

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Tentative Mapping of Transcription-Induced Interchromosomal Interaction using Chimeric EST and mRNA Data</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000254"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000254" />

    <meta name="citation_title" content="Tentative Mapping of Transcription-Induced Interchromosomal Interaction using Chimeric EST and mRNA Data"/>
    <meta itemprop="name" content="Tentative Mapping of Transcription-Induced Interchromosomal Interaction using Chimeric EST and mRNA Data"/>

      <meta name="citation_author" content="Per Unneberg"/>
            <meta name="citation_author_institution" content="Structural and Genomic Information Laboratory, Centre National de la Recherche Scientifique (CNRS) UPR-2589, Institut de Biologie Structurale et Microbiologie, Marseille, France"/>
      <meta name="citation_author" content="Jean-Michel Claverie"/>
            <meta name="citation_author_institution" content="Structural and Genomic Information Laboratory, Centre National de la Recherche Scientifique (CNRS) UPR-2589, Institut de Biologie Structurale et Microbiologie, Marseille, France"/>

    <meta name="citation_date" content="2007/2/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000254.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e254"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays.; citation_author=J Johnson; citation_author=J Castle; citation_author=P Garrett-Engele; citation_author=Z Kan; citation_author=P Loerch; citation_journal_title=Science; citation_volume=302; citation_number=1; citation_pages=2141-4; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Pseudo-messenger RNA: phantoms of the transcriptome.; citation_author=M Frith; citation_author=L Wilming; citation_author=A Forrest; citation_author=H Kawaji; citation_author=S Tan; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=2; citation_pages=e23; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A genome-wide survey demonstrates widespread non-linear mRNA in expressed sequences from multiple species.; citation_author=R Dixon; citation_author=I Eperon; citation_author=L Hall; citation_author=N Samani; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=3; citation_pages=5904-13; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Widespread occurrence of antisense transcription in the human genome.; citation_author=R Yelin; citation_author=D Dahary; citation_author=R Sorek; citation_author=E Levanon; citation_author=O Goldstein; citation_journal_title=Nat Biotechnol; citation_volume=21; citation_number=4; citation_pages=379-86; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Naturally occurring antisense: transcriptional leakage or real overlap?; citation_author=D Dahary; citation_author=O Elroy-Stein; citation_author=R Sorek; citation_journal_title=Genome Res; citation_volume=15; citation_number=5; citation_pages=364-8; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Over 20% of human transcripts might form sense-antisense pairs.; citation_author=J Chen; citation_author=M Sun; citation_author=W Kent; citation_author=X Huang; citation_author=H Xie; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=6; citation_pages=4812-20; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Non-coding RNA.; citation_author=J Mattick; citation_author=I Makunin; citation_journal_title=Hum Mol Genet; citation_volume=15 Spec No 1; citation_number=7; citation_pages=R17-29; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The transcriptional landscape of the mammalian genome.; citation_author=P Carninci; citation_author=T Kasukawa; citation_author=S Katayama; citation_author=J Gough; citation_author=M Frith; citation_journal_title=Science; citation_volume=309; citation_number=8; citation_pages=1559-63; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Fewer genes, more noncoding RNA.; citation_author=J Claverie; citation_journal_title=Science; citation_volume=309; citation_number=9; citation_pages=1529-30; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transcription-mediated gene fusion in the human genome.; citation_author=P Akiva; citation_author=A Toporik; citation_author=S Edelheit; citation_author=Y Peretz; citation_author=A Diber; citation_journal_title=Genome Res; citation_volume=16; citation_number=10; citation_pages=30-6; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Tandem chimerism as a means to increase protein complexity in the human genome.; citation_author=G Parra; citation_author=A Reymond; citation_author=N Dabbouseh; citation_author=E Dermitzakis; citation_author=R Castelo; citation_journal_title=Genome Res; citation_volume=16; citation_number=11; citation_pages=37-44; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=EST analyses predict the existence of a population of chimeric microRNA precursor-mRNA transcripts expressed in normal human and mouse tissues.; citation_author=N Smalheiser; citation_journal_title=Genome Biol; citation_volume=4; citation_number=12; citation_pages=403; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Active genes dynamically colocalize to shared sites of ongoing transcription.; citation_author=C Osborne; citation_author=L Chakalova; citation_author=K Brown; citation_author=D Carter; citation_author=A Horton; citation_journal_title=Nat Genet; citation_volume=36; citation_number=13; citation_pages=1065-71; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=CTCF mediates interchromosomal colocalization between Igf2/H19 and Wsb1/Nf1.; citation_author=J Ling; citation_author=T Li; citation_author=J Hu; citation_author=T Vu; citation_author=H Chen; citation_journal_title=Science; citation_volume=312; citation_number=14; citation_pages=269-72; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Close encounters between active genes in the nucleus.; citation_author=C Chuang; citation_author=A Belmont; citation_journal_title=Genome Biol; citation_volume=6; citation_number=15; citation_pages=237; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Intermingling of chromosome territories in interphase suggests role in translocations and transcription-dependent associations.; citation_author=M Branco; citation_author=A Pombo; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=16; citation_pages=e138; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A novel algorithm for computational identification of contaminated EST libraries.; citation_author=R Sorek; citation_author=H Safer; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=17; citation_pages=1067-74; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Natural trans-splicing in carnitine octanoyltransferase pre-mRNAs in rat liver.; citation_author=C Caudevilla; citation_author=D Serra; citation_author=A Miliar; citation_author=C Codony; citation_author=G Asins; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=18; citation_pages=12185-90; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Intergenic mRNA molecules resulting from trans-splicing.; citation_author=C Finta; citation_author=P Zaphiropoulos; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=19; citation_pages=5882-90; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=trans and cis splicing in trypanosomatids: mechanism, factors, and regulation.; citation_author=X Liang; citation_author=A Haritan; citation_author=S Uliel; citation_author=S Michaeli; citation_journal_title=Eukar Cell; citation_volume=2; citation_number=20; citation_pages=830-40; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A candidate chimeric mammalian mRNA transcript is derived from distinct chromosomes and is associated with nonconsensus splice junction motifs.; citation_author=C Zhang; citation_author=Y Xie; citation_author=J Martignetti; citation_author=T Yeo; citation_author=S Massa; citation_journal_title=DNA Cell Biol; citation_volume=22; citation_number=21; citation_pages=303-15; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Detection and analysis of spliced chimeric mRNAs in sequence databanks.; citation_author=A Romani; citation_author=E Guerra; citation_author=M Trerotola; citation_author=S Alberti; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=22; citation_pages=e17; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Finding fusion genes resulting from chromosome rearrangement by analyzing the expressed sequence databases.; citation_author=Y Hahn; citation_author=T Bera; citation_author=K Gehlhaus; citation_author=I Kirsch; citation_author=I Pastan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=23; citation_pages=13257-61; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=ChimerDB–a knowledgebase for fusion sequences.; citation_author=N Kim; citation_author=P Kim; citation_author=S Nam; citation_author=S Shin; citation_author=S Lee; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=24; citation_pages=D21-4; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcriptional maps of 10 human chromosomes at 5-nucleotide resolution.; citation_author=J Cheng; citation_author=P Kapranov; citation_author=J Drenkow; citation_author=S Dike; citation_author=S Brubaker; citation_journal_title=Science; citation_volume=308; citation_number=25; citation_pages=1149-54; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The abundance of short proteins in the mammalian proteome.; citation_author=M Frith; citation_author=A Forrest; citation_author=E Nourbakhsh; citation_author=K Pang; citation_author=C Kai; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=26; citation_pages=e52; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Interchromosomal associations between alternatively expressed loci.; citation_author=C Spilianakis; citation_author=M Lalioti; citation_author=T Town; citation_author=G Lee; citation_author=R Flavell; citation_journal_title=Nature; citation_volume=435; citation_number=27; citation_pages=637-45; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Blat–the blast-like alignment tool.; citation_author=WJ Kent; citation_journal_title=Genome Res; citation_volume=12; citation_number=28; citation_pages=656-64; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000254" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Tentative Mapping of Transcription-Induced Interchromosomal Interaction using Chimeric EST and mRNA Data"/>
    <meta name="twitter:description" content="Recent studies on chromosome conformation show that chromosomes colocalize in the nucleus, bringing together active genes in transcription factories. This spatial proximity of actively transcribing genes could provide a means for RNA interaction at the transcript level. We have screened public databases for chimeric EST and mRNA sequences with the intent of mapping transcription-induced interchromosomal interactions. We suggest that chimeric transcripts may be the result of close encounters of active genes, either as functional products or &ldquo;noise&rdquo; in the transcription process, and that they could be used as probes for chromosome interactions. We have found a total of 5,614 chimeric ESTs and 587 chimeric mRNAs that meet our selection criteria. Due to their higher quality, the mRNA findings are of particular interest and we hope that they may serve as food for thought for specialists in diverse areas of molecular biology."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000254.g004"/>

  <meta property="og:title" content="Tentative Mapping of Transcription-Induced Interchromosomal Interaction using Chimeric EST and mRNA Data" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000254" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4728'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1773'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9525&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000254">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000254" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000254&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Tentative Mapping of Transcription-Induced Interchromosomal Interaction using Chimeric EST and mRNA Data
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Per Unneberg
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:per.unneberg@igs.cnrs-mrs.fr">per.unneberg@igs.cnrs-mrs.fr</a></p>

                <p>Affiliation:
                  Structural and Genomic Information Laboratory, Centre National de la Recherche Scientifique (CNRS) UPR-2589, Institut de Biologie Structurale et Microbiologie, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean-Michel Claverie
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Structural and Genomic Information Laboratory, Centre National de la Recherche Scientifique (CNRS) UPR-2589, Institut de Biologie Structurale et Microbiologie, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000254</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000254" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000254" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000254" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000254" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000254-g001" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000254-t001" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000254-g002" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000254-g003" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000254-t002" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000254-g004" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000254-t003" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000254-t004" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000254">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000254" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Recent studies on chromosome conformation show that chromosomes colocalize in the nucleus, bringing together active genes in transcription factories. This spatial proximity of actively transcribing genes could provide a means for RNA interaction at the transcript level. We have screened public databases for chimeric EST and mRNA sequences with the intent of mapping transcription-induced interchromosomal interactions. We suggest that chimeric transcripts may be the result of close encounters of active genes, either as functional products or “noise” in the transcription process, and that they could be used as probes for chromosome interactions. We have found a total of 5,614 chimeric ESTs and 587 chimeric mRNAs that meet our selection criteria. Due to their higher quality, the mRNA findings are of particular interest and we hope that they may serve as food for thought for specialists in diverse areas of molecular biology.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Unneberg P, Claverie J-M (2007) Tentative Mapping of Transcription-Induced Interchromosomal Interaction using Chimeric EST and mRNA Data. PLoS ONE 2(2):
          e254.
            doi:10.1371/journal.pone.0000254</p><p><strong>Academic Editor: </strong>Jörg Hoheisel, Deutsches Krebsforschungszentrum, Germany</p><p><strong>Received:</strong> December 15, 2006; <strong>Accepted:</strong> February 6, 2007; <strong>Published:</strong> February 28, 2007</p><p><strong>Copyright:</strong> © 2007 Unneberg, Claverie. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Per Unneberg was supported by the Swedish Research Council.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>With the development of increasingly sophisticated large-scale sequencing and microarray techniques, the known transcriptome continues to grow. Even though the idea that one gene produces one protein product has long been abandoned for more complicated models, our understanding of transcription remains incomplete and subject to unexpected findings.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Analyses of transcript databases via bioinformatic approaches have described and uncovered numerous transcript classes. As an example, alternative splicing allows the cell to increase protein diversity – according to current estimates, 75–80% of human genes produce splice variants <a href="#pone.0000254-Johnson1">[1]</a>. Pseudo-messenger RNAs, in the form of expressed pseudogenes or disrupted splice variants with retained introns, have recently been described <a href="#pone.0000254-Frith1">[2]</a>. Tandem duplication of exons generates non-linear mRNA transcripts <a href="#pone.0000254-Dixon1">[3]</a>. Antisense transcription attests the existence of overlapping gene loci in eukaryotic genomes <a href="#pone.0000254-Yelin1">[4]</a>–<a href="#pone.0000254-Chen1">[6]</a>. Finally, the never-ending identification of various types of non-protein-coding RNAs (ncRNAs) continues to increase the size and complexity of the transcriptome <a href="#pone.0000254-Mattick1">[7]</a>–<a href="#pone.0000254-Claverie1">[9]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>A recent addition to transcript diversity is transcription induced chimerism (TIC) <a href="#pone.0000254-Akiva1">[10]</a>, <a href="#pone.0000254-Parra1">[11]</a>. Here, tandem gene pairs are transcribed into one chimeric transcript, thus generating a fusion protein. The functional role of these proteins remains unclear, but since at least 4–5% tandem gene pairs form chimeric transcripts <a href="#pone.0000254-Parra1">[11]</a>, it is not a singular event. Chimeric microRNA precursor messenger RNA (mRNA) transcripts have also recently been described <a href="#pone.0000254-Smalheiser1">[12]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In addition to the bioinformatical approaches, new cellular biology techniques are providing new insights in the 3-dimensional and topological properties of the transcription process. The concept of transcription factories – foci for nascent RNA and RNA polymerase II – has been proposed for some time. According to this model, as a gene is activated for transcription, the production of its mRNA takes place in such a transcription factory. Several active genes may occupy the same factory <a href="#pone.0000254-Osborne1">[13]</a>. Lately, it has been shown that even genes from different chromosomes may interact simultaneously in one transcription factory <a href="#pone.0000254-Ling1">[14]</a>, <a href="#pone.0000254-Chuang1">[15]</a>. In fact, the chromosome is now known to be a highly mobile structure, with territories intermingling significantly in the nucleus <a href="#pone.0000254-Branco1">[16]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>It is generally acknowledged that transcript databases in general, and EST databases in particular, contain contaminants of various kinds <a href="#pone.0000254-Sorek1">[17]</a>. Chimeric sequences is one such example. Traditionally, chimeric transcripts have been discarded as artefacts, primarily causing problems in annotation and gene indexing/clustering projects. However, in the light of the radically new findings mentioned previously, it might have been premature to dismiss all aberrant transcripts as artefacts. For instance, <em>trans</em>-splicing <a href="#pone.0000254-Caudevilla1">[18]</a>–<a href="#pone.0000254-Liang1">[20]</a>, the event of joining the exons of two heterologous transcripts, provides a mechanism for the generation of chimeric transcripts. There are few estimates of the frequency of <em>trans</em>-splicing events; one study observed ≈0.15% <a href="#pone.0000254-Finta1">[19]</a>, indicating it is a very rare event. Since <em>trans</em>-splicing products are generated by the spliceosome machinery, there has been a focus on splicing that occurs at consensus splice sites. However, a study of a rat mRNA, Leukocyte Common Antigen-Related (LAR) tyrosine phosphatase receptor, has indicated the existence of a chimeric transcript with non-consensus splice donor and acceptor sequences <a href="#pone.0000254-Zhang1">[21]</a>. More generally, all kinds of chimeric transcripts could be the product of the normal cell transcriptional process, not discarding the possibility that they may constitute the “noise” of this process, and thus represent <em>bona fide</em> biological artefacts, with or without function.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Chimeric cDNA clones, in that context, might be the unavoidable price to pay for the implementation of local, interchromosomal, gene-coding regulatory processes (<a href="#pone-0000254-g001">Figure 1</a>). Functional or not, the chimeric mRNAs might at least serve as probes for regions of chromosomal interaction taking place in transcription factories. To tentatively map such putative regions of interchromosomal contact, we revisited the specific task of identifying chimeric sequences in transcript databases. Similar procedures have been performed earlier <a href="#pone.0000254-Romani1">[22]</a>, <a href="#pone.0000254-Hahn1">[23]</a>, focusing on finding <em>bona fide</em> transcripts with exon/exon fusion boundaries. Here, we concentrate on the remaining cases, using the latest human reference genome build.</p>
<div class="figure" id="pone-0000254-g001"><div class="img"><a name="pone-0000254-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000254.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000254.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Chimeric mRNA revealing chromsome interaction.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Schematic representation of regions of two chromosomes, represented by red and blue thick lines, with accompanying mRNA transcripts in corresponding colors, represented by wavy lines, transcription factories and RNA polymerases. When chromosomes are not in proximity, mRNAs are less likely to interact (A), whereas proximal chromosomes generate a chimeric mRNA, revealing interchromosomal interaction (B).</p>
<span>doi:10.1371/journal.pone.0000254.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Validation</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A recent publication describes ChimerDB, a database that catalogues EST and mRNA fusion sequences in GenBank <a href="#pone.0000254-Kim1">[24]</a>. In this database only fusion events at exon-exon borders are considered. A natural way to validate our procedure is to compare our chimeric sequences with those found in the database. In the current version (0.8), which is based on NCBI Genome Build version 35, there are 194 EST and 137 mRNA interchromosomal chimeras, respectively. We found 88 (45.1%) of the EST chimeras, and 94 (67.1%) of the mRNA chimeras. The selection criteria used in ChimerDB were alignments of at least 100 bp and 93% identity over the entire query; this difference in imposed criteria turned out to be the main reason for the missed chimeras in the EST case. On the other hand, the missed mRNA chimeras were mainly due to the fact that the ChimerDB chimeras failed to comply with our uniqueness criterion.</p>


<h4>EST mapping results</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We mapped 7684642 EST sequences to the human genome reference sequence. 5702 ESTs passed the selection criteria outlined in Methods; 5614 remained after removing the already known fusion sequences found in the validation step. The mean chimeric EST length was 507 bp (SD 131), and in total 2844067 bp were mapped.</p>


<h4>Interaction characteristics</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>A summary of the partner interactions at the locus level is displayed in <a href="#pone-0000254-t001">Table 1</a>. Despite the fact that EST sequences are the basis for the observations, 1546 (27.8%) of the interactions have an intergenic component (<a href="#pone-0000254-t001">Table 1 <strong>A</strong></a>). Possible explanations for this are either that the EST library was contaminated with genomic DNA, or that the corresponding intergenic transcripts are not yet annotated or characterized (as were most of them until recently) <a href="#pone.0000254-Carninci1">[8]</a>, <a href="#pone.0000254-Cheng1">[25]</a>. The latter case would correspond to our hypothesis of transcription-induced interchromosomal interaction.</p>
<div class="figure" id="pone-0000254-t001"><div class="img"><a name="pone-0000254-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000254.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000254.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Observed chimeric partner interactions.</span></strong></p><span>doi:10.1371/journal.pone.0000254.t001</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The 4050 remaining cases (72.3%) represent potential gene–gene interactions. The orientation could be determined for both partners in 3621 chimeras, out of which 2651 consisted of partners that were oriented in the same direction. <a href="#pone-0000254-g002">Figure 2</a> shows the frequency of chromosome interactions for this subset, where the size of each square is proportional to the number of times a given chromosome–chromosome pair is observed. Plotted above and to the right of the cell frequencies are the gene frequencies for each chromosome. The gene frequencies tell us what to expect if the associations between chromosomes are non-preferential. For instance, since the number of genes is highest on chromosome 1, followed by chromosome 2, one would expect the highest number of observed interactions between these chromosomes; this indeed is the case. The correlation between the observed values and expected values as calculated with gene frequencies is 0.72, confirming the non-preferential nature of the observed associations.</p>
<div class="figure" id="pone-0000254-g002"><div class="img"><a name="pone-0000254-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000254.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000254.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Gene interaction plot.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Mosaic plot of gene interactions for 2651 EST chimeras where the direction of the participating partners is the same. The size of each square is proportional to the number of times a fusion event is observed between chromosomes <em>i</em> and <em>j</em>, for <em>i</em>,<em>j</em>Є1,2,…,22,<em>X</em>,<em>Y</em>. The barplots represent known gene densities on each chromosome, according to Ensembl gene counts for all chromosomes.</p>
<span>doi:10.1371/journal.pone.0000254.g002</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The ESTs with gene partners having the same direction are potential sources of <em>trans</em>-splicing observations if the fusion point occurred at exon boundaries for both partners. Exon boundaries could be uniquely defined for both partners in 1852 cases. Accepting a distance of 10 bp from a boundary, we observed only 20 potential <em>trans</em>-splicing cases. Consequently, <em>trans</em>-splicing is not a main reason for EST chimera generation in this data set.</p>


<h4>EST library distribution</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>At the time of analysis, the human EST sequences could be derived from 8618 EST libraries. The 5614 EST sequences that passed the validation criteria originated from 1537 libraries, with 756 libraries contributing one sequence. <a href="#pone-0000254-g003">Figure 3</a> illustrates a barplot of EST library counts for libraries with more than 10 contributing sequences.</p>
<div class="figure" id="pone-0000254-g003"><div class="img"><a name="pone-0000254-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000254.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000254.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Chimera distributions by EST library.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Distribution of chimeric observations grouped by dbEST library id. Only libraries with more than 10 observations are shown.</p>
<span>doi:10.1371/journal.pone.0000254.g003</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The most commonly observed library, NCI_CGAP_GC6 (dbEST library id 1402), contributes 222 chimeric sequences, followed by libraries Soares_NFL_T_GBC_S1 (library id 1042) and Fetal brain, Stratagene (cat#936206) (library id 2). <a href="#pone-0000254-t002">Table 2</a> lists the EST libraries with &gt;40 chimeric sequences. Libraries displaying high chimera counts may point to problems in library construction, rather than reflect biologically relevant observations. Contaminated libraries have been analysed by Sorek and Safer (2003), but none of the libraries listed in <a href="#pone-0000254-t002">Table 2</a> were identified in that study. Nevertheless, it remains possible that the high number of chimeras observed in these libraries is a sign of artefacts in library construction.</p>
<div class="figure" id="pone-0000254-t002"><div class="img"><a name="pone-0000254-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000254.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000254.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>EST libraries with more than 40 observations.</span></strong></p><span>doi:10.1371/journal.pone.0000254.t002</span></div>

<h4>mRNA mapping results</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Out of 200033 mapped mRNA sequence, 681 passed the selection criteria (<a href="#pone.0000254.s001">Table <strong>S</strong>1</a>), with 587 remaining after removal of known fusion events in the validation phase. In total, 1515785 bp were mapped, with a mean length of 2582 bp (SD 1349).</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The random fusion event of two gene-coding mRNA transcripts is likely to alter the reading frame properties of the fusion partners. Even if the fusion occurs at exon-exon boundaries, it is possible that an internal stop codon be introduced in the resulting fusion transcript due to frameshifts. On the other hand, if the fusion occurs between untranslated regions, the open reading frame (ORF) in either partner may remain unaltered. It is often assumed that functional peptides are longer than 100 amino acid residues, although short proteins (e.g. ribosomal proteins) have been shown to be common in the mammalian proteome <a href="#pone.0000254-Frith2">[26]</a>. 493 (84%) mRNA transcripts have open reading frame lengths ≥300 bp; 265 (45%) have lengths ≥900 bp. Consequently, the majority of mRNA chimeric transcripts code for peptides long enough to be assumed functional.</p>


<h4>Fusion point location in chimera</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Apart from reading frame length, the likelihood that a fusion event will affect reading frame characteristics is also dependant on the location of the fusion point. Approximately half (3069 or 54.7%) of the EST chimeric sequences had ORFs that overlapped the fusion point, compared to 27.3% (160 out of 587) for the mRNA chimeras. Due to the imposed selection criteria, the fusion point will be located at least 100 bp from either end of a chimeric sequence. For the shorter EST sequences, it is likely that the fusion point will be distributed halfway through the sequence. However, the mRNAs are full-length sequences, and if the fusion of two sequences occurs in a non-random fashion, the location of the fusion point might also display a non-random pattern.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>To investigate this hypothesis, we examined the distribution of the fusion point location. Denote by <em>X</em> the fusion point location, and by <em>L</em> the sequence length. Since we had imposed that each chimeric partner be at least 100 bp long, <em>X</em>Є(100,<em>L</em>−100). By letting <em>Y</em> = max(<em>X</em>,1−<em>X</em>)/(<em>L</em>−200), <em>Y</em> will be uniformly distributed in the range (0.5,1), with a mean value 0.75. <a href="#pone-0000254-g004">Figure 4</a> shows boxplots of the distribution of <em>Y</em> for EST and mRNA sequences. Both plots indicate that the fusion point is randomly distributed along sequence length.</p>
<div class="figure" id="pone-0000254-g004"><div class="img"><a name="pone-0000254-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000254.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000254.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Boxplots of fusion location.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Distribution of fusion location for 5614 EST and 587 mRNA chimeras. The fusion location is represented as a fraction <em>X</em> of sequence length. Fractions <em>Y</em>&lt;0.5 have been transformed to <em>X</em> = 1−<em>Y</em>.</p>
<span>doi:10.1371/journal.pone.0000254.g004</span></div>

<h4>Interaction examples</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>The size of each square in the mosaic plot (<a href="#pone-0000254-g002">Figure 2</a>) is proportional to the number of times an EST chimera consists of partners from two given chromosomes. Taking this a step further, a close-up view of each chromosome interaction also provides information about how many times, for instance, a given gene-gene interaction occurs. The simplest way to accomplish this task would be to examine the Ensembl gene ids of the partners in a chimera and count the number of times every pair occurs. However, this procedure would miss unannotated intergenic regions, which nevertheless may be transcribed. Thus, an alternative approach is to determine the mapping (i.e. chromosome start and stop coordinates) for each partner in a chimera. Consequently, all sequences that have chromosome coordinates that overlap are said to have the same mapping.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>The majority of EST chimera interactions (4774 out of 5630) are observed only once. However, singleton interactions do not necessarily imply artefactual origin <a href="#pone.0000254-Parra1">[11]</a>, but may reflect low expression levels. There are 12 interactions that are observed ≥5 times (<a href="#pone-0000254-t003">Table 3</a>). Here, except for two cases, all chimeric partners are mapped to genes. Although these are multiple observations, in each case all sequences are from one laboratory, most often also deriving from one EST library. Therefore, it is difficult to ascertain the significance of these findings since there are no independent observations. Due to the smaller mRNA data set, there are only 16 interactions with ≥2 observations (<a href="#pone-0000254-t004">Table 4</a>), of which 5 have an intergenic partner. As pointed out previously, intergenic sequences might indicate that the sequence is contaminated with genomic DNA <a href="#pone.0000254-Sorek1">[17]</a>. However, it is unlikely that a randomly chosen intergenic region would show up more than once in such a small sample that the mRNA data set represents. The interactions IG–GRM7, IG–CDH13, Q96PV3_HUMAN–CK016_HUMAN, SLIT3–MAGED2, and MLL–AFF1, have furthermore been independently observed. Here, it should be remarked that the interactions marked with asterisks are characterized as fusion sequence, although they were not included in ChimerDB. This highlights the difficulty of distinguishing between transcripts that are derived from potential interactions at the transcript level and transcripts that arise from chromosome translocations.</p>
<div class="figure" id="pone-0000254-t003"><div class="img"><a name="pone-0000254-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000254.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000254.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Most common interactions for EST data.</span></strong></p><span>doi:10.1371/journal.pone.0000254.t003</span></div><div class="figure" id="pone-0000254-t004"><div class="img"><a name="pone-0000254-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000254" data-uri="info:doi/10.1371/journal.pone.0000254.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000254.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000254.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000254.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000254.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000254.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Most common interactions for mRNA data.</span></strong></p><span>doi:10.1371/journal.pone.0000254.t004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have made a tentative mapping of EST and mRNA sequences to the human genome in the hope of identifying potential gene-gene or locus-locus interactions. Recent findings have shown that interchromosomal interactions upon transcription take place in transcription factories in the nucleus <a href="#pone.0000254-Ling1">[14]</a>–<a href="#pone.0000254-Branco1">[16]</a>, <a href="#pone.0000254-Spilianakis1">[27]</a>. As a result, heterologous genes, and consequently transcripts, are colocalized in the nucleus, thereby providing the spatial proximity for possible transcript interaction. Given the recent discoveries concerning transcriptome complexity <a href="#pone.0000254-Dixon1">[3]</a>, <a href="#pone.0000254-Carninci1">[8]</a>, <a href="#pone.0000254-Akiva1">[10]</a>, novel analyses of old transcript data still provide a means for discovering new features in the transcriptome. The focus on canonical splice sites will identify <em>trans</em>-splicing events, but any possible alternative mechanism of RNA interaction will go undetected. For this reason, we have applied a search for chimeric sequences without any a priori assumptions about the nature of RNA interactions.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>At a first glance, the fact that we observe several interactions more than once in both the EST and mRNA case, would suggest that there is independent evidence corroborating our hypothesis. There are 856 EST sequences that can be grouped into 366 interactions, whereas the majority of interactions are observed once (4758 cases). As shown in <a href="#pone-0000254-t003">Table 3</a>, multiple interaction observations often seem to originate from one or a few libraries. In fact, the 366 interaction groups correspond only to 433 libraries, indicating that multiple observations of an interaction come from the same library. Consequently, there are few independent experiments that validate the multiple observations, thereby questioning their significance.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The large number of rare events may however have other explanations. First, due to the nature of cDNA library construction, EST libraries mostly originate from polyadenylated sequences - the latter may consist of as little as half of all transcribed sequences <a href="#pone.0000254-Cheng1">[25]</a>. Therefore, a large amount of transcripts will be missing from these EST libraries. Second, the choice of mapping parameters has a paramount effect on the number of observed chimeras. EST data are known to contain contaminants and may have sequence error rates as high as 3%. This, in combination with the high repetitive content of the human genome, makes unambiguous mapping difficult, especially for short sequences. Third, EST sequence length limits the number of observed EST chimeras for another reason. Even if a clone is chimeric, the fusion point would, in our settings, have to be located at most 5–600 bp from either clone end (average EST length is 533 bp), and at least 100 bp from either end of the EST sequence. Indeed, the proportion <em>p</em> of chimeric sequences is significantly higher in the mRNA data set (<em>p<sub>mRNA</sub></em> = 2.9×10<sup>−3</sup> vs <em>p<sub>EST</sub></em> = 7.3×10<sup>−4</sup>, z-test, p-value&lt;2.2×10<sup>−16</sup>). A change in filtering settings would affect both data sets, with the difference remaining, and it is likely that this difference can be attributed to sequence length (average mRNA length 1784 bp).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>While it is expected that EST sequences are prone to artefacts and contamination problems, the mRNA data set consists of assembled sequences from full-length, cDNA clones, with a higher sequence quality than that of the EST sequences. Still, some 600 intergenic mRNA sequences have been found, of which most are not examples of regular splicing events. The majority have ORFs ≥300 bp; however, only a quarter of ORFs overlap the fusion point, meaning three out of four fusions do not alter reading frame and protein characteristics.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>This work presents the first tentative mapping of interchromosomal interactions using EST and mRNA data. The essence of this work is contained in the list of putative locus-locus interactions shown in <a href="#pone.0000254.s001">Table <strong>S</strong>1</a>. This should be considered as a resource on future work on the biological significance of this phenomenon. It is our hope that some of these putative chromosomal interactions might correlate with interesting phenotypes, related to such diverse topics as cytogenetic aberrations in tumours, mutational and recombinational hotspots, and disease-related chromosomal regions. These correlations could be spotted by the trained eye of specialists in their respective fields of research. In addition to the interaction resource, we propose a novel type of transcriptome component that could be derived from the juxtaposition of two regions of different chromosomes. Whether the resulting chimeric transcript solely reports an interaction, or has a specific function, remains to be assessed.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data sets</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p><em>Homo sapiens</em> sequence data was taken from GenBank, release 153 (<a href="ftp://ftp.ncbi.nih.gov/genbank/">ftp://ftp.ncbi.nih.gov/genbank/</a>). The EST division consisted of 7684642 sequences; 200033 mRNA sequences were obtained from the PRI and HTC divisions. The human genome reference build, release 36.1, was used for the alignments (<a href="ftp://ftp.ncbi.nih.gov/genomes/H_sapiens/Assembled_chromosomes">ftp://ftp.ncbi.nih.gov/genomes/H_sapiens​/Assembled_chromosomes</a>). The EST and mRNA data sets were aligned to the human genome reference build, using BLAT <a href="#pone.0000254-Kent1">[28]</a>. The Ensembl database <em>homo_sapiens_core_40_36b</em> was downloaded from <a href="ftp://ftp.ensembl.org/pub/current_homo_sapiens/data/mysql/homo_sapiens_core_40_36b">ftp://ftp.ensembl.org/pub/current_homo_s​apiens/data/mysql/homo_sapiens_core_40_3​6b</a>. Finally, dbEST report files were downloaded from <a href="ftp://ftp.ncbi.nih.gov/repository/dbEST/">ftp://ftp.ncbi.nih.gov/repository/dbEST/</a>.</p>


<h4>Selection of chimeric transcripts</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Simply put, the selection of chimeric sequences consists of selecting queries (EST or mRNA) that map to loci on two different chromosomes. For each query, all alignments to distinct chromosomes with alignment length ≥100 base pairs (bp) and identity ≥95% were collected and processed. Alignments were sorted according to score, and a transcript was reconstructed by fusing two partner sequences which are defined by the query regions of the two best alignments. A 10 bp overlap between the partner sequences was allowed at the fusion point to account for alignment uncertainties. A query was classified as a chimeric transcript if the number of identities in the two best alignments together constituted ≥95% of the query length.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>In addition, the following uniqueness criterion was imposed to make sure that the two best alignments were unambiguous: for a query with three or more alignments, the alignments ranking third or worse were compared with the second best alignment (constituting the “short” partner of the reconstructed transcript). If the ratio of correctly aligned bases between any such alignment and the second alignment was &gt;0.8 <strong>and</strong> the subject regions didn't overlap, then that part was considered to be ambiguously mapped. As a final quality check, the chimera was discarded from further analysis if a restriction recognition site for the enzyme used in library construction was found at the fusion point.</p>


<h4>Classification of interactions</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>By our selection criteria, the two partners of a chimeric sequence map to loci on two different chromosomes. As a consequence, a chimeric transcript can be viewed as the result of a potential transcriptional interaction between interchromosomal loci. For instance, a potential gene-gene interaction is observed if both partners of a chimeric sequence map to genes. Comparison with the Ensembl database allowed us to classify the fusion point in both partners according to whether they mapped to a gene or an intergenic region, denoted as <strong>GENE</strong> and <strong>IG</strong>, respectively (<a href="#pone-0000254-t001">Table 1 <strong>A</strong></a>). Moreover, mappings to genes could be classified as <strong>EXON</strong> or <strong>INTRON</strong> if the corresponding Ensembl gene contained transcript(s) with exon/intron information (<a href="#pone-0000254-t001">Table 1 <strong>B</strong></a>). Finally, if also CDS information existed for the Ensembl gene, for a partner mapping to an exon the fusion point could be further classified as <strong>5′UTR</strong>, <strong>3′UTR</strong> or <strong>CDS</strong> (<a href="#pone-0000254-t001">Table 1 <strong>C</strong></a>). Alignment of a sequence to gene regions with known transcripts also enabled the inference of sequence orientation.</p>


<h4>Data analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We stored EST and mRNA sequences in a MySQL database using the BioSQL schema (<a href="http://obda.open-bio.org">http://obda.open-bio.org</a>), adding extra tables for alignment results and dbEST clone library information. The Ensembl database <em>homo_sapiens_core_40_36b</em> was installed for the classification of chimera partners and fusion point. Statistical analyses were performed using the software R.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000254"></div><a name="pone.0000254.s001" id="pone.0000254.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000254.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Mapping results for mRNA sequences. The symbol IG indicates an alignment to an intergenic region.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000254.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.92 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Pascal Hingcamp for critical reading of the manuscript and valuable suggestions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JC PU. Performed the experiments: PU. Analyzed the data: JC PU. Wrote the paper: JC PU.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000254-Johnson1" id="pone.0000254-Johnson1"></a>Johnson J, Castle J, Garrett-Engele P, Kan Z, Loerch P, et al.  (2003) Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays. Science  302: 2141–4.  <ul class="find" data-citedArticleID="934072" data-doi="10.1126/science.1090100"><li><a href="http://dx.doi.org/10.1126/science.1090100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+survey+of+human+alternative+pre-mRNA+splicing+with+exon+junction+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+survey+of+human+alternative+pre-mRNA+splicing+with+exon+junction+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000254-Frith1" id="pone.0000254-Frith1"></a>Frith M, Wilming L, Forrest A, Kawaji H, Tan S, et al.  (2006) Pseudo-messenger RNA: phantoms of the transcriptome. PLoS Genet  2: e23.  <ul class="find" data-citedArticleID="934066" data-doi="10.1371/journal.pgen.0020023"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pseudo-messenger+RNA%3A+phantoms+of+the+transcriptome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pseudo-messenger+RNA%3A+phantoms+of+the+transcriptome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000254-Dixon1" id="pone.0000254-Dixon1"></a>Dixon R, Eperon I, Hall L, Samani N (2005) A genome-wide survey demonstrates widespread non-linear mRNA in expressed sequences from multiple species. Nucleic Acids Res  33: 5904–13.  <ul class="find" data-citedArticleID="934062" data-doi="10.1093/nar/gki893"><li><a href="http://dx.doi.org/10.1093/nar/gki893" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome-wide+survey+demonstrates+widespread+non-linear+mRNA+in+expressed+sequences+from+multiple+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome-wide+survey+demonstrates+widespread+non-linear+mRNA+in+expressed+sequences+from+multiple+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000254-Yelin1" id="pone.0000254-Yelin1"></a>Yelin R, Dahary D, Sorek R, Levanon E, Goldstein O, et al.  (2003) Widespread occurrence of antisense transcription in the human genome. Nat Biotechnol  21: 379–86.  <ul class="find" data-citedArticleID="934096" data-doi="10.1038/nbt808"><li><a href="http://dx.doi.org/10.1038/nbt808" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Widespread+occurrence+of+antisense+transcription+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Widespread+occurrence+of+antisense+transcription+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000254-Dahary1" id="pone.0000254-Dahary1"></a>Dahary D, Elroy-Stein O, Sorek R (2005) Naturally occurring antisense: transcriptional leakage or real overlap? Genome Res  15: 364–8.  <ul class="find" data-citedArticleID="934060" data-doi="10.1101/gr.3308405"><li><a href="http://dx.doi.org/10.1101/gr.3308405" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Naturally+occurring+antisense%3A+transcriptional+leakage+or+real+overlap%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Naturally+occurring+antisense%3A+transcriptional+leakage+or+real+overlap%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000254-Chen1" id="pone.0000254-Chen1"></a>Chen J, Sun M, Kent W, Huang X, Xie H, et al.  (2004) Over 20% of human transcripts might form sense-antisense pairs. Nucleic Acids Res  32: 4812–20.  <ul class="find" data-citedArticleID="934052" data-doi="10.1093/nar/gkh818"><li><a href="http://dx.doi.org/10.1093/nar/gkh818" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Over+20%25+of+human+transcripts+might+form+sense-antisense+pairs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Over+20%25+of+human+transcripts+might+form+sense-antisense+pairs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000254-Mattick1" id="pone.0000254-Mattick1"></a>Mattick J, Makunin I (2006) Non-coding RNA. Hum Mol Genet  15 Spec No 1: R17–29.  <ul class="find" data-citedArticleID="934082"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Non-coding+RNA.&amp;auth=&amp;atitle=Non-coding+RNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-coding+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-coding+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000254-Carninci1" id="pone.0000254-Carninci1"></a>Carninci P, Kasukawa T, Katayama S, Gough J, Frith M, et al.  (2005) The transcriptional landscape of the mammalian genome. Science  309: 1559–63.  <ul class="find" data-citedArticleID="934048"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+transcriptional+landscape+of+the+mammalian+genome.&amp;auth=&amp;atitle=The+transcriptional+landscape+of+the+mammalian+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcriptional+landscape+of+the+mammalian+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcriptional+landscape+of+the+mammalian+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000254-Claverie1" id="pone.0000254-Claverie1"></a>Claverie J (2005) Fewer genes, more noncoding RNA. Science  309: 1529–30.  <ul class="find" data-citedArticleID="934058" data-doi="10.1126/science.1116800"><li><a href="http://dx.doi.org/10.1126/science.1116800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fewer+genes%2C+more+noncoding+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fewer+genes%2C+more+noncoding+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000254-Akiva1" id="pone.0000254-Akiva1"></a>Akiva P, Toporik A, Edelheit S, Peretz Y, Diber A, et al.  (2006) Transcription-mediated gene fusion in the human genome. Genome Res  16: 30–6.  <ul class="find" data-citedArticleID="934044" data-doi="10.1101/gr.4137606"><li><a href="http://dx.doi.org/10.1101/gr.4137606" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription-mediated+gene+fusion+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription-mediated+gene+fusion+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000254-Parra1" id="pone.0000254-Parra1"></a>Parra G, Reymond A, Dabbouseh N, Dermitzakis E, Castelo R, et al.  (2006) Tandem chimerism as a means to increase protein complexity in the human genome. Genome Res  16: 37–44.  <ul class="find" data-citedArticleID="934086" data-doi="10.1101/gr.4145906"><li><a href="http://dx.doi.org/10.1101/gr.4145906" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tandem+chimerism+as+a+means+to+increase+protein+complexity+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tandem+chimerism+as+a+means+to+increase+protein+complexity+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000254-Smalheiser1" id="pone.0000254-Smalheiser1"></a>Smalheiser N (2003) EST analyses predict the existence of a population of chimeric microRNA precursor-mRNA transcripts expressed in normal human and mouse tissues. Genome Biol  4: 403.  <ul class="find" data-citedArticleID="934090"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=EST+analyses+predict+the+existence+of+a+population+of+chimeric+microRNA+precursor-mRNA+transcripts+expressed+in+normal+human+and+mouse+tissues.&amp;auth=&amp;atitle=EST+analyses+predict+the+existence+of+a+population+of+chimeric+microRNA+precursor-mRNA+transcripts+expressed+in+normal+human+and+mouse+tissues." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EST+analyses+predict+the+existence+of+a+population+of+chimeric+microRNA+precursor-mRNA+transcripts+expressed+in+normal+human+and+mouse+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EST+analyses+predict+the+existence+of+a+population+of+chimeric+microRNA+precursor-mRNA+transcripts+expressed+in+normal+human+and+mouse+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000254-Osborne1" id="pone.0000254-Osborne1"></a>Osborne C, Chakalova L, Brown K, Carter D, Horton A, et al.  (2004) Active genes dynamically colocalize to shared sites of ongoing transcription. Nat Genet  36: 1065–71.  <ul class="find" data-citedArticleID="934084" data-doi="10.1038/ng1423"><li><a href="http://dx.doi.org/10.1038/ng1423" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Active+genes+dynamically+colocalize+to+shared+sites+of+ongoing+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Active+genes+dynamically+colocalize+to+shared+sites+of+ongoing+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000254-Ling1" id="pone.0000254-Ling1"></a>Ling J, Li T, Hu J, Vu T, Chen H, et al.  (2006) CTCF mediates interchromosomal colocalization between Igf2/H19 and Wsb1/Nf1. Science  312: 269–72.  <ul class="find" data-citedArticleID="934080" data-doi="10.1126/science.1123191"><li><a href="http://dx.doi.org/10.1126/science.1123191" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CTCF+mediates+interchromosomal+colocalization+between+Igf2%2FH19+and+Wsb1%2FNf1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CTCF+mediates+interchromosomal+colocalization+between+Igf2%2FH19+and+Wsb1%2FNf1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000254-Chuang1" id="pone.0000254-Chuang1"></a>Chuang C, Belmont A (2005) Close encounters between active genes in the nucleus. Genome Biol  6: 237.  <ul class="find" data-citedArticleID="934056"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Close+encounters+between+active+genes+in+the+nucleus.&amp;auth=&amp;atitle=Close+encounters+between+active+genes+in+the+nucleus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Close+encounters+between+active+genes+in+the+nucleus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Close+encounters+between+active+genes+in+the+nucleus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000254-Branco1" id="pone.0000254-Branco1"></a>Branco M, Pombo A (2006) Intermingling of chromosome territories in interphase suggests role in translocations and transcription-dependent associations. PLoS Biol  4: e138.  <ul class="find" data-citedArticleID="934046" data-doi="10.1371/journal.pbio.0040138"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intermingling+of+chromosome+territories+in+interphase+suggests+role+in+translocations+and+transcription-dependent+associations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intermingling+of+chromosome+territories+in+interphase+suggests+role+in+translocations+and+transcription-dependent+associations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000254-Sorek1" id="pone.0000254-Sorek1"></a>Sorek R, Safer H (2003) A novel algorithm for computational identification of contaminated EST libraries. Nucleic Acids Res  31: 1067–74.  <ul class="find" data-citedArticleID="934092" data-doi="10.1093/nar/gkg170"><li><a href="http://dx.doi.org/10.1093/nar/gkg170" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+algorithm+for+computational+identification+of+contaminated+EST+libraries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+algorithm+for+computational+identification+of+contaminated+EST+libraries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000254-Caudevilla1" id="pone.0000254-Caudevilla1"></a>Caudevilla C, Serra D, Miliar A, Codony C, Asins G, et al.  (1998) Natural trans-splicing in carnitine octanoyltransferase pre-mRNAs in rat liver. Proc Natl Acad Sci U S A  95: 12185–90.  <ul class="find" data-citedArticleID="934050" data-doi="10.1073/pnas.95.21.12185"><li><a href="http://dx.doi.org/10.1073/pnas.95.21.12185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+trans-splicing+in+carnitine+octanoyltransferase+pre-mRNAs+in+rat+liver." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+trans-splicing+in+carnitine+octanoyltransferase+pre-mRNAs+in+rat+liver.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000254-Finta1" id="pone.0000254-Finta1"></a>Finta C, Zaphiropoulos P (2002) Intergenic mRNA molecules resulting from trans-splicing. J Biol Chem  277: 5882–90.  <ul class="find" data-citedArticleID="934064" data-doi="10.1074/jbc.m109175200"><li><a href="http://dx.doi.org/10.1074/jbc.m109175200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intergenic+mRNA+molecules+resulting+from+trans-splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intergenic+mRNA+molecules+resulting+from+trans-splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000254-Liang1" id="pone.0000254-Liang1"></a>Liang X, Haritan A, Uliel S, Michaeli S (2003) <em>trans</em> and <em>cis</em> splicing in trypanosomatids: mechanism, factors, and regulation. Eukar Cell  2: 830–40.  <ul class="find" data-citedArticleID="934078" data-doi="10.1128/ec.2.5.830-840.2003"><li><a href="http://dx.doi.org/10.1128/ec.2.5.830-840.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=trans+and+cis+splicing+in+trypanosomatids%3A+mechanism%2C+factors%2C+and+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22trans+and+cis+splicing+in+trypanosomatids%3A+mechanism%2C+factors%2C+and+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000254-Zhang1" id="pone.0000254-Zhang1"></a>Zhang C, Xie Y, Martignetti J, Yeo T, Massa S, et al.  (2003) A candidate chimeric mammalian mRNA transcript is derived from distinct chromosomes and is associated with nonconsensus splice junction motifs. DNA Cell Biol  22: 303–15.  <ul class="find" data-citedArticleID="934098" data-doi="10.1089/104454903322216653"><li><a href="http://dx.doi.org/10.1089/104454903322216653" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+candidate+chimeric+mammalian+mRNA+transcript+is+derived+from+distinct+chromosomes+and+is+associated+with+nonconsensus+splice+junction+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+candidate+chimeric+mammalian+mRNA+transcript+is+derived+from+distinct+chromosomes+and+is+associated+with+nonconsensus+splice+junction+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000254-Romani1" id="pone.0000254-Romani1"></a>Romani A, Guerra E, Trerotola M, Alberti S (2003) Detection and analysis of spliced chimeric mRNAs in sequence databanks. Nucleic Acids Res  31: e17.  <ul class="find" data-citedArticleID="934088" data-doi="10.1093/nar/gng017"><li><a href="http://dx.doi.org/10.1093/nar/gng017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+and+analysis+of+spliced+chimeric+mRNAs+in+sequence+databanks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+and+analysis+of+spliced+chimeric+mRNAs+in+sequence+databanks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000254-Hahn1" id="pone.0000254-Hahn1"></a>Hahn Y, Bera T, Gehlhaus K, Kirsch I, Pastan I, et al.  (2004) Finding fusion genes resulting from chromosome rearrangement by analyzing the expressed sequence databases. Proc Natl Acad Sci U S A  101: 13257–61.  <ul class="find" data-citedArticleID="934070" data-doi="10.1073/pnas.0405490101"><li><a href="http://dx.doi.org/10.1073/pnas.0405490101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Finding+fusion+genes+resulting+from+chromosome+rearrangement+by+analyzing+the+expressed+sequence+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Finding+fusion+genes+resulting+from+chromosome+rearrangement+by+analyzing+the+expressed+sequence+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000254-Kim1" id="pone.0000254-Kim1"></a>Kim N, Kim P, Nam S, Shin S, Lee S (2006) ChimerDB–a knowledgebase for fusion sequences. Nucleic Acids Res  34: D21–4.  <ul class="find" data-citedArticleID="934076" data-doi="10.1093/nar/gkj019"><li><a href="http://dx.doi.org/10.1093/nar/gkj019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ChimerDB%26%23x2013%3Ba+knowledgebase+for+fusion+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ChimerDB%26%23x2013%3Ba+knowledgebase+for+fusion+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000254-Cheng1" id="pone.0000254-Cheng1"></a>Cheng J, Kapranov P, Drenkow J, Dike S, Brubaker S, et al.  (2005) Transcriptional maps of 10 human chromosomes at 5-nucleotide resolution. Science  308: 1149–54.  <ul class="find" data-citedArticleID="934054" data-doi="10.1126/science.1108625"><li><a href="http://dx.doi.org/10.1126/science.1108625" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+maps+of+10+human+chromosomes+at+5-nucleotide+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+maps+of+10+human+chromosomes+at+5-nucleotide+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000254-Frith2" id="pone.0000254-Frith2"></a>Frith M, Forrest A, Nourbakhsh E, Pang K, Kai C, et al.  (2006) The abundance of short proteins in the mammalian proteome. PLoS Genet  2: e52.  <ul class="find" data-citedArticleID="934068" data-doi="10.1371/journal.pgen.0020052"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+abundance+of+short+proteins+in+the+mammalian+proteome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+abundance+of+short+proteins+in+the+mammalian+proteome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000254-Spilianakis1" id="pone.0000254-Spilianakis1"></a>Spilianakis C, Lalioti M, Town T, Lee G, Flavell R (2005) Interchromosomal associations between alternatively expressed loci. Nature  435: 637–45.  <ul class="find" data-citedArticleID="934094" data-doi="10.1038/nature03574"><li><a href="http://dx.doi.org/10.1038/nature03574" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interchromosomal+associations+between+alternatively+expressed+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interchromosomal+associations+between+alternatively+expressed+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000254-Kent1" id="pone.0000254-Kent1"></a>Kent WJ (2002) Blat–the blast-like alignment tool. Genome Res  12: 656–64.  <ul class="find" data-citedArticleID="934074" data-doi="10.1101/gr.229202."><li><a href="http://dx.doi.org/10.1101/gr.229202." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Blat%26%23x2013%3Bthe+blast-like+alignment+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Blat%26%23x2013%3Bthe+blast-like+alignment+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.XML" value="82853"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.PDF" value="267885"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g001.PNG_L" value="585572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g001.PNG_M" value="90304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g001.PNG_S" value="3452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g001.TIF" value="851926"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g001.PNG_I" value="34449"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t001.PNG_L" value="152350"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t001.PNG_M" value="101833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t001.PNG_S" value="7375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t001.TIF" value="765562"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t001.PNG_I" value="79347"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g002.PNG_L" value="328857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g002.PNG_M" value="31977"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g002.PNG_S" value="2703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g002.TIF" value="1114554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g002.PNG_I" value="21185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g003.PNG_L" value="55066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g003.PNG_M" value="9511"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g003.PNG_S" value="1489"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g003.TIF" value="335330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g003.PNG_I" value="5432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t002.PNG_L" value="132895"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t002.PNG_M" value="48397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t002.PNG_S" value="2708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t002.TIF" value="626596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t002.PNG_I" value="20474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g004.PNG_L" value="6907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g004.PNG_M" value="8012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g004.PNG_S" value="1586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g004.TIF" value="31176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.g004.PNG_I" value="4591"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t003.PNG_L" value="131467"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t003.PNG_M" value="48886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t003.PNG_S" value="2804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t003.TIF" value="671096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t003.PNG_I" value="21000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t004.PNG_L" value="134712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t004.PNG_M" value="47568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t004.PNG_S" value="2824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t004.TIF" value="738054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.t004.PNG_I" value="21065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000254.s001.DOC" value="1920512"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000254&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000254" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000254&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000254&volume=&issue=&title=Tentative Mapping of Transcription-Induced Interchromosomal Interaction using Chimeric EST and mRNA Data&author_name=Per%20Unneberg%2C%20Jean-Michel%20Claverie&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000254" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000254" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000254" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000254&amp;t=Tentative%20Mapping%20of%20Transcription-Induced%20Interchromosomal%20Interaction%20using%20Chimeric%20EST%20and%20mRNA%20Data" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000254&title=Tentative%20Mapping%20of%20Transcription-Induced%20Interchromosomal%20Interaction%20using%20Chimeric%20EST%20and%20mRNA%20Data&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000254&amp;title=Tentative%20Mapping%20of%20Transcription-Induced%20Interchromosomal%20Interaction%20using%20Chimeric%20EST%20and%20mRNA%20Data" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000254" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000254&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Tentative Mapping of Transcription-Induced Interchromosomal Interaction using Chimeric EST and mRNA Data';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000254';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Tentative%20Mapping%20of%20Transcription-Induced%20Interchromosomal%20Interaction%20using%20Chimeric%20EST%20and%20mRNA%20Data http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000254" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000254" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosome+mapping%22" title="Search for articles in the subject area:'Chromosome mapping'"><div class="flagText">Chromosome mapping</div></a>
              <div data-categoryid="19841" data-articleid="23346"
                   data-categoryname="Chromosome mapping"
                   class="flagImage" title="Flag 'Chromosome mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="23346"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="23346"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23346"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="23346"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="47429" data-articleid="23346"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40551" data-articleid="23346"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="23346"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3875'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3820'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4042&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=4847">Question and revalidating about mRNA dataset</a><br>Posted by deepreds</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
