

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Simple Fluorescent Sensors Engineered with Catalytic DNA ‘MgZ’ Based on a Non-Classic Allosteric Design</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001224"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001224" />

    <meta name="citation_title" content="Simple Fluorescent Sensors Engineered with Catalytic DNA ‘MgZ’ Based on a Non-Classic Allosteric Design"/>
    <meta itemprop="name" content="Simple Fluorescent Sensors Engineered with Catalytic DNA ‘MgZ’ Based on a Non-Classic Allosteric Design"/>

      <meta name="citation_author" content="William Chiuman"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Biomedical Sciences, McMaster University, Hamilton, Ontario, Canada"/>
      <meta name="citation_author" content="Yingfu Li"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Biomedical Sciences, McMaster University, Hamilton, Ontario, Canada"/>

    <meta name="citation_date" content="2007/11/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001224.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1224"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Selection in vitro of an RNA enzyme that specifically cleaves single-stranded DNA.; citation_author=DL Robertson; citation_author=GF Joyce; citation_journal_title=Nature; citation_volume=344; citation_number=1; citation_pages=467-468; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=In vitro selection of RNA molecules that bind specific ligands.; citation_author=AD Ellington; citation_author=JW Szostak; citation_journal_title=Nature; citation_volume=346; citation_number=2; citation_pages=818-822; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Systematic evolution of ligands by exponential enrichment: RNA ligands to bacteriophage T4 DNA polymerase.; citation_author=C Tuerk; citation_author=L Gold; citation_journal_title=Science; citation_volume=249; citation_number=3; citation_pages=505-510; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Aptamer Database.; citation_author=JF Lee; citation_author=JR Hesselberth; citation_author=LA Meyers; citation_author=AD Ellington; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=4; citation_pages=D95-D100; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=RiboaptDB: A Comprehensive Database of Ribozymes and Aptamers.; citation_author=V Thodima; citation_author=M Piroozina; citation_author=Y Deng; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=5; citation_pages=S6; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Aptamer therapeutics advance.; citation_author=JF Lee; citation_author=GM Stovall; citation_author=AD Ellington; citation_journal_title=Curr Opin Chem Biol; citation_volume=10; citation_number=6; citation_pages=282-289; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Aptamer beacons for the direct detection of proteins.; citation_author=N Hamaguchi; citation_author=AD Ellington; citation_author=M Stanton; citation_journal_title=Anal Biochem; citation_volume=294; citation_number=7; citation_pages=126-131; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Light-switching excimer probes for rapid protein monitoring in complex biological fluids.; citation_author=CJ Yang; citation_author=S Jockusch; citation_author=M Vicens; citation_author=NJ Turro; citation_author=W Tan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=8; citation_pages=17278-17283; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Modular Aptameric Sensors.; citation_author=MN Stojanovic; citation_author=DM Kolpashchikov; citation_journal_title=J Am Chem Soc; citation_volume=126; citation_number=9; citation_pages=9266-9270; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Designed Signaling Aptamers that Transduce Molecular Recognition to Changes in Fluorescence Intensity.; citation_author=SD Jhaveri; citation_author=R Kirby; citation_author=R Conrad; citation_author=EJ Maglott; citation_author=M Bowser; citation_journal_title=J Am Chem Soc; citation_volume=122; citation_number=10; citation_pages=2469-2473; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Structure-Switching Signaling Aptamers.; citation_author=R Nutiu; citation_author=Y Li; citation_journal_title=J Am Chem Soc; citation_volume=125; citation_number=11; citation_pages=4771-4778; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Inhibition of cytohesins by SecinH3 leads to hepatic insulin resistance.; citation_author=M Hafner; citation_author=A Schmitz; citation_author=I Grüne; citation_author=SG Srivatsan; citation_author=B Paul; citation_journal_title=Nature; citation_volume=444; citation_number=12; citation_pages=941-944; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Rational design of allosteric ribozymes.; citation_author=J Tang; citation_author=RR Breaker; citation_journal_title=Chem Biol; citation_volume=4; citation_number=13; citation_pages=453-459; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Engineering precision RNA molecular switches.; citation_author=GA Soukup; citation_author=RR Breaker; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=14; citation_pages=3584-3589; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=In vitro selection of an allosteric ribozyme that transduces analytes to amplicons.; citation_author=MP Robertson; citation_author=AD Ellington; citation_journal_title=Nat Biotechnol; citation_volume=17; citation_number=15; citation_pages=62-66; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=ATP-Dependent Allosteric DNA Enzymes.; citation_author=M Levy; citation_author=AD Ellington; citation_journal_title=Chem Biol; citation_volume=9; citation_number=16; citation_pages=417-426; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Protein-dependent ribozymes report molecular interactions in real time.; citation_author=JS Hartig; citation_author=SH Najafi-Shoushtari; citation_author=I Grüne; citation_author=A Yan; citation_author=AD Ellington; citation_journal_title=Nat Biotechnol; citation_volume=20; citation_number=17; citation_pages=717-722; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Monitoring post-translational modification of proteins with allosteric ribozymes.; citation_author=NK Vaish; citation_author=F Dong; citation_author=L Andrews; citation_author=RE Schweppe; citation_author=NG Ahn; citation_journal_title=Nat Biotechnol; citation_volume=20; citation_number=18; citation_pages=810-815; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A General Strategy for Effector-mediated Control of RNA-cleaving Ribozymes and DNA Enzymes.; citation_author=DY Wang; citation_author=BH Lai; citation_author=D Sen; citation_journal_title=J Mol Biol; citation_volume=318; citation_number=19; citation_pages=33-43; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Group I aptazymes as genetic regulatory switches.; citation_author=KM Thompson; citation_author=HA Syrett; citation_author=SM Knudsen; citation_author=AD Ellington; citation_journal_title=BMC Biotechnology; citation_volume=2; citation_number=20; citation_pages=21; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Competitive regulation of modular allosteric aptazymes by a small molecule and oligonucleotide effector.; citation_author=SH Najafi-Shoushtari; citation_author=M Famulok; citation_journal_title=RNA; citation_volume=11; citation_number=21; citation_pages=1514-1520; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Controlling the rate of organic reactions: rational design of allosteric Diels-Alderase ribozymes.; citation_author=S Amontov; citation_author=A Jäschke; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=22; citation_pages=5032-5038; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Catalysis and rational engineering of trans-acting pH6DZ1, an RNA-cleaving and fluorescence-signaling deoxyribozyme with a four-way junction structure.; citation_author=Y Shen; citation_author=W Chiuman; citation_author=JD Brennan; citation_author=Y Li; citation_journal_title=ChemBioChem; citation_volume=7; citation_number=23; citation_pages=1343-1348; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA aptamer-mediated regulation of the hairpin ribozyme by human alpha-thrombin.; citation_author=SH Najafi-Shoushtari; citation_author=M Famulok; citation_journal_title=Blood Cells Mol Dis; citation_volume=38; citation_number=24; citation_pages=19-24; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Aptamer Displacement Identifies Alternative Small-Molecule Target Sites that Escape Viral Resistance.; citation_author=S Yamazaki; citation_author=L Tan; citation_author=G Mayer; citation_author=JS Hartig; citation_author=JN Song; citation_journal_title=Chem Biol; citation_volume=14; citation_number=25; citation_pages=804-812; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Allosteric selection of ribozymes that respond to the second messengers cGMP and cAMP.; citation_author=M Koizumi; citation_author=GA Soukup; citation_author=JN Kerr; citation_author=RR Breaker; citation_journal_title=Nat Struc Biol; citation_volume=6; citation_number=26; citation_pages=1062-1071; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Altering molecular recognition of RNA aptamers by allosteric selection.; citation_author=GA Soukup; citation_author=GM Emilsson; citation_author=RR Breaker; citation_journal_title=J Mol Biol; citation_volume=298; citation_number=27; citation_pages=623-632; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Generating new ligand-binding RNAs by affinity maturation and disintegration of allosteric ribozymes; citation_author=GA Soukup; citation_author=EC DeRose; citation_author=M Koizumi; citation_author=RR Breaker; citation_journal_title=RNA; citation_volume=7; citation_number=28; citation_pages=524-536; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=In vitro selection of nucleoprotein enzymes.; citation_author=MP Robertson; citation_author=AD Ellington; citation_journal_title=Nat Biotechnol; citation_volume=19; citation_number=29; citation_pages=650-655; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=In vitro selection of allosteric ribozymes: theory and experimental validation.; citation_author=N Piganeau; citation_author=V Thuillier; citation_author=M Famulok; citation_journal_title=J Mol Biol; citation_volume=312; citation_number=30; citation_pages=1177-1190; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=In vitro selection of ribozymes dependent on peptides for activity.; citation_author=MP Robertson; citation_author=SM Knudsen; citation_author=AD Ellington; citation_journal_title=RNA; citation_volume=10; citation_number=31; citation_pages=114-127; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Engineered allosteric ribozymes that respond to specific divalent metal ions.; citation_author=M Zivarts; citation_author=Y Liu; citation_author=RR Breaker; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=32; citation_pages=622-631; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The catalytic diversity of RNAs.; citation_author=MJ Fedor; citation_author=JR Williamson; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=6; citation_number=33; citation_pages=399-412; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=In vitro selection, characterization, and application of deoxyribozymes that cleave RNA.; citation_author=SK Silverman; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=34; citation_pages=6151-6163; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Real-time characterization of ribozymes by fluorescence resonance energy transfer (FRET).; citation_author=A Jenne; citation_author=W Gmlein; citation_author=N Raffler; citation_author=M Famulok; citation_journal_title=Angew Chem Int Ed; citation_volume=38; citation_number=35; citation_pages=1300-1303; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Intracellular ribozyme-catalyzed trans-cleavage of RNA monitored by fluorescence resonance energy transfer.; citation_author=D Vitiello; citation_author=B Pecchia; citation_author=JM Burke; citation_journal_title=RNA; citation_volume=6; citation_number=36; citation_pages=628-637; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Homogeneous assays based on deoxyribozyme catalysis.; citation_author=MN Stojanovic; citation_author=P De Prada; citation_author=DW Landry; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=37; citation_pages=2915-2918; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Detection of small organic analytes by fluorescing molecular switches.; citation_author=C Frauendorf; citation_author=A Jäschke; citation_journal_title=Bioorg Med Chem; citation_volume=9; citation_number=38; citation_pages=2521-2524; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Intermolecular energy migration and fluorescence.; citation_author=T Förster; citation_journal_title=Ann Phys; citation_volume=2; citation_number=39; citation_pages=55-75; citation_date=1948; " />
      <meta name="citation_reference" content="citation_title=An Efficient RNA-Cleaving DNA Enzyme that Synchronizes Catalysis with Fluorescence Signaling.; citation_author=SH Mei; citation_author=Z Liu; citation_author=JD Brennan; citation_author=Y Li; citation_journal_title=J Am Chem Soc; citation_volume=125; citation_number=40; citation_pages=412-420; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Assemblage of Signaling DNA Enzymes with Intriguing Metal-Ion Specificities and pH Dependences.; citation_author=Z Liu; citation_author=SH Mei; citation_author=JD Brennan; citation_author=Y Li; citation_journal_title=J Am Chem Soc; citation_volume=125; citation_number=41; citation_pages=7539-7545; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Revitalization of Six Abandoned Catalytic DNA Species Reveals a Common Three-way Junction Framework and Diverse Catalytic Cores.; citation_author=W Chiuman; citation_author=Y Li; citation_journal_title=J Mol Biol; citation_volume=357; citation_number=42; citation_pages=748-754; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolution of High-Branching Deoxyribozymes from a Catalytic DNA with a Three-Way Junction.; citation_author=W Chiuman; citation_author=Y Li; citation_journal_title=Chem Biol; citation_volume=13; citation_number=43; citation_pages=1061-1069; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Quenching of Fluorophore-Labeled DNA Oligonucleotides by Divalent Metal Ions: Implications for Selection, Design, and Applications of Signaling Aptamers and Signaling Deoxyribozymes.; citation_author=N Rupcich; citation_author=W Chiuman; citation_author=R Nutiu; citation_author=S Mei; citation_author=KK Flora; citation_journal_title=J Am Chem Soc; citation_volume=128; citation_number=44; citation_pages=780-790; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A DNA Aptamer That Binds Adenosine and ATP.; citation_author=DE Huizenga; citation_author=JW Szostak; citation_journal_title=Biochemistry; citation_volume=34; citation_number=45; citation_pages=656-665; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Structural basis of DNA folding and recognition in an AMP-DNA aptamer complex: distinct architectures but common recognition motifs for DNA and RNA aptamers complexed to AMP.; citation_author=CH Lin; citation_author=DJ Patel; citation_journal_title=Chem Biol; citation_volume=4; citation_number=46; citation_pages=817-832; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=ADP-Specific Sensors Enable Universal Assay of Protein Kinase Activity.; citation_author=J Srinivasan; citation_author=ST Cload; citation_author=N Hamaguchi; citation_author=J Kurz; citation_author=S Keene; citation_journal_title=Chem Biol; citation_volume=11; citation_number=47; citation_pages=499-508; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Emergence of a Replicating Species from an in vitro RNA Evolution Reaction.; citation_author=RR Breaker; citation_author=GF Joyce; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=48; citation_pages=6093-6097; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A DNA enzyme with Mg2+-dependent RNA phosphoesterase activity.; citation_author=RR Breaker; citation_author=GF Joyce; citation_journal_title=Chem Biol; citation_volume=2; citation_number=49; citation_pages=655-660; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Sequence Diversity, Metal Specificity, and Catalytic Proficiency of Metal-Dependent Phosphorylating DNA Enzymes.; citation_author=W Wang; citation_author=LP Billen; citation_author=Y Li; citation_journal_title=Chem Biol; citation_volume=9; citation_number=50; citation_pages=507-517; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Efficiencies of fluorescence resonance energy transfer and contact-mediated quenching in oligonucleotide probes.; citation_author=SAE Marras; citation_author=FR Kramer; citation_author=S Tyagi; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=51; citation_pages=e122; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Improving Fluorescent DNAzyme Biosensors by Combining Inter- and Intramolecular Quenchers.; citation_author=J Liu; citation_author=Y Lu; citation_journal_title=Anal Chem; citation_volume=75; citation_number=52; citation_pages=6666-6672; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A catalytic beacon sensor for uranium with parts-per-trillion sensitivity and millionfold selectivity.; citation_author=J Liu; citation_author=AK Brown; citation_author=X Meng; citation_author=DM Cropek; citation_author=JD Istok; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=53; citation_pages=2056-2061; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Efficient signaling platforms built from a small catalytic DNA and doubly labeled fluorogenic substrates.; citation_author=W Chiuman; citation_author=Y Li; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=54; citation_pages=401-405; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A general purpose RNA-cleaving DNA enzyme.; citation_author=SW Santoro; citation_author=GF Joyce; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=55; citation_pages=4262-4266; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=In vitro selection and characterization of a highly efficient Zn(II)-dependent RNA-cleaving deoxyribozyme.; citation_author=J Li; citation_author=W Zheng; citation_author=AH Kwon; citation_author=Y Lu; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=56; citation_pages=481-488; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Preferential activation of the 8–17 deoxyribozyme by Ca(2+) ions. Evidence for the identity of 8–17 with the catalytic domain of the Mg5 deoxyribozyme.; citation_author=A Peracchi; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=57; citation_pages=11693-11697; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Thermodynamic Parameters To Predict Stability of RNA/DNA Hybrid Duplexes.; citation_author=N Sugimoto; citation_author=S Nakano; citation_author=M Katoh; citation_author=A Matsumura; citation_author=H Nakamuta; citation_journal_title=Biochemistry; citation_volume=34; citation_number=58; citation_pages=11211-11216; citation_date=1995; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001224" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Simple Fluorescent Sensors Engineered with Catalytic DNA ‘MgZ’ Based on a Non-Classic Allosteric Design"/>
    <meta name="twitter:description" content="Most NAE (nucleic acid enzyme) sensors are composed of an RNA-cleaving catalytic motif and an aptameric receptor. They operate by activating or repressing the catalytic activity of a relevant NAE through the conformational change in the aptamer upon target binding. To transduce a molecular recognition event to a fluorescence signal, a fluorophore-quencher pair is attached to opposite ends of the RNA substrate such that when the NAE cleaves the substrate, an increased level of fluorescence can be generated. However, almost all NAE sensors to date harbor either NAEs that cannot accommodate a fluorophore-quencher pair near the cleavage site or those that can accept such a modification but require divalent transition metal ions for catalysis. Therefore, the signaling magnitude and the versatility of current NAE sensors might not suffice for analytical and biological applications. Here we report an RNA-cleaving DNA enzyme, termed &lsquo;MgZ&rsquo;, which depends on Mg2+ for its activity and can accommodate bulky dye moieties next to the cleavage site. MgZ was created by in vitro selection. The selection scheme entailed acidic buffering and ethanol-based reaction stoppage to remove selfish DNAs. Characterization of MgZ revealed a three-way junction structure, a cleavage rate of 1 min&minus;1, and 26-fold fluorescence enhancement. Two ligand-responsive NAE sensors were rationally designed by linking an aptamer sequence to the substrate of MgZ. In the absence of the target, the aptamer-linked substrate is locked into a conformation that prohibits MgZ from accessing the substrate. In the presence of the target, the aptamer releases the substrate, which induces MgZ-mediated RNA cleavage. The discovery of MgZ and the introduction of the above NAE sensor design strategy should facilitate future efforts in sensor engineering."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001224.g007"/>

  <meta property="og:title" content="Simple Fluorescent Sensors Engineered with Catalytic DNA ‘MgZ’ Based on a Non-Classic Allosteric Design" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001224" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6265'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4168'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8397&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001224">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001224" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001224&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Simple Fluorescent Sensors Engineered with Catalytic DNA ‘MgZ’ Based on a Non-Classic Allosteric Design
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            William Chiuman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry and Biomedical Sciences, McMaster University, Hamilton, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yingfu Li
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:liying@mcmaster.ca">liying@mcmaster.ca</a></p>

                <p>Affiliation:
                  Department of Biochemistry and Biomedical Sciences, McMaster University, Hamilton, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001224</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001224" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001224" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001224" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001224" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001224-g001" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001224-g002" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001224-g003" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001224-g004" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001224-g005" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001224-g006" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001224-g007" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001224">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001224" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Most NAE (nucleic acid enzyme) sensors are composed of an RNA-cleaving catalytic motif and an aptameric receptor. They operate by activating or repressing the catalytic activity of a relevant NAE through the conformational change in the aptamer upon target binding. To transduce a molecular recognition event to a fluorescence signal, a fluorophore-quencher pair is attached to opposite ends of the RNA substrate such that when the NAE cleaves the substrate, an increased level of fluorescence can be generated. However, almost all NAE sensors to date harbor either NAEs that cannot accommodate a fluorophore-quencher pair near the cleavage site or those that can accept such a modification but require divalent transition metal ions for catalysis. Therefore, the signaling magnitude and the versatility of current NAE sensors might not suffice for analytical and biological applications. Here we report an RNA-cleaving DNA enzyme, termed ‘MgZ’, which depends on Mg<sup>2+</sup> for its activity and can accommodate bulky dye moieties next to the cleavage site. MgZ was created by in vitro selection. The selection scheme entailed acidic buffering and ethanol-based reaction stoppage to remove selfish DNAs. Characterization of MgZ revealed a three-way junction structure, a cleavage rate of 1 min<sup>−1</sup>, and 26-fold fluorescence enhancement. Two ligand-responsive NAE sensors were rationally designed by linking an aptamer sequence to the substrate of MgZ. In the absence of the target, the aptamer-linked substrate is locked into a conformation that prohibits MgZ from accessing the substrate. In the presence of the target, the aptamer releases the substrate, which induces MgZ-mediated RNA cleavage. The discovery of MgZ and the introduction of the above NAE sensor design strategy should facilitate future efforts in sensor engineering.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Chiuman W, Li Y (2007) Simple Fluorescent Sensors Engineered with Catalytic DNA ‘MgZ’ Based on a Non-Classic Allosteric Design. PLoS ONE 2(11):
          e1224.
            doi:10.1371/journal.pone.0001224</p><p><strong>Academic Editor: </strong>Martin Egli, Vanderbilt University, United States of America</p><p><strong>Received:</strong> August 25, 2007; <strong>Accepted:</strong> October 30, 2007; <strong>Published:</strong> November 21, 2007</p><p><strong>Copyright:</strong> © 2007 Chiuman, Li. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a research grant from Canadian Institutes for Health Research (grant number MOP37964).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>NAE (nucleic acid enzyme) sensors have two essential components: a receptor and a transducer. Similar to protein enzymes that are allosterically regulated by the presence of intra- or extracellular signals, an NAE sensor (also referred to as allosteric NAE) operates by activating or suppressing the catalytic activity of the NAE when the receptor recognizes its target. For NAEs that require a cofactor or an environmental cue to perform catalysis, they can be directly utilized for sensing applications as they already contain an inherent receptor. However, to detect a substance other than the catalytic constraint, an external receptor must be introduced.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Aptamers are single-stranded DNA or RNA molecules that can selectively bind to a specific ligand with high affinity and specificity. They can be routinely isolated from random sequence libraries through in vitro selection <a href="#pone.0001224-Robertson1">[1]</a>, <a href="#pone.0001224-Ellington1">[2]</a> or SELEX (Systematic Evolution of Ligands by EXponential enrichment) <a href="#pone.0001224-Tuerk1">[3]</a>. The SELEX technique has been applied to a wide variety of targets ranging from small organic compounds to proteins to whole cells <a href="#pone.0001224-Lee1">[4]</a>, <a href="#pone.0001224-Thodima1">[5]</a>. With such target diversity, aptamers are regarded as a new class of receptors that can rival antibodies <a href="#pone.0001224-Lee2">[6]</a>. Aptamers (particularly DNA aptamers) offer several features that are attractive for biosensor development, which include high tolerance to denaturation, high chemical stability, ease of synthesis and manipulation, and low cost. However, aptamers alone usually cannot function as sensors because they do not posses an inherent ability to report their binding activities.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>There are several strategies that can transduce the ligand-binding event of an aptamer to a physically detectable signal, such as change in fluorescence. These include aptamer beacon <a href="#pone.0001224-Hamaguchi1">[7]</a>, excimer aptamer <a href="#pone.0001224-Yang1">[8]</a>, modular aptameric sensor <a href="#pone.0001224-Stojanovic1">[9]</a>, signaling aptamer <a href="#pone.0001224-Jhaveri1">[10]</a>, structure-switching aptamer <a href="#pone.0001224-Nutiu1">[11]</a>, fluorescence polarization <a href="#pone.0001224-Hafner1">[12]</a> and allosteric NAE <a href="#pone.0001224-Tang1">[13]</a>–<a href="#pone.0001224-Yamazaki1">[25]</a>. We chose allosteric NAE as the basis of our sensor development program simply because NAE is able to convert one binding event to multiple signals during catalytic turnovers, and it could be utilized as a platform to conduct aptamer selection <a href="#pone.0001224-Koizumi1">[26]</a>–<a href="#pone.0001224-Zivarts1">[32]</a>. The enzymatic domains of allosteric NAEs reported to date are mostly RNA-cleaving ribozymes found in nature <a href="#pone.0001224-Fedor1">[33]</a> or RNA-cleaving DNA enzymes (DNAzymes or deoxyribozymes) artificially created by in vitro selection <a href="#pone.0001224-Silverman1">[34]</a>. There are three main reasons why RNA-cleaving systems are usually favored over NAEs that catalyze other chemical reactions: (1) RNA-cleaving NAEs have been exhaustively studied because of their high occurrence in nature and their potential application in gene therapeutics. As a consequence, there are many valuable data that can facilitate biosensor engineering; (2) the progress of RNA cleavage can be easily monitored, as a typical RNA cleavage reaction produces two shorter oligonucleotide fragments; (3) the catalyzed rates of RNA cleavage are among the fastest of all reactions catalyzed by NAEs.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>To synchronize cleavage activity with a change in fluorescence, molecular engineers typically attach a fluorophore and a quencher to opposite ends of an RNA substrate such that when the substrate is cleaved, the fluorophore and the quencher are physically separated from each other, with a concomitant increase in fluorescent intensity. This specific dye arrangement usually yields less than 10-fold fluorescence enhancement <a href="#pone.0001224-Jenne1">[35]</a>–<a href="#pone.0001224-Frauendorf1">[38]</a>. It is conceivable that the signaling magnitude can be greatly improved by simply moving the fluorophore and the quencher closer to the cleavage site, given that the efficiency of resonance energy transfer between any two dyes is inversely proportional to the sixth power of the separating distance <a href="#pone.0001224-Frster1">[39]</a>. Nonetheless, such operation might make an NAE incompetent at cleaving its substrate for the following reasons: (1) most RNA-cleaving NAEs were not intentionally selected to cleave substrates that have dyes at the cleavage site; (2) dye-bulkiness might sterically hinder the enzyme to access the cleavage site; (3) considering that most dyes also contain chemically functional moieties, the interactions between the scissile ribo-linkage and the enzyme might be disrupted due to the formation of an alternative interaction network.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Previous efforts from our group have led to the report of at least 9 DNAzymes that could efficiently cleave a chimeric DNA/RNA substrate that contains a lone RNA linkage (rA, adenine ribonucleotide) sandwiched between two deoxyribothymidines modified with a fluorescein (F) and a DABCYL (Q; 4-(4-dimethylaminophenylazo) benzoic acid), respectively (<a href="#pone-0001224-g001">Figure 1</a> inset, underlined) <a href="#pone.0001224-Mei1">[40]</a>–<a href="#pone.0001224-Chiuman2">[43]</a>. These DNAzymes were isolated by in vitro selection and they were anticipated to yield a large fluorescence increase upon substrate cleavage (owing to the close proximity of F and Q on the uncleaved substrate). However, due to the fact that divalent transition metal ions (such as Mn<sup>2+</sup>, Cd<sup>2+</sup>, Co<sup>2+</sup>, Ni<sup>2+</sup>) are required for the DNAzyme function and that these metal ions have been found to be strong fluorescence quenchers <a href="#pone.0001224-Rupcich1">[44]</a>, the best signal enhancement attained was only ~3-fold better than that of the end-to-end dye arrangement described above. The necessity of transition metal ions also hinders the recruitment of these DNAzymes in biological applications because the metal toxicity could become a concern. We were intrigued, however, by the fact that none of these DNAzymes recruited Mg<sup>2+</sup> as a cofactor, even though Mg<sup>2+</sup> was included in the selection buffer. To investigate whether it is a chemical challenge for DNA to catalyze the cleavage of the same substrate in the presence of Mg<sup>2+</sup>, we set out to carry out an in vitro selection study to isolate similar DNAzymes when Mg<sup>2+</sup> was given as the only divalent metal ion in the reaction mixture.</p>
<div class="figure" id="pone-0001224-g001"><div class="img"><a name="pone-0001224-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>(A) In vitro selection scheme. The self-cleaving construct of the library consists of a DNA substrate (blue bars), 2 primer binding sites (open bars) and a random library (black bars). The substrate has one ribonucleotide, rA, flanked immediately by a fluorescein (F)- and a DABCYL (Q)-modified deoxyribothymidine. DNA species that are able to self-cleave in the presence of Mg<sup>2+</sup> are isolated by denaturing PAGE and subsequently subjected to PCR- amplification. A ribo-linkage (rA) is introduced to the top strand of the PCR product during PCR 2, which enables the recovery of the DNAzyme strand after NaOH treatment followed by PAGE purification. To regenerate the self-cleaving construct, the DNA molecules are phosphorylated at the 5′ ends and ligated to the substrates. The selection cycle continues until a desired cleavage activity is reached. N represents A, G, C or T; P, phosphate. (B) MgZ in cis. Filled circles, absolutely conserved residues; Open bars, covariations; Filled triangle, cleavage site; Filled arrows, DMS methylation interfered residues; F, fluorescein-dT; Q, DABCYL-dT. Inset: random library in the self-cleaving construct (L) and the primers used (PM1-3). Substrate sequence is underlined.</p>
<span>doi:10.1371/journal.pone.0001224.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>In vitro selection</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The starting library contained, in the 5′ to 3′ direction, the substrate, the 5′ primer-binding site, a 60-nucleotide random region (N<sub>60</sub>) and the 3′ primer-binding site (<a href="#pone-0001224-g001">Figure 1A</a>). The random region in one-third of the library was composed of equal ratio of A, G, C and T at every position, while the random region of the remaining library was composed of the sequences of the known ‘OA’ DNAzymes with a degeneracy of 0.3 (70% wild type nucleotide and 10% each of the other nucleotides for every nucleotide position within the random region). OAs are six RNA-cleaving DNAzymes previously isolated by our group and all of them use divalent transition metal ions as cofactors <a href="#pone.0001224-Chiuman1">[42]</a>. They were included in the starting pool because we speculated that some OA variants might be able to use Mg<sup>2+</sup> as well. To isolate Mg<sup>2+</sup>-dependent DNAzymes, the DNA molecules with the covalently attached substrate were allowed to self-cleave with 10 mM Mg<sup>2+</sup> at pH 7.5 for 4 hours. Cleaved sequences were isolated by denaturing PAGE and PCR-amplified with primers PM1 and PM2 (<a href="#pone-0001224-g001">Figure 1</a> inset). A tiny amount of the PCR product was used to seed the second PCR where a ribo-linkage was introduced to the sense strand by using a ribo-terminated primer (PM3). After boiling the second PCR product in NaOH, the ribo-linkage was hydrolyzed, and as a result, three oligonucleotides of unequal lengths were produced. The DNAzyme strand was purified by denaturing PAGE, phosphorylated at the 5′, and ligated to the substrate. Such assembled DNA pool was then used for the next round of selection. Another 11 cycles were performed with the reaction time progressively reduced from 4 h to 6 s for the purpose of enriching the most efficient DNA sequences (see <a href="#s4">Materials and Methods</a> for the reaction time allowed for each cycle). The final generation (G12) was cloned and sequenced.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Twenty-one sequences were obtained (<a href="#pone.0001224.s002">Figure S1</a>) and they are termed G12SD-X, where X is the clone number. Two sequences, G12SD-1 and G12SD-2, from the two dominating sequence classes were arbitrarily chosen for further examination. To our surprise, both DNA species were able to cleave the RNA linkage in the absence of Mg<sup>2+</sup>. However, the best cleavage yields in any cases were less than 10% (data not shown). We then attempted to optimize G12SD-1 and G12SD-2 through reselection (i.e., in vitro selection using degenerate libraries). Without much success, the optimized species only showed a small increase in self-cleavage yield (≤10%) and no improvement in intermolecular cleavage (<a href="#pone.0001224.s003">Figure S2</a>, only the optimized G12SD-1 is shown).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>As we later found out that the activity of G12SD-1 could be severely suppressed in low pH conditions, several procedures in the selection scheme were modified, especially after the substrates were ligated to the DNAzymes. The changes made include: (1) running denaturing PAGE in 1× TME solution (30 mM Tris, 54 mM MES, 1 mM Na<sub>2</sub>EDTA, pH 6.2) instead of 1× TBE (89 mM Tris, 89 mM boric acid, 1 mM EDTA, pH 8.3); (2) eluting DNA from the gel in 10 mM MES (pH 5.5) rather than in the solution containing 10 mM Tris-HCl (pH 7.5), 200 mM NaCl and 1 mM EDTA; (3) reducing the elution time from 30 min to 10 min; (4) stopping the reaction by adding 2.5× equivalent volume of 100% ethanol. Using G5 (where a strong cleavage activity was first observed) as our starting pool, a new selection experiment was conducted with the modified selection scheme, in hope of isolating DNAzymes that are Mg<sup>2+</sup>-dependent and more efficient than G12SD. The reaction was restricted to 1 min in the first round of selection and progressively shortened to 10 s over the next 6 rounds (see <a href="#s4">Materials and Methods</a> for the reaction time allowed for each cycle). Four sequence classes in the final generation were found to use Mg<sup>2+</sup> for substrate cleavage (<a href="#pone.0001224.s002">Figure S1</a>); sequences that perform Mg<sup>2+</sup>-dependent RNA cleavage are termed MgZ-X. One sequence from each class was arbitrarily chosen, and their trans-cleavage activities were assessed. The two sequences (MgZ-5 and MgZ-7) that showed higher activities were subjected to reselection. Given that the reselected MgZ-5 showed higher cleavage activity than the reselected MgZ-7, one of the MgZ-5 variants (56 in total) was selected for the remaining study. It is referred to as ‘MgZ’ for convenience (<a href="#pone-0001224-g001">Figure 1B</a>).</p>


<h4>Identification of catalytic motifs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>According to our proposed secondary structure model (<a href="#pone-0001224-g001">Figure 1B</a>), MgZ adopts a three-way junction structure when binding to the substrate. Stems P1 and P2 are formed by the hybridization between the substrate (blue) and the two binding arms located at the 5′ and the 3′ ends of the DNAzyme (black), thus creating a bulge at the cleavage site. Stem P3 is formed on the DNAzyme strand, and it interlinks P1 and P2 through two stretches of 8 nucleotides, namely J2/3 and J3/1 (J2/3 represents the junction between P2 and P3; similar nomenclature applies to J3/1). Sequence analysis of the reselected clones revealed two observations: (1) the existence of P3 is strongly supported by the covariable nucleotide pairs found in the region between G<sub>48</sub> and T<sub>64</sub> (highlighted with open bars); (2) the highly conserved nucleotides (black circles) are concentrated mostly in J2/3 and J3/1. Since some of these nucleotides are also prone to dimethyl sulfate (DMS) methylation interference (highlighted with filled arrows), it is reasonable to assume that J2/3, J3/1 and the open end of P3 harbor the catalytically essential nucleotides or presumably, the catalytic core of MgZ. For details in the nucleotide conservation patterns and the DMS methylation interference result, see <a href="#pone.0001224.s004">Figures S3</a> and <a href="#pone.0001224.s005">S4</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To validate the proposed secondary structure, MgZ was first converted to the trans format by disjoining the DNAzyme strand from the substrate (<a href="#pone-0001224-g001">Figure 1B</a>). The 3′ tail of the substrate was then extended with 5′-TTCTTGATCAA-3′ such that a stronger P2 could be formed (<a href="#pone.0001224.s006">Figure S5</a>). Based on these constructs, a series of substrate and MgZ variants were synthesized and tested to assess the catalytic importance of P1, P2 and P3. In brief, all 3 stems are required for cleavage activity. Deletion of either DNA strand of any stem or incorporation of several mismatches within the stem motifs is detrimental to catalytic function. P1, P2 and P3 can be shortened to 6, 4 and 6 base pairs, respectively without a drop in cleavage yield. The sequence contexts of P1 and P2 can be altered as long as the Watson-Crick base pairs are retained. However, the catalytic activity would be compromised to different extents, depending on the modifications. P3 can be covaried with no effect on the cleavage activity, except the two C≡G base pairs and the following G•T wobble pair at the open end (<a href="#pone-0001224-g001">Figure 1B</a>). The sequence context of loop L3 is not important for catalysis. Details of individual mutant constructs and experimental results are presented in <a href="#pone.0001224.s006">Figures S5</a>, <a href="#pone.0001224.s007">S6</a>, <a href="#pone.0001224.s008">S7</a> and <a href="#pone.0001224.s009">S8</a>.</p>


<h4>Signaling property of the DNA enzyme</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Radioactive and fluorescence-monitored cleavage assays were carried out in a single-turnover condition with several combinations of substrate constructs and MgZ variants that had been used to characterize the structure of MgZ (see <a href="#pone.0001224.s010">Figure S9</a> for results). The combination that gave the best signaling performance is shown in <a href="#pone-0001224-g002">Figure 2A</a>. Its cleavage and fluorescence kinetics are plotted in <a href="#pone-0001224-g002">Figure 2B</a>. Cleavage yields over time (γ) refer to the left y-axis (% cleavage yield); other data points refer to the right y-axis (<em>F</em>/<em>F</em><sub>o</sub>, fluorescence enhancement). The cleavage reaction was initiated by adding excess DNAzymes to the substrate in 70 mM HEPES (pH 8.0), 40 mM MgCl<sub>2</sub> and 0.001% Tween-20 at 30°C (the optimal condition with pH≤8.0). More fluorescence was generated as the reaction proceeded. Since the level of fluorescence enhancement increased at a similar rate as that of the cleavage yield, the release of cleavage fragments (at least one of the two) from the DNAzyme was not a limiting step for fluorescence development. The cleavage rate of the DNAzyme was 1 min<sup>−1</sup> with 91% final cleavage yield and the signaling rate was 0.8-fold min<sup>−1</sup> with 26-fold maximum signal enhancement. It should be noted that the observed fluorescence increase was specific to the DNAzyme-catalyzed substrate cleavage, as no fluorescence enhancement was observed when either Mg<sup>2+</sup> or MgZ was excluded from the reaction mixture (<a href="#pone-0001224-g002">Figure 2B</a>).</p>
<div class="figure" id="pone-0001224-g002"><div class="img"><a name="pone-0001224-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) MgZ in trans. Shown here is a specific combination of substrate and MgZ variant that generates the largest signal enhancement upon substrate cleavage. (B) Kinetic analyses. □ refers to the left y-axis (% cleavage); ○, +(−Mg) and×(−MgZ) refer to the right y-axis (<em>F</em>/<em>F</em><sub>o</sub>). The % cleavage vs. time data were fitted to <em>Y</em> = <em>Y</em><sub>f</sub>(1–e<sup>−<em>k</em><sub>c</sub></sup><em><sup>t</sup></em>). Fitting-curve is shown as ---. <em>Y</em>, cleavage yield; <em>Y</em><sub>f</sub>, final cleavage yield; <em>k</em><sub>c</sub>, observed cleavage rate. The <em>F</em>/<em>F</em><sub>o</sub> vs. time data were fitted to <em>F</em>/<em>F</em><sub>o</sub> = initial <em>F</em>/<em>F</em><sub>o</sub>+(final <em>F</em>/<em>F</em><sub>o</sub>−initial <em>F</em>/<em>F</em><sub>o</sub>)×(1−e<sup>−<em>k</em><sub>s</sub></sup><em><sup>t</sup></em>); initial <em>F</em>/<em>F</em><sub>o</sub> = 1.0. Fitting curve is shown as —. <em>F</em>/<em>F</em><sub>o</sub>, fluorescence enhancement; <em>k</em><sub>s</sub>, signal-enhancement rate. Cleavage rate = 1 min<sup>−1</sup> with Y<sub>f</sub> = 91%. Signaling rate = 0.8-fold min<sup>−1</sup> with final <em>F</em>/<em>F</em><sub>o</sub> = 26-fold. Reaction condition: 5 nM substrate, 1 µM DNAzyme, 70 mM HEPES (pH 8.0), 40 mM MgCl<sub>2</sub>, 0.001% Tween-20, 30°C.</p>
<span>doi:10.1371/journal.pone.0001224.g002</span></div>

<h4>Sensor engineering</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>There are a variety of engineering methods to assemble NAE sensors. The most well known method is to link a particular aptamer to an RNA-cleaving NAE (such as hammerhead ribozyme) through a communication module <a href="#pone.0001224-Soukup1">[14]</a>. Communication module is usually a weak duplex scaffold that changes its base-pairing pattern, depending on whether the aptamer domain is in the target-bound state or the target-free state. Alteration in the base-pairing pattern, in turn, causes either activation or repression of the activity of NAE. In spite of many engineering approaches, the common theme in all of the existing NAE sensors is to put NAE under allosteric control <a href="#pone.0001224-Tang1">[13]</a>–<a href="#pone.0001224-Zivarts1">[32]</a>. Here we used an alternative approach to design an ATP sensor and an ADP sensor using two existing aptamers and MgZ. The purposes of these experiments were mainly two-fold: (1) to show the utility of MgZ in sensor development; (2) to demonstrate that NAE sensors can also be engineered on the basis of a mechanism that regulates the accessibility of the substrate through ligand binding. For convenience, we term our strategy ‘non-classic allosteric design’.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The sensor design and mechanism are depicted in <a href="#pone-0001224-g003">Figure 3</a> and described as follows. An aptamer (cyan) is engineered to harbor an antisense element (magenta) in its variable region. By conjugating such aptamer to the fluorogenic substrate of MgZ, the substrate is sequestered in an extended conformation (such as a duplex) by hybridization with the antisense element. With this arrangement, MgZ has no access to the substrate and no cleavage occurs. In the presence of the target, however, the aptamer pulls away the antisense element from the substrate and folds into the target-bound conformation. Consequently, MgZ is able to bind and cleave the substrate. Due to the physical separation of F and Q, higher level of fluorescence will be generated in response to the presence of target.</p>
<div class="figure" id="pone-0001224-g003"><div class="img"><a name="pone-0001224-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Sensor design and mechanism.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Blue, substrate; cyan, aptamer; magenta, antisense element. See text for details.</p>
<span>doi:10.1371/journal.pone.0001224.g003</span></div>

<h4>ATP sensor</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The receptor of the ATP sensor is an ATP-binding DNA aptamer that was previously isolated by the Szostak group <a href="#pone.0001224-Huizenga1">[45]</a>. It has high affinity to ATP and its analogs such as adenosine and AMP; however, the aptamer does not bind to other nucleotides. The NMR model of the AMP-aptamer complex reveals that the aptamer adopts a stem-loop structure with two binding sites <a href="#pone.0001224-Lin1">[46]</a>. Given that the loop is not involved directly in target binding, it poses a good site to incorporate an antisense element. A series of aptamer-antisense sequences were designed and annealed to the 5′ or the 3′ ends of the substrates. These constructs were screened by radioactive cleavage assay for the ability to block the substrate from being cleaved by MgZ in the absence of ATP, as well as the plasticity to switch from the substrate-blocking conformation (closed) to the target-bound conformation (open), which in essence allows MgZ to bind and cleave the substrate, when ATP is added.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>One of the constructs, which showed the largest increase in cleavage yield over the background, has a 7-nucleotide antisense element that, in conjunction with the flanking nucleotides of the aptamer, locks the substrate in a 13-base pair bulged duplex (<a href="#pone-0001224-g004">Figure 4A</a>; same color scheme as in <a href="#pone-0001224-g003">Figure 3</a>). Although minor cleavage by MgZ still occurred with this scaffold (<a href="#pone-0001224-g004">Figure 4B</a>), the amount of cleavage products was elevated by at least 20-fold (within the 90 min time frame) in response to 1 mM ATP. This observation clearly indicates that substrate cleavage was triggered only through the recognition of the target and not by means of random events. To assess the signaling performance of the sensor, fluorescence enhancement kinetics correlated to various concentrations of ATP were surveyed. As shown in <a href="#pone-0001224-g004">Figure 4C</a>, higher rate of fluorescence increase and larger fluorescence enhancement correspond to higher concentration of ATP up to 1 mM (a smaller signal enhancement with 2 mM was attributable to the pH effect of ATP; see <a href="#pone.0001224.s011">Figure S10</a>). The sensor has a detection limit of 5–10 µM and a greater sensitivity in the range of 10–250 µM. Similar to the “0 µM ATP” kinetics (black), the signal enhancement with 1 mM GTP (grey) was also negligible over time, which reiterates the fact that the aptamer (therefore, the sensor) is selective for ATP.</p>
<div class="figure" id="pone-0001224-g004"><div class="img"><a name="pone-0001224-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>ATP sensor.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) ATP-ASAP. (B) Phosphorimage of the radioactive cleavage assays. Cleavage reactions were carried out with 5 nM ATP-ASAP, 50 nM MgZ and 1 mM ATP (in the case of ATP-induced cleavage) in 50 mM HEPES (pH 7.0), 20 mM MgCl<sub>2</sub>, 0.001% Tween-20 at room temperature. m, minutes; % clv, % cleavage; ND, not detectable; ▸, substrate; ▹, 5′ cleavage fragment. (C) Fluorescence kinetics in response to various concentrations of ligands. Similar reaction conditions in panel B were applied.</p>
<span>doi:10.1371/journal.pone.0001224.g004</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>In comparison with the unmodified substrate (<a href="#pone-0001224-g002">Figure 2</a>), the assemblage of <span class="underline">a</span>ntisense element, <span class="underline">s</span>ubstrate and <span class="underline">ap</span>tamer into a tripartite structure-switching substrate (termed “ASAP”) did not allow RNA cleavage to occur with a similar rate, cleavage yield or fluorescence enhancement even under the ATP-saturating condition (<a href="#pone-0001224-g004">Figures 4B and 4C</a>). There are at least three plausible reasons: (1) the reaction was not conducted in the optimal condition for substrate cleavage (<a href="#pone-0001224-g002">Figure 2</a> legend) but in a condition specifically optimized for ATP-induced cleavage (<a href="#pone-0001224-g004">Figure 4</a> legend); (2) the ASAP complexes might have folded into alternative structures that prohibited conformational transition and substrate access; (3) the closed conformation is strongly favored over the open conformation. To determine if the binding affinity of the aptamer was deteriorated in the context of ASAP, the initial rates of fluorescence enhancement were calculated and plotted against the ATP concentrations (<a href="#pone-0001224-g005">Figure 5</a>). Curve-fitting with the one site binding model revealed an apparent <em>K</em><sub>d</sub> of 456±47 µM, which is significantly higher than that of the aptamer alone (~6 µM) <a href="#pone.0001224-Huizenga1">[45]</a>. Since an increase in the apparent <em>K</em><sub>d</sub> was also observed in several allosteric NAEs <a href="#pone.0001224-Levy1">[16]</a>, <a href="#pone.0001224-Vaish1">[18]</a>, <a href="#pone.0001224-Soukup2">[27]</a>, <a href="#pone.0001224-Soukup3">[28]</a>, it might implicate that an extra free energy is usually required to drive the conformational transition in both allosteric and ASAP systems. Based on the initial rates of fluorescence enhancement, the sensor was found to have &gt;445-fold higher selectivity for ATP over GTP. It is important to note that the data in <a href="#pone-0001224-g005">Figure 5</a> could only be fitted with the one site binding model, despite the NMR structure reveals two binding sites <a href="#pone.0001224-Lin1">[46]</a> and previous studies have suggested that the aptamer binds cooperatively to 2 ATP molecules <a href="#pone.0001224-Jhaveri1">[10]</a>, <a href="#pone.0001224-Levy1">[16]</a>. This discrepancy might be due to that the transition of ASAP from the closed to the open conformation can only be induced through the binding of 2 ATP molecules at the same time. Such a change in the binding mode relative to that of the parent aptamer might also contribute to the apparent decrease in binding affinity. Regardless, the sensor described here is comparable to most DNAzyme-based <a href="#pone.0001224-Levy1">[16]</a> and even aptamer-based ATP sensors <a href="#pone.0001224-Stojanovic1">[9]</a>–<a href="#pone.0001224-Nutiu1">[11]</a> in terms of sensitivity, selectivity and the dynamic range of detection.</p>
<div class="figure" id="pone-0001224-g005"><div class="img"><a name="pone-0001224-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Initial rate of fluorescence enhancement vs. ATP concentration.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>Fitting curve: <em>v</em><sub>o</sub> = (<em>v</em><sub>max</sub>×[ligand])/(<em>K</em><sub>d</sub>+[ligand]). <em>v</em><sub>o</sub>, initial rate of signal enhancement (see <a href="#s4">Materials and Methods</a>); <em>v</em><sub>max</sub>, maximal rate of signal enhancement; <em>K</em><sub>d</sub>, dissociation constant. Error bar represents the standard deviation of three independent assays. <em>v</em><sub>o</sub> with 2 mM ATP was omitted in the curve fitting process due to substantial fluorescence quenching by high [ATP] (<a href="#pone.0001224.s011">Figure S10</a>). Inset: <em>v</em><sub>o</sub> vs. low [ATP].</p>
<span>doi:10.1371/journal.pone.0001224.g005</span></div>

<h4>ADP sensor</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The receptor of the ADP sensor is an ADP-RNA aptamer isolated by the Diener group <a href="#pone.0001224-Srinivasan1">[47]</a>. The aptamer has ~300-fold higher selectivity for ADP over ATP and low affinity to other nucleotides. The ADP aptamer was proposed to have a stem loop structure, in which the loop is not important for its function and thus is a good site to incorporate an antisense element. A series of antisense elements were screened for ADP-induced substrate cleavage. One of the best ASAPs, shown in <a href="#pone-0001224-g006">Figure 6A</a>, has a 30-nucleotide long antisense element that locks the substrate in a 3-way junction scaffold. By combining this ASAP with MgZ, an ADP sensor was created. It should be emphasized that the antisense element is composed entirely of DNA. Although it would be more convenient to synthesize an RNA antisense element along with the aptamer sequence in one piece, we have screened 10 different ASAP constructs that contain all RNA aptamer-antisense sequences, and none of them was able to inhibit substrate cleavage in the absence of the target, even with an antisense element that forms 14 base pairs with the substrate. A similar DNA counterpart, however, could show some substrate-blocking activity (data not shown).</p>
<div class="figure" id="pone-0001224-g006"><div class="img"><a name="pone-0001224-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>ADP sensor.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) ADP-ASAP. (B) Phosphorimage of the radioactive cleavage assays. Cleavage reactions were carried out with 5 nM ADP-ASAP, 1 µM MgZ and 1 mM ADP (in the case of ADP-induced cleavage) in 50 mM HEPES (pH 7.0), 20 mM MgCl<sub>2</sub>, 0.001% Tween-20 at room temperature. ▸, substrate; ▹, 3′ cleavage fragment. (C) Fluorescence kinetics in response to various ADP concentrations. Similar reaction conditions in panel B were applied.</p>
<span>doi:10.1371/journal.pone.0001224.g006</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>To evaluate the sensor performance, both radioactive and fluorescent cleavage assays were performed. Phosphorimage of the radioactive cleavage assays (<a href="#pone-0001224-g006">Figure 6B</a>) shows that the sensor, in response to 1 mM ADP, had &gt;10-fold increase in the cleavage yield over the background (no ADP) in the first 5 min. That difference, however, fell progressively to ~4-fold over the next 55 min, as more ASAPs collapsed to alternative conformations that allowed substrate cleavage in the absence of the target whilst the ADP-induced cleavage went to near completion. Fluorescence enhancement kinetics correlated to various concentrations of ADP are plotted in <a href="#pone-0001224-g006">Figure 6C</a>. As indicated in the figure, higher ADP concentrations resulted in higher rate of fluorescence increase and larger fluorescence enhancement. The sensor has a low detection limit of 2 µM and a greater sensitivity in the range of 2–500 µM. The sensitivity is better resolved by plotting the initial rate of fluorescence enhancement against the ADP concentration, as shown in <a href="#pone-0001224-g007">Figure 7A</a>. By fitting the data from this plot into the one site binding model, an apparent <em>K</em><sub>d</sub> of 35±3 µM was revealed, which is higher than that of the aptamer alone (3 µM) <a href="#pone.0001224-Srinivasan1">[47]</a>. This again suggests that more free energy is required for the aptamer to undergo conformational transition in the context of ASAP.</p>
<div class="figure" id="pone-0001224-g007"><div class="img"><a name="pone-0001224-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001224" data-uri="info:doi/10.1371/journal.pone.0001224.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001224.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001224.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001224.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001224.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Sensitivity and specificity of ADP sensor.</span></strong></p><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1"></a><p>(A) Initial rate of fluorescence enhancement vs. ADP concentration. Fitting curve: <em>v</em><sub>o</sub> = (<em>v</em><sub>max</sub>×[ligand])/(<em>K</em><sub>d</sub>+[ligand]). Error bar: standard deviation of three independent assays. <em>v</em><sub>o</sub> with 2 mM ADP was omitted in the curve fitting process due to fluorescence quenching by high [ADP] (<a href="#pone.0001224.s011">Figure S10</a>). Inset: <em>v</em><sub>o</sub> vs. low [ADP]. (B) Fluorescence kinetics in response to non-cognate ligands. ado, adenosine; ×, 1 mM ado; +, 1 mM AMP; □, no ligand; ▵, 200 µM ATP; ▿, 300 µM ATP; ○, 400 µM ATP; •, 1 mM ATP.</p>
<span>doi:10.1371/journal.pone.0001224.g007</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>Shown in <a href="#pone-0001224-g007">Figure 7B</a> are the fluorescence enhancement kinetics corresponding to adenosine, AMP and ATP at various concentrations. In comparison with the background signal (no ligand), no extra fluorescence was generated with 1 mM adenosine, 1 mM AMP and 200 µM ATP. Nevertheless, with 300, 400 and 1000 µM ATP, larger fluorescence enhancements up to ~3, 3.4 and 3.9-fold were detected within the 60 min time frame. Based on the initial rates of fluorescence enhancement corresponding to various ADP analogs at 1 mM, the sensor has &gt;22-fold selectivity over adenosine and AMP, and 9-fold selectivity over ATP. However, at 200 µM of analyte, the sensor has &gt;18-fold selectivity over ATP.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The key to the success of in vitro selection is to devise an effective selection method to separate the nucleic acid species with the function of interest apart from the inert species. However, even with a well-established selection protocol, some unwanted species with unexpected characteristics might still bypass the selection criteria and in our case, dominated the whole population. These molecules are identified as “selfish species” <a href="#pone.0001224-Breaker1">[48]</a>. To eradicate these molecules in the course of selection, we first characterized them and then modified the selection scheme accordingly so as to reduce their chances of survival. Although our strategy was not perfect as some of them were still able to pass through, the pool distribution of Mg<sup>2+</sup>-utilizing DNAzymes must have grown much faster than the selfish species in consecutive selection cycles, and allowed the successful isolation of MgZ.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The discovery of MgZ has brought us to the conclusion that beside the normal RNA substrate and the chimeric DNA/RNA substrate <a href="#pone.0001224-Silverman1">[34]</a>, <a href="#pone.0001224-Breaker2">[49]</a>, the chemical challenge to cleave a DNA substrate with a crowded RNA linkage can also be overcome by catalytic DNAs assisted by Mg<sup>2+</sup>. However, it was intriguing to observe that MgZ also adopts a simple 3-way junction secondary structure that is common to all OA DNAzymes <a href="#pone.0001224-Chiuman1">[42]</a>. This prompted us to wonder why none of the sequenced clones (~130) from our previous selection study recruited Mg<sup>2+</sup> as the divalent metal ion cofactor <a href="#pone.0001224-Mei1">[40]</a>, <a href="#pone.0001224-Chiuman1">[42]</a>, if a similar structural framework was exploited. One possible reason is that the diversity of transition metal ion-dependent DNAzymes was significantly higher than that of the Mg<sup>2+</sup>-dependent DNAzymes in the initial library. Thus, the chances for the minor Mg<sup>2+</sup>-dependent species to survive the entire in vitro selection process would be small, as they could be easily lost. This notion could be further supported by a previous study where transition metal ion-promoted self-phosphorylating DNAzymes were found to be significantly more populated than the ones assisted by Mg<sup>2+</sup> <a href="#pone.0001224-Wang2">[50]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>As discussed earlier, molecular engineers typically attach a pair of fluorophore and quencher at opposite ends of the substrate to couple the RNA-cleaving activity with fluorescence signals. To improve the signal enhancement attained by this end-to-end dye arrangement (&lt;10-fold), the Lu group has added another quencher to one end of the NAE, which also positioned the quencher adjacent to the fluorophore for static quenching <a href="#pone.0001224-Marras1">[51]</a>. This new dye arrangement has been applied to two DNAzymes that are lead-dependent <a href="#pone.0001224-Liu2">[52]</a> and uranyl-specific <a href="#pone.0001224-Liu3">[53]</a>, respectively, and the resulting constructs produced 10- and 16-fold signal enhancements upon substrate cleavage. Recently, we have studied the effects of the nature of dye moieties as well as their attaching positions along the substrate strand on the catalytic and signaling performance of an RNA-cleaving DNAzyme termed 8–17 <a href="#pone.0001224-Chiuman3">[54]</a>. 8–17 is very versatile in terms of its metal usage for catalysis (including Mg<sup>2+</sup>) <a href="#pone.0001224-Liu2">[52]</a>, <a href="#pone.0001224-Santoro1">[55]</a>–<a href="#pone.0001224-Peracchi1">[57]</a>. Although we learnt that 8–17 is able to accommodate dye compounds that are only one nucleotide away from the cleavage site and to generate ≤85-fold fluorescence enhancement, the cleavage activity will be severely impaired if Mn<sup>2+</sup> is not supplemented in the reaction mixture. Clearly, the advent of new engineering methods described above have significantly advanced the development of better fluorogenic NAE platforms and eventually, NAE sensors. Nevertheless, these platforms might not be the ideal choices for engineers to apply in building sensors for biological applications. Here we took advantage of the in vitro selection technique and successfully isolated a Mg<sup>2+</sup>-dependent DNAzyme that could efficiently cleave a sterically hindered (and perhaps, chemically challenging) RNA linkage that is flanked immediately by fluorescein and DABCYL. Due to the close proximity of these two chromophores, substrate cleavage could lead to high fluorescence generation and in this case, 26-fold fluorescence enhancement above the background (substrate itself), which doubles the best fluorescence enhancement attained by the catalytic platforms previously isolated by the Li group <a href="#pone.0001224-Mei1">[40]</a>–<a href="#pone.0001224-Chiuman2">[43]</a>. Given that MgZ-catalyzed RNA cleavage went to almost completion in 5 min (<a href="#pone-0001224-g002">Figure 2B</a>), we believe MgZ will be a great signaling molecule to be exploited in sensor engineering for both analytical and biological applications.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>To demonstrate how to build sensors using the MgZ system, we have chosen the ATP-binding DNA aptamer and the ADP-binding RNA aptamer as our model receptors. Using a new engineering approach, the aptamers were incorporated with antisense elements and annealed to the substrates to form ASAP complexes. In the absence of the target, ASAPs remained in the closed state; in the presence of the target, ASAPs switched to the open state and allowed MgZ to cleave the substrate, leading to high fluorescence generation. As depicted in <a href="#pone-0001224-g003">Figure 3</a>, ASAP is an isolated unit where both target recognition and conformational transition occur in chorus and independently from the DNAzyme. Since no modification is applied to the DNAzyme, its folding and catalytic activity would not be affected (presumably) to any extent when it binds to the substrate domain of ASAP, as the aptamer is attached to one extreme end of the substrate. Likewise, the aptamer should also not be disrupted by the folding of the DNAzyme through steric clashes. Therefore, the only constraint to build a similar type of sensor is to find a region of the aptamer that could be replaced by or incorporated with an antisense element. The classic allosteric engineering approaches, however, usually require the covalent attachment of the aptamer to NAE. Without NMR or crystal structures, it is very difficult to assemble them in such a way that steric clashes could be avoided. Although in vitro selection could be used to search for aptamer-NAE sequences that have the capacity to synchronize both functions and at the same time, maintain their full activities, there could be several outcomes based on the previous studies <a href="#pone.0001224-Soukup1">[14]</a>, <a href="#pone.0001224-Levy1">[16]</a>, <a href="#pone.0001224-Koizumi1">[26]</a>–<a href="#pone.0001224-Zivarts1">[32]</a>: (1) allosteric NAEs that maintain the full activities of both aptamer and NAE; (2) only one domain maintains its full activity; (3) both activities are affected; (4) effective allosteric control is limited to suppression only; (5) poor allosteric NAEs due to high background signal; (6) no allosteric activity.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In comparison with the optimized NAE-based ATP and ADP sensors reported in the literature <a href="#pone.0001224-Levy1">[16]</a>, <a href="#pone.0001224-Srinivasan1">[47]</a>, the sensors described in this study, which were created solely by rational design, have similar detection limit, detection range, specificity and selectivity. Given that the cleavage and fluorescence-signaling performance of MgZ were poorer when coupled with the ASAPs than with the unmodified substrate, the sensor performance might be further improved by screening a larger library of antisense elements via in vitro selection. It should be emphasized that in this study, we did not intend to optimize the sensors for practical applications but to illustrate how to incorporate aptamers in both DNA and RNA formats into the MgZ system. It is unclear, however, for what reasons that the all-RNA antisense elements we examined were not able to block the substrate from MgZ-catalyzed RNA cleavage, given that RNA-DNA hybridization, in general, has similar thermodynamic stability and instability to that of RNA-RNA duplex and DNA-DNA mismatches, respectively <a href="#pone.0001224-Sugimoto1">[58]</a>. Currently, we are exploring the ASAP-MgZ system for aptamer selection. Further assessment of the ASAP design is thus warranted in the near future.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Chemicals and Oligonucleotides</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>DMS, piperidine, tetrabutyl-ammonium fluoride (TBAF) solution (1M in tetrahydrofuran), Tween-20 and nucleotide analogs including adenosine, AMP, ADP, ATP and GTP were obtained from Sigma-Aldrich. dNTPs were purchased from MBI Fermentas. [γ-<sup>32</sup>P]ATP and [α-<sup>32</sup>P]dGTP were from GE Healthcare Life Sciences or Perkin Elmer. All other chemicals were from BioShop (Burlington, Canada). Water was purified with a Milli-Q Synthesis A10 water purification system and then autoclaved. Standard and modified oligonucleotides were prepared by solid phase synthesis (Integrated DNA Technologies or Keck Biotechnology Resource Laboratory, Yale University). All oligonucleotides were purified by 8% denaturing PAGE before use. The 2′-OH protecting TOM (triisopropylsilyloxymethyl) group of the RNA linkage within the chimeric DNA/RNA substrate was removed by adding 500 µL of TBAF to ~250 pmol of substrate which is in the form of a dry pellet, followed by overnight incubation at 60°C on a mixer. The reaction was quenched by adding 200 µL of H<sub>2</sub>O. The resulting product was precipitated with ethanol, resuspended in H<sub>2</sub>O, desalted on a Nanosep 3K device (Pall Life Sciences), and further purified by 8% denaturing PAGE.</p>


<h4>In vitro selection</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The starting library consists of 500 pmol of random sequences in the context of 5′-TTCTTGATCAA-N<sub>60</sub>-TGGCACCCGCATCG-3′, where N is 25% A, 25% G, 25% C and 25% T, plus 165 pmol of each of the six OA degenerate libraries <a href="#pone.0001224-Chiuman1">[42]</a>. To prepare the self-cleaving DNA construct (L) shown in <a href="#pone-0001224-g001">Figure 1</a> inset, the library was first <sup>32</sup>P-labeled at the 5′-end in the presence of T4 PNK (T4 polynucleotide kinase; MBI Fermentas) and 10 µCi [γ-<sup>32</sup>P]ATP for 30 min. Non-radioactive ATP was then added to a final concentration of 1 mM. The mixture was further incubated for 30 min to ensure complete phosphorylation. The resulting product was ligated to the chimeric DNA/RNA substrate, 5′-GATGTGTCCGTGCFrAQGGTTCGA-3′, in the presence of an appropriate template and T4 DNA ligase (MBI Fermentas). The ligated products were purified by 8% denaturing PAGE and resuspended in H<sub>2</sub>O.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>The library was heated at 90°C for 30 s and cooled to room temperature (~23°C) for ~10 min. Self-cleavage reaction was initiated by adding 5× selection buffer (250 mM HEPES, pH 7.5 at 23°C, 750 mM NaCl, 50 mM MgCl<sub>2</sub> and 0.005% Tween-20) to the library to produce the final concentrations of 1× selection buffer and 0.2 µM of the library. The reaction was first incubated at room temperature for 2 h. Then, the reaction condition was alternated between room temperature and 37°C every 30 min for another 2 h. The reaction was stopped by adding EDTA (pH 8.0 at 23°C) to a final concentration of 20 mM. The self-cleaved oligonucleotides were separated from the inactive species by 8% denaturing PAGE and used as the templates for PCR amplification.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>The oligonucleotides were PCR-amplified in a volume of 50 µL containing 75 mM Tris-HCl (pH 9.0), 2 mM MgCl<sub>2</sub>, 50 mM KCl, 20 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 0.2 mM each of the four dNTPs, 1.25 U <em>Thermus thermophilus</em> DNA polymerase (Biotools), 0.5 µM DNA primer 1 (5′-TTACA- TCTACAAACATGGTTCGATTCTTGA-3′) and 0.5 µM primer 2 (5′-CGATGCGGGTGCCA-3′). The PCR was carried out for 10 thermocycles of which the temperature was altered in the following order: 94°C, 30 s (2 min for the first cycle); 45°C, 45 s; 72°C, 45 s. A small portion of the PCR products was amplified in the second PCR using a similar condition described above but with 0.3 µM primers 2 and 3 (5′-TTACATCTACAAACATGGTTCGrA-3′) for 15 cycles of 94°C, 30 s (2 min for the first cycle); 53°C, 45 s; 72°C, 45 s. <sup>32</sup>P-labeling of the oligonucleotides was conducted similarly as in the second PCR but with 30 µM each of the four dNTPs, 10 µCi [α-<sup>32</sup>P]dGTP, 0.2 µM primers 2 and 3 in a total volume of 25 µL. The non-radiolabeled and <sup>32</sup>P-labeled PCR products were combined, precipitated with ethanol, resuspended in 90 µL of 0.25 M NaOH and incubated at 90°C for 10 min to cleave the single ribonucleotide linkage (rA) within one of the two strands. 10 µL of 3M NaOAc (pH 5.2) was subsequently added to neutralize the solution. The cleaved DNA strand was isolated by 8% denaturing PAGE, eluted from the gel, precipitated with ethanol and restored into the self-cleaving construct by DNA phosphorylation followed by DNA ligation, as described above. The selection was performed for another 11 cycles as described for the first round, except that the final DNA concentration in the self-cleavage reaction was reduced to ~0.1 µM and the reaction time was progressively shortened to 1 h in round 6, 10 min in round 7, 1 min in rounds 8 and 9, 30 s in round 10 and 6 s in rounds 11 and 12. Population G12 was cloned and sequenced for further analyses.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>As we found that G12 was dominated by selfish DNAs, some changes in the selection scheme have been made in order to reduce their chances of survival (see <a href="#s2">Results</a> section for details). The selection experiment was re-initiated using the G5 population and continued for another 7 cycles, where the incubation time was shortened from 1 min in round 6 (G5) to 8 s in rounds 7 and 8, 5 min in rounds 9 and 10, and 10 s in rounds 11 and 12. G12 was cloned and sequenced for further analyses.</p>


<h4>Reselection</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Two partially randomized libraries were chemically synthesized with a degeneracy of 0.3 based on the sequences of truncated MgZ-5 and MgZ-7 (<a href="#pone.0001224.s003">Figure S2</a>). The wild type sequence of MgZ-5 is 5′-TTCTTGATCAA<span class="underline">CCAGGTCGGGGCCGAAATATAAGATGTTTTGGGAGGCT- AAGCTAGGAAG</span>GACCACCCGCATCG-3′, where the randomized region is underlined; the wild type sequence of MgZ-7 is 5′-TTCTTGATCAA<span class="underline">GGATTATTACCAGGTCGGGGCCAAA- TTAACGGAGTTAATTAGGGAGGCTGGCACCC</span>GCATCGTCGGTAGTC-3′. Note that the 3′ primer-binding sites are different from that in the starting library. They were changed by design to avoid cross-contamination by each pool or by the parent population from the initial selection experiment. Consequently, primers 2 for PCR-amplification were revised to 5′-CGATGCGGGTGGTC-3′ and 5′-GACTACCGACGATGC-3′ for MgZ-5 and MgZ-7, respectively. 705 pmol of MgZ-5 sequence variants and 718 pmol of MgZ-7 variants were subjected to 6 rounds of selection based on the original selection scheme described above. The incubation times for rounds 1 to 6 were 30 min, 5 min, 30 s, 10 s, 5 min and 30 s. G6 populations were cloned and sequenced for further analyses.</p>


<h4>Kinetic analyses of the DNA enzymes</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Radioactive assays were carried out at 30°C under a single turnover condition with 5 nM substrate (see <a href="#pone.0001224.s001">Text S1</a> for substrate preparation) and 1 µM DNAzyme in 70 mM HEPES (pH 8.0) containing 40 mM MgCl<sub>2</sub>, 0.001% Tween-20. Reactions were initiated by adding each DNAzyme to a relevant substrate that was pre-incubated in the reaction buffer, and stopped by adding EDTA to a final concentration of 80 mM. The cleavage products were precipitated with ethanol and separated by 10% denaturing PAGE. Cleavage fractions were quantitated by using the Storm 820 phosphorimager along with the Molecular Dynamics software. Cleavage assays were conducted over 10 time points over the course of 1 h or 9 time points over the course of 30 min (in the cases of M18 and M19 constructs; see <a href="#pone.0001224.s008">Figures S7</a> and <a href="#pone.0001224.s010">S9</a>). Each time point was repeated in at least three independent experiments. Rate constants were obtained by fitting the cleavage fraction vs. time data to either a single (<em>Y</em> = <em>Y</em><sub>f</sub>[1−e<sup>−<em>k</em><sub>c</sub></sup><em><sup>t</sup></em>]) or a double (<em>Y</em> = <em>Y</em><sub>f1</sub>[1–e<sup>−<em>k</em><sub>c1</sub></sup><em><sup>t</sup></em>]+<em>Y</em><sub>f2</sub>[1−e<sup>−<em>k</em><sub>c2</sub></sup><em><sup>t</sup></em>]) exponential equation with <em>R</em><sup>2</sup> &gt;0.99 by using GraphPad software Prism 4.03. <em>Y</em> represents cleavage yield; <em>Y</em><sub>f</sub> represents final cleavage yield; <em>k</em><sub>c</sub> represents the observed cleavage rate.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Fluorescent assays were performed on a Cary Eclipse Fluorescence Spectrophotometer (Varian). Fluorescence signals were recorded every 15 s at 800 V. Fluorescein was excited at 495 nm (5 nm bandwidth) and fluorescence was monitored at 520 nm (5 nm bandpass). Fluorescence of the substrate in the reaction buffer was monitored for at least 5 min before the addition of the DNAzyme. Fluorescence after DNAzyme addition was monitored for at least 30 min. Background signal was determined from the fluorescence output of the reaction buffer alone. The average background signal over the course of at least 5 min of fluorescence monitoring was subtracted from the sample readings before <em>F</em>/<em>F</em><sub>o</sub> was computed. Thus, <em>F</em>/<em>F</em><sub>o</sub> = (<em>F</em>−average background)/(average <em>F</em><sub>o</sub>−average background). <em>F</em> represents fluorescence reading; average <em>F</em><sub>o</sub> is the average signal of the substrate during the initial ~5 min of fluorescence monitoring; <em>F</em>/<em>F</em><sub>o</sub> represents signal enhancement. The final reaction condition was the same as described above for the radioactive assays. Fluorescence kinetics for every DNAzyme construct was done in triplicate. The signaling rates were determined by fitting the <em>F</em>/<em>F<sub>o</sub></em> vs. time data to a modified single exponential equation (<em>F</em>/<em>F</em><sub>o</sub> = initial <em>F</em>/<em>F</em><sub>o</sub>+(final <em>F</em>/<em>F</em><sub>o</sub>−initial <em>F</em>/<em>F</em><sub>o</sub>)×(1−e<sup>−<em>k</em><sub>s</sub></sup><em><sup>t</sup></em>); initial <em>F</em>/<em>F</em><sub>o</sub> = 1.0) with <em>R</em><sup>2</sup>&gt;0.93. <em>k</em><sub>s</sub> represents the rate of signal enhancement.</p>


<h4>Assays to test sensor performance</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Radioactive cleavage assays were carried out at room temperature with 5 nM substrate-aptamer complex and 50 nM (or 1 µM in the case of ADP sensor) DNAzyme in 50 mM HEPES (pH 7.0) containing 20 mM MgCl<sub>2</sub>, 0.001% Tween-20. Note: these reaction conditions represent the optimal conditions that yield the best signal enhancement in the presence of 1 mM target. Reactions were initiated by adding the DNAzyme to the substrate-aptamer complex that was pre-incubated in the reaction buffer containing the ligand at various concentrations. Each reaction was stopped by adding EDTA to a final concentration of 40 mM. The cleavage products were precipitated with ethanol and separated by 10% denaturing PAGE. Cleavage fractions were quantitated by phosphorimaging.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Fluorescent assays were performed similarly. The fluorescence intensity of the substrate-aptamer complex in the absence or presence of each relevant target was monitored for at least 5 min before the addition of the DNAzyme. Fluorescence was monitored for ~90 min (for ATP sensor) or ~60 min (for ADP sensor) after the DNAzyme addition. Signals were recorded every 30 s. <em>F</em>/<em>F</em><sub>o</sub> was computed as described above. Note that <em>F</em><sub>o</sub> here is the signal of the substrate-aptamer complex with or without the ligand present at various concentrations before the addition of the DNAzyme. Fluorescence kinetics for every ligand type or concentration was done in triplicate. The initial rate of signal enhancement (<em>v</em><sub>o</sub>) was approximated by using a linear equation (<em>F</em>/<em>F</em><sub>o</sub> = <em>v</em><sub>o</sub><em>t</em>+1.0; <em>F</em>/<em>F</em><sub>o</sub> = 1.0 at <em>t</em> = 0 min) to fit the <em>F</em>/<em>F<sub>o</sub></em> vs. time data, which were collected during the first ~10 min (for ATP sensor) or ~3 min (for ADP sensor) once DNAzyme was added to the reaction mixture. The dissociation constant of the ligand (<em>K</em><sub>d</sub>) was extracted by fitting the <em>v</em><sub>o</sub> vs. [ligand] data to the one site binding hyperbola equation (<em>v</em><sub>o</sub> = (<em>v</em><sub>max</sub>×[ligand])/(<em>K</em><sub>d</sub>+[ligand])) with <em>R</em><sup>2</sup> &gt;0.98 (note: <em>v</em><sub>max</sub> = maximal rate of signal enhancement; [ligand] = ligand concentration).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001224"></div><a name="pone.0001224.s001" id="pone.0001224.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001224.s001">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Supplementary Methods and Figure Legends</p>
<p class="siDoi">doi:10.1371/journal.pone.0001224.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0001224.s002" id="pone.0001224.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001224.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">MgZ species and selfish DNAs from G12</p>
<p class="siDoi">doi:10.1371/journal.pone.0001224.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(3.38 MB TIF)</p>
<a name="pone.0001224.s003" id="pone.0001224.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001224.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Selfish DNA G12SD-1</p>
<p class="siDoi">doi:10.1371/journal.pone.0001224.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.95 MB TIF)</p>
<a name="pone.0001224.s004" id="pone.0001224.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001224.s004">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Mutational analysis of MgZ-5 through reselection</p>
<p class="siDoi">doi:10.1371/journal.pone.0001224.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.43 MB TIF)</p>
<a name="pone.0001224.s005" id="pone.0001224.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001224.s005">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">DMS methylation interference pattern of a selected MgZ</p>
<p class="siDoi">doi:10.1371/journal.pone.0001224.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(1.56 MB TIF)</p>
<a name="pone.0001224.s006" id="pone.0001224.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001224.s006">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Initial truncation study of MgZ</p>
<p class="siDoi">doi:10.1371/journal.pone.0001224.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(1.72 MB TIF)</p>
<a name="pone.0001224.s007" id="pone.0001224.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001224.s007">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Characterization of stem P3</p>
<p class="siDoi">doi:10.1371/journal.pone.0001224.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(1.70 MB TIF)</p>
<a name="pone.0001224.s008" id="pone.0001224.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001224.s008">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Deletion study of P1 and P2</p>
<p class="siDoi">doi:10.1371/journal.pone.0001224.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(2.41 MB TIF)</p>
<a name="pone.0001224.s009" id="pone.0001224.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001224.s009">Figure S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Substrate flexibility</p>
<p class="siDoi">doi:10.1371/journal.pone.0001224.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(1.50 MB TIF)</p>
<a name="pone.0001224.s010" id="pone.0001224.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001224.s010">Figure S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Kinetic analyses of various DNAzyme constructs coupled with S3 and S4</p>
<p class="siDoi">doi:10.1371/journal.pone.0001224.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.63 MB TIF)</p>
<a name="pone.0001224.s011" id="pone.0001224.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001224.s011">Figure S10. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">pH effects of nucleic acid analogs</p>
<p class="siDoi">doi:10.1371/journal.pone.0001224.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(3.21 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Li lab members for comments on the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YL WC. Performed the experiments: WC. Analyzed the data: WC. Contributed reagents/materials/analysis tools: YL. Wrote the paper: YL WC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001224-Robertson1" id="pone.0001224-Robertson1"></a>Robertson DL, Joyce GF (1990) Selection in vitro of an RNA enzyme that specifically cleaves single-stranded DNA. Nature  344: 467–468.  <ul class="find" data-citedArticleID="1022044" data-doi="10.1038/344467a0"><li><a href="http://dx.doi.org/10.1038/344467a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+in+vitro+of+an+RNA+enzyme+that+specifically+cleaves+single-stranded+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+in+vitro+of+an+RNA+enzyme+that+specifically+cleaves+single-stranded+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001224-Ellington1" id="pone.0001224-Ellington1"></a>Ellington AD, Szostak JW (1990) In vitro selection of RNA molecules that bind specific ligands. Nature  346: 818–822.  <ul class="find" data-citedArticleID="1021992" data-doi="10.1038/346818a0"><li><a href="http://dx.doi.org/10.1038/346818a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+selection+of+RNA+molecules+that+bind+specific+ligands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+selection+of+RNA+molecules+that+bind+specific+ligands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001224-Tuerk1" id="pone.0001224-Tuerk1"></a>Tuerk C, Gold L (1990) Systematic evolution of ligands by exponential enrichment: RNA ligands to bacteriophage T4 DNA polymerase. Science  249: 505–510.  <ul class="find" data-citedArticleID="1022080" data-doi="10.1126/science.2200121"><li><a href="http://dx.doi.org/10.1126/science.2200121" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+evolution+of+ligands+by+exponential+enrichment%3A+RNA+ligands+to+bacteriophage+T4+DNA+polymerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+evolution+of+ligands+by+exponential+enrichment%3A+RNA+ligands+to+bacteriophage+T4+DNA+polymerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001224-Lee1" id="pone.0001224-Lee1"></a>Lee JF, Hesselberth JR, Meyers LA, Ellington AD (2004) Aptamer Database. Nucleic Acids Res  32: D95–D100.  <ul class="find" data-citedArticleID="1022014"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Aptamer+Database.&amp;auth=&amp;atitle=Aptamer+Database." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aptamer+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aptamer+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001224-Thodima1" id="pone.0001224-Thodima1"></a>Thodima V, Piroozina M, Deng Y (2006) RiboaptDB: A Comprehensive Database of Ribozymes and Aptamers. BMC Bioinformatics  7: Suppl. 2S6.  <ul class="find" data-citedArticleID="1022076" data-doi="10.1186/1471-2105-7-s2-s6"><li><a href="http://dx.doi.org/10.1186/1471-2105-7-s2-s6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RiboaptDB%3A+A+Comprehensive+Database+of+Ribozymes+and+Aptamers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RiboaptDB%3A+A+Comprehensive+Database+of+Ribozymes+and+Aptamers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001224-Lee2" id="pone.0001224-Lee2"></a>Lee JF, Stovall GM, Ellington AD (2006) Aptamer therapeutics advance. Curr Opin Chem Biol  10: 282–289.  <ul class="find" data-citedArticleID="1022016" data-doi="10.1016/j.cbpa.2006.03.015"><li><a href="http://dx.doi.org/10.1016/j.cbpa.2006.03.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aptamer+therapeutics+advance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aptamer+therapeutics+advance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001224-Hamaguchi1" id="pone.0001224-Hamaguchi1"></a>Hamaguchi N, Ellington AD, Stanton M (2001) Aptamer beacons for the direct detection of proteins. Anal Biochem  294: 126–131.  <ul class="find" data-citedArticleID="1022002" data-doi="10.1006/abio.2001.5169"><li><a href="http://dx.doi.org/10.1006/abio.2001.5169" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aptamer+beacons+for+the+direct+detection+of+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aptamer+beacons+for+the+direct+detection+of+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001224-Yang1" id="pone.0001224-Yang1"></a>Yang CJ, Jockusch S, Vicens M, Turro NJ, Tan W (2005) Light-switching excimer probes for rapid protein monitoring in complex biological fluids. Proc Natl Acad Sci U S A  102: 17278–17283.  <ul class="find" data-citedArticleID="1022092" data-doi="10.1073/pnas.0508821102"><li><a href="http://dx.doi.org/10.1073/pnas.0508821102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Light-switching+excimer+probes+for+rapid+protein+monitoring+in+complex+biological+fluids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Light-switching+excimer+probes+for+rapid+protein+monitoring+in+complex+biological+fluids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001224-Stojanovic1" id="pone.0001224-Stojanovic1"></a>Stojanovic MN, Kolpashchikov DM (2004) Modular Aptameric Sensors. J Am Chem Soc  126: 9266–9270.  <ul class="find" data-citedArticleID="1022068" data-doi="10.1021/ja032013t"><li><a href="http://dx.doi.org/10.1021/ja032013t" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modular+Aptameric+Sensors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modular+Aptameric+Sensors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001224-Jhaveri1" id="pone.0001224-Jhaveri1"></a>Jhaveri SD, Kirby R, Conrad R, Maglott EJ, Bowser M, et al.  (2000) Designed Signaling Aptamers that Transduce Molecular Recognition to Changes in Fluorescence Intensity. J Am Chem Soc  122: 2469–2473.  <ul class="find" data-citedArticleID="1022010" data-doi="10.1021/ja992393b"><li><a href="http://dx.doi.org/10.1021/ja992393b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Designed+Signaling+Aptamers+that+Transduce+Molecular+Recognition+to+Changes+in+Fluorescence+Intensity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Designed+Signaling+Aptamers+that+Transduce+Molecular+Recognition+to+Changes+in+Fluorescence+Intensity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001224-Nutiu1" id="pone.0001224-Nutiu1"></a>Nutiu R, Li Y (2003) Structure-Switching Signaling Aptamers. J Am Chem Soc  125: 4771–4778.  <ul class="find" data-citedArticleID="1022038" data-doi="10.1021/ja028962o"><li><a href="http://dx.doi.org/10.1021/ja028962o" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure-Switching+Signaling+Aptamers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure-Switching+Signaling+Aptamers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001224-Hafner1" id="pone.0001224-Hafner1"></a>Hafner M, Schmitz A, Grüne I, Srivatsan SG, Paul B, et al.  (2006) Inhibition of cytohesins by SecinH3 leads to hepatic insulin resistance. Nature  444: 941–944.  <ul class="find" data-citedArticleID="1022000" data-doi="10.1038/nature05415"><li><a href="http://dx.doi.org/10.1038/nature05415" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+cytohesins+by+SecinH3+leads+to+hepatic+insulin+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+cytohesins+by+SecinH3+leads+to+hepatic+insulin+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001224-Tang1" id="pone.0001224-Tang1"></a>Tang J, Breaker RR (1997) Rational design of allosteric ribozymes. Chem Biol  4: 453–459.  <ul class="find" data-citedArticleID="1022074" data-doi="10.1016/s1074-5521(97)90197-6"><li><a href="http://dx.doi.org/10.1016/s1074-5521(97)90197-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rational+design+of+allosteric+ribozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rational+design+of+allosteric+ribozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001224-Soukup1" id="pone.0001224-Soukup1"></a>Soukup GA, Breaker RR (1999) Engineering precision RNA molecular switches. Proc Natl Acad Sci U S A  96: 3584–3589.  <ul class="find" data-citedArticleID="1022060" data-doi="10.1073/pnas.96.7.3584"><li><a href="http://dx.doi.org/10.1073/pnas.96.7.3584" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Engineering+precision+RNA+molecular+switches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Engineering+precision+RNA+molecular+switches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001224-Robertson2" id="pone.0001224-Robertson2"></a>Robertson MP, Ellington AD (1999) In vitro selection of an allosteric ribozyme that transduces analytes to amplicons. Nat Biotechnol  17: 62–66.  <ul class="find" data-citedArticleID="1022046"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+vitro+selection+of+an+allosteric+ribozyme+that+transduces+analytes+to+amplicons.&amp;auth=&amp;atitle=In+vitro+selection+of+an+allosteric+ribozyme+that+transduces+analytes+to+amplicons." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+selection+of+an+allosteric+ribozyme+that+transduces+analytes+to+amplicons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+selection+of+an+allosteric+ribozyme+that+transduces+analytes+to+amplicons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001224-Levy1" id="pone.0001224-Levy1"></a>Levy M, Ellington AD (2002) ATP-Dependent Allosteric DNA Enzymes. Chem Biol  9: 417–426.  <ul class="find" data-citedArticleID="1022018" data-doi="10.1016/s1074-5521(02)00123-0"><li><a href="http://dx.doi.org/10.1016/s1074-5521(02)00123-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ATP-Dependent+Allosteric+DNA+Enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ATP-Dependent+Allosteric+DNA+Enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001224-Hartig1" id="pone.0001224-Hartig1"></a>Hartig JS, Najafi-Shoushtari SH, Grüne I, Yan A, Ellington AD, et al.  (2002) Protein-dependent ribozymes report molecular interactions in real time. Nat Biotechnol  20: 717–722.  <ul class="find" data-citedArticleID="1022004" data-doi="10.1038/nbt0702-717"><li><a href="http://dx.doi.org/10.1038/nbt0702-717" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein-dependent+ribozymes+report+molecular+interactions+in+real+time." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein-dependent+ribozymes+report+molecular+interactions+in+real+time.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001224-Vaish1" id="pone.0001224-Vaish1"></a>Vaish NK, Dong F, Andrews L, Schweppe RE, Ahn NG, et al.  (2002) Monitoring post-translational modification of proteins with allosteric ribozymes. Nat Biotechnol  20: 810–815.  <ul class="find" data-citedArticleID="1022082" data-doi="10.1038/nbt719"><li><a href="http://dx.doi.org/10.1038/nbt719" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monitoring+post-translational+modification+of+proteins+with+allosteric+ribozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monitoring+post-translational+modification+of+proteins+with+allosteric+ribozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001224-Wang1" id="pone.0001224-Wang1"></a>Wang DY, Lai BH, Sen D (2002) A General Strategy for Effector-mediated Control of RNA-cleaving Ribozymes and DNA Enzymes. J Mol Biol  318: 33–43.  <ul class="find" data-citedArticleID="1022086" data-doi="10.1016/s0022-2836(02)00046-3"><li><a href="http://dx.doi.org/10.1016/s0022-2836(02)00046-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+General+Strategy+for+Effector-mediated+Control+of+RNA-cleaving+Ribozymes+and+DNA+Enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+General+Strategy+for+Effector-mediated+Control+of+RNA-cleaving+Ribozymes+and+DNA+Enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001224-Thompson1" id="pone.0001224-Thompson1"></a>Thompson KM, Syrett HA, Knudsen SM, Ellington AD (2002) Group I aptazymes as genetic regulatory switches. BMC Biotechnology  2: 21.  <ul class="find" data-citedArticleID="1022078" data-doi="10.1186/1472-6750-2-21"><li><a href="http://dx.doi.org/10.1186/1472-6750-2-21" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Group+I+aptazymes+as+genetic+regulatory+switches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Group+I+aptazymes+as+genetic+regulatory+switches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001224-NajafiShoushtari1" id="pone.0001224-NajafiShoushtari1"></a>Najafi-Shoushtari SH, Famulok M (2005) Competitive regulation of modular allosteric aptazymes by a small molecule and oligonucleotide effector. RNA  11: 1514–1520.  <ul class="find" data-citedArticleID="1022034" data-doi="10.1261/rna.2840805"><li><a href="http://dx.doi.org/10.1261/rna.2840805" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Competitive+regulation+of+modular+allosteric+aptazymes+by+a+small+molecule+and+oligonucleotide+effector." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Competitive+regulation+of+modular+allosteric+aptazymes+by+a+small+molecule+and+oligonucleotide+effector.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001224-Amontov1" id="pone.0001224-Amontov1"></a>Amontov S, Jäschke A (2006) Controlling the rate of organic reactions: rational design of allosteric Diels-Alderase ribozymes. Nucleic Acids Res  34: 5032–5038.  <ul class="find" data-citedArticleID="1021980" data-doi="10.1093/nar/gkl613"><li><a href="http://dx.doi.org/10.1093/nar/gkl613" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Controlling+the+rate+of+organic+reactions%3A+rational+design+of+allosteric+Diels-Alderase+ribozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Controlling+the+rate+of+organic+reactions%3A+rational+design+of+allosteric+Diels-Alderase+ribozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001224-Shen1" id="pone.0001224-Shen1"></a>Shen Y, Chiuman W, Brennan JD, Li Y (2006) Catalysis and rational engineering of trans-acting pH6DZ1, an RNA-cleaving and fluorescence-signaling deoxyribozyme with a four-way junction structure. ChemBioChem  7: 1343–1348.  <ul class="find" data-citedArticleID="1022056" data-doi="10.1002/cbic.200600195"><li><a href="http://dx.doi.org/10.1002/cbic.200600195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Catalysis+and+rational+engineering+of+trans-acting+pH6DZ1%2C+an+RNA-cleaving+and+fluorescence-signaling+deoxyribozyme+with+a+four-way+junction+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Catalysis+and+rational+engineering+of+trans-acting+pH6DZ1%2C+an+RNA-cleaving+and+fluorescence-signaling+deoxyribozyme+with+a+four-way+junction+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001224-NajafiShoushtari2" id="pone.0001224-NajafiShoushtari2"></a>Najafi-Shoushtari SH, Famulok M (2007) DNA aptamer-mediated regulation of the hairpin ribozyme by human alpha-thrombin. Blood Cells Mol Dis  38: 19–24.  <ul class="find" data-citedArticleID="1022036" data-doi="10.1016/j.bcmd.2006.10.007"><li><a href="http://dx.doi.org/10.1016/j.bcmd.2006.10.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+aptamer-mediated+regulation+of+the+hairpin+ribozyme+by+human+alpha-thrombin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+aptamer-mediated+regulation+of+the+hairpin+ribozyme+by+human+alpha-thrombin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001224-Yamazaki1" id="pone.0001224-Yamazaki1"></a>Yamazaki S, Tan L, Mayer G, Hartig JS, Song JN, et al.  (2007) Aptamer Displacement Identifies Alternative Small-Molecule Target Sites that Escape Viral Resistance. Chem Biol  14: 804–812.  <ul class="find" data-citedArticleID="1022090" data-doi="10.1016/j.chembiol.2007.06.003"><li><a href="http://dx.doi.org/10.1016/j.chembiol.2007.06.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aptamer+Displacement+Identifies+Alternative+Small-Molecule+Target+Sites+that+Escape+Viral+Resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aptamer+Displacement+Identifies+Alternative+Small-Molecule+Target+Sites+that+Escape+Viral+Resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001224-Koizumi1" id="pone.0001224-Koizumi1"></a>Koizumi M, Soukup GA, Kerr JN, Breaker RR (1999) Allosteric selection of ribozymes that respond to the second messengers cGMP and cAMP. Nat Struc Biol  6: 1062–1071.  <ul class="find" data-citedArticleID="1022012" data-doi="10.1093/nass/42.1.275"><li><a href="http://dx.doi.org/10.1093/nass/42.1.275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Allosteric+selection+of+ribozymes+that+respond+to+the+second+messengers+cGMP+and+cAMP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Allosteric+selection+of+ribozymes+that+respond+to+the+second+messengers+cGMP+and+cAMP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001224-Soukup2" id="pone.0001224-Soukup2"></a>Soukup GA, Emilsson GM, Breaker RR (2000) Altering molecular recognition of RNA aptamers by allosteric selection. J Mol Biol  298: 623–632.  <ul class="find" data-citedArticleID="1022062" data-doi="10.1006/jmbi.2000.3704"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3704" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Altering+molecular+recognition+of+RNA+aptamers+by+allosteric+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Altering+molecular+recognition+of+RNA+aptamers+by+allosteric+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001224-Soukup3" id="pone.0001224-Soukup3"></a>Soukup GA, DeRose EC, Koizumi M, Breaker RR (2001) Generating new ligand-binding RNAs by affinity maturation and disintegration of allosteric ribozymes RNA  7: 524–536.  <ul class="find" data-citedArticleID="1022064" data-doi="10.1017/s1355838201002175"><li><a href="http://dx.doi.org/10.1017/s1355838201002175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generating+new+ligand-binding+RNAs+by+affinity+maturation+and+disintegration+of+allosteric+ribozymes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generating+new+ligand-binding+RNAs+by+affinity+maturation+and+disintegration+of+allosteric+ribozymes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001224-Robertson3" id="pone.0001224-Robertson3"></a>Robertson MP, Ellington AD (2001) In vitro selection of nucleoprotein enzymes. Nat Biotechnol  19: 650–655.  <ul class="find" data-citedArticleID="1022048" data-doi="10.1038/90256"><li><a href="http://dx.doi.org/10.1038/90256" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+selection+of+nucleoprotein+enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+selection+of+nucleoprotein+enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001224-Piganeau1" id="pone.0001224-Piganeau1"></a>Piganeau N, Thuillier V, Famulok M (2001) In vitro selection of allosteric ribozymes: theory and experimental validation. J Mol Biol  312: 1177–1190.  <ul class="find" data-citedArticleID="1022042" data-doi="10.1006/jmbi.2001.4981"><li><a href="http://dx.doi.org/10.1006/jmbi.2001.4981" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+selection+of+allosteric+ribozymes%3A+theory+and+experimental+validation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+selection+of+allosteric+ribozymes%3A+theory+and+experimental+validation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001224-Robertson4" id="pone.0001224-Robertson4"></a>Robertson MP, Knudsen SM, Ellington AD (2004) In vitro selection of ribozymes dependent on peptides for activity. RNA  10: 114–127.  <ul class="find" data-citedArticleID="1022050" data-doi="10.1261/rna.5900204"><li><a href="http://dx.doi.org/10.1261/rna.5900204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+selection+of+ribozymes+dependent+on+peptides+for+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+selection+of+ribozymes+dependent+on+peptides+for+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001224-Zivarts1" id="pone.0001224-Zivarts1"></a>Zivarts M, Liu Y, Breaker RR (2005) Engineered allosteric ribozymes that respond to specific divalent metal ions. Nucleic Acids Res  33: 622–631.  <ul class="find" data-citedArticleID="1022094" data-doi="10.1093/nar/gki182"><li><a href="http://dx.doi.org/10.1093/nar/gki182" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Engineered+allosteric+ribozymes+that+respond+to+specific+divalent+metal+ions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Engineered+allosteric+ribozymes+that+respond+to+specific+divalent+metal+ions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001224-Fedor1" id="pone.0001224-Fedor1"></a>Fedor MJ, Williamson JR (2005) The catalytic diversity of RNAs. Nat Rev Mol Cell Biol  6: 399–412.  <ul class="find" data-citedArticleID="1021994" data-doi="10.1038/nrm1647"><li><a href="http://dx.doi.org/10.1038/nrm1647" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+catalytic+diversity+of+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+catalytic+diversity+of+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001224-Silverman1" id="pone.0001224-Silverman1"></a>Silverman SK (2005) In vitro selection, characterization, and application of deoxyribozymes that cleave RNA. Nucleic Acids Res  33: 6151–6163.  <ul class="find" data-citedArticleID="1022058" data-doi="10.1093/nar/gki930"><li><a href="http://dx.doi.org/10.1093/nar/gki930" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+selection%2C+characterization%2C+and+application+of+deoxyribozymes+that+cleave+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+selection%2C+characterization%2C+and+application+of+deoxyribozymes+that+cleave+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001224-Jenne1" id="pone.0001224-Jenne1"></a>Jenne A, Gmlein W, Raffler N, Famulok M (1999) Real-time characterization of ribozymes by fluorescence resonance energy transfer (FRET). Angew Chem Int Ed  38: 1300–1303.  <ul class="find" data-citedArticleID="1022008" data-doi="10.1002/(sici)1521-3773(19990503)38:9<1300::aid-anie1300&gt;3.0.co;2-q"><li><a href="http://dx.doi.org/10.1002/(sici)1521-3773(19990503)38:9<1300::aid-anie1300&gt;3.0.co;2-q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Real-time+characterization+of+ribozymes+by+fluorescence+resonance+energy+transfer+%28FRET%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Real-time+characterization+of+ribozymes+by+fluorescence+resonance+energy+transfer+%28FRET%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001224-Vitiello1" id="pone.0001224-Vitiello1"></a>Vitiello D, Pecchia B, Burke JM (2000) Intracellular ribozyme-catalyzed trans-cleavage of RNA monitored by fluorescence resonance energy transfer. RNA  6: 628–637.  <ul class="find" data-citedArticleID="1022084" data-doi="10.1017/s1355838200990964"><li><a href="http://dx.doi.org/10.1017/s1355838200990964" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intracellular+ribozyme-catalyzed+trans-cleavage+of+RNA+monitored+by+fluorescence+resonance+energy+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intracellular+ribozyme-catalyzed+trans-cleavage+of+RNA+monitored+by+fluorescence+resonance+energy+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001224-Stojanovic2" id="pone.0001224-Stojanovic2"></a>Stojanovic MN, De Prada P, Landry DW (2000) Homogeneous assays based on deoxyribozyme catalysis. Nucleic Acids Res  28: 2915–2918.  <ul class="find" data-citedArticleID="1022070" data-doi="10.1093/nar/28.15.2915"><li><a href="http://dx.doi.org/10.1093/nar/28.15.2915" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homogeneous+assays+based+on+deoxyribozyme+catalysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homogeneous+assays+based+on+deoxyribozyme+catalysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001224-Frauendorf1" id="pone.0001224-Frauendorf1"></a>Frauendorf C, Jäschke A (2001) Detection of small organic analytes by fluorescing molecular switches. Bioorg Med Chem  9: 2521–2524.  <ul class="find" data-citedArticleID="1021996" data-doi="10.1016/s0968-0896(01)00027-x"><li><a href="http://dx.doi.org/10.1016/s0968-0896(01)00027-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+small+organic+analytes+by+fluorescing+molecular+switches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+small+organic+analytes+by+fluorescing+molecular+switches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001224-Frster1" id="pone.0001224-Frster1"></a>Förster T (1948) Intermolecular energy migration and fluorescence. Ann Phys  2: 55–75.  <ul class="find" data-citedArticleID="1021998"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Intermolecular+energy+migration+and+fluorescence.&amp;auth=&amp;atitle=Intermolecular+energy+migration+and+fluorescence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intermolecular+energy+migration+and+fluorescence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intermolecular+energy+migration+and+fluorescence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001224-Mei1" id="pone.0001224-Mei1"></a>Mei SH, Liu Z, Brennan JD, Li Y (2003) An Efficient RNA-Cleaving DNA Enzyme that Synchronizes Catalysis with Fluorescence Signaling. J Am Chem Soc  125: 412–420.  <ul class="find" data-citedArticleID="1022032" data-doi="10.1021/ja0281232"><li><a href="http://dx.doi.org/10.1021/ja0281232" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Efficient+RNA-Cleaving+DNA+Enzyme+that+Synchronizes+Catalysis+with+Fluorescence+Signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Efficient+RNA-Cleaving+DNA+Enzyme+that+Synchronizes+Catalysis+with+Fluorescence+Signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001224-Liu1" id="pone.0001224-Liu1"></a>Liu Z, Mei SH, Brennan JD, Li Y (2003) Assemblage of Signaling DNA Enzymes with Intriguing Metal-Ion Specificities and pH Dependences. J Am Chem Soc  125: 7539–7545.  <ul class="find" data-citedArticleID="1022024" data-doi="10.1021/ja035208+"><li><a href="http://dx.doi.org/10.1021/ja035208+" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assemblage+of+Signaling+DNA+Enzymes+with+Intriguing+Metal-Ion+Specificities+and+pH+Dependences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assemblage+of+Signaling+DNA+Enzymes+with+Intriguing+Metal-Ion+Specificities+and+pH+Dependences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001224-Chiuman1" id="pone.0001224-Chiuman1"></a>Chiuman W, Li Y (2006) Revitalization of Six Abandoned Catalytic DNA Species Reveals a Common Three-way Junction Framework and Diverse Catalytic Cores. J Mol Biol  357: 748–754.  <ul class="find" data-citedArticleID="1021986" data-doi="10.1016/j.jmb.2006.01.036"><li><a href="http://dx.doi.org/10.1016/j.jmb.2006.01.036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Revitalization+of+Six+Abandoned+Catalytic+DNA+Species+Reveals+a+Common+Three-way+Junction+Framework+and+Diverse+Catalytic+Cores." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Revitalization+of+Six+Abandoned+Catalytic+DNA+Species+Reveals+a+Common+Three-way+Junction+Framework+and+Diverse+Catalytic+Cores.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001224-Chiuman2" id="pone.0001224-Chiuman2"></a>Chiuman W, Li Y (2006) Evolution of High-Branching Deoxyribozymes from a Catalytic DNA with a Three-Way Junction. Chem Biol  13: 1061–1069.  <ul class="find" data-citedArticleID="1021988" data-doi="10.1016/j.chembiol.2006.08.009"><li><a href="http://dx.doi.org/10.1016/j.chembiol.2006.08.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+High-Branching+Deoxyribozymes+from+a+Catalytic+DNA+with+a+Three-Way+Junction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+High-Branching+Deoxyribozymes+from+a+Catalytic+DNA+with+a+Three-Way+Junction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001224-Rupcich1" id="pone.0001224-Rupcich1"></a>Rupcich N, Chiuman W, Nutiu R, Mei S, Flora KK, et al.  (2006) Quenching of Fluorophore-Labeled DNA Oligonucleotides by Divalent Metal Ions: Implications for Selection, Design, and Applications of Signaling Aptamers and Signaling Deoxyribozymes. J Am Chem Soc  128: 780–790.  <ul class="find" data-citedArticleID="1022052" data-doi="10.1021/ja053336n"><li><a href="http://dx.doi.org/10.1021/ja053336n" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quenching+of+Fluorophore-Labeled+DNA+Oligonucleotides+by+Divalent+Metal+Ions%3A+Implications+for+Selection%2C+Design%2C+and+Applications+of+Signaling+Aptamers+and+Signaling+Deoxyribozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quenching+of+Fluorophore-Labeled+DNA+Oligonucleotides+by+Divalent+Metal+Ions%3A+Implications+for+Selection%2C+Design%2C+and+Applications+of+Signaling+Aptamers+and+Signaling+Deoxyribozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001224-Huizenga1" id="pone.0001224-Huizenga1"></a>Huizenga DE, Szostak JW (1995) A DNA Aptamer That Binds Adenosine and ATP. Biochemistry  34: 656–665.  <ul class="find" data-citedArticleID="1022006" data-doi="10.1021/bi00002a033"><li><a href="http://dx.doi.org/10.1021/bi00002a033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+DNA+Aptamer+That+Binds+Adenosine+and+ATP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+DNA+Aptamer+That+Binds+Adenosine+and+ATP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001224-Lin1" id="pone.0001224-Lin1"></a>Lin CH, Patel DJ (1997) Structural basis of DNA folding and recognition in an AMP-DNA aptamer complex: distinct architectures but common recognition motifs for DNA and RNA aptamers complexed to AMP. Chem Biol  4: 817–832.  <ul class="find" data-citedArticleID="1022022" data-doi="10.1016/s1074-5521(97)90115-0"><li><a href="http://dx.doi.org/10.1016/s1074-5521(97)90115-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+of+DNA+folding+and+recognition+in+an+AMP-DNA+aptamer+complex%3A+distinct+architectures+but+common+recognition+motifs+for+DNA+and+RNA+aptamers+complexed+to+AMP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+of+DNA+folding+and+recognition+in+an+AMP-DNA+aptamer+complex%3A+distinct+architectures+but+common+recognition+motifs+for+DNA+and+RNA+aptamers+complexed+to+AMP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001224-Srinivasan1" id="pone.0001224-Srinivasan1"></a>Srinivasan J, Cload ST, Hamaguchi N, Kurz J, Keene S, et al.  (2004) ADP-Specific Sensors Enable Universal Assay of Protein Kinase Activity. Chem Biol  11: 499–508.  <ul class="find" data-citedArticleID="1022066" data-doi="10.1016/j.chembiol.2004.03.014"><li><a href="http://dx.doi.org/10.1016/j.chembiol.2004.03.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ADP-Specific+Sensors+Enable+Universal+Assay+of+Protein+Kinase+Activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ADP-Specific+Sensors+Enable+Universal+Assay+of+Protein+Kinase+Activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001224-Breaker1" id="pone.0001224-Breaker1"></a>Breaker RR, Joyce GF (1994) Emergence of a Replicating Species from an in vitro RNA Evolution Reaction. Proc Natl Acad Sci U S A  91: 6093–6097.  <ul class="find" data-citedArticleID="1021982" data-doi="10.1073/pnas.91.13.6093"><li><a href="http://dx.doi.org/10.1073/pnas.91.13.6093" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emergence+of+a+Replicating+Species+from+an+in+vitro+RNA+Evolution+Reaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emergence+of+a+Replicating+Species+from+an+in+vitro+RNA+Evolution+Reaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001224-Breaker2" id="pone.0001224-Breaker2"></a>Breaker RR, Joyce GF (1995) A DNA enzyme with Mg<sup>2+</sup>-dependent RNA phosphoesterase activity. Chem Biol  2: 655–660.  <ul class="find" data-citedArticleID="1021984" data-doi="10.1016/1074-5521(95)90028-4"><li><a href="http://dx.doi.org/10.1016/1074-5521(95)90028-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+DNA+enzyme+with+Mg2%2B-dependent+RNA+phosphoesterase+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+DNA+enzyme+with+Mg2%2B-dependent+RNA+phosphoesterase+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001224-Wang2" id="pone.0001224-Wang2"></a>Wang W, Billen LP, Li Y (2002) Sequence Diversity, Metal Specificity, and Catalytic Proficiency of Metal-Dependent Phosphorylating DNA Enzymes. Chem Biol  9: 507–517.  <ul class="find" data-citedArticleID="1022088" data-doi="10.1016/s1074-5521(02)00127-8"><li><a href="http://dx.doi.org/10.1016/s1074-5521(02)00127-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+Diversity%2C+Metal+Specificity%2C+and+Catalytic+Proficiency+of+Metal-Dependent+Phosphorylating+DNA+Enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+Diversity%2C+Metal+Specificity%2C+and+Catalytic+Proficiency+of+Metal-Dependent+Phosphorylating+DNA+Enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001224-Marras1" id="pone.0001224-Marras1"></a>Marras SAE, Kramer FR, Tyagi S (2002) Efficiencies of fluorescence resonance energy transfer and contact-mediated quenching in oligonucleotide probes. Nucleic Acids Res  30: e122.  <ul class="find" data-citedArticleID="1022030" data-doi="10.1093/nar/gnf121"><li><a href="http://dx.doi.org/10.1093/nar/gnf121" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficiencies+of+fluorescence+resonance+energy+transfer+and+contact-mediated+quenching+in+oligonucleotide+probes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficiencies+of+fluorescence+resonance+energy+transfer+and+contact-mediated+quenching+in+oligonucleotide+probes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001224-Liu2" id="pone.0001224-Liu2"></a>Liu J, Lu Y (2003) Improving Fluorescent DNAzyme Biosensors by Combining Inter- and Intramolecular Quenchers. Anal Chem  75: 6666–6672.  <ul class="find" data-citedArticleID="1022026" data-doi="10.1021/ac034924r"><li><a href="http://dx.doi.org/10.1021/ac034924r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improving+Fluorescent+DNAzyme+Biosensors+by+Combining+Inter-+and+Intramolecular+Quenchers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improving+Fluorescent+DNAzyme+Biosensors+by+Combining+Inter-+and+Intramolecular+Quenchers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001224-Liu3" id="pone.0001224-Liu3"></a>Liu J, Brown AK, Meng X, Cropek DM, Istok JD, et al.  (2007) A catalytic beacon sensor for uranium with parts-per-trillion sensitivity and millionfold selectivity. Proc Natl Acad Sci U S A  104: 2056–2061.  <ul class="find" data-citedArticleID="1022028" data-doi="10.1073/pnas.0607875104"><li><a href="http://dx.doi.org/10.1073/pnas.0607875104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+catalytic+beacon+sensor+for+uranium+with+parts-per-trillion+sensitivity+and+millionfold+selectivity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+catalytic+beacon+sensor+for+uranium+with+parts-per-trillion+sensitivity+and+millionfold+selectivity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001224-Chiuman3" id="pone.0001224-Chiuman3"></a>Chiuman W, Li Y (2007) Efficient signaling platforms built from a small catalytic DNA and doubly labeled fluorogenic substrates. Nucleic Acids Res  35: 401–405.  <ul class="find" data-citedArticleID="1021990" data-doi="10.1093/nar/gkl1056"><li><a href="http://dx.doi.org/10.1093/nar/gkl1056" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+signaling+platforms+built+from+a+small+catalytic+DNA+and+doubly+labeled+fluorogenic+substrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+signaling+platforms+built+from+a+small+catalytic+DNA+and+doubly+labeled+fluorogenic+substrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001224-Santoro1" id="pone.0001224-Santoro1"></a>Santoro SW, Joyce GF (1997) A general purpose RNA-cleaving DNA enzyme. Proc Natl Acad Sci U S A  94: 4262–4266.  <ul class="find" data-citedArticleID="1022054" data-doi="10.1073/pnas.94.9.4262"><li><a href="http://dx.doi.org/10.1073/pnas.94.9.4262" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+purpose+RNA-cleaving+DNA+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+purpose+RNA-cleaving+DNA+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001224-Li1" id="pone.0001224-Li1"></a>Li J, Zheng W, Kwon AH, Lu Y (2000) In vitro selection and characterization of a highly efficient Zn(II)-dependent RNA-cleaving deoxyribozyme. Nucleic Acids Res  28: 481–488.  <ul class="find" data-citedArticleID="1022020" data-doi="10.1093/nar/28.2.481"><li><a href="http://dx.doi.org/10.1093/nar/28.2.481" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+selection+and+characterization+of+a+highly+efficient+Zn%28II%29-dependent+RNA-cleaving+deoxyribozyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+selection+and+characterization+of+a+highly+efficient+Zn%28II%29-dependent+RNA-cleaving+deoxyribozyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001224-Peracchi1" id="pone.0001224-Peracchi1"></a>Peracchi A (2000) Preferential activation of the 8–17 deoxyribozyme by Ca(2+) ions. Evidence for the identity of 8–17 with the catalytic domain of the Mg5 deoxyribozyme. J Biol Chem  275: 11693–11697.  <ul class="find" data-citedArticleID="1022040" data-doi="10.1074/jbc.275.16.11693"><li><a href="http://dx.doi.org/10.1074/jbc.275.16.11693" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preferential+activation+of+the+8%E2%80%9317+deoxyribozyme+by+Ca%282%2B%29+ions.+Evidence+for+the+identity+of+8%E2%80%9317+with+the+catalytic+domain+of+the+Mg5+deoxyribozyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preferential+activation+of+the+8%E2%80%9317+deoxyribozyme+by+Ca%282%2B%29+ions.+Evidence+for+the+identity+of+8%E2%80%9317+with+the+catalytic+domain+of+the+Mg5+deoxyribozyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001224-Sugimoto1" id="pone.0001224-Sugimoto1"></a>Sugimoto N, Nakano S, Katoh M, Matsumura A, Nakamuta H, et al.  (1995) Thermodynamic Parameters To Predict Stability of RNA/DNA Hybrid Duplexes. Biochemistry  34: 11211–11216.  <ul class="find" data-citedArticleID="1022072" data-doi="10.1021/bi00035a029"><li><a href="http://dx.doi.org/10.1021/bi00035a029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thermodynamic+Parameters+To+Predict+Stability+of+RNA%2FDNA+Hybrid+Duplexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thermodynamic+Parameters+To+Predict+Stability+of+RNA%2FDNA+Hybrid+Duplexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.XML" value="114455"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.PDF" value="509384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g001.PNG_L" value="619104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g001.PNG_M" value="85006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g001.PNG_S" value="5736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g001.TIF" value="1028940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g001.PNG_I" value="32842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g002.PNG_L" value="529110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g002.PNG_M" value="91062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g002.PNG_S" value="9254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g002.TIF" value="901146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g002.PNG_I" value="72659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g003.PNG_L" value="234088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g003.PNG_M" value="81811"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g003.PNG_S" value="6274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g003.TIF" value="389980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g003.PNG_I" value="32473"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g004.PNG_L" value="1160915"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g004.PNG_M" value="105604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g004.PNG_S" value="5258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g004.TIF" value="1726304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g004.PNG_I" value="38302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g005.PNG_L" value="90925"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g005.PNG_M" value="27580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g005.PNG_S" value="3352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g005.TIF" value="154806"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g005.PNG_I" value="11685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g006.PNG_L" value="1406360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g006.PNG_M" value="135682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g006.PNG_S" value="6032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g006.TIF" value="2145760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g006.PNG_I" value="49222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g007.PNG_L" value="756422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g007.PNG_M" value="133830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g007.PNG_S" value="9788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g007.TIF" value="1252470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.g007.PNG_I" value="82942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.s001.DOC" value="54272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.s002.TIF" value="3379627"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.s003.TIF" value="511037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.s004.TIF" value="431728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.s005.TIF" value="872662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.s006.TIF" value="1720981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.s007.TIF" value="1697173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.s008.TIF" value="2412507"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.s009.TIF" value="1500841"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.s010.TIF" value="627049"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001224.s011.TIF" value="3212923"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001224&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001224" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001224&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001224&volume=&issue=&title=Simple Fluorescent Sensors Engineered with Catalytic DNA ‘MgZ’ Based on a Non-Classic Allosteric Design&author_name=William%20Chiuman%2C%20Yingfu%20Li&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001224" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001224" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001224" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001224&amp;t=Simple%20Fluorescent%20Sensors%20Engineered%20with%20Catalytic%20DNA%20%E2%80%98MgZ%E2%80%99%20Based%20on%20a%20Non-Classic%20Allosteric%20Design" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001224&title=Simple%20Fluorescent%20Sensors%20Engineered%20with%20Catalytic%20DNA%20%E2%80%98MgZ%E2%80%99%20Based%20on%20a%20Non-Classic%20Allosteric%20Design&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001224&amp;title=Simple%20Fluorescent%20Sensors%20Engineered%20with%20Catalytic%20DNA%20%E2%80%98MgZ%E2%80%99%20Based%20on%20a%20Non-Classic%20Allosteric%20Design" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001224" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001224&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Simple Fluorescent Sensors Engineered with Catalytic DNA ‘MgZ’ Based on a Non-Classic Allosteric Design';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001224';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Simple%20Fluorescent%20Sensors%20Engineered%20with%20Catalytic%20DNA%20%E2%80%98MgZ%E2%80%99%20Based%20on%20a%20Non-Classic%20Allosteric%20Design http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001224" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001224" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+cleavage%22" title="Search for articles in the subject area:'DNA cleavage'"><div class="flagText">DNA cleavage</div></a>
              <div data-categoryid="31951" data-articleid="25280"
                   data-categoryname="DNA cleavage"
                   class="flagImage" title="Flag 'DNA cleavage' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+isolation%22" title="Search for articles in the subject area:'DNA isolation'"><div class="flagText">DNA isolation</div></a>
              <div data-categoryid="21441" data-articleid="25280"
                   data-categoryname="DNA isolation"
                   class="flagImage" title="Flag 'DNA isolation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25280"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Engineering+and+technology%22" title="Search for articles in the subject area:'Engineering and technology'"><div class="flagText">Engineering and technology</div></a>
              <div data-categoryid="2605" data-articleid="25280"
                   data-categoryname="Engineering and technology"
                   class="flagImage" title="Flag 'Engineering and technology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="25280"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribozymes%22" title="Search for articles in the subject area:'Ribozymes'"><div class="flagText">Ribozymes</div></a>
              <div data-categoryid="17535" data-articleid="25280"
                   data-categoryname="Ribozymes"
                   class="flagImage" title="Flag 'Ribozymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sensory+receptors%22" title="Search for articles in the subject area:'Sensory receptors'"><div class="flagText">Sensory receptors</div></a>
              <div data-categoryid="17769" data-articleid="25280"
                   data-categoryname="Sensory receptors"
                   class="flagImage" title="Flag 'Sensory receptors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="25280"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8942'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4577'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2301&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=793">Referee Comments: Referee 2 </a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
