

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: MHC Adaptive Divergence between Closely Related and Sympatric African Cichlids</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000734"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000734" />

    <meta name="citation_title" content="MHC Adaptive Divergence between Closely Related and Sympatric African Cichlids"/>
    <meta itemprop="name" content="MHC Adaptive Divergence between Closely Related and Sympatric African Cichlids"/>

      <meta name="citation_author" content="Jonatan Blais"/>
            <meta name="citation_author_institution" content="Département de Biologie, Québec-Océan, Université Laval, Quebec, Québec, Canada"/>
            <meta name="citation_author_institution" content="Estación Biológica de Doñana, Consejo Superior de Investigaciones Científicas, Sevilla, Spain"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, University of Hull, Hull, United Kingdom"/>
      <meta name="citation_author" content="Ciro Rico"/>
            <meta name="citation_author_institution" content="Estación Biológica de Doñana, Consejo Superior de Investigaciones Científicas, Sevilla, Spain"/>
      <meta name="citation_author" content="Cock van Oosterhout"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, University of Hull, Hull, United Kingdom"/>
      <meta name="citation_author" content="Joanne Cable"/>
            <meta name="citation_author_institution" content="School of Biological Sciences, Cardiff University, Cardiff, United Kingdom"/>
      <meta name="citation_author" content="George F. Turner"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, University of Hull, Hull, United Kingdom"/>
      <meta name="citation_author" content="Louis Bernatchez"/>
            <meta name="citation_author_institution" content="Département de Biologie, Québec-Océan, Université Laval, Quebec, Québec, Canada"/>

    <meta name="citation_date" content="2007/8/15"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000734.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e734"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Latitudinal gradients of biodiversity: pattern, process, scale, and synthesis.; citation_author=MR Willig; citation_author=DM Kaufman; citation_author=RD Stevens; citation_journal_title=Ann Rev Ecol Evol Syst; citation_volume=34; citation_number=1; citation_pages=273-309; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The role of parasites in sympatric and allopatric host diversification.; citation_author=A Buckling; citation_author=PB Rainey; citation_journal_title=Nature; citation_volume=420; citation_number=2; citation_pages=496-499; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Parasitic exploitation as an engine of diversity.; citation_author=K Summers; citation_author=S McKeon; citation_author=J Sellars; citation_author=M Keusenkothen; citation_author=J Morris; citation_journal_title=Biol Rev; citation_volume=78; citation_number=3; citation_pages=639-675; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Disease and evolution.; citation_author=JBS Haldane; citation_journal_title=La Ricerca Scientifica Suppl A; citation_volume=19; citation_number=4; citation_pages=68-76; citation_date=1949; " />
      <meta name="citation_reference" content="citation_title=J. B. S. Haldane (1949) on infectious disease and evolution.; citation_author=J Lederberg; citation_journal_title=Genetics; citation_volume=153; citation_number=5; citation_pages=1-3; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Theory and speciation.; citation_author=M Turelli; citation_author=NH Barton; citation_author=JA Coyne; citation_journal_title=Trends Ecol Evol; citation_volume=16; citation_number=6; citation_pages=330-343; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Ancient co-speciation of simian foamy viruses and primates.; citation_author=WM Switzer; citation_author=M Salemi; citation_author=V Shanmugam; citation_author=F Gao; citation_author=M-e Cong; citation_journal_title=Nature; citation_volume=434; citation_number=7; citation_pages=376-380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Geographic Mosaic of Coevolution.; citation_author=JN Thompson; citation_number=8; citation_date=2005; citation_publisher=The University of Chicago Press; " />
      <meta name="citation_reference" content="citation_title=Parasites and the evolutionary diversification of primate clades.; citation_author=CL Nunn; citation_author=S Altizer; citation_author=W Sechrest; citation_author=KE Jones; citation_author=RA Barton; citation_journal_title=Am Nat; citation_volume=164; citation_number=9; citation_pages=S90-S103; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Relationship between host diversity and parasite diversity: flea assemblages on small mammals.; citation_author=BR Krasnov; citation_author=GI Shenbrot; citation_author=IS Khokhlova; citation_author=AA Degen; citation_journal_title=J Biogeogr; citation_volume=31; citation_number=10; citation_pages=1857-1866; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Comparative sequencing of human and chimpanzee MHC class I regions unveils insertions/deletions as the major path to genomic divergence.; citation_author=T Anzai; citation_author=T Shiina; citation_author=N Kimura; citation_author=K Yanagiya; citation_author=S Kohara; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=100; citation_number=11; citation_pages=7708-7713; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Major histocompatibility genes in the Lake Tana African large barb species flock: evidence for complete partitioning of class II B, but not class I, genes among different species.; citation_author=CP Kruiswijk; citation_author=T Hermsen; citation_author=Jv Heerwaarden; citation_author=B Dixon; citation_author=HFJ Savelkoul; citation_journal_title=Immunog; citation_volume=56; citation_number=12; citation_pages=894-908; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=initial sequencing and comparative analysis of the mouse genome.; citation_author=RH Waterston; citation_author=K Lindblad-Toh; citation_author=E Birney; citation_author=J Rogers; citation_author=JF Abril; citation_journal_title=Nature; citation_volume=420; citation_number=13; citation_pages=520-562; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of the Brown Norway rat yields insights into mammalian evolution.; citation_author=RA Gibbs; citation_author=GM Weinstock; citation_journal_title=Nature; citation_volume=428; citation_number=14; citation_pages=493-521; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MHC studies in nonmodel vertebrates: what have we learned about natural selection in 15 years?; citation_author=L Bernatchez; citation_author=C Landry; citation_journal_title=J Evol Biol; citation_volume=16; citation_number=15; citation_pages=363-377; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The foreign antigen binding site and T cell recognition regions of class I histocompatibility antigens.; citation_author=PJ Bjorkman; citation_author=MA Saper; citation_author=B Samraoui; citation_author=WS Bennett; citation_author=JL Strominger; citation_journal_title=Nature; citation_volume=329; citation_number=16; citation_pages=512-518; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Paternally inherited HLA alleles are associated with women's choice of male odor.; citation_author=S Jacob; citation_author=MK McClintock; citation_author=B Zelano; citation_author=C Ober; citation_journal_title=Nat Gen; citation_volume=30; citation_number=17; citation_pages=175-179; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Discrimination of MHC-derived odors by untrained mice is consistent with divergence in peptide-binding region residues.; citation_author=LS Carroll; citation_author=DJ Penn; citation_author=WK Potts; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=18; citation_pages=2187-2192; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Major histocompatibility complex and mate choice in a monogamous rodent.; citation_author=S Sommer; citation_journal_title=Behav Ecol Sociobiol; citation_volume=58; citation_number=19; citation_pages=181-189; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=“Good genes as heterozygosity”: MHC and mate choice in Atlantic salmon (Salmo salar) Proc R Soc Lond B; citation_author=C Landry; citation_author=D Garant; citation_author=P Duchesne; citation_author=L Bernatchez; citation_volume=268; citation_number=20; citation_pages=1279-1285; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mate choice decisions of stickleback females predictably modified by MHC peptide ligands.; citation_author=M Milinski; citation_author=S Griffiths; citation_author=KM Wegner; citation_author=TBH Reusch; citation_author=A Haas-Assenbaum; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=21; citation_pages=4414-4418; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution and explosive speciation: The cichlid fish model.; citation_author=TD Kocher; citation_journal_title=Nat Rev Gen; citation_volume=5; citation_number=22; citation_pages=288-298; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The mbuna cichlids of Lake Malawi: a model for rapid speciation and adaptive radiation.; citation_author=MJ Genner; citation_author=GF Turner; citation_journal_title=Fish and Fisheries; citation_volume=6; citation_number=23; citation_pages=1-34; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A preliminary survey of the cichlid fishes of rocky habitats in Lake Malawi.; citation_author=AJ Ribbink; citation_author=BA Marsh; citation_author=AC Marsh; citation_author=AC Ribbink; citation_author=BJ Sharp; citation_journal_title=S Afr J Zool; citation_volume=18; citation_number=24; citation_pages=147-310; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Explosive speciation of African cichlid fishes.; citation_author=GF Turner; citation_number=25; citation_pages=113-129; citation_date=1999; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Mol Ecol; citation_author=C Rico; citation_author=GF Turner; citation_volume=11; citation_number=26; citation_pages=1585-1590; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Parasite transmission: reconciling theory and reality.; citation_author=A Fenton; citation_author=JP Fairbairn; citation_author=R Norman; citation_author=PJ Hudson; citation_journal_title=J Anim Ecol; citation_volume=71; citation_number=27; citation_pages=893-905; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Sexual selection, parasites and bower height skew in a bower-building cichlid fish.; citation_author=MI Taylor; citation_author=GF Turner; citation_author=RL Robinson; citation_author=JR Stauffer; citation_journal_title=Anim Behav; citation_volume=56; citation_number=28; citation_pages=379-384; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Fitness correlates of male coloration in a Lake Victoria cichlid fish.; citation_author=ME Maan; citation_author=M van der Spoel; citation_author=PQ Jimenez; citation_author=JJM van Alphen; citation_author=O Seehausen; citation_journal_title=Behav Ecol; citation_volume=17; citation_number=29; citation_pages=691-699; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Female preference for conspecific males based on olfactory cues in a Lake Malawi cichlid fish.; citation_author=M Plenderleith; citation_author=C van Oosterhout; citation_author=RL Robinson; citation_author=GF Turner; citation_journal_title=Biol Lett; citation_volume=1; citation_number=30; citation_pages=411-414; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ecology and the origin of species.; citation_author=D Schluter; citation_journal_title=Trends Ecol Evol; citation_volume=16; citation_number=31; citation_pages=372-380; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Linkage Relationship and Haplotype Polymorphism Among Cichlid Mhc Class II B Loci.; citation_author=E Málaga-Trillo; citation_author=Z Zaleska-Rutczynska; citation_author=B McAndrew; citation_author=V Vincek; citation_author=F Figueroa; citation_journal_title=Genetics; citation_volume=149; citation_number=32; citation_pages=1527-1537; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Major histocompatibility complex class II genes of zebrafish.; citation_author=H Ono; citation_author=D Klein; citation_author=V Vincek; citation_author=F Figueroa; citation_author=C O'hUigin; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=89; citation_number=33; citation_pages=11886-11890; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Evolution and comparative genomics of odorant- and pheromone-associated genes in rodents.; citation_author=RD Emes; citation_author=SA Beatson; citation_author=CP Ponting; citation_author=L Goodstadt; citation_journal_title=Genome Research; citation_volume=14; citation_number=34; citation_pages=591-602; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolution of introns and exons of class II major histocompatibility complex genes of vertebrates.; citation_author=AL Hughes; citation_journal_title=Immunog; citation_volume=51; citation_number=35; citation_pages=473-486; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Inter- and Intralocus recombination drive MHC class IIB gene diversification in a teleost, the three-spined stickleback Gasterosteus aculeatus; citation_author=TBH Reusch; citation_author=Å Langefors; citation_journal_title=J Mol Evol; citation_volume=61; citation_number=36; citation_pages=531-541; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Automated phylogenetic detection of recombination using a genetic algorithm.; citation_author=SL Kosakovsky Pond; citation_author=D Posada; citation_author=MB Gravenor; citation_author=CH Woelk; citation_author=SDW Frost; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=37; citation_pages=1891-1901; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ecologically meaningful transformations for ordination of species data.; citation_author=P Legendre; citation_author=ED Gallagher; citation_journal_title=Oecologia; citation_volume=129; citation_number=38; citation_pages=271-280; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Recognition and polymorphism in host-parasite genetics.; citation_author=SA Frank; citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_volume=346; citation_number=39; citation_pages=283-293; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The Ecology of Adaptive Radiation;; citation_author=D Schluter; citation_number=40; citation_date=2000; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=The effect of hitch-hiking on genes linked to a balanced polymorphism in a subdivided population.; citation_author=MH Schierup; citation_author=X Vekemans; citation_author=D Charlesworth; citation_journal_title=Gen Res; citation_volume=76; citation_number=41; citation_pages=63-73; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Balancing selection, random genetic drift, and genetic variation at the major histocompatibility complex in two wild populations of guppies (Poecilia reticulata).; citation_author=C van Oosterhout; citation_author=DA Joyce; citation_author=SM Cummings; citation_author=J Blais; citation_author=NJ Barson; citation_journal_title=Evolution; citation_volume=60; citation_number=42; citation_pages=2562-2574; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Spawning colouration, parasites and habitat selection in Salvelinus alpinus: initiating speciation by sexual selection?; citation_author=F Skarstein; citation_author=I Folstad; citation_author=HP Rønning; citation_journal_title=J Fish Biol; citation_volume=67; citation_number=43; citation_pages=969-980; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Strong positive selection and habitat-specific amino acid substitution patterns in Mhc from an estuarine fish under intense pollution stress.; citation_author=S Cohen; citation_journal_title=Mol Biol Evol; citation_volume=10; citation_number=44; citation_pages=1870-1880; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=African cichlid fishes: Model systems for evolutionary biology.; citation_author=I Kornfield; citation_author=PF Smith; citation_journal_title=Ann Rev Ecol Syst; citation_volume=31; citation_number=45; citation_pages=163-196; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Colour diversification in Malawi cichlids: evidence for adaptation, reinforcement or sexual selection?; citation_author=JC Deutsch; citation_journal_title=Biol J Linn Soc; citation_volume=62; citation_number=46; citation_pages=1-14; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Phylogeny of a rapidly evolving clade: The cichlid fishes of Lake Malawi, East Africa.; citation_author=RC Albertson; citation_author=JA Markert; citation_author=PD Danley; citation_author=TD Kocher; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=96; citation_number=47; citation_pages=5107-5110; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Divergent selection on opsins drives incipient speciation in Lake Victoria cichlids.; citation_author=Y Terai; citation_author=O Seehausen; citation_author=T Sasaki; citation_author=K Takahashi; citation_author=S Mizoiri; citation_journal_title=PLoS Biology; citation_volume=4; citation_number=48; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Effects of sexual selection and life history on speciation: Species flocks in African cichlids and Hawaiian Drosophila.; citation_author=WJ Dominey; citation_number=49; citation_pages=231-249; citation_date=1984; citation_publisher=University of Maine at Orono Press; " />
      <meta name="citation_reference" content="citation_title=Can sympatric speciation by disruptive sexual selection explain rapid evolution of cichlid diversity in Lake Victoria?; citation_author=O Seehausen; citation_author=JJM van Alphen; citation_journal_title=Ecol Lett; citation_volume=2; citation_number=50; citation_pages=262-271; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Divergent selection during speciation of Lake Malawi cichlid fishes inferred from parallel radiations in nuptial coloration.; citation_author=CJ Allender; citation_author=O Seehausen; citation_author=ME Knight; citation_author=GF Turner; citation_author=N Maclean; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=100; citation_number=51; citation_pages=14074-14079; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Heritable true fitness and bright birds: a role for parasites?; citation_author=WD Hamilton; citation_author=M Zuk; citation_journal_title=Science; citation_volume=218; citation_number=52; citation_pages=384-387; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Sexual reproduction as an adaptation to resist parasites.; citation_author=WD Hamilton; citation_author=R Axelrod; citation_author=R Tanese; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=87; citation_number=53; citation_pages=3566-3573; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=MHC genotype and male ornamentation: Genetic evidence for the Hamilton-Zuk model.; citation_author=T Von Schantz; citation_author=H Wittzell; citation_author=G Goransson; citation_author=M Grahn; citation_author=K Persson; citation_journal_title=Proc R Soc Lond B; citation_volume=263; citation_number=54; citation_pages=265-271; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Alternative hypotheses linking the immune system and mate choice for good genes.; citation_author=DF Westneat; citation_author=TR Birkhead; citation_journal_title=Proc R Soc Lond B; citation_volume=265; citation_number=55; citation_pages=1065-1073; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Signaling health versus parasites.; citation_author=T Getty; citation_journal_title=Am Nat; citation_volume=159; citation_number=56; citation_pages=363-371; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Parasites as a viability cost of sexual selection in natural populations of mammals.; citation_author=SL Moore; citation_author=K Wilson; citation_journal_title=Science; citation_volume=297; citation_number=57; citation_pages=2015-2018; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The evolution of immune defense and song complexity in birds.; citation_author=LZ Garamszegi; citation_author=AP Moller; citation_author=J Erritzoe; citation_journal_title=Evolution; citation_volume=57; citation_number=58; citation_pages=905-912; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Nonlinear effects of female mate choice in wild threespine sticklebacks.; citation_author=J Blais; citation_author=C Rico; citation_author=L Bernatchez; citation_journal_title=Evolution; citation_volume=58; citation_number=59; citation_pages=2498-2510; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Female sticklebacks count alleles in a strategy of sexual selection explaining MHC polymorphism.; citation_author=TBH Reusch; citation_author=MA Haberli; citation_author=PB Aeschlimann; citation_author=M Milinski; citation_journal_title=Nature; citation_volume=414; citation_number=60; citation_pages=300-302; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The optimal number of major histocompatibility complex molecules in an individual.; citation_author=MA Nowak; citation_author=K Tarczy-Hornoch; citation_author=JM Austyn; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=89; citation_number=61; citation_pages=10896-10899; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Thymic selection does not limit the individual MHC diversity.; citation_author=JAM Borghans; citation_author=AJ Noest; citation_author=RJ De Boer; citation_journal_title=Eur J Immunol; citation_volume=33; citation_number=62; citation_pages=3353-3358; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Antigen presentation on MHC molecules as a diversity filter that enhances immune efficacy.; citation_author=HA van den Berg; citation_author=DA Rand; citation_journal_title=J Theor Biol; citation_volume=224; citation_number=63; citation_pages=249-267; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Skepticism towards Santa Rosalia, or why are there so few kinds of animals?; citation_author=J Felsenstein; citation_journal_title=Evolution; citation_volume=35; citation_number=64; citation_pages=124-138; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Malawi cichlids in their natural habitat.; citation_author=A Konings; citation_number=65; citation_pages=352; citation_date=2001; citation_publisher=Cichlid Press; " />
      <meta name="citation_reference" content="citation_title=Distinguishing migration from isolation: a Markov chain Monte Carlo approach.; citation_author=R Nielsen; citation_author=J Wakeley; citation_journal_title=Genetics; citation_volume=158; citation_number=66; citation_pages=885-896; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Pattern of nucleotide substitution at major histocompatibility complex class I loci reveals overdominant selection.; citation_author=AL Hughes; citation_author=M Nei; citation_journal_title=Nature; citation_volume=335; citation_number=67; citation_pages=167-170; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Codon-substitution models to detect adaptive evolution that account for heterogeneous selective pressures among site classes.; citation_author=Z Yang; citation_author=WJ Swanson; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=68; citation_pages=49-57; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Major-histocompatibility-complex variation in two species of cichlid fishes from Lake Malawi.; citation_author=H Ono; citation_author=C O'hUigin; citation_author=H Tichy; citation_author=D Klein; citation_journal_title=Mol Biol Evol; citation_volume=10; citation_number=69; citation_pages=1060-1072; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.; citation_author=JD Thompson; citation_author=TJ Gibson; citation_author=F Plewniak; citation_author=F Jeanmougin; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Res; citation_volume=24; citation_number=70; citation_pages=4876-4882; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees.; citation_author=N Saitou; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=4; citation_number=71; citation_pages=406-425; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Evaluation of methods for detecting recombination from DNA sequences: computer simulations.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=72; citation_pages=13757-13762; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Population biology of antigen presentation by MHC class I molecules.; citation_author=P Parham; citation_author=T Ohta; citation_journal_title=Science; citation_volume=272; citation_number=73; citation_pages=67-74; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=High MHC diversity maintained by balancing selection in an otherwise genetically monomorphic mammal.; citation_author=A Aguilar; citation_author=G Roemer; citation_author=S Debenham; citation_author=M Binns; citation_author=D Garcelon; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=101; citation_number=74; citation_pages=3490-3494; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Adaptation to Different Human Populations by HIV-1 Revealed by Codon-Based Analyses.; citation_author=SL Kosakovsky Pond; citation_author=SDW Frost; citation_author=Z Grossman; citation_author=MB Gravenor; citation_author=DD Richman; citation_journal_title=Plos Computational Biology; citation_volume=2; citation_number=75; citation_pages=e62; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Site-to-site variation in synonymous substitution rates.; citation_author=SL Kosakovsky Pond; citation_author=SV Muse; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=76; citation_pages=2375-2385; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evidence for positive selection on a sexual reproduction gene in the diatom genus Thalassiosira (Bacillariophyta).; citation_author=U Sorhannus; citation_author=SL Kosakovsky Pond; citation_journal_title=J Mol Evol; citation_volume=63; citation_number=77; citation_pages=231-239; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Not so different after all: A comparison of methods for detecting amino acid sites under selection.; citation_author=SL Kosakovsky Pond; citation_author=SDW Frost; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=78; citation_pages=1208-1222; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of shared populations of human immunodeficiency virus type 1 infecting microglia and tissue macrophages outside the central nervous system.; citation_author=TH Wang; citation_author=YK Donaldson; citation_author=RP Brettle; citation_author=JE Bell; citation_author=P Simmonds; citation_journal_title=J Virol; citation_volume=75; citation_number=79; citation_pages=11686-11699; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Estimation of levels of gene flow from DNA sequence data.; citation_author=R Hudson; citation_author=M Slatkin; citation_author=W Maddison; citation_journal_title=Genetics; citation_volume=132; citation_number=80; citation_pages=583-589; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The Exchangeability of Amino Acids in Proteins Genetics; citation_author=LY Yampolsky; citation_author=A Stoltzfus; citation_volume=170; citation_number=81; citation_pages=1459-1472; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of gene diversity in subdivided populations.; citation_author=M Nei; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=70; citation_number=82; citation_pages=3321-3323; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=Highly variable loci and their interpretation in evolution and conservation.; citation_author=PW Hedrick; citation_journal_title=Evolution; citation_volume=53; citation_number=83; citation_pages=313-318; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A new statistic for detecting genetic differentiation.; citation_author=RR Hudson; citation_journal_title=Genetics; citation_volume=155; citation_number=84; citation_pages=2011-2014; citation_date=2000; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000734" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="MHC Adaptive Divergence between Closely Related and Sympatric African Cichlids"/>
    <meta name="twitter:description" content="BackgroundThe haplochromine cichlid species assemblages of Lake Malawi and Victoria represent some of the most important study systems in evolutionary biology. Identifying adaptive divergence between closely-related species can provide important insights into the processes that may have contributed to these spectacular radiations. Here, we studied a pair of sympatric Lake Malawi species, Pseudotropheus fainzilberi and P. emmiltos, whose reproductive isolation depends on olfactory communication. We tested the hypothesis that these species have undergone divergent selection at MHC class II genes, which are known to contribute to olfactory-based mate choice in other taxa.Methodology/Principal FindingsDivergent selection on functional alleles was inferred from the higher genetic divergence at putative antigen binding sites (ABS) amino acid sequences than at putatively neutrally evolving sites at intron 1, exon 2 synonymous sequences and exon 2 amino acid residues outside the putative ABS. In addition, sympatric populations of these fish species differed significantly in communities of eukaryotic parasites.Conclusions/SignificanceWe propose that local host-parasite coevolutionary dynamics may have driven adaptive divergence in MHC alleles, influencing odor-mediated mate choice and leading to reproductive isolation. These results provide the first evidence for a novel mechanism of adaptive speciation and the first evidence of adaptive divergence at the MHC in closely related African cichlid fishes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000734.g004"/>

  <meta property="og:title" content="MHC Adaptive Divergence between Closely Related and Sympatric African Cichlids" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000734" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6326'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2574'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4491&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000734">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000734" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000734&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    MHC Adaptive Divergence between Closely Related and Sympatric African Cichlids
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jonatan Blais, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Département de Biologie, Québec-Océan, Université Laval, Quebec, Québec, Canada, 
                  Estación Biológica de Doñana, Consejo Superior de Investigaciones Científicas, Sevilla, Spain, 
                  Department of Biological Sciences, University of Hull, Hull, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ciro Rico, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Estación Biológica de Doñana, Consejo Superior de Investigaciones Científicas, Sevilla, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Cock van Oosterhout, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, University of Hull, Hull, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joanne Cable, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biological Sciences, Cardiff University, Cardiff, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            George F. Turner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, University of Hull, Hull, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Louis Bernatchez
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Louis.Bernatchez@bio.ulaval.ca">Louis.Bernatchez@bio.ulaval.ca</a></p>

                <p>Affiliation:
                  Département de Biologie, Québec-Océan, Université Laval, Quebec, Québec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 15, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000734</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000734" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000734" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000734" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000734" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000734-g001" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000734-g002" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000734-g003" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000734-g004" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000734-t001" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000734-t002" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000734-t003" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000734-t004" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000734">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000734" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The haplochromine cichlid species assemblages of Lake Malawi and Victoria represent some of the most important study systems in evolutionary biology. Identifying adaptive divergence between closely-related species can provide important insights into the processes that may have contributed to these spectacular radiations. Here, we studied a pair of sympatric Lake Malawi species, <em>Pseudotropheus fainzilberi</em> and <em>P. emmiltos</em>, whose reproductive isolation depends on olfactory communication. We tested the hypothesis that these species have undergone divergent selection at MHC class II genes, which are known to contribute to olfactory-based mate choice in other taxa.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Divergent selection on functional alleles was inferred from the higher genetic divergence at putative antigen binding sites (ABS) amino acid sequences than at putatively neutrally evolving sites at intron 1, exon 2 synonymous sequences and exon 2 amino acid residues outside the putative ABS. In addition, sympatric populations of these fish species differed significantly in communities of eukaryotic parasites.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We propose that local host-parasite coevolutionary dynamics may have driven adaptive divergence in MHC alleles, influencing odor-mediated mate choice and leading to reproductive isolation. These results provide the first evidence for a novel mechanism of adaptive speciation and the first evidence of adaptive divergence at the MHC in closely related African cichlid fishes.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Blais J, Rico C, van Oosterhout C, Cable J, Turner GF, et al.  (2007) MHC Adaptive Divergence between Closely Related and Sympatric African Cichlids. PLoS ONE 2(8):
          e734.
            doi:10.1371/journal.pone.0000734</p><p><strong>Academic Editor: </strong>Richard Copley, Wellcome Trust Centre for Human Genetics, United Kingdom</p><p><strong>Received:</strong> April 19, 2007; <strong>Accepted:</strong> July 5, 2007; <strong>Published:</strong> August 15, 2007</p><p><strong>Copyright:</strong> © 2007 Blais et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>JB was supported by a post-graduate scholarship from the Natural Science and Engineering Research Council of Canada and Fond Québécois pour la Nature et les Technologies of Quebec. JC was funded by an Advanced Research Fellowship from the Natural and Environment Research Council (NER/J/S/2002/00706) of the UK. Field work by CvO., JB, and JC was funded by the British Ecological Society. CR was supported by the Consejo Superior de Investigacion Scientifica of Spain. GFT was funded by the Biotechnology and Biological Sciences Research Council of the UK and field work by the Natural and Environment Research Council of the UK. LB was funded by the Natural Science and Engineering Research Council of Canada (Discovery Grant Program).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Explaining why certain ecosystems harbor disproportionately high species richness is a hotly debated issue <a href="#pone.0000734-Willig1">[1]</a>. Antagonistic coevolution between the sexes or between hosts and parasites is recognized as a powerful force capable of driving rapid evolutionary changes, which contributes significantly to biodiversity <a href="#pone.0000734-Buckling1">[2]</a>, <a href="#pone.0000734-Summers1">[3]</a>. Since Haldane's <a href="#pone.0000734-Haldane1">[4]</a> proposal on the importance of parasites resulting in genetic adaptations of the host, researchers have acknowledged their potentially important role in speciation <a href="#pone.0000734-Lederberg1">[5]</a>, <a href="#pone.0000734-Turelli1">[6]</a>. More recently, the concept of cospeciation between parasites and their hosts has received increased attention <a href="#pone.0000734-Switzer1">[7]</a>. The theory of the geographic mosaic of coevolution predicts the diversification of allopatric populations of both host and parasite through three-way interactions between their genotypes and the environment <a href="#pone.0000734-Thompson1">[8]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Consistent with these models, the diversity of parasite species infecting host lineages has been shown to be positively correlated with host species richness in primates <a href="#pone.0000734-Nunn1">[9]</a> and rodents <a href="#pone.0000734-Krasnov1">[10]</a>. Some closely-related species also display striking differences in their response to infectious diseases, as exemplified by the difference between humans and chimpanzees (<em>Pan troglodyte</em>) in their response to infection by hepatitis B and C, HIV and <em>Plasmodium falciparum</em> <a href="#pone.0000734-Anzai1">[11]</a>. Parasitism has also been correlated with increased bacterial host diversity and specialization in microbial communities <a href="#pone.0000734-Buckling1">[2]</a>. Moreover, divergence in MHC genes has been associated with the adaptive radiation of African large barbs from Lake Tana, Ethiopia <a href="#pone.0000734-Kruiswijk1">[12]</a>. Finally, genes showing signs of significant adaptive evolution among rodent and human lineages were found to be related to reproduction, olfaction and immunity <a href="#pone.0000734-Waterston1">[13]</a>, <a href="#pone.0000734-Gibbs1">[14]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In vertebrates, one of the most important factors determining an individual's susceptibility and resistance (defined here as the likelihood of suffering fitness cost from exposure to a pathogen) to specific parasites is its MHC (major histocompatibility complex) genotype <a href="#pone.0000734-Bernatchez1">[15]</a>. Classical MHC molecules are cell surface glycoproteins whose primary role is binding and presentation of antigens to T cell receptors as part of a “self/non-self” recognition mechanism representing the first step of adaptive immune response <a href="#pone.0000734-Bjorkman1">[16]</a>. Antigenic peptides are anchored at specific residues commonly found to be under positive selection and called Antigen Binding Sites (ABS). MHC genes that differ at ABS produce molecules that effectively bind and present different antigens to T cells. This will in turn determine whether or not an effective immune response can be mounted against the infectious agent <a href="#pone.0000734-Bjorkman1">[16]</a>. The MHC thus initiates the adaptive immune defense which is why these genes are thought to be among the most important targets of the host-parasite coevolutionary arms race <a href="#pone.0000734-Bernatchez1">[15]</a>. Besides their central role in immunity, MHC molecules are also known for their influence, through olfactory signals, on mate choice and kin recognition, producing either assortative or disassortative associations in various species including humans <a href="#pone.0000734-Jacob1">[17]</a>, rodents <a href="#pone.0000734-Carroll1">[18]</a>, <a href="#pone.0000734-Sommer1">[19]</a> and fishes <a href="#pone.0000734-Landry1">[20]</a>, <a href="#pone.0000734-Milinski1">[21]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Some of the most spectacular examples of tropical biodiversity are the adaptive radiations of East African cichlid fishes of Lake Victoria, Malawi and Tanganyika <a href="#pone.0000734-Kocher1">[22]</a>, <a href="#pone.0000734-Genner1">[23]</a>. How these hundreds of endemic species evolved from a few founding lineages in a few million years and can coexist within single lakes remains an unresolved issue in evolutionary biology. In Lake Malawi, about 500 endemic species are believed to have diverged from a single lineage within the last 5–7 million years <a href="#pone.0000734-Genner1">[23]</a>. Particularly intriguing is the apparent lack of clear morphological or ecological adaptive differences among many congeneric species, which are distinguished largely on the basis of colour <a href="#pone.0000734-Ribbink1">[24]</a>, <a href="#pone.0000734-Turner1">[25]</a>. Little is known about adaptive genetic differentiation among closely-related species which may lead to evolution of reproductive isolation.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The possibility that host-parasite coevolution has made a significant contribution to East African cichlid diversity has hitherto received scant attention. Nonetheless, several observations suggest that it might be worthy of consideration. Firstly, the most species-rich groups of haplochromines in Lake Victoria and Malawi (i.e. the rock dwelling Mbipi and Mbuna respectively) have highly disjointed and fragmented populations with pronounced genetic structure <a href="#pone.0000734-Rico1">[26]</a>. This would make them prone to diverge under local coevolutionary dynamics according to the geographic mosaic theory <a href="#pone.0000734-Thompson1">[8]</a>. Secondly, population densities of rock dwelling haplochromines are typically very high, in the order of tens of thousands per km<sup>2</sup> or more <a href="#pone.0000734-Ribbink1">[24]</a>, which can favor the spread of infectious diseases <a href="#pone.0000734-Fenton1">[27]</a>. Thirdly, interspecific interactions, like parasitism, are thought to be more intense and complex in tropical ecosystems and might contribute to the higher speciation and specialization observed in these regions <a href="#pone.0000734-Willig1">[1]</a>. Finally, parasitism has previously been shown to negatively influence male mating success through female choice in a Lake Malawi cichlid of the genus <em>Mchenga</em> (formerly <em>Copadichromis</em>) <a href="#pone.0000734-Taylor1">[28]</a>, and to be negatively correlated with the intensity of carotenoid-based male nuptial coloration and territory size in <em>Pundamilia nyererei</em> from Lake Victoria <a href="#pone.0000734-Maan1">[29]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Recently, it has been demonstrated that olfactory communication plays an essential role in assortative mating between the sympatric Lake Malawi cichlids <em>Pseudotropheus emmiltos</em> and <em>P. fainzilberi</em> <a href="#pone.0000734-Plenderleith1">[30]</a>. Divergent natural selection can cause ecological differentiation and speciation simultaneously when its action on ecological traits affects mate choice as a by-product <a href="#pone.0000734-Schluter1">[31]</a>. Considering the documented influence of MHC genotype on odor-mediated mate choice, we hypothesized that odor differences between these species may involve differences in MHC genes accumulated under divergent host-parasite coevolution. In this context, the goal of this study was to document patterns of genetic differentiation between a pair of closely-related sympatric species at MHC class II B, as predicted if host-parasite coevolution has played a role in the divergence of the closely related <em>P. emmiltos</em> and <em>P. fainzilberi.</em> More, specifically, we predict that such divergence should be particularly prominent when comparing nonsynonymous sequence variation at the exon 2-encoded ABS of different species, as diversifying selection is thought to act most stringently at these sites. Ratios of nonsynonymous to synonymous substitutions (dN/dS) higher than unity were used to identify putative ABS, as these are indicative of positive selection. Comparisons with neutral divergence at non-coding intron or synonymous exon 2 sequences of the same gene were used to assess whether divergent selection had to be invoked to explain observed allelic compositions. We also predicted that divergence in MHC class II should be correlated with detectable differences in parasite communities currently infecting the two species in sympatry. The results show clear evidence of divergent selection at MHC class II B genes between the species. Moreover, we found significant differences in parasite communities collected on each sympatric host species.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Positive Selection and Recombination</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>MHC class II B intron 1 and exon 2 sequences were obtained from a total of 1022 clones, representing 74 <em>Pseudotropheus emmiltos</em> individuals from two populations (Luwino Reef N = 31; Mpanga Rocks N = 43) and 106 <em>P. fainzilberi</em> individuals from three populations (Luwino Reef N = 31; Mpanga Rocks N = 36; Chirwa Island N = 37) (<a href="#pone-0000734-g001">Fig. 1</a>; <a href="#pone.0000734.s001">Fig. S1</a>). A mean±standard deviation of 5.74±2.22 clones per individual were sequenced. The final dataset contained 419 individual exon 2 haplotypes (GenBank accession numbers: EF539902-EF540320) and a high level of polymorphism and haplotype diversity comprising 196 distinct exon 2 alleles (172 at the amino acid level) and 164 distinct intron 1 alleles (<a href="#pone.0000734.s002">Fig. S2</a>). The mean number of alleles per individual was 2.31 and 2.29, and ranged from one to five and one to six, for exon 2 and intron 1 respectively (<a href="#pone.0000734.s001">Fig. S1</a>). These data are consistent with Malaga-Trillo et al. <a href="#pone.0000734-MlagaTrillo1">[32]</a> that concluded that the number of MHC class II loci per individual varied between one and six (and therefore that the number of alleles varied between one and 12) in <em>Pseudotropheus zebra</em>. Although we cannot be certain that we have detected all alleles present in each individual, our sampling nevertheless provide an unbiased and adequate estimate of population allele frequencies.</p>
<div class="figure" id="pone-0000734-g001"><div class="img"><a name="pone-0000734-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000734.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000734.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Sampling Locations.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Individuals of a) <em>Pseudotropheus fainzilberi</em> and b) <em>P. emmiltos</em> were collected at Mpanga Rocks, Luwino Reef, and Chirwa Island off the North Western shore of Lake Malawi. The two species are sympatric at Mpanga Rocks and Luwino Reef. Photograhs show males in full nuptial dress and are courtesy of Ad Konings.</p>
<span>doi:10.1371/journal.pone.0000734.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Likelihood ratio tests comparing selective regimes affecting MHC class II B exon 2 in <em>P. emmiltos</em> and <em>P. fainzilberi</em> showed that global selection pressures affecting exon 2 were not significantly different between species, neither in terms of the proportion of sites under selection nor in terms of strength of selection (<a href="#pone.0000734.s006">Table S1</a>). Site-by-site analyses revealed three sites (72, 85 and 87) evolving under different selective pressures (with p&lt;0.05) between the two species along internal branches and three sites (50, 74 and 87) at the tip of the phylogeny. However, these results were not significant after Bonferroni correction. The identification of codons under significant positive selection was therefore conducted on sequences pooled from both species.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Constraining dN≤dS revealed clear evidence of global positive selection in exon 2 since the likelihood of this model was significantly lower than the one allowing dN&gt;dS (LRT = 648.554; d.f. = 3; p&lt;0.001). The bivariate REL analysis identified four dN/dS rate categories across exon 2, with mean rate estimates of: 0.43, 1.11, 2.40 and 5.75. We considered the two lower rate categories as representing purifying selection and (nearly) neutral evolution respectively, and the two higher rate categories as indicating positive selection. A total of 19 out of 84 codons had an empirical Bayesian posterior probability greater than 0.95 of belonging to one of the two positive selection rate categories (<a href="#pone.0000734.s007">Table S2</a>). Of these, 12 corresponded with human ABS identified from crystallography <a href="#pone.0000734-Ono1">[33]</a> (<a href="#pone-0000734-g002">Fig. 2</a>), suggesting that although the functional structure of mammalian and fish MHC class II molecules are approximately similar (<a href="#pone-0000734-g003">Fig. 3</a>; <a href="#pone.0000734.s003">Fig. S3</a>), there appear to be noteworthy differences in the precise position of ABS codons. In mammals and fish, 59 and 58% respectively of MHC class II B ABS are found in the alpha-helices or in coils between adjacent helices forming the edges of the peptide binding groove. A similar proportion (52%) of positively selected sites was found in the alpha-helices of the peptide binding groove of non-classical mhc class Ib genes thought to be involved in pheromone detection in rodents <a href="#pone.0000734-Emes1">[34]</a>.</p>
<div class="figure" id="pone-0000734-g002"><div class="img"><a name="pone-0000734-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000734.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000734.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Three-Dimensional Model of the β<sub>1</sub> Domain of Cichlid MHC Class II.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>The tertiary structure prediction was based on Psemmil_m58_c exon 2 sequence (Accession number: EF540049) of 41% homology with mouse MHC sequence gi|13399459 in the Protein Data Bank (<a href="http://www.pdb.org/pdb/home/home.do">http://www.pdb.org/pdb/home/home.do</a>) using the 3D-jigsaw server v.2.0 (<a href="http://www.bmm.icnet.uk/~3djigsaw/">http://www.bmm.icnet.uk/~3djigsaw/</a>). The graphical representation was created using the program Pymol v.0.99. Amino acid residues under significant positive selection in <em>Pseudotropheus fainzilberi</em> and <em>P. emmiltos</em> and corresponding to peptide binding sites in humans are highlighted in orange. Residues shown in red were under significant positive selection in <em>P. fainzilberi</em> and <em>P. emmiltos</em> but do not correspond with peptide binding sites in humans. Residues in yellow are human peptide binding sites that were not found to evolve under positive selection in cichlids. The 84 amplified exon 2 codons are numbered 1-84 on the graph. Those correspond to codons 6-89 of the mature protein.</p>
<span>doi:10.1371/journal.pone.0000734.g002</span></div><div class="figure" id="pone-0000734-g003"><div class="img"><a name="pone-0000734-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000734.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000734.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Comparison of Secondary Structure of Mouse and Cichlid MHC Class II Exon 2.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Structure predictions were obtained from the PSIPRED server (<a href="http://bioinf.cs.ucl.ac.uk/psipred/psiform.html">http://bioinf.cs.ucl.ac.uk/psipred/psifo​rm.html</a>) for a) mouse sequence gi:13399459 and b) cichlid sequence Psemmil_m58_c (Accession number: EF540049). Mammalian and fish secondary structures are similar except for an additional short alpha-helix at position 46–49 predicted for the mouse sequence, and an additional short strand at position 74 predicted for the cichlid sequence.</p>
<span>doi:10.1371/journal.pone.0000734.g003</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>We compared intron 1 and exon 2 mean genetic distances (<em>d</em> and <em>dS</em> respectively) within 14 functional clades defined using these positively selected sites (from an amino acid p-distance neighbor-joining phylogeny, <a href="#pone.0000734.s004">Fig. S4</a>). Significantly different mean genetic distances between intron 1 and exon 2 were detected in at least eight clades, which is evidence for intron-exon recombination <a href="#pone.0000734-Hughes1">[35]</a> (<a href="#pone.0000734.s008">Table S3</a>). In three of these, mean <em>d</em> at intron 1 was significantly lower than mean <em>dS</em> at exon 2, suggesting the action of within-locus recombination and balancing selection on functional exon 2 alleles <a href="#pone.0000734-Hughes1">[35]</a>. The remaining five cases had mean <em>d</em> at intron 1 higher than mean <em>d</em>S at exon 2, suggesting inter-locus recombination <a href="#pone.0000734-Reusch1">[36]</a>. However, among those eight clades, only clade 14 had bootstrap support higher than 60% and thus results from the other seven clades must be viewed with great caution. Nevertheless, a single breakpoint recombination analysis <a href="#pone.0000734-KosakovskyPond1">[37]</a> revealed that the most likely breakpoint was at nucleotide position 231. Including this recombination breakpoint greatly improved the phylogenetic model of intron 1 and exon 2 evolution (ΔAICc = 13001.0; ΔBIC = 3667.85). This position is located only 15 nucleotides upstream from the intron 1-exon 2 junction of the alignment, corroborating the scenario of recombination between intron 1 and exon 2.</p>


<h4>Functional Divergence</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The association index (AI) value calculated from the intron 1 phylogeny (0.69) was higher than that computed from the exon 2 phylogeny (0.61), suggesting that phylogenetic divergence was higher at exon 2 than at non-coding intron 1 (<a href="#pone-0000734-t001">Table 1</a>). Similarly, genetic differentiation measured by Fst value based on overall mean pairwise genetic distances was higher at exon 2 putative ABS amino acid sites (Fst = 0.097; p&lt;0.01) than at neutrally evolving intron 1 (Fst = 0.021; p&lt;0.01) or synonymous exon 2 alleles (Fst = 0.021; p&lt;0.01). Both findings are contrary to what is predicted from a scenario of balancing selection with no divergent selection at exon 2. In addition, within exon 2, Fst was higher for positively selected putative ABS (Fst = 0.097; p&lt;0.01) than for sites outside this region (Fst = 0.016; p = 0.015) (<a href="#pone-0000734-t001">Table 1</a>). Moreover, higher divergence at putative ABS compared to neutral alleles was observed for both heterospecific sympatric comparisons, but for none of the conspecific allopatric comparisons where functional Fst estimates at ABS were all negative and therefore technically equal to zero (<a href="#pone-0000734-g004">Fig. 4</a>).</p>
<div class="figure" id="pone-0000734-g004"><div class="img"><a name="pone-0000734-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000734.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000734.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Comparison of Genetic Divergence Between Heterospecific Sympatric Populations and Conspecific Allopatric Populations.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Fst values (HudSon et al. 1992) from pairwise comparisons of conspecific allopatric populations (Mpanga Rocks (Mpg), Luwino Reef (Lwo), and Chirwa Island (Chw)) and between heterospecific sympatric populations of <em>P. fainzilberi</em> (fainz.), and <em>P. emmiltos</em> (emm.). Neutral Fst estimates were obtained from intron 1 p-distances, modified Nei and Gojobori synonymous distances at exon 2, and from amino acid EX distance at exon 2 sites outside the putative ABS region (dN/dS≤1) and functional Fst were obtained from EX distance at putative exon 2 ABS (dN/dS&gt;1). Negative Fst estimates were forced to zero. Evidence of significant genetic divergence between pairs of allopatric populations was found only for neutral Fst estimates between <em>P. emmiltos</em> populations from Luwino Reef and Mpanga Rocks. Evidence of significant genetic divergence between pairs of <em>P. fainzilberi</em> and <em>P. emmiltos</em> sympatric populations was found only at putative ABS at Mpanga Rocks. <sup>*</sup>Significantly different from zero (p&lt;0.05; 200 permutations)</p>
<span>doi:10.1371/journal.pone.0000734.g004</span></div><div class="figure" id="pone-0000734-t001"><div class="img"><a name="pone-0000734-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000734.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000734.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Neutral and functional genetic differentiation estimates between <em>Pseudotropheus fainzilberi</em> and <em>P. emmiltos</em> samples.</span></strong></p><span>doi:10.1371/journal.pone.0000734.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Finally, there were 31 shared and 23 private alleles at intron 1 between <em>P. fainzilberi</em> and <em>P. emmiltos</em>, compared to 22 shared and 41 private alleles at exon 2 amino acid sequences. These raw proportions were significantly different (two-tailed Fisher's Exact Test: p = 0.017). There was no significant difference in allele frequency distributions between exon 2 and intron 1 (<a href="#pone.0000734.s005">Fig. S5</a>), (mean <em>f</em> intron 1: 0.015, mean <em>f</em> exon 2: 0.013, Mann-Whitney U = 1558.000, p = 0.437; intron 1 σ<sup>2</sup>: 0.246, exon 2 σ<sup>2</sup>: 0.227; Levene's test F = 0.435, p = 0.511). There was, however, a highly significant difference in shared and private allele composition as measured by <em>Dc</em> (<a href="#pone-0000734-t002">Table 2</a>). This difference was significant between intron 1 and exon 2 alleles when considering the whole exon 2 sequence, and between intron 1 and exon 2 alleles at the 19 positively selected putative ABS, but not between intron 1 and the 65 exon 2 residues outside this region (<a href="#pone-0000734-t002">Table 2</a>).</p>
<div class="figure" id="pone-0000734-t002"><div class="img"><a name="pone-0000734-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000734.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000734.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Proportions of private and shared alleles between <em>Pseudotropheus fainzilberi</em> and <em>P. emmiltos</em> at MHC class II B intron 1 alleles and exon 2 amino acid alleles.</span></strong></p><span>doi:10.1371/journal.pone.0000734.t002</span></div>

<h4>Comparison of Parasite Communities</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Sixteen categories of parasites were identified in both species and their abundance recorded on each fish (<a href="#pone-0000734-t003">Table 3</a>). A Principal Component Analysis on the dataset <a href="#pone.0000734-Legendre1">[38]</a> returned four factors explaining 84% of the original variance (<a href="#pone-0000734-t003">Table 3</a>). PC1 represented mainly a negative correlation between infection by acanthocephalans and stomach cysts versus infection by muscle-encysted digenean metacercaria, PC2 between infection by muscle-encysted digenean metacercaria and by stomach cysts, and PC3 between infection by ciliate <em>Trichodina</em> sp. and gill parasitic copepod sp. 3 versus infection by acanthocephalans. Finally, PC4 largely represented a contrast between infection by gill parasitic copepod sp. 3 and ciliate <em>Trichodina</em> sp.</p>
<div class="figure" id="pone-0000734-t003"><div class="img"><a name="pone-0000734-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000734.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000734.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Parasite infections of <em>Pseudotropheus emmiltos</em> and <em>P. fainzilberi</em>: eigenvectors of the first four factors explaining 84% of the original variance returned from a principal component analysis on incidence and abundance of 16 types of parasites transformed according to equation 12 in Legendre and Gallagher <a href="#pone.0000734-Legendre1">[38]</a>.</span></strong></p><span>doi:10.1371/journal.pone.0000734.t003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>A multivariate linear model of the relationship between these first four factors representing infecting parasite communities and host species, host sex, host standard length and collection site returned R<sup>2</sup> values of 0.754, 0.473, 0.178 and 0.010 for PC1 to PC4 respectively. Overall, 26% of the variance in these four factors was explained by variation among collection sites, while host standard length, host sex and host species accounted for 3.6, 3.3, and 1.8% of the variance, respectively (<a href="#pone-0000734-t004">Table 4</a>). There were significant differences between <em>P. fainzilberi</em> and <em>P. emmiltos</em> in PC1 and PC3 (<a href="#pone-0000734-t004">Table 4</a>). Least square means for PC1 and PC3 were 0.263 and −0.372 for <em>P. emmiltos</em> and 0.027 and 0.067 for <em>P. fainzilberi</em> respectively. Hence, <em>P. emmiltos</em> tended to have more stomach cysts and muscle encysted digenean metacercaria relative to acanthocephalan infections (PC1) than <em>P. fainzilberi</em>. Conversely, <em>P. emmiltos</em> had less trichodinids and gill infecting parasitic copepod sp 3 relative to acanthocephalan infection (PC3) than <em>P. fainzilberi</em>.</p>
<div class="figure" id="pone-0000734-t004"><div class="img"><a name="pone-0000734-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000734" data-uri="info:doi/10.1371/journal.pone.0000734.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000734.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000734.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000734.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000734.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Sum of squares (SS) and degrees of freedom (df) of a multivariate general linear model testing the effect of host species, collection site, host sex, and host standard length on four principal component analysis factors (PC1-PC4) representing 84% of the original variance of the incidence and prevalence of 16 different types of parasites.</span></strong></p><span>doi:10.1371/journal.pone.0000734.t004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Antagonistic host-parasite interactions involving coevolutionary responses have the potential to produce rapid evolutionary changes <a href="#pone.0000734-Summers1">[3]</a>. Genes directly implicated in these interactions are often polymorphic <a href="#pone.0000734-Frank1">[39]</a> and may diverge quickly if there are differences in selective pressures among populations <a href="#pone.0000734-Thompson1">[8]</a>. Such defense genes are likely candidates to show the first signs of genetic divergence in recently-speciated taxa. Whereas the importance of divergence in traits related to defense against parasites and other enemies (e.g. phytophagous insects) has long been recognized in studies of plant radiations, it has remained largely ignored in the study of adaptive radiations of animals <a href="#pone.0000734-Schluter2">[40]</a>. Here, we tested the hypothesis that two recently diverged species experienced divergent selection at important immunological genes of the MHC class II. Furthermore, we also tested for possible differentiation in parasite communities infecting the species in sympatry, which we hypothesized could be a driver of divergent evolution at the MHC.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Evidence for strong balancing and positive selection on MHC class II loci was manifested by the high polymorphism and haplotype diversity and a clear signal of positive selection across the 84 codons of exon 2. Differentiation levels expressed in the association index (AI) and Fst, as well as proportion of shared and private alleles for exon 2 of MHC class II B were contrasted with neutral divergence at intron 1 and synonymous exon 2 polymorphisms. These comparisons revealed that the observed patterns of functional allelic composition of the two species cannot be explained by divergence due to drift alone.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Four arguments support the contention that diversifying selection augmented genetic differentiation of parts of the MHC genes between the two cichlid species. Firstly, without divergent selection, genes under balancing selection are expected to show less differentiation between populations than neutral loci <a href="#pone.0000734-Schierup1">[41]</a>, <a href="#pone.0000734-vanOosterhout1">[42]</a>. However, despite recombination between intron 1 and exon 2, divergence estimated by AI values was higher for the exon 2 phylogeny than for the intron 1 phylogeny. Secondly, neutral Fst from intron 1, exon 2 dS, or sites with dN/dS≤1 in exon 2 was about 2%, while this value was almost five times higher in putative ABS. Thirdly, this pattern of higher divergence at putative ABS was observed between both heterospecific sympatric population pairs but for none of the allopatric conspecific population pairs, suggesting that this is a species-specific adaptive divergence that persists despite potential gene flow between species in sympatry. Finally, the significantly higher proportion of private alleles at exon 2 amino acid sequences compared with non-coding intron 1, provides strong additional evidence that despite close physical linkage between intron and exon sequences, divergent selection contributed to the shaping of the functional allelic composition of the species.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Adaptive divergence at immune system genes is likely to be correlated with differences in resistance and susceptibility to parasites. Our results are consistent with this prediction as there were significant, albeit relatively small, interspecific differences for PC1 and PC3 while controlling for host sex, standard length and collection site. Apart from acanthocephalans and unidentified stomach cysts, the other parasites represented by PC1 and PC3 (namely digenean metacercaria, trichodinids and gill infecting parasitic copepod) are known to actively infect the fish from the water column. Acanthocephalans are normally acquired via the food chain. There is therefore two distinct modes of transmission involved. Thus we cannot rule out the possibility that species differences in susceptibility stems from differences in microhabitat or diet rather than from genetic differences <a href="#pone.0000734-Skarstein1">[43]</a>. However, there is substantial overlap in microhabitat and both species feed by combing attached filamentous algae with their specialized bi- and tricuspid teeth <a href="#pone.0000734-Ribbink1">[24]</a>. In addition, at sampling locations, males of both species have overlapping territories which they use for foraging.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>A substantial amount of variance (26%) in infecting parasite communities was explained by variation among collection sites. Local variation in abundance of specific parasites can be due to transient epidemics or alternatively, to more stable variation in population densities caused by habitat or environmental differences. The latter case provides the raw material for coevolutionary divergence in genes involved in parasite resistance such as MHC between allopatric populations exposed to different parasite environments <a href="#pone.0000734-Thompson1">[8]</a>. Upon secondary contact, such differences accumulated in allopatry could be manifested by distinct infection patterns of the species such as those detected here. Alternatively, initial divergence among allopatric populations in trophic niche could have lead to different levels of exposures to parasites transmitted via the food chain such as acanthocephalans, which in turn may select for different MHC alleles. These processes of differentiation could be particularly important in species with pronounced genetic structure such as rock-dwelling haplochromines. Divergence in immune system does not imply detectable divergence in morphology and may therefore be overlooked in many animal systems. Nevertheless, divergence in immunity could have important consequences on foraging or reproductive behaviors that may in turn promote speciation or species coexistence. The phenomenon could be more widespread in animal radiations than currently appreciated. For example, divergence in MHC class II has been observed in the radiation of large barbs (cyprinid fishes) in Ethiopia's Lake Tana <a href="#pone.0000734-Kruiswijk1">[12]</a>. In the cyprinodont <em>Fundulus heteroclitus</em>, divergent selection in MHC class II among populations was inferred from population specific amino acid replacement at putative ABS <a href="#pone.0000734-Cohen1">[44]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Identifying adaptive genetic differentiation between closely related species is an important step toward understanding speciation processes. Unfortunately, this critical information has been lacking in the case of African haplochromine cichlids. The impressive species diversity and endemism of cichlid species assemblages from Lake Malawi, Victoria and Tanganyika have established the systems as some of the most important models in evolutionary biology <a href="#pone.0000734-Kornfield1">[45]</a>. Part of this fascination comes from the surprising ecological and morphological similarity of congeners, which led some to suggest that natural selection had little importance in recent speciation events <a href="#pone.0000734-Deutsch1">[46]</a>, <a href="#pone.0000734-Kocher1">[22]</a>. The results presented here provide one of the first evidence of adaptive genetic evolution in African cichlids from the African Great Lake radiations <a href="#pone.0000734-Albertson1">[see also 47]</a>, <a href="#pone.0000734-Terai1">[48]</a>. We hypothesize that this adaptive differentiation may have fuelled the speciation process as a by-product by augmenting MHC-mediated mate choice signals. Natural selection from host-parasite coevolution may thus have played a more important role in recent speciation of endemic African cichlid lineages than suggested by morphological similarities.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Sexual selection has long been regarded as an important factor in haplochromine speciation <a href="#pone.0000734-Dominey1">[49]</a>–<a href="#pone.0000734-Allender1">[51]</a> while the importance of parasitism in sexual selection and mate choice has been discussed extensively <a href="#pone.0000734-Hamilton1">[52]</a>–<a href="#pone.0000734-Blais1">[59]</a>. Several cases of assortative MHC-based mate choice have been documented. Thus, MHC genotypes represent an ecological trait whose divergence by natural selection may directly produce reproductive isolation as a by-product. For instance, threespine stickleback females prefer to mate with males having an intermediate rather than maximal number of MHC class II alleles different from those the females possess <a href="#pone.0000734-Reusch2">[60]</a>. Female preference for intermediate MHC similarity has also been found in humans <a href="#pone.0000734-Jacob1">[17]</a>. Malagasy giant jumping rat (<em>Hypogeomys antimena</em>) females preferred males with MHC genotypes more similar to themselves <a href="#pone.0000734-Sommer1">[19]</a>. Possible causes of mate preferences aimed at limiting offsprings' MHC diversity include detrimental effect of excessive MHC diversity caused by: negative thymic selection of T cell lines <a href="#pone.0000734-Nowak1">[61]</a>, <a href="#pone.0000734-Borghans1">[62]</a>, cross-reactive autoimmune diseases <a href="#pone.0000734-Borghans1">[62]</a> and reduced concentration of specific MHC-peptide ligands on the surface of antigen presenting cells for effective T cell activation <a href="#pone.0000734-vandenBerg1">[63]</a>. We therefore speculate that multilocus MHC genotype compatibility may exclude matings between <em>P. fainzilberi</em> and <em>P. emmiltos</em> through olfactory signals part of a mate choice rule including some limits on MHC dissimilarity of mates. This would represent a case of “one-allele” speciation mechanism <a href="#pone.0000734-Felsenstein1">[64]</a> as the same mate choice alleles in both species would reduce gene flow. Behavioral studies on the precise influence of MHC genotype on mate choice in these two species may elucidate the role that this divergence has played in the evolution of reproductive isolation.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Study Sites and Field Sampling</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p><em>Pseudotropheus emmiltos</em> and <em>P. fainzilberi</em> are among the 17–44 currently recognized species of the <em>P. zebra</em> species complex (subgenus <em>Maylandia</em>), members of which are found all around in Lake Malawi <a href="#pone.0000734-Genner1">[23]</a>. The two species are sympatric at Mpanga Rock and Luwino Reef off the north western coast (<a href="#pone-0000734-g001">Fig. 1</a>), the only two sites where <em>P. emmiltos</em> is known to occur <a href="#pone.0000734-Konings1">[65]</a>. <em>P. fainzilberi</em> is found at several other localities both north and south of the zone of sympatry. Thus, the range of <em>P. emmiltos</em> is nested within the range of <em>P. fainzilberi</em>. Mitochondrial DNA analyses using the model of isolation with migration implemented in the program IM <a href="#pone.0000734-Nielsen1">[66]</a> revealed that the two species diverged around 30–45 000 years ago and that the most probable speciation scenario was one of allopatric divergence followed by secondary contact (Turner et al. in prep.). The most noticeable morphological difference between the taxa is the color of the dorsal fin, which is orange in <em>P. emmiltos</em> and blue with a black longitudinal stripe in <em>P. fainzilberi</em> (<a href="#pone-0000734-g001">Fig. 1</a>). The two species are behaviorally reproductively isolated and olfactory signals are involved in mate discrimination <a href="#pone.0000734-Plenderleith1">[30]</a>. Fifty individuals of each species where caught at Mpanga Rocks and Luwino Reef by SCUBA divers using monofilament gill nets. Fifty individuals of <em>P. fainzilberi</em> were also captured at Chirwa Island, two kilometers south of the zone of sympatry (<a href="#pone-0000734-g001">Fig. 1</a>).</p>


<h4>Cloning and DNA Sequencing</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>MHC class II molecules are expressed on the surface of antigen presenting cells and bind antigenic peptides in a cleft formed by the α<sub>1</sub> and β<sub>1</sub> domains in which peptides are anchored in pockets of specific residues (ABS). Bound peptides are then presented to CD4+ T cells. Sequence polymorphisms at these sites are believed to result from positive and balancing selection <a href="#pone.0000734-Hughes2">[67]</a>, <a href="#pone.0000734-Yang1">[68]</a>. Intron 1 along with 252 bp of exon 2 (codons 6-89 of the mature protein), coding for the β<sub>1</sub> domain of MHC class II genes <a href="#pone.0000734-Ono2">[69]</a> were PCR amplified using the degenerate primer pair TU377-TU383 <a href="#pone.0000734-MlagaTrillo1">[32]</a>, which amplifies up to eight expressed putative loci in African cichlids according to comparisons of individual haplotypes obtained from genomic and cDNA clones [32; Blais et al. unpublished results]. PCR conditions consisted of 30 cycles of 30 sec at 95°, 60 sec at 56°, 60 sec at 72° in a 20 µl reaction volume containing 0.25mM of each dNTPs, 2 µl of 10× reaction buffer, 2 mM MgCl<sub>2</sub>, 0.5U of Taq DNA polymerase, and 25 ρmol of each primer.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>PCR products were purified by ExoSAP-IT (GE healthcare, Chalfont St. Giles, U.K.) following manufacturer's instructions, ligated into pGEM®-T vector (Promega, Madison, WI, USA) and cloned using JM109 Competent Cells (Promega, Madison, WI, USA). Between eight and 30 recombinant clones per individual were amplified with primers OMNI (5′-ACA GGA AAC AGC TAT GAC CAT GAT-3′) and UNI (5′-CGA CGT TGT AAA ACG AGG CCA GT-3′) under the same PCR conditions described above, and sized on 1.5% agarose gel. Between four and 23 clones containing inserts of expected size (~320–500 bp depending on intron 1 length) per individual were purified with ExoSAP-IT (GE healthcare, Chalfont St. Giles, U.K.), sequenced using Bigdye 2.0 dye terminator kit (Applied Biosystems, Foster City, CA, USA), cleaned with Sephadex™ and analyzed on an ABI 3100 DNA Analyzer (Applied Biosystems, Foster City, CA, USA). Sequence alignments produced by ClustalX v.1.83 <a href="#pone.0000734-Thompson2">[70]</a> were examined to detect false allelic variants due to PCR errors by comparing sequences within and among individuals. First, sequence differences of less than 3 bp when compared to multiple identical sequence replicates from different clones within individuals were considered to be PCR or cloning artifacts and removed from the alignment. Second, only sequences found in at least two independent PCR (within or among individuals) were used for subsequent analyses <a href="#pone.0000734-vanOosterhout1">[42]</a>.</p>


<h4>Identification of ABS</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>There is at present no available information from X-ray diffraction or NMR analyses about the three-dimensional structure of non-mammalian MHC. It is therefore impossible to identify with certainty which sites are involved in peptide binding. In order to identify putative antigen binding sites (ABS) within the peptide binding cleft involved in the coevolutionary arms race, we determined which codons were evolving under positive selection, i.e. having a nonsynonymous/synonymous substitution ratio (dN/dS) significantly higher than unity. We also compared the location of these with mammalian ABS identified from crystallography. The following and all subsequent sequence analyses were conducted using the HYPHY package and programming language running on a 32 processors SGI Altix3700 computer at the Bioinformatics and Computational Biology Centre of University Laval, under a maximum likelihood framework, unless otherwise stated .</p>


<h4>Recombination</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>According to Kosakovsky Pond et al. <a href="#pone.0000734-KosakovskyPond1">[37]</a> and as implemented by SingleBreakpointRecomb.bf in HYPHY, we determined the most likely recombination breakpoint by computing the likelihood of neighbor-joining phylogenies <a href="#pone.0000734-Saitou1">[71]</a> for the entire intron-exon alignment and for models fitting branch length independently for partition blocks on each side of a single break point located at each nucleotide position sequentially. Akaike information criterion with second-order correction (AICc) and Bayesian information criterion (BIC) were used to determine which was the most likely recombination breakpoint in the sequence and if including recombination improved the fit of the phylogenetic model (SingleBreakpointRecomb.bf in HYPHY). Using a method assuming a single breakpoint was shown <a href="#pone.0000734-KosakovskyPond1">[37]</a> to perform as well or better than 14 different recombination detection methods evaluated by Posada and Crandall <a href="#pone.0000734-Posada1">[72]</a> and is particularly well suited for relatively short alignments like the one studied here where one is interested in evaluating the likelihood of recombination at a specific location along the sequence.</p>


<h4>Selection</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>In the next sections, we explain the methodology used to examine evidence for different selection pressures: (1) positive selection, (2) balancing selection and (3) divergent selection. Here, we define positive selection as selection for improved recognition of parasites. Positive selection pressure favors nonsynonymous substitutions particularly in the ABS.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Balancing selection is thought to be the dominant force maintaining polymorphism in classical MHC genes <a href="#pone.0000734-Hughes2">[67]</a>, <a href="#pone.0000734-Parham1">[73]</a>, <a href="#pone.0000734-Aguilar1">[74]</a>. Overdominance and negative frequency dependent selection are types of balancing selection that operate within populations. In the absence of divergent selection, balancing selection is expected to prevent or delay population divergence <a href="#pone.0000734-Schierup1">[41]</a>, and some empirical studies suggested that genetic differentiation between populations is lower for functional MHC alleles than for neutrally evolving loci <a href="#pone.0000734-vanOosterhout1">[42]</a>. Balancing selection thus maintains a high level of polymorphism within populations but reduces genetic divergence between populations.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>Finally, divergent selection occurs when different selection pressures act between populations, resulting in higher genetic differentiation, particularly at dN of ABS of allopatric populations. If populations differ in local (positive) selective pressures, divergent selection may maintain different sets of alleles and result in increased differentiation <a href="#pone.0000734-Cohen1">[44]</a>. Divergent selection could thus counteract the affect of balancing selection and increase the extent of population differentiation at a level above that expected for neutral genes [reviewed in 15].</p>


<h4>Positive Selection</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Before determining which individual codons were under positive selection, we first verified whether the selective regimes affecting each species were the same in terms of proportion of sites under selection and the strength of selection. We therefore constrained the substitution rate parameters by setting it equal for both species, and compared the fit of this model with models that could use different parameter setting for each species. The fit of the models were compared using likelihood ratio tests (LRT) with CompareDistribution.bf in HYPHY. A codon-based site-by-site fixed effect likelihood (FEL) analysis comparing selective pressures between the two species along internal branches and leaves of the phylogenetic tree was also performed (ComparesSelectivePressure.bf in HYPHY) <a href="#pone.0000734-KosakovskyPond2">[75]</a>. This analysis tests the null hypothesis that the ratio of nonsynonymous to synonymous substitutions is significantly different between the two species for each codon site using likelihood ratio tests.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>Positive selection acting on exon 2 was tested by comparing a random effect likelihood (REL) dual model <a href="#pone.0000734-KosakovskyPond3">[76]</a> where both dN and dS are free to vary and are drawn from a general discrete distribution with N = 4 rate classes but where dN is constrained to be≤dS, with a model where dN is free to be&gt;dS <a href="#pone.0000734-Sorhannus1">[77]</a>. A likelihood ratio test (LRT) based on the Chi-square distribution with N–1 degrees of freedom was used to determine whether allowing dN&gt;dS significantly improved the fit of the model. Individual codons evolving under significant positive selection were identified using a REL method where both dN and dS are free to vary and were drawn from a bivariate general discrete distribution with four dN and four dS rate categories (dNdSBivariateRates.bf in HYPHY). The model fits the parameters of this bivariate general discrete distribution of rates across the sequences given the phylogeny. Positive selection at specific codons is inferred for sites having an empirical Bayesian posterior probability≥0.95 to be in one of the dN/dS&gt;1 categories. The empirical Bayesian approach is based on the maximum likelihood estimates of the rate distribution parameters <a href="#pone.0000734-KosakovskyPond4">[78]</a>. The results were then compared with sites of the cichlid MHC sequence corresponding to human ABS deduced from three-dimensional information.</p>


<h4>Balancing Selection</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Following Hughes <a href="#pone.0000734-Hughes1">[35]</a>, evidence of both balancing selection and recombination between exon 2 and intron 1 in each species was evaluated by comparing mean genetic distance (<em>d</em>) among intron 1 alleles and mean synonymous genetic distance (<em>dS</em>) at functional exon 2 alleles within functional clades defined using putative ABS identified by the method outlined above. Recombination between introns and exons creates incongruence between intron and exon phylogenies. This translates into significantly lower mean genetic distance among intron alleles than exon 2 alleles within exon 2 functional clades, assuming recombination occurs within locus and that balancing selection acts on exon 2 functional allelic variants <a href="#pone.0000734-Hughes1">[35]</a>. Indeed, balancing selection on the ABS can maintain synonymous sequence variation in exon 2 due to genetic hitchhiking, while recombination causes intron 1 to loose polymorphism by genetic drift. On the other hand, higher genetic distance at introns than at exon 2 within functional clades is expected when recombination occurs predominantly between loci <a href="#pone.0000734-Reusch1">[36]</a>.</p>


<h4>Divergent Selection</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>We tested for the presence of divergent selection using three different approaches. First, the level of phylogenetic compartmentalization at non coding intron 1 was compared with that observed at exon 2 using the association index (AI) developed by Wang et al. <a href="#pone.0000734-Wang1">[79]</a> and implemented in HYPHY. An association value, <em>d</em>, for the tree is calculated by summation of <em>d</em>-values from each node, according to <em>d</em> = (1–<em>f</em>)/2<em><sup>n</sup></em><sup>−1</sup>, where <em>n</em> is the number of sequences below the node and <em>f</em> is the frequency of sequences from the most common species in the clade. Values of <em>d</em> expected from the null hypothesis (i.e. no phylogenetic grouping) were calculated by random reassignment of the sequences to the different species. The association index (AI) represents the mean ratio of 100 bootstrap replicates of the association value calculated from the test sequences to those of species-reassigned controls from 200 permutations <a href="#pone.0000734-Wang1">[79]</a>. Smaller AI values indicate higher divergence of samples (i.e. more homogenous clades). Smaller AI values are predicted for intron 1 phylogeny than for exon 2 phylogeny under a pure scenario of balancing selection acting on ABS in exon 2 involving intron-exon recombination <a href="#pone.0000734-Hughes1">[35]</a>. By contrast, divergent selection between species at exon 2 is predicted to result in relatively high AI value for intron 1.</p>
<a id="article1.body1.sec4.sec8.p2" name="article1.body1.sec4.sec8.p2"></a><p>Secondly, estimates of Fst based on pairwise genetic distances <a href="#pone.0000734-Hudson1">[80]</a> were contrasted between neutral (non-coding or synonymous alleles) and functional alleles (i.e. alleles differing in putative ABS amino acid residues). Divergent selection can be inferred in cases where Fst estimates from functional alleles exceed those obtained from neutrally evolving alleles. We calculated neutral Fst estimates using nucleotide p-distance for intron 1, as well as modified Nei and Gojobori <em>dS</em> at exon 2. We then computed functional divergence at exon 2 using the amino acid experimental exchangeability distance (EX) <a href="#pone.0000734-Yampolsky1">[81]</a> estimated by maximum likelihood in HYPHY, for the 65 sites with dN/dS≤1 and for the 19 sites with dN/dS&gt;1 separately. The EX distance is based on experimental assays of the severity-of-effect of amino acid substitutions on protein function and was specifically designed to reflect functional differences in amino acid sequences while being free of mutational bias. Adaptive divergence between species was first tested using Fst values calculated from sequences of all individuals. In order to test whether observed divergence patterns were species specific and thus could be associated with putative speciation processes, we also compared neutral and functional Fst estimates obtained from the two heterospecific sympatric pairs of populations with those obtained from all pairwise comparisons of allopatric conspecific populations.</p>
<a id="article1.body1.sec4.sec8.p3" name="article1.body1.sec4.sec8.p3"></a><p>Third, we compared haplotype compositions of exon 2 amino acid alleles with non-coding intron 1 alleles. In genes such as classical MHC where haplotype diversity is typically very high, most haplotypes may appear only once in the sample and methods based on haplotype frequencies, such as Gst <a href="#pone.0000734-Nei1">[82]</a>, have little or no power <a href="#pone.0000734-Hedrick1">[83]</a>, <a href="#pone.0000734-Hudson2">[84]</a>. To circumvent this problem, we contrasted the relative proportion of shared and private alleles at intron 1 and exon 2 amino acid sequences. Divergent selection for alternative sets of functional exon 2 alleles predicts that the proportion of private alleles should be higher for exon 2 amino acid sequences than for non coding intron 1 sequences. Alleles that were observed only once were omitted as these cannot be shared by both species. We then tallied the number of ‘private alleles’ (i.e. alleles that were observed at least twice in only one of the two species) and used Fisher's exact test on observed proportions of shared and private alleles at intron 1 and exon 2. We also quantified the relative proportion of private and shared alleles by assigning allelic distinctness scores <em>Dc</em> to each allele defined as allele frequency <em>f</em> in the case of private alleles, and as-(1-<em>f<sub>s</sub></em>) in the case of shared alleles, where <em>f<sub>s</sub></em> is the frequency of the allele in the species where it is most common. Thus, according to this index, <em>Dc</em> is bounded by −1 and 1 (private alleles have positive scores and shared alleles have negative scores), and more weight is given to common private alleles and to rare shared alleles, reflecting the likelihood of sampling each category. We tested for a significant difference between intron 1 and exon 2 in the composition of private and shared alleles as measured by <em>Dc</em> using the non-parametric Mann-Whitney <em>U</em> test.</p>


<h4>Parasite Screening and Analysis</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Heterospecific populations in sympatry were predicted to show different parasite communities due to independent host-parasite coevolution (i.e. a prediction consistent with divergent selection). We therefore dissected a total of 81 fish captured at Mpanga Rocks (<em>P. fainzilberi</em> N = 16; <em>P. emmiltos</em> N = 18), Luwino reef (<em>P. fainzilberi</em> N = 18; <em>P. emmiltos</em> N = 17) and Chirwa Island (<em>P. fainzilberi</em> N = 12) by SCUBA diving using gill nets in September 2005. The fish were examined for all eukaryote parasites within 24 h of collection. Parasites incidence and abundance were recorded by killing the fish in an overdose of 0.2% MS222 and examining the dissected organs under a stereo-microscope with cold light illumination. The resulting parasite community dataset was subjected to a linear transformation according to <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000734.e001&amp;representation=PNG" class="inline-graphic"></span>, where y<sub>ij</sub> denotes the observation for parasite type j, on host i and y<sub>i+ </sub>is the sum of observations for host i [eq. 12 in 38]. As a result of this transformation, a principal component analysis (PCA) performed on the transformed data is based on Hellinger distances, suitable for community data comparisons, rather than Euclidian distances <a href="#pone.0000734-Legendre1">[38]</a>. Thus, to reduce the dimensionality of the data, the minimum number of factors required to explain at least 80% of the original variance were retained from a PCA on the transformed dataset. These factors were then used in a multivariate general linear model to test for differences in relative abundance and composition of parasite communities infecting the two host species. The model included host species as categorical predictor, as well as collection site, host sex and host standard length as additional categorical and continuous covariates. Confidence intervals around model coefficients were obtained from 1000 bootstrap replicates of the original dataset in S-Plus.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000734"></div><a name="pone.0000734.s001" id="pone.0000734.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000734.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">List of intron 1 and exon 2 alleles found in each sampled individual. Each individual fish identifier is given in the leftmost column followed by its species, population and its alleles. Alleles are numbered 1-164 and 1-196 for intron 1 and exon 2 alleles respectively. Individual sequences were deposited in the GenBank database under accession number EF539902-EF540320.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000734.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.09 MB XLS)</p>
<a name="pone.0000734.s002" id="pone.0000734.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000734.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">GenBank accession numbers for intron 1 and exon 2 alleles. Accession numbers for sequences corresponding to intron 1 and exon 2 alleles as numbered and listed in <a href="#pone.0000734.s001">figure S1</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000734.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.10 MB XLS)</p>
<a name="pone.0000734.s003" id="pone.0000734.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000734.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Structural Alignment of Cichlid MHC Class II Exon 2 Amino Acid Sequences to Protein Data Bases. The HHpred server (<a href="http://toolkit.tuebingen.mpg.de/hhpred">http://toolkit.tuebingen.mpg.de/hhpred</a>) using a profil hidden Markov model was used to align cichlid sequences to the 18 protein family alignment data bases currently available on the server. Q ss_pred: query secondary structure as predicted by PSIPRED; Q ss_conf: PSIPRED confidence values (0–9); Q query_name: query sequence; Q Consensus: query alignment consensus sequence; −quality of colum-column match (very bad = ; bad −; neutral .; good +; very good |; T Consensus: template alignment consensus sequence; T templ_name: template sequence; T ss_dssp: template secondary structure as determined by DSSP; T ss_pred: template secondary structure as predicted by PSIPRED; T ss_conf: PSIPRED confidence values (0–9); H: alpha-helix; E: extended beta sheet; C: coil; S: bend; B : beta brigded residue; T : turn; G : 3_10 helix residue.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000734.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.16 MB TXT)</p>
<a name="pone.0000734.s004" id="pone.0000734.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000734.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Neighbor-Joining Phylogeny of MHC Class II B Functional Alleles. Neighbor-joining phylogeny of the 130 distinct alleles at 19 positively selected putative ABS amino acid sequences based on p-distances. Node bootstrap support (from 500 replicates) of more than 50% are indicated. Labeled clades were those used in intron 1 d and exon 2 dS comparisons.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000734.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.05 MB EPS)</p>
<a name="pone.0000734.s005" id="pone.0000734.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000734.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Exon 2 and Intron 1 Allele Frequency Distributions in Pseudotropheus fainzilberi and P. emmiltos. Histograms of allele frequencies showing similar distributions for exon 2 amino acid sequences and intron 1 sequences. Private and shared alleles between the two species are also plotted separately. All alleles with an absolute frequency≥2 were included.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000734.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.07 MB EPS)</p>
<a name="pone.0000734.s006" id="pone.0000734.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000734.s006">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Likelihood ratio tests comparing selective regimes affecting MHC class II B exon 2 in Pseudotropheus fainzilberi and P. emmiltos. Likelihood values under each of the alternative hypotheses were obtained from fitting models where species specific parameters were free to vary independently. Selective regimes were not significantly different between the species indicating that functional constraints were similar and therefore that selective pressures at individual sites could be inferred from the combined datasets.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000734.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000734.s007" id="pone.0000734.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000734.s007">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Empirical Bayesian posterior probabilities of belonging to each of the four substitution rate categories defined by a bivariate REL analysis. The selective pressures associated with each categories were defined as follow: purifying selection: dN/dS = 0.432; (nearly) neutral evolution: dN/dS = 1.107; positive selection dN/dS = 2.404, dN/dS = 5.746. Probabilities are reported for each of the 19 codons under positive selection of MHC class II β exon 2 in Pseudotropheus fainzilberi and P. emmiltos. Codon position numbers are those of the mature protein. Probabilities higher than 5% are highlighted in bold.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000734.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000734.s008" id="pone.0000734.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000734.s008">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Mean d, dS, and dN within clades of a neighbor joining tree built using amino acid p-distances at 19 positively selected putative ABS codons of MHC class II β. dS and dN were computed using the modified Nei and Gojobori method with a transition transversion ration of 1.26 (estimated from the data). d at intron 1 was measured using the Jukes and Cantor distance. Standard errors (SE) were computed from 500 bootstrap replicates. All calculations were done in MEGA v.3.1. Significantly (<em>α</em> = 0.05) different mean d in intron 1 and mean dS in exon 2 are marked in bold.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000734.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Jérôme Laroche from the Centre de Bio-Informatique et de Biologie Computationnelle de l'Université Laval for bioinformatics support. We are also grateful to Ad Konings for the photographs of specimens.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: LB GT JB CR. Performed the experiments: JB. Analyzed the data: JB. Contributed reagents/materials/analysis tools: CR. Wrote the paper: LB GT JB CR Cv JC. Other: Provided parasitological expertise in parasite collection and identification: JC. Contributed theoretical and analytical discussions on interpretation of MHC evolution: Cv.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000734-Willig1" id="pone.0000734-Willig1"></a>Willig MR, Kaufman DM, Stevens RD (2003) Latitudinal gradients of biodiversity: pattern, process, scale, and synthesis. Ann Rev Ecol Evol Syst  34: 273–309.  <ul class="find" data-citedArticleID="977212" data-doi="10.1146/annurev.ecolsys.34.012103.144032"><li><a href="http://dx.doi.org/10.1146/annurev.ecolsys.34.012103.144032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Latitudinal+gradients+of+biodiversity%3A+pattern%2C+process%2C+scale%2C+and+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Latitudinal+gradients+of+biodiversity%3A+pattern%2C+process%2C+scale%2C+and+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000734-Buckling1" id="pone.0000734-Buckling1"></a>Buckling A, Rainey PB (2002) The role of parasites in sympatric and allopatric host diversification. Nature  420: 496–499.  <ul class="find" data-citedArticleID="977070" data-doi="10.1038/nature01164"><li><a href="http://dx.doi.org/10.1038/nature01164" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+parasites+in+sympatric+and+allopatric+host+diversification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+parasites+in+sympatric+and+allopatric+host+diversification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000734-Summers1" id="pone.0000734-Summers1"></a>Summers K, McKeon S, Sellars J, Keusenkothen M, Morris J, et al.  (2003) Parasitic exploitation as an engine of diversity. Biol Rev  78: 639–675.  <ul class="find" data-citedArticleID="977188" data-doi="10.1017/s146479310300616x"><li><a href="http://dx.doi.org/10.1017/s146479310300616x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parasitic+exploitation+as+an+engine+of+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parasitic+exploitation+as+an+engine+of+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000734-Haldane1" id="pone.0000734-Haldane1"></a>Haldane JBS (1949) Disease and evolution. La Ricerca Scientifica Suppl A  19: 68–76.  <ul class="find" data-citedArticleID="977096"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Disease+and+evolution.&amp;auth=&amp;atitle=Disease+and+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Disease+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Disease+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000734-Lederberg1" id="pone.0000734-Lederberg1"></a>Lederberg J (1999) J. B. S. Haldane (1949) on infectious disease and evolution. Genetics  153: 1–3.  <ul class="find" data-citedArticleID="977134"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=J.+B.+S.+Haldane+%281949%29+on+infectious+disease+and+evolution.&amp;auth=&amp;atitle=J.+B.+S.+Haldane+%281949%29+on+infectious+disease+and+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=J.+B.+S.+Haldane+%281949%29+on+infectious+disease+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22J.+B.+S.+Haldane+%281949%29+on+infectious+disease+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000734-Turelli1" id="pone.0000734-Turelli1"></a>Turelli M, Barton NH, Coyne JA (2001) Theory and speciation. Trends Ecol Evol  16: 330–343.  <ul class="find" data-citedArticleID="977200" data-doi="10.1016/s0169-5347(01)02177-2"><li><a href="http://dx.doi.org/10.1016/s0169-5347(01)02177-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Theory+and+speciation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Theory+and+speciation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000734-Switzer1" id="pone.0000734-Switzer1"></a>Switzer WM, Salemi M, Shanmugam V, Gao F, Cong M-e, et al.  (2005) Ancient co-speciation of simian foamy viruses and primates. Nature  434: 376–380.  <ul class="find" data-citedArticleID="977190" data-doi="10.1038/nature03341"><li><a href="http://dx.doi.org/10.1038/nature03341" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+co-speciation+of+simian+foamy+viruses+and+primates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+co-speciation+of+simian+foamy+viruses+and+primates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000734-Thompson1" id="pone.0000734-Thompson1"></a>Thompson JN (2005) The Geographic Mosaic of Coevolution. Chicago: The University of Chicago Press.   <ul class="find-nolinks"></ul></li><li><span class="label">9.
              </span><a name="pone.0000734-Nunn1" id="pone.0000734-Nunn1"></a>Nunn CL, Altizer S, Sechrest W, Jones KE, Barton RA, et al.  (2004) Parasites and the evolutionary diversification of primate clades. Am Nat  164: S90–S103.  <ul class="find" data-citedArticleID="977152" data-doi="10.1086/424608"><li><a href="http://dx.doi.org/10.1086/424608" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parasites+and+the+evolutionary+diversification+of+primate+clades." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parasites+and+the+evolutionary+diversification+of+primate+clades.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000734-Krasnov1" id="pone.0000734-Krasnov1"></a>Krasnov BR, Shenbrot GI, Khokhlova IS, Degen AA (2004) Relationship between host diversity and parasite diversity: flea assemblages on small mammals. J Biogeogr  31: 1857–1866.  <ul class="find" data-citedArticleID="977128" data-doi="10.1111/j.1365-2699.2004.01132.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2699.2004.01132.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+between+host+diversity+and+parasite+diversity%3A+flea+assemblages+on+small+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+between+host+diversity+and+parasite+diversity%3A+flea+assemblages+on+small+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000734-Anzai1" id="pone.0000734-Anzai1"></a>Anzai T, Shiina T, Kimura N, Yanagiya K, Kohara S, et al.  (2003) Comparative sequencing of human and chimpanzee MHC class I regions unveils insertions/deletions as the major path to genomic divergence. Proc Natl Acad Sci USA  100: 7708–7713.  <ul class="find" data-citedArticleID="977060" data-doi="10.1073/pnas.1230533100"><li><a href="http://dx.doi.org/10.1073/pnas.1230533100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+sequencing+of+human+and+chimpanzee+MHC+class+I+regions+unveils+insertions%2Fdeletions+as+the+major+path+to+genomic+divergence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+sequencing+of+human+and+chimpanzee+MHC+class+I+regions+unveils+insertions%2Fdeletions+as+the+major+path+to+genomic+divergence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000734-Kruiswijk1" id="pone.0000734-Kruiswijk1"></a>Kruiswijk CP, Hermsen T, Heerwaarden Jv, Dixon B, Savelkoul HFJ, et al.  (2005) Major histocompatibility genes in the Lake Tana African large barb species flock: evidence for complete partitioning of class II B, but not class I, genes among different species. Immunog  56: 894–908.  <ul class="find" data-citedArticleID="977130" data-doi="10.1007/s00251-005-0767-5"><li><a href="http://dx.doi.org/10.1007/s00251-005-0767-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Major+histocompatibility+genes+in+the+Lake+Tana+African+large+barb+species+flock%3A+evidence+for+complete+partitioning+of+class+II+B%2C+but+not+class+I%2C+genes+among+different+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Major+histocompatibility+genes+in+the+Lake+Tana+African+large+barb+species+flock%3A+evidence+for+complete+partitioning+of+class+II+B%2C+but+not+class+I%2C+genes+among+different+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000734-Waterston1" id="pone.0000734-Waterston1"></a>Waterston RH, Lindblad-Toh K, Birney E, Rogers J, Abril JF, et al.  (2002) initial sequencing and comparative analysis of the mouse genome. Nature  420: 520–562.  <ul class="find" data-citedArticleID="977208"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=initial+sequencing+and+comparative+analysis+of+the+mouse+genome.&amp;auth=&amp;atitle=initial+sequencing+and+comparative+analysis+of+the+mouse+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=initial+sequencing+and+comparative+analysis+of+the+mouse+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22initial+sequencing+and+comparative+analysis+of+the+mouse+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000734-Gibbs1" id="pone.0000734-Gibbs1"></a>Gibbs RA, Weinstock GM, et al.  (2004) Genome sequence of the Brown Norway rat yields insights into mammalian evolution. Nature  428: 493–521.  <ul class="find" data-citedArticleID="977094"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+sequence+of+the+Brown+Norway+rat+yields+insights+into+mammalian+evolution.&amp;auth=&amp;atitle=Genome+sequence+of+the+Brown+Norway+rat+yields+insights+into+mammalian+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+the+Brown+Norway+rat+yields+insights+into+mammalian+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+the+Brown+Norway+rat+yields+insights+into+mammalian+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000734-Bernatchez1" id="pone.0000734-Bernatchez1"></a>Bernatchez L, Landry C (2003) MHC studies in nonmodel vertebrates: what have we learned about natural selection in 15 years? J Evol Biol  16: 363–377.  <ul class="find" data-citedArticleID="977062" data-doi="10.1046/j.1420-9101.2003.00531.x"><li><a href="http://dx.doi.org/10.1046/j.1420-9101.2003.00531.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MHC+studies+in+nonmodel+vertebrates%3A+what+have+we+learned+about+natural+selection+in+15+years%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MHC+studies+in+nonmodel+vertebrates%3A+what+have+we+learned+about+natural+selection+in+15+years%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000734-Bjorkman1" id="pone.0000734-Bjorkman1"></a>Bjorkman PJ, Saper MA, Samraoui B, Bennett WS, Strominger JL, et al.  (1987) The foreign antigen binding site and T cell recognition regions of class I histocompatibility antigens. Nature  329: 512–518.  <ul class="find" data-citedArticleID="977064" data-doi="10.1038/329512a0"><li><a href="http://dx.doi.org/10.1038/329512a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+foreign+antigen+binding+site+and+T+cell+recognition+regions+of+class+I+histocompatibility+antigens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+foreign+antigen+binding+site+and+T+cell+recognition+regions+of+class+I+histocompatibility+antigens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000734-Jacob1" id="pone.0000734-Jacob1"></a>Jacob S, McClintock MK, Zelano B, Ober C (2002) Paternally inherited HLA alleles are associated with women's choice of male odor. Nat Gen  30: 175–179.  <ul class="find" data-citedArticleID="977112" data-doi="10.1038/ng830"><li><a href="http://dx.doi.org/10.1038/ng830" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Paternally+inherited+HLA+alleles+are+associated+with+women%27s+choice+of+male+odor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Paternally+inherited+HLA+alleles+are+associated+with+women%27s+choice+of+male+odor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000734-Carroll1" id="pone.0000734-Carroll1"></a>Carroll LS, Penn DJ, Potts WK (2002) Discrimination of MHC-derived odors by untrained mice is consistent with divergence in peptide-binding region residues. Proc Natl Acad Sci USA  99: 2187–2192.  <ul class="find" data-citedArticleID="977072" data-doi="10.1073/pnas.042244899"><li><a href="http://dx.doi.org/10.1073/pnas.042244899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discrimination+of+MHC-derived+odors+by+untrained+mice+is+consistent+with+divergence+in+peptide-binding+region+residues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discrimination+of+MHC-derived+odors+by+untrained+mice+is+consistent+with+divergence+in+peptide-binding+region+residues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000734-Sommer1" id="pone.0000734-Sommer1"></a>Sommer S (2005) Major histocompatibility complex and mate choice in a monogamous rodent. Behav Ecol Sociobiol  58: 181–189.  <ul class="find" data-citedArticleID="977184" data-doi="10.1007/s00265-005-0909-7"><li><a href="http://dx.doi.org/10.1007/s00265-005-0909-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Major+histocompatibility+complex+and+mate+choice+in+a+monogamous+rodent." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Major+histocompatibility+complex+and+mate+choice+in+a+monogamous+rodent.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000734-Landry1" id="pone.0000734-Landry1"></a>Landry C, Garant D, Duchesne P, Bernatchez L (2001) “Good genes as heterozygosity”: MHC and mate choice in Atlantic salmon (<em>Salmo salar</em>) Proc R Soc Lond B 268: 1279–1285.  <ul class="find" data-citedArticleID="977132" data-doi="10.1098/rspb.2001.1659"><li><a href="http://dx.doi.org/10.1098/rspb.2001.1659" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%26%23x201c%3BGood+genes+as+heterozygosity%26%23x201d%3B%3A+MHC+and+mate+choice+in+Atlantic+salmon+%28Salmo+salar%29+Proc+R+Soc+Lond+B" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%26%23x201c%3BGood+genes+as+heterozygosity%26%23x201d%3B%3A+MHC+and+mate+choice+in+Atlantic+salmon+%28Salmo+salar%29+Proc+R+Soc+Lond+B%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000734-Milinski1" id="pone.0000734-Milinski1"></a>Milinski M, Griffiths S, Wegner KM, Reusch TBH, Haas-Assenbaum A, et al.  (2005) Mate choice decisions of stickleback females predictably modified by MHC peptide ligands. Proc Natl Acad Sci USA  102: 4414–4418.  <ul class="find" data-citedArticleID="977140" data-doi="10.1073/pnas.0408264102"><li><a href="http://dx.doi.org/10.1073/pnas.0408264102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mate+choice+decisions+of+stickleback+females+predictably+modified+by+MHC+peptide+ligands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mate+choice+decisions+of+stickleback+females+predictably+modified+by+MHC+peptide+ligands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000734-Kocher1" id="pone.0000734-Kocher1"></a>Kocher TD (2004) Adaptive evolution and explosive speciation: The cichlid fish model. Nat Rev Gen  5: 288–298.  <ul class="find" data-citedArticleID="977114" data-doi="10.1038/nrg1316"><li><a href="http://dx.doi.org/10.1038/nrg1316" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+and+explosive+speciation%3A+The+cichlid+fish+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+and+explosive+speciation%3A+The+cichlid+fish+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000734-Genner1" id="pone.0000734-Genner1"></a>Genner MJ, Turner GF (2005) The mbuna cichlids of Lake Malawi: a model for rapid speciation and adaptive radiation. Fish and Fisheries  6: 1–34.  <ul class="find" data-citedArticleID="977090" data-doi="10.1111/j.1467-2679.2005.00173.x"><li><a href="http://dx.doi.org/10.1111/j.1467-2679.2005.00173.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mbuna+cichlids+of+Lake+Malawi%3A+a+model+for+rapid+speciation+and+adaptive+radiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mbuna+cichlids+of+Lake+Malawi%3A+a+model+for+rapid+speciation+and+adaptive+radiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000734-Ribbink1" id="pone.0000734-Ribbink1"></a>Ribbink AJ, Marsh BA, Marsh AC, Ribbink AC, Sharp BJ (1983) A preliminary survey of the cichlid fishes of rocky habitats in Lake Malawi. S Afr J Zool  18: 147–310.  <ul class="find" data-citedArticleID="977168" data-doi="10.1007/bf00005391"><li><a href="http://dx.doi.org/10.1007/bf00005391" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+preliminary+survey+of+the+cichlid+fishes+of+rocky+habitats+in+Lake+Malawi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+preliminary+survey+of+the+cichlid+fishes+of+rocky+habitats+in+Lake+Malawi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000734-Turner1" id="pone.0000734-Turner1"></a>Turner GF (1999)  Explosive speciation of African cichlid fishes. In: Maguran AE, May RM, editors. Evolution of Biological Diversity. Oxford: Oxford University Press. pp. 113–129.  <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0000734-Rico1" id="pone.0000734-Rico1"></a>Rico C, Turner GF (2002) Mol Ecol 11: 1585–1590.  <ul class="find" data-citedArticleID="977170" data-doi="10.1046/j.1365-294x.2002.01537.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2002.01537.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mol+Ecol" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mol+Ecol%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000734-Fenton1" id="pone.0000734-Fenton1"></a>Fenton A, Fairbairn JP, Norman R, Hudson PJ (2002) Parasite transmission: reconciling theory and reality. J Anim Ecol  71: 893–905.  <ul class="find" data-citedArticleID="977084" data-doi="10.1046/j.1365-2656.2002.00656.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2656.2002.00656.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parasite+transmission%3A+reconciling+theory+and+reality." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parasite+transmission%3A+reconciling+theory+and+reality.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000734-Taylor1" id="pone.0000734-Taylor1"></a>Taylor MI, Turner GF, Robinson RL, Stauffer JR JR (1998) Sexual selection, parasites and bower height skew in a bower-building cichlid fish. Anim Behav  56: 379–384.  <ul class="find" data-citedArticleID="977192" data-doi="10.1006/anbe.1998.0795"><li><a href="http://dx.doi.org/10.1006/anbe.1998.0795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sexual+selection%2C+parasites+and+bower+height+skew+in+a+bower-building+cichlid+fish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sexual+selection%2C+parasites+and+bower+height+skew+in+a+bower-building+cichlid+fish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000734-Maan1" id="pone.0000734-Maan1"></a>Maan ME, van der Spoel M, Jimenez PQ, van Alphen JJM, Seehausen O (2006) Fitness correlates of male coloration in a Lake Victoria cichlid fish. Behav Ecol  17: 691–699.  <ul class="find" data-citedArticleID="977138" data-doi="10.1093/beheco/ark020"><li><a href="http://dx.doi.org/10.1093/beheco/ark020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitness+correlates+of+male+coloration+in+a+Lake+Victoria+cichlid+fish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fitness+correlates+of+male+coloration+in+a+Lake+Victoria+cichlid+fish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000734-Plenderleith1" id="pone.0000734-Plenderleith1"></a>Plenderleith M, van Oosterhout C, Robinson RL, Turner GF (2005) Female preference for conspecific males based on olfactory cues in a Lake Malawi cichlid fish. Biol Lett  1: 411–414.  <ul class="find" data-citedArticleID="977160" data-doi="10.1098/rsbl.2005.0355"><li><a href="http://dx.doi.org/10.1098/rsbl.2005.0355" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Female+preference+for+conspecific+males+based+on+olfactory+cues+in+a+Lake+Malawi+cichlid+fish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Female+preference+for+conspecific+males+based+on+olfactory+cues+in+a+Lake+Malawi+cichlid+fish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000734-Schluter1" id="pone.0000734-Schluter1"></a>Schluter D (2001) Ecology and the origin of species. Trends Ecol Evol  16: 372–380.  <ul class="find" data-citedArticleID="977176" data-doi="10.1016/s0169-5347(01)02198-x"><li><a href="http://dx.doi.org/10.1016/s0169-5347(01)02198-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecology+and+the+origin+of+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecology+and+the+origin+of+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000734-MlagaTrillo1" id="pone.0000734-MlagaTrillo1"></a>Málaga-Trillo E, Zaleska-Rutczynska Z, McAndrew B, Vincek V, Figueroa F, et al.  (1998) Linkage Relationship and Haplotype Polymorphism Among Cichlid <em>Mhc</em> Class II <em>B</em> Loci. Genetics  149: 1527–1537.  <ul class="find" data-citedArticleID="977142"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Linkage+Relationship+and+Haplotype+Polymorphism+Among+Cichlid+Mhc+Class+II+B+Loci.&amp;auth=&amp;atitle=Linkage+Relationship+and+Haplotype+Polymorphism+Among+Cichlid+Mhc+Class+II+B+Loci." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+Relationship+and+Haplotype+Polymorphism+Among+Cichlid+Mhc+Class+II+B+Loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+Relationship+and+Haplotype+Polymorphism+Among+Cichlid+Mhc+Class+II+B+Loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000734-Ono1" id="pone.0000734-Ono1"></a>Ono H, Klein D, Vincek V, Figueroa F, O'hUigin C, et al.  (1992) Major histocompatibility complex class II genes of zebrafish. Proc Natl Acad Sci USA  89: 11886–11890.  <ul class="find" data-citedArticleID="977154" data-doi="10.1073/pnas.89.24.11886"><li><a href="http://dx.doi.org/10.1073/pnas.89.24.11886" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Major+histocompatibility+complex+class+II+genes+of+zebrafish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Major+histocompatibility+complex+class+II+genes+of+zebrafish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000734-Emes1" id="pone.0000734-Emes1"></a>Emes RD, Beatson SA, Ponting CP, Goodstadt L (2004) Evolution and comparative genomics of odorant- and pheromone-associated genes in rodents. Genome Research  14: 591–602.  <ul class="find" data-citedArticleID="977080" data-doi="10.1101/gr.1940604"><li><a href="http://dx.doi.org/10.1101/gr.1940604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+and+comparative+genomics+of+odorant-+and+pheromone-associated+genes+in+rodents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+and+comparative+genomics+of+odorant-+and+pheromone-associated+genes+in+rodents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000734-Hughes1" id="pone.0000734-Hughes1"></a>Hughes AL (2000) Evolution of introns and exons of class II major histocompatibility complex genes of vertebrates. Immunog  51: 473–486.  <ul class="find" data-citedArticleID="977108" data-doi="10.1007/s002510050646"><li><a href="http://dx.doi.org/10.1007/s002510050646" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+introns+and+exons+of+class+II+major+histocompatibility+complex+genes+of+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+introns+and+exons+of+class+II+major+histocompatibility+complex+genes+of+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000734-Reusch1" id="pone.0000734-Reusch1"></a>Reusch TBH, Langefors Å (2005) Inter- and Intralocus recombination drive MHC class IIB gene diversification in a teleost, the three-spined stickleback <em>Gasterosteus aculeatus</em> J Mol Evol  61: 531–541.  <ul class="find" data-citedArticleID="977164" data-doi="10.1007/s00239-004-0340-0"><li><a href="http://dx.doi.org/10.1007/s00239-004-0340-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inter-+and+Intralocus+recombination+drive+MHC+class+IIB+gene+diversification+in+a+teleost%2C+the+three-spined+stickleback+Gasterosteus+aculeatus" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inter-+and+Intralocus+recombination+drive+MHC+class+IIB+gene+diversification+in+a+teleost%2C+the+three-spined+stickleback+Gasterosteus+aculeatus%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000734-KosakovskyPond1" id="pone.0000734-KosakovskyPond1"></a>Kosakovsky Pond SL, Posada D, Gravenor MB, Woelk CH, Frost SDW (2006) Automated phylogenetic detection of recombination using a genetic algorithm. Mol Biol Evol  23: 1891–1901.  <ul class="find" data-citedArticleID="977120" data-doi="10.1093/molbev/msl051"><li><a href="http://dx.doi.org/10.1093/molbev/msl051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automated+phylogenetic+detection+of+recombination+using+a+genetic+algorithm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automated+phylogenetic+detection+of+recombination+using+a+genetic+algorithm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000734-Legendre1" id="pone.0000734-Legendre1"></a>Legendre P, Gallagher ED (2001) Ecologically meaningful transformations for ordination of species data. Oecologia  129: 271–280.  <ul class="find" data-citedArticleID="977136" data-doi="10.1007/s004420100716"><li><a href="http://dx.doi.org/10.1007/s004420100716" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecologically+meaningful+transformations+for+ordination+of+species+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecologically+meaningful+transformations+for+ordination+of+species+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000734-Frank1" id="pone.0000734-Frank1"></a>Frank SA (1994) Recognition and polymorphism in host-parasite genetics. Philos Trans R Soc Lond B Biol Sci  346: 283–293.  <ul class="find" data-citedArticleID="977086"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Recognition+and+polymorphism+in+host-parasite+genetics.&amp;auth=&amp;atitle=Recognition+and+polymorphism+in+host-parasite+genetics." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+and+polymorphism+in+host-parasite+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+and+polymorphism+in+host-parasite+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000734-Schluter2" id="pone.0000734-Schluter2"></a>Schluter D (2000)  The Ecology of Adaptive Radiation;. In: May RM, Harvey PH, editors. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0000734-Schierup1" id="pone.0000734-Schierup1"></a>Schierup MH, Vekemans X, Charlesworth D (2000) The effect of hitch-hiking on genes linked to a balanced polymorphism in a subdivided population. Gen Res  76: 63–73.  <ul class="find" data-citedArticleID="977174" data-doi="10.1017/s0016672300004547"><li><a href="http://dx.doi.org/10.1017/s0016672300004547" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+hitch-hiking+on+genes+linked+to+a+balanced+polymorphism+in+a+subdivided+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+hitch-hiking+on+genes+linked+to+a+balanced+polymorphism+in+a+subdivided+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000734-vanOosterhout1" id="pone.0000734-vanOosterhout1"></a>van Oosterhout C, Joyce DA, Cummings SM, Blais J, Barson NJ, et al.  (2006) Balancing selection, random genetic drift, and genetic variation at the major histocompatibility complex in two wild populations of guppies (<em>Poecilia reticulata</em>). Evolution  60: 2562–2574.  <ul class="find" data-citedArticleID="977218" data-doi="10.1111/j.0014-3820.2006.tb01890.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2006.tb01890.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Balancing+selection%2C+random+genetic+drift%2C+and+genetic+variation+at+the+major+histocompatibility+complex+in+two+wild+populations+of+guppies+%28Poecilia+reticulata%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Balancing+selection%2C+random+genetic+drift%2C+and+genetic+variation+at+the+major+histocompatibility+complex+in+two+wild+populations+of+guppies+%28Poecilia+reticulata%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000734-Skarstein1" id="pone.0000734-Skarstein1"></a>Skarstein F, Folstad I, Rønning HP (2005) Spawning colouration, parasites and habitat selection in <em>Salvelinus alpinus</em>: initiating speciation by sexual selection? J Fish Biol  67: 969–980.  <ul class="find" data-citedArticleID="977182" data-doi="10.1111/j.0022-1112.2005.00796.x"><li><a href="http://dx.doi.org/10.1111/j.0022-1112.2005.00796.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spawning+colouration%2C+parasites+and+habitat+selection+in+Salvelinus+alpinus%3A+initiating+speciation+by+sexual+selection%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spawning+colouration%2C+parasites+and+habitat+selection+in+Salvelinus+alpinus%3A+initiating+speciation+by+sexual+selection%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000734-Cohen1" id="pone.0000734-Cohen1"></a>Cohen S (2002) Strong positive selection and habitat-specific amino acid substitution patterns in Mhc from an estuarine fish under intense pollution stress. Mol Biol Evol  10: 1870–1880.  <ul class="find" data-citedArticleID="977074" data-doi="10.1093/oxfordjournals.molbev.a004011"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strong+positive+selection+and+habitat-specific+amino+acid+substitution+patterns+in+Mhc+from+an+estuarine+fish+under+intense+pollution+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strong+positive+selection+and+habitat-specific+amino+acid+substitution+patterns+in+Mhc+from+an+estuarine+fish+under+intense+pollution+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000734-Kornfield1" id="pone.0000734-Kornfield1"></a>Kornfield I, Smith PF (2000) African cichlid fishes: Model systems for evolutionary biology. Ann Rev Ecol Syst  31: 163–196.  <ul class="find" data-citedArticleID="977118" data-doi="10.1146/annurev.ecolsys.31.1.163"><li><a href="http://dx.doi.org/10.1146/annurev.ecolsys.31.1.163" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=African+cichlid+fishes%3A+Model+systems+for+evolutionary+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22African+cichlid+fishes%3A+Model+systems+for+evolutionary+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000734-Deutsch1" id="pone.0000734-Deutsch1"></a>Deutsch JC (1997) Colour diversification in Malawi cichlids: evidence for adaptation, reinforcement or sexual selection? Biol J Linn Soc  62: 1–14.  <ul class="find" data-citedArticleID="977076" data-doi="10.1111/j.1095-8312.1997.tb01614.x"><li><a href="http://dx.doi.org/10.1111/j.1095-8312.1997.tb01614.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Colour+diversification+in+Malawi+cichlids%3A+evidence+for+adaptation%2C+reinforcement+or+sexual+selection%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Colour+diversification+in+Malawi+cichlids%3A+evidence+for+adaptation%2C+reinforcement+or+sexual+selection%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000734-Albertson1" id="pone.0000734-Albertson1"></a>Albertson RC, Markert JA, Danley PD, Kocher TD (1999) Phylogeny of a rapidly evolving clade: The cichlid fishes of Lake Malawi, East Africa. Proc Natl Acad Sci USA  96: 5107–5110.  <ul class="find" data-citedArticleID="977056" data-doi="10.1073/pnas.96.9.5107"><li><a href="http://dx.doi.org/10.1073/pnas.96.9.5107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+of+a+rapidly+evolving+clade%3A+The+cichlid+fishes+of+Lake+Malawi%2C+East+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+of+a+rapidly+evolving+clade%3A+The+cichlid+fishes+of+Lake+Malawi%2C+East+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000734-Terai1" id="pone.0000734-Terai1"></a>Terai Y, Seehausen O, Sasaki T, Takahashi K, Mizoiri S, et al.  (2006) Divergent selection on opsins drives incipient speciation in Lake Victoria cichlids. PLoS Biology  4:   doi:10.1371/journal.pbio.0040433.  <ul class="find" data-citedArticleID="977194" data-doi="10.1371/journal.pbio.0040433"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Divergent+selection+on+opsins+drives+incipient+speciation+in+Lake+Victoria+cichlids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Divergent+selection+on+opsins+drives+incipient+speciation+in+Lake+Victoria+cichlids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000734-Dominey1" id="pone.0000734-Dominey1"></a>Dominey WJ (1984)  Effects of sexual selection and life history on speciation: Species flocks in African cichlids and Hawaiian Drosophila. In: Echelle AA, Kornfield I, editors. Evolution of fish species flocks. Orono, Maine: University of Maine at Orono Press. pp. 231–249.  <ul class="find-nolinks"></ul></li><li><span class="label">50.
              </span><a name="pone.0000734-Seehausen1" id="pone.0000734-Seehausen1"></a>Seehausen O, van Alphen JJM (1999) Can sympatric speciation by disruptive sexual selection explain rapid evolution of cichlid diversity in Lake Victoria? Ecol Lett  2: 262–271.  <ul class="find" data-citedArticleID="977180" data-doi="10.1046/j.1461-0248.1999.00082.x"><li><a href="http://dx.doi.org/10.1046/j.1461-0248.1999.00082.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Can+sympatric+speciation+by+disruptive+sexual+selection+explain+rapid+evolution+of+cichlid+diversity+in+Lake+Victoria%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Can+sympatric+speciation+by+disruptive+sexual+selection+explain+rapid+evolution+of+cichlid+diversity+in+Lake+Victoria%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000734-Allender1" id="pone.0000734-Allender1"></a>Allender CJ, Seehausen O, Knight ME, Turner GF, Maclean N (2003) Divergent selection during speciation of Lake Malawi cichlid fishes inferred from parallel radiations in nuptial coloration. Proc Natl Acad Sci USA  100: 14074–14079.  <ul class="find" data-citedArticleID="977058" data-doi="10.1073/pnas.2332665100"><li><a href="http://dx.doi.org/10.1073/pnas.2332665100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Divergent+selection+during+speciation+of+Lake+Malawi+cichlid+fishes+inferred+from+parallel+radiations+in+nuptial+coloration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Divergent+selection+during+speciation+of+Lake+Malawi+cichlid+fishes+inferred+from+parallel+radiations+in+nuptial+coloration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000734-Hamilton1" id="pone.0000734-Hamilton1"></a>Hamilton WD, Zuk M (1982) Heritable true fitness and bright birds: a role for parasites? Science  218: 384–387.  <ul class="find" data-citedArticleID="977098" data-doi="10.1126/science.7123238"><li><a href="http://dx.doi.org/10.1126/science.7123238" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heritable+true+fitness+and+bright+birds%3A+a+role+for+parasites%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heritable+true+fitness+and+bright+birds%3A+a+role+for+parasites%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000734-Hamilton2" id="pone.0000734-Hamilton2"></a>Hamilton WD, Axelrod R, Tanese R (1990) Sexual reproduction as an adaptation to resist parasites. Proc Natl Acad Sci USA  87: 3566–3573.  <ul class="find" data-citedArticleID="977100" data-doi="10.1073/pnas.87.9.3566"><li><a href="http://dx.doi.org/10.1073/pnas.87.9.3566" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sexual+reproduction+as+an+adaptation+to+resist+parasites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sexual+reproduction+as+an+adaptation+to+resist+parasites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000734-VonSchantz1" id="pone.0000734-VonSchantz1"></a>Von Schantz T, Wittzell H, Goransson G, Grahn M, Persson K (1996) MHC genotype and male ornamentation: Genetic evidence for the Hamilton-Zuk model. Proc R Soc Lond B  263: 265–271.  <ul class="find" data-citedArticleID="977204" data-doi="10.1098/rspb.1996.0041"><li><a href="http://dx.doi.org/10.1098/rspb.1996.0041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MHC+genotype+and+male+ornamentation%3A+Genetic+evidence+for+the+Hamilton-Zuk+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MHC+genotype+and+male+ornamentation%3A+Genetic+evidence+for+the+Hamilton-Zuk+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000734-Westneat1" id="pone.0000734-Westneat1"></a>Westneat DF, Birkhead TR (1998) Alternative hypotheses linking the immune system and mate choice for good genes. Proc R Soc Lond B  265: 1065–1073.  <ul class="find" data-citedArticleID="977210" data-doi="10.1098/rspb.1998.0400"><li><a href="http://dx.doi.org/10.1098/rspb.1998.0400" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+hypotheses+linking+the+immune+system+and+mate+choice+for+good+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+hypotheses+linking+the+immune+system+and+mate+choice+for+good+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000734-Getty1" id="pone.0000734-Getty1"></a>Getty T (2002) Signaling health versus parasites. Am Nat  159: 363–371.  <ul class="find" data-citedArticleID="977092" data-doi="10.1086/338992"><li><a href="http://dx.doi.org/10.1086/338992" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signaling+health+versus+parasites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signaling+health+versus+parasites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000734-Moore1" id="pone.0000734-Moore1"></a>Moore SL, Wilson K (2002) Parasites as a viability cost of sexual selection in natural populations of mammals. Science  297: 2015–2018.  <ul class="find" data-citedArticleID="977144" data-doi="10.1126/science.1074196"><li><a href="http://dx.doi.org/10.1126/science.1074196" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parasites+as+a+viability+cost+of+sexual+selection+in+natural+populations+of+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parasites+as+a+viability+cost+of+sexual+selection+in+natural+populations+of+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000734-Garamszegi1" id="pone.0000734-Garamszegi1"></a>Garamszegi LZ, Moller AP, Erritzoe J (2003) The evolution of immune defense and song complexity in birds. Evolution  57: 905–912.  <ul class="find" data-citedArticleID="977088" data-doi="10.1554/0014-3820(2003)057[0905:teoida]2.0.co;2"><li><a href="http://dx.doi.org/10.1554/0014-3820(2003)057[0905:teoida]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+immune+defense+and+song+complexity+in+birds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+immune+defense+and+song+complexity+in+birds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000734-Blais1" id="pone.0000734-Blais1"></a>Blais J, Rico C, Bernatchez L (2004) Nonlinear effects of female mate choice in wild threespine sticklebacks. Evolution  58: 2498–2510.  <ul class="find" data-citedArticleID="977066" data-doi="10.1554/04-189"><li><a href="http://dx.doi.org/10.1554/04-189" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonlinear+effects+of+female+mate+choice+in+wild+threespine+sticklebacks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonlinear+effects+of+female+mate+choice+in+wild+threespine+sticklebacks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000734-Reusch2" id="pone.0000734-Reusch2"></a>Reusch TBH, Haberli MA, Aeschlimann PB, Milinski M (2001) Female sticklebacks count alleles in a strategy of sexual selection explaining MHC polymorphism. Nature  414: 300–302.  <ul class="find" data-citedArticleID="977166" data-doi="10.1038/35104547"><li><a href="http://dx.doi.org/10.1038/35104547" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Female+sticklebacks+count+alleles+in+a+strategy+of+sexual+selection+explaining+MHC+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Female+sticklebacks+count+alleles+in+a+strategy+of+sexual+selection+explaining+MHC+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000734-Nowak1" id="pone.0000734-Nowak1"></a>Nowak MA, Tarczy-Hornoch K, Austyn JM (1992) The optimal number of major histocompatibility complex molecules in an individual. Proc Natl Acad Sci USA  89: 10896–10899.  <ul class="find" data-citedArticleID="977150" data-doi="10.1073/pnas.89.22.10896"><li><a href="http://dx.doi.org/10.1073/pnas.89.22.10896" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+optimal+number+of+major+histocompatibility+complex+molecules+in+an+individual." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+optimal+number+of+major+histocompatibility+complex+molecules+in+an+individual.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000734-Borghans1" id="pone.0000734-Borghans1"></a>Borghans JAM, Noest AJ, De Boer RJ (2003) Thymic selection does not limit the individual MHC diversity. Eur J Immunol  33: 3353–3358.  <ul class="find" data-citedArticleID="977068" data-doi="10.1002/eji.200324365"><li><a href="http://dx.doi.org/10.1002/eji.200324365" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thymic+selection+does+not+limit+the+individual+MHC+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thymic+selection+does+not+limit+the+individual+MHC+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000734-vandenBerg1" id="pone.0000734-vandenBerg1"></a>van den Berg HA, Rand DA (2003) Antigen presentation on MHC molecules as a diversity filter that enhances immune efficacy. J Theor Biol  224: 249–267.  <ul class="find" data-citedArticleID="977220" data-doi="10.1016/s0022-5193(03)00162-0"><li><a href="http://dx.doi.org/10.1016/s0022-5193(03)00162-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antigen+presentation+on+MHC+molecules+as+a+diversity+filter+that+enhances+immune+efficacy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antigen+presentation+on+MHC+molecules+as+a+diversity+filter+that+enhances+immune+efficacy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000734-Felsenstein1" id="pone.0000734-Felsenstein1"></a>Felsenstein J (1981) Skepticism towards Santa Rosalia, or why are there so few kinds of animals? Evolution  35: 124–138.  <ul class="find" data-citedArticleID="977082" data-doi="10.2307/2407946"><li><a href="http://dx.doi.org/10.2307/2407946" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Skepticism+towards+Santa+Rosalia%2C+or+why+are+there+so+few+kinds+of+animals%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Skepticism+towards+Santa+Rosalia%2C+or+why+are+there+so+few+kinds+of+animals%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000734-Konings1" id="pone.0000734-Konings1"></a>Konings A (2001) Malawi cichlids in their natural habitat. El Paso, TX: Cichlid Press.   <ul class="find-nolinks"></ul></li><li><span class="label">66.
              </span><a name="pone.0000734-Nielsen1" id="pone.0000734-Nielsen1"></a>Nielsen R, Wakeley J (2001) Distinguishing migration from isolation: a Markov chain Monte Carlo approach. Genetics  158: 885–896.  <ul class="find" data-citedArticleID="977148"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Distinguishing+migration+from+isolation%3A+a+Markov+chain+Monte+Carlo+approach.&amp;auth=&amp;atitle=Distinguishing+migration+from+isolation%3A+a+Markov+chain+Monte+Carlo+approach." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinguishing+migration+from+isolation%3A+a+Markov+chain+Monte+Carlo+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinguishing+migration+from+isolation%3A+a+Markov+chain+Monte+Carlo+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000734-Hughes2" id="pone.0000734-Hughes2"></a>Hughes AL, Nei M (1988) Pattern of nucleotide substitution at major histocompatibility complex class I loci reveals overdominant selection. Nature  335: 167–170.  <ul class="find" data-citedArticleID="977110" data-doi="10.1038/335167a0"><li><a href="http://dx.doi.org/10.1038/335167a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pattern+of+nucleotide+substitution+at+major+histocompatibility+complex+class+I+loci+reveals+overdominant+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pattern+of+nucleotide+substitution+at+major+histocompatibility+complex+class+I+loci+reveals+overdominant+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000734-Yang1" id="pone.0000734-Yang1"></a>Yang Z, Swanson WJ (2002) Codon-substitution models to detect adaptive evolution that account for heterogeneous selective pressures among site classes. Mol Biol Evol  19: 49–57.  <ul class="find" data-citedArticleID="977216" data-doi="10.1093/oxfordjournals.molbev.a003981"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003981" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon-substitution+models+to+detect+adaptive+evolution+that+account+for+heterogeneous+selective+pressures+among+site+classes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon-substitution+models+to+detect+adaptive+evolution+that+account+for+heterogeneous+selective+pressures+among+site+classes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000734-Ono2" id="pone.0000734-Ono2"></a>Ono H, O'hUigin C, Tichy H, Klein D (1993) Major-histocompatibility-complex variation in two species of cichlid fishes from Lake Malawi. Mol Biol Evol  10: 1060–1072.  <ul class="find" data-citedArticleID="977156"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Major-histocompatibility-complex+variation+in+two+species+of+cichlid+fishes+from+Lake+Malawi.&amp;auth=&amp;atitle=Major-histocompatibility-complex+variation+in+two+species+of+cichlid+fishes+from+Lake+Malawi." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Major-histocompatibility-complex+variation+in+two+species+of+cichlid+fishes+from+Lake+Malawi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Major-histocompatibility-complex+variation+in+two+species+of+cichlid+fishes+from+Lake+Malawi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000734-Thompson2" id="pone.0000734-Thompson2"></a>Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG (1997) The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res  24: 4876–4882.  <ul class="find" data-citedArticleID="977198" data-doi="10.1093/nar/25.24.4876"><li><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ClustalX+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ClustalX+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000734-Saitou1" id="pone.0000734-Saitou1"></a>Saitou N, Nei M (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evol  4: 406–425.  <ul class="find" data-citedArticleID="977172"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.&amp;auth=&amp;atitle=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000734-Posada1" id="pone.0000734-Posada1"></a>Posada D, Crandall KA (2001) Evaluation of methods for detecting recombination from DNA sequences: computer simulations. Proc Natl Acad Sci USA  98: 13757–13762.  <ul class="find" data-citedArticleID="977162" data-doi="10.1073/pnas.241370698"><li><a href="http://dx.doi.org/10.1073/pnas.241370698" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+methods+for+detecting+recombination+from+DNA+sequences%3A+computer+simulations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+methods+for+detecting+recombination+from+DNA+sequences%3A+computer+simulations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000734-Parham1" id="pone.0000734-Parham1"></a>Parham P, Ohta T (1996) Population biology of antigen presentation by MHC class I molecules. Science  272: 67–74.  <ul class="find" data-citedArticleID="977158" data-doi="10.1126/science.272.5258.67"><li><a href="http://dx.doi.org/10.1126/science.272.5258.67" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+biology+of+antigen+presentation+by+MHC+class+I+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+biology+of+antigen+presentation+by+MHC+class+I+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000734-Aguilar1" id="pone.0000734-Aguilar1"></a>Aguilar A, Roemer G, Debenham S, Binns M, Garcelon D, et al.  (2004) High MHC diversity maintained by balancing selection in an otherwise genetically monomorphic mammal. Proc Natl Acad Sci USA  101: 3490–3494.  <ul class="find" data-citedArticleID="977054" data-doi="10.1073/pnas.0306582101"><li><a href="http://dx.doi.org/10.1073/pnas.0306582101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+MHC+diversity+maintained+by+balancing+selection+in+an+otherwise+genetically+monomorphic+mammal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+MHC+diversity+maintained+by+balancing+selection+in+an+otherwise+genetically+monomorphic+mammal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0000734-KosakovskyPond2" id="pone.0000734-KosakovskyPond2"></a>Kosakovsky Pond SL, Frost SDW, Grossman Z, Gravenor MB, Richman DD, et al.  (2006) Adaptation to Different Human Populations by HIV-1 Revealed by Codon-Based Analyses. Plos Computational Biology  2: e62.  <ul class="find" data-citedArticleID="977122" data-doi="10.1371/journal.pcbi.0020062.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020062.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptation+to+Different+Human+Populations+by+HIV-1+Revealed+by+Codon-Based+Analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptation+to+Different+Human+Populations+by+HIV-1+Revealed+by+Codon-Based+Analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000734-KosakovskyPond3" id="pone.0000734-KosakovskyPond3"></a>Kosakovsky Pond SL, Muse SV (2005) Site-to-site variation in synonymous substitution rates. Mol Biol Evol  22: 2375–2385.  <ul class="find" data-citedArticleID="977124" data-doi="10.1093/molbev/msi232"><li><a href="http://dx.doi.org/10.1093/molbev/msi232" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Site-to-site+variation+in+synonymous+substitution+rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Site-to-site+variation+in+synonymous+substitution+rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0000734-Sorhannus1" id="pone.0000734-Sorhannus1"></a>Sorhannus U, Kosakovsky Pond SL (2006) Evidence for positive selection on a sexual reproduction gene in the diatom genus Thalassiosira (Bacillariophyta). J Mol Evol  63: 231–239.  <ul class="find" data-citedArticleID="977186" data-doi="10.1007/s00239-006-0016-z"><li><a href="http://dx.doi.org/10.1007/s00239-006-0016-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+positive+selection+on+a+sexual+reproduction+gene+in+the+diatom+genus+Thalassiosira+%28Bacillariophyta%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+positive+selection+on+a+sexual+reproduction+gene+in+the+diatom+genus+Thalassiosira+%28Bacillariophyta%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0000734-KosakovskyPond4" id="pone.0000734-KosakovskyPond4"></a>Kosakovsky Pond SL, Frost SDW (2005) Not so different after all: A comparison of methods for detecting amino acid sites under selection. Mol Biol Evol  22: 1208–1222.  <ul class="find" data-citedArticleID="977126" data-doi="10.1093/molbev/msi105"><li><a href="http://dx.doi.org/10.1093/molbev/msi105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Not+so+different+after+all%3A+A+comparison+of+methods+for+detecting+amino+acid+sites+under+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Not+so+different+after+all%3A+A+comparison+of+methods+for+detecting+amino+acid+sites+under+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0000734-Wang1" id="pone.0000734-Wang1"></a>Wang TH, Donaldson YK, Brettle RP, Bell JE, Simmonds P (2001) Identification of shared populations of human immunodeficiency virus type 1 infecting microglia and tissue macrophages outside the central nervous system. J Virol  75: 11686–11699.  <ul class="find" data-citedArticleID="977206" data-doi="10.1128/jvi.75.23.11686-11699.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.23.11686-11699.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+shared+populations+of+human+immunodeficiency+virus+type+1+infecting+microglia+and+tissue+macrophages+outside+the+central+nervous+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+shared+populations+of+human+immunodeficiency+virus+type+1+infecting+microglia+and+tissue+macrophages+outside+the+central+nervous+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0000734-Hudson1" id="pone.0000734-Hudson1"></a>Hudson R, Slatkin M, Maddison W (1992) Estimation of levels of gene flow from DNA sequence data. Genetics  132: 583–589.  <ul class="find" data-citedArticleID="977104"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Estimation+of+levels+of+gene+flow+from+DNA+sequence+data.&amp;auth=&amp;atitle=Estimation+of+levels+of+gene+flow+from+DNA+sequence+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+levels+of+gene+flow+from+DNA+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+levels+of+gene+flow+from+DNA+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0000734-Yampolsky1" id="pone.0000734-Yampolsky1"></a>Yampolsky LY, Stoltzfus A (2005) The Exchangeability of Amino Acids in Proteins Genetics 170: 1459–1472.  <ul class="find" data-citedArticleID="977214" data-doi="10.1534/genetics.104.039107"><li><a href="http://dx.doi.org/10.1534/genetics.104.039107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Exchangeability+of+Amino+Acids+in+Proteins+Genetics" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Exchangeability+of+Amino+Acids+in+Proteins+Genetics%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0000734-Nei1" id="pone.0000734-Nei1"></a>Nei M (1973) Analysis of gene diversity in subdivided populations. Proc Natl Acad Sci USA  70: 3321–3323.  <ul class="find" data-citedArticleID="977146" data-doi="10.1073/pnas.70.12.3321"><li><a href="http://dx.doi.org/10.1073/pnas.70.12.3321" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+gene+diversity+in+subdivided+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+gene+diversity+in+subdivided+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0000734-Hedrick1" id="pone.0000734-Hedrick1"></a>Hedrick PW (1999) Highly variable loci and their interpretation in evolution and conservation. Evolution  53: 313–318.  <ul class="find" data-citedArticleID="977102" data-doi="10.2307/2640768"><li><a href="http://dx.doi.org/10.2307/2640768" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+variable+loci+and+their+interpretation+in+evolution+and+conservation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+variable+loci+and+their+interpretation+in+evolution+and+conservation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0000734-Hudson2" id="pone.0000734-Hudson2"></a>Hudson RR (2000) A new statistic for detecting genetic differentiation. Genetics  155: 2011–2014.  <ul class="find" data-citedArticleID="977106"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+new+statistic+for+detecting+genetic+differentiation.&amp;auth=&amp;atitle=A+new+statistic+for+detecting+genetic+differentiation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+statistic+for+detecting+genetic+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+statistic+for+detecting+genetic+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.XML" value="161866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.PDF" value="429120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g001.PNG_L" value="591046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g001.PNG_M" value="138152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g001.PNG_S" value="13093"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g001.TIF" value="1011336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g001.PNG_I" value="60885"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g002.PNG_L" value="255958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g002.PNG_M" value="118943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g002.PNG_S" value="8766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g002.TIF" value="416916"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g002.PNG_I" value="43776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g003.PNG_L" value="115303"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g003.PNG_M" value="130208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g003.PNG_S" value="13779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g003.TIF" value="493390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g003.PNG_I" value="93130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g004.PNG_L" value="29652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g004.PNG_M" value="38250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g004.PNG_S" value="7391"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g004.TIF" value="59496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.g004.PNG_I" value="13692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t001.PNG_L" value="203489"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t001.PNG_M" value="242145"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t001.PNG_S" value="22815"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t001.TIF" value="792250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t001.PNG_I" value="106189"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t002.PNG_L" value="121540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t002.PNG_M" value="71841"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t002.PNG_S" value="9675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t002.TIF" value="500424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t002.PNG_I" value="22341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t003.PNG_L" value="158783"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t003.PNG_M" value="176255"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t003.PNG_S" value="21369"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t003.TIF" value="725302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t003.PNG_I" value="87554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t004.PNG_L" value="153757"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t004.PNG_M" value="60822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t004.PNG_S" value="9430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t004.TIF" value="656956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.t004.PNG_I" value="18335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.e001.PNG" value="7377"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.e001.TIF" value="23408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.s001.XLS" value="91648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.s002.XLS" value="99840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.s003.TXT" value="156983"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.s004.EPS" value="51578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.s005.EPS" value="67039"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.s006.DOC" value="26112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.s007.DOC" value="40448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000734.s008.DOC" value="56832"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000734&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000734" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000734&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000734&volume=&issue=&title=MHC Adaptive Divergence between Closely Related and Sympatric African Cichlids&author_name=Jonatan%20Blais%2C%20Ciro%20Rico%2C%20Cock%20van%20Oosterhout%2C%20Joanne%20Cable%2C%20George%20F.%20Turner%2C%20Louis%20Bernatchez&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000734" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000734" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000734" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000734&amp;t=MHC%20Adaptive%20Divergence%20between%20Closely%20Related%20and%20Sympatric%20African%20Cichlids" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000734&title=MHC%20Adaptive%20Divergence%20between%20Closely%20Related%20and%20Sympatric%20African%20Cichlids&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000734&amp;title=MHC%20Adaptive%20Divergence%20between%20Closely%20Related%20and%20Sympatric%20African%20Cichlids" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000734" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000734&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'MHC Adaptive Divergence between Closely Related and Sympatric African Cichlids';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000734';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20MHC%20Adaptive%20Divergence%20between%20Closely%20Related%20and%20Sympatric%20African%20Cichlids http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000734" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000734" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="24306"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+recombination%22" title="Search for articles in the subject area:'DNA recombination'"><div class="flagText">DNA recombination</div></a>
              <div data-categoryid="33483" data-articleid="24306"
                   data-categoryname="DNA recombination"
                   class="flagImage" title="Flag 'DNA recombination' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24306"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Host-pathogen+interactions%22" title="Search for articles in the subject area:'Host-pathogen interactions'"><div class="flagText">Host-pathogen interactions</div></a>
              <div data-categoryid="19119" data-articleid="24306"
                   data-categoryname="Host-pathogen interactions"
                   class="flagImage" title="Flag 'Host-pathogen interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Introns%22" title="Search for articles in the subject area:'Introns'"><div class="flagText">Introns</div></a>
              <div data-categoryid="20277" data-articleid="24306"
                   data-categoryname="Introns"
                   class="flagImage" title="Flag 'Introns' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="24306"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Parasitic+diseases%22" title="Search for articles in the subject area:'Parasitic diseases'"><div class="flagText">Parasitic diseases</div></a>
              <div data-categoryid="2663" data-articleid="24306"
                   data-categoryname="Parasitic diseases"
                   class="flagImage" title="Flag 'Parasitic diseases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="24306"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1857'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5371'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7494&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=6615">host age</a><br>Posted by wormboy</p>
      <p><a href="/annotation/listThread.action?root=4475">MHC mate choice in inbreeding avoidance and speciation?</a><br>Posted by wormboy</p>
      <p><a href="/annotation/listThread.action?root=44461">A great system and thorough piece of work</a><br>Posted by wormboy</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
