

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Role of PSIP1/LEDGF/p75 in Lentiviral Infectivity and Integration Targeting</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001340"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001340" />

    <meta name="citation_title" content="Role of PSIP1/LEDGF/p75 in Lentiviral Infectivity and Integration Targeting"/>
    <meta itemprop="name" content="Role of PSIP1/LEDGF/p75 in Lentiviral Infectivity and Integration Targeting"/>

      <meta name="citation_author" content="Heather M. Marshall"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Keshet Ronen"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Charles Berry"/>
            <meta name="citation_author_institution" content="Department of Family, Preventive Medicine, San Diego School of Medicine, University of California at San Diego, San Diego, California, United States of America"/>
      <meta name="citation_author" content="Manuel Llano"/>
            <meta name="citation_author_institution" content="Molecular Medicine Program, Mayo Clinic College of Medicine, Rochester, Minnesota, United States of America"/>
      <meta name="citation_author" content="Heidi Sutherland"/>
            <meta name="citation_author_institution" content="Medical Research Council (MRC) Human Genetics Unit, Edinburgh, United Kingdom"/>
      <meta name="citation_author" content="Dyana Saenz"/>
            <meta name="citation_author_institution" content="Department of Family, Preventive Medicine, San Diego School of Medicine, University of California at San Diego, San Diego, California, United States of America"/>
      <meta name="citation_author" content="Wendy Bickmore"/>
            <meta name="citation_author_institution" content="Medical Research Council (MRC) Human Genetics Unit, Edinburgh, United Kingdom"/>
      <meta name="citation_author" content="Eric Poeschla"/>
            <meta name="citation_author_institution" content="Molecular Medicine Program, Mayo Clinic College of Medicine, Rochester, Minnesota, United States of America"/>
      <meta name="citation_author" content="Frederic D. Bushman"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, United States of America"/>

    <meta name="citation_date" content="2007/12/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001340.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1340"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_author=JM Coffin; citation_author=SH Hughes; citation_author=HE Varmus; citation_number=1; citation_date=1997; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_author=FD Bushman; citation_number=2; citation_date=2001; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=The site of HIV-1 integration in the human genome determines basal transcriptional activity and response to tat transactivation.; citation_author=A Jordan; citation_author=P Defechereux; citation_author=E Verdin; citation_journal_title=EMBO J; citation_volume=20; citation_number=3; citation_pages=1726-1738; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=HIV reporducibly establishes a latent infection after acute infection of T cells in vitro.; citation_author=A Jordan; citation_author=D Bisgrove; citation_author=E Verdin; citation_journal_title=EMBO J; citation_volume=22; citation_number=4; citation_pages=1868-1877; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of chromosomal features repressing HIV transcription.; citation_author=M Lewinski; citation_author=D Bisgrove; citation_author=P Shinn; citation_author=H Chen; citation_author=E Verdin; citation_journal_title=J Virol; citation_volume=79; citation_number=5; citation_pages=6610-6619; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular mechanisms of HIV-1 proviral latency.; citation_author=D Bisgrove; citation_author=M Lewinski; citation_author=FD Bushman; citation_author=E Verdin; citation_journal_title=Expert Rev Anti Infect Ther; citation_volume=3; citation_number=6; citation_pages=805-814; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A serious adverse event after successful gene therapy for X-linked severe combined immunodeficiency.; citation_author=S Hacein-Bey-Abina; citation_author=C von Kalle; citation_author=M Schmidt; citation_author=F Le Deist; citation_author=N Wulffraat; citation_journal_title=N Engl J Med; citation_volume=348; citation_number=7; citation_pages=255-256; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=LMO2-associated clonal T cell proliferation in two patients after gene therapy for SCID-X1.; citation_author=S Hacein-Bey-Abina; citation_author=C Von Kalle; citation_author=M Schmidt; citation_author=MP McCormack; citation_author=N Wulffraat; citation_journal_title=Science; citation_volume=302; citation_number=8; citation_pages=415-419; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=HIV-1 integrase forms stable tetramers and associates with LEDGF/p75 protein in human cells.; citation_author=P Cherepanov; citation_author=G Maertens; citation_author=P Proost; citation_author=B Devreese; citation_author=J Van Beeumen; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=9; citation_pages=372-381; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=LEDGF/p75 is essential for nuclear and chromosomal targeting of HIV-1 integrase in human cells.; citation_author=G Maertens; citation_author=P Cherepanov; citation_author=W Pluymers; citation_author=K Busschots; citation_author=E De Clercq; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=10; citation_pages=33528-33539; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Human cell proteins and human immunodeficiency virus DNA integration.; citation_author=F Turlure; citation_author=E Devroe; citation_author=PA Silver; citation_author=A Engelman; citation_journal_title=Front Biosci; citation_volume=9; citation_number=11; citation_pages=3187-3208; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=LEDGF/p75 determines cellular trafficking of diverse lentiviral but not murine oncoretroviral integrase proteins and is a component of functional lentiviral preintegration complexes.; citation_author=M Llano; citation_author=M Vanegas; citation_author=O Fregoso; citation_author=D Saenz; citation_author=S Chung; citation_journal_title=J Virol; citation_volume=78; citation_number=12; citation_pages=9524-9537; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Integrase mutants defective for interaction with LEDGF/p75 are impaired in chromosome tethering and HIV-1 replication.; citation_author=S Emiliani; citation_author=A Mousnier; citation_author=K Busschots; citation_author=M Maroun; citation_author=B Van Maele; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=13; citation_pages=25517-23; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of an evolutionarily-conserved domain of LEDGF/p75 that bind HIV-1 integrase.; citation_author=P Cherepanov; citation_author=E Devroe; citation_author=PA Silver; citation_author=A Engelman; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=14; citation_pages=48883-48892; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of the LEDGF/p75 HIV-1 integrase interaction domain and NLS reveals NLS-independent chromatin tethering.; citation_author=M Vanegas; citation_author=M Llano; citation_author=S Delgado; citation_author=D Thompson; citation_author=M Peretz; citation_journal_title=J Cell Sci; citation_volume=118; citation_number=15; citation_pages=1733-1743; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification and characterization of the chromatin-binding domains of the HIV-1 integrase interactor LEDGF/p75.; citation_author=M Llano; citation_author=M Vanegas; citation_author=N Hutchins; citation_author=D Thompson; citation_author=S Delgado; citation_journal_title=J Mol Biol; citation_volume=360; citation_number=16; citation_pages=760-773; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A tripartite DNA-binding element, comprised of the nuclear localization signal and two AT-hook motifs, mediates the association of LEDGF/p75 with chromatin in vivo.; citation_author=F Turlure; citation_author=G Maertens; citation_author=S Rahman; citation_author=P Cherepanov; citation_author=A Engelman; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=17; citation_pages=1653-1675; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=LEDGF/p75 prevents proteasomal degradation of HIV-1 integrase.; citation_author=M Llano; citation_author=S Delgado; citation_author=M Vanegas; citation_author=EM Poeschla; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=18; citation_pages=55570-55577; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The interaction of LEDGF/p75 with integrase is lentivirus-specific and promotes DNA binding.; citation_author=K Busschots; citation_author=J Vercammen; citation_author=S Emiliani; citation_author=R Benarous; citation_author=Y Engelborghs; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=19; citation_pages=17841-17847; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=LEDGF/p75 interacts with divergent lentiviral integrases and modulates their enzymatic activity in vitro.; citation_author=P Cherepanov; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=20; citation_pages=113-124; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=HIV-1 integration in the human genome favors active genes and local hotspots.; citation_author=AR Schroder; citation_author=P Shinn; citation_author=H Chen; citation_author=C Berry; citation_author=JR Ecker; citation_journal_title=Cell; citation_volume=110; citation_number=21; citation_pages=521-529; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Retroviral DNA integration: ASLV, HIV, and MLV show distinct target site preferences.; citation_author=RS Mitchell; citation_author=BF Beitzel; citation_author=AR Schroder; citation_author=P Shinn; citation_author=H Chen; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=22; citation_pages=e234; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Distinct genomic integration of MLV and SIV vectors in primate hematopoietic stem and progenitor cells.; citation_author=P Hematti; citation_author=BK Hong; citation_author=C Ferguson; citation_author=R Adler; citation_author=H Hanawa; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=23; citation_pages=e423; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Fusions of LEDGF/p75 to lambda repressor promote HIV DNA integration near lambda operators in vitro.; citation_author=A Ciuffi; citation_author=T Diamond; citation_author=Y Hwang; citation_author=H Marshall; citation_author=FD Bushman; citation_journal_title=Human Gene Therapy; citation_volume=17; citation_number=24; citation_pages=960-967; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HIV integration site selection: Targeting in macrophages and the effects of different routes of viral entry.; citation_author=SD Barr; citation_author=A Ciuffi; citation_author=J Leipzig; citation_author=P Shinn; citation_author=JR Ecker; citation_journal_title=Mol Ther; citation_volume=14; citation_number=25; citation_pages=218-225; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Integration targeting by avian sarcoma-leukosis virus and human immunodeficiency virus in the chicken genome.; citation_author=SD Barr; citation_author=J Leipzig; citation_author=P Shinn; citation_author=JR Ecker; citation_author=FD Bushman; citation_journal_title=J Virol; citation_volume=79; citation_number=26; citation_pages=12035-12044; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Simian immunodeficiency virus integration preference is similar to that of human immunodeficiency virus type 1.; citation_author=B Crise; citation_author=Y Li; citation_author=C Yuan; citation_author=DR Morcock; citation_author=D Whitby; citation_journal_title=J Virol; citation_volume=79; citation_number=27; citation_pages=12199-121204; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Retroviral DNA integration: Viral and cellular determinants of target-site selection.; citation_author=MK Lewinski; citation_author=M Yamashita; citation_author=M Emerman; citation_author=A Ciuffi; citation_author=H Marshall; citation_journal_title=PLoS Pathog; citation_volume=2; citation_number=28; citation_pages=e60; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of retroviral DNA integration.; citation_author=F Bushman; citation_author=M Lewinski; citation_author=A Ciuffi; citation_author=S Barr; citation_author=J Leipzig; citation_journal_title=Nat Rev Microbiol; citation_volume=3; citation_number=29; citation_pages=848-858; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Integration site choice of a feline immunodeficiency virus vector.; citation_author=Y Kang; citation_author=CJ Moressi; citation_author=TE Scheetz; citation_author=L Xie; citation_author=DT Tran; citation_journal_title=J Virol; citation_volume=80; citation_number=30; citation_pages=8820-8823; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The integration profile of EIAV-based vectors.; citation_author=CV Hacker; citation_author=CA Vink; citation_author=TW Wardell; citation_author=S Lee; citation_author=P Treasure; citation_journal_title=Mol Ther; citation_volume=14; citation_number=31; citation_pages=536-545; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genomic sites of human immunodeficiency virus type 2 (HIV-2) integration: Similarities to HIV-1 in vitro and possible differences in vivo.; citation_author=A MacNeil; citation_author=JL Sankale; citation_author=ST Meloni; citation_author=AD Sarr; citation_author=S Mboup; citation_journal_title=J Virol; citation_volume=80; citation_number=32; citation_pages=7316-7321; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structural basis for the recognition between HIV-1 integrase and transcriptional coactivator p75.; citation_author=P Cherepanov; citation_author=AL Ambrosio; citation_author=S Rahman; citation_author=T Ellenberger; citation_author=A Engelman; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=33; citation_pages=17308-17313; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Solution structure of the HIV integrase-binding domain in LEDGF/p75.; citation_author=P Cherepanov; citation_author=ZY Sun; citation_author=S Rahman; citation_author=G Maertens; citation_author=G Wagner; citation_journal_title=Nat Struct Mol Biol; citation_volume=12; citation_number=34; citation_pages=526-532; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Modest but reproducible inhibition of human immunodeficiency virus type 1 infection in macrophages following LEDGFp75 silencing.; citation_author=SP Zielske; citation_author=M Stevenson; citation_journal_title=J Virol; citation_volume=80; citation_number=35; citation_pages=7275-7280; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biochemical and genetic analyses of integrase-interacting protein lens epithelium-derived growth factor (LEDGF)/p75 and hepatoma-derived growth factor related protein 2 (HRP2) in preintegration complex function and HIV-1 replication.; citation_author=N Vandegraaff; citation_author=E Devroe; citation_author=F Turlure; citation_author=PA Silver; citation_author=A Engelman; citation_journal_title=Virology; citation_volume=346; citation_number=36; citation_pages=415-426; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An Essential Role for LEDGF/p75 in HIV Integration.; citation_author=M Llano; citation_author=DT Saenz; citation_author=A Meehan; citation_author=P Wongthida; citation_author=M Peretz; citation_author=WH Walker; citation_author=W Teo; citation_author=EM Poeschla; citation_journal_title=Science; citation_volume=314; citation_number=37; citation_pages=461-464; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transient and stable knockdown of the integrase cofactor LEDGF/p75 reveals its role in the replication cycle of human immunodeficiency virus.; citation_author=L Vandekerckhove; citation_author=F Christ; citation_author=B Van Maele; citation_author=J De Rijck; citation_author=R Gijsbers; citation_journal_title=J Virol; citation_volume=80; citation_number=38; citation_pages=1886-1896; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Overexpression of the lens epithelium-derived growth factor/p75 integrase binding domain inhibits human immunodeficiency virus replication.; citation_author=J De Rijck; citation_author=L Vandekerckhove; citation_author=R Gijsbers; citation_author=A Hombrouck; citation_author=J Hendrix; citation_journal_title=J Virol; citation_volume=80; citation_number=39; citation_pages=11498-11509; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A role for LEDGF/p75 in targeting HIV DNA integration.; citation_author=A Ciuffi; citation_author=M Llano; citation_author=E Poeschla; citation_author=C Hoffmann; citation_author=J Leipzig; citation_journal_title=Nat Med; citation_volume=11; citation_number=40; citation_pages=1287-1289; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=LEDGF/p75 functions downstream from preintegration complex formation to effect gene-specific HIV-1 integration.; citation_author=MC Shun; citation_author=NK Raghavendra; citation_author=N Vandegraaff; citation_author=JE Daigle; citation_author=S Hughes; citation_journal_title=Genes Dev; citation_volume=21; citation_number=41; citation_pages=1767-1778; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Disruption of Ledgf/Psip1 results in perinatal mortality and homeotic skeletal transformations.; citation_author=HG Sutherland; citation_author=K Newton; citation_author=DG Brownstein; citation_author=MC Holmes; citation_author=C Kress; citation_journal_title=Mol Cell Biol; citation_volume=26; citation_number=42; citation_pages=7201-7210; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HIV cDNA metabolism studied by fluorescence-monitored PCR: Notable stability of two-LTR circles.; citation_author=SL Butler; citation_author=EP Johnson; citation_author=FD Bushman; citation_journal_title=J Virol; citation_volume=76; citation_number=43; citation_pages=3739-3747; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A quantitative assay for HIV cDNA integration in vivo.; citation_author=S Butler; citation_author=M Hansen; citation_author=FD Bushman; citation_journal_title=Nat Med; citation_volume=7; citation_number=44; citation_pages=631-634; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors.; citation_author=M Margulies; citation_author=M Egholm; citation_author=WE Altman; citation_author=S Attiya; citation_author=JS Bader; citation_journal_title=Nature; citation_volume=437; citation_number=45; citation_pages=376-380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=HIV integration site selection: Analysis by massively parallel pyrosequencing reveals association with epigenetic modifications.; citation_author=GP Wang; citation_author=A Ciuffi; citation_author=J Leipzig; citation_author=CC Berry; citation_author=FD Bushman; citation_journal_title=Genome Res; citation_volume=17; citation_number=46; citation_pages=1186-1194; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Quantitative phenotypic analysis of yeast deletion mutants using a highly parallel molecular bar-coding strategy.; citation_author=DD Shoemaker; citation_author=DA Lashkari; citation_author=D Morris; citation_author=M Mittmann; citation_author=RW Davis; citation_journal_title=Nat Genet; citation_volume=14; citation_number=47; citation_pages=450-456; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=DNA bar coding and pyrosequencing to identify rare HIV drug resistance mutations.; citation_author=C Hoffmann; citation_author=N Minkah; citation_author=J Leipzig; citation_author=G Wang; citation_author=MQ Arens; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=48; citation_pages=e91; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The use of coded PCR primers enables high-throughput sequencing of multiple homolog amplification products by 454 parallel sequencing.; citation_author=J Binladen; citation_author=MT Gilbert; citation_author=JP Bollback; citation_author=F Panitz; citation_author=C Bendixen; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=49; citation_pages=e197; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Retroviral integration into minichromosomes in vitro.; citation_author=PM Pryciak; citation_author=A Sil; citation_author=HE Varmus; citation_journal_title=EMBO J; citation_volume=11; citation_number=50; citation_pages=291-303; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Sequence analysis of the human DNA flanking sites of human immunodeficiency virus type 1 integration.; citation_author=SW Stevens; citation_author=JD Griffith; citation_journal_title=J Virol; citation_volume=70; citation_number=51; citation_pages=6459-6462; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Chromosome structure and HIV-1 cDNA integration: Centromeric alphoid repeats are a disfavored target.; citation_author=S Carteau; citation_author=C Hoffmann; citation_author=FD Bushman; citation_journal_title=J Virol; citation_volume=72; citation_number=52; citation_pages=4005-4014; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Weak palindromic consensus sequences are a common feature found at the integration target sites of many retroviruses.; citation_author=X Wu; citation_author=Y Li; citation_author=B Crise; citation_author=SM Burgess; citation_author=DJ Munroe; citation_journal_title=J Virol; citation_volume=79; citation_number=53; citation_pages=5211-5214; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Symmetrical base preferences surrounding HIV-1, avian sarcoma/leukosis virus, and murine leukemia virus integration sites.; citation_author=AG Holman; citation_author=JM Coffin; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=54; citation_pages=6103-6107; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Selection of target sites for mobile DNA integration in the human genome.; citation_author=C Berry; citation_author=S Hannenhalli; citation_author=J Leipzig; citation_author=FD Bushman; citation_journal_title=PLoS Comput Biol; citation_volume=2; citation_number=55; citation_pages=e157; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Modulating target site selection during human immunodeficiency virus DNA integration in vitro with an engineered tethering factor.; citation_author=A Ciuffi; citation_author=TL Diamond; citation_author=Y Hwang; citation_author=HM Marshall; citation_author=FD Bushman; citation_journal_title=Hum Gene Ther; citation_volume=17; citation_number=56; citation_pages=960-967; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcription start regions in the human genome are favored targets for MLV integration.; citation_author=X Wu; citation_author=Y Li; citation_author=B Crise; citation_author=SM Burgess; citation_journal_title=Science; citation_volume=300; citation_number=57; citation_pages=1749-1751; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Large-scale analysis of adeno-associated virus vector integration sites in normal human cells.; citation_author=DG Miller; citation_author=GD Trobridge; citation_author=LM Petek; citation_author=MA Jacobs; citation_author=R Kaul; citation_journal_title=J Virol; citation_volume=79; citation_number=58; citation_pages=11434-11442; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of a small molecule binding site at the dimer interface of the HIV integrase catalytic domain.; citation_author=V Molteni; citation_author=J Greenwald; citation_author=D Rhodes; citation_author=Y Hwang; citation_author=W Kwiatkowski; citation_journal_title=Acta Crystallogr D Biol Crystallogr; citation_volume=57; citation_number=59; citation_pages=536-544; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cross-validatory chosice and assesment of statistical predictions.; citation_author=M Stone; citation_journal_title=Journal of the Royal Statistical Society, Series B–Methodological; citation_volume=36; citation_number=60; citation_pages=111-147; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Quantitative studies of the growth of mouse embryo cells in culture and their development into established lines.; citation_author=G Todaro; citation_author=H Green; citation_journal_title=J Cell Biol; citation_volume=17; citation_number=61; citation_pages=299-313; citation_date=1963; " />
      <meta name="citation_reference" content="citation_title=Gene transfer by lentiviral vectors is limited by nuclear translocation and rescued by HIV-1 pol sequences.; citation_author=A Follenzi; citation_author=LE Ailes; citation_author=S Bakovic; citation_author=M Gueuna; citation_author=L Naldini; citation_journal_title=Nat Genetics; citation_volume=25; citation_number=62; citation_pages=217-222; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=In vivo gene delivery and stable transduction of nondividing cells by a lentiviral vector.; citation_author=L Naldini; citation_author=U Blomer; citation_author=P Gallay; citation_author=D Ory; citation_author=R Mulligan; citation_journal_title=Science; citation_volume=272; citation_number=63; citation_pages=263-267; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Gene transfer vectors derived from equine infectious anemia virus.; citation_author=JC Olsen; citation_journal_title=Gene Ther; citation_volume=5; citation_number=64; citation_pages=1481-1487; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Capsid is a dominant determinant of retrovirus infectivity in nondividing cells.; citation_author=M Yamashita; citation_author=M Emerman; citation_journal_title=J Virol; citation_volume=78; citation_number=65; citation_pages=5670-5678; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Integration site selection by HIV-based vectors in dividing and growth-arrested IMR-90 lung fibroblasts.; citation_author=A Ciuffi; citation_author=RS Mitchell; citation_author=C Hoffmann; citation_author=J Leipzig; citation_author=P Shinn; citation_journal_title=Mol Ther; citation_volume=13; citation_number=66; citation_pages=366-373; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gene transfer in humans using a conditionally replicating lentiviral vector.; citation_author=BL Levine; citation_author=LM Humeau; citation_author=J Boyer; citation_author=RR MacGregor; citation_author=T Rebello; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=67; citation_pages=17372-17377; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001340" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Role of PSIP1/LEDGF/p75 in Lentiviral Infectivity and Integration Targeting"/>
    <meta name="twitter:description" content="BackgroundTo replicate, lentiviruses such as HIV must integrate DNA copies of their RNA genomes into host cell chromosomes. Lentiviral integration is favored in active transcription units, which allows efficient viral gene expression after integration, but the mechanisms directing integration targeting are incompletely understood. A cellular protein, PSIP1/LEDGF/p75, binds tightly to the lentiviral-encoded integrase protein (IN), and has been reported to be important for HIV infectivity and integration targeting.MethodologyHere we report studies of lentiviral integration targeting in 1) human cells with intensified RNAi knockdowns of PSIP1/LEDGF/p75, and 2) murine cells with homozygous gene trap mutations in the PSIP1/LEDGF/p75 locus. Infections with vectors derived from equine infections anemia virus (EIAV) and HIV were compared. Integration acceptor sites were analyzed by DNA bar coding and pyrosequencing.Conclusions/SignificanceIn both PSIP1/LEDGF/p75-depleted cell lines, reductions were seen in lentiviral infectivity compared to controls. For the human cells, integration was reduced in transcription units in the knockdowns, and this reduction was greater than in our previous studies of human cells less completely depleted for PSIP1/LEDGF/p75. For the homozygous mutant mouse cells, similar reductions in integration in transcription units were seen, paralleling a previous study of a different mutant mouse line. Integration did not become random, however&ndash;integration in transcription units in both cell types was still favored, though to a reduced degree. New trends also appeared, including favored integration near CpG islands. In addition, we carried out a bioinformatic study of 15 HIV integration site data sets in different cell types, which showed that the frequency of integration in transcription units was correlated with the cell-type specific levels of PSIP1/LEDGF/p75 expression."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001340.g009"/>

  <meta property="og:title" content="Role of PSIP1/LEDGF/p75 in Lentiviral Infectivity and Integration Targeting" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001340" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5211'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8501'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6416&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001340">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001340" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001340&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Role of PSIP1/LEDGF/p75 in Lentiviral Infectivity and Integration Targeting
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Heather M. Marshall
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Heather M. Marshall, Keshet Ronen</p>

              
              

                <p>Affiliation:
                  Department of Microbiology, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Keshet Ronen
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Heather M. Marshall, Keshet Ronen</p>

              
              

                <p>Affiliation:
                  Department of Microbiology, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Charles Berry, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Family, Preventive Medicine, San Diego School of Medicine, University of California at San Diego, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Manuel Llano, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Molecular Medicine Program, Mayo Clinic College of Medicine, Rochester, Minnesota, United States of America
                </p>

                <p>
                    Current address: Department of Biological Sciences, University of Texas at El Paso, El Paso, Texas, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Heidi Sutherland, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Medical Research Council (MRC) Human Genetics Unit, Edinburgh, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dyana Saenz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Family, Preventive Medicine, San Diego School of Medicine, University of California at San Diego, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wendy Bickmore, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Medical Research Council (MRC) Human Genetics Unit, Edinburgh, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eric Poeschla, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Molecular Medicine Program, Mayo Clinic College of Medicine, Rochester, Minnesota, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frederic D. Bushman
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:bushman@mail.med.upenn.edu">bushman@mail.med.upenn.edu</a></p>

                <p>Affiliation:
                  Department of Microbiology, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001340</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001340" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001340" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001340" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001340" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001340-g001" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001340-g002" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001340-t001" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001340-g003" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001340-g004" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001340-t002" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001340-g005" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001340-g006" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001340-g007" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001340-t003" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001340-g008" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001340-g009" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001340">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001340" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>To replicate, lentiviruses such as HIV must integrate DNA copies of their RNA genomes into host cell chromosomes. Lentiviral integration is favored in active transcription units, which allows efficient viral gene expression after integration, but the mechanisms directing integration targeting are incompletely understood. A cellular protein, PSIP1/LEDGF/p75, binds tightly to the lentiviral-encoded integrase protein (IN), and has been reported to be important for HIV infectivity and integration targeting.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here we report studies of lentiviral integration targeting in 1) human cells with intensified RNAi knockdowns of PSIP1/LEDGF/p75, and 2) murine cells with homozygous gene trap mutations in the <em>PSIP1/LEDGF/p75</em> locus. Infections with vectors derived from equine infections anemia virus (EIAV) and HIV were compared. Integration acceptor sites were analyzed by DNA bar coding and pyrosequencing.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>In both PSIP1/LEDGF/p75-depleted cell lines, reductions were seen in lentiviral infectivity compared to controls. For the human cells, integration was reduced in transcription units in the knockdowns, and this reduction was greater than in our previous studies of human cells less completely depleted for PSIP1/LEDGF/p75. For the homozygous mutant mouse cells, similar reductions in integration in transcription units were seen, paralleling a previous study of a different mutant mouse line. Integration did not become random, however–integration in transcription units in both cell types was still favored, though to a reduced degree. New trends also appeared, including favored integration near CpG islands. In addition, we carried out a bioinformatic study of 15 HIV integration site data sets in different cell types, which showed that the frequency of integration in transcription units was correlated with the cell-type specific levels of <em>PSIP1/LEDGF/p75</em> expression.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Marshall HM, Ronen K, Berry C, Llano M, Sutherland H, et al.  (2007) Role of PSIP1/LEDGF/p75 in Lentiviral Infectivity and Integration Targeting. PLoS ONE 2(12):
          e1340.
            doi:10.1371/journal.pone.0001340</p><p><strong>Academic Editor: </strong>Debbie Fox, The Research Institute for Children, United States of America</p><p><strong>Received:</strong> September 17, 2007; <strong>Accepted:</strong> November 26, 2007; <strong>Published:</strong> December 19, 2007</p><p><strong>Copyright:</strong> © 2007 Marshall et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by NIH grants AI52845, AI66290, AI47536, the Tietze Foundation, the James B. Pendleton Charitable Trust, The Penn Genomics Institute, and Robin and Frederic Withington. Keshet Ronen was supported by T32 07229-33. Heather Marshall was supported by AI007632.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Early steps of retroviral replication involve reverse transcription to generate a DNA copy of the viral RNA genome, and integration, which results in the covalent connection of the viral DNA to host cell DNA (for reviews see <a href="#pone.0001340-Coffin1">[1]</a>, <a href="#pone.0001340-Bushman1">[2]</a>). The question of where retroviruses target DNA integration is central to understanding viral host interactions. For the virus, selection of favorable sites for viral DNA integration assists efficient expression of the viral genome after integration <a href="#pone.0001340-Jordan1">[3]</a>–<a href="#pone.0001340-Bisgrove1">[6]</a>. For the host, viral DNA integration can either activate or inactivate gene transcription. One consequence of integration can be insertional activation of oncogenes and transformation to malignant growth <a href="#pone.0001340-Coffin1">[1]</a>, <a href="#pone.0001340-Bushman1">[2]</a>, <a href="#pone.0001340-HaceinBeyAbina1">[7]</a>, <a href="#pone.0001340-HaceinBeyAbina2">[8]</a>. Here we present data on the role of a host-cell encoded protein, PSIP1/LEDGF/p75, that guides integration site selection by lentiviruses, the viral genus including HIV (henceforth we use “LEDGF/p75” because this name is widely used in the HIV field).</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>LEDGF/p75 first came to the attention of the retrovirus field when it was identified in affinity-based screens for its tight binding to HIV IN <a href="#pone.0001340-Cherepanov1">[9]</a>–<a href="#pone.0001340-Turlure1">[11]</a>. LEDGF/p75 tethers ectopically-expressed HIV IN to chromatin <a href="#pone.0001340-Cherepanov1">[9]</a>, <a href="#pone.0001340-Maertens1">[10]</a>, <a href="#pone.0001340-Llano1">[12]</a>, <a href="#pone.0001340-Emiliani1">[13]</a>, through specific binding domains <a href="#pone.0001340-Cherepanov2">[14]</a>–<a href="#pone.0001340-Turlure2">[17]</a>, and also protects IN from proteasomal degradation <a href="#pone.0001340-Llano3">[18]</a>. LEDGF/p75 binding is specific for lentiviral IN proteins (e. g. those of HIV, SIV, FIV, and EIAV)<a href="#pone.0001340-Llano1">[12]</a>, <a href="#pone.0001340-Busschots1">[19]</a>, <a href="#pone.0001340-Cherepanov3">[20]</a>, which makes it appealing as a candidate tethering factor since all the lentiviruses tested (HIV, SIV, FIV, and EIAV) show favored integration in active transcription units <a href="#pone.0001340-Lewinski1">[5]</a>, <a href="#pone.0001340-Schroder1">[21]</a>–<a href="#pone.0001340-MacNeil1">[32]</a>. The crystal structure of the catalytic domain of HIV IN (residues 50–212) bound to the integrase binding domain (IBD) was solved, which showed that a pair of LEDGF/p75-IBD molecules could bind at symmetry-related positions at the interface of the IN catalytic domain dimer <a href="#pone.0001340-Cherepanov4">[33]</a>, <a href="#pone.0001340-Cherepanov5">[34]</a></p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Early attempts to determine whether LEDGF/p75 was important for efficient HIV replication used RNAi knockdowns in human cells, which had either no effect or quantitatively modest effects on infection <a href="#pone.0001340-Llano1">[12]</a>, <a href="#pone.0001340-Emiliani1">[13]</a>, <a href="#pone.0001340-Zielske1">[35]</a>, <a href="#pone.0001340-Vandegraaff1">[36]</a>. This now appears to be because incomplete knockdowns left biologically significant amounts of protein present. More recently, human SupT1 cells with intensified RNAi knockdowns showed drops of 30-fold for infection by either HIV or another lentivirus, feline immunodeficiency virus (FIV), and combining this with dominant interfering proteins derived from the LEDGF/p75-IBD produced 560-fold inhibition of infection <a href="#pone.0001340-Llano4">[37]</a>. These findings are supported by additional studies in human cell lines <a href="#pone.0001340-Zielske1">[35]</a>, <a href="#pone.0001340-Vandekerckhove1">[38]</a>, <a href="#pone.0001340-DeRijck1">[39]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Early knockdowns of LEDGF/p75 were also analyzed for effects on targeting of HIV integration <a href="#pone.0001340-Ciuffi2">[40]</a>. Knockdowns in three cell types were studied, and in each integration frequency within transcription units was reduced. In addition, other effects were seen, including an increase in the content of G/C bases around sites of HIV integration in the knockdown cells. These data supported the idea that LEDGF/p75 acted as a tethering factor, binding to both HIV and chromatin to direct HIV integration into active genes. In support of the tethering model, artificial fusion proteins in which the LEDGF/p75 IBD was fused to the sequence specific DNA binding domain of phage lambda repressor were shown to direct favored integration <em>in vitro</em> near repressor binding sites <a href="#pone.0001340-Ciuffi1">[24]</a>. Also supporting the tethering idea, function of LEDGF/p75 in promoting HIV replication requires that both ends of the putative LEDGF/p75 tether be intact <a href="#pone.0001340-Llano4">[37]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>However, key questions still remain on the role of LEDGF/p75. In all the models studied, HIV continued to favor integration within active transcription units. This could either be because residual LEDGF/p75 remaining in the knockdown was sufficient for residual targeting activity, or because additional host cell factors also contribute independently to targeting HIV integration. In an effort to address this issue, Shun et al. prepared a mouse strain in which part of the <em>LEDGF/p75</em> locus was flanked by Cre recombination sites <a href="#pone.0001340-Shun1">[41]</a>, and the <em>LEDGF/p75</em> exon was deleted by exposure to Cre recombinase. Mouse embryonic fibroblasts were then studied for effects on infection with HIV reporter viruses. These cells showed a 20-fold reduction in infectivity by HIV, and also a reduction in integration frequency in transcription units that was stronger than that reported in human cell knockdowns by Ciuffi et al. <a href="#pone.0001340-Ciuffi2">[40]</a>. However, HIV did still infect at a reduced rate, and integration in transcription units was still significantly favored. The mouse cells also showed some new targeting features in the LEDGF/p75-depleted cells, including increased integration near CpG islands.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>These studies were helpful in clarifying the effects of strong LEDGF/p75 depletion, but several issues remain. We wished to obtain lentiviral integration targeting data for human cells with stronger knockdowns of LEDGF/p75 to investigate possible effects of the host cell species. We also wished to obtained data from an additional murine cell line depleted for LEDGF/p75 to check the generality of conclusions from Shun et al. <a href="#pone.0001340-Shun1">[41]</a>. We thus studied the human SupT1 T-cell line with intensified RNAi developed by Llano et al. <a href="#pone.0001340-Llano4">[37]</a>, and mouse cells containing homozygous gene trap mutations at the <em>LEDGF/p75</em> locus developed by Sutherland and coworkers <a href="#pone.0001340-Sutherland1">[42]</a>. Vectors derived from equine infectious anemia virus (EIAV) were used in many of the experiments, allowing effects on HIV and EIAV to be compared. Studies of both cell models and both lentiviruses provided strong evidence for the role of LEDGF/p75 in promoting efficient infection and targeting integration in transcription units. In addition to these data on manipulated cell models, we also present additional bioinformatic studies of 15 published HIV integration site data sets in different cell types, which revealed a strong correlation between cell type specific <em>LEDGF/p75</em> expression levels and the proportion of HIV integration sites in transcription units. These data provide further support for the generality of LEDGF/p75 as a determinant of integration target site selection for lentiviruses in primary cells where LEDGF/p75 levels were not artificially reduced.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Efficiency of lentivirus infection in human SupT1 cells with intensified knockdown of LEDGF/p75</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Initially cells depleted for LEDGF/p75 were tested for effects on lentiviral infection. For the human SupT1 cells with the intensified LEDGF/p75 knockdown (the TC2 and TL2 cell lines in <a href="#pone.0001340-Llano4">[37]</a>), there were technical complications in studying HIV integration targeting. To generate the cells, shRNAs were introduced using HIV-based vectors. Thus the modified cells already contain integrated HIV sequences, which would complicate sequence analysis of newly integrated HIV proviruses. For that reason, we studied the lentivirus equine infectious anemia virus (EIAV). Like HIV IN, EIAV IN is known to bind LEDGF/p75 <a href="#pone.0001340-Cherepanov3">[20]</a>, and EIAV is also known to integrate in active transcription units <a href="#pone.0001340-Hacker1">[31]</a>, so EIAV is a suitable model for analysis of the influence of LEDGF/p75 on lentivirus infection.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p><a href="#pone-0001340-g001">Figure 1</a> shows the efficiency of infection by HIV and EIAV in the modified SupT1 cells. HIV infection efficiency was characterized two days after infection using a luciferase-transducing HIV vector. Activity was compared for unmodified SupT1 cells or control cells containing a scrambled shRNA sequence (SCRAM). Luciferase activity was reduced ~10 fold in the LEDGF/p75 knockdown but not in the control cells, and similar effects were seen at two multiplicities of infection (<a href="#pone-0001340-g001">Figure 1A and B</a>), paralleling previously published data from Llano et al. <a href="#pone.0001340-Llano4">[37]</a>.</p>
<div class="figure" id="pone-0001340-g001"><div class="img"><a name="pone-0001340-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Effects of intensified knockdown of LEDGF/p75 in SupT1 cells on the efficiency of lentiviral infection.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A) and B) HIV luc activity was compared for wild-type SupT1 cells, SupT1 containing a control scrambled shRNA (SCRAM), and LEDGF/p75 knockdown (KD) cells. A) High multiplicity of infection (80 ng p24); B) Lower multiplicity of infection (20 ng p24). The designation “p24” indicates the amount of viral stock, measured by the weight of the p24 capsid antigen applied to cells. C) and D) EIAV infectivity was compared in the SupT1 cell set as assayed by quantitative PCR for viral cDNA: C) high multiplicity (100 µl stock), D) lower multiplicity (25 µl stock).</p>
<span>doi:10.1371/journal.pone.0001340.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>An EIAV vector was also tested (<a href="#pone-0001340-g001">Figure 1C and D</a>). Infection through the step of integration was monitored by infecting cells, then growing the cells for two weeks, so that only covalently integrated DNA persisted (unintegrated DNA is degraded or lost by dilution during prolonged cell growth <a href="#pone.0001340-Butler1">[43]</a>, <a href="#pone.0001340-Butler2">[44]</a>). EIAV DNA was then quantified in genomic DNA samples using quantitative PCR. The LEDGF/p75 knockdown cells showed only between 8 and 24% of the amount of viral DNA seen in the control cells, indicating that for EIAV as well LEDGF/p75 is important for completing the early steps of replication.</p>


<h4>Efficiency of lentivirus infection in murine cells disrupted at <em>LEDGF/p75</em></h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We also compared lentiviral infection in murine cells containing the gene trap disruption of <em>LEDGF/p75</em> reported by Sutherland and colleagues <a href="#pone.0001340-Sutherland1">[42]</a>. Because residual expression is sometimes detected in gene trap alleles, we used quantitative RT-PCR to determine the fraction of <em>LEDGF/p75</em> messages disrupted by the gene trap insertion. In samples from homozygous mutant (−/−) cells, amplification of correct <em>LEDGF/p75</em> message was sporadically detected at high PCR cycle numbers, suggesting that rare correctly spliced messages were formed. However, quantification of correct message formation using SyberGreen quantitative PCR showed expression of <em>LEDGF/p75</em> to be below the limit of detection in the −/− cells, corresponding to a reduction of at least 32-fold compared to the wild type (+/+) cells (unpublished data). Sutherland and coworkers reported LEDGF/p75 protein to be undetectable <a href="#pone.0001340-Sutherland1">[42]</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We analyzed infection of murine embryonic fibroblasts (MEFs) isolated from embryos of +/+ and homozygous mutant −/− mice after infection with HIV and EIAV. Integration was measured by infecting cells, maintaining the cells in culture for two weeks to allow loss of unintegrated DNA <a href="#pone.0001340-Butler2">[44]</a>, then quantifying the viral DNA by TaqMan PCR. HIV integration was reduced ~five fold in the <em>LEDGF/p75</em> −/− MEFs (<a href="#pone-0001340-g002">Figure 2A</a>), and EIAV integration was reduced &gt;50 fold. Thus in the presence of a homozygous mutation of <em>LEDGF/p75</em>, lentiviral integration was strongly reduced but not eliminated.</p>
<div class="figure" id="pone-0001340-g002"><div class="img"><a name="pone-0001340-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Efficiency of lentiviral infection in control (+/+) and homozygous <em>LEDGF/p75</em>-disrupted (−/−) murine cells, measured by quantitative PCR.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A) HIV infectivity. B) EIAV infectivity.</p>
<span>doi:10.1371/journal.pone.0001340.g002</span></div>

<h4>DNA bar coding and pyrosequencing to analyze integration site placement</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Below we first describe studies of EIAV integration targeting in the SupT1 cells with intensified RNAi knockdowns, then HIV and EIAV targeting in the mouse cells disrupted at <em>LEDGF/p75</em>.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>For each of our studies, we used the pyrosequencing technology commercialized by 454 Life Sciences <a href="#pone.0001340-Margulies1">[45]</a> to sequence genomic DNA flanking integrated proviruses. Briefly, genomic DNA was isolated and cleaved with restriction enzymes. DNA linkers were ligated onto the cleaved ends, then host-virus DNA junctions were amplified using one primer complementary to the linker and one complementary to the viral DNA end. A second round of PCR was used to improve specificity and to add recognition sites for the 454 primers necessary for the emulsion PCR step preceding pyrosequencing <a href="#pone.0001340-Wang1">[46]</a>. Pooled DNAs were then subjected to pyrosequencing.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Use of DNA bar coding allowed multiple integration site populations to be studied in parallel <a href="#pone.0001340-Shoemaker1">[47]</a>–<a href="#pone.0001340-Binladen1">[49]</a>. The viral DNA primer used in the second round of amplification contained a short recognition sequence (4–8 bases) abutting the 454 primer that was different for each sample tested. These 4–8 bases are the first determined in pyrosequencing reads. Thus use of bar coding allowed many samples to be pooled for sequence determination, then the reads could be sorted into individual experiments by bar code. A total of 3566 unique integration site sequences from different virus and cell combinations were determined using this method (<a href="#pone-0001340-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001340-t001"><div class="img"><a name="pone-0001340-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001340.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001340.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Integration site data sets used in this study.</span></strong></p><span>doi:10.1371/journal.pone.0001340.t001</span></div>

<h4>Consensus sequences at EIAV integration sites in human SupT1 cells</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The EIAV vector was used to infect SupT1 cells with intensified RNAi against LEDGF/p75 and compared to controls consisting of either SupT1 cells with a scrambled shRNA (SCRAM) or untreated SupT1 cells. Integration sites were sequenced and placed on the hg18 draft human genome sequence. As a first step in the analysis, the favored target DNA sequences at the point of integration were compared in the presence and absence of LEDGF/p75. Alignment of target DNA sequences at integration sites has revealed weak inverted repeat consensus sequences <a href="#pone.0001340-Pryciak1">[50]</a>–<a href="#pone.0001340-Berry1">[55]</a>, the symmetry arising because the favored sequence features at each of the two viral DNA ends are the same. The presence of this consensus sequence can be a strong predictor of integration targeting specificity, particularly over short intervals <a href="#pone.0001340-Berry1">[55]</a>. For HIV, the favored consensus sequence has been synthesized and shown to be a favored integration target site for HIV preintegration complexes <em>in vitro</em> <a href="#pone.0001340-Carteau1">[52]</a>.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>EIAV has been reported to favor integration in an A/T rich palindromic consensus sequence <a href="#pone.0001340-Hacker1">[31]</a>, which matched that seen here for EIAV integration in the control SupT1 and SupT1 SCRAM cell lines (<a href="#pone-0001340-g003">Figure 3A and B</a>). The LEDGF/p75 knockdown cells showed an indistinguishable consensus sequence (<a href="#pone-0001340-g003">Figure 3C</a>), providing evidence against the view that LEDGF/p75 is involved in specifying the target sequence preference.</p>
<div class="figure" id="pone-0001340-g003"><div class="img"><a name="pone-0001340-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Integration site consensus at sequences flanking EIAV proviruses in control and LEDGF/p75-knockdown SupT1 cells.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>A) Unmodified SupT1 cells. B) Control SCRAM cells. C) LEDGF KD cells. The diagrams were generated using the WebLOGO program (weblogo.berkeley.edu/logo.cgi). The y-axis indicates bits of information–perfect conservation of a base would score as two bits.</p>
<span>doi:10.1371/journal.pone.0001340.g003</span></div>

<h4>EIAV integration targeting in human SupT1 cells depleted for LEDGF/p75</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The genomic distribution of EIAV integration sites was then compared in the presence and absence of LEDGF/p75 (<a href="#pone-0001340-t002">Table 2</a> and <a href="#pone-0001340-g004">Figure 4</a>). Integration site data sets were compared systematically relative to many forms of genomic annotation. Complete reports are in Statistical <a href="#pone.0001340.s002">Reports S1</a> and <a href="#pone.0001340.s003">S2</a>. Major findings are summarized below.</p>
<div class="figure" id="pone-0001340-g004"><div class="img"><a name="pone-0001340-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>EIAV integration site distributions in control and LEDGF/p75-knockdown SupT1 cells.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Integration site distributions are shown relative to A) RefSeq genes, B) CpG islands (plus or minus 1 kb), C) relative G/C content (Integration sites from unmodified and knockdown cells were pooled and divided into 10 equal bins of increasing GC content, and sites in each cell type plotted for each bin), D) gene density, and E) relative gene expression intensity. For each value in A–B) and D), the measured value for the integration site population was divided by that of the matched random control to emphasize the departure of the experimental data from random. P values shown are based on regression analysis (A–C) or Chi Square test for trend (D–E).</p>
<span>doi:10.1371/journal.pone.0001340.g004</span></div><div class="figure" id="pone-0001340-t002"><div class="img"><a name="pone-0001340-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001340.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001340.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Integration frequency in the presence and absence of LEDGF/p75 near mapped genomic features in the human genome.</span></strong></p><span>doi:10.1371/journal.pone.0001340.t002</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Three catalogs of human gene annotation were used to analyze EIAV integration site distributions, since LEDGF/p75 had previously been implicated in directing HIV integration to transcription units. From 60 to 69% of EIAV integration sites were in genes (<a href="#pone-0001340-t002">Table 2</a>), while a computationally generated random distribution showed only 37 to 43 % in genes. In the cell line strongly depleted for LEDGF/p75, integration frequency in genes ranged from 45% to 52%, a significant reduction compared to the pooled SupT1 and SupT1 SCRAM controls (P&lt;0.0001 for Known genes, P&lt;0.0001 for RefSeq, P = 0.027 for Unigenes; comparison to pooled controls by the Fisher's exact test). However, even in the absence of LEDGF/p75, integration in genes was still significantly favored over random in two out of three sets of gene calls (<a href="#pone-0001340-t002">Table 2</a>). <a href="#pone-0001340-g004">Figure 4A</a> shows the extent of favoring of integration in RefSeq transcription units normalized to the random expectation.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>In some data sets integration by lentiviruses has been found to be disfavored near CpG islands <a href="#pone.0001340-Mitchell1">[22]</a>, which are genomic regions enriched in the rare CpG dinucleotide and commonly associated with transcription start sites and regulatory regions. EIAV also showed disfavored integration near CpG islands (P&lt;0.0001 for comparison random sites by the Fisher's exact test). In the LEDGF/p75-depleted cells, integration frequency within 2 Kb of CpG islands went up, so that CpG islands were no longer disfavored (<a href="#pone-0001340-t002">Table 2</a>), and the difference between pooled SupT1 control sites and LEDGF/p75-depleted cells achieved significance (P&lt;0.0001, Fisher's exact test). <a href="#pone-0001340-g004">Figure 4B</a> shows the frequency within 1 kb, plotted to emphasize the enrichment over random.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>CpG islands are often associated with transcription start sites. Analysis of integration frequency showed a trend toward more frequent integration near transcription start sites in the knockdown (6% in pooled SupT1 controls versus 10% in the knockdown) though the trend did not achieve significance with this sample size (P = 0.083 by the Fisher's exact test).</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>In the previous study of weaker LEDGF/p75-knockdowns <a href="#pone.0001340-Ciuffi2">[40]</a>, HIV integration in knockdown cells was associated with an increase in the relative G/C content at integration sites. One speculation was that this was because LEDGF/p75 contains an A/T hook DNA binding domain, which may promote integration in A/T-rich regions in LEDGF/p75-positive cells <a href="#pone.0001340-Ciuffi2">[40]</a>. <a href="#pone-0001340-g004">Figure 4C</a> shows that in the SupT1 cell model as well, strong depletion of LEDGF/p75 resulted in increased G/C content at integration sites (P = 0.0003 by regression analysis).</p>
<a id="article1.body1.sec2.sec5.p6" name="article1.body1.sec2.sec5.p6"></a><p>One of the main questions at the start of this study was whether a stronger knockdown of LEDGF/p75 would result in stronger effects on lentivirus integration targeting. <a href="#pone-0001340-g005">Figure 5</a> shows a comparison of two HIV integration site data sets from Ciuffi et al. for HIV integration in Jurkat or 293T cells <a href="#pone.0001340-Ciuffi2">[40]</a>, which harbored less complete knockdowns of LEDGF/p75. In the control cells (<a href="#pone-0001340-g005">Figure 5, blue shading</a>) integration was enriched in transcription units in all cases. In the LEDGF/p75 knockdowns (<a href="#pone-0001340-g005">Figure 5, orange and yellow shading</a>), the proportion of integration sites in genes was reduced, with the percent change significantly greater in the intensified SupT1 knockdown over many of the gene catalogs studied.</p>
<div class="figure" id="pone-0001340-g005"><div class="img"><a name="pone-0001340-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Comparison of LEDGF/p75 knockdowns in different human cell types.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>The Jurkat and 293T data sets are described in detail in <a href="#pone.0001340-Ciuffi2">[40]</a>. Integration frequency was compared within RefSeq genes.</p>
<span>doi:10.1371/journal.pone.0001340.g005</span></div><a id="article1.body1.sec2.sec5.p7" name="article1.body1.sec2.sec5.p7"></a><p>Integration frequency at some of the genomic features studied was not detectably affected by the LEDGF/p75 knockdown. For example, when integration frequency was assessed relative to gene density, no strong effect was seen (<a href="#pone-0001340-g004">Figure 4D</a>). Similarly, the relationship between gene activity and integration frequency was not significantly altered (<a href="#pone-0001340-g004">Figure 4E</a>). Integration frequency near open chromatin as marked by DNAse I hypersensitive sites was also not significantly altered by the knockdown (data not shown). This implies that either there is residual LEDGF/p75 present even in the intensified knockdowns that is sufficient to influence targeting, or else other cellular systems contribute to integration targeting as well.</p>


<h4>Consensus sequences at lentiviral integration sites in murine cells disrupted at <em>LEDGF/p75</em></h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>We analyzed integration sites in murine embryonic fibroblasts (MEFs) derived from the <em>LEDGF/p75</em> homozygous gene trap (−/−) and control (+/+) mice <a href="#pone.0001340-Sutherland1">[42]</a> after infection with HIV and EIAV. Cells that had been immortalized in culture (iMEF) were compared to primary MEFs (prMEFs). For all the features discussed below the results were identical for iMEFs and prMEFs (data not shown), so the two data sets were pooled in what follows.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Integration site sequences were aligned to determine the consensus palindromic sequence at the point of integration, and results were compared for the +/+ and −/− MEFs for each virus (<a href="#pone-0001340-g006">Figure 6</a>). In both cases, integration in the +/+ MEFs showed the weak consensus seen previously for HIV and EIAV. No major differences were seen in the −/− MEFs, consistent with findings described above for human cells and previously <a href="#pone.0001340-Ciuffi2">[40]</a>, <a href="#pone.0001340-Shun1">[41]</a>.</p>
<div class="figure" id="pone-0001340-g006"><div class="img"><a name="pone-0001340-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Integration site consensus sequence for lentiviral infection of murine control and <em>LEDGF/p75</em>-disrupted cells.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>A) HIV in +/+ MEFs. B) HIV integration in −/− MEFs. C) EIAV integration in +/+ MEFs. D) EIAV integration in −/− MEFs. Markings as in <a href="#pone-0001340-g003">Figure 3</a>.</p>
<span>doi:10.1371/journal.pone.0001340.g006</span></div>

<h4>EIAV integration targeting in murine cells disrupted at <em>LEDGF/p75</em></h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Genome-wide studies of EIAV integration targeting in murine cells are presented in this section and analysis of HIV integration in murine cells is described in the next section. Extensive further analysis of EIAV and HIV integration in MEFs is presented in Statistical <a href="#pone.0001340.s003">Reports S2</a>.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>EIAV integration in transcription units was decreased in the −/− <em>LEDGF/p75</em> gene trap cells compared with wild-type. In wild-type cells, 58.6% of experimental integration sites were in RefSeq genes (see <a href="#pone-0001340-t003">Table 3</a>), a significant enrichment over the 28% seen in the matched random controls (see <a href="#pone-0001340-g007">Figure 7A</a>). In −/− MEFs, 38.4% of sites were in RefSeq transcription units, a value that is significantly less than in the +/+ MEFs (p = 0.016 by the Fisher's exact test). Significant differences were seen when the analysis was repeated using other gene catalogs as well (<a href="#pone-0001340-t003">Table 3</a>).</p>
<div class="figure" id="pone-0001340-g007"><div class="img"><a name="pone-0001340-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>EIAV integration distributions in murine control and <em>LEDGF/p75</em>-disrupted cells.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Integration frequencies are shown relative to A) RefSeq genes, B) CpG islands (1 kb window; note that there were no control sites within &lt;1 kb), C) G/C content, D) Gene density (250 kb window), E) Gene activity. Markings as in <a href="#pone-0001340-g004">Figure 4</a>.</p>
<span>doi:10.1371/journal.pone.0001340.g007</span></div><div class="figure" id="pone-0001340-t003"><div class="img"><a name="pone-0001340-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001340.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001340.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Integration frequency in the presence and absence of LEDGF/p75 near mapped genomic features in the murine genome.</span></strong></p><span>doi:10.1371/journal.pone.0001340.t003</span></div><a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>We also analyzed the proximity of EIAV integration sites to CpG islands (<a href="#pone-0001340-g007">Figure 7B</a> and <a href="#pone-0001340-t003">Table 3</a>). In wild-type cells integration within 2 kb of CpG islands was not significantly different from random, while in knockout cells integration was 13-fold enriched over random (P = 0.0086; Fisher's exact test).</p>
<a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>The frequency of integration within 5 kb of RefSeq gene 5′ ends showed a similar pattern (<a href="#pone-0001340-t003">Table 3</a>). Integration levels around gene 5′ ends were not significantly different from random in the +/+ cells (5.7% of sites), whereas in the knockout a significant enrichment was observed (25.6% of sites) achieving P = 0.014 for the comparison between cell types (Fisher's exact test).</p>
<a id="article1.body1.sec2.sec7.p5" name="article1.body1.sec2.sec7.p5"></a><p>We analyzed the correlation between integration frequency and G/C content using a 5 kb window around the integration site. A significant difference between genotypes was found (P = 0.001, using regression analysis, <a href="#pone-0001340-g007">Figure 7C</a>).</p>
<a id="article1.body1.sec2.sec7.p6" name="article1.body1.sec2.sec7.p6"></a><p>A variety of features analyzed did not show significant differences between genotypes, including the response to gene density (<a href="#pone-0001340-g007">Figure 7D</a>) and the relationship between gene activity and integration frequency (<a href="#pone-0001340-g007">Figure 7E</a>). We return to the implications of these findings in the Discussion.</p>


<h4>HIV integration targeting in murine cells disrupted at <em>LEDGF/p75</em></h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>Data on HIV integration site distributions in MEFs closely matched the data for EIAV integration (<a href="#pone-0001340-g008">Figure 8</a> and <a href="#pone-0001340-t003">Table 3</a>). HIV integration in +/+ MEFs showed a strong preference for transcription units (<a href="#pone-0001340-t003">Table 3</a> and <a href="#pone-0001340-g008">Figure 8</a>), which was strongly reduced in the −/− MEFs (P&lt;0.0001 for comparison between genotypes).</p>
<div class="figure" id="pone-0001340-g008"><div class="img"><a name="pone-0001340-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>HIV integration distributions in murine control and <em>LEDGF/p75</em>-disrupted cells.</span></strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>A) RefSeq genes, B) CpG islands (1 kb window), C) G/C content, D) Gene density (250 kb window), E) Gene activity. Markings as in <a href="#pone-0001340-g004">Figure 4</a>.</p>
<span>doi:10.1371/journal.pone.0001340.g008</span></div><a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>HIV integration within 2 kb of CpG islands was found to be disfavored compared with matched random controls, and this was the case in +/+ MEFs (<a href="#pone-0001340-g008">Figure 8B</a> and <a href="#pone-0001340-t003">Table 3</a>). Integration in −/− MEFs was greatly increased within 2 kb of CpG islands or 5 kb of transcription start sites (P&lt;0.0001 and P = 0.014 or the respective comparisons between genotypes).</p>
<a id="article1.body1.sec2.sec8.p3" name="article1.body1.sec2.sec8.p3"></a><p>Knockdown of LEDGF/p75 has previously been shown to result in an increase in the G/C content of HIV integration site sequences <a href="#pone.0001340-Ciuffi2">[40]</a>. We therefore analyzed the frequency of integration in regions of varying G/C content (<a href="#pone-0001340-g008">Figure 8C</a>), revealing that integration was significantly increased in more G/C rich regions in the −/− MEFs (P = 4e-16).</p>
<a id="article1.body1.sec2.sec8.p4" name="article1.body1.sec2.sec8.p4"></a><p>As seen above for EIAV, the frequency of integration near a variety of features was not detectably altered. <a href="#pone-0001340-g008">Figure 8D</a> shows that integration frequency was similarly favored in gene-rich regions in both the +/+ and −/− MEFs. <a href="#pone-0001340-g008">Figure 8E</a> shows that the relative activity of genes hosting integration events was also not distinguishable for the +/+ and −/− MEFs (<a href="#pone-0001340-g008">Figure 8E</a>).</p>


<h4>Correlation between <em>LEDGF/p75</em> expression and the frequency of HIV integration in transcription units analyzed over many cell types</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>In addition to studying cells with artificially reduced levels of <em>LEDGF/p75</em> expression, we were interested in natural variation in cellular <em>LEDGF/p75</em> expression levels. Different primary cell types and cell lines show different steady state levels of <em>LEDGF/p75</em> mRNA. Different cell types also show reproducibly different frequencies of HIV integration in transcription units (see <a href="#pone.0001340-Ciuffi2">[40]</a> for examples). We thus asked whether cell types with higher LEDGF/p75 levels showed higher frequencies of HIV integration transcription units.</p>
<a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>We analyzed data from 15 HIV integration site data sets for which we also had transcriptional profiling data on gene activity for that cell type. For each microarray data set, the expression level of <em>LEDGF/p75</em>-specific probe sets was ranked relative to all other probe sets on the array for that cell type, thus yielding a value for relative <em>LEDGF/p75</em> expression. These values were then plotted against the proportion of HIV integration sites in transcription units for that cell type (<a href="#pone-0001340-g009">Figure 9</a>). This analysis showed that increased relative <em>LEDGF/p75</em> mRNA abundance positively correlated with increased HIV integration frequency in transcription units (R<sup>2</sup> = 0.61; P&lt;0.0001). <a href="#pone-0001340-g009">Figure 9</a> shows data with experimental LEDGF/p75 knockdowns included (triangles), but the correlation was still significant when the experimental knockdowns were excluded (P&lt;0.0001), indicating that natural variation in LEDGF/p75 levels was functionally significant.</p>
<div class="figure" id="pone-0001340-g009"><div class="img"><a name="pone-0001340-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001340" data-uri="info:doi/10.1371/journal.pone.0001340.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001340.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001340.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001340.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001340.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Correlation between LEDGF/p75 expression and the frequency of HIV integration in genes.</span></strong></p><a id="article1.body1.sec2.sec9.fig1.caption1.p1" name="article1.body1.sec2.sec9.fig1.caption1.p1"></a><p>Data is shown for 15 HIV integration site data sets in 10 cell types. The y-axis shows the percentage of integration events within transcription units of the “known gene” set of human genes for each integration site data set. The x-axis shows relative expression values for <em>LEDGF/p75</em> derived from Affymetrix array data (see <a href="#s4">methods</a> for details). The R-squared value for the fit is 0.6148 (P&lt;0.0001). The references for the data sets used are as follows: Macrophage 1 is the VSV-G set in <a href="#pone.0001340-Barr1">[25]</a>; Macrophage 2 is the CCR5 set in <a href="#pone.0001340-Barr1">[25]</a>; SupT1 <a href="#pone.0001340-Schroder1">[21]</a>; IMR90 1 is the dividing set in <a href="#pone.0001340-Ciuffi4">[66]</a>; IMR90 2 is the growth-arrested set in <a href="#pone.0001340-Ciuffi4">[66]</a>; CD4 T <a href="#pone.0001340-Levine1">[67]</a>; PBMC <a href="#pone.0001340-Mitchell1">[22]</a>; Jurkat 1 is the Mse set in <a href="#pone.0001340-Wang1">[46]</a>; Jurkat 2 is the Avr set in <a href="#pone.0001340-Wang1">[46]</a>; Jurkat 3 is the initially bright set in <a href="#pone.0001340-Lewinski1">[5]</a>; Jurkat 4 is the initially dark set in <a href="#pone.0001340-Lewinski1">[5]</a>; Jurkat p75 knockdown <a href="#pone.0001340-Ciuffi2">[40]</a>,<a href="#pone.0001340-Wang1">[46]</a>; 293T <a href="#pone.0001340-Ciuffi2">[40]</a>; 293T Scram <a href="#pone.0001340-Ciuffi2">[40]</a>; 293T p75 knockdown <a href="#pone.0001340-Ciuffi2">[40]</a>.</p>
<span>doi:10.1371/journal.pone.0001340.g009</span></div><a id="article1.body1.sec2.sec9.p3" name="article1.body1.sec2.sec9.p3"></a><p>Some of the data in <a href="#pone-0001340-g009">Figure 9</a> and in previous studies was generated using transformed cell lines, leaving open the question of whether natural variation in LEDGF/p75 levels was functionally important in human primary cells. We repeated the analysis in <a href="#pone-0001340-g009">Figure 9</a> using only data from human primary cells where LEDGF/p75 levels had not been altered experimentally, and again found a significant positive correlation between integration frequency in genes and <em>LEDGF/p75</em> mRNA levels (P = 0.044). These data indicate that natural variation in <em>LEDGF/p75</em> expression levels is a significant determinant of integration frequency in transcription units in human primary cells.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Here we report studies of lentiviral integration in two cell types with strong depletions of LEDGF/p75. In the first, we studied the SupT1 human T-cell line with intensified RNAi against LEDGF/p75 described in <a href="#pone.0001340-Llano4">[37]</a>. Extensive characterization has shown that these cells have stronger knockdowns than those studied previously (e. g. <a href="#pone.0001340-Llano1">[12]</a>, <a href="#pone.0001340-Ciuffi2">[40]</a>), providing an improved model for the role of LEDGF/p75 in lentiviral integration targeting in human T-cells. In the second cell model, we studied murine cells with a homozygous gene-trap mutation disrupting the <em>LEDGF/p75</em> locus <a href="#pone.0001340-Sutherland1">[42]</a>. We also presented data on EIAV, extending the collection of lentiviruses shown functionally to be affected by LEDGF/p75. Infectivity for both HIV and EIAV was reduced 5–50 fold in LEDGF/p75-depleted cells, in good agreement with data on HIV and FIV published previously <a href="#pone.0001340-Llano4">[37]</a>, <a href="#pone.0001340-Shun1">[41]</a>–taken together, these studies firmly establishing that strong LEDGF-p75 knockdowns strongly reduce HIV infectivity. The data reported on target site selection in human cells and murine cells were closely parallel with each other, and also parallel with studies of another murine <em>LEDGF/p75</em> mutant <a href="#pone.0001340-Llano4">[37]</a>, <a href="#pone.0001340-Shun1">[41]</a>. Comparison of integration targeting data reported here to earlier data with weaker knockdowns <a href="#pone.0001340-Ciuffi2">[40]</a> showed that indeed intensifying the LEDGF/p75 depletion further diminished the proportion of lentiviral integration sites in transcription units. Because more than half of the favoring of transcription units was eliminated by the stronger depletion of LEDGF/p75, we can conclude that the LEDGF/p75-dependent pathway is the predominant pathway for targeting integration to transcription units.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Published studies of integration targeting by LEDGF/p75 have relied on analysis of cells where the LEDGF/p75 levels were artificially reduced—thus there is interest in obtaining data on the effects of LEDGF/p75 in cells naturally expressing different levels of the protein. We took advantage of the observation that different cell types differ reproducibly in their frequency of integration in transcription units <a href="#pone.0001340-Ciuffi2">[40]</a> to investigate this question. A bioinformatic comparison (<a href="#pone-0001340-g009">Figure 9</a>) showed that higher levels of <em>LEDGF/p75</em> expression correlated with higher frequencies of integration in transcription units. The trend achieved significance even when the analysis was restricted to human primary cells only. Thus the study of natural variation in <em>LEDGF/p75</em> expression allowed us to extend the idea that LEDGF/p75 directs HIV integration to transcription units in human primary cells without artificially reduced LEDGF/p75 levels.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>A simple model holds that LEDGF/p75 directs favored integration into transcription units by tethering. According to this model, one domain of LEDGF/p75 binds to HIV preintegration complexes and the other binds chromatin at active transcription units. Data from artificial tethering studies <em>in vitro</em> with fusions of the LEDGF/p75 IBD to a sequence-specific binding domain support this model <a href="#pone.0001340-Ciuffi3">[56]</a>. The tethering model predicts that LEDGF/p75 should accumulate on active transcription units, but so far this has not been demonstrated experimentally. Similarly, it is not known how LEDGF/p75 recognizes active transcription units. One possible model would be that histone post-translational modifications mark active transcription units and guide LEDGF/p75 binding. Potentially consistent with this idea is the finding that HIV integration is positively correlated with several types of histone post-translational modifications <a href="#pone.0001340-Wang1">[46]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Curiously, both this study and Shun et al. <a href="#pone.0001340-Shun1">[41]</a> showed not only a loss of integration targeting in LEDGF/p75-depleted cells, but new favored genomic regions as well. From the previous study alone this might have been an idiosyncrasy of the murine model, but data presented here shows a similar response in human cells. In all LEDGF/p75-depleted cell types in both studies, integration became more favored near transcription start sites and associated CpG islands. The basis for this trend is unknown. It may be that preintegration complexes normally associated with LEDGF/p75 become free to integrate near these sites once LEDGF/p75 was removed. Possibly chromatin at start site regions is particularly accessible and so represents a default target. It is also possible that a more active mechanism is involved. In support of this idea is the finding that MLV integration is strongly favored at start sites <a href="#pone.0001340-Lewinski2">[28]</a>, <a href="#pone.0001340-Wu2">[57]</a>, while several other integrating elements show near random distributions <a href="#pone.0001340-Mitchell1">[22]</a>, <a href="#pone.0001340-Berry1">[55]</a>, <a href="#pone.0001340-Miller1">[58]</a>, suggesting that mechanisms exist to guide preferential integration near start sites. A variety of genomic features showed positive correlation with lentiviral integration in both the depleted cells and controls, indicating that cellular systems in addition to LEDGF/p75 also influence integration. As increasingly deep annotation of the human genome accumulates, it may be possible to detect additional associations between lentiviral integration and particular bound proteins, potentially allowing identification of host cell factors operating in the absence of LEDGF/p75.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Finally, data presented here and in <a href="#pone.0001340-Llano4">[37]</a>, <a href="#pone.0001340-Shun1">[41]</a> emphasizes that LEDGF/p75 is important for efficient HIV replication, suggesting that the interaction between IN and LEDGF/p75 may be a tractable target for antiviral therapy. The structure of a complex of the LEDGF/p75 IBD and the IN catalytic domain have been solved by X-ray crystallography<a href="#pone.0001340-Cherepanov4">[33]</a>, and the interaction surface was found to overlap with the binding site seen previously for the integrase inhibitor tetraphenylarsonium <a href="#pone.0001340-Molteni1">[59]</a>. This supports the idea that small molecule inhibitors, if of high enough affinity, may be able to disrupt binding of LEDGF/p75 to integrase and so abrogate HIV replication.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cell lines</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>MEFs were extracted from wild-type and knockout embryos at 13.5 dpc <a href="#pone.0001340-Stone1">[60]</a> and cultured in DMEM with 10% FBS, 50 µg/ml gentamycin, 110 µM beta-mercaptoethanol, 1× non-essential amino acids, 100 µM sodium pyruvate. Primary MEFs (prMEFs) were immortalized by the 3T3 protocol, by splitting cells every 3 days to a density of 6×10<sup>4</sup> cells/ml <a href="#pone.0001340-Todaro1">[61]</a>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>TC2 and TL2 are control (“scramble” sequence) and active shRNA-expressing SupT1 cell lines derived in parallel by intensified RNAi. They were established simultaneously from the same parental population, using equivalent MOI transduction with lentiviral vectors that differed only in the 19 nt of the shRNA <a href="#pone.0001340-Llano4">[37]</a>.</p>


<h4>Viral particle production and infections</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>VSV-G pseudotyped HIV vector particles were produced by Lipofectamine transfection of 293T cells with p156RRLsin-PPTCMVGFPWPRE <a href="#pone.0001340-Follenzi1">[62]</a>, the packaging construct pCMVdeltaR9 <a href="#pone.0001340-Naldini1">[63]</a>, and the vesicular stomatitis virus G-producing pMD.G construct. EIAV vector particles were likewise produced by transfection with p6.1G3CeGFPw (M. Patel and J. Olsen, University of North Carolina, Chapel Hill unpublished), the packaging construct pEV53B <a href="#pone.0001340-Olsen1">[64]</a>, and the vesicular stomatitis virus G-producing plasmid pVSVG into 293T cells. Viral supernatant was harvested 38 hours after transfection, filtered through 0.22 µm filters, concentrated by filtration through a Centricon, treated with DNase I, and stored frozen at −80°C. HIV titer was quantified by p24 ELISA.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>For EIAV infection of SupT1 cells, cells were plated at 1×10<sup>5</sup> cells per well of a 24-well plate, infected with between 25–100 µl concentrated DNase I treated virus stock, and all wells were brought to 200 µl final volume with fresh RPMI containing 10% heat-inactivated FBS, 10 units/ml penicillin, 10 µg/ml streptomycin and 50 µg/ml gentamycin (R-10). At 5 hours all well contents were transferred to a 1.5 ml Eppendorf and spun for 10 min at 1000RPMs to pellet cells. Cells were resuspended in 1 ml R-10 and cultured for an additional 76 hrs for integration site cloning or 2 weeks for QPCR analysis. Upon collection, 30–50% of cells expressed GFP as analyzed by fluorescence microscopy.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>For HIV infection of MEFs, cells were plated onto 6-well plates at a density of 3×10<sup>5</sup> cells per well and each well infected with 1 µg p24. For EIAV, cells were plated into 24-well plates at a density of 4×10<sup>4</sup> cells per well, and each well infected with 100 µl concentrated virus. Infections were performed overnight in the presence of 10 µg/ml DEAE-dextran. 10 independent HIV infections and 5 EIAV infections were performed per genotype. 48 hours after infection, 90% of cells were harvested for integration site cloning and the remainder passaged for an additional 2 weeks to dilute unintegrated products of reverse transcription and used for QPCR analysis of integration efficiency.</p>


<h4>Infectivity tests</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>For quantitative PCR analysis, infected cells were passaged for 2 weeks following infection to dilute unintegrated products of reverse transcription, then genomic DNA was extracted using the Qiagen DNeasy tissue extraction kit. QPCR using HIV late-RT primers and probe was carried out as described in <a href="#pone.0001340-Butler2">[44]</a> using 50 ng genomic DNA as template. For EIAV, primer and probe sequences are described in <a href="#pone.0001340.s001">Table S1</a>. 25 ng of SupT1 genomic DNA was used as template, 50 ng of MEF genomic DNA. QPCR was performed using Applied Biosystems 2× FAST universal master mix and Applied Biosystems FAST PCR machine.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>For luciferase assays, HIV luciferase reporter virus stock was prepared by transfection of pLai3_envLuc2 <a href="#pone.0001340-Yamashita1">[65]</a> and the vesicular stomatitis virus G-producing plasmid pVSVG into 293T cells. Viral supernatant was collected 36 h after transfection, filtered through 0.22-µm filters, concentrated, assayed by p24 and stored frozen at –80°C. For infectivity assay, SupT1 cells plated at 1×10<sup>5</sup> cells per well of a 24-well plate were infected with various amounts of concentrated DNase treated virus stock. All wells were brought to 1 ml final volume with fresh R-10. Three days later, cells were lysed in 0.5% Triton-X 100 in PBS and luciferase levels were determined using Luciferase Assay System and a Thermo Luminoskan Ascent luminescence counter. All infections were performed in triplicate.</p>


<h4>Integration site cloning</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Integration sites were isolated and sequenced by linker-mediated PCR essentially as described previously <a href="#pone.0001340-Wang1">[46]</a>. Genomic DNA was extracted from infected cells using the Qiagen DNeasy tissue extraction kit. Up to 2 µg of DNA from each infection was digested overnight using MseI. This was followed by digestion to prevent amplification of internal viral fragments (from the 5′ LTR) and plasmid backbone with SacI and DpnI in the case on HIV, and XmaI and DpnI in the case of EIAV. Linkers were then ligated onto digestion products (oligonucleotide sequences can be found in <a href="#pone.0001340.s001">Table S1</a>) and nested PCR performed from ligation products. Nested PCR primers contained 4 or 8 nt barcode sequences between the sequencing primer and LTR-binding portions. These enabled pooling of all PCR products into one sequencing reaction and subsequent separation of sequences by decoding the barcodes. Amplification products were gel-purified and sent to the Interdisciplinary Center for Biotechnology Research at the University of Florida and the Virginia Bioinformatics Institute Core Laboratory Facility for pyrosequencing.</p>


<h4>Bioinformatic analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Integration sites were judged to be authentic if the sequences had a best unique hit when aligned to the murine or human genome as appropriate (mm8 and hg18 respectively) using BLAT, and the alignment began within 3bp of the viral LTR end and had &gt;98% sequence identity. Detailed statistical methods are described in <a href="#pone.0001340-Berry1">[55]</a> and Statistical <a href="#pone.0001340.s002">Reports S1</a> and <a href="#pone.0001340.s003">S2</a>.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>To control for possible biases in isolating integration sites due to restriction enzyme sequence distribution, three-ten matched random controls were computationally generated for each experimental integration site that were the same distance from the closest MseI restriction site as the experimental site.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>Integration site counts in various genomic annotations were compared with matched random controls by the Fisher's exact test. Additionally, multiple regression models for integration intensity were applied, as described in <a href="#pone.0001340-Berry1">[55]</a>.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>For analysis of correlations with gene activity in murine integration sites (<a href="#pone-0001340-g007">Figure 7</a> and <a href="#pone-0001340-g008">8</a>), transcriptional profiling data from wild-type MEFs analyzed on the MGU74Av2 Affymetrix microarray were used. Genes represented on the microarray were ranked by expression level and divided into 4 bins based on expression level. Integration sites found within genes in each bin were counted as a proportion of sites found within genes in all bins. For human expression analysis (<a href="#pone-0001340-g004">Figure 4</a>) data was from <a href="#pone.0001340-Lewinski1">[5]</a>.</p>
<a id="article1.body1.sec4.sec5.p5" name="article1.body1.sec4.sec5.p5"></a><p>For the analysis of relative gene activity in <a href="#pone-0001340-g009">Figure 9</a>, data from two types Affymetrix chips were used (HU95A and HU133A). Two probe sets querying <em>LEDGF/p75</em> but not <em>p52</em> were available on each chip (For HU95: 39243_s_at and 37622_r_at; for HU133: 209337_at and 205961_s_at). To account for differences in the sensitivities arising from the different chip designs and probe sets, the values for each cell type were first ranked for each probe set and chip combination, then the ranked values pooled in the final data set.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001340"></div><a name="pone.0001340.s001" id="pone.0001340.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001340.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Oligonucleotides used in this study</p>
<p class="siDoi">doi:10.1371/journal.pone.0001340.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.01 MB XLS)</p>
<a name="pone.0001340.s002" id="pone.0001340.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001340.s002">Statistical Report S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">EIAV integration in human cells</p>
<p class="siDoi">doi:10.1371/journal.pone.0001340.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.35 MB PDF)</p>
<a name="pone.0001340.s003" id="pone.0001340.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001340.s003">Statistical Report S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">EIAV and HIV integration in murine cells</p>
<p class="siDoi">doi:10.1371/journal.pone.0001340.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.44 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank members of the Bickmore, Bushman, and Poeschla laboratory for materials and helpful discussions. We particularly thank Dr. Roderick Jensen and members of the Virginia Bioinformatics Institute Core Laboratory Facility for sequencing support. We also thank John C. Olsen and Manij Patel, at the Cystic Fibrosis/Pulmonary Research and Treatment Center, University of North Carolina at Chapel Hill for the development and donation of the EIAV vectors, maps and infection advice.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: FB. Performed the experiments: HM KR. Analyzed the data: FB HM KR. Contributed reagents/materials/analysis tools: WB FB CB ML HS DS EP. Wrote the paper: FB HM KR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001340-Coffin1" id="pone.0001340-Coffin1"></a>Coffin JM, Hughes SH, Varmus HE (1997) Retroviruses. Cold Spring Harbor: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001340-Bushman1" id="pone.0001340-Bushman1"></a>Bushman FD (2001) Lateral DNA transfer: Mechanisms and consequences. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0001340-Jordan1" id="pone.0001340-Jordan1"></a>Jordan A, Defechereux P, Verdin E (2001) The site of HIV-1 integration in the human genome determines basal transcriptional activity and response to tat transactivation. EMBO J  20: 1726–1738.  <ul class="find" data-citedArticleID="1032508" data-doi="10.1093/emboj/20.7.1726"><li><a href="http://dx.doi.org/10.1093/emboj/20.7.1726" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+site+of+HIV-1+integration+in+the+human+genome+determines+basal+transcriptional+activity+and+response+to+tat+transactivation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+site+of+HIV-1+integration+in+the+human+genome+determines+basal+transcriptional+activity+and+response+to+tat+transactivation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001340-Jordan2" id="pone.0001340-Jordan2"></a>Jordan A, Bisgrove D, Verdin E (2003) HIV reporducibly establishes a latent infection after acute infection of T cells in vitro. EMBO J  22: 1868–1877.  <ul class="find" data-citedArticleID="1032510" data-doi="10.1093/emboj/cdg188"><li><a href="http://dx.doi.org/10.1093/emboj/cdg188" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+reporducibly+establishes+a+latent+infection+after+acute+infection+of+T+cells+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+reporducibly+establishes+a+latent+infection+after+acute+infection+of+T+cells+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001340-Lewinski1" id="pone.0001340-Lewinski1"></a>Lewinski M, Bisgrove D, Shinn P, Chen H, Verdin E, et al.  (2005) Genome-wide analysis of chromosomal features repressing HIV transcription. J Virol  79: 6610–6619.  <ul class="find" data-citedArticleID="1032516" data-doi="10.1128/jvi.79.11.6610-6619.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.11.6610-6619.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+chromosomal+features+repressing+HIV+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+chromosomal+features+repressing+HIV+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001340-Bisgrove1" id="pone.0001340-Bisgrove1"></a>Bisgrove D, Lewinski M, Bushman FD, Verdin E (2005) Molecular mechanisms of HIV-1 proviral latency. Expert Rev Anti Infect Ther  3: 805–814.  <ul class="find" data-citedArticleID="1032454" data-doi="10.1586/14787210.3.5.805"><li><a href="http://dx.doi.org/10.1586/14787210.3.5.805" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+mechanisms+of+HIV-1+proviral+latency." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+mechanisms+of+HIV-1+proviral+latency.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001340-HaceinBeyAbina1" id="pone.0001340-HaceinBeyAbina1"></a>Hacein-Bey-Abina S, von Kalle C, Schmidt M, Le Deist F, Wulffraat N, et al.  (2003) A serious adverse event after successful gene therapy for X-linked severe combined immunodeficiency. N Engl J Med  348: 255–256.  <ul class="find" data-citedArticleID="1032496" data-doi="10.1056/nejm200301163480314"><li><a href="http://dx.doi.org/10.1056/nejm200301163480314" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+serious+adverse+event+after+successful+gene+therapy+for+X-linked+severe+combined+immunodeficiency." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+serious+adverse+event+after+successful+gene+therapy+for+X-linked+severe+combined+immunodeficiency.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001340-HaceinBeyAbina2" id="pone.0001340-HaceinBeyAbina2"></a>Hacein-Bey-Abina S, Von Kalle C, Schmidt M, McCormack MP, Wulffraat N, et al.  (2003) LMO2-associated clonal T cell proliferation in two patients after gene therapy for SCID-X1. Science  302: 415–419.  <ul class="find" data-citedArticleID="1032498" data-doi="10.1126/science.1088547"><li><a href="http://dx.doi.org/10.1126/science.1088547" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LMO2-associated+clonal+T+cell+proliferation+in+two+patients+after+gene+therapy+for+SCID-X1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LMO2-associated+clonal+T+cell+proliferation+in+two+patients+after+gene+therapy+for+SCID-X1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001340-Cherepanov1" id="pone.0001340-Cherepanov1"></a>Cherepanov P, Maertens G, Proost P, Devreese B, Van Beeumen J, et al.  (2003) HIV-1 integrase forms stable tetramers and associates with LEDGF/p75 protein in human cells. J Biol Chem  278: 372–381.  <ul class="find" data-citedArticleID="1032468" data-doi="10.1074/jbc.m209278200"><li><a href="http://dx.doi.org/10.1074/jbc.m209278200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+integrase+forms+stable+tetramers+and+associates+with+LEDGF%2Fp75+protein+in+human+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+integrase+forms+stable+tetramers+and+associates+with+LEDGF%2Fp75+protein+in+human+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001340-Maertens1" id="pone.0001340-Maertens1"></a>Maertens G, Cherepanov P, Pluymers W, Busschots K, De Clercq E, et al.  (2003) LEDGF/p75 is essential for nuclear and chromosomal targeting of HIV-1 integrase in human cells. J Biol Chem  278: 33528–33539.  <ul class="find" data-citedArticleID="1032530" data-doi="10.1074/jbc.m303594200"><li><a href="http://dx.doi.org/10.1074/jbc.m303594200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LEDGF%2Fp75+is+essential+for+nuclear+and+chromosomal+targeting+of+HIV-1+integrase+in+human+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LEDGF%2Fp75+is+essential+for+nuclear+and+chromosomal+targeting+of+HIV-1+integrase+in+human+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001340-Turlure1" id="pone.0001340-Turlure1"></a>Turlure F, Devroe E, Silver PA, Engelman A (2004) Human cell proteins and human immunodeficiency virus DNA integration. Front Biosci  9: 3187–3208.  <ul class="find" data-citedArticleID="1032560" data-doi="10.2741/1472"><li><a href="http://dx.doi.org/10.2741/1472" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+cell+proteins+and+human+immunodeficiency+virus+DNA+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+cell+proteins+and+human+immunodeficiency+virus+DNA+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001340-Llano1" id="pone.0001340-Llano1"></a>Llano M, Vanegas M, Fregoso O, Saenz D, Chung S, et al.  (2004) LEDGF/p75 determines cellular trafficking of diverse lentiviral but not murine oncoretroviral integrase proteins and is a component of functional lentiviral preintegration complexes. J Virol  78: 9524–9537.  <ul class="find" data-citedArticleID="1032520" data-doi="10.1128/jvi.78.17.9524-9537.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.17.9524-9537.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LEDGF%2Fp75+determines+cellular+trafficking+of+diverse+lentiviral+but+not+murine+oncoretroviral+integrase+proteins+and+is+a+component+of+functional+lentiviral+preintegration+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LEDGF%2Fp75+determines+cellular+trafficking+of+diverse+lentiviral+but+not+murine+oncoretroviral+integrase+proteins+and+is+a+component+of+functional+lentiviral+preintegration+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001340-Emiliani1" id="pone.0001340-Emiliani1"></a>Emiliani S, Mousnier A, Busschots K, Maroun M, Van Maele B, et al.  (2005) Integrase mutants defective for interaction with LEDGF/p75 are impaired in chromosome tethering and HIV-1 replication. J Biol Chem  280: 25517–23.  <ul class="find" data-citedArticleID="1032492" data-doi="10.1074/jbc.m501378200"><li><a href="http://dx.doi.org/10.1074/jbc.m501378200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrase+mutants+defective+for+interaction+with+LEDGF%2Fp75+are+impaired+in+chromosome+tethering+and+HIV-1+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrase+mutants+defective+for+interaction+with+LEDGF%2Fp75+are+impaired+in+chromosome+tethering+and+HIV-1+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001340-Cherepanov2" id="pone.0001340-Cherepanov2"></a>Cherepanov P, Devroe E, Silver PA, Engelman A (2004) Identification of an evolutionarily-conserved domain of LEDGF/p75 that bind HIV-1 integrase. J Biol Chem  279: 48883–48892.  <ul class="find" data-citedArticleID="1032470" data-doi="10.1074/jbc.m406307200"><li><a href="http://dx.doi.org/10.1074/jbc.m406307200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+an+evolutionarily-conserved+domain+of+LEDGF%2Fp75+that+bind+HIV-1+integrase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+an+evolutionarily-conserved+domain+of+LEDGF%2Fp75+that+bind+HIV-1+integrase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001340-Vanegas1" id="pone.0001340-Vanegas1"></a>Vanegas M, Llano M, Delgado S, Thompson D, Peretz M, et al.  (2005) Identification of the LEDGF/p75 HIV-1 integrase interaction domain and NLS reveals NLS-independent chromatin tethering. J Cell Sci  118: 1733–1743.  <ul class="find" data-citedArticleID="1032568" data-doi="10.1242/jcs.02299"><li><a href="http://dx.doi.org/10.1242/jcs.02299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+the+LEDGF%2Fp75+HIV-1+integrase+interaction+domain+and+NLS+reveals+NLS-independent+chromatin+tethering." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+the+LEDGF%2Fp75+HIV-1+integrase+interaction+domain+and+NLS+reveals+NLS-independent+chromatin+tethering.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001340-Llano2" id="pone.0001340-Llano2"></a>Llano M, Vanegas M, Hutchins N, Thompson D, Delgado S, et al.  (2006) Identification and characterization of the chromatin-binding domains of the HIV-1 integrase interactor LEDGF/p75. J Mol Biol  360: 760–773.  <ul class="find" data-citedArticleID="1032522" data-doi="10.1016/j.jmb.2006.04.073"><li><a href="http://dx.doi.org/10.1016/j.jmb.2006.04.073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+the+chromatin-binding+domains+of+the+HIV-1+integrase+interactor+LEDGF%2Fp75." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+the+chromatin-binding+domains+of+the+HIV-1+integrase+interactor+LEDGF%2Fp75.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001340-Turlure2" id="pone.0001340-Turlure2"></a>Turlure F, Maertens G, Rahman S, Cherepanov P, Engelman A (2006) A tripartite DNA-binding element, comprised of the nuclear localization signal and two AT-hook motifs, mediates the association of LEDGF/p75 with chromatin in vivo. Nucleic Acids Res  34: 1653–1675.  <ul class="find" data-citedArticleID="1032562" data-doi="10.1093/nar/gkl052"><li><a href="http://dx.doi.org/10.1093/nar/gkl052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+tripartite+DNA-binding+element%2C+comprised+of+the+nuclear+localization+signal+and+two+AT-hook+motifs%2C+mediates+the+association+of+LEDGF%2Fp75+with+chromatin+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+tripartite+DNA-binding+element%2C+comprised+of+the+nuclear+localization+signal+and+two+AT-hook+motifs%2C+mediates+the+association+of+LEDGF%2Fp75+with+chromatin+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001340-Llano3" id="pone.0001340-Llano3"></a>Llano M, Delgado S, Vanegas M, Poeschla EM (2004) LEDGF/p75 prevents proteasomal degradation of HIV-1 integrase. J Biol Chem  279: 55570–55577.  <ul class="find" data-citedArticleID="1032524" data-doi="10.1074/jbc.m408508200"><li><a href="http://dx.doi.org/10.1074/jbc.m408508200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LEDGF%2Fp75+prevents+proteasomal+degradation+of+HIV-1+integrase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LEDGF%2Fp75+prevents+proteasomal+degradation+of+HIV-1+integrase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001340-Busschots1" id="pone.0001340-Busschots1"></a>Busschots K, Vercammen J, Emiliani S, Benarous R, Engelborghs Y, et al.  (2005) The interaction of LEDGF/p75 with integrase is lentivirus-specific and promotes DNA binding. J Biol Chem  280: 17841–17847.  <ul class="find" data-citedArticleID="1032460" data-doi="10.1074/jbc.m411681200"><li><a href="http://dx.doi.org/10.1074/jbc.m411681200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+interaction+of+LEDGF%2Fp75+with+integrase+is+lentivirus-specific+and+promotes+DNA+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+interaction+of+LEDGF%2Fp75+with+integrase+is+lentivirus-specific+and+promotes+DNA+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001340-Cherepanov3" id="pone.0001340-Cherepanov3"></a>Cherepanov P (2007) LEDGF/p75 interacts with divergent lentiviral integrases and modulates their enzymatic activity in vitro. Nucleic Acids Res  35: 113–124.  <ul class="find" data-citedArticleID="1032472" data-doi="10.1093/nar/gkl885"><li><a href="http://dx.doi.org/10.1093/nar/gkl885" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LEDGF%2Fp75+interacts+with+divergent+lentiviral+integrases+and+modulates+their+enzymatic+activity+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LEDGF%2Fp75+interacts+with+divergent+lentiviral+integrases+and+modulates+their+enzymatic+activity+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001340-Schroder1" id="pone.0001340-Schroder1"></a>Schroder AR, Shinn P, Chen H, Berry C, Ecker JR, et al.  (2002) HIV-1 integration in the human genome favors active genes and local hotspots. Cell  110: 521–529.  <ul class="find" data-citedArticleID="1032546" data-doi="10.1016/s0092-8674(02)00864-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)00864-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+integration+in+the+human+genome+favors+active+genes+and+local+hotspots." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+integration+in+the+human+genome+favors+active+genes+and+local+hotspots.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001340-Mitchell1" id="pone.0001340-Mitchell1"></a>Mitchell RS, Beitzel BF, Schroder AR, Shinn P, Chen H, et al.  (2004) Retroviral DNA integration: ASLV, HIV, and MLV show distinct target site preferences. PLoS Biol  2: e234.  <ul class="find" data-citedArticleID="1032536" data-doi="10.1371/journal.pbio.0020234"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020234" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroviral+DNA+integration%3A+ASLV%2C+HIV%2C+and+MLV+show+distinct+target+site+preferences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroviral+DNA+integration%3A+ASLV%2C+HIV%2C+and+MLV+show+distinct+target+site+preferences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001340-Hematti1" id="pone.0001340-Hematti1"></a>Hematti P, Hong BK, Ferguson C, Adler R, Hanawa H, et al.  (2004) Distinct genomic integration of MLV and SIV vectors in primate hematopoietic stem and progenitor cells. PLoS Biol  2: e423.  <ul class="find" data-citedArticleID="1032502" data-doi="10.1371/journal.pbio.0020423"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020423" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+genomic+integration+of+MLV+and+SIV+vectors+in+primate+hematopoietic+stem+and+progenitor+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+genomic+integration+of+MLV+and+SIV+vectors+in+primate+hematopoietic+stem+and+progenitor+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001340-Ciuffi1" id="pone.0001340-Ciuffi1"></a>Ciuffi A, Diamond T, Hwang Y, Marshall H, Bushman FD (2006) Fusions of LEDGF/p75 to lambda repressor promote HIV DNA integration near lambda operators in vitro. Human Gene Therapy  17: 960–967.  <ul class="find" data-citedArticleID="1032478"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fusions+of+LEDGF%2Fp75+to+lambda+repressor+promote+HIV+DNA+integration+near+lambda+operators+in+vitro.&amp;auth=&amp;atitle=Fusions+of+LEDGF%2Fp75+to+lambda+repressor+promote+HIV+DNA+integration+near+lambda+operators+in+vitro." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fusions+of+LEDGF%2Fp75+to+lambda+repressor+promote+HIV+DNA+integration+near+lambda+operators+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fusions+of+LEDGF%2Fp75+to+lambda+repressor+promote+HIV+DNA+integration+near+lambda+operators+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001340-Barr1" id="pone.0001340-Barr1"></a>Barr SD, Ciuffi A, Leipzig J, Shinn P, Ecker JR, et al.  (2006) HIV integration site selection: Targeting in macrophages and the effects of different routes of viral entry. Mol Ther  14: 218–225.  <ul class="find" data-citedArticleID="1032446" data-doi="10.1016/j.ymthe.2006.03.012"><li><a href="http://dx.doi.org/10.1016/j.ymthe.2006.03.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+integration+site+selection%3A+Targeting+in+macrophages+and+the+effects+of+different+routes+of+viral+entry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+integration+site+selection%3A+Targeting+in+macrophages+and+the+effects+of+different+routes+of+viral+entry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001340-Barr2" id="pone.0001340-Barr2"></a>Barr SD, Leipzig J, Shinn P, Ecker JR, Bushman FD (2005) Integration targeting by avian sarcoma-leukosis virus and human immunodeficiency virus in the chicken genome. J Virol  79: 12035–12044.  <ul class="find" data-citedArticleID="1032448" data-doi="10.1128/jvi.79.18.12035-12044.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.18.12035-12044.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integration+targeting+by+avian+sarcoma-leukosis+virus+and+human+immunodeficiency+virus+in+the+chicken+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integration+targeting+by+avian+sarcoma-leukosis+virus+and+human+immunodeficiency+virus+in+the+chicken+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001340-Crise1" id="pone.0001340-Crise1"></a>Crise B, Li Y, Yuan C, Morcock DR, Whitby D, et al.  (2005) Simian immunodeficiency virus integration preference is similar to that of human immunodeficiency virus type 1. J Virol  79: 12199–121204.  <ul class="find" data-citedArticleID="1032488" data-doi="10.1128/jvi.79.19.12199-12204.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.19.12199-12204.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simian+immunodeficiency+virus+integration+preference+is+similar+to+that+of+human+immunodeficiency+virus+type+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simian+immunodeficiency+virus+integration+preference+is+similar+to+that+of+human+immunodeficiency+virus+type+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001340-Lewinski2" id="pone.0001340-Lewinski2"></a>Lewinski MK, Yamashita M, Emerman M, Ciuffi A, Marshall H, et al.  (2006) Retroviral DNA integration: Viral and cellular determinants of target-site selection. PLoS Pathog  2: e60.  <ul class="find" data-citedArticleID="1032518" data-doi="10.1371/journal.ppat.0020060"><li><a href="http://dx.doi.org/10.1371/journal.ppat.0020060" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroviral+DNA+integration%3A+Viral+and+cellular+determinants+of+target-site+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroviral+DNA+integration%3A+Viral+and+cellular+determinants+of+target-site+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001340-Bushman2" id="pone.0001340-Bushman2"></a>Bushman F, Lewinski M, Ciuffi A, Barr S, Leipzig J, et al.  (2005) Genome-wide analysis of retroviral DNA integration. Nat Rev Microbiol  3: 848–858.  <ul class="find" data-citedArticleID="1032458" data-doi="10.1038/nrmicro1263"><li><a href="http://dx.doi.org/10.1038/nrmicro1263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+retroviral+DNA+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+retroviral+DNA+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001340-Kang1" id="pone.0001340-Kang1"></a>Kang Y, Moressi CJ, Scheetz TE, Xie L, Tran DT, et al.  (2006) Integration site choice of a feline immunodeficiency virus vector. J Virol  80: 8820–8823.  <ul class="find" data-citedArticleID="1032512" data-doi="10.1128/jvi.00719-06"><li><a href="http://dx.doi.org/10.1128/jvi.00719-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integration+site+choice+of+a+feline+immunodeficiency+virus+vector." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integration+site+choice+of+a+feline+immunodeficiency+virus+vector.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001340-Hacker1" id="pone.0001340-Hacker1"></a>Hacker CV, Vink CA, Wardell TW, Lee S, Treasure P, et al.  (2006) The integration profile of EIAV-based vectors. Mol Ther  14: 536–545.  <ul class="find" data-citedArticleID="1032500" data-doi="10.1016/j.ymthe.2006.06.006"><li><a href="http://dx.doi.org/10.1016/j.ymthe.2006.06.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+integration+profile+of+EIAV-based+vectors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+integration+profile+of+EIAV-based+vectors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001340-MacNeil1" id="pone.0001340-MacNeil1"></a>MacNeil A, Sankale JL, Meloni ST, Sarr AD, Mboup S, et al.  (2006) Genomic sites of human immunodeficiency virus type 2 (HIV-2) integration: Similarities to HIV-1 in vitro and possible differences in vivo. J Virol  80: 7316–7321.  <ul class="find" data-citedArticleID="1032528" data-doi="10.1128/jvi.00604-06"><li><a href="http://dx.doi.org/10.1128/jvi.00604-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+sites+of+human+immunodeficiency+virus+type+2+%28HIV-2%29+integration%3A+Similarities+to+HIV-1+in+vitro+and+possible+differences+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+sites+of+human+immunodeficiency+virus+type+2+%28HIV-2%29+integration%3A+Similarities+to+HIV-1+in+vitro+and+possible+differences+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001340-Cherepanov4" id="pone.0001340-Cherepanov4"></a>Cherepanov P, Ambrosio AL, Rahman S, Ellenberger T, Engelman A (2005) Structural basis for the recognition between HIV-1 integrase and transcriptional coactivator p75. Proc Natl Acad Sci USA  102: 17308–17313.  <ul class="find" data-citedArticleID="1032474" data-doi="10.1073/pnas.0506924102"><li><a href="http://dx.doi.org/10.1073/pnas.0506924102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+the+recognition+between+HIV-1+integrase+and+transcriptional+coactivator+p75." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+the+recognition+between+HIV-1+integrase+and+transcriptional+coactivator+p75.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001340-Cherepanov5" id="pone.0001340-Cherepanov5"></a>Cherepanov P, Sun ZY, Rahman S, Maertens G, Wagner G, et al.  (2005) Solution structure of the HIV integrase-binding domain in LEDGF/p75. Nat Struct Mol Biol  12: 526–532.  <ul class="find" data-citedArticleID="1032476" data-doi="10.1038/nsmb937"><li><a href="http://dx.doi.org/10.1038/nsmb937" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Solution+structure+of+the+HIV+integrase-binding+domain+in+LEDGF%2Fp75." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Solution+structure+of+the+HIV+integrase-binding+domain+in+LEDGF%2Fp75.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001340-Zielske1" id="pone.0001340-Zielske1"></a>Zielske SP, Stevenson M (2006) Modest but reproducible inhibition of human immunodeficiency virus type 1 infection in macrophages following LEDGFp75 silencing. J Virol  80: 7275–7280.  <ul class="find" data-citedArticleID="1032578" data-doi="10.1128/jvi.02470-05"><li><a href="http://dx.doi.org/10.1128/jvi.02470-05" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modest+but+reproducible+inhibition+of+human+immunodeficiency+virus+type+1+infection+in+macrophages+following+LEDGFp75+silencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modest+but+reproducible+inhibition+of+human+immunodeficiency+virus+type+1+infection+in+macrophages+following+LEDGFp75+silencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001340-Vandegraaff1" id="pone.0001340-Vandegraaff1"></a>Vandegraaff N, Devroe E, Turlure F, Silver PA, Engelman A (2006) Biochemical and genetic analyses of integrase-interacting protein lens epithelium-derived growth factor (LEDGF)/p75 and hepatoma-derived growth factor related protein 2 (HRP2) in preintegration complex function and HIV-1 replication. Virology  346: 415–426.  <ul class="find" data-citedArticleID="1032564" data-doi="10.1016/j.virol.2005.11.022"><li><a href="http://dx.doi.org/10.1016/j.virol.2005.11.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical+and+genetic+analyses+of+integrase-interacting+protein+lens+epithelium-derived+growth+factor+%28LEDGF%29%2Fp75+and+hepatoma-derived+growth+factor+related+protein+2+%28HRP2%29+in+preintegration+complex+function+and+HIV-1+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical+and+genetic+analyses+of+integrase-interacting+protein+lens+epithelium-derived+growth+factor+%28LEDGF%29%2Fp75+and+hepatoma-derived+growth+factor+related+protein+2+%28HRP2%29+in+preintegration+complex+function+and+HIV-1+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001340-Llano4" id="pone.0001340-Llano4"></a>Llano M, Saenz DT, Meehan A, Wongthida P, Peretz M, Walker WH, Teo W, Poeschla EM (2006) An Essential Role for LEDGF/p75 in HIV Integration. Science  314: 461–464.  <ul class="find" data-citedArticleID="1032526" data-doi="10.1126/science.1132319"><li><a href="http://dx.doi.org/10.1126/science.1132319" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Essential+Role+for+LEDGF%2Fp75+in+HIV+Integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Essential+Role+for+LEDGF%2Fp75+in+HIV+Integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001340-Vandekerckhove1" id="pone.0001340-Vandekerckhove1"></a>Vandekerckhove L, Christ F, Van Maele B, De Rijck J, Gijsbers R, et al.  (2006) Transient and stable knockdown of the integrase cofactor LEDGF/p75 reveals its role in the replication cycle of human immunodeficiency virus. J Virol  80: 1886–1896.  <ul class="find" data-citedArticleID="1032566" data-doi="10.1128/jvi.80.4.1886-1896.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.4.1886-1896.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transient+and+stable+knockdown+of+the+integrase+cofactor+LEDGF%2Fp75+reveals+its+role+in+the+replication+cycle+of+human+immunodeficiency+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transient+and+stable+knockdown+of+the+integrase+cofactor+LEDGF%2Fp75+reveals+its+role+in+the+replication+cycle+of+human+immunodeficiency+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001340-DeRijck1" id="pone.0001340-DeRijck1"></a>De Rijck J, Vandekerckhove L, Gijsbers R, Hombrouck A, Hendrix J, et al.  (2006) Overexpression of the lens epithelium-derived growth factor/p75 integrase binding domain inhibits human immunodeficiency virus replication. J Virol  80: 11498–11509.  <ul class="find" data-citedArticleID="1032490" data-doi="10.1128/jvi.00801-06"><li><a href="http://dx.doi.org/10.1128/jvi.00801-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overexpression+of+the+lens+epithelium-derived+growth+factor%2Fp75+integrase+binding+domain+inhibits+human+immunodeficiency+virus+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overexpression+of+the+lens+epithelium-derived+growth+factor%2Fp75+integrase+binding+domain+inhibits+human+immunodeficiency+virus+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001340-Ciuffi2" id="pone.0001340-Ciuffi2"></a>Ciuffi A, Llano M, Poeschla E, Hoffmann C, Leipzig J, et al.  (2005) A role for LEDGF/p75 in targeting HIV DNA integration. Nat Med  11: 1287–1289.  <ul class="find" data-citedArticleID="1032480" data-doi="10.1038/nm1329"><li><a href="http://dx.doi.org/10.1038/nm1329" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+role+for+LEDGF%2Fp75+in+targeting+HIV+DNA+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+role+for+LEDGF%2Fp75+in+targeting+HIV+DNA+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001340-Shun1" id="pone.0001340-Shun1"></a>Shun MC, Raghavendra NK, Vandegraaff N, Daigle JE, Hughes S, et al.  (2007) LEDGF/p75 functions downstream from preintegration complex formation to effect gene-specific HIV-1 integration. Genes Dev  21: 1767–1778.  <ul class="find" data-citedArticleID="1032550" data-doi="10.1101/gad.1565107"><li><a href="http://dx.doi.org/10.1101/gad.1565107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LEDGF%2Fp75+functions+downstream+from+preintegration+complex+formation+to+effect+gene-specific+HIV-1+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LEDGF%2Fp75+functions+downstream+from+preintegration+complex+formation+to+effect+gene-specific+HIV-1+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001340-Sutherland1" id="pone.0001340-Sutherland1"></a>Sutherland HG, Newton K, Brownstein DG, Holmes MC, Kress C, et al.  (2006) Disruption of Ledgf/Psip1 results in perinatal mortality and homeotic skeletal transformations. Mol Cell Biol  26: 7201–7210.  <ul class="find" data-citedArticleID="1032556" data-doi="10.1128/mcb.00459-06"><li><a href="http://dx.doi.org/10.1128/mcb.00459-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Disruption+of+Ledgf%2FPsip1+results+in+perinatal+mortality+and+homeotic+skeletal+transformations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Disruption+of+Ledgf%2FPsip1+results+in+perinatal+mortality+and+homeotic+skeletal+transformations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001340-Butler1" id="pone.0001340-Butler1"></a>Butler SL, Johnson EP, Bushman FD (2002) HIV cDNA metabolism studied by fluorescence-monitored PCR: Notable stability of two-LTR circles. J Virol  76: 3739–3747.  <ul class="find" data-citedArticleID="1032462" data-doi="10.1128/jvi.76.8.3739-3747.2002"><li><a href="http://dx.doi.org/10.1128/jvi.76.8.3739-3747.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+cDNA+metabolism+studied+by+fluorescence-monitored+PCR%3A+Notable+stability+of+two-LTR+circles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+cDNA+metabolism+studied+by+fluorescence-monitored+PCR%3A+Notable+stability+of+two-LTR+circles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001340-Butler2" id="pone.0001340-Butler2"></a>Butler S, Hansen M, Bushman FD (2001) A quantitative assay for HIV cDNA integration in vivo. Nat Med  7: 631–634.  <ul class="find" data-citedArticleID="1032464" data-doi="10.1038/87979"><li><a href="http://dx.doi.org/10.1038/87979" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+quantitative+assay+for+HIV+cDNA+integration+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+quantitative+assay+for+HIV+cDNA+integration+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001340-Margulies1" id="pone.0001340-Margulies1"></a>Margulies M, Egholm M, Altman WE, Attiya S, Bader JS, et al.  (2005) Genome sequencing in microfabricated high-density picolitre reactors. Nature  437: 376–380.  <ul class="find" data-citedArticleID="1032532"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.&amp;auth=&amp;atitle=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001340-Wang1" id="pone.0001340-Wang1"></a>Wang GP, Ciuffi A, Leipzig J, Berry CC, Bushman FD (2007) HIV integration site selection: Analysis by massively parallel pyrosequencing reveals association with epigenetic modifications. Genome Res  17: 1186–1194.  <ul class="find" data-citedArticleID="1032570" data-doi="10.1101/gr.6286907"><li><a href="http://dx.doi.org/10.1101/gr.6286907" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+integration+site+selection%3A+Analysis+by+massively+parallel+pyrosequencing+reveals+association+with+epigenetic+modifications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+integration+site+selection%3A+Analysis+by+massively+parallel+pyrosequencing+reveals+association+with+epigenetic+modifications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001340-Shoemaker1" id="pone.0001340-Shoemaker1"></a>Shoemaker DD, Lashkari DA, Morris D, Mittmann M, Davis RW (1996) Quantitative phenotypic analysis of yeast deletion mutants using a highly parallel molecular bar-coding strategy. Nat Genet  14: 450–456.  <ul class="find" data-citedArticleID="1032548" data-doi="10.1038/ng1296-450"><li><a href="http://dx.doi.org/10.1038/ng1296-450" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+phenotypic+analysis+of+yeast+deletion+mutants+using+a+highly+parallel+molecular+bar-coding+strategy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+phenotypic+analysis+of+yeast+deletion+mutants+using+a+highly+parallel+molecular+bar-coding+strategy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001340-Hoffmann1" id="pone.0001340-Hoffmann1"></a>Hoffmann C, Minkah N, Leipzig J, Wang G, Arens MQ, et al.  (2007) DNA bar coding and pyrosequencing to identify rare HIV drug resistance mutations. Nucleic Acids Res  35: e91.  <ul class="find" data-citedArticleID="1032504" data-doi="10.1093/nar/gkm435"><li><a href="http://dx.doi.org/10.1093/nar/gkm435" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+bar+coding+and+pyrosequencing+to+identify+rare+HIV+drug+resistance+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+bar+coding+and+pyrosequencing+to+identify+rare+HIV+drug+resistance+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001340-Binladen1" id="pone.0001340-Binladen1"></a>Binladen J, Gilbert MT, Bollback JP, Panitz F, Bendixen C, et al.  (2007) The use of coded PCR primers enables high-throughput sequencing of multiple homolog amplification products by 454 parallel sequencing. PLoS ONE  2: e197.  <ul class="find" data-citedArticleID="1032452" data-doi="10.1371/journal.pone.0000197"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+coded+PCR+primers+enables+high-throughput+sequencing+of+multiple+homolog+amplification+products+by+454+parallel+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+coded+PCR+primers+enables+high-throughput+sequencing+of+multiple+homolog+amplification+products+by+454+parallel+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001340-Pryciak1" id="pone.0001340-Pryciak1"></a>Pryciak PM, Sil A, Varmus HE (1992) Retroviral integration into minichromosomes <em>in vitro</em>. EMBO J  11: 291–303.  <ul class="find" data-citedArticleID="1032544"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Retroviral+integration+into+minichromosomes+in+vitro.&amp;auth=&amp;atitle=Retroviral+integration+into+minichromosomes+in+vitro." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroviral+integration+into+minichromosomes+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroviral+integration+into+minichromosomes+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001340-Stevens1" id="pone.0001340-Stevens1"></a>Stevens SW, Griffith JD (1996) Sequence analysis of the human DNA flanking sites of human immunodeficiency virus type 1 integration. J Virol  70: 6459–6462.  <ul class="find" data-citedArticleID="1032552"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sequence+analysis+of+the+human+DNA+flanking+sites+of+human+immunodeficiency+virus+type+1+integration.&amp;auth=&amp;atitle=Sequence+analysis+of+the+human+DNA+flanking+sites+of+human+immunodeficiency+virus+type+1+integration." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+analysis+of+the+human+DNA+flanking+sites+of+human+immunodeficiency+virus+type+1+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+analysis+of+the+human+DNA+flanking+sites+of+human+immunodeficiency+virus+type+1+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001340-Carteau1" id="pone.0001340-Carteau1"></a>Carteau S, Hoffmann C, Bushman FD (1998) Chromosome structure and HIV-1 cDNA integration: Centromeric alphoid repeats are a disfavored target. J Virol  72: 4005–4014.  <ul class="find" data-citedArticleID="1032466"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chromosome+structure+and+HIV-1+cDNA+integration%3A+Centromeric+alphoid+repeats+are+a+disfavored+target.&amp;auth=&amp;atitle=Chromosome+structure+and+HIV-1+cDNA+integration%3A+Centromeric+alphoid+repeats+are+a+disfavored+target." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome+structure+and+HIV-1+cDNA+integration%3A+Centromeric+alphoid+repeats+are+a+disfavored+target." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome+structure+and+HIV-1+cDNA+integration%3A+Centromeric+alphoid+repeats+are+a+disfavored+target.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001340-Wu1" id="pone.0001340-Wu1"></a>Wu X, Li Y, Crise B, Burgess SM, Munroe DJ (2005) Weak palindromic consensus sequences are a common feature found at the integration target sites of many retroviruses. J Virol  79: 5211–5214.  <ul class="find" data-citedArticleID="1032572" data-doi="10.1128/jvi.79.8.5211-5214.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.8.5211-5214.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Weak+palindromic+consensus+sequences+are+a+common+feature+found+at+the+integration+target+sites+of+many+retroviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Weak+palindromic+consensus+sequences+are+a+common+feature+found+at+the+integration+target+sites+of+many+retroviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001340-Holman1" id="pone.0001340-Holman1"></a>Holman AG, Coffin JM (2005) Symmetrical base preferences surrounding HIV-1, avian sarcoma/leukosis virus, and murine leukemia virus integration sites. Proc Natl Acad Sci USA  102: 6103–6107.  <ul class="find" data-citedArticleID="1032506" data-doi="10.1073/pnas.0501646102"><li><a href="http://dx.doi.org/10.1073/pnas.0501646102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Symmetrical+base+preferences+surrounding+HIV-1%2C+avian+sarcoma%2Fleukosis+virus%2C+and+murine+leukemia+virus+integration+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Symmetrical+base+preferences+surrounding+HIV-1%2C+avian+sarcoma%2Fleukosis+virus%2C+and+murine+leukemia+virus+integration+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001340-Berry1" id="pone.0001340-Berry1"></a>Berry C, Hannenhalli S, Leipzig J, Bushman FD (2006) Selection of target sites for mobile DNA integration in the human genome. PLoS Comput Biol  2: e157.  <ul class="find" data-citedArticleID="1032450" data-doi="10.1371/journal.pcbi.0020157"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+of+target+sites+for+mobile+DNA+integration+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+of+target+sites+for+mobile+DNA+integration+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001340-Ciuffi3" id="pone.0001340-Ciuffi3"></a>Ciuffi A, Diamond TL, Hwang Y, Marshall HM, Bushman FD (2006) Modulating target site selection during human immunodeficiency virus DNA integration in vitro with an engineered tethering factor. Hum Gene Ther  17: 960–967.  <ul class="find" data-citedArticleID="1032482" data-doi="10.1089/hum.2006.17.960"><li><a href="http://dx.doi.org/10.1089/hum.2006.17.960" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulating+target+site+selection+during+human+immunodeficiency+virus+DNA+integration+in+vitro+with+an+engineered+tethering+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulating+target+site+selection+during+human+immunodeficiency+virus+DNA+integration+in+vitro+with+an+engineered+tethering+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001340-Wu2" id="pone.0001340-Wu2"></a>Wu X, Li Y, Crise B, Burgess SM (2003) Transcription start regions in the human genome are favored targets for MLV integration. Science  300: 1749–1751.  <ul class="find" data-citedArticleID="1032574" data-doi="10.1126/science.1083413"><li><a href="http://dx.doi.org/10.1126/science.1083413" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+start+regions+in+the+human+genome+are+favored+targets+for+MLV+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+start+regions+in+the+human+genome+are+favored+targets+for+MLV+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001340-Miller1" id="pone.0001340-Miller1"></a>Miller DG, Trobridge GD, Petek LM, Jacobs MA, Kaul R, et al.  (2005) Large-scale analysis of adeno-associated virus vector integration sites in normal human cells. J Virol  79: 11434–11442.  <ul class="find" data-citedArticleID="1032534" data-doi="10.1128/jvi.79.17.11434-11442.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.17.11434-11442.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+analysis+of+adeno-associated+virus+vector+integration+sites+in+normal+human+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+analysis+of+adeno-associated+virus+vector+integration+sites+in+normal+human+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001340-Molteni1" id="pone.0001340-Molteni1"></a>Molteni V, Greenwald J, Rhodes D, Hwang Y, Kwiatkowski W, et al.  (2001) Identification of a small molecule binding site at the dimer interface of the HIV integrase catalytic domain. Acta Crystallogr D Biol Crystallogr  57: 536–544.  <ul class="find" data-citedArticleID="1032538" data-doi="10.1107/s0907444901001652"><li><a href="http://dx.doi.org/10.1107/s0907444901001652" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+small+molecule+binding+site+at+the+dimer+interface+of+the+HIV+integrase+catalytic+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+small+molecule+binding+site+at+the+dimer+interface+of+the+HIV+integrase+catalytic+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001340-Stone1" id="pone.0001340-Stone1"></a>Stone M (1974) Cross-validatory chosice and assesment of statistical predictions. Journal of the Royal Statistical Society, Series B–Methodological  36: 111–147.  <ul class="find" data-citedArticleID="1032554"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cross-validatory+chosice+and+assesment+of+statistical+predictions.&amp;auth=&amp;atitle=Cross-validatory+chosice+and+assesment+of+statistical+predictions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cross-validatory+chosice+and+assesment+of+statistical+predictions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cross-validatory+chosice+and+assesment+of+statistical+predictions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001340-Todaro1" id="pone.0001340-Todaro1"></a>Todaro G, Green H (1963) Quantitative studies of the growth of mouse embryo cells in culture and their development into established lines. J Cell Biol  17: 299–313.  <ul class="find" data-citedArticleID="1032558" data-doi="10.1083/jcb.17.2.299"><li><a href="http://dx.doi.org/10.1083/jcb.17.2.299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+studies+of+the+growth+of+mouse+embryo+cells+in+culture+and+their+development+into+established+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+studies+of+the+growth+of+mouse+embryo+cells+in+culture+and+their+development+into+established+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001340-Follenzi1" id="pone.0001340-Follenzi1"></a>Follenzi A, Ailes LE, Bakovic S, Gueuna M, Naldini L (2000) Gene transfer by lentiviral vectors is limited by nuclear translocation and rescued by HIV-1 pol sequences. Nat Genetics  25: 217–222.  <ul class="find" data-citedArticleID="1032494"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+transfer+by+lentiviral+vectors+is+limited+by+nuclear+translocation+and+rescued+by+HIV-1+pol+sequences.&amp;auth=&amp;atitle=Gene+transfer+by+lentiviral+vectors+is+limited+by+nuclear+translocation+and+rescued+by+HIV-1+pol+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+transfer+by+lentiviral+vectors+is+limited+by+nuclear+translocation+and+rescued+by+HIV-1+pol+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+transfer+by+lentiviral+vectors+is+limited+by+nuclear+translocation+and+rescued+by+HIV-1+pol+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001340-Naldini1" id="pone.0001340-Naldini1"></a>Naldini L, Blomer U, Gallay P, Ory D, Mulligan R, et al.  (1996) In vivo gene delivery and stable transduction of nondividing cells by a lentiviral vector. Science  272: 263–267.  <ul class="find" data-citedArticleID="1032540" data-doi="10.1126/science.272.5259.263"><li><a href="http://dx.doi.org/10.1126/science.272.5259.263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+gene+delivery+and+stable+transduction+of+nondividing+cells+by+a+lentiviral+vector." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+gene+delivery+and+stable+transduction+of+nondividing+cells+by+a+lentiviral+vector.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001340-Olsen1" id="pone.0001340-Olsen1"></a>Olsen JC (1998) Gene transfer vectors derived from equine infectious anemia virus. Gene Ther  5: 1481–1487.  <ul class="find" data-citedArticleID="1032542" data-doi="10.1038/sj.gt.3300768"><li><a href="http://dx.doi.org/10.1038/sj.gt.3300768" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+transfer+vectors+derived+from+equine+infectious+anemia+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+transfer+vectors+derived+from+equine+infectious+anemia+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001340-Yamashita1" id="pone.0001340-Yamashita1"></a>Yamashita M, Emerman M (2004) Capsid is a dominant determinant of retrovirus infectivity in nondividing cells. J Virol  78: 5670–5678.  <ul class="find" data-citedArticleID="1032576" data-doi="10.1128/jvi.78.11.5670-5678.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.11.5670-5678.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Capsid+is+a+dominant+determinant+of+retrovirus+infectivity+in+nondividing+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Capsid+is+a+dominant+determinant+of+retrovirus+infectivity+in+nondividing+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001340-Ciuffi4" id="pone.0001340-Ciuffi4"></a>Ciuffi A, Mitchell RS, Hoffmann C, Leipzig J, Shinn P, et al.  (2006) Integration site selection by HIV-based vectors in dividing and growth-arrested IMR-90 lung fibroblasts. Mol Ther  13: 366–373.  <ul class="find" data-citedArticleID="1032484" data-doi="10.1016/j.ymthe.2005.10.009"><li><a href="http://dx.doi.org/10.1016/j.ymthe.2005.10.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integration+site+selection+by+HIV-based+vectors+in+dividing+and+growth-arrested+IMR-90+lung+fibroblasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integration+site+selection+by+HIV-based+vectors+in+dividing+and+growth-arrested+IMR-90+lung+fibroblasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001340-Levine1" id="pone.0001340-Levine1"></a>Levine BL, Humeau LM, Boyer J, MacGregor RR, Rebello T, et al.  (2006) Gene transfer in humans using a conditionally replicating lentiviral vector. Proc Natl Acad Sci USA  103: 17372–17377.  <ul class="find" data-citedArticleID="1032514" data-doi="10.1073/pnas.0608138103"><li><a href="http://dx.doi.org/10.1073/pnas.0608138103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+transfer+in+humans+using+a+conditionally+replicating+lentiviral+vector." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+transfer+in+humans+using+a+conditionally+replicating+lentiviral+vector.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.XML" value="139043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.PDF" value="437139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g001.PNG_L" value="78683"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g001.PNG_M" value="103149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g001.PNG_S" value="11166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g001.TIF" value="480060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g001.PNG_I" value="37766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g002.PNG_L" value="48290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g002.PNG_M" value="59306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g002.PNG_S" value="7307"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g002.TIF" value="265780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g002.PNG_I" value="22527"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t001.PNG_L" value="197619"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t001.PNG_M" value="118327"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t001.PNG_S" value="12022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t001.TIF" value="808958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t001.PNG_I" value="35883"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g003.PNG_L" value="71937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g003.PNG_M" value="138179"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g003.PNG_S" value="13327"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g003.TIF" value="343158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g003.PNG_I" value="50804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g004.PNG_L" value="79150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g004.PNG_M" value="97016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g004.PNG_S" value="13546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g004.TIF" value="659102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g004.PNG_I" value="44447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t002.PNG_L" value="95746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t002.PNG_M" value="55510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t002.PNG_S" value="8978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t002.TIF" value="425654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t002.PNG_I" value="17352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g005.PNG_L" value="24846"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g005.PNG_M" value="72252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g005.PNG_S" value="10143"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g005.TIF" value="129318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g005.PNG_I" value="27424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g006.PNG_L" value="74396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g006.PNG_M" value="94928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g006.PNG_S" value="8065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g006.TIF" value="341010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g006.PNG_I" value="34040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g007.PNG_L" value="55862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g007.PNG_M" value="90359"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g007.PNG_S" value="13911"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g007.TIF" value="352988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g007.PNG_I" value="50380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t003.PNG_L" value="99335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t003.PNG_M" value="59168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t003.PNG_S" value="9526"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t003.TIF" value="457928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.t003.PNG_I" value="18480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g008.PNG_L" value="55482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g008.PNG_M" value="85071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g008.PNG_S" value="13605"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g008.TIF" value="372036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g008.PNG_I" value="51125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g009.PNG_L" value="31712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g009.PNG_M" value="67836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g009.PNG_S" value="8370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g009.TIF" value="132228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.g009.PNG_I" value="26035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.s001.XLS" value="11264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.s002.PDF" value="351150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001340.s003.PDF" value="1435992"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001340&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001340" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001340&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001340&volume=&issue=&title=Role of PSIP1/LEDGF/p75 in Lentiviral Infectivity and Integration Targeting&author_name=Heather%20M.%20Marshall%2C%20Keshet%20Ronen%2C%20Charles%20Berry%2C%20Manuel%20Llano%2C%20Heidi%20Sutherland%2C%20Dyana%20Saenz%2C%20Wendy%20Bickmore%2C%20Eric%20Poeschla%2C%20Frederic%20D.%20Bushman&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001340" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001340" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001340" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001340&amp;t=Role%20of%20PSIP1%2FLEDGF%2Fp75%20in%20Lentiviral%20Infectivity%20and%20Integration%20Targeting" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001340&title=Role%20of%20PSIP1%2FLEDGF%2Fp75%20in%20Lentiviral%20Infectivity%20and%20Integration%20Targeting&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001340&amp;title=Role%20of%20PSIP1%2FLEDGF%2Fp75%20in%20Lentiviral%20Infectivity%20and%20Integration%20Targeting" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001340" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001340&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Role of PSIP1/LEDGF/p75 in Lentiviral Infectivity and Integration Targeting';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001340';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Role%20of%20PSIP1%2FLEDGF%2Fp75%20in%20Lentiviral%20Infectivity%20and%20Integration%20Targeting http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001340" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001340" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">





















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22293T+cells%22" title="Search for articles in the subject area:'293T cells'"><div class="flagText">293T cells</div></a>
              <div data-categoryid="20253" data-articleid="25510"
                   data-categoryname="293T cells"
                   class="flagImage" title="Flag '293T cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+methylation%22" title="Search for articles in the subject area:'DNA methylation'"><div class="flagText">DNA methylation</div></a>
              <div data-categoryid="32275" data-articleid="25510"
                   data-categoryname="DNA methylation"
                   class="flagImage" title="Flag 'DNA methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25510"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="25510"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV%22" title="Search for articles in the subject area:'HIV'"><div class="flagText">HIV</div></a>
              <div data-categoryid="42663" data-articleid="25510"
                   data-categoryname="HIV"
                   class="flagImage" title="Flag 'HIV' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV+infections%22" title="Search for articles in the subject area:'HIV infections'"><div class="flagText">HIV infections</div></a>
              <div data-categoryid="47749" data-articleid="25510"
                   data-categoryname="HIV infections"
                   class="flagImage" title="Flag 'HIV infections' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="25510"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+vector+techniques%22" title="Search for articles in the subject area:'Viral vector techniques'"><div class="flagText">Viral vector techniques</div></a>
              <div data-categoryid="50687" data-articleid="25510"
                   data-categoryname="Viral vector techniques"
                   class="flagImage" title="Flag 'Viral vector techniques' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5281'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9571'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2515&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
