

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Global Genetic Population Structure of Bacillus anthracis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000461"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000461" />

    <meta name="citation_title" content="Global Genetic Population Structure of Bacillus anthracis"/>
    <meta itemprop="name" content="Global Genetic Population Structure of Bacillus anthracis"/>

      <meta name="citation_author" content="Matthew N. Van Ert"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="W. Ryan Easterday"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Lynn Y. Huynh"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Richard T. Okinaka"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
            <meta name="citation_author_institution" content="Biosciences, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America"/>
      <meta name="citation_author" content="Martin E. Hugh-Jones"/>
            <meta name="citation_author_institution" content="Department of Environmental Studies, Louisiana State University, Baton Rouge, Louisiana, United States of America"/>
      <meta name="citation_author" content="Jacques Ravel"/>
            <meta name="citation_author_institution" content="The Institute for Genomic Research, Rockville, Maryland, United States of America"/>
      <meta name="citation_author" content="Shaylan R. Zanecki"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Talima Pearson"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Tatum S. Simonson"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Jana M. U'Ren"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Sergey M. Kachur"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Rebecca R. Leadem-Dougherty"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Shane D. Rhoton"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Guenevier Zinser"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Jason Farlow"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Pamala R. Coker"/>
            <meta name="citation_author_institution" content="Department of Environmental Studies, Louisiana State University, Baton Rouge, Louisiana, United States of America"/>
      <meta name="citation_author" content="Kimothy L. Smith"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Bingxiang Wang"/>
            <meta name="citation_author_institution" content="Lanzhou Institute of Biological Products, Lanzhou, China"/>
      <meta name="citation_author" content="Leo J. Kenefic"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Claire M. Fraser-Liggett"/>
            <meta name="citation_author_institution" content="The Institute for Genomic Research, Rockville, Maryland, United States of America"/>
      <meta name="citation_author" content="David M. Wagner"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Paul Keim"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
            <meta name="citation_author_institution" content="Biosciences, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America"/>
            <meta name="citation_author_institution" content="Pathogen Genomics Division, Translational Genomics Research Institute, Phoenix, Arizona, United States of America"/>

    <meta name="citation_date" content="2007/5/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000461.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e461"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Molecular subtyping of Bacillus anthracis and the 2001 bioterrorism-associated anthrax outbreak, United States.; citation_author=AR Hoffmaster; citation_author=CC Fitzgerald; citation_author=E Ribot; citation_author=LW Mayer; citation_author=T Popovic; citation_journal_title=Emerg Infect Dis; citation_volume=8; citation_number=1; citation_pages=1111-1116; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular investigation of the Aum Shinrikyo anthrax release in Kameido, Japan.; citation_author=P Keim; citation_author=KL Smith; citation_author=C Keys; citation_author=H Takahashi; citation_author=T Kurata; citation_journal_title=J Clin Microbiol; citation_volume=39; citation_number=2; citation_pages=4566-4567; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Anthrax molecular epidemiology and forensics: using the appropriate marker for different evolutionary scales.; citation_author=P Keim; citation_author=MN Van Ert; citation_author=T Pearson; citation_author=AJ Vogler; citation_author=LY Huynh; citation_journal_title=Infect Genet Evol; citation_volume=4; citation_number=3; citation_pages=205-213; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Multiple-locus variable-number tandem repeat analysis reveals genetic relationships within Bacillus anthracis.; citation_author=P Keim; citation_author=LB Price; citation_author=AM Klevytska; citation_author=KL Smith; citation_author=JM Schupp; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=4; citation_pages=2928-2936; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic discovery bias in Bacillus anthracis using single-nucleotide polymorphisms from whole-genome sequencing.; citation_author=T Pearson; citation_author=JD Busch; citation_author=J Ravel; citation_author=TD Read; citation_author=SD Rhoton; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=5; citation_pages=13536-13541; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Diversity among French Bacillus anthracis isolates.; citation_author=A Fouet; citation_author=KL Smith; citation_author=C Keys; citation_author=J Vaissaire; citation_author=C Le Doujet; citation_journal_title=J Clin Microbiol; citation_volume=40; citation_number=6; citation_pages=4732-4734; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Intriguing diversity of Bacillus anthracis in eastern Poland–the molecular echoes of the past outbreaks.; citation_author=R Gierczynski; citation_author=S Kaluzewski; citation_author=A Rakin; citation_author=M Jagielski; citation_author=A Zasada; citation_journal_title=FEMS Microbiol Lett; citation_volume=239; citation_number=7; citation_pages=235-240; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular diversity of Bacillus anthracis in Italy.; citation_author=A Fasanella; citation_author=M Van Ert; citation_author=SA Altamura; citation_author=G Garofolo; citation_author=C Buonavoglia; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=8; citation_pages=3398-3401; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bacillus anthracis diversity in Kruger National Park.; citation_author=KL Smith; citation_author=V DeVos; citation_author=H Bryden; citation_author=LB Price; citation_author=ME Hugh-Jones; citation_journal_title=J Clin Microbiol; citation_volume=38; citation_number=9; citation_pages=3780-3784; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Antibiotic susceptibility and molecular diversity of Bacillus anthracis strains in Chad: detection of a new phylogenetic subgroup.; citation_author=A Maho; citation_author=A Rossano; citation_author=H Hachler; citation_author=A Holzer; citation_author=E Schelling; citation_journal_title=J Clin Microbiol; citation_volume=44; citation_number=10; citation_pages=3422-3425; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Strain-Specific Single-Nucleotide Polymorphism Assays for the Bacillus anthracis Ames Strain.; citation_author=MN Van Ert; citation_author=WR Easterday; citation_author=TS Simonson; citation_author=JM U'Ren; citation_author=T Pearson; citation_journal_title=J Clin Microbiol; citation_volume=45; citation_number=11; citation_pages=47-53; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Methicillin-resistant Staphylococcus aureus genotyping using a small set of polymorphisms.; citation_author=AJ Stephens; citation_author=F Huygens; citation_author=J Inman-Bamber; citation_author=EP Price; citation_author=GR Nimmo; citation_journal_title=J Med Microbiol; citation_volume=55; citation_number=12; citation_pages=43-51; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comparative genome sequencing for discovery of novel polymorphisms in Bacillus anthracis.; citation_author=TD Read; citation_author=SL Salzberg; citation_author=M Pop; citation_author=M Shumway; citation_author=L Umayam; citation_journal_title=Science; citation_volume=296; citation_number=13; citation_pages=2028-2033; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The International HapMap Consortium: A haplotype map of the human genome.; citation_author=D Altshuler; citation_author=LD Brooks; citation_author=A Chakravarti; citation_author=FS Collins; citation_author=MJ Daly; citation_journal_title=Nature; citation_volume=437; citation_number=14; citation_pages=1299-1320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Distinct mutations in PlcR explain why some strains of the Bacillus cereus group are nonhemolytic.; citation_author=L Slamti; citation_author=S Perchat; citation_author=M Gominet; citation_author=G Vilas-Boas; citation_author=A Fouet; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=15; citation_pages=3531-3538; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Use of single nucleotide polymorphisms in the plcR gene for specific identification of Bacillus anthracis.; citation_author=WR Easterday; citation_author=MN Van Ert; citation_author=TS Simonson; citation_author=DM Wagner; citation_author=LJ Kenefic; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=16; citation_pages=1995-1997; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Anthrax but not Bacillus anthracis?; citation_author=RT Okinaka; citation_author=T Pearson; citation_author=PS Keim; citation_journal_title=PLoS-Pathogens; citation_volume=2; citation_number=17; citation_pages=1025-1027; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolution of Bacillus anthracis, causative agent of anthrax.; citation_author=PS Keim; citation_author=T Pearson; citation_author=RT Okinaka; citation_number=18; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Ecological constraints on diversification in a model adaptive radiation.; citation_author=R Kassen; citation_author=M Llewellyn; citation_author=PB Rainey; citation_journal_title=Nature; citation_volume=431; citation_number=19; citation_pages=984-988; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A History of Anthrax.; citation_author=DM Klemm; citation_author=WR Klemm; citation_journal_title=J Am Vet Med Assoc; citation_volume=135; citation_number=20; citation_pages=458-462; citation_date=1959; " />
      <meta name="citation_reference" content="citation_title=Evolution of anthrax, report II: History of the spread of the disease.; citation_author=GV Kolonin; citation_journal_title=Zhurnal Mikrobiol Epidemiol; citation_volume=48; citation_number=21; citation_pages=118-122; citation_date=1971; " />
      <meta name="citation_reference" content="citation_author=J Diamond; citation_number=22; citation_date=1997; citation_publisher=W.W. Norton and Company; " />
      <meta name="citation_reference" content="citation_title=Evolution, consequences and future of plant and animal domestication.; citation_author=J Diamond; citation_journal_title=Nature; citation_volume=418; citation_number=23; citation_pages=700; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mitochodrial diversity and the origins of African and European cattle.; citation_author=DG Bradley; citation_author=DE MacHugh; citation_author=P Cunningham; citation_author=RT Loftus; citation_journal_title=PNAS; citation_volume=93; citation_number=24; citation_pages=5131-5135; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Ecology of anthrax.; citation_author=GB Van Ness; citation_journal_title=Science; citation_volume=172; citation_number=25; citation_pages=1303-1307; citation_date=1971; " />
      <meta name="citation_reference" content="citation_title=The earliest account of anthrax in man and animals in North America.; citation_author=RP Hanson; citation_journal_title=J Am Vet Mde Assoc; citation_volume=135; citation_number=26; citation_date=1959; " />
      <meta name="citation_reference" content="citation_title=A review of anthrax in Canada and implications for research on the disease in northern bison.; citation_author=DC Dragon; citation_author=BT Elkin; citation_author=JS Nishi; citation_author=TR Ellsworth; citation_journal_title=J Appl Microbiol; citation_volume=87; citation_number=27; citation_pages=208-213; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Milroy Lectures on industrial anthrax.; citation_author=TM Legge; citation_journal_title=Lancet; citation_volume=165; citation_number=28; citation_pages=842; citation_date=1905; " />
      <meta name="citation_reference" content="citation_title=Epidemiology of a small outbreak of anthrax.; citation_author=AT Doig; citation_author=JS Gemmill; citation_journal_title=Lancet; citation_volume=1; citation_number=29; citation_pages=1011-1012; citation_date=1951; " />
      <meta name="citation_reference" content="citation_title=Anthrax and bone-meal fertiliser.; citation_author=WM Jamieson; citation_author=DM Green; citation_journal_title=Lancet; citation_volume=268; citation_number=30; citation_pages=560; citation_date=1955; " />
      <meta name="citation_reference" content="citation_title=Anthrax and bone-meal fertiliser.; citation_author=DM Green; citation_author=WM Jamieson; citation_journal_title=Lancet; citation_volume=2; citation_number=31; citation_pages=153-154; citation_date=1958; " />
      <meta name="citation_reference" content="citation_author=W Geering; citation_number=32; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Identifying sources of human exposure to plague.; citation_author=JL Lowell; citation_author=DM Wagner; citation_author=B Atshabar; citation_author=MF Antolin; citation_author=AJ Vogler; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=33; citation_pages=650-656; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Characterization of a Bacillus anthracis isolate causing a rare case of fatal anthrax in a 2-year-old boy from Hong Kong.; citation_author=DT Cheung; citation_author=KM Kam; citation_author=KL Hau; citation_author=TK Au; citation_author=CK Marston; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=34; citation_pages=1992-1994; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Anthrax and the art of war (against ascertainment bias).; citation_author=M Worobey; citation_journal_title=Heredity; citation_volume=94; citation_number=35; citation_pages=459-460; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The incompatibility between the PlcR- and AtxA-controlled regulons may have selected a nonsense mutation in Bacillus anthracis.; citation_author=T Mignot; citation_author=M Mock; citation_author=D Robichon; citation_author=A Landier; citation_author=D Lereclus; citation_journal_title=Mol Microbiol; citation_volume=42; citation_number=36; citation_pages=1189-1198; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=High-throughput single nucleotide polymorphism genotyping by fluorescent 5′ exonuclease activity.; citation_author=PA Morin; citation_author=R Saiz; citation_author=A Monjazeb; citation_journal_title=Biotechniques; citation_volume=27; citation_number=37; citation_pages=538-552; citation_date=1999; " />
      <meta name="citation_reference" content="citation_author=G Zinser; citation_number=38; citation_pages=34; citation_date=2002; citation_publisher=Northern Arizona University; " />
      <meta name="citation_reference" content="citation_title=MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Brief Bioinform; citation_volume=5; citation_number=39; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular analysis of rifampin resistance in Bacillus anthracis and Bacillus cereus.; citation_author=AJ Vogler; citation_author=JD Busch; citation_author=S Percy-Fine; citation_author=C Tipton-Hunton; citation_author=KL Smith; citation_journal_title=Antimicrob Agents Chemother; citation_volume=46; citation_number=40; citation_pages=511-513; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The distribution of rates of spontaneous mutation over viruses, prokaryotes, and eukaryotes.; citation_author=JW Drake; citation_journal_title=Ann N Y Acad Sci; citation_volume=870; citation_number=41; citation_pages=100-107; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Rates of Spontaneous Mutation.; citation_author=JW Drake; citation_author=B Charlesworth; citation_author=D Charlesworth; citation_author=JF Crow; citation_journal_title=Genetics; citation_volume=148; citation_number=42; citation_pages=1667-1686; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=In vivo growth and distribution of anthrax bacilli in resistant, susceptible, and immunized hosts.; citation_author=WI Jones; citation_author=F Klein; citation_author=JS Walker; citation_author=BG Mahlandt; citation_author=JP Dobbs; citation_journal_title=J Bacteriol; citation_volume=94; citation_number=43; citation_pages=600-608; citation_date=1967; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000461" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Global Genetic Population Structure of Bacillus anthracis"/>
    <meta name="twitter:description" content="Anthrax, caused by the bacterium Bacillus anthracis, is a disease of historical and current importance that is found throughout the world. The basis of its historical transmission is anecdotal and its true global population structure has remained largely cryptic. Seven diverse B. anthracis strains were whole-genome sequenced to identify rare single nucleotide polymorphisms (SNPs), followed by phylogenetic reconstruction of these characters onto an evolutionary model. This analysis identified SNPs that define the major clonal lineages within the species. These SNPs, in concert with 15 variable number tandem repeat (VNTR) markers, were used to subtype a collection of 1,033 B. anthracis isolates from 42 countries to create an extensive genotype data set. These analyses subdivided the isolates into three previously recognized major lineages (A, B, and C), with further subdivision into 12 clonal sub-lineages or sub-groups and, finally, 221 unique MLVA15 genotypes. This rare genomic variation was used to document the evolutionary progression of B. anthracis and to establish global patterns of diversity. Isolates in the A lineage are widely dispersed globally, whereas the B and C lineages occur on more restricted spatial scales. Molecular clock models based upon genome-wide synonymous substitutions indicate there was a massive radiation of the A lineage that occurred in the mid-Holocene (3,064&ndash;6,127 ybp). On more recent temporal scales, the global population structure of B. anthracis reflects colonial-era importation of specific genotypes from the Old World into the New World, as well as the repeated industrial importation of diverse genotypes into developed countries via spore-contaminated animal products. These findings indicate humans have played an important role in the evolution of anthrax by increasing the proliferation and dispersal of this now global disease. Finally, the value of global genotypic analysis for investigating bioterrorist-mediated outbreaks of anthrax is demonstrated."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000461.g003"/>

  <meta property="og:title" content="Global Genetic Population Structure of Bacillus anthracis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000461" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9336'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5138'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6346&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000461">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000461" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000461&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Global Genetic Population Structure of <i>Bacillus anthracis</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthew N. Van Ert, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>

                <p>
                    Current address: Midwest Research Institute, Palm Bay, Florida, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            W. Ryan Easterday, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lynn Y. Huynh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>

                <p>
                    Current address: Graduate Division of Biological and Biomedical Sciences, Emory University, Atlanta, Georgia, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Richard T. Okinaka, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America, 
                  Biosciences, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Martin E. Hugh-Jones, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Environmental Studies, Louisiana State University, Baton Rouge, Louisiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jacques Ravel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Institute for Genomic Research, Rockville, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shaylan R. Zanecki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Talima Pearson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tatum S. Simonson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jana M. U'Ren, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sergey M. Kachur, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rebecca R. Leadem-Dougherty, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shane D. Rhoton, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guenevier Zinser, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jason Farlow, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>

                <p>
                    Current address: Biodesign Institute, Arizona State University, Phoenix, Arizona, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pamala R. Coker, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Environmental Studies, Louisiana State University, Baton Rouge, Louisiana, United States of America
                </p>

                <p>
                    Current address: Pathogen Research & Consulting, Shreveport, Louisiana, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kimothy L. Smith, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>

                <p>
                    Current address: Office of Research and Development, Science and Technology Directorate, Department of Homeland Security, Washington, D. C., United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bingxiang Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Lanzhou Institute of Biological Products, Lanzhou, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Leo J. Kenefic, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claire M. Fraser-Liggett, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Institute for Genomic Research, Rockville, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David M. Wagner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paul Keim
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Paul.Keim@nau.edu">Paul.Keim@nau.edu</a></p>

                <p>Affiliations:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America, 
                  Biosciences, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America, 
                  Pathogen Genomics Division, Translational Genomics Research Institute, Phoenix, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 23, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000461</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000461" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000461" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000461" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000461" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000461-g001" data-doi="info:doi/10.1371/journal.pone.0000461" data-uri="info:doi/10.1371/journal.pone.0000461.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000461-t001" data-doi="info:doi/10.1371/journal.pone.0000461" data-uri="info:doi/10.1371/journal.pone.0000461.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000461-g002" data-doi="info:doi/10.1371/journal.pone.0000461" data-uri="info:doi/10.1371/journal.pone.0000461.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000461-g003" data-doi="info:doi/10.1371/journal.pone.0000461" data-uri="info:doi/10.1371/journal.pone.0000461.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000461-t002" data-doi="info:doi/10.1371/journal.pone.0000461" data-uri="info:doi/10.1371/journal.pone.0000461.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000461">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000461" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Anthrax, caused by the bacterium <em>Bacillus anthracis</em>, is a disease of historical and current importance that is found throughout the world. The basis of its historical transmission is anecdotal and its true global population structure has remained largely cryptic. Seven diverse <em>B. anthracis</em> strains were whole-genome sequenced to identify rare single nucleotide polymorphisms (SNPs), followed by phylogenetic reconstruction of these characters onto an evolutionary model. This analysis identified SNPs that define the major clonal lineages within the species. These SNPs, in concert with 15 variable number tandem repeat (VNTR) markers, were used to subtype a collection of 1,033 <em>B. anthracis</em> isolates from 42 countries to create an extensive genotype data set. These analyses subdivided the isolates into three previously recognized major lineages (A, B, and C), with further subdivision into 12 clonal sub-lineages or sub-groups and, finally, 221 unique MLVA15 genotypes. This rare genomic variation was used to document the evolutionary progression of <em>B. anthracis</em> and to establish global patterns of diversity. Isolates in the A lineage are widely dispersed globally, whereas the B and C lineages occur on more restricted spatial scales. Molecular clock models based upon genome-wide synonymous substitutions indicate there was a massive radiation of the A lineage that occurred in the mid-Holocene (3,064–6,127 ybp). On more recent temporal scales, the global population structure of <em>B. anthracis</em> reflects colonial-era importation of specific genotypes from the Old World into the New World, as well as the repeated industrial importation of diverse genotypes into developed countries via spore-contaminated animal products. These findings indicate humans have played an important role in the evolution of anthrax by increasing the proliferation and dispersal of this now global disease. Finally, the value of global genotypic analysis for investigating bioterrorist-mediated outbreaks of anthrax is demonstrated.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Van Ert MN, Easterday WR, Huynh LY, Okinaka RT, Hugh-Jones ME, et al.  (2007) Global Genetic Population Structure of <em>Bacillus anthracis</em>. PLoS ONE 2(5):
          e461.
            doi:10.1371/journal.pone.0000461</p><p><strong>Academic Editor: </strong>Joseph Petrosino, Baylor College of Medicine, United States of America</p><p><strong>Received:</strong> February 28, 2007; <strong>Accepted:</strong> March 20, 2007; <strong>Published:</strong> May 23, 2007</p><p><strong>Copyright:</strong> © 2007 van ert et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Funding for this work was provided by the National Institutes of Health, General Medical Sciences and the U.S. Departments of Energy and Homeland Security.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Anthrax, caused by the bacterium <em>Bacillus anthracis</em>, is a disease with a natural transmission cycle involving wildlife, livestock, and, occasionally, humans. Recently <em>B. anthracis</em> received notoriety for its use as an agent of bioterrorism in the 2001 letter attacks in the United States <a href="#pone.0000461-Hoffmaster1">[1]</a>, and an unsuccessful aerosol attack in Japan in 1993 <a href="#pone.0000461-Keim1">[2]</a>. Prior to its use as a bioterrorism agent, <em>B. anthracis</em> was developed as a biological weapon by the governments of several countries, including the United States, the United Kingdom, and the former Soviet Union <a href="#pone.0000461-Keim2">[3]</a>. Despite the emphasis on its role as an agent of bioterrorism or biological warfare, anthrax has been and continues to be an important global disease of wildlife and livestock. Global dispersal of spores via commodities has been prevalent, such that there are currently endemic anthrax foci on all continents except Antarctica (<a href="http://www.vetmed.lsu.edu/whocc/">http://www.vetmed.lsu.edu/whocc/</a>). In the environment, <em>B. anthracis</em> primarily exists as a dormant, highly stable spore, which is central to the ecology, evolution, and contemporary weaponization of this pathogen. During the spore phase, which may persist for decades, evolution is static or at least greatly reduced in rate, which limits the amount of genetic diversity found among isolates of this species.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In the past the genetic homogeneity of <em>B. anthracis</em> severely compromised efforts to reconstruct its evolutionary history. Two molecular approaches, multiple locus variable number tandem repeat analysis (MLVA) and whole genome single nucleotide polymorphism (SNP) discovery and analysis, have greatly enhanced the identification of genetic markers that help to establish the phylogenetic relationships among <em>B. anthracis</em> isolates <a href="#pone.0000461-Keim3">[4]</a>, <a href="#pone.0000461-Pearson1">[5]</a>. For example, Keim <em>et al.</em><a href="#pone.0000461-Keim3">[4]</a> used eight variable number tandem repeat (VNTR) markers to examine a worldwide collection of over 400 <em>B. anthracis</em> isolates and described two major clonal lineages (A and B) and 89 unique MLVA8 genotypes. This same VNTR typing scheme also has been used to examine the diversity of <em>B. anthracis</em> in France, <a href="#pone.0000461-Fouet1">[6]</a> Poland, <a href="#pone.0000461-Gierczynski1">[7]</a>, Italy <a href="#pone.0000461-Fasanella1">[8]</a>, and countries in southern <a href="#pone.0000461-Smith1">[9]</a> and northern Africa <a href="#pone.0000461-Maho1">[10]</a>. This process has now been expanded to 15 marker-loci, MLVA15 <a href="#pone.0000461-VanErt1">[11]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Although individual SNPs have limited resolving power relative to MLVA, researchers have used phylogenetic approaches to identify SNPs that efficiently partition bacterial strains into genetic groups consistent with their recognized population structure <a href="#pone.0000461-Keim2">[3]</a>, <a href="#pone.0000461-VanErt1">[11]</a>, <a href="#pone.0000461-Stephens1">[12]</a>. Recent whole genome sequencing efforts discovered approximately 3,500 SNPs among five strains of <em>B. anthracis</em><a href="#pone.0000461-Pearson1">[5]</a>, <a href="#pone.0000461-Read1">[13]</a> (J. Ravel, unpublished). Pearson <em>et al.</em><a href="#pone.0000461-Pearson1">[5]</a> mapped nearly 1,000 of these SNPs across 27 diverse isolates and proposed an extremely robust and conserved phylogenetic model for <em>Bacillus anthracis</em>. The conserved distribution of SNPs within the <em>B. anthracis</em> phylogenetic tree was reflected in the observation that only a single character conflict (homoplasy) was detected from &gt;25,000 data points. These results indicated that that a select number of SNPs representative of specific branches and nodes in the <em>B. anthracis</em> SNP-derived tree would be sufficient to accurately determine the current phylogenetic position of any <em>B. anthracis</em> isolate. A working hypothesis was formulated <a href="#pone.0000461-Keim2">[3]</a> where a small number of canonical SNPs (canSNPs) located at key phylogenetic junctions along the <em>B. anthracis</em> SNP tree could replace a tedious genome-wide SNP analysis. This strategy is analogous to the TagSNP concept that has been suggested by the International HapMap Consortium for the human genome<a href="#pone.0000461-Altshuler1">[14]</a> that “only a minority of sites need to be examined” to fully capture the genotype information in various conserved regions throughout the genome. CanSNPs in <em>B. anthracis</em> represent an extreme example of the TagSNP concept where a single SNP can represent the entire genome of an isolate.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In this study, the canSNP hypothesis for <em>Bacillus anthracis</em> was tested against a diverse global collection containing &gt;1,000 isolates. An initial set of 12 canSNPs representing different points in the evolutionary history of <em>Bacillus anthracis</em> were queried against DNA preparations from this entire collection. These experiments demonstrate that all of the <em>B. anthracis</em> isolates can be placed into one of 12 conserved groups or lineages. The slowly evolving canSNP data set was then coupled to the more rapidly evolving MLVA15 marker set to greatly enhance the resolution beyond the original 89 <em>B. anthracis</em> genotypes <a href="#pone.0000461-Keim3">[4]</a>. The analysis of slowly evolving canSNPs allowed the definition of major clonal lineages in <em>B. anthracis</em>, whereas the more rapidly evolving MLVA15 markers elucidated younger population-level structure in the species. We also utilized molecular clock models, based upon simple assumptions and exhaustive whole genome synonymous SNP surveys of representative strains, to estimate the age of major events in the evolution of <em>B. anthracis</em>. Collectively, our phylogenetic and molecular clock analyses, as well as information on isolate frequencies and global geographic distribution, facilitate the most comprehensive description to date of the global diversity and historical transmission patterns of this pathogen.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Canonical SNP analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>CanSNP analysis subdivided all of the <em>B. anthracis</em> isolates into three previously recognized major lineages (A, B and C), with further subdivisions into one of 12 distinct sub-lineages (<a href="#pone-0000461-g001">Figure 1, stars</a>) or sub-groups (circles). Seven completed whole genome sequences (C.USA.A1055, KrugerB, CNEVA.9066, Ames, Australia94, Vollum, Western North America, see <a href="#pone-0000461-t001">Table 1</a>) defined endpoints (stars) that describe 7 distinct sub-lineages within the canonical SNP tree. These seven strains were picked to represent previously recognized diversity within <em>B. anthracis</em><a href="#pone.0000461-Keim3">[4]</a>, <a href="#pone.0000461-Pearson1">[5]</a>. In addition to the 7 lineages the canSNP analysis identified 5 sub-groups labelled as positions along the branches in the canSNP tree. The positions of each of the canSNPs are illustrated in <a href="#pone-0000461-g001">Figure 1</a> and the canSNP genotype for each of the 7 sub-lineages and the 5 sub-groups is shown in <a href="#pone-0000461-t001">Table 1</a>. It is important to note that all of the 1,033 isolates in this <em>B. anthracis</em> collection fell into one of these 12 subdivisions and that the specific sequenced lineage isolates are only representative of a cluster of related isolates within that lineage.</p>
<div class="figure" id="pone-0000461-g001"><div class="img"><a name="pone-0000461-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000461" data-uri="info:doi/10.1371/journal.pone.0000461.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000461.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000461.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The relationship between canSNPs, sub-lineages and/or sub-groups: The stars in this dendrogram represent specific lineages that are defined by one of the seven sequenced genomes of B. anthracis.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The circles represent branch points along the lineages that contain specific subgroups of isolates. These sub-groups are named after the canSNPs that flank these positions. Indicated in red are the positions and names for each of the canSNPs (also see <a href="#pone-0000461-t001">Table 1</a>).</p>
<span>doi:10.1371/journal.pone.0000461.g001</span></div><div class="figure" id="pone-0000461-t001"><div class="img"><a name="pone-0000461-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000461" data-uri="info:doi/10.1371/journal.pone.0000461.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000461.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000461.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>CANONICAL SNPs.</span></strong></p><span>doi:10.1371/journal.pone.0000461.t001</span></div>

<h4>MLVA15 analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>UPGMA cluster analysis of the MLVA15 data alone clearly identifies the three major genetic lineages (A, B, and C; <a href="#pone-0000461-g002">Figure 2</a>). The longer B and C branch lengths (<a href="#pone-0000461-g001">Figure 1</a>) are underestimated in this analysis (<a href="#pone-0000461-g002">Figure 2</a>) due to mutational saturation of the rapidly evolving MLVA markers. This dataset also increased the number of unique <em>B. anthracis</em> MLVA genotypes from 89 (MLVA8, <a href="#pone.0000461-Keim2">[3]</a>) to 221 owing to both a larger subset of isolates and the expanded resolving power of the MLVA15 marker set (<a href="#pone-0000461-g002">Figure 2</a>, <a href="#pone.0000461.s001">Tables S1</a> and <a href="#pone.0000461.s002">S2</a>). The MLVA15 tree (<a href="#pone-0000461-g002">Figure 2</a>) illustrates that the majority of isolates are located in shallow branches within the A lineage whereas the B and C lineages have rarer genotypes and fewer isolates. The MLVA15 dataset indicates that 89.6% (198) MLVA genotypes are from the A branch, 10% (22 MLVA genotypes) are from the B branch, and only 0.4% (1 MLVA genotype) are from the C branch.</p>
<div class="figure" id="pone-0000461-g002"><div class="img"><a name="pone-0000461-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000461" data-uri="info:doi/10.1371/journal.pone.0000461.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000461.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000461.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>UPGMA dendrogram of VNTR data from worldwide B. anthracis isolates: Fifteen VNTR loci and UPGMA cluster analysis were used to establish genetic relationships among the 1,033 B. anthracis isolates.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>In this UPGMA dendrogram, which was created using MEGA software <a href="#pone.0000461-Kumar1">[39]</a>, groups of genetically similar isolates are collapsed into black triangles that are sized in proportion to the number of isolates in that particular lineage. VNTR loci mutate at faster rates than SNPs and, hence, provide greater resolution for terminal branches. Longer branches, such as the B and C lineages, have length underestimation in this analysis due to mutational saturation. The scale bar indicates genetic distance. Also illustrated on this figure is the distribution of the canonical SNP groups relative to the MLVA phylogeny (right columns). The number of isolates (N) associated with each canSNP group is shown in the second column. The correlation between the phylogenetic clusters identified by the canSNP and MLVA analysis with regards to the world wide geographic distribution of these clusters can be seen in <a href="#pone-0000461-g003">Figure 3</a>.</p>
<span>doi:10.1371/journal.pone.0000461.g002</span></div>

<h4>Geographical distribution of clonal sub-lineages and sub-groups</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>
          <a href="#pone-0000461-g002">Figures 2</a> and <a href="#pone-0000461-g003">3A</a> graphically depict the distribution of the 1,033 isolates into the 12 canSNP sub-lineages and/or sub-groups (Column N in <a href="#pone-0000461-g002">Fig. 2</a> and <a href="#pone-0000461-g003">3A</a>) and also indicates the number of unique MLVA15 genotypes that were found in each of the 12 canSNP groupings (<a href="#pone-0000461-g003">Figure 3A, column G</a>; also see <a href="#pone.0000461.s001">Table S1</a> in the Supplemental Section). The canSNP sub-lineages and sub-groups in <a href="#pone-0000461-g003">Figure 3A</a> also were assigned unique color codes to assist in establishing correlations between these 12 canSNP groupings and the geographic origins of each isolate. These data are presented in <a href="#pone-0000461-g003">Figure 3B</a> as color-coded pie charts for various geographic regions. Each pie chart illustrates the proportion of each canSNP grouping and the total number of isolates that originated from a particular geographic region. North America, Europe, China and parts of Africa are very well represented in these studies, whereas South America and Australia have reasonable representation. Countries from the Middle East and the former Soviet Union are under-represented. These sample biases are important considerations but do not appear to mitigate major genetic and geographic trends in this data set.</p>
<div class="figure" id="pone-0000461-g003"><div class="img"><a name="pone-0000461-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000461" data-uri="info:doi/10.1371/journal.pone.0000461.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000461.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000461.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Worldwide distribution of B. anthracis clonal lineages:Phylogenetic and geographic relationships among 1,033 B. anthracis isolates.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Population structure based upon analysis of data from 12 canSNP (Protocol S1). The numbers of isolates (N) and associated MLVA genotypes (G) within each sub-lineage are indicated as well as the average Hamming distance (D) as estimated from VNTR data. The major lineages (A, B, C) are labelled, as are the derived sub-lineages (1–12), which are also color-coded. (B) Frequency and geographic distribution of the B. anthracis sub-lineages. The colors represented in the pie charts correspond to the sub-lineage color designations in panel A.</p>
<span>doi:10.1371/journal.pone.0000461.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>There are distinct differences in the global distributions of the major <em>B. anthracis</em> clonal lineages (A, B, and C). The A lineage isolates are widely distributed and are found in all countries included in this study. In contrast, the geographic distributions of the B and C lineage isolates are restricted, for example, the B lineage is primarily found in South Africa [B.Br.Kruger B sub-lineage and B.Br.001/002 sub-group <a href="#pone.0000461-Smith1">[9]</a> and portions of Europe [B.CNEVA-9006 sub-lineage; <a href="#pone.0000461-Keim3">[4]</a>, <a href="#pone.0000461-Fouet1">[6]</a>, <a href="#pone.0000461-Gierczynski1">[7]</a> with geographical differentiation at the sub-group level. Examples of these sub-lineages are rarely found outside of these regions.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Although A branch genotypes appear to be scattered throughout the world, there are distinct subgroup geographic compositions for many regions. The dominant genotypes in Southern Africa, for example, belong to the A.Br.Vollum sub-lineage, whereas in Europe isolates from A.Br.008/009 sub-group are dominant. Although central Asia is poorly represented in our collection, the genetic diversity in Eurasia appears to change along a longitudinal axis. Isolate collections from the west (Europe) are dominated by A.Br.008/009 sub-group isolates, and collections from western and south-central Asia (Turkey, India) and western China are dominated by genotypes belonging to A.Br.Aust94 sub-Lineage, (regional data not shown). Further into central and eastern China the genotypes are dominated by isolates belonging to the A.Br001/002 sub-group and A.Br.Ames sub-lineage (regional data not shown). Distinctive genotype compositions are also observed in the western hemisphere, which is dominated by unique clonal lineages that are not observed in the eastern hemisphere. Within the Americas, North and South America contain different genetic groups of <em>B. anthracis</em>: North American genotypes belong mainly to the A.Br.WNA sub-lineage and South American genotypes belong mainly to A.Br.003/004 sub-group.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>A striking feature of isolate collections from the Americas is that the dominant clonal groups are rarely observed outside of these regions. These collections also exhibit low genetic diversity even when analyzed using high-resolution MLVA markers (<a href="#pone-0000461-g002">Figure 2</a>). For instance, in South America isolates from the A.Br.003/004 sub-group (mean within-group VNTR distance = 0.08; <a href="#pone-0000461-g003">Fig. 3A</a>) comprise more than 80% of the total isolates from this region yet are rarely observed elsewhere in the world. A similar trend is observed in the more extensive isolate collection from North America, which is dominated (70%) by a single group (sub-lineage A.Br.WNA; mean within-group VNTR distance = 0.06; <a href="#pone-0000461-g003">Fig. 3A</a>) that is not observed outside of North America. In contrast, the dominant sub-lineages in Europe, Asia, and Africa exhibit greater within-group genetic distances [Europe = A.Br.008/009 sub-group, mean within-group genetic distance = 0.17; South Asia (India, Turkey) = A.Br.Aust94 sub-lineage, mean within-group genetic distance = 0.22; East Asia (China) = A.Br.001/002 sub-group, mean within-group genetic distance = 0.14; Southern Africa = A.Br.Vollum sub-lineage, mean within-group genetic distance = 0.19]. In more industrialized regions, such as Western Europe and the United States, we observe dominant clonal lineages but also the co-occurrence of greatly differing genetic types. Important “donor” regions can be identified and differentiated from “recipient” regions based upon their strain diversity and the positions of these strains in phylogenetic models.</p>


<h4>Molecular Clock Estimates</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Our models, based upon simple assumptions and whole genome synonymous SNP surveys, allowed us to generate age estimates for the major events in the evolutionary history of <em>B. anthracis</em> (<a href="#s4">Methods</a>). The divergence of the rare C branch isolates from the lineage containing the A and B branches appears to have occurred approximately 12,857 to 25,714 ybp. The more recent divergence of the A and B branch from a common ancestor occurred approximately 8,746 to 17,493 ybp. On a more recent time scale, we estimate that the primary A-radiation in <em>B. anthracis</em>, which is clearly evident in <a href="#pone-0000461-g002">Figures 2</a> and <a href="#pone-0000461-g003">3A</a>, occurred approximately 3,277 to 6,555 ybp, or in the mid-Holocene (<a href="#pone-0000461-t002">Table 2</a>).</p>
<div class="figure" id="pone-0000461-t002"><div class="img"><a name="pone-0000461-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000461" data-uri="info:doi/10.1371/journal.pone.0000461.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000461.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000461.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000461.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000461.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Molecular clock estimates of separation times among <em>B. anthracis</em> sub-lineages.</span></strong></p><span>doi:10.1371/journal.pone.0000461.t002</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>
        <em>B. anthracis</em> is thought to have diverged from a <em>B. cereus</em> ancestor by the evolutionary acquisition of two virulence plasmids (pX01 and pX02) and several important chromosomal mutations, such as the nonsense mutation in <em>plcR</em><a href="#pone.0000461-Slamti1">[15]</a>–<a href="#pone.0000461-Keim4">[18]</a>. Subsequent evolution within this pathogen is evidenced by differences in the global distribution and abundance of isolates from the major clonal lineages (A, B, and C). In <em>B. anthracis</em>, the more common genotypes and the majority of isolates are located in shallow branches within the A lineage (<a href="#pone-0000461-g002">Figures 2</a>, <a href="#pone-0000461-g003">3A</a>); whereas the B and C lineages are associated with rarer genotypes and fewer isolates. If isolate abundance is used as a fitness estimator, with rare genotypes considered less fit than common types, genotypes from the C branch and, to a lesser extent, the B branch appear to have very low fitness relative to the A branch genotypes (<a href="#pone-0000461-g002">Figures 2</a>, <a href="#pone-0000461-g003">3</a>). Indeed, the C branch has significantly slower evolutionary rates than the A branch (<a href="#pone.0000461-Pearson1">[5]</a>; <a href="#pone-0000461-g002">Figure 2</a>), suggestive of fewer infective cycles in nature.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The A branch of <em>B. anthracis</em> has experienced a recent and massive radiation (<a href="#pone-0000461-g002">Figures 2</a> and <a href="#pone-0000461-g003">3A</a>) that was clearly a very important event in the evolution of anthrax. Evidence for this event includes the great success of the A branch and its clonal derivatives, the involvement of A genotypes in most of the recent anthrax outbreaks around the world, and short phylogenetic branch lengths within this group. This last point is best illustrated in the dendrogram generated from the MLVA data alone (<a href="#pone-0000461-g002">Figure 2</a>), which capitalizes upon the rapid evolution of VNTR loci to depict the recently-derived radiative lineages within the A branch. The domination of A branch genotypes on a global scale is indicative of great reproductive success (hence, fitness) and considerable long-distance dispersal (<a href="#pone-0000461-g003">Figure 3B</a>). In the absence of the A-lineage expansion, anthrax likely would be a highly restricted and rare disease.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>There are several possible explanations for the differences in global distribution and abundance observed among the major lineages of <em>B. anthracis</em>. One explanation is adaptive genetic differences that affect survival and propagation in either the environment or hosts. A comparison of A vs. B isolates from Kruger National Park, South Africa indicated that A strains were adapted to more diverse environments than B strains, which were restricted to more narrow environmental conditions <a href="#pone.0000461-Smith1">[9]</a>. This trend is also reflected on a global scale, where the B and C types may be successful locally or regionally but, unlike the A strains, are not a dominant presence worldwide. The limited abundance and geographic distribution of these rarer lineages may arise from fitness costs associated with niche specialization <a href="#pone.0000461-Smith1">[9]</a>, <a href="#pone.0000461-Kassen1">[19]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In addition to possible adaptive differences among lineages, stochastic processes such as human-mediated dispersal may explain the greater success of particular genetic groups. The global genetic population structure of <em>B. anthracis</em> suggests human activities have played a role in the proliferation and dispersal of this now global disease and we see evidence for these human impacts on several time scales. For example, models based upon simple assumptions and whole genome synonymous SNP surveys suggest the primary A-radiation in <em>B. anthracis</em> occurred approximately 3,277 to 6,555 ybp, or in the mid-Holocene (<a href="#pone-0000461-t002">Table 2</a>). These age estimates coincide with periods of increased human activities in animal domestication and domesticate population expansion <a href="#pone.0000461-Klemm1">[20]</a>–<a href="#pone.0000461-Bradley1">[24]</a>. Although the importance of the development of human civilization and animal domestication in the natural history of anthrax has been recognized <a href="#pone.0000461-Klemm1">[20]</a>, <a href="#pone.0000461-Kolonin1">[21]</a>, our study presents genetic evidence that it dramatically influenced the global population structure of <em>B. anthracis</em>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>As an important disease of livestock, it seems logical that major evolutionary events in anthrax, such as the A radiation, coincide with human developments in agriculture, animal domestication, and Old World trade routes. Animal husbandry and farming practices, which forced animals into confined areas, are likely to have increased <em>B. anthracis</em> infection and evolutionary rates, which would rapidly increase genotypic diversification. Similarly, the population expansion of large mammal domesticates from the centers of domestication in Eurasia and North Africa would function to disperse <em>B. anthracis</em> genotypes. Molecular clock models suggest that African and Eurasian cattle populations expanded 9,000 ybp and 5,000 ybp, respectively <a href="#pone.0000461-Bradley1">[24]</a>; a time period that roughly corresponds to the A lineage radiation (3,277 to 6,555 ybp) and the divergence of the two major B branches from a common ancestor (4,661–9,322 ybp).</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Independent domestication and domesticate expansion events may provide an explanation for the different assortments of A and B lineages on these two continents. For example, the two major B lineages are spatially separated, one is found in southern Africa (B.Br.001/002 sub-group and B.Br.KrugerB sub-lineage) and the other (B.Br.CNEVA-9066 sub-lineage) is found in portions of Europe, suggesting that after diverging from a common ancestor, these two groups experienced independent evolutionary histories. The divergence of the B.Br.CNEVA and B.Br.Kruger sub-lineages are similar in molecular clock estimates to the A radiation and, again, could represent human influences on this pathogen. Taken together, human-mediated events in the mid-Holocene provide plausible explanations for both the dramatic events in <em>B. anthracis</em> evolution observed during this time period and the diversity among and within clonal lineages on the African and Eurasian landmasses.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The dispersal of <em>B. anthracis</em> to the western hemisphere was probably via intercontinental transport of animal products during European colonization <a href="#pone.0000461-VanNess1">[25]</a>, <a href="#pone.0000461-Hanson1">[26]</a>. Evidence for this includes isolate collections from the western hemisphere that are dominated by clonal groups that are rarely observed outside of these regions and exhibit low genetic diversity when analyzed using high-resolution markers. These patterns are consistent with single, relatively recent introductions followed by widespread dispersal, ecological establishment, and local differentiation. The close derived genetic relationship between the North American sub-lineage A.Br.WNA and the dominant European sub-group A.Br.008/009 is consistent with an introduction to North America from Europe, possibly via French or Spanish colonization <a href="#pone.0000461-VanNess1">[25]</a>, <a href="#pone.0000461-Hanson1">[26]</a>.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>More recent human activities in commerce and industrialization also appear to have impacted the global population structure of <em>B. anthracis</em>. For instance, in addition to a single dominant genetic type, North America also contains a cosmopolitan assortment of rarer <em>B. anthracis</em> genotypes that are likely a consequence of international industrial trade (<em>e.g.,</em> wool, skins, bone meal, shaving brushes). A similar phenomenon is observed in other industrialized regions, such as Western Europe, where we observe the co-occurrence of greatly differing genetic types. The dispersal of these genotypes to industrialized regions has been tied to the trade of spore-infected items <a href="#pone.0000461-VanNess1">[25]</a>, <a href="#pone.0000461-Dragon1">[27]</a>. For instance, in the United Kingdom, the presence of minor genetic types that are dominant in portions of southern and eastern Asia (sub-lineages A.Br.Aust94, A.Br.001/002, A.Br.Ames; <a href="#pone-0000461-g003">Figure 3</a>) is consistent with reports tracing anthrax infections to imported animal products from these regions during the 19<sup>th</sup> and early 20<sup>th</sup> century <a href="#pone.0000461-Legge1">[28]</a>–<a href="#pone.0000461-Green1">[31]</a>. Certainly, the highly-stable <em>B. anthracis</em> spore plays an important role in the importation of diverse genotypes into industrialized countries via transport and trade of contaminated commodities across large distances.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Trade also seems the likely source of <em>B. anthracis</em> in Australia. It has been hypothesized that anthrax was first introduced to Australia in 1847 via contaminated bone meal-based fertilizer shipped from India. Following this initial introduction at Sydney, the disease is thought to have spread along stock routes to the interior of the country <a href="#pone.0000461-Geering1">[32]</a>. Our genetic data provide some support for this hypothesis. All ten of the isolates we examined from India were assigned to sub-lineage A.Br.Aust94, which also appears as the dominant sub-lineage in Australia. It must be noted that the preponderance of isolates from A.Br.Aust94 lineage in Australia stems in part from a collection that is dominated by isolates from a single anthrax outbreak. Our genetic data, in fact, indicates separate introductions into Australia of isolates that belong to the A.Br.005/006 and A.Br.001/002 sub-groups; sub-groups that are more commonly found in Southern Africa and Eastern Asia, respectively.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>
        <em>B. anthracis</em> has been developed as a biological weapon by several nations and terrorists groups and this has greatly increased the value of genotyping analysis for applications that attempt to differentiate between natural and bioterrorist-mediated outbreaks of anthrax. This is illustrated in the identification of the Ames strain as the source for the weaponized material from the 2001 anthrax letter attacks in the USA <a href="#pone.0000461-Hoffmaster1">[1]</a>, <a href="#pone.0000461-Keim2">[3]</a>, <a href="#pone.0000461-Read1">[13]</a>. We found that the Ames strain genotype, which was originally obtained from a dead cow in Texas in 1981, is unique in this isolate collection and, hence, apparently rare in nature. North America is well represented in this study with 273 isolates spanning 44 MLVA genotypes (A.Br.WNA plus isolates from other sub-lineages, <a href="#pone-0000461-g003">Figure 3B</a>). However, the Ames genotype was present only once (although genetically similar isolates to the Ames strain were also identified in Texas, USA). The rarity of the Ames genotype in nature, coupled with its widespread use as a laboratory strain, makes it unlikely that the source material utilized in the 2001 bioterrorist attack was acquired directly from nature. These findings further highlight the importance of large genetic-geographic databases for distinguishing between intentional and environment-acquired infections caused by organisms that are both potential biological weapons and widespread in the environment <a href="#pone.0000461-Fasanella1">[8]</a>, <a href="#pone.0000461-Lowell1">[33]</a>, <a href="#pone.0000461-Cheung1">[34]</a>.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>In summary, our analyses of both canSNP and MLVA data provide a description of the global diversity and historical transmission patterns of <em>B. anthracis</em>. Our data suggest that although <em>B. anthracis</em> is a naturally occurring pathogen, human activities have dramatically influenced its current distribution and occurrence. We observe the effects of human activities at three levels: 1) the massive radiation of the A-branch in the mid-Holocene, 2) the more recent colonial-era importation of specific <em>B. anthracis</em> genotypes from the Old World into the New World, which lead to their ecological establishment, and 3) the repeated industrial importation of rare diverse genotypes into developed countries through animal products (e.g. wool, hides, and bone meal). The genetic population structure of <em>B. anthracis</em> is indicative of these long distance transmission events and illustrates its ability to become ecologically established in new locations. Fortunately, natural outbreaks of anthrax can be managed effectively through vaccination and public health efforts. However, due to actual and potential nefarious use of the pathogen, anthrax will likely remain of great social and scientific importance.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Nomenclature</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The tree in <a href="#pone-0000461-g001">Figure 1</a> is based upon an analysis of &gt;1,000 SNPs discovered amongst seven complete or draft genomes of <em>B. anthracis</em>, which yielded a branched phylogeny containing seven lineages corresponding to the sequenced “discovery” genomes <a href="#pone.0000461-Pearson1">[5]</a>. In a strictly clonal species, like <em>B. anthracis</em>, these genomes will be situated at the end of each branch. These terminal lineages are depicted as stars <a href="#pone-0000461-g001">Figure 1</a> and each of these lineages is named after the sequenced isolate (e.g. Ames, KrugerB, Vollum, etc.). The canSNPs are named after one of the three main clades (e.g. A, B, or C) followed by a three digit number (A.Br.001, A.Br.002, A.Br.003). Where possible, we have tried to be systematic in naming the canSNPs. For example, the first canSNP in the A branch was proximal to the Ames genome sequence (or the lineage terminus) and is named A.Br.001 (red labels in <a href="#pone-0000461-g001">Figure 1</a>). The second canSNP position defines a position between canSNP A.Br.001 and the circled position called A.Br.001/002. Such a systematic naming scheme for canSNPs may be compromised by future studies that define additional lineages and branches (<em>i.e.</em> the order of the canSNPs from the terminus will be inconsistent with their names). Hence, this should only be considered an arbitrary numbering system, but it will function effectively as new phylgenetic discoveries are made. The circles in the dendrogram represent branches or branch points defined by flanking characters (canSNPs). The branch points and the ends of lineages (the circles and stars in <a href="#pone-0000461-g001">Fig 1</a>) encompass all 1,033 <em>B. anthracis</em> isolates (ranging from 2 isolates in the C lineage (C.Br.A1055) to 228 isolates in the Western North American lineage (A.Br.WNA). Branch points also have been defined and named by their flanking canSNPs (<em>e.g.</em> B.Br.001/002). The near total absence of homoplasy (character conflicts in the tree), coupled with character discovery bias, has caused “branch collapse” in this clonally propagating pathogen <a href="#pone.0000461-Pearson1">[5]</a>, <a href="#pone.0000461-Worobey1">[35]</a>. A collapsed branch is still defined by its flanking canSNP characters.</p>


<h4>
          <em>B. anthracis</em> isolates</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We examined a global collection of 1,033 <em>B. anthracis</em> isolates. <a href="#pone.0000461.s003">Table S3</a> contains information on the numbers and distribution of strains used in this study. These isolates were obtained from known anthrax cases, environmental sources, or other materials associated with the disease. Our isolate collection is biased toward anthrax outbreaks that occurred in the last several decades and towards countries actively engaged in the international exchange of scientific material. It is important to note that all of the isolates analyzed in this study were shown to possess the <em>plcR</em> inactivating mutation as detected by the PCR assay described in Easterday <em>et al.</em><a href="#pone.0000461-Easterday1">[16]</a>. This nonsense mutation is considered essential for maintenance of virulence plasmids and represents a definitive character of <em>B. anthracis</em><a href="#pone.0000461-Easterday1">[16]</a>, <a href="#pone.0000461-Mignot1">[36]</a>.</p>


<h4>DNA isolation</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A 1.0 µl inoculating loop was used to transfer <em>B. anthracis</em> colony material into 200 µl of Brain-Heart Infusion broth (Hardy Diagnostics, Santa Maria, CA) within the wells of a sterile, untreated polycarbonate 96-well culture plate (Costar Corning Inc., Acton, MA). The plate was then covered with an adhesive plastic film, placed in a secondary containment device, and incubated overnight at 37°C without shaking. Following incubation, 10.0 µl of broth was transferred to a Microseal™ Polypropylene Microplate (MJ Research, Waltham, MA). The samples were then flash-frozen in 96-well cold block (−80°C) for 15 s and then immediately thawed in a 96-well heat block (96°C) for 15 s. This freeze-thaw cycle was repeated two additional times. The cell lysates were then transferred into a 96-well GV 0.2 µM Durapore Multiscreen Plate (Millipore, Billarica, MA) containing 100 µl of TE (10 mM Tris-HCl [pH 8.0], 1.0 mM EDTA) per well. Cellular debris and spores were removed from the 96-well filter plate by vacuum filtration using a MultiScreen Separations System Manifold (Millipore, Bedford, MA). The filtrate was collected into a 96-well plate and used to support PCR for downstream SNP and MLVA genotyping. The sterility of each sample was confirmed by plating 1.0 µl of each filtrate onto a TSA II 5% Sheep Blood prepared media plate (Becton Dickinson and Company, Cockeysville, MD) and incubating at 37°C for 48 hr.</p>


<h4>Genetic Markers</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Two types of genetic markers were used to analyze the <em>B. anthracis</em> collection: canonical single nucleotide polymorphisms (canSNPs) and variable number tandem repeats (VNTRs). We used data from the Pearson <em>et al.</em><a href="#pone.0000461-Pearson1">[5]</a> and unpublished genomic sequence data (Ravel <em>et al.</em>, unpublished data) to identify canSNPs that can be used for identifying a particular phylogenetic point in the evolutionary history of <em>B. anthracis</em>. In total, 2 <em>B. anthracis</em> specific SNPs and 12 canSNPs to analyze DNA from the 1,033 <em>B. anthracis</em> isolates. CanSNP alleles were determined using TaqMan™ -Minor Groove Binding (MGB) allelic discrimination assays. TaqMan™ MGB probes and primers for the canSNPs were designed using ABI Primer Express software and guidelines, with the exception that allele-specific probe lengths were manually adjusted to match melting temperatures <a href="#pone.0000461-Morin1">[37]</a>. The genomic location for each of the canSNPs can be found in <a href="#pone.0000461.s004">Table S4</a> while the probe and primer sequences for each are listed in <a href="#pone.0000461.s005">Table S5</a>. Each 10.0 µl reaction contained 1× ABI Universal Master Mix, 250 nM of each probe, and 600 nM each of forward and reverse primers and 1.0 µl of approximately 350 pg/µl template DNA. For all assays, thermal cycling parameters were 50° C for 2 min., 95° C for 10 min., followed by 40–50 cycles of 95° C for 15 sec and 60° C for 1 min. Endpoint fluorescent data were measured on the ABI 7900.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>DNA from the isolates was also analyzed using 15 VNTR loci; eight of these VNTRs are described by Keim <em>et al.</em><a href="#pone.0000461-Keim3">[4]</a> MLVA8 and the additional 7 markers are described by Zinser <a href="#pone.0000461-Zinser1">[38]</a>. These markers were compiled together into a multiple-locus VNTR analysis (MLVA15) subtyping system (see Protocol S1, <a href="#pone.0000461.s006">Table S6</a> for details on the markers and methods).</p>


<h4>Phylogenetic analyses</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Two basic approaches were used to analyze genetic relationships among the 1,033 <em>B. anthracis</em> isolates. First, canSNP and VNTR data were used in a hierarchical approach to analyze phylogenetic relationships: data from the slowly evolving canSNPs loci were used to categorize the isolates into clonal lineages and followed by the use of data from the 15 rapidly-evolving VNTR loci to measure diversity and determine the number of genotypes within each of these clonal categories. This system allowed us to effectively analyze both older phylogenetic relationships and younger population-level structure <a href="#pone.0000461-Keim2">[3]</a>. Second, we used UPGMA cluster analyses of the MLVA15 data alone to illustrate the global population genetic structure in an unbiased manner. All phylogenetic analyses were conducted using MEGA3 software <a href="#pone.0000461-Kumar1">[39]</a>.</p>


<h4>Geographic distribution of clonal lineages</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>To examine genetic-geographic patterns in <em>B. anthracis</em>, we mapped the worldwide distribution of the clonal lineages that were identified by the analysis of the canSNP data.</p>


<h4>Age Estimates</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>To estimate the age of several events in the evolutionary history of <em>B. anthracis</em>, we performed whole genome synonymous SNP comparisons of strains that represent major clonal lineages. We utilized the following equation to estimate the time since pairs of strains last shared a common ancestor:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.e001&amp;representation=PNG" class="inline-graphic"></span><br>where <em>sSNPs</em> is the total number of synonymous SNPs between two strains as determined by whole-genome comparisons, <em>MR</em> is the per site synonymous mutation rate in <em>B. anthracis</em> (5.2×10<sup>−10</sup> mutations/generation; <a href="#pone.0000461-Vogler1">[40]</a>, <em>sSites</em> is the number of synonymous sites in common between the two strains, and <em>generations</em> is the estimated number of generations undergone by a given lineage in each year (estimated as 43 per transmission cycle). The number of generations per year is based upon an ungulate transmission model and the number of infection/death cycles per year (see detailed descriptions below). The age estimates are particularly sensitive to the number of infection/death cycles per year. As such, we calculated the estimates using both 1 (43 generations/year) and 0.5 (21.5 generations/year) infection/death cycles per year (<a href="#pone-0000461-t002">Table 2</a>).</p>


<h4>Details of the Age Estimates</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>The use of sSNPs for the substitution rate restricts these estimates to nearly neutral evolutionary characters. While all SNPs are relatively infrequent among <em>B. anthracis</em> isolates, the use of whole genome analysis has identified many sSNPs (<a href="#pone-0000461-t001">Table 1</a>) and resulted in highly robust estimates of relationships among isolates <a href="#pone.0000461-Pearson1">[5]</a>. sSNP occurrence between two strains is modeled well by the Poisson probability distribution. The relative large number of observations makes the error in this estimate small. When the expected number is high, the Poisson become fairly symmetrical with a standard deviation equal to the square root of the expected number. Thus, two standard deviations from the maxima are very close to the 95% confidence interval.</p>
<a id="article1.body1.sec4.sec8.p2" name="article1.body1.sec4.sec8.p2"></a><p>The mutations rates for single nucleotide changes have been reported in <em>B. anthracis</em> based upon selection for antibiotic resistance (Rif) and are very similar to the rates observed for other well-studied bacteria (2). In this case, Vogler et al. <a href="#pone.0000461-Vogler1">[40]</a> estimated the rate using the Luria-Delbruck fluctuation test and then partitioned the phenotypic mutation rate (1.55E-09 mutants per generation) to different nucleotide positions in the <em>rpoB</em> gene by sequencing this gene in the mutants. Hence, we have a per site mutation rate (5.2E-10 mutations per generation) instead of merely a phenotypic rate.</p>
<a id="article1.body1.sec4.sec8.p3" name="article1.body1.sec4.sec8.p3"></a><p>While Drake <a href="#pone.0000461-Drake1">[41]</a>, <a href="#pone.0000461-Drake2">[42]</a> has argued for a universal substitution rate for microbial genomes, the extremely episodic nature of anthrax transmission makes this hard to justify among the clonal lineage of <em>B. anthracis.</em> Indeed, this is clearly the most sensitive aspect of the substitution rate model with certain parameters highly influential in the final estimates.</p>


<h4>Ungulate transmission model</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>The number of <em>Bacillus anthracis</em> generations (<em>G</em>) in a single infected ungulate was determined using the following equation:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000461.e002&amp;representation=PNG" class="inline-graphic"></span><br>where <em>t</em> = terminal number of <em>B. anthracis</em> organisms in a 100 kg ungulate (100 kg × <em>d</em>), <em>i</em> = initial number of <em>B. anthracis</em> organisms in the ungulate as obtained from an environmental source (10 organisms), and <em>d</em> = terminal density of <em>B. anthracis</em> organisms per unit body weight in a mammal (10<sup>8.8</sup> organisms per kg) <a href="#pone.0000461-Jones1">[43]</a>. Based on these parameters, it was estimated that <em>G</em> = 43.1, which was rounded to 43. The model is not particularly sensitive to this particular parameter. Changing the size of the animal and, hence, the final <em>B. anthracis</em> population size is mitigated by the log<sub>2</sub> transformation. The number of generations is altered only by 3.3 for every 10-fold increase in population size. This has a minimal affect upon the final number of generations.</p>


<h4>Infection/death cycles per year</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>Estimating the number of infection/death cycles per year is difficult for anthrax. While hundreds or even thousands of individual animals might die in a single outbreak, it is unlikely that these multiple victims are sequential infection/death cycles. Rather, these clusters are likely to be from a single source, or due to environmental induction of the outbreak. For this reason, we believe that the average annual number of death/infection cycles will be one or less, even in the most endemic regions. <em>B. anthracis</em> spores are known to survive long periods of time; though very long-term spore survival is unlike to be important in the overall evolutionary rates as the viability does drop with time. In this study, we are primarily interested in the most highly fit branch of <em>B. anthracis</em> (A). Its worldwide distribution and fitness argues for a higher rate of transmission, probably close to one infection/death cycle per year. Because this is one of the most sensitive parameters in the model, we have modeled the molecular clock estimates using both 1 and 0.5 deaths per year. These values translate into 43 or 21.5 generations per year when combined with the population size estimates from a typical host (see above).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000461"></div><a name="pone.0000461.s001" id="pone.0000461.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000461.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">The 221 MLVA Genotypes and Associated Can SNPs. The 221 genotypes (1–221, Column A) are organized according to their Keim Genetics Lab ID Designation (“A” number - Column B), prior designations when available (“K” numbers - Column C), their original MLVA8 GenoTyping designation (“GT” numbers: 1–89 - Column D) from Keim et al., (2000) and the alternative strain designations and original source codes for each isolate (Column E). This is followed by the isolate's canSNP lineage/group (Column F, also see <a href="#pone-0000461-g001">Fig 1</a>), two B. anthracis specific SNPs (Columns G and H), the13 canSNP scores (Columns I–U) and the 15 marker MLVA profile for that isolate (Columns W–AK). The first two SNPs (Column G and H) are Bacillus anthracis specific SNPs originally identified in the plcR and gyrA loci and are not part of the canSNP profile. There are 221 unique MLVA genotypes listed in this table.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000461.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.15 MB XLS)</p>
<a name="pone.0000461.s002" id="pone.0000461.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000461.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">The MLVA Sizing Code. The VNTR alleles for each MLVA marker in Supplemental <a href="#pone.0000461.s001">Table S1</a> are letter coded according to size to allow these data sets to be utilized by various tree drawing programs. Apparent MLVA fragment sizes vary from instrument to instrument and even with various size standards. Allele codes provide a common designation in the face of this variation. <a href="#pone.0000461.s002">Table S2</a> provides a code that describes the fragment sizes for these alleles based on analysis on an ABI 3100 Genetic Analyzer (see Protocol S3), a custom made LIZ®-labeled internal size standard (5), and subsequent analysis using Genotyper. The numeral 1 appears as a code when a fragment failed to amplify; eg., an isolate lacking the pXO1 plasmid would not be able to amplify the pXO1.1AAT VNTR marker.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000461.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0000461.s003" id="pone.0000461.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000461.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Geographical Composition of B. anthracis isolates used in this study</p>
<p class="siDoi">doi:10.1371/journal.pone.0000461.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0000461.s004" id="pone.0000461.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000461.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">CanSNPs Description and Chromosomal Location</p>
<p class="siDoi">doi:10.1371/journal.pone.0000461.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000461.s005" id="pone.0000461.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000461.s005">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Canonical SNP Primers/Probes used in molecular typing of B. anthracis</p>
<p class="siDoi">doi:10.1371/journal.pone.0000461.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000461.s006" id="pone.0000461.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000461.s006">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">15 VNTR loci in the B. anthracis 15 VNTR MLVA system.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000461.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PK TP Mv DW WE LH TS JU SK KS LK. Performed the experiments: Mv WE LH SZ TS JU SK SR GZ LK RL. Analyzed the data: JR PK TP Mv DW WE LH. Contributed reagents/materials/analysis tools: CF JR TP Mv DW WE LH JF PC KS BW MH. Wrote the paper: PK Mv DW RO.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000461-Hoffmaster1" id="pone.0000461-Hoffmaster1"></a>Hoffmaster AR, Fitzgerald CC, Ribot E, Mayer LW, Popovic T (2002) Molecular subtyping of Bacillus anthracis and the 2001 bioterrorism-associated anthrax outbreak, United States. Emerg Infect Dis  8: 1111–1116.  <ul class="find" data-citedArticleID="953120" data-doi="10.3201/eid0810.020394"><li><a href="http://dx.doi.org/10.3201/eid0810.020394" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+subtyping+of+Bacillus+anthracis+and+the+2001+bioterrorism-associated+anthrax+outbreak%2C+United+States." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+subtyping+of+Bacillus+anthracis+and+the+2001+bioterrorism-associated+anthrax+outbreak%2C+United+States.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000461-Keim1" id="pone.0000461-Keim1"></a>Keim P, Smith KL, Keys C, Takahashi H, Kurata T, et al.  (2001) Molecular investigation of the Aum Shinrikyo anthrax release in Kameido, Japan. J Clin Microbiol  39: 4566–4567.  <ul class="find" data-citedArticleID="953128" data-doi="10.1128/jcm.39.12.4566-4567.2001"><li><a href="http://dx.doi.org/10.1128/jcm.39.12.4566-4567.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+investigation+of+the+Aum+Shinrikyo+anthrax+release+in+Kameido%2C+Japan." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+investigation+of+the+Aum+Shinrikyo+anthrax+release+in+Kameido%2C+Japan.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000461-Keim2" id="pone.0000461-Keim2"></a>Keim P, Van Ert MN, Pearson T, Vogler AJ, Huynh LY, et al.  (2004) Anthrax molecular epidemiology and forensics: using the appropriate marker for different evolutionary scales. Infect Genet Evol  4: 205–213.  <ul class="find" data-citedArticleID="953130" data-doi="10.1016/j.meegid.2004.02.005"><li><a href="http://dx.doi.org/10.1016/j.meegid.2004.02.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anthrax+molecular+epidemiology+and+forensics%3A+using+the+appropriate+marker+for+different+evolutionary+scales." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anthrax+molecular+epidemiology+and+forensics%3A+using+the+appropriate+marker+for+different+evolutionary+scales.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000461-Keim3" id="pone.0000461-Keim3"></a>Keim P, Price LB, Klevytska AM, Smith KL, Schupp JM, et al.  (2000) Multiple-locus variable-number tandem repeat analysis reveals genetic relationships within Bacillus anthracis. J Bacteriol  182: 2928–2936.  <ul class="find" data-citedArticleID="953132" data-doi="10.1128/jb.182.10.2928-2936.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.10.2928-2936.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple-locus+variable-number+tandem+repeat+analysis+reveals+genetic+relationships+within+Bacillus+anthracis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple-locus+variable-number+tandem+repeat+analysis+reveals+genetic+relationships+within+Bacillus+anthracis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000461-Pearson1" id="pone.0000461-Pearson1"></a>Pearson T, Busch JD, Ravel J, Read TD, Rhoton SD, et al.  (2004) Phylogenetic discovery bias in Bacillus anthracis using single-nucleotide polymorphisms from whole-genome sequencing. Proc Natl Acad Sci U S A  101: 13536–13541.  <ul class="find" data-citedArticleID="953154" data-doi="10.1073/pnas.0403844101"><li><a href="http://dx.doi.org/10.1073/pnas.0403844101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+discovery+bias+in+Bacillus+anthracis+using+single-nucleotide+polymorphisms+from+whole-genome+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+discovery+bias+in+Bacillus+anthracis+using+single-nucleotide+polymorphisms+from+whole-genome+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000461-Fouet1" id="pone.0000461-Fouet1"></a>Fouet A, Smith KL, Keys C, Vaissaire J, Le Doujet C, et al.  (2002) Diversity among French Bacillus anthracis isolates. J Clin Microbiol  40: 4732–4734.  <ul class="find" data-citedArticleID="953110" data-doi="10.1128/jcm.40.12.4732-4734.2002"><li><a href="http://dx.doi.org/10.1128/jcm.40.12.4732-4734.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+among+French+Bacillus+anthracis+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+among+French+Bacillus+anthracis+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000461-Gierczynski1" id="pone.0000461-Gierczynski1"></a>Gierczynski R, Kaluzewski S, Rakin A, Jagielski M, Zasada A, et al.  (2004) Intriguing diversity of Bacillus anthracis in eastern Poland–the molecular echoes of the past outbreaks. FEMS Microbiol Lett  239: 235–240.  <ul class="find" data-citedArticleID="953114" data-doi="10.1016/j.femsle.2004.08.038"><li><a href="http://dx.doi.org/10.1016/j.femsle.2004.08.038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intriguing+diversity+of+Bacillus+anthracis+in+eastern+Poland%26%23x2013%3Bthe+molecular+echoes+of+the+past+outbreaks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intriguing+diversity+of+Bacillus+anthracis+in+eastern+Poland%26%23x2013%3Bthe+molecular+echoes+of+the+past+outbreaks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000461-Fasanella1" id="pone.0000461-Fasanella1"></a>Fasanella A, Van Ert M, Altamura SA, Garofolo G, Buonavoglia C, et al.  (2005) Molecular diversity of Bacillus anthracis in Italy. J Clin Microbiol  43: 3398–3401.  <ul class="find" data-citedArticleID="953108" data-doi="10.1128/jcm.43.7.3398-3401.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.7.3398-3401.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+diversity+of+Bacillus+anthracis+in+Italy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+diversity+of+Bacillus+anthracis+in+Italy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000461-Smith1" id="pone.0000461-Smith1"></a>Smith KL, DeVos V, Bryden H, Price LB, Hugh-Jones ME, et al.  (2000) Bacillus anthracis diversity in Kruger National Park. J Clin Microbiol  38: 3780–3784.  <ul class="find" data-citedArticleID="953160"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bacillus+anthracis+diversity+in+Kruger+National+Park.&amp;auth=&amp;atitle=Bacillus+anthracis+diversity+in+Kruger+National+Park." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacillus+anthracis+diversity+in+Kruger+National+Park." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacillus+anthracis+diversity+in+Kruger+National+Park.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000461-Maho1" id="pone.0000461-Maho1"></a>Maho A, Rossano A, Hachler H, Holzer A, Schelling E, et al.  (2006) Antibiotic susceptibility and molecular diversity of Bacillus anthracis strains in Chad: detection of a new phylogenetic subgroup. J Clin Microbiol  44: 3422–3425.  <ul class="find" data-citedArticleID="953146" data-doi="10.1128/jcm.01269-06"><li><a href="http://dx.doi.org/10.1128/jcm.01269-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antibiotic+susceptibility+and+molecular+diversity+of+Bacillus+anthracis+strains+in+Chad%3A+detection+of+a+new+phylogenetic+subgroup." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antibiotic+susceptibility+and+molecular+diversity+of+Bacillus+anthracis+strains+in+Chad%3A+detection+of+a+new+phylogenetic+subgroup.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000461-VanErt1" id="pone.0000461-VanErt1"></a>Van Ert MN, Easterday WR, Simonson TS, U'Ren JM, Pearson T, et al.  (2007) Strain-Specific Single-Nucleotide Polymorphism Assays for the Bacillus anthracis Ames Strain. J Clin Microbiol  45: 47–53.  <ul class="find" data-citedArticleID="953164" data-doi="10.1128/jcm.01233-06"><li><a href="http://dx.doi.org/10.1128/jcm.01233-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strain-Specific+Single-Nucleotide+Polymorphism+Assays+for+the+Bacillus+anthracis+Ames+Strain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strain-Specific+Single-Nucleotide+Polymorphism+Assays+for+the+Bacillus+anthracis+Ames+Strain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000461-Stephens1" id="pone.0000461-Stephens1"></a>Stephens AJ, Huygens F, Inman-Bamber J, Price EP, Nimmo GR, et al.  (2006) Methicillin-resistant Staphylococcus aureus genotyping using a small set of polymorphisms. J Med Microbiol  55: 43–51.  <ul class="find" data-citedArticleID="953162" data-doi="10.1099/jmm.0.46157-0"><li><a href="http://dx.doi.org/10.1099/jmm.0.46157-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methicillin-resistant+Staphylococcus+aureus+genotyping+using+a+small+set+of+polymorphisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methicillin-resistant+Staphylococcus+aureus+genotyping+using+a+small+set+of+polymorphisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000461-Read1" id="pone.0000461-Read1"></a>Read TD, Salzberg SL, Pop M, Shumway M, Umayam L, et al.  (2002) Comparative genome sequencing for discovery of novel polymorphisms in Bacillus anthracis. Science  296: 2028–2033.  <ul class="find" data-citedArticleID="953156" data-doi="10.1126/science.1071837"><li><a href="http://dx.doi.org/10.1126/science.1071837" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genome+sequencing+for+discovery+of+novel+polymorphisms+in+Bacillus+anthracis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genome+sequencing+for+discovery+of+novel+polymorphisms+in+Bacillus+anthracis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000461-Altshuler1" id="pone.0000461-Altshuler1"></a>Altshuler D, Brooks LD, Chakravarti A, Collins FS, Daly MJ, et al.  (2005) The International HapMap Consortium: A haplotype map of the human genome. Nature  437: 1299–1320.  <ul class="find" data-citedArticleID="953088" data-doi="10.1038/nature04226"><li><a href="http://dx.doi.org/10.1038/nature04226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+International+HapMap+Consortium%3A+A+haplotype+map+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+International+HapMap+Consortium%3A+A+haplotype+map+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000461-Slamti1" id="pone.0000461-Slamti1"></a>Slamti L, Perchat S, Gominet M, Vilas-Boas G, Fouet A, et al.  (2004) Distinct mutations in PlcR explain why some strains of the Bacillus cereus group are nonhemolytic. J Bacteriol  186: 3531–3538.  <ul class="find" data-citedArticleID="953158" data-doi="10.1128/jb.186.11.3531-3538.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.11.3531-3538.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+mutations+in+PlcR+explain+why+some+strains+of+the+Bacillus+cereus+group+are+nonhemolytic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+mutations+in+PlcR+explain+why+some+strains+of+the+Bacillus+cereus+group+are+nonhemolytic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000461-Easterday1" id="pone.0000461-Easterday1"></a>Easterday WR, Van Ert MN, Simonson TS, Wagner DM, Kenefic LJ, et al.  (2005) Use of single nucleotide polymorphisms in the plcR gene for specific identification of Bacillus anthracis. J Clin Microbiol  43: 1995–1997.  <ul class="find" data-citedArticleID="953106" data-doi="10.1128/jcm.43.4.1995-1997.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.4.1995-1997.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+single+nucleotide+polymorphisms+in+the+plcR+gene+for+specific+identification+of+Bacillus+anthracis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+single+nucleotide+polymorphisms+in+the+plcR+gene+for+specific+identification+of+Bacillus+anthracis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000461-Okinaka1" id="pone.0000461-Okinaka1"></a>Okinaka RT, Pearson T, Keim PS (2006) Anthrax but not Bacillus anthracis? PLoS-Pathogens  2: 1025–1027.  <ul class="find" data-citedArticleID="953152" data-doi="10.1371/journal.ppat.0020122"><li><a href="http://dx.doi.org/10.1371/journal.ppat.0020122" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anthrax+but+not+Bacillus+anthracis%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anthrax+but+not+Bacillus+anthracis%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000461-Keim4" id="pone.0000461-Keim4"></a>Keim PS, Pearson T, Okinaka RT (2007)  Evolution of Bacillus anthracis, causative agent of anthrax. In: Cassell GH, Baquero F, Nombela C, Gutierrez-Fuentes JA, editors. Introduction to Evolutionary Biology of Bacterial and Fungal Pathogens.   in press. <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0000461-Kassen1" id="pone.0000461-Kassen1"></a>Kassen R, Llewellyn M, Rainey PB (2004) Ecological constraints on diversification in a model adaptive radiation. Nature  431: 984–988.  <ul class="find" data-citedArticleID="953126" data-doi="10.1038/nature02923"><li><a href="http://dx.doi.org/10.1038/nature02923" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecological+constraints+on+diversification+in+a+model+adaptive+radiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecological+constraints+on+diversification+in+a+model+adaptive+radiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000461-Klemm1" id="pone.0000461-Klemm1"></a>Klemm DM, Klemm WR (1959) A History of Anthrax. J Am Vet Med Assoc  135: 458–462.  <ul class="find" data-citedArticleID="953136"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+History+of+Anthrax.&amp;auth=&amp;atitle=A+History+of+Anthrax." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+History+of+Anthrax." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+History+of+Anthrax.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000461-Kolonin1" id="pone.0000461-Kolonin1"></a>Kolonin GV (1971) Evolution of anthrax, report II: History of the spread of the disease. Zhurnal Mikrobiol Epidemiol  48: 118–122.  <ul class="find" data-citedArticleID="953138"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+of+anthrax%2C+report+II%3A+History+of+the+spread+of+the+disease.&amp;auth=&amp;atitle=Evolution+of+anthrax%2C+report+II%3A+History+of+the+spread+of+the+disease." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+anthrax%2C+report+II%3A+History+of+the+spread+of+the+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+anthrax%2C+report+II%3A+History+of+the+spread+of+the+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000461-Diamond1" id="pone.0000461-Diamond1"></a>Diamond J (1997) Guns, Germs and Steel. New York: W.W. Norton and Company.   <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0000461-Diamond2" id="pone.0000461-Diamond2"></a>Diamond J (2002) Evolution, consequences and future of plant and animal domestication. Nature  418: 700.  <ul class="find" data-citedArticleID="953096" data-doi="10.1038/nature01019"><li><a href="http://dx.doi.org/10.1038/nature01019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution%2C+consequences+and+future+of+plant+and+animal+domestication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution%2C+consequences+and+future+of+plant+and+animal+domestication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000461-Bradley1" id="pone.0000461-Bradley1"></a>Bradley DG, MacHugh DE, Cunningham P, Loftus RT (1996) Mitochodrial diversity and the origins of African and European cattle. PNAS  93: 5131–5135.  <ul class="find" data-citedArticleID="953090" data-doi="10.1073/pnas.93.10.5131"><li><a href="http://dx.doi.org/10.1073/pnas.93.10.5131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochodrial+diversity+and+the+origins+of+African+and+European+cattle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochodrial+diversity+and+the+origins+of+African+and+European+cattle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000461-VanNess1" id="pone.0000461-VanNess1"></a>Van Ness GB (1971) Ecology of anthrax. Science  172: 1303–1307.  <ul class="find" data-citedArticleID="953166" data-doi="10.1126/science.172.3990.1303"><li><a href="http://dx.doi.org/10.1126/science.172.3990.1303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecology+of+anthrax." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecology+of+anthrax.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000461-Hanson1" id="pone.0000461-Hanson1"></a>Hanson RP (1959) The earliest account of anthrax in man and animals in North America. J Am Vet Mde Assoc  135:   <ul class="find" data-citedArticleID="953118"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+earliest+account+of+anthrax+in+man+and+animals+in+North+America.&amp;auth=&amp;atitle=The+earliest+account+of+anthrax+in+man+and+animals+in+North+America." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+earliest+account+of+anthrax+in+man+and+animals+in+North+America." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+earliest+account+of+anthrax+in+man+and+animals+in+North+America.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000461-Dragon1" id="pone.0000461-Dragon1"></a>Dragon DC, Elkin BT, Nishi JS, Ellsworth TR (1999) A review of anthrax in Canada and implications for research on the disease in northern bison. J Appl Microbiol  87: 208–213.  <ul class="find" data-citedArticleID="953100" data-doi="10.1046/j.1365-2672.1999.00872.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2672.1999.00872.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+review+of+anthrax+in+Canada+and+implications+for+research+on+the+disease+in+northern+bison." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+review+of+anthrax+in+Canada+and+implications+for+research+on+the+disease+in+northern+bison.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000461-Legge1" id="pone.0000461-Legge1"></a>Legge TM (1905) Milroy Lectures on industrial anthrax. Lancet  165: 842.  <ul class="find" data-citedArticleID="953142" data-doi="10.1016/s0140-6736(01)45882-x"><li><a href="http://dx.doi.org/10.1016/s0140-6736(01)45882-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Milroy+Lectures+on+industrial+anthrax." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Milroy+Lectures+on+industrial+anthrax.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000461-Doig1" id="pone.0000461-Doig1"></a>Doig AT, Gemmill JS (1951) Epidemiology of a small outbreak of anthrax. Lancet  1: 1011–1012.  <ul class="find" data-citedArticleID="953098" data-doi="10.1016/s0140-6736(51)92521-4"><li><a href="http://dx.doi.org/10.1016/s0140-6736(51)92521-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epidemiology+of+a+small+outbreak+of+anthrax." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epidemiology+of+a+small+outbreak+of+anthrax.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000461-Jamieson1" id="pone.0000461-Jamieson1"></a>Jamieson WM, Green DM (1955) Anthrax and bone-meal fertiliser. Lancet  268: 560.  <ul class="find" data-citedArticleID="953122" data-doi="10.1016/s0140-6736(55)91238-1"><li><a href="http://dx.doi.org/10.1016/s0140-6736(55)91238-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anthrax+and+bone-meal+fertiliser." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anthrax+and+bone-meal+fertiliser.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000461-Green1" id="pone.0000461-Green1"></a>Green DM, Jamieson WM (1958) Anthrax and bone-meal fertiliser. Lancet  2: 153–154.  <ul class="find" data-citedArticleID="953116" data-doi="10.1016/s0140-6736(58)92237-2"><li><a href="http://dx.doi.org/10.1016/s0140-6736(58)92237-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anthrax+and+bone-meal+fertiliser." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anthrax+and+bone-meal+fertiliser.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000461-Geering1" id="pone.0000461-Geering1"></a>Geering W (1997) Anthrax in Australia. Kathmandu, Nepal.  <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0000461-Lowell1" id="pone.0000461-Lowell1"></a>Lowell JL, Wagner DM, Atshabar B, Antolin MF, Vogler AJ, et al.  (2005) Identifying sources of human exposure to plague. J Clin Microbiol  43: 650–656.  <ul class="find" data-citedArticleID="953144" data-doi="10.1128/jcm.43.2.650-656.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.2.650-656.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+sources+of+human+exposure+to+plague." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+sources+of+human+exposure+to+plague.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000461-Cheung1" id="pone.0000461-Cheung1"></a>Cheung DT, Kam KM, Hau KL, Au TK, Marston CK, et al.  (2005) Characterization of a Bacillus anthracis isolate causing a rare case of fatal anthrax in a 2-year-old boy from Hong Kong. J Clin Microbiol  43: 1992–1994.  <ul class="find" data-citedArticleID="953092" data-doi="10.1128/jcm.43.4.1992-1994.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.4.1992-1994.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+Bacillus+anthracis+isolate+causing+a+rare+case+of+fatal+anthrax+in+a+2-year-old+boy+from+Hong+Kong." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+Bacillus+anthracis+isolate+causing+a+rare+case+of+fatal+anthrax+in+a+2-year-old+boy+from+Hong+Kong.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000461-Worobey1" id="pone.0000461-Worobey1"></a>Worobey M (2005) Anthrax and the art of war (against ascertainment bias). Heredity  94: 459–460.  <ul class="find" data-citedArticleID="953170" data-doi="10.1038/sj.hdy.6800636"><li><a href="http://dx.doi.org/10.1038/sj.hdy.6800636" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anthrax+and+the+art+of+war+%28against+ascertainment+bias%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anthrax+and+the+art+of+war+%28against+ascertainment+bias%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000461-Mignot1" id="pone.0000461-Mignot1"></a>Mignot T, Mock M, Robichon D, Landier A, Lereclus D, et al.  (2001) The incompatibility between the PlcR- and AtxA-controlled regulons may have selected a nonsense mutation in Bacillus anthracis. Mol Microbiol  42: 1189–1198.  <ul class="find" data-citedArticleID="953148" data-doi="10.1046/j.1365-2958.2001.02692.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02692.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+incompatibility+between+the+PlcR-+and+AtxA-controlled+regulons+may+have+selected+a+nonsense+mutation+in+Bacillus+anthracis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+incompatibility+between+the+PlcR-+and+AtxA-controlled+regulons+may+have+selected+a+nonsense+mutation+in+Bacillus+anthracis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000461-Morin1" id="pone.0000461-Morin1"></a>Morin PA, Saiz R, Monjazeb A (1999) High-throughput single nucleotide polymorphism genotyping by fluorescent 5′ exonuclease activity. Biotechniques  27: 538–552.  <ul class="find" data-citedArticleID="953150"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=High-throughput+single+nucleotide+polymorphism+genotyping+by+fluorescent+5%26%23x2032%3B+exonuclease+activity.&amp;auth=&amp;atitle=High-throughput+single+nucleotide+polymorphism+genotyping+by+fluorescent+5%26%23x2032%3B+exonuclease+activity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-throughput+single+nucleotide+polymorphism+genotyping+by+fluorescent+5%26%23x2032%3B+exonuclease+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-throughput+single+nucleotide+polymorphism+genotyping+by+fluorescent+5%26%23x2032%3B+exonuclease+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000461-Zinser1" id="pone.0000461-Zinser1"></a>Zinser G (2002) Evolutionary relationships and mutation rate estimates in Bacillus anthracis. Flagstaff: Northern Arizona University.   <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0000461-Kumar1" id="pone.0000461-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment. Brief Bioinform  5: 150–163.  <ul class="find" data-citedArticleID="953140" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000461-Vogler1" id="pone.0000461-Vogler1"></a>Vogler AJ, Busch JD, Percy-Fine S, Tipton-Hunton C, Smith KL, et al.  (2002) Molecular analysis of rifampin resistance in Bacillus anthracis and Bacillus cereus. Antimicrob Agents Chemother  46: 511–513.  <ul class="find" data-citedArticleID="953168" data-doi="10.1128/aac.46.2.511-513.2002"><li><a href="http://dx.doi.org/10.1128/aac.46.2.511-513.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+analysis+of+rifampin+resistance+in+Bacillus+anthracis+and+Bacillus+cereus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+analysis+of+rifampin+resistance+in+Bacillus+anthracis+and+Bacillus+cereus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000461-Drake1" id="pone.0000461-Drake1"></a>Drake JW (1999) The distribution of rates of spontaneous mutation over viruses, prokaryotes, and eukaryotes. Ann N Y Acad Sci  870: 100–107.  <ul class="find" data-citedArticleID="953102" data-doi="10.1111/j.1749-6632.1999.tb08870.x"><li><a href="http://dx.doi.org/10.1111/j.1749-6632.1999.tb08870.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+distribution+of+rates+of+spontaneous+mutation+over+viruses%2C+prokaryotes%2C+and+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+distribution+of+rates+of+spontaneous+mutation+over+viruses%2C+prokaryotes%2C+and+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000461-Drake2" id="pone.0000461-Drake2"></a>Drake JW, Charlesworth B, Charlesworth D, Crow JF (1998) Rates of Spontaneous Mutation. Genetics  148: 1667–1686.  <ul class="find" data-citedArticleID="953104"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rates+of+Spontaneous+Mutation.&amp;auth=&amp;atitle=Rates+of+Spontaneous+Mutation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rates+of+Spontaneous+Mutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rates+of+Spontaneous+Mutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000461-Jones1" id="pone.0000461-Jones1"></a>Jones WI Jr, Klein F, Walker JS, Mahlandt BG, Dobbs JP, et al.  (1967) In vivo growth and distribution of anthrax bacilli in resistant, susceptible, and immunized hosts. J Bacteriol  94: 600–608.  <ul class="find" data-citedArticleID="953124"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+vivo+growth+and+distribution+of+anthrax+bacilli+in+resistant%2C+susceptible%2C+and+immunized+hosts.&amp;auth=&amp;atitle=In+vivo+growth+and+distribution+of+anthrax+bacilli+in+resistant%2C+susceptible%2C+and+immunized+hosts." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+growth+and+distribution+of+anthrax+bacilli+in+resistant%2C+susceptible%2C+and+immunized+hosts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+growth+and+distribution+of+anthrax+bacilli+in+resistant%2C+susceptible%2C+and+immunized+hosts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.XML" value="123055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.PDF" value="364834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g001.PNG_L" value="446187"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g001.PNG_M" value="73462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g001.PNG_S" value="2873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g001.TIF" value="739528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g001.PNG_I" value="31089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.t001.PNG_L" value="283876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.t001.PNG_M" value="73347"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.t001.PNG_S" value="3008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.t001.TIF" value="1182710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.t001.PNG_I" value="29287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g002.PNG_L" value="512899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g002.PNG_M" value="58651"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g002.PNG_S" value="3719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g002.TIF" value="928450"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g002.PNG_I" value="45806"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g003.PNG_L" value="1307550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g003.PNG_M" value="133390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g003.PNG_S" value="4261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g003.TIF" value="1460766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.g003.PNG_I" value="57974"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.t002.PNG_L" value="234137"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.t002.PNG_M" value="89141"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.t002.PNG_S" value="3255"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.t002.TIF" value="906688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.t002.PNG_I" value="37202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.e001.PNG" value="5682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.e001.TIF" value="26892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.e002.PNG" value="1903"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.e002.TIF" value="23284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.s001.XLS" value="145408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.s002.XLS" value="22528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.s003.DOC" value="60416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.s004.DOC" value="27648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.s005.DOC" value="25088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000461.s006.DOC" value="54784"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000461&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000461" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000461&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000461&volume=&issue=&title=Global Genetic Population Structure of Bacillus anthracis&author_name=Matthew%20N.%20Van%20Ert%2C%20W.%20Ryan%20Easterday%2C%20Lynn%20Y.%20Huynh%2C%20Richard%20T.%20Okinaka%2C%20Martin%20E.%20Hugh-Jones%2C%20Jacques%20Ravel%2C%20Shaylan%20R.%20Zanecki%2C%20Talima%20Pearson%2C%20Tatum%20S.%20Simonson%2C%20Jana%20M.%20U'Ren%2C%20Sergey%20M.%20Kachur%2C%20Rebecca%20R.%20Leadem-Dougherty%2C%20Shane%20D.%20Rhoton%2C%20Guenevier%20Zinser%2C%20Jason%20Farlow%2C%20Pamala%20R.%20Coker%2C%20Kimothy%20L.%20Smith%2C%20Bingxiang%20Wang%2C%20Leo%20J.%20Kenefic%2C%20Claire%20M.%20Fraser-Liggett%2C%20David%20M.%20Wagner%2C%20Paul%20Keim&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000461" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000461" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000461" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000461&amp;t=Global%20Genetic%20Population%20Structure%20of%20Bacillus%20anthracis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000461&title=Global%20Genetic%20Population%20Structure%20of%20Bacillus%20anthracis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000461&amp;title=Global%20Genetic%20Population%20Structure%20of%20Bacillus%20anthracis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000461" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000461&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Global Genetic Population Structure of Bacillus anthracis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000461';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Global%20Genetic%20Population%20Structure%20of%20Bacillus%20anthracis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000461" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000461" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
























          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Anthrax%22" title="Search for articles in the subject area:'Anthrax'"><div class="flagText">Anthrax</div></a>
              <div data-categoryid="34881" data-articleid="23760"
                   data-categoryname="Anthrax"
                   class="flagImage" title="Flag 'Anthrax' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacillus+anthracis%22" title="Search for articles in the subject area:'Bacillus anthracis'"><div class="flagText">Bacillus anthracis</div></a>
              <div data-categoryid="43355" data-articleid="23760"
                   data-categoryname="Bacillus anthracis"
                   class="flagImage" title="Flag 'Bacillus anthracis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+dominance%22" title="Search for articles in the subject area:'Genetic dominance'"><div class="flagText">Genetic dominance</div></a>
              <div data-categoryid="24915" data-articleid="23760"
                   data-categoryname="Genetic dominance"
                   class="flagImage" title="Flag 'Genetic dominance' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23760"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="23760"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="23760"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogeography%22" title="Search for articles in the subject area:'Phylogeography'"><div class="flagText">Phylogeography</div></a>
              <div data-categoryid="32107" data-articleid="23760"
                   data-categoryname="Phylogeography"
                   class="flagImage" title="Flag 'Phylogeography' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="23760"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4643'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9300'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1389&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=9271">Additional Funding Information</a><br>Posted by ochrenaka</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
