

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Synthetic Biology of Proteins: Tuning GFPs Folding and Stability with Fluoroproline</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001680"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001680" />

    <meta name="citation_title" content="Synthetic Biology of Proteins: Tuning GFPs Folding and Stability with Fluoroproline"/>
    <meta itemprop="name" content="Synthetic Biology of Proteins: Tuning GFPs Folding and Stability with Fluoroproline"/>

      <meta name="citation_author" content="Thomas Steiner"/>
            <meta name="citation_author_institution" content="Max Planck Institute of Biochemistry, Martinsried, Germany"/>
      <meta name="citation_author" content="Petra Hess"/>
            <meta name="citation_author_institution" content="Max Planck Institute of Biochemistry, Martinsried, Germany"/>
      <meta name="citation_author" content="Jae Hyun Bae"/>
            <meta name="citation_author_institution" content="Max Planck Institute of Biochemistry, Martinsried, Germany"/>
      <meta name="citation_author" content="Birgit Wiltschi"/>
            <meta name="citation_author_institution" content="Max Planck Institute of Biochemistry, Martinsried, Germany"/>
      <meta name="citation_author" content="Luis Moroder"/>
            <meta name="citation_author_institution" content="Max Planck Institute of Biochemistry, Martinsried, Germany"/>
      <meta name="citation_author" content="Nediljko Budisa"/>
            <meta name="citation_author_institution" content="Max Planck Institute of Biochemistry, Martinsried, Germany"/>

    <meta name="citation_date" content="2008/2/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001680.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1680"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Optimized codon usage and chromophore mutations provide enhanced sensitivity with the green fluorescent protein.; citation_author=T Yang; citation_author=L Cheng; citation_author=S Kain; citation_journal_title=Nucl Acids Res; citation_volume=24; citation_number=1; citation_pages=4592-4593; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Spectral variants of green fluorescent protein.; citation_author=GJ Palm; citation_author=A Wlodawer; citation_journal_title=Methods Enzymol; citation_volume=302; citation_number=2; citation_pages=378-394; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Crystal structure and refolding properties of the mutant F99S/M153T/V163A of the green fluorescent protein.; citation_author=R Battistutta; citation_author=A Negro; citation_author=G Zanotti; citation_journal_title=Proteins; citation_volume=41; citation_number=3; citation_pages=429-437; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Chromophore Formation in Green Fluorescent Protein.; citation_author=BG Reid; citation_author=GC Flynn; citation_journal_title=Biochemistry; citation_volume=36; citation_number=4; citation_pages=6786-6791; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Regulation of peptide bond cis/trans isomerization by enzyme catalysis and its implication in physiological processes.; citation_author=G Fischer; citation_author=T Aumüller; citation_number=5; citation_pages=105-150; citation_date=2004; citation_publisher=Springer Verlag; " />
      <meta name="citation_reference" content="citation_title=Proline Cis-Trans Isomerization and Protein Folding.; citation_author=WJ Wedemeyer; citation_author=E Welker; citation_author=HA Scheraga; citation_journal_title=Biochemistry; citation_volume=41; citation_number=6; citation_pages=14637-14644; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Peptidyl-Prolyl cis/trans Isomerases and Their Effectors.; citation_author=G Fischer; citation_journal_title=Angew Chem Int Ed Engl; citation_volume=33; citation_number=7; citation_pages=1415-1436; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Prolyl Isomerase: Enzymatic Catalysis of Slow Protein-Folding Reactions.; citation_author=FX Schmid; citation_journal_title=Annu Rev Biophys Biomol Struct; citation_volume=22; citation_number=8; citation_pages=123-143; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Consideration of the Possibility that the slow step in protein denaturation reactions is due to cis-trans isomerism of proline residues.; citation_author=JF Brandts; citation_author=HR Halvorson; citation_author=M Brennan; citation_journal_title=Biochemistry; citation_volume=14; citation_number=9; citation_pages=4953-4963; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=Cis-Trans Isomerization of Organic Molecules and Biomolecules: Implications and Applications.; citation_author=C Dugave; citation_author=L Demange; citation_journal_title=Chem Rev; citation_volume=103; citation_number=10; citation_pages=2475-2532; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cis peptide bonds in proteins: residues involved, their conformations, interactions and locations.; citation_author=D Pal; citation_author=P Chakrabarti; citation_journal_title=J Mol Biol; citation_volume=294; citation_number=11; citation_pages=271-288; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Pyrrolidine ring puckering in cis and trans-proline residues in proteins and polypeptides : Different puckers are favoured in certain situations.; citation_author=EJ Milner-White; citation_author=LH Bell; citation_author=PH Maccallum; citation_journal_title=J Mol Biol; citation_volume=228; citation_number=12; citation_pages=725-734; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Inductive Effects on the Energetics of Prolyl Peptide Bond Isomerization: Implications for Collagen Folding and Stability.; citation_author=ES Eberhardt; citation_author=N Panasik; citation_author=RT Raines; citation_journal_title=J Am Chem Soc; citation_volume=118; citation_number=13; citation_pages=12261-12266; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Fluoroprolines as Tools for Protein Design and Engineering.; citation_author=C Renner; citation_author=S Alefelder; citation_author=JH Bae; citation_author=N Budisa; citation_author=R Huber; citation_journal_title=Angew Chem Int Ed Engl; citation_volume=40; citation_number=14; citation_pages=923-925; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Code for collagen's stability deciphered.; citation_author=SK Holmgren; citation_author=KM Taylor; citation_author=LE Bretscher; citation_author=RT Raines; citation_journal_title=Nature; citation_volume=392; citation_number=15; citation_pages=666-667; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A hyperstable collagen mimic.; citation_author=SK Holmgren; citation_author=LE Bretscher; citation_author=KM Taylor; citation_author=RT Raines; citation_journal_title=Chem Biol; citation_volume=6; citation_number=16; citation_pages=63-70; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Folding of barstar C40A/C82A/P27A and catalysis of the peptidyl-prolyl cis/trans isomerization by human cytosolic cyclophilin (Cyp18).; citation_author=R Golbik; citation_author=G Fischer; citation_author=A Fersht; citation_journal_title=Protein Sci; citation_volume=8; citation_number=17; citation_pages=1505-1514; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Peptidyl Prolyl cis/trans-Isomerases: Comparative Reactivities of Cyclophilins, FK506-Binding Proteins, and Parvulins with Fluorinated Oligopeptide and Protein Substrates.; citation_author=R Golbik; citation_author=C Yu; citation_author=E Weyher-Stingl; citation_author=R Huber; citation_author=L Moroder; citation_journal_title=Biochemistry; citation_volume=44; citation_number=18; citation_pages=16026-16034; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Single Proline Residues can Dictate the Oxidative Folding Pathways of Cysteine-rich Peptides.; citation_author=C Boulègue; citation_author=AG Milbradt; citation_author=C Renner; citation_author=L Moroder; citation_journal_title=J Mol Biol; citation_volume=358; citation_number=19; citation_pages=846-856; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structural and Spectral Response of Green Fluorescent Protein Variants to Changes in pH.; citation_author=M-A Elsliger; citation_author=RM Wachter; citation_author=GT Hanson; citation_author=K Kallio; citation_author=SJ Remington; citation_journal_title=Biochemistry; citation_volume=38; citation_number=20; citation_pages=5296-5301; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Non-Canonical Amino Acids in Protein Polymer Design.; citation_author=RE Connor; citation_author=DA Tirrell; citation_journal_title=Polymer Rev; citation_volume=47; citation_number=21; citation_pages=9-28; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Natural history and experimental evolution of the genetic code.; citation_author=B Wiltschi; citation_author=N Budisa; citation_journal_title=Appl Microbiol Biotechnol; citation_volume=74; citation_number=22; citation_pages=739-753; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Refolding of inclusion body proteins.; citation_author=M Mayer; citation_author=J Buchner; citation_journal_title=Methods Mol Med; citation_volume=94; citation_number=23; citation_pages=239-254; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Engineering and characterization of a superfolder green fluorescent protein.; citation_author=J-D Pedelacq; citation_author=S Cabantous; citation_author=T Tran; citation_author=TC Terwilliger; citation_author=GS Waldo; citation_journal_title=Nat Biotechnol; citation_volume=24; citation_number=24; citation_pages=79-88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolution of a fluorinated green fluorescent protein.; citation_author=TH Yoo; citation_author=AJ Link; citation_author=DA Tirrell; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=25; citation_pages=13887-13890; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Chemical aspects of peptide bond isomerisation.; citation_author=G Fischer; citation_journal_title=Chem Soc Rev; citation_volume=29; citation_number=26; citation_pages=119-127; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A novel approach for studies of the molecular conformations in flexible polypeptides.; citation_author=K Wüthrich; citation_author=C Grathwohl; citation_journal_title=FEBS Lett; citation_volume=43; citation_number=27; citation_pages=337-340; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Multiple roles of prolyl residues in structure and folding.; citation_author=SJ Eyles; citation_author=LM Gierasch; citation_journal_title=J Mol Biol; citation_volume=301; citation_number=28; citation_pages=737-747; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cyclic Green Fluorescent Protein Produced in Vivo Using an Artificially Split PI-PfuI Intein from Pyrococcus furiosus.; citation_author=H Iwai; citation_author=A Lingel; citation_author=A Plückthun; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=29; citation_pages=16548-16554; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Primary structure of the Aequorea victoria green-fluorescent protein.; citation_author=DC Prasher; citation_author=VK Eckenrode; citation_author=WW Ward; citation_author=FG Prendergast; citation_author=MJ Cormier; citation_journal_title=Gene; citation_volume=111; citation_number=30; citation_pages=229-233; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Probing the role of tryptophans in Aequorea victoria green fluorescent proteins with an expanded genetic code.; citation_author=N Budisa; citation_author=PP Pal; citation_author=S Alefelder; citation_author=P Birle; citation_author=T Krywcun; citation_journal_title=Biol Chem; citation_volume=385; citation_number=31; citation_pages=191-202; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Organic Fluorine: Odd Man Out.; citation_author=JD Dunitz; citation_journal_title=Chembiochem; citation_volume=5; citation_number=32; citation_pages=614-621; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Atomic Mutations at the Single Tryptophan Residue of Human Recombinant Annexin V: Effects on Structure, Stability, and Activity.; citation_author=C Minks; citation_author=R Huber; citation_author=L Moroder; citation_author=N Budisa; citation_journal_title=Biochemistry; citation_volume=38; citation_number=33; citation_pages=10649-10659; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=High-level Biosynthetic Substitution of Methionine in Proteins by its Analogs 2-Aminohexanoic Acid, Selenomethionine, Telluromethionine and Ethionine in Escherichia coli.; citation_author=N Budisa; citation_author=B Steipe; citation_author=P Demange; citation_author=C Eckerskorn; citation_author=J Kellermann; citation_journal_title=Eur J Biochem; citation_volume=230; citation_number=34; citation_pages=788-796; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Processing of X-ray Diffraction Data Collected in Oscillation Mode.; citation_author=Z Otwinowski; citation_author=W Minor; citation_journal_title=Methods Enzymol; citation_volume=276; citation_number=35; citation_pages=307-326; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Crystallography & NMR System: A New Software Suite for Macromolecular Structure Determination.; citation_author=AT Brünger; citation_author=PD Adams; citation_author=GM Clore; citation_author=WL DeLano; citation_author=P Gros; citation_journal_title=Acta Crystallogr D Biol Crystallogr; citation_volume=54; citation_number=36; citation_pages=905-921; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Improved methods for building protein models in electron density maps and the location of errors in these models.; citation_author=TA Jones; citation_author=J-Y Zou; citation_author=SW Cowan; citation_author=M Kjeldgaard; citation_journal_title=Acta Crystallogr A; citation_volume=47; citation_number=37; citation_pages=110-119; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=PROCHECK: a program to check the stereochemical quality of protein structures.; citation_author=RA Laskowski; citation_author=MW MacArthur; citation_author=DS Moss; citation_author=JM Thornton; citation_journal_title=J Appl Crystallogr; citation_volume=26; citation_number=38; citation_pages=283-291; citation_date=1993; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001680" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Synthetic Biology of Proteins: Tuning GFPs Folding and Stability with Fluoroproline"/>
    <meta name="twitter:description" content="BackgroundProline residues affect protein folding and stability via cis/trans isomerization of peptide bonds and by the C&gamma;-exo or -endo puckering of their pyrrolidine rings. Peptide bond conformation as well as puckering propensity can be manipulated by proper choice of ring substituents, e.g. C&gamma;-fluorination. Synthetic chemistry has routinely exploited ring-substituted proline analogs in order to change, modulate or control folding and stability of peptides.Methodology/Principal FindingsIn order to transmit this synthetic strategy to complex proteins, the ten proline residues of enhanced green fluorescent protein (EGFP) were globally replaced by (4R)- and (4S)-fluoroprolines (FPro). By this approach, we expected to affect the cis/trans peptidyl-proline bond isomerization and pyrrolidine ring puckering, which are responsible for the slow folding of this protein. Expression of both protein variants occurred at levels comparable to the parent protein, but the (4R)-FPro-EGFP resulted in irreversibly unfolded inclusion bodies, whereas the (4S)-FPro-EGFP led to a soluble fluorescent protein. Upon thermal denaturation, refolding of this variant occurs at significantly higher rates than the parent EGFP. Comparative inspection of the X-ray structures of EGFP and (4S)-FPro-EGFP allowed to correlate the significantly improved refolding with the C&gamma;-endo puckering of the pyrrolidine rings, which is favored by 4S-fluorination, and to lesser extents with the cis/trans isomerization of the prolines.Conclusions/SignificanceWe discovered that the folding rates and stability of GFP are affected to a lesser extent by cis/trans isomerization of the proline bonds than by the puckering of pyrrolidine rings. In the C&gamma;-endo conformation the fluorine atoms are positioned in the structural context of the GFP such that a network of favorable local interactions is established. From these results the combined use of synthetic amino acids along with detailed structural knowledge and existing protein engineering methods can be envisioned as a promising strategy for the design of complex tailor-made proteins and even cellular structures of superior properties compared to the native forms."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001680.g005"/>

  <meta property="og:title" content="Synthetic Biology of Proteins: Tuning GFPs Folding and Stability with Fluoroproline" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001680" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7469'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6154'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=921&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001680">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001680" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001680&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Synthetic Biology of Proteins: Tuning GFPs Folding and Stability with Fluoroproline
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas Steiner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Planck Institute of Biochemistry, Martinsried, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Petra Hess, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Planck Institute of Biochemistry, Martinsried, Germany
                </p>

                <p>
                    Current address: Salutas Pharma GmbH, Barleben, Germany
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jae Hyun Bae, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Planck Institute of Biochemistry, Martinsried, Germany
                </p>

                <p>
                    Current address: Department of Pharmacology, Yale University School of Medicine, New Haven, Connecticut, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Birgit Wiltschi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Planck Institute of Biochemistry, Martinsried, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Luis Moroder, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Planck Institute of Biochemistry, Martinsried, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nediljko Budisa
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:budisa@biochem.mpg.de">budisa@biochem.mpg.de</a></p>

                <p>Affiliation:
                  Max Planck Institute of Biochemistry, Martinsried, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 27, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001680</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001680" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001680" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001680" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001680" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001680-g001" data-doi="info:doi/10.1371/journal.pone.0001680" data-uri="info:doi/10.1371/journal.pone.0001680.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001680-g002" data-doi="info:doi/10.1371/journal.pone.0001680" data-uri="info:doi/10.1371/journal.pone.0001680.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001680-g003" data-doi="info:doi/10.1371/journal.pone.0001680" data-uri="info:doi/10.1371/journal.pone.0001680.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001680-t001" data-doi="info:doi/10.1371/journal.pone.0001680" data-uri="info:doi/10.1371/journal.pone.0001680.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001680-g004" data-doi="info:doi/10.1371/journal.pone.0001680" data-uri="info:doi/10.1371/journal.pone.0001680.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001680-g005" data-doi="info:doi/10.1371/journal.pone.0001680" data-uri="info:doi/10.1371/journal.pone.0001680.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001680">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001680" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Proline residues affect protein folding and stability via <em>cis</em>/<em>trans</em> isomerization of peptide bonds and by the C<sup>γ</sup>-<em>exo</em> or -<em>endo</em> puckering of their pyrrolidine rings. Peptide bond conformation as well as puckering propensity can be manipulated by proper choice of ring substituents, e.g. C<sup>γ</sup>-fluorination. Synthetic chemistry has routinely exploited ring-substituted proline analogs in order to change, modulate or control folding and stability of peptides.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>In order to transmit this synthetic strategy to complex proteins, the ten proline residues of enhanced green fluorescent protein (EGFP) were globally replaced by (4<em>R</em>)- and (4<em>S</em>)-fluoroprolines (FPro). By this approach, we expected to affect the <em>cis</em>/<em>trans</em> peptidyl-proline bond isomerization and pyrrolidine ring puckering, which are responsible for the slow folding of this protein. Expression of both protein variants occurred at levels comparable to the parent protein, but the (4<em>R</em>)-FPro-EGFP resulted in irreversibly unfolded inclusion bodies, whereas the (4<em>S</em>)-FPro-EGFP led to a soluble fluorescent protein. Upon thermal denaturation, refolding of this variant occurs at significantly higher rates than the parent EGFP. Comparative inspection of the X-ray structures of EGFP and (4<em>S</em>)-FPro-EGFP allowed to correlate the significantly improved refolding with the C<sup>γ</sup>-<em>endo</em> puckering of the pyrrolidine rings, which is favored by 4<em>S</em>-fluorination, and to lesser extents with the <em>cis</em>/<em>trans</em> isomerization of the prolines.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We discovered that the folding rates and stability of GFP are affected to a lesser extent by <em>cis</em>/<em>trans</em> isomerization of the proline bonds than by the puckering of pyrrolidine rings. In the C<sup>γ</sup>-<em>endo</em> conformation the fluorine atoms are positioned in the structural context of the GFP such that a network of favorable local interactions is established. From these results the combined use of synthetic amino acids along with detailed structural knowledge and existing protein engineering methods can be envisioned as a promising strategy for the design of complex tailor-made proteins and even cellular structures of superior properties compared to the native forms.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Steiner T, Hess P, Bae JH, Wiltschi B, Moroder L, et al.  (2008) Synthetic Biology of Proteins: Tuning GFPs Folding and Stability with Fluoroproline. PLoS ONE 3(2):
          e1680.
            doi:10.1371/journal.pone.0001680</p><p><strong>Academic Editor: </strong>Wenqing Xu, University of Washington, United States of America</p><p><strong>Received:</strong> November 13, 2007; <strong>Accepted:</strong> January 27, 2008; <strong>Published:</strong> February 27, 2008</p><p><strong>Copyright:</strong> © 2008 Steiner et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the BioFuture Program of the Federal Ministry of Education and Research of Germany and by the Munich Center for Integrated Protein Science (CIPSM).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Enhanced green fluorescent protein (EGFP) is the Phe64Leu/Ser65Thr mutant of GFP <a href="#pone.0001680-Yang1">[1]</a> (<a href="#pone-0001680-g001">Fig. 1A</a>) and one of the most widely used autofluorescent tags in molecular and cell biology <a href="#pone.0001680-Palm1">[2]</a>. GFPs are frequently used as reporters for both <em>in vitro</em> and <em>in vivo</em> protein folding, but their (re)folding rates are known to be very slow (10–1000 s) <a href="#pone.0001680-Palm1">[2]</a>. Therefore, an improvement of the folding properties still represents a challenge for the design and engineering of fast folding autofluorescent proteins. GFPs contain ten proline residues in their primary sequence. These prolines affect the folding rates in a decisive manner because of their known slow <em>cis/trans</em> isomerization <a href="#pone.0001680-Battistutta1">[3]</a>–<a href="#pone.0001680-Brandts1">[9]</a>. We have therefore focused the present study on the role of these proline residues in the GFP folding process.</p>
<div class="figure" id="pone-0001680-g001"><div class="img"><a name="pone-0001680-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001680" data-uri="info:doi/10.1371/journal.pone.0001680.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001680.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001680.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Fluoroproline variants of EGFP.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>(A) Characteristic β-barrel structure of EGFP with the 10 Pro residues highlighted. Cro66 indicates the fluorophore. (B) Chemical structures of proline and the two proline analogs, (2<em>S</em>,4<em>S</em>)-4-fluoroproline ((4<em>S</em>)-FPro), and (2<em>S</em>,4<em>R</em>)-4-fluoroproline ((4<em>R</em>)-FPro). (C) Expression profile of EGFP and its 4-FPro variants in <em>E. coli</em>. EGFP and (4<em>S</em>)-FPro-EGFP are predominantly soluble, whereas (4<em>R</em>)-FPro-EGFP is insoluble. Purified EGFP was applied as the molecular weight marker (M) and is indicated by the arrow; S, soluble protein fraction; I, insoluble protein fraction. Proteins were separated by SDS-PAGE and stained with Coomassie Brillant Blue.</p>
<span>doi:10.1371/journal.pone.0001680.g001</span></div><a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Among the twenty naturally occurring amino acids, proline occupies a special place. Its five-membered pyrrolidine structure causes an exceptional conformational rigidity, which is responsible for the α-helix or β-sheet disrupting properties of this residue in proteins. More importantly, <em>cis</em>/<em>trans</em> isomerization of peptidyl-proline bonds is one of the rate-determining steps in protein folding <a href="#pone.0001680-Wedemeyer1">[6]</a>, <a href="#pone.0001680-Dugave1">[10]</a>. The pyrrolidine ring of proline adopts two alternative conformations that differ in the position of the C<sup>γ</sup> atom relative to the plane of the ring. These are referred to as either C<sup>γ</sup>-<em>exo</em> or C<sup>γ</sup>-<em>endo</em> pucker (<a href="#pone.0001680-Pal1">[11]</a>, <a href="#pone.0001680-MilnerWhite1">[12]</a> and references therein). The <em>cis</em> and <em>trans</em> peptidyl-proline bond conformation and the C<sup>γ</sup>-<em>exo</em> and C<sup>γ</sup>-<em>endo</em> pucker of the pyrrolidine ring are correlated properties in proteins <a href="#pone.0001680-Pal1">[11]</a>, <a href="#pone.0001680-MilnerWhite1">[12]</a>, which can be affected by appropriate ring substituents such as C<sup>γ</sup>(C-4) fluorine atoms. Indeed, (2<em>S</em>, 4<em>R</em>)-4-fluoroproline ((4<em>R</em>)-FPro) (<a href="#pone-0001680-g001">Fig. 1B</a>) favors by stereoelectronic effects the <em>trans</em> conformation and C<sup>γ</sup>-<em>exo</em> puckering, while the epimeric (2<em>S</em>, 4<em>S</em>)-4-fluoroproline ((4<em>S</em>)-FPro) (<a href="#pone-0001680-g001">Fig. 1B</a>) promotes the <em>cis</em> conformation and C<sup>γ</sup><em>-endo</em> puckering <a href="#pone.0001680-Eberhardt1">[13]</a>, <a href="#pone.0001680-Renner1">[14]</a>. These properties were exploited for the synthesis of hyperstable collagen triple helices by replacing the hydroxyproline residues with (4<em>R</em>)-FPro <a href="#pone.0001680-Holmgren1">[15]</a>, <a href="#pone.0001680-Holmgren2">[16]</a>. Conversely, with (4<em>S</em>)-FPro folding rates of the pseudo-wildtype barstar C40A/C82A/P27A mutant <a href="#pone.0001680-Golbik1">[17]</a> were enhanced and its structure stabilized by residue-specific replacement of the single Pro48 residue with (4<em>S</em>)-FPro to favor its <em>cis</em>-conformation <a href="#pone.0001680-Eberhardt1">[13]</a>, <a href="#pone.0001680-Golbik2">[18]</a>. Similarly, the folding rates of the N-terminal domain of minicollagen from <em>Hydra</em> nematocysts containing a single <em>cis</em> Pro bond were significantly and contrariwise affected by (4<em>R</em>)- or (4<em>S</em>)-FPro <a href="#pone.0001680-Boulgue1">[19]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Based on these previous experiences it was reasonable to expect a marked effect of the two stereochemically distinct fluoroprolines (4<em>R</em>)-FPro and (4<em>S</em>)-FPro on folding and stability of EGFP where out of the 10 Pro residues 9 are involved in <em>trans</em> and only one in a <em>cis</em> peptide bond (Pro89) <a href="#pone.0001680-Elsliger1">[20]</a>. Upon replacement of all Pro residues in EGFP by either (4<em>R</em>)-FPro or (4<em>S</em>)-FPro we were not only able to control protein folding, but also to dissect the contributions of various factors to the folding of a complex protein molecule.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>For replacement of all 10 Pro residues in EGFP, the residue-specific method for expansion of the amino acid repertoire <a href="#pone.0001680-Connor1">[21]</a>, <a href="#pone.0001680-Wiltschi1">[22]</a> was applied. The C-terminally (His)<sub>6</sub>-tagged EGFP was expressed in the Pro-auxotrophic <em>E. coli</em> K-12 strain JM83, in the presence of Pro, (4<em>S</em>)-FPro and (4<em>R</em>)-FPro, respectively. The parent EGFP and the variants (4<em>S</em>)-FPro-EGFP and (4<em>R</em>)-FPro-EGFP were formed in comparably good yields (<a href="#pone-0001680-g001">Fig. 1C</a>). Rather surprisingly, cell pellets with (4<em>R</em>)-FPro-EGFP were colorless, indicating deposition of unfolded non-fluorescent protein in inclusion bodies. Indeed, in an SDS-gel, the (4<em>R</em>)-FPro-EGFP variant was detected exclusively in the insoluble protein fraction (<a href="#pone-0001680-g001">Fig. 1C</a>), and all attempts for its recovery by standard inclusion body refolding protocols failed <a href="#pone.0001680-Mayer1">[23]</a>. Conversely, the parent EGFP and (4<em>S</em>)-FPro-EGFP were detected mainly in the soluble fraction as folded, fluorescent proteins (<a href="#pone-0001680-g001">Fig. 1C</a>). The C-terminally (His)<sub>6</sub>-tagged parent EGFP and (4<em>S</em>)-FPro-EGFP were purified from the soluble protein fraction and ESI-MS analysis confirmed that both proteins were isolated as monomers. In (4<em>S</em>)-FPro-EGFP all 10 Pro residues were replaced by (4<em>S</em>)-FPro (theoretical mass: 27924.5 Da; found mass: 27923.9±3.0 Da). This was further evidenced by the X-ray structure analysis of (4<em>S</em>)-FPro-EGFP (<a href="#pone-0001680-g002">Fig. 2</a>).</p>
<div class="figure" id="pone-0001680-g002"><div class="img"><a name="pone-0001680-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001680" data-uri="info:doi/10.1371/journal.pone.0001680.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001680.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001680.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Stereo image of the crystal structure of (4<em>S</em>)-FPro-EGFP.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>All Pro residues were replaced by (4<em>S</em>)-FPro. Fluoroprolines (13, 54, 56, 58, 75, 89, 187, 192, 196, 211) as well as the C- and N-termini (C, N) are indicated. The chromophore is shown in green and fluorines in cyan. Note that all fluorinated Pro residues except (4<em>S</em>)-FPro56 exhibit a C<sup>γ</sup>-<em>endo</em> pucker. Only (4<em>S</em>)-FPro56 shows a C<sup>γ</sup>-<em>exo</em> puckered pyrrolidine ring.</p>
<span>doi:10.1371/journal.pone.0001680.g002</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>For a comparative analysis of the folding properties of EGFP and (4<em>S</em>)-FPro-EGFP, the proteins were unfolded in boiling 8 M urea and then refolded at room temperature after 100-fold dilution into buffer. Refolding kinetics were monitored fluorometrically over a time period of at least 30 min, and the refolding efficiency was assessed after 24 h incubation under non-denaturing conditions. (4<em>S</em>)-FPro-EGFP recovered more than 95% of its fluorescence before denaturation (<a href="#pone-0001680-g003">Fig. 3A</a>), whereas the parent EGFP retrieved only up to 60% of its initial fluorescence (<a href="#pone-0001680-g003">Fig. 3B</a>). The refolding kinetics of (4<em>S</em>)-FPro-EGFP and EGFP (<a href="#pone-0001680-g003">Fig. 3C</a>) show an initial fast phase with rate constants of 3.01×10<sup>−2</sup> s<sup>−1</sup> and 1.41×10<sup>−2</sup> s<sup>−1</sup>, respectively, that is followed by a slower refolding phase (rate constants 0.36×10<sup>−2</sup> s<sup>−1</sup> and 0.15×10<sup>−2</sup> s<sup>−1</sup>, respectively; <a href="#pone-0001680-g003">Fig. 3C</a>). Surprisingly, (4<em>S</em>)-FPro-EGFP exhibited superior refolding properties when compared to the parent EGFP as the rate is 2.1 times faster than that of the parent EGFP in both phases. A ‘superfolder’ GFP mutant has been reported by Pédelacq <em>et al.</em> <a href="#pone.0001680-Pedelacq1">[24]</a>, which refolds upon thermal denaturation in a two-step process as well, with an initial fast rate of 5.0×10<sup>−1</sup> s<sup>−1</sup>, which is one order of magnitude faster than that of (4<em>S</em>)-FPro-EGFP. This GFP mutant was developed from an already well-folding ‘cycle 3’ GFP mutant and relative to this parent mutant a 3.5 times enhanced folding rate was achieved.</p>
<div class="figure" id="pone-0001680-g003"><div class="img"><a name="pone-0001680-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001680" data-uri="info:doi/10.1371/journal.pone.0001680.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001680.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001680.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Fluorescence recovery of EGFP and (4<em>S</em>)-FPro-EGFP.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>The proteins were denatured by boiling (95°C, 5 min) in 8 M urea and refolded by 100-fold dilution into the buffer without urea (see <a href="#s3">Methods</a> section for details). Fluorescence emission profiles of (A) (4<em>S</em>)-FPro-EGFP and (B) EGFP upon excitation of the chromophore at 488 nm before denaturation and after 24 h refolding at room temperature. (4<em>S</em>)-FPro-EGFP recovers more than 95% of its fluorescence before denaturation, whereas EGFP recovers only up to 60% of its initial fluorescence (this is in agreement with literature data). (C) The refolding kinetics of both proteins starts with an initial fast phase that is followed by a slow refolding phase. (4<em>S</em>)-FPro-EGFP refolds approximately 2 times faster than EGFP. The percentage of refolding was calculated on the basis of the final constant amount of fluorescence, corresponding to 100% of refolding. Normalized fluorescence in arbitrary units (au) was plotted against time.</p>
<span>doi:10.1371/journal.pone.0001680.g003</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>From the results it is evident that, in contrast to our expectation, incorporation of (4<em>R</em>)-FPro into EGFP, which should favor the <em>trans</em> conformation of 9 Pro bonds, interferes with a correct protein folding, whereas (4<em>S</em>)-FPro with its opposite effect obviously does not. It is at least equally well accommodated by the protein structure as Pro and it enhances EGFP folding rates, whereas incorporation of (4<em>R</em>)-FPro apparently leads to effects that exceed the plasticity of the protein structure, and thus to irreversibly unfolded protein. A similar observation was reported recently by the group of Tirrell <a href="#pone.0001680-Yoo1">[25]</a>. Indeed, global replacement of the leucine residues in GFP with 5,5,5-trifluoroleucine resulted in unfolded protein, and proper folding could only be restored after evolution of a mutant GFP that was able to accommodate the fluorinated residues <a href="#pone.0001680-Yoo1">[25]</a>. In contrast to our and Pédelacq's superfolding GFP, the physical and spectroscopic properties of the fluorinated GFP mutant evolved by Tirrell's group were not superior to those of the parent GFP <a href="#pone.0001680-Yoo1">[25]</a>.</p>
<a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>An inspection of the EGFP structure (PDB entry: 1EMG <a href="#pone.0001680-Elsliger1">[20]</a>) reveals that except Pro89 all other Pro residues are involved in <em>trans</em> peptide bonds. However, the resolution of the structure does not allow unambiguous assignment of the proline puckers. Indeed, in the process of the 3D-structure elucidation the C<sup>γ</sup>-<em>exo</em> or -<em>endo</em> puckering attracts little attention due to its low relevance for the overall crystallographic data quality. In folded proteins ~5% of peptidyl-Pro bonds are in the <em>cis</em> conformation as derived by inspection of the protein structure database (<a href="#pone.0001680-Fischer3">[26]</a> and references therein). In unstructured polypeptide chains the content of <em>cis</em> peptidyl-Pro bonds can reach significantly higher values particularly with aromatic amino acids directly preceding the Pro residues <a href="#pone.0001680-Brandts1">[9]</a>, <a href="#pone.0001680-Fischer3">[26]</a>, <a href="#pone.0001680-Wthrich1">[27]</a>. It is well known that <em>trans/cis</em> isomerization dramatically affects folding kinetics of proteins with native <em>cis</em> peptidyl-Pro bonds <a href="#pone.0001680-Fischer1">[5]</a>–<a href="#pone.0001680-Brandts1">[9]</a>, <a href="#pone.0001680-Eyles1">[28]</a>. Correspondingly, the fast initial refolding phase of EGFP should involve molecules with the Met88-Pro89 bond already in <em>cis</em> conformation whereas the slow phase should originate from denatured protein molecules with this peptide bond in <em>trans</em> <a href="#pone.0001680-Iwai1">[29]</a>. In our (4<em>S</em>)-FPro-EGFP variant, the <em>cis</em> conformation of Met88-(4<em>S</em>)-FPro89 bond is favored and, thus, should accelerate the folding rates. In contrast to the experimental findings, the (4<em>S</em>)-FPro residues in all other nine positions were expected to disfavor the <em>trans</em> conformation and, thus, refolding properties. Obviously, the enhanced refolding rates of (4<em>S</em>)-FPro-EGFP have to originate from other factors.</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>The crystal structure of (4<em>S</em>)-FPro-EGFP (PDB entry: 2Q6P) solved at 2.1 Å resolution (<a href="#pone-0001680-g002">Fig. 2</a>; refinement statistics are reported in <a href="#pone-0001680-t001">Table 1</a>; for details on crystallization conditions see <a href="#s3">Materials and Methods</a>) confirmed that incorporation of the 10 (4<em>S</em>)-FPro residues did not affect the overall protein fold. All (4<em>S</em>)-FPro residues display C<sup>γ</sup>-<em>endo</em> puckered pyrrolidine rings apart from Pro56, which adopts a C<sup>γ</sup>-<em>exo</em> configuration. Indeed, the fluoroprolines are well defined and characterized by low B-factors indicating rigid local conformations in the protein matrix. As outlined above, (4<em>S</em>)-fluorination of Pro promotes C<sup>γ</sup>-<em>endo</em> puckering. Apparently, such spatial display with preferred C<sup>γ</sup>-<em>endo</em> puckering of 9 out of 10 Pro residues dramatically improves the folding properties.</p>
<div class="figure" id="pone-0001680-t001"><div class="img"><a name="pone-0001680-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001680" data-uri="info:doi/10.1371/journal.pone.0001680.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001680.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001680.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>X-ray data collection and refinement statistics.</span></strong></p><span>doi:10.1371/journal.pone.0001680.t001</span></div><a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>Among the 10 Pro residues, five (13, 75, 89,192 and 211) are surface-exposed in EGFP, one is partially exposed (187) and the other residues are buried in the protein core (54, 56, 58 and 196). Fluorination of the buried residues increases their hydrophobicity and thus stabilizes the folded protein. We observed that the fluorinated EGFP was less prone to aggregation over the time and that related samples crystallized faster (overnight) than those of the parent protein (few days). Three of the buried proline residues are located in the characteristic proline-rich pentapeptide (4<em>S</em>)-FPro54-Val55-(4<em>S</em>)-FPro56-Trp57-(4<em>S</em>)-FPro58 (PVPWP motif; <a href="#pone-0001680-g004">Fig. 4</a>). The average B-factors for the prolines in the PVPWP motif as well as of the chromophore atoms are generally low (~12 Å<sup>2</sup> in (4<em>S</em>)-FPro-EGFP). Furthermore, in the crystalline state of (4<em>S</em>)-FPro-EGFP neighboring residues of fluorinated PVPWP exhibit lower average B-factors (~5–7 Å<sup>2</sup>) as well.</p>
<div class="figure" id="pone-0001680-g004"><div class="img"><a name="pone-0001680-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001680" data-uri="info:doi/10.1371/journal.pone.0001680.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001680.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001680.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>X-ray structure of the proline-rich pentapeptide (4<em>S</em>)-FPro54-Val55-(4<em>S</em>)-FPro56-Trp57-(4<em>S</em>)-FPro58 (PVPWP).</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>The continuous electron density (grey, 2Fo-Fc; contouring levels 1 σ) indicates fluorine atoms at the 4<em>S</em>-position in three buried Pro residues (54, 56, 58). Their experimental electron densities are localized unambiguously (image preparation with PYMOL (<a href="http://pymol.sourceforge.net/">http://pymol.sourceforge.net/</a>)). Out of the three Pro residues forming <em>trans</em> peptide bonds, only Pro56 exhibits predominant C<sup>γ</sup>-<em>exo</em> pucker whereas the other two have pyrrolidine rings with C<sup>γ</sup>-<em>endo</em> conformation. The rigid local secondary structure of this motif forces the (4<em>S</em>)-fluorinated pyrrolidine ring of (4<em>S</em>)-FPro56 into a stereochemically unfavorable C<sup>γ</sup>-<em>exo</em> pucker.</p>
<span>doi:10.1371/journal.pone.0001680.g004</span></div><a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>The presence of the PVPWP pentapeptide in the GFP sequence has long been recognized <a href="#pone.0001680-Prasher1">[30]</a>, however, its significance is still unclear. Searching different protein databases (SwissProt, NCBI databases) we found the PVPWP motif in various proteins as different as cytochromes and eukaryotic voltage-activated potassium channels. Furthermore, we observed that the PVPWP motif is crucial for the EGFP function since site directed mutagenesis of Val55 abolishes protein fluorescence (unpublished data). Similarly, Trp57 cannot be replaced by any of the other 19 amino acids <a href="#pone.0001680-Budisa1">[31]</a>. Thus, we speculate that the function of this proline-rich pentapeptide in GFP is to control the spatial orientation of the relatively bulky hydrophobic Val55 and Trp57 side chains. This is required for protecting the fluorophore from collisional quenching, e.g., by oxygen or other diffusible ligands.</p>
<a id="article1.body1.sec2.p8" name="article1.body1.sec2.p8"></a><p>The (4<em>S</em>) H→F replacement in Pro residues endows the pyrrolidine rings with large dipole moments because of the highly polar C–F bonds. This may promote strong dipole interactions in the local environments with polar groups such as amides, hydroxy or carbonyl groups. Indeed, 12 new interactions were detected in the (4<em>S</em>)-FPro-EGFP structure that were not present in EGFP (see <a href="#pone-0001680-g005">Fig. 5</a>). The majority of the fluorine atoms in (4<em>S</em>)-FPro-EGFP is involved in interactions with hydrogen atoms from neighboring backbone -NH- groups on their ‘own’ strand or on strands in the near vicinity (<a href="#pone-0001680-g005">Fig. 5A–F</a>). Only for the 4<em>S</em>-fluorine atoms at positions 187 and 192 direct interactions could not be detected. As outlined above, (4<em>S</em>)-FPro56 is the only fluoroproline having a C<sup>γ</sup>-<em>exo</em> pucker. This puckering directs the (4<em>S</em>)-fluorine atom towards an unfavorable position as it is involved in a repulsive interaction (3.07 Å) with the backbone carbonyl group of Asn153 on the neighboring strand (<a href="#pone-0001680-g005">Fig. 5B</a>). However, the destabilizing effect of this repulsion is apparently largely outweighed by the other stabilizing interactions. The crystallographic distances detected in the (4<em>S</em>)-FPro-EGFP structure are well compatible with C–F—H–N/O electrostatic interactions, which are more favorable in the <em>endo</em> than they would be in the <em>exo</em> pucker conformation. We are well aware that hydrogen bonding to organic fluorine is a matter of considerable controversy <a href="#pone.0001680-Dunitz1">[32]</a>; thus, higher resolution three-dimensional structures of fluorinated proteins are required to shed more light on this disputed matter.</p>
<div class="figure" id="pone-0001680-g005"><div class="img"><a name="pone-0001680-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001680" data-uri="info:doi/10.1371/journal.pone.0001680.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001680.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001680.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001680.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001680.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001680.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Local microenvironments of the fluorinated prolines in (4<em>S</em>)-FPro-EGFP.</span></strong></p><a id="article1.body1.sec2.fig4.caption1.p1" name="article1.body1.sec2.fig4.caption1.p1"></a><p>The high resolution (2.1 Å) X-ray crystallographic structure of (4<em>S</em>)-FPro-EGFP allowed identification of new interactions introduced by 4<em>S</em>-fluorination. The fluorine atoms are characterized by well defined electron densities at the H→F replacement sites and facilitated unambiguous determination of the conformation of the pyrrolidine rings (see also <a href="#pone-0001680-g002">Fig. 2</a>). Fluorines are cyan, the new interactions are shown in yellow except one repulsive interaction, which is indicated in grey. All images were prepared with PYMOL (<a href="http://pymol.sourceforge.net/">http://pymol.sourceforge.net/</a>). (A) (4<em>S</em>)-FPro13 interacts with the backbone -NH- of Asp117 (3.46 Å) and with Oγ2 of Thr118 (3.03 Å) on the neighboring strand. (B) The fluorinated PVPWP motif: the 4<em>S</em>-fluorine of (4<em>S</em>)-FPro56 is in a stereochemically unfavored position; it is most probably involved in a repulsive interaction with the backbone carbonyl group of Asn153 on the neighboring strand (measured crystallographic distance: 3.07 Å). The other two fluorinated prolines are involved in dipole interactions with neighboring backbone -NH- groups: (4<em>S</em>)-FPro54 with Val55 (3.40 Å) and (4<em>S</em>)-FPro58 with Thr59 (3.17 Å). (C) (4<em>S</em>)-FPro75 interacts with the backbone -NH- of Met78 (3.32 Å) and establishes a contact (~3.4 Å) with the -NH- of the His77 imidazole ring. (D) (4<em>S</em>)-FPro196 interacts with the backbone -NH- of the adjacent Ala154 (3.43 Å), and (E) (4<em>S</em>)-FPro89 with that of the succeeding Glu90 (3.46 Å). Finally, (F) (4<em>S</em>)-FPro211 interacts with both, the backbone -NH- (3.40 Å) and Nб (3.06 Å) of the succeeding Asn212. In total, the fluorine atoms in (4<em>S</em>)-FPro-EGFP establish 12 novel interactions that are absent in EGFP.</p>
<span>doi:10.1371/journal.pone.0001680.g005</span></div><a id="article1.body1.sec2.p9" name="article1.body1.sec2.p9"></a><p>It is obvious that fluorination of the Pro residues in EGFP is the main source of the superior refolding rates which may originate from several synergistic effects. The energy difference between the <em>cis</em> and <em>trans</em> Pro bond conformation is significantly larger for the <em>exo</em> than for the <em>endo</em> pucker and the activation energy for the <em>cis</em>→<em>trans</em> isomerization of (4<em>S</em>)-FPro is almost identical to that of Pro <a href="#pone.0001680-Renner1">[14]</a>. Correspondingly, the <em>cis</em>/<em>trans</em> isomerization of the proline bonds affects folding of the EGFP variant to significantly lesser extents than the preferred <em>endo</em> puckering of the (4<em>S</em>)-FPro pyrrolidine ring. In the structural context, this generates stabilizing interactions of the fluorine atoms in (4<em>S</em>)-FPro-EGFP that are absent in the parent EGFP. Conversely, the “superfolding GFP” of Pédelacq <em>et al.</em> <a href="#pone.0001680-Pedelacq1">[24]</a> was generated by random mutagenesis of amino acids that likewise resulted in intramolecular interaction networks not present in native GFP.</p>
<a id="article1.body1.sec2.p10" name="article1.body1.sec2.p10"></a><p>Although our study reports a serendipitous discovery, we are convinced that the insights we gained here can be generally useful for the design and engineering of proteins where proline residues play decisive roles. We think that the stability of a protein can be rationally manipulated by choosing the appropriate amino acid to fit into the 3D fine structure of the target protein. Alternatively, the structure of a target protein can be optimized for the incorporation of synthetic amino acids by guided evolution <a href="#pone.0001680-Yoo1">[25]</a>. Thus, detailed protein models and classical engineering methods in combination with an expanded genetic code could open a new era of synthetic biology.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Chemicals, analog incorporation, fermentation and protein purification</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The amino acids Pro, (4<em>R</em>)-FPro and (4<em>S</em>)-FPro were purchased from Bachem AG (Bubendorf, Switzerland). Unless otherwise stated all chemicals were from Sigma (Steinheim, Germany) or Merck (Merck KGaA, Darmstadt, Germany). The Pro-auxotrophic <em>Escherichia coli</em> K-12 strain JM83 from ATCC (catalogue number 35607; genotype: F<sup>−</sup> Δ(<em>lac</em>-<em>pro</em>AB) φ80 Δ(<em>lac</em>Z) M15 <em>ara rps</em>L <em>thi</em> λ<sup>−</sup>) served as the host organism for expression experiments. The EGFP expression plasmid was constructed as follows: The <em>Nco</em>I-<em>Hind</em>III fragment form the pEGFP plasmid (BD Biosciences, San Jose, CA) comprising the complete EGFP (Phe64Leu/Ser65Thr) coding sequence (with 10 coding triplets for the amino acid proline) was inserted into the pQE60 vector (Qiagen, Hilden, Germany) cleaved with the same enzymes. EGFP is expressed with a C-terminal (His)<sub>6</sub>-tag from the resulting expression vector pQE60-EGFP. Cells were routinely co-transformed with pREP4 (Qiagen) encoding the repressor gene <em>lacI<sup>q</sup></em> and pQE60-EGFP (lacking <em>lacI<sup>q</sup></em>). Transformed host cells <em>E. coli</em> JM83 were grown in New Minimal Medium <a href="#pone.0001680-Minks1">[33]</a>, <a href="#pone.0001680-Budisa2">[34]</a> (NMM) which contains 22 mM KH<sub>2</sub>PO<sub>4</sub>, 50 mM K<sub>2</sub>HPO<sub>4</sub>, 8.5 mM NaCl, 7.5 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 1 mM MgSO<sub>4</sub>, 20 mM glucose, 1 µg/ml Ca<sup>2+</sup>, 1 µg/ml Fe<sup>2+</sup>, 0.001 µg/ml trace elements (Cu<sup>2+</sup>, Zn<sup>2+</sup>, Mn<sup>2+</sup>, MoO<sub>4</sub><sup>2−</sup>), 10 µg/ml thiamine and 10 µg/ml biotin and the appropriate antibiotics (100 µg/ml ampicillin and 70 µg/ml kanamycin). The cells were first grown in NMM in the presence of 0.05 mM Pro as the natural substrate until its depletion from the culture in the mid-logarithmic growth phase (OD<sub>600</sub> 0.5–0.8) as described elsewhere <a href="#pone.0001680-Budisa2">[34]</a>. At that point, 1 mM of the non-canonical analog was added and at the same time the translation of the target gene product induced. Expression of EGFP in the presence of (4<em>S</em>)-FPro and (4<em>R</em>)-FPro produced proteins in yields similar to that of the parent EGFP protein (10–30 mg/L), although (4<em>R</em>)-FPro-EGFP was deposited in inclusion bodies. EGFP and its variant (4<em>S</em>)-FPro-EGFP were purified by two successive chromatographic steps: (i) Ni-NTA agarose (Qiagen) followed by elution with an imidazole gradient (0–100 mM) in 100 mM Na-phosphate buffer pH 8.0 and 0.5 M NaCl, and (ii) phenyl-sepharose (GE Healthcare Bio-Sciences AB, Uppsala, Sweden) eluted with an ammonium sulphate gradient 20–0% in 20 mM TrisCl, pH 8.0 and 1 mM EDTA.</p>


<h4>Crystallization and Structure Elucidation</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The (4<em>S</em>)-FPro-EGFP was crystallized under the same conditions as the parent protein: (4<em>S</em>)-FPro-EGFP (16 mg/ml) was crystallized in 0.2 M Mg<sup>2+</sup> acetate, 0.1 M sodium cacodylate, pH 6.5, and 13% (w/v) polyethylene glycol 8000 using the sitting drop vapor diffusion method. 2 µl of protein solution were mixed with 1 µl of precipitant solution at 20°C. The structure of the EGFP variant was solved by the molecular replacement technique using 1EMG <a href="#pone.0001680-Elsliger1">[20]</a> as a model. The data set was collected on an X-ray image plate system (Mar Research, Hamburg, Germany) using CuKα-radiation generated by a Rigaku rotating anode at 100°K. Crystals were transferred to their mother solution containing 20% (v/v) glycerol as a cryo-protectant and shock-frozen in a nitrogen stream.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Reflections were integrated with the program DENZO, scaled and reduced using SCALEPACK <a href="#pone.0001680-Otwinowski1">[35]</a>. Model building and refinement was performed with CNS <a href="#pone.0001680-Brnger1">[36]</a>. The initial model was refined by alternating automatic minimization protocols performed with CNS inspecting visual electron density map and manually adjusted using the program O <a href="#pone.0001680-Jones1">[37]</a>. Except for a small part of the N- and the C-terminus the whole model (G4-I229) could be built. The data collection and refinement statistics are presented in <a href="#pone-0001680-t001">Table 1</a>. Accession codes: Protein Data Bank: Coordinates and structure factor amplitudes were deposited with accession code 2Q6P.</p>


<h4>UV-absorbance and fluorescence of proteins</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>UV-absorption spectra of the proteins in phosphate buffered saline (PBS; 137 mM NaCl, 2.7 mM KCl, 4.3 mM Na<sub>2</sub>HPO<sub>4</sub>·7H<sub>2</sub>O, 1.4 mM KH<sub>2</sub>PO<sub>4</sub>, pH~7.3) were recorded at 20°C on a Perkin-Elmer Lambda 17 UV/VIS spectrophotometer. Fluorescence spectra were excited at 488 nm by using excitation/emission slits of 5.0 nm and were recorded on a Perkin-Elmer spectrometer (LS50B) equipped with digital software. Protein concentrations were determined as described elsewhere <a href="#pone.0001680-Budisa2">[34]</a>.</p>


<h4>Denaturation and refolding of the different GFP variants</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>GFPs are generally conformationally very stable proteins once their structures have formed. Their denaturation only occurs under extremely harsh conditions, e.g., strong denaturants in combination with high temperature. Denaturation of purified (4<em>S</em>)-FPro-EGFP and EGFP (30 µM each) was performed in PBS containing 8 M urea and 5 mM DTT for 5 min at 95°C. Urea-denatured samples were renatured at room temperature by 100-fold dilution into PBS with 5 mM DTT but without urea. Protein refolding was monitored for 30 min by fluorescence recovery at 509 nm by using the option ‘Timedrive’ of Perkin-Elmer spectrometer (LS50B) with an interval of 3 sec and a slit of 2.5 nm. The concentrations of denatured proteins were adjusted so that the dilution yielded about 0.3 µM protein. Raw data were imported into Origin 6.1 (OriginLab Corporation, Northampton, MA) and normalized before plotting. Data were fitted with Sigma Plot (Systat Software Inc., San Jose, CA) using equations as described elsewhere <a href="#pone.0001680-Pedelacq1">[24]</a>.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>In order to assess the end point fluorescence recovery of EGFP and (4<em>S</em>)-FPro-EGFP, fluorescence spectra were recorded before denaturation and after renaturation at room temperature for 24 h.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Petra Birle, Tatjana Krywcun and Waltraud Wenger for their invaluable technical assistance in protein preparation and crystallization. We are grateful to Dr. Christian Benda and Dr. Christian Renner for discussions on the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: NB. Performed the experiments: PH JB. Analyzed the data: LM TS BW. Contributed reagents/materials/analysis tools: PH. Wrote the paper: LM BW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001680-Yang1" id="pone.0001680-Yang1"></a>Yang T, Cheng L, Kain S (1996) Optimized codon usage and chromophore mutations provide enhanced sensitivity with the green fluorescent protein. Nucl Acids Res  24: 4592–4593.  <ul class="find" data-citedArticleID="1063140" data-doi="10.1093/nar/24.22.4592"><li><a href="http://dx.doi.org/10.1093/nar/24.22.4592" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimized+codon+usage+and+chromophore+mutations+provide+enhanced+sensitivity+with+the+green+fluorescent+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimized+codon+usage+and+chromophore+mutations+provide+enhanced+sensitivity+with+the+green+fluorescent+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001680-Palm1" id="pone.0001680-Palm1"></a>Palm GJ, Wlodawer A (1999) Spectral variants of green fluorescent protein. Methods Enzymol  302: 378–394.  <ul class="find" data-citedArticleID="1063122" data-doi="10.1016/s0076-6879(99)02034-0"><li><a href="http://dx.doi.org/10.1016/s0076-6879(99)02034-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spectral+variants+of+green+fluorescent+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spectral+variants+of+green+fluorescent+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001680-Battistutta1" id="pone.0001680-Battistutta1"></a>Battistutta R, Negro A, Zanotti G (2000) Crystal structure and refolding properties of the mutant F99S/M153T/V163A of the green fluorescent protein. Proteins  41: 429–437.  <ul class="find" data-citedArticleID="1063068" data-doi="10.1002/1097-0134(20001201)41:4<429::aid-prot10&gt;3.0.co;2-d"><li><a href="http://dx.doi.org/10.1002/1097-0134(20001201)41:4<429::aid-prot10&gt;3.0.co;2-d" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+and+refolding+properties+of+the+mutant+F99S%2FM153T%2FV163A+of+the+green+fluorescent+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+and+refolding+properties+of+the+mutant+F99S%2FM153T%2FV163A+of+the+green+fluorescent+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001680-Reid1" id="pone.0001680-Reid1"></a>Reid BG, Flynn GC (1997) Chromophore Formation in Green Fluorescent Protein. Biochemistry  36: 6786–6791.  <ul class="find" data-citedArticleID="1063128" data-doi="10.1021/bi970281w"><li><a href="http://dx.doi.org/10.1021/bi970281w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromophore+Formation+in+Green+Fluorescent+Protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromophore+Formation+in+Green+Fluorescent+Protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001680-Fischer1" id="pone.0001680-Fischer1"></a>Fischer G, Aumüller T (2004) Regulation of peptide bond cis/trans isomerization by enzyme catalysis and its implication in physiological processes. Reviews of Physiology, Biochemistry and Pharmacology. Berlin: Springer Verlag. pp. 105–150.  <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0001680-Wedemeyer1" id="pone.0001680-Wedemeyer1"></a>Wedemeyer WJ, Welker E, Scheraga HA (2002) Proline Cis-Trans Isomerization and Protein Folding. Biochemistry  41: 14637–14644.  <ul class="find" data-citedArticleID="1063134" data-doi="10.1021/bi020574b"><li><a href="http://dx.doi.org/10.1021/bi020574b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proline+Cis-Trans+Isomerization+and+Protein+Folding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proline+Cis-Trans+Isomerization+and+Protein+Folding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001680-Fischer2" id="pone.0001680-Fischer2"></a>Fischer G (1994) Peptidyl-Prolyl <em>cis</em>/<em>trans</em> Isomerases and Their Effectors. Angew Chem Int Ed Engl  33: 1415–1436.  <ul class="find" data-citedArticleID="1063094" data-doi="10.1002/anie.199414151"><li><a href="http://dx.doi.org/10.1002/anie.199414151" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peptidyl-Prolyl+cis%2Ftrans+Isomerases+and+Their+Effectors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peptidyl-Prolyl+cis%2Ftrans+Isomerases+and+Their+Effectors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001680-Schmid1" id="pone.0001680-Schmid1"></a>Schmid FX (1993) Prolyl Isomerase: Enzymatic Catalysis of Slow Protein-Folding Reactions. Annu Rev Biophys Biomol Struct  22: 123–143.  <ul class="find" data-citedArticleID="1063132" data-doi="10.1146/annurev.bb.22.060193.001011"><li><a href="http://dx.doi.org/10.1146/annurev.bb.22.060193.001011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prolyl+Isomerase%3A+Enzymatic+Catalysis+of+Slow+Protein-Folding+Reactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prolyl+Isomerase%3A+Enzymatic+Catalysis+of+Slow+Protein-Folding+Reactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001680-Brandts1" id="pone.0001680-Brandts1"></a>Brandts JF, Halvorson HR, Brennan M (1975) Consideration of the Possibility that the slow step in protein denaturation reactions is due to cis-trans isomerism of proline residues. Biochemistry  14: 4953–4963.  <ul class="find" data-citedArticleID="1063072" data-doi="10.1021/bi00693a026"><li><a href="http://dx.doi.org/10.1021/bi00693a026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consideration+of+the+Possibility+that+the+slow+step+in+protein+denaturation+reactions+is+due+to+cis-trans+isomerism+of+proline+residues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consideration+of+the+Possibility+that+the+slow+step+in+protein+denaturation+reactions+is+due+to+cis-trans+isomerism+of+proline+residues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001680-Dugave1" id="pone.0001680-Dugave1"></a>Dugave C, Demange L (2003) Cis-Trans Isomerization of Organic Molecules and Biomolecules: Implications and Applications. Chem Rev  103: 2475–2532.  <ul class="find" data-citedArticleID="1063082" data-doi="10.1021/cr0104375"><li><a href="http://dx.doi.org/10.1021/cr0104375" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cis-Trans+Isomerization+of+Organic+Molecules+and+Biomolecules%3A+Implications+and+Applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cis-Trans+Isomerization+of+Organic+Molecules+and+Biomolecules%3A+Implications+and+Applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001680-Pal1" id="pone.0001680-Pal1"></a>Pal D, Chakrabarti P (1999) Cis peptide bonds in proteins: residues involved, their conformations, interactions and locations. J Mol Biol  294: 271–288.  <ul class="find" data-citedArticleID="1063120" data-doi="10.1006/jmbi.1999.3217"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.3217" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cis+peptide+bonds+in+proteins%3A+residues+involved%2C+their+conformations%2C+interactions+and+locations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cis+peptide+bonds+in+proteins%3A+residues+involved%2C+their+conformations%2C+interactions+and+locations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001680-MilnerWhite1" id="pone.0001680-MilnerWhite1"></a>Milner-White EJ, Bell LH, Maccallum PH (1992) Pyrrolidine ring puckering in cis and trans-proline residues in proteins and polypeptides : Different puckers are favoured in certain situations. J Mol Biol  228: 725–734.  <ul class="find" data-citedArticleID="1063114" data-doi="10.1016/0022-2836(92)90859-i"><li><a href="http://dx.doi.org/10.1016/0022-2836(92)90859-i" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pyrrolidine+ring+puckering+in+cis+and+trans-proline+residues+in+proteins+and+polypeptides+%3A+Different+puckers+are+favoured+in+certain+situations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pyrrolidine+ring+puckering+in+cis+and+trans-proline+residues+in+proteins+and+polypeptides+%3A+Different+puckers+are+favoured+in+certain+situations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001680-Eberhardt1" id="pone.0001680-Eberhardt1"></a>Eberhardt ES, Panasik N, Raines RT (1996) Inductive Effects on the Energetics of Prolyl Peptide Bond Isomerization: Implications for Collagen Folding and Stability. J Am Chem Soc  118: 12261–12266.  <ul class="find" data-citedArticleID="1063086" data-doi="10.1021/ja9623119"><li><a href="http://dx.doi.org/10.1021/ja9623119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inductive+Effects+on+the+Energetics+of+Prolyl+Peptide+Bond+Isomerization%3A+Implications+for+Collagen+Folding+and+Stability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inductive+Effects+on+the+Energetics+of+Prolyl+Peptide+Bond+Isomerization%3A+Implications+for+Collagen+Folding+and+Stability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001680-Renner1" id="pone.0001680-Renner1"></a>Renner C, Alefelder S, Bae JH, Budisa N, Huber R, et al.  (2001) Fluoroprolines as Tools for Protein Design and Engineering. Angew Chem Int Ed Engl  40: 923–925.  <ul class="find" data-citedArticleID="1063130" data-doi="10.1002/1521-3773(20010302)40:5<923::aid-anie923&gt;3.3.co;2-r"><li><a href="http://dx.doi.org/10.1002/1521-3773(20010302)40:5<923::aid-anie923&gt;3.3.co;2-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fluoroprolines+as+Tools+for+Protein+Design+and+Engineering." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fluoroprolines+as+Tools+for+Protein+Design+and+Engineering.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001680-Holmgren1" id="pone.0001680-Holmgren1"></a>Holmgren SK, Taylor KM, Bretscher LE, Raines RT (1998) Code for collagen's stability deciphered. Nature  392: 666–667.  <ul class="find" data-citedArticleID="1063102" data-doi="10.1038/33573"><li><a href="http://dx.doi.org/10.1038/33573" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Code+for+collagen%27s+stability+deciphered." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Code+for+collagen%27s+stability+deciphered.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001680-Holmgren2" id="pone.0001680-Holmgren2"></a>Holmgren SK, Bretscher LE, Taylor KM, Raines RT (1999) A hyperstable collagen mimic. Chem Biol  6: 63–70.  <ul class="find" data-citedArticleID="1063104" data-doi="10.1016/s1074-5521(99)80003-9"><li><a href="http://dx.doi.org/10.1016/s1074-5521(99)80003-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+hyperstable+collagen+mimic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+hyperstable+collagen+mimic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001680-Golbik1" id="pone.0001680-Golbik1"></a>Golbik R, Fischer G, Fersht A (1999) Folding of barstar C40A/C82A/P27A and catalysis of the peptidyl-prolyl cis/trans isomerization by human cytosolic cyclophilin (Cyp18). Protein Sci  8: 1505–1514.  <ul class="find" data-citedArticleID="1063098" data-doi="10.1110/ps.8.7.1505"><li><a href="http://dx.doi.org/10.1110/ps.8.7.1505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Folding+of+barstar+C40A%2FC82A%2FP27A+and+catalysis+of+the+peptidyl-prolyl+cis%2Ftrans+isomerization+by+human+cytosolic+cyclophilin+%28Cyp18%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Folding+of+barstar+C40A%2FC82A%2FP27A+and+catalysis+of+the+peptidyl-prolyl+cis%2Ftrans+isomerization+by+human+cytosolic+cyclophilin+%28Cyp18%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001680-Golbik2" id="pone.0001680-Golbik2"></a>Golbik R, Yu C, Weyher-Stingl E, Huber R, Moroder L, et al.  (2005) Peptidyl Prolyl <em>cis</em>/<em>trans</em>-Isomerases: Comparative Reactivities of Cyclophilins, FK506-Binding Proteins, and Parvulins with Fluorinated Oligopeptide and Protein Substrates. Biochemistry  44: 16026–16034.  <ul class="find" data-citedArticleID="1063100" data-doi="10.1021/bi051442w"><li><a href="http://dx.doi.org/10.1021/bi051442w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peptidyl+Prolyl+cis%2Ftrans-Isomerases%3A+Comparative+Reactivities+of+Cyclophilins%2C+FK506-Binding+Proteins%2C+and+Parvulins+with+Fluorinated+Oligopeptide+and+Protein+Substrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peptidyl+Prolyl+cis%2Ftrans-Isomerases%3A+Comparative+Reactivities+of+Cyclophilins%2C+FK506-Binding+Proteins%2C+and+Parvulins+with+Fluorinated+Oligopeptide+and+Protein+Substrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001680-Boulgue1" id="pone.0001680-Boulgue1"></a>Boulègue C, Milbradt AG, Renner C, Moroder L (2006) Single Proline Residues can Dictate the Oxidative Folding Pathways of Cysteine-rich Peptides. J Mol Biol  358: 846–856.  <ul class="find" data-citedArticleID="1063070" data-doi="10.1016/j.jmb.2006.02.031"><li><a href="http://dx.doi.org/10.1016/j.jmb.2006.02.031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single+Proline+Residues+can+Dictate+the+Oxidative+Folding+Pathways+of+Cysteine-rich+Peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single+Proline+Residues+can+Dictate+the+Oxidative+Folding+Pathways+of+Cysteine-rich+Peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001680-Elsliger1" id="pone.0001680-Elsliger1"></a>Elsliger M-A, Wachter RM, Hanson GT, Kallio K, Remington SJ (1999) Structural and Spectral Response of Green Fluorescent Protein Variants to Changes in pH. Biochemistry  38: 5296–5301.  <ul class="find" data-citedArticleID="1063088" data-doi="10.1021/bi9902182"><li><a href="http://dx.doi.org/10.1021/bi9902182" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+and+Spectral+Response+of+Green+Fluorescent+Protein+Variants+to+Changes+in+pH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+and+Spectral+Response+of+Green+Fluorescent+Protein+Variants+to+Changes+in+pH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001680-Connor1" id="pone.0001680-Connor1"></a>Connor RE, Tirrell DA (2007) Non-Canonical Amino Acids in Protein Polymer Design. Polymer Rev  47: 9–28.  <ul class="find" data-citedArticleID="1063080" data-doi="10.1080/15583720601109552"><li><a href="http://dx.doi.org/10.1080/15583720601109552" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-Canonical+Amino+Acids+in+Protein+Polymer+Design." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-Canonical+Amino+Acids+in+Protein+Polymer+Design.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001680-Wiltschi1" id="pone.0001680-Wiltschi1"></a>Wiltschi B, Budisa N (2007) Natural history and experimental evolution of the genetic code. Appl Microbiol Biotechnol  74: 739–753.  <ul class="find" data-citedArticleID="1063136" data-doi="10.1007/s00253-006-0823-6"><li><a href="http://dx.doi.org/10.1007/s00253-006-0823-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+history+and+experimental+evolution+of+the+genetic+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+history+and+experimental+evolution+of+the+genetic+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001680-Mayer1" id="pone.0001680-Mayer1"></a>Mayer M, Buchner J (2004) Refolding of inclusion body proteins. Methods Mol Med  94: 239–254.  <ul class="find" data-citedArticleID="1063112" data-doi="10.1385/1-59259-679-7:239"><li><a href="http://dx.doi.org/10.1385/1-59259-679-7:239" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Refolding+of+inclusion+body+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Refolding+of+inclusion+body+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001680-Pedelacq1" id="pone.0001680-Pedelacq1"></a>Pedelacq J-D, Cabantous S, Tran T, Terwilliger TC, Waldo GS (2006) Engineering and characterization of a superfolder green fluorescent protein. Nat Biotechnol  24: 79–88.  <ul class="find" data-citedArticleID="1063124" data-doi="10.1038/nbt1172"><li><a href="http://dx.doi.org/10.1038/nbt1172" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Engineering+and+characterization+of+a+superfolder+green+fluorescent+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Engineering+and+characterization+of+a+superfolder+green+fluorescent+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001680-Yoo1" id="pone.0001680-Yoo1"></a>Yoo TH, Link AJ, Tirrell DA (2007) Evolution of a fluorinated green fluorescent protein. Proc Natl Acad Sci U S A  104: 13887–13890.  <ul class="find" data-citedArticleID="1063142" data-doi="10.1073/pnas.0701904104"><li><a href="http://dx.doi.org/10.1073/pnas.0701904104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+a+fluorinated+green+fluorescent+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+a+fluorinated+green+fluorescent+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001680-Fischer3" id="pone.0001680-Fischer3"></a>Fischer G (2000) Chemical aspects of peptide bond isomerisation. Chem Soc Rev  29: 119–127.  <ul class="find" data-citedArticleID="1063096" data-doi="10.1039/a803742f"><li><a href="http://dx.doi.org/10.1039/a803742f" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemical+aspects+of+peptide+bond+isomerisation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemical+aspects+of+peptide+bond+isomerisation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001680-Wthrich1" id="pone.0001680-Wthrich1"></a>Wüthrich K, Grathwohl C (1974) A novel approach for studies of the molecular conformations in flexible polypeptides. FEBS Lett  43: 337–340.  <ul class="find" data-citedArticleID="1063138" data-doi="10.1016/0014-5793(74)80674-5"><li><a href="http://dx.doi.org/10.1016/0014-5793(74)80674-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+approach+for+studies+of+the+molecular+conformations+in+flexible+polypeptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+approach+for+studies+of+the+molecular+conformations+in+flexible+polypeptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001680-Eyles1" id="pone.0001680-Eyles1"></a>Eyles SJ, Gierasch LM (2000) Multiple roles of prolyl residues in structure and folding. J Mol Biol  301: 737–747.  <ul class="find" data-citedArticleID="1063090" data-doi="10.1006/jmbi.2000.4002"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+roles+of+prolyl+residues+in+structure+and+folding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+roles+of+prolyl+residues+in+structure+and+folding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001680-Iwai1" id="pone.0001680-Iwai1"></a>Iwai H, Lingel A, Plückthun A (2001) Cyclic Green Fluorescent Protein Produced in Vivo Using an Artificially Split PI-PfuI Intein from Pyrococcus furiosus. J Biol Chem  276: 16548–16554.  <ul class="find" data-citedArticleID="1063106" data-doi="10.1074/jbc.m011639200"><li><a href="http://dx.doi.org/10.1074/jbc.m011639200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cyclic+Green+Fluorescent+Protein+Produced+in+Vivo+Using+an+Artificially+Split+PI-PfuI+Intein+from+Pyrococcus+furiosus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cyclic+Green+Fluorescent+Protein+Produced+in+Vivo+Using+an+Artificially+Split+PI-PfuI+Intein+from+Pyrococcus+furiosus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001680-Prasher1" id="pone.0001680-Prasher1"></a>Prasher DC, Eckenrode VK, Ward WW, Prendergast FG, Cormier MJ (1992) Primary structure of the Aequorea victoria green-fluorescent protein. Gene  111: 229–233.  <ul class="find" data-citedArticleID="1063126" data-doi="10.1016/0378-1119(92)90691-h"><li><a href="http://dx.doi.org/10.1016/0378-1119(92)90691-h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Primary+structure+of+the+Aequorea+victoria+green-fluorescent+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Primary+structure+of+the+Aequorea+victoria+green-fluorescent+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001680-Budisa1" id="pone.0001680-Budisa1"></a>Budisa N, Pal PP, Alefelder S, Birle P, Krywcun T, et al.  (2004) Probing the role of tryptophans in Aequorea victoria green fluorescent proteins with an expanded genetic code. Biol Chem  385: 191–202.  <ul class="find" data-citedArticleID="1063076" data-doi="10.1515/bc.2004.038"><li><a href="http://dx.doi.org/10.1515/bc.2004.038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probing+the+role+of+tryptophans+in+Aequorea+victoria+green+fluorescent+proteins+with+an+expanded+genetic+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probing+the+role+of+tryptophans+in+Aequorea+victoria+green+fluorescent+proteins+with+an+expanded+genetic+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001680-Dunitz1" id="pone.0001680-Dunitz1"></a>Dunitz JD (2004) Organic Fluorine: Odd Man Out. Chembiochem  5: 614–621.  <ul class="find" data-citedArticleID="1063084" data-doi="10.1002/cbic.200300801"><li><a href="http://dx.doi.org/10.1002/cbic.200300801" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Organic+Fluorine%3A+Odd+Man+Out." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Organic+Fluorine%3A+Odd+Man+Out.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001680-Minks1" id="pone.0001680-Minks1"></a>Minks C, Huber R, Moroder L, Budisa N (1999) Atomic Mutations at the Single Tryptophan Residue of Human Recombinant Annexin V: Effects on Structure, Stability, and Activity. Biochemistry  38: 10649–10659.  <ul class="find" data-citedArticleID="1063116" data-doi="10.1021/bi990580g"><li><a href="http://dx.doi.org/10.1021/bi990580g" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Atomic+Mutations+at+the+Single+Tryptophan+Residue+of+Human+Recombinant+Annexin+V%3A+Effects+on+Structure%2C+Stability%2C+and+Activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Atomic+Mutations+at+the+Single+Tryptophan+Residue+of+Human+Recombinant+Annexin+V%3A+Effects+on+Structure%2C+Stability%2C+and+Activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001680-Budisa2" id="pone.0001680-Budisa2"></a>Budisa N, Steipe B, Demange P, Eckerskorn C, Kellermann J, et al.  (1995) High-level Biosynthetic Substitution of Methionine in Proteins by its Analogs 2-Aminohexanoic Acid, Selenomethionine, Telluromethionine and Ethionine in Escherichia coli. Eur J Biochem  230: 788–796.  <ul class="find" data-citedArticleID="1063078" data-doi="10.1111/j.1432-1033.1995.tb20622.x"><li><a href="http://dx.doi.org/10.1111/j.1432-1033.1995.tb20622.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-level+Biosynthetic+Substitution+of+Methionine+in+Proteins+by+its+Analogs+2-Aminohexanoic+Acid%2C+Selenomethionine%2C+Telluromethionine+and+Ethionine+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-level+Biosynthetic+Substitution+of+Methionine+in+Proteins+by+its+Analogs+2-Aminohexanoic+Acid%2C+Selenomethionine%2C+Telluromethionine+and+Ethionine+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001680-Otwinowski1" id="pone.0001680-Otwinowski1"></a>Otwinowski Z, Minor W (1997) Processing of X-ray Diffraction Data Collected in Oscillation Mode. Methods Enzymol  276: 307–326.  <ul class="find" data-citedArticleID="1063118" data-doi="10.1016/s0076-6879(97)76066-x"><li><a href="http://dx.doi.org/10.1016/s0076-6879(97)76066-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Processing+of+X-ray+Diffraction+Data+Collected+in+Oscillation+Mode." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Processing+of+X-ray+Diffraction+Data+Collected+in+Oscillation+Mode.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001680-Brnger1" id="pone.0001680-Brnger1"></a>Brünger AT, Adams PD, Clore GM, DeLano WL, Gros P, et al.  (1998) Crystallography &amp; NMR System: A New Software Suite for Macromolecular Structure Determination. Acta Crystallogr D Biol Crystallogr  54: 905–921.  <ul class="find" data-citedArticleID="1063074" data-doi="10.1107/s0907444998003254"><li><a href="http://dx.doi.org/10.1107/s0907444998003254" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystallography+%26amp%3B+NMR+System%3A+A+New+Software+Suite+for+Macromolecular+Structure+Determination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystallography+%26amp%3B+NMR+System%3A+A+New+Software+Suite+for+Macromolecular+Structure+Determination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001680-Jones1" id="pone.0001680-Jones1"></a>Jones TA, Zou J-Y, Cowan SW, Kjeldgaard M (1991) Improved methods for building protein models in electron density maps and the location of errors in these models. Acta Crystallogr A  47: 110–119.  <ul class="find" data-citedArticleID="1063108" data-doi="10.1107/s0108767390010224"><li><a href="http://dx.doi.org/10.1107/s0108767390010224" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+methods+for+building+protein+models+in+electron+density+maps+and+the+location+of+errors+in+these+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+methods+for+building+protein+models+in+electron+density+maps+and+the+location+of+errors+in+these+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001680-Laskowski1" id="pone.0001680-Laskowski1"></a>Laskowski RA, MacArthur MW, Moss DS, Thornton JM (1993) PROCHECK: a program to check the stereochemical quality of protein structures. J Appl Crystallogr  26: 283–291.  <ul class="find" data-citedArticleID="1063110" data-doi="10.1107/s0021889892009944"><li><a href="http://dx.doi.org/10.1107/s0021889892009944" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.XML" value="77352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.PDF" value="742454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g001.PNG_L" value="900858"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g001.PNG_M" value="172385"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g001.PNG_S" value="10518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g001.TIF" value="1246004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g001.PNG_I" value="59022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g002.PNG_L" value="908294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g002.PNG_M" value="197059"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g002.PNG_S" value="11906"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g002.TIF" value="1170604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g002.PNG_I" value="65598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g003.PNG_L" value="1014323"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g003.PNG_M" value="80329"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g003.PNG_S" value="15585"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g003.TIF" value="2101512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g003.PNG_I" value="113427"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.t001.PNG_L" value="147877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.t001.PNG_M" value="108628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.t001.PNG_S" value="16436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.t001.TIF" value="844220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.t001.PNG_I" value="83756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g004.PNG_L" value="666073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g004.PNG_M" value="303539"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g004.PNG_S" value="12165"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g004.TIF" value="755434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g004.PNG_I" value="97606"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g005.PNG_L" value="4243240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g005.PNG_M" value="209812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g005.PNG_S" value="15940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g005.TIF" value="6813428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001680.g005.PNG_I" value="95488"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001680&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001680" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001680&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001680&volume=&issue=&title=Synthetic Biology of Proteins: Tuning GFPs Folding and Stability with Fluoroproline&author_name=Thomas%20Steiner%2C%20Petra%20Hess%2C%20Jae%20Hyun%20Bae%2C%20Birgit%20Wiltschi%2C%20Luis%20Moroder%2C%20Nediljko%20Budisa&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001680" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001680" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001680" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001680&amp;t=Synthetic%20Biology%20of%20Proteins%3A%20Tuning%20GFPs%20Folding%20and%20Stability%20with%20Fluoroproline" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001680&title=Synthetic%20Biology%20of%20Proteins%3A%20Tuning%20GFPs%20Folding%20and%20Stability%20with%20Fluoroproline&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001680&amp;title=Synthetic%20Biology%20of%20Proteins%3A%20Tuning%20GFPs%20Folding%20and%20Stability%20with%20Fluoroproline" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001680" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001680&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Synthetic Biology of Proteins: Tuning GFPs Folding and Stability with Fluoroproline';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001680';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Synthetic%20Biology%20of%20Proteins%3A%20Tuning%20GFPs%20Folding%20and%20Stability%20with%20Fluoroproline http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001680" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001680" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->

<div class="block">
    <div class="header">
        <h3>Included in the Following Collection</h3>
    </div>
    <p><dd><a href="http://www.ploscollections.org/article/browse/issue/info%3Adoi%2F10.1371%2Fissue.pcol.v02.i30" title="Browse the Open-Access Collection">Protein Engineering</a></dd><dd><a href="http://www.ploscollections.org/article/browse/issue/info%3Adoi%2F10.1371%2Fissue.pcol.v02.i29" title="Browse the Open-Access Collection">Biological Parts</a></dd></p>
</div>

<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystal+structure%22" title="Search for articles in the subject area:'Crystal structure'"><div class="flagText">Crystal structure</div></a>
              <div data-categoryid="17301" data-articleid="26186"
                   data-categoryname="Crystal structure"
                   class="flagImage" title="Flag 'Crystal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fluorescence%22" title="Search for articles in the subject area:'Fluorescence'"><div class="flagText">Fluorescence</div></a>
              <div data-categoryid="17783" data-articleid="26186"
                   data-categoryname="Fluorescence"
                   class="flagImage" title="Flag 'Fluorescence' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fluorine%22" title="Search for articles in the subject area:'Fluorine'"><div class="flagText">Fluorine</div></a>
              <div data-categoryid="27721" data-articleid="26186"
                   data-categoryname="Fluorine"
                   class="flagImage" title="Flag 'Fluorine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Isomerization%22" title="Search for articles in the subject area:'Isomerization'"><div class="flagText">Isomerization</div></a>
              <div data-categoryid="34507" data-articleid="26186"
                   data-categoryname="Isomerization"
                   class="flagImage" title="Flag 'Isomerization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proline%22" title="Search for articles in the subject area:'Proline'"><div class="flagText">Proline</div></a>
              <div data-categoryid="40969" data-articleid="26186"
                   data-categoryname="Proline"
                   class="flagImage" title="Flag 'Proline' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+folding%22" title="Search for articles in the subject area:'Protein folding'"><div class="flagText">Protein folding</div></a>
              <div data-categoryid="33387" data-articleid="26186"
                   data-categoryname="Protein folding"
                   class="flagImage" title="Flag 'Protein folding' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure%22" title="Search for articles in the subject area:'Protein structure'"><div class="flagText">Protein structure</div></a>
              <div data-categoryid="31877" data-articleid="26186"
                   data-categoryname="Protein structure"
                   class="flagImage" title="Flag 'Protein structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Urea%22" title="Search for articles in the subject area:'Urea'"><div class="flagText">Urea</div></a>
              <div data-categoryid="32247" data-articleid="26186"
                   data-categoryname="Urea"
                   class="flagImage" title="Flag 'Urea' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4327'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9337'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7595&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
