

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Modeling Insertional Mutagenesis Using Gene Length and Expression in Murine Embryonic Stem Cells</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000617"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000617" />

    <meta name="citation_title" content="Modeling Insertional Mutagenesis Using Gene Length and Expression in Murine Embryonic Stem Cells"/>
    <meta itemprop="name" content="Modeling Insertional Mutagenesis Using Gene Length and Expression in Murine Embryonic Stem Cells"/>

      <meta name="citation_author" content="Alex S. Nord"/>
            <meta name="citation_author_institution" content="Department of Medicine, MacDonald Medical Research Laboratories, University of California at Los Angeles, California, United States of America"/>
            <meta name="citation_author_institution" content="Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America"/>
            <meta name="citation_author_institution" content="Departments of Biopharmaceutical Sciences and Pharmaceutical Chemistry, University of California at San Francisco, California, United States of America"/>
      <meta name="citation_author" content="Karen Vranizan"/>
            <meta name="citation_author_institution" content="Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America"/>
            <meta name="citation_author_institution" content="Functional Genomics Laboratory, University of California at Berkeley, California, United States of America"/>
      <meta name="citation_author" content="Whittemore Tingley"/>
            <meta name="citation_author_institution" content="Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Medicine, University of California at San Francisco, California, United States of America"/>
      <meta name="citation_author" content="Alexander C. Zambon"/>
            <meta name="citation_author_institution" content="Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Medicine, University of California at San Francisco, California, United States of America"/>
      <meta name="citation_author" content="Kristina Hanspers"/>
            <meta name="citation_author_institution" content="Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Medicine, University of California at San Francisco, California, United States of America"/>
      <meta name="citation_author" content="Loren G. Fong"/>
            <meta name="citation_author_institution" content="Department of Medicine, MacDonald Medical Research Laboratories, University of California at Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Yan Hu"/>
            <meta name="citation_author_institution" content="Department of Medicine, MacDonald Medical Research Laboratories, University of California at Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Peter Bacchetti"/>
            <meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, University of California at San Francisco, California, United States of America"/>
      <meta name="citation_author" content="Thomas E. Ferrin"/>
            <meta name="citation_author_institution" content="Departments of Biopharmaceutical Sciences and Pharmaceutical Chemistry, University of California at San Francisco, California, United States of America"/>
      <meta name="citation_author" content="Patricia C. Babbitt"/>
            <meta name="citation_author_institution" content="Departments of Biopharmaceutical Sciences and Pharmaceutical Chemistry, University of California at San Francisco, California, United States of America"/>
      <meta name="citation_author" content="Scott W. Doniger"/>
            <meta name="citation_author_institution" content="Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America"/>
            <meta name="citation_author_institution" content="Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="William C. Skarnes"/>
            <meta name="citation_author_institution" content="Wellcome Trust Sanger Institute Hinxton, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Stephen G. Young"/>
            <meta name="citation_author_institution" content="Department of Medicine, MacDonald Medical Research Laboratories, University of California at Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Bruce R. Conklin"/>
            <meta name="citation_author_institution" content="Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Medicine, University of California at San Francisco, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular and Cellular Pharmacology, University of California at San Francisco, California, United States of America"/>

    <meta name="citation_date" content="2007/7/18"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000617.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e617"/>
    <meta name="citation_issue" content="7"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The BDGP gene disruption project: single transposon insertions associated with 40% of Drosophila genes.; citation_author=H Bellen; citation_author=R Levis; citation_author=G Liao; citation_author=Y He; citation_author=J Carlson; citation_journal_title=Genetics,; citation_volume=167(2); citation_number=1; citation_pages=761-781; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=FlyBase: genes and gene models.; citation_author=R Drysdale; citation_author=M Crosby; citation_journal_title=Nucleic Acids Res; citation_volume=33(Database issue); citation_number=2; citation_pages=D390-395; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Research resources for Drosophila: the expanding universe.; citation_author=K Matthews; citation_author=T Kaufman; citation_author=W Gelbart; citation_journal_title=Nat Rev Genet; citation_volume=6(3); citation_number=3; citation_pages=179-193; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Construction of Escherichia coli K-12 in frame, single-gene knockout mutants: the Keio collection.; citation_author=T Baba; citation_author=T Ara; citation_author=M Hasegawa; citation_author=Y Takai; citation_author=Y Okumura; citation_journal_title=Mol Syst Biol; citation_volume=2; citation_number=4; citation_pages=2006-2008; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products.; citation_author=K Datsenko; citation_author=B Wanner; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97(12); citation_number=5; citation_pages=6640-6645; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The knockout mouse project.; citation_author=C Austin; citation_author=J Battey; citation_author=A Bradley; citation_author=M Bucan; citation_author=M Capecchi; citation_journal_title=Nat Genet; citation_volume=36(9); citation_number=6; citation_pages=921-924; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gene-trap mutagenesis: past, present and beyond.; citation_author=W Stanford; citation_author=J Cohn; citation_author=S Cordes; citation_journal_title=Nat Rev Genet; citation_volume=2(10); citation_number=7; citation_pages=756-768; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A public gene trap resource for mouse functional genomics.; citation_author=W Skarnes; citation_author=Hv Melchner; citation_author=G Hicks; citation_author=A Nord; citation_author=T Cox; citation_journal_title=Nat Genet; citation_volume=36(6); citation_number=8; citation_pages=543-544; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A large-scale, gene-driven mutagenesis approach for the functional analysis of the mouse genome.; citation_author=J Hansen; citation_author=T Floss; citation_author=PV Sloun; citation_author=E Fuchtbauer; citation_author=H Arnold; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100(17); citation_number=9; citation_pages=9918-9922; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene targeting using a promoterless gene trap vector (“targeted trapping”) is an efficient method to mutate a large fraction of genes.; citation_author=R Friedel; citation_author=A Plump; citation_author=X Lu; citation_author=K Spilker; citation_author=C Jolicoeur; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102(37); citation_number=10; citation_pages=13188-13193; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A gene trap approach in mouse embryonic stem cells: the lacZ reported is activated by splicing, reflects endogenous gene expression, and is mutagenic in mice.; citation_author=W Skarnes; citation_author=B Auerbach; citation_author=A Joyner; citation_journal_title=Genes Dev; citation_volume=6(6); citation_number=11; citation_pages=903-918; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Identification of cellular promoters by using a retrovirus promoter trap.; citation_author=Hv Melchner; citation_author=H Ruley; citation_journal_title=J Virol; citation_volume=63(8); citation_number=12; citation_pages=3227-3233; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The Centre for Modeling Human Disease Gene Trap resource.; citation_author=C To; citation_author=T Reid; citation_author=Q Lan; citation_author=M Yu; citation_author=C Li; citation_journal_title=Nucleic Acids Res; citation_volume=32(Database issue); citation_number=13; citation_pages=D557-559; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=DNA bending creates favored sites for retroviral integration: an explanation for preferred insertion sites in nucleosomes.; citation_author=H Muller; citation_author=H Varmus; citation_journal_title=EMBO J; citation_volume=13(19); citation_number=14; citation_pages=4704-4714; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analyses of avian sarcoma virus integration sites.; citation_author=A Narezkina; citation_author=K Taganov; citation_author=S Litwin; citation_author=R Stoyanova; citation_author=C Seeger; citation_journal_title=J Virol; citation_volume=78(21); citation_number=15; citation_pages=11656-11663; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transcriptionally active genome regions are preferred targets for retrovirus integration.; citation_author=U Scherdin; citation_author=K Rhodes; citation_author=M Breindl; citation_journal_title=J Virol; citation_volume=64(2); citation_number=16; citation_pages=907-912; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Transcription start regions in the human genome are favored targets for MLV integration.; citation_author=X Wu; citation_author=Y Li; citation_author=B Crise; citation_author=S Burgess; citation_journal_title=Science; citation_volume=300(5626); citation_number=17; citation_pages=1749-1751; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Suppression of nonsense-mediated mRNA decay permits unbiased gene trapping in mouse embryonic stem cells.; citation_author=T Shigeoka; citation_author=M Kawaichi; citation_author=Y Ishida; citation_journal_title=Nucleic Acids Res; citation_volume=33(2); citation_number=18; citation_pages=e20; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of retroviral DNA integration.; citation_author=F Bushman; citation_author=M Lewinski; citation_author=A Ciuffi; citation_author=S Barr; citation_author=J Leipzig; citation_journal_title=Nat Rev Microbiol; citation_volume=3(11); citation_number=19; citation_pages=848-858; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RTCGD: retroviral tagged cancer gene database.; citation_author=K Akagi; citation_author=T Suzuki; citation_author=RM Stephens; citation_author=NA Jenkins; citation_author=NG Copeland; citation_journal_title=Nucleic Acids Res; citation_volume=32(Database issue); citation_number=20; citation_pages=D523-527; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Integration of human immunodeficiency virus type 1 in untreated infection occurs preferentially within genes.; citation_author=H Liu; citation_author=E Dow; citation_author=R Arora; citation_author=J Kimata; citation_author=L Bull; citation_journal_title=J Virol; citation_volume=80(15); citation_number=21; citation_pages=7765-7768; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Targeting survival: integration site selection by retroviruses and LTR-retrotransposons.; citation_author=F Bushman; citation_journal_title=Cell; citation_volume=115(2); citation_number=22; citation_pages=135-138; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular genetics and target site specificity of retroviral integration.; citation_author=L Holmes-Son; citation_author=R Appa; citation_author=S Chow; citation_journal_title=Adv Genet; citation_volume=43; citation_number=23; citation_pages=33-69; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Integrase-specific enhancement and suppression of retroviral DNA integration by compacted chromatin structure in vitro.; citation_author=K Taganov; citation_author=I Cuesta; citation_author=R Daniel; citation_author=L Cirillo; citation_author=R Katz; citation_journal_title=J Virol; citation_volume=78(11); citation_number=24; citation_pages=5848-5855; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Human gene targeting by adeno-associated virus vectors is enhanced by DNA double-strand breaks.; citation_author=D Miller; citation_author=L Petek; citation_author=D Russell; citation_journal_title=Mol Cell Biol; citation_volume=23(10); citation_number=25; citation_pages=3550-3557; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Analysis of gene targeting and intrachromosomal homologous recombination stimulated by genomic double-strand breaks in mouse embryonic stem cells.; citation_author=G Donoho; citation_author=M Jasin; citation_author=P Berg; citation_journal_title=Mol Cell Biol; citation_volume=18(7); citation_number=26; citation_pages=4070-4078; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Non-homologous DNA end joining in plant cells is associated with deletions and filler DNA insertions.; citation_author=A Gorbunova; citation_author=V Levy; citation_journal_title=Nucleic Acids Res; citation_volume=25(22); citation_number=27; citation_pages=4650-4657; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Wnk1 kinase deficiency lowers blood pressure in mice: a gene-trap screen to identify potential targets for therapeutic intervention.; citation_author=B Zambrowicz; citation_author=A Abuin; citation_author=R Ramirez-Solis; citation_author=L Richter; citation_author=J Piggott; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100(24); citation_number=28; citation_pages=14109-14114; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The International Gene Trap Consortium Website: a portal to all publicly available gene trap cell lines in mouse.; citation_author=A Nord; citation_author=P Chang; citation_author=B Conklin; citation_author=A Cox; citation_author=C Harper; citation_journal_title=Nucleic Acids Res; citation_volume=34(Database issue); citation_number=29; citation_pages=D642-648; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information.; citation_author=D Wheeler; citation_author=T Barrett; citation_author=D Benson; citation_author=S Bryant; citation_author=K Canese; citation_journal_title=Nucleic Acids Res; citation_volume=34(Database issue); citation_number=30; citation_pages=D173-180; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genomewide production of multipurpose alleles for the functional analysis of the mouse genome.; citation_author=F Schnutgen; citation_author=S De-Zolt; citation_author=P Van Sloun; citation_author=M Hollatz; citation_author=T Floss; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=2005, 102(20); citation_number=31; citation_pages=7221-7226; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Functional genomics in mice by tagged sequence mutagenesis.; citation_author=G Hicks; citation_author=E Shi; citation_author=X Li; citation_author=C Li; citation_author=M Pawlak; citation_journal_title=Nat Genet; citation_volume=16(4); citation_number=32; citation_pages=338-344; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Activation of cryptic 3′ splice sites within introns of cellular genes following gene entrapment.; citation_author=A Osipovich; citation_author=E White-Grindley; citation_author=G Hicks; citation_author=M Roshon; citation_author=C Shaffer; citation_journal_title=Nucleic Acids Res; citation_volume=32(9); citation_number=33; citation_pages=2912-2924; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=High-throughput trapping of secretory pathway genes in mouse embryonic stem cells.; citation_author=S De-Zolt; citation_author=F Schnutgen; citation_author=C Seisenberger; citation_author=J Hansen; citation_author=M Hollatz; citation_journal_title=Nucleic Acids Res; citation_volume=34(3); citation_number=34; citation_pages=e25; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Functional analysis of secreted and transmembrane proteins critical to mouse development.; citation_author=K Mitchell; citation_author=K Pinson; citation_author=O Kelly; citation_author=J Brennan; citation_author=J Zupicich; citation_journal_title=Nat Genet; citation_volume=28(3); citation_number=35; citation_pages=241-249; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Ensembl 2006.; citation_author=E Birney; citation_author=D Andrews; citation_author=M Caccamo; citation_author=Y Chen; citation_author=L Clarke; citation_journal_title=Nucleic Acids Res; citation_volume=34(Database issue); citation_number=36; citation_pages=D556-561; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=BayGenomics: A resource of insertional mutations in mouse embryonic stem cells.; citation_author=D Stryke; citation_author=M Kawamoto; citation_author=C Huang; citation_author=S Johns; citation_author=L King; citation_journal_title=Nucl Acids Res; citation_volume=31(1); citation_number=37; citation_pages=278-281; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Comparison of Affymetrix GeneChip expression measures.; citation_author=R Irizarry; citation_author=Z Wu; citation_author=H Jaffee; citation_journal_title=Bioinformatics; citation_volume=22(7); citation_number=38; citation_pages=789-794; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A Practical Guide to Splines.; citation_author=Cd Boor; citation_number=39; citation_date=1978; citation_publisher=Springer Verlag;1978; " />
      <meta name="citation_reference" content="citation_title=Information theory and an extension of the maximum likelihood principle.; citation_author=H Akaike; citation_number=40; citation_pages=267-281; citation_date=1973; citation_publisher=Akademia Kiado; " />
      <meta name="citation_reference" content="citation_title=Generalized Linear Models, 2nd edn.; citation_author= McCullagh; citation_number=41; citation_date=1989; citation_publisher=Chapman & Hall; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000617" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Modeling Insertional Mutagenesis Using Gene Length and Expression in Murine Embryonic Stem Cells"/>
    <meta name="twitter:description" content="BackgroundHigh-throughput mutagenesis of the mammalian genome is a powerful means to facilitate analysis of gene function. Gene trapping in embryonic stem cells (ESCs) is the most widely used form of insertional mutagenesis in mammals. However, the rules governing its efficiency are not fully understood, and the effects of vector design on the likelihood of gene-trapping events have not been tested on a genome-wide scale.Methodology/Principal FindingsIn this study, we used public gene-trap data to model gene-trap likelihood. Using the association of gene length and gene expression with gene-trap likelihood, we constructed spline-based regression models that characterize which genes are susceptible and which genes are resistant to gene-trapping techniques. We report results for three classes of gene-trap vectors, showing that both length and expression are significant determinants of trap likelihood for all vectors. Using our models, we also quantitatively identified hotspots of gene-trap activity, which represent loci where the high likelihood of vector insertion is controlled by factors other than length and expression. These formalized statistical models describe a high proportion of the variance in the likelihood of a gene being trapped by expression-dependent vectors and a lower, but still significant, proportion of the variance for vectors that are predicted to be independent of endogenous gene expression.Conclusions/SignificanceThe findings of significant expression and length effects reported here further the understanding of the determinants of vector insertion. Results from this analysis can be applied to help identify other important determinants of this important biological phenomenon and could assist planning of large-scale mutagenesis efforts."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000617.g003"/>

  <meta property="og:title" content="Modeling Insertional Mutagenesis Using Gene Length and Expression in Murine Embryonic Stem Cells" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000617" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2559'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2933'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7907&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000617">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000617" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000617&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Modeling Insertional Mutagenesis Using Gene Length and Expression in Murine Embryonic Stem Cells
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alex S. Nord
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:nordalex@u.washington.edu">nordalex@u.washington.edu</a> (AN); <a href="mailto:bconklin@gladstone.ucsf.edu">bconklin@gladstone.ucsf.edu</a> (BC)</p>

                <p>Affiliations:
                  Department of Medicine, MacDonald Medical Research Laboratories, University of California at Los Angeles, California, United States of America, 
                  Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America, 
                  Departments of Biopharmaceutical Sciences and Pharmaceutical Chemistry, University of California at San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Karen Vranizan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America, 
                  Functional Genomics Laboratory, University of California at Berkeley, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Whittemore Tingley, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America, 
                  Department of Medicine, University of California at San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexander C. Zambon, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America, 
                  Department of Medicine, University of California at San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kristina Hanspers, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America, 
                  Department of Medicine, University of California at San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Loren G. Fong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Medicine, MacDonald Medical Research Laboratories, University of California at Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yan Hu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Medicine, MacDonald Medical Research Laboratories, University of California at Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter Bacchetti, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Epidemiology and Biostatistics, University of California at San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas E. Ferrin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departments of Biopharmaceutical Sciences and Pharmaceutical Chemistry, University of California at San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patricia C. Babbitt, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departments of Biopharmaceutical Sciences and Pharmaceutical Chemistry, University of California at San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Scott W. Doniger, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America, 
                  Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            William C. Skarnes, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Wellcome Trust Sanger Institute Hinxton, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen G. Young, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Medicine, MacDonald Medical Research Laboratories, University of California at Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bruce R. Conklin
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:nordalex@u.washington.edu">nordalex@u.washington.edu</a> (AN); <a href="mailto:bconklin@gladstone.ucsf.edu">bconklin@gladstone.ucsf.edu</a> (BC)</p>

                <p>Affiliations:
                  Gladstone Institute of Cardiovascular Disease, San Francisco, California, United States of America, 
                  Department of Medicine, University of California at San Francisco, California, United States of America, 
                  Department of Molecular and Cellular Pharmacology, University of California at San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: July 18, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000617</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000617" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000617" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000617" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000617" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000617-g001" data-doi="info:doi/10.1371/journal.pone.0000617" data-uri="info:doi/10.1371/journal.pone.0000617.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000617-g002" data-doi="info:doi/10.1371/journal.pone.0000617" data-uri="info:doi/10.1371/journal.pone.0000617.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000617-g003" data-doi="info:doi/10.1371/journal.pone.0000617" data-uri="info:doi/10.1371/journal.pone.0000617.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000617-t001" data-doi="info:doi/10.1371/journal.pone.0000617" data-uri="info:doi/10.1371/journal.pone.0000617.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000617-t002" data-doi="info:doi/10.1371/journal.pone.0000617" data-uri="info:doi/10.1371/journal.pone.0000617.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000617">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000617" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>High-throughput mutagenesis of the mammalian genome is a powerful means to facilitate analysis of gene function. Gene trapping in embryonic stem cells (ESCs) is the most widely used form of insertional mutagenesis in mammals. However, the rules governing its efficiency are not fully understood, and the effects of vector design on the likelihood of gene-trapping events have not been tested on a genome-wide scale.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>In this study, we used public gene-trap data to model gene-trap likelihood. Using the association of gene length and gene expression with gene-trap likelihood, we constructed spline-based regression models that characterize which genes are susceptible and which genes are resistant to gene-trapping techniques. We report results for three classes of gene-trap vectors, showing that both length and expression are significant determinants of trap likelihood for all vectors. Using our models, we also quantitatively identified hotspots of gene-trap activity, which represent loci where the high likelihood of vector insertion is controlled by factors other than length and expression. These formalized statistical models describe a high proportion of the variance in the likelihood of a gene being trapped by expression-dependent vectors and a lower, but still significant, proportion of the variance for vectors that are predicted to be independent of endogenous gene expression.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The findings of significant expression and length effects reported here further the understanding of the determinants of vector insertion. Results from this analysis can be applied to help identify other important determinants of this important biological phenomenon and could assist planning of large-scale mutagenesis efforts.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Nord AS, Vranizan K, Tingley W, Zambon AC, Hanspers K, et al.  (2007) Modeling Insertional Mutagenesis Using Gene Length and Expression in Murine Embryonic Stem Cells. PLoS ONE 2(7):
          e617.
            doi:10.1371/journal.pone.0000617</p><p><strong>Academic Editor: </strong>Cecile Fairhead, Pasteur Institute, France</p><p><strong>Received:</strong> April 20, 2007; <strong>Accepted:</strong> May 31, 2007; <strong>Published:</strong> July 18, 2007</p><p><strong>Copyright:</strong> © 2007 Nord et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>NIH Grants HL66621 and HG002766</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Complete collections of well-defined mutants have helped shed light on the biology of model organisms, such as flies <a href="#pone.0000617-Bellen1">[1]</a>–<a href="#pone.0000617-Matthews1">[3]</a> and bacteria <a href="#pone.0000617-Baba1">[4]</a>, <a href="#pone.0000617-Datsenko1">[5]</a>. Likewise, the development of a complete collection of mouse mutants would enhance our ability to understand mammalian biology <a href="#pone.0000617-Austin1">[6]</a>. Libraries of mutant mouse embryonic stem cells (ESCs) are particularly valuable because they can be readily cryopreserved and used to generate mutant mice. Gene trapping in ESCs is an effective, high-throughput technique for generating insertional mutations in the mouse genome <a href="#pone.0000617-Stanford1">[7]</a>. Ultimately, however, non-targeted trapping becomes inefficient; some genes are repeatedly trapped, and others are trapped rarely, if at all <a href="#pone.0000617-Skarnes1">[8]</a>, <a href="#pone.0000617-Hansen1">[9]</a>. A better understanding of the characteristics that determine susceptibility (or resistance) to trapping would be useful, as it would further understanding of vector insertion into the genome and could help guide large-scale mouse mutagenesis efforts.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The factors that determine the “trappability” of individual genes (<em>i.e.</em>, their likelihood of being inactivated by gene trapping) are unclear. The integration of gene-trapping vectors into chromosomal DNA is potentially influenced by a number of factors, including the intrinsic properties of the vector, the expression level of the gene in mouse ESCs, chromatin structure, DNA substrate recognition, and gene size. In addition the existence of highly favored integration sites (hotspots) complicates efforts to understand the factors that control trappability. <a href="#pone.0000617-Friedel1">[10]</a></p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Gene expression levels in ESCs are believed to positively correlate with trapping efficiency with expression-dependent vectors, but the extent of the expression effect in different gene-trap vectors has not been systematically quantified or compared. Splice-acceptor (SA) gene-trap vectors depend on the integration of a new SA sequence to interrupt the trapped gene <a href="#pone.0000617-Skarnes2">[11]</a>, <a href="#pone.0000617-Melchner1">[12]</a>. When successful, SA-trap vectors inactivate the trapped gene and result in an antibiotic-resistance gene product that allows for selection of the mutant cell lines. These vectors lack a promoter, so endogenous gene expression is required to drive transcription of the vector product. However, gene expression has not been tested on a large scale while controlling for gene length, which is also thought to affect trappability.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In polyadenylation (poly-A) gene-trap vectors, by contrast, the antibiotic-resistance gene is driven by a strong promoter within the vector. The stability of the transcript for the antibiotic-resistance gene depends on the poly-A signal from the trapped gene <a href="#pone.0000617-To1">[13]</a>. Because the transcription of the antibiotic-resistance gene product does not depend on the endogenous expression of the trapped gene, poly-A trap vectors are predicted to trap genes regardless of whether the gene is expressed in ESCs.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The method of vector delivery to cells (retroviral vector versus plasmid DNA) may also influence which genes are inactivated by gene trapping. Retroviruses are predicted to insert at the 5′ end of transcriptionally active genes and may recognize specific substrates in genomic DNA. Little is known about the insertion of plasmid vectors. Both plasmid and retroviral methods have been used in SA gene trapping, while poly-A approaches have exclusively used retroviral delivery methods.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>The recent release of a near-complete mouse genome, advances in techniques for estimating the levels of gene expression in a cell, and the availability of a public gene-trapping database (<a href="http://www.genetrap.org">www.genetrap.org</a>) make it possible to globally assess the likelihood that a gene will be inactivated by gene trapping. In this study, we used regression techniques to model the effects of gene length and gene-expression levels on gene trapping in different gene-trap vectors. We also sought to define hotspots for gene-trapping events by using the regression models to identify genes trapped more frequently than predicted by the models. Our findings provide an improved understanding of the factors that control vector insertion in the genome.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Association of gene expression and length with gene-trap likelihood</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We sought to formally test the hypothesis that length and/or expression influence the probability that a gene will be trapped. We applied stringent criteria to the genes included in this analysis, limiting the dataset to genes for which accurate genomic mapping and curated annotation were available. Because absolute gene expression estimates, as opposed to fold changes, were necessary for this analysis, we employed Affymetrix Gene Chips and the GCRMA (GeneChip Robust Multi-array Analysis) gene expression estimation method (<a href="http://www.bioconductor.org">http://www.bioconductor.org</a>) on a representative sample of E14 mouse embryonic stem cells. GCRMA expression estimates were validated by comparisons to RT-PCR data in the same E14 mouse ESC line <a href="#pone.0000617-Friedel1">[10]</a> (<a href="#pone.0000617.s001">Table S1</a>). The correlation between GCRMA and the RT-PCR-derived expression was high (Spearman's <em>r</em> = 0.67, P-value&lt;.0001), and the relationship between expression and gene-trap likelihood in endogenous expression-dependent vectors is consistent with previous analysis of trap likelihood with SA-plasmid vectors <a href="#pone.0000617-Friedel1">[10]</a>. This level of quality control and validation gave us confidence that we accurately assessed relative gene expression throughout the full range of transcriptional activity.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>For this study, we focused on three major types of gene-trap vectors (<a href="#pone-0000617-g001">Figure 1</a>), for which enough genes had been trapped to allow robust comparisons. We analyzed 16322 gene-trap cell lines in the public database (<a href="http://www.genetrap.org">www.genetrap.org</a>) (<a href="#pone-0000617-t001">Table 1</a>). We first constructed scatter plots of the trapping frequencies for genes versus known gene length and our expression estimates in E14 ESCs for each vector type (<a href="#pone-0000617-g002">Figure 2</a>). We then used regression modeling to test length and expression simultaneously, so that we could assess the effects of each variable on trapping, independently of the other. For each vector, we fit a regression model to the number of times each gene was trapped as a function of gene length and expression. Spline-based modeling methods were used to accommodate potential nonlinearity in the models of trap likelihood. The expected number of traps for each gene per million trapping events, as predicted by the fitted models, was plotted against a grid of length and expression values (<a href="#pone-0000617-g003">Figure 3</a>).</p>
<div class="figure" id="pone-0000617-g001"><div class="img"><a name="pone-0000617-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000617" data-uri="info:doi/10.1371/journal.pone.0000617.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000617.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000617.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Diagram of major mechanisms of gene trapping of an endogenous gene with two exons.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) In the SA-trap, the SA site allows trapping when inserted into any part of the gene <em>via</em> plasmid or viral integration. (B) The poly-A trap relies on the poly-A (pA) of the endogenous gene because the neomycin-resistance gene does not have a poly-A tail. Note that the poly-A trap has its own constitutive promoter (prom). Also indicated are the splice donor (SD), splice acceptor (SA), and neomycin resistance (NeoR). The major components of each trap were excluded from this diagram to emphasize on the essential elements needed to understand the trapping models. Detailed maps of each major vector type are referenced in the <a href="#s3">Methods</a> section.</p>
<span>doi:10.1371/journal.pone.0000617.g001</span></div><div class="figure" id="pone-0000617-g002"><div class="img"><a name="pone-0000617-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000617" data-uri="info:doi/10.1371/journal.pone.0000617.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000617.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000617.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Trapped genes by length and expression.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>For each vector type, genes were plotted according to their size and level of expression in ESCs. Genes that have been trapped are shown in red. The circle size is proportional to the number of times a gene has been trapped.</p>
<span>doi:10.1371/journal.pone.0000617.g002</span></div><div class="figure" id="pone-0000617-g003"><div class="img"><a name="pone-0000617-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000617" data-uri="info:doi/10.1371/journal.pone.0000617.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000617.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000617.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Models of trap likelihood for gene-trap vectors.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Models of the likelihood of trapping a gene with particular length (<em>x</em>-axis) and expression (<em>y</em>-axis) values for each gene-trap event were created through an iterative process, in which outliers (<em>P</em>&lt;0.001) were removed before the final model was created. Probability (<em>z</em>-axis) is given as events per million traps.</p>
<span>doi:10.1371/journal.pone.0000617.g003</span></div><div class="figure" id="pone-0000617-t001"><div class="img"><a name="pone-0000617-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000617" data-uri="info:doi/10.1371/journal.pone.0000617.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000617.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000617.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of gene trap data sets</span></strong></p><span>doi:10.1371/journal.pone.0000617.t001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The probability that a gene would be trapped with SA-plasmid and SA-viral vectors increased with both gene expression and gene length. SA vectors showed highly statistically significant (P&lt;0.0001) effects due to gene expression (<a href="#pone-0000617-t002">Table 2</a>). In comparison, the poly-A vectors showed much weaker, albeit statistically significant, expression effects (P&lt;0.013). The trapping efficiency of the SA-plasmid and poly-A vectors also increased with gene size (P&lt;0.0001). SA-retroviral vectors exhibited a similar length trend (P&lt;0.0001), but for unknown reasons, these vectors displayed a plateau effect in the largest genes, where trapping likelihood did not increase.</p>
<div class="figure" id="pone-0000617-t002"><div class="img"><a name="pone-0000617-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000617" data-uri="info:doi/10.1371/journal.pone.0000617.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000617.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000617.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000617.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000617.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Hotspot Effects and Model Summary</span></strong></p><span>doi:10.1371/journal.pone.0000617.t002</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>The strong effect of expression on trap likelihood is likely due to two factors. First, this effect is an inherent property of antibiotic selection, and differences in the expression trends of endogenous expression-dependent vectors stem from differences between cell-culture and sequencing protocols. Second, the presence of the small expression effect in poly-A vectors, where none is expected, suggests that gene-trap likelihood is, at some level, dependent on transcriptional activity and chromatin structure. Previous studies of retroviral insertion with vectors similar to gene-trap vectors yielded contradictory results concerning the effect of gene-expression level on vector insertion <a href="#pone.0000617-Muller1">[14]</a>–<a href="#pone.0000617-Scherdin1">[16]</a>; however, retroviruses are known to integrate preferentially into transcribed genes, likely owing to the effects of chromatin structure <a href="#pone.0000617-Wu1">[17]</a>.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>Although poly-A vectors do not appear to depend substantially on gene expression levels, their use poses additional challenges. Poly-A vectors do not require endogenous regulation of transcription, so there is a potentially greater chance that insertion in a “non-genic” locus could still confer antibiotic resistance. This could account for the diminished proportion of poly-A gene-trap events that can be mapped to a gene (<a href="#pone-0000617-t001">Table 1</a>). In addition, preferential integration at the 3′ end of genes in these vectors is due to nonsense-mediated decay of transcripts of the antibiotic-resistance gene-trap products. This decay typically occurs when the vector inserts upstream of the final intron <a href="#pone.0000617-Shigeoka1">[18]</a>. As a result, insertion of poly-A-trap vectors at the 3′ end more frequently yields drug-resistant colonies. This bias is worrisome because the likelihood that a gene-trap mutation will cause a null allele decreases as the insertion site moves towards the 3′ end of a gene. Newer poly-A trap vectors may overcome the nonsense-mediated decay issue <a href="#pone.0000617-Shigeoka1">[18]</a> and could be an attractive alternative to expression-based gene-trapping vectors.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>In addition to expression, we found that gene length affected trapping likelihood for all three vectors. This finding was somewhat surprising because certain vector types are thought to insert primarily into the ends of genes and therefore might not be expected to exhibit significant gene-length effects. For example, retroviral vectors preferentially inserted at the 5′ end of genes in one study <a href="#pone.0000617-Hansen1">[9]</a>. Likewise, the poly-A trap vectors included in this study insert preferentially into the 3′ ends of genes <a href="#pone.0000617-Shigeoka1">[18]</a>. Such preferential insertion could eliminate the effects of overall length. In addition, the first or last introns may be the dominant determinants of this length effect, and that total gene length might not capture this effect. Nevertheless, we found a clear enrichment in the trapping of long genes with all retroviral vectors.</p>
<a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7"></a><p>While individual insertion-specific intron length may be of ultimate importance to the length effect observed in this study, measurement of intron size and identification of the intron of insertion are less reliable due to the prominence of alternative splicing and the difficulty of mapping specific gene-trap vector insertion sites. Further characterization of trends affecting the intron of insertion is necessary to better understand the gene-length effect described here.</p>


<h4>Hotspots of vector insertion</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Though well known in the field of gene-trapping <a href="#pone.0000617-Hansen1">[9]</a>, hotspots are difficult to define rigorously <a href="#pone.0000617-Bushman1">[19]</a>. These loci, in which vector insertion is highly enriched, are of interest not only to the gene-trap community, but also to the gene therapy <a href="#pone.0000617-Bushman1">[19]</a>, cancer biology <a href="#pone.0000617-Akagi1">[20]</a>, and HIV fields <a href="#pone.0000617-Liu1">[21]</a>. Prior attempts to define hotspots have likely been confounded by ignoring effects of expression and length that we define here. To identify loci trapped with frequencies elevated beyond those predicted by expression and length, we flagged genes significantly outside the model prediction space and defined them as hotspots. We used an iterative fitting process to identify these genes separately for each vector (hotspots listed in <a href="#pone.0000617.s002">Table S2</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Each vector type had a unique set of hotspots, with marked differences based upon the method of vector delivery (plasmid or retroviral), as shown in <a href="#pone-0000617-t002">Table 2</a>. Hotspot insertions were more frequent with SA-retroviral retroviral vectors (30% of total traps) than with SA-plasmid vectors (10% of total traps). Poly-A vectors showed a smaller hotspot effect (21% of total traps) than other retroviral vectors. These proportions could underestimate the actual number of genomic hotspots, as we only considered well-defined genes.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>This increased presence of hotspots in retroviral-based gene trapping could reflect the tendency of retroviral insertional machinery to interact with specific sites in the genome <a href="#pone.0000617-Bushman1">[19]</a>, <a href="#pone.0000617-Bushman2">[22]</a>–<a href="#pone.0000617-Taganov1">[24]</a>. The mechanism driving hotspots with plasmid vectors is less well understood but might involve genomic regions with high recombination frequencies and high rates of double-stranded break repair <a href="#pone.0000617-Miller1">[25]</a>–<a href="#pone.0000617-Gorbunova1">[27]</a>. The quantitative method of identifying hotspots used here may help future investigations to identify and characterize cellular and genomic factors that underlie insertional hotspots.</p>


<h4>Summary of gene-trap likelihood models</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The ability of these expression and length-dependent models to explain trapping probabilities was quantified by the percent reduction in deviance compared to a null model with no covariates in the datasets after hot-spot removal, analogous to the use of R-squared in linear regression models. The percent reduction in deviance was greater for SA-plasmid and SA-retroviral models (34% and 19%, respectively) than for the poly-A model (6%). For the expression-dependent SA vectors, these models explain a substantial proportion of the deviance. The low explanatory ability of the poly-A vector model reflects the relatively lower effect of expression on gene-trap likelihood, and to some extent the smaller effect of length.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Regardless of whether a gene has been inactivated by gene trapping in our experimental data, our models can predict the likelihood that a gene will be trapped in a single trapping event. These predictions serve as “trapping scores” for each gene. The raw trapping scores were corrected for the effects of both hotspots and gene-trap events that could not be mapped to a gene. The corrected scores, reported for the 7435 well-defined genes included in our dataset in <a href="#pone.0000617.s002">Table S2</a>, allow the overall trapping efficiencies of different vectors to be compared.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>To validate our model, we compared the expected number of traps from the SA-retroviral model with the observed number of traps in gene-trap cell lines produced by Lexicon Genetics <a href="#pone.0000617-Zambrowicz1">[28]</a>. This validation set contained 48,809 cell lines from Lexicon Genetics that could be annotated to our gene set (data in <a href="#pone.0000617.s002">Table S2</a>). The Spearman's correlation coefficient for the comparison was 0.429 (P&lt;0.005). This level of concordance gives us further confidence that gene-trap likelihood was successfully modeled.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Although these models describe a significant proportion of the variance in trap likelihood between mouse genes, other factors undoubtedly contribute to trap likelihood, and therefore, trapping scores should be interpreted on a limited scope for individual genes. For instance, the expression-based vectors used in this study fail to trap secreted and membrane-bound proteins (data not shown). Other genomic factors that control vector insertion, transcription, splicing, translation and protein localization all likely play some role in determining trap likelihood. Further examination of genes where the number of observed traps departs from model predictions may help identify other important mechanisms affecting insertional mutagenesis.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Our models have other limitations relating to experimental and modeling constraints. The restricted number of total genes for which high-confidence annotation and ESC expression data were available reduced our dataset size. We also were limited to using expression measurements from a single ESC line, and different global expression states potentially exist between the different ESC lines used in gene trapping. These constraints may affect the accuracy of our models, contributing to the overall noise, and these effects may be stronger at the end of the length and expression scales, where there are fewer data points.</p>


<h4>Conclusion</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Our findings offer a more complete understanding of factors governing the accessibility of genes to trapping. We report the first formal testing of the effects of gene expression and gene length on trapping likelihood. While the effects of expression in SA-trap vectors is confirmatory, the detection of an expression effect in poly-A vectors is an important finding and matches previous reports of a role of transcriptional state in vector insertion likelihood. The length effect reported for all vectors described in this study is a novel finding that requires further characterization to understand the relative importance of the underlying biology. In addition the identification of expression and length-independent insertional hotspots is an important result and could benefit fields other than gene-trapping. Ideally, the empirically quantified relationships we provide here can be generalized to all genes in the mouse genome. Mutagenesis in mouse and human ESCs will continue to evolve with new and more powerful techniques, and the results from this initial analysis will hopefully aid future mammalian gene mutagenesis efforts.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Gene-trap data</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Data for gene-trap cell lines were generated with the International Gene Trap Consortium (IGTC) identification and annotation pipeline <a href="#pone.0000617-Nord1">[29]</a>. Annotations were obtained by genome and transcript-based homology searching. Publicly available gene-trap cell lines included in the IGTC database were used for the gene-trap data sets, and all gene-trap cell-line sequences used in our analysis can be found in the NCBI Genome Survey Sequence database <a href="#pone.0000617-Wheeler1">[30]</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Because numerous gene-trap vectors were used to create the cell lines represented by the IGTC, we chose three representative vector groups for analysis: the plasmid pGTlxf series (SA-plasmid), the retroviral FlEx vectors (SA-retroviral), and a combination of poly-A trap vectors (poly-A). The SA-plasmid vectors were produced by BayGenomics (<a href="http://baygenomics.ucsf.edu">http://baygenomics.ucsf.edu</a>). The SA-viral vectors are a conditional system used by the German Gene Trap Consortium (<a href="http://www.tikus.gsf.de">http://www.tikus.gsf.de</a>) <a href="#pone.0000617-Schnutgen1">[31]</a>. Poly-A cell lines are from the Centre for Modeling Human Disease gene-trap project (<a href="http://www.cmhd.ca/genetrap">http://www.cmhd.ca/genetrap</a>) <a href="#pone.0000617-To1">[13]</a>. Poly-A vectors designed to take advantage of nonsense-mediated decay <a href="#pone.0000617-Shigeoka1">[18]</a> were not included in this analysis. Exon trap vectors are similar to SA-trap vectors but depend on direct, in-frame integration into the open-reading frame of a gene <a href="#pone.0000617-Hicks1">[32]</a>; however, these vectors behave similarly to SA vectors due to cryptic splicing <a href="#pone.0000617-Osipovich1">[33]</a> and were therefore omitted from this study. Secretory vectors containing transmembrane signal sequences <a href="#pone.0000617-DeZolt1">[34]</a>, <a href="#pone.0000617-Mitchell1">[35]</a> were also excluded. Vector maps are available on each gene-trap resource website. More information on gene-trap data is available on the IGTC website (<a href="http://www.genetrap.org">http://www.genetrap.org</a>).</p>


<h4>Gene Data</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Gene length and annotation data were from mouse build 36 of the Ensembl database <a href="#pone.0000617-Birney1">[36]</a>. Length was computed as the full transcribed genomic region, including the UTR when present. This analysis used a set of 7435 well-characterized genes (<a href="#pone.0000617.s002">Table S2</a>) for which complete sequence data were available. Similar sets of “sentinel genes” have been used in analyses of gene-trap data <a href="#pone.0000617-Zambrowicz1">[28]</a>. For this study, this set includes known genes annotated to a Mouse Genome Informatics (MGI) symbol and an Entrez Gene ID and not primarily identified by Riken clones. Single-exon genes were omitted from the model, because they lack of a splice donor site necessary for proper function of SA and poly-A trap vectors.</p>


<h4>Expression Data</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>For all gene-expression data, mouse E14 ESCs were prepared as described <a href="#pone.0000617-Stryke1">[37]</a> (<a href="http://www.baygenomics.ucsf.edu">http://www.baygenomics.ucsf.edu</a>). For GeneChip studies, we performed four biological replicates using Affymetrix 430 2.0 arrays, and RNA samples were prepared as described by the manufacturer (Affymetrix, Sunnyvale, CA). The 430 2.0 GeneChip contains 45,101 probe sets, including 9242 probe sets that were unambiguously mapped to a single Ensembl identification. Only probes marked as type “_at” were used for the final analysis, because we had the highest confidence of proper hybridization in these probe sets. All other probe sets were discarded, because the probes may cross-hybridize to mRNA products of other genes (Affymetrix). We selected the probe set with maximal expression when there was more than one representing a single gene.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Expression values were calculated with GCRMA (v. 1.1.5; <a href="http://www.bioconductor.org">http://www.bioconductor.org</a>), a method that purports to give good estimates of expression in the entire expression spectrum <a href="#pone.0000617-Irizarry1">[38]</a>. While on/off calls and removal of genes based on a low signal-to-noise ratio may allow elimination of some spurious expression results, use of GCRMA and full data inclusion were necessary to model the likelihood of trapping, especially at the lower boundaries of expression. Expression values for a subsample of genes from the same cell type in the same tissue-culture conditions were confirmed in TaqMan quantitative RT-PCR experiments <a href="#pone.0000617-Stryke1">[37]</a>. Raw and normalized data for these experiments can be accessed using GEO series accession GSE8128.</p>


<h4>Analysis</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Spline-based methods were used in multinomial regression models, with gene length and expression as model predictors and the number of trap events in each gene as the outcome. Knot placement for splines was based on gene-distribution percentiles. Models were iteratively fitted to genes remaining at each round after cumulative removal of hotspot genes, defined as genes with an observed trapping frequency far above the expected frequency. We identified such genes, using a cut-off of P&lt;0.001 and re-ran the analyses without them. These probabilities were calculated by using the Poisson approximation to the binomial distribution with a large number of trials and a low probability of success and were corrected for the estimated model overdispersion. We then re-fit the model with the hotspots deleted, repeating the process until no additional hotspots were identified. The level of significance 0.001 <em>P</em> value was selected to be conservative in the culling of genes that did not fit the model, as we wanted to limit the number of genes removed to only those that far exceeded predicted trap likelihood. <em>P</em> values for the length and expression effects in the final models are reported, and deviance that can be explained for each model was computed.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Trapping scores were computed directly from the fitted model as the predicted probability of trapping, and corrected by multiplying the proportion of events that trapped a modeled gene rather than a hotspot or gene-trap event that could not be mapped to a gene. Statistical analysis was performed with SAS (SAS Institute, Cary, NC) and the R statistical environment (<a href="http://www.r-project.org">http://www.r-project.org</a>).</p>


<h4>Model Specification</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Because each experiment (trap event) selects one of a known set of genes that could be trapped, the data fit the statistical framework of multinomial regression. Let <em>n</em> = 1 to <em>N</em> index experiments that trapped a gene. For each experiment, we assumed that the probability that gene <em>j</em> is the one that is trapped is a function of covariates. Let <strong>x</strong> be the matrix with a row for each gene and a column for each covariate. A multinomial model for which gene is trapped in each experiment is then defined by: <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where the sum in the denominator is over all genes that might be trapped, <em>f</em> is a function of the covariates, and <strong>x</strong><em><sub>i</sub></em> is the vector of covariates for gene <em>i</em>. For example, a simple linear model with two covariates would be f(<strong>x</strong><em><sub>i</sub></em>) = β<sub>1</sub><em>x<sub>i1</sub></em>+β<sub>2</sub><em>x<sub>i2</sub></em>. Here we restrict attention to genes whose length and expression are known and to the experiments where one of these genes was trapped. Letting <em>j</em> denote the gene trapped in the <em>n</em>th experiment, we can write the log-likelihood (up to a constant that does not depend on the covariates) for experiment <em>n</em> as: <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>Letting <em>h<sub>i</sub></em> denote the number of times gene <em>i</em> was trapped, we can then write the log-likelihood for the entire set of experiments as: <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000617.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br></p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>For any given parametric form for the function <em>f</em>, we can estimate the parameters by finding those that maximize this log-likelihood, with the general optimization features of the NLMIXED procedure in SAS. For both covariates (gene length and expression), we applied logarithmic transformations and then used cubic parametric splines <a href="#pone.0000617-Boor1">[39]</a>, choosing among models with different degrees of freedom with the Akaike information criterion <a href="#pone.0000617-Akaike1">[40]</a> adjusted for overdispersion <a href="#pone.0000617-McCullagh1">[41]</a>. We assumed that the effects of these two covariates were additive, <em>f</em>(<strong>x</strong><em><sub>i</sub></em>) = <em>f<sub>1</sub></em>(<em>x<sub>i1</sub></em>)+<em>f<sub>2</sub></em>(<em>x<sub>i2</sub></em>). Adding interaction terms did not substantially improve fits to the data.</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>To calculate a fitted probability of trapping for each gene, we used equation (1) with the best-fitting parameters for <em>f</em><sub>1</sub> and <em>f</em><sub>2</sub></p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000617"></div><a name="pone.0000617.s001" id="pone.0000617.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000617.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Compares the expression as measured using RT-PCR with expression estimates derived using GCRMA methods.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000617.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0000617.s002" id="pone.0000617.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000617.s002">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Summarizes the dataset used for this analysis. Included are the length and expression values for genes included in the analysis, number of traps in each gene by gene-trap vector, and the derived trap score for each gene. Hotspots genes are tagged in the trap score column for each vector. Omniback gene-trap events, used here as a validation set, are listed in the final column.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000617.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(1.60 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank members of the BayGenomics Consortium, members of the Conklin and Young laboratories for valuable discussions and reading of the manuscript, Gary Howard and Stephen Ordway, and Wendy Lilliedoll for editorial assistance, and Mary Weglarz and Amanda Bradford for manuscript preparation.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BC AZ WS WT SY AN KV SD. Performed the experiments: BC AZ PB WT KH LF YH SY AN KV SD TF. Analyzed the data: BC WS KH AN KV PB. Contributed reagents/materials/analysis tools: BC. Wrote the paper: BC WS SY AN KV PB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000617-Bellen1" id="pone.0000617-Bellen1"></a>Bellen H, Levis R, Liao G, He Y, Carlson J, et al.  (2004) The BDGP gene disruption project: single transposon insertions associated with 40% of Drosophila genes. Genetics,  167(2): 761–781.  <ul class="find" data-citedArticleID="966494" data-doi="10.1534/genetics.104.026427"><li><a href="http://dx.doi.org/10.1534/genetics.104.026427" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+BDGP+gene+disruption+project%3A+single+transposon+insertions+associated+with+40%25+of+Drosophila+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+BDGP+gene+disruption+project%3A+single+transposon+insertions+associated+with+40%25+of+Drosophila+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000617-Drysdale1" id="pone.0000617-Drysdale1"></a>Drysdale R, Crosby M (2005) FlyBase: genes and gene models. Nucleic Acids Res  33(Database issue): D390–395.  <ul class="find" data-citedArticleID="966510" data-doi="10.1093/nar/gki046"><li><a href="http://dx.doi.org/10.1093/nar/gki046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FlyBase%3A+genes+and+gene+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FlyBase%3A+genes+and+gene+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000617-Matthews1" id="pone.0000617-Matthews1"></a>Matthews K, Kaufman T, Gelbart W (2005) Research resources for Drosophila: the expanding universe. Nat Rev Genet  6(3): 179–193.  <ul class="find" data-citedArticleID="966526" data-doi="10.1038/nrg1554"><li><a href="http://dx.doi.org/10.1038/nrg1554" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Research+resources+for+Drosophila%3A+the+expanding+universe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Research+resources+for+Drosophila%3A+the+expanding+universe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000617-Baba1" id="pone.0000617-Baba1"></a>Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, et al.  (2006) Construction of Escherichia coli K-12 in frame, single-gene knockout mutants: the Keio collection. Mol Syst Biol  2: 2006–2008.  <ul class="find" data-citedArticleID="966492" data-doi="10.1038/msb4100050"><li><a href="http://dx.doi.org/10.1038/msb4100050" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+Escherichia+coli+K-12+in+frame%2C+single-gene+knockout+mutants%3A+the+Keio+collection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+Escherichia+coli+K-12+in+frame%2C+single-gene+knockout+mutants%3A+the+Keio+collection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000617-Datsenko1" id="pone.0000617-Datsenko1"></a>Datsenko K, Wanner B (2000) One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proc Natl Acad Sci U S A  97(12): 6640–6645.  <ul class="find" data-citedArticleID="966504" data-doi="10.1073/pnas.120163297"><li><a href="http://dx.doi.org/10.1073/pnas.120163297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=One-step+inactivation+of+chromosomal+genes+in+Escherichia+coli+K-12+using+PCR+products." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22One-step+inactivation+of+chromosomal+genes+in+Escherichia+coli+K-12+using+PCR+products.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000617-Austin1" id="pone.0000617-Austin1"></a>Austin C, Battey J, Bradley A, Bucan M, Capecchi M, et al.  (2004) The knockout mouse project. Nat Genet  36(9): 921–924.  <ul class="find" data-citedArticleID="966490" data-doi="10.1038/ng0904-921"><li><a href="http://dx.doi.org/10.1038/ng0904-921" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+knockout+mouse+project." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+knockout+mouse+project.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000617-Stanford1" id="pone.0000617-Stanford1"></a>Stanford W, Cohn J, Cordes S (2001) Gene-trap mutagenesis: past, present and beyond. Nat Rev Genet  2(10): 756–768.  <ul class="find" data-citedArticleID="966554" data-doi="10.1038/35093548"><li><a href="http://dx.doi.org/10.1038/35093548" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene-trap+mutagenesis%3A+past%2C+present+and+beyond." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene-trap+mutagenesis%3A+past%2C+present+and+beyond.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000617-Skarnes1" id="pone.0000617-Skarnes1"></a>Skarnes W, Melchner Hv, Hicks G, Nord A, Cox T, et al.  (2004) A public gene trap resource for mouse functional genomics. Nat Genet  36(6): 543–544.  <ul class="find" data-citedArticleID="966550" data-doi="10.1038/ng0604-543"><li><a href="http://dx.doi.org/10.1038/ng0604-543" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+public+gene+trap+resource+for+mouse+functional+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+public+gene+trap+resource+for+mouse+functional+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000617-Hansen1" id="pone.0000617-Hansen1"></a>Hansen J, Floss T, Sloun PV, Fuchtbauer E, Arnold H, et al.  (2003) A large-scale, gene-driven mutagenesis approach for the functional analysis of the mouse genome. Proc Natl Acad Sci U S A  100(17): 9918–9922.  <ul class="find" data-citedArticleID="966516" data-doi="10.1073/pnas.1633296100"><li><a href="http://dx.doi.org/10.1073/pnas.1633296100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+large-scale%2C+gene-driven+mutagenesis+approach+for+the+functional+analysis+of+the+mouse+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+large-scale%2C+gene-driven+mutagenesis+approach+for+the+functional+analysis+of+the+mouse+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000617-Friedel1" id="pone.0000617-Friedel1"></a>Friedel R, Plump A, Lu X, Spilker K, Jolicoeur C, et al.  (2005) Gene targeting using a promoterless gene trap vector (“targeted trapping”) is an efficient method to mutate a large fraction of genes. Proc Natl Acad Sci U S A  102(37): 13188–13193.  <ul class="find" data-citedArticleID="966512" data-doi="10.1073/pnas.0505474102"><li><a href="http://dx.doi.org/10.1073/pnas.0505474102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+targeting+using+a+promoterless+gene+trap+vector+%28%26%23x201c%3Btargeted+trapping%26%23x201d%3B%29+is+an+efficient+method+to+mutate+a+large+fraction+of+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+targeting+using+a+promoterless+gene+trap+vector+%28%26%23x201c%3Btargeted+trapping%26%23x201d%3B%29+is+an+efficient+method+to+mutate+a+large+fraction+of+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000617-Skarnes2" id="pone.0000617-Skarnes2"></a>Skarnes W, Auerbach B, Joyner A (1992) A gene trap approach in mouse embryonic stem cells: the lacZ reported is activated by splicing, reflects endogenous gene expression, and is mutagenic in mice. Genes Dev  6(6): 903–918.  <ul class="find" data-citedArticleID="966552" data-doi="10.1101/gad.6.6.903"><li><a href="http://dx.doi.org/10.1101/gad.6.6.903" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+trap+approach+in+mouse+embryonic+stem+cells%3A+the+lacZ+reported+is+activated+by+splicing%2C+reflects+endogenous+gene+expression%2C+and+is+mutagenic+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+trap+approach+in+mouse+embryonic+stem+cells%3A+the+lacZ+reported+is+activated+by+splicing%2C+reflects+endogenous+gene+expression%2C+and+is+mutagenic+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000617-Melchner1" id="pone.0000617-Melchner1"></a>Melchner Hv, Ruley H (1989) Identification of cellular promoters by using a retrovirus promoter trap. J Virol  63(8): 3227–3233.  <ul class="find" data-citedArticleID="966530"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+cellular+promoters+by+using+a+retrovirus+promoter+trap.&amp;auth=&amp;atitle=Identification+of+cellular+promoters+by+using+a+retrovirus+promoter+trap." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+cellular+promoters+by+using+a+retrovirus+promoter+trap." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+cellular+promoters+by+using+a+retrovirus+promoter+trap.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000617-To1" id="pone.0000617-To1"></a>To C, Reid T, Lan Q, Yu M, Li C, et al.  (2004) The Centre for Modeling Human Disease Gene Trap resource. Nucleic Acids Res  32(Database issue): D557–559.  <ul class="find" data-citedArticleID="966560"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Centre+for+Modeling+Human+Disease+Gene+Trap+resource.&amp;auth=&amp;atitle=The+Centre+for+Modeling+Human+Disease+Gene+Trap+resource." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Centre+for+Modeling+Human+Disease+Gene+Trap+resource." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Centre+for+Modeling+Human+Disease+Gene+Trap+resource.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000617-Muller1" id="pone.0000617-Muller1"></a>Muller H, Varmus H (1994) DNA bending creates favored sites for retroviral integration: an explanation for preferred insertion sites in nucleosomes. EMBO J  13(19): 4704–4714.  <ul class="find" data-citedArticleID="966536"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+bending+creates+favored+sites+for+retroviral+integration%3A+an+explanation+for+preferred+insertion+sites+in+nucleosomes.&amp;auth=&amp;atitle=DNA+bending+creates+favored+sites+for+retroviral+integration%3A+an+explanation+for+preferred+insertion+sites+in+nucleosomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+bending+creates+favored+sites+for+retroviral+integration%3A+an+explanation+for+preferred+insertion+sites+in+nucleosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+bending+creates+favored+sites+for+retroviral+integration%3A+an+explanation+for+preferred+insertion+sites+in+nucleosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000617-Narezkina1" id="pone.0000617-Narezkina1"></a>Narezkina A, Taganov K, Litwin S, Stoyanova R, Seeger C, et al.  (2004) Genome-wide analyses of avian sarcoma virus integration sites. J Virol  78(21): 11656–11663.  <ul class="find" data-citedArticleID="966538" data-doi="10.1128/jvi.78.21.11656-11663.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.21.11656-11663.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analyses+of+avian+sarcoma+virus+integration+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analyses+of+avian+sarcoma+virus+integration+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000617-Scherdin1" id="pone.0000617-Scherdin1"></a>Scherdin U, Rhodes K, Breindl M (1990) Transcriptionally active genome regions are preferred targets for retrovirus integration. J Virol  64(2): 907–912.  <ul class="find" data-citedArticleID="966544"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transcriptionally+active+genome+regions+are+preferred+targets+for+retrovirus+integration.&amp;auth=&amp;atitle=Transcriptionally+active+genome+regions+are+preferred+targets+for+retrovirus+integration." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptionally+active+genome+regions+are+preferred+targets+for+retrovirus+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptionally+active+genome+regions+are+preferred+targets+for+retrovirus+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000617-Wu1" id="pone.0000617-Wu1"></a>Wu X, Li Y, Crise B, Burgess S (2003) Transcription start regions in the human genome are favored targets for MLV integration. Science  300(5626): 1749–1751.  <ul class="find" data-citedArticleID="966564" data-doi="10.1126/science.1083413"><li><a href="http://dx.doi.org/10.1126/science.1083413" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+start+regions+in+the+human+genome+are+favored+targets+for+MLV+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+start+regions+in+the+human+genome+are+favored+targets+for+MLV+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000617-Shigeoka1" id="pone.0000617-Shigeoka1"></a>Shigeoka T, Kawaichi M, Ishida Y (2005) Suppression of nonsense-mediated mRNA decay permits unbiased gene trapping in mouse embryonic stem cells. Nucleic Acids Res  33(2): e20.  <ul class="find" data-citedArticleID="966548" data-doi="10.1093/nar/gni022"><li><a href="http://dx.doi.org/10.1093/nar/gni022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suppression+of+nonsense-mediated+mRNA+decay+permits+unbiased+gene+trapping+in+mouse+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suppression+of+nonsense-mediated+mRNA+decay+permits+unbiased+gene+trapping+in+mouse+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000617-Bushman1" id="pone.0000617-Bushman1"></a>Bushman F, Lewinski M, Ciuffi A, Barr S, Leipzig J, et al.  (2005) Genome-wide analysis of retroviral DNA integration. Nat Rev Microbiol  3(11): 848–858.  <ul class="find" data-citedArticleID="966500" data-doi="10.1038/nrmicro1263"><li><a href="http://dx.doi.org/10.1038/nrmicro1263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+retroviral+DNA+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+retroviral+DNA+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000617-Akagi1" id="pone.0000617-Akagi1"></a>Akagi K, Suzuki T, Stephens RM, Jenkins NA, Copeland NG (2004) RTCGD: retroviral tagged cancer gene database. Nucleic Acids Res  32(Database issue): D523–527.  <ul class="find" data-citedArticleID="966486" data-doi="10.1093/nar/gkh013"><li><a href="http://dx.doi.org/10.1093/nar/gkh013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RTCGD%3A+retroviral+tagged+cancer+gene+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RTCGD%3A+retroviral+tagged+cancer+gene+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000617-Liu1" id="pone.0000617-Liu1"></a>Liu H, Dow E, Arora R, Kimata J, Bull L, et al.  (2006) Integration of human immunodeficiency virus type 1 in untreated infection occurs preferentially within genes. J Virol  80(15): 7765–7768.  <ul class="find" data-citedArticleID="966524" data-doi="10.1128/jvi.00542-06"><li><a href="http://dx.doi.org/10.1128/jvi.00542-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integration+of+human+immunodeficiency+virus+type+1+in+untreated+infection+occurs+preferentially+within+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integration+of+human+immunodeficiency+virus+type+1+in+untreated+infection+occurs+preferentially+within+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000617-Bushman2" id="pone.0000617-Bushman2"></a>Bushman F (2003) Targeting survival: integration site selection by retroviruses and LTR-retrotransposons. Cell  115(2): 135–138.  <ul class="find" data-citedArticleID="966502" data-doi="10.1016/s0092-8674(03)00760-8"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)00760-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeting+survival%3A+integration+site+selection+by+retroviruses+and+LTR-retrotransposons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeting+survival%3A+integration+site+selection+by+retroviruses+and+LTR-retrotransposons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000617-HolmesSon1" id="pone.0000617-HolmesSon1"></a>Holmes-Son L, Appa R, Chow S (2001) Molecular genetics and target site specificity of retroviral integration. Adv Genet  43: 33–69.  <ul class="find" data-citedArticleID="966520" data-doi="10.1016/s0065-2660(01)43003-3"><li><a href="http://dx.doi.org/10.1016/s0065-2660(01)43003-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+genetics+and+target+site+specificity+of+retroviral+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+genetics+and+target+site+specificity+of+retroviral+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000617-Taganov1" id="pone.0000617-Taganov1"></a>Taganov K, Cuesta I, Daniel R, Cirillo L, Katz R, et al.  (2004) Integrase-specific enhancement and suppression of retroviral DNA integration by compacted chromatin structure in vitro. J Virol  78(11): 5848–5855.  <ul class="find" data-citedArticleID="966558" data-doi="10.1128/jvi.78.11.5848-5855.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.11.5848-5855.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrase-specific+enhancement+and+suppression+of+retroviral+DNA+integration+by+compacted+chromatin+structure+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrase-specific+enhancement+and+suppression+of+retroviral+DNA+integration+by+compacted+chromatin+structure+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000617-Miller1" id="pone.0000617-Miller1"></a>Miller D, Petek L, Russell D (2003) Human gene targeting by adeno-associated virus vectors is enhanced by DNA double-strand breaks. Mol Cell Biol  23(10): 3550–3557.  <ul class="find" data-citedArticleID="966532" data-doi="10.1128/mcb.23.10.3550-3557.2003"><li><a href="http://dx.doi.org/10.1128/mcb.23.10.3550-3557.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+gene+targeting+by+adeno-associated+virus+vectors+is+enhanced+by+DNA+double-strand+breaks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+gene+targeting+by+adeno-associated+virus+vectors+is+enhanced+by+DNA+double-strand+breaks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000617-Donoho1" id="pone.0000617-Donoho1"></a>Donoho G, Jasin M, Berg P (1998) Analysis of gene targeting and intrachromosomal homologous recombination stimulated by genomic double-strand breaks in mouse embryonic stem cells. Mol Cell Biol  18(7): 4070–4078.  <ul class="find" data-citedArticleID="966508"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+gene+targeting+and+intrachromosomal+homologous+recombination+stimulated+by+genomic+double-strand+breaks+in+mouse+embryonic+stem+cells.&amp;auth=&amp;atitle=Analysis+of+gene+targeting+and+intrachromosomal+homologous+recombination+stimulated+by+genomic+double-strand+breaks+in+mouse+embryonic+stem+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+gene+targeting+and+intrachromosomal+homologous+recombination+stimulated+by+genomic+double-strand+breaks+in+mouse+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+gene+targeting+and+intrachromosomal+homologous+recombination+stimulated+by+genomic+double-strand+breaks+in+mouse+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000617-Gorbunova1" id="pone.0000617-Gorbunova1"></a>Gorbunova A, Levy V (1997) Non-homologous DNA end joining in plant cells is associated with deletions and filler DNA insertions. Nucleic Acids Res  25(22): 4650–4657.  <ul class="find" data-citedArticleID="966514" data-doi="10.1093/nar/25.22.4650"><li><a href="http://dx.doi.org/10.1093/nar/25.22.4650" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-homologous+DNA+end+joining+in+plant+cells+is+associated+with+deletions+and+filler+DNA+insertions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-homologous+DNA+end+joining+in+plant+cells+is+associated+with+deletions+and+filler+DNA+insertions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000617-Zambrowicz1" id="pone.0000617-Zambrowicz1"></a>Zambrowicz B, Abuin A, Ramirez-Solis R, Richter L, Piggott J, et al.  (2003) Wnk1 kinase deficiency lowers blood pressure in mice: a gene-trap screen to identify potential targets for therapeutic intervention. Proc Natl Acad Sci U S A  100(24): 14109–14114.  <ul class="find" data-citedArticleID="966566" data-doi="10.1073/pnas.2336103100"><li><a href="http://dx.doi.org/10.1073/pnas.2336103100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wnk1+kinase+deficiency+lowers+blood+pressure+in+mice%3A+a+gene-trap+screen+to+identify+potential+targets+for+therapeutic+intervention." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Wnk1+kinase+deficiency+lowers+blood+pressure+in+mice%3A+a+gene-trap+screen+to+identify+potential+targets+for+therapeutic+intervention.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000617-Nord1" id="pone.0000617-Nord1"></a>Nord A, Chang P, Conklin B, Cox A, Harper C, et al.  (2006) The International Gene Trap Consortium Website: a portal to all publicly available gene trap cell lines in mouse. Nucleic Acids Res  34(Database issue): D642–648.  <ul class="find" data-citedArticleID="966540" data-doi="10.1093/nar/gkj097"><li><a href="http://dx.doi.org/10.1093/nar/gkj097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+International+Gene+Trap+Consortium+Website%3A+a+portal+to+all+publicly+available+gene+trap+cell+lines+in+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+International+Gene+Trap+Consortium+Website%3A+a+portal+to+all+publicly+available+gene+trap+cell+lines+in+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000617-Wheeler1" id="pone.0000617-Wheeler1"></a>Wheeler D, Barrett T, Benson D, Bryant S, Canese K, et al.  (2006) Database resources of the National Center for Biotechnology Information. Nucleic Acids Res  34(Database issue): D173–180.  <ul class="find" data-citedArticleID="966562" data-doi="10.1093/nar/gkj158"><li><a href="http://dx.doi.org/10.1093/nar/gkj158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Database+resources+of+the+National+Center+for+Biotechnology+Information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Database+resources+of+the+National+Center+for+Biotechnology+Information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000617-Schnutgen1" id="pone.0000617-Schnutgen1"></a>Schnutgen F, De-Zolt S, Van Sloun P, Hollatz M, Floss T, et al.  (2005) Genomewide production of multipurpose alleles for the functional analysis of the mouse genome. Proc Natl Acad Sci USA  2005, 102(20): 7221–7226.  <ul class="find" data-citedArticleID="966546" data-doi="10.1073/pnas.0502273102"><li><a href="http://dx.doi.org/10.1073/pnas.0502273102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomewide+production+of+multipurpose+alleles+for+the+functional+analysis+of+the+mouse+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomewide+production+of+multipurpose+alleles+for+the+functional+analysis+of+the+mouse+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000617-Hicks1" id="pone.0000617-Hicks1"></a>Hicks G, Shi E, Li X, Li C, Pawlak M, et al.  (1997) Functional genomics in mice by tagged sequence mutagenesis. Nat Genet  16(4): 338–344.  <ul class="find" data-citedArticleID="966518" data-doi="10.1038/ng0897-338"><li><a href="http://dx.doi.org/10.1038/ng0897-338" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+genomics+in+mice+by+tagged+sequence+mutagenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+genomics+in+mice+by+tagged+sequence+mutagenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000617-Osipovich1" id="pone.0000617-Osipovich1"></a>Osipovich A, White-Grindley E, Hicks G, Roshon M, Shaffer C, et al.  (2004) Activation of cryptic 3′ splice sites within introns of cellular genes following gene entrapment. Nucleic Acids Res  32(9): 2912–2924.  <ul class="find" data-citedArticleID="966542" data-doi="10.1093/nar/gkh604"><li><a href="http://dx.doi.org/10.1093/nar/gkh604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+of+cryptic+3%26%23x2032%3B+splice+sites+within+introns+of+cellular+genes+following+gene+entrapment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+of+cryptic+3%26%23x2032%3B+splice+sites+within+introns+of+cellular+genes+following+gene+entrapment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000617-DeZolt1" id="pone.0000617-DeZolt1"></a>De-Zolt S, Schnutgen F, Seisenberger C, Hansen J, Hollatz M, et al.  (2006) High-throughput trapping of secretory pathway genes in mouse embryonic stem cells. Nucleic Acids Res  34(3): e25.  <ul class="find" data-citedArticleID="966506" data-doi="10.1093/nar/gnj026"><li><a href="http://dx.doi.org/10.1093/nar/gnj026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-throughput+trapping+of+secretory+pathway+genes+in+mouse+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-throughput+trapping+of+secretory+pathway+genes+in+mouse+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000617-Mitchell1" id="pone.0000617-Mitchell1"></a>Mitchell K, Pinson K, Kelly O, Brennan J, Zupicich J, et al.  (2001) Functional analysis of secreted and transmembrane proteins critical to mouse development. Nat Genet  28(3): 241–249.  <ul class="find" data-citedArticleID="966534"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Functional+analysis+of+secreted+and+transmembrane+proteins+critical+to+mouse+development.&amp;auth=&amp;atitle=Functional+analysis+of+secreted+and+transmembrane+proteins+critical+to+mouse+development." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+analysis+of+secreted+and+transmembrane+proteins+critical+to+mouse+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+analysis+of+secreted+and+transmembrane+proteins+critical+to+mouse+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000617-Birney1" id="pone.0000617-Birney1"></a>Birney E, Andrews D, Caccamo M, Chen Y, Clarke L, et al.  (2006) Ensembl 2006. Nucleic Acids Res  34(Database issue): D556–561.  <ul class="find" data-citedArticleID="966496" data-doi="10.1093/nar/gkj133"><li><a href="http://dx.doi.org/10.1093/nar/gkj133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ensembl+2006." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ensembl+2006.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000617-Stryke1" id="pone.0000617-Stryke1"></a>Stryke D, Kawamoto M, Huang C, Johns S, King L, et al.  (2003) BayGenomics: A resource of insertional mutations in mouse embryonic stem cells. Nucl Acids Res  31(1): 278–281.  <ul class="find" data-citedArticleID="966556" data-doi="10.1093/nar/gkg064"><li><a href="http://dx.doi.org/10.1093/nar/gkg064" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BayGenomics%3A+A+resource+of+insertional+mutations+in+mouse+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BayGenomics%3A+A+resource+of+insertional+mutations+in+mouse+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000617-Irizarry1" id="pone.0000617-Irizarry1"></a>Irizarry R, Wu Z, Jaffee H (2006) Comparison of Affymetrix GeneChip expression measures. Bioinformatics  22(7): 789–794.  <ul class="find" data-citedArticleID="966522" data-doi="10.1093/bioinformatics/btk046"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btk046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+Affymetrix+GeneChip+expression+measures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+Affymetrix+GeneChip+expression+measures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000617-Boor1" id="pone.0000617-Boor1"></a>Boor Cd (1978) A Practical Guide to Splines. Berlin: Springer Verlag;1978.   <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0000617-Akaike1" id="pone.0000617-Akaike1"></a>Akaike H (1973) Information theory and an extension of the maximum likelihood principle. Second International Symposium on Information Theory: 1973. Budapest: Akademia Kiado. pp. 267–281.  <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0000617-McCullagh1" id="pone.0000617-McCullagh1"></a>McCullagh  (1989) Generalized Linear Models, 2nd edn. London: Chapman &amp; Hall.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.XML" value="86186"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.PDF" value="381884"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g001.PNG_L" value="1055389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g001.PNG_M" value="113770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g001.PNG_S" value="12292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g001.TIF" value="1617480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g001.PNG_I" value="39827"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g002.PNG_L" value="1645054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g002.PNG_M" value="180477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g002.PNG_S" value="7111"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g002.TIF" value="2028500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g002.PNG_I" value="57717"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g003.PNG_L" value="1694724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g003.PNG_M" value="152863"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g003.PNG_S" value="29316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g003.TIF" value="2047662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.g003.PNG_I" value="329516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.t001.PNG_L" value="78557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.t001.PNG_M" value="104170"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.t001.PNG_S" value="14383"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.t001.TIF" value="327146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.t001.PNG_I" value="36707"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.t002.PNG_L" value="112741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.t002.PNG_M" value="70560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.t002.PNG_S" value="9635"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.t002.TIF" value="483004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.t002.PNG_I" value="22108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.e001.PNG" value="10962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.e001.TIF" value="27308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.e002.PNG" value="9809"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.e002.TIF" value="25848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.e003.PNG" value="11038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.e003.TIF" value="26956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.s001.XLS" value="28160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000617.s002.XLS" value="1600000"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000617&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000617" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000617&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000617&volume=&issue=&title=Modeling Insertional Mutagenesis Using Gene Length and Expression in Murine Embryonic Stem Cells&author_name=Alex%20S.%20Nord%2C%20Karen%20Vranizan%2C%20Whittemore%20Tingley%2C%20Alexander%20C.%20Zambon%2C%20Kristina%20Hanspers%2C%20Loren%20G.%20Fong%2C%20Yan%20Hu%2C%20Peter%20Bacchetti%2C%20Thomas%20E.%20Ferrin%2C%20Patricia%20C.%20Babbitt%2C%20Scott%20W.%20Doniger%2C%20William%20C.%20Skarnes%2C%20Stephen%20G.%20Young%2C%20Bruce%20R.%20Conklin&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000617" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000617" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000617" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000617&amp;t=Modeling%20Insertional%20Mutagenesis%20Using%20Gene%20Length%20and%20Expression%20in%20Murine%20Embryonic%20Stem%20Cells" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000617&title=Modeling%20Insertional%20Mutagenesis%20Using%20Gene%20Length%20and%20Expression%20in%20Murine%20Embryonic%20Stem%20Cells&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000617&amp;title=Modeling%20Insertional%20Mutagenesis%20Using%20Gene%20Length%20and%20Expression%20in%20Murine%20Embryonic%20Stem%20Cells" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000617" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000617&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Modeling Insertional Mutagenesis Using Gene Length and Expression in Murine Embryonic Stem Cells';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000617';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Modeling%20Insertional%20Mutagenesis%20Using%20Gene%20Length%20and%20Expression%20in%20Murine%20Embryonic%20Stem%20Cells http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000617" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000617" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24072"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="24072"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomics%22" title="Search for articles in the subject area:'Genomics'"><div class="flagText">Genomics</div></a>
              <div data-categoryid="18221" data-articleid="24072"
                   data-categoryname="Genomics"
                   class="flagImage" title="Flag 'Genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Introns%22" title="Search for articles in the subject area:'Introns'"><div class="flagText">Introns</div></a>
              <div data-categoryid="20277" data-articleid="24072"
                   data-categoryname="Introns"
                   class="flagImage" title="Flag 'Introns' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="24072"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plasmid+vector+techniques%22" title="Search for articles in the subject area:'Plasmid vector techniques'"><div class="flagText">Plasmid vector techniques</div></a>
              <div data-categoryid="30683" data-articleid="24072"
                   data-categoryname="Plasmid vector techniques"
                   class="flagImage" title="Flag 'Plasmid vector techniques' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plasmid+vectors%22" title="Search for articles in the subject area:'Plasmid vectors'"><div class="flagText">Plasmid vectors</div></a>
              <div data-categoryid="24907" data-articleid="24072"
                   data-categoryname="Plasmid vectors"
                   class="flagImage" title="Flag 'Plasmid vectors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="24072"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3505'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2701'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8250&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
