

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Selection against Spurious Promoter Motifs Correlates with Translational Efficiency across Bacteria</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000745"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000745" />

    <meta name="citation_title" content="Selection against Spurious Promoter Motifs Correlates with Translational Efficiency across Bacteria"/>
    <meta itemprop="name" content="Selection against Spurious Promoter Motifs Correlates with Translational Efficiency across Bacteria"/>

      <meta name="citation_author" content="Jeffrey L. Froula"/>
            <meta name="citation_author_institution" content="Evolutionary Genomics Program, DOE Joint Genome Institute, Walnut Creek, California, United States of America"/>
      <meta name="citation_author" content="M. Pilar Francino"/>
            <meta name="citation_author_institution" content="Evolutionary Genomics Program, DOE Joint Genome Institute, Walnut Creek, California, United States of America"/>

    <meta name="citation_date" content="2007/8/15"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000745.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e745"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms.; citation_author=T Ikemura; citation_journal_title=Mol Biol Evol; citation_volume=2; citation_number=1; citation_pages=13-34; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=An evolutionary perspective on synonymous codon usage in unicellular organisms.; citation_author=PM Sharp; citation_author=WH Li; citation_journal_title=J Mol Evol; citation_volume=24; citation_number=2; citation_pages=28-38; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=The selection-mutation-drift theory of synonymous codon usage.; citation_author=M Bulmer; citation_journal_title=Genetics; citation_volume=129; citation_number=3; citation_pages=897-907; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of Bacillus subtilis tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis.; citation_author=S Kanaya; citation_author=Y Yamada; citation_author=Y Kudo; citation_author=T Ikemura; citation_journal_title=Gene; citation_volume=238; citation_number=4; citation_pages=143-155; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Codon usage bias from tRNA's point of view: redundancy, specialization, and efficient decoding for translation optimization.; citation_author=EP Rocha; citation_journal_title=Genome Res; citation_volume=14; citation_number=5; citation_pages=2279-2286; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Tethering sigma70 to RNA polymerase reveals high in vivo activity of sigma factors and sigma70-dependent pausing at promoter-distal locations.; citation_author=RA Mooney; citation_author=R Landick; citation_journal_title=Genes Dev; citation_volume=17; citation_number=6; citation_pages=2839-2851; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Association of RNA polymerase with transcribed regions in Escherichia coli.; citation_author=JT Wade; citation_author=K Struhl; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=7; citation_pages=17777-17782; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The effects of selection against spurious transcription factor binding sites.; citation_author=MW Hahn; citation_author=JE Stajich; citation_author=GA Wray; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=8; citation_pages=901-906; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Compilation and analysis of Escherichia coli promoter DNA sequences.; citation_author=DK Hawley; citation_author=WR McClure; citation_journal_title=Nucleic Acids Res; citation_volume=11; citation_number=9; citation_pages=2237-2255; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Compilation of E. coli mRNA promoter sequences.; citation_author=S Lisser; citation_author=H Margalit; citation_journal_title=Nucleic Acids Res; citation_volume=21; citation_number=10; citation_pages=1507-1516; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Hierarchies of base pair preferences in the P22 ant promoter.; citation_author=H Moyle; citation_author=C Waldburger; citation_author=MM Susskind; citation_journal_title=J Bacteriol; citation_volume=173; citation_number=11; citation_pages=1944-1950; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Non-canonical sequence elements in the promoter structure. Cluster analysis of promoters recognized by Escherichia coli RNA polymerase.; citation_author=ON Ozoline; citation_author=AA Deev; citation_author=MV Arkhipova; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=12; citation_pages=4703-4709; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Rapid evolution of cis-regulatory sequences via local point mutations.; citation_author=JR Stone; citation_author=GA Wray; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=13; citation_pages=1764-1770; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Sigma70 promoters in Escherichia coli: specific transcription in dense regions of overlapping promoter-like signals.; citation_author=AM Huerta; citation_author=J Collado-Vides; citation_journal_title=J Mol Biol; citation_volume=333; citation_number=14; citation_pages=261-278; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Proceedings of the SMBE Tri-National Young Investigators' Workshop 2005. Positional conservation of clusters of overlapping promoter-like sequences in enterobacterial genomes.; citation_author=AM Huerta; citation_author=J Collado-Vides; citation_author=MP Francino; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=15; citation_pages=997-1010; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Selection for unequal densities of sigma70 promoter-like signals in different regions of large bacterial genomes.; citation_author=AM Huerta; citation_author=MP Francino; citation_author=E Morett; citation_author=J Collado-Vides; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=16; citation_pages=e185; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The sigma 70 family: sequence conservation and evolutionary relationships.; citation_author=M Lonetto; citation_author=M Gribskov; citation_author=CA Gross; citation_journal_title=J Bacteriol; citation_volume=174; citation_number=17; citation_pages=3843-3849; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Compilation and analysis of Bacillus subtilis sigma A-dependent promoter sequences: evidence for extended contact between RNA polymerase and upstream promoter DNA.; citation_author=JD Helmann; citation_journal_title=Nucleic Acids Res; citation_volume=23; citation_number=18; citation_pages=2351-2360; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Experimental and computational analysis of transcriptional start sites in the cyanobacterium Prochlorococcus MED4.; citation_author=J Vogel; citation_author=IM Axmann; citation_author=H Herzel; citation_author=WR Hess; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=19; citation_pages=2890-2899; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of mycoplasmal promoters in Escherichia coli using a promoter probe vector with Green Fluorescent Protein as reporter system.; citation_author=S Dhandayuthapani; citation_author=WG Rasmussen; citation_author=JB Baseman; citation_journal_title=Gene; citation_volume=215; citation_number=20; citation_pages=213-222; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Transcription in Mycoplasma pneumoniae.; citation_author=J Weiner; citation_author=R Herrmann; citation_author=GF Browning; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=21; citation_pages=4488-4496; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Regulation of the phosphate regulon of Escherichia coli. Activation of pstS transcription by PhoB protein in vitro.; citation_author=K Makino; citation_author=H Shinagawa; citation_author=M Amemura; citation_author=S Kimura; citation_author=A Nakata; citation_journal_title=J Mol Biol; citation_volume=203; citation_number=22; citation_pages=85-95; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=The minus 35-recognition region of Escherichia coli sigma 70 is inessential for initiation of transcription at an “extended minus 10” promoter.; citation_author=A Kumar; citation_author=RA Malloch; citation_author=N Fujita; citation_author=DA Smillie; citation_author=A Ishihama; citation_journal_title=J Mol Biol; citation_volume=232; citation_number=23; citation_pages=406-418; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Recognition of overlapping nucleotides by AraC and the sigma subunit of RNA polymerase.; citation_author=A Dhiman; citation_author=R Schleif; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=24; citation_pages=5076-5081; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=RegulonDB (version 4.0): transcriptional regulation, operon organization and growth conditions in Escherichia coli K-12.; citation_author=H Salgado; citation_author=S Gama-Castro; citation_author=A Martinez-Antonio; citation_author=E Diaz-Peredo; citation_author=F Sanchez-Solano; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=25; citation_pages=D303-306; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A common origin of rickettsiae and certain plant pathogens.; citation_author=WG Weisburg; citation_author=CR Woese; citation_author=ME Dobson; citation_author=E Weiss; citation_journal_title=Science; citation_volume=230; citation_number=26; citation_pages=556-558; citation_date=1985; " />
      <meta name="citation_reference" content="citation_author=B Lewin; citation_number=27; citation_date=2000; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Translation is a non-uniform process. Effect of tRNA availability on the rate of elongation of nascent polypeptide chains.; citation_author=S Varenne; citation_author=J Buc; citation_author=R Lloubes; citation_author=C Lazdunski; citation_journal_title=J Mol Biol; citation_volume=180; citation_number=28; citation_pages=549-576; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=tRNomics: analysis of tRNA genes from 50 genomes of Eukarya, Archaea, and Bacteria reveals anticodon-sparing strategies and domain-specific features.; citation_author=C Marck; citation_author=H Grosjean; citation_journal_title=Rna; citation_volume=8; citation_number=29; citation_pages=1189-1232; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genomic changes following host restriction in bacteria.; citation_author=NA Moran; citation_author=GR Plague; citation_journal_title=Curr Opin Genet Dev; citation_volume=14; citation_number=30; citation_pages=627-633; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Inferring evolutionary processes from phylogenies.; citation_author=M Pagel; citation_journal_title=Zoologica Scripta; citation_volume=26; citation_number=31; citation_pages=331-348; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Inferring the historical patterns of biological evolution.; citation_author=M Pagel; citation_journal_title=Nature; citation_volume=401; citation_number=32; citation_pages=877-884; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The sigma 70 subunit of RNA polymerase induces lacUV5 promoter-proximal pausing of transcription.; citation_author=K Brodolin; citation_author=N Zenkin; citation_author=A Mustaev; citation_author=D Mamaeva; citation_author=H Heumann; citation_journal_title=Nat Struct Mol Biol; citation_volume=11; citation_number=33; citation_pages=551-557; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The sigma 70 subunit of RNA polymerase mediates a promoter-proximal pause at the lac promoter.; citation_author=BE Nickels; citation_author=J Mukhopadhyay; citation_author=SJ Garrity; citation_author=RH Ebright; citation_author=A Hochschild; citation_journal_title=Nat Struct Mol Biol; citation_volume=11; citation_number=34; citation_pages=544-550; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Accelerated evolution and Muller's rachet in endosymbiotic bacteria.; citation_author=NA Moran; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=35; citation_pages=2873-2878; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Sequence evolution in bacterial endosymbionts having extreme base compositions.; citation_author=MA Clark; citation_author=NA Moran; citation_author=P Baumann; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=36; citation_pages=1586-1598; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Evidence for genetic drift in endosymbionts (Buchnera): analyses of protein-coding genes.; citation_author=JJ Wernegreen; citation_author=NA Moran; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=37; citation_pages=83-97; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Reductive genome evolution in Buchnera aphidicola.; citation_author=RC van Ham; citation_author=J Kamerbeek; citation_author=C Palacios; citation_author=C Rausell; citation_author=F Abascal; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=38; citation_pages=581-586; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Consequences of reductive evolution for gene expression in an obligate endosymbiont.; citation_author=JL Wilcox; citation_author=HE Dunbar; citation_author=RD Wolfinger; citation_author=NA Moran; citation_journal_title=Mol Microbiol; citation_volume=48; citation_number=39; citation_pages=1491-1500; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Did the loss of sigma factors initiate pseudogene accumulation in M. leprae?; citation_author=M Madan Babu; citation_journal_title=Trends Microbiol; citation_volume=11; citation_number=40; citation_pages=59-61; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Regulation of transcription in a reduced bacterial genome: nutrient-provisioning genes of the obligate symbiont Buchnera aphidicola.; citation_author=NA Moran; citation_author=HE Dunbar; citation_author=JL Wilcox; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=41; citation_pages=4229-4237; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene expression level influences amino acid usage, but not codon usage, in the tsetse fly endosymbiont Wigglesworthia.; citation_author=JT Herbeck; citation_author=DP Wall; citation_author=JJ Wernegreen; citation_journal_title=Microbiology; citation_volume=149; citation_number=42; citation_pages=2585-2596; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A conservative test of genetic drift in the endosymbiotic bacterium Buchnera: slightly deleterious mutations in the chaperonin groEL.; citation_author=JT Herbeck; citation_author=DJ Funk; citation_author=PH Degnan; citation_author=JJ Wernegreen; citation_journal_title=Genetics; citation_volume=165; citation_number=43; citation_pages=1651-1660; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identifying DNA and protein patterns with statistically significant alignments of multiple sequences.; citation_author=GZ Hertz; citation_author=GD Stormo; citation_journal_title=Bioinformatics; citation_volume=15; citation_number=44; citation_pages=563-577; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=GenBank.; citation_author=DA Benson; citation_author=I Karsch-Mizrachi; citation_author=DJ Lipman; citation_author=J Ostell; citation_author=DL Wheeler; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=45; citation_pages=D34-38; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A powerful non-homology method for the prediction of operons in prokaryotes.; citation_author=G Moreno-Hagelsieb; citation_author=J Collado-Vides; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=46; citation_pages=S329-336; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Significance of nucleotide sequence alignments: a method for random sequence permutation that preserves dinucleotide and codon usage.; citation_author=SF Altschul; citation_author=BW Erickson; citation_journal_title=Mol Biol Evol; citation_volume=2; citation_number=47; citation_pages=526-538; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=MUSCLE: a multiple sequence alignment method with reduced time and space complexity.; citation_author=RC Edgar; citation_journal_title=BMC Bioinformatics; citation_volume=5; citation_number=48; citation_pages=113; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput.; citation_author=RC Edgar; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=49; citation_pages=1792-1797; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MRBAYES: Bayesian inference of phylogenetic trees.; citation_author=JP Huelsenbeck; citation_author=F Ronquist; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=50; citation_pages=754-755; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The rapid generation of mutation data matrices from protein sequences.; citation_author=D Jones; citation_author=W Taylor; citation_author=J Thornton; citation_journal_title=Comput Appl Biosci; citation_volume=8; citation_number=51; citation_pages=275-282; citation_date=1992; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000745" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Selection against Spurious Promoter Motifs Correlates with Translational Efficiency across Bacteria"/>
    <meta name="twitter:description" content="Because binding of RNAP to misplaced sites could compromise the efficiency of transcription, natural selection for the optimization of gene expression should regulate the distribution of DNA motifs capable of RNAP-binding across the genome. Here we analyze the distribution of the &minus;10 promoter motifs that bind the &sigma;70 subunit of RNAP in 42 bacterial genomes. We show that selection on these motifs operates across the genome, maintaining an over-representation of &minus;10 motifs in regulatory sequences while eliminating them from the nonfunctional and, in most cases, from the protein coding regions. In some genomes, however, &minus;10 sites are over-represented in the coding sequences; these sites could induce pauses effecting regulatory roles throughout the length of a transcriptional unit. For nonfunctional sequences, the extent of motif under-representation varies across genomes in a manner that broadly correlates with the number of tRNA genes, a good indicator of translational speed and growth rate. This suggests that minimizing the time invested in gene transcription is an important selective pressure against spurious binding. However, selection against spurious binding is detectable in the reduced genomes of host-restricted bacteria that grow at slow rates, indicating that components of efficiency other than speed may also be important. Minimizing the number of RNAP molecules per cell required for transcription, and the corresponding energetic expense, may be most relevant in slow growers. These results indicate that genome-level properties affecting the efficiency of transcription and translation can respond in an integrated manner to optimize gene expression. The detection of selection against promoter motifs in nonfunctional regions also confirms previous results indicating that no sequence may evolve free of selective constraints, at least in the relatively small and unstructured genomes of bacteria."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000745.g005"/>

  <meta property="og:title" content="Selection against Spurious Promoter Motifs Correlates with Translational Efficiency across Bacteria" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000745" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5355'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2906'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8727&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000745">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000745" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000745&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Selection against Spurious Promoter Motifs Correlates with Translational Efficiency across Bacteria
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jeffrey L. Froula, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Evolutionary Genomics Program, DOE Joint Genome Institute, Walnut Creek, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            M. Pilar Francino
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mpfrancino@lbl.gov">mpfrancino@lbl.gov</a></p>

                <p>Affiliation:
                  Evolutionary Genomics Program, DOE Joint Genome Institute, Walnut Creek, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 15, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000745</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000745" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000745" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000745" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000745" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000745-g001" data-doi="info:doi/10.1371/journal.pone.0000745" data-uri="info:doi/10.1371/journal.pone.0000745.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000745-g002" data-doi="info:doi/10.1371/journal.pone.0000745" data-uri="info:doi/10.1371/journal.pone.0000745.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000745-t001" data-doi="info:doi/10.1371/journal.pone.0000745" data-uri="info:doi/10.1371/journal.pone.0000745.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000745-g003" data-doi="info:doi/10.1371/journal.pone.0000745" data-uri="info:doi/10.1371/journal.pone.0000745.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000745-g004" data-doi="info:doi/10.1371/journal.pone.0000745" data-uri="info:doi/10.1371/journal.pone.0000745.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000745-g005" data-doi="info:doi/10.1371/journal.pone.0000745" data-uri="info:doi/10.1371/journal.pone.0000745.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000745">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000745" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Because binding of RNAP to misplaced sites could compromise the efficiency of transcription, natural selection for the optimization of gene expression should regulate the distribution of DNA motifs capable of RNAP-binding across the genome. Here we analyze the distribution of the −10 promoter motifs that bind the σ<sup>70</sup> subunit of RNAP in 42 bacterial genomes. We show that selection on these motifs operates across the genome, maintaining an over-representation of −10 motifs in regulatory sequences while eliminating them from the nonfunctional and, in most cases, from the protein coding regions. In some genomes, however, −10 sites are over-represented in the coding sequences; these sites could induce pauses effecting regulatory roles throughout the length of a transcriptional unit. For nonfunctional sequences, the extent of motif under-representation varies across genomes in a manner that broadly correlates with the number of tRNA genes, a good indicator of translational speed and growth rate. This suggests that minimizing the time invested in gene transcription is an important selective pressure against spurious binding. However, selection against spurious binding is detectable in the reduced genomes of host-restricted bacteria that grow at slow rates, indicating that components of efficiency other than speed may also be important. Minimizing the number of RNAP molecules per cell required for transcription, and the corresponding energetic expense, may be most relevant in slow growers. These results indicate that genome-level properties affecting the efficiency of transcription and translation can respond in an integrated manner to optimize gene expression. The detection of selection against promoter motifs in nonfunctional regions also confirms previous results indicating that no sequence may evolve free of selective constraints, at least in the relatively small and unstructured genomes of bacteria.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Froula JL, Francino MP (2007) Selection against Spurious Promoter Motifs Correlates with Translational Efficiency across Bacteria. PLoS ONE 2(8):
          e745.
            doi:10.1371/journal.pone.0000745</p><p><strong>Academic Editor: </strong>Matthew Hahn, Indiana University, United States of America</p><p><strong>Received:</strong> May 16, 2007; <strong>Accepted:</strong> July 13, 2007; <strong>Published:</strong> August 15, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>This work has been performed under the auspices of the US Department of Energy's Office of Science, Biological and Environmental Research Program and by the University of California, Lawrence Berkeley National Laboratory under contract No. DE-AC03-76SF00098.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Increased efficiency of cellular functions may impose selective pressures on many factors beyond the amino acid sequences of the cell's proteome. The capacity to produce the necessary proteins at the right time and in the required amount is a system-wide cellular property that requires integration of numerous signals and processes. In bacteria, finely tuned gene expression is the main recourse for immediate interaction with the environment, and should be selected to proceed in a rapid, precise and cost-efficient manner. The efficiency of bacterial translation is known to be under measurable selective pressure; this is evidenced by the fact that, in many bacteria, the most highly expressed genes utilize restricted sets of codons that allow for fast and accurate translation of mRNA, by virtue of their optimal interaction with the most abundant tRNA species for a given amino acid <a href="#pone.0000745-Ikemura1">[1]</a>–<a href="#pone.0000745-Rocha1">[5]</a>. In addition, recent comparative analyses across numerous bacterial genomes have revealed that the abundance and variety of the tRNA gene sets encoded in a genome is highly correlated with the maximal growth rate of the bacterial species. The genomes of fast-growing bacteria contain large numbers of tRNA genes, with the diversity of the encoded tRNA set being low and well adjusted to the codon usage in that species. As a result, the codon usage bias in the highly expressed genes of fast growers is extreme <a href="#pone.0000745-Rocha1">[5]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The selective pressures operating on the general efficiency of transcription are less well understood. Although recent experimental studies suggest that the efficiency of transcription may be regulated at every stage of the process <a href="#pone.0000745-Mooney1">[6]</a>, <a href="#pone.0000745-Wade1">[7]</a>, the frequency of transcription initiation is likely to remain the strongest determinant of total transcriptional output, as evidenced by the complexity of regulatory mechanisms operating at this step. In particular, rapid and accurate localization and binding of correct transcription promoter sites by the RNA polymerase (RNAP) holoenzyme may be one of the critical steps limiting the overall efficiency of gene expression, especially in bacteria, where genomic DNA is not protected by nucleosome-like structures. Spurious binding could result in draining of RNAP molecules away from real promoters, in wasteful attempts to initiate transcription at unproductive sites, or in the generation of unwanted transcripts <a href="#pone.0000745-Hahn1">[8]</a>. However, promoter recognition is likely to be compromised by the fact that RNAP is capable of binding, and possibly initiating transcription, at a multitude of DNA sequences.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Although promoters follow general rules in terms of their nucleotide sequence, structure, length and position, many variations on the canonical promoter model allow for function. In the case of σ<sup>70</sup> promoters, which regulate the majority of <em>E. coli</em> genes, the canonical model is characterized by two hexamers, centered around positions −35 and −10 from the +1 transcription initiation site and separated by 15 to 21 bp, which are bound by distinct regions of the σ<sup>70</sup> molecule. Comparisons of numerous promoter sequences have defined TTGACA and TATAAT as the consensus sequences for the −35 and −10 promoter motifs <a href="#pone.0000745-Hawley1">[9]</a>, <a href="#pone.0000745-Lisser1">[10]</a>. However, transcription initiation does not proceed from promoters having strict consensus sequences, because the binding of the RNAP to a perfect consensus promoter is so strong that it prevents the necessary steps towards promoter clearance and elongation initiation <a href="#pone.0000745-Moyle1">[11]</a>. Rather, variations of the consensi are used which have lower but more functional binding strengths. On average, <em>E. coli</em> promoters preserve only 8 of the 12 canonical bases of the −35 and −10 hexamers <a href="#pone.0000745-Lisser1">[10]</a>, <a href="#pone.0000745-Ozoline1">[12]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Given the variety of sites recognized by RNAP, promoter-like motifs are likely to appear across DNA sequences by neutral mutation and random genetic drift. Computer simulations have shown that this is indeed the case for eukaryotic transcription factor binding sites, which can appear neutrally within microevolutionary timescales <a href="#pone.0000745-Stone1">[13]</a>. Selection for efficient gene expression should therefore act to maintain low frequencies of potential RNAP-binding sites in locations where they are not needed. Several lines of evidence have emerged that suggest that selection may indeed operate to control the genomic distribution of DNA motifs capable of binding to σ<sup>70</sup>. In a series of recent computational analyses, it has been established that the densities of promoter-like sequence motifs differ between regulatory and nonregulatory regions of the genome for a majority of bacteria. The density of promoter-like sequences is high within regulatory regions, in contrast to coding regions and regions located between convergently-transcribed genes, where functional promoters are not required <a href="#pone.0000745-Huerta1">[14]</a>–<a href="#pone.0000745-Huerta3">[16]</a>. However, comparisons among genomic regions can't determine whether σ<sup>70</sup> binding sites are over-represented in regulatory regions and/or under-represented in nonregulatory regions beyond their expected values. To address this issue, actual motif counts need to be compared to the expected occurrences of the motifs given the base composition of each type of region. Hahn, Stajich and Wray <a href="#pone.0000745-Hahn1">[8]</a> conducted such an analysis for the consensus promoter sequences TTGACA and TATAAT, showing that the numbers of these words are below the expectations in both coding and noncoding regions of numerous bacterial genomes. This implies that natural selection acts to remove spurious occurrences of the consensus σ<sup>70</sup>-binding motifs.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Given that σ<sup>70</sup> recognizes many sequences beyond its canonical consensi, we were interested in determining the collective behavior of motifs capable of σ<sup>70</sup>-binding, and in understanding the role of natural selection in shaping their genomic distribution. In order to investigate the specific selective pressures acting on the distribution of these motifs, we decided to study and compare their densities in different genomic regions for bacteria having different lifestyles, growth rates and levels of optimization of the translational process. Such a broad comparison is warranted because several factors indicate that all bacteria contain promoter motifs similar to those found in <em>E. coli,</em> particularly for the −10 region. RNAP is evolutionarily conserved across bacteria, and, in particular, the orthologues of σ<sup>70</sup> can be clearly aligned and contain highly similar motifs for the recognition and binding of promoter sequences <a href="#pone.0000745-Lonetto1">[17]</a>. Recent computational analyses have shown that when genomes of bacterial species belonging to different phyla are searched with <em>E. coli</em> position frequency matrices, collections of promoter-like motifs are detected which are similar to those of <em>E. coli</em> in several respects, including consensus sequence and average score <a href="#pone.0000745-Huerta2">[15]</a>, <a href="#pone.0000745-Huerta3">[16]</a>. Experimental evidence from several bacteria belonging to different phyla also indicates that the −10 box is conserved and displays the same consensus, although the −35 region may be more variable and can't be defined for all genes <a href="#pone.0000745-Helmann1">[18]</a>–<a href="#pone.0000745-Weiner1">[21]</a>. Accordingly, the −10 motif has been shown to be the most essential component of the promoter in <em>E. coli</em>, whereas the −35 motif can be replaced by other combinations of elements upstream of the −10 <a href="#pone.0000745-Makino1">[22]</a>–<a href="#pone.0000745-Dhiman1">[24]</a>. Therefore, we focused our analyses on the −10 motifs present in the genomes of numerous bacterial species belonging to distant phylogenetic groups.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Over- and under-representation of −10 motifs in different genomic regions of <em>E. coli</em></h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We first analyzed the distribution of −10 motifs capable of σ<sup>70</sup>-binding in the genome of <em>E. coli</em>. To this aim, we defined a list of 185 unique −10 hexamers (<a href="#pone.0000745.s001">Table S1</a>) derived from 584 experimentally detected σ<sup>70</sup> <em>E. coli</em> promoters reported in RegulonDB <a href="#pone.0000745-Salgado1">[25]</a>. We obtained the total number of occurrences of these hexamers for three distinct categories of genomic sequence: regulatory, coding and nonfunctional. In order to establish whether these sites are under or overrepresented in each sequence category, we compared the observed counts to the expected occurrences of the hexamers given the base composition of each type of region (see <a href="#s4">Methods</a>).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p><a href="#pone-0000745-g001">Figure 1</a> compares the observed hexamer counts in <em>E. coli</em> with the values obtained for sets of 1000 shuffled sequences for each sequence category, as well as for a control set of randomly generated DNA sequence. The control shows that there were no procedural biases, as the observed value from the original random sequence lies within the curve for shuffled random DNA. The regulatory and nonregulatory sequences behave in an opposite manner. In regulatory regions, the observed total hexamer count is beyond any value obtained in the population of shuffled sequences, whereas the observed count in coding and nonfunctional DNA is below any of the simulated values. We therefore conclude that the hexamers corresponding to the −10 σ<sup>70</sup>-binding motif are significantly over-represented in regulatory regions and under-represented in nonregulatory regions.</p>
<div class="figure" id="pone-0000745-g001"><div class="img"><a name="pone-0000745-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000745" data-uri="info:doi/10.1371/journal.pone.0000745.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000745.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000745.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Comparison of observed and expected −10 motif frequencies in different genomic regions of <em>E. coli</em>.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Distributions were obtained by counting and summing the occurrences of 185 −10 hexamers in sets of 1000 shuffled sequences. This procedure was performed independently for (A) randomly generated, (B) nonfunctional, (C) regulatory, and (D) coding sequences. In each panel, the blue lines represent ± 2 Standard Deviations (SD) from the mean value in the population of shuffled sequences and the red line represents the observed count in the unshuffled <em>E. coli</em> sequence.</p>
<span>doi:10.1371/journal.pone.0000745.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>This implies that, in <em>E.coli</em>, natural selection is favoring a high density of σ<sup>70</sup>-binding sites in regulatory regions, while keeping their frequencies low in other types of sequences. The under-representation of potential σ<sup>70</sup>-binding sites in nonregulatory regions is likely to be related to selection against spurious RNAP-binding to achieve higher transcriptional efficiency. The functional and evolutionary causes that might be responsible for maintaining high densities of promoter-like signals in regulatory regions have been previously described <a href="#pone.0000745-Huerta1">[14]</a>–<a href="#pone.0000745-Huerta3">[16]</a>. Some of the σ<sup>70</sup>-binding sites in regulatory regions could be part of distinct functional promoters that direct transcriptional initiation under different environmental conditions. Alternatively, these sites may not serve as locations of transcription initiation, but rather may affect the function of the real start site in different ways. For instance, these sites could bind RNAP to maintain a local abundance of RNAP molecules close to the transcription start site. They could also play a more specific regulatory role, by affecting the interactions of the functional promoter with regulatory proteins, or by causing the transcription complex to pause after initiation during the early steps of elongation. Finally, several modes of natural selection could operate over regulatory regions that would result in a rapid turnover of functional binding sites and maintenance of numerous cryptic signals, such as compensatory selection, stabilizing selection or selection for robustness to mutation.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>In order to achieve some understanding of the relative importance of selection for and against potential σ<sup>70</sup>-binding sites in different bacteria, we have analyzed the distribution of −10 σ<sup>70</sup> motifs in different regions of 41 additional bacterial genomes (<a href="#pone-0000745-t001">Table 1</a> and <a href="#pone-0000745-g002">Fig. 2</a>).</p>
<div class="figure" id="pone-0000745-g002"><div class="img"><a name="pone-0000745-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000745" data-uri="info:doi/10.1371/journal.pone.0000745.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000745.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000745.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Phylogeny of the 42 bacterial species analyzed.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Phylogeny based on a set of 51 genes present in single copy in most bacterial genomes. Concatenated multiple sequence alignments were used as input for the bayesian tree reconstruction program MrBayes <a href="#pone.0000745-Huelsenbeck1">[50]</a>. The tree topology and branch lengths shown correspond to the most likely tree obtained in two independent runs of MrBayes spanning 250,000 generations each. All clades were recovered with 100% posterior probability. The tree is rooted on the branch leading to the hyperthermophiles <em>Aquifex aeolicus</em> and <em>Thermotoga maritima</em>.</p>
<span>doi:10.1371/journal.pone.0000745.g002</span></div><div class="figure" id="pone-0000745-t001"><div class="img"><a name="pone-0000745-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000745" data-uri="info:doi/10.1371/journal.pone.0000745.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000745.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000745.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Bacterial species analyzed and their general features.</span></strong></p><span>doi:10.1371/journal.pone.0000745.t001</span></div>

<h4>Over- and under-representation of −10 σ<sup>70</sup> motifs in different bacterial genomes</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Given that the motif responsible for binding the −10 promoter region is highly conserved in all bacterial σ<sup>70</sup> orthologues, we used the collection of 185 −10 hexamers defined in <em>E. coli</em> for analyses of all bacterial species. For each genome, we defined regulatory, coding and nonfunctional regions as previously done for <em>E. coli</em> and we obtained the sum of the number of occurrences of each hexamer in every region (<a href="#pone.0000745.s002">Tables S2</a>, <a href="#pone.0000745.s003">S3</a> and <a href="#pone.0000745.s004">S4</a>). Shuffled sequences were generated for every sequence category in every genome analyzed, in order to produce independent genome-specific distributions of expected hexamer counts.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p><a href="#pone-0000745-g003">Figure 3</a> shows the overall pattern obtained across bacterial genomes. Regulatory regions present a significant over-representation of −10 σ<sup>70</sup> motifs in a majority of bacteria, whereas the same sites are significantly under-represented in a majority of the nonfunctional and coding regions. <a href="#pone-0000745-g004">Figure 4</a> shows the deviations in the observed counts of −10 σ<sup>70</sup> motifs measured in numbers of standard deviations from the expected mean (NSD), for the three types of genomic regions (see also <a href="#pone.0000745.s002">Tables S2</a>, <a href="#pone.0000745.s003">S3</a> and <a href="#pone.0000745.s004">S4</a>). Species are ordered according to ascending NSD values for the nonfunctional regions. Because there is no reason to expect any selection for −10 σ<sup>70</sup> motifs in nonfunctional regions, these NSD values may be interpreted as the baseline strength of selection against spurious binding in a given genome. Overall, the NSD in the three regions covary across genomes (regulatory vs. nonfunctional: r = 0.44 p&lt;0.005; coding vs. nonfunctional: r = 0.52 p&lt;0.001 and coding vs. regulatory r = 0.38 p&lt;0.02). Moreover, in nearly all cases, the nonfunctional regions and the regulatory regions present distinguishable patterns of abundance of −10 sites. In the left area of the graph in <a href="#pone-0000745-g004">Figure 4</a>, genomes show the hallmark of selection against spurious sites (under-representation) in nonfunctional regions, whereas regulatory regions do not deviate from expectation or display an over-representation of −10 motifs. In contrast, in the right area of the graph, nonfunctional regions do not deviate from expectation, but most genomes display selection for −10 site over-representation in the regulatory regions. Thus, as a result of selection for −10 sites in the regulatory regions, selection against them in the nonfunctional regions, or the combination of both processes, the localization of RNAP molecules should be biased towards regulatory regions in all genomes. This indicates that the optimization of transcriptional efficiency via the distribution of potential −10 binding sites is an important selective constraint across bacteria. Remarkably, although under-representation of potential σ<sup>70</sup> binding sites is the most common pattern in coding regions (24/42), there also exist several genomes with significant over-representations (9/42), indicating selection of these sites and suggesting some functional effect on the transcription process.</p>
<div class="figure" id="pone-0000745-g003"><div class="img"><a name="pone-0000745-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000745" data-uri="info:doi/10.1371/journal.pone.0000745.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000745.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000745.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Patterns of over- and under-representation of −10 motifs across species.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>For every species, observed and expected −10 motif frequencies in the three different genomic regions were compared as in <a href="#pone-0000745-g001">Figure 1</a>. Over- and under-representation were considered significant when the observed frequency was beyond ±2 SD from the mean value for shuffled sequences.</p>
<span>doi:10.1371/journal.pone.0000745.g003</span></div><div class="figure" id="pone-0000745-g004"><div class="img"><a name="pone-0000745-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000745" data-uri="info:doi/10.1371/journal.pone.0000745.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000745.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000745.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Deviations in the observed counts of −10 motifs in different genomic regions.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>NSD values measure the deviation in the observed counts of −10 motifs in numbers of standard deviations from the expected mean in the corresponding genome, and were obtained by comparing observed counts to distributions of motif frequencies in shuffled sequences. Species are ordered according to ascending NSD values for the nonfunctional regions. NSD values in the white band are not significantly different from expectation. The red line separates species with under-representation of −10 sites in nonfunctional regions (left) from those without (right).</p>
<span>doi:10.1371/journal.pone.0000745.g004</span></div>

<h4>Selection against spurious σ70 sites in nonfunctional regions correlates with translational efficiency</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Next, we explored possible factors influencing the observed patterns of distribution of −10 σ<sup>70</sup> sites in different genomes. The 42 bacterial species analyzed belong to many different phyla and have substantially different lifestyles reflected in their overall cellular and genomic features. In particular, the species considered show a large range of variation in growth rate as well as in the level of optimization of the translational process, as evidenced in the number of encoded tRNA genes (<a href="#pone-0000745-t001">Table 1</a>) and the degree of adaptive bias in codon usage <a href="#pone.0000745-Rocha1">[5]</a>.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Given that the bacterial cell is not highly compartmentalized, the two components of gene expression, transcription and translation, are tightly coupled, with ribosomes actively synthesizing polypetides from the 5′ ends of mRNAs while the 3′ ends have yet to be completed <a href="#pone.0000745-Weisburg1">[26]</a>, <a href="#pone.0000745-Lewin1">[27]</a>. The speed of gene expression should therefore depend on the speeds of both transcription and translation, and conditions, such as high growth rate, that select for an increased rate of gene expression should affect each of the two processes. Consequently, if σ<sup>70</sup>-binding site avoidance in nonregulatory regions is selected for in order to increase the speed of transcription, the degree of under-representation of these sites in a genome should respond to selection for high growth rate and vary in a manner that correlates with translational optimization.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The number of tRNA genes encoded in a bacterial genome is a good measure of the degree of optimization for translational speed. Given that the rate-limiting step during protein synthesis is the diffusion of charged tRNAs to the A site of the ribosome, where peptide bond formation is accomplished <a href="#pone.0000745-Varenne1">[28]</a>, a high concentration of tRNA molecules in the cell will allow for translation to occur more rapidly. Indeed, tRNA abundance varies widely across bacteria and has been shown to evolve rapidly and correlate strongly with maximal growth rate and adaptive codon usage <a href="#pone.0000745-Rocha1">[5]</a>, <a href="#pone.0000745-Marck1">[29]</a>. This indicates that this parameter effectively tracks the needs of bacteria for translational speed and efficiency.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>The abundance of tRNA genes varies greatly across the bacterial genomes employed in this analysis, ranging from 30 in <em>Ureaplasma urealyticum</em> to 98 in <em>Vibrio cholerae</em>, with an overall average of 55 tRNAs per genome (<a href="#pone-0000745-t001">Table 1</a> and <a href="#pone-0000745-g005">Fig. 5</a>). The color-coding in <a href="#pone-0000745-g005">Figure 5</a> distinguishes species according to their maximal growth rate to stress the association between this trait and the number of tRNA genes. Most of the slowest and fastest growers display binding site under-representation in nonfunctional regions (NSD&lt;−2, below the red line) and are distributed across the range of tRNA values in accordance with their varying growth rates <a href="#pone.0000745-Rocha1">[5]</a>. There is a distinct cluster at the extreme of low tRNA abundance (&lt;40) containing the smallest genomes (≤1.5 Mb), which correspond to species that have independently evolved by genome reduction in several bacterial phyla and generally have very slow growth rates <a href="#pone.0000745-Moran1">[30]</a>. In contrast, most of the intermediate growers, with minimal generation times between one and three hours <a href="#pone.0000745-Rocha1">[5]</a>, present a clustered distribution deviating moderately from the average tRNA gene number but display no site under-representation in nonfunctional regions (NSD&gt;−2, above the red line).</p>
<div class="figure" id="pone-0000745-g005"><div class="img"><a name="pone-0000745-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000745" data-uri="info:doi/10.1371/journal.pone.0000745.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000745.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000745.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000745.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000745.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000745.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>NSD of −10 motif counts in nonfunctional DNA correlate negatively with the number of tRNA genes in the genome.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>NSD values are defined as in the text and <a href="#pone-0000745-g004">Figure 4</a>. The red line represents −2 SD from the mean expected value of −10 motif counts in the nonfunctional DNA of the corresponding genome. The green line indicates the average number of tRNA genes in the genomes analyzed (55). The color-coding distinguishes species according to their maximal growth rate (see <a href="#pone-0000745-t001">Table 1</a>); no growth rate estimate was available for <em>Corynebacterium glutamicum</em> (not colored). Black dotted line: regression including all analyzed genomes (r = 0.38 and p&lt;0.02). Black solid line: regression for the subset of genomes with NSD&lt;−2 in nonfunctional DNA (r = 0.63 and p&lt;0.001).</p>
<span>doi:10.1371/journal.pone.0000745.g005</span></div><a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Even though σ<sup>70</sup> site under-representation is more frequently detected in slow growers than in intermediate growers, when all species analyzed are considered the degree of deviation in nonfunctional DNA correlates negatively with the number of tRNA genes in the genome (r = 0.38 and p&lt;0.02, dotted line in <a href="#pone-0000745-g005">Fig. 5</a>). This is consistent with the notion that both characters are affected by selection for high rates of gene expression, although the relative placement of slow and intermediate growers indicates that this selective pressure is not the only factor at play. As selection against σ<sup>70</sup> sites in the nonfunctional DNA appears to be mostly absent from species with intermediate growth rates, we also analyzed separately the subset of slow and fast growers where this selection is actually detected (NSD&lt;−2, below the red line). When only such genomes are included, the correlation between the degree of deviation in nonfunctional DNA and the number of tRNA genes increases (r = 0.63 and p&lt;0.001, solid line in <a href="#pone-0000745-g005">Fig. 5</a>). In order to ensure that the obtained correlations are not due to phylogenetic relatedness, we employed a method that takes into account the topology and branch lengths of the phylogenetic tree relating the analyzed species (<a href="#pone-0000745-g002">Fig. 2</a>). This methodology implements a generalized least squares (GLS) model and likelihood ratio tests to resolve whether correlation between two characters is dependent on the underlying phylogeny <a href="#pone.0000745-Pagel1">[31]</a>, <a href="#pone.0000745-Pagel2">[32]</a>. The correlation among NSD and tRNA genes is recovered when phylogeny is considered (p&lt;0.05 for all species and p&lt;0.0005 for those with NSD&lt;−2).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Overall, the results of our analyses indicate that selection operates on almost all bacterial genomes to shape patterns of distribution of σ<sup>70</sup>-binding sites that distinguish regulatory and nonfunctional DNA. The distinct behavior of these regions may be due to three different situations: 1) selection for under-representation in nonfunctional regions, 2) selection for over-representation in regulatory regions, or 3) the joint effects of both processes. Selection for under-representation of σ<sup>70</sup> sites is also detected in the coding regions of 24 of the 42 genomes analyzed. However, in contrast to the general trend, the coding regions of some genomes (9/42) do not avoid potential σ<sup>70</sup> binding sites, but rather display an over-representation similar to that seen in regulatory DNA. This indicates that some form of natural selection is favoring the presence of σ<sup>70</sup> sites in coding regions beyond random expectations. Although it is impossible to rule out that some selective pressure at the protein level may influence the occurrence of given nucleotide motifs in coding DNA, the fact that we took into account trinucleotide frequencies in generating our baseline expectations implies that only biases at the level of amino acid pairs or triplets could possibly influence our site counts. Therefore, it seems probable that selection for some other function, presumably related to transcription, is favoring the observed over-representation of −10 σ<sup>70</sup> sites in coding DNA.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Of the possible explanations for σ<sup>70</sup> site over-representation advanced for regulatory regions <a href="#pone.0000745-Huerta1">[14]</a>–<a href="#pone.0000745-Huerta3">[16]</a>, only one seems to be equally applicable to coding DNA. It is very unlikely that σ<sup>70</sup>-binding sites in coding regions could function as promoters or regulate transcription by interaction with the transcription start site. However, these sites could affect the rate of transcription elongation by causing the elongation complex to pause at different points as it advances through a transcriptional unit, as has been demonstrated to occur in the earliest phase of elongation after promoter clearance <a href="#pone.0000745-Brodolin1">[33]</a>, <a href="#pone.0000745-Nickels1">[34]</a>. Recent experimental evidence indeed suggests that motifs with affinity for σ<sup>70</sup> may influence the activity of the transcription elongation complex along the extent of a transcriptional unit <a href="#pone.0000745-Mooney1">[6]</a>, <a href="#pone.0000745-Wade1">[7]</a> . This could have a fine-tuning regulatory role, by modulating the speed of elongation and/or by providing pausing times at specific sites to allow for interaction with additional regulatory molecules.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The different types of selective pressures that can affect the presence of σ<sup>70</sup> binding sites across the genome seem to be operating at different extents in different bacteria. Moreover, these patterns of selection are not independent of those affecting gene expression at the translational level, and their relationships to different bacterial lifestyles suggest that they may actually correspond to distinct biological needs. We will now summarize the patterns of selection on transcriptional and translational efficiency in different groups of bacteria and how they might relate to the general biology of the species in which they're found.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Selection against spurious binding is expected to be strongest in species that grow at the fastest rates and therefore require the highest speed and efficiency during transcription and translation. The genomes of fast-growing species bear the expected hallmarks of selective pressure for fast growth at the translational level, such as elevated numbers of tRNA genes (average 78; <a href="#pone-0000745-t001">Table 1</a> and <a href="#pone-0000745-g005">Fig. 5</a>) and high adaptive biases in codon usage <a href="#pone.0000745-Rocha1">[5]</a>. Here we show that 9 of the 11 analyzed genomes from species with minimum generation times under 40 minutes undergo selection against spurious sites in the nonfunctional DNA, to a degree which correlates well with their abundance of tRNA genes (<a href="#pone-0000745-g004">Figs. 4</a> and <a href="#pone-0000745-g005">5</a>). In fact, most fast-growing species respond to selection on −10 sites across all genomic regions, as these sites are also under-represented in the coding DNA and over-represented in the regulatory regions. Thus, selection against spurious binding is pervasive across the non-regulatory DNA of these genomes, but is often overturned in regulatory DNA, indicating that an excess of −10 sites at these regions can confer some advantageous function(s) related to gene expression even at high growth rate. However, it is interesting that the regulatory regions of the two fastest growers, with minimum generation times of 12 minutes -<em>Vibrio cholerae</em> and <em>Clostridium perfringens</em>- <a href="#pone.0000745-Rocha1">[5]</a>, do not deviate from the expected abundances of −10 sites, suggesting that the potential advantages conferred by over-representation of these sites are lost at the extreme of rapid growth.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Most of the analyzed genomes having minimum generation times between 40 minutes and 3 hours do not display any evidence of selection against spurious binding, suggesting that high efficiency during transcription is not critical at intermediate growth rates. Also absent from these genomes are the hallmarks of selection for efficiency at the translational level, as these species have moderate numbers of tRNA genes (average 52) and a moderate degree of adaptive bias in codon usage <a href="#pone.0000745-Rocha1">[5]</a>. These observations suggest that selection on the general efficiency of gene expression is relaxed in species growing at intermediate rates. In contrast, most of these genomes show an over-representation of −10 sites in both the regulatory DNA and the coding regions (<a href="#pone.0000745.s002">Tables S2</a>, <a href="#pone.0000745.s003">S3</a>, <a href="#pone.0000745.s004">S4</a> and <a href="#pone-0000745-g004">Fig. 4</a>). This suggests that bacteria growing at moderate speed do benefit from the potential advantages conferred by an excess of such sites at these regions, presumably through functional effects related to gene expression. In particular, these are the only species where over-representations of −10 sites are detected in the coding regions, suggesting that regulation of transcription elongation by σ<sup>70</sup>-induced pausing might be most effective at moderate rates of growth. The presence of −10 sites in coding regions could potentially diminish the efficiency of transcription initiation through competition with RNAP-binding sites in the upstream regulatory regions, which could be disadvantageous at very fast growth rates. But even the strongest over-representation of −10 sites in the coding regions can be compatible with a moderate rate of growth, as <em>Brucella melitensis</em>, for instance, has a minimal generation time of 2 h <a href="#pone.0000745-Rocha1">[5]</a> in spite of exceeding its expected value of −10 sites in coding regions by nearly 15 NSD. In this context, it is important to notice that −10 sites may strongly exceed their expected value in the coding regions of a genome and still occur in numbers substantially lower than those detected in the regulatory DNA. This is due to the fact that GC content is higher in coding regions, which results in lower baseline expectations for the AT-rich −10 sites.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The slowest growers among the species analyzed are bacteria that can only survive in close association with a eukaryotic host. Most are animal symbionts or parasites whose lifestyle is completely host-restricted. As a consequence, these bacteria exist in small and subdivided populations prone to genetic drift, and are known to be subject to a process of genome reduction and degradation, commonly undergoing rapid sequence evolution with an AT biased mutational pressure <a href="#pone.0000745-Moran1">[30]</a>, <a href="#pone.0000745-Moran2">[35]</a>. As expected, these reduced genomes (≤1.5 Mb; <a href="#pone-0000745-t001">Table 1</a>) bear no evidence of selection on translational efficiency, as they have low tRNA gene numbers (average 36) and display no adaptive biases in codon usage <a href="#pone.0000745-Rocha1">[5]</a>, <a href="#pone.0000745-Clark1">[36]</a>, <a href="#pone.0000745-Wernegreen1">[37]</a>. But, remarkably, these host-restricted species display significant under-representations of −10 sites in the nonregulatory regions of the genome, both coding and nonfunctional. This suggests that minimizing spurious binding is advantageous in these species even if there is no selective pressure for rapid growth. An alternative selective pressure for increasing efficiency during transcription may be energetic economy. Given that genome degradation results in general problems of protein stability and functionality <a href="#pone.0000745-Moran2">[35]</a>, <a href="#pone.0000745-vanHam1">[38]</a>, <a href="#pone.0000745-Wilcox1">[39]</a>, host-restricted species may be under strong selective pressure to perform basic cellular functions with limited numbers of active molecules. The avoidance of spurious binding could represent considerable energetic savings by allowing transcription to proceed with minimal numbers of RNAP molecules per cell. In contrast to nonregulatory regions, the abundances of −10 sites in the regulatory regions of most reduced genomes conform to expectation, with moderate deviations only in <em>Wigglesworthia glossinidia</em> (2.09 NSD) and the spirochaetes <em>Treponema pallidum</em> (2.78 NSD) and <em>Borrelia burgdorferii</em> (2.89 NSD). This suggests that selective pressures for site over-representation are of little effect in host-restricted bacteria. Recent evidence indicates that several other aspects of gene regulation are partially degraded in reduced genomes. Losses of certain promoter sequences, specialized σ factors and regulatory proteins have been documented <a href="#pone.0000745-Wilcox1">[39]</a>–<a href="#pone.0000745-Moran3">[41]</a>. In addition, the collections of predicted −10 and −35 promoter motifs in these genomes deviate from those of most other bacteria, in that they have lower average scores and G to T changes in the consensus sequences <a href="#pone.0000745-Huerta2">[15]</a>, <a href="#pone.0000745-Huerta3">[16]</a>. Such a pattern of moderate degradation of regulatory functions in host-restricted bacteria is most likely due to the accumulation of deleterious mutations by genetic drift, which also causes other moderately maladaptive consequences, including accumulation of deleterious amino acid substitutions and loss of adaptive codon biases <a href="#pone.0000745-Clark1">[36]</a>, <a href="#pone.0000745-Herbeck1">[42]</a>, <a href="#pone.0000745-Herbeck2">[43]</a>.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Because the population structure of host-restricted bacteria only allows them to respond to natural selection when it's relatively strong (in relation to bacterial population size), the fact that these species are mostly responding to selection against spurious binding and not to selection for −10 site over-representation in regulatory regions allows us to gauge the relative strength of these two selective forces. The different response to selection suggests that the deleterious fitness effects of spurious binding are much larger than the potential benefits of finely tuned regulation of gene expression via over-representation of −10 sites, at least in host-associated bacteria. By an analogous reasoning, the low numbers of tRNA genes and the absence of adaptive codon biases in these species suggest that avoidance of RNAP spurious binding is a stronger constraint than optimization of speed and accuracy during translation.</p>

<h4>Conclusions</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Unlike the highly localized selective pressures on amino acid sequence, selection to optimize gene expression can affect all portions of the genome, including sequences not directly involved in regulatory processes. The implications for genome sequence evolution are large, as this distributed selective process signifies that no sequence may be free of selective constraints, at least in the relatively small and unstructured genomes of bacteria <a href="#pone.0000745-Hahn1">[8]</a>. In addition, our study suggests that different genomic traits can respond to specific selective pressures in an integrated manner. Understanding the coevolution among the different molecules, pathways, subsystems and processes of the cell will be of central importance as we enter a new era of synthetically designed genomes for biotechnology and medicine. In the same way that evolutionary studies of individual molecules have provided many insights into organismal adaptation, the analysis of whole genomes in a broad comparative framework has the potential to uncover such coevolutionary patterns.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Obtaining a list of −10 σ<sup>70</sup> sites</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Current information on all aspects of gene regulation in <em>E.coli</em> K12 is summarized in RegulonDB <a href="#pone.0000745-Salgado1">[25]</a>. At the time we started these analyses, Regulon DB contained 584 experimentally detected σ<sup>70</sup> promoters. The hexamers predicted to be the most likely −10 motifs in each of these 584 promoters were obtained using the program WCONSENSUS <a href="#pone.0000745-Hertz1">[44]</a> and algorithms developed by Huerta and Collado-Vides <a href="#pone.0000745-Huerta1">[14]</a>. This set of hexamers contained 185 different words that were used as −10 σ<sup>70</sup> sites in all subsequent analyses (<a href="#pone.0000745.s001">Table S1</a>).</p>


<h4>Subdividing genomes into different regions</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Completely sequenced bacterial genomes were downloaded from GenBank <a href="#pone.0000745-Benson1">[45]</a>, and only chromosomal DNA was considered. Transcription units were defined within each genome according to a method which takes into account distributions of intergenic distances, and genomic DNA was then parsed into three categories: coding, regulatory, and nonfunctional <a href="#pone.0000745-MorenoHagelsieb1">[46]</a>. Regulatory DNA was defined to be 250 bases upstream of the +1 start site of an operon or single transcription unit. 90% of the promoters in <em>E. coli</em> are known to occur within this range. Nonfunctional DNA was defined as sequence located between adjacent genes transcribed in convergent directions, which is not expected to contain promoter elements. Noncoding DNA located beyond 250 bp from +1 or between genes belonging to the same transcriptional unit was not included in the analysis, as the possibility that transcription might start within these sequences can not be excluded. Open reading frames over 1kb were included in the coding DNA category. For every genome, three sequence concatenations were produced by adjoining all the sequences in a DNA category. An “N” was inserted between concatenated sequences to avoid introducing artificial motifs at each concatenation point.</p>


<h4>Generating shuffled sequence</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>To obtain the neutral expectations for hexamer frequencies, we generated shuffled DNA with the same background biases as the original genomic sequences. To this aim we employed a strategy based on a limited eularian walk <a href="#pone.0000745-Altschul1">[47]</a>, which produced shuffled sequences that preserved the trinucleotide frequencies present in the original DNA. An algorithm written in Perl was produced to implement this approach and employed to independently shuffle the concatenations of regulatory, nonfunctional and coding sequences. Shuffling was restricted to 1Mb samples of the total concatenation when actual length exceeded this value (<a href="#pone.0000745.s002">Tables S2</a>, <a href="#pone.0000745.s003">S3</a> and <a href="#pone.0000745.s004">S4</a>). We generated sets of 1000 shuffled sequences for each sequence category in every one of the genomes analyzed. As a procedural control, we also generated and shuffled a pseudo-random sequence totaling 1 Mb.</p>


<h4>Counting hexamers and establishing over- or under-representation</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>In order to establish whether −10 σ<sup>70</sup> sites are under- or over-represented in each sequence category, we compared their observed counts to their expected occurrences given the base composition of each type of region. Neutral expectations were determined separately for each sequence category, by counting the occurrences of the −10 σ<sup>70</sup> hexamers in sets of 1000 shuffled sequences. The number of occurrences of each of the 185 hexamers was independently counted within a given sequence using a sliding window approach, and the counts for each word were summed to obtain the total number of −10 σ<sup>70</sup> sites. For every genome and sequence category, we obtained the mean and standard deviation of the total hexamer counts in the sets of 1000 shuffled sequences, and the observed hexamer counts were compared to these values. Over- and under-representation were considered significant when the observed counts lied beyond 2 Standard Deviations (SD) from the mean. One megabase of random sequence with no nucleotide biases was produced with Perl's random number generator and used as a control where the detected hexamer counts in the original sequence should closely approximate the frequencies obtained in the shuffled set.</p>


<h4>Phylogenetically-based correlation analyses</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>In order to detect possible covariation between different genomic properties, we performed correlation analyses employing comparative methodologies that take into account the phylogenetic relationships among the analyzed bacterial species. Species phylogeny was based on a Bayesian analysis of a set of 51 genes present in single copy in most bacterial genomes. This core of genes is best suited for deep phylogenetic reconstruction because it evolves under high and similar selective pressures across all organisms and maximally avoids gene loss, duplication, horizontal transfer and accelerations of evolutionary rate. Concatenated multiple sequence alignments were constructed using the MUSCLE 3.52 program <a href="#pone.0000745-Edgar1">[48]</a>, <a href="#pone.0000745-Edgar2">[49]</a>, and were used as input for the bayesian tree reconstruction program MrBayes <a href="#pone.0000745-Huelsenbeck1">[50]</a>. MrBayes was run twice independently, using the JTT amino acid substitution matrix <a href="#pone.0000745-Jones1">[51]</a> and 4 gamma rate categories to model among-site rate variation. Each run included four Markov chains, three heated and one cold, starting from random trees and proceeding for 250,000 generations. The tree topology and branch lengths of the most likely tree obtained were used as input for CONTINUOUS, a computer program that implements the generalized least squares (GLS) model for across-species analysis of comparative data <a href="#pone.0000745-Pagel1">[31]</a>, <a href="#pone.0000745-Pagel2">[32]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000745"></div><a name="pone.0000745.s001" id="pone.0000745.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000745.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">List of 185 unique −10 hexamers derived from 584 experimentally detected s70 E. coli promoters.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000745.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000745.s002" id="pone.0000745.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000745.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Observed vs. expected counts of −10 motifs in regulatory regions. Total Sequence Length is the total number of base pairs analyzed for the corresponding type of region in each genome. NSD is the number of standard deviations that separate the observed motif count from the expected mean in that genome, NSD = (Obs-Mean)/SD.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000745.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000745.s003" id="pone.0000745.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000745.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Observed vs. expected counts of −10 motifs in nonfunctional regions. Total Sequence Length is the total number of base pairs analyzed for the corresponding type of region in each genome. NSD is the number of standard deviations that separate the observed motif count from the expected mean in that genome, NSD = (Obs-Mean)/SD.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000745.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0000745.s004" id="pone.0000745.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000745.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Observed vs. expected counts of −10 motifs in coding regions. Total Sequence Length is the total number of base pairs analyzed for the corresponding type of region in each genome. NSD is the number of standard deviations that separate the observed motif count from the expected mean in that genome, NSD = (Obs-Mean)/SD.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000745.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Araceli Huerta Moreno for her help during the initial stages of computational analysis, and Richard Baker and Paramvir Dehal for their help and advise on statistical procedures and phylogenetics. We also thank Matthew Hahn for his thoughtful review that helped us improve the final version of the paper.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MF. Performed the experiments: JF. Analyzed the data: MF JF. Contributed reagents/materials/analysis tools: JF. Wrote the paper: MF JF.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000745-Ikemura1" id="pone.0000745-Ikemura1"></a>Ikemura T (1985) Codon usage and tRNA content in unicellular and multicellular organisms. Mol Biol Evol  2: 13–34.  <ul class="find" data-citedArticleID="978008"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Codon+usage+and+tRNA+content+in+unicellular+and+multicellular+organisms.&amp;auth=&amp;atitle=Codon+usage+and+tRNA+content+in+unicellular+and+multicellular+organisms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+usage+and+tRNA+content+in+unicellular+and+multicellular+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+usage+and+tRNA+content+in+unicellular+and+multicellular+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000745-Sharp1" id="pone.0000745-Sharp1"></a>Sharp PM, Li WH (1986) An evolutionary perspective on synonymous codon usage in unicellular organisms. J Mol Evol  24: 28–38.  <ul class="find" data-citedArticleID="978052" data-doi="10.1007/bf02099948"><li><a href="http://dx.doi.org/10.1007/bf02099948" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+evolutionary+perspective+on+synonymous+codon+usage+in+unicellular+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+evolutionary+perspective+on+synonymous+codon+usage+in+unicellular+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000745-Bulmer1" id="pone.0000745-Bulmer1"></a>Bulmer M (1991) The selection-mutation-drift theory of synonymous codon usage. Genetics  129: 897–907.  <ul class="find" data-citedArticleID="977976"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+selection-mutation-drift+theory+of+synonymous+codon+usage.&amp;auth=&amp;atitle=The+selection-mutation-drift+theory+of+synonymous+codon+usage." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+selection-mutation-drift+theory+of+synonymous+codon+usage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+selection-mutation-drift+theory+of+synonymous+codon+usage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000745-Kanaya1" id="pone.0000745-Kanaya1"></a>Kanaya S, Yamada Y, Kudo Y, Ikemura T (1999) Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of Bacillus subtilis tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis. Gene  238: 143–155.  <ul class="find" data-citedArticleID="978012" data-doi="10.1016/s0378-1119(99)00225-5"><li><a href="http://dx.doi.org/10.1016/s0378-1119(99)00225-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Studies+of+codon+usage+and+tRNA+genes+of+18+unicellular+organisms+and+quantification+of+Bacillus+subtilis+tRNAs%3A+gene+expression+level+and+species-specific+diversity+of+codon+usage+based+on+multivariate+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Studies+of+codon+usage+and+tRNA+genes+of+18+unicellular+organisms+and+quantification+of+Bacillus+subtilis+tRNAs%3A+gene+expression+level+and+species-specific+diversity+of+codon+usage+based+on+multivariate+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000745-Rocha1" id="pone.0000745-Rocha1"></a>Rocha EP (2004) Codon usage bias from tRNA's point of view: redundancy, specialization, and efficient decoding for translation optimization. Genome Res  14: 2279–2286.  <ul class="find" data-citedArticleID="978048" data-doi="10.1101/gr.2896904"><li><a href="http://dx.doi.org/10.1101/gr.2896904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+usage+bias+from+tRNA%27s+point+of+view%3A+redundancy%2C+specialization%2C+and+efficient+decoding+for+translation+optimization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+usage+bias+from+tRNA%27s+point+of+view%3A+redundancy%2C+specialization%2C+and+efficient+decoding+for+translation+optimization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000745-Mooney1" id="pone.0000745-Mooney1"></a>Mooney RA, Landick R (2003) Tethering sigma70 to RNA polymerase reveals high in vivo activity of sigma factors and sigma70-dependent pausing at promoter-distal locations. Genes Dev  17: 2839–2851.  <ul class="find" data-citedArticleID="978028" data-doi="10.1101/gad.1142203"><li><a href="http://dx.doi.org/10.1101/gad.1142203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tethering+sigma70+to+RNA+polymerase+reveals+high+in+vivo+activity+of+sigma+factors+and+sigma70-dependent+pausing+at+promoter-distal+locations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tethering+sigma70+to+RNA+polymerase+reveals+high+in+vivo+activity+of+sigma+factors+and+sigma70-dependent+pausing+at+promoter-distal+locations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000745-Wade1" id="pone.0000745-Wade1"></a>Wade JT, Struhl K (2004) Association of RNA polymerase with transcribed regions in Escherichia coli. Proc Natl Acad Sci U S A  101: 17777–17782.  <ul class="find" data-citedArticleID="978060" data-doi="10.1073/pnas.0404305101"><li><a href="http://dx.doi.org/10.1073/pnas.0404305101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+of+RNA+polymerase+with+transcribed+regions+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+of+RNA+polymerase+with+transcribed+regions+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000745-Hahn1" id="pone.0000745-Hahn1"></a>Hahn MW, Stajich JE, Wray GA (2003) The effects of selection against spurious transcription factor binding sites. Mol Biol Evol  20: 901–906.  <ul class="find" data-citedArticleID="977988" data-doi="10.1093/molbev/msg096"><li><a href="http://dx.doi.org/10.1093/molbev/msg096" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effects+of+selection+against+spurious+transcription+factor+binding+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effects+of+selection+against+spurious+transcription+factor+binding+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000745-Hawley1" id="pone.0000745-Hawley1"></a>Hawley DK, McClure WR (1983) Compilation and analysis of Escherichia coli promoter DNA sequences. Nucleic Acids Res  11: 2237–2255.  <ul class="find" data-citedArticleID="977990" data-doi="10.1093/nar/11.8.2237"><li><a href="http://dx.doi.org/10.1093/nar/11.8.2237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compilation+and+analysis+of+Escherichia+coli+promoter+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compilation+and+analysis+of+Escherichia+coli+promoter+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000745-Lisser1" id="pone.0000745-Lisser1"></a>Lisser S, Margalit H (1993) Compilation of E. coli mRNA promoter sequences. Nucleic Acids Res  21: 1507–1516.  <ul class="find" data-citedArticleID="978018" data-doi="10.1093/nar/21.7.1507"><li><a href="http://dx.doi.org/10.1093/nar/21.7.1507" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compilation+of+E.+coli+mRNA+promoter+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compilation+of+E.+coli+mRNA+promoter+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000745-Moyle1" id="pone.0000745-Moyle1"></a>Moyle H, Waldburger C, Susskind MM (1991) Hierarchies of base pair preferences in the P22 ant promoter. J Bacteriol  173: 1944–1950.  <ul class="find" data-citedArticleID="978038"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hierarchies+of+base+pair+preferences+in+the+P22+ant+promoter.&amp;auth=&amp;atitle=Hierarchies+of+base+pair+preferences+in+the+P22+ant+promoter." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hierarchies+of+base+pair+preferences+in+the+P22+ant+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hierarchies+of+base+pair+preferences+in+the+P22+ant+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000745-Ozoline1" id="pone.0000745-Ozoline1"></a>Ozoline ON, Deev AA, Arkhipova MV (1997) Non-canonical sequence elements in the promoter structure. Cluster analysis of promoters recognized by Escherichia coli RNA polymerase. Nucleic Acids Res  25: 4703–4709.  <ul class="find" data-citedArticleID="978042" data-doi="10.1093/nar/25.23.4703"><li><a href="http://dx.doi.org/10.1093/nar/25.23.4703" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-canonical+sequence+elements+in+the+promoter+structure.+Cluster+analysis+of+promoters+recognized+by+Escherichia+coli+RNA+polymerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-canonical+sequence+elements+in+the+promoter+structure.+Cluster+analysis+of+promoters+recognized+by+Escherichia+coli+RNA+polymerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000745-Stone1" id="pone.0000745-Stone1"></a>Stone JR, Wray GA (2001) Rapid evolution of cis-regulatory sequences via local point mutations. Mol Biol Evol  18: 1764–1770.  <ul class="find" data-citedArticleID="978054" data-doi="10.1093/oxfordjournals.molbev.a003964"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003964" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+evolution+of+cis-regulatory+sequences+via+local+point+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+evolution+of+cis-regulatory+sequences+via+local+point+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000745-Huerta1" id="pone.0000745-Huerta1"></a>Huerta AM, Collado-Vides J (2003) Sigma70 promoters in Escherichia coli: specific transcription in dense regions of overlapping promoter-like signals. J Mol Biol  333: 261–278.  <ul class="find" data-citedArticleID="978002" data-doi="10.1016/j.jmb.2003.07.017"><li><a href="http://dx.doi.org/10.1016/j.jmb.2003.07.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sigma70+promoters+in+Escherichia+coli%3A+specific+transcription+in+dense+regions+of+overlapping+promoter-like+signals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sigma70+promoters+in+Escherichia+coli%3A+specific+transcription+in+dense+regions+of+overlapping+promoter-like+signals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000745-Huerta2" id="pone.0000745-Huerta2"></a>Huerta AM, Collado-Vides J, Francino MP (2006) Proceedings of the SMBE Tri-National Young Investigators' Workshop 2005. Positional conservation of clusters of overlapping promoter-like sequences in enterobacterial genomes. Mol Biol Evol  23: 997–1010.  <ul class="find" data-citedArticleID="978004" data-doi="10.1093/molbev/msk004"><li><a href="http://dx.doi.org/10.1093/molbev/msk004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proceedings+of+the+SMBE+Tri-National+Young+Investigators%27+Workshop+2005.+Positional+conservation+of+clusters+of+overlapping+promoter-like+sequences+in+enterobacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proceedings+of+the+SMBE+Tri-National+Young+Investigators%27+Workshop+2005.+Positional+conservation+of+clusters+of+overlapping+promoter-like+sequences+in+enterobacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000745-Huerta3" id="pone.0000745-Huerta3"></a>Huerta AM, Francino MP, Morett E, Collado-Vides J (2006) Selection for unequal densities of sigma70 promoter-like signals in different regions of large bacterial genomes. PLoS Genet  2: e185.  <ul class="find" data-citedArticleID="978006" data-doi="10.1371/journal.pgen.0020185.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020185.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+for+unequal+densities+of+sigma70+promoter-like+signals+in+different+regions+of+large+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+for+unequal+densities+of+sigma70+promoter-like+signals+in+different+regions+of+large+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000745-Lonetto1" id="pone.0000745-Lonetto1"></a>Lonetto M, Gribskov M, Gross CA (1992) The sigma 70 family: sequence conservation and evolutionary relationships. J Bacteriol  174: 3843–3849.  <ul class="find" data-citedArticleID="978020"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+sigma+70+family%3A+sequence+conservation+and+evolutionary+relationships.&amp;auth=&amp;atitle=The+sigma+70+family%3A+sequence+conservation+and+evolutionary+relationships." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sigma+70+family%3A+sequence+conservation+and+evolutionary+relationships." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sigma+70+family%3A+sequence+conservation+and+evolutionary+relationships.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000745-Helmann1" id="pone.0000745-Helmann1"></a>Helmann JD (1995) Compilation and analysis of Bacillus subtilis sigma A-dependent promoter sequences: evidence for extended contact between RNA polymerase and upstream promoter DNA. Nucleic Acids Res  23: 2351–2360.  <ul class="find" data-citedArticleID="977992" data-doi="10.1093/nar/23.13.2351"><li><a href="http://dx.doi.org/10.1093/nar/23.13.2351" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compilation+and+analysis+of+Bacillus+subtilis+sigma+A-dependent+promoter+sequences%3A+evidence+for+extended+contact+between+RNA+polymerase+and+upstream+promoter+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compilation+and+analysis+of+Bacillus+subtilis+sigma+A-dependent+promoter+sequences%3A+evidence+for+extended+contact+between+RNA+polymerase+and+upstream+promoter+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000745-Vogel1" id="pone.0000745-Vogel1"></a>Vogel J, Axmann IM, Herzel H, Hess WR (2003) Experimental and computational analysis of transcriptional start sites in the cyanobacterium Prochlorococcus MED4. Nucleic Acids Res  31: 2890–2899.  <ul class="find" data-citedArticleID="978058" data-doi="10.1093/nar/gkg398"><li><a href="http://dx.doi.org/10.1093/nar/gkg398" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+and+computational+analysis+of+transcriptional+start+sites+in+the+cyanobacterium+Prochlorococcus+MED4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+and+computational+analysis+of+transcriptional+start+sites+in+the+cyanobacterium+Prochlorococcus+MED4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000745-Dhandayuthapani1" id="pone.0000745-Dhandayuthapani1"></a>Dhandayuthapani S, Rasmussen WG, Baseman JB (1998) Identification of mycoplasmal promoters in Escherichia coli using a promoter probe vector with Green Fluorescent Protein as reporter system. Gene  215: 213–222.  <ul class="find" data-citedArticleID="977980" data-doi="10.1016/s0378-1119(98)00260-1"><li><a href="http://dx.doi.org/10.1016/s0378-1119(98)00260-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+mycoplasmal+promoters+in+Escherichia+coli+using+a+promoter+probe+vector+with+Green+Fluorescent+Protein+as+reporter+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+mycoplasmal+promoters+in+Escherichia+coli+using+a+promoter+probe+vector+with+Green+Fluorescent+Protein+as+reporter+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000745-Weiner1" id="pone.0000745-Weiner1"></a>Weiner J 3rd, Herrmann R, Browning GF (2000) Transcription in Mycoplasma pneumoniae. Nucleic Acids Res  28: 4488–4496.  <ul class="find" data-citedArticleID="978062" data-doi="10.1093/nar/28.22.4488"><li><a href="http://dx.doi.org/10.1093/nar/28.22.4488" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+in+Mycoplasma+pneumoniae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+in+Mycoplasma+pneumoniae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000745-Makino1" id="pone.0000745-Makino1"></a>Makino K, Shinagawa H, Amemura M, Kimura S, Nakata A, et al.  (1988) Regulation of the phosphate regulon of Escherichia coli. Activation of pstS transcription by PhoB protein in vitro. J Mol Biol  203: 85–95.  <ul class="find" data-citedArticleID="978024" data-doi="10.1016/0022-2836(88)90093-9"><li><a href="http://dx.doi.org/10.1016/0022-2836(88)90093-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+the+phosphate+regulon+of+Escherichia+coli.+Activation+of+pstS+transcription+by+PhoB+protein+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+the+phosphate+regulon+of+Escherichia+coli.+Activation+of+pstS+transcription+by+PhoB+protein+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000745-Kumar1" id="pone.0000745-Kumar1"></a>Kumar A, Malloch RA, Fujita N, Smillie DA, Ishihama A, et al.  (1993) The minus 35-recognition region of Escherichia coli sigma 70 is inessential for initiation of transcription at an “extended minus 10” promoter. J Mol Biol  232: 406–418.  <ul class="find" data-citedArticleID="978014" data-doi="10.1006/jmbi.1993.1400"><li><a href="http://dx.doi.org/10.1006/jmbi.1993.1400" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+minus+35-recognition+region+of+Escherichia+coli+sigma+70+is+inessential+for+initiation+of+transcription+at+an+%26%23x201c%3Bextended+minus+10%26%23x201d%3B+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+minus+35-recognition+region+of+Escherichia+coli+sigma+70+is+inessential+for+initiation+of+transcription+at+an+%26%23x201c%3Bextended+minus+10%26%23x201d%3B+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000745-Dhiman1" id="pone.0000745-Dhiman1"></a>Dhiman A, Schleif R (2000) Recognition of overlapping nucleotides by AraC and the sigma subunit of RNA polymerase. J Bacteriol  182: 5076–5081.  <ul class="find" data-citedArticleID="977982" data-doi="10.1128/jb.182.18.5076-5081.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.18.5076-5081.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+overlapping+nucleotides+by+AraC+and+the+sigma+subunit+of+RNA+polymerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+overlapping+nucleotides+by+AraC+and+the+sigma+subunit+of+RNA+polymerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000745-Salgado1" id="pone.0000745-Salgado1"></a>Salgado H, Gama-Castro S, Martinez-Antonio A, Diaz-Peredo E, Sanchez-Solano F, et al.  (2004) RegulonDB (version 4.0): transcriptional regulation, operon organization and growth conditions in Escherichia coli K-12. Nucleic Acids Res  32: D303–306.  <ul class="find" data-citedArticleID="978050" data-doi="10.1093/nar/gkh140"><li><a href="http://dx.doi.org/10.1093/nar/gkh140" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RegulonDB+%28version+4.0%29%3A+transcriptional+regulation%2C+operon+organization+and+growth+conditions+in+Escherichia+coli+K-12." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RegulonDB+%28version+4.0%29%3A+transcriptional+regulation%2C+operon+organization+and+growth+conditions+in+Escherichia+coli+K-12.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000745-Weisburg1" id="pone.0000745-Weisburg1"></a>Weisburg WG, Woese CR, Dobson ME, Weiss E (1985) A common origin of rickettsiae and certain plant pathogens. Science  230: 556–558.  <ul class="find" data-citedArticleID="978064" data-doi="10.1126/science.3931222"><li><a href="http://dx.doi.org/10.1126/science.3931222" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+common+origin+of+rickettsiae+and+certain+plant+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+common+origin+of+rickettsiae+and+certain+plant+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000745-Lewin1" id="pone.0000745-Lewin1"></a>Lewin B (2000) Genes VII. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0000745-Varenne1" id="pone.0000745-Varenne1"></a>Varenne S, Buc J, Lloubes R, Lazdunski C (1984) Translation is a non-uniform process. Effect of tRNA availability on the rate of elongation of nascent polypeptide chains. J Mol Biol  180: 549–576.  <ul class="find" data-citedArticleID="978056" data-doi="10.1016/0022-2836(84)90027-5"><li><a href="http://dx.doi.org/10.1016/0022-2836(84)90027-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translation+is+a+non-uniform+process.+Effect+of+tRNA+availability+on+the+rate+of+elongation+of+nascent+polypeptide+chains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translation+is+a+non-uniform+process.+Effect+of+tRNA+availability+on+the+rate+of+elongation+of+nascent+polypeptide+chains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000745-Marck1" id="pone.0000745-Marck1"></a>Marck C, Grosjean H (2002) tRNomics: analysis of tRNA genes from 50 genomes of Eukarya, Archaea, and Bacteria reveals anticodon-sparing strategies and domain-specific features. Rna  8: 1189–1232.  <ul class="find" data-citedArticleID="978026" data-doi="10.1017/s1355838202022021"><li><a href="http://dx.doi.org/10.1017/s1355838202022021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNomics%3A+analysis+of+tRNA+genes+from+50+genomes+of+Eukarya%2C+Archaea%2C+and+Bacteria+reveals+anticodon-sparing+strategies+and+domain-specific+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNomics%3A+analysis+of+tRNA+genes+from+50+genomes+of+Eukarya%2C+Archaea%2C+and+Bacteria+reveals+anticodon-sparing+strategies+and+domain-specific+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000745-Moran1" id="pone.0000745-Moran1"></a>Moran NA, Plague GR (2004) Genomic changes following host restriction in bacteria. Curr Opin Genet Dev  14: 627–633.  <ul class="find" data-citedArticleID="978030" data-doi="10.1016/j.gde.2004.09.003"><li><a href="http://dx.doi.org/10.1016/j.gde.2004.09.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+changes+following+host+restriction+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+changes+following+host+restriction+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000745-Pagel1" id="pone.0000745-Pagel1"></a>Pagel M (1997) Inferring evolutionary processes from phylogenies. Zoologica Scripta  26: 331–348.  <ul class="find" data-citedArticleID="978044" data-doi="10.1111/j.1463-6409.1997.tb00423.x"><li><a href="http://dx.doi.org/10.1111/j.1463-6409.1997.tb00423.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+evolutionary+processes+from+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+evolutionary+processes+from+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000745-Pagel2" id="pone.0000745-Pagel2"></a>Pagel M (1999) Inferring the historical patterns of biological evolution. Nature  401: 877–884.  <ul class="find" data-citedArticleID="978046" data-doi="10.1038/44766"><li><a href="http://dx.doi.org/10.1038/44766" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+the+historical+patterns+of+biological+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+the+historical+patterns+of+biological+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000745-Brodolin1" id="pone.0000745-Brodolin1"></a>Brodolin K, Zenkin N, Mustaev A, Mamaeva D, Heumann H (2004) The sigma 70 subunit of RNA polymerase induces lacUV5 promoter-proximal pausing of transcription. Nat Struct Mol Biol  11: 551–557.  <ul class="find" data-citedArticleID="977974" data-doi="10.1038/nsmb768"><li><a href="http://dx.doi.org/10.1038/nsmb768" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sigma+70+subunit+of+RNA+polymerase+induces+lacUV5+promoter-proximal+pausing+of+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sigma+70+subunit+of+RNA+polymerase+induces+lacUV5+promoter-proximal+pausing+of+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000745-Nickels1" id="pone.0000745-Nickels1"></a>Nickels BE, Mukhopadhyay J, Garrity SJ, Ebright RH, Hochschild A (2004) The sigma 70 subunit of RNA polymerase mediates a promoter-proximal pause at the lac promoter. Nat Struct Mol Biol  11: 544–550.  <ul class="find" data-citedArticleID="978040" data-doi="10.1038/nsmb757"><li><a href="http://dx.doi.org/10.1038/nsmb757" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sigma+70+subunit+of+RNA+polymerase+mediates+a+promoter-proximal+pause+at+the+lac+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sigma+70+subunit+of+RNA+polymerase+mediates+a+promoter-proximal+pause+at+the+lac+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000745-Moran2" id="pone.0000745-Moran2"></a>Moran NA (1996) Accelerated evolution and Muller's rachet in endosymbiotic bacteria. Proc Natl Acad Sci U S A  93: 2873–2878.  <ul class="find" data-citedArticleID="978032" data-doi="10.1073/pnas.93.7.2873"><li><a href="http://dx.doi.org/10.1073/pnas.93.7.2873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accelerated+evolution+and+Muller%27s+rachet+in+endosymbiotic+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accelerated+evolution+and+Muller%27s+rachet+in+endosymbiotic+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000745-Clark1" id="pone.0000745-Clark1"></a>Clark MA, Moran NA, Baumann P (1999) Sequence evolution in bacterial endosymbionts having extreme base compositions. Mol Biol Evol  16: 1586–1598.  <ul class="find" data-citedArticleID="977978" data-doi="10.1093/oxfordjournals.molbev.a026071"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026071" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+evolution+in+bacterial+endosymbionts+having+extreme+base+compositions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+evolution+in+bacterial+endosymbionts+having+extreme+base+compositions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000745-Wernegreen1" id="pone.0000745-Wernegreen1"></a>Wernegreen JJ, Moran NA (1999) Evidence for genetic drift in endosymbionts (Buchnera): analyses of protein-coding genes. Mol Biol Evol  16: 83–97.  <ul class="find" data-citedArticleID="978066" data-doi="10.1093/oxfordjournals.molbev.a026040"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+genetic+drift+in+endosymbionts+%28Buchnera%29%3A+analyses+of+protein-coding+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+genetic+drift+in+endosymbionts+%28Buchnera%29%3A+analyses+of+protein-coding+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000745-vanHam1" id="pone.0000745-vanHam1"></a>van Ham RC, Kamerbeek J, Palacios C, Rausell C, Abascal F, et al.  (2003) Reductive genome evolution in Buchnera aphidicola. Proc Natl Acad Sci U S A  100: 581–586.  <ul class="find" data-citedArticleID="978070" data-doi="10.1073/pnas.0235981100"><li><a href="http://dx.doi.org/10.1073/pnas.0235981100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reductive+genome+evolution+in+Buchnera+aphidicola." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reductive+genome+evolution+in+Buchnera+aphidicola.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000745-Wilcox1" id="pone.0000745-Wilcox1"></a>Wilcox JL, Dunbar HE, Wolfinger RD, Moran NA (2003) Consequences of reductive evolution for gene expression in an obligate endosymbiont. Mol Microbiol  48: 1491–1500.  <ul class="find" data-citedArticleID="978068" data-doi="10.1046/j.1365-2958.2003.03522.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03522.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consequences+of+reductive+evolution+for+gene+expression+in+an+obligate+endosymbiont." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consequences+of+reductive+evolution+for+gene+expression+in+an+obligate+endosymbiont.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000745-MadanBabu1" id="pone.0000745-MadanBabu1"></a>Madan Babu M (2003) Did the loss of sigma factors initiate pseudogene accumulation in M. leprae? Trends Microbiol  11: 59–61.  <ul class="find" data-citedArticleID="978022" data-doi="10.1016/s0966-842x(02)00031-8"><li><a href="http://dx.doi.org/10.1016/s0966-842x(02)00031-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Did+the+loss+of+sigma+factors+initiate+pseudogene+accumulation+in+M.+leprae%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Did+the+loss+of+sigma+factors+initiate+pseudogene+accumulation+in+M.+leprae%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000745-Moran3" id="pone.0000745-Moran3"></a>Moran NA, Dunbar HE, Wilcox JL (2005) Regulation of transcription in a reduced bacterial genome: nutrient-provisioning genes of the obligate symbiont Buchnera aphidicola. J Bacteriol  187: 4229–4237.  <ul class="find" data-citedArticleID="978034" data-doi="10.1128/jb.187.12.4229-4237.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.12.4229-4237.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+transcription+in+a+reduced+bacterial+genome%3A+nutrient-provisioning+genes+of+the+obligate+symbiont+Buchnera+aphidicola." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+transcription+in+a+reduced+bacterial+genome%3A+nutrient-provisioning+genes+of+the+obligate+symbiont+Buchnera+aphidicola.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000745-Herbeck1" id="pone.0000745-Herbeck1"></a>Herbeck JT, Wall DP, Wernegreen JJ (2003) Gene expression level influences amino acid usage, but not codon usage, in the tsetse fly endosymbiont Wigglesworthia. Microbiology  149: 2585–2596.  <ul class="find" data-citedArticleID="977994" data-doi="10.1099/mic.0.26381-0"><li><a href="http://dx.doi.org/10.1099/mic.0.26381-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+level+influences+amino+acid+usage%2C+but+not+codon+usage%2C+in+the+tsetse+fly+endosymbiont+Wigglesworthia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+level+influences+amino+acid+usage%2C+but+not+codon+usage%2C+in+the+tsetse+fly+endosymbiont+Wigglesworthia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000745-Herbeck2" id="pone.0000745-Herbeck2"></a>Herbeck JT, Funk DJ, Degnan PH, Wernegreen JJ (2003) A conservative test of genetic drift in the endosymbiotic bacterium Buchnera: slightly deleterious mutations in the chaperonin groEL. Genetics  165: 1651–1660.  <ul class="find" data-citedArticleID="977996"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+conservative+test+of+genetic+drift+in+the+endosymbiotic+bacterium+Buchnera%3A+slightly+deleterious+mutations+in+the+chaperonin+groEL.&amp;auth=&amp;atitle=A+conservative+test+of+genetic+drift+in+the+endosymbiotic+bacterium+Buchnera%3A+slightly+deleterious+mutations+in+the+chaperonin+groEL." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+conservative+test+of+genetic+drift+in+the+endosymbiotic+bacterium+Buchnera%3A+slightly+deleterious+mutations+in+the+chaperonin+groEL." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+conservative+test+of+genetic+drift+in+the+endosymbiotic+bacterium+Buchnera%3A+slightly+deleterious+mutations+in+the+chaperonin+groEL.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000745-Hertz1" id="pone.0000745-Hertz1"></a>Hertz GZ, Stormo GD (1999) Identifying DNA and protein patterns with statistically significant alignments of multiple sequences. Bioinformatics  15: 563–577.  <ul class="find" data-citedArticleID="977998" data-doi="10.1093/bioinformatics/15.7.563"><li><a href="http://dx.doi.org/10.1093/bioinformatics/15.7.563" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+DNA+and+protein+patterns+with+statistically+significant+alignments+of+multiple+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+DNA+and+protein+patterns+with+statistically+significant+alignments+of+multiple+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000745-Benson1" id="pone.0000745-Benson1"></a>Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (2005) GenBank. Nucleic Acids Res  33: D34–38.  <ul class="find" data-citedArticleID="977972" data-doi="10.1093/nar/gkm929"><li><a href="http://dx.doi.org/10.1093/nar/gkm929" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GenBank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GenBank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000745-MorenoHagelsieb1" id="pone.0000745-MorenoHagelsieb1"></a>Moreno-Hagelsieb G, Collado-Vides J (2002) A powerful non-homology method for the prediction of operons in prokaryotes. Bioinformatics  18: Suppl 1S329–336.  <ul class="find" data-citedArticleID="978036" data-doi="10.1093/bioinformatics/18.suppl_1.s329"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.suppl_1.s329" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+powerful+non-homology+method+for+the+prediction+of+operons+in+prokaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+powerful+non-homology+method+for+the+prediction+of+operons+in+prokaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000745-Altschul1" id="pone.0000745-Altschul1"></a>Altschul SF, Erickson BW (1985) Significance of nucleotide sequence alignments: a method for random sequence permutation that preserves dinucleotide and codon usage. Mol Biol Evol  2: 526–538.  <ul class="find" data-citedArticleID="977970"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Significance+of+nucleotide+sequence+alignments%3A+a+method+for+random+sequence+permutation+that+preserves+dinucleotide+and+codon+usage.&amp;auth=&amp;atitle=Significance+of+nucleotide+sequence+alignments%3A+a+method+for+random+sequence+permutation+that+preserves+dinucleotide+and+codon+usage." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Significance+of+nucleotide+sequence+alignments%3A+a+method+for+random+sequence+permutation+that+preserves+dinucleotide+and+codon+usage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Significance+of+nucleotide+sequence+alignments%3A+a+method+for+random+sequence+permutation+that+preserves+dinucleotide+and+codon+usage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000745-Edgar1" id="pone.0000745-Edgar1"></a>Edgar RC (2004) MUSCLE: a multiple sequence alignment method with reduced time and space complexity. BMC Bioinformatics  5: 113.  <ul class="find" data-citedArticleID="977984"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity.&amp;auth=&amp;atitle=MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000745-Edgar2" id="pone.0000745-Edgar2"></a>Edgar RC (2004) MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res  32: 1792–1797.  <ul class="find" data-citedArticleID="977986" data-doi="10.1093/nar/gkh340"><li><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000745-Huelsenbeck1" id="pone.0000745-Huelsenbeck1"></a>Huelsenbeck JP, Ronquist F (2001) MRBAYES: Bayesian inference of phylogenetic trees. Bioinformatics  17: 754–755.  <ul class="find" data-citedArticleID="978000" data-doi="10.1093/bioinformatics/17.8.754"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.8.754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MRBAYES%3A+Bayesian+inference+of+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MRBAYES%3A+Bayesian+inference+of+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000745-Jones1" id="pone.0000745-Jones1"></a>Jones D, Taylor W, Thornton J (1992) The rapid generation of mutation data matrices from protein sequences. Comput Appl Biosci  8: 275–282.  <ul class="find" data-citedArticleID="978010" data-doi="10.1093/bioinformatics/8.3.275"><li><a href="http://dx.doi.org/10.1093/bioinformatics/8.3.275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+rapid+generation+of+mutation+data+matrices+from+protein+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+rapid+generation+of+mutation+data+matrices+from+protein+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.XML" value="110118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.PDF" value="343249"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g001.PNG_L" value="883807"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g001.PNG_M" value="128508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g001.PNG_S" value="10773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g001.TIF" value="3123674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g001.PNG_I" value="46383"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g002.PNG_L" value="42984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g002.PNG_M" value="29422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g002.PNG_S" value="9074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g002.TIF" value="115744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g002.PNG_I" value="18199"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.t001.PNG_L" value="368207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.t001.PNG_M" value="149978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.t001.PNG_S" value="16833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.t001.TIF" value="1738242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.t001.PNG_I" value="59001"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g003.PNG_L" value="162350"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g003.PNG_M" value="48954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g003.PNG_S" value="7728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g003.TIF" value="869264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g003.PNG_I" value="18303"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g004.PNG_L" value="647429"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g004.PNG_M" value="101112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g004.PNG_S" value="8452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g004.TIF" value="2614092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g004.PNG_I" value="36241"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g005.PNG_L" value="209778"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g005.PNG_M" value="48582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g005.PNG_S" value="6285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g005.TIF" value="465628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.g005.PNG_I" value="18386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.s001.DOC" value="37376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.s002.DOC" value="54272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.s003.DOC" value="68608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000745.s004.DOC" value="59392"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000745&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000745" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000745&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000745&volume=&issue=&title=Selection against Spurious Promoter Motifs Correlates with Translational Efficiency across Bacteria&author_name=Jeffrey%20L.%20Froula%2C%20M.%20Pilar%20Francino&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000745" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000745" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000745" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000745&amp;t=Selection%20against%20Spurious%20Promoter%20Motifs%20Correlates%20with%20Translational%20Efficiency%20across%20Bacteria" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000745&title=Selection%20against%20Spurious%20Promoter%20Motifs%20Correlates%20with%20Translational%20Efficiency%20across%20Bacteria&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000745&amp;title=Selection%20against%20Spurious%20Promoter%20Motifs%20Correlates%20with%20Translational%20Efficiency%20across%20Bacteria" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000745" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000745&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Selection against Spurious Promoter Motifs Correlates with Translational Efficiency across Bacteria';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000745';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Selection%20against%20Spurious%20Promoter%20Motifs%20Correlates%20with%20Translational%20Efficiency%20across%20Bacteria http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000745" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000745" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="24328"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24328"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="24328"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24328"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24328"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="24328"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="24328"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transfer+RNA%22" title="Search for articles in the subject area:'Transfer RNA'"><div class="flagText">Transfer RNA</div></a>
              <div data-categoryid="21239" data-articleid="24328"
                   data-categoryname="Transfer RNA"
                   class="flagImage" title="Flag 'Transfer RNA' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1340'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2160'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5472&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
