

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Evidence for a Minimal Eukaryotic Phosphoproteome?</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000777"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000777" />

    <meta name="citation_title" content="Evidence for a Minimal Eukaryotic Phosphoproteome?"/>
    <meta itemprop="name" content="Evidence for a Minimal Eukaryotic Phosphoproteome?"/>

      <meta name="citation_author" content="Sander H. Diks"/>
            <meta name="citation_author_institution" content="Kinome Profiling Unit, Department of Cell Biology, University Medical Center Groningen, University of Groningen, Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Kaushal Parikh"/>
            <meta name="citation_author_institution" content="Kinome Profiling Unit, Department of Cell Biology, University Medical Center Groningen, University of Groningen, Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Marijke van der Sijde"/>
            <meta name="citation_author_institution" content="Kinome Profiling Unit, Department of Cell Biology, University Medical Center Groningen, University of Groningen, Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Jos Joore"/>
            <meta name="citation_author_institution" content="Pepscan Presto, Lelystad, Flevoland, The Netherlands"/>
      <meta name="citation_author" content="Tita Ritsema"/>
            <meta name="citation_author_institution" content="Phytopathology, Institute of Environmental Biology, Utrecht University, Utrecht, Utrecht, The Netherlands"/>
      <meta name="citation_author" content="Maikel P. Peppelenbosch"/>
            <meta name="citation_author_institution" content="Kinome Profiling Unit, Department of Cell Biology, University Medical Center Groningen, University of Groningen, Groningen, Groningen, The Netherlands"/>

    <meta name="citation_date" content="2007/8/22"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000777.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e777"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Signaling–2000 and beyond.; citation_author=T Hunter; citation_journal_title=Cell; citation_volume=100; citation_number=1; citation_pages=113-127; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The protein kinase complement of the human genome.; citation_author=G Manning; citation_author=DB Whyte; citation_author=R Martinez; citation_author=T Hunter; citation_author=S Sudarsanam; citation_journal_title=Science; citation_volume=298; citation_number=2; citation_pages=1912-1934; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The repertoire of protein kinases encoded in the draft version of the human genome: atypical variations and uncommon domain combinations.; citation_author=A Krupa; citation_author=N Srinivasan; citation_journal_title=Genome Biol; citation_volume=3; citation_number=3; citation_pages=RESEARCH0066; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Kinomics: characterizing the therapeutically validated kinase space.; citation_author=M Vieth; citation_author=JJ Sutherland; citation_author=DH Robertson; citation_author=RM Campbell; citation_journal_title=Drug Discov Today; citation_volume=10; citation_number=4; citation_pages=839-846; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genomic analysis of the eukaryotic protein kinase superfamily: a perspective.; citation_author=SK Hanks; citation_journal_title=Genome Biol; citation_volume=4; citation_number=5; citation_pages=111; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Novel families of putative protein kinases in bacteria and archaea: evolution of the “eukaryotic” protein kinase superfamily.; citation_author=CJ Leonard; citation_author=L Aravind; citation_author=EV Koonin; citation_journal_title=Genome Res; citation_volume=8; citation_number=6; citation_pages=1038-1047; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Evolution of protein kinase signaling from yeast to man.; citation_author=G Manning; citation_author=GD Plowman; citation_author=T Hunter; citation_author=S Sudarsanam; citation_journal_title=Trends Biochem Sci; citation_volume=27; citation_number=7; citation_pages=514-520; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Classification and functional annotation of eukaryotic protein kinases.; citation_author=D Miranda-Saavedra; citation_author=GJ Barton; citation_journal_title=Proteins; citation_number=8; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis kinome: after the casting.; citation_author=A Champion; citation_author=M Kreis; citation_author=K Mockaitis; citation_author=A Picaud; citation_author=Y Henry; citation_journal_title=Funct Integr Genomics; citation_volume=4; citation_number=9; citation_pages=163-187; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The sea urchin kinome: A first look.; citation_author=CA Bradham; citation_author=KR Foltz; citation_author=WS Beane; citation_author=MI Arnone; citation_author=F Rizzo; citation_journal_title=Dev Biol; citation_volume=300; citation_number=10; citation_pages=180-193; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The protein kinases of budding yeast: six score and more.; citation_author=T Hunter; citation_author=GD Plowman; citation_journal_title=Trends Biochem Sci; citation_volume=22; citation_number=11; citation_pages=18-22; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Protein kinases of the human malaria parasite Plasmodium falciparum: the kinome of a divergent eukaryote.; citation_author=P Ward; citation_author=L Equinet; citation_author=J Packer; citation_author=C Doerig; citation_journal_title=BMC Genomics; citation_volume=5; citation_number=12; citation_pages=79; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Novel protein kinase of Arabidopsis thaliana (APK1) that phosphorylates tyrosine, serine and threonine.; citation_author=T Hirayama; citation_author=A Oka; citation_journal_title=Plant Mol Biol; citation_volume=20; citation_number=13; citation_pages=653-662; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Catalytic roles of yeast GSK3beta/shaggy homolog Rim11p in meiotic activation.; citation_author=K Malathi; citation_author=Y Xiao; citation_author=AP Mitchell; citation_journal_title=Genetics; citation_volume=153; citation_number=14; citation_pages=1145-1152; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Spk1, a new kinase from Saccharomyces cerevisiae, phosphorylates proteins on serine, threonine, and tyrosine.; citation_author=DF Stern; citation_author=P Zheng; citation_author=DR Beidler; citation_author=C Zerillo; citation_journal_title=Mol Cell Biol; citation_volume=11; citation_number=15; citation_pages=987-1001; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Analysis of yeast protein kinases using protein chips.; citation_author=H Zhu; citation_author=JF Klemic; citation_author=S Chang; citation_author=P Bertone; citation_author=A Casamayor; citation_journal_title=Nat Genet; citation_volume=26; citation_number=16; citation_pages=283-289; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of Schizosaccharomyces pombe.; citation_author=V Wood; citation_author=R Gwilliam; citation_author=MA Rajandream; citation_author=M Lyne; citation_author=R Lyne; citation_journal_title=Nature; citation_volume=415; citation_number=17; citation_pages=871-880; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The dictyostelium kinome–analysis of the protein kinases from a simple model organism.; citation_author=JM Goldberg; citation_author=G Manning; citation_author=A Liu; citation_author=P Fey; citation_author=KE Pilcher; citation_author=Y Xu; citation_author=JL Smith; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=18; citation_pages=e38; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=PhosphoBase, a database of phosphorylation sites: release 2.0.; citation_author=A Kreegipuu; citation_author=N Blom; citation_author=S Brunak; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=19; citation_pages=237-239; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Rapid immunosuppressive effects of glucocorticoids mediated through Lck and Fyn.; citation_author=M Lowenberg; citation_author=J Tuynman; citation_author=J Bilderbeek; citation_author=T Gaber; citation_author=F Buttgereit; citation_journal_title=Blood; citation_volume=106; citation_number=20; citation_pages=1703-1710; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Kinome profiling for studying lipopolysaccharide signal transduction in human peripheral blood mononuclear cells.; citation_author=SH Diks; citation_author=K Kok; citation_author=T O'Toole; citation_author=DW Hommes; citation_author=P van Dijken; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=21; citation_pages=49206-49213; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Autopoiesis: the organization of living systems, its characterization and a model.; citation_author=FG Varela; citation_author=HR Maturana; citation_author=R Uribe; citation_journal_title=Curr Mod Biol; citation_volume=5; citation_number=22; citation_pages=187-196; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Expression of the small tyrosine phosphatase (Stp1) in Saccharomyces cerevisiae: a study on protein tyrosine phosphorylation.; citation_author=A Modesti; citation_author=L Bini; citation_author=L Carraresi; citation_author=F Magherini; citation_author=S Liberatori; citation_author=V Pallini; citation_author=G Manao; citation_author=LA Pinna; citation_author=G Raugei; citation_author=G Ramponi; citation_journal_title=Electrophoresis; citation_volume=22; citation_number=23; citation_pages=576-585; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis and experimentation of plant serine/threonine/tyrosine-specific protein kinases.; citation_author=P Rudrabhatla; citation_author=MM Reddy; citation_author=R Rajasekharan; citation_journal_title=Plant Mol Biol; citation_volume=60; citation_number=24; citation_pages=293-319; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mammalian protein histidine kinases.; citation_author=PG Besant; citation_author=E Tan; citation_author=PV Attwood; citation_journal_title=Int J Biochem Cell Biol; citation_volume=35; citation_number=25; citation_pages=297-309; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Protein kinases and phosphatases that act on histidine, lysine, or arginine residues in eukaryotic proteins: a possible regulator of the mitogen-activated protein kinase cascade.; citation_author=HR Matthews; citation_journal_title=Pharmacol Ther; citation_volume=67; citation_number=26; citation_pages=323-350; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Histidine kinases and histidine phosphorylated proteins in mammalian cell biology, signal transduction and cancer.; citation_author=PS Steeg; citation_author=D Palmieri; citation_author=T Ouatas; citation_author=M Salerno; citation_journal_title=Cancer Lett; citation_volume=190; citation_number=27; citation_pages=1-12; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Histidine protein kinases: key signal transducers outside the animal kingdom.; citation_author=PM Wolanin; citation_author=PA Thomason; citation_author=JB Stock; citation_journal_title=Genome Biol; citation_volume=3; citation_number=28; citation_pages=REVIEWS3013; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Hierarchical clustering schemes.; citation_author=SC Johnson; citation_journal_title=Psychometrika; citation_volume=32; citation_number=29; citation_pages=241-254; citation_date=1967; " />
      <meta name="citation_reference" content="citation_title=Early evolution of animal cell signaling and adhesion genes.; citation_author=SA Nichols; citation_author=W Dirks; citation_author=JS Pearse; citation_author=N King; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=30; citation_pages=12451-12456; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Functional development of Src tyrosine kinases during evolution from a unicellular ancestor to multicellular animals.; citation_author=Y Segawa; citation_author=H Suga; citation_author=N Iwabe; citation_author=C Oneyama; citation_author=T Akagi; citation_author=T Miyata; citation_author=M Okada; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=31; citation_pages=12021-12026; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Introductory Mycology. Wiley.; citation_author=CJ Alexopoulos; citation_author=CW Mims; citation_author=M Blackwell; citation_number=32; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The Fungal Dimension of Biodiversity-Magnitude, Significance, and Conservation.; citation_author=DL Hawksworth; citation_journal_title=Mycological Research; citation_volume=95; citation_number=33; citation_pages=641-655; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Structural basis and prediction of substrate specificity in protein serine/threonine kinases.; citation_author=RI Brinkworth; citation_author=RA Breinl; citation_author=B Kobe; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=34; citation_pages=74-79; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Dual specificity kinases–a new family of signal transducers.; citation_author=E Douville; citation_author=P Duncan; citation_author=N Abraham; citation_author=JC Bell; citation_journal_title=Cancer Metastasis Rev; citation_volume=13; citation_number=35; citation_pages=1-7; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Dual-specificity protein kinases: will any hydroxyl do?; citation_author=RA Lindberg; citation_author=AM Quinn; citation_author=T Hunter; citation_journal_title=Trends Biochem Sci; citation_volume=17; citation_number=36; citation_pages=114-119; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=p107wee1 is a dual-specificity kinase that phosphorylates p34cdc2 on tyrosine 15.; citation_author=LL Parker; citation_author=S Atherton-Fessler; citation_author=H Piwnica-Worms; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=89; citation_number=37; citation_pages=2917-2921; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Dyrk, a dual specificity protein kinase with unique structural features whose activity is dependent on tyrosine residues between subdomains VII and VIII.; citation_author=H Kentrup; citation_author=W Becker; citation_author=J Heukelbach; citation_author=A Wilmes; citation_author=A Schurmann; citation_author=C Huppertz; citation_author=H Kainulainen; citation_author=HG Joost; citation_journal_title=J Biol Chem; citation_volume=271; citation_number=38; citation_pages=3488-3495; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Protein tyrosine phosphorylation in the transition to light state 2 of chloroplast thylakoids.; citation_author=J Forsberg; citation_author=JF Allen; citation_journal_title=Photosynth Res; citation_volume=68; citation_number=39; citation_pages=71-79; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Functional characterization of peanut serine/threonine/tyrosine protein kinase: molecular docking and inhibition kinetics with tyrosine kinase inhibitors.; citation_author=P Rudrabhatla; citation_author=R Rajasekharan; citation_journal_title=Biochemistry; citation_volume=43; citation_number=40; citation_pages=12123-12132; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structural evolution of the protein kinase-like superfamily.; citation_author=ED Scheeff; citation_author=PE Bourne; citation_journal_title=PLoS Comput Biol; citation_volume=1; citation_number=41; citation_pages=e49; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000777" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Evidence for a Minimal Eukaryotic Phosphoproteome?"/>
    <meta name="twitter:description" content="BackgroundReversible phosphorylation catalysed by kinases is probably the most important regulatory mechanism in eukaryotes.Methodology/Principal FindingsWe studied the in vitro phosphorylation of peptide arrays exhibiting the majority of PhosphoBase-deposited protein sequences, by factors in cell lysates from representatives of various branches of the eukaryotic species. We derived a set of substrates from the PhosphoBase whose phosphorylation by cellular extracts is common to the divergent members of different kingdoms and thus may be considered a minimal eukaryotic phosphoproteome. The protein kinases (or kinome) responsible for phosphorylation of these substrates are involved in a variety of processes such as transcription, translation, and cytoskeletal reorganisation.Conclusions/SignificanceThese results indicate that the divergence in eukaryotic kinases is not reflected at the level of substrate phosphorylation, revealing the presence of a limited common substrate space for kinases in eukaryotes and suggests the presence of a set of kinase substrates and regulatory mechanisms in an ancestral eukaryote that has since remained constant in eukaryotic life."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000777.g002"/>

  <meta property="og:title" content="Evidence for a Minimal Eukaryotic Phosphoproteome?" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000777" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6985'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=464'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=981&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000777">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000777" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000777&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Evidence for a Minimal Eukaryotic Phosphoproteome?
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sander H. Diks
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:m.p.peppelenbosch@med.umcg.nl">m.p.peppelenbosch@med.umcg.nl</a></p>

                <p>Affiliation:
                  Kinome Profiling Unit, Department of Cell Biology, University Medical Center Groningen, University of Groningen, Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kaushal Parikh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Kinome Profiling Unit, Department of Cell Biology, University Medical Center Groningen, University of Groningen, Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marijke van der Sijde, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Kinome Profiling Unit, Department of Cell Biology, University Medical Center Groningen, University of Groningen, Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jos Joore, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Pepscan Presto, Lelystad, Flevoland, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tita Ritsema, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Phytopathology, Institute of Environmental Biology, Utrecht University, Utrecht, Utrecht, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maikel P. Peppelenbosch
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Kinome Profiling Unit, Department of Cell Biology, University Medical Center Groningen, University of Groningen, Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 22, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000777</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000777" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000777" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000777" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000777" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000777-t001" data-doi="info:doi/10.1371/journal.pone.0000777" data-uri="info:doi/10.1371/journal.pone.0000777.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000777-g001" data-doi="info:doi/10.1371/journal.pone.0000777" data-uri="info:doi/10.1371/journal.pone.0000777.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000777-g002" data-doi="info:doi/10.1371/journal.pone.0000777" data-uri="info:doi/10.1371/journal.pone.0000777.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000777-t002" data-doi="info:doi/10.1371/journal.pone.0000777" data-uri="info:doi/10.1371/journal.pone.0000777.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000777-t003" data-doi="info:doi/10.1371/journal.pone.0000777" data-uri="info:doi/10.1371/journal.pone.0000777.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000777-t004" data-doi="info:doi/10.1371/journal.pone.0000777" data-uri="info:doi/10.1371/journal.pone.0000777.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000777">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000777" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Reversible phosphorylation catalysed by kinases is probably the most important regulatory mechanism in eukaryotes.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We studied the <em>in vitro</em> phosphorylation of peptide arrays exhibiting the majority of PhosphoBase-deposited protein sequences, by factors in cell lysates from representatives of various branches of the eukaryotic species. We derived a set of substrates from the PhosphoBase whose phosphorylation by cellular extracts is common to the divergent members of different kingdoms and thus may be considered a minimal eukaryotic phosphoproteome. The protein kinases (or kinome) responsible for phosphorylation of these substrates are involved in a variety of processes such as transcription, translation, and cytoskeletal reorganisation.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>These results indicate that the divergence in eukaryotic kinases is not reflected at the level of substrate phosphorylation, revealing the presence of a limited common substrate space for kinases in eukaryotes and suggests the presence of a set of kinase substrates and regulatory mechanisms in an ancestral eukaryote that has since remained constant in eukaryotic life.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Diks SH, Parikh K, van der Sijde M, Joore J, Ritsema T, et al.  (2007) Evidence for a Minimal Eukaryotic Phosphoproteome? PLoS ONE 2(8):
          e777.
            doi:10.1371/journal.pone.0000777</p><p><strong>Academic Editor: </strong>Robert Insall, The University of Birmingham, United Kingdom</p><p><strong>Received:</strong> April 26, 2007; <strong>Accepted:</strong> July 22, 2007; <strong>Published:</strong> August 22, 2007</p><p><strong>Copyright:</strong> © 2007 Diks et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported in part by grants from the regional programme of innovative actions in Groningen (IAG) and Top Institute Pharma (TI Pharma). SHD is supported by a Casimir fellowship funded by the Netherlands Organisation for Scientific Research (NWO).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Kinases are enzymes that transfer a phosphate to an acceptor, which can be carbohydrates, lipids or proteins. The superfamily of eukaryotic protein kinases responsible for phosphorylation of specific tyrosine, serine, and threonine residues is generally recognised as the major regulator of virtually all metabolic activities in eukaryotic cells including proliferation, gene expression, motility, vesicular transport, and programmed cell death <a href="#pone.0000777-Hunter1">[1]</a>. Dysregulation of protein phosphorylation plays a major role in many diseases such as cancer and neurodegenerative disorders, and characterisation of the human kinome space revealed that 244 of 518 putative protein kinase genes are currently mapped to disease loci or cancer amplicons <a href="#pone.0000777-Manning1">[2]</a>, <a href="#pone.0000777-Krupa1">[3]</a>. Accordingly, drugs targeting protein kinases are promising avenues for the therapeutic treatment of a plethora of different diseases <a href="#pone.0000777-Vieth1">[4]</a>. In addition, elucidating kinase cascades has proved pivotal for understanding and manipulating cellular behaviour in a variety of divergent eukaryotes.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Most members of the protein kinase superfamily of enzymes can be recognized from their primary sequences by the presence of a catalytic eukaryotic protein kinase (ePK) domain of approximately 250 amino acids, whereas a small number of protein kinases do not share this catalytic domain and are often collectively called atypical kinases <a href="#pone.0000777-Hanks1">[5]</a>, <a href="#pone.0000777-Leonard1">[6]</a>. A comparison of kinase domains both within and between species displays substantial diversity, which is further increased by the non-catalytic functional domains of kinases that are involved in regulation, interactions with other protein partners, or subcellular localisation. This diversity in catalytic and non-catalytic domains explains the functional diversification of kinases within the eukaryotic kingdom. Eukaryotic protein kinases are now generally classified into several major groups <a href="#pone.0000777-Manning2">[7]</a>, <a href="#pone.0000777-MirandaSaavedra1">[8]</a>: the cyclic nucleotide- and Ca2+-/phoshospholipid-dependent kinases (AGC); a group consisting of the cyclin-dependent and cyclin-dependent-like kinases, mitogen-activated kinases, and glycogen synthase kinases (CMGC); the tyrosine kinases (TK); the tyrosine kinase-like group (which are in fact serine/threonine protein kinases) (TKL); the calmodulin-dependent kinases (CAMK); the casein kinase 1 group (CK); and the STE group (first identified in analyses of sterile yeast mutants) that includes the enzymes acting upstream of the mitogen-activated kinases (STE), summarised in <a href="#pone-0000777-t001">table 1</a> which is an extension on the table published by Manning et al. 2002. Plants were considered not have a TK group but instead have a large receptor-like kinase group (RLK). However, recently Miranda–Saavedra <em>et al.</em> have shown using a new library that this is not the case. This new library is outperforms BLASTP and general Pfam hidden Markov models in the classification of kinase domains. They show that plants do contain tyrosine kinases and that diverse classes of organisms have a large overlap in kinase families <a href="#pone.0000777-MirandaSaavedra1">[8]</a>. It should be noted, however, that many eukaryotes also have kinase sequences that are not easily assigned to one of these groups and are referred to as “other protein kinases.” Thus far, pan-eukaryotic classification of kinase substrate sequences has not been attempted and would give better insight in the evolution and variability of substrates and their kinases.</p>
<div class="figure" id="pone-0000777-t001"><div class="img"><a name="pone-0000777-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000777" data-uri="info:doi/10.1371/journal.pone.0000777.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000777.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000777.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Classification of the different kinases in major groups and the numbers of members detected in different organisms by genetic screens (general estimates).</span></strong></p><span>doi:10.1371/journal.pone.0000777.t001</span></div><a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Comparative analyses of genomes have already demonstrated substantial differences in the kinomes of different eukaryotes. These differences are partly reflected in the highly variable number of protein kinase genes present in the genomes of different eukaryotes (<em>e.g.</em>, the <em>A. thaliana</em> genome contains 973 apparent protein kinases <a href="#pone.0000777-Champion1">[9]</a>, the <em>H. sapiens</em> genome contains 518 <a href="#pone.0000777-Manning1">[2]</a>, <em>S. purpuratus</em> is predicted to have 353 protein kinases <a href="#pone.0000777-Bradham1">[10]</a> <em>D. melanogaster</em> appears to have 240 <a href="#pone.0000777-Manning2">[7]</a>, <em>S. cerevisiae</em> has 115 protein kinase genes <a href="#pone.0000777-Hunter2">[11]</a>, and <em>P. falciparum</em> exhibits only 65 putative protein kinases) <a href="#pone.0000777-Ward1">[12]</a>, as well as in highly divergent kinase structures. For instance, plant and unicellular eukaryotic genomes do not contain any apparent kinases from the tyrosine kinase group, despite the detection of phosphorylated tyrosine residues in plants, suggesting that tyrosine phosphorylation in these organisms is possible or that it is mediated via other types of kinases <a href="#pone.0000777-Hirayama1">[13]</a>–<a href="#pone.0000777-Zhu1">[16]</a>. Strikingly, of the 106 putative protein kinases identified in <em>S. pombe</em> on the basis of primary sequence, only 67 have orthologues in <em>S. cerevisiea</em> but 47 have an orthologue in <em>H. sapiens</em> <a href="#pone.0000777-Wood1">[17]</a>, indicating a great deal of conservation in kinases between different organisms. This high degree of overlap might indicate the presence of conservatism in kinase substrates too. In the <em>P. falciparum</em> kinome, 30% of protein kinases belong to the FIKK family of protein kinases that is apicomplexa-specific and not found in other groups of eukaryotes <a href="#pone.0000777-Ward1">[12]</a>. As mentioned previously, plants contain a large group of serine/threonine kinases (receptor-like kinases) not found in other eukaryotes. These RLKs most likely share a common evolutionary origin with the receptor tyrosine kinases present in animals and are thus sometimes collectively referred to as receptor kinases and providing an explanation that tyrosine containing motifs on the PepChip can be phosphorylated by these lysates <a href="#pone.0000777-Champion1">[9]</a>. Interestingly, a recent <em>in silico</em> report on the kinome of the sea urchin has provided new evidence on the evolution of different kinase subfamilies as being an intermediate eukaryote between animals and plants <a href="#pone.0000777-Bradham1">[10]</a>. Fungi such as yeast and <em>Neurospora</em> do not appear to have representatives of the receptor kinase group, whereas the slime mould <em>D. discoideum</em> does have receptor kinases, which fits with the role of receptor kinases in multicellular organisms <a href="#pone.0000777-Goldberg1">[18]</a>. Thus, the eukaryotic family of protein kinases displays substantial diversity at the genetic level between different eukaryotic families.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Whether a kinase is able to phosphorylate its substrate depends on multiple factors such as the physical localisation of both molecules, availability of the substrate to the kinase, but a very important factor, in case of a protein kinase, is the amino acid context surrounding the phospho acceptor. The amino acids surrounding the substrate amino acid confer specificity to which kinase can bind correctly to the substrate and confer a phosphate group to the acceptor. The fact that different kinases have different target substrates is being exploited for phosphoproteome profiling using peptide arrays. In this approach, kinase substrates described in the PhosphoBase phosphorylation site database <a href="#pone.0000777-Kreegipuu1">[19]</a> are spotted on a glass slide and incubated with cell lysates and 33P-labelled γ-ATP. Phosphorylation of target peptides in arrays has provided substrate phosphorylation profiles for LPS-stimulated monocytes and was instrumental for the discovery of Lck and Fyn kinases as early targets of glucocorticoids <a href="#pone.0000777-Lowenberg1">[20]</a>, <a href="#pone.0000777-Diks1">[21]</a>. Importantly, the extent to which the diversity of kinases at the genetic level is reflected in differences in substrate specificity has not been investigated on a large scale.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In the present study, we investigated substrate requirements of phosphoproteomes of several divergent eukaryotes by employing peptide arrays on resting, unstimulated cellular lysates. Our results show that the divergence of eukaryotic protein kinases observed at the level of primary sequence is not completely reflected at the level of substrate phosphorylation, revealing a large overlap in the phosphorylation profiles from lysates of different eukaryotic origins. Furthermore, the identified minimal eukaryotic phosphoproteome suggests the presence of a set of kinase substrates in an ancestral eukaryote that has since remained invariant in eukaryotic life. The phosphoproteome seems to be involved in the maintenance of cell homeostasis as judged from the source of the peptides involved and thus may be a requisite for eukaryotic life <a href="#pone.0000777-Varela1">[22]</a>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Phosphorylation of peptide arrays exhibiting mammalian-biased kinase substrates by divergent eukaryote sources</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A peptide array (PepChip) was employed to determine the preference of cell lysates for kinase substrates. We used the PhosphoBase resource (version 2.0) (now called Phospho.Elm: <a href="http://phospho.elm.eu.org">http://phospho.elm.eu.org</a>) as a source of diverse peptide substrates for kinases <a href="#pone.0000777-Kreegipuu1">[19]</a>. This database contains kinase substrate peptides from diverse organisms, including yeast and plant peptides, but is strongly biased towards mammalian peptide sequences (<a href="#pone-0000777-g001">Figure 1A</a> and <a href="#pone.0000777.s001">Table S1</a>). It must be noted that this set of substrates is just a small subset of known protein kinase substrates and the complete phosphoproteome which is considered to be a lot bigger. Arrays were constructed by covalently coupling chemically synthesized, soluble peptides to glass substrates as described previously <a href="#pone.0000777-Diks1">[21]</a>. Arrays contained 1152 different oligopeptides, covering the majority of substrate peptides available through PhosphoBase (version 2.0). On each carrier, the array was spotted twice to allow assessment of variability in substrate phosphorylation. The final physical dimensions of the array were 25×75 mm. Each peptide spot had a diameter of approximately 250 µm, and each spot was 620 µm from adjacent spots. When the arrays were incubated with [33P-γ] ATP and cell lysates from diverse eukaryotic sources, radioactivity was efficiently incorporated. In contrast, no radioactivity was incorporated when arrays were incubated with [33P-α] ATP and lysates, demonstrating that spot phosphorylation was mediated by specific attachment of the γ-phosphate of ATP to the oligopeptides in the array (<a href="#pone-0000777-g001">Figure 1B</a>). Both the technical replicates (same peptide on the same chip) and the biological replicates were generally of good quality (see supplementary data). Remarkably, the efficiencies by which cell lysates derived from divergent eukaryotic sources phosphorylated specific peptides in the array overlapped substantially, with mammalian lysates showing <sup>33</sup>P incorporation in a large number of spots (<a href="#pone-0000777-g001">Figure 1C</a>). This overlap in phosphorylation of a strongly mammalian-biased set of kinase substrates indicates that a subgroup of kinases is present in divergent eukaryotes has similar peptide sequence requirements for catalysing phosphorylation reactions.</p>
<div class="figure" id="pone-0000777-g001"><div class="img"><a name="pone-0000777-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000777" data-uri="info:doi/10.1371/journal.pone.0000777.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000777.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000777.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>(A) Distribution of the primary origin of substrates spotted on the PepChip by regnum and species.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(B) Incubation of a lysate on a PepChip with equal amounts of [<sup>33</sup>P-α]- and [<sup>33</sup>P-γ]-labelled ATP to show functional phosphorylation (C) Weighted average of at least three PepChip profiles of the different samples.</p>
<span>doi:10.1371/journal.pone.0000777.g001</span></div>

<h4>Serine (S), threonine (T), and tyrosine (Y) phosphorylation is similar in divergent eukaryotes</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Eukaryotic organisms from the plant and fungal kingdoms were not thought to express archetypical tyrosine kinases, as judged from the primary sequences of kinases present in their genomes. However, such organisms have been reported to be capable of phosphorylating tyrosine residues via dual-specificity kinases <a href="#pone.0000777-Hunter2">[11]</a>, <a href="#pone.0000777-Malathi1">[14]</a>–<a href="#pone.0000777-Zhu1">[16]</a>, <a href="#pone.0000777-Modesti1">[23]</a>, <a href="#pone.0000777-Rudrabhatla1">[24]</a>. Another explanation for tyrosine phosphorylation by these lysates is the fact that serine, threonine, and tyrosine are not the only phosphate acceptors in eukaryotes. Several lines of research have already shown that histidine and aspartate are also phosphorylated in eukaryotic cells (reviewed in <a href="#pone.0000777-Besant1">[25]</a>–<a href="#pone.0000777-Steeg1">[27]</a>). Therefore, another explanation could be that histidine and/or aspartate kinases were a possible confounder in our minimal phosphoproteome set (<a href="#pone-0000777-t002">Table 2</a>). This is boosted by the observation that of the 353 monophospho-substrates, only 35% of the serine/threonine motifs contained a histidine (H) or aspartate (D) and 60% of the tyrosine motifs. The difference in the distribution of the H and D amino acids between S/T and Y containing motifs could imply that phosphorylation of histidine (H), aspartate (D) and tyrosine (Y) might have a common ancestry and a coupled evolutionary background which is not unlikely as remarkable similarities exists between these two classes of kinases (reviewed by Wolanin et al. and references therein)<a href="#pone.0000777-Wolanin1">[28]</a>. However, most the tyrosine substrates in our minimal phosphoproteome panel do not contain a histidine or aspartate and therefore common evolutionary backgrounds for histidine, aspartate and tyrosine seems less likely. Thus, the absence of obvious tyrosine kinases in the plant and fungal kingdoms does not result in the inability to phosphorylate tyrosine containing substrates in these organisms. Thus, we compared the relative capacities of animal-derived cell lysates to phosphorylate tyrosine-containing peptide substrates with lysates obtained from the other two eukaryotic kingdoms. To this end, we compared the contribution of serine, threonine, or tyrosine amino acid-containing substrates to the total phosphorylation of all peptide substrates, correcting for the relative abundance of the amino acid in the entire set of substrates. Peptides that can be phosphorylated at more than one residue would bias the results towards a particular amino acid. For example, a peptide that is phosphorylated at two adjacent serines could result in higher signal intensity than a peptide phosphorylated on one threonine. Thus, only those peptides with a single serine, threonine, or tyrosine phosphorylation site were considered (see <a href="#pone.0000777.s002">Table S2</a>). When array phosphorylation was studied in this manner, it appeared that the relative capacities of cell lysates to phosphorylate serine, threonine, or tyrosine substrates were remarkably similar, independent of the kingdom (<a href="#pone-0000777-g002">Figure 2A and B</a>).</p>
<div class="figure" id="pone-0000777-g002"><div class="img"><a name="pone-0000777-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000777" data-uri="info:doi/10.1371/journal.pone.0000777.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000777.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000777.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>(A) Distribution of serine, threonine, and tyrosine substrates of the different phyla on the PepChip, before (left) and after (right) correction for the abundance of each phosphate acceptor on the PepChip.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(B) Venn diagram of spots phosphorylated by the different regna. (C) Hierarchical clustering (according to Johnson <a href="#pone.0000777-Johnson1">[29]</a>) of the phosphorylated motifs among the lysates tested.</p>
<span>doi:10.1371/journal.pone.0000777.g002</span></div><div class="figure" id="pone-0000777-t002"><div class="img"><a name="pone-0000777-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000777" data-uri="info:doi/10.1371/journal.pone.0000777.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000777.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000777.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Distribution of the other phospho acceptors, histidine (H) and aspartate (D) in monophospho motifs (containing only one S, T or Y).</span></strong></p><span>doi:10.1371/journal.pone.0000777.t002</span></div>

<h4>Clustering of array phosphorylation patterns along phylogenetic lines</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We wished to determine whether the patterns of array phosphorylation reflect phylogenetic relations among the various sources of the cell lysates. To this end, we calculated the Spearman correlation coefficient among the array results using all datasets separately (<a href="#pone.0000777.s003">Table S3</a>), combining datasets with similar origin (<a href="#pone.0000777.s004">Table S4</a>) or combining datasets to organisms (<a href="#pone.0000777.s005">Table S5</a>) and then clustered the results according to Johnson (<a href="#pone-0000777-g002">Figure 2C</a>) <a href="#pone.0000777-Johnson1">[29]</a>. Histograms the distributions of positive spots of these three datasets analysis show a normal distribution which is shifted to the right (<a href="#pone.0000777.s007">Histogram S1</a>, <a href="#pone.0000777.s008">S2</a> and <a href="#pone.0000777.s009">S3</a>). Cell lysates from plant and animal sources clustered <em>intra regna</em>, with plants showing less intraregnal variation than animals. This finding could arise from the fact that plant cell lysates were produced from entire organisms, whereas animal lysates were from specialised tissues. Strikingly, the variation in array phosphorylation was comparable between different human or different mouse lysates and between mammalian lysates and a Drosophila lysate. Substrate preferences for kinases do seem to have undergone some diversification after the separation of the animal and plant branches of the eukaryotes. For example, intraregnal variation in phosphorylation between monocotyledons and dicotyledons is smaller than the variation between <em>M. musculus</em> B-cells and <em>H. sapiens</em> macrophages. However, diversity in substrate preferences apparently has not increased after the separation of the Arthropoda and Chordata phyla, and the animal phosphoproteome was established early in animal evolution. This observation corresponds well with analyses of the animal phosphoproteome employing the primary sequences of kinases from divergent animals, as well as with very recent data showing that all major signalling pathways are present in the Porifera phylum, which separated from other animals very early in animal evolution <a href="#pone.0000777-Nichols1">[30]</a>, <a href="#pone.0000777-Segawa1">[31]</a>. Lysates obtained from the fungal kingdom show much more diversity in array phosphorylation than animal lysates, with a <em>P. pastoris</em> lysate actually clustering with plants rather than with other members of the fungal kingdom. A possible explanation is that fungi consist of a diverse group of organisms closely related to plants <a href="#pone.0000777-Alexopoulos1">[32]</a>, <a href="#pone.0000777-Hawksworth1">[33]</a>. It must be noted however, that the other two fungi in the set are also not clustered together, again indicating a large diversity. The diversity in the phosphoproteomes can of course also be caused by the changes in evolutionary pressure on the different samples. It is possible that the evolutionary pressure on metabolic processes in organisms like fungi is of a different level when compared to plants or animals. When the average phosphorylation patterns of the plant, fungal, and animal kingdoms were compared (<a href="#pone-0000777-g002">Figure 2C</a>), the phosphorylation pattern of plants was found to more closely resemble the animal phosphorylation pattern than the fungal pattern.</p>


<h4>Extraction of a minimal phosphoproteome</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The clustering analysis indicated that a significant subset of peptide substrates has remained evolutionarily stabile in terms of phosphorylation, irrespective of the eukaryotic source of the cell lysate. Hence, we decided to investigate the set of substrates whose phosphorylation is shared by all organisms tested in the present study. It appeared that phosphorylation of a set of 128 substrates was common to all organisms tested (If phosphorylation is random, one would expect only 0.6 substrates common in different tissues (binomial distribution 13 positive, 1152 total, cumulative chance 0,02; <em>p</em>&lt;0.01) (supplementary information in <a href="#pone.0000777.s006">Table S6</a>). <a href="#pone-0000777-t003">Table 3</a> lists the set of substrates that are phosphorylated by the divergent eukaryote cell lysates tested. Some of the substrates in the set are highly similar, <em>e.g.</em>, 12 slightly different peptides containing Ser15 of glycogen phosphorylase that were apparently deposited in PhosphoBase as separate substrates. When the list of pan-eukaryotic kinase targets is corrected for essentially identical peptide substrates, 71 different peptide substrates remained. These peptides are, in our set, the substrates for what may be termed a minimal eukaryotic phosphoproteome.</p>
<div class="figure" id="pone-0000777-t003"><div class="img"><a name="pone-0000777-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000777" data-uri="info:doi/10.1371/journal.pone.0000777.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000777.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000777.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Unique substrates phosphorylated in the majority of the profiles tested (supplementary info). Distribution in other species and the conservation of each substrate are also indicated.</span></strong></p><span>doi:10.1371/journal.pone.0000777.t003</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Remarkably, all substrates in <a href="#pone-0000777-t003">table 3</a> contain one or more lysines (K), suggesting a bias in sequence composition or kinase. However, this seems unlikely as studies by Brinkworth <em>et al.</em> showed that, when using prediction models for substrates of kinases, the basic amino acids lysine (K) and arginine (R) are often required for optimal recognition of substrates <a href="#pone.0000777-Brinkworth1">[34]</a>. Therefore the fact that lysine and arginine are present in the substrates in <a href="#pone-0000777-t003">table 3</a> is not completely unexpected. Furthermore, it must be noted that the annotation of the substrates is based on the available data at present, and therefore incomplete. Profiling fungal lysates on a primarily mammalian set of substrates can cause the phosphorylation of irrelevant motifs. However the fact that these motifs are still phosphorylated clearly indicates the possible presence for kinase↔substrate interactions in other organisms even though no direct <em>in vivo</em> relevance is apparent. <a href="#pone-0000777-t004">Table 4</a> shows the distribution of peptide substrates with regard to the molecular functions of their source proteins (according to Gene Ontology, based on human homologues in the Swiss-Prot database whenever possible). These data suggest that the phosphorylation events of this minimal phosphoproteome are associated with cell homeostasis; DNA replication, organisation, and stability; RNA translation; cytoskeletal organisation; motility; transmembrane ion transport; and signal transduction. Indeed, these are functions associated with every eukaryotic cell. When all of the peptides on the chip were subjected to a Blastp search (results are listed on <a href="http://www.koskov.nl">http://www.koskov.nl</a>), not all of the peptides included in the minimal kinome scored higher (<em>p</em>&lt;0.01) for multiregnal homology hits than peptides not present in the pan-eukaryotic kinase substrate set. A possible explanation for this observation is that knowledge of non-mammalian regulation of phosphorylation is not as elaborate as that in mammals.</p>
<div class="figure" id="pone-0000777-t004"><div class="img"><a name="pone-0000777-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000777" data-uri="info:doi/10.1371/journal.pone.0000777.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000777.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000777.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000777.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000777.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000777.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Gene Ontology according to the Swiss-Prot database for the substrates of the minimal kinome, shown for humanized substrate set.</span></strong></p><span>doi:10.1371/journal.pone.0000777.t004</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>For most substrates in this minimal phosphoproteome set, a kinase capable of phosphorylating the peptide has been described (<a href="#pone-0000777-t003">Table 3</a>). Although most of the kinases in this list are common to all eukaryotes (<em>e.g.</em>, phosphorylase kinase and S6 kinase), some are unique to animals. This is especially true for the tyrosine kinases Src, Ros, and c-Fms, which do not have orthologues in plants or fungi. Hence, phosphorylation of tyrosine in the substrates by plant or fungal cell lysates proceeds through other kinases that have similar substrate specificities as the members of the tyrosine kinase family in animals. Possible candidates for such phosphorylation are the kinases belonging to the dual specificity DYRK, STE7, and Wee family of kinases, which are thought to be capable of tyrosine phosphorylation <a href="#pone.0000777-Douville1">[35]</a>–<a href="#pone.0000777-Kentrup1">[38]</a>. However, unique groups of kinases in these species could also be candidates. Interestingly, a recent analysis of the <em>D. discoideum</em> kinome identified a number of kinases that, based on their primary sequences, may act as tyrosine kinases <a href="#pone.0000777-Goldberg1">[18]</a>. In <em>A. thaliana</em>, APK1 is capable of tyrosine phosphorylation <a href="#pone.0000777-Hirayama1">[13]</a>. It would be interesting to investigate whether any of these kinases are responsible for this minimal phosphoproteome tyrosine phosphorylation events observed in the present study. Interestingly, inhibitors of animal tyrosine kinases also function in plants, suggesting substantial structural homology between the kinases responsible for tyrosine phosphorylation in both kingdoms <a href="#pone.0000777-Forsberg1">[39]</a>, <a href="#pone.0000777-Rudrabhatla2">[40]</a>. Further insights into kinase evolution and specificity in different species are needed.</p>


<h4>Peptides in this minimal phosphoproteome are not general kinase substrates</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>An important question concerns the necessity of this minimal eukaryotic phosphoproteome for cell function. The finding that a set of peptide substrates is phosphorylated by cell lysates from highly divergent eukaryotes may indicate that such kinase activity is essential for eukaryotic life and that strong evolutionary pressure exists to prevent its loss. An alternative explanation would be that these substrates act as so-called über-substrates that are relatively non-specifically phosphorylated by multiple kinases. To investigate this question, we incubated chips with relatively high concentrations of purified kinases, <em>e.g.</em>, human Tpl2 (MAP3K8). We observed that the substrates phosphorylated by these purified kinases did not overlap with the set of substrates comprising this minimal eukaryotic phosphoproteome (<em>R</em>2 = 0.11). Thus, phosphorylation of the substrates in the minimal phosphoproteome likely reflects the specific activities of multiple kinases in the eukaryotic cell lysates. However, this can only be validated when the phosphorylation profile all kinases are analysed separately. Apparently, strong evolutionary pressure on a minimal phosphoproteome exists, counteracting changes in substrate specificity for the kinases responsible for these phosphorylation events. By inference, this set of substrate motifs was probably present in an ancestral eukaryotic progenitor cell. This notion is in agreement with a recent study by Scheeff and Bourne provides convincing evidence for the evolution of the various kinase families from a common ancestor <a href="#pone.0000777-Scheeff1">[41]</a>. It is tempting to speculate that this ancestral protein kinase, or other kinases that appeared relatively early in the history of eukaryotic life, delivered the foundation of essential kinase substrate motifs (the minimal eukaryotic phosphoproteome) that remained stabile ever since.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Concluding, in this paper we described the presence of a set of kinase substrates that is recognised and phosphorylated by a diverse panel of eukaryotic cell lysates. This is remarkable since this set is biased towards mammalian motifs, but can still be a target of non mammalian lysates. The fact that this occurs indicates that some level of conservation exists in the eukaryotic linage. Analysis of the preferred substrates revealed that lysine and arginine have an important role in primary sequence of kinase substrates. The possibility that the minimal kinome is produced by a few kinases seems unlikely since single kinase experiments reproduce a very limited part of this panel. However a limited set of kinases can very well be able to reproduce this set. This seems not unlikely since the major function of this set is to maintain cell homeostasis, other more specialised functions require specialised kinases.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Organisms</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Whole extracts of <em>C. albicans</em>, <em>P. pastoris</em>, <em>F. Solani</em>, <em>D. melanogaster</em>, <em>T. aestivium</em> and <em>A. thaliana</em> were used and cell types of <em>M. musculus</em> and <em>H. sapiens</em> were used as mentioned in the text.</p>


<h4>Peptide Array Analysis</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>For kinome array samples, 10<sup>6</sup> ceq or 500 µg were lysed or homogenised in 100 µl of cell lysis buffer (20 mM Tris-HCl, pH 7.5, 150 mM NaCl, 1 mM Na2EDTA, 1 mM EGTA, 1% Triton X-100, 2.5 mM sodium pyrophosphate, 1 mM MgCl2, 1 mM β-glycerophosphate, 1 mM Na<sub>3</sub>VO<sub>4</sub>, 1 mM NaF, 1 µg/ml leupeptin, 1 µg/ml aprotinin, 1 mM PMSF). The cell lysates were subsequently cleared on a 0.22-µm filter. Peptide array incubation mix was produced by adding 10 µl of filter-cleared activation mix (50% glycerol, 50 µM [γ-33P] ATP, 0.05% v/v Brij-35, 0.25 mg/ml bovine serum albumin, [γ-33P] ATP (1000 kBq)). Next, the peptide array mix was added onto the chip, and the chip was kept at 37°C in a humidified stove for 90 min. Subsequently the peptide array was washed twice with Tris-buffered saline with Tween, twice in 2 M NaCl, and twice in demineralized H<sub>2</sub>O and then air-dried. The experiments were performed three times in duplicate.</p>


<h4>Analysis of Peptide Array</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The chips were exposed to a phosphorimager plate for 72 hours, and the density of the spots was measured and analyzed with array software.</p>


<h4>Analysis</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>For the analysis clustering using the spearman correlation coefficient was calculated for each combination of sets and clustering was performed using Johnston hierarchical clustering schemes. Inclusion parameters for each of the kinome profiles are described in supplemental data, <a href="#pone.0000777.s004">Table S4</a>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000777"></div><a name="pone.0000777.s001" id="pone.0000777.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000777.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Substrates spotted on the PepChip.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000777.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.17 MB XLS)</p>
<a name="pone.0000777.s002" id="pone.0000777.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000777.s002">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">List of monophospho-acceptor motifs, with the distribution of histidine and aspartate residues indicated.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000777.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.11 MB XLS)</p>
<a name="pone.0000777.s003" id="pone.0000777.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000777.s003">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Presence (1) or absence (0) of spots phosphorylated by the different lysates tested. This table is used to determine the clustering of the different lysates.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000777.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.32 MB XLS)</p>
<a name="pone.0000777.s004" id="pone.0000777.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000777.s004">Table S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Presence (1) or absence (0) of spots phosphorylated averaged for the different sample background.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000777.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.23 MB XLS)</p>
<a name="pone.0000777.s005" id="pone.0000777.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000777.s005">Table S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Presence (1) or absence (0) of spots phosphorylated averaged for the different organisms.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000777.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.25 MB XLS)</p>
<a name="pone.0000777.s006" id="pone.0000777.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000777.s006">Table S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Calculation of the probability that 116 trials (substrates) are positive (in at least 90% of the samples, corrected for origin bias) in a total number of 1152 trials ( = whole PepChip) using a binominal distribution calculation (<a href="http://www.stat.sc.edu/%E2%88%BCwest/applets/binomialdemo.html">http://www.stat.sc.edu/~west/applets/bin​omialdemo.html</a>). The p-value for success in the binomial distribution is calculated by using the cumulative relative amount of positive spots for every organism. The result of this test shows the chance that a spot is phosphorylated in every set.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000777.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.32 MB XLS)</p>
<a name="pone.0000777.s007" id="pone.0000777.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000777.s007">Histogram S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Histogram of frequency distribution of <a href="#pone.0000777.s003">Table S3</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000777.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0000777.s008" id="pone.0000777.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000777.s008">Histogram S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material8.caption1.p1" name="article1.body1.sec4.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Histogram of frequency distribution of <a href="#pone.0000777.s004">Table S4</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000777.s008</p><a id="article1.body1.sec4.supplementary-material8.caption1.p2" name="article1.body1.sec4.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0000777.s009" id="pone.0000777.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000777.s009">Histogram S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material9.caption1.p1" name="article1.body1.sec4.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Histogram of frequency distribution of <a href="#pone.0000777.s005">Table S5</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000777.s009</p><a id="article1.body1.sec4.supplementary-material9.caption1.p2" name="article1.body1.sec4.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors are grateful for the material that was provided by the different groups.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JJ SD MP TR. Performed the experiments: SD TR Mv KP. Analyzed the data: JJ SD MP Mv KP. Contributed reagents/materials/analysis tools: JJ. Wrote the paper: SD MP TR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000777-Hunter1" id="pone.0000777-Hunter1"></a>Hunter T (2000) Signaling–2000 and beyond. Cell  100: 113–127.  <ul class="find" data-citedArticleID="981082" data-doi="10.1016/s0092-8674(00)81688-8"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81688-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signaling%26%23x2013%3B2000+and+beyond." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signaling%26%23x2013%3B2000+and+beyond.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000777-Manning1" id="pone.0000777-Manning1"></a>Manning G, Whyte DB, Martinez R, Hunter T, Sudarsanam S (2002) The protein kinase complement of the human genome. Science  298: 1912–1934.  <ul class="find" data-citedArticleID="981102" data-doi="10.1126/science.1075762"><li><a href="http://dx.doi.org/10.1126/science.1075762" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+protein+kinase+complement+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+protein+kinase+complement+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000777-Krupa1" id="pone.0000777-Krupa1"></a>Krupa A, Srinivasan N (2002) The repertoire of protein kinases encoded in the draft version of the human genome: atypical variations and uncommon domain combinations. Genome Biol  3: RESEARCH0066.  <ul class="find" data-citedArticleID="981092"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+repertoire+of+protein+kinases+encoded+in+the+draft+version+of+the+human+genome%3A+atypical+variations+and+uncommon+domain+combinations.&amp;auth=&amp;atitle=The+repertoire+of+protein+kinases+encoded+in+the+draft+version+of+the+human+genome%3A+atypical+variations+and+uncommon+domain+combinations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+repertoire+of+protein+kinases+encoded+in+the+draft+version+of+the+human+genome%3A+atypical+variations+and+uncommon+domain+combinations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+repertoire+of+protein+kinases+encoded+in+the+draft+version+of+the+human+genome%3A+atypical+variations+and+uncommon+domain+combinations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000777-Vieth1" id="pone.0000777-Vieth1"></a>Vieth M, Sutherland JJ, Robertson DH, Campbell RM (2005) Kinomics: characterizing the therapeutically validated kinase space. Drug Discov Today  10: 839–846.  <ul class="find" data-citedArticleID="981130" data-doi="10.1016/s1359-6446(05)03477-x"><li><a href="http://dx.doi.org/10.1016/s1359-6446(05)03477-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kinomics%3A+characterizing+the+therapeutically+validated+kinase+space." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kinomics%3A+characterizing+the+therapeutically+validated+kinase+space.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000777-Hanks1" id="pone.0000777-Hanks1"></a>Hanks SK (2003) Genomic analysis of the eukaryotic protein kinase superfamily: a perspective. Genome Biol  4: 111.  <ul class="find" data-citedArticleID="981076"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genomic+analysis+of+the+eukaryotic+protein+kinase+superfamily%3A+a+perspective.&amp;auth=&amp;atitle=Genomic+analysis+of+the+eukaryotic+protein+kinase+superfamily%3A+a+perspective." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+analysis+of+the+eukaryotic+protein+kinase+superfamily%3A+a+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+analysis+of+the+eukaryotic+protein+kinase+superfamily%3A+a+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000777-Leonard1" id="pone.0000777-Leonard1"></a>Leonard CJ, Aravind L, Koonin EV (1998) Novel families of putative protein kinases in bacteria and archaea: evolution of the “eukaryotic” protein kinase superfamily. Genome Res  8: 1038–1047.  <ul class="find" data-citedArticleID="981094"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Novel+families+of+putative+protein+kinases+in+bacteria+and+archaea%3A+evolution+of+the+%26%23x201c%3Beukaryotic%26%23x201d%3B+protein+kinase+superfamily.&amp;auth=&amp;atitle=Novel+families+of+putative+protein+kinases+in+bacteria+and+archaea%3A+evolution+of+the+%26%23x201c%3Beukaryotic%26%23x201d%3B+protein+kinase+superfamily." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+families+of+putative+protein+kinases+in+bacteria+and+archaea%3A+evolution+of+the+%26%23x201c%3Beukaryotic%26%23x201d%3B+protein+kinase+superfamily." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+families+of+putative+protein+kinases+in+bacteria+and+archaea%3A+evolution+of+the+%26%23x201c%3Beukaryotic%26%23x201d%3B+protein+kinase+superfamily.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000777-Manning2" id="pone.0000777-Manning2"></a>Manning G, Plowman GD, Hunter T, Sudarsanam S (2002) Evolution of protein kinase signaling from yeast to man. Trends Biochem Sci  27: 514–520.  <ul class="find" data-citedArticleID="981104" data-doi="10.1016/s0968-0004(02)02179-5"><li><a href="http://dx.doi.org/10.1016/s0968-0004(02)02179-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+protein+kinase+signaling+from+yeast+to+man." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+protein+kinase+signaling+from+yeast+to+man.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000777-MirandaSaavedra1" id="pone.0000777-MirandaSaavedra1"></a>Miranda-Saavedra D, Barton GJ (2007) Classification and functional annotation of eukaryotic protein kinases. Proteins.   <ul class="find" data-citedArticleID="981108" data-doi="10.1002/prot.21444"><li><a href="http://dx.doi.org/10.1002/prot.21444" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Classification+and+functional+annotation+of+eukaryotic+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Classification+and+functional+annotation+of+eukaryotic+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000777-Champion1" id="pone.0000777-Champion1"></a>Champion A, Kreis M, Mockaitis K, Picaud A, Henry Y (2004) Arabidopsis kinome: after the casting. Funct Integr Genomics  4: 163–187.  <ul class="find" data-citedArticleID="981066" data-doi="10.1007/s10142-003-0096-4"><li><a href="http://dx.doi.org/10.1007/s10142-003-0096-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+kinome%3A+after+the+casting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+kinome%3A+after+the+casting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000777-Bradham1" id="pone.0000777-Bradham1"></a>Bradham CA, Foltz KR, Beane WS, Arnone MI, Rizzo F, et al.  (2006) The sea urchin kinome: A first look. Dev Biol  300: 180–193.  <ul class="find" data-citedArticleID="981062" data-doi="10.1016/j.ydbio.2006.08.074"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2006.08.074" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sea+urchin+kinome%3A+A+first+look." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sea+urchin+kinome%3A+A+first+look.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000777-Hunter2" id="pone.0000777-Hunter2"></a>Hunter T, Plowman GD (1997) The protein kinases of budding yeast: six score and more. Trends Biochem Sci  22: 18–22.  <ul class="find" data-citedArticleID="981084" data-doi="10.1016/s0968-0004(96)10068-2"><li><a href="http://dx.doi.org/10.1016/s0968-0004(96)10068-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+protein+kinases+of+budding+yeast%3A+six+score+and+more." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+protein+kinases+of+budding+yeast%3A+six+score+and+more.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000777-Ward1" id="pone.0000777-Ward1"></a>Ward P, Equinet L, Packer J, Doerig C (2004) Protein kinases of the human malaria parasite Plasmodium falciparum: the kinome of a divergent eukaryote. BMC Genomics  5: 79.  <ul class="find" data-citedArticleID="981132" data-doi="10.1186/1471-2164-5-79"><li><a href="http://dx.doi.org/10.1186/1471-2164-5-79" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+kinases+of+the+human+malaria+parasite+Plasmodium+falciparum%3A+the+kinome+of+a+divergent+eukaryote." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+kinases+of+the+human+malaria+parasite+Plasmodium+falciparum%3A+the+kinome+of+a+divergent+eukaryote.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000777-Hirayama1" id="pone.0000777-Hirayama1"></a>Hirayama T, Oka A (1992) Novel protein kinase of Arabidopsis thaliana (APK1) that phosphorylates tyrosine, serine and threonine. Plant Mol Biol  20: 653–662.  <ul class="find" data-citedArticleID="981080" data-doi="10.1007/bf00046450"><li><a href="http://dx.doi.org/10.1007/bf00046450" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+protein+kinase+of+Arabidopsis+thaliana+%28APK1%29+that+phosphorylates+tyrosine%2C+serine+and+threonine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+protein+kinase+of+Arabidopsis+thaliana+%28APK1%29+that+phosphorylates+tyrosine%2C+serine+and+threonine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000777-Malathi1" id="pone.0000777-Malathi1"></a>Malathi K, Xiao Y, Mitchell AP (1999) Catalytic roles of yeast GSK3beta/shaggy homolog Rim11p in meiotic activation. Genetics  153: 1145–1152.  <ul class="find" data-citedArticleID="981100"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Catalytic+roles+of+yeast+GSK3beta%2Fshaggy+homolog+Rim11p+in+meiotic+activation.&amp;auth=&amp;atitle=Catalytic+roles+of+yeast+GSK3beta%2Fshaggy+homolog+Rim11p+in+meiotic+activation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Catalytic+roles+of+yeast+GSK3beta%2Fshaggy+homolog+Rim11p+in+meiotic+activation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Catalytic+roles+of+yeast+GSK3beta%2Fshaggy+homolog+Rim11p+in+meiotic+activation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000777-Stern1" id="pone.0000777-Stern1"></a>Stern DF, Zheng P, Beidler DR, Zerillo C (1991) Spk1, a new kinase from Saccharomyces cerevisiae, phosphorylates proteins on serine, threonine, and tyrosine. Mol Cell Biol  11: 987–1001.  <ul class="find" data-citedArticleID="981126"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Spk1%2C+a+new+kinase+from+Saccharomyces+cerevisiae%2C+phosphorylates+proteins+on+serine%2C+threonine%2C+and+tyrosine.&amp;auth=&amp;atitle=Spk1%2C+a+new+kinase+from+Saccharomyces+cerevisiae%2C+phosphorylates+proteins+on+serine%2C+threonine%2C+and+tyrosine." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spk1%2C+a+new+kinase+from+Saccharomyces+cerevisiae%2C+phosphorylates+proteins+on+serine%2C+threonine%2C+and+tyrosine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spk1%2C+a+new+kinase+from+Saccharomyces+cerevisiae%2C+phosphorylates+proteins+on+serine%2C+threonine%2C+and+tyrosine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000777-Zhu1" id="pone.0000777-Zhu1"></a>Zhu H, Klemic JF, Chang S, Bertone P, Casamayor A, et al.  (2000) Analysis of yeast protein kinases using protein chips. Nat Genet  26: 283–289.  <ul class="find" data-citedArticleID="981138" data-doi="10.1038/81576"><li><a href="http://dx.doi.org/10.1038/81576" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+yeast+protein+kinases+using+protein+chips." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+yeast+protein+kinases+using+protein+chips.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000777-Wood1" id="pone.0000777-Wood1"></a>Wood V, Gwilliam R, Rajandream MA, Lyne M, Lyne R, et al.  (2002) The genome sequence of Schizosaccharomyces pombe. Nature  415: 871–880.  <ul class="find" data-citedArticleID="981136" data-doi="10.1038/nature724"><li><a href="http://dx.doi.org/10.1038/nature724" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000777-Goldberg1" id="pone.0000777-Goldberg1"></a>Goldberg JM, Manning G, Liu A, Fey P, Pilcher KE, Xu Y, Smith JL (2006) The dictyostelium kinome–analysis of the protein kinases from a simple model organism. PLoS Genet  2: e38.  <ul class="find" data-citedArticleID="981074" data-doi="10.1371/journal.pgen.0020038"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dictyostelium+kinome%26%23x2013%3Banalysis+of+the+protein+kinases+from+a+simple+model+organism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dictyostelium+kinome%26%23x2013%3Banalysis+of+the+protein+kinases+from+a+simple+model+organism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000777-Kreegipuu1" id="pone.0000777-Kreegipuu1"></a>Kreegipuu A, Blom N, Brunak S (1999) PhosphoBase, a database of phosphorylation sites: release 2.0. Nucleic Acids Res  27: 237–239.  <ul class="find" data-citedArticleID="981090" data-doi="10.1093/nar/27.1.237"><li><a href="http://dx.doi.org/10.1093/nar/27.1.237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PhosphoBase%2C+a+database+of+phosphorylation+sites%3A+release+2.0." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PhosphoBase%2C+a+database+of+phosphorylation+sites%3A+release+2.0.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000777-Lowenberg1" id="pone.0000777-Lowenberg1"></a>Lowenberg M, Tuynman J, Bilderbeek J, Gaber T, Buttgereit F, et al.  (2005) Rapid immunosuppressive effects of glucocorticoids mediated through Lck and Fyn. Blood  106: 1703–1710.  <ul class="find" data-citedArticleID="981098" data-doi="10.1182/blood-2004-12-4790"><li><a href="http://dx.doi.org/10.1182/blood-2004-12-4790" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+immunosuppressive+effects+of+glucocorticoids+mediated+through+Lck+and+Fyn." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+immunosuppressive+effects+of+glucocorticoids+mediated+through+Lck+and+Fyn.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000777-Diks1" id="pone.0000777-Diks1"></a>Diks SH, Kok K, O'Toole T, Hommes DW, van Dijken P, et al.  (2004) Kinome profiling for studying lipopolysaccharide signal transduction in human peripheral blood mononuclear cells. J Biol Chem  279: 49206–49213.  <ul class="find" data-citedArticleID="981068" data-doi="10.1074/jbc.m405028200"><li><a href="http://dx.doi.org/10.1074/jbc.m405028200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kinome+profiling+for+studying+lipopolysaccharide+signal+transduction+in+human+peripheral+blood+mononuclear+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kinome+profiling+for+studying+lipopolysaccharide+signal+transduction+in+human+peripheral+blood+mononuclear+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000777-Varela1" id="pone.0000777-Varela1"></a>Varela FG, Maturana HR, Uribe R (1974) Autopoiesis: the organization of living systems, its characterization and a model. Curr Mod Biol  5: 187–196.  <ul class="find" data-citedArticleID="981128" data-doi="10.1016/0303-2647(74)90031-8"><li><a href="http://dx.doi.org/10.1016/0303-2647(74)90031-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Autopoiesis%3A+the+organization+of+living+systems%2C+its+characterization+and+a+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Autopoiesis%3A+the+organization+of+living+systems%2C+its+characterization+and+a+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000777-Modesti1" id="pone.0000777-Modesti1"></a>Modesti A, Bini L, Carraresi L, Magherini F, Liberatori S, Pallini V, Manao G, Pinna LA, Raugei G, Ramponi G (2001) Expression of the small tyrosine phosphatase (Stp1) in Saccharomyces cerevisiae: a study on protein tyrosine phosphorylation. Electrophoresis  22: 576–585.  <ul class="find" data-citedArticleID="981110" data-doi="10.1002/1522-2683(200102)22:3<576::aid-elps576&gt;3.0.co;2-p"><li><a href="http://dx.doi.org/10.1002/1522-2683(200102)22:3<576::aid-elps576&gt;3.0.co;2-p" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+the+small+tyrosine+phosphatase+%28Stp1%29+in+Saccharomyces+cerevisiae%3A+a+study+on+protein+tyrosine+phosphorylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+the+small+tyrosine+phosphatase+%28Stp1%29+in+Saccharomyces+cerevisiae%3A+a+study+on+protein+tyrosine+phosphorylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000777-Rudrabhatla1" id="pone.0000777-Rudrabhatla1"></a>Rudrabhatla P, Reddy MM, Rajasekharan R (2006) Genome-wide analysis and experimentation of plant serine/threonine/tyrosine-specific protein kinases. Plant Mol Biol  60: 293–319.  <ul class="find" data-citedArticleID="981116" data-doi="10.1007/s11103-005-4109-7"><li><a href="http://dx.doi.org/10.1007/s11103-005-4109-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+and+experimentation+of+plant+serine%2Fthreonine%2Ftyrosine-specific+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+and+experimentation+of+plant+serine%2Fthreonine%2Ftyrosine-specific+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000777-Besant1" id="pone.0000777-Besant1"></a>Besant PG, Tan E, Attwood PV (2003) Mammalian protein histidine kinases. Int J Biochem Cell Biol  35: 297–309.  <ul class="find" data-citedArticleID="981060" data-doi="10.1016/s1357-2725(02)00257-1"><li><a href="http://dx.doi.org/10.1016/s1357-2725(02)00257-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+protein+histidine+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+protein+histidine+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000777-Matthews1" id="pone.0000777-Matthews1"></a>Matthews HR (1995) Protein kinases and phosphatases that act on histidine, lysine, or arginine residues in eukaryotic proteins: a possible regulator of the mitogen-activated protein kinase cascade. Pharmacol Ther  67: 323–350.  <ul class="find" data-citedArticleID="981106" data-doi="10.1016/0163-7258(95)00020-8"><li><a href="http://dx.doi.org/10.1016/0163-7258(95)00020-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+kinases+and+phosphatases+that+act+on+histidine%2C+lysine%2C+or+arginine+residues+in+eukaryotic+proteins%3A+a+possible+regulator+of+the+mitogen-activated+protein+kinase+cascade." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+kinases+and+phosphatases+that+act+on+histidine%2C+lysine%2C+or+arginine+residues+in+eukaryotic+proteins%3A+a+possible+regulator+of+the+mitogen-activated+protein+kinase+cascade.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000777-Steeg1" id="pone.0000777-Steeg1"></a>Steeg PS, Palmieri D, Ouatas T, Salerno M (2003) Histidine kinases and histidine phosphorylated proteins in mammalian cell biology, signal transduction and cancer. Cancer Lett  190: 1–12.  <ul class="find" data-citedArticleID="981124" data-doi="10.1016/s0304-3835(02)00499-8"><li><a href="http://dx.doi.org/10.1016/s0304-3835(02)00499-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histidine+kinases+and+histidine+phosphorylated+proteins+in+mammalian+cell+biology%2C+signal+transduction+and+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histidine+kinases+and+histidine+phosphorylated+proteins+in+mammalian+cell+biology%2C+signal+transduction+and+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000777-Wolanin1" id="pone.0000777-Wolanin1"></a>Wolanin PM, Thomason PA, Stock JB (2002) Histidine protein kinases: key signal transducers outside the animal kingdom. Genome Biol  3: REVIEWS3013.  <ul class="find" data-citedArticleID="981134"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Histidine+protein+kinases%3A+key+signal+transducers+outside+the+animal+kingdom.&amp;auth=&amp;atitle=Histidine+protein+kinases%3A+key+signal+transducers+outside+the+animal+kingdom." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histidine+protein+kinases%3A+key+signal+transducers+outside+the+animal+kingdom." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histidine+protein+kinases%3A+key+signal+transducers+outside+the+animal+kingdom.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000777-Johnson1" id="pone.0000777-Johnson1"></a>Johnson SC (1967) Hierarchical clustering schemes. Psychometrika  32: 241–254.  <ul class="find" data-citedArticleID="981086" data-doi="10.1007/bf02289588"><li><a href="http://dx.doi.org/10.1007/bf02289588" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hierarchical+clustering+schemes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hierarchical+clustering+schemes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000777-Nichols1" id="pone.0000777-Nichols1"></a>Nichols SA, Dirks W, Pearse JS, King N (2006) Early evolution of animal cell signaling and adhesion genes. Proc Natl Acad Sci U S A  103: 12451–12456.  <ul class="find" data-citedArticleID="981112" data-doi="10.1073/pnas.0604065103"><li><a href="http://dx.doi.org/10.1073/pnas.0604065103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+evolution+of+animal+cell+signaling+and+adhesion+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+evolution+of+animal+cell+signaling+and+adhesion+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000777-Segawa1" id="pone.0000777-Segawa1"></a>Segawa Y, Suga H, Iwabe N, Oneyama C, Akagi T, Miyata T, Okada M (2006) Functional development of Src tyrosine kinases during evolution from a unicellular ancestor to multicellular animals. Proc Natl Acad Sci U S A  103: 12021–12026.  <ul class="find" data-citedArticleID="981122" data-doi="10.1073/pnas.0600021103"><li><a href="http://dx.doi.org/10.1073/pnas.0600021103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+development+of+Src+tyrosine+kinases+during+evolution+from+a+unicellular+ancestor+to+multicellular+animals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+development+of+Src+tyrosine+kinases+during+evolution+from+a+unicellular+ancestor+to+multicellular+animals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000777-Alexopoulos1" id="pone.0000777-Alexopoulos1"></a>Alexopoulos CJ, Mims CW, Blackwell M (1996) Introductory Mycology. Wiley.  <ul class="find" data-citedArticleID="981058" data-doi="10.2307/3244124"><li><a href="http://dx.doi.org/10.2307/3244124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Introductory+Mycology.+Wiley." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Introductory+Mycology.+Wiley.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000777-Hawksworth1" id="pone.0000777-Hawksworth1"></a>Hawksworth DL (1991) The Fungal Dimension of Biodiversity-Magnitude, Significance, and Conservation. Mycological Research  95: 641–655.  <ul class="find" data-citedArticleID="981078" data-doi="10.1016/s0953-7562(09)80810-1"><li><a href="http://dx.doi.org/10.1016/s0953-7562(09)80810-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Fungal+Dimension+of+Biodiversity-Magnitude%2C+Significance%2C+and+Conservation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Fungal+Dimension+of+Biodiversity-Magnitude%2C+Significance%2C+and+Conservation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000777-Brinkworth1" id="pone.0000777-Brinkworth1"></a>Brinkworth RI, Breinl RA, Kobe B (2003) Structural basis and prediction of substrate specificity in protein serine/threonine kinases. Proc Natl Acad Sci U S A  100: 74–79.  <ul class="find" data-citedArticleID="981064" data-doi="10.1073/pnas.0134224100"><li><a href="http://dx.doi.org/10.1073/pnas.0134224100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+and+prediction+of+substrate+specificity+in+protein+serine%2Fthreonine+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+and+prediction+of+substrate+specificity+in+protein+serine%2Fthreonine+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000777-Douville1" id="pone.0000777-Douville1"></a>Douville E, Duncan P, Abraham N, Bell JC (1994) Dual specificity kinases–a new family of signal transducers. Cancer Metastasis Rev  13: 1–7.  <ul class="find" data-citedArticleID="981070"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Dual+specificity+kinases%26%23x2013%3Ba+new+family+of+signal+transducers.&amp;auth=&amp;atitle=Dual+specificity+kinases%26%23x2013%3Ba+new+family+of+signal+transducers." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dual+specificity+kinases%26%23x2013%3Ba+new+family+of+signal+transducers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dual+specificity+kinases%26%23x2013%3Ba+new+family+of+signal+transducers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000777-Lindberg1" id="pone.0000777-Lindberg1"></a>Lindberg RA, Quinn AM, Hunter T (1992) Dual-specificity protein kinases: will any hydroxyl do? Trends Biochem Sci  17: 114–119.  <ul class="find" data-citedArticleID="981096" data-doi="10.1016/0968-0004(92)90248-8"><li><a href="http://dx.doi.org/10.1016/0968-0004(92)90248-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dual-specificity+protein+kinases%3A+will+any+hydroxyl+do%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dual-specificity+protein+kinases%3A+will+any+hydroxyl+do%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000777-Parker1" id="pone.0000777-Parker1"></a>Parker LL, Atherton-Fessler S, Piwnica-Worms H (1992) p107wee1 is a dual-specificity kinase that phosphorylates p34cdc2 on tyrosine 15. Proc Natl Acad Sci U S A  89: 2917–2921.  <ul class="find" data-citedArticleID="981114" data-doi="10.1073/pnas.89.7.2917"><li><a href="http://dx.doi.org/10.1073/pnas.89.7.2917" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=p107wee1+is+a+dual-specificity+kinase+that+phosphorylates+p34cdc2+on+tyrosine+15." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22p107wee1+is+a+dual-specificity+kinase+that+phosphorylates+p34cdc2+on+tyrosine+15.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000777-Kentrup1" id="pone.0000777-Kentrup1"></a>Kentrup H, Becker W, Heukelbach J, Wilmes A, Schurmann A, Huppertz C, Kainulainen H, Joost HG (1996) Dyrk, a dual specificity protein kinase with unique structural features whose activity is dependent on tyrosine residues between subdomains VII and VIII. J Biol Chem  271: 3488–3495.  <ul class="find" data-citedArticleID="981088" data-doi="10.1074/jbc.271.7.3488"><li><a href="http://dx.doi.org/10.1074/jbc.271.7.3488" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dyrk%2C+a+dual+specificity+protein+kinase+with+unique+structural+features+whose+activity+is+dependent+on+tyrosine+residues+between+subdomains+VII+and+VIII." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dyrk%2C+a+dual+specificity+protein+kinase+with+unique+structural+features+whose+activity+is+dependent+on+tyrosine+residues+between+subdomains+VII+and+VIII.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000777-Forsberg1" id="pone.0000777-Forsberg1"></a>Forsberg J, Allen JF (2001) Protein tyrosine phosphorylation in the transition to light state 2 of chloroplast thylakoids. Photosynth Res  68: 71–79.  <ul class="find" data-citedArticleID="981072"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Protein+tyrosine+phosphorylation+in+the+transition+to+light+state+2+of+chloroplast+thylakoids.&amp;auth=&amp;atitle=Protein+tyrosine+phosphorylation+in+the+transition+to+light+state+2+of+chloroplast+thylakoids." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+tyrosine+phosphorylation+in+the+transition+to+light+state+2+of+chloroplast+thylakoids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+tyrosine+phosphorylation+in+the+transition+to+light+state+2+of+chloroplast+thylakoids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000777-Rudrabhatla2" id="pone.0000777-Rudrabhatla2"></a>Rudrabhatla P, Rajasekharan R (2004) Functional characterization of peanut serine/threonine/tyrosine protein kinase: molecular docking and inhibition kinetics with tyrosine kinase inhibitors. Biochemistry  43: 12123–12132.  <ul class="find" data-citedArticleID="981118" data-doi="10.1021/bi0497042"><li><a href="http://dx.doi.org/10.1021/bi0497042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+characterization+of+peanut+serine%2Fthreonine%2Ftyrosine+protein+kinase%3A+molecular+docking+and+inhibition+kinetics+with+tyrosine+kinase+inhibitors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+characterization+of+peanut+serine%2Fthreonine%2Ftyrosine+protein+kinase%3A+molecular+docking+and+inhibition+kinetics+with+tyrosine+kinase+inhibitors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000777-Scheeff1" id="pone.0000777-Scheeff1"></a>Scheeff ED, Bourne PE (2005) Structural evolution of the protein kinase-like superfamily. PLoS Comput Biol  1: e49.  <ul class="find" data-citedArticleID="981120" data-doi="10.1371/journal.pcbi.0010049.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0010049.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+evolution+of+the+protein+kinase-like+superfamily." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+evolution+of+the+protein+kinase-like+superfamily.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.XML" value="141421"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.PDF" value="1962330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t001.PNG_L" value="238239"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t001.PNG_M" value="139768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t001.PNG_S" value="14473"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t001.TIF" value="1035574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t001.PNG_I" value="42204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.g001.PNG_L" value="1420397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.g001.PNG_M" value="252758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.g001.PNG_S" value="15648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.g001.TIF" value="1861590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.g001.PNG_I" value="143341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.g002.PNG_L" value="782604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.g002.PNG_M" value="91932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.g002.PNG_S" value="10858"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.g002.TIF" value="1699716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.g002.PNG_I" value="51456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t002.PNG_L" value="58121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t002.PNG_M" value="78545"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t002.PNG_S" value="12484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t002.TIF" value="236378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t002.PNG_I" value="25456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t003.PNG_L" value="865020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t003.PNG_M" value="182398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t003.PNG_S" value="24001"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t003.TIF" value="3573410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t003.PNG_I" value="182033"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t004.PNG_L" value="1144268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t004.PNG_M" value="210400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t004.PNG_S" value="20089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t004.TIF" value="4857892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.t004.PNG_I" value="128900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.s001.XLS" value="169984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.s002.XLS" value="114176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.s003.XLS" value="323584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.s004.XLS" value="231936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.s005.XLS" value="249344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.s006.XLS" value="319488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.s007.XLS" value="15872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.s008.XLS" value="16896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000777.s009.XLS" value="26624"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000777&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000777" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000777&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000777&volume=&issue=&title=Evidence for a Minimal Eukaryotic Phosphoproteome?&author_name=Sander%20H.%20Diks%2C%20Kaushal%20Parikh%2C%20Marijke%20van%20der%20Sijde%2C%20Jos%20Joore%2C%20Tita%20Ritsema%2C%20Maikel%20P.%20Peppelenbosch&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000777" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000777" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000777" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000777&amp;t=Evidence%20for%20a%20Minimal%20Eukaryotic%20Phosphoproteome%3F" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000777&title=Evidence%20for%20a%20Minimal%20Eukaryotic%20Phosphoproteome%3F&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000777&amp;title=Evidence%20for%20a%20Minimal%20Eukaryotic%20Phosphoproteome%3F" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000777" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000777&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Evidence for a Minimal Eukaryotic Phosphoproteome?';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000777';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Evidence%20for%20a%20Minimal%20Eukaryotic%20Phosphoproteome%3F http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000777" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000777" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+binding%22" title="Search for articles in the subject area:'Cell binding'"><div class="flagText">Cell binding</div></a>
              <div data-categoryid="19609" data-articleid="24392"
                   data-categoryname="Cell binding"
                   class="flagImage" title="Flag 'Cell binding' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="24392"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microtubule-associated+proteins%22" title="Search for articles in the subject area:'Microtubule-associated proteins'"><div class="flagText">Microtubule-associated proteins</div></a>
              <div data-categoryid="17617" data-articleid="24392"
                   data-categoryname="Microtubule-associated proteins"
                   class="flagImage" title="Flag 'Microtubule-associated proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phosphorylation%22" title="Search for articles in the subject area:'Phosphorylation'"><div class="flagText">Phosphorylation</div></a>
              <div data-categoryid="42723" data-articleid="24392"
                   data-categoryname="Phosphorylation"
                   class="flagImage" title="Flag 'Phosphorylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+kinase+signaling+cascade%22" title="Search for articles in the subject area:'Protein kinase signaling cascade'"><div class="flagText">Protein kinase signaling cascade</div></a>
              <div data-categoryid="21139" data-articleid="24392"
                   data-categoryname="Protein kinase signaling cascade"
                   class="flagImage" title="Flag 'Protein kinase signaling cascade' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22STAT+proteins%22" title="Search for articles in the subject area:'STAT proteins'"><div class="flagText">STAT proteins</div></a>
              <div data-categoryid="23235" data-articleid="24392"
                   data-categoryname="STAT proteins"
                   class="flagImage" title="Flag 'STAT proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transmembrane+receptors%22" title="Search for articles in the subject area:'Transmembrane receptors'"><div class="flagText">Transmembrane receptors</div></a>
              <div data-categoryid="20299" data-articleid="24392"
                   data-categoryname="Transmembrane receptors"
                   class="flagImage" title="Flag 'Transmembrane receptors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Tyrosine%22" title="Search for articles in the subject area:'Tyrosine'"><div class="flagText">Tyrosine</div></a>
              <div data-categoryid="40505" data-articleid="24392"
                   data-categoryname="Tyrosine"
                   class="flagImage" title="Flag 'Tyrosine' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1523'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3296'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6230&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=20255">RLKs are part of the TKL group</a><br>Posted by Gerard_Manning</p>
      <p><a href="/annotation/listThread.action?root=287">Not quite true</a><br>Posted by Gerard_Manning</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
