

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Automated Builder and Database of Protein/Membrane Complexes for Molecular Dynamics Simulations</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000880"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000880" />

    <meta name="citation_title" content="Automated Builder and Database of Protein/Membrane Complexes for Molecular Dynamics Simulations"/>
    <meta itemprop="name" content="Automated Builder and Database of Protein/Membrane Complexes for Molecular Dynamics Simulations"/>

      <meta name="citation_author" content="Sunhwan Jo"/>
            <meta name="citation_author_institution" content="Department of Chemistry, The University of Kansas, Lawrence, Kansas, United States of America"/>
      <meta name="citation_author" content="Taehoon Kim"/>
            <meta name="citation_author_institution" content="Department of Molecular Biosciences, Center for Bioinformatics, The University of Kansas, Lawrence, Kansas, United States of America"/>
      <meta name="citation_author" content="Wonpil Im"/>
            <meta name="citation_author_institution" content="Department of Molecular Biosciences, Center for Bioinformatics, The University of Kansas, Lawrence, Kansas, United States of America"/>

    <meta name="citation_date" content="2007/9/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000880.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e880"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Signaling networks: the origins of cellular multitasking.; citation_author=JD Jordan; citation_author=EM Landau; citation_author=R Iyengar; citation_journal_title=Cell; citation_volume=103; citation_number=1; citation_pages=193-200; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Signaling–2000 and beyond.; citation_author=T Hunter; citation_journal_title=Cell; citation_volume=100; citation_number=2; citation_pages=113-127; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Structure of a glycerol-conducting channel and the basis for its selectivity.; citation_author=D Fu; citation_author=A Libson; citation_author=LJ Miercke; citation_author=C Weitzman; citation_author=P Nollert; citation_journal_title=Science; citation_volume=290; citation_number=3; citation_pages=481-486; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mechanism of ammonia transport by Amt/MEP/Rh: structure of AmtB at 1.35 A.; citation_author=S Khademi; citation_author=J O'Connell; citation_author=J Remis; citation_author=Y Robles-Colmenares; citation_author=LJ Miercke; citation_journal_title=Science; citation_volume=305; citation_number=4; citation_pages=1587-1594; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The voltage-gated potassium channels and their relatives.; citation_author=G Yellen; citation_journal_title=Nature; citation_volume=419; citation_number=5; citation_pages=35-42; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Crystal structure and mechanism of a calcium-gated potassium channel.; citation_author=Y Jiang; citation_author=A Lee; citation_author=J Chen; citation_author=M Cadene; citation_author=BT Chait; citation_journal_title=Nature; citation_volume=417; citation_number=6; citation_pages=515-522; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structural determinants of water permeation through aquaporin-1.; citation_author=K Murata; citation_author=K Mitsuoka; citation_author=T Hirai; citation_author=T Walz; citation_author=P Agre; citation_journal_title=Nature; citation_volume=407; citation_number=7; citation_pages=599-605; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Structural basis of energy transduction in the transport cycle of MsbA.; citation_author=J Dong; citation_author=G Yang; citation_author=HS McHaourab; citation_journal_title=Science; citation_volume=308; citation_number=8; citation_pages=1023-1028; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Energy transduction in ATP synthase.; citation_author=T Elston; citation_author=H Wang; citation_author=G Oster; citation_journal_title=Nature; citation_volume=391; citation_number=9; citation_pages=510-513; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Molecular biology of the cell.; citation_author=B Alberts; citation_number=10; citation_pages=xxxiv, 1463, [1486]; citation_date=2002; citation_publisher=Garland Science; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of integral membrane proteins from eubacterial, archaean, and eukaryotic organisms.; citation_author=E Wallin; citation_author=G von Heijne; citation_journal_title=Protein Sci; citation_volume=7; citation_number=11; citation_pages=1029-1038; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=In silico research in drug discovery.; citation_author=GC Terstappen; citation_author=A Reggiani; citation_journal_title=Trends Pharmacol Sci; citation_volume=22; citation_number=12; citation_pages=23-26; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=White S Membrane Proteins of Known 3D Structure.; citation_number=13; " />
      <meta name="citation_reference" content="citation_title=The Protein Data Bank.; citation_author=HM Berman; citation_author=T Battistuz; citation_author=TN Bhat; citation_author=WF Bluhm; citation_author=PE Bourne; citation_journal_title=Acta Crystallogr D Biol Crystallogr; citation_volume=58; citation_number=14; citation_pages=899-907; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Water permeation across biological membranes: mechanism and dynamics of aquaporin-1 and GlpF.; citation_author=BL de Groot; citation_author=H Grubmuller; citation_journal_title=Science; citation_volume=294; citation_number=15; citation_pages=2353-2357; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Molecular basis of proton blockage in aquaporins.; citation_author=N Chakrabarti; citation_author=E Tajkhorshid; citation_author=B Roux; citation_author=R Pomes; citation_journal_title=Structure; citation_volume=12; citation_number=16; citation_pages=65-74; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Computational studies of membrane channels.; citation_author=B Roux; citation_author=K Schulten; citation_journal_title=Structure; citation_volume=12; citation_number=17; citation_pages=1343-1351; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Control of ion selectivity in potassium channels by electrostatic and dynamic properties of carbonyl ligands.; citation_author=SY Noskov; citation_author=S Berneche; citation_author=B Roux; citation_journal_title=Nature; citation_volume=431; citation_number=18; citation_pages=830-834; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Modulation of glycophorin A transmembrane helix interactions by lipid bilayers: molecular dynamics calculations.; citation_author=HI Petrache; citation_author=A Grossfield; citation_author=KR MacKenzie; citation_author=DM Engelman; citation_author=TB Woolf; citation_journal_title=J Mol Biol; citation_volume=302; citation_number=19; citation_pages=727-746; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular dynamics simulation of the gramicidin channel in a phospholipid bilayer.; citation_author=TB Woolf; citation_author=B Roux; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=20; citation_pages=11631-11635; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Structure, energetics, and dynamics of lipid-protein interactions: A molecular dynamics study of the gramicidin A channel in a DMPC bilayer.; citation_author=TB Woolf; citation_author=B Roux; citation_journal_title=Proteins; citation_volume=24; citation_number=21; citation_pages=92-114; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Transmembrane helix structure, dynamics, and interactions: multi-nanosecond molecular dynamics simulations.; citation_author=L Shen; citation_author=D Bassolino; citation_author=T Stouch; citation_journal_title=Biophys J; citation_volume=73; citation_number=22; citation_pages=3-20; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A molecular dynamics study of the pores formed by Escherichia coli OmpF porin in a fully hydrated palmitoyloleoylphosphatidylcholine bilayer.; citation_author=DP Tieleman; citation_author=HJ Berendsen; citation_journal_title=Biophys J; citation_volume=74; citation_number=23; citation_pages=2786-2801; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Setting up and optimization of membrane protein simulations.; citation_author=JD Faraldo-Gomez; citation_author=GR Smith; citation_author=MS Sansom; citation_journal_title=Eur Biophys J; citation_volume=31; citation_number=24; citation_pages=217-227; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=OPM: orientations of proteins in membranes database.; citation_author=MA Lomize; citation_author=AL Lomize; citation_author=ID Pogozheva; citation_author=HI Mosberg; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=25; citation_pages=623-625; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structure of lipid bilayers.; citation_author=JF Nagle; citation_author=S Tristram-Nagle; citation_journal_title=Biochim Biophys Acta; citation_volume=1469; citation_number=26; citation_pages=159-195; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Ions and counterions in a biological channel: a molecular dynamics simulation of OmpF porin from Escherichia coli in an explicit membrane with 1 M KCl aqueous salt solution.; citation_author=W Im; citation_author=B Roux; citation_journal_title=J Mol Biol; citation_volume=319; citation_number=27; citation_pages=1177-1197; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=CHARMM: a program for macromolecular energy, minimization, and dynamics calculations.; citation_author=BR Brooks; citation_author=RE Bruccoleri; citation_author=BD Olafson; citation_author=DJ States; citation_author=S Swaminathan; citation_journal_title=J Comp Chem; citation_volume=4; citation_number=28; citation_pages=187-217; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=All-atom empirical potential for molecular modeling and dynamics studies of proteins.; citation_author=AD MacKerell; citation_author=D Bashford; citation_author=M Bellott; citation_author=RL Dunbrack; citation_author=JD Evanseck; citation_journal_title=Journal of Physical Chemistry B; citation_volume=102; citation_number=29; citation_pages=3586-3616; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Extending the treatment of backbone energetics in protein force fields: limitations of gas-phase quantum mechanics in reproducing protein conformational distributions in molecular dynamics simulations.; citation_author=AD Mackerell; citation_author=M Feig; citation_author=CL Brooks; citation_journal_title=J Comput Chem; citation_volume=25; citation_number=30; citation_pages=1400-1415; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Comparison of Simple Potential Functions for Simulating Liquid Water.; citation_author=WL Jorgensen; citation_author=J Chandrasekhar; citation_author=JD Madura; citation_author=RW Impey; citation_author=ML Klein; citation_journal_title=Journal of Chemical Physics; citation_volume=79; citation_number=31; citation_pages=926-935; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=An ab initio study on the torsional surface of alkanes and its effect on molecular simulations of alkanes and a DPPC bilayer.; citation_author=JB Klauda; citation_author=BR Brooks; citation_author=AD MacKerell; citation_author=RM Venable; citation_author=RW Pastor; citation_journal_title=Journal of Physical Chemistry B; citation_volume=109; citation_number=32; citation_pages=5300-5311; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Computer-Simulation of Liquid/Liquid Interfaces .2. Surface-Tension Area Dependence of a Bilayer and Monolayer.; citation_author=SE Feller; citation_author=YH Zhang; citation_author=RW Pastor; citation_journal_title=Journal of Chemical Physics; citation_volume=103; citation_number=33; citation_pages=10267-10276; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Simulations of membranes and other interfacial systems using P2(1) and Pc periodic boundary conditions.; citation_author=EA Dolan; citation_author=RM Venable; citation_author=RW Pastor; citation_author=BR Brooks; citation_journal_title=Biophys J; citation_volume=82; citation_number=34; citation_pages=2317-2325; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=GROMACS: fast, flexible, and free.; citation_author=D Van Der Spoel; citation_author=E Lindahl; citation_author=B Hess; citation_author=G Groenhof; citation_author=AE Mark; citation_journal_title=J Comput Chem; citation_volume=26; citation_number=35; citation_pages=1701-1718; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Scalable molecular dynamics with NAMD.; citation_author=JC Phillips; citation_author=R Braun; citation_author=W Wang; citation_author=J Gumbart; citation_author=E Tajkhorshid; citation_journal_title=J Comput Chem; citation_volume=26; citation_number=36; citation_pages=1781-1802; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000880" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Automated Builder and Database of Protein/Membrane Complexes for Molecular Dynamics Simulations"/>
    <meta name="twitter:description" content="Molecular dynamics simulations of membrane proteins have provided deeper insights into their functions and interactions with surrounding environments at the atomic level. However, compared to solvation of globular proteins, building a realistic protein/membrane complex is still challenging and requires considerable experience with simulation software. Membrane Builder in the CHARMM-GUI website (http://www.charmm-gui.org) helps users to build such a complex system using a web browser with a graphical user interface. Through a generalized and automated building process including system size determination as well as generation of lipid bilayer, pore water, bulk water, and ions, a realistic membrane system with virtually any kinds and shapes of membrane proteins can be generated in 5 minutes to 2 hours depending on the system size. Default values that were elaborated and tested extensively are given in each step to provide reasonable options and starting points for both non-expert and expert users. The efficacy of Membrane Builder is illustrated by its applications to 12 transmembrane and 3 interfacial membrane proteins, whose fully equilibrated systems with three different types of lipid molecules (DMPC, DPPC, and POPC) and two types of system shapes (rectangular and hexagonal) are freely available on the CHARMM-GUI website. One of the most significant advantages of using the web environment is that, if a problem is found, users can go back and re-generate the whole system again before quitting the browser. Therefore, Membrane Builder provides the intuitive and easy way to build and simulate the biologically important membrane system."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000880.g007"/>

  <meta property="og:title" content="Automated Builder and Database of Protein/Membrane Complexes for Molecular Dynamics Simulations" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000880" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4031'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8196'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1378&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000880">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000880" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000880&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Automated Builder and Database of Protein/Membrane Complexes for Molecular Dynamics Simulations
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sunhwan Jo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Chemistry, The University of Kansas, Lawrence, Kansas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Taehoon Kim, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Biosciences, Center for Bioinformatics, The University of Kansas, Lawrence, Kansas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wonpil Im
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:wonpil@ku.edu">wonpil@ku.edu</a></p>

                <p>Affiliation:
                  Department of Molecular Biosciences, Center for Bioinformatics, The University of Kansas, Lawrence, Kansas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 12, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000880</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000880" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000880" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000880" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000880" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000880-g001" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000880-g002" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000880-g003" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000880-g004" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000880-g005" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000880-t001" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000880-g006" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000880-g007" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000880-t002" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000880">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000880" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Molecular dynamics simulations of membrane proteins have provided deeper insights into their functions and interactions with surrounding environments at the atomic level. However, compared to solvation of globular proteins, building a realistic protein/membrane complex is still challenging and requires considerable experience with simulation software. Membrane Builder in the CHARMM-GUI website (<a href="http://www.charmm-gui.org">http://www.charmm-gui.org</a>) helps users to build such a complex system using a web browser with a graphical user interface. Through a generalized and automated building process including system size determination as well as generation of lipid bilayer, pore water, bulk water, and ions, a realistic membrane system with virtually any kinds and shapes of membrane proteins can be generated in 5 minutes to 2 hours depending on the system size. Default values that were elaborated and tested extensively are given in each step to provide reasonable options and starting points for both non-expert and expert users. The efficacy of Membrane Builder is illustrated by its applications to 12 transmembrane and 3 interfacial membrane proteins, whose fully equilibrated systems with three different types of lipid molecules (DMPC, DPPC, and POPC) and two types of system shapes (rectangular and hexagonal) are freely available on the CHARMM-GUI website. One of the most significant advantages of using the web environment is that, if a problem is found, users can go back and re-generate the whole system again before quitting the browser. Therefore, Membrane Builder provides the intuitive and easy way to build and simulate the biologically important membrane system.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Jo S, Kim T, Im W (2007) Automated Builder and Database of Protein/Membrane Complexes for Molecular Dynamics Simulations. PLoS ONE 2(9):
          e880.
            doi:10.1371/journal.pone.0000880</p><p><strong>Academic Editor: </strong>Adam Yuan, Temasek Life Sciences Laboratory, Singapore</p><p><strong>Received:</strong> July 30, 2007; <strong>Accepted:</strong> August 22, 2007; <strong>Published:</strong> September 12, 2007</p><p><strong>Copyright:</strong> © 2007 Jo et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by institutional funding from The University of Kansas.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Membrane proteins play important roles in many cellular processes, such as cell signaling <a href="#pone.0000880-Jordan1">[1]</a>, <a href="#pone.0000880-Hunter1">[2]</a>, translocation of ions and small molecules <a href="#pone.0000880-Fu1">[3]</a>–<a href="#pone.0000880-Murata1">[7]</a>, energy transduction process <a href="#pone.0000880-Dong1">[8]</a>, <a href="#pone.0000880-Elston1">[9]</a>, and cell-cell recognition <a href="#pone.0000880-Alberts1">[10]</a>. They are quantitatively significant as well. For example, 20–30% of the protein-encoding regions of known genomes encode membrane proteins <a href="#pone.0000880-Wallin1">[11]</a>, suggesting that the human genome encodes about 8,000 membrane proteins. Furthermore, about 50% of these membrane proteins are considered putative drug targets <a href="#pone.0000880-Terstappen1">[12]</a>. Due to the biological and pharmaceutical importance of membrane proteins, considerable experimental efforts have been made to better understand their structure and function in membrane environments. As a result of technological advances in various fields, more and more membrane protein structures have become available <a href="#pone.0000880-1">[13]</a>, although their numbers are still much smaller than those of globular proteins <a href="#pone.0000880-Berman1">[14]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Molecular dynamics (MD) simulations of membrane proteins have provided deeper insights into their functions and interactions with surrounding environments at the atomic level <a href="#pone.0000880-deGroot1">[15]</a>–<a href="#pone.0000880-Noskov1">[18]</a>. In particular, when combined with computationally sophisticated and intensive free energy techniques, MD simulations can elucidate the microscopic origins or driving forces of biological functions. Compared to solvation of globular proteins, however, building a realistic protein/membrane complex is still challenging and requires considerable experience with simulation software. The main difficulty in building such a complex system arises from how to insert a protein into a lipid bilayer.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>There are two popular methods commonly used to build a realistic protein/membrane complex. In the first method, lipid-like pseudo atoms are first distributed around a protein and then replaced by lipid molecules one at a time <a href="#pone.0000880-Petrache1">[19]</a>–<a href="#pone.0000880-Woolf2">[21]</a>. Individual lipid molecules are randomly selected from a lipid library that contains various conformations of lipid molecules. This method allows one to easily control the system size and the number of lipid molecules while it generates a lipid bilayer nicely packed around the protein. In the second method, a hole is first created in a pre-equilibrated lipid bilayer and then the membrane protein is inserted into the hole <a href="#pone.0000880-Shen1">[22]</a>–<a href="#pone.0000880-FaraldoGomez1">[24]</a>. In general, weak repulsive radial forces perpendicular to the membrane normal are applied to a lipid bilayer until the hole is large enough to accommodate the protein. This method provides a well-equilibrated lipid bilayer, and one might expect less equilibration time than in the first method. For the sake of convenience, the first method is called the “replacement method” and the second method is called the “insertion method” hereinafter. Although both methods are well explained in the literature <a href="#pone.0000880-Petrache1">[19]</a>–<a href="#pone.0000880-FaraldoGomez1">[24]</a>, considerable efforts and experiences with MD simulation software are required to build a realistic protein/membrane system.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The CHARMM-GUI website (<a href="http://www.charmm-gui.org">http://www.charmm-gui.org</a>) provides a graphical user interface (GUI) of a collection of modules that helps users to setup their MD simulations in a web browser. Membrane Builder, one of the modules in CHARMM-GUI, aims to help users to build a sophisticated protein/membrane system easily and interactively in a web browser through a generalized and automated building process including system size determination as well as generation of lipid bilayer, pore water, bulk water, and ions. The main idea is to let users define various parameters, such as the system size and shape, ion concentration, lipid type, and bilayer generation method in a full GUI fashion, and to provide the generated complex structure and input files. However, default values that were elaborated and tested extensively are given in each step to provide reasonable options and starting points for both non-expert and expert users. We have further developed and generalized both replacement and insertion methods to adopt them in Membrane Builder to build a membrane bilayer efficiently.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In this paper, we describe and illustrate the generalized protein/membrane complex building process. Its efficiency has been examined by building a total of 90 protein/membrane complexes of 12 transmembrane and 3 interfacial membrane proteins, that have considerably different shapes and sizes, with three types of lipid bilayers (dimyristoylphosphatidylcholine (DMPC), dipalmitoylphosphatidylcholine (DPPC), and palmitoyloleoylphosphatidylcholine (POPC)) and two different system shapes (rectangular and hexagonal). Each equilibrated system is available at the archive on the CHARMM-GUI website.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Generalized Protein/membrane Complex Building Process</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>As shown in <a href="#pone-0000880-g001">Fig. 1</a>, the overall process to build a protein/membrane complex has been generalized and automated in six subsequent steps. Each step was designed to incorporate users' specific options through a web browser, and generate and run CHARMM input files. Individual input and output files including generated structures as well as an archive of all the created files are available in each step. One can visualize the generated system in each step so that, if necessary, one can go back to the previous step and modify the options interactively. Video demonstrations on how to use Membrane Builder (“Membrane Builder Lecture” and “Membrane Builder Demo”) are available on the CHARMM-GUI website.</p>
<div class="figure" id="pone-0000880-g001"><div class="img"><a name="pone-0000880-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Overview of generalized process of building protein/membrane complex system.</span></strong></p><span>doi:10.1371/journal.pone.0000880.g001</span></div><h5>Step 1–PDB Reading.</h5><a id="article1.body1.sec2.sec1.sec1.p1" name="article1.body1.sec2.sec1.sec1.p1"></a><p>The objective of this step is to read a protein structure, manipulate it if necessary, and generate a structure topology file. Users can upload their own pre-oriented protein structure, or specify PDB entry ID and a database, either of PDB database <a href="#pone.0000880-Berman1">[14]</a> or OPM database <a href="#pone.0000880-Lomize1">[25]</a>. Protein structures from OPM database are pre-oriented with respect to the membrane normal (the <em>Z</em>-axis by definition), so that one may use them without any modification of the protein orientation (see Step 2 below). In this step, users can select a model structure (in the case of NMR ensemble structures) or a part of residues in each chain. After the model and chains are selected, users can rename or remove engineered residues, choose terminal groups for each chain, protonate and/or phosphorylate residues, and make disulfide bond between two residues.</p>

<h5>Step 2–Orient Protein.</h5><a id="article1.body1.sec2.sec1.sec2.p1" name="article1.body1.sec2.sec1.sec2.p1"></a><p>In general, a membrane protein structure from PDB database <a href="#pone.0000880-Berman1">[14]</a> does not have proper information on relative disposition of the protein in a membrane bilayer. In Membrane Builder, one can place the protein appropriately in a lipid bilayer by aligning its principal axis or a vector between two specific C<sub>α</sub> atoms along the <em>Z</em>-axis, by translating the protein along the <em>Z</em>-axis, and by rotating it around the <em>X</em>-axis. It is assumed that the membrane normal is parallel to the <em>Z</em>-axis and its center is located at <em>Z = </em>0 Å. Alignment with the principal axis is useful for the proteins that have a symmetric shape. The usage of a vector defined by specific two residues is useful for irregular-shaped proteins if one can identify two residues that are parallel to the <em>Z</em>-axis. In the case that the protein structure is not easily aligned with the available options, one should align it in a local machine and then upload it. One can skip the alignment step for the protein structures from OPM database <a href="#pone.0000880-Lomize1">[25]</a> or pre-oriented structures.</p>
<a id="article1.body1.sec2.sec1.sec2.p2" name="article1.body1.sec2.sec1.sec2.p2"></a><p>After alignment, Membrane Builder generates pore water, if specified, and calculates the protein cross-sectional area along the <em>Z</em>-axis. We have developed a new method to generate water molecules inside pores with any kind of shapes and numbers (see Pore Water Generation for details). The cross-sectional area profile is used to determine protein areas that correspond to the maximum values at 10 Å&lt;<em>Z</em>&lt;20 Å and −20 Å&lt;<em>Z</em>&lt;−10 Å, respectively. <a href="#pone-0000880-g002">Figure 2</a> shows the calculated profile for PDB:2HAC (ζ-ζ transmembrane dimer). The protein areas in the top and bottom lipid leaflets are used to determine the system size in the next step. The cross-sectional area of pore water, if generated, is incorporated into the protein area so that the system size in the <em>XY</em> plane is determined as the sum of the protein area and the lipid area (see below).</p>
<div class="figure" id="pone-0000880-g002"><div class="img"><a name="pone-0000880-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Cross-sectional area profile of PDB:2HAC along the <em>Z</em>-axis.</span></strong></p><span>doi:10.1371/journal.pone.0000880.g002</span></div>
<h5>Step 3–Determine System Size.</h5><a id="article1.body1.sec2.sec1.sec3.p1" name="article1.body1.sec2.sec1.sec3.p1"></a><p>There are currently three types of lipid molecules available to build homogenous lipid bilayers, such as DMPC (experimental cross-sectional area of 60.7 Å<sup>2</sup> at 303 K), DPPC (64.0 Å<sup>2</sup> at 323 K), and POPC (68.3 Å<sup>2</sup> at 303 K) <a href="#pone.0000880-Nagle1">[26]</a>. By using the protein areas determined in the previous step and the experimental cross-sectional areas of lipid molecules, one can estimate the number of lipid molecules to build a system with specific dimension and shape (rectangular or hexagonal) in the <em>XY</em> plane, or vice versa. Membrane Builder facilitates this process by determining the system size based on users' inputs for (1) number of lipid layers, (2) number of lipid molecules, or (3) system dimensions. The number of lipid layers corresponds to the number of lipid molecules from a protein to the closest system edge so that this option can be used to quickly estimate the total number of lipid molecules. For example, “1.5 lipid layers” means, at least, three lipid molecules will be placed between two proteins in the primary and image systems. The system size along the <em>Z</em>-axis is determined by specifying the thickness of bulk water from the protein extent along the <em>Z-</em>axis, as shown in <a href="#pone-0000880-g003">Fig. 3</a>. In the case of membrane interfacial proteins whose extents do not cover the lipid bilayer, a minimum protein extent is set to <em>Z</em> = 20 Å and <em>Z</em> = −20 Å.</p>
<div class="figure" id="pone-0000880-g003"><div class="img"><a name="pone-0000880-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Thickness of bulk water from the protein extent (PDB:2HAC) along the <em>Z</em>-axis.</span></strong></p><span>doi:10.1371/journal.pone.0000880.g003</span></div><a id="article1.body1.sec2.sec1.sec3.p2" name="article1.body1.sec2.sec1.sec3.p2"></a><p>After the system size determination is completed, Membrane Builder provides a detailed summary of determined system size and a possible lipid-packing image to help users to check if the system size is proper. <a href="#pone-0000880-g004">Figure 4</a> shows a packing image of PDB:2HAC, which corresponds to a configuration generated by MD simulations of lipid-like pseudo atoms whose cross-sectional area is similar to the lipid molecule. The image systems are also displayed to provide a better idea about the system size. If the system is smaller or larger than expected, users can go back and specify the numbers of lipid molecules (option 2) or system dimensions (option 3) to enlarge or reduce the system size.</p>
<div class="figure" id="pone-0000880-g004"><div class="img"><a name="pone-0000880-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Packing image with lipid-like pseudo atoms around PDB:2HAC.</span></strong></p><a id="article1.body1.sec2.sec1.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec1.sec3.fig2.caption1.p1"></a><p>An option of 1.5 lipid layer was used to estimate the number of lipid molecules. Both primary (blue region) and image systems (yellow region) in <em>XY</em> plane are visualized to provide a general idea about the lipid packing in the protein/membrane complex.</p>
<span>doi:10.1371/journal.pone.0000880.g004</span></div>
<h5>Step 4–Build Components.</h5><a id="article1.body1.sec2.sec1.sec4.p1" name="article1.body1.sec2.sec1.sec4.p1"></a><p>Based on the system size information, Membrane Builder generates individual components to fully solvate the protein, such as a lipid bilayer, bulk water, and counter ions. If pore water molecules were generated in Step 2, users can refine them in this step to make sure that there is no water molecule left outside of the protein in the membrane hydrophobic core region (see Pore Water Generation for details). If the ion concentration (<em>C</em>) is specified, the numbers of KCl ions (<em>N</em><sub>K</sub> and <em>N</em><sub>Cl</sub>) are determined by the ion-accessible volume (<em>V</em>) and the total charge of the system (<em>Q</em><sub>sys</sub>), i.e., <em>N</em><sub>K</sub> = <em>C</em>·<em>V</em>−<em>Q</em><sub>sys</sub>/2 and <em>N</em><sub>Cl</sub> = <em>C</em>·<em>V</em>+<em>Q</em><sub>sys</sub>/2, to neutralize the system charge. The initial configuration of ions is then determined through Monte Carlo simulations using a primitive model, i.e., scaled Coulombic and van der Waals interactions. Note that only monovalent K and Cl ions are currently available, but more ion types can be incorporated easily upon users' request. It is this step that users select the lipid bilayer generation method, i.e., the insertion method or the replacement method (see below for details).</p>

<h5>Step 5, 6–Assemble Components and equilibrate the system.</h5><a id="article1.body1.sec2.sec1.sec5.p1" name="article1.body1.sec2.sec1.sec5.p1"></a><p>Each component that has been generated in the previous step will be assembled in this step as shown in <a href="#pone-0000880-g005">Fig. 5</a>. One of the most significant advantages of using the web environment is that, if a problem is found, users can go back and re-generate the whole system again before quitting the browser. Therefore, the visualization of the initially assembled structure is important to verify if the system is reasonable.</p>
<div class="figure" id="pone-0000880-g005"><div class="img"><a name="pone-0000880-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Assembled PDB:2HAC/DMPC structures with water thickness of 10 Å (see <a href="#pone-0000880-g003">Figure 3</a>) and 0.15 M of KCl.</span></strong></p><a id="article1.body1.sec2.sec1.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec1.sec5.fig1.caption1.p1"></a><p>(A) top and (B) side views. Green and magenta spheres represent chloride and potassium ions, respectively.</p>
<span>doi:10.1371/journal.pone.0000880.g005</span></div><a id="article1.body1.sec2.sec1.sec5.p2" name="article1.body1.sec2.sec1.sec5.p2"></a><p>After the protein/membrane complex is assembled, the equilibration must be performed to relax the uncorrelated initial system before MD production simulations. However, due to the significant amount of computing power that cannot be hosted in our server, Membrane Builder provides six consecutive CHARMM input files for equilibration, which can be modified for continual production simulations. As shown in <a href="#pone-0000880-t001">Table 1</a>, to assure gradual equilibration of the initially assembled system, various restraints are applied to the protein, water, ions, and lipid molecules during the equilibration <a href="#pone.0000880-Im1">[27]</a>: (1) harmonic restraints to ions and heavy atoms of the protein, (2) repulsive planar restraints to prevent water from entering into the membrane hydrophobic region, and (3) planar restraints to hold the position of head groups of membranes along the <em>Z</em>-axis. These restraint forces are slowly reduced as the equilibration progresses. To warrant the successful equilibration, i.e., to avoid instability of dynamics integrations during equilibration, the NVT dynamics (constant volume and temperature) is used for the first and second steps, and the NPAT (constant pressure, area, and temperature) dynamics for the rest at 303.15 K (DMPC and POPC) and 323.15 K (DPPC).</p>
<div class="figure" id="pone-0000880-t001"><div class="img"><a name="pone-0000880-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000880.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000880.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Detailed information on each equilibration step</span></strong></p><span>doi:10.1371/journal.pone.0000880.t001</span></div>


<h4>Lipid Bilayer Building Methods: Insertion and Replacement</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To generalize and automate the lipid bilayer building method, we have extended and further refined both insertion and replacement methods. <a href="#pone-0000880-g006">Figure 6</a> shows a clear distinction between insertion and replacement methods. The insertion method creates a hole in a bilayer by applying weak repulsive radial forces during the simulation of the lipid bilayer, and insert a protein into the hole, as shown in <a href="#pone-0000880-g006">Figs. 6A and 6B</a>. By utilizing a library that has various sizes of holes in pre-equilibrated lipid bilayers, the insertion method can generate a lipid bilayer less than a minute. Currently, a library for each lipid molecule (DMPC, DPPC, and POPC) is composed of 90 pre-equilibrated lipid bilayers with a hole of radius from 1 Å to 45 Å for two different number of lipid molecules (128 or 256 lipid molecules) (see <a href="#s4">Materials and Methods</a> for details). The size of the largest lipid bilayer is 90 Å by 90 Å in the <em>XY</em> plane with 256 lipid molecules (128 for each leaflet). Therefore, if a system dimension is larger than 90 Å by 90 Å in <em>XY</em>, or the number of lipid molecules in one leaflet is more than 128, Membrane Builder will ask users to select the replacement method. By assuming that the protein has a cylindrical shape, the hole sizes are first estimated from the protein areas in the top and bottom lipid leaflets. Then, the top and bottom lipid leaflets with the estimated hole sizes are selected from the library and the numbers of lipid molecules are reduced to their target numbers determined in Step 3. Therefore, the insertion method is useful for a protein that has a regular and cylindrical shape. Otherwise, the replacement method would be a better choice.</p>
<div class="figure" id="pone-0000880-g006"><div class="img"><a name="pone-0000880-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>DMPC lipid bilayers generated by (A and B) the insertion method and (C and D) the replacement method for PDB:2HAC.</span></strong></p><span>doi:10.1371/journal.pone.0000880.g006</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The replacement method distributes lipid-like pseudo atoms around the protein, and substituting each atom with a lipid molecule that is randomly selected from a lipid molecular library. In Membrane Builder, the library is composed of 2,000 different conformations that were gathered from MD simulations of pure lipid bilayers (see <a href="#s4">Materials and Methods</a> for details). <a href="#pone-0000880-g006">Figure 6D</a> shows a DMPC lipid bilayer generated by the replacement method for PDB:2HAC. Unlike the insertion method, lipid molecules wrap around the protein such that the replacement method can be used for virtually any kinds of membrane proteins, including transmembrane proteins, monotopic proteins, and interfacial peptides. Furthermore, this method should be able to generate a mixed lipid bilayer with slight modification in the lipid substitution process.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>There are limitations and advantages in each method. The insertion method can build a lipid bilayer quickly, and the generated lipid bilayer is pre-equilibrated. However, the insertion method has a couple of limitations related to the system size and the protein shape. On the other hand, the replacement method can generate a lipid bilayer for virtually any kinds of shape and size of membrane proteins. However, it takes more time to generate a lipid bilayer, e.g., 5 to 50 minutes depending on the system size, and the generated bilayer is not pre-equilibrated. The library of lipid bilayers with holes and the library of lipid molecules for each lipid type are freely available at the archive on the CHARMM-GUI website.</p>


<h4>Pore Water Generation</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Ion channels, transporters, and porins have pores that are large enough to accommodate water molecules. A traditional way of generating the so-called pore water is to build a cylindrical water box, place the box inside the pore, and remove water molecules that overlap with the protein. It is easy to implement and works well for some cases. During the tests of Membrane Builder, however, we have encountered many cases that the traditional method cannot be applied directly. For example, a closed form of mechanosensitive channel (PDB:2OAR) has a pore opened only half of the transmembrane region, and the cross-sectional area profile varies markedly. If the pore were filled by the traditional method, large number of water molecules would be left in the protein exterior near the closed region of the protein. Another example is bacterial porins (e.g., PDB:1PHO) with three pores, so that it is not conceivable to fill the pores automatically with the traditional method.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In order to make the generation process of pore water more general and persuasive, we have developed a new method based on a simple idea that water molecules inside the pore would remain while those outside of the protein would evaporate during a high temperature dynamics. As shown in <a href="#pone-0000880-t001">Table 1</a>, water molecules can access to the pore vestibules in the lipid head group regions, but not into the hydrophobic region due to the restraints for water during equilibration. Therefore, Membrane Builder only solvates the transmembrane region, as shown in <a href="#pone-0000880-g007">Fig. 7A</a>, and runs molecular dynamics at a high temperature (5,000 K) with protein fixed and planar restraint potentials on the top and bottom of the water box to keep pore water molecules in the transmembrane region. Most water molecules that are outside of the protein are quickly removed after a couple of dynamics cycles, as shown in <a href="#pone-0000880-g007">Fig. 7B</a>. However, it is conceivable that a few water molecules may stay near the protein exterior if they interact with protein strongly. To resolve this problem, we have introduced the refinement step of pore water generation in Step 4 to remove those water molecules interactively by selecting residue numbers of water molecules after users' visual inspection. Users can verify this step by visualizing the pore region again after removing selected water molecules.</p>
<div class="figure" id="pone-0000880-g007"><div class="img"><a name="pone-0000880-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000880.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Pore water generation.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) The transmembrane region of the protein is solvated with a waterbox. (B) Only pore water molecules had remained after pore water generation through high-temperature dynamics.</p>
<span>doi:10.1371/journal.pone.0000880.g007</span></div>

<h4>Test cases</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>In order to illustrate and verify the efficiency and applicability of the web-based Membrane Builder, we have generated the protein/membrane complexes for 15 membrane proteins of various sizes and shapes, 12 transmembrane and 3 interfacial proteins (see <a href="#pone-0000880-t002">Table 2</a>). Three kinds of lipid molecules (DMPC, DPPC, and POPC) and two kinds of system shapes (rectangular and hexagonal) were used to build a total of 90 complex systems. The test was facilitated by using pre-oriented protein structures from OPM database <a href="#pone.0000880-Lomize1">[25]</a> and the default values in each step, i.e., 1.5 lipid layers to determine the system size in <em>XY</em>, 12 Å water thickness to determine the system size along <em>Z</em>, and 0.15 M KCl concentration (the physiological ion concentration), unless specified explicitly. It should be stressed that these default values were elaborated and tested extensively by building the large number of protein/membrane complexes. <a href="#pone-0000880-t002">Table 2</a> shows the detailed information on the systems with POPC membranes and the full list of 90 complex systems is shown in <a href="#pone.0000880.s001">Table S1</a> in the Supporting Information. Note that ligands such as ions or small molecules in original PDB structures are not included.</p>
<div class="figure" id="pone-0000880-t002"><div class="img"><a name="pone-0000880-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000880" data-uri="info:doi/10.1371/journal.pone.0000880.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000880.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000880.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000880.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000880.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000880.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Testcases and system information with POPC membranes.</span></strong></p><span>doi:10.1371/journal.pone.0000880.t002</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>For given testcases, the whole process of generating a protein/membrane system took 5 minutes to 2 hours depending on the system size. The smallest protein and the largest protein in the testcases are PDB:1UYN (translocation domain of autotransporter NalP) and PDB:1SU4 (calcium ATPase), respectively. We have applied the insertion method for the proteins that have a cylindrical and regular shape, such as PDB:1GZM (bovine rhodopsin), 1ZLL (human phospholamban pentamer), 1I78 (outer membrane protease OmpT), 2GFP (multidrug transporter EmrD), 1OKC (mitochondrial ADP/ATP carrier), 2OAR (mechanosensitive channel of large conductance), 1UYN (translocator domain of autotransporter NalP). The replacement method was used for the other proteins that have rather irregular shape, including interfacial peptides, such as PDB:1H2S (sensory rhodopsin II), 2A65 (a bacterial homolog of Na+/Cl− dependent neurotransmitter transporters), 1SU4 (calcium ATPase), 1PHO (E. Coli porins), 1XQ8 (human micelle-bound alpha-synuclein), 2DEO (membrane protease specific for a stomatin homolog from Pyrococcus horikoshii), PDB:1UUN (main porin from Mycobacteria), and 1D5R (PTEN tumor suppressor). Pore water molecules were generated for PDB:2OAR, 1I78, 1OKC, 1UYN, 1UUN and 1ZLL. There are three membrane proteins such as PDB:1SU4, 1UUN, and 1XQ8 for which the default options could not generate the complex system properly. Both PDB:1SU4 and 1UUN have soluble cellular domains that are much larger than their transmembrane domain. Since Membrane Builder determines the system size in <em>XY</em> only based on the size of the transmembrane domain, the system size generated by the default “1.5 lipid layers” was not big enough to accommodate the cellular domain. Therefore, we had to manually adjust their system size. Since the final system size of PDB:1UUN was too big to use the insertion method, we had to use the replacement method, although it has a cylindrical and regular shape. In particular, PDB:1XQ8 has a very elongated conformation along the membrane surface. Since Membrane Builder assumes that the protein shape in the <em>XY</em> plane is more or less symmetric, the estimated system size by the default values was much smaller than it should be, such that we had to adjust the system size manually.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Each system was equilibrated for 375 ps in our local machines using the equilibration input files that were generated by Membrane Builder (see <a href="#pone-0000880-t001">Table 1</a>). All the calculations were performed with CHARMM <a href="#pone.0000880-Brooks1">[28]</a>, and the all-atom parameter set PARAM22 for proteins <a href="#pone.0000880-MacKerell1">[29]</a> including dihedral cross-term corrections (CMAP) <a href="#pone.0000880-Mackerell1">[30]</a> and a modified TIP3P water model <a href="#pone.0000880-Jorgensen1">[31]</a>, as well as recently optimized lipid parameters <a href="#pone.0000880-Klauda1">[32]</a>. The NVT dynamics (constant volume and temperature) is used for the first and second steps, and the NPAT (constant pressure, area, and temperature) dynamics for the rest at 303.15K (DMPC and POPC) and 323.15K (DPPC). The equilibrated protein/membrane complexes are freely available at the archive on the CHARMM-GUI website.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have described the generalized and automated process to build a protein/membrane complex interactively and easily using Membrane Builder in the CHARMM-GUI website. Its efficacy was illustrated by building protein/membrane complexes for 15 membrane proteins of various sizes and shapes. Each and every step is translated into a web interface so that a realistic protein/membrane complex can be generated through a web browser with a graphical user interface. The significance of this work is that the web interface provides not only the intuitive and elegant way to generate the membrane system, but also the detailed input files that are already optimized and tested extensively. We have adopted and further developed two lipid bilayer generation methods, “replacement method” and “insertion method.” The insertion method uses a pre-equilibrated lipid bilayer with a hole of an appropriate size into which a protein is inserted. This method is particularly useful for a protein with cylindrical and symmetrical shape. If a protein has an irregular shape, the replacement method would be a better choice. The replacement method uses a packing of lipid-like pseudo atoms, and replaces them by lipid molecules one at a time. Therefore, it generates a nicely packed lipid bilayer around the protein.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>While Membrane Builder currently supports three types of lipid molecules such as DMPC, DPPC, and POPC, we will make more lipid molecules available by generating the necessary lipid libraries. In addition, we will incorporate a method of building mixed lipid bilayers by introducing a step for choosing different types of lipids randomly in the replacement method. Furthermore, Membrane Builder will provide the CPγT (constant pressure, surface tension, and temperature) dynamics <a href="#pone.0000880-Feller1">[33]</a> to allow the system size along the <em>X</em> and <em>Y</em> axes to vary during the simulation, and the P2<sub>1</sub> image transformation <a href="#pone.0000880-Dolan1">[34]</a> to allow the number of lipid molecules in the top and bottom leaflets to vary during the simulation. Although Membrane Builder has been developed to use CHARMM <a href="#pone.0000880-Brooks1">[28]</a> as a MD simulation engine, we will provide the equilibration input files for other MD software such as GROMACS <a href="#pone.0000880-VanDerSpoel1">[35]</a> and NAMD <a href="#pone.0000880-Phillips1">[36]</a> in near future so that users can continue the equilibration of the generated membrane system with their own MD software.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Lipid Bilyaer Library and Lipid Molecule Library</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>To facilitate the building process of a membrane bilayer using the insertion and replacement methods, the library of lipid bilayers with holes and the library of lipid molecules have been generated. There are 90 pre-equilibrated lipid bilayers with a hole of radius from 1 Å to 45 Å for two different number of lipid molecules (128 or 256 lipid molecules) in the lipid library for DMPC, DPPC, and POPC. Starting from an equilibrated membrane bilayer of each type of lipid, the cylindrical pore has been generated by increasing the pore radius by 1 Å every 50 ps with a cylindrical harmonic restraint at 303.15 K (DMPC and POPC) and at 323.15 K (DPPC). The maximum number of lipid molecules available in each lipid bilayer is 256 (128 for each leaflet), and the largest bilayer size is 90 Å by 90 Å. The lipid molecule library for each type of lipid contains 2,000 different conformations. Lipid molecules are randomly selected from a 2.5 ns trajectory of each lipid bilayer. All the calculations were performed with CHARMM <a href="#pone.0000880-Brooks1">[28]</a>, and the all-atom parameter set PARAM22 for proteins <a href="#pone.0000880-MacKerell1">[29]</a> including dihedral cross-term corrections (CMAP) <a href="#pone.0000880-Mackerell1">[30]</a> and a modified TIP3P water model <a href="#pone.0000880-Jorgensen1">[31]</a>, as well as recently optimized lipid parameters <a href="#pone.0000880-Klauda1">[32]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000880"></div><a name="pone.0000880.s001" id="pone.0000880.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000880.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000880.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.23 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Jeffery Klauda, Patrick Lague, and Richard Pastor for providing equilibrated DMPC, DPPC, and POPC lipid coordinate. Sunhwan Jo is the&nbsp;recipient&nbsp;of the Undergraduate Research Assistant Fund from The University of Kansas.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: WI. Performed the experiments: SJ TK. Analyzed the data: SJ TK. Wrote the paper: WI SJ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000880-Jordan1" id="pone.0000880-Jordan1"></a>Jordan JD, Landau EM, Iyengar R (2000) Signaling networks: the origins of cellular multitasking. Cell  103: 193–200.  <ul class="find" data-citedArticleID="990962" data-doi="10.1016/s0092-8674(00)00112-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)00112-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signaling+networks%3A+the+origins+of+cellular+multitasking." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signaling+networks%3A+the+origins+of+cellular+multitasking.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000880-Hunter1" id="pone.0000880-Hunter1"></a>Hunter T (2000) Signaling–2000 and beyond. Cell  100: 113–127.  <ul class="find" data-citedArticleID="990956" data-doi="10.1016/s0092-8674(00)81688-8"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81688-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signaling%26%23x2013%3B2000+and+beyond." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signaling%26%23x2013%3B2000+and+beyond.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000880-Fu1" id="pone.0000880-Fu1"></a>Fu D, Libson A, Miercke LJ, Weitzman C, Nollert P, et al.  (2000) Structure of a glycerol-conducting channel and the basis for its selectivity. Science  290: 481–486.  <ul class="find" data-citedArticleID="990954" data-doi="10.1126/science.290.5491.481"><li><a href="http://dx.doi.org/10.1126/science.290.5491.481" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+a+glycerol-conducting+channel+and+the+basis+for+its+selectivity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+a+glycerol-conducting+channel+and+the+basis+for+its+selectivity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000880-Khademi1" id="pone.0000880-Khademi1"></a>Khademi S, O'Connell J 3rd, Remis J, Robles-Colmenares Y, Miercke LJ, et al.  (2004) Mechanism of ammonia transport by Amt/MEP/Rh: structure of AmtB at 1.35 A. Science  305: 1587–1594.  <ul class="find" data-citedArticleID="990966" data-doi="10.1126/science.1101952"><li><a href="http://dx.doi.org/10.1126/science.1101952" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+of+ammonia+transport+by+Amt%2FMEP%2FRh%3A+structure+of+AmtB+at+1.35+A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+of+ammonia+transport+by+Amt%2FMEP%2FRh%3A+structure+of+AmtB+at+1.35+A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000880-Yellen1" id="pone.0000880-Yellen1"></a>Yellen G (2002) The voltage-gated potassium channels and their relatives. Nature  419: 35–42.  <ul class="find" data-citedArticleID="991002" data-doi="10.1038/nature00978"><li><a href="http://dx.doi.org/10.1038/nature00978" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+voltage-gated+potassium+channels+and+their+relatives." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+voltage-gated+potassium+channels+and+their+relatives.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000880-Jiang1" id="pone.0000880-Jiang1"></a>Jiang Y, Lee A, Chen J, Cadene M, Chait BT, et al.  (2002) Crystal structure and mechanism of a calcium-gated potassium channel. Nature  417: 515–522.  <ul class="find" data-citedArticleID="990960" data-doi="10.1038/417515a"><li><a href="http://dx.doi.org/10.1038/417515a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+and+mechanism+of+a+calcium-gated+potassium+channel." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+and+mechanism+of+a+calcium-gated+potassium+channel.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000880-Murata1" id="pone.0000880-Murata1"></a>Murata K, Mitsuoka K, Hirai T, Walz T, Agre P, et al.  (2000) Structural determinants of water permeation through aquaporin-1. Nature  407: 599–605.  <ul class="find" data-citedArticleID="990976" data-doi="10.1038/35036519"><li><a href="http://dx.doi.org/10.1038/35036519" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+determinants+of+water+permeation+through+aquaporin-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+determinants+of+water+permeation+through+aquaporin-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000880-Dong1" id="pone.0000880-Dong1"></a>Dong J, Yang G, McHaourab HS (2005) Structural basis of energy transduction in the transport cycle of MsbA. Science  308: 1023–1028.  <ul class="find" data-citedArticleID="990946" data-doi="10.1126/science.1106592"><li><a href="http://dx.doi.org/10.1126/science.1106592" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+of+energy+transduction+in+the+transport+cycle+of+MsbA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+of+energy+transduction+in+the+transport+cycle+of+MsbA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000880-Elston1" id="pone.0000880-Elston1"></a>Elston T, Wang H, Oster G (1998) Energy transduction in ATP synthase. Nature  391: 510–513.  <ul class="find" data-citedArticleID="990948" data-doi="10.1038/35185"><li><a href="http://dx.doi.org/10.1038/35185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Energy+transduction+in+ATP+synthase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Energy+transduction+in+ATP+synthase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000880-Alberts1" id="pone.0000880-Alberts1"></a>Alberts B (2002) Molecular biology of the cell. New York: Garland Science.   <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0000880-Wallin1" id="pone.0000880-Wallin1"></a>Wallin E, von Heijne G (1998) Genome-wide analysis of integral membrane proteins from eubacterial, archaean, and eukaryotic organisms. Protein Sci  7: 1029–1038.  <ul class="find" data-citedArticleID="990996" data-doi="10.1002/pro.5560070420"><li><a href="http://dx.doi.org/10.1002/pro.5560070420" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+integral+membrane+proteins+from+eubacterial%2C+archaean%2C+and+eukaryotic+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+integral+membrane+proteins+from+eubacterial%2C+archaean%2C+and+eukaryotic+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000880-Terstappen1" id="pone.0000880-Terstappen1"></a>Terstappen GC, Reggiani A (2001) In silico research in drug discovery. Trends Pharmacol Sci  22: 23–26.  <ul class="find" data-citedArticleID="990990" data-doi="10.1016/s0165-6147(00)01584-4"><li><a href="http://dx.doi.org/10.1016/s0165-6147(00)01584-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+silico+research+in+drug+discovery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+silico+research+in+drug+discovery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000880-1" id="pone.0000880-1"></a>White S Membrane Proteins of Known 3D Structure.   <a href="http://blanco.biomol.uci.edu/Membrane_Proteins_xtal.html">http://blanco.biomol.uci.edu/Membrane_Pr​oteins_xtal.html</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000880-Berman1" id="pone.0000880-Berman1"></a>Berman HM, Battistuz T, Bhat TN, Bluhm WF, Bourne PE, et al.  (2002) The Protein Data Bank. Acta Crystallogr D Biol Crystallogr  58: 899–907.  <ul class="find" data-citedArticleID="990938" data-doi="10.1107/s0907444902003451"><li><a href="http://dx.doi.org/10.1107/s0907444902003451" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Protein+Data+Bank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Protein+Data+Bank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000880-deGroot1" id="pone.0000880-deGroot1"></a>de Groot BL, Grubmuller H (2001) Water permeation across biological membranes: mechanism and dynamics of aquaporin-1 and GlpF. Science  294: 2353–2357.  <ul class="find" data-citedArticleID="991004" data-doi="10.1126/science.1062459"><li><a href="http://dx.doi.org/10.1126/science.1062459" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Water+permeation+across+biological+membranes%3A+mechanism+and+dynamics+of+aquaporin-1+and+GlpF." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Water+permeation+across+biological+membranes%3A+mechanism+and+dynamics+of+aquaporin-1+and+GlpF.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000880-Chakrabarti1" id="pone.0000880-Chakrabarti1"></a>Chakrabarti N, Tajkhorshid E, Roux B, Pomes R (2004) Molecular basis of proton blockage in aquaporins. Structure  12: 65–74.  <ul class="find" data-citedArticleID="990942" data-doi="10.1016/j.str.2003.11.017"><li><a href="http://dx.doi.org/10.1016/j.str.2003.11.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+basis+of+proton+blockage+in+aquaporins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+basis+of+proton+blockage+in+aquaporins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000880-Roux1" id="pone.0000880-Roux1"></a>Roux B, Schulten K (2004) Computational studies of membrane channels. Structure  12: 1343–1351.  <ul class="find" data-citedArticleID="990986" data-doi="10.1016/j.str.2004.06.013"><li><a href="http://dx.doi.org/10.1016/j.str.2004.06.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+studies+of+membrane+channels." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+studies+of+membrane+channels.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000880-Noskov1" id="pone.0000880-Noskov1"></a>Noskov SY, Berneche S, Roux B (2004) Control of ion selectivity in potassium channels by electrostatic and dynamic properties of carbonyl ligands. Nature  431: 830–834.  <ul class="find" data-citedArticleID="990980" data-doi="10.1038/nature02943"><li><a href="http://dx.doi.org/10.1038/nature02943" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+ion+selectivity+in+potassium+channels+by+electrostatic+and+dynamic+properties+of+carbonyl+ligands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+ion+selectivity+in+potassium+channels+by+electrostatic+and+dynamic+properties+of+carbonyl+ligands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000880-Petrache1" id="pone.0000880-Petrache1"></a>Petrache HI, Grossfield A, MacKenzie KR, Engelman DM, Woolf TB (2000) Modulation of glycophorin A transmembrane helix interactions by lipid bilayers: molecular dynamics calculations. J Mol Biol  302: 727–746.  <ul class="find" data-citedArticleID="990982" data-doi="10.1006/jmbi.2000.4072"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulation+of+glycophorin+A+transmembrane+helix+interactions+by+lipid+bilayers%3A+molecular+dynamics+calculations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulation+of+glycophorin+A+transmembrane+helix+interactions+by+lipid+bilayers%3A+molecular+dynamics+calculations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000880-Woolf1" id="pone.0000880-Woolf1"></a>Woolf TB, Roux B (1994) Molecular dynamics simulation of the gramicidin channel in a phospholipid bilayer. Proc Natl Acad Sci U S A  91: 11631–11635.  <ul class="find" data-citedArticleID="990998" data-doi="10.1073/pnas.91.24.11631"><li><a href="http://dx.doi.org/10.1073/pnas.91.24.11631" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+dynamics+simulation+of+the+gramicidin+channel+in+a+phospholipid+bilayer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+dynamics+simulation+of+the+gramicidin+channel+in+a+phospholipid+bilayer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000880-Woolf2" id="pone.0000880-Woolf2"></a>Woolf TB, Roux B (1996) Structure, energetics, and dynamics of lipid-protein interactions: A molecular dynamics study of the gramicidin A channel in a DMPC bilayer. Proteins  24: 92–114.  <ul class="find" data-citedArticleID="991000" data-doi="10.1002/(sici)1097-0134(199601)24:1<92::aid-prot7&gt;3.0.co;2-q"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0134(199601)24:1<92::aid-prot7&gt;3.0.co;2-q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure%2C+energetics%2C+and+dynamics+of+lipid-protein+interactions%3A+A+molecular+dynamics+study+of+the+gramicidin+A+channel+in+a+DMPC+bilayer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure%2C+energetics%2C+and+dynamics+of+lipid-protein+interactions%3A+A+molecular+dynamics+study+of+the+gramicidin+A+channel+in+a+DMPC+bilayer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000880-Shen1" id="pone.0000880-Shen1"></a>Shen L, Bassolino D, Stouch T (1997) Transmembrane helix structure, dynamics, and interactions: multi-nanosecond molecular dynamics simulations. Biophys J  73: 3–20.  <ul class="find" data-citedArticleID="990988" data-doi="10.1016/s0006-3495(97)78042-1"><li><a href="http://dx.doi.org/10.1016/s0006-3495(97)78042-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transmembrane+helix+structure%2C+dynamics%2C+and+interactions%3A+multi-nanosecond+molecular+dynamics+simulations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transmembrane+helix+structure%2C+dynamics%2C+and+interactions%3A+multi-nanosecond+molecular+dynamics+simulations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000880-Tieleman1" id="pone.0000880-Tieleman1"></a>Tieleman DP, Berendsen HJ (1998) A molecular dynamics study of the pores formed by Escherichia coli OmpF porin in a fully hydrated palmitoyloleoylphosphatidylcholine bilayer. Biophys J  74: 2786–2801.  <ul class="find" data-citedArticleID="990992" data-doi="10.1016/s0006-3495(98)77986-x"><li><a href="http://dx.doi.org/10.1016/s0006-3495(98)77986-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+molecular+dynamics+study+of+the+pores+formed+by+Escherichia+coli+OmpF+porin+in+a+fully+hydrated+palmitoyloleoylphosphatidylcholine+bilayer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+molecular+dynamics+study+of+the+pores+formed+by+Escherichia+coli+OmpF+porin+in+a+fully+hydrated+palmitoyloleoylphosphatidylcholine+bilayer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000880-FaraldoGomez1" id="pone.0000880-FaraldoGomez1"></a>Faraldo-Gomez JD, Smith GR, Sansom MS (2002) Setting up and optimization of membrane protein simulations. Eur Biophys J  31: 217–227.  <ul class="find" data-citedArticleID="990950" data-doi="10.1007/s00249-002-0207-5"><li><a href="http://dx.doi.org/10.1007/s00249-002-0207-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Setting+up+and+optimization+of+membrane+protein+simulations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Setting+up+and+optimization+of+membrane+protein+simulations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000880-Lomize1" id="pone.0000880-Lomize1"></a>Lomize MA, Lomize AL, Pogozheva ID, Mosberg HI (2006) OPM: orientations of proteins in membranes database. Bioinformatics  22: 623–625.  <ul class="find" data-citedArticleID="990970" data-doi="10.1093/bioinformatics/btk023"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btk023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OPM%3A+orientations+of+proteins+in+membranes+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OPM%3A+orientations+of+proteins+in+membranes+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000880-Nagle1" id="pone.0000880-Nagle1"></a>Nagle JF, Tristram-Nagle S (2000) Structure of lipid bilayers. Biochim Biophys Acta  1469: 159–195.  <ul class="find" data-citedArticleID="990978" data-doi="10.1016/s0304-4157(00)00016-2"><li><a href="http://dx.doi.org/10.1016/s0304-4157(00)00016-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+lipid+bilayers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+lipid+bilayers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000880-Im1" id="pone.0000880-Im1"></a>Im W, Roux B (2002) Ions and counterions in a biological channel: a molecular dynamics simulation of OmpF porin from Escherichia coli in an explicit membrane with 1 M KCl aqueous salt solution. J Mol Biol  319: 1177–1197.  <ul class="find" data-citedArticleID="990958" data-doi="10.1016/s0022-2836(02)00380-7"><li><a href="http://dx.doi.org/10.1016/s0022-2836(02)00380-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ions+and+counterions+in+a+biological+channel%3A+a+molecular+dynamics+simulation+of+OmpF+porin+from+Escherichia+coli+in+an+explicit+membrane+with+1+M+KCl+aqueous+salt+solution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ions+and+counterions+in+a+biological+channel%3A+a+molecular+dynamics+simulation+of+OmpF+porin+from+Escherichia+coli+in+an+explicit+membrane+with+1+M+KCl+aqueous+salt+solution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000880-Brooks1" id="pone.0000880-Brooks1"></a>Brooks BR, Bruccoleri RE, Olafson BD, States DJ, Swaminathan S, et al.  (1983) CHARMM: a program for macromolecular energy, minimization, and dynamics calculations. J Comp Chem  4: 187–217.  <ul class="find" data-citedArticleID="990940" data-doi="10.1002/jcc.540040211"><li><a href="http://dx.doi.org/10.1002/jcc.540040211" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CHARMM%3A+a+program+for+macromolecular+energy%2C+minimization%2C+and+dynamics+calculations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CHARMM%3A+a+program+for+macromolecular+energy%2C+minimization%2C+and+dynamics+calculations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000880-MacKerell1" id="pone.0000880-MacKerell1"></a>MacKerell AD, Bashford D, Bellott M, Dunbrack RL, Evanseck JD, et al.  (1998) All-atom empirical potential for molecular modeling and dynamics studies of proteins. Journal of Physical Chemistry B  102: 3586–3616.  <ul class="find" data-citedArticleID="990972" data-doi="10.1021/jp973084f"><li><a href="http://dx.doi.org/10.1021/jp973084f" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=All-atom+empirical+potential+for+molecular+modeling+and+dynamics+studies+of+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22All-atom+empirical+potential+for+molecular+modeling+and+dynamics+studies+of+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000880-Mackerell1" id="pone.0000880-Mackerell1"></a>Mackerell AD Jr, Feig M, Brooks CL 3rd (2004) Extending the treatment of backbone energetics in protein force fields: limitations of gas-phase quantum mechanics in reproducing protein conformational distributions in molecular dynamics simulations. J Comput Chem  25: 1400–1415.  <ul class="find" data-citedArticleID="990974" data-doi="10.1002/jcc.20065"><li><a href="http://dx.doi.org/10.1002/jcc.20065" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extending+the+treatment+of+backbone+energetics+in+protein+force+fields%3A+limitations+of+gas-phase+quantum+mechanics+in+reproducing+protein+conformational+distributions+in+molecular+dynamics+simulations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extending+the+treatment+of+backbone+energetics+in+protein+force+fields%3A+limitations+of+gas-phase+quantum+mechanics+in+reproducing+protein+conformational+distributions+in+molecular+dynamics+simulations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000880-Jorgensen1" id="pone.0000880-Jorgensen1"></a>Jorgensen WL, Chandrasekhar J, Madura JD, Impey RW, Klein ML (1983) Comparison of Simple Potential Functions for Simulating Liquid Water. Journal of Chemical Physics  79: 926–935.  <ul class="find" data-citedArticleID="990964" data-doi="10.1063/1.445869"><li><a href="http://dx.doi.org/10.1063/1.445869" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+Simple+Potential+Functions+for+Simulating+Liquid+Water." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+Simple+Potential+Functions+for+Simulating+Liquid+Water.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000880-Klauda1" id="pone.0000880-Klauda1"></a>Klauda JB, Brooks BR, MacKerell AD, Venable RM, Pastor RW (2005) An ab initio study on the torsional surface of alkanes and its effect on molecular simulations of alkanes and a DPPC bilayer. Journal of Physical Chemistry B  109: 5300–5311.  <ul class="find" data-citedArticleID="990968" data-doi="10.1021/jp0468096"><li><a href="http://dx.doi.org/10.1021/jp0468096" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+ab+initio+study+on+the+torsional+surface+of+alkanes+and+its+effect+on+molecular+simulations+of+alkanes+and+a+DPPC+bilayer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+ab+initio+study+on+the+torsional+surface+of+alkanes+and+its+effect+on+molecular+simulations+of+alkanes+and+a+DPPC+bilayer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000880-Feller1" id="pone.0000880-Feller1"></a>Feller SE, Zhang YH, Pastor RW (1995) Computer-Simulation of Liquid/Liquid Interfaces .2. Surface-Tension Area Dependence of a Bilayer and Monolayer. Journal of Chemical Physics  103: 10267–10276.  <ul class="find" data-citedArticleID="990952" data-doi="10.1063/1.469928"><li><a href="http://dx.doi.org/10.1063/1.469928" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computer-Simulation+of+Liquid%2FLiquid+Interfaces+.2.+Surface-Tension+Area+Dependence+of+a+Bilayer+and+Monolayer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computer-Simulation+of+Liquid%2FLiquid+Interfaces+.2.+Surface-Tension+Area+Dependence+of+a+Bilayer+and+Monolayer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000880-Dolan1" id="pone.0000880-Dolan1"></a>Dolan EA, Venable RM, Pastor RW, Brooks BR (2002) Simulations of membranes and other interfacial systems using P2(1) and Pc periodic boundary conditions. Biophys J  82: 2317–2325.  <ul class="find" data-citedArticleID="990944" data-doi="10.1016/s0006-3495(02)75577-x"><li><a href="http://dx.doi.org/10.1016/s0006-3495(02)75577-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simulations+of+membranes+and+other+interfacial+systems+using+P2%281%29+and+Pc+periodic+boundary+conditions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simulations+of+membranes+and+other+interfacial+systems+using+P2%281%29+and+Pc+periodic+boundary+conditions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000880-VanDerSpoel1" id="pone.0000880-VanDerSpoel1"></a>Van Der Spoel D, Lindahl E, Hess B, Groenhof G, Mark AE, et al.  (2005) GROMACS: fast, flexible, and free. J Comput Chem  26: 1701–1718.  <ul class="find" data-citedArticleID="990994" data-doi="10.1002/jcc.20291"><li><a href="http://dx.doi.org/10.1002/jcc.20291" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GROMACS%3A+fast%2C+flexible%2C+and+free." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GROMACS%3A+fast%2C+flexible%2C+and+free.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000880-Phillips1" id="pone.0000880-Phillips1"></a>Phillips JC, Braun R, Wang W, Gumbart J, Tajkhorshid E, et al.  (2005) Scalable molecular dynamics with NAMD. J Comput Chem  26: 1781–1802.  <ul class="find" data-citedArticleID="990984" data-doi="10.1002/jcc.20289"><li><a href="http://dx.doi.org/10.1002/jcc.20289" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scalable+molecular+dynamics+with+NAMD." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Scalable+molecular+dynamics+with+NAMD.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.XML" value="91176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.PDF" value="627594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g001.PNG_L" value="252126"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g001.PNG_M" value="78758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g001.PNG_S" value="12370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g001.TIF" value="853316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g001.PNG_I" value="66523"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g002.PNG_L" value="240872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g002.PNG_M" value="59530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g002.PNG_S" value="10662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g002.TIF" value="537512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g002.PNG_I" value="52937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g003.PNG_L" value="5367658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g003.PNG_M" value="500597"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g003.PNG_S" value="15906"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g003.TIF" value="6753658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g003.PNG_I" value="154366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g004.PNG_L" value="4850677"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g004.PNG_M" value="358344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g004.PNG_S" value="25220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g004.TIF" value="6762606"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g004.PNG_I" value="229176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g005.PNG_L" value="6627306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g005.PNG_M" value="354346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g005.PNG_S" value="31101"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g005.TIF" value="8400408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g005.PNG_I" value="400642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.t001.PNG_L" value="119619"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.t001.PNG_M" value="70477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.t001.PNG_S" value="10260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.t001.TIF" value="550834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.t001.PNG_I" value="22082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g006.PNG_L" value="6956531"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g006.PNG_M" value="470590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g006.PNG_S" value="16920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g006.TIF" value="7952244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g006.PNG_I" value="152658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g007.PNG_L" value="1939231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g007.PNG_M" value="212425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g007.PNG_S" value="9479"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g007.TIF" value="2433014"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.g007.PNG_I" value="68121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.t002.PNG_L" value="300275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.t002.PNG_M" value="170346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.t002.PNG_S" value="18256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.t002.TIF" value="1415304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.t002.PNG_I" value="51764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000880.s001.DOC" value="225280"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000880&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000880" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000880&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000880&volume=&issue=&title=Automated Builder and Database of Protein/Membrane Complexes for Molecular Dynamics Simulations&author_name=Sunhwan%20Jo%2C%20Taehoon%20Kim%2C%20Wonpil%20Im&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000880" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000880" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000880" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000880&amp;t=Automated%20Builder%20and%20Database%20of%20Protein%2FMembrane%20Complexes%20for%20Molecular%20Dynamics%20Simulations" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000880&title=Automated%20Builder%20and%20Database%20of%20Protein%2FMembrane%20Complexes%20for%20Molecular%20Dynamics%20Simulations&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000880&amp;title=Automated%20Builder%20and%20Database%20of%20Protein%2FMembrane%20Complexes%20for%20Molecular%20Dynamics%20Simulations" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000880" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000880&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Automated Builder and Database of Protein/Membrane Complexes for Molecular Dynamics Simulations';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000880';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Automated%20Builder%20and%20Database%20of%20Protein%2FMembrane%20Complexes%20for%20Molecular%20Dynamics%20Simulations http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000880" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000880" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biochemical+simulations%22" title="Search for articles in the subject area:'Biochemical simulations'"><div class="flagText">Biochemical simulations</div></a>
              <div data-categoryid="31873" data-articleid="24598"
                   data-categoryname="Biochemical simulations"
                   class="flagImage" title="Flag 'Biochemical simulations' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lipid+bilayer%22" title="Search for articles in the subject area:'Lipid bilayer'"><div class="flagText">Lipid bilayer</div></a>
              <div data-categoryid="43135" data-articleid="24598"
                   data-categoryname="Lipid bilayer"
                   class="flagImage" title="Flag 'Lipid bilayer' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lipids%22" title="Search for articles in the subject area:'Lipids'"><div class="flagText">Lipids</div></a>
              <div data-categoryid="17941" data-articleid="24598"
                   data-categoryname="Lipids"
                   class="flagImage" title="Flag 'Lipids' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+characteristics%22" title="Search for articles in the subject area:'Membrane characteristics'"><div class="flagText">Membrane characteristics</div></a>
              <div data-categoryid="22189" data-articleid="24598"
                   data-categoryname="Membrane characteristics"
                   class="flagImage" title="Flag 'Membrane characteristics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+protein+complexes%22" title="Search for articles in the subject area:'Membrane protein complexes'"><div class="flagText">Membrane protein complexes</div></a>
              <div data-categoryid="35893" data-articleid="24598"
                   data-categoryname="Membrane protein complexes"
                   class="flagImage" title="Flag 'Membrane protein complexes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+proteins%22" title="Search for articles in the subject area:'Membrane proteins'"><div class="flagText">Membrane proteins</div></a>
              <div data-categoryid="17569" data-articleid="24598"
                   data-categoryname="Membrane proteins"
                   class="flagImage" title="Flag 'Membrane proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+structures%22" title="Search for articles in the subject area:'Membrane structures'"><div class="flagText">Membrane structures</div></a>
              <div data-categoryid="17215" data-articleid="24598"
                   data-categoryname="Membrane structures"
                   class="flagImage" title="Flag 'Membrane structures' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure+databases%22" title="Search for articles in the subject area:'Protein structure databases'"><div class="flagText">Protein structure databases</div></a>
              <div data-categoryid="33455" data-articleid="24598"
                   data-categoryname="Protein structure databases"
                   class="flagImage" title="Flag 'Protein structure databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=322'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4931'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3846&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
