

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: High Throughput Genetic Analysis of Congenital Myasthenic Syndromes Using Resequencing Microarrays</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000918"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000918" />

    <meta name="citation_title" content="High Throughput Genetic Analysis of Congenital Myasthenic Syndromes Using Resequencing Microarrays"/>
    <meta itemprop="name" content="High Throughput Genetic Analysis of Congenital Myasthenic Syndromes Using Resequencing Microarrays"/>

      <meta name="citation_author" content="Lisa Denning"/>
            <meta name="citation_author_institution" content="Department of Neurology, University of California at Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="Jennifer A. Anderson"/>
            <meta name="citation_author_institution" content="Department of Neurology, University of California at Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="Ryan Davis"/>
            <meta name="citation_author_institution" content="Department of Pathology, Medical Investigation of Neurodevelopmental Disorders Institute, University of California at Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="Jeffrey P. Gregg"/>
            <meta name="citation_author_institution" content="Department of Pathology, Medical Investigation of Neurodevelopmental Disorders Institute, University of California at Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="Jennifer Kuzdenyi"/>
            <meta name="citation_author_institution" content="Department of Neurology, University of California at Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="Ricardo A. Maselli"/>
            <meta name="citation_author_institution" content="Department of Neurology, University of California at Davis, Davis, California, United States of America"/>

    <meta name="citation_date" content="2007/9/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000918.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e918"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Current understanding of congenital myasthenic syndromes.; citation_author=AG Engel; citation_author=SM Sine; citation_journal_title=Curr Opin Pharmacol; citation_volume=5; citation_number=1; citation_pages=308-321; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Choline acetyltransferase mutations cause myasthenic syndrome associated with episodic apnea in humans.; citation_author=K Ohno; citation_author=A Tsujino; citation_author=JM Brengman; citation_author=CM Harper; citation_author=Z Bajzer; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=2; citation_pages=2017-2022; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Human endplate acetylcholinesterase deficiency caused by mutations in the collagen-like tail subunit (COLQ) of the asymmetric enzyme.; citation_author=K Ohno; citation_author=J Brengman; citation_author=A Tsujino; citation_author=AG Engel; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=3; citation_pages=9654-9659; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Congenital myasthenic syndromes: progress over the past decade.; citation_author=AG Engel; citation_author=K Ohno; citation_author=SM Sine; citation_journal_title=Muscle Nerve; citation_volume=27; citation_number=4; citation_pages=4-25; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Rapsyn mutations in humans cause endplate acetylcholine-receptor deficiency and myasthenic syndrome.; citation_author=K Ohno; citation_author=AG Engel; citation_author=XM Shen; citation_author=D Selcen; citation_author=J Brengman; citation_journal_title=Am J Hum Genet; citation_volume=70; citation_number=5; citation_pages=875-885; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=MUSK, a new target for mutations causing congenital myasthenic syndrome.; citation_author=F Chevessier; citation_author=B Faraut; citation_author=A Ravel-Chapuis; citation_author=P Richard; citation_author=K Gaudon; citation_journal_title=Hum Mol Genet; citation_volume=13; citation_number=6; citation_pages=3229-3240; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Dok-7 mutations underlie a neuromuscular junction synaptopathy.; citation_author=D Beeson; citation_author=O Higuchi; citation_author=J Palace; citation_author=J Cossins; citation_author=H Spearman; citation_journal_title=Science; citation_volume=313; citation_number=7; citation_pages=1975-1978; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mutation in the AChR ion channel gate underlies a fast channel congenital myasthenic syndrome.; citation_author=R Webster; citation_author=M Brydson; citation_author=R Croxen; citation_author=J Newsom-Davis; citation_author=A Vincent; citation_journal_title=Neurology; citation_volume=62; citation_number=8; citation_pages=1090-1096; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Splicing abnormalities in congenital myasthenic syndromes.; citation_author=K Ohno; citation_author=AG Engel; citation_journal_title=Acta Myol; citation_volume=24; citation_number=9; citation_pages=50-54; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mutation of the acetylcholine receptor epsilon-subunit promoter in congenital myasthenic syndrome.; citation_author=P Nichols; citation_author=R Croxen; citation_author=A Vincent; citation_author=R Rutter; citation_author=M Hutchinson; citation_journal_title=Ann Neurol; citation_volume=45; citation_number=10; citation_pages=439-443; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A newly identified chromosomal microdeletion of the rapsyn gene causes a congenital myasthenic syndrome.; citation_author=JS Muller; citation_author=A Abicht; citation_author=HJ Christen; citation_author=R Stucka; citation_author=U Schara; citation_journal_title=Neuromuscul Disord; citation_volume=14; citation_number=11; citation_pages=744-749; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An intronic base alteration of the CHRNE gene leading to a congenital myasthenic syndrome.; citation_author=JS Muller; citation_author=R Stucka; citation_author=S Neudecker; citation_author=S Zierz; citation_author=C Schmidt; citation_journal_title=Neurology; citation_volume=65; citation_number=12; citation_pages=463-465; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=126th International Workshop: congenital myasthenic syndromes, 24-26 September 2004, Naarden, the Netherlands.; citation_author=D Beeson; citation_author=D Hantai; citation_author=H Lochmuller; citation_author=AG Engel; citation_journal_title=Neuromuscul Disord; citation_volume=15; citation_number=13; citation_pages=498-512; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sequencing arrays for screening multiple genes associated with early-onset human retinal degenerations on a high-throughput platform.; citation_author=MN Mandal; citation_author=JR Heckenlively; citation_author=T Burch; citation_author=L Chen; citation_author=V Vasireddy; citation_journal_title=Invest Ophthalmol Vis Sci; citation_volume=46; citation_number=14; citation_pages=3355-3362; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A transforming MET mutation discovered in non-small cell lung cancer using microarray-based resequencing.; citation_author=T Tengs; citation_author=JC Lee; citation_author=J Guillermo Paez; citation_author=X Zhao; citation_author=T Laframboise; citation_journal_title=Cancer Lett; citation_volume=229; citation_number=15; citation_pages=227-233; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Screening candidate genes for mutations in patients with hypogonadotropic hypogonadism using custom genome resequencing microarrays.; citation_author=N Xu; citation_author=RH Podolsky; citation_author=P Chudgar; citation_author=LP Chorich; citation_author=C Liu; citation_journal_title=Am J Obstet Gynecol; citation_volume=192; citation_number=16; citation_pages=1274-1282; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A map of human genome sequence variation containing 1.42 million single nucleotide polymorphisms.; citation_author=R Sachidanandam; citation_author=D Weissman; citation_author=SC Schmidt; citation_author=JM Kakol; citation_author=LD Stein; citation_journal_title=Nature; citation_volume=409; citation_number=17; citation_pages=928-933; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=High-throughput variation detection and genotyping using microarrays.; citation_author=DJ Cutler; citation_author=ME Zwick; citation_author=MM Carrasquillo; citation_author=CT Yohn; citation_author=KP Tobin; citation_journal_title=Genome Res; citation_volume=11; citation_number=18; citation_pages=1913-1925; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=SNiPer: improved SNP genotype calling for Affymetrix 10K GeneChip microarray data.; citation_author=MJ Huentelman; citation_author=DW Craig; citation_author=AD Shieh; citation_author=JJ Corneveaux; citation_author=D Hu-Lince; citation_journal_title=BMC Genomics; citation_volume=6; citation_number=19; citation_pages=149; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comparisons of substitution, insertion and deletion probes for resequencing and mutational analysis using oligonucleotide microarrays.; citation_author=MW Karaman; citation_author=S Groshen; citation_author=CC Lee; citation_author=BL Pike; citation_author=JG Hacia; citation_journal_title=Nucl Acids Res; citation_volume=33; citation_number=20; citation_pages=e33; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Resequencing and mutational analysis using oligonucleotide microarrays.; citation_author=JG Hacia; citation_journal_title=Nat Genet; citation_volume=21; citation_number=21; citation_pages=42-47 (Suppl.); citation_date=1999; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000918" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="High Throughput Genetic Analysis of Congenital Myasthenic Syndromes Using Resequencing Microarrays"/>
    <meta name="twitter:description" content="BackgroundThe use of resequencing microarrays for screening multiple, candidate disease loci is a promising alternative to conventional capillary sequencing. We describe the performance of a custom resequencing microarray for mutational analysis of Congenital Myasthenic Syndromes (CMSs), a group of disorders in which the normal process of neuromuscular transmission is impaired.Methodology/Principal FindingsOur microarray was designed to assay the exons and flanking intronic regions of 8 genes linked to CMSs. A total of 31 microarrays were hybridized with genomic DNA from either individuals with known CMS mutations or from healthy controls. We estimated an overall microarray call rate of 93.61%, and we found the percentage agreement between the microarray and capillary sequencing techniques to be 99.95%. In addition, our microarray exhibited 100% specificity and 99.99% reproducibility. Finally, the microarray detected 22 out of the 23 known missense mutations, but it failed to detect all 7 known insertion and deletion (indels) mutations, indicating an overall sensitivity of 73.33% and a sensitivity with respect to missense mutations of 95.65%.Conclusions/SignificanceOverall, our microarray prototype exhibited strong performance and proved highly efficient for screening genes associated with CMSs. Until indels can be efficiently assayed with this technology, however, we recommend using resequencing microarrays for screening CMS mutations after common indels have been first assayed by capillary sequencing."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000918.g002"/>

  <meta property="og:title" content="High Throughput Genetic Analysis of Congenital Myasthenic Syndromes Using Resequencing Microarrays" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000918" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3824'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7703'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1139&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000918">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000918" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000918&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    High Throughput Genetic Analysis of Congenital Myasthenic Syndromes Using Resequencing Microarrays
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lisa Denning, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Neurology, University of California at Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jennifer A. Anderson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Neurology, University of California at Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ryan Davis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology, Medical Investigation of Neurodevelopmental Disorders Institute, University of California at Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jeffrey P. Gregg, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology, Medical Investigation of Neurodevelopmental Disorders Institute, University of California at Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jennifer Kuzdenyi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Neurology, University of California at Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ricardo A. Maselli
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:ramaselli@ucdavis.edu">ramaselli@ucdavis.edu</a></p>

                <p>Affiliation:
                  Department of Neurology, University of California at Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000918</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000918" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000918" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000918" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000918" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000918-t001" data-doi="info:doi/10.1371/journal.pone.0000918" data-uri="info:doi/10.1371/journal.pone.0000918.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000918-g001" data-doi="info:doi/10.1371/journal.pone.0000918" data-uri="info:doi/10.1371/journal.pone.0000918.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000918-g002" data-doi="info:doi/10.1371/journal.pone.0000918" data-uri="info:doi/10.1371/journal.pone.0000918.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000918-t002" data-doi="info:doi/10.1371/journal.pone.0000918" data-uri="info:doi/10.1371/journal.pone.0000918.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000918-t003" data-doi="info:doi/10.1371/journal.pone.0000918" data-uri="info:doi/10.1371/journal.pone.0000918.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000918">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000918" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The use of resequencing microarrays for screening multiple, candidate disease loci is a promising alternative to conventional capillary sequencing. We describe the performance of a custom resequencing microarray for mutational analysis of Congenital Myasthenic Syndromes (CMSs), a group of disorders in which the normal process of neuromuscular transmission is impaired.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Our microarray was designed to assay the exons and flanking intronic regions of 8 genes linked to CMSs. A total of 31 microarrays were hybridized with genomic DNA from either individuals with known CMS mutations or from healthy controls. We estimated an overall microarray call rate of 93.61%, and we found the percentage agreement between the microarray and capillary sequencing techniques to be 99.95%. In addition, our microarray exhibited 100% specificity and 99.99% reproducibility. Finally, the microarray detected 22 out of the 23 known missense mutations, but it failed to detect all 7 known insertion and deletion (indels) mutations, indicating an overall sensitivity of 73.33% and a sensitivity with respect to missense mutations of 95.65%.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Overall, our microarray prototype exhibited strong performance and proved highly efficient for screening genes associated with CMSs. Until indels can be efficiently assayed with this technology, however, we recommend using resequencing microarrays for screening CMS mutations after common indels have been first assayed by capillary sequencing.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Denning L, Anderson JA, Davis R, Gregg JP, Kuzdenyi J, et al.  (2007) High Throughput Genetic Analysis of Congenital Myasthenic Syndromes Using Resequencing Microarrays. PLoS ONE 2(9):
          e918.
            doi:10.1371/journal.pone.0000918</p><p><strong>Academic Editor: </strong>Philip Awadalla, North Carolina State University, United States of America</p><p><strong>Received:</strong> June 21, 2007; <strong>Accepted:</strong> August 22, 2007; <strong>Published:</strong> September 19, 2007</p><p><strong>Copyright:</strong> © 2007 Denning et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was funded by awards from the Muscular Dystrophy Association and the Myasthenia Gravis Foundation of America. In addition, the microarray core facility at the UC Davis M.I.N.D. Institute was supported by NIEHS 1 P01 ES11269-01. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Congenital myasthenic syndromes (CMSs) comprise a distinctive group of disorders in which the normal process of neuromuscular transmission is impaired by one or more pathogenic mechanisms. To date, nine genes have been demonstrated to harbor causative, mostly recessive, mutations for CMSs (<a href="#pone-0000918-t001">Table 1</a>; <a href="#pone.0000918-Engel1">[1]</a>–<a href="#pone.0000918-Webster1">[8]</a>). In the majority of these cases, patients present as compound heterozygotes, usually combining a missense mutation in one allele with a missense, nonsense, or frameshift mutation in the other allele of the same gene <a href="#pone.0000918-Engel2">[4]</a>. Other less frequent defects involve splice junctions <a href="#pone.0000918-Ohno4">[9]</a>, promoter regions <a href="#pone.0000918-Nichols1">[10]</a>, chromosomal micro-deletions <a href="#pone.0000918-Muller1">[11]</a>, and intronic areas outside the splice junction consensuses <a href="#pone.0000918-Muller2">[12]</a>. In addition, with few exceptions, mutations responsible for CMSs are private, so that considerable effort is required to detect the mutation or mutations present in each individual. Furthermore, only a few phenotypic clues can point to mutations in one or a limited number of genes <a href="#pone.0000918-Beeson2">[13]</a>. In the absence of these clues, determining the exact genetic causes of CMS in each patient requires that all genes linked to CMSs be amplified and sequenced, a labor and time-intensive undertaking. Thus, there is a real need for a high-throughput technique to efficiently screen the DNA sequences of genes associated with CMSs.</p>
<div class="figure" id="pone-0000918-t001"><div class="img"><a name="pone-0000918-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000918" data-uri="info:doi/10.1371/journal.pone.0000918.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000918.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000918.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Genes associated with congenital myasthenic syndromes.</span></strong></p><span>doi:10.1371/journal.pone.0000918.t001</span></div><a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Sequence analysis based on custom resequencing microarrays has recently emerged as a powerful strategy for screening mutations in multiple genes linked to a common phenotype <a href="#pone.0000918-Mandal1">[14]</a>–<a href="#pone.0000918-Xu1">[16]</a>. This report describes our design and evaluation of a resequencing microarray for mutational analysis of CMSs. We find that with respect to the detection of missense mutations, our microarray performs well. Moreover, it exhibits high specificity and reproducibility. However, this technology is not able to efficiently assay indels. We therefore suggest that resequencing microarrays be employed for mutational analysis after common indels have been screened by capillary sequencing.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Resequencing Microarray Design</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Our microarray was designed to sequence all exons and 8 base pairs (bp) of flanking intronic regions from the splice junctions of <em>CHRNA1</em>, <em>CHRNB1</em>, <em>CHRND</em>, <em>RAPSN</em>, <em>COLQ</em>, <em>CHAT</em>, and <em>MUSK</em> (<a href="#pone-0000918-t001">Table 1</a>). Additionally, 250 bp of the <em>RAPSN</em> and <em>CHRNE</em> promoter regions as well as the entire genomic sequence of <em>CHRNE</em> were tiled on the microarray. These latter sequences were added because promoter mutations and exonic mutations have been reported in <em>RAPSN</em> <a href="#pone.0000918-Ohno3">[5]</a>, <a href="#pone.0000918-Mandal1">[14]</a>, and promoter, exonic, and intronic mutations have been reported in <em>CHRNE</em> <a href="#pone.0000918-Ohno3">[5]</a>, <a href="#pone.0000918-Nichols1">[10]</a>, <a href="#pone.0000918-Muller2">[12]</a>. The sequence for each gene was obtained from GenBank (see <a href="#pone.0000918.s001">Table S1</a>) and subjected to Repeat Masker (Institute for Systems Biology, Seattle, WA), a program that identifies repetitive elements (e.g. <em>SINEs</em>, <em>LINEs</em>, and <em>ALU</em>s) and internal duplications. In addition, because the association between CMS and <em>DOK7</em> mutations was not known at the time of the design, this gene was not included in the microarray (<a href="#pone-0000918-t001">Table 1</a>).</p>


<h4>Subjects</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The sensitivity of the microarray was determined using DNA from 21 CMS patients possessing mutations previously characterized by capillary sequencing. In addition, both the specificity and reproducibility of the microarray were determined using DNA from 5 healthy individuals. This study was approved by the Institutional Review Board of the University of California, Davis. All subjects were informed of their rights and the details of the research, and they all signed an ‘informed consent’ form.</p>


<h4>DNA extraction and PCR</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>DNA was extracted from blood samples using the QIAamp DNA Blood Mini Kit (Qiagen, Valencia, CA). We used a combination of traditional PCR and long distance PCR to reduce the overall number of reactions required. The size of the PCR amplicons ranged from 170 bp to nearly 13 kb. All primers were designed using Primer3 (<a href="http://frodo.wi.mit.edu/cgi-bin/primer3/primer3_www.cgi">http://frodo.wi.mit.edu/cgi-bin/primer3/​primer3_www.cgi</a>). Primer sequences and reaction conditions are available upon request.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>A 7.5 kb plasmid (IQ-EX) included in the manufacturer's assay (GeneChip® Resequencing Assay Kit, Affymetrix, Santa Clara, CA, USA) was amplified according to the manufacturer's instructions and was used as a positive internal control.</p>


<h4>Quantitation, pooling, fragmentation, and labeling of products</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The PCR products were purified of residual reagents using a PCR purification kit (Qiagen) according to the manufacturer's instructions. The DNA concentration of each purified product was measured (ng/µl) (NanoDrop Technologies, Wilmington, DE). After calculating the molarity of each sample, equimolar amounts of the products were pooled to achieve even hybridization across the microarray.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The MicroArray Core Facility at the UC Davis Medical Center (Sacramento, CA) M.I.N.D. Institute performed all of the experimental procedures for the arrays. The pooled PCR products were fragmented using Fragmentation Reagent (0.15U Dnase We/ug DNA, GeneChip® Resequencing Assay Kit, Affymetrix) at 37°C for 15 minutes, followed by inactivation at 95°C for 15 minutes. Pooled and fragmented PCR products were end-labeled using a biotin-labeling reagent (GeneChip® DNA Labeling Reagent, 5 mM, Affymetrix) and terminal deoxynucleotidyl transferase (TdT, Affymetrix) at 37°C for 2 hours, followed by inactivation at 95°C for 15 minutes.&nbsp;The amplified plasmid control (IQ-EX) was fragmented and labeled for use in the hybridization cocktail.</p>


<h4>Microarray Hybridization and Analysis</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Hybridization was performed according to the manufacturer's protocol. The microarrays were placed in a Hybridization Oven 640 (Affymetrix) at 45°C for 16 hours, rotating at 60 RPM. The arrays were then washed and stained on a fluidics station using the manufacturer's wash and stain protocol (DNAARRAY_WS4_450). They were subsequently scanned on a GeneChip® 3000 Scanner (Affymetrix), and the data from each scan were analyzed using GeneChip® Resequencing Analysis Software, Version 4.0 (GSEQ v4.0, Affymetrix). See <a href="#pone-0000918-g001">Figure 1</a> for an example of the software output.</p>
<div class="figure" id="pone-0000918-g001"><div class="img"><a name="pone-0000918-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000918" data-uri="info:doi/10.1371/journal.pone.0000918.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000918.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000918.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Conversion of microarray probe intensities into sequence by GSEQ v4.0: an illustration of software output.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>A. A digital color image of the scanned data for the entire microarray. The sense and antisense strands of the DNA fragment are highlighted. B. A close-up view of part of the sense strand highlighted in A. C. The converted sequence displayed in B. D. Probe intensity of the heterozygous site at position 52 shown in C.</p>
<span>doi:10.1371/journal.pone.0000918.g001</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>This software uses an algorithm based on the Adaptive Background genotype Calling Scheme (ABACUS) created by Cutler et al. (2001; <a href="#pone.0000918-Cutler1">[18]</a>). The algorithm allows for 11 possible models: A, C, G, T, AC, AG, AT, CG, CT, GT, and no-call (N). It calculates the likelihood of each model representing the observed data independently for the forward and reverse strands of each position, and it uses these data to calculate an overall likelihood of a particular model fitting the data for that position. There are three results that can arise from the overall likelihood calculation for a particular site: a near perfect fit where the forward and reverse strand fit the same model, an imperfect fit where data from one strand fit the model well but data from the other do not, or a no-call where no model fits the data from either strand. Once an initial call is made, the data must fit reliability rules to ensure the data are reliable; the user has control over certain settings of the reliability rules. A final call is then made for each position. Because some of the reliability rules require that data from each sample be compared to data from other samples at the same position, it is recommended that a minimum of 15 microarrays be analyzed together for optimal algorithm performance.</p>


<h4>Capillary Sequencing</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To initially identify the mutations carried by each patient as well as to estimate the percentage agreement between the capillary and microarray sequencing techniques, DNA was amplified as previously described and then sequenced at the UC Davis Division of Biological Sciences DNA Sequencing Facility (Davis, CA) using an ABI 3730 DNA Analyzer (Applied Biosystems, Foster City, CA).</p>


<h4>Analysis</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>The sensitivity of the microarray methodology with respect to the known pathogenic mutations was defined as the proportion of true positives detected by the microarray, while the specificity was defined as the proportion of true negatives detected. Additionally, the reproducibility was defined as the proportion of identical calls made across the five microarrays assayed with identical DNA. Finally, the percentage agreement between the microarray data and the data produced by capillary sequencing represents the proportion of identical calls between the two methods.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Design of the microarray</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Screening with RepeatMasker indicated that no repetitive elements or internal duplications were present in the regions resequenced by the microarray. The total number of base pairs resequenced by the microarray was 24,056; 22,214 bp represented genomic sequence covering the eight genes included in the array, and 1,842 bp corresponded to common mutations tiled in duplicate (<a href="#pone-0000918-t001">Tables 1</a> and <a href="#pone-0000918-t002">2</a>). An 814 bp internal control, representing the 7.5 kb plasmid control (IQ-EX) provided by the manufacturer was also tiled on the microarray as a means of evaluating individual microarray performance. The microarrays were designed with 25 by 20 micron feature size.</p>


<h4>Sequencing with the CMS1 microarray</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>DNA from 26 individuals (21 patients and 5 controls) was sequenced, and for one control, an additional five microarrays were used to determine the reproducibility of the resequencing data. Therefore, a total of 745,736 bases were sequenced across the 31 arrays (21 patient arrays and 10 control arrays; <a href="#pone-0000918-t002">Table 2</a>). The sequence analysis software assigned calls to 698,059 of these bases for an overall call rate of 93.61% (<a href="#pone-0000918-t002">Table 2</a>). Call rates for individual microarrays varied from 92.14%–94.87%. The median GC content of sites assigned a no-call designation (N) for all microarrays was 66%, while the GC content of the entire sequence tiled on the microarray was 57%, a significant difference (<em>p&lt;</em>0.01, t-test). A significant negative correlation was also detected between GC content and the median call rate (R<sup>2</sup> = 0.0873, <em>p&lt;</em>0.01; see <a href="#pone-0000918-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0000918-g002"><div class="img"><a name="pone-0000918-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000918" data-uri="info:doi/10.1371/journal.pone.0000918.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000918.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000918.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Relationship between GC content and call rate.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>The median call rate of each fragment across all microarrays is plotted against the GC content of each fragment.</p>
<span>doi:10.1371/journal.pone.0000918.g002</span></div><div class="figure" id="pone-0000918-t002"><div class="img"><a name="pone-0000918-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000918" data-uri="info:doi/10.1371/journal.pone.0000918.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000918.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000918.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Summary of the microarray data.</span></strong></p><span>doi:10.1371/journal.pone.0000918.t002</span></div>

<h4>Findings in patients</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>According to the capillary sequence data, the 21 patients in this study exhibit a total of 30 mutations, including 23 missense, 4 insertion, and 3 deletion mutations (<a href="#pone.0000918.s002">Table S2</a>). The microarray was able to detect 22 of the missense mutations and none of the insertion or deletion mutations (<a href="#pone.0000918.s002">Table S2</a>). Of the 21 patients, 13 were accurately detected as positive for their respective mutations. All 13 patients were either homozygous or heterozygous for missense mutations. Four additional patients carried one missense mutation and one insertion or deletion mutation, and the missense mutations in these four patients were accurately detected. Three of the 21 patients carried one or two insertion or deletion mutations, none of which was detected by the microarray. Finally, one of the 21 patients was heterozygous for a missense mutation which was not detected by the microarray (<a href="#pone.0000918.s002">Table S2</a>). Overall, the sensitivity of the microarray was estimated to be 73.3%, while the sensitivity of the microarray with respect to missense mutations was estimated to be 95.65% (22/23 detected; <a href="#pone-0000918-t002">Table 2</a>).</p>


<h4>Findings in controls</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>No pathogenic mutations were detected among the five healthy controls in this study, even though multiple SNPs were detected. Interestingly, a unique nonsynonymous variant was found in a single healthy control. This variant causes an arginine to tryptophan replacement (at site 22 of the amino acid sequence) and was confirmed by capillary sequencing. Although this is a highly conserved residue across mammals, there is no indication this mutation is pathogenic, and the individual was considered to have been correctly identified as a true healthy control. As a result, the specificity of the microarray was 100% (<a href="#pone-0000918-t002">Table 2</a>).</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>To assess the reproducibility of the microarray approach, five microarrays were hybridized to DNA from the same individual. Calls were assigned to 113,474 of the 120,280 tiled bases, yielding an overall call rate of 94.34% (<a href="#pone-0000918-t002">Table 2</a>). Call rates for each of the five microarrays varied from 93.81%–94.97%, and the average number of calls per microarray was 22,695 out of 24,056. A single discrepancy was found among the five microarrays: two of the microarrays were called ‘G ’, matching the reference sequence, two others were called heterozygotes for ‘A’ and ‘G’, and the last microarray showed a no-call in this position. One discrepancy among an average of 22,695 base calls yielded a reproducibility estimate of 99.996% (<a href="#pone-0000918-t002">Table 2</a>).</p>


<h4>SNPs and comparison between techniques</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Heterozygosity estimates have revealed that, on average, SNPs occur at a frequency of 1 in every 1000 bases across the human genome <a href="#pone.0000918-Sachidanandam1">[17]</a>. For this study, a total of 102 SNPs were detected by the microarray, including the 22 missense mutations considered to be pathogenic (<a href="#pone-0000918-t003">Table 3</a>). Of the 80 remaining SNPs, 37 have been documented in the GenBank SNP database (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db%E2%80%8A=%E2%80%8ASnp">http://www.ncbi.nlm.nih.gov/entrez/query​.fcgi?db= Snp</a>; see <a href="#pone.0000918.s003">Table S3</a>). Using the 80, nonpathogenic SNPs, the expected heterozygosity (often referred to as ‘pi’) per site per gene region was estimated (see <a href="#pone-0000918-t003">Table 3</a>). Although there is variation across gene regions, these estimates are close to the genome-wide average of ~0.001 (<a href="#pone-0000918-t003">Table 3</a>).</p>
<div class="figure" id="pone-0000918-t003"><div class="img"><a name="pone-0000918-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000918" data-uri="info:doi/10.1371/journal.pone.0000918.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000918.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000918.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000918.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000918.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000918.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Description of SNPs detected.</span></strong></p><span>doi:10.1371/journal.pone.0000918.t003</span></div><a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>To determine how well sequence data from the microarray correspond to sequence data from capillary sequencing, 96,686 base calls from across 26 microarrays (21 patients and 5 controls) were compared to capillary sequence data at the same positions. This degree of capillary sequencing allowed us to assess agreement between the two methods for 70 of the 80 SNPs. Of the remaining 10 SNPs, five were verified electronically using the GenBank SNP database (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db%E2%80%8A=%E2%80%8ASnp">http://www.ncbi.nlm.nih.gov/entrez/query​.fcgi?db= Snp</a>). It remains to be determined whether the remaining five SNPs are detected by both sequencing approaches.</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>A total of 20 discrepant sites were found between the microarray and capillary sequencing data (<a href="#pone-0000918-t003">Table 3</a>). All of these sites involved SNPs detected by the microarray but not by capillary sequencing, and seven of the sites were singletons; that is, they were detected in just a single microarray. Interestingly, six additional SNPs were detected exclusively by the capillary sequencing method, one of which was a singleton; we did not consider these six sites to represent discrepancies, however, as they were called ‘N’s by the microarray (<a href="#pone-0000918-t003">Table 3</a>). None of the discrepant sites called by the microarray is documented in the SNP database, whereas three of the six sites detected by capillary sequencing are corroborated in the database. Overall, 96,635 of the 96,686 assayed bases were identical between the two methods. Therefore, the percentage agreement was 99.947%. However, with respect to sites segregating for SNPs, the percentage agreement between these two methods was only 73.3% (55/75 SNPs; these 55 SNPs are reported in <a href="#pone.0000918.s003">Table S3</a>). Despite this degree of disagreement, estimates of heterozygosity do not change appreciably when using SNP information from the capillary sequencing method (data not shown).</p>
<a id="article1.body1.sec3.sec5.p4" name="article1.body1.sec3.sec5.p4"></a><p>For all of the discrepant sites, the raw trace data from both sequencing techniques were examined. It appears that many of the SNPs called by the microarray software, but not by the capillary technique, were in regions of poor data quality (i.e. several surrounding sites called Ns). In fact, most of the polymorphisms detected in regions of poor data quality were not compelling when examining the raw data, and for many SNP sites, just one of the strands was of good quality (both strands were sequenced for both methods). However, for 8 of the 20 total discrepant sites, convincing and contradictory data were found for both techniques. One approach to addressing this conflict is to employ a third and independent methodology, such as SSCP (single-stranded conformation polymorphism), to re-assay these regions. Alternatively, comparing only data that are validated by a predetermined quality score for both techniques may reveal many fewer, if any, discrepancies <a href="#pone.0000918-Cutler1">[18; see below]</a>.</p>
<a id="article1.body1.sec3.sec5.p5" name="article1.body1.sec3.sec5.p5"></a><p>As another approach to assessing the quality of the microarray data, it was determined whether the 80 SNPs exhibited Hardy-Weinberg genotype proportions across the sample. Six of the SNPs were found to deviate significantly from Hardy-Weinberg expectations (<em>i.e</em>. χ<sup>2</sup>&gt;3.84, <em>p&lt;</em>0.05). To address this, these six loci were re-sequenced in all 26 individuals using capillary sequencing. Overall, departures from Hardy-Weinberg expectations can be explained by the failure of the microarray software to detect all of the heterozygous individuals, either because it called these sites ‘Ns’, or because these sites were called homozygous. In fact, when the genotype proportions provided by the capillary sequence data are used, these SNPs were all found to be in Hardy-Weinberg Equilibrium (data not shown).</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Resequencing microarrays provide a rapid and cost effective method for screening mutations in genetically heterogeneous diseases such as CMSs. Indeed, this technique uses many fewer PCR reactions to amplify and sequence long segments of genomic DNA, and it assays multiple genes using a single hybridization reaction. In this study, we were able to analyze more than 24 kilobases of gene regions linked to CMSs from the products of just 35 PCR reactions. In contrast, a capillary sequencing approach requires greater than 100 PCR reactions. Moreover, our resequencing microarray exhibited a high degree of sensitivity with respect to the detection of missense mutations, its average call rate exceeded the 90% rate guaranteed by the manufacturer, it was highly reproducible, and it showed a high level of agreement with the data produced by capillary sequencing.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Although other groups have also reported high performance from custom resequencing microarrays <a href="#pone.0000918-Mandal1">[14]</a>–<a href="#pone.0000918-Xu1">[16]</a>, there are limitations associated with the technology. For example, the call rate of different resequencing microarray designs may vary considerably. In fact, studies have reported anywhere from fewer than 50% of their microarrays achieving a call rate of greater than 90% to nearly 100% of their microarrays achieving a call rate of 97.5% <a href="#pone.0000918-Tengs1">[15]</a>, <a href="#pone.0000918-Xu1">[16]</a>. Importantly, some of this variability can be explained by differences in the user-chosen settings of the CustomSeq™ Algorithm. For example, settings for data filters that capture features with either minimal intensity or intensity approaching the saturation level of the detector can be changed by the user to be more stringent or more relaxed. If the settings are more relaxed, fewer sites will be assigned as no-calls by the filter, but the calls made will be less reliable. If the settings are more stringent, more sites will be assigned as no-calls by the filter and the overall call rate will be reduced, but calls made will be more reliable. Depending on the particular microarray as well as the goals of the study, the optimal set of parameters will vary.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The GC content of a region may also affect the call rate. In fact, our finding that the call rate of the fragments tiled on our microarray decreased as the GC content increased (see <a href="#pone-0000918-g002">Figure 2</a>) corroborates the result of another study, in which 98.4% of the bases assigned as no-calls were either G or C <a href="#pone.0000918-Mandal1">[14]</a>. It is possible that GC-rich probes bind more strongly to the target DNA, thereby increasing the chance of signal saturation and a no-call at a particular position. Interestingly, Cutler et al. (2001; <a href="#pone.0000918-Cutler1">[18]</a>) found that fluorescence intensity declines with the G-richness of a probe, leading to a lower call rate due to the decreased reliability of such probes. However, another study found no correlation between probe content and call rates <a href="#pone.0000918-Huentelman1">[19]</a>. Clearly, additional data and analysis are required to understand the relationship, if any, between these two variables.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Despite the high percentage of agreement found in our study, the agreement between the microarray and capillary sequencing techniques with respect to sites segregating for SNPs was only 73.3%. This number is much lower than the overall agreement, in part, because the level of variation in humans tends to be low (i.e. there are many more invariant sites; also see <a href="#pone-0000918-t003">Table 3</a> for heterozygosity estimates). Consequently, using the microarray technology for high throughput SNP discovery and genotyping has been controversial, and efforts have been made to identify the sites with the highest likelihood of being correct (see <a href="#pone.0000918-Cutler1">[18]</a>, and references therein). As mentioned above, this can be accomplished by increasing the stringency settings of the calling algorithm <a href="#pone.0000918-Cutler1">[18]</a>. Accompanying an increase in the reliability of each call, however, is a reduction in the overall call rate. In fact, in their paper introducing the ABACUS algorithm, Cutler et al. (2001; <a href="#pone.0000918-Cutler1">[18]</a>) found that only ~80% of the sites could be called with these high stringency settings. From our perspective, this may not be an ideal solution, as it is possible that a mutation will be overlooked when screening DNA from a patient with an unknown mutation profile. Such a scenario is of greatest concern for CMS studies when single heterozygous mutations have dominant effects, as in the case of slow-channel syndrome or in rare cases of fast-channel syndrome due to dominant negative mutations <a href="#pone.0000918-Webster1">[8]</a>. Undoubtedly, the trade-offs between call rate and call quality will continue to be an important issue surrounding the use of resequencing microarrays for mutational analysis.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>Another serious limitation of this technology is that it cannot detect insertion and deletion mutations. This is of particular concern for mutations in <em>CHRNE</em> and <em>DOK7</em>, which together account for a large number of CMS patients, and in which indels are often encountered in the homozygous state. Therefore, we suggest that resequencing microarrays should be used for screening CMS mutations after <em>CHRNE</em> and <em>DOK7</em> have been screened by capillary sequencing. We hope that the increased capability of the recently available, more powerful 100K and 300K microarray platforms will allow the inclusion of probes complementary to common insertions and deletions at each sequence position to overcome this technological limitation. In fact, such an approach has recently been shown to be feasible <a href="#pone.0000918-Karaman1">[20]</a>. Alternatively, as reviewed by Hacia (1999; <a href="#pone.0000918-Hacia1">[21]</a>), heterozygous indels have been successfully detected when using a loss-of-signal hybridization approach (in contrast to the gain-of-signal approach used in this study). By definition, this approach can only identify the presence of indels; however, capillary sequencing can then be used to determine the actual sequence changes. Clearly, the technology or combination of technologies with the highest power of detection will depend on the particular genes being assayed as well as the goals of the study.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000918"></div><a name="pone.0000918.s001" id="pone.0000918.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000918.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Genomic Sequences from Genbank.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000918.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000918.s002" id="pone.0000918.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000918.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Missense and indel mutations carried by the 21 studied patients.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000918.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0000918.s003" id="pone.0000918.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000918.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Information on the 55 validated SNPs (also see <a href="#pone-0000918-t003">Table 3</a>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000918.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.14 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Dr. Ruth Ballard and two anonymous reviewers for useful comments and suggestions. We also thank Dr. Christopher Gomez for his referral of patients with slow-channel syndrome. Finally, we thank Jarae Ng, Orla Cagney, Josh DeClerq, and Anna Szeto for their assistance with PCR and capillary sequencing.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RM LD. Performed the experiments: LD RD JK. Analyzed the data: JA LD. Contributed reagents/materials/analysis tools: JG. Wrote the paper: JA RM LD.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000918-Engel1" id="pone.0000918-Engel1"></a>Engel AG, Sine SM (2005) Current understanding of congenital myasthenic syndromes. Curr Opin Pharmacol  5: 308–321.  <ul class="find" data-citedArticleID="994422" data-doi="10.1016/j.coph.2004.12.007"><li><a href="http://dx.doi.org/10.1016/j.coph.2004.12.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Current+understanding+of+congenital+myasthenic+syndromes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Current+understanding+of+congenital+myasthenic+syndromes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000918-Ohno1" id="pone.0000918-Ohno1"></a>Ohno K, Tsujino A, Brengman JM, Harper CM, Bajzer Z, et al.  (2001) Choline acetyltransferase mutations cause myasthenic syndrome associated with episodic apnea in humans. Proc Natl Acad Sci U S A  98: 2017–2022.  <ul class="find" data-citedArticleID="994440" data-doi="10.1073/pnas.98.4.2017"><li><a href="http://dx.doi.org/10.1073/pnas.98.4.2017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Choline+acetyltransferase+mutations+cause+myasthenic+syndrome+associated+with+episodic+apnea+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Choline+acetyltransferase+mutations+cause+myasthenic+syndrome+associated+with+episodic+apnea+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000918-Ohno2" id="pone.0000918-Ohno2"></a>Ohno K, Brengman J, Tsujino A, Engel AG (1998) Human endplate acetylcholinesterase deficiency caused by mutations in the collagen-like tail subunit (<em>COLQ</em>) of the asymmetric enzyme. Proc Natl Acad Sci U S A  95: 9654–9659.  <ul class="find" data-citedArticleID="994442" data-doi="10.1073/pnas.95.16.9654"><li><a href="http://dx.doi.org/10.1073/pnas.95.16.9654" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+endplate+acetylcholinesterase+deficiency+caused+by+mutations+in+the+collagen-like+tail+subunit+%28COLQ%29+of+the+asymmetric+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+endplate+acetylcholinesterase+deficiency+caused+by+mutations+in+the+collagen-like+tail+subunit+%28COLQ%29+of+the+asymmetric+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000918-Engel2" id="pone.0000918-Engel2"></a>Engel AG, Ohno K, Sine SM (2003) Congenital myasthenic syndromes: progress over the past decade. Muscle Nerve  27: 4–25.  <ul class="find" data-citedArticleID="994424" data-doi="10.1002/mus.10269"><li><a href="http://dx.doi.org/10.1002/mus.10269" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Congenital+myasthenic+syndromes%3A+progress+over+the+past+decade." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Congenital+myasthenic+syndromes%3A+progress+over+the+past+decade.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000918-Ohno3" id="pone.0000918-Ohno3"></a>Ohno K, Engel AG, Shen XM, Selcen D, Brengman J, et al.  (2002) Rapsyn mutations in humans cause endplate acetylcholine-receptor deficiency and myasthenic syndrome. Am J Hum Genet  70: 875–885.  <ul class="find" data-citedArticleID="994444" data-doi="10.1086/339465"><li><a href="http://dx.doi.org/10.1086/339465" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapsyn+mutations+in+humans+cause+endplate+acetylcholine-receptor+deficiency+and+myasthenic+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapsyn+mutations+in+humans+cause+endplate+acetylcholine-receptor+deficiency+and+myasthenic+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000918-Chevessier1" id="pone.0000918-Chevessier1"></a>Chevessier F, Faraut B, Ravel-Chapuis A, Richard P, Gaudon K, et al.  (2004) <em>MUSK</em>, a new target for mutations causing congenital myasthenic syndrome. Hum Mol Genet  13: 3229–3240.  <ul class="find" data-citedArticleID="994418" data-doi="10.1093/hmg/ddh333"><li><a href="http://dx.doi.org/10.1093/hmg/ddh333" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUSK%2C+a+new+target+for+mutations+causing+congenital+myasthenic+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUSK%2C+a+new+target+for+mutations+causing+congenital+myasthenic+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000918-Beeson1" id="pone.0000918-Beeson1"></a>Beeson D, Higuchi O, Palace J, Cossins J, Spearman H, et al.  (2006) Dok-7 mutations underlie a neuromuscular junction synaptopathy. Science  313: 1975–1978.  <ul class="find" data-citedArticleID="994414" data-doi="10.1126/science.1130837"><li><a href="http://dx.doi.org/10.1126/science.1130837" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dok-7+mutations+underlie+a+neuromuscular+junction+synaptopathy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dok-7+mutations+underlie+a+neuromuscular+junction+synaptopathy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000918-Webster1" id="pone.0000918-Webster1"></a>Webster R, Brydson M, Croxen R, Newsom-Davis J, Vincent A, et al.  (2004) Mutation in the AChR ion channel gate underlies a fast channel congenital myasthenic syndrome. Neurology  62: 1090–1096.  <ul class="find" data-citedArticleID="994452" data-doi="10.1212/01.wnl.0000118205.99701.41"><li><a href="http://dx.doi.org/10.1212/01.wnl.0000118205.99701.41" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+in+the+AChR+ion+channel+gate+underlies+a+fast+channel+congenital+myasthenic+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+in+the+AChR+ion+channel+gate+underlies+a+fast+channel+congenital+myasthenic+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000918-Ohno4" id="pone.0000918-Ohno4"></a>Ohno K, Engel AG (2005) Splicing abnormalities in congenital myasthenic syndromes. Acta Myol  24: 50–54.  <ul class="find" data-citedArticleID="994446"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Splicing+abnormalities+in+congenital+myasthenic+syndromes.&amp;auth=&amp;atitle=Splicing+abnormalities+in+congenital+myasthenic+syndromes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Splicing+abnormalities+in+congenital+myasthenic+syndromes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Splicing+abnormalities+in+congenital+myasthenic+syndromes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000918-Nichols1" id="pone.0000918-Nichols1"></a>Nichols P, Croxen R, Vincent A, Rutter R, Hutchinson M, et al.  (1999) Mutation of the acetylcholine receptor epsilon-subunit promoter in congenital myasthenic syndrome. Ann Neurol  45: 439–443.  <ul class="find" data-citedArticleID="994438" data-doi="10.1002/1531-8249(199904)45:4<439::aid-ana4&gt;3.3.co;2-n"><li><a href="http://dx.doi.org/10.1002/1531-8249(199904)45:4<439::aid-ana4&gt;3.3.co;2-n" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+of+the+acetylcholine+receptor+epsilon-subunit+promoter+in+congenital+myasthenic+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+of+the+acetylcholine+receptor+epsilon-subunit+promoter+in+congenital+myasthenic+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000918-Muller1" id="pone.0000918-Muller1"></a>Muller JS, Abicht A, Christen HJ, Stucka R, Schara U, et al.  (2004) A newly identified chromosomal microdeletion of the rapsyn gene causes a congenital myasthenic syndrome. Neuromuscul Disord  14: 744–749.  <ul class="find" data-citedArticleID="994434" data-doi="10.1016/j.nmd.2004.06.010"><li><a href="http://dx.doi.org/10.1016/j.nmd.2004.06.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+newly+identified+chromosomal+microdeletion+of+the+rapsyn+gene+causes+a+congenital+myasthenic+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+newly+identified+chromosomal+microdeletion+of+the+rapsyn+gene+causes+a+congenital+myasthenic+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000918-Muller2" id="pone.0000918-Muller2"></a>Muller JS, Stucka R, Neudecker S, Zierz S, Schmidt C, et al.  (2005) An intronic base alteration of the <em>CHRNE</em> gene leading to a congenital myasthenic syndrome. Neurology  65: 463–465.  <ul class="find" data-citedArticleID="994436" data-doi="10.1212/01.wnl.0000172346.26219.fd"><li><a href="http://dx.doi.org/10.1212/01.wnl.0000172346.26219.fd" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+intronic+base+alteration+of+the+CHRNE+gene+leading+to+a+congenital+myasthenic+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+intronic+base+alteration+of+the+CHRNE+gene+leading+to+a+congenital+myasthenic+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000918-Beeson2" id="pone.0000918-Beeson2"></a>Beeson D, Hantai D, Lochmuller H, Engel AG (2005) 126th International Workshop: congenital myasthenic syndromes, 24-26 September 2004, Naarden, the Netherlands. Neuromuscul Disord  15: 498–512.  <ul class="find" data-citedArticleID="994416" data-doi="10.1016/j.nmd.2005.05.001"><li><a href="http://dx.doi.org/10.1016/j.nmd.2005.05.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=126th+International+Workshop%3A+congenital+myasthenic+syndromes%2C+24-26+September+2004%2C+Naarden%2C+the+Netherlands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22126th+International+Workshop%3A+congenital+myasthenic+syndromes%2C+24-26+September+2004%2C+Naarden%2C+the+Netherlands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000918-Mandal1" id="pone.0000918-Mandal1"></a>Mandal MN, Heckenlively JR, Burch T, Chen L, Vasireddy V, et al.  (2005) Sequencing arrays for screening multiple genes associated with early-onset human retinal degenerations on a high-throughput platform. Invest Ophthalmol Vis Sci  46: 3355–3362.  <ul class="find" data-citedArticleID="994432" data-doi="10.1167/iovs.05-0007"><li><a href="http://dx.doi.org/10.1167/iovs.05-0007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+arrays+for+screening+multiple+genes+associated+with+early-onset+human+retinal+degenerations+on+a+high-throughput+platform." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+arrays+for+screening+multiple+genes+associated+with+early-onset+human+retinal+degenerations+on+a+high-throughput+platform.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000918-Tengs1" id="pone.0000918-Tengs1"></a>Tengs T, Lee JC, Guillermo Paez J, Zhao X, Laframboise T, et al.  (2005) A transforming MET mutation discovered in non-small cell lung cancer using microarray-based resequencing. Cancer Lett  229: 227–233.  <ul class="find" data-citedArticleID="994450" data-doi="10.1016/j.canlet.2005.08.007"><li><a href="http://dx.doi.org/10.1016/j.canlet.2005.08.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+transforming+MET+mutation+discovered+in+non-small+cell+lung+cancer+using+microarray-based+resequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+transforming+MET+mutation+discovered+in+non-small+cell+lung+cancer+using+microarray-based+resequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000918-Xu1" id="pone.0000918-Xu1"></a>Xu N, Podolsky RH, Chudgar P, Chorich LP, Liu C, et al.  (2005) Screening candidate genes for mutations in patients with hypogonadotropic hypogonadism using custom genome resequencing microarrays. Am J Obstet Gynecol  192: 1274–1282.  <ul class="find" data-citedArticleID="994454" data-doi="10.1016/j.ajog.2004.12.066"><li><a href="http://dx.doi.org/10.1016/j.ajog.2004.12.066" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Screening+candidate+genes+for+mutations+in+patients+with+hypogonadotropic+hypogonadism+using+custom+genome+resequencing+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Screening+candidate+genes+for+mutations+in+patients+with+hypogonadotropic+hypogonadism+using+custom+genome+resequencing+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000918-Sachidanandam1" id="pone.0000918-Sachidanandam1"></a>Sachidanandam R, Weissman D, Schmidt SC, Kakol JM, Stein LD, et al.  (2001) A map of human genome sequence variation containing 1.42 million single nucleotide polymorphisms. Nature  409: 928–933.  <ul class="find" data-citedArticleID="994448" data-doi="10.1038/35057149"><li><a href="http://dx.doi.org/10.1038/35057149" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+map+of+human+genome+sequence+variation+containing+1.42+million+single+nucleotide+polymorphisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+map+of+human+genome+sequence+variation+containing+1.42+million+single+nucleotide+polymorphisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000918-Cutler1" id="pone.0000918-Cutler1"></a>Cutler DJ, Zwick ME, Carrasquillo MM, Yohn CT, Tobin KP, et al.  (2001) High-throughput variation detection and genotyping using microarrays. Genome Res  11: 1913–1925.  <ul class="find" data-citedArticleID="994420"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=High-throughput+variation+detection+and+genotyping+using+microarrays.&amp;auth=&amp;atitle=High-throughput+variation+detection+and+genotyping+using+microarrays." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-throughput+variation+detection+and+genotyping+using+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-throughput+variation+detection+and+genotyping+using+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000918-Huentelman1" id="pone.0000918-Huentelman1"></a>Huentelman MJ, Craig DW, Shieh AD, Corneveaux JJ, Hu-Lince D, et al.  (2005) SNiPer: improved SNP genotype calling for Affymetrix 10K GeneChip microarray data. BMC Genomics  6: 149.  <ul class="find" data-citedArticleID="994428"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=SNiPer%3A+improved+SNP+genotype+calling+for+Affymetrix+10K+GeneChip+microarray+data.&amp;auth=&amp;atitle=SNiPer%3A+improved+SNP+genotype+calling+for+Affymetrix+10K+GeneChip+microarray+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SNiPer%3A+improved+SNP+genotype+calling+for+Affymetrix+10K+GeneChip+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SNiPer%3A+improved+SNP+genotype+calling+for+Affymetrix+10K+GeneChip+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000918-Karaman1" id="pone.0000918-Karaman1"></a>Karaman MW, Groshen S, Lee CC, Pike BL, Hacia JG (2005) Comparisons of substitution, insertion and deletion probes for resequencing and mutational analysis using oligonucleotide microarrays. Nucl Acids Res  33: e33.  <ul class="find" data-citedArticleID="994430" data-doi="10.1093/nar/gni034"><li><a href="http://dx.doi.org/10.1093/nar/gni034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparisons+of+substitution%2C+insertion+and+deletion+probes+for+resequencing+and+mutational+analysis+using+oligonucleotide+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparisons+of+substitution%2C+insertion+and+deletion+probes+for+resequencing+and+mutational+analysis+using+oligonucleotide+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000918-Hacia1" id="pone.0000918-Hacia1"></a>Hacia JG (1999) Resequencing and mutational analysis using oligonucleotide microarrays. Nat Genet  21: 42–47 (Suppl.).  <ul class="find" data-citedArticleID="994426" data-doi="10.1038/4469"><li><a href="http://dx.doi.org/10.1038/4469" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resequencing+and+mutational+analysis+using+oligonucleotide+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resequencing+and+mutational+analysis+using+oligonucleotide+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.XML" value="67311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.PDF" value="172538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t001.PNG_L" value="143229"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t001.PNG_M" value="76866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t001.PNG_S" value="10124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t001.TIF" value="572988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t001.PNG_I" value="22824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.g001.PNG_L" value="2595264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.g001.PNG_M" value="171233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.g001.PNG_S" value="11978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.g001.TIF" value="4279508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.g001.PNG_I" value="52946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.g002.PNG_L" value="292506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.g002.PNG_M" value="30936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.g002.PNG_S" value="6318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.g002.TIF" value="379558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.g002.PNG_I" value="10210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t002.PNG_L" value="82397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t002.PNG_M" value="115937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t002.PNG_S" value="15073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t002.TIF" value="365408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t002.PNG_I" value="37605"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t003.PNG_L" value="93896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t003.PNG_M" value="51341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t003.PNG_S" value="9372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t003.TIF" value="462402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.t003.PNG_I" value="15885"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.s001.DOC" value="39936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.s002.DOC" value="65024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000918.s003.DOC" value="136704"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000918&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000918" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000918&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000918&volume=&issue=&title=High Throughput Genetic Analysis of Congenital Myasthenic Syndromes Using Resequencing Microarrays&author_name=Lisa%20Denning%2C%20Jennifer%20A.%20Anderson%2C%20Ryan%20Davis%2C%20Jeffrey%20P.%20Gregg%2C%20Jennifer%20Kuzdenyi%2C%20Ricardo%20A.%20Maselli&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000918" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000918" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000918" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000918&amp;t=High%20Throughput%20Genetic%20Analysis%20of%20Congenital%20Myasthenic%20Syndromes%20Using%20Resequencing%20Microarrays" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000918&title=High%20Throughput%20Genetic%20Analysis%20of%20Congenital%20Myasthenic%20Syndromes%20Using%20Resequencing%20Microarrays&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000918&amp;title=High%20Throughput%20Genetic%20Analysis%20of%20Congenital%20Myasthenic%20Syndromes%20Using%20Resequencing%20Microarrays" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000918" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000918&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'High Throughput Genetic Analysis of Congenital Myasthenic Syndromes Using Resequencing Microarrays';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000918';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20High%20Throughput%20Genetic%20Analysis%20of%20Congenital%20Myasthenic%20Syndromes%20Using%20Resequencing%20Microarrays http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000918" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000918" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Capillaries%22" title="Search for articles in the subject area:'Capillaries'"><div class="flagText">Capillaries</div></a>
              <div data-categoryid="48067" data-articleid="24674"
                   data-categoryname="Capillaries"
                   class="flagImage" title="Flag 'Capillaries' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+deletion%22" title="Search for articles in the subject area:'Gene deletion'"><div class="flagText">Gene deletion</div></a>
              <div data-categoryid="33017" data-articleid="24674"
                   data-categoryname="Gene deletion"
                   class="flagImage" title="Flag 'Gene deletion' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Insertion+mutation%22" title="Search for articles in the subject area:'Insertion mutation'"><div class="flagText">Insertion mutation</div></a>
              <div data-categoryid="35247" data-articleid="24674"
                   data-categoryname="Insertion mutation"
                   class="flagImage" title="Flag 'Insertion mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="24674"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Missense+mutation%22" title="Search for articles in the subject area:'Missense mutation'"><div class="flagText">Missense mutation</div></a>
              <div data-categoryid="41093" data-articleid="24674"
                   data-categoryname="Missense mutation"
                   class="flagImage" title="Flag 'Missense mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="24674"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="47289" data-articleid="24674"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="17487" data-articleid="24674"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1251'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2813'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2478&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
