

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Novel OxyR Sensor and Regulator of Hydrogen Peroxide Stress with One Cysteine Residue in Deinococcus radiodurans</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001602"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001602" />

    <meta name="citation_title" content="A Novel OxyR Sensor and Regulator of Hydrogen Peroxide Stress with One Cysteine Residue in Deinococcus radiodurans"/>
    <meta itemprop="name" content="A Novel OxyR Sensor and Regulator of Hydrogen Peroxide Stress with One Cysteine Residue in Deinococcus radiodurans"/>

      <meta name="citation_author" content="Huan Chen"/>
            <meta name="citation_author_institution" content="Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China"/>
            <meta name="citation_author_institution" content="James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, China"/>
      <meta name="citation_author" content="Guangzhi Xu"/>
            <meta name="citation_author_institution" content="Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China"/>
      <meta name="citation_author" content="Ye Zhao"/>
            <meta name="citation_author_institution" content="Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China"/>
      <meta name="citation_author" content="Bing Tian"/>
            <meta name="citation_author_institution" content="Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China"/>
      <meta name="citation_author" content="Huiming Lu"/>
            <meta name="citation_author_institution" content="Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China"/>
      <meta name="citation_author" content="Xiaomin Yu"/>
            <meta name="citation_author_institution" content="Key Laboratory of Genome Science and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Zhenjian Xu"/>
            <meta name="citation_author_institution" content="Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China"/>
      <meta name="citation_author" content="Nanjiao Ying"/>
            <meta name="citation_author_institution" content="Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China"/>
      <meta name="citation_author" content="Songnian Hu"/>
            <meta name="citation_author_institution" content="James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, China"/>
            <meta name="citation_author_institution" content="Key Laboratory of Genome Science and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Yuejin Hua"/>
            <meta name="citation_author_institution" content="Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China"/>

    <meta name="citation_date" content="2008/2/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001602.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1602"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Pathways of oxidative damage.; citation_author=JA Imlay; citation_journal_title=Annu Rev Microbiol; citation_volume=57; citation_number=1; citation_pages=395-418; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Activation of the OxyR transcription factor by reversible disulfide bond formation.; citation_author=M Zheng; citation_author=F Aslund; citation_author=G Storz; citation_journal_title=Science; citation_volume=279; citation_number=2; citation_pages=1718-1721; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Positive control of a regulon for defenses against oxidative stress and some heat-shock proteins in Salmonella typhimurium.; citation_author=MF Christman; citation_author=RW Morgan; citation_author=FS Jacobson; citation_author=BN Ames; citation_journal_title=Cell; citation_volume=41; citation_number=3; citation_pages=753-762; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=OxyR: a molecular code for redox sensing?; citation_author=JD Helmann; citation_journal_title=Sci STKE; citation_volume=2002; citation_number=4; citation_pages=PE46; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=OxyR acts as a repressor of catalase expression in Neisseria gonorrhoeae.; citation_author=HJ Tseng; citation_author=AG McEwan; citation_author=MA Apicella; citation_author=MP Jennings; citation_journal_title=Infect Immun; citation_volume=71; citation_number=5; citation_pages=550-556; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Redox regulation of OxyR requires specific disulfide bond formation involving a rapid kinetic reaction path.; citation_author=C Lee; citation_author=SM Lee; citation_author=P Mukhopadhyay; citation_author=SJ Kim; citation_author=SC Lee; citation_journal_title=Nat Struct Mol Biol; citation_volume=11; citation_number=6; citation_pages=1179-1185; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Redox-dependent shift of OxyR-DNA contacts along an extended DNA-binding site: a mechanism for differential promoter selection.; citation_author=MB Toledano; citation_author=I Kullik; citation_author=F Trinh; citation_author=PT Baird; citation_author=TD Schneider; citation_journal_title=Cell; citation_volume=78; citation_number=7; citation_pages=897-909; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=OxyR: a molecular code for redox-related signaling.; citation_author=SO Kim; citation_author=K Merchant; citation_author=R Nudelman; citation_author=WF Beyer; citation_author=T Keng; citation_journal_title=Cell; citation_volume=109; citation_number=8; citation_pages=383-396; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Why is Deinococcus radiodurans so resistant to ionizing radiation?; citation_author=JR Battista; citation_author=AM Earl; citation_author=MJ Park; citation_journal_title=Trends Microbiol; citation_volume=7; citation_number=9; citation_pages=362-365; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=DNA repair in the extremely radioresistant bacterium Deinococcus radiodurans.; citation_author=KW Minton; citation_journal_title=Mol Microbiol; citation_volume=13; citation_number=10; citation_pages=9-15; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Characterization of pathways dependent on the uvsE, uvrA1, or uvrA2 gene product for UV resistance in Deinococcus radiodurans.; citation_author=M Tanaka; citation_author=I Narumi; citation_author=T Funayama; citation_author=M Kikuchi; citation_author=H Watanabe; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=11; citation_pages=3693-3697; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic evidence that the uvsE gene product of Deinococcus radiodurans R1 is a UV damage endonuclease.; citation_author=AM Earl; citation_author=SK Rankin; citation_author=KP Kim; citation_author=ON Lamendola; citation_author=JR Battista; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=12; citation_pages=1003-1009; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Targeted mutagenesis by duplication insertion in the radioresistant bacterium Deinococcus radiodurans: radiation sensitivities of catalase (katA) and superoxide dismutase (sodA) mutants.; citation_author=LM Markillie; citation_author=SM Varnum; citation_author=P Hradecky; citation_author=KK Wong; citation_journal_title=J Bacteriol; citation_volume=181; citation_number=13; citation_pages=666-669; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Analysis of Deinococcus radiodurans's transcriptional response to ionizing radiation and desiccation reveals novel proteins that contribute to extreme radioresistance.; citation_author=M Tanaka; citation_author=AM Earl; citation_author=HA Howell; citation_author=MJ Park; citation_author=JA Eisen; citation_journal_title=Genetics; citation_volume=168; citation_number=14; citation_pages=21-33; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=How radiation kills cells: survival of Deinococcus radiodurans and Shewanella oneidensis under oxidative stress.; citation_author=D Ghosal; citation_author=MV Omelchenko; citation_author=EK Gaidamakova; citation_author=VY Matrosova; citation_author=A Vasilenko; citation_journal_title=FEMS Microbiol Rev; citation_volume=29; citation_number=15; citation_pages=361-375; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Characterization of the OxyR regulon of Neisseria gonorrhoeae.; citation_author=KL Seib; citation_author=HJ Wu; citation_author=YN Srikhanta; citation_author=JL Edwards; citation_author=ML Falsetta; citation_journal_title=Mol Microbiol; citation_volume=63; citation_number=16; citation_pages=54-68; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mutational analysis of the redox-sensitive transcriptional regulator OxyR: regions important for DNA binding and multimerization.; citation_author=I Kullik; citation_author=J Stevens; citation_author=MB Toledano; citation_author=G Storz; citation_journal_title=J Bacteriol; citation_volume=177; citation_number=17; citation_pages=1285-1291; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mutational analysis to define an activating region on the redox-sensitive transcriptional regulator OxyR.; citation_author=X Wang; citation_author=P Mukhopadhyay; citation_author=MJ Wood; citation_author=FW Outten; citation_author=JA Opdyke; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=18; citation_pages=8335-8342; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=OxyR, a positive regulator of hydrogen peroxide-inducible genes in Escherichia coli and Salmonella typhimurium, is homologous to a family of bacterial regulatory proteins.; citation_author=MF Christman; citation_author=G Storz; citation_author=BN Ames; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=86; citation_number=19; citation_pages=3484-3488; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Novel application of 7-chloro-4-nitrobenzo-2-oxa-1,3-diazole to identify cysteine sulfenic acid in the AhpC component of alkyl hydroperoxide reductase.; citation_author=HR Ellis; citation_author=LB Poole; citation_journal_title=Biochemistry; citation_volume=36; citation_number=20; citation_pages=15013-15018; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=DNA microarray-mediated transcriptional profiling of the Escherichia coli response to hydrogen peroxide.; citation_author=M Zheng; citation_author=X Wang; citation_author=LJ Templeton; citation_author=DR Smulski; citation_author=RA LaRossa; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=21; citation_pages=4562-4570; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Regulation of hydrogen peroxide-dependent gene expression in Rhodobacter sphaeroides: regulatory functions of OxyR.; citation_author=T Zeller; citation_author=MA Mraheil; citation_author=OV Moskvin; citation_author=K Li; citation_author=M Gomelsky; citation_journal_title=J Bacteriol; citation_volume=189; citation_number=22; citation_pages=3784-3792; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The OxyR regulon in nontypeable Haemophilus influenzae.; citation_author=A Harrison; citation_author=WC Ray; citation_author=BD Baker; citation_author=DW Armbruster; citation_author=LO Bakaletz; citation_journal_title=J Bacteriol; citation_volume=189; citation_number=23; citation_pages=1004-1012; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Role of oxyR in the oral anaerobe Porphyromonas gingivalis.; citation_author=PI Diaz; citation_author=N Slakeski; citation_author=EC Reynolds; citation_author=R Morona; citation_author=AH Rogers; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=24; citation_pages=2454-2462; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Regulation of Salmonella enterica serovar Typhimurium mntH transcription by H(2)O(2), Fe(2+), and Mn(2+).; citation_author=DG Kehres; citation_author=A Janakiraman; citation_author=JM Slauch; citation_author=ME Maguire; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=25; citation_pages=3151-3158; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of Dps-1, a functionally distinct Dps protein from Deinococcus radiodurans.; citation_author=SG Kim; citation_author=G Bhattacharyya; citation_author=A Grove; citation_author=YH Lee; citation_journal_title=J Mol Biol; citation_volume=361; citation_number=26; citation_pages=105-114; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The crystal structure of the Dps2 from Deinococcus radiodurans reveals an unusual pore profile with a non-specific metal binding site.; citation_author=MG Cuypers; citation_author=EP Mitchell; citation_author=CV Romao; citation_author=SM McSweeney; citation_journal_title=J Mol Biol; citation_volume=371; citation_number=27; citation_pages=787-799; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Transcriptional profile in response to ionizing radiation at low dose in Deinococcus radiodurans.; citation_author=H Chen; citation_author=ZJ Xu; citation_author=B Tian; citation_author=WW Chen; citation_author=SN Hu; citation_journal_title=Progress in Natural Science; citation_volume=17; citation_number=28; citation_pages=525-536; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Accumulation of Mn(II) in Deinococcus radiodurans facilitates gamma-radiation resistance.; citation_author=MJ Daly; citation_author=EK Gaidamakova; citation_author=VY Matrosova; citation_author=A Vasilenko; citation_author=M Zhai; citation_journal_title=Science; citation_volume=306; citation_number=29; citation_pages=1025-1028; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bacillus subtilis contains multiple Fur homologues: identification of the iron uptake (Fur) and peroxide regulon (PerR) repressors.; citation_author=N Bsat; citation_author=A Herbig; citation_author=L Casillas-Martinez; citation_author=P Setlow; citation_author=JD Helmann; citation_journal_title=Mol Microbiol; citation_volume=29; citation_number=30; citation_pages=189-198; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The complex oxidative stress response of Bacteroides fragilis: the role of OxyR in control of gene expression.; citation_author=ER Rocha; citation_author=CD Herren; citation_author=DJ Smalley; citation_author=CJ Smith; citation_journal_title=Anaerobe; citation_volume=9; citation_number=31; citation_pages=165-173; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Role and regulation of the Shigella flexneri sit and MntH systems.; citation_author=L Runyen-Janecky; citation_author=E Dazenski; citation_author=S Hawkins; citation_author=L Warner; citation_journal_title=Infect Immun; citation_volume=74; citation_number=32; citation_pages=4666-4672; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An essential role for free radicals and derived species in signal transduction.; citation_author=HM Lander; citation_journal_title=Faseb J; citation_volume=11; citation_number=33; citation_pages=118-124; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Hydrogen peroxide sensing and signaling.; citation_author=EA Veal; citation_author=AM Day; citation_author=BA Morgan; citation_journal_title=Mol Cell; citation_volume=26; citation_number=34; citation_pages=1-14; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Transcriptome dynamics of Deinococcus radiodurans recovering from ionizing radiation.; citation_author=Y Liu; citation_author=J Zhou; citation_author=MV Omelchenko; citation_author=AS Beliaev; citation_author=A Venkateswaran; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=35; citation_pages=4191-4196; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Preserving genome integrity: the DdrA protein of Deinococcus radiodurans R1.; citation_author=DR Harris; citation_author=M Tanaka; citation_author=SV Saveliev; citation_author=E Jolivet; citation_author=AM Earl; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=36; citation_pages=e304; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification and functional analysis of a phytoene desaturase gene from the extremely radioresistant bacterium Deinococcus radiodurans.; citation_author=Z Xu; citation_author=B Tian; citation_author=Z Sun; citation_author=J Lin; citation_author=Y Hua; citation_journal_title=Microbiology; citation_volume=153; citation_number=37; citation_pages=1642-1652; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Protein oxidation implicated as the primary determinant of bacterial radioresistance.; citation_author=MJ Daly; citation_author=EK Gaidamakova; citation_author=VY Matrosova; citation_author=A Vasilenko; citation_author=M Zhai; citation_journal_title=PLoS Biol; citation_volume=5; citation_number=38; citation_pages=e92; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The OhrR repressor senses organic hydroperoxides by reversible formation of a cysteine-sulfenic acid derivative.; citation_author=M Fuangthong; citation_author=JD Helmann; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=39; citation_pages=6690-6695; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A complex thiolate switch regulates the Bacillus subtilis organic peroxide sensor OhrR.; citation_author=JW Lee; citation_author=S Soonsanga; citation_author=JD Helmann; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=40; citation_pages=8743-8748; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=N-Acetyltransferase Mpr1 confers ethanol tolerance on Saccharomyces cerevisiae by reducing reactive oxygen species.; citation_author=X Du; citation_author=H Takagi; citation_journal_title=Appl Microbiol Biotechnol; citation_volume=75; citation_number=41; citation_pages=1343-1351; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phase variation of Ag43 is independent of the oxidation state of OxyR.; citation_author=A Wallecha; citation_author=J Correnti; citation_author=V Munster; citation_author=M van der Woude; citation_journal_title=J Bacteriol; citation_volume=185; citation_number=42; citation_pages=2203-2209; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Construction of DNA damage response gene pprI function deficient and function complementary mutants in Deinococcus radiodurans.; citation_author=GJ Gao; citation_author=HM Lu; citation_author=LF Huang; citation_author=YJ Hua; citation_journal_title=Chinese Science Bulletin; citation_volume=50; citation_number=43; citation_pages=311-316; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=PprI: a general switch responsible for extreme radioresistance of Deinococcus radiodurans.; citation_author=Y Hua; citation_author=I Narumi; citation_author=G Gao; citation_author=B Tian; citation_author=K Satoh; citation_journal_title=Biochem Biophys Res Commun; citation_volume=306; citation_number=44; citation_pages=354-360; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The PerR regulon in peroxide resistance and virulence of Streptococcus pyogenes.; citation_author=A Brenot; citation_author=KY King; citation_author=MG Caparon; citation_journal_title=Mol Microbiol; citation_volume=55; citation_number=45; citation_pages=221-234; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Paper disk-agar diffusion assay of penicillin in the presence of streptomycin.; citation_author=D Raahave; citation_journal_title=Antimicrob Agents Chemother; citation_volume=6; citation_number=46; citation_pages=603-605; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Three tandem HRDC domains have synergistic effect on the RecQ functions in Deinococcus radiodurans.; citation_author=L Huang; citation_author=X Hua; citation_author=H Lu; citation_author=G Gao; citation_author=B Tian; citation_journal_title=DNA Repair (Amst); citation_volume=6; citation_number=47; citation_pages=167-176; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Chemiluminescence assay for reactive oxygen species scavenging activities and inhibition on oxidative damage of DNA in Deinococcus radiodurans.; citation_author=B Tian; citation_author=Y Wu; citation_author=D Sheng; citation_author=Z Zheng; citation_author=G Gao; citation_journal_title=Luminescence; citation_volume=19; citation_number=48; citation_pages=78-84; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Catalase activity in human healthy and inflamed dental pulps.; citation_author=P Esposito; citation_author=G Varvara; citation_author=S Caputi; citation_author=G Perinetti; citation_journal_title=Int Endod J; citation_volume=36; citation_number=49; citation_pages=599-603; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Characterization of a mammalian peroxiredoxin that contains one conserved cysteine.; citation_author=SW Kang; citation_author=IC Baines; citation_author=SG Rhee; citation_journal_title=J Biol Chem; citation_volume=273; citation_number=50; citation_pages=6303-6311; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Novel organic hydroperoxide-sensing and responding mechanisms for OhrR, a major bacterial sensor and regulator of organic hydroperoxide stress.; citation_author=W Panmanee; citation_author=P Vattanaviboon; citation_author=LB Poole; citation_author=S Mongkolsuk; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=51; citation_pages=1389-1395; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcriptional and proteomic analysis of a ferric uptake regulator (fur) mutant of Shewanella oneidensis: possible involvement of fur in energy metabolism, transcriptional regulation, and oxidative stress.; citation_author=DK Thompson; citation_author=AS Beliaev; citation_author=CS Giometti; citation_author=SL Tollaksen; citation_author=T Khare; citation_journal_title=Appl Environ Microbiol; citation_volume=68; citation_number=52; citation_pages=881-892; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=limmaGUI: a graphical user interface for linear modeling of microarray data.; citation_author=JM Wettenhall; citation_author=GK Smyth; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=53; citation_pages=3705-3706; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Use of within-array replicate spots for assessing differential expression in microarray experiments.; citation_author=GK Smyth; citation_author=J Michaud; citation_author=HS Scott; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=54; citation_pages=2067-2075; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bacterioferritin A modulates catalase A (KatA) activity and resistance to hydrogen peroxide in Pseudomonas aeruginosa.; citation_author=JF Ma; citation_author=UA Ochsner; citation_author=MG Klotz; citation_author=VK Nanayakkara; citation_author=ML Howell; citation_journal_title=J Bacteriol; citation_volume=181; citation_number=55; citation_pages=3730-3742; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The DtxR protein acting as dual transcriptional regulator directs a global regulatory network involved in iron metabolism of Corynebacterium glutamicum.; citation_author=I Brune; citation_author=H Werner; citation_author=AT Huser; citation_author=J Kalinowski; citation_author=A Puhler; citation_journal_title=BMC Genomics; citation_volume=7; citation_number=56; citation_pages=21; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Crystal structure and mutational study of RecOR provide insight into its mode of DNA binding.; citation_author=J Timmins; citation_author=I Leiros; citation_author=S McSweeney; citation_journal_title=Embo J; citation_number=57; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Structural basis of the redox switch in the OxyR transcription factor.; citation_author=H Choi; citation_author=S Kim; citation_author=P Mukhopadhyay; citation_author=S Cho; citation_author=J Woo; citation_journal_title=Cell; citation_volume=105; citation_number=58; citation_pages=103-113; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Studies on a radio-resistant micrococcus. I. Isolation, morphology, cultural characteristics, and resistance to gamma radiation.; citation_author=AW Anderson; citation_author=HC Nordon; citation_author=RF Cain; citation_author=G Parrish; citation_author=G Duggan; citation_journal_title=Food Technol; citation_volume=10; citation_number=59; citation_pages=575-578; citation_date=1956; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001602" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Novel OxyR Sensor and Regulator of Hydrogen Peroxide Stress with One Cysteine Residue in Deinococcus radiodurans"/>
    <meta name="twitter:description" content="In bacteria, OxyR is a peroxide sensor and transcription regulator, which can sense the presence of reactive oxygen species and induce antioxidant system. When the cells are exposed to H2O2, OxyR protein is activated via the formation of a disulfide bond between the two conserved cysteine residues (C199 and C208). In Deinococcus radiodurans, a previously unreported special characteristic of DrOxyR (DR0615) is found with only one conserved cysteine. dr0615 gene mutant is hypersensitive to H2O2, but only a little to ionizing radiation. Site-directed mutagenesis and subsequent in vivo functional analyses revealed that the conserved cysteine (C210) is necessary for sensing H2O2, but its mutation did not alter the binding characteristics of OxyR on DNA. Under oxidant stress, DrOxyR is oxidized to sulfenic acid form, which can be reduced by reducing reagents. In addition, quantitative real-time PCR and global transcription profile results showed that OxyR is not only a transcriptional activator (e.g., katE, drb0125), but also a transcriptional repressor (e.g., dps, mntH). Because OxyR regulates Mn and Fe ion transporter genes, Mn/Fe ion ratio is changed in dr0615 mutant, suggesting that the genes involved in Mn/Fe ion homeostasis, and the genes involved in antioxidant mechanism are highly cooperative under extremely oxidant stress. In conclusion, these findings expand the OxyR family, which could be divided into two classes: typical 2-Cys OxyR and 1-Cys OxyR."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001602.g007"/>

  <meta property="og:title" content="A Novel OxyR Sensor and Regulator of Hydrogen Peroxide Stress with One Cysteine Residue in Deinococcus radiodurans" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001602" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=87'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1566'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3482&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001602">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001602" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001602&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Novel OxyR Sensor and Regulator of Hydrogen Peroxide Stress with One Cysteine Residue in <i>Deinococcus radiodurans</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Huan Chen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China, 
                  James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guangzhi Xu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ye Zhao, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bing Tian, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Huiming Lu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiaomin Yu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Key Laboratory of Genome Science and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhenjian Xu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nanjiao Ying, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Songnian Hu
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:husn@big.ac.cn">husn@big.ac.cn</a> (SH); <a href="mailto:yjhua@zju.edu.cn">yjhua@zju.edu.cn</a> (YH)</p>

                <p>Affiliations:
                  James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, China, 
                  Key Laboratory of Genome Science and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yuejin Hua
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:husn@big.ac.cn">husn@big.ac.cn</a> (SH); <a href="mailto:yjhua@zju.edu.cn">yjhua@zju.edu.cn</a> (YH)</p>

                <p>Affiliation:
                  Key Laboratory of Chinese Ministry of Agriculture for Nuclear-Agricultural Sciences, Institute of Nuclear-Agricultural Sciences, Zhejiang University, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 13, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001602</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001602" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001602" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001602" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001602" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001602-g001" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-t001" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-t002" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-g002" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-g003" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-g004" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-g005" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-t003" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-t004" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-t005" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-t006" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-t007" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t007" title="Table 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-g006" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001602-g007" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001602">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001602" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>In bacteria, OxyR is a peroxide sensor and transcription regulator, which can sense the presence of reactive oxygen species and induce antioxidant system. When the cells are exposed to H<sub>2</sub>O<sub>2</sub>, OxyR protein is activated via the formation of a disulfide bond between the two conserved cysteine residues (C199 and C208). In <em>Deinococcus radiodurans</em>, a previously unreported special characteristic of DrOxyR (DR0615) is found with only one conserved cysteine. <em>dr0615</em> gene mutant is hypersensitive to H<sub>2</sub>O<sub>2</sub>, but only a little to ionizing radiation. Site-directed mutagenesis and subsequent <em>in vivo</em> functional analyses revealed that the conserved cysteine (C210) is necessary for sensing H<sub>2</sub>O<sub>2</sub>, but its mutation did not alter the binding characteristics of OxyR on DNA. Under oxidant stress, DrOxyR is oxidized to sulfenic acid form, which can be reduced by reducing reagents. In addition, quantitative real-time PCR and global transcription profile results showed that OxyR is not only a transcriptional activator (e.g., <em>katE</em>, <em>drb0125</em>), but also a transcriptional repressor (e.g., <em>dps</em>, <em>mntH</em>). Because OxyR regulates Mn and Fe ion transporter genes, Mn/Fe ion ratio is changed in <em>dr0615</em> mutant, suggesting that the genes involved in Mn/Fe ion homeostasis, and the genes involved in antioxidant mechanism are highly cooperative under extremely oxidant stress. In conclusion, these findings expand the OxyR family, which could be divided into two classes: typical 2-Cys OxyR and 1-Cys OxyR.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Chen H, Xu G, Zhao Y, Tian B, Lu H, et al.  (2008) A Novel OxyR Sensor and Regulator of Hydrogen Peroxide Stress with One Cysteine Residue in <em>Deinococcus radiodurans</em>. PLoS ONE 3(2):
          e1602.
            doi:10.1371/journal.pone.0001602</p><p><strong>Academic Editor: </strong>Axel Imhof, University of Munich and Center of Integrated Protein Science, Germany</p><p><strong>Received:</strong> November 10, 2007; <strong>Accepted:</strong> January 18, 2008; <strong>Published:</strong> February 13, 2008</p><p><strong>Copyright:</strong> © 2008 Chen et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work is supported by a grant from the National Basic Research Program (2004CB19604), a grant from the National Hi-Tech Development Program (2007AA021305), a grant for Distinguished Young Scientist (30425038), and a key project from the National Natural Science Foundation of China (30330020) to YJH.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Reactive oxygen species (ROS), including hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>), superoxide, and hydroxyl radical, are toxic to cells due to their ability to damage DNA and especially proteins containing iron-sulfur clusters or sulfur atoms <a href="#pone.0001602-Imlay1">[1]</a>. In bacteria, many transcription factors have been found to sense the presence of ROS and induce antioxidant system. OxyR is such a peroxide sensor and transcription regulator. It was originally identified in <em>Salmonella enterica</em> serovar Typhimurium and <em>Escherichia coli</em> <a href="#pone.0001602-Zheng1">[2]</a>, <a href="#pone.0001602-Christman1">[3]</a>. In <em>E. coli</em>, OxyR is a positive regulator of <em>dps</em> (a DNA-binding ferritin-like protein), <em>groA</em> (GSH), <em>grxA</em> (glutaredoxin), <em>katG</em> (catalase), <em>ahpCF</em> (alkylhydroperoxide-NADPH oxido-reductase subunits F and C), <em>fur</em> (Fe-homeostasis regulation), and <em>oxyS</em> (a regulatory RNA) <a href="#pone.0001602-Helmann1">[4]</a>. However, OxyR acts as a repressor of catalase expression in <em>Neisseria gonorrhoeae</em> <a href="#pone.0001602-Tseng1">[5]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>As a redox-responsive protein of the LysR family, OxyR has conserved regions consisting of a helix-turn-helix motif and a LysR-substrate binding domain. When the cells are exposed to H<sub>2</sub>O<sub>2</sub>, OxyR protein is thought to be activated via the formation of a disulfide bond between the two cysteine residues (C199 and C208) <a href="#pone.0001602-Zheng1">[2]</a>, <a href="#pone.0001602-Lee1">[6]</a>. Detailed footprinting studies indicate that oxidized OxyR binds to its target promoters as a tetramer, occupying four adjacent major grooves upstream of the genes to be transcriptionally activated <a href="#pone.0001602-Toledano1">[7]</a>. However, Kim <em>et al</em>. argued that OxyR activation does not involve disulfide bond formation at all, and that only one thiol in OxyR is critical for protecting against H<sub>2</sub>O<sub>2 </sub><a href="#pone.0001602-Kim1">[8]</a>. Their work disclosed that OxyR is not involved in disulfide bond formation when it was activated by S-nitrosylation, and that mutation of C208 (which was reported to form a disulfide bond with C199) would not result in the cell hypersensitivity to H<sub>2</sub>O<sub>2</sub>, whereas the mutation of C199 did <a href="#pone.0001602-Kim1">[8]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The gram-positive bacterium <em>Deinococcus radiodurans</em> is well known for its extreme resistance to ionizing radiation <a href="#pone.0001602-Battista1">[9]</a>, <a href="#pone.0001602-Minton1">[10]</a>, ultraviolet radiation <a href="#pone.0001602-Tanaka1">[11]</a>, <a href="#pone.0001602-Earl1">[12]</a>, oxidizing agent <a href="#pone.0001602-Markillie1">[13]</a>, and desiccation <a href="#pone.0001602-Tanaka2">[14]</a>. It has been suggested that protective mechanisms against oxidative damage is also involved in this extreme radiation resistance <a href="#pone.0001602-Markillie1">[13]</a>, <a href="#pone.0001602-Ghosal1">[15]</a>. <em>D. radiodurans</em> possesses a powerful enzymatic antioxidant system, including three catalases, three superoxide dismutases, two Dps, etc. However, the mechanism of its response to oxidant stress has not been well clarified. Here, we demonstrate an OxyR in <em>D. radiodurans</em>, which is different from all reported homologs in containing only one cysteine residue. Based on quantitative real-time PCR (QRT-PCR), we found that DrOxyR is both an activator, and a repressor. The binding of purified His-tagged OxyR protein to the upstream region of the respective genes was verified <em>in vitro</em> by DNA band shift assays. Furthermore, we investigated the global trancriptome variation due to disruption of <em>droxyR</em>, and the comparative analysis reveals pathways significantly impacted either directly or indirectly by <em>droxyR</em>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Identification of oxyR in <em>D. radiodurans</em></h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In <em>D. radiodurans</em> genome database (TIGRE), there is a potential <em>oxyR</em> homolog (DR0615, designated as <em>droxyR</em>) <a href="#pone.0001602-Ghosal1">[15]</a>, which encodes a protein of 317 amino acids. BLASTP analysis showed that DR0615 exhibited 31% identity to <em>E. coli</em> OxyR and 29% identity to <em>N. gonorrhoeae</em> OxyR, respectively <a href="#pone.0001602-Seib1">[16]</a>. Five conserved residues in its helix-turn-helix region (between amino acids 3 to 62) involved in DNA binding are identical (<a href="#pone-0001602-g001">Figure 1</a>) <a href="#pone.0001602-Kullik1">[17]</a>. Other functional domains are conserved at its LysR-substrate binding domain (between amino acids 86 to 297), including a hydrophobic core, a tetramerization domain, and a RNA polymerase binding domain <a href="#pone.0001602-Kullik1">[17]</a>, <a href="#pone.0001602-Wang1">[18]</a>. Interestingly, DR0615 has a single sensing cysteine residue (C210), compared with other organisms. This difference in the primary structure of <em>oxyR</em> raised the possibility that <em>droxyR</em> need not, indeed can not form an intramolecular disulfide bond, and that DrOxyR activation can be caused by the modification of just one cysteine residue (C<sub>210</sub>) (<a href="#pone-0001602-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0001602-g001"><div class="img"><a name="pone-0001602-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Alignment of OxyR homologs from different organisms.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Using CLUSTAL W software aligned amino acid sequences of the <em>Streptomyces coelicolor</em> A3(2), <em>Neisseria gonorrhoeae</em>, <em>E. coli</em>, and <em>D. radiodurans</em>. Identical amino acids are highlighted in black, and conserved residues are highlighted with grey. The DrOxyR helix-turn-helix region has four conserved residues (R4, L32, S33 and R50) <a href="#pone.0001602-Kullik1">[17]</a>. At its LysR-substrate binding domain, D142 and R273 possibly define an activating region on OxyR (contact with RNA polymerase)<a href="#pone.0001602-Wang1">[18]</a>, A233 residue is involved in tetramerization<a href="#pone.0001602-Kullik1">[17]</a>, V110, L124, and A233 form a hydrophobic core<a href="#pone.0001602-Choi1">[58]</a>. Numbering is based on the <em>E. coli</em> OxyR sequence.</p>
<span>doi:10.1371/journal.pone.0001602.g001</span></div>

<h4>Phenotypic characterization of <em>MOxyR</em></h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To test its role in the antioxidant mechanism of <em>D. radiodurans</em>, a <em>droxyR</em> disruptant strain (MOxyR) was constructed (<a href="#pone-0001602-t001">Table 1</a>). The coding region of the <em>dr0615</em> was replaced with a kanamycin resistance cassette under a constitutively expressed <em>D. radiodurans groEL</em> promoter. The primers are used for mutation listed in <a href="#pone-0001602-t002">Table 2</a>.</p>
<div class="figure" id="pone-0001602-t001"><div class="img"><a name="pone-0001602-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Bacterial strains and plasmids used in this study</span></strong></p><span>doi:10.1371/journal.pone.0001602.t001</span></div><div class="figure" id="pone-0001602-t002"><div class="img"><a name="pone-0001602-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Primers used in this study.</span></strong></p><span>doi:10.1371/journal.pone.0001602.t002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Disruption of <em>dr0615</em> did not show a growth defect (data not shown). However, as shown in <a href="#pone-0001602-g002">Figure 2A</a>, the sensitivity of the mutant after 20 mM H<sub>2</sub>O<sub>2 </sub>treatment was increased, compared to the wild type strain. After complemention with wild type <em>droxyR</em> gene (plasmid pRAD<em>oxyR</em>) (<a href="#pone-0001602-t001">Table 1</a>), H<sub>2</sub>O<sub>2</sub> resistance of the MOxyR_wtC was significantly increased. In contrast, the MOxyR_sdC strains, which is the <em>oxyR</em> disruption mutant complemented with the <em>droxyR</em> C210A site-directed mutant (plasmid pRAD<em>oxyR</em>sdC), still showed sensitivity to H<sub>2</sub>O<sub>2</sub>. In addition, a little difference was observed between the ionizing radiation resistance of wild type strains and that of MOxyR (<a href="#pone-0001602-g002">Figure 2B</a>).</p>
<div class="figure" id="pone-0001602-g002"><div class="img"><a name="pone-0001602-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Survival curves of D. radiodurans strains exposed to (A) H<sub>2</sub>O<sub>2</sub> and (B) ionizing irradiation.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) Wild-type D. radiodurans R1 (○) compared to MOxyR (▪), MOxyR_wtC (•), and MOxyR_sdC (□) under 20 mM H<sub>2</sub>O<sub>2</sub> treatment at the five recovery time points (0, 20, 40, 60, and 80 min). (B) Wild-type D. radiodurans R1 (○) compared to MOxyR (▪) after ionizing irradiation. Error bars represent standard deviations from four replicate experiments.</p>
<span>doi:10.1371/journal.pone.0001602.g002</span></div>

<h4>Differences in catalase activities and ROS levels between the MOxyR and wild type strains</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In order to investigate the regulatory role of OxyR on enzymatic antioxidants of <em>D. radiodurans</em> after treatment of H<sub>2</sub>O<sub>2</sub>, we assayed the catalase activity in wild type, MOxyR, MOxyR_wtC, and MOxyR_sdC stains with H<sub>2</sub>O<sub>2</sub> treatment or not (<a href="#pone-0001602-t001">Table 1</a>). When log-phase cells were treated with 20 mM H<sub>2</sub>O<sub>2</sub>, the wild type showed a 1.5-fold increase in catalase activity, whilst, neither MOxyR nor MOxyR_sdC showed an increase (<a href="#pone-0001602-g003">Figure 3A</a>). On the other hand, complement strain MOxyR_wtC showed a higher catalase activity than the wild-type under normal conditions. It is well accepted that <em>oxyR</em> expression is auto-regulated via negative feedback in <em>E.coli</em> <a href="#pone.0001602-Christman2">[19]</a>, so we presume that the <em>droxyR</em> gene is under the control of the stronger <em>groEL</em> promoter in pRAD<em>oxyR</em>, the transcription of which destroys the negative feedback. As a result, catalase production may have been activated by an abundance of OxyR, which is likely oxidized by H<sub>2</sub>O<sub>2</sub> from normal metabolism. This might be the reason why MOxyRC is more resistant to hydrogen peroxide than the wild type strain. These results indicate the <em>droxyR</em> gene is responsible for the regulation of catalase activity.</p>
<div class="figure" id="pone-0001602-g003"><div class="img"><a name="pone-0001602-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Effect of <em>oxyR</em> disruption on the antioxidant ability of <em>D. radiodurans</em>.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Catalase activities of R1, MOxyR, MOxyR_wtC, and MOxyR_sdC after H<sub>2</sub>O<sub>2</sub> treatment (grey bar) or not (white bar). (B) ROS accumulate in four strains after H<sub>2</sub>O<sub>2</sub> treatment (grey bar) or not (white bar). Data reported represent the average and standard deviations of three independent experiments.</p>
<span>doi:10.1371/journal.pone.0001602.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>To determine whether the <em>droxyR</em> disruption has effect on the total ROS scavenging ability of the cell, we also measured the intracellular ROS level. <a href="#pone-0001602-g003">Figure 3B</a> shows that ROS level in all the strains increased after the H<sub>2</sub>O<sub>2</sub> treatment, and that MOxyR and MOxyR_sdC accumulated more ROS than wild type and MOxyR_wtC.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Combined with the survival phenotypic data, it could be inferred that the sensitivity of mutant to H<sub>2</sub>O<sub>2 </sub>is due to the loss of induction effect of <em>oxyR</em> on antioxidant enzymes (e.g. catalase), and that <em>oxyR</em> acts as a positive regulator of catalase. Moreover, the C210A mutant showed the same phenotype as MOxyR, indicating that C210 is a site key to OxyR gene regulation.</p>


<h4>C210 is a sensing cysteine</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The site-directed mutagenesis of <em>droxyR</em> revealed that residue C210 plays essential roles in the function of the protein. This finding poses an intriguing question that whether C210 is a sensing cysteine. To verify this hypothesis, experiments were carried out to obtain C<sub>210</sub>-SOH formation by either CHP or air oxidation, followed by the use of an electrophile (NBD-Cl) to trap the C-SOH <a href="#pone.0001602-Ellis1">[20]</a>. As expected, OxyR treated with CHP (Cys-210SO-NBD) exhibited a maximal absorbance at 347 nm (<a href="#pone-0001602-g004">Figure 4</a>), on the hand, the reducing form Cys-210S-NBD showed its maximal absorbance at 420 nm. This data identified that C210 is the peroxidatic center of the molecular.</p>
<div class="figure" id="pone-0001602-g004"><div class="img"><a name="pone-0001602-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Absorbance of NBD chloride-treated purified OxyR. Air-oxidized OxyR (▪), along with the CHP (10 mM) treated OxyR (□), shows maximal absorption at 347 nm.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The shoulder at 420 nm is reduced OxyR (▴) reacted with NBD chloride. The absorbance spectrum is from 285 to 600 nm.</p>
<span>doi:10.1371/journal.pone.0001602.g004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Because sulfenic acid is highly unstable, and reacts further to produce a disulfide, we used nonreducing SDS-PAGE to identity whether C210 is involved in intermolecular disulfide bond formation. As typical OxyR protein, DrOxyR protein did not form intermolecular disulfide linkages after CHP (organic hydroperoxide) or H<sub>2</sub>O<sub>2 </sub>(inorganic hydroperoxide) treatment (data not shown).</p>


<h4>QRT-PCR analysis disclose two classes of OxyR-dependent genes</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>As in the <em>oxyR</em> knockout of <em>E. coli</em>., disruption of <em>oxyR</em> makes <em>D. radiodurans</em> much more sensitive to H<sub>2</sub>O<sub>2</sub>. As we have known, this sensitivity is attributed to the inhibition of normal transcription of OxyR-dependent genes <a href="#pone.0001602-Seib1">[16]</a>, <a href="#pone.0001602-Zheng2">[21]</a>–<a href="#pone.0001602-Harrison1">[23]</a>. Therefore, eight homolog transcripts which were reported as OxyR-dependent genes in other bacteria <a href="#pone.0001602-Zheng2">[21]</a>, <a href="#pone.0001602-Harrison1">[23]</a>–<a href="#pone.0001602-Kehres1">[25]</a> were selected and compared in wild type and MOxyR using QRT-PCR. QRT-PCR was also used to analyze the expression patterns of these genes in wild type and MOxyR after treatment with H<sub>2</sub>O<sub>2</sub>. Of these genes, <em>dr1998</em> codes a major catalase KatE in <em>D. radiodurans</em>, and it has been shown to play a role in protection of <em>D. radiodurans</em> from oxidative stress and ionizing radiation <a href="#pone.0001602-Markillie1">[13]</a>. <em>dr2263</em> and <em>drB0092</em> are two <em>dps</em> genes, with functions of protection against oxidative stress and iron uptake and storage <a href="#pone.0001602-Kim2">[26]</a>, <a href="#pone.0001602-Cuypers1">[27]</a>, but their expression patterns are different under ionizing radiation <a href="#pone.0001602-Chen1">[28]</a>, indicating that their regulator may be different. Furthermore, <em>D. radiodurans</em> accumulates high intracellular manganese and low iron levels compared with radiation-sensitive bacteria and this is regarded as an important contribution to its resistance <a href="#pone.0001602-Daly1">[29]</a>. Three ion transport genes were selected to test whether they are regulated by OxyR, including <em>dr1219</em> (<em>feoB,</em> coding ferrous iron transport protein B), <em>drB0125</em> (coding Iron(III) dicitrate-binding periplasmic protein), and <em>dr1709</em> (<em>mntH</em>). In addition, DR1982 is an alkylhydroperoxide reductase subunits F, which could transfers electrons from NADH to AhpC. DR0865 is a Fur or PerR homolog, and Fur proteins control iron uptake in many Gram-negative bacteria, while PerR is postulated to be the peroxide regulon repressor <a href="#pone.0001602-Bsat1">[30]</a>.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Consistent with catalase activity assay, the <em>katE</em> (<em>dr1998</em>) transcript was repressed approximately 1.74-fold in the MOxyR relative to that of wild type, and induction of <em>katE</em> expression by H<sub>2</sub>O<sub>2</sub> was eliminated in strain MOxyR (<a href="#pone-0001602-g005">Figure 5A</a>), suggesting that OxyR is a positive regulator of <em>katE</em>. In addition, both iron transporter genes (<em>dr1219</em> and <em>drB0125</em>) showed the same expression pattern as <em>katE</em>, indicating that expression of these genes were also mediated by OxyR, a finding similar to the OxyR regulon in <em>Haemophilus influenzae</em> <a href="#pone.0001602-Harrison1">[23]</a>.</p>
<div class="figure" id="pone-0001602-g005"><div class="img"><a name="pone-0001602-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Q-RT PCR of the expression of potential OxyR-dependent genes in <em>D. radiodurans</em> R1 wild-type compared to wild type after H<sub>2</sub>O<sub>2</sub> treatment (white bar), MOxyR (grey bar), and MOxyR H<sub>2</sub>O<sub>2</sub> treatment (black bar).</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) genes positively regulated by DrOxyR; (B) genes negatively regulated by DrOxyR; (C) the expression patterns of DR1709 and DR2263 were measured in MOxyR_sdC; (D) genes not regulated by DrOxyR.</p>
<span>doi:10.1371/journal.pone.0001602.g005</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>When the wild type cells were exposed to H<sub>2</sub>O<sub>2</sub>, the transcripts of <em>dps</em> (<em>dr2263</em>) and <em>mntH</em> (<em>dr1709</em>) decreased, whereas, transcripts of these genes in the MOxyR cell were identified either under normal growth conditions or after H<sub>2</sub>O<sub>2 </sub>challenge (<a href="#pone-0001602-g005">Figure 5B</a>). These data informed us that oxidized DrOxyR might act as a transcription repressor of <em>dr2263</em> and <em>dr1709</em>. Since the expression levels of <em>dr2263</em> and <em>dr1709</em> are higher in MOxyR than those of in wild type strains under normal condition, we deduce that reduced DrOxyR could also be a repressor of both of them. To verify this hypothesis, we measured the expression patterns of DR1709 and DR2263 in MOxyR_sdC, which contained a reduced OxyR protein due to its C210 mutation. Opposite to their expression patterns in MOxyR, both of them did not show a significant increase under normal condition (<a href="#pone-0001602-g005">Figure 5C</a>). Although they were activated under H<sub>2</sub>O<sub>2</sub> treatment in MOxyR_sdC, the change fold is less than that of in MOxyR. The data confirmed that reduced OxyR could still negatively regulate these genes expression. DrOxyR action pattern is opposite to its homologs in <em>E. coli</em> <a href="#pone.0001602-Zheng2">[21]</a>, <em>Bacteroides fragilis</em> <a href="#pone.0001602-Rocha1">[31]</a>, and <em>Shigella flexneri</em> <a href="#pone.0001602-RunyenJanecky1">[32]</a>. In <em>D. radiodurans</em>, two types of Mn(II) import systems have been identified, DR1709 belongs to the Nramp family of transporters <a href="#pone.0001602-Daly1">[29]</a>. Another type of predicted Mn transporter is an ATP-dependent ABC-type transporter (DR2283-DR2284, DR2523) <a href="#pone.0001602-Daly1">[29]</a>. However, QRT-PCR results showed that the second Mn ion transporter system was not regulated by DrOxyR protein (Data not shown).</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Additionally, <em>dr1982</em> (<em>ahpF</em>) and <em>drB0092</em> (<em>dps</em>) did not show significant changes at oxidant stress (<a href="#pone-0001602-g005">Figure 5D</a>). We deduced that OxyR did not regulate both of them, and the transcription patterns of the two <em>dps</em> genes (<em>dr2263</em> and <em>drB0092</em>) are different. Furthermore, we also measured the expression of <em>dr0865</em>, which is a putative <em>perR</em> homolog. We found that <em>katE</em> was significantly activated after deletion of <em>dr0865</em>, and that the mutant strain exhibited greater resistance to H<sub>2</sub>O<sub>2</sub> than wild type strain (unpublished data). Interestingly, the expression level of <em>perR</em> was repressed in MOxyR, as well as in the wild type strain after H<sub>2</sub>O<sub>2</sub> challenge. Given the expression pattern of <em>perR</em>, the oxidant stress occurred after disruption of <em>oxyR</em>, which was consistent with intracellular ROS accumulation assay results.</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>Since the Mn(II) transporter gene (<em>dr1709</em>) is induced, and the iron transporter genes (<em>dr1219</em>, <em>drB0125</em>) are repressed in <em>oxyR</em> mutant, we assayed the intracellular Fe ion and Mn ion levels in MOxyR. As expected, compared to that Fe ion levels are three times higher than Mn ion in wild type strain, Fe ion levels are only two times higher than Mn ion in MOxyR (<a href="#pone-0001602-t003">Table 3</a>).</p>
<div class="figure" id="pone-0001602-t003"><div class="img"><a name="pone-0001602-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Intracelluar Mn and Fe levels in wild type and MOxyR.</span></strong></p><span>doi:10.1371/journal.pone.0001602.t003</span></div>

<h4>Transcriptome changes in response to disruption of <em>droxyR</em></h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The results of intracellular ROS accumulation assay showed that MOxyR accumulates higher levels of ROS than the wild type strain. It is well known that ROS is a signaling molecule, and has an important role in the regulation of a variety of biological processes <a href="#pone.0001602-Lander1">[33]</a>, <a href="#pone.0001602-Veal1">[34]</a>. Therefore, in order to identify other OxyR-dependent genes, and to measure the consequences of higher levels of ROS, we carried out a microarray comparison of the wild type and the OxyR mutants grown under normal conditions. <a href="#pone.0001602.s003">Table S1</a> and <a href="#pone.0001602.s004">Table S2</a> exhibits that a total of 280 genes showed at least a 2-fold change (<em>p</em>&lt;0.05). A higher percentage of genes were repressed (150 genes, <a href="#pone.0001602.s003">Table S1</a>), whereas, 130 genes were induced (<a href="#pone.0001602.s004">Table S2</a>). <a href="#pone-0001602-t004">Table 4</a> and <a href="#pone-0001602-t005">Table 5</a> showed the 36 most highly repressed and induced genes, many with known roles in oxidative stress response, including catalase, oxidoreductase, N-acetyltransferase. Furthermore, functional classification of these genes revealed that signal transduction mechanisms, inorganic ion transport and metabolism, lipid metabolism, energy production and conversion, and amino acid transport and metabolism showed altered expression patterns in the <em>oxyR</em> mutant (<a href="#pone.0001602.s005">Table S3</a>).</p>
<div class="figure" id="pone-0001602-t004"><div class="img"><a name="pone-0001602-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>The 36 most highly repressed genes in MOxyR.</span></strong></p><span>doi:10.1371/journal.pone.0001602.t004</span></div><div class="figure" id="pone-0001602-t005"><div class="img"><a name="pone-0001602-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>The 36 most highly induced genes in MOxyR.</span></strong></p><span>doi:10.1371/journal.pone.0001602.t005</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p><a href="#pone-0001602-t006">Table 6</a> and <a href="#pone-0001602-t007">Table 7</a> show genes with the same expression pattern in wild type strain after 20 mM H<sub>2</sub>O<sub>2</sub> treatment (our unpublished data) and these in MOxyR. This confirmed that oxidative stress occurs in MOxyR.</p>
<div class="figure" id="pone-0001602-t006"><div class="img"><a name="pone-0001602-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Genes with a decreased level of expression both in wild-type strains treated with H<sub>2</sub>O<sub>2</sub> (20 mM) and in MOxyR.</span></strong></p><span>doi:10.1371/journal.pone.0001602.t006</span></div><div class="figure" id="pone-0001602-t007"><div class="img"><a name="pone-0001602-t007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.t007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.t007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.t007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.t007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.t007.TIF"></span>)
                </a></li></ul></div><p><strong>Table 7.  <span>Genes with an increased level of expression both in wild-type strains treated with H<sub>2</sub>O<sub>2</sub> (20 mM) an in MOxyR.</span></strong></p><span>doi:10.1371/journal.pone.0001602.t007</span></div>

<h4>Gel mobility shift assays to confirm OxyR-regulated genes</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Our global transcriptome analysis results suggested that the expression patterns of many genes were altered as a consequence of <em>oxyR</em> deletion. However, microarray data could not distinguish those genes directly controlled by OxyR form those controlled by indirect mechanisms. To support the reliability of both QRT-PCR and microarray data, we used a DNA mobility shift assay to determine whether purified OxyR protein could bind <em>in vitro</em> to the two potentially positively regulated gene (<em>dr1998</em> and <em>drB0125</em>) and two potentially negatively regulated genes (<em>dr1709</em> and <em>dr2263</em>). In addition, microarray data showed that many oxidoreductase genes were repressed, so we cloned the promoter sequence of <em>drB0036</em> which is induced after ionizing radiation <a href="#pone.0001602-Chen1">[28]</a> to identify whether <em>oxyR</em> is a regulator of oxidoreductase. <em>dr0207</em> was used as a negative control, which is up-regulated after ionizing radiation<a href="#pone.0001602-Liu1">[35]</a>. As shown in <a href="#pone-0001602-g006">Figure 6</a>, both oxidized and reduced forms of the protein could bind these promoters. Given that OxyR was not completely reduced with DTT and the DTT was probably quickly electrophoresed away from the protein in the mobility shift assays <a href="#pone.0001602-Toledano1">[7]</a>, we also examined the binding of the C210A mutant protein to these genes and observed a same binding pattern (data not shown). This data indicate that OxyR protein is bound to its recognition sequences even in the absence of oxidative stress, and the binding ability supports the result that reduced OxyR could regulate <em>dr1709</em> and <em>dr2263</em> expression.</p>
<div class="figure" id="pone-0001602-g006"><div class="img"><a name="pone-0001602-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Binding of reduced and oxidized OxyR to the upstream region of (A) negative control (<em>dr0207</em> and coding sequence for <em>dr1709</em>); (B) positivly regulated genes (<em>dr1998</em> and <em>drB0125</em>); (C) negativly regulated genes (<em>dr1709</em>, <em>dr2263</em>, and <em>drB0036</em>).</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>To generate reduced protein, 200 mM DTT was added to the binding reactions. Column 1, 2, and 3 indicate non-added protein, reduced OxyR added, and oxidized OxyR added, respectively.</p>
<span>doi:10.1371/journal.pone.0001602.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p><em>D. radiodurans</em> exhibits extreme radiation resistance. In addition to its powerful DNA repair systems which include some novel components <a href="#pone.0001602-Harris1">[36]</a>, free radical scavengers are regarded as important contributors to this resistant mechanism <a href="#pone.0001602-Ghosal1">[15]</a>, <a href="#pone.0001602-Xu1">[37]</a>. Recently, Daly <em>et al.</em> reported that accumulation of Mn (II) in <em>D. radiodurans</em> facilitates its radiation resistance through high levels of protein protection from oxidation <a href="#pone.0001602-Daly1">[29]</a>, <a href="#pone.0001602-Daly2">[38]</a>. However, despite these efforts, the molecular regulatory mechanism underlying its oxidative resistance remains poorly understood. Therefore, in this work, we demonstrate that DR0615 is an OxyR homologue with an important role in oxidative stress sensing and regulation mechanism.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Unlike many other OxyR homologs, the sequence analyses showed that DrOxyR contains only one cysteine, which is absolutely conserved in other OxyR homologs (<a href="#pone-0001602-g001">Figure 1</a>). Further comparisons of the amino acid sequences of DrOxyR with those from other bacteria indicated that the other activating regions are well conserved in the DrOxyR protein, except the T238 residue which is involved in C199–C208 disulfide bond formation is absent (<a href="#pone-0001602-g001">Figure 1</a>) <a href="#pone.0001602-Wang1">[18]</a>. This sequence characteristic informed that DrOxyR could not form intramolecular disulfide bond under oxidative stress. However, disulfide bonds are not the only cysteine oxidation product important for redox sensing. As <em>Bacillus subtilis's</em> OhrR protein <a href="#pone.0001602-Fuangthong1">[39]</a>, our NBD chloride assay <em>in vitro</em> showed that the sole cysteine could be oxidized into sulfenic acid and did not generate an intermolecular disulfide bond. The <em>in vivo</em> functional analyses of the cysteine mutant and wild type OxyR showed that the single C210 is sufficient for DrOxyR to act as a sensor of and a regulator responding to oxidant stress. But the protein may react with low molecular weight thiols (e.g. Cysteine) to make a mixed disulfide <em>in vivo,</em> such as that of BsOhrR <a href="#pone.0001602-Lee2">[40]</a>. This is a major mechanistic difference from the sensing mechanism of the 2-Cys OxyR, for which sulfenic acid is an intermediate in the pathway to intramolecular disulfide bond formation <a href="#pone.0001602-Zheng1">[2]</a>. Nevertheless, this activating pathway was challenged by the report of Stamler's group, whose research showed that sulfenic acid is stable in per monomer <a href="#pone.0001602-Kim1">[8]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Furthermore, the <em>droxyR</em> could not complement the defect in the <em>E. coli oxyR</em> mutant strain (GS09) (<a href="#pone.0001602.s001">Figure S1</a>), even when GS09 was complemented with <em>droxyR</em>::T201C, which is put the missing cysteine back in DrOxyR (data not shown). One explanation for the inability of DrOxyR to complement to GS09 is an inability to properly contact or communicate with <em>E. coli</em> RNA polymerase. Another explanation is that these two families (based on the number of cysteine residues present) use different mechanisms to activate downstream process.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Based on QRT-PCR results, eight potential OxyR-regulated genes were classified into three classes due to their different expression pattern in wild type and mutant strains before or after H<sub>2</sub>O<sub>2 </sub>stress. Particularly interesting is, excepting as a regulator of <em>katE</em>, DrOxyR also acts as an activator of iron transport genes and as a repressor of manganese transport gene. As seen in recent studies, the high intracellular Mn/Fe ratio in <em>D. radiodurans</em> plays an important role in resistance ability by protecting cells from ROS generation during recovery <a href="#pone.0001602-Ghosal1">[15]</a>, <a href="#pone.0001602-Daly1">[29]</a>, <a href="#pone.0001602-Daly2">[38]</a>. In addition, our findings do not preclude the existence of other regulators of Mn/Fe transport genes, or <em>katE</em>. Indeed, <em>dr0865</em> (<em>fur or perR</em>) and <em>dr2519</em> (<em>mntR</em> or <em>dtxR</em>) also showed the abilities to regulate these genes (our unpublished data), indicating that the oxidative stress response network is much more complex than we initial prediction.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>From the microarray data, we found a total of 280 genes (about 9% of genome) that showed at least 2-fold change, suggesting that these genes were regulated by <em>droxyR</em> through either direct or indirect mechanisms. Several genes annotated as <em>N</em>-acetyltransferase (<em>dr0763</em>, <em>dr1057</em>, <em>dr1978</em>, <em>dr2441</em>, and <em>drA0019</em>) were repressed. In <em>Saccharomyces cerevisiae</em>, <em>N</em>-acetyltransferase could reduce intracellular oxidant levels and protect cells from oxidative stress <a href="#pone.0001602-Du1">[41]</a>. Furthermore, genes involved in electron transport were also repressed, including <em>dr0343</em>, <em>dr1493</em>, <em>dr1502</em>, <em>dr1505</em>, and <em>dr2618</em>. This may result in a decrease of the production of ROS generated from the electron transport process <a href="#pone.0001602-Imlay1">[1]</a>, <a href="#pone.0001602-Ghosal1">[15]</a>. In addition, an iron-sulfur protein (DR1907) was also significantly inhibited to avoid the attack by ROS. Conversely, many oxidoreductases were overexpressed, some of which are involved in regulating the oxidation state and activities of several proteins <a href="#pone.0001602-Veal1">[34]</a>. Thioredoxin (DRA0164) is such an oxidoreductase whose expression level was highly elevated in MOxyR. It directly regulates the activation of specific signal transduction proteins through hydrogen peroxide-sensitive noncovalent interactions <a href="#pone.0001602-Veal1">[34]</a>. Moreover, gel mobility shift assays showed that DRB0036 (oxidoreductase) is under the control of OxyR, suggesting that OxyR may be directly involved in the oxidoreductase processes of <em>D. radiodurans</em>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>As a signalling molecule, hydrogen peroxide has an important roles in the regulation of a variety of biological processes, such as stimulate cell proliferation <a href="#pone.0001602-Lander1">[33]</a>, <a href="#pone.0001602-Veal1">[34]</a>. In this work, some genes involved in transcriptional regulation, transport and cell proliferation also showed an altered expression pattern. Thus, it was deduced that these genes were not regulated by <em>oxyR</em>, but were affected as a consequence of ROS accumulation. For example, we have shown the expression levels of two genes, <em>minE</em> (<em>dr0751</em>) and <em>minD</em> (<em>dr2383</em>), were changed. The MinE protein, which is known to prevent the division inhibitor from acting at internal division sites, was activated, whereas the <em>minD</em> gene, which is a cell division inhibitor, was repressed. Based on these expression patterns, it could be assumed that the cell was stimulated due to ROS accumulation. Other interesting genes that were down-regulated in MOxyR were some DNA damage response genes, including <em>recA</em>, <em>cinA</em>, <em>ligT</em>, <em>dinB</em>, <em>ddrB</em>, <em>ddrC</em>, <em>ddrH</em>, <em>ddrJ</em>, <em>ddrK</em>, <em>ddrM</em>, and <em>ddrO</em> <a href="#pone.0001602-Tanaka2">[14]</a>, whereas they were up-regulated in MOxyR after 20 mM H<sub>2</sub>O<sub>2</sub> treatment (data not shown), indicating that they were not regulated by <em>oxyR</em> only.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Based on the gene expression patterns, two classes of OxyR-dependent genes were identified and DrOxyR can function not only as a transcriptional activator but also as a transcriptional repressor. Our DNA band shift assay showed that either reduced OxyR protein or oxidized protein can bind both classes of genes. As transcriptional activator, reduced OxyR binds to two pairs of adjacent major grooves separated by one helical turn of the DNA duplex and acts to repress its own synthesis. When oxidized, the OxyR tetramer binds four adjacent major grooves upstream of those genes that are transcriptionally activated by OxyR <a href="#pone.0001602-Helmann1">[4]</a>, <a href="#pone.0001602-Toledano1">[7]</a> (<a href="#pone-0001602-g007">Figure 7</a>). Whereas a repressor, the reduced form of OxyR produces a basal expression level in the absence of exogenous H<sub>2</sub>O<sub>2</sub> and the oxidized form of OxyR significantly repressed the genes' expression level. Moreover, after deletion of OxyR, the expression of these genes was significantly induced in MOxyR (<a href="#pone-0001602-g007">Figure 7</a>). This agrees with published data showing that oxidized OxyR also acts as a repressor <a href="#pone.0001602-Tseng1">[5]</a>, <a href="#pone.0001602-Wallecha1">[42]</a>. Nonetheless, no obvious binding sites, such as those identified in <em>E. coli</em>, were observed in these genes. This is consistent with the results of OxyR-binding sites of genes in other bacteria <a href="#pone.0001602-Tseng1">[5]</a>, <a href="#pone.0001602-Zeller1">[22]</a>.</p>
<div class="figure" id="pone-0001602-g007"><div class="img"><a name="pone-0001602-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001602" data-uri="info:doi/10.1371/journal.pone.0001602.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001602.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001602.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001602.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Model for reduced and oxidized OxyR binding to and activation at the two classes genes.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>For Class I (<em>katE</em>): OxyR activates gene in the presence of H<sub>2</sub>O<sub>2</sub>, whereas under non-stressed conditions, reduced OxyR is bound to two pairs of adjacent major grooves separated by one helical turn of the DNA duplex and acts to repress its own synthesis. For Class II (<em>mntH</em>): mutation of OxyR can greatly enhance gene expression, reduced OxyR binds to DNA and minimally induced genes, whereas oxidized OxyR significantly decreases the gene expression levels.</p>
<span>doi:10.1371/journal.pone.0001602.g007</span></div><a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>In summary, our work presents a biochemical mechanism for hydrogen peroxide sensing of <em>D. radiodurans</em> OxyR, which contains only one conserved cysteine. The gene transcription induction by hydrogen peroxide requires only one cysteine that can be reversibly oxidized by peroxides to a sulfenic acid form. Moreover, based on QRT-PCR and globe transcriptome analysis, we provide evidence that DrOxyR functions as not only a positive regulator but also as a negative regulator of different classes of genes. These results show that genes participating in the Mn/Fe homeostatic and antioxidant system are highly cooperative under extreme conditions, and that cooperation contributes to resistance. More research is needed to establish the detailed mechanism of OxyR regulation of these important genes, and whether communication between OxyR and other regulators such as PerR (DR0865) existed and is required for the intricate coordination of oxygen radical detoxification.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Strains and growth conditions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Bacterial strains and plasmids are listed in <a href="#pone-0001602-t001">Table 1</a>. All <em>D. radiodurans</em> strains used in this work were grown at 30°C in TGY (0.5%Tryptone, 0.3% yeast extract, 0.1% glucose) broth or on TGY plates supplemented with 1.5% Bacto-agar. Overnight cultures were incubated into fresh TGY medium and exponential-phase cells were used in all experiments. <em>E. coli</em> strains were grown in Luria-Bertani (LB) broth or on LB agar plates at 37°C.</p>


<h4>Disruption of the <em>dr0615</em> locus in D. radiodurans</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>A three-step gene splicing by overlap extension was used to generate the DR0615 mutant strain (designed MOxyR) <a href="#pone.0001602-Gao1">[43]</a>. Primers OxyR1 and OxyR2 were used to amplify a <em>Bam</em>HI fragment upstream of targeted genes, and primers OxyR3 and OxyR4 were used to obtain a <em>Hin</em>dIII fragment downstream of targeted genes respectively (<a href="#pone-0001602-t002">Table 2</a>). The kanamycin resistance cassette containing the <em>groEL</em> promoter was obtained from a shuttle plasmid, pRADK <a href="#pone.0001602-Gao1">[43]</a>. After these three DNA fragments were digested and ligated, the ligation products were used as template for PCR to amplify the resulting PCR fragment (OxyR1 and OxyR5 used as primer), which was then transformed into exponential phase cells by CaCl<sub>2</sub> treatment. The mutant strains were selected on TGY agar plates supplemented with 20 µg/mL kanamycin were confirmed by PCR product sizes, enzyme-digested electrophoresis (<a href="#pone.0001602.s002">Figure S2</a>), and DNA sequencing.</p>


<h4>Complementation of <em>oxyR</em> mutant</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Complementation plasmid construction was constructed as previously described by Hua <em>et al</em> <a href="#pone.0001602-Hua1">[44]</a>. Briefly, chromosomal DNA was isolated from wild type strains. A 1000-bp region containing the <em>oxyR</em> gene was amplified by OxyR<sub>com</sub>F and OxyR<sub>com</sub>R (<a href="#pone-0001602-t002">Table 2</a>), and ligated to pMD18 T-Easy vector (Takara, JP), designed as pMD<em>oxyR</em>. After digested by <em>Nde</em>I and <em>Bam</em>HI, the target gene <em>oxyR</em> was ligated to <em>Nde</em>I and <em>Bam</em>HI-pre-digested pRADK <a href="#pone.0001602-Gao1">[43]</a>, which named as pRAD<em>oxyR</em>. The complementation plasmids were confirmed by PCR and DNA sequence analysis, and transformed into MOxyR and GS09 (K12 <em>oxyR</em>::<em>Kan</em> of <em>E. coli</em>, a gift from Dr Gisela Storz) <a href="#pone.0001602-Toledano1">[7]</a>, resulting in two functional complementation strains: MOxyR_wtC (<em>D. radiodurans oxyR</em> mutant strain complemented with pRAD<em>oxyR</em>) and GS09C (GS09 complemented with pRAD<em>oxyR</em>) (<a href="#pone-0001602-t001">Table 1</a>).</p>


<h4>PCR mutagenesis C<sub>210</sub> of OxyR</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The first PCR fragment was obtained using primer OxyR<sub>com</sub>F and a mutagenic antisense primer C<sub>210</sub>AR (<a href="#pone-0001602-t002">Table 2</a>, the mutated bases are underlined). The second PCR fragment was obtained using primer OxyR<sub>com</sub>R and the mutagenic sense primer C<sub>210</sub>AF, which is complementary to the C<sub>210</sub>AR. The mutagenesis PCR was generated by using OxyR<sub>com</sub>F and OxyR<sub>com</sub>R <a href="#pone.0001602-Fuangthong1">[39]</a>, and ligated to pMD18 T-Easy vector (Takara), designed as pMD18<em>oxyR</em>sd. Then, pMD18<em>oxyR</em>sd was digested with <em>Nde</em>I and <em>Bam</em>HI, and cloned into pRADK. The resulted pRAD<em>oxyR</em>sdC plasmid was transformed into DrOxyR. The <em>oxyR</em> site-directed mutation sequence was verified by sequencing.</p>


<h4>Measurement of cell survival rate</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The sensitivity of <em>D. radiodurans</em> cells to hydrogen peroxide was assayed following the method as previously described with some modifications <a href="#pone.0001602-Brenot1">[45]</a>. Cells were harvested in early stationary phase, washed twice with and re-suspended in phosphate buffer (20mM, pH 7.4). An aliquot was removed as control and the remaining aliquot was treated with hydrogen peroxide to a final concentration of 20 mM. The mixture was incubated at 30°C in an orbital shaker. At the indicated recovery time points (20, 40, 60, and 80 min), an aliquot was removed and catalase (Sigma, St. Louis, MO) was added in excess (100 g/mL) to stop the H<sub>2</sub>O<sub>2 </sub>treatment. The cells were diluted and spread on solid TGY media for determining the numbers of colonies forming units (cfu). Survival rates are defined as a percentage of the number of colonies obtained comparing with the control. Hydrogen peroxide disk assay was used to assay the survival rate of <em>E. coli</em> <a href="#pone.0001602-Raahave1">[46]</a>.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Cell survival rate after ionizing radiation was determined by the method described previously <a href="#pone.0001602-Huang1">[47]</a>. In short, after the cells were harvested, 200 µL of various MOxyR strains and wild type strains were irradiated at room temperature for 1h with <sup>60</sup>Co γ-rays at various distances from the source, which correspond to doses from 0 to 24 kGy. After irradiation, the various MOxyR and wild type strains were plated on TGY plates and incubated at 30°C for 3 days prior to colony enumeration.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>All the data provided here represent the mean and standard deviation of at least three independent experiments.</p>


<h4>Catalase activity assay</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Whole-cell protein extracts were obtained from <em>D. radiodurans</em> cells in exponential phase growth according to the method of Tian <em>et al</em> <a href="#pone.0001602-Tian1">[48]</a>. Catalase activity was determined as in <a href="#pone.0001602-Esposito1">[49]</a>: 2 µL of 0.2 mg/mL protein was diluted with 38 µL of PBS buffer, 10 µL of 250 mM H<sub>2</sub>O<sub>2</sub> was then added followed by incubation at 25°C for 2 min. The reaction was quenched with 450 µL of 1% sodium azide. 10 µL of this mixture was diluted with 200 µL of an appropriate chromogenic reagent miture (Beyotime, CHN), in a 1-cm path-length polystyrene cuvette. The quantity of H<sub>2</sub>O<sub>2</sub> remaining in the mixture can be determined by the oxidative coupling of 4-aminophenazone (4-aminoantipyrene, AAP) and 3,5-dichloro-2-hydroxybenzenesulfonic acid (DHBS) in the presence of H<sub>2</sub>O<sub>2</sub> when catalyzed by horseradish peroxidase (HRP) contained in the chromogenic reagent mixture. After 15 min of incubation at 25°C, the resultant quinoneimine dye (N-(4-antipyrl)-3-chloro-5-sulfonate-p-b​enzoquinonemonoimine)was quantitated at 520 nm. H<sub>2</sub>O<sub>2</sub> concentrations were determined by reference to a calibration curve generated from H<sub>2</sub>O<sub>2</sub> solutions in the range 0–0.5 µM. The activity of catalase (expressed in micromoles of H<sub>2</sub>O<sub>2</sub> decomposed per minute per milligram of total protein) was calculated as follows:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001602.e001&amp;representation=PNG" class="inline-graphic"></span><br></p>


<h4>Intracellular ROS accumulation assay</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>ROS generation in cells was assayed as reported <a href="#pone.0001602-Kang1">[50]</a>. In short, 10<sup>7</sup> cells (100 µL, OD<sub>600</sub> = 1.0, washed three times with PBS) were resuspended in 1 mL DCFH-DA (10 µM) and incubated at 37°C for 20 minutes. After incubation, the cells were washed twice with PBS and re-suspended in 1 mL PBS. Then, the sample was divided in two parts, half (0.5 mL) was exposed to 1 µL of 10M H<sub>2</sub>O<sub>2</sub> for 20 minutes at room temperature, the other half served as the nonexposed control culture. The fluorescence intensities were measured using a fluorescence spectrophotometer with an excitation wavelength of 485 nm and an emission wavelength of 525 nm.</p>


<h4>Expression and purification of OxyR wild-type and mutant proteins</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Wild-type <em>oxyR</em> was produced from pMD<em>oxyR</em>, which was digested with <em>Nde</em>I and <em>Bam</em>HI. Mutant <em>oxyR</em> was obtained from pMD18<em>oxyR</em>sd digested with <em>Nde</em>I and <em>Bam</em>HI. The products were ligated into pET28a (Novagen, San Diego, CA), and the resulting plasmids were transformed into <em>E. coli</em> BL21 for overexpression of His-tagged proteins, respectively. Protein expression and purification as previously described <a href="#pone.0001602-Zeller1">[22]</a>. Briefly, an overnight culture was diluted 1: 50 in fresh media to an OD<sub>600</sub> of 0.3 at 37°C and followed by a shift to 4°C. After 0.5 hours incubation at 4°C, cells were induced with 0.1 mM isopropyl-β-D-thiogalactoside (IPTG) for 15 hours at 20°C. The harvested cells were resuspended in lysis buffer (20 mM NaH<sub>2</sub>PO<sub>4</sub>; 20 mM Na<sub>2</sub>HPO<sub>4</sub>; 400 mM NaCl; 15 mM imidazole; 1 mM DTT) and disrupted at 4°C in a sonicator. After centrifugation at 12, 000 rpm for 20 min at 4°C, the supernatant was loaded onto Ni-NTA agarose columns (Qiagen, Valencia, CA). The purified OxyR protein was applied to a Superdex 300 HR 16/70. The purity of protein samples was determined using 12% SDS-PAGE, and only fractions with pure OxyR protein were used for further experiments.</p>


<h4>Reversible formation of cysteine-sulfenic acid trapping with NBD chloride</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Modification of OxyR protein by NBD chloride (Sigma) at C210 was detected as previously described <a href="#pone.0001602-Kim1">[8]</a>, <a href="#pone.0001602-Ellis1">[20]</a>, <a href="#pone.0001602-Fuangthong1">[39]</a>, <a href="#pone.0001602-Panmanee1">[51]</a>. Briefly, reduced or oxidized proteins were mixed with 1 mM NBD-Cl in dimethyl sulfoxide and incubated for 60 min at 25°C in the dark. Then the NBD-Cl was removed by ultrafiltration with YM-10 (Millipore, Bedford, MA) dialysed three times in 50 mM pH 7.0 potassium phosphate buffer containing 150 mM NaCl and 1 mM EDTA. The absorbance spectra (300 to 600 nm) of the modified proteins were measured on a ND-1000 spcetrophotometer (NanoDrop, Wilmingon, De. US).</p>


<h4>RNA isolation, quantitative real-time PCR (QRT-PCR) experiments</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>The wild-type strain and MOxyR strain were grown in TGY to mid-exponential phase. For H<sub>2</sub>O<sub>2</sub> treatment, the cultures were divided in two; one half of the culture was treated with H<sub>2</sub>O<sub>2 </sub>at a final concentration of 20 mM, while the other half was used as non-treatment control. RNA isolation, and QRT-PCR were carried out as previously described <a href="#pone.0001602-Chen1">[28]</a>. Total RNA was extracted from cells using TRI Reagent (Invitrogen, Carlsbad, CA), after liquid nitrogen grinding. Then the RNA samples were treated with Rnase free Dnase I (Promega, Madison, WI) and purified using phenol chloroform extraction. RNA quality and quantity were evaluated by UV absorbance at 260 and 280 nm.</p>
<a id="article1.body1.sec4.sec10.p2" name="article1.body1.sec4.sec10.p2"></a><p>QRT-PCR assay utilized RNA samples obtained from different conditions and first-strand cDNA synthesis was carried out in 20 µL of reaction containing 1 µg of RNA sample combined with 3 µg of random hexamers using SuperScript III Reverse Transcriptase kit (Invitrogen). Then Quant SYBR Green PCR kit (TIANGEN, BJ) was used to amplification following the manufacturer's instructions. As an internal control, a house-keeping gene encoding glycosyl transferase, <em>dr0089</em>, was used as a house-keeping gene, encoding the glycosyl transferase <a href="#pone.0001602-Chen1">[28]</a>. In our hands, DR0089 was unaffected by any of our treatments. cDNA probes for microarray hybridization were prepared from four biological replicate total RNA samples each of wild type and MOxyR cultures. All primers used in QRT-PCR are shown in <a href="#pone-0001602-t002">Table 2</a>.</p>


<h4>Microarray hybridization and data analysis</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>Microarray design and constructions were carried out as our previous work <a href="#pone.0001602-Chen1">[28]</a>. Total RNA for microarray hybridization were obtained from four biological replicate samples of each of wild type and MOxyR under normal condition. First, RNA (4 µg) was annealed to 9 µg of random hexamer primers (Takara) in total volume of 20 µL at 70°C for 10 min and subsequent keep on ice for 2 min. cDNA was synthesized at 42°C overnight in total 31 µL using SuperScript III Reverse Transcriptase kit (Invitrogen) with 0.5 mM dNTP mix containing amino allyl-dUTP (GE, Piscataway, NJ). The reaction was terminated by adding 20 µL EDTA (0.5 M), and RNA was hydrolysed by adding 20 µL NaOH (1 M), then incubating at 65°C for 20 min. After neutralized with 50 µL Hepes (1M, pH 7.0), unincorporated free amino allyl-dUTPs were removed by ultrafiltration with YM 30 (Millipore), and resultant cDNA samples were coupled to 1 pmol Cy3 or Cy5 dyes (GE) in 0.1 M sodium carbonate buffer for 2 h at room temperature in the dark. Unincorporated free Cy3 or Cy5 were removed by ultrafiltration with YM 30. Two labeled cDNA pool (wild type and MOxyR) to be compared were mixed and hybridized simultaneously to the array in a solution containing 3×saline sodium citrate (SSC), 0.3 % SDS, and 24 µg of unlabeled herring sperm DNA (Gibco BRL, Gaithersburg, MD) <a href="#pone.0001602-Thompson1">[52]</a>. Following hybridization, slides were washed as published paper <a href="#pone.0001602-Thompson1">[52]</a>.</p>
<a id="article1.body1.sec4.sec11.p2" name="article1.body1.sec4.sec11.p2"></a><p>Measurement of spot intensity and normalization were carried out as our paper <a href="#pone.0001602-Chen1">[28]</a>. In short, slides were scanned with a GenePix 4000B imager (Axon, Union City, CA), and spot intensities were obtained by software GenePix pro 5.1. Normalization and statistical analysis were carried out in the R computing environment (2.11, Raqua on the Windows) using the linear models for microarray data package (Limma) <a href="#pone.0001602-Wettenhall1">[53]</a>. Within Limma, prior to channel normalization, microarray outputs were filtered to remove spots of poor signal quality by excluding those data points with mean intensity less than two standard deviations above background in both channels. Then, global LOESS normalization was used to normalize all data, and the 2-replicate spots per gene in each array were used to maximize the robustness of differential expression measurement of each gene via the “lmFit” function <a href="#pone.0001602-Smyth1">[54]</a>. The microarray data have been deposited in the Gene Expression Omnibus database under accession no. GSE9636.</p>


<h4>Assay of intracellular Fe and Mn ion concentration</h4>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1"></a><p>Total iron and Mn concentration were detected as described previously with some modification <a href="#pone.0001602-Ma1">[55]</a>. Bacteria were grown aerobically to OD600 0.8 in 600 ml of TGY broth. After centrifugation at 10,000 g for 10 min at 4°C, cells were washed twice in 200 ml of phosphate-buffered saline (PBS) with 1 mM EDTA (pH 7.4) and resuspended in 200 ml of PBS without EDTA. After centrifugation, the pellet was resuspended in 10 ml of PBS, 8 ml of which was used for iron and manganese analysis. Cell dry weight was estimated with the remaining 2ml suspension. For iron and manganese analysis, pelleted bacteria were resuspended in 1 ml of Ultrex II nitric acid (Fluka AG., Buchs, Switzerland) and incubated at 80°C for 1 hour. After centrifugation at 20,000 g for 20 minutes, the supernatant was filtered against 0.45 µM membrane. The concentration of samples was analyzed for iron and Mn content by inductively coupled plasma-optical emission spectroscopy (ICP-MS, Model Agilent 7500a, Hewlett-Packard, Yokogawa Analytical Systems, Tokyo, Japan). All buffers and nitric acid solutions were analyzed as described above to correct for background.</p>


<h4>Gel mobility shift assays</h4>
<a id="article1.body1.sec4.sec13.p1" name="article1.body1.sec4.sec13.p1"></a><p>Gel mobility shift assays were performed with FITC-labeled DNA fragments (0.05 pmol) mixed with purified OxyR protein (oxidized protein or reduced protein, 200 nmol) in a total volume of 20 µL. The binding buffer contained 10 mM Tris-Cl (pH 8.0), 50 mM NaCl, 1 mM EDTA, 5% Glycerol, 50 µg/mL BSA and 5 µg/mL calf thymus DNA <a href="#pone.0001602-Zeller1">[22]</a>, <a href="#pone.0001602-Fuangthong1">[39]</a>. The reaction mixture was incubated at room temperature for 30 min and loaded onto a 1.5% agarose gel in 0.5×TBE. Electrophoresis was performed at 80 V for 1 h at 4°C and followed by photographed <a href="#pone.0001602-Brune1">[56]</a>, <a href="#pone.0001602-Timmins1">[57]</a>. For generation of the labeled DNA, the appropriate operator fragment of target genes (<em>dr1709</em>, <em>dr1998</em>, <em>dr2263</em>, <em>drB0036</em>, and <em>drB0125</em>) was amplified by PCR with genomic DNA and cloned into pMD18, followed by FITC-labeled RV-M and reverse priming of target genes. In addition, <em>dr0207</em> was used as negative control. All primers are listed in <a href="#pone-0001602-t002">Table 2</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001602"></div><a name="pone.0001602.s001" id="pone.0001602.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001602.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">H2O2 disk assay. Photos showing the zones of inhibition by H2O2 in E. coli K-12 (wild type) (A), GS09 (oxyR::kan mutant) (B), and E. coli K-12 strain GS09 complemented with the droxyR gene (C). (D) Histogram showing the results of H2O2 disk assay in E. coli. In the assay, E. coli cells were grown in Luria-Bertani (LB) broth at 37 °C with overnight shaking. 200 µl of the overnight cultures were added to LB top agar and spread onto LB agar. Then, 10 µl of 3% H2O2 was pipetted onto 3MM Whatman paper disks (0.7-cm diameter), and these disks were placed on top of the agar and incubated at 37 °C overnight. The zone of inhibition, in mm, was taken as a measure of H2O2 sensitivity. The zone of inhibition was measured in three dimensions, and the mean values and standard deviations were calculated.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001602.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(4.13 MB TIF)</p>
<a name="pone.0001602.s002" id="pone.0001602.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001602.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Disruption of D. radiodurans droxyR gene. Verification of droxyR gene disruption by PCR analysis. Purified PCR fragments were amplified from the genomic DNA of strain R1 and MOxyR using primers (OxyR1 and OxyR5) that flank the coding sequences for droxyR. The PCR products of R1 revealed a band of ~2550 bp length (band 1), whereas those of MOxyR resulted in a ~3500 bp fragment (band 2). Bands 3 and 4 denote PCR products of R1 and MOxyR were digested with BamHI, respectively. Bands 5 and 6 denote PCR products of R1 and MOxyR were digested with HindIII, respectively. M denotes molecular weight standards.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001602.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(2.63 MB TIF)</p>
<a name="pone.0001602.s003" id="pone.0001602.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001602.s003">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">The repressed genes showed in MOxyR. All genes are sorted by fold induction or repression.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001602.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0001602.s004" id="pone.0001602.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001602.s004">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">The induced genes showed in MOxyR. All genes are sorted by fold induction or repression.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001602.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.09 MB DOC)</p>
<a name="pone.0001602.s005" id="pone.0001602.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001602.s005">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Functional classification of genes with statistically significantly induction or repression in untreated wild-type strains compared to untreated oxyR mutant.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001602.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Dr Gisela Storz (National Institutes of Health) for providing the GS09 strains. Microarray and Q-RT PCR were performed at the Centre of Analysis &amp; Measurement of Zhejiang University, China. The article is contributed to the 50th anniversary of Institute of Nuclear-Agricultural Sciences, Zhejiang University.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YH HC SH. Performed the experiments: HC GX YZ HL XY ZX NY. Analyzed the data: HC GX. Contributed reagents/materials/analysis tools: YH SH. Wrote the paper: YH HC BT SH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001602-Imlay1" id="pone.0001602-Imlay1"></a>Imlay JA (2003) Pathways of oxidative damage. Annu Rev Microbiol  57: 395–418.  <ul class="find" data-citedArticleID="1056152" data-doi="10.1146/annurev.micro.57.030502.090938"><li><a href="http://dx.doi.org/10.1146/annurev.micro.57.030502.090938" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathways+of+oxidative+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathways+of+oxidative+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001602-Zheng1" id="pone.0001602-Zheng1"></a>Zheng M, Aslund F, Storz G (1998) Activation of the OxyR transcription factor by reversible disulfide bond formation. Science  279: 1718–1721.  <ul class="find" data-citedArticleID="1056216" data-doi="10.1126/science.279.5357.1718"><li><a href="http://dx.doi.org/10.1126/science.279.5357.1718" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+of+the+OxyR+transcription+factor+by+reversible+disulfide+bond+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+of+the+OxyR+transcription+factor+by+reversible+disulfide+bond+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001602-Christman1" id="pone.0001602-Christman1"></a>Christman MF, Morgan RW, Jacobson FS, Ames BN (1985) Positive control of a regulon for defenses against oxidative stress and some heat-shock proteins in Salmonella typhimurium. Cell  41: 753–762.  <ul class="find" data-citedArticleID="1056116" data-doi="10.1016/s0092-8674(85)80056-8"><li><a href="http://dx.doi.org/10.1016/s0092-8674(85)80056-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+control+of+a+regulon+for+defenses+against+oxidative+stress+and+some+heat-shock+proteins+in+Salmonella+typhimurium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+control+of+a+regulon+for+defenses+against+oxidative+stress+and+some+heat-shock+proteins+in+Salmonella+typhimurium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001602-Helmann1" id="pone.0001602-Helmann1"></a>Helmann JD (2002) OxyR: a molecular code for redox sensing? Sci STKE  2002: PE46.  <ul class="find" data-citedArticleID="1056146" data-doi="10.1126/stke.2002.157.pe46"><li><a href="http://dx.doi.org/10.1126/stke.2002.157.pe46" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OxyR%3A+a+molecular+code+for+redox+sensing%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OxyR%3A+a+molecular+code+for+redox+sensing%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001602-Tseng1" id="pone.0001602-Tseng1"></a>Tseng HJ, McEwan AG, Apicella MA, Jennings MP (2003) OxyR acts as a repressor of catalase expression in Neisseria gonorrhoeae. Infect Immun  71: 550–556.  <ul class="find" data-citedArticleID="1056202" data-doi="10.1128/iai.71.1.550-556.2003"><li><a href="http://dx.doi.org/10.1128/iai.71.1.550-556.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OxyR+acts+as+a+repressor+of+catalase+expression+in+Neisseria+gonorrhoeae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OxyR+acts+as+a+repressor+of+catalase+expression+in+Neisseria+gonorrhoeae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001602-Lee1" id="pone.0001602-Lee1"></a>Lee C, Lee SM, Mukhopadhyay P, Kim SJ, Lee SC, et al.  (2004) Redox regulation of OxyR requires specific disulfide bond formation involving a rapid kinetic reaction path. Nat Struct Mol Biol  11: 1179–1185.  <ul class="find" data-citedArticleID="1056166" data-doi="10.1038/nsmb856"><li><a href="http://dx.doi.org/10.1038/nsmb856" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Redox+regulation+of+OxyR+requires+specific+disulfide+bond+formation+involving+a+rapid+kinetic+reaction+path." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Redox+regulation+of+OxyR+requires+specific+disulfide+bond+formation+involving+a+rapid+kinetic+reaction+path.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001602-Toledano1" id="pone.0001602-Toledano1"></a>Toledano MB, Kullik I, Trinh F, Baird PT, Schneider TD, et al.  (1994) Redox-dependent shift of OxyR-DNA contacts along an extended DNA-binding site: a mechanism for differential promoter selection. Cell  78: 897–909.  <ul class="find" data-citedArticleID="1056200" data-doi="10.1016/s0092-8674(94)90702-1"><li><a href="http://dx.doi.org/10.1016/s0092-8674(94)90702-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Redox-dependent+shift+of+OxyR-DNA+contacts+along+an+extended+DNA-binding+site%3A+a+mechanism+for+differential+promoter+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Redox-dependent+shift+of+OxyR-DNA+contacts+along+an+extended+DNA-binding+site%3A+a+mechanism+for+differential+promoter+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001602-Kim1" id="pone.0001602-Kim1"></a>Kim SO, Merchant K, Nudelman R, Beyer WF Jr, Keng T, et al.  (2002) OxyR: a molecular code for redox-related signaling. Cell  109: 383–396.  <ul class="find" data-citedArticleID="1056158" data-doi="10.1016/s0092-8674(02)00723-7"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)00723-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OxyR%3A+a+molecular+code+for+redox-related+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OxyR%3A+a+molecular+code+for+redox-related+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001602-Battista1" id="pone.0001602-Battista1"></a>Battista JR, Earl AM, Park MJ (1999) Why is Deinococcus radiodurans so resistant to ionizing radiation? Trends Microbiol  7: 362–365.  <ul class="find" data-citedArticleID="1056104" data-doi="10.1016/s0966-842x(99)01566-8"><li><a href="http://dx.doi.org/10.1016/s0966-842x(99)01566-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+is+Deinococcus+radiodurans+so+resistant+to+ionizing+radiation%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+is+Deinococcus+radiodurans+so+resistant+to+ionizing+radiation%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001602-Minton1" id="pone.0001602-Minton1"></a>Minton KW (1994) DNA repair in the extremely radioresistant bacterium Deinococcus radiodurans. Mol Microbiol  13: 9–15.  <ul class="find" data-citedArticleID="1056176" data-doi="10.1111/j.1365-2958.1994.tb00397.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1994.tb00397.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+repair+in+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+repair+in+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001602-Tanaka1" id="pone.0001602-Tanaka1"></a>Tanaka M, Narumi I, Funayama T, Kikuchi M, Watanabe H, et al.  (2005) Characterization of pathways dependent on the uvsE, uvrA1, or uvrA2 gene product for UV resistance in Deinococcus radiodurans. J Bacteriol  187: 3693–3697.  <ul class="find" data-citedArticleID="1056190" data-doi="10.1128/jb.187.11.3693-3697.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.11.3693-3697.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+pathways+dependent+on+the+uvsE%2C+uvrA1%2C+or+uvrA2+gene+product+for+UV+resistance+in+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+pathways+dependent+on+the+uvsE%2C+uvrA1%2C+or+uvrA2+gene+product+for+UV+resistance+in+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001602-Earl1" id="pone.0001602-Earl1"></a>Earl AM, Rankin SK, Kim KP, Lamendola ON, Battista JR (2002) Genetic evidence that the uvsE gene product of Deinococcus radiodurans R1 is a UV damage endonuclease. J Bacteriol  184: 1003–1009.  <ul class="find" data-citedArticleID="1056130" data-doi="10.1128/jb.184.4.1003-1009.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.4.1003-1009.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+evidence+that+the+uvsE+gene+product+of+Deinococcus+radiodurans+R1+is+a+UV+damage+endonuclease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+evidence+that+the+uvsE+gene+product+of+Deinococcus+radiodurans+R1+is+a+UV+damage+endonuclease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001602-Markillie1" id="pone.0001602-Markillie1"></a>Markillie LM, Varnum SM, Hradecky P, Wong KK (1999) Targeted mutagenesis by duplication insertion in the radioresistant bacterium Deinococcus radiodurans: radiation sensitivities of catalase (katA) and superoxide dismutase (sodA) mutants. J Bacteriol  181: 666–669.  <ul class="find" data-citedArticleID="1056174"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Targeted+mutagenesis+by+duplication+insertion+in+the+radioresistant+bacterium+Deinococcus+radiodurans%3A+radiation+sensitivities+of+catalase+%28katA%29+and+superoxide+dismutase+%28sodA%29+mutants.&amp;auth=&amp;atitle=Targeted+mutagenesis+by+duplication+insertion+in+the+radioresistant+bacterium+Deinococcus+radiodurans%3A+radiation+sensitivities+of+catalase+%28katA%29+and+superoxide+dismutase+%28sodA%29+mutants." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeted+mutagenesis+by+duplication+insertion+in+the+radioresistant+bacterium+Deinococcus+radiodurans%3A+radiation+sensitivities+of+catalase+%28katA%29+and+superoxide+dismutase+%28sodA%29+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeted+mutagenesis+by+duplication+insertion+in+the+radioresistant+bacterium+Deinococcus+radiodurans%3A+radiation+sensitivities+of+catalase+%28katA%29+and+superoxide+dismutase+%28sodA%29+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001602-Tanaka2" id="pone.0001602-Tanaka2"></a>Tanaka M, Earl AM, Howell HA, Park MJ, Eisen JA, et al.  (2004) Analysis of Deinococcus radiodurans's transcriptional response to ionizing radiation and desiccation reveals novel proteins that contribute to extreme radioresistance. Genetics  168: 21–33.  <ul class="find" data-citedArticleID="1056192" data-doi="10.1534/genetics.104.029249"><li><a href="http://dx.doi.org/10.1534/genetics.104.029249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+Deinococcus+radiodurans%27s+transcriptional+response+to+ionizing+radiation+and+desiccation+reveals+novel+proteins+that+contribute+to+extreme+radioresistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+Deinococcus+radiodurans%27s+transcriptional+response+to+ionizing+radiation+and+desiccation+reveals+novel+proteins+that+contribute+to+extreme+radioresistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001602-Ghosal1" id="pone.0001602-Ghosal1"></a>Ghosal D, Omelchenko MV, Gaidamakova EK, Matrosova VY, Vasilenko A, et al.  (2005) How radiation kills cells: survival of Deinococcus radiodurans and Shewanella oneidensis under oxidative stress. FEMS Microbiol Rev  29: 361–375.  <ul class="find" data-citedArticleID="1056140" data-doi="10.1016/j.fmrre.2004.12.007"><li><a href="http://dx.doi.org/10.1016/j.fmrre.2004.12.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+radiation+kills+cells%3A+survival+of+Deinococcus+radiodurans+and+Shewanella+oneidensis+under+oxidative+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+radiation+kills+cells%3A+survival+of+Deinococcus+radiodurans+and+Shewanella+oneidensis+under+oxidative+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001602-Seib1" id="pone.0001602-Seib1"></a>Seib KL, Wu HJ, Srikhanta YN, Edwards JL, Falsetta ML, et al.  (2007) Characterization of the OxyR regulon of Neisseria gonorrhoeae. Mol Microbiol  63: 54–68.  <ul class="find" data-citedArticleID="1056186" data-doi="10.1111/j.1365-2958.2006.05478.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2006.05478.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+OxyR+regulon+of+Neisseria+gonorrhoeae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+OxyR+regulon+of+Neisseria+gonorrhoeae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001602-Kullik1" id="pone.0001602-Kullik1"></a>Kullik I, Stevens J, Toledano MB, Storz G (1995) Mutational analysis of the redox-sensitive transcriptional regulator OxyR: regions important for DNA binding and multimerization. J Bacteriol  177: 1285–1291.  <ul class="find" data-citedArticleID="1056162"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mutational+analysis+of+the+redox-sensitive+transcriptional+regulator+OxyR%3A+regions+important+for+DNA+binding+and+multimerization.&amp;auth=&amp;atitle=Mutational+analysis+of+the+redox-sensitive+transcriptional+regulator+OxyR%3A+regions+important+for+DNA+binding+and+multimerization." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutational+analysis+of+the+redox-sensitive+transcriptional+regulator+OxyR%3A+regions+important+for+DNA+binding+and+multimerization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutational+analysis+of+the+redox-sensitive+transcriptional+regulator+OxyR%3A+regions+important+for+DNA+binding+and+multimerization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001602-Wang1" id="pone.0001602-Wang1"></a>Wang X, Mukhopadhyay P, Wood MJ, Outten FW, Opdyke JA, et al.  (2006) Mutational analysis to define an activating region on the redox-sensitive transcriptional regulator OxyR. J Bacteriol  188: 8335–8342.  <ul class="find" data-citedArticleID="1056208" data-doi="10.1128/jb.01318-06"><li><a href="http://dx.doi.org/10.1128/jb.01318-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutational+analysis+to+define+an+activating+region+on+the+redox-sensitive+transcriptional+regulator+OxyR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutational+analysis+to+define+an+activating+region+on+the+redox-sensitive+transcriptional+regulator+OxyR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001602-Christman2" id="pone.0001602-Christman2"></a>Christman MF, Storz G, Ames BN (1989) OxyR, a positive regulator of hydrogen peroxide-inducible genes in Escherichia coli and Salmonella typhimurium, is homologous to a family of bacterial regulatory proteins. Proc Natl Acad Sci U S A  86: 3484–3488.  <ul class="find" data-citedArticleID="1056118" data-doi="10.1073/pnas.86.10.3484"><li><a href="http://dx.doi.org/10.1073/pnas.86.10.3484" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OxyR%2C+a+positive+regulator+of+hydrogen+peroxide-inducible+genes+in+Escherichia+coli+and+Salmonella+typhimurium%2C+is+homologous+to+a+family+of+bacterial+regulatory+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OxyR%2C+a+positive+regulator+of+hydrogen+peroxide-inducible+genes+in+Escherichia+coli+and+Salmonella+typhimurium%2C+is+homologous+to+a+family+of+bacterial+regulatory+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001602-Ellis1" id="pone.0001602-Ellis1"></a>Ellis HR, Poole LB (1997) Novel application of 7-chloro-4-nitrobenzo-2-oxa-1,3-diazole to identify cysteine sulfenic acid in the AhpC component of alkyl hydroperoxide reductase. Biochemistry  36: 15013–15018.  <ul class="find" data-citedArticleID="1056132" data-doi="10.1021/bi972191x"><li><a href="http://dx.doi.org/10.1021/bi972191x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+application+of+7-chloro-4-nitrobenzo-2-oxa-1%2C3-diazole+to+identify+cysteine+sulfenic+acid+in+the+AhpC+component+of+alkyl+hydroperoxide+reductase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+application+of+7-chloro-4-nitrobenzo-2-oxa-1%2C3-diazole+to+identify+cysteine+sulfenic+acid+in+the+AhpC+component+of+alkyl+hydroperoxide+reductase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001602-Zheng2" id="pone.0001602-Zheng2"></a>Zheng M, Wang X, Templeton LJ, Smulski DR, LaRossa RA, et al.  (2001) DNA microarray-mediated transcriptional profiling of the Escherichia coli response to hydrogen peroxide. J Bacteriol  183: 4562–4570.  <ul class="find" data-citedArticleID="1056218" data-doi="10.1128/jb.183.15.4562-4570.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.15.4562-4570.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+microarray-mediated+transcriptional+profiling+of+the+Escherichia+coli+response+to+hydrogen+peroxide." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+microarray-mediated+transcriptional+profiling+of+the+Escherichia+coli+response+to+hydrogen+peroxide.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001602-Zeller1" id="pone.0001602-Zeller1"></a>Zeller T, Mraheil MA, Moskvin OV, Li K, Gomelsky M, et al.  (2007) Regulation of hydrogen peroxide-dependent gene expression in Rhodobacter sphaeroides: regulatory functions of OxyR. J Bacteriol  189: 3784–3792.  <ul class="find" data-citedArticleID="1056214" data-doi="10.1128/jb.01795-06"><li><a href="http://dx.doi.org/10.1128/jb.01795-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+hydrogen+peroxide-dependent+gene+expression+in+Rhodobacter+sphaeroides%3A+regulatory+functions+of+OxyR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+hydrogen+peroxide-dependent+gene+expression+in+Rhodobacter+sphaeroides%3A+regulatory+functions+of+OxyR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001602-Harrison1" id="pone.0001602-Harrison1"></a>Harrison A, Ray WC, Baker BD, Armbruster DW, Bakaletz LO, et al.  (2007) The OxyR regulon in nontypeable Haemophilus influenzae. J Bacteriol  189: 1004–1012.  <ul class="find" data-citedArticleID="1056144" data-doi="10.1128/jb.01040-06"><li><a href="http://dx.doi.org/10.1128/jb.01040-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+OxyR+regulon+in+nontypeable+Haemophilus+influenzae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+OxyR+regulon+in+nontypeable+Haemophilus+influenzae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001602-Diaz1" id="pone.0001602-Diaz1"></a>Diaz PI, Slakeski N, Reynolds EC, Morona R, Rogers AH, et al.  (2006) Role of oxyR in the oral anaerobe Porphyromonas gingivalis. J Bacteriol  188: 2454–2462.  <ul class="find" data-citedArticleID="1056126" data-doi="10.1128/jb.188.7.2454-2462.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.7.2454-2462.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+oxyR+in+the+oral+anaerobe+Porphyromonas+gingivalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+oxyR+in+the+oral+anaerobe+Porphyromonas+gingivalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001602-Kehres1" id="pone.0001602-Kehres1"></a>Kehres DG, Janakiraman A, Slauch JM, Maguire ME (2002) Regulation of Salmonella enterica serovar Typhimurium mntH transcription by H(2)O(2), Fe(2+), and Mn(2+). J Bacteriol  184: 3151–3158.  <ul class="find" data-citedArticleID="1056156" data-doi="10.1128/jb.184.12.3151-3158.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.12.3151-3158.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+Salmonella+enterica+serovar+Typhimurium+mntH+transcription+by+H%282%29O%282%29%2C+Fe%282%2B%29%2C+and+Mn%282%2B%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+Salmonella+enterica+serovar+Typhimurium+mntH+transcription+by+H%282%29O%282%29%2C+Fe%282%2B%29%2C+and+Mn%282%2B%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001602-Kim2" id="pone.0001602-Kim2"></a>Kim SG, Bhattacharyya G, Grove A, Lee YH (2006) Crystal structure of Dps-1, a functionally distinct Dps protein from Deinococcus radiodurans. J Mol Biol  361: 105–114.  <ul class="find" data-citedArticleID="1056160" data-doi="10.1016/j.jmb.2006.06.010"><li><a href="http://dx.doi.org/10.1016/j.jmb.2006.06.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+Dps-1%2C+a+functionally+distinct+Dps+protein+from+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+Dps-1%2C+a+functionally+distinct+Dps+protein+from+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001602-Cuypers1" id="pone.0001602-Cuypers1"></a>Cuypers MG, Mitchell EP, Romao CV, McSweeney SM (2007) The crystal structure of the Dps2 from Deinococcus radiodurans reveals an unusual pore profile with a non-specific metal binding site. J Mol Biol  371: 787–799.  <ul class="find" data-citedArticleID="1056120" data-doi="10.1016/j.jmb.2006.11.032"><li><a href="http://dx.doi.org/10.1016/j.jmb.2006.11.032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+crystal+structure+of+the+Dps2+from+Deinococcus+radiodurans+reveals+an+unusual+pore+profile+with+a+non-specific+metal+binding+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+crystal+structure+of+the+Dps2+from+Deinococcus+radiodurans+reveals+an+unusual+pore+profile+with+a+non-specific+metal+binding+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001602-Chen1" id="pone.0001602-Chen1"></a>Chen H, Xu ZJ, Tian B, Chen WW, Hu SN, et al.  (2007) Transcriptional profile in response to ionizing radiation at low dose in Deinococcus radiodurans. Progress in Natural Science  17: 525–536.  <ul class="find" data-citedArticleID="1056112" data-doi="10.1080/10020070708541032"><li><a href="http://dx.doi.org/10.1080/10020070708541032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+profile+in+response+to+ionizing+radiation+at+low+dose+in+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+profile+in+response+to+ionizing+radiation+at+low+dose+in+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001602-Daly1" id="pone.0001602-Daly1"></a>Daly MJ, Gaidamakova EK, Matrosova VY, Vasilenko A, Zhai M, et al.  (2004) Accumulation of Mn(II) in Deinococcus radiodurans facilitates gamma-radiation resistance. Science  306: 1025–1028.  <ul class="find" data-citedArticleID="1056122" data-doi="10.1126/science.1103185"><li><a href="http://dx.doi.org/10.1126/science.1103185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accumulation+of+Mn%28II%29+in+Deinococcus+radiodurans+facilitates+gamma-radiation+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accumulation+of+Mn%28II%29+in+Deinococcus+radiodurans+facilitates+gamma-radiation+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001602-Bsat1" id="pone.0001602-Bsat1"></a>Bsat N, Herbig A, Casillas-Martinez L, Setlow P, Helmann JD (1998) Bacillus subtilis contains multiple Fur homologues: identification of the iron uptake (Fur) and peroxide regulon (PerR) repressors. Mol Microbiol  29: 189–198.  <ul class="find" data-citedArticleID="1056110" data-doi="10.1046/j.1365-2958.1998.00921.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1998.00921.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacillus+subtilis+contains+multiple+Fur+homologues%3A+identification+of+the+iron+uptake+%28Fur%29+and+peroxide+regulon+%28PerR%29+repressors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacillus+subtilis+contains+multiple+Fur+homologues%3A+identification+of+the+iron+uptake+%28Fur%29+and+peroxide+regulon+%28PerR%29+repressors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001602-Rocha1" id="pone.0001602-Rocha1"></a>Rocha ER, Herren CD, Smalley DJ, Smith CJ (2003) The complex oxidative stress response of Bacteroides fragilis: the role of OxyR in control of gene expression. Anaerobe  9: 165–173.  <ul class="find" data-citedArticleID="1056182" data-doi="10.1016/s0378-1119(03)00759-5"><li><a href="http://dx.doi.org/10.1016/s0378-1119(03)00759-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complex+oxidative+stress+response+of+Bacteroides+fragilis%3A+the+role+of+OxyR+in+control+of+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complex+oxidative+stress+response+of+Bacteroides+fragilis%3A+the+role+of+OxyR+in+control+of+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001602-RunyenJanecky1" id="pone.0001602-RunyenJanecky1"></a>Runyen-Janecky L, Dazenski E, Hawkins S, Warner L (2006) Role and regulation of the Shigella flexneri sit and MntH systems. Infect Immun  74: 4666–4672.  <ul class="find" data-citedArticleID="1056184" data-doi="10.1128/iai.00562-06"><li><a href="http://dx.doi.org/10.1128/iai.00562-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+and+regulation+of+the+Shigella+flexneri+sit+and+MntH+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+and+regulation+of+the+Shigella+flexneri+sit+and+MntH+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001602-Lander1" id="pone.0001602-Lander1"></a>Lander HM (1997) An essential role for free radicals and derived species in signal transduction. Faseb J  11: 118–124.  <ul class="find" data-citedArticleID="1056164"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+essential+role+for+free+radicals+and+derived+species+in+signal+transduction.&amp;auth=&amp;atitle=An+essential+role+for+free+radicals+and+derived+species+in+signal+transduction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+essential+role+for+free+radicals+and+derived+species+in+signal+transduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+essential+role+for+free+radicals+and+derived+species+in+signal+transduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001602-Veal1" id="pone.0001602-Veal1"></a>Veal EA, Day AM, Morgan BA (2007) Hydrogen peroxide sensing and signaling. Mol Cell  26: 1–14.  <ul class="find" data-citedArticleID="1056204" data-doi="10.1016/j.molcel.2007.03.016"><li><a href="http://dx.doi.org/10.1016/j.molcel.2007.03.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hydrogen+peroxide+sensing+and+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hydrogen+peroxide+sensing+and+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001602-Liu1" id="pone.0001602-Liu1"></a>Liu Y, Zhou J, Omelchenko MV, Beliaev AS, Venkateswaran A, et al.  (2003) Transcriptome dynamics of Deinococcus radiodurans recovering from ionizing radiation. Proc Natl Acad Sci U S A  100: 4191–4196.  <ul class="find" data-citedArticleID="1056170" data-doi="10.1073/pnas.0630387100"><li><a href="http://dx.doi.org/10.1073/pnas.0630387100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptome+dynamics+of+Deinococcus+radiodurans+recovering+from+ionizing+radiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptome+dynamics+of+Deinococcus+radiodurans+recovering+from+ionizing+radiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001602-Harris1" id="pone.0001602-Harris1"></a>Harris DR, Tanaka M, Saveliev SV, Jolivet E, Earl AM, et al.  (2004) Preserving genome integrity: the DdrA protein of Deinococcus radiodurans R1. PLoS Biol  2: e304.  <ul class="find" data-citedArticleID="1056142" data-doi="10.1371/journal.pbio.0020304"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020304" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preserving+genome+integrity%3A+the+DdrA+protein+of+Deinococcus+radiodurans+R1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preserving+genome+integrity%3A+the+DdrA+protein+of+Deinococcus+radiodurans+R1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001602-Xu1" id="pone.0001602-Xu1"></a>Xu Z, Tian B, Sun Z, Lin J, Hua Y (2007) Identification and functional analysis of a phytoene desaturase gene from the extremely radioresistant bacterium Deinococcus radiodurans. Microbiology  153: 1642–1652.  <ul class="find" data-citedArticleID="1056212" data-doi="10.1099/mic.0.2006/002857-0"><li><a href="http://dx.doi.org/10.1099/mic.0.2006/002857-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+functional+analysis+of+a+phytoene+desaturase+gene+from+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+functional+analysis+of+a+phytoene+desaturase+gene+from+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001602-Daly2" id="pone.0001602-Daly2"></a>Daly MJ, Gaidamakova EK, Matrosova VY, Vasilenko A, Zhai M, et al.  (2007) Protein oxidation implicated as the primary determinant of bacterial radioresistance. PLoS Biol  5: e92.  <ul class="find" data-citedArticleID="1056124" data-doi="10.1371/journal.pbio.0050092"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0050092" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+oxidation+implicated+as+the+primary+determinant+of+bacterial+radioresistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+oxidation+implicated+as+the+primary+determinant+of+bacterial+radioresistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001602-Fuangthong1" id="pone.0001602-Fuangthong1"></a>Fuangthong M, Helmann JD (2002) The OhrR repressor senses organic hydroperoxides by reversible formation of a cysteine-sulfenic acid derivative. Proc Natl Acad Sci U S A  99: 6690–6695.  <ul class="find" data-citedArticleID="1056136" data-doi="10.1073/pnas.102483199"><li><a href="http://dx.doi.org/10.1073/pnas.102483199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+OhrR+repressor+senses+organic+hydroperoxides+by+reversible+formation+of+a+cysteine-sulfenic+acid+derivative." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+OhrR+repressor+senses+organic+hydroperoxides+by+reversible+formation+of+a+cysteine-sulfenic+acid+derivative.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001602-Lee2" id="pone.0001602-Lee2"></a>Lee JW, Soonsanga S, Helmann JD (2007) A complex thiolate switch regulates the Bacillus subtilis organic peroxide sensor OhrR. Proc Natl Acad Sci U S A  104: 8743–8748.  <ul class="find" data-citedArticleID="1056168" data-doi="10.1073/pnas.0702081104"><li><a href="http://dx.doi.org/10.1073/pnas.0702081104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+complex+thiolate+switch+regulates+the+Bacillus+subtilis+organic+peroxide+sensor+OhrR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+complex+thiolate+switch+regulates+the+Bacillus+subtilis+organic+peroxide+sensor+OhrR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001602-Du1" id="pone.0001602-Du1"></a>Du X, Takagi H (2007) N-Acetyltransferase Mpr1 confers ethanol tolerance on Saccharomyces cerevisiae by reducing reactive oxygen species. Appl Microbiol Biotechnol  75: 1343–1351.  <ul class="find" data-citedArticleID="1056128" data-doi="10.1007/s00253-007-0940-x"><li><a href="http://dx.doi.org/10.1007/s00253-007-0940-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=N-Acetyltransferase+Mpr1+confers+ethanol+tolerance+on+Saccharomyces+cerevisiae+by+reducing+reactive+oxygen+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22N-Acetyltransferase+Mpr1+confers+ethanol+tolerance+on+Saccharomyces+cerevisiae+by+reducing+reactive+oxygen+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001602-Wallecha1" id="pone.0001602-Wallecha1"></a>Wallecha A, Correnti J, Munster V, van der Woude M (2003) Phase variation of Ag43 is independent of the oxidation state of OxyR. J Bacteriol  185: 2203–2209.  <ul class="find" data-citedArticleID="1056206" data-doi="10.1128/jb.185.7.2203-2209.2003"><li><a href="http://dx.doi.org/10.1128/jb.185.7.2203-2209.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phase+variation+of+Ag43+is+independent+of+the+oxidation+state+of+OxyR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phase+variation+of+Ag43+is+independent+of+the+oxidation+state+of+OxyR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001602-Gao1" id="pone.0001602-Gao1"></a>Gao GJ, Lu HM, Huang LF, Hua YJ (2005) Construction of DNA damage response gene pprI function deficient and function complementary mutants in Deinococcus radiodurans. Chinese Science Bulletin  50: 311–316.  <ul class="find" data-citedArticleID="1056138" data-doi="10.1360/982004-719"><li><a href="http://dx.doi.org/10.1360/982004-719" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+DNA+damage+response+gene+pprI+function+deficient+and+function+complementary+mutants+in+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+DNA+damage+response+gene+pprI+function+deficient+and+function+complementary+mutants+in+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001602-Hua1" id="pone.0001602-Hua1"></a>Hua Y, Narumi I, Gao G, Tian B, Satoh K, et al.  (2003) PprI: a general switch responsible for extreme radioresistance of Deinococcus radiodurans. Biochem Biophys Res Commun  306: 354–360.  <ul class="find" data-citedArticleID="1056148" data-doi="10.1016/s0006-291x(03)00965-3"><li><a href="http://dx.doi.org/10.1016/s0006-291x(03)00965-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PprI%3A+a+general+switch+responsible+for+extreme+radioresistance+of+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PprI%3A+a+general+switch+responsible+for+extreme+radioresistance+of+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001602-Brenot1" id="pone.0001602-Brenot1"></a>Brenot A, King KY, Caparon MG (2005) The PerR regulon in peroxide resistance and virulence of Streptococcus pyogenes. Mol Microbiol  55: 221–234.  <ul class="find" data-citedArticleID="1056106" data-doi="10.1111/j.1365-2958.2004.04370.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04370.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+PerR+regulon+in+peroxide+resistance+and+virulence+of+Streptococcus+pyogenes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+PerR+regulon+in+peroxide+resistance+and+virulence+of+Streptococcus+pyogenes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001602-Raahave1" id="pone.0001602-Raahave1"></a>Raahave D (1974) Paper disk-agar diffusion assay of penicillin in the presence of streptomycin. Antimicrob Agents Chemother  6: 603–605.  <ul class="find" data-citedArticleID="1056180" data-doi="10.1128/aac.6.5.603"><li><a href="http://dx.doi.org/10.1128/aac.6.5.603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Paper+disk-agar+diffusion+assay+of+penicillin+in+the+presence+of+streptomycin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Paper+disk-agar+diffusion+assay+of+penicillin+in+the+presence+of+streptomycin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001602-Huang1" id="pone.0001602-Huang1"></a>Huang L, Hua X, Lu H, Gao G, Tian B, et al.  (2007) Three tandem HRDC domains have synergistic effect on the RecQ functions in Deinococcus radiodurans. DNA Repair (Amst)  6: 167–176.  <ul class="find" data-citedArticleID="1056150" data-doi="10.1016/j.dnarep.2006.09.006"><li><a href="http://dx.doi.org/10.1016/j.dnarep.2006.09.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three+tandem+HRDC+domains+have+synergistic+effect+on+the+RecQ+functions+in+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three+tandem+HRDC+domains+have+synergistic+effect+on+the+RecQ+functions+in+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001602-Tian1" id="pone.0001602-Tian1"></a>Tian B, Wu Y, Sheng D, Zheng Z, Gao G, et al.  (2004) Chemiluminescence assay for reactive oxygen species scavenging activities and inhibition on oxidative damage of DNA in Deinococcus radiodurans. Luminescence  19: 78–84.  <ul class="find" data-citedArticleID="1056196" data-doi="10.1002/bio.761"><li><a href="http://dx.doi.org/10.1002/bio.761" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemiluminescence+assay+for+reactive+oxygen+species+scavenging+activities+and+inhibition+on+oxidative+damage+of+DNA+in+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemiluminescence+assay+for+reactive+oxygen+species+scavenging+activities+and+inhibition+on+oxidative+damage+of+DNA+in+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001602-Esposito1" id="pone.0001602-Esposito1"></a>Esposito P, Varvara G, Caputi S, Perinetti G (2003) Catalase activity in human healthy and inflamed dental pulps. Int Endod J  36: 599–603.  <ul class="find" data-citedArticleID="1056134" data-doi="10.1046/j.1365-2591.2003.00692.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2591.2003.00692.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Catalase+activity+in+human+healthy+and+inflamed+dental+pulps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Catalase+activity+in+human+healthy+and+inflamed+dental+pulps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001602-Kang1" id="pone.0001602-Kang1"></a>Kang SW, Baines IC, Rhee SG (1998) Characterization of a mammalian peroxiredoxin that contains one conserved cysteine. J Biol Chem  273: 6303–6311.  <ul class="find" data-citedArticleID="1056154" data-doi="10.1074/jbc.273.11.6303"><li><a href="http://dx.doi.org/10.1074/jbc.273.11.6303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+mammalian+peroxiredoxin+that+contains+one+conserved+cysteine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+mammalian+peroxiredoxin+that+contains+one+conserved+cysteine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001602-Panmanee1" id="pone.0001602-Panmanee1"></a>Panmanee W, Vattanaviboon P, Poole LB, Mongkolsuk S (2006) Novel organic hydroperoxide-sensing and responding mechanisms for OhrR, a major bacterial sensor and regulator of organic hydroperoxide stress. J Bacteriol  188: 1389–1395.  <ul class="find" data-citedArticleID="1056178" data-doi="10.1128/jb.188.4.1389-1395.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.4.1389-1395.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+organic+hydroperoxide-sensing+and+responding+mechanisms+for+OhrR%2C+a+major+bacterial+sensor+and+regulator+of+organic+hydroperoxide+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+organic+hydroperoxide-sensing+and+responding+mechanisms+for+OhrR%2C+a+major+bacterial+sensor+and+regulator+of+organic+hydroperoxide+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001602-Thompson1" id="pone.0001602-Thompson1"></a>Thompson DK, Beliaev AS, Giometti CS, Tollaksen SL, Khare T, et al.  (2002) Transcriptional and proteomic analysis of a ferric uptake regulator (fur) mutant of Shewanella oneidensis: possible involvement of fur in energy metabolism, transcriptional regulation, and oxidative stress. Appl Environ Microbiol  68: 881–892.  <ul class="find" data-citedArticleID="1056194" data-doi="10.1128/aem.68.2.881-892.2002"><li><a href="http://dx.doi.org/10.1128/aem.68.2.881-892.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+and+proteomic+analysis+of+a+ferric+uptake+regulator+%28fur%29+mutant+of+Shewanella+oneidensis%3A+possible+involvement+of+fur+in+energy+metabolism%2C+transcriptional+regulation%2C+and+oxidative+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+and+proteomic+analysis+of+a+ferric+uptake+regulator+%28fur%29+mutant+of+Shewanella+oneidensis%3A+possible+involvement+of+fur+in+energy+metabolism%2C+transcriptional+regulation%2C+and+oxidative+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001602-Wettenhall1" id="pone.0001602-Wettenhall1"></a>Wettenhall JM, Smyth GK (2004) limmaGUI: a graphical user interface for linear modeling of microarray data. Bioinformatics  20: 3705–3706.  <ul class="find" data-citedArticleID="1056210" data-doi="10.1093/bioinformatics/bth449"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth449" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=limmaGUI%3A+a+graphical+user+interface+for+linear+modeling+of+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22limmaGUI%3A+a+graphical+user+interface+for+linear+modeling+of+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001602-Smyth1" id="pone.0001602-Smyth1"></a>Smyth GK, Michaud J, Scott HS (2005) Use of within-array replicate spots for assessing differential expression in microarray experiments. Bioinformatics  21: 2067–2075.  <ul class="find" data-citedArticleID="1056188" data-doi="10.1093/bioinformatics/bti270"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti270" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+within-array+replicate+spots+for+assessing+differential+expression+in+microarray+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+within-array+replicate+spots+for+assessing+differential+expression+in+microarray+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001602-Ma1" id="pone.0001602-Ma1"></a>Ma JF, Ochsner UA, Klotz MG, Nanayakkara VK, Howell ML, et al.  (1999) Bacterioferritin A modulates catalase A (KatA) activity and resistance to hydrogen peroxide in Pseudomonas aeruginosa. J Bacteriol  181: 3730–3742.  <ul class="find" data-citedArticleID="1056172"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bacterioferritin+A+modulates+catalase+A+%28KatA%29+activity+and+resistance+to+hydrogen+peroxide+in+Pseudomonas+aeruginosa.&amp;auth=&amp;atitle=Bacterioferritin+A+modulates+catalase+A+%28KatA%29+activity+and+resistance+to+hydrogen+peroxide+in+Pseudomonas+aeruginosa." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterioferritin+A+modulates+catalase+A+%28KatA%29+activity+and+resistance+to+hydrogen+peroxide+in+Pseudomonas+aeruginosa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterioferritin+A+modulates+catalase+A+%28KatA%29+activity+and+resistance+to+hydrogen+peroxide+in+Pseudomonas+aeruginosa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001602-Brune1" id="pone.0001602-Brune1"></a>Brune I, Werner H, Huser AT, Kalinowski J, Puhler A, et al.  (2006) The DtxR protein acting as dual transcriptional regulator directs a global regulatory network involved in iron metabolism of Corynebacterium glutamicum. BMC Genomics  7: 21.  <ul class="find" data-citedArticleID="1056108"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+DtxR+protein+acting+as+dual+transcriptional+regulator+directs+a+global+regulatory+network+involved+in+iron+metabolism+of+Corynebacterium+glutamicum.&amp;auth=&amp;atitle=The+DtxR+protein+acting+as+dual+transcriptional+regulator+directs+a+global+regulatory+network+involved+in+iron+metabolism+of+Corynebacterium+glutamicum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+DtxR+protein+acting+as+dual+transcriptional+regulator+directs+a+global+regulatory+network+involved+in+iron+metabolism+of+Corynebacterium+glutamicum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+DtxR+protein+acting+as+dual+transcriptional+regulator+directs+a+global+regulatory+network+involved+in+iron+metabolism+of+Corynebacterium+glutamicum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001602-Timmins1" id="pone.0001602-Timmins1"></a>Timmins J, Leiros I, McSweeney S (2007) Crystal structure and mutational study of RecOR provide insight into its mode of DNA binding. Embo J.   <ul class="find" data-citedArticleID="1056198" data-doi="10.1038/sj.emboj.7601760"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601760" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+and+mutational+study+of+RecOR+provide+insight+into+its+mode+of+DNA+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+and+mutational+study+of+RecOR+provide+insight+into+its+mode+of+DNA+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001602-Choi1" id="pone.0001602-Choi1"></a>Choi H, Kim S, Mukhopadhyay P, Cho S, Woo J, et al.  (2001) Structural basis of the redox switch in the OxyR transcription factor. Cell  105: 103–113.  <ul class="find" data-citedArticleID="1056114" data-doi="10.1016/s0092-8674(01)00300-2"><li><a href="http://dx.doi.org/10.1016/s0092-8674(01)00300-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+of+the+redox+switch+in+the+OxyR+transcription+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+of+the+redox+switch+in+the+OxyR+transcription+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001602-Anderson1" id="pone.0001602-Anderson1"></a>Anderson AW, Nordon HC, Cain RF, Parrish G, Duggan G (1956) Studies on a radio-resistant micrococcus. I. Isolation, morphology, cultural characteristics, and resistance to gamma radiation. Food Technol  10: 575–578.  <ul class="find" data-citedArticleID="1056102"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Studies+on+a+radio-resistant+micrococcus.+I.+Isolation%2C+morphology%2C+cultural+characteristics%2C+and+resistance+to+gamma+radiation.&amp;auth=&amp;atitle=Studies+on+a+radio-resistant+micrococcus.+I.+Isolation%2C+morphology%2C+cultural+characteristics%2C+and+resistance+to+gamma+radiation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Studies+on+a+radio-resistant+micrococcus.+I.+Isolation%2C+morphology%2C+cultural+characteristics%2C+and+resistance+to+gamma+radiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Studies+on+a+radio-resistant+micrococcus.+I.+Isolation%2C+morphology%2C+cultural+characteristics%2C+and+resistance+to+gamma+radiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.XML" value="178052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.PDF" value="496653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g001.PNG_L" value="197494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g001.PNG_M" value="337614"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g001.PNG_S" value="17537"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g001.TIF" value="1391644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g001.PNG_I" value="113023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t001.PNG_L" value="249912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t001.PNG_M" value="148335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t001.PNG_S" value="22746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t001.TIF" value="1174050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t001.PNG_I" value="159884"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t002.PNG_L" value="281868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t002.PNG_M" value="125304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t002.PNG_S" value="23615"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t002.TIF" value="1487222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t002.PNG_I" value="220386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g002.PNG_L" value="311783"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g002.PNG_M" value="50177"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g002.PNG_S" value="8355"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g002.TIF" value="670044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g002.PNG_I" value="17714"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g003.PNG_L" value="270004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g003.PNG_M" value="50210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g003.PNG_S" value="9320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g003.TIF" value="1040556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g003.PNG_I" value="17492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g004.PNG_L" value="23645"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g004.PNG_M" value="35295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g004.PNG_S" value="7197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g004.TIF" value="40922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g004.PNG_I" value="12784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g005.PNG_L" value="270073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g005.PNG_M" value="54481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g005.PNG_S" value="9842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g005.TIF" value="1050954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g005.PNG_I" value="18798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t003.PNG_L" value="42041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t003.PNG_M" value="53865"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t003.PNG_S" value="10474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t003.TIF" value="179270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t003.PNG_I" value="18263"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t004.PNG_L" value="341649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t004.PNG_M" value="149167"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t004.PNG_S" value="25722"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t004.TIF" value="1533634"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t004.PNG_I" value="230715"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t005.PNG_L" value="348548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t005.PNG_M" value="155855"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t005.PNG_S" value="26026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t005.TIF" value="1582150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t005.PNG_I" value="238596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t006.PNG_L" value="190295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t006.PNG_M" value="110354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t006.PNG_S" value="12561"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t006.TIF" value="859620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t006.PNG_I" value="33139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t007.PNG_L" value="314650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t007.PNG_M" value="179612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t007.PNG_S" value="17001"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t007.TIF" value="1462408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.t007.PNG_I" value="53671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g006.PNG_L" value="615317"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g006.PNG_M" value="91277"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g006.PNG_S" value="8607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g006.TIF" value="1837620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g006.PNG_I" value="30717"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g007.PNG_L" value="451621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g007.PNG_M" value="66782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g007.PNG_S" value="9742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g007.TIF" value="1058806"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.g007.PNG_I" value="22770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.e001.PNG" value="16150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.e001.TIF" value="33204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.s001.TIF" value="5059954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.s002.TIF" value="1155766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.s003.DOC" value="80384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.s004.DOC" value="89088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001602.s005.DOC" value="27648"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001602&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001602" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001602&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001602&volume=&issue=&title=A Novel OxyR Sensor and Regulator of Hydrogen Peroxide Stress with One Cysteine Residue in Deinococcus radiodurans&author_name=Huan%20Chen%2C%20Guangzhi%20Xu%2C%20Ye%20Zhao%2C%20Bing%20Tian%2C%20Huiming%20Lu%2C%20Xiaomin%20Yu%2C%20Zhenjian%20Xu%2C%20Nanjiao%20Ying%2C%20Songnian%20Hu%2C%20Yuejin%20Hua&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001602" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001602" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001602" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001602&amp;t=A%20Novel%20OxyR%20Sensor%20and%20Regulator%20of%20Hydrogen%20Peroxide%20Stress%20with%20One%20Cysteine%20Residue%20in%20Deinococcus%20radiodurans" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001602&title=A%20Novel%20OxyR%20Sensor%20and%20Regulator%20of%20Hydrogen%20Peroxide%20Stress%20with%20One%20Cysteine%20Residue%20in%20Deinococcus%20radiodurans&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001602&amp;title=A%20Novel%20OxyR%20Sensor%20and%20Regulator%20of%20Hydrogen%20Peroxide%20Stress%20with%20One%20Cysteine%20Residue%20in%20Deinococcus%20radiodurans" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001602" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001602&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Novel OxyR Sensor and Regulator of Hydrogen Peroxide Stress with One Cysteine Residue in Deinococcus radiodurans';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001602';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Novel%20OxyR%20Sensor%20and%20Regulator%20of%20Hydrogen%20Peroxide%20Stress%20with%20One%20Cysteine%20Residue%20in%20Deinococcus%20radiodurans http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001602" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001602" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Catalases%22" title="Search for articles in the subject area:'Catalases'"><div class="flagText">Catalases</div></a>
              <div data-categoryid="22887" data-articleid="26030"
                   data-categoryname="Catalases"
                   class="flagImage" title="Flag 'Catalases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cysteine%22" title="Search for articles in the subject area:'Cysteine'"><div class="flagText">Cysteine</div></a>
              <div data-categoryid="40627" data-articleid="26030"
                   data-categoryname="Cysteine"
                   class="flagImage" title="Flag 'Cysteine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="26030"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Disulfide+bonds%22" title="Search for articles in the subject area:'Disulfide bonds'"><div class="flagText">Disulfide bonds</div></a>
              <div data-categoryid="20255" data-articleid="26030"
                   data-categoryname="Disulfide bonds"
                   class="flagImage" title="Flag 'Disulfide bonds' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="26030"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="26030"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="47289" data-articleid="26030"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Regulator+genes%22" title="Search for articles in the subject area:'Regulator genes'"><div class="flagText">Regulator genes</div></a>
              <div data-categoryid="18579" data-articleid="26030"
                   data-categoryname="Regulator genes"
                   class="flagImage" title="Flag 'Regulator genes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5246'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6877'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2348&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
