

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genome Dynamics of Short Oligonucleotides: The Example of Bacterial DNA Uptake Enhancing Sequences</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000741"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000741" />

    <meta name="citation_title" content="Genome Dynamics of Short Oligonucleotides: The Example of Bacterial DNA Uptake Enhancing Sequences"/>
    <meta itemprop="name" content="Genome Dynamics of Short Oligonucleotides: The Example of Bacterial DNA Uptake Enhancing Sequences"/>

      <meta name="citation_author" content="Mohammed Bakkali"/>
            <meta name="citation_author_institution" content="Institute of Genetics, Queen's Medical Center, University of Nottingham, Nottingham, United Kingdom"/>

    <meta name="citation_date" content="2007/8/15"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000741.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e741"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Bacterial gene transfer by natural genetic transformation in the environment.; citation_author=MG Lorenz; citation_author=W Wackernagel; citation_journal_title=Microbiol Rev; citation_volume=58; citation_number=1; citation_pages=563-602; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=DNA uptake in bacteria.; citation_author=D Dubnau; citation_journal_title=Annu Rev Microbiol; citation_volume=53; citation_number=2; citation_pages=217-244; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Construction of DNA recognition sites active in Haemophilus transformation.; citation_author=DB Danner; citation_author=HO Smith; citation_author=SA Narang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=79; citation_number=3; citation_pages=2393-2397; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=DNA uptake in Haemophilus transformation.; citation_author=SH Goodgal; citation_journal_title=Annu Rev Genet; citation_volume=16; citation_number=4; citation_pages=169-192; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Haemophilus influenzae and Neisseria gonorrhoeae recognize different specificity determinants in the DNA uptake step of genetic transformation.; citation_author=LS Mathis; citation_author=JJ Scocca; citation_journal_title=J Gen Microbiol; citation_volume=128; citation_number=5; citation_pages=1159-1161; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Natural transformation and DNA uptake signal sequences in Actinobacillus actinomycetemcomitans.; citation_author=Y Wang; citation_author=SD Goodman; citation_author=RJ Redfield; citation_author=C Chen; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=6; citation_pages=3442-3449; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Species-specific uptake of DNA by gonococci is mediated by a 10-base-pair sequence.; citation_author=C Elkins; citation_author=CE Thomas; citation_author=HS Seifert; citation_author=PF Sparling; citation_journal_title=J Bacteriol; citation_volume=173; citation_number=7; citation_pages=3911-3913; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Sequence-specific DNA uptake in transformation of Neisseria gonorrhoeae.; citation_author=JF Graves; citation_author=GD Biswas; citation_author=PF Sparling; citation_journal_title=J Bacteriol; citation_volume=152; citation_number=8; citation_pages=1071-1077; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Genomic distribution and functions of uptake signal sequences in Actinobacillus actinomycetemcomitans.; citation_author=Y Wang; citation_author=J Orvis; citation_author=D Dyer; citation_author=C Chen; citation_journal_title=Microbiology; citation_volume=152; citation_number=9; citation_pages=3319-3325; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of Neisseria meningitidis serogroup B strain MC58.; citation_author=H Tettelin; citation_author=NJ Saunders; citation_author=J Heidelberg; citation_author=AC Jeffries; citation_author=KE Nelson; citation_journal_title=Science; citation_volume=287; citation_number=10; citation_pages=1809-1815; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Biased distribution of DNA uptake sequences towards genome maintenance genes.; citation_author=T Davidsen; citation_author=EA Rodland; citation_author=K Lagesen; citation_author=E Seeberg; citation_author=T Rognes; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=11; citation_pages=1050-1058; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complete DNA sequence of a serogroup A strain of Neisseria meningitidis Z2491.; citation_author=J Parkhill; citation_author=M Achtman; citation_author=KD James; citation_author=SD Bentley; citation_author=C Churcher; citation_journal_title=Nature; citation_volume=404; citation_number=12; citation_pages=502-506; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=DNA uptake signal sequences in naturally transformable bacteria.; citation_author=HO Smith; citation_author=ML Gwinn; citation_author=SL Salzberg; citation_journal_title=Res Microbiol; citation_volume=150; citation_number=13; citation_pages=603-616; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Frequency and distribution of DNA uptake signal sequences in the Haemophilus influenzae Rd genome.; citation_author=HO Smith; citation_author=JF Tomb; citation_author=BA Dougherty; citation_author=RD Fleischmann; citation_author=JC Venter; citation_journal_title=Science; citation_volume=269; citation_number=14; citation_pages=538-540; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Evolutionary stability of DNA uptake signal sequences in the Pasteurellaceae.; citation_author=M Bakkali; citation_author=TY Chen; citation_author=HC Lee; citation_author=RJ Redfield; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=15; citation_pages=4513-4518; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolution of competence and DNA uptake specificity in the Pasteurellaceae.; citation_author=RJ Redfield; citation_author=WA Findlay; citation_author=J Bosse; citation_author=JS Kroll; citation_author=AD Cameron; citation_journal_title=BMC Evol Biol; citation_volume=6; citation_number=16; citation_pages=82; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Development of a genetic manipulation system for Haemophilus parasuis.; citation_author=A Bigas; citation_author=ME Garrido; citation_author=AM de Rozas; citation_author=I Badiola; citation_author=J Barbe; citation_journal_title=Vet Microbiol; citation_volume=105; citation_number=17; citation_pages=223-228; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of Mannheimia haemolytica A1: insights into virulence, natural competence, and Pasteurellaceae phylogeny.; citation_author=J Gioia; citation_author=X Qin; citation_author=H Jiang; citation_author=K Clinkenbeard; citation_author=R Lo; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=18; citation_pages=7257-7266; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The ins and outs of DNA transfer in bacteria.; citation_author=I Chen; citation_author=PJ Christie; citation_author=D Dubnau; citation_journal_title=Science; citation_volume=310; citation_number=19; citation_pages=1456-1460; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA transport during transformation.; citation_author=I Chen; citation_author=D Dubnau; citation_journal_title=Front Biosci; citation_volume=8; citation_number=20; citation_pages=s544-556; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=DNA uptake during bacterial transformation.; citation_author=I Chen; citation_author=D Dubnau; citation_journal_title=Nat Rev Microbiol; citation_volume=2; citation_number=21; citation_pages=241-249; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification and arrangement of the DNA sequence recognized in specific transformation of Neisseria gonorrhoeae.; citation_author=SD Goodman; citation_author=JJ Scocca; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=85; citation_number=22; citation_pages=6982-6986; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Frequent oligonucleotides and peptides of the Haemophilus influenzae genome.; citation_author=S Karlin; citation_author=J Mrazek; citation_author=AM Campbell; citation_journal_title=Nucleic Acids Res; citation_volume=24; citation_number=23; citation_pages=4263-4272; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Bacteria mating preferences.; citation_author=RJ Redfield; citation_journal_title=Nature; citation_volume=352; citation_number=24; citation_pages=25-26; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Evolution of DNA uptake signal sequences.; citation_author=D Chu; citation_author=HC Lee; citation_author=T Lenaerts; citation_journal_title=Artif Life; citation_volume=11; citation_number=25; citation_pages=317-338; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genes for breakfast: the have-your-cake-and-eat-it-too of bacterial transformation.; citation_author=RJ Redfield; citation_journal_title=J Hered; citation_volume=84; citation_number=26; citation_pages=400-404; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Do bacteria have sex?; citation_author=RJ Redfield; citation_journal_title=Nat Rev Genet; citation_volume=2; citation_number=27; citation_pages=634-639; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Who's competent and when: regulation of natural genetic competence in bacteria.; citation_author=JM Solomon; citation_author=AD Grossman; citation_journal_title=Trends Genet; citation_volume=12; citation_number=28; citation_pages=150-155; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Escherichia coli competence gene homologs are essential for competitive fitness and the use of DNA as a nutrient.; citation_author=V Palchevskiy; citation_author=SE Finkel; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=29; citation_pages=3902-3910; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA as a nutrient: novel role for bacterial competence gene homologs.; citation_author=SE Finkel; citation_author=R Kolter; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=30; citation_pages=6288-6293; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Natural genetic transformation of clinical isolates of Escherichia coli in urine and water.; citation_author=M Woegerbauer; citation_author=B Jenni; citation_author=F Thalhammer; citation_author=W Graninger; citation_author=H Burgmann; citation_journal_title=Appl Environ Microbiol; citation_volume=68; citation_number=31; citation_pages=440-443; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Heterospecific transformation in the genus Haemophilus.; citation_author=WL Albritton; citation_author=JK Setlow; citation_author=M Thomas; citation_author=F Sottnek; citation_author=AG Steigerwalt; citation_journal_title=Mol Gen Genet; citation_volume=193; citation_number=32; citation_pages=358-363; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Uptake of heterologous DNA by Haemophilus influenzae.; citation_author=SH Goodgal; citation_author=M Mitchell; citation_journal_title=J Bacteriol; citation_volume=157; citation_number=33; citation_pages=785-788; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Uptake-sequence-independent DNA transformation exists in Neisseria gonorrhoeae.; citation_author=S Boyle-Vavra; citation_author=HS Seifert; citation_journal_title=Microbiology; citation_volume=142; citation_number=34; citation_pages=2839-2845; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Evaluation of the current models for the evolution of bacterial DNA uptake signal sequences.; citation_author=D Chu; citation_author=J Rowe; citation_author=HC Lee; citation_journal_title=J Theor Biol; citation_volume=238; citation_number=35; citation_pages=157-166; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Factors affecting genetic transformation of Neisseria gonorrhoeae.; citation_author=GD Biswas; citation_author=T Sox; citation_author=E Blackman; citation_author=PF Sparling; citation_journal_title=J Bacteriol; citation_volume=129; citation_number=36; citation_pages=983-992; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Entry of double-stranded deoxyribonucleic acid during transformation of Neisseria gonorrhoeae.; citation_author=GD Biswas; citation_author=PF Sparling; citation_journal_title=J Bacteriol; citation_volume=145; citation_number=37; citation_pages=638-640; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Studies on transformations of Haemophilus influenzae. I. Competence.; citation_author=SH Goodgal; citation_author=RM Herriott; citation_journal_title=J Gen Physiol; citation_volume=44; citation_number=38; citation_pages=1201-1227; citation_date=1961; " />
      <meta name="citation_reference" content="citation_title=Defined nongrowth media for stage II development of competence in Haemophilus influenzae.; citation_author=RM Herriott; citation_author=EM Meyer; citation_author=M Vogt; citation_journal_title=J Bacteriol; citation_volume=101; citation_number=39; citation_pages=517-524; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Sequence and uptake specificity of cloned sonicated fragments of Haemophilus influenzae DNA.; citation_author=SH Goodgal; citation_author=MA Mitchell; citation_journal_title=J Bacteriol; citation_volume=172; citation_number=40; citation_pages=5924-5928; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Specificity in deoxyribonucleic acid uptake by transformable Haemophilus influenzae.; citation_author=JJ Scocca; citation_author=RL Poland; citation_author=KC Zoon; citation_journal_title=J Bacteriol; citation_volume=118; citation_number=41; citation_pages=369-373; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Bacterial species and speciation.; citation_author=FM Cohan; citation_journal_title=Syst Biol; citation_volume=50; citation_number=42; citation_pages=513-524; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=In search of a bacterial species definition.; citation_author=E Moreno; citation_journal_title=Rev Biol Trop; citation_volume=45; citation_number=43; citation_pages=753-771; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The bacterial species definition in the genomic era.; citation_author=KT Konstantinidis; citation_author=A Ramette; citation_author=JM Tiedje; citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_volume=361; citation_number=44; citation_pages=1929-1940; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=What are bacterial species?; citation_author=FM Cohan; citation_journal_title=Annu Rev Microbiol; citation_volume=56; citation_number=45; citation_pages=457-487; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Fuzzy species among recombinogenic bacteria.; citation_author=WP Hanage; citation_author=C Fraser; citation_author=BG Spratt; citation_journal_title=BMC Biol; citation_volume=3; citation_number=46; citation_pages=6; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=ReadOut: structure-based calculation of direct and indirect readout energies and specificities for protein-DNA recognition.; citation_author=S Ahmad; citation_author=H Kono; citation_author=MJ Arauzo-Bravo; citation_author=A Sarai; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=47; citation_pages=W124-127; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phase variation of gonococcal pili by frameshift mutation in pilC, a novel gene for pilus assembly.; citation_author=AB Jonsson; citation_author=G Nyberg; citation_author=S Normark; citation_journal_title=Embo J; citation_volume=10; citation_number=48; citation_pages=477-488; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Role of pili and the phase-variable PilC protein in natural competence for transformation of Neisseria gonorrhoeae.; citation_author=T Rudel; citation_author=D Facius; citation_author=R Barten; citation_author=I Scheuerpflug; citation_author=E Nonnenmacher; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=49; citation_pages=7986-7990; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Neisseria PilC protein identified as type-4 pilus tip-located adhesin.; citation_author=T Rudel; citation_author=I Scheurerpflug; citation_author=TF Meyer; citation_journal_title=Nature; citation_volume=373; citation_number=50; citation_pages=357-359; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The fimbrial gene cluster of Haemophilus influenzae type b.; citation_author=SM van Ham; citation_author=L van Alphen; citation_author=FR Mooi; citation_author=JP van Putten; citation_journal_title=Mol Microbiol; citation_volume=13; citation_number=51; citation_pages=673-684; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Haemophilus influenzae pili are composite structures assembled via the HifB chaperone.; citation_author=JW St Geme; citation_author=JS Pinkner; citation_author=GP Krasan; citation_author=J Heuser; citation_author=E Bullitt; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=52; citation_pages=11913-11918; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Identification of hifD and hifE in the pilus gene cluster of Haemophilus influenzae type b strain Eagan.; citation_author=KW McCrea; citation_author=WJ Watson; citation_author=JR Gilsdorf; citation_author=CF Marrs; citation_journal_title=Infect Immun; citation_volume=62; citation_number=53; citation_pages=4922-4928; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Identification of two minor subunits in the pilus of Haemophilus influenzae.; citation_author=KW McCrea; citation_author=WJ Watson; citation_author=JR Gilsdorf; citation_author=CF Marrs; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=54; citation_pages=4227-4231; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Role of pili in Haemophilus influenzae adherence and colonization.; citation_author=JR Gilsdorf; citation_author=KW McCrea; citation_author=CF Marrs; citation_journal_title=Infect Immun; citation_volume=65; citation_number=55; citation_pages=2997-3002; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Over- and under-representation of short oligonucleotides in DNA sequences.; citation_author=C Burge; citation_author=AM Campbell; citation_author=S Karlin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=89; citation_number=56; citation_pages=1358-1362; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Statistical analyses of counts and distributions of restriction sites in DNA sequences.; citation_author=S Karlin; citation_author=C Burge; citation_author=AM Campbell; citation_journal_title=Nucleic Acids Res; citation_volume=20; citation_number=57; citation_pages=1363-1370; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Comparative DNA analysis across diverse genomes.; citation_author=S Karlin; citation_author=AM Campbell; citation_author=J Mrazek; citation_journal_title=Annu Rev Genet; citation_volume=32; citation_number=58; citation_pages=185-225; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Compositional biases of bacterial genomes and evolutionary implications.; citation_author=S Karlin; citation_author=J Mrazek; citation_author=AM Campbell; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=59; citation_pages=3899-3913; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Discovery of novel transcription factor binding sites by statistical overrepresentation.; citation_author=S Sinha; citation_author=M Tompa; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=60; citation_pages=5549-5560; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=YMF: A program for discovery of novel transcription factor binding sites by statistical overrepresentation.; citation_author=S Sinha; citation_author=M Tompa; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=61; citation_pages=3586-3588; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Fast and systematic genome-wide discovery of conserved regulatory elements using a non-alignment based approach.; citation_author=O Elemento; citation_author=S Tavazoie; citation_journal_title=Genome Biol; citation_volume=6; citation_number=62; citation_pages=R18; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Specificity and robustness in transcription control networks.; citation_author=AM Sengupta; citation_author=M Djordjevic; citation_author=BI Shraiman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=63; citation_pages=2072-2077; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=On the selection and evolution of regulatory DNA motifs.; citation_author=U Gerland; citation_author=T Hwa; citation_journal_title=J Mol Evol; citation_volume=55; citation_number=64; citation_pages=386-400; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Specificity, free energy and information content in protein-DNA interactions.; citation_author=GD Stormo; citation_author=DS Fields; citation_journal_title=Trends Biochem Sci; citation_volume=23; citation_number=65; citation_pages=109-113; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Sequence logos: a new way to display consensus sequences.; citation_author=TD Schneider; citation_author=RM Stephens; citation_journal_title=Nucleic Acids Res; citation_volume=18; citation_number=66; citation_pages=6097-6100; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=WebLogo: a sequence logo generator.; citation_author=GE Crooks; citation_author=G Hon; citation_author=JM Chandonia; citation_author=SE Brenner; citation_journal_title=Genome Res; citation_volume=14; citation_number=67; citation_pages=1188-1190; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transformation of Haemophilus influenzae.; citation_author=G Poje; citation_author=RJ Redfield; citation_journal_title=Methods Mol Med; citation_volume=71; citation_number=68; citation_pages=57-70; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The recombination hotspot Chi is recognized by the translocating RecBCD enzyme as the single strand of DNA containing the sequence 5′-GCTGGTGG-3′.; citation_author=PR Bianco; citation_author=SC Kowalczykowski; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=69; citation_pages=6706-6711; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Singular over-representation of an octameric palindrome, HIP1, in DNA from many cyanobacteria.; citation_author=NJ Robinson; citation_author=PJ Robinson; citation_author=A Gupta; citation_author=AJ Bleasby; citation_author=BA Whitton; citation_journal_title=Nucleic Acids Res; citation_volume=23; citation_number=70; citation_pages=729-735; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The cyanobacterium Synechocystis sp. strain PCC 6803 expresses a DNA methyltransferase specific for the recognition sequence of the restriction endonuclease PvuI.; citation_author=M Scharnagl; citation_author=S Richter; citation_author=M Hagemann; citation_journal_title=J Bacteriol; citation_volume=180; citation_number=71; citation_pages=4116-4122; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A chicken middle-repetitive DNA sequence which shares homology with mammalian ubiquitous repeats.; citation_author=WE Stumph; citation_author=P Kristo; citation_author=MJ Tsai; citation_author=BW O'Malley; citation_journal_title=Nucleic Acids Res; citation_volume=9; citation_number=72; citation_pages=5383-5397; citation_date=1981; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000741" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genome Dynamics of Short Oligonucleotides: The Example of Bacterial DNA Uptake Enhancing Sequences"/>
    <meta name="twitter:description" content="Among the many bacteria naturally competent for transformation by DNA uptake&mdash;a phenomenon with significant clinical and financial implications&mdash; Pasteurellaceae and Neisseriaceae species preferentially take up DNA containing specific short sequences. The genomic overrepresentation of these DNA uptake enhancing sequences (DUES) causes preferential uptake of conspecific DNA, but the function(s) behind this overrepresentation and its evolution are still a matter for discovery. Here I analyze DUES genome dynamics and evolution and test the validity of the results to other selectively constrained oligonucleotides. I use statistical methods and computer simulations to examine DUESs accumulation in Haemophilus influenzae and Neisseria gonorrhoeae genomes. I analyze DUESs sequence and nucleotide frequencies, as well as those of all their mismatched forms, and prove the dependence of DUESs genomic overrepresentation on their preferential uptake by quantifying and correlating both characteristics. I then argue that mutation, uptake bias, and weak selection against DUESs in less constrained parts of the genome combined are sufficient enough to cause DUESs accumulation in susceptible parts of the genome with no need for other DUES function. The distribution of overrepresentation values across sequences with different mismatch loads compared to the DUES suggests a gradual yet not linear molecular drive of DNA sequences depending on their similarity to the DUES. Other genomically overrepresented sequences, both pro- and eukaryotic, show similar distribution of frequencies suggesting that the molecular drive reported above applies to other frequent oligonucleotides. Rare oligonucleotides, however, seem to be gradually drawn to genomic underrepresentation, thus, suggesting a molecular drag. To my knowledge this work provides the first clear evidence of the gradual evolution of selectively constrained oligonucleotides, including repeated, palindromic and protein/transcription factor-binding DNAs."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000741.g005"/>

  <meta property="og:title" content="Genome Dynamics of Short Oligonucleotides: The Example of Bacterial DNA Uptake Enhancing Sequences" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000741" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3394'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9160'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8371&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000741">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000741" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000741&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genome Dynamics of Short Oligonucleotides: The Example of Bacterial DNA Uptake Enhancing Sequences
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mohammed Bakkali
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mbakkali@gmail.com">mbakkali@gmail.com</a></p>

                <p>Affiliation:
                  Institute of Genetics, Queen's Medical Center, University of Nottingham, Nottingham, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 15, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000741</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000741" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000741" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000741" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000741" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000741-g001" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000741-t001" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000741-g002" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000741-g003" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000741-t002" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000741-t003" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000741-t004" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000741-t005" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000741-g004" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000741-g005" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000741-t006" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000741-t007" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t007" title="Table 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000741">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000741" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Among the many bacteria naturally competent for transformation by DNA uptake—a phenomenon with significant clinical and financial implications— Pasteurellaceae and Neisseriaceae species preferentially take up DNA containing specific short sequences. The genomic overrepresentation of these DNA uptake enhancing sequences (DUES) causes preferential uptake of conspecific DNA, but the function(s) behind this overrepresentation and its evolution are still a matter for discovery. Here I analyze DUES genome dynamics and evolution and test the validity of the results to other selectively constrained oligonucleotides. I use statistical methods and computer simulations to examine DUESs accumulation in <em>Haemophilus influenzae</em> and <em>Neisseria gonorrhoeae</em> genomes. I analyze DUESs sequence and nucleotide frequencies, as well as those of all their mismatched forms, and prove the dependence of DUESs genomic overrepresentation on their preferential uptake by quantifying and correlating both characteristics. I then argue that mutation, uptake bias, and weak selection against DUESs in less constrained parts of the genome combined are sufficient enough to cause DUESs accumulation in susceptible parts of the genome with no need for other DUES function. The distribution of overrepresentation values across sequences with different mismatch loads compared to the DUES suggests a gradual yet not linear molecular drive of DNA sequences depending on their similarity to the DUES. Other genomically overrepresented sequences, both pro- and eukaryotic, show similar distribution of frequencies suggesting that the molecular drive reported above applies to other frequent oligonucleotides. Rare oligonucleotides, however, seem to be gradually drawn to genomic underrepresentation, thus, suggesting a molecular drag. To my knowledge this work provides the first clear evidence of the gradual evolution of selectively constrained oligonucleotides, including repeated, palindromic and protein/transcription factor-binding DNAs.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Bakkali M (2007) Genome Dynamics of Short Oligonucleotides: The Example of Bacterial DNA Uptake Enhancing Sequences. PLoS ONE 2(8):
          e741.
            doi:10.1371/journal.pone.0000741</p><p><strong>Academic Editor: </strong>Cecile Fairhead, Pasteur Institute, France</p><p><strong>Received:</strong> June 5, 2007; <strong>Accepted:</strong> June 29, 2007; <strong>Published:</strong> August 15, 2007</p><p><strong>Copyright:</strong> © 2007 Mohammed Bakkali. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The author has no support or funding to report.</p><p><strong>Competing interests:</strong> The author has declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Many bacteria are naturally competent for transformation by spontaneous uptake of DNA from their surrounding environments <a href="#pone.0000741-Lorenz1">[1]</a>, <a href="#pone.0000741-Dubnau1">[2]</a>. Among these, Pasteurellaceae and Neisseriaceae species preferentially take up DNA containing specific short sequences, called uptake signal sequences (USS) and DNA uptake sequences (DUS), respectively <a href="#pone.0000741-Danner1">[3]</a>–<a href="#pone.0000741-Wang2">[9]</a>. Both USSs and DUSs are overrepresented in their respective genomes and, while only one DUS (<span class="gene">5′-GCCGTCTGAA</span>) has been described <a href="#pone.0000741-Tettelin1">[10]</a>–<a href="#pone.0000741-Smith1">[13]</a>, USS was reported in two slightly different versions: <em>(i)</em> <span class="gene">5′-AAGTGCGGT</span> in <em>Haemophilus influenzae</em> <a href="#pone.0000741-Smith1">[13]</a>–<a href="#pone.0000741-Bakkali1">[15]</a>, <em>Actinobacillus actinomycetemcomitans</em> <a href="#pone.0000741-Wang1">[6]</a>, <a href="#pone.0000741-Wang2">[9]</a>, <a href="#pone.0000741-Bakkali1">[15]</a>, <em>Haemophilus somnus</em> and <em>Pasteurella multocida</em> <a href="#pone.0000741-Bakkali1">[15]</a>, <em>Mannheimia succiniciproducens</em> <a href="#pone.0000741-Redfield1">[16]</a> and, probably <em>Haemophilus parasuis</em> <a href="#pone.0000741-Redfield1">[16]</a>, <a href="#pone.0000741-Bigas1">[17]</a> and <em>(ii)</em> <span class="gene">5′-ACAAGCGGTC</span> in <em>Mannheimia haemolytica</em> <a href="#pone.0000741-Redfield1">[16]</a>, <a href="#pone.0000741-Gioia1">[18]</a> and <em>Actinobacillus pleuropneumoniae</em> <a href="#pone.0000741-Redfield1">[16]</a>. Bacterial competence for natural transformation is best characterized in <em>Bacillus subtilis</em> (Bacillaceae), <em>H. influenzae</em> (Pasteurellaceae), <em>Neisseria gonorrhoeae</em> (Neisseriaceae) and <em>Streptococcus pneumoniae</em> (Streptococcaceae) (see <a href="#pone.0000741-Lorenz1">[1]</a>, <a href="#pone.0000741-Dubnau1">[2]</a>, <a href="#pone.0000741-Chen1">[19]</a>–<a href="#pone.0000741-Chen3">[21]</a>). This work therefore focuses on <em>H. influenzae</em> and <em>N. gonorrhoeae</em> USS/DUS—for which I will henceforth use the unifying name DNA Uptake Enhancing Sequence (DUES).</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Less than 10 DUESs are expected in random sequences of similar length and base composition as <em>H. influenzae</em> Rd Kw20 and <em>N. gonorrhoeae</em> FA1090 genomes (see <a href="#pone.0000741.s001">Table S1</a>). However, Smith et al. <a href="#pone.0000741-Smith1">[13]</a>, <a href="#pone.0000741-Smith2">[14]</a> found 1471 DUES in the first genome, and Davidsen et al. <a href="#pone.0000741-Davidsen1">[11]</a> counted 1965 in the latter. The search for a function to account for the evolution of these sequences is still inconclusive. Goodman and Scocca <a href="#pone.0000741-Goodman1">[22]</a> suggested a DUES role in transcription termination, and Karlin et al. <a href="#pone.0000741-Karlin1">[23]</a> viewed the significant even spacing of these sequences around the genome as a sign of their role in DNA replication, repair, or compaction. However, any transcription termination activity is difficult to envisage for at least three reasons: <em>(i)</em> 65% of <em>H. influenzae</em> and 35% of <em>N. gonorrhoeae</em> DUESs are located within open reading frames <a href="#pone.0000741-Davidsen1">[11]</a>, <a href="#pone.0000741-Smith1">[13]</a>, <a href="#pone.0000741-Smith2">[14]</a>, <em>(ii)</em> DUESs are not palindromes and only ~10% occur as inverted repeats (M. Bakkali, unpublished), and <em>(iii)</em> there are more DUESs in non-coding regions than expected for an unbiased distribution <a href="#pone.0000741-Bakkali1">[15]</a>. Furthermore, no intracellular DUES-binding protein was identified, and DUESs show no orientation bias around the chromosome <a href="#pone.0000741-Smith1">[13]</a>–<a href="#pone.0000741-Bakkali1">[15]</a>—features expected from a sequence that interacts with the replication machinery.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>DUESs as bacterial mate recognition systems <a href="#pone.0000741-Redfield2">[24]</a> is the only uptake related function proposed for these sequences. In this case, DUESs could be tags for ‘safe sex’ among bacterial cells seeking recombination trough competence for natural transformation. Given the striking DUES genomic overrepresentation, any preferential uptake of DUES-containing DNA will inevitably result in preferential uptake of conspecific DNA, and DNA from species sharing the same DUES (<em>i.e.</em>, closely related species). Hence, DUES-biased system of DNA uptake could evolve given a higher selective advantage of recombination with DNA from conspecifics compared to DNA from unrelated species, and computer simulations seem to confirm this possibility <a href="#pone.0000741-Chu1">[25]</a>. However, bacteria were suggested to take up DNA mainly as a nutrient rather than for recombination <a href="#pone.0000741-Redfield3">[26]</a>, <a href="#pone.0000741-Redfield4">[27]</a>. Increasingly supported by recent findings <a href="#pone.0000741-Solomon1">[28]</a>–<a href="#pone.0000741-Woegerbauer1">[31]</a>, this hypothesis raises further questions about the evolution of a DUES-biased DNA uptake system that significantly limits the quantity of DNA available to the competent cells.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p><em>H. influenzae</em> has ‘only’ 764 singly-mismatched DUESs <a href="#pone.0000741-Smith1">[13]</a>–<a href="#pone.0000741-Bakkali1">[15]</a>, even though these can arise from the species' 1471 nine base pairs (bp) long DUESs by any of the 27 possible mutations. Testing this ostensible sequence homogeneity may therefore help understand DUES evolution. High sequence homogeneity is expected in repeats arising by copying, such as transposable elements and telomeres, and Smith et al. <a href="#pone.0000741-Smith1">[13]</a> suggested that the unexpectedly low frequency of singly-mismatched DUESs relative to the non-mismatched one could be due to a balance between mutation away from the latter and its restoration by preferential uptake. However, DUESs are not known to be transposable, and we have no reason to discard uptake bias in favor of singly-mismatched ones. Both <em>H. influenzae</em> and <em>Neisseria meningitidis</em> even take up DUES-lacking DNAs, though less efficiently <a href="#pone.0000741-Goodman1">[22]</a>, <a href="#pone.0000741-Albritton1">[32]</a>–<a href="#pone.0000741-BoyleVavra1">[34]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Based on sequence comparison of homologous pasteurellacean genes, DUESs were suggested to evolve by gradual accumulation of point mutations in preexisting sequences rather than by insertion/deletion of entire sequences <a href="#pone.0000741-Bakkali1">[15]</a>. As incoming DNA can replace homologous chromosomal regions by recombination, and provided there is enough bias towards some mismatched DUESs as well, DNA uptake could generate a drive that gradually imposes the DUES and some of its mismatched forms in the susceptible parts of the genome (<em>i.e.</em>, non-coding and unconstrained coding regions). The resulting molecular drive, tested in <a href="#pone.0000741-Chu2">[35]</a>, offers a simpler explanation of DUES accumulation where the perceived sequence homogeneity could be a consequence of stronger uptake bias towards the DUES than its mismatched forms.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Evaluation of these interpretations requires careful analysis of the genomic representation of all the DUES-like sequences as well as the actual bias of DNA uptake and its contribution to DUES evolution. In this work I examine and computer simulate the accumulation of DUESs in bacterial genomes by estimating and analyzing the frequencies of <em>H. influenzae</em> and <em>N. gonorrhoeae</em> DUESs and all their mutated forms. I subsequently monitor the overrepresentation of DUES nucleotides in sequences with different mismatch loads (<em>i.e.</em>, number of mismatched positions compared to the DUES) to detect the genomic footprints of the DNA uptake bias and, thus, infer DUES evolutionary history. I then test whether uptake bias in itself can explain DUES accumulation by correlating the strength of DUES-like sequences, as estimated from the genomic frequencies of their nucleotides, to the uptake bias in their favor. Finally, I analyze the distribution of the genomic frequencies of several pro- and eukaryotic sequences to test whether the DUES mode of evolution could be extrapolated to other selectively constrained oligonucleotides. DUES offers a precious system for the study of such sequences—especially protein/transcription factor-binding—as it is selectively bound by the extracellular receptor for DNA uptake, and the uptake bias could be analogous to any other function/selective force.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Sequence frequencies and overrepresentation</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>This analysis aims at answering two questions: (i) Do DUESs evolve by gradual accumulation of mutations? If so, then (ii) what is the minimum number of matches a sequence needs to share with the DUES for its uptake to be significantly preferential?</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>For a gradual evolution of DUES to take place, a degree of DNA uptake bias towards some of its mutated forms is also required. In such conditions, and assuming that there is no interference from other evolutionary forces than mutation and DNA uptake bias, over evolutionary time the latter should leave a trace in the genome in form of significant overrepresentation of the DUES and its mutated yet preferentially taken up forms. Furthermore, the magnitude of overrepresentation should inversely correlate to the mismatch load of the sequence compared to the DUES. However, no significant overrepresentation is expected for sequences not preferentially taken up and, if any, it does not have to correlate with the mismatch load.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Only DUESs in <em>H. influenzae</em> and <em>N. gonorrhoeae</em> genomes show significant overrepresentation that consistently decreases with the increase in mismatch load until underrepresentation at mismatch load 6 (<a href="#pone-0000741-g001">Figure 1</a>, <a href="#pone-0000741-t001">Table 1</a>, and <a href="#pone.0000741.s001">Table S1</a>). Significant overrepresentation then reappears at the last two sequence categories where it increases with the mismatch load. As expected from non-accumulating DNAs, the control tests show neither overrepresentation of the sequences analyzed, nor any consistent trend in sequence frequencies with respect to the mismatch load. Thus, if sequence overrepresentation is indicative of DNA uptake, these results suggest that DUESs gradually evolve in the genome by accumulation of point mutations.</p>
<div class="figure" id="pone-0000741-g001"><div class="img"><a name="pone-0000741-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000741.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000741.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Ratio of the observed to the expected number of sequences across mismatch loads.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>X-axis: Mismatch load. A: DUESs. B: Control sequences. HDUESH and HDUESS refer to <em>H. influenzae</em> DUES in <em>H. influenzae</em> and <em>S. pyogenes</em> genomes, respectively. NDUESN and NDUESC refer to <em>N. gonorrhoeae</em> DUES in <em>N. gonorrhoeae</em> and <em>C. tepidum</em> genomes. CSH1, 2, and 3 and CSN1, 2, and 3 refer to the three control sequences in <em>H. influenzae</em> and <em>N. gonorrhoeae</em> (see <a href="#s4">Material and methods</a>).</p>
<span>doi:10.1371/journal.pone.0000741.g001</span></div><div class="figure" id="pone-0000741-t001"><div class="img"><a name="pone-0000741-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Sequence representation (equation 2) of DUESs, control sequences, and all their mismatched forms.</span></strong></p><span>doi:10.1371/journal.pone.0000741.t001</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>The distribution of the overrepresentation values is strikingly similar between <em>H. influenzae</em> and <em>N. gonorrhoeae</em>, suggesting similar DUES genome dynamics and evolution. Nonetheless, overall, <em>N. gonorrhoeae</em> values are noticeably higher than those of <em>H. influenzae</em>, possibly due to more efficient/frequent DNA uptake. Two major drops in overrepresentation values can easily be identified in both species: The first between mismatch loads 0 and 1 (92 fold in <em>H. influenzae</em> and 82 in <em>H. gonorrhoeae</em>), and the second between loads 1 and 2 (9 fold in <em>H. influenzae</em> and 7 in <em>N. gonorrhoeae</em>). This reflects a non-linear relation between the mismatch load and DNA uptake, possibly due to high specificity of the receptor binding the DUES. Coinciding with the base length difference between <em>N. gonorrhoeae</em> and <em>H. influenzae</em> DUESs, significant overrepresentation values reach one mismatch further in the first species than in the latter. This highlights that for DNA uptake what matters is the number of matches to the DUES not the mismatches.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>At four matches (<em>i.e.</em>, mismatch load 5 in <em>H. influenzae</em>, and 6 in <em>N. gonorrhoeae</em>), the results are ambiguous with non-significant overrepresentation in <em>H. influenzae</em>, but significant underrepresentation in <em>N. gonorrhoeae</em>. This may be due to: <em>(i)</em> Non-preferential uptake, <em>(ii)</em> insufficient uptake for balancing sequence loss by mutational decay, <em>(iii)</em> insufficient drive from higher mismatch loads to balance the drive towards lower ones, or <em>(iv)</em> functional constraint selecting against some sequences at this mismatch loads. Sequences with 3 and 2 matches to the DUES are significantly underrepresented, thus, probably not preferentially taken up. They could possibly comprise the pool from which the drive imposed by the DNA uptake bias receives new sequences after mutation. Overrepresentation of sequences with 0 and 1 match to the DUES shows no negative correlation with the mismatch load and is clearly due to uptake independent factor(s) (<em>e.g.</em>, functional and mutational constraints or codon bias).</p>


<h4>Computer simulations</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Overrepresentation of mismatched sequences suggests that they are preferentially taken up. Still, it could also be due to mutational decay of sequences with fewer mismatches that are preferentially taken up. DUESs are so overrepresented that singly-mismatched sequences could reach overrepresentation only by DUES mutational decay—as suggested in <a href="#pone.0000741-Smith1">[13]</a>. It is also possible that the two major drops in sequence overrepresentation reported above reflect uptake bias only towards sequences with less than 2 mismatches to the DUES. I therefore simulated the evolution of sequence frequencies in a population of 10<sup>18</sup> genomes with the same distribution of sequence frequencies as those expected for randomized <em>H. influenzae</em> and <em>N. gonorrhoeae</em> genomes (see <a href="#pone.0000741.s001">Table S1</a>) evolving at 10<sup>−2</sup> mutations per base per generation. Mutations alone are not able to drive neither <em>H. influenzae</em> nor <em>N. gonorrhoeae</em> DUES to their respective observed overrepresentation levels (data not shown), and an uptake bias of 104.4 and 477.5 respectively is needed (<a href="#pone-0000741-g002">Figure 2A;D</a>). However, the results do not support the possibility of mutational decay. By themselves, these uptake biases can neither drive the singly-mismatched sequences to their observed overrepresentation levels, nor give similar distributions of sequence frequencies as the observed ones (<a href="#pone-0000741-g002">Figure 2A;D</a>). An additional 1.02 and 5.284 uptake bias in favor of singly-mismatched <em>H. influenzae</em> and <em>N. gonorrhoeae</em> DUESs was needed to attain their observed frequencies, yet it did not give similar overall sequence frequency distribution as the observed one (<a href="#pone-0000741-g002">Figure 2B;E</a>). The same was true for additional uptake biases towards other mismatch loads (data not shown). An approximation to the observed distributions of sequence frequencies with an error margin of less than 0.01% was obtained only with the distributions of uptake biases shown in <a href="#pone-0000741-g002">Figure 2C;F</a>. This distribution agrees with my interpretation of the sequence overrepresentation results, as it supports uptake bias towards mismatched sequences and negative correlation between the mismatch load and the uptake bias. The exception seems to be <em>H. influenzae</em> sequences at mismatch loads 4 to 9, where uptake bias value increases with the mismatch load, which may reflect the involvement of other selective forces than uptake. The values of uptake bias used in the simulations certainly include the other selective forces, such as functional constraints, that led to the observed distributions of sequence frequencies in the real genomes.</p>
<div class="figure" id="pone-0000741-g002"><div class="img"><a name="pone-0000741-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000741.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000741.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Computer simulation of the accumulation of <em>H. influenzae</em> and <em>N. gonorrhoeae</em> DUESs.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>X-axis: Cycle (<em>i.e.</em>, generation). Y-axis: Ratio of the number of sequences after each simulation cycle to the observed (<em>i.e.</em>, real) number. A and D: Genomes of <em>H. influenzae</em> (A) and <em>N. gonorrhoeae</em> (D) evolving with 104.4 and 477.5 uptake bias towards their respective DUES. B and E: The same genomes evolving with an additional 1.02 and 5.284 uptake bias towards the singly-mismatched DUESs. C and F: The same genomes evolving with the uptake bias combinations (156.4; 2.016; 0.352; 0.295; 0.340; 0.395; 0.449; 0.505; 0.563; 0.623) and (499.2; 5.567; 0.857; 0.493; 0.306; 0.175; 0.056; 0; 0; 0.051; 0.184) towards sequences with 0 to 9 and 0 to 10 mismatches to the respective DUES—semicolons separate uptake biases for different mismatch loads.</p>
<span>doi:10.1371/journal.pone.0000741.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To reach the currently observed genomic distributions of sequence frequencies, simulations needed higher uptake bias values and more cycles (<em>i.e.</em>, generations) for <em>N. gonorrhoeae</em> than for <em>H. influenzae</em> (<a href="#pone-0000741-g002">Figure 2C;F</a>). This may be due to: <em>(i)</em> Differences in the ancestral organization of the real genomes which, obviously, was not random as assumed in the simulations, <em>(ii)</em> more frequent uptake of DNA by the almost permanently competent <em>N. gonorrhoeae</em> <a href="#pone.0000741-Biswas1">[36]</a>, <a href="#pone.0000741-Biswas2">[37]</a> than by the occasionally competent <em>H. influenzae</em> <a href="#pone.0000741-Goodgal1">[4]</a>, <a href="#pone.0000741-Goodgal3">[38]</a>, <a href="#pone.0000741-Herriott1">[39]</a>, or <em>(iii)</em> earlier evolution of the DUES-biased DNA uptake in <em>N. gonorrhoeae</em> than in <em>H. influenzae</em>.</p>


<h4>Sequence logos and evolution of DUES nucleotides</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The results reported above clearly indicate gradual evolution of DUESs by accumulation of point mutations. However, this is only certain if we see a gradual increase in the overrepresentation of the actual DUES nucleotides; as nucleotide overrepresentation at any particular position of the DUES may not show any trend with regards to the mismatch level.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>So far I focused solely on the 9 and 10 bp H. influenzae and N. gonorrhoeae DUESs. However, for H. influenzae, this sequence is only the core of a larger (29 bp) DUES consensus containing two additional less conserved regions <a href="#pone.0000741-Smith2">[14]</a>, <a href="#pone.0000741-Redfield1">[16]</a>. Conversely, there is no report of less conserved DUES regions in N. gonorrhoeae <a href="#pone.0000741-Elkins1">[7]</a>. DNA sequence logos were therefore generated for 100 pb sequences, containing the DUES cores or one of their mismatched forms, in order to include the less conserved regions and look for possible additional ones. H. influenzae DUES consensus is 5′-aAAGTGCGGTnrwttttnnnnnnrwtttw, where r = A or G and w = A or T (<a href="#pone-0000741-g003">Figure 3A</a>), which is similar to the consensus reported in <a href="#pone.0000741-Smith2">[14]</a>. N. gonorrhoeae DUES also seems to show less conserved nucleotides (<a href="#pone-0000741-g003">Figure 3B</a>), especially an adenine and a thymine at the 5′ side. Its consensus sequence thus being 5′-mdatGCCGTCTGAAvv, where d = A, G or T, m = A or C and v = A, G or C.</p>
<div class="figure" id="pone-0000741-g003"><div class="img"><a name="pone-0000741-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000741.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000741.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>DNA sequence logos showing the accumulation of <em>H. influenzae</em> and <em>N. gonorrhoeae</em> DUES nucleotides.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The heights of the characters reflect the conservation of the nucleotides (<a href="#pone.0000741-Schneider1">[equation 3 in 66]</a>) as well as the genomic representation of sequences at the mismatch load (equation 2). The number to the side of each logo is the mismatch load analyzed. The height of the logo should be multiplied by the number at the top, if applicable.</p>
<span>doi:10.1371/journal.pone.0000741.g003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The logos support previous interpretations, as they show clear gradual accumulation of DUES nucleotides starting from the third match, with some nucleotides gaining overrepresentation earlier/faster than others, probably due to their importance for uptake. This, together with the results of the simulations, solves the ambiguity regarding sequences with 4 matches and suggests a certain bias in their uptake, though not sufficient to drive them to significant overrepresentation. Coinciding with the biases in genomic base composition, DUES Ws emerge first in H. influenzae whilst, in N. gonorrhoeae, the whole DUES core emerges at the same time with a slightly higher overrepresentation of Ss than Ws. Unlike the core, DUES nucleotides of the less conserved regions emerge only after mismatch load 2 and, in H. influenzae positions 14 and 15, there is a switch in the nature of the most overrepresented nucleotide; probably because of similarity in their overrepresentation. In neither of the species does overrepresentation seem to depend on uptake at less than 3 matches, since the logos do not show resemblance to the respective DUES cores.</p>


<h4>Dependence of the uptake bias on the strength of the sequence</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Dependence of sequence overrepresentation on DNA uptake is the key to interpreting the results described above. I therefore used the 28 DNA fragments tested for uptake efficiency by competent <em>H. influenzae</em> in <a href="#pone.0000741-Goodgal4">[40]</a>—which are the best experimental data set available on DNA uptake—to test the dependence of their uptake on the strength (equation 19) of their 29 bp DUES-like sequences. The small differences in the naming and sizes of some of the DNA fragments between what is reported in <a href="#pone.0000741-Goodgal4">[40]</a> and the sequences available at the database are insignificant and will not affect the intended analyses. 15 of these fragments were classified by Goodgal and Mitchell <a href="#pone.0000741-Goodgal4">[40]</a> as uptake fragments and the remaining 13 as non-uptake. As <a href="#pone-0000741-t002">Table 2</a> shows, the uptake fragments are, overall, larger than the non-uptake ones, and the average strength of their strongest 29 bp sequences is higher, whilst the average mismatch load of these sequences is lower. The mean mismatch load of all the 29 bp sequences of every uptake and non-uptake fragment is similar, whereas their mean strength is lower in the uptake than the non-uptake fragments. In principle, this may be sufficient to infer dependence of the uptake efficiency on the strength of the strongest sequence of a DNA fragment. However, DNAs were named by Goodgal and Mitchell <a href="#pone.0000741-Goodgal4">[40]</a> prior to quantification of their uptake, and their classification into uptake and non-uptake fragments does not reflect the clear division between fragments taken up at more than 60 molecules per cell and those taken up at 30 or less. Three fragments originally classified as non-uptake are better seen as uptake fragments (14nu, 27nu and 58nu), and six <em>vice versa</em> (159u, 201u, 205u, 30u, 62u and 71u). After this reclassification, the overall uptake fragments (<em>i.e.</em>, all their 29 bp sequences), on average, show less mismatches, but less strength, than the non-uptake ones, whereas their strongest 29 bp sequences show less mismatches and more strength.</p>
<div class="figure" id="pone-0000741-t002"><div class="img"><a name="pone-0000741-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Characteristics of the DNA fragments tested for uptake in <a href="#pone.0000741-Goodgal4">[40]</a>.</span></strong></p><span>doi:10.1371/journal.pone.0000741.t002</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Regression analysis to detect dependence of the number of DNA molecules taken up by competent <em>H. influenzae</em> on the strength and mismatch load of their sequences shows very significant values for the strongest 29 bp sequence in each of the 28 DNA fragments (<a href="#pone-0000741-t003">Table 3</a>). This suggests that the efficiency of DNA uptake depends on the strength of the best region of the DNA fragment. However, the results are also significant for the second 29 bp sequences in strength, implying that the preferential uptake might not always target the strongest sequence in a DNA fragment. Nevertheless, weaker sequences show no significant results, and the significance of the regression is much higher for the strongest sequence. The significant results of the second sequences in strength could, thus, be due to similarities with the strongest ones in some DNA fragments. In fact, 11 DNA fragments show less than 5% difference in strength between their strongest and the following sequence (fragments 8u2, 30u, 37nu2, 39nu, 45nu, 48u, 51nu, 71u, 159u, 201u, 205u). At 10% cut-off, the number of these fragments already reaches half the sample size (due to addition of fragments 43u, 48nu, 62nu).</p>
<div class="figure" id="pone-0000741-t003"><div class="img"><a name="pone-0000741-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Dependence of the uptake efficiency of the DNA fragments in <a href="#pone.0000741-Goodgal4">[40]</a> on the strength and mismatch load of their 29 bp sequences.</span></strong></p><span>doi:10.1371/journal.pone.0000741.t003</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Even though uptake DNA fragments on average are larger than the non-uptake ones (<a href="#pone-0000741-t002">Table 2</a>), DNA uptake efficiency does not depend on the length of the DNA fragment taken up (<a href="#pone-0000741-t004">Table 4</a>). However, had I tested more DNA fragments, such effect could have been detected as, in theory, the larger a DNA fragment is, the more likely it is to have a strong DUES. Furthermore, neither the mean mismatch load of all the 29 bp sequences of a DNA fragment, their mean strength, the position of the strongest one in the fragment, nor its orientation have any significant effect on DNA uptake (<a href="#pone-0000741-t004">Table 4</a>). Randomizing the 29 positions of the strongest sequence in each DNA fragment abolishes the significant dependence of the original uptake values on the strength of these sequences (<em>e.g.</em>, <em>Beta</em> = 0.271, <em>t<sub>26</sub></em> = 1.433 and <em>p-level</em> = 0.164).</p>
<div class="figure" id="pone-0000741-t004"><div class="img"><a name="pone-0000741-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Lack of dependence of uptake efficiency on the length of the DNA fragment, its average mismatch load, strength, and the position and orientation of its strongest DUES.</span></strong></p><span>doi:10.1371/journal.pone.0000741.t004</span></div><a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>Ultimately, the results suggest that the efficiency of DNA uptake significantly depends on the strength of one sequence only—the strongest—which is the most likely to be bound and taken up independently of its position or orientation in the DNA fragment. Since the strength of sequences was estimated based on the genomic overrepresentation (conservation) of their nucleotides (see equation 19), these results suggest that DNA uptake bias alone can explain the genomic overrepresentation of DUESs and some of their mismatched forms.</p>


<h4>Experimental verification of the theoretical results</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>For experimental confirmation of the abovementioned results, uptake was measured for 222 bp DNA fragments generated <em>in vitro</em> to contain the ‘best’ DUES consensus, consensuses mutated to the least frequent nucleotide at positions 2–11, 13 and 14, 20, or 25 and 26, or the worst 29 bp sequence. Uptake experiments were replicated several times and for different amounts of DNA (10, 20, 30 and 40 ηg). The data show considerable variance between experiments, and a nested ANCOVA was performed on the uptake values and their standard deviations to detect possible dependence of uptake on the availability of DNA and the strength of the DUES, as well as to estimate the magnitude of the noise (<em>i.e.</em>, error) emanating from the variation between experiments. The results (<a href="#pone-0000741-t005">Table 5</a>) suggest that the amount of DNA taken up strongly depends on the nature of the DUES (<em>i.e.</em>, stronger DUESs are taken up more efficiently), and on the amount of DNA given to the cells (<em>i.e.</em>, at non-saturating concentrations, as in these experiments, more DNA in the medium increases the possibility for cells to ‘find’ it, bind it, and take it up). Nonetheless, the percentage of DNA taken up depends only on the nature of the DUES. The variability between experiments, however, could be due to differences in the uptake/transformation efficiency (<em>i.e.</em>, competence state) of the cells used in each experiment and/or the use of different DNA concentrations and/or DUESs in different experiments. This may be the reason why the variation in the amount of DNA taken up is only explained by variation in the strength of the DUES and the amount of DNA given to the cells, whereas the variation in the percentage of DNA taken up solely depends on the nature of the DUES tested (<em>i.e.</em>, position mutated).</p>
<div class="figure" id="pone-0000741-t005"><div class="img"><a name="pone-0000741-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Effect of the DNA concentration in the medium, the strength of the DUES, and the experiment on DNA uptake efficiency by competent <em>H. influenzae</em>.</span></strong></p><span>doi:10.1371/journal.pone.0000741.t005</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Overall, the experimental findings agree with those of the theoretical analysis reported above, and mutations of theoretically important DUES positions also result in less DNA uptake (<a href="#pone-0000741-g004">Figure 4</a>). The little effect on uptake after mutating the second, third and fourth positions of the DUES consensus, however, do not agree with their conservation results. Still, this result could reflect the evolutionary variability of nucleotides at these positions as they show differences between the two pasteurellacean DUESs (<a href="#pone-0000741-g004">Figure 4</a>). The experiment confirms that, even within the DUES core, the importance of the different positions for DNA uptake varies, with the evolutionarily conserved ones being the most important. Indeed, Karlin et al. <a href="#pone.0000741-Karlin1">[23]</a> reported a biased distribution of singly mismatched DUESs in <em>H. influenzae</em> genome.</p>
<div class="figure" id="pone-0000741-g004"><div class="img"><a name="pone-0000741-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000741.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000741.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Relative uptake efficiency of <em>H. influenzae</em> DUES and some of its mismatched forms.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>X-axis: Sequence tested. Best DUES: <em>H. influenzae</em> best DUES consensus. Worst Sequence: The 29 bp sequence of the least frequent nucleotides. PxN: Best DUES mutated at position x (Px) towards the least frequent nucleotide (N). Y-axis: Percentage of DNA taken up. SD: Standard deviation of the mean, SE: Standard error. Note that the critical part of the DUES largely coincides with the evolutionarily conserved positions of the two pasteurellacean DUESs currently known.</p>
<span>doi:10.1371/journal.pone.0000741.g004</span></div>

<h4>The findings on DUES could be generalized to other oligonucleotides</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>In this work I took advantage of the significant amount of experimental data on DUES mediated DNA uptake and the convenience of working with bacterial genomes, and used <em>H. influenzae</em> and <em>N. gonorrhoeae</em> DUESs as examples of selectively driven—protein-binding—oligonucleotides. As <a href="#pone-0000741-g005">Figure 5A;B</a> and <a href="#pone-0000741-t006">Tables 6</a> and <a href="#pone-0000741-t007">7</a> show, the gradual mode of evolution suggested by the results on these DUESs seems to be a common feature of other selectively constrained oligonucleotides, including the other pasteurellacean DUES as well as several other pro- and eukaryotic short repeated, palindromic and protein-binding DNAs. Sequences showing genomic overrepresentation seem to gradually evolve in the same way as the DUES (<em>i.e.</em>, by molecular drive), whilst those underrepresented seem to evolve in an inversed fashion (<em>i.e.</em>, by molecular drag). The differences in over- or underrepresentation levels between sequences and mismatch loads reflect the differences in strength and specificity of the selective force(s) directing the evolution of these sequences.</p>
<div class="figure" id="pone-0000741-g005"><div class="img"><a name="pone-0000741-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000741.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000741.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Ratio of the observed to the expected number of several pro- and eukaryotic oligonucleotides in their respective genome/chromosome.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>X-axis: Mismatch load. A: Sequences genomically overrepresented. B: Sequences genomically underrepresented. The sequences are the same as in <a href="#pone-0000741-t006">Tables 6</a> and <a href="#pone-0000741-t007">7</a>.</p>
<span>doi:10.1371/journal.pone.0000741.g005</span></div><div class="figure" id="pone-0000741-t006"><div class="img"><a name="pone-0000741-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Genomic frequencies of various prokaryotic oligonucleotides and their mismatched forms showing signs of gradual drive or drag towards genomic over- or underrepresentation.</span></strong></p><span>doi:10.1371/journal.pone.0000741.t006</span></div><div class="figure" id="pone-0000741-t007"><div class="img"><a name="pone-0000741-t007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000741" data-uri="info:doi/10.1371/journal.pone.0000741.t007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.t007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000741.t007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000741.t007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000741.t007.TIF"></span>)
                </a></li></ul></div><p><strong>Table 7.  <span>Genomic frequencies of various eukaryotic oligonucleotides and their mismatched forms showing signs of gradual drive or drag towards genomic over- or underrepresentation.</span></strong></p><span>doi:10.1371/journal.pone.0000741.t007</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The current names of pasteurellacean uptake signal sequences (USS) and neisseriacean DNA uptake sequences (DUS) could be misleading, as they do not accurately describe the effect of these sequences on DNA uptake and imply that they represent different genetic elements. Both sequences show similar genomic distributions in the corresponding genomes <a href="#pone.0000741-Wang2">[9]</a>, <a href="#pone.0000741-Tettelin1">[10]</a>, <a href="#pone.0000741-Parkhill1">[12]</a>–<a href="#pone.0000741-Smith2">[14]</a>, and have the same effect on DNA uptake by the corresponding competent bacteria <a href="#pone.0000741-Mathis1">[5]</a>, <a href="#pone.0000741-Graves1">[8]</a>, <a href="#pone.0000741-Wang2">[9]</a>, <a href="#pone.0000741-Goodman1">[22]</a>, <a href="#pone.0000741-Albritton1">[32]</a>, <a href="#pone.0000741-Scocca1">[41]</a>. Therefore, USS and DUS refer to sequence variants of the same genetic element; which makes the case for a common nomenclature. This should not be USS, given that neither USSs nor DUSs actually signal DNA uptake, as bacteria need to be competent beforehand (<em>i.e.</em>, non-competent cells will not take up DNA, no matter how many USSs or DUSs one gives them). DUS is not a satisfactory name either, as both competent Pasteurellaceae and Neisseriaceae species also take up DNA that lacks USS/DUS, though less efficiently <a href="#pone.0000741-Goodman1">[22]</a>, <a href="#pone.0000741-Albritton1">[32]</a>–<a href="#pone.0000741-BoyleVavra1">[34]</a>. Here I suggest the unifying and more accurately descriptive name of DNA Uptake Enhancing Sequences (DUES) to highlight that both sequences are variants of the same genetic element which enhance the uptake of DNA rather than cause it.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>DUESs genomic overrepresentation raises questions about their function and how the normally small bacterial genomes tolerate them in such large quantities. Among the several intracellular functions suggested <a href="#pone.0000741-Goodman1">[22]</a>, <a href="#pone.0000741-Karlin1">[23]</a> none seems to be supported by the genomic distributions of these sequences (see <a href="#s1">Introduction</a>) as: <em>(i)</em> DUESs are not palindromes, and most are neither inverted repeats nor biased towards locations downstream coding sequences—as it would normally have been expected from sequences terminating transcription, <em>(ii)</em> they show no orientation bias around the chromosome—feature expected from sequences interacting with the replication machinery—and we do not know of any intracellular DUES-binding protein, and <em>(iii)</em> they are not as evenly spaced as expected from sequences that help compact the chromosome. Bacterial mate recognition system, discussed in <a href="#pone.0000741-Redfield2">[24]</a>, is the only non-intracellular function proposed for these sequences, and the only one to account for their involvement in DNA uptake. The differences between the DUESs of the two groups of Pasteurellaceae species <a href="#pone.0000741-Redfield1">[16]</a>, and between these and the Neisseriaceae DUES (<a href="#pone.0000741-Mathis1">[5]</a>, <a href="#pone.0000741-Smith1">[13]</a>; <a href="#pone-0000741-g003">Figure 3A;B</a>) clearly supports this last hypothesis. Nevertheless, competence is suggested to be a mechanism more for nutritional than for sex and recombination purposes <a href="#pone.0000741-Redfield3">[26]</a>–<a href="#pone.0000741-Woegerbauer1">[31]</a> and, even if this was not the case, some significantly distant bacterial species share the same DUES. These include: <em>(i) H. influenzae</em>, <em>P. multocida</em>, <em>H. somnus</em>, <em>A. actinomycetemcomitans</em> and <em>M. succiniciproducens</em>, <em>(ii) A. pleuropneumoniae</em> and <em>M. haemolytica</em> <a href="#pone.0000741-Wang1">[6]</a>, <a href="#pone.0000741-Wang2">[9]</a>, <a href="#pone.0000741-Smith1">[13]</a>–<a href="#pone.0000741-Redfield1">[16]</a>, <a href="#pone.0000741-Gioia1">[18]</a> and <em>(iii) N. gonorrhoeae</em> and <em>N. meningitidis</em> <a href="#pone.0000741-Elkins1">[7]</a>, <a href="#pone.0000741-Tettelin1">[10]</a>–<a href="#pone.0000741-Smith1">[13]</a>. On the other hand, the concept of bacterial species is still a matter for debate <a href="#pone.0000741-Cohan1">[42]</a>–<a href="#pone.0000741-Cohan2">[45]</a> and, in some cases, including <em>Neisseria</em> <a href="#pone.0000741-Hanage1">[46]</a>, differentiation between species seems to be rather fuzzy. Furthermore, even if competence was for nutritional purposes, some of the DNA taken up still survives digestion and recombines with the chromosome. Occasional recombination with potentially harmful DNA from distantly related species may, thus, be a sufficient driving force for DUESs to evolve as a mechanism for minimizing such unwanted recombination. DUESs might also not be that advantageous, thus representing a sort of ‘lucky DNA’ probably driven to overrepresentation in genomic locations where they cause insignificant loss of fitness due to a possibly coincidental specificity of the protein that binds DNA (<em>i.e.</em>, receptor) at the cell surface during competence. Every DNA-binding protein has some sort of inherent specificity towards a combination of nucleotides due to its amino acids sequence and spatial configuration <a href="#pone.0000741-Ahmad1">[47]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The possible implication of DNA uptake bias towards DUESs on the accumulation of these sequences in the genome was oddly overlooked. Earlier we <a href="#pone.0000741-Bakkali1">[15]</a>, hypothesized that the biased DNA receptor at the cell surface of a competent species may gradually enrich its genome with DNA fragments containing the preferred DNA sequence (<em>i.e.</em>, DUES) and some of its mutated forms. Mutations towards the preferred DUES in the DNA of the ‘donor’ cell may preferentially spread to other cells, by uptake and recombination, when the ‘donor’ cell dies. On the other hand, mutations away from the DUES in the recipient chromosomes may be restored by uptake and recombination with incoming DNA from ‘donors’ with less-mutated sequences. This way, the combined action of mutation, uptake bias and reduced constraints in some genomic regions could be sufficient for DUESs to gradually accumulate in susceptible parts the genome and no additional function is needed. This would explain DUESs preferential location in non-coding regions <a href="#pone.0000741-Bakkali1">[15]</a>, and in parts of the coding regions where they have little influence on protein configuration/function <a href="#pone.0000741-Karlin1">[23]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>If such interpretation is correct, DNA uptake should leave a footprint on the genome, in form of higher overrepresentation of sequences depending on their similarity to the DUES, which is exactly what this work shows. Sequence overrepresentation is observed even for some mutated DUESs, where it negatively correlates to the mismatch load, and DUES nucleotides seem to emerge and start accumulating as soon as the third match. Simulations of DUES evolution in hypothetical genomes subjected to mutation and uptake bias towards the DUES only fail to reproduce the observed distribution of sequence overrepresentation across mismatch loads, whilst consideration of additional uptake bias towards mismatched sequences did. This, together with the clear correlation between genomic overrepresentation and uptake efficiency, suggests that mutations and DNA uptake bias result in a force sufficient enough to allow DUES gradual evolution by accumulation of point mutations in susceptible parts of the genome (<em>i.e.</em>, by molecular drive). This is supported by the experimentally detectable uptake of DNAs from unrelated species (<em>i.e.</em>, with mismatched or no DUESs) by competent Pasteurellaceae and Neisseriaceae species <a href="#pone.0000741-Goodman1">[22]</a>, <a href="#pone.0000741-Albritton1">[32]</a>–<a href="#pone.0000741-BoyleVavra1">[34]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The higher overrepresentation of <em>N. gonorrhoeae</em> DUES compared to that of <em>H. influenzae</em> could be due to the almost permanent character of competence in the first species <a href="#pone.0000741-Biswas1">[36]</a>, <a href="#pone.0000741-Biswas2">[37]</a> as opposed to its occasional nature in the latter <a href="#pone.0000741-Goodgal1">[4]</a>, <a href="#pone.0000741-Goodgal3">[38]</a>, <a href="#pone.0000741-Herriott1">[39]</a>. Even so, the increase in sequence overrepresentation accompanying the decrease in mismatch load is not linear in any of the two species, as we see sharp drops between the DUES and its singly-mismatched forms and, to a lower degree, between the latter and sequences with two mismatches. This distribution could reflect a strong specificity of the DNA-binding receptor, as well as the length of the evolutionary history of the DNA uptake bias and/or its efficiency. In principle, the more specific the receptor is, the more pronounced the overrepresentation of the DUES compared to its mismatched forms will be. Similarly, the longer the uptake bias was running for, or the more efficient it is, the closer genomes will be to an equilibrium, where the relative overrepresentations of the DUES and its mismatched forms better reflect the differences in the relative efficiency of their uptake.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In spite of all the similarities in genomic frequencies and evolutionary dynamics between <em>H. influenzae</em> and <em>N. gonorrhoeae</em> DUESs, their actual sequence consensuses are different both in shape and composition. In this work I identify less conserved nucleotides previously unreported at each side of the <em>N. gonorrhoeae</em> DUES core, bringing the consensus up to 16 bp, with four additional nucleotides at the 5′ side of the core (two of which strongly conserved) and two at the 3′ side. The resulting consensus is identical to that of <em>N. meningitidis</em> DUES reported in <a href="#pone.0000741-Smith1">[13]</a>, suggesting common ancestry of the DUESs of these two sister species. Therefore, just like the Pasteurellaceae <a href="#pone.0000741-Bakkali1">[15]</a>, <em>Neisseria</em> species seem to have ancestral DUES-mediated preferential uptake of conspecific DNA. The obvious reason for the differences between <em>H. influenzae</em> and <em>N. gonorrhoeae</em> DUESs is differences in structure and binding specificity of the DNA receptors at the cell surfaces of both species. PilC is the extracellular 110 kDa adhesin <a href="#pone.0000741-Jonsson1">[48]</a> suggested as binding DNA at the tips of <em>N. gonorrhoeae</em>'s type IV pili <a href="#pone.0000741-Rudel1">[49]</a>, <a href="#pone.0000741-Rudel2">[50]</a>. Its equivalent at the tips of <em>H. influenzae</em> type IV pili are a 216 amino acids protein (called HifD) and two units of a 435 amino acids protein (called HifE) <a href="#pone.0000741-vanHam1">[51]</a>–<a href="#pone.0000741-Gilsdorf1">[55]</a>. Given this difference, it is hard to refrain from speculating that whilst PilC could be the receptor driving the entire <em>N. gonorrhoeae</em> DUES, HifD might be binding to and driving the core sequence of <em>H. influenzae</em> DUES, whereas the two HifE proteins could have specificity towards the two almost identical less conserved regions of this DUES. However, this can only be confirmed after proper experimental testing (<em>e.g.</em>, gene knockouts, DNA-protein cross-linking, southern-blotting, band-shifts and DNaseI foot-printing, antibodies…).</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The results of this work also show that, at least for <em>H. influenzae</em>, only one DUES seems to be bound by the receptor at the cell surface at any given time, independently of its orientation or position in the DNA fragment. A similar result was experimentally obtained in <em>A. actinomycetemcomitans</em> <a href="#pone.0000741-Wang2">[9]</a>, suggesting that this is a general characteristic of DUES mediated DNA uptake. In addition, the most similar sequence to the DUES in a DNA fragment seems to be the one most likely to be bound. Both characteristics are of importance to the molecular drive model of DUES evolution postulated in <a href="#pone.0000741-Bakkali1">[15]</a>, since the simultaneous binding of more than one DUES would not explain the genomic distribution of these sequences—it favors significant clustering in some chromosomal regions. In addition, bias towards DUESs in a particular orientation would have caused bias in their orientation around the chromosome. Indiscriminate binding of the receptor to sequences independently of their degree of similarity to the DUES, however, would loosen or even abolish the drive responsible for the emergence and gradual accumulation of DUESs in the genome.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Several genomically overrepresented oligonucleotides, both pro- and eukaryotic, seem to be selectively driven in a similar mode as the DUES, whilst those underrepresented seem to be gradually eliminated (dragged) in an inverse fashion. This suggests that the gradual mode of sequence evolution discussed above might be a general feature of short DNAs selectively driven or dragged to genomic over- or underrepresentation, including protein/transcription factor-binding, palindromic, repeated and other oligonucleotides. Statistical deviation from the expected frequencies <a href="#pone.0000741-Burge1">[56]</a>–<a href="#pone.0000741-Karlin4">[59]</a> is used as indicator of DNA functionality, including transcription-factor binding <a href="#pone.0000741-Sinha1">[60]</a>–<a href="#pone.0000741-Elemento1">[62]</a>. DUESs themselves resemble sequence families that interact with sequence-specific DNA-binding proteins (<em>e.g.</em>, CRP, LexA and other pro- and eukaryotic transcription factor-binding DNAs); which typically consist of a short conserved core extended by less conserved regions clustering around a consensus sequence without exactly reproducing it (see <a href="#pone.0000741-Sengupta1">[63]</a>–<a href="#pone.0000741-Stormo1">[65]</a>).</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>In conclusion, the results of this work suggest a gradual accumulation of genomic oligonucleotides by molecular drive towards overrepresentation due to the combined effects of mutation, the function of the oligonucleotide and some of its mutated forms, and weak selection against changes in the functionally unconstrained regions of the genome. Underrepresented oligonucleotides, however, seem to be gradually eliminated by a molecular drag emanating from the combined effects of mutation, the negative effect of the oligonucleotide and some of its mutated forms on the fitness of the carrier, and selection against changes in the functionally constrained regions of the genome. Several oligonucleotides seem to evolve in this way, including transcription factor/protein-binding and functionally constrained palindromic and repeated DNAs. DUESs themselves might be a sort of ‘lucky DNA’ driven to genomic overrepresentation, in the susceptible parts of the genome, not due to any other function but the possibly coincidental specificity of the receptor that binds DNA for uptake during natural competence for transformation.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Theoretical analysis</h4>
<h5>Sequence frequencies and overrepresentation.</h5><a id="article1.body1.sec4.sec1.sec1.p1" name="article1.body1.sec4.sec1.sec1.p1"></a><p>The programs Sequence_Extractor.pl (a modified version of the Perl based program search_USSmissmatches_sequence.pl, see acknowledgements) and BioEdit version 7.0.1 were used to search both strands of the genomes of H. influenzae str. Rd KW20 (Pasteurellaceae, 1.83 Mb genome size, 38.1% G+C, Accession no. NC_000907) and N. gonorrhoeae str. FA1090 (Neisseriaceae, 2.154 Mb genome size, 51.5% G+C, Accession no. NC_002946) and count the number of their respective DUES and all its mutated forms. The same was done for three control sequences (CS) with the same base composition but no match to the DUES. These were CSH1 = 5′-GGTCAGTAG, CSH2 = 5′-TGCGAGATG and CSH3 = 5′-CGAGTGTAG for H. influenzae, and CSN1 = 5′-ATTCAGCCGG, CSN2 = 5′-TAATGGCCCG and CSN3 = 5′-AGGAGTCTCC for N. gonorrhoeae. In addition, similar analysis was carried out for H. influenzae and N. gonorrhoeae DUESs in the genomes of Streptococcus pyogenes str. M1 GAS (Streptococcaceae, 1.85 Mb genome size, 38.5% G+C, Accession no. NC_002737) and Chlorobium tepidum str. TLS (Chlorobiaceae, 2.155 Mb genome size, 56.5% G+C, Accession no. NC_002932), which have no DUES-biased DNA uptake and their genomes are of similar sizes and G+C contents as those of H. influenzae and N. gonorrhoeae.</p>
<a id="article1.body1.sec4.sec1.sec1.p2" name="article1.body1.sec4.sec1.sec1.p2"></a><p>The expected number of sequences at each mismatch load was calculated as: <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.e001&amp;representation=PNG" class="inline-graphic"></span><br>(equation 1), where <em>m</em> is the mismatch load analyzed, τ is the number of adenines and thymines (Ws) in the DUES, σ is the number of cytosines and guanines (Ss) in the same DUES, <em>x</em> is the maximum number of mismatches that could affect DUES W positions at the mismatch load <em>m</em>, θ is the genomic frequency of Ws, ς is the genomic frequency of Ss and <em>L</em> is the genome's length in base pairs. Sequence representation was calculated at each mismatch load as <em>Rep<sub>Seq</sub></em> = (<em>Seq<sub>obs</sub></em>−<em>Seq<sub>exp</sub></em>)/<em>Seq<sub>exp</sub></em> (equation 2), where <em>Seq<sub>obs</sub></em> is the observed number of sequences. Significant deviation of the observed numbers of sequences from the expected ones was tested using the goodness of fit Chi-squared (χ<em><sup>2</sup></em>), which values were multiplied by |<em>Seq<sub>obs</sub></em>−<em>Seq<sub>exp</sub></em>|/(<em>Seq<sub>obs</sub></em>−<em>Seq<sub>exp</sub></em>) (equation 3) to differentiate overrepresentation from underrepresentation.</p>

<h5>Computer simulations.</h5><a id="article1.body1.sec4.sec1.sec2.p1" name="article1.body1.sec4.sec1.sec2.p1"></a><p>To test the contribution of DNA uptake bias to the observed distribution of sequence frequencies, I wrote the Perl based program Genome_Dynamics.pl (program available on request). It simulates the evolution of the genomic frequencies of a given sequence and all its mismatched forms starting from a given distribution until the equilibrium or any other target. Each generation, a mismatch load looses the sequences mutated to the immediately higher and lower mismatch loads while gaining some of the sequences mutating from the same. Assuming there is no other evolutionary force than single (i.e., point) mutation, sequences at a mismatch load m should decay (i.e., mutate to mismatch load m+1) at the proportion μ(<em>Seq</em>)(1−(<em>m</em>/<em>n</em>)) (equation 4) per mutation per generation, and improve (i.e., mutate to mismatch load m-1) at μ(<em>Seq</em>)(<em>m</em>/(<em>3n</em>)) (equation 5), where μ is the mutation rate per base per generation, Seq is the number of sequences at the mismatch load and generation analysed, and n is the size of the sequence in bp. Uptake biases U<sub>0</sub> to U<sub>n</sub> in favour of sequences at the mismatch loads 0 to n should result in additional generational increase of the frequency of sequences at any mismatch load m by the proportion U<sub>m</sub> of sequences mutated from the mismatch loads m−1 and m+1. In addition, there will be a generational decrease by the proportion U<sub>m−1</sub> of sequences improving from the mismatch load m to m−1, and the proportion U<sub>m+1</sub> of sequences decaying from the load m to m+1—note that U<sub>m</sub> could also include other selective forces than DNA uptake. Repetition of these calculations at each mismatch load for many generations allows simulating the evolution of a given initial distribution of sequence frequencies in a population under a given mutation rate and distribution of uptake bias values towards sequences at different mismatch loads. Genome_Dynamics.pl assumes panmixia and I used a population of 10<sup>18</sup> individuals (i.e., cells) evolving at 10<sup>−2</sup> mutation per base per generation.</p>

<h5>DNA Sequence logos and DUES nucleotides evolution.</h5><a id="article1.body1.sec4.sec1.sec3.p1" name="article1.body1.sec4.sec1.sec3.p1"></a><p>DNA sequence logos were generated in order to detect possible nucleotide conservation outside the 9 or 10 bp DUES, and to graphically monitor the emergence and evolution of DUES nucleotides in the respective genomes. I used the program Sequences_Extractor.pl to search both strands of H. influenzae and N. gonorrhoeae genomes and extract all the 100 bp sequences containing the respective DUES or any of its mismatched forms at position 38 from the 5′ end of the sequence. I then used WebLogo version 2.8.2 <a href="#pone.0000741-Schneider1">[66]</a>, <a href="#pone.0000741-Crooks1">[67]</a> to generate logos for the sequences extracted at each mismatch load. Deviation of the genomic G+C content from the 50% assumed by WebLogo was considered by amending its source code and adding a correction factor F to the value of each base frequency used for generating the logo. F was calculated as |<em>V</em>|(<em>0.5</em>−θ)/θ (equation 6) for Ws, and |<em>V</em>|(0.5−ζ)/ζ (equation 7) for Ss, where V is the value being corrected. This adjustment permits more accurate assessment of the selective pressure on DUES Ss and Ws by minimising underestimation of the importance of the less frequent ones.</p>

<h5>Nucleotide genomic overrepresentation and sequence quantification.</h5><a id="article1.body1.sec4.sec1.sec4.p1" name="article1.body1.sec4.sec1.sec4.p1"></a><p>The genomic representation of each nucleotide at each position of the sequences at each mismatch load was calculated as <em>Rep<sub>Nuc</sub></em> = (<em>Nuc<sub>obs</sub></em>−<em>Nuc<sub>exp</sub></em>)/<em>Nuc<sub>exp</sub></em> (equation 8), where Nuc<sub>obs</sub> and Nuc<sub>exp</sub> are the observed and the expected numbers of the nucleotide at the position and sequences analysed.</p>
<a id="article1.body1.sec4.sec1.sec4.p2" name="article1.body1.sec4.sec1.sec4.p2"></a><p>Contrarily to the unbiased distribution of nucleotides in sequence parts not targeted by the extraction program (positions outside the DUES core), the distribution of nucleotides within DUES core positions was biased due to targeting of specific combinations of nucleotides by the program Sequences_Extractor.pl. Thus, for nucleotides in the first type of positions, Nuc<sub>exp</sub> was calculated as θ<em>Seq<sub>obs</sub></em>/<em>2</em> (equation 9) for Ws, and ζ<em>Seq<sub>obs</sub></em>/<em>2</em> (equation 10) for Ss. Whereas for matches at DUES core positions, Nuc<sub>exp</sub> was calculated using equation 11 for Ws, and 12 for Ss. For mismatches at DUES core positions, however, calculation of Nuc<sub>exp</sub> depends on the nature of both the mismatched nucleotide in question and the DUES match at the same position. Thus, for a W as mismatch at a DUES core position, Nuc<sub>exp</sub> was calculated using equation 13, if the match at the same position is also a W, and 14, if it was an S. For an S as mismatch at a DUES core position, Nuc<sub>exp</sub> was calculated using equation 15, if the match at the same position is a W, and 16, if it was an S as well.</p>
<a id="article1.body1.sec4.sec1.sec4.p3" name="article1.body1.sec4.sec1.sec4.p3"></a><p>Equation 11:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.e002&amp;representation=PNG" class="inline-graphic"></span><br>Equation 12:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.e003&amp;representation=PNG" class="inline-graphic"></span><br>Equation 13:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.e004&amp;representation=PNG" class="inline-graphic"></span><br>Equation 14:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.e005&amp;representation=PNG" class="inline-graphic"></span><br>Equation 15:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.e006&amp;representation=PNG" class="inline-graphic"></span><br>Equation 16:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.e007&amp;representation=PNG" class="inline-graphic"></span><br>A modified version of equation 3 in <a href="#pone.0000741-Schneider1">[66]</a> was used to calculate a nucleotide conservation index (Con<sub>Nuc</sub>) for each nucleotide at each position of sequences at each mismatch load as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.e008&amp;representation=PNG" class="inline-graphic"></span> (equation 17), where Max_Rep<sub>Nuc</sub> is the highest representation possible for a nucleotide at the mismatch load analyzed. <em>Max_Rep<sub>Nuc</sub></em> = (<em>Tot<sub>Seq</sub></em>−<em>Nuc<sub>exp</sub></em>)/<em>Nuc<sub>exp</sub></em> (equation 18), where Tot<sub>Seq </sub>is the total number of sequences at the mismatch load. This way the strength of any given sequence, of the same length (C) as the DUES consensus, can be quantified from the overall conservation of its nucleotides across mismatch loads as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000741.e009&amp;representation=PNG" class="inline-graphic"></span> (equation 19), where p is the position.</p>

<h5>Dependence of the uptake bias on the strength of the sequence.</h5><a id="article1.body1.sec4.sec1.sec5.p1" name="article1.body1.sec4.sec1.sec5.p1"></a><p>I searched the 28 DNA fragments tested for DNA uptake in <a href="#pone.0000741-Goodgal4">[40]</a> (Accession no. M33432 to M33459) and extracted then calculated the strength of all their 29 bp sequences using the program <em>Sequences_Extractor.pl</em> and equation 19. To test whether the genomic overrepresentation of DUES nucleotides reflects the efficiency of their uptake, I tested the dependence of the uptake of these 28 DNA fragments on the strength of their 29 bp sequences by regression analyses using <em>Statistica version 5.1</em>. As negative control the positions of these 29 bp sequences were randomized, using a script written in <em>Perl</em>, and a similar regression analysis performed on the resulting sequence strengths and the uptake of the DNA fragment from where the sequences were originally extracted.</p>

<h5>Experimental analysis.</h5><a id="article1.body1.sec4.sec1.sec6.p1" name="article1.body1.sec4.sec1.sec6.p1"></a><p>To further test the theoretical results, synthetic DNA sequences containing the most conserved <em>H. influenzae</em> DUES consensus or one of its mutated forms were tested for uptake efficiency by competent <em>H. influenzae</em>. Both a 29 bp sequence of the most frequent nucleotide at each <em>H. influenzae</em> DUES consensus position (best DUES) and another of the least frequent ones (worst sequence) were synthesized and cloned by blunt-end ligation into the <em>Sma</em>I site of the plasmid pGEM-7Zf(-) (Accession no. X65311). Control 222 bp DNA template fragments containing the best DUES or the worst sequence were then PCR amplified from the constructs. Sequences carrying point mutations towards the least frequent nucleotide were generated from the best DUES construct using a three-step cohesive-end PCR process. For each desired mutant, two half-fragments (113 bp and 135 bp, with 26 bp overlap) were first produced using overlapping internal primers mutated at the chosen DUES position. After gel electrophoresis and purification, both PCR products were combined as template for a third cohesive-end PCR reaction. The final 222 bp PCR products were then gel purified and sequenced before use as templates for Klenow radio-labeling reactions. These were carried out for three hours at room temperature and, for the initial hour, contained limiting α-<sup>33</sup>P-dATP (6.0 µM). Subsequent addition of unlabeled (cold) dATP ensured complete replication of each molecule. DNA was then gel purified, and incorporation of α-<sup>33</sup>P-dATP checked by autoradiography after electrophoresis of an aliquot in acrylamide gel. Radio-labeled DNAs were then mixed with unlabelled DNAs of the same sequence to a specific radioactivity of 1,000 counts per minute (cpm) per ηg of DNA. 10, 20, 30 or 40 ηg of each DNA sequence was then separately added to 0.5 ml of competent <em>H. influenzae</em> cells for transformation as described in <a href="#pone.0000741-Poje1">[68]</a>. After 15 min incubation at 37 C in rotating tubes, 25 µl of ice cold DNaseI (1 mg/ml) was added to each tube before gentle vortexing and incubation for 5 min on ice. 50 µl of ice cold 5 M NaCl was then added and the tubes gently vortexed then centrifuged at 16,000 g for 1 min at 4 °C. After re-suspension in ice-cold MIV medium <a href="#pone.0000741-Herriott1">[39]</a> containing 1 M NaCl, gentle vortexing and centrifugation, the final pellets were re-suspended in scintillation vials containing 200 µl MIV medium and 1 ml aqueous scintillation fluid at room temperature. Scintillation count was carried out in a Beckman Scintillation Counter. For standardization and accuracy, vials were counted simultaneously, P<sup>33</sup> decay factor considered, and the background's P<sup>33</sup> cpm subtracted. The nanograms of DNA taken up were estimated using its specific radioactivity. Experiments were carried out in triplicate.</p>


</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000741"></div><a name="pone.0000741.s001" id="pone.0000741.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000741.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Goodness of fit Chi-squared test (<em>χ</em><sup>2</sup>) on the observed and expected numbers of DUESs, control sequences, as well as all their mismatched forms.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000741.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.12 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>I am very thankful to my wife Mrs. Pernille Lavgesen for tolerating my working hours and for her continuous support. I also wish to thank Dr. Rosemary J. Redfield for allowing me to use the experimental data obtained in her laboratory thanks to the appreciable help of Mrs. Becky Dinesen M.Sc., Dr. Ta-Yuan Chen for providing me with the program search_USSmissmatches_sequence.pl and allowing me to modify it, and Dr. Chris M. Wade for allowing me access to his powerful computer. To the daughter or son my wife and I are expecting.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Wrote the paper: MB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000741-Lorenz1" id="pone.0000741-Lorenz1"></a>Lorenz MG, Wackernagel W (1994) Bacterial gene transfer by natural genetic transformation in the environment. Microbiol Rev  58: 563–602.  <ul class="find" data-citedArticleID="977688"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bacterial+gene+transfer+by+natural+genetic+transformation+in+the+environment.&amp;auth=&amp;atitle=Bacterial+gene+transfer+by+natural+genetic+transformation+in+the+environment." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+gene+transfer+by+natural+genetic+transformation+in+the+environment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+gene+transfer+by+natural+genetic+transformation+in+the+environment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000741-Dubnau1" id="pone.0000741-Dubnau1"></a>Dubnau D (1999) DNA uptake in bacteria. Annu Rev Microbiol  53: 217–244.  <ul class="find" data-citedArticleID="977646" data-doi="10.1146/annurev.micro.53.1.217"><li><a href="http://dx.doi.org/10.1146/annurev.micro.53.1.217" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+uptake+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+uptake+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000741-Danner1" id="pone.0000741-Danner1"></a>Danner DB, Smith HO, Narang SA (1982) Construction of DNA recognition sites active in <em>Haemophilus</em> transformation. Proc Natl Acad Sci U S A  79: 2393–2397.  <ul class="find" data-citedArticleID="977642" data-doi="10.1073/pnas.79.7.2393"><li><a href="http://dx.doi.org/10.1073/pnas.79.7.2393" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+DNA+recognition+sites+active+in+Haemophilus+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+DNA+recognition+sites+active+in+Haemophilus+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000741-Goodgal1" id="pone.0000741-Goodgal1"></a>Goodgal SH (1982) DNA uptake in <em>Haemophilus</em> transformation. Annu Rev Genet  16: 169–192.  <ul class="find" data-citedArticleID="977660" data-doi="10.1146/annurev.ge.16.120182.001125"><li><a href="http://dx.doi.org/10.1146/annurev.ge.16.120182.001125" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+uptake+in+Haemophilus+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+uptake+in+Haemophilus+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000741-Mathis1" id="pone.0000741-Mathis1"></a>Mathis LS, Scocca JJ (1982) <em>Haemophilus influenzae</em> and <em>Neisseria gonorrhoeae</em> recognize different specificity determinants in the DNA uptake step of genetic transformation. J Gen Microbiol  128: 1159–1161.  <ul class="find" data-citedArticleID="977690" data-doi="10.1099/00221287-128-5-1159"><li><a href="http://dx.doi.org/10.1099/00221287-128-5-1159" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haemophilus+influenzae+and+Neisseria+gonorrhoeae+recognize+different+specificity+determinants+in+the+DNA+uptake+step+of+genetic+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Haemophilus+influenzae+and+Neisseria+gonorrhoeae+recognize+different+specificity+determinants+in+the+DNA+uptake+step+of+genetic+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000741-Wang1" id="pone.0000741-Wang1"></a>Wang Y, Goodman SD, Redfield RJ, Chen C (2002) Natural transformation and DNA uptake signal sequences in <em>Actinobacillus actinomycetemcomitans</em>. J Bacteriol  184: 3442–3449.  <ul class="find" data-citedArticleID="977744" data-doi="10.1128/jb.184.13.3442-3449.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.13.3442-3449.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+transformation+and+DNA+uptake+signal+sequences+in+Actinobacillus+actinomycetemcomitans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+transformation+and+DNA+uptake+signal+sequences+in+Actinobacillus+actinomycetemcomitans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000741-Elkins1" id="pone.0000741-Elkins1"></a>Elkins C, Thomas CE, Seifert HS, Sparling PF (1991) Species-specific uptake of DNA by gonococci is mediated by a 10-base-pair sequence. J Bacteriol  173: 3911–3913.  <ul class="find" data-citedArticleID="977650"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Species-specific+uptake+of+DNA+by+gonococci+is+mediated+by+a+10-base-pair+sequence.&amp;auth=&amp;atitle=Species-specific+uptake+of+DNA+by+gonococci+is+mediated+by+a+10-base-pair+sequence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Species-specific+uptake+of+DNA+by+gonococci+is+mediated+by+a+10-base-pair+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Species-specific+uptake+of+DNA+by+gonococci+is+mediated+by+a+10-base-pair+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000741-Graves1" id="pone.0000741-Graves1"></a>Graves JF, Biswas GD, Sparling PF (1982) Sequence-specific DNA uptake in transformation of <em>Neisseria gonorrhoeae</em>. J Bacteriol  152: 1071–1077.  <ul class="find" data-citedArticleID="977670"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sequence-specific+DNA+uptake+in+transformation+of+Neisseria+gonorrhoeae.&amp;auth=&amp;atitle=Sequence-specific+DNA+uptake+in+transformation+of+Neisseria+gonorrhoeae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence-specific+DNA+uptake+in+transformation+of+Neisseria+gonorrhoeae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence-specific+DNA+uptake+in+transformation+of+Neisseria+gonorrhoeae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000741-Wang2" id="pone.0000741-Wang2"></a>Wang Y, Orvis J, Dyer D, Chen C (2006) Genomic distribution and functions of uptake signal sequences in <em>Actinobacillus actinomycetemcomitans</em>. Microbiology  152: 3319–3325.  <ul class="find" data-citedArticleID="977746" data-doi="10.1099/mic.0.29018-0"><li><a href="http://dx.doi.org/10.1099/mic.0.29018-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+distribution+and+functions+of+uptake+signal+sequences+in+Actinobacillus+actinomycetemcomitans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+distribution+and+functions+of+uptake+signal+sequences+in+Actinobacillus+actinomycetemcomitans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000741-Tettelin1" id="pone.0000741-Tettelin1"></a>Tettelin H, Saunders NJ, Heidelberg J, Jeffries AC, Nelson KE, et al.  (2000) Complete genome sequence of <em>Neisseria meningitidis</em> serogroup B strain MC58. Science  287: 1809–1815.  <ul class="find" data-citedArticleID="977742" data-doi="10.1126/science.287.5459.1809"><li><a href="http://dx.doi.org/10.1126/science.287.5459.1809" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+Neisseria+meningitidis+serogroup+B+strain+MC58." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+Neisseria+meningitidis+serogroup+B+strain+MC58.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000741-Davidsen1" id="pone.0000741-Davidsen1"></a>Davidsen T, Rodland EA, Lagesen K, Seeberg E, Rognes T, et al.  (2004) Biased distribution of DNA uptake sequences towards genome maintenance genes. Nucleic Acids Res  32: 1050–1058.  <ul class="find" data-citedArticleID="977644" data-doi="10.1093/nar/gkh255"><li><a href="http://dx.doi.org/10.1093/nar/gkh255" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biased+distribution+of+DNA+uptake+sequences+towards+genome+maintenance+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biased+distribution+of+DNA+uptake+sequences+towards+genome+maintenance+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000741-Parkhill1" id="pone.0000741-Parkhill1"></a>Parkhill J, Achtman M, James KD, Bentley SD, Churcher C, et al.  (2000) Complete DNA sequence of a serogroup A strain of <em>Neisseria meningitidis</em> Z2491. Nature  404: 502–506.  <ul class="find" data-citedArticleID="977700" data-doi="10.1038/35006655"><li><a href="http://dx.doi.org/10.1038/35006655" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+DNA+sequence+of+a+serogroup+A+strain+of+Neisseria+meningitidis+Z2491." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+DNA+sequence+of+a+serogroup+A+strain+of+Neisseria+meningitidis+Z2491.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000741-Smith1" id="pone.0000741-Smith1"></a>Smith HO, Gwinn ML, Salzberg SL (1999) DNA uptake signal sequences in naturally transformable bacteria. Res Microbiol  150: 603–616.  <ul class="find" data-citedArticleID="977730" data-doi="10.1016/s0923-2508(99)00130-8"><li><a href="http://dx.doi.org/10.1016/s0923-2508(99)00130-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+uptake+signal+sequences+in+naturally+transformable+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+uptake+signal+sequences+in+naturally+transformable+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000741-Smith2" id="pone.0000741-Smith2"></a>Smith HO, Tomb JF, Dougherty BA, Fleischmann RD, Venter JC (1995) Frequency and distribution of DNA uptake signal sequences in the <em>Haemophilus influenzae</em> Rd genome. Science  269: 538–540.  <ul class="find" data-citedArticleID="977732" data-doi="10.1126/science.7542802"><li><a href="http://dx.doi.org/10.1126/science.7542802" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frequency+and+distribution+of+DNA+uptake+signal+sequences+in+the+Haemophilus+influenzae+Rd+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frequency+and+distribution+of+DNA+uptake+signal+sequences+in+the+Haemophilus+influenzae+Rd+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000741-Bakkali1" id="pone.0000741-Bakkali1"></a>Bakkali M, Chen TY, Lee HC, Redfield RJ (2004) Evolutionary stability of DNA uptake signal sequences in the Pasteurellaceae. Proc Natl Acad Sci U S A  101: 4513–4518.  <ul class="find" data-citedArticleID="977612" data-doi="10.1073/pnas.0306366101"><li><a href="http://dx.doi.org/10.1073/pnas.0306366101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+stability+of+DNA+uptake+signal+sequences+in+the+Pasteurellaceae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+stability+of+DNA+uptake+signal+sequences+in+the+Pasteurellaceae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000741-Redfield1" id="pone.0000741-Redfield1"></a>Redfield RJ, Findlay WA, Bosse J, Kroll JS, Cameron AD, et al.  (2006) Evolution of competence and DNA uptake specificity in the Pasteurellaceae. BMC Evol Biol  6: 82.  <ul class="find" data-citedArticleID="977704" data-doi="10.1186/1471-2148-6-82"><li><a href="http://dx.doi.org/10.1186/1471-2148-6-82" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+competence+and+DNA+uptake+specificity+in+the+Pasteurellaceae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+competence+and+DNA+uptake+specificity+in+the+Pasteurellaceae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000741-Bigas1" id="pone.0000741-Bigas1"></a>Bigas A, Garrido ME, de Rozas AM, Badiola I, Barbe J, et al.  (2005) Development of a genetic manipulation system for <em>Haemophilus parasuis</em>. Vet Microbiol  105: 223–228.  <ul class="find" data-citedArticleID="977616" data-doi="10.1016/j.vetmic.2004.10.015"><li><a href="http://dx.doi.org/10.1016/j.vetmic.2004.10.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+a+genetic+manipulation+system+for+Haemophilus+parasuis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+a+genetic+manipulation+system+for+Haemophilus+parasuis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000741-Gioia1" id="pone.0000741-Gioia1"></a>Gioia J, Qin X, Jiang H, Clinkenbeard K, Lo R, et al.  (2006) The genome sequence of <em>Mannheimia haemolytica</em> A1: insights into virulence, natural competence, and Pasteurellaceae phylogeny. J Bacteriol  188: 7257–7266.  <ul class="find" data-citedArticleID="977658" data-doi="10.1128/jb.00675-06"><li><a href="http://dx.doi.org/10.1128/jb.00675-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+Mannheimia+haemolytica+A1%3A+insights+into+virulence%2C+natural+competence%2C+and+Pasteurellaceae+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+Mannheimia+haemolytica+A1%3A+insights+into+virulence%2C+natural+competence%2C+and+Pasteurellaceae+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000741-Chen1" id="pone.0000741-Chen1"></a>Chen I, Christie PJ, Dubnau D (2005) The ins and outs of DNA transfer in bacteria. Science  310: 1456–1460.  <ul class="find" data-citedArticleID="977626" data-doi="10.1126/science.1114021"><li><a href="http://dx.doi.org/10.1126/science.1114021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ins+and+outs+of+DNA+transfer+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ins+and+outs+of+DNA+transfer+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000741-Chen2" id="pone.0000741-Chen2"></a>Chen I, Dubnau D (2003) DNA transport during transformation. Front Biosci  8: s544–556.  <ul class="find" data-citedArticleID="977628" data-doi="10.2741/1047"><li><a href="http://dx.doi.org/10.2741/1047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+transport+during+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+transport+during+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000741-Chen3" id="pone.0000741-Chen3"></a>Chen I, Dubnau D (2004) DNA uptake during bacterial transformation. Nat Rev Microbiol  2: 241–249.  <ul class="find" data-citedArticleID="977630" data-doi="10.1038/nrmicro844"><li><a href="http://dx.doi.org/10.1038/nrmicro844" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+uptake+during+bacterial+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+uptake+during+bacterial+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000741-Goodman1" id="pone.0000741-Goodman1"></a>Goodman SD, Scocca JJ (1988) Identification and arrangement of the DNA sequence recognized in specific transformation of <em>Neisseria gonorrhoeae</em>. Proc Natl Acad Sci U S A  85: 6982–6986.  <ul class="find" data-citedArticleID="977668" data-doi="10.1073/pnas.85.18.6982"><li><a href="http://dx.doi.org/10.1073/pnas.85.18.6982" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+arrangement+of+the+DNA+sequence+recognized+in+specific+transformation+of+Neisseria+gonorrhoeae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+arrangement+of+the+DNA+sequence+recognized+in+specific+transformation+of+Neisseria+gonorrhoeae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000741-Karlin1" id="pone.0000741-Karlin1"></a>Karlin S, Mrazek J, Campbell AM (1996) Frequent oligonucleotides and peptides of the <em>Haemophilus influenzae</em> genome. Nucleic Acids Res  24: 4263–4272.  <ul class="find" data-citedArticleID="977678" data-doi="10.1093/nar/24.21.4263"><li><a href="http://dx.doi.org/10.1093/nar/24.21.4263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frequent+oligonucleotides+and+peptides+of+the+Haemophilus+influenzae+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frequent+oligonucleotides+and+peptides+of+the+Haemophilus+influenzae+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000741-Redfield2" id="pone.0000741-Redfield2"></a>Redfield RJ (1991) Bacteria mating preferences. Nature  352: 25–26.  <ul class="find" data-citedArticleID="977706" data-doi="10.1038/352025b0"><li><a href="http://dx.doi.org/10.1038/352025b0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteria+mating+preferences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteria+mating+preferences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000741-Chu1" id="pone.0000741-Chu1"></a>Chu D, Lee HC, Lenaerts T (2005) Evolution of DNA uptake signal sequences. Artif Life  11: 317–338.  <ul class="find" data-citedArticleID="977632" data-doi="10.1162/1064546054407176"><li><a href="http://dx.doi.org/10.1162/1064546054407176" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+DNA+uptake+signal+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+DNA+uptake+signal+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000741-Redfield3" id="pone.0000741-Redfield3"></a>Redfield RJ (1993) Genes for breakfast: the have-your-cake-and-eat-it-too of bacterial transformation. J Hered  84: 400–404.  <ul class="find" data-citedArticleID="977708"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genes+for+breakfast%3A+the+have-your-cake-and-eat-it-too+of+bacterial+transformation.&amp;auth=&amp;atitle=Genes+for+breakfast%3A+the+have-your-cake-and-eat-it-too+of+bacterial+transformation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+for+breakfast%3A+the+have-your-cake-and-eat-it-too+of+bacterial+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+for+breakfast%3A+the+have-your-cake-and-eat-it-too+of+bacterial+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000741-Redfield4" id="pone.0000741-Redfield4"></a>Redfield RJ (2001) Do bacteria have sex? Nat Rev Genet  2: 634–639.  <ul class="find" data-citedArticleID="977710" data-doi="10.1038/35084593"><li><a href="http://dx.doi.org/10.1038/35084593" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Do+bacteria+have+sex%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Do+bacteria+have+sex%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000741-Solomon1" id="pone.0000741-Solomon1"></a>Solomon JM, Grossman AD (1996) Who's competent and when: regulation of natural genetic competence in bacteria. Trends Genet  12: 150–155.  <ul class="find" data-citedArticleID="977734" data-doi="10.1016/0168-9525(96)10014-7"><li><a href="http://dx.doi.org/10.1016/0168-9525(96)10014-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Who%27s+competent+and+when%3A+regulation+of+natural+genetic+competence+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Who%27s+competent+and+when%3A+regulation+of+natural+genetic+competence+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000741-Palchevskiy1" id="pone.0000741-Palchevskiy1"></a>Palchevskiy V, Finkel SE (2006) <em>Escherichia coli</em> competence gene homologs are essential for competitive fitness and the use of DNA as a nutrient. J Bacteriol  188: 3902–3910.  <ul class="find" data-citedArticleID="977698" data-doi="10.1128/jb.01974-05"><li><a href="http://dx.doi.org/10.1128/jb.01974-05" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Escherichia+coli+competence+gene+homologs+are+essential+for+competitive+fitness+and+the+use+of+DNA+as+a+nutrient." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Escherichia+coli+competence+gene+homologs+are+essential+for+competitive+fitness+and+the+use+of+DNA+as+a+nutrient.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000741-Finkel1" id="pone.0000741-Finkel1"></a>Finkel SE, Kolter R (2001) DNA as a nutrient: novel role for bacterial competence gene homologs. J Bacteriol  183: 6288–6293.  <ul class="find" data-citedArticleID="977652" data-doi="10.1128/jb.183.21.6288-6293.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.21.6288-6293.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+as+a+nutrient%3A+novel+role+for+bacterial+competence+gene+homologs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+as+a+nutrient%3A+novel+role+for+bacterial+competence+gene+homologs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000741-Woegerbauer1" id="pone.0000741-Woegerbauer1"></a>Woegerbauer M, Jenni B, Thalhammer F, Graninger W, Burgmann H (2002) Natural genetic transformation of clinical isolates of <em>Escherichia coli</em> in urine and water. Appl Environ Microbiol  68: 440–443.  <ul class="find" data-citedArticleID="977748" data-doi="10.1128/aem.68.1.440-443.2002"><li><a href="http://dx.doi.org/10.1128/aem.68.1.440-443.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+genetic+transformation+of+clinical+isolates+of+Escherichia+coli+in+urine+and+water." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+genetic+transformation+of+clinical+isolates+of+Escherichia+coli+in+urine+and+water.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000741-Albritton1" id="pone.0000741-Albritton1"></a>Albritton WL, Setlow JK, Thomas M, Sottnek F, Steigerwalt AG (1984) Heterospecific transformation in the genus <em>Haemophilus</em>. Mol Gen Genet  193: 358–363.  <ul class="find" data-citedArticleID="977610" data-doi="10.1007/bf00330693"><li><a href="http://dx.doi.org/10.1007/bf00330693" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterospecific+transformation+in+the+genus+Haemophilus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterospecific+transformation+in+the+genus+Haemophilus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000741-Goodgal2" id="pone.0000741-Goodgal2"></a>Goodgal SH, Mitchell M (1984) Uptake of heterologous DNA by <em>Haemophilus influenzae</em>. J Bacteriol  157: 785–788.  <ul class="find" data-citedArticleID="977662"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Uptake+of+heterologous+DNA+by+Haemophilus+influenzae.&amp;auth=&amp;atitle=Uptake+of+heterologous+DNA+by+Haemophilus+influenzae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uptake+of+heterologous+DNA+by+Haemophilus+influenzae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Uptake+of+heterologous+DNA+by+Haemophilus+influenzae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000741-BoyleVavra1" id="pone.0000741-BoyleVavra1"></a>Boyle-Vavra S, Seifert HS (1996) Uptake-sequence-independent DNA transformation exists in <em>Neisseria gonorrhoeae</em>. Microbiology  142(Pt 10): 2839–2845.  <ul class="find" data-citedArticleID="977622" data-doi="10.1099/13500872-142-10-2839"><li><a href="http://dx.doi.org/10.1099/13500872-142-10-2839" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uptake-sequence-independent+DNA+transformation+exists+in+Neisseria+gonorrhoeae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Uptake-sequence-independent+DNA+transformation+exists+in+Neisseria+gonorrhoeae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000741-Chu2" id="pone.0000741-Chu2"></a>Chu D, Rowe J, Lee HC (2006) Evaluation of the current models for the evolution of bacterial DNA uptake signal sequences. J Theor Biol  238: 157–166.  <ul class="find" data-citedArticleID="977634" data-doi="10.1016/j.jtbi.2005.05.024"><li><a href="http://dx.doi.org/10.1016/j.jtbi.2005.05.024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+the+current+models+for+the+evolution+of+bacterial+DNA+uptake+signal+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+the+current+models+for+the+evolution+of+bacterial+DNA+uptake+signal+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000741-Biswas1" id="pone.0000741-Biswas1"></a>Biswas GD, Sox T, Blackman E, Sparling PF (1977) Factors affecting genetic transformation of <em>Neisseria gonorrhoeae</em>. J Bacteriol  129: 983–992.  <ul class="find" data-citedArticleID="977618"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Factors+affecting+genetic+transformation+of+Neisseria+gonorrhoeae.&amp;auth=&amp;atitle=Factors+affecting+genetic+transformation+of+Neisseria+gonorrhoeae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Factors+affecting+genetic+transformation+of+Neisseria+gonorrhoeae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Factors+affecting+genetic+transformation+of+Neisseria+gonorrhoeae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000741-Biswas2" id="pone.0000741-Biswas2"></a>Biswas GD, Sparling PF (1981) Entry of double-stranded deoxyribonucleic acid during transformation of <em>Neisseria gonorrhoeae</em>. J Bacteriol  145: 638–640.  <ul class="find" data-citedArticleID="977620"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Entry+of+double-stranded+deoxyribonucleic+acid+during+transformation+of+Neisseria+gonorrhoeae.&amp;auth=&amp;atitle=Entry+of+double-stranded+deoxyribonucleic+acid+during+transformation+of+Neisseria+gonorrhoeae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Entry+of+double-stranded+deoxyribonucleic+acid+during+transformation+of+Neisseria+gonorrhoeae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Entry+of+double-stranded+deoxyribonucleic+acid+during+transformation+of+Neisseria+gonorrhoeae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000741-Goodgal3" id="pone.0000741-Goodgal3"></a>Goodgal SH, Herriott RM (1961) Studies on transformations of <em>Haemophilus influenzae</em>. I. Competence. J Gen Physiol  44: 1201–1227.  <ul class="find" data-citedArticleID="977664" data-doi="10.1085/jgp.44.6.1201"><li><a href="http://dx.doi.org/10.1085/jgp.44.6.1201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Studies+on+transformations+of+Haemophilus+influenzae.+I.+Competence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Studies+on+transformations+of+Haemophilus+influenzae.+I.+Competence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000741-Herriott1" id="pone.0000741-Herriott1"></a>Herriott RM, Meyer EM, Vogt M (1970) Defined nongrowth media for stage II development of competence in <em>Haemophilus influenzae</em>. J Bacteriol  101: 517–524.  <ul class="find" data-citedArticleID="977674"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Defined+nongrowth+media+for+stage+II+development+of+competence+in+Haemophilus+influenzae.&amp;auth=&amp;atitle=Defined+nongrowth+media+for+stage+II+development+of+competence+in+Haemophilus+influenzae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Defined+nongrowth+media+for+stage+II+development+of+competence+in+Haemophilus+influenzae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Defined+nongrowth+media+for+stage+II+development+of+competence+in+Haemophilus+influenzae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000741-Goodgal4" id="pone.0000741-Goodgal4"></a>Goodgal SH, Mitchell MA (1990) Sequence and uptake specificity of cloned sonicated fragments of <em>Haemophilus influenzae</em> DNA. J Bacteriol  172: 5924–5928.  <ul class="find" data-citedArticleID="977666"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sequence+and+uptake+specificity+of+cloned+sonicated+fragments+of+Haemophilus+influenzae+DNA.&amp;auth=&amp;atitle=Sequence+and+uptake+specificity+of+cloned+sonicated+fragments+of+Haemophilus+influenzae+DNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+and+uptake+specificity+of+cloned+sonicated+fragments+of+Haemophilus+influenzae+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+and+uptake+specificity+of+cloned+sonicated+fragments+of+Haemophilus+influenzae+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000741-Scocca1" id="pone.0000741-Scocca1"></a>Scocca JJ, Poland RL, Zoon KC (1974) Specificity in deoxyribonucleic acid uptake by transformable <em>Haemophilus influenzae</em>. J Bacteriol  118: 369–373.  <ul class="find" data-citedArticleID="977722"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Specificity+in+deoxyribonucleic+acid+uptake+by+transformable+Haemophilus+influenzae.&amp;auth=&amp;atitle=Specificity+in+deoxyribonucleic+acid+uptake+by+transformable+Haemophilus+influenzae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specificity+in+deoxyribonucleic+acid+uptake+by+transformable+Haemophilus+influenzae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specificity+in+deoxyribonucleic+acid+uptake+by+transformable+Haemophilus+influenzae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000741-Cohan1" id="pone.0000741-Cohan1"></a>Cohan FM (2001) Bacterial species and speciation. Syst Biol  50: 513–524.  <ul class="find" data-citedArticleID="977636"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bacterial+species+and+speciation.&amp;auth=&amp;atitle=Bacterial+species+and+speciation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+species+and+speciation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+species+and+speciation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000741-Moreno1" id="pone.0000741-Moreno1"></a>Moreno E (1997) In search of a bacterial species definition. Rev Biol Trop  45: 753–771.  <ul class="find" data-citedArticleID="977696"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+search+of+a+bacterial+species+definition.&amp;auth=&amp;atitle=In+search+of+a+bacterial+species+definition." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+search+of+a+bacterial+species+definition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+search+of+a+bacterial+species+definition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000741-Konstantinidis1" id="pone.0000741-Konstantinidis1"></a>Konstantinidis KT, Ramette A, Tiedje JM (2006) The bacterial species definition in the genomic era. Philos Trans R Soc Lond B Biol Sci  361: 1929–1940.  <ul class="find" data-citedArticleID="977686" data-doi="10.1098/rstb.2006.1920"><li><a href="http://dx.doi.org/10.1098/rstb.2006.1920" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+bacterial+species+definition+in+the+genomic+era." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+bacterial+species+definition+in+the+genomic+era.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000741-Cohan2" id="pone.0000741-Cohan2"></a>Cohan FM (2002) What are bacterial species? Annu Rev Microbiol  56: 457–487.  <ul class="find" data-citedArticleID="977638" data-doi="10.1146/annurev.micro.56.012302.160634"><li><a href="http://dx.doi.org/10.1146/annurev.micro.56.012302.160634" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=What+are+bacterial+species%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22What+are+bacterial+species%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000741-Hanage1" id="pone.0000741-Hanage1"></a>Hanage WP, Fraser C, Spratt BG (2005) Fuzzy species among recombinogenic bacteria. BMC Biol  3: 6.  <ul class="find" data-citedArticleID="977672" data-doi="10.1186/1741-7007-3-6"><li><a href="http://dx.doi.org/10.1186/1741-7007-3-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fuzzy+species+among+recombinogenic+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fuzzy+species+among+recombinogenic+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000741-Ahmad1" id="pone.0000741-Ahmad1"></a>Ahmad S, Kono H, Arauzo-Bravo MJ, Sarai A (2006) ReadOut: structure-based calculation of direct and indirect readout energies and specificities for protein-DNA recognition. Nucleic Acids Res  34: W124–127.  <ul class="find" data-citedArticleID="977608" data-doi="10.1093/nar/gkl104"><li><a href="http://dx.doi.org/10.1093/nar/gkl104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ReadOut%3A+structure-based+calculation+of+direct+and+indirect+readout+energies+and+specificities+for+protein-DNA+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ReadOut%3A+structure-based+calculation+of+direct+and+indirect+readout+energies+and+specificities+for+protein-DNA+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000741-Jonsson1" id="pone.0000741-Jonsson1"></a>Jonsson AB, Nyberg G, Normark S (1991) Phase variation of gonococcal pili by frameshift mutation in pilC, a novel gene for pilus assembly. Embo J  10: 477–488.  <ul class="find" data-citedArticleID="977676"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phase+variation+of+gonococcal+pili+by+frameshift+mutation+in+pilC%2C+a+novel+gene+for+pilus+assembly.&amp;auth=&amp;atitle=Phase+variation+of+gonococcal+pili+by+frameshift+mutation+in+pilC%2C+a+novel+gene+for+pilus+assembly." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phase+variation+of+gonococcal+pili+by+frameshift+mutation+in+pilC%2C+a+novel+gene+for+pilus+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phase+variation+of+gonococcal+pili+by+frameshift+mutation+in+pilC%2C+a+novel+gene+for+pilus+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000741-Rudel1" id="pone.0000741-Rudel1"></a>Rudel T, Facius D, Barten R, Scheuerpflug I, Nonnenmacher E, et al.  (1995) Role of pili and the phase-variable PilC protein in natural competence for transformation of <em>Neisseria gonorrhoeae</em>. Proc Natl Acad Sci U S A  92: 7986–7990.  <ul class="find" data-citedArticleID="977714" data-doi="10.1073/pnas.92.17.7986"><li><a href="http://dx.doi.org/10.1073/pnas.92.17.7986" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+pili+and+the+phase-variable+PilC+protein+in+natural+competence+for+transformation+of+Neisseria+gonorrhoeae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+pili+and+the+phase-variable+PilC+protein+in+natural+competence+for+transformation+of+Neisseria+gonorrhoeae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000741-Rudel2" id="pone.0000741-Rudel2"></a>Rudel T, Scheurerpflug I, Meyer TF (1995) <em>Neisseria</em> PilC protein identified as type-4 pilus tip-located adhesin. Nature  373: 357–359.  <ul class="find" data-citedArticleID="977716" data-doi="10.1038/373357a0"><li><a href="http://dx.doi.org/10.1038/373357a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neisseria+PilC+protein+identified+as+type-4+pilus+tip-located+adhesin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neisseria+PilC+protein+identified+as+type-4+pilus+tip-located+adhesin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000741-vanHam1" id="pone.0000741-vanHam1"></a>van Ham SM, van Alphen L, Mooi FR, van Putten JP (1994) The fimbrial gene cluster of <em>Haemophilus influenzae</em> type b. Mol Microbiol  13: 673–684.  <ul class="find" data-citedArticleID="977750" data-doi="10.1111/j.1365-2958.1994.tb00461.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1994.tb00461.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fimbrial+gene+cluster+of+Haemophilus+influenzae+type+b." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fimbrial+gene+cluster+of+Haemophilus+influenzae+type+b.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000741-StGeme1" id="pone.0000741-StGeme1"></a>St Geme JW, Pinkner JS 3rd, Krasan GP, Heuser J, Bullitt E, et al.  (1996) <em>Haemophilus influenzae</em> pili are composite structures assembled via the HifB chaperone. Proc Natl Acad Sci U S A  93: 11913–11918.  <ul class="find" data-citedArticleID="977736" data-doi="10.1073/pnas.93.21.11913"><li><a href="http://dx.doi.org/10.1073/pnas.93.21.11913" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haemophilus+influenzae+pili+are+composite+structures+assembled+via+the+HifB+chaperone." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Haemophilus+influenzae+pili+are+composite+structures+assembled+via+the+HifB+chaperone.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000741-McCrea1" id="pone.0000741-McCrea1"></a>McCrea KW, Watson WJ, Gilsdorf JR, Marrs CF (1994) Identification of hifD and hifE in the pilus gene cluster of <em>Haemophilus influenzae</em> type b strain Eagan. Infect Immun  62: 4922–4928.  <ul class="find" data-citedArticleID="977692"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+hifD+and+hifE+in+the+pilus+gene+cluster+of+Haemophilus+influenzae+type+b+strain+Eagan.&amp;auth=&amp;atitle=Identification+of+hifD+and+hifE+in+the+pilus+gene+cluster+of+Haemophilus+influenzae+type+b+strain+Eagan." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+hifD+and+hifE+in+the+pilus+gene+cluster+of+Haemophilus+influenzae+type+b+strain+Eagan." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+hifD+and+hifE+in+the+pilus+gene+cluster+of+Haemophilus+influenzae+type+b+strain+Eagan.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000741-McCrea2" id="pone.0000741-McCrea2"></a>McCrea KW, Watson WJ, Gilsdorf JR, Marrs CF (1997) Identification of two minor subunits in the pilus of <em>Haemophilus influenzae</em>. J Bacteriol  179: 4227–4231.  <ul class="find" data-citedArticleID="977694"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+two+minor+subunits+in+the+pilus+of+Haemophilus+influenzae.&amp;auth=&amp;atitle=Identification+of+two+minor+subunits+in+the+pilus+of+Haemophilus+influenzae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+two+minor+subunits+in+the+pilus+of+Haemophilus+influenzae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+two+minor+subunits+in+the+pilus+of+Haemophilus+influenzae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000741-Gilsdorf1" id="pone.0000741-Gilsdorf1"></a>Gilsdorf JR, McCrea KW, Marrs CF (1997) Role of pili in <em>Haemophilus influenzae</em> adherence and colonization. Infect Immun  65: 2997–3002.  <ul class="find" data-citedArticleID="977656"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Role+of+pili+in+Haemophilus+influenzae+adherence+and+colonization.&amp;auth=&amp;atitle=Role+of+pili+in+Haemophilus+influenzae+adherence+and+colonization." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+pili+in+Haemophilus+influenzae+adherence+and+colonization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+pili+in+Haemophilus+influenzae+adherence+and+colonization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000741-Burge1" id="pone.0000741-Burge1"></a>Burge C, Campbell AM, Karlin S (1992) Over- and under-representation of short oligonucleotides in DNA sequences. Proc Natl Acad Sci U S A  89: 1358–1362.  <ul class="find" data-citedArticleID="977624" data-doi="10.1073/pnas.89.4.1358"><li><a href="http://dx.doi.org/10.1073/pnas.89.4.1358" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Over-+and+under-representation+of+short+oligonucleotides+in+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Over-+and+under-representation+of+short+oligonucleotides+in+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000741-Karlin2" id="pone.0000741-Karlin2"></a>Karlin S, Burge C, Campbell AM (1992) Statistical analyses of counts and distributions of restriction sites in DNA sequences. Nucleic Acids Res  20: 1363–1370.  <ul class="find" data-citedArticleID="977680" data-doi="10.1093/nar/20.6.1363"><li><a href="http://dx.doi.org/10.1093/nar/20.6.1363" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+analyses+of+counts+and+distributions+of+restriction+sites+in+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+analyses+of+counts+and+distributions+of+restriction+sites+in+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000741-Karlin3" id="pone.0000741-Karlin3"></a>Karlin S, Campbell AM, Mrazek J (1998) Comparative DNA analysis across diverse genomes. Annu Rev Genet  32: 185–225.  <ul class="find" data-citedArticleID="977682" data-doi="10.1146/annurev.genet.32.1.185"><li><a href="http://dx.doi.org/10.1146/annurev.genet.32.1.185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+DNA+analysis+across+diverse+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+DNA+analysis+across+diverse+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000741-Karlin4" id="pone.0000741-Karlin4"></a>Karlin S, Mrazek J, Campbell AM (1997) Compositional biases of bacterial genomes and evolutionary implications. J Bacteriol  179: 3899–3913.  <ul class="find" data-citedArticleID="977684"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Compositional+biases+of+bacterial+genomes+and+evolutionary+implications.&amp;auth=&amp;atitle=Compositional+biases+of+bacterial+genomes+and+evolutionary+implications." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compositional+biases+of+bacterial+genomes+and+evolutionary+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compositional+biases+of+bacterial+genomes+and+evolutionary+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000741-Sinha1" id="pone.0000741-Sinha1"></a>Sinha S, Tompa M (2002) Discovery of novel transcription factor binding sites by statistical overrepresentation. Nucleic Acids Res  30: 5549–5560.  <ul class="find" data-citedArticleID="977726" data-doi="10.1093/nar/gkf669"><li><a href="http://dx.doi.org/10.1093/nar/gkf669" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovery+of+novel+transcription+factor+binding+sites+by+statistical+overrepresentation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovery+of+novel+transcription+factor+binding+sites+by+statistical+overrepresentation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000741-Sinha2" id="pone.0000741-Sinha2"></a>Sinha S, Tompa M (2003) YMF: A program for discovery of novel transcription factor binding sites by statistical overrepresentation. Nucleic Acids Res  31: 3586–3588.  <ul class="find" data-citedArticleID="977728" data-doi="10.1093/nar/gkg618"><li><a href="http://dx.doi.org/10.1093/nar/gkg618" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=YMF%3A+A+program+for+discovery+of+novel+transcription+factor+binding+sites+by+statistical+overrepresentation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22YMF%3A+A+program+for+discovery+of+novel+transcription+factor+binding+sites+by+statistical+overrepresentation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000741-Elemento1" id="pone.0000741-Elemento1"></a>Elemento O, Tavazoie S (2005) Fast and systematic genome-wide discovery of conserved regulatory elements using a non-alignment based approach. Genome Biol  6: R18.  <ul class="find" data-citedArticleID="977648"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fast+and+systematic+genome-wide+discovery+of+conserved+regulatory+elements+using+a+non-alignment+based+approach.&amp;auth=&amp;atitle=Fast+and+systematic+genome-wide+discovery+of+conserved+regulatory+elements+using+a+non-alignment+based+approach." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fast+and+systematic+genome-wide+discovery+of+conserved+regulatory+elements+using+a+non-alignment+based+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fast+and+systematic+genome-wide+discovery+of+conserved+regulatory+elements+using+a+non-alignment+based+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000741-Sengupta1" id="pone.0000741-Sengupta1"></a>Sengupta AM, Djordjevic M, Shraiman BI (2002) Specificity and robustness in transcription control networks. Proc Natl Acad Sci U S A  99: 2072–2077.  <ul class="find" data-citedArticleID="977724" data-doi="10.1073/pnas.022388499"><li><a href="http://dx.doi.org/10.1073/pnas.022388499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specificity+and+robustness+in+transcription+control+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specificity+and+robustness+in+transcription+control+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000741-Gerland1" id="pone.0000741-Gerland1"></a>Gerland U, Hwa T (2002) On the selection and evolution of regulatory DNA motifs. J Mol Evol  55: 386–400.  <ul class="find" data-citedArticleID="977654" data-doi="10.1007/s00239-002-2335-z"><li><a href="http://dx.doi.org/10.1007/s00239-002-2335-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+selection+and+evolution+of+regulatory+DNA+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+selection+and+evolution+of+regulatory+DNA+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000741-Stormo1" id="pone.0000741-Stormo1"></a>Stormo GD, Fields DS (1998) Specificity, free energy and information content in protein-DNA interactions. Trends Biochem Sci  23: 109–113.  <ul class="find" data-citedArticleID="977738" data-doi="10.1016/s0968-0004(98)01187-6"><li><a href="http://dx.doi.org/10.1016/s0968-0004(98)01187-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specificity%2C+free+energy+and+information+content+in+protein-DNA+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specificity%2C+free+energy+and+information+content+in+protein-DNA+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000741-Schneider1" id="pone.0000741-Schneider1"></a>Schneider TD, Stephens RM (1990) Sequence logos: a new way to display consensus sequences. Nucleic Acids Res  18: 6097–6100.  <ul class="find" data-citedArticleID="977720" data-doi="10.1093/nar/18.20.6097"><li><a href="http://dx.doi.org/10.1093/nar/18.20.6097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+logos%3A+a+new+way+to+display+consensus+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+logos%3A+a+new+way+to+display+consensus+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000741-Crooks1" id="pone.0000741-Crooks1"></a>Crooks GE, Hon G, Chandonia JM, Brenner SE (2004) WebLogo: a sequence logo generator. Genome Res  14: 1188–1190.  <ul class="find" data-citedArticleID="977640" data-doi="10.1101/gr.849004"><li><a href="http://dx.doi.org/10.1101/gr.849004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WebLogo%3A+a+sequence+logo+generator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WebLogo%3A+a+sequence+logo+generator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000741-Poje1" id="pone.0000741-Poje1"></a>Poje G, Redfield RJ (2003) Transformation of <em>Haemophilus influenzae</em>. Methods Mol Med  71: 57–70.  <ul class="find" data-citedArticleID="977702" data-doi="10.1385/1-59259-321-6:57"><li><a href="http://dx.doi.org/10.1385/1-59259-321-6:57" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transformation+of+Haemophilus+influenzae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transformation+of+Haemophilus+influenzae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000741-Bianco1" id="pone.0000741-Bianco1"></a>Bianco PR, Kowalczykowski SC (1997) The recombination hotspot Chi is recognized by the translocating RecBCD enzyme as the single strand of DNA containing the sequence 5′-GCTGGTGG-3′. Proc Natl Acad Sci U S A  94: 6706–6711.  <ul class="find" data-citedArticleID="977614" data-doi="10.1073/pnas.94.13.6706"><li><a href="http://dx.doi.org/10.1073/pnas.94.13.6706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+recombination+hotspot+Chi+is+recognized+by+the+translocating+RecBCD+enzyme+as+the+single+strand+of+DNA+containing+the+sequence+5%26%23x2032%3B-GCTGGTGG-3%26%23x2032%3B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+recombination+hotspot+Chi+is+recognized+by+the+translocating+RecBCD+enzyme+as+the+single+strand+of+DNA+containing+the+sequence+5%26%23x2032%3B-GCTGGTGG-3%26%23x2032%3B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000741-Robinson1" id="pone.0000741-Robinson1"></a>Robinson NJ, Robinson PJ, Gupta A, Bleasby AJ, Whitton BA, et al.  (1995) Singular over-representation of an octameric palindrome, HIP1, in DNA from many cyanobacteria. Nucleic Acids Res  23: 729–735.  <ul class="find" data-citedArticleID="977712" data-doi="10.1093/nar/23.5.729"><li><a href="http://dx.doi.org/10.1093/nar/23.5.729" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Singular+over-representation+of+an+octameric+palindrome%2C+HIP1%2C+in+DNA+from+many+cyanobacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Singular+over-representation+of+an+octameric+palindrome%2C+HIP1%2C+in+DNA+from+many+cyanobacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000741-Scharnagl1" id="pone.0000741-Scharnagl1"></a>Scharnagl M, Richter S, Hagemann M (1998) The cyanobacterium <em>Synechocystis</em> sp. strain PCC 6803 expresses a DNA methyltransferase specific for the recognition sequence of the restriction endonuclease PvuI. J Bacteriol  180: 4116–4122.  <ul class="find" data-citedArticleID="977718"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+cyanobacterium+Synechocystis+sp.+strain+PCC+6803+expresses+a+DNA+methyltransferase+specific+for+the+recognition+sequence+of+the+restriction+endonuclease+PvuI.&amp;auth=&amp;atitle=The+cyanobacterium+Synechocystis+sp.+strain+PCC+6803+expresses+a+DNA+methyltransferase+specific+for+the+recognition+sequence+of+the+restriction+endonuclease+PvuI." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+cyanobacterium+Synechocystis+sp.+strain+PCC+6803+expresses+a+DNA+methyltransferase+specific+for+the+recognition+sequence+of+the+restriction+endonuclease+PvuI." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+cyanobacterium+Synechocystis+sp.+strain+PCC+6803+expresses+a+DNA+methyltransferase+specific+for+the+recognition+sequence+of+the+restriction+endonuclease+PvuI.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000741-Stumph1" id="pone.0000741-Stumph1"></a>Stumph WE, Kristo P, Tsai MJ, O'Malley BW (1981) A chicken middle-repetitive DNA sequence which shares homology with mammalian ubiquitous repeats. Nucleic Acids Res  9: 5383–5397.  <ul class="find" data-citedArticleID="977740" data-doi="10.1093/nar/9.20.5383"><li><a href="http://dx.doi.org/10.1093/nar/9.20.5383" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+chicken+middle-repetitive+DNA+sequence+which+shares+homology+with+mammalian+ubiquitous+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+chicken+middle-repetitive+DNA+sequence+which+shares+homology+with+mammalian+ubiquitous+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.XML" value="205321"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.PDF" value="548538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g001.PNG_L" value="373759"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g001.PNG_M" value="78011"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g001.PNG_S" value="10962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g001.TIF" value="821990"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g001.PNG_I" value="27812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t001.PNG_L" value="133054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t001.PNG_M" value="91600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t001.PNG_S" value="12145"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t001.TIF" value="632268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t001.PNG_I" value="30421"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g002.PNG_L" value="575428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g002.PNG_M" value="82818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g002.PNG_S" value="12248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g002.TIF" value="1521168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g002.PNG_I" value="38971"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g003.PNG_L" value="2256380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g003.PNG_M" value="179056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g003.PNG_S" value="18393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g003.TIF" value="4662980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g003.PNG_I" value="94367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t002.PNG_L" value="227150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t002.PNG_M" value="130188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t002.PNG_S" value="13147"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t002.TIF" value="953096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t002.PNG_I" value="40211"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t003.PNG_L" value="85852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t003.PNG_M" value="119927"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t003.PNG_S" value="17007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t003.TIF" value="381692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t004.PNG_L" value="55362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t004.PNG_M" value="75830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t004.PNG_S" value="12077"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t004.TIF" value="228532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t004.PNG_I" value="25018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t005.PNG_L" value="130404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t005.PNG_M" value="69930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t005.PNG_S" value="10667"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t005.TIF" value="622774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t005.PNG_I" value="20985"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g004.PNG_L" value="290753"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g004.PNG_M" value="73272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g004.PNG_S" value="11517"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g004.TIF" value="747862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g004.PNG_I" value="26641"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g005.PNG_L" value="513775"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g005.PNG_M" value="69308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g005.PNG_S" value="10445"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g005.TIF" value="1104956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.g005.PNG_I" value="37328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t006.PNG_L" value="449054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t006.PNG_M" value="189513"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t006.PNG_S" value="16655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t006.TIF" value="1863954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t006.PNG_I" value="57657"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t007.PNG_L" value="402825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t007.PNG_M" value="171474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t007.PNG_S" value="15824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t007.TIF" value="1669474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.t007.PNG_I" value="52249"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e001.PNG" value="17825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e001.TIF" value="34788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e002.PNG" value="18410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e002.TIF" value="35456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e003.PNG" value="17841"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e003.TIF" value="34780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e004.PNG" value="23671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e004.TIF" value="40660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e005.PNG" value="23420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e005.TIF" value="40140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e006.PNG" value="23633"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e006.TIF" value="40476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e007.PNG" value="23202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e007.TIF" value="39816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e008.PNG" value="11591"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e008.TIF" value="28060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e009.PNG" value="12767"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.e009.TIF" value="28680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000741.s001.DOC" value="121856"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000741&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000741" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000741&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000741&volume=&issue=&title=Genome Dynamics of Short Oligonucleotides: The Example of Bacterial DNA Uptake Enhancing Sequences&author_name=Mohammed%20Bakkali&start_page=1&end_page=16" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000741" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000741" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000741" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000741&amp;t=Genome%20Dynamics%20of%20Short%20Oligonucleotides%3A%20The%20Example%20of%20Bacterial%20DNA%20Uptake%20Enhancing%20Sequences" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000741&title=Genome%20Dynamics%20of%20Short%20Oligonucleotides%3A%20The%20Example%20of%20Bacterial%20DNA%20Uptake%20Enhancing%20Sequences&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000741&amp;title=Genome%20Dynamics%20of%20Short%20Oligonucleotides%3A%20The%20Example%20of%20Bacterial%20DNA%20Uptake%20Enhancing%20Sequences" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000741" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000741&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genome Dynamics of Short Oligonucleotides: The Example of Bacterial DNA Uptake Enhancing Sequences';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000741';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genome%20Dynamics%20of%20Short%20Oligonucleotides%3A%20The%20Example%20of%20Bacterial%20DNA%20Uptake%20Enhancing%20Sequences http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000741" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000741" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">



















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="24320"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="24320"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24320"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24320"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haemophilus+influenzae%22" title="Search for articles in the subject area:'Haemophilus influenzae'"><div class="flagText">Haemophilus influenzae</div></a>
              <div data-categoryid="44471" data-articleid="24320"
                   data-categoryname="Haemophilus influenzae"
                   class="flagImage" title="Flag 'Haemophilus influenzae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="24320"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Neisseria+gonorrhoeae%22" title="Search for articles in the subject area:'Neisseria gonorrhoeae'"><div class="flagText">Neisseria gonorrhoeae</div></a>
              <div data-categoryid="43649" data-articleid="24320"
                   data-categoryname="Neisseria gonorrhoeae"
                   class="flagImage" title="Flag 'Neisseria gonorrhoeae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotide+sequencing%22" title="Search for articles in the subject area:'Nucleotide sequencing'"><div class="flagText">Nucleotide sequencing</div></a>
              <div data-categoryid="19663" data-articleid="24320"
                   data-categoryname="Nucleotide sequencing"
                   class="flagImage" title="Flag 'Nucleotide sequencing' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=342'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4095'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7357&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
