

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Detailed Enzyme Kinetics in Terms of Biochemical Species: Study of Citrate Synthase</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001825"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001825" />

    <meta name="citation_title" content="Detailed Enzyme Kinetics in Terms of Biochemical Species: Study of Citrate Synthase"/>
    <meta itemprop="name" content="Detailed Enzyme Kinetics in Terms of Biochemical Species: Study of Citrate Synthase"/>

      <meta name="citation_author" content="Daniel A. Beard"/>
            <meta name="citation_author_institution" content="Biotechnology and Bioengineering Center and Department of Physiology, Medical College of Wisconsin, Milwaukee, Wisconsin, United States of America"/>
      <meta name="citation_author" content="Kalyan C. Vinnakota"/>
            <meta name="citation_author_institution" content="Biotechnology and Bioengineering Center and Department of Physiology, Medical College of Wisconsin, Milwaukee, Wisconsin, United States of America"/>
      <meta name="citation_author" content="Fan Wu"/>
            <meta name="citation_author_institution" content="Biotechnology and Bioengineering Center and Department of Physiology, Medical College of Wisconsin, Milwaukee, Wisconsin, United States of America"/>

    <meta name="citation_date" content="2008/3/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001825.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1825"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Relations between biochemical thermodynamics and biochemical kinetics.; citation_author=RA Alberty; citation_journal_title=Biophys Chem; citation_volume=124; citation_number=1; citation_pages=11-17; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Levels of thermodynamic treatment of biochemical reaction systems.; citation_author=RA Alberty; citation_journal_title=Biophys J; citation_volume=65; citation_number=2; citation_pages=1243-1254; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Can yeast glycolysis be understood in terms of in vitro kinetics of the constituent enzymes? Testing biochemistry.; citation_author=B Teusink; citation_author=J Passarge; citation_author=CA Reijenga; citation_author=E Esgalhado; citation_author=CC van der Weijden; citation_journal_title=Eur J Biochem; citation_volume=267; citation_number=3; citation_pages=5313-5329; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Dynamics of muscle glycogenolysis modeled with pH time course computation and pH-dependent reaction equilibria and enzyme kinetics.; citation_author=K Vinnakota; citation_author=ML Kemp; citation_author=MJ Kushmerick; citation_journal_title=Biophys J; citation_volume=91; citation_number=4; citation_pages=1264-1287; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A computational model for glycogenolysis in skeletal muscle.; citation_author=MJ Lambeth; citation_author=MJ Kushmerick; citation_journal_title=Ann Biomed Eng; citation_volume=30; citation_number=5; citation_pages=808-827; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Dynamic simulation of the human red blood cell metabolic network.; citation_author=N Jamshidi; citation_author=JS Edwards; citation_author=T Fahland; citation_author=GM Church; citation_author=BO Palsson; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=6; citation_pages=286-287; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Model of 2,3-bisphosphoglycerate metabolism in the human erythrocyte based on detailed enzyme kinetic equations: equations and parameter refinement.; citation_author=PJ Mulquiney; citation_author=PW Kuchel; citation_journal_title=Biochem J; citation_volume=342 Pt 3; citation_number=7; citation_pages=581-596; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The determination of enzyme dissocation constants.; citation_author=H Lineweaver; citation_author=D Burk; citation_journal_title=J Am Chem Soc; citation_volume=56; citation_number=8; citation_pages=658-666; citation_date=1934; " />
      <meta name="citation_reference" content="citation_title=The effect of pH on fumarase activity in acetate buffer.; citation_author=C Frieden; citation_author=RA Alberty; citation_journal_title=J Biol Chem; citation_volume=212; citation_number=9; citation_pages=859-868; citation_date=1955; " />
      <meta name="citation_reference" content="citation_author=IH Segel; citation_number=10; citation_pages=xxii, 957; citation_date=1975; citation_publisher=Wiley; " />
      <meta name="citation_reference" content="citation_author=A Cornish-Bowden; citation_number=11; citation_date=2004; citation_publisher=Portland Press; " />
      <meta name="citation_reference" content="citation_title=A schematic method of deriving the rate laws for enzyme-catalyzed reactions.; citation_author=EL King; citation_author=C Altman; citation_journal_title=J Phys Chem; citation_volume=60; citation_number=12; citation_pages=1375-1378; citation_date=1956; " />
      <meta name="citation_reference" content="citation_author=RA Alberty; citation_number=13; citation_date=2003; citation_publisher=Wiley-Interscience; " />
      <meta name="citation_reference" content="citation_title=Kinetic studies of citrate synthase from rat kidney and rat brain.; citation_author=Y Matsuoka; citation_author=PA Srere; citation_journal_title=J Biol Chem; citation_volume=248; citation_number=14; citation_pages=8022-8030; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=The kinetic properties of citrate synthase from rat liver mitochondria.; citation_author=D Shepherd; citation_author=PB Garland; citation_journal_title=Biochem J; citation_volume=114; citation_number=15; citation_pages=597-610; citation_date=1969; " />
      <meta name="citation_reference" content="citation_title=Citrate-condensing enzyme-oxalacetate binary complex. Studies on its physical and chemical properties.; citation_author=PA Srere; citation_journal_title=J Biol Chem; citation_volume=241; citation_number=16; citation_pages=2157-2165; citation_date=1966; " />
      <meta name="citation_reference" content="citation_title=Inhibition of citrate synthase by succinyl-CoA and other metabolites.; citation_author=CM Smith; citation_author=JR Williamson; citation_journal_title=FEBS Lett; citation_volume=18; citation_number=17; citation_pages=35-38; citation_date=1971; " />
      <meta name="citation_reference" content="citation_author=DA Beard; citation_author=H Qian; citation_number=18; citation_pages=69-104; citation_date=2008; citation_publisher=Cambridge University Press; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001825" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Detailed Enzyme Kinetics in Terms of Biochemical Species: Study of Citrate Synthase"/>
    <meta name="twitter:description" content="The compulsory-ordered ternary catalytic mechanism for two-substrate two-product enzymes is analyzed to account for binding of inhibitors to each of the four enzyme states and to maintain the relationship between the kinetic constants and the reaction equilibrium constant. The developed quasi-steady flux expression is applied to the analysis of data from citrate synthase to determine and parameterize a kinetic scheme in terms of biochemical species, in which the effects of pH, ionic strength, and cation binding to biochemical species are explicitly accounted for in the analysis of the data. This analysis provides a mechanistic model that is consistent with the data that have been used support competing hypotheses regarding the catalytic mechanism of this enzyme."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001825.g007"/>

  <meta property="og:title" content="Detailed Enzyme Kinetics in Terms of Biochemical Species: Study of Citrate Synthase" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001825" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3118'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=328'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4707&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001825">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001825" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001825&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Detailed Enzyme Kinetics in Terms of Biochemical Species: Study of Citrate Synthase
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel A. Beard
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:dbeard@mcw.edu">dbeard@mcw.edu</a></p>

                <p>Affiliation:
                  Biotechnology and Bioengineering Center and Department of Physiology, Medical College of Wisconsin, Milwaukee, Wisconsin, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kalyan C. Vinnakota, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology and Bioengineering Center and Department of Physiology, Medical College of Wisconsin, Milwaukee, Wisconsin, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fan Wu
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology and Bioengineering Center and Department of Physiology, Medical College of Wisconsin, Milwaukee, Wisconsin, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 19, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001825</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001825" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001825" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001825" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001825" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001825-g001" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001825-t001" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001825-g002" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001825-g003" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001825-g004" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001825-g005" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001825-t002" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001825-g006" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001825-g007" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001825">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001825" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The compulsory-ordered ternary catalytic mechanism for two-substrate two-product enzymes is analyzed to account for binding of inhibitors to each of the four enzyme states and to maintain the relationship between the kinetic constants and the reaction equilibrium constant. The developed quasi-steady flux expression is applied to the analysis of data from citrate synthase to determine and parameterize a kinetic scheme in terms of biochemical species, in which the effects of pH, ionic strength, and cation binding to biochemical species are explicitly accounted for in the analysis of the data. This analysis provides a mechanistic model that is consistent with the data that have been used support competing hypotheses regarding the catalytic mechanism of this enzyme.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Beard DA, Vinnakota KC, Wu F (2008) Detailed Enzyme Kinetics in Terms of Biochemical Species: Study of Citrate Synthase. PLoS ONE 3(3):
          e1825.
            doi:10.1371/journal.pone.0001825</p><p><strong>Editor: </strong>Timothy Secomb, University of Arizona, United States of America</p><p><strong>Received:</strong> November 29, 2007; <strong>Accepted:</strong> February 20, 2008; <strong>Published:</strong> March 19, 2008</p><p><strong>Copyright:</strong> © 2008 Beard et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>NIH grants GM068610 and HL072011</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>While the study of the catalytic kinetics of enzymes represents one of the most established and well documented fields in biochemical research, the impact of biochemical state (pH, ionic strength, temperature, and certain cation concentrations) is typically not formally accounted for in kinetic studies <a href="#pone.0001825-Alberty1">[1]</a>, <a href="#pone.0001825-Alberty2">[2]</a>. <em>In vitro</em> experiments using purified proteins and controlled substrate concentrations to characterize enzyme kinetics are conducted under conditions that do not necessarily match the physiological environment, but are determined based on a number of factors, including the requirements of the assays used to measure the kinetics. Therefore it is difficult to compare results obtained from different studies and to use available kinetic data to predict <em>in vivo</em> function without ambiguity.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The need for credible validated models (such as enzymatic rate laws and associated parameter values) for the individual components of a given biochemical system is apparent in developing simulations of cellular biochemical systems. For example, simulations of metabolic systems, such as the glycolytic pathway in yeast <a href="#pone.0001825-Teusink1">[3]</a>, skeletal muscle <a href="#pone.0001825-Vinnakota1">[4]</a>, <a href="#pone.0001825-Lambeth1">[5]</a>, and mammalian red blood cells <a href="#pone.0001825-Jamshidi1">[6]</a>, <a href="#pone.0001825-Mulquiney1">[7]</a>, are based on integrating the individual components together. Simulations of other cellular systems, such as signaling networks and membrane electrophysiology, are also based on kinetic models for mechanisms of relevant individual enzymes and transporter proteins. To apply these models to simulate and predict cellular behavior, they must not only match the available data but also properly account for biochemical state.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Outlining these and other issues in somewhat greater detail, the following specific challenges associated with interpreting <em>in vitro</em> kinetic data must be overcome to make optimal use of them.</p>


<ol class="order">

<li>While a great deal of high quality data may be available for a particular enzyme, much of these data were obtained in the 1960's and 1970's when tools for proper analysis of the data were not available. As a result, the reported kinetic parameter values (typically obtained from double reciprocal plots of inverse flux versus inverse substrate <a href="#pone.0001825-Lineweaver1">[8]</a>) may not optimally match the reported data.</li>

<li>Data on biochemical kinetics are typically obtained under nonphysiological pH and ionic conditions. Therefore the reported kinetic constants must be corrected to apply to simulations of physiological systems.</li>

<li>A third problem related to the second is that kinetic constants are associated with apparent mechanisms that operate on biochemical reactants, which are sums of biochemical species <a href="#pone.0001825-Alberty2">[2]</a>. The result is that the reported mechanisms and associated parameter values are dependent on biochemical state and not easily translated to apply to different biochemical states or to simulations in which the biochemical state changes.</li>

<li>The reported kinetic mechanisms and parameters are often not constrained to match the thermodynamic data for a given reaction. Since the basic thermodynamics of a given reaction is typically characterized with greater precision than the kinetics of an enzyme catalyzing the reaction, putative kinetic mechanisms should be constrained to match the biochemical reaction thermodynamics.</li>

</ol><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>We propose addressing and correcting these problems by posing reaction mechanisms in terms of species and ensuring that mechanisms properly account for thermodynamics. This basic approach was first introduced by Frieden and Alberty <a href="#pone.0001825-Frieden1">[9]</a>, yet has received little attention. Here, we reanalyze legacy data from a variety of sources of kinetic data on citrate synthase. Rather than estimating apparent Michaelis-Menten parameter values from slopes of double reciprocal plots, we use nonlinear curve fitting to simultaneously estimate parameter values from several sets of data from kinetic studies on specific isoforms of the enzyme. Through this analysis we are able to show that data used to support competing models of the mechanism for this enzyme are all consistent with the compulsory-order ternary-complex mechanism. In addition, certain conclusions drawn from the original studies are shown to be not consistent with the data presented in these studies.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>To perform this analysis on citrate synthase it is first necessary to derive the general rate law (quasi-steady flux expression) for the compulsory-order ternary-complex mechanism that can account for potential nonproductive binding of inhibitors at any of the four distinct enzyme states. Although the derivation of quasi-steady rate laws for multi-state catalytic mechanisms is a rich and established field, the general form for this mechanism with potential inhibition at any site, has not previously been presented. Therefore it is expected that this expression will be useful in the analysis of a number of other two-substrate two-product catalytic mechanisms.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods and Results"></a><h3>Methods and Results</h3>
<h4>Kinetic equations for compulsory-order ternary-complex enzyme mechanism</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The basic compulsory-order ternary-complex mechanism, also called the ordered bi-bi mechanism, is illustrated in <a href="#pone-0001825-g001">Figure 1</a> for the general reaction <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e001&amp;representation=PNG" class="inline-graphic"></span>. The mechanism involves four enzyme state transitions:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where each state transition is assumed to proceed by mass action <a href="#pone.0001825-Segel1">[10]</a>, <a href="#pone.0001825-CornishBowden1">[11]</a>. Here E<sub>1</sub> represents free (unbound) enzyme; E<sub>2</sub> represents the complex formed between enzyme and the species A, which binds first; E<sub>3</sub> is the ternary complex that represents enzyme bound to both substrates or both products; and E<sub>4</sub> represents the complex formed between enzyme and the species Q. In <a href="#pone-0001825-g001">Figure 1</a> the substrate and product concentrations are denoted <em>a</em> = [A], <em>b</em> = [B], <em>p</em> = [P], and <em>q</em> = [Q] and the reactant concentrations are incorporated into apparent mass-action rate constants for the state transitions between enzyme states 1, 2, 3, and 4.</p>
<div class="figure" id="pone-0001825-g001"><div class="img"><a name="pone-0001825-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Basic compulsory-order ternary-complex mechanism.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The basic ordered mechanism for the general reaction <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e028&amp;representation=PNG" class="inline-graphic"></span>, with <em>a</em> = [A], <em>b</em> = [B], <em>p</em> = [P], and <em>q</em> = [Q] is illustrated. The four states refer to unbound enzyme (state 1), enzyme-substrate A complex (E·A, state 2), enzyme-substrate A-substrate B complex (E·AB, state 3), and enzyme-product Q complex (E·Q, state 4). The four steps of the catalytic cycle are detailed in Equation (1).</p>
<span>doi:10.1371/journal.pone.0001825.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>From the four-state diagram of <a href="#pone-0001825-g001">Figure 1</a>, the expression for the steady-state flux through the reaction can be obtained from diagrammatic method of King and Altman <a href="#pone.0001825-King1">[12]</a>. The flux <em>J</em> may be expressed<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>where<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>and<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>The constant <em>K<sub>eq</sub></em> is the equilibrium constant for the reaction; <em>E<sub>o</sub></em> = <em>E<sub>1</sub></em>+<em>E<sub>2</sub></em>+<em>E<sub>3</sub></em>+<em>E<sub>4</sub></em> is the total enzyme concentration; and the <em>I<sub>i</sub></em> factors in Equation (4) account for nonproductive binding (inhibition) of inhibitors to each of the enzyme states. These inhibition factors are computed<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e006&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>where <em>K<sub>ij</sub></em> is the binding constant for nonproductive binding of species <em>j</em> to enzyme state <em>i</em> and <em>c<sub>j</sub></em> is the concentration of species <em>j</em>.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Defining<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e007&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br>and<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e008&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>the flux is <em>J</em> = <em>n</em>/<em>d</em>, where the numerator and denominator expressed in terms of kinetic constants are<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e009&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br>and<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e010&amp;representation=PNG" class="inline-graphic"><span class="note">(9)</span></span><br>Here the kinetic constants are defined<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(10)</span></span><br></p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Expressing the steady-state kinetics in terms of these parameters, only the <em>V<sub>m</sub></em> parameter, which has units of mass per unit time per unit volume, has units that include time. All other parameters have units of concentration (mass per unit volume). In addition, the eight concentration parameters cannot vary independently. For example we can compute <em>K<sub>eQ</sub></em> in terms of the other parameters if the equilibrium constant of the reaction is known:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(11)</span></span><br></p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>The novelty of the expression of Equation (9) for the denominator of the flux expression is that inhibitive binding at each enzyme state is considered. It is expected that a wide class catalytic mechanisms for two-substrate two-product reactions will conform to this general form.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>Here we apply this general form to the analysis of data from citrate synthase to determine kinetic parameters for several isoforms of this enzyme and to elucidate the mechanisms behind inhibition by products and other species not part of the overall chemical reaction.</p>


<h4>Model of citrate synthase</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Citrate synthase is the first step in the oxidation of acetyl-CoA in the citric acid cycle. The reference chemical reaction, is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(12)</span></span><br>where the abbreviations for the biochemical species are listed in <a href="#pone-0001825-t001">Table 1</a>. The biochemical reaction, involving biochemical reactants that are sums of species is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e014&amp;representation=PNG" class="inline-graphic"><span class="note">(13)</span></span><br></p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Note that we have introduced the convention that the charge of a species appears as a superscript, even when the charge is zero. This convention conveniently differentiates between, for example, the species ACCOA<sup>0</sup> and the reactant ACCOA.</p>
<div class="figure" id="pone-0001825-t001"><div class="img"><a name="pone-0001825-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001825.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001825.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Thermodynamic Parameter Values for Citrate Synthase (298.15 K, 1 M reactants, I = 0.17 M, P = 1 atm).</span></strong></p><span>doi:10.1371/journal.pone.0001825.t001</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The standard Gibbs free energy is computed <a href="#pone.0001825-Alberty3">[13]</a>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e015&amp;representation=PNG" class="inline-graphic"><span class="note">(14)</span></span><br>where the basic thermodynamic data are listed in <a href="#pone-0001825-t001">Table 1</a>. The equilibrium constant for reaction in Equation (12) is computed from the standard Gibbs free energy<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e016&amp;representation=PNG" class="inline-graphic"><span class="note">(15)</span></span><br>where we have introduced the definition <em>h</em> = 10<sup>−pH</sup> and this equilibrium constant explicitly accounts for pH. Therefore <em>K</em><sup>0</sup><em><sub>eq</sub></em><sub>,cits</sub> represents the equilibrium ratio of [COAS<sup>−</sup>][CIT<sup>3−</sup>]/[OAA<sup>2−</sup>][ACCOA<sup>0</sup>].The relationships between species and reactant concentrations depend on the pH and concentration of metal ions that reversibly bind to biochemical species. To compute species concentrations and the apparent equilibrium constant and Gibbs free energy, we introduce binding polynomials for reactants and other species that we consider in our model of citrate synthase kinetics:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e017&amp;representation=PNG" class="inline-graphic"><span class="note">(16)</span></span><br></p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>These polynomials include terms for H<sup>+</sup>-, Mg<sup>2+</sup>-, and K<sup>+</sup>-bound states <a href="#pone.0001825-Alberty3">[13]</a>. Note that only states that are expected to be significant in the pH and ionic range studied are included in these calculations. Therefore some binding polynomials do not include terms for all possible cation-bound states. Given these forms of the binding polynomials, the relationships between the reference species concentrations and the reactant concentrations are<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e018&amp;representation=PNG" class="inline-graphic"><span class="note">(17)</span></span><br></p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Under typical physiological conditions (pH = 7, [K<sup>+</sup>] = 120 mM, and [Mg<sup>2+</sup>] = 1 mM) the fraction of these reactants present as their unbound reference species may be smaller than 10%. For example, the molar fraction of ATP<sup>4−</sup>, HATP<sup>3−</sup>, MgATP<sup>2−</sup>, KATP<sup>3−</sup>, predicted based on the parameters in <a href="#pone-0001825-t001">Table 1</a> are approximately 0.085, 0.024, 0.32, and 0.57.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>The apparent equilibrium constant for the biochemical reaction is computed as a function of pH, [K<sup>+</sup>], and [Mg<sup>2+</sup>]<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e019&amp;representation=PNG" class="inline-graphic"><span class="note">(18)</span></span><br></p>
<a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>Citrate synthase is believe to operate by the compulsory-order ternary-complex mechanism, although investigations have lead to proposing more complex behavior, involving cooperativity and random order and dead-end binding of substrates <a href="#pone.0001825-Matsuoka1">[14]</a>. Here we postulate the standard compulsory-order ternary-complex mechanism derived above can explain the kinetic behavior of citrate synthase with substrate and products identified as: <em>a</em> = [OAA<sup>2−</sup>], <em>b</em> = [ACCOA<sup>0</sup>], <em>p</em> = [COAS<sup>−</sup>], <em>q</em> = [CIT<sup>3−</sup>]. The specific mechanism proposed is:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e020&amp;representation=PNG" class="inline-graphic"><span class="note">(19)</span></span><br></p>
<a id="article1.body1.sec2.sec2.p8" name="article1.body1.sec2.sec2.p8"></a><p>As an alternative mechanism, we could postulate that the second and third reactions have the form<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e021&amp;representation=PNG" class="inline-graphic"><span class="note">(20)</span></span><br>where both the second and third reactions generate protons. Based on the data analyzed we are not able to distinguish between these models. The analysis here applies to the mechanism of Equation (19). In this case only the third reaction (in which hydrogen ion explicitly appears) depends on pH. Since <em>K<sub>eP</sub></em>, the equilibrium constant for the third reaction, depends on pH while the others do not, we compute <em>K<sub>eP</sub></em>, as a function of the equilibrium constant for the reference reaction<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e022&amp;representation=PNG" class="inline-graphic"><span class="note">(21)</span></span><br>The rate constant <em>k</em><sub>43</sub> is assumed to depend on pH according to the formula <em>k</em><sub>43</sub> = (<em>h</em>/10<sup>−7</sup>)<sup>2</sup><em>k</em>′<sub>43</sub> where <em>k</em>′<sub>43</sub> is independent of pH. Therefore the kinetic constant <em>K<sub>mP</sub></em> is defined to depend on pH as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e023&amp;representation=PNG" class="inline-graphic"><span class="note">(22)</span></span><br>where <em>K</em>′<em><sub>mP</sub></em> is a kinetic constant that is independent of pH. In addition to the pH-dependency of the kinetic constants, the overall enzyme activity is assumed to depend on pH, with the numerator of the flux expression taking the form:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e024&amp;representation=PNG" class="inline-graphic"><span class="note">(23)</span></span><br>which is used to reproduce the pH dependency observed by Shepherd and Garland <a href="#pone.0001825-Shepherd1">[15]</a>. Equation (23) assumes that the enzyme is a monobasic acid, with dissociation constant <em>K<sub>iH</sub></em>.</p>
<a id="article1.body1.sec2.sec2.p9" name="article1.body1.sec2.sec2.p9"></a><p>Previous studies have revealed that a number of substances, including succinyl-coenzyme A and adenine nucleotides, act as inhibitors of citrate synthase. Our analysis of kinetic data on citrate synthase from rat liver and bovine heart (see below) revealed that ATP, ADP, and AMP inhibit the enzyme by forming unproductive complexes with enzyme state 2. Because we were able to obtain less data on SCOA inhibition than on adenine nucleotide inhibition, we were not able to elucidate the site of SCOA binding: models assuming binding at either state 1 or state 2 are equally well able to explain the observed data. Since the adenine nucleotide inhibition was determined to occur at enzyme state 2, here we parameterize the model assuming that SCOA binds to this complex as well. Based on this formulation of the model, the inhibition term <em>I<sub>2</sub></em> is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e025&amp;representation=PNG" class="inline-graphic"><span class="note">(24)</span></span><br>and inhibition at other complexes is not considered: <em>I<sub>1</sub></em> = <em>I<sub>3</sub></em> = <em>I<sub>4</sub></em> = 1.</p>


<h4>Analysis of data for the kidney enzyme from rat</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Matsuoka and Srere reported a comprehensive study on the forward and reverse kinetics of citrate synthase from rat kidney <a href="#pone.0001825-Matsuoka1">[14]</a> that is useful in identifying the kinetic parameters for this enzyme. Data used here are plotted in <a href="#pone-0001825-g002">Figures 2</a> and <a href="#pone-0001825-g003">3</a>.</p>
<div class="figure" id="pone-0001825-g002"><div class="img"><a name="pone-0001825-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Fits to kinetic data from <a href="#pone.0001825-Matsuoka1">[14]</a> on the forward operation of kidney enzyme.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Measured flux as a function of substrate concentrations was obtained from <a href="#pone-0001825-g002">Figures 2</a>, <a href="#pone-0001825-g003">3</a>, <a href="#pone-0001825-g006">6</a>, <a href="#pone-0001825-g007">7</a>, and 9 of <a href="#pone.0001825-Matsuoka1">[14]</a>. Initial fluxes (µmol of COASH (or CIT) synthesized per minute per µg of enzyme) measured at the substrate concentrations indicated in the figures. For A, B, and D, the initial product (CIT and COASH) concentrations are zero. C. Flux measured with COASH added in various concentrations to investigate the kinetics of product inhibition. All data were obtained at pH = 8.1 at 28°C. Model fits are plotted as solid lines.</p>
<span>doi:10.1371/journal.pone.0001825.g002</span></div><div class="figure" id="pone-0001825-g003"><div class="img"><a name="pone-0001825-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Fits to kinetic data from <a href="#pone.0001825-Matsuoka1">[14]</a> on the reverse operation of kidney enzyme.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Measured reverse flux as a function of concentrations of CIT and COASH was obtained from <a href="#pone-0001825-g004">Figures 4</a> and <a href="#pone-0001825-g005">5</a> of <a href="#pone.0001825-Matsuoka1">[14]</a>. Initial fluxes (µmol of COASH (or CIT) synthesized per minute per µg of enzyme) measured at the substrate concentrations indicated in the figures. All data were obtained at pH = 8.1 at 28°C. Model fits are plotted as solid lines.</p>
<span>doi:10.1371/journal.pone.0001825.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p><a href="#pone-0001825-g002">Figure 2</a> plots data on the forward reaction flux as functions of the concentrations of substrates OAA and ACCOA, corresponding to data from <a href="#pone-0001825-g002">Figures 2</a>, <a href="#pone-0001825-g003">3</a>, <a href="#pone-0001825-g006">6</a>, <a href="#pone-0001825-g007">7</a>, and 9 of Matsuoka and Srere <a href="#pone.0001825-Matsuoka1">[14]</a>. <a href="#pone-0001825-g002">Figure 2A</a> plots flux in units of µmol of COASH (or CIT) synthesized per minute per µg of enzyme as a function of [ACCOA] at different concentrations of [OAA] while <a href="#pone-0001825-g002">Figure 2B</a> plots flux versus [OAA] at different concentrations of [ACCOA], as indicated in the figure. Product inhibition by COASH is illustrated in <a href="#pone-0001825-g002">Figure 2C</a>, which plots flux versus [ACCOA] at [OAA] = 0.5 mM and concentrations of [COASH] ranging from 0 to 100 µM. The data plotted in <a href="#pone-0001825-g002">Figure 2D</a> are analogous to that of <a href="#pone-0001825-g002">Figure 2A</a>, with the difference that the range of substrate concentrations in <a href="#pone-0001825-g002">Figure 2D</a> are significantly lower than in <a href="#pone-0001825-g002">Figure 2A</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p><a href="#pone-0001825-g006">Figures 6</a> and <a href="#pone-0001825-g007">7</a> Matsuoka and Srere <a href="#pone.0001825-Matsuoka1">[14]</a> report data obtained for low concentrations of ACCOA, while <a href="#pone-0001825-g002">Figures 2</a> and <a href="#pone-0001825-g003">3</a> report data at the higher concentration range. However, where the concentration ranges intersect, the reported fluxes (in units of nmole/min) are approximately five times higher in their <a href="#pone-0001825-g002">Figures 2</a> and <a href="#pone-0001825-g003">3</a> than in their <a href="#pone-0001825-g006">Figures 6</a> and <a href="#pone-0001825-g007">7</a>. Since the assays in their <a href="#pone-0001825-g002">Figures 2</a> and <a href="#pone-0001825-g003">3</a> were carried out in a 1 ml cuvette and the assays in their <a href="#pone-0001825-g006">Figures 6</a> and <a href="#pone-0001825-g007">7</a> were carried out in a 5 ml cuvette, we have deduced that the data in <a href="#pone-0001825-g006">Figures 6</a> and <a href="#pone-0001825-g007">7</a> were normalized to the cuvette size. This finding is verified based on their reported estimated <em>V<sub>max</sub></em> values, which are consistent with this scaling. Therefore, we have scaled the data of <a href="#pone-0001825-g006">Figures 6</a> and <a href="#pone-0001825-g007">7</a> of Matsuoka and Srere <a href="#pone.0001825-Matsuoka1">[14]</a> (replotted here in panel 2D) by a factor of five compared to the other figures.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Data on the reverse flux of the enzyme are plotted in <a href="#pone-0001825-g003">Figure 3</a>. <a href="#pone-0001825-g003">Figures 3A and 3B</a> correspond to <a href="#pone-0001825-g004">Figures 4</a> and <a href="#pone-0001825-g005">5</a>, respectively, of Matsuoka and Srere <a href="#pone.0001825-Matsuoka1">[14]</a>, where reverse flux is reported over a range of [COASH] and [CIT] values.</p>
<div class="figure" id="pone-0001825-g004"><div class="img"><a name="pone-0001825-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Fits to kinetic data from <a href="#pone.0001825-Shepherd1">[15]</a> on the forward operation of liver enzyme.</span></strong></p><a id="article1.body1.sec2.sec3.fig3.caption1.p1" name="article1.body1.sec2.sec3.fig3.caption1.p1"></a><p>Measured flux in arbitrary units was obtained from <a href="#pone-0001825-g001">Figures 1</a>,<a href="#pone-0001825-g002">2</a>,<a href="#pone-0001825-g005">5</a>, and <a href="#pone-0001825-g006">6</a> of <a href="#pone.0001825-Shepherd1">[15]</a>. For all cases the product (CIT and COASH) concentrations are zero and total substrate and inhibitor concentrations are indicated in the figure. A and B report data obtained with no inhibitors present. C. The relative activity (normalized to its maximum) of the enzyme is plotted as functions of [ATP], [ADP], and [AMP] measured at [ACCOA] = 11 µM and [OAA] = 1.9 µM. D. The measured flux is plotted as a function of [ACCOA] at [OAA] = 34 µM with ATP, ADP, and AMP present as indicated in the figure. All data were obtained at pH = 7.4 at 25°C. Model fits are plotted as solid lines.</p>
<span>doi:10.1371/journal.pone.0001825.g004</span></div><div class="figure" id="pone-0001825-g005"><div class="img"><a name="pone-0001825-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Impact of [Mg<sup>2+</sup>] and pH on liver enzyme.</span></strong></p><a id="article1.body1.sec2.sec3.fig4.caption1.p1" name="article1.body1.sec2.sec3.fig4.caption1.p1"></a><p>Measured flux in arbitrary units was obtained from Figures 13 and 14 of <a href="#pone.0001825-Shepherd1">[15]</a>. A. The relative activity (normalized to its maximum) of the enzyme is plotted as functions of [ATP] at [Mg<sup>2+</sup>] = 0 mM (shaded circles), 0.5 mM (shaded triangles), 1.0 mM (shaded squares), 2.0 mM (open circles), and 4.0 mM (diamonds). B. Relative activity is plotted as a function of pH. Substrate concentations are [ACCOA] = 21 µM and [OAA] = 8.6 µM. All data were obtained at 25°C. pH is fixed a 7.4 for A. Model fits are plotted as solid lines.</p>
<span>doi:10.1371/journal.pone.0001825.g005</span></div><a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>All experiments for the data of <a href="#pone-0001825-g002">Figures 2</a> and <a href="#pone-0001825-g003">3</a> were conducted at pH = 8.1 at 28°C. We assume and overall ionic strength of 0.17 M, and [K<sup>+</sup>] = 100 mM, and [Mg<sup>2+</sup>] = 0. The complete set of data from <a href="#pone-0001825-g002">Figures 2</a> and <a href="#pone-0001825-g003">3</a> were used to estimate parameter values for the kidney enzyme. Specifically, we use these data to estimate <em>V<sub>m</sub></em>, <em>K<sub>mA</sub></em>, <em>K<sub>mB</sub></em>, <em>K</em>′<em><sub>mP</sub></em>, <em>K<sub>mQ</sub></em>, <em>K<sub>eA</sub></em>, <em>K<sub>eB</sub></em>, and <em>K<sub>eQ</sub></em>. The parameter values associated with the best fits to the data are listed in <a href="#pone-0001825-t002">Table 2</a> and the corresponding model predictions are plotted as solid lines in the figures. The agreement between the data and the model is satisfactory, with values of the eight estimated parameters sensitive to the observed data. Estimated sensitivity coefficients (defined below) are given in parenthesis for each parameter value in <a href="#pone-0001825-t002">Table 2</a>.</p>
<div class="figure" id="pone-0001825-t002"><div class="img"><a name="pone-0001825-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001825.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001825.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Kinetic Parameter Values for Citrate Synthase.</span></strong></p><span>doi:10.1371/journal.pone.0001825.t002</span></div><a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>Based on their data, Matsuoka and Srere report estimates for the kinetic parameters, some of which can be compared to those estimated here. It is not surprising that the estimates of <em>V<sub>m</sub></em> (estimated at high ACCOA concentrations by Matsuoka and Srere to be 0.316 µmol·min<sup>−1</sup>·µg<sup>−1</sup> and estimated here to be 0.336 µmol·min<sup>−1</sup>·µg<sup>−1</sup>) are similar. In addition, Matsuoka and Srere's reported estimates kinetic parameter values for the “medium” ACCOA concentration range (5–50 µM) are <em>K<sub>mA</sub></em> = 5 µM, <em>K<sub>mB</sub></em> = 4.5 µM, <em>K<sub>mP</sub></em> = 39 µM, <em>K<sub>mQ</sub></em> = 3 mM, <em>K<sub>eA</sub></em> = 5 µM, and <em>K<sub>eQ</sub></em> = 4.3 mM. These values can be directly compared to our estimates of <em>K<sub>mA</sub></em> = 8.227 µM, <em>K<sub>mB</sub></em> = 7.402 µM, <em>K<sub>mP</sub></em> = 24.72 µM (at pH 8.1), <em>K<sub>mQ</sub></em> = 4.548 mM, <em>K<sub>eA</sub></em> = 0.8879 µM, and <em>K<sub>eQ</sub></em> = 3.618 mM. (The parameters <em>K<sub>eB</sub></em> and <em>K<sub>eP</sub></em> are not directly comparable to those estimated by Matsuoka and Srere because they are defined differently in the two studies.)</p>
<a id="article1.body1.sec2.sec3.p7" name="article1.body1.sec2.sec3.p7"></a><p>Parameter estimates from our study correspond to species, while those from Matsuoka and Srere correspond to reactants; thus the applied models are not exactly identical. More significantly, we performed our model fits by simultaneously matching all of the data in <a href="#pone-0001825-g002">Figures 2</a> and <a href="#pone-0001825-g003">3</a> to the model using a single set of parameter estimates rather than estimating kinetic constants from the slopes of double reciprocal plots. The result is that we are able to explain the data based on a single mechanism operating at all observed concentration ranges, while Matsuoka and Srere reported different estimates of apparent <em>V<sub>m</sub></em> and other kinetic constants operating at low (&lt;5 µM), medium (5–50 µM) and high (&gt;50 µM) concentrations of ACCOA.</p>
<a id="article1.body1.sec2.sec3.p8" name="article1.body1.sec2.sec3.p8"></a><p>Matsuoka and Srere speculated that their findings may be explained by the existence of cooperativity in ACCOA binding or by a random binding mechanism. Yet while we are able to explain the high-ACCOA data (Figure 2 of <a href="#pone.0001825-Matsuoka1">[14]</a>) and the low-ACCOA data (Figure 7 of <a href="#pone.0001825-Matsuoka1">[14]</a>), our model is not able to reproduce the data presented in Figure 8 of <a href="#pone.0001825-Matsuoka1">[14]</a>, which shows a discontinuity in the slope of the double reciprocal plot of <em>V</em><sup>−1</sup> versus [ACCOA]<sup>−1</sup>. In fact, we are at a loss to explain Figure 8 of Matsuoka and Srere because the concentration ranges explored in their Figures 2 and 7 overlap and extend beyond the range explored in their Figure 8. The data of their Figure 2 and 7, when plotted together, all fall on the same straight line. Therefore it is possible that an incorrect scaling was applied to a portion of the data presented in <a href="#pone.0001825-Matsuoka1">[14]</a>. In any case, it is apparent that there exists an inconsistency in the reported data. Since to our knowledge no other study reproduced the slope discontinuity reported in <a href="#pone.0001825-Matsuoka1">[14]</a>, we chose to leave this data set out of our analysis.</p>
<a id="article1.body1.sec2.sec3.p9" name="article1.body1.sec2.sec3.p9"></a><p>The largest disagreement between our estimated kinetic constants and those estimated by Matsuoka and Srere for the medium ACCOA concentration range is in the estimate of <em>K<sub>eA</sub></em>, the dissociation constant for OAA. Yet our estimate of 0.8879 µM is in agreement with an independent study by Srere <a href="#pone.0001825-Srere1">[16]</a> that estimated <em>K<sub>eA</sub></em> to be 0.6 µM for citrate synthase obtained from pig heart, providing further validation of our proposed model.</p>


<h4>Analysis of data for the liver enzyme from rat</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>A detailed kinetic study on citrate synthase obtained from rat liver <a href="#pone.0001825-Shepherd1">[15]</a> provides an opportunity to check the basic model developed above based on and independent data set and to assess the impact of inhibition of the enzyme by ATP, ADP, and AMP. While we do not expect the kinetic constants to be the same for the liver isoform as for the kidney enzyme studied above, we hypothesize that the basic kinetic mechanism is the same for both isoforms.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p><a href="#pone-0001825-g004">Figures 4</a> and <a href="#pone-0001825-g005">5</a> plot data obtained from <a href="#pone-0001825-g001">Figures 1</a>, <a href="#pone-0001825-g002">2</a>, <a href="#pone-0001825-g005">5</a>, <a href="#pone-0001825-g006">6</a>, 13, and 14 of Shepherd and Garland <a href="#pone.0001825-Shepherd1">[15]</a>, all corresponding to the forward reaction flux with no product present in the assays. Experiments were conducted at pH = 7.4 (except those plotted in <a href="#pone-0001825-g005">Figure 5B</a>) at 25°C. Again, we assume and overall ionic strength of 0.17 M, and [K<sup>+</sup>] = 100 mM, and [Mg<sup>2+</sup>] = 0. The experiments of <a href="#pone-0001825-g004">Figures 4A and 4B</a> are analogous to those of <a href="#pone-0001825-g002">Figures 2A and 2B</a>, in which the substrate concentrations are varied to obtain estimates of <em>K<sub>mA</sub></em>, <em>K<sub>mB</sub></em>, and <em>K<sub>eA</sub></em>. (Since the total mass of enzyme used in these experiments is not specified, it is not possible to estimate <em>V<sub>m</sub></em> from these data.)</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Experimental data plotted in <a href="#pone-0001825-g004">Figure 4C and 4D</a> provide information on inhibition of the enzyme due to binding of ATP, ADP, and AMP. <a href="#pone-0001825-g004">Figure 4C</a> plots the inhibition (as relative activity normalized to its maximum) as functions of [ATP], [ADP], and [AMP] measured at [ACCOA] = 11 µM and [OAA] = 1.9 µM. It is clear that ATP is the strongest and AMP the weakest inhibitor. In <a href="#pone-0001825-g004">Figure 4D</a> the measured flux is plotted as a function of [ACCOA] at [OAA] = 34 µM with ATP, ADP, and AMP present as indicated in the figure.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p><a href="#pone-0001825-g005">Figure 5</a> illustrates the affects of [Mg<sup>2+</sup>] and pH on inhibition by ATP, ADP, and AMP, and on overall catalytic activity. In <a href="#pone-0001825-g005">Figure 5A</a> the ATP-inhibition curve is plotted at different levels of free Mg<sup>2+</sup> ion concentration. Mg<sup>2+</sup> is shown to diminish the inhibition effect, suggesting that the species Mg·ATP<sup>2−</sup> does not bind as significantly as ATP<sup>4−</sup> and supporting our assumption that the free unbound species of ATP, ADP, and AMP are the important actors in competitive inhibition of the enzyme. <a href="#pone-0001825-g005">Figure 5B</a> plots the overall catalytic activity as a function of pH at fixed substrate concentrations, as indicated in the figure legend.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>The data of <a href="#pone-0001825-g004">Figures 4A, 4B, 4C, 4D</a>, and <a href="#pone-0001825-g005">5B</a>, were used to provide estimates of the kinetic parameters <em>K<sub>mA</sub></em>, <em>K<sub>mB</sub></em>, and <em>K<sub>eA</sub></em> and the inhibition parameters <em>K<sub>iATP</sub></em>, <em>K<sub>iADP</sub></em>, <em>K<sub>iAMP</sub></em>, and <em>K<sub>iH</sub></em> for the liver enzyme. These estimates are listed in <a href="#pone-0001825-t002">Table 2</a>. The solid lines plotted in <a href="#pone-0001825-g004">Figures 4</a> and <a href="#pone-0001825-g005">5</a> represent model predictions corresponding to these data sets. The data of <a href="#pone-0001825-g005">Figure 5A</a> were not used to estimate model parameters; thus the model predictions in <a href="#pone-0001825-g005">Figure 5A</a> represent validation of the overall model based on the prediction of how relative activity increases with [Mg<sup>2+</sup>] in the presence of ATP. Note that Shepherd and Garland reported a maximum in the plot of <a href="#pone-0001825-g005">Figure 5A</a> (their Figure 14) near [ATP] = 0.5 mM for [Mg<sup>2+</sup>] = 2 mM that is not captured by our model. (The [Mg<sup>2+</sup>] = 2 mM data are plotted as open circles in <a href="#pone-0001825-g005">Figure 5A</a>.) However, this apparent maxima is based on a single experimental data point; Smith and Williamson <a href="#pone.0001825-Smith1">[17]</a> report that they were unable to reproduce this observation.</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>Based on these data from Shepherd and Garland, we are able to determine that the likely site of ATP, ADP, and AMP binding is enzyme state 2—the complex E·OAA<sup>−2</sup> in the proposed scheme of Equation (19). Alternative models, with nonproductive binding at states 1, 3, or 4, are not able to explain the observed data nearly as well as the model with inhibition at state 2. However, our model with a single inhibition site did not reproduce the data of <a href="#pone-0001825-g007">Figure 7</a> of <a href="#pone.0001825-Shepherd1">[15]</a>, which indicate that AMP may also act as a competitive inhibitor against OAA, while ATP and ADP do not. It may be possible to explain the AMP data from <a href="#pone-0001825-g007">Figure 7</a> of <a href="#pone.0001825-Shepherd1">[15]</a> with a model that includes binding of AMP at both states 1 and 2. However, doing this would require increasing the complexity of the model based on a handful of unreplicated data points from a single study. The current model, in not including binding of ATP, ADP, or AMP to free enzyme (state 1), explains the consensus of the available data.</p>


<h4>Analysis of data for the heart enzyme from cow</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The study of Smith and Williamson <a href="#pone.0001825-Smith1">[17]</a> allows us to test the validity compulsory-order ternary-complex mechanism model for data from citrate synthase obtained from bovine heart. <a href="#pone-0001825-g006">Figure 6</a> plots data obtained from <a href="#pone-0001825-g001">Figures 1</a> and <a href="#pone-0001825-g002">2</a> of <a href="#pone.0001825-Smith1">[17]</a>, which allow us to estimate five parameters of the model for this isoform. Specifically, <a href="#pone-0001825-g006">Figure 6A</a> plots relative activity as a function of [ATP] at two different concentrations of OAA, as specific in the figure, and at fixed [ACCOA] = 16 µM. <a href="#pone-0001825-g006">Figure 6B</a> plots flux measured as a function of [ACCOA] at with different concentrations of the inhibitor ATP present and at fixed [OAA] = 5 µM. Finally, <a href="#pone-0001825-g006">Figure 6C</a> plots measured flux as function of [ACCOA] with [OAA] fixed at 3.1 µM and the inhibitor [SCOA] added to different concentrations as indicated in the figure. Experiments for this study were conducted at pH = 7.4 and at 21°C. As for the previous studies, we assume and overall ionic strength of 0.17 M, and [K<sup>+</sup>] = 100 mM, and [Mg<sup>2+</sup>] = 0.</p>
<div class="figure" id="pone-0001825-g006"><div class="img"><a name="pone-0001825-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Inhibition of cardiac enzyme.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Measured flux in arbitrary units was obtained from <a href="#pone-0001825-g001">Figures 1</a> and <a href="#pone-0001825-g002">2</a> of <a href="#pone.0001825-Smith1">[17]</a>. A. Flux is plotted as a function inhibitor ATP concentration for [ACCOA] = 16 µM and [OAA] = 1.13 and 2.25 µM. B. Flux is plotted as a function of [ACCOA] at [OAA] = 5 µM at three different concentrations of ATP indicated in figure. C. Flux is plotted as a function of [ACCOA] at [OAA] = 3.1 µM at three different concentrations of SCOA indicated in figure. All data were obtained at pH = 7.4 at 21°C. Model fits are plotted as solid lines.</p>
<span>doi:10.1371/journal.pone.0001825.g006</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Taken together, the data of <a href="#pone-0001825-g006">Figure 6</a> provide the means to estimate kinetic parameters <em>K<sub>mA</sub></em>, <em>K<sub>mB</sub></em>, and <em>K<sub>eA</sub></em> and the inhibition parameters <em>K<sub>iATP</sub></em> and <em>K<sub>iSCOA</sub></em> for the isoform of this enzyme from bovine heart. The solid lines plotted in the Figure correspond to the parameter estimates listed in <a href="#pone-0001825-t002">Table 2</a>.</p>


<h4>Sensitivity analysis</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To estimate the sensitivity of the model prediction to finite changes in parameter values, the sensitivity was computed as the relative change in mean squared error due to a 10% change in a given parameter value. For each parameter estimate a sensitivity coefficient is defined as follows,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e026&amp;representation=PNG" class="inline-graphic"><span class="note">(25)</span></span><br>where <em>E</em><sup>*</sup> represents the minimum mean squared difference between model predictions and experimental data, and <em>x<sub>i</sub></em> is the optimal value of the <em>i</em>th parameter. The term <em>E</em><sup>*</sup>(<em>x<sub>i</sub></em>±0.1<em>x<sub>i</sub></em>) is the error computed from setting parameter <em>x<sub>i</sub></em> to 10% above and below its optimal value and reoptimizing all of the remaining parameter estimates. The sensitivity coefficients are listed in <a href="#pone-0001825-t002">Table 2</a> in parenthesis following each parameter estimate. When the sensitivity coefficient is high, the data used to estimate that parameter value are able to provide a sensitive estimate of that parameter value. For the twenty parameter estimates reported in <a href="#pone-0001825-t002">Table 2</a>, three are associated with sensitivity coefficients of less than 1% and three others have sensitivity coefficients in the range 1–5%. Thus not all kinetic parameters are identified with high sensitivity.</p>


<h4>Alternative model</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Previous analyses of the data used in this study have suggested a rapid-equilibrium random-order ternary-complex mechanism for citrate synthase <a href="#pone.0001825-Matsuoka1">[14]</a>, <a href="#pone.0001825-Shepherd1">[15]</a>. To provide an alternative to our proposed model and determine if the observed kinetics may be explained by this mechanism, we fit the data of Matsuoka and Srere <a href="#pone.0001825-Matsuoka1">[14]</a> to the flux expression for the rapid-equilibrium random-order ternary-complex mechanism, given below <a href="#pone.0001825-CornishBowden1">[11]</a>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.e027&amp;representation=PNG" class="inline-graphic"><span class="note">(26)</span></span><br>Here we analyzed the Matsuoka and Srere data to avoid the complicating effects of inhibitors and because this data set is rich enough to exclude this model as a competing hypothesis.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p><a href="#pone-0001825-g007">Figure 7</a> replots the data of <a href="#pone-0001825-g002">Figure 2</a> and <a href="#pone-0001825-g003">3</a> of the present study and illustrates two sets of fits using Equation (26), obtained by varying the seven adjustable parameters in this model. Parameter values are reported in the figure legend. The first fit (solid lines) is obtained as the best fit to the data in panels 7A through 7D on the forward operation of the enzyme. It is apparent that the forward flux data can be matched closely by the alternative model, at the expense of not being able to reproduce the reverse flux data of panel 7E and 7F. When the model is optimized to match both the forward and reverse data, the dashed-line model fits are obtained. In this case, none of the forward flux data are effectively matched. In particular, for the concentration range in <a href="#pone-0001825-g007">Figure 7C</a>, the dashed-line model fit shows no product inhibition by COASH, while the other panels at least reproduce the qualitative trends in the data. Since there is no parameter set that can reasonable match all of these data, this alternative model can be ruled out as inadequate.</p>
<div class="figure" id="pone-0001825-g007"><div class="img"><a name="pone-0001825-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001825" data-uri="info:doi/10.1371/journal.pone.0001825.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001825.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001825.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001825.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001825.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Analysis using random-order model of Equation (26).</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Data and conditions in A, B, C, and D are the same as for <a href="#pone-0001825-g002">Figure 2</a>. Data and conditions for E and F are the same as for <a href="#pone-0001825-g003">Figures 3A and 3B</a>, respectively Parameter values for solid line model predictions are <em>V<sub>max</sub></em> = 0.320 µmol·min<sup>−1</sup>· µg<sup>−1</sup>, <em>K<sub>mB</sub></em> = 6.20 µM, <em>K<sub>mP</sub></em> = 8.00 µM, <em>K<sub>eA</sub></em> = 1.35 µM, <em>K<sub>eB</sub></em> = 1.10 µM, <em>K<sub>eP</sub></em> = 21.6 nM, <em>K<sub>eQ</sub></em> = 0.150 µM. Parameter values for dashed line model predictions are <em>V<sub>max</sub></em> = 0.526 µmol·min<sup>−1</sup>·µg<sup>−1</sup>, <em>K<sub>mB</sub></em> = 36.6 µM, <em>K<sub>mP</sub></em> = 80.792 mM, <em>K<sub>eA</sub></em> = 3.08 nM, <em>K<sub>eB</sub></em> = 10.8 nM, <em>K<sub>eP</sub></em> = 0.152 µM, <em>K<sub>eQ</sub></em> = 17.0 µM.</p>
<span>doi:10.1371/journal.pone.0001825.g007</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Here we have introduced a model for the compulsory order ternary-complex catalytic mechanism that is formulated in terms of chemical species, allowing the model to account for variable state, including pH and metal ion concentrations. In addition, a general form of the model, with potential inhibitory binding at each enzyme state is introduced. The model is used to analyze independent data sets from a number of labs on different isoforms of citrate synthase and develop a consensus mechanism that explains the available data. This consensus mechanism provides a detailed understanding of the basic mechanism of this enzyme and can be useful in computational simulation of biochemical systems including this enzyme.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>More generally, the basic model developed may serve as template for other two-substrate two-product reaction mechanisms. Given the depth and breath of the field of enzyme kinetics, it is surprising that the basic flux expression for the ordered ternary-complex mechanism with inhibition at all possible enzyme complexes has not been previously presented. Yet to our knowledge the flux expression introduced here has not been previously reported. Based on this general form, it is possible to systematically test models with inhibition at one or more potential sites and determine which model or models are consistent with the available data. In the case of citrate synthase examined here, the majority of data on inhibition by ATP, ADP, and AMP are best explained by nonproductive binding of these species to state 2 in the catalytic mechanism of <a href="#pone-0001825-g001">Figure 1</a>. Based on data at different pH values and with different concentrations of [Mg<sup>2+</sup>] in the media, we conclude that the unbound species of these reactants (ATP<sup>4−</sup>, ADP<sup>3−</sup>, AMP<sup>2−</sup>) are the species that bind to the enzyme complex.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The data analyzed here are not sensitive to distinguish the site of inhibition by SCOA. Models with binding to either enzyme state 1 or state 2 are equally able to explain the data. The parameters and model fits reported here correspond to the inhibition model of Equation (24) with binding to state 2.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Note that analysis of the data sets from isoforms from rat kidney, rat liver, and bovine heart does not provide a complete set of parameters for any of these isoforms. The data from the rat kidney isoform provide a detailed characterization of the basic kinetic parameters for this isoform, but without information on the inhibitors. The studies for the other isoforms did not probe the reverse reaction, therefore provide no information on kinetic parameters that appear only in the terms involving concentrations of products P and Q in the Equation (9). From cases where estimates of a given parameter are available for different isoforms (e.g., <em>K<sub>mA</sub></em>, <em>K<sub>mB</sub></em>, <em>K<sub>eA</sub></em>) it is apparent that the estimates vary between the different species and/or tissue types from which the enzyme was obtained. Therefore, from these data a complete model parameterization for a specific species and tissue type is not possible.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>It should also be noted that while we have ruled out the rapid-equilibrium random-order ternary-complex mechanism as an alternative model to explain observed data, our analysis does not rule out all possible alternatives. Rather, the model serves as a hypothesis that can explain, and is not disproved by, the observed data. While the data do not allow us to distinguish between certain alternative forms of the model, such as between the mechanisms of Equations (19) and (20), it is clear that widely reported mechanism for this enzyme is not able to explain the observed kinetics. Here the general equations for compulsory-order ternary-complex mechanism, with generalized inhibitor binding to any state in the catalytic cycle, are developed and shown to be consistent with the available data. This model will be useful in integrated modeling of biochemical systems involving this enzyme and in analyzing kinetic data from other enzymes thought to follow this general mechanism.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank Athel Cornish-Bowden for advice and feedback on the manuscript.</p>
<a id="article1.back1.ack1.p2" name="article1.back1.ack1.p2"></a><p>Note on Prior Publication:</p>
<a id="article1.back1.ack1.p3" name="article1.back1.ack1.p3"></a><p>The model and kinetic analysis reported here was first published, in abbreviated form, as an example in a textbook <a href="#pone.0001825-Beard1">[18]</a>.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DB KV FW. Analyzed the data: DB. Wrote the paper: DB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001825-Alberty1" id="pone.0001825-Alberty1"></a>Alberty RA (2006) Relations between biochemical thermodynamics and biochemical kinetics. Biophys Chem  124: 11–17.  <ul class="find" data-citedArticleID="1077006" data-doi="10.1016/j.bpc.2006.05.024"><li><a href="http://dx.doi.org/10.1016/j.bpc.2006.05.024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relations+between+biochemical+thermodynamics+and+biochemical+kinetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relations+between+biochemical+thermodynamics+and+biochemical+kinetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001825-Alberty2" id="pone.0001825-Alberty2"></a>Alberty RA (1993) Levels of thermodynamic treatment of biochemical reaction systems. Biophys J  65: 1243–1254.  <ul class="find" data-citedArticleID="1077008" data-doi="10.1016/s0006-3495(93)81146-9"><li><a href="http://dx.doi.org/10.1016/s0006-3495(93)81146-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Levels+of+thermodynamic+treatment+of+biochemical+reaction+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Levels+of+thermodynamic+treatment+of+biochemical+reaction+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001825-Teusink1" id="pone.0001825-Teusink1"></a>Teusink B, Passarge J, Reijenga CA, Esgalhado E, van der Weijden CC, et al.  (2000) Can yeast glycolysis be understood in terms of in vitro kinetics of the constituent enzymes? Testing biochemistry. Eur J Biochem  267: 5313–5329.  <ul class="find" data-citedArticleID="1077038" data-doi="10.1046/j.1432-1327.2000.01527.x"><li><a href="http://dx.doi.org/10.1046/j.1432-1327.2000.01527.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Can+yeast+glycolysis+be+understood+in+terms+of+in+vitro+kinetics+of+the+constituent+enzymes%3F+Testing+biochemistry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Can+yeast+glycolysis+be+understood+in+terms+of+in+vitro+kinetics+of+the+constituent+enzymes%3F+Testing+biochemistry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001825-Vinnakota1" id="pone.0001825-Vinnakota1"></a>Vinnakota K, Kemp ML, Kushmerick MJ (2006) Dynamics of muscle glycogenolysis modeled with pH time course computation and pH-dependent reaction equilibria and enzyme kinetics. Biophys J  91: 1264–1287.  <ul class="find" data-citedArticleID="1077040" data-doi="10.1529/biophysj.105.073296"><li><a href="http://dx.doi.org/10.1529/biophysj.105.073296" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamics+of+muscle+glycogenolysis+modeled+with+pH+time+course+computation+and+pH-dependent+reaction+equilibria+and+enzyme+kinetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamics+of+muscle+glycogenolysis+modeled+with+pH+time+course+computation+and+pH-dependent+reaction+equilibria+and+enzyme+kinetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001825-Lambeth1" id="pone.0001825-Lambeth1"></a>Lambeth MJ, Kushmerick MJ (2002) A computational model for glycogenolysis in skeletal muscle. Ann Biomed Eng  30: 808–827.  <ul class="find" data-citedArticleID="1077022" data-doi="10.1114/1.1492813"><li><a href="http://dx.doi.org/10.1114/1.1492813" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+computational+model+for+glycogenolysis+in+skeletal+muscle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+computational+model+for+glycogenolysis+in+skeletal+muscle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001825-Jamshidi1" id="pone.0001825-Jamshidi1"></a>Jamshidi N, Edwards JS, Fahland T, Church GM, Palsson BO (2001) Dynamic simulation of the human red blood cell metabolic network. Bioinformatics  17: 286–287.  <ul class="find" data-citedArticleID="1077018" data-doi="10.1093/bioinformatics/17.3.286"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.3.286" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+simulation+of+the+human+red+blood+cell+metabolic+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+simulation+of+the+human+red+blood+cell+metabolic+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001825-Mulquiney1" id="pone.0001825-Mulquiney1"></a>Mulquiney PJ, Kuchel PW (1999) Model of 2,3-bisphosphoglycerate metabolism in the human erythrocyte based on detailed enzyme kinetic equations: equations and parameter refinement. Biochem J  342 Pt 3: 581–596.  <ul class="find" data-citedArticleID="1077028" data-doi="10.1042/0264-6021:3420581"><li><a href="http://dx.doi.org/10.1042/0264-6021:3420581" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Model+of+2%2C3-bisphosphoglycerate+metabolism+in+the+human+erythrocyte+based+on+detailed+enzyme+kinetic+equations%3A+equations+and+parameter+refinement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Model+of+2%2C3-bisphosphoglycerate+metabolism+in+the+human+erythrocyte+based+on+detailed+enzyme+kinetic+equations%3A+equations+and+parameter+refinement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001825-Lineweaver1" id="pone.0001825-Lineweaver1"></a>Lineweaver H, Burk D (1934) The determination of enzyme dissocation constants. J Am Chem Soc  56: 658–666.  <ul class="find" data-citedArticleID="1077024" data-doi="10.1021/ja01318a036"><li><a href="http://dx.doi.org/10.1021/ja01318a036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+determination+of+enzyme+dissocation+constants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+determination+of+enzyme+dissocation+constants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001825-Frieden1" id="pone.0001825-Frieden1"></a>Frieden C, Alberty RA (1955) The effect of pH on fumarase activity in acetate buffer. J Biol Chem  212: 859–868.  <ul class="find" data-citedArticleID="1077016"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+effect+of+pH+on+fumarase+activity+in+acetate+buffer.&amp;auth=&amp;atitle=The+effect+of+pH+on+fumarase+activity+in+acetate+buffer." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+pH+on+fumarase+activity+in+acetate+buffer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+pH+on+fumarase+activity+in+acetate+buffer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001825-Segel1" id="pone.0001825-Segel1"></a>Segel IH (1975) Enzyme kinetics : behavior and analysis of rapid equilibrium and steady-state enzyme systems. New York: Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0001825-CornishBowden1" id="pone.0001825-CornishBowden1"></a>Cornish-Bowden A (2004) Fundamentals of enzyme kinetics. London: Portland Press.   <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0001825-King1" id="pone.0001825-King1"></a>King EL, Altman C (1956) A schematic method of deriving the rate laws for enzyme-catalyzed reactions. J Phys Chem  60: 1375–1378.  <ul class="find" data-citedArticleID="1077020" data-doi="10.1021/j150544a010"><li><a href="http://dx.doi.org/10.1021/j150544a010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+schematic+method+of+deriving+the+rate+laws+for+enzyme-catalyzed+reactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+schematic+method+of+deriving+the+rate+laws+for+enzyme-catalyzed+reactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001825-Alberty3" id="pone.0001825-Alberty3"></a>Alberty RA (2003) Thermodynamics of Biochemical Reactions. Hoboken, N.J.: Wiley-Interscience.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001825-Matsuoka1" id="pone.0001825-Matsuoka1"></a>Matsuoka Y, Srere PA (1973) Kinetic studies of citrate synthase from rat kidney and rat brain. J Biol Chem  248: 8022–8030.  <ul class="find" data-citedArticleID="1077026"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Kinetic+studies+of+citrate+synthase+from+rat+kidney+and+rat+brain.&amp;auth=&amp;atitle=Kinetic+studies+of+citrate+synthase+from+rat+kidney+and+rat+brain." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kinetic+studies+of+citrate+synthase+from+rat+kidney+and+rat+brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kinetic+studies+of+citrate+synthase+from+rat+kidney+and+rat+brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001825-Shepherd1" id="pone.0001825-Shepherd1"></a>Shepherd D, Garland PB (1969) The kinetic properties of citrate synthase from rat liver mitochondria. Biochem J  114: 597–610.  <ul class="find" data-citedArticleID="1077032"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+kinetic+properties+of+citrate+synthase+from+rat+liver+mitochondria.&amp;auth=&amp;atitle=The+kinetic+properties+of+citrate+synthase+from+rat+liver+mitochondria." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+kinetic+properties+of+citrate+synthase+from+rat+liver+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+kinetic+properties+of+citrate+synthase+from+rat+liver+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001825-Srere1" id="pone.0001825-Srere1"></a>Srere PA (1966) Citrate-condensing enzyme-oxalacetate binary complex. Studies on its physical and chemical properties. J Biol Chem  241: 2157–2165.  <ul class="find" data-citedArticleID="1077036"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Citrate-condensing+enzyme-oxalacetate+binary+complex.+Studies+on+its+physical+and+chemical+properties.&amp;auth=&amp;atitle=Citrate-condensing+enzyme-oxalacetate+binary+complex.+Studies+on+its+physical+and+chemical+properties." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Citrate-condensing+enzyme-oxalacetate+binary+complex.+Studies+on+its+physical+and+chemical+properties." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Citrate-condensing+enzyme-oxalacetate+binary+complex.+Studies+on+its+physical+and+chemical+properties.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001825-Smith1" id="pone.0001825-Smith1"></a>Smith CM, Williamson JR (1971) Inhibition of citrate synthase by succinyl-CoA and other metabolites. FEBS Lett  18: 35–38.  <ul class="find" data-citedArticleID="1077034" data-doi="10.1016/0014-5793(71)80400-3"><li><a href="http://dx.doi.org/10.1016/0014-5793(71)80400-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+citrate+synthase+by+succinyl-CoA+and+other+metabolites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+citrate+synthase+by+succinyl-CoA+and+other+metabolites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001825-Beard1" id="pone.0001825-Beard1"></a>Beard DA, Qian H (2008) Enyzme-catalyzed reactions: cycles, transients, and non-equilibrium steady state. Chemical Biophysics: Quantitative Analysis of Cellular Processes. Cambridge, UK: Cambridge University Press. pp. 69–104.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.XML" value="84054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.PDF" value="557732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e001.PNG" value="6857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e001.TIF" value="22724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e002.PNG" value="15604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e002.TIF" value="31088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e028.PNG" value="6569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e028.TIF" value="22476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g001.PNG_L" value="29197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g001.PNG_M" value="32747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g001.PNG_S" value="8624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g001.TIF" value="56822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g001.PNG_I" value="12168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e003.PNG" value="6129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e003.TIF" value="22080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e004.PNG" value="9653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e004.TIF" value="25560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e005.PNG" value="27929"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e005.TIF" value="45584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e006.PNG" value="7834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e006.TIF" value="23768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e007.PNG" value="8978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e007.TIF" value="24836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e008.PNG" value="9119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e008.TIF" value="24968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e009.PNG" value="9899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e009.TIF" value="25460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e010.PNG" value="40820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e010.TIF" value="55380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e011.PNG" value="29098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e011.TIF" value="44512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e012.PNG" value="8785"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e012.TIF" value="24532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e013.PNG" value="10004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e013.TIF" value="26172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e014.PNG" value="9115"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e014.TIF" value="25252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.t001.PNG_L" value="131364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.t001.PNG_M" value="74384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.t001.PNG_S" value="11364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.t001.TIF" value="746304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.t001.PNG_I" value="23093"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e015.PNG" value="13044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e015.TIF" value="29208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e016.PNG" value="10907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e016.TIF" value="26428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e017.PNG" value="42963"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e017.TIF" value="58128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e018.PNG" value="35724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e018.TIF" value="53216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e019.PNG" value="11161"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e019.TIF" value="26648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e020.PNG" value="24624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e020.TIF" value="40760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e021.PNG" value="15475"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e021.TIF" value="31912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e022.PNG" value="9655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e022.TIF" value="25288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e023.PNG" value="9343"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e023.TIF" value="25000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e024.PNG" value="12632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e024.TIF" value="27904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e025.PNG" value="13550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e025.TIF" value="29964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g002.PNG_L" value="442476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g002.PNG_M" value="58234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g002.PNG_S" value="11027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g002.TIF" value="865972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g002.PNG_I" value="19436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g003.PNG_L" value="217925"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g003.PNG_M" value="31761"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g003.PNG_S" value="8275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g003.TIF" value="433288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g003.PNG_I" value="10587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g004.PNG_L" value="437814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g004.PNG_M" value="58291"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g004.PNG_S" value="11183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g004.TIF" value="847080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g004.PNG_I" value="19652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g005.PNG_L" value="181634"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g005.PNG_M" value="25831"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g005.PNG_S" value="7610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g005.TIF" value="374442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g005.PNG_I" value="8666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.t002.PNG_L" value="106919"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.t002.PNG_M" value="138451"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.t002.PNG_S" value="18476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.t002.TIF" value="514652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.t002.PNG_I" value="46001"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g006.PNG_L" value="289182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g006.PNG_M" value="39013"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g006.PNG_S" value="8997"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g006.TIF" value="669578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g006.PNG_I" value="12741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e026.PNG" value="10375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e026.TIF" value="26192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e027.PNG" value="14270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.e027.TIF" value="29820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g007.PNG_L" value="656066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g007.PNG_M" value="74134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g007.PNG_S" value="12560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g007.TIF" value="1277410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001825.g007.PNG_I" value="30522"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001825&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001825" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001825&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001825&volume=&issue=&title=Detailed Enzyme Kinetics in Terms of Biochemical Species: Study of Citrate Synthase&author_name=Daniel%20A.%20Beard%2C%20Kalyan%20C.%20Vinnakota%2C%20Fan%20Wu&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001825" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001825" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001825" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001825&amp;t=Detailed%20Enzyme%20Kinetics%20in%20Terms%20of%20Biochemical%20Species%3A%20Study%20of%20Citrate%20Synthase" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001825&title=Detailed%20Enzyme%20Kinetics%20in%20Terms%20of%20Biochemical%20Species%3A%20Study%20of%20Citrate%20Synthase&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001825&amp;title=Detailed%20Enzyme%20Kinetics%20in%20Terms%20of%20Biochemical%20Species%3A%20Study%20of%20Citrate%20Synthase" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001825" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001825&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Detailed Enzyme Kinetics in Terms of Biochemical Species: Study of Citrate Synthase';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001825';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Detailed%20Enzyme%20Kinetics%20in%20Terms%20of%20Biochemical%20Species%3A%20Study%20of%20Citrate%20Synthase http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001825" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001825" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biochemical+simulations%22" title="Search for articles in the subject area:'Biochemical simulations'"><div class="flagText">Biochemical simulations</div></a>
              <div data-categoryid="17939" data-articleid="26474"
                   data-categoryname="Biochemical simulations"
                   class="flagImage" title="Flag 'Biochemical simulations' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chemical+equilibrium%22" title="Search for articles in the subject area:'Chemical equilibrium'"><div class="flagText">Chemical equilibrium</div></a>
              <div data-categoryid="21113" data-articleid="26474"
                   data-categoryname="Chemical equilibrium"
                   class="flagImage" title="Flag 'Chemical equilibrium' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzyme+inhibitors%22" title="Search for articles in the subject area:'Enzyme inhibitors'"><div class="flagText">Enzyme inhibitors</div></a>
              <div data-categoryid="22575" data-articleid="26474"
                   data-categoryname="Enzyme inhibitors"
                   class="flagImage" title="Flag 'Enzyme inhibitors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzyme+kinetics%22" title="Search for articles in the subject area:'Enzyme kinetics'"><div class="flagText">Enzyme kinetics</div></a>
              <div data-categoryid="27783" data-articleid="26474"
                   data-categoryname="Enzyme kinetics"
                   class="flagImage" title="Flag 'Enzyme kinetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzyme+kinetics+characterization%22" title="Search for articles in the subject area:'Enzyme kinetics characterization'"><div class="flagText">Enzyme kinetics characterization</div></a>
              <div data-categoryid="31329" data-articleid="26474"
                   data-categoryname="Enzyme kinetics characterization"
                   class="flagImage" title="Flag 'Enzyme kinetics characterization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzymes%22" title="Search for articles in the subject area:'Enzymes'"><div class="flagText">Enzymes</div></a>
              <div data-categoryid="17863" data-articleid="26474"
                   data-categoryname="Enzymes"
                   class="flagImage" title="Flag 'Enzymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Kidneys%22" title="Search for articles in the subject area:'Kidneys'"><div class="flagText">Kidneys</div></a>
              <div data-categoryid="47351" data-articleid="26474"
                   data-categoryname="Kidneys"
                   class="flagImage" title="Flag 'Kidneys' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reactants%22" title="Search for articles in the subject area:'Reactants'"><div class="flagText">Reactants</div></a>
              <div data-categoryid="21823" data-articleid="26474"
                   data-categoryname="Reactants"
                   class="flagImage" title="Flag 'Reactants' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=535'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8994'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4045&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=16043">UPDATE</a><br>Posted by dbeard</p>
      <p><a href="/annotation/listThread.action?root=209">&quot;will alternate models (i.e. parameters) fit the data equally well&quot;</a><br>Posted by ehpardo</p>
      <p><a href="/annotation/listThread.action?root=2665">Referee Comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
