

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Network Analysis of the Human T-Cell Activation Gene Network Identifies Jagged1 as a Therapeutic Target for Autoimmune Diseases</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001222"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001222" />

    <meta name="citation_title" content="A Network Analysis of the Human T-Cell Activation Gene Network Identifies Jagged1 as a Therapeutic Target for Autoimmune Diseases"/>
    <meta itemprop="name" content="A Network Analysis of the Human T-Cell Activation Gene Network Identifies Jagged1 as a Therapeutic Target for Autoimmune Diseases"/>

      <meta name="citation_author" content="Ricardo Palacios"/>
            <meta name="citation_author_institution" content="Department of Neurology, Clinica Universitaria de Navarra and Center for Applied Medical Research, University of Navarra, Pamplona, Spain"/>
      <meta name="citation_author" content="Joaquin Goni"/>
            <meta name="citation_author_institution" content="Department of Neurology, Clinica Universitaria de Navarra and Center for Applied Medical Research, University of Navarra, Pamplona, Spain"/>
            <meta name="citation_author_institution" content="Department of Physics and Applied Mathematics, University of Navarra, Pamplona, Spain"/>
      <meta name="citation_author" content="Ivan Martinez-Forero"/>
            <meta name="citation_author_institution" content="Department of Neurology, Clinica Universitaria de Navarra and Center for Applied Medical Research, University of Navarra, Pamplona, Spain"/>
            <meta name="citation_author_institution" content="Department of Physics and Applied Mathematics, University of Navarra, Pamplona, Spain"/>
      <meta name="citation_author" content="Jaime Iranzo"/>
            <meta name="citation_author_institution" content="Department of Neurology, Clinica Universitaria de Navarra and Center for Applied Medical Research, University of Navarra, Pamplona, Spain"/>
      <meta name="citation_author" content="Jorge Sepulcre"/>
            <meta name="citation_author_institution" content="Department of Neurology, Clinica Universitaria de Navarra and Center for Applied Medical Research, University of Navarra, Pamplona, Spain"/>
      <meta name="citation_author" content="Ignacio Melero"/>
            <meta name="citation_author_institution" content="Hepathology and Gene Therapy, Center for Applied Medical Research, University of Navarra, Pamplona, Spain"/>
      <meta name="citation_author" content="Pablo Villoslada"/>
            <meta name="citation_author_institution" content="Department of Neurology, Clinica Universitaria de Navarra and Center for Applied Medical Research, University of Navarra, Pamplona, Spain"/>

    <meta name="citation_date" content="2007/11/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001222.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1222"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Network biology: understanding the cell's functional organization.; citation_author=AL Barabasi; citation_author=ZN Oltvai; citation_journal_title=Nat Rev Genet; citation_volume=5(2); citation_number=1; citation_pages=101-113; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Understanding the generation and function of memory T cell subsets.; citation_author=A Lanzavecchia; citation_author=F Sallusto; citation_journal_title=Curr Opin Immunol; citation_volume=17(3); citation_number=2; citation_pages=326-332; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Contingent genetic regulatory events in T lymphocyte activation.; citation_author=GR Crabtree; citation_journal_title=Science; citation_volume=243(4889); citation_number=3; citation_pages=355-361; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Cytokines: promoters and dampeners of autoimmunity.; citation_author=N Hill; citation_author=N Sarvetnick; citation_journal_title=Curr Opin Immunol; citation_volume=14(6); citation_number=4; citation_pages=791-797; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Transcriptional analysis of targets in multiple sclerosis.; citation_author=L Steinman; citation_author=S Zamvil; citation_journal_title=Nat Rev Immunol; citation_volume=3(6); citation_number=5; citation_pages=483-492; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Immunology of multiple sclerosis.; citation_author=M Sospedra; citation_author=R Martin; citation_journal_title=Annu Rev Immunol; citation_volume=23; citation_number=6; citation_pages=683-747; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The art of the probable: system control in the adaptive immune system.; citation_author=RN Germain; citation_journal_title=Science; citation_volume=293(5528); citation_number=7; citation_pages=240-245; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Inverse correlation between the incidences of autoimmune disease and infection predicted by a model of T cell mediated tolerance.; citation_author=K Leon; citation_author=J Faro; citation_author=A Lage; citation_author=J Carneiro; citation_journal_title=J Autoimmun; citation_volume=22(1); citation_number=8; citation_pages=31-42; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cellular and genetic mechanisms of self tolerance and autoimmunity.; citation_author=CC Goodnow; citation_author=J Sprent; citation_author=B Fazekas de St Groth; citation_author=CG Vinuesa; citation_journal_title=Nature; citation_volume=435(7042); citation_number=9; citation_pages=590-597; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=On the role of feedback in promoting conflicting goals of the adaptive immune system.; citation_author=LA Segel; citation_author=RL Bar-Or; citation_journal_title=J Immunol; citation_volume=163(3); citation_number=10; citation_pages=1342-1349; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The many sounds of T lymphocyte silence.; citation_author=I Melero; citation_author=A Arina; citation_author=L Chen; citation_journal_title=Immunol Res; citation_volume=33(2); citation_number=11; citation_pages=135-147; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Metabolic Syndrome and Robustness Tradeoffs.; citation_author=H Kitano; citation_author=O K; citation_author=K T; citation_author=M Y; citation_author=M Csete; citation_journal_title=Diabetes; citation_volume=53(S3); citation_number=12; citation_pages=S6-S15; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A robustness-based approach to systems-oriented drug design.; citation_author=H Kitano; citation_journal_title=Nat Rev Drug Discov; citation_volume=6(3); citation_number=13; citation_pages=202-210; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The neurobiology of multiple sclerosis: genes, inflammation, and neurodegeneration.; citation_author=SL Hauser; citation_author=JR Oksenberg; citation_journal_title=Neuron; citation_volume=52(1); citation_number=14; citation_pages=61-76; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Lethality and centrality in protein networks.; citation_author=H Jeong; citation_author=SP Mason; citation_author=AL Barabasi; citation_author=ZN Oltvai; citation_journal_title=Nature; citation_volume=411(6833); citation_number=15; citation_pages=41-42; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Overview of computational methods for the inference of gene regulatory networks.; citation_author=SG Styczynski MP; citation_journal_title=Computers & Chemical Engineering; citation_volume=29; citation_number=16; citation_pages=519-534; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Causal protein-signaling networks derived from multiparameter single-cell data.; citation_author=K Sachs; citation_author=O Perez; citation_author=D Pe'er; citation_author=DA Lauffenburger; citation_author=GP Nolan; citation_journal_title=Science; citation_volume=308(5721); citation_number=17; citation_pages=523-529; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=How to infer gene networks from expression profiles.; citation_author=M Bansal; citation_author=V Belcastro; citation_author=A Ambesi-Impiombato; citation_author=D di Bernardo; citation_journal_title=Mol Syst Biol; citation_volume=3; citation_number=18; citation_pages=78; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease.; citation_author=J Lamb; citation_author=ED Crawford; citation_author=D Peck; citation_author=JW Modell; citation_author=IC Blat; citation_journal_title=Science; citation_volume=313(5795); citation_number=19; citation_pages=1929-1935; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Linking the genes: inferring quantitative gene networks from microarray data.; citation_author=A de la Fuente; citation_author=P Brazhnik; citation_author=P Mendes; citation_journal_title=Trends Genet; citation_volume=18(8); citation_number=20; citation_pages=395-398; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Inferring dynamic architecture of cellular networks using time series of gene expression, protein and metabolite data.; citation_author=E Sontag; citation_author=A Kiyatkin; citation_author=BN Kholodenko; citation_journal_title=Bioinformatics; citation_volume=20(12); citation_number=21; citation_pages=1877-1886; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Generalized concept of minimal cut sets in biochemical networks.; citation_author=S Klamt; citation_journal_title=Biosystems; citation_volume=83(2–3); citation_number=22; citation_pages=233-247; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MS treatment: Postmarketing studies.; citation_author=X Montalban; citation_journal_title=J Neurol Sci; citation_number=23; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cytokines in autoimmune disorders.; citation_author=M Feldmann; citation_author=FM Brennan; citation_author=R Maini; citation_journal_title=Int Rev Immunol; citation_volume=17(1–4); citation_number=24; citation_pages=217-228; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Targeting cytokines of the interleukin-12 family in autoimmunity.; citation_author=BY Kang; citation_author=TS Kim; citation_journal_title=Curr Med Chem; citation_volume=13(10); citation_number=25; citation_pages=1149-1156; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecules involved in T-B co-stimulation and B cell homeostasis: possible targets for an immunological intervention in autoimmunity.; citation_author=HH Peter; citation_author=K Warnatz; citation_journal_title=Expert Opin Biol Ther; citation_volume=5; citation_number=26; citation_pages=S61-71; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The role of interleukin 10 in the control of autoimmunity.; citation_author=MG Roncarolo; citation_author=M Battaglia; citation_author=S Gregori; citation_journal_title=J Autoimmun; citation_volume=20(4); citation_number=27; citation_pages=269-272; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The immunopharmacological properties of transforming growth factor beta.; citation_author=Y Le; citation_author=X Yu; citation_author=L Ruan; citation_author=O Wang; citation_author=D Qi; citation_journal_title=Int Immunopharmacol; citation_volume=5(13–14); citation_number=28; citation_pages=1771-1782; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Anti-alpha4 integrin therapy for multiple sclerosis: mechanisms and rationale.; citation_author=GP Rice; citation_author=HP Hartung; citation_author=PA Calabresi; citation_journal_title=Neurology; citation_volume=64(8); citation_number=29; citation_pages=1336-1342; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Linkage disequilibrium screening for multiple sclerosis implicates JAG1 and POU2AF1 as susceptibility genes in Europeans.; citation_journal_title=J Neuroimmunol; citation_volume=23; citation_number=30; citation_pages=23; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Notch signalling during peripheral T-cell activation and differentiation.; citation_author=BA Osborne; citation_author=LM Minter; citation_journal_title=Nat Rev Immunol; citation_volume=7(1); citation_number=31; citation_pages=64-75; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Notch ligands Delta-like1, Delta-like4 and Jagged1 differentially regulate activation of peripheral T helper cells.; citation_author=S Rutz; citation_author=B Mordmuller; citation_author=S Sakano; citation_author=A Scheffold; citation_journal_title=Eur J Immunol; citation_volume=35(8); citation_number=32; citation_pages=2443-2451; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The human homolog of rat Jagged1 expressed by marrow stroma inhibits differentiation of 32D cells through interaction with Notch1.; citation_author=L Li; citation_author=LA Milner; citation_author=Y Deng; citation_author=M Iwata; citation_author=A Banta; citation_journal_title=Immunity; citation_volume=8(1); citation_number=33; citation_pages=43-55; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=From molecular to modular cell biology.; citation_author=LH Hartwell; citation_author=JJ Hopfield; citation_author=S Leibler; citation_author=AW Murray; citation_journal_title=Nature; citation_volume=402; citation_number=34; citation_pages=C47-52; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The human disease network.; citation_author=KI Goh; citation_author=ME Cusick; citation_author=D Valle; citation_author=B Childs; citation_author=M Vidal; citation_journal_title=Proc Natl Acad Sci U S A; citation_number=35; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Computational systems biology.; citation_author=H Kitano; citation_journal_title=Nature; citation_volume=420(6912); citation_number=36; citation_pages=206-210; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Inferring cellular networks using probabilistic graphical models.; citation_author=N Friedman; citation_journal_title=Science; citation_volume=303(5659); citation_number=37; citation_pages=799-805; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Human regulatory T cells and their role in autoimmune disease.; citation_author=C Baecher-Allan; citation_author=DA Hafler; citation_journal_title=Immunol Rev; citation_volume=212; citation_number=38; citation_pages=203-216; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Instruction of distinct CD4 T helper cell fates by different notch ligands on antigen-presenting cells.; citation_author=D Amsen; citation_author=JM Blander; citation_author=GR Lee; citation_author=K Tanigaki; citation_author=T Honjo; citation_journal_title=Cell; citation_volume=117(4); citation_number=39; citation_pages=515-526; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cloning of murine Stat6 and human Stat6, Stat proteins that are tyrosine phosphorylated in responses to IL-4 and IL-3 but are not required for mitogenesis.; citation_author=FW Quelle; citation_author=K Shimoda; citation_author=W Thierfelder; citation_author=C Fischer; citation_author=A Kim; citation_journal_title=Mol Cell Biol; citation_volume=15(6); citation_number=40; citation_pages=3336-3343; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=IPC: professional type 1 interferon-producing cells and plasmacytoid dendritic cell precursors.; citation_author=YJ Liu; citation_journal_title=Annu Rev Immunol; citation_volume=23; citation_number=41; citation_pages=275-306; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Activation of human CD4+ cells with CD3 and CD46 induces a T-regulatory cell 1 phenotype.; citation_author=C Kemper; citation_author=AC Chan; citation_author=JM Green; citation_author=KA Brett; citation_author=KM Murphy; citation_journal_title=Nature; citation_volume=421(6921); citation_number=42; citation_pages=388-392; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=CD45-induced tumor necrosis factor alpha production in monocytes is phosphatidylinositol 3-kinase-dependent and nuclear factor-kappaB-independent.; citation_author=AL Hayes; citation_author=C Smith; citation_author=BM Foxwell; citation_author=FM Brennan; citation_journal_title=J Biol Chem; citation_volume=274(47); citation_number=43; citation_pages=33455-33461; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Molecular mechanism of TNF signaling and beyond.; citation_author=ZG Liu; citation_journal_title=Cell Res; citation_volume=15(1); citation_number=44; citation_pages=24-27; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulatory mechanisms and their therapeutic implications of interleukin-12 production in immune cells.; citation_author=BY Kang; citation_author=E Kim; citation_author=TS Kim; citation_journal_title=Cell Signal; citation_volume=17(6); citation_number=45; citation_pages=665-673; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Autoimmune concepts of multiple sclerosis as a basis for selective immunotherapy: from pipe dreams to (therapeutic) pipelines.; citation_author=R Hohlfeld; citation_author=H Wekerle; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=46; citation_pages=14599-14606; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Chemokine network in multiple sclerosis: role in pathogenesis and targeting for future treatments.; citation_author=D Galimberti; citation_author=N Bresolin; citation_author=E Scarpini; citation_journal_title=Expert Rev Neurother; citation_volume=4(3); citation_number=47; citation_pages=439-453; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Immunomodulation in multiple sclerosis: from immunosuppression to neuroprotection.; citation_author=O Neuhaus; citation_author=JJ Archelos; citation_author=HP Hartung; citation_journal_title=Trends Pharmacol Sci; citation_volume=24(3); citation_number=48; citation_pages=131-138; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Small interfering RNA-mediated knockdown of notch ligands in primary CD4+ T cells and dendritic cells enhances cytokine production.; citation_author=Y Stallwood; citation_author=E Briend; citation_author=KM Ray; citation_author=GA Ward; citation_author=BJ Smith; citation_journal_title=J Immunol; citation_volume=177(2); citation_number=49; citation_pages=885-895; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Multiple sclerosis: re-expression of a developmental pathway that restricts oligodendrocyte maturation.; citation_author=GR John; citation_author=SL Shankar; citation_author=B Shafit-Zagardo; citation_author=A Massimi; citation_author=SC Lee; citation_journal_title=Nat Med; citation_volume=8(10); citation_number=50; citation_pages=1115-1121; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Diagnostic criteria for multiple sclerosis: 2005 revisions to the “McDonald Criteria”.; citation_author=CH Polman; citation_author=SC Reingold; citation_author=G Edan; citation_author=M Filippi; citation_author=HP Hartung; citation_journal_title=Ann Neurol; citation_volume=58(6); citation_number=51; citation_pages=840-846; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes.; citation_author=J Vandesompele; citation_author=K De Preter; citation_author=F Pattyn; citation_author=B Poppe; citation_author=N Van Roy; citation_journal_title=Genome Biol; citation_volume=3(7); citation_number=52; citation_pages=RESEARCH0034; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Processing of gene expression data generated by quantitative real-time RT-PCR.; citation_author=PY Muller; citation_author=H Janovjak; citation_author=AR Miserez; citation_author=Z Dobbie; citation_journal_title=Biotechniques; citation_volume=32(6); citation_number=53; citation_pages=1372-1374, 1376, 1378–1379; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Methylthioadenosine reverses brain autoimmune disease.; citation_author=B Moreno; citation_author=H Hevia; citation_author=M Santamaria; citation_author=J Sepulcre; citation_author=J Munoz; citation_journal_title=AnnNeurol; citation_volume=60; citation_number=54; citation_pages=323-334; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Tabu Search: Kluwer Academic Publishers; citation_author=F Glover; citation_author=M Laguna; citation_number=55; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=On information and sufficiency.; citation_author=S Kullback; citation_author=RA Leibler; citation_journal_title=Annals of Mathematical Statics; citation_volume=22; citation_number=56; citation_pages=79-86; citation_date=1951; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001222" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Network Analysis of the Human T-Cell Activation Gene Network Identifies Jagged1 as a Therapeutic Target for Autoimmune Diseases"/>
    <meta name="twitter:description" content="Understanding complex diseases will benefit the recognition of the properties of the gene networks that control biological functions. Here, we set out to model the gene network that controls T-cell activation in humans, which is critical for the development of autoimmune diseases such as Multiple Sclerosis (MS). The network was established on the basis of the quantitative expression from 104 individuals of 20 genes of the immune system, as well as on biological information from the Ingenuity database and Bayesian inference. Of the 31 links (gene interactions) identified in the network, 18 were identified in the Ingenuity database and 13 were new and we validated 7 of 8 interactions experimentally. In the MS patients network, we found an increase in the weight of gene interactions related to Th1 function and a decrease in those related to Treg and Th2 function. Indeed, we found that IFN-&szlig; therapy induces changes in gene interactions related to T cell proliferation and adhesion, although these gene interactions were not restored to levels similar to controls. Finally, we identify JAG1 as a new therapeutic target whose differential behaviour in the MS network was not modified by immunomodulatory therapy. In vitro treatment with a Jagged1 agonist peptide modulated the T-cell activation network in PBMCs from patients with MS. Moreover, treatment of mice with experimental autoimmune encephalomyelitis with the Jagged1 agonist ameliorated the disease course, and modulated Th2, Th1 and Treg function. This study illustrates how network analysis can predict therapeutic targets for immune intervention and identified the immunomodulatory properties of Jagged1 making it a new therapeutic target for MS and other autoimmune diseases."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001222.g005"/>

  <meta property="og:title" content="A Network Analysis of the Human T-Cell Activation Gene Network Identifies Jagged1 as a Therapeutic Target for Autoimmune Diseases" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001222" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6728'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2761'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=628&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001222">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001222" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001222&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Network Analysis of the Human T-Cell Activation Gene Network Identifies Jagged1 as a Therapeutic Target for Autoimmune Diseases
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ricardo Palacios, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Neurology, Clinica Universitaria de Navarra and Center for Applied Medical Research, University of Navarra, Pamplona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joaquin Goni, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Neurology, Clinica Universitaria de Navarra and Center for Applied Medical Research, University of Navarra, Pamplona, Spain, 
                  Department of Physics and Applied Mathematics, University of Navarra, Pamplona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ivan Martinez-Forero, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Neurology, Clinica Universitaria de Navarra and Center for Applied Medical Research, University of Navarra, Pamplona, Spain, 
                  Department of Physics and Applied Mathematics, University of Navarra, Pamplona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jaime Iranzo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Neurology, Clinica Universitaria de Navarra and Center for Applied Medical Research, University of Navarra, Pamplona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jorge Sepulcre, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Neurology, Clinica Universitaria de Navarra and Center for Applied Medical Research, University of Navarra, Pamplona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ignacio Melero, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Hepathology and Gene Therapy, Center for Applied Medical Research, University of Navarra, Pamplona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pablo Villoslada
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pvilloslada@unav.es">pvilloslada@unav.es</a></p>

                <p>Affiliation:
                  Department of Neurology, Clinica Universitaria de Navarra and Center for Applied Medical Research, University of Navarra, Pamplona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001222</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001222" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001222" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001222" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001222" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001222-g001" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001222-t001" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001222-t002" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001222-t003" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001222-g002" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001222-t004" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001222-g003" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001222-t005" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001222-g004" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001222-t006" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001222-g005" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001222">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001222" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Understanding complex diseases will benefit the recognition of the properties of the gene networks that control biological functions. Here, we set out to model the gene network that controls T-cell activation in humans, which is critical for the development of autoimmune diseases such as Multiple Sclerosis (MS). The network was established on the basis of the quantitative expression from 104 individuals of 20 genes of the immune system, as well as on biological information from the Ingenuity database and Bayesian inference. Of the 31 links (gene interactions) identified in the network, 18 were identified in the Ingenuity database and 13 were new and we validated 7 of 8 interactions experimentally. In the MS patients network, we found an increase in the weight of gene interactions related to Th1 function and a decrease in those related to Treg and Th2 function. Indeed, we found that IFN-ß therapy induces changes in gene interactions related to T cell proliferation and adhesion, although these gene interactions were not restored to levels similar to controls. Finally, we identify JAG1 as a new therapeutic target whose differential behaviour in the MS network was not modified by immunomodulatory therapy. In vitro treatment with a Jagged1 agonist peptide modulated the T-cell activation network in PBMCs from patients with MS. Moreover, treatment of mice with experimental autoimmune encephalomyelitis with the Jagged1 agonist ameliorated the disease course, and modulated Th2, Th1 and Treg function. This study illustrates how network analysis can predict therapeutic targets for immune intervention and identified the immunomodulatory properties of Jagged1 making it a new therapeutic target for MS and other autoimmune diseases.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Palacios R, Goni J, Martinez-Forero I, Iranzo J, Sepulcre J, et al.  (2007) A Network Analysis of the Human T-Cell Activation Gene Network Identifies Jagged1 as a Therapeutic Target for Autoimmune Diseases. PLoS ONE 2(11):
          e1222.
            doi:10.1371/journal.pone.0001222</p><p><strong>Academic Editor: </strong>Jacques Zimmer, Centre de Recherche Public-Santé, Luxembourg</p><p><strong>Received:</strong> August 21, 2007; <strong>Accepted:</strong> October 30, 2007; <strong>Published:</strong> November 21, 2007</p><p><strong>Copyright:</strong> © 2007 Palacios et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants from the Spanish Ministry of Health (FIS PI051201, FIS PI060117) and from the Fundacion Uriach to PV.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Understanding the structure and dynamics of biological networks may prove critical to unravel complex traits and diseases, such as autoimmune diseases <a href="#pone.0001222-Barabasi1">[1]</a>. In the immune response, T cells interact with antigen-presenting cells in a complex process that generates changes in gene expression. These changes underlie cell differentiation, and effector and regulatory events, as well as promoting the acquisition of a panel of adhesion molecules that guide cells to the appropriate tissues <a href="#pone.0001222-Lanzavecchia1">[2]</a>, <a href="#pone.0001222-Crabtree1">[3]</a>. Several evidences indicates gene deregulation within the immune system in autoimmune diseases <a href="#pone.0001222-Hill1">[4]</a>, <a href="#pone.0001222-Steinman1">[5]</a>, such as in Multiple Sclerosis (MS) <a href="#pone.0001222-Sospedra1">[6]</a>. Several studies suggest that T-cell activation and the ensuing differentiation to effector cells or is one of the most critical process in controlling autoimmunity, as well as maintaining the balance between effector and regulatory mechanisms <a href="#pone.0001222-Germain1">[7]</a>–<a href="#pone.0001222-Melero1">[11]</a>. However, despite the many molecular and cellular studies, we still lack a comprehensive understanding of how the immune system is controlled and how autoimmune diseases arise. Given the complex interactions between the cells and molecules that regulate this process, a systems approach to analyse these processes might identify critical functional interactions that are disturbed in autoimmune diseases. Moreover, the identification of such pathological interactions might facilitate the development of new therapeutic targets <a href="#pone.0001222-Kitano1">[12]</a>, <a href="#pone.0001222-Kitano2">[13]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>MS is a chronic inflammatory and neurodegenerative disease of the central nervous system <a href="#pone.0001222-Hauser1">[14]</a>. MS is characterized by the presence of plaques composed by chronic inflammatory infiltrates, including T and B cells as well as monocytes into the brain, accompanied by the presence of large areas of demyelination and axonal loss <a href="#pone.0001222-Sospedra1">[6]</a>. MS is the second cause of permanent disability in young adults after spinal cord injury and due to its chronic nature imposes a significant health and social cost in western countries. Although current immunotherapies are able to modify disease course, we still need to develop more effective and safe therapies for improving the quality of life of patients.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The development of network theory is providing important insights into gene and protein networks <a href="#pone.0001222-Jeong1">[15]</a> . However, the translation of such advances to humans complex diseases such as autoimmune diseases is confronted with many challenges, such as incomplete knowledge of the molecules involved, lack of quantitative data, the higher degree of complexity and the limited availability of analytical methods. Among several methods of network analysis for reconstructing network topology from experimental datasets <a href="#pone.0001222-StyczynskiMP1">[16]</a>, Bayesian networks are those that offer the best results <a href="#pone.0001222-Sachs1">[17]</a>, <a href="#pone.0001222-Bansal1">[18]</a>. In human complex diseases, the use of different clinical phenotypes such as quantitative traits, disease subtypes or therapies, can introduce meaningful perturbations into a network to help infer its topology <a href="#pone.0001222-Lamb1">[19]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The aim of our study was to assess the functional properties of the gene network that controls the T-cell activation processes in healthy circumstances and in an autoimmune disease such as MS. Furthermore, we assessed the effect of immunotherapy in such a gene network. In addition we were interested in identifying new therapeutic targets at the systems level. In order to achieve this objective, we performed a network analysis using quantitative measurements of gene expression obtained by real time PCR from a small number of well-known genes involved in T-cell activation, as well as using prior biological information. We limited our study to a set of 20 genes for two reasons: 1) we were interested in obtaining a balanced matrix considering number of genes, subjects and perturbations; and 2) the limited amount of RNA from every individual in which assess the gene expression levels by real time PCR in the same sample. We employed a Bayesian approach to obtain an accurate reconstruction of the network and indeed, we assessed the qualitative and quantitative network properties using several new methods from systems biology. This systems approach to autoimmunity revealed functional differences in the gene network that controls T-cell activation that cannot be captured with previous methods. Moreover, we show how this approach can be useful in translational clinical research by evaluating the effect of current therapies, and by identifying new therapeutic targets for immunotherapy, such as the Jagged1-Notch pathway.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Network analysis of the T-cell activation gene network</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In order to obtain a more accurate network, we decided to focus on a small set of genes for which the biological information was more complete, they were previously implicated in MS and having them distributed among five basic functions associated with T-cell activation. In this way, we were able to obtain quantitative data and a good balance between the size of the network and the number of possible perturbations related to the different disease phenotypes and therapies <a href="#pone.0001222-Sachs1">[17]</a>. Thus, the experimental dataset was obtained by assessing the gene expression levels of the 20 genes by real time PCR in a cohort of 104 subjects, composed by healthy controls and patients with MS (see <a href="#pone-0001222-t001">Table 1</a> for details of the clinical characteristics and <a href="#pone.0001222.s001">Table S1</a> for statistically summary of gene expression levels). Our approach profits the presence of different phenotypes (healthy, patient condition or different immunotherapies) as perturbations to the system in order to improve the ability of Bayesian inference to identify the right interactions. In the other hand, since Bayesian algorithms might lead to several different results using the same experimental dataset, we also feed the algorithm with a structural network template to decrease the number of different outputs and to increase its accuracy. We obtained the template (<em>structural network</em>) by applying biological knowledge from co-expression studies available at the Ingenuity database (<a href="http://www.ingenuity.com/">http://www.ingenuity.com/</a>). The resulting <em>structural network</em> identified 50 links (<a href="#pone-0001222-g001">Fig 1</a>). Using this <em>structural network</em> as a template, we inferred the topology of the human T-cell activation network by using the experimental dataset performing a Bayesian network approach (see <a href="#s4">methods</a>).</p>
<div class="figure" id="pone-0001222-g001"><div class="img"><a name="pone-0001222-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001222.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001222.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Network analysis of the human T-cell activation network:</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The structural network was obtained from co-expression analysis using the Ingenuity database. The structural network has 20 genes and we identified 50 links. Using the structural network as a template and the experimental dataset (gene expression levels of the 20 genes from 104 subjects quantified by real time-PCR), we reconstructed the T-cell activation network. The network contains the 20 genes and we identified 31 links (see <a href="#pone-0001222-t001">Table 1</a> for information about the weight, direction and previous biological knowledge of the links).</p>
<span>doi:10.1371/journal.pone.0001222.g001</span></div><div class="figure" id="pone-0001222-t001"><div class="img"><a name="pone-0001222-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001222.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001222.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Demographic and clinical data of patients and controls</span></strong></p><span>doi:10.1371/journal.pone.0001222.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The inferred network model (<a href="#pone-0001222-g001">Fig 1</a> and <a href="#pone-0001222-t002">Table 2</a>) has 31 links between the 20 genes, of which 26 were arcs and 4 were edges. We found that hubs in the T-cell activation network differ from those in the <em>structural</em> network, indicating that the experimental dataset was informative to overcome the literature bias by which genes studied for many years (e.g. IFNγ) are cited more often than new genes (e.g. transcription factors). Moreover, because cycles are not permitted in Bayesian networks, the links identified represent a selection of the most relevant interactions in the network, including the removal of the redundant ones such as autocrine loops. Of the 31 links identified, 18 were identified in the gene co-expression searches in the Ingenuity database, or were described in the biological literature (supplementary <a href="#pone.0001222.s002">Table S2</a> and <a href="#pone.0001222.s003">S3</a>) and as such, they were already present in the <em>structural</em> network. The remaining 32 links present in the structural network and absent in the final network were discarded by the algorithm for the following reasons: 1) the gene interaction is not functional in the tissue studied (PBMC); 2) the magnitude of the interaction was not enough for being detected based in the gene co-expression levels; 3) removal of cycles imposed by the Bayesian inference. For 15 of the 18 predicted links, the direction of the interaction was correctly inferred, while for the other three interactions there was insufficient information for the Bayesian algorithm to define this parameter. The Bayesian algorithm found 13 new links that have not been previously reported or identified by bioinformatics analyses of the gene co-expression database. Due to the small size of our network, we did not calculate common network parameters such as degree, mean path-length or clustering coefficient.</p>
<div class="figure" id="pone-0001222-t002"><div class="img"><a name="pone-0001222-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001222.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001222.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Gene interactions identified in the T-cell activation network.</span></strong></p><span>doi:10.1371/journal.pone.0001222.t002</span></div>

<h4>Validation of the new inferred links</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In order to assess the accuracy of our network analysis method and the validity of our T-cell activation network, we validated the newly inferred links as well some of the predicted ones in a new dataset of 16 healthy individuals. In addition, the validation of new links would probe that differences in the topology of the gene network have biological implications, at least at the gene expression level. In vitro assays were performed by stimulating PBMCs with the recombinant proteins involved in such links (the parent node), to assess their influence on the target gene (the child node) in terms of expression. We were able to experimentally validate the interaction between 7 of the 8 links assessed (87% validation), defining a statistical association between the levels of the parent and the child node (<a href="#pone-0001222-t003">Table 3</a>). We found a significant increase in CD28, IL12A and ITGA4 gene expression after 12 or 24 hours in cells stimulated with IL-10 (p&lt;0.05, <a href="#pone-0001222-t003">Table 3</a>), validating the predicted arcs IL10–CD28 and IL10–ITGA4, and the new arc IL10–IL12A. Furthermore, JAG1 gene expression augmented after 24 hours in IL-4 stimulated cells (p = 0.017), validating the predicted edge JAG1–IL4. Finally, in CTLA-4 IgG stimulated cells JAG1, MX1 and PTPRC gene expression all increased significantly (p&lt;0.05), validating the newly identified arcs CTLA4–JAG1, and CTLA4–MX1 and the predicted edge PTPRC–CTLA4. Thus, these results confirm the accuracy of the reconstructed T-cell activation network.</p>
<div class="figure" id="pone-0001222-t003"><div class="img"><a name="pone-0001222-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001222.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001222.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Validation of the gene interactions in the T-cell activation network.</span></strong></p><span>doi:10.1371/journal.pone.0001222.t003</span></div>

<h4>The topology of the network reveals pleiotropy of genes in biological functions</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We performed a qualitative analysis of the functional properties of the topology of the network <a href="#pone.0001222-delaFuente1">[20]</a>, <a href="#pone.0001222-Sontag1">[21]</a> using network analysis <a href="#pone.0001222-Klamt1">[22]</a>. First we obtained the dependence matrix that assesses the role of every gene in a given function based on the definition of such function and the constraints imposed by the topology of our network (<a href="#pone-0001222-g002">Figure 2</a>). We found that the topology of the reconstructed network identified some of the biological function of the participating genes, such as the opposing roles of Th1-Th2 activity, as described by the stimulatory effect of IFNG and TBX21 in Th1 function, and of GATA3 and STAT6 in Th2 function, as well as by the inhibitory effect of IFNG and TBX21 in Th2 function, and of GATA3 and STAT6 in Th1 function (<a href="#pone-0001222-g002">Figure 2</a>). Indeed, the topological analysis identified HLA molecules as activators in the antigen presentation process and the role of CTLA4, IL-10, TGFß and JAG1 in Treg function. Moreover, the dependence matrix also highlighted the pleiotropic activity of many genes in T-cell activation, as represented by the dual role (in yellow) of several genes in the majority of the functions analyzed. In fact, the majority of the genes were activators of migration, indicating that the expression of adhesion molecules by T cells must be regulated after activation so that they may migrate and act in the target tissue. Furthermore, MX1 was included to assess the effect of IFN-ß therapy, although it does not influence T-cell activation, and we found that it did not affect the dependency matrix of the T-cell activation network, reinforcing the specificity of the analysis. However, the small size of the network and limitation of the network analysis process might prevent the right identification of the implication of every single gene in the functions studied, exemplified by the fact that in some cases the Cell Net Analyzer (CNA, see <a href="#s4">methods</a>) was not able to discriminate a stimulatory or inhibitory effect and release a dual influence (yellow) or no influence (white) for a given gene. We identified the minimal cut sets (MCS, see <a href="#s4">methods</a>) for each function in order to assess the structural robustness and fragility of the given function (<a href="#pone-0001222-t004">Table 4</a>). We found that the process of T-cell activation was very robust (robustness = 1, <a href="#pone-0001222-t004">Table 4</a>), mainly due to the selection of the genes critical for this process. Migration displayed an intermediate robustness (robustness = 0.58, <a href="#pone-0001222-t004">Table 4</a>) because most of the genes contribute to generate a specific pattern of adhesion molecule expression after T-cell activation. In contrast, Th1-Th2 and Tr function were not very robust (robustness = 0.27 and 0.21 respectively, <a href="#pone-0001222-t004">Table 4</a>), indicating that T-cell fate after activation is less fixed and that it is more sensitive to stochastic events or environmental signals <a href="#pone.0001222-Lanzavecchia1">[2]</a>.</p>
<div class="figure" id="pone-0001222-g002"><div class="img"><a name="pone-0001222-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001222.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001222.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Network Analysis: Dependence matrix.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The role of each gene in every T-cell activation function (antigen (Ag) presentation; Th1 differentiation; Th2 differentiation; Treg function; migration) based on the topology of the network is displayed using the following colour code: yellow: dual role (activator or inhibitor); green: full activator; red: full inhibitor; white: no influence.</p>
<span>doi:10.1371/journal.pone.0001222.g002</span></div><div class="figure" id="pone-0001222-t004"><div class="img"><a name="pone-0001222-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001222.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001222.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Network Analysis: Minimal cut-set analysis (MCS).</span></strong></p><span>doi:10.1371/journal.pone.0001222.t004</span></div>

<h4>Differences in the T-cell activation network between healthy individuals and patients with Multiple Sclerosis</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To quantify the importance of each interaction of the network in healthy controls, MS patients and MS patients treated with INF β, we calculated the Kullback-Leibler divergence (KL-divergence) of each interaction of the network for each diagnosis. To do that, we used the inferred topology of the network of T-cell activation and the gene expression levels of each of the groups (see <a href="#s4">methods</a>). Thus, we obtained quantitative values that describe the behaviour of each interaction of the T-cell activation network under the different diagnosis. These values are indicative of the importance of the interaction, or to put it another way, they are indicative of the activation of that interaction. We found that the weight of some gene interactions differed in the two networks (<a href="#pone-0001222-g003">Figure 3A</a> represented by the colour of the arrow; <a href="#pone-0001222-t005">Table 5</a>). Accordingly, we found a significant decrease in the weight of the interaction between JAG1–IL4, IL10–IL12A, and IL12A–STAT6 in patients with MS (p&lt;0.05), and a significant increase in the weight of the interaction between TGFB1–IL12A and PTPRC–JAG1 when compared to HC (p&lt;0.05).</p>
<div class="figure" id="pone-0001222-g003"><div class="img"><a name="pone-0001222-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001222.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001222.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Comparison of the T-cell activation network between patients and controls.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>A) HC versus untreated MS patients; B) untreated MS patients versus MS patients treated with IFN-ß; C) HC versus patients treated with IFN-ß. Comparisons between gene interaction weights are described using the following colour code: black: no change; green: decreased; red: increased. See <a href="#pone-0001222-t004">Table 4</a> for statistical analysis.</p>
<span>doi:10.1371/journal.pone.0001222.g003</span></div><div class="figure" id="pone-0001222-t005"><div class="img"><a name="pone-0001222-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001222.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001222.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Comparison in the weight of the interaction in the T-cell activation network between controls, MS patients and MS patients treated with IFN-ß.</span></strong></p><span>doi:10.1371/journal.pone.0001222.t005</span></div>

<h4>Effect of immunotherapy in the T cell activation network</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Because immunomodulatory treatments might exert their activity in a pleiotropic manner, we were interested in evaluating the effect of one of the therapies for MS in the network, such as INF-ß therapy. Our aim was to identify the functions targeted by IFN-ß in order to validate our approach to study immunotherapies at the systems level. Following the same procedure, we reconstructed the T-cell activation network using the gene expression levels from IFN-ß treated and untreated patients, and we compared the differences in interaction weight between pairs of genes (<a href="#pone-0001222-g003">Figure 3B</a>). When the IFN-ß treated and untreated patients were compared, we found a significant increase in the weight of the interaction between ITGA4–TGFB1 and IL4–HLA-DRA (p&lt;0.001) and a significant decrease in the weight of the interaction between JAG1–TNF, TGFB1–GATA3, IL10–IL12A, IL10–CD28, TGFB1–ITGB7, ITGA4–ITGB7 and IL10–ITGA4 (p&lt;0.005, <a href="#pone-0001222-t005">Table 5</a>). Thus, our results indicate that IFN-ß modifies different regions of the T-cell activation network, affecting different functions and thereby confirming its pleiotropic activity. While we identified the immunomodulatory effect of IFN-ß therapy in the T-cell activation network, we found that it did not restore their interaction weight to the baseline levels represented by the HC network (<a href="#pone-0001222-g003">Figure 3C</a>). Hence, IFN-ß therapy fails to completely restore the T-cell activation network indicating that this therapy is unable to completely normalize T cell function. This is in agreement with the clinical experience that IFN-ß therapy is only partially effective in MS <a href="#pone.0001222-Montalban1">[23]</a>.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We were also interested in identifying new therapeutic targets not addressed by IFN-ß therapy. Thus, we analyzed the differences between the HC and the IFN-ß treated patient networks. Ideally, a therapy will restore the functional state close to that of the healthy gene network and any deviation from such state can be considered as a therapeutic target <a href="#pone.0001222-Kitano1">[12]</a>. The comparison between IFN-ß treated patients and HC networks (<a href="#pone-0001222-g003">Figure 3C</a>, <a href="#pone-0001222-t005">Table 5</a>) showed a significant increase in the weight of the gene interaction between the ITGA4–TGFB1, and the PTPRC–JAG1 genes (p&lt;0.05) and a decrease in the weight of the interactions between JAG1–IL4, JAG1–TNF, TGFB1–GATA3, IL10–IL12A, IL12A–STAT6, IL10–CD28, TGFB1–ITGB7, ITGA4–ITGB7 and IL10–ITGA4 (p&lt;0.05). Overall, we found that IFN-ß therapy failed to restore the balance between pro-inflammatory cytokines and Treg function. Pro-inflammatory cytokines, such as TNF or IL12A, the T-cell activation molecule CD28, the regulatory molecules IL10, TGFB1 or the adhesion molecules ITGA4 and ITGB1 have all been evaluated previously as therapeutic targets for autoimmune diseases, including MS <a href="#pone.0001222-Feldmann1">[24]</a>–<a href="#pone.0001222-Rice1">[29]</a>. Hence, we focused on the JAG1 gene as a potential target for new therapies to treat autoimmune diseases. JAG1 plays a critical role in our network since it was consistently modified in the disease state and its interactions were almost unmodified by IFN-ß therapy, making it an excellent therapeutic target in our model, even if JAG1 mRNA levels were not significantly different between patients and controls (<a href="#pone.0001222.s001">Table S1</a>). Moreover, JAG1 has recently being identified as a candidate gene for MS <a href="#pone.0001222-The1">[30]</a>.</p>


<h4>Validation of JAG1 as a therapeutic target in Multiple Sclerosis</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Notch signalling in the adult immune system has several functions depending on the ligand involved <a href="#pone.0001222-Osborne1">[31]</a>, <a href="#pone.0001222-Rutz1">[32]</a>. Jagged1 signalling promotes Treg function whereas Delta1 promotes pro-inflammatory responses. In order to validate the JAG1 gene and its protein jagged1 as a therapeutic target, first we assessed the effect of stimulating human PBMCs from MS patients <em>in vitro</em> with a Jagged1 peptide agonist <a href="#pone.0001222-Li1">[33]</a>. We used the HES5 and MX1 genes as reporters of the Jagged1-Notch pathway and of the IFN-ß-IFNR1 pathway, respectively. Accordingly, PBMCs cultures stimulated with either the Jagged1 peptide agonist or IFN-ß displayed a significant increase in HES5 or MX1 gene expression (Supplementary <a href="#pone.0001222.s004">Table S4</a>). Moreover, the Jagged1 stimulated network differed from the untreated and the IFN-ß treated network (<a href="#pone-0001222-t006">Table 6</a>, <a href="#pone-0001222-g004">Fig 4A</a>). Compared to the network from untreated patients, Jagged1 therapy induced an increase in the interaction weight between JAG1-TNF and the downstream interaction TNF-IFNG, as well as a decrease in the interaction weight between CD28-TNF (<a href="#pone-0001222-g004">Figure 4A</a>). Thus, our network analysis was able to capture the biological effect of Jagged1-Notch signalling in T-cell activation. Our results suggested that the Jagged1-Notch pathway modulates Th1 function. Exposure of PBMCs from untreated patients to IFN-ß <em>in vitro</em> yielded a network similar to that obtained <em>ex-vivo</em> from IFN-ß treated patients (<a href="#pone-0001222-g003">Figure 3B</a> and <a href="#pone-0001222-g004">4B</a>), although some quantitative differences in the interaction weight between the pairs of genes involved in the effect of IFN-ß were detected. Indeed, IFN-ß therapy impaired the effect of JAG1 on TNF that is required for Jagged1 to suppress Th1 function in our model. Finally, the combination of Jagged1 and IFN-ß treatment indicates a synergistic effect of both therapies (<a href="#pone-0001222-t006">Table 6</a>, <a href="#pone-0001222-g004">Fig 4C</a>), and the effect of each therapy could be identified in the different network interactions, which might produce different functional effects. Moreover, Jagged1 therapy was able to recover the effect of JAG1 on TNF that was lost with IFN-ß therapy, suggesting that this combined therapy will maintain the Th1 function suppression induced by Jagged1.</p>
<div class="figure" id="pone-0001222-g004"><div class="img"><a name="pone-0001222-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001222.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001222.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Comparison of the T-cell activation network after in vitro stimulation with jagged1, IFN-ß or Jagged1+IFN-ß:</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>A) untreated patients versus stimulation with the Jagged1 agonistic peptide for 24 h; B) untreated patients versus stimulation with IFN-B for 24 h; C) untreated patients versus stimulation with JGA1+IFN- ß for 24 h. Comparisons between gene interaction weights are described using the following colour code: black: no change; green: decreased; red: increased. See <a href="#pone-0001222-t005">Table 5</a> for statistical analysis.</p>
<span>doi:10.1371/journal.pone.0001222.g004</span></div><div class="figure" id="pone-0001222-t006"><div class="img"><a name="pone-0001222-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001222.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001222.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Comparison in the weight of the interaction in the T-cell activation network between jagged1, IFN-ß and jagged1 plus IFN-ß treatments in vitro.</span></strong></p><span>doi:10.1371/journal.pone.0001222.t006</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>In order to confirm the results from our network analysis, we evaluated the effect of the Jagged1 agonist peptide in the animal model of MS. C57B6 mice immunized with MOG<sub>35–55</sub> peptide were treated with the Jagged1 peptide i.p. from day 0 to day 30 and this produced a milder progression of the disease (<a href="#pone-0001222-g005">Figure 5A</a>) and lower histological scores (<a href="#pone-0001222-g005">Figure 5B and 5C</a>) than in placebo animals. Hence, Jagged1-Notch signalling appears to exert an immunomodulatory effect in brain autoimmunity. We studied the possible mode of action of Jagged1 therapy by assessing several immune responses involved in autoimmunity. In addition, we tested the <em>in vivo</em> effect of Jagged1 therapy in the genes of the T-cell activation network. Previous studies suggested a role for Jagged1-Notch signalling in promoting Treg and Th2 function <a href="#pone.0001222-Osborne1">[31]</a>. We found that by day 9 after immunization, animals treated with Jagged1 peptide have increased percentage of CD25+Foxp3+ cells that placebo animals (21.6% compared to 13.6%, p = 0.032; <a href="#pone-0001222-g005">Fig. 5D</a>) and such differences were lost by day 30 (<a href="#pone-0001222-g005">Fig. 5D</a>). In addition, we also found that by day 9 after immunization, Jagged1 treated animals have significant higher numbers of Th2 cells (IL-4 secreting cells; p = 0.032) and a trend for decreased numbers of Th1 cells (IFNγ secreting cells; p = 0.056) than placebo animals, and not changes in numbers of IL-17 secreting cells (<a href="#pone-0001222-g005">Fig. 5E</a>). By day 30, differences were lost and frequency of such populations were similar between Jagged1 and placebo treated animals (data not shown). We also analyzed the effect of Jagged1 peptide therapy in gene expression levels in splenocytes by day 9 and 30 after immunization. We found that by day 9 p.i. Jagged1 treated animals have increased gene expression levels of IL-10 (p = 0.029) and decreased levels of TNFα and IL-17a (p&lt;0.05 in both cases, <a href="#pone-0001222-g005">Fig. 5F</a> and <a href="#pone.0001222.s005">Table S5</a>). By day 30, we found a decrease in TGFß1, TNFα, and ITGB7 gene expression levels in animals treated with Jagged1 when compared with untreated animals (p&lt;0.05 in all cases; <a href="#pone.0001222.s005">Table S5</a>). These results might suggest that Jagged1-Notch signalling promotes Treg and Th2 responses and suppress Th1 function. Thus, our findings indicate that the Jagged1-Notch pathway may be a therapeutic target to treat MS and other autoimmune diseases. Moreover, our network approach allowed us to predict a mechanism of action that was not expected from previous biological knowledge.</p>
<div class="figure" id="pone-0001222-g005"><div class="img"><a name="pone-0001222-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001222" data-uri="info:doi/10.1371/journal.pone.0001222.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001222.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001222.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001222.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001222.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Validation of Jagged1 as a therapeutic target in the animal model of MS:</span></strong></p><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1"></a><p>C57B6 mice (n = 60) were immunized with MOG<sub>35–55</sub> and treated with Jagged1 agonist peptide i.p. (n = 30) or placebo (n = 30) from day 0 to day 30 in two different experiments. Twelve animals (6 from each treatment group) were sacrificed by day 9 in order to perform immunological studies. Animals treated with the Jagged1 agonist peptide have a clinical (A) and histological (B) score lower than placebo animals. C) Representative spinal cord sections from placebo (a) and jagged1 (b) treated animals stained with Luxol-fast blue showing a decrease in the presence of inflammatory infiltrates and the extend of demyelination in the Jagged1 treated animals. FACS analysis assessing the percentage of CD25+Foxp3+ Treg cells (D) ELISPOT studies assessing the in vitro secretion of Th1 (INFγ), Th2 (IL-4) and Th17 (IL-17) cytokines (E) and real-time PCR (F) studies in splenocytes from Jagged1 treated and untreated mice. D) Mice treated with Jagged1 peptide have higher percentage of Treg cells than placebo animals by day 9 p.i. (p = 0.032), and such difference disappeared by day 30. E) Splenocytes from Jagged1 treated mice expressed higher levels of IL-4, lower levels of IFNγ and similar levels of Il-17 than placebo animals at day 9 p.i. F) Jagged1 treated animals expressed higher levels of IL-10 and lower levels of TNFα and IL-17 than placebo animals at day 9 p.i. Results are expressed as the mean±SEM</p>
<span>doi:10.1371/journal.pone.0001222.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In the present study we have provided proof of concept that a gene network analysis approach is feasible to study human systems and diseases, providing valuable information about the complex interactions involved in biological process and in complex diseases. This is important since most systems biology studies have been applied to lower organisms and its application to higher animals and humans has been restricted by the lack of biological knowledge, technological and analytical tools, as well as by the higher degree of complexity of such organisms. Biological functions, as well as complex traits and diseases, can only rarely be attributed to an individual molecule. On the contrary, complex interactions between dozens of molecules lead to a specific biological function <a href="#pone.0001222-Hartwell1">[34]</a>, and altering the relationships between these elements may disrupt the activity in such systems. Network analysis has emerged as a powerful tool to understand complex intercellular interactions that contribute to the structure and function of living systems <a href="#pone.0001222-Barabasi1">[1]</a> and it can be used to study complex traits and diseases in order to discover new therapies <a href="#pone.0001222-Goh1">[35]</a>, <a href="#pone.0001222-Kitano3">[36]</a>. Indeed, the application of a Bayesian approach to define cell networks has been successfully used and as well as to infer classic, well understood signalling networks <a href="#pone.0001222-Friedman1">[37]</a>. Such an approach has also provided new insights into specific systems that have not been previously identified through hypothesis-directed research <a href="#pone.0001222-Sachs1">[17]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In our network analysis, we identified 31 interactions between 20 genes acting in the immune system of which, 18 were predicted either from the literature or by bioinformatics analysis of co-expression databases. However, the other 13 interactions were new and had not been predicted using bioinformatics strategies. These latter interactions could be more specific for autoimmunity such that our experimental dataset from patients suffering an autoimmune disease was informative to highlight the importance of these interactions when compared to the healthy state. It must to be remembered that the gene interactions identified in our network does not mean that they are direct molecular interactions and intermediary molecules might account for such effect. Indeed, among the many interactions between the genes in our network, we were able to identify either the most robust or the ones that in the specific condition we are studying (MS) are most relevant, implying that some others could be missed for our analysis, even if they have been biologically demonstrated. The validation of the majority of these interactions, both the predicted and new interactions, indicates that our approach was sufficiently powerful and accurate to identify true gene interactions in human cells. In addition, the validation of new gene interactions indicates that differences in gene interaction weight have biological implications, at least at the gene expression level. The high validation rate was due to the balanced design between the number of genes studied, the use of accurate quantitative methods and the biological knowledge available in the literature, as well as benefiting from the use of system biology methods such as Bayesian inference, previously validated in other settings. In addition, in the majority of the cases we were also able to correctly identify the direction of the interactions. Identifying the direction of an interaction is an important step in reverse network engineering and requires larger datasets for network inference <a href="#pone.0001222-Sachs1">[17]</a>. Although we use a mixed cell population such as PBMCs, our results indicates that the network analysis was able to identify the contribution of every gene to the T cell activation process, which is a multicellular process by definition implying antigen presenting cells and T cells. Finally, the qualitative analysis of the network using CNA shows that our method was able to identify a network topology, which is related with the biological functions involved. This analysis was more an attempt to confirm the influence of network topology in biological functions even if such analysis was limited by the size of our network and was not able to predict all functions studied (e.g. the role of IL-4 in Th2). Thus, our results suggest that the reconstruction of biological networks from experimental data is a feasible strategy to study human system and diseases.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Biological networks are robust <a href="#pone.0001222-Barabasi1">[1]</a>, and we predicted that the differences between the healthy and disease state for complex and polygenic disease would not be based on the deletion of nodes (genes or proteins) or even links (gene or protein interactions), but rather in more subtle changes in the weight of the interactions. For this reason we were more interested in performing a quantitative analysis of the network than in the topological differences between such networks. Our quantitative analysis of the network that controls T-cell activation revealed several significant differences between the healthy state and MS. MS is an inflammatory disease in which T-cell activation is dysfunctional <a href="#pone.0001222-Sospedra1">[6]</a> and regulatory T cell function is impaired <a href="#pone.0001222-BaecherAllan1">[38]</a>. The comparison between the T-cell activation networks from HC and MS patients showed an overall decrease in the activity (defined here by the change in the weight of the interaction between pair of genes) of Treg and Th2 function, and an increase in Th1 activity. For example, JAG1 constitutes an instructive signal for Th2 differentiation by inducing GATA3 and by directly regulating IL4 gene transcription <a href="#pone.0001222-Amsen1">[39]</a>. Thus, the reduction in the weight of the JAG1–IL4 gene interaction in the MS patient network indicates a decrease in Th2 differentiation. Furthermore, the reduced weight of the IL12A–STAT6 interaction also suggests diminished Th2 differentiation due to the fact that STAT6 plays a central role in exerting IL4 mediated biological responses <a href="#pone.0001222-Quelle1">[40]</a> and IL12 is important in the regulation of the Th2 response <a href="#pone.0001222-Liu1">[41]</a>. Indeed, a reduction in the weight of the IL10–IL12A interaction indicates a decrease in the suppression mediated by IL-10 over Th1 function <a href="#pone.0001222-Kemper1">[42]</a>. On the other hand, an increase in the weight of the TGFB1–IL12A and PTPRC–JAG1 interactions suggests more pronounced Th1 activity in patients with MS. IL12 is required for the induction of IFNγ and PTPRC induces the expression of TNFα <a href="#pone.0001222-Hayes1">[43]</a>, which can stimulate cell proliferation via Jagged1 <a href="#pone.0001222-Liu2">[44]</a>. Thus, our results identify JAG1 and IL12A as critical genes in the pathogenesis of MS. IL12 plays a central role in brain autoimmunity <a href="#pone.0001222-Kang2">[45]</a> and recent genetic studies have associated JAG1 to MS susceptibility <a href="#pone.0001222-The1">[30]</a>. Of course, several other genes not included in our study may play a role in the pathogenesis of MS, but our results indicate that even using a small set of genes, a quantitative network analysis is able to identify functional differences between healthy and disease states.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Moreover, we show that a quantitative network analysis provides valuable information about how therapies work at a system level, showing that in most cases they exert a pleiotropic activity that is difficult to capture in single molecule assays. But more importantly, our approach allowed us to identify new therapeutic targets that are not modified by current therapies, based on the quantitative changes in the network produced by the disease. Current immunotherapies for autoimmune diseases such as interferon beta, glatiramer acetate or natalizumab exert their action through pleiotropic activities that are poorly understood <a href="#pone.0001222-Hohlfeld1">[46]</a>. In our study, the increase in the weight of the ITGA4–TGFB1 interaction and the decrease in weight of the JAG1–TNF, TGFB1–GATA3, IL10–CD28, TGFB1–ITGB7, ITGA4–ITGB7 and IL10–ITGA4 interactions in patients treated with IFN-beta when compared with untreated MS patients and HC, suggest a direct influence of IFN-ß on these pathways. The decrease in the weight of the JAG1–TNF, TGFB1–GATA3 and IL10–CD28 interactions support the idea that IFN-ß activity involves the suppression of T cell proliferation, the induction of Th2 cytokine production, inhibition of Th1 cytokine production and inhibition of monocyte activation <a href="#pone.0001222-Galimberti1">[47]</a>. Also, the changes in the links involving the adhesion molecules ITGA4, ITGB1 and ITGB7 confirm the role of IFN-ß treatment in decreasing the surface expression of adhesion molecules, which reduces the migratory potential of T cells <a href="#pone.0001222-Neuhaus1">[48]</a>. The <em>in vitro</em> validation of the IL10–IL12A interaction deserves special mention, as it suggests that IFN-ß affects this pathway. However, based on our analysis, IFN-ß therapy failed to restore such an interaction to the normal levels in the healthy state. Indeed, differences in the weight of the JAG1–IL4, IL12A–STAT6 and PTPRC–JAG1 interactions in treated or untreated MS patients compared with HC would suggest that IFN-ß treatment has no effect on these pathways and as such, they may be considered as novel therapeutic targets. Taking into account that JAG1 seems to be involved in the genetic susceptibility of MS, its role in the immune system, and the failure of IFN-ß treatment to modulate its function, we propose JAG1 as a new therapeutic target in the treatment of the MS.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>As a proof of concept that network analysis might identify valuable therapeutic targets, we validated the Jagged1-Notch pathway as a therapeutic target for MS. The Notch pathway exerts several activities in the developing, as well as in the mature peripheral immune system <a href="#pone.0001222-Osborne1">[31]</a>. The outcome of Notch signalling depends on the ligand involved and accordingly, it has been shown that Jagged1 activates Treg and Th2 function whereas Delta1 promotes the Th1 response. We were able to demonstrate that treating immune cells from patients with MS with an agonistic peptide of Jagged1 <em>in vitro</em> modulates the T-cell activation network, mainly suppressing Th1 function (represented by the strong effect of JAG1 over TNF and the downstream effect of TNF on IFNG). Interfering with Jagged1 has been shown to suppress IFNγ production by dendritic cells <a href="#pone.0001222-Stallwood1">[49]</a>. Moreover, we found that IFN-ß and Jagged1 therapy exerted a synergy in vitro. In our network analysis we were able to identify the individual and non-overlapping effects of both therapies and the predominant role of Jagged1 in suppressing Th1 activation, overcoming the suppressor effect of IFN-ß therapy on the JAG1-TNF interaction. Finally, we were able to identify the therapeutic potential of the Jagged1-Notch pathway in the animal model of MS, confirming its immunomodulatory effect. We found that Jagged1-Notch signalling enhances Treg and Th2 function and suppressed Th1 function in mice suffering an autoimmune disease. Although we found no effect of Jagged1 therapy in Th-17 subset (no changes in levels of secreted IL-17 in the ELISpot assays), indirect evidence might suggest that Jagged1 might influence also this function, since critical molecules for Th-17 induction were modified by Jagged1 therapy at the RNA level such as IL17a, TNFα and TGFß. Finally, it has been described that Notch signalling in MS brains might contribute to failure of regenerative process <a href="#pone.0001222-John1">[50]</a>. We found no evidences that treating animals with an agonistic peptide of Jagged1 might impair recovery for EAE. However, further studies will be required to fully validate the therapeutic opportunities associated with the Jagged1-Notch pathway in autoimmune diseases, mainly considering the critical and pleiotropic functions that this pathway plays in many cell types that might lead to undesired side effects.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In summary, our results suggest that a quantitative network approach is a useful tool in medicine to understand complex diseases and discover new therapeutic targets. In particular, the Jagged1-Notch pathway seems to be a good candidate to participate in the susceptibility and therapy of MS. Hence, a more profound study of its regulation may help to understand its implication in MS and its potential for therapy.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Human subjects and biological assays</h4>
<h5>1. Human subjects.</h5><a id="article1.body1.sec4.sec1.sec1.p1" name="article1.body1.sec4.sec1.sec1.p1"></a><p>We studied 52 patients with MS <a href="#pone.0001222-Polman1">[51]</a> and 52 sex and age matched healthy controls (HC) from the same population with no history of autoimmune diseases. For validation assays of the identified links in the network we used a second set of 16 healthy individuals. For the in vitro assays with PBMCs for testing the role of Jagged1 peptide and IFNB, we used a new set of 24 patients with MS that were not receiving immunomodulatory therapy. The demographic and clinical data from the subjects are shown in Supplementary <a href="#pone.0001222.s001">Table S1</a>. Patients were recruited by their neurologist after obtaining written informed consent. This study was approved by the Institutional Review Board at the University of Navarra.</p>

<h5>2. RNA extraction, probes and rtPCR.</h5><a id="article1.body1.sec4.sec1.sec2.p1" name="article1.body1.sec4.sec1.sec2.p1"></a><p>Peripheral blood mononuclear cells (PBMCs) were isolated using Ficoll-Paque (Pharmacia Biotech) and they were immediately submerged in RNAlater RNA Stabilization Reagent (Qiagen) to preserve the gene expression patterns. Total RNA was isolated using the RNeasy Mini Kit (Qiagen), removing DNA with the RNase-Free DNase Set (Qiagen), and the High-Capacity cDNA Archive Kit (Applied Biosystems) was used to synthesise cDNA from the total RNA. Primer sequences and target-specific fluorescence-labelled TaqMan probes were purchased from Applied Biosystems (TaqMan Gene Expression Assays, Supplementary <a href="#pone.0001222.s002">Table S2</a>).</p>
<a id="article1.body1.sec4.sec1.sec2.p2" name="article1.body1.sec4.sec1.sec2.p2"></a><p>Quantitative real-time PCR (rt-PCR) was performed with the DNA Engine Opticon2 (MJ Research). Each sample was run in triplicate and the target and endogenous control gene were amplified in different wells on each plate. We used GAPDH and B2M as controls because both genes were described as a good choice to normalize leukocyte expression levels <a href="#pone.0001222-Vandesompele1">[52]</a>. Cycle threshold (c(t)) values were acquired with the Opticon Monitor 2.01 software (MJ Research) and using this information, we subtracted the baseline signal as the average of the fluorescence measured from cycle 1 to 40 and we set the c(t) line to a standard deviation of 1.00. The normalized gene expression was calculated using the Q-Gene software application <a href="#pone.0001222-Muller1">[53]</a>, which does not assume that the PCR amplification efficiencies of target and reference genes are equivalent. Real-time PCR efficiencies (E) were calculated by amplifying a series of 2-fold dilutions of each factor. The mean c(t) values were plotted against the log of the amount of cDNA added. The linear graphs obtained (correlation coefficients&gt;0.99) were used to calculate the corresponding E. The mean normalized expression was given by calculating the average c(t) values of the target and reference triplicates. Single gene expression values and the comparison between groups are shown in supplementary <a href="#pone.0001222.s001">Table S1</a>.</p>

<h5>3. Validation of the new inferred links.</h5><a id="article1.body1.sec4.sec1.sec3.p1" name="article1.body1.sec4.sec1.sec3.p1"></a><p>Freshly isolated PBMCs, from a new set of 16 HC not previously used in the network construction, were stimulated with 20 ng/ml recombinant human IL10 (R&amp;D Systems), 4 ng/ml recombinant human IL4 (R&amp;D Systems) or 40 ng/ml recombinant human CTLA4/Fc chimera (R&amp;D Systems) for 12 and 24 hours, or left unstimulated as controls. At the end of the assay, RNA was isolated from the cells and gene expression was measured by rtPCR as described above.</p>

<h5>4. In vitro Jagged1 validation assays.</h5><a id="article1.body1.sec4.sec1.sec4.p1" name="article1.body1.sec4.sec1.sec4.p1"></a><p>Jagged1 validation assays were performed using PBMC from patients with MS untreated and stimulated <em>in vitro</em> with 20 µg/ml of human jagged1 peptide 188–204 (CDDYYYGFGCNKFCRPR; Sigma-Aldrich) <a href="#pone.0001222-Li1">[33]</a>, or with 1.000 u/ml of recombinant human INF-β (PBL Biomedical Laboratories) or both. Cells were incubated for 24 h and RNA was extracted and quantified by real time PCR as described before. KL-divergence values were calculated for each group in the T-cell activation network as described above.</p>

<h5>5. Experimental Autoimmune encephalomyelitis.</h5><a id="article1.body1.sec4.sec1.sec5.p1" name="article1.body1.sec4.sec1.sec5.p1"></a><p>Female C57B6 mice obtained from Charles River (6–8 weeks old; 20 gr. body weight) were immunized in the lateral flank with a 100 µl emulsion of saline and incomplete Freund´s adjuvant containing 100 µg Myelin Olygrodendrocyte Glycoprotein (MOG<sub>35–55</sub>) peptide from Sigma (Germany) supplemented with 4 mg/ml <em>Mycobacterium tuberculosis</em> (H37Ra strain from Difco, Detroit, MI). The animals were weighed on a daily basis and inspected for clinical signs of experimental autoimmune encephalomyelitis (EAE) by a blind observer as described previously <a href="#pone.0001222-Moreno1">[54]</a>. Animals were treated with 1 mg/day of human jagged1 peptide 188–204 (CDDYYYGFGCNKFCRPR; Sigma-Aldrich) <a href="#pone.0001222-Li1">[33]</a> beginning at the day of immunization until day 30 post-immunization. Disease severity was assessed according to the following scale: 0 = normal; 1 = limp tail; 2 = mild paraparesis of the hind limbs, unsteady gait; 3 = moderate paraparesis, voluntary movements still possible; 4 = paraplegia or tetraparesis; 5 = moribund state. Data shown for the EAE studies are representative of two independent experiments performed with the number of animals indicated. The University of Navarra Committee for Animal Care approved the entire animal studies carried out.</p>
<a id="article1.body1.sec4.sec1.sec5.p2" name="article1.body1.sec4.sec1.sec5.p2"></a><p>Histological evaluation was performed on paraformaldehyde-fixed, paraffin embedded sections of the brain and spinal cord, as described previously <a href="#pone.0001222-Moreno1">[54]</a>. Sections (10 µm thick) were stained with haematoxylin and eosin (H&amp;E) and with Luxol fast-blue to assess inflammation and demyelination. We examined 20 consecutive sagittal sections from each region examined (brain, cervical, thoracic and lumbar spinal cord) of every animal in the study. Semi-quantitative histological evaluation for inflammation and demyelination was carried out and scored blindly using the following scale: 0, normal; 1, 1-3/section perivascular cuffs with minimal demyelination; 2, 3-10 perivascular cuffs/section accompanied by moderate demyelination; 3, widespread perivascular cuffing, extensive demyelination with large confluent lesions.</p>

<h5>6. Flow Cytometry analysis.</h5><a id="article1.body1.sec4.sec1.sec6.p1" name="article1.body1.sec4.sec1.sec6.p1"></a><p>Cells were phenotyped by three-colour flow cytometry (FACSaria) according to the expression of CD25 and Foxp3 using the mouse regulatory T cell staining kit from Ebiosciences, UK. We analyzed the splenocytes from naïve or treated C57B6 mice suffering EAE at day 9 (n = 6 placebo and 6 Jagged1 treated mice) and day 30 (n = 24 placebo and 24 Jagged1 treated mice) postinjection (p.i.).</p>

<h5>7. ELISpot.</h5><a id="article1.body1.sec4.sec1.sec7.p1" name="article1.body1.sec4.sec1.sec7.p1"></a><p>We assessed the secretion of IFN-γ, IL-4 and IL-17 in splenocytes from naïve or treated C57B6 mice suffering EAE at day 9 using the mouse-IFNγ and mouse-IL4 ELISpot plus kit from Mabtech (Mabtech, US), and the mouse-IL17 ELISpot from Ebioscience (Ebioscience,UK) according to the manufacturer's instructions. Antigen stimulation was carried out with the immunizing antigen (MOG<sub>35–55</sub> 100 µg/ml) for 48 h and we used either PHA or ConA (5 µg/ml) as a positive control for Th1 and Th2, or for Th17, respectively. ELISpot quantification was performed with the Immunospot S4 Pro Analyzer (Cellular Technology Ltd, US).</p>



<h4>Network Analysis</h4>
<h5>1. Ingenuity Pathways Knowledge Base analysis.</h5><a id="article1.body1.sec4.sec2.sec1.p1" name="article1.body1.sec4.sec2.sec1.p1"></a><p>Because Bayesian algorithms might provide several different results with the same experimental dataset, feeding the algorithm with a template of the network will diminish the number of different outputs and increase their accuracy. Thus, in order to obtain a template (<em>structural network</em>) of the gene network that controls the T-cell activation, we used biological knowledge from co-expression data available at the Ingenuity database, using the Ingenuity Pathways Knowledge Base (Ingenuity Systems Inc. Redwood City, USA). We were interested in evaluating five critical processes in T-cell activation, namely: 1) antigen presentation and co-stimulation, 2) Th1 differentiation; 3) Th2 differentiation; 4) T regulatory (Treg) function; and 5) migration to tissues. Genes were distributed in five biological functions based in biological knowledge (<a href="#pone.0001222.s002">Table S2</a>) as follows: 1) antigen presentation, signalling threshold modulation and co-stimulation-HLA-DRA, HLA-DQB1, PTPRC (CD45) and CD28; 2) Th1 differentiation-TBX21 (T-bet), STAT1, IFNG, TNFA and IL12A; 3) Th2 differentiation-GATA3, STAT6, IL4, JAG1; 4) Treg function-CTLA4, IL10, TGFB1 and JAG1; and 5) Migration-ITGA4, ITGB1, ITGB7. There is significant biological knowledge regarding each of these genes and they have all been implicated in autoimmune diseases, including MS (see supplementary <a href="#pone.0001222.s002">Table S2</a> and <a href="#pone.0001222.s003">S3</a>).</p>

<h5>2. Bayesian network algorithm.</h5><a id="article1.body1.sec4.sec2.sec2.p1" name="article1.body1.sec4.sec2.sec2.p1"></a><p>Gene expression data from MS patients and HC were made discrete in two intervals with the same number of associated cases. To model the T-cell activation network, we performed structural learning using a Tabu Search <a href="#pone.0001222-Glover1">[55]</a>, based on the experimental data and retaining the structure of the network obtained in the Ingenuity database search described above. Because the inference of the Bayesian network does not accept network cycles, the <em>structural network</em> template was introduced after manually curated the presence of such cycles by changing the orientation of links in order to maintain the link and avoid cycles. During this procedure, no links were removed. The overall approach of the Tabu Search is to avoid entrainment in cycles by forbidding or penalizing moves that, in the next iteration, take the solution to points in the solution space previously visited <a href="#pone.0001222-Glover1">[55]</a>. The Tabu Search algorithm constructs a graphic in which the nodes represent the measured gene levels and the arcs represent statistically meaningful relationships and the dependency between these genes. A Tabu Search was performed using a tabu list size of 10 and a structural complexity influence of 1. Bayesian analysis was processed using the BayesiaLab 3.3 software (Bayesia SA. Laval Cedex, France). The algorithm considered links as arcs (links in which the orientation of the connection cannot be changed without changing the probabilistic relations encoded) if the available information allow the algorithm to define the direction of the interaction, or as edges (links in which the orientation can be inverted) if the algorithm was not able to definitively confirm a unique direction for the interaction.</p>

<h5>3. Qualitative analysis of the network.</h5><a id="article1.body1.sec4.sec2.sec3.p1" name="article1.body1.sec4.sec2.sec3.p1"></a><p>We took advantage of tools developed for metabolomics studies to assess the functional consequences of the network topology <a href="#pone.0001222-delaFuente1">[20]</a>, <a href="#pone.0001222-Sontag1">[21]</a>. The Cell Net Analysis (CNA) approach and CNA 6.0 software (Steffen Klamt, Max-Planck Institute, Magdeburg, Germany) was used to measure the overall network function since gene expression patterns could be considered stable states. We first evaluated the dependence matrix in order to assess the functional consistency of the reconstructed network. We then identified the minimal cut sets (MCS) <a href="#pone.0001222-Klamt1">[22]</a> required for the loss of a defined cell function. The concept of minimal cut sets has been introduced to study structural fragility and to identify knock-out strategies in biochemical reaction networks <a href="#pone.0001222-Klamt1">[22]</a>. In this study, a MCS was defined as a minimal set of interactions whose removal blocks cell function (e.g. Th1 differentiation). We defined the maintenance of a given function based on biological knowledge using logical rules as follow: 1) Antigen presentation and co-stimulation: CD28 AND PTPRC AND (HLA-DQB1 OR HLA-DRA) NOT (CTLA4 OR JAG1OR (CTLA4 AND JAG1) OR IL10 OR TGFB1 OR (IL10 AND TGFB1) OR (CTLA4 OR JAG1 OR (CTLA4 AND JAG1) AND (IL10 OR TGFB1 OR (IL10 AND TGFB1)))); 2) Th1 differentiation: INFG AND IL12A AND STAT1 AND TBX21 AND TNF NOT (IL4 AND STAT6 AND GATA3 AND JAG1); 3) Th2 differentiation: IL4 AND STAT6 AND GATA3 AND JAG1 NOT (INFG AND IL12A AND STAT1 AND TBX21 AND TNF); 4) Treg function: CTLA4 OR JAG1OR (CTLA4 AND JAG1) OR IL10 OR TGFB1 OR (IL10 AND TGFB1) OR (CTLA4 OR JAG1 OR (CTLA4 AND JAG1) AND (IL10 OR TGFB1 OR (IL10 AND TGFB1))); 5) migration: ITGA4 AND (ITGB1 OR ITGB7) (which builds the VLA4 or LPAM adhesion molecule respectively). We then calculated the fragility of the network by using the F coefficient <a href="#pone.0001222-Klamt1">[22]</a>, which is defined as the inverse of the average value of the number of interactions participating in a minimal cut set. Robustness is defined as 1–fragility. The five T-cell activation functions described were considered as outputs of the network.</p>

<h5>4. Quantitative analysis of the network.</h5><a id="article1.body1.sec4.sec2.sec4.p1" name="article1.body1.sec4.sec2.sec4.p1"></a><p>In order to measure the weight of each link in the different conditions we calculated the Kullback-Leibler divergence (KL-divergence) for each link of the network using ten groups of samples for each diagnosis, randomly constructed with half of the total samples for each condition. The KL-divergence is a measure that can be used to evaluate the differences between the probability distributions represented by the network, both with the arc corresponding to the relation and without this arc <a href="#pone.0001222-Kullback1">[56]</a>.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001222.e001&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec4.sec2.sec4.p2" name="article1.body1.sec4.sec2.sec4.p2"></a><p>In this case, it is a measure of the mutual information between the parent and the child node in the network. The KL-divergence is a natural distance measure from a “true” probability distribution P to an arbitrary probability distribution Q. Typically P represents data, observations, or a precise calculated probability distribution. The measure Q typically represents a theory, a model, a description or an approximation of P.</p>



<h4>Statistical analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The Shapiro-Wilk Test was carried out on each group to assess the normal distribution. Those data groups that were not normal were transformed to base-e-logarithm. We used the <em>student-</em>t test to compare individual gene expression levels between MS patients and HC, the influence of IFN-ß treatment in patients with MS, KL-divergence values and gene expression in animal studies. Differences in the clinical course of EAE, histological scores, ELISpot and flow cytometry studies were assessed with the Mann-Whitney U test. The Bonferroni correction for multiple testing was applied as required. For all tests p values of &lt;0.05 were considered as significant. Data was analyzed and processed using the SPSS 13.0 statistical package (SPSS Inc. Chicago, USA).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001222"></div><a name="pone.0001222.s001" id="pone.0001222.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001222.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001222.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.11 MB DOC)</p>
<a name="pone.0001222.s002" id="pone.0001222.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001222.s002">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001222.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.20 MB DOC)</p>
<a name="pone.0001222.s003" id="pone.0001222.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001222.s003">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001222.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0001222.s004" id="pone.0001222.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001222.s004">Table S4. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001222.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0001222.s005" id="pone.0001222.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001222.s005">Table S5. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001222.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="postSiDOI">(0.11 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We wish to thank Begoña Fernández-Díez for her technical support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PV RP IM. Performed the experiments: RP. Analyzed the data: PV RP JG JI JS IM. Contributed reagents/materials/analysis tools: JG JI JS IM. Wrote the paper: PV RP IM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001222-Barabasi1" id="pone.0001222-Barabasi1"></a>Barabasi AL, Oltvai ZN (2004) Network biology: understanding the cell's functional organization. Nat Rev Genet  5(2): 101–113.  <ul class="find" data-citedArticleID="1021818" data-doi="10.1038/nrg1272"><li><a href="http://dx.doi.org/10.1038/nrg1272" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+biology%3A+understanding+the+cell%27s+functional+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+biology%3A+understanding+the+cell%27s+functional+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001222-Lanzavecchia1" id="pone.0001222-Lanzavecchia1"></a>Lanzavecchia A, Sallusto F (2005) Understanding the generation and function of memory T cell subsets. Curr Opin Immunol  17(3): 326–332.  <ul class="find" data-citedArticleID="1021868" data-doi="10.1016/j.coi.2005.04.010"><li><a href="http://dx.doi.org/10.1016/j.coi.2005.04.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+the+generation+and+function+of+memory+T+cell+subsets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+the+generation+and+function+of+memory+T+cell+subsets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001222-Crabtree1" id="pone.0001222-Crabtree1"></a>Crabtree GR (1989) Contingent genetic regulatory events in T lymphocyte activation. Science  243(4889): 355–361.  <ul class="find" data-citedArticleID="1021820" data-doi="10.1126/science.2783497"><li><a href="http://dx.doi.org/10.1126/science.2783497" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contingent+genetic+regulatory+events+in+T+lymphocyte+activation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contingent+genetic+regulatory+events+in+T+lymphocyte+activation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001222-Hill1" id="pone.0001222-Hill1"></a>Hill N, Sarvetnick N (2002) Cytokines: promoters and dampeners of autoimmunity. Curr Opin Immunol  14(6): 791–797.  <ul class="find" data-citedArticleID="1021842" data-doi="10.1016/s0952-7915(02)00403-x"><li><a href="http://dx.doi.org/10.1016/s0952-7915(02)00403-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytokines%3A+promoters+and+dampeners+of+autoimmunity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytokines%3A+promoters+and+dampeners+of+autoimmunity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001222-Steinman1" id="pone.0001222-Steinman1"></a>Steinman L, Zamvil S (2003) Transcriptional analysis of targets in multiple sclerosis. Nat Rev Immunol  3(6): 483–492.  <ul class="find" data-citedArticleID="1021914" data-doi="10.1038/nri1108"><li><a href="http://dx.doi.org/10.1038/nri1108" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+analysis+of+targets+in+multiple+sclerosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+analysis+of+targets+in+multiple+sclerosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001222-Sospedra1" id="pone.0001222-Sospedra1"></a>Sospedra M, Martin R (2005) Immunology of multiple sclerosis. Annu Rev Immunol  23: 683–747.  <ul class="find" data-citedArticleID="1021910" data-doi="10.1146/annurev.immunol.23.021704.115707"><li><a href="http://dx.doi.org/10.1146/annurev.immunol.23.021704.115707" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunology+of+multiple+sclerosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunology+of+multiple+sclerosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001222-Germain1" id="pone.0001222-Germain1"></a>Germain RN (2001) The art of the probable: system control in the adaptive immune system. Science  293(5528): 240–245.  <ul class="find" data-citedArticleID="1021828" data-doi="10.1126/science.1062946"><li><a href="http://dx.doi.org/10.1126/science.1062946" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+art+of+the+probable%3A+system+control+in+the+adaptive+immune+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+art+of+the+probable%3A+system+control+in+the+adaptive+immune+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001222-Leon1" id="pone.0001222-Leon1"></a>Leon K, Faro J, Lage A, Carneiro J (2004) Inverse correlation between the incidences of autoimmune disease and infection predicted by a model of T cell mediated tolerance. J Autoimmun  22(1): 31–42.  <ul class="find" data-citedArticleID="1021872" data-doi="10.1016/j.jaut.2003.10.002"><li><a href="http://dx.doi.org/10.1016/j.jaut.2003.10.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inverse+correlation+between+the+incidences+of+autoimmune+disease+and+infection+predicted+by+a+model+of+T+cell+mediated+tolerance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inverse+correlation+between+the+incidences+of+autoimmune+disease+and+infection+predicted+by+a+model+of+T+cell+mediated+tolerance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001222-Goodnow1" id="pone.0001222-Goodnow1"></a>Goodnow CC, Sprent J, Fazekas de St Groth B, Vinuesa CG (2005) Cellular and genetic mechanisms of self tolerance and autoimmunity. Nature  435(7042): 590–597.  <ul class="find" data-citedArticleID="1021834" data-doi="10.1038/nature03724"><li><a href="http://dx.doi.org/10.1038/nature03724" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cellular+and+genetic+mechanisms+of+self+tolerance+and+autoimmunity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cellular+and+genetic+mechanisms+of+self+tolerance+and+autoimmunity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001222-Segel1" id="pone.0001222-Segel1"></a>Segel LA, Bar-Or RL (1999) On the role of feedback in promoting conflicting goals of the adaptive immune system. J Immunol  163(3): 1342–1349.  <ul class="find" data-citedArticleID="1021906"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+the+role+of+feedback+in+promoting+conflicting+goals+of+the+adaptive+immune+system.&amp;auth=&amp;atitle=On+the+role+of+feedback+in+promoting+conflicting+goals+of+the+adaptive+immune+system." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+role+of+feedback+in+promoting+conflicting+goals+of+the+adaptive+immune+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+role+of+feedback+in+promoting+conflicting+goals+of+the+adaptive+immune+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001222-Melero1" id="pone.0001222-Melero1"></a>Melero I, Arina A, Chen L (2005) The many sounds of T lymphocyte silence. Immunol Res  33(2): 135–147.  <ul class="find" data-citedArticleID="1021880" data-doi="10.1385/ir:33:2:135"><li><a href="http://dx.doi.org/10.1385/ir:33:2:135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+many+sounds+of+T+lymphocyte+silence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+many+sounds+of+T+lymphocyte+silence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001222-Kitano1" id="pone.0001222-Kitano1"></a>Kitano H, K O, T K, Y M, Csete M, et al.  (2004) Metabolic Syndrome and Robustness Tradeoffs. Diabetes  53(S3): S6–S15.  <ul class="find" data-citedArticleID="1021856" data-doi="10.2337/diabetes.53.suppl_3.s6"><li><a href="http://dx.doi.org/10.2337/diabetes.53.suppl_3.s6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolic+Syndrome+and+Robustness+Tradeoffs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolic+Syndrome+and+Robustness+Tradeoffs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001222-Kitano2" id="pone.0001222-Kitano2"></a>Kitano H (2007) A robustness-based approach to systems-oriented drug design. Nat Rev Drug Discov  6(3): 202–210.  <ul class="find" data-citedArticleID="1021858" data-doi="10.1038/nrd2195"><li><a href="http://dx.doi.org/10.1038/nrd2195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+robustness-based+approach+to+systems-oriented+drug+design." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+robustness-based+approach+to+systems-oriented+drug+design.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001222-Hauser1" id="pone.0001222-Hauser1"></a>Hauser SL, Oksenberg JR (2006) The neurobiology of multiple sclerosis: genes, inflammation, and neurodegeneration. Neuron  52(1): 61–76.  <ul class="find" data-citedArticleID="1021838" data-doi="10.1016/j.neuron.2006.09.011"><li><a href="http://dx.doi.org/10.1016/j.neuron.2006.09.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+neurobiology+of+multiple+sclerosis%3A+genes%2C+inflammation%2C+and+neurodegeneration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+neurobiology+of+multiple+sclerosis%3A+genes%2C+inflammation%2C+and+neurodegeneration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001222-Jeong1" id="pone.0001222-Jeong1"></a>Jeong H, Mason SP, Barabasi AL, Oltvai ZN (2001) Lethality and centrality in protein networks. Nature  411(6833): 41–42.  <ul class="find" data-citedArticleID="1021846" data-doi="10.1038/35075138"><li><a href="http://dx.doi.org/10.1038/35075138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lethality+and+centrality+in+protein+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lethality+and+centrality+in+protein+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001222-StyczynskiMP1" id="pone.0001222-StyczynskiMP1"></a>Styczynski MP SG (2005) Overview of computational methods for the inference of gene regulatory networks. Computers &amp; Chemical Engineering  29: 519–534.  <ul class="find" data-citedArticleID="1021916" data-doi="10.1016/j.compchemeng.2004.08.029"><li><a href="http://dx.doi.org/10.1016/j.compchemeng.2004.08.029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overview+of+computational+methods+for+the+inference+of+gene+regulatory+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overview+of+computational+methods+for+the+inference+of+gene+regulatory+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001222-Sachs1" id="pone.0001222-Sachs1"></a>Sachs K, Perez O, Pe'er D, Lauffenburger DA, Nolan GP (2005) Causal protein-signaling networks derived from multiparameter single-cell data. Science  308(5721): 523–529.  <ul class="find" data-citedArticleID="1021904" data-doi="10.1126/science.1105809"><li><a href="http://dx.doi.org/10.1126/science.1105809" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Causal+protein-signaling+networks+derived+from+multiparameter+single-cell+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Causal+protein-signaling+networks+derived+from+multiparameter+single-cell+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001222-Bansal1" id="pone.0001222-Bansal1"></a>Bansal M, Belcastro V, Ambesi-Impiombato A, di Bernardo D (2007) How to infer gene networks from expression profiles. Mol Syst Biol  3: 78.  Epub 2007 Feb 2013.  <ul class="find" data-citedArticleID="1021816" data-doi="10.1038/msb4100120"><li><a href="http://dx.doi.org/10.1038/msb4100120" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+to+infer+gene+networks+from+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+to+infer+gene+networks+from+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001222-Lamb1" id="pone.0001222-Lamb1"></a>Lamb J, Crawford ED, Peck D, Modell JW, Blat IC, et al.  (2006) The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease. Science  313(5795): 1929–1935.  <ul class="find" data-citedArticleID="1021866" data-doi="10.1126/science.1132939"><li><a href="http://dx.doi.org/10.1126/science.1132939" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001222-delaFuente1" id="pone.0001222-delaFuente1"></a>de la Fuente A, Brazhnik P, Mendes P (2002) Linking the genes: inferring quantitative gene networks from microarray data. Trends Genet  18(8): 395–398.  <ul class="find" data-citedArticleID="1021922" data-doi="10.1016/s0168-9525(02)02692-6"><li><a href="http://dx.doi.org/10.1016/s0168-9525(02)02692-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linking+the+genes%3A+inferring+quantitative+gene+networks+from+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linking+the+genes%3A+inferring+quantitative+gene+networks+from+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001222-Sontag1" id="pone.0001222-Sontag1"></a>Sontag E, Kiyatkin A, Kholodenko BN (2004) Inferring dynamic architecture of cellular networks using time series of gene expression, protein and metabolite data. Bioinformatics  20(12): 1877–1886.  <ul class="find" data-citedArticleID="1021908" data-doi="10.1093/bioinformatics/bth173"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth173" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+dynamic+architecture+of+cellular+networks+using+time+series+of+gene+expression%2C+protein+and+metabolite+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+dynamic+architecture+of+cellular+networks+using+time+series+of+gene+expression%2C+protein+and+metabolite+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001222-Klamt1" id="pone.0001222-Klamt1"></a>Klamt S (2006) Generalized concept of minimal cut sets in biochemical networks. Biosystems  83(2–3): 233–247.  <ul class="find" data-citedArticleID="1021862" data-doi="10.1016/j.biosystems.2005.04.009"><li><a href="http://dx.doi.org/10.1016/j.biosystems.2005.04.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generalized+concept+of+minimal+cut+sets+in+biochemical+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generalized+concept+of+minimal+cut+sets+in+biochemical+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001222-Montalban1" id="pone.0001222-Montalban1"></a>Montalban X (2007) MS treatment: Postmarketing studies. J Neurol Sci.   <ul class="find" data-citedArticleID="1021882" data-doi="10.1016/j.jns.2006.10.022"><li><a href="http://dx.doi.org/10.1016/j.jns.2006.10.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MS+treatment%3A+Postmarketing+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MS+treatment%3A+Postmarketing+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001222-Feldmann1" id="pone.0001222-Feldmann1"></a>Feldmann M, Brennan FM, Maini R (1998) Cytokines in autoimmune disorders. Int Rev Immunol  17(1–4): 217–228.  <ul class="find" data-citedArticleID="1021822" data-doi="10.3109/08830189809084493"><li><a href="http://dx.doi.org/10.3109/08830189809084493" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytokines+in+autoimmune+disorders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytokines+in+autoimmune+disorders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001222-Kang1" id="pone.0001222-Kang1"></a>Kang BY, Kim TS (2006) Targeting cytokines of the interleukin-12 family in autoimmunity. Curr Med Chem  13(10): 1149–1156.  <ul class="find" data-citedArticleID="1021850" data-doi="10.2174/092986706776360879"><li><a href="http://dx.doi.org/10.2174/092986706776360879" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeting+cytokines+of+the+interleukin-12+family+in+autoimmunity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeting+cytokines+of+the+interleukin-12+family+in+autoimmunity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001222-Peter1" id="pone.0001222-Peter1"></a>Peter HH, Warnatz K (2005) Molecules involved in T-B co-stimulation and B cell homeostasis: possible targets for an immunological intervention in autoimmunity. Expert Opin Biol Ther  5: Suppl 1S61–71.  <ul class="find" data-citedArticleID="1021892" data-doi="10.1517/14712598.5.1.s61"><li><a href="http://dx.doi.org/10.1517/14712598.5.1.s61" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecules+involved+in+T-B+co-stimulation+and+B+cell+homeostasis%3A+possible+targets+for+an+immunological+intervention+in+autoimmunity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecules+involved+in+T-B+co-stimulation+and+B+cell+homeostasis%3A+possible+targets+for+an+immunological+intervention+in+autoimmunity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001222-Roncarolo1" id="pone.0001222-Roncarolo1"></a>Roncarolo MG, Battaglia M, Gregori S (2003) The role of interleukin 10 in the control of autoimmunity. J Autoimmun  20(4): 269–272.  <ul class="find" data-citedArticleID="1021900" data-doi="10.1016/s0896-8411(03)00047-7"><li><a href="http://dx.doi.org/10.1016/s0896-8411(03)00047-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+interleukin+10+in+the+control+of+autoimmunity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+interleukin+10+in+the+control+of+autoimmunity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001222-Le1" id="pone.0001222-Le1"></a>Le Y, Yu X, Ruan L, Wang O, Qi D, et al.  (2005) The immunopharmacological properties of transforming growth factor beta. Int Immunopharmacol  5(13–14): 1771–1782.  Epub 2005 Aug 1772. <ul class="find" data-citedArticleID="1021870" data-doi="10.1016/j.intimp.2005.07.006"><li><a href="http://dx.doi.org/10.1016/j.intimp.2005.07.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+immunopharmacological+properties+of+transforming+growth+factor+beta." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+immunopharmacological+properties+of+transforming+growth+factor+beta.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001222-Rice1" id="pone.0001222-Rice1"></a>Rice GP, Hartung HP, Calabresi PA (2005) Anti-alpha4 integrin therapy for multiple sclerosis: mechanisms and rationale. Neurology  64(8): 1336–1342.  <ul class="find" data-citedArticleID="1021898" data-doi="10.1212/01.wnl.0000158329.30470.d0"><li><a href="http://dx.doi.org/10.1212/01.wnl.0000158329.30470.d0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anti-alpha4+integrin+therapy+for+multiple+sclerosis%3A+mechanisms+and+rationale." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anti-alpha4+integrin+therapy+for+multiple+sclerosis%3A+mechanisms+and+rationale.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001222-The1" id="pone.0001222-The1"></a>The Games Collaborative G (2006) Linkage disequilibrium screening for multiple sclerosis implicates JAG1 and POU2AF1 as susceptibility genes in Europeans. J Neuroimmunol  23: 23.  <ul class="find" data-citedArticleID="1021918" data-doi="10.1016/j.jneuroim.2006.06.003"><li><a href="http://dx.doi.org/10.1016/j.jneuroim.2006.06.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+disequilibrium+screening+for+multiple+sclerosis+implicates+JAG1+and+POU2AF1+as+susceptibility+genes+in+Europeans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+disequilibrium+screening+for+multiple+sclerosis+implicates+JAG1+and+POU2AF1+as+susceptibility+genes+in+Europeans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001222-Osborne1" id="pone.0001222-Osborne1"></a>Osborne BA, Minter LM (2007) Notch signalling during peripheral T-cell activation and differentiation. Nat Rev Immunol  7(1): 64–75.  Epub 2006 Dec 2015. <ul class="find" data-citedArticleID="1021890" data-doi="10.1038/nri1998"><li><a href="http://dx.doi.org/10.1038/nri1998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Notch+signalling+during+peripheral+T-cell+activation+and+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Notch+signalling+during+peripheral+T-cell+activation+and+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001222-Rutz1" id="pone.0001222-Rutz1"></a>Rutz S, Mordmuller B, Sakano S, Scheffold A (2005) Notch ligands Delta-like1, Delta-like4 and Jagged1 differentially regulate activation of peripheral T helper cells. Eur J Immunol  35(8): 2443–2451.  <ul class="find" data-citedArticleID="1021902" data-doi="10.1002/eji.200526294"><li><a href="http://dx.doi.org/10.1002/eji.200526294" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Notch+ligands+Delta-like1%2C+Delta-like4+and+Jagged1+differentially+regulate+activation+of+peripheral+T+helper+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Notch+ligands+Delta-like1%2C+Delta-like4+and+Jagged1+differentially+regulate+activation+of+peripheral+T+helper+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001222-Li1" id="pone.0001222-Li1"></a>Li L, Milner LA, Deng Y, Iwata M, Banta A, et al.  (1998) The human homolog of rat Jagged1 expressed by marrow stroma inhibits differentiation of 32D cells through interaction with Notch1. Immunity  8(1): 43–55.  <ul class="find" data-citedArticleID="1021874" data-doi="10.1016/s1074-7613(00)80457-4"><li><a href="http://dx.doi.org/10.1016/s1074-7613(00)80457-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+homolog+of+rat+Jagged1+expressed+by+marrow+stroma+inhibits+differentiation+of+32D+cells+through+interaction+with+Notch1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+homolog+of+rat+Jagged1+expressed+by+marrow+stroma+inhibits+differentiation+of+32D+cells+through+interaction+with+Notch1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001222-Hartwell1" id="pone.0001222-Hartwell1"></a>Hartwell LH, Hopfield JJ, Leibler S, Murray AW (1999) From molecular to modular cell biology. Nature  402: 6761 SupplC47–52.  <ul class="find" data-citedArticleID="1021836" data-doi="10.1038/35011540"><li><a href="http://dx.doi.org/10.1038/35011540" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+molecular+to+modular+cell+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+molecular+to+modular+cell+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001222-Goh1" id="pone.0001222-Goh1"></a>Goh KI, Cusick ME, Valle D, Childs B, Vidal M, et al.  (2007) The human disease network. Proc Natl Acad Sci U S A.   <ul class="find" data-citedArticleID="1021832" data-doi="10.1073/pnas.0701361104"><li><a href="http://dx.doi.org/10.1073/pnas.0701361104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+disease+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+disease+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001222-Kitano3" id="pone.0001222-Kitano3"></a>Kitano H (2002) Computational systems biology. Nature  420(6912): 206–210.  <ul class="find" data-citedArticleID="1021860" data-doi="10.1038/nature01254"><li><a href="http://dx.doi.org/10.1038/nature01254" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+systems+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+systems+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001222-Friedman1" id="pone.0001222-Friedman1"></a>Friedman N (2004) Inferring cellular networks using probabilistic graphical models. Science  303(5659): 799–805.  <ul class="find" data-citedArticleID="1021824" data-doi="10.1126/science.1094068"><li><a href="http://dx.doi.org/10.1126/science.1094068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+cellular+networks+using+probabilistic+graphical+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+cellular+networks+using+probabilistic+graphical+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001222-BaecherAllan1" id="pone.0001222-BaecherAllan1"></a>Baecher-Allan C, Hafler DA (2006) Human regulatory T cells and their role in autoimmune disease. Immunol Rev  212: 203–216.  <ul class="find" data-citedArticleID="1021814" data-doi="10.1111/j.0105-2896.2006.00417.x"><li><a href="http://dx.doi.org/10.1111/j.0105-2896.2006.00417.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+regulatory+T+cells+and+their+role+in+autoimmune+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+regulatory+T+cells+and+their+role+in+autoimmune+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001222-Amsen1" id="pone.0001222-Amsen1"></a>Amsen D, Blander JM, Lee GR, Tanigaki K, Honjo T, et al.  (2004) Instruction of distinct CD4 T helper cell fates by different notch ligands on antigen-presenting cells. Cell  117(4): 515–526.  <ul class="find" data-citedArticleID="1021812" data-doi="10.1016/s0092-8674(04)00451-9"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00451-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Instruction+of+distinct+CD4+T+helper+cell+fates+by+different+notch+ligands+on+antigen-presenting+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Instruction+of+distinct+CD4+T+helper+cell+fates+by+different+notch+ligands+on+antigen-presenting+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001222-Quelle1" id="pone.0001222-Quelle1"></a>Quelle FW, Shimoda K, Thierfelder W, Fischer C, Kim A, et al.  (1995) Cloning of murine Stat6 and human Stat6, Stat proteins that are tyrosine phosphorylated in responses to IL-4 and IL-3 but are not required for mitogenesis. Mol Cell Biol  15(6): 3336–3343.  <ul class="find" data-citedArticleID="1021896"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cloning+of+murine+Stat6+and+human+Stat6%2C+Stat+proteins+that+are+tyrosine+phosphorylated+in+responses+to+IL-4+and+IL-3+but+are+not+required+for+mitogenesis.&amp;auth=&amp;atitle=Cloning+of+murine+Stat6+and+human+Stat6%2C+Stat+proteins+that+are+tyrosine+phosphorylated+in+responses+to+IL-4+and+IL-3+but+are+not+required+for+mitogenesis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+of+murine+Stat6+and+human+Stat6%2C+Stat+proteins+that+are+tyrosine+phosphorylated+in+responses+to+IL-4+and+IL-3+but+are+not+required+for+mitogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+of+murine+Stat6+and+human+Stat6%2C+Stat+proteins+that+are+tyrosine+phosphorylated+in+responses+to+IL-4+and+IL-3+but+are+not+required+for+mitogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001222-Liu1" id="pone.0001222-Liu1"></a>Liu YJ (2005) IPC: professional type 1 interferon-producing cells and plasmacytoid dendritic cell precursors. Annu Rev Immunol  23: 275–306.  <ul class="find" data-citedArticleID="1021876" data-doi="10.1146/annurev.immunol.23.021704.115633"><li><a href="http://dx.doi.org/10.1146/annurev.immunol.23.021704.115633" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=IPC%3A+professional+type+1+interferon-producing+cells+and+plasmacytoid+dendritic+cell+precursors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22IPC%3A+professional+type+1+interferon-producing+cells+and+plasmacytoid+dendritic+cell+precursors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001222-Kemper1" id="pone.0001222-Kemper1"></a>Kemper C, Chan AC, Green JM, Brett KA, Murphy KM, et al.  (2003) Activation of human CD4+ cells with CD3 and CD46 induces a T-regulatory cell 1 phenotype. Nature  421(6921): 388–392.  <ul class="find" data-citedArticleID="1021854" data-doi="10.1038/nature01315"><li><a href="http://dx.doi.org/10.1038/nature01315" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+of+human+CD4%2B+cells+with+CD3+and+CD46+induces+a+T-regulatory+cell+1+phenotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+of+human+CD4%2B+cells+with+CD3+and+CD46+induces+a+T-regulatory+cell+1+phenotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001222-Hayes1" id="pone.0001222-Hayes1"></a>Hayes AL, Smith C, Foxwell BM, Brennan FM (1999) CD45-induced tumor necrosis factor alpha production in monocytes is phosphatidylinositol 3-kinase-dependent and nuclear factor-kappaB-independent. J Biol Chem  274(47): 33455–33461.  <ul class="find" data-citedArticleID="1021840" data-doi="10.1074/jbc.274.47.33455"><li><a href="http://dx.doi.org/10.1074/jbc.274.47.33455" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD45-induced+tumor+necrosis+factor+alpha+production+in+monocytes+is+phosphatidylinositol+3-kinase-dependent+and+nuclear+factor-kappaB-independent." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD45-induced+tumor+necrosis+factor+alpha+production+in+monocytes+is+phosphatidylinositol+3-kinase-dependent+and+nuclear+factor-kappaB-independent.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001222-Liu2" id="pone.0001222-Liu2"></a>Liu ZG (2005) Molecular mechanism of TNF signaling and beyond. Cell Res  15(1): 24–27.  <ul class="find" data-citedArticleID="1021878" data-doi="10.1038/sj.cr.7290259"><li><a href="http://dx.doi.org/10.1038/sj.cr.7290259" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+mechanism+of+TNF+signaling+and+beyond." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+mechanism+of+TNF+signaling+and+beyond.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001222-Kang2" id="pone.0001222-Kang2"></a>Kang BY, Kim E, Kim TS (2005) Regulatory mechanisms and their therapeutic implications of interleukin-12 production in immune cells. Cell Signal  17(6): 665–673.  Epub 2005 Jan 2015. <ul class="find" data-citedArticleID="1021852" data-doi="10.1016/j.cellsig.2004.12.010"><li><a href="http://dx.doi.org/10.1016/j.cellsig.2004.12.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulatory+mechanisms+and+their+therapeutic+implications+of+interleukin-12+production+in+immune+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulatory+mechanisms+and+their+therapeutic+implications+of+interleukin-12+production+in+immune+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001222-Hohlfeld1" id="pone.0001222-Hohlfeld1"></a>Hohlfeld R, Wekerle H (2004) Autoimmune concepts of multiple sclerosis as a basis for selective immunotherapy: from pipe dreams to (therapeutic) pipelines. Proc Natl Acad Sci U S A  101: Suppl 214599–14606.  <ul class="find" data-citedArticleID="1021844" data-doi="10.1073/pnas.0404874101"><li><a href="http://dx.doi.org/10.1073/pnas.0404874101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Autoimmune+concepts+of+multiple+sclerosis+as+a+basis+for+selective+immunotherapy%3A+from+pipe+dreams+to+%28therapeutic%29+pipelines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Autoimmune+concepts+of+multiple+sclerosis+as+a+basis+for+selective+immunotherapy%3A+from+pipe+dreams+to+%28therapeutic%29+pipelines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001222-Galimberti1" id="pone.0001222-Galimberti1"></a>Galimberti D, Bresolin N, Scarpini E (2004) Chemokine network in multiple sclerosis: role in pathogenesis and targeting for future treatments. Expert Rev Neurother  4(3): 439–453.  <ul class="find" data-citedArticleID="1021826" data-doi="10.1586/14737175.4.3.439"><li><a href="http://dx.doi.org/10.1586/14737175.4.3.439" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemokine+network+in+multiple+sclerosis%3A+role+in+pathogenesis+and+targeting+for+future+treatments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemokine+network+in+multiple+sclerosis%3A+role+in+pathogenesis+and+targeting+for+future+treatments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001222-Neuhaus1" id="pone.0001222-Neuhaus1"></a>Neuhaus O, Archelos JJ, Hartung HP (2003) Immunomodulation in multiple sclerosis: from immunosuppression to neuroprotection. Trends Pharmacol Sci  24(3): 131–138.  <ul class="find" data-citedArticleID="1021888" data-doi="10.1016/s0165-6147(03)00028-2"><li><a href="http://dx.doi.org/10.1016/s0165-6147(03)00028-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunomodulation+in+multiple+sclerosis%3A+from+immunosuppression+to+neuroprotection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunomodulation+in+multiple+sclerosis%3A+from+immunosuppression+to+neuroprotection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001222-Stallwood1" id="pone.0001222-Stallwood1"></a>Stallwood Y, Briend E, Ray KM, Ward GA, Smith BJ, et al.  (2006) Small interfering RNA-mediated knockdown of notch ligands in primary CD4+ T cells and dendritic cells enhances cytokine production. J Immunol  177(2): 885–895.  <ul class="find" data-citedArticleID="1021912"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Small+interfering+RNA-mediated+knockdown+of+notch+ligands+in+primary+CD4%2B+T+cells+and+dendritic+cells+enhances+cytokine+production.&amp;auth=&amp;atitle=Small+interfering+RNA-mediated+knockdown+of+notch+ligands+in+primary+CD4%2B+T+cells+and+dendritic+cells+enhances+cytokine+production." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Small+interfering+RNA-mediated+knockdown+of+notch+ligands+in+primary+CD4%2B+T+cells+and+dendritic+cells+enhances+cytokine+production." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Small+interfering+RNA-mediated+knockdown+of+notch+ligands+in+primary+CD4%2B+T+cells+and+dendritic+cells+enhances+cytokine+production.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001222-John1" id="pone.0001222-John1"></a>John GR, Shankar SL, Shafit-Zagardo B, Massimi A, Lee SC, et al.  (2002) Multiple sclerosis: re-expression of a developmental pathway that restricts oligodendrocyte maturation. Nat Med  8(10): 1115–1121.  <ul class="find" data-citedArticleID="1021848" data-doi="10.1038/nm781"><li><a href="http://dx.doi.org/10.1038/nm781" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+sclerosis%3A+re-expression+of+a+developmental+pathway+that+restricts+oligodendrocyte+maturation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+sclerosis%3A+re-expression+of+a+developmental+pathway+that+restricts+oligodendrocyte+maturation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001222-Polman1" id="pone.0001222-Polman1"></a>Polman CH, Reingold SC, Edan G, Filippi M, Hartung HP, et al.  (2005) Diagnostic criteria for multiple sclerosis: 2005 revisions to the “McDonald Criteria”. Ann Neurol  58(6): 840–846.  <ul class="find" data-citedArticleID="1021894" data-doi="10.1002/ana.20703"><li><a href="http://dx.doi.org/10.1002/ana.20703" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diagnostic+criteria+for+multiple+sclerosis%3A+2005+revisions+to+the+%E2%80%9CMcDonald+Criteria%E2%80%9D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diagnostic+criteria+for+multiple+sclerosis%3A+2005+revisions+to+the+%E2%80%9CMcDonald+Criteria%E2%80%9D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001222-Vandesompele1" id="pone.0001222-Vandesompele1"></a>Vandesompele J, De Preter K, Pattyn F, Poppe B, Van Roy N, et al.  (2002) Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes. Genome Biol  3(7): RESEARCH0034.  Epub 2002 Jun 0018.  <ul class="find" data-citedArticleID="1021920"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Accurate+normalization+of+real-time+quantitative+RT-PCR+data+by+geometric+averaging+of+multiple+internal+control+genes.&amp;auth=&amp;atitle=Accurate+normalization+of+real-time+quantitative+RT-PCR+data+by+geometric+averaging+of+multiple+internal+control+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accurate+normalization+of+real-time+quantitative+RT-PCR+data+by+geometric+averaging+of+multiple+internal+control+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accurate+normalization+of+real-time+quantitative+RT-PCR+data+by+geometric+averaging+of+multiple+internal+control+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001222-Muller1" id="pone.0001222-Muller1"></a>Muller PY, Janovjak H, Miserez AR, Dobbie Z (2002) Processing of gene expression data generated by quantitative real-time RT-PCR. Biotechniques  32(6): 1372–1374, 1376, 1378–1379.  <ul class="find" data-citedArticleID="1021886"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Processing+of+gene+expression+data+generated+by+quantitative+real-time+RT-PCR.&amp;auth=&amp;atitle=Processing+of+gene+expression+data+generated+by+quantitative+real-time+RT-PCR." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Processing+of+gene+expression+data+generated+by+quantitative+real-time+RT-PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Processing+of+gene+expression+data+generated+by+quantitative+real-time+RT-PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001222-Moreno1" id="pone.0001222-Moreno1"></a>Moreno B, Hevia H, Santamaria M, Sepulcre J, Munoz J, et al.  (2006) Methylthioadenosine reverses brain autoimmune disease. AnnNeurol  60: 323–334.  <ul class="find" data-citedArticleID="1021884" data-doi="10.1002/ana.20895"><li><a href="http://dx.doi.org/10.1002/ana.20895" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methylthioadenosine+reverses+brain+autoimmune+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methylthioadenosine+reverses+brain+autoimmune+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001222-Glover1" id="pone.0001222-Glover1"></a>Glover F, Laguna M (1997) Tabu Search: Kluwer Academic Publishers.   <ul class="find-nolinks"></ul></li><li><span class="label">56.
              </span><a name="pone.0001222-Kullback1" id="pone.0001222-Kullback1"></a>Kullback S, Leibler RA (1951) On information and sufficiency. Annals of Mathematical Statics  22: 79–86.  <ul class="find" data-citedArticleID="1021864" data-doi="10.1214/aoms/1177729694"><li><a href="http://dx.doi.org/10.1214/aoms/1177729694" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+information+and+sufficiency." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+information+and+sufficiency.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.XML" value="176258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.PDF" value="558387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g001.PNG_L" value="1206678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g001.PNG_M" value="79610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g001.PNG_S" value="10324"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g001.TIF" value="1885346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g001.PNG_I" value="28086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t001.PNG_L" value="47857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t001.PNG_M" value="64710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t001.PNG_S" value="11095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t001.TIF" value="236642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t001.PNG_I" value="20797"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t002.PNG_L" value="265158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t002.PNG_M" value="144253"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t002.PNG_S" value="24675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t002.TIF" value="1222206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t002.PNG_I" value="177704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t003.PNG_L" value="163149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t003.PNG_M" value="163634"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t003.PNG_S" value="20911"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t003.TIF" value="740668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t003.PNG_I" value="94309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g002.PNG_L" value="339290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g002.PNG_M" value="37590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g002.PNG_S" value="13482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g002.TIF" value="2140414"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g002.PNG_I" value="21857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t004.PNG_L" value="115758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t004.PNG_M" value="103971"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t004.PNG_S" value="16207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t004.TIF" value="690406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t004.PNG_I" value="68741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g003.PNG_L" value="1497487"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g003.PNG_M" value="95378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g003.PNG_S" value="12792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g003.TIF" value="2186116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g003.PNG_I" value="35403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t005.PNG_L" value="384158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t005.PNG_M" value="161071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t005.PNG_S" value="18197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t005.TIF" value="1923204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t005.PNG_I" value="73425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g004.PNG_L" value="1440393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g004.PNG_M" value="92639"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g004.PNG_S" value="12816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g004.TIF" value="2108222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g004.PNG_I" value="34588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t006.PNG_L" value="399555"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t006.PNG_M" value="183724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t006.PNG_S" value="19611"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t006.TIF" value="2128502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.t006.PNG_I" value="88344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g005.PNG_L" value="4837068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g005.PNG_M" value="218042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g005.PNG_S" value="15943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g005.TIF" value="8344232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.g005.PNG_I" value="73959"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.e001.PNG" value="10874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.e001.TIF" value="26700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.s001.DOC" value="113152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.s002.DOC" value="199680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.s003.DOC" value="67072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.s004.DOC" value="75264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001222.s005.DOC" value="105984"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001222&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001222" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001222&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001222&volume=&issue=&title=A Network Analysis of the Human T-Cell Activation Gene Network Identifies Jagged1 as a Therapeutic Target for Autoimmune Diseases&author_name=Ricardo%20Palacios%2C%20Joaquin%20Goni%2C%20Ivan%20Martinez-Forero%2C%20Jaime%20Iranzo%2C%20Jorge%20Sepulcre%2C%20Ignacio%20Melero%2C%20Pablo%20Villoslada&start_page=1&end_page=15" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001222" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001222" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001222" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001222&amp;t=A%20Network%20Analysis%20of%20the%20Human%20T-Cell%20Activation%20Gene%20Network%20Identifies%20Jagged1%20as%20a%20Therapeutic%20Target%20for%20Autoimmune%20Diseases" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001222&title=A%20Network%20Analysis%20of%20the%20Human%20T-Cell%20Activation%20Gene%20Network%20Identifies%20Jagged1%20as%20a%20Therapeutic%20Target%20for%20Autoimmune%20Diseases&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001222&amp;title=A%20Network%20Analysis%20of%20the%20Human%20T-Cell%20Activation%20Gene%20Network%20Identifies%20Jagged1%20as%20a%20Therapeutic%20Target%20for%20Autoimmune%20Diseases" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001222" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001222&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Network Analysis of the Human T-Cell Activation Gene Network Identifies Jagged1 as a Therapeutic Target for Autoimmune Diseases';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001222';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Network%20Analysis%20of%20the%20Human%20T-Cell%20Activation%20Gene%20Network%20Identifies%20Jagged1%20as%20a%20Therapeutic%20Target%20for%20Autoimmune%20Diseases http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001222" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001222" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">




















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Autoimmune+diseases%22" title="Search for articles in the subject area:'Autoimmune diseases'"><div class="flagText">Autoimmune diseases</div></a>
              <div data-categoryid="20419" data-articleid="25276"
                   data-categoryname="Autoimmune diseases"
                   class="flagImage" title="Flag 'Autoimmune diseases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bayes+theorem%22" title="Search for articles in the subject area:'Bayes theorem'"><div class="flagText">Bayes theorem</div></a>
              <div data-categoryid="32251" data-articleid="25276"
                   data-categoryname="Bayes theorem"
                   class="flagImage" title="Flag 'Bayes theorem' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25276"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="47601" data-articleid="25276"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+networks%22" title="Search for articles in the subject area:'Genetic networks'"><div class="flagText">Genetic networks</div></a>
              <div data-categoryid="31807" data-articleid="25276"
                   data-categoryname="Genetic networks"
                   class="flagImage" title="Flag 'Genetic networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Multiple+sclerosis%22" title="Search for articles in the subject area:'Multiple sclerosis'"><div class="flagText">Multiple sclerosis</div></a>
              <div data-categoryid="33571" data-articleid="25276"
                   data-categoryname="Multiple sclerosis"
                   class="flagImage" title="Flag 'Multiple sclerosis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Network+analysis%22" title="Search for articles in the subject area:'Network analysis'"><div class="flagText">Network analysis</div></a>
              <div data-categoryid="3497" data-articleid="25276"
                   data-categoryname="Network analysis"
                   class="flagImage" title="Flag 'Network analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22T+cells%22" title="Search for articles in the subject area:'T cells'"><div class="flagText">T cells</div></a>
              <div data-categoryid="40147" data-articleid="25276"
                   data-categoryname="T cells"
                   class="flagImage" title="Flag 'T cells' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9637'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4483'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1507&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=43627">Understanding complex diseases ? </a><br>Posted by nvaluy</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
