

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Invasion and Persistence of a Selfish Gene in the Cnidaria</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000003"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000003" />

    <meta name="citation_title" content="Invasion and Persistence of a Selfish Gene in the Cnidaria"/>
    <meta itemprop="name" content="Invasion and Persistence of a Selfish Gene in the Cnidaria"/>

      <meta name="citation_author" content="Matthew R. Goddard"/>
            <meta name="citation_author_institution" content="School of Biological Sciences, University of Auckland, New Zealand"/>
            <meta name="citation_author_institution" content="National Environment Research Council Centre for Population Biology, Imperial College, London, United Kingdom"/>
      <meta name="citation_author" content="Jessica Leigh"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada"/>
      <meta name="citation_author" content="Andrew J Roger"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada"/>
      <meta name="citation_author" content="Andrew J Pemberton"/>
            <meta name="citation_author_institution" content="Marine Biological Association of the United Kingdom, Plymouth, United Kingdom"/>

    <meta name="citation_date" content="2006/12/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000003.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e3"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="1"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Homing endonuclease genes: the rise and fall and rise again of a selfish genetic element.; citation_author=A Burt; citation_author=V Koufopanou; citation_journal_title=Current Opinion in Genetics and Development; citation_volume=14; citation_number=1; citation_pages=609-615; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Outcrossed sex allows a selfish gene to invade yeast populations.; citation_author=MR Goddard; citation_author=D Greig; citation_author=A Burt; citation_journal_title=Proceedings of the Royal Society of London Series B, Containing papers of a Biological character:; citation_volume=268; citation_number=2; citation_pages=2537-2542; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The natural history of group I introns.; citation_author=P Haugen; citation_author=DM Simon; citation_author=D Bhattacharya; citation_journal_title=Trends in Genetics; citation_volume=21; citation_number=3; citation_pages=111-119; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Homing endonucleases: structural and functional insights into the catalysts of intron/intein mobility.; citation_author=BS Chevalier; citation_author=BL Stoddard; citation_journal_title=Nucleic Acids Research; citation_volume=29; citation_number=4; citation_pages=3757-3774; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Homing endouncleases: keeping the house in order.; citation_author=M Belfort; citation_author=RJ Roberts; citation_journal_title=Nucleic Acids Research; citation_volume=25; citation_number=5; citation_pages=3379-3388; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Recurrent invasion and extinction of a selfish gene.; citation_author=MR Goddard; citation_author=A Burt; citation_journal_title=Proceedings of the National Academy of Sciences: USA; citation_volume=96; citation_number=6; citation_pages=13880-13885; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Adaptation for horizontal transfer in a homing endonuclease.; citation_author=V Koufopanou; citation_author=MR Goddard; citation_author=A Burt; citation_journal_title=Molecular Biology and Evolution; citation_volume=19; citation_number=7; citation_pages=239-246; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Widespread horizontal transfer of mitochondrial genes in flowering plants.; citation_author=U Bergthorsson; citation_author=K Adams; citation_author=B Thomason; citation_author=J Palmer; citation_journal_title=Nature; citation_volume=424; citation_number=8; citation_pages=197-201; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Statistical modelling and analysis of the LAGLIDADG family of site-specific endonuclease of the HNH family.; citation_author=JZ Dalgaard; citation_author=AJ Klar; citation_author=MJ Moser; citation_author=WR Holley; citation_author=A Chatterjee; citation_journal_title=Nucleic Acids Research; citation_volume=25; citation_number=9; citation_pages=4626-4638; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The evolution of homing endonuclease genes and group I introns in nuclear rDNA.; citation_author=P Haugen; citation_author=V Reeb; citation_author=F Lutzoni; citation_author=D Bhattacharya; citation_journal_title=Molecular Biology and Evolution; citation_volume=21; citation_number=10; citation_pages=129-140; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Two mitochondrial group I introns in a metazoan, the sea anemone Metridium senile: One intron contains genes for subunits 1 and 3 of NADH dehydrogenase.; citation_author=CT Beagley; citation_author=NA Okada; citation_author=DR Wolstenholme; citation_journal_title=Proceedings of the National Academy of Sciences: USA; citation_volume=93; citation_number=11; citation_pages=5619-5623; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic relationships within the class anthozoa (Phylum Cnidaria) based in nuclear 18S rDNA sequences.; citation_author=EA Berntson; citation_author=SC France; citation_author=LS Mullineaux; citation_journal_title=Molecular Phylogenetics and Evolution; citation_volume=13; citation_number=12; citation_pages=417-433; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A phylogenetic study of the Anthozoa (phylum Cnidaria) based on morphological and molecular characters.; citation_author=JH Won; citation_author=BJ Rho; citation_author=JI Song; citation_journal_title=Coral Reefs; citation_volume=20; citation_number=13; citation_pages=39-50; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Naked corals: skeleton loss in Scleratinia.; citation_author=M Medina; citation_author=AG Collins; citation_author=TL Takaoka; citation_author=JV Kuehl; citation_author=JL Boore; citation_journal_title=Proceedings of the National Academy of Sciences: USA; citation_volume=103; citation_number=14; citation_pages=9096-9100; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Multiple comparisons of log-likelihood with applications to phylogenetics inference.; citation_author=H Shimodaira; citation_author=M Hasegawa; citation_journal_title=Molecular Biology and Evolution; citation_volume=16; citation_number=15; citation_pages=1114-1116; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Bayesian phylogenetic analysis of combined data.; citation_author=JAA Nylander; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_author=JL Nieves-Aldrey; citation_journal_title=Systematic Biology; citation_volume=53; citation_number=16; citation_pages=47-67; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Computing Bayes Factors using thermodynamic integration.; citation_author=N Lartillot; citation_author=H Philippe; citation_journal_title=Systematic Biology; citation_volume=55; citation_number=17; citation_pages=195-207; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A likelihood ratio test to detect conflicting phylogenetic signal.; citation_author=JP Huelsenbeck; citation_author=JJ Bull; citation_journal_title=Systematic Biology; citation_volume=45; citation_number=18; citation_pages=92-98; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Reductio ad absurdum: testing the evolutionary relationships of Ediacaran and Paleozoic problematic fossils using molecular divergence dates.; citation_author=B Waggoner; citation_author=AG Collins; citation_journal_title=Journal of Palaeontology; citation_volume=78; citation_number=19; citation_pages=51-61; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Inferring the historical patterns of biological evolution.; citation_author=M Pagel; citation_journal_title=Nature; citation_volume=401; citation_number=20; citation_pages=877-884; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Trees within trees: phylogeny and historical associations.; citation_author=RDM Page; citation_author=MA Charleston; citation_journal_title=Trends in Ecology and Evolution; citation_volume=13; citation_number=21; citation_pages=356-359; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Drosophila mediopunctata P elements: A new example of horizontal transfer.; citation_author=E Loreto; citation_author=V Valente; citation_author=A Zaha; citation_author=J Silva; citation_author=M Kidwell; citation_journal_title=Journal of Heredity; citation_volume=92; citation_number=22; citation_pages=375-381; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Recent horizontal transfer of mellifera subfamily Mariner transposons into insect lineages representing four different orders shows that selection acts only during horizontal transfer.; citation_author=D Lampe; citation_author=D Witherspoon; citation_author=F Soto-Adames; citation_author=H Robertson; citation_journal_title=Molecular Biology and Evolution; citation_volume=20; citation_number=23; citation_pages=554-562; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Massive horizontal transfer of mitochondrial genes from diverse land plant donors to the basal angiosperm Amborella.; citation_author=U Bergthorsson; citation_author=A Richardson; citation_author=G Young; citation_author=L Goertzen; citation_author=J Palmer; citation_journal_title=Proceedings of the National Academy of Sciences: USA; citation_volume=101; citation_number=24; citation_pages=17747-17752; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An mt+ gamete-specific nuclease that targets mt− chloroplasts during sexual reproduction in C. reinhardtii.; citation_author=Y Nishimura; citation_author=O Misumi; citation_author=K Kato; citation_author=N Inada; citation_author=T Higashiyama; citation_journal_title=Genes and Development; citation_volume=16; citation_number=25; citation_pages=1116-1128; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Animal mitochondrial DNA recombination revisited.; citation_author=A Rokas; citation_author=ED Ladoukakis; citation_author=E Zouros; citation_journal_title=Trends in Ecology and Evolution; citation_volume=18; citation_number=26; citation_pages=411-417; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Slow mitochindrial DNA sequence evolution in the Anthozoa (Cnidaria).; citation_author=TL Shearer; citation_author=MJHV Oppen; citation_author=SL Romano; citation_author=G Worheide; citation_journal_title=Molecular Ecology; citation_volume=11; citation_number=27; citation_pages=2475-2487; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Analysis of complete mitochondrial DNA sequences of three members of the Montastraea annularis coral species complex (Cnidaria, Anthozoa, Scleratinia).; citation_author=H Fukami; citation_author=N Knowlton; citation_journal_title=Coral Reefs; citation_volume=24; citation_number=28; citation_pages=410-417; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA sequence of the mitochondrial COXI gene have low levels of divergence among deep-sea octocorals (Cnidaria: Anthozoa).; citation_author=SC France; citation_author=LL Hoover; citation_journal_title=Hydrobiologia; citation_volume=471; citation_number=29; citation_pages=149-155; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA of the coral Sarcophyton glaucum contains a gene for a homologue of bacterial MutS: a possible case of gene transfer from the nucleus to the mitochondria.; citation_author=G Pont-Kingdom; citation_author=NA Okada; citation_author=JL Macfarlane; citation_author=CT Beagley; citation_author=CD Watkins-Sims; citation_journal_title=Journal of Molecular Evolution; citation_volume=46; citation_number=30; citation_pages=419-431; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=DNA extraction from the sea anemone (Cnidarina: Actinaria) tissues for molecular analyses.; citation_author=SM Pinto; citation_author=FMC Fernades-Matioli; citation_author=E Schlenz; citation_journal_title=Genetics and Molecular Biology; citation_volume=23; citation_number=31; citation_pages=601-604; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Amplification and direct sequencing of fungal ribosomal RNA genes for phylogenetics.; citation_author=TJ White; citation_author=T Bruns; citation_author=S Lee; citation_author=J Taylor; citation_number=32; citation_pages=315-322; citation_date=1990; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.; citation_author=JD Thompson; citation_author=TJ Gibson; citation_author=F Plewniak; citation_author=F Jeanmougin; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Research; citation_volume=25; citation_number=33; citation_pages=4876-4882; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=PAUP*.; citation_author=DL Swofford; citation_number=34; citation_date=2002; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=35; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=JAA Nylander; citation_number=36; citation_date=2004; " />
      <meta name="citation_reference" content="citation_author=S Wolfram; citation_number=37; citation_date=1996; citation_publisher=Wolfram Media; " />
      <meta name="citation_reference" content="citation_title=A simple, fast and accurate algorithm to estimate large phylogenies by maximum likelihood.; citation_author=S Guindon; citation_author=O Gascuel; citation_journal_title=Systematic Biology; citation_volume=52; citation_number=38; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing.; citation_author=HA Schmidt; citation_author=K Strimmer; citation_author=M Vingron; citation_author=A von Haeseler; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=39; citation_pages=502-504; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000003" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Invasion and Persistence of a Selfish Gene in the Cnidaria"/>
    <meta name="twitter:description" content="BackgroundHoming endonuclease genes (HEGs) are superfluous, but are capable of invading populations that mix alleles by biasing their inheritance patterns through gene conversion. One model suggests that their long-term persistence is achieved through recurrent invasion. This circumvents evolutionary degeneration, but requires reasonable rates of transfer between species to maintain purifying selection. Although HEGs are found in a variety of microbes, we found the previous discovery of this type of selfish genetic element in the mitochondria of a sea anemone surprising.Methods/Principal FindingsWe surveyed 29 species of Cnidaria for the presence of the COXI HEG. Statistical analyses provided evidence for HEG invasion. We also found that 96 individuals of Metridium senile, from five different locations in the UK, had identical HEG sequences. This lack of sequence divergence illustrates the stable nature of Anthozoan mitochondria. Our data suggests this HEG conforms to the recurrent invasion model of evolution.ConclusionsOrdinarily such low rates of HEG transfer would likely be insufficient to enable major invasion. However, the slow rate of Anthozoan mitochondrial change lengthens greatly the time to HEG degeneration: this significantly extends the periodicity of the HEG life-cycle. We suggest that a combination of very low substitution rates and rare transfers facilitated metazoan HEG invasion."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000003.g002"/>

  <meta property="og:title" content="Invasion and Persistence of a Selfish Gene in the Cnidaria" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000003" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=576'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5944'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8666&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000003">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000003" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000003&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Invasion and Persistence of a Selfish Gene in the Cnidaria
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthew R. Goddard
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:m.goddard@auckland.ac.nz">m.goddard@auckland.ac.nz</a>
            </p>

                <p>Affiliations:
                  School of Biological Sciences, University of Auckland, New Zealand, 
                  National Environment Research Council Centre for Population Biology, Imperial College, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jessica Leigh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrew J Roger, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrew J Pemberton
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Marine Biological Association of the United Kingdom, Plymouth, United Kingdom
                </p>

                <p>
                    Current address: Zoological Museum of the University of Zurich, Zurich, Switzerland
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 20, 2006</li>
    <li>DOI: 10.1371/journal.pone.0000003</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000003" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000003" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000003" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000003" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000003-g001" data-doi="info:doi/10.1371/journal.pone.0000003" data-uri="info:doi/10.1371/journal.pone.0000003.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000003.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000003-g002" data-doi="info:doi/10.1371/journal.pone.0000003" data-uri="info:doi/10.1371/journal.pone.0000003.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000003.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000003">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000003" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Homing endonuclease genes (HEGs) are superfluous, but are capable of invading populations that mix alleles by biasing their inheritance patterns through gene conversion. One model suggests that their long-term persistence is achieved through recurrent invasion. This circumvents evolutionary degeneration, but requires reasonable rates of transfer between species to maintain purifying selection. Although HEGs are found in a variety of microbes, we found the previous discovery of this type of selfish genetic element in the mitochondria of a sea anemone surprising.</p>


<h3>Methods/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We surveyed 29 species of Cnidaria for the presence of the <em>COXI</em> HEG. Statistical analyses provided evidence for HEG invasion. We also found that 96 individuals of <em>Metridium senile</em>, from five different locations in the UK, had identical HEG sequences. This lack of sequence divergence illustrates the stable nature of Anthozoan mitochondria. Our data suggests this HEG conforms to the recurrent invasion model of evolution.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Ordinarily such low rates of HEG transfer would likely be insufficient to enable major invasion. However, the slow rate of Anthozoan mitochondrial change lengthens greatly the time to HEG degeneration: this significantly extends the periodicity of the HEG life-cycle. We suggest that a combination of very low substitution rates and rare transfers facilitated metazoan HEG invasion.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Goddard MR, Leigh J, Roger AJ, Pemberton AJ (2006) Invasion and Persistence of a Selfish Gene in the Cnidaria. PLoS ONE 1(1):
          e3.
            doi:10.1371/journal.pone.0000003</p><p><strong>Academic Editor: </strong>James Fraser, University of Queensland, Australia</p><p><strong>Received:</strong> August 1, 2006; <strong>Accepted:</strong> August 12, 2006; <strong>Published:</strong> December 20, 2006</p><p><strong>Copyright:</strong> © 2006 Goddard et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The contributions of JL and AJR were supported by an operating grant (MOP-62809) from the Canadian Institutes for Health Research. This work was funded by the UK Natural Environment Research Council and a University of Auckland Research grant (3606223) to MG.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Homing endonuclease genes (HEGs) are superfluous, but invade populations because they cheat the rules of Mendelian inheritance and over-represent themselves in the next generation <a href="#pone.0000003-Burt1">[1]</a>–<a href="#pone.0000003-Haugen1">[3]</a>. Each homing endonuclease (HE), the protein product of an HEG, recognises and cuts a specific 15–30 bp DNA recognition sequence which is typically unique within a genome <a href="#pone.0000003-Chevalier1">[4]</a>, <a href="#pone.0000003-Belfort1">[5]</a>. HEGs are located in the middle of their own recognition sequence and so their presence splits this site in two and protects against chromosome cleavage. However, in individuals heterozygous for any particular HEG, the uninterrupted recognition sequence is encountered and cut producing a potentially lethal broken chromosome <a href="#pone.0000003-Chevalier1">[4]</a>. Chromosome breaks are typically repaired using the homologous area of the intact chromosome as a template, and the HEG is copied onto the other chromosome as a consequence of the repair procedure <a href="#pone.0000003-Burt1">[1]</a>. This ‘homing’ process allows HEGs to effect super-Mendelian inheritance and increase in frequency <a href="#pone.0000003-Goddard1">[2]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The infective nature of HEGs allows them to quickly invade species <a href="#pone.0000003-Goddard1">[2]</a>, but the lack of homing opportunities near and at fixation means that selection for HEG function will become substantially relaxed. One evolutionary model suggests that purifying selection may only be maintained, and thus degeneration circumvented, by restoring homing events <a href="#pone.0000003-Goddard2">[6]</a>. The recurrent invasion of closely related uninfected species, via horizontal transfer, is a hypothesis that permits the longer-term persistence of HEGs by maintaining purifying selection. This cyclical evolutionary model allows major invasions of closely related species, and predicts the patchy distribution of both functional and non-functional HEGs, and incongruence between host-genome and HEG phylogenies <a href="#pone.0000003-Goddard2">[6]</a>–<a href="#pone.0000003-Bergthorsson1">[8]</a>. HEGs have been reported from a wide array of species and comparative data from phage, archaea, fungi, algae and plants support this HEG recurrent invasion model (see <a href="#pone.0000003-Burt1">[1]</a> and references within). One clear prediction of this model is that selection will produce HEGs that are adapted for major invasion by having recognition sites that are conserved among taxa. This seems to be the case since extant HEGs have recognition sequences usually targeted toward essential and sequentially conserved genes <a href="#pone.0000003-Koufopanou1">[7]</a>. The presence of HEGs in the nuclear genomes of eukaryotes, archebacteria and phage correlates with the fact that alleles may be bought together in each of these groups by mating, conjugation and co-infection respectively, and thus permit HEG homing and invasion <a href="#pone.0000003-Burt1">[1]</a>, <a href="#pone.0000003-Haugen1">[3]</a>, <a href="#pone.0000003-Belfort1">[5]</a>, <a href="#pone.0000003-Dalgaard1">[9]</a>, <a href="#pone.0000003-Haugen2">[10]</a>. If HEG-plus and HEG-minus alleles are not bought together there will be no opportunities for homing and spread within a species. In general HEGs will not be able to persist in strictly asexual genomes <a href="#pone.0000003-Goddard1">[2]</a>. Thus, it appears that HEGs have two requirements for major invasion: 1, a reasonable opportunity for homing; and 2, relatively frequent transfer to naive hosts to maintain purifying selection and escape degeneration.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>An HEG has been discovered in the cytochrome oxidase I (<em>COXI</em>) gene of the mitochondria of the sea anemone <em>Metridium senile</em> 
            <a href="#pone.0000003-Beagley1">[11]</a>. This HEG is located in the middle of a self splicing group I intron (as is common) which allows excision from the host gene at the RNA level: this allows the production of a functional COXI. This original report of an HEG among metazoans is one of an extremely small number of known metazoan mitochondrial introns. This discovery is puzzling. The mitochondria of animals are generally thought to be uni-parentally inherited, and not to recombine: this should deny opportunities for homing. Even if within-species HEG spread were possible for some reason, then escape from long-term degeneration would seem unlikely as one imagines horizontal transfer between metazoan germ-lines to be a prohibitively rare event.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>We investigated the distribution and evolutionary genetics of this HEG within the Cnidaria in order to test the applicability of the recurrent invasion model <a href="#pone.0000003-Goddard2">[6]</a> in metazoans. We present evidence which strongly suggests that this HEG invaded and then spread within the Actinaria (true sea anemones).</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Phylogenies and HEG distribution</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The nuclear 18S, 5.8S and ITS sequence data were assumed to represent the evolutionary history of the Cnidarian host genome (accession numbers 18S: DQ831237 - DQ831296; ITS1: DQ831297 - DQ831306; 5.8S: DQ831307 - DQ831322; ITS2: DQ831323 - DQ831332). After alignment and correct model selection (see <a href="#pone.0000003.s001">Supporting Material 1</a>), Bayesian, likelihood and parsimony optimality criteria produced extremely similar phylogenetic trees; the tree recovered using Bayesian methods is shown in <a href="#pone-0000003-g001">figure 1</a>. The partitioning of the taxa in this phylogeny agrees with previous estimates of Cnidarian phylogeny <a href="#pone.0000003-Berntson1">[12]</a>–<a href="#pone.0000003-Medina1">[14]</a>, and with their taxonomic classifications.</p>
<div class="figure" id="pone-0000003-g001"><div class="img"><a name="pone-0000003-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000003.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000003" data-uri="info:doi/10.1371/journal.pone.0000003.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000003.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000003.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000003.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000003.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000003.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000003.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000003.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000003.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000003.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Cnidaria host genome phylogeny reconstructed using Bayesian methods from 18S, 5.8S and ITS data, and is a 50% majority rule consensus taken from the MCMC analyses after plateau (i.e. ‘burn-in’ trees discarded).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The Bayesian posterior probabilities are given for each node.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p2" name="article1.body1.sec2.sec1.fig1.caption1.p2"></a><p>Taxa containing a HEG are highlighted; the non-functional HEG is boxed.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p3" name="article1.body1.sec2.sec1.fig1.caption1.p3"></a><p>Taxonomic classification is also distinguished; O(A)  =  Octocorallia (Alcyonaria); C  =  Corrimorpharia; S  =  Scleratinia; Z  =  Zoanthiniaria.</p>
<span>doi:10.1371/journal.pone.0000003.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>All twenty nine taxa produced a single <em>COXI</em> PCR product, and seventeen of these harboured the correct sized larger product indicating the presence of an HEG in the <em>COXI</em> gene. When HEG presence is mapped onto the host phylogeny the distribution appears non-random, with a strong cluster in members of the Actinaria and Corrimorpharia (<a href="#pone-0000003-g001">figure 1</a>). Such a pattern seems to minimise the change of HEG status across the tree, and implies that horizontal transfer has not played a large part in this HEG's evolutionary history. Indeed, if we assume that HEG gains and losses are equally likely, then the inferred three character changes across the tree (two gains and one loss) are significantly less than that observed when intron status is randomised (<em>P</em>&lt;0.01; <em>n</em> = 100 randomisations). This clumped distribution, and simple test, suggests this HEG has not been subject to rampant horizontal transfer across all these taxa, and, for some reason, that the HEG is restricted to specific clades.</p>


<h4>Tests for HEG horizontal transfer</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A more robust test for horizontal transfer makes use of sequence data from both areas of interest. We obtained sequence for fourteen of the HEGs and their flanking group I introns (accession numbers DQ831333 - DQ831345). All HEGs were putatively functional, with the exception of that found in <em>Megalactis sp</em>. which had four deletions of between four and 12 bp which destroyed the HEG reading frame (one deletion even removes the HEG start codon). Overall the HEG sequences showed extremely good alignment allowing un-ambiguous translation for all but the non-functional version in <em>Megalactis sp</em>. The protein motifs which place this HEG in the LAGLIDADG family <a href="#pone.0000003-Dalgaard1">[9]</a> may be readily identified. Phylogenies reconstructed exclusively from the Actinarian derived HEGs using Bayesian, likelihood and parsimony criteria all agreed (see <a href="#pone.0000003.s001">Supporting Material 1</a> for model details). A phylogeny reconstructed from the translated amino acid data was also congruent with the DNA derived phylogenies. Statistical comparisons of host and HEG data allowed us clear tests for differences in evolutionary histories. An Actinaria host tree was reconstructed using only those taxa for which we have HEG sequence (see <a href="#pone-0000003-g002">figure 2A</a>). First, this was compared to the HEG tree using the likelihood based Shimodaira-Hasegawa (SH) test as implemented by PAUP* (full optimisation, 1000 replicates). Comparisons performed on each relevant data set returned a <em>P</em> value of &lt;0.001 in each case (four tests total, bootstrap and best trees tested on both data partitions) which suggested that the two data sets contained different phylogenetic signals. One criticism with the SH tests, as carried out here, is that only two topologies were considered at a time: the test was really designed to consider a large multiplicity of trees <a href="#pone.0000003-Shimodaira1">[15]</a>. In addition, the support for some of the nodes seen in <a href="#pone-0000003-g002">figure 2</a> is low – ideally tests which do not rely on fixed topologies should be employed.</p>
<div class="figure" id="pone-0000003-g002"><div class="img"><a name="pone-0000003-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000003.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000003" data-uri="info:doi/10.1371/journal.pone.0000003.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000003.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000003.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000003.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000003.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000003.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000003.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000003.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000003.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000003.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Comparison of Actinarian and HEG phylogenies.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>
                     <strong>A.</strong> The numbers above each branch are the Bayesian posterior probabilities (again after burn-in trees discarded), and those below the branch are the support generated from a likelihood bootstrap analysis (n = 1000).</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p2" name="article1.body1.sec2.sec2.fig1.caption1.p2"></a><p>The HEG tree is unrooted (it is unknown) and displayed in such a way as to minimise the differences with the host tree.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p3" name="article1.body1.sec2.sec2.fig1.caption1.p3"></a><p>
                     <strong>B.</strong> The null distribution for the likelihood ratio test generated by evaluating δ for 1000 non-parametric bootstrap resampled data sets, 50% of which are drawn entirely from the host data and 50% from the HEG data.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p4" name="article1.body1.sec2.sec2.fig1.caption1.p4"></a><p>The likelihood ratio for the observed partition is shown.</p>
<span>doi:10.1371/journal.pone.0000003.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>A more appropriate test for incongruence uses Bayes factors (BF) to compare evolutionary histories <a href="#pone.0000003-Nylander1">[16]</a>. The marginal log-likelihood of an analysis where the HEG and the host partitions were forced to share the same topology (the ‘one tree’ model) was subtracted from the marginal log-likelihood of an analysis which allowed a separate tree from each partition (the ‘two tree’ model): this yielded an estimate of the log-Bayes factor. Four separate MCMCMC runs of the ‘one’ tree versus ‘two tree’ models were completed, and the resulting average log-BF was 23.48. This test circumvents the problems mentioned above, and provides strong evidence against accepting the simpler one tree model since 2(log-BF) &gt;10 (see <a href="#pone.0000003-Nylander1">[16]</a>): the two tree model is clearly favoured. However, some caution is warranted in interpreting this result, as a recent analysis suggests that the harmonic mean estimate of the marginal log-likelihood for these Bayes factor calculations may be biased in favour of more complex models <a href="#pone.0000003-Lartillot1">[17]</a>.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Thus, as a final test for incongruence between HEG and host histories, we used a non-parametric version of Huelsenbeck and Bull's likelihood ratio test for conflicting phylogenetic signal <a href="#pone.0000003-Huelsenbeck1">[18]</a>. In this test, a likelihood ratio test statistic is calculated for the ‘two tree’ versus ‘one tree’ models, and compared to a null distribution generated by non-parametric bootstrapping (see <a href="#s4">Methods</a>). For our data, the test statistic, δ = 27.88, was markedly larger than any value from the null bootstrap distribution. This allows the null hypothesis to be firmly rejected with <em>p</em>&lt;0.001 (<a href="#pone-0000003-g002">figure 2B</a>). Collectively these three tests show that the evolutionary histories of the host and HEG data significantly differ, and therefore provide evidence for the horizontal transfer of this HEG among these actinarians.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Having demonstrated that HEG horizontal transfer has very likely occurred, we were keen to estimate a rate of HEG movement: is this a frequent or rare event? Recent work based on fossil data suggests that an approximate age for the basal split of the host phylogeny in <a href="#pone-0000003-g001">figure 1</a> is 500 MY <a href="#pone.0000003-Waggoner1">[19]</a>. Using branch lengths calculated by maximum likelihood, with a molecular clock enforced, the total time encompassed by the most likely resolution of the tree seen in <a href="#pone-0000003-g001">figure 1</a> is approximately 5500 MY (estimated by summing the lengths of all the branches and converting to MY). We examined the dynamics of recurrent HEG gain and loss across this phylogeny using maximum likelihood methods as implemented by Pagel's Multistate (v0.8) program <a href="#pone.0000003-Pagel1">[20]</a>. We found no significant difference between a one parameter model where rates of HEG gain and loss were constrained to be the same (ln likelihood = 13.478; 1 d.f.), and a two parameter model where rates were free to vary (12.691; 2 d.f.). A one parameter model which constrained gains to be zero was also significantly worse than a model which included gains (16.218, 1 d.f., <em>p</em>&lt;0.025). The model where gains and losses were constrained to be equal produced an extremely tentative estimate of one transition (either gain or loss) every 500 MY, with 2 log likelihood support limits (equivalent to 95% confidence intervals) between once every 100 MY and 0 (never). Reconstructing specific transfer events between lineages is an extremely hard problem <a href="#pone.0000003-Page1">[21]</a>, but this method offers an alternative approach and provides a likely rate of movement. This very approximate estimate suggest that around 11 gain and loss cycles occurred across this phylogeny, and this equates to a very low horizontal transfer rate of 0.002 events per MY, with bounds from 0–0.01.</p>


<h4>HEG population study</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>One little studied area of HEG biology concerns their intra-population variance. All of the 95 <em>M. senile</em> individuals (from four UK collection sites, see <a href="#pone.0000003.s001">Supporting Material 1</a>) contained a complete HEG as indicated by the presence of an appropriately sized PCR product. All of the 51 different one-way sequences we obtained (approximately half and half 5′ and 3′) were identical to one another and identical to the published sequence from a single individual from the west coast of the USA <a href="#pone.0000003-Beagley1">[11]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The observation of functional, non-functional and absent HEG states, combined with our inference of horizontal transfer, strongly suggests that this HEG, like many others, conforms to a cyclical evolutionary model. These inferences suggest that this HEG invaded the Actinaria <a href="#pone.0000003-Goddard2">[6]</a>. It is possible that whole mitochondrial genomes moved between species, but our supposition of HEG transfer is directly corroborated by a very recent study examining a number of complete anthozoan mitochondria: Medina et al <a href="#pone.0000003-Medina1">[14]</a> concluded this HEG (and not whole mitochondrial genomes) has been gained many times by the Hexacorallia (see <a href="#pone-0000003-g001">figure 1</a>). Our tentative estimate for a rate HEGs transfer among metazoans is approximately two orders of magnitude lower than that for HEGs in the mitochondria and nucleus of Saccharomycete yeasts <a href="#pone.0000003-Goddard2">[6]</a>, <a href="#pone.0000003-Koufopanou1">[7]</a>. There are other reports of horizontal transfer events among metazoans, notably in insects because of the dynamics of other types of selfish element (e.g., TEs and P-elements in <em>Drosophila</em> 
            <a href="#pone.0000003-Loreto1">[22]</a>, <a href="#pone.0000003-Lampe1">[23]</a>). There are also reports of large mitochondrial horizontal transfer events and intron invasions of plants <a href="#pone.0000003-Bergthorsson1">[8]</a>, <a href="#pone.0000003-Bergthorsson2">[24]</a>. Our conclusions are therefore not only consistent with other data concerning this HEG <a href="#pone.0000003-Medina1">[14]</a>, but also with the idea that horizontal transfer and intron invasion occurs among multicellular organisms. These data suggest that, surprisingly, horizontal transfer rates on the order of once every 100 MY may be frequent enough to allow the invasion, cycling and persistence of HEGs within the Cnidaria.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Once transferred, the <em>COXI</em> HEG must home in order to restore purifying selection, and this requires that the mitochondria of the recipients must, at least on occasion, mix. The mitochondria of both mating types of the algae <em>Chlamydomonas</em> co-exist briefly after zygote formation before one is degraded, and this provides enough opportunity for an HEG to home <a href="#pone.0000003-Nishimura1">[25]</a> and therefore presumably invade a species where uni-parental mitochondrial inheritance seems the norm. Although we know of no direct evidence from sea anemones, mitochondrial leakage has been observed in a wide variety of animal species and may provide a window of opportunity for HEG homing <a href="#pone.0000003-Rokas1">[26]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Based on the distribution of this HEG (see <a href="#pone-0000003-g001">figure 1</a>), one has to infer that either the ancestor of the Cnidaria harboured an HEG, and that it has been lost a number of times, yet retained for some reason in the Hexacorallia, or else infer that the HEG was gained from some other lineage. Dalgaard et al <a href="#pone.0000003-Dalgaard1">[9]</a> have reconstructed the relationships between over 130 HEGs, and the most closely related HEG to that in <em>M. senile</em> (apart from the sequences reported here) is found in <em>Neurospora crassa</em>. A seemingly strange best estimate would be that sea anemones received this HEG from fungi. However, this suggestion directly parallels data from angiosperms, which also appear to have gained a mitochondrial <em>COXI</em> HEG from fungi <a href="#pone.0000003-Bergthorsson1">[8]</a>, <a href="#pone.0000003-Bergthorsson2">[24]</a> which then rapidly invaded the angiosperms via homing and horizontal transfer events.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The HEG survey results suggest that it is at or near fixation within <em>M. senile</em>. The lack of sequence divergence may imply a recent invasion and sweep within this species. However, it must be pointed out that the mitochondria of anthozoans are extraordinarily stable <a href="#pone.0000003-Shearer1">[27]</a>, with rates of evolution around only 0.03% per MY <a href="#pone.0000003-Fukami1">[28]</a>. Moreover, the <em>COXI</em> genes of Anthozoans have extremely low levels of divergence <a href="#pone.0000003-France1">[29]</a>, and one explanation for this highlights the discovery of a homologue to a mismatch repair gene <a href="#pone.0000003-PontKingdom1">[30]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Overall these data suggest that the HEG within the mitochondria of the Actinaria behaves in a similar evolutionary manner to HEGs in a variety of other organisms <a href="#pone.0000003-Burt1">[1]</a>, <a href="#pone.0000003-Haugen1">[3]</a>, and conforms to the recurrent invasion model <a href="#pone.0000003-Goddard2">[6]</a>. The low rates of horizontal transfer among metazoans will likely ensure that HEGs degenerate long before an opportunity to ‘escape’ arises, and mean that major clade-wide invasion is prevented. It might be that HEGs were able to invade anthozoan mitochondria only because of their extremely slow rates of substitution: this greatly extends the time to HEG degeneration, and damps down the periodicity of the HEG recurrent invasion cycle. It is only under these sluggish conditions that very rare horizontal transfer events are sufficient for long term HEG persistence and invasion. In general it seems that there need be a minimum lower bound on the rate of horizontal transfer, and frequency of homing opportunities, which enable HEGs to invade and persist in any clade. The likelihood of these events appears greater in simpler single celled organisms, but it also seems that whist the opportunities for homing and horizontal transfer are less in higher organisms, this does not necessarily put them out of HEGs reach. It is possible that HEGs were only able to invade sea anemones because of the slow evolutionary rate of their mitochondrial DNA <a href="#pone.0000003-Shearer1">[27]</a>, <a href="#pone.0000003-Fukami1">[28]</a>. The extent to which HEGs have not invaded other metazoans may either be a lack of opportunity, or that most metazoans have faster substitution rates, which mean the more rapid degeneration of HEGs.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sample collection</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Twenty-nine different species of Cnidaria were sampled (see <a href="#pone.0000003.s002">Table S1</a>); these spanned the Anthozoa and Hydrozoa, but as a result of the tentative distribution of introns reported by Beagley et al <a href="#pone.0000003-Beagley1">[11]</a> we mainly concentrated within the Anthozoa:Actinaria (true sea anemones). Samples were predominantly from the south coast of the UK but also included tropical examples. To investigate the within-species variation of HEGs we focused on <em>Metridium senile</em>. Ninety-five individuals from four different marinas on the south coast of the UK were sampled (see <a href="#pone.0000003.s001">Supporting Material 1</a> for details).</p>


<h4>Molecular methods</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>When necessary DNA was isolated from tissues using the method described by Pinto et al <a href="#pone.0000003-Pinto1">[31]</a>. HEG presence and absence was scored by amplification with the PCR primers described in Beagley et al <a href="#pone.0000003-Beagley1">[11]</a>; these were designed such that the size of the product indicated the HEG status of each sample. The three 5′ most regions of the nuclear 18S and the ITS1-5.8S-ITS2 region were also amplified for each taxon using the primers described by White et al <a href="#pone.0000003-White1">[32]</a>. PCR products were directly sequenced and analysed with an ABI 3700 instrument. Complete and partial two-way sequences of the <em>COXI</em> intron were obtained for 14 of the intron containing taxa. Fifty-one one way <em>COXI</em> intron sequence were obtained from the 95 <em>Metridium senile</em> samples. Partial 18S sequence was obtained for all taxa but replaced with the more complete genbank 18S sequences AJ133552, AF254377, AJ877002, U19553, AY372249, AF052892 and AF052889 for <em>Actinia equina, Edwardsia gilbertensis, Lophelia pertusa, Tubularia indivisa, Sinularia sp, Palythoa variabilis</em> and <em>Metridium senile</em> respectively.</p>


<h4>Analyses</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>After manual inspection and editing where necessary, sequences were aligned using ClustalX <a href="#pone.0000003-Thompson1">[33]</a>; the intron alignment was conducted by taking into account secondary structure following Goddard and Burt <a href="#pone.0000003-Goddard2">[6]</a>. The resulting alignments were inspected and ambiguous areas were removed before phylogenetic analyses. The 18S data alone were insufficient to resolve some shallower nodes; we therefore added 5.8S and ITS data for these taxa since there was no conflict between data partitions. PAUP* (v 4.0b10) <a href="#pone.0000003-Swofford1">[34]</a> and MrBayes (3.1.2) <a href="#pone.0000003-Ronquist1">[35]</a> for Windows were used for phylogenetic reconstruction. The appropriate likelihood models were selected using MrModeltest (v2.2) <a href="#pone.0000003-Nylander2">[36]</a>, and are shown in <a href="#pone.0000003.s001">Supporting Material 1</a>. Bayesian, likelihood and parsimony methods were used to analyse the host and HEG data sets. The loss test was conducted by constructing randomised intron assignments with Mathematica (v5.2) <a href="#pone.0000003-Wolfram1">[37]</a> and then using PAUP* to score the number of losses. Multistate (v0.8) <a href="#pone.0000003-Pagel1">[20]</a> was used to estimate the most likely transition rates for HEG gain and loss across the host phylogeny.</p>


<h4>Phylogenetic congruence tests</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>To assess congruence between data partitions three methods were used. First, as a heuristic test, reciprocal Shimodaira-Hasegawa (SH) tests <a href="#pone.0000003-Shimodaira1">[15]</a>, as implemented in PAUP*, were used to test whether optimal trees from different partitions were incongruent. Congruence was more rigorously tested using a Bayesian model selection framework (Bayes factors) <a href="#pone.0000003-Nylander1">[16]</a> and lastly, a modified version of the likelihood ratio test proposed by Huelsenbeck and Bull <a href="#pone.0000003-Huelsenbeck1">[18]</a> (Leigh, Susko and Roger, unpublished) was employed.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>The Bayes factor model selection was carried out with MrBayes as follows. Two models were tested. Under the ‘one tree’ model, the two data partitions were constrained to shared a single topology; however, the model parameters and branch-lengths were unlinked across the partitions for the Metropolis-coupled Markov chain Monte Carlo (MCMCMC) analysis. The ‘two tree’ model was identical to the ‘one tree’ model, except the tree topology was also unlinked, allowing tree-space to be separately explored for each data partition. The harmonic mean likelihood from the Metropolis-coupled Markov chain Monte Carlo (MCMCMC) iterations (after burnin was discarded) was used as an approximation to the marginal log-likelihood of the data <a href="#pone.0000003-Nylander1">[16]</a>. For both the ‘one-tree’ and ‘two tree’ analyses, four independent runs from random starting trees were completed: two with one million MCMCMC iterations and two with two million MCMCMC iterations. Inspection of log-likelihood plots indicated that stationarity was achieved &lt;50,000 iterations, and a conservative burnin of 150,000 iterations was discarded. Convergence was further assessed by monitoring the standard deviation in split frequencies between MCMCMC runs from different random starting trees. In all cases, at the end of the runs these standard errors were &lt;0.01. Log-Bayes factors were computed by taking the differences in the harmonic mean log-likelihoods averaged over all four runs between the ‘two tree’ model and the ‘one tree’ model.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>For the likelihood ratio test, the log-likelihood of two data sets (e.g. A and B) is maximized under the null hypothesis where A and B share a single tree (<em>T<sub>AB</sub>
               </em>) but each gene is allowed to have its own model and branch-length parameters (θ<sub>A</sub> and θ<sub>B</sub>). The log-likelihood of this hypothesis is given by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000003.e001&amp;representation=PNG" class="inline-graphic"></span><br>Under the alternative hypothesis each data set is allowed to have a separate tree (<em>T<sub>A</sub>
               </em> and <em>T<sub>B</sub>
               </em>) and its own model and branch-length parameters with maximum log-likelihood given by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000003.e002&amp;representation=PNG" class="inline-graphic"></span><br>The log-likelihood ratio statistic (δ) is then defined as:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000003.e003&amp;representation=PNG" class="inline-graphic"></span><br>A null distribution for this statistic is generated by evaluating δ for 1000 non-parametric bootstrap resampled data sets, 50% of which are drawn entirely from data set A and 50% from dataset B. This procedure was accomplished with a software tool, <span class="small-caps">CONCATERPILLAR</span> (Leigh, Susko and Roger, unpublished) that calculates trees and likelihoods under the General Time Reversible plus invariable sites plus gamma model (GTR+I+Γ with the programs PHYML <a href="#pone.0000003-Guindon1">[38]</a> and TREE-PUZZLE version 5.2 <a href="#pone.0000003-Schmidt1">[39]</a>).</p>

</div>

<div id="section5" class="section"><a id="s6" name="s6" toc="s6" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000003"></div><a name="pone.0000003.s001" id="pone.0000003.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000003.s001">Supporting Material 1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Details concerning the intra-specific sampling of <em>Metridium senile</em> and information concerning the models used in phylogenetic reconstruction, which were derived using Mr Model Test</p>
<p class="siDoi">doi:10.1371/journal.pone.0000003.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000003.s002" id="pone.0000003.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000003.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">
                  <a href="#pone.0000003.s002">Table S1</a> shows the taxonomy, sampling location and personnel involved in the collection and identification of samples. The taxonomic hierarchy is mainly based on Fautin, Daphne G. 2005. Hexacorallians of the World. http://hercules.kgs.ku.edu/hexacoral/ane​mone2/index.cfm.‘-’ indicates that this taxonomical level is not in common use in this taxa. Collection locations: ‘MBA tanks’ - sea water tanks of the Marine Biological Association of the UK, Plymouth; species may have recruited naturally from the circulating sea water system or been brought in from local shores by staff; ‘MBA standard haul’ is an offshore benthic trawl as described in Genner, M.J. et al Proc. R. Soc. Lond. B (2004) 271, 655-661; ‘Pet store’ - sample purchased from commercial pet store therefore original collection location is unknown. Personnel: ‘JB’ - John Bishop, MBA; ‘MD’ - Marymegan Daly, U. of Kansas, USA; ‘JD’ - Jo Davy, MBA; ‘SD’ - Simon Davy, MBA; ‘MD’ - Marie Le Goff, Southampton Oceanography Centre, UK; ‘KH’ - Keith Hiscock, MBA; and ‘KN’ - Ken Neal, MBA. All other collections and/or identification by Andrew Pemberton (AP) <em>Aglaophenia kirchenpaueri</em> - this species is known to occur at this location; however, the specimen was collected outside the reproductive season so lacked the gonads required for certain identification to species level.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000003.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank John Bishop, Marymegan Daly, Jo Davy, Simon Davy, Marie Le Goff, Keith Hiscock and Ken Neal for providing samples and helping with identification. We thank Callum Brandon for technical assistance and David Bryant for discussions regarding phylogenetic congruence testing.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MG. Analyzed the data: AR MG JL. Contributed reagents/materials/analysis tools: AR MG AP. Wrote the paper: MG AP</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000003-Burt1" id="pone.0000003-Burt1"></a>Burt A, Koufopanou V (2004) Homing endonuclease genes: the rise and fall and rise again of a selfish genetic element. Current Opinion in Genetics and Development  14: 609–615.  <ul class="find" data-citedArticleID="913008" data-doi="10.1016/j.gde.2004.09.010"><li><a href="http://dx.doi.org/10.1016/j.gde.2004.09.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homing+endonuclease+genes%3A+the+rise+and+fall+and+rise+again+of+a+selfish+genetic+element." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homing+endonuclease+genes%3A+the+rise+and+fall+and+rise+again+of+a+selfish+genetic+element.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000003-Goddard1" id="pone.0000003-Goddard1"></a>Goddard MR, Greig D, Burt A (2001) Outcrossed sex allows a selfish gene to invade yeast populations. Proceedings of the Royal Society of London Series B, Containing papers of a Biological character:  268: 2537–2542.  <ul class="find" data-citedArticleID="913018" data-doi="10.1098/rspb.2001.1830"><li><a href="http://dx.doi.org/10.1098/rspb.2001.1830" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Outcrossed+sex+allows+a+selfish+gene+to+invade+yeast+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Outcrossed+sex+allows+a+selfish+gene+to+invade+yeast+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000003-Haugen1" id="pone.0000003-Haugen1"></a>Haugen P, Simon DM, Bhattacharya D (2005) The natural history of group I introns. Trends in Genetics  21: 111–119.  <ul class="find" data-citedArticleID="913024" data-doi="10.1016/j.tig.2004.12.007"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.12.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+natural+history+of+group+I+introns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+natural+history+of+group+I+introns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000003-Chevalier1" id="pone.0000003-Chevalier1"></a>Chevalier BS, Stoddard BL (2001) Homing endonucleases: structural and functional insights into the catalysts of intron/intein mobility. Nucleic Acids Research  29: 3757–3774.  <ul class="find" data-citedArticleID="913010" data-doi="10.1093/nar/29.18.3757"><li><a href="http://dx.doi.org/10.1093/nar/29.18.3757" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homing+endonucleases%3A+structural+and+functional+insights+into+the+catalysts+of+intron%2Fintein+mobility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homing+endonucleases%3A+structural+and+functional+insights+into+the+catalysts+of+intron%2Fintein+mobility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000003-Belfort1" id="pone.0000003-Belfort1"></a>Belfort M, Roberts RJ (1997) Homing endouncleases: keeping the house in order. Nucleic Acids Research  25: 3379–3388.  <ul class="find" data-citedArticleID="913000" data-doi="10.1093/nar/25.17.3379"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3379" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homing+endouncleases%3A+keeping+the+house+in+order." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homing+endouncleases%3A+keeping+the+house+in+order.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000003-Goddard2" id="pone.0000003-Goddard2"></a>Goddard MR, Burt A (1999) Recurrent invasion and extinction of a selfish gene. Proceedings of the National Academy of Sciences: USA  96: 13880–13885.  <ul class="find" data-citedArticleID="913020" data-doi="10.1073/pnas.96.24.13880"><li><a href="http://dx.doi.org/10.1073/pnas.96.24.13880" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recurrent+invasion+and+extinction+of+a+selfish+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recurrent+invasion+and+extinction+of+a+selfish+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000003-Koufopanou1" id="pone.0000003-Koufopanou1"></a>Koufopanou V, Goddard MR, Burt A (2002) Adaptation for horizontal transfer in a homing endonuclease. Molecular Biology and Evolution  19: 239–246.  <ul class="find" data-citedArticleID="913030" data-doi="10.1093/oxfordjournals.molbev.a004077"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004077" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptation+for+horizontal+transfer+in+a+homing+endonuclease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptation+for+horizontal+transfer+in+a+homing+endonuclease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000003-Bergthorsson1" id="pone.0000003-Bergthorsson1"></a>Bergthorsson U, Adams K, Thomason B, Palmer J (2003) Widespread horizontal transfer of mitochondrial genes in flowering plants. Nature  424: 197–201.  <ul class="find" data-citedArticleID="913002" data-doi="10.1038/nature01743"><li><a href="http://dx.doi.org/10.1038/nature01743" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Widespread+horizontal+transfer+of+mitochondrial+genes+in+flowering+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Widespread+horizontal+transfer+of+mitochondrial+genes+in+flowering+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000003-Dalgaard1" id="pone.0000003-Dalgaard1"></a>Dalgaard JZ, Klar AJ, Moser MJ, Holley WR, Chatterjee A, et al.  (1997) Statistical modelling and analysis of the LAGLIDADG family of site-specific endonuclease of the HNH family. Nucleic Acids Research  25: 4626–4638.  <ul class="find" data-citedArticleID="913012" data-doi="10.1093/nar/25.22.4626"><li><a href="http://dx.doi.org/10.1093/nar/25.22.4626" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+modelling+and+analysis+of+the+LAGLIDADG+family+of+site-specific+endonuclease+of+the+HNH+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+modelling+and+analysis+of+the+LAGLIDADG+family+of+site-specific+endonuclease+of+the+HNH+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000003-Haugen2" id="pone.0000003-Haugen2"></a>Haugen P, Reeb V, Lutzoni F, Bhattacharya D (2004) The evolution of homing endonuclease genes and group I introns in nuclear rDNA. Molecular Biology and Evolution  21: 129–140.  <ul class="find" data-citedArticleID="913026" data-doi="10.1093/molbev/msh005"><li><a href="http://dx.doi.org/10.1093/molbev/msh005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+homing+endonuclease+genes+and+group+I+introns+in+nuclear+rDNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+homing+endonuclease+genes+and+group+I+introns+in+nuclear+rDNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000003-Beagley1" id="pone.0000003-Beagley1"></a>Beagley CT, Okada NA, Wolstenholme DR (1996) Two mitochondrial group I introns in a metazoan, the sea anemone <em>Metridium senile</em>: One intron contains genes for subunits 1 and 3 of NADH dehydrogenase. Proceedings of the National Academy of Sciences: USA  93: 5619–5623.  <ul class="find" data-citedArticleID="912998" data-doi="10.1073/pnas.93.11.5619"><li><a href="http://dx.doi.org/10.1073/pnas.93.11.5619" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+mitochondrial+group+I+introns+in+a+metazoan%2C+the+sea+anemone+Metridium+senile%3A+One+intron+contains+genes+for+subunits+1+and+3+of+NADH+dehydrogenase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+mitochondrial+group+I+introns+in+a+metazoan%2C+the+sea+anemone+Metridium+senile%3A+One+intron+contains+genes+for+subunits+1+and+3+of+NADH+dehydrogenase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000003-Berntson1" id="pone.0000003-Berntson1"></a>Berntson EA, France SC, Mullineaux LS (1999) Phylogenetic relationships within the class anthozoa (Phylum Cnidaria) based in nuclear 18S rDNA sequences. Molecular Phylogenetics and Evolution  13: 417–433.  <ul class="find" data-citedArticleID="913006" data-doi="10.1006/mpev.1999.0649"><li><a href="http://dx.doi.org/10.1006/mpev.1999.0649" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+relationships+within+the+class+anthozoa+%28Phylum+Cnidaria%29+based+in+nuclear+18S+rDNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+relationships+within+the+class+anthozoa+%28Phylum+Cnidaria%29+based+in+nuclear+18S+rDNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000003-Won1" id="pone.0000003-Won1"></a>Won JH, Rho BJ, Song JI (2001) A phylogenetic study of the Anthozoa (phylum Cnidaria) based on morphological and molecular characters. Coral Reefs  20: 39–50.  <ul class="find" data-citedArticleID="913074" data-doi="10.1007/s003380000132"><li><a href="http://dx.doi.org/10.1007/s003380000132" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+phylogenetic+study+of+the+Anthozoa+%28phylum+Cnidaria%29+based+on+morphological+and+molecular+characters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+phylogenetic+study+of+the+Anthozoa+%28phylum+Cnidaria%29+based+on+morphological+and+molecular+characters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000003-Medina1" id="pone.0000003-Medina1"></a>Medina M, Collins AG, Takaoka TL, Kuehl JV, Boore JL (2006) Naked corals: skeleton loss in Scleratinia. Proceedings of the National Academy of Sciences: USA  103: 9096–9100.  <ul class="find" data-citedArticleID="913038" data-doi="10.1073/pnas.0602444103"><li><a href="http://dx.doi.org/10.1073/pnas.0602444103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Naked+corals%3A+skeleton+loss+in+Scleratinia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Naked+corals%3A+skeleton+loss+in+Scleratinia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000003-Shimodaira1" id="pone.0000003-Shimodaira1"></a>Shimodaira H, Hasegawa M (1999) Multiple comparisons of log-likelihood with applications to phylogenetics inference. Molecular Biology and Evolution  16: 1114–1116.  <ul class="find" data-citedArticleID="913062" data-doi="10.1093/oxfordjournals.molbev.a026201"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+comparisons+of+log-likelihood+with+applications+to+phylogenetics+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+comparisons+of+log-likelihood+with+applications+to+phylogenetics+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000003-Nylander1" id="pone.0000003-Nylander1"></a>Nylander JAA, Ronquist F, Huelsenbeck JP, Nieves-Aldrey JL (2004) Bayesian phylogenetic analysis of combined data. Systematic Biology  53: 47–67.  <ul class="find" data-citedArticleID="913042"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bayesian+phylogenetic+analysis+of+combined+data.&amp;auth=&amp;atitle=Bayesian+phylogenetic+analysis+of+combined+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+phylogenetic+analysis+of+combined+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+phylogenetic+analysis+of+combined+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000003-Lartillot1" id="pone.0000003-Lartillot1"></a>Lartillot N, Philippe H (2006) Computing Bayes Factors using thermodynamic integration. Systematic Biology  55: 195–207.  <ul class="find" data-citedArticleID="913034"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Computing+Bayes+Factors+using+thermodynamic+integration.&amp;auth=&amp;atitle=Computing+Bayes+Factors+using+thermodynamic+integration." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computing+Bayes+Factors+using+thermodynamic+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computing+Bayes+Factors+using+thermodynamic+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000003-Huelsenbeck1" id="pone.0000003-Huelsenbeck1"></a>Huelsenbeck JP, Bull JJ (1996) A likelihood ratio test to detect conflicting phylogenetic signal. Systematic Biology  45: 92–98.  <ul class="find" data-citedArticleID="913028"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+likelihood+ratio+test+to+detect+conflicting+phylogenetic+signal.&amp;auth=&amp;atitle=A+likelihood+ratio+test+to+detect+conflicting+phylogenetic+signal." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+likelihood+ratio+test+to+detect+conflicting+phylogenetic+signal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+likelihood+ratio+test+to+detect+conflicting+phylogenetic+signal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000003-Waggoner1" id="pone.0000003-Waggoner1"></a>Waggoner B, Collins AG (2004) Reductio ad absurdum: testing the evolutionary relationships of Ediacaran and Paleozoic problematic fossils using molecular divergence dates. Journal of Palaeontology  78: 51–61.  <ul class="find" data-citedArticleID="913068" data-doi="10.1666/0022-3360(2004)078<0051:raatte&gt;2.0.co;2"><li><a href="http://dx.doi.org/10.1666/0022-3360(2004)078<0051:raatte&gt;2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reductio+ad+absurdum%3A+testing+the+evolutionary+relationships+of+Ediacaran+and+Paleozoic+problematic+fossils+using+molecular+divergence+dates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reductio+ad+absurdum%3A+testing+the+evolutionary+relationships+of+Ediacaran+and+Paleozoic+problematic+fossils+using+molecular+divergence+dates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000003-Pagel1" id="pone.0000003-Pagel1"></a>Pagel M (1999) Inferring the historical patterns of biological evolution. Nature  401: 877–884.  <ul class="find" data-citedArticleID="913048" data-doi="10.1038/44766"><li><a href="http://dx.doi.org/10.1038/44766" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+the+historical+patterns+of+biological+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+the+historical+patterns+of+biological+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000003-Page1" id="pone.0000003-Page1"></a>Page RDM, Charleston MA (1998) Trees within trees: phylogeny and historical associations. Trends in Ecology and Evolution  13: 356–359.  <ul class="find" data-citedArticleID="913046" data-doi="10.1016/s0169-5347(98)01438-4"><li><a href="http://dx.doi.org/10.1016/s0169-5347(98)01438-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trees+within+trees%3A+phylogeny+and+historical+associations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trees+within+trees%3A+phylogeny+and+historical+associations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000003-Loreto1" id="pone.0000003-Loreto1"></a>Loreto E, Valente V, Zaha A, Silva J, Kidwell M (2001) Drosophila mediopunctata P elements: A new example of horizontal transfer. Journal of Heredity  92: 375–381.  <ul class="find" data-citedArticleID="913036" data-doi="10.1093/jhered/92.5.375"><li><a href="http://dx.doi.org/10.1093/jhered/92.5.375" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Drosophila+mediopunctata+P+elements%3A+A+new+example+of+horizontal+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Drosophila+mediopunctata+P+elements%3A+A+new+example+of+horizontal+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000003-Lampe1" id="pone.0000003-Lampe1"></a>Lampe D, Witherspoon D, Soto-Adames F, Robertson H (2003) Recent horizontal transfer of mellifera subfamily Mariner transposons into insect lineages representing four different orders shows that selection acts only during horizontal transfer. Molecular Biology and Evolution  20: 554–562.  <ul class="find" data-citedArticleID="913032" data-doi="10.1093/molbev/msg069"><li><a href="http://dx.doi.org/10.1093/molbev/msg069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+horizontal+transfer+of+mellifera+subfamily+Mariner+transposons+into+insect+lineages+representing+four+different+orders+shows+that+selection+acts+only+during+horizontal+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+horizontal+transfer+of+mellifera+subfamily+Mariner+transposons+into+insect+lineages+representing+four+different+orders+shows+that+selection+acts+only+during+horizontal+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000003-Bergthorsson2" id="pone.0000003-Bergthorsson2"></a>Bergthorsson U, Richardson A, Young G, Goertzen L, Palmer J (2004) Massive horizontal transfer of mitochondrial genes from diverse land plant donors to the basal angiosperm Amborella. Proceedings of the National Academy of Sciences: USA  101: 17747–17752.  <ul class="find" data-citedArticleID="913004" data-doi="10.1073/pnas.0408336102"><li><a href="http://dx.doi.org/10.1073/pnas.0408336102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Massive+horizontal+transfer+of+mitochondrial+genes+from+diverse+land+plant+donors+to+the+basal+angiosperm+Amborella." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Massive+horizontal+transfer+of+mitochondrial+genes+from+diverse+land+plant+donors+to+the+basal+angiosperm+Amborella.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000003-Nishimura1" id="pone.0000003-Nishimura1"></a>Nishimura Y, Misumi O, Kato K, Inada N, Higashiyama T, et al.  (2002) An mt<sup>+</sup> gamete-specific nuclease that targets mt<sup>−</sup> chloroplasts during sexual reproduction in <em>C. reinhardtii</em>. Genes and Development  16: 1116–1128.  <ul class="find" data-citedArticleID="913040" data-doi="10.1101/gad.979902"><li><a href="http://dx.doi.org/10.1101/gad.979902" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+mt%2B+gamete-specific+nuclease+that+targets+mt%26%23x2212%3B+chloroplasts+during+sexual+reproduction+in+C.+reinhardtii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+mt%2B+gamete-specific+nuclease+that+targets+mt%26%23x2212%3B+chloroplasts+during+sexual+reproduction+in+C.+reinhardtii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000003-Rokas1" id="pone.0000003-Rokas1"></a>Rokas A, Ladoukakis ED, Zouros E (2003) Animal mitochondrial DNA recombination revisited. Trends in Ecology and Evolution  18: 411–417.  <ul class="find" data-citedArticleID="913054" data-doi="10.1016/s0169-5347(03)00125-3"><li><a href="http://dx.doi.org/10.1016/s0169-5347(03)00125-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Animal+mitochondrial+DNA+recombination+revisited." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Animal+mitochondrial+DNA+recombination+revisited.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000003-Shearer1" id="pone.0000003-Shearer1"></a>Shearer TL, Oppen MJHV, Romano SL, Worheide G (2002) Slow mitochindrial DNA sequence evolution in the Anthozoa (Cnidaria). Molecular Ecology  11: 2475–2487.  <ul class="find" data-citedArticleID="913060" data-doi="10.1046/j.1365-294x.2002.01652.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2002.01652.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Slow+mitochindrial+DNA+sequence+evolution+in+the+Anthozoa+%28Cnidaria%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Slow+mitochindrial+DNA+sequence+evolution+in+the+Anthozoa+%28Cnidaria%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000003-Fukami1" id="pone.0000003-Fukami1"></a>Fukami H, Knowlton N (2005) Analysis of complete mitochondrial DNA sequences of three members of the <em>Montastraea annularis</em> coral species complex (Cnidaria, Anthozoa, Scleratinia). Coral Reefs  24: 410–417.  <ul class="find" data-citedArticleID="913016" data-doi="10.1007/s00338-005-0023-3"><li><a href="http://dx.doi.org/10.1007/s00338-005-0023-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+complete+mitochondrial+DNA+sequences+of+three+members+of+the+Montastraea+annularis+coral+species+complex+%28Cnidaria%2C+Anthozoa%2C+Scleratinia%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+complete+mitochondrial+DNA+sequences+of+three+members+of+the+Montastraea+annularis+coral+species+complex+%28Cnidaria%2C+Anthozoa%2C+Scleratinia%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000003-France1" id="pone.0000003-France1"></a>France SC, Hoover LL (2002) DNA sequence of the mitochondrial <em>COXI</em> gene have low levels of divergence among deep-sea octocorals (Cnidaria: Anthozoa). Hydrobiologia  471: 149–155.  <ul class="find" data-citedArticleID="913014"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+sequence+of+the+mitochondrial+COXI+gene+have+low+levels+of+divergence+among+deep-sea+octocorals+%28Cnidaria%3A+Anthozoa%29.&amp;auth=&amp;atitle=DNA+sequence+of+the+mitochondrial+COXI+gene+have+low+levels+of+divergence+among+deep-sea+octocorals+%28Cnidaria%3A+Anthozoa%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+sequence+of+the+mitochondrial+COXI+gene+have+low+levels+of+divergence+among+deep-sea+octocorals+%28Cnidaria%3A+Anthozoa%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+sequence+of+the+mitochondrial+COXI+gene+have+low+levels+of+divergence+among+deep-sea+octocorals+%28Cnidaria%3A+Anthozoa%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000003-PontKingdom1" id="pone.0000003-PontKingdom1"></a>Pont-Kingdom G, Okada NA, Macfarlane JL, Beagley CT, Watkins-Sims CD, et al.  (1997) Mitochondrial DNA of the coral <em>Sarcophyton glaucum</em> contains a gene for a homologue of bacterial MutS: a possible case of gene transfer from the nucleus to the mitochondria. Journal of Molecular Evolution  46: 419–431.  <ul class="find" data-citedArticleID="913052" data-doi="10.1007/pl00006321"><li><a href="http://dx.doi.org/10.1007/pl00006321" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+of+the+coral+Sarcophyton+glaucum+contains+a+gene+for+a+homologue+of+bacterial+MutS%3A+a+possible+case+of+gene+transfer+from+the+nucleus+to+the+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+of+the+coral+Sarcophyton+glaucum+contains+a+gene+for+a+homologue+of+bacterial+MutS%3A+a+possible+case+of+gene+transfer+from+the+nucleus+to+the+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000003-Pinto1" id="pone.0000003-Pinto1"></a>Pinto SM, Fernades-Matioli FMC, Schlenz E (2000) DNA extraction from the sea anemone (Cnidarina: Actinaria) tissues for molecular analyses. Genetics and Molecular Biology  23: 601–604.  <ul class="find" data-citedArticleID="913050" data-doi="10.1590/s1415-47572000000300017"><li><a href="http://dx.doi.org/10.1590/s1415-47572000000300017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+extraction+from+the+sea+anemone+%28Cnidarina%3A+Actinaria%29+tissues+for+molecular+analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+extraction+from+the+sea+anemone+%28Cnidarina%3A+Actinaria%29+tissues+for+molecular+analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000003-White1" id="pone.0000003-White1"></a>White TJ, Bruns T, Lee S, Taylor J (1990)  Amplification and direct sequencing of fungal ribosomal RNA genes for phylogenetics. In: Innes MA, Gelfand DH, Sninsky TT, White TJ, editors. PCR Protocols: a guide to methods and applications. San Diego: Academic Press. pp. 315–322.  <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0000003-Thompson1" id="pone.0000003-Thompson1"></a>Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG (1997) The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Research  25: 4876–4882.  <ul class="find" data-citedArticleID="913066" data-doi="10.1093/nar/25.24.4876"><li><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ClustalX+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ClustalX+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000003-Swofford1" id="pone.0000003-Swofford1"></a>Swofford DL (2002) PAUP*. Phylogenetic analysis using parsimony (*and other methods). Sunderland, Massachsettes: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">35.
              </span><a name="pone.0000003-Ronquist1" id="pone.0000003-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics  19: 1572–1574.  <ul class="find" data-citedArticleID="913056" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000003-Nylander2" id="pone.0000003-Nylander2"></a>Nylander JAA (2004) MrModeltest v2.   Uppsala University: Evolutionary Biology Centre. pp. Program distributed by the author.  <ul class="find-nolinks"></ul></li><li><span class="label">37.
              </span><a name="pone.0000003-Wolfram1" id="pone.0000003-Wolfram1"></a>Wolfram S (1996) Mathematica. Champaign: Wolfram Media.   5.0 ed.  <ul class="find-nolinks"></ul></li><li><span class="label">38.
              </span><a name="pone.0000003-Guindon1" id="pone.0000003-Guindon1"></a>Guindon S, Gascuel O (2003) A simple, fast and accurate algorithm to estimate large phylogenies by maximum likelihood. Systematic Biology  52: 696–704.  <ul class="find" data-citedArticleID="913022"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple%2C+fast+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.&amp;auth=&amp;atitle=A+simple%2C+fast+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+fast+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+fast+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000003-Schmidt1" id="pone.0000003-Schmidt1"></a>Schmidt HA, Strimmer K, Vingron M, von Haeseler A (2002) TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing. Bioinformatics  18: 502–504.  <ul class="find" data-citedArticleID="913058" data-doi="10.1093/bioinformatics/18.3.502"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.3.502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TREE-PUZZLE%3A+maximum+likelihood+phylogenetic+analysis+using+quartets+and+parallel+computing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TREE-PUZZLE%3A+maximum+likelihood+phylogenetic+analysis+using+quartets+and+parallel+computing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.XML" value="89281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.PDF" value="246888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.g001.PNG_L" value="276109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.g001.PNG_M" value="73067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.g001.PNG_S" value="3910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.g001.TIF" value="427708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.g001.PNG_I" value="48681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.g002.PNG_L" value="102851"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.g002.PNG_M" value="26923"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.g002.PNG_S" value="2178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.g002.TIF" value="583846"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.g002.PNG_I" value="14757"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.e001.PNG" value="6041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.e001.TIF" value="7726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.e002.PNG" value="5601"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.e002.TIF" value="7248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.e003.PNG" value="1233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.e003.TIF" value="3742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.s001.DOC" value="28160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000003.s002.PDF" value="8431"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000003&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000003" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000003&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000003&volume=&issue=&title=Invasion and Persistence of a Selfish Gene in the Cnidaria&author_name=Matthew%20R.%20Goddard%2C%20Jessica%20Leigh%2C%20Andrew%20J%20Roger%2C%20Andrew%20J%20Pemberton&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000003" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000003" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000003" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000003&amp;t=Invasion%20and%20Persistence%20of%20a%20Selfish%20Gene%20in%20the%20Cnidaria" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000003&title=Invasion%20and%20Persistence%20of%20a%20Selfish%20Gene%20in%20the%20Cnidaria&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000003&amp;title=Invasion%20and%20Persistence%20of%20a%20Selfish%20Gene%20in%20the%20Cnidaria" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000003" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000003&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Invasion and Persistence of a Selfish Gene in the Cnidaria';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000003';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Invasion%20and%20Persistence%20of%20a%20Selfish%20Gene%20in%20the%20Cnidaria http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000003" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000003" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="22846"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Introns%22" title="Search for articles in the subject area:'Introns'"><div class="flagText">Introns</div></a>
              <div data-categoryid="20277" data-articleid="22846"
                   data-categoryname="Introns"
                   class="flagImage" title="Flag 'Introns' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Invasive+species%22" title="Search for articles in the subject area:'Invasive species'"><div class="flagText">Invasive species</div></a>
              <div data-categoryid="46115" data-articleid="22846"
                   data-categoryname="Invasive species"
                   class="flagImage" title="Flag 'Invasive species' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondria%22" title="Search for articles in the subject area:'Mitochondria'"><div class="flagText">Mitochondria</div></a>
              <div data-categoryid="32453" data-articleid="22846"
                   data-categoryname="Mitochondria"
                   class="flagImage" title="Flag 'Mitochondria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="22846"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="22846"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sea+anemones%22" title="Search for articles in the subject area:'Sea anemones'"><div class="flagText">Sea anemones</div></a>
              <div data-categoryid="44383" data-articleid="22846"
                   data-categoryname="Sea anemones"
                   class="flagImage" title="Flag 'Sea anemones' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="22846"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7383'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=211'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5884&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=13615">better citation</a><br>Posted by mgog018</p>
      <p><a href="/annotation/listThread.action?root=20727">better citiation</a><br>Posted by mgog018</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
