

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Protistan Diversity in the Arctic: A Case of Paleoclimate Shaping Modern Biodiversity?</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000728"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000728" />

    <meta name="citation_title" content="Protistan Diversity in the Arctic: A Case of Paleoclimate Shaping Modern Biodiversity?"/>
    <meta itemprop="name" content="Protistan Diversity in the Arctic: A Case of Paleoclimate Shaping Modern Biodiversity?"/>

      <meta name="citation_author" content="Thorsten Stoeck"/>
            <meta name="citation_author_institution" content="Department of Ecology, University of Kaiserslautern, Kaiserslautern, Germany"/>
      <meta name="citation_author" content="Jennifer Kasper"/>
            <meta name="citation_author_institution" content="Department of Ecology, University of Kaiserslautern, Kaiserslautern, Germany"/>
      <meta name="citation_author" content="John Bunge"/>
            <meta name="citation_author_institution" content="Department of Statistical Science, Cornell University, Ithaca, New York, United States of America"/>
      <meta name="citation_author" content="Chesley Leslin"/>
            <meta name="citation_author_institution" content="Department of Biology, Northeastern University, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Valya Ilyin"/>
            <meta name="citation_author_institution" content="Department of Biology, Northeastern University, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Slava Epstein"/>
            <meta name="citation_author_institution" content="Department of Biology, Northeastern University, Boston, Massachusetts, United States of America"/>
            <meta name="citation_author_institution" content="Marine Science Center, Northeastern University, Nahant, Massachusetts, United States of America"/>

    <meta name="citation_date" content="2007/8/15"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000728.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e728"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The Cenozoic palaeoenvironment of the Arctic Ocean.; citation_author=K Moran; citation_author=J Backman; citation_author=H Brinkhuis; citation_author=SC Clemens; citation_author=T Cronin; citation_journal_title=Nature; citation_volume=441; citation_number=1; citation_pages=601-605; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A neoproterozoic snowball earth.; citation_author=PF Hoffman; citation_author=AJ Kaufman; citation_author=GP Halverson; citation_author=DP Schrag; citation_journal_title=Science; citation_volume=281; citation_number=2; citation_pages=1342-1346; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Late proterozoic low-latitude global glaciation: The snowball earth.; citation_author=JL Kirschvink; citation_number=3; citation_pages=51-52; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Life, geology and snowball Earth.; citation_author=DP Schrag; citation_author=PF Hoffman; citation_journal_title=Nature; citation_volume=409; citation_number=4; citation_pages=306; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The snowball Earth hypothesis: testing the limits of global change.; citation_author=PF Hoffman; citation_author=DP Schrag; citation_journal_title=Terra Nova; citation_volume=14; citation_number=5; citation_pages=129-155; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Paleoproterozoic snowball earth: extreme climatic and geochemical global change and its biological consequences.; citation_author=JL Kirschvink; citation_author=EJ Gaidos; citation_author=LE Bertani; citation_author=NJ Beukes; citation_author=J Gutzmer; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=6; citation_pages=1400-1405; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Estimating duration and intensity of Neoproterozoic snowball glaciations from Ir anomalies.; citation_author=B Bodiselitsch; citation_author=C Koeberl; citation_author=S Master; citation_author=WU Reimold; citation_journal_title=Science; citation_volume=308; citation_number=7; citation_pages=239-242; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Environment and evolution through the Paleocene-Eocene thermal maximum.; citation_author=PD Gingerich; citation_journal_title=Trends Ecol Evol; citation_volume=21; citation_number=8; citation_pages=246-253; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The rate and pattern of cladogenesis in microbes.; citation_author=AP Martin; citation_author=EKM Costello; citation_author=F A; citation_author=DR Nemergut; citation_author=SK Schmidt; citation_journal_title=Evolution; citation_volume=58; citation_number=9; citation_pages=946-955; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution and explosive speciation: the cichlid fish model.; citation_author=TD Kocher; citation_journal_title=Nat Rev Genet; citation_volume=5; citation_number=10; citation_pages=288-298; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Submarine hot springs and the origin of life.; citation_author=SL Miller; citation_author=JL Bada; citation_journal_title=Nature; citation_volume=334; citation_number=11; citation_pages=609-611; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Subtropical Arctic Ocean temperatures during the Palaeocene/Eocene thermal maximum.; citation_author=A Sluijs; citation_author=S Schouten; citation_author=M Pagani; citation_author=M Woltering; citation_author=H Brinkhuis; citation_journal_title=Nature; citation_volume=441; citation_number=12; citation_pages=610-613; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High temperatures in the Late Cretaceous Arctic Ocean.; citation_author=HC Jenkyns; citation_author=A Forster; citation_author=S Schouten; citation_author=JS Sinninghe Damste; citation_journal_title=Nature; citation_volume=432; citation_number=13; citation_pages=888-892; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Warm tropical ocean surface and global anoxia during the mid-Cretaceous period.; citation_author=PA Wilson; citation_author=RD Norris; citation_journal_title=Nature; citation_volume=412; citation_number=14; citation_pages=425-429; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Eocene bipolar glaciation associated with global carbon cycle changes.; citation_author=A Tripati; citation_author=J Backman; citation_author=H Elderfield; citation_author=P Ferretti; citation_journal_title=Nature; citation_volume=436; citation_number=15; citation_pages=341-346; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Latitudinal variations in organic diversity.; citation_author=AG Fischer; citation_journal_title=Evolution; citation_volume=14; citation_number=16; citation_pages=64-81; citation_date=1960; " />
      <meta name="citation_reference" content="citation_title=Benthic eukaryotic diversity in the Guaymas Basin hydrothermal vent environment.; citation_author=VP Edgcomb; citation_author=DT Kysela; citation_author=A Teske; citation_author=A de Vera Gomez; citation_author=ML Sogin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=17; citation_pages=7658-7662; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Autochthonous eukaryotic diversity in hydrothermal sediment and experimental microcolonizers at the Mid-Atlantic Ridge.; citation_author=P Lopez-Garcia; citation_author=H Philippe; citation_author=F Gail; citation_author=D Moreira; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=100; citation_number=18; citation_pages=697-702; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Labrador Sea boundary currents and the fate of the Irminger Sea Water.; citation_author=J Cuny; citation_author=PB Rhines; citation_author=PP Niiler; citation_author=S Bacon; citation_journal_title=J of Phys Ocean; citation_volume=32; citation_number=19; citation_pages=627-647; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Novel eukaryotic lineages inferred from small-subunit rRNA analyses of oxygen-depleted marine environments.; citation_author=T Stoeck; citation_author=S Epstein; citation_journal_title=Appl Environ Microbiol; citation_volume=69; citation_number=20; citation_pages=2657-2663; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Novel kingdom-level eukaryotic diversity in anoxic environments.; citation_author=SC Dawson; citation_author=NR Pace; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=21; citation_pages=8324-8329; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Synthetic statistical approach reveals a high degree of richness of microbial eukaryotes in an anoxic water column.; citation_author=SO Jeon; citation_author=J Bunge; citation_author=T Stoeck; citation_author=KJ Barger; citation_author=SH Hong; citation_journal_title=Appl Environ Microbiol; citation_volume=72; citation_number=22; citation_pages=6578-6583; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A multiple PCR-primer approach to access the microeukaryotic diversity in the anoxic Cariaco Basin (Caribbean Sea).; citation_author=T Stoeck; citation_author=B Hayward; citation_author=GT Taylor; citation_author=R Varela; citation_author=SS Epstein; citation_journal_title=Protist; citation_volume=157; citation_number=23; citation_pages=31-43; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Novel eukaryotes from a permanently anoxic Cariaco Basin (Caribbean Sea).; citation_author=T Stoeck; citation_author=G Taylor; citation_author=SS Epstein; citation_journal_title=Appl Environ Microbiol; citation_volume=69; citation_number=24; citation_pages=5656-5663; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Predicting microbial species richness.; citation_author=SH Hong; citation_author=J Bunge; citation_author=SO Jeon; citation_author=SS Epstein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=25; citation_pages=117-122; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biogeography.; citation_author=MV Lomolino; citation_author=BR Riddle; citation_author=JH Brown; citation_number=26; citation_pages=845; citation_date=2006; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=Molecular analysis of geographic patterns of eukaryotic diversity in Antarctic soils.; citation_author=B Lawley; citation_author=S Ripley; citation_author=P Bridge; citation_author=P Convey; citation_journal_title=Appl Environ Microbiol; citation_volume=70; citation_number=27; citation_pages=5963-5972; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bias caused by template annealing in the amplification of mixtures of 16S rRNA genes by PCR.; citation_author=MT Suzuki; citation_author=SJ Giovannoni; citation_journal_title=Appl Environ Microbiol; citation_volume=62; citation_number=28; citation_pages=625-630; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Diversity and distribution of marine microbial eukaryotes in the Arctic Ocean and adjacent seas.; citation_author=C Lovejoy; citation_author=R Massana; citation_author=C Pedros-Alio; citation_journal_title=Appl Environ Microbiol; citation_volume=72; citation_number=29; citation_pages=3085-3095; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microeukaryote community patterns along an O2/H2S gradient in a super-sulfidic anoxic fjord (Framvaren, Norway).; citation_author=A Behnke; citation_author=J Bunge; citation_author=K Barger; citation_author=H-W Breiner; citation_author=V Alla; citation_journal_title=Appl Environ Microbiol; citation_volume=72; citation_number=30; citation_pages=3626-3636; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Diversity estimates of microeukaryotes below the chemocline of the anoxic Mariager Fjord, Denmark.; citation_author=A Zuendorf; citation_author=J Bunge; citation_author=A Behnke; citation_author=KJ Barger; citation_author=T Stoeck; citation_journal_title=FEMS Microbiol Ecol; citation_volume=58; citation_number=31; citation_pages=476-491; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sand DNA—a genetic library of life at the water's edge.; citation_author=RK Naviaux; citation_author=B Good; citation_author=JD McPherson; citation_author=DL Steffen; citation_author=D Markusic; citation_journal_title=Mar Ecol Prog Ser; citation_volume=301; citation_number=32; citation_pages=9-22; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Microbiology: eukaryotic diversity in Spain's River of Fire.; citation_author=LA Amaral Zettler; citation_author=F Gomez; citation_author=E Zettler; citation_author=BG Keenan; citation_author=R Amils; citation_journal_title=Nature; citation_volume=417; citation_number=33; citation_pages=137; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Diversity of free-living prokaryotes from a deep-sea site at the Antarctic Polar Front.; citation_author=P Lopez-Garcia; citation_author=A Lopez-Lopez; citation_author=D Moreira; citation_author=F Rodriguez-Valera; citation_journal_title=FEMS Microbiol Ecol; citation_volume=36; citation_number=34; citation_pages=193-202; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Oceanic 18S rDNA sequences from picoplankton reveal unsuspected eukaryotic diversity.; citation_author=SY Moon-van der Staay; citation_author=R De Wachter; citation_author=D Vaulot; citation_journal_title=Nature; citation_volume=409; citation_number=35; citation_pages=607-610; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The stramenopiles from a molecular perspective: small subunit rDNA sequences from Labyrinthuloides minuta and Cafeteria roenbergensis.; citation_author=DD Leipe; citation_author=PO Wainright; citation_author=JH Gunderson; citation_author=D Porter; citation_author=DJ Patterson; citation_journal_title=Phycologia; citation_volume=33; citation_number=36; citation_pages=369-377; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The protistan origins of animals and fungi.; citation_author=ET Steenkamp; citation_author=J Wright; citation_author=SL Baldauf; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=37; citation_pages=93-106; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The phylogeny of colpodellids (Alveolata) using small subunit rRNA gene sequences suggests they are the free-living sister group to apicomplexans.; citation_author=ON Kuvardina; citation_author=BS Leander; citation_author=VV Aleshin; citation_author=AP Myl'nikov; citation_author=PJ Keeling; citation_journal_title=J Eukaryot Microbiol; citation_volume=49; citation_number=38; citation_pages=498-504; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny and surface morphology of Colpodella edax (Alveolata): insights into the phagotrophic ancestry of apicomplexans.; citation_author=BS Leander; citation_author=ON Kuvardina; citation_author=VV Aleshin; citation_author=AP Mylnikov; citation_author=PJ Keeling; citation_journal_title=J Eukaryot Microbiol; citation_volume=50; citation_number=39; citation_pages=334-340; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial evolution.; citation_author=MW Gray; citation_author=G Burger; citation_author=BF Lang; citation_journal_title=Science; citation_volume=283; citation_number=40; citation_pages=1476-1481; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=An ancestral mitochondrial DNA resembling a eubacterial genome in miniature.; citation_author=BF Lang; citation_author=G Burger; citation_author=CJ O'Kelly; citation_author=R Cedergren; citation_author=GB Golding; citation_journal_title=Nature; citation_volume=387; citation_number=41; citation_pages=493-497; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The jakobid flagellates: structural features of Jakoba, Reclinomonas and Histiona and implications for the early diversification of eukaryotes.; citation_author=CJ O'Kelly; citation_journal_title=J Euk Microbiol; citation_volume=40; citation_number=42; citation_pages=627-636; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Enigmatic Late Cretaceous high palaeo-latitude lonestones in Chukotka, northeasternmost Asia.; citation_author=A Ahlberg; citation_author=AB Herman; citation_author=M Raikevich; citation_author=A Rees; citation_author=RA Spicer; citation_journal_title=GFF; citation_volume=124; citation_number=43; citation_pages=197-199; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The diodiversity of Greenland–a country study.; citation_number=44; citation_pages=165; citation_date=2003; citation_publisher=Gronlands Naturinstitut; " />
      <meta name="citation_reference" content="citation_title=Phylogeny of marine Gregarines (Apicomplexa)–Pterospora, Lithocystis and Lankesteria–and the origin(s) of coelomic parasitism.; citation_author=BS Leander; citation_author=SA Lloyd; citation_author=W Marshall; citation_author=SC Landers; citation_journal_title=Protist; citation_volume=157; citation_number=45; citation_pages=45-60; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ultrastructure and identification of the predatory flagellate Colpodella pugnax Cienkowski (Apicomplexa) with a description of Colpodella turpis n. sp. and a review of the genus.; citation_author=AGB Simpson; citation_author=DJ Patterson; citation_journal_title=Syst Parasitol; citation_volume=33; citation_number=46; citation_pages=187-198; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Identification and activity of a series of azole-based compounds with lactate dehydrogenase-directed anti-malarial activity.; citation_author=A Cameron; citation_author=J Read; citation_author=R Tranter; citation_author=VJ Winter; citation_author=RB Sessions; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=47; citation_pages=31429-31439; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Metabolic changes of the malaria parasite during the transition from the human to the mosquito host.; citation_author=N Lang-Unnasch; citation_author=AD Murphy; citation_journal_title=Annu Rev Microbiol; citation_volume=52; citation_number=48; citation_pages=561-590; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A hydrogenosome with pyruvate formate-lyase: anaerobic chytrid fungi use an alternative route for pyruvate catabolism.; citation_author=A Akhmanova; citation_author=FG Voncken; citation_author=KM Hosea; citation_author=H Harhangi; citation_author=JT Keltjens; citation_journal_title=Mol Microbiol; citation_volume=32; citation_number=49; citation_pages=1103-1114; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Multiple origins of hydrogenosomes: functional and phylogenetic evidence from the ADP/ATP carrier of the anaerobic chytrid Neocallimastix sp.; citation_author=F Voncken; citation_author=B Boxma; citation_author=J Tjaden; citation_author=A Akhmanova; citation_author=M Huynen; citation_journal_title=Mol Microbiol; citation_volume=44; citation_number=50; citation_pages=1441-1454; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: Improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties, and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgens; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=51; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The RDP-II (Ribosomal Database Project).; citation_author=BL Maidak; citation_author=JR Cole; citation_author=TG Lilburn; citation_author=CT Parker; citation_author=PR Saxman; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=52; citation_pages=173-174; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evaluation of nearest-neighbor methods for detection of chimeric small-subunit rRNA sequences.; citation_author=JF Robison-Cox; citation_author=MM Bateson; citation_author=DM Ward; citation_journal_title=Appl Environ Microbiol; citation_volume=61; citation_number=53; citation_pages=1240-1245; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mixtures of exponential distributions to describe the distribution of Poisson means in estimating the number of unobserved classes.; citation_author=KJ-A Barger; citation_number=54; citation_date=2006; citation_publisher=Cornell University; " />
      <meta name="citation_reference" content="citation_title=Estimating the number of species in a stochastic abundance model.; citation_author=A Chao; citation_author=J Bunge; citation_journal_title=Biometrics; citation_volume=58; citation_number=55; citation_pages=531-539; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Predicting the number of new species in a further taxonomic sampling.; citation_author=T-J Shen; citation_author=A Chao; citation_author=C-F Lin; citation_journal_title=Ecology; citation_volume=84; citation_number=56; citation_pages=798-804; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madde; citation_author=AA Schaffer; citation_author=JH Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=57; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=ARB: a software environment for sequence data.; citation_author=W Ludwig; citation_author=O Strunk; citation_author=R Westram; citation_author=L Richter; citation_author=H Meier; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=58; citation_pages=1363-1371; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=PAUP*. Phylogenetic analysis using parsimony (and other methods). 4.0b6. ed.; citation_author=DL Swofford; citation_number=59; citation_date=2001; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=MODELTEST: testing the model of DNA substitution.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=60; citation_pages=817-818; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The characterization of enzymatically amplified eukaryotic 16S-like rRNA-coding regions.; citation_author=L Medlin; citation_author=HJ Elwood; citation_author=S Stickel; citation_author=ML Sogin; citation_journal_title=Gene; citation_volume=71; citation_number=61; citation_pages=491-499; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Unexpected diversity of small eukaryotes in deep-sea Antarctic plankton.; citation_author=P Lopez-Garcia; citation_author=F Rodriguez-Valera; citation_author=C Pedros-Alio; citation_author=D Moreira; citation_journal_title=Nature; citation_volume=409; citation_number=62; citation_pages=603-607; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=16S/23S rRNA sequencing.; citation_author=DJ Lane; citation_number=63; citation_pages=115-175; citation_date=1991; citation_publisher=John Wiley&Sons; " />
      <meta name="citation_reference" content="citation_title=Evaluation of protein A gene polymorphic region DNA sequencing for typing of Staphylococcus aureus strains.; citation_author=B Shopsin; citation_author=M Gomez; citation_author=SO Montgomery; citation_author=DH Smith; citation_author=M Waddington; citation_journal_title=J Clin Microbiol; citation_volume=37; citation_number=64; citation_pages=3556-3563; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Improvement of ciliate identification and quantification: a new protocol for fluorescence in situ hybridization (FISH) in combination with silver stain techniques.; citation_author=J Fried; citation_author=W Ludwig; citation_author=R Psenner; citation_author=KH Schleifer; citation_journal_title=Syst Appl Microbiol; citation_volume=25; citation_number=65; citation_pages=555-571; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The European ribosomal RNA database.; citation_author=J Wuyts; citation_author=G Perriere; citation_author=Y Van de Peer; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=66; citation_pages=D101-D103; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The small-subunit ribosomal RNA gene sequences from the hypotrichous ciliates Oxytricha nova and Stylonychia pustulata.; citation_author=HJ Elwood; citation_author=GJ Olsen; citation_author=ML Sogin; citation_journal_title=Mol Biol Evol; citation_volume=2; citation_number=67; citation_pages=399-410; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Highly divergent SSU rRNA genes found in the marine ciliates Myrionecta rubra and Mesodinium pulex.; citation_author=MD Johnson; citation_author=T Tengs; citation_author=DW Oldach; citation_author=CF Delwiche; citation_author=K Stoecker; citation_journal_title=Protist; citation_volume=155; citation_number=68; citation_pages=347-359; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000728" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Protistan Diversity in the Arctic: A Case of Paleoclimate Shaping Modern Biodiversity?"/>
    <meta name="twitter:description" content="
Background
The impact of climate on biodiversity is indisputable. Climate changes over geological time must have significantly influenced the evolution of biodiversity, ultimately leading to its present pattern. Here we consider the paleoclimate data record, inferring that present-day hot and cold environments should contain, respectively, the largest and the smallest diversity of ancestral lineages of microbial eukaryotes.

Methodology/Principal Findings
We investigate this hypothesis by analyzing an original dataset of 18S rRNA gene sequences from Western Greenland in the Arctic, and data from the existing literature on 18S rRNA gene diversity in hydrothermal vent, temperate sediments, and anoxic water column communities. Unexpectedly, the community from the cold environment emerged as one of the richest observed to date in protistan species, and most diverse in ancestral lineages.

Conclusions/Significance
This pattern is consistent with natural selection sweeps on aerobic non-psychrophilic microbial eukaryotes repeatedly caused by low temperatures and global anoxia of snowball Earth conditions. It implies that cold refuges persisted through the periods of greenhouse conditions, which agrees with some, although not all, current views on the extent of the past global cooling and warming events. We therefore identify cold environments as promising targets for microbial discovery.
"/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000728.g010"/>

  <meta property="og:title" content="Protistan Diversity in the Arctic: A Case of Paleoclimate Shaping Modern Biodiversity?" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000728" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4487'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7199'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9439&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000728">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000728" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000728&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Protistan Diversity in the Arctic: A Case of Paleoclimate Shaping Modern Biodiversity?
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thorsten Stoeck, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Ecology, University of Kaiserslautern, Kaiserslautern, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jennifer Kasper, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Ecology, University of Kaiserslautern, Kaiserslautern, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John Bunge, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Statistical Science, Cornell University, Ithaca, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chesley Leslin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Northeastern University, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Valya Ilyin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Northeastern University, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Slava Epstein
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:s.epstein@neu.edu">s.epstein@neu.edu</a></p>

                <p>Affiliations:
                  Department of Biology, Northeastern University, Boston, Massachusetts, United States of America, 
                  Marine Science Center, Northeastern University, Nahant, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 15, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000728</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000728" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000728" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000728" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000728" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000728-t001" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000728-t002" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000728-g001" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000728-g002" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000728-g003" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000728-g004" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000728-g005" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000728-g006" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000728-g007" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000728-g008" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000728-g009" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000728-g010" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g010" title="Figure 10">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g010&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000728">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000728" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The impact of climate on biodiversity is indisputable. Climate changes over geological time must have significantly influenced the evolution of biodiversity, ultimately leading to its present pattern. Here we consider the paleoclimate data record, inferring that present-day hot and cold environments should contain, respectively, the largest and the smallest diversity of ancestral lineages of microbial eukaryotes.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We investigate this hypothesis by analyzing an original dataset of 18S rRNA gene sequences from Western Greenland in the Arctic, and data from the existing literature on 18S rRNA gene diversity in hydrothermal vent, temperate sediments, and anoxic water column communities. Unexpectedly, the community from the cold environment emerged as one of the richest observed to date in protistan species, and most diverse in ancestral lineages.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This pattern is consistent with natural selection sweeps on aerobic non-psychrophilic microbial eukaryotes repeatedly caused by low temperatures and global anoxia of snowball Earth conditions. It implies that cold refuges persisted through the periods of greenhouse conditions, which agrees with some, although not all, current views on the extent of the past global cooling and warming events. We therefore identify cold environments as promising targets for microbial discovery.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Stoeck T, Kasper J, Bunge J, Leslin C, Ilyin V, et al.  (2007) Protistan Diversity in the Arctic: A Case of Paleoclimate Shaping Modern Biodiversity? PLoS ONE 2(8):
          e728.
            doi:10.1371/journal.pone.0000728</p><p><strong>Academic Editor: </strong>Rees Kassen, University of Ottawa, Canada</p><p><strong>Received:</strong> January 14, 2007; <strong>Accepted:</strong> July 5, 2007; <strong>Published:</strong> August 15, 2007</p><p><strong>Copyright:</strong> © 2007 Stoeck et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported by the US NSF grant OPP-0214231 to SSE and Deutsche Forschungsgemeinschaft grants STO414/2-2 and STO 414/2-3 to TS. The research was conducted using the computing facilities of the Northeastern University, and resources of the Cornell Theory Center, which receives funding from Cornell University, New York State, federal agencies, foundations, and corporate partners.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Periods of steady climate on our planet have been punctuated by extraordinary paleoclimate events, from extreme greenhouses, with Arctic ocean temperatures soaring above 20°C <a href="#pone.0000728-Moran1">[1]</a>, to the freezing conditions of snowball Earth <a href="#pone.0000728-Hoffman1">[2]</a>–<a href="#pone.0000728-Schrag1">[4]</a>. These dramatic fluctuations in temperature, and global anoxia that likely coincided with the freezing events <a href="#pone.0000728-Hoffman2">[5]</a>, must have had a pronounced effect on the evolution of biological diversity. Each event probably acted as a powerful natural selection filter, with different events selecting for exactly opposite traits. It is possible that linking the history of paleoclimate to evolution of biodiversity may help to explain the present pattern of this biodiversity. We are interested in what biodiversity predictions can be made based on the paleoclimate dynamics, and whether such predictions can be verified. Here we examine a link between the diversity of microbial eukaryotes and paleoclimate events over the course of their evolution.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>There is increasing evidence that several snowball Earth conditions occurred in the history of our planet (e.g., in the Paleoproterozoic 2.4 billion years ago <a href="#pone.0000728-Kirschvink2">[6]</a>, and more recently in the Neoproterozoic, 710 and 635 million years ago <a href="#pone.0000728-Hoffman1">[2]</a>–<a href="#pone.0000728-Kirschvink1">[3]</a>, <a href="#pone.0000728-Hoffman2">[5]</a>, <a href="#pone.0000728-Bodiselitsch1">[7]</a>), as did the ultra-greenhouse temperatures that likely followed such conditions <a href="#pone.0000728-Hoffman1">[2]</a>, <a href="#pone.0000728-Hoffman2">[5]</a>, and more moderate greenhouse events, the latest of which occurred in the Cenozoic 55 million years ago <a href="#pone.0000728-Moran1">[1]</a>, <a href="#pone.0000728-Gingerich1">[8]</a>. Regardless of when and at what temperature the eukaryotes originated, they must have survived dramatic temperature and oxygen level fluctuations. Extreme cold would obviously select for extreme psychrophiles, making all other organisms less competitive. The opposite would be expected during extreme global warming periods. We postulate that one principal factor determining what kind of biodiversity would survive a cataclysmic shift is the presence or absence of a refuge. Depending on whether or not temperature refuges persisted throughout the history of eukaryotic life, we can envision four scenarios of how paleoclimate might-and in fact should–have influenced the evolution of microbial eukaryotes:</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>1. Refuges did not exist, and temperature extremes reached every corner of the biosphere. If so, neither psychrophiles nor thermophiles would be expected to survive an adverse temperature swing, and would have to evolve anew after each such event.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>2. Only cold refuges persisted through time. If so, today's cold environments would be enriched with basal lineages with a long history of continuous evolution in the cold. Thermophilic lineages of today would be descendants of psychrophiles. Having evolved after the last global freezing event, these lineages would be expected to root within clades of psychrophilic origin.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>3. Only hot refuges persisted through time. This would lead to a scenario opposite to #2: assemblages in hot environments would be enriched with basal lineages, with psychrophiles appearing as evolutionary newcomers.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>4. Both types of refuges continuously existed throughout the history of eukaryotic life. In this case, today both low- and high-temperature environments would be expected to show a diversity of microeukaryotic life, each characterized by its own basal lineages. Continuous evolution of cold adaptations in thermophiles would produce evolutionarily young psychrophilic lineages within clades of thermophilic origin, and vice versa, leading to clades represented by mixes of thermo- and psychrophiles.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Two additional considerations add to the picture of predicted patterns of microeukaryotic diversity. First, since the snowball Earth was largely anoxic, the majority of survivors must have been obligate and facultative anaerobes. One would expect this to manifest today as a diversity of deeply rooted microbial eukaryotes with anaerobic life style. Second, it is reasonable to expect that older habitats would harbor larger biological richness by providing longer periods of time for this richness to evolve, especially since the diversification rate of microorganisms may have exceeded their extinction rate for billions of years <a href="#pone.0000728-Martin1">[9]</a>. This richness should be distinct from that of a quick explosive origin (<em>e.g.</em>, cichlid fish in Africa <a href="#pone.0000728-Kocher1">[10]</a>): the first should appear as a collection of lineages rooted deeply, whereas the second as a number of short lineages branching off from a distinct node of relatively recent origin.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>The four scenarios above are not all equally plausible. The first is the least realistic, because hydrothermal environments have likely been a persistent feature of the ocean floor since the origin of our planet <a href="#pone.0000728-Miller1">[11]</a>. Though rather ephemeral at individual sites, the hydrothermal vent environment in general would still provide a safe harbor for at least a selection of thermophilic (and perhaps some mesophilic) organisms, even at the peaks of the snowball Earth events.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>In contrast, there is little evidence to support the persistence of cold refuges. The paleoceanographic record constructed from a sediment core from a recent drilling expedition in the Arctic Ocean suggests that early Cenozoic was very warm, with polar waters staying above 20°C year round, and no cold deep ocean <a href="#pone.0000728-Moran1">[1]</a>, <a href="#pone.0000728-Sluijs1">[12]</a>. The mid-Cretaceous is also thought to have been exceptionally warm <a href="#pone.0000728-Jenkyns1">[13]</a>–<a href="#pone.0000728-Wilson1">[14]</a>. It appears that cold environments might have been completely–and repeatedly-wiped out during such times, making scenarios 2 and 4 rather unlikely.</p>
<a id="article1.body1.sec1.p10" name="article1.body1.sec1.p10"></a><p>This leaves case 3 as the most plausible, at least in terms of current paleoclimate views. This scenario implies that microbial eukaryotes at elevated temperatures, such as in hydrothermal vent environments, should be uniquely rich in ancestral lineages, and exhibit a relatively large number of species, especially with anaerobic lifestyle. In contrast, protistan assemblages in cold environments would be expected to have lower species diversity and to be represented by lineages of recent evolutionary origin. The peak of the latest extreme greenhouse event was approximately 55 million years ago <a href="#pone.0000728-Moran1">[1]</a>, and glaciation in the northern hemisphere did not start until 6–10 million years ago <a href="#pone.0000728-Tripati1">[15]</a>. Following the logic of the “center of origin” hypothesis <a href="#pone.0000728-Fischer1">[16]</a>, it seems reasonable to expect that protists from the polar regions, having evolved over that relatively short period of time, would form few (perhaps no) “old” unique clades, and would fall into groups characterized by deeply rooted thermophiles. Here we test these predictions by comparing the 18S rRNA gene diversity in two hydrothermal vent communities (HV1 <a href="#pone.0000728-Edgcomb1">[17]</a> and HV2 <a href="#pone.0000728-LopezGarcia1">[18]</a>), to the diversity in an original dataset we obtained from a tidal flat in the Arctic (DI; Disko Island, Western Greenland). This sediment habitat represents one of the coldest tidal flats studied because it stays ice-free for only 3–4 months a year, with water temperature averaging −1.8°C <a href="#pone.0000728-Cuny1">[19]</a>, and rarely exceeding 4–6°C even on a sunny summer day. For comparative purposes, we include in our analyses the only available 18S rRNA surveys of temperate sediments (WH <a href="#pone.0000728-Stoeck1">[20]</a>, and CA1 and CA2 <a href="#pone.0000728-Dawson1">[21]</a>), as well as one of the best-studied (to date) protistan assemblages from an anoxic system (CB; Cariaco Basin in the Caribbean <a href="#pone.0000728-Jeon1">[22]</a>, <a href="#pone.0000728-Stoeck2">[23]</a>, <a href="#pone.0000728-Stoeck3">[24]</a>). We compare the results of these surveys by examining (1) their respective overall species richness; (2) the presence/absence, phylogeny, and richness of unique clades of high taxonomic order; and (3) the biology of relevant organisms.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Species richness</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The rRNA surveys in question were of unequal size, and the observed numbers of individual phylotypes detected were not directly comparable. We recently developed a statistical procedure to estimate biological richness based on observed data of this type, and tested it on both prokaryotic and protistan datasets <a href="#pone.0000728-Jeon1">[22]</a>, <a href="#pone.0000728-Hong1">[25]</a>. We applied this approach here to estimate the total richness of the samples from the target communities, which allows a direct comparison between such samples (<a href="#pone-0000728-t001">Table 1</a>). We note that the true richness of hydrothermal vent communities may be overestimated here because some of the 18S sequences detected at the richer site (HV1) likely originated from the photic zone of the overlying water column <a href="#pone.0000728-LopezGarcia1">[18]</a>. When the uppermost layer of the sediment was excluded from analyses, the number of allochthonous sequences was reduced <a href="#pone.0000728-LopezGarcia1">[18]</a>. We note that even though this lowered the estimate of total protistan richness (compare HV1 and HV2 in <a href="#pone-0000728-t001">Table 1</a>), it remained higher than those for temperate tidal flat. This may appear surprising because the ultimate community food base (i.e., fixed organic material) is likely less diverse and abundant in deep sea than in shallow water environments. However, the apparently higher richness of hydrothermal vent protistan communities can be easily explained if this environment was evolutionarily old, and its inhabitants had a long uninterrupted history of diversification.</p>
<div class="figure" id="pone-0000728-t001"><div class="img"><a name="pone-0000728-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000728.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000728.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>18S rDNA phylotype richness estimates of protistan assemblages in Disko Island (DI, Arctic); hydrothermal vents HV1 <a href="#pone.0000728-Edgcomb1">[17]</a> and HV2 <a href="#pone.0000728-LopezGarcia1">[18]</a>; temperate sediment communities WH <a href="#pone.0000728-Stoeck1">[20]</a>, CA1 and CA2 <a href="#pone.0000728-Dawson1">[21]</a>; and anoxic water column CB <a href="#pone.0000728-Stoeck2">[23]</a>.</span></strong></p><span>doi:10.1371/journal.pone.0000728.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Unexpectedly, the Arctic protistan community emerged as one of the richest in species to date. This contrasts with the prevailing general view that Arctic cold conditions do not favor high biological diversity <a href="#pone.0000728-Lomolino1">[26]</a>, but is tentatively supported by recent findings of a latitudinal increase in eukaryotic soil richness towards the southernmost sampling points in Antarctica <a href="#pone.0000728-Lawley1">[27]</a>. Considering the small size of our sediment sample for DNA extraction (3 g), and the fact that any estimate based on the rRNA approach is conservative due to the well-known PCR and primer biases <a href="#pone.0000728-Suzuki1">[28]</a>, the 605 phylotypes predicted to co-exist in the sample tell a story of notable diversity and richness. A recently detected high diversity of planktonic protists in the Arctic <a href="#pone.0000728-Lovejoy1">[29]</a> lends additional support to our findings.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>While comparing the richness estimates (<a href="#pone-0000728-t001">Table 1</a>), we considered several factors that could bias these estimates, thus potentially limiting the usefulness of such comparison.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>1. Different sites were sampled using somewhat different methodologies. Similarly to our research, one of the two temperate tidal flat studies used both anoxic and suboxic parts of the sediment cores <a href="#pone.0000728-Stoeck1">[20]</a>, whereas the second used only the anoxic sediment zone <a href="#pone.0000728-Dawson1">[21]</a>. Nonetheless, the richness estimates are essentially the same between the latter two (<a href="#pone-0000728-t001">Table 1</a>), and equally low relative to the one obtained here. This indicates that the presence of aerobic protists in our samples cannot explain the large difference between their richness and that in the temperate samples. This is in line with the growing understanding that protistan diversity in anoxic environments significantly exceeds that of oxic habitats <a href="#pone.0000728-Behnke1">[30]</a>–<a href="#pone.0000728-Zuendorf1">[31]</a>.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>2. Sand binds dissolved DNA <a href="#pone.0000728-Naviaux1">[32]</a>, and the rRNA gene diversity uncovered here might contain signatures of allochthonous species. While there is a good chance this might be the case, the artifact in question is probably characteristic of all sediment works. The temperate and vent sediments considered in our analyses were all micro- to mesoporal sands, and all could be enriched with DNA from nearby sources. While assessing the impact of this artifact, it is important to consider the richness of the contributing sources. Temperate sediments must be influenced by, among other factors, adjacent soils, which are known to be rich in microbial life. In contrast, the contribution of this source in the present study was minimal, as the soils immediately adjacent to the sampling site were barren stony desert, with partial snow cover. The likely higher diversity of allochthonous DNA in temperate sediments makes the observed difference between protistan diversity in temperate vs. Arctic sediments conservative.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>3. The original dataset was obtained using a multiple primer approach, which could detect more protistan rRNA genes than a more traditional single primer approach. This is an important difference between this and some (but not all) published research, and therefore deserves careful analysis. In addition to predicting the DI protistan richness based on pooled clone data (<a href="#pone-0000728-t001">Table 1</a>), we conducted separate analyses using rRNA gene sequences obtained with individual PCR primers. Using primer sets D1, 2, and 3 (<a href="#pone-0000728-t002">Table 2</a>), we obtained and sequenced 192 clones for each clone library, of which 143, 170, and 138, respectively, produced protistan 18S rRNA gene sequences. We grouped these into OTUs at 99% and 98% rRNA gene sequence identity, and obtained respectively 79 and 62 OTUs for clone D2 and D1 clone libraries, and 52 and 53 OTUs for D1 and D2 clone libraries. We then reconstructed the OTU frequency distribution in the above 4 groups of OTUs, and applied to these frequency distributions the same statistical approach we used with the pooled data (<a href="#pone-0000728-t001">Table 1</a>) to produce estimates of total protistan richness. Because these analyses use the data obtained with single primer sets, their results are comparable with those from the majority of published reports.</p>
<div class="figure" id="pone-0000728-t002"><div class="img"><a name="pone-0000728-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000728.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000728.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Primer sequences used in this study for specific PCR-amplification of 18S rRNA gene sequences from environmental Disko Island DNA.</span></strong></p><span>doi:10.1371/journal.pone.0000728.t002</span></div><a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7"></a><p>As expected, predictions of protistan richness based on single PCR primer datasets were lower than those based on pooled clone data (<a href="#pone-0000728-t001">Table 1</a>). These predictions are: 225±72 (SE) OTUs at 99% level of gene identity based on primer set 3 individual dataset, and 126±42 (SE) and 132±30 (SE) OTUs at 98% level based on primer sets 1 and 2 individual datasets, respectively (the 4<sup>th</sup> analysis that used primer set 2 at 99% level OTUs did not produce statistically valid total richness predictions). Though these estimates are substantially lower than those based on the pooled data (<a href="#pone-0000728-t001">Table 1</a>), it is important to note that they are higher than an estimate of protistan richness in a temperate tidal flat community <a href="#pone.0000728-Stoeck1">[20]</a>, and twice as high as either of the two anoxic tidal flats we considered <a href="#pone.0000728-Dawson1">[21]</a>, even though the latter study employed a multiple PCR primer approach. The above estimates appear on a par with HV2 and, with sediment communities in mind, are only second to HV1 (<a href="#pone-0000728-t001">Table 1</a>). It should be noted, however, that the HV1 data <a href="#pone.0000728-Edgcomb1">[17]</a> contains a significant number of signatures of photosynthetic protists and is believed to be artificially enriched with allochthonous species from the overlying water column <a href="#pone.0000728-LopezGarcia1">[18]</a>. Therefore, the data obtained here, whether considered as a library based on the multiple PCR primer approach, or as a collection of libraries each based on an individual PCR primer, indicate a surprising level of protistan richness in the Arctic community.</p>
<a id="article1.body1.sec2.sec1.p8" name="article1.body1.sec2.sec1.p8"></a><p>We conclude that the evidence for this richness is not due to methodological differences between this and prior studies of 18S rRNA gene diversity in sediment communities, but rather points to the Polar environment as a “hot spot” of protistan biodiversity. This does not support the idea that this community is evolutionarily young and principally derived from thermophilic ancestors.</p>


<h4>Comparative phylogeny of Arctic microbial eukaryotes</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The phylogenetic diversity of microbial eukaryotes from a marine tidal flat in Western Greenland is substantial, matching or exceeding that detected in hydrothermal vents <a href="#pone.0000728-Edgcomb1">[17]</a>–<a href="#pone.0000728-LopezGarcia1">[18]</a>, temperate sediments <a href="#pone.0000728-Stoeck1">[20]</a>–<a href="#pone.0000728-Dawson1">[21]</a>, or any other community studied to date <a href="#pone.0000728-AmaralZettler1">[33]</a>–<a href="#pone.0000728-MoonvanderStaay1">[35]</a>. We uncovered representatives of all major protistan clades known to science, except microsporidians, trichomonads, and uncultured Alveolates (<a href="#pone-0000728-g001">Figure 1</a>; for a detailed account of the phylogenetic positions of all 18S rRNA gene sequences discovered see <a href="#pone-0000728-g002">Figures 2</a>–<a href="#pone-0000728-g003"></a><a href="#pone-0000728-g004"></a><a href="#pone-0000728-g005"></a><a href="#pone-0000728-g006"></a><a href="#pone-0000728-g007"></a><a href="#pone-0000728-g008"></a><a href="#pone-0000728-g009"></a><a href="#pone-0000728-g010">10</a>).</p>
<div class="figure" id="pone-0000728-g001"><div class="img"><a name="pone-0000728-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>18S rDNA Maximum Parsimony (MP) tree showing the assignment of Disko Island phylotypes to major eukaryote clades.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The numbers a/b indicate the total number of GenBank sequences representing the clade/the number of phylotypes detected. Ancestral sequences to a specific clade were included in the clade itself. Only protistan and fungal sequences are shown in the tree. Detailed phylogenies (partial treeing analyses) can be found in <a href="#pone-0000728-g002">Figures 2</a>–<a href="#pone-0000728-g003"></a><a href="#pone-0000728-g004"></a><a href="#pone-0000728-g005"></a><a href="#pone-0000728-g006"></a><a href="#pone-0000728-g007"></a><a href="#pone-0000728-g008"></a><a href="#pone-0000728-g009"></a><a href="#pone-0000728-g010">10</a>.</p>
<span>doi:10.1371/journal.pone.0000728.g001</span></div><div class="figure" id="pone-0000728-g002"><div class="img"><a name="pone-0000728-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Minimum evolution phylogenetic tree of 18S rDNA sequences showing the position of apicomplexan and perkinsozoan Disko Island phylotypes (Alveolata).</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>The tree was constructed under Maximum Likelihood criteria (ML) using a General Time Reversible (GTR) DNA substitution model with the variable-site gamma distribution shape parameter (G) at 0.6830 and the proportion of invariable sites (I) at 0.1125, and is based on 784 unambiguously aligned and conserved positions. Distance bootstrap values over 50% from an analysis of 1000 bootstrap replicates are given at the respective nodes; dots identify nodes with 100% bootstrap support. Clone names in red, blue, and green identify sequences reported in this study (DI) and from hydrothermal vent and temperate environments, respectively. The first two identifiers of the DI sequences (D1–D5) designate the different PCR primer sets used in this study (detailed information in <a href="#pone-0000728-t002">Table 2</a>).</p>
<span>doi:10.1371/journal.pone.0000728.g002</span></div><div class="figure" id="pone-0000728-g003"><div class="img"><a name="pone-0000728-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Minimum evolution phylogenetic tree of 18S rDNA sequences showing the position of basal-branching Disko Island sequences.</span></strong></p><a id="article1.body1.sec2.sec2.fig3.caption1.p1" name="article1.body1.sec2.sec2.fig3.caption1.p1"></a><p>The tree was constructed under Maximum Likelihood criteria (ML) using a General Time Reversible (GTR) DNA substitution model with the variable-site gamma distribution shape parameter (G) at 1.3667, the proportion of invariable sites (I) at 0.0658, and base frequencies and a rate matrix for the substitution model as suggested by Modeltest (see <a href="#s3">Material and Methods</a>), based on 553 unambiguously aligned and conserved positions. Because of their uncertain position in phylogenetic 18S rDNA sequence trees, the branching orders of the primitive jakobid flagellates and the <em>Myrionecta</em>/<em>Mesodinium</em> (Ciliophora) group are not supported. For a detailed explanation of the basal branching of the latter group see e.g. <a href="#pone.0000728-Zuendorf1">[31]</a>, <a href="#pone.0000728-Johnson1">[68]</a>. Distance bootstrap values over 50% from an analysis of 1000 bootstrap replicates are given at the respective nodes; dots identify nodes with 100% bootstrap support. Clone names in red, blue, and green identify sequences reported in this study (DI) and from hydrothermal vent and temperate environments, respectively. The first two identifiers of the DI sequences (D1–D5) designate the different PCR primer sets used in this study (detailed information in <a href="#pone-0000728-t002">Table 2</a>).</p>
<span>doi:10.1371/journal.pone.0000728.g003</span></div><div class="figure" id="pone-0000728-g004"><div class="img"><a name="pone-0000728-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Minimum evolution phylogenetic tree of 18S rDNA sequences showing the position of heterotrophic stramenopile Disko Island sequences.</span></strong></p><a id="article1.body1.sec2.sec2.fig4.caption1.p1" name="article1.body1.sec2.sec2.fig4.caption1.p1"></a><p>The tree was constructed using a Tamura Nei DNA substitution model with the variable-site gamma distribution shape parameter (G) at 0.6830 and the proportion of invariable sites (I) at 0.1125, and is based on 784 unambiguously aligned and conserved positions. Distance bootstrap values over 50% from an analysis of 1000 bootstrap replicates are given at the respective nodes; dots identify nodes with 100% bootstrap support. Clone names in red, blue, and green identify sequences reported in this study (DI) and from hydrothermal vent and temperate environments, respectively. The first two identifiers of the DI sequences (D1–D5) designate the different PCR primer sets used in this study (detailed information in <a href="#pone-0000728-t002">Table 2</a>).</p>
<span>doi:10.1371/journal.pone.0000728.g004</span></div><div class="figure" id="pone-0000728-g005"><div class="img"><a name="pone-0000728-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Minimum evolution phylogenetic tree of 18S rDNA sequences showing the position of autotrophic stramenopile Disko Island sequences.</span></strong></p><a id="article1.body1.sec2.sec2.fig5.caption1.p1" name="article1.body1.sec2.sec2.fig5.caption1.p1"></a><p>The tree was constructed under using a Tamura Nei substitution model with the variable-site gamma distribution shape parameter (G) at 0.7606 and the proportion of invariable sites (I) at 0.4458, and is based on 851 unambiguously aligned and conserved positions; dots identify nodes with 100% bootstrap support. Clone names in red, blue, and green identify sequences reported in this study (DI) and from hydrothermal vent and temperate environments, respectively. The first two identifiers of the DI sequences (D1–D5) designate the different PCR primer sets used in this study (detailed information in <a href="#pone-0000728-t002">Table 2</a>).</p>
<span>doi:10.1371/journal.pone.0000728.g005</span></div><div class="figure" id="pone-0000728-g006"><div class="img"><a name="pone-0000728-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Minimum evolution phylogenetic tree of 18S rDNA sequences showing the position of rhizarian Disko Island sequences.</span></strong></p><a id="article1.body1.sec2.sec2.fig6.caption1.p1" name="article1.body1.sec2.sec2.fig6.caption1.p1"></a><p>The tree was constructed under Maximum Likelihood criteria (ML) using a General Time Reversible (GTR) DNA substitution model with the variable-site gamma distribution shape parameter (G) at 0.8312 and the proportion of invariable sites (I) at 0.2533, and is based on 817 unambiguously aligned and conserved positions. Distance bootstrap values over 50% from an analysis of 1000 bootstrap replicates are given at the respective nodes; dots identify nodes with 100% bootstrap support. Clone names in red, blue, and green identify sequences reported in this study (DI) and from hydrothermal vent and temperate environments, respectively. The first two identifiers of the DI sequences (D1–D5) designate the different PCR primer sets used in this study (detailed information in <a href="#pone-0000728-t002">Table 2</a>).</p>
<span>doi:10.1371/journal.pone.0000728.g006</span></div><div class="figure" id="pone-0000728-g007"><div class="img"><a name="pone-0000728-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Minimum evolution phylogenetic tree of 18S rDNA sequences showing the position of unicellular algal Disko Island sequences (Plantae).</span></strong></p><a id="article1.body1.sec2.sec2.fig7.caption1.p1" name="article1.body1.sec2.sec2.fig7.caption1.p1"></a><p>The tree was constructed using a Tamura Nei DNA substitution model with the variable-site gamma distribution shape parameter (G) at 0.6123, and is based on 813 unambiguously aligned and conserved positions. Distance bootstrap values over 50% from an analysis of 1000 bootstrap replicates are given at the respective nodes; dots identify nodes with 100% bootstrap support. Clone names in red, blue, and green identify sequences reported in this study (DI) and from hydrothermal vent and temperate environments, respectively. The first two identifiers of the DI sequences (D1–D5) designate the different PCR primer sets used in this study (detailed information in <a href="#pone-0000728-t002">Table 2</a>).</p>
<span>doi:10.1371/journal.pone.0000728.g007</span></div><div class="figure" id="pone-0000728-g008"><div class="img"><a name="pone-0000728-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Minimum evolution phylogenetic tree of 18S rDNA sequences showing the position of dinophycean Disko Island sequences.</span></strong></p><a id="article1.body1.sec2.sec2.fig8.caption1.p1" name="article1.body1.sec2.sec2.fig8.caption1.p1"></a><p>The tree was constructed using a Tamura Nei DNA substitution model with the variable-site gamma distribution shape parameter (G) at 0.7606 and the proportion of invariable sites (I) at 0.4458, and is based on 851 unambiguously aligned and conserved positions. Distance bootstrap values over 50% from an analysis of 1000 bootstrap replicates are given at the respective nodes; dots identify nodes with 100% bootstrap support. Clone names in red, blue, and green identify sequences reported in this study (DI) and from hydrothermal vent and temperate environments, respectively. The first two identifiers of the DI sequences (D1–D5) designate the different PCR primer sets used in this study (detailed information in <a href="#pone-0000728-t002">Table 2</a>).</p>
<span>doi:10.1371/journal.pone.0000728.g008</span></div><div class="figure" id="pone-0000728-g009"><div class="img"><a name="pone-0000728-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Minimum evolution phylogenetic tree of 18S rDNA sequences showing the position of euglenozoan Disko Island sequences.</span></strong></p><a id="article1.body1.sec2.sec2.fig9.caption1.p1" name="article1.body1.sec2.sec2.fig9.caption1.p1"></a><p>The tree was constructed under Maximum Likelihood criteria (ML) using a General Time Reversible (GTR) DNA substitution model with the variable-site gamma distribution shape parameter (G) at 0.8904 and the proportion of invariable sites (I) at 0.1517, and is based on 730 unambiguously aligned and conserved positions. Distance bootstrap values over 50% from an analysis of 1000 bootstrap replicates are given at the respective nodes; dots identify nodes with 100% bootstrap support. Clone names in red, blue, and green identify sequences reported in this study (DI) and from hydrothermal vent and temperate environments, respectively. The first two identifiers of the DI sequences (D1–D5) designate the different PCR primer sets used in this study (detailed information in <a href="#pone-0000728-t002">Table 2</a>).</p>
<span>doi:10.1371/journal.pone.0000728.g009</span></div><div class="figure" id="pone-0000728-g010"><div class="img"><a name="pone-0000728-g010" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g010&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000728" data-uri="info:doi/10.1371/journal.pone.0000728.g010"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000728.g010&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g010/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g010/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g010/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g010/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g010.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000728.g010/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000728.g010/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000728.g010.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 10.  <span>Minimum evolution phylogenetic tree of 18S rDNA sequences showing the position of ophistokont Disko Island sequences.</span></strong></p><a id="article1.body1.sec2.sec2.fig10.caption1.p1" name="article1.body1.sec2.sec2.fig10.caption1.p1"></a><p>The tree was constructed using a Tamura Nei DNA substitution model with the variable-site gamma distribution shape parameter (G) at 0.6200, and is based on 669 unambiguously aligned and conserved positions. Distance bootstrap values over 50% from an analysis of 1000 bootstrap replicates are given at the respective nodes; dots identify nodes with 100% bootstrap support. Clone names in red, blue, and green identify sequences reported in this study (DI) and from hydrothermal vent and temperate environments, respectively. The first two identifiers of the DI sequences (D1–D5) designate the different PCR primer sets used in this study (detailed information in <a href="#pone-0000728-t002">Table 2</a>).</p>
<span>doi:10.1371/journal.pone.0000728.g010</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Apart from its sheer breadth, one interesting aspect of the DI phylotypes' diversity is that it is skewed in favor of several specific taxonomic groups. For example, apicomplexans are represented remarkably well in our Arctic clone libraries and constituted over 12% of all unique protistan rRNA gene sequences. This proportion is the highest ever recorded; it exceeds the second highest by 5-fold (2.4%; temperate intertidal community <a href="#pone.0000728-Dawson1">[21]</a>). Further statistical analyses estimated the total number of distinct apicomplexan OTUs (&gt;99% sequence identity) at the Arctic site to be 174±60, exceeding the entire protistan richness at any of the three temperate tidal flats studied to date (<a href="#pone-0000728-t001">Table 1</a>). Perhaps even more importantly, a number of the Arctic apicomplexan lineages are deeply rooted within the clade (<a href="#pone-0000728-g002">Figure 2</a>), which makes the hypothesis of their recent diversification implausible.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Similarly, chytrid fungi, absent from the hydrothermal or temperate sediment sites studied so far, were well represented in the DI data and constituted almost 5% of all unique 18S rRNA gene sequences detected there. We estimated the total number of OTUs (&gt;99% sequence identity) of all fungi to be 60±22; the data on chytrid fungi alone was insufficient for direct richness estimation, but a conservative lower bound (based on the equal-OTU-abundance or unmixed Poisson model) for the number of chytrid fungi OTUs was 14±2. Interestingly, the only other study of 18S rRNA gene diversity in a polar terrestrial environment (Antarctic soil, <a href="#pone.0000728-Lawley1">[27]</a>) also noted an elevated diversity of chytrid fungi. Similarly to apicomplexans, chytrid lineages uncovered here do not seem to be a result of recent radiation: they either form novel sister clades to the known chytrids, or appear ancestral to them (<a href="#pone-0000728-g009">Figure 9</a>).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Jakobid-like organisms showed a strong presence in Arctic as well, constituting over 5% of all unique 18S rRNA sequences (<a href="#pone-0000728-g003">Figure 3</a>). However, they are notably absent at hydrothermal sites. Also noteworthy is a substantial diversity of Labyrinthulida in the DI data, some of which appear ancestral to the Labyrinthulids detected at hydrothermal vent sites (<a href="#pone-0000728-g004">Figure 4</a>).</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>We note that those clades that appear disproportionately rich in the Arctic are also evolutionarily older. Labyrinthulids are almost certainly ancestral to one of the largest protistan clades, Stramenopiles <a href="#pone.0000728-Leipe1">[36]</a>. Chytrids are undoubtedly primitive and are thought to be ancestral to Fungi <a href="#pone.0000728-Steenkamp1">[37]</a>. Apicomplexans possess a specialized apical complex suggesting their derived nature, but other ultrastructural characteristics point to their position as ancestral to another large eukaryotic clade, the Alveolates <a href="#pone.0000728-Kuvardina1">[38]</a>–<a href="#pone.0000728-Leander1">[39]</a>. Most interestingly, Jakobids are characterized by the most bacteria-like mitochondrial genome known, and are regarded as the most ancestral mitochondria-bearing eukaryotes <a href="#pone.0000728-Gray1">[40]</a>–<a href="#pone.0000728-OKelly1">[42]</a>. Collectively, Stramenopiles, Alveolates, Fungi, and Jakobids represent a substantial part of the entire eukaryotic diversity known today, and finding an extensive richness of their ancestral lineages in one community identifies this community as an important target for future efforts, and potentially informative for reconstructing eukaryotic evolution.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>This comparative phylogenetic analysis leads to an inescapable conclusion: not only does the Arctic appear to be a “hot spot” of protistan biodiversity, but it is also notably rich in lineages ancestral to several principal eukaryotic clades, if not to the eukaryotes in general. It seems unlikely that communities with such traits are evolutionarily young and derived (<em>e.g.</em>, from thermophiles). A simpler explanation is that communities in the cold environment of the Arctic have been evolving in this or a similar environment for a substantial period of time, possibly comparable to the age of eukaryotes. This suggests that cold refuges might have persisted even through the warmest periods of the last one to two billion years, which is in conflict with the current view on the extent of global warming episodes (<a href="#pone.0000728-Moran1">[1]</a>, but see also <a href="#pone.0000728-Ahlberg1">[43]</a>).</p>


<h4>Biology of Arctic microbial eukaryotes</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The four clades that are significantly–if not uniquely–rich in the Arctic sediment clone libraries are Apicomplexans, Chytrids, Labyrinthulids, and Jakobids. All these organisms are heterotrophs; many are saprophytes or parasites. Interpreting these observations meets with two difficulties.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>First, the source of organic material necessary to support such a diversity of heterotrophs is not obvious. Autochthonous material is a doubtful source of organics given the uncharacteristically low number of autotrophic protists detected. Seaweeds are typically absent from Arctic tidal flats because of the action of winter ice. Diatoms and uncultured alveolates, which are ubiquitous components of the water column communities, mayrepresent a potential source of carbon, but this is hypothetical as our (sediment) dataset contains few to no signatures of these organisms. The source of allochthonous energy supply is also unclear. The sampling site is surrounded by a stony arctic desert with small patches of high latitude tundra, an unlikely supplier of organic material. By exclusion, this leads to subtidal fields of seaweeds, dominated principally by brown and red algae of substantial diversity (over 100 species in Greenland, <a href="#pone.0000728-Jensen1">[44]</a>), as potential suppliers of carbon. We hypothesize that algal biomass which has washed ashore, together with drift seaweeds, constitute an important source of “missing” energy. This would explain why so many of the protists detected belong to groups rich in saprophytes (<em>e.g.,</em> Chytrids, Labyrinthulids).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The second difficulty is to explain an unparalleled richness of Apicomplexans, a group with very few known free-living representatives. We hypothesize that the apicomplexan life cycle, which includes resilient cysts, makes this group particularly competitive, and thus particularly successful, in the harsh environment of the Arctic. However, this cannot expect the puzzling aspect of high apicomplexan diversity because the Arctic has a low diversity of prospective hosts (animals, primarily mammals). Benthic macrofauna is also known to decrease in diversity in the northern direction <a href="#pone.0000728-Fischer1">[16]</a>, and it is reasonable to think the diversity of their parasites would mirror the trend. If so, the diversity of parasitic apicomplexans and chytrids would be expected to be higher in the temperate zone, not in Disko Bay. Our data show a strong opposite trend. The simplest explanation is that our rRNA survey detected a diversity of previously unknown free-living Apicomplexans and chytrids. Some of them may be close to the free-living ancestor of the present day parasitic forms, which is line with our finding deeply rooted apicomplexan and chytrid lineages. This is also indirectly supported by our detection of several 18S rRNA gene sequences related to <em>Colpodella</em>, a rare example of free-living apicomplexan, which is thought to be ancestral to the clade <a href="#pone.0000728-Leander2">[45]</a>–<a href="#pone.0000728-Simpson1">[46]</a>.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>One important common theme in the biology of species from the four clades in question is their energy metabolism. The few Apicomplexans studied in this regard rely principally on substrate level phosphorylation at least at some stages of their life cycles (cf., <em>Plasmodium</em>, <a href="#pone.0000728-Cameron1">[47]</a>), and some are thought to lack the full complement of the TCA cycle <a href="#pone.0000728-LangUnnasch1">[48]</a>. Chytrids are among the few eukaryotes possessing hydrogenosomes in place of mitochondria <a href="#pone.0000728-Akhmanova1">[49]</a>–<a href="#pone.0000728-Voncken1">[50]</a>, and are almost certainly anaerobes. There is virtually no information on bioenergetics of Labyrinthulids and Jakobids, but both groups are often found in anoxic environments <a href="#pone.0000728-Edgcomb1">[17]</a>, <a href="#pone.0000728-Stoeck1">[20]</a>, <a href="#pone.0000728-Stoeck2">[23]</a>–<a href="#pone.0000728-Stoeck3">[24]</a>, <a href="#pone.0000728-Behnke1">[30]</a>. This leads us to conclude that the Arctic protistan community studied exhibits an elevated richness of cold-adapted, ancestral protistan lineages with anaerobic/microaerophilic life styles. This is in line with an accepted view that snowball Earth events coincided with global anoxia <a href="#pone.0000728-Hoffman2">[5]</a>, and the idea that cold environments have been a persistent feature of our planet for at least a part of eukaryotic evolution.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>In conclusion, we identified the cold environment of the Arctic as a “hot spot” of protistan biodiversity rich in evolutionarily old lineages. The communities of microbial eukaryotes in temperate habitats appear less diverse and not as rich. Polar environments therefore hold a high potential for future research on microbial discovery. To account for the observed biodiversity patterns, we postulate that cold refuges have persisted throughout the history of eukaryotic evolution, which is in conflict with some current views on the extent of past global warming events.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Study site and sampling</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Sediment samples were collected in July 2003 in Unqussivik, a small bay on Disko Island, Greenland (69°26′03″N; 54°14′13″W). The tidal flat of the bay is composed of gravel and sand of varying porosity. We sampled the fine sand areas (median grain size 1801 µm) with a well developed sulfidic zone below the 5–10 mm depth. Five samples were collected by coring the sediments to 10 cm depth. The samples were transported to the Arctic Station, University of Copenhagen, Qeqertarsuaq, Greenland, and kept at the temperature registered at the time of sampling (6°C) for 12 hr. After thoroughly mixing the samples, we obtained three 3-g subsamples as sources of DNA for PCR reaction employing 3 different primer sets, as described below.</p>


<h4>Nucleic acid isolation and PCR-amplification of 18S rDNA</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>High molecular weight DNA was obtained from 3-g aliquots of sediment samples as described previously <a href="#pone.0000728-Stoeck1">[20]</a>. The sample was heated to 65°C for 2 h in a high-salt extraction buffer (100 mM Tris-HCl [pH8], 100 mM Na<sub>2</sub>EDTA [pH8], 100 mM NaPO<sub>4</sub> [pH8]) with SDS (20%), CTAB (1%), and Proteinase K (0.1 mg ml<sup>−1</sup> final). The lysates were purified twice by extraction with an equal volume of chloroform-isoamyl alcohol (24:1) and precipitated with a 0.6 volume isopropanol. Humic acids and other potential inhibitors of downstream applications were removed by DNA purification with the resin-based Wizard DNA clean up system (Promega, Madison, WI). The integrity of the total DNA was checked by agarose gel electrophoresis (0.8%), and the DNA yield was quantified using a Nanodrop ND-1000 UV-Vis spectrophotometer (Nanodrop Technologies, Wilmington, DE). The molecular weight of extracted genomic DNA was <em>c.</em> 13 Kb (as determined by comparison to a molecular weight standard). We extracted 17 µg DNA per g of sediment. Following the multiple-primer approach <a href="#pone.0000728-Stoeck2">[23]</a>, we amplified fragments of the 18S rRNA gene ranging from ≈1,500 bp to≈1,100 bp, using five different primer sets (<a href="#pone-0000728-t002">Table 2</a>). PCR reactions were performed as described in detail previously <a href="#pone.0000728-Stoeck3">[24]</a>.</p>


<h4>Clone library construction</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The PCR products were used to construct a clone library for each primer set with the pGEM-T Vector System cloning kit (Promega, Madison, WI). According to the primer set identifiers (<a href="#pone-0000728-t002">Table 2</a>), the five clone libraries were designated D1–D5. Plasmids were isolated from overnight cultures using the 96well Directprep kit (Qiagen, Valencia, CA). Nearly 1000 clones evenly distributed among the different primer sets (<a href="#pone-0000728-t002">Table 2</a>), were partially sequenced (M13F sequencing primer) at MWG-biotech (Ebersheim, Germany) using an Applied Biosystems (ABI) 3730 DNA Stretch Sequencer, with the XL Upgrade and the ABI Prism BigDye Terminator version 3.1 Cycle Sequencing Ready Reaction Kit. Initially, partial sequences of each individual primer set were grouped separately into operational taxonomic units (OTUs) based on a 99.0% sequence similarity cutoff value (separately for 5′-3′ and 3′-5′ vector-inserted gene fragments). Selected representative clones of each OTU were then sequenced bidirectionally. Sequence quality assessments, PHRED and PHRAP analysis and assembling were performed with the program CodonCode Aligner v. 1.2.4 (CodonCode corporation, Dedham, MA, USA).</p>


<h4>Sequence clustering</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The 18S rDNA sequences were grouped into OTUs based on 99, 98, and 97% sequence similarity cutoff values. This was achieved, as described previously <a href="#pone.0000728-Jeon1">[22]</a>, <a href="#pone.0000728-Hong1">[25]</a>, by first making all possible pairwise sequence alignments using ClusatlW at default settings <a href="#pone.0000728-Thompson1">[51]</a> and calculating percent sequence similarities, followed by clustering of the sequences into OTUs using the mean UPGMA linkages as implemented in the OC clustering program (<a href="http://www.compbio.dundee.ac.uk/Software/OC/oc.html">http://www.compbio.dundee.ac.uk/Software​/OC/oc.html</a>). The OTU grouping was checked manually to verify that all OTUs were assembled at the cutoff level desired. Potentially chimeric sequences were identified using secondary structure predictions, the Chimera_Check command version 2.7 provided by the Ribosomal database project <a href="#pone.0000728-Maidak1">[52]</a>, and partial treeing analyses <a href="#pone.0000728-RobisonCox1">[53]</a>. Representative gene sequences of each OTU have been deposited in the GenBank database (accession numbers EF100195-EF100415).</p>


<h4>Statistical analyses</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Our estimates of species richness are based on sample frequency count data; for mathematical and computational details and recent applications see <a href="#pone.0000728-Jeon1">[22]</a>, <a href="#pone.0000728-Hong1">[25]</a>, <a href="#pone.0000728-Barger1">[54]</a>–<a href="#pone.0000728-Chao1">[55]</a>. In brief, we assume that each species independently contributes representatives to the sample according to a Poisson process, and that the rates of these processes are randomly generated by a stochastic “abundance” distribution. We currently test seven candidate abundance distributions: point mass or unmixed Poisson (equal species sizes); gamma (negative binomial); inverse Gaussian; lognormal; Pareto, and mixtures of two and of three exponential distributions, fitting these by maximum likelihood using custom software (available on request). Datasets of this type typically have long right tails; we therefore fit every model to every right-truncated subset of each dataset. For each subset we compare models using the Akaike Information Criterion (AIC); across subsets we compare models using an asymptotically-corrected Pearson chi-square goodness-of-fit statistic. We also compute the nonparametric Abundance-based Coverage Estimator <a href="#pone.0000728-Dawson1">[21]</a> and its variants in each case, using the software SPADE <a href="#pone.0000728-Shen1">[56]</a>. This computationally intensive process is carried out on a supercomputer in the Cornell Theory Center; the work described here entailed about 4,000 separate analyses.</p>


<h4>Phylogenetic analyses</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Low quality sequence reads and non-target metazoan sequences were excluded from the phylogenetic analyses. Environmental 18S rRNA gene sequences initially were compared to those in GenBank using gapped BLAST analysis <a href="#pone.0000728-Altschul1">[57]</a> to determine their approximate phylogenetic affiliation. Environmental sequence data together with their closest GenBank matches were compiled in ARB <a href="#pone.0000728-Ludwig1">[58]</a> and aligned using the ARB FastAligner utility. Alignments were manually refined using phylogenetically conserved secondary structures. The conserved and unambiguously aligned positions (their number is indicated in <a href="#pone-0000728-g002">Figures 2</a>–<a href="#pone-0000728-g003"></a><a href="#pone-0000728-g004"></a><a href="#pone-0000728-g005"></a><a href="#pone-0000728-g006"></a><a href="#pone-0000728-g007"></a><a href="#pone-0000728-g008"></a><a href="#pone-0000728-g009"></a><a href="#pone-0000728-g010">10</a>) were used in subsequent phylogenetic analyses. Classification of unique phylotypes was performed via two phylogenetic inference methods: minimum evolutionary distance (ME, <a href="#pone-0000728-g002">Figures 2</a>–<a href="#pone-0000728-g003"></a><a href="#pone-0000728-g004"></a><a href="#pone-0000728-g005"></a><a href="#pone-0000728-g006"></a><a href="#pone-0000728-g007"></a><a href="#pone-0000728-g008"></a><a href="#pone-0000728-g009"></a><a href="#pone-0000728-g010">10</a>) and maximum parsimony (MP, <a href="#pone-0000728-g001">Figure 1</a>). Trees were constructed using the PAUP* software package 4.0b10 <a href="#pone.0000728-Swofford1">[59]</a>. All heuristic searches were performed using random, stepwise addition of taxa with the TBR branch-swapping algorithm. We applied the program Modeltest <a href="#pone.0000728-Posada1">[60]</a> to choose the models of DNA substitution that best fit our datasets from among 56 possible models. Modeltest was run for each individual dataset. The DNA substitution models as well as the parameter settings for each tree constructed are described in detail in the legend of the respective figure. We assessed the relative stability of tree topologies using 1000 distance bootstrap replicates. Heuristic searches for bootstrap analyses employed stepwise addition starting trees with simple addition of sequences and TBR branch-swapping.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>All datasets used for phylogenetic analyses are available from the authors upon request.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors are indebted to Dr. Paul Hoffman for his critical comments on this manuscript and its ideas, and his advice on paleoclimate dynamics. The authors sincerely thank the Scientific Leader of the Artic Station of the University of Copenhagen Dr. Director Bente Jessen Graae for her unwavering support of our research program and warm hospitality, and the Station's Chief of Logistics Kjeld Mølgaard for his untiring assistance in organizing boat access. Dmitry Epstein and Alice Epstein bravely helped locating and sampling this remote area of Greenland. We are deeply grateful to Drs. S. Dawson, V. Edgcomb, P. López-García, and N. Pace for generously sharing their data on OTU frequencies in their rRNA gene surveys, and to Dr. David Patterson for his advice on protistan taxonomy and evolution. We thank Linda Woodard for constructing the database system used for richness estimation, and Sara Laurentz for overseeing the richness computations. We are very appreciative of the constructive criticisms of anonymous reviewers that substantially improved the manuscript. Comments and suggestion made by one reviewer were particularly helpful, especially on the possibility that multiple PCR primer approach increased the detected richness vis-à-vis that reported in the literature. This is contribution #257 of the Marine Science Center of the Northeastern University, Boston MA, USA.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SE TS JB. Performed the experiments: TS JK JB VI CL. Analyzed the data: SE TS JK JB. Contributed reagents/materials/analysis tools: SE TS JB VI CL. Wrote the paper: SE TS JB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000728-Moran1" id="pone.0000728-Moran1"></a>Moran K, Backman J, Brinkhuis H, Clemens SC, Cronin T, et al.  (2006) The Cenozoic palaeoenvironment of the Arctic Ocean. Nature  441: 601–605.  <ul class="find" data-citedArticleID="976442" data-doi="10.1038/nature04800"><li><a href="http://dx.doi.org/10.1038/nature04800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Cenozoic+palaeoenvironment+of+the+Arctic+Ocean." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Cenozoic+palaeoenvironment+of+the+Arctic+Ocean.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000728-Hoffman1" id="pone.0000728-Hoffman1"></a>Hoffman PF, Kaufman AJ, Halverson GP, Schrag DP (1998) A neoproterozoic snowball earth. Science  281: 1342–1346.  <ul class="find" data-citedArticleID="976392" data-doi="10.1126/science.281.5381.1342"><li><a href="http://dx.doi.org/10.1126/science.281.5381.1342" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+neoproterozoic+snowball+earth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+neoproterozoic+snowball+earth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000728-Kirschvink1" id="pone.0000728-Kirschvink1"></a>Kirschvink JL (1992)  Late proterozoic low-latitude global glaciation: The snowball earth. In: Schopf JW, Klein C, Des Maris D, editors. The proterozoic biosphere: A multidisciplinary study: Cambridge University Press. pp. 51–52.  <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0000728-Schrag1" id="pone.0000728-Schrag1"></a>Schrag DP, Hoffman PF (2001) Life, geology and snowball Earth. Nature  409: 306.  <ul class="find" data-citedArticleID="976484" data-doi="10.1038/35053170"><li><a href="http://dx.doi.org/10.1038/35053170" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Life%2C+geology+and+snowball+Earth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Life%2C+geology+and+snowball+Earth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000728-Hoffman2" id="pone.0000728-Hoffman2"></a>Hoffman PF, Schrag DP (2002) The snowball Earth hypothesis: testing the limits of global change. Terra Nova  14: 129–155.  <ul class="find" data-citedArticleID="976408" data-doi="10.1046/j.1365-3121.2002.00408.x"><li><a href="http://dx.doi.org/10.1046/j.1365-3121.2002.00408.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+snowball+Earth+hypothesis%3A+testing+the+limits+of+global+change." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+snowball+Earth+hypothesis%3A+testing+the+limits+of+global+change.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000728-Kirschvink2" id="pone.0000728-Kirschvink2"></a>Kirschvink JL, Gaidos EJ, Bertani LE, Beukes NJ, Gutzmer J, et al.  (2000) Paleoproterozoic snowball earth: extreme climatic and geochemical global change and its biological consequences. Proc Natl Acad Sci U S A  97: 1400–1405.  <ul class="find" data-citedArticleID="976368" data-doi="10.1073/pnas.97.4.1400"><li><a href="http://dx.doi.org/10.1073/pnas.97.4.1400" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Paleoproterozoic+snowball+earth%3A+extreme+climatic+and+geochemical+global+change+and+its+biological+consequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Paleoproterozoic+snowball+earth%3A+extreme+climatic+and+geochemical+global+change+and+its+biological+consequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000728-Bodiselitsch1" id="pone.0000728-Bodiselitsch1"></a>Bodiselitsch B, Koeberl C, Master S, Reimold WU (2005) Estimating duration and intensity of Neoproterozoic snowball glaciations from Ir anomalies. Science  308: 239–242.  <ul class="find" data-citedArticleID="976416" data-doi="10.1126/science.1104657"><li><a href="http://dx.doi.org/10.1126/science.1104657" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+duration+and+intensity+of+Neoproterozoic+snowball+glaciations+from+Ir+anomalies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+duration+and+intensity+of+Neoproterozoic+snowball+glaciations+from+Ir+anomalies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000728-Gingerich1" id="pone.0000728-Gingerich1"></a>Gingerich PD (2006) Environment and evolution through the Paleocene-Eocene thermal maximum. Trends Ecol Evol  21: 246–253.  <ul class="find" data-citedArticleID="976480" data-doi="10.1016/j.tree.2006.03.006"><li><a href="http://dx.doi.org/10.1016/j.tree.2006.03.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Environment+and+evolution+through+the+Paleocene-Eocene+thermal+maximum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Environment+and+evolution+through+the+Paleocene-Eocene+thermal+maximum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000728-Martin1" id="pone.0000728-Martin1"></a>Martin AP, Costello EKM, A F, Nemergut DR, Schmidt SK (2004) The rate and pattern of cladogenesis in microbes. Evolution  58: 946–955.  <ul class="find" data-citedArticleID="976456" data-doi="10.1111/j.0014-3820.2004.tb00429.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2004.tb00429.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+rate+and+pattern+of+cladogenesis+in+microbes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+rate+and+pattern+of+cladogenesis+in+microbes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000728-Kocher1" id="pone.0000728-Kocher1"></a>Kocher TD (2004) Adaptive evolution and explosive speciation: the cichlid fish model. Nat Rev Genet  5: 288–298.  <ul class="find" data-citedArticleID="976422" data-doi="10.1038/nrg1316"><li><a href="http://dx.doi.org/10.1038/nrg1316" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+and+explosive+speciation%3A+the+cichlid+fish+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+and+explosive+speciation%3A+the+cichlid+fish+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000728-Miller1" id="pone.0000728-Miller1"></a>Miller SL, Bada JL (1988) Submarine hot springs and the origin of life. Nature  334: 609–611.  <ul class="find" data-citedArticleID="976394" data-doi="10.1038/334609a0"><li><a href="http://dx.doi.org/10.1038/334609a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Submarine+hot+springs+and+the+origin+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Submarine+hot+springs+and+the+origin+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000728-Sluijs1" id="pone.0000728-Sluijs1"></a>Sluijs A, Schouten S, Pagani M, Woltering M, Brinkhuis H, et al.  (2006) Subtropical Arctic Ocean temperatures during the Palaeocene/Eocene thermal maximum. Nature  441: 610–613.  <ul class="find" data-citedArticleID="976398" data-doi="10.1038/nature04668"><li><a href="http://dx.doi.org/10.1038/nature04668" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subtropical+Arctic+Ocean+temperatures+during+the+Palaeocene%2FEocene+thermal+maximum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subtropical+Arctic+Ocean+temperatures+during+the+Palaeocene%2FEocene+thermal+maximum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000728-Jenkyns1" id="pone.0000728-Jenkyns1"></a>Jenkyns HC, Forster A, Schouten S, Sinninghe Damste JS (2004) High temperatures in the Late Cretaceous Arctic Ocean. Nature  432: 888–892.  <ul class="find" data-citedArticleID="976500" data-doi="10.1038/nature03143"><li><a href="http://dx.doi.org/10.1038/nature03143" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+temperatures+in+the+Late+Cretaceous+Arctic+Ocean." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+temperatures+in+the+Late+Cretaceous+Arctic+Ocean.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000728-Wilson1" id="pone.0000728-Wilson1"></a>Wilson PA, Norris RD (2001) Warm tropical ocean surface and global anoxia during the mid-Cretaceous period. Nature  412: 425–429.  <ul class="find" data-citedArticleID="976424" data-doi="10.1038/35086553"><li><a href="http://dx.doi.org/10.1038/35086553" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Warm+tropical+ocean+surface+and+global+anoxia+during+the+mid-Cretaceous+period." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Warm+tropical+ocean+surface+and+global+anoxia+during+the+mid-Cretaceous+period.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000728-Tripati1" id="pone.0000728-Tripati1"></a>Tripati A, Backman J, Elderfield H, Ferretti P (2005) Eocene bipolar glaciation associated with global carbon cycle changes. Nature  436: 341–346.  <ul class="find" data-citedArticleID="976490" data-doi="10.1038/nature03874"><li><a href="http://dx.doi.org/10.1038/nature03874" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eocene+bipolar+glaciation+associated+with+global+carbon+cycle+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eocene+bipolar+glaciation+associated+with+global+carbon+cycle+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000728-Fischer1" id="pone.0000728-Fischer1"></a>Fischer AG (1960) Latitudinal variations in organic diversity. Evolution  14: 64–81.  <ul class="find" data-citedArticleID="976450" data-doi="10.2307/2405923"><li><a href="http://dx.doi.org/10.2307/2405923" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Latitudinal+variations+in+organic+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Latitudinal+variations+in+organic+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000728-Edgcomb1" id="pone.0000728-Edgcomb1"></a>Edgcomb VP, Kysela DT, Teske A, de Vera Gomez A, Sogin ML (2002) Benthic eukaryotic diversity in the Guaymas Basin hydrothermal vent environment. Proc Natl Acad Sci U S A  99: 7658–7662.  <ul class="find" data-citedArticleID="976436" data-doi="10.1073/pnas.062186399"><li><a href="http://dx.doi.org/10.1073/pnas.062186399" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Benthic+eukaryotic+diversity+in+the+Guaymas+Basin+hydrothermal+vent+environment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Benthic+eukaryotic+diversity+in+the+Guaymas+Basin+hydrothermal+vent+environment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000728-LopezGarcia1" id="pone.0000728-LopezGarcia1"></a>Lopez-Garcia P, Philippe H, Gail F, Moreira D (2003) Autochthonous eukaryotic diversity in hydrothermal sediment and experimental microcolonizers at the Mid-Atlantic Ridge. Proc Natl Acad Sci USA  100: 697–702.  <ul class="find" data-citedArticleID="976426" data-doi="10.1073/pnas.0235779100"><li><a href="http://dx.doi.org/10.1073/pnas.0235779100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Autochthonous+eukaryotic+diversity+in+hydrothermal+sediment+and+experimental+microcolonizers+at+the+Mid-Atlantic+Ridge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Autochthonous+eukaryotic+diversity+in+hydrothermal+sediment+and+experimental+microcolonizers+at+the+Mid-Atlantic+Ridge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000728-Cuny1" id="pone.0000728-Cuny1"></a>Cuny J, Rhines PB, Niiler PP, Bacon S (2002) Labrador Sea boundary currents and the fate of the Irminger Sea Water. J of Phys Ocean  32: 627–647.  <ul class="find" data-citedArticleID="976434" data-doi="10.1175/1520-0485(2002)032<0627:lsbcat&gt;2.0.co;2"><li><a href="http://dx.doi.org/10.1175/1520-0485(2002)032<0627:lsbcat&gt;2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Labrador+Sea+boundary+currents+and+the+fate+of+the+Irminger+Sea+Water." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Labrador+Sea+boundary+currents+and+the+fate+of+the+Irminger+Sea+Water.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000728-Stoeck1" id="pone.0000728-Stoeck1"></a>Stoeck T, Epstein S (2003) Novel eukaryotic lineages inferred from small-subunit rRNA analyses of oxygen-depleted marine environments. Appl Environ Microbiol  69: 2657–2663.  <ul class="find" data-citedArticleID="976458" data-doi="10.1128/aem.69.5.2657-2663.2003"><li><a href="http://dx.doi.org/10.1128/aem.69.5.2657-2663.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+eukaryotic+lineages+inferred+from+small-subunit+rRNA+analyses+of+oxygen-depleted+marine+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+eukaryotic+lineages+inferred+from+small-subunit+rRNA+analyses+of+oxygen-depleted+marine+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000728-Dawson1" id="pone.0000728-Dawson1"></a>Dawson SC, Pace NR (2002) Novel kingdom-level eukaryotic diversity in anoxic environments. Proc Natl Acad Sci U S A  99: 8324–8329.  <ul class="find" data-citedArticleID="976420" data-doi="10.1073/pnas.062169599"><li><a href="http://dx.doi.org/10.1073/pnas.062169599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+kingdom-level+eukaryotic+diversity+in+anoxic+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+kingdom-level+eukaryotic+diversity+in+anoxic+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000728-Jeon1" id="pone.0000728-Jeon1"></a>Jeon SO, Bunge J, Stoeck T, Barger KJ, Hong SH, et al.  (2006) Synthetic statistical approach reveals a high degree of richness of microbial eukaryotes in an anoxic water column. Appl Environ Microbiol  72: 6578–6583.  <ul class="find" data-citedArticleID="976388" data-doi="10.1128/aem.00787-06"><li><a href="http://dx.doi.org/10.1128/aem.00787-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synthetic+statistical+approach+reveals+a+high+degree+of+richness+of+microbial+eukaryotes+in+an+anoxic+water+column." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synthetic+statistical+approach+reveals+a+high+degree+of+richness+of+microbial+eukaryotes+in+an+anoxic+water+column.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000728-Stoeck2" id="pone.0000728-Stoeck2"></a>Stoeck T, Hayward B, Taylor GT, Varela R, Epstein SS (2006) A multiple PCR-primer approach to access the microeukaryotic diversity in the anoxic Cariaco Basin (Caribbean Sea). Protist  157: 31–43.  <ul class="find" data-citedArticleID="976400" data-doi="10.1016/j.protis.2005.10.004"><li><a href="http://dx.doi.org/10.1016/j.protis.2005.10.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+multiple+PCR-primer+approach+to+access+the+microeukaryotic+diversity+in+the+anoxic+Cariaco+Basin+%28Caribbean+Sea%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+multiple+PCR-primer+approach+to+access+the+microeukaryotic+diversity+in+the+anoxic+Cariaco+Basin+%28Caribbean+Sea%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000728-Stoeck3" id="pone.0000728-Stoeck3"></a>Stoeck T, Taylor G, Epstein SS (2003) Novel eukaryotes from a permanently anoxic Cariaco Basin (Caribbean Sea). Appl Environ Microbiol  69: 5656–5663.  <ul class="find" data-citedArticleID="976452" data-doi="10.1128/aem.69.9.5656-5663.2003"><li><a href="http://dx.doi.org/10.1128/aem.69.9.5656-5663.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+eukaryotes+from+a+permanently+anoxic+Cariaco+Basin+%28Caribbean+Sea%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+eukaryotes+from+a+permanently+anoxic+Cariaco+Basin+%28Caribbean+Sea%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000728-Hong1" id="pone.0000728-Hong1"></a>Hong SH, Bunge J, Jeon SO, Epstein SS (2006) Predicting microbial species richness. Proc Natl Acad Sci U S A  103: 117–122.  <ul class="find" data-citedArticleID="976406" data-doi="10.1073/pnas.0507245102"><li><a href="http://dx.doi.org/10.1073/pnas.0507245102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+microbial+species+richness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+microbial+species+richness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000728-Lomolino1" id="pone.0000728-Lomolino1"></a>Lomolino MV, Riddle BR, Brown JH (2006) Biogeography. Sunderland, MA: Sinauer.   <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0000728-Lawley1" id="pone.0000728-Lawley1"></a>Lawley B, Ripley S, Bridge P, Convey P (2004) Molecular analysis of geographic patterns of eukaryotic diversity in Antarctic soils. Appl Environ Microbiol  70: 5963–5972.  <ul class="find" data-citedArticleID="976390" data-doi="10.1128/aem.70.10.5963-5972.2004"><li><a href="http://dx.doi.org/10.1128/aem.70.10.5963-5972.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+analysis+of+geographic+patterns+of+eukaryotic+diversity+in+Antarctic+soils." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+analysis+of+geographic+patterns+of+eukaryotic+diversity+in+Antarctic+soils.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000728-Suzuki1" id="pone.0000728-Suzuki1"></a>Suzuki MT, Giovannoni SJ (1996) Bias caused by template annealing in the amplification of mixtures of 16S rRNA genes by PCR. Appl Environ Microbiol  62: 625–630.  <ul class="find" data-citedArticleID="976402"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bias+caused+by+template+annealing+in+the+amplification+of+mixtures+of+16S+rRNA+genes+by+PCR.&amp;auth=&amp;atitle=Bias+caused+by+template+annealing+in+the+amplification+of+mixtures+of+16S+rRNA+genes+by+PCR." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bias+caused+by+template+annealing+in+the+amplification+of+mixtures+of+16S+rRNA+genes+by+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bias+caused+by+template+annealing+in+the+amplification+of+mixtures+of+16S+rRNA+genes+by+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000728-Lovejoy1" id="pone.0000728-Lovejoy1"></a>Lovejoy C, Massana R, Pedros-Alio C (2006) Diversity and distribution of marine microbial eukaryotes in the Arctic Ocean and adjacent seas. Appl Environ Microbiol  72: 3085–3095.  <ul class="find" data-citedArticleID="976386" data-doi="10.1128/aem.72.5.3085-3095.2006"><li><a href="http://dx.doi.org/10.1128/aem.72.5.3085-3095.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+and+distribution+of+marine+microbial+eukaryotes+in+the+Arctic+Ocean+and+adjacent+seas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+and+distribution+of+marine+microbial+eukaryotes+in+the+Arctic+Ocean+and+adjacent+seas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000728-Behnke1" id="pone.0000728-Behnke1"></a>Behnke A, Bunge J, Barger K, Breiner H-W, Alla V, et al.  (2006) Microeukaryote community patterns along an O2/H2S gradient in a super-sulfidic anoxic fjord (Framvaren, Norway). Appl Environ Microbiol  72: 3626–3636.  <ul class="find" data-citedArticleID="976466" data-doi="10.1128/aem.72.5.3626-3636.2006"><li><a href="http://dx.doi.org/10.1128/aem.72.5.3626-3636.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microeukaryote+community+patterns+along+an+O2%2FH2S+gradient+in+a+super-sulfidic+anoxic+fjord+%28Framvaren%2C+Norway%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microeukaryote+community+patterns+along+an+O2%2FH2S+gradient+in+a+super-sulfidic+anoxic+fjord+%28Framvaren%2C+Norway%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000728-Zuendorf1" id="pone.0000728-Zuendorf1"></a>Zuendorf A, Bunge J, Behnke A, Barger KJ, Stoeck T (2006) Diversity estimates of microeukaryotes below the chemocline of the anoxic Mariager Fjord, Denmark. FEMS Microbiol Ecol  58: 476–491.  <ul class="find" data-citedArticleID="976384" data-doi="10.1111/j.1574-6941.2006.00171.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6941.2006.00171.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+estimates+of+microeukaryotes+below+the+chemocline+of+the+anoxic+Mariager+Fjord%2C+Denmark." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+estimates+of+microeukaryotes+below+the+chemocline+of+the+anoxic+Mariager+Fjord%2C+Denmark.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000728-Naviaux1" id="pone.0000728-Naviaux1"></a>Naviaux RK, Good B, McPherson JD, Steffen DL, Markusic D, et al.  (2005) Sand DNA—a genetic library of life at the water's edge. Mar Ecol Prog Ser  301: 9–22.  <ul class="find" data-citedArticleID="976496" data-doi="10.3354/meps301009"><li><a href="http://dx.doi.org/10.3354/meps301009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sand+DNA%E2%80%94a+genetic+library+of+life+at+the+water%27s+edge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sand+DNA%E2%80%94a+genetic+library+of+life+at+the+water%27s+edge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000728-AmaralZettler1" id="pone.0000728-AmaralZettler1"></a>Amaral Zettler LA, Gomez F, Zettler E, Keenan BG, Amils R, et al.  (2002) Microbiology: eukaryotic diversity in Spain's River of Fire. Nature  417: 137.  <ul class="find" data-citedArticleID="976502" data-doi="10.1038/417137a"><li><a href="http://dx.doi.org/10.1038/417137a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbiology%3A+eukaryotic+diversity+in+Spain%27s+River+of+Fire." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbiology%3A+eukaryotic+diversity+in+Spain%27s+River+of+Fire.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000728-LopezGarcia2" id="pone.0000728-LopezGarcia2"></a>Lopez-Garcia P, Lopez-Lopez A, Moreira D, Rodriguez-Valera F (2001) Diversity of free-living prokaryotes from a deep-sea site at the Antarctic Polar Front. FEMS Microbiol Ecol  36: 193–202.  <ul class="find" data-citedArticleID="976446" data-doi="10.1016/s0168-6496(01)00133-7"><li><a href="http://dx.doi.org/10.1016/s0168-6496(01)00133-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+of+free-living+prokaryotes+from+a+deep-sea+site+at+the+Antarctic+Polar+Front." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+of+free-living+prokaryotes+from+a+deep-sea+site+at+the+Antarctic+Polar+Front.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000728-MoonvanderStaay1" id="pone.0000728-MoonvanderStaay1"></a>Moon-van der Staay SY, De Wachter R, Vaulot D (2001) Oceanic 18S rDNA sequences from picoplankton reveal unsuspected eukaryotic diversity. Nature  409: 607–610.  <ul class="find" data-citedArticleID="976430" data-doi="10.1038/35054541"><li><a href="http://dx.doi.org/10.1038/35054541" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oceanic+18S+rDNA+sequences+from+picoplankton+reveal+unsuspected+eukaryotic+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oceanic+18S+rDNA+sequences+from+picoplankton+reveal+unsuspected+eukaryotic+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000728-Leipe1" id="pone.0000728-Leipe1"></a>Leipe DD, Wainright PO, Gunderson JH, Porter D, Patterson DJ, et al.  (1994) The stramenopiles from a molecular perspective: small subunit rDNA sequences from <em>Labyrinthuloides minuta</em> and <em>Cafeteria roenbergensis</em>. Phycologia  33: 369–377.  <ul class="find" data-citedArticleID="976474" data-doi="10.2216/i0031-8884-33-5-369.1"><li><a href="http://dx.doi.org/10.2216/i0031-8884-33-5-369.1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+stramenopiles+from+a+molecular+perspective%3A+small+subunit+rDNA+sequences+from+Labyrinthuloides+minuta+and+Cafeteria+roenbergensis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+stramenopiles+from+a+molecular+perspective%3A+small+subunit+rDNA+sequences+from+Labyrinthuloides+minuta+and+Cafeteria+roenbergensis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000728-Steenkamp1" id="pone.0000728-Steenkamp1"></a>Steenkamp ET, Wright J, Baldauf SL (2006) The protistan origins of animals and fungi. Mol Biol Evol  23: 93–106.  <ul class="find" data-citedArticleID="976404" data-doi="10.1093/molbev/msj011"><li><a href="http://dx.doi.org/10.1093/molbev/msj011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+protistan+origins+of+animals+and+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+protistan+origins+of+animals+and+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000728-Kuvardina1" id="pone.0000728-Kuvardina1"></a>Kuvardina ON, Leander BS, Aleshin VV, Myl'nikov AP, Keeling PJ, et al.  (2002) The phylogeny of colpodellids (Alveolata) using small subunit rRNA gene sequences suggests they are the free-living sister group to apicomplexans. J Eukaryot Microbiol  49: 498–504.  <ul class="find" data-citedArticleID="976374" data-doi="10.1111/j.1550-7408.2002.tb00235.x"><li><a href="http://dx.doi.org/10.1111/j.1550-7408.2002.tb00235.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+phylogeny+of+colpodellids+%28Alveolata%29+using+small+subunit+rRNA+gene+sequences+suggests+they+are+the+free-living+sister+group+to+apicomplexans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+phylogeny+of+colpodellids+%28Alveolata%29+using+small+subunit+rRNA+gene+sequences+suggests+they+are+the+free-living+sister+group+to+apicomplexans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000728-Leander1" id="pone.0000728-Leander1"></a>Leander BS, Kuvardina ON, Aleshin VV, Mylnikov AP, Keeling PJ (2003) Molecular phylogeny and surface morphology of <em>Colpodella edax</em> (Alveolata): insights into the phagotrophic ancestry of apicomplexans. J Eukaryot Microbiol  50: 334–340.  <ul class="find" data-citedArticleID="976414" data-doi="10.1111/j.1550-7408.2003.tb00145.x"><li><a href="http://dx.doi.org/10.1111/j.1550-7408.2003.tb00145.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogeny+and+surface+morphology+of+Colpodella+edax+%28Alveolata%29%3A+insights+into+the+phagotrophic+ancestry+of+apicomplexans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogeny+and+surface+morphology+of+Colpodella+edax+%28Alveolata%29%3A+insights+into+the+phagotrophic+ancestry+of+apicomplexans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000728-Gray1" id="pone.0000728-Gray1"></a>Gray MW, Burger G, Lang BF (1999) Mitochondrial evolution. Science  283: 1476–1481.  <ul class="find" data-citedArticleID="976432" data-doi="10.1126/science.283.5407.1476"><li><a href="http://dx.doi.org/10.1126/science.283.5407.1476" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000728-Lang1" id="pone.0000728-Lang1"></a>Lang BF, Burger G, O'Kelly CJ, Cedergren R, Golding GB, et al.  (1997) An ancestral mitochondrial DNA resembling a eubacterial genome in miniature. Nature  387: 493–497.  <ul class="find" data-citedArticleID="976476" data-doi="10.1038/387493a0"><li><a href="http://dx.doi.org/10.1038/387493a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+ancestral+mitochondrial+DNA+resembling+a+eubacterial+genome+in+miniature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+ancestral+mitochondrial+DNA+resembling+a+eubacterial+genome+in+miniature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000728-OKelly1" id="pone.0000728-OKelly1"></a>O'Kelly CJ (1993) The jakobid flagellates: structural features of <em>Jakoba</em>, <em>Reclinomonas</em> and <em>Histiona</em> and implications for the early diversification of eukaryotes. J Euk Microbiol  40: 627–636.  <ul class="find" data-citedArticleID="976438" data-doi="10.1111/j.1550-7408.1993.tb06120.x"><li><a href="http://dx.doi.org/10.1111/j.1550-7408.1993.tb06120.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+jakobid+flagellates%3A+structural+features+of+Jakoba%2C+Reclinomonas+and+Histiona+and+implications+for+the+early+diversification+of+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+jakobid+flagellates%3A+structural+features+of+Jakoba%2C+Reclinomonas+and+Histiona+and+implications+for+the+early+diversification+of+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000728-Ahlberg1" id="pone.0000728-Ahlberg1"></a>Ahlberg A, Herman AB, Raikevich M, Rees A, Spicer RA (2002) Enigmatic Late Cretaceous high palaeo-latitude lonestones in Chukotka, northeasternmost Asia. GFF  124: 197–199.  <ul class="find" data-citedArticleID="976494" data-doi="10.1080/11035890201244197"><li><a href="http://dx.doi.org/10.1080/11035890201244197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enigmatic+Late+Cretaceous+high+palaeo-latitude+lonestones+in+Chukotka%2C+northeasternmost+Asia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enigmatic+Late+Cretaceous+high+palaeo-latitude+lonestones+in+Chukotka%2C+northeasternmost+Asia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000728-Jensen1" id="pone.0000728-Jensen1"></a> Jensen DB, Christensen KD, editors.  (2003) The diodiversity of Greenland–a country study. Pinngortitalerifik: Gronlands Naturinstitut.   No. 55.  <ul class="find-nolinks"></ul></li><li><span class="label">45.
              </span><a name="pone.0000728-Leander2" id="pone.0000728-Leander2"></a>Leander BS, Lloyd SA, Marshall W, Landers SC (2006) Phylogeny of marine Gregarines (Apicomplexa)–<em>Pterospora</em>, <em>Lithocystis</em> and <em>Lankesteria</em>–and the origin(s) of coelomic parasitism. Protist  157: 45–60.  <ul class="find" data-citedArticleID="976448" data-doi="10.1016/j.protis.2005.10.002"><li><a href="http://dx.doi.org/10.1016/j.protis.2005.10.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+of+marine+Gregarines+%28Apicomplexa%29%E2%80%93Pterospora%2C+Lithocystis+and+Lankesteria%E2%80%93and+the+origin%28s%29+of+coelomic+parasitism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+of+marine+Gregarines+%28Apicomplexa%29%E2%80%93Pterospora%2C+Lithocystis+and+Lankesteria%E2%80%93and+the+origin%28s%29+of+coelomic+parasitism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000728-Simpson1" id="pone.0000728-Simpson1"></a>Simpson AGB, Patterson DJ (1996) Ultrastructure and identification of the predatory flagellate <em>Colpodella pugnax</em> Cienkowski (Apicomplexa) with a description of <em>Colpodella turpis</em> n. sp. and a review of the genus. Syst Parasitol  33: 187–198.  <ul class="find" data-citedArticleID="976498" data-doi="10.1007/bf01531200"><li><a href="http://dx.doi.org/10.1007/bf01531200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ultrastructure+and+identification+of+the+predatory+flagellate+Colpodella+pugnax+Cienkowski+%28Apicomplexa%29+with+a+description+of+Colpodella+turpis+n.+sp.+and+a+review+of+the+genus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ultrastructure+and+identification+of+the+predatory+flagellate+Colpodella+pugnax+Cienkowski+%28Apicomplexa%29+with+a+description+of+Colpodella+turpis+n.+sp.+and+a+review+of+the+genus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000728-Cameron1" id="pone.0000728-Cameron1"></a>Cameron A, Read J, Tranter R, Winter VJ, Sessions RB, et al.  (2004) Identification and activity of a series of azole-based compounds with lactate dehydrogenase-directed anti-malarial activity. J Biol Chem  279: 31429–31439.  <ul class="find" data-citedArticleID="976472" data-doi="10.1074/jbc.m402433200"><li><a href="http://dx.doi.org/10.1074/jbc.m402433200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+activity+of+a+series+of+azole-based+compounds+with+lactate+dehydrogenase-directed+anti-malarial+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+activity+of+a+series+of+azole-based+compounds+with+lactate+dehydrogenase-directed+anti-malarial+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000728-LangUnnasch1" id="pone.0000728-LangUnnasch1"></a>Lang-Unnasch N, Murphy AD (1998) Metabolic changes of the malaria parasite during the transition from the human to the mosquito host. Annu Rev Microbiol  52: 561–590.  <ul class="find" data-citedArticleID="976470" data-doi="10.1146/annurev.micro.52.1.561"><li><a href="http://dx.doi.org/10.1146/annurev.micro.52.1.561" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolic+changes+of+the+malaria+parasite+during+the+transition+from+the+human+to+the+mosquito+host." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolic+changes+of+the+malaria+parasite+during+the+transition+from+the+human+to+the+mosquito+host.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000728-Akhmanova1" id="pone.0000728-Akhmanova1"></a>Akhmanova A, Voncken FG, Hosea KM, Harhangi H, Keltjens JT, et al.  (1999) A hydrogenosome with pyruvate formate-lyase: anaerobic chytrid fungi use an alternative route for pyruvate catabolism. Mol Microbiol  32: 1103–1114.  <ul class="find" data-citedArticleID="976478" data-doi="10.1046/j.1365-2958.1999.01434.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01434.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+hydrogenosome+with+pyruvate+formate-lyase%3A+anaerobic+chytrid+fungi+use+an+alternative+route+for+pyruvate+catabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+hydrogenosome+with+pyruvate+formate-lyase%3A+anaerobic+chytrid+fungi+use+an+alternative+route+for+pyruvate+catabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000728-Voncken1" id="pone.0000728-Voncken1"></a>Voncken F, Boxma B, Tjaden J, Akhmanova A, Huynen M, et al.  (2002) Multiple origins of hydrogenosomes: functional and phylogenetic evidence from the ADP/ATP carrier of the anaerobic chytrid <em>Neocallimastix</em> sp. Mol Microbiol  44: 1441–1454.  <ul class="find" data-citedArticleID="976376" data-doi="10.1046/j.1365-2958.2002.02959.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.02959.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+origins+of+hydrogenosomes%3A+functional+and+phylogenetic+evidence+from+the+ADP%2FATP+carrier+of+the+anaerobic+chytrid+Neocallimastix+sp." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+origins+of+hydrogenosomes%3A+functional+and+phylogenetic+evidence+from+the+ADP%2FATP+carrier+of+the+anaerobic+chytrid+Neocallimastix+sp.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000728-Thompson1" id="pone.0000728-Thompson1"></a>Thompson JD, Higgens DG, Gibson TJ (1994) CLUSTAL W: Improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties, and weight matrix choice. Nucleic Acids Res  33: 4673–4680.  <ul class="find" data-citedArticleID="976440" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+Improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+positions-specific+gap+penalties%2C+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+Improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+positions-specific+gap+penalties%2C+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000728-Maidak1" id="pone.0000728-Maidak1"></a>Maidak BL, Cole JR, Lilburn TG, Parker CT Jr, Saxman PR, et al.  (2001) The RDP-II (Ribosomal Database Project). Nucleic Acids Res  29: 173–174.  <ul class="find" data-citedArticleID="976396" data-doi="10.1093/nar/29.1.173"><li><a href="http://dx.doi.org/10.1093/nar/29.1.173" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+RDP-II+%28Ribosomal+Database+Project%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+RDP-II+%28Ribosomal+Database+Project%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000728-RobisonCox1" id="pone.0000728-RobisonCox1"></a>Robison-Cox JF, Bateson MM, Ward DM (1995) Evaluation of nearest-neighbor methods for detection of chimeric small-subunit rRNA sequences. Appl Environ Microbiol  61: 1240–1245.  <ul class="find" data-citedArticleID="976462"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evaluation+of+nearest-neighbor+methods+for+detection+of+chimeric+small-subunit+rRNA+sequences.&amp;auth=&amp;atitle=Evaluation+of+nearest-neighbor+methods+for+detection+of+chimeric+small-subunit+rRNA+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+nearest-neighbor+methods+for+detection+of+chimeric+small-subunit+rRNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+nearest-neighbor+methods+for+detection+of+chimeric+small-subunit+rRNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000728-Barger1" id="pone.0000728-Barger1"></a>Barger KJ-A (2006) Mixtures of exponential distributions to describe the distribution of Poisson means in estimating the number of unobserved classes. Ithaca: Cornell University.   <a href="http://hdl.handle.net/1813/2953">2953</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">55.
              </span><a name="pone.0000728-Chao1" id="pone.0000728-Chao1"></a>Chao A, Bunge J (2002) Estimating the number of species in a stochastic abundance model. Biometrics  58: 531–539.  <ul class="find" data-citedArticleID="976380" data-doi="10.1111/j.0006-341x.2002.00531.x"><li><a href="http://dx.doi.org/10.1111/j.0006-341x.2002.00531.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+the+number+of+species+in+a+stochastic+abundance+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+the+number+of+species+in+a+stochastic+abundance+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000728-Shen1" id="pone.0000728-Shen1"></a>Shen T-J, Chao A, Lin C-F (2003) Predicting the number of new species in a further taxonomic sampling. Ecology  84: 798–804.  <ul class="find" data-citedArticleID="976486" data-doi="10.1890/0012-9658(2003)084[0798:ptnons]2.0.co;2"><li><a href="http://dx.doi.org/10.1890/0012-9658(2003)084[0798:ptnons]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+the+number+of+new+species+in+a+further+taxonomic+sampling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+the+number+of+new+species+in+a+further+taxonomic+sampling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000728-Altschul1" id="pone.0000728-Altschul1"></a>Altschul SF, Madde TL, Schaffer AA, Zhang JH, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="976428" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000728-Ludwig1" id="pone.0000728-Ludwig1"></a>Ludwig W, Strunk O, Westram R, Richter L, Meier H, et al.  (2004) ARB: a software environment for sequence data. Nucleic Acids Res  32: 1363–1371.  <ul class="find" data-citedArticleID="976410" data-doi="10.1093/nar/gkh293"><li><a href="http://dx.doi.org/10.1093/nar/gkh293" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ARB%3A+a+software+environment+for+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ARB%3A+a+software+environment+for+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000728-Swofford1" id="pone.0000728-Swofford1"></a>Swofford DL (2001) PAUP*. Phylogenetic analysis using parsimony (and other methods). 4.0b6. ed. Sunderland, MA: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">60.
              </span><a name="pone.0000728-Posada1" id="pone.0000728-Posada1"></a>Posada D, Crandall KA (1998) MODELTEST: testing the model of DNA substitution. Bioinformatics  14: 817–818.  <ul class="find" data-citedArticleID="976460" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MODELTEST%3A+testing+the+model+of+DNA+substitution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MODELTEST%3A+testing+the+model+of+DNA+substitution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000728-Medlin1" id="pone.0000728-Medlin1"></a>Medlin L, Elwood HJ, Stickel S, Sogin ML (1988) The characterization of enzymatically amplified eukaryotic 16S-like rRNA-coding regions. Gene  71: 491–499.  <ul class="find" data-citedArticleID="976412" data-doi="10.1016/0378-1119(88)90066-2"><li><a href="http://dx.doi.org/10.1016/0378-1119(88)90066-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+characterization+of+enzymatically+amplified+eukaryotic+16S-like+rRNA-coding+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+characterization+of+enzymatically+amplified+eukaryotic+16S-like+rRNA-coding+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000728-LopezGarcia3" id="pone.0000728-LopezGarcia3"></a>Lopez-Garcia P, Rodriguez-Valera F, Pedros-Alio C, Moreira D (2001) Unexpected diversity of small eukaryotes in deep-sea Antarctic plankton. Nature  409: 603–607.  <ul class="find" data-citedArticleID="976488" data-doi="10.1038/35054537"><li><a href="http://dx.doi.org/10.1038/35054537" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unexpected+diversity+of+small+eukaryotes+in+deep-sea+Antarctic+plankton." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unexpected+diversity+of+small+eukaryotes+in+deep-sea+Antarctic+plankton.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000728-Lane1" id="pone.0000728-Lane1"></a>Lane DJ (1991)  16S/23S rRNA sequencing. In: Stackebrandt E, Goodfellow M, editors. Nucleic acid techniques in bacterial systematics. Chichester, UK: John Wiley&amp;Sons. pp. 115–175.  <ul class="find-nolinks"></ul></li><li><span class="label">64.
              </span><a name="pone.0000728-Shopsin1" id="pone.0000728-Shopsin1"></a>Shopsin B, Gomez M, Montgomery SO, Smith DH, Waddington M, et al.  (1999) Evaluation of protein A gene polymorphic region DNA sequencing for typing of <em>Staphylococcus aureus</em> strains. J Clin Microbiol  37: 3556–3563.  <ul class="find" data-citedArticleID="976468"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evaluation+of+protein+A+gene+polymorphic+region+DNA+sequencing+for+typing+of+Staphylococcus+aureus+strains.&amp;auth=&amp;atitle=Evaluation+of+protein+A+gene+polymorphic+region+DNA+sequencing+for+typing+of+Staphylococcus+aureus+strains." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+protein+A+gene+polymorphic+region+DNA+sequencing+for+typing+of+Staphylococcus+aureus+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+protein+A+gene+polymorphic+region+DNA+sequencing+for+typing+of+Staphylococcus+aureus+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000728-Fried1" id="pone.0000728-Fried1"></a>Fried J, Ludwig W, Psenner R, Schleifer KH (2002) Improvement of ciliate identification and quantification: a new protocol for fluorescence in situ hybridization (FISH) in combination with silver stain techniques. Syst Appl Microbiol  25: 555–571.  <ul class="find" data-citedArticleID="976372" data-doi="10.1078/07232020260517706"><li><a href="http://dx.doi.org/10.1078/07232020260517706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improvement+of+ciliate+identification+and+quantification%3A+a+new+protocol+for+fluorescence+in+situ+hybridization+%28FISH%29+in+combination+with+silver+stain+techniques." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improvement+of+ciliate+identification+and+quantification%3A+a+new+protocol+for+fluorescence+in+situ+hybridization+%28FISH%29+in+combination+with+silver+stain+techniques.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000728-Wuyts1" id="pone.0000728-Wuyts1"></a>Wuyts J, Perriere G, Van de Peer Y (2004) The European ribosomal RNA database. Nucleic Acids Res  32: D101–D103.  <ul class="find" data-citedArticleID="976444" data-doi="10.1093/nar/gkh065"><li><a href="http://dx.doi.org/10.1093/nar/gkh065" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+European+ribosomal+RNA+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+European+ribosomal+RNA+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000728-Elwood1" id="pone.0000728-Elwood1"></a>Elwood HJ, Olsen GJ, Sogin ML (1985) The small-subunit ribosomal RNA gene sequences from the hypotrichous ciliates <em>Oxytricha nova</em> and <em>Stylonychia pustulata</em>. Mol Biol Evol  2: 399–410.  <ul class="find" data-citedArticleID="976370"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+small-subunit+ribosomal+RNA+gene+sequences+from+the+hypotrichous+ciliates+Oxytricha+nova+and+Stylonychia+pustulata.&amp;auth=&amp;atitle=The+small-subunit+ribosomal+RNA+gene+sequences+from+the+hypotrichous+ciliates+Oxytricha+nova+and+Stylonychia+pustulata." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+small-subunit+ribosomal+RNA+gene+sequences+from+the+hypotrichous+ciliates+Oxytricha+nova+and+Stylonychia+pustulata." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+small-subunit+ribosomal+RNA+gene+sequences+from+the+hypotrichous+ciliates+Oxytricha+nova+and+Stylonychia+pustulata.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000728-Johnson1" id="pone.0000728-Johnson1"></a>Johnson MD, Tengs T, Oldach DW, Delwiche CF, Stoecker K (2004) Highly divergent SSU rRNA genes found in the marine ciliates <em>Myrionecta rubra</em> and <em>Mesodinium pulex</em>. Protist  155: 347–359.  <ul class="find" data-citedArticleID="976464" data-doi="10.1078/1434461041844222"><li><a href="http://dx.doi.org/10.1078/1434461041844222" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+divergent+SSU+rRNA+genes+found+in+the+marine+ciliates+Myrionecta+rubra+and+Mesodinium+pulex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+divergent+SSU+rRNA+genes+found+in+the+marine+ciliates+Myrionecta+rubra+and+Mesodinium+pulex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.XML" value="135517"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.PDF" value="289246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.t001.PNG_L" value="215800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.t001.PNG_M" value="298481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.t001.PNG_S" value="24942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.t001.TIF" value="1076252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.t001.PNG_I" value="40121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.t002.PNG_L" value="161630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.t002.PNG_M" value="240185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.t002.PNG_S" value="18566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.t002.TIF" value="705162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.t002.PNG_I" value="33533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g001.PNG_L" value="152606"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g001.PNG_M" value="116438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g001.PNG_S" value="13029"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g001.TIF" value="325690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g001.PNG_I" value="31729"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g002.PNG_L" value="169238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g002.PNG_M" value="394084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g002.PNG_S" value="23546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g002.TIF" value="788148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g002.PNG_I" value="76681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g003.PNG_L" value="187741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g003.PNG_M" value="375074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g003.PNG_S" value="31327"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g003.TIF" value="903288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g003.PNG_I" value="115688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g004.PNG_L" value="202693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g004.PNG_M" value="399540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g004.PNG_S" value="32007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g004.TIF" value="939346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g004.PNG_I" value="124799"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g005.PNG_L" value="182594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g005.PNG_M" value="377856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g005.PNG_S" value="27452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g005.TIF" value="845122"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g005.PNG_I" value="92285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g006.PNG_L" value="129904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g006.PNG_M" value="299020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g006.PNG_S" value="24211"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g006.TIF" value="689172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g006.PNG_I" value="68424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g007.PNG_L" value="149309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g007.PNG_M" value="312338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g007.PNG_S" value="26899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g007.TIF" value="751406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g007.PNG_I" value="85593"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g008.PNG_L" value="129569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g008.PNG_M" value="304505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g008.PNG_S" value="25129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g008.TIF" value="685572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g008.PNG_I" value="80486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g009.PNG_L" value="83897"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g009.PNG_M" value="189474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g009.PNG_S" value="22912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g009.TIF" value="480014"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g009.PNG_I" value="63524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g010.PNG_L" value="132692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g010.PNG_M" value="297224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g010.PNG_S" value="26428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g010.TIF" value="671474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000728.g010.PNG_I" value="74532"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000728&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000728" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000728&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000728&volume=&issue=&title=Protistan Diversity in the Arctic: A Case of Paleoclimate Shaping Modern Biodiversity?&author_name=Thorsten%20Stoeck%2C%20Jennifer%20Kasper%2C%20John%20Bunge%2C%20Chesley%20Leslin%2C%20Valya%20Ilyin%2C%20Slava%20Epstein&start_page=1&end_page=16" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000728" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000728" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000728" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000728&amp;t=Protistan%20Diversity%20in%20the%20Arctic%3A%20A%20Case%20of%20Paleoclimate%20Shaping%20Modern%20Biodiversity%3F" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000728&title=Protistan%20Diversity%20in%20the%20Arctic%3A%20A%20Case%20of%20Paleoclimate%20Shaping%20Modern%20Biodiversity%3F&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000728&amp;title=Protistan%20Diversity%20in%20the%20Arctic%3A%20A%20Case%20of%20Paleoclimate%20Shaping%20Modern%20Biodiversity%3F" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000728" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000728&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Protistan Diversity in the Arctic: A Case of Paleoclimate Shaping Modern Biodiversity?';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000728';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Protistan%20Diversity%20in%20the%20Arctic%3A%20A%20Case%20of%20Paleoclimate%20Shaping%20Modern%20Biodiversity%3F http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000728" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000728" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24294"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hydrothermal+vents%22" title="Search for articles in the subject area:'Hydrothermal vents'"><div class="flagText">Hydrothermal vents</div></a>
              <div data-categoryid="21099" data-articleid="24294"
                   data-categoryname="Hydrothermal vents"
                   class="flagImage" title="Flag 'Hydrothermal vents' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24294"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="24294"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="24294"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribosomal+RNA%22" title="Search for articles in the subject area:'Ribosomal RNA'"><div class="flagText">Ribosomal RNA</div></a>
              <div data-categoryid="21371" data-articleid="24294"
                   data-categoryname="Ribosomal RNA"
                   class="flagImage" title="Flag 'Ribosomal RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sediment%22" title="Search for articles in the subject area:'Sediment'"><div class="flagText">Sediment</div></a>
              <div data-categoryid="21117" data-articleid="24294"
                   data-categoryname="Sediment"
                   class="flagImage" title="Flag 'Sediment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="24294"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9980'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9660'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4728&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
