

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: One-Pot, Mix-and-Read Peptide-MHC Tetramers</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001678"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001678" />

    <meta name="citation_title" content="One-Pot, Mix-and-Read Peptide-MHC Tetramers"/>
    <meta itemprop="name" content="One-Pot, Mix-and-Read Peptide-MHC Tetramers"/>

      <meta name="citation_author" content="Christian Leisner"/>
            <meta name="citation_author_institution" content="Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Nina Loeth"/>
            <meta name="citation_author_institution" content="Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Kasper Lamberth"/>
            <meta name="citation_author_institution" content="Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Sune Justesen"/>
            <meta name="citation_author_institution" content="Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Christina Sylvester-Hvid"/>
            <meta name="citation_author_institution" content="Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Esben G. Schmidt"/>
            <meta name="citation_author_institution" content="Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Mogens Claesson"/>
            <meta name="citation_author_institution" content="Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Soren Buus"/>
            <meta name="citation_author_institution" content="Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Anette Stryhn"/>
            <meta name="citation_author_institution" content="Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark"/>

    <meta name="citation_date" content="2008/2/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001678.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1678"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A novel two colour ELISPOT assay. I. Simultaneous detection of distinct types of antibody-secreting cells.; citation_author=C Czerkinsky; citation_author=Z Moldoveanu; citation_author=J Mestecky; citation_author=LA Nilsson; citation_author=O Ouchterlony; citation_journal_title=J Immunol Methods; citation_volume=115; citation_number=1; citation_pages=31-37; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=A sensitive ELISPOT assay for detection of CD8+ T lymphocytes specific for HLA class I-binding peptide epitopes derived from influenza proteins in the blood of healthy donors and melanoma patients.; citation_author=C Scheibenbogen; citation_author=KH Lee; citation_author=S Mayer; citation_author=S Stevanovic; citation_author=U Moebius; citation_journal_title=Clin Cancer Res; citation_volume=3; citation_number=2; citation_pages=221-226; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Idiotype-specific T cells in multiple myeloma stage I: an evaluation by four different functional tests.; citation_author=A Osterborg; citation_author=Q Yi; citation_author=S Bergenbrant; citation_author=G Holm; citation_author=AK Lefvert; citation_journal_title=Br J Haematol; citation_volume=89; citation_number=3; citation_pages=110-116; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Novel methods to monitor antigen-specific cytotoxic T-cell responses in cancer immunotherapy.; citation_author=P Romero; citation_author=JC Cerottini; citation_author=GA Waanders; citation_journal_title=Mol Med Today; citation_volume=4; citation_number=4; citation_pages=305-312; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Detection of antigen-specific T cell cytokine expression in whole blood by flow cytometry.; citation_author=MA Suni; citation_author=LJ Picker; citation_author=VC Maino; citation_journal_title=J Immunol Methods; citation_volume=212; citation_number=5; citation_pages=89-98; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A reproducible method for the enumeration of functional (cytokine producing) versus non-functional peptide-specific cytotoxic T lymphocytes in human peripheral blood.; citation_author=SN Markovic; citation_author=WK Nevala; citation_author=CB Uhl; citation_author=E Celis; citation_author=DJ McKean; citation_journal_title=Clin Exp Immunol; citation_volume=145; citation_number=6; citation_pages=438-447; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phenotypic analysis of antigen-specific T lymphocytes.; citation_author=JD Altman; citation_author=PA Moss; citation_author=PJ Goulder; citation_author=DH Barouch; citation_author=MG McHeyzer-Williams; citation_journal_title=Science; citation_volume=274; citation_number=7; citation_pages=94-96; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Purification of cytomegalovirus-specific CD8 T cells from peripheral blood using HLA-peptide tetramers.; citation_author=RD Keenan; citation_author=J Ainsworth; citation_author=N Khan; citation_author=R Bruton; citation_author=M Cobbold; citation_journal_title=Br J Haematol; citation_volume=115; citation_number=8; citation_pages=428-434; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Expansion of CD8+ cytotoxic T cells in vitro and in vivo using MHC class I tetramers.; citation_author=P Savage; citation_author=M Millrain; citation_author=S Dimakou; citation_author=J Stebbing; citation_author=J Dyson; citation_journal_title=Tumour Biol; citation_volume=28; citation_number=9; citation_pages=70-76; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Induction of viral and tumour specific CTL responses using antibody targeted HLA class I peptide complexes.; citation_author=P Savage; citation_author=P Cowburn; citation_author=A Clayton; citation_author=S Man; citation_author=A McMichael; citation_journal_title=Br J Cancer; citation_volume=86; citation_number=10; citation_pages=1336-1342; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Purification of correctly oxidized MHC class I heavy-chain molecules under denaturing conditions: a novel strategy exploiting disulfide assisted protein folding.; citation_author=H Ferre; citation_author=E Ruffet; citation_author=T Blicher; citation_author=C Sylvester-Hvid; citation_author=LL Nielsen; citation_journal_title=Protein Sci; citation_volume=12; citation_number=11; citation_pages=551-559; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Biotinylation of proteins in vivo and in vitro using small peptide tags.; citation_author=MG Cull; citation_author=PJ Schatz; citation_journal_title=Methods Enzymol; citation_volume=326; citation_number=12; citation_pages=430-440; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=In vivo biotinylation of the major histocompatibility complex (MHC) class II/peptide complex by coexpression of BirA enzyme for the generation of MHC class II/tetramers.; citation_author=J Yang; citation_author=A Jaramillo; citation_author=R Shi; citation_author=WW Kwok; citation_author=T Mohanakumar; citation_journal_title=Hum Immunol; citation_volume=65; citation_number=13; citation_pages=692-699; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Establishment of a quantitative ELISA capable of determining peptide - MHC class I interaction.; citation_author=C Sylvester-Hvid; citation_author=N Kristensen; citation_author=T Blicher; citation_author=H Ferre; citation_author=SL Lauemoller; citation_journal_title=Tissue Antigens; citation_volume=59; citation_number=14; citation_pages=251-258; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Efficient assembly of recombinant major histocompatibility complex class I molecules with preformed disulfide bonds.; citation_author=PL Ostergaard; citation_author=MH Nissen; citation_author=NJ Hansen; citation_author=LL Nielsen; citation_author=SL Lauenmoller; citation_volume=31; citation_number=15; citation_pages=2986-2996; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=CD8 expression allows T cell signaling by monomeric peptide-MHC complexes.; citation_author=J Delon; citation_author=C Gregoire; citation_author=B Malissen; citation_author=S Darche; citation_author=F Lemaitre; citation_journal_title=Immunity; citation_volume=9; citation_number=16; citation_pages=467-473; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Generation of peptide-MHC class I complexes through UV-mediated ligand exchange.; citation_author=B Rodenko; citation_author=M Toebes; citation_author=SR Hadrup; citation_author=WJ van Esch; citation_author=AM Molenaar; citation_journal_title=Nat Protoc; citation_volume=1; citation_number=17; citation_pages=1120-1132; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001678" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="One-Pot, Mix-and-Read Peptide-MHC Tetramers"/>
    <meta name="twitter:description" content="BackgroundCytotoxic T Lymphocytes (CTL) recognize complexes of peptide ligands and Major Histocompatibility Complex (MHC) class I molecules presented at the surface of Antigen Presenting Cells (APC). Detection and isolation of CTL's are of importance for research on CTL immunity, and development of vaccines and adoptive immune therapy. Peptide-MHC tetramers have become important reagents for detection and enumeration of specific CTL's. Conventional peptide-MHC-tetramer production involves recombinant MHC production, in vitro refolding, biotinylation and tetramerization; each step followed by various biochemical steps such as chromatographic purification, concentration etc. Such cumbersome production protocols have limited dissemination and restricted availability of peptide-MHC tetramers effectively precluding large-scale screening strategies involving many different peptide-MHC tetramers.Methodology/Principal FindingsWe have developed an approach whereby any given tetramer specificity can be produced within 2 days with very limited effort and hands-on time. The strategy is based on the isolation of correctly oxidized, in vivo biotinylated recombinant MHC I heavy chain (HC). Such biotinylated MHC I HC molecules can be refolded in vitro, tetramerized with streptavidin, and used for specific T cell staining-all in a one-pot reaction without any intervening purification steps.Conclusions/SignificanceWe have developed an efficient &ldquo;one-pot, mix-and-read&rdquo; strategy for peptide-MHC tetramer generation, and demonstrated specific T cell straining comparable to a commercially available MHC-tetramer. Here, seven peptide-MHC tetramers representing four different human MHC (HLA) class I proteins have been generated. The technique should be readily extendable to any binding peptide and pre-biotinylated MHC (at this time we have over 40 different pre-biotinylated HLA proteins). It is simple, robust, and versatile technique with a very broad application potential as it can be adapted both to small- and large-scale production of one or many different peptide-MHC tetramers for T cell isolation, or epitope screening."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001678.g009"/>

  <meta property="og:title" content="One-Pot, Mix-and-Read Peptide-MHC Tetramers" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001678" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2211'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2966'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9111&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001678">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001678" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001678&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    One-Pot, Mix-and-Read Peptide-MHC Tetramers
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christian Leisner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark
                </p>

                <p>
                    Current address: Swiss Federal Institute of Technology (ETH), Institute of Biochemistry, Zurich, Switzerland
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nina Loeth, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kasper Lamberth, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sune Justesen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christina Sylvester-Hvid, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark
                </p>

                <p>
                    Current address: Wiborg Aps, Hørsholm, Denmark
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Esben G. Schmidt, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mogens Claesson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Soren Buus
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:s.buus@immi.ku.dk">s.buus@immi.ku.dk</a></p>

                <p>Affiliation:
                  Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anette Stryhn
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratories of Experimental Immunology, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 27, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001678</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001678" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001678" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001678" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001678" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001678-g001" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001678-t001" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001678-g002" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001678-g003" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001678-g004" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001678-g005" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001678-g006" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001678-g007" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001678-g008" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001678-g009" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001678">Reader Comments (6)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001678" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Cytotoxic T Lymphocytes (CTL) recognize complexes of peptide ligands and Major Histocompatibility Complex (MHC) class I molecules presented at the surface of Antigen Presenting Cells (APC). Detection and isolation of CTL's are of importance for research on CTL immunity, and development of vaccines and adoptive immune therapy. Peptide-MHC tetramers have become important reagents for detection and enumeration of specific CTL's. Conventional peptide-MHC-tetramer production involves recombinant MHC production, in vitro refolding, biotinylation and tetramerization; each step followed by various biochemical steps such as chromatographic purification, concentration etc. Such cumbersome production protocols have limited dissemination and restricted availability of peptide-MHC tetramers effectively precluding large-scale screening strategies involving many different peptide-MHC tetramers.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We have developed an approach whereby any given tetramer specificity can be produced within 2 days with very limited effort and hands-on time. The strategy is based on the isolation of correctly oxidized, in vivo biotinylated recombinant MHC I heavy chain (HC). Such biotinylated MHC I HC molecules can be refolded in vitro, tetramerized with streptavidin, and used for specific T cell staining-all in a one-pot reaction without any intervening purification steps.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We have developed an efficient “one-pot, mix-and-read” strategy for peptide-MHC tetramer generation, and demonstrated specific T cell straining comparable to a commercially available MHC-tetramer. Here, seven peptide-MHC tetramers representing four different human MHC (HLA) class I proteins have been generated. The technique should be readily extendable to any binding peptide and pre-biotinylated MHC (at this time we have over 40 different pre-biotinylated HLA proteins). It is simple, robust, and versatile technique with a very broad application potential as it can be adapted both to small- and large-scale production of one or many different peptide-MHC tetramers for T cell isolation, or epitope screening.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Leisner C, Loeth N, Lamberth K, Justesen S, Sylvester-Hvid C, et al.  (2008) One-Pot, Mix-and-Read Peptide-MHC Tetramers. PLoS ONE 3(2):
          e1678.
            doi:10.1371/journal.pone.0001678</p><p><strong>Academic Editor: </strong>Jacques Zimmer, Centre de Recherche Public-Santé, Luxembourg</p><p><strong>Received:</strong> October 31, 2007; <strong>Accepted:</strong> January 21, 2008; <strong>Published:</strong> February 27, 2008</p><p><strong>Copyright:</strong> © 2008 Leisner et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the 6FP EU grant 503231, the NIH contract 266200400025C, the Danish FTP 274-05-0582 and by the Danish Cancer Society DP06015.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Antigen-specific, MHC-restricted T cells (hereafter referred to as “specific T cells”) play many important roles in the generation of immune responses. It is therefore important to be able to identify, enumerate and characterize specific T cells, e.g. during infections, vaccinations or immunotherapies. In the past, assays of proliferation, cytotoxicity, and/or bulk cytokine production have been used as functional evidence of the presence of specific T cells. Combined with laborious limiting dilution experiments, these assays have even provided indirect measurements of the frequency of specific T cells. More recent and less cumbersome assays have detected and counted specific T cell responses at the single cell level using either ELISPOT assays of cytokine release <a href="#pone.0001678-Czerkinsky1">[1]</a>–<a href="#pone.0001678-Romero1">[4]</a>, or flow cytometry-based assays for intracellular cytokine production <a href="#pone.0001678-Suni1">[5]</a>. These assays, however, are still based upon functional assays and the frequency of responding T cells may not necessarily be the same as the frequency of T cells expressing the appropriate T cell receptor <a href="#pone.0001678-Markovic1">[6]</a>. In a 1996 landmark achievement, Altman and co-workers <a href="#pone.0001678-Altman1">[7]</a> generated fluorescence labeled, peptide-MHC multimers of defined specificity (generally known in the field of T cell immunology as “MHC tetramers”) and demonstrated their ability to stain specific T cells. They surmised that peptide-MHC multimers would engage several of the T cell receptors expressed on a specific T cell surface and stabilize an otherwise very short lived interaction, thus allowing direct and specific staining of the T cell. Whereas functional assays involves some kind of re-stimulation, MHC tetramers can be used to identify specific T cells without further in vitro manipulation, and they allow for a simultaneous evaluation of the differentiation state through co-staining for various cell surface differentiation markers. In addition, MHC tetramers can be used to purify <a href="#pone.0001678-Keenan1">[8]</a>, manipulate, and stimulate <a href="#pone.0001678-Savage1">[9]</a>, <a href="#pone.0001678-Savage2">[10]</a> specific T cells.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Thus, MHC tetramers provide a simple, fast and efficient approach for monitoring and handling specific T cells. Unfortunately, the production of MHC tetramers is technically demanding. The Altman tetramer production can be sub-divided into three major steps all of which requires biochemical expertise: peptide-MHC monomer production, biotinylation of peptide-MHC monomers, and finally addition of fluorescent labeled streptavidin to effect tetramerization. After each of these steps, the product has to be processed biochemically (e.g. purified by column chromatography, concentrated etc.) before being subjected to the next step. This need for a biochemical set-up and proficiency limits the dissemination of the technology and effectively prevents the application of MHC tetramers to large-scale analysis and use. Indeed, the NIH has created a core facility to generate MHC tetramers; however, the use of the NIH MHC tetramer facility requires approval and may be limited.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We have previously generated pre-oxidized recombinant MHC class I molecules of a quality that allows virtually complete folding and peptide-MHC complex formation<a href="#pone.0001678-Ferre1">[11]</a>. Here, we have generated fully pre-biotinylated versions of the MHC class I molecules and demonstrated that these can be used as convenient reagents in the preparation of biotinylated peptide-MHC monomers. Adding streptavidin to such MHC complexes efficiently leads to MHC tetramer formation. Given the efficiency of our folding, biotinylation and tetramerization processes, we reasoned that it might be possible to omit most, if not all, of the preparative biochemical work involved in tetramer production and create a “one-pot, mix-and-read” protocol. In this scenario, recombinant, pre-biotinylated MHC molecules would be successively reacted with peptide, streptavidin and eventually T cells without any intervening purification or concentration processes. Indeed, we have developed several examples of such “one-pot, mix-and-read” tetramers and demonstrated specific T cell staining of a quality fully comparable with a commercially available counterpart. Pre-oxidized and pre-biotinylated MHC class I heavy chain molecules could be produced in large quantities for wide distribution to laboratories, where they could be readily used to generate tetramers with peptides of choice. Thus, this tetramer technology could allow virtually any end-user to generate his or hers own MHC tetramers at will, and it should allow large-scale MHC tetramer screening approaches.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Production of recombinant biotinylated MHC-I heavy chains</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To assure one single specific biotinylation of each recombinant MHC molecule, a biotinylation substrate peptide (BSP) sequence specific for the E.<em>coli</em> biotin protein ligase, BirA, enzyme was attached to the C-terminal end of the recombinant MHC-I HC. This enables specific biotinylation of MHC-I HC by BirA. For MHC tetramer production, biotinylation of the BSP site has mostly been based on biochemical in vitro biotinylation of the folded MHC-I complex. This needs to be followed by chromatographic purification steps, which are time consuming and potentially associated with significant loss of material. Induction of efficient in vivo biotinylation of the HC would therefore save both time and material. The endogenous BirA expression in E. coli is insufficient for an efficient biotinylation of over-expressed proteins. However, co-expression of the BirA enzyme either in the E. coli or insect cell expression systems has been shown to increase the biotinylation efficiency<a href="#pone.0001678-Cull1">[12]</a>, <a href="#pone.0001678-Yang1">[13]</a>. Therefore, the BL21 (DE3) E. coli strain was co-transformed with a pET28a+ vector encoding the MHC-I HC-BSP and a pASYC vector encoding the BirA enzyme. Production of HC and BirA were induced by addition of IPTG for the last 3 h of culture (<a href="#pone-0001678-g001">Figure 1A</a>). Biotin was added to the bacterial culture immediately before induction of expression of HC and BirA. The biotinylation efficiency of the HC was evaluated in a SDS-PAGE gel-shift assay. Samples of HC were incubated with increasing amount of Streptavidin (SA), and subsequently separated by SDS-PAGE. Biotinylated HC will bind to SA and thereby significantly increase the molecular weight of the complex. This resulted in disappearance of the HC-band and appearance of bands of higher molecular weights (<a href="#pone-0001678-g001">Figure 1B</a>). Using densitometry, the residual HC-band representing the non-biotinylated HC were quantified and compared to the total amount of HC (lane without SA added). We have produced and analyzed 27 HLA-A alleles and 14 HLA-B alleles this way, and in all cases obtained biotinylation efficiencies in the range of 85–100% (<a href="#pone-0001678-t001">Table 1</a>). More than ¾ of these preparations were more than 95% biotinylated the rest were between 82 and 94% biotinylated. Thus, the in vivo biotinylation procedure was highly efficient.</p>
<div class="figure" id="pone-0001678-g001"><div class="img"><a name="pone-0001678-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Production of MHC-I heavy chains.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Clones of Escherichia coli were selected for high expression of HLA-A*0201-HAT-BSP HC and BirA enzyme and the HLA-A*0201-HAT-BSP product was tested for biotinylation efficiency. In A) Samples of the clone were withdrawn before and after IPTG induction and analyzed on reducing SDS-PAGE. The bacterial cell pellet was resuspended in 50 µL MgCl2/SDS lysis buffer to solubilize heavy-chain inclusion bodies. After centrifugation at 20,000 g for 2 min, 15 µL of the supernatant was loaded directly on the gel. Lane 1, protein marker; lane 2, before induction; and lane 3 after induction with IPTG. Positions of HC monomer and BirA are shown with arrows. In B) HLA-A*0201-HAT-BSP was tested for biotinylation efficiency by a streptavidin gel-shift assay. Purified in vivo biotinylated HLA-A*0201-HAT-BSP incubated for 20 min with either no SA, equal amount of SA, or SA in 2-fold excess. The samples were diluted in 2×Laemmli buffer and electrophoresed on a 12% polyacrylamide gel. For the purpose of quantitative densitometric analysis the HC sample without SA was applied to the SDS-PAGE in 3 different amounts (undiluted, 5-fold, and 25-fold diluted). The percentage of biotinylated HC was determined from the ratio of density of the monomer HC band in the sample with excess SA compared to the band in the diluted sample without SA.</p>
<span>doi:10.1371/journal.pone.0001678.g001</span></div><div class="figure" id="pone-0001678-t001"><div class="img"><a name="pone-0001678-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001678.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001678.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>HLA preparations</span></strong></p><span>doi:10.1371/journal.pone.0001678.t001</span></div>

<h4>Biological activity of biotinylated MHC-I</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To ensure that the in vivo biotinylation did not affect the specificity and subsequent folding of the MHC class I complex, we compared the peptide binding affinities of HLA-HLA-A*0201-HAT-BSP with and without biotinylation to the unmodified recombinant HLA-A*0201. A panel of small pox peptides representing good binders of K<sub>D</sub>&lt;10 nM (YLDYDTIYV, FLRDNLYHV, and YLSDSAINI), intermediate binders of K<sub>D</sub> 10-100 nM (YLSTERDHV and FLETDAGRV), and non binding peptides of K<sub>D</sub>&gt;1 µM (ALSDACKKI), were analyzed for binding affinity to each of HLA-A*0201 HC constructs. The HLA-A*0201 constructs were folded in the presence of ß<sub>2</sub>m and a titration of the peptides and the K<sub>D</sub> of each combination was determined in a quantitative ELISA assay <a href="#pone.0001678-SylvesterHvid1">[14]</a> (<a href="#pone-0001678-g002">Figure 2</a>). Each peptide, strong as well as poor binders, gave essentially the same K<sub>D</sub> value for all three HLA-A*0201 constructs. Thus, neither the addition of the BSP site nor the biotinylation prior to folding affected the binding specificity of the HLA-A*0201 molecule.</p>
<div class="figure" id="pone-0001678-g002"><div class="img"><a name="pone-0001678-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Peptide binding affinity of biotinylated HLA-A*0201 versus non-biotinylated.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The peptide binding affinities of the biotinylated HLA-A*0201-HAT-BSP was compared to the standard truncated HLA-A*0201 molecule. The HLA-A*0201 peptide binding affinities of a panel of pox peptides ranging from god binders (0–10 nM) over intermediary (10–100 nM) to non-binders (&gt;1 µM) were determined in a quantitative ELISA <a href="#pone.0001678-SylvesterHvid1">[14]</a>. The HC were folded with various concentrations of peptide and excess of hβ<sub>2</sub>m. The K<sub>D</sub> values for each peptide MHC combination were calculated and the results presented as means of two independent experiments with the corresponding confidence intervals.</p>
<span>doi:10.1371/journal.pone.0001678.g002</span></div>

<h4>Refolding efficiency</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In addition to a high biotinylation frequency, our approach also depends upon a high refolding efficiency. To this end we have exploited two previous observations; the first demonstrating that denatured class I MHC heavy chains with pre-oxidized disulfide bonds efficiently refold (when provided with β<sub>2</sub>m and an appropriate peptide)<a href="#pone.0001678-Ostergaard1">[15]</a>, and the second demonstrating that the efficiency of this process can be further improved by purification of denatured heavy chain molecules with correct disulfide conformations<a href="#pone.0001678-Ferre1">[11]</a>. Thus, we have previously shown that subspecies of correctly oxidized recombinant denatured MHC-I HC can be purified from the inclusion bodies, and that these upon dilution folds into functional complex with a &gt;90% efficiency<a href="#pone.0001678-Ferre1">[11]</a>. In this paper, we have used 4 different MHC-I HC preparations for tetramer production: HLA-A*0101, HLA-A*0201, HLA-B*0702, and HLA-B*4402. These HC's were in vivo biotinylated, purified, and subsequently folded in the presence of a relevant peptide, pp65<sub>363–373</sub>, pp65<sub>495–503</sub>, pp65<sub>417–426</sub>, and EBNA3b<sub>657–666</sub> respectively. These complexes were titrated and measured in a quantitative ELISA. The folding efficiency was determined by comparison to a standard of known complex concentration. As seen in <a href="#pone-0001678-t001">Table 1</a>, we obtain 95–100% correctly folded complex and these HC were 97–100% biotinylated. Note, that although we don't know where and how these disulfide bonds are generated, we have at this point successfully generated tetramers from HLA preparations representing 14 different HLA alleles, and in no case have we failed. We conclude that these molecules in principle are ready to be tetramerized with SA without further purification.</p>


<h4>Tetramerization of MHC and streptavidin</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>In order to follow the stoichiometric distribution of SA-MHC-I complexes during the formation of peptide-MHC tetramers, the folding reaction of biotinylated HLA-A*0201-HAT-BSP was spiked with radioactively labeled peptide. MHC-tetramers were the generated at different molar ratios of MHC-I and SA; and the formations of various SA-MHC-I multimers were followed by size-exclusion chromatography (SEC) through detection of radioactivity (<a href="#pone-0001678-g003">Figure 3</a>). SA binds MHC-I at a 1 to 4 molar ratio. Gradually increasing the SA to MHC-I ratio from MHC-I in excess to SA in excess, saturated and unsaturated SA molecules could be generated at different stoichiometric ratios; SA:MHC<sub>4</sub>, SA:MHC<sub>3</sub>, SA:MHC<sub>2</sub>, SA:MHC, and eventually MHC monomers. Quantification of molecular weights according to standard marker proteins showed agreement between theoretical and empirical molecular sizes (<a href="#pone-0001678-g003">Figure 3</a>). All four SA-MHC-I multimers were generated dependent upon the ratio between SA and MHC-I. At SA to MHC-I ratio of 1:4, the tetramer form was almost quantitatively produced, generating none of the other multimer species and only leaving a small fraction of non-reacted monomers. This presumably represents a small fraction of non-biotinylated HC.</p>
<div class="figure" id="pone-0001678-g003"><div class="img"><a name="pone-0001678-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Binding of SA and biotinylated HLA-A*0201-peptide complexes.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The folding reaction of biotinylated HLA-A*0201 was spiked with radioactively labeled peptide. These complexes were reacted with SA at various different molar ratios in order to produce MHC-I tetramers, trimers, dimers, and monomers. The various SA-MHC-I complexes were run on a calibrated SEC column. The eluate fraction of the SA-MHC-I complexes was determined through radioactivity measurements. (A) Size exclusion chromatogram of 6 different stoichiometric mixtures of SA to HLA-A*0201-peptide complex. In (B) is listed the calculated MW's corresponding of the individual peaks of the chromatogram.</p>
<span>doi:10.1371/journal.pone.0001678.g003</span></div>

<h4>Staining specific T cells using “one pot, mix and read” tetramers</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The efficiency of the biotinylation and folding processes suggested to us that it should be possible to generate MHC tetramers in a “one-pot” reaction omitting most, if not all, of the preparative biochemical work normally involved in tetramer production. To analyze whether our “one-pot” tetramers were able to stain specific T cells in a “mix-and-read” mode, we initially selected the HLA-A*0201 restricted dominant CMV T cell epitopes, CMV pp65<sub>495–503</sub> peptide as model system. CMV pp65<sub>495–503</sub> specific T cells from healthy donors were generated and HLA-A*0201/pp65<sub>495–503</sub>-SA-phycoerythrin (PE) tetramers were produced as described above. The tetramers were not concentrated nor further purified, but used directly to stain the specific T cells.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The specificity and sensitivity of the tetramer staining were compared to that of the commercially available HLA-A*0201/pp65<sub>495–503</sub> pentamers (®ProImmune, <a href="http://www.proimmune.com">www.proimmune.com</a>) (<a href="#pone-0001678-g004">Figure 4</a>). The tetramers stain about 60% of the CD8 T cells with a baseline separation from the negative population. The same frequency of specific T cells was detected using the commercial pentamers. The staining intensity of the tetramers were slightly lower than that of the pentamers (FI 1480 vs. 2042), however, the background staining of the negative population was also lower (FI 4 vs. 10) resulting in a higher signal to noise ratio for the in-house generated tetramers (370 vs. 202). Thus the “one-pot, mix-and-read” strategy for the HLA-A*0201/pp65<sub>495–503</sub> tetramer production appeared to work as well as the MHC pentamer of the same specificity.</p>
<div class="figure" id="pone-0001678-g004"><div class="img"><a name="pone-0001678-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Tetramer staining comparison to pentamer.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>PBMC from a healthy donor were stimulated for 8 day with the pp65<sub>495–503</sub> peptide and stained with APC labeled anti-CD8 and PE-labeled HLA-A*0201-pp65<sub>495–503</sub> tetramer (right panel) or with PE-labeled HLA-A*0201-pp65<sub>495–503</sub> pentamer (left panel). The dot plot shows the gated CD8 T cells. Numbers in the upper right quadrant of each plot are the percentage of epitope specific CD8 T cells. The fluorescence intensity (FI) of the staining of the positive and negative population is given in the upper and lower right quadrant, respectively.</p>
<span>doi:10.1371/journal.pone.0001678.g004</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The MHC-I monomers were produced at a rather low concentration compared to conventional tetramer production. These monomers were mixed with streptavidin directly, without any concentration, to generate the tetramers. The stability, yield, and robustness of this production strategy were of concern. We found that the tetramers could be stored for more that 9 months at 4°C with no change in activity (data not shown). Another concern was whether there was a lower concentration limit of monomers needed for production of tetramers in order to give an optimal staining of the specific CD8 T cells. To analyze this HLA-A*0201/pp65<sub>495–503</sub> monomers were folded at 200 nM, 100 nM and 50 nM and each mixed with streptavidin to produce tetramers. Thus, the maximum tetramer concentration of the 200 nM monomers was 50 nM tetramer, of the 100 nM monomers 25 nM tetramer, and of the 50 nM monomers 12.5 nM tetramer. Two fold titrations were made of each of these three productions and used to stain pp65<sub>495–503</sub> specific T cells. Baseline separation between the positive and negative cell population was seen for all the concentrations used. The relative fluorescence intensity is depicted as a function of the actual tetramer concentration used to stain the cells (<a href="#pone-0001678-g005">Figure 5</a>). The curves for the three-tetramer productions were super imposable showing considerable robustness in set-up and staining reactions. The actual tetramer concentration used in the staining reaction, and not the concentration of monomers used for the tetramer production, was important for the staining intensity. The staining intensity reached a plateau and the T cell labeling was saturated at tetramer concentrations above 12.5 nM. Therefore, saturable T cell staining can be achieved using tetramers produced from concentration of monomers higher than 50 nM.</p>
<div class="figure" id="pone-0001678-g005"><div class="img"><a name="pone-0001678-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Titration of complex concentration for tetramer production.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>Three different concentrations of HLA-A*0201-pp65<sub>495–503</sub> complexes were folded (50, 100, and 200 nM) and tetramers were produced from these. PBMC from a CMV pp65<sub>495–503</sub> responsive donor (3.5% specific CD8 T cells) were stained with a titration of each of the three tetramer productions. The x-axis is the actual calculated concentration of tetramers in the cell staining reaction. The results are depicted as the relative fluorescence intensity (FI signal of the positive population/FI signal of the negative population).</p>
<span>doi:10.1371/journal.pone.0001678.g005</span></div>

<h4>Unreacted monomers do not inhibit HLA tetramer staining</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>It is inherent to the “mix-and-read” tetramer strategy that the staining reaction is done with the entire tetramerization reaction mixture including any unreacted monomer such as that of any un-biotinylated HLA molecule. A priori, the tetramerized peptide-HLA complexes would have a much stronger reaction with the appropriate T cell receptor than any unreacted monomers would have. Addressing whether the monomers could inhibit tetramer staining, we added increasing concentrations (up to 180 Nm) of monomers to specific T cells, incubated the cells at room temperature for 5 minutes, and then added a fixed concentration of tetramer (5 nM tetramer corresponding to 20 nM monomer). In no case was any inhibition of tetramer staining observed (<a href="#pone-0001678-g006">Figure 6</a>) suggesting that unreacted monomers do not compromise staining with our one-pot, mix-and-read tetramers. Note, that we have not addressed any physiological effect of peptide-HLA monomers such as the partial activation demonstrated by Malissen and co-workers <a href="#pone.0001678-Delon1">[16]</a>.</p>
<div class="figure" id="pone-0001678-g006"><div class="img"><a name="pone-0001678-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Unreacted monomers do not inhibit tetramer staining.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>PBMC from a healthy donor was re-stimulated for 8 day with the CMV pp65<sub>495–503</sub> peptide. They were washed and adjusted to a final concentration of 2.5 million cells/ml containing the indicated concentration of CMV pp65<sub>495–503</sub>/HLA-A*0201 monomer. After 5 min incubation at room temperature, a final concentration of 5 nM CMV pp65<sub>495–503</sub>/HLA-A*0201 tetramer was added. The cells were further stained for CD8 and analyzed by flow cytometry as described in <a href="#s3">materials and methods</a>.</p>
<span>doi:10.1371/journal.pone.0001678.g006</span></div>

<h4>Extending to other HLA-A*0201-peptide epitopes</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>To validate the generality of the “one-pot, mix-and-read” strategy for tetramer production, a different set of tetramers were produced from 3 different HLA-A*0201 EBV peptide complexes: HLA-A*0201/LMP1<sub>125–133</sub>, HLA-A*0201/LMP2<sub>356–364</sub> and HLA-A*0201/LMP2<sub>416–424</sub>. PBMC's from four different HLA-A2 positive donors were restimulated with the three different peptides for 8 days and then stained with the tetramers (<a href="#pone-0001678-g007">Figure 7</a>). PBMC's from all four donors could be stained by one or more of the three tetramers. In all cases, the specific staining showed baseline separation between positive and negative populations. Three of the donors had LMP2 <sub>356–364</sub> specific T cells varying from about 3 to 11% of the CD8 T cells being labeled with HLA-A*0201/LMP2 <sub>356–364</sub>. Two of the donors were positive for LMP2 <sub>416–424</sub> with about 3 and 4% specific CD8 T cells. Only one out of four donors was positive for LMP1<sub>125–133 </sub>with about 3% specific CD8 T cells. The same results were obtained when the frequency of peptide specific T cells producing IFNγ was evaluated by intracellular staining (data not shown). Thus this “one-pot, mix-and-read” strategy can be used to produce several different HLA-A*0201 tetramers and can directly be used for epitope screening analysis.</p>
<div class="figure" id="pone-0001678-g007"><div class="img"><a name="pone-0001678-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>HLA-A*0201-tetramer screening of EBV reactive donors.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>PBMC from four healthy donors were each re-stimulated for 8 day with three different EBV peptides, LMP1<sub>125–133</sub>, LMP2<sub>356–364</sub>, or LMP2<sub>416–424</sub>, and stained with both HLA-A*0201-LMP1<sub>125–133</sub> (top panel, HLA-A*0201-LMP2<sub>356–364</sub> (middle panel) and HLA-A*0201-LMP2<sub>416–424</sub> tetramers (bottom panel). The dot plot shows gated CD8 T cells and the percentage of tetramer labeled CD8 T cells are given in the upper right quadrant of each plot.</p>
<span>doi:10.1371/journal.pone.0001678.g007</span></div>

<h4>Extending to other HLA-alleles</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>We wanted to extend this application to other HLA and peptide-HLA combinations. Complexes of two dominant CMV pp65 T cell epitopes, HLA-A*0101/pp65<sub>363–373</sub> and HLA-B*0702/pp65<sub>417–426</sub>, were folded. As for the HLA-A*0201, the folding efficiency of these complexes were &gt;96% (<a href="#pone-0001678-t001">Table 1</a>). Tetramers of these complexes and the HLA-A*0201/pp65<sub>495–503</sub> were produced and used to analyze the specificity of a CMV response. PBMC's from four different healthy donors were stimulated for 8 days with a mixture of 134 overlapping 15-mer peptides representing the entire pp65 CMV protein and subsequently analyzed for pp65 specific responses using intracellular IFNγ (IC-IFNγ) staining. All four donors showed a pp65 specific response, having between 16% and 46% pp65 specific CD8 T cells (<a href="#pone-0001678-g008">Figure 8</a>, top panel). Using the HLA-A*0101/pp65<sub>363–373</sub>, HLA-A*0201/pp65<sub>495–503</sub>, and HLA-B*0702/pp65<sub>417–426</sub> tetramers these donors were analyzed for the specificity distribution against these three dominant pp65 T cell epitopes (<a href="#pone-0001678-g008">Figure 8</a>). All three tetramers showed specific staining and we were clearly able to differentiate the donors as responder and non-responder to the given epitope. Two of the four donors (Donor 1 and 4; <a href="#pone-0001678-g008">Figure 8</a>) stained positive for the HLA-A*0101/pp65<sub>363–373</sub> tetramer; one having a relative high frequency of positive CD8 T cells and the other a very low frequency (17.6% versus 0.06%). Two donors (Donor 1 and 2; <a href="#pone-0001678-g008">Figure 8</a>) stained positive for the HLA-A*0201/pp65<sub>495–503</sub>, tetramer, both with relative high frequencies (28.5 and 15.6%). Only donor 4 stained positive for HLA-B*0702/pp65<sub>417–426</sub> with 36% of the CD8 T cells being specific for this epitope. Except for one donor, the frequencies of tetramer positive CD8 T cells for each donor fully covered the entire pp65 response measured by intracellular staining for that donor. Donor 3 was not stained by any of the three tetramers. Therefore, the pp65 specific CD8 T cells response, must be represented by pp65 epitopes different from the three used for tetramers.</p>
<div class="figure" id="pone-0001678-g008"><div class="img"><a name="pone-0001678-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Screening for CMV specific T cells with three different Class I-tetramers.</span></strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>PBMC from 4 healthy CMV pp65 responding donor were stimulated for 8 day with a mixture of 15 amino acid long overlapping peptides spanning the entire pp65 protein. The cells were analyzed pp65 specific responses using the IC-IFNγ assay and stained for CD8, CD69 and intracellular IFNγ (top panel). The donors were screened for HLA-A*0101 restricted pp65<sub>363–373</sub>, the HLA-A*0201 restricted pp65<sub>495–503</sub>, and the HLA-B*0702 restricted pp65<sub>417–426</sub> specific T cells. Staining the cells with CD8 and each of the three tetramers, HLA-A*0101-pp65<sub>363–373</sub>, HLA-A*0201-pp65<sub>495–503</sub>, and HLA-B*0702-pp65<sub>417–426</sub>. The dot plot shows the gated CD8 T cells. Numbers in the upper right quadrant of each plot are the percentage of epitope specific CD8 T cells.</p>
<span>doi:10.1371/journal.pone.0001678.g008</span></div>

<h4>Correlating staining with functionality</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>Finally, we correlated tetramer staining with functionality. Though the frequency of tetramer positive cells and IFNγ producing cells correlated nicely we wished to assure that the cells, which stained tetramer positive, were in fact the IFNγ producing cells. Three different tetramers were produced: HLA-A*0101/pp65<sub>363–373</sub>, HLA-A*0201/pp65<sub>495–503</sub>, and B*4401/EBNA3b<sub>657–666</sub>. Three different donors were restimulated for 8 day with the three peptides, CMV pp65<sub>363–373</sub>, CMV pp65<sub>495–503</sub>, and EBV EBNA3b<sub>657–666</sub>, respectively. T cells from each donor were double labeled for tetramer and IC-IFNγ (<a href="#pone-0001678-g009">Figure 9</a>). For all combinations, the specific IFNγ producing CD8 T cells were also stained with the relevant tetramers. For the two donors specific for pp65<sub>363–373</sub> and pp65<sub>495–503</sub>, a few percentage of the tetramer labeled cells did not produce IFNγ. Since, the majority of the CMV specific CD8 T cells produced both IFNγ and TNFα and only a few percentage produced either one or the other alone, these tetramer+, IFNγ- cells were most likely TNFα producing. Thus, the CD8 T cells stained with these tetramers were in fact the specific CD8 T cells that produce IFNγ in response to the relevant peptide epitope.</p>
<div class="figure" id="pone-0001678-g009"><div class="img"><a name="pone-0001678-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001678" data-uri="info:doi/10.1371/journal.pone.0001678.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001678.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001678.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001678.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001678.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Double staining with tetramer and IC-IFNγ.</span></strong></p><a id="article1.body1.sec2.sec9.fig1.caption1.p1" name="article1.body1.sec2.sec9.fig1.caption1.p1"></a><p>Τhree donors responding to either HLA-A*0101 restricted pp65<sub>363–373</sub>, HLA-A*0201 restricted pp65<sub>495–503</sub>, or HLA-B*4401 restricted EBV EBNA3b<sub>657–666</sub>, were restimulated with the respective peptides for 3 h in the presence of Brefeldin A. The cells were stained with the relevant tetramers, HLA-A*0101-pp65<sub>363–373</sub>, HLA-A*0201-pp65<sub>495–503</sub>, or HLA-B*4401-EBNA3b<sub>657–666</sub>, prior to the staining for CD8 and intracellular IC-IFNγ. The dot plot shows the gated CD8 T cells.</p>
<span>doi:10.1371/journal.pone.0001678.g009</span></div>

<h4>Conclusion</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>In summary, we have developed a rapid “one-pot, mix-and-read” strategy for tetramer production. Due to highly efficient biotinylation, folding, and tetramerization processes, all the preparative biochemical work normally needed during MHC tetramer generation can be omitted and the production can run as a fast sequential addition of the relevant reagents. Others have also attempted to generate a high-throughput production of MHC class I tetramers<a href="#pone.0001678-Rodenko1">[17]</a>. Their strategy is based on peptide exchange of MHC complexes folded with an UV cleavable peptide. These degradable complexes are produced in the conventional way being time consuming and associated with significant loss of material. At this point, this peptide cleavable approach has been applied to one human MHC class I allele (HLA-A*0201) and a mouse allele (D<sup>b</sup>). The broad application of their strategy is limited by the need to define a unique UV-labile epitope for each MHC class I allele. In contrast, our approach is based on the ability to produce fully in vivo biotinylated and readily foldable MHC I HC. Ready-to-use tetramers with the desired specificity and scale can be produced within 2 days in any lab independent of special biochemical skills or equipment. We have demonstrated the application of our “one-pot, mix-and-read” strategy for 7 different MHC I restricted T cells epitopes spanning 4 different HLA class I alleles. We have produced an additional 37 fully biotinylated HLA-class I HC using this strategy (<a href="#pone-0001678-t001">Table 1</a>). This procedure is suitable for large-scale production of a single peptide-MHC tetramer for repeated analysis or purification of specific T cells as well as for a small-scale production of a larger number of different class I-peptide complexes e.g. for a comprehensive T cell screening program.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Peptides</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Peptides were synthesized by standard 9-fluorenylmethyloxycarbonyl (FMOC) chemistry, purified by reversed-phase high-performance liquid chromatography (at least 80%, usually &gt;95% purity) (Shafer-N, Copenhagen, Denmark). Mixes of 134 15 amino acid long peptides spanning the entire CMV pp65 were obtained from PT Peptide Technologies GmbH, Germany.</p>


<h4>Production of human MHC-I heavy chains</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Recombinant MHC-I heavy chains (HC's) have been generated from synthetic codon optimized and truncated genes without the transmembrane region. Sequences encoding HAT (a histidine affinity tag) and BSP (a biotinylation tag) were fused to the 3′ end of the truncated HC gene and cloned into a pET28a+ <em>E. coli</em> expression vector (Novagen). For expression, the BL21 (DE3) <em>E. coli</em> strain was co-transformed with a pET28a+ vector encoding the tagged HC's and a pASYC vector encoding the BirA biotinylation holoenzyme. Clones producing high amounts of HC and BirA upon isopropylthiogalactoside (IPTG) induction were selected. 100 mg/L biotin (Sigma) was included during the 3h IPTG induction.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Large-scale production and purification of proteins were performed as previously described<a href="#pone.0001678-Ferre1">[11]</a>. Where and how the disulfides of the HLA heavy chains are formed is not known. The details of protein handling may be extremely important, including pH, temperature and time. We have not examined these parameters, however, preparations of tetramer generating quality were consistently generated.</p>


<h4>Biotinylation assay</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The level of biotinylation of the purified HC was examined by a streptavidin gel-shift SDS-PAGE assay. Purified HC was incubated for 20 min at RT with a functional excess of streptavidin (SA, each streptavidin molecule can bind four biotinylated molecules), and analyzed by 12% polyacrylamide gel (SDS-PAGE, without boiling the samples). SA interacts with biotinylated HC and shifts the approximate 36 kDa monomer band to approximately 100 kDa. For comparison, serial dilutions of HC, which have not been reacted with SA, were analyzed. The intensities of the SA reacted and non-reacted HC bands were determined by densitometric analysis using 1D Gel Analysis software (Kodak Digital Science). The percentage of biotinylated HC could be calculated from the ratio of intensity of the HC band of the sample without SA compared to the sample with excess SA.</p>


<h4>Quantitative measurement of peptide MHC-I complex formation</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The peptide binding affinity of biotinylated versus non-biotinylated HLA-A*0201 were compared using our previously described quantitative ELISA assay<a href="#pone.0001678-SylvesterHvid1">[14]</a>. Briefly, peptide-MHC-I complexes were folded at 18°C for 48 hours with a constant concentration of hß<sub>2</sub>m and a titration of peptide concentrations. The complex formation was measured with a sandwich ELISA using the conformationally sensitive monoclonal antibody, W6/32, for capturing peptide-MHC monomers and a HRP conjugated polyclonal rabbit anti-human ß<sub>2</sub>m antibody (DAKO) for detection of correctly folded complexes. The ELISA was developed with EnVision™ (DAKO #K4003). MHC-I concentration was measured by spectrophotometric analysis in a Microplate Scanning Spectrophotometer, Discovery HT-R (BIO-TEK Instruments). A peptide-MHC monomer complex of known concentration was used as reference for conversion of absorbance data to concentrations. A sigmoid fit (y = B<sub>max</sub>*x/(K<sub>D</sub>+x) of each peptide titration curve on semi logarithmic coordinate scales provides B<sub>max</sub> and K<sub>D</sub> values.</p>


<h4>Size exclusion chromatography of peptide-MHC tetramers</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>The tetramerization of peptide-MHC-I monomer complexes with SA was visualized by Superdex 200 (Amersham Biosciences) chromatography. Iodinated (<sup>125</sup>I) peptide-MHC-I monomers were generated using radioactively labeled peptides, and subsequently used to spike a preparation of unlabeled peptide-MHC-I monomers. The spiked peptide-MHC-I complexes were then reacted with various stoichiometric ratios of SA. A 100 µl sample of MHC-SA complexes were analyzed on a 23 ml pre-packed and calibrated Superdex 200 SEC column; 200 µl fractions were collected, and analyzed by gamma-spectrometry (Cobra 5010, Packard).</p>


<h4>“One-pot” MHC tetramer production</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Peptide-MHC-I monomer complexes were generated by diluting the HC (typical stock concentration 15–40 µM)) 100-fold into a reaction buffer containing 50 mM tris-maleate pH 6.6, 0.1% pluronic F86 NF (BASF, a surfactant compatible with cellular use), a 20-fold molar excess of ß<sub>2</sub>m, and 50-fold molar excess of peptide. The mixture was vortex'ed briefly and incubated at 18°C for 48 hours. Streptavidin-R-Phycoerythrin (SA-PE, Becton Dickinson) was then added (sequentially in 10 small aliquots over 90 min) at a molar ratio of four MHC to one SA. This reaction mixture containing tetramers was stored at 4°C.</p>


<h4>Cell lines and culture condition</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>Buffy coats were obtained from healthy volunteer Danish donors from The Blood Bank at the national Hospital (Copenhagen, Denmark). Peripheral blood mononuclear cells (PBMC) were isolated by density gradient centrifugation using Ficoll-Paque (GE-healthcare). To stimulate specific CTL, 2 µg/ml peptide were added to the PBMC and incubated for 8–9 days in Xvivo15 (Lonza) supplemented with 5% autologous serum. IL2 at 50 U/ml were added at day 1, 4, and 6. Dendritic cells (DC) were used as APC. PBMC were adhered for 2 h at 37°C in 24-well plates. Adherent cells were cultured for 9 days in Xvivo15 with 5% AB serum, 100 ng/ml GM-CSF, and 100 ng/ml IL4. At day 7 the DC were activated with a mixture of 10 ng/ml TNFα, 5 ng/ml IL1β, 20 ng/ml IL6, and 1 µg/ml PGE2.</p>


<h4>Tetramer staining</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>PBMC or T cells, harvested day 8, were aliquoted at 2*10<sup>5</sup>/well in a 96-well round bottom microtiter well. The cells were pelleted and resuspended in 25 µl tetramer, incubated for 20 min at RT followed by 30 min incubation with anti CD8-APC (Pharmingen) at 4°C. After two washes the cells were analyzed on a FACScalibur.</p>


<h4>Intracellular staining for IFNγ</h4>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1"></a><p>DC's were pulsed with or without peptide for 1 h and added to the T cells. The cells were incubated for 3h with Brefeldin A at 37°C. Subsequently, staining for CD8, CD69 and intracellular IFNγ was carried out according to the Fast Immune protocol (Pharmingen). In some cases prior to the Fast Immune staining the cells were stained with tetramers as described above. The cells were analyzed on a FACScalibur.</p>

</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SB AS. Performed the experiments: KL SJ CL NL CS AS ES. Analyzed the data: SB SJ CL NL CS MC AS ES. Contributed reagents/materials/analysis tools: KL. Wrote the paper: SB MC AS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001678-Czerkinsky1" id="pone.0001678-Czerkinsky1"></a>Czerkinsky C, Moldoveanu Z, Mestecky J, Nilsson LA, Ouchterlony O (1988) A novel two colour ELISPOT assay. I. Simultaneous detection of distinct types of antibody-secreting cells. J Immunol Methods  115: 31–37.  <ul class="find" data-citedArticleID="1062954" data-doi="10.1016/0022-1759(88)90306-7"><li><a href="http://dx.doi.org/10.1016/0022-1759(88)90306-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+two+colour+ELISPOT+assay.+I.+Simultaneous+detection+of+distinct+types+of+antibody-secreting+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+two+colour+ELISPOT+assay.+I.+Simultaneous+detection+of+distinct+types+of+antibody-secreting+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001678-Scheibenbogen1" id="pone.0001678-Scheibenbogen1"></a>Scheibenbogen C, Lee KH, Mayer S, Stevanovic S, Moebius U, et al.  (1997) A sensitive ELISPOT assay for detection of CD8+ T lymphocytes specific for HLA class I-binding peptide epitopes derived from influenza proteins in the blood of healthy donors and melanoma patients. Clin Cancer Res  3: 221–226.  <ul class="find" data-citedArticleID="1062976"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+sensitive+ELISPOT+assay+for+detection+of+CD8%2B+T+lymphocytes+specific+for+HLA+class+I-binding+peptide+epitopes+derived+from+influenza+proteins+in+the+blood+of+healthy+donors+and+melanoma+patients.&amp;auth=&amp;atitle=A+sensitive+ELISPOT+assay+for+detection+of+CD8%2B+T+lymphocytes+specific+for+HLA+class+I-binding+peptide+epitopes+derived+from+influenza+proteins+in+the+blood+of+healthy+donors+and+melanoma+patients." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+sensitive+ELISPOT+assay+for+detection+of+CD8%2B+T+lymphocytes+specific+for+HLA+class+I-binding+peptide+epitopes+derived+from+influenza+proteins+in+the+blood+of+healthy+donors+and+melanoma+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+sensitive+ELISPOT+assay+for+detection+of+CD8%2B+T+lymphocytes+specific+for+HLA+class+I-binding+peptide+epitopes+derived+from+influenza+proteins+in+the+blood+of+healthy+donors+and+melanoma+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001678-Osterborg1" id="pone.0001678-Osterborg1"></a>Osterborg A, Yi Q, Bergenbrant S, Holm G, Lefvert AK, et al.  (1995) Idiotype-specific T cells in multiple myeloma stage I: an evaluation by four different functional tests. Br J Haematol  89: 110–116.  <ul class="find" data-citedArticleID="1062964" data-doi="10.1111/j.1365-2141.1995.tb08902.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2141.1995.tb08902.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Idiotype-specific+T+cells+in+multiple+myeloma+stage+I%3A+an+evaluation+by+four+different+functional+tests." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Idiotype-specific+T+cells+in+multiple+myeloma+stage+I%3A+an+evaluation+by+four+different+functional+tests.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001678-Romero1" id="pone.0001678-Romero1"></a>Romero P, Cerottini JC, Waanders GA (1998) Novel methods to monitor antigen-specific cytotoxic T-cell responses in cancer immunotherapy. Mol Med Today  4: 305–312.  <ul class="find" data-citedArticleID="1062970" data-doi="10.1016/s1357-4310(98)01280-5"><li><a href="http://dx.doi.org/10.1016/s1357-4310(98)01280-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+methods+to+monitor+antigen-specific+cytotoxic+T-cell+responses+in+cancer+immunotherapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+methods+to+monitor+antigen-specific+cytotoxic+T-cell+responses+in+cancer+immunotherapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001678-Suni1" id="pone.0001678-Suni1"></a>Suni MA, Picker LJ, Maino VC (1998) Detection of antigen-specific T cell cytokine expression in whole blood by flow cytometry. J Immunol Methods  212: 89–98.  <ul class="find" data-citedArticleID="1062978" data-doi="10.1016/s0022-1759(98)00004-0"><li><a href="http://dx.doi.org/10.1016/s0022-1759(98)00004-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+antigen-specific+T+cell+cytokine+expression+in+whole+blood+by+flow+cytometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+antigen-specific+T+cell+cytokine+expression+in+whole+blood+by+flow+cytometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001678-Markovic1" id="pone.0001678-Markovic1"></a>Markovic SN, Nevala WK, Uhl CB, Celis E, McKean DJ (2006) A reproducible method for the enumeration of functional (cytokine producing) versus non-functional peptide-specific cytotoxic T lymphocytes in human peripheral blood. Clin Exp Immunol  145: 438–447.  <ul class="find" data-citedArticleID="1062962" data-doi="10.1111/j.1365-2249.2006.03157.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2249.2006.03157.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+reproducible+method+for+the+enumeration+of+functional+%28cytokine+producing%29+versus+non-functional+peptide-specific+cytotoxic+T+lymphocytes+in+human+peripheral+blood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+reproducible+method+for+the+enumeration+of+functional+%28cytokine+producing%29+versus+non-functional+peptide-specific+cytotoxic+T+lymphocytes+in+human+peripheral+blood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001678-Altman1" id="pone.0001678-Altman1"></a>Altman JD, Moss PA, Goulder PJ, Barouch DH, McHeyzer-Williams MG, et al.  (1996) Phenotypic analysis of antigen-specific T lymphocytes. Science  274: 94–96.  <ul class="find" data-citedArticleID="1062950" data-doi="10.1126/science.274.5284.94"><li><a href="http://dx.doi.org/10.1126/science.274.5284.94" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypic+analysis+of+antigen-specific+T+lymphocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypic+analysis+of+antigen-specific+T+lymphocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001678-Keenan1" id="pone.0001678-Keenan1"></a>Keenan RD, Ainsworth J, Khan N, Bruton R, Cobbold M, et al.  (2001) Purification of cytomegalovirus-specific CD8 T cells from peripheral blood using HLA-peptide tetramers. Br J Haematol  115: 428–434.  <ul class="find" data-citedArticleID="1062960" data-doi="10.1046/j.1365-2141.2001.03106.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2141.2001.03106.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification+of+cytomegalovirus-specific+CD8+T+cells+from+peripheral+blood+using+HLA-peptide+tetramers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification+of+cytomegalovirus-specific+CD8+T+cells+from+peripheral+blood+using+HLA-peptide+tetramers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001678-Savage1" id="pone.0001678-Savage1"></a>Savage P, Millrain M, Dimakou S, Stebbing J, Dyson J (2007) Expansion of CD8+ cytotoxic T cells in vitro and in vivo using MHC class I tetramers. Tumour Biol  28: 70–76.  <ul class="find" data-citedArticleID="1062972" data-doi="10.1159/000099152"><li><a href="http://dx.doi.org/10.1159/000099152" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expansion+of+CD8%2B+cytotoxic+T+cells+in+vitro+and+in+vivo+using+MHC+class+I+tetramers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expansion+of+CD8%2B+cytotoxic+T+cells+in+vitro+and+in+vivo+using+MHC+class+I+tetramers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001678-Savage2" id="pone.0001678-Savage2"></a>Savage P, Cowburn P, Clayton A, Man S, McMichael A, et al.  (2002) Induction of viral and tumour specific CTL responses using antibody targeted HLA class I peptide complexes. Br J Cancer  86: 1336–1342.  <ul class="find" data-citedArticleID="1062974"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Induction+of+viral+and+tumour+specific+CTL+responses+using+antibody+targeted+HLA+class+I+peptide+complexes.&amp;auth=&amp;atitle=Induction+of+viral+and+tumour+specific+CTL+responses+using+antibody+targeted+HLA+class+I+peptide+complexes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+viral+and+tumour+specific+CTL+responses+using+antibody+targeted+HLA+class+I+peptide+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+viral+and+tumour+specific+CTL+responses+using+antibody+targeted+HLA+class+I+peptide+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001678-Ferre1" id="pone.0001678-Ferre1"></a>Ferre H, Ruffet E, Blicher T, Sylvester-Hvid C, Nielsen LL, et al.  (2003) Purification of correctly oxidized MHC class I heavy-chain molecules under denaturing conditions: a novel strategy exploiting disulfide assisted protein folding. Protein Sci  12: 551–559.  <ul class="find" data-citedArticleID="1062958" data-doi="10.1110/ps.0233003"><li><a href="http://dx.doi.org/10.1110/ps.0233003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification+of+correctly+oxidized+MHC+class+I+heavy-chain+molecules+under+denaturing+conditions%3A+a+novel+strategy+exploiting+disulfide+assisted+protein+folding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification+of+correctly+oxidized+MHC+class+I+heavy-chain+molecules+under+denaturing+conditions%3A+a+novel+strategy+exploiting+disulfide+assisted+protein+folding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001678-Cull1" id="pone.0001678-Cull1"></a>Cull MG, Schatz PJ (2000) Biotinylation of proteins in vivo and in vitro using small peptide tags. Methods Enzymol  326: 430–440.  <ul class="find" data-citedArticleID="1062952" data-doi="10.1016/s0076-6879(00)26068-0"><li><a href="http://dx.doi.org/10.1016/s0076-6879(00)26068-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biotinylation+of+proteins+in+vivo+and+in+vitro+using+small+peptide+tags." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biotinylation+of+proteins+in+vivo+and+in+vitro+using+small+peptide+tags.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001678-Yang1" id="pone.0001678-Yang1"></a>Yang J, Jaramillo A, Shi R, Kwok WW, Mohanakumar T (2004) In vivo biotinylation of the major histocompatibility complex (MHC) class II/peptide complex by coexpression of BirA enzyme for the generation of MHC class II/tetramers. Hum Immunol  65: 692–699.  <ul class="find" data-citedArticleID="1062982" data-doi="10.1016/j.humimm.2004.04.001"><li><a href="http://dx.doi.org/10.1016/j.humimm.2004.04.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+biotinylation+of+the+major+histocompatibility+complex+%28MHC%29+class+II%2Fpeptide+complex+by+coexpression+of+BirA+enzyme+for+the+generation+of+MHC+class+II%2Ftetramers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+biotinylation+of+the+major+histocompatibility+complex+%28MHC%29+class+II%2Fpeptide+complex+by+coexpression+of+BirA+enzyme+for+the+generation+of+MHC+class+II%2Ftetramers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001678-SylvesterHvid1" id="pone.0001678-SylvesterHvid1"></a>Sylvester-Hvid C, Kristensen N, Blicher T, Ferre H, Lauemoller SL, et al.  (2002) Establishment of a quantitative ELISA capable of determining peptide - MHC class I interaction. Tissue Antigens  59: 251–258.  <ul class="find" data-citedArticleID="1062980" data-doi="10.1034/j.1399-0039.2002.590402.x"><li><a href="http://dx.doi.org/10.1034/j.1399-0039.2002.590402.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Establishment+of+a+quantitative+ELISA+capable+of+determining+peptide+-+MHC+class+I+interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Establishment+of+a+quantitative+ELISA+capable+of+determining+peptide+-+MHC+class+I+interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001678-Ostergaard1" id="pone.0001678-Ostergaard1"></a>Ostergaard PL, Nissen MH, Hansen NJ, Nielsen LL, Lauenmoller SL, et al.  (2001) Efficient assembly of recombinant major histocompatibility complex class I molecules with preformed disulfide bonds. 31: 2986–2996.  <ul class="find" data-citedArticleID="1062966" data-doi="10.1002/1521-4141(2001010)31:10<2986::aid-immu2986&gt;3.0.co;2-r"><li><a href="http://dx.doi.org/10.1002/1521-4141(2001010)31:10<2986::aid-immu2986&gt;3.0.co;2-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+assembly+of+recombinant+major+histocompatibility+complex+class+I+molecules+with+preformed+disulfide+bonds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+assembly+of+recombinant+major+histocompatibility+complex+class+I+molecules+with+preformed+disulfide+bonds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001678-Delon1" id="pone.0001678-Delon1"></a>Delon J, Gregoire C, Malissen B, Darche S, Lemaitre F, et al.  (1998) CD8 expression allows T cell signaling by monomeric peptide-MHC complexes. Immunity  9: 467–473.  <ul class="find" data-citedArticleID="1062956" data-doi="10.1016/s1074-7613(00)80630-5"><li><a href="http://dx.doi.org/10.1016/s1074-7613(00)80630-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD8+expression+allows+T+cell+signaling+by+monomeric+peptide-MHC+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD8+expression+allows+T+cell+signaling+by+monomeric+peptide-MHC+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001678-Rodenko1" id="pone.0001678-Rodenko1"></a>Rodenko B, Toebes M, Hadrup SR, van Esch WJ, Molenaar AM, et al.  (2006) Generation of peptide-MHC class I complexes through UV-mediated ligand exchange. Nat Protoc  1: 1120–1132.  <ul class="find" data-citedArticleID="1062968" data-doi="10.1038/nprot.2006.121"><li><a href="http://dx.doi.org/10.1038/nprot.2006.121" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generation+of+peptide-MHC+class+I+complexes+through+UV-mediated+ligand+exchange." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generation+of+peptide-MHC+class+I+complexes+through+UV-mediated+ligand+exchange.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.XML" value="73588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.PDF" value="852953"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g001.PNG_L" value="1133744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g001.PNG_M" value="95723"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g001.PNG_S" value="4245"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g001.TIF" value="1077796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g001.PNG_I" value="31217"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.t001.PNG_L" value="216407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.t001.PNG_M" value="86728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.t001.PNG_S" value="20006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.t001.TIF" value="1251986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.t001.PNG_I" value="142920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g002.PNG_L" value="1779250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g002.PNG_M" value="69931"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g002.PNG_S" value="3847"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g002.TIF" value="2311686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g002.PNG_I" value="21006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g003.PNG_L" value="984213"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g003.PNG_M" value="56351"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g003.PNG_S" value="3920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g003.TIF" value="1629434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g003.PNG_I" value="21325"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g004.PNG_L" value="1707028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g004.PNG_M" value="114805"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g004.PNG_S" value="4832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g004.TIF" value="3064864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g004.PNG_I" value="39804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g005.PNG_L" value="405201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g005.PNG_M" value="78052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g005.PNG_S" value="5439"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g005.TIF" value="729304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g005.PNG_I" value="29509"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g006.PNG_L" value="2021102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g006.PNG_M" value="75890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g006.PNG_S" value="6509"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g006.TIF" value="3555382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g006.PNG_I" value="24720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g007.PNG_L" value="3262390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g007.PNG_M" value="170786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g007.PNG_S" value="6900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g007.TIF" value="5203766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g007.PNG_I" value="55658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g008.PNG_L" value="5792311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g008.PNG_M" value="270102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g008.PNG_S" value="9281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g008.TIF" value="8702582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g008.PNG_I" value="86394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g009.PNG_L" value="2939571"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g009.PNG_M" value="158045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g009.PNG_S" value="4935"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g009.TIF" value="4416870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001678.g009.PNG_I" value="50368"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001678&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001678" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001678&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001678&volume=&issue=&title=One-Pot, Mix-and-Read Peptide-MHC Tetramers&author_name=Christian%20Leisner%2C%20Nina%20Loeth%2C%20Kasper%20Lamberth%2C%20Sune%20Justesen%2C%20Christina%20Sylvester-Hvid%2C%20Esben%20G.%20Schmidt%2C%20Mogens%20Claesson%2C%20Soren%20Buus%2C%20Anette%20Stryhn&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001678" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001678" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001678" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001678&amp;t=One-Pot%2C%20Mix-and-Read%20Peptide-MHC%20Tetramers" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001678&title=One-Pot%2C%20Mix-and-Read%20Peptide-MHC%20Tetramers&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001678&amp;title=One-Pot%2C%20Mix-and-Read%20Peptide-MHC%20Tetramers" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001678" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001678&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'One-Pot, Mix-and-Read Peptide-MHC Tetramers';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001678';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20One-Pot%2C%20Mix-and-Read%20Peptide-MHC%20Tetramers http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001678" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001678" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Binders%22" title="Search for articles in the subject area:'Binders'"><div class="flagText">Binders</div></a>
              <div data-categoryid="22201" data-articleid="26182"
                   data-categoryname="Binders"
                   class="flagImage" title="Flag 'Binders' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biotinylation%22" title="Search for articles in the subject area:'Biotinylation'"><div class="flagText">Biotinylation</div></a>
              <div data-categoryid="23137" data-articleid="26182"
                   data-categoryname="Biotinylation"
                   class="flagImage" title="Flag 'Biotinylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+staining%22" title="Search for articles in the subject area:'Cell staining'"><div class="flagText">Cell staining</div></a>
              <div data-categoryid="17401" data-articleid="26182"
                   data-categoryname="Cell staining"
                   class="flagImage" title="Flag 'Cell staining' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzyme-linked+immunoassays%22" title="Search for articles in the subject area:'Enzyme-linked immunoassays'"><div class="flagText">Enzyme-linked immunoassays</div></a>
              <div data-categoryid="20331" data-articleid="26182"
                   data-categoryname="Enzyme-linked immunoassays"
                   class="flagImage" title="Flag 'Enzyme-linked immunoassays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ion+channel+gating%22" title="Search for articles in the subject area:'Ion channel gating'"><div class="flagText">Ion channel gating</div></a>
              <div data-categoryid="51859" data-articleid="26182"
                   data-categoryname="Ion channel gating"
                   class="flagImage" title="Flag 'Ion channel gating' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Negative+staining%22" title="Search for articles in the subject area:'Negative staining'"><div class="flagText">Negative staining</div></a>
              <div data-categoryid="22325" data-articleid="26182"
                   data-categoryname="Negative staining"
                   class="flagImage" title="Flag 'Negative staining' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Peptides%22" title="Search for articles in the subject area:'Peptides'"><div class="flagText">Peptides</div></a>
              <div data-categoryid="17847" data-articleid="26182"
                   data-categoryname="Peptides"
                   class="flagImage" title="Flag 'Peptides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22T+cells%22" title="Search for articles in the subject area:'T cells'"><div class="flagText">T cells</div></a>
              <div data-categoryid="40147" data-articleid="26182"
                   data-categoryname="T cells"
                   class="flagImage" title="Flag 'T cells' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8599'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5101'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1181&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=15759">Fast peptide loading is possible</a><br>Posted by SorenBuus</p>
      <p><a href="/annotation/listThread.action?root=21467">Is this technology truly preferable?</a><br>Posted by Ton_Schumacher</p>
      <p><a href="/annotation/listThread.action?root=3961">Referee comments: Referee 3</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
