

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Sequence Evidence in the Archaeal Genomes that tRNAs Emerged Through the Combination of Ancestral Genes as 5′ and 3′ tRNA Halves</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001622"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001622" />

    <meta name="citation_title" content="Sequence Evidence in the Archaeal Genomes that tRNAs Emerged Through the Combination of Ancestral Genes as 5′ and 3′ tRNA Halves"/>
    <meta itemprop="name" content="Sequence Evidence in the Archaeal Genomes that tRNAs Emerged Through the Combination of Ancestral Genes as 5′ and 3′ tRNA Halves"/>

      <meta name="citation_author" content="Kosuke Fujishima"/>
            <meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan"/>
            <meta name="citation_author_institution" content="Systems Biology Program, Graduate School of Media and Governance, Keio University, Fujisawa, Japan"/>
      <meta name="citation_author" content="Junichi Sugahara"/>
            <meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan"/>
            <meta name="citation_author_institution" content="Systems Biology Program, Graduate School of Media and Governance, Keio University, Fujisawa, Japan"/>
      <meta name="citation_author" content="Masaru Tomita"/>
            <meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan"/>
            <meta name="citation_author_institution" content="Systems Biology Program, Graduate School of Media and Governance, Keio University, Fujisawa, Japan"/>
            <meta name="citation_author_institution" content="Department of Environment and Information, Keio University, Fujisawa, Japan"/>
      <meta name="citation_author" content="Akio Kanai"/>
            <meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan"/>
            <meta name="citation_author_institution" content="Systems Biology Program, Graduate School of Media and Governance, Keio University, Fujisawa, Japan"/>
            <meta name="citation_author_institution" content="Department of Environment and Information, Keio University, Fujisawa, Japan"/>

    <meta name="citation_date" content="2008/2/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001622.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1622"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Phylogeny from function: evidence from the molecular fossil record that tRNA originated in replication, not translation.; citation_author=N Maizels; citation_author=AM Weiner; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=1; citation_pages=6729-6734; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The genomic tag hypothesis: modern viruses as molecular fossils of ancient strategies for genomic replication, and clues regarding the origin of protein synthesis.; citation_author=AM Weiner; citation_author=N Maizels; citation_journal_title=Biol Bull; citation_volume=196; citation_number=2; citation_pages=327-330; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The non-monophyletic origin of the tRNA molecule and the origin of genes only after the evolutionary stage of the last universal common ancestor (LUCA).; citation_author=M Di Giulio; citation_journal_title=J Theor Biol; citation_volume=240; citation_number=3; citation_pages=343-352; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=RNA ligation and the origin of tRNA.; citation_author=U Nagaswamy; citation_author=GE Fox; citation_journal_title=Orig Life Evol Biosph; citation_volume=33; citation_number=4; citation_pages=199-209; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The non-monophyletic origin of the tRNA molecule.; citation_author=M Di Giulio; citation_journal_title=J Theor Biol; citation_volume=197; citation_number=5; citation_pages=403-414; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=tRNomics: analysis of tRNA genes from 50 genomes of Eukarya, Archaea, and Bacteria reveals anticodon-sparing strategies and domain-specific features.; citation_author=C Marck; citation_author=H Grosjean; citation_journal_title=Rna; citation_volume=8; citation_number=6; citation_pages=1189-1232; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=SPLITS: a new program for predicting split and intron-containing tRNA genes at the genome level.; citation_author=J Sugahara; citation_author=N Yachie; citation_author=Y Sekine; citation_author=A Soma; citation_author=M Matsui; citation_journal_title=In Silico Biol; citation_volume=6; citation_number=7; citation_pages=411-418; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=In silico screening of archaeal tRNA-encoding genes having multiple introns with bulge-helix-bulge splicing motifs.; citation_author=J Sugahara; citation_author=N Yachie; citation_author=K Arakawa; citation_author=M Tomita; citation_journal_title=Rna; citation_volume=13; citation_number=8; citation_pages=671-681; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Nanoarchaeum equitans creates functional tRNAs from separate genes for their 5′- and 3′-halves.; citation_author=L Randau; citation_author=R Munch; citation_author=MJ Hohn; citation_author=D Jahn; citation_author=D Soll; citation_journal_title=Nature; citation_volume=433; citation_number=9; citation_pages=537-541; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Coevolution of tRNA intron motifs and tRNA endonuclease architecture in Archaea.; citation_author=GD Tocchini-Valentini; citation_author=P Fruscoloni; citation_author=GP Tocchini-Valentini; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=10; citation_pages=15418-15422; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The heteromeric Nanoarchaeum equitans splicing endonuclease cleaves noncanonical bulge-helix-bulge motifs of joined tRNA halves.; citation_author=L Randau; citation_author=K Calvin; citation_author=M Hall; citation_author=J Yuan; citation_author=M Podar; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=11; citation_pages=17934-17939; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of BHB splicing motifs in intron-containing tRNAs from 18 archaea: evolutionary implications.; citation_author=C Marck; citation_author=H Grosjean; citation_journal_title=Rna; citation_volume=9; citation_number=12; citation_pages=1516-1531; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Solution structure of a substrate for the archaeal pre-tRNA splicing endonucleases: the bulge-helix-bulge motif.; citation_author=JL Diener; citation_author=PB Moore; citation_journal_title=Mol Cell; citation_volume=1; citation_number=13; citation_pages=883-894; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Archaeal pre-mRNA splicing: a connection to hetero-oligomeric splicing endonuclease.; citation_author=S Yoshinari; citation_author=T Itoh; citation_author=SJ Hallam; citation_author=EF DeLong; citation_author=S Yokobori; citation_journal_title=Biochem Biophys Res Commun; citation_volume=346; citation_number=14; citation_pages=1024-1032; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Introns in protein-coding genes in Archaea.; citation_author=Y Watanabe; citation_author=S Yokobori; citation_author=T Inaba; citation_author=A Yamagishi; citation_author=T Oshima; citation_journal_title=FEBS Lett; citation_volume=510; citation_number=15; citation_pages=27-30; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=RNomics in Archaea reveals a further link between splicing of archaeal introns and rRNA processing.; citation_author=TH Tang; citation_author=TS Rozhdestvensky; citation_author=BC d'Orval; citation_author=ML Bortolin; citation_author=H Huber; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=16; citation_pages=921-930; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Crystal structure and evolution of a transfer RNA splicing enzyme.; citation_author=H Li; citation_author=CR Trotta; citation_author=J Abelson; citation_journal_title=Science; citation_volume=280; citation_number=17; citation_pages=279-284; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Another bridge between kingdoms: tRNA splicing in archaea and eukaryotes.; citation_author=M Belfort; citation_author=A Weiner; citation_journal_title=Cell; citation_volume=89; citation_number=18; citation_pages=1003-1006; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Permuted tRNA genes expressed via a circular RNA intermediate in Cyanidioschyzon merolae.; citation_author=A Soma; citation_author=A Onodera; citation_author=J Sugahara; citation_author=A Kanai; citation_author=N Yachie; citation_journal_title=Science; citation_volume=318; citation_number=19; citation_pages=450-453; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Nanoarchaeum equitans is a living fossil.; citation_author=M Di Giulio; citation_journal_title=J Theor Biol; citation_volume=242; citation_number=20; citation_pages=257-260; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=On the origin of the transfer RNA molecule.; citation_author=M Di Giulio; citation_journal_title=J Theor Biol; citation_volume=159; citation_number=21; citation_pages=199-214; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Operational RNA code for amino acids: species-specific aminoacylation of minihelices switched by a single nucleotide.; citation_author=D Hipps; citation_author=K Shiba; citation_author=B Henderson; citation_author=P Schimmel; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=22; citation_pages=5550-5552; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The genome of Nanoarchaeum equitans: insights into early archaeal evolution and derived parasitism.; citation_author=E Waters; citation_author=MJ Hohn; citation_author=I Ahel; citation_author=DE Graham; citation_author=MD Adams; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=23; citation_pages=12984-12988; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=An emerging phylogenetic core of Archaea: phylogenies of transcription and translation machineries converge following addition of new genome sequences.; citation_author=C Brochier; citation_author=P Forterre; citation_author=S Gribaldo; citation_journal_title=BMC Evol Biol; citation_volume=5; citation_number=24; citation_pages=36; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.; citation_author=TM Lowe; citation_author=SR Eddy; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=25; citation_pages=955-964; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A co-evolution theory of the genetic code.; citation_author=J Wong; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=72; citation_number=26; citation_pages=1909-1912; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=Comparing two evolutionary mechanisms of modern tRNAs.; citation_author=F Wei; citation_author=M Meng; citation_author=S Li; citation_author=H Ma; citation_journal_title=Mol Phylogenet Evol; citation_volume=38; citation_number=27; citation_pages=1-11; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Emergence of scaling in random networks.; citation_author=AL Barabasi; citation_author=R Albert; citation_journal_title=Science; citation_volume=286; citation_number=28; citation_pages=509-512; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Collective dynamics of ‘small-world’ networks.; citation_author=DJ Watts; citation_author=SH Strogatz; citation_journal_title=Nature; citation_volume=393; citation_number=29; citation_pages=440-442; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The complete set of tRNA species in Nanoarchaeum equitans.; citation_author=L Randau; citation_author=M Pearson; citation_author=D Soll; citation_journal_title=FEBS Lett; citation_volume=579; citation_number=30; citation_pages=2945-2947; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A human 28S ribosomal RNA retropseudogene.; citation_author=S Wang; citation_author=IL Pirtle; citation_author=RM Pirtle; citation_journal_title=Gene; citation_volume=196; citation_number=31; citation_pages=105-111; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Retroposition of processed pseudogenes: the impact of RNA stability and translational control.; citation_author=A Pavlicek; citation_author=AJ Gentles; citation_author=J Paces; citation_author=V Paces; citation_author=J Jurka; citation_journal_title=Trends Genet; citation_volume=22; citation_number=32; citation_pages=69-73; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The presence of tRNA pseudogenes in mammalia and plants and their absence in yeast may account for different specificities of pre-tRNA processing enzymes.; citation_author=Y Kacar; citation_author=H Beier; citation_author=HJ Gross; citation_journal_title=Gene; citation_volume=156; citation_number=33; citation_pages=129-132; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Several short interspersed repetitive elements (SINEs) in distant species may have originated from a common ancestral retrovirus: characterization of a squid SINE and a possible mechanism for generation of tRNA-derived retroposons.; citation_author=K Ohshima; citation_author=R Koishi; citation_author=M Matsuo; citation_author=N Okada; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=34; citation_pages=6260-6264; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A novel class of mammalian-specific tailless retropseudogenes.; citation_author=J Schmitz; citation_author=G Churakov; citation_author=H Zischler; citation_author=J Brosius; citation_journal_title=Genome Res; citation_volume=14; citation_number=35; citation_pages=1911-1915; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The Mauriceville retroplasmid reverse transcriptase initiates cDNA synthesis de novo at the 3′ end of tRNAs.; citation_author=CC Chiang; citation_author=AM Lambowitz; citation_journal_title=Mol Cell Biol; citation_volume=17; citation_number=36; citation_pages=4526-4535; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=16S rRNA phylogenetic investigation of the candidate division “Korarchaeota”.; citation_author=TA Auchtung; citation_author=CD Takacs-Vesbach; citation_author=CM Cavanaugh; citation_journal_title=Appl Environ Microbiol; citation_volume=72; citation_number=37; citation_pages=5077-5082; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.; citation_author=JD Thompson; citation_author=TJ Gibson; citation_author=F Plewniak; citation_author=F Jeanmougin; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=38; citation_pages=4876-4882; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=An enhanced RNA alignment benchmark for sequence alignment programs.; citation_author=A Wilm; citation_author=I Mainz; citation_author=G Steger; citation_journal_title=Algorithms Mol Biol; citation_volume=1; citation_number=39; citation_pages=19; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MRBAYES: Bayesian inference of phylogenetic trees.; citation_author=JP Huelsenbeck; citation_author=F Ronquist; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=40; citation_pages=754-755; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Visualizing large hierarchical clusters in hyperbolic space.; citation_author=J Bingham; citation_author=S Sudarsanam; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=41; citation_pages=660-661; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks.; citation_author=P Shannon; citation_author=A Markiel; citation_author=O Ozier; citation_author=NS Baliga; citation_author=JT Wang; citation_journal_title=Genome Res; citation_volume=13; citation_number=42; citation_pages=2498-2504; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001622" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Sequence Evidence in the Archaeal Genomes that tRNAs Emerged Through the Combination of Ancestral Genes as 5′ and 3′ tRNA Halves"/>
    <meta name="twitter:description" content="The discovery of separate 5&prime; and 3&prime; halves of transfer RNA (tRNA) molecules&mdash;so-called split tRNA&mdash;in the archaeal parasite Nanoarchaeum equitans made us wonder whether ancestral tRNA was encoded on 1 or 2 genes. We performed a comprehensive phylogenetic analysis of tRNAs in 45 archaeal species to explore the relationship between the three types of tRNAs (nonintronic, intronic and split). We classified 1953 mature tRNA sequences into 22 clusters. All split tRNAs have shown phylogenetic relationships with other tRNAs possessing the same anticodon. We also mimicked split tRNA by artificially separating the tRNA sequences of 7 primitive archaeal species at the anticodon and analyzed the sequence similarity and diversity of the 5&prime; and 3&prime; tRNA halves. Network analysis revealed specific characteristics of and topological differences between the 5&prime; and 3&prime; tRNA halves: the 5&prime; half sequences were categorized into 6 distinct groups with a sequence similarity of &gt;80%, while the 3&prime; half sequences were categorized into 9 groups with a higher sequence similarity of &gt;88%, suggesting different evolutionary backgrounds of the 2 halves. Furthermore, the combinations of 5&prime; and 3&prime; halves corresponded with the variation of amino acids in the codon table. We found not only universally conserved combinations of 5&prime;&ndash;3&prime; tRNA halves in tRNAiMet, tRNAThr, tRNAIle, tRNAGly, tRNAGln, tRNAGlu, tRNAAsp, tRNALys, tRNAArg and tRNALeu but also phylum-specific combinations in tRNAPro, tRNAAla, and tRNATrp. Our results support the idea that tRNA emerged through the combination of separate genes and explain the sequence diversity that arose during archaeal tRNA evolution."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001622.g006"/>

  <meta property="og:title" content="Sequence Evidence in the Archaeal Genomes that tRNAs Emerged Through the Combination of Ancestral Genes as 5′ and 3′ tRNA Halves" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001622" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7378'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7008'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8687&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001622">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001622" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001622&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Sequence Evidence in the Archaeal Genomes that tRNAs Emerged Through the Combination of Ancestral Genes as 5′ and 3′ tRNA Halves
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kosuke Fujishima, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan, 
                  Systems Biology Program, Graduate School of Media and Governance, Keio University, Fujisawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Junichi Sugahara, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan, 
                  Systems Biology Program, Graduate School of Media and Governance, Keio University, Fujisawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Masaru Tomita, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan, 
                  Systems Biology Program, Graduate School of Media and Governance, Keio University, Fujisawa, Japan, 
                  Department of Environment and Information, Keio University, Fujisawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Akio Kanai
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:akio@sfc.keio.ac.jp">akio@sfc.keio.ac.jp</a></p>

                <p>Affiliations:
                  Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan, 
                  Systems Biology Program, Graduate School of Media and Governance, Keio University, Fujisawa, Japan, 
                  Department of Environment and Information, Keio University, Fujisawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 20, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001622</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001622" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001622" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001622" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001622" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001622-g001" data-doi="info:doi/10.1371/journal.pone.0001622" data-uri="info:doi/10.1371/journal.pone.0001622.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001622-g002" data-doi="info:doi/10.1371/journal.pone.0001622" data-uri="info:doi/10.1371/journal.pone.0001622.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001622-g003" data-doi="info:doi/10.1371/journal.pone.0001622" data-uri="info:doi/10.1371/journal.pone.0001622.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001622-g004" data-doi="info:doi/10.1371/journal.pone.0001622" data-uri="info:doi/10.1371/journal.pone.0001622.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001622-g005" data-doi="info:doi/10.1371/journal.pone.0001622" data-uri="info:doi/10.1371/journal.pone.0001622.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001622-g006" data-doi="info:doi/10.1371/journal.pone.0001622" data-uri="info:doi/10.1371/journal.pone.0001622.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001622">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001622" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The discovery of separate 5′ and 3′ halves of transfer RNA (tRNA) molecules—so-called split tRNA—in the archaeal parasite <em>Nanoarchaeum equitans</em> made us wonder whether ancestral tRNA was encoded on 1 or 2 genes. We performed a comprehensive phylogenetic analysis of tRNAs in 45 archaeal species to explore the relationship between the three types of tRNAs (nonintronic, intronic and split). We classified 1953 mature tRNA sequences into 22 clusters. All split tRNAs have shown phylogenetic relationships with other tRNAs possessing the same anticodon. We also mimicked split tRNA by artificially separating the tRNA sequences of 7 primitive archaeal species at the anticodon and analyzed the sequence similarity and diversity of the 5′ and 3′ tRNA halves. Network analysis revealed specific characteristics of and topological differences between the 5′ and 3′ tRNA halves: the 5′ half sequences were categorized into 6 distinct groups with a sequence similarity of &gt;80%, while the 3′ half sequences were categorized into 9 groups with a higher sequence similarity of &gt;88%, suggesting different evolutionary backgrounds of the 2 halves. Furthermore, the combinations of 5′ and 3′ halves corresponded with the variation of amino acids in the codon table. We found not only universally conserved combinations of 5′–3′ tRNA halves in tRNA<sup>iMet</sup>, tRNA<sup>Thr</sup>, tRNA<sup>Ile</sup>, tRNA<sup>Gly</sup>, tRNA<sup>Gln</sup>, tRNA<sup>Glu</sup>, tRNA<sup>Asp</sup>, tRNA<sup>Lys</sup>, tRNA<sup>Arg</sup> and tRNA<sup>Leu</sup> but also phylum-specific combinations in tRNA<sup>Pro</sup>, tRNA<sup>Ala</sup>, and tRNA<sup>Trp</sup>. Our results support the idea that tRNA emerged through the combination of separate genes and explain the sequence diversity that arose during archaeal tRNA evolution.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Fujishima K, Sugahara J, Tomita M, Kanai A (2008) Sequence Evidence in the Archaeal Genomes that tRNAs Emerged Through the Combination of Ancestral Genes as 5′ and 3′ tRNA Halves. PLoS ONE 3(2):
          e1622.
            doi:10.1371/journal.pone.0001622</p><p><strong>Academic Editor: </strong>Mukund Thattai, Tata Institute of Fundamental Research, India</p><p><strong>Received:</strong> November 11, 2007; <strong>Accepted:</strong> January 22, 2008; <strong>Published:</strong> February 20, 2008</p><p><strong>Copyright:</strong> © 2008 Fujishima et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported in part by a Grant-in-Aid for Scientific Research on Priority Areas; a Grant-in-Aid for the 21st Century Center of Excellence (COE) Program, entitled ‘Understanding and Control of Life's Function via Systems Biology’ (Keio University); research funds from the Yamagata Prefectural Government and Tsuruoka City, Japan; and a grant from Keio University.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The origin and evolution of tRNA is an exciting topic widely discussed in the field of molecular evolution <a href="#pone.0001622-Maizels1">[1]</a>–<a href="#pone.0001622-DiGiulio2">[5]</a>. Three types of tRNA genes have been identified in archaeal genomes: “nonintronic tRNA”, which is encoded on a single gene with no intron sequence; “intron-containing tRNA” (intronic tRNA), which is encoded on a single gene with 1 to 3 introns punctuating the mature tRNA sequence at various locations <a href="#pone.0001622-Marck1">[6]</a>–<a href="#pone.0001622-Sugahara2">[8]</a>; and “split tRNA”, which has been found only in the hyperthermophilic archaeal parasite <em>Nanoarchaeum equitans</em>, of which the 5′ and 3′ halves are encoded on separate genes <a href="#pone.0001622-Randau1">[9]</a>. The discovery of these split tRNA genes raised the question of whether ancestral tRNA was encoded on a single gene or separate genes. No redundant tRNAs with the same anticodon have been found in the <em>N. equitans</em> genome, and experimental data suggest that these split tRNAs are transcribed and processed into functional tRNA molecules <a href="#pone.0001622-Randau1">[9]</a>. The joiner region of the pre-tRNA product of the 2 split tRNA genes consists of a bulge–helix–loop (BHL) motif, a relaxed form of bulge–helix–bulge (BHB) motif which is recognized and cleaved by homodimer or heterotetramer splicing endonucleases during tRNA processing <a href="#pone.0001622-TocchiniValentini1">[10]</a>, <a href="#pone.0001622-Randau2">[11]</a>. Interestingly, BHB and BHL motif is known as a distinctive structure for protein-spliced intron and are observed at the exon–intron boundary of archaeal intronic tRNA <a href="#pone.0001622-Marck2">[12]</a>, <a href="#pone.0001622-Diener1">[13]</a>, mRNA <a href="#pone.0001622-Yoshinari1">[14]</a>, <a href="#pone.0001622-Watanabe1">[15]</a>, rRNA <a href="#pone.0001622-Tang1">[16]</a> and eukaryal intronic tRNAs <a href="#pone.0001622-Li1">[17]</a>, <a href="#pone.0001622-Belfort1">[18]</a>. Recently, a novel permuted structure of tRNA gene was discovered in a genome of primitive red alga, <em>Cyanidioschyzon merolae</em>, which tRNAs are encoded on a single gene in the order of 3′ half–5′ half forming a potential BHB motif around intron-exon junction <a href="#pone.0001622-Soma1">[19]</a>. These common processing machineries suggest an evolutionary link among various types of tRNAs.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In this context, Di Giulio suggested that ancestral tRNA was encoded by 2 separate minigenes, which later fused to encode modern tRNAs <a href="#pone.0001622-DiGiulio1">[3]</a>, <a href="#pone.0001622-DiGiulio2">[5]</a>, <a href="#pone.0001622-DiGiulio3">[20]</a>. This conclusion is based on the probabilistic argument that the boundaries of the split tRNAs and of many of the introns found in the tRNA sequences are located at anticodon loops. In addition, it has been proposed that the tRNA molecule originated by direct duplication of an RNA hairpin structure <a href="#pone.0001622-DiGiulio1">[3]</a>, <a href="#pone.0001622-Nagaswamy1">[4]</a>, <a href="#pone.0001622-DiGiulio4">[21]</a>. Experimental studies have shown that a single hairpin minihelix that recapitulates the domain of top half tRNA sequence (acceptor stem+TψC arm helix) can be aminoacylated by modern tRNA synthetases <a href="#pone.0001622-Hipps1">[22]</a>. The definition of the minihelix is different from that of split tRNA (separated at the anticodon loop) but the idea of tRNA emerged from two distinct minigenes has been supported by the genomic tag hypothesis, which proposes that the top half tRNA sequence must have originated among ancient life to catalyze RNA replication in the ‘RNA world’ ahead of bottom half tRNA sequence (D arm helix+anticodon stem) on account of the independent structure and functionality of the 5′ and 3′ tRNA sequences <a href="#pone.0001622-Maizels1">[1]</a>, <a href="#pone.0001622-Weiner1">[2]</a>. Nevertheless, the phylogenetic position of <em>N. equitans</em> remains uncertain, as it has two incongruent positions in the phylogenetic tree of the Archaea depending on the dataset <a href="#pone.0001622-Waters1">[23]</a>, <a href="#pone.0001622-Brochier1">[24]</a>, and so the question remains as to whether the split tRNA genes really exemplify the ancestral form of tRNA.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We have recently developed software called SPLITS <a href="#pone.0001622-Sugahara1">[7]</a>, <a href="#pone.0001622-Sugahara2">[8]</a>, which has predicted most of the archaeal missing tRNAs that cannot be predicted by tRNAscan-SE, the most widely used software for tRNA annotation <a href="#pone.0001622-Lowe1">[25]</a>. The recent accumulation of complete archaeal genome sequences in the public databases enabled us to perform comprehensive phylogenetic analysis of 1953 archaeal tRNA sequences in 45 archaeal species. Here, we report on the overall phylogeny of the archaeal tRNAs, focusing on the evolutionary relationships among nonintronic, intronic, and split tRNAs. Further, by separating all kinds of tRNAs at the anticodon region and mimicking the split tRNA sequences, we show that network topologies based on the sequence similarities of 5′ and 3′ tRNA halves differ significantly. Moreover, the specific combination of 5′ and 3′ tRNA halves from different groups explained the variation of amino acids in the codon table. Here, we provide sequence evidence supporting the hypothesis that tRNA evolved through a combination of 5′ and 3′ tRNA sequences.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Phylogenetic analysis of mature tRNAs in 45 archaeal species</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We predicted 1977 putative tRNA candidates from the genome sequences of 45 archaeal species with 2 tRNA predicting programs, SPLITS <a href="#pone.0001622-Sugahara1">[7]</a>, <a href="#pone.0001622-Sugahara2">[8]</a> and tRNAscan-SE <a href="#pone.0001622-Lowe1">[25]</a>. All tRNA sequences were manually checked, and 24 false candidates (tRNA-like sequences used for viral integration, and pseudogenes) were eliminated from the dataset. The resulting 1953 archaeal tRNAs, including 6 known split tRNAs and 423 intronic tRNAs, were used as a dataset for phylogenetic analysis. We performed structural alignment based on their mature tRNA sequences (from which introns and leader sequences were deleted) by manually improving the multiple alignment data through complete matching of the consensus nucleotides conserved among archaeal tRNAs (see <a href="#s3">Methods</a> for detail). An unrooted neighbor-joining (NJ) tree was then produced. As a result, 1953 tRNAs were separated into 22 clusters: 12 dominated by tRNAs with its anticodon corresponding to a single type of amino acid (e.g., a tRNA for Ala), and 10 consisting of tRNAs with anticodons corresponding to 2 to 4 amino acids (e.g., a tRNA for Arg-Lys-Trp [i.e., 3 amino acids]) (<a href="#pone-0001622-g001">Fig. 1</a>). For example, 89 out of 90 archaeal tRNA<sup>Gln</sup> were clustered in the same branch (cluster 16); and all tRNA<sup>Asp</sup> and tRNA<sup>Glu</sup> except the split tRNA<sup>Glu</sup> were clustered indistinguishably in the same branch (cluster 17). Like as Asp-Glu cluster, there are several indistinguishable pairs of amino acids clustered in the same branch that are in precursor-product relationship or in biosynthetic relationship (ex: Ala-Val, Arg-Lys, Phe-Tyr, Cys-Ser) supporting the coevolution theory of the origin of the genetic code <a href="#pone.0001622-Wong1">[26]</a>.</p>
<div class="figure" id="pone-0001622-g001"><div class="img"><a name="pone-0001622-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001622" data-uri="info:doi/10.1371/journal.pone.0001622.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001622.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001622.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Phylogenetic tree of mature tRNA sequences in 45 archaeal species.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The phylogenetic neighbor-joining tree was constructed by using mature sequences of 1953 predicted tRNAs from the complete genomes of 45 archaeal species. Clusters are numbered from 1 to 22, and tRNAs within each cluster are denoted by amino acids corresponding to the anticodon. Clusters including tRNAs from all 3 archaeal phyla (Euryarchaeota, Crenarchaeota, and Nanoarchaeota) are shaded in blue. The amino acids corresponding to the split tRNAs are boxed.</p>
<span>doi:10.1371/journal.pone.0001622.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The split tRNAs were placed in 5 different clusters: 4 split tRNAs (tRNA<sup>iMet</sup> [cluster 1], tRNA<sup>His</sup> [cluster 6], tRNA<sup>Gln</sup> [cluster 16], and tRNA<sup>Lys</sup> [cluster 22]) were placed adjacent to other tRNAs with synonymous codons. Only the two split tRNA<sup>Glu</sup> (cluster 15) was distant from any other tRNAs that branched at the root of the tRNA<sup>Gln</sup> cluster. Accordingly, only 13 tRNA clusters (shaded in <a href="#pone-0001622-g001">Fig. 1</a>) included tRNAs from all 3 archaeal phyla (Nanoarchaeota, Crenarchaeota, and Euryarchaeota), suggesting that about half of the tRNAs corresponding to specific amino acids evolved from a monophyletic origin. For a clearer understanding of the phylogenetic distribution of archaeal tRNAs, we tabulated the phylogenetic location of tRNAs from 16 archaeal genera against each tRNA's cluster number (Supplemental <a href="#pone.0001622.s001">Fig. S1</a>). Further, we have found that none of the <em>N. equitans</em> tRNAs branched adjacent to the tRNAs of the host species <em>Ignicoccus hospitalis</em> (data not shown), thus we assume that no tRNA genes were laterally transferred between the 2 species after <em>N. equitans</em> became parasite of <em>Ignicoccus</em> cell.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Since 4 out of the 6 split tRNAs possessed clear sequence similarity with other tRNAs with the same anticodons, we focused on the precise phylogeny of the 6 split tRNAs from <em>N. equitans</em> in relation to other archaeal tRNAs. We performed detailed phylogenetic analysis for each of the 6 split tRNAs based on the Bayesian method (Supplemental <a href="#pone.0001622.s002">Fig. S2</a>). Mature sequences of 3 split tRNAs (tRNA<sup>iMet</sup>, tRNA<sup>Lys</sup> and tRNA<sup>Gln</sup>) branched with other tRNAs with same anticodon from Crenarchaeota and Euryarchaeota lineages. Split tRNA<sup>His</sup> clustered with tRNA<sup>His</sup> from the Crenarchaeota lineage and <em>M. kandleri</em>. The most notable was split tRNA<sup>Glu</sup>, which located at the root of tRNA<sup>Glu</sup> cluster. The phylogenetic position of split tRNA<sup>Glu</sup> in NJ tree was adjacent to tRNA<sup>Gln</sup> and tRNA<sup>Trp</sup> cluster, although this contradiction exemplifies the sequence ambiguity of split tRNA<sup>Glu</sup>. Thus, split tRNAs reveal various characteristics in the phylogeny of archaeal tRNAs—universal (tRNA<sup>iMet</sup>, tRNA<sup>Lys</sup> and tRNA<sup>Gln</sup>), crenarchaeal-specific (tRNA<sup>His</sup>), and unique (tRNA<sup>Glu</sup>) phylogenetic positions—suggesting that split tRNA could be the ancestral form of tRNAs. Besides, intronic tRNAs were scattered throughout tRNA phylogeny in almost every tRNA clusters with introns positioned at the same location as the 5′–3′ boundary of the split tRNAs. We found an intronic tRNA<sup>Arg</sup> with an intron sequence possessing 58% identity to that of the split tRNA<sup>Lys</sup> leader sequence located at the same position as that of the intron position (<a href="#pone-0001622-g002">Fig. 2</a>). Both tRNA belongs to the same cluster (cluster 22 in <a href="#pone-0001622-g001">Fig. 1</a>) suggesting that some intronic tRNAs may have emerged from integrated split tRNA in the archaeal genome.</p>
<div class="figure" id="pone-0001622-g002"><div class="img"><a name="pone-0001622-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001622" data-uri="info:doi/10.1371/journal.pone.0001622.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001622.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001622.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Comparison of the 3 types of tRNA sequences.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>(A) Full nucleotide sequences of pre-tRNAs (1 split tRNA<sup>Lys</sup> [Neq] and 2 intronic tRNA<sup>Arg</sup>s [Pae]) and 2 nonintronic tRNA<sup>Lys</sup> (Pfu and Afu) were aligned. Black bar marks the intron of the intronic tRNAs and the leader sequences of the split tRNAs, which are inserted at tRNA nucleotide position 32/33. Red bar marks the anticodon. (B) Comparison of the secondary structures and nucleotide sequences around the exon–intron boundary of the 3 types of tRNAs. Nucleotides that are identical between leader sequence and intron are shown in red. Red bar marks the anticodon.</p>
<span>doi:10.1371/journal.pone.0001622.g002</span></div>

<h4>Sequence analysis of the 5′ and 3′ tRNA halves reveals their different evolutionary backgrounds</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>As described in the previous section, comprehensive phylogenetic analysis of 1953 mature archaeal tRNA sequences revealed an evolutionary relationship between the three types of tRNAs. If split genes were the ancestral form of tRNA, can we explain the origin and evolution of modern tRNA sequences by the combination patterns of 5′ and 3′ tRNA halves? We attempted to corroborate this hypothesis by analyzing the evolutionary backgrounds of 5′ and 3′ halves. We selected 7 archaeal species, each from the closest taxon to the last archaeal common ancestor: 1 Nanoarchaeota (<em>N. equitans</em>, Neq), 3 Crenarchaeota (<em>P. aerophilum</em>, Pae; <em>A. pernix</em>, Ape; and <em>S. solfataricus</em>; Sso), and 3 Euryarchaeota (<em>P. furiosus</em>, Pfu; <em>M. kandleri</em>, Mka; and <em>M. jannaschii</em>, Mja). In total, 296 tRNA sequences were extracted and separated into the 5′ half (positions 1–33) and the 3′ half (positions 37–73) at the anticodon region (positions 34–36) to mimic split tRNA. We considered each tRNA half as a network node, and nodes were linked when sequence identity between the 2 tRNA halves was above certain threshold. Accordingly, we have constructed 2 different networks (5′ half network and 3′ half network) based on their sequence similarities. Previously, a network-based approach was used to explain 2 mechanisms of tRNA evolution: point mutation and complementary mutation <a href="#pone.0001622-Wei1">[27]</a>. Thus, concept of representing the sequence similarities among tRNA halves as a network provides comprehensive understanding of sequence characteristics and its diversity based on the global statistics. <a href="#pone-0001622-g003">Figure. 3</a> shows the sequence similarity networks of 5′ and 3′ tRNA halves with thresholds of &gt;70%, &gt;75%, and &gt;80% for 5′ tRNA halves (<a href="#pone-0001622-g003">Fig. 3A</a>), and &gt;80%, &gt;85%, and &gt;88% for 3′ tRNA halves (<a href="#pone-0001622-g003">Fig. 3B</a>). A noticeable topological difference is apparent between the sequence similarity networks of the 5′ and 3′ halves. Sequence diversity of the 5′ half sequences appeared between sequence similarities of &gt;70% and &gt;75%, where the single large network started to localize into small clusters corresponding to specific amino acids. This feature was more prominent at a similarity of &gt;80%, where 5′ halves clearly separated into 6 groups (1–6 in <a href="#pone-0001622-g003">Fig. 3A</a>). In contrast, all 3′ half sequences except tRNA<sup>Ser</sup> and tRNA<sup>Leu</sup> (possessing long variable stem loops shown as yellow and orange nodes in <a href="#pone-0001622-g003">Fig. 3B</a>) gathered into 1 large group even at a sequence similarity of &gt;80%. The 3′ half sequences were finally differentiated at a sequence similarity of &gt;88% into 9 groups (A–I in <a href="#pone-0001622-g003">Fig. 3B</a>).</p>
<div class="figure" id="pone-0001622-g003"><div class="img"><a name="pone-0001622-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001622" data-uri="info:doi/10.1371/journal.pone.0001622.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001622.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001622.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Network analysis based on the sequence similarities of 5′ and 3′ tRNA halves.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A total of 296 mature tRNA sequences from 7 archaeal species (Neq, Sso, Ape, Pae, Mka, Pfu, Mja) were artificially split into 5′ and 3′ halves at the anti-codon region. Each node (colored dot) represents a tRNA half, and its color indicates the charged amino acid's chemical properties (DE, mid-green; MNQ, light green; RKH, blue; FWY, purple; AGP, red; ILV, orange; CST, yellow; iMet, gray). Nodes are linked by a white line (edge) when the sequence similarity is above the threshold. (A) Network created by set of 5′ half sequences with thresholds of &gt;70%, &gt;75%, and &gt;80%. The sequences are classified into 6 clusters (1–6) at a threshold of &gt;80%. (B) Network created by set of 3′ half sequences with thresholds of &gt;80%, &gt;85%, and &gt;88%. The sequences are classified into 9 clusters (A–I) at a threshold of &gt;88%.</p>
<span>doi:10.1371/journal.pone.0001622.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To characterize the overall network topology of the 5′ and 3′ halves, we compared the 2 networks by measuring the distribution of the number of connections per node (connectivity distribution). Network topology can be classified into 2 distinct types: ‘scale-free networks’, in which a few nodes have many links but most have only a few links, and which follow a power-law distribution <a href="#pone.0001622-Barabasi1">[28]</a>; and ‘small-world networks’, which consist of nodes linked randomly and follow a Poisson or exponential distribution <a href="#pone.0001622-Watts1">[29]</a>. We found obvious difference in degree distribution between the 2 networks at all similarity thresholds (<a href="#pone-0001622-g004">Fig. 4</a>). At a similarity threshold of &gt;75%, the clustering coefficient <em>c</em> (degree of linkage between nodes) of 5′ half was only 0.17 while 3′ half was 0.47, which means that 47% of all possible connection are used. The power-law-like distribution of the 5′ half network was enhanced as the similarity threshold increased, while the distribution of 3′ half network changed from Poisson-like (&gt;75%) to irregular (&gt;80–85%) then finally a power-law-like distribution at a sequence similarity of &gt;88%. Meanwhile, the ratio of the cluster coefficient between 5′ and 3′ halves were constantly 1:2.5–1:4 showing that characteristics of the 2 tRNA domains significantly differ at all level of sequence threshold (75%–88%). We suggest that high commonality of the 3′ half sequences is one of the evidence of which 3′ half could have had a single origin. On the other hand, the scale-free distribution of the 5′ half represents a specific sequence characteristic within each sequence group, suggesting a non-monophyletic origin of the 5′ half sequences. These different sequence characteristics of the 5′ and 3′ tRNA halves further support the idea that ancient archaeal tRNA emerged from a specific combination of 5′ and 3′ halves.</p>
<div class="figure" id="pone-0001622-g004"><div class="img"><a name="pone-0001622-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001622" data-uri="info:doi/10.1371/journal.pone.0001622.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001622.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001622.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Distribution of networks based on the 5′ and 3′ tRNA sequences.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Relation between the number of tRNA sequences and the number of links is represented at 4 sequence similarity thresholds (A–D). Clustering coefficient <em>c</em> is denoted for each tRNA halves.</p>
<span>doi:10.1371/journal.pone.0001622.g004</span></div>

<h4>Specific combinations of 5′ with 3′ tRNA halves strongly correlate with amino acid distribution in the codon table</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Since the 5′ and 3′ halves are suggested to have had different evolutionary backgrounds, we hypothesized that the sequence variety of tRNAs was derived directly from different combinations of 5′ and 3′ tRNA sequences in the early stage of archaeal evolution. To examine the correlation between specific combinations of 5′ and 3′ halves and the corresponding amino acids in the codon table, we filled the codon table of 7 archaeal species by annotating each of the anticodons with the 5′ and 3′ group IDs denoted in <a href="#pone-0001622-g003">Figure. 3</a>. We have selected different sequence similarity thresholds (80% for 5′ half and 88% for 3′ half) for grouping the tRNA sequences, since the two networks possessed similar clustering coefficient of <em>c</em> = 0.08 and 0.05. <a href="#pone-0001622-g005">Figure. 5</a> shows a codon table filled by the 5′ and 3′ group IDs of <em>N. equitans</em> tRNA sequences. Codon tables of the other 6 species are shown in Supplemental <a href="#pone.0001622.s003">Figure S3</a>. Group A is the largest group in the 3′ half network, accounting for 104/296 (35%) sequences of tRNAs corresponding to various amino acids. In <em>N. equitans</em>, 3′ half sequences of tRNA<sup>Arg</sup>, tRNA<sup>Ile</sup>, tRNA<sup>Val</sup>, tRNA<sup>Ala</sup>, tRNA<sup>Pro</sup>, tRNA<sup>Trp</sup>, and tRNA<sup>Phe</sup> were all categorized as group A. Adding the 5′ sequence information further classified these tRNAs into small groups possessing the same 5′–3′ combination. For example, in all 7 species, the same 4–A combination was observed for tRNA<sup>Arg</sup> and tRNA<sup>Lys</sup>; this, together with the fact that split tRNA<sup>Lys</sup> is clustered among the Arg-Lys cluster (<a href="#pone-0001622-g001">Fig. 1</a>), suggests that these tRNAs originated from a combination of ancestral minigenes of the 2 sequence groups. The same features are apparent in the 5–A combination of tRNA<sup>Val</sup>. Exceptionally, however, 2 tRNA<sup>Val</sup>s in <em>M. kandleri</em> both have the 4–A combination located in unique cluster (cluster 11 in <a href="#pone-0001622-g001">Fig. 1</a>). <em>M. kandleri</em> is an intriguing species which possesses many unique tRNAs clustered at different phylogenetic positions from other tRNAs with the same anticodon (Supplemental <a href="#pone.0001622.s001">Fig. 1</a>). The <em>N. equitans</em> genome also encodes unique tRNA<sup>Ile</sup>, which displays the only UAU (TAT) anticodon found in the Archaea. tRNA<sup>Ile</sup> (Neq) is considered to be evolutionarily unique, as no homologous sequence has been found in any other archaeal species and therefore it is located within an isolated branch with various tRNAs from <em>M. kandleri</em> (cluster 11 in <a href="#pone-0001622-g001">Fig. 1</a>). However, by splitting the sequence of tRNA<sup>Ile</sup> (Neq), we found tRNA<sup>Gly</sup>(GCC) in <em>N. equitans</em> genome which possessed identical 3′ half but different 5′half (<a href="#pone-0001622-g006">Fig. 6A</a>). This sequence evidence supports our idea of the ancient integration of split tRNA genes.</p>
<div class="figure" id="pone-0001622-g005"><div class="img"><a name="pone-0001622-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001622" data-uri="info:doi/10.1371/journal.pone.0001622.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001622.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001622.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Representation of <em>N. equitans</em> codon table filled with 5′ and 3′ tRNA halves.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The table is filled with the group IDs (see <a href="#pone-0001622-g003">Fig. 3</a>) corresponding to each of the 5′ and 3′ tRNA halves in <em>N. equitans</em> (Neq). The anti-codon corresponding to the 6 split tRNAs is shown in red. An asterisk indicates that a sequence does not have a similar sequence above the threshold (5′ half, 80%; 3′ half, 88%).</p>
<span>doi:10.1371/journal.pone.0001622.g005</span></div><div class="figure" id="pone-0001622-g006"><div class="img"><a name="pone-0001622-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001622" data-uri="info:doi/10.1371/journal.pone.0001622.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001622.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001622.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001622.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001622.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Examples of tRNA sequences explained by specific combinations of 5′ and 3′ tRNA halves.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>(A) Examples of tRNA sequences with different 5′ half sequences but a common 3′ half sequence. (B) Example of tRNA sequences with different 3′ half sequences but a common 5′ half sequence. Asterisk denotes matching nucleotides in the aligned tRNA sequences. Boxes delineate sequences belonging to the same group in the sequence similarity network (<a href="#pone-0001622-g003">Fig. 3</a>). The corresponding group ID is labeled in a black box.</p>
<span>doi:10.1371/journal.pone.0001622.g006</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Comparing the codon tables of all 3 archaeal species groups (Supplemental <a href="#pone.0001622.s003">Fig. S3</a>), we found several interesting features related to the evolutionary selection of 5′–3′ combinations. We identified a phylum-specific distribution of tRNA halves corresponding to the same amino acids. For example, the 5′ tRNA sequences of euryarchaeal tRNA<sup>Pro</sup> belong to group 6, but those of the nanoarchaeal and crenarchaeal tRNA<sup>Pro</sup>s belong to group 5. In contrast, the 3′ sequences of tRNA<sup>Pro</sup> all belong to group C except <em>N. equitans</em>. tRNA<sup>Trp</sup> and tRNA<sup>Ala</sup> also showed phylum-specific 5′ half sequences. This different ancestry of the 5′ half sequences clearly explains the non-monophyletic origins of tRNA<sup>Pro</sup>, tRNA<sup>Trp</sup> and tRNA<sup>Ala</sup>. Secondly, specific 5′–3′ combinations of tRNA<sup>iMet</sup>, tRNA<sup>Thr</sup>, tRNA<sup>Ile</sup>, tRNA<sup>Gly</sup>, tRNA<sup>Gln</sup>, tRNA<sup>Glu</sup>, tRNA<sup>Asp</sup>, tRNA<sup>Lys</sup>, tRNA<sup>Arg</sup> and tRNA<sup>Leu</sup> are conserved among all 7 species, placing these tRNAs among the most ancient unchanged tRNAs in the genome since the last archaeal common ancestor. Finally we observed a clear rule in the sequence divergence of 5′ and 3′ halves. For example in 5′ half network, group 1 has connection with group 3 but not with group 5 nor group 6 and in 3′ half network, group D has connection with group C and group B but not with group A (<a href="#pone-0001622-g003">Fig. 3</a>). Based on these rules, we found that group 5 and group C were placed at the center of network diversity, suggesting that these cluster should have been the origin of the whole network. When conbining the two (5′ and 3′) rules together, tRNA<sup>Gly</sup> appeared as the only 5–C combination conserved among 7 species. Thus we suggest that minigenes encoding 5′ and 3′ tRNA sequence of tRNA<sup>Gly</sup> were the origins of other tRNA genes in the very early stage of tRNA evolution. Later, gene duplication and sequence divergence occurred and selection of specific 5′–3′ tRNA sequences would have increased the specificity of tRNAs along with the usage of various amino acids. As shown in <a href="#pone-0001622-g006">Figure 6</a>, many tRNAs share identical 3′ halves but have different 5′ halves (<a href="#pone-0001622-g006">Fig. 6A–6C</a>), or vice versa (<a href="#pone-0001622-g006">Fig. 6D</a>). In that circumstance, the anticodon sequence could have been supplied by either sequences, a feature observed in split tRNAs <a href="#pone.0001622-Randau3">[30]</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In conclusion, our hypothesis that modern tRNAs evolved through the combination of 5′ and 3′ tRNA fragments is supported strongly by the variation in amino acids in the codon table and explains the sequence diversity in archaeal tRNAs. Some tRNAs show a highly conserved 3′ half but different 5′ half sequences (or vice versa), while other tRNAs consist of specific 5′–3′ combinations and correspond to single types of amino acids, features that are conserved throughout all archaeal phyla. We need natural or experimental evidence of how these split genes evolved into modern tRNA genes. According to current studies of processed pseudogenes, at least some rRNAs, mRNAs, and tRNAs can retropose (reverse-transcribe and integrate) into the genome in Mammalia and plants <a href="#pone.0001622-Wang1">[31]</a>–<a href="#pone.0001622-Kacar1">[33]</a>. A good example is known as short interspread nuclear element (SINE); a retrotransposon which is known to be originated from tRNA via retropositional mechanism <a href="#pone.0001622-Ohshima1">[34]</a>, <a href="#pone.0001622-Schmitz1">[35]</a>. Experimentally, <em>Escherichia coli</em>–derived tRNA<sup>Tyr</sup> was reverse-transcribed as mono-cDNA by using retroplasmid-derived reverse transcriptase without any template DNA <a href="#pone.0001622-Chiang1">[36]</a>. Therefore, we speculate that ancestral 5′ and 3′ tRNA minigenes integrated at the transcript level, along with the help of reverse transcriptase or integrase. If so, both intronic and nonintronic tRNAs could have been produced from transcribed split tRNA, resulting in the integration of the leader sequence of split tRNA as an intron sequence containing the BHB motif, and the deposition of the CCA sequence added by nucleotidyltransferase downstream of the 3′ end of retroposed tRNA sequences. This hypothesis could also be adapted to mRNAs and rRNAs, because some of these functional RNAs in the <em>N. equitans</em> genome are encoded on separate genes just like split tRNA <a href="#pone.0001622-Waters1">[23]</a>. Recently we have reported a novel type of functional tRNA gene in a primitive red alga, <em>Cyanidioschyzon merolae</em>, in which the 3′ half of the tRNA sequence lies upstream of the 5′ half in the genome <a href="#pone.0001622-Soma1">[19]</a>. This ‘permuted’ tRNA might also be considered as a result of the adaptive integration of 5′ and 3′ tRNA halves of ancient eukaryotes or a retroposed product of circular pre-tRNA transcript. Consequently, we have provided a molecular basis for our current understanding of the origin and evolution of tRNAs. Novel tRNA sequences from the upcoming fourth archaeal phylum, the Korarchaeota <a href="#pone.0001622-Auchtung1">[37]</a>, and other primitive Eukarya and Prokarya separated near the root of the phylogenetic tree will allow further testing of our hypothesis.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Preparation of tRNA sequences</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>tRNA sequences were predicted from the genome sequences of the following 45 archaeal species: <strong>Crenarchaeota (15 species):</strong> <em>Aeropyrum pernix</em> K1 (Ape), <em>Caldivirga maquilingensis</em> IC-167 (Cma), <em>Cenarchaeum symbiosum</em> (Csy), <em>Hyperthermus butylicus</em> DSM 5456 (Hbu), <em>Ignicoccus hospitalis</em> KIN4/I (Ign), <em>Metallosphaera sedula</em> DSM 5348 (Mse), <em>Pyrobaculum aerophilum</em> str. IM2 (Pae), <em>Pyrobaculum arsenaticum</em> DSM 13514 (Par), <em>Pyrobaculum calidifontis</em> JCM 11548 (Pca), <em>Pyrobaculum islandicum</em> DSM 4184 (Pis), <em>Staphylothermus marinus</em> F1 (Sma), <em>Sulfolobus acidocaldarius</em> DSM 639 (Sac), <em>Sulfolobus solfataricus</em> P2 (Sso), <em>Sulfolobus tokodaii</em> str. 7 (Sto), <em>Thermofilum pendens</em> Hrk 5 (Tpe). <strong>Euryarchaeota (29 species):</strong> <em>Archaeoglobus fulgidus</em> DSM 4304 (Afu), <em>Candidatus</em> Methanoregula boonei 6A8 (Cme), <em>Haloarcula marismortui</em> ATCC 43049 (HmaI [chromosome I], HmaII [chromosome II]), <em>Halobacterium</em> sp. NRC-1 (Hsa), <em>Haloquadratum walsbyi</em> DSM 16790 (Hwa), <em>Methanocaldococcus jannaschii</em> DSM 2661 (Mja), <em>Methanococcoides burtonii</em> DSM 6242 (Mbu), <em>Methanococcus maripaludis</em> C5 (MmaC5), <em>Methanococcus maripaludis</em> S2 (MmaS2), <em>Methanococcus vannielii</em> SB (Mva), <em>Methanocorpusculum labreanum</em> Z (Mla), <em>Methanoculleus marisnigri</em> JR1 (Mcu), <em>Methanopyrus kandleri</em> AV19 (Mka), <em>Methanosaeta thermophila</em> PT (Msa), <em>Methanosarcina acetivorans</em> C2A (Mac), <em>Methanosarcina barkeri</em> str. Fusaro (Mba), <em>Methanosarcina mazei</em> Go1 (Mma), <em>Methanosphaera stadtmanae</em> DSM 3091 (Mst), <em>Methanospirillum hungatei</em> JF-1 (Mhu), <em>Methanothermobacter thermautotrophicus</em> str. Delta H (Mth), <em>Natronomonas pharaonis</em> DSM 2160 (Nph), <em>Picrophilus torridus</em> DSM 9790 (Pto), <em>Pyrococcus abyssi</em> GE5 (Pab), <em>Pyrococcus furiosus</em> DSM 3638 (Pfu), <em>Pyrococcus horikoshii</em> OT3 (Pho), <em>Thermococcus kodakarensis</em> KOD1 (Tko), <em>Thermoplasma acidophilum</em> DSM 1728 (Tac), <em>Thermoplasma volcanium</em> GSS1 (Tvo), uncultured methanogenic archaeon RC-I (Mrc). <strong>Nanoarchaeota (1 species):</strong> <em>Nanoarchaeum equitans</em> Kin4-M (Neq).</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>All genomic information was downloaded from the Microbial Genomes resource of the National Center for Biotechnology Information (NCBI) (<a href="http://www.ncbi.nlm.nih.gov/genomes/MICROBES/microbial_taxtree.html">http://www.ncbi.nlm.nih.gov/genomes/MICR​OBES/microbial_taxtree.html</a>), except for the raw sequence data of the <em>Caldivirga maquilingensis</em> IC-167 draft genome, which was downloaded from the Department of Energy's Joint Genome Institute (<a href="http://genome.jgi-psf.org/draft_microbes/calma/calma.home.html">http://genome.jgi-psf.org/draft_microbes​/calma/calma.home.html</a>).</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Both nonintronic and intronic tRNAs were predicted by 2 tRNA-predicting programs, SPLITS (7, 8) and tRNAscan-SE (17). tRNAscan-SE was run with default parameters for predicting nonintronic tRNAs and intronic tRNAs with canonical introns. SPLITS was run parameters -d 2, -p 0.51, -H 2, and –F 3 for predicting intronic tRNAs with multiple introns. The split tRNA sequences were taken from Randau <a href="#pone.0001622-Randau3">[30]</a>. All predicted tRNA sequences were manually checked to eliminate false candidates and to correct the intron sequences. All predicted archaeal tRNA sequences are registered in a new tRNA database (Sugahara and Kikuta <em>et al.</em>, to be published separately).</p>


<h4>Accession number of Archaeal genomes</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>[Species (GenBank ID)]: Aeropyrum pernix K1 (BA000002.3), Cenarchaeum symbiosum (DP000238), Hyperthermus butylicus DSM 5456 (CP000493.1), Ignicoccus hospitalis KIN4/I (CP000816.1), Metallosphaera sedula DSM 5348 (CP000682.1), Pyrobaculum aerophilum str. IM2 (AE009441.1), Pyrobaculum arsenaticum DSM 13514 (CP000660.1), Pyrobaculum calidifontis JCM 11548 (CP000561.1), Pyrobaculum islandicum DSM 4184 (CP000504.1), Staphylothermus marinus F1 (CP000575.1), Sulfolobus acidocaldarius DSM 639 (CP000077.1), Sulfolobus solfataricus P2 (AE006641.1), Sulfolobus tokodaii str. 7 (BA000023.2), Thermofilum pendens Hrk 5 (CP000505.1), Archaeoglobus fulgidus DSM 4304 (AE000782.1), Candidatus Methanoregula boonei 6A8 (CP000780.1), Haloarcula marismortui ATCC 43049 (AY596297.1), Halobacterium sp. NRC-1 (AE004437.1), Haloquadratum walsbyi DSM 16790 (AM180088.1), Methanocaldococcus jannaschii DSM 2661 (L77117.1), Methanococcoides burtonii DSM 6242 (CP000300.1), Methanococcus aeolicus Nankai-3 (CP000743.1), Methanococcus maripaludis C5 (CP000609.1), Methanococcus maripaludis C7 (CP000745.1), Methanococcus maripaludis S2 (BX950229.1), Methanococcus vannielii SB (CP000742.1), Methanocorpusculum labreanum Z (CP000559.1), Methanoculleus marisnigri JR1 (CP000562.1), Methanopyrus kandleri AV19 (AE009439.1), Methanosaeta thermophila PT (CP000477.1), Methanosarcina acetivorans C2A (AE010299.1), Methanosarcina barkeri str. Fusaro (CP000099.1), Methanosarcina mazei Go1 (AE008384.1), Methanosphaera stadtmanae DSM 3091 (CP000102.1), Methanospirillum hungatei JF-1 (CP000254.1), Methanothermobacter thermautotrophicus str. Delta H (AE000666.1), Natronomonas pharaonis DSM 2160 (CR936257.1), Picrophilus torridus DSM 9790 (AE017261.1), Pyrococcus abyssi GE5 (AL096836.1), Pyrococcus furiosus DSM 3638 (AE009950.1), Pyrococcus horikoshii OT3 (BA000001.2), Thermococcus kodakarensis KOD1 (AP006878.1), Thermoplasma acidophilum DSM 1728 (AL139299.1), Thermoplasma volcanium GSS1 (BA000011.4), uncultured methanogenic archaeon RC-I (AM114193.2), Nanoarchaeum equitans Kin4-M (AE017199.1).</p>


<h4>tRNA sequence analysis</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>All mature tRNA sequences (input as tDNA sequences) were aligned using CLUSTALW <a href="#pone.0001622-Thompson1">[38]</a> with the following parameters: Multiple Alignment parameter “Gap Opening” = 22.5; Multiple Alignment parameter “Gap Extension Penalty” = 0.83; Pairwise Alignment parameter “Gap Opening” = 22.5; Multiple Alignment parameter “Delay Divergent Sequence” = 25%. These parameters gave the highest performance in RNA sequence alignment <a href="#pone.0001622-Wilm1">[39]</a>. Aligned tRNA sequences (.aln file) were manually improved by matching the consensus nucleotides conserved among archaeal tRNAs: base 8 (T or C), base 14-15 (AG), base 18-19 (GG), base 21 (A or G), base 33 (T), base 48 (T or C) base 53-58 (GTTC[AG]A), base 60-61 (TC) and base 72 (T or C) <a href="#pone.0001622-Marck1">[6]</a> to obtain a structural alignment. The neighbor joining tree was constructed using CLUSTALW. Bayesian trees were constructed using MrBayes software v. 3.1.2 <a href="#pone.0001622-Huelsenbeck1">[40]</a>. The model of sequence evolution was determined by MrModeltest software v. 2.2. The phylogenetic tree files were visualized using Hypertree <a href="#pone.0001622-Bingham1">[41]</a> and Treeview X software (<a href="http://darwin.zoology.gla.ac.uk/rpage/treeviewx/">http://darwin.zoology.gla.ac.uk/rpage/tr​eeviewx/</a>). The mimicry of split tRNA halves were conducted by separating tRNA sequences at positions 34–36 (anti-codon); we defined the 2 separated sequences as 5′ half tRNA (positions 1–33) and 3′ half tRNA (positions 37–73). Sequence similarity was calculated based on the aligned file of mature tRNA sequences (.aln). The anti-codon and CCA sequence was excluded to avoid bias. Networks were visualized with Cytoscape software v. 2.5 <a href="#pone.0001622-Shannon1">[42]</a>. Clustering coefficient <em>c</em> is calculated as given equation:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001622.e001&amp;representation=PNG" class="inline-graphic"></span><br>where <em>i</em>(<em>i</em>-1) refers to the total number of possible connection among <em>i</em> nodes, <em>j</em> refers to the total number of edge found in the network. Clustering coefficient measures relative linkage among nodes and it is used to understand the topological characteristics of a network (18, 20).</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001622"></div><a name="pone.0001622.s001" id="pone.0001622.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001622.s001">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001622.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001622.s002" id="pone.0001622.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001622.s002">Figure S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001622.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.18 MB DOC)</p>
<a name="pone.0001622.s003" id="pone.0001622.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001622.s003">Figure S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001622.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.76 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Motomu Matsui, Mikiko Hattori, and Kaoru Kikuta (Keio University, Japan) for their helpful discussions. We also thank the Department of Energy's Joint Genome Institute (<a href="http://www.jggi.doe.gov/">http://www.jggi.doe.gov/</a>) for permission to use the raw sequence data of <em>Caldivirga maquilingensis</em> IC-167.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AK. Performed the experiments: KF. Analyzed the data: MT AK KF JS. Contributed reagents/materials/analysis tools: JS. Wrote the paper: AK KF.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001622-Maizels1" id="pone.0001622-Maizels1"></a>Maizels N, Weiner AM (1994) Phylogeny from function: evidence from the molecular fossil record that tRNA originated in replication, not translation. Proc Natl Acad Sci U S A  91: 6729–6734.  <ul class="find" data-citedArticleID="1058274" data-doi="10.1073/pnas.91.15.6729"><li><a href="http://dx.doi.org/10.1073/pnas.91.15.6729" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+from+function%3A+evidence+from+the+molecular+fossil+record+that+tRNA+originated+in+replication%2C+not+translation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+from+function%3A+evidence+from+the+molecular+fossil+record+that+tRNA+originated+in+replication%2C+not+translation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001622-Weiner1" id="pone.0001622-Weiner1"></a>Weiner AM, Maizels N (1999) The genomic tag hypothesis: modern viruses as molecular fossils of ancient strategies for genomic replication, and clues regarding the origin of protein synthesis. Biol Bull  196: 327–330.  discussion 329-30.  <ul class="find" data-citedArticleID="1058318" data-doi="10.2307/1542962"><li><a href="http://dx.doi.org/10.2307/1542962" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genomic+tag+hypothesis%3A+modern+viruses+as+molecular+fossils+of+ancient+strategies+for+genomic+replication%2C+and+clues+regarding+the+origin+of+protein+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genomic+tag+hypothesis%3A+modern+viruses+as+molecular+fossils+of+ancient+strategies+for+genomic+replication%2C+and+clues+regarding+the+origin+of+protein+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001622-DiGiulio1" id="pone.0001622-DiGiulio1"></a>Di Giulio M (2006) The non-monophyletic origin of the tRNA molecule and the origin of genes only after the evolutionary stage of the last universal common ancestor (LUCA). J Theor Biol  240: 343–352.  <ul class="find" data-citedArticleID="1058254" data-doi="10.1016/j.jtbi.2005.09.023"><li><a href="http://dx.doi.org/10.1016/j.jtbi.2005.09.023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+non-monophyletic+origin+of+the+tRNA+molecule+and+the+origin+of+genes+only+after+the+evolutionary+stage+of+the+last+universal+common+ancestor+%28LUCA%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+non-monophyletic+origin+of+the+tRNA+molecule+and+the+origin+of+genes+only+after+the+evolutionary+stage+of+the+last+universal+common+ancestor+%28LUCA%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001622-Nagaswamy1" id="pone.0001622-Nagaswamy1"></a>Nagaswamy U, Fox GE (2003) RNA ligation and the origin of tRNA. Orig Life Evol Biosph  33: 199–209.  <ul class="find" data-citedArticleID="1058280"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=RNA+ligation+and+the+origin+of+tRNA.&amp;auth=&amp;atitle=RNA+ligation+and+the+origin+of+tRNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+ligation+and+the+origin+of+tRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+ligation+and+the+origin+of+tRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001622-DiGiulio2" id="pone.0001622-DiGiulio2"></a>Di Giulio M (1999) The non-monophyletic origin of the tRNA molecule. J Theor Biol  197: 403–414.  <ul class="find" data-citedArticleID="1058256" data-doi="10.1006/jtbi.1998.0882"><li><a href="http://dx.doi.org/10.1006/jtbi.1998.0882" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+non-monophyletic+origin+of+the+tRNA+molecule." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+non-monophyletic+origin+of+the+tRNA+molecule.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001622-Marck1" id="pone.0001622-Marck1"></a>Marck C, Grosjean H (2002) tRNomics: analysis of tRNA genes from 50 genomes of Eukarya, Archaea, and Bacteria reveals anticodon-sparing strategies and domain-specific features. Rna  8: 1189–1232.  <ul class="find" data-citedArticleID="1058276" data-doi="10.1017/s1355838202022021"><li><a href="http://dx.doi.org/10.1017/s1355838202022021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNomics%3A+analysis+of+tRNA+genes+from+50+genomes+of+Eukarya%2C+Archaea%2C+and+Bacteria+reveals+anticodon-sparing+strategies+and+domain-specific+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNomics%3A+analysis+of+tRNA+genes+from+50+genomes+of+Eukarya%2C+Archaea%2C+and+Bacteria+reveals+anticodon-sparing+strategies+and+domain-specific+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001622-Sugahara1" id="pone.0001622-Sugahara1"></a>Sugahara J, Yachie N, Sekine Y, Soma A, Matsui M, et al.  (2006) SPLITS: a new program for predicting split and intron-containing tRNA genes at the genome level. In Silico Biol  6: 411–418.  <ul class="find" data-citedArticleID="1058298"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=SPLITS%3A+a+new+program+for+predicting+split+and+intron-containing+tRNA+genes+at+the+genome+level.&amp;auth=&amp;atitle=SPLITS%3A+a+new+program+for+predicting+split+and+intron-containing+tRNA+genes+at+the+genome+level." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SPLITS%3A+a+new+program+for+predicting+split+and+intron-containing+tRNA+genes+at+the+genome+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SPLITS%3A+a+new+program+for+predicting+split+and+intron-containing+tRNA+genes+at+the+genome+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001622-Sugahara2" id="pone.0001622-Sugahara2"></a>Sugahara J, Yachie N, Arakawa K, Tomita M (2007) In silico screening of archaeal tRNA-encoding genes having multiple introns with bulge-helix-bulge splicing motifs. Rna  13: 671–681.  <ul class="find" data-citedArticleID="1058300" data-doi="10.1261/rna.309507"><li><a href="http://dx.doi.org/10.1261/rna.309507" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+silico+screening+of+archaeal+tRNA-encoding+genes+having+multiple+introns+with+bulge-helix-bulge+splicing+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+silico+screening+of+archaeal+tRNA-encoding+genes+having+multiple+introns+with+bulge-helix-bulge+splicing+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001622-Randau1" id="pone.0001622-Randau1"></a>Randau L, Munch R, Hohn MJ, Jahn D, Soll D (2005) Nanoarchaeum equitans creates functional tRNAs from separate genes for their 5′- and 3′-halves. Nature  433: 537–541.  <ul class="find" data-citedArticleID="1058286" data-doi="10.1038/nature03233"><li><a href="http://dx.doi.org/10.1038/nature03233" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nanoarchaeum+equitans+creates+functional+tRNAs+from+separate+genes+for+their+5%E2%80%B2-+and+3%E2%80%B2-halves." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nanoarchaeum+equitans+creates+functional+tRNAs+from+separate+genes+for+their+5%E2%80%B2-+and+3%E2%80%B2-halves.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001622-TocchiniValentini1" id="pone.0001622-TocchiniValentini1"></a>Tocchini-Valentini GD, Fruscoloni P, Tocchini-Valentini GP (2005) Coevolution of tRNA intron motifs and tRNA endonuclease architecture in Archaea. Proc Natl Acad Sci U S A  102: 15418–15422.  <ul class="find" data-citedArticleID="1058306" data-doi="10.1073/pnas.0506750102"><li><a href="http://dx.doi.org/10.1073/pnas.0506750102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coevolution+of+tRNA+intron+motifs+and+tRNA+endonuclease+architecture+in+Archaea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coevolution+of+tRNA+intron+motifs+and+tRNA+endonuclease+architecture+in+Archaea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001622-Randau2" id="pone.0001622-Randau2"></a>Randau L, Calvin K, Hall M, Yuan J, Podar M, et al.  (2005) The heteromeric Nanoarchaeum equitans splicing endonuclease cleaves noncanonical bulge-helix-bulge motifs of joined tRNA halves. Proc Natl Acad Sci U S A  102: 17934–17939.  <ul class="find" data-citedArticleID="1058288" data-doi="10.1073/pnas.0509197102"><li><a href="http://dx.doi.org/10.1073/pnas.0509197102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+heteromeric+Nanoarchaeum+equitans+splicing+endonuclease+cleaves+noncanonical+bulge-helix-bulge+motifs+of+joined+tRNA+halves." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+heteromeric+Nanoarchaeum+equitans+splicing+endonuclease+cleaves+noncanonical+bulge-helix-bulge+motifs+of+joined+tRNA+halves.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001622-Marck2" id="pone.0001622-Marck2"></a>Marck C, Grosjean H (2003) Identification of BHB splicing motifs in intron-containing tRNAs from 18 archaea: evolutionary implications. Rna  9: 1516–1531.  <ul class="find" data-citedArticleID="1058278" data-doi="10.1261/rna.5132503"><li><a href="http://dx.doi.org/10.1261/rna.5132503" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+BHB+splicing+motifs+in+intron-containing+tRNAs+from+18+archaea%3A+evolutionary+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+BHB+splicing+motifs+in+intron-containing+tRNAs+from+18+archaea%3A+evolutionary+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001622-Diener1" id="pone.0001622-Diener1"></a>Diener JL, Moore PB (1998) Solution structure of a substrate for the archaeal pre-tRNA splicing endonucleases: the bulge-helix-bulge motif. Mol Cell  1: 883–894.  <ul class="find" data-citedArticleID="1058262" data-doi="10.1016/s1097-2765(00)80087-8"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)80087-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Solution+structure+of+a+substrate+for+the+archaeal+pre-tRNA+splicing+endonucleases%3A+the+bulge-helix-bulge+motif." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Solution+structure+of+a+substrate+for+the+archaeal+pre-tRNA+splicing+endonucleases%3A+the+bulge-helix-bulge+motif.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001622-Yoshinari1" id="pone.0001622-Yoshinari1"></a>Yoshinari S, Itoh T, Hallam SJ, DeLong EF, Yokobori S, et al.  (2006) Archaeal pre-mRNA splicing: a connection to hetero-oligomeric splicing endonuclease. Biochem Biophys Res Commun  346: 1024–1032.  <ul class="find" data-citedArticleID="1058324" data-doi="10.1016/j.bbrc.2006.06.011"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2006.06.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Archaeal+pre-mRNA+splicing%3A+a+connection+to+hetero-oligomeric+splicing+endonuclease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Archaeal+pre-mRNA+splicing%3A+a+connection+to+hetero-oligomeric+splicing+endonuclease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001622-Watanabe1" id="pone.0001622-Watanabe1"></a>Watanabe Y, Yokobori S, Inaba T, Yamagishi A, Oshima T, et al.  (2002) Introns in protein-coding genes in Archaea. FEBS Lett  510: 27–30.  <ul class="find" data-citedArticleID="1058310" data-doi="10.1016/s0014-5793(01)03219-7"><li><a href="http://dx.doi.org/10.1016/s0014-5793(01)03219-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Introns+in+protein-coding+genes+in+Archaea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Introns+in+protein-coding+genes+in+Archaea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001622-Tang1" id="pone.0001622-Tang1"></a>Tang TH, Rozhdestvensky TS, d'Orval BC, Bortolin ML, Huber H, et al.  (2002) RNomics in Archaea reveals a further link between splicing of archaeal introns and rRNA processing. Nucleic Acids Res  30: 921–930.  <ul class="find" data-citedArticleID="1058302" data-doi="10.1093/nar/30.4.921"><li><a href="http://dx.doi.org/10.1093/nar/30.4.921" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNomics+in+Archaea+reveals+a+further+link+between+splicing+of+archaeal+introns+and+rRNA+processing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNomics+in+Archaea+reveals+a+further+link+between+splicing+of+archaeal+introns+and+rRNA+processing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001622-Li1" id="pone.0001622-Li1"></a>Li H, Trotta CR, Abelson J (1998) Crystal structure and evolution of a transfer RNA splicing enzyme. Science  280: 279–284.  <ul class="find" data-citedArticleID="1058270" data-doi="10.1126/science.280.5361.279"><li><a href="http://dx.doi.org/10.1126/science.280.5361.279" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+and+evolution+of+a+transfer+RNA+splicing+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+and+evolution+of+a+transfer+RNA+splicing+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001622-Belfort1" id="pone.0001622-Belfort1"></a>Belfort M, Weiner A (1997) Another bridge between kingdoms: tRNA splicing in archaea and eukaryotes. Cell  89: 1003–1006.  <ul class="find" data-citedArticleID="1058246" data-doi="10.1016/s0092-8674(00)80287-1"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80287-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Another+bridge+between+kingdoms%3A+tRNA+splicing+in+archaea+and+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Another+bridge+between+kingdoms%3A+tRNA+splicing+in+archaea+and+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001622-Soma1" id="pone.0001622-Soma1"></a>Soma A, Onodera A, Sugahara J, Kanai A, Yachie N, et al.  (2007) Permuted tRNA genes expressed via a circular RNA intermediate in Cyanidioschyzon merolae. Science  318: 450–453.  <ul class="find" data-citedArticleID="1058296" data-doi="10.1126/science.1145718"><li><a href="http://dx.doi.org/10.1126/science.1145718" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Permuted+tRNA+genes+expressed+via+a+circular+RNA+intermediate+in+Cyanidioschyzon+merolae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Permuted+tRNA+genes+expressed+via+a+circular+RNA+intermediate+in+Cyanidioschyzon+merolae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001622-DiGiulio3" id="pone.0001622-DiGiulio3"></a>Di Giulio M (2006) Nanoarchaeum equitans is a living fossil. J Theor Biol  242: 257–260.  <ul class="find" data-citedArticleID="1058258" data-doi="10.1016/j.jtbi.2006.01.034"><li><a href="http://dx.doi.org/10.1016/j.jtbi.2006.01.034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nanoarchaeum+equitans+is+a+living+fossil." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nanoarchaeum+equitans+is+a+living+fossil.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001622-DiGiulio4" id="pone.0001622-DiGiulio4"></a>Di Giulio M (1992) On the origin of the transfer RNA molecule. J Theor Biol  159: 199–214.  <ul class="find" data-citedArticleID="1058260" data-doi="10.1016/s0022-5193(05)80702-7"><li><a href="http://dx.doi.org/10.1016/s0022-5193(05)80702-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+origin+of+the+transfer+RNA+molecule." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+origin+of+the+transfer+RNA+molecule.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001622-Hipps1" id="pone.0001622-Hipps1"></a>Hipps D, Shiba K, Henderson B, Schimmel P (1995) Operational RNA code for amino acids: species-specific aminoacylation of minihelices switched by a single nucleotide. Proc Natl Acad Sci U S A  92: 5550–5552.  <ul class="find" data-citedArticleID="1058264" data-doi="10.1073/pnas.92.12.5550"><li><a href="http://dx.doi.org/10.1073/pnas.92.12.5550" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Operational+RNA+code+for+amino+acids%3A+species-specific+aminoacylation+of+minihelices+switched+by+a+single+nucleotide." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Operational+RNA+code+for+amino+acids%3A+species-specific+aminoacylation+of+minihelices+switched+by+a+single+nucleotide.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001622-Waters1" id="pone.0001622-Waters1"></a>Waters E, Hohn MJ, Ahel I, Graham DE, Adams MD, et al.  (2003) The genome of Nanoarchaeum equitans: insights into early archaeal evolution and derived parasitism. Proc Natl Acad Sci U S A  100: 12984–12988.  <ul class="find" data-citedArticleID="1058312" data-doi="10.1073/pnas.1735403100"><li><a href="http://dx.doi.org/10.1073/pnas.1735403100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+of+Nanoarchaeum+equitans%3A+insights+into+early+archaeal+evolution+and+derived+parasitism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+of+Nanoarchaeum+equitans%3A+insights+into+early+archaeal+evolution+and+derived+parasitism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001622-Brochier1" id="pone.0001622-Brochier1"></a>Brochier C, Forterre P, Gribaldo S (2005) An emerging phylogenetic core of Archaea: phylogenies of transcription and translation machineries converge following addition of new genome sequences. BMC Evol Biol  5: 36.  <ul class="find" data-citedArticleID="1058250"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+emerging+phylogenetic+core+of+Archaea%3A+phylogenies+of+transcription+and+translation+machineries+converge+following+addition+of+new+genome+sequences.&amp;auth=&amp;atitle=An+emerging+phylogenetic+core+of+Archaea%3A+phylogenies+of+transcription+and+translation+machineries+converge+following+addition+of+new+genome+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+emerging+phylogenetic+core+of+Archaea%3A+phylogenies+of+transcription+and+translation+machineries+converge+following+addition+of+new+genome+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+emerging+phylogenetic+core+of+Archaea%3A+phylogenies+of+transcription+and+translation+machineries+converge+following+addition+of+new+genome+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001622-Lowe1" id="pone.0001622-Lowe1"></a>Lowe TM, Eddy SR (1997) tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence. Nucleic Acids Res  25: 955–964.  <ul class="find" data-citedArticleID="1058272" data-doi="10.1093/nar/25.5.955"><li><a href="http://dx.doi.org/10.1093/nar/25.5.955" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNAscan-SE%3A+a+program+for+improved+detection+of+transfer+RNA+genes+in+genomic+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNAscan-SE%3A+a+program+for+improved+detection+of+transfer+RNA+genes+in+genomic+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001622-Wong1" id="pone.0001622-Wong1"></a>Wong J (1975) A co-evolution theory of the genetic code. Proc Natl Acad Sci U S A  72: 1909–1912.  <ul class="find" data-citedArticleID="1058322"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+co-evolution+theory+of+the+genetic+code.&amp;auth=&amp;atitle=A+co-evolution+theory+of+the+genetic+code." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+co-evolution+theory+of+the+genetic+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+co-evolution+theory+of+the+genetic+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001622-Wei1" id="pone.0001622-Wei1"></a>Wei F, Meng M, Li S, Ma H (2006) Comparing two evolutionary mechanisms of modern tRNAs. Mol Phylogenet Evol  38: 1–11.  <ul class="find" data-citedArticleID="1058316" data-doi="10.1016/j.ympev.2005.09.017"><li><a href="http://dx.doi.org/10.1016/j.ympev.2005.09.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparing+two+evolutionary+mechanisms+of+modern+tRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparing+two+evolutionary+mechanisms+of+modern+tRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001622-Barabasi1" id="pone.0001622-Barabasi1"></a>Barabasi AL, Albert R (1999) Emergence of scaling in random networks. Science  286: 509–512.  <ul class="find" data-citedArticleID="1058244" data-doi="10.1126/science.286.5439.509"><li><a href="http://dx.doi.org/10.1126/science.286.5439.509" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emergence+of+scaling+in+random+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emergence+of+scaling+in+random+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001622-Watts1" id="pone.0001622-Watts1"></a>Watts DJ, Strogatz SH (1998) Collective dynamics of ‘small-world’ networks. Nature  393: 440–442.  <ul class="find" data-citedArticleID="1058314" data-doi="10.1038/30918"><li><a href="http://dx.doi.org/10.1038/30918" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Collective+dynamics+of+%E2%80%98small-world%E2%80%99+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Collective+dynamics+of+%E2%80%98small-world%E2%80%99+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001622-Randau3" id="pone.0001622-Randau3"></a>Randau L, Pearson M, Soll D (2005) The complete set of tRNA species in Nanoarchaeum equitans. FEBS Lett  579: 2945–2947.  <ul class="find" data-citedArticleID="1058290" data-doi="10.1016/j.febslet.2005.04.051"><li><a href="http://dx.doi.org/10.1016/j.febslet.2005.04.051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+set+of+tRNA+species+in+Nanoarchaeum+equitans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+set+of+tRNA+species+in+Nanoarchaeum+equitans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001622-Wang1" id="pone.0001622-Wang1"></a>Wang S, Pirtle IL, Pirtle RM (1997) A human 28S ribosomal RNA retropseudogene. Gene  196: 105–111.  <ul class="find" data-citedArticleID="1058308" data-doi="10.1016/s0378-1119(97)00214-x"><li><a href="http://dx.doi.org/10.1016/s0378-1119(97)00214-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+human+28S+ribosomal+RNA+retropseudogene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+human+28S+ribosomal+RNA+retropseudogene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001622-Pavlicek1" id="pone.0001622-Pavlicek1"></a>Pavlicek A, Gentles AJ, Paces J, Paces V, Jurka J (2006) Retroposition of processed pseudogenes: the impact of RNA stability and translational control. Trends Genet  22: 69–73.  <ul class="find" data-citedArticleID="1058284" data-doi="10.1016/j.tig.2005.11.005"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.11.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroposition+of+processed+pseudogenes%3A+the+impact+of+RNA+stability+and+translational+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroposition+of+processed+pseudogenes%3A+the+impact+of+RNA+stability+and+translational+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001622-Kacar1" id="pone.0001622-Kacar1"></a>Kacar Y, Beier H, Gross HJ (1995) The presence of tRNA pseudogenes in mammalia and plants and their absence in yeast may account for different specificities of pre-tRNA processing enzymes. Gene  156: 129–132.  <ul class="find" data-citedArticleID="1058268" data-doi="10.1016/0378-1119(95)00079-l"><li><a href="http://dx.doi.org/10.1016/0378-1119(95)00079-l" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+presence+of+tRNA+pseudogenes+in+mammalia+and+plants+and+their+absence+in+yeast+may+account+for+different+specificities+of+pre-tRNA+processing+enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+presence+of+tRNA+pseudogenes+in+mammalia+and+plants+and+their+absence+in+yeast+may+account+for+different+specificities+of+pre-tRNA+processing+enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001622-Ohshima1" id="pone.0001622-Ohshima1"></a>Ohshima K, Koishi R, Matsuo M, Okada N (1993) Several short interspersed repetitive elements (SINEs) in distant species may have originated from a common ancestral retrovirus: characterization of a squid SINE and a possible mechanism for generation of tRNA-derived retroposons. Proc Natl Acad Sci U S A  90: 6260–6264.  <ul class="find" data-citedArticleID="1058282" data-doi="10.1073/pnas.90.13.6260"><li><a href="http://dx.doi.org/10.1073/pnas.90.13.6260" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Several+short+interspersed+repetitive+elements+%28SINEs%29+in+distant+species+may+have+originated+from+a+common+ancestral+retrovirus%3A+characterization+of+a+squid+SINE+and+a+possible+mechanism+for+generation+of+tRNA-derived+retroposons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Several+short+interspersed+repetitive+elements+%28SINEs%29+in+distant+species+may+have+originated+from+a+common+ancestral+retrovirus%3A+characterization+of+a+squid+SINE+and+a+possible+mechanism+for+generation+of+tRNA-derived+retroposons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001622-Schmitz1" id="pone.0001622-Schmitz1"></a>Schmitz J, Churakov G, Zischler H, Brosius J (2004) A novel class of mammalian-specific tailless retropseudogenes. Genome Res  14: 1911–1915.  <ul class="find" data-citedArticleID="1058292" data-doi="10.1101/gr.2720104"><li><a href="http://dx.doi.org/10.1101/gr.2720104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+class+of+mammalian-specific+tailless+retropseudogenes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+class+of+mammalian-specific+tailless+retropseudogenes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001622-Chiang1" id="pone.0001622-Chiang1"></a>Chiang CC, Lambowitz AM (1997) The Mauriceville retroplasmid reverse transcriptase initiates cDNA synthesis de novo at the 3′ end of tRNAs. Mol Cell Biol  17: 4526–4535.  <ul class="find" data-citedArticleID="1058252"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Mauriceville+retroplasmid+reverse+transcriptase+initiates+cDNA+synthesis+de+novo+at+the+3%3F+end+of+tRNAs.&amp;auth=&amp;atitle=The+Mauriceville+retroplasmid+reverse+transcriptase+initiates+cDNA+synthesis+de+novo+at+the+3%3F+end+of+tRNAs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Mauriceville+retroplasmid+reverse+transcriptase+initiates+cDNA+synthesis+de+novo+at+the+3%E2%80%B2+end+of+tRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Mauriceville+retroplasmid+reverse+transcriptase+initiates+cDNA+synthesis+de+novo+at+the+3%E2%80%B2+end+of+tRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001622-Auchtung1" id="pone.0001622-Auchtung1"></a>Auchtung TA, Takacs-Vesbach CD, Cavanaugh CM (2006) 16S rRNA phylogenetic investigation of the candidate division “Korarchaeota”. Appl Environ Microbiol  72: 5077–5082.  <ul class="find" data-citedArticleID="1058242" data-doi="10.1128/aem.00052-06"><li><a href="http://dx.doi.org/10.1128/aem.00052-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=16S+rRNA+phylogenetic+investigation+of+the+candidate+division+%E2%80%9CKorarchaeota%E2%80%9D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%2216S+rRNA+phylogenetic+investigation+of+the+candidate+division+%E2%80%9CKorarchaeota%E2%80%9D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001622-Thompson1" id="pone.0001622-Thompson1"></a>Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG (1997) The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res  25: 4876–4882.  <ul class="find" data-citedArticleID="1058304" data-doi="10.1093/nar/25.24.4876"><li><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001622-Wilm1" id="pone.0001622-Wilm1"></a>Wilm A, Mainz I, Steger G (2006) An enhanced RNA alignment benchmark for sequence alignment programs. Algorithms Mol Biol  1: 19.  <ul class="find" data-citedArticleID="1058320"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+enhanced+RNA+alignment+benchmark+for+sequence+alignment+programs.&amp;auth=&amp;atitle=An+enhanced+RNA+alignment+benchmark+for+sequence+alignment+programs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+enhanced+RNA+alignment+benchmark+for+sequence+alignment+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+enhanced+RNA+alignment+benchmark+for+sequence+alignment+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001622-Huelsenbeck1" id="pone.0001622-Huelsenbeck1"></a>Huelsenbeck JP, Ronquist F (2001) MRBAYES: Bayesian inference of phylogenetic trees. Bioinformatics  17: 754–755.  <ul class="find" data-citedArticleID="1058266" data-doi="10.1093/bioinformatics/17.8.754"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.8.754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MRBAYES%3A+Bayesian+inference+of+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MRBAYES%3A+Bayesian+inference+of+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001622-Bingham1" id="pone.0001622-Bingham1"></a>Bingham J, Sudarsanam S (2000) Visualizing large hierarchical clusters in hyperbolic space. Bioinformatics  16: 660–661.  <ul class="find" data-citedArticleID="1058248" data-doi="10.1093/bioinformatics/16.7.660"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.7.660" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Visualizing+large+hierarchical+clusters+in+hyperbolic+space." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Visualizing+large+hierarchical+clusters+in+hyperbolic+space.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001622-Shannon1" id="pone.0001622-Shannon1"></a>Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, et al.  (2003) Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res  13: 2498–2504.  <ul class="find" data-citedArticleID="1058294" data-doi="10.1101/gr.1239303"><li><a href="http://dx.doi.org/10.1101/gr.1239303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.XML" value="83200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.PDF" value="553654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g001.PNG_L" value="2261910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g001.PNG_M" value="205417"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g001.PNG_S" value="12955"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g001.TIF" value="3647998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g001.PNG_I" value="69594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g002.PNG_L" value="1264871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g002.PNG_M" value="123229"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g002.PNG_S" value="7708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g002.TIF" value="2751478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g002.PNG_I" value="41802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g003.PNG_L" value="2203661"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g003.PNG_M" value="124950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g003.PNG_S" value="9528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g003.TIF" value="3724400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g003.PNG_I" value="42563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g004.PNG_L" value="1127921"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g004.PNG_M" value="62320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g004.PNG_S" value="8487"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g004.TIF" value="1392280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g004.PNG_I" value="20977"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g005.PNG_L" value="1342707"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g005.PNG_M" value="245125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g005.PNG_S" value="20598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g005.TIF" value="3078888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g005.PNG_I" value="100423"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g006.PNG_L" value="1054855"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g006.PNG_M" value="58600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g006.PNG_S" value="7334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g006.TIF" value="1396606"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.g006.PNG_I" value="19676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.e001.PNG" value="6748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.e001.TIF" value="22700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.s001.DOC" value="37376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.s002.DOC" value="1631232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001622.s003.DOC" value="3169792"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001622&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001622" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001622&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001622&volume=&issue=&title=Sequence Evidence in the Archaeal Genomes that tRNAs Emerged Through the Combination of Ancestral Genes as 5′ and 3′ tRNA Halves&author_name=Kosuke%20Fujishima%2C%20Junichi%20Sugahara%2C%20Masaru%20Tomita%2C%20Akio%20Kanai&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001622" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001622" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001622" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001622&amp;t=Sequence%20Evidence%20in%20the%20Archaeal%20Genomes%20that%20tRNAs%20Emerged%20Through%20the%20Combination%20of%20Ancestral%20Genes%20as%205%E2%80%B2%20and%203%E2%80%B2%20tRNA%20Halves" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001622&title=Sequence%20Evidence%20in%20the%20Archaeal%20Genomes%20that%20tRNAs%20Emerged%20Through%20the%20Combination%20of%20Ancestral%20Genes%20as%205%E2%80%B2%20and%203%E2%80%B2%20tRNA%20Halves&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001622&amp;title=Sequence%20Evidence%20in%20the%20Archaeal%20Genomes%20that%20tRNAs%20Emerged%20Through%20the%20Combination%20of%20Ancestral%20Genes%20as%205%E2%80%B2%20and%203%E2%80%B2%20tRNA%20Halves" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001622" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001622&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Sequence Evidence in the Archaeal Genomes that tRNAs Emerged Through the Combination of Ancestral Genes as 5′ and 3′ tRNA Halves';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001622';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Sequence%20Evidence%20in%20the%20Archaeal%20Genomes%20that%20tRNAs%20Emerged%20Through%20the%20Combination%20of%20Ancestral%20Genes%20as%205%E2%80%B2%20and%203%E2%80%B2%20tRNA%20Halves http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001622" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001622" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Anticodons%22" title="Search for articles in the subject area:'Anticodons'"><div class="flagText">Anticodons</div></a>
              <div data-categoryid="20289" data-articleid="26070"
                   data-categoryname="Anticodons"
                   class="flagImage" title="Flag 'Anticodons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Archaean+biology%22" title="Search for articles in the subject area:'Archaean biology'"><div class="flagText">Archaean biology</div></a>
              <div data-categoryid="43097" data-articleid="26070"
                   data-categoryname="Archaean biology"
                   class="flagImage" title="Flag 'Archaean biology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="26070"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="26070"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="26070"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="26070"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26070"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transfer+RNA%22" title="Search for articles in the subject area:'Transfer RNA'"><div class="flagText">Transfer RNA</div></a>
              <div data-categoryid="21239" data-articleid="26070"
                   data-categoryname="Transfer RNA"
                   class="flagImage" title="Flag 'Transfer RNA' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5150'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2898'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7501&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=19991">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=14423">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
