

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Is Transcriptional Regulation of Metabolic Pathways an Optimal Strategy for Fitness?</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000855"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000855" />

    <meta name="citation_title" content="Is Transcriptional Regulation of Metabolic Pathways an Optimal Strategy for Fitness?"/>
    <meta itemprop="name" content="Is Transcriptional Regulation of Metabolic Pathways an Optimal Strategy for Fitness?"/>

      <meta name="citation_author" content="Carl Troein"/>
            <meta name="citation_author_institution" content="Computational Biology and Biological Physics, Department of Theoretical Physics, Lund University, Lund, Sweden"/>
      <meta name="citation_author" content="Dag Ahrén"/>
            <meta name="citation_author_institution" content="Microbial Ecology, Department of Ecology, Lund University, Lund, Sweden"/>
      <meta name="citation_author" content="Morten Krogh"/>
            <meta name="citation_author_institution" content="Computational Biology and Biological Physics, Department of Theoretical Physics, Lund University, Lund, Sweden"/>
      <meta name="citation_author" content="Carsten Peterson"/>
            <meta name="citation_author_institution" content="Computational Biology and Biological Physics, Department of Theoretical Physics, Lund University, Lund, Sweden"/>

    <meta name="citation_date" content="2007/9/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000855.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e855"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Optimality and evolutionary tuning of the expression level of a protein.; citation_author=E Dekel; citation_author=U Alon; citation_journal_title=Nature; citation_volume=436; citation_number=1; citation_pages=588-592; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=In silico predictions of Escherichia coli metabolic capabilities are consistent with experimental data.; citation_author=JS Edwards; citation_author=RU Ibarra; citation_author=BO Palsson; citation_journal_title=Nat Biotech; citation_volume=19; citation_number=2; citation_pages=125-130; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Metabolic switching in the sugar phosphotransferase system of Escherichia coli.; citation_author=M Thattai; citation_author=BI Shraiman; citation_journal_title=Biophys J; citation_volume=85; citation_number=3; citation_pages=744-754; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Demand theory of gene regulation. I. Quantitative development of the theory.; citation_author=MA Savageau; citation_journal_title=Genetics; citation_volume=149; citation_number=4; citation_pages=1665-1676; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Evolution of digital organisms at high mutation rates leads to survival of the flattest.; citation_author=CO Wilke; citation_author=JL Wang; citation_author=C Ofria; citation_author=RE Lenski; citation_author=C Adami; citation_journal_title=Nature; citation_volume=412; citation_number=5; citation_pages=331-333; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Enzyme induction as an all-or-none phenomenon.; citation_author=A Novick; citation_author=M Weiner; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=43; citation_number=6; citation_pages=553-566; citation_date=1957; " />
      <meta name="citation_reference" content="citation_title=The protein burden of lac operon products.; citation_author=AL Koch; citation_journal_title=J Mol Evol; citation_volume=19; citation_number=7; citation_pages=455-462; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Design of molecular control mechanisms and the demand for gene expression.; citation_author=MA Savageau; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=74; citation_number=8; citation_pages=5647-5651; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Demand theory of gene regulation. II. Quantitative application to the lactose and maltose operons of Escherichia coli.; citation_author=M Savageau; citation_journal_title=Genetics; citation_volume=149; citation_number=9; citation_pages=1677-1691; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=In silico evolved lac operons exhibit bistability for artificial inducers, but not for lactose.; citation_author=MJA van Hoek; citation_author=P Hogeweg; citation_journal_title=Biophys J; citation_volume=91; citation_number=10; citation_pages=2833-2843; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulation in constraints-based metabolic models of Escherichia coli.; citation_author=MW Covert; citation_author=BO Palsson; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=11; citation_pages=28058-28064; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The dynamics of single-substrate continuous cultures: the role of transport enzymes.; citation_author=J Shoemaker; citation_author=GT Reeves; citation_author=S Gupta; citation_author=SS Pilyugin; citation_author=T Egli; citation_journal_title=J Theor Biol; citation_volume=222; citation_number=12; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A note on the kinetics of enzyme action.; citation_author=GE Briggs; citation_author=JBS Haldane; citation_journal_title=Biochem J; citation_volume=19; citation_number=13; citation_pages=338-339; citation_date=1925; " />
      <meta name="citation_reference" content="citation_author=P Nelson; citation_number=14; citation_date=2004; citation_publisher=W. H. Freeman, first edition; " />
      <meta name="citation_reference" content="citation_title=A new model of uptake of multiple sugars by S. cerevisiae.; citation_author=JP Barford; citation_author=PJ Phillips; citation_author=JH Orlowski; citation_journal_title=Bioprocess Eng; citation_volume=7; citation_number=15; citation_pages=297-302; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Regulation of transcription in a reduced bacterial genome: nutrient-provisioning genes of the obligate symbiont Buchnera aphidicola.; citation_author=NA Moran; citation_author=HE Dunbar; citation_author=JL Wilcox; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=16; citation_pages=4229-4237; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolution by gene duplication: an update.; citation_author=J Zhang; citation_journal_title=Trends Ecol Evol; citation_volume=18; citation_number=17; citation_pages=292-298; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The evolutionary fate and consequences of duplicate genes.; citation_author=M Lynch; citation_author=JS Conery; citation_journal_title=Science; citation_volume=290; citation_number=18; citation_pages=1151-1155; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The probability of duplicate gene preservation by subfunctionalization.; citation_author=M Lynch; citation_author=A Force; citation_journal_title=Genetics; citation_volume=154; citation_number=19; citation_pages=459-473; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Subfunctionalization of duplicated genes as a transition state to neofunctionalization.; citation_author=S Rastogi; citation_author=DA Liberles; citation_journal_title=BMC Evol Biol; citation_volume=5; citation_number=20; citation_pages=28; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Computational studies of gene regulatory networks: in numero molecular biology.; citation_author=J Hasty; citation_author=D McMillen; citation_author=F Isaacs; citation_author=JJ Collins; citation_journal_title=Nat Rev Genet; citation_volume=2; citation_number=21; citation_pages=268-279; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Bistability in bacteria.; citation_author=D Dubnau; citation_author=R Losick; citation_journal_title=Mol Microbiol; citation_volume=61; citation_number=22; citation_pages=564-572; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Stochastic gene expression in fluctuating environments.; citation_author=M Thattai; citation_author=A van Oudenaarden; citation_journal_title=Genetics; citation_volume=167; citation_number=23; citation_pages=523-530; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Multistability in the lactose utilization network of Escherichia coli.; citation_author=EM Ozbudak; citation_author=M Thattai; citation_author=HN Lim; citation_author=BI Shraiman; citation_author=A van Oudenaarden; citation_journal_title=Nature; citation_volume=427; citation_number=24; citation_pages=737-740; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolution of complexity in signaling pathways.; citation_author=OS Soyer; citation_author=S Bonhoeffer; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=104; citation_number=25; citation_pages=16337-16342; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Proteome analysis of yeast response to various nutrient limitations.; citation_author=A Kolkman; citation_author=P Daran-Lapujade; citation_author=A Fullaondo; citation_author=MM Olsthoorn; citation_author=JT Pronk; citation_journal_title=Mol Syst Biol; citation_volume=2; citation_number=26; citation_pages=2006.0026; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000855" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Is Transcriptional Regulation of Metabolic Pathways an Optimal Strategy for Fitness?"/>
    <meta name="twitter:description" content="BackgroundTranscriptional regulation of the genes in metabolic pathways is a highly successful strategy, which is virtually universal in microorganisms. The lac operon of E. coli is but one example of how enzyme and transporter production can be made conditional on the presence of a nutrient to catabolize.MethodologyWith a minimalist model of metabolism, cell growth and transcriptional regulation in a microorganism, we explore how the interaction between environmental conditions and gene regulation set the growth rate of cells in the phase of exponential growth. This in silico model, which is based on biochemical rate equations, does not describe a specific organism, but the magnitudes of its parameters are chosen to match realistic values. Optimizing the parameters of the regulatory system allows us to quantify the fitness benefit of regulation. When a second nutrient and its metabolic pathway are introduced, the system must further decide whether and how to activate both pathways.ConclusionsEven the crudest transcriptional network is shown to substantially increase the fitness of the organism, and this effect persists even when the range of nutrient levels is kept very narrow. We show that maximal growth is achieved when pathway activation is a more or less steeply graded function of the nutrient concentration. Furthermore, we predict that bistability of the system is a rare phenomenon in this context, but outline a situation where it may be selected for."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000855.g005"/>

  <meta property="og:title" content="Is Transcriptional Regulation of Metabolic Pathways an Optimal Strategy for Fitness?" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000855" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1152'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3310'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7828&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000855">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000855" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000855&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Is Transcriptional Regulation of Metabolic Pathways an Optimal Strategy for Fitness?
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carl Troein
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:carl@thep.lu.se">carl@thep.lu.se</a></p>

                <p>Affiliation:
                  Computational Biology and Biological Physics, Department of Theoretical Physics, Lund University, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dag Ahrén, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Microbial Ecology, Department of Ecology, Lund University, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Morten Krogh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computational Biology and Biological Physics, Department of Theoretical Physics, Lund University, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carsten Peterson
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computational Biology and Biological Physics, Department of Theoretical Physics, Lund University, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 05, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000855</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000855" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000855" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000855" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000855" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000855-g001" data-doi="info:doi/10.1371/journal.pone.0000855" data-uri="info:doi/10.1371/journal.pone.0000855.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000855-g002" data-doi="info:doi/10.1371/journal.pone.0000855" data-uri="info:doi/10.1371/journal.pone.0000855.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000855-g003" data-doi="info:doi/10.1371/journal.pone.0000855" data-uri="info:doi/10.1371/journal.pone.0000855.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000855-g004" data-doi="info:doi/10.1371/journal.pone.0000855" data-uri="info:doi/10.1371/journal.pone.0000855.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000855-g005" data-doi="info:doi/10.1371/journal.pone.0000855" data-uri="info:doi/10.1371/journal.pone.0000855.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000855">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000855" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Transcriptional regulation of the genes in metabolic pathways is a highly successful strategy, which is virtually universal in microorganisms. The <em>lac</em> operon of <em>E. coli</em> is but one example of how enzyme and transporter production can be made conditional on the presence of a nutrient to catabolize.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>With a minimalist model of metabolism, cell growth and transcriptional regulation in a microorganism, we explore how the interaction between environmental conditions and gene regulation set the growth rate of cells in the phase of exponential growth. This <em>in silico</em> model, which is based on biochemical rate equations, does not describe a specific organism, but the magnitudes of its parameters are chosen to match realistic values. Optimizing the parameters of the regulatory system allows us to quantify the fitness benefit of regulation. When a second nutrient and its metabolic pathway are introduced, the system must further decide whether and how to activate both pathways.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Even the crudest transcriptional network is shown to substantially increase the fitness of the organism, and this effect persists even when the range of nutrient levels is kept very narrow. We show that maximal growth is achieved when pathway activation is a more or less steeply graded function of the nutrient concentration. Furthermore, we predict that bistability of the system is a rare phenomenon in this context, but outline a situation where it may be selected for.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Troein C, Ahrén D, Krogh M, Peterson C (2007) Is Transcriptional Regulation of Metabolic Pathways an Optimal Strategy for Fitness? PLoS ONE 2(9):
          e855.
            doi:10.1371/journal.pone.0000855</p><p><strong>Academic Editor: </strong>Mukund Thattai, Tata Institute of Fundamental Research, India</p><p><strong>Received:</strong> May 21, 2007; <strong>Accepted:</strong> August 14, 2007; <strong>Published:</strong> September 5, 2007</p><p><strong>Copyright:</strong> © 2007 Troein et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>CT acknowledges support from the Swedish National Research School in Genomics and Bioinformatics. DA was supported by the Knut and Alice Wallenberg foundation. CP and MK were supported by the Swedish Foundation for Strategic Research (Senior Individual Grant). MK was also supported by the Knut and Alice Wallenberg Foundation through the Swegene consortium and the Strategic Science Foundation (Lund Strategic Center for Clinical Cancer Research).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Transcriptional regulation of effector genes is a highly successful strategy, as evidenced by our tendency to ask how rather than whether a gene is regulated. A very natural place to study gene regulation is in the metabolism of the cell, and then specifically in the regulation of genes that code for enzymes and transporter proteins. Here, the function of regulation is quite clear: expressing the right genes at the right time will enable the cell to make the most of the resources within its reach, by maximizing the uptake and use of rate-limiting resources such as carbon and energy.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In unicellular organisms like <em>E. coli</em> and yeast, the benefits of a well-adapted regulatory system are readily quantified, as the fitness of an individual can be estimated by its growth rate in culture. A number of studies have explored how regulation of metabolic pathways affects the growth rate of microorganisms, both in the steady state and in response to changes in the local environment. In a typical experimental setup, <em>E. coli</em> is grown in a chemostat for some period of time, with one or two carbon sources present at levels that are perturbed at some point in time (see, e.g., <a href="#pone.0000855-Dekel1">[1]</a>–<a href="#pone.0000855-Thattai1">[3]</a>).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Over evolutionary time scales, regulation must provide a fitness benefit that offsets the costs of maintaining the regulatory system. Most immediately, precious resources will be spent on synthesizing transcription factors and replicating extra DNA, rather than going directly into growth of the cell. However, this cost can easily be dwarfed by the cost of a failure to regulate gene expression optimally, as enzymes are typically produced at far higher rates than transcription factors.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>There is also an entropic cost involved in maintaining a regulatory system, stemming from random mutations that tend to destroy transcription factors and binding sites. As elucidated by Savageau <a href="#pone.0000855-Savageau1">[4]</a>, for functioning regulation to be present in the wild type, the population of that genotype must offset losses due to mutations by having a higher grow rate than the mutants with broken regulation. The design of the regulatory system affects the growth rate not only when the system is intact but also when it is broken, which in realistic situations can severly constrain the regulatory options. This is “survival of the flattest” <a href="#pone.0000855-Wilke1">[5]</a> at work.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In the case of the <em>lac</em> operon of <em>E. coli</em>, a well-studied system for detecting and metabolizing lactose, it is known that the overall effect of expressing the <em>lac</em> genes in vain is a drop in the growth rate of as much as 5% <a href="#pone.0000855-Dekel1">[1]</a>, <a href="#pone.0000855-Novick1">[6]</a>. It has been argued, based on the cost in energy and carbon, that a number around 0.2% would be expected, and that the difference is more or less specific to the <em>lac</em> operon <a href="#pone.0000855-Koch1">[7]</a>. Utilization of lactose, when present, has a positive effect on the order of 10–15% <a href="#pone.0000855-Dekel1">[1]</a>. With a such an asymmetry between potential cost and benefit, regulation can make the most difference to the long-term growth rate if the resource in question is only available a similarly small fraction of the time. More generally, and at least to a first approximation, it is obvious that gene regulation only is useful if the environmental conditions vary with time. Experimental data show that a repressive mode, where the presence of a resource disables the binding of a repressor to the DNA, is preferred when demand for expression is rare, whereas an activating mode is preferred in the opposite situation <a href="#pone.0000855-Savageau2">[8]</a>. For the <em>lac</em> system and realistic time variations, the entropic effects appear to be so large that only a repressive regulatory mode is possible <a href="#pone.0000855-Savageau3">[9]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>To concretize the question of how to regulate metabolic processes, we consider the simplified view presented in <a href="#pone-0000855-g001">figure 1(A)</a>. Here, an organism grows in a medium with one relevant rate-limiting resource, presumably a combination of carbon and energy, and several nutrients may be available to provide this resource. Nutrients present in the medium can be absorbed by the cell and converted into a useful form by the actions of different enzymes and other proteins. The proteins are replenished at the cost of slower growth, and evolution will optimize the growth rate over a set of environmental conditions by tuning the regulation of the protein production rates.</p>
<div class="figure" id="pone-0000855-g001"><div class="img"><a name="pone-0000855-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000855" data-uri="info:doi/10.1371/journal.pone.0000855.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000855.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000855.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>A) A general view of an organism with one or more metabolic pathways.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Solid lines represent the flow of biomass, while dotted lines show regulatory interactions, with arrows for positive regulation and circles where the sign of the regulation can be varied. Nutrients are transported into the cell, processed by enzymes, and spent on cell growth and on replenishing the enzymes. Optionally, transcription factors may detect the presence of nutrients and regulate the enzyme production accordingly. B) A sketch of the most complete model considered here, where two nutrients, A and B, are turned into C by their respective enzymes, E<em><sub>X</sub></em>. Nutrients inside the cell activate the transcription factors, T<em><sub>X</sub></em>→T<sup>*</sup><em><sub>X</sub></em>, which in turn activate or repress the production of enzymes and each other. For parts of this paper, pieces of the model are removed, including the entire B side. When the transcription factors are excluded, the rate constants <em>p</em><sub>E<sub>A</sub></sub> and <em>p</em><sub>E<sub>B</sub></sub> control the production of enzymes.</p>
<span>doi:10.1371/journal.pone.0000855.g001</span></div><a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>In this paper we study how the levels of one or two nutrients interact with the transcriptional regulation of their respective metabolic pathways to determine the growth rate of an organism under steady state conditions. Transient responses are an important aspect of metabolic regulation (see, e.g., <a href="#pone.0000855-Dekel1">[1]</a>, <a href="#pone.0000855-Thattai1">[3]</a>, <a href="#pone.0000855-vanHoek1">[10]</a>), but it is also hypothesized, and in some cases known, that microorganisms excel at optimizing their growth rate in the steady state <a href="#pone.0000855-Edwards1">[2]</a>, <a href="#pone.0000855-Covert1">[11]</a>. Because of this, we may approximate the fitness of an organism from its growth rate in a set of static environments. From such a treatment we can hope to gain further insights into how evolution chooses a mode of regulation, if any, depending on the environments that a species is exposed to.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>The model</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We have implemented a minimalist model of an organism that grows by metabolizing one or more compounds found in its environment. These compounds could represent different sugars, alcohols, or other nutrients, and although the model is not tied to any one particular example, we have glanced at the <em>lac</em> operon of Escherichia coli and the carbon metabolism of yeast. The model as such is rather similar both in spirit and form to that of Shoemaker et al. <a href="#pone.0000855-Shoemaker1">[12]</a>. We have, as far as possible, derived equations and constants from first principles, in order to maximize the generality of the results. Some assumptions and approximations greatly simplify the model:</p>


<ul class="bulletlist">

<li>Chemistry can be dealt with in terms of rate equations.</li>

<li>Michaelis–Menten kinetics <a href="#pone.0000855-Briggs1">[13]</a> are always appropriate.</li>

<li>The cell contents are homogeneous, and transport processes are purely diffusional.</li>

<li>Cell growth and division are one continuous process, resulting only in the dilution of the contents of the cell.</li>

<li>Molecular concentrations have no hard upper limit.</li>

</ul><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Furthermore, when the environment is held constant, it is natural to seek steady state solutions to the equations. If only one stable solution exists, we use the corresponding growth rate as a fitness measure for the organism.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The differential equations that define the model can be found in <a href="#s4">Materials and Methods</a>. Their parameters are largely given by basic physics and rudimentary knowledge about living cells, although in some cases only to within a few orders of magnitude. It should be pointed out that no attempt has been made to fit the model to experimental data, so although predictions made from the model may be qualitatively sound, they cannot be expected to agree quantitatively with any particular organism.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p><a href="#pone-0000855-g001">Figure 1(B)</a> summarizes the interactions of the model, of which we will first consider a reduced version. A compound to be catabolized, call it A, exists outside the cell at concentration <em>A</em><sub>ext</sub>, and can diffuse into and out of the cell. Inside the cell, A is converted into metabolite C with the help of an enzyme, E<sub>A</sub>. The metabolite C is what the cell needs in order to grow and make more E<sub>A</sub>. If A represents glucose, then C might represent pyruvate and ATP, and E<sub>A</sub> is a whole set of enzymes, transporters, and other proteins. The generality of Michaelis–Menten kinetics as an approximation for one-way reactions (see, e.g., <a href="#pone.0000855-Nelson1">[14]</a>) is what keeps this vagueness acceptable. In particular, note that even though we speak of E<sub>A</sub> as an enzyme, the same rate equations arise if the main rate limiting step involves transport across the plasma membrane. Hence our minimalist model is not at odds with the more detailed model of Barford et al. for sugar uptake by yeast <a href="#pone.0000855-Barford1">[15]</a>, given a sensible choice of parameters for the kinetics.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>Let <em>A</em> denote the level of A, and so on. The production of C follows Michaelis–Menten kinetics, depending linearly on each of <em>E</em><sub>A</sub> and <em>A</em> but leveling off at high <em>A</em>. The rates of protein synthesis and cell growth are limited by C at low nutrient levels, but by translation and DNA replication, respectively, at higher C. Just how great the production of E<sub>A</sub>, <em>p</em><sub>E<sub>A</sub></sub>, is depends on the rate constant <em>p</em><sub>E<sub>A</sub></sub>, which captures both transcriptional regulation and the amount of protein synthesized per mRNA. Degradation of E<sub>A</sub> is assumed to occur in the form of exponential decay with a half-life on the order of an hour.</p>


<h4>Optimal enzyme production rate</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Even with a single nutrient, before attempting to regulate the enzyme production rate, we need to look at how much difference this can make to the growth rate over the whole range of environments. Only then can we know if and when there can be any point in employing transcriptional regulation.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>All model parameters except the rate constant for enzyme production, <em>p</em><sub>E<sub>A</sub></sub>, are fixed, and regulation amounts to adapting <em>p</em><sub>E<sub>A</sub></sub> to the nutrient concentration, <em>A</em><sub>ext</sub>. Hence, we have explored how the steady state growth rate depends on <em>p</em><sub>E<sub>A</sub></sub> for many fixed values of <em>A</em><sub>ext</sub>, by integrating the equations of the system from some initial state until reaching a fixed point. The growth rate thus found is normalized such that a value of 1 corresponds to the best possible growth rate for a given <em>A</em><sub>ext</sub>. This quantity, which is shown in <a href="#pone-0000855-g002">figure 2</a>, is a fitness score that reflects how well an organism can compete with others that are perfectly adapted to a single environment. A tiny difference in growth rate becomes significant over many generations. Therefore, the gray scale in <a href="#pone-0000855-g002">figure 2</a> was chosen to resolve small deviations from the maximum growth rate.</p>
<div class="figure" id="pone-0000855-g002"><div class="img"><a name="pone-0000855-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000855" data-uri="info:doi/10.1371/journal.pone.0000855.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000855.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000855.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Growth rate as a function of the enzyme production rate constant, <em>p</em><sub>E<sub>A</sub></sub>, over a range of nutrient concentrations.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Dark areas mark where the growth rate is maximized for the respective <em>A</em><sub>ext</sub>, and the solid and dashed white lines indicate 90% and 99% of the maximum, respectively. Values of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.e001&amp;representation=PNG" class="inline-graphic"></span> are not physically realizable, but are included to show the insensitivity to <em>p</em><sub>E<sub>A</sub></sub> at high <em>A</em><sub>ext</sub>. Note the nonlinear brightness scale, which accentuates small deviations from the maximum.</p>
<span>doi:10.1371/journal.pone.0000855.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Interpreting <a href="#pone-0000855-g002">figure 2</a> is easier if we also consider <a href="#pone-0000855-g003">figure 3</a>, which shows the optimal <em>p</em><sub>E<sub>A</sub></sub> and the resulting protein production rate, <em>p</em><sub>E<sub>A</sub></sub> (determined by <em>p</em><sub>E<sub>A</sub></sub> and <em>C</em>), as functions of <em>A</em><sub>ext</sub>, along with the growth rate and the fraction of resources (C) spent on enzyme production as opposed to growth. At very low concentrations, the cell barely has enough of the nutrient to survive, and it must focus almost all its energy on building enzyme. Because of the degradation of E<sub>A</sub>, there is a point (near <em>A</em><sub>ext</sub> = 20 µM) where each molecule of E<sub>A</sub> costs more molecules of C to synthesize than it can bring about during its lifetime, and starvation is inevitable.</p>
<div class="figure" id="pone-0000855-g003"><div class="img"><a name="pone-0000855-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000855" data-uri="info:doi/10.1371/journal.pone.0000855.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000855.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000855.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>The transcriptional regulation, <em>p</em><sub>E<sub>A</sub></sub>, that maximizes the growth rate of <a href="#pone-0000855-g002">figure 2</a> (solid) and the corresponding actual enzyme production rate, <em>p</em><sub>E<sub>A</sub></sub> (dot-dashed), in units of molecules per cell and second.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Also shown are the growth rate as fraction per hour (dashed, right scale) and the cost of producing the enzyme E<sub>A</sub>, expressed in terms of the total resource expenditure (dotted line, right scale).</p>
<span>doi:10.1371/journal.pone.0000855.g003</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>When the nutrient is more abundant, the resulting rise in <em>C</em> will lead to greater enzyme production. Our model assumes a linear relationship between <em>C</em> and the protein synthesis rate for low-to-moderate <em>C</em>, but the dash-dotted line in <a href="#pone-0000855-g003">figure 3</a> indicates that with our model the enzyme production should approximately follow <em>C</em><sup>0.6</sup> to maximize the growth rate. The exact relationship is hardly significant, but in any case it seems unlikely that the overall rate of protein synthesis would be adjusted to meet the needs of a single protein, and it is up to <em>p</em><sub>E<sub>A</sub></sub> to bridge this gap.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>With higher enzyme concentrations it is primarily the diffusion of A into the cell that limits growth. There is a range of intermediate <em>A</em><sub>ext</sub> for which the optimal value of <em>p</em><sub>E<sub>A</sub></sub> varies relatively little, but with even higher <em>A</em><sub>ext</sub> and faster growth comes greater dilution of the enzyme, and this forces <em>p</em><sub>E<sub>A</sub></sub> to rise. If <em>A</em><sub>ext</sub> is higher still, the cell divides as fast as it possibly can and yet has much C to spare. As we do not take into account that protein synthesis may tie up machinery also needed for rapid growth and cell division, there is no incentive to keep <em>p</em><sub>E<sub>A</sub></sub> down in this limit. Thus, for very high <em>A</em><sub>ext</sub> the optimal value of <em>p</em><sub>E<sub>A</sub></sub> is the maximum allowed by the model. At the same time, the growth rate is insensitive to the value of <em>p</em><sub>E<sub>A</sub></sub>; when resources are plentiful, how they are spent is less important.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>We see that the fitness conferred by the metabolic pathway depends strongly on the enzyme production rate constant <em>p</em><sub>E<sub>A</sub></sub>, except if the nutrient level is always kept within a limited range. Such static conditions may apply to some obligate symbionts, which can indeed lack the ability to regulate genes involved in important metabolic processes. For example, bacterial symbionts in aphids have lost most of their transcriptional regulation of amino acid synthesis <a href="#pone.0000855-Moran1">[16]</a>, in a situation analogous to the one studied here in that the fitness gain from regulation must be weighed against the cost over a range of external conditions. For the vast majority of single-celled organisms, the environment can not be expected to be so constant. Selection pressure necessitates regulation of metabolic pathways.</p>


<h4>Transcriptional regulation</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To incorporate transcriptional regulation into the model, we add a transcription factor, T<sub>A</sub>, which like E<sub>A</sub> is produced from C. T<sub>A</sub> can act as an enhancer or repressor for the production of E<sub>A</sub>, with the sign and strength of the regulation as two independent parameters (see <a href="#s4">Materials and Methods</a>). However, T<sub>A</sub> can only function when activated by the presence of A. The fraction in the active form is directly determined by the level of A as compared to a half-maximum parameter. Now <em>p</em><sub>E<sub>A</sub></sub> is no longer a constant, but a function of the levels of T<sub>A</sub> and A. In addition to the dilution caused by growth, T<sub>A</sub> is degraded, albeit at a much lower rate than E<sub>A</sub>. It would be quite natural to add autoregulation to T<sub>A</sub>, to allow it to stabilize its level over a wider range of growth rates or possibly be more sensitive to <em>A</em><sub>ext</sub>, but doing so would make the results less transparent.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>By wiring and tuning a transcriptional network, evolution will tend to maximize the average growth rate over a succession of environmental conditions. In the limit where the environment very rarely changes, and the organism spends long time intervals in each setting, only the distribution of conditions matters, not the transitions between them. To keep things simple, we have defined the fitness of an organism as the mean of the normalized growth rate across an arbitrary selection of nutrient levels, as shown in <a href="#pone-0000855-g004">figure 4</a>. This fitness corresponds to the average growth rate when a similar number of generations is spent in each of these environments. In other words, the amount of time spent at each <em>A</em><sub>ext</sub> is inversely proportional to the dashed line in <a href="#pone-0000855-g003">figure 3</a>; not an entirely unreasonable first assumption.</p>
<div class="figure" id="pone-0000855-g004"><div class="img"><a name="pone-0000855-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000855" data-uri="info:doi/10.1371/journal.pone.0000855.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000855.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000855.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The behavior of a cell with well-tuned transcriptional regulation of the enzyme production.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The normalized growth rate (see <a href="#pone-0000855-g002">figure 2</a>) was optimized for those <em>A</em><sub>ext</sub> indicated by points, with an equal number of generations spent in each environment. The resulting system responds to <em>A</em><sub>ext</sub> as indicated by the thick curve. For comparison, the straight white line shows the best <em>p</em><sub>E<sub>A</sub></sub> in absence of regulation.</p>
<span>doi:10.1371/journal.pone.0000855.g004</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In our equations, the transcriptional network is defined by five parameters, and taking the view that evolution will have had ample time to globally optimize a system with so few parameters, we have used simulated annealing to pinpoint the parameter values that maximize the fitness measure. The thick line in <a href="#pone-0000855-g004">figure 4</a> shows the effect that transcriptional regulation has on <em>p</em><sub>E<sub>A</sub></sub>, and thereby on enzyme production, for the network of optimal fitness. The points mark the <em>A</em><sub>ext</sub> through which the fitness measure is defined, and the color filling the points indicates that the achieved fitness is almost indistinguishable from the maximal one, for those <em>A</em><sub>ext</sub> that we have optimized for. A brief explanation of how the evolved transcriptional network operates is in order.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>The transcription factor has become a repressor for the enzyme, and the downward slope for low-to-moderate <em>A</em><sub>ext</sub> is caused by the steadily more produced and activated T<sub>A</sub> repressing the expression of E<sub>A</sub>. Some transcriptional leakage occurs, and there is never that much T<sub>A</sub> present, so <em>p</em><sub>E<sub>A</sub></sub> is kept from falling too low. This explains how an increase in <em>A</em><sub>ext</sub> can result in a well-adjusted decrease in <em>p</em><sub>E<sub>A</sub></sub>. Perhaps more surprising, then, is the increase that <em>p</em><sub>E<sub>A</sub></sub> shows for high <em>A</em><sub>ext</sub>. The explanation: the time scale for decay and regeneration of T<sub>A</sub> is roughly one day, and when the generation length is shorter than that, the level of T<sub>A</sub> drops because of dilution.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>In all, the transcriptional network is extremely good at maximizing the fitness. Without regulation, the best possible fitness is 0.81 (with <em>p</em><sub>E<sub>A</sub></sub>≈1.8·10<sup>5</sup>), but with regulation the fitness rises above 0.99. Adding autoregulation to the transcription factor will further increase the fitness. We have also examined the effect of limiting the range of nutrient concentrations to between 100 µM and 1 mM. Then, the fitness scores of the best unregulated and regulated systems are 0.997 and 0.9996, respectively. Finally, forcing the regulatory mode to be activating was seen to substantially reduce the fitness score. Even with <em>A</em><sub>ext</sub> limited to values above 1 mM, the regulatory system was then incapable of giving a strong positive response to counter the dilution effect. We interpret this as a sign that the activating mode of regulation requires the transcription factor T<sub>A</sub> to be positively autoregulating, or its activation by A to be nonlinear, or both. In either case, there are added requirements on T<sub>A</sub>, which may make activators more difficult to evolve. In addition, it could them more sensitive to random mutations, which would quantitatively affect the results of <a href="#pone.0000855-Savageau3">[9]</a>.</p>
<a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>The difference in average growth rate due to regulation should be compared to the effect of adding the gene for the transcription factor to the genome of the organism, an effect not included in our model. Adding one gene should slow the growth rate by no more than one part in 1 000, and often far less (see <a href="#s4">Materials and Methods</a>). This slowdown is far outweighed by the boost that regulation brings, as long as the nutrient level is not too constant. We conclude that when the nutrient level varies, transcriptional regulation of the metabolic pathway carries great benefits for the organism.</p>


<h4>An additional nutrient</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Gene duplication plays a major role in the evolution of new functions. A duplicated enzyme or transcription factor will, if not lost, likely be subject to subfunctionalization <a href="#pone.0000855-Zhang1">[17]</a>, <a href="#pone.0000855-Lynch1">[18]</a>, a process that may in turn lead to neofunctionalization <a href="#pone.0000855-Lynch2">[19]</a>, <a href="#pone.0000855-Rastogi1">[20]</a>. To extend the model to the case where a second nutrient is present in the environment, we mimic a gene duplication event by adding a set of variables where “A” is replaced with “B”. Thus B is the alternative nutrient, turned into C by enzyme E<sub>B</sub>, whose production is governed by the rate constant <em>p</em><sub>E<sub>B</sub></sub>. Clearly C should then be seen as the first common point along the metabolic pathways of A and B. Generally, A and B need not be equally useful for producing C, and the parameters that describe E<sub>A</sub> and E<sub>B</sub> may be quite different. Such a state of affairs may, e.g., apply to the preference for glucose over fructose in yeast. Still, we will assume that all the relevant parameters are equal between A and B, because a small difference between them should only distort our conclusions slightly, for instance by shifting the point of preference for one nutrient over the other.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>With the introduction of the second nutrient comes an increase in the dimensionality of the state space that makes it impossible to visualize the full gamut of the system. However, we have observed that for any given environment the growth rate behaves nicely, with a single, rounded peak around some optimal <em>p</em><sub>E<sub>A</sub></sub> and <em>p</em><sub>E<sub>B</sub></sub> (data not shown). The optimum with respect to one parameter depends very weakly on the value of the other. It is therefore meaningful to keep <em>A</em><sub>ext</sub> and <em>p</em><sub>E<sub>A</sub></sub> fixed while studying the behavior as a function of <em>B</em><sub>ext</sub> and <em>p</em><sub>E<sub>B</sub></sub>. Under the assumptions of our model, it is not necessarily optimal for the organism to metabolize the most abundant nutrient only, in contrast to the optimal strategy in <a href="#pone.0000855-Thattai1">[3]</a>. The difference here is that when a nutrient is metabolized, its level is inside the cell drops due to the finite diffusion rate.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p><a href="#pone-0000855-g005">Figure 5</a> shows how the growth rate depends on <em>p</em><sub>E<sub>B</sub></sub> for different <em>B</em><sub>ext</sub>, with <em>A</em><sub>ext</sub> fixed at 0, 50 µM, or 1 mM and <em>p</em><sub>E<sub>A</sub></sub> optimized at <em>p</em><sub>E<sub>B</sub></sub> = 0 (to <em>p</em><sub>E<sub>A</sub></sub> = 0, <em>p</em><sub>E<sub>A</sub></sub>≈1.5·10<sup>5</sup>, or <em>p</em><sub>E<sub>A</sub></sub>≈9·10<sup>4</sup>, respectively, as per <a href="#pone-0000855-g003">figure 3</a>). As expected, the figure reveals that when little B is available, it is best for the cell not to bother with B at all. However, when <em>B</em><sub>ext</sub> comes within a factor of about ten of <em>A</em><sub>ext</sub>, the optimal <em>p</em><sub>E<sub>B</sub></sub> rises quickly, from zero to a value near where the optimum would be in the absence of A. That is, when <em>B</em><sub>ext</sub>≳<em>A</em><sub>ext</sub>, the dependence of <em>p</em><sub>E<sub>B</sub></sub> on <em>B</em><sub>ext</sub> is described by <a href="#pone-0000855-g005">figure 5(A)</a> (which is identical to <a href="#pone-0000855-g002">figure 2</a>). For other values of <em>A</em><sub>ext</sub> than those shown in <a href="#pone-0000855-g005">figure 5</a>, the only notable trend is that the transition region is narrower and closer to <em>A</em><sub>ext</sub> at low <em>A</em><sub>ext</sub>. An organism that is well adapted to a slowly changing environment will show a strong, often nonlinear response to the level of B in the region where 3≲<em>A</em><sub>ext</sub>/<em>B</em><sub>ext</sub>≲30.</p>
<div class="figure" id="pone-0000855-g005"><div class="img"><a name="pone-0000855-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000855" data-uri="info:doi/10.1371/journal.pone.0000855.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000855.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000855.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000855.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000855.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Growth rate as a function of the activation of a second metabolic pathway, presented as in <a href="#pone-0000855-g002">figure 2</a>.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>In each plot, the level of metabolite B is varied, while metabolite A is present at a constant level, and <em>p</em><sub>E<sub>A</sub></sub> is pegged at its optimal value in absence of B. The sharp transition of the optimal <em>p</em><sub>E<sub>B</sub></sub> in (B) signals a need for nonlinearity when <em>p</em><sub>E<sub>B</sub></sub> is actively regulated. The thick lines indicate what the simple transcriptional network of <a href="#pone-0000855-g001">figure 1(B)</a> can accomplish, when optimized over a wide range of environments, either emphasizing high (solid) or low (dashed) nutrient levels. Comparison with <a href="#pone-0000855-g004">figure 4</a> shows that it is far more difficult to achieve near-optimal regulation when the second metabolic pathway is added.</p>
<span>doi:10.1371/journal.pone.0000855.g005</span></div>

<h4>Bistability</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>After a change in the environmental conditions, and after transients have died down, the internal state of a cell may depend on the history of the cell, rather than on the new environmental conditions alone. Bistability, the coexistence of two stable fixed points in the dynamics, implies such hysteresis, because the initial state of the system generally determines which fixed point it goes to. In principle, stochastic effects always make the two states metastable, but the time scale for spontaneous switching may vary greatly. See <a href="#pone.0000855-Hasty1">[21]</a> for a review.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Evolution does not indiscriminately optimize the steady state growth rate. The environment changes from time to time, and if moving to a different operating region transiently carries a cost in the form of slowed growth, that cost must eventually be recuperated. Depending on the typical pattern of changes in the environment, the best solution for the individual cell may involve hysteresis. In clonal populations, phenotypic heterogeneity caused by bistability can be advantageous to the genome if it allows at least part of the population to survive a disaster <a href="#pone.0000855-Dubnau1">[22]</a>. Genes can thus hedge their bets by making individuals take risks, but only if a timely response is impossible is this a better strategy than for the cells to individually adapt to new conditions <a href="#pone.0000855-Thattai2">[23]</a>.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The <em>lac</em> operon of <em>E. coli</em> is a well-known example of a regulatory module tasked with deciding whether to make use of a specific metabolite. This system, in short, contains a positive feedback loop that activates lactose transport and metabolism when a metabolite of lactose is detected, but only if the glucose level in the environment is comparatively low. Experiments have shown that the <em>lac</em> system has the potential for bistable behavior when subjected to artificial inducers <a href="#pone.0000855-Ozbudak1">[24]</a>. This property of the system is appealing, and we can picture how it could fit into our results in <a href="#pone-0000855-g005">figure 5</a>. Although the optimal <em>p</em><sub>E<sub>B</sub></sub> does not follow an S-shaped curve, the steepness and width of the region where <em>p</em><sub>E<sub>B</sub></sub> switches are, at low <em>A</em><sub>ext</sub>, great enough to allow for bistability at a relatively low fitness cost in the steady state.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>However, it appears that the response of the <em>lac</em> operon to lactose itself, as opposed to an artificial inducer, is only a steeply graded monostable function, not a bistable one. A convincing explanation is that with time-varying lactose levels, a graded response provides significantly faster switching between operating modes <a href="#pone.0000855-vanHoek1">[10]</a>. Based on <a href="#pone-0000855-g005">figure 5</a>, we posit that a graded response is advantageous also when the lactose level varies very slowly, because unless the <em>lac</em> system is dissimilar to our model in some unforeseen way, the optimal <em>lac</em> expression is ever a smooth function. Consequently, we expect bistability in this context to be a rare phenomenon.</p>


<h4>Transcriptional regulation revisited</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Regulation of the production of the two enzymes, E<sub>A</sub> and E<sub>B</sub>, requires signaling from both A and B, and we have seen that some degree of mutual exclusivity is desirable. If a new transcription factor, T<sub>B</sub>, were to be created by duplication of an autoregulating T<sub>A</sub>, it might be prudent to retain a full set of regulations, with T<sub>A</sub> and T<sub>B</sub> regulating each other, themselves, and both enzymes. However, this would introduce an excessive number of parameters and obscure the issue of how easy it is to achieve beneficial transcriptional regulation. Therefore, we again use a stripped-down network, this time one that only includes regulation of E<sub>A</sub> by T<sub>A</sub>, of E<sub>B</sub> by T<sub>B</sub>, and mutually between T<sub>A</sub> and T<sub>B</sub>.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>In the same spirit as for the one-nutrient system, we have optimized the fitness of the regulatory network over two different sets of <em>A</em><sub>ext</sub> and <em>B</em><sub>ext</sub>, differing in their emphasis on high-nutrient or low-nutrient environments. The thick lines in <a href="#pone-0000855-g005">figure 5</a> illustrate how the resulting networks perform, which in either case certainly is better than with any constant value for <em>p</em><sub>E<sub>B</sub></sub>.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>When emphasis is placed on high-nutrient emphasis, the two transcription factors become mutual repressors, as one may have expected. This makes bistability possible even without autoregulation, but only when we add autoregulation to the model or modify the parameters do we see bistable behavior (data not shown). The way we assess the growth rate in each environment implicitly assumes monostability by denying the system a history (see <a href="#s4">Materials and Methods</a>), which is an extreme case of the idea outlined earlier: changes to the environment are infinitely rare, and bistability can evolve only because it never gets a chance to do any harm. Nevertheless, this emergence of bistability demonstrates a point: nontrivial behavior may appear even when not selected for, because of the constraints that evolution has to work with.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have developed a model for how the steady state growth rate depends on the activity of metabolic pathways, including their transcriptional control, in an idealized organism. This represents an implicit way to model the evolution of transcriptional networks subject to simple metabolic tasks.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>A key finding is that even when no alternative pathways exist, transcriptional regulation confers a substantial fitness advantage in all but the most static environments. In other words, transcriptional control is required for an organism to be competitive, even for a very simple metabolism. Furthermore, we have shown that this fitness advantage can be well exploited by a remarkably crude regulatory system, which relies on transcriptional repression. The observation that the mode of regulation correlates with the demand for expression for many metabolic genes <a href="#pone.0000855-Savageau2">[8]</a> is explained by Savageau from the perspective of resilience to mutations <a href="#pone.0000855-Savageau3">[9]</a>, but we believe that our results point to a complementary explanation, namely that negative regulation is easier to accomplish. For genes that are rarely used, it is relatively important to minimize the resources spent on the regulatory system, as opposed to fine-tuning the expression level and dynamical behavior, and this would tend to favor repression over activation.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>When two pathways are available, as would be the case when two catabolizable sugars are present, the highest growth rate is achieved by sharply activating the pathway of one nutrient, when the level of that nutrient nears the level of the other. At low nutrient concentrations this optimal response is distinctly nonlinear.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In line with previous work <a href="#pone.0000855-vanHoek1">[10]</a>, we have demonstrated that bistability is not a desirable feature of metabolic pathway regulation, at least not in a steady state limit. Nevertheless, optimizing the fitness of a regulatory system can cause bistability at some nutrient levels, suggesting that such complex behavior can emerge even when it in itself is unfavorable, merely because the underlying mutations have a positive net effect on fitness. This is reminiscent of, yet different from, how pathway complexity can be increased by the actions of the evolutionary mechanisms per se <a href="#pone.0000855-Soyer1">[25]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>We can also picture conditions that would promote bistable switching of metabolic pathways. From <a href="#pone-0000855-g005">figure 5</a>, one can imagine the existence of two distinct overall environmental states, whose respective ranges for the nutrient concentration overlap. If transitions between the two states are rare, and intra-state fluctuations of the nutrient level are too rapid for regulation to follow suit, bistability in the overlapping range can be expected to increase fitness, as it prevents needless state changes. Such a scheme could perhaps be applied to the lactose level in a chemostat with <em>E. coli</em> over many generations. As demonstrated in <a href="#pone.0000855-Dekel1">[1]</a>, the expression level of the <em>lac</em> genes can be significantly altered by fitness-raising mutations in a few hundred generations of growth in a constant lactose/glycerol medium. Thus it may well be possible to achieve bistability in the lactose response of <em>E. coli</em> through evolution in a laboratory setting.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Our model predicts that cells growing in a very low nutrient medium will have their nutrient uptake and catabolism up-regulated as compared to when the nutrient is more abundant. Such behavior has indeed been seen in yeast under glucose limited conditions <a href="#pone.0000855-Kolkman1">[26]</a>, where the hexose transporter HXT6 and the glycolysis enzymes HXK1 and GLK1, along with several other glycolysis proteins, were found to be significantly up-regulated. Our results predict that this effect only comes into play at sustained growth close to the long-term starvation limit, and only when there is no appreciable level of alternative substrates present.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Our predictions are primarily qualitative, not quantitative. For example, the exact location of the point where the model breaks down in <a href="#pone-0000855-g002">figure 2</a> should not be considered as realistic, as the degradation rate of the enzyme is higher than it would be in an organism adapted to very low nutrient levels. Still, a real microorganism may survive periods of starvation through sporulation and other mechanisms outside our model, but will eventually die if the nutrient levels are too low.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Material and Methods"></a><h3>Material and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>We here give the equations and parameters of the full model with two nutrients and transcriptional regulation. For the smaller models discussed in the text, parameters are set to zero as appropriate. C is the universal currency of our cell, and takes on the roles of energy carrier and building block. Being small molecules, A, B, and C have a mass of about <em>m</em><sub>A</sub> = 100 Da. The proteins E and T weigh in at 10<sup>3</sup><em>m</em><sub>A</sub>, and synthesis of one E or T consumes <em>s<sub>protein</sub></em> = 10<sup>4</sup> molecules of C. Let <em>C</em>, <em>A</em>, <em>E</em><sub>A</sub>, <em>T</em><sub>A</sub>, etc. denote the number of molecules in a cell of fixed volume <em>V<sub>cell</sub></em> = 1 µm<sup>3</sup>. Brownian motion sets a limit of about one collision per molecule pair per second, and this affects many parameters. For brevity we let <em>X</em> represent A or B where their equations are identical.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Transcription factor activation:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.e002&amp;representation=PNG" class="inline-graphic"></span><br>where <em>K<sub>X</sub></em>Є[10<sup>2</sup>,10<sup>6</sup>]<em>V</em><sup>−1</sup><em><sub>cell</sub></em> (or 1.7·10<sup>−7</sup>,10<sup>−3</sup>M), so activation can be made very sensitive to A or B.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Transcriptional regulation of enzyme production:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.e003&amp;representation=PNG" class="inline-graphic"></span><br>where the maximum rates <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.e004&amp;representation=PNG" class="inline-graphic"></span> are limited by ribosome count and speed. The parameters <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.e005&amp;representation=PNG" class="inline-graphic"></span> determine the type of regulation, ranging from strong repression to linear activation via leakiness and indifference. The strength of <em>T</em><sup>*</sup><em><sub>X</sub></em> as a regulator is <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.e006&amp;representation=PNG" class="inline-graphic"></span>, meaning that hundreds to thousands of TF molecules are needed for regulation. Similarly for the regulation of <em>p</em><sub>T<sub>A</sub></sub> by <em>p</em><sub>T<sub>B</sub></sub> and vice versa, <em>mutatis mutandis</em>.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Enzyme activity:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.e007&amp;representation=PNG" class="inline-graphic"></span><br>where <em>v<sub>X</sub></em> = 10<sup>3</sup> s<sup>−1</sup> and <em>K</em><sub>E</sub> = 10<sup>8</sup><em>V</em><sup>−1</sup><em><sub>cell</sub></em>≈1.7mM, which loosely means that one in 1 000 collisions between <em>X</em> and E<em><sub>X</sub></em> leads to a reaction that takes 1ns.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>Protein production, cell growth, and dilution:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.e008&amp;representation=PNG" class="inline-graphic"></span><br>where <em>K<sub>cell</sub></em> = 10<sup>8</sup><em>V</em><sup>−1</sup><em><sub>cell</sub></em> and <em>K<sub>prot</sub></em> = 10<sup>6</sup><em>V</em><sup>−1</sup><em><sub>cell</sub></em> reflect bottlenecks in cell replication and protein synthesis, s<em><sub>cell</sub></em> = 10<sup>10</sup><em>V</em><sup>−1</sup><em><sub>cell</sub></em> because a cell takes that many molecules of C to build, and <em>p<sub>cell</sub></em> = 10<sup>7</sup><em>V</em><sup>−1</sup><em><sub>cell</sub>s</em><sup>−1</sup> should give a maximum growth rate of about 20 minutes per generation.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>The rate equations:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.e009&amp;representation=PNG" class="inline-graphic"></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.e010&amp;representation=PNG" class="inline-graphic"></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.e011&amp;representation=PNG" class="inline-graphic"></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000855.e012&amp;representation=PNG" class="inline-graphic"></span><br>where <em>D<sub>X</sub></em> = 1 s<sup>−1</sup> lies between an upper limit set by diffusion and a lower limit set by the maximum growth rate, <em>d</em><sub>E</sub> = 10<sup>−3</sup> s<sup>−1</sup> makes E<em><sub>X</sub></em> rather short-lived but keeps its level reasonable even at the maximum production rate, and <em>d</em><sub>T</sub> = 10<sup>−5</sup> s<sup>−1</sup> is a realistic decay rate far slower than <em>d</em><sub>E</sub>.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>For an upper bound to the reproductive cost of adding an extra gene, consider a prokaryote for which the DNA replication rate is limiting. If this rate is proportional to the genome size, and there are 1 000 genes, the fitness cost is on the order of 10<sup>−3</sup>. By comparison, the cost of synthesizing the extra DNA is closer to 10<sup>−5</sup><em>s<sub>cell</sub></em> .</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>To find the growth rate in the equations' steady state, we performed a numeric integration from the state with all variables zero except <em>C</em> = 10<sup>8</sup><em>V</em><sup>−1</sup><em><sub>cell</sub></em>, using the bsimp stepper from GSL (<a href="http://www.gnu.org/software/gsl/">http://www.gnu.org/software/gsl/</a>). For <a href="#pone-0000855-g005">figure 5</a> we probed for bistability by gradually going from low to high <em>B</em><sub>ext</sub>, and vice versa.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>CT acknowledges support from the Swedish National Research School in Genomics and Bioinformatics. DA was supported by the Knut and Alice Wallenberg foundation. CP and MK were supported by the Swedish Foundation for Strategic Research (Senior Individual Grant). MK was also supported by the Knut and Alice Wallenberg Foundation through the Swegene consortium and the Strategic Science Foundation (Lund Strategic Center for Clinical Cancer Research). We thank Björn Samuelsson for assistance with the figures, and are grateful to an anonymous referee for many helpful comments.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: CP MK CT. Wrote the paper: CP MK CT DA. Other: Provided biological knowledge: DA. Performed the simulations: CT.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000855-Dekel1" id="pone.0000855-Dekel1"></a>Dekel E, Alon U (2005) Optimality and evolutionary tuning of the expression level of a protein. Nature  436: 588–592.  <ul class="find" data-citedArticleID="988714" data-doi="10.1038/nature03842"><li><a href="http://dx.doi.org/10.1038/nature03842" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimality+and+evolutionary+tuning+of+the+expression+level+of+a+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimality+and+evolutionary+tuning+of+the+expression+level+of+a+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000855-Edwards1" id="pone.0000855-Edwards1"></a>Edwards JS, Ibarra RU, Palsson BO (2001) In silico predictions of <em>Escherichia coli</em> metabolic capabilities are consistent with experimental data. Nat Biotech  19: 125–130.  <ul class="find" data-citedArticleID="988718"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+silico+predictions+of+Escherichia+coli+metabolic+capabilities+are+consistent+with+experimental+data.&amp;auth=&amp;atitle=In+silico+predictions+of+Escherichia+coli+metabolic+capabilities+are+consistent+with+experimental+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+silico+predictions+of+Escherichia+coli+metabolic+capabilities+are+consistent+with+experimental+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+silico+predictions+of+Escherichia+coli+metabolic+capabilities+are+consistent+with+experimental+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000855-Thattai1" id="pone.0000855-Thattai1"></a>Thattai M, Shraiman BI (2003) Metabolic switching in the sugar phosphotransferase system of Escherichia coli. Biophys J  85: 744–754.  <ul class="find" data-citedArticleID="988750" data-doi="10.1016/s0006-3495(03)74517-2"><li><a href="http://dx.doi.org/10.1016/s0006-3495(03)74517-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolic+switching+in+the+sugar+phosphotransferase+system+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolic+switching+in+the+sugar+phosphotransferase+system+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000855-Savageau1" id="pone.0000855-Savageau1"></a>Savageau MA (1998) Demand theory of gene regulation. I. Quantitative development of the theory. Genetics  149: 1665–1676.  <ul class="find" data-citedArticleID="988740"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Demand+theory+of+gene+regulation.+I.+Quantitative+development+of+the+theory.&amp;auth=&amp;atitle=Demand+theory+of+gene+regulation.+I.+Quantitative+development+of+the+theory." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Demand+theory+of+gene+regulation.+I.+Quantitative+development+of+the+theory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Demand+theory+of+gene+regulation.+I.+Quantitative+development+of+the+theory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000855-Wilke1" id="pone.0000855-Wilke1"></a>Wilke CO, Wang JL, Ofria C, Lenski RE, Adami C (2001) Evolution of digital organisms at high mutation rates leads to survival of the flattest. Nature  412: 331–333.  <ul class="find" data-citedArticleID="988754" data-doi="10.1038/35085569"><li><a href="http://dx.doi.org/10.1038/35085569" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+digital+organisms+at+high+mutation+rates+leads+to+survival+of+the+flattest." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+digital+organisms+at+high+mutation+rates+leads+to+survival+of+the+flattest.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000855-Novick1" id="pone.0000855-Novick1"></a>Novick A, Weiner M (1957) Enzyme induction as an all-or-none phenomenon. Proc Natl Acad Sci USA  43: 553–566.  <ul class="find" data-citedArticleID="988734" data-doi="10.1073/pnas.43.7.553"><li><a href="http://dx.doi.org/10.1073/pnas.43.7.553" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enzyme+induction+as+an+all-or-none+phenomenon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enzyme+induction+as+an+all-or-none+phenomenon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000855-Koch1" id="pone.0000855-Koch1"></a>Koch AL (1983) The protein burden of <em>lac</em> operon products. J Mol Evol  19: 455–462.  <ul class="find" data-citedArticleID="988722" data-doi="10.1007/bf02102321"><li><a href="http://dx.doi.org/10.1007/bf02102321" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+protein+burden+of+lac+operon+products." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+protein+burden+of+lac+operon+products.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000855-Savageau2" id="pone.0000855-Savageau2"></a>Savageau MA (1977) Design of molecular control mechanisms and the demand for gene expression. Proc Natl Acad Sci USA  74: 5647–5651.  <ul class="find" data-citedArticleID="988742" data-doi="10.1073/pnas.74.12.5647"><li><a href="http://dx.doi.org/10.1073/pnas.74.12.5647" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Design+of+molecular+control+mechanisms+and+the+demand+for+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Design+of+molecular+control+mechanisms+and+the+demand+for+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000855-Savageau3" id="pone.0000855-Savageau3"></a>Savageau M (1998) Demand theory of gene regulation. II. Quantitative application to the lactose and maltose operons of <em>Escherichia coli</em>. Genetics  149: 1677–1691.  <ul class="find" data-citedArticleID="988744"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Demand+theory+of+gene+regulation.+II.+Quantitative+application+to+the+lactose+and+maltose+operons+of+Escherichia+coli.&amp;auth=&amp;atitle=Demand+theory+of+gene+regulation.+II.+Quantitative+application+to+the+lactose+and+maltose+operons+of+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Demand+theory+of+gene+regulation.+II.+Quantitative+application+to+the+lactose+and+maltose+operons+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Demand+theory+of+gene+regulation.+II.+Quantitative+application+to+the+lactose+and+maltose+operons+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000855-vanHoek1" id="pone.0000855-vanHoek1"></a>van Hoek MJA, Hogeweg P (2006) In silico evolved <em>lac</em> operons exhibit bistability for artificial inducers, but not for lactose. Biophys J  91: 2833–2843.  <ul class="find" data-citedArticleID="988758" data-doi="10.1529/biophysj.105.077420"><li><a href="http://dx.doi.org/10.1529/biophysj.105.077420" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+silico+evolved+lac+operons+exhibit+bistability+for+artificial+inducers%2C+but+not+for+lactose." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+silico+evolved+lac+operons+exhibit+bistability+for+artificial+inducers%2C+but+not+for+lactose.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000855-Covert1" id="pone.0000855-Covert1"></a>Covert MW, Palsson BO (2002) Transcriptional regulation in constraints-based metabolic models of <em>Escherichia coli</em>. J Biol Chem  277: 28058–28064.  <ul class="find" data-citedArticleID="988712" data-doi="10.1074/jbc.m201691200"><li><a href="http://dx.doi.org/10.1074/jbc.m201691200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulation+in+constraints-based+metabolic+models+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulation+in+constraints-based+metabolic+models+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000855-Shoemaker1" id="pone.0000855-Shoemaker1"></a>Shoemaker J, Reeves GT, Gupta S, Pilyugin SS, Egli T, et al.  (2003) The dynamics of single-substrate continuous cultures: the role of transport enzymes. J Theor Biol  222:   <ul class="find" data-citedArticleID="988746" data-doi="10.1016/s0022-5193(03)00036-5"><li><a href="http://dx.doi.org/10.1016/s0022-5193(03)00036-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dynamics+of+single-substrate+continuous+cultures%3A+the+role+of+transport+enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dynamics+of+single-substrate+continuous+cultures%3A+the+role+of+transport+enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000855-Briggs1" id="pone.0000855-Briggs1"></a>Briggs GE, Haldane JBS (1925) A note on the kinetics of enzyme action. Biochem J  19: 338–339.  <ul class="find" data-citedArticleID="988710"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+note+on+the+kinetics+of+enzyme+action.&amp;auth=&amp;atitle=A+note+on+the+kinetics+of+enzyme+action." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+note+on+the+kinetics+of+enzyme+action." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+note+on+the+kinetics+of+enzyme+action.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000855-Nelson1" id="pone.0000855-Nelson1"></a>Nelson P (2004) Biological Physics. New York: W. H. Freeman, first edition.   <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0000855-Barford1" id="pone.0000855-Barford1"></a>Barford JP, Phillips PJ, Orlowski JH (1992) A new model of uptake of multiple sugars by <em>S. cerevisiae</em>. Bioprocess Eng  7: 297–302.  <ul class="find" data-citedArticleID="988708" data-doi="10.1007/bf00705158"><li><a href="http://dx.doi.org/10.1007/bf00705158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+model+of+uptake+of+multiple+sugars+by+S.+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+model+of+uptake+of+multiple+sugars+by+S.+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000855-Moran1" id="pone.0000855-Moran1"></a>Moran NA, Dunbar HE, Wilcox JL (2005) Regulation of transcription in a reduced bacterial genome: nutrient-provisioning genes of the obligate symbiont <em>Buchnera aphidicola</em>. J Bacteriol  187: 4229–4237.  <ul class="find" data-citedArticleID="988730" data-doi="10.1128/jb.187.12.4229-4237.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.12.4229-4237.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+transcription+in+a+reduced+bacterial+genome%3A+nutrient-provisioning+genes+of+the+obligate+symbiont+Buchnera+aphidicola." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+transcription+in+a+reduced+bacterial+genome%3A+nutrient-provisioning+genes+of+the+obligate+symbiont+Buchnera+aphidicola.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000855-Zhang1" id="pone.0000855-Zhang1"></a>Zhang J (2003) Evolution by gene duplication: an update. Trends Ecol Evol  18: 292–298.  <ul class="find" data-citedArticleID="988756" data-doi="10.1016/s0169-5347(03)00033-8"><li><a href="http://dx.doi.org/10.1016/s0169-5347(03)00033-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+by+gene+duplication%3A+an+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+by+gene+duplication%3A+an+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000855-Lynch1" id="pone.0000855-Lynch1"></a>Lynch M, Conery JS (2000) The evolutionary fate and consequences of duplicate genes. Science  290: 1151–1155.  <ul class="find" data-citedArticleID="988726" data-doi="10.1126/science.290.5494.1151"><li><a href="http://dx.doi.org/10.1126/science.290.5494.1151" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+fate+and+consequences+of+duplicate+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+fate+and+consequences+of+duplicate+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000855-Lynch2" id="pone.0000855-Lynch2"></a>Lynch M, Force A (2005) The probability of duplicate gene preservation by subfunctionalization. Genetics  154: 459–473.  <ul class="find" data-citedArticleID="988728"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+probability+of+duplicate+gene+preservation+by+subfunctionalization.&amp;auth=&amp;atitle=The+probability+of+duplicate+gene+preservation+by+subfunctionalization." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+probability+of+duplicate+gene+preservation+by+subfunctionalization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+probability+of+duplicate+gene+preservation+by+subfunctionalization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000855-Rastogi1" id="pone.0000855-Rastogi1"></a>Rastogi S, Liberles DA (2005) Subfunctionalization of duplicated genes as a transition state to neofunctionalization. BMC Evol Biol  5: 28.  <ul class="find" data-citedArticleID="988738"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Subfunctionalization+of+duplicated+genes+as+a+transition+state+to+neofunctionalization.&amp;auth=&amp;atitle=Subfunctionalization+of+duplicated+genes+as+a+transition+state+to+neofunctionalization." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subfunctionalization+of+duplicated+genes+as+a+transition+state+to+neofunctionalization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subfunctionalization+of+duplicated+genes+as+a+transition+state+to+neofunctionalization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000855-Hasty1" id="pone.0000855-Hasty1"></a>Hasty J, McMillen D, Isaacs F, Collins JJ (2001) Computational studies of gene regulatory networks: <em>in numero</em> molecular biology. Nat Rev Genet  2: 268–279.  <ul class="find" data-citedArticleID="988720" data-doi="10.1038/35066056"><li><a href="http://dx.doi.org/10.1038/35066056" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+studies+of+gene+regulatory+networks%3A+in+numero+molecular+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+studies+of+gene+regulatory+networks%3A+in+numero+molecular+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000855-Dubnau1" id="pone.0000855-Dubnau1"></a>Dubnau D, Losick R (2006) Bistability in bacteria. Mol Microbiol  61: 564–572.  <ul class="find" data-citedArticleID="988716" data-doi="10.1111/j.1365-2958.2006.05249.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2006.05249.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bistability+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bistability+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000855-Thattai2" id="pone.0000855-Thattai2"></a>Thattai M, van Oudenaarden A (2004) Stochastic gene expression in fluctuating environments. Genetics  167: 523–530.  <ul class="find" data-citedArticleID="988752" data-doi="10.1534/genetics.167.1.523"><li><a href="http://dx.doi.org/10.1534/genetics.167.1.523" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stochastic+gene+expression+in+fluctuating+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stochastic+gene+expression+in+fluctuating+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000855-Ozbudak1" id="pone.0000855-Ozbudak1"></a>Ozbudak EM, Thattai M, Lim HN, Shraiman BI, van Oudenaarden A (2004) Multistability in the lactose utilization network of <em>Escherichia coli</em>. Nature  427: 737–740.  <ul class="find" data-citedArticleID="988736" data-doi="10.1038/nature02298"><li><a href="http://dx.doi.org/10.1038/nature02298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multistability+in+the+lactose+utilization+network+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multistability+in+the+lactose+utilization+network+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000855-Soyer1" id="pone.0000855-Soyer1"></a>Soyer OS, Bonhoeffer S (2006) Evolution of complexity in signaling pathways. Proc Natl Acad Sci USA  104: 16337–16342.  <ul class="find" data-citedArticleID="988748" data-doi="10.1073/pnas.0604449103"><li><a href="http://dx.doi.org/10.1073/pnas.0604449103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+complexity+in+signaling+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+complexity+in+signaling+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000855-Kolkman1" id="pone.0000855-Kolkman1"></a>Kolkman A, Daran-Lapujade P, Fullaondo A, Olsthoorn MM, Pronk JT, et al.  (2006) Proteome analysis of yeast response to various nutrient limitations. Mol Syst Biol  2: 2006.0026.  <ul class="find" data-citedArticleID="988724" data-doi="10.1038/msb4100069"><li><a href="http://dx.doi.org/10.1038/msb4100069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteome+analysis+of+yeast+response+to+various+nutrient+limitations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteome+analysis+of+yeast+response+to+various+nutrient+limitations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.XML" value="74274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.PDF" value="319240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g001.PNG_L" value="76888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g001.PNG_M" value="65457"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g001.PNG_S" value="8022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g001.TIF" value="369236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g001.PNG_I" value="24818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e001.PNG" value="7172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e001.TIF" value="23128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g002.PNG_L" value="112870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g002.PNG_M" value="137145"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g002.PNG_S" value="13759"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g002.TIF" value="274218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g002.PNG_I" value="50500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g003.PNG_L" value="38354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g003.PNG_M" value="42891"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g003.PNG_S" value="10253"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g003.TIF" value="70430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g003.PNG_I" value="13983"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g004.PNG_L" value="118179"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g004.PNG_M" value="143266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g004.PNG_S" value="14444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g004.TIF" value="290280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g004.PNG_I" value="52995"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g005.PNG_L" value="304164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g005.PNG_M" value="105797"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g005.PNG_S" value="24652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g005.TIF" value="773140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.g005.PNG_I" value="200597"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e002.PNG" value="8009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e002.TIF" value="23824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e003.PNG" value="12034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e003.TIF" value="27632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e004.PNG" value="7500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e004.TIF" value="23528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e005.PNG" value="7236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e005.TIF" value="23276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e006.PNG" value="8110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e006.TIF" value="24124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e007.PNG" value="8914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e007.TIF" value="24704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e008.PNG" value="11688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e008.TIF" value="27236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e009.PNG" value="9433"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e009.TIF" value="25172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e010.PNG" value="9794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e010.TIF" value="25480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e011.PNG" value="9535"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e011.TIF" value="25280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e012.PNG" value="10409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000855.e012.TIF" value="26112"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000855&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000855" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000855&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000855&volume=&issue=&title=Is Transcriptional Regulation of Metabolic Pathways an Optimal Strategy for Fitness?&author_name=Carl%20Troein%2C%20Dag%20Ahr%C3%A9n%2C%20Morten%20Krogh%2C%20Carsten%20Peterson&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000855" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000855" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000855" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000855&amp;t=Is%20Transcriptional%20Regulation%20of%20Metabolic%20Pathways%20an%20Optimal%20Strategy%20for%20Fitness%3F" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000855&title=Is%20Transcriptional%20Regulation%20of%20Metabolic%20Pathways%20an%20Optimal%20Strategy%20for%20Fitness%3F&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000855&amp;title=Is%20Transcriptional%20Regulation%20of%20Metabolic%20Pathways%20an%20Optimal%20Strategy%20for%20Fitness%3F" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000855" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000855&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Is Transcriptional Regulation of Metabolic Pathways an Optimal Strategy for Fitness?';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000855';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Is%20Transcriptional%20Regulation%20of%20Metabolic%20Pathways%20an%20Optimal%20Strategy%20for%20Fitness%3F http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000855" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000855" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzyme+metabolism%22" title="Search for articles in the subject area:'Enzyme metabolism'"><div class="flagText">Enzyme metabolism</div></a>
              <div data-categoryid="20267" data-articleid="24548"
                   data-categoryname="Enzyme metabolism"
                   class="flagImage" title="Flag 'Enzyme metabolism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzyme+regulation%22" title="Search for articles in the subject area:'Enzyme regulation'"><div class="flagText">Enzyme regulation</div></a>
              <div data-categoryid="19973" data-articleid="24548"
                   data-categoryname="Enzyme regulation"
                   class="flagImage" title="Flag 'Enzyme regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzymes%22" title="Search for articles in the subject area:'Enzymes'"><div class="flagText">Enzymes</div></a>
              <div data-categoryid="17863" data-articleid="24548"
                   data-categoryname="Enzymes"
                   class="flagImage" title="Flag 'Enzymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24548"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="24548"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lactose%22" title="Search for articles in the subject area:'Lactose'"><div class="flagText">Lactose</div></a>
              <div data-categoryid="34563" data-articleid="24548"
                   data-categoryname="Lactose"
                   class="flagImage" title="Flag 'Lactose' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Metabolic+pathways%22" title="Search for articles in the subject area:'Metabolic pathways'"><div class="flagText">Metabolic pathways</div></a>
              <div data-categoryid="20601" data-articleid="24548"
                   data-categoryname="Metabolic pathways"
                   class="flagImage" title="Flag 'Metabolic pathways' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcription+factors%22" title="Search for articles in the subject area:'Transcription factors'"><div class="flagText">Transcription factors</div></a>
              <div data-categoryid="17335" data-articleid="24548"
                   data-categoryname="Transcription factors"
                   class="flagImage" title="Flag 'Transcription factors' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1311'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3688'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1762&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=10207">Testing the hypothesis: metabolic vs. non-metabolic regulators</a><br>Posted by mthattai</p>
      <p><a href="/annotation/listThread.action?root=42851">Valuable insights into evolution of regulation</a><br>Posted by clementkent</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
