

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Normal-Mode-Analysis–Monitored Energy Minimization Procedure for Generating Small–Molecule Bound Conformations</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001025"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001025" />

    <meta name="citation_title" content="Normal-Mode-Analysis–Monitored Energy Minimization Procedure for Generating Small–Molecule Bound Conformations"/>
    <meta itemprop="name" content="Normal-Mode-Analysis–Monitored Energy Minimization Procedure for Generating Small–Molecule Bound Conformations"/>

      <meta name="citation_author" content="Qi Wang"/>
            <meta name="citation_author_institution" content="Computer–Aided Molecular Design Laboratory, Mayo Clinic, Rochester, Minnesota, United States of America"/>
      <meta name="citation_author" content="Yuan-Ping Pang"/>
            <meta name="citation_author_institution" content="Computer–Aided Molecular Design Laboratory, Mayo Clinic, Rochester, Minnesota, United States of America"/>

    <meta name="citation_date" content="2007/10/10"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001025.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1025"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The key-lock theory and the induced fit theory.; citation_author=DE Koshland; citation_journal_title=Angew Chem Int Ed Engl; citation_volume=33; citation_number=1; citation_pages=2375-2378; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Conformational changes and drug action.; citation_author=AS Burgen; citation_journal_title=Fed Proc; citation_volume=40; citation_number=2; citation_pages=2723-2728; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Conformational induction versus conformational selection: evidence from allosteric enhancers.; citation_author=RF Bruns; citation_journal_title=Trends Pharmacol Sci; citation_volume=17; citation_number=3; citation_pages=189; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Receptor conformational induction versus selection: all part of the same energy landscape, agonists can differentially stabilize multiple active states of receptors.; citation_author=T Kenakin; citation_journal_title=Trends Pharmacol Sci; citation_volume=17; citation_number=4; citation_pages=190-191; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Docking studies on the complexed and uncomplexed FKBP12 structures with bound and unbound ligands: an implication of conformational selection mechanism for binding.; citation_author=Y-P Pang; citation_author=ND Silva; citation_author=C Hydock; citation_author=FG Prendergast; citation_journal_title=J Mol Model; citation_volume=3; citation_number=5; citation_pages=240-248; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=EUDOC: a computer program for identification of drug interaction sites in macromolecules and drug leads from chemical databases.; citation_author=Y-P Pang; citation_author=E Perola; citation_author=K Xu; citation_author=FG Prendergast; citation_journal_title=J Comput Chem; citation_volume=22; citation_number=6; citation_pages=1750-1771; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=In silico drug discovery: solving the “target-rich and lead-poor” imbalance using the genome-to-drug-lead paradigm.; citation_author=Y-P Pang; citation_journal_title=Clin Pharmacol Ther; citation_volume=81; citation_number=7; citation_pages=30-34; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=EUDOC on Blue Gene: accelerating the transfer of drug discoveries from laboratory to patient.; citation_author=Y-P Pang; citation_author=T Mullins; citation_author=B Swartz; citation_author=J McAllister; citation_author=B Smith; citation_journal_title=IBM J Res Dev in press; citation_number=8; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Preference of small molecules for local minimum conformations when binding to proteins.; citation_author=Q Wang; citation_author=Y-P Pang; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=9; citation_pages=e820; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Normal mode theory and harmonic potential approximations;; citation_author=K Hinsen; citation_number=10; citation_pages=1-16; citation_date=2006; citation_publisher=Chapman & Hall/CRC; " />
      <meta name="citation_reference" content="citation_title=Rigidity theory and applications;; citation_author=D Case; citation_number=11; citation_pages=329-344; citation_date=1999; citation_publisher=Kluwer Academic/Plenum Publishers; " />
      <meta name="citation_reference" content="citation_title=Introduction to computational chemistry.; citation_author=F Jensen; citation_number=12; citation_pages=312-315; citation_date=1998; citation_publisher=John Wiley & Sons; " />
      <meta name="citation_reference" content="citation_title=Essentials of computational chemistry.; citation_author=CJ Cramer; citation_number=13; citation_pages=303 p; citation_date=2002; citation_publisher=John Wiley & Sons; " />
      <meta name="citation_reference" content="citation_title=Accurate reproduction of 161 small-molecule complex crystal structures using the EUDOC program: expanding the use of EUDOC to supramolecular chemistry.; citation_author=Q Wang; citation_author=Y-P Pang; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=14; citation_pages=e531; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Cambridge Structural Database: a quarter of a million crystal structures and rising.; citation_author=FH Allen; citation_journal_title=Acta Crystallogr Sect B: Struct Sci; citation_volume=58; citation_number=15; citation_pages=380-388; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Systematic search in conformational analysis.; citation_author=DD Beusen; citation_author=EFB Shands; citation_author=SF Karasek; citation_author=GR Marshall; citation_author=RA Dammkoehler; citation_journal_title=J Mol Struc (Theochem); citation_volume=370; citation_number=16; citation_pages=157-171; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A survey of methods for searching the conformational space of small and medium-sized molecules.; citation_author=AR Leach; citation_number=17; citation_pages=1-55; citation_date=1991; citation_publisher=Reviews in Computational Chemistry: John Wiley & Sons, Inc; " />
      <meta name="citation_reference" content="citation_title=Conformational energy penalties of protein-bound ligands.; citation_author=J Boström; citation_author=P-O Norrby; citation_author=T Liljefors; citation_journal_title=J Comput-Aided Mol Design; citation_volume=12; citation_number=18; citation_pages=383-396; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Sampling conformational hyperspace: techniques for improving completeness.; citation_author=RA Dammkoehler; citation_author=SF Karasek; citation_author=EF Shands; citation_author=GR Marshall; citation_journal_title=J Comput Aided Mol Design; citation_volume=9; citation_number=19; citation_pages=491-499; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Locally accessible conformations of proteins - multiple molecular dynamics simulations of crambin.; citation_author=LSD Caves; citation_author=JD Evanseck; citation_author=M Karplus; citation_journal_title=Protein Sci; citation_volume=7; citation_number=20; citation_pages=649-666; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Assessing equilibration and convergence in biomolecular simulations.; citation_author=LJ Smith; citation_author=X Daura; citation_author=WF van Gunsteren; citation_journal_title=Proteins; citation_volume=48; citation_number=21; citation_pages=487-496; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Absolute comparison of simulated and experimental protein-folding dynamics.; citation_author=CD Snow; citation_author=N Nguyen; citation_author=VS Pande; citation_author=M Gruebele; citation_journal_title=Nature; citation_volume=420; citation_number=22; citation_pages=102-106; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Simulation of folding of a small alpha-helical protein in atomistic detail using worldwide-distributed computing.; citation_author=B Zagrovic; citation_author=CD Snow; citation_author=MR Shirts; citation_author=VS Pande; citation_journal_title=J Mol Biol; citation_volume=323; citation_number=23; citation_pages=927-937; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Modeling domino effects in enzymes: molecular basis of the substrate specificity of the bacterial metallo-beta-lactamases IMP-1 and IMP-6.; citation_author=P Oelschlaeger; citation_author=RD Schmid; citation_author=J Pleiss; citation_journal_title=Biochemistry; citation_volume=42; citation_number=24; citation_pages=8945-8956; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Three-dimensional model of a substrate-bound SARS chymotrypsin-like cysteine proteinase predicted by multiple molecular dynamics simulations: catalytic efficiency regulated by substrate binding.; citation_author=Y-P Pang; citation_journal_title=Proteins; citation_volume=57; citation_number=25; citation_pages=747-757; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Application of the multimolecule and multiconformational resp methodology to biopolymers: charge derivation for DNA, RNA, and proteins.; citation_author=P Cieplak; citation_author=WD Cornell; citation_author=C Bayly; citation_author=PA Kollman; citation_journal_title=J Comput Chem; citation_volume=16; citation_number=26; citation_pages=1357-1377; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=GAUSSIAN 98, Revision A.7.; citation_author=MJ Frisch; citation_author=GW Trucks; citation_author=HB Schlegel; citation_author=PMW Gill; citation_author=BG Hohnson; citation_number=27; citation_date=1999; citation_publisher=Gaussian, Inc; " />
      <meta name="citation_reference" content="citation_title=AMBER, a package of computer programs for applying molecular mechanics, normal mode analysis, molecular dynamics and free energy calculations to simulate the structural and energetic properties of molecules.; citation_author=DA Pearlman; citation_author=DA Case; citation_author=JW Caldwell; citation_author=WS Ross; citation_author=TE Cheatham; citation_journal_title=Comput Phys Commun; citation_volume=91; citation_number=28; citation_pages=1-41; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=A second generation force field for the simulation of proteins, nucleic acids, and organic molecules.; citation_author=WD Cornell; citation_author=P Cieplak; citation_author=CI Bayly; citation_author=IR Gould; citation_author=KM Merz; citation_journal_title=J Am Chem Soc; citation_volume=117; citation_number=29; citation_pages=5179-5197; citation_date=1995; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001025" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Normal-Mode-Analysis–Monitored Energy Minimization Procedure for Generating Small–Molecule Bound Conformations"/>
    <meta name="twitter:description" content="The energy minimization of a small molecule alone does not automatically stop at a local minimum of the potential energy surface of the molecule if the minimum is shallow, thus leading to folding of the molecule and consequently hampering the generation of the bound conformation of a guest in the absence of its host. This questions the practicality of virtual screening methods that use conformations at local minima of their potential energy surfaces (local minimum conformations) as potential bound conformations. Here we report a normal-mode-analysis&ndash;monitored energy minimization (NEM) procedure that generates local minimum conformations as potential bound conformations. Of 22 selected guest&ndash;host complex crystal structures with guest structures possessing up to four rotatable bonds, all complexes were reproduced, with guest mass&ndash;weighted root mean square deviations of &lt;1.0 &Aring;, through docking with the NEM&ndash;generated guest local minimum conformations. An analysis of the potential energies of these local minimum conformations showed that 22 (100%), 18 (82%), 16 (73%), and 12 (55%) of the 22 guest bound conformations in the crystal structures had conformational strain energies of less than or equal to 3.8, 2.0, 0.6, and 0.0 kcal/mol, respectively. These results suggest that (1) the NEM procedure can generate small&ndash;molecule bound conformations, and (2) guests adopt low-strain&ndash;energy conformations for complexation, thus supporting the virtual screening methods that use local minimum conformations."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001025.g005"/>

  <meta property="og:title" content="Normal-Mode-Analysis–Monitored Energy Minimization Procedure for Generating Small–Molecule Bound Conformations" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001025" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4618'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5319'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4739&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001025">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001025" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001025&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Normal-Mode-Analysis–Monitored Energy Minimization Procedure for Generating Small–Molecule Bound Conformations
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Qi Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computer–Aided Molecular Design Laboratory, Mayo Clinic, Rochester, Minnesota, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yuan-Ping Pang
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pang@mayo.edu">pang@mayo.edu</a></p>

                <p>Affiliation:
                  Computer–Aided Molecular Design Laboratory, Mayo Clinic, Rochester, Minnesota, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 10, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001025</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001025" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001025" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001025" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001025" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001025-g001" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001025-g002" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001025-g003" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001025-g004" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001025-g005" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001025-t001" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001025-t002" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001025">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001025" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The energy minimization of a small molecule alone does not automatically stop at a local minimum of the potential energy surface of the molecule if the minimum is shallow, thus leading to folding of the molecule and consequently hampering the generation of the bound conformation of a guest in the absence of its host. This questions the practicality of virtual screening methods that use conformations at local minima of their potential energy surfaces (local minimum conformations) as potential bound conformations. Here we report a normal-mode-analysis–monitored energy minimization (NEM) procedure that generates local minimum conformations as potential bound conformations. Of 22 selected guest–host complex crystal structures with guest structures possessing up to four rotatable bonds, all complexes were reproduced, with guest mass–weighted root mean square deviations of &lt;1.0 Å, through docking with the NEM–generated guest local minimum conformations. An analysis of the potential energies of these local minimum conformations showed that 22 (100%), 18 (82%), 16 (73%), and 12 (55%) of the 22 guest bound conformations in the crystal structures had conformational strain energies of less than or equal to 3.8, 2.0, 0.6, and 0.0 kcal/mol, respectively. These results suggest that (1) the NEM procedure can generate small–molecule bound conformations, and (2) guests adopt low-strain–energy conformations for complexation, thus supporting the virtual screening methods that use local minimum conformations.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wang Q, Pang Y-P (2007) Normal-Mode-Analysis–Monitored Energy Minimization Procedure for Generating Small–Molecule Bound Conformations. PLoS ONE 2(10):
          e1025.
            doi:10.1371/journal.pone.0001025</p><p><strong>Academic Editor: </strong>Mark Isalan, Center for Genomic Regulation, Spain</p><p><strong>Received:</strong> July 16, 2007; <strong>Accepted:</strong> September 25, 2007; <strong>Published:</strong> October 10, 2007</p><p><strong>Copyright:</strong> © 2007 Wang, Pang. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the U.S. Army Medical Research Acquisition Activity (W81XWH-04-2-0001), the National Institutes of Health (5R01AI054574-03 and 5R01GM061300-06), Mayo Graduate School, and the Mayo Foundation for Medical Education and Research.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Molecular complexation in biology is best described by the conformational induction theory <a href="#pone.0001025-Koshland1">[1]</a>—namely, a guest binds initially to a less compatible conformation of its host and then adjusts its conformation to induce the most compatible conformation of the host. The conformation induction theory is not ideal for computationally addressing the conformational flexibility of both guest and host in docking studies, however, because computing the mutually dependent conformational changes of both partners on the fly is time–consuming and unsuitable for parallel computing. Alternatively, the conformation selection theory describes that both guest and host select their <em>preformed</em> conformations that are most compatible with one another to effect binding by shifting two equilibriums progressively from less compatible to most compatible conformations for both partners <a href="#pone.0001025-Burgen1">[2]</a>–<a href="#pone.0001025-Pang1">[5]</a>. These preformed and most compatible conformations are conformations at local minima of their potential energy surfaces (local minimum conformations). When the most compatible conformations of both partners are most prevalent, the conformation selection theory becomes the lock–key theory <a href="#pone.0001025-Koshland1">[1]</a>. The conformation selection theory is ideal to computationally account for molecular flexibility in docking because it can convert a guest–host association best described by the conformational induction theory to a series of associations each of which can be described by the lock–key theory <a href="#pone.0001025-Pang2">[6]</a>. The conformation selection theory thereby affords parallel computing and enables a docking study to be performed using thousands of IBM Blue Gene processors with high processor utilization <a href="#pone.0001025-Pang2">[6]</a>–<a href="#pone.0001025-Pang4">[8]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In a recently reported study of 100 small-molecule–protein complex crystal structures, we found that the energy minimization of these small molecules alone does not automatically stop at minima of the potential energy surfaces of these molecules if the minima are shallow, thus leading to the folding of the molecules <a href="#pone.0001025-Wang1">[9]</a>; we also found that the small–molecule conformations in all 100 crystal structures are nearly identical to their local minimum conformations identified by normal mode analysis <a href="#pone.0001025-Hinsen1">[10]</a>–<a href="#pone.0001025-Cramer1">[13]</a> that uses analytic means to analyze harmonic potential wells and classify possible deformations of these molecules according to their energetic costs <a href="#pone.0001025-Wang1">[9]</a>. These findings suggest that small molecules prefer to adopt local minimum conformations when binding to proteins and theoretically support the virtual screening methods that use local minimum conformations to enable massively parallel docking <a href="#pone.0001025-Pang2">[6]</a>, <a href="#pone.0001025-Pang4">[8]</a>, <a href="#pone.0001025-Wang2">[14]</a>. In practice, the folding of small molecules caused by energy minimization in the absence of their partners hampers the generation of small–molecule bound conformations from their two–dimensional (2D) structures. This questions the practicality of the virtual screening methods that use local minimum conformations as potential bound conformations.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Herein we report a normal-mode-analysis–monitored energy minimization (NEM) procedure that generates bound conformations of small molecules from their 2D structures and we discuss our test of the NEM procedure. We also report an analysis of conformational strain energies of small–molecule bound conformations. The conformational strain energy is defined herein as the potential energy difference between a conformation of interest and its global minimum conformation. Knowing the strain energy is useful in triaging energetically less stable local minimum conformations in docking studies (see below). Our results suggest that the NEM procedure can generate small–molecule bound conformations and that guests adopt low-strain–energy conformations for complexation, thus offering additional support for the virtual screening methods that use local minimum conformations.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Normal-mode-analysis–monitored energy minimization procedure for generating bound conformations</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>As shown in <a href="#pone-0001025-g001">Figure 1</a>, the NEM procedure begins with 10 steps of energy minimization on a guest conformation generated by a torsion driver. The energy–minimized guest conformation is then subject to normal mode analysis to check whether the guest is in its local minimum conformation. The 10–step energy minimization uses a gradient cut-off of 10<sup>−7</sup> kcal/(mol•Å) and is repeated until the normal mode analysis shows that the guest is in a local minimum conformation. After each 10–step energy minimization, the gradient of the guest potential energy is checked. If the gradient is &gt;0.06 kcal/(mol•Å), the normal mode analysis is aborted, and the guest is considered not to be in its local minimum conformation. If the gradient is ≤0.06 kcal/(mol•Å), the normal mode analysis is performed, and the magnitudes of three translational and three rotational frequencies are checked. If the magnitudes of all translational frequencies are &lt;0.01 cm<sup>−1</sup> and the magnitudes of all rotational frequencies are &lt;10 cm<sup>−1</sup>, all vibrational frequencies are checked; otherwise, the analysis of vibrational frequencies is aborted and the guest is considered not to be in its local minimum conformation. If all vibrational frequencies are positive, the guest is considered to be in its local minimum conformation <a href="#pone.0001025-Jensen1">[12]</a>, <a href="#pone.0001025-Cramer1">[13]</a>. The cut-offs for the gradient and the translational and rotational frequencies are obtained from reference 12, and are based on the fact that geometry cannot be optimized to a gradient of exact zero because of numeric truncations <a href="#pone.0001025-Jensen1">[12]</a>. The NEM procedure is automated by a Perl script shown in <a href="#pone.0001025.s001">Figure S1</a>.</p>
<div class="figure" id="pone-0001025-g001"><div class="img"><a name="pone-0001025-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001025.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001025.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Flowchart of the normal-mode-analysis–monitored energy minimization procedure.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>mwRMSD stands for mass–weighted root mean square deviation.</p>
<span>doi:10.1371/journal.pone.0001025.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The essence of the NEM procedure is that it generates a local minimum conformation closest to the starting conformation. Different local minimum conformations can therefore be generated from rotamers obtained from the 2D structure by systematically varying the conformation–governing rotatable bonds of the molecule. In theory one of these local minimum conformations is a bound conformation for its particular host molecule according to the conformation selection theory described above.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>To test whether the NEM procedure can generate a set of guest local minimum conformations one of which is indeed a bound conformation to its known host, the crystal structure of a crown ether 18-crown-6 in complex with dithiobiurea {Cambridge Structural Database (CSD) code <a href="#pone.0001025-Allen1">[15]</a>: AJUXUY} was used as a model system because dithiobiurea has three conformation–governing rotatable bonds. As shown in <a href="#pone-0001025-g002">Figure 2</a>, 216 rotamers were generated from the 2D structure of dithiobiurea by systematically changing all conformation–governing rotatable bonds of the molecule in increments of 60° of arc starting from 0°. These rotamers were then optimized using the NEM procedure, and a cluster analysis with consideration of molecular symmetry of the 216 optimized rotamers identified six different local minimum conformations of dithiobiurea (<a href="#pone-0001025-g003">Figure 3</a>).</p>
<div class="figure" id="pone-0001025-g002"><div class="img"><a name="pone-0001025-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001025.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001025.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Process for generating the six different local minimum conformations of dithiobiurea used in a docking study to reproduce the dithiobiurea–18-crown-6 crystal structure.</span></strong></p><span>doi:10.1371/journal.pone.0001025.g002</span></div><div class="figure" id="pone-0001025-g003"><div class="img"><a name="pone-0001025-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001025.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001025.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Six different local minimum conformations of dithiobiurea generated by the normal-mode-analysis–monitored energy minimization procedure.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>The carbon, nitrogen, and sulfur atoms are green, blue, and orange, respectively.</p>
<span>doi:10.1371/journal.pone.0001025.g003</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>These local minimum conformations were then docked into the three–dimensional (3D) structure of 18-crown-6, which was taken from the complex crystal structure using the EUDOC program <a href="#pone.0001025-Pang2">[6]</a>, <a href="#pone.0001025-Pang4">[8]</a>, <a href="#pone.0001025-Wang2">[14]</a>. Initially, EUDOC failed to identify the bound conformation of dithiobiurea found in the crystal structure because the differences in the EUDOC–calculated interaction energy among the six dithiobiurea conformations were &lt;0.7 kcal/mol, which is the estimated uncertainty in calculating the interaction energy using EUDOC <a href="#pone.0001025-Pang2">[6]</a>. Visual inspection of the six EUDOC–generated 18-crown-6–dithiobiurea complexes revealed that there was only one hydrogen bond interaction between 18-crown-6 and dithiobiurea (<a href="#pone-0001025-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0001025-g004"><div class="img"><a name="pone-0001025-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001025.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001025.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Six energetically indistinguishable dithiobiurea–18-crown-6 complexes generated by the EUDOC program using local minimum conformations of dithiobiurea.</span></strong></p><a id="article1.body1.sec2.sec1.fig4.caption1.p1" name="article1.body1.sec2.sec1.fig4.caption1.p1"></a><p>The carbon, nitrogen, and sulfur atoms are green, blue, and orange, respectively.</p>
<span>doi:10.1371/journal.pone.0001025.g004</span></div><a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>Repeating the docking study with consideration to the influence of crystal packing on molecular complexation <a href="#pone.0001025-Wang2">[14]</a>, however, identified only one local minimum conformation of dithiobiurea as the bound conformation (<a href="#pone-0001025-g005">Figure 5</a>). The intermolecular interaction energy of this conformation is at least 7.5 kcal/mol lower (stronger) than those of the other five conformations (<a href="#pone-0001025-t001">Table 1</a>); the 18-crown-6 complex with this conformation has a guest mass–weighted root mean square deviation (mwRMSD) of 0.34 Å relative to that of crystal structure AJUXUY. In the 0.34–Å complex, dithiobiurea has both van der Waals and electrostatic interactions with 18-crown-6 as shown by the decomposed interaction energies in <a href="#pone-0001025-t001">Table 1</a>. These results suggested that the NEM procedure could generate a set of guest local minimum conformations one of which is a bound conformation to its host.</p>
<div class="figure" id="pone-0001025-g005"><div class="img"><a name="pone-0001025-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001025.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001025.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The dithiobiurea–18-crown-6 complex with the strongest intermolecular interaction energy that was identified by the EUDOC program using local minimum conformations of dithiobiurea.</span></strong></p><a id="article1.body1.sec2.sec1.fig5.caption1.p1" name="article1.body1.sec2.sec1.fig5.caption1.p1"></a><p>The nitrogen and sulfur atoms are blue and orange, respectively. The carbon atoms of the primary and neighboring hosts are green and yellow, respectively.</p>
<span>doi:10.1371/journal.pone.0001025.g005</span></div><div class="figure" id="pone-0001025-t001"><div class="img"><a name="pone-0001025-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001025.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001025.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Energies and structural differences of dithiobiurea–18-crown-6 complexes identified by the EUDOC program using six local minimum conformations of dithiobiurea.</span></strong></p><span>doi:10.1371/journal.pone.0001025.t001</span></div>

<h4>Testing the normal-mode-analysis–monitored energy minimization procedure</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To test the generality of the NEM procedure for generating bound conformations, the validation study with crystal structure AJUXUY was repeated with 21 additional small–molecule complex crystal structures (<a href="#pone-0001025-t002">Table 2</a>). These crystal structures were selected from a previously reported study <a href="#pone.0001025-Wang2">[14]</a> and have guest structures possessing fewer than five conformation–governing rotatable bonds. The use of this selection criterion reduced the demand for computing resources and allowed better estimation of conformational strain energies of bound conformations (see below). <a href="#pone-0001025-t002">Table 2</a> lists the results of the validation studies with the 21 crystal structures. The influence of crystal packing was taken into account in all of these studies.</p>
<div class="figure" id="pone-0001025-t002"><div class="img"><a name="pone-0001025-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001025" data-uri="info:doi/10.1371/journal.pone.0001025.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001025.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001025.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001025.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001025.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Accurate reproduction of 22 guest–host complex crystal structures using guest local minimum conformations generated by the normal-mode-analysis–monitored energy minimization procedure.</span></strong></p><span>doi:10.1371/journal.pone.0001025.t002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Of the 22 small–molecule complex crystal structures, including complex AJUXUY described above, 22 and 15 of them were produced with guest mwRMSDs of less than 1.0 and 0.5 Å, respectively by docking the NEM–generated guest local minimum conformations into their host structures that were taken from the corresponding complex crystal structures (<a href="#pone-0001025-t002">Table 2</a>). These results show that the NEM procedure can generate bound conformations in the absence of their host structures regardless of the number of local minimum conformations or the molecular complexity. It also demonstrates the generality of the NEM procedure for generating small–molecule bound conformations.</p>


<h4>Analysis of conformational strain energies of bound conformations</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In this study, the number of conformation–governing rotatable bonds in all 22 guest structures was fewer than five, and the rotamers were optimized using the NEM procedure. Sampling of guest conformations and identification of the global minimum conformation could therefore be done at a relatively fine granularity. Accordingly, the conformational strain energies of the 22 bound guest conformations in the complex crystal structures were determined from the potential energy difference between the EUDOC–identified bound conformation and the global minimum conformation. Of the 22 guest bound conformations in the guest-host complex crystal structures studied, 22 (100%), 18 (82%), and 16 (73%) of them have the conformational strain energies of less than or equal to 3.8, 2.0, and 0.6 kcal/mol, respectively (<a href="#pone-0001025-t002">Table 2</a>); 12 of them (55%) are in their global minimum conformations.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>The conformation sampling resolution</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>In this study, a cluster width of 60° of arc was used in cluster analysis. Although this width has been widely used and proven adequate for sampling the energy landscape of small to medium size molecules <a href="#pone.0001025-Beusen1">[16]</a>, <a href="#pone.0001025-Leach1">[17]</a>, it was desirable to confirm that this cluster width is narrow enough to identify distinct local minimum conformations. In repeating the cluster analysis and subsequent docking studies for the 22 complexes (see above) using a cluster width of 30° of arc, we identified new guest local minimum conformations for only two complexes (CSD codes: CECMEC10 and DOXWAO), but found that none of the new conformations have a lower potential energy than the global minimum conformation that was identified with the cluster width of 60° of arc and that none of these new conformations can form a complex with an interaction energy that is stronger than that of the complex obtained with the cluster width of 60° of arc. These results confirm that the 60°–of–arc conformation sampling resolution is adequate for generating distinct local minimum conformations.</p>


<h4>The conformational strain energies of the bound conformations</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The analysis of the conformational strain energies described above showed that 22 (100%), 18 (82%), 16 (73%), and 12 (55%) of the 22 guest bound conformations in the guest-host complex crystal structures have the conformational strain energies of less than or equal to 3.8, 2.0, 0.6, and 0.0 kcal/mol, respectively (<a href="#pone-0001025-t002">Table 2</a>). These observations are consistent with the report that approximately 70% of the small–molecule bound conformations in their protein-bound crystal structures have conformational strain energies of ≤3.0 kcal/mol <a href="#pone.0001025-Bostrm1">[18]</a>. These data are also consistent with our recently reported study of six small–molecule–protein complex crystal structures <a href="#pone.0001025-Wang1">[9]</a> in which 6 (100%), 5 (83%), 4 (67%), and 1 (17%) small–molecule bound conformations have the conformational strain energies of less than or equal to 2.3, 1.5, 0.88, and 0 kcal/mol, respectively. In this context, we propose to use a cut-off of 5.0 kcal/mol for the conformational strain energy to triage energetically less stable local minimum conformations in docking studies. This cut-off can significantly reduce the number of conformations used in a docking study and shorten the computing time for docking. For example, for the guest structure in one of the 22 complexes (CSD code: BAPRAM), rotamer generation, optimization with the NEM procedure, and conformational clustering identified 24 different local minimum conformations, but only eight of them (33%) need to be docked if a conformational strain energy cut-off of 5.0 kcal/mol is used to remove energetically less stable local minimum conformations. It is conceivable that using this cut-off the number of local minimum conformations will be markedly reduced, thus shortening the docking process greatly, when molecules to be docked have more than five conformation–governing rotatable bonds. Although the cut-off of 5.0 kcal/mol is a good starting point, more studies of various molecular complexes are needed to refine it.</p>


<h4>Generality of the NEM procedure for generating bound conformations</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>In this study the NEM procedure was used in conjunction with a rotamer sampling approach to generate local minimum conformations of a molecule possessing fewer than five conformation–governing rotatable bonds. Given the U.S. National Science Foundation's petascale science and engineering initiative (<a href="http://www.nsf.gov/pubs/2005/nsf05625/nsf05625.htm">http://www.nsf.gov/pubs/2005/nsf05625/ns​f05625.htm</a>) and the current cost reduction rate for disk space, it is conceivable that generation of large numbers of local minimum conformations for a molecule with more than four conformation–governing rotatable bonds is computationally feasible, because the calculations to search for different local minimum conformations are embarrassingly parallel over the commodity–driven multicore/multithread computer hardware. The NEM procedure can be applied to other conformation sampling approaches as well. When the number of conformation–governing rotatable bonds of a molecule is too large (e.g., &gt;10), the rotamer sampling approach can be computationally expensive. In that case, other approaches such as distance/conformational constraints <a href="#pone.0001025-Beusen1">[16]</a>, radial or adaptive sampling technique <a href="#pone.0001025-Dammkoehler1">[19]</a>, or stochastic sampling with multiple molecular dynamics simulations <a href="#pone.0001025-Caves1">[20]</a>–<a href="#pone.0001025-Pang5">[25]</a> can be used in conjunction with the NEM procedure to generate local minimum conformations. Given our finding that small molecules prefer to adopt local minimum conformations when binding to their partners <a href="#pone.0001025-Wang1">[9]</a>, the NEM procedure, which can generate a local minimum conformation closest to the starting conformation, appears to be a plausible procedure for generating small–molecule bound conformations that are useful for docking studies and for large–scale virtual screening of chemical databases for drug leads <a href="#pone.0001025-Pang3">[7]</a>, <a href="#pone.0001025-Pang4">[8]</a>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Selection of the 22 guest–host complex crystal structures</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We selected 22 guest–host complex crystal structures from a published study of 161 small–molecule complex crystal structures, all of which were reproduced with guest mwRMSDs of &lt;1.0 Å by the EUDOC program using the bound conformations of guests and hosts taken from crystal structures <a href="#pone.0001025-Wang2">[14]</a>. The selection criterion was that the number of conformation–governing rotatable bonds was fewer than five. The conformation–governing rotatable bond is defined as a torsion whose rotation changes the conformation of the molecule. A terminal torsion (e.g., the torsion of CH<sub>3</sub>CH<sub>3</sub>) is generally considered not to be a conformation–governing rotatable bond; however, a terminal torsion comprising the OH group or the F atom is treated as a conformation–governing rotatable bond because the hydroxyl H or F atom is a hydrogen bond donor or acceptor, respectively. The CSD codes for the 22 selected complexes are listed in <a href="#pone-0001025-t002">Table 2</a>.</p>


<h4>Preparation of the bound conformations of the hosts</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The 22 guest–bound host conformations were taken from the guest–host complex crystal structures. The atomic charges of these hosts were generated according to the RESP procedure <a href="#pone.0001025-Cieplak1">[26]</a> with <em>ab initio</em> calculations at the HF/6-31G* level using the Gaussian 98 program <a href="#pone.0001025-Frisch1">[27]</a> (<a href="#pone.0001025.s002">Table S1</a>). The force field parameters of these hosts were generated using the ANTECHAMBER module of the AMBER 7 program <a href="#pone.0001025-Pearlman1">[28]</a> using the Cornell et al. force field (parm99.dat/gaff.dat) <a href="#pone.0001025-Cornell1">[29]</a> (<a href="#pone.0001025.s003">Table S2</a>).</p>


<h4>Generation of the local minimum conformations of the guests</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>For each guest structure, a set of local minimum conformations was generated according to the following steps: (1) A 2D structure was converted to a 3D structure using the QUANTA97 program (Accelrys Software, Inc, San Diego, California). The atomic charges of the 3D structure were generated using the same method used for generating the host charges. (2) New conformations of the 3D structure were generated by systematically changing all conformation–governing rotatable bonds using the INTERFACE module of the AMBER 5 program <a href="#pone.0001025-Pearlman1">[28]</a> at a torsion increment of 60° of arc starting from 0°. The INTERFACE module generated 6<sup>n</sup> conformations in total, where n is the number of conformation–governing rotatable bonds. The torsional restraints used by the module were set as parabolic to the designated angle ±40° of arc and linear sides beyond that torsion range. The force constant used to restrain the conformation–governing rotatable bonds was 50 kcal/(mol•rad<sup>2</sup>). (3) Each conformation generated by the INTERFACE module was then subjected to the NEM procedure for energy minimization. (4) Cluster analysis was performed on each conformation–governing rotatable bond of the energy–minimized conformations. Each cluster contains all the conformations each of which has a torsion angle within ±30° of the average values of all the members in the corresponding cluster (cluster center). (5) One conformation was randomly chosen from each cluster as a representative conformation.</p>


<h4>Docking studies using the EUDOC program</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The algorithm of the EUDOC program has been reported elsewhere <a href="#pone.0001025-Pang2">[6]</a>. Briefly, it uses a systematic search protocol, translating and rotating a guest in a putative binding pocket of a host and repeating the translations and rotations with different conformations of both guest and host to search for energetically favorable conformations, orientations, and positions of the guest relative to the host. A docking box is defined within the binding pocket to confine the translation of the guest. The intermolecular interaction energy is the potential energy of the guest–host complex relative to the potential energies of the two partners in their free states. This energy is calculated according to Equations 1 and 2 using the second–generation AMBER force field <a href="#pone.0001025-Cornell1">[29]</a>. In calculating the intermolecular interaction energy, the multiplicative dielectric constant is set to 1.0, and the distance cut-offs for steric and electrostatic interactions are set to 10<sup>9</sup> Å.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(Eq.1)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001025.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(Eq.2)</span></span><br>In this study, a docking box was defined to enclose the guest structure in the host structure; each dimension of the box is ≥6 Å, the size of the docking box and the cut-off for the interaction energy used by the EUDOC program are listed in <a href="#pone.0001025.s004">Table S3</a>; the complex–prediction module of EUDOC was used to translate and rotate the guest around the host at increments of 1.0 Å and 10° of arc, respectively; all different guest local minimum conformations were automatically docked into the host structure taken from the corresponding guest–host complex crystal structure using EUDOC.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>To consider the influence of crystal packing, the PyMOL program (DeLano Scientific LLC, South San Francisco, California) was used to generate a multimeric host system by applying the symmetry of the space group of the crystal structure. Host or guest structures were excluded from the multimeric host system if the shortest distance of a heavy atom of the guest structure to be docked to the heavy atom of the host/guest structure in neighboring unit cells was &gt;4.0 Å.</p>


<h4>Energy minimization monitored with normal mode analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Energy minimization used (1) 10<sup>6 </sup>steps of energy minimization, (2) a dielectric multiplicative constant of 80.0, (3) the steepest descent or conjugate gradient method, (4) a nonbonded cut-off of 12 Å, (9) a 10<sup>−7</sup>-kcal/(mole•Å) cut-off for the root-mean-square of the Cartesian elements of the gradient, and (10) defaults for other inputs of the SANDER module of the AMBER 5 program <a href="#pone.0001025-Pearlman1">[28]</a>. NMA used (1) a dielectric multiplicative constant of 80.0, (2) a nonbonded cut-off of 12 Å, and (3) defaults for other inputs of the NMODE module of the AMBER 8 program <a href="#pone.0001025-Pearlman1">[28]</a>.</p>


<h4>Mass–weighted root mean square deviations</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The mwRMSDs were calculated by superimposing the host portion of the EUDOC–generated complex over the corresponding host portion of the crystal structure. The mwRMSD of all atoms of the guest portion in the two superimposed complexes were determined using the PTRAJ module of the AMBER 8 program <a href="#pone.0001025-Pearlman1">[28]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001025"></div><a name="pone.0001025.s001" id="pone.0001025.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001025.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Perl script for the normal-mode-analysis-monitored energy minimization procedure.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001025.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.05 MB PDF)</p>
<a name="pone.0001025.s002" id="pone.0001025.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001025.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">The RESP charges and the AMBER atom types of 22 host-guest complexes. Atom names, the AMBER atom types, Cartesian coordinates x,y and z, and the RESP charges are at columns 3, 4, 6, 7, 8 and 9. Suffixes “h” and “g” specifies the host and the guest of the complex crystal structure.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001025.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.34 MB PDF)</p>
<a name="pone.0001025.s003" id="pone.0001025.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001025.s003">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">The AMBER force field parameters for the 22 guest structures.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001025.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0001025.s004" id="pone.0001025.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001025.s004">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Detailed information with regard to the docking studies of the 22 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001025.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank an anonymous reviewer for insightful comments and suggestions. The authors also acknowledge the computing support from the University of Minnesota Supercomputing Institute.</p>
<a id="article1.back1.ack1.p2" name="article1.back1.ack1.p2"></a><p><strong>Disclaimer</strong></p>
<a id="article1.back1.ack1.p3" name="article1.back1.ack1.p3"></a><p>The opinions or assertions contained herein belong to the authors and are not necessarily the official views of the U.S. Army, the U.S. Department of Defense, or the U.S. National Institutes of Health.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YP. Performed the experiments: QW. Analyzed the data: YP QW. Wrote the paper: YP QW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001025-Koshland1" id="pone.0001025-Koshland1"></a>Koshland DE (1995) The key-lock theory and the induced fit theory. Angew Chem Int Ed Engl  33: 2375–2378.  <ul class="find" data-citedArticleID="1004492" data-doi="10.1002/anie.199423751"><li><a href="http://dx.doi.org/10.1002/anie.199423751" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+key-lock+theory+and+the+induced+fit+theory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+key-lock+theory+and+the+induced+fit+theory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001025-Burgen1" id="pone.0001025-Burgen1"></a>Burgen AS (1981) Conformational changes and drug action. Fed Proc  40: 2723–2728.  <ul class="find" data-citedArticleID="1004470"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Conformational+changes+and+drug+action.&amp;auth=&amp;atitle=Conformational+changes+and+drug+action." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+changes+and+drug+action." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+changes+and+drug+action.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001025-Bruns1" id="pone.0001025-Bruns1"></a>Bruns RF (1996) Conformational induction versus conformational selection: evidence from allosteric enhancers. Trends Pharmacol Sci  17: 189.  <ul class="find" data-citedArticleID="1004468" data-doi="10.1016/0165-6147(96)20027-6"><li><a href="http://dx.doi.org/10.1016/0165-6147(96)20027-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+induction+versus+conformational+selection%3A+evidence+from+allosteric+enhancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+induction+versus+conformational+selection%3A+evidence+from+allosteric+enhancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001025-Kenakin1" id="pone.0001025-Kenakin1"></a>Kenakin T (1996) Receptor conformational induction versus selection: all part of the same energy landscape, agonists can differentially stabilize multiple active states of receptors. Trends Pharmacol Sci  17: 190–191.  <ul class="find" data-citedArticleID="1004490" data-doi="10.1016/0165-6147(96)20026-4"><li><a href="http://dx.doi.org/10.1016/0165-6147(96)20026-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Receptor+conformational+induction+versus+selection%3A+all+part+of+the+same+energy+landscape%2C+agonists+can+differentially+stabilize+multiple+active+states+of+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Receptor+conformational+induction+versus+selection%3A+all+part+of+the+same+energy+landscape%2C+agonists+can+differentially+stabilize+multiple+active+states+of+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001025-Pang1" id="pone.0001025-Pang1"></a>Pang Y-P, Silva ND, Hydock C, Prendergast FG (1997) Docking studies on the complexed and uncomplexed FKBP12 structures with bound and unbound ligands: an implication of conformational selection mechanism for binding. J Mol Model  3: 240–248.  <ul class="find" data-citedArticleID="1004498" data-doi="10.1007/s008940050036"><li><a href="http://dx.doi.org/10.1007/s008940050036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Docking+studies+on+the+complexed+and+uncomplexed+FKBP12+structures+with+bound+and+unbound+ligands%3A+an+implication+of+conformational+selection+mechanism+for+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Docking+studies+on+the+complexed+and+uncomplexed+FKBP12+structures+with+bound+and+unbound+ligands%3A+an+implication+of+conformational+selection+mechanism+for+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001025-Pang2" id="pone.0001025-Pang2"></a>Pang Y-P, Perola E, Xu K, Prendergast FG (2001) EUDOC: a computer program for identification of drug interaction sites in macromolecules and drug leads from chemical databases. J Comput Chem  22: 1750–1771.  <ul class="find" data-citedArticleID="1004500" data-doi="10.1002/jcc.1129"><li><a href="http://dx.doi.org/10.1002/jcc.1129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EUDOC%3A+a+computer+program+for+identification+of+drug+interaction+sites+in+macromolecules+and+drug+leads+from+chemical+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EUDOC%3A+a+computer+program+for+identification+of+drug+interaction+sites+in+macromolecules+and+drug+leads+from+chemical+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001025-Pang3" id="pone.0001025-Pang3"></a>Pang Y-P (2007) In silico drug discovery: solving the “target-rich and lead-poor” imbalance using the genome-to-drug-lead paradigm. Clin Pharmacol Ther  81: 30–34.  <ul class="find" data-citedArticleID="1004502"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+silico+drug+discovery%3A+solving+the+%3Ftarget-rich+and+lead-poor%3F+imbalance+using+the+genome-to-drug-lead+paradigm.&amp;auth=&amp;atitle=In+silico+drug+discovery%3A+solving+the+%3Ftarget-rich+and+lead-poor%3F+imbalance+using+the+genome-to-drug-lead+paradigm." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+silico+drug+discovery%3A+solving+the+%E2%80%9Ctarget-rich+and+lead-poor%E2%80%9D+imbalance+using+the+genome-to-drug-lead+paradigm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+silico+drug+discovery%3A+solving+the+%E2%80%9Ctarget-rich+and+lead-poor%E2%80%9D+imbalance+using+the+genome-to-drug-lead+paradigm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001025-Pang4" id="pone.0001025-Pang4"></a>Pang Y-P, Mullins T, Swartz B, McAllister J, Smith B, et al.  (2007) EUDOC on Blue Gene: accelerating the transfer of drug discoveries from laboratory to patient. IBM J Res Dev in press.   <ul class="find" data-citedArticleID="1004504" data-doi="10.1147/rd.521.0069"><li><a href="http://dx.doi.org/10.1147/rd.521.0069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EUDOC+on+Blue+Gene%3A+accelerating+the+transfer+of+drug+discoveries+from+laboratory+to+patient." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EUDOC+on+Blue+Gene%3A+accelerating+the+transfer+of+drug+discoveries+from+laboratory+to+patient.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001025-Wang1" id="pone.0001025-Wang1"></a>Wang Q, Pang Y-P (2007) Preference of small molecules for local minimum conformations when binding to proteins. PLoS ONE  2: e820.  <ul class="find" data-citedArticleID="1004514" data-doi="10.1371/journal.pone.0000820"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000820" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preference+of+small+molecules+for+local+minimum+conformations+when+binding+to+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preference+of+small+molecules+for+local+minimum+conformations+when+binding+to+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001025-Hinsen1" id="pone.0001025-Hinsen1"></a>Hinsen K (2006)  Normal mode theory and harmonic potential approximations;. In: Cui Q, Bahar I, editors. Boca Raton: Chapman &amp; Hall/CRC. pp. 1–16.  <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0001025-Case1" id="pone.0001025-Case1"></a>Case D (1999)  Rigidity theory and applications;. In: Thorpe M, Duxbury P, editors. New York: Kluwer Academic/Plenum Publishers. pp. 329–344.  <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0001025-Jensen1" id="pone.0001025-Jensen1"></a>Jensen F (1998) Introduction to computational chemistry. New York: John Wiley &amp; Sons. pp. 312–315.  <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0001025-Cramer1" id="pone.0001025-Cramer1"></a>Cramer CJ (2002) Essentials of computational chemistry. New York: John Wiley &amp; Sons.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001025-Wang2" id="pone.0001025-Wang2"></a>Wang Q, Pang Y-P (2007) Accurate reproduction of 161 small-molecule complex crystal structures using the EUDOC program: expanding the use of EUDOC to supramolecular chemistry. PLoS ONE  2: e531.  <ul class="find" data-citedArticleID="1004516" data-doi="10.1371/journal.pone.0000531"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000531" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accurate+reproduction+of+161+small-molecule+complex+crystal+structures+using+the+EUDOC+program%3A+expanding+the+use+of+EUDOC+to+supramolecular+chemistry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accurate+reproduction+of+161+small-molecule+complex+crystal+structures+using+the+EUDOC+program%3A+expanding+the+use+of+EUDOC+to+supramolecular+chemistry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001025-Allen1" id="pone.0001025-Allen1"></a>Allen FH (2002) The Cambridge Structural Database: a quarter of a million crystal structures and rising. Acta Crystallogr Sect B: Struct Sci  58: 380–388.  <ul class="find" data-citedArticleID="1004462" data-doi="10.1107/s0108768102003890"><li><a href="http://dx.doi.org/10.1107/s0108768102003890" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Cambridge+Structural+Database%3A+a+quarter+of+a+million+crystal+structures+and+rising." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Cambridge+Structural+Database%3A+a+quarter+of+a+million+crystal+structures+and+rising.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001025-Beusen1" id="pone.0001025-Beusen1"></a>Beusen DD, Shands EFB, Karasek SF, Marshall GR, Dammkoehler RA (1996) Systematic search in conformational analysis. J Mol Struc (Theochem)  370: 157–171.  <ul class="find" data-citedArticleID="1004464" data-doi="10.1016/s0166-1280(96)04565-4"><li><a href="http://dx.doi.org/10.1016/s0166-1280(96)04565-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+search+in+conformational+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+search+in+conformational+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001025-Leach1" id="pone.0001025-Leach1"></a>Leach AR (1991)  A survey of methods for searching the conformational space of small and medium-sized molecules. In: Lipkowitz BK, Boyd BD, editors. Reviews in Computational Chemistry: John Wiley &amp; Sons, Inc. pp. 1–55.  <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0001025-Bostrm1" id="pone.0001025-Bostrm1"></a>Boström J, Norrby P-O, Liljefors T (1998) Conformational energy penalties of protein-bound ligands. J Comput-Aided Mol Design  12: 383–396.  <ul class="find" data-citedArticleID="1004466"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Conformational+energy+penalties+of+protein-bound+ligands.&amp;auth=&amp;atitle=Conformational+energy+penalties+of+protein-bound+ligands." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+energy+penalties+of+protein-bound+ligands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+energy+penalties+of+protein-bound+ligands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001025-Dammkoehler1" id="pone.0001025-Dammkoehler1"></a>Dammkoehler RA, Karasek SF, Shands EF, Marshall GR (1995) Sampling conformational hyperspace: techniques for improving completeness. J Comput Aided Mol Design  9: 491–499.  <ul class="find" data-citedArticleID="1004482" data-doi="10.1007/bf00124320"><li><a href="http://dx.doi.org/10.1007/bf00124320" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sampling+conformational+hyperspace%3A+techniques+for+improving+completeness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sampling+conformational+hyperspace%3A+techniques+for+improving+completeness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001025-Caves1" id="pone.0001025-Caves1"></a>Caves LSD, Evanseck JD, Karplus M (1998) Locally accessible conformations of proteins - multiple molecular dynamics simulations of crambin. Protein Sci  7: 649–666.  <ul class="find" data-citedArticleID="1004474" data-doi="10.1002/pro.5560070314"><li><a href="http://dx.doi.org/10.1002/pro.5560070314" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Locally+accessible+conformations+of+proteins+-+multiple+molecular+dynamics+simulations+of+crambin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Locally+accessible+conformations+of+proteins+-+multiple+molecular+dynamics+simulations+of+crambin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001025-Smith1" id="pone.0001025-Smith1"></a>Smith LJ, Daura X, van Gunsteren WF (2002) Assessing equilibration and convergence in biomolecular simulations. Proteins  48: 487–496.  <ul class="find" data-citedArticleID="1004510" data-doi="10.1002/prot.10144"><li><a href="http://dx.doi.org/10.1002/prot.10144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+equilibration+and+convergence+in+biomolecular+simulations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+equilibration+and+convergence+in+biomolecular+simulations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001025-Snow1" id="pone.0001025-Snow1"></a>Snow CD, Nguyen N, Pande VS, Gruebele M (2002) Absolute comparison of simulated and experimental protein-folding dynamics. Nature  420: 102–106.  <ul class="find" data-citedArticleID="1004512" data-doi="10.1038/nature01160"><li><a href="http://dx.doi.org/10.1038/nature01160" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Absolute+comparison+of+simulated+and+experimental+protein-folding+dynamics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Absolute+comparison+of+simulated+and+experimental+protein-folding+dynamics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001025-Zagrovic1" id="pone.0001025-Zagrovic1"></a>Zagrovic B, Snow CD, Shirts MR, Pande VS (2002) Simulation of folding of a small alpha-helical protein in atomistic detail using worldwide-distributed computing. J Mol Biol  323: 927–937.  <ul class="find" data-citedArticleID="1004518" data-doi="10.1016/s0022-2836(02)00997-x"><li><a href="http://dx.doi.org/10.1016/s0022-2836(02)00997-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simulation+of+folding+of+a+small+alpha-helical+protein+in+atomistic+detail+using+worldwide-distributed+computing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simulation+of+folding+of+a+small+alpha-helical+protein+in+atomistic+detail+using+worldwide-distributed+computing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001025-Oelschlaeger1" id="pone.0001025-Oelschlaeger1"></a>Oelschlaeger P, Schmid RD, Pleiss J (2003) Modeling domino effects in enzymes: molecular basis of the substrate specificity of the bacterial metallo-beta-lactamases IMP-1 and IMP-6. Biochemistry  42: 8945–8956.  <ul class="find" data-citedArticleID="1004496" data-doi="10.1021/bi0300332"><li><a href="http://dx.doi.org/10.1021/bi0300332" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+domino+effects+in+enzymes%3A+molecular+basis+of+the+substrate+specificity+of+the+bacterial+metallo-beta-lactamases+IMP-1+and+IMP-6." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+domino+effects+in+enzymes%3A+molecular+basis+of+the+substrate+specificity+of+the+bacterial+metallo-beta-lactamases+IMP-1+and+IMP-6.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001025-Pang5" id="pone.0001025-Pang5"></a>Pang Y-P (2004) Three-dimensional model of a substrate-bound SARS chymotrypsin-like cysteine proteinase predicted by multiple molecular dynamics simulations: catalytic efficiency regulated by substrate binding. Proteins  57: 747–757.  <ul class="find" data-citedArticleID="1004506" data-doi="10.1002/prot.20249"><li><a href="http://dx.doi.org/10.1002/prot.20249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three-dimensional+model+of+a+substrate-bound+SARS+chymotrypsin-like+cysteine+proteinase+predicted+by+multiple+molecular+dynamics+simulations%3A+catalytic+efficiency+regulated+by+substrate+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three-dimensional+model+of+a+substrate-bound+SARS+chymotrypsin-like+cysteine+proteinase+predicted+by+multiple+molecular+dynamics+simulations%3A+catalytic+efficiency+regulated+by+substrate+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001025-Cieplak1" id="pone.0001025-Cieplak1"></a>Cieplak P, Cornell WD, Bayly C, Kollman PA (1995) Application of the multimolecule and multiconformational resp methodology to biopolymers: charge derivation for DNA, RNA, and proteins. J Comput Chem  16: 1357–1377.  <ul class="find" data-citedArticleID="1004476" data-doi="10.1002/jcc.540161106"><li><a href="http://dx.doi.org/10.1002/jcc.540161106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+the+multimolecule+and+multiconformational+resp+methodology+to+biopolymers%3A+charge+derivation+for+DNA%2C+RNA%2C+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+the+multimolecule+and+multiconformational+resp+methodology+to+biopolymers%3A+charge+derivation+for+DNA%2C+RNA%2C+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001025-Frisch1" id="pone.0001025-Frisch1"></a>Frisch MJ, Trucks GW, Schlegel HB, Gill PMW, Hohnson BG, et al.  (1999) GAUSSIAN 98, Revision A.7. Pittsburgh, , PA: Gaussian, Inc.   <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0001025-Pearlman1" id="pone.0001025-Pearlman1"></a>Pearlman DA, Case DA, Caldwell JW, Ross WS, Cheatham TE III, et al.  (1995) AMBER, a package of computer programs for applying molecular mechanics, normal mode analysis, molecular dynamics and free energy calculations to simulate the structural and energetic properties of molecules. Comput Phys Commun  91: 1–41.  <ul class="find" data-citedArticleID="1004508" data-doi="10.1016/0010-4655(95)00041-d"><li><a href="http://dx.doi.org/10.1016/0010-4655(95)00041-d" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AMBER%2C+a+package+of+computer+programs+for+applying+molecular+mechanics%2C+normal+mode+analysis%2C+molecular+dynamics+and+free+energy+calculations+to+simulate+the+structural+and+energetic+properties+of+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AMBER%2C+a+package+of+computer+programs+for+applying+molecular+mechanics%2C+normal+mode+analysis%2C+molecular+dynamics+and+free+energy+calculations+to+simulate+the+structural+and+energetic+properties+of+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001025-Cornell1" id="pone.0001025-Cornell1"></a>Cornell WD, Cieplak P, Bayly CI, Gould IR, Merz KM Jr, et al.  (1995) A second generation force field for the simulation of proteins, nucleic acids, and organic molecules. J Am Chem Soc  117: 5179–5197.  <ul class="find" data-citedArticleID="1004478" data-doi="10.1021/ja00124a002"><li><a href="http://dx.doi.org/10.1021/ja00124a002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+second+generation+force+field+for+the+simulation+of+proteins%2C+nucleic+acids%2C+and+organic+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+second+generation+force+field+for+the+simulation+of+proteins%2C+nucleic+acids%2C+and+organic+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.XML" value="75981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.PDF" value="200863"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g001.PNG_L" value="75283"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g001.PNG_M" value="30399"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g001.PNG_S" value="5788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g001.TIF" value="146520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g001.PNG_I" value="10676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g002.PNG_L" value="205233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g002.PNG_M" value="62230"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g002.PNG_S" value="8568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g002.TIF" value="312888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g002.PNG_I" value="68840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g003.PNG_L" value="108669"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g003.PNG_M" value="38414"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g003.PNG_S" value="7079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g003.TIF" value="203430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g003.PNG_I" value="24843"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g004.PNG_L" value="374102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g004.PNG_M" value="67398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g004.PNG_S" value="7406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g004.TIF" value="736964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g004.PNG_I" value="24441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g005.PNG_L" value="266187"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g005.PNG_M" value="49969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g005.PNG_S" value="6772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g005.TIF" value="562312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.g005.PNG_I" value="18194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.t001.PNG_L" value="116613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.t001.PNG_M" value="67028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.t001.PNG_S" value="9557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.t001.TIF" value="522824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.t001.PNG_I" value="21071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.t002.PNG_L" value="246117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.t002.PNG_M" value="142969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.t002.PNG_S" value="15158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.t002.TIF" value="1168098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.t002.PNG_I" value="44939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.e001.PNG" value="12925"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.e001.TIF" value="28376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.e002.PNG" value="9566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.e002.TIF" value="25496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.s001.PDF" value="45023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.s002.PDF" value="340386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.s003.PDF" value="16095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001025.s004.PDF" value="17031"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001025&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001025" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001025&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001025&volume=&issue=&title=Normal-Mode-Analysis–Monitored Energy Minimization Procedure for Generating Small–Molecule Bound Conformations&author_name=Qi%20Wang%2C%20Yuan-Ping%20Pang&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001025" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001025" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001025" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001025&amp;t=Normal-Mode-Analysis%E2%80%93Monitored%20Energy%20Minimization%20Procedure%20for%20Generating%20Small%E2%80%93Molecule%20Bound%20Conformations" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001025&title=Normal-Mode-Analysis%E2%80%93Monitored%20Energy%20Minimization%20Procedure%20for%20Generating%20Small%E2%80%93Molecule%20Bound%20Conformations&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001025&amp;title=Normal-Mode-Analysis%E2%80%93Monitored%20Energy%20Minimization%20Procedure%20for%20Generating%20Small%E2%80%93Molecule%20Bound%20Conformations" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001025" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001025&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Normal-Mode-Analysis–Monitored Energy Minimization Procedure for Generating Small–Molecule Bound Conformations';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001025';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Normal-Mode-Analysis%E2%80%93Monitored%20Energy%20Minimization%20Procedure%20for%20Generating%20Small%E2%80%93Molecule%20Bound%20Conformations http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001025" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001025" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Carbon%22" title="Search for articles in the subject area:'Carbon'"><div class="flagText">Carbon</div></a>
              <div data-categoryid="21023" data-articleid="24884"
                   data-categoryname="Carbon"
                   class="flagImage" title="Flag 'Carbon' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Computer+software%22" title="Search for articles in the subject area:'Computer software'"><div class="flagText">Computer software</div></a>
              <div data-categoryid="2287" data-articleid="24884"
                   data-categoryname="Computer software"
                   class="flagImage" title="Flag 'Computer software' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystal+structure%22" title="Search for articles in the subject area:'Crystal structure'"><div class="flagText">Crystal structure</div></a>
              <div data-categoryid="17301" data-articleid="24884"
                   data-categoryname="Crystal structure"
                   class="flagImage" title="Flag 'Crystal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Deformation%22" title="Search for articles in the subject area:'Deformation'"><div class="flagText">Deformation</div></a>
              <div data-categoryid="17439" data-articleid="24884"
                   data-categoryname="Deformation"
                   class="flagImage" title="Flag 'Deformation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Molecular+structure%22" title="Search for articles in the subject area:'Molecular structure'"><div class="flagText">Molecular structure</div></a>
              <div data-categoryid="19867" data-articleid="24884"
                   data-categoryname="Molecular structure"
                   class="flagImage" title="Flag 'Molecular structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Potential+energy%22" title="Search for articles in the subject area:'Potential energy'"><div class="flagText">Potential energy</div></a>
              <div data-categoryid="23753" data-articleid="24884"
                   data-categoryname="Potential energy"
                   class="flagImage" title="Flag 'Potential energy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Small+molecules%22" title="Search for articles in the subject area:'Small molecules'"><div class="flagText">Small molecules</div></a>
              <div data-categoryid="17289" data-articleid="24884"
                   data-categoryname="Small molecules"
                   class="flagImage" title="Flag 'Small molecules' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Vibration%22" title="Search for articles in the subject area:'Vibration'"><div class="flagText">Vibration</div></a>
              <div data-categoryid="20931" data-articleid="24884"
                   data-categoryname="Vibration"
                   class="flagImage" title="Flag 'Vibration' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9394'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8043'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3495&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
