

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Phage Lambda CIII: A Protease Inhibitor Regulating the Lysis-Lysogeny Decision</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000363"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000363" />

    <meta name="citation_title" content="Phage Lambda CIII: A Protease Inhibitor Regulating the Lysis-Lysogeny Decision"/>
    <meta itemprop="name" content="Phage Lambda CIII: A Protease Inhibitor Regulating the Lysis-Lysogeny Decision"/>

      <meta name="citation_author" content="Oren Kobiler"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics and Biotechnology, The Hebrew University-Hadassah Medical School, Jerusalem, Israel"/>
      <meta name="citation_author" content="Assaf Rokney"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics and Biotechnology, The Hebrew University-Hadassah Medical School, Jerusalem, Israel"/>
      <meta name="citation_author" content="Amos B. Oppenheim"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics and Biotechnology, The Hebrew University-Hadassah Medical School, Jerusalem, Israel"/>

    <meta name="citation_date" content="2007/4/11"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000363.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e363"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Proteases and their targets in Escherichia coli.; citation_author=S Gottesman; citation_journal_title=Annu Rev Genet; citation_volume=30; citation_number=1; citation_pages=465-506; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Marked instability of the sigma(32) heat shock transcription factor at high temperature. Implications for heat shock regulation.; citation_author=M Kanemori; citation_author=H Yanagi; citation_author=T Yura; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=2; citation_pages=22002-22007; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Regulation by proteolysis in bacterial cells.; citation_author=U Jenal; citation_author=R Hengge-Aronis; citation_journal_title=Curr Opin Microbiol; citation_volume=6; citation_number=3; citation_pages=163-172; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Global role for ClpP-containing proteases in stationary-phase adaptation of Escherichia coli.; citation_author=D Weichart; citation_author=N Querfurth; citation_author=M Dreger; citation_author=R Hengge-Aronis; citation_journal_title=J Bacteriol; citation_volume=185; citation_number=4; citation_pages=115-125; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Recruitment of host ATP-dependent proteases by bacteriophage lambda.; citation_author=O Kobiler; citation_author=AB Oppenheim; citation_author=C Herman; citation_journal_title=J Struct Biol; citation_volume=146; citation_number=5; citation_pages=72-78; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Protein folding and degradation in bacteria: to degrade or not to degrade? That is the question.; citation_author=DA Dougan; citation_author=A Mogk; citation_author=B Bukau; citation_journal_title=Cell Mol Life Sci; citation_volume=59; citation_number=6; citation_pages=1607-1616; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Proteolysis in bacterial regulatory circuits.; citation_author=S Gottesman; citation_journal_title=Annu Rev Cell Dev Biol; citation_volume=19; citation_number=7; citation_pages=565-587; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Proteolysis: Adaptor, adaptor, catch me a catch.; citation_author=SE Ades; citation_journal_title=Curr Biol; citation_volume=14; citation_number=8; citation_pages=R924-926; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=PinA inhibits ATP hydrolysis and energy-dependent protein degradation by Lon protease.; citation_author=JJ Hilliard; citation_author=LD Simon; citation_author=L Van Melderen; citation_author=MR Maurizi; citation_journal_title=J Biol Chem; citation_volume=273; citation_number=9; citation_pages=524-527; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=SpoVM, a small protein essential to development in Bacillus subtilis, interacts with the ATP-dependent protease FtsH.; citation_author=S Cutting; citation_author=M Anderson; citation_author=E Lysenko; citation_author=A Page; citation_author=T Tomoyasu; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=10; citation_pages=5534-5542; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The HflB protease of Escherichia coli degrades its inhibitor lambda cIII.; citation_author=C Herman; citation_author=D Thevenet; citation_author=R D'Ari; citation_author=P Bouloc; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=11; citation_pages=358-363; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The phage lambda CII transcriptional activator carries a C-terminal domain signaling for rapid proteolysis.; citation_author=O Kobiler; citation_author=S Koby; citation_author=D Teff; citation_author=D Court; citation_author=AB Oppenheim; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=12; citation_pages=14964-14969; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The activity of the CIII regulator of lambdoid bacteriophages resides within a 24-amino acid protein domain.; citation_author=D Kornitzer; citation_author=S Altuvia; citation_author=AB Oppenheim; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=88; citation_number=13; citation_pages=5217-5221; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Structural and functional studies on an FtsH inhibitor from Bacillus subtilis.; citation_author=RS Prajapati; citation_author=T Ogura; citation_author=SM Cutting; citation_journal_title=Biochim Biophys Acta; citation_volume=1475; citation_number=14; citation_pages=353-359; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Escherichia coli FtsH is a membrane-bound, ATP-dependent protease which degrades the heat-shock transcription factor sigma 32.; citation_author=T Tomoyasu; citation_author=J Gamer; citation_author=B Bukau; citation_author=M Kanemori; citation_author=H Mori; citation_journal_title=Embo J; citation_volume=14; citation_number=15; citation_pages=2551-2560; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Cellular functions, mechanism of action, and regulation of FtsH protease.; citation_author=K Ito; citation_author=Y Akiyama; citation_journal_title=Annu Rev Microbiol; citation_volume=59; citation_number=16; citation_pages=211-231; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Different pathways for protein degradation by the FtsH/HflKC membrane-embedded protease complex: an implication from the interference by a mutant form of a new substrate protein, YccA.; citation_author=A Kihara; citation_author=Y Akiyama; citation_author=K Ito; citation_journal_title=J Mol Biol; citation_volume=279; citation_number=17; citation_pages=175-188; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A Genetic Switch: Phage Lambda Revisted: CSHL Press.; citation_author=M Ptashne; citation_number=18; citation_pages=154 p; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Switches in bacteriophage lambda development.; citation_author=AB Oppenheim; citation_author=O Kobiler; citation_author=J Stavans; citation_author=DL Court; citation_author=S Adhya; citation_journal_title=Annu Rev Genet; citation_volume=39; citation_number=19; citation_pages=409-429; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Stability of CII is a key element in the cold stress response of bacteriophage lambda infection.; citation_author=M Obuchowski; citation_author=Y Shotland; citation_author=S Koby; citation_author=H Giladi; citation_author=M Gabig; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=20; citation_pages=5987-5991; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Lysogenization by bacteriophage lambda. III. Multiplicity dependent phenomena occuring upon infection by lambda.; citation_author=P Kourilsky; citation_author=A Knapp; citation_journal_title=Biochimie; citation_volume=56; citation_number=21; citation_pages=1517-1523; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Mutations in a temperate bacteriophage affecting its ability to lysogenize Escherichia coli.; citation_author=AD Kaiser; citation_journal_title=Virology; citation_volume=3; citation_number=22; citation_pages=42-61; citation_date=1957; " />
      <meta name="citation_reference" content="citation_title=Control of lambda repressor synthesis.; citation_author=L Reichardt; citation_author=AD Kaiser; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=68; citation_number=23; citation_pages=2185-2189; citation_date=1971; " />
      <meta name="citation_reference" content="citation_title=The operator and early promoter region of the Shiga toxin type 2-encoding bacteriophage 933W and control of toxin expression.; citation_author=JS Tyler; citation_author=MJ Mills; citation_author=DI Friedman; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=24; citation_pages=7670-7679; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structure and function of the nun gene and the immunity region of the lambdoid phage HK022.; citation_author=J Oberto; citation_author=RA Weisberg; citation_author=ME Gottesman; citation_journal_title=J Mol Biol; citation_volume=207; citation_number=25; citation_pages=675-693; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Isolation, characterization and deletion mapping of amber mutations in the cll gene of phage lambda.; citation_author=M Belfort; citation_author=D Noff; citation_author=AB Oppenheim; citation_journal_title=Virology; citation_volume=63; citation_number=26; citation_pages=147-159; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=The lysis-lysogeny decision of phage lambda: explicit programming and responsiveness.; citation_author=I Herskowitz; citation_author=D Hagen; citation_journal_title=Annu Rev Genet; citation_volume=14; citation_number=27; citation_pages=399-445; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Quantitative kinetic analysis of the bacteriophage lambda genetic network.; citation_author=O Kobiler; citation_author=A Rokney; citation_author=N Friedman; citation_author=DL Court; citation_author=J Stavans; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=28; citation_pages=4470-4475; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Translational regulatory signals within the coding region of the bacteriophage lambda cIII gene.; citation_author=S Altuvia; citation_author=AB Oppenheim; citation_journal_title=J Bacteriol; citation_volume=167; citation_number=29; citation_pages=415-419; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=In vivo recombineering of bacteriophage lambda by PCR fragments and single-strand oligonucleotides.; citation_author=AB Oppenheim; citation_author=AJ Rattray; citation_author=M Bubunenko; citation_author=LC Thomason; citation_author=DL Court; citation_journal_title=Virology; citation_volume=319; citation_number=30; citation_pages=185-189; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Balanced biosynthesis of major membrane components through regulated degradation of the committed enzyme of lipid A biosynthesis by the AAA protease FtsH (HflB) in Escherichia coli.; citation_author=T Ogura; citation_author=K Inoue; citation_author=T Tatsuta; citation_author=T Suzaki; citation_author=K Karata; citation_journal_title=Mol Microbiol; citation_volume=31; citation_number=31; citation_pages=833-844; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Degradation of sigma 32, the heat shock regulator in Escherichia coli, is governed by HflB.; citation_author=C Herman; citation_author=D Thevenet; citation_author=R D'Ari; citation_author=P Bouloc; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=32; citation_pages=3516-3520; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Proteolysis of the phage lambda CII regulatory protein by FtsH (HflB) of Escherichia coli.; citation_author=Y Shotland; citation_author=S Koby; citation_author=D Teff; citation_author=N Mansur; citation_author=DA Oren; citation_journal_title=Mol Microbiol; citation_volume=24; citation_number=33; citation_pages=1303-1310; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=FtsH is required for proteolytic elimination of uncomplexed forms of SecY, an essential protein translocase subunit.; citation_author=A Kihara; citation_author=Y Akiyama; citation_author=K Ito; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=34; citation_pages=4532-4536; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Impaired lysogenisation of the Escherichia coli rpoA341 mutant by bacteriophage lambda is due to the inability of CII to act as a transcriptional activator.; citation_author=M Obuchowski; citation_author=H Giladi; citation_author=S Koby; citation_author=A Szalewska-Palasz; citation_author=A Wegrzyn; citation_journal_title=Mol Gen Genet; citation_volume=254; citation_number=35; citation_pages=304-311; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000363" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Phage Lambda CIII: A Protease Inhibitor Regulating the Lysis-Lysogeny Decision"/>
    <meta name="twitter:description" content="The ATP-dependent protease FtsH (HflB) complexed with HflKC participates in post-translational control of the lysis-lysogeny decision of bacteriophage lambda by rapid degradation of lambda CII. Both phage-encoded proteins, the CII transcription activator and the CIII polypeptide, are required for efficient lysogenic response. The conserved CIII is both an inhibitor and substrate of FtsH. Here we show that the protease inhibitor CIII is present as oligomeric amphipathic &alpha; helical structures and functions as a competitive inhibitor of FtsH by preventing binding of the CII substrate. We identified single alanine substitutions in CIII that abolish its activity. We characterize a dominant negative effect of a CIII mutant. Thus, we suggest that CIII oligomrization is required for its function. Real-time analysis of CII activity demonstrates that the effect of CIII is not seen in the absence of either FtsH or HflKC. When CIII is provided ectopically, CII activity increases linearly as a function of the multiplicity of infection, suggesting that CIII enhances CII stability and the lysogenic response. FtsH function is essential for cellular viability as it regulates the balance in the synthesis of phospholipids and lipopolysaccharides. Genetic experiments confirmed that the CIII bacteriostatic effects are due to inhibition of FtsH. Thus, the early presence of CIII following infection stimulates the lysogenic response, while its degradation at later times ensures the reactivation of FtsH allowing the growth of the established lysogenic cell."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000363.g008"/>

  <meta property="og:title" content="Phage Lambda CIII: A Protease Inhibitor Regulating the Lysis-Lysogeny Decision" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000363" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2859'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6556'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9630&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000363">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000363" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000363&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Phage Lambda CIII: A Protease Inhibitor Regulating the Lysis-Lysogeny Decision
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Oren Kobiler
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:orenk@ekmd.huji.ac.il">orenk@ekmd.huji.ac.il</a></p>

                <p>Affiliation:
                  Department of Molecular Genetics and Biotechnology, The Hebrew University-Hadassah Medical School, Jerusalem, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Assaf Rokney, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Genetics and Biotechnology, The Hebrew University-Hadassah Medical School, Jerusalem, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Amos B. Oppenheim
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Genetics and Biotechnology, The Hebrew University-Hadassah Medical School, Jerusalem, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 11, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000363</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000363" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000363" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000363" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000363" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000363-g001" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000363-g002" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000363-g003" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000363-g004" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000363-g005" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000363-g006" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000363-g007" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000363-t001" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000363-t002" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000363-g008" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000363">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000363" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The ATP-dependent protease FtsH (HflB) complexed with HflKC participates in post-translational control of the lysis-lysogeny decision of bacteriophage lambda by rapid degradation of lambda CII. Both phage-encoded proteins, the CII transcription activator and the CIII polypeptide, are required for efficient lysogenic response. The conserved CIII is both an inhibitor and substrate of FtsH. Here we show that the protease inhibitor CIII is present as oligomeric amphipathic α helical structures and functions as a competitive inhibitor of FtsH by preventing binding of the CII substrate. We identified single alanine substitutions in CIII that abolish its activity. We characterize a dominant negative effect of a CIII mutant. Thus, we suggest that CIII oligomrization is required for its function. Real-time analysis of CII activity demonstrates that the effect of CIII is not seen in the absence of either FtsH or HflKC. When CIII is provided ectopically, CII activity increases linearly as a function of the multiplicity of infection, suggesting that CIII enhances CII stability and the lysogenic response. FtsH function is essential for cellular viability as it regulates the balance in the synthesis of phospholipids and lipopolysaccharides. Genetic experiments confirmed that the CIII bacteriostatic effects are due to inhibition of FtsH. Thus, the early presence of CIII following infection stimulates the lysogenic response, while its degradation at later times ensures the reactivation of FtsH allowing the growth of the established lysogenic cell.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kobiler O, Rokney A, Oppenheim AB (2007) Phage Lambda CIII: A Protease Inhibitor Regulating the Lysis-Lysogeny Decision. PLoS ONE 2(4):
          e363.
            doi:10.1371/journal.pone.0000363</p><p><strong>Academic Editor: </strong>Christophe Herman, Baylor College of Medicine, United States of America</p><p><strong>Received:</strong> December 11, 2006; <strong>Accepted:</strong> March 21, 2007; <strong>Published:</strong> April 11, 2007</p><p><strong>Copyright:</strong> © 2007 Kobiler et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported in part by the Israel Science Foundation (grant # 489/01-1 and 340/04) and by an NIH grant R21A I071240-01.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Proteolysis of key regulatory factors is an important control element of gene activity both in eukaryotic and prokaryotic cells. In bacteria degradation by ATP-dependent proteases, belonging to the AAA+ superfamily, participates in regulation of many developmental pathways: the heat shock response, starvation adaptation, DNA damage repair, capsular polysaccharide biosynthesis, sporulation and control of bacteriophage development <a href="#pone.0000363-Gottesman1">[1]</a>–<a href="#pone.0000363-Kobiler1">[5]</a>. Specific adaptor proteins are known to modify the interaction of substrates with ATP-dependent proteases (reviewed in <a href="#pone.0000363-Dougan1">[6]</a>–<a href="#pone.0000363-Ades1">[8]</a>). However, there are only three known intracellular inhibitory polypeptides. The phage T4 PinA protein inhibits the Lon protease <a href="#pone.0000363-Hilliard1">[9]</a>, and both the <em>Bacillus</em> species sporulation regulator SpoVM <a href="#pone.0000363-Cutting1">[10]</a> and the phage λ CIII <a href="#pone.0000363-Herman1">[11]</a>, <a href="#pone.0000363-Kobiler2">[12]</a> inhibit the FtsH protease. Both FtsH inhibitors, SpoVM and CIII, were predicted to form amphipathic α helices and are degraded by FtsH <a href="#pone.0000363-Kobiler2">[12]</a>–<a href="#pone.0000363-Prajapati1">[14]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The FtsH protease is the only essential ATP-dependent protease in <em>E. coli</em>. <a href="#pone.0000363-Tomoyasu1">[15]</a>, <a href="#pone.0000363-Ito1">[16]</a>. It is a membrane-bound homohexamer enzyme made of three major domains: a trans-membrane domain, an ATPase domain and a protease domain. FtsH is complexed with HflKC forming an FtsH<sub>6</sub>-HflKC<sub>6</sub> holoenzyme, which is present in the cell in less than 100 copies. FtsH degrades membrane proteins <a href="#pone.0000363-Kihara1">[17]</a> and a number of cytoplasmic proteins such as LpxC, σ32, SsrA-tagged proteins and the bacteriophage λCII and CIII proteins. Degradation of LpxC by FtsH is required for <em>Escherichia coli</em> viability, as the levels of LpxC are essential for maintaining the balance in the synthesis of phospholipids and lipopolysaccarides.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Bacteriophage λ infection may activate either the lytic or the lysogenic developmental pathway <a href="#pone.0000363-Ptashne1">[18]</a>, <a href="#pone.0000363-Oppenheim1">[19]</a>. In λ infection, physiological conditions as low temperature <a href="#pone.0000363-Obuchowski1">[20]</a>, starvation of the cells and high multiplicity of infection (MOI) <a href="#pone.0000363-Kourilsky1">[21]</a> are known to favor lysogeny. A few phage functions are specifically required for the lysogenic response <a href="#pone.0000363-Kaiser1">[22]</a>, <a href="#pone.0000363-Reichardt1">[23]</a>. The CII transcriptional activator, which is a key regulator of the lysis-lysogeny decision, induces three promoters essential for the lysogenic pathway. CII is required for the initial synthesis of the CI repressor from the pE promoter and of the integration protein Int, from the pI promoter. In addition, CII activates the paQ promoter and thus inhibits the Q antiterminator essential for lytic gene expression. The CII transcriptional activator is subjected to multilevel controls <a href="#pone.0000363-Oppenheim1">[19]</a>. High levels of the CII protein, that are required for the activation of the lysogenic developmental pathway, are facilitated by λ CIII, a 54-residue peptide which protects CII from rapid degradation by FtsH <a href="#pone.0000363-Herman1">[11]</a>, <a href="#pone.0000363-Kobiler2">[12]</a>. The CIII protein was also shown to induce the heat shock response by stabilizing σ32 <a href="#pone.0000363-Kornitzer1">[13]</a>. A 24-amino acid (residues 14–37) region of the λ CIII protein, which is essential and sufficient for CIII activity, was predicted to form a conserved amphipathic α helix <a href="#pone.0000363-Kornitzer1">[13]</a>. <em>In vitro</em> assays in a purified system showed that CIII inhibits FtsH proteolysis activity and can be degraded by the enzyme <a href="#pone.0000363-Kobiler2">[12]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In this work we present novel findings on the structure and mechanism of action of CIII <em>in vitro</em> and analyze its <em>in vivo</em> functions. We demonstrate that CIII possesses an amphipathic alpha helical structure. It is present in solution as higher order complex structures and acts as a competitive inhibitor of FtsH by preventing the binding of CII. We further show that both FtsH and HlfKC contribute to the down-regulation of CII activity following infection. Moreover, real-time measurements of GFP reporter fusions demonstrate that CIII levels have a profound influence on CII stability <em>in vivo</em> suggesting that CIII may control the lysis-lysogeny decision. Finally, we demonstrate that the cause for the bacteriostatic effect of CIII is inhibition of FtsH that affects the balance in lipid membrane composition.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Conservation of CIII</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The recent surge in sequencing of bacteriophages and of prophages yielded an increasing number of conserved <em>cIII</em> encoding genes. These can be classified into two classes: the λ CIII protein homologs and the HK022 CIII homologs (<a href="#pone-0000363-g001">Figure 1A</a>). The larger λ CIII group includes the phage 933W, which is the best studied phage encoding for the Stx2 Shiga toxin (see <a href="#pone.0000363-Tyler1">[24]</a>). The HK022 CIII <a href="#pone.0000363-Oberto1">[25]</a> group includes in addition, only HK097 and H-19B that encode for the Stx1 Shiga toxin. In <a href="#pone-0000363-g001">figure 1A</a> only the HK022 CIII is shown as all three proteins have identical sequence. The 24-amino acid (residues 14–37 underlined in <a href="#pone-0000363-g001">figure 1</a>) region essential for λ CIII activity <a href="#pone.0000363-Kornitzer1">[13]</a> is the most conserved sequence. This sequence was previously predicted to form an amphipathic α helix containing a short coiled coil structure <a href="#pone.0000363-Kornitzer1">[13]</a>. In this region the a and d coiled coil positions are mostly occupied by hydrophobic amino acids. Additionally, a relatively high number of conserved positive and negative residues are found. Interestingly, five spaced residues, including two glycines, are highly conserved at the amino terminus. The function of these residues is not known.</p>
<div class="figure" id="pone-0000363-g001"><div class="img"><a name="pone-0000363-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Conservation of CIII in phage and bacteria.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The multi sequences alignment was done using the ClustalX (version 1.81) program. Conserved or partially conserved residues are colored according to their biochemical properties. The essential region for CIII activity is underlined and the positions predicted coiled coil (by the MARCOIL) is marked above the sequence. (A) Alignment of CIII proteins from different phages and prophages (represented by the phage name or the bacterial strain carrying the prophage, respectively). CIII proteins were collected from the BLAST program by using the lambda CIII sequence as a query for the non-redundant database. A separate search using the HK022 CIII sequence was done. The HK022 CIII sequence is identical to that of HK097A and H-19B. A complete list of all the phages and prophages in which CIII homologs were recognized, is presented in the supplementary material. (B) Alignment of CIII and SpoVM proteins was carried out with the lambda CIII protein aligned with three SpoVM proteins from different Bacillus strains. (C) Alignment of CIII and YjhS hypothetical proteins that carry the DUF1737 and DUF303, was carried out. The three YjhS-like proteins were derived from different Shigella toxin carrying phages. The HP stands for hypothetical protein without a specific name.</p>
<span>doi:10.1371/journal.pone.0000363.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The other FtsH inhibitors, the conserved SpoVM from <em>Bacillus</em> species, are also predicted to form an amphipathic α helix. <a href="#pone-0000363-g001">Figure 1B</a> shows alignment of CIII with the SpoVM peptides. The alignment suggests that these inhibitors are structurally similar.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Search of the genome database for CIII-related peptides, using the region essential for CIII activity as query, yielded one protein domain of unknown function, DUF1737 (<a href="#pone-0000363-g001">Figure 1C</a>). This domain was found at the N-terminus of 53 bacterial and bacteriophages open reading frames, and is joined to a domain of unknown function, DUF303, in 42 cases. These open reading frames are found downstream to Shiga toxin encoding genes in Stx converting phages, indicating a possible role in bacterial virulence.</p>


<h4>Purified CIII possesses an α helical structure and is able to oligomerize</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A highly purified preparation of His<sub>6</sub>-CIII was prepared as previously described <a href="#pone.0000363-Kobiler2">[12]</a> (<a href="#pone-0000363-g002">Figure 2A</a>). The CIII structure was analyzed in solution by circular dichroism (CD) at 37°C (<a href="#pone-0000363-g002">Figure 2B</a>). Analysis of the CD spectra suggests that the protein forms an α helical structure. Furthermore the α helical content increases from 56% to 78% as the concentrations of CIII increases from 3.5 µM to 40 µM, suggesting that interactions between CIII protomers contribute to secondary structure stability. <a href="#pone-0000363-g003">Figure 3A</a> shows that multiple oligomers can be visualized using western blot analysis and that CIII oligomers can partially resist electrophoresis in the presence of SDS. To determine the oligomerization state of CIII in solution, His<sub>6</sub>-CIII was chemically crosslinked by glutaraldehyde, the oligomers separated by electrophoresis and visualized by His<sub>6</sub> antibodies (<a href="#pone-0000363-g003">Figure 3B</a>). A ladder of ordered protein bands indicating increasing numbers of monomers up to a 12mer could be clearly detected. We conclude that CIII is present in solution in multiple oligomeric states. Furthermore, we detected CIII oligomers by electrophoresis of cells following λ infection (data not shown).</p>
<div class="figure" id="pone-0000363-g002"><div class="img"><a name="pone-0000363-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Circular dichroism of CIII.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) 200 pmol of purified His<sub>6</sub>-CIII (right lane) were loaded on 4–12% NuPAGE (invitrogen) and visualized by Coomassie staining. On the left lane protein size marker was loaded (sizes 49, 38, 28, 14, 6 and 3 kDa). (B) The Circular dichroism spectrum (wavelengths 195–255 nm) of the purified His<sub>6</sub>-CIII at 3.5 µM (light gray), 27 µM (dark gray) and 40 µM (black).</p>
<span>doi:10.1371/journal.pone.0000363.g002</span></div><div class="figure" id="pone-0000363-g003"><div class="img"><a name="pone-0000363-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Oligomerization of CIII.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>(A) 10 µl of different concentrations of purified His<sub>6</sub>-CIII protein were loaded on 4–12% NuPAGE (invitrogen) and visualized by western blotting using commercial antibodies against the His6 tag. (B) 10 µM of His<sub>6</sub>-CIII was incubated in the presence (lanes 2,3) or absence (lane 1) of 0.25% Glutaraldehyde and the cross-linking reaction was stopped after 1 or 5 minutes by 200 mM Glycine (lanes 2 and 3 respectively). The reactions were loaded on 4–12% NuPAGE (invitrogen) and visualized by western blotting using commercial antibodies against the His6 tag.</p>
<span>doi:10.1371/journal.pone.0000363.g003</span></div>

<h4>CIII functions as a competitive inhibitor of FtsH</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To determine the mechanism of FtsH inhibition by CIII we performed an <em>in vitro</em> proteolysis assay of CII in the presence of increasing CIII to FtsH ratios (<a href="#pone-0000363-g004">Figure 4A</a>). We found a linear inverse correlation between CIII concentration and the ability of FtsH to degrade CII. These results suggest a simple dose-dependent inhibition. We found that degradation of 1 µM CII is completely inhibited by 30 µM CIII whereas for 4 µM CII, more than 60 µM CIII are required for complete inhibition (<a href="#pone-0000363-g004">Figure 4B</a>). These findings suggest a competitive inhibitory mechanism. To test if CIII competes with CII at the level of binding to FtsH we carried out cross-linking experiments. We found that CIII inhibits the formation of CII-FtsH complexes (<a href="#pone-0000363-g004">Figure 4C and D</a>). The concentrations of CIII required for inhibition of CII binding to FtsH correlate with the levels needed to inhibit CII proteolysis, suggesting that CIII acts by competing for the substrate recognition site of FtsH. During the proteolysis assay the CIII concentrations also decrease, therefore when comparing the proteolysis assay to the binding assay higher levels of CIII are required in the former to obtain similar inhibition.</p>
<div class="figure" id="pone-0000363-g004"><div class="img"><a name="pone-0000363-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Purified CIII acts as a competitive inhibitor of FtsH.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) 4 µM of CII were incubated with 1 µM GST-FtsH at 42<sup>0</sup>C for 5 minutes in the presence of increasing concentrations (as specified) of His<sub>6</sub>-CIII. The reactions were loaded on 4–12% NuPAGE (invitrogen) and visualized by Coomassie stain. First lane indicates the amount of CII at starting time. (B) The CII levels remaining after degradation experiment shown in A, were calculated using the “NIH image” program and plotted versus CIII levels (full diamonds and full lines). A similar experiment with only 1 µM of CII being degraded is shown in empty diamonds and a dotted line. (C) The binding assays were done in the presence of 10mM AMP-PNP as described in the <a href="#s4">materials and methods</a>. The reactions were loaded on 4–12% NuPAGE (invitrogen) and visualized by western blotting using antibodies raised against the CII protein. (D) CII levels bound in complex to GST-FtsH were calculated using the “NIH image” program and average from two different experiments was plotted versus CIII levels. The error bars are +/− SD.</p>
<span>doi:10.1371/journal.pone.0000363.g004</span></div>

<h4>The <em>ftsH</em> and <em>hflKC</em> mutant strains allow over expression of CII independent of CIII</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Following phage infection, CII and CIII are required for efficient lysogeny. CII induces the expression of the CI repressor from the pE promoter. FtsH, which is found in holoenzyme complexes with HflKC, degrades both CII and CIII. It was shown that mutations in <em>ftsH</em> or <em>hflKC</em> promote high frequency of lysogenization that is independent of CIII, while mutations in the <em>cII</em> gene allow for phage growth <a href="#pone.0000363-Belfort1">[26]</a>, <a href="#pone.0000363-Herskowitz1">[27]</a>. To obtain quantitative data on the role of CIII in the absence of FtsH or HflK we carried out infections of bacterial mutant strains with λ wt and mutants under conditions that greatly favor the lysogenic pathway. CII activity was assayed by a real-time kinetic readout from the pE-GFP reporter (<a href="#pone-0000363-g005">Figure 5, blue lines</a>) <a href="#pone.0000363-Kobiler3">[28]</a>. In parallel we assayed the lytic pathway by measuring Q activity using the pR′-tR′-GFP reporter (<a href="#pone-0000363-g005">Figure 5, red lines</a>) <a href="#pone.0000363-Kobiler3">[28]</a>. The results show that levels of CII in both <em>ftsH<sup>−</sup></em> or <em>hflKC<sup>−</sup></em> strains are much higher than those obtained following infection of the isogenic wild-type host. Furthermore, both host mutants show efficient suppression of the <em>cIII</em> mutation, suggesting that in the absence of FtsH and HflKC, CIII is dispensable. However, the loss of CII activity, following a peak, suggests that additional proteases participate in the degradation of CII.</p>
<div class="figure" id="pone-0000363-g005"><div class="img"><a name="pone-0000363-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>FtsH and HflA affect the λ genetic network.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>CII activity is reported by pE-<em>gfp</em> fusions (blue diamonds), whereas Q activity is reported by pR‘-tR‘-<em>gfp</em> fusions (red circles). The promoter activity as function of time of cultures of wild type strain (A, D, G) <em>hflA</em>- strain (B, E, H) or <em>ftsH</em> deleted strain (C, F, I) infected with λ<em>c+</em> (A, B, C), λ<em>cIII<sup>−</sup></em> (D, E, F) and λ<em>cII<sup>−</sup></em> (G, H, I) is shown. All experiments were carried out as described in the <a href="#s4">materials and methods</a> at a MOI of 6.</p>
<span>doi:10.1371/journal.pone.0000363.g005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We have previously shown that the Q function, measured from the pR′-tR′-GFP, is very low under conditions favoring the lysogenic pathway, mainly due to the presence of high CII levels <a href="#pone.0000363-Kobiler3">[28]</a>. Consistent with our previous findings, <a href="#pone-0000363-g005">Figure 5</a> shows that infections with a λ <em>cII<sup>−</sup></em> mutant phage lead to high Q activity. Q activity is observed even in the absence of FtsH or HflKC, suggesting that the effect of these proteins on the lytic pathway is mediated mainly by CII. We note however, that infections of the <em>hflK</em> mutant result in significantly lower levels of Q activity. Presently, we do not have a simple explanation for this result.</p>


<h4>The level of CIII activity controls the output from the CII-dependent pE promoter</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Genetic experiments indicated that approximately 3-fold increase in the translation of CIII greatly favors the lysogenic response <a href="#pone.0000363-Altuvia1">[29]</a>. To determine how the continuous supply of CIII would affect CII activity we infected cells carrying a double plasmid system with λ strains. A compatible plasmid (pCTCIII) carrying the <em>cIII</em> gene under the <em>pTac</em> inducible promoter was introduced into wild type cells carrying one of the λ promoter-<em>gfp</em> fusion plasmids. Following infection of these cells in the presence or absence of the IPTG inducer, the CII and Q activities were monitored (<a href="#pone-0000363-g006">Figure 6</a>, blue and red lines respectively). The results show that ectopic supply of CIII increases both the time and the amplitude of CII activity expressed by the infecting phages. As expected, the effect of CIII supplied in trans on CII and Q activities is much more dramatic in infection with a λ <em>cIII<sup>−</sup></em> mutant than infection by the wild-type phage. Interestingly, ectopic addition of CIII reduces the output signal from the pR′-tR′-GFP plasmid even in the absence of CII. It is possible that by inducing the heat-shock response CIII can further modulate Q antitermination activity <a href="#pone.0000363-Kornitzer1">[13]</a>. Consistent with this hypothesis, we found that following infection either by λ <em>c+</em> or by λ <em>cII<sup>−</sup></em> at 42°C, a significant decrease in Q activity was observed (data not shown).</p>
<div class="figure" id="pone-0000363-g006"><div class="img"><a name="pone-0000363-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>CIII levels effect on CII and Q activity <em>in vivo.</em></span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>CII activity is reported by pE-<em>gfp</em> fusions (blue diamonds), whereas Q activity is reported by pR‘-tR‘-<em>gfp</em> fusions (red circles). Promoter activity is given as a function of time of infected cultures carrying the pCTCIII plasmid in the presence or absence of 0.1mM IPTG (B, D, F or A, C, E respectively) with λ<em>c+</em> (A, B), λ<em>cIII<sup>−</sup></em> (C, D), and λ<em>cII<sup>−</sup></em> (E, F). All measurements were carried out as described in the <a href="#s4">materials and methods</a> at a MOI of 6.</p>
<span>doi:10.1371/journal.pone.0000363.g006</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The experimental setup allowed us to test the dependence of CII activity on the number of <em>cII</em> copies carried by the infecting phages in the presence of equal levels of CIII that is ectopically expressed. The results show a linear increase in CII activity as function of MOI (<a href="#pone-0000363-g007">Figure 7</a>). By comparing infection of λ c+ and λ <em>cIII<sup>−</sup></em> phages, we found that the amount of CIII made from the infecting phage has an effect when MOI is greater than 4, and little or no effect at low MOI. However, CIII supplied from the plasmid increases CII function at all MOI's, allowing the phage to follow the lysogenic pathway even at low MOI. These results indicate that CIII can modulate the phage genetic network.</p>
<div class="figure" id="pone-0000363-g007"><div class="img"><a name="pone-0000363-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>CIII levels affect multiplicity of infection.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>The total fluorescences obtain from the pE-GFP plasmid, 80 minutes after infection were plotted against the MOI of each experiment. λ<em>cIII<sup>−</sup></em> infections of bacteria carrying the pCTCIII plasmid were followed in the absence (light blue squares) or presence of 0.1 mM IPTG (blue squares) or of 0.2 mM IPTG (purple squares). λ<em>c+</em> (red×symbol) and λ<em>cIII<sup>−</sup></em> (orange×symbol) infections of bacteria without the pCTCIII plasmid are shown as reference.</p>
<span>doi:10.1371/journal.pone.0000363.g007</span></div>

<h4>CIII dominant negative mutant</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>We hypothesized that CIII oligomerization is required for efficient FtsH inhibition. We thus predict that it should be possible to identify CIII mutants that would be dominant negative due to formation of inactive CIII<sub>mutant</sub>-CIII<sub>wild-type</sub> hetrooligomers. To find such a mutant we introduced, using recombineering <a href="#pone.0000363-Oppenheim2">[30]</a>, a set of five alanine substitutions into the essential region of CIII within the λ genome. The mutants S17A, R24A, L26A, R27A and R32A show a <em>cIII<sup>−</sup></em> phenotype in plaque morphology and in CII activity, using the pE-GFP fluorescence assay (data not shown). Two mutants CIII<sub>R27A</sub> and CIII<sub>R32A</sub> were inserted into a plasmid under the tac promoter, for further analysis. Next, we confirmed by electrophoresis that these mutants formed oligomers at efficiency similar to that of wild type CIII (data not shown). We then tested their ability to cause dominant negative inhibition of wild type CIII activity. As shown in <a href="#pone-0000363-t001">Table 1</a>, the plating of wild type phage on a strain carrying the CIII<sub>R32A</sub> plasmid leads to a clear plaque formation. Consistent with these results, we found reduction in the frequency of lysogeny from 10% to 0.22% in the strain carrying the R32A mutant plasmid. These results suggest that the CIII<sub>R32A</sub> mutant inhibits wild type CIII activity and support the hypothesis that CIII oligomerization enhances FtsH inhibition.</p>
<div class="figure" id="pone-0000363-t001"><div class="img"><a name="pone-0000363-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000363.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000363.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>The dominant negative effect of CIII mutants as observed by plaque morphology.</span></strong></p><span>doi:10.1371/journal.pone.0000363.t001</span></div>

<h4>The host <em>sfhC21</em> mutation eliminates the bacteriostatic effect of CIII</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>LpxC catalyses the limiting reaction in lipopolysaccharide biosynthesis and controls the ratio between lipopolysaccharides and lipophospholipids in Gram-negative bacteria. Overproduction of LpxC causes toxic accumulation of membranes in the periplasm. LpxC levels are tightly regulated by the FtsH protease, which is therefore an essential protease in <em>E. coli.</em> <a href="#pone.0000363-Ogura1">[31]</a>. Genetic experiments showed that <em>sfhC21</em>, a mutation that up-regulated the FabA dehydrase (of the type II fatty acid synthase system) suppresses a deletion in <em>ftsH</em> <a href="#pone.0000363-Ogura1">[31]</a>.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>It was shown that overexpression of CIII impairs host cell growth and that this inhibition can be overcome by compensating high levels of FtsH capable of degrading CIII <a href="#pone.0000363-Kornitzer1">[13]</a>, <a href="#pone.0000363-Tomoyasu1">[15]</a>, <a href="#pone.0000363-Herman2">[32]</a>. To identify if FtsH is the sole target responsible for CIII growth inhibition we constructed a plasmid carrying the pBAD-CIII-GFP fusion. This construct prevents colony formation when introduced into a wild type bacterial strain, indicating that the CIII-GFP fusion protein retains a strong inhibitory effect on cell growth (<a href="#pone-0000363-t002">Table 2</a>). However expression of CIII-GFP did not impair bacterial growth of the Δ<em>ftsH sfhC21</em> double mutant strain or the <em>sfhC21</em> single mutant. These findings corroborate the results shown by Ogura et al. <a href="#pone.0000363-Ogura1">[31]</a> which demonstrated that the primary target of CIII in inhibiting cellular growth is FtsH proteolytic activity.</p>
<div class="figure" id="pone-0000363-t002"><div class="img"><a name="pone-0000363-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000363.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000363.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>CIII-GFP plasmid inhibits bacterial growth.</span></strong></p><span>doi:10.1371/journal.pone.0000363.t002</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The phage λ decision-making process has been studied extensively as a paradigm for genetic networks. The λ CIII inhibitor peptide is required for efficient lysogenic activation following phage infection. In this work we describe biochemically and genetically properties of this FtsH inhibitor. We identified similarities between CIII and the Bacillus SpoVM FtsH inhibitory peptide. Both inhibitors are degraded by FtsH and are found in association with the cell membrane <a href="#pone.0000363-Cutting1">[10]</a>, <a href="#pone.0000363-Herman1">[11]</a>. We found that CIII, as SpoVM, forms an amphipathic α helix. Sequence similarities suggest that the relatively conserved amphipathic α helix may be important for the inhibitory activity of both peptides.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>It is interesting to note that CIII homologs are found in a growing number of temperate phages. As FtsH is highly conserved in prokaryotic organisms as well as in the mitochondria and the chloroplasts of eukaryotic cells, one might expect that the inhibitory function of this protease will also be conserved. However, no CIII-like proteins are found to be present in the genome database. It is possible that CIII-like functions having different primary sequences do exist or less likely, efficient temporal inhibition of FtsH did not find its use in bacterial evolution.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Both CII and CIII are tightly regulated at the levels of transcription and translation <a href="#pone.0000363-Oppenheim1">[19]</a>. By inhibiting FtsH, CIII leads to an increase in the levels of CII activity. Thus, the CII/CIII/FtsH/HflKC act as a post-translational regulatory module (<a href="#pone-0000363-g008">Figure 8A</a>). Here we found that CIII acts as a competitive inhibitor of the host FtsH interfering with the binding of the CII substrate to the enzyme.</p>
<div class="figure" id="pone-0000363-g008"><div class="img"><a name="pone-0000363-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000363" data-uri="info:doi/10.1371/journal.pone.0000363.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000363.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000363.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000363.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000363.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Schematic model for CIII activity.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>(A) The role of the CIII/CII/FtsH/HflKC module in the lambda genetic decision. The FtsH and HflKC complex regulates CII and CIII levels and CIII controls FtsH/HflKC proteolytic activity. (B) A schematic drawing of the CIII mechanism of inhibition (based on the model presented in <a href="#pone.0000363-Ito1">[16]</a>). In panel 1 in the absence of CIII, CII binds the FtsH protease and is translocated into the enzymatic cavity where it is first unfolded and then degraded. In panel 2, CIII binds the FtsH protease and prevents CII from binding. In panel 3, the CIII protein, which is found as an oligomer, is also subjected to degradation.</p>
<span>doi:10.1371/journal.pone.0000363.g008</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>A number of biochemical properties of the CII/CIII/FtsH/HflKC module (<a href="#pone-0000363-g008">Figure 8B</a>) provide for its ability to finely tune CII levels, thus to tightly control the lysis-lysogeny decision. First, the FtsH/HflKC is present in the cell as huge, membrane-bound, highly active enzyme complexes, FtsH<sub>6</sub>HflKC<sub>6</sub> of which there are probably less the 100 molecules in a cell <a href="#pone.0000363-Ito1">[16]</a>. Furthermore, FtsH degrades CII very rapidly without requiring adaptor or chaperone functions <a href="#pone.0000363-Shotland1">[33]</a>. The CIII inhibitor is also subject to proteolysis by FtsH, which limits its activity to a short time window and allows for its rapid elimination once the lysogenic state is established. The elimination of CIII is required as CIII is deleterious to bacterial host. Our results suggest that FtsH is the only target of CIII responsible for the bacteriostatic effect of CIII.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The structure-function relationships of CIII are not known. The role of the amphipathic region may be for improved binding to FtsH or for the interaction with the cytoplasmic membrane favoring its binding to the membrane-bound FtsH <a href="#pone.0000363-Herman1">[11]</a>. We identified the ability of CIII to form oligomers, which may interact via the predicted coiled coil motif of this amphipathic region. The dominant negative effect of the CIII<sub>R32A</sub> mutant over the wild type CIII strongly suggests that CIII functions <em>in vivo</em> in oligomeric form.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Many proteins of bacteriophage λ are regulated by rapid proteolysis by various proteases <a href="#pone.0000363-Kobiler1">[5]</a>. Interestingly, the key elements of the lysis lysogeny decision, the CII and CIII proteins, are mainly degraded by FtsH. We suggest that coevolutionary forces maintaining the balance between bacteria and the infecting phages preferred cells that carry the active protease critical for the regulation of lysis-lysogeny decision. The fittest mechanism was obtained by selecting the only essential ATP-dependent protease in <em>E. coli</em>.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and methods"></a><h3>Materials and methods</h3>
<h4>Bacterial strains and plasmids</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The <em>ftsH</em> deletion strain is W3110 <em>sfhC</em> ZAD220::Tn10 <em>ftsH</em>3::<em>Kan</em>R <a href="#pone.0000363-Ogura1">[31]</a>. The <em>hflK</em> mutant strain is W3110 <em>hflK</em>::<em>Kan</em>R prepared by P1 transduction. AD16 is strain carrying <em>lacI<sup>q</sup></em> <a href="#pone.0000363-Kihara2">[34]</a> The expression systems and purification processes of the FtsH, CII and CIII proteins were described before <a href="#pone.0000363-Kobiler2">[12]</a>. The CIII-GFP plasmid was constructed by insertion of the CIII gene into the pJHK5 plasmid (received from Marcia Goldberg) using the EcoRI and PstI restrictions sites for the 5′ and 3′ respectively. The lambda promoters-GFP fusion plasmids <a href="#pone.0000363-Kobiler3">[28]</a> were used for monitoring the lambda infection process. Plasmid pCTCIII <a href="#pone.0000363-Obuchowski2">[35]</a> carries the λ cIII gene downstream of the p<em>Tac</em> promoter such that expression of the λ cIII gene can be conditionally induced by addition of the isopropyl-thio-β-D-galactopyranoside (IPTG) inducer. The point mutations in the <em>cIII</em> gene were introduced using single strand DNA oligos as described <a href="#pone.0000363-Oppenheim2">[30]</a> and selected by formation of clear plaques. The mutations were confirmed by sequencing. The <em>cIII</em> mutant genes were amplified by PCR and cloned into the pQE30 plasmid (Qiagen) as described for the wild type CIII <a href="#pone.0000363-Kobiler2">[12]</a>.</p>


<h4>Fluorescence assays</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Strains carrying both one of the promoter-gfp plasmids and the pCT-CIII plasmid were grown overnight in LB medium supplemented with 0.2% maltose, 50 µg/ml ampicillin and 20 µg/ml chloramphenicol. Cells were concentrated by centrifugation to 6×10<sup>9</sup> cells per ml, and 25 µl samples then were infected with different amounts of freshly prepared phage lysates (at the specified MOI) on ice. After incubation for 30 min on ice, the infected cells were diluted to 0.5 ml in M9 supplemented with 0.5% glycerol, 0.2% maltose, 0.01% B1, 50 µg/ml ampicillin and 20 µg/ml chloramphenicol, in the presence or absence of isopropyl -D-thiogalactoside (IPTG) as specified. Immediately after dilution, 200 µl cultures (6×10<sup>7</sup> cells) were assayed in duplicates in 96-well plates for GFP activity. Plates were incubated at 37°C and assayed at 2-min intervals in a SPECTRAFluor Plus fluorimeter (Tecan, Maennedorf, Switzerland) with a short shaking interval between assays. The FtsH and HflK mutants experiments were done similarly with the exception that no chloramphenicol or IPTG were used in these experiments. The data analysis was carried out as described based on the average of two experiments <a href="#pone.0000363-Kobiler3">[28]</a>.</p>


<h4>Circular Dichroism</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The CD spectra (wavelengths 195–255nm) of a highly purified preparation of the His<sub>6</sub>-CIII diluted in PBS solution were recorded in a 0.5 mm path-length cuvette at 37°C on a Jobin et Yvon CD 6 spectropolarimeter with sampling intervals of 1 nm. The average of six scans is shown after smoothing the data by averaging each wavelength point with the previous two and next two points.</p>


<h4><em>In vitro</em> assays</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Proteolysis assays of GST-FtsH were performed as described <a href="#pone.0000363-Shotland1">[33]</a>. Cross-linking reactions contained 4 µM CII, 1 µM GST-FtsH, CIII in different concentrations and 0.1mg/ml BSA in PBS solution. The reactions were incubated in the presence or absence of 10mM AMP-PNP (no nucleotide dependence was observed) for 5 minutes at 42°C. 0.25% Glutaraldehyde was added and the reaction was incubated for two more minutes at 42°C and terminated by the addition of glycine to a final concentration of 200 mM and transferring the reaction mix to 0°C. For CIII oligomerization 10 µM of His<sub>6</sub>-CIII were incubated in PBS solution with 0.1 mg/ml BSA at 42°C. The cross-linking reaction started by addition of 0.25% Glutaraldehyde and was stopped by 200 mM Glycine and transferring the reaction mix to 0°C.</p>


<h4>Frequency of lysogeny assay</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Bacterial cultures (AD16; AD16/pCIII<sub>R32A</sub>) were grown, in LB medium and in LB-ampicilin medium, respectively. λ <em>c+ kn<sup>R</sup></em> phage <a href="#pone.0000363-Kobiler2">[12]</a> was adsorbed on ice for 30 min, at a multiplicity of 0.01. The infected cells were diluted into LB medium in the presence of 10 µM IPTG isopropyl β-d-thiogalactoside and incubated at 37°C for 45 min before spreading on LB plates containing 30 µg/ml kanamycin. Lysogens were scored as percentage Km<sup>R</sup> colonies of the input-infecting phage.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Gali Prag for fruitful discussions, Ariella Oppenheim and Ilan Rosenshine for the support and critical reading of the manuscript, Dr. Marcia Goldberg for plasmid pJHK5 and Dr. Teru Ogura for bacterial strains.</p>
<a id="article1.back1.ack1.p2" name="article1.back1.ack1.p2"></a><p>This article is dedicated to the memory of our mentor Amos B. Oppenheim (1934–2006), who passed away recently.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: OK AO. Performed the experiments: OK AR AO. Analyzed the data: OK AR AO. Contributed reagents/materials/analysis tools: OK AR AO. Wrote the paper: OK AR AO.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000363-Gottesman1" id="pone.0000363-Gottesman1"></a>Gottesman S (1996) Proteases and their targets in Escherichia coli. Annu Rev Genet  30: 465–506.  <ul class="find" data-citedArticleID="943490" data-doi="10.1146/annurev.genet.30.1.465"><li><a href="http://dx.doi.org/10.1146/annurev.genet.30.1.465" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteases+and+their+targets+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteases+and+their+targets+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000363-Kanemori1" id="pone.0000363-Kanemori1"></a>Kanemori M, Yanagi H, Yura T (1999) Marked instability of the sigma(32) heat shock transcription factor at high temperature. Implications for heat shock regulation. J Biol Chem  274: 22002–22007.  <ul class="find" data-citedArticleID="943508" data-doi="10.1074/jbc.274.31.22002"><li><a href="http://dx.doi.org/10.1074/jbc.274.31.22002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Marked+instability+of+the+sigma%2832%29+heat+shock+transcription+factor+at+high+temperature.+Implications+for+heat+shock+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Marked+instability+of+the+sigma%2832%29+heat+shock+transcription+factor+at+high+temperature.+Implications+for+heat+shock+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000363-Jenal1" id="pone.0000363-Jenal1"></a>Jenal U, Hengge-Aronis R (2003) Regulation by proteolysis in bacterial cells. Curr Opin Microbiol  6: 163–172.  <ul class="find" data-citedArticleID="943504" data-doi="10.1016/s1369-5274(03)00029-8"><li><a href="http://dx.doi.org/10.1016/s1369-5274(03)00029-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+by+proteolysis+in+bacterial+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+by+proteolysis+in+bacterial+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000363-Weichart1" id="pone.0000363-Weichart1"></a>Weichart D, Querfurth N, Dreger M, Hengge-Aronis R (2003) Global role for ClpP-containing proteases in stationary-phase adaptation of Escherichia coli. J Bacteriol  185: 115–125.  <ul class="find" data-citedArticleID="943548" data-doi="10.1128/jb.185.1.115-125.2003"><li><a href="http://dx.doi.org/10.1128/jb.185.1.115-125.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+role+for+ClpP-containing+proteases+in+stationary-phase+adaptation+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+role+for+ClpP-containing+proteases+in+stationary-phase+adaptation+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000363-Kobiler1" id="pone.0000363-Kobiler1"></a>Kobiler O, Oppenheim AB, Herman C (2004) Recruitment of host ATP-dependent proteases by bacteriophage lambda. J Struct Biol  146: 72–78.  <ul class="find" data-citedArticleID="943514" data-doi="10.1016/j.jsb.2003.10.021"><li><a href="http://dx.doi.org/10.1016/j.jsb.2003.10.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recruitment+of+host+ATP-dependent+proteases+by+bacteriophage+lambda." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recruitment+of+host+ATP-dependent+proteases+by+bacteriophage+lambda.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000363-Dougan1" id="pone.0000363-Dougan1"></a>Dougan DA, Mogk A, Bukau B (2002) Protein folding and degradation in bacteria: to degrade or not to degrade? That is the question. Cell Mol Life Sci  59: 1607–1616.  <ul class="find" data-citedArticleID="943488" data-doi="10.1007/pl00012487"><li><a href="http://dx.doi.org/10.1007/pl00012487" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+folding+and+degradation+in+bacteria%3A+to+degrade+or+not+to+degrade%3F+That+is+the+question." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+folding+and+degradation+in+bacteria%3A+to+degrade+or+not+to+degrade%3F+That+is+the+question.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000363-Gottesman2" id="pone.0000363-Gottesman2"></a>Gottesman S (2003) Proteolysis in bacterial regulatory circuits. Annu Rev Cell Dev Biol  19: 565–587.  <ul class="find" data-citedArticleID="943492" data-doi="10.1146/annurev.cellbio.19.110701.153228"><li><a href="http://dx.doi.org/10.1146/annurev.cellbio.19.110701.153228" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteolysis+in+bacterial+regulatory+circuits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteolysis+in+bacterial+regulatory+circuits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000363-Ades1" id="pone.0000363-Ades1"></a>Ades SE (2004) Proteolysis: Adaptor, adaptor, catch me a catch. Curr Biol  14: R924–926.  <ul class="find" data-citedArticleID="943480" data-doi="10.1016/j.cub.2004.10.015"><li><a href="http://dx.doi.org/10.1016/j.cub.2004.10.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteolysis%3A+Adaptor%2C+adaptor%2C+catch+me+a+catch." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteolysis%3A+Adaptor%2C+adaptor%2C+catch+me+a+catch.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000363-Hilliard1" id="pone.0000363-Hilliard1"></a>Hilliard JJ, Simon LD, Van Melderen L, Maurizi MR (1998) PinA inhibits ATP hydrolysis and energy-dependent protein degradation by Lon protease. J Biol Chem  273: 524–527.  <ul class="find" data-citedArticleID="943500" data-doi="10.1074/jbc.273.1.524"><li><a href="http://dx.doi.org/10.1074/jbc.273.1.524" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PinA+inhibits+ATP+hydrolysis+and+energy-dependent+protein+degradation+by+Lon+protease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PinA+inhibits+ATP+hydrolysis+and+energy-dependent+protein+degradation+by+Lon+protease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000363-Cutting1" id="pone.0000363-Cutting1"></a>Cutting S, Anderson M, Lysenko E, Page A, Tomoyasu T, et al.  (1997) SpoVM, a small protein essential to development in Bacillus subtilis, interacts with the ATP-dependent protease FtsH. J Bacteriol  179: 5534–5542.  <ul class="find" data-citedArticleID="943486"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=SpoVM%2C+a+small+protein+essential+to+development+in+Bacillus+subtilis%2C+interacts+with+the+ATP-dependent+protease+FtsH.&amp;auth=&amp;atitle=SpoVM%2C+a+small+protein+essential+to+development+in+Bacillus+subtilis%2C+interacts+with+the+ATP-dependent+protease+FtsH." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SpoVM%2C+a+small+protein+essential+to+development+in+Bacillus+subtilis%2C+interacts+with+the+ATP-dependent+protease+FtsH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SpoVM%2C+a+small+protein+essential+to+development+in+Bacillus+subtilis%2C+interacts+with+the+ATP-dependent+protease+FtsH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000363-Herman1" id="pone.0000363-Herman1"></a>Herman C, Thevenet D, D'Ari R, Bouloc P (1997) The HflB protease of Escherichia coli degrades its inhibitor lambda cIII. J Bacteriol  179: 358–363.  <ul class="find" data-citedArticleID="943494"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+HflB+protease+of+Escherichia+coli+degrades+its+inhibitor+lambda+cIII.&amp;auth=&amp;atitle=The+HflB+protease+of+Escherichia+coli+degrades+its+inhibitor+lambda+cIII." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+HflB+protease+of+Escherichia+coli+degrades+its+inhibitor+lambda+cIII." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+HflB+protease+of+Escherichia+coli+degrades+its+inhibitor+lambda+cIII.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000363-Kobiler2" id="pone.0000363-Kobiler2"></a>Kobiler O, Koby S, Teff D, Court D, Oppenheim AB (2002) The phage lambda CII transcriptional activator carries a C-terminal domain signaling for rapid proteolysis. Proc Natl Acad Sci U S A  99: 14964–14969.  <ul class="find" data-citedArticleID="943516" data-doi="10.1073/pnas.222172499"><li><a href="http://dx.doi.org/10.1073/pnas.222172499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+phage+lambda+CII+transcriptional+activator+carries+a+C-terminal+domain+signaling+for+rapid+proteolysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+phage+lambda+CII+transcriptional+activator+carries+a+C-terminal+domain+signaling+for+rapid+proteolysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000363-Kornitzer1" id="pone.0000363-Kornitzer1"></a>Kornitzer D, Altuvia S, Oppenheim AB (1991) The activity of the CIII regulator of lambdoid bacteriophages resides within a 24-amino acid protein domain. Proc Natl Acad Sci U S A  88: 5217–5221.  <ul class="find" data-citedArticleID="943520" data-doi="10.1073/pnas.88.12.5217"><li><a href="http://dx.doi.org/10.1073/pnas.88.12.5217" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+activity+of+the+CIII+regulator+of+lambdoid+bacteriophages+resides+within+a+24-amino+acid+protein+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+activity+of+the+CIII+regulator+of+lambdoid+bacteriophages+resides+within+a+24-amino+acid+protein+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000363-Prajapati1" id="pone.0000363-Prajapati1"></a>Prajapati RS, Ogura T, Cutting SM (2000) Structural and functional studies on an FtsH inhibitor from Bacillus subtilis. Biochim Biophys Acta  1475: 353–359.  <ul class="find" data-citedArticleID="943536" data-doi="10.1016/s0304-4165(00)00089-1"><li><a href="http://dx.doi.org/10.1016/s0304-4165(00)00089-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+and+functional+studies+on+an+FtsH+inhibitor+from+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+and+functional+studies+on+an+FtsH+inhibitor+from+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000363-Tomoyasu1" id="pone.0000363-Tomoyasu1"></a>Tomoyasu T, Gamer J, Bukau B, Kanemori M, Mori H, et al.  (1995) Escherichia coli FtsH is a membrane-bound, ATP-dependent protease which degrades the heat-shock transcription factor sigma 32. Embo J  14: 2551–2560.  <ul class="find" data-citedArticleID="943544"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Escherichia+coli+FtsH+is+a+membrane-bound%2C+ATP-dependent+protease+which+degrades+the+heat-shock+transcription+factor+sigma+32.&amp;auth=&amp;atitle=Escherichia+coli+FtsH+is+a+membrane-bound%2C+ATP-dependent+protease+which+degrades+the+heat-shock+transcription+factor+sigma+32." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Escherichia+coli+FtsH+is+a+membrane-bound%2C+ATP-dependent+protease+which+degrades+the+heat-shock+transcription+factor+sigma+32." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Escherichia+coli+FtsH+is+a+membrane-bound%2C+ATP-dependent+protease+which+degrades+the+heat-shock+transcription+factor+sigma+32.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000363-Ito1" id="pone.0000363-Ito1"></a>Ito K, Akiyama Y (2005) Cellular functions, mechanism of action, and regulation of FtsH protease. Annu Rev Microbiol  59: 211–231.  <ul class="find" data-citedArticleID="943502" data-doi="10.1146/annurev.micro.59.030804.121316"><li><a href="http://dx.doi.org/10.1146/annurev.micro.59.030804.121316" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cellular+functions%2C+mechanism+of+action%2C+and+regulation+of+FtsH+protease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cellular+functions%2C+mechanism+of+action%2C+and+regulation+of+FtsH+protease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000363-Kihara1" id="pone.0000363-Kihara1"></a>Kihara A, Akiyama Y, Ito K (1998) Different pathways for protein degradation by the FtsH/HflKC membrane-embedded protease complex: an implication from the interference by a mutant form of a new substrate protein, YccA. J Mol Biol  279: 175–188.  <ul class="find" data-citedArticleID="943510" data-doi="10.1006/jmbi.1998.1781"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.1781" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Different+pathways+for+protein+degradation+by+the+FtsH%2FHflKC+membrane-embedded+protease+complex%3A+an+implication+from+the+interference+by+a+mutant+form+of+a+new+substrate+protein%2C+YccA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Different+pathways+for+protein+degradation+by+the+FtsH%2FHflKC+membrane-embedded+protease+complex%3A+an+implication+from+the+interference+by+a+mutant+form+of+a+new+substrate+protein%2C+YccA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000363-Ptashne1" id="pone.0000363-Ptashne1"></a>Ptashne M (2004) A Genetic Switch: Phage Lambda Revisted: CSHL Press.   <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0000363-Oppenheim1" id="pone.0000363-Oppenheim1"></a>Oppenheim AB, Kobiler O, Stavans J, Court DL, Adhya S (2005) Switches in bacteriophage lambda development. Annu Rev Genet  39: 409–429.  <ul class="find" data-citedArticleID="943532" data-doi="10.1146/annurev.genet.39.073003.113656"><li><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.113656" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Switches+in+bacteriophage+lambda+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Switches+in+bacteriophage+lambda+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000363-Obuchowski1" id="pone.0000363-Obuchowski1"></a>Obuchowski M, Shotland Y, Koby S, Giladi H, Gabig M, et al.  (1997) Stability of CII is a key element in the cold stress response of bacteriophage lambda infection. J Bacteriol  179: 5987–5991.  <ul class="find" data-citedArticleID="943526"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Stability+of+CII+is+a+key+element+in+the+cold+stress+response+of+bacteriophage+lambda+infection.&amp;auth=&amp;atitle=Stability+of+CII+is+a+key+element+in+the+cold+stress+response+of+bacteriophage+lambda+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stability+of+CII+is+a+key+element+in+the+cold+stress+response+of+bacteriophage+lambda+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stability+of+CII+is+a+key+element+in+the+cold+stress+response+of+bacteriophage+lambda+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000363-Kourilsky1" id="pone.0000363-Kourilsky1"></a>Kourilsky P, Knapp A (1974) Lysogenization by bacteriophage lambda. III. Multiplicity dependent phenomena occuring upon infection by lambda. Biochimie  56: 1517–1523.  <ul class="find" data-citedArticleID="943522" data-doi="10.1016/s0300-9084(75)80275-6"><li><a href="http://dx.doi.org/10.1016/s0300-9084(75)80275-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lysogenization+by+bacteriophage+lambda.+III.+Multiplicity+dependent+phenomena+occuring+upon+infection+by+lambda." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lysogenization+by+bacteriophage+lambda.+III.+Multiplicity+dependent+phenomena+occuring+upon+infection+by+lambda.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000363-Kaiser1" id="pone.0000363-Kaiser1"></a>Kaiser AD (1957) Mutations in a temperate bacteriophage affecting its ability to lysogenize Escherichia coli. Virology  3: 42–61.  <ul class="find" data-citedArticleID="943506" data-doi="10.1016/0042-6822(57)90022-3"><li><a href="http://dx.doi.org/10.1016/0042-6822(57)90022-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+a+temperate+bacteriophage+affecting+its+ability+to+lysogenize+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+a+temperate+bacteriophage+affecting+its+ability+to+lysogenize+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000363-Reichardt1" id="pone.0000363-Reichardt1"></a>Reichardt L, Kaiser AD (1971) Control of lambda repressor synthesis. Proc Natl Acad Sci U S A  68: 2185–2189.  <ul class="find" data-citedArticleID="943540" data-doi="10.1073/pnas.68.9.2185"><li><a href="http://dx.doi.org/10.1073/pnas.68.9.2185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+lambda+repressor+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+lambda+repressor+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000363-Tyler1" id="pone.0000363-Tyler1"></a>Tyler JS, Mills MJ, Friedman DI (2004) The operator and early promoter region of the Shiga toxin type 2-encoding bacteriophage 933W and control of toxin expression. J Bacteriol  186: 7670–7679.  <ul class="find" data-citedArticleID="943546" data-doi="10.1128/jb.186.22.7670-7679.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.22.7670-7679.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+operator+and+early+promoter+region+of+the+Shiga+toxin+type+2-encoding+bacteriophage+933W+and+control+of+toxin+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+operator+and+early+promoter+region+of+the+Shiga+toxin+type+2-encoding+bacteriophage+933W+and+control+of+toxin+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000363-Oberto1" id="pone.0000363-Oberto1"></a>Oberto J, Weisberg RA, Gottesman ME (1989) Structure and function of the nun gene and the immunity region of the lambdoid phage HK022. J Mol Biol  207: 675–693.  <ul class="find" data-citedArticleID="943524" data-doi="10.1016/0022-2836(89)90237-4"><li><a href="http://dx.doi.org/10.1016/0022-2836(89)90237-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+function+of+the+nun+gene+and+the+immunity+region+of+the+lambdoid+phage+HK022." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+function+of+the+nun+gene+and+the+immunity+region+of+the+lambdoid+phage+HK022.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000363-Belfort1" id="pone.0000363-Belfort1"></a>Belfort M, Noff D, Oppenheim AB (1975) Isolation, characterization and deletion mapping of amber mutations in the cll gene of phage lambda. Virology  63: 147–159.  <ul class="find" data-citedArticleID="943484" data-doi="10.1016/0042-6822(75)90380-3"><li><a href="http://dx.doi.org/10.1016/0042-6822(75)90380-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation%2C+characterization+and+deletion+mapping+of+amber+mutations+in+the+cll+gene+of+phage+lambda." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation%2C+characterization+and+deletion+mapping+of+amber+mutations+in+the+cll+gene+of+phage+lambda.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000363-Herskowitz1" id="pone.0000363-Herskowitz1"></a>Herskowitz I, Hagen D (1980) The lysis-lysogeny decision of phage lambda: explicit programming and responsiveness. Annu Rev Genet  14: 399–445.  <ul class="find" data-citedArticleID="943498" data-doi="10.1146/annurev.ge.14.120180.002151"><li><a href="http://dx.doi.org/10.1146/annurev.ge.14.120180.002151" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+lysis-lysogeny+decision+of+phage+lambda%3A+explicit+programming+and+responsiveness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+lysis-lysogeny+decision+of+phage+lambda%3A+explicit+programming+and+responsiveness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000363-Kobiler3" id="pone.0000363-Kobiler3"></a>Kobiler O, Rokney A, Friedman N, Court DL, Stavans J, et al.  (2005) Quantitative kinetic analysis of the bacteriophage lambda genetic network. Proc Natl Acad Sci U S A  102: 4470–4475.  <ul class="find" data-citedArticleID="943518" data-doi="10.1073/pnas.0500670102"><li><a href="http://dx.doi.org/10.1073/pnas.0500670102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+kinetic+analysis+of+the+bacteriophage+lambda+genetic+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+kinetic+analysis+of+the+bacteriophage+lambda+genetic+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000363-Altuvia1" id="pone.0000363-Altuvia1"></a>Altuvia S, Oppenheim AB (1986) Translational regulatory signals within the coding region of the bacteriophage lambda cIII gene. J Bacteriol  167: 415–419.  <ul class="find" data-citedArticleID="943482"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Translational+regulatory+signals+within+the+coding+region+of+the+bacteriophage+lambda+cIII+gene.&amp;auth=&amp;atitle=Translational+regulatory+signals+within+the+coding+region+of+the+bacteriophage+lambda+cIII+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translational+regulatory+signals+within+the+coding+region+of+the+bacteriophage+lambda+cIII+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translational+regulatory+signals+within+the+coding+region+of+the+bacteriophage+lambda+cIII+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000363-Oppenheim2" id="pone.0000363-Oppenheim2"></a>Oppenheim AB, Rattray AJ, Bubunenko M, Thomason LC, Court DL (2004) In vivo recombineering of bacteriophage lambda by PCR fragments and single-strand oligonucleotides. Virology  319: 185–189.  <ul class="find" data-citedArticleID="943534" data-doi="10.1016/j.virol.2003.11.007"><li><a href="http://dx.doi.org/10.1016/j.virol.2003.11.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+recombineering+of+bacteriophage+lambda+by+PCR+fragments+and+single-strand+oligonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+recombineering+of+bacteriophage+lambda+by+PCR+fragments+and+single-strand+oligonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000363-Ogura1" id="pone.0000363-Ogura1"></a>Ogura T, Inoue K, Tatsuta T, Suzaki T, Karata K, et al.  (1999) Balanced biosynthesis of major membrane components through regulated degradation of the committed enzyme of lipid A biosynthesis by the AAA protease FtsH (HflB) in Escherichia coli. Mol Microbiol  31: 833–844.  <ul class="find" data-citedArticleID="943530" data-doi="10.1046/j.1365-2958.1999.01221.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01221.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Balanced+biosynthesis+of+major+membrane+components+through+regulated+degradation+of+the+committed+enzyme+of+lipid+A+biosynthesis+by+the+AAA+protease+FtsH+%28HflB%29+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Balanced+biosynthesis+of+major+membrane+components+through+regulated+degradation+of+the+committed+enzyme+of+lipid+A+biosynthesis+by+the+AAA+protease+FtsH+%28HflB%29+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000363-Herman2" id="pone.0000363-Herman2"></a>Herman C, Thevenet D, D'Ari R, Bouloc P (1995) Degradation of sigma 32, the heat shock regulator in Escherichia coli, is governed by HflB. Proc Natl Acad Sci U S A  92: 3516–3520.  <ul class="find" data-citedArticleID="943496" data-doi="10.1073/pnas.92.8.3516"><li><a href="http://dx.doi.org/10.1073/pnas.92.8.3516" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Degradation+of+sigma+32%2C+the+heat+shock+regulator+in+Escherichia+coli%2C+is+governed+by+HflB." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Degradation+of+sigma+32%2C+the+heat+shock+regulator+in+Escherichia+coli%2C+is+governed+by+HflB.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000363-Shotland1" id="pone.0000363-Shotland1"></a>Shotland Y, Koby S, Teff D, Mansur N, Oren DA, et al.  (1997) Proteolysis of the phage lambda CII regulatory protein by FtsH (HflB) of Escherichia coli. Mol Microbiol  24: 1303–1310.  <ul class="find" data-citedArticleID="943542" data-doi="10.1046/j.1365-2958.1997.4231796.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1997.4231796.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteolysis+of+the+phage+lambda+CII+regulatory+protein+by+FtsH+%28HflB%29+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteolysis+of+the+phage+lambda+CII+regulatory+protein+by+FtsH+%28HflB%29+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000363-Kihara2" id="pone.0000363-Kihara2"></a>Kihara A, Akiyama Y, Ito K (1995) FtsH is required for proteolytic elimination of uncomplexed forms of SecY, an essential protein translocase subunit. Proc Natl Acad Sci U S A  92: 4532–4536.  <ul class="find" data-citedArticleID="943512" data-doi="10.1073/pnas.92.10.4532"><li><a href="http://dx.doi.org/10.1073/pnas.92.10.4532" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FtsH+is+required+for+proteolytic+elimination+of+uncomplexed+forms+of+SecY%2C+an+essential+protein+translocase+subunit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FtsH+is+required+for+proteolytic+elimination+of+uncomplexed+forms+of+SecY%2C+an+essential+protein+translocase+subunit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000363-Obuchowski2" id="pone.0000363-Obuchowski2"></a>Obuchowski M, Giladi H, Koby S, Szalewska-Palasz A, Wegrzyn A, et al.  (1997) Impaired lysogenisation of the Escherichia coli rpoA341 mutant by bacteriophage lambda is due to the inability of CII to act as a transcriptional activator. Mol Gen Genet  254: 304–311.  <ul class="find" data-citedArticleID="943528" data-doi="10.1007/s004380050420"><li><a href="http://dx.doi.org/10.1007/s004380050420" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impaired+lysogenisation+of+the+Escherichia+coli+rpoA341+mutant+by+bacteriophage+lambda+is+due+to+the+inability+of+CII+to+act+as+a+transcriptional+activator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impaired+lysogenisation+of+the+Escherichia+coli+rpoA341+mutant+by+bacteriophage+lambda+is+due+to+the+inability+of+CII+to+act+as+a+transcriptional+activator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.XML" value="76915"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.PDF" value="784520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g001.PNG_L" value="1552638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g001.PNG_M" value="187506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g001.PNG_S" value="5073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g001.TIF" value="2236336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g001.PNG_I" value="74859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g002.PNG_L" value="131911"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g002.PNG_M" value="54826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g002.PNG_S" value="2534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g002.TIF" value="145376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g002.PNG_I" value="18737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g003.PNG_L" value="175362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g003.PNG_M" value="68345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g003.PNG_S" value="2951"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g003.TIF" value="179172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g003.PNG_I" value="23552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g004.PNG_L" value="183780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g004.PNG_M" value="23204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g004.PNG_S" value="4147"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g004.TIF" value="216956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g004.PNG_I" value="39153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g005.PNG_L" value="690095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g005.PNG_M" value="94408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g005.PNG_S" value="3564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g005.TIF" value="1077988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g005.PNG_I" value="38523"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g006.PNG_L" value="326393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g006.PNG_M" value="114143"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g006.PNG_S" value="4903"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g006.TIF" value="483622"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g006.PNG_I" value="62082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g007.PNG_L" value="98939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g007.PNG_M" value="47436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g007.PNG_S" value="2923"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g007.TIF" value="197692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g007.PNG_I" value="24109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.t001.PNG_L" value="65714"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.t001.PNG_M" value="68411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.t001.PNG_S" value="2870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.t001.TIF" value="315590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.t001.PNG_I" value="31928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.t002.PNG_L" value="76841"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.t002.PNG_M" value="77625"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.t002.PNG_S" value="2934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.t002.TIF" value="361108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.t002.PNG_I" value="36874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g008.PNG_L" value="324569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g008.PNG_M" value="84599"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g008.PNG_S" value="7252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g008.TIF" value="549374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000363.g008.PNG_I" value="80391"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000363&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000363" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000363&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000363&volume=&issue=&title=Phage Lambda CIII: A Protease Inhibitor Regulating the Lysis-Lysogeny Decision&author_name=Oren%20Kobiler%2C%20Assaf%20Rokney%2C%20Amos%20B.%20Oppenheim&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000363" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000363" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000363" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000363&amp;t=Phage%20Lambda%20CIII%3A%20A%20Protease%20Inhibitor%20Regulating%20the%20Lysis-Lysogeny%20Decision" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000363&title=Phage%20Lambda%20CIII%3A%20A%20Protease%20Inhibitor%20Regulating%20the%20Lysis-Lysogeny%20Decision&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000363&amp;title=Phage%20Lambda%20CIII%3A%20A%20Protease%20Inhibitor%20Regulating%20the%20Lysis-Lysogeny%20Decision" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000363" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000363&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Phage Lambda CIII: A Protease Inhibitor Regulating the Lysis-Lysogeny Decision';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000363';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Phage%20Lambda%20CIII%3A%20A%20Protease%20Inhibitor%20Regulating%20the%20Lysis-Lysogeny%20Decision http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000363" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000363" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alpha+helix%22" title="Search for articles in the subject area:'Alpha helix'"><div class="flagText">Alpha helix</div></a>
              <div data-categoryid="35251" data-articleid="23564"
                   data-categoryname="Alpha helix"
                   class="flagImage" title="Flag 'Alpha helix' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+pathogens%22" title="Search for articles in the subject area:'Bacterial pathogens'"><div class="flagText">Bacterial pathogens</div></a>
              <div data-categoryid="42717" data-articleid="23564"
                   data-categoryname="Bacterial pathogens"
                   class="flagImage" title="Flag 'Bacterial pathogens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacteriophages%22" title="Search for articles in the subject area:'Bacteriophages'"><div class="flagText">Bacteriophages</div></a>
              <div data-categoryid="43083" data-articleid="23564"
                   data-categoryname="Bacteriophages"
                   class="flagImage" title="Flag 'Bacteriophages' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Competitive+inhibitors%22" title="Search for articles in the subject area:'Competitive inhibitors'"><div class="flagText">Competitive inhibitors</div></a>
              <div data-categoryid="24251" data-articleid="23564"
                   data-categoryname="Competitive inhibitors"
                   class="flagImage" title="Flag 'Competitive inhibitors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oligomers%22" title="Search for articles in the subject area:'Oligomers'"><div class="flagText">Oligomers</div></a>
              <div data-categoryid="20627" data-articleid="23564"
                   data-categoryname="Oligomers"
                   class="flagImage" title="Flag 'Oligomers' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteases%22" title="Search for articles in the subject area:'Proteases'"><div class="flagText">Proteases</div></a>
              <div data-categoryid="19447" data-articleid="23564"
                   data-categoryname="Proteases"
                   class="flagImage" title="Flag 'Proteases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="23564"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="23564"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5210'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5614'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3535&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=16819">Corresponding author</a><br>Posted by okobiler</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
