

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Evolution and Diversity of Clonal Bacteria: The Paradigm of Mycobacterium tuberculosis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001538"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001538" />

    <meta name="citation_title" content="Evolution and Diversity of Clonal Bacteria: The Paradigm of Mycobacterium tuberculosis"/>
    <meta itemprop="name" content="Evolution and Diversity of Clonal Bacteria: The Paradigm of Mycobacterium tuberculosis"/>

      <meta name="citation_author" content="Tiago Dos Vultos"/>
            <meta name="citation_author_institution" content="Unité de Génétique mycobactérienne, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Olga Mestre"/>
            <meta name="citation_author_institution" content="Unité de Génétique mycobactérienne, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Jean Rauzier"/>
            <meta name="citation_author_institution" content="Unité de Génétique mycobactérienne, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Marcin Golec"/>
            <meta name="citation_author_institution" content="Unité de Génétique mycobactérienne, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Nalin Rastogi"/>
            <meta name="citation_author_institution" content="Unité de la Tuberculose et des Mycobactéries, Institut Pasteur de Guadeloupe, Abymes, Guadeloupe"/>
      <meta name="citation_author" content="Voahangy Rasolofo"/>
            <meta name="citation_author_institution" content="Unité de la Tuberculose et des Mycobactéries, Institut Pasteur de Madagascar, Antananarivo, Madagascar"/>
      <meta name="citation_author" content="Tone Tonjum"/>
            <meta name="citation_author_institution" content="Centre for Molecular Biology and Neuroscience and Institute of Microbiology, University of Oslo, Oslo, Norway"/>
            <meta name="citation_author_institution" content="Centre for Molecular Biology and Neuroscience and Institute of Microbiology, Rikshospitalet, Oslo, Norway"/>
      <meta name="citation_author" content="Christophe Sola"/>
            <meta name="citation_author_institution" content="Unité de Génétique mycobactérienne, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Ivan Matic"/>
            <meta name="citation_author_institution" content="Institut National de la Santé et de la Recherche Médicale U571, Faculté de Médicine, Université Paris V, Paris, France"/>
      <meta name="citation_author" content="Brigitte Gicquel"/>
            <meta name="citation_author_institution" content="Unité de Génétique mycobactérienne, Institut Pasteur, Paris, France"/>

    <meta name="citation_date" content="2008/2/6"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001538.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1538"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Evolutionary history of Salmonella typhi.; citation_author=P Roumagnac; citation_author=FX Weill; citation_author=C Dolecek; citation_author=S Baker; citation_author=S Brisse; citation_journal_title=Science; citation_volume=314; citation_number=1; citation_pages=1301-1304; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A new evolutionary scenario for the Mycobacterium tuberculosis complex.; citation_author=R Brosch; citation_author=SV Gordon; citation_author=M Marmiesse; citation_author=P Brodin; citation_author=C Buchrieser; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=2; citation_pages=3684-3689; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mycobacterium tuberculosis complex genetic diversity: mining the fourth international spoligotyping database (SpolDB4) for classification, population genetics and epidemiology.; citation_author=K Brudey; citation_author=JR Driscoll; citation_author=L Rigouts; citation_author=WM Prodinger; citation_author=A Gori; citation_journal_title=BMC Microbiol; citation_volume=6; citation_number=3; citation_pages=23; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Global phylogeny of Mycobacterium tuberculosis based on single nucleotide polymorphism (SNP) analysis: insights into tuberculosis evolution, phylogenetic accuracy of other DNA fingerprinting systems, and recommendations for a minimal standard SNP set.; citation_author=I Filliol; citation_author=AS Motiwala; citation_author=M Cavatore; citation_author=W Qi; citation_author=MH Hazbon; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=4; citation_pages=759-772; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity in the Mycobacterium tuberculosis complex based on variable numbers of tandem DNA repeats.; citation_author=R Frothingham; citation_author=WA Meeker-O'Connell; citation_journal_title=Microbiology; citation_volume=144 ( Pt 5); citation_number=5; citation_pages=1189-1196; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Nature of DNA polymorphism in the direct repeat cluster of Mycobacterium tuberculosis; application for strain differentiation by a novel typing method.; citation_author=PM Groenen; citation_author=AE Bunschoten; citation_author=D van Soolingen; citation_author=JD van Embden; citation_journal_title=Mol Microbiol; citation_volume=10; citation_number=6; citation_pages=1057-1065; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of synonymous single nucleotide polymorphisms in Mycobacterium tuberculosis complex organisms: resolution of genetic relationships among closely related microbial strains.; citation_author=MM Gutacker; citation_author=JC Smoot; citation_author=CA Migliaccio; citation_author=SM Ricklefs; citation_author=S Hua; citation_journal_title=Genetics; citation_volume=162; citation_number=7; citation_pages=1533-1543; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Restricted structural gene polymorphism in the Mycobacterium tuberculosis complex indicates evolutionarily recent global dissemination.; citation_author=S Sreevatsan; citation_author=X Pan; citation_author=KE Stockbauer; citation_author=ND Connell; citation_author=BN Kreiswirth; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=8; citation_pages=9869-9874; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Variable human minisatellite-like regions in the Mycobacterium tuberculosis genome.; citation_author=P Supply; citation_author=E Mazars; citation_author=S Lesjean; citation_author=V Vincent; citation_author=B Gicquel; citation_journal_title=Mol Microbiol; citation_volume=36; citation_number=9; citation_pages=762-771; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evidence for recombination in Mycobacterium tuberculosis.; citation_author=X Liu; citation_author=MM Gutacker; citation_author=JM Musser; citation_author=YX Fu; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=10; citation_pages=8169-8177; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Horizontal transfer of a virulence operon to the ancestor of Mycobacterium tuberculosis.; citation_author=V Rosas-Magallanes; citation_author=P Deschavanne; citation_author=L Quintana-Murci; citation_author=R Brosch; citation_author=B Gicquel; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=11; citation_pages=1129-1135; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of Nudix hydrolase family members with an antimutator role in Mycobacterium tuberculosis and Mycobacterium smegmatis.; citation_author=T Dos Vultos; citation_author=J Blazquez; citation_author=J Rauzier; citation_author=I Matic; citation_author=B Gicquel; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=12; citation_pages=3159-3161; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mutations in putative mutator genes of Mycobacterium tuberculosis strains of the W-Beijing family.; citation_author=ME Rad; citation_author=P Bifani; citation_author=C Martin; citation_author=K Kremer; citation_author=S Samper; citation_journal_title=Emerg Infect Dis; citation_volume=9; citation_number=13; citation_pages=838-845; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=DNA repair in Mycobacterium tuberculosis. What have we learnt from the genome sequence?; citation_author=V Mizrahi; citation_author=SJ Andersen; citation_journal_title=Mol Microbiol; citation_volume=29; citation_number=14; citation_pages=1331-1339; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Median-joining networks for inferring intraspecific phylogenies.; citation_author=HJ Bandelt; citation_author=P Forster; citation_author=A Rohl; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=15; citation_pages=37-48; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A recently evolved sublineage of the Mycobacterium tuberculosis Beijing strain family is associated with an increased ability to spread and cause disease.; citation_author=M Hanekom; citation_author=GD van der Spuy; citation_author=E Streicher; citation_author=SL Ndabambi; citation_author=CR McEvoy; citation_journal_title=J Clin Microbiol; citation_volume=45; citation_number=16; citation_pages=1483-1490; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A phylogenomic study of DNA repair genes, proteins, and processes.; citation_author=JA Eisen; citation_author=PC Hanawalt; citation_journal_title=Mutat Res; citation_volume=435; citation_number=17; citation_pages=171-213; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Uracil incorporation: a source of pulse-labeled DNA fragments in the replication of the Escherichia coli chromosome.; citation_author=BK Tye; citation_author=J Chien; citation_author=IR Lehman; citation_author=BK Duncan; citation_author=HR Warner; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=75; citation_number=18; citation_pages=233-237; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Silent nucleotide polymorphisms and a phylogeny for Mycobacterium tuberculosis.; citation_author=L Baker; citation_author=T Brown; citation_author=MC Maiden; citation_author=F Drobniewski; citation_journal_title=Emerg Infect Dis; citation_volume=10; citation_number=19; citation_pages=1568-1577; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Comparisons of dN/dS are time dependent for closely related bacterial genomes.; citation_author=EP Rocha; citation_author=JM Smith; citation_author=LD Hurst; citation_author=MT Holden; citation_author=JE Cooper; citation_journal_title=J Theor Biol; citation_volume=239; citation_number=20; citation_pages=226-235; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular signatures of natural selection.; citation_author=R Nielsen; citation_journal_title=Annu Rev Genet; citation_volume=39; citation_number=21; citation_pages=197-218; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolution of mutation rates in bacteria.; citation_author=E Denamur; citation_author=I Matic; citation_journal_title=Mol Microbiol; citation_volume=60; citation_number=22; citation_pages=820-827; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microbial fitness and genome dynamics.; citation_author=T Tonjum; citation_author=E Seeberg; citation_journal_title=Trends Microbiol; citation_volume=9; citation_number=23; citation_pages=356-358; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Brief Bioinform; citation_volume=5; citation_number=24; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=DnaSP, DNA polymorphism analyses by the coalescent and other methods.; citation_author=J Rozas; citation_author=JC Sanchez-DelBarrio; citation_author=X Messeguer; citation_author=R Rozas; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=25; citation_pages=2496-2497; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001538" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Evolution and Diversity of Clonal Bacteria: The Paradigm of Mycobacterium tuberculosis"/>
    <meta name="twitter:description" content="BackgroundMycobacterium tuberculosis complex species display relatively static genomes and 99.9% nucleotide sequence identity. Studying the evolutionary history of such monomorphic bacteria is a difficult and challenging task.Principal FindingsWe found that single-nucleotide polymorphism (SNP) analysis of DNA repair, recombination and replication (3R) genes in a comprehensive selection of M. tuberculosis complex strains from across the world, yielded surprisingly high levels of polymorphisms as compared to house-keeping genes, making it possible to distinguish between 80% of clinical isolates analyzed in this study. Bioinformatics analysis suggests that a large number of these polymorphisms are potentially deleterious. Site frequency spectrum comparison of synonymous and non-synonymous variants and Ka/Ks ratio analysis suggest a general negative/purifying selection acting on these sets of genes that may lead to suboptimal 3R system activity. In turn, the relaxed fidelity of 3R genes may allow the occurrence of adaptive variants, some of which will survive. Furthermore, 3R-based phylogenetic trees are a new tool for distinguishing between M. tuberculosis complex strains.Conclusions/SignificanceThis situation, and the consequent lack of fidelity in genome maintenance, may serve as a starting point for the evolution of antibiotic resistance, fitness for survival and pathogenicity, possibly conferring a selective advantage in certain stressful situations. These findings suggest that 3R genes may play an important role in the evolution of highly clonal bacteria, such as M. tuberculosis. They also facilitate further epidemiological studies of these bacteria, through the development of high-resolution tools. With many more microbial genomes being sequenced, our results open the door to 3R gene-based studies of adaptation and evolution of other, highly clonal bacteria."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001538.g005"/>

  <meta property="og:title" content="Evolution and Diversity of Clonal Bacteria: The Paradigm of Mycobacterium tuberculosis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001538" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1677'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1642'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7885&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001538">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001538" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001538&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Evolution and Diversity of Clonal Bacteria: The Paradigm of <i>Mycobacterium tuberculosis</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tiago Dos Vultos, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité de Génétique mycobactérienne, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Olga Mestre, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité de Génétique mycobactérienne, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean Rauzier, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité de Génétique mycobactérienne, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marcin Golec, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité de Génétique mycobactérienne, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nalin Rastogi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité de la Tuberculose et des Mycobactéries, Institut Pasteur de Guadeloupe, Abymes, Guadeloupe
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Voahangy Rasolofo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité de la Tuberculose et des Mycobactéries, Institut Pasteur de Madagascar, Antananarivo, Madagascar
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tone Tonjum, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Centre for Molecular Biology and Neuroscience and Institute of Microbiology, University of Oslo, Oslo, Norway, 
                  Centre for Molecular Biology and Neuroscience and Institute of Microbiology, Rikshospitalet, Oslo, Norway
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christophe Sola, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité de Génétique mycobactérienne, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ivan Matic, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institut National de la Santé et de la Recherche Médicale U571, Faculté de Médicine, Université Paris V, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Brigitte Gicquel
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:bgicquel@pasteur.fr">bgicquel@pasteur.fr</a></p>

                <p>Affiliation:
                  Unité de Génétique mycobactérienne, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 06, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001538</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001538" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001538" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001538" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001538" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001538-g001" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001538-g002" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001538-g003" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001538-g004" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001538-t001" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001538-t002" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001538-t003" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001538-t004" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001538-t005" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001538-t006" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001538-g005" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001538">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001538" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p><em>Mycobacterium tuberculosis</em> complex species display relatively static genomes and 99.9% nucleotide sequence identity. Studying the evolutionary history of such monomorphic bacteria is a difficult and challenging task.</p>


<h3>Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We found that single-nucleotide polymorphism (SNP) analysis of DNA repair, recombination and replication (3R) genes in a comprehensive selection of <em>M. tuberculosis</em> complex strains from across the world, yielded surprisingly high levels of polymorphisms as compared to house-keeping genes, making it possible to distinguish between 80% of clinical isolates analyzed in this study. Bioinformatics analysis suggests that a large number of these polymorphisms are potentially deleterious. Site frequency spectrum comparison of synonymous and non-synonymous variants and Ka/Ks ratio analysis suggest a general negative/purifying selection acting on these sets of genes that may lead to suboptimal 3R system activity. In turn, the relaxed fidelity of 3R genes may allow the occurrence of adaptive variants, some of which will survive. Furthermore, 3R-based phylogenetic trees are a new tool for distinguishing between <em>M. tuberculosis</em> complex strains.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This situation, and the consequent lack of fidelity in genome maintenance, may serve as a starting point for the evolution of antibiotic resistance, fitness for survival and pathogenicity, possibly conferring a selective advantage in certain stressful situations. These findings suggest that 3R genes may play an important role in the evolution of highly clonal bacteria, such as <em>M. tuberculosis</em>. They also facilitate further epidemiological studies of these bacteria, through the development of high-resolution tools. With many more microbial genomes being sequenced, our results open the door to 3R gene-based studies of adaptation and evolution of other, highly clonal bacteria.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Dos Vultos T, Mestre O, Rauzier J, Golec M, Rastogi N, et al.  (2008) Evolution and Diversity of Clonal Bacteria: The Paradigm of <em>Mycobacterium tuberculosis</em>. PLoS ONE 3(2):
          e1538.
            doi:10.1371/journal.pone.0001538</p><p><strong>Academic Editor: </strong>Niyaz Ahmed, Centre for DNA Fingerprinting and Diagnostics, India</p><p><strong>Received:</strong> October 26, 2007; <strong>Accepted:</strong> December 30, 2007; <strong>Published:</strong> February 6, 2008</p><p><strong>Copyright:</strong> © 2008 Dos Vultos et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work received support from the European Commission (TB Adapt project LSHP-CT-2006-037919 and VACSIS project ICA4-CT-2002-10052).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Despite their different tropisms, phenotypes and pathogenicities, <em>M. tuberculosis</em> complex (MTC) strains are highly clonal: their nucleotide sequences are 99.9% identical and 16S rRNA sequences do not differ between MTC members, with the exception of <em>M. canetti</em>. It is difficult to study the evolutionary history of such mono-morphic bacteria <a href="#pone.0001538-Roumagnac1">[1]</a>. Several methods based on polymorphic loci or the sequencing of housekeeping genes have been used to distinguish between <em>M. tuberculosis</em> complex isolates <a href="#pone.0001538-Brosch1">[2]</a>–<a href="#pone.0001538-Supply1">[9]</a>. However, they have provided only low resolution and sparse functional information on how strains evolve and adapt to changes in environmental selection pressures, such as immune pressures and antimicrobial drug treatment.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Allelic variations in bacteria arise from random mutation, which may or may not be subject to selective pressure, horizontal gene transfer or recombination events. Evidence for horizontal transfer and recombination has recently been obtained, but exchange of genetic material in <em>M. tuberculosis</em> seems only to have occurred in the distant past <a href="#pone.0001538-Liu1">[10]</a>, <a href="#pone.0001538-RosasMagallanes1">[11]</a>. Other mechanisms, such as DNA repair, recombination and replication (3R) may have driven more recent <em>M. tuberculosis</em> evolution. <em>M. tuberculosis</em> may be regarded as a possible natural mutator, as there are no genes for components of the DNA mismatch repair system in its genome. In addition, within the W-Beijing family of strains, characteristic variations have already been found in DNA repair genes, null alleles of which have been shown to lead to an increase in spontaneous mutation frequency in <em>M. smegmatis</em> <a href="#pone.0001538-DosVultos1">[12]</a>, <a href="#pone.0001538-Rad1">[13]</a>. A previous analysis of the <em>M. tuberculosis</em> H37Rv genome identified homologs of genes involved in the reversal or repair of DNA damage in <em>E. coli</em> and related organisms <a href="#pone.0001538-Mizrahi1">[14]</a>. We analyzed most of these homologs, comprising a comprehensive set of 56 3R system components</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Polymorphisms in global MTC strains</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A comprehensive set of 56 genes encoding 3R system components was analyzed by sequencing (<a href="#pone-0001538-t001">Tables 1</a> and <a href="#pone-0001538-t002">2</a>) in a spoligotype-based set of 92 clinical strains; 45 of these strains are representative of global MTC diversity <a href="#pone.0001538-Brudey1">[3]</a> and were included to ascertain the global diversity of 3R genes in <em>M. tuberculosis</em>. The other 47 strains were chosen to allow evaluation of the resolution power of 3R-SNP-based variations in strains from very precise geographical locations. One group of strains was from Bangui, CAR, where the predominance of two major families of strains has been described: they were used to determine whether this approach could discriminate between these strains. The second group was from Madagascar, a country where both human and bacterial diversity is high. Analysis of 6.7 Mbp of MTC nucleotide sequence, corresponding to roughly 1.5 times the genome of <em>M. tuberculosis</em> H37Rv, showed an unexpectedly large set of highly polymorphic genes, implicating 3R systems in MTC evolution. We identified 259 polymorphisms, in 52 variable genes from 92 clinical isolates. These polymorphisms comprised 161 non synonymous (ns) SNPs, including three encoding stop codons, 91 synonymous (s) SNPs, and 7 deletions (<a href="#pone-0001538-t001">Tables 1</a>, <a href="#pone-0001538-t003">3</a>, <a href="#pone-0001538-t004">4</a>, <a href="#pone-0001538-t005">5</a> and <a href="#pone-0001538-t006">6</a>, see Supplementary Information <a href="#pone.0001538.s001">Text S1</a>, <a href="#pone.0001538.s002">Table S1</a>). As previously reported, nsSNPs were much more abundant than sSNPs (<a href="#pone-0001538-g001">Figure 1</a>, <a href="#pone-0001538-t001">Table 1</a>) <a href="#pone.0001538-Gutacker1">[7]</a>. SOS repair, Holliday junction-resolving genes and NER were the classes of genes that showed nucleotide diversity lower, although similar, than that of the housekeeping genes. This surely reflects the importance of these genes for mycobacteria. It seems logical that an obligate intracellular pathogen such as <em>M. tuberculosis</em> maintains a stable SOS repair machinery and consequently stable Holliday junction-resolving genes, which are induced as part of the SOS response. NER stabillity might indicate the importance of UV radiation resistance for <em>M.tuberculosis</em>. Nevertheless, these results reveal a wealth of polymorphisms, very different from the restricted allelic variation generally observed for <em>M. tuberculosis</em> housekeeping genes (<a href="#pone-0001538-t003">Tables 3</a>, <a href="#pone-0001538-t004">4</a> and <a href="#pone-0001538-t005">5</a>). We identified 74 haplotypes, with a nucleotide diversity per site of 0.00024, approximately twice that reported for the control group of housekeeping genes (Mann-Whitney p&lt; = 0.01082). Due to technical limitations, the analysis of housekeeping genes was restricted to a control group of strains whose genomic sequences were available online (see <a href="#s3">Materials and Methods</a>). In the control group, 3R and housekeeping genes showed a nucleotide diversity of 0,00042 and 0,00012, respectively, which represents a 3.5-fold difference. In total, 115 informative sites marking the evolutionary history of <em>M. tuberculosis</em> and 137 non-informative sites specific to single strains were identified. No recombination events were detected. <a href="#pone-0001538-g002">Figures 2</a> and <a href="#pone-0001538-g003">3</a> show the phylogenetic networks constructed using the data obtained <a href="#pone.0001538-Bandelt1">[15]</a>. Polymorphic site parsimony was perfectly correlated with spoligotype signature and could therefore be used to trace the evolutionary history of MTC (<a href="#pone-0001538-g004">Figure 4</a>). Principal genetic group 1 (PGG1) strains, such as the W-Beijing, CAS, EAI and <em>M. bovis</em> families of strains, appear to be only very distantly related to the strains of PGG2 and PGG3, providing a strong argument for the use of ecotype categorization for MTC members rather than the traditional subspecies classification. Our results suggest a high degree of functional redundancy among 3R genes. The occurrence of an ns variation in one gene in a particular 3R system is generally accompanied by neutral mutations or wild-type copies of other genes from the same system. This may reflect the existence of an equilibrium, demonstrating the importance of these 3R systems for the genomic integrity in mycobacteria and the significance that even individual nsSNPs may have. This is demonstrated by the analysis of average nucleotide diversity per classes of genes (<a href="#pone-0001538-g001">Figure 1</a>), where only the groups involved in direct repair and alkylation damage, ligases and AP endonucleases show a bigger than 2-fold average non-synonymous diversity in comparison with synonymous diversity.</p>
<div class="figure" id="pone-0001538-g001"><div class="img"><a name="pone-0001538-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001538.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001538.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Average nucleotide diversity by gene class.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>It was calculated based on the results for the clinical strains according to the class of 3R genes analyzed. Holliday Junction resolving genes, 4407 nucleotides-4 genes. SOS repair, 16893 nucleotides-10 genes. NER genes, 18108 nucleotides-5 genes. AP endonucleases, 1635 nucleotides-2 genes. GO repair, 5850 nucleotides-8 genes. Recombination involved genes, 30567 nucleotides-18 genes. Ligases, 6957 nucleotides-2 genes. BER genes, 8328 nucleotides-10 genes. Alkylation damage, 3216 nucleotides-4 genes. RecBCD, 8307 nucleotides-3 genes. RecFOR, 2568 nucleotides-3 genes. Polymerases, 7857 nucleotides-5 genes. Direct repair, 1989 nucleotides-2 genes. Uracil related repair, 1149 nucleotides-2.</p>
<span>doi:10.1371/journal.pone.0001538.g001</span></div><div class="figure" id="pone-0001538-g002"><div class="img"><a name="pone-0001538-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001538.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001538.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>(A) Phylogenetic network based on the total set of SNPs. This phylogenetic network was constructed using the median-joining algorithm with a final set of 252 SNPs characterized in 92 clinical strains of the <em>Mycobacterium tuberculosis</em> complex (MTC).</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>(B) Phylogenetic network based on the nsSNPs. This phylogenetic network was constructed using the median-joining algorithm with a final set of 163nsSNPs characterized in 92 clinical strains of the MTC. (C) Phylogenetic network based on the sSNPs. This phylogenetic network was constructed using the median-joining algorithm with a final set of 89 sSNPs characterized in 92 clinical strains of the MTC. Deletions were excluded from the analysis. Clinical isolates are classified with a color code, according to their spoligotype-based family. Node sizes indicate the number of strains belonging to the same haplotype.</p>
<span>doi:10.1371/journal.pone.0001538.g002</span></div><div class="figure" id="pone-0001538-g003"><div class="img"><a name="pone-0001538-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001538.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001538.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Geographic origin of the haplotypes identified.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>This phylogenetic network constructed using the median-joining algorithm with a final set of 252 SNPs characterized in 92 clinical strains of the <em>Mycobacterium tuberculosis</em> complex (MTC). Deletions were excluded from the analysis. Geographical origin is classified with a color code. Node sizes indicate the number of strains belonging to the same haplotype.</p>
<span>doi:10.1371/journal.pone.0001538.g003</span></div><div class="figure" id="pone-0001538-g004"><div class="img"><a name="pone-0001538-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001538.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001538.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Spoligotype based unrooted tree of the strains analyzed.</span></strong></p><a id="article1.body1.sec2.sec1.fig4.caption1.p1" name="article1.body1.sec2.sec1.fig4.caption1.p1"></a><p>This unrooted neighbor-joining tree was built with the Mega software on the same dataset as in <a href="#pone-0001538-g001">Figure 1</a>. The upper part of the tree describes Principal Genetic Group (PGG) 2 &amp; 3 strains and the lower part relates to PGG1. The spoligotypes are indicated next to the tree to show the excellent congruence. Clades are named according to SpolDB4 and to the recent SNP-cluster group(SCG) nomenclature.</p>
<span>doi:10.1371/journal.pone.0001538.g004</span></div><div class="figure" id="pone-0001538-t001"><div class="img"><a name="pone-0001538-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001538.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001538.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Putative gene function and distribution of synonymous and non-synonymous SNPS, deletions and stop codons found in this study.</span></strong></p><span>doi:10.1371/journal.pone.0001538.t001</span></div><div class="figure" id="pone-0001538-t002"><div class="img"><a name="pone-0001538-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001538.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001538.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>List of oligonucleotides (5′-3′) used in this study.</span></strong></p><span>doi:10.1371/journal.pone.0001538.t002</span></div><div class="figure" id="pone-0001538-t003"><div class="img"><a name="pone-0001538-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001538.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001538.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>DNA polymorphism and divergence data.</span></strong></p><span>doi:10.1371/journal.pone.0001538.t003</span></div><div class="figure" id="pone-0001538-t004"><div class="img"><a name="pone-0001538-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001538.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001538.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>DNA polymorphism data on the control group of strains.</span></strong></p><span>doi:10.1371/journal.pone.0001538.t004</span></div><div class="figure" id="pone-0001538-t005"><div class="img"><a name="pone-0001538-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001538.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001538.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>DNA polymorphism data on the control group of strains.</span></strong></p><span>doi:10.1371/journal.pone.0001538.t005</span></div><div class="figure" id="pone-0001538-t006"><div class="img"><a name="pone-0001538-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001538.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001538.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Outcome of correlating the location of non-synonymous single nucleotide polymorphisms (ns SNPs) inside genes, the amino acids they are predicted to encode and predicted enzymatic signature motifs and active sites.</span></strong></p><span>doi:10.1371/journal.pone.0001538.t006</span></div>

<h4>Major 3R findings</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We investigated the location of ns SNPs and compared the amino acids they encoded with predicted enzymatic signature motifs and active sites. Significant polymorphisms in 3R genes were observed for particular MTC families, that may be progenitors for altered mutator phenotypes (see SI <a href="#pone.0001538.s001">Text S1</a> for further information about the genes studied, the SNPs found and inferences about their significance). For example, one W-Beijing strain shows an accumulation of ns variations in the <em>tagA</em> and <em>alkA</em> genes. The <em>tagA</em> gene encodes a 3-methyladenine DNA glycosylase I, is constitutively expressed and highly specific, whereas <em>alkA</em> encodes a 3-methyladenine DNA glycosylase II—an alkylation damage-inducible protein capable of catalyzing the excision of a wide variety of alkylated bases. The <em>tagA</em> gene was one of the most conserved genes in our panel, with ns variants found in only two strains. The observation of such variants in only one of the W-Beijing strains is most interesting, and consistent with recent observations that the pathogenic characteristics of W-Beijing strains are not conserved, with strains within individual W-Beijing lineages having evolved unique pathogenic characteristics <a href="#pone.0001538-Hanekom1">[16]</a>. Another case concerns <em>M. bovis</em> strains, which displayed the greatest accumulation of mutations in <em>recBCD</em> genes. RecBCD processes DNA ends resulting from double strand breaks, acting as a bipolar helicase that splits the duplex into its component strands and digests them until a recombinational hot-spot (chi site) is encountered. This association is of interest because the formation of deletions has been identified as a common feature for RecB mutants, including in <em>M. bovis</em> strains <a href="#pone.0001538-Brosch1">[2]</a>. In addition, the gene encoding the recombination factor RecO had ns SNPs predicted to cause amino acid substitutions affecting component locations critical to enzymatic function. Furthermore, two SNPs were found in the gene encoding the DNA glycosylase End (codon-167 coupled with a codon-170 Gly-Ser variation) and a combination of two ns variations was found in the gene encoding the DNA polymerase PolA (codon-186 and codon-188) . However, only the ns SNPs in the <em>polA</em> gene were at locations that could affect active sites in the expression product. MTC strains are highly clonal, so the occurrence of two variations in the same codon coupled with another variation only two codons away seems unlikely to be either random or entirely fortuitous; rather is indicative of strong natural selection either seperately or due to epistasis. However, this does not exclude a possible recombination or horizontal transfer event. Some of the most polymorphic genes, such as those encoding components of the RecBCD pathway, made it possible to distinguish 24 haplotypes among the strains analyzed. RecB, RecC and RecD orthologs have a limited species distribution, being found in only a few enterobacteria in addition to <em>M. tuberculosis</em> and <em>B. burgdorferi</em>. This has led to the suggestion that <em>M. tuberculosis</em> acquired these genes through a lateral transfer event <a href="#pone.0001538-Eisen1">[17]</a>. The highly polymorphic genes also included <em>polA</em>, dinP, dinX and <em>dnaQ</em>, which displayed a remarkable accumulation of ns variations, suggestive of changes in PolIII proofreading in the strains possessing them. Furthermore, significant ns SNPs were detected in the genes encoding LigC and MutT2, and MutT2 has already been suggested to be involved as a source of variation in w-Beijing strains (13). Other molecules potentially affected by SNP-encoded amino-acid substitutions, albeit to a lesser extent, included DNA polymerases DinP and DinX, the recombination factors RecB, RecC, RecD and RecN, the ligases LigA, LigB, LigC and LigD, the nucleotide excision repair <a href="#pone.0001538-Tye1">[18]</a> components UvrA, UvrB, and UvrC. The nsSNPs in the genes encoding the BER DNA glycosylases MutY, Nth, Nei and Fpg, recombination proteins RecA, RecC, RecG, RecN and RecR, LexA, the NER components Uvr, UvrC and the helicase UvrD, and the double-stranded DNA translocase and ATPase RuvB led to predicted amino-acid changes at component sites which could potentially induce steric changes only indirectly (<a href="#pone-0001538-t006">Table 6</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Although the 3R genes were unexpectedly polymorphic, these results are fully consistent with the idea that <em>M. tuberculosis</em> is a strictly clonal organism, and provide no evidence of recent lateral gene transfer <a href="#pone.0001538-Baker1">[19]</a>. It has previously been suggested that the occurrence of amino-acid substitutions in <em>M. tuberculosis</em> strains is strongly indicative of possible functional consequences of these substitutions <a href="#pone.0001538-Sreevatsan1">[8]</a>. This might induce slack in the fidelity of genome maintenance and could be regarded as compensation for the genetic isolation of MTC strains, devoid as they are of horizontal gene transfer. In addition to the lack of a recognizable mismatch repair system, the predicted reduced stringency/precision in DNA repair resulting from the polymorphisms detected, might facilitate or even allow adaptation. However, this does not necessarily mean that the selective consequences of non synonymous changes are immediately effective <a href="#pone.0001538-Rocha1">[20]</a>.</p>


<h4>Effect on evolution</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We investigated whether some form of natural selection could account for the patterns of diversity observed, by comparing the site frequency spectrum (SFS) of synonymous and non synonymous variants (<a href="#pone-0001538-g005">Figure 5</a>). The frequency spectrum is a count of the number of mutations that exist at a frequency of xi = i/n for i = 1, 2,…, n−1, in a sample of size n. In other words, it represents a summary of the allele frequencies of the various mutations in the sample. In a standard neutral model (i.e., a model with random mating, constant population size, no population subdivision, etc.), the expected value of xi is proportional to 1/i. Selection against deleterious mutations will increase the fraction of mutations segregating at low frequencies in the sample. A selective sweep has roughly the same effect on the frequency spectrum. Conversely, positive selection will tend to increase the frequency in a sample of mutations segregating at high frequencies. Under a strictly neutral model, these two classes of genetic variants should present a similar SFS <a href="#pone.0001538-Nielsen1">[21]</a>. The higher values observed for the singleton ns SNPs than for s SNPs are suggestive of negative/purifying selection. However, caution is required in interpretation, because different selective/demographic scenarios may mimic similar patterns of diversity. Negative selection alone and/or population growth might be equally likely to account for the patterns observed <a href="#pone.0001538-Nielsen1">[21]</a>. We compared the non synonymous and synonymous substitution rates for each gene, by calculating the ratio of non synonymous mutations per non synonymous site (<em>Ka</em>) to synonymous mutations per synonymous site (<em>Ks</em>) (<a href="#pone-0001538-t003">Tables 3</a>, <a href="#pone-0001538-t004">4</a> and <a href="#pone-0001538-t005">5</a>). Under a strictly neutral model of evolution, this ratio should be equal to one. For this particular analysis, we used the oldest strain from the panel analyzed (as determined by the number of spoligotype spacers) as the outgroup. Nine of the 52 genes presented K<sub>A</sub>/K<sub>S</sub> values higher than 1. Six of these nine genes had considerably higher K<sub>A</sub>/K<sub>S </sub>ratios, suggesting that the evolution of these genes might have been driven by positive natural selection. The remaining genes had K<sub>A</sub>/K<sub>S</sub> ratios below one, consistent with negative/purifying selection, as suggested by the SFS spectrum. Further detailed evolutionary studies will be required to elucidate the evolutionary forces that may account for the patterns observed, and to determine which of these genes have contributed significantly to the evolution of <em>M. tuberculosis</em>.</p>
<div class="figure" id="pone-0001538-g005"><div class="img"><a name="pone-0001538-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001538" data-uri="info:doi/10.1371/journal.pone.0001538.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001538.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001538.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001538.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001538.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001538.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Site frequency spectrum of sSNPs and nsSNPs.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>This spectrum summarizes the allele frequencies of the various mutations in the sample.</p>
<span>doi:10.1371/journal.pone.0001538.g005</span></div>

<h4>New phylogenetic tool</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p><em>M. tuberculosis</em> strains are highly clonal. However, SNP analysis of 3R genes seems to be a robust phylogenetic method with very high resolution, even for a generally monomorphic, recent pathogen, such as <em>M. tuberculosis</em>. Genome stability is a key factor in maintenance of the integrity of an organism. Nevertheless, genome variability may sometimes be a selective advantage. Pathogenic bacteria are constantly exposed to hostile conditions, in which factors such as host defenses and antibiotic treatments are continuously changing their environments. Provided that it is in balance with bacterial fitness, a mutator phenotype may act as a driving force facilitating strain evolution, through, for example, the acquisition of antibiotic resistance, virulence factor variation and adaptation to the genetic stress conditions exerted by the environment (e.g. host defense mechanisms). Changes in mutation rates generally result from allelic variation in the genes controlling 3R fidelity <a href="#pone.0001538-Denamur1">[22]</a>, <a href="#pone.0001538-Tonjum1">[23]</a>. The 3R polymorphisms observed in this study suggest that these genes in general may be subjected to negative/purifying selection pressure. In this model, a large number of the variations observed would be expected to be deleterious, at least to some extent. We consistently found 3R polymorphisms to be frequent in a global panel of <em>M. tuberculosis</em> families, indicating that most of these mutations can be only slightly deleterious, as fitness costs would otherwise be too high for these MTC strains to sustain with such a wide range of human hosts; highly deleterious mutations would be expected to give rise to non-viable cells and would therefore be selected against. These classes of “slightly deleterious” mutations may also result in suboptimal 3R system activity. Deficiencies in polymerase proofreading activity, for example, might cause an increase in mutation rates, whereas incorrect non-homologous end joining might result in deletions or other polymorphisms. These events could potentially increase genomic variability and might therefore be a selective advantage to the strains possessing them under certain stressful conditions, whereas selection against them would be expected in changing environments <a href="#pone.0001538-Denamur1">[22]</a>. Overall, this study shows that 3R gene family polymorphisms can be used to study the evolution of highly clonal bacteria, and in particular MTC strains. It also provides a powerful new high-resolution tool for strain discrimination for clinicians. The high-resolution surveillance of haplotypes with particular characteristics could be used to provide early warning of the spread of localized epidemics, making it easier to deal with outbreaks caused by MDR and XDR MTC strains, for example, and facilitating their dissemination.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>DNA was sequenced directly, with fragments amplified by the dideoxy chain-termination method from the strains described above. In the comparison of the nucleotide diversity of 3R and housekeeping genes in the control group of strains, the analysis of housekeeping genes was restricted to a control group of strains whose genomic sequences were available online: <em>M. bovis subsp. bovis</em> AF2122/97 and <em>M. tuberculosis</em> CDC1551 strains from the TIGR website at <a href="http://cmr.tigr.org">http://cmr.tigr.org</a>, <em>M. microti</em> and <em>M. africanum</em> strains from the Sanger Institute at <a href="http://www.sanger.ac.uk">http://www.sanger.ac.uk</a> and strains F11, C and <em>Haarlem</em> from Broad Institute available at <a href="http://www.broad.mit.edu">http://www.broad.mit.edu</a>. The sSNPs and nsSNPs were concatenated, resulting in a single character string (nucleotide sequence) for each clinical isolate analyzed. Network software <a href="#pone.0001538-Bandelt1">[15]</a> was initially used for phylogenetic and molecular evolution analysis. This software assumes that there is no recombination between genomes. Phylogenetic trees were built with the neighbor-joining method and MEGA software <a href="#pone.0001538-Kumar1">[24]</a>. The DNAsp package <a href="#pone.0001538-Rozas1">[25]</a> was used to analyze the average nucleotide diversity of the MTC and interspecies Ka/Ks tests. Prediction of the 3R protein secondary structure was performed based on sequence alignments with various 3R homologs by using the JPred program. A search for functional domains or signatures in the 3R gene deduced amino acid sequences was carried out using the DOLOP, the PROSITE and the Pfam databases. The presence of recognized DNA binding motifs and active sites was assessed by using the Expasy site and PFAM bioinformatics algorithms available at <a href="http://us.expasy.org/cgi-bin/protscale.pl">http://us.expasy.org/cgi-bin/protscale.p​l</a>, and the electrostatic charge was calculated by using the EMBOSS package (<a href="http://proteas.uio.no/EMBOSS">http://proteas.uio.no/EMBOSS</a>). Thereby, the significance of nsSNPs in relation to predicted DNA binding and enzymatic signature motifs and active sites was predicted.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001538"></div><a name="pone.0001538.s001" id="pone.0001538.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001538.s001">Text S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Supporting information about the genes studied, the SNPs found and inferences about their significance.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001538.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.24 MB DOC)</p>
<a name="pone.0001538.s002" id="pone.0001538.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001538.s002">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Full results from this study. The First line, in red, represents the strains to which the results refer. A denomination starting with (B) means that the strains belong to the Bangui CAR group, conversingly an (M) and a (C) indicates that the strains belong to the Madagascar or Global groups, respectively. The first column indicates the gene where the mutation is present. The second column indicates the genomic position where the polymorphism was found. Polymorphism are marked in red. Non-synonymous polymorphism are indicated by a red genomic position.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001538.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.48 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Luis Barreiro for assistance in the writing of the manuscript and critical discussions. We thank Thierry Zozio (Institut Pasteur de Guadeloupe) for rechecking some of the spoligotyping results and Stephan A. Frye for bioinformatics support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: IM BG TT TD. Performed the experiments: JR TT TD OM MG. Analyzed the data: JR CS TT TD OM. Contributed reagents/materials/analysis tools: NR CS VR. Wrote the paper: IM BG NR CS TT TD VR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001538-Roumagnac1" id="pone.0001538-Roumagnac1"></a>Roumagnac P, Weill FX, Dolecek C, Baker S, Brisse S, et al.  (2006) Evolutionary history of Salmonella typhi. Science  314: 1301–1304.  <ul class="find" data-citedArticleID="1050280" data-doi="10.1126/science.1134933"><li><a href="http://dx.doi.org/10.1126/science.1134933" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+history+of+Salmonella+typhi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+history+of+Salmonella+typhi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001538-Brosch1" id="pone.0001538-Brosch1"></a>Brosch R, Gordon SV, Marmiesse M, Brodin P, Buchrieser C, et al.  (2002) A new evolutionary scenario for the Mycobacterium tuberculosis complex. Proc Natl Acad Sci U S A  99: 3684–3689.  <ul class="find" data-citedArticleID="1050246" data-doi="10.1073/pnas.052548299"><li><a href="http://dx.doi.org/10.1073/pnas.052548299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+evolutionary+scenario+for+the+Mycobacterium+tuberculosis+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+evolutionary+scenario+for+the+Mycobacterium+tuberculosis+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001538-Brudey1" id="pone.0001538-Brudey1"></a>Brudey K, Driscoll JR, Rigouts L, Prodinger WM, Gori A, et al.  (2006) Mycobacterium tuberculosis complex genetic diversity: mining the fourth international spoligotyping database (SpolDB4) for classification, population genetics and epidemiology. BMC Microbiol  6: 23.  <ul class="find" data-citedArticleID="1050248"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mycobacterium+tuberculosis+complex+genetic+diversity%3A+mining+the+fourth+international+spoligotyping+database+%28SpolDB4%29+for+classification%2C+population+genetics+and+epidemiology.&amp;auth=&amp;atitle=Mycobacterium+tuberculosis+complex+genetic+diversity%3A+mining+the+fourth+international+spoligotyping+database+%28SpolDB4%29+for+classification%2C+population+genetics+and+epidemiology." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mycobacterium+tuberculosis+complex+genetic+diversity%3A+mining+the+fourth+international+spoligotyping+database+%28SpolDB4%29+for+classification%2C+population+genetics+and+epidemiology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mycobacterium+tuberculosis+complex+genetic+diversity%3A+mining+the+fourth+international+spoligotyping+database+%28SpolDB4%29+for+classification%2C+population+genetics+and+epidemiology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001538-Filliol1" id="pone.0001538-Filliol1"></a>Filliol I, Motiwala AS, Cavatore M, Qi W, Hazbon MH, et al.  (2006) Global phylogeny of Mycobacterium tuberculosis based on single nucleotide polymorphism (SNP) analysis: insights into tuberculosis evolution, phylogenetic accuracy of other DNA fingerprinting systems, and recommendations for a minimal standard SNP set. J Bacteriol  188: 759–772.  <ul class="find" data-citedArticleID="1050256" data-doi="10.1128/jb.188.2.759-772.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.2.759-772.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+phylogeny+of+Mycobacterium+tuberculosis+based+on+single+nucleotide+polymorphism+%28SNP%29+analysis%3A+insights+into+tuberculosis+evolution%2C+phylogenetic+accuracy+of+other+DNA+fingerprinting+systems%2C+and+recommendations+for+a+minimal+standard+SNP+set." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+phylogeny+of+Mycobacterium+tuberculosis+based+on+single+nucleotide+polymorphism+%28SNP%29+analysis%3A+insights+into+tuberculosis+evolution%2C+phylogenetic+accuracy+of+other+DNA+fingerprinting+systems%2C+and+recommendations+for+a+minimal+standard+SNP+set.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001538-Frothingham1" id="pone.0001538-Frothingham1"></a>Frothingham R, Meeker-O'Connell WA (1998) Genetic diversity in the Mycobacterium tuberculosis complex based on variable numbers of tandem DNA repeats. Microbiology  144 ( Pt 5): 1189–1196.  <ul class="find" data-citedArticleID="1050258" data-doi="10.1099/00221287-144-5-1189"><li><a href="http://dx.doi.org/10.1099/00221287-144-5-1189" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+diversity+in+the+Mycobacterium+tuberculosis+complex+based+on+variable+numbers+of+tandem+DNA+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+diversity+in+the+Mycobacterium+tuberculosis+complex+based+on+variable+numbers+of+tandem+DNA+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001538-Groenen1" id="pone.0001538-Groenen1"></a>Groenen PM, Bunschoten AE, van Soolingen D, van Embden JD (1993) Nature of DNA polymorphism in the direct repeat cluster of Mycobacterium tuberculosis; application for strain differentiation by a novel typing method. Mol Microbiol  10: 1057–1065.  <ul class="find" data-citedArticleID="1050260" data-doi="10.1111/j.1365-2958.1993.tb00976.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1993.tb00976.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nature+of+DNA+polymorphism+in+the+direct+repeat+cluster+of+Mycobacterium+tuberculosis%3B+application+for+strain+differentiation+by+a+novel+typing+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nature+of+DNA+polymorphism+in+the+direct+repeat+cluster+of+Mycobacterium+tuberculosis%3B+application+for+strain+differentiation+by+a+novel+typing+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001538-Gutacker1" id="pone.0001538-Gutacker1"></a>Gutacker MM, Smoot JC, Migliaccio CA, Ricklefs SM, Hua S, et al.  (2002) Genome-wide analysis of synonymous single nucleotide polymorphisms in Mycobacterium tuberculosis complex organisms: resolution of genetic relationships among closely related microbial strains. Genetics  162: 1533–1543.  <ul class="find" data-citedArticleID="1050262"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome-wide+analysis+of+synonymous+single+nucleotide+polymorphisms+in+Mycobacterium+tuberculosis+complex+organisms%3A+resolution+of+genetic+relationships+among+closely+related+microbial+strains.&amp;auth=&amp;atitle=Genome-wide+analysis+of+synonymous+single+nucleotide+polymorphisms+in+Mycobacterium+tuberculosis+complex+organisms%3A+resolution+of+genetic+relationships+among+closely+related+microbial+strains." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+synonymous+single+nucleotide+polymorphisms+in+Mycobacterium+tuberculosis+complex+organisms%3A+resolution+of+genetic+relationships+among+closely+related+microbial+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+synonymous+single+nucleotide+polymorphisms+in+Mycobacterium+tuberculosis+complex+organisms%3A+resolution+of+genetic+relationships+among+closely+related+microbial+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001538-Sreevatsan1" id="pone.0001538-Sreevatsan1"></a>Sreevatsan S, Pan X, Stockbauer KE, Connell ND, Kreiswirth BN, et al.  (1997) Restricted structural gene polymorphism in the Mycobacterium tuberculosis complex indicates evolutionarily recent global dissemination. Proc Natl Acad Sci U S A  94: 9869–9874.  <ul class="find" data-citedArticleID="1050284" data-doi="10.1073/pnas.94.18.9869"><li><a href="http://dx.doi.org/10.1073/pnas.94.18.9869" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Restricted+structural+gene+polymorphism+in+the+Mycobacterium+tuberculosis+complex+indicates+evolutionarily+recent+global+dissemination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Restricted+structural+gene+polymorphism+in+the+Mycobacterium+tuberculosis+complex+indicates+evolutionarily+recent+global+dissemination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001538-Supply1" id="pone.0001538-Supply1"></a>Supply P, Mazars E, Lesjean S, Vincent V, Gicquel B, et al.  (2000) Variable human minisatellite-like regions in the Mycobacterium tuberculosis genome. Mol Microbiol  36: 762–771.  <ul class="find" data-citedArticleID="1050286" data-doi="10.1046/j.1365-2958.2000.01905.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01905.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variable+human+minisatellite-like+regions+in+the+Mycobacterium+tuberculosis+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variable+human+minisatellite-like+regions+in+the+Mycobacterium+tuberculosis+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001538-Liu1" id="pone.0001538-Liu1"></a>Liu X, Gutacker MM, Musser JM, Fu YX (2006) Evidence for recombination in Mycobacterium tuberculosis. J Bacteriol  188: 8169–8177.  <ul class="find" data-citedArticleID="1050268" data-doi="10.1128/jb.01062-06"><li><a href="http://dx.doi.org/10.1128/jb.01062-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+recombination+in+Mycobacterium+tuberculosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+recombination+in+Mycobacterium+tuberculosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001538-RosasMagallanes1" id="pone.0001538-RosasMagallanes1"></a>Rosas-Magallanes V, Deschavanne P, Quintana-Murci L, Brosch R, Gicquel B, et al.  (2006) Horizontal transfer of a virulence operon to the ancestor of Mycobacterium tuberculosis. Mol Biol Evol  23: 1129–1135.  <ul class="find" data-citedArticleID="1050278" data-doi="10.1093/molbev/msj120"><li><a href="http://dx.doi.org/10.1093/molbev/msj120" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Horizontal+transfer+of+a+virulence+operon+to+the+ancestor+of+Mycobacterium+tuberculosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Horizontal+transfer+of+a+virulence+operon+to+the+ancestor+of+Mycobacterium+tuberculosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001538-DosVultos1" id="pone.0001538-DosVultos1"></a>Dos Vultos T, Blazquez J, Rauzier J, Matic I, Gicquel B (2006) Identification of Nudix hydrolase family members with an antimutator role in Mycobacterium tuberculosis and Mycobacterium smegmatis. J Bacteriol  188: 3159–3161.  <ul class="find" data-citedArticleID="1050252" data-doi="10.1128/jb.188.8.3159-3161.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.8.3159-3161.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+Nudix+hydrolase+family+members+with+an+antimutator+role+in+Mycobacterium+tuberculosis+and+Mycobacterium+smegmatis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+Nudix+hydrolase+family+members+with+an+antimutator+role+in+Mycobacterium+tuberculosis+and+Mycobacterium+smegmatis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001538-Rad1" id="pone.0001538-Rad1"></a>Rad ME, Bifani P, Martin C, Kremer K, Samper S, et al.  (2003) Mutations in putative mutator genes of Mycobacterium tuberculosis strains of the W-Beijing family. Emerg Infect Dis  9: 838–845.  <ul class="find" data-citedArticleID="1050274" data-doi="10.3201/eid0907.020803"><li><a href="http://dx.doi.org/10.3201/eid0907.020803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+putative+mutator+genes+of+Mycobacterium+tuberculosis+strains+of+the+W-Beijing+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+putative+mutator+genes+of+Mycobacterium+tuberculosis+strains+of+the+W-Beijing+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001538-Mizrahi1" id="pone.0001538-Mizrahi1"></a>Mizrahi V, Andersen SJ (1998) DNA repair in Mycobacterium tuberculosis. What have we learnt from the genome sequence? Mol Microbiol  29: 1331–1339.  <ul class="find" data-citedArticleID="1050270" data-doi="10.1046/j.1365-2958.1998.01038.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1998.01038.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+repair+in+Mycobacterium+tuberculosis.+What+have+we+learnt+from+the+genome+sequence%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+repair+in+Mycobacterium+tuberculosis.+What+have+we+learnt+from+the+genome+sequence%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001538-Bandelt1" id="pone.0001538-Bandelt1"></a>Bandelt HJ, Forster P, Rohl A (1999) Median-joining networks for inferring intraspecific phylogenies. Mol Biol Evol  16: 37–48.  <ul class="find" data-citedArticleID="1050244" data-doi="10.1093/oxfordjournals.molbev.a026036"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Median-joining+networks+for+inferring+intraspecific+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Median-joining+networks+for+inferring+intraspecific+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001538-Hanekom1" id="pone.0001538-Hanekom1"></a>Hanekom M, van der Spuy GD, Streicher E, Ndabambi SL, McEvoy CR, et al.  (2007) A recently evolved sublineage of the Mycobacterium tuberculosis Beijing strain family is associated with an increased ability to spread and cause disease. J Clin Microbiol  45: 1483–1490.  <ul class="find" data-citedArticleID="1050264" data-doi="10.1128/jcm.02191-06"><li><a href="http://dx.doi.org/10.1128/jcm.02191-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+recently+evolved+sublineage+of+the+Mycobacterium+tuberculosis+Beijing+strain+family+is+associated+with+an+increased+ability+to+spread+and+cause+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+recently+evolved+sublineage+of+the+Mycobacterium+tuberculosis+Beijing+strain+family+is+associated+with+an+increased+ability+to+spread+and+cause+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001538-Eisen1" id="pone.0001538-Eisen1"></a>Eisen JA, Hanawalt PC (1999) A phylogenomic study of DNA repair genes, proteins, and processes. Mutat Res  435: 171–213.  <ul class="find" data-citedArticleID="1050254" data-doi="10.1016/s0921-8777(99)00050-6"><li><a href="http://dx.doi.org/10.1016/s0921-8777(99)00050-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+phylogenomic+study+of+DNA+repair+genes%2C+proteins%2C+and+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+phylogenomic+study+of+DNA+repair+genes%2C+proteins%2C+and+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001538-Tye1" id="pone.0001538-Tye1"></a>Tye BK, Chien J, Lehman IR, Duncan BK, Warner HR (1978) Uracil incorporation: a source of pulse-labeled DNA fragments in the replication of the Escherichia coli chromosome. Proc Natl Acad Sci U S A  75: 233–237.  <ul class="find" data-citedArticleID="1050290" data-doi="10.1073/pnas.75.1.233"><li><a href="http://dx.doi.org/10.1073/pnas.75.1.233" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uracil+incorporation%3A+a+source+of+pulse-labeled+DNA+fragments+in+the+replication+of+the+Escherichia+coli+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Uracil+incorporation%3A+a+source+of+pulse-labeled+DNA+fragments+in+the+replication+of+the+Escherichia+coli+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001538-Baker1" id="pone.0001538-Baker1"></a>Baker L, Brown T, Maiden MC, Drobniewski F (2004) Silent nucleotide polymorphisms and a phylogeny for Mycobacterium tuberculosis. Emerg Infect Dis  10: 1568–1577.  <ul class="find" data-citedArticleID="1050242" data-doi="10.3201/eid1009.040046"><li><a href="http://dx.doi.org/10.3201/eid1009.040046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Silent+nucleotide+polymorphisms+and+a+phylogeny+for+Mycobacterium+tuberculosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Silent+nucleotide+polymorphisms+and+a+phylogeny+for+Mycobacterium+tuberculosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001538-Rocha1" id="pone.0001538-Rocha1"></a>Rocha EP, Smith JM, Hurst LD, Holden MT, Cooper JE, et al.  (2006) Comparisons of dN/dS are time dependent for closely related bacterial genomes. J Theor Biol  239: 226–235.  <ul class="find" data-citedArticleID="1050276" data-doi="10.1016/j.jtbi.2005.08.037"><li><a href="http://dx.doi.org/10.1016/j.jtbi.2005.08.037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparisons+of+dN%2FdS+are+time+dependent+for+closely+related+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparisons+of+dN%2FdS+are+time+dependent+for+closely+related+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001538-Nielsen1" id="pone.0001538-Nielsen1"></a>Nielsen R (2005) Molecular signatures of natural selection. Annu Rev Genet  39: 197–218.  <ul class="find" data-citedArticleID="1050272" data-doi="10.1146/annurev.genet.39.073003.112420"><li><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.112420" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+signatures+of+natural+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+signatures+of+natural+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001538-Denamur1" id="pone.0001538-Denamur1"></a>Denamur E, Matic I (2006) Evolution of mutation rates in bacteria. Mol Microbiol  60: 820–827.  <ul class="find" data-citedArticleID="1050250" data-doi="10.1111/j.1365-2958.2006.05150.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2006.05150.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+mutation+rates+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+mutation+rates+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001538-Tonjum1" id="pone.0001538-Tonjum1"></a>Tonjum T, Seeberg E (2001) Microbial fitness and genome dynamics. Trends Microbiol  9: 356–358.  <ul class="find" data-citedArticleID="1050288" data-doi="10.1016/s0966-842x(01)02099-6"><li><a href="http://dx.doi.org/10.1016/s0966-842x(01)02099-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+fitness+and+genome+dynamics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+fitness+and+genome+dynamics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001538-Kumar1" id="pone.0001538-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment. Brief Bioinform  5: 150–163.  <ul class="find" data-citedArticleID="1050266" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001538-Rozas1" id="pone.0001538-Rozas1"></a>Rozas J, Sanchez-DelBarrio JC, Messeguer X, Rozas R (2003) DnaSP, DNA polymorphism analyses by the coalescent and other methods. Bioinformatics  19: 2496–2497.  <ul class="find" data-citedArticleID="1050282" data-doi="10.1093/bioinformatics/btg359"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DnaSP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DnaSP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.XML" value="151822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.PDF" value="474613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g001.PNG_L" value="463085"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g001.PNG_M" value="87686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g001.PNG_S" value="12439"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g001.TIF" value="660804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g001.PNG_I" value="29600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g002.PNG_L" value="1163943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g002.PNG_M" value="115907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g002.PNG_S" value="9737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g002.TIF" value="1529542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g002.PNG_I" value="39845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g003.PNG_L" value="782381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g003.PNG_M" value="138117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g003.PNG_S" value="11922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g003.TIF" value="1095946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g003.PNG_I" value="47197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g004.PNG_L" value="2408804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g004.PNG_M" value="103219"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g004.PNG_S" value="9881"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g004.TIF" value="4074938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g004.PNG_I" value="58877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t001.PNG_L" value="402221"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t001.PNG_M" value="83055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t001.PNG_S" value="30400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t001.TIF" value="2058942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t001.PNG_I" value="247629"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t002.PNG_L" value="586596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t002.PNG_M" value="285757"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t002.PNG_S" value="20395"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t002.TIF" value="2831228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t002.PNG_I" value="147602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t003.PNG_L" value="267671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t003.PNG_M" value="94643"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t003.PNG_S" value="24627"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t003.TIF" value="1391380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t003.PNG_I" value="175713"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t004.PNG_L" value="226857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t004.PNG_M" value="73600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t004.PNG_S" value="18466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t004.TIF" value="1283626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t004.PNG_I" value="131939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t005.PNG_L" value="107738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t005.PNG_M" value="141941"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t005.PNG_S" value="17239"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t005.TIF" value="491442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t005.PNG_I" value="49880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t006.PNG_L" value="110116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t006.PNG_M" value="157194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t006.PNG_S" value="17347"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t006.TIF" value="416418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.t006.PNG_I" value="52953"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g005.PNG_L" value="347836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g005.PNG_M" value="64119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g005.PNG_S" value="7497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g005.TIF" value="616488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.g005.PNG_I" value="23132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.s001.DOC" value="236032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001538.s002.XLS" value="478720"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001538&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001538" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001538&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001538&volume=&issue=&title=Evolution and Diversity of Clonal Bacteria: The Paradigm of Mycobacterium tuberculosis&author_name=Tiago%20Dos%20Vultos%2C%20Olga%20Mestre%2C%20Jean%20Rauzier%2C%20Marcin%20Golec%2C%20Nalin%20Rastogi%2C%20Voahangy%20Rasolofo%2C%20Tone%20Tonjum%2C%20Christophe%20Sola%2C%20Ivan%20Matic%2C%20Brigitte%20Gicquel&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001538" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001538" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001538" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001538&amp;t=Evolution%20and%20Diversity%20of%20Clonal%20Bacteria%3A%20The%20Paradigm%20of%20Mycobacterium%20tuberculosis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001538&title=Evolution%20and%20Diversity%20of%20Clonal%20Bacteria%3A%20The%20Paradigm%20of%20Mycobacterium%20tuberculosis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001538&amp;title=Evolution%20and%20Diversity%20of%20Clonal%20Bacteria%3A%20The%20Paradigm%20of%20Mycobacterium%20tuberculosis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001538" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001538&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Evolution and Diversity of Clonal Bacteria: The Paradigm of Mycobacterium tuberculosis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001538';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Evolution%20and%20Diversity%20of%20Clonal%20Bacteria%3A%20The%20Paradigm%20of%20Mycobacterium%20tuberculosis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001538" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001538" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="25902"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA%22" title="Search for articles in the subject area:'DNA'"><div class="flagText">DNA</div></a>
              <div data-categoryid="32449" data-articleid="25902"
                   data-categoryname="DNA"
                   class="flagImage" title="Flag 'DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+repair%22" title="Search for articles in the subject area:'DNA repair'"><div class="flagText">DNA repair</div></a>
              <div data-categoryid="31937" data-articleid="25902"
                   data-categoryname="DNA repair"
                   class="flagImage" title="Flag 'DNA repair' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="25902"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="25902"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mycobacterium+tuberculosis%22" title="Search for articles in the subject area:'Mycobacterium tuberculosis'"><div class="flagText">Mycobacterium tuberculosis</div></a>
              <div data-categoryid="42725" data-articleid="25902"
                   data-categoryname="Mycobacterium tuberculosis"
                   class="flagImage" title="Flag 'Mycobacterium tuberculosis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotides%22" title="Search for articles in the subject area:'Nucleotides'"><div class="flagText">Nucleotides</div></a>
              <div data-categoryid="17443" data-articleid="25902"
                   data-categoryname="Nucleotides"
                   class="flagImage" title="Flag 'Nucleotides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="25902"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9852'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9479'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7851&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=8945">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
