

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Optimal Foraging Predicts the Ecology but Not the Evolution of Host Specialization in Bacteriophages</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001946"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001946" />

    <meta name="citation_title" content="Optimal Foraging Predicts the Ecology but Not the Evolution of Host Specialization in Bacteriophages"/>
    <meta itemprop="name" content="Optimal Foraging Predicts the Ecology but Not the Evolution of Host Specialization in Bacteriophages"/>

      <meta name="citation_author" content="Sébastien Guyader"/>
            <meta name="citation_author_institution" content="Department of Biology, University of North Carolina, Chapel Hill, North Carolina, United States of America"/>
      <meta name="citation_author" content="Christina L. Burch"/>
            <meta name="citation_author_institution" content="Department of Biology, University of North Carolina, Chapel Hill, North Carolina, United States of America"/>

    <meta name="citation_date" content="2008/4/16"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001946.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1946"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The evolution of ecological specialization.; citation_author=DJ Futuyma; citation_author=G Moreno; citation_journal_title=Annual Review of Ecology and Systematics; citation_volume=19; citation_number=1; citation_pages=207-233; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Trade-offs, food web structure, and the coexistence of habitat specialists and generalists.; citation_author=MA McPeek; citation_journal_title=American Naturalist; citation_volume=148; citation_number=2; citation_pages=S124-S138; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Cost of host radiation in an RNA virus.; citation_author=PE Turner; citation_author=SF Elena; citation_journal_title=Genetics; citation_volume=156; citation_number=3; citation_pages=1465-1470; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Ecological genetics and host adaptation in herbivorous insects - the experimental study of evolution in natural and agricultural systems.; citation_author=S Via; citation_journal_title=Annual review of entomology; citation_volume=35; citation_number=4; citation_pages=421-446; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Optimal patch residence time in egg parasitoids: innate versus learned estimate of patch quality.; citation_author=G Boivin; citation_author=X Fauvergue; citation_author=E Wajnberg; citation_journal_title=Oecologia; citation_volume=138; citation_number=5; citation_pages=640-647; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Niche breadth in parasites: an evolutionarily stable strategy model, with special reference to the protozoan parasite Leishmania.; citation_author=E Garnick; citation_journal_title=Theoretical Population Biology; citation_volume=42; citation_number=6; citation_pages=62-103; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Optimal foraging and size selection of prey by bluegill sunfish (Lepomis macrochirus).; citation_author=EE Werner; citation_author=DJ Hall; citation_journal_title=Ecology; citation_volume=55; citation_number=7; citation_pages=1042-1052; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=The role of foraging profitability and experience in habitat use by the bluegill sunfish.; citation_author=EE Werner; citation_author=GG Mittelbach; citation_author=DJ Hall; citation_journal_title=Ecology; citation_volume=62; citation_number=8; citation_pages=116-125; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Optimal prey selection in great tit (Parus major).; citation_author=JR Krebs; citation_author=JT Erichsen; citation_author=MI Webber; citation_author=EL Charnov; citation_journal_title=Animal Behaviour; citation_volume=25; citation_number=9; citation_pages=30-38; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Bacteriophage latent-period evolution as a response to resource availability.; citation_author=ST Abedon; citation_author=TD Herschler; citation_author=D Stopar; citation_journal_title=Applied and Environmental Microbiology; citation_volume=67; citation_number=10; citation_pages=4233-4241; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Optimal foraging: Attack strategy of a mantid.; citation_author=EL Charnov; citation_journal_title=The American Naturalist; citation_volume=110; citation_number=11; citation_pages=141-151; citation_date=1976; " />
      <meta name="citation_reference" content="citation_title=Preliminary observations on the concentration of marine bacteriophages in the water around Helgoland.; citation_author=K Moebus; citation_journal_title=Helgolander Meeresuntersuchungen; citation_volume=45; citation_number=12; citation_pages=411-422; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Bacteriophage diversity in the North Sea.; citation_author=A Wichels; citation_author=SS Biel; citation_author=HR Gelderblom; citation_author=T Brinkhoff; citation_author=G Muyzer; citation_journal_title=Applied and Environmental Microbiology; citation_volume=64; citation_number=13; citation_pages=4128-4133; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Virioplankton: Viruses in aquatic ecosystems.; citation_author=KE Wommack; citation_author=RR Colwell; citation_journal_title=Microbiology and Molecular Biology Reviews; citation_volume=64; citation_number=14; citation_pages=69-114; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A method for the detection of bacteriophages from ocean water.; citation_author=K Moebus; citation_journal_title=Helgolander Meeresuntersuchungen; citation_volume=34; citation_number=15; citation_pages=1-14; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Bacteriophage sensitivity patterns among bacteria isolated from marine waters.; citation_author=K Moebus; citation_author=H Nattkemper; citation_journal_title=Helgolander Meeresuntersuchungen; citation_volume=34; citation_number=16; citation_pages=375-&; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Further invetigations on the concentration of marine bacteriophages in the water around Helgoland, with refrence to the phage-host system encountered.; citation_author=K Moebus; citation_journal_title=Helgolander Meeresuntersuchungen; citation_volume=46; citation_number=17; citation_pages=275-292; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Bacterial diversity in aquatic and other environments: what 16S rDNA libraries can tell us.; citation_author=PF Kemp; citation_author=JY Aller; citation_journal_title=FEMS Microbiology Ecology; citation_volume=47; citation_number=18; citation_pages=161-177; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Species-energy theory - an exptension of species-area theory.; citation_author=DH Wright; citation_journal_title=Oikos; citation_volume=41; citation_number=19; citation_pages=496-506; citation_date=1983; " />
      <meta name="citation_reference" content="citation_author=GS Stent; citation_title=Molecular biology of bacterial viruses; citation_number=20; citation_date=1963; citation_publisher=W. H. Freeman; " />
      <meta name="citation_reference" content="citation_author=EL Charnov; citation_title=Optimal foraging: some theoretical explorations [Ph.D.]; citation_number=21; citation_pages=157; citation_date=1973; citation_publisher=University of Washington; " />
      <meta name="citation_reference" content="citation_title=Optimal foraging: a selective review of theory and tests.; citation_author=GH Pyke; citation_author=HR Pulliam; citation_author=EL Charnov; citation_journal_title=The Quarterly Review of Biology; citation_volume=52; citation_number=22; citation_pages=137-154; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Exceptional convergent evolution in a virus.; citation_author=JJ Bull; citation_author=MR Badgett; citation_author=HA Wichman; citation_author=JP Huelsenbeck; citation_author=DM Hillis; citation_journal_title=Genetics; citation_volume=147; citation_number=23; citation_pages=1497-1507; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Profiles of adaptation in two similar viruses.; citation_author=KK Holder; citation_author=JJ Bull; citation_journal_title=Genetics; citation_volume=159; citation_number=24; citation_pages=1393-1404; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mutagenic treatment of double- and single- stranded DNA phages T4 and S13 with hydroxylamine.; citation_author=I Tessman; citation_journal_title=Virology; citation_volume=35; citation_number=25; citation_pages=330-&; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=High abundance of viruses found in aquatic environments.; citation_author=O Bergh; citation_author=KY Borsheim; citation_author=G Bratbak; citation_author=M Heldal; citation_journal_title=Nature; citation_volume=340; citation_number=26; citation_pages=467-468; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The first two steps of the invasion of host cells by bacterial viruses. II.; citation_author=A Garen; citation_author=TT Puck; citation_journal_title=Journal of Experimental Medicine; citation_volume=94; citation_number=27; citation_pages=177-189; citation_date=1951; " />
      <meta name="citation_reference" content="citation_title=Constraints on the coevolution of bacteria and virulent phage - A oodel, some experiments, and predictions for natural communities.; citation_author=RE Lenski; citation_author=BR Levin; citation_journal_title=American Naturalist; citation_volume=125; citation_number=28; citation_pages=585-602; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Marine cyanophages infecting oceanic and coastal strains of Synechococcus - Abundance, morphology, cross-infectivity and growth characteristics.; citation_author=CA Suttle; citation_author=AM Chan; citation_journal_title=Marine Ecology-Progress Series; citation_volume=92; citation_number=29; citation_pages=99-109; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The effect of phosphate status on the kinetics of cyanophage infection in the oceanic cyanobacterium Synechococcus sp WH7803.; citation_author=WH Wilson; citation_author=NG Carr; citation_author=NH Mann; citation_journal_title=Journal of Phycology; citation_volume=32; citation_number=30; citation_pages=506-516; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Characterization of a cold-active bacteriophage on two psychrophilic marine hosts.; citation_author=L Wells; citation_author=J Deming; citation_journal_title=Aquatic Microbial Ecology; citation_volume=45; citation_number=31; citation_pages=15-29; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genomic sequence and evolution of marine cyanophage P60: a new insight on lytic and lysogenic phages.; citation_author=F Chen; citation_author=JR Lu; citation_journal_title=Applied and Environmental Microbiology; citation_volume=68; citation_number=32; citation_pages=2589-2594; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Isolation and characterization of a cyanophage infecting the toxic cyanobacterium Microcystis aeruginosa.; citation_author=T Yoshida; citation_author=Y Takashima; citation_author=Y Tomaru; citation_author=Y Shirai; citation_author=Y Takao; citation_journal_title=Applied and Environmental Microbiology; citation_volume=72; citation_number=33; citation_pages=1239-1247; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Characterization of marine temperate phage-host systems isolated from Mamala Bay, Oahu, Hawaii.; citation_author=SC Jiang; citation_author=CA Kellogg; citation_author=JH Paul; citation_journal_title=Applied and Environmental Microbiology; citation_volume=64; citation_number=34; citation_pages=535-542; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Nucleic acids from the host bacterium as a major source of nucleotides for 3 marine bacteriophages.; citation_author=J Wikner; citation_author=JJ Vallino; citation_author=GF Steward; citation_author=DC Smith; citation_author=F Azam; citation_journal_title=FEMS Microbiology Ecology; citation_volume=12; citation_number=35; citation_pages=237-248; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Effects of bacteriophages on the population dynamics of four strains of pelagic marine bacteria.; citation_author=M Middelboe; citation_author=A Hagstrom; citation_author=N Blackburn; citation_author=B Sinn; citation_author=U Fischer; citation_journal_title=Microbial Ecology; citation_volume=42; citation_number=36; citation_pages=395-406; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Impact of initial phage/host ratio and nutrient addition on coexistence in a phage-host system.; citation_author=A Wolf; citation_author=TL Zheng; citation_author=KP Witzel; citation_author=G Jost; citation_journal_title=Aquatic Microbial Ecology; citation_volume=35; citation_number=37; citation_pages=131-139; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cyanophages infecting the oceanic cyanobacterium Prochlorococcus.; citation_author=MB Sullivan; citation_author=JB Waterbury; citation_author=SW Chisholm; citation_journal_title=Nature; citation_volume=424; citation_number=38; citation_pages=1047-1051; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Population Biology of Bacterial-Viruses - Why Be Temperate.; citation_author=FM Stewart; citation_author=BR Levin; citation_journal_title=Theoretical Population Biology; citation_volume=26; citation_number=39; citation_pages=93-117; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Dynamics of the pseudolysogenic response in slowly growing cells of Pseudomonas aeruginosa.; citation_author=S Ripp; citation_author=RV Miller; citation_journal_title=Microbiology; citation_volume=144; citation_number=40; citation_pages=2225-2232; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Ecology of prokaryotic viruses.; citation_author=MG Weinbauer; citation_journal_title=FEMS Microbiology Reviews; citation_volume=28; citation_number=41; citation_pages=127-181; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Plankton blooms - Lysogeny in marine Synechococcus.; citation_author=L McDaniel; citation_author=LA Houchin; citation_author=SJ Williamson; citation_author=JH Paul; citation_journal_title=Nature; citation_volume=415; citation_number=42; citation_pages=496-496; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Seasonal variation in lysogeny as depicted by prophage induction in Tampa Bay, Florida.; citation_author=SJ Williamson; citation_author=LA Houchin; citation_author=L McDaniel; citation_author=JH Paul; citation_journal_title=Applied and Environmental Microbiology; citation_volume=68; citation_number=43; citation_pages=4307-4314; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Selection for bacteriophage latent period length by bacterial density - a theretical examination.; citation_author=ST Abedon; citation_journal_title=Microbial Ecology; citation_volume=18; citation_number=44; citation_pages=79-88; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The evolution of phage lysis timing.; citation_author=I-N Wang; citation_author=DE Dykhuizen; citation_author=LB Slobodkin; citation_journal_title=Evolutionary Ecology; citation_volume=10; citation_number=45; citation_pages=545-558; citation_date=1996; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001946" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Optimal Foraging Predicts the Ecology but Not the Evolution of Host Specialization in Bacteriophages"/>
    <meta name="twitter:description" content="We explore the ability of optimal foraging theory to explain the observation among marine bacteriophages that host range appears to be negatively correlated with host abundance in the local marine environment. We modified Charnov's classic diet composition model to describe the ecological dynamics of the related generalist and specialist bacteriophages &phi;X174 and G4, and confirmed that specialist phages are ecologically favored only at high host densities. Our modified model accurately predicted the ecological dynamics of phage populations in laboratory microcosms, but had only limited success predicting evolutionary dynamics. We monitored evolution of attachment rate, the phenotype that governs diet breadth, in phage populations adapting to both low and high host density microcosms. Although generalist &phi;X174 populations evolved even broader diets at low host density, they did not show a tendency to evolve the predicted specialist foraging strategy at high host density. Similarly, specialist G4 populations were unable to evolve the predicted generalist foraging strategy at low host density. These results demonstrate that optimal foraging models developed to explain the behaviorally determined diets of predators may have only limited success predicting the genetically determined diets of bacteriophage, and that optimal foraging probably plays a smaller role than genetic constraints in the evolution of host specialization in bacteriophages."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001946.g005"/>

  <meta property="og:title" content="Optimal Foraging Predicts the Ecology but Not the Evolution of Host Specialization in Bacteriophages" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001946" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4017'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=860'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2517&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001946">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001946" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001946&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Optimal Foraging Predicts the Ecology but Not the Evolution of Host Specialization in Bacteriophages
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sébastien Guyader, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, University of North Carolina, Chapel Hill, North Carolina, United States of America
                </p>

                <p>
                    Current address: INRA, UR979, Productions Végétales, Petit-Bourg, France
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christina L. Burch
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:cburch@bio.unc.edu">cburch@bio.unc.edu</a></p>

                <p>Affiliation:
                  Department of Biology, University of North Carolina, Chapel Hill, North Carolina, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 16, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001946</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001946" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001946" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001946" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001946" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001946-g001" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001946-t001" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001946-g002" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001946-t002" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001946-g003" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001946-g004" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001946-g005" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001946">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001946" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>We explore the ability of optimal foraging theory to explain the observation among marine bacteriophages that host range appears to be negatively correlated with host abundance in the local marine environment. We modified Charnov's classic diet composition model to describe the ecological dynamics of the related generalist and specialist bacteriophages φX174 and G4, and confirmed that specialist phages are ecologically favored only at high host densities. Our modified model accurately predicted the ecological dynamics of phage populations in laboratory microcosms, but had only limited success predicting evolutionary dynamics. We monitored evolution of attachment rate, the phenotype that governs diet breadth, in phage populations adapting to both low and high host density microcosms. Although generalist φX174 populations evolved even broader diets at low host density, they did not show a tendency to evolve the predicted specialist foraging strategy at high host density. Similarly, specialist G4 populations were unable to evolve the predicted generalist foraging strategy at low host density. These results demonstrate that optimal foraging models developed to explain the behaviorally determined diets of predators may have only limited success predicting the genetically determined diets of bacteriophage, and that optimal foraging probably plays a smaller role than genetic constraints in the evolution of host specialization in bacteriophages.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Guyader S, Burch CL (2008) Optimal Foraging Predicts the Ecology but Not the Evolution of Host Specialization in Bacteriophages. PLoS ONE 3(4):
          e1946.
            doi:10.1371/journal.pone.0001946</p><p><strong>Editor: </strong>Rees Kassen, University of Ottawa, Canada</p><p><strong>Received:</strong> December 5, 2007; <strong>Accepted:</strong> February 26, 2008; <strong>Published:</strong> April 16, 2008</p><p><strong>Copyright:</strong> © 2008 Guyader, Burch. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by an NSF grant to CLB.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The advantages of host range expansion for parasites include reduced intraspecific competition for hosts, reduced time spent searching for hosts, and an increased probability of transmission to a suitable host <a href="#pone.0001946-Futuyma1">[1]</a>. Despite these advantages, parasites are most often observed to specialize on a limited number of host species. Explanations for the prevalence of host specialists may be ecological or evolutionary, and include ecological tradeoffs between diet breadth and diet quality <a href="#pone.0001946-McPeek1">[2]</a>, and genetic tradeoffs in the ability to utilize alternative hosts <a href="#pone.0001946-Turner1">[3]</a>, <a href="#pone.0001946-Via1">[4]</a>. Unlike investigations of predator foraging strategies that focus on the ecological explanations for specialization, investigations of parasite foraging strategies have focused on evolutionary explanations <a href="#pone.0001946-Turner1">[e.g. 3]</a>, <a href="#pone.0001946-Via1">[4]</a> often ignoring ecological explanations entirely <a href="#pone.0001946-Boivin1">[but see 5]</a>, <a href="#pone.0001946-Garnick1">[6]</a>. The reason for this difference in focus is most likely methodological. Parasites are more amenable to genetic analysis and manipulation than predators, but parasites are less amenable to ecological observation.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The ecological explanation for the prevalence of host specialists comes from an optimal foraging model developed to describe the consequences of predator diet compositions. The model explores the trade-off between diet breadth and diet quality, and generally predicts that predators should specialize on a small number of highly profitable prey types unless prey abundance is low. Diet composition models have accurately predicted the diet composition of predators such as bluegill sunfish <a href="#pone.0001946-Werner1">[7]</a>, <a href="#pone.0001946-Werner2">[8]</a> and great tits <a href="#pone.0001946-Krebs1">[9]</a>, and should be equally applicable to parasites <a href="#pone.0001946-Garnick1">[e.g. 6]</a>, <a href="#pone.0001946-Abedon1">[e.g. 10]</a> as long as their encounter with hosts is governed by a first-order process in which encounter rates increase linearly with host abundance <a href="#pone.0001946-Charnov1">[11]</a>. Thus, diet composition models should accurately describe the ecology of parasites that are transmitted through the environment, but may not apply to parasites that require direct contact between infected and uninfected hosts or transmission by a specialist insect vector.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In this study we investigated the ability of optimal foraging theory to explain the prevalence of host specialists among bacteriophage, parasites that are both transmitted through the environment and amenable to experimental manipulation. Our interest in bacteriophages arose from recent observations that phages collected from higher host density coastal waters have a greater tendency toward host specialization than phages collected from lower host density pelagic ocean waters <a href="#pone.0001946-Moebus1">[12]</a>–<a href="#pone.0001946-Wommack1">[14]</a>. The major observations come from studies that sampled phages by performing plaque assays on numerous bacterial strains that were co-isolated from the same marine waters, as in <a href="#pone.0001946-Moebus2">[15]</a>. The isolated phages were then tested for the ability to grow on bacterial strains other than the strain used to isolate that phage. Only 41 out of 217 phages (18.9%) collected from low nutrient pelagic waters of the North Atlantic were specialists that grew only on the original host strain <a href="#pone.0001946-Moebus3">[16]</a>, whereas 88 out of 194 (45%) phages collected from productive waters off the German coast were host specialists <a href="#pone.0001946-Moebus1">[12]</a>, <a href="#pone.0001946-Moebus4">[17]</a>. These data suggest a relationship between host density and host specialization, as long as bacterial diversity was not higher in the low density environment – a possibility that seems unlikely <a href="#pone.0001946-Kemp1">[18]</a>, <a href="#pone.0001946-Wright1">[19]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Although optimal foraging theory offers an intuitively appealing explanation for this observed relationship between host density and host specialization, even the richest marine environments achieve host densities that seem too low to explain the prevalence of specialist phages in the marine environment <a href="#pone.0001946-Wommack1">[14]</a>. This latter observation suggests that either the ecological dynamics of bacteriophage populations are not accurately described by diet composition models, or that the evolutionary dynamics are not well predicted by such models. To distinguish between these possibilities, we tested the predictions of a diet composition model by monitoring the ecological and evolutionary dynamics in laboratory microcosms of populations of two related phages, the generalist phage φX174 that infects both <em>Escherichia coli</em> and <em>Salmonella enterica</em> serovar Typhimurium (hereafter <em>S. typhimurium</em>), and the specialist phage G4 that infects only <em>E. coli</em>.</p>

<h4>Optimal foraging in bacteriophages</h4>
<a id="article1.body1.sec1.sec1.p1" name="article1.body1.sec1.sec1.p1"></a><p>In order to use the diet composition model of Charnov <a href="#pone.0001946-Charnov1">[11]</a> to describe the consequences of host specialization for bacteriophages, we modified the model slightly to account for the exponential growth dynamics of phages. Although the qualitative prediction that host specialization should be positively correlated with host abundance remains unchanged, the modification is necessary to quantify the host density above which specialization is favored. In Charnov's model, diet breadth changes as a function of the differences in search time (<em>s</em>), handling time (<em>h</em>), and energy content (<em>E</em>) between alternative hosts. These parameters are directly related to the phage life history parameters attachment rate (<em>k</em>), lysis time (<em>L</em>), and burst size (<em>B</em>), respectively.</p>
<a id="article1.body1.sec1.sec1.p2" name="article1.body1.sec1.sec1.p2"></a><p>In phages, attachment to a host bacterium is governed by first-order kinetics <a href="#pone.0001946-Stent1">[20]</a>, thus phages bind to bacterial hosts according to the equation:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>P</em> is the density of unattached phage, <em>N</em> is the density of hosts, and <em>k</em> is an attachment rate constant. Search time can be calculated from equation 1 as the average time required for attachment to a host:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>Following attachment to a host, phages replicate inside the host until they lyse (kill) the host cell. Thus, handling time is equivalent to lysis time (<em>L</em>), the time that elapses from attachment to lysis. Finally, the burst size (<em>B</em>), or number of progeny released after lysis is equivalent to the relative energy content of the host.</p>
<a id="article1.body1.sec1.sec1.p3" name="article1.body1.sec1.sec1.p3"></a><p>Foraging in phages differs somewhat from the original diet width models <a href="#pone.0001946-Charnov2">[21]</a>, <a href="#pone.0001946-Pyke1">[22]</a> because phage growth rates do not increase linearly with the energy content of the host. Instead, within-host replication of phages leads to an exponential increase in progeny numbers. In this case, phages increase over time according to:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where <em>P</em><sub>0</sub> and <em>P<sub>t</sub></em> are the number of phages present initially and at time <em>t</em>. Note that <em>t</em>/(<em>s</em>+<em>L</em>) is the average number of generations (<em>g</em>) achieved in <em>t</em> minutes, and that the number of progeny produced by a single phage in <em>t</em> minutes is <em>B<sup>g</sup></em>. Because phage populations grow exponentially, the rate of energy intake can be linearized by considering the rate of change in ln(<em>P</em>):<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.e005&amp;representation=PNG" class="inline-graphic"></span> is the average energy content of hosts included in the current diet, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.e006&amp;representation=PNG" class="inline-graphic"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.e007&amp;representation=PNG" class="inline-graphic"></span> are the average search and handling times for hosts already included in the diet.</p>
<a id="article1.body1.sec1.sec1.p4" name="article1.body1.sec1.sec1.p4"></a><p>Using this framework we now consider the following question: Given that the <em>i</em>−1 most profitable hosts – ranked in decreasing order according to their profitability = ln(<em>B<sub>i</sub></em>)/<em>L<sub>i</sub></em> – are already included in the diet, should the phage expand its diet by including the next most profitable host as well? To answer this question, imagine that the phage has encountered (attached to) a new host of rank <em>i</em>. If the phage consumes this host, it will achieve an instantaneous rate of energy intake of ln(<em>B<sub>i</sub></em>)/<em>L<sub>i</sub></em>. If the phage, instead, ignores this host and continues to search for a more profitable host, it will spend <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.e008&amp;representation=PNG" class="inline-graphic"></span> time searching for and handling a more profitable host in order to take in <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.e009&amp;representation=PNG" class="inline-graphic"></span> energy content. In this scenario the optimal strategy is to consume hosts of rank <em>i</em> if, and only if,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.e010&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br></p>
<a id="article1.body1.sec1.sec1.p5" name="article1.body1.sec1.sec1.p5"></a><p>The primary prediction of equation 5 is that host range should be negatively correlated with host abundance. When host abundance is low, search times are large, making it easy to satisfy the inequality. Bacteriophage should consume any host they encounter. As host abundance increases, search times approach zero, and it becomes increasingly difficult to satisfy the inequality. In this case, bacteriophage should consume only the most profitable hosts. We note that these are the classical predictions of the diet composition model <a href="#pone.0001946-Charnov2">[21]</a>. We have only fitted the classical model to describe the particular case of bacteriophage dynamics.</p>
<a id="article1.body1.sec1.sec1.p6" name="article1.body1.sec1.sec1.p6"></a><p>In the experiments that follow, we use equation 5 to identify the threshold host density above which specialists should be favored in laboratory microcosms containing two hosts. We then test the primary prediction of the model – that host range should be negatively correlated with host abundance – by examining the ecological and evolutionary dynamics of phage populations in laboratory microcosms maintained at host densities above and below this threshold.</p>

</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Phage and bacterial strains</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The bacteriophages φX174 and G4 are laboratory clones described in Bull et al. <a href="#pone.0001946-Bull1">[23]</a> and Holder and Bull <a href="#pone.0001946-Holder1">[24]</a>, respectively. These phages and their standard laboratory hosts <em>E. coli</em> strain C and <em>S. typhimurium</em> strain LT2 were obtained from Holly Wichman (University of Idaho). φX174 infects both hosts; G4 infects only <em>E. coli</em> C.</p>


<h4>General culture conditions</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>All phages and bacteria were grown at 37°C with shaking in LB broth (10 g NaCl, 10 g Bacto-tryptone, and 5 g yeast extract per liter of water) supplemented with 2 mM CaCl<sub>2</sub>, and were stored in LB at 4°C for immediate use or in 4:6 glycerol/LB (v/v) at −80°C for later use. The phage lysates used to initiate experiments were prepared by mixing approximately 10<sup>3</sup> phages, 200 µL of an overnight <em>E. coli</em> culture (about 10<sup>9</sup> bacteria/mL), and 3 mL of LB top agar (0.7% agar), and overlaying this mixture on an LB agar plate (1.5% agar). After a 4 hour incubation at 37°C, the top agar surface was harvested and mixed with 2 mL of LB. The top agar and bacteria were pelleted by centrifugation at 3000 rpm for 10 minutes, and the supernatant was stored as described. When needed, phage lysates were titered by plating a small amount of phages as described above, and counting the resulting number of plaques.</p>


<h4>Attachment Rate Assays</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Attachment rate constants were determined as in <a href="#pone.0001946-Stent1">[20]</a>. Approximately 10<sup>3</sup> phages were mixed with 1 mL of an exponentially growing culture at a density of ~10<sup>8</sup> bacteria/mL and incubated with shaking at 37°C. Immediately (time <em>t</em> = 0), and then again after a few minutes (2, 4, 6 and 8 minutes for foraging model parameterization; 5 minutes only for evolution experiments), 150 µL of this mixture was centrifuged for one minute at 10K rpm and 100 µL of the supernatant was plated to count the unbound phages. The rate at which the concentration of unbound phages decreases is determined by the following equation:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br>where <em>P<sub>t</sub></em> is the concentration of unbound phages at time <em>t</em> minutes, <em>k</em> is the attachment rate constant, and <em>N</em> is the concentration of bacteria. Therefore, we calculated the attachment rate constant (<em>k</em>) as the slope of the regression of ln(<em>P<sub>t</sub></em>/<em>P<sub>0</sub></em>) over time divided by the bacteria concentration, <em>N</em>. For each assay, <em>N</em> was measured by plating a 10<sup>−6</sup> dilution of the bacterial cultures on LB plates. Attachment rates were measured in triplicate, except for the evolution experiment where it was measured one time from each of 6 independent plaque-purified clones.</p>


<h4>Lysis Time and Burst Size</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Lysis times were estimated from one step growth curves (<a href="#pone-0001946-g001">Figure 1</a>). Approximately 10<sup>3</sup> phages were mixed with 3 mL of an exponentially growing culture at a density of 10<sup>8</sup> bacteria/mL, and incubated shaking at 37°C. Aliquots of this mixture were plated at different time intervals to monitor the increase in phage concentration. Preliminary experiments indicated that lysis time occurred at about 15 minutes, so we plated to estimate phage concentration every 5 minutes for the first 10 minutes (well before lysis), and every 2 minutes between 14 and 22 minutes, and between 26 and 32 minutes (the time intervals in which lysis events were expected to occur). An initial latent period in which the concentration of plaque forming units (PFU) remains constant was followed by a lysis period in which the PFU concentration increases sharply (note that before lysis, the number of progeny phages increases, but as they remain inside the same bacterial cell, the apparent count in PFU is unchanged). The first time point at which the phage concentration increased to more than twice the initial concentration was identified, and lysis time (<em>L</em>) was calculated as the mean between this time point (immediately post lysis) and the previous time point (immediately pre lysis). Burst sizes were also estimated from one step growth curves (<a href="#pone-0001946-g001">Figure 1</a>). Once all infected cells have been lysed, the concentration of PFU levels off and achieves a maximum value before the next lysis period begins. Burst size was then determined by dividing this maximum value (obtained at time <em>t</em> = 2<em>L</em>) by the initial PFU concentration. Lysis times and burst sizes were measured in triplicate.</p>
<div class="figure" id="pone-0001946-g001"><div class="img"><a name="pone-0001946-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001946.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001946.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>G4 growth curve illustrates how lysis time (<em>L</em>) and burst size (<em>B</em>) were determined.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Data are means and standard errors for the number of plaque-forming units (PFU = number of infected cells+free phage particles) at various times during the course of infection. The curve was drawn by hand through the data for illustrative purposes. Two bursts occur during this time window. The first burst started around 15 minutes, the second burst started around 30 minutes.</p>
<span>doi:10.1371/journal.pone.0001946.g001</span></div>

<h4>Growth Rate Assays</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Host bacteria were grown to a density of 4×10<sup>8</sup> bacteria/mL (exponential growth phase), at which point the appropriate volume of bacterial culture(s) was mixed with approximately 10<sup>3</sup> phages (5×10<sup>2</sup> φX174 and 5×10<sup>2</sup> G4) and sterile LB, both preheated to 37°C, to achieve a final volume of 2 mL. The final bacterial density differed according to 4 different treatments (2 host densities×2 host compositions): 10<sup>8</sup> <em>E. coli</em>/mL (high density×one host), 10<sup>6</sup> <em>E. coli</em>/mL (low density×one host), 10<sup>8</sup> <em>E. coli</em> and 10<sup>8</sup> <em>S. typhimurium</em>/mL (high density×two hosts), or 10<sup>6</sup> <em>E. coli</em> and 10<sup>6</sup> <em>S. typhimurium</em>/mL (low density×two hosts). These mixtures were incubated shaking for 45 minutes at 37°C, allowing 2 full cycles of phage replication and release. The infection process was stopped by adding 50 µL of chloroform to each tube and vortexing. This process destroys bacterial cells, but does not release phage particles trapped inside infected cells. Initially (<em>t</em><sub>0</sub>) and after 45 minutes (<em>t</em><sub>45</sub>), phage concentrations were measured by plating on a mixed lawn consisting of 200 microliters of an overnight <em>E. coli</em> culture and 60 microliters of an onvernight <em>S. typhimurium</em> culture. 200 µL of <em>E. coli</em> ensures that both phages produce clearly visible plaques on the mixed lawn, and 60 µL of <em>S. typhimurium</em> is enough to make G4 plaques appear turbid, but not so much that G4 plaques are no longer clearly visible. φX174 forms clear plaques on this mixed lawn. The growth rates for each phage were calculated as the ratio of phage concentration at <em>t</em><sub>45</sub> to that at <em>t</em><sub>0</sub>, and log transformed for statistical analyses. Each of these experiments were replicated in 5 complete blocks (<em>i.e.</em> one replicate of each treatment), with each block performed on a different day.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We measured the growth rates of φX174 and G4 simultaneously in individual assays to standardize the conditions experienced by generalist and specialist phages, but recognized that the presence of interspecific phages in each assay had the potential to reduce phage growth rates by increasing competition for hosts. To minimize this potential, we conducted the assays at a low multiplicity of infection (MOI, or ratio of phage to hosts) of 10<sup>−5</sup>. In addition, we confirmed that these mixed-phage assays yield the same results as assays conducted separately on the two phages by repeating a subset of the experiments using separate assays to measure the growth rates of φX174 and G4 (data not shown).</p>


<h4>Experimental evolution</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Because the phages used in this study have a low spontaneous mutation rate and are genetically homogeneous (clones), we used chemical mutagenesis to generate genetic variation upon which selection could act. Virus lysates were treated with a 0.5 M hydroxylamine (HA) solution for 30 minutes, at which time 99% of phages were unable to form plaques. At this dose, phages receive an average of 4.65 lethal mutations per genome, and the surviving phages are expected to carry numerous non-lethal mutations. Even at an average dose of 1 lethal mutation per genome, 7% of the surviving phages acquire temperature sensitive mutations <a href="#pone.0001946-Tessman1">[25]</a>. About 10<sup>3</sup> mutagenized phages were plated and harvested in order to both eliminate remaining HA and obtain high titer lysates for experiments. Four independently mutagenized phage populations were obtained and used for serial passage experiments. High host density passages were initiated by adding 10<sup>4</sup> mutagenized phages to 1 mL of LC containing 10<sup>8</sup> cells each of <em>E. coli</em> and <em>S. typhimurium</em> (10<sup>8</sup> bacteria/mL for each host type). Low host density passages were initiated by adding 10<sup>4</sup> mutagenized phages to 100 mL containing 10<sup>6</sup> cells each of <em>E. coli</em> and <em>S. typhimurium</em> (10<sup>6</sup> bacteria/mL for each host type). We aimed to transfer 10<sup>4</sup> phages after each generation (every 30 minutes) by diluting a small volume into new flasks containing host bacteria, so as to keep the MOI below 1 (actual MOI varied from 1.13×10<sup>−4</sup> to 1.2×10<sup>−1</sup>). Ten passages were completed for each of the 4 replicate mutagenized populations, for each host density treatment. After completion of these experiments, samples were treated with chloroform and stored in 40% v/v glycerol/LC at −80°C.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>We screened mutagenized phages for the ability to infect <em>S. typhimurium</em> by plating on mixed lawns of <em>E. coli</em> and <em>S. typhimurium</em>, and looking for clear plaques. Mutagenized phages must be screened on mixed lawns because they do not express their mutant phenotypes until after they replicate once in <em>E. coli</em>.</p>


<h4>Statistical analyses</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>All statistical tests were performed using SAS version 9.1 (SAS Institute, Cary, NC). ANOVAs were implemented using the GLM procedure, and the ANOVA assumptions of homogeneous and normally distributed residuals were confirmed using Shapiro-Wilk's and Levene's tests, respectively. In all cases, block was treated as a random factor and the other factors as fixed. Planned comparisons were made using the TDIFF and PDIFF statements to conduct t-tests and paired t-tests, respectively.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Optimal foraging model predictions</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>To parameterize the optimal foraging model described above, we conducted attachment assays and single-step growth curves to measure attachment rates (<em>k</em>), lysis times (<em>L</em>), and burst sizes (<em>B</em>) of both phage (φX174 and G4) on both hosts (<em>E. coli</em> and <em>S. typhimurium</em>). The resulting parameter values are shown in <a href="#pone-0001946-t001">Table 1</a>. The measurements reveal two major properties of phage growth dynamics on these hosts. First, <em>E. coli</em> is a more profitable host than <em>S. typhimurium</em> for the generalist phage φX174. The higher profitability results both from a shorter handling time (<em>L</em>), and from a higher energy content (ln<em>B</em>) on <em>E. coli</em>. Second, G4 achieves a higher growth rate than φX174 on the shared host, <em>E. coli</em>. The higher growth rate of G4 results from a larger burst size; the two phage have similar attachment rates and lysis times on <em>E. coli</em>.</p>
<div class="figure" id="pone-0001946-t001"><div class="img"><a name="pone-0001946-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001946.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001946.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Life history parameters of φX174 and G4.</span></strong></p><span>doi:10.1371/journal.pone.0001946.t001</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Incorporating the measured parameter values into the left and right sides of equation 5, we determined the density of <em>E. coli</em> below which it should be advantageous for φX174 to include <em>S. typhimurium</em> (the less profitable host) in the diet. In <a href="#pone-0001946-g002">Figure 2</a> we plot the expected rate of energy intake as a function of <em>E. coli</em> density for a diet consisting only of <em>E. coli</em> (right side of equation 5). In addition we plot the profitability of <em>S. typhimurium</em> (left side of equation 5), a quantity that does not depend on host density. The two lines intersect at a host density of 5.03×10<sup>7</sup> bacteria/mL, indicating that inclusion of <em>S. typhimurium</em> in the diet should be advantageous for <em>E. coli</em> densities below 5×10<sup>7</sup> bacteria/mL. In other words, the optimal foraging model predicts that the generalist phenotype of φX174 will be advantageous below this cell density, but costly above it.</p>
<div class="figure" id="pone-0001946-g002"><div class="img"><a name="pone-0001946-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001946.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001946.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Predictions of the optimal foraging model using parameters measured in φX174.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Given that φX174 consumes the more profitable host <em>E. coli</em>, we determined the <em>E. coli</em> densities over which φX174 benefits by also consuming the less profitable host <em>S. typhimurium</em>. φX174 should consume <em>S. typhimurium</em> only at host densities where the profitability of <em>S. typhimurium</em> (dashed line) exceeds the growth rate achieved on a diet of only <em>E. coli</em> (solid line). Both quantities are shown on the same scale as the experimental measures, in which growth rate is calculated as the phage concentration after 45 minutes divided by the initial phage concentration. Growth rate on <em>E. coli</em> was calculated as ln<em>B<sub>Ec</sub></em>/(1/(<em>k<sub>Ec</sub>N</em>)+<em>L<sub>Ec</sub></em>)×45 minutes, and profitability of <em>S. typhimurium</em> was calcualated as ln<em>B<sub>St</sub></em>/<em>L<sub>St</sub></em>×45 minutes. Both quantities are described in equation 5.</p>
<span>doi:10.1371/journal.pone.0001946.g002</span></div>

<h4>Ecological dynamics</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>We measured the growth rates of the generalist (φX174) and specialist (G4) phages under conditions of high (10<sup>8</sup> bacteria/mL) and low (10<sup>6</sup> bacteria/mL) host densities, and in the presence or absence of <em>S. typhimurium</em>. We expect the presence of <em>S. typhimurium</em> to have a positive effect on phage growth rate at low host density and a negative effect at high host density (a host composition×host density interaction), but only for the generalist phage φX174. The specialist phage G4 provides a negative control in this experiment because it is expected to be unaffected by the presence of <em>S. typhimurium</em>. ANOVAs performed separately on the data collected for each phage confirmed these predictions (<a href="#pone-0001946-t002">Table 2</a>), revealing a significant host composition×host density interaction effect on the growth rate of φX174, but not on the growth rate of G4.</p>
<div class="figure" id="pone-0001946-t002"><div class="img"><a name="pone-0001946-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001946.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001946.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>ANOVA test of host density and diet effects on phage growth rates.</span></strong></p><span>doi:10.1371/journal.pone.0001946.t002</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>In <a href="#pone-0001946-g003">figure 3</a>, we illustrated the nature of this interaction effect by plotting the differences between <em>ln</em>(growth rate) in mixed host cultures containing <em>E. coli</em> and <em>S. typhimurium</em> and <em>ln</em>(growth rate) in pure host cultures containing only <em>E. coli</em>. For the generalist phage φX174, the presence of <em>S. typhimurium</em> (the less profitable host) in experimental microcosms significantly decreased growth rate at high host density, but significantly increased growth rate at low density. In contrast, host composition had no significant effect on the growth rate of G4 at either host density.</p>
<div class="figure" id="pone-0001946-g003"><div class="img"><a name="pone-0001946-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001946.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001946.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>The generalist phenotype of φX174 is costly at high host density (10<sup>8</sup> bacteria/mL) but advantageous at low host density (10<sup>6</sup> bacteria/mL).</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>Data are the differences between <em>ln</em>(growth rate) in mixed host cultures containing <em>E. coli</em> and <em>S. typhimurium</em> (ES) and <em>ln</em>(growth rate) in pure host cultures containing only <em>E. coli</em> (E). The dotted line represents equal growth in both host composition treatments. Each data point represents the mean of 5 replicates±95% confidence intervals.</p>
<span>doi:10.1371/journal.pone.0001946.g003</span></div>

<h4>Evolutionary dynamics</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>To determine whether adaptation to low host density – but not high host density – is accomplished by the evolution of generalist traits, we adapted replicate phage populations to experimental low density and high density microcosms containing an equal mixture of <em>E. coli</em> and <em>S. typhimurium</em>. Phage populations were subjected to chemical mutagenesis followed by 10 serial transfers (10–20 generations) into either low or high host density cultures.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Low host density was expected to impose selection to incorporate the less profitable host <em>S. typhimurium</em> into the diet, and to result in an increased rate of attachment to <em>S. typhimurium</em>. In contrast, high host density was expected to impose selection to specialize on the more profitable host <em>E. coli</em>, and to result in a reduced rate of attachment to <em>S. typhimurium</em>. To determine if attachment rate was, in fact, a target of selection in our experiments, we measured the attachment rates to <em>S. typhimurium</em> for each mutagenized population both before and after evolution (<a href="#pone-0001946-g004">fig. 4</a>). As expected, the mean attachment rate of populations adapted to the low host density treatment was significantly higher than the mean of the ancestral populations (<em>t</em> = 5.6076, <em>P</em> = 0.0056), whereas the mean attachment rates of populations adapted to the high density treatment did not change significantly over the course of the experiment (<em>t</em> = 0.9633, <em>P</em> = 0.2032).</p>
<div class="figure" id="pone-0001946-g004"><div class="img"><a name="pone-0001946-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001946.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001946.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Evolutionary response in the attachment rate to <em>S. typhimurium</em> during adaptation to two-host microcosms.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>For individual populations, responses were calculated as <em>k</em><sub>10</sub>−<em>k</em><sub>0,</sub> where <em>k</em><sub>i</sub> was the mean attachment rate measured after <em>i</em> generations (passages) of adaptation to growth at high or low host density. Data are grand means±95% confidence intervals based on four replicate evolution experiments. The black dot (low density) indicates a difference significantly greater than 0 (1-tailed paired <em>t</em>-test, <em>P</em> = 0.0056).</p>
<span>doi:10.1371/journal.pone.0001946.g004</span></div><a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>We had hoped to perform identical experiments using G4 populations, but were unable to generate any mutant phage capable of growth on <em>S. typhimurium</em>. We screened 1.46×10<sup>9</sup> un-mutagenized phages and 2×10<sup>4</sup> mutagenized phages for the ability to infect <em>S. typhimurium</em>, but found no such mutants. 2×10<sup>4</sup> mutagenized phages is 20 times greater than the number of phages used to initiate individual φX174 evolution experiments, allowing us to conclude that genetic constraints prevent the evolution of host generalization in G4 populations.</p>


<h4>Optimal foraging in natural populations</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>We also investigated whether conclusions drawn from observations in φX174 (i.e. specialists favored when <em>N</em> = 10<sup>8</sup>; generalists favored when <em>N</em> = 10<sup>6</sup>) can be generalized to other phages with other life history strategies. In particular, we wanted to determine whether the life history strategies observed in marine phages, and the host densities observed in marine environments are predicted to favor specialist or generalist phages.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>We asked the following question. Given that a phage already consumes the most profitable host (host 1) with profitability <em>Pr</em><sub>1</sub> = ln<em>B</em><sub>1</sub>/<em>L</em><sub>1</sub>, and inhabits an environment with host density <em>N</em>, should the phage consume host 2 that is a fraction <em>x</em> as profitable (i.e. <em>Pr</em><sub>2</sub> = <em>x Pr</em><sub>1</sub>)? By substituting <em>xPr</em><sub>1</sub> for <em>Pr</em><sub>2</sub> in equation 5, and rearranging the resulting inequality, we find that phage should consume host 2 only if the following inequality is met:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>We used equation (7) to identify the attachment rate and lysis time combinations that favor specialists when host densities are close to the maximum (5×10<sup>6</sup> hosts/ml), mean (1×10<sup>6</sup>), and median (5×10<sup>5</sup>) observations from natural marine environments <a href="#pone.0001946-Wommack1">[14]</a>, <a href="#pone.0001946-Bergh1">[26]</a> (<a href="#pone-0001946-g005">Figure 5A</a>), and when the second most profitable host in the environment is 75%, 50%, or 25% as profitable as the most profitable host (<a href="#pone-0001946-g005">Figure 5B</a>).</p>
<div class="figure" id="pone-0001946-g005"><div class="img"><a name="pone-0001946-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001946" data-uri="info:doi/10.1371/journal.pone.0001946.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001946.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001946.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001946.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001946.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001946.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Optimal diet breadth given observed life history strategies in phages.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>Lines depict the attachment rate and lysis time combinations at which generalist and specialist phages should achieve equal growth rates for particular host combinations and densities, as predicted by equation 7 (<em>k</em><sub>1</sub> = <em>x</em>/[(1<em>−x</em>)<em>L</em><sub>1</sub><em>N</em>]). Here, we assume that the second best host is a proportion <em>x</em> as profitable as the best host, and specify a particular host density <em>N</em>. <em>k</em><sub>1</sub> and <em>L</em><sub>1</sub> represent the attachment rate and lysis time, respectively, of the most profitable host. Combinations above each line favor specialists; combinations below each line favor generalists. The shaded portion of the graph indicates the region over which phage life history traits have been observed. (A) Observed attachment rate and lysis times overlap the region that favors specialists only when host density is high. Here we fixed <em>x</em> = 0.75, and examined the cases where <em>N</em> = 5×10<sup>5</sup> (solid line), 1×10<sup>6</sup> (dashed line), and 5×10<sup>6</sup> hosts/ml (dotted line). (B) Observed attachment rate and lysis times overlap the region that favors specialists only when differences in host profitability are large. Here we fixed <em>N</em> = 1×10<sup>6</sup>, and examined the cases where the second best host is 75% (solid line), 50% (dashed line), and 25% (dotted line) as profitable as the best host.</p>
<span>doi:10.1371/journal.pone.0001946.g005</span></div><a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>Among marine phages, observed attachment rate constants fall between 4×10<sup>−11</sup> and 6×10<sup>−9</sup> mL/min <a href="#pone.0001946-Stent1">[20]</a>, <a href="#pone.0001946-Garen1">[27]</a>–<a href="#pone.0001946-Wells1">[31]</a> and observed lysis times fall between 40 minutes and 12 hours <a href="#pone.0001946-Wilson1">[30]</a>, <a href="#pone.0001946-Chen1">[32]</a>–<a href="#pone.0001946-Wolf1">[37]</a> – describing a region of parameter space generally predicted to favors generalists (<a href="#pone-0001946-g005">Figure 5</a>). Although phages with the highest attachment rates and lysis times are predicted to specialize, the prediction only holds in rare environments where hosts are close to their maximum observed density (i.e. <em>N</em>~5×10<sup>6</sup>) or in which the second best host is much less profitable than the best host (e.g., <em>Pr</em><sub>2</sub>&lt;0.5 <em>Pr</em><sub>1</sub>).</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Our results confirm that optimal foraging models of diet composition developed to describe the ecological consequences of predator foraging decisions accurately describe the ecological dynamics of bacteriophage populations in laboratory microcosms. Reproductive rates of the generalist and specialist bacteriophages φX174 and G4 conformed to the theoretical prediction that the reproductive advantage associated with host specialization depends on host density. The generalist phenotype of φX174 was advantageous at low host density (10<sup>6</sup> hosts/ml), but costly at high host density (10<sup>8</sup> hosts/ml).</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>The qualitative similarity between these results and observations from marine environments – that phages collected from higher host density coastal waters have a greater tendency toward host specialization than phages collected from lower host density pelagic ocean waters <a href="#pone.0001946-Moebus1">[12]</a>–<a href="#pone.0001946-Wommack1">[14]</a> – suggests that diet breadth models may also explain bacteriophage host ranges in natural populations. However, the quantitative match between our data and observations from marine environments is not as good. In particular, the average marine environment achieves a bacterial density of approximately 10<sup>6</sup>/ml. As this bacterial density represents then entire bacterial community, the density of particular bacterial species is likely to be much lower. Our analysis suggests that this density is not sufficient to favor the evolution of specialist phages, and yet most phages isolated from nature infect only one or a few bacterial hosts <a href="#pone.0001946-Sullivan1">[38]</a>.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Although optimal foraging does not appear to explain the prevalence of specialist phages in marine environments, the explanation may still be primarily ecological. For example, the scarcity of hosts (relative to phages) in the marine environment may impose intense resource competition, and host specialization may evolve as a mechanism to reduce this competition. On the other hand, we provide evidence that genetic constraints also contribute to the prevalence of specialist phages in nature.</p>

<h4>Optimal foraging when diet breadth is determined genetically</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>A second explanation for the failure of the diet width model to explain the foraging patterns of bacteriophages in nature is that bacteriophages are parasites and not predators. Unlike predators that can change diet width behaviorally, bacteriophages change diet width only through genetic mutations. Thus, bacteriophages may be constrained in their diet choice in a way that predators are not. Our data support this expectation. Although the diet width model accurately predicted the <em>direction</em> of selection, it had only a limited ability to predict the <em>response</em> to selection. The ability to predict the response to selection depended on the mutational accessibility of appropriate genotypes. φX174 populations that were capable of generating appropriate genetic variation adapted to low host density as predicted, by increasing utilization of the less profitable host. In contrast, G4 populations were unable to produce mutations that allowed infection of <em>S. typhimurium</em> and were, therefore, unable to evolve a generalist phenotype.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>We considered whether the inability of G4 populations to evolve a generalist phenotype resulted because G4 has less evolutionary potential than other phages, but we think this possibility unlikely. In particular, the fact that the φX174 genome encodes an identical set of genes, but possesses a generalist phenotype, indicates that these phages are not characterized by unusual functional constraints that prevent a generalist phenotype. Although phages that use alternative mechanisms of attachment (e.g. tailed phages which attach to their hosts through an appendage instead of directly with their capsid components) may be less evolutionarily constrained than G4, it is also possible that specialist phages often occupy regions of sequence space from which generalist genotypes are inaccessible.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Genetic constraints that act to limit host range in environments where hosts are rare would seem to pose a huge problem for bacteriophage survival and reproduction. One way that bacteriophage may get around this problem is to use a vertical mode of transmission (lysogeny) when host density is too low for horizontal transmission (lysis) to be effective <a href="#pone.0001946-Stewart1">[39]</a>. Lysogeny is a process by which bacteriophage genomes integrate into the genome of the host, and are transmitted to the progeny during replication of the host genome. Some phages are also capable of entering into a pseudolysogeny state in which the viruses stay in a limited lytic mode, but still allow the host bacteria to grow and divide, ensuring their vertical transmission <a href="#pone.0001946-Ripp1">[40]</a>. Both lysogeny and pseudolysogeny occur in marine environments <a href="#pone.0001946-Weinbauer1">[41]</a>, and moreover, there is evidence that the proportion of bacteria containing lysogenic phage is negatively correlated with bacterial density <a href="#pone.0001946-McDaniel1">[42]</a>, <a href="#pone.0001946-Williamson1">[43]</a>. This correlation is consistent with the optimal foraging predictions made here. The disadvantage of host specialization, and the corresponding advantage of lysogeny, is predicted to decrease with increasing bacterial density.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>We conclude with the final caveat that attempts to explain the evolution of bacteriophage diet width using purely ecological models may be limited by the extent to which the underlying parameters search time, handling time, and energy content can themselves evolve. The difficulty arises from the difference between bacteriophage parasites and predators discussed above – bacteriophage diets are determined genetically and approach optimality over an evolutionary rather than ecological timescale. The limited utility of the diet width model for predicting phenotypic evolution was apparent in the evolution experiments in which we adapted phage populations to a high host density. The diet width model and our ecological experiments both indicated that high host density would impose selection for reduced consumption of the inferior host <em>S. typhimurium</em>. Consumption of <em>S. typhimurium</em> is mediated by the rate of attachment to <em>S. typhimurium</em>, but this trait did not evolve in the high host density evolution experiments. A possible explanation for this outcome is that selection was also acting on lysis time and burst size <a href="#pone.0001946-Abedon1">[10]</a>, <a href="#pone.0001946-Abedon2">[44]</a>, <a href="#pone.0001946-Wang1">[45]</a> to increase the profitability of <em>S. typhimurium</em>. If host profitabilities and diet width evolve simultaneously, optimal diet choice becomes a chicken and egg problem. In cases where parasite diets do appear optimal, two explanations are possible. Parasite diet breadth could have evolved to match available host profitabilities, or host profitabilities could have evolved to match the current diet.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank C.E. Mitchell and members of the Burch lab for comments on the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: CB SG. Performed the experiments: SG. Analyzed the data: CB SG. Wrote the paper: CB SG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001946-Futuyma1" id="pone.0001946-Futuyma1"></a>Futuyma DJ, Moreno G (1988) The evolution of ecological specialization. Annual Review of Ecology and Systematics  19: 207–233.  <ul class="find" data-citedArticleID="1087422" data-doi="10.1146/annurev.es.19.110188.001231"><li><a href="http://dx.doi.org/10.1146/annurev.es.19.110188.001231" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+ecological+specialization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+ecological+specialization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001946-McPeek1" id="pone.0001946-McPeek1"></a>McPeek MA (1996) Trade-offs, food web structure, and the coexistence of habitat specialists and generalists. American Naturalist  148: S124–S138.  <ul class="find" data-citedArticleID="1087440" data-doi="10.1086/285906"><li><a href="http://dx.doi.org/10.1086/285906" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trade-offs%2C+food+web+structure%2C+and+the+coexistence+of+habitat+specialists+and+generalists." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trade-offs%2C+food+web+structure%2C+and+the+coexistence+of+habitat+specialists+and+generalists.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001946-Turner1" id="pone.0001946-Turner1"></a>Turner PE, Elena SF (2000) Cost of host radiation in an RNA virus. Genetics  156: 1465–1470.  <ul class="find" data-citedArticleID="1087466"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cost+of+host+radiation+in+an+RNA+virus.&amp;auth=&amp;atitle=Cost+of+host+radiation+in+an+RNA+virus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cost+of+host+radiation+in+an+RNA+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cost+of+host+radiation+in+an+RNA+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001946-Via1" id="pone.0001946-Via1"></a>Via S (1990) Ecological genetics and host adaptation in herbivorous insects - the experimental study of evolution in natural and agricultural systems. Annual review of entomology  35: 421–446.  <ul class="find" data-citedArticleID="1087468" data-doi="10.1146/annurev.ento.35.1.421"><li><a href="http://dx.doi.org/10.1146/annurev.ento.35.1.421" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecological+genetics+and+host+adaptation+in+herbivorous+insects+-+the+experimental+study+of+evolution+in+natural+and+agricultural+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecological+genetics+and+host+adaptation+in+herbivorous+insects+-+the+experimental+study+of+evolution+in+natural+and+agricultural+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001946-Boivin1" id="pone.0001946-Boivin1"></a>Boivin G, Fauvergue X, Wajnberg E (2004) Optimal patch residence time in egg parasitoids: innate versus learned estimate of patch quality. Oecologia  138: 640–647.  <ul class="find" data-citedArticleID="1087412" data-doi="10.1007/s00442-003-1469-z"><li><a href="http://dx.doi.org/10.1007/s00442-003-1469-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimal+patch+residence+time+in+egg+parasitoids%3A+innate+versus+learned+estimate+of+patch+quality." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimal+patch+residence+time+in+egg+parasitoids%3A+innate+versus+learned+estimate+of+patch+quality.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001946-Garnick1" id="pone.0001946-Garnick1"></a>Garnick E (1992) Niche breadth in parasites: an evolutionarily stable strategy model, with special reference to the protozoan parasite Leishmania. Theoretical Population Biology  42: 62–103.  <ul class="find" data-citedArticleID="1087426" data-doi="10.1016/0040-5809(92)90005-e"><li><a href="http://dx.doi.org/10.1016/0040-5809(92)90005-e" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Niche+breadth+in+parasites%3A+an+evolutionarily+stable+strategy+model%2C+with+special+reference+to+the+protozoan+parasite+Leishmania." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Niche+breadth+in+parasites%3A+an+evolutionarily+stable+strategy+model%2C+with+special+reference+to+the+protozoan+parasite+Leishmania.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001946-Werner1" id="pone.0001946-Werner1"></a>Werner EE, Hall DJ (1974) Optimal foraging and size selection of prey by bluegill sunfish (Lepomis macrochirus). Ecology  55: 1042–1052.  <ul class="find" data-citedArticleID="1087476" data-doi="10.2307/1940354"><li><a href="http://dx.doi.org/10.2307/1940354" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimal+foraging+and+size+selection+of+prey+by+bluegill+sunfish+%28Lepomis+macrochirus%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimal+foraging+and+size+selection+of+prey+by+bluegill+sunfish+%28Lepomis+macrochirus%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001946-Werner2" id="pone.0001946-Werner2"></a>Werner EE, Mittelbach GG, Hall DJ (1981) The role of foraging profitability and experience in habitat use by the bluegill sunfish. Ecology  62: 116–125.  <ul class="find" data-citedArticleID="1087478" data-doi="10.2307/1936675"><li><a href="http://dx.doi.org/10.2307/1936675" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+foraging+profitability+and+experience+in+habitat+use+by+the+bluegill+sunfish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+foraging+profitability+and+experience+in+habitat+use+by+the+bluegill+sunfish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001946-Krebs1" id="pone.0001946-Krebs1"></a>Krebs JR, Erichsen JT, Webber MI, Charnov EL (1977) Optimal prey selection in great tit (Parus major). Animal Behaviour  25: 30–38.  <ul class="find" data-citedArticleID="1087434" data-doi="10.1016/0003-3472(77)90064-1"><li><a href="http://dx.doi.org/10.1016/0003-3472(77)90064-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimal+prey+selection+in+great+tit+%28Parus+major%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimal+prey+selection+in+great+tit+%28Parus+major%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001946-Abedon1" id="pone.0001946-Abedon1"></a>Abedon ST, Herschler TD, Stopar D (2001) Bacteriophage latent-period evolution as a response to resource availability. Applied and Environmental Microbiology  67: 4233–4241.  <ul class="find" data-citedArticleID="1087406" data-doi="10.1128/aem.67.9.4233-4241.2001"><li><a href="http://dx.doi.org/10.1128/aem.67.9.4233-4241.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteriophage+latent-period+evolution+as+a+response+to+resource+availability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteriophage+latent-period+evolution+as+a+response+to+resource+availability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001946-Charnov1" id="pone.0001946-Charnov1"></a>Charnov EL (1976) Optimal foraging: Attack strategy of a mantid. The American Naturalist  110: 141–151.  <ul class="find" data-citedArticleID="1087416" data-doi="10.1086/283054"><li><a href="http://dx.doi.org/10.1086/283054" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimal+foraging%3A+Attack+strategy+of+a+mantid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimal+foraging%3A+Attack+strategy+of+a+mantid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001946-Moebus1" id="pone.0001946-Moebus1"></a>Moebus K (1992) Preliminary observations on the concentration of marine bacteriophages in the water around Helgoland. Helgolander Meeresuntersuchungen  45: 411–422.  <ul class="find" data-citedArticleID="1087444" data-doi="10.1007/bf02367176"><li><a href="http://dx.doi.org/10.1007/bf02367176" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preliminary+observations+on+the+concentration+of+marine+bacteriophages+in+the+water+around+Helgoland." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preliminary+observations+on+the+concentration+of+marine+bacteriophages+in+the+water+around+Helgoland.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001946-Wichels1" id="pone.0001946-Wichels1"></a>Wichels A, Biel SS, Gelderblom HR, Brinkhoff T, Muyzer G, et al.  (1998) Bacteriophage diversity in the North Sea. Applied and Environmental Microbiology  64: 4128–4133.  <ul class="find" data-citedArticleID="1087480"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bacteriophage+diversity+in+the+North+Sea.&amp;auth=&amp;atitle=Bacteriophage+diversity+in+the+North+Sea." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteriophage+diversity+in+the+North+Sea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteriophage+diversity+in+the+North+Sea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001946-Wommack1" id="pone.0001946-Wommack1"></a>Wommack KE, Colwell RR (2000) Virioplankton: Viruses in aquatic ecosystems. Microbiology and Molecular Biology Reviews  64: 69–114.  <ul class="find" data-citedArticleID="1087490" data-doi="10.1128/mmbr.64.1.69-114.2000"><li><a href="http://dx.doi.org/10.1128/mmbr.64.1.69-114.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Virioplankton%3A+Viruses+in+aquatic+ecosystems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Virioplankton%3A+Viruses+in+aquatic+ecosystems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001946-Moebus2" id="pone.0001946-Moebus2"></a>Moebus K (1980) A method for the detection of bacteriophages from ocean water. Helgolander Meeresuntersuchungen  34: 1–14.  <ul class="find" data-citedArticleID="1087446" data-doi="10.1007/bf01983537"><li><a href="http://dx.doi.org/10.1007/bf01983537" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+method+for+the+detection+of+bacteriophages+from+ocean+water." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+method+for+the+detection+of+bacteriophages+from+ocean+water.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001946-Moebus3" id="pone.0001946-Moebus3"></a>Moebus K, Nattkemper H (1981) Bacteriophage sensitivity patterns among bacteria isolated from marine waters. Helgolander Meeresuntersuchungen  34: 375–&amp;.  <ul class="find" data-citedArticleID="1087448" data-doi="10.1007/bf02074130"><li><a href="http://dx.doi.org/10.1007/bf02074130" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteriophage+sensitivity+patterns+among+bacteria+isolated+from+marine+waters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteriophage+sensitivity+patterns+among+bacteria+isolated+from+marine+waters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001946-Moebus4" id="pone.0001946-Moebus4"></a>Moebus K (1992) Further invetigations on the concentration of marine bacteriophages in the water around Helgoland, with refrence to the phage-host system encountered. Helgolander Meeresuntersuchungen  46: 275–292.  <ul class="find" data-citedArticleID="1087450" data-doi="10.1007/bf02367099"><li><a href="http://dx.doi.org/10.1007/bf02367099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Further+invetigations+on+the+concentration+of+marine+bacteriophages+in+the+water+around+Helgoland%2C+with+refrence+to+the+phage-host+system+encountered." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Further+invetigations+on+the+concentration+of+marine+bacteriophages+in+the+water+around+Helgoland%2C+with+refrence+to+the+phage-host+system+encountered.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001946-Kemp1" id="pone.0001946-Kemp1"></a>Kemp PF, Aller JY (2004) Bacterial diversity in aquatic and other environments: what 16S rDNA libraries can tell us. FEMS Microbiology Ecology  47: 161–177.  <ul class="find" data-citedArticleID="1087432" data-doi="10.1016/s0168-6496(03)00257-5"><li><a href="http://dx.doi.org/10.1016/s0168-6496(03)00257-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+diversity+in+aquatic+and+other+environments%3A+what+16S+rDNA+libraries+can+tell+us." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+diversity+in+aquatic+and+other+environments%3A+what+16S+rDNA+libraries+can+tell+us.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001946-Wright1" id="pone.0001946-Wright1"></a>Wright DH (1983) Species-energy theory - an exptension of species-area theory. Oikos  41: 496–506.  <ul class="find" data-citedArticleID="1087492" data-doi="10.2307/3544109"><li><a href="http://dx.doi.org/10.2307/3544109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Species-energy+theory+-+an+exptension+of+species-area+theory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Species-energy+theory+-+an+exptension+of+species-area+theory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001946-Stent1" id="pone.0001946-Stent1"></a>Stent GS (1963) Molecular biology of bacterial viruses. San Francisco: W. H. Freeman.   <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0001946-Charnov2" id="pone.0001946-Charnov2"></a>Charnov EL (1973) Optimal foraging: some theoretical explorations [Ph.D.]. Seattle: University of Washington.   <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0001946-Pyke1" id="pone.0001946-Pyke1"></a>Pyke GH, Pulliam HR, Charnov EL (1977) Optimal foraging: a selective review of theory and tests. The Quarterly Review of Biology  52: 137–154.  <ul class="find" data-citedArticleID="1087452" data-doi="10.1086/409852"><li><a href="http://dx.doi.org/10.1086/409852" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimal+foraging%3A+a+selective+review+of+theory+and+tests." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimal+foraging%3A+a+selective+review+of+theory+and+tests.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001946-Bull1" id="pone.0001946-Bull1"></a>Bull JJ, Badgett MR, Wichman HA, Huelsenbeck JP, Hillis DM, et al.  (1997) Exceptional convergent evolution in a virus. Genetics  147: 1497–1507.  <ul class="find" data-citedArticleID="1087414"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Exceptional+convergent+evolution+in+a+virus.&amp;auth=&amp;atitle=Exceptional+convergent+evolution+in+a+virus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exceptional+convergent+evolution+in+a+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exceptional+convergent+evolution+in+a+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001946-Holder1" id="pone.0001946-Holder1"></a>Holder KK, Bull JJ (2001) Profiles of adaptation in two similar viruses. Genetics  159: 1393–1404.  <ul class="find" data-citedArticleID="1087428"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Profiles+of+adaptation+in+two+similar+viruses.&amp;auth=&amp;atitle=Profiles+of+adaptation+in+two+similar+viruses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Profiles+of+adaptation+in+two+similar+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Profiles+of+adaptation+in+two+similar+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001946-Tessman1" id="pone.0001946-Tessman1"></a>Tessman I (1968) Mutagenic treatment of double- and single- stranded DNA phages T4 and S13 with hydroxylamine. Virology  35: 330–&amp;.  <ul class="find" data-citedArticleID="1087464" data-doi="10.1016/0042-6822(68)90275-4"><li><a href="http://dx.doi.org/10.1016/0042-6822(68)90275-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutagenic+treatment+of+double-+and+single-+stranded+DNA+phages+T4+and+S13+with+hydroxylamine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutagenic+treatment+of+double-+and+single-+stranded+DNA+phages+T4+and+S13+with+hydroxylamine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001946-Bergh1" id="pone.0001946-Bergh1"></a>Bergh O, Borsheim KY, Bratbak G, Heldal M (1989) High abundance of viruses found in aquatic environments. Nature  340: 467–468.  <ul class="find" data-citedArticleID="1087410" data-doi="10.1038/340467a0"><li><a href="http://dx.doi.org/10.1038/340467a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+abundance+of+viruses+found+in+aquatic+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+abundance+of+viruses+found+in+aquatic+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001946-Garen1" id="pone.0001946-Garen1"></a>Garen A, Puck TT (1951) The first two steps of the invasion of host cells by bacterial viruses. II. Journal of Experimental Medicine  94: 177–189.  <ul class="find" data-citedArticleID="1087424" data-doi="10.1084/jem.94.3.177"><li><a href="http://dx.doi.org/10.1084/jem.94.3.177" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+first+two+steps+of+the+invasion+of+host+cells+by+bacterial+viruses.+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+first+two+steps+of+the+invasion+of+host+cells+by+bacterial+viruses.+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001946-Lenski1" id="pone.0001946-Lenski1"></a>Lenski RE, Levin BR (1985) Constraints on the coevolution of bacteria and virulent phage - A oodel, some experiments, and predictions for natural communities. American Naturalist  125: 585–602.  <ul class="find" data-citedArticleID="1087436" data-doi="10.1086/284364"><li><a href="http://dx.doi.org/10.1086/284364" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Constraints+on+the+coevolution+of+bacteria+and+virulent+phage+-+A+oodel%2C+some+experiments%2C+and+predictions+for+natural+communities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Constraints+on+the+coevolution+of+bacteria+and+virulent+phage+-+A+oodel%2C+some+experiments%2C+and+predictions+for+natural+communities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001946-Suttle1" id="pone.0001946-Suttle1"></a>Suttle CA, Chan AM (1993) Marine cyanophages infecting oceanic and coastal strains of Synechococcus - Abundance, morphology, cross-infectivity and growth characteristics. Marine Ecology-Progress Series  92: 99–109.  <ul class="find" data-citedArticleID="1087462" data-doi="10.3354/meps092099"><li><a href="http://dx.doi.org/10.3354/meps092099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Marine+cyanophages+infecting+oceanic+and+coastal+strains+of+Synechococcus+-+Abundance%2C+morphology%2C+cross-infectivity+and+growth+characteristics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Marine+cyanophages+infecting+oceanic+and+coastal+strains+of+Synechococcus+-+Abundance%2C+morphology%2C+cross-infectivity+and+growth+characteristics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001946-Wilson1" id="pone.0001946-Wilson1"></a>Wilson WH, Carr NG, Mann NH (1996) The effect of phosphate status on the kinetics of cyanophage infection in the oceanic cyanobacterium Synechococcus sp WH7803. Journal of Phycology  32: 506–516.  <ul class="find" data-citedArticleID="1087486" data-doi="10.1111/j.0022-3646.1996.00506.x"><li><a href="http://dx.doi.org/10.1111/j.0022-3646.1996.00506.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+phosphate+status+on+the+kinetics+of+cyanophage+infection+in+the+oceanic+cyanobacterium+Synechococcus+sp+WH7803." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+phosphate+status+on+the+kinetics+of+cyanophage+infection+in+the+oceanic+cyanobacterium+Synechococcus+sp+WH7803.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001946-Wells1" id="pone.0001946-Wells1"></a>Wells L, Deming J (2006) Characterization of a cold-active bacteriophage on two psychrophilic marine hosts. Aquatic Microbial Ecology  45: 15–29.  <ul class="find" data-citedArticleID="1087474" data-doi="10.3354/ame045015"><li><a href="http://dx.doi.org/10.3354/ame045015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+cold-active+bacteriophage+on+two+psychrophilic+marine+hosts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+cold-active+bacteriophage+on+two+psychrophilic+marine+hosts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001946-Chen1" id="pone.0001946-Chen1"></a>Chen F, Lu JR (2002) Genomic sequence and evolution of marine cyanophage P60: a new insight on lytic and lysogenic phages. Applied and Environmental Microbiology  68: 2589–2594.  <ul class="find" data-citedArticleID="1087420" data-doi="10.1128/aem.68.5.2589-2594.2002"><li><a href="http://dx.doi.org/10.1128/aem.68.5.2589-2594.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+sequence+and+evolution+of+marine+cyanophage+P60%3A+a+new+insight+on+lytic+and+lysogenic+phages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+sequence+and+evolution+of+marine+cyanophage+P60%3A+a+new+insight+on+lytic+and+lysogenic+phages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001946-Yoshida1" id="pone.0001946-Yoshida1"></a>Yoshida T, Takashima Y, Tomaru Y, Shirai Y, Takao Y, et al.  (2006) Isolation and characterization of a cyanophage infecting the toxic cyanobacterium Microcystis aeruginosa. Applied and Environmental Microbiology  72: 1239–1247.  <ul class="find" data-citedArticleID="1087494" data-doi="10.1128/aem.72.2.1239-1247.2006"><li><a href="http://dx.doi.org/10.1128/aem.72.2.1239-1247.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+characterization+of+a+cyanophage+infecting+the+toxic+cyanobacterium+Microcystis+aeruginosa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+characterization+of+a+cyanophage+infecting+the+toxic+cyanobacterium+Microcystis+aeruginosa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001946-Jiang1" id="pone.0001946-Jiang1"></a>Jiang SC, Kellogg CA, Paul JH (1998) Characterization of marine temperate phage-host systems isolated from Mamala Bay, Oahu, Hawaii. Applied and Environmental Microbiology  64: 535–542.  <ul class="find" data-citedArticleID="1087430"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+marine+temperate+phage-host+systems+isolated+from+Mamala+Bay%2C+Oahu%2C+Hawaii.&amp;auth=&amp;atitle=Characterization+of+marine+temperate+phage-host+systems+isolated+from+Mamala+Bay%2C+Oahu%2C+Hawaii." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+marine+temperate+phage-host+systems+isolated+from+Mamala+Bay%2C+Oahu%2C+Hawaii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+marine+temperate+phage-host+systems+isolated+from+Mamala+Bay%2C+Oahu%2C+Hawaii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001946-Wikner1" id="pone.0001946-Wikner1"></a>Wikner J, Vallino JJ, Steward GF, Smith DC, Azam F (1993) Nucleic acids from the host bacterium as a major source of nucleotides for 3 marine bacteriophages. FEMS Microbiology Ecology  12: 237–248.  <ul class="find" data-citedArticleID="1087482" data-doi="10.1111/j.1574-6941.1993.tb00036.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6941.1993.tb00036.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleic+acids+from+the+host+bacterium+as+a+major+source+of+nucleotides+for+3+marine+bacteriophages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleic+acids+from+the+host+bacterium+as+a+major+source+of+nucleotides+for+3+marine+bacteriophages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001946-Middelboe1" id="pone.0001946-Middelboe1"></a>Middelboe M, Hagstrom A, Blackburn N, Sinn B, Fischer U, et al.  (2001) Effects of bacteriophages on the population dynamics of four strains of pelagic marine bacteria. Microbial Ecology  42: 395–406.  <ul class="find" data-citedArticleID="1087442" data-doi="10.1007/s00248-001-0012-1"><li><a href="http://dx.doi.org/10.1007/s00248-001-0012-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+bacteriophages+on+the+population+dynamics+of+four+strains+of+pelagic+marine+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+bacteriophages+on+the+population+dynamics+of+four+strains+of+pelagic+marine+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001946-Wolf1" id="pone.0001946-Wolf1"></a>Wolf A, Zheng TL, Witzel KP, Jost G (2004) Impact of initial phage/host ratio and nutrient addition on coexistence in a phage-host system. Aquatic Microbial Ecology  35: 131–139.  <ul class="find" data-citedArticleID="1087488" data-doi="10.3354/ame035131"><li><a href="http://dx.doi.org/10.3354/ame035131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impact+of+initial+phage%2Fhost+ratio+and+nutrient+addition+on+coexistence+in+a+phage-host+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impact+of+initial+phage%2Fhost+ratio+and+nutrient+addition+on+coexistence+in+a+phage-host+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001946-Sullivan1" id="pone.0001946-Sullivan1"></a>Sullivan MB, Waterbury JB, Chisholm SW (2003) Cyanophages infecting the oceanic cyanobacterium Prochlorococcus. Nature  424: 1047–1051.  <ul class="find" data-citedArticleID="1087460" data-doi="10.1038/nature01929"><li><a href="http://dx.doi.org/10.1038/nature01929" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cyanophages+infecting+the+oceanic+cyanobacterium+Prochlorococcus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cyanophages+infecting+the+oceanic+cyanobacterium+Prochlorococcus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001946-Stewart1" id="pone.0001946-Stewart1"></a>Stewart FM, Levin BR (1984) The Population Biology of Bacterial-Viruses - Why Be Temperate. Theoretical Population Biology  26: 93–117.  <ul class="find" data-citedArticleID="1087458" data-doi="10.1016/0040-5809(84)90026-1"><li><a href="http://dx.doi.org/10.1016/0040-5809(84)90026-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Population+Biology+of+Bacterial-Viruses+-+Why+Be+Temperate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Population+Biology+of+Bacterial-Viruses+-+Why+Be+Temperate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001946-Ripp1" id="pone.0001946-Ripp1"></a>Ripp S, Miller RV (1998) Dynamics of the pseudolysogenic response in slowly growing cells of Pseudomonas aeruginosa. Microbiology  144: 2225–2232.  <ul class="find" data-citedArticleID="1087454" data-doi="10.1099/00221287-144-8-2225"><li><a href="http://dx.doi.org/10.1099/00221287-144-8-2225" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamics+of+the+pseudolysogenic+response+in+slowly+growing+cells+of+Pseudomonas+aeruginosa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamics+of+the+pseudolysogenic+response+in+slowly+growing+cells+of+Pseudomonas+aeruginosa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001946-Weinbauer1" id="pone.0001946-Weinbauer1"></a>Weinbauer MG (2004) Ecology of prokaryotic viruses. FEMS Microbiology Reviews  28: 127–181.  <ul class="find" data-citedArticleID="1087472" data-doi="10.1016/j.femsre.2003.08.001"><li><a href="http://dx.doi.org/10.1016/j.femsre.2003.08.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecology+of+prokaryotic+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecology+of+prokaryotic+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001946-McDaniel1" id="pone.0001946-McDaniel1"></a>McDaniel L, Houchin LA, Williamson SJ, Paul JH (2002) Plankton blooms - Lysogeny in marine Synechococcus. Nature  415: 496–496.  <ul class="find" data-citedArticleID="1087438" data-doi="10.1038/415496a"><li><a href="http://dx.doi.org/10.1038/415496a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plankton+blooms+-+Lysogeny+in+marine+Synechococcus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plankton+blooms+-+Lysogeny+in+marine+Synechococcus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001946-Williamson1" id="pone.0001946-Williamson1"></a>Williamson SJ, Houchin LA, McDaniel L, Paul JH (2002) Seasonal variation in lysogeny as depicted by prophage induction in Tampa Bay, Florida. Applied and Environmental Microbiology  68: 4307–4314.  <ul class="find" data-citedArticleID="1087484" data-doi="10.1128/aem.68.9.4307-4314.2002"><li><a href="http://dx.doi.org/10.1128/aem.68.9.4307-4314.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Seasonal+variation+in+lysogeny+as+depicted+by+prophage+induction+in+Tampa+Bay%2C+Florida." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Seasonal+variation+in+lysogeny+as+depicted+by+prophage+induction+in+Tampa+Bay%2C+Florida.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001946-Abedon2" id="pone.0001946-Abedon2"></a>Abedon ST (1989) Selection for bacteriophage latent period length by bacterial density - a theretical examination. Microbial Ecology  18: 79–88.  <ul class="find" data-citedArticleID="1087408" data-doi="10.1007/bf02030117"><li><a href="http://dx.doi.org/10.1007/bf02030117" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+for+bacteriophage+latent+period+length+by+bacterial+density+-+a+theretical+examination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+for+bacteriophage+latent+period+length+by+bacterial+density+-+a+theretical+examination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001946-Wang1" id="pone.0001946-Wang1"></a>Wang I-N, Dykhuizen DE, Slobodkin LB (1996) The evolution of phage lysis timing. Evolutionary Ecology  10: 545–558.  <ul class="find" data-citedArticleID="1087470" data-doi="10.1007/bf01237884"><li><a href="http://dx.doi.org/10.1007/bf01237884" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+phage+lysis+timing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+phage+lysis+timing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.XML" value="92094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.PDF" value="213035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e001.PNG" value="7134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e001.TIF" value="23040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e002.PNG" value="6077"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e002.TIF" value="22092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e003.PNG" value="7009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e003.TIF" value="22932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e004.PNG" value="8393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e004.TIF" value="24276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e005.PNG" value="5397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e005.TIF" value="21328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e006.PNG" value="4773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e006.TIF" value="20700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e007.PNG" value="4890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e007.TIF" value="20836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e008.PNG" value="5346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e008.TIF" value="21304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e009.PNG" value="5397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e009.TIF" value="21340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e010.PNG" value="8094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e010.TIF" value="24004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e011.PNG" value="7600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e011.TIF" value="23548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g001.PNG_L" value="23827"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g001.PNG_M" value="22954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g001.PNG_S" value="6090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g001.TIF" value="58972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g001.PNG_I" value="8338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.t001.PNG_L" value="54521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.t001.PNG_M" value="74166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.t001.PNG_S" value="12907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.t001.TIF" value="261456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.t001.PNG_I" value="24750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g002.PNG_L" value="26834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g002.PNG_M" value="36659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g002.PNG_S" value="7804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g002.TIF" value="55252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g002.PNG_I" value="13396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.t002.PNG_L" value="88004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.t002.PNG_M" value="116232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.t002.PNG_S" value="17191"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.t002.TIF" value="417182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.t002.PNG_I" value="39986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g003.PNG_L" value="23860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g003.PNG_M" value="27762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g003.PNG_S" value="6430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g003.TIF" value="51102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g003.PNG_I" value="10382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g004.PNG_L" value="16385"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g004.PNG_M" value="30269"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g004.PNG_S" value="5620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g004.TIF" value="84160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g004.PNG_I" value="6993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e012.PNG" value="7648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.e012.TIF" value="23644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g005.PNG_L" value="25656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g005.PNG_M" value="23030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g005.PNG_S" value="5819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g005.TIF" value="69732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001946.g005.PNG_I" value="9151"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001946&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001946" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001946&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001946&volume=&issue=&title=Optimal Foraging Predicts the Ecology but Not the Evolution of Host Specialization in Bacteriophages&author_name=S%C3%A9bastien%20Guyader%2C%20Christina%20L.%20Burch&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001946" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001946" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001946" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001946&amp;t=Optimal%20Foraging%20Predicts%20the%20Ecology%20but%20Not%20the%20Evolution%20of%20Host%20Specialization%20in%20Bacteriophages" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001946&title=Optimal%20Foraging%20Predicts%20the%20Ecology%20but%20Not%20the%20Evolution%20of%20Host%20Specialization%20in%20Bacteriophages&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001946&amp;title=Optimal%20Foraging%20Predicts%20the%20Ecology%20but%20Not%20the%20Evolution%20of%20Host%20Specialization%20in%20Bacteriophages" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001946" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001946&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Optimal Foraging Predicts the Ecology but Not the Evolution of Host Specialization in Bacteriophages';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001946';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Optimal%20Foraging%20Predicts%20the%20Ecology%20but%20Not%20the%20Evolution%20of%20Host%20Specialization%20in%20Bacteriophages http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001946" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001946" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacteriophages%22" title="Search for articles in the subject area:'Bacteriophages'"><div class="flagText">Bacteriophages</div></a>
              <div data-categoryid="43083" data-articleid="26716"
                   data-categoryname="Bacteriophages"
                   class="flagImage" title="Flag 'Bacteriophages' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+ecology%22" title="Search for articles in the subject area:'Evolutionary ecology'"><div class="flagText">Evolutionary ecology</div></a>
              <div data-categoryid="48159" data-articleid="26716"
                   data-categoryname="Evolutionary ecology"
                   class="flagImage" title="Flag 'Evolutionary ecology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Foraging%22" title="Search for articles in the subject area:'Foraging'"><div class="flagText">Foraging</div></a>
              <div data-categoryid="17309" data-articleid="26716"
                   data-categoryname="Foraging"
                   class="flagImage" title="Flag 'Foraging' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Host-pathogen+interactions%22" title="Search for articles in the subject area:'Host-pathogen interactions'"><div class="flagText">Host-pathogen interactions</div></a>
              <div data-categoryid="19119" data-articleid="26716"
                   data-categoryname="Host-pathogen interactions"
                   class="flagImage" title="Flag 'Host-pathogen interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lysis+%28medicine%29%22" title="Search for articles in the subject area:'Lysis (medicine)'"><div class="flagText">Lysis (medicine)</div></a>
              <div data-categoryid="47609" data-articleid="26716"
                   data-categoryname="Lysis (medicine)"
                   class="flagImage" title="Flag 'Lysis (medicine)' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+density%22" title="Search for articles in the subject area:'Population density'"><div class="flagText">Population density</div></a>
              <div data-categoryid="43529" data-articleid="26716"
                   data-categoryname="Population density"
                   class="flagImage" title="Flag 'Population density' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Salmonella+typhimurium%22" title="Search for articles in the subject area:'Salmonella typhimurium'"><div class="flagText">Salmonella typhimurium</div></a>
              <div data-categoryid="51943" data-articleid="26716"
                   data-categoryname="Salmonella typhimurium"
                   class="flagImage" title="Flag 'Salmonella typhimurium' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Trophic+interactions%22" title="Search for articles in the subject area:'Trophic interactions'"><div class="flagText">Trophic interactions</div></a>
              <div data-categoryid="47297" data-articleid="26716"
                   data-categoryname="Trophic interactions"
                   class="flagImage" title="Flag 'Trophic interactions' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6682'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3242'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5216&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=13443">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
