

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genomic Diversity among Beijing and non-Beijing Mycobacterium tuberculosis Isolates from Myanmar</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001973"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001973" />

    <meta name="citation_title" content="Genomic Diversity among Beijing and non-Beijing Mycobacterium tuberculosis Isolates from Myanmar"/>
    <meta itemprop="name" content="Genomic Diversity among Beijing and non-Beijing Mycobacterium tuberculosis Isolates from Myanmar"/>

      <meta name="citation_author" content="Ruth Stavrum"/>
            <meta name="citation_author_institution" content="Section for Microbiology and Immunology, The Gade Institute, University of Bergen, Bergen, Norway"/>
            <meta name="citation_author_institution" content="Department of Microbiology and Immunology, Haukeland University Hospital, Bergen, Norway"/>
      <meta name="citation_author" content="Håvard Valvatne"/>
            <meta name="citation_author_institution" content="Section for Microbiology and Immunology, The Gade Institute, University of Bergen, Bergen, Norway"/>
            <meta name="citation_author_institution" content="Department of Microbiology and Immunology, Haukeland University Hospital, Bergen, Norway"/>
      <meta name="citation_author" content="Trond H. Bø"/>
            <meta name="citation_author_institution" content="Bergen Centre for Computational Science, Department of Informatics, University of Bergen, Bergen, Norway"/>
      <meta name="citation_author" content="Inge Jonassen"/>
            <meta name="citation_author_institution" content="Bergen Centre for Computational Science, Department of Informatics, University of Bergen, Bergen, Norway"/>
      <meta name="citation_author" content="Jason Hinds"/>
            <meta name="citation_author_institution" content="Bacterial Microarray Group, Division of Cellular and Molecular Medicine, St. George's, University of London, London, United Kingdom"/>
      <meta name="citation_author" content="Philip D. Butcher"/>
            <meta name="citation_author_institution" content="Bacterial Microarray Group, Division of Cellular and Molecular Medicine, St. George's, University of London, London, United Kingdom"/>
      <meta name="citation_author" content="Harleen M. S. Grewal"/>
            <meta name="citation_author_institution" content="Section for Microbiology and Immunology, The Gade Institute, University of Bergen, Bergen, Norway"/>
            <meta name="citation_author_institution" content="Department of Microbiology and Immunology, Haukeland University Hospital, Bergen, Norway"/>

    <meta name="citation_date" content="2008/4/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001973.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1973"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The Mycobacterium tuberculosis 19-kilodalton lipoprotein inhibits gamma interferon-regulated HLA-DR and Fc gamma R1 on human macrophages through Toll-like receptor 2.; citation_author=AJ Gehring; citation_author=RE Rojas; citation_author=DH Canaday; citation_author=DL Lakey; citation_author=CV Harding; citation_journal_title=Infect Immun; citation_volume=71; citation_number=1; citation_pages=4487-4497; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Tuberculosis: latency and reactivation.; citation_author=JL Flynn; citation_author=J Chan; citation_journal_title=Infect Immun; citation_volume=69; citation_number=2; citation_pages=4195-4201; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Effects of genetic variability of Mycobacterium tuberculosis strains on the presentation of disease.; citation_author=AN Malik; citation_author=P Godfrey-Faussett; citation_journal_title=Lancet Infect Dis; citation_volume=5; citation_number=3; citation_pages=174-183; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A marked difference in pathogenesis and immune response induced by different Mycobacterium tuberculosis genotypes.; citation_author=B Lopez; citation_author=D Aguilar; citation_author=H Orozco; citation_author=M Burger; citation_author=C Espitia; citation_journal_title=Clin Exp Immunol; citation_volume=133; citation_number=4; citation_pages=30-37; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A glycolipid of hypervirulent tuberculosis strains that inhibits the innate immune response.; citation_author=MB Reed; citation_author=P Domenech; citation_author=C Manca; citation_author=H Su; citation_author=AK Barczak; citation_journal_title=Nature; citation_volume=431; citation_number=5; citation_pages=84-87; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Negligible genetic diversity of mycobacterium tuberculosis host immune system protein targets: evidence of limited selective pressure.; citation_author=JM Musser; citation_author=A Amin; citation_author=S Ramaswamy; citation_journal_title=Genetics; citation_volume=155; citation_number=6; citation_pages=7-16; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genomic deletions suggest a phylogeny for the Mycobacterium tuberculosis complex.; citation_author=S Mostowy; citation_author=D Cousins; citation_author=J Brinkman; citation_author=A Aranaz; citation_author=MA Behr; citation_journal_title=J Infect Dis; citation_volume=186; citation_number=7; citation_pages=74-80; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Whole-genome comparison of Mycobacterium tuberculosis clinical and laboratory strains.; citation_author=RD Fleischmann; citation_author=D Alland; citation_author=JA Eisen; citation_author=L Carpenter; citation_author=O White; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=8; citation_pages=5479-5490; citation_date=2002; " />
      <meta name="citation_reference" content="citation_number=9; " />
      <meta name="citation_reference" content="citation_title=Worldwide occurrence of Beijing/W strains of Mycobacterium tuberculosis: a systematic review.; citation_author=JR Glynn; citation_author=J Whiteley; citation_author=PJ Bifani; citation_author=K Kremer; citation_author=D van Soolingen; citation_journal_title=Emerg Infect Dis; citation_volume=8; citation_number=10; citation_pages=843-849; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Heterogeneity of Mycobacterium tuberculosis isolates in Yangon, Myanmar.; citation_author=S Phyu; citation_author=R Jureen; citation_author=T Ti; citation_author=UR Dahle; citation_author=HM Grewal; citation_journal_title=J Clin Microbiol; citation_volume=41; citation_number=11; citation_pages=4907-4908; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mycobacterium tuberculosis Beijing genotype.; citation_author=T Lillebaek; citation_author=AB Andersen; citation_author=A Dirksen; citation_author=JR Glynn; citation_author=K Kremer; citation_journal_title=Emerg Infect Dis; citation_volume=9; citation_number=12; citation_pages=1553-1557; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Global dissemination of the Mycobacterium tuberculosis W-Beijing family strains.; citation_author=PJ Bifani; citation_author=B Mathema; citation_author=NE Kurepina; citation_author=BN Kreiswirth; citation_journal_title=Trends Microbiol; citation_volume=10; citation_number=13; citation_pages=45-52; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Definition of the Beijing/W lineage of Mycobacterium tuberculosis on the basis of genetic markers.; citation_author=K Kremer; citation_author=JR Glynn; citation_author=T Lillebaek; citation_author=S Niemann; citation_author=NE Kurepina; citation_journal_title=J Clin Microbiol; citation_volume=42; citation_number=14; citation_pages=4040-4049; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genomic deletions classify the Beijing/W strains as a distinct genetic lineage of Mycobacterium tuberculosis.; citation_author=AG Tsolaki; citation_author=S Gagneux; citation_author=AS Pym; citation_author=YO Goguet de la Salmoniere; citation_author=BN Kreiswirth; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=15; citation_pages=3185-3191; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Drug-resistant tuberculosis in Yangon, Myanmar.; citation_author=S Phyu; citation_author=T Lwin; citation_author=T Ti; citation_author=W Maung; citation_author=WW Mar; citation_journal_title=Scand J Infect Dis; citation_volume=37; citation_number=16; citation_pages=846-851; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular techniques: applications in epidemiologic studies. Mycobacterium bovis infection in animals and humans:; citation_author=JDA van Embden; citation_author=LM Schouls; citation_author=D van Soolingen; citation_number=17; citation_pages=15-27; citation_date=1995; citation_publisher=Iowa State University Press; " />
      <meta name="citation_reference" content="citation_title=Re-annotation of the genome sequence of Mycobacterium tuberculosis H37Rv.; citation_author=JC Camus; citation_author=MJ Pryor; citation_author=C Medigue; citation_author=ST Cole; citation_journal_title=Microbiology; citation_volume=148; citation_number=18; citation_pages=2967-2973; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Dissection of the heat-shock response in Mycobacterium tuberculosis using mutants and microarrays.; citation_author=GR Stewart; citation_author=L Wernisch; citation_author=R Stabler; citation_author=JA Mangan; citation_author=J Hinds; citation_journal_title=Microbiology; citation_volume=148; citation_number=19; citation_pages=3129-3138; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Glass slide microarrays for bacterial genomes.; citation_author=JLKG Hinds; citation_author=JA Mangan; citation_author=PD Butcher; citation_number=20; citation_pages=83-99; citation_date=2002; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=A comparison of normalization methods for high density oligonucleotide array data based on variance and bias.; citation_author=BM Bolstad; citation_author=RA Irizarry; citation_author=M Astrand; citation_author=TP Speed; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=21; citation_pages=185-193; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Variance stabilization applied to microarray data calibration and to the quantification of differential expression.; citation_author=W Huber; citation_author=A von Heydebreck; citation_author=H Sultmann; citation_author=A Poustka; citation_author=M Vingron; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=22; citation_pages=S96-104; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Approximate variance-stabilizing transformations for gene-expression microarray data.; citation_author=DM Rocke; citation_author=B Durbin; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=23; citation_pages=966-972; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Patients with active tuberculosis often have different strains in the same sputum specimen.; citation_author=RM Warren; citation_author=TC Victor; citation_author=EM Streicher; citation_author=M Richardson; citation_author=N Beyers; citation_journal_title=Am J Respir Crit Care Med; citation_volume=169; citation_number=24; citation_pages=610-614; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Proposal for standardization of optimized mycobacterial interspersed repetitive unit-variable-number tandem repeat typing of Mycobacterium tuberculosis.; citation_author=P Supply; citation_author=C Allix; citation_author=S Lesjean; citation_author=M Cardoso-Oelemann; citation_author=S Rusch-Gerdes; citation_journal_title=J Clin Microbiol; citation_volume=44; citation_number=25; citation_pages=4498-4510; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Role of the pks15/1 gene in the biosynthesis of phenolglycolipids in the Mycobacterium tuberculosis complex. Evidence that all strains synthesize glycosylated p-hydroxybenzoic methyl esters and that strains devoid of phenolglycolipids harbor a frameshift mutation in the pks15/1 gene.; citation_author=P Constant; citation_author=E Perez; citation_author=W Malaga; citation_author=MA Laneelle; citation_author=O Saurel; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=26; citation_pages=38148-38158; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Seventy percent of the Mycobacterium tuberculosis isolates in Hong Kong represent the Beijing genotype.; citation_author=MY Chan; citation_author=M Borgdorff; citation_author=CW Yip; citation_author=PE de Haas; citation_author=WS Wong; citation_journal_title=Epidemiol Infect; citation_volume=127; citation_number=27; citation_pages=169-171; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Predominance of a single genotype of Mycobacterium tuberculosis in countries of east Asia.; citation_author=D van Soolingen; citation_author=L Qian; citation_author=PE de Haas; citation_author=JT Douglas; citation_author=H Traore; citation_journal_title=J Clin Microbiol; citation_volume=33; citation_number=28; citation_pages=3234-3238; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Global distribution of Mycobacterium tuberculosis spoligotypes.; citation_author=I Filliol; citation_author=JR Driscoll; citation_author=D Van Soolingen; citation_author=BN Kreiswirth; citation_author=K Kremer; citation_journal_title=Emerg Infect Dis; citation_volume=8; citation_number=29; citation_pages=1347-1349; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The PE-PGRS glycine-rich proteins of Mycobacterium tuberculosis: a new family of fibronectin-binding proteins?; citation_author=C Espitia; citation_author=JP Laclette; citation_author=M Mondragon-Palomino; citation_author=A Amador; citation_author=J Campuzano; citation_journal_title=Microbiology; citation_volume=145 ( Pt 12); citation_number=30; citation_pages=3487-3495; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Deciphering the biology of Mycobacterium tuberculosis from the complete genome sequence.; citation_author=ST Cole; citation_author=R Brosch; citation_author=J Parkhill; citation_author=T Garnier; citation_author=C Churcher; citation_journal_title=Nature; citation_volume=393; citation_number=31; citation_pages=537-544; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The complete genome sequence of Mycobacterium bovis.; citation_author=T Garnier; citation_author=K Eiglmeier; citation_author=JC Camus; citation_author=N Medina; citation_author=H Mansoor; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=32; citation_pages=7877-7882; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Variable host-pathogen compatibility in Mycobacterium tuberculosis.; citation_author=S Gagneux; citation_author=K DeRiemer; citation_author=T Van; citation_author=M Kato-Maeda; citation_author=BC de Jong; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=33; citation_pages=2869-2873; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Predominance of ancestral lineages of Mycobacterium tuberculosis in India.; citation_author=MC Gutierrez; citation_author=N Ahmed; citation_author=E Willery; citation_author=S Narayanan; citation_author=SE Hasnain; citation_journal_title=Emerg Infect Dis; citation_volume=12; citation_number=34; citation_pages=1367-1374; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Signature gene expression profiles discriminate between isoniazid-, thiolactomycin-, and triclosan-treated Mycobacterium tuberculosis.; citation_author=JC Betts; citation_author=A McLaren; citation_author=MG Lennon; citation_author=FM Kelly; citation_author=PT Lukey; citation_journal_title=Antimicrob Agents Chemother; citation_volume=47; citation_number=35; citation_pages=2903-2913; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A new evolutionary scenario for the Mycobacterium tuberculosis complex.; citation_author=R Brosch; citation_author=SV Gordon; citation_author=M Marmiesse; citation_author=P Brodin; citation_author=C Buchrieser; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=36; citation_pages=3684-3689; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Analysis of genomic downsizing on the basis of region-of-difference polymorphism profiling of Mycobacterium tuberculosis patient isolates reveals geographic partitioning.; citation_author=KR Rao; citation_author=F Kauser; citation_author=S Srinivas; citation_author=S Zanetti; citation_author=LA Sechi; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=37; citation_pages=5978-5982; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001973" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genomic Diversity among Beijing and non-Beijing Mycobacterium tuberculosis Isolates from Myanmar"/>
    <meta name="twitter:description" content="BackgroundThe Beijing family of Mycobacterium tuberculosis is dominant in countries in East Asia. Genomic polymorphisms are a source of diversity within the M. tuberculosis genome and may account for the variation of virulence among M. tuberculosis isolates. Till date there are no studies that have examined the genomic composition of M. tuberculosis isolates from the high TB-burden country, Myanmar.Methodology/Principle FindingsTwenty-two M. tuberculosis isolates from Myanmar were screened on whole-genome arrays containing genes from M. tuberculosis H37Rv, M. tuberculosis CDC1551 and M. bovis AF22197. Screening identified 198 deletions or extra regions in the clinical isolates compared to H37Rv. Twenty-two regions differentiated between Beijing and non-Beijing isolates and were verified by PCR on an additional 40 isolates. Six regions (Rv0071-0074 [RD105], Rv1572-1576c [RD149], Rv1585c-1587c [RD149], MT1798-Rv1755c [RD152], Rv1761c [RD152] and Rv0279c) were deleted in Beijing isolates, of which 4 (Rv1572-1576c, Rv1585c-1587c, MT1798-Rv1755c and Rv1761c) were variably deleted among ST42 isolates, indicating a closer relationship between the Beijing and ST42 lineages. The TbD1 region, Mb1582-Mb1583 was deleted in Beijing and ST42 isolates. One M. bovis gene of unknown function, Mb3184c was present in all isolates, except 11 of 13 ST42 isolates. The CDC1551 gene, MT1360 coding for a putative adenylate cyclase, was present in all Beijing and ST42 isolates (except 1). The pks15/1 gene, coding for a putative virulence factor, was intact in all Beijing and non-Beijing isolates, except in ST42 and ST53 isolates.ConclusionThis study describes previously unreported deletions/extra regions in Beijing and non-Beijing M. tuberculosis isolates. The modern and highly frequent ST42 lineage showed a closer relationship to the hypervirulent Beijing lineage than to the ancient non-Beijing lineages. The pks15/1 gene was disrupted only in modern non-Beijing isolates. This is the first report of an in-depth analysis on the genomic diversity of M. tuberculosis isolates from Myanmar."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001973.g004"/>

  <meta property="og:title" content="Genomic Diversity among Beijing and non-Beijing Mycobacterium tuberculosis Isolates from Myanmar" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001973" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8317'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1750'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6184&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001973">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001973" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001973&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genomic Diversity among Beijing and non-Beijing <i>Mycobacterium tuberculosis</i> Isolates from Myanmar
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ruth Stavrum, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Section for Microbiology and Immunology, The Gade Institute, University of Bergen, Bergen, Norway, 
                  Department of Microbiology and Immunology, Haukeland University Hospital, Bergen, Norway
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Håvard Valvatne, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Section for Microbiology and Immunology, The Gade Institute, University of Bergen, Bergen, Norway, 
                  Department of Microbiology and Immunology, Haukeland University Hospital, Bergen, Norway
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Trond H. Bø, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bergen Centre for Computational Science, Department of Informatics, University of Bergen, Bergen, Norway
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Inge Jonassen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bergen Centre for Computational Science, Department of Informatics, University of Bergen, Bergen, Norway
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jason Hinds, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bacterial Microarray Group, Division of Cellular and Molecular Medicine, St. George's, University of London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Philip D. Butcher, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bacterial Microarray Group, Division of Cellular and Molecular Medicine, St. George's, University of London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Harleen M. S. Grewal
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Harleen.Grewal@Gades.uib.no">Harleen.Grewal@Gades.uib.no</a></p>

                <p>Affiliations:
                  Section for Microbiology and Immunology, The Gade Institute, University of Bergen, Bergen, Norway, 
                  Department of Microbiology and Immunology, Haukeland University Hospital, Bergen, Norway
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 09, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001973</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001973" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001973" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001973" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001973" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001973-t001" data-doi="info:doi/10.1371/journal.pone.0001973" data-uri="info:doi/10.1371/journal.pone.0001973.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001973-g001" data-doi="info:doi/10.1371/journal.pone.0001973" data-uri="info:doi/10.1371/journal.pone.0001973.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001973-g002" data-doi="info:doi/10.1371/journal.pone.0001973" data-uri="info:doi/10.1371/journal.pone.0001973.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001973-g003" data-doi="info:doi/10.1371/journal.pone.0001973" data-uri="info:doi/10.1371/journal.pone.0001973.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001973-g004" data-doi="info:doi/10.1371/journal.pone.0001973" data-uri="info:doi/10.1371/journal.pone.0001973.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001973">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001973" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The Beijing family of <em>Mycobacterium tuberculosis</em> is dominant in countries in East Asia. Genomic polymorphisms are a source of diversity within the <em>M. tuberculosis</em> genome and may account for the variation of virulence among <em>M. tuberculosis</em> isolates. Till date there are no studies that have examined the genomic composition of <em>M. tuberculosis</em> isolates from the high TB-burden country, Myanmar.</p>


<h3>Methodology/Principle Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Twenty-two <em>M. tuberculosis</em> isolates from Myanmar were screened on whole-genome arrays containing genes from <em>M. tuberculosis</em> H37Rv, <em>M. tuberculosis</em> CDC1551 and <em>M. bovis</em> AF22197. Screening identified 198 deletions or extra regions in the clinical isolates compared to H37Rv. Twenty-two regions differentiated between Beijing and non-Beijing isolates and were verified by PCR on an additional 40 isolates. Six regions <em>(Rv0071-0074</em> [RD105], <em>Rv1572-1576c</em> [RD149], <em>Rv1585c-1587c</em> [RD149], <em>MT1798-Rv1755c</em> [RD152], <em>Rv1761c</em> [RD152] and <em>Rv0279c</em>) were deleted in Beijing isolates, of which 4 (<em>Rv1572-1576c, Rv1585c-1587c, MT1798-Rv1755c</em> and <em>Rv1761c</em>) were variably deleted among ST42 isolates, indicating a closer relationship between the Beijing and ST42 lineages. The TbD1 region, <em>Mb1582-Mb1583</em> was deleted in Beijing and ST42 isolates. One <em>M. bovis</em> gene of unknown function, <em>Mb3184c</em> was present in all isolates, except 11 of 13 ST42 isolates. The CDC1551 gene, <em>MT1360</em> coding for a putative adenylate cyclase, was present in all Beijing and ST42 isolates (except 1). The <em>pks15/1</em> gene, coding for a putative virulence factor, was intact in all Beijing and non-Beijing isolates, except in ST42 and ST53 isolates.</p>


<h3>Conclusion</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This study describes previously unreported deletions/extra regions in Beijing and non-Beijing <em>M. tuberculosis</em> isolates. The modern and highly frequent ST42 lineage showed a closer relationship to the hypervirulent Beijing lineage than to the ancient non-Beijing lineages. The <em>pks15/1</em> gene was disrupted only in modern non-Beijing isolates. This is the first report of an in-depth analysis on the genomic diversity of <em>M. tuberculosis</em> isolates from Myanmar.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Stavrum R, Valvatne H, Bø TH, Jonassen I, Hinds J, et al.  (2008) Genomic Diversity among Beijing and non-Beijing <em>Mycobacterium tuberculosis</em> Isolates from Myanmar. PLoS ONE 3(4):
          e1973.
            doi:10.1371/journal.pone.0001973</p><p><strong>Editor: </strong>Leonardo A. Sechi, Università di Sassari, Italy</p><p><strong>Received:</strong> January 11, 2008; <strong>Accepted:</strong> February 29, 2008; <strong>Published:</strong> April 9, 2008</p><p><strong>Copyright:</strong> © 2008 Stavrum et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was funded by Helse Vest, the Norwegian Research Council and University of Bergen. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p><em>Mycobacterium tuberculosis</em>, the causative agent of tuberculosis (TB) is an intracellular pathogen which has evolved to infect and persist inside the host macrophage, a cell which is specialized to kill intracellular bacteria <a href="#pone.0001973-Gehring1">[1]</a>. Once inside the macrophage the bacilli may evade or subvert the host immune response causing either a latent infection or active disease <a href="#pone.0001973-Flynn1">[2]</a>. The outcome of infection is dependent on both host-specific and pathogen-specific factors <a href="#pone.0001973-Malik1">[3]</a>. Different strains of <em>M. tuberculosis</em> possess varying degrees of virulence resulting in different immune responses <a href="#pone.0001973-Lopez1">[4]</a> . The genetic determinants causing variations in virulence and transmissibility have not yet been elucidated. However, a polyketide synthase (pks)-derived phenolic glycolipid (PGL) has been suggested to be involved in down-regulation of the immune response in W/Beijing strains <a href="#pone.0001973-Reed1">[5]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Although the <em>M. tuberculosis</em> genome exhibits very little genomic sequence diversity <a href="#pone.0001973-Musser1">[6]</a> the sequencing and comparison of the genomes of <em>M. tuberculosis</em> H37Rv and <em>Mycobacterium bovis</em> has revealed large sequence polymorphisms (LSPs) distinguishing these two strains from each other <a href="#pone.0001973-Mostowy1">[7]</a>. Due to considerable <em>in vitro</em> passages of laboratory strains, the relevance of the <em>M. tuberculosis</em> H37Rv genome to clinical <em>M. tuberculosis</em> strains has been questioned <a href="#pone.0001973-Fleischmann1">[8]</a>. The sequencing of the <em>M. tuberculosis</em> strain CDC1551, which is a more recent clinical isolate known to be transmissible and virulent in humans, may therefore provide information about genes involved in <em>in vivo</em> infection that are retained in clinical isolates, but may have been lost during <em>in vitro</em> passage of <em>M. tuberculosis</em> H37Rv <a href="#pone.0001973-Fleischmann1">[8]</a>. According to WHO, 40% of the estimated 10 million new cases of TB each year are located in Southeast-Asia. Myanmar is one of the 22 countries that combined account for 80% of the world's total new TB cases <a href="#pone.0001973-WHO1">[9]</a>. One family of <em>M. tuberculosis</em> strains, Beijing, has attracted special attention. This hypervirulent family is reported to be common in several Asian studies and may possess selective advantages compared to other genotypes <a href="#pone.0001973-Glynn1">[10]</a>, <a href="#pone.0001973-Phyu1">[11]</a>. This family is also more often associated with multi-drug resistance <a href="#pone.0001973-Lillebaek1">[12]</a>. Strains belonging to the Beijing family share genetic markers, such as similar IS<em>6110</em> restriction fragment length polymorphism (RFLP) and spoligotype pattern <a href="#pone.0001973-Bifani1">[13]</a>, <a href="#pone.0001973-Kremer1">[14]</a>. The similarities between different Beijing strains may suggest clonal expansion, and the detection of LSP's can provide insights into the evolution and biology of this family. A study by Tsolaki <em>et al</em> using comparative whole-genome hybridization of Beijing/W strains against <em>M. tuberculosis</em> H37Rv, revealed 7 previously unreported LSP's, of which one (RD105) could serve as a genetic marker for Beijing/W strains and 3 additional LSP's (RD142, RD150 and RD181) that could be used for sub-typing of this family <a href="#pone.0001973-Tsolaki1">[15]</a>. Thus, with the aim to understand the genetic diversity of Beijing and non-Beijing isolates from Myanmar and to determine if previously reported genetic markers can differentiate between Beijing and non-Beijing <em>M. tuberculosis</em> from a defined area, we have screened <em>M. tuberculosis</em> isolates on whole-genome microarrays that represent all genes contained in <em>M. tuberculosis</em> H37Rv, <em>M. tuberculosis</em> CDC1551 and <em>M. bovis</em> 2122/97.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4><em>M. tuberculosis</em> isolates</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Sputum specimens were collected from pulmonary TB patients attending four district diagnostic and treatment centers run by the National TB Program, Yangon division, Myanmar. Following ethical approval to obtain sputum samples (REK Vest, Norway; 03/09931 and Ethical committee on Medical Research, Yangon, Myanmar; 3/2002) in the periods April to August 2002 and October to December 2002, 567 patients older than 14 years of age with pulmonary TB were included <a href="#pone.0001973-Phyu2">[16]</a>. Following verbal and written consent, two smear-positive sputum samples from each patient were obtained and all participants were interviewed by trained physicians using structured questionnaires, as described previously <a href="#pone.0001973-Phyu2">[16]</a>. The information collected included socio-economic and demographic characteristics, current and previous history of <em>M. tuberculosis</em> infection, history of contact with a TB case, history of previous anti-tuberculosis treatment and chest X-ray findings. The Yangon division with its ~6 million habitants is the most densely populated area in Myanmar representing about 10% of the country's population. Of 310 well-characterized <em>M. tuberculosis</em> isolates (S. Phyu, <em>et al.,</em> unpublished data), 22 (11 Beijing, 4 ST42, 2 ST89 and 3 previously unreported [UR] shared-types [ST]) were chosen for further characterization by comparative genomic hybridization (CGH) based on a high degree of IS<em>6110</em> RFLP clustering isolates within each genotype. Among the 11 Beijing isolates, two pairs of isolates (TB29/TB30 and TB33/TB35) with identical IS<em>6110</em> RFLP pattern were included. In addition, two isolates with low-copy number of IS<em>6110</em> by RFLP (1 ST947 and 1 UR) were selected. <em>M. tuberculosis</em> H37Rv was used as a reference strain. An additional 24 (8 Beijing, 2 ST42 and 14 other non-Beijing) isolates from the same collection of clinical isolates from Myanmar and 16 (9 Beijing and 7 ST42) well-characterized isolates (M. Mphahlele, <em>et al.,</em> unpublished data) from South Africa were included in the study in order to verify the results observed by CGH for the respective genotypes.</p>


<h4>DNA isolation</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Chromosomal DNA used in array hybridization experiments and in PCR reactions was isolated as described by van Embden <em>et al</em> <a href="#pone.0001973-vanEmbden1">[17]</a> or by heat-extraction using a loopful of cells suspended in 200 µl of TE buffer (10 mM Tris-Cl, 1 mM EDTA), and heat-killed by incubation at 95°C for 20 min. The supernatant containing the extracted DNA was collected by centrifugation at 12000 rpm for 10 min.</p>


<h4>Microarray hybridization</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Twenty-two <em>M. tuberculosis</em> isolates (11 Beijing and 11 non-Beijing) were analyzed by whole-genome comparative microarrays, which include genes from <em>M. tuberculosis</em> H37Rv with updated genome annotations <a href="#pone.0001973-Camus1">[18]</a>, <em>M. tuberculosis</em> CDC1551 and <em>M.bovis</em> AF2122/97 as previously described <a href="#pone.0001973-Stewart1">[19]</a>. The array design is available in BµG@Sbase (accession number: A-BUGS-34; <a href="http://bugs.sgul.ac.uk/A-BUGS-34">http://bugs.sgul.ac.uk/A-BUGS-34</a>) and also ArrayExpress (accession number: A-BUGS-34). Each array consisted of a total of 10944 features and included duplicates of each of the 4410 genes as PCR-generated elements (size-range 60–1000 bp; mean 517 bp) which were spotted onto Corning UltraGaps aminosilane coated glass slides <a href="#pone.0001973-Hinds1">[20]</a>. The signal intensities observed were the result of a competitive hybridization between 5 µg of clinical genomic <em>M. tuberculosis</em> DNA and 5 µg genomic control DNA (<em>M. tuberculosis</em> H37Rv). The DNA was labeled with Cy3 (test strain) or Cy5-labeled (control strain) dCTP (Amersham) by random priming (Invitrogen Life Technologies) <a href="#pone.0001973-Stewart1">[19]</a> and purified (Qiagen MinElute PCR purification kit). Purified Cy5-labelled <em>M. tuberculosis</em> H37Rv and Cy3-labelled genomic DNA from the <em>M. tuberculosis</em> isolates from Myanmar were mixed and added to the hybridization solution and applied to the array <a href="#pone.0001973-Stewart1">[19]</a>. The slides were then sealed in a humid hybridization cassette and incubated in the dark at 65°C for 16 hrs. Following washing <a href="#pone.0001973-Stewart1">[19]</a>, slides were scanned using a dual-laser scanner at a level just below saturation of the most intensely fluorescent spots on each array. The images were quantified using GenePix Pro Image Analysis 5.</p>


<h4><em>M. tuberculosis</em> CGH microarray data analysis</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Background subtracted intensities in the red and green channel of non-flagged spots on each array were submitted to quantile normalization <a href="#pone.0001973-Bolstad1">[21]</a> and transformed using glog transformation <a href="#pone.0001973-Huber1">[22]</a>. The channel difference obtained by using glog transformation, referred to as the glog ratio, was set to 0 for flagged spots. The array-probes were organized in the same manner as their respective genes appear on the <em>M. tuberculosis</em> H37Rv genome, with the addition of probes from the genome of <em>M. bovis</em> and <em>M. tuberculosis</em> CDC1551 interspersed between the probes from <em>M. tuberculosis</em> H37Rv. The glog ratios for each spot were ordered accordingly. To estimate the parameters needed for glog transformation, we minimized the dependency of the variance of the glog ratios to the signal intensity as described previously <a href="#pone.0001973-Huber1">[22]</a>, <a href="#pone.0001973-Rocke1">[23]</a>. During this search, 5% of the extreme outlier glog ratios compared to the global median glog ratio were excluded. To find deleted or extra regions in the clinical isolates compared to <em>M. tuberculosis</em> H37Rv, the ordered glog ratios were scanned to locate regions with outlier populations of glog ratios, and the standard deviation <em>s</em> of non-flagged glog ratios was estimated with the inter-quartile range (IQR, <em>s</em> = IQR/1.34898). The z-score was calculated by taking the average glog ratio of a region, trying region sizes from one to ten probes, divided by the expected standard deviation of an average coming from an N(0,<em>s</em>) distribution. Regions with z-scores of more than ±4.42 (absolute value) occurring in one or more samples, corresponding to a standard normal P-value of 0.00001 (1e-5), were reported and the data in these regions inspected to define borders for deletions/extra regions. If at least one of the isolates gave a z-score above the threshold in a region, it was taken as evidence to make a call, and a lower z-score threshold of ±2.58 (corresponding to P = 0.001) was used in the analysis of the other isolates in calling deleted or extra regions. As signal intensity and hence, channel difference could be affected by sequence divergence, copy number or cross-hybridization effects, the degree of evidence was graded into categories for each call: deletions or extra regions with z-scores corresponding to P-values between 1e-3 and 1e-5 were not strong enough in themselves to call a region, z-scores corresponding to P-values between 1e-5 and 1e-9 indicated sufficient evidence to call a region, strong evidence was supported by z-scores corresponding to P-values between 1e-9 and 1e-16, and even stronger evidence was indicated by z-scores corresponding to P-values lower than 1e-16. The criteria for selecting regions to be verified by PCR were: regions with three or more isolates containing a deletion or an extra region with a z-score corresponding to a P-value of at most 1e-5 that could differentiate the Beijing from the non-Beijing isolates.</p>


<h4>Verification of extra regions and deletions in clinical isolates</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Deletions or extra regions detected by CGH, meeting the criteria stated above, were analyzed further by PCR. Thus, for all reactions amplifications were performed in a total volume of 25 µl containing 1 µl of DNA, 0.4 µM of each primer (<a href="#pone-0001973-t001">Table 1</a>), and Hotstart <em>Taq</em> polymerase Master Mix (Qiagen). All reactions were subjected to 95°C for 15 min, followed by 30 cycles of 40 sec at 94°C, 40 sec at 55°C, 1–4 min at 72°C and terminated by 7 min at 72°C. Fragments were visualized by UV transillumination or by Bioanalyzer 2100 DNA 1000 LabChip kit in accordance with the manufacturer's instructions (Agilent Technologies, USA).</p>
<div class="figure" id="pone-0001973-t001"><div class="img"><a name="pone-0001973-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001973" data-uri="info:doi/10.1371/journal.pone.0001973.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001973.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001973.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Primers used for the PCR verification of deleted and extra regions as observed by CGH analysis.</span></strong></p><span>doi:10.1371/journal.pone.0001973.t001</span></div>

<h4>RD207 verification</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>PCR of the RD207 region was performed on isolates which had discrepant results between spoligotyping, CGH and RD105 verification, or if a mixed <em>M. tuberculosis</em> sub-population in a sample was suspected. The RD207 deletion has only been observed in strains of the Beijing lineage and is reported to be a Beijing defining deletion <a href="#pone.0001973-Tsolaki1">[15]</a>. Four sets of primers were used for RD207 PCR verification; two overlapping sets unique to an IS<em>6110</em> insertion only found in <em>Rv2820</em> in strains of the Beijing lineage, and two sets complementary to <em>Rv2819</em>, which will only produce a PCR product in strains of the non-Beijing lineage <a href="#pone.0001973-Warren1">[24]</a>.</p>


<h4>MIRU-VNTR</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>In cases where mixed infections were suspected, the isolates were genotyped by variable number tandem repeat of mycobacterial interspersed repetitive units (MIRU-VNTR) by amplification of 15 loci as described by Supply <em>et al</em> <a href="#pone.0001973-Supply1">[25]</a>. PCR amplification was performed in a total volume of 20 µl containing 1 µl DNA, 0.04–0.4 µM of all 15 primer sets, and Hotstart <em>Taq</em> Plus polymerase Master Mix (Qiagen). All reactions were subjected to 95°C for 5 min, followed by 30 cycles of 30 sec at 94°C, 1 min at 55°C, 1.5 min at 72°C and terminated by 7 min at 72°C. Genotyping was performed using multiplex PCR with a Rox-labeled MapMarker 1000 size standard (PE Applied Biosystems) for sizing of the PCR products. The PCR fragments were analysed using a capillary-based electrophoresis sequencer (ABI 3700), and sizing of the various VNTR alleles were done using the Peak Scanner Software v1.0 (PE Applied Biosystems).</p>


<h4>Further sub-typing of clinical isolates</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>Two regions, RD142 and RD181, previously described by Tsolaki <em>et al</em> <a href="#pone.0001973-Tsolaki1">[15]</a> as variably deleted in strains of the Beijing/W lineage, were chosen for further sub-typing of the isolates. The primers used for the PCR were: RD142: L_primer <span class="gene">GGACCTCGCGTTTCGTATGGTA</span> and R_primer <span class="gene">GTGACTCCAGTGGCGCAGGAT</span> (<em>Rv1189</em>), and L_primer <span class="gene">CTGTTGCCTGTGT and R_primer <span class="gene">GATGGCGAATTCATGCACGTCT</span> (<em>Rv1192</em>); RD181: L_primer <span class="gene">CGCAACGGCCGCGGTGAACTCT</span> and R_primer <span class="gene">CGGGCGGCTGCGGGAACCTT</span> <a href="#pone.0001973-Tsolaki1">[15]</a>.</span></p>


<h4><em>pks15/1</em> polymorphism</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>The presence of the <em>pks15/1</em> polymorphism in the <em>M. tuberculosis</em> isolates was determined by PCR and sequencing as described <a href="#pone.0001973-Tsolaki1">[15]</a>. The sequencing reaction consisted of an initial 5 min for denaturation at 96°C, 25 cycles of 96°C for 15 sec, 50°C for 10 sec, 60°C for 4 min and elongation time of 72°C for 15 min using the BigDye 3.1 Terminator Cycle Sequencing Kit (Applied Biosystems Inc). The sequencing was carried out in an ABI 377 automatic DNA sequencer (Applied Biosystems Inc., Foster, CA) and nucleotide sequences were analyzed using DNASTAR software (DNASTAR, Inc., USA).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The CGH analysis of 22 <em>M. tuberculosis</em> isolates from Myanmar revealed a total of 198 regions where the genomic composition of the isolates tested were different (deletions or extra regions) from that of the reference strain, <em>M. tuberculosis</em> H37Rv (<a href="#pone-0001973-g001">Figure 1</a>). The number of deletions/extra regions detected by CGH ranged from 29–78 for the Beijing isolates, with an average of 41.7 per isolate, 23–61 for the ST42 isolates, with an average of 40.4 per isolate, and 17–56 for other non-Beijing isolates, with an average of 37.8 per isolate. CGH also revealed several strain-specific genomic variations between the Beijing and non-Beijing strains. Furthermore, the two pairs of isolates (TB33/TB35, TB29/TB30) with identical <em>IS6110</em> RFLP pattern, and where no close epidemiologic connection between the patients could be established, could be distinguished from one another by CGH analysis.</p>
<div class="figure" id="pone-0001973-g001"><div class="img"><a name="pone-0001973-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001973" data-uri="info:doi/10.1371/journal.pone.0001973.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001973.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001973.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Genetic diversity among 22 <em>Mycobacterium tuberculosis</em> isolates tested by CGH.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>The dendogram was produced by weighted pair group method (WPGMA) clustering using a Pearson correlation based distance measure defined on regional z-score values. The red color indicates deletions in the clinical isolates compared to the reference strain <em>M. tuberculosis</em> H37Rv, whereas the green color shows extra regions in the clinical isolates not present in the reference strain <em>M. tuberculosis</em> H37Rv.</p>
<span>doi:10.1371/journal.pone.0001973.g001</span></div><a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Of the 198 regions that showed either deletions or extra regions compared to H37Rv on the arrays, 22 (<a href="#pone-0001973-g002">Figure 2</a>) were selected according to the predefined criteria for further verification by PCR on the extended battery of isolates. Fully annotated microarray data has been deposited in BµG@Sbase (accession number: E-BUGS-66; <a href="http://bugs.sgul.ac.uk/E-BUGS-66">http://bugs.sgul.ac.uk/E-BUGS-66</a>) and also ArrayExpress (accession number: E-BUGS-66).</p>
<div class="figure" id="pone-0001973-g002"><div class="img"><a name="pone-0001973-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001973" data-uri="info:doi/10.1371/journal.pone.0001973.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001973.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001973.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Regions chosen for PCR verification based on the CGH array analysis.</span></strong></p><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1"></a><p>The dendogram based on CGH array analysis of 22 <em>M. tuberculosis</em> isolates from Myanmar (11 Beijing, 4 ST42, 2 ST89 and 5 previously unreported [UR] shared-types [ST]) was produced by weighted pair group method (WPGMA) clustering using a Pearson correlation based distance measure defined on regional z-score values, corresponding to P-values between 1e-3 and 1e-16. The green color indicates extra regions that are present in the clinical isolates, but not in the reference strain <em>M. tuberculosis</em> H37Rv, whereas the red color depicts deletions in the clinical isolates compared to the reference strain <em>M. tuberculosis</em> H37Rv.</p>
<span>doi:10.1371/journal.pone.0001973.g002</span></div>
<h4>Deletions</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>RD207 (<em>Rv2815c-Rv2820c</em>) was deleted in all Beijing isolates, but present in all non-Beijing isolates, except TB21. RD207 is a Beijing-defining deletion, which is also detected by spoligotyping.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p><em>Rv0279c</em> was deleted in 8 of the 11 Beijing isolates on array, but present in all non-Beijing isolates, except TB21 (<a href="#pone-0001973-g002">Figure 2</a>). PCR verification detected a deletion in this region for all Beijing isolates in the study, and TB21 (<a href="#pone-0001973-g003">Figure 3</a>).</p>
<div class="figure" id="pone-0001973-g003"><div class="img"><a name="pone-0001973-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001973" data-uri="info:doi/10.1371/journal.pone.0001973.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001973.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001973.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>PCR results from the 22 regions chosen for verification in 60 clinical <em>M. tuberculosis</em> isolates.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>The dendogram based on PCR results from 16 deletions and 6 extra regions chosen for verification on 60 clinical <em>M. tuberculosis</em> isolates (28 Beijing, 13 ST42 and 19 other non-Beijing) was produced by unweighted pair group method (UPGMA) clustering using a Pearson correlation based distance measure.</p>
<span>doi:10.1371/journal.pone.0001973.g003</span></div><a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>RD105 was by CGH, deleted in all Beijing isolates, except TB23 and TB25. PCR verification confirmed a deletion in RD105 for all Beijing isolates, including TB23 and TB25. However, PCR analysis also revealed an RD105 deletion in two non-Beijing isolates, TB21 and ST126.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>RD149 (<em>Rv1573</em>-<em>Rv1584c</em>) was, by CGH, observed to be deleted in all Beijing isolates, except TB23, and all 4 ST42 isolates. PCR verification confirmed a deletion in <em>Rv1572c-1576c</em> for all Beijing isolates, including TB23, and in 8 of 13 ST42 isolates as well as 3 other non-Beijing isolates (ST48, ST947 and TB50). <em>Rv1585c-1587c</em> was deleted in all Beijing, 10 ST42 and 5 other non-Beijing isolates (<a href="#pone-0001973-g003">Figure 3</a>).</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p><strong>T</strong>he RD152 region (<em>Rv1754c-1765c</em>) was variably deleted in isolates of the Beijing and ST42 lineages, as well as in 1 other non-Beijing isolate (TB21) [<a href="#pone-0001973-g002">Figure 2</a>]. However, as shown in <a href="#pone-0001973-g004">Figure 4</a>, not all genes within this region were deleted. Thus, PCR verification of <em>MT1798</em>-<em>Rv1755c</em> revealed a deletion in all Beijing and 10 of 13 ST42 isolates (<a href="#pone-0001973-g003">Figure 3</a>). <em>Rv1761c</em> was deleted in all Beijing, 8 of 13 ST42 isolates and 2 other non-Beijing isolates (ST89 and TB21), whereas <em>Rv1765c</em> was variably deleted only among Beijing isolates (2 of 28) [<a href="#pone-0001973-g003">Figure 3</a>].</p>
<div class="figure" id="pone-0001973-g004"><div class="img"><a name="pone-0001973-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001973" data-uri="info:doi/10.1371/journal.pone.0001973.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001973.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001973.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001973.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001973.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001973.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Gene graph of the <em>M.tuberculosis</em> H37Rv <em>1754c-1765c, Mb1785-1787</em>, <em>MT1808</em> and <em>MT1812-1813</em> regions.</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>The graph is based on region z-scores, with each line representing one isolate analyzed by CGH. Region z-scores of more than ±4.42 (absolute value) occurring in one or more samples, corresponding to a standard normal P-value of 0.00001 (1e-5) indicate deletions (above the 0-line) or extra regions (below 0-line). Red lines represent isolates of the Beijing lineage, green lines; ST42 lineage and blue lines; non-Beijing isolates other than ST42.</p>
<span>doi:10.1371/journal.pone.0001973.g004</span></div><a id="article1.body1.sec3.sec1.p6" name="article1.body1.sec3.sec1.p6"></a><p>Two regions, RD174 (<em>Rv1994</em>-<em>1997</em>) and <em>Rv3351c-3353c</em> were deleted in all ST42 isolates analyzed by CGH. PCR confirmed a deletion in all 6 ST42 isolates from Myanmar, whereas only 2 out of 7 South-African ST42 isolates had deletions in these two regions.</p>
<a id="article1.body1.sec3.sec1.p7" name="article1.body1.sec3.sec1.p7"></a><p>CGH analysis of RD6 (<em>Rv3425-3429</em>) showed a deletion in four ST42 isolates and in ST947. PCR validated a deletion in 3 of the ST42 isolates tested on array, whereas 1 isolate, TB28, gave a product of the expected size. Furthermore, PCR also detected an RD6 deletion in 3 additional ST42 isolates, and 3 other non-Beijing isolates (<a href="#pone-0001973-g003">Figure 3</a>). A PCR product that was larger than the expected size was obtained for 22 of 28 Beijing isolates.</p>
<a id="article1.body1.sec3.sec1.p8" name="article1.body1.sec3.sec1.p8"></a><p>Two regions, <em>Rv1524</em>-<em>1525</em> and <em>Rv3651</em> were, by CGH analysis, deleted in all non-Beijing isolates, except the ST42 isolates and TB21. PCR verification confirmed a deletion of <em>Rv1524</em>-<em>1525</em> among the non-Beijing isolates, except 4 isolates; TB21, ST26, ST53 and ST126. PCR of <em>Rv3651</em> showed a deletion in all non-Beijing isolates, except TB21, ST26 and ST53.</p>
<a id="article1.body1.sec3.sec1.p9" name="article1.body1.sec3.sec1.p9"></a><p><em>Rv3179</em> was, by CGH, variably deleted among Beijing isolates. PCR analysis on the extended battery of strains detected a deletion in two additional Beijing isolates and 2 other non-Beijing isolates; ST48 and 1 ST292. Two ST42 isolates contained an extra insert of ~900 bp.</p>
<a id="article1.body1.sec3.sec1.p10" name="article1.body1.sec3.sec1.p10"></a><p>CGH and PCR analysis showed that all non-Beijing isolates had an intact RD150 (<em>Rv1672c-1673c</em>) region. Among the 11 Beijing isolates screened by CGH, 3 were observed to have a deletion in this region. PCR also detected two additional Beijing isolates with a deletion in RD150 (<a href="#pone-0001973-g003">Figure 3</a>).</p>
<a id="article1.body1.sec3.sec1.p11" name="article1.body1.sec3.sec1.p11"></a><p>The region <em>Rv3018c-3020c</em>, was on array examination reported to be deleted in 2 out of 11 Beijing isolates (TB27 and TB34 ) and 4 out of 6 non-Beijing isolates; 2 ST89 (TB24 and TB26) and 2 UR shared-types (TB31 and TB36) [<a href="#pone-0001973-g002">Figure 2</a>]. Further examination of this region using the J-Express Pro 2.7 package (Molmine) showed that the two Beijing isolates (TB27 and TB34) only had <em>Rv3019c</em>, deleted. The extended PCR verification of this region showed that <em>Rv3018c</em> was deleted in 1 additional Beijing isolate and 1 additional non-Beijing isolate (ST292), whereas TB27 and TB34 gave a product of the expected size (<a href="#pone-0001973-g003">Figure 3</a>). <em>Rv3019c</em> was deleted in 15 Beijing isolates (including TB27 and TB34) and 13 non-Beijing isolates (<a href="#pone-0001973-g003">Figure 3</a>). <em>Rv3018c</em> and <em>3019c</em> were present in all ST42 isolates.</p>


<h4>Extra regions</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>CGH analysis indicated the presence of two extra regions, <em>M. bovis</em> AF2122/97 (Mb) <em>1785-1787</em> and CDC1551 (MT) <em>1812</em>-<em>1813</em> in 6 of 11 non-Beijing isolates tested. PCR confirmed the presence of <em>Mb1785-1787</em> in all non-Beijing isolates, except 9 ST42 and TB21. The PCR analysis for <em>MT1812-1813</em> revealed that all non-Beijing isolates, except 6 ST42 and TB21, contained this extra region. PCR also detected the presence of this extra region in 12 Beijing isolates.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p><em>Mb1582-Mb1583</em> (TbD1) was, by CGH analysis, present in 5 of 6 non-Beijing isolates, but absent from the non-Beijing isolate TB21 as well as all Beijing and ST42 isolates. PCR on the larger battery of strains verified the presence of <em>Mb1582-1583</em> in all non-Beijing isolates, except ST53, ST126, TB21 and all ST42.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p><em>MT1360</em> was by CGH, present in 8 of 11 Beijing, 4 non-Beijing isolates; 3 ST42 isolates and TB21. The extended PCR verification showed that all Beijing isolates, 4 non-Beijing (TB21, ST26, ST53 and ST126) and 12 of 13 ST42 isolates had this extra region. Of the 12 ST42 isolates with this extra region, 10 had a PCR product which was 900 bp larger than in the Beijing and the other 6 non-Beijing isolates (<a href="#pone-0001973-g003">Figure 3</a>).</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p><em>Mb3184c</em> was, by CGH, detected in all Beijing and non-Beijing isolates, except ST42. PCR verified the CGH results for all isolates, except for 1 ST42 (TB20) which gave a product of the expected size. All isolates, except 11 ST42 from the extended battery, contained this region (<a href="#pone-0001973-g003">Figure 3</a>).</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>Four Beijing isolates and 7 non-Beijing isolates, appeared by array examination to contain the extra <em>MT1968-Mb1951c</em> region. One Beijing (TB13) and 1 ST42 (TB20) isolate showed deletions in this region. PCR analysis showed that all isolates had this extra region, except 10 ST42 isolates (<a href="#pone-0001973-g003">Figure 3</a>).</p>


<h4>Sub-typing</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>RD181 (<em>Rv2262c-2263</em>), which has previously been reported to be variably deleted in Beijing strains, appeared to be present in all isolates tested on the arrays. However, PCR verification revealed that RD181 was deleted in all but one Beijing isolates tested.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>RD142 (<em>Rv1189-1192</em>) has previously been reported to be variably deleted in Beijing/W strains. The array analysis indicated that this region was intact in all isolates tested on array. Further, PCR verification on the larger battery of isolates confirmed the array hybridization results and did not detect any isolates with deletions in RD142.</p>


<h4>MIRU-VNTR</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>As RD105 is reported to be a Beijing-specific deletion, two non-Beijing isolates (by spoligotyping) [TB21 and ST126], that showed an RD105 deletion, were subjected to PCR verification of the RD207 region using 4 primer sets which are designed to discriminate specifically between Beijing and non-Beijing strains <a href="#pone.0001973-Warren1">[24]</a>. PCR of RD207 for TB21 and ST126 produced products for all four primer sets, suggesting a mixed <em>M. tuberculosis</em> sub-population. Thus, further characterization of these two isolates was undertaken using MIRU-VNTR. MIRU-VNTR of TB21 showed two PCR products for locus Mtub04 (645/681) and Mtub30 (362/471 bp), whereas ST126 showed two PCR products for the MIRU 10 (596/633 bp), Qub-4156c (581/702), Qub-11b (311/542 bp), Mtub21 loci (202/257) and Qub-26 (734/887). Two other isolates (ST53 and ST26), where a mixed <em>M. tuberculosis</em> sub-population was suspected, were also subjected to PCR of the RD207 region. PCR produced products for primer sets 3 and 4, indicating the presence of non-Beijing DNA only. MIRU-VNTR analysis showed single PCR products for all loci in both isolates.</p>


<h4><em>pks15/1</em> polymorphism</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>The <em>pks15/1</em> operon was screened for the previously described 1 or 7 base pair deletions by PCR and sequencing <a href="#pone.0001973-Constant1">[26]</a>. The results from this analysis revealed an intact open reading frame in all Beijing and non-Beijing isolates, except for 14 isolates, of which 13 belonged to the ST42 lineage and 1 to the ST53 lineage, all having a 7-bp deletion at this locus.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The <em>M. tuberculosis</em> Beijing family is highly prevalent in Asian countries <a href="#pone.0001973-Chan1">[27]</a>, <a href="#pone.0001973-vanSoolingen1">[28]</a> and the identification of strain-specific LSP's in the genome of this hypervirulent family may aid in the identification of important virulence factors. The Beijing/W family of strains share highly similar multicopy IS<em>6110</em> RFLP pattern <a href="#pone.0001973-Kremer1">[14]</a> as well as possess both strain-specific LSPs and polymorphisms which it shares with other families <a href="#pone.0001973-Tsolaki1">[15]</a>. We have recently shown (S. Phyu, <em>et al,</em> unpublished data), that isolates of the ST42 lineage from Myanmar also share highly similar RFLP patterns. ST42 strains belong to the LAM superfamily and are less common in Asia <a href="#pone.0001973-Filliol1">[29]</a>. Thus, 22 <em>M. tuberculosis</em> isolates from Myanmar, were screened for genomic deletions or extra regions by CGH, and selected regions which on array analysis potentially differentiated the Beijing isolates from the non-Beijing isolates were verified by PCR.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Array examination revealed no significant difference in the number of deletions/extra regions between the Beijing, ST42 and the other non-Beijing lineages. A previously unreported Beijing specific deletion in <em>Rv0279c</em> was detected. PCR analysis verified this deletion in all Beijing isolates, whereas all but one (TB21) non-Beijing isolates had this region intact. <em>Rv0279c</em> belongs to a family consisting of other PE-PGRS genes whose transcripts have been suggested to have fibronectin-binding abilities <a href="#pone.0001973-Espitia1">[30]</a>. <em>Rv0278c</em> and <em>Rv0279c</em> are contiguous sequences and share a high sequence homology with PE-PGRS81 <a href="#pone.0001973-Espitia1">[30]</a><strong>.</strong> As a result of high sequence homology among members of the PE-PGRS family and the many repetitive sequences in these genes, PCR and hybridization-based characterization of the PE-PGRS regions may be difficult to interpret. However, in this study, the 164 bp deletion observed in <em>Rv0279c</em> could consistently differentiate between the Beijing and the non-Beijing isolates included.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>RD105 and RD207 deletions have previously been reported to be Beijing specific <a href="#pone.0001973-Tsolaki1">[15]</a>. As expected, all Beijing isolates included in the study had RD105 deleted. However, two non-Beijing isolates, 1 UR (TB21) and 1 ST126, also showed deletions in the RD105 region. Further investigation of these two isolates using PCR <a href="#pone.0001973-Tsolaki1">[15]</a> and MIRU-VNTR <a href="#pone.0001973-Supply1">[25]</a>, confirmed the presence of a mixed sub-population of <em>M. tuberculosis</em>. Two other non-Beijing isolates, 1 ST53 and 1 ST26 were, based on PCR verification of the 22 regions, also suspected of having a mixed <em>M. tuberculosis</em> sub-population. PCR of the RD207 region showed the presence of only non-Beijing DNA and MIRU-VNTR indicated the presence of a homogeneous <em>M. tuberculosis</em> population.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>One <em>M. bovis</em> gene, with unknown function, <em>Mb3184c</em>, was present in all isolates, except 11 ST42 isolates. As compared to the annotated H37Rv strain <a href="#pone.0001973-Cole1">[31]</a>, <em>Mb3184c</em> is located between PPE53 (<em>Rv3159c</em>) and <em>Rv3160c</em>. The <em>Mb3184c</em> insert codes for an additional hypothetical protein, PPE70, which is equivalent to MT3248 of the <em>M. tuberculosis</em> CDC1551 strain <a href="#pone.0001973-Garnier1">[32]</a>.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The previously reported 7-bp deletion of the <em>pks15/1</em> locus <a href="#pone.0001973-Constant1">[26]</a> was observed in all ST42 isolates and in one ST53 isolate. <em>Pks15/1</em> codes for an enzyme involved in the production of PGL, which is a virulence factor in Beijing/W isolates. PGL has been shown to inhibit the release of pro-inflammatory mediators in monocyte-derived macrophages, and loss of PGL correlated with an increase in the release of pro-inflammatory cytokines, tumor necrosis factor-α and interleukins 6 and 12 <a href="#pone.0001973-Reed1">[5]</a>. As can be seen in <a href="#pone-0001973-g001">Figure 1</a> and <a href="#pone-0001973-g002">2</a>, the ST42 isolates separate from the other non-Beijing isolates and appear to be more closely related to the Beijing isolates. Furthermore, RD174 (<em>Rv1994c-1997c</em>) and <em>Rv3351c-3353c</em>, were deleted only in isolates of the ST42 lineage. In a study, where the 7-bp deletion in the <em>pks15/1</em> locus was used as a phylogenetic marker for global population analysis, the deletion of RD174 was observed only in isolates with a deletion in <em>pks15/1</em> <a href="#pone.0001973-Gagneux1">[33]</a>. In this study, all ST42 isolates from Yangon, and 2 of 7 ST42 isolates from South-Africa (TB76 and TB79) showed combined deletions of <em>pks15/1</em>, RD174 and <em>Rv3351c-3353c</em>, suggesting that there could be different circulating sub-lineages of ST42. Two previously unreported deletions, <em>Rv1524-1525</em> and <em>Rv3651</em> were detected in 17 and 18 non-Beijing isolates, respectively. These deletions may therefore have occurred in a common ancestor after the divergence of the Beijing and ST42 lineages.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>Tsolaki <em>et al</em> <a href="#pone.0001973-Tsolaki1">[15]</a> have shown that 5 regions (RD142, RD149, RD150, RD152 and RD181) were variably deleted in strains of the Beijing/W lineage. In this study, RD142 was intact in all isolates, RD150 was deleted in 5 of 28 Beijing isolates, whereas RD181, which appeared to be intact in all isolates by CGH, was shown by PCR to be deleted in all Beijing isolates, except TB42. The reason for the discrepancy between the array and PCR results for RD181 is that the deleted fragment from the RD181 region is not included in the PCR amplicons spotted onto the array. CGH and PCR analysis showed that <em>Rv1572c-1576c</em> and <em>Rv1585c-1587c</em> of the RD149 region were deleted in all Beijing and ST42 isolates as well as in 3 of 21 other non-Beijing isolates. Furthermore, CGH results for RD152 indicated large variations in the genes deleted within this region in the Beijing and ST42 strains (<a href="#pone-0001973-g004">Figure 4</a>). CGH and PCR analysis of the RD152 region showed that the genes <em>Rv1756c-1757c</em> and <em>Rv1759c</em> (<em>wag22</em>) were present in all isolates tested, whereas <em>Rv1755c</em> and <em>Rv1761</em> were deleted in all Beijing and ST42 isolates. <em>Rv1759c</em> codes for a protein (wag22) probably expressed during <em>in vivo</em> infection <a href="#pone.0001973-Espitia1">[30]</a> and is a member of a PE-PGRS family.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p><em>Mb1582-Mb1583</em>, belonging to the TbD1 region, was deleted in all Beijing and ST42 isolates, and 3 other non-Beijing isolates (ST26, ST53 and ST126). The presence of the TbD1 region is characteristic of ancestral isolates and the deletion of this region is thought to have occurred as a single evolutionary event in a common ancestor <a href="#pone.0001973-Gutierrez1">[34]</a>. Eighteen of the 21 (86%) non-Beijing isolates, other than ST42, had the TbD1 region intact (TbD1+) suggesting that the majority of the non-Beijing isolates included in this study belong to ancestral lineages. MmpS6 (<em>Mb1582</em>) and MmpL6 (<em>Mb1583</em>) are conserved membrane proteins and transmembrane transport proteins, respectively <a href="#pone.0001973-Garnier1">[32]</a>. Transcription of <em>mmpL6</em> has been shown to be induced by isoniazid, an antibiotic used in the treatment of tuberculosis <a href="#pone.0001973-Betts1">[35]</a>. Interestingly, only 2 of the 20 TbD1+ isolates (10%) in this study were resistant to INH, whereas 21 of 42 (50%) of the TbD1- isolates were resistant to isoniazid <a href="#pone.0001973-Phyu2">[16]</a>.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>In addition to the deletions common for the Beijing and ST42 isolates, the presence of an extra region, <em>MT1360</em>, distinguished the ST42 isolates from other non-Beijing isolates. Thus, all isolates from the Beijing and ST42 lineages (except TB75) contained <em>MT1360</em> (coding for a putative adenylate cyclase involved in signal transduction) <a href="#pone.0001973-Fleischmann1">[8]</a>. All non-Beijing isolates had the extra regions <em>Mb1785c-1787c</em> and <em>MT1812-1813,</em> except 9 and 6 of the 13 ST42 isolates, respectively.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p><em>Rv3425-3429</em> (RD6) has been shown to be deleted in <em>M. bovis</em> and <em>M. tuberculosis</em> Erdmann <a href="#pone.0001973-Malik1">[3]</a>, <a href="#pone.0001973-Brosch1">[36]</a>. RD6 is an insertion element shown to be variably deleted in mycobacterial strains irrespective of origin <a href="#pone.0001973-Rao1">[37]</a>. The array results indicated a deletion of this region only in isolates of the ST42 and ST947 lineages. PCR analysis revealed deletions in RD6 also among isolates belonging to other lineages. PCR results detected an extra insert in this region for 23 of 28 Beijing isolates. These results support previous studies of large variations within this region <a href="#pone.0001973-Brosch1">[36]</a>, <a href="#pone.0001973-Rao1">[37]</a>.</p>
<a id="article1.body1.sec4.p10" name="article1.body1.sec4.p10"></a><p>Of the 198 polymorphic regions detected in this study, one deletion, <em>Rv0279c</em>, was specific for the isolates of the Beijing lineage, in addition to the previously described RD105, RD150, RD181 <a href="#pone.0001973-Tsolaki1">[15]</a>. RD149 and RD152 were variably deleted among isolates of the Beijing and ST42 lineage. In addition, RvD2 (<em>Mb1785-1787</em>) and RvD3 (<em>MT1812-1813</em>) were only deleted among Beijing and ST42 isolates. This may suggest that strains of the ST42 lineage and the highly virulent strains of the Beijing lineage share a more recent common ancestor and that the polymorphic regions common between these two highly frequent lineages may be important for virulence, or that selective pressure has led these strains to converge through independent deletions. We show that, a combined deletion in <em>pks15/1</em>, RD174 and <em>Rv3351c-3353c</em> was useful for sub-grouping strains belonging to the ST42 lineage. Further studies are required to determine if the ST42 isolates, like strains belonging to the Beijing family, show higher transmissibility and increased virulence. In addition, we identified 6 regions from <em>M. tuberculosis</em> CDC1551 and <em>M. bovis</em> among the clinical isolates, which are not present in <em>M. tuberculosis</em> H37Rv. The problem with detecting small genomic variations leaves open the possibility that there may still be many undetected strain-specific genomic variations which could explain the increased virulence seen in some strains.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank Thandar Lwin and Ti Ti, NTP, Myanmar for their invaluable contributions, and the reference laboratory staff for the collection of isolates. We also acknowledge the valuable contribution of Sabai Phyu in the studies performed in Myanmar and for the genotyping of the <em>M. tuberculosis</em> isolates. We thank Matsie Mphahlele and Karin Weyer, MRC, Pretoria, South Africa for collection and typing of the South African isolates, and Anne Fjellbirkeland, Kristi Øvreås and Karl-Henning Kalland for their constructive advice. We also thank Lucy Brooks at the Bacterial Microarray Group at St. George's, University of London for assistance in depositing data in <a href="mailto:BuGSbase">BuGSbase</a> and ArrayExpress. We acknowledge NIH, NIAID Contract No. HHSN266200400091C, entitled “Tuberculosis Vaccine Testing and Research Materials,” which was awarded to Colorado State University for the supply of <em>M.tuberculosis</em> H37Rv DNA. We acknowledge the Wellcome Trust (St. George's, University of London) for funding the microbial pathogen microarray facility under its Functional Genomics Resources Initiative. Results from this study were presented in part at the meeting of The Wellcome Trust Funded Multi-Collaborative Microbial Pathogen Microarray Facility, Hinxton, Cambridge (2005).</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RS HG HV. Performed the experiments: RS HG HV. Analyzed the data: JH RS HG HV TB. Contributed reagents/materials/analysis tools: PB IJ JH TB. Wrote the paper: RS HG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001973-Gehring1" id="pone.0001973-Gehring1"></a>Gehring AJ, Rojas RE, Canaday DH, Lakey DL, Harding CV, et al.  (2003) The Mycobacterium tuberculosis 19-kilodalton lipoprotein inhibits gamma interferon-regulated HLA-DR and Fc gamma R1 on human macrophages through Toll-like receptor 2. Infect Immun  71: 4487–4497.  <ul class="find" data-citedArticleID="1089720" data-doi="10.1128/iai.71.8.4487-4497.2003"><li><a href="http://dx.doi.org/10.1128/iai.71.8.4487-4497.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Mycobacterium+tuberculosis+19-kilodalton+lipoprotein+inhibits+gamma+interferon-regulated+HLA-DR+and+Fc+gamma+R1+on+human+macrophages+through+Toll-like+receptor+2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Mycobacterium+tuberculosis+19-kilodalton+lipoprotein+inhibits+gamma+interferon-regulated+HLA-DR+and+Fc+gamma+R1+on+human+macrophages+through+Toll-like+receptor+2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001973-Flynn1" id="pone.0001973-Flynn1"></a>Flynn JL, Chan J (2001) Tuberculosis: latency and reactivation. Infect Immun  69: 4195–4201.  <ul class="find" data-citedArticleID="1089714" data-doi="10.1128/iai.69.7.4195-4201.2001"><li><a href="http://dx.doi.org/10.1128/iai.69.7.4195-4201.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tuberculosis%3A+latency+and+reactivation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tuberculosis%3A+latency+and+reactivation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001973-Malik1" id="pone.0001973-Malik1"></a>Malik AN, Godfrey-Faussett P (2005) Effects of genetic variability of Mycobacterium tuberculosis strains on the presentation of disease. Lancet Infect Dis  5: 174–183.  <ul class="find" data-citedArticleID="1089736" data-doi="10.1016/s1473-3099(05)01310-1"><li><a href="http://dx.doi.org/10.1016/s1473-3099(05)01310-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+genetic+variability+of+Mycobacterium+tuberculosis+strains+on+the+presentation+of+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+genetic+variability+of+Mycobacterium+tuberculosis+strains+on+the+presentation+of+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001973-Lopez1" id="pone.0001973-Lopez1"></a>Lopez B, Aguilar D, Orozco H, Burger M, Espitia C, et al.  (2003) A marked difference in pathogenesis and immune response induced by different Mycobacterium tuberculosis genotypes. Clin Exp Immunol  133: 30–37.  <ul class="find" data-citedArticleID="1089734" data-doi="10.1046/j.1365-2249.2003.02171.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2249.2003.02171.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+marked+difference+in+pathogenesis+and+immune+response+induced+by+different+Mycobacterium+tuberculosis+genotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+marked+difference+in+pathogenesis+and+immune+response+induced+by+different+Mycobacterium+tuberculosis+genotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001973-Reed1" id="pone.0001973-Reed1"></a>Reed MB, Domenech P, Manca C, Su H, Barczak AK, et al.  (2004) A glycolipid of hypervirulent tuberculosis strains that inhibits the innate immune response. Nature  431: 84–87.  <ul class="find" data-citedArticleID="1089748" data-doi="10.1038/nature02837"><li><a href="http://dx.doi.org/10.1038/nature02837" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+glycolipid+of+hypervirulent+tuberculosis+strains+that+inhibits+the+innate+immune+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+glycolipid+of+hypervirulent+tuberculosis+strains+that+inhibits+the+innate+immune+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001973-Musser1" id="pone.0001973-Musser1"></a>Musser JM, Amin A, Ramaswamy S (2000) Negligible genetic diversity of mycobacterium tuberculosis host immune system protein targets: evidence of limited selective pressure. Genetics  155: 7–16.  <ul class="find" data-citedArticleID="1089740"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Negligible+genetic+diversity+of+mycobacterium+tuberculosis+host+immune+system+protein+targets%3A+evidence+of+limited+selective+pressure.&amp;auth=&amp;atitle=Negligible+genetic+diversity+of+mycobacterium+tuberculosis+host+immune+system+protein+targets%3A+evidence+of+limited+selective+pressure." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Negligible+genetic+diversity+of+mycobacterium+tuberculosis+host+immune+system+protein+targets%3A+evidence+of+limited+selective+pressure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Negligible+genetic+diversity+of+mycobacterium+tuberculosis+host+immune+system+protein+targets%3A+evidence+of+limited+selective+pressure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001973-Mostowy1" id="pone.0001973-Mostowy1"></a>Mostowy S, Cousins D, Brinkman J, Aranaz A, Behr MA (2002) Genomic deletions suggest a phylogeny for the Mycobacterium tuberculosis complex. J Infect Dis  186: 74–80.  <ul class="find" data-citedArticleID="1089738" data-doi="10.1086/341068"><li><a href="http://dx.doi.org/10.1086/341068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+deletions+suggest+a+phylogeny+for+the+Mycobacterium+tuberculosis+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+deletions+suggest+a+phylogeny+for+the+Mycobacterium+tuberculosis+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001973-Fleischmann1" id="pone.0001973-Fleischmann1"></a>Fleischmann RD, Alland D, Eisen JA, Carpenter L, White O, et al.  (2002) Whole-genome comparison of Mycobacterium tuberculosis clinical and laboratory strains. J Bacteriol  184: 5479–5490.  <ul class="find" data-citedArticleID="1089712" data-doi="10.1128/jb.184.19.5479-5490.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.19.5479-5490.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+comparison+of+Mycobacterium+tuberculosis+clinical+and+laboratory+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+comparison+of+Mycobacterium+tuberculosis+clinical+and+laboratory+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001973-WHO1" id="pone.0001973-WHO1"></a>  WHO WHO Report 2005. Global Tuberculosis Control - surveillance, planning, financing.  <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0001973-Glynn1" id="pone.0001973-Glynn1"></a>Glynn JR, Whiteley J, Bifani PJ, Kremer K, van Soolingen D (2002) Worldwide occurrence of Beijing/W strains of Mycobacterium tuberculosis: a systematic review. Emerg Infect Dis  8: 843–849.  <ul class="find" data-citedArticleID="1089722" data-doi="10.3201/eid0808.020002"><li><a href="http://dx.doi.org/10.3201/eid0808.020002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Worldwide+occurrence+of+Beijing%2FW+strains+of+Mycobacterium+tuberculosis%3A+a+systematic+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Worldwide+occurrence+of+Beijing%2FW+strains+of+Mycobacterium+tuberculosis%3A+a+systematic+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001973-Phyu1" id="pone.0001973-Phyu1"></a>Phyu S, Jureen R, Ti T, Dahle UR, Grewal HM (2003) Heterogeneity of Mycobacterium tuberculosis isolates in Yangon, Myanmar. J Clin Microbiol  41: 4907–4908.  <ul class="find" data-citedArticleID="1089742" data-doi="10.1128/jcm.41.10.4907-4908.2003"><li><a href="http://dx.doi.org/10.1128/jcm.41.10.4907-4908.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterogeneity+of+Mycobacterium+tuberculosis+isolates+in+Yangon%2C+Myanmar." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterogeneity+of+Mycobacterium+tuberculosis+isolates+in+Yangon%2C+Myanmar.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001973-Lillebaek1" id="pone.0001973-Lillebaek1"></a>Lillebaek T, Andersen AB, Dirksen A, Glynn JR, Kremer K (2003) Mycobacterium tuberculosis Beijing genotype. Emerg Infect Dis  9: 1553–1557.  <ul class="find" data-citedArticleID="1089732" data-doi="10.3201/eid0912.030276"><li><a href="http://dx.doi.org/10.3201/eid0912.030276" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mycobacterium+tuberculosis+Beijing+genotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mycobacterium+tuberculosis+Beijing+genotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001973-Bifani1" id="pone.0001973-Bifani1"></a>Bifani PJ, Mathema B, Kurepina NE, Kreiswirth BN (2002) Global dissemination of the Mycobacterium tuberculosis W-Beijing family strains. Trends Microbiol  10: 45–52.  <ul class="find" data-citedArticleID="1089694" data-doi="10.1016/s0966-842x(01)02277-6"><li><a href="http://dx.doi.org/10.1016/s0966-842x(01)02277-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+dissemination+of+the+Mycobacterium+tuberculosis+W-Beijing+family+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+dissemination+of+the+Mycobacterium+tuberculosis+W-Beijing+family+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001973-Kremer1" id="pone.0001973-Kremer1"></a>Kremer K, Glynn JR, Lillebaek T, Niemann S, Kurepina NE, et al.  (2004) Definition of the Beijing/W lineage of Mycobacterium tuberculosis on the basis of genetic markers. J Clin Microbiol  42: 4040–4049.  <ul class="find" data-citedArticleID="1089730" data-doi="10.1128/jcm.42.9.4040-4049.2004"><li><a href="http://dx.doi.org/10.1128/jcm.42.9.4040-4049.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Definition+of+the+Beijing%2FW+lineage+of+Mycobacterium+tuberculosis+on+the+basis+of+genetic+markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Definition+of+the+Beijing%2FW+lineage+of+Mycobacterium+tuberculosis+on+the+basis+of+genetic+markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001973-Tsolaki1" id="pone.0001973-Tsolaki1"></a>Tsolaki AG, Gagneux S, Pym AS, Goguet de la Salmoniere YO, Kreiswirth BN, et al.  (2005) Genomic deletions classify the Beijing/W strains as a distinct genetic lineage of Mycobacterium tuberculosis. J Clin Microbiol  43: 3185–3191.  <ul class="find" data-citedArticleID="1089756" data-doi="10.1128/jcm.43.7.3185-3191.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.7.3185-3191.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+deletions+classify+the+Beijing%2FW+strains+as+a+distinct+genetic+lineage+of+Mycobacterium+tuberculosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+deletions+classify+the+Beijing%2FW+strains+as+a+distinct+genetic+lineage+of+Mycobacterium+tuberculosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001973-Phyu2" id="pone.0001973-Phyu2"></a>Phyu S, Lwin T, Ti T, Maung W, Mar WW, et al.  (2005) Drug-resistant tuberculosis in Yangon, Myanmar. Scand J Infect Dis  37: 846–851.  <ul class="find" data-citedArticleID="1089744" data-doi="10.1080/00365540500265065"><li><a href="http://dx.doi.org/10.1080/00365540500265065" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Drug-resistant+tuberculosis+in+Yangon%2C+Myanmar." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Drug-resistant+tuberculosis+in+Yangon%2C+Myanmar.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001973-vanEmbden1" id="pone.0001973-vanEmbden1"></a>van Embden JDA, Schouls LM, van Soolingen D (1995) Molecular techniques: applications in epidemiologic studies. Mycobacterium bovis infection in animals and humans:. Ames, Iowa: Iowa State University Press. pp. 15–27.  <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0001973-Camus1" id="pone.0001973-Camus1"></a>Camus JC, Pryor MJ, Medigue C, Cole ST (2002) Re-annotation of the genome sequence of Mycobacterium tuberculosis H37Rv. Microbiology  148: 2967–2973.  <ul class="find" data-citedArticleID="1089700"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Re-annotation+of+the+genome+sequence+of+Mycobacterium+tuberculosis+H37Rv.&amp;auth=&amp;atitle=Re-annotation+of+the+genome+sequence+of+Mycobacterium+tuberculosis+H37Rv." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Re-annotation+of+the+genome+sequence+of+Mycobacterium+tuberculosis+H37Rv." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Re-annotation+of+the+genome+sequence+of+Mycobacterium+tuberculosis+H37Rv.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001973-Stewart1" id="pone.0001973-Stewart1"></a>Stewart GR, Wernisch L, Stabler R, Mangan JA, Hinds J, et al.  (2002) Dissection of the heat-shock response in Mycobacterium tuberculosis using mutants and microarrays. Microbiology  148: 3129–3138.  <ul class="find" data-citedArticleID="1089752"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Dissection+of+the+heat-shock+response+in+Mycobacterium+tuberculosis+using+mutants+and+microarrays.&amp;auth=&amp;atitle=Dissection+of+the+heat-shock+response+in+Mycobacterium+tuberculosis+using+mutants+and+microarrays." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dissection+of+the+heat-shock+response+in+Mycobacterium+tuberculosis+using+mutants+and+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dissection+of+the+heat-shock+response+in+Mycobacterium+tuberculosis+using+mutants+and+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001973-Hinds1" id="pone.0001973-Hinds1"></a>Hinds JLKG, Mangan JA, Butcher PD (2002)  Glass slide microarrays for bacterial genomes. In: WBWaD N, editor. Methods in Microbiology: Functional Microbial Genomics. London: Academic Press. pp. 83–99.  <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0001973-Bolstad1" id="pone.0001973-Bolstad1"></a>Bolstad BM, Irizarry RA, Astrand M, Speed TP (2003) A comparison of normalization methods for high density oligonucleotide array data based on variance and bias. Bioinformatics  19: 185–193.  <ul class="find" data-citedArticleID="1089696" data-doi="10.1093/bioinformatics/19.2.185"><li><a href="http://dx.doi.org/10.1093/bioinformatics/19.2.185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+normalization+methods+for+high+density+oligonucleotide+array+data+based+on+variance+and+bias." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+normalization+methods+for+high+density+oligonucleotide+array+data+based+on+variance+and+bias.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001973-Huber1" id="pone.0001973-Huber1"></a>Huber W, von Heydebreck A, Sultmann H, Poustka A, Vingron M (2002) Variance stabilization applied to microarray data calibration and to the quantification of differential expression. Bioinformatics  18: Suppl 1S96–104.  <ul class="find" data-citedArticleID="1089728" data-doi="10.1093/bioinformatics/18.suppl_1.s96"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.suppl_1.s96" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variance+stabilization+applied+to+microarray+data+calibration+and+to+the+quantification+of+differential+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variance+stabilization+applied+to+microarray+data+calibration+and+to+the+quantification+of+differential+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001973-Rocke1" id="pone.0001973-Rocke1"></a>Rocke DM, Durbin B (2003) Approximate variance-stabilizing transformations for gene-expression microarray data. Bioinformatics  19: 966–972.  <ul class="find" data-citedArticleID="1089750" data-doi="10.1093/bioinformatics/btg107"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Approximate+variance-stabilizing+transformations+for+gene-expression+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Approximate+variance-stabilizing+transformations+for+gene-expression+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001973-Warren1" id="pone.0001973-Warren1"></a>Warren RM, Victor TC, Streicher EM, Richardson M, Beyers N, et al.  (2004) Patients with active tuberculosis often have different strains in the same sputum specimen. Am J Respir Crit Care Med  169: 610–614.  <ul class="find" data-citedArticleID="1089760" data-doi="10.1164/rccm.200305-714oc"><li><a href="http://dx.doi.org/10.1164/rccm.200305-714oc" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patients+with+active+tuberculosis+often+have+different+strains+in+the+same+sputum+specimen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patients+with+active+tuberculosis+often+have+different+strains+in+the+same+sputum+specimen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001973-Supply1" id="pone.0001973-Supply1"></a>Supply P, Allix C, Lesjean S, Cardoso-Oelemann M, Rusch-Gerdes S, et al.  (2006) Proposal for standardization of optimized mycobacterial interspersed repetitive unit-variable-number tandem repeat typing of Mycobacterium tuberculosis. J Clin Microbiol  44: 4498–4510.  <ul class="find" data-citedArticleID="1089754" data-doi="10.1128/jcm.01392-06"><li><a href="http://dx.doi.org/10.1128/jcm.01392-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proposal+for+standardization+of+optimized+mycobacterial+interspersed+repetitive+unit-variable-number+tandem+repeat+typing+of+Mycobacterium+tuberculosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proposal+for+standardization+of+optimized+mycobacterial+interspersed+repetitive+unit-variable-number+tandem+repeat+typing+of+Mycobacterium+tuberculosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001973-Constant1" id="pone.0001973-Constant1"></a>Constant P, Perez E, Malaga W, Laneelle MA, Saurel O, et al.  (2002) Role of the pks15/1 gene in the biosynthesis of phenolglycolipids in the Mycobacterium tuberculosis complex. Evidence that all strains synthesize glycosylated p-hydroxybenzoic methyl esters and that strains devoid of phenolglycolipids harbor a frameshift mutation in the pks15/1 gene. J Biol Chem  277: 38148–38158.  <ul class="find" data-citedArticleID="1089706" data-doi="10.1074/jbc.m206538200"><li><a href="http://dx.doi.org/10.1074/jbc.m206538200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+pks15%2F1+gene+in+the+biosynthesis+of+phenolglycolipids+in+the+Mycobacterium+tuberculosis+complex.+Evidence+that+all+strains+synthesize+glycosylated+p-hydroxybenzoic+methyl+esters+and+that+strains+devoid+of+phenolglycolipids+harbor+a+frameshift+mutation+in+the+pks15%2F1+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+pks15%2F1+gene+in+the+biosynthesis+of+phenolglycolipids+in+the+Mycobacterium+tuberculosis+complex.+Evidence+that+all+strains+synthesize+glycosylated+p-hydroxybenzoic+methyl+esters+and+that+strains+devoid+of+phenolglycolipids+harbor+a+frameshift+mutation+in+the+pks15%2F1+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001973-Chan1" id="pone.0001973-Chan1"></a>Chan MY, Borgdorff M, Yip CW, de Haas PE, Wong WS, et al.  (2001) Seventy percent of the Mycobacterium tuberculosis isolates in Hong Kong represent the Beijing genotype. Epidemiol Infect  127: 169–171.  <ul class="find" data-citedArticleID="1089702" data-doi="10.1017/s0950268801005659"><li><a href="http://dx.doi.org/10.1017/s0950268801005659" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Seventy+percent+of+the+Mycobacterium+tuberculosis+isolates+in+Hong+Kong+represent+the+Beijing+genotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Seventy+percent+of+the+Mycobacterium+tuberculosis+isolates+in+Hong+Kong+represent+the+Beijing+genotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001973-vanSoolingen1" id="pone.0001973-vanSoolingen1"></a>van Soolingen D, Qian L, de Haas PE, Douglas JT, Traore H, et al.  (1995) Predominance of a single genotype of Mycobacterium tuberculosis in countries of east Asia. J Clin Microbiol  33: 3234–3238.  <ul class="find" data-citedArticleID="1089764"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Predominance+of+a+single+genotype+of+Mycobacterium+tuberculosis+in+countries+of+east+Asia.&amp;auth=&amp;atitle=Predominance+of+a+single+genotype+of+Mycobacterium+tuberculosis+in+countries+of+east+Asia." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predominance+of+a+single+genotype+of+Mycobacterium+tuberculosis+in+countries+of+east+Asia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predominance+of+a+single+genotype+of+Mycobacterium+tuberculosis+in+countries+of+east+Asia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001973-Filliol1" id="pone.0001973-Filliol1"></a>Filliol I, Driscoll JR, Van Soolingen D, Kreiswirth BN, Kremer K, et al.  (2002) Global distribution of Mycobacterium tuberculosis spoligotypes. Emerg Infect Dis  8: 1347–1349.  <ul class="find" data-citedArticleID="1089710" data-doi="10.3201/eid0811.020125"><li><a href="http://dx.doi.org/10.3201/eid0811.020125" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+distribution+of+Mycobacterium+tuberculosis+spoligotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+distribution+of+Mycobacterium+tuberculosis+spoligotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001973-Espitia1" id="pone.0001973-Espitia1"></a>Espitia C, Laclette JP, Mondragon-Palomino M, Amador A, Campuzano J, et al.  (1999) The PE-PGRS glycine-rich proteins of Mycobacterium tuberculosis: a new family of fibronectin-binding proteins? Microbiology  145 ( Pt 12): 3487–3495.  <ul class="find" data-citedArticleID="1089708"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+PE-PGRS+glycine-rich+proteins+of+Mycobacterium+tuberculosis%3A+a+new+family+of+fibronectin-binding+proteins%3F&amp;auth=&amp;atitle=The+PE-PGRS+glycine-rich+proteins+of+Mycobacterium+tuberculosis%3A+a+new+family+of+fibronectin-binding+proteins%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+PE-PGRS+glycine-rich+proteins+of+Mycobacterium+tuberculosis%3A+a+new+family+of+fibronectin-binding+proteins%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+PE-PGRS+glycine-rich+proteins+of+Mycobacterium+tuberculosis%3A+a+new+family+of+fibronectin-binding+proteins%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001973-Cole1" id="pone.0001973-Cole1"></a>Cole ST, Brosch R, Parkhill J, Garnier T, Churcher C, et al.  (1998) Deciphering the biology of Mycobacterium tuberculosis from the complete genome sequence. Nature  393: 537–544.  <ul class="find" data-citedArticleID="1089704" data-doi="10.1038/31159"><li><a href="http://dx.doi.org/10.1038/31159" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deciphering+the+biology+of+Mycobacterium+tuberculosis+from+the+complete+genome+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deciphering+the+biology+of+Mycobacterium+tuberculosis+from+the+complete+genome+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001973-Garnier1" id="pone.0001973-Garnier1"></a>Garnier T, Eiglmeier K, Camus JC, Medina N, Mansoor H, et al.  (2003) The complete genome sequence of Mycobacterium bovis. Proc Natl Acad Sci U S A  100: 7877–7882.  <ul class="find" data-citedArticleID="1089718" data-doi="10.1073/pnas.1130426100"><li><a href="http://dx.doi.org/10.1073/pnas.1130426100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+genome+sequence+of+Mycobacterium+bovis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+genome+sequence+of+Mycobacterium+bovis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001973-Gagneux1" id="pone.0001973-Gagneux1"></a>Gagneux S, DeRiemer K, Van T, Kato-Maeda M, de Jong BC, et al.  (2006) Variable host-pathogen compatibility in Mycobacterium tuberculosis. Proc Natl Acad Sci U S A  103: 2869–2873.  <ul class="find" data-citedArticleID="1089716" data-doi="10.1073/pnas.0511240103"><li><a href="http://dx.doi.org/10.1073/pnas.0511240103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variable+host-pathogen+compatibility+in+Mycobacterium+tuberculosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variable+host-pathogen+compatibility+in+Mycobacterium+tuberculosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001973-Gutierrez1" id="pone.0001973-Gutierrez1"></a>Gutierrez MC, Ahmed N, Willery E, Narayanan S, Hasnain SE, et al.  (2006) Predominance of ancestral lineages of Mycobacterium tuberculosis in India. Emerg Infect Dis  12: 1367–1374.  <ul class="find" data-citedArticleID="1089724" data-doi="10.3201/eid1209.050017"><li><a href="http://dx.doi.org/10.3201/eid1209.050017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predominance+of+ancestral+lineages+of+Mycobacterium+tuberculosis+in+India." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predominance+of+ancestral+lineages+of+Mycobacterium+tuberculosis+in+India.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001973-Betts1" id="pone.0001973-Betts1"></a>Betts JC, McLaren A, Lennon MG, Kelly FM, Lukey PT, et al.  (2003) Signature gene expression profiles discriminate between isoniazid-, thiolactomycin-, and triclosan-treated Mycobacterium tuberculosis. Antimicrob Agents Chemother  47: 2903–2913.  <ul class="find" data-citedArticleID="1089692" data-doi="10.1128/aac.47.9.2903-2913.2003"><li><a href="http://dx.doi.org/10.1128/aac.47.9.2903-2913.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signature+gene+expression+profiles+discriminate+between+isoniazid-%2C+thiolactomycin-%2C+and+triclosan-treated+Mycobacterium+tuberculosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signature+gene+expression+profiles+discriminate+between+isoniazid-%2C+thiolactomycin-%2C+and+triclosan-treated+Mycobacterium+tuberculosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001973-Brosch1" id="pone.0001973-Brosch1"></a>Brosch R, Gordon SV, Marmiesse M, Brodin P, Buchrieser C, et al.  (2002) A new evolutionary scenario for the Mycobacterium tuberculosis complex. Proc Natl Acad Sci U S A  99: 3684–3689.  <ul class="find" data-citedArticleID="1089698" data-doi="10.1073/pnas.052548299"><li><a href="http://dx.doi.org/10.1073/pnas.052548299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+evolutionary+scenario+for+the+Mycobacterium+tuberculosis+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+evolutionary+scenario+for+the+Mycobacterium+tuberculosis+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001973-Rao1" id="pone.0001973-Rao1"></a>Rao KR, Kauser F, Srinivas S, Zanetti S, Sechi LA, et al.  (2005) Analysis of genomic downsizing on the basis of region-of-difference polymorphism profiling of Mycobacterium tuberculosis patient isolates reveals geographic partitioning. J Clin Microbiol  43: 5978–5982.  <ul class="find" data-citedArticleID="1089746" data-doi="10.1128/jcm.43.12.5978-5982.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.12.5978-5982.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+genomic+downsizing+on+the+basis+of+region-of-difference+polymorphism+profiling+of+Mycobacterium+tuberculosis+patient+isolates+reveals+geographic+partitioning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+genomic+downsizing+on+the+basis+of+region-of-difference+polymorphism+profiling+of+Mycobacterium+tuberculosis+patient+isolates+reveals+geographic+partitioning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.XML" value="109280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.PDF" value="612074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.t001.PNG_L" value="421002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.t001.PNG_M" value="149037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.t001.PNG_S" value="17156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.t001.TIF" value="2207238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.t001.PNG_I" value="88001"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g001.PNG_L" value="224197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g001.PNG_M" value="105516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g001.PNG_S" value="10180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g001.TIF" value="1576062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g001.PNG_I" value="44750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g002.PNG_L" value="1261027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g002.PNG_M" value="127578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g002.PNG_S" value="14253"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g002.TIF" value="4398616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g002.PNG_I" value="55187"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g003.PNG_L" value="1648439"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g003.PNG_M" value="130099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g003.PNG_S" value="10458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g003.TIF" value="4213074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g003.PNG_I" value="109884"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g004.PNG_L" value="4069714"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g004.PNG_M" value="250251"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g004.PNG_S" value="10390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g004.TIF" value="6405576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001973.g004.PNG_I" value="85204"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001973&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001973" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001973&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001973&volume=&issue=&title=Genomic Diversity among Beijing and non-Beijing Mycobacterium tuberculosis Isolates from Myanmar&author_name=Ruth%20Stavrum%2C%20H%C3%A5vard%20Valvatne%2C%20Trond%20H.%20B%C3%B8%2C%20Inge%20Jonassen%2C%20Jason%20Hinds%2C%20Philip%20D.%20Butcher%2C%20Harleen%20M.%20S.%20Grewal&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001973" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001973" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001973" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001973&amp;t=Genomic%20Diversity%20among%20Beijing%20and%20non-Beijing%20Mycobacterium%20tuberculosis%20Isolates%20from%20Myanmar" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001973&title=Genomic%20Diversity%20among%20Beijing%20and%20non-Beijing%20Mycobacterium%20tuberculosis%20Isolates%20from%20Myanmar&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001973&amp;title=Genomic%20Diversity%20among%20Beijing%20and%20non-Beijing%20Mycobacterium%20tuberculosis%20Isolates%20from%20Myanmar" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001973" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001973&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genomic Diversity among Beijing and non-Beijing Mycobacterium tuberculosis Isolates from Myanmar';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001973';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genomic%20Diversity%20among%20Beijing%20and%20non-Beijing%20Mycobacterium%20tuberculosis%20Isolates%20from%20Myanmar http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001973" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001973" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bovine+tuberculosis%22" title="Search for articles in the subject area:'Bovine tuberculosis'"><div class="flagText">Bovine tuberculosis</div></a>
              <div data-categoryid="49279" data-articleid="26770"
                   data-categoryname="Bovine tuberculosis"
                   class="flagImage" title="Flag 'Bovine tuberculosis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="26770"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Myanmar%22" title="Search for articles in the subject area:'Myanmar'"><div class="flagText">Myanmar</div></a>
              <div data-categoryid="41213" data-articleid="26770"
                   data-categoryname="Myanmar"
                   class="flagImage" title="Flag 'Myanmar' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mycobacterium+bovis%22" title="Search for articles in the subject area:'Mycobacterium bovis'"><div class="flagText">Mycobacterium bovis</div></a>
              <div data-categoryid="42909" data-articleid="26770"
                   data-categoryname="Mycobacterium bovis"
                   class="flagImage" title="Flag 'Mycobacterium bovis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mycobacterium+tuberculosis%22" title="Search for articles in the subject area:'Mycobacterium tuberculosis'"><div class="flagText">Mycobacterium tuberculosis</div></a>
              <div data-categoryid="42725" data-articleid="26770"
                   data-categoryname="Mycobacterium tuberculosis"
                   class="flagImage" title="Flag 'Mycobacterium tuberculosis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="26770"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Restriction+fragment+length+polymorphism+analysis%22" title="Search for articles in the subject area:'Restriction fragment length polymorphism analysis'"><div class="flagText">Restriction fragment length polymorphism analysis</div></a>
              <div data-categoryid="50165" data-articleid="26770"
                   data-categoryname="Restriction fragment length polymorphism analysis"
                   class="flagImage" title="Flag 'Restriction fragment length polymorphism analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Tuberculosis%22" title="Search for articles in the subject area:'Tuberculosis'"><div class="flagText">Tuberculosis</div></a>
              <div data-categoryid="47969" data-articleid="26770"
                   data-categoryname="Tuberculosis"
                   class="flagImage" title="Flag 'Tuberculosis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8457'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=811'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9907&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=1515">Referee Comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=21137">Academic Editor Comments: Leonardo Sechi</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
