

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Horizontal Transfer of a Nitrate Assimilation Gene Cluster and Ecological Transitions in Fungi: A Phylogenetic Study</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001097"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001097" />

    <meta name="citation_title" content="Horizontal Transfer of a Nitrate Assimilation Gene Cluster and Ecological Transitions in Fungi: A Phylogenetic Study"/>
    <meta itemprop="name" content="Horizontal Transfer of a Nitrate Assimilation Gene Cluster and Ecological Transitions in Fungi: A Phylogenetic Study"/>

      <meta name="citation_author" content="Jason C. Slot"/>
            <meta name="citation_author_institution" content="Department of Biology, Clark University, Worcester, Massachusetts, United States of America"/>
      <meta name="citation_author" content="David S. Hibbett"/>
            <meta name="citation_author_institution" content="Department of Biology, Clark University, Worcester, Massachusetts, United States of America"/>

    <meta name="citation_date" content="2007/10/31"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001097.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1097"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Isolation and characterisation of the crnA-niiA-niaD gene cluster for nitrate assimilation in Aspergillus nidulans.; citation_author=IL Johnstone; citation_author=PC McCabe; citation_author=P Greaves; citation_author=SJ Gurr; citation_author=GE Cole; citation_journal_title=Gene; citation_volume=90; citation_number=1; citation_pages=181-192; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=The genes YNI1 and YNR1, encoding nitrite reductase and nitrate reductase respectively in the yeast Hansenula polymorpha, are clustered and co-ordinately regulated.; citation_author=N Brito; citation_author=J Avila; citation_author=MD Perez; citation_author=C Gonzalez; citation_author=JM Siverio; citation_journal_title=Biochem J; citation_volume=317; citation_number=2; citation_pages=89-95; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Characterisation and expression analysis of a nitrate transporter and nitrite reductase genes, two members of a gene cluster for nitrate assimilation from the symbiotic basidiomycete Hebeloma cylindrosporum.; citation_author=P Jargeat; citation_author=D Rekangalt; citation_author=MC Verner; citation_author=G Gay; citation_author=JC Debaud; citation_journal_title=Curr Genet; citation_volume=43; citation_number=3; citation_pages=199-205; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetic regulation of nitrogen metabolism in the fungi.; citation_author=GA Marzluf; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=61; citation_number=4; citation_pages=17-32; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Diversification of NRT2 and the Origin of Its Fungal Homolog.; citation_author=JC Slot; citation_author=K Hallstrom; citation_author=PB Matheny; citation_author=DS Hibbett; citation_journal_title=Mol Biol Evol; citation_number=5; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Operon formation is driven by co-regulation and not by horizontal gene transfer.; citation_author=MN Price; citation_author=KH Huang; citation_author=AP Arkin; citation_author=EJ Alm; citation_journal_title=Genome Res; citation_volume=15; citation_number=6; citation_pages=809-819; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Selfish operons and speciation by gene transfer.; citation_author=JG Lawrence; citation_journal_title=Trends Microbiol; citation_volume=5; citation_number=7; citation_pages=355-359; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Selfish operons: the evolutionary impact of gene clustering in prokaryotes and eukaryotes.; citation_author=J Lawrence; citation_journal_title=Curr Opin Genet Dev; citation_volume=9; citation_number=8; citation_pages=642-648; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Role of Horizontal Gene Transfer in the Evolution of Fungi.; citation_author=UL Rosewich; citation_author=HC Kistler; citation_journal_title=Annu Rev Phytopathol; citation_volume=38; citation_number=9; citation_pages=325-363; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Horizontal gene transfer and the evolution of secondary metabolite gene clusters in fungi: an hypothesis.; citation_author=JD Walton; citation_journal_title=Fungal Genet Biol; citation_volume=30; citation_number=10; citation_pages=167-171; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Phylogenomic analysis of type I polyketide synthase genes in pathogenic and saprobic ascomycetes.; citation_author=S Kroken; citation_author=NL Glass; citation_author=JW Taylor; citation_author=OC Yoder; citation_author=BG Turgeon; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=11; citation_pages=15670-15675; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genes determining pathogenicity to pea are clustered on a supernumerary chromosome in the fungal plant pathogen Nectria haematococca.; citation_author=Y Han; citation_author=X Liu; citation_author=U Benny; citation_author=HC Kistler; citation_author=HD VanEtten; citation_journal_title=Plant J; citation_volume=25; citation_number=12; citation_pages=305-314; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Recent evolution of the human pathogen Cryptococcus neoformans by intervarietal transfer of a 14-gene fragment.; citation_author=LA Kavanaugh; citation_author=JA Fraser; citation_author=FS Dietrich; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=13; citation_pages=1879-1890; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transfer of a supernumerary chromosome between vegetatively incompatible biotypes of the fungus Colletotrichum gloeosporioides.; citation_author=C He; citation_author=AG Rusu; citation_author=AM Poplawski; citation_author=JA Irwin; citation_author=JM Manners; citation_journal_title=Genetics; citation_volume=150; citation_number=14; citation_pages=1459-1466; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Nitrite reductase gene from Synechococcus sp. PCC 7942: homology between cyanobacterial and higher-plant nitrite reductases.; citation_author=I Luque; citation_author=E Flores; citation_author=A Herrero; citation_journal_title=Plant Mol Biol; citation_volume=21; citation_number=15; citation_pages=1201-1205; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Evolution of nitrate reductase: molecular and structural variations on a common function.; citation_author=JF Stolz; citation_author=P Basu; citation_journal_title=Chembiochem; citation_volume=3; citation_number=16; citation_pages=198-206; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Reconstructing the early evolution of Fungi using a six-gene phylogeny.; citation_author=TY James; citation_author=F Kauff; citation_author=CL Schoch; citation_author=PB Matheny; citation_author=V Hofstetter; citation_journal_title=Nature; citation_volume=443; citation_number=17; citation_pages=818-822; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulation of biomass-degrading enzymes in the filamentous fungus Trichoderma reesei.; citation_author=PK Foreman; citation_author=D Brown; citation_author=L Dankmeyer; citation_author=R Dean; citation_author=S Diener; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=18; citation_pages=31988-31997; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Role of the bga1-encoded extracellular {beta}-galactosidase of Hypocrea jecorina in cellulase induction by lactose.; citation_author=B Seiboth; citation_author=L Hartl; citation_author=N Salovuori; citation_author=K Lanthaler; citation_author=GD Robson; citation_journal_title=Appl Environ Microbiol; citation_volume=71; citation_number=19; citation_pages=851-857; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome analysis of the smallest free-living eukaryote Ostreococcus tauri unveils many unique features.; citation_author=E Derelle; citation_author=C Ferraz; citation_author=S Rombauts; citation_author=P Rouze; citation_author=AZ Worden; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=20; citation_pages=11647-11652; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Five nitrate assimilation-related loci are clustered in Chlamydomonas reinhardtii.; citation_author=A Quesada; citation_author=A Galvan; citation_author=RA Schnell; citation_author=PA Lefebvre; citation_author=E Fernandez; citation_journal_title=Mol Gen Genet; citation_volume=240; citation_number=21; citation_pages=387-394; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Fungal pathogenesis: gene clusters unveiled as secrets within the Ustilago maydis code.; citation_author=BJ Howlett; citation_author=A Idnurm; citation_author=J Heitman; citation_journal_title=Curr Biol; citation_volume=17; citation_number=22; citation_pages=R87-90; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning and characterization of an ML-236B (compactin) biosynthetic gene cluster in Penicillium citrinum.; citation_author=Y Abe; citation_author=T Suzuki; citation_author=C Ono; citation_author=K Iwamoto; citation_author=M Hosobuchi; citation_journal_title=Mol Genet Genomics; citation_volume=267; citation_number=23; citation_pages=636-646; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A complex ergovaline gene cluster in epichloe endophytes of grasses.; citation_author=DJ Fleetwood; citation_author=B Scott; citation_author=GA Lane; citation_author=A Tanaka; citation_author=RD Johnson; citation_journal_title=Appl Environ Microbiol; citation_volume=73; citation_number=24; citation_pages=2571-2579; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Indole-diterpene gene cluster from Aspergillus flavus.; citation_author=S Zhang; citation_author=BJ Monahan; citation_author=JS Tkacz; citation_author=B Scott; citation_journal_title=Appl Environ Microbiol; citation_volume=70; citation_number=25; citation_pages=6875-6883; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Production of penicillin by fungi growing on food products: identification of a complete penicillin gene cluster in Penicillium griseofulvum and a truncated cluster in Penicillium verrucosum.; citation_author=F Laich; citation_author=F Fierro; citation_author=JF Martin; citation_journal_title=Appl Environ Microbiol; citation_volume=68; citation_number=26; citation_pages=1211-1219; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Recombination, balancing selection and adaptive evolution in the aflatoxin gene cluster of Aspergillus parasiticus.; citation_author=I Carbone; citation_author=JL Jakobek; citation_author=JH Ramirez-Prado; citation_author=BW Horn; citation_journal_title=Mol Ecol; citation_number=27; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Birth of a metabolic gene cluster in yeast by adaptive gene relocation.; citation_author=S Wong; citation_author=KH Wolfe; citation_journal_title=Nat Genet; citation_volume=37; citation_number=28; citation_pages=777-782; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Trichoderma: Systematics, the Sexual State, and Ecology.; citation_author=GJ Samuels; citation_journal_title=Phytopathology; citation_volume=96; citation_number=29; citation_pages=195-206; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sexually competent, sucrose- and nitrate-assimilating strains of Hypocrea jecorina (Trichoderma reesei) from South American soils. .; citation_author=E Lieckfeldt; citation_author=C Kullnig; citation_author=GJ Samuels; citation_author=CP Kubicek; citation_journal_title=Mycologia; citation_volume=92; citation_number=30; citation_pages=374-380; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Influence of nitrogen sources on cellulase biosynthesis by a mutant strain Trichoderma viride 44.; citation_author=NA Ostrikova; citation_author=SA Konovalov; citation_journal_title=Applied Biochemistry and Microbiology; citation_volume=19; citation_number=31; citation_pages=392-395; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Effect of Certain Carbon and Nitrogen Sources on the Antagonistic Activities of some Biocontrol Agents against Colletotrichum falcatum Went.; citation_author=R Saikia; citation_author=P Azad; citation_journal_title=Environment and Ecology; citation_volume=19; citation_number=32; citation_pages=849-852; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Effect of certain nitrogenous sources on the in vitro growth, sporulation and production of antifungal substances by Trichoderma harzianum.; citation_author=J Jayaraj; citation_author=R Ramabadran; citation_journal_title=Journal of Mycology and Plant Pathology; citation_volume=28; citation_number=33; citation_pages=23-25; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Saprophytic growth in soil of a strain of Trichoderma koningii.; citation_author=SA Wakelin; citation_author=K Sivasithamparam; citation_author=ALJ Cole; citation_author=RA Skipp; citation_journal_title=New Zealand Journal of Agricultural Research; citation_volume=42; citation_number=34; citation_pages=337-345; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Competition for ammonium and nitrate forms of nitrogen between some phytopathogenic and antagonistic soil fungi.; citation_author=F Celar; citation_journal_title=Biological Control; citation_volume=28; citation_number=35; citation_pages=19-24; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Trichoderma atroviride G-protein alpha-subunit gene tga1 is involved in mycoparasitic coiling and conidiation.; citation_author=V Rocha-Ramirez; citation_author=C Omero; citation_author=I Chet; citation_author=BA Horwitz; citation_author=A Herrera-Estrella; citation_journal_title=Eukaryot Cell; citation_volume=1; citation_number=36; citation_pages=594-605; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Histopathological studies of sclerotia of phytopathogenic fungi parasitized by a GFP transformed Trichoderma virens antagonistic strain.; citation_author=S Sarrocco; citation_author=L Mikkelsen; citation_author=M Vergara; citation_author=DF Jensen; citation_author=M Lubeck; citation_journal_title=Mycol Res; citation_volume=110; citation_number=37; citation_pages=179-187; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Large-scale gene discovery in the oomycete Phytophthora infestans reveals likely components of phytopathogenicity shared with true fungi.; citation_author=TA Randall; citation_author=RA Dwyer; citation_author=E Huitema; citation_author=K Beyer; citation_author=C Cvitanich; citation_journal_title=Mol Plant Microbe Interact; citation_volume=18; citation_number=38; citation_pages=229-243; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolution of filamentous plant pathogens: gene exchange across eukaryotic kingdoms.; citation_author=TA Richards; citation_author=JB Dacks; citation_author=JM Jenkinson; citation_author=CR Thornton; citation_author=NJ Talbot; citation_journal_title=Curr Biol; citation_volume=16; citation_number=39; citation_pages=1857-1864; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microbial activity as a major factor in the mobilization of iron in the humid tropics.; citation_author=SJ Stemmler; citation_author=J Berthelin; citation_journal_title=European Journal of Soil Science; citation_volume=54; citation_number=40; citation_pages=725-733; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A mesoscale iron enrichment in the western subarctic Pacific induces a large centric diatom bloom.; citation_author=A Tsuda; citation_author=S Takeda; citation_author=H Saito; citation_author=J Nishioka; citation_author=Y Nojiri; citation_journal_title=Science; citation_volume=300; citation_number=41; citation_pages=958-961; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=How does a symbiotic fungus modulate expression of its host-plant nitrite reductase?; citation_author=J Bailly; citation_author=JC Debaud; citation_author=MC Verner; citation_author=C Plassard; citation_author=M Chalot; citation_journal_title=New Phytol; citation_volume=175; citation_number=42; citation_pages=155-165; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Utilization of microbial siderophores by mycorrhizal and non-mycorrhizal pine roots.; citation_author=C Leyval; citation_author=CPP Reid; citation_journal_title=New Phytologist; citation_volume=119; citation_number=43; citation_pages=93-98; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The GATA factor AreA is essential for chromatin remodelling in a eukaryotic bidirectional promoter.; citation_author=MI Muro-Pastor; citation_author=R Gonzalez; citation_author=J Strauss; citation_author=F Narendja; citation_author=C Scazzocchio; citation_journal_title=Embo J; citation_volume=18; citation_number=44; citation_pages=1584-1597; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The intergenic region between the divergently transcribed niiA and niaD genes of Aspergillus nidulans contains multiple NirA binding sites which act bidirectionally.; citation_author=PJ Punt; citation_author=J Strauss; citation_author=R Smit; citation_author=JR Kinghorn; citation_author=CA van den Hondel; citation_journal_title=Mol Cell Biol; citation_volume=15; citation_number=45; citation_pages=5688-5699; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Molecular evidence for the early colonization of land by fungi and plants.; citation_author=DS Heckman; citation_author=DM Geiser; citation_author=BR Eidell; citation_author=RL Stauffer; citation_author=NL Kardos; citation_journal_title=Science; citation_volume=293; citation_number=46; citation_pages=1129-1133; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=47; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.; citation_author=JD Thompson; citation_author=TJ Gibson; citation_author=F Plewniak; citation_author=F Jeanmougin; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=48; citation_pages=4876-4882; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=PAUP*. Phylogenetic Analysis Using Parsimony (*and Other Methods). Version 4.; citation_author=DL Swofford; citation_number=49; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MRBAYES: Bayesian inference of phylogenetic trees.; citation_author=JP Huelsenbeck; citation_author=F Ronquist; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=50; citation_pages=754-755; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=51; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Parallel Metropolis coupled Markov chain Monte Carlo for Bayesian phylogenetic inference.; citation_author=G Altekar; citation_author=S Dwarkadas; citation_author=JP Huelsenbeck; citation_author=F Ronquist; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=52; citation_pages=407-415; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models.; citation_author=A Stamatakis; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=53; citation_pages=2688-2690; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Multiple Comparisons of Log-Likelihoods with Applications to Phylogenetic Inference.; citation_author=H Shimodaira; citation_author=M Hasegawa; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=54; citation_pages=1114-1116; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing.; citation_author=HA Schmidt; citation_author=K Strimmer; citation_author=M Vingron; citation_author=A von Haeseler; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=55; citation_pages=502-504; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform.; citation_author=K Katoh; citation_author=K Misawa; citation_author=K Kuma; citation_author=T Miyata; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=56; citation_pages=3059-3066; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The deep roots of eukaryotes.; citation_author=SL Baldauf; citation_journal_title=Science; citation_volume=300; citation_number=57; citation_pages=1703-1706; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001097" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Horizontal Transfer of a Nitrate Assimilation Gene Cluster and Ecological Transitions in Fungi: A Phylogenetic Study"/>
    <meta name="twitter:description" content="High affinity nitrate assimilation genes in fungi occur in a cluster (fHANT-AC) that can be coordinately regulated. The clustered genes include nrt2, which codes for a high affinity nitrate transporter; euknr, which codes for nitrate reductase; and NAD(P)H-nir, which codes for nitrite reductase. Homologs of genes in the fHANT-AC occur in other eukaryotes and prokaryotes, but they have only been found clustered in the oomycete Phytophthora (heterokonts). We performed independent and concatenated phylogenetic analyses of homologs of all three genes in the fHANT-AC. Phylogenetic analyses limited to fungal sequences suggest that the fHANT-AC has been transferred horizontally from a basidiomycete (mushrooms and smuts) to an ancestor of the ascomycetous mold Trichoderma reesei. Phylogenetic analyses of sequences from diverse eukaryotes and eubacteria, and cluster structure, are consistent with a hypothesis that the fHANT-AC was assembled in a lineage leading to the oomycetes and was subsequently transferred to the Dikarya (Ascomycota+Basidiomycota), which is a derived fungal clade that includes the vast majority of terrestrial fungi. We propose that the acquisition of high affinity nitrate assimilation contributed to the success of Dikarya on land by allowing exploitation of nitrate in aerobic soils, and the subsequent transfer of a complete assimilation cluster improved the fitness of T. reesei in a new niche. Horizontal transmission of this cluster of functionally integrated genes supports the &ldquo;selfish operon&rdquo; hypothesis for maintenance of gene clusters."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001097.g002"/>

  <meta property="og:title" content="Horizontal Transfer of a Nitrate Assimilation Gene Cluster and Ecological Transitions in Fungi: A Phylogenetic Study" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001097" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3119'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1155'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7483&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001097">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001097" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001097&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Horizontal Transfer of a Nitrate Assimilation Gene Cluster and Ecological Transitions in Fungi: A Phylogenetic Study
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jason C. Slot
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jslot@clarku.edu">jslot@clarku.edu</a></p>

                <p>Affiliation:
                  Department of Biology, Clark University, Worcester, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David S. Hibbett
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Clark University, Worcester, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 31, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001097</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001097" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001097" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001097" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001097" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001097-g001" data-doi="info:doi/10.1371/journal.pone.0001097" data-uri="info:doi/10.1371/journal.pone.0001097.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001097.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001097-t001" data-doi="info:doi/10.1371/journal.pone.0001097" data-uri="info:doi/10.1371/journal.pone.0001097.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001097.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001097-g002" data-doi="info:doi/10.1371/journal.pone.0001097" data-uri="info:doi/10.1371/journal.pone.0001097.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001097.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001097-t002" data-doi="info:doi/10.1371/journal.pone.0001097" data-uri="info:doi/10.1371/journal.pone.0001097.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001097.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001097">Reader Comments (5)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001097" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>High affinity nitrate assimilation genes in fungi occur in a cluster (fHANT-AC) that can be coordinately regulated. The clustered genes include <em>nrt2</em>, which codes for a high affinity nitrate transporter; <em>euknr</em>, which codes for nitrate reductase; and <em>NAD(P)H-nir</em>, which codes for nitrite reductase. Homologs of genes in the fHANT-AC occur in other eukaryotes and prokaryotes, but they have only been found clustered in the oomycete <em>Phytophthora</em> (heterokonts). We performed independent and concatenated phylogenetic analyses of homologs of all three genes in the fHANT-AC. Phylogenetic analyses limited to fungal sequences suggest that the fHANT-AC has been transferred horizontally from a basidiomycete (mushrooms and smuts) to an ancestor of the ascomycetous mold <em>Trichoderma reesei</em>. Phylogenetic analyses of sequences from diverse eukaryotes and eubacteria, and cluster structure, are consistent with a hypothesis that the fHANT-AC was assembled in a lineage leading to the oomycetes and was subsequently transferred to the Dikarya (Ascomycota+Basidiomycota), which is a derived fungal clade that includes the vast majority of terrestrial fungi. We propose that the acquisition of high affinity nitrate assimilation contributed to the success of Dikarya on land by allowing exploitation of nitrate in aerobic soils, and the subsequent transfer of a complete assimilation cluster improved the fitness of <em>T. reesei</em> in a new niche. Horizontal transmission of this cluster of functionally integrated genes supports the “selfish operon” hypothesis for maintenance of gene clusters.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Slot JC, Hibbett DS (2007) Horizontal Transfer of a Nitrate Assimilation Gene Cluster and Ecological Transitions in Fungi: A Phylogenetic Study. PLoS ONE 2(10):
          e1097.
            doi:10.1371/journal.pone.0001097</p><p><strong>Academic Editor: </strong>James Fraser, University of Queensland, Australia</p><p><strong>Received:</strong> July 20, 2007; <strong>Accepted:</strong> October 8, 2007; <strong>Published:</strong> October 31, 2007</p><p><strong>Copyright:</strong> © 2007 Slot, Hibbett. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was made possible by a National Science Foundation Doctoral Dissertation Improvement grant (DEB0608017, to DSH and JCS). The National Science Foundation played no role in the production of data or of the manuscript and this manuscript does not imply its approval of the research or publication of the results.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A cluster of nitrate assimilation genes occurs in the ascomycetes <em>Aspergillus nidulans</em> <a href="#pone.0001097-Johnstone1">[1]</a> and <em>Pichia angusta</em> <a href="#pone.0001097-Brito1">[2]</a> and in the basidiomycetes <em>Hebeloma cylindrosporum</em>, which is mycorrhizal, and <em>Phanerochaete chrysosporium</em> <a href="#pone.0001097-Jargeat1">[3]</a>, which is a wood-decayer. This cluster, abbreviated fHANT-AC, encodes a high affinity nitrate transporter (NRT2, TCDB 2.A.1.8.5) along with a nitrate reductase (EUKNR, EC1.7.1.3) and a ferredoxin-independent assimilatory nitrite reductase (NAD[P]H-NIR, EC1.7.1.4). The latter two proteins are required for the reduction of nitrate to ammonium <a href="#pone.0001097-Marzluf1">[4]</a>. Ascomycetes and basidiomycetes are sister taxa that together form a clade called Dikarya, which contains the vast majority of described species of filamentous, terrestrial fungi. Without regard to clustering, <em>nrt2</em> is widely distributed in bacteria, plants, heterokonts and other groups, but not in opisthokonts outside Dikarya <a href="#pone.0001097-Slot1">[5]</a>. <em>Euknr</em> is restricted to eukaryotes, and absent from opisthokonts outside Dikarya, and <em>NAD(P)H-nir</em> is known to occur in Dikarya, heterokonts and bacteria.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The clustering of functionally related genes may be selectively favorable because it facilitates coordinated expression of the constituent genes <a href="#pone.0001097-Price1">[6]</a>. The “selfish operon” hypothesis puts forth that clusters can result from the selective maintenance of fitness-improving genes following horizontal gene transfer (HGT) events <a href="#pone.0001097-Lawrence1">[7]</a>. The clustering of genes in selfish operons greatly increases success of HGT events when a phenotypic advantage or ability to exploit a new niche is conferred to the recipient <a href="#pone.0001097-Lawrence2">[8]</a>. Gene clusters may facilitate the acquisition of entire metabolic pathways in the fungi <a href="#pone.0001097-Rosewich1">[9]</a>, <a href="#pone.0001097-Walton1">[10]</a>. Examples mainly involve secondary metabolite clusters, which do not necessarily require HGT between fungal species to explain their distribution <a href="#pone.0001097-Kroken1">[11]</a>, and pathogenicity gene clusters <a href="#pone.0001097-Rosewich1">[9]</a>, <a href="#pone.0001097-Han1">[12]</a>. Transfers of supernumerary chromosomes and large fragments of chromatin remain the most compelling mechanisms <a href="#pone.0001097-Han1">[12]</a>, <a href="#pone.0001097-Kavanaugh1">[13]</a>, <a href="#pone.0001097-He1">[14]</a> for the transfer of metabolic pathways.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>After discovering a potential horizontal transfer of <em>nrt2</em> previously <a href="#pone.0001097-Slot1">[5]</a>, we conducted a series of phylogenetic analyses to reconstruct the history of the fHANT-AC and its constituent genes. We first searched available eukaryotic genome sequences for homologs of genes in fHANT-AC. We then estimated the organismal phylogeny of ascomycetes, basidiomycetes, and oomycetes using rRNA and RPB2 gene sequences. We contrasted the organismal phylogeny with results of independent and concatenated phylogenetic analyses of genes in the fHANT-AC, which we performed at two phylogenetic scales: within the fungi (rooted with the heterokont <em>Phytophthora</em>), and across diverse eukaryotes and eubacteria (rooted with selected prokaryotes). Phylogenetic analyses within the fungi suggest that the nitrate assimilation cluster in the ascomycete <em>T. reesei</em> was obtained by HGT from a basidiomycete and corresponds to a change in nutritional mode. This is the best evidence to date that horizontal transfer of a metabolic pathway can facilitate niche shift in fungi. Less conclusive phylogenetic and structural evidence from analyses across diverse eukaryotes and eubacteria cannot reject a heterokont origin of the nitrate assimilation cluster in Dikarya.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Distribution of Genes of the fHANT-AC in Eukaryotes is Patchy</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Homologs of each of the three genes of the fHANT-AC (SI <a href="#pone.0001097.s004">Table S1</a>) are limited to Viridiplantae, Rhodophyta, heterokonts (oomycetes+diatoms in this dataset) and Dikarya, with one exception in <em>Nematostella vectensis</em> (Metazoa), which possesses bacteria-like <em>nrt2</em> and <em>nitrite reductase</em> homologs (<a href="http://genome.jgi-psf.org/Nemve1/Nemve1.home.html">http://genome.jgi-psf.org/Nemve1/Nemve1.​home.html</a>). Viridiplantae and Rhodophyta possess genes homologous to the dikarya/heterokont <em>nrt2</em> and <em>euknr</em>, but not the fungal/heterokont <em>NAD(P)H-nir</em>. Instead, these groups maintain a ferredoxin-dependent nitrite reductase gene (<em>cpnir</em>, EC1.7.7.1) of probable plastid/cyanobacterial origin <a href="#pone.0001097-Luque1">[15]</a>. Diatoms possess the three Dikarya/oomycete homologs in addition to <em>cpnir</em> (<a href="http://genome.jgi-psf.org/Thaps3/Thaps3.home.html">http://genome.jgi-psf.org/Thaps3/Thaps3.​home.html</a>). Diverse eubacteria possess <em>nrt2</em> and <em>NAD(P)H-nir</em> homologs, but not <em>euknr</em> which has an exclusively eukaryotic distribution <a href="#pone.0001097-Stolz1">[16]</a>. From the <em>Populus trichocarpa</em> genome, we recovered a β proteobacteria-like <em>NAD(P)H-nir</em>. Homologs of the dikarya/oomycete genes are absent from most Dikarya yeasts (with the exception of <em>Pichia angusta</em>), filamentous fungi outside Dikarya (<em>Rhizopus oryzae</em> and <em>Phycomyces blakesleeanus</em>), and the chytrid <em>Batrachochytrium dendrobatis</em>.</p>


<h4>Phylogenetic Analyses Suggest Horizontal Gene Transfer of the fHANT-AC</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A comparison of the organismal phylogeny and the phylogeny of fHANT-AC within fungi provides support for the hypothesis that fHANT-AC has been transferred horizontally from basidiomycetes to an ascomycete (<a href="#pone-0001097-g001">Fig. 1a</a>). In separate and combined analyses of nuclear ribosomal RNA genes (rDNA) and translated <em>rpb2</em> (RPB2) sequences using Bayesian, maximum likelihood (ML) and maximum parsimony (MP) methods (BPP = Bayesian Posterior Probabilities, MLB = Maximum Likelihood Bootstrap, MPB = Maximum Parsimony Bootstrap), we recovered species relationships that are congruent with previous molecular phylogenies <a href="#pone.0001097-James1">[17]</a>. We received strong support (1.0 BPP, 100 MPB combined analysis and 100MLB rDNA and RPB2 analyses unless otherwise noted) for the monophyly of Basidiomycota (RPB2 = 97 MLB, rDNA = 63 MLB) including <em>Ustilago maydis</em>, the ascomycetes (rDNA = 81 MLB), Pezizomycotina (non-yeast ascomycete lineages), and Sordariomycetes including <em>Trichoderma reesei</em>.</p>
<div class="figure" id="pone-0001097-g001"><div class="img"><a name="pone-0001097-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001097.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001097" data-uri="info:doi/10.1371/journal.pone.0001097.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001097.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001097.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001097.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001097.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001097.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001097.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001097.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001097.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001097.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Phylogeny and clustering of the fHANT-AC.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(a) Organismal phylogeny based on rRNA and RPB2 genes (right), and gene phylogeny of the fHANT-AC cluster (left), along with data on internal gene order and extent of clustering (center). Phylograms are from 50% majority rule Bayesian consensus. Shading of genes indicates confirmed expression of sequences in GenBank (dbEST:CF878787.1, CB906451.1, CF872359, CF865713, CB895628). Thick black branches denote strong support for the complete combined sequence (Bayesian posterior probabilities [BPP]&gt;0.95, maximum parsimony bootstrap percentages [MPB]&gt;85), and thick gray branches denote strong support from two of three individual genes (BPP 0.99 and/or MPB 70%). Strong support from maximum likelihood bootstraps ([MLB]&gt;80%) is indicated by an *. Actual support values for critical nodes are in <a href="#pone-0001097-t001">Table 1</a>. Clades labeled A (Ascomycota), B (Basidiomycota) and S (Sordariomycetes) follow James <em>et al.</em> (2006). Contributing phylogenetic analyses are in SI <a href="#pone.0001097.s001">Fig. S1a,b</a>. (b) Description of open reading frames flanking the fHANT-AC in <em>T. reesei</em> and <em>U. maydis</em>. Complete descriptions are available at genome project websites (see SI <a href="#pone.0001097.s004">Table S1</a>).</p>
<span>doi:10.1371/journal.pone.0001097.g001</span></div><div class="figure" id="pone-0001097-t001"><div class="img"><a name="pone-0001097-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001097.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001097" data-uri="info:doi/10.1371/journal.pone.0001097.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001097.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001097.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001097.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001097.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001097.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001097.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001097.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001097.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001097.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Alignments including fungal and <em>Phythophthora</em> sequences.</span></strong></p><span>doi:10.1371/journal.pone.0001097.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The phylogeny of the fHANT-AC genes in fungi is similar to the organismal phylogeny inferred with rRNA and RPB2 genes, with one striking exception: in independent and combined analyses, the three genes of the fHANT-AC of the ascomycete <em>T. reesei</em> were strongly supported (1.0 BPP, 100 MPB, 100 MLB) as sister to those of the basidiomycete <em>Ustilago maydis</em>. Additionally, a Shimodaira-Hasegawa test (<a href="#pone-0001097-t001">Table 1</a>) rejected the monophyly of ascomycete nitrate assimilation clusters when <em>T. reesei</em> sequences were included (p&lt;0.0001). Together, these analyses indicate strong conflict between the organismal and nitrate assimilation gene phylogenies that is best explained by HGT of the fHANT-AC from the Ustilaginales to <em>Trichoderma</em>. A search of the EST database at GenBank confirms that at least two of the horizontally transferred genes (<em>nrt2</em> and <em>euknr</em>) are expressed in <em>T. reesei</em>. We recovered no evidence of vertically derived homologs of these genes in <em>T. reesei</em></p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Higher-level analyses (<a href="#pone-0001097-g002">Fig. 2</a>) of HANT-AC proteins including diverse eubacteria sequences along with sequences from fungi, heterokonts and 25-29 (depending on gene distribution) eukaryotic lineages are not inconsistent with a heterokont origin of the fungal HANT-AC, although patchy distribution prevents a strong conclusion from phylogenetic analyses. NAD(P)H-NIR sequences from three species of <em>Phytophthora</em> (oomycetes) and <em>Phaeodactylum</em> and <em>Thalassiosira</em> (diatoms) do not form a heterokont clade. <em>Phytophthora</em> NAD(P)H-NIR sequences are sister to fungal sequences (1.0 BPP, 81 MPB, 57 MLB), and heterokonts plus fungi are supported as monophyletic by MP analyses (77 MPB), but not by ML or Bayesian analyses. Analyses of NRT2 suggest a red alga+heterokont+fungi clade (1.0 BPP, &lt;50 MPB, 74 MLB) that excludes green algae and plants in a strongly supported (1.0 BPP, 92 MPB, 99 MLB) eukaryote clade. These results are consistent with previous analyses extensively focused on fungi, which suggested that fungal NRT2 sequences form a monophyletic group that is nested within a paraphyletic assemblage of sequences from heterokonts <a href="#pone.0001097-Slot1">[5]</a>. Analyses of EUKNR sequences , which were restricted to the eukaryotic distribution of the gene, revealed little phylogenetic resolution outside the fungi. We found no synapomorphic insertion or deletion events in any of these sequences that exclusively support a sister relationship between fungi and oomycetes. Future analyses that include eukaryotic and fungal lineages not represented in our survey could falsify the hypothesis of an ancient transfer from heterokonts to dikarya.</p>
<div class="figure" id="pone-0001097-g002"><div class="img"><a name="pone-0001097-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001097.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001097" data-uri="info:doi/10.1371/journal.pone.0001097.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001097.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001097.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001097.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001097.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001097.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001097.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001097.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001097.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001097.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Distribution and phylogeny of HANT-AC gene homologs.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>(a) Unrooted phylograms of maximum likelihood analyses performed in RaxML-VI-HPC ver. 2.2.3. Support values for critical nodes can be found in <a href="#pone-0001097-t002">Table 2</a>. Contributing phylogenetic analyses are in SI <a href="#pone-0001097-g002">Fig. 2a,b,c</a>. (b) Distribution and clustering of HANT-AC genes in eukaryotes. Clustering indicated if most or all genomes in clade show clustering. Phylogeny of eukaryotes is adapted from Baldauf <a href="#pone.0001097-Baldauf1">[57]</a>. <sup>1</sup>A homolog similar to bacterial sequences is present in <em>Nematostella vectensis.</em> <sup>2</sup><em>N. vectensis</em> may also possesses bacteria-like nitrite reductase homologs.</p>
<span>doi:10.1371/journal.pone.0001097.g002</span></div><div class="figure" id="pone-0001097-t002"><div class="img"><a name="pone-0001097-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001097.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001097" data-uri="info:doi/10.1371/journal.pone.0001097.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001097.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001097.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001097.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001097.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001097.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001097.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001097.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001097.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001097.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Alignments including diverse eukaryotic sequences.</span></strong></p><span>doi:10.1371/journal.pone.0001097.t002</span></div>

<h4>Gene Clustering of fHANT-AC genes is Conserved, but Order is Not</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>While fHANT-AC genes are usually clustered, gene orders within and flanking the cluster are not well conserved between lineages (<a href="#pone-0001097-g001">Fig. 1a</a>). A comparison of genes flanking the fHANT-AC (<a href="#pone-0001097-g001">Fig. 1b</a>) in <em>Ustilago maydis</em> and <em>T. reesei</em> genomes did not reveal additional genes that appear to have been obtained by HGT. Most genes on the <em>T. reesei</em> scaffold 28 are highly similar to ascomycete genes (data not shown). In addition, the <em>Cip2</em> gene that occurs three genes downstream from the <em>nrt2</em> gene is a confirmed <em>T. reesei</em> sequence <a href="#pone.0001097-Foreman1">[18]</a>. These results suggest that the incongruous fHANT-AC was incorporated by chromosomal recombination involving a small number of genes rather than by acquisition of an additional chromosome. Genes flanking the fHANT-AC in <em>T. reesei</em> appear to be related to carbohydrate metabolism. We speculate that the occurrence of this cluster could influence carbohydrate metabolism by either upregulating or inhibiting transcription of these genes, and thereby contribute downstream to the superlative propensity of <em>T. reesei</em> to produce cellulases, <em>e.g.</em>in <a href="#pone.0001097-Seiboth1">[19]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>The fHANT-AC persists with genomic rearrangement</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The only groups found to maintain the three genes of the fHANT-AC in a cluster (as of June 2007) are the dikarya fungi and oomycete heterokonts (SI <a href="#pone.0001097.s004">Table S1</a>). <em>Euknr</em> and <em>nrt2</em> are widely distributed in dikarya fungi, chromalveolates (alveolates+heterokonts), and plants, whereas <em>NAD(P)H-nir</em> does not occur outside heterokonts and Dikarya (<a href="#pone-0001097-g002">Fig. 2</a>). We cannot yet falsify the hypothesis that the fHANT-AC cluster originated in the lineage leading to heterokonts (SI <a href="#pone.0001097.s003">Fig. S3A</a>) and was subsequently transferred to Dikarya. Because of the limited sample of non-dikarya fungi (SI <a href="#pone.0001097.s004">Table S1</a>) we cannot rule out the possibility of an earlier fungal origin. Nitrate assimilation clusters also occur in at least two green algae <a href="#pone.0001097-Derelle1">[20]</a>, <a href="#pone.0001097-Quesada1">[21]</a>, but each of these clusters contains a unique complement of genes that are not all homologous to those in the fHANT-AC. The convergent evolution of a nitrate assimilation cluster in green algae could suggest a general selective advantage to clustering of nitrate assimilation genes.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Clustering of nitrate assimilation genes is observed in varying degrees within Dikarya (<a href="#pone-0001097-g001">Fig. 1a</a>). In Basidiomycota where the genes are present, the clustered orientation of the three genes predominates with one known exception. A retroviral sequence interrupts the cluster between <em>nrt2</em> and <em>NAD(P)H-nir</em> in <em>Laccaria bicolor</em>, but the genes remain in the same locus. The cluster has disassembled in certain ascomycetes. At least two lineages have lost the clustering of the three genes while maintaining the genes themselves, most notably the Sordariomycetes, of which <em>T. reesei</em> is a member. <em>Botryotinia fuckeliana</em> maintains a cluster of two of the genes, while the closely related <em>Sclerotinia sclerotiorum</em> lacks clustering entirely. The orientation of the genes within the cluster is not phylogenetically conserved within Dikarya, suggesting that while rearrangement is common, these rearrangements do not necessarily disassemble the cluster. We propose three scenarios to explain the disassembly of the fHANT-AC in ascomycetes (SI <a href="#pone.0001097.s003">Fig. S3B, C, D</a>). If cluster formation is assumed to be an unlikely event, then that would favor a scenario (SI <a href="#pone.0001097.s003">Fig. S3D</a>) that only requires the assembly of the fHANT-AC to have occurred once. Metabolic gene clusters have been reported in several fungal lineages <a href="#pone.0001097-Howlett1">[22]</a>, <a href="#pone.0001097-Abe1">[23]</a>, <a href="#pone.0001097-Fleetwood1">[24]</a>, <a href="#pone.0001097-Zhang1">[25]</a>, <a href="#pone.0001097-Laich1">[26]</a>, <a href="#pone.0001097-Carbone1">[27]</a>, and one occurrence of rapid assembly of a cluster has been suggested in yeast <a href="#pone.0001097-Wong1">[28]</a>. However, there is currently no consensus regarding methods of measuring the relative rates of cluster assembly, disassembly and horizontal transfer in fungi.</p>


<h4>The Anomalous fHANT-AC in <em>Trichoderma reesei</em> coincides with Niche Shift</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p><em>T. reesei,</em> an ascomycete adapted to wood decay occurs in a genus consisting largely of endophytes and parasites <a href="#pone.0001097-Samuels1">[29]</a>. <em>T. reesei</em> strain QM9414, the subject of the <em>T. reesei</em> genome project, was derived from QM6a, a strain originally defined by its inability to grow on nitrate as a sole nitrogen source, likely due to defects in nitrite reductase <a href="#pone.0001097-Lieckfeldt1">[30]</a>. Several sexually competent strains of <em>T. reesei,</em> however, are known to utilize nitrate <a href="#pone.0001097-Lieckfeldt1">[30]</a>. We know of no direct evidence of nitrate assimilation by other <em>Trichoderma</em> species, although the presence of nitrate has been linked to physiological responses <a href="#pone.0001097-Ostrikova1">[31]</a>, <a href="#pone.0001097-Saikia1">[32]</a>, <a href="#pone.0001097-Jayaraj1">[33]</a> and suppression of growth <a href="#pone.0001097-Wakelin1">[34]</a> in some species. Attempts to amplify <em>nrt2</em> from various <em>Trichoderma</em> species using PCR were not successful. A shift away from a tightly regulated nitrate assimilation pathway may have preceded the acquisition of a basidiomycete fHANT-AC by <em>T. reesei</em>. Analyses of gene synteny (<a href="#pone-0001097-g001">Fig. 1a</a>) suggest that loss of the ascomycete nitrate assimilation genes in <em>T. reesei</em> was preceded by the disassembly of the cluster in Sordariomycetes. In one study, <em>Trichoderma</em> species (not including <em>T. reesei</em>) and <em>Sclerotinia sclerotiorum</em> (which also lacks an intact cluster) were shown to utilize ammonium preferentially, and possibly exclusively (in <em>Trichoderma)</em>, over nitrate in ammonium nitrate feeding experiments <a href="#pone.0001097-Celar1">[35]</a>. The effect was also observed, but less pronounced, for other sordariomycetes (<em>Fusarium spp</em>.). These observations suggest that there was a change in the mode of nitrate assimilation in the evolution of sordariomycetes. This could have favored the acquisition of the basidiomycete fHANT-AC and subsequent loss of vertically inherited genes when <em>T. reesei</em> switched to a nitrogen-deprived, woody substrate. The horizontal transmission leading to the <em>T. reesei</em> cluster is a plausible scenario because some <em>Trichoderma</em> species are intracellular parasites of basidiomycetes <a href="#pone.0001097-RochaRamirez1">[36]</a>, <a href="#pone.0001097-Sarrocco1">[37]</a>.</p>


<h4>Analysis of the HANT-AC Suggests Widespread Loss, Convergence or a “Selfish Operon”</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The distribution of HANT-AC genes could suggest either widespread loss outside Dikarya, heterokonts and plants or an HGT origin of the fungal homologs. We can reject the null hypotheses (<a href="#pone-0001097-t002">Table 2</a>) that chromalveolate and plant NRT2 form a clade that excludes fungi (p = .038), that heterokont NAD(P)H-NIR are monophyletic exclusive of fungi (p&lt;.0001), and that an organismal phylogeny based on five nuclear protein coding genes (SI <a href="#pone.0001097.s002">Fig. S2d</a>), restricted to taxa bearing HANT-AC genes, is consistent with the fungi+<em>Phytophthora</em> clade (p&lt;.0001) suggested by NAD(P)H-NIR analyses. Each of these results is consistent with an HGT origin of these genes in fungi. The eukaryotic <em>NAD(P)H-nir</em> could have been derived from a mitochondrial source, which would require multiple losses in different lineages of eukaryotes, or there could have been a more recent transfer from bacteria to a lineage leading to the heterokonts. The absence of <em>euknr</em> from other opisthokonts also suggests that this gene was secondarily acquired in Dikarya, although phylogenetic analyses do not clearly identify a source. The existence in <em>Phytophthora</em> of a cluster (oHANT-AC) homologous to the fHANT-AC, some of whose components appear to be vertically derived, is the strongest evidence of a heterokont origin of the fungal genes, implying that a cluster could have facilitated HGT. We present an hypothesis of the origin of the fHANT-AC in SI <a href="#pone.0001097.s003">Fig. S3</a>, which parsimoniously accounts for the observed distribution of HANT-AC genes in eukaryotes. There we show a single origin of the cluster in heterokonts from genes present in common ancestors of heterokonts and plants, followed by a transfer to the ancestor of Dikarya.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Recent studies have suggested closer genetic links between oomycetes and fungi than would be predicted by organismal phylogeny. One suggested that a number of shared EST sequences relate to a mutual tendency toward pathogenicity <a href="#pone.0001097-Randall1">[38]</a>. It was not clear if this is evidence of genomic convergence or genetic exchange. Another study suggested that at least four genes have been transferred from ascomycete fungi to oomycetes <a href="#pone.0001097-Richards1">[39]</a>. A failure to detect homologous sequences in other heterokonts and non-dikarya fungi in that study leaves the origin of these sequences ambiguous, requiring the invocation of additional prokaryotic donors. In our study, the presence of homologous genes in other heterokonts and related lineages, but not in other fungi suggests that the direction of the hypothetical transfer of the fHANT-AC would be from a heterokont to a common ancestor of dikarya fungi. A limited sample of non-dikarya fungi leaves an earlier fungal origin also possible. In this case, the restriction of clustered HANT-AC genes to oomycetes suggests that the heterokonts could have produced the species that donated the fHANT-AC to dikarya fungi. While the current study alone is not conclusive as to genetic exchange between these groups, it contributes to a developing hypothesis of an ancient, intimate association involving Dikarya and heterokonts.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>The alternate hypothesis is that the HANT-AC arose convergently in fungi and oomycetes under similar environmental pressures. For example, in aerobic soils, reduced iron for denitrification is limited, leading to elevated nitrate <a href="#pone.0001097-Stemmler1">[40]</a>. Iron-containing ferredoxin is required to donate electrons in plant nitrite reduction, but this function can be performed by NAD(P)H (derived from cellular respiration) in fungi and heterokonts; they could be partly freed from iron limitations through this mechanism. That certain diatoms are dominant in high-nitrate, low-iron environments <a href="#pone.0001097-Tsuda1">[41]</a> seems to support this. This nutritional mode could benefit photosynthetic partners in ectomycorrhizal and lichen symbioses under aerobic conditions. In fact, ectomycorrhizal fungi both down-regulate plant nitrite reductase <a href="#pone.0001097-Bailly1">[42]</a> and increase iron uptake <a href="#pone.0001097-Leyval1">[43]</a>. In contrast, oomycete plant pathogens could out-compete their host for nitrate in iron-limited conditions. Alternatively, the absence of photosystems to efficiently replenish reduced ferredoxin in fungi and oomycetes could have necessitated convergent acquisition of a respiration-linked source of electrons for nitrite reduction.</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>Co-regulation of the three genes in the fHANT-AC in <em>H. cylindrosporum</em> was recently demonstrated <a href="#pone.0001097-Jargeat1">[3]</a>. This was earlier demonstrated in the ascomycetous yeast, <em>Pichia angusta</em> <a href="#pone.0001097-Brito1">[2]</a>. The intergenic region between <em>NAD(P)H-nir</em> and <em>euknr</em> in <em>A. nidulans</em> serves as a bidirectional promoter for both genes <a href="#pone.0001097-MuroPastor1">[44]</a>, <a href="#pone.0001097-Punt1">[45]</a>. These findings collectively suggest that the fHANT-AC resembles a eukaryotic operon. The “selfish operon” is a coordinately regulated cluster of genes, which receive improved fitness because they are more readily maintained when transferred between organisms <a href="#pone.0001097-Lawrence2">[8]</a>. While not truly a “selfish” element due to its likely positive influence on host fitness, evidence of HGT of the fHANT-AC makes it a candidate for a “selfish” eukaryotic operon. The current results suggest that acquisition of nutritional operons may promote ecological shifts in fungi. Differential utilization of the fHANT-AC in various strains of <em>T. reesei</em> could suggest this species is currently undergoing such an evolutionary event.</p>


<h4>A key acquisition in colonization of dry land?</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The Dikarya experienced a neoproterozoic diversification <a href="#pone.0001097-Heckman1">[46]</a> that was not paralleled in other fungal lineages, including some that also associate with plants <a href="#pone.0001097-James1">[17]</a>. We speculate that the horizontally acquired ability to derive nitrogen from oxidized substrates, and to thus benefit photosynthetic hosts, may have been a key innovation that allowed the Dikarya to diversify in terrestrial habitats as free-living saprotrophs or mycorrhizal symbionts.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Phylogenetic Analysis of Organisms Containing the High Affinity Nitrate Assimilation Gene Cluster (fHANT-AC) using rRNA and RPB2 gene sequences</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We used a combination of tblastn <a href="#pone.0001097-Altschul1">[47]</a> and searches for annotated genes to obtain phylogenetically relevant sequences for species noted with an * in SI <a href="#pone.0001097.s004">Table S1</a>. Substitutions were made for <em>Pichia angusta</em> RNA polymerase II second largest subunit (RPB2, AAS67502, <em>Pichia guilliermondii</em>) and <em>Hebeloma cylindrosporum</em> (partial nuclear small ribosomal RNA subunit gene [nssu] AY745703 and <em>rpb2</em> DQ472718 from <em>H. velutipes</em> specimen PBM 2277) where genome projects were not available. We generated a partial nuclear large ribosomal RNA subunit gene (nlsu) sequence from <em>H.cylindrosporum</em> strain CBS558.96 (EF219136) using the primers LROR and LR5 (sequencing was performed on an ABI377 Automated DNA Sequencer using BigDye ver1.1; Applied Biosystems, Foster City, CA, USA). We concatenated nlsu and nssu nucleotide sequences and the inferred amino acid sequence of RPB2 for all available fungal genomes. Sequences were aligned with ClustalX <a href="#pone.0001097-Thompson1">[48]</a>, and adjusted manually. We created independent alignments (SI <a href="#pone.0001097.s005">Table S2</a>) for each gene by excluding two of the three genes, and a combined analysis that included all three genes. Ambiguously aligned characters were excluded from further analysis. Each alignment was analyzed by (A) 1000 maximum parsimony bootstrap replicates with ten random addition sequence replicates per bootstrap replicate, saving multiple trees at each replicate in PAUP* 4.0b <a href="#pone.0001097-Swofford1">[49]</a>, (B) two Bayesian analyses using mixed protein models for amino acids and a GTR plus gamma model of evolution for nucleotides in MrBayes ver. 3.2 <a href="#pone.0001097-Huelsenbeck1">[50]</a>, <a href="#pone.0001097-Ronquist1">[51]</a>, <a href="#pone.0001097-Altekar1">[52]</a>; we ran the Bayesian analyses for one million generations, sampled trees every 100 generations and removed trees sampled before likelihoods for two runs converged and stabilized as the burnin, and (C) 100 Maximum Likelihood Bootstrap replicates of RPB2 with mixed protein models and of combined rDNA with mixed GTR models as implemented in RaxML-VI-HPC ver. 2.2.3. We also performed (D) maximum likelihood searches of a combined rDNA dataset and an RPB2 dataset in RaxML-VI-HPC ver. 2.2.3 <a href="#pone.0001097-Stamatakis1">[53]</a> (mixed GTR and protein models) for the optimal topologies and also for the best topology in which <em>T. reesei</em> sequences were constrained to form a clade with <em>U. maydis</em>. We then used a Shimodaira-Hasegawa test <a href="#pone.0001097-Shimodaira1">[54]</a>, as implemented in TreePuzzle ver. 5.2 <a href="#pone.0001097-Schmidt1">[55]</a>, to compare the likelihoods of resulting topologies.</p>


<h4>Analysis of the nitrate assimilation gene cluster in fungi</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We concatenated amino acid sequences inferred from three genes in the fHANT-AC (<em>nrt2, euknr</em> and <em>NAD(P)H-nir</em>) of <em>Hebeloma cylindrosporum</em>. We used tBlastn <a href="#pone.0001097-Altschul1">[47]</a> with this sequence to search genome projects shown in supporting materials (SI <a href="#pone.0001097.s004">Table S1</a>) for homologous sequences. We obtained <em>H. cylindrosporum</em> and <em>P. angusta</em> fHANT-AC sequences, and <em>T. reesei</em> EST sequences from GenBank. The identity of each sequence was verified by a blastp search of GenBank to determine the most similar sequences and conservation of functional domain architecture. We assembled an alignment consisting of concatenated amino acid sequences from the fHANT-AC for all available fungal genomes (as of May, 2006, shown with an * in SI <a href="#pone.0001097.s004">Table S1</a>) and also <em>Phytophthora sojae</em> and <em>Phytophthora ramorum</em> genomes, which were included as the outgroup. Alignments and analyses for independent genes and combined sequences were conducted as described for amino acid sequences in the phylogenetic analysis. We performed maximum likelihood searches of the combined fHANT-AC dataset using mixed protein models in RaxML-VI-HPC (see above) for the optimal topology and also for the best topology in which ascomycete sequences were constrained to be monophyletic. We then compared the likelihoods of resulting topologies using a Shimodaira-Hasegawa test as described above for RPB2.</p>


<h4>Analyses of components of the fHANT-AC in diverse eukaryotes and prokaryotes</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>We conducted higher level analyses of NRT2, EUKNR and NAD(P)H-NIR amino acid sequences retrieved from eukaryotic genome projects along with bacterial sequences from GenBank. For NRT2, approximately 200 bacterial sequences and for NAD(P)H-NIR approximately 500 bacterial sequences were initially obtained. These were combined with the eukaryotic genome sequences, aligned with mafft ver. 5.861 <a href="#pone.0001097-Katoh1">[56]</a> using the default settings. Long ends of the alignment were trimmed and a neighbor joining analysis was performed in PAUP* 4.0b. Thirty-nine clades of NRT2 and eighty-three clades of NAD(P)H-NIR were then analyzed as described above for the fHANT-AC genes, except 500 maximum likelihood bootstraps were performed on each gene separately. We confirmed that major clades from previous NRT2 analyses <a href="#pone.0001097-Slot1">[5]</a> were represented. Twenty-nine eukaryotic EUKNR sequences were also analyzed as described for the other protein sequences. Constraint analyses as described above forced the monophyly of heterokont NAD(P)H-NIR sequences. We performed analyses as described for a combined dataset of α-tubulin, β-tubulin, RNA polymerase II largest and second largest subunits, and elongation factor 1-alpha limited to taxa bearing homologs of genes in the fHANT-AC. We then performed constraint analyses as described above, which forced either fungi and <em>Phytophthora</em> or fungi and heterokonts to be monophyletic.</p>


<h4>Analysis of gene order and location in fungi</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We used the browser at the <em>T. reesei</em> and <em>Ustilago maydis</em> genome websites to catalog genes flanking the fHANT-AC in each species. We cataloged genes of up to four inferred or hypothetical proteins for each species. We used tBlastn <a href="#pone.0001097-Altschul1">[47]</a> with a concatenated amino acid sequence of the three proteins encoded by the fHANT-AC of <em>Coprinopsis cinerea</em> in order of nitrate metabolism (NRT2-EUKNR-NAD(P)H-NIR) to obtain the location and relative direction of transcription of each gene in the eukaryotic genomes and bacterial genomes shown in supporting materials, <a href="#pone.0001097.s004">Table S1</a>. We also performed tBlastn using the individual genes, and used 45% amino acid similarity to determine the absence of homologs for each gene in eukaryotes and 40% amino acid similarity in bacteria. This was repeated using the <em>Arabidopsis thaliana</em> ferredoxin-dependent nitrite reductase (NP_179164.1) as the query.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001097"></div><a name="pone.0001097.s001" id="pone.0001097.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001097.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI"><a href="#pone.0001097.s001">Figure S1</a> depicts phylogenetic trees of fungal sequences, rooted with an oomycete (heterokont) outgroup. <a href="#pone.0001097.s001">Figure S1a</a> shows phylogenetic trees based on independent and combined analyses of ribosomal RNA and RPB2 gene loci, which reflect the organismal phylogeny. <a href="#pone.0001097.s001">Figure S1b</a> presents phylogenetic trees based on independent and combined analyses of genes in the fHANT-AC cluster. All analyses of the organismal phylogeny group Trichoderma reesei with Gibberella zeae (ascomycetes), and all analyses of genes in the fHANT-AC cluster group Trichoderma reesei with Ustilago maydis (basidiomycetes).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001097.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.49 MB PDF)</p>
<a name="pone.0001097.s002" id="pone.0001097.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001097.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Maximum likelihood trees showing relationships among (a) NRT2, (b) EUKNR and (c) NAD(P)H NIR amino acid sequences from eukaryotic and eubacteria genomes. (d) Maximum likelihood trees showing relationships among concatenated nuclear gene products (α- and β-tubulin, rpb1, rpb2 and ef1-α), from eukaryotic genomes. Branch lengths reflect number of changes per site.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001097.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.88 MB PDF)</p>
<a name="pone.0001097.s003" id="pone.0001097.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001097.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Evolution of the nitrate assimilation cluster in Fungi and other eukaryotes. A. Evolution of the nitrate assimilation cluster across the eukaryote phylogeny. B. Cluster evolution within the fungi. C, D. Alternative reconstructions of cluster dissociation and formation events in fungi. 1. Nrt2 and NAD(P)H nir were acquired from bacteria, and euknr was derived from the sulfite oxidase gene family in the nuclear genome<a href="#pone.0001097-Johnstone1">[1]</a>. 2. The common ancestor of the Chromalveolates and Plantae retained nrt2, NAD(P)H-nir, and euknr. 3. NAD(P)H-nir was lost in the lineage leading to Plantae, after the divergence of Chromalveolates, and Cp-nir was acquired during the primary endosymbiotic origin of the chloroplast (not necessarily in that order). 4. Cp-nir was horizontally transferred from the Rhodophytes to the Heterokonts (Stramenopiles) during a secondary endosymbiotic origin of chloroplasts <a href="#pone.0001097-Brito1">[2]</a>. Cp-nir was retained in the lineage leading to the diatoms, but was lost in the lineage leading to the Oomycetes. 5. Nrt2, NAD(P)H-nir, and euknr were each lost in the lineage leading to the Alveolates. 6. The nitrate assimilation cluster, including nrt2, NAD(P)H-nir, and euknr, was formed in the lineage leading to the Oomycetes. 7. The nitrate assimilation cluster was horizontally transferred from the lineage leading to the Oomycetes to the lineage leading to Dikarya. 8. The nitrate assimilation cluster was horizontally transferred from the lineage leading to Ustilago maydis (Basidiomycota) to the lineage leading to Trichoderma reesei (Ascomycota). The Ascomycota-derived components of the nitrate assimilation cluster were lost in the lineage leading to Trichoderma reesei (this could have occurred before or after the horizontal transfer event). 9. The nitrate assimilation cluster became dissociated twice during the evolution of Ascomycota. Aspergillus retains an ancestral clustered condition. Two component genes of the nitrate assimilation cluster (euk-nr and NAD(P)H-nir) were rejoined in the lineage leading to Botryotinia fuckeliana. 10. Alternatively (B), the nitrate assimilation cluster became dissociated only once in the evolution of Ascomycota, and was reconstituted in the lineage leading to Aspergillus; euk-nr and NAD(P)H-nir were rejoined in the lineage leading to Botryotinia fuckeliana. 11. Another possible scenario (D) suggests that the nitrate assimilation cluster became dissociated in four separate events in the Ascomycota, and that the clustered condition in Aspergillus spp. and Botryotinia fuckeliana are both plesiomorphic. Although this scenario involves the same number of breakpoint/association events as those described in (B) and (C), it does not require that clusters of genes be reconstituted after having been disrupted. 12. All three component genes of the nitrate assimilation cluster were lost in the lineages leading to Cryptococcus neoformans (Basidiomycota) and Saccharomyces cerevisiae (Ascomycota). Vertical order of branching points does not reflect absolute timing of such events. 1. Stolz JF, Basu P (2002) Evolution of nitrate reductase: molecular and structural variations on a common function. Chembiochem 3: 198–206. 2. Bhattacharya D, Yoon HS, Hackett JD (2004) Photosynthetic eukaryotes unite: endosymbiosis connects the dots. Bioessays 26: 50–60.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001097.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.93 MB PDF)</p>
<a name="pone.0001097.s004" id="pone.0001097.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001097.s004">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI"><sup>1</sup> Highest blastp sequence in GenBank, 88% similar to AAQ21342, uncultured bacterium nitrite reductase. <sup>2</sup> Highest tblastn hit in GenBank feature in acc. #CP000082 Psychrobacter arcticus 273-4, nitrate transporter. <sup>3</sup> Tblastn 62%similar to Arabidopsis thaliana GenBank acc. # NM_127123, nitrite reductase, 42% similar to feature in NT_165926, Aspergillus terreus sulfite reductase, beta subunit. <sup>4</sup> Tblastn 69% similar to 489AA of C.cinerea NAD(P)Hnir. <sup>5</sup> Locus ZP_00980374, nitrite/sulfite reductase, Tblastn 44% similar to Arabidopsis thaliana GenBank acc. #NP_179164.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001097.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.11 MB DOC)</p>
<a name="pone.0001097.s005" id="pone.0001097.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001097.s005">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">For each dataset, two simultaneous Bayesian analyses were run for one million generations, using MrBayes ver. 3.1.2 <a href="#pone.0001097-Johnstone1">[1]</a>, <a href="#pone.0001097-Brito1">[2]</a>. Amino acid data was analyzed under mixed protein models, and nucleotide data under a GTR+Gamma model. Trees were sampled every 100 generations. Trees sampled before likelihood convergence and stabilization of independent chains (usually 500–2000) were removed as the burnin. Each alignment in SI <a href="#pone-0001097-t002">Table 2</a> was also analyzed by maximum parsimony bootstrapping and maximum likelihood bootstrapping. In parsimony analyses, performed in PAUP*4.0b <a href="#pone.0001097-Jargeat1">[3]</a>, characters were equally weighted. 1000 bootstrap replicates were performed. Each bootstrap replicate performed a heuristic search with 10 random addition sequence replicates, with the multitrees option turned on, using the TBR branch-swapping algorithm. Likelihood analyses were performed under mixed protein models or mixed GTR (for nucleotides). 100 bootstraps were performed on fungal alignments (<a href="#pone.0001097.s001">Fig. S1</a>), and 500 on eukaryotic alignments (<a href="#pone.0001097.s002">Fig. S2</a>) in RaxML-VI-HPC ver. 2.2.3<a href="#pone.0001097-Marzluf1">[4]</a>. Literature Cited: 1. Huelsenbeck JP, Ronquist F (2001) MRBAYES: Bayesian inference of phylogenetic trees. Bioinformatics 17: 754–755. 2. Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics 19: 1572–1574. 3. Swofford DL (2003) PAUP*. Phylogenetic Analysis Using Parsimony (*and Other Methods). Version 4. Sinauer Associates, Sunderland, Massachusetts. 4. Stamatakis A (2006) RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models. Bioinformatics 22: 2688–2690.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001097.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Z. Wang and several anonymous reviewers for helpful suggestions, and P. Matheny, M. Binder and C. Khatchikian for editorial help.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DH JS. Performed the experiments: JS. Analyzed the data: JS. Wrote the paper: DH JS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001097-Johnstone1" id="pone.0001097-Johnstone1"></a>Johnstone IL, McCabe PC, Greaves P, Gurr SJ, Cole GE, et al.  (1990) Isolation and characterisation of the crnA-niiA-niaD gene cluster for nitrate assimilation in Aspergillus nidulans. Gene  90: 181–192.  <ul class="find" data-citedArticleID="1010872" data-doi="10.1016/0378-1119(90)90178-t"><li><a href="http://dx.doi.org/10.1016/0378-1119(90)90178-t" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+characterisation+of+the+crnA-niiA-niaD+gene+cluster+for+nitrate+assimilation+in+Aspergillus+nidulans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+characterisation+of+the+crnA-niiA-niaD+gene+cluster+for+nitrate+assimilation+in+Aspergillus+nidulans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001097-Brito1" id="pone.0001097-Brito1"></a>Brito N, Avila J, Perez MD, Gonzalez C, Siverio JM (1996) The genes YNI1 and YNR1, encoding nitrite reductase and nitrate reductase respectively in the yeast Hansenula polymorpha, are clustered and co-ordinately regulated. Biochem J  317(Pt 1): 89–95.  <ul class="find" data-citedArticleID="1010844" data-doi="10.1016/0014-5793(95)00511-7"><li><a href="http://dx.doi.org/10.1016/0014-5793(95)00511-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genes+YNI1+and+YNR1%2C+encoding+nitrite+reductase+and+nitrate+reductase+respectively+in+the+yeast+Hansenula+polymorpha%2C+are+clustered+and+co-ordinately+regulated." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genes+YNI1+and+YNR1%2C+encoding+nitrite+reductase+and+nitrate+reductase+respectively+in+the+yeast+Hansenula+polymorpha%2C+are+clustered+and+co-ordinately+regulated.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001097-Jargeat1" id="pone.0001097-Jargeat1"></a>Jargeat P, Rekangalt D, Verner MC, Gay G, Debaud JC, et al.  (2003) Characterisation and expression analysis of a nitrate transporter and nitrite reductase genes, two members of a gene cluster for nitrate assimilation from the symbiotic basidiomycete Hebeloma cylindrosporum. Curr Genet  43: 199–205.  <ul class="find" data-citedArticleID="1010868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterisation+and+expression+analysis+of+a+nitrate+transporter+and+nitrite+reductase+genes%2C+two+members+of+a+gene+cluster+for+nitrate+assimilation+from+the+symbiotic+basidiomycete+Hebeloma+cylindrosporum.&amp;auth=&amp;atitle=Characterisation+and+expression+analysis+of+a+nitrate+transporter+and+nitrite+reductase+genes%2C+two+members+of+a+gene+cluster+for+nitrate+assimilation+from+the+symbiotic+basidiomycete+Hebeloma+cylindrosporum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterisation+and+expression+analysis+of+a+nitrate+transporter+and+nitrite+reductase+genes%2C+two+members+of+a+gene+cluster+for+nitrate+assimilation+from+the+symbiotic+basidiomycete+Hebeloma+cylindrosporum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterisation+and+expression+analysis+of+a+nitrate+transporter+and+nitrite+reductase+genes%2C+two+members+of+a+gene+cluster+for+nitrate+assimilation+from+the+symbiotic+basidiomycete+Hebeloma+cylindrosporum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001097-Marzluf1" id="pone.0001097-Marzluf1"></a>Marzluf GA (1997) Genetic regulation of nitrogen metabolism in the fungi. Microbiol Mol Biol Rev  61: 17–32.  <ul class="find" data-citedArticleID="1010892"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+regulation+of+nitrogen+metabolism+in+the+fungi.&amp;auth=&amp;atitle=Genetic+regulation+of+nitrogen+metabolism+in+the+fungi." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+regulation+of+nitrogen+metabolism+in+the+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+regulation+of+nitrogen+metabolism+in+the+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001097-Slot1" id="pone.0001097-Slot1"></a>Slot JC, Hallstrom K, Matheny PB, Hibbett DS (2007) Diversification of NRT2 and the Origin of Its Fungal Homolog. Mol Biol Evol.   <ul class="find" data-citedArticleID="1010926" data-doi="10.1093/molbev/msm098"><li><a href="http://dx.doi.org/10.1093/molbev/msm098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversification+of+NRT2+and+the+Origin+of+Its+Fungal+Homolog." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversification+of+NRT2+and+the+Origin+of+Its+Fungal+Homolog.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001097-Price1" id="pone.0001097-Price1"></a>Price MN, Huang KH, Arkin AP, Alm EJ (2005) Operon formation is driven by co-regulation and not by horizontal gene transfer. Genome Res  15: 809–819.  <ul class="find" data-citedArticleID="1010898" data-doi="10.1101/gr.3368805"><li><a href="http://dx.doi.org/10.1101/gr.3368805" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Operon+formation+is+driven+by+co-regulation+and+not+by+horizontal+gene+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Operon+formation+is+driven+by+co-regulation+and+not+by+horizontal+gene+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001097-Lawrence1" id="pone.0001097-Lawrence1"></a>Lawrence JG (1997) Selfish operons and speciation by gene transfer. Trends Microbiol  5: 355–359.  <ul class="find" data-citedArticleID="1010882" data-doi="10.1016/s0966-842x(97)01110-4"><li><a href="http://dx.doi.org/10.1016/s0966-842x(97)01110-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selfish+operons+and+speciation+by+gene+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selfish+operons+and+speciation+by+gene+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001097-Lawrence2" id="pone.0001097-Lawrence2"></a>Lawrence J (1999) Selfish operons: the evolutionary impact of gene clustering in prokaryotes and eukaryotes. Curr Opin Genet Dev  9: 642–648.  <ul class="find" data-citedArticleID="1010884" data-doi="10.1016/s0959-437x(99)00025-8"><li><a href="http://dx.doi.org/10.1016/s0959-437x(99)00025-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selfish+operons%3A+the+evolutionary+impact+of+gene+clustering+in+prokaryotes+and+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selfish+operons%3A+the+evolutionary+impact+of+gene+clustering+in+prokaryotes+and+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001097-Rosewich1" id="pone.0001097-Rosewich1"></a>Rosewich UL, Kistler HC (2000) Role of Horizontal Gene Transfer in the Evolution of Fungi. Annu Rev Phytopathol  38: 325–363.  <ul class="find" data-citedArticleID="1010912" data-doi="10.1146/annurev.phyto.38.1.325"><li><a href="http://dx.doi.org/10.1146/annurev.phyto.38.1.325" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+Horizontal+Gene+Transfer+in+the+Evolution+of+Fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+Horizontal+Gene+Transfer+in+the+Evolution+of+Fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001097-Walton1" id="pone.0001097-Walton1"></a>Walton JD (2000) Horizontal gene transfer and the evolution of secondary metabolite gene clusters in fungi: an hypothesis. Fungal Genet Biol  30: 167–171.  <ul class="find" data-citedArticleID="1010942" data-doi="10.1006/fgbi.2000.1224"><li><a href="http://dx.doi.org/10.1006/fgbi.2000.1224" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Horizontal+gene+transfer+and+the+evolution+of+secondary+metabolite+gene+clusters+in+fungi%3A+an+hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Horizontal+gene+transfer+and+the+evolution+of+secondary+metabolite+gene+clusters+in+fungi%3A+an+hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001097-Kroken1" id="pone.0001097-Kroken1"></a>Kroken S, Glass NL, Taylor JW, Yoder OC, Turgeon BG (2003) Phylogenomic analysis of type I polyketide synthase genes in pathogenic and saprobic ascomycetes. Proc Natl Acad Sci U S A  100: 15670–15675.  <ul class="find" data-citedArticleID="1010878" data-doi="10.1073/pnas.2532165100"><li><a href="http://dx.doi.org/10.1073/pnas.2532165100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenomic+analysis+of+type+I+polyketide+synthase+genes+in+pathogenic+and+saprobic+ascomycetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenomic+analysis+of+type+I+polyketide+synthase+genes+in+pathogenic+and+saprobic+ascomycetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001097-Han1" id="pone.0001097-Han1"></a>Han Y, Liu X, Benny U, Kistler HC, VanEtten HD (2001) Genes determining pathogenicity to pea are clustered on a supernumerary chromosome in the fungal plant pathogen Nectria haematococca. Plant J  25: 305–314.  <ul class="find" data-citedArticleID="1010856" data-doi="10.1046/j.1365-313x.2001.00969.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.2001.00969.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+determining+pathogenicity+to+pea+are+clustered+on+a+supernumerary+chromosome+in+the+fungal+plant+pathogen+Nectria+haematococca." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+determining+pathogenicity+to+pea+are+clustered+on+a+supernumerary+chromosome+in+the+fungal+plant+pathogen+Nectria+haematococca.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001097-Kavanaugh1" id="pone.0001097-Kavanaugh1"></a>Kavanaugh LA, Fraser JA, Dietrich FS (2006) Recent evolution of the human pathogen Cryptococcus neoformans by intervarietal transfer of a 14-gene fragment. Mol Biol Evol  23: 1879–1890.  <ul class="find" data-citedArticleID="1010876" data-doi="10.1093/molbev/msl070"><li><a href="http://dx.doi.org/10.1093/molbev/msl070" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+evolution+of+the+human+pathogen+Cryptococcus+neoformans+by+intervarietal+transfer+of+a+14-gene+fragment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+evolution+of+the+human+pathogen+Cryptococcus+neoformans+by+intervarietal+transfer+of+a+14-gene+fragment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001097-He1" id="pone.0001097-He1"></a>He C, Rusu AG, Poplawski AM, Irwin JA, Manners JM (1998) Transfer of a supernumerary chromosome between vegetatively incompatible biotypes of the fungus Colletotrichum gloeosporioides. Genetics  150: 1459–1466.  <ul class="find" data-citedArticleID="1010858" data-doi="10.1007/s002940050249"><li><a href="http://dx.doi.org/10.1007/s002940050249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transfer+of+a+supernumerary+chromosome+between+vegetatively+incompatible+biotypes+of+the+fungus+Colletotrichum+gloeosporioides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transfer+of+a+supernumerary+chromosome+between+vegetatively+incompatible+biotypes+of+the+fungus+Colletotrichum+gloeosporioides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001097-Luque1" id="pone.0001097-Luque1"></a>Luque I, Flores E, Herrero A (1993) Nitrite reductase gene from Synechococcus sp. PCC 7942: homology between cyanobacterial and higher-plant nitrite reductases. Plant Mol Biol  21: 1201–1205.  <ul class="find" data-citedArticleID="1010890" data-doi="10.1007/bf00023618"><li><a href="http://dx.doi.org/10.1007/bf00023618" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nitrite+reductase+gene+from+Synechococcus+sp.+PCC+7942%3A+homology+between+cyanobacterial+and+higher-plant+nitrite+reductases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nitrite+reductase+gene+from+Synechococcus+sp.+PCC+7942%3A+homology+between+cyanobacterial+and+higher-plant+nitrite+reductases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001097-Stolz1" id="pone.0001097-Stolz1"></a>Stolz JF, Basu P (2002) Evolution of nitrate reductase: molecular and structural variations on a common function. Chembiochem  3: 198–206.  <ul class="find" data-citedArticleID="1010932" data-doi="10.1002/1439-7633(20020301)3:2/3<198::aid-cbic198&gt;3.0.co;2-c"><li><a href="http://dx.doi.org/10.1002/1439-7633(20020301)3:2/3<198::aid-cbic198&gt;3.0.co;2-c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+nitrate+reductase%3A+molecular+and+structural+variations+on+a+common+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+nitrate+reductase%3A+molecular+and+structural+variations+on+a+common+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001097-James1" id="pone.0001097-James1"></a>James TY, Kauff F, Schoch CL, Matheny PB, Hofstetter V, et al.  (2006) Reconstructing the early evolution of Fungi using a six-gene phylogeny. Nature  443: 818–822.  <ul class="find" data-citedArticleID="1010866" data-doi="10.1038/nature05110"><li><a href="http://dx.doi.org/10.1038/nature05110" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstructing+the+early+evolution+of+Fungi+using+a+six-gene+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstructing+the+early+evolution+of+Fungi+using+a+six-gene+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001097-Foreman1" id="pone.0001097-Foreman1"></a>Foreman PK, Brown D, Dankmeyer L, Dean R, Diener S, et al.  (2003) Transcriptional regulation of biomass-degrading enzymes in the filamentous fungus Trichoderma reesei. J Biol Chem  278: 31988–31997.  <ul class="find" data-citedArticleID="1010854" data-doi="10.1074/jbc.m304750200"><li><a href="http://dx.doi.org/10.1074/jbc.m304750200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulation+of+biomass-degrading+enzymes+in+the+filamentous+fungus+Trichoderma+reesei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulation+of+biomass-degrading+enzymes+in+the+filamentous+fungus+Trichoderma+reesei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001097-Seiboth1" id="pone.0001097-Seiboth1"></a>Seiboth B, Hartl L, Salovuori N, Lanthaler K, Robson GD, et al.  (2005) Role of the bga1-encoded extracellular {beta}-galactosidase of Hypocrea jecorina in cellulase induction by lactose. Appl Environ Microbiol  71: 851–857.  <ul class="find" data-citedArticleID="1010922" data-doi="10.1128/aem.71.2.851-857.2005"><li><a href="http://dx.doi.org/10.1128/aem.71.2.851-857.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+bga1-encoded+extracellular+%7Bbeta%7D-galactosidase+of+Hypocrea+jecorina+in+cellulase+induction+by+lactose." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+bga1-encoded+extracellular+%7Bbeta%7D-galactosidase+of+Hypocrea+jecorina+in+cellulase+induction+by+lactose.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001097-Derelle1" id="pone.0001097-Derelle1"></a>Derelle E, Ferraz C, Rombauts S, Rouze P, Worden AZ, et al.  (2006) Genome analysis of the smallest free-living eukaryote Ostreococcus tauri unveils many unique features. Proc Natl Acad Sci U S A  103: 11647–11652.  <ul class="find" data-citedArticleID="1010850" data-doi="10.1073/pnas.0604795103"><li><a href="http://dx.doi.org/10.1073/pnas.0604795103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+analysis+of+the+smallest+free-living+eukaryote+Ostreococcus+tauri+unveils+many+unique+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+analysis+of+the+smallest+free-living+eukaryote+Ostreococcus+tauri+unveils+many+unique+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001097-Quesada1" id="pone.0001097-Quesada1"></a>Quesada A, Galvan A, Schnell RA, Lefebvre PA, Fernandez E (1993) Five nitrate assimilation-related loci are clustered in Chlamydomonas reinhardtii. Mol Gen Genet  240: 387–394.  <ul class="find" data-citedArticleID="1010902"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Five+nitrate+assimilation-related+loci+are+clustered+in+Chlamydomonas+reinhardtii.&amp;auth=&amp;atitle=Five+nitrate+assimilation-related+loci+are+clustered+in+Chlamydomonas+reinhardtii." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Five+nitrate+assimilation-related+loci+are+clustered+in+Chlamydomonas+reinhardtii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Five+nitrate+assimilation-related+loci+are+clustered+in+Chlamydomonas+reinhardtii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001097-Howlett1" id="pone.0001097-Howlett1"></a>Howlett BJ, Idnurm A, Heitman J (2007) Fungal pathogenesis: gene clusters unveiled as secrets within the Ustilago maydis code. Curr Biol  17: R87–90.  <ul class="find" data-citedArticleID="1010862" data-doi="10.1016/j.cub.2006.11.047"><li><a href="http://dx.doi.org/10.1016/j.cub.2006.11.047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fungal+pathogenesis%3A+gene+clusters+unveiled+as+secrets+within+the+Ustilago+maydis+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fungal+pathogenesis%3A+gene+clusters+unveiled+as+secrets+within+the+Ustilago+maydis+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001097-Abe1" id="pone.0001097-Abe1"></a>Abe Y, Suzuki T, Ono C, Iwamoto K, Hosobuchi M, et al.  (2002) Molecular cloning and characterization of an ML-236B (compactin) biosynthetic gene cluster in Penicillium citrinum. Mol Genet Genomics  267: 636–646.  <ul class="find" data-citedArticleID="1010834" data-doi="10.1007/s00438-002-0697-y"><li><a href="http://dx.doi.org/10.1007/s00438-002-0697-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+and+characterization+of+an+ML-236B+%28compactin%29+biosynthetic+gene+cluster+in+Penicillium+citrinum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+and+characterization+of+an+ML-236B+%28compactin%29+biosynthetic+gene+cluster+in+Penicillium+citrinum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001097-Fleetwood1" id="pone.0001097-Fleetwood1"></a>Fleetwood DJ, Scott B, Lane GA, Tanaka A, Johnson RD (2007) A complex ergovaline gene cluster in epichloe endophytes of grasses. Appl Environ Microbiol  73: 2571–2579.  <ul class="find" data-citedArticleID="1010852" data-doi="10.1128/aem.00257-07"><li><a href="http://dx.doi.org/10.1128/aem.00257-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+complex+ergovaline+gene+cluster+in+epichloe+endophytes+of+grasses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+complex+ergovaline+gene+cluster+in+epichloe+endophytes+of+grasses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001097-Zhang1" id="pone.0001097-Zhang1"></a>Zhang S, Monahan BJ, Tkacz JS, Scott B (2004) Indole-diterpene gene cluster from Aspergillus flavus. Appl Environ Microbiol  70: 6875–6883.  <ul class="find" data-citedArticleID="1010946" data-doi="10.1128/aem.70.11.6875-6883.2004"><li><a href="http://dx.doi.org/10.1128/aem.70.11.6875-6883.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Indole-diterpene+gene+cluster+from+Aspergillus+flavus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Indole-diterpene+gene+cluster+from+Aspergillus+flavus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001097-Laich1" id="pone.0001097-Laich1"></a>Laich F, Fierro F, Martin JF (2002) Production of penicillin by fungi growing on food products: identification of a complete penicillin gene cluster in Penicillium griseofulvum and a truncated cluster in Penicillium verrucosum. Appl Environ Microbiol  68: 1211–1219.  <ul class="find" data-citedArticleID="1010880" data-doi="10.1128/aem.68.3.1211-1219.2002"><li><a href="http://dx.doi.org/10.1128/aem.68.3.1211-1219.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Production+of+penicillin+by+fungi+growing+on+food+products%3A+identification+of+a+complete+penicillin+gene+cluster+in+Penicillium+griseofulvum+and+a+truncated+cluster+in+Penicillium+verrucosum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Production+of+penicillin+by+fungi+growing+on+food+products%3A+identification+of+a+complete+penicillin+gene+cluster+in+Penicillium+griseofulvum+and+a+truncated+cluster+in+Penicillium+verrucosum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001097-Carbone1" id="pone.0001097-Carbone1"></a>Carbone I, Jakobek JL, Ramirez-Prado JH, Horn BW (2007) Recombination, balancing selection and adaptive evolution in the aflatoxin gene cluster of Aspergillus parasiticus. Mol Ecol.   <ul class="find" data-citedArticleID="1010846" data-doi="10.1111/j.1365-294x.2007.03464.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2007.03464.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination%2C+balancing+selection+and+adaptive+evolution+in+the+aflatoxin+gene+cluster+of+Aspergillus+parasiticus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination%2C+balancing+selection+and+adaptive+evolution+in+the+aflatoxin+gene+cluster+of+Aspergillus+parasiticus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001097-Wong1" id="pone.0001097-Wong1"></a>Wong S, Wolfe KH (2005) Birth of a metabolic gene cluster in yeast by adaptive gene relocation. Nat Genet  37: 777–782.  <ul class="find" data-citedArticleID="1010944" data-doi="10.1038/ng1584"><li><a href="http://dx.doi.org/10.1038/ng1584" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Birth+of+a+metabolic+gene+cluster+in+yeast+by+adaptive+gene+relocation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Birth+of+a+metabolic+gene+cluster+in+yeast+by+adaptive+gene+relocation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001097-Samuels1" id="pone.0001097-Samuels1"></a>Samuels GJ (2006) <em>Trichoderma</em>: Systematics, the Sexual State, and Ecology. Phytopathology  96: 195–206.  <ul class="find" data-citedArticleID="1010916" data-doi="10.1094/phyto-96-0195"><li><a href="http://dx.doi.org/10.1094/phyto-96-0195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trichoderma%3A+Systematics%2C+the+Sexual+State%2C+and+Ecology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trichoderma%3A+Systematics%2C+the+Sexual+State%2C+and+Ecology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001097-Lieckfeldt1" id="pone.0001097-Lieckfeldt1"></a>Lieckfeldt E, Kullnig C, Samuels GJ, Kubicek CP (2000) Sexually competent, sucrose- and nitrate-assimilating strains of Hypocrea jecorina (Trichoderma reesei) from South American soils. . Mycologia  92: 374–380.  <ul class="find" data-citedArticleID="1010888" data-doi="10.2307/3761493"><li><a href="http://dx.doi.org/10.2307/3761493" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sexually+competent%2C+sucrose-+and+nitrate-assimilating+strains+of+Hypocrea+jecorina+%28Trichoderma+reesei%29+from+South+American+soils.+." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sexually+competent%2C+sucrose-+and+nitrate-assimilating+strains+of+Hypocrea+jecorina+%28Trichoderma+reesei%29+from+South+American+soils.+.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001097-Ostrikova1" id="pone.0001097-Ostrikova1"></a>Ostrikova NA, Konovalov SA (1983) Influence of nitrogen sources on cellulase biosynthesis by a mutant strain Trichoderma viride 44. Applied Biochemistry and Microbiology  19: 392–395.  <ul class="find" data-citedArticleID="1010896"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Influence+of+nitrogen+sources+on+cellulase+biosynthesis+by+a+mutant+strain+Trichoderma+viride+44.&amp;auth=&amp;atitle=Influence+of+nitrogen+sources+on+cellulase+biosynthesis+by+a+mutant+strain+Trichoderma+viride+44." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+nitrogen+sources+on+cellulase+biosynthesis+by+a+mutant+strain+Trichoderma+viride+44." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+nitrogen+sources+on+cellulase+biosynthesis+by+a+mutant+strain+Trichoderma+viride+44.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001097-Saikia1" id="pone.0001097-Saikia1"></a>Saikia R, Azad P (2001) Effect of Certain Carbon and Nitrogen Sources on the Antagonistic Activities of some Biocontrol Agents against Colletotrichum falcatum Went. Environment and Ecology  19: 849–852.  <ul class="find" data-citedArticleID="1010914"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effect+of+Certain+Carbon+and+Nitrogen+Sources+on+the+Antagonistic+Activities+of+some+Biocontrol+Agents+against+Colletotrichum+falcatum+Went.&amp;auth=&amp;atitle=Effect+of+Certain+Carbon+and+Nitrogen+Sources+on+the+Antagonistic+Activities+of+some+Biocontrol+Agents+against+Colletotrichum+falcatum+Went." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+Certain+Carbon+and+Nitrogen+Sources+on+the+Antagonistic+Activities+of+some+Biocontrol+Agents+against+Colletotrichum+falcatum+Went." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+Certain+Carbon+and+Nitrogen+Sources+on+the+Antagonistic+Activities+of+some+Biocontrol+Agents+against+Colletotrichum+falcatum+Went.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001097-Jayaraj1" id="pone.0001097-Jayaraj1"></a>Jayaraj J, Ramabadran R (1998) Effect of certain nitrogenous sources on the in vitro growth, sporulation and production of antifungal substances by Trichoderma harzianum. Journal of Mycology and Plant Pathology  28: 23–25.  <ul class="find" data-citedArticleID="1010870"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effect+of+certain+nitrogenous+sources+on+the+in+vitro+growth%2C+sporulation+and+production+of+antifungal+substances+by+Trichoderma+harzianum.&amp;auth=&amp;atitle=Effect+of+certain+nitrogenous+sources+on+the+in+vitro+growth%2C+sporulation+and+production+of+antifungal+substances+by+Trichoderma+harzianum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+certain+nitrogenous+sources+on+the+in+vitro+growth%2C+sporulation+and+production+of+antifungal+substances+by+Trichoderma+harzianum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+certain+nitrogenous+sources+on+the+in+vitro+growth%2C+sporulation+and+production+of+antifungal+substances+by+Trichoderma+harzianum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001097-Wakelin1" id="pone.0001097-Wakelin1"></a>Wakelin SA, Sivasithamparam K, Cole ALJ, Skipp RA (1999) Saprophytic growth in soil of a strain of Trichoderma koningii. New Zealand Journal of Agricultural Research  42: 337–345.  <ul class="find" data-citedArticleID="1010940" data-doi="10.1080/00288233.1999.9513383"><li><a href="http://dx.doi.org/10.1080/00288233.1999.9513383" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Saprophytic+growth+in+soil+of+a+strain+of+Trichoderma+koningii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Saprophytic+growth+in+soil+of+a+strain+of+Trichoderma+koningii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001097-Celar1" id="pone.0001097-Celar1"></a>Celar F (2003) Competition for ammonium and nitrate forms of nitrogen between some phytopathogenic and antagonistic soil fungi. Biological Control  28: 19–24.  <ul class="find" data-citedArticleID="1010848" data-doi="10.1016/s1049-9644(03)00049-5"><li><a href="http://dx.doi.org/10.1016/s1049-9644(03)00049-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Competition+for+ammonium+and+nitrate+forms+of+nitrogen+between+some+phytopathogenic+and+antagonistic+soil+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Competition+for+ammonium+and+nitrate+forms+of+nitrogen+between+some+phytopathogenic+and+antagonistic+soil+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001097-RochaRamirez1" id="pone.0001097-RochaRamirez1"></a>Rocha-Ramirez V, Omero C, Chet I, Horwitz BA, Herrera-Estrella A (2002) Trichoderma atroviride G-protein alpha-subunit gene tga1 is involved in mycoparasitic coiling and conidiation. Eukaryot Cell  1: 594–605.  <ul class="find" data-citedArticleID="1010908" data-doi="10.1128/ec.1.4.594-605.2002"><li><a href="http://dx.doi.org/10.1128/ec.1.4.594-605.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trichoderma+atroviride+G-protein+alpha-subunit+gene+tga1+is+involved+in+mycoparasitic+coiling+and+conidiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trichoderma+atroviride+G-protein+alpha-subunit+gene+tga1+is+involved+in+mycoparasitic+coiling+and+conidiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001097-Sarrocco1" id="pone.0001097-Sarrocco1"></a>Sarrocco S, Mikkelsen L, Vergara M, Jensen DF, Lubeck M, et al.  (2006) Histopathological studies of sclerotia of phytopathogenic fungi parasitized by a GFP transformed Trichoderma virens antagonistic strain. Mycol Res  110: 179–187.  <ul class="find" data-citedArticleID="1010918" data-doi="10.1016/j.mycres.2005.08.005"><li><a href="http://dx.doi.org/10.1016/j.mycres.2005.08.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histopathological+studies+of+sclerotia+of+phytopathogenic+fungi+parasitized+by+a+GFP+transformed+Trichoderma+virens+antagonistic+strain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histopathological+studies+of+sclerotia+of+phytopathogenic+fungi+parasitized+by+a+GFP+transformed+Trichoderma+virens+antagonistic+strain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001097-Randall1" id="pone.0001097-Randall1"></a>Randall TA, Dwyer RA, Huitema E, Beyer K, Cvitanich C, et al.  (2005) Large-scale gene discovery in the oomycete Phytophthora infestans reveals likely components of phytopathogenicity shared with true fungi. Mol Plant Microbe Interact  18: 229–243.  <ul class="find" data-citedArticleID="1010904" data-doi="10.1094/mpmi-18-0229"><li><a href="http://dx.doi.org/10.1094/mpmi-18-0229" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+gene+discovery+in+the+oomycete+Phytophthora+infestans+reveals+likely+components+of+phytopathogenicity+shared+with+true+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+gene+discovery+in+the+oomycete+Phytophthora+infestans+reveals+likely+components+of+phytopathogenicity+shared+with+true+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001097-Richards1" id="pone.0001097-Richards1"></a>Richards TA, Dacks JB, Jenkinson JM, Thornton CR, Talbot NJ (2006) Evolution of filamentous plant pathogens: gene exchange across eukaryotic kingdoms. Curr Biol  16: 1857–1864.  <ul class="find" data-citedArticleID="1010906" data-doi="10.1016/j.cub.2006.07.052"><li><a href="http://dx.doi.org/10.1016/j.cub.2006.07.052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+filamentous+plant+pathogens%3A+gene+exchange+across+eukaryotic+kingdoms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+filamentous+plant+pathogens%3A+gene+exchange+across+eukaryotic+kingdoms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001097-Stemmler1" id="pone.0001097-Stemmler1"></a>Stemmler SJ, Berthelin J (2003) Microbial activity as a major factor in the mobilization of iron in the humid tropics. European Journal of Soil Science  54: 725–733.  <ul class="find" data-citedArticleID="1010930" data-doi="10.1046/j.1351-0754.2003.0571.x"><li><a href="http://dx.doi.org/10.1046/j.1351-0754.2003.0571.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+activity+as+a+major+factor+in+the+mobilization+of+iron+in+the+humid+tropics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+activity+as+a+major+factor+in+the+mobilization+of+iron+in+the+humid+tropics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001097-Tsuda1" id="pone.0001097-Tsuda1"></a>Tsuda A, Takeda S, Saito H, Nishioka J, Nojiri Y, et al.  (2003) A mesoscale iron enrichment in the western subarctic Pacific induces a large centric diatom bloom. Science  300: 958–961.  <ul class="find" data-citedArticleID="1010938" data-doi="10.1126/science.1082000"><li><a href="http://dx.doi.org/10.1126/science.1082000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mesoscale+iron+enrichment+in+the+western+subarctic+Pacific+induces+a+large+centric+diatom+bloom." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mesoscale+iron+enrichment+in+the+western+subarctic+Pacific+induces+a+large+centric+diatom+bloom.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001097-Bailly1" id="pone.0001097-Bailly1"></a>Bailly J, Debaud JC, Verner MC, Plassard C, Chalot M, et al.  (2007) How does a symbiotic fungus modulate expression of its host-plant nitrite reductase? New Phytol  175: 155–165.  <ul class="find" data-citedArticleID="1010840" data-doi="10.1111/j.1469-8137.2007.02066.x"><li><a href="http://dx.doi.org/10.1111/j.1469-8137.2007.02066.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+does+a+symbiotic+fungus+modulate+expression+of+its+host-plant+nitrite+reductase%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+does+a+symbiotic+fungus+modulate+expression+of+its+host-plant+nitrite+reductase%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001097-Leyval1" id="pone.0001097-Leyval1"></a>Leyval C, Reid CPP (1991) Utilization of microbial siderophores by mycorrhizal and non-mycorrhizal pine roots. New Phytologist  119: 93–98.  <ul class="find" data-citedArticleID="1010886" data-doi="10.1111/j.1469-8137.1991.tb01011.x"><li><a href="http://dx.doi.org/10.1111/j.1469-8137.1991.tb01011.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Utilization+of+microbial+siderophores+by+mycorrhizal+and+non-mycorrhizal+pine+roots." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Utilization+of+microbial+siderophores+by+mycorrhizal+and+non-mycorrhizal+pine+roots.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001097-MuroPastor1" id="pone.0001097-MuroPastor1"></a>Muro-Pastor MI, Gonzalez R, Strauss J, Narendja F, Scazzocchio C (1999) The GATA factor AreA is essential for chromatin remodelling in a eukaryotic bidirectional promoter. Embo J  18: 1584–1597.  <ul class="find" data-citedArticleID="1010894" data-doi="10.1093/emboj/18.6.1584"><li><a href="http://dx.doi.org/10.1093/emboj/18.6.1584" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+GATA+factor+AreA+is+essential+for+chromatin+remodelling+in+a+eukaryotic+bidirectional+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+GATA+factor+AreA+is+essential+for+chromatin+remodelling+in+a+eukaryotic+bidirectional+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001097-Punt1" id="pone.0001097-Punt1"></a>Punt PJ, Strauss J, Smit R, Kinghorn JR, van den Hondel CA, et al.  (1995) The intergenic region between the divergently transcribed niiA and niaD genes of Aspergillus nidulans contains multiple NirA binding sites which act bidirectionally. Mol Cell Biol  15: 5688–5699.  <ul class="find" data-citedArticleID="1010900"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+intergenic+region+between+the+divergently+transcribed+niiA+and+niaD+genes+of+Aspergillus+nidulans+contains+multiple+NirA+binding+sites+which+act+bidirectionally.&amp;auth=&amp;atitle=The+intergenic+region+between+the+divergently+transcribed+niiA+and+niaD+genes+of+Aspergillus+nidulans+contains+multiple+NirA+binding+sites+which+act+bidirectionally." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+intergenic+region+between+the+divergently+transcribed+niiA+and+niaD+genes+of+Aspergillus+nidulans+contains+multiple+NirA+binding+sites+which+act+bidirectionally." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+intergenic+region+between+the+divergently+transcribed+niiA+and+niaD+genes+of+Aspergillus+nidulans+contains+multiple+NirA+binding+sites+which+act+bidirectionally.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001097-Heckman1" id="pone.0001097-Heckman1"></a>Heckman DS, Geiser DM, Eidell BR, Stauffer RL, Kardos NL, et al.  (2001) Molecular evidence for the early colonization of land by fungi and plants. Science  293: 1129–1133.  <ul class="find" data-citedArticleID="1010860" data-doi="10.1126/science.1061457"><li><a href="http://dx.doi.org/10.1126/science.1061457" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evidence+for+the+early+colonization+of+land+by+fungi+and+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evidence+for+the+early+colonization+of+land+by+fungi+and+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001097-Altschul1" id="pone.0001097-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="1010838" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001097-Thompson1" id="pone.0001097-Thompson1"></a>Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG (1997) The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res  25: 4876–4882.  <ul class="find" data-citedArticleID="1010936" data-doi="10.1093/nar/25.24.4876"><li><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001097-Swofford1" id="pone.0001097-Swofford1"></a>Swofford DL (2003) PAUP*. Phylogenetic Analysis Using Parsimony (*and Other Methods). Version 4.   Sinauer Associates, Sunderland, Massachusetts.  <ul class="find-nolinks"></ul></li><li><span class="label">50.
              </span><a name="pone.0001097-Huelsenbeck1" id="pone.0001097-Huelsenbeck1"></a>Huelsenbeck JP, Ronquist F (2001) MRBAYES: Bayesian inference of phylogenetic trees. Bioinformatics  17: 754–755.  <ul class="find" data-citedArticleID="1010864" data-doi="10.1093/bioinformatics/17.8.754"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.8.754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MRBAYES%3A+Bayesian+inference+of+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MRBAYES%3A+Bayesian+inference+of+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001097-Ronquist1" id="pone.0001097-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics  19: 1572–1574.  <ul class="find" data-citedArticleID="1010910" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001097-Altekar1" id="pone.0001097-Altekar1"></a>Altekar G, Dwarkadas S, Huelsenbeck JP, Ronquist F (2004) Parallel Metropolis coupled Markov chain Monte Carlo for Bayesian phylogenetic inference. Bioinformatics  20: 407–415.  <ul class="find" data-citedArticleID="1010836" data-doi="10.1093/bioinformatics/btg427"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg427" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+Metropolis+coupled+Markov+chain+Monte+Carlo+for+Bayesian+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+Metropolis+coupled+Markov+chain+Monte+Carlo+for+Bayesian+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001097-Stamatakis1" id="pone.0001097-Stamatakis1"></a>Stamatakis A (2006) RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models. Bioinformatics  22: 2688–2690.  <ul class="find" data-citedArticleID="1010928" data-doi="10.1093/bioinformatics/btl446"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl446" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RAxML-VI-HPC%3A+maximum+likelihood-based+phylogenetic+analyses+with+thousands+of+taxa+and+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RAxML-VI-HPC%3A+maximum+likelihood-based+phylogenetic+analyses+with+thousands+of+taxa+and+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001097-Shimodaira1" id="pone.0001097-Shimodaira1"></a>Shimodaira H, Hasegawa M (1999) Multiple Comparisons of Log-Likelihoods with Applications to Phylogenetic Inference. Mol Biol Evol  16: 1114–1116.  <ul class="find" data-citedArticleID="1010924" data-doi="10.1093/oxfordjournals.molbev.a026201"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+Comparisons+of+Log-Likelihoods+with+Applications+to+Phylogenetic+Inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+Comparisons+of+Log-Likelihoods+with+Applications+to+Phylogenetic+Inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001097-Schmidt1" id="pone.0001097-Schmidt1"></a>Schmidt HA, Strimmer K, Vingron M, von Haeseler A (2002) TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing. Bioinformatics  18: 502–504.  <ul class="find" data-citedArticleID="1010920" data-doi="10.1093/bioinformatics/18.3.502"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.3.502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TREE-PUZZLE%3A+maximum+likelihood+phylogenetic+analysis+using+quartets+and+parallel+computing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TREE-PUZZLE%3A+maximum+likelihood+phylogenetic+analysis+using+quartets+and+parallel+computing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001097-Katoh1" id="pone.0001097-Katoh1"></a>Katoh K, Misawa K, Kuma K, Miyata T (2002) MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform. Nucleic Acids Res  30: 3059–3066.  <ul class="find" data-citedArticleID="1010874" data-doi="10.1093/nar/gkf436"><li><a href="http://dx.doi.org/10.1093/nar/gkf436" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAFFT%3A+a+novel+method+for+rapid+multiple+sequence+alignment+based+on+fast+Fourier+transform." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAFFT%3A+a+novel+method+for+rapid+multiple+sequence+alignment+based+on+fast+Fourier+transform.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001097-Baldauf1" id="pone.0001097-Baldauf1"></a>Baldauf SL (2003) The deep roots of eukaryotes. Science  300: 1703–1706.  <ul class="find" data-citedArticleID="1010842" data-doi="10.1126/science.1085544"><li><a href="http://dx.doi.org/10.1126/science.1085544" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+deep+roots+of+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+deep+roots+of+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.XML" value="109297"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.PDF" value="319545"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.g001.PNG_L" value="673560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.g001.PNG_M" value="82272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.g001.PNG_S" value="9391"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.g001.TIF" value="1146610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.g001.PNG_I" value="27744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.t001.PNG_L" value="135953"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.t001.PNG_M" value="80950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.t001.PNG_S" value="10776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.t001.TIF" value="630160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.t001.PNG_I" value="24758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.g002.PNG_L" value="333437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.g002.PNG_M" value="39421"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.g002.PNG_S" value="6575"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.g002.TIF" value="588574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.g002.PNG_I" value="13921"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.t002.PNG_L" value="119579"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.t002.PNG_M" value="73838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.t002.PNG_S" value="10056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.t002.TIF" value="509170"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.t002.PNG_I" value="23019"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.s001.PDF" value="492446"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.s002.PDF" value="879661"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.s003.PDF" value="929686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.s004.DOC" value="107008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001097.s005.DOC" value="32256"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001097&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001097" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001097&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001097&volume=&issue=&title=Horizontal Transfer of a Nitrate Assimilation Gene Cluster and Ecological Transitions in Fungi: A Phylogenetic Study&author_name=Jason%20C.%20Slot%2C%20David%20S.%20Hibbett&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001097" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001097" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001097" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001097&amp;t=Horizontal%20Transfer%20of%20a%20Nitrate%20Assimilation%20Gene%20Cluster%20and%20Ecological%20Transitions%20in%20Fungi%3A%20A%20Phylogenetic%20Study" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001097&title=Horizontal%20Transfer%20of%20a%20Nitrate%20Assimilation%20Gene%20Cluster%20and%20Ecological%20Transitions%20in%20Fungi%3A%20A%20Phylogenetic%20Study&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001097&amp;title=Horizontal%20Transfer%20of%20a%20Nitrate%20Assimilation%20Gene%20Cluster%20and%20Ecological%20Transitions%20in%20Fungi%3A%20A%20Phylogenetic%20Study" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001097" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001097&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Horizontal Transfer of a Nitrate Assimilation Gene Cluster and Ecological Transitions in Fungi: A Phylogenetic Study';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001097';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Horizontal%20Transfer%20of%20a%20Nitrate%20Assimilation%20Gene%20Cluster%20and%20Ecological%20Transitions%20in%20Fungi%3A%20A%20Phylogenetic%20Study http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001097" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001097" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amino+acid+sequence+analysis%22" title="Search for articles in the subject area:'Amino acid sequence analysis'"><div class="flagText">Amino acid sequence analysis</div></a>
              <div data-categoryid="20173" data-articleid="25028"
                   data-categoryname="Amino acid sequence analysis"
                   class="flagImage" title="Flag 'Amino acid sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ascomycetes%22" title="Search for articles in the subject area:'Ascomycetes'"><div class="flagText">Ascomycetes</div></a>
              <div data-categoryid="44323" data-articleid="25028"
                   data-categoryname="Ascomycetes"
                   class="flagImage" title="Flag 'Ascomycetes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+genetics%22" title="Search for articles in the subject area:'Fungal genetics'"><div class="flagText">Fungal genetics</div></a>
              <div data-categoryid="18483" data-articleid="25028"
                   data-categoryname="Fungal genetics"
                   class="flagImage" title="Flag 'Fungal genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+genomics%22" title="Search for articles in the subject area:'Fungal genomics'"><div class="flagText">Fungal genomics</div></a>
              <div data-categoryid="33097" data-articleid="25028"
                   data-categoryname="Fungal genomics"
                   class="flagImage" title="Flag 'Fungal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungi%22" title="Search for articles in the subject area:'Fungi'"><div class="flagText">Fungi</div></a>
              <div data-categoryid="43047" data-articleid="25028"
                   data-categoryname="Fungi"
                   class="flagImage" title="Flag 'Fungi' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nitrates%22" title="Search for articles in the subject area:'Nitrates'"><div class="flagText">Nitrates</div></a>
              <div data-categoryid="21435" data-articleid="25028"
                   data-categoryname="Nitrates"
                   class="flagImage" title="Flag 'Nitrates' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="25028"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="25028"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9148'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7548'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5625&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=11807">Figure S3 B,C,D contain errors based on previously noted errors</a><br>Posted by JasonSlot</p>
      <p><a href="/annotation/listThread.action?root=8581">This statement is no longer supported after reexamination of data.</a><br>Posted by JasonSlot</p>
      <p><a href="/annotation/listThread.action?root=10529">There is an error in this figure that led to an erroneous statement in the conclusions</a><br>Posted by JasonSlot</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
