

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Correlation Index-Based Responsible-Enzyme Gene Screening (CIRES), a Novel DNA Microarray-Based Method for Enzyme Gene Involved in Glycan Biosynthesis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001232"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001232" />

    <meta name="citation_title" content="Correlation Index-Based Responsible-Enzyme Gene Screening (CIRES), a Novel DNA Microarray-Based Method for Enzyme Gene Involved in Glycan Biosynthesis"/>
    <meta itemprop="name" content="Correlation Index-Based Responsible-Enzyme Gene Screening (CIRES), a Novel DNA Microarray-Based Method for Enzyme Gene Involved in Glycan Biosynthesis"/>

      <meta name="citation_author" content="Harumi Yamamoto"/>
            <meta name="citation_author_institution" content="Laboratory of Membrane Biochemistry and Biophysics, Graduate School of Biostudies, Kyoto University, Sakyo, Kyoto, Japan"/>
            <meta name="citation_author_institution" content="Supra-Biomolecular System Research Group, RIKEN Frontier Research System, RIKEN, Wako, Saitama, Japan"/>
      <meta name="citation_author" content="Hiromu Takematsu"/>
            <meta name="citation_author_institution" content="Laboratory of Membrane Biochemistry and Biophysics, Graduate School of Biostudies, Kyoto University, Sakyo, Kyoto, Japan"/>
            <meta name="citation_author_institution" content="Core Research for Evolutional Science and Technology (CREST), Japan Science and Technology Corporation (JST), Kawaguchi, Saitama, Japan"/>
      <meta name="citation_author" content="Reiko Fujinawa"/>
            <meta name="citation_author_institution" content="Supra-Biomolecular System Research Group, RIKEN Frontier Research System, RIKEN, Wako, Saitama, Japan"/>
      <meta name="citation_author" content="Yuko Naito"/>
            <meta name="citation_author_institution" content="Laboratory of Membrane Biochemistry and Biophysics, Graduate School of Biostudies, Kyoto University, Sakyo, Kyoto, Japan"/>
            <meta name="citation_author_institution" content="Core Research for Evolutional Science and Technology (CREST), Japan Science and Technology Corporation (JST), Kawaguchi, Saitama, Japan"/>
      <meta name="citation_author" content="Yasushi Okuno"/>
            <meta name="citation_author_institution" content="Department of PharmacoInformatics, Graduate School of Pharmaceutical Sciences, Kyoto University, Sakyo, Kyoto, Japan"/>
      <meta name="citation_author" content="Gozoh Tsujimoto"/>
            <meta name="citation_author_institution" content="Department of Genomic Drug Discovery, Graduate School of Pharmaceutical Sciences, Kyoto University, Sakyo, Kyoto, Japan"/>
      <meta name="citation_author" content="Akemi Suzuki"/>
            <meta name="citation_author_institution" content="Supra-Biomolecular System Research Group, RIKEN Frontier Research System, RIKEN, Wako, Saitama, Japan"/>
      <meta name="citation_author" content="Yasunori Kozutsumi"/>
            <meta name="citation_author_institution" content="Laboratory of Membrane Biochemistry and Biophysics, Graduate School of Biostudies, Kyoto University, Sakyo, Kyoto, Japan"/>
            <meta name="citation_author_institution" content="Supra-Biomolecular System Research Group, RIKEN Frontier Research System, RIKEN, Wako, Saitama, Japan"/>
            <meta name="citation_author_institution" content="Core Research for Evolutional Science and Technology (CREST), Japan Science and Technology Corporation (JST), Kawaguchi, Saitama, Japan"/>

    <meta name="citation_date" content="2007/11/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001232.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1232"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Monosaccharide Metabolism; citation_author=HH Freeze; citation_number=1; citation_pages=69-84; citation_date=1999; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Transporters of nucleotide sugars, ATP, and nucleotide sulfate in the endoplasmic reticulum and Golgi apparatus; citation_author=CB Hirschberg; citation_author=PW Robbins; citation_author=C Abeijon; citation_journal_title=Annu Rev Biochem; citation_volume=67; citation_number=2; citation_pages=49-69 9759482; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Assembly of Asparagine-Linked Oligosaccharides; citation_author=R Kornfeld; citation_author=S Kornfeld; citation_journal_title=Annu Rev Biochem; citation_volume=54; citation_number=3; citation_pages=631-664; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning of a CD28 cDNA by a high-efficiency COS cell expression system; citation_author=A Aruffo; citation_author=B Seed; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=84; citation_number=4; citation_pages=8573-7 2825196; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning of the CD2 antigen, the T-cell erythrocyte receptor, by a rapid immunoselection procedure; citation_author=B Seed; citation_author=A Aruffo; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=84; citation_number=5; citation_pages=3365-9 2437578; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Isolation of a cDNA encoding a murine UDPgalactose:beta-D-galactosyl- 1,4-N-acetyl-D-glucosaminide alpha-1,3-galactosyltransferase: expression cloning by gene transfer; citation_author=RD Larsen; citation_author=VP Rajan; citation_author=MM Ruff; citation_author=J Kukowska-Latallo; citation_author=RD Cummings; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=86; citation_number=6; citation_pages=8227-31 2510162; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Construction of a human glycogene library and comprehensive functional analysis; citation_author=H Narimatsu; citation_journal_title=Glycoconj J; citation_volume=21; citation_number=7; citation_pages=17-24 15467393; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A focused microarray approach to functional glycomics: transcriptional regulation of the glycome; citation_author=EM Comelli; citation_author=SR Head; citation_author=T Gilmartin; citation_author=T Whisenant; citation_author=SM Haslam; citation_journal_title=Glycobiology; citation_volume=16; citation_number=8; citation_pages=117-31 16237199; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA microarray in glycobiology; citation_author=H Takematsu; citation_author=Y Kozutsumi; citation_number=9; citation_pages=428-448; citation_date=2007; citation_publisher=Elsevier; " />
      <meta name="citation_reference" content="citation_title=Prediction of glycan structures from gene expression data based on glycosyltransferase reactions; citation_author=S Kawano; citation_author=K Hashimoto; citation_author=T Miyama; citation_author=S Goto; citation_author=M Kanehisa; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=10; citation_pages=3976-82 16159923; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Germinal center marker GL7 probes activation-dependent repression of N-glycolylneuraminic acid, a sialic acid species involved in the negative modulation of B cell activation; citation_author=Y Naito; citation_author=H Takematsu; citation_author=S Koyama; citation_author=S Miyake; citation_author=H Yamamoto; citation_journal_title=Mol Cell Biol; citation_volume=27; citation_number=11; citation_pages=3008-3022; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Beta 1-6 branched oligosaccharides as a marker of tumor progression in human breast and colon neoplasia; citation_author=B Fernandes; citation_author=U Sagman; citation_author=M Auger; citation_author=M Demetrio; citation_author=JW Dennis; citation_journal_title=Cancer Res; citation_volume=51; citation_number=12; citation_pages=718-23 1985789; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Beta1,6-N-acetylglucosamine-bearing N-glycans in human gliomas: implications for a role in regulating invasivity; citation_author=H Yamamoto; citation_author=J Swoger; citation_author=S Greene; citation_author=T Saito; citation_author=J Hurh; citation_journal_title=Cancer Res; citation_volume=60; citation_number=13; citation_pages=134-42 10646865; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Suppression of tumor growth and metastasis in Mgat5-deficient mice; citation_author=M Granovsky; citation_author=J Fata; citation_author=J Pawling; citation_author=WJ Muller; citation_author=R Khokha; citation_journal_title=Nat Med; citation_volume=6; citation_number=14; citation_pages=306-12 10700233; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Regulation of cytokine receptors by Golgi N-glycan processing and endocytosis; citation_author=EA Partridge; citation_author=C Le Roy; citation_author=GM Di Guglielmo; citation_author=J Pawling; citation_author=P Cheung; citation_journal_title=Science; citation_volume=306; citation_number=15; citation_pages=120-4 15459394; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Immune regulation by the ST6Gal sialyltransferase; citation_author=T Hennet; citation_author=D Chui; citation_author=JC Paulson; citation_author=JD Marth; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=16; citation_pages=4504-9 9539767; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Evanescent-field fluorescence-assisted lectin microarray: a new strategy for glycan profiling; citation_author=A Kuno; citation_author=N Uchiyama; citation_author=S Koseki-Kuno; citation_author=Y Ebe; citation_author=S Takashima; citation_journal_title=Nat Methods; citation_volume=2; citation_number=17; citation_pages=851-6 16278656; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulation of alpha 2,3-sialyltransferase expression correlates with conversion of peanut agglutinin (PNA)+ to PNA- phenotype in developing thymocytes; citation_author=W Gillespie; citation_author=JC Paulson; citation_author=S Kelm; citation_author=M Pang; citation_author=LG Baum; citation_journal_title=J Biol Chem; citation_volume=268; citation_number=18; citation_pages=3801-4 8440675; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The ST3Gal-I sialyltransferase controls CD8+ T lymphocyte homeostasis by modulating O-glycan biosynthesis; citation_author=JJ Priatel; citation_author=D Chui; citation_author=N Hiraoka; citation_author=CJ Simmons; citation_author=KB Richardson; citation_journal_title=Immunity; citation_volume=12; citation_number=19; citation_pages=273-83 10755614; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Synthesis of 2-O-(2-acetamido-2-deoxy-beta-D-glucopyranosyl)-D-mannose, and its interaction with D-mannose-specific lectins; citation_author=R Kaifu; citation_author=T Osawa; citation_author=RW Jeanloz; citation_journal_title=Carbohydr Res; citation_volume=40; citation_number=20; citation_pages=111-7 1125946; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=The absence of fucose but not the presence of galactose or bisecting N-acetylglucosamine of human IgG1 complex-type oligosaccharides shows the critical role of enhancing antibody-dependent cellular cytotoxicity; citation_author=T Shinkawa; citation_author=K Nakamura; citation_author=N Yamane; citation_author=E Shoji-Hosaka; citation_author=Y Kanda; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=21; citation_pages=3466-73 12427744; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Dysregulation of TGF-beta1 receptor activation leads to abnormal lung development and emphysema-like phenotype in core fucose-deficient mice; citation_author=X Wang; citation_author=S Inoue; citation_author=J Gu; citation_author=E Miyoshi; citation_author=K Noda; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=22; citation_pages=15791-6 16236725; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Purification and characterization of an anti-H(O) phytohemagglutinin of Ulex europeus; citation_author=I Matsumoto; citation_author=T Osawa; citation_journal_title=Biochim Biophys Acta; citation_volume=194; citation_number=23; citation_pages=180-9 5353123; citation_date=1969; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning of a novel alpha2,3-sialyltransferase (ST3Gal VI) that sialylates type II lactosamine structures on glycoproteins and glycolipids; citation_author=T Okajima; citation_author=S Fukumoto; citation_author=H Miyazaki; citation_author=H Ishida; citation_author=M Kiso; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=24; citation_pages=11479-86 10206952; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Localization of binding sites of Ulex europaeus I, Helix pomatia and Griffonia simplicifolia I-B4 lectins and analysis of their backbone structures by several glycosidases and poly-N-acetyllactosamine-specific lectins in human breast carcinomas; citation_author=N Ito; citation_author=S Imai; citation_author=S Haga; citation_author=C Nagaike; citation_author=Y Morimura; citation_journal_title=Histochem Cell Biol; citation_volume=106; citation_number=25; citation_pages=331-9 8897074; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Beta-1,4-galactosylation of N-glycans is a complex process; citation_author=K Furukawa; citation_author=T Sato; citation_journal_title=Biochim Biophys Acta; citation_volume=1473; citation_number=26; citation_pages=54-66 10580129; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Carbohydrate binding studies on the lectin from Datura stramonium seeds; citation_author=JF Crowley; citation_author=IJ Goldstein; citation_author=J Arnarp; citation_author=J Lonngren; citation_journal_title=Arch Biochem Biophys; citation_volume=231; citation_number=27; citation_pages=524-33 6203486; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Carbohydrate binding properties of complex-type oligosaccharides on immobilized Datura stramonium lectin; citation_author=K Yamashita; citation_author=K Totani; citation_author=T Ohkura; citation_author=S Takasaki; citation_author=IJ Goldstein; citation_journal_title=J Biol Chem; citation_volume=262; citation_number=28; citation_pages=1602-7 3805046; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Effects of dibutyryl cAMP and bromodeoxyuridine on expression of N-acetylglucosaminyltransferases III and V in GOTO neuroblastoma cells; citation_author=Y Ihara; citation_author=A Nishikawa; citation_author=N Taniguchi; citation_journal_title=Glycoconj J; citation_volume=12; citation_number=29; citation_pages=787-94 8748156; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The action of N-acetylglucosaminyltransferase-V is prevented by the bisecting GlcNAc residue at the catalytic step; citation_author=K Sasai; citation_author=Y Ikeda; citation_author=H Eguchi; citation_author=T Tsuda; citation_author=K Honke; citation_journal_title=FEBS Lett; citation_volume=522; citation_number=30; citation_pages=151-5 12095636; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Selective suppression of N-acetylglucosaminyltransferase III activity in a human hepatoblastoma cell line transfected with hepatitis; citation_author=E Miyoshi; citation_author=A Nishikawa; citation_author=Y Ihara; citation_author=N Hayashi; citation_author=H Fusamoto; citation_journal_title=B virus Cancer Res; citation_volume=54; citation_number=31; citation_pages=1854-8 8137300; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Bisecting GlcNAc structures act as negative sorting signals for cell surface glycoproteins in forskolin-treated rat hepatoma cells; citation_author=AS Sultan; citation_author=E Miyoshi; citation_author=Y Ihara; citation_author=A Nishikawa; citation_author=Y Tsukada; citation_journal_title=J Biol Chem; citation_volume=272; citation_number=32; citation_pages=2866-72 9006930; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The immobilized leukoagglutinin from the seeds of Maackia amurensis binds with high affinity to complex-type Asn-linked oligosaccharides containing terminal sialic acid-linked alpha-2,3 to penultimate galactose residues; citation_author=WC Wang; citation_author=RD Cummings; citation_journal_title=J Biol Chem; citation_volume=263; citation_number=33; citation_pages=4576-85 3350806; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Strong affinity of Maackia amurensis hemagglutinin (MAH) for sialic acid-containing Ser/Thr-linked carbohydrate chains of N-terminal octapeptides from human glycophorin; citation_author=Y Konami; citation_author=K Yamamoto; citation_author=T Osawa; citation_author=T Irimura; citation_journal_title=A FEBS Lett; citation_volume=342; citation_number=34; citation_pages=334-8 8150094; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Characterization of the carbohydrate binding specificity of the leukoagglutinating lectin from Maackia amurensis. Comparison with other sialic acid-specific lectins; citation_author=RN Knibbs; citation_author=IJ Goldstein; citation_author=RM Ratcliffe; citation_author=N Shibuya; citation_journal_title=J Biol Chem; citation_volume=266; citation_number=35; citation_pages=83-8 1985926; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Selection and characterization of eight phenotypically distinct lines of lectin-resistant Chinese hamster ovary cell; citation_author=P Stanley; citation_author=V Caillibot; citation_author=L Siminovitch; citation_journal_title=Cell; citation_volume=6; citation_number=36; citation_pages=121-8 1182798; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=Differential expression of five sialyltransferase genes in human tissues; citation_author=H Kitagawa; citation_author=JC Paulson; citation_journal_title=J Biol Chem; citation_volume=269; citation_number=37; citation_pages=17872-8 8027041; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Expression of the developmental I antigen by a cloned human cDNA encoding a member of a beta-1,6-N-acetylglucosaminyltransferase gene family; citation_author=MF Bierhuizen; citation_author=MG Mattei; citation_author=M Fukuda; citation_journal_title=Genes Dev; citation_volume=7; citation_number=38; citation_pages=468-78 8449405; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Golgi alpha-mannosidase II deficiency in vertebrate systems: implications for asparagine-linked oligosaccharide processing in mammals; citation_author=KW Moremen; citation_journal_title=Biochim Biophys Acta; citation_volume=1573; citation_number=39; citation_pages=225-35 12417404; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Use of sialylated or sulfated derivatives and acrylamide copolymers of Gal beta 1,3GalNAc alpha- and GalNAc alpha- to determine the specificities of blood group T- and Tn-specific lectins and the copolymers to measure anti-T and anti-Tn antibody levels in cancer patients; citation_author=Y Chen; citation_author=RK Jain; citation_author=EV Chandrasekaran; citation_author=KL Matta; citation_journal_title=Glycoconj J; citation_volume=12; citation_number=40; citation_pages=55-62 7795413; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Evidence that Agaricus bisporus agglutinin (ABA) has dual sugar-binding specificity; citation_author=S Nakamura-Tsuruta; citation_author=J Kominami; citation_author=A Kuno; citation_author=J Hirabayashi; citation_journal_title=Biochem Biophys Res Commun; citation_volume=347; citation_number=41; citation_pages=215-20 16824489; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Carbohydrate-to-carbohydrate interaction in basic cell biology: a brief overview; citation_author=S Hakomori; citation_journal_title=Arch Biochem Biophys; citation_volume=426; citation_number=42; citation_pages=173-81 15158668; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sweet spots in functional glycomics; citation_author=JC Paulson; citation_author=O Blixt; citation_author=BE Collins; citation_journal_title=Nat Chem Biol; citation_volume=2; citation_number=43; citation_pages=238-48 16619023; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Oligosaccharide microarrays for high-throughput detection and specificity assignments of carbohydrate-protein interactions; citation_author=S Fukui; citation_author=T Feizi; citation_author=C Galustian; citation_author=AM Lawson; citation_author=W Chai; citation_journal_title=Nat Biotechnol; citation_volume=20; citation_number=44; citation_pages=1011-7 12219077; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Printed covalent glycan array for ligand profiling of diverse glycan binding proteins; citation_author=O Blixt; citation_author=S Head; citation_author=T Mondala; citation_author=C Scanlan; citation_author=ME Huflejt; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=45; citation_pages=17033-8 15563589; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A combined strategy for glycan profiling: a model study with pyridylaminated oligosaccharides; citation_author=N Kamekawa; citation_author=K Hayama; citation_author=S Nakamura-Tsuruta; citation_author=A Kuno; citation_author=J Hirabayashi; citation_journal_title=J Biochem (Tokyo); citation_volume=140; citation_number=46; citation_pages=337-47 16861248; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Non-linear normalization and background correction in one-channel cDNA microarray studies; citation_author=D Edwards; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=47; citation_pages=825-33 12724292; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Limma: linear models for microarray data.; citation_author=GK Smyth; citation_number=48; citation_pages=Chapter 23; citation_date=2005; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing.; citation_author=RDC Team; citation_number=49; citation_date=2004; citation_publisher=R Foundation for Statistical Computing; " />
      <meta name="citation_reference" content="citation_title=Plat-E: An efficient and stable system for transient packaging of retroviruses; citation_author=S Morita; citation_author=T Kojima; citation_author=T Kitamura; citation_journal_title=Gene Ther; citation_volume=7; citation_number=50; citation_pages=1063-6 10871756; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Plant Lectins.; citation_author=R Cummings; citation_number=51; citation_pages=455-468; citation_date=1999; citation_publisher=Cold Spring Harbor Laboratory Press; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001232" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Correlation Index-Based Responsible-Enzyme Gene Screening (CIRES), a Novel DNA Microarray-Based Method for Enzyme Gene Involved in Glycan Biosynthesis"/>
    <meta name="twitter:description" content="BackgroundGlycan biosynthesis occurs though a multi-step process that requires a variety of enzymes ranging from glycosyltransferases to those involved in cytosolic sugar metabolism. In many cases, glycan biosynthesis follows a glycan-specific, linear pathway. As glycosyltransferases are generally regulated at the level of transcription, assessing the overall transcriptional profile for glycan biosynthesis genes seems warranted. However, a systematic approach for assessing the correlation between glycan expression and glycan-related gene expression has not been reported previously.MethodologyTo facilitate genetic analysis of glycan biosynthesis, we sought to correlate the expression of genes involved in cell-surface glycan formation with the expression of the glycans, as detected by glycan-recognizing probes. We performed cross-sample comparisons of gene expression profiles using a newly developed, glycan-focused cDNA microarray. Cell-surface glycan expression profiles were obtained using flow cytometry of cells stained with plant lectins. Pearson's correlation coefficients were calculated for these profiles and were used to identify enzyme genes correlated with glycan biosynthesis.ConclusionsThis method, designated correlation index-based responsible-enzyme gene screening (CIRES), successfully identified genes already known to be involved in the biosynthesis of certain glycans. Our evaluation of CIRES indicates that it is useful for identifying genes involved in the biosynthesis of glycan chains that can be probed with lectins using flow cytometry."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001232.g006"/>

  <meta property="og:title" content="Correlation Index-Based Responsible-Enzyme Gene Screening (CIRES), a Novel DNA Microarray-Based Method for Enzyme Gene Involved in Glycan Biosynthesis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001232" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1177'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5993'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6149&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001232">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001232" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001232&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Correlation Index-Based Responsible-Enzyme Gene Screening (CIRES), a Novel DNA Microarray-Based Method for Enzyme Gene Involved in Glycan Biosynthesis
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Harumi Yamamoto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Laboratory of Membrane Biochemistry and Biophysics, Graduate School of Biostudies, Kyoto University, Sakyo, Kyoto, Japan, 
                  Supra-Biomolecular System Research Group, RIKEN Frontier Research System, RIKEN, Wako, Saitama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hiromu Takematsu
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:htakema@pharm.kyoto-u.ac.jp">htakema@pharm.kyoto-u.ac.jp</a></p>

                <p>Affiliations:
                  Laboratory of Membrane Biochemistry and Biophysics, Graduate School of Biostudies, Kyoto University, Sakyo, Kyoto, Japan, 
                  Core Research for Evolutional Science and Technology (CREST), Japan Science and Technology Corporation (JST), Kawaguchi, Saitama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Reiko Fujinawa, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Supra-Biomolecular System Research Group, RIKEN Frontier Research System, RIKEN, Wako, Saitama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yuko Naito, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Laboratory of Membrane Biochemistry and Biophysics, Graduate School of Biostudies, Kyoto University, Sakyo, Kyoto, Japan, 
                  Core Research for Evolutional Science and Technology (CREST), Japan Science and Technology Corporation (JST), Kawaguchi, Saitama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yasushi Okuno, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of PharmacoInformatics, Graduate School of Pharmaceutical Sciences, Kyoto University, Sakyo, Kyoto, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gozoh Tsujimoto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genomic Drug Discovery, Graduate School of Pharmaceutical Sciences, Kyoto University, Sakyo, Kyoto, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Akemi Suzuki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Supra-Biomolecular System Research Group, RIKEN Frontier Research System, RIKEN, Wako, Saitama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yasunori Kozutsumi
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Laboratory of Membrane Biochemistry and Biophysics, Graduate School of Biostudies, Kyoto University, Sakyo, Kyoto, Japan, 
                  Supra-Biomolecular System Research Group, RIKEN Frontier Research System, RIKEN, Wako, Saitama, Japan, 
                  Core Research for Evolutional Science and Technology (CREST), Japan Science and Technology Corporation (JST), Kawaguchi, Saitama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001232</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001232" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001232" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001232" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001232" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001232-g001" data-doi="info:doi/10.1371/journal.pone.0001232" data-uri="info:doi/10.1371/journal.pone.0001232.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001232-g002" data-doi="info:doi/10.1371/journal.pone.0001232" data-uri="info:doi/10.1371/journal.pone.0001232.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001232-g003" data-doi="info:doi/10.1371/journal.pone.0001232" data-uri="info:doi/10.1371/journal.pone.0001232.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001232-g004" data-doi="info:doi/10.1371/journal.pone.0001232" data-uri="info:doi/10.1371/journal.pone.0001232.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001232-g005" data-doi="info:doi/10.1371/journal.pone.0001232" data-uri="info:doi/10.1371/journal.pone.0001232.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001232-g006" data-doi="info:doi/10.1371/journal.pone.0001232" data-uri="info:doi/10.1371/journal.pone.0001232.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001232">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001232" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Glycan biosynthesis occurs though a multi-step process that requires a variety of enzymes ranging from glycosyltransferases to those involved in cytosolic sugar metabolism. In many cases, glycan biosynthesis follows a glycan-specific, linear pathway. As glycosyltransferases are generally regulated at the level of transcription, assessing the overall transcriptional profile for glycan biosynthesis genes seems warranted. However, a systematic approach for assessing the correlation between glycan expression and glycan-related gene expression has not been reported previously.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>To facilitate genetic analysis of glycan biosynthesis, we sought to correlate the expression of genes involved in cell-surface glycan formation with the expression of the glycans, as detected by glycan-recognizing probes. We performed cross-sample comparisons of gene expression profiles using a newly developed, glycan-focused cDNA microarray. Cell-surface glycan expression profiles were obtained using flow cytometry of cells stained with plant lectins. Pearson's correlation coefficients were calculated for these profiles and were used to identify enzyme genes correlated with glycan biosynthesis.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This method, designated correlation index-based responsible-enzyme gene screening (CIRES), successfully identified genes already known to be involved in the biosynthesis of certain glycans. Our evaluation of CIRES indicates that it is useful for identifying genes involved in the biosynthesis of glycan chains that can be probed with lectins using flow cytometry.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Yamamoto H, Takematsu H, Fujinawa R, Naito Y, Okuno Y, et al.  (2007) Correlation Index-Based Responsible-Enzyme Gene Screening (CIRES), a Novel DNA Microarray-Based Method for Enzyme Gene Involved in Glycan Biosynthesis. PLoS ONE 2(11):
          e1232.
            doi:10.1371/journal.pone.0001232</p><p><strong>Academic Editor: </strong>Stefan Wölfl, Universität Heidelberg, Germany</p><p><strong>Received:</strong> March 19, 2007; <strong>Accepted:</strong> November 4, 2007; <strong>Published:</strong> November 28, 2007</p><p><strong>Copyright:</strong> © 2007 Yamamoto et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by CREST, JST, a grant-in-aid program from the Ministry of Education, Culture, Sports, Science and Technology of Japan, and by RIKEN.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The biosynthesis of glycan chains is a multi-step process. First, free sugars are biosynthesized by sugar-specific metabolic pathways. Then, these sugar molecules are further metabolized to nucleotide sugars, which serve as donors for glycosyltransferases <a href="#pone.0001232-Freeze1">[1]</a>. Specific transporters move the nucleotide sugars to the endoplasmic reticulum (ER) or Golgi apparatus <a href="#pone.0001232-Hirschberg1">[2]</a>, where they are utilized by glycosyltransferases for the tandem addition of sugars to the termini of nascent glycan chains in a sugar- and linkage-specific manner <a href="#pone.0001232-Kornfeld1">[3]</a>. This lengthy glycosylation process requires a great number of different enzymes operating at various levels of synthesis.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Thus far, more than 300 enzymes and transporter genes have been reported to be involved in the metabolism and biosynthesis of different glycans in diverse cell types and at various stages. Each glycan structure has its own specific biosynthetic pathway. The introduction of cloning expression methodology <a href="#pone.0001232-Aruffo1">[4]</a>, <a href="#pone.0001232-Seed1">[5]</a> has led to the successful cloning of a glycosyltransferase and to the demonstration that overexpression of a glycosyltransferase cDNA clone can confer the capability of glycan biosynthesis in overexpressing cells <a href="#pone.0001232-Larsen1">[6]</a>. This mechanism is in contrast to that used by protein kinases, which also act via pathway-like processes but are often positively or negatively regulated by phosphorylation.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>DNA microarray technology is very powerful because it can simultaneously detect changes in the expression levels of a large number of genes. In the field of glycobiology, extensive efforts have been made to identify the genes involved in glycan biosynthesis, and many have been shown to encode glycosyltransferases of the ER or Golgi apparatus. Many of these genes have been cloned, including those encoding large enzyme families <a href="#pone.0001232-Narimatsu1">[7]</a>. Given the important role of gene transcription in the regulation of glycan biosynthesis, a glycan-focused cDNA microarray was developed to obtain the transcriptome of glycan-related genes <a href="#pone.0001232-Comelli1">[8]</a>, <a href="#pone.0001232-Takematsu1">[9]</a>. As the presentation of glycomic information on a cell surface is likely to be regulated at the level of transcription of the enzymes in biosynthetic pathways, a glycan-focused DNA microarray may prove useful in elucidating glycan expression <a href="#pone.0001232-Comelli1">[8]</a>, <a href="#pone.0001232-Kawano1">[10]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In the present study, we analyzed the glycan-related gene expression profiles for possible correlations with cellular glycan expression profiles in a cross-sample manner, using Pearson's correlation coefficient. This analysis successfully identified specific genes encoding regulatory enzymes for the biosynthesis of specific glycans, from among the candidate genes of the glycan biosynthesis pathways. We designated this method correlation index-based responsible-enzyme gene screening, or CIRES (<a href="#pone-0001232-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0001232-g001"><div class="img"><a name="pone-0001232-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001232" data-uri="info:doi/10.1371/journal.pone.0001232.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001232.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001232.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Schematic of the CIRES concept.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>The expression patterns of about 1000 glycan-related genes were profiled in a set of six different cell lines (A–F) by comparing the microarray binding of cellular cDNA and reference polyA(+) RNA and calculating the relative expression values (<a href="#pone.0001232.s001">Table S1</a>). The polygons in the left web graphs represent the relative gene expression profiles of eight glycan-related genes selected as examples. In these graphs, the difference in relative gene expression is expressed on a log scale, where the edge of the polygon corresponds to the strongest expression in each cell line (A–F). The same set of six cell lines were examined for cell-surface glycan expression using fluorescently labeled plant lectins and flow cytometry; the strength of the glycan expression is plotted as relative values among the six lines, where the edge of the polygon represents the strongest expression (web graph on top right). The glycan expression profiles were analyzed for correlations with the glycan-related gene expression profiles. Similarities and dissimilarities between the profiles were assessed using Pearson's correlation coefficient, which has values ranging from -1 (no correlation) to 1 (perfect correlation). A complete list of the genes found to be positively or negatively correlated with plant lectin staining patterns is presented in <a href="#pone.0001232.s002">Table S2</a>. Genes known to affect the biosynthesis of an epitope were selected from among the correlated genes (shown for each lectin in the tables on the right in <a href="#pone-0001232-g002">Figures 2</a>–<a href="#pone-0001232-g003"></a><a href="#pone-0001232-g004"></a><a href="#pone-0001232-g005"></a><a href="#pone-0001232-g006">6</a>). A correlated gene identified by CIRES was confirmed as the gene responsible for regulating the biosynthesis of a particular glycan by transferring the gene into another cell line of the set, via gene transfer techniques such as retrovirus-mediated overexpression, and looking for a related change in epitope expression.</p>
<span>doi:10.1371/journal.pone.0001232.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Glycan-related gene-expression profiling using cDNA microarrays</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The rat monoclonal antibody GL7 specifically stains germinal center B cells upon T cell-dependent antigen immunization. We recently demonstrated that GL7 recognizes the glycan Neu5Acα<sub>2–6</sub>-Galβ<sub>1–4</sub>-GlcNAc-R and that the sialyltransferase gene <em>ST6GAL1</em> is responsible for the biosynthesis of the glycan epitope recognized by GL7 <a href="#pone.0001232-Naito1">[11]</a>. By analyzing the correlation between the expression profiles for sialic acid (Sia) metabolism-related genes and the expression profiles for the GL7 epitope in a cross-sample manner, we showed that this type of correlation analysis was useful in screening for genes involved in the biosynthesis of the glycan epitope. In the present study, we further developed this systematic methodology by analyzing the correlations for cross-sample comparisons between the expression profiles for glycan-related genes and the expression profiles for various glycans, as determined by specific lectin binding (<a href="#pone-0001232-g001">Figure 1</a>).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The glycan-related gene expression profiles were obtained using total RNA isolated from six human B-cell lines cultured under optimal conditions, and cross-sample comparisons of these profiles were made in relation to a commercially available universal reference RNA consisting of a mixture of polyA(+) RNA from various organs. The gene expression profiles were determined as a ratio of the gene expression level to the universal reference cDNA expression level on the glycan-focused microarray (<a href="#pone.0001232-Naito1">[11]</a>; the complete relative gene expression profiles are shown in <a href="#pone.0001232.s001">Table S1</a>). Thus, the glycan-related gene expression profiles are expressed as the ratio of the gene expression signal at each spot on the microarray relative to the reference RNA signal.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>After staining the cells with various anti-glycan probes (lectins) of known specificity, we determined the glycan expression profiles using flow cytometry. For each cell line, the transcriptional profile of glycan-related genes was used for cross-sample correlation analysis with the glycan expression profile.</p>


<h4>Cell-surface glycan expression profiling using flow cytometric detection of lectin staining</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Cell-surface glycan expression has been extensively studied using plant lectins that recognize specific glycan epitopes. To evaluate whether correlation analyses of lectin staining and glycan-related gene expression might provide useful information, we first performed lectin staining of a set of human B cells (Daudi, KMS-12BM, KMS-12PE, Namalwa, Raji, and Ramos) to obtain their cross-sample profiles of lectin epitope expression. We analyzed the strength of the correlations using Pearson's correlation coefficient, which is a standard, well-established method for assessing correlation. To prevent possible bias in the lectin choice, we used 15 plant lectins supplied in two commercially available sets.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To evaluate the efficacy of the calculations, the lectins were first divided into two groups based on the presence or absence of previous reports asserting a correlation between cell surface expression of a specific lectin epitope and expression of a certain glycosyltransferase gene. The lectins that lacked a reported correlation were divided into highly specific (or narrow) and broadly specific groups. The highly specific (narrow) lectins were further assigned to one of two subgroups according to the position of the epitope (terminal or interior) on the glycan chain.</p>


<h4>CIRES correlation analyses of lectin staining profiles obtained using lectins with epitopes regulated by known biosynthetic enzyme genes</h4>
<h5><em>Phaseolus vulgaris</em> leukoaggulutinin (PHA-L4).</h5><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1"></a><p>PHA-L4 recognizes tri- or tetraantennary N-glycans with β<sub>1–6</sub> branching of N-acetylglucosamine (GlcNAc), which often correlates with tumor progression <a href="#pone.0001232-Fernandes1">[12]</a>. Histochemical and immunoblot analyses have shown that PHA-L4 epitope expression correlates with the expression of the <em>MGAT5</em> (<em>GnT-V</em>) gene <a href="#pone.0001232-Yamamoto1">[13]</a>, and this lectin is commonly used as a marker for β<sub>1–6</sub>-branched N-glycans. PHA-L4 epitope expression is diminished in <em>Mgat5</em>-null mice <a href="#pone.0001232-Granovsky1">[14]</a>, and these mice exhibit enhanced rates of cytokine receptor internalization and subsequent cytokine signaling <a href="#pone.0001232-Partridge1">[15]</a>. <em>MGAT5</em> expression was strongly correlated with the PHA-L4 staining profile, as shown in <a href="#pone-0001232-g002">Figure 2A</a>. The possible values of Pearson's correlation coefficient range between 1 and −1, where a value of 1 indicates complete correlation; therefore, the coefficient index between PHA-L4 staining and <em>MGAT5</em> expression (CI = 0.93) represents a highly significant correlation. Other correlated glycan-related genes were judged to be irrelevant to the biosynthesis of this epitope and are listed in <a href="#pone.0001232.s002">Table S2</a>, which contains the complete list of microarray-wide correlations for glycan-biosynthesizing genes.</p>
<div class="figure" id="pone-0001232-g002"><div class="img"><a name="pone-0001232-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001232" data-uri="info:doi/10.1371/journal.pone.0001232.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001232.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001232.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>CIRES analyses of staining profiles obtained using lectins with known epitope expression-regulating enzymes.</span></strong></p><a id="article1.body1.sec2.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec3.sec1.fig1.caption1.p1"></a><p>(A, C, D) Expected glycan structures for lectin recognition (left), web graphs of the lectin staining profiles (depicted as polygons) obtained using a set of six B-cell lines (middle), and the correlation indexes (CI, Pearson's correlation coefficient for profile matching) of the relevant genes that correlated with the plant lectin staining profiles and the <em>P</em> values of the correlations (right). The correlation orders of the glycan-related genes selected from the complete list of correlated genes (<a href="#pone.0001232.s002">Table S2</a>) are indicated as numbers in parentheses in the box for each gene, with a smaller number indicating a stronger correlation between gene expression and glycan expression profiles. Genes with a negative correlation are indicated by an N before the order number. The lectins used were (A) PHA-L4, (C) SSA, and (D) PNA. Lectin epitopes shown in the figures are taken from the literature unless otherwise specified <a href="#pone.0001232-Kuno1">[17]</a>, <a href="#pone.0001232-Cummings1">[51]</a>. (B) Namalwa cells were infected with MSCV harboring <em>MGAT5-IRES-EGFP</em>. Control cells were infected with empty vector (<em>IRES-EGFP</em>) or the same vector encoding <em>B3GNT2</em> or <em>MGAT3</em>. Flow cytometry results for PHA-L4 staining were compared between EGFP-positive cells (solid line) and EGFP-negative cells (dashed line).</p>
<span>doi:10.1371/journal.pone.0001232.g002</span></div><a id="article1.body1.sec2.sec3.sec1.p2" name="article1.body1.sec2.sec3.sec1.p2"></a><p>The CIRES analysis correctly predicted that the <em>MGAT5</em> gene was responsible for expression of the PHA-L4 epitope. This prediction was confirmed by retrovirus-mediated gene expression in Namalwa B cells (<a href="#pone-0001232-g002">Figure 2B</a>). When a modified murine stem cell virus (MSCV) vector carrying genes for MGAT5 and enhanced green fluorescent protein (EGFP) divided with internal ribosomal entry site (IRES) <em>(MGAT5-IRES-EGFP)</em> was introduced into Namalwa cells, the level of PHA-L4 epitope expression was higher in the EGFP-positive population than in the EGFP-negative population. To rule out the possibility that viral infection somehow altered the cell surface glycan independently of glycosyltransferase expression, the vector carrying only <em>IRES-EGFP</em> was used as a negative control. In Namalwa cells expressing only EGFP, the EGFP-positive and EGFP-negative populations expressed identical levels of the PHA-L4 determinant (<a href="#pone-0001232-g002">Figure 2B</a>).</p>

<h5><em>Sambucus sieboldiana</em> aggulutinin (SSA).</h5><a id="article1.body1.sec2.sec3.sec2.p1" name="article1.body1.sec2.sec3.sec2.p1"></a><p>SSA recognizes α<sub>2–6</sub>-linked Sia bound to galactose (Gal) or N-acetylgalactosamine (GalNAc). We previously showed that SSA epitope expression is induced in CHO cells by stable transfection with the rat <em>ST6GAL1</em> gene <a href="#pone.0001232-Naito1">[11]</a>. The deletion of <em>St6gal1</em> in mice eliminated the expression of the <em>Sambucus nigra</em> agglutinin (SNA) epitope <a href="#pone.0001232-Hennet1">[16]</a>, which is also recognized by SSA. In the present study, the correlation index was assessed to determine whether <em>ST6GAL1</em> gene expression correlated with SSA epitope expression, as determined by flow cytometry, in six B-cell lines. Although SSA staining in the six B-cell lines varied in intensity (<a href="#pone-0001232-g002">Figure 2C</a>), the staining profiles correlated with the gene expression profiles for <em>ST6GAL1</em> and a few other GlcNAc-transferase genes, including two β<sub>1–6</sub> GlcNAc transferases and B3GNT5, which is involved in the biosynthesis of N-acetyllactosamine (LacNAc) units on glycan chains. These findings indicate that the SSA epitope detected by flow cytometry might be located at the terminus of poly-LacNAc units, which are often found on the β<sub>1–6</sub> branch, and might extend beyond the glycocalyx of the cell surface.</p>

<h5><em>Arachis hypogaea</em> agglutinin (PNA).</h5><a id="article1.body1.sec2.sec3.sec3.p1" name="article1.body1.sec2.sec3.sec3.p1"></a><p>PNA recognizes the Gal-exposed core-1 structure (Galβ<sub>1–3</sub>GalNAc-Thr/Ser), and the capping of this epitope by sialylation severely reduces the affinity of the interaction <a href="#pone.0001232-Kuno1">[17]</a>. Diminished PNA epitope expression reportedly coincides with an increase in α<sub>2–3</sub> sialyltransferase activity, which sialylates the Gal residue <a href="#pone.0001232-Gillespie1">[18]</a>. This change was shown to occur during thymocyte maturation, in which PNA-positive cortex cells mature into PNA-negative medulla thymocytes. In a mouse model, the deletion of <em>St3gal1</em> caused a deficiency in the derepression of PNA reactivity during thymocyte development and eventually resulted in deficient CD8<sup>+</sup> T cell maturation <a href="#pone.0001232-Priatel1">[19]</a>.</p>
<a id="article1.body1.sec2.sec3.sec3.p2" name="article1.body1.sec2.sec3.sec3.p2"></a><p>The above findings suggest that the expression pattern of the PNA epitope might be positively affected by core-1 glycan biosynthesis and negatively affected by capping. Indeed, we found a negative correlation between PNA epitope expression and the <em>ST3GAL1</em> expression profile (<a href="#pone-0001232-g002">Figure 2D</a>). Thus, our correlation index analysis is able to not only identify a positive correlation but also reliably predict a negative correlation for a gene involved in the expression of a lectin glycan epitope.</p>
<a id="article1.body1.sec2.sec3.sec3.p3" name="article1.body1.sec2.sec3.sec3.p3"></a><p>Taken together, these results suggest that correlation indexing can be used to identify genes responsible for regulating cell surface expression of glycan epitopes, as determined by flow cytometry based on lectin binding. We designated this methodology as correlation index-based responsible-enzyme gene screening, or CIRES. After confirming that CIRES could be used to predict the genes involved in the biosynthesis of the glycan epitopes for these lectins (<a href="#pone-0001232-g002">Figure 2</a>), we used CIRES to assess the genes responsible for the staining profiles of other plant lectins, as determined by flow cytometry.</p>



<h4>CIRES correlation analyses of lectin staining profiles obtained using lectins that recognize specific terminal glycan structures and have unknown epitope expression-regulating enzyme genes</h4>
<h5><em>Lens culinaris</em> aggulutinin (LCA).</h5><a id="article1.body1.sec2.sec4.sec1.p1" name="article1.body1.sec2.sec4.sec1.p1"></a><p>We assessed the staining profiles of lectins that recognize terminal structures of glycan chains. LCA recognizes the biantennary N-glycan chain with core α<sub>1–6</sub> linked fucose (Fuc) attached to the chitobiose <a href="#pone.0001232-Kaifu1">[20]</a>. The presence of a core Fuc in the N-glycan of the Fc region of IgG severely represses the antibody-dependent cellular cytotoxicity activity of the antibody <a href="#pone.0001232-Shinkawa1">[21]</a>. The expression of <em>FUT8</em> has been shown to be responsible for the biosynthesis of a core Fuc on N-glycans <a href="#pone.0001232-Wang1">[22]</a>, but a correlation between <em>FUT8</em> gene expression and cell surface LCA staining has not been demonstrated in flow cytometry experiments.</p>
<a id="article1.body1.sec2.sec4.sec1.p2" name="article1.body1.sec2.sec4.sec1.p2"></a><p>Our analysis of the LCA staining profile and <em>FUT8</em> expression profile revealed a correlation (<a href="#pone-0001232-g003">Figure 3A</a>), although it was weaker than those for the three lectins described above (<a href="#pone-0001232-g002">Figure 2</a>). We also noted that <em>MGAT4b</em> gene expression negatively correlated with the LCA staining profile (<a href="#pone.0001232.s002">Table S2</a>). Considering that the presence of additional antennae on the N-glycan inhibits LCA binding <a href="#pone.0001232-Kuno1">[17]</a>, this type of negative correlation could be quite informative; however, in this case, no evidence was reported indicating that <em>MGAT4b</em> expression reduces the detection of the LCA epitope by flow cytometry. Nevertheless, CIRES is useful in predicting the genes involved positively or negatively in the biosynthesis of glycan epitopes.</p>
<div class="figure" id="pone-0001232-g003"><div class="img"><a name="pone-0001232-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001232" data-uri="info:doi/10.1371/journal.pone.0001232.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001232.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001232.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>CIRES analyses of staining profiles obtained using lectins that recognize terminal glycan structures and have unknown epitope-expression-regulating enzymes.</span></strong></p><a id="article1.body1.sec2.sec4.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec4.sec1.fig1.caption1.p1"></a><p>Presentation is the same as in <a href="#pone-0001232-g002">Fig. 2</a> except that the plant lectins used were (A) LCA, (B) UEA-I, and (C) RCA 120. N.D. in the gene order list indicates that no gene was determined to have a correlation with the lectin staining.</p>
<span>doi:10.1371/journal.pone.0001232.g003</span></div>
<h5><em>Ulex europaeus</em> agglutinin-I (UEA-I).</h5><a id="article1.body1.sec2.sec4.sec2.p1" name="article1.body1.sec2.sec4.sec2.p1"></a><p>UEA-I recognizes α<sub>1–2</sub>-linked Fuc on type-2 LacNAc, which is involved in forming the epitope of H-type human red blood cell antigen <a href="#pone.0001232-Matsumoto1">[23]</a>. UEA-I staining did not reveal a significant positive correlation with the expression of the gene for α<sub>1–2</sub> fucosyltransferase, which is involved in the biosynthesis of this linkage (<a href="#pone-0001232-g003">Figure 3B</a>). Instead, a prominent negative correlation was found with the expression profile of <em>ST3GAL6</em>, which has a preference for type-2 LacNAc substrates on both glycoproteins and glycolipids <a href="#pone.0001232-Okajima1">[24]</a>.</p>
<a id="article1.body1.sec2.sec4.sec2.p2" name="article1.body1.sec2.sec4.sec2.p2"></a><p>In theory, UEA-I binding should be affected by the expression of <em>FUT1</em> or <em>FUT2</em>, as they encode the proteins responsible for H antigen biosynthesis, and by the expression of A or B transferase, which can cap the H antigen to reduce the affinity <a href="#pone.0001232-Ito1">[25]</a>. However, the sequence similarity between the A and B (and also O) transferase genes prevented their differentiation in the microarray experiments. Redundant regulation by <em>FUT1</em> and <em>FUT2</em> in these cells may be the reason that no positive correlation with UEA-I epitope expression was observed. Alternatively, these data may suggest that negatively correlated ST3GAL6, which utilizes the same substrate as fucosyltransferases, may compete with the biosynthesis of this epitope by prior sialylation of the fucosyltransferase substrate(s).</p>

<h5><em>Ricinus communis</em> agglutinin (RCA120).</h5><a id="article1.body1.sec2.sec4.sec3.p1" name="article1.body1.sec2.sec4.sec3.p1"></a><p>RCA120 preferentially recognizes terminal LacNAc structures found in various classes of glycans. These LacNAc structures are biosynthesized by a large group of B4GalT <a href="#pone.0001232-Furukawa1">[26]</a> and proximal GlcNAc-transferase family enzymes. The RCA120 staining profile revealed no obvious correlation with the genes for the enzymes known to be involved in this biosynthetic pathway (<a href="#pone-0001232-g003">Figure 3C</a>). Given our earlier correlation results identifying a terminal glycosyltransferase as being responsible for LacNAc expression (<em>i.e</em>., sialyltransferase for SSA), this result was not surprising. It indicates that the abundant expression of LacNAc structures ensures the detection of a correlation between a terminal enzyme expression profile and the expression of a terminal glycan detected by flow cytometry. Moreover, capping of LacNAc should have a negative effect on its recognition by RCA120, which would make the detection of epitope expression more complex. It was clear that this procedure is not universally effective for lectin epitopes but that the effectiveness of the procedure depends on the type of glycan recognized by a lectin.</p>



<h4>CIRES correlation analyses of lectin staining profiles obtained using lectins that recognize specific internal glycan structures and have unknown epitope expression-regulating enzyme genes</h4>
<h5><em>Datura stramonium</em> agglutinin (DSA).</h5><a id="article1.body1.sec2.sec5.sec1.p1" name="article1.body1.sec2.sec5.sec1.p1"></a><p>DSA recognizes tri- and tetraantennary N-glycans. It is specific for GlcNAc β<sub>1–4</sub>-Man α<sub>1–3</sub>-branched triantennary N-glycan <a href="#pone.0001232-Crowley1">[27]</a>, <a href="#pone.0001232-Yamashita1">[28]</a>, which is biosynthesized by MGAT4a and MGAT4b. In our experiments, the DSA staining profile resembled that of PHA-L4, and thus the two lectins correlated with similar genes, most prominently <em>MGAT5</em> (<a href="#pone-0001232-g004">Figure 4A</a>). This result could be explained by the fact that the addition of a β<sub>1–6</sub> branch increases the preference of DSA for a ligand, even though MGAT4a/b activity is required. Ihara et al. have reported that DSA staining correlates with the expression level of <em>MGAT5</em> in <em>in vitro</em>-differentiated GOTO cells <a href="#pone.0001232-Ihara1">[29]</a>, suggesting that <em>MGAT5</em> may also be involved in the biosynthesis of the optimal DSA epitope, with a tetraanntenary glycan. Consistent with this idea, the introduction of <em>MGAT5</em> into Namalwa cells resulted in a 60% increase in DSA staining (Mean fluorescence intensity (MFI), 1986), compared with control (MFI, 1247) (<a href="#pone-0001232-g004">Figure 4B</a>). Interestingly, when <em>MGAT3</em> was introduced into Namalwa cells, DSA epitope expression was subtly suppressed (MFI, 961) compared with control expression (MFI, 1222), possibly due to the competitive relationship between MGAT3 and MGAT5 <a href="#pone.0001232-Sasai1">[30]</a> (<a href="#pone-0001232-g004">Figure 4B</a>). These effects appeared to be specific, because no obvious shift was seen in cells with introduced <em>B3GNT2</em>.</p>
<div class="figure" id="pone-0001232-g004"><div class="img"><a name="pone-0001232-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001232" data-uri="info:doi/10.1371/journal.pone.0001232.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001232.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001232.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>CIRES analyses of staining profiles obtained using lectins that recognize internal glycan structures and have unknown epitope-expression-regulating enzymes.</span></strong></p><a id="article1.body1.sec2.sec5.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec5.sec1.fig1.caption1.p1"></a><p>Presentation is the same as in <a href="#pone-0001232-g002">Fig. 2</a> except that the plant lectins used were (A, B) DSA and (C–E) PHA-E4. N.D. in the gene order list indicates that no significant correlation was detected. (B, D) Flow cytometric staining patterns for EGFP-positive Namalwa cells are shown in bold lines, and those for EGFP-negative (control) cells are shown in gray dashed lines. The overexpression of <em>MGAT5</em> resulted in a subtle (60%) increase in DSA staining. The overexpression of <em>MGAT3</em> resulted in a 2-fold increase in PHA-E4 staining. (E) PHA-E4 lectin blotting was performed using the membrane fraction of the same set of cell lines. A plot of the quantified signals reveals differences in the PHA-E4 staining profile among the six cell lines (C, E), as discussed in the text.</p>
<span>doi:10.1371/journal.pone.0001232.g004</span></div>
<h5><em>Phaseolus vulgaris</em> erythroagglutinin (PHA-E4).</h5><a id="article1.body1.sec2.sec5.sec2.p1" name="article1.body1.sec2.sec5.sec2.p1"></a><p>The staining profile of PHA-E4 was similar to those of PHA-L4 and DSA. This result was unexpected because PHA-E4 recognizes bisecting GlcNAc-containing biantennary N-glycans, which comprise a type of glycan distinct from the PHA-L4 epitope. Owing to the similarity among the staining patterns of these three lectins, a correlation was also found between PHA-E4 staining and <em>MGAT5</em> (<a href="#pone.0001232.s002">Table S2</a>), but PHA-E4 staining did not correlate with <em>MGAT3</em> (<em>GnT-III</em>), which is the GlcNAc transferase gene expected to correlate by virtue of its known epitope specificity (<a href="#pone-0001232-g004">Figure 4C</a>). However, when we overexpressed <em>MGAT3</em> in Namalwa cells, the MFI value of PHA-E4 staining increased, from 873 in the control population to 1868 in the EGFP-positive population (<a href="#pone-0001232-g004">Figure 4D</a>). Thus, the expression level of <em>MGAT3</em> appears to be important for PHA-E4 epitope biosynthesis, as expected. The overexpression of <em>MGAT5</em> had no effect on PHA-E4 binding; the MFI value was 899 in the control population and 866 in the EGFP-positive population.</p>
<a id="article1.body1.sec2.sec5.sec2.p2" name="article1.body1.sec2.sec5.sec2.p2"></a><p>When we stained the membrane fractions from the six B-cell lines using PHA-E4 in lectin-blot analyses, the blot and FACS signal strengths differed, as seen in the shape of the staining profile (<a href="#pone-0001232-g004">Figure 4E</a>), and <em>MGAT3</em> expression did not correlate with the signal strength on the lectin blot. Somewhat consistent with our result, Miyoshi et al. reported that <em>MGAT3</em> expression levels did not necessarily correlate with cell-surface staining of the PHA-E4 ligand in flow cytometry experiments, although co-expression was found in lectin-blot experiments <a href="#pone.0001232-Miyoshi1">[31]</a>. Thus, our results support the suggestion of Miyoshi et al. that the cell-surface expression level of the PHA-E4 ligand epitope may be regulated by factor(s) other than <em>MGAT3</em> expression. Consistent with this idea, they also reported that the presence of bisecting GlcNAc negatively affected the sorting of glycoproteins to the cell surface <a href="#pone.0001232-Sultan1">[32]</a>.</p>



<h4>CIRES correlation analyses of lectin staining profiles obtained using lectins that recognize multiple glycan structures</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Some of the lectins used in the present study had mixed or heterologous specificity. We assessed the correlation indexes for the staining profiles of these lectins.</p>
<h5><em>Maackia amurensis</em> lectin (MAM).</h5><a id="article1.body1.sec2.sec6.sec1.p1" name="article1.body1.sec2.sec6.sec1.p1"></a><p>MAM is a mixture of two lectin subunits, MAL and MAH. MAL binds to Sia α<sub>2–3</sub>-LacNAc structures <a href="#pone.0001232-Wang2">[33]</a>, whereas MAH preferentially recognizes disialylated structures found in O-glycans <a href="#pone.0001232-Konami1">[34]</a>. Of the known sialyltransferases, ST3GAL3, ST3GAL4, or ST3GAL6 may synthesize the MAL epitope, and ST8s may synthesize disialylated glycans. Correlation-index analyses showed that <em>ST3GAL3</em> and <em>B3GNT2</em> may be responsible for the expression of the epitope in the six B-cell lines (<a href="#pone-0001232-g005">Figure 5A</a>). This is consistent with a previous report that repeating LacNAc units enhance MAL binding <a href="#pone.0001232-Knibbs1">[35]</a>. The MAL binding preference seemed to be more important than that of MAH in this CIRES prediction based on MAM staining and flow cytometry. As expected from its positive correlation with <em>B3GNT2</em> expression, the MAM epitope showed increased levels in Namalwa cells overexpressing <em>B3GNT2</em>, whereas overexpressed <em>MGAT3</em> was negatively correlated with the MAM staining profile, owing to the suppression of MAM epitope expression (<a href="#pone-0001232-g005">Figure 5B</a>). Thus, the MAM epitope may be preferentially biosynthesized on LacNAc units of the β<sub>1–6</sub> branch of N-glycans. Alternatively, MGAT3 expression may change the sorting of the protein carrying the MAM epitope. Taken together, these results indicate that the expression of correlated genes can have an additive regulatory effect (positive or negative) on the cell-surface presentation of a lectin epitope.</p>
<div class="figure" id="pone-0001232-g005"><div class="img"><a name="pone-0001232-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001232" data-uri="info:doi/10.1371/journal.pone.0001232.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001232.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001232.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>CIRES analyses of staining profiles obtained using lectins that recognize multiple glycan structures and have unknown epitope expression-regulating enzymes.</span></strong></p><a id="article1.body1.sec2.sec6.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec6.sec1.fig1.caption1.p1"></a><p>Presentation is the same as in <a href="#pone-0001232-g002">Fig. 2</a> except that the plant lectins used were (A–B) MAM, (C) WGA, and (D) Con-A. The epitopes of the two different lectins of MAM, MAL and MAH, are illustrated separately. WGA essentially recognizes a cluster of N-acetyl groups, as indicated, and thus required Neu5Ac as a Sia species. Con-A recognizes mannose-containing glycans with varying affinities. High-mannose-type glycans (upper diagram in (D)) bind best to this lectin. (B) Namalwa cells were infected with retroviruses encoding various GlcNAc transferases. The MAM staining patterns of the EGFP-positive populations of each infectant are shown. <em>MGAT5</em> overexpression did not shift the staining pattern in comparison with the vector control (data not shown).</p>
<span>doi:10.1371/journal.pone.0001232.g005</span></div>
<h5><em>Triticum vulgaris</em> agglutinin, wheat germ agglutinin (WGA).</h5><a id="article1.body1.sec2.sec6.sec2.p1" name="article1.body1.sec2.sec6.sec2.p1"></a><p>WGA is thought to preferentially recognize clustered N-acetyl groups found in N-acetylneuraminic acid (Neu5Ac), GlcNAc, and GalNAc. Neu5Ac is often a major WGA ligand because the Sia density on the termini of glycan chains tends to increase for the highly branched N-linked glycans <a href="#pone.0001232-Kuno1">[17]</a>. The affinity of WGA for Sia was exploited in the isolation of Lec mutants in CHO cells <a href="#pone.0001232-Stanley1">[36]</a>. The density of the N-acetyl group can also be high in the I-branched β<sub>1–6</sub> GlcNAc-containing glycans <a href="#pone.0001232-Kuno1">[17]</a>.</p>
<a id="article1.body1.sec2.sec6.sec2.p2" name="article1.body1.sec2.sec6.sec2.p2"></a><p>The WGA staining profile correlated strongly with the expression profile of the <em>ST6GAL1</em> gene and weakly with that of the <em>ST3GAL3</em> gene (<a href="#pone-0001232-g005">Figure 5C</a>). (These genes were previously known as <em>ST6N</em> and <em>ST3N</em>, respectively <a href="#pone.0001232-Kitagawa1">[37]</a>.) Since WGA binding to sialylated glycans increases with the degree of sialylation, this correlation pattern seems to indicate that the supply of the substrate LacNAc is ample and that expression of the terminal sialyltransferases determines the expression level of the WGA epitope. Among these sialyltransferases, <em>ST6GAL1</em> appeared to play a more prominent role in biosynthesis in the B-cell lines used in this study. In addition to the I-branching β<sub>1–6</sub> GlcNAc transferase <a href="#pone.0001232-Bierhuizen1">[38]</a>, <em>GCNT2</em> expression also correlated with WGA staining, probably reflecting the GlcNAc-binding aspect of WGA. In a case such as this one, the correlation of the genes identified by CIRES may appear to be additive, but a lack of exclusivity tends to reduce the correlation index for each gene.</p>

<h5><em>Canavalia ensiformis</em> lectin (Con-A).</h5><a id="article1.body1.sec2.sec6.sec3.p1" name="article1.body1.sec2.sec6.sec3.p1"></a><p>The modification of N-glycans occurs following the transfer of lipid-linked Glc<sub>3</sub>-Man<sub>9</sub>-GlcNAc<sub>2</sub> to nascent N-glycosylated protein by oligosaccharyltransferase in the ER <a href="#pone.0001232-Moremen1">[39]</a>. Con-A recognizes mannose (Man)-containing N-glycans to various degrees; it binds preferentially to high-mannose N-glycans, followed by hybrid-type N-glycans, and has the least affinity for complex-type N-glycans. In theory, oligosaccharyltransferase activity is required for the expression of the Con-A epitope, whereas mannosidase expression is inhibitory. Con-A staining profiles revealed a subtle but significant negative correlation with the expression of the mannosidase genes <em>MAN1B1</em> and <em>MAN2A2</em> (<a href="#pone-0001232-g005">Figure 5D</a>). Thus, the mannosidases encoded by these genes may control the expression of high-mannose N-glycans on the cell surface. Alternatively, other factors such as the expression of the protein moiety of the glycoconjugate could determine the expression of the Con-A epitope, given that the number of proteins reported to carry high-mannose glycans is limited.</p>

<h5><em>Agaricus bisporus</em> aggulutinin (ABA).</h5><a id="article1.body1.sec2.sec6.sec4.p1" name="article1.body1.sec2.sec6.sec4.p1"></a><p>ABA recognizes Gal-exposed core-1 structures that are similar to the PNA epitope, but in contrast to PNA, ABA can recognize sialylated structures <a href="#pone.0001232-Chen1">[40]</a>. In B cells, ABA staining was similar to PNA staining and was negatively correlated with <em>ST3GAL1</em> expression (<a href="#pone-0001232-g006">Figure 6A</a>). Consistent with this negative correlation, ABA binding was increased in Daudi cells treated with sialidase (<a href="#pone-0001232-g006">Figure 6B</a>), similar to the enhanced binding of PNA observed upon sialidase treatment. This effect occurred with both a broad-range sialidase (<em>Arthrobacter ureafaciens</em> sialidase, AUS) and the α<sub>2,3</sub>-linked Sia-specific <em>Salmonella typhimurium</em> sialidase (<a href="#pone-0001232-g006">Figure 6B</a>), indicating that sialylation, which probably occurs on the core-1 Gal residue, somehow inhibits recognition by ABA.</p>
<div class="figure" id="pone-0001232-g006"><div class="img"><a name="pone-0001232-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001232" data-uri="info:doi/10.1371/journal.pone.0001232.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001232.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001232.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001232.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001232.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001232.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>CIRES analyses of staining profiles obtained using the lectin ABA.</span></strong></p><a id="article1.body1.sec2.sec6.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec6.sec4.fig1.caption1.p1"></a><p>(A) Presentation is the same as in <a href="#pone-0001232-g002">Fig. 2</a> except that ABA was used. (B) Effect of sialidase treatment on the binding of ABA and PNA in B cells. (See text for the specificity of the sialidase and <a href="#pone-0001232-g002">Fig. 2D</a> for the PNA epitope.) Mean fluorescence intensity (MFI) values for the staining with each lectin (bold lines) are shown at the top. Dashed lines indicate the results from the non-staining control.</p>
<span>doi:10.1371/journal.pone.0001232.g006</span></div><a id="article1.body1.sec2.sec6.sec4.p2" name="article1.body1.sec2.sec6.sec4.p2"></a><p><em>B3GNT5</em> expression positively correlated with ABA staining. In fact, a recent study has shown that ABA has dual specificity for glycan chains, recognizing both Gal-exposed O-glycans and GlcNAc-exposed N-glycans <a href="#pone.0001232-NakamuraTsuruta1">[41]</a>. Whether the GlcNAc residue biosynthesized by the GlcNAc transferase is uncapped on the cell surface is unknown, but some ABA binding to GlcNAc may contribute to the increased correlation index of <em>B3GNT5</em> as compared with that for PNA, which showed an otherwise similar staining profile for the correlated genes.</p>



<h4>Overall findings</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>As confirmed by staining with various plant lectins, CIRES successfully identified enzyme genes known to be involved in the biosynthesis of lectin-binding determinants. When an unbiased set of 15 lectins was analyzed for binding to six B-cell lines, 12 of the lectins showed significant staining. Correlation assessment of these staining profiles identified the enzyme genes that are apparently responsible for the expression of the specific epitopes for nine lectins. In general, lectins that recognize terminal structures of the glycan chain tended to yield the most reliable correlation with the responsible genes.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>Interestingly, CIRES also found negative correlations for some epitopes, which is consistent with the fact that CIRES results are highly dependent on the regulatory mechanisms of glycan epitope expression, some of which are negative (<em>i.e.</em>, capping of an epitope by further glycosylation). Finding negative relationships is difficult in normal biological experimental setups but may have been possible using CIRES because of the unbiased correlation coefficient calculation resulting from the large set of cross-sample comparisons.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>CIRES correlation analysis of glycan-related gene expression and binding of anti-glycan probes such as lectins</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The functional glycans expressed on a cell surface can encode biological information. Although glycan-glycan interactions are important in determining the biological consequences of some glycosylations <a href="#pone.0001232-Hakomori1">[42]</a>, glycan-binding proteins are the major target of functional glycans. Thus, the binding of glycan-specific proteins can be highly informative in decoding the glycomic information of an organism <a href="#pone.0001232-Paulson1">[43]</a>, making glycan-binding proteins a rational choice for the analysis of glycan expression. Although the identification of the proteins that bind best to each glycan is no doubt important given the role of glycan-binding proteins in glycan recognition in an organism <a href="#pone.0001232-Fukui1">[44]</a>, <a href="#pone.0001232-Blixt1">[45]</a>, here we opted to identify the genes involved in the biosynthesis of specific glycans because, as the glycan-biosynthesis enzymes are responsible for producing the glycan ligands, this information will help elucidate the regulation of biological events mediated by glycan-binding proteins.</p>


<h4>Systematic methodological merit of CIRES</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The experimental setup for CIRES is relatively simple. Briefly, we first made a cross-sample comparison of glycan-related gene expression profiles using cDNA microarrays. This technique yielded the relative expression level for each gene as compared with a universal reference RNA, producing an expression profile for each gene. We then used anti-glycan lectin probes to obtain binding profiles in the same cell lines and compared the two types of profiles. We used the same cross-sample profiles of glycan-related gene expression for the correlation analyses of all the lectins examined in this study.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The cells that we used for gene expression profiling are available either commercially or from a non-profit cell resource center (Japanese Collection of Research Bioresources; JCRB). All cells were cultured under commonly used conditions and were used in logarithmic growth phase. Under these conditions, the glycan expression phenotype obtained by lectin staining tended to be uniform within a cell line and unique for each cell line. Thus, the data obtained by lectin staining with excess probe were suitable for determining the mean fluorescence intensity (MFI) of lectin staining.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>This method compared the relative glycan expression profiles of cell lines whose cellular size and character might not be uniform; therefore, we normalized the staining signal by using the ratio of the stained sample MFI to the control MFI. This eliminates the absolute glycan expression signals and normalizes the relative expression results to the staining and data acquisition conditions.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>The statistical analysis is the core of the CIRES method. Since glycosylation follows a defined pathway, we could have set up an algorithm suitable for the correlation analyses. However, we chose to use the well-established Pearson's correlation coefficient for the analyses. Regardless of the calculation method, correlations were detected even when the profiles did not completely match. Based on these calculations, a list of genes can be ordered according to the strength of their correlation. This list might be useful in designing further experiments to confirm biological functions.</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>In glycan biosynthesis, not all relationships are positive in nature, and some lectins used in this study yielded negative correlations. Thus, it is possible to detect dominant inhibitors of specific glycan biosynthesis steps using CIRES. Moreover, in theory, CIRES is not limited to glycan biosynthesis but could be used in any system for which numeric phenotypic expression results (such as glycan expression) can be obtained for a set of cultured cells. The correlation of non-glycosylation phenotypes with gene expression in the same system should be possible.</p>


<h4>Systematic methodology for using microarray data in CIRES</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>One limitation of the standard microarray technique is that it can only detect the relative cDNA expression levels of two samples. This was actually useful when we took the ratio of the signal for each gene relative to that for reference RNA, thus circumventing the microarray problems associated with spot-to-spot variation and hybridization variation for different nucleotide sequences. Almost all of the spots hybridized with universal reference RNA (data not shown). Normal, uniform cell culture conditions were used in this experiment to maximize reproducibility.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>In order to calculate the gene expression ratios in a cross-sample manner, we wanted to avoid negative signals in the microarray scans. To do this, we directly used the raw data, instead of applying a cut-off value to the microarray signal by deducting the local background signal from the raw spot signals. We consider this to be a valid option, because we confirmed that the scanner readout after hybridization was reproducible, even for spots yielding weaker signals, and that both options yielded a similar order of relative strength (data not shown).</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Normalization was also important because the array results were compared in a cross-sample manner. Sum-based normalization was applied to obtain better correlation coefficients for proven lectins (data not shown). The validity of the calculations was dependent on the use of a consistent standard, making a universal reference essential for the cross-sample comparison. In general, the microarray results tended to be more consistent for stronger hybridization signals than for weaker signals.</p>


<h4>Expected and unexpected findings with CIRES</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>For an epitope whose expression is regulated by a single gene product in its biosynthetic pathway and for which the supply of substrate for that biosynthetic reaction is abundant, CIRES will identify the gene involved in the biosynthesis of the epitope. The finding that the RCA120staining profile did not correlate with a specific glycosyltransferase was therefore not surprising, because it binds LacNAc. Generally, we obtained better correlation coefficients for epitopes created by terminal modifications of LacNAc. CIRES was also useful in identifying major genes responsible for epitope capping, which is negatively correlated with glycan expression, as in the case of PNA.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Our CIRES results tended to show correlations with the GlcNAc transferases. This may result from the use of flow cytometry to detect lectin binding, and thus glycan expression, because lectin staining preferentially detects glycans that extend beyond the glycocalyx. Moreover, the level of epitope expression may not linearly correlate with the lectin signal, because lectins tend to form multimers. As poly-LacNAcs are the usual core units for the presentation of some functional glycan epitopes, GlcNAc transferases found to correlate could also be important for the epitope presentation suitable for lectin-based recognition.</p>
<a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>Thus far, we have not identified any enzymes other than glycosyltransferases that strongly correlate with epitope expression levels. However, a subtle negative correlation was noted between the expression of the Con-A epitope and mannosidase expression. Our failure to identify additional enzymes might have resulted from the limited number of lectins available to us. In theory, other factors involved in biosynthetic pathways, such as factors involved in sugar-nucleotide biosynthesis or transport, could also regulate glycan expression on the cell surface.</p>


<h4>Limitations of CIRES</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Specific probes are unquestionably important for the successful application of CIRES. The most important aspect of lectin specificity is not affinity for the glycan ligand but rather the exclusivity of the enzyme. To calculate statistically significant correlations, a set of cell lines expressing different amounts of the target glycan should be used. The uniform expression of the target glycan in the cell set is undesirable.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>To evaluate this methodology, we used the plant lectins that have been used extensively in other glycan studies. Since the correlations are identified by statistical calculations alone, we first assessed those lectins that had already been correlated with specific enzyme genes, to test for the identification of genes that matched glycan expression only by chance. Therefore, when CIRES is applied using uncharacterized glycan-binding probes, the biological relevance of the correlated genes must be confirmed by altering their expression in the cells. This requirement could be viewed as a limitation of the system. However, the alteration of target glycan expression has been an enduring and major objective of glycobiological experiments concerning lectins. As indicated in <a href="#pone-0001232-g001">Figure 1</a>, the CIRES methodology includes confirmation of the correlation by using glycan-related gene overexpression or silencing. Confirming biological relevance via the transfer of a correlated gene has the additional benefit of providing cells with a different expression level of the epitope of interest, which can be valuable for further assessment of lectin recognition in biological systems. Changing the cell surface expression of glycans for specific anti-glycan probes has been difficult, because the rate-limiting glycosyltransferase must be identified and overexpressed. Thus, allowing for a glycan-related gene transfer procedure that produces glycan-altered cells for further experiments is a major merit of the CIRES methodology.</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>CIRES alone cannot identify the specific glycan structure to which an anti-glycan probe binds; at the same time, knowledge of the structure of interest does not necessarily mean that modifying its expression on cells is possible. Using the CIRES method will likely result in the production of cells with modified glycan expression, but the actual structure bound by the probe may be unknown. This situation is somewhat similar to that of glycan-array binding studies, in which the probe that binds best on the glycan array does not necessarily bind the same glycan on the array as on cells and the identification of the glycan structure does not always result in the identification of the relevant biosynthetic enzyme. Thus, CIRES could be combined with conventional glycan-binding assays <a href="#pone.0001232-Fukui1">[44]</a>–<a href="#pone.0001232-Kamekawa1">[46]</a> to determine the specificity of glycan binding in a set of glycans and to identify the gene that modulates glycan expression on the cell surface, from among the pathway component enzymes. Thus, CIRES is a highly useful genetic strategy for studying the functionality of the interactions between glycans and glycan-binding proteins in cell-based systems.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Reagents and cell culture</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Two commercially available sets of biotin-conjugated plant lectins (Plant Lectin Set I and II) were obtained from Seikagaku (Tokyo, Japan). R-Phycoerythrin-conjugated streptavidin was obtained from Caltag (USA). The B-cell lines Daudi, Namalwa, Raji, Ramos, KMS-BM, and KMS-PE were obtained from the Japanese Collection of Research Bioresources and were cultured in RPMI 1640 medium supplemented with 10% fetal bovine serum, sodium pyruvate, non-essential amino acids, and 2-mercaptoethanol.</p>


<h4>DNA microarrays</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Gene expression profiling of the six B-cell lines was performed using a glycan-focused cDNA microarray (RIKEN human glycogene microarray, version 1) and a GEO Platform (GPL #3465) and compared with reference RNA (Clontech, USA) to create GEO Series GSE 4407, as reported elsewhere <a href="#pone.0001232-Naito1">[11]</a>.</p>


<h4>Flow cytometry</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A total of 2.5 × 10<sup>5</sup> B cells in 100 µl FACS buffer (1% BSA and NaN<sub>3</sub> in PBS(–)) was incubated with excess biotinylated plant lectin probes at room temperature for 30 min. R-Phycoerythrin- or FITC-conjugated streptavidin was used to detect lectin binding. Data were obtained using FACScan or FACSCaliber (BD Biosciences, USA) and analyzed using FlowJo (Tristar, USA) or Cellquest software (BD Biosciences, USA). To cross-compare staining signals between cell lines, the mean fluorescence intensity (MFI) of the background staining was adjusted to around 10, and the relative staining signal was expressed as the ratio of sample MFI divided by the control MFI.</p>


<h4>Statistical analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Fluorochrome signals on the microarray were acquired by an array scanner (Affymetrix 428) without background subtraction and were then background-corrected using a smoothing function <a href="#pone.0001232-Edwards1">[47]</a>. They were then Lowess normalized using Linear Models for Microarray Data (LIMMA) <a href="#pone.0001232-Smyth1">[48]</a> and the software program R <a href="#pone.0001232-Team1">[49]</a>. Inter-array normalization was not used in cross-sample comparisons, as it seemed to cause over-normalization.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>The signal from the B-cell lines was divided by the signal from the universal reference RNA <a href="#pone.0001232-Naito1">[11]</a> to obtain the relative expression profile for each gene in each cell line (<a href="#pone.0001232.s001">Table S1</a>). The gene expression profiles were compared with the lectin staining profiles obtained by flow cytometry. Similarities between the profiles were evaluated with Pearson's correlation coefficient, and probability values (<em>P</em>) were calculated using the correlation coefficient test. For a sample size of six, a correlation coefficient of 0.81 indicates a statistical significance level of 5%. The genes that correlated with lectin staining by this method were ranked according to correlation strength (<a href="#pone.0001232.s002">Table S2</a>), and this list was examined for genes that appeared to be relevant to previously reported lectin glycan epitopes and glycan biosynthetic pathways (<a href="#pone-0001232-g001">Figure 1</a>).</p>


<h4>Retrovirus-mediated transduction of glycan-related genes</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Retroviruses were prepared and were used to infect Namalwa cells, as reported previously <a href="#pone.0001232-Naito1">[11]</a>. Briefly, full-length glycosyltransferase cDNA was cloned into a modified MSCV vector, which expressed cDNA and EGFP via an internal ribosome entry site. The plasmid was transiently transfected into Plat-A packaging cells <a href="#pone.0001232-Morita1">[50]</a>, and retrovirus-containing supernatants were collected. Namalwa cells were spin-infected (2000 rpm, 32°C, 120 min) with the retrovirus in the presence of 6 µg/ml polybrene.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>The retrovirus-infected cells were cultured for 2 days after infection before analysis by flow cytometry. EGFP-positive and -negative cells were regarded as infected and non-infected cells, respectively. The staining of these two populations was used as the control.</p>


<h4>Lectin blotting</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>B cells were sonicated in detergent-free lysis buffer [10 mM Tris-HCl (pH 7.6), 1 mM DTT, 1 mM EDTA, 0.25 M sucrose, and protease inhibitor cocktail (Roche)]. Postnuclear supernatants were further ultracentrifuged, and the pellets (membrane fraction) were resuspended in lysis buffer. The suspensions were subjected to lectin blotting with HRP-conjugated PHA-E4. The signal intensity was measured by exposure of the membrane to LAS300 (Fujifilm, Japan).</p>


<h4>Sialidase treatment</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Sialidase treatment was carried out as reported elsewhere <a href="#pone.0001232-Naito1">[11]</a>. Briefly, Daudi cells were incubated with sialidase in 100 mM sodium acetate (pH 5.2) at room temperature prior to lectin staining. Sialidases from <em>Arthrobacter ureafaciens</em> (AUS; Calbiochem, San Diego, USA) and <em>Salmonella typhimurium</em> (Takara, Kusatsu, Japan) were used.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001232"></div><a name="pone.0001232.s001" id="pone.0001232.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001232.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Complete gene expression profiles obtained from cDNA microarray: Fluorescent Cy3 (universal reference) and Cy5 (each Bcell line) readout data were acquired from the array scanner and following statistical calculations were applied using the software program R. First, background was corrected using “Edward method”. Each data was then normalized using Linear Models for Microarray Data (LIMMA) to correct bias between fluorescent dyes. Finally, the Cy5 signal from the B cell lines was divided by the Cy3 signal to obtain the relative expression profile for each spot and resultant relative values were shown in columns I through N for each cells. Column A shows serial number of all spots on microarray and column B through E show physical location of all spots on glass slide-based cDNA microarray.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001232.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.31 MB XLS)</p>
<a name="pone.0001232.s002" id="pone.0001232.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001232.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Lists of correlated genes with lectin staining: This is an Exel file consists of 14 worksheets. First sheet shows full list of genes spotted on the Glycan-focuesed microarray and their gene ID numbers. Following sheets are the lists of genes that exhibited stronger correlation with indicated lectin staining profiles. See “<a href="#s4">Materials and Methods</a>” for the method applied for calculation.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001232.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.40 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank Dr. Hisashi Narimatsu (AIST) for full-length cDNA clones of glycosyltransferases and Dr. Toshio Kitamura (University of Tokyo) for Plat-A packaging cells. We also thank Drs. Atsushi Kuno (AIST) and Eiji Miyoshi (Osaka University) for helpful discussions about lectin determinants.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: HT YK HY AS YO. Performed the experiments: YN RF HY. Analyzed the data: HT YN RF HY YO GT. Contributed reagents/materials/analysis tools: HT YK AS YO GT. Wrote the paper: HT.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001232-Freeze1" id="pone.0001232-Freeze1"></a>Freeze HH (1999)  Monosaccharide Metabolism. In: Varki A, Cummings R, Esko JD, Freeze HH, Hart GW, Marth J, editors. Essentials of Glycobiology. New York: Cold Spring Harbor Laboratory Press. pp. 69–84.  <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001232-Hirschberg1" id="pone.0001232-Hirschberg1"></a>Hirschberg CB, Robbins PW, Abeijon C (1998) Transporters of nucleotide sugars, ATP, and nucleotide sulfate in the endoplasmic reticulum and Golgi apparatus Annu Rev Biochem  67: 49–69 9759482.  <ul class="find" data-citedArticleID="1022832" data-doi="10.1146/annurev.biochem.67.1.49"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.67.1.49" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transporters+of+nucleotide+sugars%2C+ATP%2C+and+nucleotide+sulfate+in+the+endoplasmic+reticulum+and+Golgi+apparatus" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transporters+of+nucleotide+sugars%2C+ATP%2C+and+nucleotide+sulfate+in+the+endoplasmic+reticulum+and+Golgi+apparatus%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001232-Kornfeld1" id="pone.0001232-Kornfeld1"></a>Kornfeld R, Kornfeld S (1985) Assembly of Asparagine-Linked Oligosaccharides Annu Rev Biochem  54: 631–664.  <ul class="find" data-citedArticleID="1022850" data-doi="10.1146/annurev.bi.54.070185.003215"><li><a href="http://dx.doi.org/10.1146/annurev.bi.54.070185.003215" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assembly+of+Asparagine-Linked+Oligosaccharides" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assembly+of+Asparagine-Linked+Oligosaccharides%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001232-Aruffo1" id="pone.0001232-Aruffo1"></a>Aruffo A, Seed B (1987) Molecular cloning of a CD28 cDNA by a high-efficiency COS cell expression system Proc Natl Acad Sci U S A  84: 8573–7 2825196.  <ul class="find" data-citedArticleID="1022800" data-doi="10.1073/pnas.84.23.8573"><li><a href="http://dx.doi.org/10.1073/pnas.84.23.8573" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+of+a+CD28+cDNA+by+a+high-efficiency+COS+cell+expression+system" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+of+a+CD28+cDNA+by+a+high-efficiency+COS+cell+expression+system%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001232-Seed1" id="pone.0001232-Seed1"></a>Seed B, Aruffo A (1987) Molecular cloning of the CD2 antigen, the T-cell erythrocyte receptor, by a rapid immunoselection procedure Proc Natl Acad Sci U S A  84: 3365–9 2437578.  <ul class="find" data-citedArticleID="1022880" data-doi="10.1073/pnas.84.10.3365"><li><a href="http://dx.doi.org/10.1073/pnas.84.10.3365" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+of+the+CD2+antigen%2C+the+T-cell+erythrocyte+receptor%2C+by+a+rapid+immunoselection+procedure" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+of+the+CD2+antigen%2C+the+T-cell+erythrocyte+receptor%2C+by+a+rapid+immunoselection+procedure%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001232-Larsen1" id="pone.0001232-Larsen1"></a>Larsen RD, Rajan VP, Ruff MM, Kukowska-Latallo J, Cummings RD, et al.  (1989) Isolation of a cDNA encoding a murine UDPgalactose:beta-D-galactosyl- 1,4-N-acetyl-D-glucosaminide alpha-1,3-galactosyltransferase: expression cloning by gene transfer Proc Natl Acad Sci U S A  86: 8227–31 2510162.  <ul class="find" data-citedArticleID="1022854" data-doi="10.1073/pnas.86.21.8227"><li><a href="http://dx.doi.org/10.1073/pnas.86.21.8227" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+a+cDNA+encoding+a+murine+UDPgalactose%3Abeta-D-galactosyl-+1%2C4-N-acetyl-D-glucosaminide+alpha-1%2C3-galactosyltransferase%3A+expression+cloning+by+gene+transfer" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+a+cDNA+encoding+a+murine+UDPgalactose%3Abeta-D-galactosyl-+1%2C4-N-acetyl-D-glucosaminide+alpha-1%2C3-galactosyltransferase%3A+expression+cloning+by+gene+transfer%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001232-Narimatsu1" id="pone.0001232-Narimatsu1"></a>Narimatsu H (2004) Construction of a human glycogene library and comprehensive functional analysis Glycoconj J  21: 17–24 15467393.  <ul class="find" data-citedArticleID="1022868" data-doi="10.1023/b:glyc.0000043742.99482.01"><li><a href="http://dx.doi.org/10.1023/b:glyc.0000043742.99482.01" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+a+human+glycogene+library+and+comprehensive+functional+analysis" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+a+human+glycogene+library+and+comprehensive+functional+analysis%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001232-Comelli1" id="pone.0001232-Comelli1"></a>Comelli EM, Head SR, Gilmartin T, Whisenant T, Haslam SM, et al.  (2006) A focused microarray approach to functional glycomics: transcriptional regulation of the glycome Glycobiology  16: 117–31 16237199.  <ul class="find" data-citedArticleID="1022808" data-doi="10.1093/glycob/cwj048"><li><a href="http://dx.doi.org/10.1093/glycob/cwj048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+focused+microarray+approach+to+functional+glycomics%3A+transcriptional+regulation+of+the+glycome" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+focused+microarray+approach+to+functional+glycomics%3A+transcriptional+regulation+of+the+glycome%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001232-Takematsu1" id="pone.0001232-Takematsu1"></a>Takematsu H, Kozutsumi Y (2007)  DNA microarray in glycobiology. In: Greet-Jan B, Lee YC, Suzuki A, Taniguchi N, Voragen AGJ, editors. Comprehensive Glycoscience vol. 2. Oxford: Elsevier. pp. 428–448.  <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0001232-Kawano1" id="pone.0001232-Kawano1"></a>Kawano S, Hashimoto K, Miyama T, Goto S, Kanehisa M (2005) Prediction of glycan structures from gene expression data based on glycosyltransferase reactions Bioinformatics  21: 3976–82 16159923.  <ul class="find" data-citedArticleID="1022842" data-doi="10.1093/bioinformatics/bti666"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti666" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+glycan+structures+from+gene+expression+data+based+on+glycosyltransferase+reactions" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+glycan+structures+from+gene+expression+data+based+on+glycosyltransferase+reactions%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001232-Naito1" id="pone.0001232-Naito1"></a>Naito Y, Takematsu H, Koyama S, Miyake S, Yamamoto H, et al.  (2007) Germinal center marker GL7 probes activation-dependent repression of N-glycolylneuraminic acid, a sialic acid species involved in the negative modulation of B cell activation Mol Cell Biol  27: 3008–3022.  <ul class="find" data-citedArticleID="1022864" data-doi="10.1128/mcb.02047-06"><li><a href="http://dx.doi.org/10.1128/mcb.02047-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Germinal+center+marker+GL7+probes+activation-dependent+repression+of+N-glycolylneuraminic+acid%2C+a+sialic+acid+species+involved+in+the+negative+modulation+of+B+cell+activation" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Germinal+center+marker+GL7+probes+activation-dependent+repression+of+N-glycolylneuraminic+acid%2C+a+sialic+acid+species+involved+in+the+negative+modulation+of+B+cell+activation%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001232-Fernandes1" id="pone.0001232-Fernandes1"></a>Fernandes B, Sagman U, Auger M, Demetrio M, Dennis JW (1991) Beta 1-6 branched oligosaccharides as a marker of tumor progression in human breast and colon neoplasia Cancer Res  51: 718–23 1985789.  <ul class="find" data-citedArticleID="1022816"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Beta+1-6+branched+oligosaccharides+as+a+marker+of+tumor+progression+in+human+breast+and+colon+neoplasia&amp;auth=&amp;atitle=Beta+1-6+branched+oligosaccharides+as+a+marker+of+tumor+progression+in+human+breast+and+colon+neoplasia" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Beta+1-6+branched+oligosaccharides+as+a+marker+of+tumor+progression+in+human+breast+and+colon+neoplasia" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Beta+1-6+branched+oligosaccharides+as+a+marker+of+tumor+progression+in+human+breast+and+colon+neoplasia%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001232-Yamamoto1" id="pone.0001232-Yamamoto1"></a>Yamamoto H, Swoger J, Greene S, Saito T, Hurh J, et al.  (2000) Beta1,6-N-acetylglucosamine-bearing N-glycans in human gliomas: implications for a role in regulating invasivity Cancer Res  60: 134–42 10646865.  <ul class="find" data-citedArticleID="1022898"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Beta1%2C6-N-acetylglucosamine-bearing+N-glycans+in+human+gliomas%3A+implications+for+a+role+in+regulating+invasivity&amp;auth=&amp;atitle=Beta1%2C6-N-acetylglucosamine-bearing+N-glycans+in+human+gliomas%3A+implications+for+a+role+in+regulating+invasivity" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Beta1%2C6-N-acetylglucosamine-bearing+N-glycans+in+human+gliomas%3A+implications+for+a+role+in+regulating+invasivity" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Beta1%2C6-N-acetylglucosamine-bearing+N-glycans+in+human+gliomas%3A+implications+for+a+role+in+regulating+invasivity%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001232-Granovsky1" id="pone.0001232-Granovsky1"></a>Granovsky M, Fata J, Pawling J, Muller WJ, Khokha R, et al.  (2000) Suppression of tumor growth and metastasis in Mgat5-deficient mice Nat Med  6: 306–12 10700233.  <ul class="find" data-citedArticleID="1022826" data-doi="10.1038/73163"><li><a href="http://dx.doi.org/10.1038/73163" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suppression+of+tumor+growth+and+metastasis+in+Mgat5-deficient+mice" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suppression+of+tumor+growth+and+metastasis+in+Mgat5-deficient+mice%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001232-Partridge1" id="pone.0001232-Partridge1"></a>Partridge EA, Le Roy C, Di Guglielmo GM, Pawling J, Cheung P, et al.  (2004) Regulation of cytokine receptors by Golgi N-glycan processing and endocytosis Science  306: 120–4 15459394.  <ul class="find" data-citedArticleID="1022872" data-doi="10.1126/science.1102109"><li><a href="http://dx.doi.org/10.1126/science.1102109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+cytokine+receptors+by+Golgi+N-glycan+processing+and+endocytosis" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+cytokine+receptors+by+Golgi+N-glycan+processing+and+endocytosis%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001232-Hennet1" id="pone.0001232-Hennet1"></a>Hennet T, Chui D, Paulson JC, Marth JD (1998) Immune regulation by the ST6Gal sialyltransferase Proc Natl Acad Sci U S A  95: 4504–9 9539767.  <ul class="find" data-citedArticleID="1022830" data-doi="10.1073/pnas.95.8.4504"><li><a href="http://dx.doi.org/10.1073/pnas.95.8.4504" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immune+regulation+by+the+ST6Gal+sialyltransferase" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immune+regulation+by+the+ST6Gal+sialyltransferase%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001232-Kuno1" id="pone.0001232-Kuno1"></a>Kuno A, Uchiyama N, Koseki-Kuno S, Ebe Y, Takashima S, et al.  (2005) Evanescent-field fluorescence-assisted lectin microarray: a new strategy for glycan profiling Nat Methods  2: 851–6 16278656.  <ul class="find" data-citedArticleID="1022852" data-doi="10.1038/nmeth803"><li><a href="http://dx.doi.org/10.1038/nmeth803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evanescent-field+fluorescence-assisted+lectin+microarray%3A+a+new+strategy+for+glycan+profiling" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evanescent-field+fluorescence-assisted+lectin+microarray%3A+a+new+strategy+for+glycan+profiling%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001232-Gillespie1" id="pone.0001232-Gillespie1"></a>Gillespie W, Paulson JC, Kelm S, Pang M, Baum LG (1993) Regulation of alpha 2,3-sialyltransferase expression correlates with conversion of peanut agglutinin (PNA)+ to PNA- phenotype in developing thymocytes J Biol Chem  268: 3801–4 8440675.  <ul class="find" data-citedArticleID="1022824"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+alpha+2%2C3-sialyltransferase+expression+correlates+with+conversion+of+peanut+agglutinin+%28PNA%29%2B+to+PNA-+phenotype+in+developing+thymocytes&amp;auth=&amp;atitle=Regulation+of+alpha+2%2C3-sialyltransferase+expression+correlates+with+conversion+of+peanut+agglutinin+%28PNA%29%2B+to+PNA-+phenotype+in+developing+thymocytes" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+alpha+2%2C3-sialyltransferase+expression+correlates+with+conversion+of+peanut+agglutinin+%28PNA%29%2B+to+PNA-+phenotype+in+developing+thymocytes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+alpha+2%2C3-sialyltransferase+expression+correlates+with+conversion+of+peanut+agglutinin+%28PNA%29%2B+to+PNA-+phenotype+in+developing+thymocytes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001232-Priatel1" id="pone.0001232-Priatel1"></a>Priatel JJ, Chui D, Hiraoka N, Simmons CJ, Richardson KB, et al.  (2000) The ST3Gal-I sialyltransferase controls CD8+ T lymphocyte homeostasis by modulating O-glycan biosynthesis Immunity  12: 273–83 10755614.  <ul class="find" data-citedArticleID="1022876" data-doi="10.1016/s1074-7613(00)80180-6"><li><a href="http://dx.doi.org/10.1016/s1074-7613(00)80180-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ST3Gal-I+sialyltransferase+controls+CD8%2B+T+lymphocyte+homeostasis+by+modulating+O-glycan+biosynthesis" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ST3Gal-I+sialyltransferase+controls+CD8%2B+T+lymphocyte+homeostasis+by+modulating+O-glycan+biosynthesis%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001232-Kaifu1" id="pone.0001232-Kaifu1"></a>Kaifu R, Osawa T, Jeanloz RW (1975) Synthesis of 2-O-(2-acetamido-2-deoxy-beta-D-glucopyr​anosyl)-D-mannose,and its interaction with D-mannose-specific lectins Carbohydr Res  40: 111–7 1125946.  <ul class="find" data-citedArticleID="1022838" data-doi="10.1016/s0008-6215(00)82673-7"><li><a href="http://dx.doi.org/10.1016/s0008-6215(00)82673-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synthesis+of+2-O-%282-acetamido-2-deoxy-beta-D-glucopyranosyl%29-D-mannose%2C+and+its+interaction+with+D-mannose-specific+lectins" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synthesis+of+2-O-%282-acetamido-2-deoxy-beta-D-glucopyranosyl%29-D-mannose%2C+and+its+interaction+with+D-mannose-specific+lectins%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001232-Shinkawa1" id="pone.0001232-Shinkawa1"></a>Shinkawa T, Nakamura K, Yamane N, Shoji-Hosaka E, Kanda Y, et al.  (2003) The absence of fucose but not the presence of galactose or bisecting N-acetylglucosamine of human IgG1 complex-type oligosaccharides shows the critical role of enhancing antibody-dependent cellular cytotoxicity J Biol Chem  278: 3466–73 12427744.  <ul class="find" data-citedArticleID="1022882" data-doi="10.1074/jbc.m210665200"><li><a href="http://dx.doi.org/10.1074/jbc.m210665200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+absence+of+fucose+but+not+the+presence+of+galactose+or+bisecting+N-acetylglucosamine+of+human+IgG1+complex-type+oligosaccharides+shows+the+critical+role+of+enhancing+antibody-dependent+cellular+cytotoxicity" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+absence+of+fucose+but+not+the+presence+of+galactose+or+bisecting+N-acetylglucosamine+of+human+IgG1+complex-type+oligosaccharides+shows+the+critical+role+of+enhancing+antibody-dependent+cellular+cytotoxicity%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001232-Wang1" id="pone.0001232-Wang1"></a>Wang X, Inoue S, Gu J, Miyoshi E, Noda K, et al.  (2005) Dysregulation of TGF-beta1 receptor activation leads to abnormal lung development and emphysema-like phenotype in core fucose-deficient mice Proc Natl Acad Sci U S A  102: 15791–6 16236725.  <ul class="find" data-citedArticleID="1022894" data-doi="10.1073/pnas.0507375102"><li><a href="http://dx.doi.org/10.1073/pnas.0507375102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dysregulation+of+TGF-beta1+receptor+activation+leads+to+abnormal+lung+development+and+emphysema-like+phenotype+in+core+fucose-deficient+mice" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dysregulation+of+TGF-beta1+receptor+activation+leads+to+abnormal+lung+development+and+emphysema-like+phenotype+in+core+fucose-deficient+mice%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001232-Matsumoto1" id="pone.0001232-Matsumoto1"></a>Matsumoto I, Osawa T (1969) Purification and characterization of an anti-H(O) phytohemagglutinin of Ulex europeus Biochim Biophys Acta  194: 180–9 5353123.  <ul class="find" data-citedArticleID="1022856" data-doi="10.1016/0005-2795(69)90193-7"><li><a href="http://dx.doi.org/10.1016/0005-2795(69)90193-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification+and+characterization+of+an+anti-H%28O%29+phytohemagglutinin+of+Ulex+europeus" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification+and+characterization+of+an+anti-H%28O%29+phytohemagglutinin+of+Ulex+europeus%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001232-Okajima1" id="pone.0001232-Okajima1"></a>Okajima T, Fukumoto S, Miyazaki H, Ishida H, Kiso M, et al.  (1999) Molecular cloning of a novel alpha2,3-sialyltransferase (ST3Gal VI) that sialylates type II lactosamine structures on glycoproteins and glycolipids J Biol Chem  274: 11479–86 10206952.  <ul class="find" data-citedArticleID="1022870" data-doi="10.1074/jbc.274.17.11479"><li><a href="http://dx.doi.org/10.1074/jbc.274.17.11479" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+of+a+novel+alpha2%2C3-sialyltransferase+%28ST3Gal+VI%29+that+sialylates+type+II+lactosamine+structures+on+glycoproteins+and+glycolipids" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+of+a+novel+alpha2%2C3-sialyltransferase+%28ST3Gal+VI%29+that+sialylates+type+II+lactosamine+structures+on+glycoproteins+and+glycolipids%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001232-Ito1" id="pone.0001232-Ito1"></a>Ito N, Imai S, Haga S, Nagaike C, Morimura Y, et al.  (1996) Localization of binding sites of Ulex europaeus I, Helix pomatia and Griffonia simplicifolia I-B4 lectins and analysis of their backbone structures by several glycosidases and poly-N-acetyllactosamine-specific lectins in human breast carcinomas Histochem Cell Biol  106: 331–9 8897074.  <ul class="find" data-citedArticleID="1022836" data-doi="10.1007/s004180050049"><li><a href="http://dx.doi.org/10.1007/s004180050049" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Localization+of+binding+sites+of+Ulex+europaeus+I%2C+Helix+pomatia+and+Griffonia+simplicifolia+I-B4+lectins+and+analysis+of+their+backbone+structures+by+several+glycosidases+and+poly-N-acetyllactosamine-specific+lectins+in+human+breast+carcinomas" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Localization+of+binding+sites+of+Ulex+europaeus+I%2C+Helix+pomatia+and+Griffonia+simplicifolia+I-B4+lectins+and+analysis+of+their+backbone+structures+by+several+glycosidases+and+poly-N-acetyllactosamine-specific+lectins+in+human+breast+carcinomas%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001232-Furukawa1" id="pone.0001232-Furukawa1"></a>Furukawa K, Sato T (1999) Beta-1,4-galactosylation of N-glycans is a complex process Biochim Biophys Acta  1473: 54–66 10580129.  <ul class="find" data-citedArticleID="1022822" data-doi="10.1016/s0304-4165(99)00169-5"><li><a href="http://dx.doi.org/10.1016/s0304-4165(99)00169-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Beta-1%2C4-galactosylation+of+N-glycans+is+a+complex+process" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Beta-1%2C4-galactosylation+of+N-glycans+is+a+complex+process%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001232-Crowley1" id="pone.0001232-Crowley1"></a>Crowley JF, Goldstein IJ, Arnarp J, Lonngren J (1984) Carbohydrate binding studies on the lectin from Datura stramonium seeds Arch Biochem Biophys  231: 524–33 6203486.  <ul class="find" data-citedArticleID="1022810" data-doi="10.1016/0003-9861(84)90417-x"><li><a href="http://dx.doi.org/10.1016/0003-9861(84)90417-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carbohydrate+binding+studies+on+the+lectin+from+Datura+stramonium+seeds" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Carbohydrate+binding+studies+on+the+lectin+from+Datura+stramonium+seeds%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001232-Yamashita1" id="pone.0001232-Yamashita1"></a>Yamashita K, Totani K, Ohkura T, Takasaki S, Goldstein IJ, et al.  (1987) Carbohydrate binding properties of complex-type oligosaccharides on immobilized Datura stramonium lectin J Biol Chem  262: 1602–7 3805046.  <ul class="find" data-citedArticleID="1022900"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Carbohydrate+binding+properties+of+complex-type+oligosaccharides+on+immobilized+Datura+stramonium+lectin&amp;auth=&amp;atitle=Carbohydrate+binding+properties+of+complex-type+oligosaccharides+on+immobilized+Datura+stramonium+lectin" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carbohydrate+binding+properties+of+complex-type+oligosaccharides+on+immobilized+Datura+stramonium+lectin" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Carbohydrate+binding+properties+of+complex-type+oligosaccharides+on+immobilized+Datura+stramonium+lectin%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001232-Ihara1" id="pone.0001232-Ihara1"></a>Ihara Y, Nishikawa A, Taniguchi N (1995) Effects of dibutyryl cAMP and bromodeoxyuridine on expression of N-acetylglucosaminyltransferases III and V in GOTO neuroblastoma cells Glycoconj J  12: 787–94 8748156.  <ul class="find" data-citedArticleID="1022834" data-doi="10.1007/bf00731240"><li><a href="http://dx.doi.org/10.1007/bf00731240" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+dibutyryl+cAMP+and+bromodeoxyuridine+on+expression+of+N-acetylglucosaminyltransferases+III+and+V+in+GOTO+neuroblastoma+cells" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+dibutyryl+cAMP+and+bromodeoxyuridine+on+expression+of+N-acetylglucosaminyltransferases+III+and+V+in+GOTO+neuroblastoma+cells%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001232-Sasai1" id="pone.0001232-Sasai1"></a>Sasai K, Ikeda Y, Eguchi H, Tsuda T, Honke K, et al.  (2002) The action of N-acetylglucosaminyltransferase-V is prevented by the bisecting GlcNAc residue at the catalytic step FEBS Lett  522: 151–5 12095636.  <ul class="find" data-citedArticleID="1022878" data-doi="10.1016/s0014-5793(02)02916-2"><li><a href="http://dx.doi.org/10.1016/s0014-5793(02)02916-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+action+of+N-acetylglucosaminyltransferase-V+is+prevented+by+the+bisecting+GlcNAc+residue+at+the+catalytic+step" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+action+of+N-acetylglucosaminyltransferase-V+is+prevented+by+the+bisecting+GlcNAc+residue+at+the+catalytic+step%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001232-Miyoshi1" id="pone.0001232-Miyoshi1"></a>Miyoshi E, Nishikawa A, Ihara Y, Hayashi N, Fusamoto H, et al.  (1994) Selective suppression of N-acetylglucosaminyltransferase III activity in a human hepatoblastoma cell line transfected with hepatitis B virus Cancer Res  54: 1854–8 8137300.  <ul class="find" data-citedArticleID="1022858"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Selective+suppression+of+N-acetylglucosaminyltransferase+III+activity+in+a+human+hepatoblastoma+cell+line+transfected+with+hepatitis&amp;auth=&amp;atitle=Selective+suppression+of+N-acetylglucosaminyltransferase+III+activity+in+a+human+hepatoblastoma+cell+line+transfected+with+hepatitis" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+suppression+of+N-acetylglucosaminyltransferase+III+activity+in+a+human+hepatoblastoma+cell+line+transfected+with+hepatitis" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+suppression+of+N-acetylglucosaminyltransferase+III+activity+in+a+human+hepatoblastoma+cell+line+transfected+with+hepatitis%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001232-Sultan1" id="pone.0001232-Sultan1"></a>Sultan AS, Miyoshi E, Ihara Y, Nishikawa A, Tsukada Y, et al.  (1997) Bisecting GlcNAc structures act as negative sorting signals for cell surface glycoproteins in forskolin-treated rat hepatoma cells J Biol Chem  272: 2866–72 9006930.  <ul class="find" data-citedArticleID="1022888" data-doi="10.1074/jbc.272.5.2866"><li><a href="http://dx.doi.org/10.1074/jbc.272.5.2866" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bisecting+GlcNAc+structures+act+as+negative+sorting+signals+for+cell+surface+glycoproteins+in+forskolin-treated+rat+hepatoma+cells" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bisecting+GlcNAc+structures+act+as+negative+sorting+signals+for+cell+surface+glycoproteins+in+forskolin-treated+rat+hepatoma+cells%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001232-Wang2" id="pone.0001232-Wang2"></a>Wang WC, Cummings RD (1988) The immobilized leukoagglutinin from the seeds of Maackia amurensis binds with high affinity to complex-type Asn-linked oligosaccharides containing terminal sialic acid-linked alpha-2,3 to penultimate galactose residues J Biol Chem  263: 4576–85 3350806.  <ul class="find" data-citedArticleID="1022896"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+immobilized+leukoagglutinin+from+the+seeds+of+Maackia+amurensis+binds+with+high+affinity+to+complex-type+Asn-linked+oligosaccharides+containing+terminal+sialic+acid-linked+alpha-2%2C3+to+penultimate+galactose+residues&amp;auth=&amp;atitle=The+immobilized+leukoagglutinin+from+the+seeds+of+Maackia+amurensis+binds+with+high+affinity+to+complex-type+Asn-linked+oligosaccharides+containing+terminal+sialic+acid-linked+alpha-2%2C3+to+penultimate+galactose+residues" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+immobilized+leukoagglutinin+from+the+seeds+of+Maackia+amurensis+binds+with+high+affinity+to+complex-type+Asn-linked+oligosaccharides+containing+terminal+sialic+acid-linked+alpha-2%2C3+to+penultimate+galactose+residues" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+immobilized+leukoagglutinin+from+the+seeds+of+Maackia+amurensis+binds+with+high+affinity+to+complex-type+Asn-linked+oligosaccharides+containing+terminal+sialic+acid-linked+alpha-2%2C3+to+penultimate+galactose+residues%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001232-Konami1" id="pone.0001232-Konami1"></a>Konami Y, Yamamoto K, Osawa T, Irimura T (1994) Strong affinity of Maackia amurensis hemagglutinin (MAH) for sialic acid-containing Ser/Thr-linked carbohydrate chains of N-terminal octapeptides from human glycophorin A FEBS Lett  342: 334–8 8150094.  <ul class="find" data-citedArticleID="1022848" data-doi="10.1016/0014-5793(94)80527-x"><li><a href="http://dx.doi.org/10.1016/0014-5793(94)80527-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strong+affinity+of+Maackia+amurensis+hemagglutinin+%28MAH%29+for+sialic+acid-containing+Ser%2FThr-linked+carbohydrate+chains+of+N-terminal+octapeptides+from+human+glycophorin" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strong+affinity+of+Maackia+amurensis+hemagglutinin+%28MAH%29+for+sialic+acid-containing+Ser%2FThr-linked+carbohydrate+chains+of+N-terminal+octapeptides+from+human+glycophorin%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001232-Knibbs1" id="pone.0001232-Knibbs1"></a>Knibbs RN, Goldstein IJ, Ratcliffe RM, Shibuya N (1991) Characterization of the carbohydrate binding specificity of the leukoagglutinating lectin from Maackia amurensis. Comparison with other sialic acid-specific lectins J Biol Chem  266: 83–8 1985926.  <ul class="find" data-citedArticleID="1022846"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+the+carbohydrate+binding+specificity+of+the+leukoagglutinating+lectin+from+Maackia+amurensis.+Comparison+with+other+sialic+acid-specific+lectins&amp;auth=&amp;atitle=Characterization+of+the+carbohydrate+binding+specificity+of+the+leukoagglutinating+lectin+from+Maackia+amurensis.+Comparison+with+other+sialic+acid-specific+lectins" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+carbohydrate+binding+specificity+of+the+leukoagglutinating+lectin+from+Maackia+amurensis.+Comparison+with+other+sialic+acid-specific+lectins" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+carbohydrate+binding+specificity+of+the+leukoagglutinating+lectin+from+Maackia+amurensis.+Comparison+with+other+sialic+acid-specific+lectins%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001232-Stanley1" id="pone.0001232-Stanley1"></a>Stanley P, Caillibot V, Siminovitch L (1975) Selection and characterization of eight phenotypically distinct lines of lectin-resistant Chinese hamster ovary cell Cell  6: 121–8 1182798.  <ul class="find" data-citedArticleID="1022886" data-doi="10.1016/0092-8674(75)90002-1"><li><a href="http://dx.doi.org/10.1016/0092-8674(75)90002-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+and+characterization+of+eight+phenotypically+distinct+lines+of+lectin-resistant+Chinese+hamster+ovary+cell" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+and+characterization+of+eight+phenotypically+distinct+lines+of+lectin-resistant+Chinese+hamster+ovary+cell%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001232-Kitagawa1" id="pone.0001232-Kitagawa1"></a>Kitagawa H, Paulson JC (1994) Differential expression of five sialyltransferase genes in human tissues J Biol Chem  269: 17872–8 8027041.  <ul class="find" data-citedArticleID="1022844"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Differential+expression+of+five+sialyltransferase+genes+in+human+tissues&amp;auth=&amp;atitle=Differential+expression+of+five+sialyltransferase+genes+in+human+tissues" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+expression+of+five+sialyltransferase+genes+in+human+tissues" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+expression+of+five+sialyltransferase+genes+in+human+tissues%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001232-Bierhuizen1" id="pone.0001232-Bierhuizen1"></a>Bierhuizen MF, Mattei MG, Fukuda M (1993) Expression of the developmental I antigen by a cloned human cDNA encoding a member of a beta-1,6-N-acetylglucosaminyltransferase gene family Genes Dev  7: 468–78 8449405.  <ul class="find" data-citedArticleID="1022802" data-doi="10.1101/gad.7.3.468"><li><a href="http://dx.doi.org/10.1101/gad.7.3.468" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+the+developmental+I+antigen+by+a+cloned+human+cDNA+encoding+a+member+of+a+beta-1%2C6-N-acetylglucosaminyltransferase+gene+family" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+the+developmental+I+antigen+by+a+cloned+human+cDNA+encoding+a+member+of+a+beta-1%2C6-N-acetylglucosaminyltransferase+gene+family%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001232-Moremen1" id="pone.0001232-Moremen1"></a>Moremen KW (2002) Golgi alpha-mannosidase II deficiency in vertebrate systems: implications for asparagine-linked oligosaccharide processing in mammals Biochim Biophys Acta  1573: 225–35 12417404.  <ul class="find" data-citedArticleID="1022860" data-doi="10.1016/s0304-4165(02)00388-4"><li><a href="http://dx.doi.org/10.1016/s0304-4165(02)00388-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Golgi+alpha-mannosidase+II+deficiency+in+vertebrate+systems%3A+implications+for+asparagine-linked+oligosaccharide+processing+in+mammals" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Golgi+alpha-mannosidase+II+deficiency+in+vertebrate+systems%3A+implications+for+asparagine-linked+oligosaccharide+processing+in+mammals%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001232-Chen1" id="pone.0001232-Chen1"></a>Chen Y, Jain RK, Chandrasekaran EV, Matta KL (1995) Use of sialylated or sulfated derivatives and acrylamide copolymers of Gal beta 1,3GalNAc alpha- and GalNAc alpha- to determine the specificities of blood group T- and Tn-specific lectins and the copolymers to measure anti-T and anti-Tn antibody levels in cancer patients Glycoconj J  12: 55–62 7795413.  <ul class="find" data-citedArticleID="1022806" data-doi="10.1007/bf00731869"><li><a href="http://dx.doi.org/10.1007/bf00731869" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+sialylated+or+sulfated+derivatives+and+acrylamide+copolymers+of+Gal+beta+1%2C3GalNAc+alpha-+and+GalNAc+alpha-+to+determine+the+specificities+of+blood+group+T-+and+Tn-specific+lectins+and+the+copolymers+to+measure+anti-T+and+anti-Tn+antibody+levels+in+cancer+patients" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+sialylated+or+sulfated+derivatives+and+acrylamide+copolymers+of+Gal+beta+1%2C3GalNAc+alpha-+and+GalNAc+alpha-+to+determine+the+specificities+of+blood+group+T-+and+Tn-specific+lectins+and+the+copolymers+to+measure+anti-T+and+anti-Tn+antibody+levels+in+cancer+patients%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001232-NakamuraTsuruta1" id="pone.0001232-NakamuraTsuruta1"></a>Nakamura-Tsuruta S, Kominami J, Kuno A, Hirabayashi J (2006) Evidence that Agaricus bisporus agglutinin (ABA) has dual sugar-binding specificity Biochem Biophys Res Commun  347: 215–20 16824489.  <ul class="find" data-citedArticleID="1022866" data-doi="10.1016/j.bbrc.2006.06.073"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2006.06.073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+that+Agaricus+bisporus+agglutinin+%28ABA%29+has+dual+sugar-binding+specificity" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+that+Agaricus+bisporus+agglutinin+%28ABA%29+has+dual+sugar-binding+specificity%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001232-Hakomori1" id="pone.0001232-Hakomori1"></a>Hakomori S (2004) Carbohydrate-to-carbohydrate interaction in basic cell biology: a brief overview Arch Biochem Biophys  426: 173–81 15158668.  <ul class="find" data-citedArticleID="1022828" data-doi="10.1016/j.abb.2004.02.032"><li><a href="http://dx.doi.org/10.1016/j.abb.2004.02.032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carbohydrate-to-carbohydrate+interaction+in+basic+cell+biology%3A+a+brief+overview" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Carbohydrate-to-carbohydrate+interaction+in+basic+cell+biology%3A+a+brief+overview%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001232-Paulson1" id="pone.0001232-Paulson1"></a>Paulson JC, Blixt O, Collins BE (2006) Sweet spots in functional glycomics Nat Chem Biol  2: 238–48 16619023.  <ul class="find" data-citedArticleID="1022874" data-doi="10.1038/nchembio785"><li><a href="http://dx.doi.org/10.1038/nchembio785" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sweet+spots+in+functional+glycomics" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sweet+spots+in+functional+glycomics%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001232-Fukui1" id="pone.0001232-Fukui1"></a>Fukui S, Feizi T, Galustian C, Lawson AM, Chai W (2002) Oligosaccharide microarrays for high-throughput detection and specificity assignments of carbohydrate-protein interactions Nat Biotechnol  20: 1011–7 12219077.  <ul class="find" data-citedArticleID="1022820" data-doi="10.1038/nbt735"><li><a href="http://dx.doi.org/10.1038/nbt735" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oligosaccharide+microarrays+for+high-throughput+detection+and+specificity+assignments+of+carbohydrate-protein+interactions" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oligosaccharide+microarrays+for+high-throughput+detection+and+specificity+assignments+of+carbohydrate-protein+interactions%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001232-Blixt1" id="pone.0001232-Blixt1"></a>Blixt O, Head S, Mondala T, Scanlan C, Huflejt ME, et al.  (2004) Printed covalent glycan array for ligand profiling of diverse glycan binding proteins Proc Natl Acad Sci U S A  101: 17033–8 15563589.  <ul class="find" data-citedArticleID="1022804" data-doi="10.1073/pnas.0407902101"><li><a href="http://dx.doi.org/10.1073/pnas.0407902101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Printed+covalent+glycan+array+for+ligand+profiling+of+diverse+glycan+binding+proteins" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Printed+covalent+glycan+array+for+ligand+profiling+of+diverse+glycan+binding+proteins%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001232-Kamekawa1" id="pone.0001232-Kamekawa1"></a>Kamekawa N, Hayama K, Nakamura-Tsuruta S, Kuno A, Hirabayashi J (2006) A combined strategy for glycan profiling: a model study with pyridylaminated oligosaccharides J Biochem (Tokyo)  140: 337–47 16861248.  <ul class="find" data-citedArticleID="1022840" data-doi="10.1093/jb/mvj154"><li><a href="http://dx.doi.org/10.1093/jb/mvj154" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+combined+strategy+for+glycan+profiling%3A+a+model+study+with+pyridylaminated+oligosaccharides" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+combined+strategy+for+glycan+profiling%3A+a+model+study+with+pyridylaminated+oligosaccharides%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001232-Edwards1" id="pone.0001232-Edwards1"></a>Edwards D (2003) Non-linear normalization and background correction in one-channel cDNA microarray studies Bioinformatics  19: 825–33 12724292.  <ul class="find" data-citedArticleID="1022814" data-doi="10.1093/bioinformatics/btg083"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg083" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-linear+normalization+and+background+correction+in+one-channel+cDNA+microarray+studies" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-linear+normalization+and+background+correction+in+one-channel+cDNA+microarray+studies%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001232-Smyth1" id="pone.0001232-Smyth1"></a>Smyth GK (2005)  Limma: linear models for microarray data. In: Gentleman VC R, Dudoit S, Irizarry R, Huber W, editors. ‘Bioinformatics and Computational Biology Solutions using R and Bioconductor’. New York: Springer.   <ul class="find-nolinks"></ul></li><li><span class="label">49.
              </span><a name="pone.0001232-Team1" id="pone.0001232-Team1"></a>Team RDC (2004) R: A Language and Environment for Statistical Computing. Vienna: R Foundation for Statistical Computing.   <a href="http://www.R-project.org">http://www.R-project.org</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">50.
              </span><a name="pone.0001232-Morita1" id="pone.0001232-Morita1"></a>Morita S, Kojima T, Kitamura T (2000) Plat-E: An efficient and stable system for transient packaging of retroviruses Gene Ther  7: 1063–6 10871756.  <ul class="find" data-citedArticleID="1022862" data-doi="10.1038/sj.gt.3301206"><li><a href="http://dx.doi.org/10.1038/sj.gt.3301206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plat-E%3A+An+efficient+and+stable+system+for+transient+packaging+of+retroviruses" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plat-E%3A+An+efficient+and+stable+system+for+transient+packaging+of+retroviruses%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001232-Cummings1" id="pone.0001232-Cummings1"></a>Cummings R (1999)  Plant Lectins. In: Varki A, Cummings R, Esko JD, Freeze HH, Hart GW, Marth J, editors. Essentials of Glycobiology. New York: Cold Spring Harbor Laboratory Press. pp. 455–468.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.XML" value="114826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.PDF" value="716992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g001.PNG_L" value="345516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g001.PNG_M" value="70607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g001.PNG_S" value="16636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g001.TIF" value="498030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g002.PNG_L" value="330037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g002.PNG_M" value="85934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g002.PNG_S" value="18595"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g002.TIF" value="459492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g002.PNG_I" value="36252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g003.PNG_L" value="206156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g003.PNG_M" value="71487"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g003.PNG_S" value="16276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g003.TIF" value="286532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g004.PNG_L" value="454265"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g004.PNG_M" value="80809"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g004.PNG_S" value="17810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g004.TIF" value="577180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g004.PNG_I" value="49093"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g005.PNG_L" value="345758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g005.PNG_M" value="79728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g005.PNG_S" value="16964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g005.TIF" value="479426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g005.PNG_I" value="38887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g006.PNG_L" value="76442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g006.PNG_M" value="70134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g006.PNG_S" value="16077"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g006.TIF" value="119908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.g006.PNG_I" value="21996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.s001.XLS" value="313344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001232.s002.XLS" value="403456"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001232&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001232" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001232&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001232&volume=&issue=&title=Correlation Index-Based Responsible-Enzyme Gene Screening (CIRES), a Novel DNA Microarray-Based Method for Enzyme Gene Involved in Glycan Biosynthesis&author_name=Harumi%20Yamamoto%2C%20Hiromu%20Takematsu%2C%20Reiko%20Fujinawa%2C%20Yuko%20Naito%2C%20Yasushi%20Okuno%2C%20Gozoh%20Tsujimoto%2C%20Akemi%20Suzuki%2C%20Yasunori%20Kozutsumi&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001232" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001232" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001232" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001232&amp;t=Correlation%20Index-Based%20Responsible-Enzyme%20Gene%20Screening%20(CIRES)%2C%20a%20Novel%20DNA%20Microarray-Based%20Method%20for%20Enzyme%20Gene%20Involved%20in%20Glycan%20Biosynthesis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001232&title=Correlation%20Index-Based%20Responsible-Enzyme%20Gene%20Screening%20(CIRES)%2C%20a%20Novel%20DNA%20Microarray-Based%20Method%20for%20Enzyme%20Gene%20Involved%20in%20Glycan%20Biosynthesis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001232&amp;title=Correlation%20Index-Based%20Responsible-Enzyme%20Gene%20Screening%20(CIRES)%2C%20a%20Novel%20DNA%20Microarray-Based%20Method%20for%20Enzyme%20Gene%20Involved%20in%20Glycan%20Biosynthesis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001232" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001232&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Correlation Index-Based Responsible-Enzyme Gene Screening (CIRES), a Novel DNA Microarray-Based Method for Enzyme Gene Involved in Glycan Biosynthesis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001232';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Correlation%20Index-Based%20Responsible-Enzyme%20Gene%20Screening%20(CIRES)%2C%20a%20Novel%20DNA%20Microarray-Based%20Method%20for%20Enzyme%20Gene%20Involved%20in%20Glycan%20Biosynthesis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001232" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001232" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22B+cells%22" title="Search for articles in the subject area:'B cells'"><div class="flagText">B cells</div></a>
              <div data-categoryid="40633" data-articleid="25296"
                   data-categoryname="B cells"
                   class="flagImage" title="Flag 'B cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biosynthesis%22" title="Search for articles in the subject area:'Biosynthesis'"><div class="flagText">Biosynthesis</div></a>
              <div data-categoryid="40369" data-articleid="25296"
                   data-categoryname="Biosynthesis"
                   class="flagImage" title="Flag 'Biosynthesis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+staining%22" title="Search for articles in the subject area:'Cell staining'"><div class="flagText">Cell staining</div></a>
              <div data-categoryid="17401" data-articleid="25296"
                   data-categoryname="Cell staining"
                   class="flagImage" title="Flag 'Cell staining' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Flow+cytometry%22" title="Search for articles in the subject area:'Flow cytometry'"><div class="flagText">Flow cytometry</div></a>
              <div data-categoryid="20135" data-articleid="25296"
                   data-categoryname="Flow cytometry"
                   class="flagImage" title="Flag 'Flow cytometry' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25296"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Glycosyltransferases%22" title="Search for articles in the subject area:'Glycosyltransferases'"><div class="flagText">Glycosyltransferases</div></a>
              <div data-categoryid="25483" data-articleid="25296"
                   data-categoryname="Glycosyltransferases"
                   class="flagImage" title="Flag 'Glycosyltransferases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lectins%22" title="Search for articles in the subject area:'Lectins'"><div class="flagText">Lectins</div></a>
              <div data-categoryid="19117" data-articleid="25296"
                   data-categoryname="Lectins"
                   class="flagImage" title="Flag 'Lectins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25296"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9179'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2038'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6885&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=17325">Referee Comments: Referee 1 (Harry Schachter)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
