

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Detection of Novel Amplicons in Prostate Cancer by Comprehensive Genomic Profiling of Prostate Cancer Cell Lines Using Oligonucleotide-Based ArrayCGH</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000769"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000769" />

    <meta name="citation_title" content="Detection of Novel Amplicons in Prostate Cancer by Comprehensive Genomic Profiling of Prostate Cancer Cell Lines Using Oligonucleotide-Based ArrayCGH"/>
    <meta itemprop="name" content="Detection of Novel Amplicons in Prostate Cancer by Comprehensive Genomic Profiling of Prostate Cancer Cell Lines Using Oligonucleotide-Based ArrayCGH"/>

      <meta name="citation_author" content="Joern Kamradt"/>
            <meta name="citation_author_institution" content="Department of Urology and Pediatric Urology, University of Saarland, Homburg/Saar, Germany"/>
      <meta name="citation_author" content="Volker Jung"/>
            <meta name="citation_author_institution" content="Department of Urology and Pediatric Urology, University of Saarland, Homburg/Saar, Germany"/>
      <meta name="citation_author" content="Kerstin Wahrheit"/>
            <meta name="citation_author_institution" content="Department of Urology and Pediatric Urology, University of Saarland, Homburg/Saar, Germany"/>
      <meta name="citation_author" content="Laura Tolosi"/>
            <meta name="citation_author_institution" content="Department of Computational Biology and Applied Algorithmics, Max-Planck Institute for Informatics, Saarbrücken, Germany"/>
      <meta name="citation_author" content="Joerg Rahnenfuehrer"/>
            <meta name="citation_author_institution" content="Statistische Methoden in der Genetik und Chemometrie Fachbereich Statistik, Universität Dortmund, Dortmund, Germany"/>
      <meta name="citation_author" content="Martin Schilling"/>
            <meta name="citation_author_institution" content="Department of General, Visceral, Vascular and Pediatric Surgery, University of the Saarland, Homburg/Saar, Germany"/>
      <meta name="citation_author" content="Robert Walker"/>
            <meta name="citation_author_institution" content="Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Sean Davis"/>
            <meta name="citation_author_institution" content="Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Michael Stoeckle"/>
            <meta name="citation_author_institution" content="Department of Urology and Pediatric Urology, University of Saarland, Homburg/Saar, Germany"/>
      <meta name="citation_author" content="Paul Meltzer"/>
            <meta name="citation_author_institution" content="Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Bernd Wullich"/>
            <meta name="citation_author_institution" content="Department of Urology and Pediatric Urology, University of Saarland, Homburg/Saar, Germany"/>

    <meta name="citation_date" content="2007/8/22"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000769.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e769"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Chromosome aberrations in solid tumors.; citation_author=DG Albertson; citation_author=C Collins; citation_author=F McCormick; citation_author=JW Gray; citation_journal_title=Nat Genet; citation_volume=34; citation_number=1; citation_pages=369-376; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Comparative genomic hybridization for molecular cytogenetic analysis of solid tumors Science; citation_author=A Kallioniemi; citation_author=OP Kallioniemi; citation_author=D Sudar; citation_author=D Rutovitz; citation_author=JW Gray; citation_volume=258; citation_number=2; citation_pages=818-821; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Comparative genomic hybridization reveals DNA copy number gains to frequently occur in human prostate cancer.; citation_author=HP Sattler; citation_author=V Rohde; citation_author=H Bonkhoff; citation_author=T Zwergel; citation_author=B Wullich; citation_journal_title=Prostate; citation_volume=39; citation_number=3; citation_pages=79-86; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Comprehensive genotypic analysis of human prostate cancer cell lines and sublines derived from metastases after orthotopic implantation in nude mice.; citation_author=R Lensch; citation_author=C Goetz; citation_author=C Andres; citation_author=A Bex; citation_author=J Lehmann; citation_journal_title=Int J Oncol; citation_volume=21; citation_number=4; citation_pages=695-706; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genetic aberrations in prostate carcinoma detected by comparative genomic hybridization and microsatellite analysis: association with progression and angiogenesis.; citation_author=DM Strohmeyer; citation_author=AP Berger; citation_author=DH Moore; citation_author=G Bartsch; citation_author=H Klocker; citation_journal_title=Prostate; citation_volume=59; citation_number=5; citation_pages=43-58; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genomic and expression analysis of the 3q25-q26 amplification unit reveals TLOC1/SEC62 as a probable target gene in prostate cancer.; citation_author=V Jung; citation_author=R Kindich; citation_author=J Kamradt; citation_author=M Jung; citation_author=M Muller; citation_journal_title=Mol Cancer Res; citation_volume=4; citation_number=6; citation_pages=169-176; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Estimating cancer survival and clinical outcome based on genetic tumor progression scores.; citation_author=J Rahnenfuehrer; citation_author=N Beerenwinkel; citation_author=WA Schulz; citation_author=C Hartmann; citation_author=A von Deimling; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=7; citation_pages=2438-2446; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Matrix-based comparative genomic hybridization: biochips to screen for genomic imbalances.; citation_author=S Solinas-Toldo; citation_author=S Lampel; citation_author=S Stilgenbauer; citation_author=J Nickolenko; citation_author=A Benner; citation_journal_title=Genes Chromosomes Cancer; citation_volume=20; citation_number=8; citation_pages=399-407; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of DNA copy-number changes using cDNA microarrays.; citation_author=JR Pollack; citation_author=CM Perou; citation_author=AA Alizadeh; citation_author=MB Eisen; citation_author=A Pergamenschikov; citation_journal_title=Nat Genet; citation_volume=23; citation_number=9; citation_pages=41-46; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=High resolution analysis of DNA copy number variation using comparative genomic hybridization to microarrays.; citation_author=D Pinkel; citation_author=R Segraves; citation_author=D Sudar; citation_author=S Clark; citation_author=I Poole; citation_journal_title=Nat Genet; citation_volume=20; citation_number=10; citation_pages=207-111; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=High-resolution global profiling of genomic alterations with longoligonucleotide microarray.; citation_author=C Brennan; citation_author=Y Zhang; citation_author=C Leo; citation_author=B Feng; citation_author=C Cauwels; citation_journal_title=Cancer Res; citation_volume=64; citation_number=11; citation_pages=4744-4748; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=High-resolution analysis of gene copy number alterations in human prostate cancer using CGH on cDNA microarrays: impact of copy number on gene expression.; citation_author=M Wolf; citation_author=S Mousses; citation_author=S Hautaniemi; citation_author=R Karhu; citation_author=P Huusko; citation_journal_title=Neoplasia; citation_volume=6; citation_number=12; citation_pages=240-247; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic aberrations in prostate cancer by microarray analysis.; citation_author=OR Saramaki; citation_author=KP Porkka; citation_author=RL Vessella; citation_author=T Visakorpi; citation_journal_title=Int J Cancer; citation_volume=119; citation_number=13; citation_pages=1322-1329; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comparative genomic hybridization using oligonucleotide microarrays and total genomic DNA.; citation_author=MT Barrett; citation_author=A Scheffer; citation_author=A Ben-Dor; citation_author=N Sampas; citation_author=D Lipson; citation_journal_title=Proc NatlAcadSciUSA; citation_volume=101; citation_number=14; citation_pages=17765-17770; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome-wide screening for complete genetic loss in prostate cancer by comparative hybridization onto cDNA microarrays.; citation_author=J Clark; citation_author=S Edwards; citation_author=A Feber; citation_author=P Flohr; citation_author=M John; citation_journal_title=Oncogene; citation_volume=22; citation_number=15; citation_pages=1247-1252; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=High-resolution analysis of paraffin-embedded and formalin-fixed prostate tumors using comparative genomic hybridization to genomic microarrays.; citation_author=PL Paris; citation_author=DG Albertson; citation_author=JC Alers; citation_author=A Andaya; citation_author=P Carroll; citation_journal_title=Am J Pathol; citation_volume=162; citation_number=16; citation_pages=763-770; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evaluation of genetic patterns in different tumor areas of intermediate-grade prostatic adenocarcinomas by high-resolution genomic array analysis.; citation_author=H Dekken van; citation_author=PL Paris; citation_author=DG Albertson; citation_author=JC Alers; citation_author=A Andaya; citation_journal_title=Genes Chromosomes Cancer; citation_volume=39; citation_number=17; citation_pages=249-256; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Loss of a small region around the PTEN locus is a major chromosome 10 alteration in prostate cancer xenografts and cell lines.; citation_author=KG Hermans; citation_author=DC van Alewijk; citation_author=JA Veltman; citation_author=W van Weerden; citation_author=AG van Kessel; citation_journal_title=Genes Chromosomes Cancer; citation_volume=39; citation_number=18; citation_pages=171-184; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Accuracy of an array comparative genomic hybridization (CGH) technique in detecting DNA copy number aberrations: comparison with conventional CGH and loss of heterozygosity analysis in prostate cancer.; citation_author=S Yano; citation_author=H Matsuyama; citation_author=K Matsuda; citation_author=H Matsumoto; citation_author=S Yoshihiro; citation_journal_title=Cancer Genet Cytogenet; citation_volume=150; citation_number=19; citation_pages=122-127; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome-wide characterization of gene expression variations and DNA copy number changes in prostate cancer cell lines.; citation_author=H Zhao; citation_author=Y Kim; citation_author=P Wang; citation_author=J Lapointe; citation_author=R Tibshirani; citation_journal_title=Prostate; citation_volume=63; citation_number=20; citation_pages=187-197; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Orthotopic implantation of human prostate cancer cell lines: a clinically relevant animal model for metastatic prostate cancer.; citation_author=K Rembrink; citation_author=JC Romijn; citation_author=TH van der Kwast; citation_author=H Ruebben; citation_author=FH Schroeder; citation_journal_title=Prostate; citation_volume=31; citation_number=21; citation_pages=168-174; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Comparison of the malignant phenotype and genotype of the human androgen-independent cell line DU 145 and a subline derived from metastasis after orthotopic implantation in nude mice.; citation_author=A Bex; citation_author=B Wullich; citation_author=V Endris; citation_author=T Otto; citation_author=K Rembrink; citation_journal_title=Cancer Genet Cytogenet; citation_volume=124; citation_number=22; citation_pages=98-104; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Ratio-based decisions and the quantitative analysis of cDNA microarray images.; citation_author=Y Chen; citation_author=ER Dougherty; citation_author=ML Bittner; citation_journal_title=J. Biomed Optics; citation_volume=2; citation_number=23; citation_pages=364-374; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Bioconductor: open software development for computational biology and bioinformatics.; citation_author=RC Gentleman; citation_author=VJ Carey; citation_author=DM Bates; citation_author=B Bolstad; citation_author=M Dettling; citation_journal_title=Genome Biol; citation_volume=5; citation_number=24; citation_pages=R80; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Analysis of array CGH data: from signal ratio to gain and loss of DNA regions.; citation_author=P Hupé; citation_author=N Stransky; citation_author=JP Thiery; citation_author=F Radvanyi; citation_author=E Barillot; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=25; citation_pages=3413-3422; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evidence of focal genetic microheterogeneity in glioblastoma multiforme by area-specific CGH on microdissected tumor cells.; citation_author=V Jung; citation_author=BFM Romeike; citation_author=W Henn; citation_author=W Feiden; citation_author=JR Moringlane; citation_journal_title=J Neuropathol Exp Neurol; citation_volume=58; citation_number=26; citation_pages=993-999; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Application of a modified real time PCR technique for relative gene copy number quantification to the determination of the relationship between NKX3.1 loss and MYC gain in prostate cancer.; citation_author=R Kindich; citation_author=AR Florl; citation_author=V Jung; citation_author=R Engers; citation_author=M Mueller; citation_journal_title=Clin Chem; citation_volume=51; citation_number=27; citation_pages=649-652; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Unbiased whole-genome amplification directly from clinical samples.; citation_author=S Hosono; citation_author=AF Faruqi; citation_author=FB Dean; citation_author=Y Du; citation_author=Z Sun; citation_journal_title=Genome Res; citation_volume=13; citation_number=28; citation_pages=954-964; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Increased expression of the interleukin-11 receptor and evidence of STAT3 activation in prostate carcinoma.; citation_author=CL Campbell; citation_author=Z Jiang; citation_author=DM Savarese; citation_author=TM Savarese; citation_journal_title=Am J Pathol; citation_volume=158; citation_number=29; citation_pages=25-32; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Combinatorial screenings in patients: the interleukin-11 receptor alpha as a candidate target in the progression of human prostate cancer.; citation_author=AJ Zurita; citation_author=P Troncoso; citation_author=M Cardo-Vila; citation_author=CJ Logothetis; citation_author=R Pasqualini; citation_journal_title=Cancer Res; citation_volume=64; citation_number=30; citation_pages=435-439; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Interleukin-6 family of cytokines and gp130.; citation_author=T Kishimoto; citation_author=S Akira; citation_author=M Narazaki; citation_author=T Taga; citation_journal_title=Blood; citation_volume=86; citation_number=31; citation_pages=1243-1254; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Down-regulation of signal transducer and activator of transcription 3 expression using vector-based small interfering RNAs suppresses growth of human prostate tumor in vivo.; citation_author=L Gao; citation_author=L Zhang; citation_author=J Hu; citation_author=F Li; citation_author=Y Shao; citation_journal_title=Clin Cancer Res; citation_volume=11; citation_number=32; citation_pages=6333-6341; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The androgen receptor and signal-transduction pathways in hormone-refractory prostate cancer. Part 1: Modifications to the androgen receptor Androgen-receptor cofactors and bypass pathways.; citation_author=J Edwards; citation_author=JM Bartlett; citation_journal_title=BJU Int; citation_volume=95; citation_number=33; citation_pages=1320-1326; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000769" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Detection of Novel Amplicons in Prostate Cancer by Comprehensive Genomic Profiling of Prostate Cancer Cell Lines Using Oligonucleotide-Based ArrayCGH"/>
    <meta name="twitter:description" content="BackgroundThe purpose of this study was to prove the feasibility of a longmer oligonucleotide microarray platform to profile gene copy number alterations in prostate cancer cell lines and to quickly indicate novel candidate genes, which may play a role in carcinogenesis.Methods/Results and FindingsGenome-wide screening for regions of genetic gains and losses on nine prostate cancer cell lines (PC3, DU145, LNCaP, CWR22, and derived sublines) was carried out using comparative genomic hybridization on a 35,000 feature oligonucleotide microarray (arrayCGH). Compared to conventional chromosomal CGH, more deletions and small regions of gains, particularly in pericentromeric regions and regions next to the telomeres, were detected. As validation of the high-resolution of arrayCGH we further analyzed a small amplicon of 1.7 MB at 9p13.3, which was found in CWR22 and CWR22-Rv1. Increased copy number was confirmed by fluorescence in situ hybridization using the BAC clone RP11-165H19 from the amplified region comprising the two genes interleukin 11 receptor alpha (IL11-RA) and dynactin 3 (DCTN3). Using quantitative real time PCR (qPCR) we could demonstrate that IL11-RA is the gene with the highest copy number gain in the cell lines compared to DCTN3 suggesting IL11-RA to be the amplification target. Screening of 20 primary prostate carcinomas by qPCR revealed an IL11-RA copy number gain in 75% of the tumors analyzed. Gain of DCTN3 was only found in two cases together with a gain of IL11-RA.Conclusions/SignificanceArrayCGH using longmer oligonucleotide microarrays is feasible for high-resolution analysis of chomosomal imbalances. Characterization of a small gained region at 9p13.3 in prostate cancer cell lines and primary prostate cancer samples by fluorescence in situ hybridization and quantitative PCR has revealed interleukin 11 receptor alpha gene as a candidate target of amplification with an amplification frequency of 75% in prostate carcinomas. Frequent amplification of IL11-RA in prostate cancer is a potential mechanism of IL11-RA overexpression in this tumor type."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000769.g004"/>

  <meta property="og:title" content="Detection of Novel Amplicons in Prostate Cancer by Comprehensive Genomic Profiling of Prostate Cancer Cell Lines Using Oligonucleotide-Based ArrayCGH" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000769" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7976'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8994'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6942&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000769">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000769" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000769&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Detection of Novel Amplicons in Prostate Cancer by Comprehensive Genomic Profiling of Prostate Cancer Cell Lines Using Oligonucleotide-Based ArrayCGH
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joern Kamradt
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Joern Kamradt, Volker Jung</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:joern.kamradt@uks.eu">joern.kamradt@uks.eu</a></p>

                <p>Affiliation:
                  Department of Urology and Pediatric Urology, University of Saarland, Homburg/Saar, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Volker Jung
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Joern Kamradt, Volker Jung</p>

              
              

                <p>Affiliation:
                  Department of Urology and Pediatric Urology, University of Saarland, Homburg/Saar, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kerstin Wahrheit, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Urology and Pediatric Urology, University of Saarland, Homburg/Saar, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laura Tolosi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Computational Biology and Applied Algorithmics, Max-Planck Institute for Informatics, Saarbrücken, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joerg Rahnenfuehrer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Statistische Methoden in der Genetik und Chemometrie Fachbereich Statistik, Universität Dortmund, Dortmund, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Martin Schilling, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of General, Visceral, Vascular and Pediatric Surgery, University of the Saarland, Homburg/Saar, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robert Walker, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sean Davis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael Stoeckle, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Urology and Pediatric Urology, University of Saarland, Homburg/Saar, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paul Meltzer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bernd Wullich
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Urology and Pediatric Urology, University of Saarland, Homburg/Saar, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 22, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000769</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000769" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000769" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000769" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000769" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000769-g001" data-doi="info:doi/10.1371/journal.pone.0000769" data-uri="info:doi/10.1371/journal.pone.0000769.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000769-t001" data-doi="info:doi/10.1371/journal.pone.0000769" data-uri="info:doi/10.1371/journal.pone.0000769.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000769-g002" data-doi="info:doi/10.1371/journal.pone.0000769" data-uri="info:doi/10.1371/journal.pone.0000769.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000769-g003" data-doi="info:doi/10.1371/journal.pone.0000769" data-uri="info:doi/10.1371/journal.pone.0000769.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000769-g004" data-doi="info:doi/10.1371/journal.pone.0000769" data-uri="info:doi/10.1371/journal.pone.0000769.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000769">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000769" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The purpose of this study was to prove the feasibility of a longmer oligonucleotide microarray platform to profile gene copy number alterations in prostate cancer cell lines and to quickly indicate novel candidate genes, which may play a role in carcinogenesis.</p>


<h3>Methods/Results and Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Genome-wide screening for regions of genetic gains and losses on nine prostate cancer cell lines (PC3, DU145, LNCaP, CWR22, and derived sublines) was carried out using comparative genomic hybridization on a 35,000 feature oligonucleotide microarray (arrayCGH). Compared to conventional chromosomal CGH, more deletions and small regions of gains, particularly in pericentromeric regions and regions next to the telomeres, were detected. As validation of the high-resolution of arrayCGH we further analyzed a small amplicon of 1.7 MB at 9p13.3, which was found in CWR22 and CWR22-Rv1. Increased copy number was confirmed by fluorescence <em>in situ</em> hybridization using the BAC clone RP11-165H19 from the amplified region comprising the two genes interleukin 11 receptor alpha (<em>IL11-RA</em>) and dynactin 3 (<em>DCTN3</em>). Using quantitative real time PCR (qPCR) we could demonstrate that <em>IL11-RA</em> is the gene with the highest copy number gain in the cell lines compared to <em>DCTN3</em> suggesting <em>IL11-RA</em> to be the amplification target. Screening of 20 primary prostate carcinomas by qPCR revealed an <em>IL11-RA</em> copy number gain in 75% of the tumors analyzed. Gain of <em>DCTN3</em> was only found in two cases together with a gain of <em>IL11-RA</em>.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>ArrayCGH using longmer oligonucleotide microarrays is feasible for high-resolution analysis of chomosomal imbalances. Characterization of a small gained region at 9p13.3 in prostate cancer cell lines and primary prostate cancer samples by fluorescence <em>in situ</em> hybridization and quantitative PCR has revealed interleukin 11 receptor alpha gene as a candidate target of amplification with an amplification frequency of 75% in prostate carcinomas. Frequent amplification of <em>IL11-RA</em> in prostate cancer is a potential mechanism of <em>IL11-RA</em> overexpression in this tumor type.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kamradt J, Jung V, Wahrheit K, Tolosi L, Rahnenfuehrer J, et al.  (2007) Detection of Novel Amplicons in Prostate Cancer by Comprehensive Genomic Profiling of Prostate Cancer Cell Lines Using Oligonucleotide-Based ArrayCGH. PLoS ONE 2(8):
          e769.
            doi:10.1371/journal.pone.0000769</p><p><strong>Academic Editor: </strong>Stefan Maas, Lehigh University, United States of America</p><p><strong>Received:</strong> July 5, 2007; <strong>Accepted:</strong> July 18, 2007; <strong>Published:</strong> August 22, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>Deutsche Forschungsgemeinschaft, grant number KA1793/1-1, Bundesministerium für Bildung und Forschung, grant number 01GR0453, BioSapiens Network of Excellence, EU contract number LSHG-CT-2003-503265</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Genetic alterations are believed to be key events in the development of most tumors, including prostate cancer <a href="#pone.0000769-Albertson1">[1]</a>. Tumor progression seems to depend on the successive acquisition of chromosomal aberrations leading to gains or losses of part of the tumor cell genome. Characterization of these genomic abnormalities in prostate cancer may therefore help to understand the molecular pathogenesis and may unveil genetic markers of progression.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Since its first description by Kallioniemi et al. (1992) <a href="#pone.0000769-Kallioniemi1">[2]</a> chromosomal comparative genomic hybridization (cCGH) has become the most frequently used technique to detect DNA copy number changes in tumor genomes. We and others have analyzed the genome of prostate cancer cell lines and primary prostate cancer samples with this technique <a href="#pone.0000769-Sattler1">[3]</a>–<a href="#pone.0000769-Strohmeyer1">[5]</a>. Fluorescence <em>in situ</em> DNA hybridization (FISH) and quantitative real time PCR have been demonstrated to be valuable tools for target gene discovery within identified chromosomal regions of gain, e.g. the <em>TLOC1/SEC62</em> gene at 3q26.2 in prostate cancer <a href="#pone.0000769-Jung1">[6]</a>. Applying advanced bioinformatic models on cCGH data demonstrated that the patterns of chromosomal aberrations contain valuable prognostic information of a tumor <a href="#pone.0000769-Rahnenfuehrer1">[7]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Because of the relative low spatial resolution (~20MB) of cCGH and its inaccuracy in centromeric as well as telomeric regions this technique is neither able to adequately detect small regions of gains or loses nor genomic alterations next to the centromere or telomere. Also, for target gene identification in gained regions as found by cCGH, fine-mapping with techniques like FISH is laborious and time-consuming.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Compared to cCGH, microarray-based CGH, referred to as array CGH (aCGH) or matrix CGH <a href="#pone.0000769-SolinasToldo1">[8]</a>–<a href="#pone.0000769-Pollack1">[9]</a>, has a roughly 1.000-fold higher resolution (or even higher) and allows analysis of chromosomal regions close to the centromere and telomere. Different approaches of aCGH have been followed over the years. Several groups utilized genomic BAC arrays <a href="#pone.0000769-Pinkel1">[10]</a> whereas others have chosen cDNA or oligonucleotide arrays that were originally designed for expression analysis <a href="#pone.0000769-Pollack1">[9]</a>, <a href="#pone.0000769-Brennan1">[11]</a>. Arrays designed for gene expression are advantageous for direct comparison of genomic alterations and gene expression on the same platform. Several studies have demonstrated that this approach shows a significant association between gene copy number and expression level <a href="#pone.0000769-Wolf1">[12]</a>, <a href="#pone.0000769-Saramaki1">[13]</a>. Lately, use of oligonucleotide arrays specifically for aCGH designed longer was reported <a href="#pone.0000769-Barrett1">[14]</a> and is now commercially available as an aCGH platform. For the aCGH analyses of prostate cancer cell lines as well as clinical specimens either BAC or cDNA arrays have been utilized <a href="#pone.0000769-Wolf1">[12]</a>, <a href="#pone.0000769-Saramaki1">[13]</a>, <a href="#pone.0000769-Clark1">[15]</a>–<a href="#pone.0000769-Zhao1">[20]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Here we present the first study utilizing a 35,000 feature 70-mer oligonucleotide array, originally designed for expression analysis, for detailed genomic characterization of nine prostate cancer cell lines. Resulting aCGH profiles are compared to cCGH results. The occurrence of a newly detected small amplicon in the pericentromeric 9p13.3 subband in various cell lines is validated by FISH and quantitative real time PCR and is also confirmed in primary prostate cancer samples.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Material and Methods"></a><h3>Material and Methods</h3>
<h4>Tumor cell lines and DNA isolation</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The human prostate cancer cell lines DU145, PC3, LNCaP, CWR22 and CWR22-Rv1 were obtained from American Type Cell Culture Collection (ATCC, Rockville, MD, USA) and cultured according to the protocols recommended by the ATCC. From PC3 and DU145, two different branches were available, one held in the laboratory of the Cancer Genetic Branch, NHGRI, NIH (PC3<em><sub>NIH</sub></em>, DU145<em><sub>NIH</sub></em>), the other in the urological laboratory in Homburg (PC3<em><sub>HOM</sub></em>, DU145<em><sub>HOM</sub></em>). PC3-N, PC-125-1L, and DU145-MN1 were established as previously described <a href="#pone.0000769-Rembrink1">[21]</a>, <a href="#pone.0000769-Bex1">[22]</a>. LNCaP-CN4-2 was kindly provided by Z. Culig, Department of Urology, Medical University, Innsbruck, Austria.</p>


<h4>Primary prostate cancer samples</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Quantitative gene copy number measurements were done on 20 primary prostate adenocarcinoma samples, which were obtained after radical prostatectomy from previously untreated prostate cancer patients. Following prostatectomy, the specimens were dissected by a pathologist, snap frozen, and stored at −80°C. Only samples containing &gt;50% tumor cells were included in the study.</p>


<h4>DNA isolation and quantification</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>DNA was isolated using the QiAmp DNA isolation kit (Qiagen, Hilden, Germany) and subsequently quantified by a fluorometric assay (Quant-iT Pico Green dsDNA Kit, Invitrogen, Karlsruhe, Germany). Fluorescence was measured using a TECAN (Salzburg, Austria) SpectrafluorPLUS microplate fluorescence reader with an excitation wavelength of 485 nm and an emission wavelength of 535 nm. DNA concentrations were calculated from a standard curve of double-stranded control DNA provided with the kit that was measured in triplicate at the concentrations 30, 3, 0.3, and 0.03 ng/µl (measured by fluorometry).</p>


<h4>Whole genome amplification and purification</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>A volume of 2.5 µl out of 4 ng/µl dilutions from cell lines and control DNA was used as starting material for the amplification. The phi29-amplification was carried out according to the Repli-G kit manufacturer's instructions (Qiagen) using an incubation time of 16 h. Repli-G reactions were checked by a real time based intra <em>Alu</em>-PCR assay. Additionally DNA concentration was fluorimetrically quantified with Quant-iT Pico Green dsDNA Kit (Invitrogen) and ranged between 10–30 μg.</p>


<h4>Alu-PCR assay</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Due to a frequently observed background synthesis in the Repli-G amplified no-template control, we performed an extra quality control on 1 µl (1:50 dilution) of unpurified phi29-amplified DNA. An <em>Alu</em> specific band should be present in all Repli-G amplified samples and absent in the Repli-G amplified no-template control. Each 2 µl of sample was compared to 1 µl serial dilutions of male control DNA (Promega, WI, USA) (10, 5, 2, 1, 0.5, 0.1, 0.01, 0.001 ng/µl). The 25 µl reactions contained 1× Hot start SYBR green master mix (Qiagen), 1.5 mM MgCl<sub>2</sub>, 0.2 mM dNTPs Mix, 400 nM primers each (<em>Alu</em>-forw: 5′-GTGGGCTGAAAAGCTCCCGATTAT-3′ and <em>Alu</em>-rev: 5′-ATTCAAAGGGTATCTGGGC TCTGG-3′). The cycling conditions were as follows: 1 min at 94°C; 35 cycles of 20 s at 94°C, 20 s at 55°C and 20 s at 72°C; 10 min at 72°C. The amplification products were checked by melting curve analysis using the LightCycler™ quantification software 3.5 (Roche Diagnostics, Mannheim, Germany) and gel electrophoresis. Amplification was considered successful when the amount of human <em>Alu</em> sequences in phi29 generated DNA fragments ranged from 10–30% and when a smear of DNA fragments, ranging from 1 to 20 kb, was visible by gel electrophoresis.</p>


<h4>ArrayCGH</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>10 µg of amplified and purified DNA were labeled using the BioPrime Array CGH Genomic Labeling kit (Invitrogen) according to the manufacturer's instructions in a volume of 50 µl with a modified dNTP mix containing 120 µM each of dATP, dGTP, and dCTP; 60 µM dTTP; and 60 µM Cy5-dUTP or Cy3-dUTP. Labeled DNA was purified using QIAquick PCR purification columns (Qiagen). Tumor and reference DNA were pooled, mixed with 50 µg Cot-1 DNA (Invitrogen) and concentrated to a volume of 20 µl in a vacuum centrifuge. DNA was then mixed with an equal amount of 2× formamide buffer, denaturated at 95°C for 5 min and preincubated at 37°C for 30 min in a waterbath.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Hybridizations were carried out on custom (printed at the NHGRI/NIH Microarray Core Facility) glass slides containing the Operon 70mer oligonucleotide set version 3 with 34.580 oligonucleotides. The oligonucleotides were originally designed for expression analysis. For arrayCGH all oligonucleotides were sequence aligned against the human genome (build 34), EnsEMBL, RefSeq, dbEST and UCSC known genes resulting in 29.383 oligonucleotides with a specific chromosomal mapping.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>DNA samples were hybridized onto the array for 16–18 hours at 42°C utilizing the MAUI 4-bay hybridization system and MAUI Mixer A0 (BioMicro System, Salt Lake City, UT, USA). After hybridization arrays and MAUI Mixer were disassembled in 42°C 1×SSC and 0.05% SDS (wash solution 1) and subsequently washed two times in wash solution 1 for 5 minutes followed by two washes in 0.1×SSC (wash solution 2) for 5 minutes. Dried array slides were scanned using a ScanLite Express microarray scanner (Perkin Elmer, Wellesley, MA, USA). Raw image files of the arrays were processed using DeArray software <a href="#pone.0000769-Chen1">[23]</a> and resulting image data were imported into R environment using bioconductor packages <a href="#pone.0000769-Gentleman1">[24]</a> for further analysis.</p>


<h4>Data processing and statistical analysis of microarray data</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>The statistical analysis for identifying chromosomal regions with altered copy numbers in single arrays consisted of two steps. First, after mapping the measured gene copy number log-ratios to their respective chromosome locations, smoothing was applied. The adaptive weights smoothing-based algorithm GLAD <a href="#pone.0000769-Hup1">[25]</a> for identifying regions of constant copy number was used. This algorithm fits a piecewise constant function along the chromosome to the log-ratios. Then, for every sample regions that are amplified or deleted were identified. Here, normal distributions were fitted in a robust way to the smoothed log-ratios of each array. In particular, the median and the interquartile range were calculated, and a normal distribution was fitted to these parameters. Finally, constant regions with values above or below specified cutoffs were detected as gained or lost regions, respectively. In order to discriminate between weak and strong signals, the median plus or minus one or two standard deviations were selected as cutoffs for weak and strong aberrations, respectively. The algorithms were implemented in the R programming language (<a href="http://www.r-project.org">www.r-project.org</a>). The results were obtained using R version 2.3 and the GLAD library provided by the Bioconductor project (<a href="http://www.bioconductor.org">www.bioconductor.org</a>), version 1.7.</p>


<h4>Chromosomal CGH</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>Hybridization was done as described previously with minor modifications <a href="#pone.0000769-Jung2">[26]</a>. 500 ng biotin labeled probe DNA, 500 ng digoxigenin labeled reference DNA (human male reference DNA, Promega, WI, USA) and 50 µg human unlabeled cot-DNA (Invitrogen, Karlsruhe, Germany) were precipitated with 0.3 M sodium acetate in the 2.5 fold volume of ethanol and dissolved in 2.5 µl deionized formamide. After 30 minutes, the 2.5 µl of the double hybridization mix (100 mM sodium phosphate buffer, 4×SSC and 20% dextrane sulfate, pH 7.0) were added and denatured for 5 min at 75°C followed by 15 min preannealing at 37°C. The hybridization was performed under sealed coverslips for 3 days at 37°C in a moist chamber. After hybridization, the slides were washed three times for 5 min in a washing solution (50% formamide in 2×SSC pH 7.0) at 45°C, twice in 2×SSC (pH 7.0) at 45°C, and once in 0.1×SSC (pH 7.0) at 45°C.</p>
<a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>Biotinylated probe DNA was detected with FITC labeled streptavidin (Vector Laboratories, Burlingame, CA, USA). The digoxigenized control DNA was detected with rhodamine labeled anti-digoxigenin antibodies (Roche Diagnostics, Mannheim, Germany). Finally the slides were counterstained, and an antifade solution applied in the same step (Vectashield with DAPI, Vector Laboratories).</p>
<a id="article1.body1.sec2.sec8.p3" name="article1.body1.sec2.sec8.p3"></a><p>Slides were analyzed using a digital image analysis system (MetaSystems, Altlussheim, Germany), based on an Olympus AX 70 microscope equipped with a cooled CCD camera (Photometrics, Tuscon, AZ, USA). For analyzing differential fluorescence data we used the MetaSystems ISIS 3 software. The three-color images with red, green, and blue were acquired from 15–20 metaphases. Chromosome imbalances were detected on the basis of the fluorescence ratio profile, deviating from the balanced value (FITC:rhodamine = 1). For each chromosome the final ratio values were prepared from mean values of at least ten chromosome homologues from separate metaphase spreads. CGH results were plotted as a series of green to red ratio profiles, and the interpretation of results followed previously described protocols <a href="#pone.0000769-Sattler1">[3]</a>.</p>


<h4>Fluorescence in situ hybridization</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>FISH was performed on nuclei and metaphase spreads of the cell lines CWR22 and CWR22-Rv1. Metaphase spreads of lymphocytes from a healthy donor were used as a control. The BAC clone RP11-165H19 (9p13.3, GenBank accession number AQ382511) was obtained from BACPAC Resource Center Children's Hospital (Oakland Research Institute, Oakland, CA, USA) and cohybridized with a probe specific for the centromere of chromosome 9 (D9Z1; Oncor, Gaithersburg, MD, USA). Bacterial cultures and DNA isolation were done according to the BACPAC Miniprep protocol (<a href="http://www.biologia.uniba.it/rmc">http://www.biologia.uniba.it/rmc</a>). <em>Alu</em>-PCR products of the BAC were used as probes and were biotinylated using nick translation. Dual color fluorescence <em>in situ</em> hybridization and detection of fluorescence signals were done as described previously <a href="#pone.0000769-Jung1">[6]</a>.</p>


<h4>Quantitative real time PCR</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>Quantitative real time PCR for gene copy number measurement was based on a recently described approach <a href="#pone.0000769-Kindich1">[27]</a>. In the present study, we used predesigned and validated SNP assays (Applied Biosystems, CA, USA) with the AB 7900 system (Applied Biosystems). For the validation of the 9p13.3 amplicon two genes within the amplicon (IL-11RA, AssayID: C__11340987_10 and DCTN3, AssayID: C__25472566_10) and a gene at at 3p24.2 located in a not altered region in prostate cancer (TOP2B, AssayID: C__8063527_10) were analyzed.</p>
<a id="article1.body1.sec2.sec10.p2" name="article1.body1.sec2.sec10.p2"></a><p>In a first step, all SNP assays used in this study were run on a dilution series of normal blood DNA to determine PCR efficiency (E) of each primer set by the formula E = 10<sup>−1/s</sup>, where s represents the absolute value of the slope in a plot of the threshold cycle (C<sub>T</sub>) against log of input amount. All primers were shown to have an equal efficiency of approximately E = 2 and in relative efficiency plots comparing the different primers (log of input amount <em>vs.</em> ΔC<sub>T</sub>) the absolute value of the slope was less than 0.1. Relative gene copy number was then calculated following the equitation: 2<sup>−ΔΔCT</sup> (User bulletin#2, Applied Biosystems). For calibration normal blood DNA was added in each PCR run. For genes having both alleles detected by the assay the C<sub>T</sub> value was subtracted by 1 based on the shown efficiency of 2. To determine whether results obtained by real time PCR analysis of the prostate cancer samples were significantly different from those obtained for samples from healthy individuals, we determined a tolerance interval (TI) for the relative gene copy numbers, using the standard deviation (SD) of C<sub>T</sub> values for target and reference genes in 8 healthy individuals according to the equation: TI = 2±(SDΔC<sub>T</sub>×2). The TI ranged from 1.62 to 3.17 for <em>DCTN3</em> and from 1.77 to 2.94 for <em>IL11-RA</em>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Minimal recurrent regions of alterations</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Because of their highly altered genomes and the large volume of data generated by array analyses, simple visual inspection of altered loci proved inefficient to identify minimal recurrent regions of aberration. To analyze the data set, we combined automatic aberration identification with a modified frequency plot procedure. Application of weighted frequency analysis to the autosomes of prostate cancer cell lines revealed multiple regions of highly recurrent changes. In comparison to our cCGH analyses several small amplifications and deletion units were only detected by aCGH, especially in pericentromeric regions and regions near the telomeres (<a href="#pone-0000769-g001">Fig. 1</a>). The size of the smallest regions of aberration could be determined to 110 kb for losses at 19q13.41 in cell lines DU145<em><sub>NIH</sub></em>, DU145-MN1, PC3 125-1L and CWR22-Rv1, and 760 kb for gains at 14q32.11-q32.12 in PC3<em><sub>HOM</sub></em>, PC3-N and PC3-125-1L. Most chromosomal aberrations as detected by cCGH were also seen in aCGH. However, more losses were detected with microarrays and more large regions of gains with metaphase technique. For the commonly detected regions of aberrations, no discrepancy between the two methods in assigning gains or losses to the individual region was observed (<a href="#pone-0000769-t001">table 1</a>).</p>
<div class="figure" id="pone-0000769-g001"><div class="img"><a name="pone-0000769-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000769" data-uri="info:doi/10.1371/journal.pone.0000769.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000769.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000769.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Comparison of cCGH (A, C) and arrayCGH (B, D) results from chromosome 9 of CWR22 and chromosome 14 of DU145-MN1.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>A small region of gain on 9p close to the centromer (B, arrowhead) and a small deletion on 14q (D, arrowhead) are only detected by arrayCGH.</p>
<span>doi:10.1371/journal.pone.0000769.g001</span></div><div class="figure" id="pone-0000769-t001"><div class="img"><a name="pone-0000769-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000769" data-uri="info:doi/10.1371/journal.pone.0000769.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000769.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000769.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Overview of detected chromosomal aberrations by cCGH and aCGH</span></strong></p><span>doi:10.1371/journal.pone.0000769.t001</span></div>

<h4>Cytogenetic constitution of different branches of prostate cancer cell lines and parental cell line vs. subline comparison</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The aCGH results of the parental cell lines PC3<em><sub>HOM</sub></em>, DU145<em><sub>HOM</sub></em>, LNCaP and CWR22 and their sublines are summarized in <a href="#pone-0000769-g002">figure 2</a>. For PC3 and DU145, two different branches could be compared (<em>HOM vs. NIH</em>). Most amazingly, the cytogenetic constitution of PC3<em><sub>HOM</sub></em> differed markedly from PC3<em><sub>NIH</sub></em>. From a total of 54 aberrations in both cell lines, only six strong aberrations could be found in common: losses at 1q, 8p, 10p and 13 and gains at 10q and 17q. The gain of 8q11.2-8q24.3 in PC3<em><sub>HOM</sub></em> was also observed in PC3<em><sub>NIH</sub></em> showing a minimal size variation of 100 kb. For DU145<em><sub>HOM</sub></em> and DU145<em><sub>NIH</sub></em>, a good overall concordance of the genetic composition was found.</p>
<div class="figure" id="pone-0000769-g002"><div class="img"><a name="pone-0000769-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000769" data-uri="info:doi/10.1371/journal.pone.0000769.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000769.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000769.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Whole genome plot of gained (A) and lost (B) chromosomal regions of 11 prostate cancer cell lines, as detected with aCGH.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>Numbers above the plot indicate chromosome numbers and vertical lines boundaries between chromosomes.</p>
<span>doi:10.1371/journal.pone.0000769.g002</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Comparing cell lines PC3<em><sub>HOM</sub></em>, DU145<em><sub>HOM</sub></em>, LNCaP and CWR22 with their derived sublines PC3-N, PC3-125-1L, DU145-N, LNCaP-CN4-2 and CWR22-Rv1, respectively, a high congruency of the cytogenetic aberrations between the corresponding cell lines was yielded by aCGH. It is worth mentioning that besides minor variations of the number of extra regions with gains or losses of genetic material differences between parental cell lines and sublines predominantly concerned the extent of corresponding regions with copy number alterations, as for example was observed for the regions with gains at 12q and 15q in PC3<em><sub>HOM </sub>vs.</em> PC3-N and for the regions with losses at 2q, 4q, 6q and 13q21.33 in LNCaP <em>vs</em>. LNCaP-CN4-2.</p>


<h4>Validation of the amplified pericentromeric region 9p13.3 using FISH analysis and quantitative real time PCR</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>A novel 1.7 Mb region with copy number gains was consistently recognized in the cell lines CWR22 and CWR22-Rv1 and was mapped to the pericentromeric 9p13.3 subband. The presence of this amplicon in the cell lines CWR22 and CWR22-Rv1 was confirmed by metaphase FISH using a BAC clone (RP11-165H19) from the amplified region and a probe specific for the centromere of chromosome 9 (<a href="#pone-0000769-g003">Fig. 3A and B</a>). Optimal signal and lack of cross hybridization was verified using normal metaphase spreads (<a href="#pone-0000769-g003">Fig. 3C</a>). This BAC clone contained two genes, <em>IL-11RA</em> and <em>DCTN3</em>, which were already thought to play a role in prostate cancer growth <a href="#pone.0000769-Saramaki1">[13]</a>.</p>
<div class="figure" id="pone-0000769-g003"><div class="img"><a name="pone-0000769-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000769" data-uri="info:doi/10.1371/journal.pone.0000769.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000769.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000769.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Validation of amplicons by FISH analysis on cell lines.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>Identification of increased copies of signals of the BAC clone RP11-165H19 mapped to 9p13.3 amplicon in cell line CWR22 and CWR22-Rv1 (A and B). C Optimal signal and lack of cross hybridization was verified using normal metaphase spreads showing two signals for each probe. BAC clone RP11-165H19 signals are green; red signals indicate chromosome 9 centromere (D9Z1).</p>
<span>doi:10.1371/journal.pone.0000769.g003</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Based on gene localization and annotated gene function, we selected these two candidate genes for gene copy number measurements in the prostate cancer cell lines and 20 primary prostate carcinoma samples using quantitative real time PCR. <em>IL-11RA</em> showed a significant increase in gene copy number above normal in CWR22 and CWR22-Rv1 and in 15 out of 20 (75%) primary prostate cancer tumor samples (<a href="#pone-0000769-g004">Fig. 4</a>). For <em>DCTN3</em>, no copy number gain was detected in any cell line and in only 2 out of the 20 (10%) prostate cancer tumor samples. The control gene <em>TOP2B</em> at 3p24.2 was proved unchanged compared to normal blood (data not shown).</p>
<div class="figure" id="pone-0000769-g004"><div class="img"><a name="pone-0000769-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000769" data-uri="info:doi/10.1371/journal.pone.0000769.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000769.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000769.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000769.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000769.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000769.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>DNA copy number quantification of <em>DCTN3</em> and <em>IL11-RA</em> genes in prostate cancer cell lines and 20 primary prostate carcinoma samples using quantitative real time PCR.</span></strong></p><a id="article1.body1.sec3.sec3.fig2.caption1.p1" name="article1.body1.sec3.sec3.fig2.caption1.p1"></a><p><em>IL-11RA</em> showed a significant increase in gene copy number above normal in CWR22, CWR22-Rv1, DU145<em>HOM</em> and DU145-MN1 and in 15 out of the 20 (75%) prostate cancer samples. For <em>DCTN3</em>, no copy number gain was detected in any cell line and in only 2 out of the 20 (10%) prostate cancer samples. Values above cut off line being assigned as increased gene copy number compared to normal.</p>
<span>doi:10.1371/journal.pone.0000769.g004</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Although prostate cancer cell lines have been previously analyzed by aCGH utilizing cDNA microarray platforms <a href="#pone.0000769-Wolf1">[12]</a>, <a href="#pone.0000769-Saramaki1">[13]</a>, <a href="#pone.0000769-Clark1">[15]</a>–<a href="#pone.0000769-Zhao1">[20]</a>, we demonstrate in this study the feasibility of a longmer oligonucleotide set, originally designed for expression analysis, for a high-resolution genomic profiling of nine prostate cancer cell lines. The microarray data for this study have been submitted to NCBI GEO with accession GSE7376. Compared to cCGH, we detected more deletions and small regions of gains with aCGH, especially in pericentromeric regions and regions near the telomeres, where cCGH is known not to deliver reliable informations on DNA imbalances. On the other hand, large regions of gains as revealed by cCGH encompassing almost the whole arm of chromosomes were overlooked by aCGH. Similar observations were also reported by Saramaki et al. (2006) <a href="#pone.0000769-Saramaki1">[13]</a> using cDNA-based aCGH. Besides normalization artifacts one might argue that these differences could result from the DNA amplification step for aCGH in our study, whereas non-amplified DNA was used for cCGH. Although whole-genome amplification by phi29 polymerase may add some bias and increase background noise, it is regarded the most unbiased approach compared to PCR-based DNA amplification procedures <a href="#pone.0000769-Hosono1">[28]</a>. As any DNA amplification step seems inevitable in the study of primary prostate cancer samples, we used an amplification step in our study even though DNA quantity was not a limiting factor when working with cell lines.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Our aCGH findings are in good agreement with the data reported in the literature for PC3, DU145, LNCaP, and CWR22 <a href="#pone.0000769-Wolf1">[12]</a>, <a href="#pone.0000769-Saramaki1">[13]</a>, <a href="#pone.0000769-Clark1">[15]</a>–<a href="#pone.0000769-Zhao1">[20]</a>. However, an interesting side aspect of our study is the observation of a marked cytogenetic difference between the two PC3 branches, which were held in two different labs (PC3<em><sub>HOM</sub> vs.</em> PC3<em><sub>NIH</sub></em>). The presumed genomic instability of cell lines may lead to genetic divergency, what has to be considered when comparing findings from different labs on seemingly the same cell lines. A quality management system, which includes details of the genetic composition of the individually used cell lines, seems prudent.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Concerning the comparison of parental cell lines and sublines, no gross differences in the cytogenetic composition were found by aCGH. Differences predominantly concerned the boundaries of corresponding regions with copy number alterations. These findings are in contrast to our previous cCGH studies demonstrating several extra regions of DNA copy number alterations in the sublines compared to the parental cell lines <a href="#pone.0000769-Lensch1">[4]</a>. An explanation might be technical, as the previously reported differences mainly involved large chromosome regions, which are detected with a lower sensitivity by aCGH compared to cCGH, as was discussed above.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>As a validation of the high resolution of aCGH, we further analyzed a small amplification unit of 1.7 MB in the pericentromeric region of chromosome 9 at 9p13.3, which was found in the xenograft-derived cell lines CWR22 and CWR22-Rv1 by aCGH but not by cCGH. Interestingly, Saramaki et al. (2006) <a href="#pone.0000769-Saramaki1">[13]</a> also found this region by aCGH to be amplified in the prostate cancer xenograft LuCaP35. They confirmed their results by BAC-FISH and demonstrated amplification and overexpression of the ubiquitin-conjugating enzyme gene E2R2 (<em>UBE2R2</em>), the dynactin 3 gene (<em>DCTN3</em>) and the WD repeat domain 40A gene (<em>WDR40A</em>) at 9p13.3. Using a real time PCR technique, we further quantified the copy number of the interleukin 11 receptor alpha gene (<em>IL-11RA</em>) and the dynactin 3 gene (<em>DCTN3</em>) both revealing the highest log2 ratio in aCGH within 9p13.3. <em>IL-11RA</em> was found with the highest copy number gain in CWR22, CWR22-Rv1, DU145<em><sub>HOM</sub></em> and DU145MN1 suggesting <em>IL-11RA</em> as the putative target gene of this amplicon. We, therefore, screened 20 primary prostate cancer samples and found 75% of the tumors harboring an <em>IL-11RA</em> copy number gain alone, none of <em>DCTN3</em> alone, and 10% of both genes. Mean copy number gain for <em>IL-11RA</em> was approximately 4-fold. These data emphasize our initial assumption that <em>IL-11RA</em> represents the amplification target rather than <em>DCTN3</em>. This hypothesis is further strengthened by immunohistochemical studies <a href="#pone.0000769-Campbell1">[29]</a>, <a href="#pone.0000769-Zurita1">[30]</a> and the cancer profiling database Oncomine™ (<a href="http://www.oncomine.org">www.oncomine.org</a>) which both reveal high overexpression of <em>IL-11RA</em> in prostate cancer compared to normal prostate tissue. <em>IL-11RA</em> encodes a specific receptor for IL-11 and belongs to the family of gp130-dependent cytokine receptors, which include receptors for IL-6, leukemia inhibitory factor, ciliary neurotrophic factor, oncostatin M, and cardiotrophin <a href="#pone.0000769-Kishimoto1">[31]</a>. An important signaling system activated by <em>IL-11RA</em> and other members of this receptor family is the Janus kinase-signal transducer and activator of transcription (Jak-STAT) pathway with STAT3 having a well studied importance in prostate carcinogenesis <a href="#pone.0000769-Gao1">[32]</a>. Moreover, activated STAT3 is believed to play a key role in androgen receptor activation in the absence of androgens, one explanation of hormone refractory growth of prostate cancer <a href="#pone.0000769-Edwards1">[33]</a>. Whether <em>IL-11RA</em> is causative in prostate cancer growth needs to be investigated in further studies.</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PM JK VJ RW. Performed the experiments: JK VJ KW. Analyzed the data: SD JK VJ LT JR. Contributed reagents/materials/analysis tools: PM SD MS MS BW. Wrote the paper: JK VJ BW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000769-Albertson1" id="pone.0000769-Albertson1"></a>Albertson DG, Collins C, McCormick F, Gray JW (2003) Chromosome aberrations in solid tumors. Nat Genet  34: 369–376.  <ul class="find" data-citedArticleID="980326" data-doi="10.1038/ng1215"><li><a href="http://dx.doi.org/10.1038/ng1215" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome+aberrations+in+solid+tumors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome+aberrations+in+solid+tumors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000769-Kallioniemi1" id="pone.0000769-Kallioniemi1"></a>Kallioniemi A, Kallioniemi OP, Sudar D, Rutovitz D, Gray JW, et al.  (1992) Comparative genomic hybridization for molecular cytogenetic analysis of solid tumors Science 258: 818–821.  <ul class="find" data-citedArticleID="980358" data-doi="10.1126/science.1359641"><li><a href="http://dx.doi.org/10.1126/science.1359641" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomic+hybridization+for+molecular+cytogenetic+analysis+of+solid+tumors+Science" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomic+hybridization+for+molecular+cytogenetic+analysis+of+solid+tumors+Science%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000769-Sattler1" id="pone.0000769-Sattler1"></a>Sattler HP, Rohde V, Bonkhoff H, Zwergel T, Wullich B (1999) Comparative genomic hybridization reveals DNA copy number gains to frequently occur in human prostate cancer. Prostate  39: 79–86.  <ul class="find" data-citedArticleID="980378" data-doi="10.1002/(sici)1097-0045(19990501)39:2<79::aid-pros1&gt;3.3.co;2-u"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0045(19990501)39:2<79::aid-pros1&gt;3.3.co;2-u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomic+hybridization+reveals+DNA+copy+number+gains+to+frequently+occur+in+human+prostate+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomic+hybridization+reveals+DNA+copy+number+gains+to+frequently+occur+in+human+prostate+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000769-Lensch1" id="pone.0000769-Lensch1"></a>Lensch R, Goetz C, Andres C, Bex A, Lehmann J, et al.  (2002) Comprehensive genotypic analysis of human prostate cancer cell lines and sublines derived from metastases after orthotopic implantation in nude mice. Int J Oncol  21: 695–706.  <ul class="find" data-citedArticleID="980364" data-doi="10.3892/ijo.21.4.695"><li><a href="http://dx.doi.org/10.3892/ijo.21.4.695" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+genotypic+analysis+of+human+prostate+cancer+cell+lines+and+sublines+derived+from+metastases+after+orthotopic+implantation+in+nude+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+genotypic+analysis+of+human+prostate+cancer+cell+lines+and+sublines+derived+from+metastases+after+orthotopic+implantation+in+nude+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000769-Strohmeyer1" id="pone.0000769-Strohmeyer1"></a>Strohmeyer DM, Berger AP, Moore DH 2nd, Bartsch G, Klocker H, et al.  (2004) Genetic aberrations in prostate carcinoma detected by comparative genomic hybridization and microsatellite analysis: association with progression and angiogenesis. Prostate  59: 43–58.  <ul class="find" data-citedArticleID="980382" data-doi="10.1002/pros.20028"><li><a href="http://dx.doi.org/10.1002/pros.20028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+aberrations+in+prostate+carcinoma+detected+by+comparative+genomic+hybridization+and+microsatellite+analysis%3A+association+with+progression+and+angiogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+aberrations+in+prostate+carcinoma+detected+by+comparative+genomic+hybridization+and+microsatellite+analysis%3A+association+with+progression+and+angiogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000769-Jung1" id="pone.0000769-Jung1"></a>Jung V, Kindich R, Kamradt J, Jung M, Muller M, et al.  (2006) Genomic and expression analysis of the 3q25-q26 amplification unit reveals <em>TLOC1/SEC62</em> as a probable target gene in prostate cancer. Mol Cancer Res  4: 169–176.  <ul class="find" data-citedArticleID="980354" data-doi="10.1158/1541-7786.mcr-05-0165"><li><a href="http://dx.doi.org/10.1158/1541-7786.mcr-05-0165" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+and+expression+analysis+of+the+3q25-q26+amplification+unit+reveals+TLOC1%2FSEC62+as+a+probable+target+gene+in+prostate+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+and+expression+analysis+of+the+3q25-q26+amplification+unit+reveals+TLOC1%2FSEC62+as+a+probable+target+gene+in+prostate+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000769-Rahnenfuehrer1" id="pone.0000769-Rahnenfuehrer1"></a>Rahnenfuehrer J, Beerenwinkel N, Schulz WA, Hartmann C, von Deimling A, et al.  (2005) Estimating cancer survival and clinical outcome based on genetic tumor progression scores. Bioinformatics  21: 2438–2446.  <ul class="find" data-citedArticleID="980372" data-doi="10.1093/bioinformatics/bti312"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti312" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+cancer+survival+and+clinical+outcome+based+on+genetic+tumor+progression+scores." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+cancer+survival+and+clinical+outcome+based+on+genetic+tumor+progression+scores.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000769-SolinasToldo1" id="pone.0000769-SolinasToldo1"></a>Solinas-Toldo S, Lampel S, Stilgenbauer S, Nickolenko J, Benner A, et al.  (1997) Matrix-based comparative genomic hybridization: biochips to screen for genomic imbalances. Genes Chromosomes Cancer  20: 399–407.  <ul class="find" data-citedArticleID="980380" data-doi="10.1002/(sici)1098-2264(199712)20:4<399::aid-gcc12&gt;3.0.co;2-i"><li><a href="http://dx.doi.org/10.1002/(sici)1098-2264(199712)20:4<399::aid-gcc12&gt;3.0.co;2-i" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Matrix-based+comparative+genomic+hybridization%3A+biochips+to+screen+for+genomic+imbalances." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Matrix-based+comparative+genomic+hybridization%3A+biochips+to+screen+for+genomic+imbalances.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000769-Pollack1" id="pone.0000769-Pollack1"></a>Pollack JR, Perou CM, Alizadeh AA, Eisen MB, Pergamenschikov A, et al.  (1999) Genome-wide analysis of DNA copy-number changes using cDNA microarrays. Nat Genet  23: 41–46.  <ul class="find" data-citedArticleID="980370" data-doi="10.1038/14385"><li><a href="http://dx.doi.org/10.1038/14385" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+DNA+copy-number+changes+using+cDNA+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+DNA+copy-number+changes+using+cDNA+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000769-Pinkel1" id="pone.0000769-Pinkel1"></a>Pinkel D, Segraves R, Sudar D, Clark S, Poole I, et al.  (1998) High resolution analysis of DNA copy number variation using comparative genomic hybridization to microarrays. Nat Genet  20: 207–111.  <ul class="find" data-citedArticleID="980368"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=High+resolution+analysis+of+DNA+copy+number+variation+using+comparative+genomic+hybridization+to+microarrays.&amp;auth=&amp;atitle=High+resolution+analysis+of+DNA+copy+number+variation+using+comparative+genomic+hybridization+to+microarrays." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+resolution+analysis+of+DNA+copy+number+variation+using+comparative+genomic+hybridization+to+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+resolution+analysis+of+DNA+copy+number+variation+using+comparative+genomic+hybridization+to+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000769-Brennan1" id="pone.0000769-Brennan1"></a>Brennan C, Zhang Y, Leo C, Feng B, Cauwels C, et al.  (2004) High-resolution global profiling of genomic alterations with longoligonucleotide microarray. Cancer Res  64: 4744–4748.  <ul class="find" data-citedArticleID="980332" data-doi="10.1158/0008-5472.can-04-1241"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-04-1241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-resolution+global+profiling+of+genomic+alterations+with+longoligonucleotide+microarray." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-resolution+global+profiling+of+genomic+alterations+with+longoligonucleotide+microarray.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000769-Wolf1" id="pone.0000769-Wolf1"></a>Wolf M, Mousses S, Hautaniemi S, Karhu R, Huusko P, et al.  (2004) High-resolution analysis of gene copy number alterations in human prostate cancer using CGH on cDNA microarrays: impact of copy number on gene expression. Neoplasia  6: 240–247.  <ul class="find" data-citedArticleID="980384" data-doi="10.1593/neo.03439"><li><a href="http://dx.doi.org/10.1593/neo.03439" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-resolution+analysis+of+gene+copy+number+alterations+in+human+prostate+cancer+using+CGH+on+cDNA+microarrays%3A+impact+of+copy+number+on+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-resolution+analysis+of+gene+copy+number+alterations+in+human+prostate+cancer+using+CGH+on+cDNA+microarrays%3A+impact+of+copy+number+on+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000769-Saramaki1" id="pone.0000769-Saramaki1"></a>Saramaki OR, Porkka KP, Vessella RL, Visakorpi T (2006) Genetic aberrations in prostate cancer by microarray analysis. Int J Cancer  119: 1322–1329.  <ul class="find" data-citedArticleID="980376" data-doi="10.1002/ijc.21976"><li><a href="http://dx.doi.org/10.1002/ijc.21976" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+aberrations+in+prostate+cancer+by+microarray+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+aberrations+in+prostate+cancer+by+microarray+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000769-Barrett1" id="pone.0000769-Barrett1"></a>Barrett MT, Scheffer A, Ben-Dor A, Sampas N, Lipson D, et al.  (2004) Comparative genomic hybridization using oligonucleotide microarrays and total genomic DNA. Proc NatlAcadSciUSA  101: 17765–17770.  <ul class="find" data-citedArticleID="980328" data-doi="10.1073/pnas.0407979101"><li><a href="http://dx.doi.org/10.1073/pnas.0407979101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomic+hybridization+using+oligonucleotide+microarrays+and+total+genomic+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomic+hybridization+using+oligonucleotide+microarrays+and+total+genomic+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000769-Clark1" id="pone.0000769-Clark1"></a>Clark J, Edwards S, Feber A, Flohr P, John M, et al.  (2003) Genome-wide screening for complete genetic loss in prostate cancer by comparative hybridization onto cDNA microarrays. Oncogene  22: 1247–1252.  <ul class="find" data-citedArticleID="980338" data-doi="10.1038/sj.onc.1206247"><li><a href="http://dx.doi.org/10.1038/sj.onc.1206247" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+screening+for+complete+genetic+loss+in+prostate+cancer+by+comparative+hybridization+onto+cDNA+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+screening+for+complete+genetic+loss+in+prostate+cancer+by+comparative+hybridization+onto+cDNA+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000769-Paris1" id="pone.0000769-Paris1"></a>Paris PL, Albertson DG, Alers JC, Andaya A, Carroll P, et al.  (2003) High-resolution analysis of paraffin-embedded and formalin-fixed prostate tumors using comparative genomic hybridization to genomic microarrays. Am J Pathol  162: 763–770.  <ul class="find" data-citedArticleID="980366" data-doi="10.1016/s0002-9440(10)63873-4"><li><a href="http://dx.doi.org/10.1016/s0002-9440(10)63873-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-resolution+analysis+of+paraffin-embedded+and+formalin-fixed+prostate+tumors+using+comparative+genomic+hybridization+to+genomic+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-resolution+analysis+of+paraffin-embedded+and+formalin-fixed+prostate+tumors+using+comparative+genomic+hybridization+to+genomic+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000769-Dekkenvan1" id="pone.0000769-Dekkenvan1"></a>Dekken van H, Paris PL, Albertson DG, Alers JC, Andaya A, et al.  (2004) Evaluation of genetic patterns in different tumor areas of intermediate-grade prostatic adenocarcinomas by high-resolution genomic array analysis. Genes Chromosomes Cancer  39: 249–256.  <ul class="find" data-citedArticleID="980340" data-doi="10.1002/gcc.20001"><li><a href="http://dx.doi.org/10.1002/gcc.20001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+genetic+patterns+in+different+tumor+areas+of+intermediate-grade+prostatic+adenocarcinomas+by+high-resolution+genomic+array+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+genetic+patterns+in+different+tumor+areas+of+intermediate-grade+prostatic+adenocarcinomas+by+high-resolution+genomic+array+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000769-Hermans1" id="pone.0000769-Hermans1"></a>Hermans KG, van Alewijk DC, Veltman JA, van Weerden W, van Kessel AG, et al.  (2004) Loss of a small region around the PTEN locus is a major chromosome 10 alteration in prostate cancer xenografts and cell lines. Genes Chromosomes Cancer  39: 171–184.  <ul class="find" data-citedArticleID="980348" data-doi="10.1002/gcc.10311"><li><a href="http://dx.doi.org/10.1002/gcc.10311" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Loss+of+a+small+region+around+the+PTEN+locus+is+a+major+chromosome+10+alteration+in+prostate+cancer+xenografts+and+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Loss+of+a+small+region+around+the+PTEN+locus+is+a+major+chromosome+10+alteration+in+prostate+cancer+xenografts+and+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000769-Yano1" id="pone.0000769-Yano1"></a>Yano S, Matsuyama H, Matsuda K, Matsumoto H, Yoshihiro S, et al.  (2004) Accuracy of an array comparative genomic hybridization (CGH) technique in detecting DNA copy number aberrations: comparison with conventional CGH and loss of heterozygosity analysis in prostate cancer. Cancer Genet Cytogenet  150: 122–127.  <ul class="find" data-citedArticleID="980386" data-doi="10.1016/j.cancergencyto.2003.09.004"><li><a href="http://dx.doi.org/10.1016/j.cancergencyto.2003.09.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accuracy+of+an+array+comparative+genomic+hybridization+%28CGH%29+technique+in+detecting+DNA+copy+number+aberrations%3A+comparison+with+conventional+CGH+and+loss+of+heterozygosity+analysis+in+prostate+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accuracy+of+an+array+comparative+genomic+hybridization+%28CGH%29+technique+in+detecting+DNA+copy+number+aberrations%3A+comparison+with+conventional+CGH+and+loss+of+heterozygosity+analysis+in+prostate+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000769-Zhao1" id="pone.0000769-Zhao1"></a>Zhao H, Kim Y, Wang P, Lapointe J, Tibshirani R, et al.  (2005) Genome-wide characterization of gene expression variations and DNA copy number changes in prostate cancer cell lines. Prostate  63: 187–197.  <ul class="find" data-citedArticleID="980388" data-doi="10.1002/pros.20158"><li><a href="http://dx.doi.org/10.1002/pros.20158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+characterization+of+gene+expression+variations+and+DNA+copy+number+changes+in+prostate+cancer+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+characterization+of+gene+expression+variations+and+DNA+copy+number+changes+in+prostate+cancer+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000769-Rembrink1" id="pone.0000769-Rembrink1"></a>Rembrink K, Romijn JC, van der Kwast TH, Ruebben H, Schroeder FH (1997) Orthotopic implantation of human prostate cancer cell lines: a clinically relevant animal model for metastatic prostate cancer. Prostate  31: 168–174.  <ul class="find" data-citedArticleID="980374" data-doi="10.1002/(sici)1097-0045(19970515)31:3<168::aid-pros4&gt;3.3.co;2-m"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0045(19970515)31:3<168::aid-pros4&gt;3.3.co;2-m" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Orthotopic+implantation+of+human+prostate+cancer+cell+lines%3A+a+clinically+relevant+animal+model+for+metastatic+prostate+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Orthotopic+implantation+of+human+prostate+cancer+cell+lines%3A+a+clinically+relevant+animal+model+for+metastatic+prostate+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000769-Bex1" id="pone.0000769-Bex1"></a>Bex A, Wullich B, Endris V, Otto T, Rembrink K, et al.  (2001) Comparison of the malignant phenotype and genotype of the human androgen-independent cell line DU 145 and a subline derived from metastasis after orthotopic implantation in nude mice. Cancer Genet Cytogenet  124: 98–104.  <ul class="find" data-citedArticleID="980330" data-doi="10.1016/s0165-4608(00)00332-0"><li><a href="http://dx.doi.org/10.1016/s0165-4608(00)00332-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+the+malignant+phenotype+and+genotype+of+the+human+androgen-independent+cell+line+DU+145+and+a+subline+derived+from+metastasis+after+orthotopic+implantation+in+nude+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+the+malignant+phenotype+and+genotype+of+the+human+androgen-independent+cell+line+DU+145+and+a+subline+derived+from+metastasis+after+orthotopic+implantation+in+nude+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000769-Chen1" id="pone.0000769-Chen1"></a>Chen Y, Dougherty ER, Bittner ML (1997) Ratio-based decisions and the quantitative analysis of cDNA microarray images. J. Biomed Optics  2: 364–374.  <ul class="find" data-citedArticleID="980336" data-doi="10.1117/12.281504"><li><a href="http://dx.doi.org/10.1117/12.281504" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ratio-based+decisions+and+the+quantitative+analysis+of+cDNA+microarray+images." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ratio-based+decisions+and+the+quantitative+analysis+of+cDNA+microarray+images.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000769-Gentleman1" id="pone.0000769-Gentleman1"></a>Gentleman RC, Carey VJ, Bates DM, Bolstad B, Dettling M, et al.  (2004) Bioconductor: open software development for computational biology and bioinformatics. Genome Biol  5: R80.  <ul class="find" data-citedArticleID="980346" data-doi="10.1186/gb-2004-5-10-r80"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-10-r80" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bioconductor%3A+open+software+development+for+computational+biology+and+bioinformatics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bioconductor%3A+open+software+development+for+computational+biology+and+bioinformatics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000769-Hup1" id="pone.0000769-Hup1"></a>Hupé P, Stransky N, Thiery JP, Radvanyi F, Barillot E (2004) Analysis of array CGH data: from signal ratio to gain and loss of DNA regions. Bioinformatics  20: 3413–3422.  <ul class="find" data-citedArticleID="980352" data-doi="10.1093/bioinformatics/bth418"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth418" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+array+CGH+data%3A+from+signal+ratio+to+gain+and+loss+of+DNA+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+array+CGH+data%3A+from+signal+ratio+to+gain+and+loss+of+DNA+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000769-Jung2" id="pone.0000769-Jung2"></a>Jung V, Romeike BFM, Henn W, Feiden W, Moringlane JR, et al.  (1999) Evidence of focal genetic microheterogeneity in glioblastoma multiforme by area-specific CGH on microdissected tumor cells. J Neuropathol Exp Neurol  58: 993–999.  <ul class="find" data-citedArticleID="980356" data-doi="10.1097/00005072-199909000-00009"><li><a href="http://dx.doi.org/10.1097/00005072-199909000-00009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+of+focal+genetic+microheterogeneity+in+glioblastoma+multiforme+by+area-specific+CGH+on+microdissected+tumor+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+of+focal+genetic+microheterogeneity+in+glioblastoma+multiforme+by+area-specific+CGH+on+microdissected+tumor+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000769-Kindich1" id="pone.0000769-Kindich1"></a>Kindich R, Florl AR, Jung V, Engers R, Mueller M, et al.  (2005) Application of a modified real time PCR technique for relative gene copy number quantification to the determination of the relationship between NKX3.1 loss and MYC gain in prostate cancer. Clin Chem  51: 649–652.  <ul class="find" data-citedArticleID="980360" data-doi="10.1373/clinchem.2004.045013"><li><a href="http://dx.doi.org/10.1373/clinchem.2004.045013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+a+modified+real+time+PCR+technique+for+relative+gene+copy+number+quantification+to+the+determination+of+the+relationship+between+NKX3.1+loss+and+MYC+gain+in+prostate+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+a+modified+real+time+PCR+technique+for+relative+gene+copy+number+quantification+to+the+determination+of+the+relationship+between+NKX3.1+loss+and+MYC+gain+in+prostate+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000769-Hosono1" id="pone.0000769-Hosono1"></a>Hosono S, Faruqi AF, Dean FB, Du Y, Sun Z, et al.  (2003) Unbiased whole-genome amplification directly from clinical samples. Genome Res  13: 954–964.  <ul class="find" data-citedArticleID="980350" data-doi="10.1101/gr.816903"><li><a href="http://dx.doi.org/10.1101/gr.816903" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unbiased+whole-genome+amplification+directly+from+clinical+samples." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unbiased+whole-genome+amplification+directly+from+clinical+samples.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000769-Campbell1" id="pone.0000769-Campbell1"></a>Campbell CL, Jiang Z, Savarese DM, Savarese TM (2001) Increased expression of the interleukin-11 receptor and evidence of STAT3 activation in prostate carcinoma. Am J Pathol  158: 25–32.  <ul class="find" data-citedArticleID="980334" data-doi="10.1016/s0002-9440(10)63940-5"><li><a href="http://dx.doi.org/10.1016/s0002-9440(10)63940-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Increased+expression+of+the+interleukin-11+receptor+and+evidence+of+STAT3+activation+in+prostate+carcinoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Increased+expression+of+the+interleukin-11+receptor+and+evidence+of+STAT3+activation+in+prostate+carcinoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000769-Zurita1" id="pone.0000769-Zurita1"></a>Zurita AJ, Troncoso P, Cardo-Vila M, Logothetis CJ, Pasqualini R, et al.  (2004) Combinatorial screenings in patients: the interleukin-11 receptor alpha as a candidate target in the progression of human prostate cancer. Cancer Res  64: 435–439.  <ul class="find" data-citedArticleID="980390" data-doi="10.1158/0008-5472.can-03-2675"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-03-2675" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combinatorial+screenings+in+patients%3A+the+interleukin-11+receptor+alpha+as+a+candidate+target+in+the+progression+of+human+prostate+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combinatorial+screenings+in+patients%3A+the+interleukin-11+receptor+alpha+as+a+candidate+target+in+the+progression+of+human+prostate+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000769-Kishimoto1" id="pone.0000769-Kishimoto1"></a>Kishimoto T, Akira S, Narazaki M, Taga T (1995) Interleukin-6 family of cytokines and gp130. Blood  86: 1243–1254.  <ul class="find" data-citedArticleID="980362"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Interleukin-6+family+of+cytokines+and+gp130.&amp;auth=&amp;atitle=Interleukin-6+family+of+cytokines+and+gp130." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interleukin-6+family+of+cytokines+and+gp130." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interleukin-6+family+of+cytokines+and+gp130.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000769-Gao1" id="pone.0000769-Gao1"></a>Gao L, Zhang L, Hu J, Li F, Shao Y, et al.  (2005) Down-regulation of signal transducer and activator of transcription 3 expression using vector-based small interfering RNAs suppresses growth of human prostate tumor in vivo. Clin Cancer Res  11: 6333–6341.  <ul class="find" data-citedArticleID="980344" data-doi="10.1158/1078-0432.ccr-05-0148"><li><a href="http://dx.doi.org/10.1158/1078-0432.ccr-05-0148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Down-regulation+of+signal+transducer+and+activator+of+transcription+3+expression+using+vector-based+small+interfering+RNAs+suppresses+growth+of+human+prostate+tumor+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Down-regulation+of+signal+transducer+and+activator+of+transcription+3+expression+using+vector-based+small+interfering+RNAs+suppresses+growth+of+human+prostate+tumor+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000769-Edwards1" id="pone.0000769-Edwards1"></a>Edwards J, Bartlett JM (2005) The androgen receptor and signal-transduction pathways in hormone-refractory prostate cancer. Part 1: Modifications to the androgen receptor Androgen-receptor cofactors and bypass pathways. BJU Int  95: 1320–1326.  <ul class="find" data-citedArticleID="980342" data-doi="10.1111/j.1464-410x.2005.05526.x"><li><a href="http://dx.doi.org/10.1111/j.1464-410x.2005.05526.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+androgen+receptor+and+signal-transduction+pathways+in+hormone-refractory+prostate+cancer.+Part+1%3A+Modifications+to+the+androgen+receptor+Androgen-receptor+cofactors+and+bypass+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+androgen+receptor+and+signal-transduction+pathways+in+hormone-refractory+prostate+cancer.+Part+1%3A+Modifications+to+the+androgen+receptor+Androgen-receptor+cofactors+and+bypass+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.XML" value="77638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.PDF" value="290685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g001.PNG_L" value="3130100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g001.PNG_M" value="87682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g001.PNG_S" value="6901"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g001.TIF" value="4230498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g001.PNG_I" value="28652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.t001.PNG_L" value="158607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.t001.PNG_M" value="102198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.t001.PNG_S" value="11696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.t001.TIF" value="731086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.t001.PNG_I" value="31090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g002.PNG_L" value="133162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g002.PNG_M" value="32442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g002.PNG_S" value="5361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g002.TIF" value="204228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g002.PNG_I" value="12738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g003.PNG_L" value="318774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g003.PNG_M" value="56535"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g003.PNG_S" value="8866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g003.TIF" value="622474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g003.PNG_I" value="20623"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g004.PNG_L" value="1728752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g004.PNG_M" value="38553"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g004.PNG_S" value="6992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g004.TIF" value="2220364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000769.g004.PNG_I" value="13112"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000769&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000769" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000769&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000769&volume=&issue=&title=Detection of Novel Amplicons in Prostate Cancer by Comprehensive Genomic Profiling of Prostate Cancer Cell Lines Using Oligonucleotide-Based ArrayCGH&author_name=Joern%20Kamradt%2C%20Volker%20Jung%2C%20Kerstin%20Wahrheit%2C%20Laura%20Tolosi%2C%20Joerg%20Rahnenfuehrer%2C%20Martin%20Schilling%2C%20Robert%20Walker%2C%20Sean%20Davis%2C%20Michael%20Stoeckle%2C%20Paul%20Meltzer%2C%20Bernd%20Wullich&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000769" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000769" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000769" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000769&amp;t=Detection%20of%20Novel%20Amplicons%20in%20Prostate%20Cancer%20by%20Comprehensive%20Genomic%20Profiling%20of%20Prostate%20Cancer%20Cell%20Lines%20Using%20Oligonucleotide-Based%20ArrayCGH" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000769&title=Detection%20of%20Novel%20Amplicons%20in%20Prostate%20Cancer%20by%20Comprehensive%20Genomic%20Profiling%20of%20Prostate%20Cancer%20Cell%20Lines%20Using%20Oligonucleotide-Based%20ArrayCGH&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000769&amp;title=Detection%20of%20Novel%20Amplicons%20in%20Prostate%20Cancer%20by%20Comprehensive%20Genomic%20Profiling%20of%20Prostate%20Cancer%20Cell%20Lines%20Using%20Oligonucleotide-Based%20ArrayCGH" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000769" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000769&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Detection of Novel Amplicons in Prostate Cancer by Comprehensive Genomic Profiling of Prostate Cancer Cell Lines Using Oligonucleotide-Based ArrayCGH';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000769';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Detection%20of%20Novel%20Amplicons%20in%20Prostate%20Cancer%20by%20Comprehensive%20Genomic%20Profiling%20of%20Prostate%20Cancer%20Cell%20Lines%20Using%20Oligonucleotide-Based%20ArrayCGH http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000769" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000769" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="24376"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Comparative+genomics%22" title="Search for articles in the subject area:'Comparative genomics'"><div class="flagText">Comparative genomics</div></a>
              <div data-categoryid="32937" data-articleid="24376"
                   data-categoryname="Comparative genomics"
                   class="flagImage" title="Flag 'Comparative genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DU145+cells%22" title="Search for articles in the subject area:'DU145 cells'"><div class="flagText">DU145 cells</div></a>
              <div data-categoryid="27423" data-articleid="24376"
                   data-categoryname="DU145 cells"
                   class="flagImage" title="Flag 'DU145 cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Metaphase%22" title="Search for articles in the subject area:'Metaphase'"><div class="flagText">Metaphase</div></a>
              <div data-categoryid="18755" data-articleid="24376"
                   data-categoryname="Metaphase"
                   class="flagImage" title="Flag 'Metaphase' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="24376"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oligonucleotides%22" title="Search for articles in the subject area:'Oligonucleotides'"><div class="flagText">Oligonucleotides</div></a>
              <div data-categoryid="19655" data-articleid="24376"
                   data-categoryname="Oligonucleotides"
                   class="flagImage" title="Flag 'Oligonucleotides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="47289" data-articleid="24376"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Prostate+cancer%22" title="Search for articles in the subject area:'Prostate cancer'"><div class="flagText">Prostate cancer</div></a>
              <div data-categoryid="19371" data-articleid="24376"
                   data-categoryname="Prostate cancer"
                   class="flagImage" title="Flag 'Prostate cancer' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1225'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6649'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7267&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
