

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Comparative Genomic Analyses of Copper Transporters and Cuproproteomes Reveal Evolutionary Dynamics of Copper Utilization and Its Link to Oxygen</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001378"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001378" />

    <meta name="citation_title" content="Comparative Genomic Analyses of Copper Transporters and Cuproproteomes Reveal Evolutionary Dynamics of Copper Utilization and Its Link to Oxygen"/>
    <meta itemprop="name" content="Comparative Genomic Analyses of Copper Transporters and Cuproproteomes Reveal Evolutionary Dynamics of Copper Utilization and Its Link to Oxygen"/>

      <meta name="citation_author" content="Perry G. Ridge"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Nebraska, Lincoln, Nebraska, United States of America"/>
      <meta name="citation_author" content="Yan Zhang"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Nebraska, Lincoln, Nebraska, United States of America"/>
      <meta name="citation_author" content="Vadim N. Gladyshev"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Nebraska, Lincoln, Nebraska, United States of America"/>

    <meta name="citation_date" content="2008/1/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001378.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1378"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Role of the copper-binding domain in the copper transport function of ATP7B, the P-type ATPase defective in Wilson's Disease.; citation_author=JR Forbes; citation_author=G His; citation_author=DW Cox; citation_journal_title=J Bio Chem; citation_volume=274; citation_number=1; citation_pages=12408-12413; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Metal transporters and disease.; citation_author=NC Andrews; citation_journal_title=Curr Opin Chem Biol; citation_volume=6; citation_number=2; citation_pages=181-186; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Motor neuron disease.; citation_author=K Talbot; citation_journal_title=Postgrad Med J; citation_volume=78; citation_number=3; citation_pages=513-519; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Chemistry and biochemistry of oxidative stress in neurodegenerative disease.; citation_author=LM Sayre; citation_author=MA Smith; citation_author=G Perry; citation_journal_title=Current Medicinal Chemistry; citation_volume=8; citation_number=4; citation_pages=721-738; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Early anaerobic metabolisms.; citation_author=DE Canfield; citation_author=MT Rosing; citation_author=C Bjerrum; citation_journal_title=Philosophical Transactions of The Royal Society B; citation_volume=361; citation_number=5; citation_pages=1819-1836; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The stepwise evolution of early life driven by energy conservation.; citation_author=JG Ferry; citation_author=CH House; citation_journal_title=Molecular Biology and Evolution; citation_volume=23; citation_number=6; citation_pages=1286-1292; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The bioinorganic chemistry of the ancient ocean: the co-evolution of cyanobacterial metal requirements and biogeochemical cycles at the archean-proterozoic boundary?; citation_author=MA Saito; citation_author=DM Sigman; citation_author=FMM Morel; citation_journal_title=Inorganica Chimica Acta; citation_volume=356; citation_number=7; citation_pages=308-318; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=RJP Williams; citation_author=JJR Fraústo da Silva; citation_number=8; citation_pages=253-262; citation_date=2006; citation_publisher=Elsevier BV; " />
      <meta name="citation_reference" content="citation_title=Computational reconstruction of iron- and manganese-responsive transcriptional networks in α-proteobacteria.; citation_author=DA Rodionov; citation_author=MS Gelfand; citation_author=JD Todd; citation_author=ARJ Curson; citation_author=AWB Johnston; citation_journal_title=PLoS Computational Biology; citation_volume=2; citation_number=9; citation_pages=1568-1585; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dynamic evolution of selenocysteine utilization in bacteria: a balance between selenoprotein loss and evolution of selenocysteine from redox-active cysteine residues.; citation_author=Y Zhang; citation_author=H Romero; citation_author=G Salinas; citation_author=VN Gladyshev; citation_journal_title=Genome Biology; citation_volume=7; citation_number=10; citation_pages=R94; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolutionary dynamics of eukaryotic selenoproteomes: large selenoproteomes may associate with aquatic and small with terrestrial life.; citation_author=AV Lobanov; citation_author=DE Fomenko; citation_author=Y Zhang; citation_author=A Sengupta; citation_author=DL Hatfield; citation_journal_title=Genome Biol; citation_volume=8; citation_number=11; citation_pages=R198; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Comparative and functional genomic analysis of prokaryotic nickel and cobalt uptake transporters: evidence for a novel group of ATP-binding cassette transporters.; citation_author=DA Rodionov; citation_author=P Hebbeln; citation_author=MS Gelfand; citation_author=T Eitinger; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=12; citation_pages=317-327; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A bacterial view of the periodic table: genes and proteins for toxic inorganic ions.; citation_author=S Silver; citation_author=LT Phung; citation_journal_title=J Ind Microbiol Biotechnol; citation_volume=32; citation_number=13; citation_pages=587-605; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cellular copper transport.; citation_author=CD Vulpe; citation_author=S Packman; citation_journal_title=Annu Rev Nutr; citation_volume=15; citation_number=14; citation_pages=293-322; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Transition metal speciation in the cell: insights from the chemistry of metal ion receptors.; citation_author=LA Finney; citation_author=TV O'Halloran; citation_journal_title=Science; citation_volume=300; citation_number=15; citation_pages=931-936; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Copper homeostasis in Enterococcus hirae.; citation_author=M Solioz; citation_author=JV Stoyanov; citation_journal_title=FEMS Microbiol Rev; citation_volume=27; citation_number=16; citation_pages=183-195; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Solution structures of a cyanobacterial metallchaperone: insight into an atypical copper-binding motif.; citation_author=L Banci; citation_author=I Bertini; citation_author=S Ciofi-Baffoni; citation_author=X Su; citation_author=GPM Borrelly; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=17; citation_pages=27502-27510; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Zn, Cu and Co in cyanobacteria: selective control of metal availability.; citation_author=JS Cavet; citation_author=GPM Borrelly; citation_author=NJ Robinson; citation_journal_title=FEMS Microbiol Rev; citation_volume=27; citation_number=18; citation_pages=165-181; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Two Menkes-type ATPases supply copper for photosynthesis in Synechocystis PCC 6803.; citation_author=S Tottey; citation_author=PR Rich; citation_author=SAM Rondet; citation_author=NJ Robinson; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=19; citation_pages=19999-20004; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Metal homeostasis in cyanobacteria and chloroplasts. Balancing benefits and risks to the photosynthetic apparatus.; citation_author=S Shcolnick; citation_author=N Keren; citation_journal_title=Plant Physiology; citation_volume=141; citation_number=20; citation_pages=805-810; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=New findings on evolution of metal homeostasis genes: evidence from comparative genome analysis of bacteria and archaea.; citation_author=JM Coombs; citation_author=T Barkay; citation_journal_title=Appl Environ Microbiol; citation_volume=71; citation_number=21; citation_pages=7083-7091; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulation of resistance to copper in Xanthomonas axonopodis pv. vesicatoria.; citation_author=AE Voloudakis; citation_author=TM Reignier; citation_author=DA Cooksey; citation_journal_title=Applied and Environmental Microbiology; citation_volume=71; citation_number=22; citation_pages=782-789; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Cu(II)-reductase NADH dehydrogenase-2 of Escherichia coli improves the bacterial growth in extreme copper concentrations and increases the resistance to the damage caused by copper and hydroperoxide.; citation_author=L Rodríguez-Montelongo; citation_author=SI Volentini; citation_author=RN Farias; citation_author=EM Massa; citation_author=VA Rapisarda; citation_journal_title=Archives of Biochemistry and Biophysics; citation_volume=451; citation_number=23; citation_pages=1-7; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence for Cu(I)-thiolate ligation and prediction of a putative copper-binding site in the Escherichia coli NADH dehydrogenase-2.; citation_author=VA Rapisarda; citation_author=RN Chehin; citation_author=J De Las Rivas; citation_author=L Rodriguez-Montelongo; citation_author=RN Farias; citation_journal_title=Arch Biochem Biophysics; citation_volume=405; citation_number=24; citation_pages=87-97; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Superoxide dismutase multigene family: a comparison of the CuZn-SOD (SOD1), Mn-SOD (SOD2), and EC-SOD (SOD3) gene structures, evolution, and expression.; citation_author=IN Zelko; citation_author=TJ Marianl; citation_author=RJ Folz; citation_journal_title=Free Radical Biology and Medicine; citation_volume=33; citation_number=25; citation_pages=337-349; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Nitrosocyanin, a red cupredoxin-like protein from Nitrosomonas europaea.; citation_author=DM Arciero; citation_author=BS Pierce; citation_author=MP Hendrich; citation_author=AB Hooper; citation_journal_title=Biochemistry; citation_volume=41; citation_number=26; citation_pages=1703-1709; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genomic analysis reveals widespread occurrence of new classes of copper nitrite reductases.; citation_author=MJ Ellis; citation_author=JG Grossman; citation_author=RR Eady; citation_author=SS Hasnain; citation_journal_title=J Biol Inorg Chem; citation_number=27; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Oxygen activation in a copper-containing amine oxidase.; citation_author=CM Wilmot; citation_journal_title=Biochem Soc Trans; citation_volume=31; citation_number=28; citation_pages=493-496; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Toward delineating the structure and function of the particulate methane monooxygenase from methanotropic bacteria.; citation_author=SI Chan; citation_author=KH-C Chen; citation_author=SS-F Yu; citation_author=C-L Chen; citation_author=SS-J Kuo; citation_journal_title=Biochemistry; citation_volume=43; citation_number=29; citation_pages=4421-4430; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=CotA of Bacillus subtilis is a copper-dependent laccase.; citation_author=MF Hullo; citation_author=I Moszer; citation_author=A Danchin; citation_author=I Martin-Verstraete; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=30; citation_pages=5426-5430; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The tat protein export pathway.; citation_author=B Berks; citation_author=F Sargent; citation_author=T Palmer; citation_journal_title=Molecular Microbiology; citation_volume=35; citation_number=31; citation_pages=260-274; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Escherichia coli mechanisms of copper homeostasis in a changing environment.; citation_author=C Rensing; citation_author=G Grass; citation_journal_title=FEMS Microbiol Rev; citation_volume=27; citation_number=32; citation_pages=197-213; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of cutC and cutF (nlpE) genes involved in copper tolerance in Escherichia Coli.; citation_author=SD Gupta; citation_author=BT Lee; citation_author=J Camakaris; citation_author=HC Wu; citation_journal_title=J Bacteriol; citation_volume=177; citation_number=33; citation_pages=4207-4215; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Genes involved in copper homeostasis in Escherichia coli.; citation_author=G Grass; citation_author=C Rensing; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=34; citation_pages=2145-2147; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A redox switch in CopC: an intriguing copper trafficking protein than binds copper(I) and copper(II) at different sites.; citation_author=F Arnesano; citation_author=L Banci; citation_author=I Bertini; citation_author=S Mangani; citation_author=AR Thompsett; citation_journal_title=Proceedings of the National Academy of the Sciences; citation_volume=100; citation_number=35; citation_pages=3814-3819; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Copper hypersensitivity and uptake in Pseudomonas syringae containing cloned components of the copper resistance operon.; citation_author=J Cha; citation_author=DA Cooksey; citation_journal_title=Appl Environ Microbiol; citation_volume=59; citation_number=36; citation_pages=1671-1674; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Toward automatic reconstruction of a highly resolved tree of life.; citation_author=FD Ciccarelli; citation_author=T Doerks; citation_author=C von Mering; citation_author=CJ Creevey; citation_author=B Snel; citation_journal_title=Science; citation_volume=311; citation_number=37; citation_pages=1283-1287; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dating the rise of atmospheric oxygen.; citation_author=A Bekker; citation_author=HD Holland; citation_author=P-L Wang; citation_author=III Rumble D; citation_author=HJ Stein; citation_journal_title=Nature; citation_volume=427; citation_number=38; citation_pages=117-120; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cops and robbers: putative evolution of copper oxygen-binding proteins.; citation_author=H Decker; citation_author=N Terwilliger; citation_journal_title=The Journal of Experimental Biology; citation_volume=203; citation_number=39; citation_pages=1777-1782; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The bioinorganic chemistry of the ancient ocean: the co-evolution of cyanobacterial metal requirements and biogeochemical cycles at the Archean-Proterozic boundary?; citation_author=MA Saito; citation_author=DM Signman; citation_author=FMM Morel; citation_journal_title=Inorganica Chimica Acta; citation_volume=356; citation_number=40; citation_pages=308-318; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Copper and the biological evolution.; citation_author=E Ochiai; citation_journal_title=BioSystems; citation_volume=16; citation_number=41; citation_pages=81-86; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Identification and characterization of a Streptococcus pyogenes ABC transporter with multiple specificity for metal cations.; citation_author=R Janulczyk; citation_author=J Pallon; citation_author=L Bjorck; citation_journal_title=Molecular Microbiology; citation_volume=34; citation_number=42; citation_pages=596-606; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Activation of superoxide dismutases: putting the metal to the petal.; citation_author=VC Culotta; citation_author=M Yang; citation_author=TV O'Halloran; citation_journal_title=Biochimica et Biophysica Acta; citation_volume=1763; citation_number=43; citation_pages=747-758; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High content of proteins containing 21st and 22nd amino acids, selenocysteine and pyrrolysine, in a symbiotic deltaproteobacterium of gutless worm Olavius algarvensis.; citation_author=Y Zhang; citation_author=N Gladyshev V; citation_journal_title=Nucleic Acids Res; citation_number=44; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=45; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Multiple sequence alignment with the Clustal series of programs.; citation_author=R Chenna; citation_author=H Sugawara; citation_author=T Koike; citation_author=R Lopez; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=46; citation_pages=3497-3500; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001378" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Comparative Genomic Analyses of Copper Transporters and Cuproproteomes Reveal Evolutionary Dynamics of Copper Utilization and Its Link to Oxygen"/>
    <meta name="twitter:description" content="Copper is an essential trace element in many organisms and is utilized in all domains of life. It is often used as a cofactor of redox proteins, but is also a toxic metal ion. Intracellular copper must be carefully handled to prevent the formation of reactive oxygen species which pose a threat to DNA, lipids, and proteins. In this work, we examined patterns of copper utilization in prokaryotes by analyzing the occurrence of copper transporters and copper-containing proteins. Many organisms, including those that lack copper-dependent proteins, had copper exporters, likely to protect against copper ions that inadvertently enter the cell. We found that copper use is widespread among prokaryotes, but also identified several phyla that lack cuproproteins. This is in contrast to the use of other trace elements, such as selenium, which shows more scattered and reduced usage, yet larger selenoproteomes. Copper transporters had different patterns of occurrence than cuproproteins, suggesting that the pathways of copper utilization and copper detoxification are independent of each other. We present evidence that organisms living in oxygen-rich environments utilize copper, whereas the majority of anaerobic organisms do not. In addition, among copper users, cuproproteomes of aerobic organisms were larger than those of anaerobic organisms. Prokaryotic cuproproteomes were small and dominated by a single protein, cytochrome c oxidase. The data are consistent with the idea that proteins evolved to utilize copper following the oxygenation of the Earth."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001378.g007"/>

  <meta property="og:title" content="Comparative Genomic Analyses of Copper Transporters and Cuproproteomes Reveal Evolutionary Dynamics of Copper Utilization and Its Link to Oxygen" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001378" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=76'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9966'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6182&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001378">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001378" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001378&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Comparative Genomic Analyses of Copper Transporters and Cuproproteomes Reveal Evolutionary Dynamics of Copper Utilization and Its Link to Oxygen
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Perry G. Ridge, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, University of Nebraska, Lincoln, Nebraska, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yan Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, University of Nebraska, Lincoln, Nebraska, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vadim N. Gladyshev
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:vgladyshev1@unl.edu">vgladyshev1@unl.edu</a></p>

                <p>Affiliation:
                  Department of Biochemistry, University of Nebraska, Lincoln, Nebraska, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001378</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001378" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001378" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001378" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001378" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001378-t001" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001378-g001" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001378-g002" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001378-g003" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001378-g004" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001378-g005" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001378-g006" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001378-g007" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001378">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001378" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Copper is an essential trace element in many organisms and is utilized in all domains of life. It is often used as a cofactor of redox proteins, but is also a toxic metal ion. Intracellular copper must be carefully handled to prevent the formation of reactive oxygen species which pose a threat to DNA, lipids, and proteins. In this work, we examined patterns of copper utilization in prokaryotes by analyzing the occurrence of copper transporters and copper-containing proteins. Many organisms, including those that lack copper-dependent proteins, had copper exporters, likely to protect against copper ions that inadvertently enter the cell. We found that copper use is widespread among prokaryotes, but also identified several phyla that lack cuproproteins. This is in contrast to the use of other trace elements, such as selenium, which shows more scattered and reduced usage, yet larger selenoproteomes. Copper transporters had different patterns of occurrence than cuproproteins, suggesting that the pathways of copper utilization and copper detoxification are independent of each other. We present evidence that organisms living in oxygen-rich environments utilize copper, whereas the majority of anaerobic organisms do not. In addition, among copper users, cuproproteomes of aerobic organisms were larger than those of anaerobic organisms. Prokaryotic cuproproteomes were small and dominated by a single protein, cytochrome c oxidase. The data are consistent with the idea that proteins evolved to utilize copper following the oxygenation of the Earth.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Ridge PG, Zhang Y, Gladyshev VN (2008) Comparative Genomic Analyses of Copper Transporters and Cuproproteomes Reveal Evolutionary Dynamics of Copper Utilization and Its Link to Oxygen. PLoS ONE 3(1):
          e1378.
            doi:10.1371/journal.pone.0001378</p><p><strong>Academic Editor: </strong>Jean-Nicolas Volff, Ecole Normale Supérieure de Lyon, France</p><p><strong>Received:</strong> October 9, 2007; <strong>Accepted:</strong> December 5, 2007; <strong>Published:</strong> January 2, 2008</p><p><strong>Copyright:</strong> © 2008 Ridge et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Supported in part by NIH GM061603.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>All organisms are thought to require metal ion cofactors (i.e., Fe, Zn, Mg, Mn, Co, Ni and Cu) which are involved in a wide variety of cellular processes. Additionally, certain non-metal trace elements, such as selenium (Se), and iodine (I) may be utilized. Most of these elements are necessary for redox catalysis and other enzymatic reactions, for sensing and signaling, and some serve structural roles in proteins. Defects in their homeostasis have been linked to a variety of diseases <a href="#pone.0001378-Forbes1">[1]</a>–<a href="#pone.0001378-Sayre1">[4]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Metals are particularly important for life and may have been utilized by organisms since the time life originated on Earth. For example, before the advent of oxygenic photosynthesis, the most active ecosystems were probably driven by the cycling of H<sub>2</sub> and Fe<sup>2+</sup> through primary production conducted by anoxygenic phototrophs <a href="#pone.0001378-Canfield1">[5]</a>. Additionally, it is thought that a substrate-level phosphorylation cycle may have been an early form of metabolism – one which required both iron and molybdenum <a href="#pone.0001378-Ferry1">[6]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Biological metal utilization has been driven, at least in part, by availability. As the Earth has evolved its chemical environment has significantly changed, which altered availability of certain metal ions <a href="#pone.0001378-Saito1">[7]</a>, <a href="#pone.0001378-Williams1">[8]</a>. For example, as environmental oxygen increased, various metal ions, such as Cu, Co, Ni, Zn, Cd, and Mo were released from their sulfide forms and became more soluble <a href="#pone.0001378-Williams1">[8]</a>, making them more readily available for biological utilization. Oxygenation of the Earth also changed the redox state of some metal ions, such as Fe (Fe<sup>2+</sup> to Fe<sup>3+</sup>), which reduced its availability <a href="#pone.0001378-Williams1">[8]</a>. Accordingly, organisms became less dependent on the metal ions that were scarce and adapted to use the available metal ions <a href="#pone.0001378-Saito1">[7]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Studying metal use in organisms can provide various insights. In this study, we use the term metalloproteome, which is a subset of the proteins from a particular proteome, which bind metal ions. Improved understanding of the composition and functions of metalloproteomes in prokaryotes, combined with our knowledge of the chemical evolution of the earth, can help decipher the evolutionary relationships among organisms and improve understanding of the roles metals play in biology. Analyses of evolutionary trends in trace element utilization have been performed for several trace elements, such as Fe and Mn <a href="#pone.0001378-Rodionov1">[9]</a>, Se <a href="#pone.0001378-Zhang1">[10]</a>, <a href="#pone.0001378-Lobanov1">[11]</a>, and Ni and Co <a href="#pone.0001378-Rodionov2">[12]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>One of the widely used trace elements is copper. This metal ion is known to be a cofactor in a number of proteins. Copper is redox-active (in biological systems it can exist as either Cu<sup>2+</sup> or Cu<sup>+</sup> and is more reactive in its reduced state <a href="#pone.0001378-Silver1">[13]</a>) and consequently, is a highly toxic element. The challenge, then, for copper-dependent organisms is to obtain sufficient levels of this metal ion to meet their needs, while tightly controlling intracellular copper to avoid toxicity. It is likely that little free copper exists in the cytoplasm (the same is not necessarily true for the periplasm <a href="#pone.0001378-Vulpe1">[14]</a>), both because of its toxicity, and because it exists at such low concentrations in the cytoplasm that it is unlikely to encounter its target proteins in a reasonable amount of time without assistance <a href="#pone.0001378-Finney1">[15]</a>. Thus, it is likely that copper is delivered to target proteins by metallochaperones.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>To date, 10 Cu-containing proteins (cuproproteins) have been characterized in prokaryotes, including cytochrome c oxidase (COX), NADH dehydrogenase-2 (ND2), Cu,Zn-superoxide dismutase (SOD1), nitrosocyanin, plastocyanin, Cu-containing nitrite reductase, Cu amine oxidase, particulate methane monooxygenase (pMMO), CotA, and tyrosinase. Each of these proteins is unable to substitute other metal ions for copper. Biosynthesis of cuproproteins is dependent on high-affinity uptake of the copper ion from natural environments, regulatory proteins, and other auxiliary proteins.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>In prokaryotes, two chaperones <a href="#pone.0001378-Vulpe1">[14]</a>, <a href="#pone.0001378-Solioz1">[16]</a>, <a href="#pone.0001378-Banci1">[17]</a> and 9 Cu-specific transporters have been reported and only one of these transporters, CtaA, has a clearly defined role as an importer <a href="#pone.0001378-Vulpe1">[14]</a>, <a href="#pone.0001378-Cavet1">[18]</a>–<a href="#pone.0001378-Shcolnick1">[20]</a>. CtaA imports copper from the periplasm rather than extracellular space <a href="#pone.0001378-Silver1">[13]</a>, <a href="#pone.0001378-Cavet1">[18]</a>, <a href="#pone.0001378-Tottey1">[19]</a>. Another transporter, PacS, transports copper from the cytoplasm to the thylakoid and assists in copper homeostasis <a href="#pone.0001378-Cavet1">[18]</a>, <a href="#pone.0001378-Tottey1">[19]</a>. PacS is a class of CopA <a href="#pone.0001378-Coombs1">[21]</a>, another transporter. CopA presents a unique challenge as multiple different copper transporters (and cuproproteins) have been named CopA. In <em>Escherichia coli</em>, CopA is an exporter, in <em>Enterococcus hirae</em> an importer (homologous to CtaA), and in <em>Pseudomonas</em> and <em>Xanthomonas</em> a multicopper oxidase <a href="#pone.0001378-Voloudakis1">[22]</a>. Regarding CopA transporters, there were two protein subgroups: a CopA1 subfamily containing ATPases with documented Cu<sup>+</sup> influx activity, and CopA2 proteins that are more ancient and include both influx and efflux transporters spanning the entire bacterial domain <a href="#pone.0001378-Coombs1">[21]</a>.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>In recent years, the complete genomes of numerous prokaryotic organisms have become available. Using this resource, it is possible to examine the occurrence and evolution of numerous biological pathways that organisms utilize. In this study we performed a comprehensive genome-wide investigation of both copper transport systems and cuproproteins. We analyzed 450 bacterial and 35 archaeal organisms for copper utilization. Our data revealed that copper is used by most prokaryotes and that cuproproteomes are larger in organisms living in oxygen-rich environments. Our results suggest that copper use increased as the atmosphere became more oxygenic.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Identification of Cu-utilizing prokaryotes</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We examined 450 sequenced bacterial genomes and 35 sequenced archaeal genomes for copper utilization by searching for occurrence of cuproproteins (see <a href="#pone-0001378-t001">Table 1</a> for a list of reference proteins). A majority of bacteria (326 of the 450 analyzed, or 72%) were found to be copper-utilizing or users (i.e., organisms that had at least one copper-dependent protein), while the remaining 124 bacteria (28%), appeared to be nonusers (i.e., organisms in which we were unable to identify even a single cuproprotein). In contrast, among archaea, most organisms appeared to be nonusers (24 archaea, or 69%), whereas 11 archaea, or 31%, were users. Overall, copper utilization was widely distributed in prokaryotes, with more than 2/3 of all organisms being dependent on this metal. Detailed distribution of copper utilization among prokaryotes is reported in <a href="#pone.0001378.s001">Table S1</a>.</p>
<div class="figure" id="pone-0001378-t001"><div class="img"><a name="pone-0001378-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001378.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001378.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>List of cuproproteins and copper transporters, chaperones, and regulators examined in the study.</span></strong></p><span>doi:10.1371/journal.pone.0001378.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Of 22 examined bacterial phyla represented by at least 4 completely sequenced genomes, 7 (enterobacteriales, vibrionaceae, pseudomonadaceae, xanthomonadaceae, burkholderiaceae, rhizobiaceae and cyanobacteria) consisted of bacteria that were all users. In addition, almost all bacteria from an additional 5 phyla (pasteurellaceae, rickettsiales, chloroflexi, actinobacteria, and bacillales) were users (<a href="#pone-0001378-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0001378-g001"><div class="img"><a name="pone-0001378-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Occurrence of cuproproteins in bacteria.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Phylogenetic tree was adapted from <a href="#pone.0001378-Ciccarelli1">[37]</a>. Reported is the total number of bacteria for each phylum and the numbers of bacteria that utilize a given cuproprotein. Numbers across the top refer to the cuproproteins in <a href="#pone-0001378-t001">Table 1</a>. The last two columns (Users and Nonusers) refer to the number of organisms in the specific phylum that are users and nonusers, respectively. Fields colored in red represent phyla where all bacteria belonging to that phylum were classified as either users or nonusers. The “# Cuproproteins” column shows the total number of cuproproteins utilized by all bacteria belonging to a particular phylum.</p>
<span>doi:10.1371/journal.pone.0001378.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Only a few bacterial phyla (thermotogae, chlorobi, lactobacillales, and mollicutes) consisted exclusively of apparent nonusers. In addition, all but one bacterium from each of an additional three phyla (chlamydiae, spirochaetes, and clostridia) appeared to be nonusers. The phyla with the highest number of apparent nonusers were subphyla of firmicutes, which are among the most ancient organisms. Interestingly, even though we found no connection between genome size and copper use in the bacterial domain, for firmicutes, users had an average genome size of 3.93 Mb while the genome size of nonusers was only 2.27 Mb.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>In archaea, the trend was somewhat reversed. There were 5 phyla with 4 or more representative organisms and only 2 of these, sulfobales and halobacteriales, consisted of exclusively users. Archaea from sulfobales prefer a harsh environment (temperature of 74°C or higher and pH of 3 or lower). In contrast, halobacteriales prefer a mild environment (temperature of 25°C and neutral pH).</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>Lastly, archaea from two additional phyla (methanosarcinales and thermococcales) all appeared to be nonusers, and all but one archaeon (<em>Picrophilus torridus</em> is the exception) from the final remaining phylum, thermoplasmales, were classified as nonusers (<a href="#pone-0001378-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0001378-g002"><div class="img"><a name="pone-0001378-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Occurrence of cuproproteins in Archaea.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Reported is the occurrence (by phylum) of cuproproteins and Cu exporters in archaea. No importers, repressors, or chaperones were identified in archaea. See legend to <a href="#pone-0001378-g001">Figure 1</a> for further details.</p>
<span>doi:10.1371/journal.pone.0001378.g002</span></div>

<h4>Distribution of Cu transporter systems</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p><a href="#pone-0001378-t001">Table 1</a> also shows all of the Cu importers/exporters analyzed in our study. CtaA (CopA in <em>Enterococcus hirae</em> <a href="#pone.0001378-Solioz1">[16]</a>), the only identified Cu-specific importer, was only present in cyanobacteria. 13 different cyanobacteria were included in this study and all but one had CtaA (<a href="#pone-0001378-g003">Figure 3</a>). No archaea that utilize CtaA were identified (<a href="#pone-0001378-g002">Figure 2</a>). Each of the organisms that utilize CtaA also utilizes at least one cuproprotein.</p>
<div class="figure" id="pone-0001378-g003"><div class="img"><a name="pone-0001378-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Occurrence of transporters, repressors and chaperones in bacteria.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Phylogenetic tree was adapted from <a href="#pone.0001378-Ciccarelli1">[37]</a>. Reported are the number of bacteria (by phylum) which utilized a given transporter, repressor, or chaperone. Numbers across the top refer to specific transporters or chaperones (see <a href="#pone-0001378-t001">Table 1</a> for protein names). Columns 10–17 are exporters, column 18 is the sole importer, columns 19–20 are chaperones, and column 21 is a transporter whose exact function has not been characterized. The last four columns report the number of exporters, importers, and chaperones present in each phyla.</p>
<span>doi:10.1371/journal.pone.0001378.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In bacteria, PacS (CopA in <em>Escherichia coli</em> <a href="#pone.0001378-Rensing1">[32]</a> or CopB in <em>Enterococcus hirae</em> <a href="#pone.0001378-Solioz1">[16]</a>) was the most widespread copper exporter. PacS was identified in 355 of 450 bacteria and was present in 29 of 32 phyla (absent in mollicutes, thermotogae, and rickettsiales). In archaea, CopA (PacS) was the only transporter identified (<a href="#pone-0001378-g002">Figure 2</a>). It was present in 6 of 13 phyla (archaeoglobales, halobacteriales, methanobacteriales, methanomicrobiales, methanosarcinales, and thermococcales).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>These two transporters, CtaA and PacS, have complimentary roles. CtaA is responsible for copper import from the periplasm to the cytoplasm and PacS is responsible for copper transport to the thylakoid <a href="#pone.0001378-Silver1">[13]</a>, <a href="#pone.0001378-Cavet1">[18]</a>, <a href="#pone.0001378-Tottey1">[19]</a>. Consistent with this, all of the examined organisms that appeared to utilize CtaA also utilize PacS.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>We also examined the genomes for the other 7 copper exporters and observed some interesting features. Most organisms had copper exporters, even if they appeared to be nonusers. Nearly 50% of nonusers had copper exporters while about 25% of users appeared to lack these proteins. It is interesting to note that more than half of the bacteria that did not have exporters (79 of 142, or 56%) were parasites (overall, only 29% of the examined bacteria were parasites). Regarding the occurrence of exporters in archaea, we again observed a reversal of the trend compared to bacteria. Only 2 users (18%) had exporters while exactly half the nonusers had these proteins.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>There did not appear to be any trend of importers and exporters co-occurring except for CtaA and PacS; however, PacS is nearly ubiquitous while CtaA is utilized by only a few organisms. CtaA and PacS as well as other importers and exporters are likely differentially regulated.</p>


<h4>Composition of cuproproteomes</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>For further discussion, we introduce and define the term cuproproteome. The cuproproteome of an organism is the set of proteins from its proteome which require a copper ion for their biological function (typically catalysis). Transporters, chaperones, and storage proteins constitute cellular Cu transport and maintenance machinery and are not part of the cuproproteome. Analysis of cuproproteomes revealed that organisms having cuproproteomes of identical size do not necessarily have the same set of cuproproteins. The biological function of copper is maintained by cuproproteomes, whereas the Cu transport and maintenance machinery serves an intermediate role and is not preserved during evolution when an organism lacks cuproproteins.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Distribution of cuproproteins is reported in <a href="#pone-0001378-g001">Figures 1</a> and <a href="#pone-0001378-g002">2</a> for bacteria and archaea, respectively. In bacteria, the most frequently utilized cuproprotein was cytochrome c oxidase (COX). COX was identified in 296 bacteria, or 91% of the users. Moreover, among those that appeared to utilize a single cuproprotein, about 70% (94 out of 136 organisms) only utilized COX. The same was true for archaea. COX was utilized by all 11 of the archaeal users. 7 of the 11 (64%) archaeal users only utilized COX.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>No other cuproproteins were nearly as widespread in bacteria. The second most frequently utilized cuproprotein was NADH dehydrogenase-2 (ND2) – utilized by 111 bacteria, or 34% of the users. Superoxide dismutase 1 (SOD1) was the only other cuproprotein utilized by more than 20% of the users (utilized by 21%), and was found predominantly in gram-negative bacteria. The remaining cuproproteins were utilized by 15% or less of the users and their frequencies were as follows:</p>


<ul class="bulletlist">

<li>nitrosocyanin, 15%</li>

<li>plastocyanin, 15%</li>

<li>Cu-containing nitrite reductase, 10%</li>

<li>Cu-amine oxidase, 6%</li>

<li>pMMO, 2%</li>

<li>CotA, 1%</li>

<li>tyrosinase, 1%</li>

</ul><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Only four cuproproteins were identified in archaea: COX, SOD1, nitrosocyanin, and Cu-containing nitrite reductase. As previously mentioned, COX was utilized by 11 archaea (all of the users) and SOD1, Cu-containing nitrite reductase, and nitrosocyanin, were only utilized by 2 (14% of the users), 2 (14% of the users), and 1 (7% of the users) archaea, respectively.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Although 9 cuproproteins are known, none of the examined organisms utilized all of them. In fact, cuproproteomes of most bacteria were quite small, with the largest identified bacterial cuproproteome having only 5 cuproproteins. Only a single bacterium (<em>Acidovorax avenae</em>), less than 1% of all the bacteria included in the study, had a cuproproteome consisting of 5 cuproproteins. More common, although still infrequent, were bacteria having cuproproteomes of 4 cuproproteins; 29 bacteria or 6% had 4 cuproproteins in their cuproproteomes. The most common cuproproteomes had only one cuproprotein (128, or 28%, of all bacteria). It is interesting to note that the second most common cuproproteome size was 0 cuproproteins (124, or 28% of bacteria). So, while copper use was widespread in bacteria, most copper-dependent organisms utilized few cuproproteins (<a href="#pone-0001378-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0001378-g004"><div class="img"><a name="pone-0001378-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Bacterial cuproproteomes.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Phylogenetic tree adapted from <a href="#pone.0001378-Ciccarelli1">[37]</a>. Numbers across the top (1–5) show the size of cuproproteomes (i.e., the number of cuproproteins in a phylum or organism). Displayed is the number of organisms from each phylum with the cuproproteome of the particular size.</p>
<span>doi:10.1371/journal.pone.0001378.g004</span></div><a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>Archaeal cuproproteomes appeared to be even smaller. The largest identified cuproproteome had only 3 cuproproteins (found only in <em>Haloarcula marismortui</em>). 3 archaea (<em>Sulfolobus acidocaldarius</em>, <em>Sulfolobus solfataricus</em>, and <em>Natronomonas pharaonis</em>), or 9%, had cuproproteomes consisting of two cuproproteins. Most archaea (~70%) appeared to utilize no cuproproteins (<a href="#pone-0001378-g002">Figures 2</a> and <a href="#pone-0001378-g005">5</a>).</p>
<div class="figure" id="pone-0001378-g005"><div class="img"><a name="pone-0001378-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Archaeal cuproproteomes.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>See the legend to <a href="#pone-0001378-g004">Figure 4</a>.</p>
<span>doi:10.1371/journal.pone.0001378.g005</span></div>

<h4>Copper utilization in relation to environmental factors and organism features</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We analyzed the following environmental factors and organism features to determine their effect on copper utilization:</p>


<ul class="bulletlist">

<li>Habitat</li>

<li>Oxygen requirement</li>

<li>Genome size</li>

<li>GC % in genome</li>

<li>Gram strain (bacteria)</li>

<li>Methanogen (archaea)</li>

<li>Optimal temperature</li>

<li>Optimal pH</li>

</ul><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>One feature that showed a clear association was oxygen requirement. In our study, 82 (18%) bacteria were anaerobic, 205 (46%) aerobic, 140 (31%) facultative, and 23 (5%) microaerophilic. Most anaerobic bacteria (62 of 85, or 73%) were nonusers, while most aerobic bacteria (192 of 205, or 94%) were users. 99 of 140 facultative bacteria (71%) and 15 of 23 microaerophilic bacteria (65%) were users (<a href="#pone-0001378-g006">Figure 6</a>). Excluding facultative organisms, a Chi-square test showed a statistically significant difference in Cu utilization between aerobic (including microaerophilic) and anaerobic organisms (P-value&lt;0.01).</p>
<div class="figure" id="pone-0001378-g006"><div class="img"><a name="pone-0001378-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Occurrence of Cu users and nonusers among bacteria differing in their dependence on oxygen.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Bacteria were divided based on their oxygen requirement into anaerobic, aerobic, facultative, and microaerophilic. Shown are the numbers of users/nonusers for each of these groups of organisms.</p>
<span>doi:10.1371/journal.pone.0001378.g006</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>This trend was even more pronounced in archaea (<a href="#pone-0001378-g007">Figure 7</a>). We examined 21 anaerobic, 9 aerobic, and 5 facultative archaea. All 21 anaerobic archaea were nonusers and all 9 aerobic archaea were users. Similarly, a Chi-square test suggested a significant difference in Cu utilization between aerobic and anaerobic species (P-value&lt;0.01). Thus, such a striking preference for the use of copper in aerobic organisms is present in both kingdoms of prokaryotes.</p>
<div class="figure" id="pone-0001378-g007"><div class="img"><a name="pone-0001378-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001378" data-uri="info:doi/10.1371/journal.pone.0001378.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001378.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001378.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001378.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001378.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Occurrence of Cu users and nonusers among archaea differing in their dependence on oxygen.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Archaea were divided based on their oxygen requirement into anaerobic, aerobic, facultative, and microaerophilic. Shown are the numbers of users/nonusers for each of these groups of organisms.</p>
<span>doi:10.1371/journal.pone.0001378.g007</span></div><a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>Not surprising, based on these results, aerobic bacteria had the largest average cuproproteomes with an average size of 1.8 cuproproteins, compared to anaerobic bacteria which had the smallest average cuproproteomes (0.4 cuproproteins). Surprising, however, is that among users, facultative and microaerophilic bacteria had larger average cuproproteomes (2.26 cuproproteins) than aerobic users which had an average cuproproteome of 1.93 cuproproteins. In archaea, where all the aerobic organisms are users and all the anaerobic organisms are nonusers, aerobic organisms clearly have the larger average cuproproteomes.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Oxygen levels are thought to have been quite low until 2.45 billion years ago, but by 2.32 billion years ago had sharply risen <a href="#pone.0001378-Bekker1">[38]</a>. Oxygen levels rose following the origin of oxygenic photosynthetic organisms <a href="#pone.0001378-Decker1">[39]</a>. At the same time, levels of bioavailable copper increased <a href="#pone.0001378-Saito2">[40]</a>. Prior to ocean oxygenation, copper likely existed as a sulfide and was insoluble in aqueous solutions. It was suggested <a href="#pone.0001378-Ochiai1">[41]</a> that copper was largely unavailable for cellular life; however, upon the rise of oxygen in the atmosphere, copper was converted, at least in part to Cu<sup>2+</sup>, and as such, became more soluble and more readily available for biological use <a href="#pone.0001378-Ochiai1">[41]</a>. It is possible that at least some bacteria evolved near hydrothermal vents in waters replete with heavy metal ions, where sulfides would likely have been more soluble. However, the large-scale evolution of cuproproteins likely occurred in oxygen-rich environments where copper was plentiful. Our analyses of copper utilizing prokaryotes clearly support this idea. We found that in most prokaryotes that require oxygen are copper users, while most anaerobic prokaryotes are nonusers.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Prior to the evolution of cuproproteins, organisms may have already utilized copper exporters <a href="#pone.0001378-Solioz1">[16]</a>, but with the increase in soluble copper, this function became even more important as a means of removal of this metal ion. The presence of copper exporters in nonusers and the sheer number of known copper exporters suggest that copper can be taken up accidentally and these organisms need a defense mechanism against copper ions that inadvertently enter the cell.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Although exporters have been readily identified, importers have almost completely eluded researchers. As pointed out above, only one copper-dependent importer has been identified in prokaryotes. However, it is noteworthy that at least one example of an ABC transporter has been reported which has broad specificity for a number of metal ions, including copper, and this could potentially explain copper uptake by prokaryotes <a href="#pone.0001378-Janulczyk1">[42]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Additionally, while there are examples of cuproproteins that obtain copper from the cytoplasm (for example, plastocyanin gets copper from PacS, which exports copper from the cytoplasm to the thylakoid <a href="#pone.0001378-Cavet1">[18]</a>, <a href="#pone.0001378-Tottey1">[19]</a>) most cuproproteins are periplasmic or membrane proteins and therefore it is possible that little cytoplasmic copper is needed for these proteins, which could help explain the lack of copper-specific importers.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In this study we only included proteins that are unable to function without copper. While no reports of prokaryotic enzymes evolving by changing the catalytic copper to other metals exist, it is a possibility, although unlikely, that some of the proteins we have identified as cuproproteins actually utilize a different metal ion.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Various cuproproteins have been identified by researchers over the years. The two most widespread cuproproteins are COX and ND2, both of which are involved in the electron transport chain. The redox properties of copper make this metal ion a prime candidate for use in the active sites of oxidoreductases involved in energy generation. In fact, in bacteria, 84% of the cuproproteins, including COX and ND2, are involved in energy producing pathways, and in archaea 86% of the cuproproteins are involved in ATP production. These data suggest that ATP generation via oxygen-dependent respiration is the major use of copper in biology. Being readily available in the environment and with such wide use in oxygenic organisms, it is a paradox that the utilization of this metal is restricted to only a few enzymes in prokaryotes.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Copper use in organisms does have its associated dangers though, and organisms which have evolved to use this metal have to have ways to deal with copper toxicity. Copper toxicity comes from its ability to produce reactive oxygen species (ROS). Thus, copper is intimately involved in the cellular control of redox homeostasis. SODs are important proteins which protect the cell against ROS by removing superoxide anions. Each of the known SODs requires at least one metal cofactor. Among these is a cuproprotein, SOD1, which utilizes both copper and zinc. Other SODs are known to use iron, manganese, or nickel <a href="#pone.0001378-Zelko1">[25]</a>, <a href="#pone.0001378-Culotta1">[43]</a>. Interestingly, copper is used in both respiration and the associated removal of ROS, which are by-products of oxygen-dependent respiration.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Another interesting cuproprotein is a multicopper oxidase, CotA, which is a spore coat protein. Only four bacteria (and no archaea) were found to utilize CotA, all of which were Bacillales. It appears that CotA is a recently evolved cuproprotein. Other multicopper oxidases studied (CueO and PcoA) are components of efflux systems.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Cuproproteomes appear to be quite small in prokaryotes. In both bacteria and archaea, the most common cuproproteome size for users was one, and frequently this single cuproprotein was COX. Interestingly, organisms with cuproproteomes of the same size do not necessarily utilize the same cuproproteins, and most cuproproteins are used in a limited number of organisms.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>The use of copper can be compared (or rather contrasted) with that of another trace element, selenium. The size of prokaryotic selenoproteomes is highly variable, from 0 to 56 <a href="#pone.0001378-Zhang1">[10]</a>, <a href="#pone.0001378-Zhang2">[44]</a>. However, approximately 80% of prokaryotes do not have selenoproteins. Thus, whereas prokaryotic selenium use shows a mosaic pattern and most selenoprotein-rich organisms are anaerobic, copper is utilized by a limited number of processes (essentially restricted to respiration), but in many aerobic organisms.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>Our study was based on the premise that most cuproproteins are already known. The striking occurrence of these proteins in aerobic organisms suggests that we could reliably identify most users. The possibility cannot be excluded, however, that additional, unknown Cu-dependent proteins are present in some prokaryotes. Nevertheless, while additional cuproproteins would increase the size of cuproproteomes, it is unlikely that this would lead to identification of many additional users.</p>
<a id="article1.body1.sec3.p12" name="article1.body1.sec3.p12"></a><p>It would be interesting to expand this work to the third super-kingdom, eukaryotes. By definition, these organisms evolved in oxygenated environments, but whether copper utilization remained widespread during evolution of eukaryotes is not known. There is some overlap between eukaryotes and prokaryotes on which a study of this nature could be based; for example, both prokaryotes and eukaryotes have COX and SOD1.</p>
<a id="article1.body1.sec3.p13" name="article1.body1.sec3.p13"></a><p>Finally, we have pointed out that some very important enzymes require a copper cofactor, and while many of them are periplasmic and membrane-bound, some are intracellular. In the future, it would be interesting to examine the means by which prokaryotes bring copper into the cell under conditions of deficiency in this trace element. Very little is known about copper import in prokaryotes; in many cases unspecific import systems would likely be sufficient, but whether this is the major route for copper use is not known.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Genomic sequence resources</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>In our study, we examined fully sequenced prokaryotic genomes (450 unique bacterial and 35 unique archaeal genomes were available as of January 31, 2007). Only a single strain of each species was included in this research. A list of fully sequenced prokaryotic genomes, both those used in this research and those completed since we began our research, can be found on the NCBI website at: <a href="http://www.ncbi.nlm.nih.gov/sutils/genom_table.cgi">http://www.ncbi.nlm.nih.gov/sutils/genom​_table.cgi</a>.</p>


<h4>Identification of Cu-utilizing organisms, Cu-specific transporters, and cuproproteins</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Initially, primary literature was used to identify Cu-dependent proteins, and Cu importers, exporters, and regulators. Cuproproteins were included in the set if their close homologs were experimentally demonstrated to be specific for copper. Copper transporters were included if they were previously reported to be copper-specific or if they had a much higher affinity for copper than other metal ions. A complete list of identified proteins, importers, exporters, and regulators is shown in <a href="#pone-0001378-t001">Table 1</a>.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Organisms that had at least one copper-dependent protein were considered copper-dependent, or users. If we were unable to find at least one cuproprotein in an organism it was designated as copper-independent or a nonuser. Determining if a specific organism had cuproproteins was accomplished in the following steps:</p>


<ul class="bulletlist">

<li>Using primary literature, organisms were identified that had a particular cuproprotein;</li>

<li>TBlastn <a href="#pone.0001378-Altschul1">[45]</a> search with default parameters was performed against all organisms in the study using the protein sequences from the step above;</li>

<li>The organism from each phylum having a proteome containing the examined cuproprotein and having the highest homology to the query sequence was identified;</li>

<li>A new Blast search was performed using the protein sequence from the previous step as query sequence and only searching the genomes from that particular phylum. In the case that no organism from the phylum had the original protein, this additional search was not performed;</li>

<li>Manual analysis of the resulting sequences to determine which organisms had the cuproprotein in question.</li>

</ul>

<h4>Other analysis tools</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Inferred phylogenies were used to verify similarity to putative cuproprotein sequences. Multiple sequence alignments of proteins and inferred phylogenies were created using ClustalW <a href="#pone.0001378-Chenna1">[46]</a> with default parameters. Manual analysis of the inferred phylogeny was performed to identify which candidate sequences clustered around the putative cuproprotein.</p>


<h4>Identification of environmental and other factors for each organism</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>To examine trends of copper utilization we collected information for various environmental factors (e.g., habitat, oxygen requirement, optimal temperature and optimal pH), and other factors and properties (e.g., genome size, GC content and Gram strain) which were retrieved from the NCBI prokaryotic genome project database (<a href="http://www.ncbi.nlm.nih.gov/sites/entrezcmdFiledbgenomeprj">http://www.ncbi.nlm.nih.gov/sites/entrez​cmdFiledbgenomeprj</a>) for all examined archaeal and bacterial species.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001378"></div><a name="pone.0001378.s001" id="pone.0001378.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001378.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001378.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.18 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to Drs. Gregor Grass (Molecular Microbiology, Martin-Luther-University, Halle/Saale, Germany) and Jaekwon Lee (Biochemistry, Redox Biology Center, University of Nebraska-Lincoln, Lincoln, USA) for the insightful and helpful comments.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: VG YZ PR. Performed the experiments: PR. Analyzed the data: VG YZ PR. Wrote the paper: VG PR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001378-Forbes1" id="pone.0001378-Forbes1"></a>Forbes JR, His G, Cox DW (1999) Role of the copper-binding domain in the copper transport function of ATP7B, the P-type ATPase defective in Wilson's Disease. J Bio Chem  274: 12408–12413.  <ul class="find" data-citedArticleID="1036272" data-doi="10.1074/jbc.274.18.12408"><li><a href="http://dx.doi.org/10.1074/jbc.274.18.12408" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+copper-binding+domain+in+the+copper+transport+function+of+ATP7B%2C+the+P-type+ATPase+defective+in+Wilson%27s+Disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+copper-binding+domain+in+the+copper+transport+function+of+ATP7B%2C+the+P-type+ATPase+defective+in+Wilson%27s+Disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001378-Andrews1" id="pone.0001378-Andrews1"></a>Andrews NC (2002) Metal transporters and disease. Curr Opin Chem Biol  6: 181–186.  <ul class="find" data-citedArticleID="1036236"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Metal+transporters+and+disease.&amp;auth=&amp;atitle=Metal+transporters+and+disease." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metal+transporters+and+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metal+transporters+and+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001378-Talbot1" id="pone.0001378-Talbot1"></a>Talbot K (2002) Motor neuron disease. Postgrad Med J  78: 513–519.  <ul class="find" data-citedArticleID="1036308" data-doi="10.1136/pmj.78.923.513"><li><a href="http://dx.doi.org/10.1136/pmj.78.923.513" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Motor+neuron+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Motor+neuron+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001378-Sayre1" id="pone.0001378-Sayre1"></a>Sayre LM, Smith MA, Perry G (2001) Chemistry and biochemistry of oxidative stress in neurodegenerative disease. Current Medicinal Chemistry  8: 721–738.  <ul class="find" data-citedArticleID="1036300" data-doi="10.2174/0929867013372922"><li><a href="http://dx.doi.org/10.2174/0929867013372922" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemistry+and+biochemistry+of+oxidative+stress+in+neurodegenerative+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemistry+and+biochemistry+of+oxidative+stress+in+neurodegenerative+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001378-Canfield1" id="pone.0001378-Canfield1"></a>Canfield DE, Rosing MT, Bjerrum C (2006) Early anaerobic metabolisms. Philosophical Transactions of The Royal Society B  361: 1819–1836.  <ul class="find" data-citedArticleID="1036248" data-doi="10.1098/rstb.2006.1906"><li><a href="http://dx.doi.org/10.1098/rstb.2006.1906" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+anaerobic+metabolisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+anaerobic+metabolisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001378-Ferry1" id="pone.0001378-Ferry1"></a>Ferry JG, House CH (2006) The stepwise evolution of early life driven by energy conservation. Molecular Biology and Evolution  23: 1286–1292.  <ul class="find" data-citedArticleID="1036268" data-doi="10.1093/molbev/msk014"><li><a href="http://dx.doi.org/10.1093/molbev/msk014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+stepwise+evolution+of+early+life+driven+by+energy+conservation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+stepwise+evolution+of+early+life+driven+by+energy+conservation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001378-Saito1" id="pone.0001378-Saito1"></a>Saito MA, Sigman DM, Morel FMM (2003) The bioinorganic chemistry of the ancient ocean: the co-evolution of cyanobacterial metal requirements and biogeochemical cycles at the archean-proterozoic boundary? Inorganica Chimica Acta  356: 308–318.  <ul class="find" data-citedArticleID="1036296" data-doi="10.1016/s0020-1693(03)00442-0"><li><a href="http://dx.doi.org/10.1016/s0020-1693(03)00442-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+bioinorganic+chemistry+of+the+ancient+ocean%3A+the+co-evolution+of+cyanobacterial+metal+requirements+and+biogeochemical+cycles+at+the+archean-proterozoic+boundary%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+bioinorganic+chemistry+of+the+ancient+ocean%3A+the+co-evolution+of+cyanobacterial+metal+requirements+and+biogeochemical+cycles+at+the+archean-proterozoic+boundary%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001378-Williams1" id="pone.0001378-Williams1"></a>Williams RJP, Fraústo da Silva JJR (2006) The Chemistry of Evolution: The Development of our Ecosystem. The Netherlands: Elsevier BV. pp. 253–262.  <ul class="find-nolinks"></ul></li><li><span class="label">9.
              </span><a name="pone.0001378-Rodionov1" id="pone.0001378-Rodionov1"></a>Rodionov DA, Gelfand MS, Todd JD, Curson ARJ, Johnston AWB (2006) Computational reconstruction of iron- and manganese-responsive transcriptional networks in α-proteobacteria. PLoS Computational Biology  2: 1568–1585.  <ul class="find" data-citedArticleID="1036290" data-doi="10.1371/journal.pcbi.0020163.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020163.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+reconstruction+of+iron-+and+manganese-responsive+transcriptional+networks+in+%CE%B1-proteobacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+reconstruction+of+iron-+and+manganese-responsive+transcriptional+networks+in+%CE%B1-proteobacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001378-Zhang1" id="pone.0001378-Zhang1"></a>Zhang Y, Romero H, Salinas G, Gladyshev VN (2006) Dynamic evolution of selenocysteine utilization in bacteria: a balance between selenoprotein loss and evolution of selenocysteine from redox-active cysteine residues. Genome Biology  7: R94.  <ul class="find" data-citedArticleID="1036322"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Dynamic+evolution+of+selenocysteine+utilization+in+bacteria%3A+a+balance+between+selenoprotein+loss+and+evolution+of+selenocysteine+from+redox-active+cysteine+residues.&amp;auth=&amp;atitle=Dynamic+evolution+of+selenocysteine+utilization+in+bacteria%3A+a+balance+between+selenoprotein+loss+and+evolution+of+selenocysteine+from+redox-active+cysteine+residues." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+evolution+of+selenocysteine+utilization+in+bacteria%3A+a+balance+between+selenoprotein+loss+and+evolution+of+selenocysteine+from+redox-active+cysteine+residues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+evolution+of+selenocysteine+utilization+in+bacteria%3A+a+balance+between+selenoprotein+loss+and+evolution+of+selenocysteine+from+redox-active+cysteine+residues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001378-Lobanov1" id="pone.0001378-Lobanov1"></a>Lobanov AV, Fomenko DE, Zhang Y, Sengupta A, Hatfield DL, et al.  (2007) Evolutionary dynamics of eukaryotic selenoproteomes: large selenoproteomes may associate with aquatic and small with terrestrial life. Genome Biol  8: R198.  <ul class="find" data-citedArticleID="1036282" data-doi="10.1186/gb-2007-8-9-r198"><li><a href="http://dx.doi.org/10.1186/gb-2007-8-9-r198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+dynamics+of+eukaryotic+selenoproteomes%3A+large+selenoproteomes+may+associate+with+aquatic+and+small+with+terrestrial+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+dynamics+of+eukaryotic+selenoproteomes%3A+large+selenoproteomes+may+associate+with+aquatic+and+small+with+terrestrial+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001378-Rodionov2" id="pone.0001378-Rodionov2"></a>Rodionov DA, Hebbeln P, Gelfand MS, Eitinger T (2006) Comparative and functional genomic analysis of prokaryotic nickel and cobalt uptake transporters: evidence for a novel group of ATP-binding cassette transporters. J Bacteriol  188: 317–327.  <ul class="find" data-citedArticleID="1036292" data-doi="10.1128/jb.188.1.317-327.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.1.317-327.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+and+functional+genomic+analysis+of+prokaryotic+nickel+and+cobalt+uptake+transporters%3A+evidence+for+a+novel+group+of+ATP-binding+cassette+transporters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+and+functional+genomic+analysis+of+prokaryotic+nickel+and+cobalt+uptake+transporters%3A+evidence+for+a+novel+group+of+ATP-binding+cassette+transporters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001378-Silver1" id="pone.0001378-Silver1"></a>Silver S, Phung LT (2005) A bacterial view of the periodic table: genes and proteins for toxic inorganic ions. J Ind Microbiol Biotechnol  32: 587–605.  <ul class="find" data-citedArticleID="1036304" data-doi="10.1007/s10295-005-0019-6"><li><a href="http://dx.doi.org/10.1007/s10295-005-0019-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+bacterial+view+of+the+periodic+table%3A+genes+and+proteins+for+toxic+inorganic+ions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+bacterial+view+of+the+periodic+table%3A+genes+and+proteins+for+toxic+inorganic+ions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001378-Vulpe1" id="pone.0001378-Vulpe1"></a>Vulpe CD, Packman S (1995) Cellular copper transport. Annu Rev Nutr  15: 293–322.  <ul class="find" data-citedArticleID="1036314" data-doi="10.1146/annurev.nu.15.070195.001453"><li><a href="http://dx.doi.org/10.1146/annurev.nu.15.070195.001453" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cellular+copper+transport." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cellular+copper+transport.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001378-Finney1" id="pone.0001378-Finney1"></a>Finney LA, O'Halloran TV (2003) Transition metal speciation in the cell: insights from the chemistry of metal ion receptors. Science  300: 931–936.  <ul class="find" data-citedArticleID="1036270" data-doi="10.1126/science.1085049"><li><a href="http://dx.doi.org/10.1126/science.1085049" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transition+metal+speciation+in+the+cell%3A+insights+from+the+chemistry+of+metal+ion+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transition+metal+speciation+in+the+cell%3A+insights+from+the+chemistry+of+metal+ion+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001378-Solioz1" id="pone.0001378-Solioz1"></a>Solioz M, Stoyanov JV (2003) Copper homeostasis in <em>Enterococcus hirae</em>. FEMS Microbiol Rev  27: 183–195.  <ul class="find" data-citedArticleID="1036306" data-doi="10.1016/s0168-6445(03)00053-6"><li><a href="http://dx.doi.org/10.1016/s0168-6445(03)00053-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Copper+homeostasis+in+Enterococcus+hirae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Copper+homeostasis+in+Enterococcus+hirae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001378-Banci1" id="pone.0001378-Banci1"></a>Banci L, Bertini I, Ciofi-Baffoni S, Su X, Borrelly GPM, et al.  (2004) Solution structures of a cyanobacterial metallchaperone: insight into an atypical copper-binding motif. J Biol Chem  279: 27502–27510.  <ul class="find" data-citedArticleID="1036242" data-doi="10.1074/jbc.m402005200"><li><a href="http://dx.doi.org/10.1074/jbc.m402005200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Solution+structures+of+a+cyanobacterial+metallchaperone%3A+insight+into+an+atypical+copper-binding+motif." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Solution+structures+of+a+cyanobacterial+metallchaperone%3A+insight+into+an+atypical+copper-binding+motif.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001378-Cavet1" id="pone.0001378-Cavet1"></a>Cavet JS, Borrelly GPM, Robinson NJ (2003) Zn, Cu and Co in cyanobacteria: selective control of metal availability. FEMS Microbiol Rev  27: 165–181.  <ul class="find" data-citedArticleID="1036250" data-doi="10.1016/s0168-6445(03)00050-0"><li><a href="http://dx.doi.org/10.1016/s0168-6445(03)00050-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Zn%2C+Cu+and+Co+in+cyanobacteria%3A+selective+control+of+metal+availability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Zn%2C+Cu+and+Co+in+cyanobacteria%3A+selective+control+of+metal+availability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001378-Tottey1" id="pone.0001378-Tottey1"></a>Tottey S, Rich PR, Rondet SAM, Robinson NJ (2001) Two Menkes-type ATPases supply copper for photosynthesis in <em>Synechocystis</em> PCC 6803. J Biol Chem  276: 19999–20004.  <ul class="find" data-citedArticleID="1036310" data-doi="10.1074/jbc.m011243200"><li><a href="http://dx.doi.org/10.1074/jbc.m011243200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+Menkes-type+ATPases+supply+copper+for+photosynthesis+in+Synechocystis+PCC+6803." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+Menkes-type+ATPases+supply+copper+for+photosynthesis+in+Synechocystis+PCC+6803.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001378-Shcolnick1" id="pone.0001378-Shcolnick1"></a>Shcolnick S, Keren N (2006) Metal homeostasis in cyanobacteria and chloroplasts. Balancing benefits and risks to the photosynthetic apparatus. Plant Physiology  141: 805–810.  <ul class="find" data-citedArticleID="1036302" data-doi="10.1104/pp.106.079251"><li><a href="http://dx.doi.org/10.1104/pp.106.079251" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metal+homeostasis+in+cyanobacteria+and+chloroplasts.+Balancing+benefits+and+risks+to+the+photosynthetic+apparatus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metal+homeostasis+in+cyanobacteria+and+chloroplasts.+Balancing+benefits+and+risks+to+the+photosynthetic+apparatus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001378-Coombs1" id="pone.0001378-Coombs1"></a>Coombs JM, Barkay T (2005) New findings on evolution of metal homeostasis genes: evidence from comparative genome analysis of bacteria and archaea. Appl Environ Microbiol  71: 7083–7091.  <ul class="find" data-citedArticleID="1036260" data-doi="10.1128/aem.71.11.7083-7091.2005"><li><a href="http://dx.doi.org/10.1128/aem.71.11.7083-7091.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+findings+on+evolution+of+metal+homeostasis+genes%3A+evidence+from+comparative+genome+analysis+of+bacteria+and+archaea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+findings+on+evolution+of+metal+homeostasis+genes%3A+evidence+from+comparative+genome+analysis+of+bacteria+and+archaea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001378-Voloudakis1" id="pone.0001378-Voloudakis1"></a>Voloudakis AE, Reignier TM, Cooksey DA (2005) Regulation of resistance to copper in Xanthomonas axonopodis pv. vesicatoria. Applied and Environmental Microbiology  71: 782–789.  <ul class="find" data-citedArticleID="1036312" data-doi="10.1128/aem.71.2.782-789.2005"><li><a href="http://dx.doi.org/10.1128/aem.71.2.782-789.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+resistance+to+copper+in+Xanthomonas+axonopodis+pv.+vesicatoria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+resistance+to+copper+in+Xanthomonas+axonopodis+pv.+vesicatoria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001378-RodrguezMontelongo1" id="pone.0001378-RodrguezMontelongo1"></a>Rodríguez-Montelongo L, Volentini SI, Farias RN, Massa EM, Rapisarda VA (2006) The Cu(II)-reductase NADH dehydrogenase-2 of <em>Escherichia coli</em> improves the bacterial growth in extreme copper concentrations and increases the resistance to the damage caused by copper and hydroperoxide. Archives of Biochemistry and Biophysics  451: 1–7.  <ul class="find" data-citedArticleID="1036294" data-doi="10.1016/j.abb.2006.04.019"><li><a href="http://dx.doi.org/10.1016/j.abb.2006.04.019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Cu%28II%29-reductase+NADH+dehydrogenase-2+of+Escherichia+coli+improves+the+bacterial+growth+in+extreme+copper+concentrations+and+increases+the+resistance+to+the+damage+caused+by+copper+and+hydroperoxide." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Cu%28II%29-reductase+NADH+dehydrogenase-2+of+Escherichia+coli+improves+the+bacterial+growth+in+extreme+copper+concentrations+and+increases+the+resistance+to+the+damage+caused+by+copper+and+hydroperoxide.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001378-Rapisarda1" id="pone.0001378-Rapisarda1"></a>Rapisarda VA, Chehin RN, De Las Rivas J, Rodriguez-Montelongo L, Farias RN, et al.  (2002) Evidence for Cu(I)-thiolate ligation and prediction of a putative copper-binding site in the Escherichia coli NADH dehydrogenase-2. Arch Biochem Biophysics  405: 87–97.  <ul class="find" data-citedArticleID="1036286" data-doi="10.1016/s0003-9861(02)00277-1"><li><a href="http://dx.doi.org/10.1016/s0003-9861(02)00277-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+Cu%28I%29-thiolate+ligation+and+prediction+of+a+putative+copper-binding+site+in+the+Escherichia+coli+NADH+dehydrogenase-2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+Cu%28I%29-thiolate+ligation+and+prediction+of+a+putative+copper-binding+site+in+the+Escherichia+coli+NADH+dehydrogenase-2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001378-Zelko1" id="pone.0001378-Zelko1"></a>Zelko IN, Marianl TJ, Folz RJ (2002) Superoxide dismutase multigene family: a comparison of the CuZn-SOD (SOD1), Mn-SOD (SOD2), and EC-SOD (SOD3) gene structures, evolution, and expression. Free Radical Biology and Medicine  33: 337–349.  <ul class="find" data-citedArticleID="1036320" data-doi="10.1016/s0891-5849(02)00905-x"><li><a href="http://dx.doi.org/10.1016/s0891-5849(02)00905-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Superoxide+dismutase+multigene+family%3A+a+comparison+of+the+CuZn-SOD+%28SOD1%29%2C+Mn-SOD+%28SOD2%29%2C+and+EC-SOD+%28SOD3%29+gene+structures%2C+evolution%2C+and+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Superoxide+dismutase+multigene+family%3A+a+comparison+of+the+CuZn-SOD+%28SOD1%29%2C+Mn-SOD+%28SOD2%29%2C+and+EC-SOD+%28SOD3%29+gene+structures%2C+evolution%2C+and+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001378-Arciero1" id="pone.0001378-Arciero1"></a>Arciero DM, Pierce BS, Hendrich MP, Hooper AB (2002) Nitrosocyanin, a red cupredoxin-like protein from <em>Nitrosomonas europaea</em>. Biochemistry  41: 1703–1709.  <ul class="find" data-citedArticleID="1036238" data-doi="10.1021/bi015908w"><li><a href="http://dx.doi.org/10.1021/bi015908w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nitrosocyanin%2C+a+red+cupredoxin-like+protein+from+Nitrosomonas+europaea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nitrosocyanin%2C+a+red+cupredoxin-like+protein+from+Nitrosomonas+europaea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001378-Ellis1" id="pone.0001378-Ellis1"></a>Ellis MJ, Grossman JG, Eady RR, Hasnain SS (2007) Genomic analysis reveals widespread occurrence of new classes of copper nitrite reductases. J Biol Inorg Chem.   In press.  <ul class="find" data-citedArticleID="1036266" data-doi="10.1007/s00775-007-0282-2"><li><a href="http://dx.doi.org/10.1007/s00775-007-0282-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+analysis+reveals+widespread+occurrence+of+new+classes+of+copper+nitrite+reductases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+analysis+reveals+widespread+occurrence+of+new+classes+of+copper+nitrite+reductases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001378-Wilmot1" id="pone.0001378-Wilmot1"></a>Wilmot CM (2003) Oxygen activation in a copper-containing amine oxidase. Biochem Soc Trans  31: 493–496.  <ul class="find" data-citedArticleID="1036318" data-doi="10.1042/bst0310493"><li><a href="http://dx.doi.org/10.1042/bst0310493" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oxygen+activation+in+a+copper-containing+amine+oxidase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oxygen+activation+in+a+copper-containing+amine+oxidase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001378-Chan1" id="pone.0001378-Chan1"></a>Chan SI, Chen KH-C, Yu SS-F, Chen C-L, Kuo SS-J (2004) Toward delineating the structure and function of the particulate methane monooxygenase from methanotropic bacteria. Biochemistry  43: 4421–4430.  <ul class="find" data-citedArticleID="1036254" data-doi="10.1021/bi0497603"><li><a href="http://dx.doi.org/10.1021/bi0497603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+delineating+the+structure+and+function+of+the+particulate+methane+monooxygenase+from+methanotropic+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+delineating+the+structure+and+function+of+the+particulate+methane+monooxygenase+from+methanotropic+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001378-Hullo1" id="pone.0001378-Hullo1"></a>Hullo MF, Moszer I, Danchin A, Martin-Verstraete I (2001) CotA of <em>Bacillus subtilis</em> is a copper-dependent laccase. J Bacteriol  183: 5426–5430.  <ul class="find" data-citedArticleID="1036278" data-doi="10.1128/jb.183.18.5426-5430.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.18.5426-5430.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CotA+of+Bacillus+subtilis+is+a+copper-dependent+laccase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CotA+of+Bacillus+subtilis+is+a+copper-dependent+laccase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001378-Berks1" id="pone.0001378-Berks1"></a>Berks B, Sargent F, Palmer T (2000) The tat protein export pathway. Molecular Microbiology  35: 260–274.  <ul class="find" data-citedArticleID="1036246" data-doi="10.1046/j.1365-2958.2000.01719.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01719.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+tat+protein+export+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+tat+protein+export+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001378-Rensing1" id="pone.0001378-Rensing1"></a>Rensing C, Grass G (2003) <em>Escherichia coli</em> mechanisms of copper homeostasis in a changing environment. FEMS Microbiol Rev  27: 197–213.  <ul class="find" data-citedArticleID="1036288" data-doi="10.1016/s0168-6445(03)00049-4"><li><a href="http://dx.doi.org/10.1016/s0168-6445(03)00049-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Escherichia+coli+mechanisms+of+copper+homeostasis+in+a+changing+environment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Escherichia+coli+mechanisms+of+copper+homeostasis+in+a+changing+environment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001378-Gupta1" id="pone.0001378-Gupta1"></a>Gupta SD, Lee BT, Camakaris J, Wu HC (1995) Identification of cutC and cutF (nlpE) genes involved in copper tolerance in <em>Escherichia Coli</em>. J Bacteriol  177: 4207–4215.  <ul class="find" data-citedArticleID="1036276"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+cutC+and+cutF+%28nlpE%29+genes+involved+in+copper+tolerance+in+Escherichia+Coli.&amp;auth=&amp;atitle=Identification+of+cutC+and+cutF+%28nlpE%29+genes+involved+in+copper+tolerance+in+Escherichia+Coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+cutC+and+cutF+%28nlpE%29+genes+involved+in+copper+tolerance+in+Escherichia+Coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+cutC+and+cutF+%28nlpE%29+genes+involved+in+copper+tolerance+in+Escherichia+Coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001378-Grass1" id="pone.0001378-Grass1"></a>Grass G, Rensing C (2001) Genes involved in copper homeostasis in <em>Escherichia coli</em>. J Bacteriol  183: 2145–2147.  <ul class="find" data-citedArticleID="1036274" data-doi="10.1128/jb.183.6.2145-2147.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.6.2145-2147.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+involved+in+copper+homeostasis+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+involved+in+copper+homeostasis+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001378-Arnesano1" id="pone.0001378-Arnesano1"></a>Arnesano F, Banci L, Bertini I, Mangani S, Thompsett AR (2003) A redox switch in CopC: an intriguing copper trafficking protein than binds copper(I) and copper(II) at different sites. Proceedings of the National Academy of the Sciences  100: 3814–3819.  <ul class="find" data-citedArticleID="1036240" data-doi="10.1073/pnas.0636904100"><li><a href="http://dx.doi.org/10.1073/pnas.0636904100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+redox+switch+in+CopC%3A+an+intriguing+copper+trafficking+protein+than+binds+copper%28I%29+and+copper%28II%29+at+different+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+redox+switch+in+CopC%3A+an+intriguing+copper+trafficking+protein+than+binds+copper%28I%29+and+copper%28II%29+at+different+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001378-Cha1" id="pone.0001378-Cha1"></a>Cha J, Cooksey DA (1993) Copper hypersensitivity and uptake in <em>Pseudomonas syringae</em> containing cloned components of the copper resistance operon. Appl Environ Microbiol  59: 1671–1674.  <ul class="find" data-citedArticleID="1036252"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Copper+hypersensitivity+and+uptake+in+Pseudomonas+syringae+containing+cloned+components+of+the+copper+resistance+operon.&amp;auth=&amp;atitle=Copper+hypersensitivity+and+uptake+in+Pseudomonas+syringae+containing+cloned+components+of+the+copper+resistance+operon." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Copper+hypersensitivity+and+uptake+in+Pseudomonas+syringae+containing+cloned+components+of+the+copper+resistance+operon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Copper+hypersensitivity+and+uptake+in+Pseudomonas+syringae+containing+cloned+components+of+the+copper+resistance+operon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001378-Ciccarelli1" id="pone.0001378-Ciccarelli1"></a>Ciccarelli FD, Doerks T, von Mering C, Creevey CJ, Snel B, et al.  (2006) Toward automatic reconstruction of a highly resolved tree of life. Science  311: 1283–1287.  <ul class="find" data-citedArticleID="1036258" data-doi="10.1126/science.1123061"><li><a href="http://dx.doi.org/10.1126/science.1123061" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+automatic+reconstruction+of+a+highly+resolved+tree+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+automatic+reconstruction+of+a+highly+resolved+tree+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001378-Bekker1" id="pone.0001378-Bekker1"></a>Bekker A, Holland HD, Wang P-L, Rumble D III, Stein HJ, et al.  (2004) Dating the rise of atmospheric oxygen. Nature  427: 117–120.  <ul class="find" data-citedArticleID="1036244" data-doi="10.1038/nature02260"><li><a href="http://dx.doi.org/10.1038/nature02260" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dating+the+rise+of+atmospheric+oxygen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dating+the+rise+of+atmospheric+oxygen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001378-Decker1" id="pone.0001378-Decker1"></a>Decker H, Terwilliger N (2000) Cops and robbers: putative evolution of copper oxygen-binding proteins. The Journal of Experimental Biology  203: 1777–1782.  <ul class="find" data-citedArticleID="1036264"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cops+and+robbers%3A+putative+evolution+of+copper+oxygen-binding+proteins.&amp;auth=&amp;atitle=Cops+and+robbers%3A+putative+evolution+of+copper+oxygen-binding+proteins." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cops+and+robbers%3A+putative+evolution+of+copper+oxygen-binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cops+and+robbers%3A+putative+evolution+of+copper+oxygen-binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001378-Saito2" id="pone.0001378-Saito2"></a>Saito MA, Signman DM, Morel FMM (2003) The bioinorganic chemistry of the ancient ocean: the co-evolution of cyanobacterial metal requirements and biogeochemical cycles at the Archean-Proterozic boundary? Inorganica Chimica Acta  356: 308–318.  <ul class="find" data-citedArticleID="1036298" data-doi="10.1016/s0020-1693(03)00442-0"><li><a href="http://dx.doi.org/10.1016/s0020-1693(03)00442-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+bioinorganic+chemistry+of+the+ancient+ocean%3A+the+co-evolution+of+cyanobacterial+metal+requirements+and+biogeochemical+cycles+at+the+Archean-Proterozic+boundary%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+bioinorganic+chemistry+of+the+ancient+ocean%3A+the+co-evolution+of+cyanobacterial+metal+requirements+and+biogeochemical+cycles+at+the+Archean-Proterozic+boundary%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001378-Ochiai1" id="pone.0001378-Ochiai1"></a>Ochiai E (1983) Copper and the biological evolution. BioSystems  16: 81–86.  <ul class="find" data-citedArticleID="1036284" data-doi="10.1016/0303-2647(83)90029-1"><li><a href="http://dx.doi.org/10.1016/0303-2647(83)90029-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Copper+and+the+biological+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Copper+and+the+biological+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001378-Janulczyk1" id="pone.0001378-Janulczyk1"></a>Janulczyk R, Pallon J, Bjorck L (1999) Identification and characterization of a <em>Streptococcus pyogenes</em> ABC transporter with multiple specificity for metal cations. Molecular Microbiology  34: 596–606.  <ul class="find" data-citedArticleID="1036280" data-doi="10.1046/j.1365-2958.1999.01626.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01626.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+a+Streptococcus+pyogenes+ABC+transporter+with+multiple+specificity+for+metal+cations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+a+Streptococcus+pyogenes+ABC+transporter+with+multiple+specificity+for+metal+cations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001378-Culotta1" id="pone.0001378-Culotta1"></a>Culotta VC, Yang M, O'Halloran TV (2006) Activation of superoxide dismutases: putting the metal to the petal. Biochimica et Biophysica Acta  1763: 747–758.  <ul class="find" data-citedArticleID="1036262" data-doi="10.1016/j.bbamcr.2006.05.003"><li><a href="http://dx.doi.org/10.1016/j.bbamcr.2006.05.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+of+superoxide+dismutases%3A+putting+the+metal+to+the+petal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+of+superoxide+dismutases%3A+putting+the+metal+to+the+petal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001378-Zhang2" id="pone.0001378-Zhang2"></a>Zhang Y, Gladyshev V N (2007) High content of proteins containing 21st and 22nd amino acids, selenocysteine and pyrrolysine, in a symbiotic deltaproteobacterium of gutless worm Olavius algarvensis. Nucleic Acids Res.   In Press.  <ul class="find" data-citedArticleID="1036324" data-doi="10.1093/nar/gkm514"><li><a href="http://dx.doi.org/10.1093/nar/gkm514" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+content+of+proteins+containing+21st+and+22nd+amino+acids%2C+selenocysteine+and+pyrrolysine%2C+in+a+symbiotic+deltaproteobacterium+of+gutless+worm+Olavius+algarvensis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+content+of+proteins+containing+21st+and+22nd+amino+acids%2C+selenocysteine+and+pyrrolysine%2C+in+a+symbiotic+deltaproteobacterium+of+gutless+worm+Olavius+algarvensis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001378-Altschul1" id="pone.0001378-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="1036234" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001378-Chenna1" id="pone.0001378-Chenna1"></a>Chenna R, Sugawara H, Koike T, Lopez R, Gibson TJ, et al.  (2003) Multiple sequence alignment with the Clustal series of programs. Nucleic Acids Res  31: 3497–3500.  <ul class="find" data-citedArticleID="1036256" data-doi="10.1093/nar/gkg500"><li><a href="http://dx.doi.org/10.1093/nar/gkg500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+sequence+alignment+with+the+Clustal+series+of+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+sequence+alignment+with+the+Clustal+series+of+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.XML" value="88144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.PDF" value="858408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.t001.PNG_L" value="168374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.t001.PNG_M" value="149484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.t001.PNG_S" value="19590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.t001.TIF" value="800862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.t001.PNG_I" value="98293"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g001.PNG_L" value="85689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g001.PNG_M" value="207280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g001.PNG_S" value="13054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g001.TIF" value="611214"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g001.PNG_I" value="76032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g002.PNG_L" value="73074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g002.PNG_M" value="93864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g002.PNG_S" value="8694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g002.TIF" value="480000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g002.PNG_I" value="34965"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g003.PNG_L" value="160021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g003.PNG_M" value="212426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g003.PNG_S" value="11091"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g003.TIF" value="1072696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g003.PNG_I" value="78274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g004.PNG_L" value="119802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g004.PNG_M" value="175560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g004.PNG_S" value="17626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g004.TIF" value="865664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g004.PNG_I" value="101397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g005.PNG_L" value="46447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g005.PNG_M" value="128260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g005.PNG_S" value="12630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g005.TIF" value="331952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g005.PNG_I" value="46756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g006.PNG_L" value="37572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g006.PNG_M" value="38541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g006.PNG_S" value="6912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g006.TIF" value="491574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g006.PNG_I" value="13951"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g007.PNG_L" value="28615"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g007.PNG_M" value="26328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g007.PNG_S" value="6177"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g007.TIF" value="464910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.g007.PNG_I" value="9547"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001378.s001.XLS" value="180224"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001378&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001378" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001378&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001378&volume=&issue=&title=Comparative Genomic Analyses of Copper Transporters and Cuproproteomes Reveal Evolutionary Dynamics of Copper Utilization and Its Link to Oxygen&author_name=Perry%20G.%20Ridge%2C%20Yan%20Zhang%2C%20Vadim%20N.%20Gladyshev&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001378" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001378" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001378" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001378&amp;t=Comparative%20Genomic%20Analyses%20of%20Copper%20Transporters%20and%20Cuproproteomes%20Reveal%20Evolutionary%20Dynamics%20of%20Copper%20Utilization%20and%20Its%20Link%20to%20Oxygen" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001378&title=Comparative%20Genomic%20Analyses%20of%20Copper%20Transporters%20and%20Cuproproteomes%20Reveal%20Evolutionary%20Dynamics%20of%20Copper%20Utilization%20and%20Its%20Link%20to%20Oxygen&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001378&amp;title=Comparative%20Genomic%20Analyses%20of%20Copper%20Transporters%20and%20Cuproproteomes%20Reveal%20Evolutionary%20Dynamics%20of%20Copper%20Utilization%20and%20Its%20Link%20to%20Oxygen" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001378" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001378&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Comparative Genomic Analyses of Copper Transporters and Cuproproteomes Reveal Evolutionary Dynamics of Copper Utilization and Its Link to Oxygen';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001378';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Comparative%20Genomic%20Analyses%20of%20Copper%20Transporters%20and%20Cuproproteomes%20Reveal%20Evolutionary%20Dynamics%20of%20Copper%20Utilization%20and%20Its%20Link%20to%20Oxygen http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001378" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001378" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Anaerobic+bacteria%22" title="Search for articles in the subject area:'Anaerobic bacteria'"><div class="flagText">Anaerobic bacteria</div></a>
              <div data-categoryid="43751" data-articleid="25586"
                   data-categoryname="Anaerobic bacteria"
                   class="flagImage" title="Flag 'Anaerobic bacteria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Archaean+biology%22" title="Search for articles in the subject area:'Archaean biology'"><div class="flagText">Archaean biology</div></a>
              <div data-categoryid="43097" data-articleid="25586"
                   data-categoryname="Archaean biology"
                   class="flagImage" title="Flag 'Archaean biology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="32801" data-articleid="25586"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Copper%22" title="Search for articles in the subject area:'Copper'"><div class="flagText">Copper</div></a>
              <div data-categoryid="22203" data-articleid="25586"
                   data-categoryname="Copper"
                   class="flagImage" title="Flag 'Copper' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="25586"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Organismal+evolution%22" title="Search for articles in the subject area:'Organismal evolution'"><div class="flagText">Organismal evolution</div></a>
              <div data-categoryid="17201" data-articleid="25586"
                   data-categoryname="Organismal evolution"
                   class="flagImage" title="Flag 'Organismal evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Prokaryotic+cells%22" title="Search for articles in the subject area:'Prokaryotic cells'"><div class="flagText">Prokaryotic cells</div></a>
              <div data-categoryid="17761" data-articleid="25586"
                   data-categoryname="Prokaryotic cells"
                   class="flagImage" title="Flag 'Prokaryotic cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Trace+elements%22" title="Search for articles in the subject area:'Trace elements'"><div class="flagText">Trace elements</div></a>
              <div data-categoryid="21605" data-articleid="25586"
                   data-categoryname="Trace elements"
                   class="flagImage" title="Flag 'Trace elements' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2699'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6396'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=972&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
