

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Transposon Excision from an Atypical Site: A Mechanism of Evolution of Novel Transposable Elements</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000965"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000965" />

    <meta name="citation_title" content="Transposon Excision from an Atypical Site: A Mechanism of Evolution of Novel Transposable Elements"/>
    <meta itemprop="name" content="Transposon Excision from an Atypical Site: A Mechanism of Evolution of Novel Transposable Elements"/>

      <meta name="citation_author" content="Marybeth Langer"/>
            <meta name="citation_author_institution" content="Department of Biology, University of Rochester, Rochester, New York, United States of America"/>
      <meta name="citation_author" content="Lynn F. Sniderhan"/>
            <meta name="citation_author_institution" content="Department of Biology, University of Rochester, Rochester, New York, United States of America"/>
      <meta name="citation_author" content="Ueli Grossniklaus"/>
            <meta name="citation_author_institution" content="Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, United States of America"/>
      <meta name="citation_author" content="Animesh Ray"/>
            <meta name="citation_author_institution" content="Department of Biology, University of Rochester, Rochester, New York, United States of America"/>
            <meta name="citation_author_institution" content="Keck Graduate Institute, Claremont, California, United States of America"/>

    <meta name="citation_date" content="2007/10/3"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000965.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e965"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Mobile elements: drivers of genome evolution.; citation_author=HH Kazazian; citation_journal_title=Science; citation_volume=303; citation_number=1; citation_pages=1626-1632; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Regulation of activator/dissociation transposition by replication and DNA methylation.; citation_author=F Ros; citation_author=R Kunze; citation_journal_title=Genetics; citation_volume=157; citation_number=2; citation_pages=1723-1733; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of non-random DNA repair following Ac transposition excision in maize and Arabidopsis.; citation_author=TA Rinehart; citation_author=C Dean; citation_author=CF Weil; citation_journal_title=Plant J; citation_volume=12; citation_number=3; citation_pages=1419-1427; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Transposition of maize Ac/Ds transposable elements in the yeast Saccharomyces cerevisiae.; citation_author=CF Weil; citation_author=R Kunze; citation_journal_title=Nat Genet; citation_volume=26; citation_number=4; citation_pages=187-190; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Ac-induced disruption of the double Ds structure in tomato.; citation_author=CM Rommens; citation_author=EA van der Biezen; citation_author=PB Ouwerkerk; citation_author=HJ Nijkamp; citation_author=J Hille; citation_journal_title=Mol Gen Genet; citation_volume=228; citation_number=5; citation_pages=453-458; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Identification and isolation of the FEEBLY gene from tomato by transposon tagging.; citation_author=EA Van der Biezen; citation_author=BF Brandwagt; citation_author=W van Leeuwen; citation_author=HJ Nijkamp; citation_author=J Hille; citation_journal_title=Mol Gen Genet.; citation_volume=251; citation_number=6; citation_pages=267-280; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Structural dynamics of eukaryotic chromosome evolution.; citation_author=EE Eichler; citation_author=D Sankoff; citation_journal_title=Science; citation_volume=301; citation_number=7; citation_pages=793-797; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mutations in maize.; citation_author=B McClintock; citation_journal_title=Carnegie Inst Wash Ybk; citation_volume=52; citation_number=8; citation_pages=227-237; citation_date=1953; " />
      <meta name="citation_reference" content="citation_title=Crossing over in maize in the presence of the transposable factors Activator and Modulator.; citation_author=CMW Fradkin; citation_author=RA Brink; citation_journal_title=Genetics; citation_volume=41; citation_number=9; citation_pages=901-906; citation_date=1956; " />
      <meta name="citation_reference" content="citation_title=Continuous exchange of sequence information between dispersed Tc1 transposons in the Caenorhabditis elegans genome.; citation_author=SE Fischer; citation_author=E Wienholds; citation_author=RH Plasterk; citation_journal_title=Genetics; citation_volume=164; citation_number=10; citation_pages=127-134; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Ac insertion site affects the frequency of transposon-induced homologous recombination at the maize p1 locus.; citation_author=YL Xiao; citation_author=X Li; citation_author=T Peterson; citation_journal_title=Genetics; citation_volume=156; citation_number=11; citation_pages=2007-2017; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Effects of recombination rate and gene density on transposable element distributions in Arabidopsis thaliana.; citation_author=SI Wright; citation_author=N Agrawal; citation_author=TE Bureau; citation_journal_title=Genome Res; citation_volume=13; citation_number=12; citation_pages=1897-1903; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Recombination rates between adjacent genic and retrotransposon regions in maize vary by 2 orders of magnitude.; citation_author=H Fu; citation_author=Z Zheng; citation_author=HK Dooner; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=13; citation_pages=1082-1087; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization of meiotic recombination across the 140-kb multigenic a1-sh2 interval of maize.; citation_author=H Yao; citation_author=Q Zhou; citation_author=J Li; citation_author=H Smith; citation_author=M Yandeau; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=14; citation_pages=5763-5765; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Transposable elements generate novel spatial patterns of gene expression in Antirrhinum majus.; citation_author=ES Coen; citation_author=R Carpenter; citation_author=C Martin; citation_journal_title=Cell; citation_volume=47; citation_number=15; citation_pages=285-96; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Generating novel allelic variation through Activator insertional mutagenesis in maize.; citation_author=L Bai; citation_author=M Singh; citation_author=L Pitt; citation_author=M Sweeney; citation_author=TP Brutnell; citation_journal_title=Genetics; citation_volume=175; citation_number=16; citation_pages=981-992; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Characterization of broken DNA molecules associated with V(D)J recombination.; citation_author=DB Roth; citation_author=C Zhu; citation_author=M Gellert; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=90; citation_number=17; citation_pages=10788-10792; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Cleavage at a V(D)J. recombination signal requires only RAG1 and RAG2 proteins and occurs in two steps.; citation_author=F McBlane; citation_author=D van Gent; citation_author=D Ramsden; citation_author=C Romeo; citation_author=C Cuomo; citation_journal_title=Cell; citation_volume=83; citation_number=18; citation_pages=387-395; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Patterns of gene action in plant development revealed by enhancer trap and gene trap transposable elements.; citation_author=V Sundaresan; citation_author=P Springer; citation_author=T Volpe; citation_author=S Haward; citation_author=JD Jones; citation_journal_title=Genes Dev; citation_volume=9; citation_number=19; citation_pages=1797-1810; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Transposition of hAT elements links transposable elements and V(D)J recombination.; citation_author=L Zhou; citation_author=R Mitra; citation_author=PW Atkinson; citation_author=AB Hickman; citation_author=F Dyda; citation_journal_title=Nature; citation_volume=432; citation_number=20; citation_pages=995-1001; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Microhomology-dependent end joining and repair of transposon-induced DNA hairpins by host factors in Saccharomyces cerevisiae.; citation_author=J Yu; citation_author=K Marshall; citation_author=M Yamaguchi; citation_author=JE Haber; citation_author=CF Weil; citation_journal_title=Mol Cell Biol; citation_volume=24; citation_number=21; citation_pages=1351-1364; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Adjacent sequences influence DNA repair accompanying transposon excision in maize.; citation_author=L Scott; citation_author=D LaFoe; citation_author=CF Weil; citation_journal_title=Genetics; citation_volume=142; citation_number=22; citation_pages=237-246; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Analysis of extrachromosomal Ac/Ds transposable elements.; citation_author=V Gorbunova; citation_author=AA Levy; citation_journal_title=Genetics; citation_volume=155; citation_number=23; citation_pages=349-359; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Ac transposition is impaired by a small terminal deletion.; citation_author=YL Xiao; citation_author=T Peterson; citation_journal_title=Mol Genet Genomics; citation_volume=266; citation_number=24; citation_pages=720-731; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Ac-immobilized, a stable source of Activator transposase that mediates sporophytic and gametophytic excision of Dissociation elements in maize.; citation_author=LJ Conrad; citation_author=TP Brutnell; citation_journal_title=Genetics; citation_volume=171; citation_number=25; citation_pages=1999-2012; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The double-strand-break repair model for recombination.; citation_author=JW Szostak; citation_author=TL Orr-Weaver; citation_author=RJ Rothstein; citation_author=FW Stahl; citation_journal_title=Cell; citation_volume=33; citation_number=26; citation_pages=25-35; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Germinal excisions of the maize transposon activator do not stimulate meiotic recombination or homology-dependent repair at the bz locus.; citation_author=HK Dooner; citation_author=IM Martínez-Férez; citation_journal_title=Genetics; citation_volume=147; citation_number=27; citation_pages=1923-1932; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Recombination occurs uniformly within the bronze gene, a meiotic recombination hotspot in the maize genome.; citation_author=HK Dooner; citation_author=IM Martínez-Férez; citation_journal_title=Plant Cell; citation_volume=9; citation_number=28; citation_pages=1633-1646; citation_date=1997a; " />
      <meta name="citation_reference" content="citation_title=Intrachromosomal excision of a hybrid Ds element induces large genomic deletions in Arabidopsis.; citation_author=DR Page; citation_author=C Koehler; citation_author=JA da Costa Nunes; citation_author=C Baroux; citation_author=JM Moore; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=101; citation_number=29; citation_pages=2969-2969; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Maize Activator transposase has a bipartite DNA binding domain that recognizes subterminal sequences and terminal inverted repeats.; citation_author=HA Becker; citation_author=R Kunze; citation_journal_title=Mol Gen Genet; citation_volume=254; citation_number=30; citation_pages=219-230; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Transposition in plants: a molecular model.; citation_author=H Saedler; citation_author=P Nevers; citation_journal_title=EMBO J; citation_volume=4; citation_number=31; citation_pages=585-590; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Origination of Ds elements from Ac elements in maize: evidence for rare repair synthesis at the site of Ac excision.; citation_author=X Yan; citation_author=IM Martínez-Férez; citation_author=S Kavchok; citation_author=HK Dooner; citation_journal_title=Genetics; citation_volume=152; citation_number=32; citation_pages=1733-1740; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=High-frequency P element loss in Drosophila is homolog dependent.; citation_author=WR Engels; citation_author=DM Johnson-Schlitz; citation_author=WB Eggleston; citation_author=J Sved; citation_journal_title=Cell; citation_volume=62; citation_number=33; citation_pages=515-523; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Targeted gene replacement in Drosophila via P element-induced gap repair.; citation_author=GB Gloor; citation_author=NA Nassif; citation_author=DM Johnson-Schlitz; citation_author=CR Preston; citation_author=WR Engels; citation_journal_title=Science; citation_volume=253; citation_number=34; citation_pages=1110-1117; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A versatile system for detecting transposition in Arabidopsis.; citation_author=NV Fedoroff; citation_author=DL Smith; citation_journal_title=Plant J; citation_volume=3; citation_number=35; citation_pages=273-289; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=SHORT INTEGUMENTS1 / SUSPENSOR1 / CARPEL FACTORY, a Dicer homologue, is a maternal-effect gene required for embryo development in Arabidopsis.; citation_author=T Golden; citation_author=SE Schauer; citation_author=JD Lang; citation_author=S Pien; citation_author=A Mushegian; citation_journal_title=Plant Physiol; citation_volume=130; citation_number=36; citation_pages=808-822; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=SHORT INTEGUMENTS1 (SIN1), a gene required for ovule development in Arabidopsis, also controls flowering time.; citation_author=A Ray; citation_author=JD Lang; citation_author=T Golden; citation_author=S Ray; citation_journal_title=Development; citation_volume=122; citation_number=37; citation_pages=2631-2638; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Efficient isolation and mapping of Arabidopsis thaliana T-DNA insert junction by thermal asymmetric interlaced PCR.; citation_author=YG Liu; citation_author=N Mitsukawa; citation_author=T Oosumi; citation_author=RA Whittier; citation_journal_title=Plant J; citation_volume=8; citation_number=38; citation_pages=457-463; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Maternal control of embryogenesis by MEDEA, a Polycomb group gene in Arabidopsis.; citation_author=U Grossniklaus; citation_author=JP Vielle-Calzada; citation_author=MA Hoeppner; citation_author=WB Gagliano; citation_journal_title=Science; citation_volume=280; citation_number=39; citation_pages=446-450; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000965" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Transposon Excision from an Atypical Site: A Mechanism of Evolution of Novel Transposable Elements"/>
    <meta name="twitter:description" content="The role of transposable elements in sculpting the genome is well appreciated but remains poorly understood. Some organisms, such as humans, do not have active transposons; however, transposable elements were presumably active in their ancestral genomes. Of specific interest is whether the DNA surrounding the sites of transposon excision become recombinogenic, thus bringing about homologous recombination. Previous studies in maize and Drosophila have provided conflicting evidence on whether transposon excision is correlated with homologous recombination. Here we take advantage of an atypical Dissociation (Ds) element, a maize transposon that can be mobilized by the Ac transposase gene in Arabidopsis thaliana, to address questions on the mechanism of Ds excision. This atypical Ds element contains an adjacent 598 base pairs (bp) inverted repeat; the element was allowed to excise by the introduction of an unlinked Ac transposase source through mating. Footprints at the excision site suggest a micro-homology mediated non-homologous end joining reminiscent of V(D)J recombination involving the formation of intra-helix 3&prime; to 5&prime; trans-esterification as an intermediate, a mechanism consistent with previous observations in maize, Antirrhinum and in certain insects. The proposed mechanism suggests that the broken chromosome at the excision site should not allow recombinational interaction with the homologous chromosome, and that the linked inverted repeat should also be mobilizable. To test the first prediction, we measured recombination of flanking chromosomal arms selected for the excision of Ds. In congruence with the model, Ds excision did not influence crossover recombination. Furthermore, evidence for correlated movement of the adjacent inverted repeat sequence is presented; its origin and movement suggest a novel mechanism for the evolution of repeated elements. Taken together these results suggest that the movement of transposable elements themselves may not directly influence linkage. Possibility remains, however, for novel repeated DNA sequences produced as a consequence of transposon movement to influence crossover in subsequent generations."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000965.g005"/>

  <meta property="og:title" content="Transposon Excision from an Atypical Site: A Mechanism of Evolution of Novel Transposable Elements" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000965" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9030'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2946'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8252&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000965">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000965" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000965&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Transposon Excision from an Atypical Site: A Mechanism of Evolution of Novel Transposable Elements
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marybeth Langer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, University of Rochester, Rochester, New York, United States of America
                </p>

                <p>
                    Current address: Immunobiology and Cancer Program, Oklahoma Medical Research Foundation, Oklahoma City, Oklahoma, United States of America,
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lynn F. Sniderhan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, University of Rochester, Rochester, New York, United States of America
                </p>

                <p>
                    Current address: Department of Microbiology and Immunology, University of Rochester School of Medicine and Dentistry, Rochester, New York, United States of America,
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ueli Grossniklaus, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, United States of America
                </p>

                <p>
                    Current address: Institute of Plant Biology and Zürich-Basel Plant Science Center, University of Zürich, Zürich, Switzerland
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Animesh Ray
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:aray@kgi.edu">aray@kgi.edu</a></p>

                <p>Affiliations:
                  Department of Biology, University of Rochester, Rochester, New York, United States of America, 
                  Keck Graduate Institute, Claremont, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 03, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000965</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000965" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000965" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000965" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000965" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000965-g001" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000965-g002" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000965-g003" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000965-t001" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000965-g004" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000965-t002" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000965-g005" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000965">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000965" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The role of transposable elements in sculpting the genome is well appreciated but remains poorly understood. Some organisms, such as humans, do not have active transposons; however, transposable elements were presumably active in their ancestral genomes. Of specific interest is whether the DNA surrounding the sites of transposon excision become recombinogenic, thus bringing about homologous recombination. Previous studies in maize and Drosophila have provided conflicting evidence on whether transposon excision is correlated with homologous recombination. Here we take advantage of an atypical <em>Dissociation</em> (<em>Ds</em>) element, a maize transposon that can be mobilized by the <em>Ac</em> transposase gene in <em>Arabidopsis thaliana</em>, to address questions on the mechanism of <em>Ds</em> excision. This atypical <em>Ds</em> element contains an adjacent 598 base pairs (bp) inverted repeat; the element was allowed to excise by the introduction of an unlinked <em>Ac</em> transposase source through mating. Footprints at the excision site suggest a micro-homology mediated non-homologous end joining reminiscent of V(D)J recombination involving the formation of intra-helix 3′ to 5′ trans-esterification as an intermediate, a mechanism consistent with previous observations in maize, Antirrhinum and in certain insects. The proposed mechanism suggests that the broken chromosome at the excision site should not allow recombinational interaction with the homologous chromosome, and that the linked inverted repeat should also be mobilizable. To test the first prediction, we measured recombination of flanking chromosomal arms selected for the excision of <em>Ds</em>. In congruence with the model, <em>Ds</em> excision did not influence crossover recombination. Furthermore, evidence for correlated movement of the adjacent inverted repeat sequence is presented; its origin and movement suggest a novel mechanism for the evolution of repeated elements. Taken together these results suggest that the movement of transposable elements themselves may not directly influence linkage. Possibility remains, however, for novel repeated DNA sequences produced as a consequence of transposon movement to influence crossover in subsequent generations.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Langer M, Sniderhan LF, Grossniklaus U, Ray A (2007) Transposon Excision from an Atypical Site: A Mechanism of Evolution of Novel Transposable Elements. PLoS ONE 2(10):
          e965.
            doi:10.1371/journal.pone.0000965</p><p><strong>Academic Editor: </strong>Brian Dilkes, University of California at Davis, United States of America</p><p><strong>Received:</strong> January 31, 2007; <strong>Accepted:</strong> July 26, 2007; <strong>Published:</strong> October 3, 2007</p><p><strong>Copyright:</strong> © 2007 Langer et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Work in AR's laboratory was supported by National Science Foundation grants IBN-9982414, EIA-0130059, and EIA-0205061 to AR. Work in UG's laboratory was supported by the Cold Spring Harbor Laboratory President's Council and the European Molecular Biology Organization grants to UG.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Mobile genetic elements are important forces of evolution <a href="#pone.0000965-Kazazian1">[1]</a>. <em>Ac/Ds</em> elements are DNA transposons that use a conservative “cut-and-paste” mechanism involving double-stranded DNA breaks to move within the genome. Repair of broken chromosome ends at the donor (excision) site contributes to genetic alterations in adjoining sequences <a href="#pone.0000965-Ros1">[2]</a>–<a href="#pone.0000965-Weil1">[4]</a>. Footprints or DNA sequence rearrangements left at the donor site following transposition provide clues to the repair mechanisms <a href="#pone.0000965-Rinehart1">[3]</a>, <a href="#pone.0000965-Rommens1">[5]</a>, <a href="#pone.0000965-VanderBiezen1">[6]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The exact mechanism of donor site repair remains unresolved; indeed, it appears likely that this phenomenon may be affected by host proteins and hence might vary among species <a href="#pone.0000965-Rinehart1">[3]</a>. In particular the effect of <em>Ac/Ds</em> transposition on chromosome crossover, a major mechanism behind genetic diversity <a href="#pone.0000965-Eichler1">[7]</a>, is an unanswered question. Linkage disequilibrium, the historical evidence of rare crossover exchanges between close markers, may potentially have its origin, among other factors, in rare crossovers stimulated by the activity of mobile elements. Early studies in maize of <em>Activator/Dissociation</em> (<em>Ac/Ds</em>) elements had failed to find a clear association between crossing over and transposition <a href="#pone.0000965-McClintock1">[8]</a>, <a href="#pone.0000965-Fradkin1">[9]</a>. Although transposons are frequently involved in gene conversion <a href="#pone.0000965-Fischer1">[10]</a> and stimulate crossover within repeated DNA <a href="#pone.0000965-Xiao1">[11]</a>, recent genome level analysis shows either no correlation <a href="#pone.0000965-Wright1">[12]</a> or negative correlation <a href="#pone.0000965-Fu1">[13]</a>, <a href="#pone.0000965-Yao1">[14]</a> between recombination rate and transposable element density along the chromosome. One may speculate that this may be the consequence of evolutionary selection to prevent crossover exchanges where transposon density is high. The question remains whether a pioneering transposable element in a chromosome free of prior copies of that transposon allows high frequency crossover during its movement.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>To address these questions, here we examined transposition of an active <em>Ds</em> element artificially introduced into <em>Arabidopsis thaliana.</em> The <em>Ds</em> element has an atypical insertion site, in which there is a long unrelated inverted repeat DNA sequence located immediately adjacent to, but outside of, one of the terminal inverted repeats, which is bordered on the outside by one of the duplicated target sequences. As in previous reports <a href="#pone.0000965-Rinehart1">[3]</a>, <a href="#pone.0000965-Coen1">[15]</a>, <a href="#pone.0000965-Bai1">[16]</a> we show that the excision repair footprints support a mechanism resembling immunoglobulin V(D)J class switching in vertebrates <a href="#pone.0000965-Roth1">[17]</a>, <a href="#pone.0000965-McBlane1">[18]</a>. In accordance with the proposed model of <em>Ds</em> transposition, we show that transposition does not influence crossover recombination between homologous chromosomes. This result suggests that the movement of transposable elements may not be an important factor in genome rearrangement by recombination. Furthermore, as predicted by the model, we detected movement of the unrelated DNA inverted repeat from its initial position to secondary locations. These observations, along with an analysis of the footprints of transposition, provoke a new model of DNA repair at the excision site that allows the formation of new inverted repeat sequences adjacent to the transposable element. We speculate that such inverted repeats may subsequently separate from the original transposon and form novel mobile sequences.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Origin of a DNA inverted repeat sequence associated with <em>Ds</em></h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The <em>Ds</em> site from which excision was allowed has an atypical configuration at the 3′ end: the 3′ terminal inverted repeat (3′ TIR) of the <em>Ds</em> end is situated adjacent to another inverted repeat of 598 bp (<a href="#pone-0000965-g001">Figure 1A</a>). The sequence of this repeat was derived from the parental strain <a href="#pone.0000965-Sundaresan1">[19]</a> in which the maize <em>Ds</em> element, introduced into Arabidopsis on a T-DNA vector, had previously transposed into the current donor site. The presence of the 598 bp repeat suggests that this duplicated sequence transposed into this site from its original location on a T-DNA stably incorporated into the chromosome in its parent. (<a href="#pone-0000965-g001">Figure 1B</a>). Although the 5′ end of the <em>Ds</em> insertion has the typical half-site of a target duplication, the other target half-site is separated from the <em>Ds</em> TIR by the 598 bp inverted repeat sequence derived from the T-DNA vector. To test whether this modified <em>Ds</em> element can transpose, we mobilized the element by introducing an active <em>Ac</em> transposase gene as described below.</p>
<div class="figure" id="pone-0000965-g001"><div class="img"><a name="pone-0000965-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000965.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000965.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Structure and position of the <em>Ds</em> element in strain ET500.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) Sequence of the <em>Ds</em> element and adjacent sequences showing the important elements. The insert is flanked by 7 bp target site duplication (boxed). DNA outside the duplicated target sequences are identical to the surrounding plant DNA sequence. Sequence of the 598 bp inverted repeat is identical to a fragment of the T-DNA containing 3′-octopine synthase promoter (3′OCS) and the right border (RB) element. Black circle within the boxed sequence shows the site of sequence inversion. (B) Structure of the <em>Ds</em> element. The <em>Ds</em> element contains a promoter-less bacterial <em>GUS</em> gene just inside the leftward TIR, and a bacterial <em>NPTII</em> gene with its own promoter. A plant-specific promoter-enhancer element (stippled oval) 5′ to the <em>Ds</em> insertion site confers constitutive <em>GUS</em> expression. (C) Map of the chromosomal region containing <em>Ds</em>. Numbers denote the first base on chromosome 4 (counting from the top) for the respective site. Roman numerals denote the two genetic intervals assayed for crossover. The transposase source, <em>Ac</em>, was unlinked to chromosome 4, and was present only in the experimental cross. Superscripts identify La-er (L) and No-0 (N) alleles.</p>
<span>doi:10.1371/journal.pone.0000965.g001</span></div>

<h4>Mobilization of the abnormal <em>Ds</em> element</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We crossed two strains of <em>Arabidopsis thaliana</em>, Landsberg <em>erecta</em> (La-er) and Nossen (No-0) such that the F1 provided heterozygosities at molecular markers for scoring recombinants (<a href="#pone-0000965-g001">Figure 1C</a>). The experimental cross brought an enhancer detector <em>Ds</em> element from one parent together with an unlinked and immobile <em>Ac</em> transposase gene from the other. The <em>Ds</em> contains a beta-glucuronidase (<em>GUS</em>) reporter gene and a neomycin phosphotransferase (<em>NPTII</em>) marker located within bounds of the two TIRs (<a href="#pone-0000965-g001">Figure 1B</a>), while <em>Ac</em> is adjacent to the negatively-selectable <em>TMS2</em> gene encoding an amide hydrolase that imparts sensitivity to naphthalene acetamide <a href="#pone.0000965-Sundaresan1">[19]</a>. The F1 of the corresponding control cross in the same genetic background carried the same <em>Ds</em> but not the <em>Ac</em>. The GUS reporter gene was under the control of an endogenous enhancer leading to constitutive expression such that production of GUS<sup>−</sup> somatic sectors could serve as the hallmark of <em>Ds</em> excision. The control cross did not show any evidence of somatic <em>Ds</em> excision (32 F1 plants tested) as indicated by a lack of GUS<sup>−</sup> leaf sectors (<a href="#pone-0000965-g002">Figure 2A</a>), nor of germinal excision as evidenced by transmission to the F2. By contrast, high frequency <em>Ds</em> excision occurred in 47 of 53 F1 plants of the experimental cross, as evidenced by GUS<sup>−</sup> sectors on F1 leaves (<a href="#pone-0000965-g002">Figure 2B</a>) and confirmed by polymerase chain reaction (PCR) and Southern analysis following transmission into the F2 progeny (<a href="#pone-0000965-g002">Figure 2C</a>). To prevent continued transposition during analysis, F2 plants were selected on naphthaline acetamide (NAM) to eliminate descendents inheriting <em>Ac</em>.</p>
<div class="figure" id="pone-0000965-g002"><div class="img"><a name="pone-0000965-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000965.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000965.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Excision of <em>Ds</em>.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) A leaf of a parental <em>Ds</em> strain homozygous for <em>Ds</em> (ET500), showing uniform GUS<sup>+</sup> staining. (B) GUS<sup>−</sup> somatic sectors due to <em>Ds</em> excision in F1 of ET500×Nae<em>Ac</em>. (C) Southern blot analysis showing the excision of <em>Ds</em>. Upper panel was probed with GUS; lower panel represents the same blot cut off between 2kb and 3kb markers, and probed with DCL1 cDNA sequences as a loading control <a href="#pone.0000965-Sundaresan1">[19]</a>. Lane 1, ET500 parental (before <em>Ds</em> excision). Lanes 2-3, DNA from two independent F2 derivatives of ET500×Nae<em>Ac</em> showing <em>Ds</em> excision, and lane 4, DNA from an F2 plant showing excision and reinsertion of <em>Ds</em> to another location. (D) Amplification of molecular markers: Landsberg <em>erecta</em> (L); Nossen (N); heterozygous (L/N) Left hand panel, nga8; middle panel, Ga1.1/BsaB1; right hand panel, nga1111.</p>
<span>doi:10.1371/journal.pone.0000965.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Depending on the timing and relative rates of <em>Ds</em> excision in the F1, meristem cells contributing to the germ line may contain numerous independent transposition events or one early event with numerous clonal descendents. Thus, a single F2 family from one flower could represent multiple independent excision events or one clonal event. A sampling of independent events could not be guaranteed by analyzing a number of F2's derived from a single F1 plant. To overcome this difficulty, from each F1 parent only one F2 progeny plant was scored for recombination. Each F2 plant was selected for analysis at random from among a pool of F2 plants that was proven to have an excision event by PCR analysis of the donor site and confirmed by Southern blots. Each separate pool was derived from a single F1 parent. This guaranteed that each F2 plant examined represented an independent <em>Ds</em> excision. Although it was not possible to formally distinguish between premeiotic, meiotic or postmeiotic transposition, most transposition probably occurred premeiotically because of the observed high rate of excision plants per F1 line. There was no attempt to determine whether crossover events were somatic or meiotic.</p>


<h4>Excision footprints suggest V(D)J end joining type of repair</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>DNA sequences at 11 of 17 independently derived empty <em>Ds</em> donor sites showed the same footprint and at three others a single base difference from the commonest footprint (<a href="#pone-0000965-t001">Table 1</a>). Sequences remaining at the excision sites were most consistent with the idea that excision frequently involved only one of the duplicated target sequences of the original <em>Ds</em> integration site. The footprints suggest a micro-homology mediated non-homologous end joining mechanism reminiscent of V(D)J recombination as suggested for Tam3 excision in <em>Antirrhinum</em> <a href="#pone.0000965-Coen1">[15]</a>, <em>Ds</em> excision in maize <a href="#pone.0000965-Rinehart1">[3]</a>, <a href="#pone.0000965-Bai1">[16]</a> and <em>Arabidopsis</em> <a href="#pone.0000965-Rinehart1">[3]</a>, and shown for <em>in vitro</em> transposition of the insect <em>hAT</em> element <em>Hermes</em> <a href="#pone.0000965-Zhou1">[20]</a> and <em>in vivo</em> excision of <em>Ds</em> in yeast <a href="#pone.0000965-Weil1">[4]</a>, <a href="#pone.0000965-Yu1">[21]</a>, involving pairs of staggered nicks, followed by DNA synthesis, hairpin formation by <em>trans</em>-esterification, hairpin nicking, imperfect base pairing, ligation, and mismatch correction (<a href="#pone-0000965-g003">Figure 3A</a>). Essentially similar models can explain two additional footprints (<a href="#pone-0000965-g003">Figure 3B, C</a>), and the remaining one involved a 41 base pairs deletion (<a href="#pone-0000965-g003">Figure 3D</a>). The exact site where the nick distal to the <em>Ds</em> element occurred cannot be ascertained from the footprint result; however, previous studies in maize, Arabidopsis and tobacco <a href="#pone.0000965-Rinehart1">[3]</a>, <a href="#pone.0000965-Weil1">[4]</a>, <a href="#pone.0000965-Coen1">[15]</a>, <a href="#pone.0000965-Bai1">[16]</a>, <a href="#pone.0000965-Scott1">[22]</a>, <a href="#pone.0000965-Gorbunova1">[23]</a> have determined that the footprints are most consistent with the transposase-mediated cleavage occurring at the 5′ nucleotide flanking the insertion. Our results are consistent with the previous models for 16 of 17 footprints (<a href="#pone-0000965-g003">Figure 3</a>). For the remaining footprint, involving a 41 bp deletion (<a href="#pone-0000965-g004">Fig 4D</a>), a mechanism involving an exonuclease-mediated degradation starting at a position 5′ of the Ds element is plausible, consistent with studies that show exonucleolytic deletion of sequences flanking the end of an element <a href="#pone.0000965-Xiao2">[24]</a>, <a href="#pone.0000965-Conrad1">[25]</a>. However, the model we have proposed here (<a href="#pone-0000965-g003">Fig. 3</a>) explains all footprints observed here, including footprints reminiscent of V(D)J recombination, a footprint involving the 41 bp deletion and the formation of a 598 bp inverted repeat adjacent to the 3′ TIR of the starting <em>Ds</em> used in this study.</p>
<div class="figure" id="pone-0000965-g003"><div class="img"><a name="pone-0000965-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000965.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000965.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Models explaining <em>Ds</em> excision footprints.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A–C) A model explaining the formation of the most common footprints. Boxed bases are the 7bp duplicated target sequence. Thin straight arrows show putative transposase nick sites, curved arrows depict <em>trans</em>-esterification reaction between the 3′ OH group of one strand and the 5′ phosphate of the other, and arrowheads show the points of ligation. Gap repair synthesis bases are in bold and direction of synthesis is shown by an arrow; deleted bases are underlined. Note the absence of symmetrical targets flanking the excision site. (D) Model explaining the formation of a less common footprint.</p>
<span>doi:10.1371/journal.pone.0000965.g003</span></div><div class="figure" id="pone-0000965-t001"><div class="img"><a name="pone-0000965-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000965.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000965.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Repair footprints following <em>Ds</em> excision</span></strong></p><span>doi:10.1371/journal.pone.0000965.t001</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The model suggests two independent predictions. First, the proposed mechanism of restoration of the broken chromosome does not involve any opportunity for homologous recombination initiated by the DNA double strand breaks because the broken ends are trans-esterified into hairpins, therefore, cannot initiate double strand break repair <a href="#pone.0000965-Szostak1">[26]</a> without further processing. The second more subtle prediction concerns the observed 41 bp ‘sliding’ of the transposase to produce the second nick (<a href="#pone-0000965-g003">Figure 3D</a>). It is conceivable that a slide by the transposase could make a double chain break at a site distant from the 3′TIR, upon which the new end could undergo hairpin formation, followed by a second nick adjacent to the 3′TIR (<a href="#pone-0000965-g004">Figure 4</a>). As a result, a new inverted repeat sequence could form, as explained in <a href="#pone-0000965-g004">Fig. 4</a>. While this prediction is actually prompted by the observation of the 3′ inverted repeat at the original <em>Ds</em> element site (<a href="#pone-0000965-g001">Figure 1A</a>), it also suggests that a reversal of the process may occasionally co-mobilize the inverted repeat sequence along with the <em>Ds</em> element. Tests of these predictions are as follows.</p>
<div class="figure" id="pone-0000965-g004"><div class="img"><a name="pone-0000965-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000965.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000965.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>A model proposing the formation and movement of novel inverted repeat elements.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>(A) TIRs of a transposable element are shown in filled boxes. The boxed sequences are the 7bp duplicated half-sites. Transposase molecules bind at the TIRs, and (B) make pairs of staggered nicks on each flank (triangles). Occasionally, the DNA may slip (C) or the transposase may bind to cryptic sites on adjacent host DNA, and make a paired nick, which leads to excision (D). (E) Short repair synthesis followed by trans-esterification leads to a hairpin at the host segment attached to the TIR, which is nicked again near the TIR. Helix unwinding, followed by microhomology-mediated ligation to a new host site inserts the transposon, and single strand gap filling by repair synthesis forms a novel inverted repeat derived from host sequences adjacent to the original location (F). The transposon may repeat the cycle of precise (B) or imprecise (C) excision, and may be dissociated from novel host-derived inverted repeat (G). If the host-derived sequence had a cryptic transposase-binding site, the site is now duplicated on the novel inverted repeat. This may lead to independent movement of the inverted repeat sequence as a novel mobile element so long as a transposase is available.</p>
<span>doi:10.1371/journal.pone.0000965.g004</span></div>

<h4>Flanking marker recombination with or without <em>Ds</em> excision</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Among F2 seedlings from the experimental cross <em>Ac</em> (No-0)×<em>Ds</em> (La-er), we selected progeny that had experienced a <em>Ds</em> excision in the following way. We required that each F2 plant lacked <em>Ac</em>, was uniformly GUS<sup>−</sup>, and had at least one La-er allele of nga8 (nga8<sup>L</sup>) tightly linked to <em>Ds</em> (<a href="#pone-0000965-g001">Figure 1C</a>). These plants contained at least one chromosome from which <em>Ds</em> had excised from the donor site. Single plants representing independent events from each of 47 F1 families from the experimental cross were identified, and analysed by PCR for crossover within Ga1.1—nga8—nga1111 intervals (<a href="#pone-0000965-g001">Figure 1C</a>). Of 47 independent excision lines, 8 had crossover recombination between nga1111 and Ga1.1 (<a href="#pone-0000965-t002">Table 2</a>). This recombination rate (8.5% per chromosome) is not significantly different from 10% recombination per chromosome in the control cross where no transposition was possible (29 crossovers among 144 F2 plants tested). Given 10% spontaneous crossover rate, 17 or more crossovers per 47 <em>Ds</em> excision events would have been scored as significantly correlated (χ<sup>2</sup> = 4.98; <em>P</em> ≤ 0.05, for 1 degree of freedom). Thus, the ability to detect an increase in crossover rate to 18% or higher argues that we would have detected a 50% crossover rate predicted by the double strand break-repair mechanism <a href="#pone.0000965-Szostak1">[26]</a> had the repair following <em>Ds</em> excision occurred by the strict version of this mechanism. Crossover frequency below 18% is presumably due to events unrelated to <em>Ds</em> excision.</p>
<div class="figure" id="pone-0000965-t002"><div class="img"><a name="pone-0000965-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000965.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000965.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span><em>Ds</em> excision does not influence crossover recombination</span></strong></p><span>doi:10.1371/journal.pone.0000965.t002</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>A critical condition of the above test was that selection on NAM did not influence segregation of the three markers flanking <em>Ds</em>, namely, nga8, Ga1.1 and nga1111. We tested this by crossing the <em>Ac</em> (No-0) parent to wild type La-er, and analyzing marker segregation in F2 lines that were either selected or not selected on NAM. For 26 F2 plants not selected on NAM, the number of plants segregating homozygous Nossen alleles (<em>NN</em>): heterozygous (<em>NL</em>): homozygous Landsberg alleles (<em>LL</em>) were 9:12:5 for nga8, 6:15:5 for Ga1.1, and 8:13:5 for nga1111, respectively (see <a href="#pone-0000965-g002">Figure 2D</a> for a typical result). For those selected on NAM, the corresponding numbers were 9:17:1 for nga8, 11:13:3 for Ga1.1 and 8:17:2 for nga1111. The pairs of segregation ratios of the selected versus unselected plants were indistinguishable from each other by the chi-squared test (<em>P</em>≤0.2 for nga8; <em>P</em>≤1.0 for both Ga1.1 and nga111, respectively). In these crosses, crossover frequencies in Ga1.1—nga8 and nga8—nga1111 intervals were 3/52 chromosomes and 1/52 chromosomes, respectively, in the presence of NAM selection, and 4/54 chromosomes and 2/54 chromosomes, respectively, in the absence of NAM selection. These pairs of frequencies in selected versus unselected lines are statistically indistinguishable. Moreover, for 231 F2 progeny of 53 families screened for Ds excision (selection on NAM for <em>Ac</em><sup>−</sup>, GUS<sup>−</sup> phenotype) the ratio of <em>LL</em>:<em>NL</em>:<em>NN</em> was 24%:45%:31%, showing no significant deficiency of any class with selection on NAM. The <em>TMS2</em> locus is unlinked to nga8, satisfying another requirement of the experiment. Linkage of TMS2 (<em>N</em>) to nga8<em><sup>N</sup></em> would have resulted in a deficiency of <em>NN</em> at nga8 with negative selection on NAM. PCR confirmed that NAM<sup>S </sup>plants were also <em>Ac</em><sup>−</sup>.</p>


<h4>Correlated movement of unrelated inverted repeat</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The regions flanking the excision footprints (<a href="#pone-0000965-g005">Figure 5</a> A, B) were examined by Southern analysis to determine the fate of the 598bp repeat after <em>Ds</em> excision (<a href="#pone-0000965-g005">Figure 5C, D</a>). For 43/47 independent events tested, the 598bp repeat remained at the empty donor site (<a href="#pone-0000965-g005">Figure 5C</a>, lanes 1–5). For 2/47 events it was lost from the donor site. Southern blot on F2 and sequencing through the ET500 insertion site on F3 (<em>LL</em> at nga8) confirmed the absence of <em>Ds</em> and the 598 bp repeat in these two cases. For 1/47 events, Southern analysis was consistent with a move of the repeat to another chromosomal position, not necessarily adjacent to a reinserted <em>Ds</em> (<a href="#pone-0000965-g005">Figure 5D</a>, lanes 1, 4). For the remaining one excision, the fate of the fragment was undetermined because results of Southern analysis and PCR were inconclusive.</p>
<div class="figure" id="pone-0000965-g005"><div class="img"><a name="pone-0000965-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000965" data-uri="info:doi/10.1371/journal.pone.0000965.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000965.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000965.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000965.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000965.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000965.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Excision of <em>Ds</em> with atypical flanking DNA.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) Configuration of <em>Ds</em> in ET500 parental <em>Ds</em> element with <em>NPTII</em>, <em>GUS</em>, 5′ and 3′ TIRs. The 598bp inverted repeat is immediately 3′ of the <em>Ds</em> 3′ TIR. The duplicated target half sites (gray boxes) flank the 5′ TIR (typical) and the 598bp inverted repeat (atypical). (B) Excision of <em>Ds</em> yields broken ends that are adjacent to the leftward duplicated target at the 5′ end of <em>Ds</em> and to the 598bp inverted repeat at the 3′ end of <em>Ds</em>, and is predicted to destroy a <em>Sac</em>I site. (C) Southern blot analysis of plant DNA digested with <em>Sac</em>I, confirming <em>Ds</em> excision. Upper panel (labelled GUS) was probed with GUS sequences, stripped and reprobed with one-half of the 598bp inverted repeat sequence (middle panel, labelled 598bp). Lower portion of blot was cut out and probed with <em>DCL1</em> cDNA sequences as a loading control (lower panel, labelled DCL1). Lanes 1–5 contained DNA from F2 lines with empty donor sites; lane 6, ET500 parent. The up-shift in band size between ET500 parent and F2 progeny plants in the middle panel is because of destruction of a <em>Sac</em>1 site due to excision. Lanes 1, 2, 4, and 5; F2 progeny had lost <em>Ds</em> from the genome, but retained the 598bp fragment. Lane 3; F2 progeny had a reinsertion of <em>Ds</em> but retained the 598bp fragment at the original locus. (D) Southern blot analysis of <em>Sac</em>I digested DNA showing movement of the 598bp segment. Lanes 1 and 2 had F2 DNA from two independent F1 lines, and lane 3 had ET500 parental DNA. Probes were as labelled for panel (C). Lane 1 shows an F1 line in which the <em>Ds</em> and the 598bp inverted repeat had a change of position. Lane 2 shows a line that had lost <em>Ds</em> but not the 598bp inverted repeat sequence from the original location. Hybridization signals expected from the original location are identified with triangles; signals due to sequence alteration are identified with circles (loss of <em>Sac</em>I site adjacent to 598 bp repeat), asterisks denote movement of GUS and the 598bp repeat to another location.</p>
<span>doi:10.1371/journal.pone.0000965.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have shown that within the resolution limits of our measurement the excision in Arabidopsis of a <em>Ds</em> element from a homologous chromosome does not provoke interaction of the broken chromosome with its intact homologue, and that the repair mechanism resembles the immunoglobulin V(D)J joining reaction of vertebrates.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Previous experiments showed that at the <em>bronze</em> locus in maize <em>Ac/Ds</em> activity was not associated with homologous recombination <a href="#pone.0000965-Dooner1">[27]</a>. The <em>bronze</em> locus, however, is a recombination hotspot <a href="#pone.0000965-Dooner2">[28]</a>, and thus could have been atypical in its behaviour with respect to <em>Ds</em> excision. Our observations in <em>Arabidopsis</em> together with previous observations in maize at <em>bronze</em> help generalize the conclusion that <em>Ds</em> excision does not frequently influence crossover recombination between homologous chromosome pairs. These results imply that although DNA transposable elements have evolved to provide evolutionary plasticity to the host genome through frequent rearrangements, their excision may seldom affect crossover of flanking arms of homologous chromosomes. We have shown that efficient excision of <em>Ds</em> element can occur despite the absence of canonical target site duplication at the 3′ end of the element. This was also reported for A<em>c</em> excision from the maize <em>P1</em>-vv varigated pericarp allele that lacks a target site duplication <a href="#pone.0000965-Xiao2">[24]</a>. Finally, based on Southern analysis, the movement of non-<em>Ds</em> inverted repeat elements may be facilitated in <em>cis</em> by the activity of nearby transposable elements. Although we have not provided conclusive molecular evidence for movement of the repeat, our analysis is consistent with the recent finding that endogenous repetitive sequences are involved in a subset of chromosomal rearrangements mediated by <em>Ds</em> excision <a href="#pone.0000965-Page1">[29]</a>. A possible mechanism of transposition of the 598 bp repeat could be cryptic transposase binding sites within this inverted repeat. When high-affinity transposase binds to the <em>Ds</em> TIRs, binding of further transposase molecules to nearby cryptic sites may be facilitated. Alternatively, DNA may occasionally slide through the bound transposase whereupon the enzyme makes a cleavage at a distance from the binding site (<a href="#pone-0000965-g004">Fig 4</a>). These interpretations suggest that the formation as well as further transposition of novel inverted repeat sequences are facilitated by existing transposable elements, and could contribute towards the evolution of new mobile elements. The alternative explanation, a move of the 598 bp repeat during a spontaneous rearrangement or recombination at the ET500 locus, not associated with Ds excision and repair, seems unlikely because the control cross ET500×wild type No-0 showed no such instability. More likely are the remaining two explanations: a transposition of the 598 bp repeat to a new chromosomal location, or a transposition during which the <em>Ds</em> and 598 bp repeat were rearranged within the ET500 locus. We suggest that the 598 bp repeat is mobile because we observed a loss of this repeat from the genome in association with two independent <em>Ds</em> transpositions.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>In further support of the existence of cryptic binding sites within the 598 bp repeat, motifs were identified based on the results of Becker and Kunze <a href="#pone.0000965-Becker1">[30]</a> describing <em>Ac</em> transposase binding sites as ACG, TCG and A/TCGG in the 250 bp subterminal ends inside the 11-bp terminal IRs. We found these motifs 25 times in the 598 bp repeat. Consistent with previous observations <a href="#pone.0000965-Becker1">[30]</a> these motifs are present on both strands in direct and inverted repeat orientations. The number of motifs within the 598 bp repeat is comparable to that in <em>Ac</em> ends, namely, 25 in <em>Ac</em> 5′ and 20 in <em>Ac</em> 3′ TIRs, respectively. Although the motifs in the 598 bp inverted repeat are outside the terminal IR of the <em>Ds</em> element, flexibility of the transposase in DNA binding <a href="#pone.0000965-Becker1">[30]</a> may allow the transposase to ‘slide’ and recognize these cryptic sites while still requiring the TIRs of the <em>Ds</em> element in <em>cis</em> for transposition. The presence of transposase binding sites within DNA, however, is necessary but not sufficient for transposition <a href="#pone.0000965-Conrad1">[25]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The observed lack of detectable crossover enhancement by a pioneering transposable element in a foreign genome and the formation of novel repeated DNA sequences as a consequence of transposon movement have implications on the role of transposable elements in genomic sculpting. While we did not find any evidence of correlated crossover exchange at the site of excision, the influence of transposable elements as agents causing rare crossovers cannot be eliminated. Nevertheless, it appears that movements of transposons are unlikely to be the most important direct agent for causing somatic or germinal crossovers. On the other hand, the production of novel repeated elements might indirectly create the possibility of intra-chromosomal or inter-chromosomal crossover by homologous recombination in subsequent generations.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>A consideration of the conserved and unique aspects of <em>Ds</em> footprints in maize <a href="#pone.0000965-Dooner1">[27]</a>, <a href="#pone.0000965-Saedler1">[31]</a>, <a href="#pone.0000965-Yan1">[32]</a>, <em>Arabidopsis</em> <a href="#pone.0000965-Gorbunova1">[23; this work]</a> and yeast <a href="#pone.0000965-Weil1">[4]</a>, <a href="#pone.0000965-Yu1">[21]</a> suggest that binding of the transposase molecules at <em>Ds</em> TIRs may be sufficient for accurate excision of <em>Ds</em> but that the mechanism of repair of broken ends at the donor site is dependent on host factors. In maize and tobacco, repair occurs mostly by an error prone repair synthesis followed by end joining. In <em>Arabidopsis</em> and yeast, the major footprints are consistent with hairpin formation, hairpin opening and microhomology mediated end joining. One wonders whether by manipulating host factors in plants might the ends be forced to repair by homologous recombination as normally seen with P-element transposition in <em>Drosophila</em> <a href="#pone.0000965-Engels1">[33]</a>, <a href="#pone.0000965-Gloor1">[34]</a>.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p><strong>Strains and plant growth conditions</strong></p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p><em>Arabidopsis thaliana</em> strain ET500 (La-er) was the source of <em>Ds</em> insertion element <a href="#pone.0000965-Sundaresan1">[19]</a>, and strain NaeAc (No-0) was the source of <em>Ac</em> transposase <a href="#pone.0000965-Fedoroff1">[35]</a>. Conditions of growth and propagation were as described before <a href="#pone.0000965-Golden1">[36]</a>.</p>

<h4>Verification of homozygous lines</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>ET500 and NaeAc putative homozygous insertion lines were selfed for two generations. In the second generation ET500 F2 segregated 100% (42/42) kanamycin resistant seedlings, and has GUS<sup>+</sup> leaves with 100% penetrance and expressivity throughout the life cycle. In addition, a ring of cells on the ovary receptacle expressed GUS. The <em>GUS</em> transgene segregated as a dominant marker in the cross ET500×wild type Nossen: 31/42 of F2 progeny were GUS<sup>+</sup>, indicating a single dominant <em>GUS</em> transgene (<em>P</em>&lt;0.005 for heterozygosity). Similarly, in the second generation of NaeAc selfing, 14/14 F2 plants were kanamycin resistant and tested positive for the transposase gene by PCR analysis. Additionally all 53 F1 plants of ET500×NaeAc scored positive for the <em>Ac</em> gene by PCR analysis. Sequencing of genomic DNA 301 bp to the left of the <em>Ds</em> insertion site and 341 bp to the right of the inverted repeat sequence showed 100% sequence homology between the Nae<em>Ac</em> and Landsberg <em>erecta</em> parental lines.</p>


<h4>Molecular methods</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Southern blots were performed as described by Golden and colleagues <a href="#pone.0000965-Golden1">[36]</a>. Methods for DNA isolation, PCR, SSLP typing, cloning and analysis were described before <a href="#pone.0000965-Golden1">[36]</a>, <a href="#pone.0000965-Ray1">[37]</a>. After amplification of SSLP markers, the PCR amplified DNA fragments were separated on a 4% Metaphor gel (FMC) at 65V for 4 hours. The polymorphic sizes are: nga8 (La-er 198 bp; No-0 168 bp); nga1111 (La-er 154 bp; No-0 146 bp or 150 bp); Ga1a/<em>Bsa</em>BI (La-er 0.707 kb and 0.527 kb; No-0 1.196 kb). PCR conditions for SSLP markers were: for nga1111 and Ga1a, 40 cycles of 15s (96°C)—15s (55°C)—30s (72°C); for nga8, 40 cycles of 15s (95°C)—15s (58°C)—30s (72°C). For amplifying <em>Ac</em> and <em>GUS</em> sequences, PCR condition was 36 cycles of 15s (95°C)—15s (58°C)—30s (72°C). The <em>Ac</em> primers were AC2210, 5′GTTGAGACATCATATGAGATC3′; and AC3080, 5′ATTGCAAGGCTAAGTATAGG3′. <em>GUS</em> primers were GUS43, 5′CTCGACGGCCTGTGGGCATTC3′; and GUS945, 5′GTGGTCGTGCACCATCAGCACG3′. PCR analysis of ET500 excision sites (footprints) used 36 cycles of 30s (96°C)—30s (63°C)—90s (72°C) (Primers were: 318E, 5′CAAGGTTGGATTCAGAGGAATCCTATTGTTCTTGAG3′​;T6150-EC, GCTTACGTACATGGTCGATAAGAAAAGGCAATTTGTAGAT​G3′;and 58237E-2,</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>5′TAACAAGAATGAGATATAAAGTATCACCTTCTCATCAT​GAG3′).For PCR of excision sites in plants that lost both the Ds and ocs3′-RB (Primers: T6136, 5′CCATTCGCCCTATAGTGAGTCG3′; and T6409-2, 5′TGCCAGTCAGCATCATCACACC3′), the reaction condition was 36 cycles of 30s (96°C)—30s (63°C)—90s (72°C). PCR of the ocs3′-RB inverted repeat flanked by a portion of the 3′ <em>Ds</em> and plant DNA was with 0.25 µM each of primers Ds3-2E (5′CGTTACCGGTATATCCCGTTTTCG3′) and 58179E (5′CAATGCTCTCTGCCTTCGCCGTTAAAG3′) for 36 cycles of 30s (96°C)—30s (61.7°C)—120s (72°C).</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>PCR of the 598 bp inverted repeat flanked by 3′<em>Ds</em> and plant genomic DNA was performed in a reaction volume of 50 µl with 1M betaine (Sigma) and 6.25 units <em>Taq</em> DNA polymerase (Qiagen), with 0.25 µM each of primers Ds3-2E and 52879E. The reaction was a ‘hot start’ with 1 cycle of denaturation at 95°C, then 36 cycles of 30 s (96°C)—30s (61.7°C)—2min (72°C).</p>


<h4>Single-stranded PCR and cloning of DNA flanking the 3′ and 5′ <em>Ds</em></h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Thermal asymmetric interlaced (TAIL) PCR <a href="#pone.0000965-Liu1">[38]</a> for DNA genomic flanking the 5′ <em>Ds</em> in ET500 was accomplished with primers: Ds5-1, Ds5-2, Ds5-3 and AD3, with secondary and tertiary cycles as described <a href="#pone.0000965-Grossniklaus1">[39]</a>. DNA on the 3′ flank of <em>Ds</em> was cloned by single-stranded PCR base on the “5′ Race System for Rapid Amplification of cDNA Ends, Version 2.0” (GibcoBRL), with modifications as follows: single-stranded PCR of 1 µl of ET500 genomic DNA with Ds3-1 and <em>Taq</em> Polymerase High Fidelity as follows: 1 cycle 94°C for 2 min; 40 cycles of 15 sec at 94°C, 15 sec annealing at 55°C and 1 min extension at 68°C; PCR of C-tailed DNA, Ds3-2 with 20 sec denaturation at 94°C, 15 sec annealing at 50°C and 3 min polymerization at 68°C; 40 cycles; nested PCR with CUDs3-3A, 5′CAUCAUCAUCAUGGTAGAGGTATTTTACCGACC3′, 94°C hot start 20 sec at 94°C, 15 sec at 55°C and 3 min at72°C, 40 cycles. Polymerization for the 40th cycle was extended from 3 to 7 min.DNA fragments cloned from the 3′ flank representing one-half of the inverted repeat were used probing Southern blots. PCR primers for probes were T6136 and T6409-2. Primer T6136 corresponds to 22 bases immediately (3′) of the <em>Sac</em>I site, within the inverted repeat, as shown in <a href="#pone-0000965-g001">Fig. 1</a> (The 5′ end of primer T6409-2 is complementary to base 6409 on the ET500 parental T-DNA vector; Genbank accession number M35007.1), and its 3′ end is complementary to base 6378 on the same T-DNA sequence. The 5′ end crosses the point of inversion by three bases (<a href="#pone-0000965-g001">Fig. 1</a>). Sequencing of the Nae<em>Ac</em> and wt Landsberg <em>erecta</em> parental genomic DNA flanking the ET500 insertion site for comparison of homology was accomplished with primers 318E and 58179E.</p>


<h4>DNA Sequencing</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Sequencing reactions were as stated in the ABI Prism BigDye Terminator cycle Sequencing Ready Reaction Kit (Applied Biosystems), with 16 pmoles of primer and 6ul Big dye Terminator Ready Reaction Mix (Applied Biosystems) for a 20 ul reaction. Sequencing primers and PCR primer sequences were the same. To sequence the 598 bp inverted repeat with flanking <em>Ds</em> and plant DNA, the reaction was 40 µl with 32 pmoles Ds3Sac, 5′CCGTTTTCATCCCTAGAGCTCCAATTCGC3′, 1M betaine (Sigma) 16 µl Big Dye Terminator Ready Reaction Mix, 1M betaine as follows: (98°C)-3min, then 25 cycles of 30s (96°C)—4min (64.0°C).</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Drs. Peter Gibbs, Teresa Golden, Martin Gorovsky, David Hinkle, Christopher Lawrence, Stephen Schauer, Samuel Schlagman, and Clifford Weil for discussion and advice, Dr. Nina Fedoroff for the <em>Ac</em> strain, and Wendy B. Gagliano for technical assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AR UG ML. Performed the experiments: ML LS. Analyzed the data: AR ML. Contributed reagents/materials/analysis tools: AR UG. Wrote the paper: AR ML. Other: Edited the paper: UG. In the PhD committee of Marybeth Langer: UG. Undergraduate researcher who worked under Marybeth Langer's supervision: LS. Advised Marybeth Langer's PhD research work: AR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000965-Kazazian1" id="pone.0000965-Kazazian1"></a>Kazazian HH Jr (2004) Mobile elements: drivers of genome evolution. Science  303: 1626–1632.  <ul class="find" data-citedArticleID="998692" data-doi="10.1126/science.1089670"><li><a href="http://dx.doi.org/10.1126/science.1089670" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mobile+elements%3A+drivers+of+genome+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mobile+elements%3A+drivers+of+genome+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000965-Ros1" id="pone.0000965-Ros1"></a>Ros F, Kunze R (2001) Regulation of activator/dissociation transposition by replication and DNA methylation. Genetics  157: 1723–1733.  <ul class="find" data-citedArticleID="998708"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+activator%2Fdissociation+transposition+by+replication+and+DNA+methylation.&amp;auth=&amp;atitle=Regulation+of+activator%2Fdissociation+transposition+by+replication+and+DNA+methylation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+activator%2Fdissociation+transposition+by+replication+and+DNA+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+activator%2Fdissociation+transposition+by+replication+and+DNA+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000965-Rinehart1" id="pone.0000965-Rinehart1"></a>Rinehart TA, Dean C, Weil CF (1997) Comparative analysis of non-random DNA repair following <em>Ac</em> transposition excision in maize and <em>Arabidopsis</em>. Plant J  12: 1419–1427.  <ul class="find" data-citedArticleID="998704" data-doi="10.1046/j.1365-313x.1997.12061419.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1997.12061419.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+non-random+DNA+repair+following+Ac+transposition+excision+in+maize+and+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+non-random+DNA+repair+following+Ac+transposition+excision+in+maize+and+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000965-Weil1" id="pone.0000965-Weil1"></a>Weil CF, Kunze R (2000) Transposition of maize Ac/Ds transposable elements in the yeast Saccharomyces cerevisiae. Nat Genet  26: 187–190.  <ul class="find" data-citedArticleID="998722" data-doi="10.1038/82827"><li><a href="http://dx.doi.org/10.1038/82827" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transposition+of+maize+Ac%2FDs+transposable+elements+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transposition+of+maize+Ac%2FDs+transposable+elements+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000965-Rommens1" id="pone.0000965-Rommens1"></a>Rommens CM, van der Biezen EA, Ouwerkerk PB, Nijkamp HJ, Hille J (1991) Ac-induced disruption of the double Ds structure in tomato. Mol Gen Genet  228: 453–458.  <ul class="find" data-citedArticleID="998706" data-doi="10.1007/bf00260639"><li><a href="http://dx.doi.org/10.1007/bf00260639" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ac-induced+disruption+of+the+double+Ds+structure+in+tomato." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ac-induced+disruption+of+the+double+Ds+structure+in+tomato.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000965-VanderBiezen1" id="pone.0000965-VanderBiezen1"></a>Van der Biezen EA, Brandwagt BF, van Leeuwen W, Nijkamp HJ, Hille J (1996) Identification and isolation of the FEEBLY gene from tomato by transposon tagging. Mol Gen Genet.  251: 267–280.  <ul class="find" data-citedArticleID="998720" data-doi="10.1007/bf02172517"><li><a href="http://dx.doi.org/10.1007/bf02172517" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+isolation+of+the+FEEBLY+gene+from+tomato+by+transposon+tagging." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+isolation+of+the+FEEBLY+gene+from+tomato+by+transposon+tagging.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000965-Eichler1" id="pone.0000965-Eichler1"></a>Eichler EE, Sankoff D (2003) Structural dynamics of eukaryotic chromosome evolution. Science  301: 793–797.  <ul class="find" data-citedArticleID="998672" data-doi="10.1126/science.1086132"><li><a href="http://dx.doi.org/10.1126/science.1086132" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+dynamics+of+eukaryotic+chromosome+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+dynamics+of+eukaryotic+chromosome+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000965-McClintock1" id="pone.0000965-McClintock1"></a>McClintock B (1953) Mutations in maize. Carnegie Inst Wash Ybk  52: 227–237.  <ul class="find" data-citedArticleID="998698"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mutations+in+maize.&amp;auth=&amp;atitle=Mutations+in+maize." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+maize." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+maize.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000965-Fradkin1" id="pone.0000965-Fradkin1"></a>Fradkin CMW, Brink RA (1956) Crossing over in maize in the presence of the transposable factors <em>Activator</em> and <em>Modulator</em>. Genetics  41: 901–906.  <ul class="find" data-citedArticleID="998680"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Crossing+over+in+maize+in+the+presence+of+the+transposable+factors+Activator+and+Modulator.&amp;auth=&amp;atitle=Crossing+over+in+maize+in+the+presence+of+the+transposable+factors+Activator+and+Modulator." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crossing+over+in+maize+in+the+presence+of+the+transposable+factors+Activator+and+Modulator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crossing+over+in+maize+in+the+presence+of+the+transposable+factors+Activator+and+Modulator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000965-Fischer1" id="pone.0000965-Fischer1"></a>Fischer SE, Wienholds E, Plasterk RH (2003) Continuous exchange of sequence information between dispersed Tc1 transposons in the <em>Caenorhabditis elegans</em> genome. Genetics  164: 127–134.  <ul class="find" data-citedArticleID="998678"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Continuous+exchange+of+sequence+information+between+dispersed+Tc1+transposons+in+the+Caenorhabditis+elegans+genome.&amp;auth=&amp;atitle=Continuous+exchange+of+sequence+information+between+dispersed+Tc1+transposons+in+the+Caenorhabditis+elegans+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Continuous+exchange+of+sequence+information+between+dispersed+Tc1+transposons+in+the+Caenorhabditis+elegans+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Continuous+exchange+of+sequence+information+between+dispersed+Tc1+transposons+in+the+Caenorhabditis+elegans+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000965-Xiao1" id="pone.0000965-Xiao1"></a>Xiao YL, Li X, Peterson T (2000) Ac insertion site affects the frequency of transposon-induced homologous recombination at the maize p1 locus. Genetics  156: 2007–2017.  <ul class="find" data-citedArticleID="998726"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Ac+insertion+site+affects+the+frequency+of+transposon-induced+homologous+recombination+at+the+maize+p1+locus.&amp;auth=&amp;atitle=Ac+insertion+site+affects+the+frequency+of+transposon-induced+homologous+recombination+at+the+maize+p1+locus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ac+insertion+site+affects+the+frequency+of+transposon-induced+homologous+recombination+at+the+maize+p1+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ac+insertion+site+affects+the+frequency+of+transposon-induced+homologous+recombination+at+the+maize+p1+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000965-Wright1" id="pone.0000965-Wright1"></a>Wright SI, Agrawal N, Bureau TE (2003) Effects of recombination rate and gene density on transposable element distributions in <em>Arabidopsis thaliana</em>. Genome Res  13: 1897–1903.  <ul class="find" data-citedArticleID="998724"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effects+of+recombination+rate+and+gene+density+on+transposable+element+distributions+in+Arabidopsis+thaliana.&amp;auth=&amp;atitle=Effects+of+recombination+rate+and+gene+density+on+transposable+element+distributions+in+Arabidopsis+thaliana." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+recombination+rate+and+gene+density+on+transposable+element+distributions+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+recombination+rate+and+gene+density+on+transposable+element+distributions+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000965-Fu1" id="pone.0000965-Fu1"></a>Fu H, Zheng Z, Dooner HK (2002) Recombination rates between adjacent genic and retrotransposon regions in maize vary by 2 orders of magnitude. Proc Natl Acad Sci USA  99: 1082–1087.  <ul class="find" data-citedArticleID="998682"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Recombination+rates+between+adjacent+genic+and+retrotransposon+regions+in+maize+vary+by+2+orders+of+magnitude.&amp;auth=&amp;atitle=Recombination+rates+between+adjacent+genic+and+retrotransposon+regions+in+maize+vary+by+2+orders+of+magnitude." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+rates+between+adjacent+genic+and+retrotransposon+regions+in+maize+vary+by+2+orders+of+magnitude." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+rates+between+adjacent+genic+and+retrotransposon+regions+in+maize+vary+by+2+orders+of+magnitude.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000965-Yao1" id="pone.0000965-Yao1"></a>Yao H, Zhou Q, Li J, Smith H, Yandeau M, et al.  (2002) Molecular characterization of meiotic recombination across the 140-kb multigenic a1-sh2 interval of maize. Proc Natl Acad Sci USA  99: 5763–5765.  <ul class="find" data-citedArticleID="998732" data-doi="10.1073/pnas.082562199"><li><a href="http://dx.doi.org/10.1073/pnas.082562199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+meiotic+recombination+across+the+140-kb+multigenic+a1-sh2+interval+of+maize." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+meiotic+recombination+across+the+140-kb+multigenic+a1-sh2+interval+of+maize.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000965-Coen1" id="pone.0000965-Coen1"></a>Coen ES, Carpenter R, Martin C (1986) Transposable elements generate novel spatial patterns of gene expression in Antirrhinum majus. Cell  47: 285–96.  <ul class="find" data-citedArticleID="998664" data-doi="10.1016/0092-8674(86)90451-4"><li><a href="http://dx.doi.org/10.1016/0092-8674(86)90451-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transposable+elements+generate+novel+spatial+patterns+of+gene+expression+in+Antirrhinum+majus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transposable+elements+generate+novel+spatial+patterns+of+gene+expression+in+Antirrhinum+majus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000965-Bai1" id="pone.0000965-Bai1"></a>Bai L, Singh M, Pitt L, Sweeney M, Brutnell TP (2007) Generating novel allelic variation through Activator insertional mutagenesis in maize. Genetics  175: 981–992.  <ul class="find" data-citedArticleID="998660" data-doi="10.1534/genetics.106.066837"><li><a href="http://dx.doi.org/10.1534/genetics.106.066837" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generating+novel+allelic+variation+through+Activator+insertional+mutagenesis+in+maize." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generating+novel+allelic+variation+through+Activator+insertional+mutagenesis+in+maize.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000965-Roth1" id="pone.0000965-Roth1"></a>Roth DB, Zhu C, Gellert M (1993) Characterization of broken DNA molecules associated with V(D)J recombination. Proc Natl Acad Sci USA  90: 10788–10792.  <ul class="find" data-citedArticleID="998710" data-doi="10.1073/pnas.90.22.10788"><li><a href="http://dx.doi.org/10.1073/pnas.90.22.10788" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+broken+DNA+molecules+associated+with+V%28D%29J+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+broken+DNA+molecules+associated+with+V%28D%29J+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000965-McBlane1" id="pone.0000965-McBlane1"></a>McBlane F, van Gent D, Ramsden D, Romeo C, Cuomo C, et al.  (1995) Cleavage at a V(D)J. recombination signal requires only RAG1 and RAG2 proteins and occurs in two steps. Cell  83: 387–395.  <ul class="find" data-citedArticleID="998696" data-doi="10.1016/0092-8674(95)90116-7"><li><a href="http://dx.doi.org/10.1016/0092-8674(95)90116-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cleavage+at+a+V%28D%29J.+recombination+signal+requires+only+RAG1+and+RAG2+proteins+and+occurs+in+two+steps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cleavage+at+a+V%28D%29J.+recombination+signal+requires+only+RAG1+and+RAG2+proteins+and+occurs+in+two+steps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000965-Sundaresan1" id="pone.0000965-Sundaresan1"></a>Sundaresan V, Springer P, Volpe T, Haward S, Jones JD, et al.  (1995) Patterns of gene action in plant development revealed by enhancer trap and gene trap transposable elements. Genes Dev  9: 1797–1810.  <ul class="find" data-citedArticleID="998716" data-doi="10.1101/gad.9.14.1797"><li><a href="http://dx.doi.org/10.1101/gad.9.14.1797" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+gene+action+in+plant+development+revealed+by+enhancer+trap+and+gene+trap+transposable+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+gene+action+in+plant+development+revealed+by+enhancer+trap+and+gene+trap+transposable+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000965-Zhou1" id="pone.0000965-Zhou1"></a>Zhou L, Mitra R, Atkinson PW, Hickman AB, Dyda F, et al.  (2004) Transposition of hAT elements links transposable elements and V(D)J recombination. Nature  432: 995–1001.  <ul class="find" data-citedArticleID="998736" data-doi="10.1038/nature03157"><li><a href="http://dx.doi.org/10.1038/nature03157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transposition+of+hAT+elements+links+transposable+elements+and+V%28D%29J+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transposition+of+hAT+elements+links+transposable+elements+and+V%28D%29J+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000965-Yu1" id="pone.0000965-Yu1"></a>Yu J, Marshall K, Yamaguchi M, Haber JE, Weil CF (2004) Microhomology-dependent end joining and repair of transposon-induced DNA hairpins by host factors in Saccharomyces cerevisiae. Mol Cell Biol  24: 1351–1364.  <ul class="find" data-citedArticleID="998734" data-doi="10.1128/mcb.24.3.1351-1364.2004"><li><a href="http://dx.doi.org/10.1128/mcb.24.3.1351-1364.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microhomology-dependent+end+joining+and+repair+of+transposon-induced+DNA+hairpins+by+host+factors+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microhomology-dependent+end+joining+and+repair+of+transposon-induced+DNA+hairpins+by+host+factors+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000965-Scott1" id="pone.0000965-Scott1"></a>Scott L, LaFoe D, Weil CF (1996) Adjacent sequences influence DNA repair accompanying transposon excision in maize. Genetics  142: 237–246.  <ul class="find" data-citedArticleID="998714"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Adjacent+sequences+influence+DNA+repair+accompanying+transposon+excision+in+maize.&amp;auth=&amp;atitle=Adjacent+sequences+influence+DNA+repair+accompanying+transposon+excision+in+maize." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adjacent+sequences+influence+DNA+repair+accompanying+transposon+excision+in+maize." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adjacent+sequences+influence+DNA+repair+accompanying+transposon+excision+in+maize.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000965-Gorbunova1" id="pone.0000965-Gorbunova1"></a>Gorbunova V, Levy AA (2000) Analysis of extrachromosomal Ac/Ds transposable elements. Genetics  155: 349–359.  <ul class="find" data-citedArticleID="998688"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+extrachromosomal+Ac%2FDs+transposable+elements.&amp;auth=&amp;atitle=Analysis+of+extrachromosomal+Ac%2FDs+transposable+elements." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+extrachromosomal+Ac%2FDs+transposable+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+extrachromosomal+Ac%2FDs+transposable+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000965-Xiao2" id="pone.0000965-Xiao2"></a>Xiao YL, Peterson T (2002) Ac transposition is impaired by a small terminal deletion. Mol Genet Genomics  266: 720–731.  <ul class="find" data-citedArticleID="998728" data-doi="10.1007/s00438-001-0600-2"><li><a href="http://dx.doi.org/10.1007/s00438-001-0600-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ac+transposition+is+impaired+by+a+small+terminal+deletion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ac+transposition+is+impaired+by+a+small+terminal+deletion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000965-Conrad1" id="pone.0000965-Conrad1"></a>Conrad LJ, Brutnell TP (2005) Ac-immobilized, a stable source of Activator transposase that mediates sporophytic and gametophytic excision of Dissociation elements in maize. Genetics  171: 1999–2012.  <ul class="find" data-citedArticleID="998666" data-doi="10.1534/genetics.105.046623"><li><a href="http://dx.doi.org/10.1534/genetics.105.046623" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ac-immobilized%2C+a+stable+source+of+Activator+transposase+that+mediates+sporophytic+and+gametophytic+excision+of+Dissociation+elements+in+maize." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ac-immobilized%2C+a+stable+source+of+Activator+transposase+that+mediates+sporophytic+and+gametophytic+excision+of+Dissociation+elements+in+maize.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000965-Szostak1" id="pone.0000965-Szostak1"></a>Szostak JW, Orr-Weaver TL, Rothstein RJ, Stahl FW (1983) The double-strand-break repair model for recombination. Cell  33: 25–35.  <ul class="find" data-citedArticleID="998718" data-doi="10.1016/0092-8674(83)90331-8"><li><a href="http://dx.doi.org/10.1016/0092-8674(83)90331-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+double-strand-break+repair+model+for+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+double-strand-break+repair+model+for+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000965-Dooner1" id="pone.0000965-Dooner1"></a>Dooner HK, Martínez-Férez IM (1997) Germinal excisions of the maize transposon activator do not stimulate meiotic recombination or homology-dependent repair at the bz locus. Genetics  147: 1923–1932.  <ul class="find" data-citedArticleID="998668"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Germinal+excisions+of+the+maize+transposon+activator+do+not+stimulate+meiotic+recombination+or+homology-dependent+repair+at+the+bz+locus.&amp;auth=&amp;atitle=Germinal+excisions+of+the+maize+transposon+activator+do+not+stimulate+meiotic+recombination+or+homology-dependent+repair+at+the+bz+locus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Germinal+excisions+of+the+maize+transposon+activator+do+not+stimulate+meiotic+recombination+or+homology-dependent+repair+at+the+bz+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Germinal+excisions+of+the+maize+transposon+activator+do+not+stimulate+meiotic+recombination+or+homology-dependent+repair+at+the+bz+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000965-Dooner2" id="pone.0000965-Dooner2"></a>Dooner HK, Martínez-Férez IM (1997a) Recombination occurs uniformly within the bronze gene, a meiotic recombination hotspot in the maize genome. Plant Cell  9: 1633–1646.  <ul class="find" data-citedArticleID="998670" data-doi="10.1105/tpc.9.9.1633"><li><a href="http://dx.doi.org/10.1105/tpc.9.9.1633" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+occurs+uniformly+within+the+bronze+gene%2C+a+meiotic+recombination+hotspot+in+the+maize+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+occurs+uniformly+within+the+bronze+gene%2C+a+meiotic+recombination+hotspot+in+the+maize+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000965-Page1" id="pone.0000965-Page1"></a>Page DR, Koehler C, da Costa Nunes JA, Baroux C, Moore JM, et al.  (2004) Intrachromosomal excision of a hybrid <em>Ds</em> element induces large genomic deletions in <em>Arabidopsis</em>. Proc Natl Acad Sci USA  101: 2969–2969.  <ul class="find" data-citedArticleID="998700" data-doi="10.1073/pnas.0400089101"><li><a href="http://dx.doi.org/10.1073/pnas.0400089101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intrachromosomal+excision+of+a+hybrid+Ds+element+induces+large+genomic+deletions+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intrachromosomal+excision+of+a+hybrid+Ds+element+induces+large+genomic+deletions+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000965-Becker1" id="pone.0000965-Becker1"></a>Becker HA, Kunze R (1997) Maize <em>Activator</em> transposase has a bipartite DNA binding domain that recognizes subterminal sequences and terminal inverted repeats. Mol Gen Genet  254: 219–230.  <ul class="find" data-citedArticleID="998662" data-doi="10.1007/s004380050410"><li><a href="http://dx.doi.org/10.1007/s004380050410" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maize+Activator+transposase+has+a+bipartite+DNA+binding+domain+that+recognizes+subterminal+sequences+and+terminal+inverted+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maize+Activator+transposase+has+a+bipartite+DNA+binding+domain+that+recognizes+subterminal+sequences+and+terminal+inverted+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000965-Saedler1" id="pone.0000965-Saedler1"></a>Saedler H, Nevers P (1985) Transposition in plants: a molecular model. EMBO J  4: 585–590.  <ul class="find" data-citedArticleID="998712"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transposition+in+plants%3A+a+molecular+model.&amp;auth=&amp;atitle=Transposition+in+plants%3A+a+molecular+model." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transposition+in+plants%3A+a+molecular+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transposition+in+plants%3A+a+molecular+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000965-Yan1" id="pone.0000965-Yan1"></a>Yan X, Martínez-Férez IM, Kavchok S, Dooner HK (1999) Origination of <em>Ds</em> elements from <em>Ac</em> elements in maize: evidence for rare repair synthesis at the site of <em>Ac</em> excision. Genetics  152: 1733–1740.  <ul class="find" data-citedArticleID="998730"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Origination+of+Ds+elements+from+Ac+elements+in+maize%3A+evidence+for+rare+repair+synthesis+at+the+site+of+Ac+excision.&amp;auth=&amp;atitle=Origination+of+Ds+elements+from+Ac+elements+in+maize%3A+evidence+for+rare+repair+synthesis+at+the+site+of+Ac+excision." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origination+of+Ds+elements+from+Ac+elements+in+maize%3A+evidence+for+rare+repair+synthesis+at+the+site+of+Ac+excision." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origination+of+Ds+elements+from+Ac+elements+in+maize%3A+evidence+for+rare+repair+synthesis+at+the+site+of+Ac+excision.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000965-Engels1" id="pone.0000965-Engels1"></a>Engels WR, Johnson-Schlitz DM, Eggleston WB, Sved J (1990) High-frequency P element loss in Drosophila is homolog dependent. Cell  62: 515–523.  <ul class="find" data-citedArticleID="998674" data-doi="10.1016/0092-8674(90)90016-8"><li><a href="http://dx.doi.org/10.1016/0092-8674(90)90016-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-frequency+P+element+loss+in+Drosophila+is+homolog+dependent." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-frequency+P+element+loss+in+Drosophila+is+homolog+dependent.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000965-Gloor1" id="pone.0000965-Gloor1"></a>Gloor GB, Nassif NA, Johnson-Schlitz DM, Preston CR, Engels WR (1999) Targeted gene replacement in Drosophila via P element-induced gap repair. Science  253: 1110–1117.  <ul class="find" data-citedArticleID="998684" data-doi="10.1126/science.1653452"><li><a href="http://dx.doi.org/10.1126/science.1653452" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeted+gene+replacement+in+Drosophila+via+P+element-induced+gap+repair." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeted+gene+replacement+in+Drosophila+via+P+element-induced+gap+repair.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000965-Fedoroff1" id="pone.0000965-Fedoroff1"></a>Fedoroff NV, Smith DL (1993) A versatile system for detecting transposition in Arabidopsis. Plant J  3: 273–289.  <ul class="find" data-citedArticleID="998676" data-doi="10.1111/j.1365-313x.1993.tb00178.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.1993.tb00178.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+versatile+system+for+detecting+transposition+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+versatile+system+for+detecting+transposition+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000965-Golden1" id="pone.0000965-Golden1"></a>Golden T, Schauer SE, Lang JD, Pien S, Mushegian A, et al.  (2002) SHORT INTEGUMENTS1 / SUSPENSOR1 / CARPEL FACTORY, a Dicer homologue, is a maternal-effect gene required for embryo development in Arabidopsis. Plant Physiol  130: 808–822.  <ul class="find" data-citedArticleID="998686" data-doi="10.1104/pp.003491"><li><a href="http://dx.doi.org/10.1104/pp.003491" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SHORT+INTEGUMENTS1+%2F+SUSPENSOR1+%2F+CARPEL+FACTORY%2C+a+Dicer+homologue%2C+is+a+maternal-effect+gene+required+for+embryo+development+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SHORT+INTEGUMENTS1+%2F+SUSPENSOR1+%2F+CARPEL+FACTORY%2C+a+Dicer+homologue%2C+is+a+maternal-effect+gene+required+for+embryo+development+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000965-Ray1" id="pone.0000965-Ray1"></a>Ray A, Lang JD, Golden T, Ray S (1996) <em>SHORT INTEGUMENTS1</em> (<em>SIN1</em>), a gene required for ovule development in <em>Arabidopsis</em>, also controls flowering time. Development  122: 2631–2638.  <ul class="find" data-citedArticleID="998702"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=SHORT+INTEGUMENTS1+%28SIN1%29%2C+a+gene+required+for+ovule+development+in+Arabidopsis%2C+also+controls+flowering+time.&amp;auth=&amp;atitle=SHORT+INTEGUMENTS1+%28SIN1%29%2C+a+gene+required+for+ovule+development+in+Arabidopsis%2C+also+controls+flowering+time." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SHORT+INTEGUMENTS1+%28SIN1%29%2C+a+gene+required+for+ovule+development+in+Arabidopsis%2C+also+controls+flowering+time." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SHORT+INTEGUMENTS1+%28SIN1%29%2C+a+gene+required+for+ovule+development+in+Arabidopsis%2C+also+controls+flowering+time.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000965-Liu1" id="pone.0000965-Liu1"></a>Liu YG, Mitsukawa N, Oosumi T, Whittier RA (1995) Efficient isolation and mapping of Arabidopsis thaliana T-DNA insert junction by thermal asymmetric interlaced PCR. Plant J  8: 457–463.  <ul class="find" data-citedArticleID="998694" data-doi="10.1046/j.1365-313x.1995.08030457.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1995.08030457.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+isolation+and+mapping+of+Arabidopsis+thaliana+T-DNA+insert+junction+by+thermal+asymmetric+interlaced+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+isolation+and+mapping+of+Arabidopsis+thaliana+T-DNA+insert+junction+by+thermal+asymmetric+interlaced+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000965-Grossniklaus1" id="pone.0000965-Grossniklaus1"></a>Grossniklaus U, Vielle-Calzada JP, Hoeppner MA, Gagliano WB (1998) Maternal control of embryogenesis by <em>MEDEA</em>, a <em>Polycomb</em> group gene in <em>Arabidopsis</em>. Science  280: 446–450.  <ul class="find" data-citedArticleID="998690" data-doi="10.1126/science.280.5362.446"><li><a href="http://dx.doi.org/10.1126/science.280.5362.446" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maternal+control+of+embryogenesis+by+MEDEA%2C+a+Polycomb+group+gene+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maternal+control+of+embryogenesis+by+MEDEA%2C+a+Polycomb+group+gene+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.XML" value="86964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.PDF" value="367164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g001.PNG_L" value="2513752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g001.PNG_M" value="86724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g001.PNG_S" value="8959"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g001.TIF" value="2973152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g001.PNG_I" value="27164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g002.PNG_L" value="572370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g002.PNG_M" value="139095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g002.PNG_S" value="9209"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g002.TIF" value="949382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g002.PNG_I" value="50592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g003.PNG_L" value="329628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g003.PNG_M" value="35329"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g003.PNG_S" value="8800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g003.TIF" value="522926"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g003.PNG_I" value="33945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.t001.PNG_L" value="54872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.t001.PNG_M" value="80417"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.t001.PNG_S" value="11906"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.t001.TIF" value="269620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.t001.PNG_I" value="27170"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g004.PNG_L" value="1049613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g004.PNG_M" value="98698"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g004.PNG_S" value="9086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g004.TIF" value="1615578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g004.PNG_I" value="36456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.t002.PNG_L" value="52643"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.t002.PNG_M" value="32694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.t002.PNG_S" value="7774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.t002.TIF" value="262212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.t002.PNG_I" value="10173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g005.PNG_L" value="1421985"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g005.PNG_M" value="83336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g005.PNG_S" value="8364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g005.TIF" value="2660460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000965.g005.PNG_I" value="30396"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000965&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000965" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000965&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000965&volume=&issue=&title=Transposon Excision from an Atypical Site: A Mechanism of Evolution of Novel Transposable Elements&author_name=Marybeth%20Langer%2C%20Lynn%20F.%20Sniderhan%2C%20Ueli%20Grossniklaus%2C%20Animesh%20Ray&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000965" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000965" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000965" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000965&amp;t=Transposon%20Excision%20from%20an%20Atypical%20Site%3A%20A%20Mechanism%20of%20Evolution%20of%20Novel%20Transposable%20Elements" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000965&title=Transposon%20Excision%20from%20an%20Atypical%20Site%3A%20A%20Mechanism%20of%20Evolution%20of%20Novel%20Transposable%20Elements&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000965&amp;title=Transposon%20Excision%20from%20an%20Atypical%20Site%3A%20A%20Mechanism%20of%20Evolution%20of%20Novel%20Transposable%20Elements" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000965" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000965&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Transposon Excision from an Atypical Site: A Mechanism of Evolution of Novel Transposable Elements';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000965';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Transposon%20Excision%20from%20an%20Atypical%20Site%3A%20A%20Mechanism%20of%20Evolution%20of%20Novel%20Transposable%20Elements http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000965" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000965" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">




















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24766"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24766"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Homologous+recombination%22" title="Search for articles in the subject area:'Homologous recombination'"><div class="flagText">Homologous recombination</div></a>
              <div data-categoryid="22257" data-articleid="24766"
                   data-categoryname="Homologous recombination"
                   class="flagImage" title="Flag 'Homologous recombination' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Maize%22" title="Search for articles in the subject area:'Maize'"><div class="flagText">Maize</div></a>
              <div data-categoryid="43001" data-articleid="24766"
                   data-categoryname="Maize"
                   class="flagImage" title="Flag 'Maize' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genomics%22" title="Search for articles in the subject area:'Plant genomics'"><div class="flagText">Plant genomics</div></a>
              <div data-categoryid="33365" data-articleid="24766"
                   data-categoryname="Plant genomics"
                   class="flagImage" title="Flag 'Plant genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="24766"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Repeated+sequences%22" title="Search for articles in the subject area:'Repeated sequences'"><div class="flagText">Repeated sequences</div></a>
              <div data-categoryid="20009" data-articleid="24766"
                   data-categoryname="Repeated sequences"
                   class="flagImage" title="Flag 'Repeated sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transposable+elements%22" title="Search for articles in the subject area:'Transposable elements'"><div class="flagText">Transposable elements</div></a>
              <div data-categoryid="17435" data-articleid="24766"
                   data-categoryname="Transposable elements"
                   class="flagImage" title="Flag 'Transposable elements' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1324'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1901'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1718&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
