

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Towards Alignment Independent Quantitative Assessment of Homology Detection</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000113"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000113" />

    <meta name="citation_title" content="Towards Alignment Independent Quantitative Assessment of Homology Detection"/>
    <meta itemprop="name" content="Towards Alignment Independent Quantitative Assessment of Homology Detection"/>

      <meta name="citation_author" content="Avihay Apatoff"/>
            <meta name="citation_author_institution" content="Compugen Ltd, Tel Aviv, Israel"/>
            <meta name="citation_author_institution" content="The Mina and Everard Goodman Faculty of Life Sciences, Bar-Ilan University, Ramat Gan, Israel"/>
      <meta name="citation_author" content="Eddo Kim"/>
            <meta name="citation_author_institution" content="Compugen Ltd, Tel Aviv, Israel"/>
            <meta name="citation_author_institution" content="The Mina and Everard Goodman Faculty of Life Sciences, Bar-Ilan University, Ramat Gan, Israel"/>
      <meta name="citation_author" content="Yossef Kliger"/>
            <meta name="citation_author_institution" content="Compugen Ltd, Tel Aviv, Israel"/>

    <meta name="citation_date" content="2006/12/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000113.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e113"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="1"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Mapping the protein universe.; citation_author=L Holm; citation_author=C Sander; citation_journal_title=Science; citation_volume=273; citation_number=1; citation_pages=595-603; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A systematic comparison of protein structure classifications: SCOP, CATH and FSSP.; citation_author=C Hadley; citation_author=DT Jones; citation_journal_title=Structure; citation_volume=7; citation_number=2; citation_pages=1099-1112; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=SCOP: a structural classification of proteins database for the investigation of sequences and structures.; citation_author=AG Murzin; citation_author=SE Brenner; citation_author=T Hubbard; citation_author=C Chothia; citation_journal_title=J Mol Biol; citation_volume=247; citation_number=3; citation_pages=536-540; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=SSAP: sequential structure alignment program for protein structure comparison.; citation_author=CA Orengo; citation_author=WR Taylor; citation_journal_title=Methods Enzymol; citation_volume=266; citation_number=4; citation_pages=617-635; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Searching protein structure databases has come of age.; citation_author=L Holm; citation_author=C Sander; citation_journal_title=Proteins: Structure, Function, and Genetics; citation_volume=19; citation_number=5; citation_pages=165-173; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Protein structure comparison by alignment of distance matrices.; citation_author=L Holm; citation_author=C Sander; citation_journal_title=J Mol Biol; citation_volume=233; citation_number=6; citation_pages=123-138; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Analysis of topological and nontopological structural similarities in the PDB: new examples with old structures.; citation_author=NN Alexandrov; citation_author=D Fischer; citation_journal_title=Proteins; citation_volume=25; citation_number=7; citation_pages=354-365; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The Protein Data Bank and the challenge of structural genomics.; citation_author=HM Berman; citation_author=TN Bhat; citation_author=PE Bourne; citation_author=Z Feng; citation_author=G Gilliland; citation_journal_title=Nat Struct Biol; citation_volume=7; citation_number=8; citation_pages=957-959; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003.; citation_author=B Boeckmann; citation_author=A Bairoch; citation_author=R Apweiler; citation_author=MC Blatter; citation_author=A Estreicher; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=9; citation_pages=365-370; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Basic local alignment search tool.; citation_author=SF Altschul; citation_author=W Gish; citation_author=W Miller; citation_author=EW Myers; citation_author=DJ Lipman; citation_journal_title=J Mol Biol; citation_volume=215; citation_number=10; citation_pages=403-410; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Rapid and sensitive sequence comparison with FASTP and FASTA.; citation_author=WR Pearson; citation_journal_title=Methods Enzymol; citation_volume=183; citation_number=11; citation_pages=63-98; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=12; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Profile analysis: detection of distantly related proteins.; citation_author=M Gribskov; citation_author=AD McLachlan; citation_author=D Eisenberg; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=84; citation_number=13; citation_pages=4355-4358; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Alignment of protein sequences using secondary structure: a modified dynamic programming method.; citation_author=F Fischel-Ghodsian; citation_author=G Mathiowitz; citation_author=TF Smith; citation_journal_title=Protein Eng; citation_volume=3; citation_number=14; citation_pages=577-581; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Protein fold recognition by prediction-based threading.; citation_author=B Rost; citation_author=R Schneider; citation_author=C Sander; citation_journal_title=J Mol Biol; citation_volume=270; citation_number=15; citation_pages=471-480; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Discovering hidden viral piracy.; citation_author=E Kim; citation_author=Y Kliger; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=16; citation_pages=4216-4222; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Signal peptide prediction based on analysis of experimentally verified cleavage sites.; citation_author=Z Zhang; citation_author=WJ Henzel; citation_journal_title=Protein Sci; citation_volume=13; citation_number=17; citation_pages=2819-2824; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Improved prediction of signal peptides: SignalP 3.0.; citation_author=JD Bendtsen; citation_author=H Nielsen; citation_author=G von Heijne; citation_author=S Brunak; citation_journal_title=J Mol Biol; citation_volume=340; citation_number=18; citation_pages=783-795; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The mystery of nonclassical protein secretion. A current view on cargo proteins and potential export routes.; citation_author=W Nickel; citation_journal_title=Eur J Biochem; citation_volume=270; citation_number=19; citation_pages=2109-2119; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Anti-immunology: evasion of the host immune system by bacterial and viral pathogens.; citation_author=BB Finlay; citation_author=G McFadden; citation_journal_title=Cell; citation_volume=124; citation_number=20; citation_pages=767-782; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins.; citation_author=KD Pruitt; citation_author=T Tatusova; citation_author=DR Maglott; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=21; citation_pages=D501-504; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information.; citation_author=DL Wheeler; citation_author=T Barrett; citation_author=DA Benson; citation_author=SH Bryant; citation_author=K Canese; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=22; citation_pages=D173-180; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000113" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Towards Alignment Independent Quantitative Assessment of Homology Detection"/>
    <meta name="twitter:description" content="Identification of homologous proteins provides a basis for protein annotation. Sequence alignment tools reliably identify homologs sharing high sequence similarity. However, identification of homologs that share low sequence similarity remains a challenge. Lowering the cutoff value could enable the identification of diverged homologs, but also introduces numerous false hits. Methods are being continuously developed to minimize this problem. Estimation of the fraction of homologs in a set of protein alignments can help in the assessment and development of such methods, and provides the users with intuitive quantitative assessment of protein alignment results. Herein, we present a computational approach that estimates the amount of homologs in a set of protein pairs. The method requires a prevalent and detectable protein feature that is conserved between homologs. By analyzing the feature prevalence in a set of pairwise protein alignments, the method can estimate the number of homolog pairs in the set independently of the alignments' quality. Using the HomoloGene database as a standard of truth, we implemented this approach in a proteome-wide analysis. The results revealed that this approach, which is independent of the alignments themselves, works well for estimating the number of homologous proteins in a wide range of homology values. In summary, the presented method can accompany homology searches and method development, provides validation to search results, and allows tuning of tools and methods."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000113.g004"/>

  <meta property="og:title" content="Towards Alignment Independent Quantitative Assessment of Homology Detection" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000113" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5594'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2228'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=603&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000113">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000113" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000113&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Towards Alignment Independent Quantitative Assessment of Homology Detection
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Avihay Apatoff, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Compugen Ltd, Tel Aviv, Israel, 
                  The Mina and Everard Goodman Faculty of Life Sciences, Bar-Ilan University, Ramat Gan, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eddo Kim, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Compugen Ltd, Tel Aviv, Israel, 
                  The Mina and Everard Goodman Faculty of Life Sciences, Bar-Ilan University, Ramat Gan, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yossef Kliger
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kliger@compugen.co.il">kliger@compugen.co.il</a>
            </p>

                <p>Affiliation:
                  Compugen Ltd, Tel Aviv, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 27, 2006</li>
    <li>DOI: 10.1371/journal.pone.0000113</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000113" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000113" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000113" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000113" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000113-g001" data-doi="info:doi/10.1371/journal.pone.0000113" data-uri="info:doi/10.1371/journal.pone.0000113.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000113-g002" data-doi="info:doi/10.1371/journal.pone.0000113" data-uri="info:doi/10.1371/journal.pone.0000113.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000113-g003" data-doi="info:doi/10.1371/journal.pone.0000113" data-uri="info:doi/10.1371/journal.pone.0000113.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000113-g004" data-doi="info:doi/10.1371/journal.pone.0000113" data-uri="info:doi/10.1371/journal.pone.0000113.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000113">Reader Comments (5)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000113" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Identification of homologous proteins provides a basis for protein annotation. Sequence alignment tools reliably identify homologs sharing high sequence similarity. However, identification of homologs that share low sequence similarity remains a challenge. Lowering the cutoff value could enable the identification of diverged homologs, but also introduces numerous false hits. <a href="#s4">Methods</a> are being continuously developed to minimize this problem. Estimation of the fraction of homologs in a set of protein alignments can help in the assessment and development of such methods, and provides the users with intuitive quantitative assessment of protein alignment results. Herein, we present a computational approach that estimates the amount of homologs in a set of protein pairs. The method requires a prevalent and detectable protein feature that is conserved between homologs. By analyzing the feature prevalence in a set of pairwise protein alignments, the method can estimate the number of homolog pairs in the set independently of the alignments' quality. Using the HomoloGene database as a standard of truth, we implemented this approach in a proteome-wide analysis. The results revealed that this approach, which is independent of the alignments themselves, works well for estimating the number of homologous proteins in a wide range of homology values. In summary, the presented method can accompany homology searches and method development, provides validation to search results, and allows tuning of tools and methods.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Apatoff A, Kim E, Kliger Y (2006) Towards Alignment Independent Quantitative Assessment of Homology Detection. PLoS ONE 1(1):
          e113.
            doi:10.1371/journal.pone.0000113</p><p><strong>Academic Editor: </strong>Richard Copley, Wellcome Trust Centre for Human Genetics, United Kingdom</p><p><strong>Received:</strong> September 27, 2006; <strong>Accepted:</strong> November 28, 2006; <strong>Published:</strong> December 27, 2006</p><p><strong>Copyright:</strong> © 2006 Apatoff et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors declare they did not get any financial funding.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Homology detection is a key step in predicting the function of newly discovered proteins. Different methods for homology detection are currently available, and can be divided into sequence-based and structure-based methods. Sequence-based methods rely on estimated evolutionary models that aim at reconstructing the evolutionary courses that relate the protein sequences. The structure-based approach uses protein structure data and allows searching for similar proteins over a structure classification database using structure alignment methods.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>An elementary biological dogma states that the 3D structure of a protein determines its function. Data of protein structure is therefore a superior representation of proteins over sequence data. While sequence alignment works best for closely-related homologous proteins, i.e. proteins sharing at least 40% of their amino acid residues, structure alignment produces a significant alignment even between highly diverged homologous proteins <a href="#pone.0000113-Holm1">[1]</a>. Public databases offer classifications of protein structures <a href="#pone.0000113-Hadley1">[2]</a>, e.g. SCOP database, which contains manually inspected structures that are classified in a hierarchical manner <a href="#pone.0000113-Murzin1">[3]</a>. Several methods for searching against such databases were developed <a href="#pone.0000113-Orengo1">[4]</a>–<a href="#pone.0000113-Alexandrov1">[7]</a>, and may detect homologous proteins unrecoverable in regular sequence-based searches. However, although the amount of solved protein structures grows rapidly, it lags behind sequence data. To date, the protein data bank <a href="#pone.0000113-Berman1">[8]</a> holds structure data of less than 40,000 proteins and protein fragments, while Swiss-Prot knowledgebase holds above 200,000 sequences <a href="#pone.0000113-Boeckmann1">[9]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>High level of sequence conservation is a strong indication for homology. Therefore, sequence alignment methods are frequently used when searching for homologous proteins. These methods use different heuristic algorithms for maximizing the alignment, return a score per alignment of two proteins, and the statistical significance of the pairing <a href="#pone.0000113-Altschul1">[10]</a>, <a href="#pone.0000113-Pearson1">[11]</a>. Using such methods to align closely-related homologous proteins often produces a significant long sequence alignment. For distantly-related proteins, however, both the sequence similarity and the alignment length are often reduced, resulting in a decreased statistical significance of the alignment. Moreover, distantly-related homologs might not be included in search results due to stringent default parameters. In such cases, lowering the cutoff value could allow the extraction of distant homologs, but would also amplify the background noise of randomly paired proteins.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Advanced approaches, which are being continuously developed, offer improved homology detection and a reduced background noise in alignment results <a href="#pone.0000113-Altschul2">[12]</a>–<a href="#pone.0000113-Rost1">[15]</a>. Nevertheless, there is no single superior solution that fits for all homology analyses. Different protein alignment tools, and different configurations of those tools, provide varying results for the same query and subject protein sets. An external independent method that will allow assessment and comparison of homology detection techniques is required.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Herein, we present a computational method for estimating the amount of homologous proteins in a set of protein pairs – the <em>Fhom Estimator</em> (Fraction of HOMologs Estimator). The method performs a probabilistic analysis for a set of protein pairs and estimates the fraction of homologs in it. Previously, the method was used for guiding the development of a new scoring function that improved identification of novel viral piracy events <a href="#pone.0000113-Kim1">[16]</a>. In the current study, we established the reliability of the method using a well-accepted standard of truth dataset of homologous proteins, and demonstrated that the method has a wide dynamic range, that goes down to as low as 0.01% of homologs. Most importantly, we confirmed that our method neither depends on the alignment itself, nor on the level of sequence identity. This approach allows quality assessment of homology search results.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>A method to estimate the amount of homologous proteins in a set of protein pairs</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We developed a computational method to estimate the amount of homologous proteins in a set of protein pairs (e.g. alignment hits). The estimator requires two protein sets, and a protein feature X, which is prevalent in both sets, and conserved among homologous proteins. Ideally, this feature will always be conserved among homologs, i.e., for a given pair of homologs, if a protein has X, so will its homolog, and if a protein lacks X its homolog will not have X as well. In contrast, not every pair of proteins that possess X are homologs, or else we could infer homology by simply observing the conservation of X. Of course, the ability to reliably detect the feature X is essential. Our method estimates the results of any procedure that aims at pairing homologous proteins from two sets of proteins. The prevalence of the protein feature X enables us to estimate the amount of random pairs in the results. Subtracting the estimated random pairs yields the estimated amount of real pairs of homologs in the results - <em>Fhom</em>. The rest of this section describes the method in detail, and illustrates it using a hypothetical example.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Protein pairs can be classified into one of four subgroups according to the presence (or absence) of the feature X in them. Proteins that have the feature X are labeled Y, and proteins that lack this feature are labeled N. Accordingly, protein pairs where both proteins have (or lack) the feature X, correspond to the YY (or NN) subgroup, while protein pairs where one protein has (or lacks) the feature X, but the other protein lacks (or has) it, correspond to the YN (or NY) subgroup.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>To exemplify, let us assume a hypothetical set of 1000 aligned protein pairs. Such a set may be the result of pairing the proteins of two sets, a query set and a subject set. Let us define these protein sets to have protein feature X prevalence of 15% and 20%, respectively. In addition, let us assign an observed distribution of 100, 50, 75 and 775 in the four subgroups YY, YN, NY and NN, respectively. The expected distribution can be calculated according to the prevalence of the feature X as follows:<a name="pone.0000113.e001" id="pone.0000113.e001"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.e001&amp;representation=PNG" class="inline-graphic"><span class="note">1</span></span><br>
            </p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Where, <em>QX</em> (<em>SX</em>) is the fraction of proteins in the query (subject) dataset possessing the protein feature X.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>According to the prevalence of the feature X in the query and subject protein sets, the expected distribution of 1000 protein pairs is 30, 120, 170, 680 in the four subgroups YY, YN, NY, NN.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>Each of the four observed subgroups is comprised of randomly paired proteins, and pairs of homologs. Our objective is to estimate the amount of homologs among these 1000 protein pairs. Of course, the fraction of randomly paired proteins may vary between the four subgroups of the observed distribution. This noise distribution of randomly paired proteins, which resides hidden in the observed distribution, must comply with the relations between the expected distribution subgroups. Since both the noise and expected distributions are random distributions, the noise distribution is actually a smaller version of the expected distribution. Therefore, dividing the YY subgroup by the YN subgroup of the noise distribution, for example, should yield the same result that one would get by dividing the YY subgroup by the YN subgroup of the expected distribution.</p>
<a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7"></a><p>Revealing the fraction of random protein pairs in one of the subgroups will enable the calculation of the fraction of random pairs in the remaining subgroups, according to the ratio of 30:120:170:680 between the expected subgroups. Towards this end, we first define the observed subgroup having the smallest observed-to-expected ratio (scaling subgroup), as consisting only of randomly paired proteins. This definition is likely to be incorrect, thus we overestimate the random pairs in this subgroup. In our hypothetical example, subgroup YN is selected for having the smallest ratio (50/120&lt;75/170&lt;775/680&lt;100/30). Hence, we define all 50 protein pairs in the YN subgroup to be random ones. In practice, the scaling subgroup would typically be either the YN or the NY subgroups, as most homologs reside in the NN and YY subgroups, which are therefore usually larger than expected in random. Using the scaling subgroup, and the ratio between the four subgroups in the expected distribution (30:120:170:680), it is possible to estimate the number of random pairs in the remaining three subgroups. The number of random pairs in the observed YY subgroup is 30*(50/120) = 12.5, in the NY subgroup it is 170*(50/120) = 70.8, and in the NN subgroup it is 680*(50/120) = 283.3. Next, dividing the sum of random pairs by the total number of protein pairs yields the total fraction of random pairs; its complementary fraction is our estimate of the fraction of homologs of the whole set, <em>Fhom</em>, as shown in <a href="#pone.0000113.e002">Eq. (2)</a> and <a href="#pone.0000113.e003">(3)</a>. In our example,<a name="pone.0000113.e002" id="pone.0000113.e002"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.e002&amp;representation=PNG" class="inline-graphic"><span class="note">2</span></span><br>
               <a name="pone.0000113.e003" id="pone.0000113.e003"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.e003&amp;representation=PNG" class="inline-graphic"><span class="note">3</span></span><br>Where, <em>Exp<sub>i</sub>
               </em> is the number of protein pairs in subgroup <em>i</em> according to the expected distribution, <em>Exp<sub>m</sub>
               </em> and <em>Obs<sub>m</sub>
               </em> are the expected and observed number of pairs in the subgroup having the smallest observed-to-expected ratio (scaling subgroup), and <em>N</em> is the total number of protein pairs.</p>
<a id="article1.body1.sec2.sec1.p8" name="article1.body1.sec2.sec1.p8"></a><p>The assumption that all pairs in one of the subgroups are random ones is strict, and the actual fraction of random pairs is typically lower. Thus, this method yields an underestimation of the real number of homologous protein pairs. It is important to note that the fraction of random protein pairs of the whole set cannot be larger. If this were the case, the scaling subgroup would have to be larger than it is. This is in order to comply with the larger fraction of random protein pairs. Hence, the fraction of random protein pairs is bounded by the size of this subgroup.</p>
<a id="article1.body1.sec2.sec1.p9" name="article1.body1.sec2.sec1.p9"></a><p>Finally, since we assume that homologous proteins perfectly conserve the presence of the protein feature X, the ratio between the YN and NY subgroups in the observed distribution, should be equal to the ratio in the expected distribution. Practically, as illustrated in our hypothetical example, an observed distribution may display a ratio that differs from the expected ratio (50/75 for the observed, and 120/170 for the expected). This indicates that the assumption of perfect conservation is not accurate. However, pairs of homologous proteins in which the protein feature was not conserved may only lead to further underestimation of <em>Fhom</em>, but never leads to an overestimation of <em>Fhom</em>. In summary, we present a method that provides an underestimation of the fraction of homologs in a set of protein pairs. The next sections describe implementation of this method, and its verification.</p>


<h4>Signal peptide as a feature suitable for <em>Fhom</em> estimation</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The signal peptide is an abundant protein feature, encompassing about 20% of the human proteome. The signal peptide is a 15 to 40 long amino acid sequence, recognized by the signal recognition particles (SRP). The SRP leads the nascent protein to the endoplasmic reticulum, where the signal peptide is cleaved and removed. The new protein is then secreted from the cell or becomes transmembranal. SignalP is a reliable state-of-the-art tool for detection of signal peptides <a href="#pone.0000113-Zhang1">[17]</a>, <a href="#pone.0000113-Bendtsen1">[18]</a>. Hence, the prevalence of signal peptides in the proteome of each organism in our analyses was determined by running the SignalP predictor on the curated RefSeq proteins of the organism. The results revealed that the signal peptide prevalence in the proteomes of human, mouse (<em>Mus musculus</em>), fruit fly (<em>Drosophila melanogaster</em>), and <em>Caenorhabditis elegans</em> is about 0.22, 0.22, 0.19 and 0.19, respectively.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We hypothesized that the presence of a signal peptide feature is conserved among homologous proteins. In order to test our hypothesis, sets of reliable homologous proteins were used as a standard of truth. We extracted human/mouse, human/fruit fly, and human/C. elegans homologous proteins from the HomoloGene triplets database. These organisms were selected to represent homology of different sequence divergence. In a random distribution, the expected proportion of protein pairs, in which both have or lack a signal peptide (YY+NN subgroups), is about 0.66, 0.67, and 0.67 for human/mouse, human/fruit fly, and human/C. elegans homologs, respectively (<a href="#pone-0000113-g001">Figure 1</a>, black columns). The observed proportion is clearly enriched in pairs of proteins that have the same signal peptide prediction (<a href="#pone-0000113-g001">Figure 1</a>, gray columns). Hence, the signal peptide feature is suitable for our estimation method. If homologous proteins would always have the same SignalP prediction, we expect the proportion to be 1. The actual value is below 1, meaning that not all homologous proteins have the same SignalP prediction. One explanation is that the SignalP tool is not perfect <a href="#pone.0000113-Zhang1">[17]</a>. Hence, it is possible for some homologous proteins, although both have or lack a signal peptide, to be assigned opposite predictions. Another reason why the proportion is smaller than 1, could be the fact that not all homologous proteins have identical topology, or are secreted in non-classical pathways <a href="#pone.0000113-Nickel1">[19]</a>. Nevertheless, there is a clear tendency for homologs to conserve the signal peptide.</p>
<div class="figure" id="pone-0000113-g001"><div class="img"><a name="pone-0000113-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000113" data-uri="info:doi/10.1371/journal.pone.0000113.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000113.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000113.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000113.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000113.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000113.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000113.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000113.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000113.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Signal peptide is a conserved feature of homologous proteins.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Homologous proteins of human/mouse, human/fruit fly, and human/C. elegans were extracted from HomoloGene triplets. The proportion of matching protein pairs, in which both proteins have or lack a signal peptide, was calculated for each pair of organisms (gray columns). For comparison, the expected proportions (black columns), is calculated under the assumption that signal peptide is not conserved between homologs, using the prevalence of proteins having signal peptides in each organism.</p>
<span>doi:10.1371/journal.pone.0000113.g001</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>One could argue that the reason SignalP assigns similar predictions to homologous proteins is because those proteins are very close in sequence; hence, SignalP, which uses the sequence as an input, will have similar predictions. To examine whether the signal peptide feature allows our method to be sequence alignment independent, HomoloGene protein pairs of the four selected organisms were divided into five groups according to their alignment identity, and the <em>Fhom</em> was estimated for each identity group (<a href="#pone-0000113-g002">Figure 2</a>). All of the groups comprise over 1000 protein pairs. Had our method been dependent on the alignment score values, one could expect a strong decrease in the homology estimates as a function of the decrease in the identity level. As revealed in <a href="#pone-0000113-g002">Figure 2</a> this is not the case: our method estimates high values of homology (~80%) even for the group of diverged homologous proteins that share less than 40% of their amino acid residues. In other words, the alignment itself has little effect on the homology estimation. Therefore, the method can be considered alignment independent, and the use of the signal peptide feature is applicable even where sequence alignment deteriorates considerably.</p>
<div class="figure" id="pone-0000113-g002"><div class="img"><a name="pone-0000113-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000113" data-uri="info:doi/10.1371/journal.pone.0000113.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000113.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000113.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000113.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000113.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000113.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000113.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000113.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000113.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Alignment independence of the <em>Fhom Estimator.</em>
                  </span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>HomoloGene homologous protein pairs were divided into five groups according to the identity level of their alignment. The organisms used are human, mouse, fruit fly and C. elegans. The figure reveals that the fraction of homologs estimation (Fhom) is applicable for both closely-related and distantly-related protein pairs.</p>
<span>doi:10.1371/journal.pone.0000113.g002</span></div>

<h4>Robustness of the <em>Fhom Estimator</em>
            </h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In order to test the dynamic range of <em>Fhom</em>, we estimated the HomoloGene Human/Mouse <em>Fhom</em> for a start. Next, we artificially added random protein pairs, while maintaining the original prevalence of signal peptides. <a href="#pone-0000113-g003">Figure 3</a> revealed that <em>Fhom</em> estimates well the real fraction of homologs for fractions between 0.01 and 1. The regression curve supports that the estimator always underestimates the real fraction of homologs, and that the <em>Fhom Estimator</em> has a wide dynamic range of homology.</p>
<div class="figure" id="pone-0000113-g003"><div class="img"><a name="pone-0000113-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000113" data-uri="info:doi/10.1371/journal.pone.0000113.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000113.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000113.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000113.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000113.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000113.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000113.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000113.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000113.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Robustness of the <em>Fhom Estimator.</em>
                  </span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>
                     <em>Fhom</em> was estimated to be 94% for HomoloGene human/mouse pairs. Next, random protein pairs were added according to the original signal peptide prevalence of these organisms. A linear regression curve and the coefficient of determination (R<sup>2</sup>) confirm that the <em>Fhom Estimator</em> has a wide dynamic range, and is robust to variation in signal-to-noise ratio.</p>
<span>doi:10.1371/journal.pone.0000113.g003</span></div>

<h4>Sample size effect</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The accuracy of the method can be impaired when small sets of protein alignments are being assessed. To evaluate the sample size effect on the accuracy of the Fhom Estimator, we aligned the Swiss-Prot protein sequences of fruit fly and human using Blastp, with expectation value (<em>E-Value</em>)&lt;100 cutoff. Signal peptides were removed from the protein sequences prior to the alignment. The results were sorted by their <em>E-Value</em>, and divided into 10 equal sized subsets of ~54,400 pairs. A subset that had <em>E-Value</em>s between 0.052 and 1.7 was chosen arbitrarily, divided into the four subgroups, and was estimated to comprise 52% homologs. Next, we used the proportions of these observed subgroups to simulate the observed distributions for 9 different sample sizes ranging from 10,000 to 75. Each simulation was repeated 1000 times. For each sample size, the average <em>Fhom</em>, the <em>Fhom</em> standard deviation, and the average size of the scaling subgroup were calculated. To further validate the results, more subgroups of fruit fly/human alignments, having <em>Fhom</em> estimation between 83% and 10%, were used as a model for additional simulation. The size of the scaling subgroup directly affects the susceptibility of the method to statistical noise. It is more informative than the total sample size, which ignores possible differences in the protein feature prevalence. <a href="#pone-0000113-g004">Figure 4</a> shows two simulation results for two sets of protein pairs having 52% <em>Fhom</em> and 10% <em>Fhom</em>, and their power regression curves. In both curves the standard deviation behaves like an N<sup>−½</sup> function, where N is the size of the scaling subgroup. However, as the <em>Fhom</em> of the subgroup decreases, the coefficient multiplying the N<sup>−½</sup> increases. Thus, to be on the safe side, it is recommended to make an upper bound to the standard deviation estimation by using the formula derived from a subgroup with very low <em>Fhom</em>:<a name="pone.0000113.e004" id="pone.0000113.e004"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.e004&amp;representation=PNG" class="inline-graphic"><span class="note">4</span></span><br>Where <em>S</em> is the size of the scaling subgroup, and <em>STD</em> is the standard deviation. If <em>STD</em> of less than 0.05 is required, the scaling subgroup should be larger than 55. If <em>STD</em> may be as large as 0.1, a scaling subgroup as small as 14 is enough.</p>
<div class="figure" id="pone-0000113-g004"><div class="img"><a name="pone-0000113-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000113" data-uri="info:doi/10.1371/journal.pone.0000113.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000113.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000113.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000113.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000113.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000113.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000113.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000113.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000113.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000113.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Size effect of scaling subgroup on the <em>Fhom Estimator</em> accuracy.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Two sets of protein pairs of fruit fly and human Swiss-Prot proteins were estimated to have <em>Fhom</em> of 52% and 10%. Both alignment sets were used separately for the sample size simulation. Each set of pairs was divided into the observed subgroups. The proportions of the observed subgroups were used for simulating observed distributions of 9 different sample sizes ranging from 10,000 to 75. Each simulation was repeated 1000 times, and the average <em>Fhom, Fhom</em> standard deviation, and the average size of the scaling subgroup were taken. The power regression curves of the 52% homologs (lower left curve) and the 10% homologs (upper right curve) describe the correlation between the mean size of the scaling subgroup and the standard deviation of <em>Fhom</em>, with high coefficients of determination.</p>
<span>doi:10.1371/journal.pone.0000113.g004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have developed a computational method for estimating the fraction of homologs in a set of protein pairs. The method is simple and robust. The method is based on the assumption that homologous proteins either both have or lack a protein feature. For proteome-wide analyses, the presence of signal peptide, which tends to be conserved among homologs (<a href="#pone-0000113-g001">Figure 1</a>), was found to be a feature suitable for the method (<a href="#pone-0000113-g002">Figure 2</a>). For more focused analyses (e.g. sub-clustering of a protein family) a different feature may be more suitable.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>For each pairwise alignment, sequence alignment methods calculate the <em>E-Value</em>, i.e. the number of different alignments that are expected to occur in a database search by chance, with scores equivalent to or better than the score of that specific alignment. This is an important measure, and indeed, alignments of most closely-related protein sequences have very low <em>E-Value</em>s. This way, it is possible to distinguish between closely-related homologous proteins and other protein pairs.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>When looking for distantly-related protein sequences, a relatively high threshold on the <em>E-Value</em> should be used. This, of course, adds noise to the results, i.e. randomly paired proteins will mistakenly be recognized as homologs. The <em>Fhom Estimator</em>, which is independent of the alignment itself, enables estimating the amount of noise added, and is therefore beneficial for the <em>E-Value</em> threshold fine tuning. Furthermore, we found that the <em>Fhom Estimator</em> works well in cases of high as well as low signal to noise ratio (<a href="#pone-0000113-g003">Figure 3</a>).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>An alignment <em>E-Value</em> depends on database size and composition, so it is impractical to define a universal cutoff, above which most data consists of random alignment pairs. The <em>Fhom Estimator</em> helps determine a specific <em>E-Value</em> cutoff for a particular set of alignment hits, above which the number of homologs is negligible. Therefore, the <em>Fhom Estimator</em> may be beneficial to assess sequence and structural alignment methods and fine tune their parameters.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In Kim &amp; Kliger <a href="#pone.0000113-Kim1">[16]</a>, we used the <em>Fhom Estimator</em> for the verification of a new scoring function (CTS) that facilitates the identification of viral piracy events <a href="#pone.0000113-Finlay1">[20]</a>. A set of viral/human protein sequence alignments having low identity levels (10% to 35%) was divided into two subsets according to their CTS scores. The <em>Fhom Estimator</em> revealed that the subset of high CTS scores was enriched in homologs in comparison with the low-CTS-score subset.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In estimating <em>Fhom</em>, <a href="#pone.0000113.e001">Eq. (1)</a> is used for calculating the expected random distribution. The calculation is based on the fraction of proteins in the query and subject datasets possessing the protein feature X. It should be verified that the homology search method in use pairs non homologs as expected by chance, and not in a bias manner. In Kim &amp; Kliger <a href="#pone.0000113-Kim1">[16]</a> such a verification was performed for Blastp. Proteins of dsDNA viruses were paired with human proteins using Blastp. A subset of pairs having an <em>E value</em>&gt;5, which is considered statistically insignificant homology, was used as the set of non homologs protein pairs. We verified that Blastp pairing follows the expected random distribution, by comparing the fraction of pairs having the same signal peptide prediction (YY+NN subgroups divided by the total number of pairs) in the E value&gt;5 subset, to the fraction expected by chance. Indeed, those fractions were similar. Hence, random pairing by Blastp, did not introduce a significant bias to the expected random distribution. Since we confirmed in Kim &amp; Kliger that there is no bias in non homologs pairing by Blastp, we did not repeat this verification procedure in the current study. In general, to avoid random pairing bias, we recommend removing the protein feature sequences from the proteins' sequences, before executing the homology search. In our studies, we discarded the signal peptide sequences before running Blastp between the protein sets.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>It is noteworthy that paralogs and orthologs are handled differently. Orthologous proteins are generally believed to have the same function in different organisms. Therefore, it is quite reasonable to assume that orthologous proteins will either both have or lack a signal peptide. However, the situation is more complex for paralogs. For example, there are intracellular and extracellular proteases that probably derived from the same protein ancestor. Therefore, choosing the presence of signal peptide as the feature for <em>Fhom</em> estimation might be problematic when dealing with distantly related paralogs. In summary, matching the right feature to the data is a crucial step for obtaining reliable results. In this study, pairs of paralogs had only a minute effect of further underestimating <em>Fhom</em> (data not shown), hence, they were included in all analyses.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>An additional analysis of RefSeq protein sequences was performed. RefSeq proteins can be divided to manually inspected proteins and merely predicted proteins, which are assigned ‘NP_’ and ‘XP_’ accession prefixes, respectively. We found that the <em>Fhom</em> estimate of human/mouse XP/XP homologs (57%) is much lower than the <em>Fhom</em> estimate of NP/NP homologs (94%). Thus, the reliability of XP proteins in human and mouse is questioned. While in newly sequenced genomes most proteins are predicted and were not yet curated, in well-curated genomes (e.g. human, mouse, fruit fly and C. elegans genomes) we assume that many of the remaining predicted XP proteins might not represent real proteins (e.g. false products of pseudo-genes or fused genes). For that reason, we used only NP proteins of well-curated genomes in this study.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>In conclusion, the <em>Fhom Estimator</em> is a simple and beneficial method to analyze the results of protein alignments. It can be used in quality assessments of homology search methods, which may share any level of sequence similarity, as being commonly done in bioinformatic analysis of proteins.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Data preparation</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>
               <em>Homo sapiens</em> (human), <em>Mus musculus</em> (mouse), <em>Drosophila melanogaster</em> (fruit fly), and <em>Caenorhabditis elegans</em> curated reference protein sequences (RefSeq build 11) were downloaded from NCBI's ftp web site <a href="#pone.0000113-Pruitt1">[21]</a>, <a href="#pone.0000113-Wheeler1">[22]</a>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Swissprot <a href="#pone.0000113-Boeckmann1">[9]</a> protein sequences of human and fruit fly were downloaded from NCBI Blast databases ftp web site (<a href="ftp://ftp.ncbi.nih.gov/blast/db/">ftp://ftp.ncbi.nih.gov/blast/db/</a>) on 21-June-06.</p>


<h4>Defining Homologs</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>HomoloGene triplets database of NCBI (Build 43.1) was used as the standard of truth for homologous proteins. HomoloGene databases contain pairs of homologous proteins from studied eukaryotic organisms that have a complete sequenced genome. In HomoloGene triplets database, each pair belongs to a group of at least three homologous proteins, and is considered highly reliable <a href="#pone.0000113-Wheeler1">[22]</a>.</p>


<h4>Signal peptide prediction</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Predicting whether a protein has a signal peptide was performed using the SignalP 3.0 prediction tool <a href="#pone.0000113-Bendtsen1">[18]</a>. SignalP consists of two predictors - a neural network (NN) based predictor, and a hidden Markov model (HMM) based predictor. Proteins having contradicting predictions assigned by the two predictors were discarded.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are thankful to I. Myslyuk, A. Feiglin, E. Schreiber, and A. Novik for critically reading the manuscript, for helpful suggestions, and fruitful discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YK AA EK. Performed the experiments: AA EK. Analyzed the data: YK AA EK. Wrote the paper: YK AA EK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000113-Holm1" id="pone.0000113-Holm1"></a>Holm L, Sander C (1996) Mapping the protein universe. Science  273: 595–603.  <ul class="find" data-citedArticleID="922366" data-doi="10.1126/science.273.5275.595"><li><a href="http://dx.doi.org/10.1126/science.273.5275.595" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+the+protein+universe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+the+protein+universe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000113-Hadley1" id="pone.0000113-Hadley1"></a>Hadley C, Jones DT (1999) A systematic comparison of protein structure classifications: SCOP, CATH and FSSP. Structure  7: 1099–1112.  <ul class="find" data-citedArticleID="922364" data-doi="10.1016/s0969-2126(99)80177-4"><li><a href="http://dx.doi.org/10.1016/s0969-2126(99)80177-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+systematic+comparison+of+protein+structure+classifications%3A+SCOP%2C+CATH+and+FSSP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+systematic+comparison+of+protein+structure+classifications%3A+SCOP%2C+CATH+and+FSSP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000113-Murzin1" id="pone.0000113-Murzin1"></a>Murzin AG, Brenner SE, Hubbard T, Chothia C (1995) SCOP: a structural classification of proteins database for the investigation of sequences and structures. J Mol Biol  247: 536–540.  <ul class="find" data-citedArticleID="922374" data-doi="10.1016/s0022-2836(05)80134-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80134-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SCOP%3A+a+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SCOP%3A+a+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000113-Orengo1" id="pone.0000113-Orengo1"></a>Orengo CA, Taylor WR (1996) SSAP: sequential structure alignment program for protein structure comparison. Methods Enzymol  266: 617–635.  <ul class="find" data-citedArticleID="922378" data-doi="10.1016/s0076-6879(96)66038-8"><li><a href="http://dx.doi.org/10.1016/s0076-6879(96)66038-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SSAP%3A+sequential+structure+alignment+program+for+protein+structure+comparison." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SSAP%3A+sequential+structure+alignment+program+for+protein+structure+comparison.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000113-Holm2" id="pone.0000113-Holm2"></a>Holm L, Sander C (1994) Searching protein structure databases has come of age. Proteins: Structure, Function, and Genetics  19: 165–173.  <ul class="find" data-citedArticleID="922368" data-doi="10.1002/prot.340190302"><li><a href="http://dx.doi.org/10.1002/prot.340190302" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Searching+protein+structure+databases+has+come+of+age." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Searching+protein+structure+databases+has+come+of+age.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000113-Holm3" id="pone.0000113-Holm3"></a>Holm L, Sander C (1993) Protein structure comparison by alignment of distance matrices. J Mol Biol  233: 123–138.  <ul class="find" data-citedArticleID="922370" data-doi="10.1006/jmbi.1993.1489"><li><a href="http://dx.doi.org/10.1006/jmbi.1993.1489" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+structure+comparison+by+alignment+of+distance+matrices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+structure+comparison+by+alignment+of+distance+matrices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000113-Alexandrov1" id="pone.0000113-Alexandrov1"></a>Alexandrov NN, Fischer D (1996) Analysis of topological and nontopological structural similarities in the PDB: new examples with old structures. Proteins  25: 354–365.  <ul class="find" data-citedArticleID="922346" data-doi="10.1002/(sici)1097-0134(199607)25:3<354::aid-prot7&gt;3.3.co;2-w"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0134(199607)25:3<354::aid-prot7&gt;3.3.co;2-w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+topological+and+nontopological+structural+similarities+in+the+PDB%3A+new+examples+with+old+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+topological+and+nontopological+structural+similarities+in+the+PDB%3A+new+examples+with+old+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000113-Berman1" id="pone.0000113-Berman1"></a>Berman HM, Bhat TN, Bourne PE, Feng Z, Gilliland G, et al.  (2000) The Protein Data Bank and the challenge of structural genomics. Nat Struct Biol  7: Suppl957–959.  <ul class="find" data-citedArticleID="922354" data-doi="10.1038/80734"><li><a href="http://dx.doi.org/10.1038/80734" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Protein+Data+Bank+and+the+challenge+of+structural+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Protein+Data+Bank+and+the+challenge+of+structural+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000113-Boeckmann1" id="pone.0000113-Boeckmann1"></a>Boeckmann B, Bairoch A, Apweiler R, Blatter MC, Estreicher A, et al.  (2003) The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003. Nucleic Acids Res  31: 365–370.  <ul class="find" data-citedArticleID="922356" data-doi="10.1093/nar/gkg095"><li><a href="http://dx.doi.org/10.1093/nar/gkg095" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+SWISS-PROT+protein+knowledgebase+and+its+supplement+TrEMBL+in+2003." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+SWISS-PROT+protein+knowledgebase+and+its+supplement+TrEMBL+in+2003.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000113-Altschul1" id="pone.0000113-Altschul1"></a>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. J Mol Biol  215: 403–410.  <ul class="find" data-citedArticleID="922348" data-doi="10.1016/s0022-2836(05)80360-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80360-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Basic+local+alignment+search+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Basic+local+alignment+search+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000113-Pearson1" id="pone.0000113-Pearson1"></a>Pearson WR (1990) Rapid and sensitive sequence comparison with FASTP and FASTA. Methods Enzymol  183: 63–98.  <ul class="find" data-citedArticleID="922380" data-doi="10.1016/0076-6879(90)83007-v"><li><a href="http://dx.doi.org/10.1016/0076-6879(90)83007-v" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+and+sensitive+sequence+comparison+with+FASTP+and+FASTA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+and+sensitive+sequence+comparison+with+FASTP+and+FASTA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000113-Altschul2" id="pone.0000113-Altschul2"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="922350" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000113-Gribskov1" id="pone.0000113-Gribskov1"></a>Gribskov M, McLachlan AD, Eisenberg D (1987) Profile analysis: detection of distantly related proteins. Proc Natl Acad Sci U S A  84: 4355–4358.  <ul class="find" data-citedArticleID="922362" data-doi="10.1073/pnas.84.13.4355"><li><a href="http://dx.doi.org/10.1073/pnas.84.13.4355" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Profile+analysis%3A+detection+of+distantly+related+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Profile+analysis%3A+detection+of+distantly+related+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000113-FischelGhodsian1" id="pone.0000113-FischelGhodsian1"></a>Fischel-Ghodsian F, Mathiowitz G, Smith TF (1990) Alignment of protein sequences using secondary structure: a modified dynamic programming method. Protein Eng  3: 577–581.  <ul class="find" data-citedArticleID="922360" data-doi="10.1093/protein/3.7.577"><li><a href="http://dx.doi.org/10.1093/protein/3.7.577" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alignment+of+protein+sequences+using+secondary+structure%3A+a+modified+dynamic+programming+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alignment+of+protein+sequences+using+secondary+structure%3A+a+modified+dynamic+programming+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000113-Rost1" id="pone.0000113-Rost1"></a>Rost B, Schneider R, Sander C (1997) Protein fold recognition by prediction-based threading. J Mol Biol  270: 471–480.  <ul class="find" data-citedArticleID="922384" data-doi="10.1006/jmbi.1997.1101"><li><a href="http://dx.doi.org/10.1006/jmbi.1997.1101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+fold+recognition+by+prediction-based+threading." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+fold+recognition+by+prediction-based+threading.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000113-Kim1" id="pone.0000113-Kim1"></a>Kim E, Kliger Y (2005) Discovering hidden viral piracy. Bioinformatics  21: 4216–4222.  <ul class="find" data-citedArticleID="922372" data-doi="10.1093/bioinformatics/bti706"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovering+hidden+viral+piracy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovering+hidden+viral+piracy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000113-Zhang1" id="pone.0000113-Zhang1"></a>Zhang Z, Henzel WJ (2004) Signal peptide prediction based on analysis of experimentally verified cleavage sites. Protein Sci  13: 2819–2824.  <ul class="find" data-citedArticleID="922388" data-doi="10.1110/ps.04682504"><li><a href="http://dx.doi.org/10.1110/ps.04682504" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signal+peptide+prediction+based+on+analysis+of+experimentally+verified+cleavage+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signal+peptide+prediction+based+on+analysis+of+experimentally+verified+cleavage+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000113-Bendtsen1" id="pone.0000113-Bendtsen1"></a>Bendtsen JD, Nielsen H, von Heijne G, Brunak S (2004) Improved prediction of signal peptides: SignalP 3.0. J Mol Biol  340: 783–795.  <ul class="find" data-citedArticleID="922352" data-doi="10.1016/j.jmb.2004.05.028"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.05.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+prediction+of+signal+peptides%3A+SignalP+3.0." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+prediction+of+signal+peptides%3A+SignalP+3.0.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000113-Nickel1" id="pone.0000113-Nickel1"></a>Nickel W (2003) The mystery of nonclassical protein secretion. A current view on cargo proteins and potential export routes. Eur J Biochem  270: 2109–2119.  <ul class="find" data-citedArticleID="922376" data-doi="10.1046/j.1432-1033.2003.03577.x"><li><a href="http://dx.doi.org/10.1046/j.1432-1033.2003.03577.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mystery+of+nonclassical+protein+secretion.+A+current+view+on+cargo+proteins+and+potential+export+routes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mystery+of+nonclassical+protein+secretion.+A+current+view+on+cargo+proteins+and+potential+export+routes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000113-Finlay1" id="pone.0000113-Finlay1"></a>Finlay BB, McFadden G (2006) Anti-immunology: evasion of the host immune system by bacterial and viral pathogens. Cell  124: 767–782.  <ul class="find" data-citedArticleID="922358" data-doi="10.1016/j.cell.2006.01.034"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.01.034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anti-immunology%3A+evasion+of+the+host+immune+system+by+bacterial+and+viral+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anti-immunology%3A+evasion+of+the+host+immune+system+by+bacterial+and+viral+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000113-Pruitt1" id="pone.0000113-Pruitt1"></a>Pruitt KD, Tatusova T, Maglott DR (2005) NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. Nucleic Acids Res  33: D501–504.  <ul class="find" data-citedArticleID="922382" data-doi="10.1093/nar/gkl842"><li><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NCBI+Reference+Sequence+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NCBI+Reference+Sequence+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000113-Wheeler1" id="pone.0000113-Wheeler1"></a>Wheeler DL, Barrett T, Benson DA, Bryant SH, Canese K, et al.  (2006) Database resources of the National Center for Biotechnology Information. Nucleic Acids Res  34: D173–180.  <ul class="find" data-citedArticleID="922386" data-doi="10.1093/nar/gkj158"><li><a href="http://dx.doi.org/10.1093/nar/gkj158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Database+resources+of+the+National+Center+for+Biotechnology+Information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Database+resources+of+the+National+Center+for+Biotechnology+Information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.XML" value="65728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.PDF" value="126136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.e001.PNG" value="2703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.e001.TIF" value="15224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.e002.PNG" value="4445"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.e002.TIF" value="46888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.e003.PNG" value="861"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.e003.TIF" value="4868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g001.PNG_L" value="130223"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g001.PNG_M" value="22376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g001.PNG_S" value="2041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g001.TIF" value="221198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g001.PNG_I" value="9555"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g002.PNG_L" value="7679"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g002.PNG_M" value="13613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g002.PNG_S" value="1723"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g002.TIF" value="27158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g002.PNG_I" value="6633"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g003.PNG_L" value="14321"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g003.PNG_M" value="13546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g003.PNG_S" value="1825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g003.TIF" value="29016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g003.PNG_I" value="7348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.e004.PNG" value="762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.e004.TIF" value="5180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g004.PNG_L" value="13626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g004.PNG_M" value="17345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g004.PNG_S" value="2135"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g004.TIF" value="25290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000113.g004.PNG_I" value="8900"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000113&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000113" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000113&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000113&volume=&issue=&title=Towards Alignment Independent Quantitative Assessment of Homology Detection&author_name=Avihay%20Apatoff%2C%20Eddo%20Kim%2C%20Yossef%20Kliger&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000113" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000113" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000113" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000113&amp;t=Towards%20Alignment%20Independent%20Quantitative%20Assessment%20of%20Homology%20Detection" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000113&title=Towards%20Alignment%20Independent%20Quantitative%20Assessment%20of%20Homology%20Detection&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000113&amp;title=Towards%20Alignment%20Independent%20Quantitative%20Assessment%20of%20Homology%20Detection" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000113" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000113&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Towards Alignment Independent Quantitative Assessment of Homology Detection';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000113';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Towards%20Alignment%20Independent%20Quantitative%20Assessment%20of%20Homology%20Detection http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000113" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000113" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Database+searching%22" title="Search for articles in the subject area:'Database searching'"><div class="flagText">Database searching</div></a>
              <div data-categoryid="32021" data-articleid="23064"
                   data-categoryname="Database searching"
                   class="flagImage" title="Flag 'Database searching' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drosophila%22" title="Search for articles in the subject area:'Drosophila'"><div class="flagText">Drosophila</div></a>
              <div data-categoryid="42801" data-articleid="23064"
                   data-categoryname="Drosophila"
                   class="flagImage" title="Flag 'Drosophila' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure%22" title="Search for articles in the subject area:'Protein structure'"><div class="flagText">Protein structure</div></a>
              <div data-categoryid="31877" data-articleid="23064"
                   data-categoryname="Protein structure"
                   class="flagImage" title="Flag 'Protein structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure+databases%22" title="Search for articles in the subject area:'Protein structure databases'"><div class="flagText">Protein structure databases</div></a>
              <div data-categoryid="33455" data-articleid="23064"
                   data-categoryname="Protein structure databases"
                   class="flagImage" title="Flag 'Protein structure databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="23064"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="23064"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+similarity+searching%22" title="Search for articles in the subject area:'Sequence similarity searching'"><div class="flagText">Sequence similarity searching</div></a>
              <div data-categoryid="35177" data-articleid="23064"
                   data-categoryname="Sequence similarity searching"
                   class="flagImage" title="Flag 'Sequence similarity searching' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Signal+peptides%22" title="Search for articles in the subject area:'Signal peptides'"><div class="flagText">Signal peptides</div></a>
              <div data-categoryid="19701" data-articleid="23064"
                   data-categoryname="Signal peptides"
                   class="flagImage" title="Flag 'Signal peptides' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3431'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6637'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3989&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=4035">Still valid?</a><br>Posted by dsingh</p>
      <p><a href="/annotation/listThread.action?root=7501">long standing issue</a><br>Posted by dsingh</p>
      <p><a href="/annotation/listThread.action?root=4497">Equal Contribution Statement</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
