

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Demographic Histories of ERV-K in Humans, Chimpanzees and Rhesus Monkeys</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001026"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001026" />

    <meta name="citation_title" content="Demographic Histories of ERV-K in Humans, Chimpanzees and Rhesus Monkeys"/>
    <meta itemprop="name" content="Demographic Histories of ERV-K in Humans, Chimpanzees and Rhesus Monkeys"/>

      <meta name="citation_author" content="Camila M. Romano"/>
            <meta name="citation_author_institution" content="Laboratory of Molecular Evolution and Bioinformatics, Department of Microbiology, Biomedical Sciences Institute–ICBII, University of São Paulo, Brazil"/>
      <meta name="citation_author" content="Fernando L. de Melo"/>
            <meta name="citation_author_institution" content="Laboratory of Molecular Evolution and Bioinformatics, Department of Microbiology, Biomedical Sciences Institute–ICBII, University of São Paulo, Brazil"/>
      <meta name="citation_author" content="Marco Aurelio B. Corsini"/>
            <meta name="citation_author_institution" content="Laboratory of Molecular Evolution and Bioinformatics, Department of Microbiology, Biomedical Sciences Institute–ICBII, University of São Paulo, Brazil"/>
      <meta name="citation_author" content="Edward C. Holmes"/>
            <meta name="citation_author_institution" content="Mueller Laboratory, Department of Biology, Center for Infectious Disease Dynamics, The Pennsylvania State University, University Park, Pennsylvania, United States of America"/>
            <meta name="citation_author_institution" content="Fogarty International Center, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Paolo M. de A. Zanotto"/>
            <meta name="citation_author_institution" content="Laboratory of Molecular Evolution and Bioinformatics, Department of Microbiology, Biomedical Sciences Institute–ICBII, University of São Paulo, Brazil"/>

    <meta name="citation_date" content="2007/10/10"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001026.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1026"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome.; citation_author=ES Lander; citation_author=LM Linton; citation_author=B Birren; citation_author=C Nusbaum; citation_author=MC Zody; citation_journal_title=Nature; citation_volume=409; citation_number=1; citation_pages=860-921; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Retrotransposons, endogenous retroviruses and the evolution of retroelements.; citation_author=JD Boeke; citation_author=JP Stoye; citation_number=2; citation_pages=343-435; citation_date=1997; citation_publisher=Cold Spring Harbor Laboratory Press: Cold Spring Harbor; " />
      <meta name="citation_reference" content="citation_title=Human trophoblast and choriocarcinoma expression of the growth factor pleiotrophin attributable to germ-line insertion of an endogenous retrovirus.; citation_author=AM Schulte; citation_author=S Lai; citation_author=A Kurtz; citation_author=F Czubayko; citation_author=AT Riegel; citation_author=A Wellstein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=3; citation_pages=14759-14764; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Sure facts, speculations, and open questions about the evolution of transposable element copy number.; citation_author=SV Nuzhdin; citation_journal_title=Genetica.; citation_volume=107; citation_number=4; citation_pages=129-137; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The first steps of transposable elements invasion: parasitic strategy vs. genetic drift.; citation_author=A Le Rouzic; citation_author=P Capy; citation_journal_title=Genetics; citation_volume=169; citation_number=5; citation_pages=1033-1043; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Human-specific integrations of the HERV-K endogenous retrovirus family.; citation_author=P Medstrand; citation_author=DL Mager; citation_journal_title=J Virol; citation_volume=72; citation_number=6; citation_pages=9782-9787; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Identification of human endogenous retroviruses with complex mRNA expression and particle formation.; citation_author=R Lower; citation_author=K Boller; citation_author=B Hasenmaier; citation_author=C Korbmacher; citation_author=N Muller-Lantzsch; citation_author=J Lower; citation_author=R Kurth; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=90; citation_number=7; citation_pages=4480-4484; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Longterm reinfection of the human genome by endogenous retroviruses.; citation_author=R Belshaw; citation_author=V Pereira; citation_author=A Katzourakis; citation_author=G Talbot; citation_author=J Paces; citation_author=A Burt; citation_author=M Tristem; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=101; citation_number=8; citation_pages=4894-4899; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Tempo and mode of ERV-K evolution in human and chimpanzee genomes.; citation_author=CM Romano; citation_author=RF Ramalho; citation_author=PM Zanotto; citation_journal_title=Arch Virol; citation_volume=151; citation_number=9; citation_pages=2215-2228; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biogeographic analysis of the Uhu and LOA elements in the Hawaiian Drosophila.; citation_author=RG Wisotzkey; citation_author=I Felger; citation_author=JA Hunt; citation_journal_title=Chromosoma; citation_volume=106; citation_number=10; citation_pages=465-477; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genomic demography: a life-history analysis of transposable element evolution.; citation_author=DE Promislow; citation_author=IK Jordan; citation_author=JF McDonald; citation_journal_title=Proc Biol Sci; citation_volume=266; citation_number=11; citation_pages=1555-1560; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The dynamics of transposable elements in structured populations.; citation_author=G Deceliere; citation_author=S Charles; citation_author=C Biemont; citation_journal_title=Genetics; citation_volume=169; citation_number=12; citation_pages=467-474; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Eukaryote Genome in Development and Evolution.; citation_author=B John; citation_author=G Miklos; citation_number=13; citation_pages=416; citation_date=1988; citation_publisher=Allen&Unwin; " />
      <meta name="citation_reference" content="citation_title=Chromosome Rearrangements and the Evolution of Genome Structuring and Adaptability.; citation_author=A Crombach; citation_author=P Hogeweg; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=14; citation_pages=1130-1139; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Selfish genes, the phenotype paradigm and genome evolution.; citation_author=WF Doolittle; citation_author=C Sapienza; citation_journal_title=Nature; citation_volume=284; citation_number=15; citation_pages=601-603; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Dynamics of transposable elements under the selection model.; citation_author=A Tsitrone; citation_author=S Charles; citation_author=C Biemont; citation_journal_title=Genet Res; citation_volume=74; citation_number=16; citation_pages=159-164; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Retroelement distributions in the human genome: variations associated with age and proximity to genes.; citation_author=P Medstrand; citation_author=LN van de Lagemaat; citation_author=DL Mager; citation_journal_title=Genome Res; citation_volume=12; citation_number=17; citation_pages=1483-1495; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification of a new, abundant superfamily of mammalian LTR-transposons.; citation_author=AFA Smit; citation_journal_title=Nucleic Acids Res; citation_volume=21; citation_number=18; citation_pages=1863-1872; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Population genetics of an expanding family of mobile genetic elements.; citation_author=T Ohta; citation_journal_title=Genetics; citation_volume=113; citation_number=19; citation_pages=145-159; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Ancient Adaptive Evolution of the Primate Antiviral DNA-Editing Enzyme APOBEC3G.; citation_author=LS Sawyer; citation_author=M Emerman; citation_author=HS Malik; citation_journal_title=Plos Biol; citation_volume=2; citation_number=20; citation_pages=1278-1285; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genomewide screening reveals high levels of insertional polymorphism in the human endogenous retrovirus family HERV-K (HML2): implications for present-day activity.; citation_author=R Belshaw; citation_author=AL Dawson; citation_author=J Woolven-Allen; citation_author=J Redding; citation_author=A Burt; citation_author=M Tristem; citation_journal_title=J. Virol; citation_volume=79; citation_number=21; citation_pages=12507-12514; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Rates of nucleotide substitution in primates and rodents and the generation-time effect hypothesis.; citation_author=WH Li; citation_author=DL Ellsworth; citation_author=J Krushkal; citation_author=BH Chang; citation_author=D Hewett-Emmett; citation_journal_title=Mol Phylogenet Evol; citation_volume=5; citation_number=22; citation_pages=182-187; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Variable molecular clocks in hominoids.; citation_author=N Elango; citation_author=JW Thomas; citation_author=SV Yi; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=23; citation_pages=1370-1375; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Body size, metabolic rate, generation time, and the molecular clock.; citation_author=AP Martin; citation_author=SR Palumbi; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=90; citation_number=24; citation_pages=4087-4091; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Sequences of primate insulin genes support the hypothesis of a slower rate of molecular evolution in humans and apes than in monkeys.; citation_author=S Seino; citation_author=GI Bell; citation_author=WH Li; citation_journal_title=Mol Biol Evol; citation_volume=9; citation_number=25; citation_pages=193-203; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Population Bottlenecks as a Potential Major Shaping Force of Human Genome Architecture.; citation_author=A Gherman; citation_author=PE Chen; citation_author=T Teslovich; citation_author=P Stankiewicz; citation_author=M Withers; citation_journal_title=PLoS Genetics In press.; citation_number=26; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Why sex and recombination?; citation_author=NH Barton; citation_author=B Charlesworth; citation_journal_title=Science; citation_volume=25; citation_number=27; citation_pages=1986-1990; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Fixation probability and time in subdivided populations.; citation_author=MC Whitlock; citation_journal_title=Genetics; citation_volume=164; citation_number=28; citation_pages=767-779; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Asian primate classifcation.; citation_author=D Brandon-Jones; citation_author=AA Eudey; citation_author=T Geissmann; citation_author=CP Groves; citation_author=DJ Melnick; citation_journal_title=Int J Primatol; citation_volume=25; citation_number=29; citation_pages=97-164; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Macroevolutionary inferences from primate phylogeny.; citation_author=A Purvis; citation_author=S Nee; citation_author=PH Harvey; citation_journal_title=Proc Biol Sci; citation_volume=260; citation_number=30; citation_pages=329-333; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Great ape DNA sequences reveal a reduced diversity and an expansion in humans.; citation_author=H Kaessmann; citation_author=V Wiebe; citation_author=G Weiss; citation_author=S Paabo; citation_journal_title=Nat Genet; citation_volume=27; citation_number=31; citation_pages=155-156; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=BLAT- the BLAST-like alignment tool.; citation_author=WJ Kent; citation_journal_title=Genome Res; citation_volume=12; citation_number=32; citation_pages=656-664; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput.; citation_author=RC Edgar; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=33; citation_pages=1792-1797; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=BlastAlign: a program that uses blast to align problematic nucleotide sequences.; citation_author=R Belshaw; citation_author=A Katzourakis; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=34; citation_pages=122-123; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=PAUP*: Phylogenetic analysis using parsimony (and other methods) 4.0. Sunderland (MA) Sinauer Associates.; citation_author=DL Swofford; citation_number=35; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=MODELTEST: testing the model of DNA substitution.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=36; citation_pages=817-818; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Constructing primate phylogenies from ancient retrovirus sequences.; citation_author=WE Johnson; citation_author=JM Coffin; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=96; citation_number=37; citation_pages=10254-10260; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=MEGA2: Molecular evolutionary genetics analysis software.; citation_author=S Kumar; citation_author=K Tamura; citation_author=IB Jakobsen; citation_author=M Nei; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=38; citation_pages=1244-1245; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=BEAST v1.0.; citation_author=AJ Drummond; citation_author=A Rambaut; citation_number=39; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001026" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Demographic Histories of ERV-K in Humans, Chimpanzees and Rhesus Monkeys"/>
    <meta name="twitter:description" content="We detected 19 complete endogenous retroviruses of the K family in the genome of rhesus monkey (Macaca mulatta; RhERV-K) and 12 full length elements in the genome of the common chimpanzee (Pan troglodytes; CERV-K). These sequences were compared with 55 human HERV-K and 20 CERV-K reported previously, producing a total data set of 106 full-length ERV-K genomes. Overall, 61% of the human elements compared to 21% of the chimpanzee and 47% of rhesus elements had estimated integration times less than 4.5 million years before present (MYBP), with an average integration times of 7.8 MYBP, 13.4 MYBP and 10.3 MYBP for HERV-K, CERV-K and RhERV-K, respectively. By excluding those ERV-K sequences generated by chromosomal duplication, we used 63 of the 106 elements to compare the population dynamics of ERV-K among species. This analysis indicated that both HERV-K and RhERV-K had similar demographic histories, including markedly smaller effective population sizes, compared to CERV-K. We propose that these differing ERV-K dynamics reflect underlying differences in the evolutionary ecology of the host species, such that host ecology and demography represent important determinants of ERV-K dynamics."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001026.g003"/>

  <meta property="og:title" content="Demographic Histories of ERV-K in Humans, Chimpanzees and Rhesus Monkeys" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001026" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6507'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5755'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6819&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001026">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001026" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001026&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Demographic Histories of ERV-K in Humans, Chimpanzees and Rhesus Monkeys
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Camila M. Romano, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Molecular Evolution and Bioinformatics, Department of Microbiology, Biomedical Sciences Institute–ICBII, University of São Paulo, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fernando L. de Melo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Molecular Evolution and Bioinformatics, Department of Microbiology, Biomedical Sciences Institute–ICBII, University of São Paulo, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marco Aurelio B. Corsini, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Molecular Evolution and Bioinformatics, Department of Microbiology, Biomedical Sciences Institute–ICBII, University of São Paulo, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Edward C. Holmes, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Mueller Laboratory, Department of Biology, Center for Infectious Disease Dynamics, The Pennsylvania State University, University Park, Pennsylvania, United States of America, 
                  Fogarty International Center, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paolo M. de A. Zanotto
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pzanotto@usp.br.">pzanotto@usp.br.</a></p>

                <p>Affiliation:
                  Laboratory of Molecular Evolution and Bioinformatics, Department of Microbiology, Biomedical Sciences Institute–ICBII, University of São Paulo, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 10, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001026</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001026" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001026" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001026" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001026" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001026-g001" data-doi="info:doi/10.1371/journal.pone.0001026" data-uri="info:doi/10.1371/journal.pone.0001026.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001026.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001026-g002" data-doi="info:doi/10.1371/journal.pone.0001026" data-uri="info:doi/10.1371/journal.pone.0001026.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001026.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001026-g003" data-doi="info:doi/10.1371/journal.pone.0001026" data-uri="info:doi/10.1371/journal.pone.0001026.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001026.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001026">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001026" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>We detected 19 complete endogenous retroviruses of the K family in the genome of rhesus monkey (<em>Macaca mulatta;</em> RhERV-K) and 12 full length elements in the genome of the common chimpanzee (<em>Pan troglodytes</em>; CERV-K). These sequences were compared with 55 human HERV-K and 20 CERV-K reported previously, producing a total data set of 106 full-length ERV-K genomes. Overall, 61% of the human elements compared to 21% of the chimpanzee and 47% of rhesus elements had estimated integration times less than 4.5 million years before present (MYBP), with an average integration times of 7.8 MYBP, 13.4 MYBP and 10.3 MYBP for HERV-K, CERV-K and RhERV-K, respectively. By excluding those ERV-K sequences generated by chromosomal duplication, we used 63 of the 106 elements to compare the population dynamics of ERV-K among species. This analysis indicated that both HERV-K and RhERV-K had similar demographic histories, including markedly smaller effective population sizes, compared to CERV-K. We propose that these differing ERV-K dynamics reflect underlying differences in the evolutionary ecology of the host species, such that host ecology and demography represent important determinants of ERV-K dynamics.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Romano CM, de Melo FL, Corsini MAB, Holmes EC, Zanotto PMdA (2007) Demographic Histories of ERV-K in Humans, Chimpanzees and Rhesus Monkeys. PLoS ONE 2(10):
          e1026.
            doi:10.1371/journal.pone.0001026</p><p><strong>Academic Editor: </strong>Jean Carr, Institute of Human Virology, United States of America</p><p><strong>Received:</strong> August 17, 2007; <strong>Accepted:</strong> September 21, 2007; <strong>Published:</strong> October 10, 2007</p><p><strong>Copyright:</strong> © 2007 Romano et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was funded by VGDN program FAPESP (00/04205-6). CMR and FLM hold a CAPES doctorate fellowship (DO) and PMAZ holds a CNPq PQ Research Scholarship.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A considerable proportion (~45%) of the primate genome consists of copies of mobile genetic elements <a href="#pone.0001026-Lander1">[1]</a>. These elements are divided into two classes based on their mechanism of mobilization: those involving an RNA intermediate, or those that transpose via DNA excision and reintegration into the host genome (transposons). The via-RNA elements (Class I) are represented by retrotransposons and endogenous retroviruses (ERVs). ERVs are relics of ancient viral infection events in the germ line, followed by long-term vertical transmission. They can increase in copy number by means of active replication (in <em>cis</em> or in <em>trans</em>) or by chromosomal duplication <a href="#pone.0001026-Boeke1">[2]</a>, and represent about 3% of all transposable elements (TE) related sequences. Proviral activity may occur over long periods of time until they become inactivated by loss of promoter functionality due to host chromosome rearrangements, insertions, deletions or point mutations. Because the LTRs (long terminal repeats) of proviruses carry transcriptional regulatory elements, such as promoters and enhancers, its likely that the insertion of a provirus, or only its LTRs, near genes or regulatory regions will be detrimental to host fitness <a href="#pone.0001026-Schulte1">[3]</a>–<a href="#pone.0001026-LeRouzic1">[5]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The human ERV-K (HERV-K) family includes some of the most active retroviral elements in human genome <a href="#pone.0001026-Medstrand1">[6]</a>, <a href="#pone.0001026-Lower1">[7]</a>. Although most of the proviral copies of ERV-K in the genome are inactive, some show evidence of past positive selection at the <em>env</em> gene <a href="#pone.0001026-Belshaw1">[8]</a>, <a href="#pone.0001026-Romano1">[9]</a>. ERVs, as well the other retroelements, can invade the host genome due to transposition bursts <a href="#pone.0001026-Wisotzkey1">[10]</a>, counteracted by host-driven excision and purging <a href="#pone.0001026-Promislow1">[11]</a>, <a href="#pone.0001026-Deceliere1">[12]</a>. This dynamical process plays an important role in the evolution of host genomes as a consequence of the rearrangement, transduction and inactivation of genes <a href="#pone.0001026-John1">[13]</a>, <a href="#pone.0001026-Crombach1">[14]</a>. In the absence of any host selection pressure to inhibit the fixation and replication, ERV copy number could increase to extreme levels <a href="#pone.0001026-Doolittle1">[15]</a>, <a href="#pone.0001026-Tsitrone1">[16]</a>. However, the preferential integration of LTR elements in gene-poor regions and in an antisense orientation suggests that these elements are routinely purged from gene-rich regions by purifying selection <a href="#pone.0001026-Medstrand2">[17]</a>, <a href="#pone.0001026-Smit1">[18]</a>, which is perhaps a major force restricting ERV copy number. Consequently, determining the mechanisms of transposition control, inactivation and purging are central to the understanding of proviral dynamics in the host genome <a href="#pone.0001026-LeRouzic1">[5]</a>, <a href="#pone.0001026-Deceliere1">[12]</a>, <a href="#pone.0001026-Tsitrone1">[16]</a>, <a href="#pone.0001026-Ohta1">[19]</a>, <a href="#pone.0001026-Sawyer1">[20]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>To explore the evolutionary dynamics of ERVs in more detail, we determined the demographic history of ERV-K in three primates: human (<em>Homo sapiens</em>), common chimpanzee (<em>Pan troglodytes</em>) and rhesus monkey (<em>Macaca mulatta</em>). Our findings suggest that host population size and ecology plays a major role in shaping patterns of ERV-K evolution in primates.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>ERV-K Characterization and Phylogeny</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Nineteen complete proviruses, designated RhERV-K, were found in the rhesus monkey (<em>Macaca mulatta</em>) draft assembly genome (<a href="#pone.0001026.s001">Text S1</a>). Similarly, 12 new elements in <em>Pan troglodytes</em> (CERV-K) genome were found (<a href="#pone.0001026.s001">Text S1</a>) and compared to 20 CERV and 55 human HERV-K previously reported, producing a total of 106 ERV-K genomes. Three RhERV-K proviruses had almost identical LTR, indicative of recent integration and therefore of possible recent activity. Conversely, RhERV-K19 had highly divergent 5′ and 3′ LTR that could not be aligned due to several insertion-deletion events (indels), indicating that the estimated integration time of about 46 MYBP (see below) may be misleading. As no RhERV-K orthologue was closely related to those in either the chimpanzee or human genomes, all RhERV-K proviruses appear to have arisen by active transposition rather than chromosomal duplication. In contrast, <em>Pan</em> and <em>Homo</em> share several ERV-K, and exhibit many closely related elements that most likely originated by chromosomal duplications and rearrangement events (<em>e.g.</em>, CERV-K32, CERV-K31, CERV-K34; CERV-K26, 27 and 28 on the Y chromosome).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>A phylogenetic tree (<a href="#pone-0001026-g001">Fig. 1</a>) for a 4130 bp alignment from the conserved domains (the Partial data set) shared by 106 ERV-K genomes, had a topology congruent to those obtained previously for both ERV-K genomic fragments <a href="#pone.0001026-Belshaw2">[21]</a> and complete genomes <a href="#pone.0001026-Romano1">[9]</a>. To facilitate data presentation, tree components involving two or more adjacent lineages in the same host, were collapsed and were indicated as colored wedges in <a href="#pone-0001026-g001">Figure 1</a>. Human and chimpanzee appear to share a large number of ERV-K as indicated by at least 18 <em>Pan</em>-<em>Homo</em> sister taxa pairs at the tips of the tree. Interestingly, 13 RhERV-K clustered in a distinct group, radiating within Group O <a href="#pone.0001026-Romano1">[9]</a>, represented by the largest wedge in <a href="#pone-0001026-g001">Figure 1</a>. The other six RhERV-K genomes fell in four distinct lineages within Group I. None of the six lineages of RhERV-K shared recent orthologues with <em>Homo</em> or <em>Pan</em>, and only three (RhERV-K3, RhERV-K8 and RhERV-K19) were possibly integrated into the common ancestor of all three primates. This notion was further supported by the fact that no traces of ERV-K were found in the orthologous chromosomal regions in human and chimpanzee, where we would expect to find the descendents of RhERV-K3 and the eight ERVs that predate the separation of all three lineages. Conversely, fragments of LTR and <em>gag</em> sequences were found on chromosome 9 of both human and chimpanzee at the integration site of RhERV-K19, suggesting that they the ERV-K viruses have been purged from these genomes.</p>
<div class="figure" id="pone-0001026-g001"><div class="img"><a name="pone-0001026-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001026.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001026" data-uri="info:doi/10.1371/journal.pone.0001026.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001026.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001026.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001026.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001026.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001026.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001026.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001026.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001026.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001026.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Maximum likelihood tree for 106 ERV-K genomes.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>ML tree for 4130 bp of shared (Partial) sequences from ERV-K genomes of human (<em>Homo sapiens</em>) (55 sequences), common chimpanzee (<em>Pan troglodytes</em>) (32 sequences) and, rhesus monkey (<em>Macaca mulatta</em>) (19 sequences). Thirteen RhERV-K (shown as a collapsed red wedge in the tree) arise from a single ancient branch in Group O, while four other deep lineages radiate independently from within Group I. No RhERV-K was observed in Group N. The HERV-K, CERV-K and RhERV-K elements are shown by black, green and red branches, respectively. Duplications of the same provirus appear in colored collapsed wedges.</p>
<span>doi:10.1371/journal.pone.0001026.g001</span></div>

<h4>ERV-K Population Dynamics</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Bayesian skyline plots, reflecting changes in effective population size through time, were inferred for 31 HERV-K found in <em>Homo sapiens</em> (<a href="#pone-0001026-g002">Figure 2a</a>), 21 CERV-K found in <em>Pan troglodytes</em> (<a href="#pone-0001026-g002">Figure 2b</a>) and 19 RhERV-K found in <em>Macaca mulatta</em> (<a href="#pone-0001026-g002">Figure 2c</a>). The high ESS values (near 1000) indicated that the sample sizes, although small, were sufficient for convergence during parameter estimation. Strikingly different plots were seen in the three species, and with a particularly complex dynamic in humans, although both <em>Homo</em> and rhesus ERV-K experienced an initial burst in ERV copy number followed by a significant reduction in the number of complete proviruses after 20 MYBP. In contrast, CERV-K experienced an apparently flat dynamic after a significantly (around ten-fold) higher growth in numbers up until 15 MYBP, and had very much larger effective population sizes than the other two species. Finally, and perhaps most notable of all, during the last 5 MY there was an increase in ERV-K numbers in the human genome, possibly caused by the radiation of the newer human elements (Group N) <a href="#pone.0001026-Romano1">[9]</a>.</p>
<div class="figure" id="pone-0001026-g002"><div class="img"><a name="pone-0001026-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001026.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001026" data-uri="info:doi/10.1371/journal.pone.0001026.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001026.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001026.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001026.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001026.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001026.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001026.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001026.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001026.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001026.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Bayesian skyline plots of three primate ERV-K.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A) human (<em>Homo sapiens</em>) ERV-K (HERV-K), B) common chimpanzee (<em>Pan troglodytes</em>) ERV-K (CERV-K) and, C) rhesus monkey (<em>Macaca mulatta</em>) ERV-K (RhERV-K). Time is presented in million years from the present and effective population sizes multiplied by the generation time (<em>Ne</em>.g) are presented in a logarithmic scale on the y-axis. The bold line represents the median estimate for each species while the 95% HPDs (reflecting statistical uncertainty) are shaded. Integration times for all ERV-K were estimated using a rate of 3.3×10<sup>−9</sup> substitutions per site per year (s/s/y).</p>
<span>doi:10.1371/journal.pone.0001026.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>One possible reason for differences in the dynamics observed is heterogeneity in evolutionary rate among the primate hosts. In particular, it has been established that the rate of evolution in humans suffered a slowdown relative to that of the chimpanzee <a href="#pone.0001026-Li1">[22]</a>, <a href="#pone.0001026-Elango1">[23]</a>, with an approximately two-fold reduction in evolutionary rate relative to Old World monkeys and chimpanzee <a href="#pone.0001026-Martin1">[24]</a>. Therefore, based on previous estimates on the differences among substitution rates for the species considered here <a href="#pone.0001026-Li1">[22]</a>–<a href="#pone.0001026-Seino1">[25]</a>, we repeated our analysis of population dynamics using dates of integration based on rates of 5.94×10<sup>−9</sup> s/s/y for CERV-K and 6.93×10<sup>−9</sup> s/s/y for RhERV-K (with human still at 3.3×10<sup>−9</sup> s/s/y). The comparisons shown in <a href="#pone-0001026-g003">Figure 3</a> clearly indicate that the differences in population dynamics are not changed qualitatively by host rate heterogeneity. Hence, these results indicate that the evolutionary rate of the host genome is a less important determinant of differences among ERV-Ks than host population dynamics.</p>
<div class="figure" id="pone-0001026-g003"><div class="img"><a name="pone-0001026-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001026.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001026" data-uri="info:doi/10.1371/journal.pone.0001026.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001026.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001026.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001026.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001026.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001026.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001026.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001026.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001026.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001026.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Comparative population dynamic of ERV-K.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>The figure shows the superimposed median values of <em>Ne</em>.g through time taken from the Bayesian skyline plots for the primate species. Time is presented in million years from the present and effective population time generation time (<em>Ne</em>.g) sizes are given in a linear scale without the 95% HPD values shown in</p>
<span>doi:10.1371/journal.pone.0001026.g003</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>ERV-K in primates</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Herein, we described several new complete ERV-K elements in the genomes of the common chimpanzee (<em>Pan troglodytes</em>) and rhesus monkey (<em>Macaca mulatta</em>) and compared them to those found in humans. We show, for the first time, that the demographic history of the host may be a major factor determining the dynamics of an endogenous retrovirus. Despite the draft quality of the rhesus genome assembly, we found many complete proviruses that have a marked similarity in their fluctuating demographic history to that of humans, with both these species distinct from that observed in the chimpanzee (<a href="#pone-0001026-g003">Figure 3</a>). In particular, we found a distinct group of 13 RhERV-K, which diverged around 12 MYBP that were absent in both humans and chimpanzees. Moreover, there was no evidence of RhERV-K amplification caused by chromosomal duplication. On the other hand, both <em>Homo</em> and <em>Pan</em> had many closely related ERV-K, some of which had several duplicated counterparts. Important differences between CERV-K and HERV-K were also evident. For example, four CERV-K where found on the Y chromosome, three of which were found within an apparently low complexity repeat region, as a consequence of DNA duplication (<em>i.e.</em>, CERV-K “Y chromosome quartet” in <a href="#pone-0001026-g001">Figure 1</a>). Interestingly, the human Y chromosome has the same repeat region without traces of retrovirus integration, suggesting that elements have been purged along the human lineage.</p>


<h4>Demography and Dynamics of ERV-K</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The Bayesian skyline plots revealed fluctuating ERV-K population sizes in all three primate species, although with a relatively large sampling error (<a href="#pone-0001026-g002">Figures 2</a> and <a href="#pone-0001026-g003">3</a>). Although HERV-K and RhERV-K had similarly complex skyline plots, it is striking that the latter exhibited a signal of rapid population growth up until 25 MYBP, coinciding with both fossil and molecular data for the radiation of the <em>Cercopithecidae</em>. Conversely, the signal for the initial burst for HERV-K and CERV-K occurred at approximately 17–18 MYBP, followed by a reduction of the number copy of the elements, first in <em>Homo</em> and then in <em>Pan</em>. This growth signature, common to all three primates, may reflect some of the shared history of ERV-K colonization of Catarrhines from the Oligocene (30 MYBP) to Miocene (20 MYBP).</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The rate of retrovirus-driven transposition and excision is evidently insufficient to explain their permanence and integrity. Since, in finite populations, size fluctuations have a drastic impact on genome architecture, ERV-K numbers in time must ultimately depend on host population dynamics <a href="#pone.0001026-Gherman1">[26]</a>. Nevertheless, the mechanisms of purging <a href="#pone.0001026-LeRouzic1">[5]</a>, reduction of transposition efficiency by APOBEC <a href="#pone.0001026-Sawyer1">[20]</a>, excision <a href="#pone.0001026-Promislow1">[11]</a> and stabilization under weak selection <a href="#pone.0001026-Tsitrone1">[16]</a>, or the balance between host migration rates and ERV-K transposition rates <a href="#pone.0001026-Deceliere1">[12]</a>, as well as synergistic epistasis among integrated ERV-K <a href="#pone.0001026-Barton1">[27]</a>, may have played a role in preventing the continued growth of the three ERV populations towards the present from 10 to 20 MYBP. The loss of cladogenetic signal from older ERV-K lineages could therefore be a consequence of a strong host-driven purging that is more evident in the <em>Homo</em> and rhesus lineages. This agrees with our finding that 61% percent of the human elements compared to 21% of the chimpanzee and 47% of rhesus had estimated integration times less than 4.5 MYBP.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Since all partial sequences we dismissed were likely generated by incomplete purging events it is evident that our approach has underestimated the loss of ERV proviruses. Nevertheless, by investigating complete genomes were able to estimate integration times, which is only possible when both LTRs are present. The Bayesian skyline plot for HERV-K showed a conspicuous population bottleneck in the last 17 MY, comprising a significant reduction in complete proviral numbers up until 4MYBP, after which a cladogenetic burst within ERVs from Group N <a href="#pone.0001026-Romano1">[9]</a> took place. This population bottleneck could indicate a recent loss of ancient signal in the hominids, since the difference in the skyline signatures predates the split of <em>Homo</em> and <em>Pan</em>. Possibly, bottlenecks since the Plio-Pleistocene may have played an important role, facilitating both the loss of unfixed alleles and the fixation of deleterious ones by genetic drift <a href="#pone.0001026-Whitlock1">[28]</a>, and which could help explain the observed complex dynamics of HERV-K. Intriguingly, the time frame for a “re-colonization” of the hominids by Group N HERV-K at around 1.5 MYBP coincides with the emergence of human-specific life history traits <a href="#pone.0001026-Elango1">[23]</a>, such as increased generation time.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>Unlike the single extant species of the genus <em>Homo</em>, the genus <em>Macaca</em> is represented by a large number of species (19) despite being a relative young clade <a href="#pone.0001026-BrandonJones1">[29]</a>, <a href="#pone.0001026-Purvis1">[30]</a>. <em>Macaca mulatta</em> originated from a <em>fascicularis</em>-like ancestor around 2.5 MYBP and became widely distributed within a relatively short period, from western India to the eastern coast of China. The strong decrease in RhERV-K population size (<a href="#pone-0001026-g003">Figure 3</a>) coincided with the emergence of the genus <em>Macaca</em> around 10 MYBP, which is one of the most specious groups among <em>Cercopithecidae</em> <a href="#pone.0001026-BrandonJones1">[29]</a>. The impact of the intense cladogenesis in <em>Cercopithecidae</em> on RhERV-K dynamics remains to be addressed. Nevertheless, the elevated dispersal of both <em>Homo</em> and <em>Macaca</em> compared to <em>Pan</em> may be an important factor that could explain the similarities in the demographic histories of HERV-K and RhERV-K.</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>Unlike HERV-K and RhERV-K, the chimpanzee ERV-K demographic signal was characterized by a far larger effective population size. Assuming that host dynamics impacts on ERV-K numbers, the recent flat curve of <em>Pan</em> skyline after 6 MYBP agrees with the lack of evidence for severe bottlenecks in the <em>Pan</em> lineage and a 3.2 times larger effective ancestral population size <a href="#pone.0001026-Kaessmann1">[31]</a>. The latter could have facilitated the maintenance of a higher number of integrated elements observed in the chimpanzee genome, because of a weaker effect of genetic drift, although the wide HPD values caution against over-interpretation.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>ERV Screening, Phylogenetic Inference and Sequence Analysis</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We screened the genomes of <em>Pan troglodytes</em> (build 2 v.1) and the <em>Macaca mulatta</em> draft assembly (v.1) by BLAT search <a href="#pone.0001026-Kent1">[32]</a> using complete ERV-K genomes as a query. This analysis revealed 116 complete retroviral genome sequences, 78 of which were previously reported and are deposited in GenBank as DQ112093-DQ112156. These sequences were then aligned with both MUSCLE <a href="#pone.0001026-Edgar1">[33]</a> and BlastAlign <a href="#pone.0001026-Belshaw3">[34]</a>. To minimize systematic errors caused by insertion/deletion events (indels), for which there is no adequate model of evolution, we also constructed a 4130 bp data set using gene coding regions only (designated as the ‘Partial’ data set from now on). Maximum likelihood (ML) trees of these data were then inferred by PAUP v.4.0b <a href="#pone.0001026-Swofford1">[35]</a>, using the TVM+Γ evolutionary model as determined by MODELTEST 3.7 <a href="#pone.0001026-Posada1">[36]</a>. Tree topologies were evaluated from an initial neighbor joining tree (NJ), using a heuristic search approach that implemented successively branch-swapping methods: (<em>i</em>) tree bisection-reconnection (TBR) branch-swapping, (<em>ii</em>) subtree pruning-regrafting (SPR) and, (<em>iii</em>) nearest-neighbor interchange (NNI). The integration time (T) of each provirus was estimated using the relation <em>T</em> = <em>d</em>/2<em>r</em>, where <em>d</em> is the genetic distance between 5′ and 3′ LTR and <em>r</em> is the rate of nucleotide substitution per site. Errors in <em>T</em> where assumed to be the transformed values of the standard errors for <em>d</em> estimations. Because rates of substitution for ERVs can range from 1.5–5×10<sup>−9</sup> substitutions per site per year (s/s/y), <a href="#pone.0001026-Boeke1">[2]</a>, <a href="#pone.0001026-Johnson1">[37]</a> we used an average rate of 3.3×10<sup>−9</sup>. Finally, pairwise distances among ERVs were calculated using Tamura-Nei model available in MEGA2 <a href="#pone.0001026-Kumar1">[38]</a>.</p>


<h4>Population Dynamics</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For this analysis we constructed a smaller 2530 bp region from the Partial dataset that contained those nucleotide sites shared by all proviruses. Proviruses that were both sister taxa (<em>i.e.</em> adjacent in the phylogenetic tree) and had similar flanking regions up to10 kb away from the insertion locus, were excluded from the demographic analyses as they most likely to have arisen by chromosomal duplication. Following this screening, 19 RhERV-K, 21 CERV-K and 31 HERV-K sequences were available for analysis. Rates of nucleotide substitution per site, the time to the Most Recent Common Ancestor TMRCA and the demographic history of each ERV-K group (<em>Homo</em>, <em>P. troglodytes</em> and <em>M. mulatta</em>) were estimated using a Bayesian Markov Chain Monte Carlo (MCMC) method available in the BEAST package <a href="#pone.0001026-Drummond1">[39]</a>. For this analysis, dates of integration based on LTR distances were used as “sampling dates” since, once integrated, ERV-K proviruses would behave as if they were “frozen” in the genome and so evolve at rates equivalent to those of host DNA. Such LTR-based “sampling dating” is justified since the differences in the rates of evolution of exogenous retroviruses are six orders of magnitude higher than those of their endogenous (“frozen”) counterparts. Because LTR comparisons indicate that ERV-K have been integrating into primate DNA for at least 40 million years, the assumption that all ERV-K were sampled today would entail a far greater systematic error. To infer population dynamics of the different primate ERV-K we fitted sequence data to the demographic models available in the Bayesian coalescent method in BEAST. In particular we used the Bayesian skyline plot to depict changes in effective population size through time (N<sub>e</sub>.<em>g</em>, where N<sub>e</sub> is the effective population size and <em>g</em> the generation time). For this analysis we used the HKY+Γ model of nucleotide substitution under the assumption of a relaxed (uncorrelated exponential) molecular clock. The HKY+Γ was consistently the best-supported model in MODELTEST when the data from each species were analyzed separately. In all cases chain lengths of 40–50 million were sufficient to obtain Effective Sample Sizes (ESS) greater than 100.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001026"></div><a name="pone.0001026.s001" id="pone.0001026.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001026.s001">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">GenBank information</p>
<p class="siDoi">doi:10.1371/journal.pone.0001026.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PZ. Performed the experiments: CR. Analyzed the data: EH PZ CR Fd MC. Contributed reagents/materials/analysis tools: MC. Wrote the paper: EH PZ CR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001026-Lander1" id="pone.0001026-Lander1"></a>Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, et al.  (2001) Initial sequencing and analysis of the human genome. Nature  409: 860–921.  <ul class="find" data-citedArticleID="1004556"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Initial+sequencing+and+analysis+of+the+human+genome.&amp;auth=&amp;atitle=Initial+sequencing+and+analysis+of+the+human+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+sequencing+and+analysis+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+sequencing+and+analysis+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001026-Boeke1" id="pone.0001026-Boeke1"></a>Boeke JD, Stoye JP (1997)  Retrotransposons, endogenous retroviruses and the evolution of retroelements. In: Coffin JM, Hughes SH, Varmus EH, editors. Retroviruses. NY: Cold Spring Harbor Laboratory Press: Cold Spring Harbor. pp. 343–435.  <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0001026-Schulte1" id="pone.0001026-Schulte1"></a>Schulte AM, Lai S, Kurtz A, Czubayko F, Riegel AT, Wellstein A (1996) Human trophoblast and choriocarcinoma expression of the growth factor pleiotrophin attributable to germ-line insertion of an endogenous retrovirus. Proc Natl Acad Sci U S A  93: 14759–14764.  <ul class="find" data-citedArticleID="1004584" data-doi="10.1073/pnas.93.25.14759"><li><a href="http://dx.doi.org/10.1073/pnas.93.25.14759" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+trophoblast+and+choriocarcinoma+expression+of+the+growth+factor+pleiotrophin+attributable+to+germ-line+insertion+of+an+endogenous+retrovirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+trophoblast+and+choriocarcinoma+expression+of+the+growth+factor+pleiotrophin+attributable+to+germ-line+insertion+of+an+endogenous+retrovirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001026-Nuzhdin1" id="pone.0001026-Nuzhdin1"></a>Nuzhdin SV (1999) Sure facts, speculations, and open questions about the evolution of transposable element copy number. Genetica.  107: 129–137.  <ul class="find" data-citedArticleID="1004570" data-doi="10.1007/978-94-011-4156-7_15"><li><a href="http://dx.doi.org/10.1007/978-94-011-4156-7_15" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sure+facts%2C+speculations%2C+and+open+questions+about+the+evolution+of+transposable+element+copy+number." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sure+facts%2C+speculations%2C+and+open+questions+about+the+evolution+of+transposable+element+copy+number.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001026-LeRouzic1" id="pone.0001026-LeRouzic1"></a>Le Rouzic A, Capy P (2005) The first steps of transposable elements invasion: parasitic strategy vs. genetic drift. Genetics  169: 1033–1043.  <ul class="find" data-citedArticleID="1004558" data-doi="10.1534/genetics.104.031211"><li><a href="http://dx.doi.org/10.1534/genetics.104.031211" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+first+steps+of+transposable+elements+invasion%3A+parasitic+strategy+vs.+genetic+drift." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+first+steps+of+transposable+elements+invasion%3A+parasitic+strategy+vs.+genetic+drift.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001026-Medstrand1" id="pone.0001026-Medstrand1"></a>Medstrand P, Mager DL (1998) Human-specific integrations of the HERV-K endogenous retrovirus family. J Virol  72: 9782–9787.  <ul class="find" data-citedArticleID="1004566"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Human-specific+integrations+of+the+HERV-K+endogenous+retrovirus+family.&amp;auth=&amp;atitle=Human-specific+integrations+of+the+HERV-K+endogenous+retrovirus+family." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human-specific+integrations+of+the+HERV-K+endogenous+retrovirus+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human-specific+integrations+of+the+HERV-K+endogenous+retrovirus+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001026-Lower1" id="pone.0001026-Lower1"></a>Lower R, Boller K, Hasenmaier B, Korbmacher C, Muller-Lantzsch N, Lower J, Kurth R (1993) Identification of human endogenous retroviruses with complex mRNA expression and particle formation. Proc Natl Acad Sci USA  90: 4480–4484.  <ul class="find" data-citedArticleID="1004562" data-doi="10.1073/pnas.90.10.4480"><li><a href="http://dx.doi.org/10.1073/pnas.90.10.4480" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+human+endogenous+retroviruses+with+complex+mRNA+expression+and+particle+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+human+endogenous+retroviruses+with+complex+mRNA+expression+and+particle+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001026-Belshaw1" id="pone.0001026-Belshaw1"></a>Belshaw R, Pereira V, Katzourakis A, Talbot G, Paces J, Burt A, Tristem M (2004) Longterm reinfection of the human genome by endogenous retroviruses. Proc Natl Acad Sci USA  101: 4894–4899.  <ul class="find" data-citedArticleID="1004522" data-doi="10.1073/pnas.0307800101"><li><a href="http://dx.doi.org/10.1073/pnas.0307800101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Longterm+reinfection+of+the+human+genome+by+endogenous+retroviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Longterm+reinfection+of+the+human+genome+by+endogenous+retroviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001026-Romano1" id="pone.0001026-Romano1"></a>Romano CM, Ramalho RF, Zanotto PM (2006) Tempo and mode of ERV-K evolution in human and chimpanzee genomes. Arch Virol  151: 2215–2228.  <ul class="find" data-citedArticleID="1004580" data-doi="10.1007/s00705-006-0792-1"><li><a href="http://dx.doi.org/10.1007/s00705-006-0792-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tempo+and+mode+of+ERV-K+evolution+in+human+and+chimpanzee+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tempo+and+mode+of+ERV-K+evolution+in+human+and+chimpanzee+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001026-Wisotzkey1" id="pone.0001026-Wisotzkey1"></a>Wisotzkey RG, Felger I, Hunt JA (1997) Biogeographic analysis of the Uhu and LOA elements in the Hawaiian Drosophila. Chromosoma  106: 465–477.  <ul class="find" data-citedArticleID="1004596" data-doi="10.1007/s004120050268"><li><a href="http://dx.doi.org/10.1007/s004120050268" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biogeographic+analysis+of+the+Uhu+and+LOA+elements+in+the+Hawaiian+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biogeographic+analysis+of+the+Uhu+and+LOA+elements+in+the+Hawaiian+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001026-Promislow1" id="pone.0001026-Promislow1"></a>Promislow DE, Jordan IK, McDonald JF (1999) Genomic demography: a life-history analysis of transposable element evolution. Proc Biol Sci  266: 1555–1560.  <ul class="find" data-citedArticleID="1004576" data-doi="10.1098/rspb.1999.0815"><li><a href="http://dx.doi.org/10.1098/rspb.1999.0815" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+demography%3A+a+life-history+analysis+of+transposable+element+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+demography%3A+a+life-history+analysis+of+transposable+element+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001026-Deceliere1" id="pone.0001026-Deceliere1"></a>Deceliere G, Charles S, Biemont C (2005) The dynamics of transposable elements in structured populations. Genetics  169: 467–474.  <ul class="find" data-citedArticleID="1004534" data-doi="10.1534/genetics.104.032243"><li><a href="http://dx.doi.org/10.1534/genetics.104.032243" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dynamics+of+transposable+elements+in+structured+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dynamics+of+transposable+elements+in+structured+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001026-John1" id="pone.0001026-John1"></a>John B, Miklos G (1988) The Eukaryote Genome in Development and Evolution. London: Allen&amp;Unwin.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001026-Crombach1" id="pone.0001026-Crombach1"></a>Crombach A, Hogeweg P (2007) Chromosome Rearrangements and the Evolution of Genome Structuring and Adaptability. Mol Biol Evol  24: 1130–1139.  <ul class="find" data-citedArticleID="1004532" data-doi="10.1093/molbev/msm033"><li><a href="http://dx.doi.org/10.1093/molbev/msm033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome+Rearrangements+and+the+Evolution+of+Genome+Structuring+and+Adaptability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome+Rearrangements+and+the+Evolution+of+Genome+Structuring+and+Adaptability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001026-Doolittle1" id="pone.0001026-Doolittle1"></a>Doolittle WF, Sapienza C (1980) Selfish genes, the phenotype paradigm and genome evolution. Nature  284: 601–603.  <ul class="find" data-citedArticleID="1004536" data-doi="10.1038/284601a0"><li><a href="http://dx.doi.org/10.1038/284601a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selfish+genes%2C+the+phenotype+paradigm+and+genome+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selfish+genes%2C+the+phenotype+paradigm+and+genome+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001026-Tsitrone1" id="pone.0001026-Tsitrone1"></a>Tsitrone A, Charles S, Biemont C (1999) Dynamics of transposable elements under the selection model. Genet Res  74: 159–164.  <ul class="find" data-citedArticleID="1004592" data-doi="10.1017/s0016672399003997"><li><a href="http://dx.doi.org/10.1017/s0016672399003997" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamics+of+transposable+elements+under+the+selection+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamics+of+transposable+elements+under+the+selection+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001026-Medstrand2" id="pone.0001026-Medstrand2"></a>Medstrand P, van de Lagemaat LN, Mager DL (2002) Retroelement distributions in the human genome: variations associated with age and proximity to genes. Genome Res  12: 1483–1495.  <ul class="find" data-citedArticleID="1004568" data-doi="10.1101/gr.388902"><li><a href="http://dx.doi.org/10.1101/gr.388902" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroelement+distributions+in+the+human+genome%3A+variations+associated+with+age+and+proximity+to+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroelement+distributions+in+the+human+genome%3A+variations+associated+with+age+and+proximity+to+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001026-Smit1" id="pone.0001026-Smit1"></a>Smit AFA (1993) Identification of a new, abundant superfamily of mammalian LTR-transposons. Nucleic Acids Res  21: 1863–1872.  <ul class="find" data-citedArticleID="1004588" data-doi="10.1093/nar/21.8.1863"><li><a href="http://dx.doi.org/10.1093/nar/21.8.1863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+new%2C+abundant+superfamily+of+mammalian+LTR-transposons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+new%2C+abundant+superfamily+of+mammalian+LTR-transposons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001026-Ohta1" id="pone.0001026-Ohta1"></a>Ohta T (1986) Population genetics of an expanding family of mobile genetic elements. Genetics  113: 145–159.  <ul class="find" data-citedArticleID="1004572"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Population+genetics+of+an+expanding+family+of+mobile+genetic+elements.&amp;auth=&amp;atitle=Population+genetics+of+an+expanding+family+of+mobile+genetic+elements." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+genetics+of+an+expanding+family+of+mobile+genetic+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+genetics+of+an+expanding+family+of+mobile+genetic+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001026-Sawyer1" id="pone.0001026-Sawyer1"></a>Sawyer LS, Emerman M, Malik HS (2004) Ancient Adaptive Evolution of the Primate Antiviral DNA-Editing Enzyme APOBEC3G. Plos Biol  2: 1278–1285.  <ul class="find" data-citedArticleID="1004582" data-doi="10.1371/journal.pbio.0020275"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+Adaptive+Evolution+of+the+Primate+Antiviral+DNA-Editing+Enzyme+APOBEC3G." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+Adaptive+Evolution+of+the+Primate+Antiviral+DNA-Editing+Enzyme+APOBEC3G.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001026-Belshaw2" id="pone.0001026-Belshaw2"></a>Belshaw R, Dawson AL, Woolven-Allen J, Redding J, Burt A, Tristem M (2005) Genomewide screening reveals high levels of insertional polymorphism in the human endogenous retrovirus family HERV-K (HML2): implications for present-day activity. J. Virol  79: 12507–12514.  <ul class="find" data-citedArticleID="1004524" data-doi="10.1128/jvi.79.19.12507-12514.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.19.12507-12514.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomewide+screening+reveals+high+levels+of+insertional+polymorphism+in+the+human+endogenous+retrovirus+family+HERV-K+%28HML2%29%3A+implications+for+present-day+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomewide+screening+reveals+high+levels+of+insertional+polymorphism+in+the+human+endogenous+retrovirus+family+HERV-K+%28HML2%29%3A+implications+for+present-day+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001026-Li1" id="pone.0001026-Li1"></a>Li WH, Ellsworth DL, Krushkal J, Chang BH, Hewett-Emmett D (1996) Rates of nucleotide substitution in primates and rodents and the generation-time effect hypothesis. Mol Phylogenet Evol  5: 182–187.  <ul class="find" data-citedArticleID="1004560"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rates+of+nucleotide+substitution+in+primates+and+rodents+and+the+generation-time+effect+hypothesis.&amp;auth=&amp;atitle=Rates+of+nucleotide+substitution+in+primates+and+rodents+and+the+generation-time+effect+hypothesis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rates+of+nucleotide+substitution+in+primates+and+rodents+and+the+generation-time+effect+hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rates+of+nucleotide+substitution+in+primates+and+rodents+and+the+generation-time+effect+hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001026-Elango1" id="pone.0001026-Elango1"></a>Elango N, Thomas JW, Yi SV, NISC Comparative Sequencing program (2006) Variable molecular clocks in hominoids. Proc Natl Acad Sci USA  103: 1370–1375.  <ul class="find" data-citedArticleID="1004542" data-doi="10.1073/pnas.0510716103"><li><a href="http://dx.doi.org/10.1073/pnas.0510716103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variable+molecular+clocks+in+hominoids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variable+molecular+clocks+in+hominoids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001026-Martin1" id="pone.0001026-Martin1"></a>Martin AP, Palumbi SR (1993) Body size, metabolic rate, generation time, and the molecular clock. Proc Natl Acad Sci USA  90: 4087–4091.  <ul class="find" data-citedArticleID="1004564" data-doi="10.1073/pnas.90.9.4087"><li><a href="http://dx.doi.org/10.1073/pnas.90.9.4087" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Body+size%2C+metabolic+rate%2C+generation+time%2C+and+the+molecular+clock." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Body+size%2C+metabolic+rate%2C+generation+time%2C+and+the+molecular+clock.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001026-Seino1" id="pone.0001026-Seino1"></a>Seino S, Bell GI, Li WH (1992) Sequences of primate insulin genes support the hypothesis of a slower rate of molecular evolution in humans and apes than in monkeys. Mol Biol Evol  9: 193–203.  <ul class="find" data-citedArticleID="1004586"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sequences+of+primate+insulin+genes+support+the+hypothesis+of+a+slower+rate+of+molecular+evolution+in+humans+and+apes+than+in+monkeys.&amp;auth=&amp;atitle=Sequences+of+primate+insulin+genes+support+the+hypothesis+of+a+slower+rate+of+molecular+evolution+in+humans+and+apes+than+in+monkeys." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequences+of+primate+insulin+genes+support+the+hypothesis+of+a+slower+rate+of+molecular+evolution+in+humans+and+apes+than+in+monkeys." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequences+of+primate+insulin+genes+support+the+hypothesis+of+a+slower+rate+of+molecular+evolution+in+humans+and+apes+than+in+monkeys.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001026-Gherman1" id="pone.0001026-Gherman1"></a>Gherman A, Chen PE, Teslovich T, Stankiewicz P, Withers M, et al.  (2007) Population Bottlenecks as a Potential Major Shaping Force of Human Genome Architecture. PLoS Genetics In press..   <ul class="find" data-citedArticleID="1004544" data-doi="10.1371/journal.pgen.0030119.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030119.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+Bottlenecks+as+a+Potential+Major+Shaping+Force+of+Human+Genome+Architecture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+Bottlenecks+as+a+Potential+Major+Shaping+Force+of+Human+Genome+Architecture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001026-Barton1" id="pone.0001026-Barton1"></a>Barton NH, Charlesworth B (1998) Why sex and recombination? Science  25: 1986–1990.  <ul class="find" data-citedArticleID="1004520" data-doi="10.1126/science.281.5385.1986"><li><a href="http://dx.doi.org/10.1126/science.281.5385.1986" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+sex+and+recombination%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+sex+and+recombination%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001026-Whitlock1" id="pone.0001026-Whitlock1"></a>Whitlock MC (2003) Fixation probability and time in subdivided populations. Genetics  164: 767–779.  <ul class="find" data-citedArticleID="1004594"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fixation+probability+and+time+in+subdivided+populations.&amp;auth=&amp;atitle=Fixation+probability+and+time+in+subdivided+populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fixation+probability+and+time+in+subdivided+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fixation+probability+and+time+in+subdivided+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001026-BrandonJones1" id="pone.0001026-BrandonJones1"></a>Brandon-Jones D, Eudey AA, Geissmann T, Groves CP, Melnick DJ, et al.  (2004) Asian primate classifcation. Int J Primatol  25: 97–164.  <ul class="find" data-citedArticleID="1004530" data-doi="10.1023/b:ijop.0000014647.18720.32"><li><a href="http://dx.doi.org/10.1023/b:ijop.0000014647.18720.32" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Asian+primate+classifcation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Asian+primate+classifcation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001026-Purvis1" id="pone.0001026-Purvis1"></a>Purvis A, Nee S, Harvey PH (1995) Macroevolutionary inferences from primate phylogeny. Proc Biol Sci  260: 329–333.  <ul class="find" data-citedArticleID="1004578" data-doi="10.1098/rspb.1995.0100"><li><a href="http://dx.doi.org/10.1098/rspb.1995.0100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Macroevolutionary+inferences+from+primate+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Macroevolutionary+inferences+from+primate+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001026-Kaessmann1" id="pone.0001026-Kaessmann1"></a>Kaessmann H, Wiebe V, Weiss G, Paabo S (2001) Great ape DNA sequences reveal a reduced diversity and an expansion in humans. Nat Genet  27: 155–156.  <ul class="find" data-citedArticleID="1004550" data-doi="10.1038/84773"><li><a href="http://dx.doi.org/10.1038/84773" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Great+ape+DNA+sequences+reveal+a+reduced+diversity+and+an+expansion+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Great+ape+DNA+sequences+reveal+a+reduced+diversity+and+an+expansion+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001026-Kent1" id="pone.0001026-Kent1"></a>Kent WJ (2002) BLAT- the BLAST-like alignment tool. Genome Res  12: 656–664.  <ul class="find" data-citedArticleID="1004552" data-doi="10.1101/gr.229202."><li><a href="http://dx.doi.org/10.1101/gr.229202." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BLAT-+the+BLAST-like+alignment+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BLAT-+the+BLAST-like+alignment+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001026-Edgar1" id="pone.0001026-Edgar1"></a>Edgar RC (2004) MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res  32: 1792–1797.  <ul class="find" data-citedArticleID="1004540" data-doi="10.1093/nar/gkh340"><li><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001026-Belshaw3" id="pone.0001026-Belshaw3"></a>Belshaw R, Katzourakis A (2005) BlastAlign: a program that uses blast to align problematic nucleotide sequences. Bioinformatics  21: 122–123.  <ul class="find" data-citedArticleID="1004526" data-doi="10.1093/bioinformatics/bth459"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth459" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BlastAlign%3A+a+program+that+uses+blast+to+align+problematic+nucleotide+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BlastAlign%3A+a+program+that+uses+blast+to+align+problematic+nucleotide+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001026-Swofford1" id="pone.0001026-Swofford1"></a>Swofford DL (2002) PAUP*: Phylogenetic analysis using parsimony (and other methods) 4.0. Sunderland (MA) Sinauer Associates.  <ul class="find" data-citedArticleID="1004590" data-doi="10.1017/cbo9780511819049.010"><li><a href="http://dx.doi.org/10.1017/cbo9780511819049.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAUP*%3A+Phylogenetic+analysis+using+parsimony+%28and+other+methods%29+4.0.+Sunderland+%28MA%29+Sinauer+Associates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAUP*%3A+Phylogenetic+analysis+using+parsimony+%28and+other+methods%29+4.0.+Sunderland+%28MA%29+Sinauer+Associates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001026-Posada1" id="pone.0001026-Posada1"></a>Posada D, Crandall KA (1998) MODELTEST: testing the model of DNA substitution. Bioinformatics  14: 817–818.  <ul class="find" data-citedArticleID="1004574" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MODELTEST%3A+testing+the+model+of+DNA+substitution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MODELTEST%3A+testing+the+model+of+DNA+substitution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001026-Johnson1" id="pone.0001026-Johnson1"></a>Johnson WE, Coffin JM (1999) Constructing primate phylogenies from ancient retrovirus sequences. Proc Natl Acad Sci USA  96: 10254–10260.  <ul class="find" data-citedArticleID="1004548" data-doi="10.1073/pnas.96.18.10254"><li><a href="http://dx.doi.org/10.1073/pnas.96.18.10254" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Constructing+primate+phylogenies+from+ancient+retrovirus+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Constructing+primate+phylogenies+from+ancient+retrovirus+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001026-Kumar1" id="pone.0001026-Kumar1"></a>Kumar S, Tamura K, Jakobsen IB, Nei M (2001) MEGA2: Molecular evolutionary genetics analysis software. Bioinformatics  17: 1244–1245.  <ul class="find" data-citedArticleID="1004554" data-doi="10.1093/bioinformatics/17.12.1244"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.12.1244" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA2%3A+Molecular+evolutionary+genetics+analysis+software." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA2%3A+Molecular+evolutionary+genetics+analysis+software.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001026-Drummond1" id="pone.0001026-Drummond1"></a>Drummond AJ, Rambaut A (2003) BEAST v1.0.   Available: <a href="http://evolve.zoo.ox.ac.uk/beast/">http://evolve.zoo.ox.ac.uk/beast/</a>.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.XML" value="63167"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.PDF" value="145160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g001.PNG_L" value="82593"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g001.PNG_M" value="88169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g001.PNG_S" value="11848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g001.TIF" value="121402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g001.PNG_I" value="51697"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g002.PNG_L" value="88662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g002.PNG_M" value="105770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g002.PNG_S" value="13037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g002.TIF" value="95240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g002.PNG_I" value="105085"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g003.PNG_L" value="76527"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g003.PNG_M" value="104252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g003.PNG_S" value="8723"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g003.TIF" value="112210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.g003.PNG_I" value="35703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001026.s001.DOC" value="25600"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001026&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001026" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001026&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001026&volume=&issue=&title=Demographic Histories of ERV-K in Humans, Chimpanzees and Rhesus Monkeys&author_name=Camila%20M.%20Romano%2C%20Fernando%20L.%20de%20Melo%2C%20Marco%20Aurelio%20B.%20Corsini%2C%20Edward%20C.%20Holmes%2C%20Paolo%20M.%20de%20A.%20Zanotto&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001026" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001026" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001026" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001026&amp;t=Demographic%20Histories%20of%20ERV-K%20in%20Humans%2C%20Chimpanzees%20and%20Rhesus%20Monkeys" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001026&title=Demographic%20Histories%20of%20ERV-K%20in%20Humans%2C%20Chimpanzees%20and%20Rhesus%20Monkeys&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001026&amp;title=Demographic%20Histories%20of%20ERV-K%20in%20Humans%2C%20Chimpanzees%20and%20Rhesus%20Monkeys" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001026" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001026&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Demographic Histories of ERV-K in Humans, Chimpanzees and Rhesus Monkeys';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001026';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Demographic%20Histories%20of%20ERV-K%20in%20Humans%2C%20Chimpanzees%20and%20Rhesus%20Monkeys http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001026" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001026" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chimpanzees%22" title="Search for articles in the subject area:'Chimpanzees'"><div class="flagText">Chimpanzees</div></a>
              <div data-categoryid="43093" data-articleid="24886"
                   data-categoryname="Chimpanzees"
                   class="flagImage" title="Flag 'Chimpanzees' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24886"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="24886"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24886"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+groupings%22" title="Search for articles in the subject area:'Population groupings'"><div class="flagText">Population groupings</div></a>
              <div data-categoryid="40421" data-articleid="24886"
                   data-categoryname="Population groupings"
                   class="flagImage" title="Flag 'Population groupings' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Primates%22" title="Search for articles in the subject area:'Primates'"><div class="flagText">Primates</div></a>
              <div data-categoryid="42883" data-articleid="24886"
                   data-categoryname="Primates"
                   class="flagImage" title="Flag 'Primates' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Rhesus+monkeys%22" title="Search for articles in the subject area:'Rhesus monkeys'"><div class="flagText">Rhesus monkeys</div></a>
              <div data-categoryid="35211" data-articleid="24886"
                   data-categoryname="Rhesus monkeys"
                   class="flagImage" title="Flag 'Rhesus monkeys' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transposable+elements%22" title="Search for articles in the subject area:'Transposable elements'"><div class="flagText">Transposable elements</div></a>
              <div data-categoryid="17435" data-articleid="24886"
                   data-categoryname="Transposable elements"
                   class="flagImage" title="Flag 'Transposable elements' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6263'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1737'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6747&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
