

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Novel Motif Identified in Dependence Receptors</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000463"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000463" />

    <meta name="citation_title" content="A Novel Motif Identified in Dependence Receptors"/>
    <meta itemprop="name" content="A Novel Motif Identified in Dependence Receptors"/>

      <meta name="citation_author" content="Gabriel del Rio"/>
            <meta name="citation_author_institution" content="Buck Institute for Age Research, Novato, California, United States of America"/>
      <meta name="citation_author" content="Darci J. Kane"/>
            <meta name="citation_author_institution" content="Buck Institute for Age Research, Novato, California, United States of America"/>
      <meta name="citation_author" content="Keith D. Ball"/>
            <meta name="citation_author_institution" content="Buck Institute for Age Research, Novato, California, United States of America"/>
      <meta name="citation_author" content="Dale E. Bredesen"/>
            <meta name="citation_author_institution" content="Buck Institute for Age Research, Novato, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Neurology, University of California San Francisco, San Francisco, California, United States of America"/>

    <meta name="citation_date" content="2007/5/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000463.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e463"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=SMART 4.0: towards genomic data integration.; citation_author=I Letunic; citation_author=RR Copley; citation_author=S Schmidt; citation_author=FD Ciccarelli; citation_author=T Doerks; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=1; citation_pages=D142-144; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Apoptosis and dependence receptors: a molecular basis for cellular addiction.; citation_author=DE Bredesen; citation_author=P Mehlen; citation_author=S Rabizadeh; citation_journal_title=Physiol Rev; citation_volume=84; citation_number=2; citation_pages=411-430; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The dependence receptor hypothesis.; citation_author=P Mehlen; citation_author=DE Bredesen; citation_journal_title=Apoptosis; citation_volume=9; citation_number=3; citation_pages=37-49; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Meeting report: cellular dependence–old concept, new mechanisms.; citation_author=P Mehlen; citation_author=DE Bredesen; citation_journal_title=Sci STKE; citation_volume=2003; citation_number=4; citation_pages=pe55; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Receptors that mediate cellular dependence.; citation_author=DE Bredesen; citation_author=P Mehlen; citation_author=S Rabizadeh; citation_journal_title=Cell Death Differ; citation_volume=12; citation_number=5; citation_pages=1031-1043; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Fitting a mixture model by expectation maximization to discover motifs in biopolymers.; citation_author=TL Bailey; citation_author=C Elkan; citation_journal_title=Proc Int Conf Intell Syst Mol Biol; citation_volume=2; citation_number=6; citation_pages=28-36; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003.; citation_author=B Boeckmann; citation_author=A Bairoch; citation_author=R Apweiler; citation_author=MC Blatter; citation_author=A Estreicher; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=7; citation_pages=365-370; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Combining evidence using p-values: application to sequence homology searches.; citation_author=TL Bailey; citation_author=M Gribskov; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=8; citation_pages=48-54; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=NPS@: network protein sequence analysis; citation_author=C Combet; citation_author=C Blanchet; citation_author=C Geourjon; citation_author=G Deleage; citation_journal_title=Trends Biochem Sci; citation_volume=25; citation_number=9; citation_pages=147-150; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Regulated intramembrane proteolysis: a control mechanism conserved from bacteria to humans.; citation_author=MS Brown; citation_author=J Ye; citation_author=RB Rawson; citation_author=JL Goldstein; citation_journal_title=Cell; citation_volume=100; citation_number=10; citation_pages=391-398; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A heptad motif of leucine residues found in membrane proteins can drive self-assembly of artificial transmembrane segments.; citation_author=R Gurezka; citation_author=R Laage; citation_author=B Brosig; citation_author=D Langosch; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=11; citation_pages=9265-9270; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=RGM and its receptor neogenin regulate neuronal survival.; citation_author=E Matsunaga; citation_author=S Tauszig-Delamasure; citation_author=PP Monnier; citation_author=BK Mueller; citation_author=SM Strittmatter; citation_journal_title=Nat Cell Biol; citation_volume=6; citation_number=12; citation_pages=749-755; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Caspase cleavage of members of the amyloid precursor family of proteins.; citation_author=V Galvan; citation_author=S Chen; citation_author=D Lu; citation_author=A Logvinova; citation_author=P Goldsmith; citation_journal_title=J Neurochem; citation_volume=82; citation_number=13; citation_pages=283-294; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Regulation of amyloid protein precursor (APP) binding to collagen and mapping of the binding sites on APP and collagen type I.; citation_author=D Beher; citation_author=L Hesse; citation_author=CL Masters; citation_author=G Multhaup; citation_journal_title=J Biol Chem; citation_volume=271; citation_number=14; citation_pages=1613-1620; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Interaction between Alzheimer's disease beta A4 precursor protein (APP) and the extracellular matrix: evidence for the participation of heparan sulfate proteoglycans.; citation_author=J Caceres; citation_author=E Brandan; citation_journal_title=J Cell Biochem; citation_volume=65; citation_number=15; citation_pages=145-158; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Secreted glypican binds to the amyloid precursor protein of Alzheimer's disease (APP) and inhibits APP-induced neurite outgrowth.; citation_author=TG Williamson; citation_author=SS Mok; citation_author=A Henry; citation_author=R Cappai; citation_author=AD Lander; citation_journal_title=J Biol Chem; citation_volume=271; citation_number=16; citation_pages=31215-31221; citation_date=1996; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000463" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Novel Motif Identified in Dependence Receptors"/>
    <meta name="twitter:description" content="Programmed cell death signaling is a critical feature of development, cellular turnover, oncogenesis, and neurodegeneration, among other processes. Such signaling may be transduced via specific receptors, either following ligand binding&mdash;to death receptors&mdash;or following the withdrawal of trophic ligands&mdash;from dependence receptors. Although dependence receptors display functional similarities, no common structural domains have been identified. Therefore, we employed the Multiple Expectation Maximization for Motif Elicitation and the Motif Alignment and Search Tool software programs to identify a novel transmembrane motif, dubbed dependence-associated receptor transmembrane (DART) motif, that is common to all described dependence receptors. Of 3,465 human transmembrane proteins, 25 (0.7%) display the DART motif. The predicted secondary structure features an alpha helical structure, with an unusually high percentage of valine residues. At least four of the proteins undergo regulated intramembrane proteolysis. To date, we have not identified a function for this putative domain. We speculate that the DART motif may be involved in protein processing, interaction with other proteins or lipids, or homomultimerization."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000463.g005"/>

  <meta property="og:title" content="A Novel Motif Identified in Dependence Receptors" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000463" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3500'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5763'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1827&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000463">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000463" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000463&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Novel Motif Identified in Dependence Receptors
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gabriel del Rio, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Buck Institute for Age Research, Novato, California, United States of America
                </p>

                <p>
                    Current address: Universidad Nacional Autonoma de Mexico/Instituto de Fisiologia Celular, Biochemistry Department, Ciudad Universitaria, Mexico
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Darci J. Kane, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Buck Institute for Age Research, Novato, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Keith D. Ball, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Buck Institute for Age Research, Novato, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dale E. Bredesen
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:dbredesen@buckinstitute.org">dbredesen@buckinstitute.org</a></p>

                <p>Affiliations:
                  Buck Institute for Age Research, Novato, California, United States of America, 
                  Department of Neurology, University of California San Francisco, San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 23, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000463</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000463" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000463" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000463" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000463" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000463-t001" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000463-g001" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000463-g002" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000463-g003" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000463-g004" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000463-t002" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000463-t003" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000463-g005" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000463">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000463" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Programmed cell death signaling is a critical feature of development, cellular turnover, oncogenesis, and neurodegeneration, among other processes. Such signaling may be transduced via specific receptors, either following ligand binding—to death receptors—or following the withdrawal of trophic ligands—from dependence receptors. Although dependence receptors display functional similarities, no common structural domains have been identified. Therefore, we employed the Multiple Expectation Maximization for Motif Elicitation and the Motif Alignment and Search Tool software programs to identify a novel transmembrane motif, dubbed dependence-associated receptor transmembrane (DART) motif, that is common to all described dependence receptors. Of 3,465 human transmembrane proteins, 25 (0.7%) display the DART motif. The predicted secondary structure features an alpha helical structure, with an unusually high percentage of valine residues. At least four of the proteins undergo regulated intramembrane proteolysis. To date, we have not identified a function for this putative domain. We speculate that the DART motif may be involved in protein processing, interaction with other proteins or lipids, or homomultimerization.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>del Rio G, Kane DJ, Ball KD, Bredesen DE (2007) A Novel Motif Identified in Dependence Receptors. PLoS ONE 2(5):
          e463.
            doi:10.1371/journal.pone.0000463</p><p><strong>Academic Editor: </strong>Thomas Zwaka, Baylor College of Medicine, United States of America</p><p><strong>Received:</strong> March 26, 2007; <strong>Accepted:</strong> April 23, 2007; <strong>Published:</strong> May 23, 2007</p><p><strong>Copyright:</strong> © 2007 del Rio et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a grant to the Buck Institute from American Bioscience, Inc., by NIH grants NS45093, AG12282 and NS33376 to DEB and by CoNaCyT grant SEP-2004-C01-47399/A-1 to GDR.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Protein evolution is rife with examples of structural and functional domains utilized by multiple proteins: for example, over 700 human proteins display SH3 domains, and over 450 proteins display PDZ domains. The identification of such common domains may hint at similarities in substructure, interactions, and potentially in function, for the various proteins displaying these domains. About 680 different domains have been documented to date in the proteomes of humans and other organisms (SMART database, <a href="http://smart.embl-heidelberg.de/">http://smart.embl-heidelberg.de/</a>, <a href="#pone.0000463-Letunic1">[1]</a>), and many of these domains appear, with some variation, in numerous proteins. Alternatively, the recognition of novel motifs may link proteins previously thought to be unrelated—e.g., suggesting a common function or interaction—and therefore may aid in the determination of both structure and function for proteins that display the novel motif.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>We have previously described a set of receptors that induces apoptosis following ligand withdrawal, but inhibits apoptosis following the binding of trophic ligands <a href="#pone.0000463-Bredesen1">[2]</a>–<a href="#pone.0000463-Bredesen2">[5]</a>. These receptors have been referred to as dependence receptors. Such receptors play roles in neural development, tumorigenesis (including metastasis), neurodegeneration, and possibly in subapoptotic events such as neurite retraction and somal atrophy <a href="#pone.0000463-Bredesen1">[2]</a>–<a href="#pone.0000463-Bredesen2">[5]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>To date, ten such receptors have been described (<a href="#pone-0000463-t001">Table 1</a> and <a href="#pone.0000463-Bredesen1">[2]</a>). These do not share any obvious structural similarity, nor do they display similar domains required for apoptosis induction. For example, Unc5H2 features a death domain in its intracytoplasmic region, but DCC does not; instead, apoptosis induction by DCC requires a short region in its intracytoplasmic domain (residues 1243–1264) that does not bear similarity to a death domain.</p>
<div class="figure" id="pone-0000463-t001"><div class="img"><a name="pone-0000463-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000463.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000463.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Ten dependence receptors (plus orthologues) comprise the training set for the MEME query.</span></strong></p><span>doi:10.1371/journal.pone.0000463.t001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In the present study, we attempted to determine whether dependence receptors as a group may indeed display a common motif(s) that had gone undetected by the initial comparisons of alignment and predicted (known) domains. We utilized the Multiple EM (Expectation Maximization) for Motif Elicitation program (MEME) (<a href="http://meme.sdsc.edu/meme/meme.html">http://meme.sdsc.edu/meme/meme.html</a> and <a href="#pone.0000463-Bailey1">[6]</a>) and identified a novel motif that is featured by receptors that have been described as dependence receptors. We then searched the Swiss-Prot protein database (<a href="http://www.expasy.uniprot.org/">http://www.expasy.uniprot.org/</a> and <a href="#pone.0000463-Boeckmann1">[7]</a>) using the Motif Alignment and Search Tool (MAST) program (<a href="http://meme.sdsc.edu/meme/mast.html">http://meme.sdsc.edu/meme/mast.html</a> and <a href="#pone.0000463-Bailey2">[8]</a>), to determine whether other receptors or other proteins also feature this motif, and identified an additional 16 human proteins that display this motif (see <a href="#s3">Results</a> and Discussion, below).</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The novel putative motif is in a transmembrane region, and therefore was dubbed dependence-associated receptor transmembrane motif (DART motif). Here we describe the consensus sequence and discuss the possible functions of this novel motif.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Databases and software</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The UniProt Knowledgebase (UniProtKB; <a href="http://www.expasy.uniprot.org/">http://www.expasy.uniprot.org/</a>) database Release 3.3 (consisting of Swiss-Prot Release 45.3 and TrEMBL Release 28.3) from the Swiss Institute of Bioinformatics was used for this study. This database was chosen because it is well documented and allowed us to analyze the predictions on receptors.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The MEME (<a href="http://meme.sdsc.edu/meme/meme.html">http://meme.sdsc.edu/meme/meme.html</a>) software program (version 3.0; non-commercial version) was used for the identification of motifs in non-aligned sequences, where a motif is a sequence pattern that occurs repeatedly in a group of protein or DNA sequences. MEME saves these motifs as a set of profiles. MEME uses the method of Bailey and Elkan to identify likely motifs within the input set of sequences <a href="#pone.0000463-Bailey1">[6]</a>. A range of motif widths (&gt;15 amino acids in length) and various numbers of unique motifs to search for (zero or one motifs per sequence) were specified in our queries.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The software program MAST (<a href="http://meme.sdsc.edu/meme/mast.html">http://meme.sdsc.edu/meme/mast.html</a>, version 3.0; non-commercial version) was used to search the Swiss-Prot database for other proteins displaying the motifs identified by MEME to be present in more than one dependence receptor. The algorithm in MAST calculates position scores for each profile at each possible position within a sequence. These scores are translated into p-values, which represent the likelihood of the given profile scoring that well against a randomly generated sequence. The best (i.e., lowest) position p-values for each profile are then adjusted to take into account the length of the sequence. MAST avoids allowing gaps in the profiles or in the search sequence.</p>


<h4>Training Set</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In order to search for motifs in previously described dependence receptors, we used a set of ten human receptors and their corresponding orthologues (three orthologues were used for each, in order to avoid bias generated by using more orthologues for one dependence receptor than another) found in the UniProtKB database. Thus, our training set included a total of 32 protein sequences (32 rather than 40, since the netrin receptors Unc5A, Unc5B, Unc5C were represented by a total of 4 sequences rather than 12, to prevent overweighting). This list is shown in <a href="#pone-0000463-t001">Table 1</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The option of having the training set sequences “shuffled” provided one of the controls used, ensuring that the motif(s) we detected were significant.</p>


<h4>Web Site</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To maintain updated information on the dependence receptor field and to allow researchers to identify the DART motif in their protein of interest, we have developed a website (<a href="http://bis.ifc.unam.mx/DependenceReceptors/">http://bis.ifc.unam.mx/DependenceRecepto​rs/</a>). The program at our website runs four independent predictions: three for identifying a transmembrane region and one for identifying the DART motif. The transmembrane region predictions are run through three different programs located at:</p>


<ol class="order">

<li>HMMTOP (<a href="http://www.enzim.hu/hmmtop/">http://www.enzim.hu/hmmtop/</a>),</li>

<li>SOSUI (<a href="http://sosui.proteome.bio.tuat.ac.jp/sosui_submit.html">http://sosui.proteome.bio.tuat.ac.jp/sos​ui_submit.html</a>) and</li>

<li>TMPRED (<a href="http://www.ch.embnet.org/software/TMPRED_form.html">http://www.ch.embnet.org/software/TMPRED​_form.html</a>).</li>

</ol><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>For identifying the DART motif, the website uses the MAST program.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Searching for motifs among the known dependence receptors</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The dependence receptors listed in <a href="#pone-0000463-t001">Table 1</a> have all been shown to induce programmed cell death when expressed in the absence of their respective trophic ligands, but not when bound by these same trophic ligands. The receptors' non-orthologous sequences in the training set did not show any significant sequence similarity by simple alignment searches. Hence, in order to search for novel motifs in this set, we used the MEME program. MEME allows the identification of motifs in non-aligned sequences, where a motif is a sequence pattern that occurs repeatedly in a group of protein or DNA sequences (see <a href="#s2">Materials and Methods</a> section). Although MEME has been most commonly used to identify motifs in homologous sequences, a pattern identified by MEME in non-homologous sequences may be biologically relevant if: a) the proteins in the training set share a common function and b) the proteins identified to contain the motif could reasonably be suspected to share functional features with the training set. Note that these are the same conditions that are considered when evaluating motifs identified in homologous proteins.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Using a training set consisting of 32 sequences from the 10 experimentally-proven dependence receptors (<a href="#pone-0000463-t001">Table 1</a>), we identified a novel motif that occurs in all of the training set proteins. This motif, designated “DART” (dependence-associated receptor transmembrane) motif, appeared in the transmembrane region of all proteins in the training set that include a transmembrane region, whereas it appeared in the ligand-binding region of the one protein that lacks a transmembrane region (the androgen receptor).</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>The consensus sequence of the proposed DART domain is shown in <a href="#pone-0000463-g001">Figure 1</a>, and the DART motifs from the training set proteins are aligned in <a href="#pone-0000463-g002">Figure 2</a>.</p>
<div class="figure" id="pone-0000463-g001"><div class="img"><a name="pone-0000463-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000463.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000463.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Multilevel consensus sequence and amino acid frequency of the DART motif.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>MEME motifs are represented by position-specific probability matrices that specify the probability of each possible letter appearing at each possible position in an occurrence of the motif. In order to make it easier to see which letters are most likely in each of the columns of the motif, the simplified motif shows the letter probabilities multiplied by 10 rounded to the nearest integer. Zeros are replaced by “:” (a colon) for readability. The information content diagram provides an idea of which positions in the motif are most highly conserved. Each column (position) in a motif can be characterized by the amount of information it contains (measured in bits). Highly conserved positions in the motif have high information; positions where all letters are equally likely have low information. The diagram is printed so that each column lines up with the same column in the simplified position-specific probability matrix above it. This multilevel consensus sequence says several things about the motif. First, the most likely form of the motif can be read from the top line as LLVIAVVVALVIxVLLVxL. Second, that only letter L has probability more than 0.2 in position 1 of the motif, both L and I have probability greater than 0.2 in position 2, etc. Third, a rough approximation of the motif can be made by converting the multilevel consensus sequence into the Prosite signature: L-[LI]-V-I-[AS]-V-V-V-[AGS]-L-V-[IF]-x-[​VI]-L-[LV]-V-x-[LI].</p>
<span>doi:10.1371/journal.pone.0000463.g001</span></div><div class="figure" id="pone-0000463-g002"><div class="img"><a name="pone-0000463-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000463.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000463.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Aligned DART motif within all dependence receptor training set members.</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>Occurrences (sites) of the DART motif within the sequences of the 32 dependence receptors that were used as the training set. The sites are shown aligned with each other, and the ten sequence positions preceding and following each site are also shown. Each site is identified by the name of the sequence where it occurs and the position in the sequence where the site begins. The sites are listed in order of increasing p-value (decreasing statistical significance). The p-value of a site is computed from the match score of the site with the position specific scoring matrix for the motif. The p-value gives the probability of a random string (generated from the background letter frequencies) having the same match score or higher. Amino acid residues constituting the transmembrane region of the protein are indicated by shading.</p>
<span>doi:10.1371/journal.pone.0000463.g002</span></div><a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>Using the software program MAST to search the Swiss-Prot database for other proteins that displayed the DART motif, we found an additional 54 sequences, 16 of which are human proteins (using a cut-off at E-value of 2.7, the value below which all training set members scored) (<a href="#pone-0000463-t002">Table 2</a>). Of 13,991 human proteins in the database, 3,465 are annotated as transmembrane proteins, and 25 display the DART motif—nine of the 10 training set members (the exception being, as noted above, the androgen receptor) and 16 additional human proteins (<a href="#pone-0000463-t003">Table 3</a>). Of these 16 additional proteins, all were transmembrane proteins, and all contained the DART motif within their transmembrane region. Thus the DART motif is relatively uncommon (at least as defined here), occurring in approximately 0.7% of human transmembrane proteins (25/3465). If we include slightly less similar motifs, extending the acceptable E-value from 2.7 to 10, then an additional 4 human proteins are included (data not shown). The alignment of the putative DART domains of these 16 human non-training set proteins is shown in <a href="#pone-0000463-g003">Figure 3</a>. A dendrogram of the human DART motifs is shown in <a href="#pone-0000463-g004">Figure 4</a>.</p>
<div class="figure" id="pone-0000463-g003"><div class="img"><a name="pone-0000463-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000463.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000463.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Aligned DART motif within 26 human proteins.</span></strong></p><a id="article1.body1.sec3.sec1.fig3.caption1.p1" name="article1.body1.sec3.sec1.fig3.caption1.p1"></a><p>Occurrences (sites) of the DART motif within the sequences of the 10 human dependence receptors from the training set (top 10 of list) and the 16 human proteins designated by MAST as containing the motif. The sites are shown aligned with each other, and the ten sequence positions preceding and following each site are also shown. Each site is identified by the name of the sequence where it occurs and the position in the sequence where the site begins. The sites are listed in order of increasing p-value. The p-value of a site is computed from the match score of the site with the position specific scoring matrix for the motif. The p-value gives the probability of a random string (generated from the background letter frequencies) having the same match score or higher. Amino acid residues constituting the transmembrane region of the protein are indicated by shading.</p>
<span>doi:10.1371/journal.pone.0000463.g003</span></div><div class="figure" id="pone-0000463-g004"><div class="img"><a name="pone-0000463-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000463.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000463.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Dendrogram of the 26 human DART-containing proteins.</span></strong></p><a id="article1.body1.sec3.sec1.fig4.caption1.p1" name="article1.body1.sec3.sec1.fig4.caption1.p1"></a><p>Dendrogram demonstrating the relationships of the DART motif sequences within each of the 26 human proteins found to contain DART (10 from the training set plus 16 discovered through the use of MAST).</p>
<span>doi:10.1371/journal.pone.0000463.g004</span></div><div class="figure" id="pone-0000463-t002"><div class="img"><a name="pone-0000463-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000463.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000463.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>MAST result list of 54 non-training set proteins found to display the DART motif.</span></strong></p><span>doi:10.1371/journal.pone.0000463.t002</span></div><div class="figure" id="pone-0000463-t003"><div class="img"><a name="pone-0000463-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000463.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000463.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>16 human proteins found to display the DART motif.</span></strong></p><span>doi:10.1371/journal.pone.0000463.t003</span></div>

<h4>Structure of the putative DART domain</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The predicted secondary structure of the consensus DART domain, as predicted by the SOPMA (Self Optimized Prediction Method from Alignments) method <a href="#pone.0000463-Combet1">[9]</a> is for a helical structure (<a href="#pone-0000463-g005">Figure 5</a>). This is not surprising, given that the motif lies within the transmembrane region of the proteins that display it. However, in comparison to other transmembrane regions (in randomly selected transmembrane proteins) it is valine rich (29±4% vs. 15±3%; p&lt;0.001).</p>
<div class="figure" id="pone-0000463-g005"><div class="img"><a name="pone-0000463-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000463" data-uri="info:doi/10.1371/journal.pone.0000463.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000463.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000463.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000463.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000463.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000463.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Predicted secondary structure of the consensus sequence of DART.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>SOPMA analysis demonstrates the alpha-helical nature of the putative DART domain.</p>
<span>doi:10.1371/journal.pone.0000463.g005</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The function of this novel motif is currently unknown. The finding that it exists in all dependence receptors described to date suggests that it may play a role in some biochemical process related to their function, such as the induction of apoptosis or the inhibition of apoptosis following ligand binding, or possibly an interaction with another membrane protein or membrane-associated non-proteinaceous molecule such as a lipid. For at least three of the proteins in the training set—APP, p75NTR, and DCC—this region undergoes regulated intramembrane proteolysis (RIP) <a href="#pone.0000463-Brown1">[10]</a>, releasing an intracytoplasmic fragment that may migrate to the nucleus. Thus it is possible that the other proteins that display the DART motif may be substrates that also undergo regulated intramembrane proteolysis; however, a number of proteins that have been shown to undergo such cleavage do not display a DART motif, so it is clearly not required for such processing.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>It is noteworthy that the transmembrane regions of DART-containing proteins are valine rich, with nearly twice the percentage of valine residues present in randomly-selected human Type I transmembrane protein domains (29±4% vs. 15±3%; p&lt;0.001). It has been shown that Leu heptads within transmembrane domains may serve as homomultimerization domains and that the substitution of Ala (or other residues typical of transmembrane regions, including Val) for Leu may prevent homomultimerization <a href="#pone.0000463-Gurezka1">[11]</a>. Thus one possibility for the Val-rich nature of the DART domain may relate to the inhibition of receptor homomultimerization.</p>

<h4>Proteins identified by MAST as displaying the DART motif</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>As noted above, MAST identified 16 proteins, all transmembrane proteins displaying the DART motifs and were as similar to the consensus as those of the training set (<a href="#pone-0000463-t003">Table 3</a>; <a href="#pone-0000463-g003">Figure 3</a>). Most of these have been implicated in cell death, either directly or indirectly; furthermore, several bind trophic ligands, as well, making them reasonable candidates to evaluate experimentally as possible dependence receptors.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Neogenin has recently been shown to bind RGM (repulsive guidance molecule), and to serve as a possible dependence receptor for RGM, inducing programmed cell death that is inhibited by RGM <a href="#pone.0000463-Matsunaga1">[12]</a>. Therefore, the identification of a DART motif within neogenin provides further support for RGM as a candidate dependence receptor.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>APLP2 (APP-like protein 2) has been shown previously to be similar to APP in displaying a potential caspase-cleavage site in its intracytoplasmic domain <a href="#pone.0000463-Galvan1">[13]</a>. Cleavage at this site liberates a pro-apoptotic peptide, C31, similar to what has been demonstrated for APP. Thus, although it is not yet clear whether APLP2 functions as a dependence receptor, and in particular whether APLP2 binds a trophic ligand, by analogy to APP it may bind laminin, collagen IV, glypican, or another ligand <a href="#pone.0000463-Beher1">[14]</a>–<a href="#pone.0000463-Williamson1">[16]</a>, and thus serve as a dependence receptor for one or more of those ligands.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>Notch is an extensively-studied transmembrane receptor involved in cell fate determination. It binds to ligands Delta1, Jagged1, and Jagged2, regulating differentiation, proliferation, and apoptosis. Notch, like APP, DCC, and p75NTR, undergoes regulated intramembrane proteolysis, liberating an intracytoplasmic domain, the NICD, that forms a transcriptional activator complex with RBP-J kappa, activating genes of the enhancer of split locus.</p>
<a id="article1.body1.sec4.sec1.p5" name="article1.body1.sec4.sec1.p5"></a><p>Ephrin type B receptor 3 binds both ephrin-B1 and ephrin-B2. It is not yet known whether this receptor induces programmed cell death in the absence of ephrin-B1 or –B2 binding.</p>
<a id="article1.body1.sec4.sec1.p6" name="article1.body1.sec4.sec1.p6"></a><p>Tumor-associated calcium signal transducer 2 (TACD2) may function as a trophic factor receptor, but its ligand is currently unknown.</p>
<a id="article1.body1.sec4.sec1.p7" name="article1.body1.sec4.sec1.p7"></a><p>Three of the proteins identified by MAST as displaying a DART motif are involved in neurotransmitter synthesis or release. Catechol O-methyltransferase exists in both cytosolic and membrane-spanning (type II membrane protein) forms, and this latter displays a DART motif. Syntaxin-3 is a Type IV membrane protein potentially involved in docking of synaptic vesicles at presynaptic active zones. Vesicle-associated membrane protein 5 (VAMP5) is also a Type IV membrane protein that may participate in vesicle trafficking events associated with myogenesis.</p>


<h4>Conclusion</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Ten of ten previously described dependence receptors display a region of similarity dubbed the DART (dependence-associated receptor transmembrane) motif. MAST identified this motif in an additional 16 human proteins in the SwissProt database, in all cases in the transmembrane regions. The function of this novel putative domain is unknown, but the motif is noted to be valine rich, and in at least four cases, the DART motif is a site of regulated intramembrane proteolysis (RIP). Whether or not this motif plays a functional role in cell death induction or ligand-induced inhibition mediated by dependence receptors remains to be determined, but the identification of this motif in 16 non-training-set proteins such as Notch and APLP2 raises the question of whether these proteins may also function as dependence receptors. Since the field of dependence receptors is an emerging field, we have developed a website for predicting dependence receptors at <a href="http://bis.ifc.unam.mx/DependenceReceptors/">http://bis.ifc.unam.mx/DependenceRecepto​rs/</a>.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Professor Mark Van der Laan (University of California, Berkeley, and the Buck Institute for Age Research) for his helpful discussions and assistance with data analysis.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: Gd DB DK. Performed the experiments: Gd. Analyzed the data: Gd DK KB. Wrote the paper: Gd DB DK KB. Other: Proposed the original idea and performed the initial computational analysis described in this work: Gd. Participated in the design and coordination of the study and helped to draft and finalize the manuscript: DB. Read and approved the final manuscript: DB Gd DK KB. Participated in the computational analysis and in the preparation of the manuscript: DK. Participated in the computation analysis: KB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000463-Letunic1" id="pone.0000463-Letunic1"></a>Letunic I, Copley RR, Schmidt S, Ciccarelli FD, Doerks T, et al.  (2004) SMART 4.0: towards genomic data integration. Nucleic Acids Res  32: D142–144.  <ul class="find" data-citedArticleID="953336" data-doi="10.1093/nar/gkh088"><li><a href="http://dx.doi.org/10.1093/nar/gkh088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SMART+4.0%3A+towards+genomic+data+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SMART+4.0%3A+towards+genomic+data+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000463-Bredesen1" id="pone.0000463-Bredesen1"></a>Bredesen DE, Mehlen P, Rabizadeh S (2004) Apoptosis and dependence receptors: a molecular basis for cellular addiction. Physiol Rev  84: 411–430.  <ul class="find" data-citedArticleID="953322" data-doi="10.1152/physrev.00027.2003"><li><a href="http://dx.doi.org/10.1152/physrev.00027.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Apoptosis+and+dependence+receptors%3A+a+molecular+basis+for+cellular+addiction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Apoptosis+and+dependence+receptors%3A+a+molecular+basis+for+cellular+addiction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000463-Mehlen1" id="pone.0000463-Mehlen1"></a>Mehlen P, Bredesen DE (2004) The dependence receptor hypothesis. Apoptosis  9: 37–49.  <ul class="find" data-citedArticleID="953340" data-doi="10.1023/b:appt.0000012120.66221.b2"><li><a href="http://dx.doi.org/10.1023/b:appt.0000012120.66221.b2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dependence+receptor+hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dependence+receptor+hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000463-Mehlen2" id="pone.0000463-Mehlen2"></a>Mehlen P, Bredesen DE (2003) Meeting report: cellular dependence–old concept, new mechanisms. Sci STKE  2003: pe55.  <ul class="find" data-citedArticleID="953342" data-doi="10.1126/stke.2132003pe55"><li><a href="http://dx.doi.org/10.1126/stke.2132003pe55" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Meeting+report%3A+cellular+dependence%26%23x2013%3Bold+concept%2C+new+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Meeting+report%3A+cellular+dependence%26%23x2013%3Bold+concept%2C+new+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000463-Bredesen2" id="pone.0000463-Bredesen2"></a>Bredesen DE, Mehlen P, Rabizadeh S (2005) Receptors that mediate cellular dependence. Cell Death Differ  12: 1031–1043.  <ul class="find" data-citedArticleID="953324" data-doi="10.1038/sj.cdd.4401680"><li><a href="http://dx.doi.org/10.1038/sj.cdd.4401680" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Receptors+that+mediate+cellular+dependence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Receptors+that+mediate+cellular+dependence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000463-Bailey1" id="pone.0000463-Bailey1"></a>Bailey TL, Elkan C (1994) Fitting a mixture model by expectation maximization to discover motifs in biopolymers. Proc Int Conf Intell Syst Mol Biol  2: 28–36.  <ul class="find" data-citedArticleID="953314"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fitting+a+mixture+model+by+expectation+maximization+to+discover+motifs+in+biopolymers.&amp;auth=&amp;atitle=Fitting+a+mixture+model+by+expectation+maximization+to+discover+motifs+in+biopolymers." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitting+a+mixture+model+by+expectation+maximization+to+discover+motifs+in+biopolymers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fitting+a+mixture+model+by+expectation+maximization+to+discover+motifs+in+biopolymers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000463-Boeckmann1" id="pone.0000463-Boeckmann1"></a>Boeckmann B, Bairoch A, Apweiler R, Blatter MC, Estreicher A, et al.  (2003) The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003. Nucleic Acids Res  31: 365–370.  <ul class="find" data-citedArticleID="953320" data-doi="10.1093/nar/gkg095"><li><a href="http://dx.doi.org/10.1093/nar/gkg095" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+SWISS-PROT+protein+knowledgebase+and+its+supplement+TrEMBL+in+2003." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+SWISS-PROT+protein+knowledgebase+and+its+supplement+TrEMBL+in+2003.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000463-Bailey2" id="pone.0000463-Bailey2"></a>Bailey TL, Gribskov M (1998) Combining evidence using p-values: application to sequence homology searches. Bioinformatics  14: 48–54.  <ul class="find" data-citedArticleID="953316" data-doi="10.1093/bioinformatics/14.1.48"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.1.48" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combining+evidence+using+p-values%3A+application+to+sequence+homology+searches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combining+evidence+using+p-values%3A+application+to+sequence+homology+searches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000463-Combet1" id="pone.0000463-Combet1"></a>Combet C, Blanchet C, Geourjon C, Deleage G (2000) NPS@: network protein sequence analysis Trends Biochem Sci  25: 147–150.  <ul class="find" data-citedArticleID="953330" data-doi="10.1016/s0968-0004(99)01540-6"><li><a href="http://dx.doi.org/10.1016/s0968-0004(99)01540-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NPS%40%3A+network+protein+sequence+analysis" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NPS%40%3A+network+protein+sequence+analysis%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000463-Brown1" id="pone.0000463-Brown1"></a>Brown MS, Ye J, Rawson RB, Goldstein JL (2000) Regulated intramembrane proteolysis: a control mechanism conserved from bacteria to humans. Cell  100: 391–398.  <ul class="find" data-citedArticleID="953326" data-doi="10.1016/s0092-8674(00)80675-3"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80675-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulated+intramembrane+proteolysis%3A+a+control+mechanism+conserved+from+bacteria+to+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulated+intramembrane+proteolysis%3A+a+control+mechanism+conserved+from+bacteria+to+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000463-Gurezka1" id="pone.0000463-Gurezka1"></a>Gurezka R, Laage R, Brosig B, Langosch D (1999) A heptad motif of leucine residues found in membrane proteins can drive self-assembly of artificial transmembrane segments. J Biol Chem  274: 9265–9270.  <ul class="find" data-citedArticleID="953334" data-doi="10.1074/jbc.274.14.9265"><li><a href="http://dx.doi.org/10.1074/jbc.274.14.9265" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+heptad+motif+of+leucine+residues+found+in+membrane+proteins+can+drive+self-assembly+of+artificial+transmembrane+segments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+heptad+motif+of+leucine+residues+found+in+membrane+proteins+can+drive+self-assembly+of+artificial+transmembrane+segments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000463-Matsunaga1" id="pone.0000463-Matsunaga1"></a>Matsunaga E, Tauszig-Delamasure S, Monnier PP, Mueller BK, Strittmatter SM, et al.  (2004) RGM and its receptor neogenin regulate neuronal survival. Nat Cell Biol  6: 749–755.  <ul class="find" data-citedArticleID="953338" data-doi="10.1038/ncb1157"><li><a href="http://dx.doi.org/10.1038/ncb1157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RGM+and+its+receptor+neogenin+regulate+neuronal+survival." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RGM+and+its+receptor+neogenin+regulate+neuronal+survival.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000463-Galvan1" id="pone.0000463-Galvan1"></a>Galvan V, Chen S, Lu D, Logvinova A, Goldsmith P, et al.  (2002) Caspase cleavage of members of the amyloid precursor family of proteins. J Neurochem  82: 283–294.  <ul class="find" data-citedArticleID="953332" data-doi="10.1046/j.1471-4159.2002.00970.x"><li><a href="http://dx.doi.org/10.1046/j.1471-4159.2002.00970.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Caspase+cleavage+of+members+of+the+amyloid+precursor+family+of+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Caspase+cleavage+of+members+of+the+amyloid+precursor+family+of+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000463-Beher1" id="pone.0000463-Beher1"></a>Beher D, Hesse L, Masters CL, Multhaup G (1996) Regulation of amyloid protein precursor (APP) binding to collagen and mapping of the binding sites on APP and collagen type I. J Biol Chem  271: 1613–1620.  <ul class="find" data-citedArticleID="953318" data-doi="10.1074/jbc.271.3.1613"><li><a href="http://dx.doi.org/10.1074/jbc.271.3.1613" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+amyloid+protein+precursor+%28APP%29+binding+to+collagen+and+mapping+of+the+binding+sites+on+APP+and+collagen+type+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+amyloid+protein+precursor+%28APP%29+binding+to+collagen+and+mapping+of+the+binding+sites+on+APP+and+collagen+type+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000463-Caceres1" id="pone.0000463-Caceres1"></a>Caceres J, Brandan E (1997) Interaction between Alzheimer's disease beta A4 precursor protein (APP) and the extracellular matrix: evidence for the participation of heparan sulfate proteoglycans. J Cell Biochem  65: 145–158.  <ul class="find" data-citedArticleID="953328" data-doi="10.1002/(sici)1097-4644(199705)65:2<145::aid-jcb2&gt;3.0.co;2-u"><li><a href="http://dx.doi.org/10.1002/(sici)1097-4644(199705)65:2<145::aid-jcb2&gt;3.0.co;2-u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+between+Alzheimer%27s+disease+beta+A4+precursor+protein+%28APP%29+and+the+extracellular+matrix%3A+evidence+for+the+participation+of+heparan+sulfate+proteoglycans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+between+Alzheimer%27s+disease+beta+A4+precursor+protein+%28APP%29+and+the+extracellular+matrix%3A+evidence+for+the+participation+of+heparan+sulfate+proteoglycans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000463-Williamson1" id="pone.0000463-Williamson1"></a>Williamson TG, Mok SS, Henry A, Cappai R, Lander AD, et al.  (1996) Secreted glypican binds to the amyloid precursor protein of Alzheimer's disease (APP) and inhibits APP-induced neurite outgrowth. J Biol Chem  271: 31215–31221.  <ul class="find" data-citedArticleID="953344" data-doi="10.1074/jbc.271.49.31215"><li><a href="http://dx.doi.org/10.1074/jbc.271.49.31215" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Secreted+glypican+binds+to+the+amyloid+precursor+protein+of+Alzheimer%27s+disease+%28APP%29+and+inhibits+APP-induced+neurite+outgrowth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Secreted+glypican+binds+to+the+amyloid+precursor+protein+of+Alzheimer%27s+disease+%28APP%29+and+inhibits+APP-induced+neurite+outgrowth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.XML" value="78033"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.PDF" value="730589"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t001.PNG_L" value="382236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t001.PNG_M" value="88748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t001.PNG_S" value="5673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t001.TIF" value="1986612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t001.PNG_I" value="58009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g001.PNG_L" value="556969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g001.PNG_M" value="34339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g001.PNG_S" value="3746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g001.TIF" value="783838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g001.PNG_I" value="37394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g002.PNG_L" value="3516241"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g002.PNG_M" value="137536"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g002.PNG_S" value="3949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g002.TIF" value="3719556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g002.PNG_I" value="45575"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g003.PNG_L" value="1851870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g003.PNG_M" value="87976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g003.PNG_S" value="3220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g003.TIF" value="2533732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g003.PNG_I" value="33916"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g004.PNG_L" value="183341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g004.PNG_M" value="28103"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g004.PNG_S" value="2070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g004.TIF" value="283124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g004.PNG_I" value="15905"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t002.PNG_L" value="434821"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t002.PNG_M" value="89677"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t002.PNG_S" value="16193"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t002.TIF" value="1933396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t002.PNG_I" value="240779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t003.PNG_L" value="351506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t003.PNG_M" value="129732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t003.PNG_S" value="3869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t003.TIF" value="1498132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.t003.PNG_I" value="54955"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g005.PNG_L" value="783087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g005.PNG_M" value="71839"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g005.PNG_S" value="2727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g005.TIF" value="1461378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000463.g005.PNG_I" value="29129"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000463&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000463" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000463&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000463&volume=&issue=&title=A Novel Motif Identified in Dependence Receptors&author_name=Gabriel%20del%20Rio%2C%20Darci%20J.%20Kane%2C%20Keith%20D.%20Ball%2C%20Dale%20E.%20Bredesen&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000463" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000463" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000463" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000463&amp;t=A%20Novel%20Motif%20Identified%20in%20Dependence%20Receptors" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000463&title=A%20Novel%20Motif%20Identified%20in%20Dependence%20Receptors&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000463&amp;title=A%20Novel%20Motif%20Identified%20in%20Dependence%20Receptors" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000463" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000463&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Novel Motif Identified in Dependence Receptors';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000463';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Novel%20Motif%20Identified%20in%20Dependence%20Receptors http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000463" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000463" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Apoptosis%22" title="Search for articles in the subject area:'Apoptosis'"><div class="flagText">Apoptosis</div></a>
              <div data-categoryid="17189" data-articleid="23764"
                   data-categoryname="Apoptosis"
                   class="flagImage" title="Flag 'Apoptosis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Computer+software%22" title="Search for articles in the subject area:'Computer software'"><div class="flagText">Computer software</div></a>
              <div data-categoryid="2287" data-articleid="23764"
                   data-categoryname="Computer software"
                   class="flagImage" title="Flag 'Computer software' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Database+searching%22" title="Search for articles in the subject area:'Database searching'"><div class="flagText">Database searching</div></a>
              <div data-categoryid="32021" data-articleid="23764"
                   data-categoryname="Database searching"
                   class="flagImage" title="Flag 'Database searching' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+proteins%22" title="Search for articles in the subject area:'Membrane proteins'"><div class="flagText">Membrane proteins</div></a>
              <div data-categoryid="17569" data-articleid="23764"
                   data-categoryname="Membrane proteins"
                   class="flagImage" title="Flag 'Membrane proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+receptor+signaling%22" title="Search for articles in the subject area:'Membrane receptor signaling'"><div class="flagText">Membrane receptor signaling</div></a>
              <div data-categoryid="20271" data-articleid="23764"
                   data-categoryname="Membrane receptor signaling"
                   class="flagImage" title="Flag 'Membrane receptor signaling' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="23764"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="23764"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transmembrane+receptors%22" title="Search for articles in the subject area:'Transmembrane receptors'"><div class="flagText">Transmembrane receptors</div></a>
              <div data-categoryid="20299" data-articleid="23764"
                   data-categoryname="Transmembrane receptors"
                   class="flagImage" title="Flag 'Transmembrane receptors' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=203'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4806'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4550&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
