

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Molecular and Structural Discrimination of Proline Racemase and Hydroxyproline-2-Epimerase from Nosocomial and Bacterial Pathogens</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000885"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000885" />

    <meta name="citation_title" content="Molecular and Structural Discrimination of Proline Racemase and Hydroxyproline-2-Epimerase from Nosocomial and Bacterial Pathogens"/>
    <meta itemprop="name" content="Molecular and Structural Discrimination of Proline Racemase and Hydroxyproline-2-Epimerase from Nosocomial and Bacterial Pathogens"/>

      <meta name="citation_author" content="Maira Goytia"/>
            <meta name="citation_author_institution" content="Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Nathalie Chamond"/>
            <meta name="citation_author_institution" content="Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Alain Cosson"/>
            <meta name="citation_author_institution" content="Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Nicolas Coatnoan"/>
            <meta name="citation_author_institution" content="Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Daniel Hermant"/>
            <meta name="citation_author_institution" content="Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Armand Berneman"/>
            <meta name="citation_author_institution" content="Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Paola Minoprio"/>
            <meta name="citation_author_institution" content="Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France"/>

    <meta name="citation_date" content="2007/9/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000885.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e885"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Studies on the enzymic reduction of amino acids. II. Purification and properties of D-proline reductase and a proline racemase from Clostridium sticklandii.; citation_author=TC Stadtman; citation_author=P Elliott; citation_journal_title=J Biol Chem; citation_volume=228(2); citation_number=1; citation_pages=983-997; citation_date=1957; " />
      <meta name="citation_reference" content="citation_title=Energetics of proline racemase: double fractionation experiment, a test for concertedness and for transition-state dominance.; citation_author=JG Belasco; citation_author=WJ Albery; citation_author=JR Knowles; citation_journal_title=Biochemistry; citation_volume=25(9); citation_number=2; citation_pages=2552-2558; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Energetics of proline racemase: racemization of unlabeled proline in the unsaturated, saturated, and oversaturated regimes.; citation_author=LM Fisher; citation_author=WJ Albery; citation_author=JR Knowles; citation_journal_title=Biochemistry; citation_volume=25(9); citation_number=3; citation_pages=2529-2537; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Lymphocyte polyclonal activation: a pitfall for vaccine design against infectious agents.; citation_author=B Reina-San-Martin; citation_author=A Cosson; citation_author=P Minoprio; citation_journal_title=Parasitology today (Personal ed; citation_volume=16(2); citation_number=4; citation_pages=62-67; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A B-cell mitogen from a pathogenic trypanosome is a eukaryotic proline racemase.; citation_author=B Reina-San-Martin; citation_author=W Degrave; citation_author=C Rougeot; citation_author=A Cosson; citation_author=N Chamond; citation_journal_title=Nat Med; citation_volume=6(8); citation_number=5; citation_pages=890-897; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Trypanosoma cruzi proline racemases are involved in parasite differentiation and infectivity.; citation_author=N Chamond; citation_author=M Goytia; citation_author=N Coatnoan; citation_author=JC Barale; citation_author=A Cosson; citation_journal_title=Mol Microbiol; citation_volume=58(1); citation_number=6; citation_pages=46-60; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Immunotherapy of Trypanosoma cruzi infections.; citation_author=N Chamond; citation_author=N Coatnoan; citation_author=P Minoprio; citation_journal_title=Current drug targets; citation_volume=2(3); citation_number=7; citation_pages=247-254; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Biochemical characterization of proline racemases from the human protozoan parasite Trypanosoma cruzi and definition of putative protein signatures.; citation_author=N Chamond; citation_author=C Gregoire; citation_author=N Coatnoan; citation_author=C Rougeot; citation_author=LH Freitas-Junior; citation_journal_title=J Biol Chem; citation_volume=278(18); citation_number=8; citation_pages=15484-15494; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Amino acid racemases: functions and mechanisms.; citation_author=T Yoshimura; citation_author=N Esak; citation_journal_title=Journal of bioscience and bioengineering; citation_volume=96(2); citation_number=9; citation_pages=103-109; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Crystal structure, catalytic mechanism, and mitogenic properties of Trypanosoma cruzi proline racemase.; citation_author=A Buschiazzo; citation_author=M Goytia; citation_author=F Schaeffer; citation_author=W Degrave; citation_author=W Shepard; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103(6); citation_number=10; citation_pages=1705-1710; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Hydroxyproline metabolism. I. Conversion to alpha-ketoglutarate by extracts of Pseudomonas.; citation_author=E Adams; citation_journal_title=J Biol Chem; citation_volume=234(8); citation_number=11; citation_pages=2073-2084; citation_date=1959; " />
      <meta name="citation_reference" content="citation_title=Conversion of hydroxyproline to pyrrole-2-carboxylic acid.; citation_author=AN Radhakrishnan; citation_author=A Meister; citation_journal_title=J Biol Chem; citation_volume=226(1); citation_number=12; citation_pages=559-571; citation_date=1957; " />
      <meta name="citation_reference" content="citation_title=A B lymphocyte mitogen is a Brucella abortus virulence factor required for persistent infection.; citation_author=JM Spera; citation_author=JE Ugalde; citation_author=J Mucci; citation_author=DJ Comerci; citation_author=RA Ugalde; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103(44); citation_number=13; citation_pages=16514-16519; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Metabolism of proline and of hydroxyproline.; citation_author=E Adams; citation_journal_title=Int Rev Connect Tissue Res; citation_volume=5; citation_number=14; citation_pages=1-91; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Collagens: molecular biology, diseases, and potentials for therapy.; citation_author=DJ Prockop; citation_author=KI Kivirikko; citation_journal_title=Annu Rev Biochem; citation_volume=64; citation_number=15; citation_pages=403-434; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Purification and mechanism of action of proline racemase.; citation_author=GJ Cardinale; citation_author=RH Abeles; citation_journal_title=Biochemistry; citation_volume=7(11); citation_number=16; citation_pages=3970-3978; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Purification and Properties of Inducible Hydroxyproline 2-Epimerase from Pseudomonas.; citation_author=E Adams; citation_author=IL Norton; citation_journal_title=J Biol Chem; citation_volume=239; citation_number=17; citation_pages=1525-1535; citation_date=1964; " />
      <meta name="citation_reference" content="citation_title=Hydroxyproline 2-epimerase of Pseudomonas. Subunit structure and active site studies.; citation_author=SG Ramaswamy; citation_journal_title=J Biol Chem; citation_volume=259(1); citation_number=18; citation_pages=249-254; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Clinical manifestations, treatment and control of infections caused by Clostridium difficile.; citation_author=E Bouza; citation_author=P Munoz; citation_author=R Alonso; citation_journal_title=Clin Microbiol Infect 11 Suppl; citation_volume=4; citation_number=19; citation_pages=57-64; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Clostridium difficile.; citation_author=B Stoddart; citation_author=MH Wilcox; citation_journal_title=Curr Opin Infect Dis; citation_volume=15(5); citation_number=20; citation_pages=513-518; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=From the discovery of the Malta fever's agent to the discovery of a marine mammal reservoir, brucellosis has continuously been a re-emerging zoonosis.; citation_author=J Godfroid; citation_author=A Cloeckaert; citation_author=JP Liautard; citation_author=S Kohler; citation_author=D Fretin; citation_journal_title=Vet Res; citation_volume=36(3); citation_number=21; citation_pages=313-326; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Targeting mechanisms of Pseudomonas aeruginosa pathogenesis.; citation_author=E Kipnis; citation_author=T Sawa; citation_author=J Wiener-Kronish; citation_journal_title=Med Mal Infect; citation_volume=36(2); citation_number=22; citation_pages=78-91; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Brucellosis.; citation_author=G Pappas; citation_author=N Akritidis; citation_author=M Bosilkovski; citation_author=E Tsianos; citation_journal_title=N Engl J Med; citation_volume=352(22); citation_number=23; citation_pages=2325-2336; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pathogen-host interactions in Pseudomonas aeruginosa pneumonia.; citation_author=RT Sadikot; citation_author=TS Blackwell; citation_author=JW Christman; citation_author=AS Prince; citation_journal_title=Am J Respir Crit Care Med; citation_volume=171(11); citation_number=24; citation_pages=1209-1223; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Melioidosis: insights into the pathogenicity of Burkholderia pseudomallei.; citation_author=WJ Wiersinga; citation_author=T van der Poll; citation_author=NJ White; citation_author=NP Day; citation_author=SJ Peacock; citation_journal_title=Nat Rev Microbiol; citation_volume=4(4); citation_number=25; citation_pages=272-282; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Septicaemia and septic arthritis due to Pseudomonas putida in a neutropenic patient.; citation_author=L Macfarlane; citation_author=BA Oppenheim; citation_author=P Lorrigan; citation_journal_title=The Journal of infection; citation_volume=23(3); citation_number=26; citation_pages=346-347; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Bacterial collagenases and collagen-degrading enzymes and their potential role in human disease.; citation_author=DJ Harrington; citation_journal_title=Infection and immunity; citation_volume=64(6); citation_number=27; citation_pages=1885-1891; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Microbial metalloproteases and pathogenesis.; citation_author=S Miyoshi; citation_author=S Shinoda; citation_journal_title=Microbes Infect; citation_volume=2(1); citation_number=28; citation_pages=91-98; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Pseudomonas aeruginosa virulence factors delay airway epithelial wound repair by altering the actin cytoskeleton and inducing overactivation of epithelial matrix metalloproteinase-2.; citation_author=S de Bentzmann; citation_author=M Polette; citation_author=JM Zahm; citation_author=J Hinnrasky; citation_author=C Kileztky; citation_journal_title=Laboratory investigation; a journal of technical methods and pathology; citation_volume=80(2); citation_number=29; citation_pages=209-219; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Degradation of basement membranes by Pseudomonas aeruginosa elastase.; citation_author=PA Bejarano; citation_author=JP Langeveld; citation_author=BG Hudson; citation_author=ME Noelken; citation_journal_title=Infection and immunity; citation_volume=57(12); citation_number=30; citation_pages=3783-3787; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Inducible degradation of hydroxyproline in Pseudomonas putida: pathway regulation and hydroxyproline uptake.; citation_author=RM Gryder; citation_author=E Adams; citation_journal_title=J Bacteriol; citation_volume=97(1); citation_number=31; citation_pages=292-306; citation_date=1969; " />
      <meta name="citation_reference" content="citation_title=Properties of the inducible hydroxyproline transport system of Pseudomonas putida.; citation_author=RM Gryder; citation_author=E Adams; citation_journal_title=J Bacteriol; citation_volume=101(3); citation_number=32; citation_pages=948-958; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Increased sensitivity of lymphocyte delta1-pyrroline-5-carboxylate reductase to inhibition by proline with transformation.; citation_author=D Valle; citation_author=RM Blaese; citation_author=JM Phang; citation_journal_title=Nature; citation_volume=253(5488); citation_number=33; citation_pages=214-216; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=Proline inhibition of pyrroline-5-carboxylate reductase: differences in enzymes obtained from animal and tissue culture sources.; citation_author=D Valle; citation_author=SJ Downing; citation_author=JM Phang; citation_journal_title=Biochemical and biophysical research communications; citation_volume=54(4); citation_number=34; citation_pages=1418-1424; citation_date=1973; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000885" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Molecular and Structural Discrimination of Proline Racemase and Hydroxyproline-2-Epimerase from Nosocomial and Bacterial Pathogens"/>
    <meta name="twitter:description" content="The first eukaryotic proline racemase (PRAC), isolated from the human Trypanosoma cruzi pathogen, is a validated therapeutic target against Chagas' disease. This essential enzyme is implicated in parasite life cycle and infectivity and its ability to trigger host B-cell nonspecific hypergammaglobulinemia contributes to parasite evasion and persistence. Using previously identified PRAC signatures and data mining we present the identification and characterization of a novel PRAC and five hydroxyproline epimerases (HyPRE) from pathogenic bacteria. Single-mutation of key HyPRE catalytic cysteine abrogates enzymatic activity supporting the presence of two reaction centers per homodimer. Furthermore, evidences are provided that Brucella abortus PrpA [for &lsquo;proline racemase&rsquo; virulence factor A] and homologous proteins from two Brucella spp are bona fide HyPREs and not &lsquo;one way&rsquo; directional PRACs as described elsewhere. Although the mechanisms of aminoacid racemization and epimerization are conserved between PRAC and HyPRE, our studies demonstrate that substrate accessibility and specificity partly rely on contraints imposed by aromatic or aliphatic residues distinctively belonging to the catalytic pockets. Analysis of PRAC and HyPRE sequences along with reaction center structural data disclose additional valuable elements for in silico discrimination of the enzymes. Furthermore, similarly to PRAC, the lymphocyte mitogenicity displayed by HyPREs is discussed in the context of bacterial metabolism and pathogenesis. Considering tissue specificity and tropism of infectious pathogens, it would not be surprising if upon infection PRAC and HyPRE play important roles in the regulation of the intracellular and extracellular amino acid pool profiting the microrganism with precursors and enzymatic pathways of the host."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000885.g007"/>

  <meta property="og:title" content="Molecular and Structural Discrimination of Proline Racemase and Hydroxyproline-2-Epimerase from Nosocomial and Bacterial Pathogens" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000885" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2790'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=638'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=90&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000885">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000885" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000885&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Molecular and Structural Discrimination of Proline Racemase and Hydroxyproline-2-Epimerase from Nosocomial and Bacterial Pathogens
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maira Goytia, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nathalie Chamond, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alain Cosson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nicolas Coatnoan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel Hermant, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Armand Berneman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paola Minoprio
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pmm@pasteur.fr">pmm@pasteur.fr</a></p>

                <p>Affiliation:
                  Laboratoire d'Immunobiologie des Infections à Trypanosoma, Département d'Immunologie, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 12, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000885</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000885" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000885" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000885" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000885" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000885-t001" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000885-g001" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000885-g002" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000885-g003" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000885-g004" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000885-g005" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000885-g006" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000885-g007" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000885">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000885" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The first eukaryotic proline racemase (PRAC), isolated from the human Trypanosoma cruzi pathogen, is a validated therapeutic target against Chagas' disease. This essential enzyme is implicated in parasite life cycle and infectivity and its ability to trigger host B-cell nonspecific hypergammaglobulinemia contributes to parasite evasion and persistence. Using previously identified PRAC signatures and data mining we present the identification and characterization of a novel PRAC and five hydroxyproline epimerases (HyPRE) from pathogenic bacteria. Single-mutation of key HyPRE catalytic cysteine abrogates enzymatic activity supporting the presence of two reaction centers per homodimer. Furthermore, evidences are provided that <em>Brucella abortus</em> PrpA [for ‘proline racemase’ virulence factor A] and homologous proteins from two <em>Brucella spp</em> are <em>bona fide</em> HyPREs and not ‘one way’ directional PRACs as described elsewhere. Although the mechanisms of aminoacid racemization and epimerization are conserved between PRAC and HyPRE, our studies demonstrate that substrate accessibility and specificity partly rely on contraints imposed by aromatic or aliphatic residues distinctively belonging to the catalytic pockets. Analysis of PRAC and HyPRE sequences along with reaction center structural data disclose additional valuable elements for <em>in silico</em> discrimination of the enzymes. Furthermore, similarly to PRAC, the lymphocyte mitogenicity displayed by HyPREs is discussed in the context of bacterial metabolism and pathogenesis. Considering tissue specificity and tropism of infectious pathogens, it would not be surprising if upon infection PRAC and HyPRE play important roles in the regulation of the intracellular and extracellular amino acid pool profiting the microrganism with precursors and enzymatic pathways of the host.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Goytia M, Chamond N, Cosson A, Coatnoan N, Hermant D, et al.  (2007) Molecular and Structural Discrimination of Proline Racemase and Hydroxyproline-2-Epimerase from Nosocomial and Bacterial Pathogens. PLoS ONE 2(9):
          e885.
            doi:10.1371/journal.pone.0000885</p><p><strong>Academic Editor: </strong>Adam Yuan, Temasek Life Sciences Laboratory, Singapore</p><p><strong>Received:</strong> June 25, 2007; <strong>Accepted:</strong> August 23, 2007; <strong>Published:</strong> September 12, 2007</p><p><strong>Copyright:</strong> © 2007 Goytia et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants of Institut Pasteur and Fonds Dédiés n°17 (Sanofi/Aventis/Ministère de la Recherche). MG is a fellow from Fondation pour la Recherche Médicale (FRM).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>In recent years, an increasing interest rose concerning Proline Racemases (PRAC). Originally isolated in 1957 from <em>Clostridium sticklandii</em> (<em>Cs</em>PRAC) <a href="#pone.0000885-Stadtman1">[1]</a>, PRAC has been extensively studied in the eighties by several groups at the biochemical level <a href="#pone.0000885-Belasco1">[2]</a>, <a href="#pone.0000885-Fisher1">[3]</a>. Lately, the first eukaryotic PRAC was isolated from the <em>Trypanosoma cruzi</em> pathogen (<em>Tc</em>PRAC) and shown to be involved in the mechanisms of parasite escape from host immune responses for its mitogenic properties toward B lymphocytes <a href="#pone.0000885-ReinaSanMartin1">[4]</a>, <a href="#pone.0000885-ReinaSanMartin2">[5]</a>. <em>Tc</em>PRAC is present in all <em>T. cruzi</em> life cycle stages, is essential for parasite viability and it appears to be involved in certain metabolic pathways during metacyclogenesis as parasites overexpressing <em>TcPRAC</em> genes gain better host infectivity <a href="#pone.0000885-Chamond1">[6]</a>. Similar genes in the human genome lack crucial enzyme catalytic residues thus consolidating <em>Tc</em>PRAC as a lead for drug development against trypanosomiasis <a href="#pone.0000885-Chamond2">[7]</a>, <a href="#pone.0000885-Chamond3">[8]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Racemases catalyze the deprotonation/reprotonation of the chiral carbon (C<sup>α</sup>) of both amino acid enantiomers resulting in steroinversion of chiral centers in reactions depending or not on pyridoxal phosphate (PLP) cofactor. PRAC is a member of the PLP-independent enzyme family along with Glutamate and Aspartate Racemases and Diaminopimelate Epimerase <a href="#pone.0000885-Yoshimura1">[9]</a>. Thermodynamic studies and the overall 3D-structure of homodimeric <em>Tc</em>PRAC in complex with its competitive inhibitor provided evidences that proline (Pro) racemization operates by stabilization of carbanionic transition-state species in a two-Cystein-dependent acid/base catalytic mechanism <a href="#pone.0000885-Buschiazzo1">[10]</a>. As demonstrated by site-specific mutagenesis, racemization of Pro involves two catalytic cystein (Cys) residues (Cys<sub>130</sub> and Cys<sub>300 </sub>) per <em>Tc</em>PRAC subunit.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Multiple alignment of functional PRAC amino acid sequences and the analysis of the conserved Cys has enabled the definition of minimal essential motifs (DRSP<strong>C</strong>GXGXXAXXA, i.e. MIII*, and M<strong>C</strong>GH) <a href="#pone.0000885-Chamond3">[8]</a> to identify putative PRACs. We investigated PRAC homologous genes from pathogens by screening released genoma databases to further explore novel potential therapeutic targets. When MIII* signature was used for mining, 111 hits were obtained, 92 of them possessing both catalytic residues.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The presence of functional PRAC was investigated in a collection of 9 bacterial species of pathogenic importance (i.e. Firmicute, α-, β- and γ-proteobacteria) using molecular and biochemical approaches. Current work unveils a new functional PRAC isolated from <em>Clostridium difficile</em> and 5 novel functional Hydroxyproline-2-Epimerases (HyPRE) specifically from <em>Pseudomonas aeruginosa, Burkholderia pseudomallei</em> and 3 <em>Brucella</em> species. The studies also reveal that MIII*, considered to be a minimal pattern to identify putative PRAC, is not sufficiently stringent to discriminate PRAC from HyPRE. Additional element motifs are provided for the discrimination of PRAC and HyPRE sequences based for instance on polarity constraints imposed by precise residues of the catalytic pockets that contribute to ligand specificity.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>HyPRE, a PLP-independent enzyme described in the late 1950's <a href="#pone.0000885-Adams1">[11]</a>, <a href="#pone.0000885-Radhakrishnan1">[12]</a>, presents overall sequence similarities with PRAC but reacts only with the C<sup>α</sup> of 4-hydroxyproline (OH-Pro). Enzymatic activities of bacterial PRAC and HyPREs identified here were fully characterized and specific <em>V<sub>max</sub></em> and <em>K<sub>m</sub></em> determined. Furthermore, the data discloses that HyPRE enzymatic activity equally depends on two catalytic Cys residues, as shown by single mutation of Cys<sub>88</sub> or Cys<sub>236</sub> residues of <em>P. aeruginosa</em> HyPRE which drastically impairs OH-Pro epimerization. This is the first work associating simultaneously full-length HyPRE genes and functional enzymatic activity of the encoded proteins.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>The present data challenges recently published studies <a href="#pone.0000885-Spera1">[13]</a> and establishes that a <em>Brucella abortus</em> virulence factor (PrpA), described as PRAC, as well as homologous proteins from <em>B. melitensis</em> and <em>B. suis,</em> are <em>bona fide</em> PLP-independent HyPREs that interconvert <em>trans</em> or <em>cis</em> OH-L-Pro into <em>cis</em> or <em>trans</em> OH-D-Pro respectively and no other amino acids.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Both Pro and OH-Pro are important compounds for growth and development of many organisms. They can be used as exclusive sources of carbon, nitrogen and energy and are the principal components of collagen-the most widespread molecule in higher organisms <a href="#pone.0000885-Adams2">[14]</a>, <a href="#pone.0000885-Prockop1">[15]</a>. Moreover, the importance of PRAC and HyPRE in the context of disease processes induced by pathogenic microorganisms is discussed.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>In silico gene selection of homologous PRAC genes</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Blast searches of NCBI and Swiss-Prot/TrEMBL databases with full-length <em>TcPRAC</em> sequences resulted in 184 hits from which 111 possess the minimal PRAC stringent MIII* among which 62 hits were directly annotated as ‘PRAC’, without previous validation of the enzymatic activity. The present analysis revealed that MIII* and M<strong>C</strong>GH motif <a href="#pone.0000885-Buschiazzo1">[10]</a>, encompassing the <em>Tc</em>PRAC Cys<sub>300</sub> and Cys<sub>130</sub> crucial residues respectively, were consistently present in 92 sequences. We formerly suggested that predicted proteins originated from genes lacking these key Cys residues would display functions other than Pro racemization <a href="#pone.0000885-Chamond3">[8]</a>. A collection of 15 sequences was selected for further studies accordingly to sequence identities with <em>Tc</em>PRAC, to the conservation or not of homologous Cys<sub>130</sub> and Cys<sub>300 </sub>and the recognized pathogenic importance of the microbial genomes.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>As summarized in <a href="#pone-0000885-t001">Table 1</a>, homologous genes from different pathogen strains, annotated as ‘putative PRAC’, ‘PRAC’ or ‘unknown’ proteins, display 29 to 56% homology with <em>Tc</em>PRAC, present either a conservation of the couple of catalytic Cys or replacements of one or both Cys positions by serine (Ser) and/or threonine (Thr) residues. A comparison between <em>Brucella spp</em> sequences and the previously characterized <em>Tc</em>PRAC and <em>Cs</em>PRAC was of note. Therefore, from the two available homologous sequences for each <em>Brucella</em> specie only one meets the requirements for PRAC activity and presents both key Cys residues, the other presenting Ser and Thr substitutions.</p>
<div class="figure" id="pone-0000885-t001"><div class="img"><a name="pone-0000885-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000885.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000885.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Database collection from in silico searches and complementary information on selected sequences.</span></strong></p><span>doi:10.1371/journal.pone.0000885.t001</span></div>

<h4>Functional PRAC and Hydroxyproline epimerases of pathogenic bacteria</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The function of 12 gene products and their ability to interconvert Pro residues was addressed. Purified recombinant proteins were analyzed in biochemical assays by measuring the shift in optical rotation of either L- or D-Pro. As shown in <a href="#pone-0000885-g001">Figure 1</a>, <em>C. difficile (Cd)</em> recombinant protein racemized both L- and D-Pro but not OH-L/D-Pro or any other natural amino acid. <em>Cd</em>PRAC activity is PLP-independent which closely resembles <em>Tc</em>PRAC and <em>Cs</em>PRAC <a href="#pone.0000885-ReinaSanMartin2">[5]</a>, <a href="#pone.0000885-Cardinale1">[16]</a>. Conversely, <em>B. pseudomallei</em> and <em>P. aeruginosa</em> recombinant proteins presented no measurable PRAC activity but demonstrated strong epimerization of OH-L/D-Pro behaving as genuine OH-Pro epimerases. However, as predicted, control recombinant proteins produced from <em>B. cenocepacia</em> and <em>P. aeruginosa</em> sequences that present ‘Cys-Thr’ or ‘Ser-Cys’ couple replacements respectively did not show neither PRAC nor HyPRE enzymatic activities. Unexpectedly, three tested recombinant proteins, two produced from <em>Bacillus anthracis</em> and one from <em>Vibrio parahaemolyticus</em>, annotated as ‘putative PRACs’ and presenting the ‘Cys-Cys’ couple generated recombinant proteins that did not display PRAC or HyPRE activities.</p>


<h4>Brucella abortus PrpA virulence factor is a validated hydroxyproline epimerase</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>One of the <em>B. abortus</em> sequences, presenting the ‘Cys-Cys’ couple, was reported elsewhere as a B-cell mitogen with PRAC activity (<em>Ba</em>PrpA, for proline racemase protein A) and was shown to be directly involved in bacterial virulence and immune system evasion <a href="#pone.0000885-Spera1">[13]</a>. Surprisingly, PrpA would not be able to perform reversible conversion of L- and D-Pro but only a L-Pro to D-Pro unidirectional conversion. If correct, this assertion would imply that other PRAC could behave likewise. The enzymatic activity of <em>Ba</em>PrpA produced from sequence 1 obtained <em>in silico</em> was then investigated. <em>Ba</em>Seq1, derived from <em>Ba</em>-strain 544, is 100% homologous to <em>Ba</em>-strain 9-941 and <em>Ba</em>PrpA (<em>Ba</em> strain 2308, BAB1_1800) and possesses all PRAC motifs (<a href="#pone.0000885.s001">Figure S1</a>). The present data undoubtedly demonstrate that <em>Ba</em>Seq1 displayed only HyPRE activity irrespective of the enzyme concentration, pH and buffer conditions (<a href="#pone-0000885-g002">Figures 2A and 2B</a>), in contrast to recurrent Pro racemization values obtained with <em>Tc</em>PRAC. Since <em>Ba</em>Seq1 sequence is 98% homologous to proteins annotated as ‘PRAC’ from <em>B. melitensis (Bm)</em> and <em>B. suis (Bs)</em> all three recombinant homologous proteins were tested in parallel for PRAC and HyPRE activities. These proteins were unable to catalyze Pro racemization but exhibited equivalent strong ability to perform epimerization of both OH-L-Pro and OH-D-Pro (<a href="#pone-0000885-g002">Figure 2 C and 2D</a>). Consequently, the data proves that <em>Ba</em>Seq1, and therefore PrpA, is a HyPRE, as do <em>B. melitensis</em> and <em>B. suis</em> corresponding proteins.</p>
<div class="figure" id="pone-0000885-g001"><div class="img"><a name="pone-0000885-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Enzymatic activities of PRAC and HyPRE from different pathogens.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Optimal reaction conditions consisted of 10 µg of the enzyme and 20 mM of substrate in specific buffers during 30 min at 37°C. (A) Percent of L- or D-Pro racemization in NaOAc, pH 6; (B) Percent of OH-L-Pro or OH-D-Pro epimerization in TE, pH 8. <em>P. aeruginosa</em> (SC) and <em>B. cenocepacia</em> (CT) recombinant proteins whose sequences lack one of the two Cys catalytic residues do not display any PRAC or HyPRE activities.</p>
<span>doi:10.1371/journal.pone.0000885.g001</span></div><div class="figure" id="pone-0000885-g002"><div class="img"><a name="pone-0000885-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>PrpA of B. abortus (BaSeq1) is an hydroxyproline-2-epimerase.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Reactions were performed with 3–10 µg of the enzyme and 40 mM of substrate in specific buffers during 1 h at 37°C. (A) Pro racemization reactions were performed in NaOAc, pH 6. (B) OH-Pro epimerization reactions were set up in parallel in TE, pH 9. Data from Spera et al <a href="#pone.0000885-Spera1">[13]</a> was transposed to the Figure under shade and <em>Tc</em>PRAC was used as control; <em>Ba</em>PrpA : purported ‘proline racemase protein A’; <em>Ba</em>Seq1 was produced from PrpA-corresponding sequence 1 from <em>B. abortus</em> (<a href="#pone-0000885-t001">Table 1</a> and <a href="#pone.0000885.s001">Figure S1</a>). Percent of L- or D-Pro racemization (C) and percent of OH-L-Pro or OH-D-Pro epimerization (D) using specific buffers, 10 µg of the enzyme and 20 mM of substrate during 30 min at 37°C.</p>
<span>doi:10.1371/journal.pone.0000885.g002</span></div>

<h4>Kinetic properties of bacterial hydroxyproline epimerases</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Optimum conditions for PRAC and HyPRE reactions for all bacterial enzymes were obtained in NaOAc, pH 6 and Tris/EDTA (TE), pH 8–9 buffers, respectively. On the other hand, when PRAC was radically inhibited by its specific competitive inhibitor pyrrole-2-carboxylic acid (PYC), no inhibition of HyPRE was observed with standard amounts of PYC (1 mM) (<a href="#pone-0000885-g003">Figures 3A and 3B</a>). HyPRE reactions were only affected by high amounts of PYC (10 mM) or by variable concentrations of iodoacetate and iodoacetamide inhibitors (<a href="#pone.0000885.s002">Figure S2</a>). Progress of Pro and OH-Pro catalysis was monitored polarimetrically. The interconversion of L&lt;&gt;D-Pro mediated by <em>Cd</em>PRAC reveals that the enzyme has comparable velocity and affinity constants to those of <em>Tc</em>PRAC (<a href="#pone-0000885-g004">Figure 4</a>). Graphic representation of the Michaelis-Menten equation corresponding to the initial velocity of <em>Cd</em>PRAC and <em>Pa</em>HyPRE as function of substrate concentration is shown, as well as respective <em>K<sub>m</sub></em> and <em>V<sub>max</sub></em> kinetic values (<a href="#pone-0000885-g004">Figure 4A and 4B</a>). <em>Brucella spp</em> and <em>B. peudomallei</em> HyPREs exhibited comparable <em>V<sub>max</sub></em> and apparent <em>K<sub>m</sub></em> values to those of <em>Pa</em>HyPRE (<a href="#pone-0000885-g004">Figure 4C</a>). However, at equilibrium, all HyPRE enzymes showed a clear advantage to OH-D-Pro substrate.</p>
<div class="figure" id="pone-0000885-g003"><div class="img"><a name="pone-0000885-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Pyrrole-2-carboxylic acid (PYC), the specific inhibitor of PRAC, is not an inhibitor of HyPRE.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(A) Percent of L-Pro racemization or (B) OH-L-Pro epimerization in absence (black bars) or in presence of 1 or 10 mM of PYC (white bars). Reactions were performed at 37°C for 30 min with 10 µg of the corresponding enzymes in NaOAc, pH 6 (PRAC reactions) or TE, pH 8 (HyPRE reactions) and 20 mM of substrate.</p>
<span>doi:10.1371/journal.pone.0000885.g003</span></div><div class="figure" id="pone-0000885-g004"><div class="img"><a name="pone-0000885-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Kinetic parameters of Proline racemization and Hydroxyproline epimerization.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Progress of enzymatic activities was monitored polarimetrically, as described previously <a href="#pone.0000885-Chamond3">[8]</a>. Initial rates were plotted in function of [S] and kinetic parameters determined with Kaleidagraph® program and Michaelis-Menten equation. Maximum rate (<em>V<sub>max</sub></em>) and Michaelis-Menten constant (<em>K<sub>m</sub></em>) were obtained at 37°C by incubation of 20 µg/ml of each recombinant protein with increasing concentrations of specific L- (closed circles) or D- (open squares) substrates. (A) PRAC activity is depicted for <em>C. difficile</em>; (B) HyPRE activity is depicted for <em>P. aeruginosa</em>. (C) <em>K<sub>m</sub></em> and <em>V<sub>max</sub></em> records of HyPRE reactions using L- or D- enantiomers were distinctively obtained with recombinant enzymes of <em>B. abortus</em> (<em>Ba</em>HyPRE), <em>B. melitensis</em> (<em>Bm</em>HyPRE), <em>B. suis</em> (<em>Bs</em>HyPRE) and <em>B. pseudomallei</em> (<em>Bp</em>HyPRE). <em>Tc</em>PRACA : <em>K<sub>m</sub></em> of 29 mM <em>and V<sub>max</sub></em> of 5,3×10<sup>−5 </sup>M.sec<sup>−1</sup> and TcPRACB : <em>K<sub>m</sub></em> of 75 mM and <em>V<sub>max</sub></em> of 2×10<sup>−4</sup> M.sec<sup>−1</sup>.</p>
<span>doi:10.1371/journal.pone.0000885.g004</span></div>

<h4>Additional elements to previously defined PRAC signature can discriminate proline racemases and hydroxyproline epimerases</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Following blast searches using full-length <em>Tc</em>PRAC sequence, MIII* and MCGH block, we demonstrated that a number of homologous hits corresponded to HyPRE, a PRAC-related enzyme. Sequences of PRAC and HyPRE were aligned and residues that may be useful for their discrimination were identified (<a href="#pone-0000885-g005">Figure 5</a>). Thus, although both enzymes possess the catalytic ‘Cys-Cys’ couple, three major and non dissociated differences seem to be noteworthy for substrate specificity. The first and most important particularity is an aromatic phenylalanine (Phe) residue which was shown to be capital to hydrophobic contacts of <em>Tc</em>PRAC with Pro ring carbon atoms that is missing in HyPRE (depicted in R1). In fact, Phe imposes polarity constraints precluding polar functions at the level of the substrate carbon ring. Instead, HyPRE holds Ser or valine (Val) substitutions, i.e. small polar or aliphatic amino acids, that would account for better OH-Pro accessibility into the pocket. Other sequences encoding proteins without enzymatic activity may present at that position, polar tyrosine (Tyr) or histidine (His) residues which would restrict PRAC or HyPRE catalysis, as observed with <em>B. anthracis</em> sequences. Another feature is the presence in the <em>Tc</em>PRAC pocket environment of a Cys (or a Leucine, i.e. Leu, for other PRAC) residue in position 270 while HyPREs possess in that position a consistent polar His residue (depicted in R2) optimally placed to favor H-bonding interaction with the OH- of the C<sup>γ</sup>-atom of OH-Pro. Moreover, an additional block of three residues downstream of the highly conserved MIII* (XLA, depicted in R3) is fully restrictive to discriminate HyPRE and PRAC enzymes. These three differences are complementary to the presence of the ‘Cys-Cys’ couple of the catalytic pockets as ascertained by the absence of both enzymatic activities exhibited by <em>B. anthracis</em> and <em>V. parahaemolyticus</em> proteins.</p>
<div class="figure" id="pone-0000885-g005"><div class="img"><a name="pone-0000885-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Alignments of PRAC and HyPRE protein sequences.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>MCGH and MIII* PRAC motifs are shaded respectively in yellow and green. Catalytic Cys residues are coloured in red. R1, R2 and R3 indicate critical compulsory differences allowing for the discrimination of PRAC and HyPRE. In the left margin, sequences corresponding to PRAC are underlined contrasting to HyPRE sequences (plain text). Residues involved in substrate specificity are shaded in green (PRAC) or blue (HyPRE). The proposed signature for HyPRE (squared) gather an additional block of specific residues downstream PRAC MIII*. Sequences that do not meet those requirements and thus present unknown functions are in light gray.</p>
<span>doi:10.1371/journal.pone.0000885.g005</span></div>

<h4>Abrogation of HyPRE enzymatic activity by mutation of conserved cysteine residues of the catalytic site</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>HyPRE homodimer was described as having both subunits participating in a single catalytic site <a href="#pone.0000885-Adams3">[17]</a>, <a href="#pone.0000885-Ramaswamy1">[18]</a>. The potential role of ‘Cys-Cys’ couple in HyPRE catalysis was verified through site-directed mutagenesis of <em>Pa</em>HyPRE Cys<sub>88</sub> or Cys<sub>236</sub> into Ser residues (<a href="#pone.0000885.s003">Figure S3</a> and <a href="#pone.0000885.s004">Table S1</a>). In comparison to wild type HyPRE, <sup>C88S</sup>HyPRE and <sup>C236S</sup>HyPRE single mutations induce radical loss of OH-L/D-Pro epimerization establishing that proton transfer during HyPRE reaction is indeed dependent on the presence of the catalytic ‘Cys-Cys’ couple of each subunit (<a href="#pone-0000885-g006">Figure 6</a>). To validate the weight of the Val<sub>60</sub> residue in ligand accessibility and thus in substrate specificity, the residue was mutated into glycine (Gly) (<sup>V60G</sup>HyPRE) or Phe (<sup>V60F</sup>HyPRE), meeting or not size and stability limits imposed by Val. The absence of epimerization exhibited by the two mutants revealed that the Val<sub>60</sub> aliphatic residue indeed accounts for OH-Pro ligand specificity and is consequently essential for HyPRE catalysis. Conversely, the Phe<sub>102</sub> residue on the PRAC catalytic site environment offers hydrophobic restriction area to the pocket occupancy restraining the accessibility of OH-Pro.</p>
<div class="figure" id="pone-0000885-g006"><div class="img"><a name="pone-0000885-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Site-directed mutagenesis of key residues of PaHyPRE results in loss of enzymatic activity.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) and (B) Percent of epimerization of respectively OH-L-Pro and OH-D-Pro exhibited by WT <em>Pa</em>HyPRE or C88S, C236S, V60G and V60F point mutants.</p>
<span>doi:10.1371/journal.pone.0000885.g006</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>The space and polarity constraints of PRAC and HyPRE active sites on protein–ligand interactions are visualized better by comparing the closer views of the enzyme pockets (<a href="#pone-0000885-g007">Figure 7A and 7B</a>). Therefore, despite close similarities displayed by PRAC and HyPRE 3D-structures, the presence of a sizable aromatic residue or, alternatively, of a small aliphatic or polar amino acid, unquestionably plays a determinant role on the enzyme/substrate specificity.</p>
<div class="figure" id="pone-0000885-g007"><div class="img"><a name="pone-0000885-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000885" data-uri="info:doi/10.1371/journal.pone.0000885.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000885.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000885.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000885.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000885.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>PRAC and HyPRE structural data, pocket constraints and evolution.</span></strong></p><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1"></a><p>(A) Ribbon model of <em>Tc</em>PRAC (green, PDB : 1W61) and <em>Pa</em>HyPRE (purple, PDB : 2AZP) subunits revealing the overall similarities of the 3D-structures. Cys catalytic residues (orange). (B) Close view of <em>Tc</em>PRAC (left panel) and <em>Pa</em>HyPRE (right panel) pockets. The two catalytic Cys residues of PRAC (C<sub>130</sub> and C<sub>300</sub>) and of HyPRE (C<sub>88</sub> and C<sub>236</sub>) are shown in the reaction center colored in orange sticks. Hydrophobic F<sub>102</sub> (green sticks) and aliphatic V<sub>60</sub> (blue sticks) residues are depicted respectively in PRAC and HyPRE reaction centers where Pro and OH-Pro were modeled. Polarity hindrance imposed by the aromatic PRAC F<sub>102</sub> residue and the solvent accessible area for the ligand made possible by HyPRE V<sub>60</sub> residue are shown. (C) Phylogram of PRAC and HyPRE aligned sequences showing the unrooted tree using <em>H. influenzae</em> DapE as uncontroversial outgroup. Bacterial and protozoa PRAC cluster together suggesting that divergence of PRAC and HyPRE took place before the separation of bacteria and eukaryotes.</p>
<span>doi:10.1371/journal.pone.0000885.g007</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>The significance and conservation of PRAC and HyPRE throughout evolution was investigated by a phylogram using another PLP-independent enzyme as an uncontroversial outgroup, i.e. the <em>Haemophilus influenzae</em> diaminopimelate epimerase (DapE). <a href="#pone-0000885-g007">Figure 7C</a> shows that PRAC and HyPRE cluster in three main groups. Interestingly, PRAC from <em>C. difficile</em> and <em>C. sticklandii</em> cluster together with <em>T. cruzi</em> and <em>T. vivax</em> (<em>Trypanosoma vivax</em> possesses a functional proline racemase. N. Chamond, A. Cosson, M. Goytia, P. Minoprio, 2007, manuscript to be submitted), the segregation of the tree branches reflecting their ancient origin. It is conceivable that the divergence between PRAC and HyPRE is phylogenetically older than the separation of bacteria, archea and eukaryotes. Alternatively, possible gene transfer between species can be envisaged.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The discovery of novel microbial genes and metabolic proteins through genome mining has proven to be a promising approach to identify potential candidates for drug discovery and therapy against infections. Despite increased availability of genome data, the attribution of putative functions to homologous genes annotations are at times too simple and errors can occur with the consequence of incorrect scientific dogmas. In this paper we report that from a selected database assembled from blast searches using <em>T. cruzi</em> proline racemase (<em>Tc</em>PRAC) full-length sequences, 73% of the hits were incorrectly annotated as PRAC or putative PRAC since most of the proteins do not experimentally display functional PRAC activity. Consistent with previous data determining critical residues for PRAC catalysis, the present study reveals that out of 12 ‘PRAC-like’ recombinant proteins from different pathogens only one (8%), from <em>C. difficile,</em> which is a significant nosocomial pathogen <a href="#pone.0000885-Bouza1">[19]</a>, <a href="#pone.0000885-Stoddart1">[20]</a>, demonstrates truthful PRAC activity. Other proteins isolated from bacteria responsible for human and animal health problems <a href="#pone.0000885-Godfroid1">[21]</a>–<a href="#pone.0000885-Wiersinga1">[25]</a> have been incorrectly annotated as PRAC and are in fact HyPRE, i.e. 3 <em>Brucella</em> species, <em>P. aeruginosa</em> and <em>B. pseudomallei</em>. In addition, 33% of the studied sequences were erroneously annotated though missing fundamental catalytic residues. To our knowledge, apart from previous work using purified <em>P. putida</em> HyPRE which associated the enzyme active site to 14 residues <a href="#pone.0000885-Ramaswamy1">[18]</a>, the current work is the first describing HyPRE full-length genes and may contribute in the future to better annotation of unknown ORFs.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Based on overall comparisons between PRAC and HyPRE and despite the evident identities displayed by the peptide sequences, structural evidences were presented here that allow the discrimination of both enzymatic activities. Considering the results obtained with <em>Pa</em>HyPRE mutants supporting the key role of Cys<sub>88</sub> and Cys<sub>236</sub> residues in catalysis and the large overall structural similarity with <em>Tc</em>PRAC, our data supports a reaction mechanism similar to PRAC where HyPRE equally possesses two active sites per dimer, each one including two catalytic Cys. Therefore, Cys<sub>88</sub> and Cys<sub>236</sub> residues are correctly positioned in the HyPRE pocket to perform epimerization of C<sup>α</sup> OH-Pro chiral center. However, HyPRE is not inhibited by PYC, the transition state analogue of Pro. It has previously been shown that hydrophobic Phe<sub>102</sub> (R1) and Phe<sub>290 </sub>residues present in the <em>Tc</em>PRAC pocket impose polarity restrictions that enable interactions of the enzyme with the C<sup>α</sup> Pro ring or the C2 atom of PYC. Instead, the absence of Phe residues in HyPRE pocket, most particularly Phe<sub>102</sub>, and its substitution by an aliphatic Val (or polar Ser), promotes an ideal environment for accessibility and stereoinversion of the C<sup>α</sup> of OH-Pro. Indeed, mutagenesis of Val<sub>60</sub> into Gly or Phe, results in radical loss of <em>Pa</em>HyPRE activity, attributing a significant role to Val<sub>60</sub> in the conformation of the enzyme, the pocket stability and the ligand specificity. It could though be hypothesized that a single replacement of this central Val<sub>60</sub> residue by an aromatic Phe would be sufficient to affect the hydrosolubility of the HyPRE pocket environment thus favouring the accessibility and correct positioning of ‘Pro’ and its further ‘racemization’. Nonetheless, our data shows that this hypothesis is unlike since <em>Pa</em>HyPRE V60F-mutant is unable to perform L-Pro &lt; &gt; D-Pro conversion. <em>Per se</em> this result is not surprising given that racemization of Pro by PRAC catalytic Cys is known to be assisted by neighbor residues of the pocket that are equally present on HyPRE, such as Leu<sub>127</sub>, His<sub>132</sub> and Asp<sub>296</sub>, <a href="#pone.0000885-Buschiazzo1">[10]</a>, but in addition by Phe<sub>290</sub> which is absent in <em>Pa</em>HyPRE sequence. These neighboring residues may be involved in significant hydrophobic interactions of the enzyme with its ligand and influence the <em>pKa</em> of the catalytic Cys residues thus determining the environment hydrophobicity and as such affecting the stability of the pocket and resulting catalysis.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>On the other hand, PRAC and HyPRE multiple alignments allowed identification of other important and non dissociated elements that account for the discrimination of the enzymes, such as the presence of the aliphatic Cys (or Leu) residue in <em>Tc</em>PRAC at position 270 which is absent and replaced by a polar His residue in HyPRE (R2) thus favouring its interaction with OH-Pro. Additionally, a block of residues (XLA) downstream of the previously identified minimal MIII* PRAC signature <a href="#pone.0000885-Chamond2">[7]</a>, <a href="#pone.0000885-Chamond3">[8]</a> was found to be HyPRE-specific (R3). The combination of those elements are fundamental in shaping the binding pocket and thus determining the substrate specificity as supported by the detailed structural analysis of <em>Tc</em>PRAC and <em>Pa</em>HyPRE active sites. It would be interesting to verify if multiple replacements of discriminating R1, R2 and R3 HyPRE elements by PRAC specific residues would induce any changes in substrate specificity. Nevertheless, although HyPRE and PRAC catalytic sites are structurally very similar, full-sequence disparities between these two enzymes are still substantial (app. 35% homology). Consequently, conformational factors (non-R1, non-R2, non-R3) that contribute to substrate recognition are certainly more subtle and intricate to distinguish than R1, R2 and R3 whose identification was exclusively based on sequence ‘(dis)similarities’. Eventhough, several coexistent mutations might introduce drastic reductions in <em>k<sub>cat</sub></em> and account for a restrained catalytic activity of the mutant and/or “asymmetric” preferences for a particular substrate stereoisomer, thus affecting “racemic principles”. It is also conceivable that major (extra) distortions in the pocket geometry and charge delocalization-previously shown to play a role on ligand accessibility <a href="#pone.0000885-Buschiazzo1">[10]</a>–may occur thus interfering with the analysis.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We report here a clarification of earlier work <a href="#pone.0000885-Spera1">[13]</a> concerning an immunomodulatory virulence factor (PrpA) of <em>B. abortus</em>, that possibly due to its 40% homology with <em>Tc</em>PRAC was described as a PRAC. Surprisingly, PrpA was described as displaying discrete racemization of L-Pro but as unable of catalyzing the conversion of D-Pro enantiomer. As such, this data would imply that some racemases do not follow fundamental racemic principles. The present data establishes that PrpA from <em>B. abortus, B. melitensis</em> and <em>B. suis</em> are in fact HyPRE that catalyze the interconversion of OH-L/D-Pro. These results are significant to prevent any misinterpretations of mechanisms linked to pathogenesis induced by <em>Brucella spp</em>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>HyPRE is a PLP-independent enzyme, shown to be essential in <em>P. putida</em> that, like other <em>Pseudomonas spp,</em> has been found to cause nosocomial infections with resulting septicemia and septic arthritis <a href="#pone.0000885-Kipnis1">[22]</a>, <a href="#pone.0000885-Sadikot1">[24]</a>, <a href="#pone.0000885-Macfarlane1">[26]</a>. Here we identify functional HyPRE from <em>P. aeruginosa</em> and several other important pathogens such as <em>B. pseudomallei</em> and <em>Brucella spp</em> agents of melioidosis and brucellosis, respectively <a href="#pone.0000885-Godfroid1">[21]</a>, <a href="#pone.0000885-Pappas1">[23]</a>, <a href="#pone.0000885-Wiersinga1">[25]</a>. These human and animal pathogens affect multiple systems and result in abcesses, pneumonia and fatal septicemia in immunosuppressed hosts. Bacterial meningitis can also provoke collagen degradation and break-down of blood-brain barrier, which consequently raises bacterial invasiveness and persistence resulting in brain injuries <a href="#pone.0000885-Harrington1">[27]</a>. Interestingly, OH-L-Pro and L-Pro, are the major constituents of collagen, the main component of the extra-cellular matrix, making up 25% of the total body protein content <a href="#pone.0000885-Prockop1">[15]</a>. Bacteria and viruses, deprived of collagen, have virulence factors which destroy collagen or interfere with its production by the secretion of collagenase and/or elastase <a href="#pone.0000885-Miyoshi1">[28]</a>, <a href="#pone.0000885-deBentzmann1">[29]</a>. <em>P. aeruginosa</em>, for instance, induces disruption of blood vessels through elastase by dissolution of the elastic lamina of arteries and arterioles, or by degrading major fragments of collagen IV <a href="#pone.0000885-Bejarano1">[30]</a>. It is worth noting that OH-L-Pro upregulates expression of bacterial genes whose products are involved in vital metabolic pathways, such as OH-D-Pro oxidase, deaminase and dehydrogenase <a href="#pone.0000885-Gryder1">[31]</a>. Specific membrane transporters for OH-L/D-Pro have been shown to exist in bacteria to increase the fundamental intracellular OH-Pro pool <a href="#pone.0000885-Gryder2">[32]</a>. However, mutants lacking HyPRE are unable to metabolize OH-L-Pro and hence are not viable in OH-L-Pro containing media as the sole carbon source <a href="#pone.0000885-Gryder1">[31]</a>. This fact alone confers an essential importance to HyPRE that by converting OH-L-Pro to OH-D-Pro allows its intracellular utilization by the action of OH-D-Pro oxidases. We therefore hypothesize that HyPRE, as PRAC, could serve as a good target for the development of therapies.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Finally, PRAC enzymes, as other B-cell mitogens, have been described as involved in evasion mechanisms of parasite and bacterial species through the induction of non-specific hypergammaglobulinemia and by the secretion of pleiotropic cytokines <a href="#pone.0000885-ReinaSanMartin1">[4]</a>, <a href="#pone.0000885-ReinaSanMartin2">[5]</a>. We have shown that similarly to <em>Tc</em>PRAC, PRAC from <em>C. difficile</em> and HyPREs from <em>P. aeruginosa</em> and <em>B. abortus</em> are also strong lymphocyte mitogens as they increase <em>in vitro</em> lymphoproliferation by up to 10 fold (<a href="#pone.0000885.s005">Table S2</a>). It has been shown that mitogen-induced proliferation of resting lymphocyte is associated with a marked increase in amino acid uptake and intracellular enzyme pathways to meet the demands of increased cellular protein synthesis <a href="#pone.0000885-Valle1">[33]</a>. It is relevant that enzymes of Pro biosynthesis, and not those of Pro degradation, are particularly increased with lymphocyte activation. However, with sufficient amounts of exogenous Pro, large increases are observed of pyrroline-5-carboxylate reductase (PCA reductase), a key enzyme in Pro synthesis. Isoforms of PCA reductase, sensitive and insensitive to feedback inhibition by Pro do exist <a href="#pone.0000885-Valle2">[34]</a>. Interestingly, PCA reductase from distinct tissues differs according to its sensitivity to Pro-inhibition. Considering tissue specificity and tropism of infectious pathogens, it would not be surprising if upon infection PRAC and HyPRE play important roles in the regulation of the intracellular and extracellular amino acid pool profiting the microrganism with precursors and enzymatic pathways of the host.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data mining and bioinformatics</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p><em>Tc</em>PRAC sequence (AF195522, NCBI, E.C.5.1.1.4) and PRAC motif III* were used to blast genome databases. Default settings for Blast were used. Unrooted trees and alignments were obtained with ClustalW program.</p>


<h4>Bacterial strains and DNA extraction</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Purified DNA was obtained from <em>B. anthracis</em> (strain 9131), <em>C. difficile</em> (strain VPI10463), <em>V. parahaemolyticus</em> (CNRVC 010089), <em>B. abortus</em> (strain 544), <em>B. melitensis</em> (strain 16M), <em>B. suis</em> (strain 1330) and <em>B. pseudomallei</em> (strain K96243). DNA was extracted from bacterial pellets of <em>B. cenocepacia</em> (strain J2315) and <em>P. aeruginosa</em> (strain PAK) with the DNA tissue culture extraction kit (Qiagen)</p>


<h4>Primer design, gene cloning and recombinant Proteins</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Forward and Reverse primers were designed based on <em>Tc</em>PRAC sequence toward specific sequences of the genes of interest (<a href="#pone.0000885.s005">Table S2</a>, Supplemental files). Bacterial PCR products were purified by QuickPCR Qiaprep kit (Qiagen) and cloned into BamHI/EcoRI or BamHI/NcoI sites of pET28b (Novagen/Merck) using Rapid Ligation Kit (Roche). <em>E. coli</em> DH5α cells were transformed with empty or ligated plasmids. Plasmids were extracted with the Qiaprep Spin Miniprep kit (Qiagen) from bacterial pellets from individual colony cultures and sequenced (Genome Express, Meylan/France). Sequences, ORFs and the presence of C-terminal 6x-His Tag were verified. <em>E. coli</em> BL21 (DE3) cells were transformed with ligated plasmids. Recombinant proteins were purified as described <a href="#pone.0000885-Chamond2">[7]</a>, <a href="#pone.0000885-Chamond3">[8]</a>.</p>


<h4>Enzymatic activity assays</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Optimum racemization and epimerization conditions were determined using 20 mM L-Pro or OH-L-Pro in 0.2 M NaOAc or Tris 20mM/EDTA 1 mM (TE) buffers respectively, as a function of pH. Percent of racemization or epimerization of serial concentrations of substrate was calculated by incubating 3–10 µg of recombinant protein, 20–80 mM substrate in NaOAc pH 6 or TE, pH 8 (q.s.p. 500 µl) for 30–60 min at 37°C. The reactions were stopped by incubating at −20°C and optical rotations measured in a polarimeter 241MC (Perkin Elmer) <a href="#pone.0000885-Chamond3">[8]</a>. Percent inhibition of enzymatic activities was determined incubating 10 µg of recombinant protein in presence or absence of 1–10 mM PYC, 1–25 mM iodoacetamide, or 1–25 mM iodoacetate. Control reactions were performed in presence or absence of PLP. All reagents were purchased from Sigma.</p>


<h4>Kinetic assays</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Assays were performed at 37°C with 10–160 mM of each substrate, 20 µg/ml of specific enzymes in optimum reaction buffer <a href="#pone.0000885-Chamond3">[8]</a>. Briefly, after determination of the linear part of the curve, velocity in 10–160 mM substrate was measured every 30 s during 5 min to determine <em>K<sub>m</sub></em> and <em>V<sub>max</sub></em>.</p>


<h4>Site directed mutagenesis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Site-directed mutagenesis of <em>Pa</em>HyPRE was performed using a QuikChange XL kit (Stratagene), as described <a href="#pone.0000885-Buschiazzo1">[10]</a>, to obtain the point mutants C88S, C236S, V60F and V60G. Briefly, point mutations were obtained by PCR using forward and reverse overlapping mutagenic primers (<a href="#pone.0000885.s003">Figure S3</a> and <a href="#pone.0000885.s005">Table S2</a>). Plasmids were purified and point mutations were ascertained by sequencing. Recombinant proteins were produced from each point mutant, as described above.</p>


<h4>Accession Numbers</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The following nucleotide sequences were submitted to GenBank™ Data Bank with accession numbers EF495346 (<em>Cd</em>PRAC, <em>C. difficile</em> VPI10463), EF495341 (<em>Pa</em>HyPRE, <em>P. aeruginosa</em> PAK), EF495342 (<em>Bm</em>HyPRE, <em>B. melitensis</em> 16M), EF495343 (<em>Bs</em>HyPRE, <em>B. suis</em> 1330), EF495344 (<em>Ba</em>HyPRE, <em>B. abortus</em> 544), EF495345 (<em>Bp</em>HyPRE, <em>B. pseudomallei</em> K96243).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000885"></div><a name="pone.0000885.s001" id="pone.0000885.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000885.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Sequence alignments of Brucella abortus PrpA and PrpB virulence factors.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000885.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.27 MB TIF)</p>
<a name="pone.0000885.s002" id="pone.0000885.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000885.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Inhibition of HyPRE reactions with alkylating agents.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000885.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.29 MB TIF)</p>
<a name="pone.0000885.s003" id="pone.0000885.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000885.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Strategy for PaHyPRE site specific mutagenesis.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000885.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.33 MB TIF)</p>
<a name="pone.0000885.s004" id="pone.0000885.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000885.s004">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Primers used for the production of recombinant proteins and site-directed mutagenesis.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000885.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.42 MB TIF)</p>
<a name="pone.0000885.s005" id="pone.0000885.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000885.s005">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Mitogenic activity of PRAC and HyPRE enzymes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000885.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.40 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>Special thanks to our collaborators P. Arimeter, for availability and useful tips, A. Blondel, L. Masgrau, Y. Janin and P. Alzari for helpful discussions and suggestions. We are thankful to the colleagues of the Collection Nationale de Culture de Microrganismes, Institut Pasteur, C. Fitting for testing endotoxin from our recombinant protein preparations and P. Goossens, M.R. Poppof, M. Fournier, M.L. Quilici, V. Balloy and M. Si-Tahar for providing bacterial strains or bacterial purified DNA. We are grateful to B. Garin-Bastuji and D. Albert (AFSSA, Maisons-Alfort) and F. Thibault (CRSSA, La Tronche) for bacterial DNA.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PM. Performed the experiments: MG NC AC NC DH. Analyzed the data: PM MG NC AB. Contributed reagents/materials/analysis tools: AB. Wrote the paper: PM MG NC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000885-Stadtman1" id="pone.0000885-Stadtman1"></a>Stadtman TC, Elliott P (1957) Studies on the enzymic reduction of amino acids. II. Purification and properties of D-proline reductase and a proline racemase from <em>Clostridium sticklandii</em>. J Biol Chem  228(2): 983–997.  <ul class="find" data-citedArticleID="991466"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Studies+on+the+enzymic+reduction+of+amino+acids.+II.+Purification+and+properties+of+D-proline+reductase+and+a+proline+racemase+from+Clostridium+sticklandii.&amp;auth=&amp;atitle=Studies+on+the+enzymic+reduction+of+amino+acids.+II.+Purification+and+properties+of+D-proline+reductase+and+a+proline+racemase+from+Clostridium+sticklandii." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Studies+on+the+enzymic+reduction+of+amino+acids.+II.+Purification+and+properties+of+D-proline+reductase+and+a+proline+racemase+from+Clostridium+sticklandii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Studies+on+the+enzymic+reduction+of+amino+acids.+II.+Purification+and+properties+of+D-proline+reductase+and+a+proline+racemase+from+Clostridium+sticklandii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000885-Belasco1" id="pone.0000885-Belasco1"></a>Belasco JG, Albery WJ, Knowles JR (1986) Energetics of proline racemase: double fractionation experiment, a test for concertedness and for transition-state dominance. Biochemistry  25(9): 2552–2558.  <ul class="find" data-citedArticleID="991420" data-doi="10.1021/bi00357a040"><li><a href="http://dx.doi.org/10.1021/bi00357a040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Energetics+of+proline+racemase%3A+double+fractionation+experiment%2C+a+test+for+concertedness+and+for+transition-state+dominance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Energetics+of+proline+racemase%3A+double+fractionation+experiment%2C+a+test+for+concertedness+and+for+transition-state+dominance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000885-Fisher1" id="pone.0000885-Fisher1"></a>Fisher LM, Albery WJ, Knowles JR (1986) Energetics of proline racemase: racemization of unlabeled proline in the unsaturated, saturated, and oversaturated regimes. Biochemistry  25(9): 2529–2537.  <ul class="find" data-citedArticleID="991434" data-doi="10.1021/bi00357a037"><li><a href="http://dx.doi.org/10.1021/bi00357a037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Energetics+of+proline+racemase%3A+racemization+of+unlabeled+proline+in+the+unsaturated%2C+saturated%2C+and+oversaturated+regimes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Energetics+of+proline+racemase%3A+racemization+of+unlabeled+proline+in+the+unsaturated%2C+saturated%2C+and+oversaturated+regimes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000885-ReinaSanMartin1" id="pone.0000885-ReinaSanMartin1"></a>Reina-San-Martin B, Cosson A, Minoprio P (2000) Lymphocyte polyclonal activation: a pitfall for vaccine design against infectious agents. Parasitology today (Personal ed  16(2): 62–67.  <ul class="find" data-citedArticleID="991458" data-doi="10.1016/s0169-4758(99)01591-4"><li><a href="http://dx.doi.org/10.1016/s0169-4758(99)01591-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lymphocyte+polyclonal+activation%3A+a+pitfall+for+vaccine+design+against+infectious+agents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lymphocyte+polyclonal+activation%3A+a+pitfall+for+vaccine+design+against+infectious+agents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000885-ReinaSanMartin2" id="pone.0000885-ReinaSanMartin2"></a>Reina-San-Martin B, Degrave W, Rougeot C, Cosson A, Chamond N, et al.  (2000) A B-cell mitogen from a pathogenic trypanosome is a eukaryotic proline racemase. Nat Med  6(8): 890–897.  <ul class="find" data-citedArticleID="991460" data-doi="10.1038/78651"><li><a href="http://dx.doi.org/10.1038/78651" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+B-cell+mitogen+from+a+pathogenic+trypanosome+is+a+eukaryotic+proline+racemase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+B-cell+mitogen+from+a+pathogenic+trypanosome+is+a+eukaryotic+proline+racemase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000885-Chamond1" id="pone.0000885-Chamond1"></a>Chamond N, Goytia M, Coatnoan N, Barale JC, Cosson A, et al.  (2005) <em>Trypanosoma cruzi</em> proline racemases are involved in parasite differentiation and infectivity. Mol Microbiol  58(1): 46–60.  <ul class="find" data-citedArticleID="991428" data-doi="10.1111/j.1365-2958.2005.04808.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2005.04808.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trypanosoma+cruzi+proline+racemases+are+involved+in+parasite+differentiation+and+infectivity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trypanosoma+cruzi+proline+racemases+are+involved+in+parasite+differentiation+and+infectivity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000885-Chamond2" id="pone.0000885-Chamond2"></a>Chamond N, Coatnoan N, Minoprio P (2002) Immunotherapy of <em>Trypanosoma cruzi</em> infections. Current drug targets  2(3): 247–254.  <ul class="find" data-citedArticleID="991430" data-doi="10.2174/1568008023340604"><li><a href="http://dx.doi.org/10.2174/1568008023340604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunotherapy+of+Trypanosoma+cruzi+infections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunotherapy+of+Trypanosoma+cruzi+infections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000885-Chamond3" id="pone.0000885-Chamond3"></a>Chamond N, Gregoire C, Coatnoan N, Rougeot C, Freitas-Junior LH, et al.  (2003) Biochemical characterization of proline racemases from the human protozoan parasite <em>Trypanosoma cruzi</em> and definition of putative protein signatures. J Biol Chem  278(18): 15484–15494.  <ul class="find" data-citedArticleID="991432" data-doi="10.1074/jbc.m210830200"><li><a href="http://dx.doi.org/10.1074/jbc.m210830200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical+characterization+of+proline+racemases+from+the+human+protozoan+parasite+Trypanosoma+cruzi+and+definition+of+putative+protein+signatures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical+characterization+of+proline+racemases+from+the+human+protozoan+parasite+Trypanosoma+cruzi+and+definition+of+putative+protein+signatures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000885-Yoshimura1" id="pone.0000885-Yoshimura1"></a>Yoshimura T, Esak N (2003) Amino acid racemases: functions and mechanisms. Journal of bioscience and bioengineering  96(2): 103–109.  <ul class="find" data-citedArticleID="991476" data-doi="10.1016/s1389-1723(03)90111-3"><li><a href="http://dx.doi.org/10.1016/s1389-1723(03)90111-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+acid+racemases%3A+functions+and+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+acid+racemases%3A+functions+and+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000885-Buschiazzo1" id="pone.0000885-Buschiazzo1"></a>Buschiazzo A, Goytia M, Schaeffer F, Degrave W, Shepard W, et al.  (2006) Crystal structure, catalytic mechanism, and mitogenic properties of <em>Trypanosoma cruzi</em> proline racemase. Proc Natl Acad Sci U S A  103(6): 1705–1710.  <ul class="find" data-citedArticleID="991424" data-doi="10.1073/pnas.0509010103"><li><a href="http://dx.doi.org/10.1073/pnas.0509010103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure%2C+catalytic+mechanism%2C+and+mitogenic+properties+of+Trypanosoma+cruzi+proline+racemase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure%2C+catalytic+mechanism%2C+and+mitogenic+properties+of+Trypanosoma+cruzi+proline+racemase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000885-Adams1" id="pone.0000885-Adams1"></a>Adams E (1959) Hydroxyproline metabolism. I. Conversion to alpha-ketoglutarate by extracts of <em>Pseudomonas</em>. J Biol Chem  234(8): 2073–2084.  <ul class="find" data-citedArticleID="991412"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hydroxyproline+metabolism.+I.+Conversion+to+alpha-ketoglutarate+by+extracts+of+Pseudomonas.&amp;auth=&amp;atitle=Hydroxyproline+metabolism.+I.+Conversion+to+alpha-ketoglutarate+by+extracts+of+Pseudomonas." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hydroxyproline+metabolism.+I.+Conversion+to+alpha-ketoglutarate+by+extracts+of+Pseudomonas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hydroxyproline+metabolism.+I.+Conversion+to+alpha-ketoglutarate+by+extracts+of+Pseudomonas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000885-Radhakrishnan1" id="pone.0000885-Radhakrishnan1"></a>Radhakrishnan AN, Meister A (1957) Conversion of hydroxyproline to pyrrole-2-carboxylic acid. J Biol Chem  226(1): 559–571.  <ul class="find" data-citedArticleID="991454"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Conversion+of+hydroxyproline+to+pyrrole-2-carboxylic+acid.&amp;auth=&amp;atitle=Conversion+of+hydroxyproline+to+pyrrole-2-carboxylic+acid." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conversion+of+hydroxyproline+to+pyrrole-2-carboxylic+acid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conversion+of+hydroxyproline+to+pyrrole-2-carboxylic+acid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000885-Spera1" id="pone.0000885-Spera1"></a>Spera JM, Ugalde JE, Mucci J, Comerci DJ, Ugalde RA (2006) A B lymphocyte mitogen is a <em>Brucella abortus</em> virulence factor required for persistent infection. Proc Natl Acad Sci U S A  103(44): 16514–16519.  <ul class="find" data-citedArticleID="991464" data-doi="10.1073/pnas.0603362103"><li><a href="http://dx.doi.org/10.1073/pnas.0603362103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+B+lymphocyte+mitogen+is+a+Brucella+abortus+virulence+factor+required+for+persistent+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+B+lymphocyte+mitogen+is+a+Brucella+abortus+virulence+factor+required+for+persistent+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000885-Adams2" id="pone.0000885-Adams2"></a>Adams E (1970) Metabolism of proline and of hydroxyproline. Int Rev Connect Tissue Res  5: 1–91.  <ul class="find" data-citedArticleID="991414" data-doi="10.1016/b978-0-12-363705-5.50007-5"><li><a href="http://dx.doi.org/10.1016/b978-0-12-363705-5.50007-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolism+of+proline+and+of+hydroxyproline." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolism+of+proline+and+of+hydroxyproline.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000885-Prockop1" id="pone.0000885-Prockop1"></a>Prockop DJ, Kivirikko KI (1995) Collagens: molecular biology, diseases, and potentials for therapy. Annu Rev Biochem  64: 403–434.  <ul class="find" data-citedArticleID="991452" data-doi="10.1146/annurev.bi.64.070195.002155"><li><a href="http://dx.doi.org/10.1146/annurev.bi.64.070195.002155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Collagens%3A+molecular+biology%2C+diseases%2C+and+potentials+for+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Collagens%3A+molecular+biology%2C+diseases%2C+and+potentials+for+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000885-Cardinale1" id="pone.0000885-Cardinale1"></a>Cardinale GJ, Abeles RH (1968) Purification and mechanism of action of proline racemase. Biochemistry  7(11): 3970–3978.  <ul class="find" data-citedArticleID="991426" data-doi="10.1021/bi00851a026"><li><a href="http://dx.doi.org/10.1021/bi00851a026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification+and+mechanism+of+action+of+proline+racemase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification+and+mechanism+of+action+of+proline+racemase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000885-Adams3" id="pone.0000885-Adams3"></a>Adams E, Norton IL (1964) Purification and Properties of Inducible Hydroxyproline 2-Epimerase from <em>Pseudomonas</em>. J Biol Chem  239: 1525–1535.  <ul class="find" data-citedArticleID="991416"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Purification+and+Properties+of+Inducible+Hydroxyproline+2-Epimerase+from+Pseudomonas.&amp;auth=&amp;atitle=Purification+and+Properties+of+Inducible+Hydroxyproline+2-Epimerase+from+Pseudomonas." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification+and+Properties+of+Inducible+Hydroxyproline+2-Epimerase+from+Pseudomonas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification+and+Properties+of+Inducible+Hydroxyproline+2-Epimerase+from+Pseudomonas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000885-Ramaswamy1" id="pone.0000885-Ramaswamy1"></a>Ramaswamy SG (1984) Hydroxyproline 2-epimerase of <em>Pseudomonas</em>. Subunit structure and active site studies. J Biol Chem  259(1): 249–254.  <ul class="find" data-citedArticleID="991456"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hydroxyproline+2-epimerase+of+Pseudomonas.+Subunit+structure+and+active+site+studies.&amp;auth=&amp;atitle=Hydroxyproline+2-epimerase+of+Pseudomonas.+Subunit+structure+and+active+site+studies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hydroxyproline+2-epimerase+of+Pseudomonas.+Subunit+structure+and+active+site+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hydroxyproline+2-epimerase+of+Pseudomonas.+Subunit+structure+and+active+site+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000885-Bouza1" id="pone.0000885-Bouza1"></a>Bouza E, Munoz P, Alonso R (2005) Clinical manifestations, treatment and control of infections caused by <em>Clostridium difficile</em>. Clin Microbiol Infect 11 Suppl  4: 57–64.  <ul class="find" data-citedArticleID="991422" data-doi="10.1111/j.1469-0691.2005.01165.x"><li><a href="http://dx.doi.org/10.1111/j.1469-0691.2005.01165.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clinical+manifestations%2C+treatment+and+control+of+infections+caused+by+Clostridium+difficile." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clinical+manifestations%2C+treatment+and+control+of+infections+caused+by+Clostridium+difficile.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000885-Stoddart1" id="pone.0000885-Stoddart1"></a>Stoddart B, Wilcox MH (2002) <em>Clostridium difficile</em>. Curr Opin Infect Dis  15(5): 513–518.  <ul class="find" data-citedArticleID="991468" data-doi="10.1097/00001432-200210000-00010"><li><a href="http://dx.doi.org/10.1097/00001432-200210000-00010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clostridium+difficile." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clostridium+difficile.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000885-Godfroid1" id="pone.0000885-Godfroid1"></a>Godfroid J, Cloeckaert A, Liautard JP, Kohler S, Fretin D, et al.  (2005) From the discovery of the Malta fever's agent to the discovery of a marine mammal reservoir, brucellosis has continuously been a re-emerging zoonosis. Vet Res  36(3): 313–326.  <ul class="find" data-citedArticleID="991436" data-doi="10.1051/vetres:2005003"><li><a href="http://dx.doi.org/10.1051/vetres:2005003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+the+discovery+of+the+Malta+fever%27s+agent+to+the+discovery+of+a+marine+mammal+reservoir%2C+brucellosis+has+continuously+been+a+re-emerging+zoonosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+the+discovery+of+the+Malta+fever%27s+agent+to+the+discovery+of+a+marine+mammal+reservoir%2C+brucellosis+has+continuously+been+a+re-emerging+zoonosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000885-Kipnis1" id="pone.0000885-Kipnis1"></a>Kipnis E, Sawa T, Wiener-Kronish J (2006) Targeting mechanisms of <em>Pseudomonas aeruginosa</em> pathogenesis. Med Mal Infect  36(2): 78–91.  <ul class="find" data-citedArticleID="991444" data-doi="10.1016/j.medmal.2005.10.007"><li><a href="http://dx.doi.org/10.1016/j.medmal.2005.10.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeting+mechanisms+of+Pseudomonas+aeruginosa+pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeting+mechanisms+of+Pseudomonas+aeruginosa+pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000885-Pappas1" id="pone.0000885-Pappas1"></a>Pappas G, Akritidis N, Bosilkovski M, Tsianos E (2005) Brucellosis. N Engl J Med  352(22): 2325–2336.  <ul class="find" data-citedArticleID="991450" data-doi="10.1056/nejmra050570"><li><a href="http://dx.doi.org/10.1056/nejmra050570" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Brucellosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Brucellosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000885-Sadikot1" id="pone.0000885-Sadikot1"></a>Sadikot RT, Blackwell TS, Christman JW, Prince AS (2005) Pathogen-host interactions in <em>Pseudomonas aeruginosa</em> pneumonia. Am J Respir Crit Care Med  171(11): 1209–1223.  <ul class="find" data-citedArticleID="991462" data-doi="10.1164/rccm.200408-1044so"><li><a href="http://dx.doi.org/10.1164/rccm.200408-1044so" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathogen-host+interactions+in+Pseudomonas+aeruginosa+pneumonia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathogen-host+interactions+in+Pseudomonas+aeruginosa+pneumonia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000885-Wiersinga1" id="pone.0000885-Wiersinga1"></a>Wiersinga WJ, van der Poll T, White NJ, Day NP, Peacock SJ (2006) Melioidosis: insights into the pathogenicity of <em>Burkholderia pseudomallei</em>. Nat Rev Microbiol  4(4): 272–282.  <ul class="find" data-citedArticleID="991474" data-doi="10.1038/nrmicro1385"><li><a href="http://dx.doi.org/10.1038/nrmicro1385" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Melioidosis%3A+insights+into+the+pathogenicity+of+Burkholderia+pseudomallei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Melioidosis%3A+insights+into+the+pathogenicity+of+Burkholderia+pseudomallei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000885-Macfarlane1" id="pone.0000885-Macfarlane1"></a>Macfarlane L, Oppenheim BA, Lorrigan P (1991) Septicaemia and septic arthritis due to <em>Pseudomonas putida</em> in a neutropenic patient. The Journal of infection  23(3): 346–347.  <ul class="find" data-citedArticleID="991446" data-doi="10.1016/0163-4453(91)93620-r"><li><a href="http://dx.doi.org/10.1016/0163-4453(91)93620-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Septicaemia+and+septic+arthritis+due+to+Pseudomonas+putida+in+a+neutropenic+patient." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Septicaemia+and+septic+arthritis+due+to+Pseudomonas+putida+in+a+neutropenic+patient.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000885-Harrington1" id="pone.0000885-Harrington1"></a>Harrington DJ (1996) Bacterial collagenases and collagen-degrading enzymes and their potential role in human disease. Infection and immunity  64(6): 1885–1891.  <ul class="find" data-citedArticleID="991442"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bacterial+collagenases+and+collagen-degrading+enzymes+and+their+potential+role+in+human+disease.&amp;auth=&amp;atitle=Bacterial+collagenases+and+collagen-degrading+enzymes+and+their+potential+role+in+human+disease." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+collagenases+and+collagen-degrading+enzymes+and+their+potential+role+in+human+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+collagenases+and+collagen-degrading+enzymes+and+their+potential+role+in+human+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000885-Miyoshi1" id="pone.0000885-Miyoshi1"></a>Miyoshi S, Shinoda S (2000) Microbial metalloproteases and pathogenesis. Microbes Infect  2(1): 91–98.  <ul class="find" data-citedArticleID="991448" data-doi="10.1016/s1286-4579(00)00280-x"><li><a href="http://dx.doi.org/10.1016/s1286-4579(00)00280-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+metalloproteases+and+pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+metalloproteases+and+pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000885-deBentzmann1" id="pone.0000885-deBentzmann1"></a>de Bentzmann S, Polette M, Zahm JM, Hinnrasky J, Kileztky C, et al.  (2000) <em>Pseudomonas aeruginosa</em> virulence factors delay airway epithelial wound repair by altering the actin cytoskeleton and inducing overactivation of epithelial matrix metalloproteinase-2. Laboratory investigation; a journal of technical methods and pathology  80(2): 209–219.  <ul class="find" data-citedArticleID="991478" data-doi="10.1038/labinvest.3780024"><li><a href="http://dx.doi.org/10.1038/labinvest.3780024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pseudomonas+aeruginosa+virulence+factors+delay+airway+epithelial+wound+repair+by+altering+the+actin+cytoskeleton+and+inducing+overactivation+of+epithelial+matrix+metalloproteinase-2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pseudomonas+aeruginosa+virulence+factors+delay+airway+epithelial+wound+repair+by+altering+the+actin+cytoskeleton+and+inducing+overactivation+of+epithelial+matrix+metalloproteinase-2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000885-Bejarano1" id="pone.0000885-Bejarano1"></a>Bejarano PA, Langeveld JP, Hudson BG, Noelken ME (1989) Degradation of basement membranes by <em>Pseudomonas aeruginosa</em> elastase. Infection and immunity  57(12): 3783–3787.  <ul class="find" data-citedArticleID="991418"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Degradation+of+basement+membranes+by+Pseudomonas+aeruginosa+elastase.&amp;auth=&amp;atitle=Degradation+of+basement+membranes+by+Pseudomonas+aeruginosa+elastase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Degradation+of+basement+membranes+by+Pseudomonas+aeruginosa+elastase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Degradation+of+basement+membranes+by+Pseudomonas+aeruginosa+elastase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000885-Gryder1" id="pone.0000885-Gryder1"></a>Gryder RM, Adams E (1969) Inducible degradation of hydroxyproline in Pseudomonas putida: pathway regulation and hydroxyproline uptake. J Bacteriol  97(1): 292–306.  <ul class="find" data-citedArticleID="991438"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inducible+degradation+of+hydroxyproline+in+Pseudomonas+putida%3A+pathway+regulation+and+hydroxyproline+uptake.&amp;auth=&amp;atitle=Inducible+degradation+of+hydroxyproline+in+Pseudomonas+putida%3A+pathway+regulation+and+hydroxyproline+uptake." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inducible+degradation+of+hydroxyproline+in+Pseudomonas+putida%3A+pathway+regulation+and+hydroxyproline+uptake." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inducible+degradation+of+hydroxyproline+in+Pseudomonas+putida%3A+pathway+regulation+and+hydroxyproline+uptake.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000885-Gryder2" id="pone.0000885-Gryder2"></a>Gryder RM, Adams E (1970) Properties of the inducible hydroxyproline transport system of <em>Pseudomonas putida</em>. J Bacteriol  101(3): 948–958.  <ul class="find" data-citedArticleID="991440"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Properties+of+the+inducible+hydroxyproline+transport+system+of+Pseudomonas+putida.&amp;auth=&amp;atitle=Properties+of+the+inducible+hydroxyproline+transport+system+of+Pseudomonas+putida." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Properties+of+the+inducible+hydroxyproline+transport+system+of+Pseudomonas+putida." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Properties+of+the+inducible+hydroxyproline+transport+system+of+Pseudomonas+putida.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000885-Valle1" id="pone.0000885-Valle1"></a>Valle D, Blaese RM, Phang JM (1975) Increased sensitivity of lymphocyte delta1-pyrroline-5-carboxylate reductase to inhibition by proline with transformation. Nature  253(5488): 214–216.  <ul class="find" data-citedArticleID="991470" data-doi="10.1038/253214a0"><li><a href="http://dx.doi.org/10.1038/253214a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Increased+sensitivity+of+lymphocyte+delta1-pyrroline-5-carboxylate+reductase+to+inhibition+by+proline+with+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Increased+sensitivity+of+lymphocyte+delta1-pyrroline-5-carboxylate+reductase+to+inhibition+by+proline+with+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000885-Valle2" id="pone.0000885-Valle2"></a>Valle D, Downing SJ, Phang JM (1973) Proline inhibition of pyrroline-5-carboxylate reductase: differences in enzymes obtained from animal and tissue culture sources. Biochemical and biophysical research communications  54(4): 1418–1424.  <ul class="find" data-citedArticleID="991472" data-doi="10.1016/0006-291x(73)91144-3"><li><a href="http://dx.doi.org/10.1016/0006-291x(73)91144-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proline+inhibition+of+pyrroline-5-carboxylate+reductase%3A+differences+in+enzymes+obtained+from+animal+and+tissue+culture+sources." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proline+inhibition+of+pyrroline-5-carboxylate+reductase%3A+differences+in+enzymes+obtained+from+animal+and+tissue+culture+sources.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.XML" value="89677"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.PDF" value="393767"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.t001.PNG_L" value="196370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.t001.PNG_M" value="100514"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.t001.PNG_S" value="12448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.t001.TIF" value="855608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.t001.PNG_I" value="31172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g001.PNG_L" value="163695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g001.PNG_M" value="37186"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g001.PNG_S" value="7062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g001.TIF" value="247732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g001.PNG_I" value="13075"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g002.PNG_L" value="344415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g002.PNG_M" value="61696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g002.PNG_S" value="9807"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g002.TIF" value="510030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g002.PNG_I" value="21002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g003.PNG_L" value="120997"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g003.PNG_M" value="29673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g003.PNG_S" value="6635"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g003.TIF" value="218398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g003.PNG_I" value="10727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g004.PNG_L" value="221649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g004.PNG_M" value="57080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g004.PNG_S" value="9481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g004.TIF" value="327554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g004.PNG_I" value="20144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g005.PNG_L" value="1663318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g005.PNG_M" value="118203"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g005.PNG_S" value="5293"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g005.TIF" value="1967954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g005.PNG_I" value="35796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g006.PNG_L" value="106345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g006.PNG_M" value="26206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g006.PNG_S" value="5932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g006.TIF" value="166208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g006.PNG_I" value="9456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g007.PNG_L" value="4717036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g007.PNG_M" value="211070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g007.PNG_S" value="21012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g007.TIF" value="5819094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.g007.PNG_I" value="180867"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.s001.TIF" value="869312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.s002.TIF" value="283748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.s003.TIF" value="319864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.s004.TIF" value="416044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000885.s005.TIF" value="392804"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000885&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000885" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000885&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000885&volume=&issue=&title=Molecular and Structural Discrimination of Proline Racemase and Hydroxyproline-2-Epimerase from Nosocomial and Bacterial Pathogens&author_name=Maira%20Goytia%2C%20Nathalie%20Chamond%2C%20Alain%20Cosson%2C%20Nicolas%20Coatnoan%2C%20Daniel%20Hermant%2C%20Armand%20Berneman%2C%20Paola%20Minoprio&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000885" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000885" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000885" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000885&amp;t=Molecular%20and%20Structural%20Discrimination%20of%20Proline%20Racemase%20and%20Hydroxyproline-2-Epimerase%20from%20Nosocomial%20and%20Bacterial%20Pathogens" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000885&title=Molecular%20and%20Structural%20Discrimination%20of%20Proline%20Racemase%20and%20Hydroxyproline-2-Epimerase%20from%20Nosocomial%20and%20Bacterial%20Pathogens&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000885&amp;title=Molecular%20and%20Structural%20Discrimination%20of%20Proline%20Racemase%20and%20Hydroxyproline-2-Epimerase%20from%20Nosocomial%20and%20Bacterial%20Pathogens" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000885" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000885&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Molecular and Structural Discrimination of Proline Racemase and Hydroxyproline-2-Epimerase from Nosocomial and Bacterial Pathogens';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000885';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Molecular%20and%20Structural%20Discrimination%20of%20Proline%20Racemase%20and%20Hydroxyproline-2-Epimerase%20from%20Nosocomial%20and%20Bacterial%20Pathogens http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000885" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000885" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+pathogens%22" title="Search for articles in the subject area:'Bacterial pathogens'"><div class="flagText">Bacterial pathogens</div></a>
              <div data-categoryid="42717" data-articleid="24608"
                   data-categoryname="Bacterial pathogens"
                   class="flagImage" title="Flag 'Bacterial pathogens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Brucella%22" title="Search for articles in the subject area:'Brucella'"><div class="flagText">Brucella</div></a>
              <div data-categoryid="44457" data-articleid="24608"
                   data-categoryname="Brucella"
                   class="flagImage" title="Flag 'Brucella' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzymes%22" title="Search for articles in the subject area:'Enzymes'"><div class="flagText">Enzymes</div></a>
              <div data-categoryid="17863" data-articleid="24608"
                   data-categoryname="Enzymes"
                   class="flagImage" title="Flag 'Enzymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proline%22" title="Search for articles in the subject area:'Proline'"><div class="flagText">Proline</div></a>
              <div data-categoryid="40969" data-articleid="24608"
                   data-categoryname="Proline"
                   class="flagImage" title="Flag 'Proline' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Pseudomonas+aeruginosa%22" title="Search for articles in the subject area:'Pseudomonas aeruginosa'"><div class="flagText">Pseudomonas aeruginosa</div></a>
              <div data-categoryid="42911" data-articleid="24608"
                   data-categoryname="Pseudomonas aeruginosa"
                   class="flagImage" title="Flag 'Pseudomonas aeruginosa' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombinant+proteins%22" title="Search for articles in the subject area:'Recombinant proteins'"><div class="flagText">Recombinant proteins</div></a>
              <div data-categoryid="18471" data-articleid="24608"
                   data-categoryname="Recombinant proteins"
                   class="flagImage" title="Flag 'Recombinant proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="24608"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="24608"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4142'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1206'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2881&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
