

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Unusually Long Palindromes Are Abundant in Mitochondrial Control Regions of Insects and Nematodes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000110"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000110" />

    <meta name="citation_title" content="Unusually Long Palindromes Are Abundant in Mitochondrial Control Regions of Insects and Nematodes"/>
    <meta itemprop="name" content="Unusually Long Palindromes Are Abundant in Mitochondrial Control Regions of Insects and Nematodes"/>

      <meta name="citation_author" content="K. P. Arunkumar"/>
            <meta name="citation_author_institution" content="Laboratory of Molecular Genetics, Centre for DNA Fingerprinting and Diagnostics, Nacharam, India"/>
      <meta name="citation_author" content="Javaregowda Nagaraju"/>
            <meta name="citation_author_institution" content="Laboratory of Molecular Genetics, Centre for DNA Fingerprinting and Diagnostics, Nacharam, India"/>

    <meta name="citation_date" content="2006/12/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000110.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e110"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="1"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Insertion, excision, and inversion of Tn5.; citation_author=DE Berg; citation_author=C Egner; citation_author=BJ Hirschel; citation_author=J Howard; citation_author=L Johnsrud; citation_journal_title=Cold Spring Harb Symp Quant Biol; citation_volume=45; citation_number=1; citation_pages=115-123; citation_date=1981; " />
      <meta name="citation_reference" content="citation_author=M Waterman; citation_number=2; citation_date=1995; citation_publisher=Chapman and Hall; " />
      <meta name="citation_reference" content="citation_author=E Wagner; citation_number=3; citation_date=1991; citation_publisher=CRC Press; " />
      <meta name="citation_reference" content="citation_author=A Kornberg; citation_author=T Baker; citation_number=4; citation_date=1992; citation_publisher=W. Freeman Co; " />
      <meta name="citation_reference" content="citation_title=Palindrome regeneration by template strand-switching mechanism at the origin of DNA replication of porcine circovirus via the rolling-circle melting-pot replication model.; citation_author=AK Cheung; citation_journal_title=J Virol; citation_volume=78; citation_number=5; citation_pages=9016-9029; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cloning, sequencing, and functional analysis of oriL, a herpes simplex virus type 1 origin of DNA synthesis.; citation_author=S Weller; citation_author=A Spadaro; citation_author=J Schaffer; citation_author=A Murray; citation_author=A Maxam; citation_journal_title=Mol Cell Biol; citation_volume=5; citation_number=6; citation_pages=930-942; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=A putative origin of Replication 29 of plasmids derived from Epstein-Barr virus is composed of two cis-acting components.; citation_author=D Reisman; citation_author=J Yates; citation_author=B Sugden; citation_journal_title=Mol Cell Biol; citation_volume=5; citation_number=7; citation_pages=1822-1832; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Human cytomegalovirus origin of DNA replication (oriLyt) resides within a highly complex repetitive region.; citation_author=M Masse; citation_author=S Karlin; citation_author=G Schachtel; citation_author=E Mocarski; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=89; citation_number=8; citation_pages=5246-5250; citation_date=1992; " />
      <meta name="citation_reference" content="citation_author=T Creighton; citation_number=9; citation_date=1993; citation_publisher=W. H. Freeman and Co; " />
      <meta name="citation_reference" content="citation_title=Long DNA palindromes, cruciform structures, genetic instability and secondary structure repair.; citation_author=D Leach; citation_journal_title=Bioessays; citation_volume=16; citation_number=10; citation_pages=893-900; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Yeast ARMs (DNA at-risk motifs) can reveal sources of genome instability.; citation_author=DA Gordenin; citation_author=MA Resnick; citation_journal_title=Mutat Res; citation_volume=400; citation_number=11; citation_pages=45-58; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Instability of long inverted repeats within mouse transgenes.; citation_author=A Collick; citation_author=J Drew; citation_author=J Penberth; citation_author=P Bois; citation_author=J Luckett; citation_journal_title=Embo J; citation_volume=15; citation_number=12; citation_pages=1163-1171; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Palindrome resolution and recombination in the mammalian germ line.; citation_author=E Akgun; citation_author=J Zahn; citation_author=S Baumes; citation_author=G Brown; citation_author=F Liang; citation_journal_title=Mol Cell Biol; citation_volume=17; citation_number=13; citation_pages=5559-5570; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Animal mitochondrial DNA: structure and evolution.; citation_author=DR Wolstenholme; citation_journal_title=Int Rev Cytol; citation_volume=141; citation_number=14; citation_pages=173-216; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA maintenance in vertebrates.; citation_author=GS Shadel; citation_author=DA Clayton; citation_journal_title=Annu Rev Biochem; citation_volume=66; citation_number=15; citation_pages=409-435; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The mitochondrial genome: structure, transcription, translation and replication.; citation_author=JW Taanman; citation_journal_title=Biochim Biophys Acta; citation_volume=1410; citation_number=16; citation_pages=103-123; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Replication of animal mitochondrial DNA.; citation_author=D Clayton; citation_journal_title=Cell; citation_volume=28; citation_number=17; citation_pages=693-705; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Structural heterogeneity of mitochondrial DNA molecules within the genus Drosophila.; citation_author=CM Fauron; citation_author=DR Wolstenholme; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=73; citation_number=18; citation_pages=3623-3627; citation_date=1976; " />
      <meta name="citation_reference" content="citation_title=BLAST 2 Sequences, a new tool for comparing protein and nucleotide sequences.; citation_author=TA Tatusova; citation_author=TL Madden; citation_journal_title=FEMS Microbiol Lett; citation_volume=174; citation_number=19; citation_pages=247-250; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial evolution.; citation_author=MW Gray; citation_author=G Burger; citation_author=BF Lang; citation_journal_title=Science; citation_volume=283; citation_number=20; citation_pages=1476-1481; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Replication origin of mitochondrial DNA in insects.; citation_author=S Saito; citation_author=K Tamura; citation_author=T Aotsuka; citation_journal_title=Genetics; citation_volume=171; citation_number=21; citation_pages=1695-1705; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bidirectional replication initiates at sites throughout the mitochondrial genome of birds.; citation_author=A Reyes; citation_author=MY Yang; citation_author=M Bowmaker; citation_author=IJ Holt; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=22; citation_pages=3242-3250; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular aspects of genetic instability of an artificial 68 bp perfect palindrome in Escherichia coli.; citation_author=A Shafferman; citation_author=Y Flashner; citation_author=I Hertman; citation_author=Y Olami; citation_author=S Cohen; citation_journal_title=Mol Gen Genet; citation_volume=208; citation_number=23; citation_pages=294-300; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=On the deletion of inverted repeated DNA in Escherichia coli: effects of length, thermal stability, and cruciform formation in vivo.; citation_author=RR Sinden; citation_author=GX Zheng; citation_author=RG Brankamp; citation_author=KN Allen; citation_journal_title=Genetics; citation_volume=129; citation_number=24; citation_pages=991-1005; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Inverted repeats, stem-loops, and cruciforms: significance for initiation of DNA replication.; citation_author=CE Pearson; citation_author=H Zorbas; citation_author=GB Price; citation_author=M Zannis-Hadjopoulos; citation_journal_title=J Cell Biochem; citation_volume=63; citation_number=25; citation_pages=1-22; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Instability of palindromic DNA in Escherichia coli.; citation_author=J Collins; citation_journal_title=Cold Spring Harb Symp Quant Biol; citation_volume=45; citation_number=26; citation_pages=409-416; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Precise and nearly-precise excision of the symmetrical inverted repeats of Tn5; common features of recA-independent deletion events in Escherichia coli.; citation_author=J Collins; citation_author=G Volckaert; citation_author=P Nevers; citation_journal_title=Gene; citation_volume=19; citation_number=27; citation_pages=139-146; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=On the formation of spontaneous deletions: the importance of short sequence homologies in the generation of large deletions.; citation_author=AM Albertini; citation_author=M Hofer; citation_author=MP Calos; citation_author=JH Miller; citation_journal_title=Cell; citation_volume=29; citation_number=28; citation_pages=319-328; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Structural intermediates of deletion mutagenesis: a role for palindromic DNA.; citation_author=BW Glickman; citation_author=LS Ripley; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=81; citation_number=29; citation_pages=512-516; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Spontaneous deletion formation at the aprt locus of hamster cells: the presence of short sequence homologies and dyad symmetries at deletion termini.; citation_author=J Nalbantoglu; citation_author=D Hartley; citation_author=G Phear; citation_author=G Tear; citation_author=M Meuth; citation_journal_title=Embo J; citation_volume=5; citation_number=30; citation_pages=1199-1204; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Excision of transposon Tn5 is dependent on the inverted repeats but not on the transposase function of Tn5.; citation_author=C Egner; citation_author=DE Berg; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=78; citation_number=31; citation_pages=459-463; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Three Tn10-associated excision events: relationship to transposition and role of direct and inverted repeats.; citation_author=TJ Foster; citation_author=V Lundblad; citation_author=S Hanley-Way; citation_author=SM Halling; citation_author=N Kleckner; citation_journal_title=Cell; citation_volume=23; citation_number=32; citation_pages=215-227; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=The inverted repeat as a recognizable structural feature in supercoiled DNA molecules.; citation_author=DM Lilley; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=77; citation_number=33; citation_pages=6468-6472; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Cruciform structures in supercoiled DNA.; citation_author=N Panayotatos; citation_author=RD Wells; citation_journal_title=Nature; citation_volume=289; citation_number=34; citation_pages=466-470; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Altered DNA conformations detected by mung bean nuclease occur in promoter and terminator regions of supercoiled pBR322 DNA.; citation_author=LG Sheflin; citation_author=D Kowalski; citation_journal_title=Nucleic Acids Res; citation_volume=13; citation_number=35; citation_pages=6137-6154; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Characterization of the binding specificity of two anticruciform DNA monoclonal antibodies.; citation_author=L Frappier; citation_author=GB Price; citation_author=RG Martin; citation_author=M Zannis-Hadjopoulos; citation_journal_title=J Biol Chem; citation_volume=264; citation_number=36; citation_pages=334-341; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=An intramolecular recombination mechanism for the formation of the rRNA gene palindrome of Tetrahymena thermophila.; citation_author=DK Butler; citation_author=LE Yasuda; citation_author=MC Yao; citation_journal_title=Mol Cell Biol; citation_volume=15; citation_number=37; citation_pages=7117-7126; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Short inverted repeats at a free end signal large palindromic DNA formation in Tetrahymena.; citation_author=LF Yasuda; citation_author=MC Yao; citation_journal_title=Cell; citation_volume=67; citation_number=38; citation_pages=505-516; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Induction of large DNA palindrome formation in yeast: implications for gene amplification and genome stability in eukaryotes.; citation_author=DK Butler; citation_author=LE Yasuda; citation_author=MC Yao; citation_journal_title=Cell; citation_volume=87; citation_number=39; citation_pages=1115-1122; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of sod2 gene amplification in Schizosaccharomyces pombe.; citation_author=EB Albrecht; citation_author=AB Hunyady; citation_author=GR Stark; citation_author=TE Patterson; citation_journal_title=Mol Biol Cell; citation_volume=11; citation_number=40; citation_pages=873-886; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Long palindromes formed in Streptomyces by nonrecombinational intra-strand annealing.; citation_author=Z Qin; citation_author=SN Cohen; citation_journal_title=Genes Dev; citation_volume=14; citation_number=41; citation_pages=1789-1796; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The Mre11 complex is required for repair of hairpin-capped double-strand breaks and prevention of chromosome rearrangements.; citation_author=KS Lobachev; citation_author=DA Gordenin; citation_author=MA Resnick; citation_journal_title=Cell; citation_volume=108; citation_number=42; citation_pages=183-193; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Formation of large palindromic DNA by homologous recombination of short inverted repeat sequences in Saccharomyces cerevisiae.; citation_author=DK Butler; citation_author=D Gillespie; citation_author=B Steele; citation_journal_title=Genetics; citation_volume=161; citation_number=43; citation_pages=1065-1075; citation_date=2002; " />
      <meta name="citation_reference" content="citation_author=M Gray; citation_author=D Spencer; citation_number=44; citation_pages=107-126; citation_date=1996; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of Rickettsia prowazekii and the origin of mitochondria.; citation_author=SG Andersson; citation_author=A Zomorodipour; citation_author=JO Andersson; citation_author=T Sicheritz-Ponten; citation_author=UC Alsmark; citation_journal_title=Nature; citation_volume=396; citation_number=45; citation_pages=133-140; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Evolution of organellar genomes.; citation_author=MW Gray; citation_journal_title=Curr Opin Genet Dev; citation_volume=9; citation_number=46; citation_pages=678-687; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The mitochondrial genome of the honeybee Apis mellifera: complete sequence and genome organization.; citation_author=RH Crozier; citation_author=YC Crozier; citation_journal_title=Genetics; citation_volume=133; citation_number=47; citation_pages=97-117; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Sequence and organization of the mitochondrial genome of the Chagas disease vector, Triatoma dimidiata.; citation_author=EM Dotson; citation_author=CB Beard; citation_journal_title=Insect Mol Biol; citation_volume=10; citation_number=48; citation_pages=205-215; citation_date=2001; " />
      <meta name="citation_reference" content="citation_author=W Brown; citation_number=49; citation_date=1985; citation_publisher=Plenum Press; " />
      <meta name="citation_reference" content="citation_title=Drosophila mitochondrial DNA: conserved sequences in the A+T-rich region and supporting evidence for a secondary structure model of the small ribosomal RNA.; citation_author=DO Clary; citation_author=DR Wolstenholme; citation_journal_title=J Mol Evol; citation_volume=25; citation_number=50; citation_pages=116-125; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Evolution of mitochondrial gene content: gene loss and transfer to the nucleus.; citation_author=KL Adams; citation_author=JD Palmer; citation_journal_title=Mol Phylogenet Evol; citation_volume=29; citation_number=51; citation_pages=380-395; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Sequence and gene organization of the chicken mitochondrial genome. A novel gene order in higher vertebrates.; citation_author=P Desjardins; citation_author=R Morais; citation_journal_title=J Mol Biol; citation_volume=212; citation_number=52; citation_pages=599-634; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=The main regulatory region of mammalian mitochondrial DNA: structure-function model and evolutionary pattern.; citation_author=C Saccone; citation_author=G Pesole; citation_author=E Sbisa; citation_journal_title=J Mol Evol; citation_volume=33; citation_number=53; citation_pages=83-91; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Mechanism of replication of human mitochondrial DNA. Localization of the 5′ ends of nascent daughter strands.; citation_author=DP Tapper; citation_author=DA Clayton; citation_journal_title=J Biol Chem; citation_volume=256; citation_number=54; citation_pages=5109-5115; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Priming of human mitochondrial DNA replication occurs at the light-strand promoter.; citation_author=DD Chang; citation_author=DA Clayton; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=82; citation_number=55; citation_pages=351-355; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Both the conserved stem-loop and divergent 5′-flanking sequences are required for initiation at the human mitochondrial origin of light-strand DNA replication.; citation_author=JE Hixson; citation_author=TW Wong; citation_author=DA Clayton; citation_journal_title=J Biol Chem; citation_volume=261; citation_number=56; citation_pages=2384-2390; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Drosophila melanogaster mitochondrial DNA: gene organization and evolutionary considerations.; citation_author=R Garesse; citation_journal_title=Genetics; citation_volume=118; citation_number=57; citation_pages=649-663; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Origin and direction of replication in mitochondrial DNA molecules from Drosophila melanogaster.; citation_author=JM Goddard; citation_author=DR Wolstenholme; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=75; citation_number=58; citation_pages=3886-3890; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Origin and direction of replication in mitochondrial DNA molecules from the genus Drosophila.; citation_author=JM Goddard; citation_author=DR Wolstenholme; citation_journal_title=Nucleic Acids Res; citation_volume=8; citation_number=59; citation_pages=741-757; citation_date=1980; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000110" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Unusually Long Palindromes Are Abundant in Mitochondrial Control Regions of Insects and Nematodes"/>
    <meta name="twitter:description" content="BackgroundPalindromes are known to be involved in a variety of biological processes. In the present investigation we carried out a comprehensive analysis of palindromes in the mitochondrial control regions (CRs) of several animal groups to study their frequency, distribution and architecture to gain insights into the origin of replication of mtDNA.Methodology/Principal FindingsMany species of Arthropoda, Nematoda, Mollusca and Annelida harbor palindromes and inverted repeats (IRs) in their CRs. Lower animals like cnidarians and higher animal groups like chordates are almost devoid of palindromes and IRs. The study revealed that palindrome occurrence is positively correlated with the AT content of CRs, and that IRs are likely to give rise to longer palindromes.Conclusions/SignificanceThe present study attempts to explain possible reasons and gives in silico evidence for absence of palindromes and IRs from CR of vertebrate mtDNA and acquisition and retention of the same in insects. Study of CRs of different animal phyla uncovered unique architecture of this locus, be it high abundance of long palindromes and IRs in CRs of Insecta and Nematoda, or short IRs of 10&ndash;20 nucleotides with a spacer region of 12&ndash;14 bases in subphylum Chelicerata, or nearly complete of absence of any long palindromes and IRs in Vertebrata, Cnidaria and Echinodermata."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000110.g007"/>

  <meta property="og:title" content="Unusually Long Palindromes Are Abundant in Mitochondrial Control Regions of Insects and Nematodes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000110" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9179'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1132'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7460&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000110">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000110" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000110&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Unusually Long Palindromes Are Abundant in Mitochondrial Control Regions of Insects and Nematodes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            K. P. Arunkumar, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Molecular Genetics, Centre for DNA Fingerprinting and Diagnostics, Nacharam, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Javaregowda Nagaraju
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jnagaraju@cdfd.org.in">jnagaraju@cdfd.org.in</a></p>

                <p>Affiliation:
                  Laboratory of Molecular Genetics, Centre for DNA Fingerprinting and Diagnostics, Nacharam, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 20, 2006</li>
    <li>DOI: 10.1371/journal.pone.0000110</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000110" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000110" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000110" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000110" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000110-g001" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000110-g002" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000110-g003" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000110-g004" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000110-g005" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000110-g006" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000110-g007" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000110">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000110" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Palindromes are known to be involved in a variety of biological processes. In the present investigation we carried out a comprehensive analysis of palindromes in the mitochondrial control regions (CRs) of several animal groups to study their frequency, distribution and architecture to gain insights into the origin of replication of mtDNA.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Many species of Arthropoda, Nematoda, Mollusca and Annelida harbor palindromes and inverted repeats (IRs) in their CRs. Lower animals like cnidarians and higher animal groups like chordates are almost devoid of palindromes and IRs. The study revealed that palindrome occurrence is positively correlated with the AT content of CRs, and that IRs are likely to give rise to longer palindromes.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The present study attempts to explain possible reasons and gives in silico evidence for absence of palindromes and IRs from CR of vertebrate mtDNA and acquisition and retention of the same in insects. Study of CRs of different animal phyla uncovered unique architecture of this locus, be it high abundance of long palindromes and IRs in CRs of Insecta and Nematoda, or short IRs of 10–20 nucleotides with a spacer region of 12–14 bases in subphylum Chelicerata, or nearly complete of absence of any long palindromes and IRs in Vertebrata, Cnidaria and Echinodermata.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Arunkumar KP, Nagaraju J (2006) Unusually Long Palindromes Are Abundant in Mitochondrial Control Regions of Insects and Nematodes. PLoS ONE 1(1):
          e110.
            doi:10.1371/journal.pone.0000110</p><p><strong>Academic Editor: </strong>Laszlo Orban, Temasek Life Sciences Laboratory, Singapore</p><p><strong>Received:</strong> September 27, 2006; <strong>Accepted:</strong> November 26, 2006; <strong>Published:</strong> December 20, 2006</p><p><strong>Copyright:</strong> © 2006 Arunkumar, Nagaraju. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Financial support for this project was from Department of Biotechnology, Government of India. KPA is recipient of the fellowship from Council of Scientific and Industrial Research (CSIR), India.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A DNA palindrome is a unique case of inverted repeats (IRs) <a href="#pone.0000110-Berg1">[1]</a> where a segment of nucleotides is immediately followed by its reverse complement. Palindromes are involved in a variety of biological processes, for example acting as recognition sites for bacterial restriction enzymes to cut foreign DNA <a href="#pone.0000110-Waterman1">[2]</a>. They also play important role in DNA replication and gene regulation <a href="#pone.0000110-Wagner1">[3]</a>, <a href="#pone.0000110-Kornberg1">[4]</a>. IRs flanking the origin of DNA replication with the potential of forming single-stranded stem-loop cruciform structures have been reported to be essential for replication of the circular genomes of many prokaryotic and eukaryotic systems <a href="#pone.0000110-Cheung1">[5]</a>. Several studies have reported the existence of high concentrations of palindromes in proximity to the replication origins of viruses <a href="#pone.0000110-Weller1">[6]</a>–<a href="#pone.0000110-Masse1">[8]</a>. The local two-fold symmetry created by the palindrome is thought to provide binding site for DNA-binding proteins that are often dimeric. Such double binding markedly increases the strength and specificity of the interaction <a href="#pone.0000110-Creighton1">[9]</a>. These regions have been associated with replication origins of a few herpesviruses, bacterial plasmids, etc. In an earlier study <a href="#pone.0000110-Masse1">[8]</a> it was demonstrated that by looking for palindrome clusters, along with other features such as clusters of close repeats and close inversions on the nucleotide sequence, it is possible to fish out regions from a genome that are likely to harbor replication origins. Also, perfect palindromes, quasi-palindromes and IRs separated by spacers, all have the potential to form secondary structures and are known to cause genetic instability in <em>Escherichia coli</em> <a href="#pone.0000110-Leach1">[10]</a>, yeast <a href="#pone.0000110-Gordenin1">[11]</a>, and in mouse <a href="#pone.0000110-Collick1">[12]</a>, <a href="#pone.0000110-Akgun1">[13]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Metazoan mitochondrial DNA (mtDNA) is a closed-circular, double-stranded molecule, ranging in size from 15 to 20 kb <a href="#pone.0000110-Wolstenholme1">[14]</a>. It contains a distinct replication origin on each of the DNA strands. Initiation of mtDNA replication is controlled by the interaction between nuclear-encoded proteins and regulatory sequences existing on the mtDNA <a href="#pone.0000110-Shadel1">[15]</a>–<a href="#pone.0000110-Clayton1">[17]</a>. The non-coding region of the mitochondrial genome in animals called the “control region” (CR) is believed to control the transcription and replication of mtDNA. In vertebrates the CR has been shown to contain promoters for transcription initiation and the origin of heavy-strand DNA replication <a href="#pone.0000110-Shadel1">[15]</a>. In insects this region is usually called “A-T rich region” <a href="#pone.0000110-Fauron1">[18]</a>. Information on palindromes and IRs in mitochondrial CRs and their role in origin of replication is scanty and needs further investigation. In the present study we have attempted to dissect the architecture of the origin of replication of mtDNA by analyzing CRs of several animal phyla and also carried out comprehensive analysis to study the frequency and distribution of palindromes and IRs of eight animal groups.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Source of CR sequences</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>From NCBI, sequences of mitochondrial CRs were queried in 12 most studied animal phyla (Porifera, Cnidaria, Platyhelminthes, Nemertina, Rotifera, Nematoda, Brachiopoda, Mollusca, Annelida, Arthropoda, Echinodermata and Chordata) and were downloaded from different phyla/classes/orders separately wherever necessary, by carrying out a boolean search using combination of different keywords. For example, to download the CR sequences of lepidopteran species, we used keywords like lepidoptera AND control region or lepidoptera AND D loop. Each sequence description was manually checked to ensure that we downloaded CR sequences only.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To study the abundance of palindromes and IRs in subphyla, classes and orders of the phyla Arthropoda and Chordata, CR sequences were downloaded separately from three subphyla Chelicerata, Crustacea and Uniramia of phylum Arthropoda. Subphylum Uniramia was further divided into Chilopoda and Insecta. Many CR sequences have been reported in class Insecta. Therefore, we further binned Insecta CRs based on the taxonomic order of origin. From phylum Chordata we extracted CRs from two important subphyla, Cephalochordata and Vertebrata. Details of the sample size are given in <a href="#pone-0000110-g001">figure 1</a>.</p>
<div class="figure" id="pone-0000110-g001"><div class="img"><a name="pone-0000110-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Number of species analysed for the presence of palindromes and inverted repeats in CRs. In a few phyla, significant number of CRs was not available. <strong><sup>a</sup></strong>These sequences were not used for analysis due to small sample size. <strong><sup>b</sup></strong>CRs from all the orders clubbed together and treated as one animal group ‘Insecta’. Phyla in which &lt;10% of the species harboring palindromes in their CRs are represented in light gray background and &gt;10% in dark gray background. The phylogenetic tree is only a schematic representation and is not according to distance</p>
<span>doi:10.1371/journal.pone.0000110.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Finally, based on number of sequences available in each phylum we divided the data into eight animal groups namely Cnidaria, Nematoda, Mollusca, Echinodermata, Chelicerata, Crustacea, Insecta and Vertebrata (<a href="#pone-0000110-g001">Figure 1</a>; see <a href="#pone.0000110.s001">Supplementary File S1</a> for complete list of species names and other details).</p>


<h4>Mining palindromes in control regions</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A novel strategy to identify palindromes and IRs of different lengths was devised (<a href="#pone-0000110-g002">Figure 2</a>). For this purpose we adopted ‘bl2seq’ (align 2 sequences) program available in standalone BLAST package of NCBI <a href="#pone.0000110-Tatusova1">[19]</a>. A perl script was written, which takes the sequence as query and reverse complement of the same sequence as the subject, and searches for the stretch of similar sequence between them using ‘bl2seq’ program. Pair-wise alignments with more than 70% match were printed to the file. Default parameters of ‘bl2seq’ were used to carry out BLAST. If there is a spacer region of &gt;13 bases between the inverted repeat regions, then that sequence was considered as an IR. This program was executed using multi FASTA files containing CR sequences as input and output files were manually parsed to extract the desired information. The output file was manually verified for confirming the selection of palindromes and tabulated in Microsoft excel data sheets. The program was found to be efficient in identifying palindromes and IRs of longer lengths, allowing a certain level of mismatch. The perl script used for the analyses can be downloaded from <a href="http://www.cdfd.org.in/lmgpgms.html">www.cdfd.org.in/lmgpgms.html</a>. We did not use ‘Palindrome’ program developed by EMBOSS, since the output did not give the length of sequence and was difficult to make out whether the output alignment is a palindrome or IR sequence. Also, there was no option for setting minimum percent similarity allowed.</p>
<div class="figure" id="pone-0000110-g002"><div class="img"><a name="pone-0000110-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Schema of palindrome and inverted repeats mining, adapted in this study. Example of a typical palindrome and inverted repeat unit is given at the bottom. Bars with arrows represent CR sequences. In alignments where subject and query are same but reverse complements, were considered as palindromes. In alignments where subject and query are separated by a spacer region of &gt;13 bases, were considered as IRs.</p>
<span>doi:10.1371/journal.pone.0000110.g002</span></div>

<h4>Analysis of palindromes and inverted repeats</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The perl program gave output of BLAST alignments. From these alignments information on presence, number and length of palindromes and IRs was sorted (<a href="#pone-0000110-g002">Figure 2</a>). Sequences of ≥20 bases were considered as palindromes in order to avoid restriction enzyme recognition sites. The data were tabulated to calculate the frequency of palindromes and IRs in different phyla, classes and orders. Only eight animal groups (<a href="#pone-0000110-g001">Figure 1</a>) were used for the analysis of palindromes and IRs, since the number of CR sequences was not adequate for the remaining animal groups. Since only 5% of the vertebrates possessed palindromes and IRs of ≥20 bases, we also analyzed palindromes and IRs of 10–20 bases for length variation within closely related organisms to get clues for their departure from subphylum Vertebrata. We also looked into subphylum Chelicerata where small IRs of 12–14 bases were present in more than half of the CRs studied.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>To find out the correlation of AT richness with the frequency of occurrence of palindromes in CRs we estimated AT content of all CRs using a C program written in-house. The C program can be downloaded from <a href="http://www.cdfd.org.in/lmgpgms.html">www.cdfd.org.in/lmgpgms.html</a>. AT content of all the reported complete mitochondrial sequences of different animal phyla was also calculated to draw correlation if any, between occurrence of palindromes in CRs and AT content of complete mitochondrial sequences. Further, statistical analyses were carried out to estimate the abundance of palindromes and IRs in different animal groups and to establish relationship between AT content and palindrome occurrence in CRs. All these analyses were carried out in Microsoft excel data sheets. AT content of all the CR sequences analyzed can be found in <a href="#pone.0000110.s001">Supplementary File S1</a>. Unpaired t-test was carried out on the AT content of all 8 animal groups, to find out whether the AT content values are statistically different between animal groups. In the present study more emphasis is given to insect and vertebrate species as more CR sequences are reported in these animal groups, which led us to address several basic questions like why insect CRs are rich in palindromes and IRs.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Evidence accrued so far suggests that mitochondria, once existed as free-living bacteria, were taken up by primitive ancestors of eukaryotic cells in an arrangement termed ‘endosymbiosis’ <a href="#pone.0000110-Gray1">[20]</a>. Till recently it was believed that replication mechanism in mtDNA is conserved and resembles that of plasmid replication. But recent reports suggest that replication mechanism of mtDNA varies among different animal phyla <a href="#pone.0000110-Saito1">[21]</a>, <a href="#pone.0000110-Reyes1">[22]</a>. Metazoan mtDNA codes for 13 or 14 proteins involved in the electron transport chain, 2 rRNA subunits, and 22 tRNA molecules. It contains a distinct replication origin on each of the DNA strands. Mammalian and insect mtDNAs maintain two separate and distinct origins of replication for unidirectional synthesis of each strand of the genome; however, location of these origins is not same in these two animal groups.</p>

<h4>Palindrome occurrence in different animal phyla</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>A total of 393 CR sequences of eight animal groups (<a href="#pone-0000110-g001">Figure 1</a>) was analysed for the distribution and frequency of palindromes and IRs. The analysis revealed interesting features. Palindromes were preponderant in invertebrates than vertebrates. Among invertebrates, more than half of the species in Insecta (85%) and Nematoda (50%) were found to possess palindromes, whereas 35, 20 and 15 percent of the species of Mollusca, Chelicerata and Crustacea respectively, had palindromes. Echinoderms and cnidarians were completely devoid of palindromes and IRs. On the other hand, only 5 percent of vertebrates contained palindromes and mammals were characterized by their complete absence (<a href="#pone-0000110-g003">Figure 3b</a>).</p>
<div class="figure" id="pone-0000110-g003"><div class="img"><a name="pone-0000110-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>(a) Mean AT content of CR (%) of mitochondrial CRs (n = 393) and (b) % of mitochondrial CRs having palindromes in different animal groups (n = 387). (c) Mean AT content of CR (%) of complete mitochondrial sequences of different animal groups (n = 201). Data was obtained from analysis of 8 animal groups.</p>
<span>doi:10.1371/journal.pone.0000110.g003</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Comparative genomics studies suggested that the mitochondrion is monophyletic in origin <a href="#pone.0000110-Gray1">[20]</a> and the original mitochondrial endosymbiont has evolved independently in anaerobic and aerobic environments that are inhabited by diverse eukaryotic lineages. The evolution of various kinds of DNA motifs in CRs such as palindromes, IRs and other recognition elements in different animal phyla, appeared to have occurred independently after the divergence of different animal phyla leading to variation in number of palindromes and IRs.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>In vertebrate mitochondrial CRs analysed in the present study, palindromes were found in a few avians (<em>Gallus</em> sps. and <em>Syrmaticus ellioti</em>), two fish species (<em>Apeltes quadracus</em>, <em>Cyprinodon bovines</em>) and a reptile (<em>Teratoscincus keyserlingii</em>). IRs were found only in one species, <em>T. keyserlingii</em> out of 125 vertebrates analysed.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>Subphylum Chelicerata is unique in its composition of CR. Nineteen out of 36 CRs analysed in this subphylum harbored short IRs of 10–20 nucleotides with a spacer region of 12–14 nucleotides. Except for <em>Acropora longicyanthus</em> with a short palindrome of 14 nucleotides, phylum Cnidaria was completely devoid of palindromes and IRs.</p>


<h4>Palindromes in mitochondrial CRs of arthropods</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Among invertebrates, in class Insecta higher abundance of palindromes and IRs was observed. Two important orders Lepidoptera and Diptera were analysed further for frequency and distribution of palindromes and IRs. Mitochondrial CR of lepidopteran insects turned out to be ‘hotspots’ of palindromes and IRs. All but two (<em>Erebia oeme</em> and <em>Pyronia tithonus</em>), of 53 species analysed in this order, harbored palindromes. When compared to order Diptera, lepidopterans possessed higher number of palindromes in CRs. Out of 50 dipterans, 38 had palindromes (<a href="#pone-0000110-g004">Figure 4a &amp; b</a>). We compared the number of palindromes per CR of dipteran and lepidopteran species. In dipterans CRs harboring only one palindrome were more as compared to lepidopterans which harbored more than one palindrome per CR (<a href="#pone-0000110-g004">Figure 4a &amp; b</a>). To study the length distribution of palindromes in lepidopteran and dipteran mitochondrial CRs, the length of palindromes was analyzed. The CRs, which had ≥1 palindrome(s) were considered for the analysis. The results revealed that lepidopteran CRs had longer palindromes than those of dipterans (<a href="#pone-0000110-g004">Figure 4c</a>).</p>
<div class="figure" id="pone-0000110-g004"><div class="img"><a name="pone-0000110-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>Distribution of palindromes in CRs of (a) Lepidopterans (n = 53) and (b) Dipterans (n = 50). Number of CRs is plotted against the number of palindromic DNA stretches they contain. Many CRs were having ≥2 stretches of palindromes in Lepidopterans. Dipteran CRs possess less number of palindromes when compared to Lepidopterans. (C) Distribution of palindrome lengths in dipteran (n = 82) and lepidopteran (n = 150) CRs. Palindromes falling to each class interval (20–30 to ≥100 bp) were grouped. Number of palindromes in each class was converted to percent values to compare the length distribution between Diptera and Lepidoptera. These percent values were plotted in graph against palindrome length class intervals. In Diptera, 38 of 50 CRs had palindromes whereas, 51 of 53 CRs analysed of Lepidoptera had palindromes.</p>
<span>doi:10.1371/journal.pone.0000110.g004</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Although palindromes exist naturally in the DNA sequence of many organisms, it is difficult to maintain long palindromes because of their genetic instability stability as demonstrated in <em>E. coli</em> <a href="#pone.0000110-Shafferman1">[23]</a>. The instability of palindromes is attributed to a number of causes. First, palindromes may be deleted as a result of intermolecular or intramolecular recombination. Second, the deletion of palindromes may result from the formation of a cruciform structure and the subsequent processing by nucleases. Third, in the case of palindromes containing direct repeats, misalignment between the direct repeats may be stabilized by the formation of a hairpin or other DNA secondary structures <a href="#pone.0000110-Sinden1">[24]</a>. In the present study we observed surprisingly long palindromes greater than 150 bp in three (<em>Epirrita christyi</em>, <em>Arethusana arethusa</em> and <em>Epirrita autumnata</em>) of the 53 lepidopterans studied. Even though palindromes are unstable due to several reasons listed above, they are retained in CR suggesting their possible involvement in replication initiation of lepidopteran mitochondrial genome. However, in dipterans we observed shorter palindromes of only up to 56 bp. None of the CR sequences analysed in the remaining phyla possessed palindromes greater than 150 bp. However, molecular function of these long palindromes in insect mtDNA needs further analysis.</p>


<h4>Palindromes occur in AT rich regions of CRs</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Since insect CRs are AT-rich, it prompted us to establish correlation if any between AT content and palindrome occurrence. All CRs were pooled irrespective of the phyla to which the species belonged and analysed for presence or absence and number of palindromes in relation to AT content. Positive correlation between AT content and number of palindromes with correlation co-efficient of 0.89 was observed. Through these analyses we propose that palindromes originate in CRs with more than 85% AT content (<a href="#pone-0000110-g005">Figure 5</a>).</p>
<div class="figure" id="pone-0000110-g005"><div class="img"><a name="pone-0000110-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>Distribution of number of palindromes per CR, plotted against AT content of CR (%). Majority of CRs with less than 85 percent AT content had no palindromes but most of the CRs with AT percent &gt;85 possessed palindromes. Trend is that as the AT content increased, occurrence of palindromes in CRs also increased. Number of CRs analysed was 393.</p>
<span>doi:10.1371/journal.pone.0000110.g005</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>All animal groups except Mollusca and Echinodermata differed significantly (p&lt;0.05) in their CR AT content (<a href="#pone.0000110.s002">Supplementary File S2</a>). Cnidaria, Echinodermata and Vertebrata, which harbored no or less number of palindromes and IRs in their CRs, have low AT content (an average of 58, 64, 61% respectively). Even though Chelicerata and Crustacea belong to phylum Arthropoda, to which insects belong, there is a marked difference in their AT content of CRs. Also, abundance of palindromes varies in these animal groups. Insects have high AT content and high occurrence of palindromes in the CRs, whereas Chelicerata and Crustacea have less AT and palindromes in their CRs (<a href="#pone-0000110-g003">Figure 3a</a>).</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Analysis of complete mitochondrial genomes of Insecta and Nematoda also indicated the AT richness in their genomes (<a href="#pone-0000110-g003">Figure 3c</a>). Nematoda, Chelicerata and Insecta have an average AT content of more than 70%. Though Chelicerata mitochondrial genome is rich in AT content, only a few of the species harbored palindromes and IRs in the CR, inferring that AT content of complete mitochondrial genome has no relationship with the palindrome occurrence in CRs.</p>


<h4>Inverted repeats would give rise to long palindromes</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The present study revealed surprisingly long palindromes in mitochondrial origin of replication. It is known that palindromes and IRs reside near the origin of replication in several bacteria, plasmids and viruses <a href="#pone.0000110-Pearson1">[25]</a>. But these palindromes and IRs are invariably of shorter length (10–12 bp) unlike the palindromes reported in the present study.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Long palindromic sequences are unstable since they are deleted at extremely high rates as reported in the case of <em>E. coli</em> <a href="#pone.0000110-Collins1">[26]</a>, <a href="#pone.0000110-Collins2">[27]</a>. While short palindromes and IRs are usually much more stable; they can be associated with the breakpoints of deletion mutations as observed in <em>E. coli</em> <a href="#pone.0000110-Albertini1">[28]</a>, <a href="#pone.0000110-Glickman1">[29]</a> and in mammalian cells <a href="#pone.0000110-Nalbantoglu1">[30]</a>. Distantly separated long IRs are also prone to deletion in bacteria <a href="#pone.0000110-Egner1">[31]</a>, <a href="#pone.0000110-Foster1">[32]</a>. In the present study palindromes were preponderant (in 120 CRs) over IRs (in 78 CRs). This shows that in CRs, palindromes are less prone to deletion unlike in <em>E. coli</em>. The presence of a fewer IRs led us to suspect that intervening regions between IRs would be deleted over a period of time resulting in formation of long palindromes. This can be explained in the following ways: i. when the DNA melts either during DNA replication or due to any other cellular activity, there would be intra-strand base-pairing in the regions of IRs leading to formation of hair-pin loop. The unpaired spacer regions between IRs may be eventually cleaved by DNases, as the tips of hairpin loops are sensitive to single-stranded nucleases <a href="#pone.0000110-Lilley1">[33]</a>–<a href="#pone.0000110-Butler1">[37]</a> resulting in long palindromes (<a href="#pone-0000110-g006">Figure 6</a>). Higher abundance of palindromes as compared to IRs also supports this hypothesis. ii. Alternatively, occurrence of a DNA double-strand break near short inverted repeat sequences acts as a starting point in the formation of large DNA palindromes <a href="#pone.0000110-Yasuda1">[38]</a>–<a href="#pone.0000110-Butler3">[43]</a>. Bidirectional DNA replication would then convert the giant hairpin molecule to a palindrome.</p>
<div class="figure" id="pone-0000110-g006"><div class="img"><a name="pone-0000110-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6. </strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>Schema showing possible mechanism of origin of long palindromes from IRs.</p>
<span>doi:10.1371/journal.pone.0000110.g006</span></div><a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>However, we cannot exclude other factors, which are thought to lead to formation of long palindromes in CRs of insects, as in some cases they form as a result of a precisely regulated developmental program. For example, previous reports have shown the formation of long palindromes from short IRs by single strand annealing of IRs followed by removal of nonhomologous DNA and gap-filling DNA replication <a href="#pone.0000110-Butler1">[37]</a>, <a href="#pone.0000110-Butler2">[39]</a>, <a href="#pone.0000110-Butler3">[43]</a>.</p>


<h4>Palindrome expansion and palindrome curing</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Palindromes of 4–6 bp are implicated in bacterial chromosome and plasmid replication. Since mitochondria are thought to have originated from bacteria, the origin of replication also has a propensity to harbor palindromes. But palindromes present in CRs of insects are much longer than those present in bacterial origin of replication. Phylogenetic evidence derived from rRNA <a href="#pone.0000110-Gray2">[44]</a> and protein data <a href="#pone.0000110-Andersson1">[45]</a> support the view that all mitochondrial genomes are descendents of a common proto-mitochondrial ancestor.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>Due to varied evolutionary forces acting on different species, lower animal groups like Cnidaria and higher animal groups like echinodermates may be completely devoid of palindromes and, longer palindromes would have originated or expanded from already present smaller palindromes or IRs in some species of insects and nematodes, depending on the bio-physiochemical environment within the cell. Although the genetic role of mtDNA appears to be universally conserved, this genome exhibits remarkable variation in conformation and size, as well as in actual gene content, arrangement and expression <a href="#pone.0000110-Gray3">[46]</a> including palindrome occurrence in CRs.</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>The surprising absence of long palindromes and IRs in CR of vertebrate mtDNA implies that during the course of evolution, higher animals like chordates probably have adopted a slightly different mechanism of replication of mtDNA which does not require palindromes as a recognition motif in replication initiation. The variation in distribution of palindromes and IRs in different animal groups suggests that the mode of replication origin is quite different between different animal phyla. This is supported by the previous studies on CR of vertebrates and insects <a href="#pone.0000110-Saito1">[21]</a>, <a href="#pone.0000110-Reyes1">[22]</a>.</p>
<a id="article1.body1.sec3.sec5.p4" name="article1.body1.sec3.sec5.p4"></a><p>The AT bias is generally observed in insect mitochondrial genomes, which ranges from 69.5 to 84.9% <a href="#pone.0000110-Crozier1">[47]</a>, <a href="#pone.0000110-Dotson1">[48]</a> as against 53 to 66% in vertebrates. The strongest AT bias is found in the CR. The CR contains the origin of replication for the heavy strand in vertebrates <a href="#pone.0000110-Brown1">[49]</a> and both strands in Drosophila <a href="#pone.0000110-Clary1">[50]</a>. Mitochondrial gene order variation occurs both between and within animal phyla <a href="#pone.0000110-Adams1">[51]</a>. In insects, the CRs, which account for 80–95% AT, lack any apparent signals such as conserved sequence blocks, for the initiation of replication like those observed in vertebrates <a href="#pone.0000110-Desjardins1">[52]</a>, <a href="#pone.0000110-Saccone1">[53]</a>. These differences in replication initiation mechanism of vertebrates and insects probably hold the answer as to why CRs of some phyla are rich in palindromes and IRs.</p>
<a id="article1.body1.sec3.sec5.p5" name="article1.body1.sec3.sec5.p5"></a><p>The vertebrates were also found to be deficient in short palindromes. Of the 125 species examined, only 50 harbored short palindromes. Multiple sequence alignment of these short palindromic sequences showed contraction of a few palindromes, indicating that during course of evolution palindromes are lost gradually by reduction in size. Contraction of palindromes would be due to mutations at both ends of palindromes as evident from <em>Gallus</em> sps (<a href="#pone.0000110.s003">Supplementary File S3</a>).</p>


<h4>Why long palindromes in CRs of Insecta only?</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Metazoan mtDNA contains a distinct replication origin on each of the DNA strands. The position of the replication origin and mode of replication have been studied in detail in mammalian mtDNA and also recently in insects <a href="#pone.0000110-Saito1">[21]</a>. In mammals, the replication begins from the replication origin of H-strand (O<sub>H</sub>) and DNA synthesis proceeds unidirectionally. When the synthesis of H-strand reaches two-thirds of the genome, the synthesis of L-strand (O<sub>L</sub>) is initiated from the replication origin of the L-strand located two-thirds of the genomic distance away from the replication origin of H-strand <a href="#pone.0000110-Shadel1">[15]</a>, <a href="#pone.0000110-Taanman1">[16]</a> (<a href="#pone-0000110-g007">Figure 7</a>). The previous observations indicate that regulatory sequences of mtDNA replication are different in invertebrate and vertebrate species and therefore suggest that the regulatory systems have changed through the evolution of animals <a href="#pone.0000110-Saito1">[21]</a>.</p>
<div class="figure" id="pone-0000110-g007"><div class="img"><a name="pone-0000110-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000110" data-uri="info:doi/10.1371/journal.pone.0000110.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000110.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000110.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000110.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000110.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7. </strong></p><a id="article1.body1.sec3.sec6.fig1.caption1.p1" name="article1.body1.sec3.sec6.fig1.caption1.p1"></a><p>Illustration of origin of replication of vertebrate (heavy strand, O<sub>H</sub>; light strand, O<sub>L</sub>) and insect (minor strand, O<sub>N</sub>; major strand, O<sub>J</sub>) mtDNA. In vertebrates, the synthesis of L-strand is initiated from the O<sub>L</sub>, only when the synthesis of H-strand reaches O<sub>L</sub>, which is located about two third of the distance from O<sub>H</sub>. In insects, the replication origin for major strand (O<sub>J</sub>) is located 97% of the genomic distance away from the replication origin for minor strand (O<sub>N</sub>), that is, within the CR.</p>
<span>doi:10.1371/journal.pone.0000110.g007</span></div><a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>Several regulatory sequences have been identified in the CR of the vertebrate mtDNA. These are present immediately upstream of the O<sub>H</sub> and are suggested to be implicated in the initiation of the H-strand replication <a href="#pone.0000110-Tapper1">[54]</a>, <a href="#pone.0000110-Chang1">[55]</a>. These regulatory sequences are thought to be involved in generating the 3′ ends of the RNA primers, which are required for the DNA synthesis of the H-strand. Around the O<sub>L</sub>, the IR sequence of 10–12 bp, that could form a stem-loop configuration is conserved among vertebrate species and is also proposed to be required for the initiation of replication <a href="#pone.0000110-Tapper1">[54]</a>. <em>In vitro</em> replication studies have suggested that the IR sequence serves as a recognition motif for mtDNA primase which provides a short RNA primer for L-strand synthesis, and DNA synthesis is initiated near the base of the stem-loop structure utilizing the 3′ ends of the RNA primer <a href="#pone.0000110-Hixson1">[56]</a>.</p>
<a id="article1.body1.sec3.sec6.p3" name="article1.body1.sec3.sec6.p3"></a><p>In insects, the leading and lagging strands are termed minor and major coding strands according to the relative numbers of the gene encoded on the respective DNA strand <a href="#pone.0000110-Garesse1">[57]</a>. The replication origin for minor strand (O<sub>N</sub>) is located in the middle portion of CR. Synthesis of minor coding strand proceeds unidirectionally, and the major coding strand synthesis begins after 97% of the minor coding strand synthesis is completed <a href="#pone.0000110-Goddard1">[58]</a>, <a href="#pone.0000110-Goddard2">[59]</a> (<a href="#pone-0000110-g007">Figure 7</a>). If the replication mode is similar between both insect and vertebrate mtDNAs, the replication origin for major strand (O<sub>J</sub>) must be located 97% of the genomic distance away from the replication origin for minor strand, that is, within the CR <a href="#pone.0000110-Saito1">[21]</a>. The fact that CR contains the O<sub>H</sub> in vertebrates <a href="#pone.0000110-Brown1">[49]</a> and both O<sub>N</sub> and O<sub>J</sub> in insects <a href="#pone.0000110-Clary1">[50]</a>, tempts us to speculate that occurrence of palindromes and IRs would aid in replication of insect mtDNA.</p>
<a id="article1.body1.sec3.sec6.p4" name="article1.body1.sec3.sec6.p4"></a><p>According to the strand-asynchronous, asymmetric model of vertebrate mtDNA, the replication of the L-strand is initiated when the synthesis of the H-strand passes beyond the L-strand origin, and template strand for the L-strand replication becomes single stranded. In insects O<sub>J</sub> is located 97% genomic distance away from the O<sub>N </sub>that is, both O<sub>J</sub> and O<sub>N</sub> lie within the CR. This observation is consistent with the recent findings in Drosophila mtDNA that the free 5′ ends in the CR near the tRNAIle gene, exactly 97% genomic distance away from the O<sub>N</sub> <a href="#pone.0000110-Saito1">[21]</a>. To draw similarity between O<sub>J</sub> of insects and O<sub>L</sub> of vertebrates, there should be presence of IRs in O<sub>J</sub> like in O<sub>L</sub> to form stem loop structure for the initiation of replication from O<sub>J</sub>. Indeed in the present study we have found palindromes and IRs in CRs of insects.</p>


<h4>Conclusions</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>This is a comprehensive report on the analysis of palindromes and IRs comprising 393 CRs from seven animal phyla. We are reporting that long palindromes and IRs are abundant in insect mitochondrial origin of replication. Lower animals like cnidarians and higher animal groups like chordates are almost devoid of long palindromes, where as many species of Arthropoda, Nematoda, Mollusca and Annelida harbor palindromes and IRs in their CRs (<a href="#pone-0000110-g001">Figure 1</a>). Here we have given the primary data on the effect of AT richness on palindrome occurrence and plausible reasons for origin of longer palindromes from short inverted repeats. Study of CRs of different animal phyla uncovered unique architecture of this locus, be it high abundance of long palindromes and IRs in CRs of Insecta and Nematoda, or short IRs of 10–20 nucleotides with a spacer region of 12–14 bases in subphylum Chelicerata or nearly complete of absence of any long palindromes and IRs in Vertebrata, Cnidaria and Echinodermata.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000110"></div><a name="pone.0000110.s001" id="pone.0000110.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000110.s001">Supplementary File S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Details of the control region sequences used in the present study</p>
<p class="siDoi">doi:10.1371/journal.pone.0000110.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.10 MB XLS)</p>
<a name="pone.0000110.s002" id="pone.0000110.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000110.s002">Supplementary File S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">P-values obtained by t-test performed on AT percent values of 8 animal groups.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000110.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000110.s003" id="pone.0000110.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000110.s003">Supplementary File S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">CLUSTAL X - Multiple sequence alignment of short palindromes found in vertebrates</p>
<p class="siDoi">doi:10.1371/journal.pone.0000110.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.61 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>Authors thank Ms Archana Tomer and Mr. Eshwar for assistance in the bioinformatics analysis.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JN KA. Performed the experiments: KA. Analyzed the data: KA. Contributed reagents/materials/analysis tools: JN. Wrote the paper: JN KA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000110-Berg1" id="pone.0000110-Berg1"></a>Berg DE, Egner C, Hirschel BJ, Howard J, Johnsrud L, et al.  (1981) Insertion, excision, and inversion of Tn5. Cold Spring Harb Symp Quant Biol  45: 115–123.  <ul class="find" data-citedArticleID="5292805" data-doi="10.1101/sqb.1981.045.01.020"><li><a href="http://dx.doi.org/10.1101/sqb.1981.045.01.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Berg%5Bauthor%5D+AND+Insertion%2C+excision%2C+and+inversion+of+Tn5." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABerg+%22Insertion%2C+excision%2C+and+inversion+of+Tn5.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000110-Waterman1" id="pone.0000110-Waterman1"></a>Waterman M (1995) Introduction to Computational Biology. London: Chapman and Hall.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0000110-Wagner1" id="pone.0000110-Wagner1"></a>Wagner E (1991) Herpesvirus Transcription and its Regulation. Boca Raton: CRC Press.   <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0000110-Kornberg1" id="pone.0000110-Kornberg1"></a>Kornberg A, Baker T (1992) DNA Replication. New York: W. Freeman Co.   <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0000110-Cheung1" id="pone.0000110-Cheung1"></a>Cheung AK (2004) Palindrome regeneration by template strand-switching mechanism at the origin of DNA replication of porcine circovirus via the rolling-circle melting-pot replication model. J Virol  78: 9016–9029.  <ul class="find" data-citedArticleID="5292813" data-doi="10.1128/jvi.78.17.9016-9029.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.17.9016-9029.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cheung%5Bauthor%5D+AND+Palindrome+regeneration+by+template+strand-switching+mechanism+at+the+origin+of+DNA+replication+of+porcine+circovirus+via+the+rolling-circle+melting-pot+replication+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACheung+%22Palindrome+regeneration+by+template+strand-switching+mechanism+at+the+origin+of+DNA+replication+of+porcine+circovirus+via+the+rolling-circle+melting-pot+replication+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000110-Weller1" id="pone.0000110-Weller1"></a>Weller S, Spadaro A, Schaffer J, Murray A, Maxam A, et al.  (1985) Cloning, sequencing, and functional analysis of oriL, a herpes simplex virus type 1 origin of DNA synthesis. Mol Cell Biol  5: 930–942.  <ul class="find" data-citedArticleID="5292815"><li><a href="http://www.crossref.org/guestquery/?auth2=Weller&amp;atitle2=Cloning%2C+sequencing%2C+and+functional+analysis+of+oriL%2C+a+herpes+simplex+virus+type+1+origin+of+DNA+synthesis.&amp;auth=Weller&amp;atitle=Cloning%2C+sequencing%2C+and+functional+analysis+of+oriL%2C+a+herpes+simplex+virus+type+1+origin+of+DNA+synthesis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Weller%5Bauthor%5D+AND+Cloning%2C+sequencing%2C+and+functional+analysis+of+oriL%2C+a+herpes+simplex+virus+type+1+origin+of+DNA+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWeller+%22Cloning%2C+sequencing%2C+and+functional+analysis+of+oriL%2C+a+herpes+simplex+virus+type+1+origin+of+DNA+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000110-Reisman1" id="pone.0000110-Reisman1"></a>Reisman D, Yates J, Sugden B (1985) A putative origin of Replication 29 of plasmids derived from Epstein-Barr virus is composed of two cis-acting components. Mol Cell Biol  5: 1822–1832.  <ul class="find" data-citedArticleID="5292817"><li><a href="http://www.crossref.org/guestquery/?auth2=Reisman&amp;atitle2=A+putative+origin+of+Replication+29+of+plasmids+derived+from+Epstein-Barr+virus+is+composed+of+two+cis-acting+components.&amp;auth=Reisman&amp;atitle=A+putative+origin+of+Replication+29+of+plasmids+derived+from+Epstein-Barr+virus+is+composed+of+two+cis-acting+components." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reisman%5Bauthor%5D+AND+A+putative+origin+of+Replication+29+of+plasmids+derived+from+Epstein-Barr+virus+is+composed+of+two+cis-acting+components." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AReisman+%22A+putative+origin+of+Replication+29+of+plasmids+derived+from+Epstein-Barr+virus+is+composed+of+two+cis-acting+components.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000110-Masse1" id="pone.0000110-Masse1"></a>Masse M, Karlin S, Schachtel G, Mocarski E (1992) Human cytomegalovirus origin of DNA replication (oriLyt) resides within a highly complex repetitive region. Proc Natl Acad Sci USA  89: 5246–5250.  <ul class="find" data-citedArticleID="5292819" data-doi="10.1073/pnas.89.12.5246"><li><a href="http://dx.doi.org/10.1073/pnas.89.12.5246" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Masse%5Bauthor%5D+AND+Human+cytomegalovirus+origin+of+DNA+replication+%28oriLyt%29+resides+within+a+highly+complex+repetitive+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMasse+%22Human+cytomegalovirus+origin+of+DNA+replication+%28oriLyt%29+resides+within+a+highly+complex+repetitive+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000110-Creighton1" id="pone.0000110-Creighton1"></a>Creighton T (1993) Proteins. New York: W. H. Freeman and Co.   <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0000110-Leach1" id="pone.0000110-Leach1"></a>Leach D (1994) Long DNA palindromes, cruciform structures, genetic instability and secondary structure repair. Bioessays  16: 893–900.  <ul class="find" data-citedArticleID="5292823" data-doi="10.1002/bies.950161207"><li><a href="http://dx.doi.org/10.1002/bies.950161207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Leach%5Bauthor%5D+AND+Long+DNA+palindromes%2C+cruciform+structures%2C+genetic+instability+and+secondary+structure+repair." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALeach+%22Long+DNA+palindromes%2C+cruciform+structures%2C+genetic+instability+and+secondary+structure+repair.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000110-Gordenin1" id="pone.0000110-Gordenin1"></a>Gordenin DA, Resnick MA (1998) Yeast ARMs (DNA at-risk motifs) can reveal sources of genome instability. Mutat Res  400: 45–58.  <ul class="find" data-citedArticleID="5292825" data-doi="10.1016/s0027-5107(98)00047-5"><li><a href="http://dx.doi.org/10.1016/s0027-5107(98)00047-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gordenin%5Bauthor%5D+AND+Yeast+ARMs+%28DNA+at-risk+motifs%29+can+reveal+sources+of+genome+instability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGordenin+%22Yeast+ARMs+%28DNA+at-risk+motifs%29+can+reveal+sources+of+genome+instability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000110-Collick1" id="pone.0000110-Collick1"></a>Collick A, Drew J, Penberth J, Bois P, Luckett J, et al.  (1996) Instability of long inverted repeats within mouse transgenes. Embo J  15: 1163–1171.  <ul class="find" data-citedArticleID="5292827"><li><a href="http://www.crossref.org/guestquery/?auth2=Collick&amp;atitle2=Instability+of+long+inverted+repeats+within+mouse+transgenes.&amp;auth=Collick&amp;atitle=Instability+of+long+inverted+repeats+within+mouse+transgenes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Collick%5Bauthor%5D+AND+Instability+of+long+inverted+repeats+within+mouse+transgenes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACollick+%22Instability+of+long+inverted+repeats+within+mouse+transgenes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000110-Akgun1" id="pone.0000110-Akgun1"></a>Akgun E, Zahn J, Baumes S, Brown G, Liang F, et al.  (1997) Palindrome resolution and recombination in the mammalian germ line. Mol Cell Biol  17: 5559–5570.  <ul class="find" data-citedArticleID="5292829"><li><a href="http://www.crossref.org/guestquery/?auth2=Akgun&amp;atitle2=Palindrome+resolution+and+recombination+in+the+mammalian+germ+line.&amp;auth=Akgun&amp;atitle=Palindrome+resolution+and+recombination+in+the+mammalian+germ+line." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Akgun%5Bauthor%5D+AND+Palindrome+resolution+and+recombination+in+the+mammalian+germ+line." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAkgun+%22Palindrome+resolution+and+recombination+in+the+mammalian+germ+line.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000110-Wolstenholme1" id="pone.0000110-Wolstenholme1"></a>Wolstenholme DR (1992) Animal mitochondrial DNA: structure and evolution. Int Rev Cytol  141: 173–216.  <ul class="find" data-citedArticleID="5292831" data-doi="10.1016/s0074-7696(08)62066-5"><li><a href="http://dx.doi.org/10.1016/s0074-7696(08)62066-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wolstenholme%5Bauthor%5D+AND+Animal+mitochondrial+DNA%3A+structure+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWolstenholme+%22Animal+mitochondrial+DNA%3A+structure+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000110-Shadel1" id="pone.0000110-Shadel1"></a>Shadel GS, Clayton DA (1997) Mitochondrial DNA maintenance in vertebrates. Annu Rev Biochem  66: 409–435.  <ul class="find" data-citedArticleID="5292833" data-doi="10.1146/annurev.biochem.66.1.409"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.66.1.409" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shadel%5Bauthor%5D+AND+Mitochondrial+DNA+maintenance+in+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AShadel+%22Mitochondrial+DNA+maintenance+in+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000110-Taanman1" id="pone.0000110-Taanman1"></a>Taanman JW (1999) The mitochondrial genome: structure, transcription, translation and replication. Biochim Biophys Acta  1410: 103–123.  <ul class="find" data-citedArticleID="5292835" data-doi="10.1016/s0005-2728(98)00161-3"><li><a href="http://dx.doi.org/10.1016/s0005-2728(98)00161-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Taanman%5Bauthor%5D+AND+The+mitochondrial+genome%3A+structure%2C+transcription%2C+translation+and+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATaanman+%22The+mitochondrial+genome%3A+structure%2C+transcription%2C+translation+and+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000110-Clayton1" id="pone.0000110-Clayton1"></a>Clayton D (1982) Replication of animal mitochondrial DNA. Cell  28: 693–705.  <ul class="find" data-citedArticleID="5292837" data-doi="10.1016/0092-8674(82)90049-6"><li><a href="http://dx.doi.org/10.1016/0092-8674(82)90049-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clayton%5Bauthor%5D+AND+Replication+of+animal+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AClayton+%22Replication+of+animal+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000110-Fauron1" id="pone.0000110-Fauron1"></a>Fauron CM, Wolstenholme DR (1976) Structural heterogeneity of mitochondrial DNA molecules within the genus Drosophila. Proc Natl Acad Sci U S A  73: 3623–3627.  <ul class="find" data-citedArticleID="5292839" data-doi="10.1073/pnas.73.10.3623"><li><a href="http://dx.doi.org/10.1073/pnas.73.10.3623" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fauron%5Bauthor%5D+AND+Structural+heterogeneity+of+mitochondrial+DNA+molecules+within+the+genus+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFauron+%22Structural+heterogeneity+of+mitochondrial+DNA+molecules+within+the+genus+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000110-Tatusova1" id="pone.0000110-Tatusova1"></a>Tatusova TA, Madden TL (1999) BLAST 2 Sequences, a new tool for comparing protein and nucleotide sequences. FEMS Microbiol Lett  174: 247–250.  <ul class="find" data-citedArticleID="5292841" data-doi="10.1111/j.1574-6968.1999.tb13575.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.1999.tb13575.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tatusova%5Bauthor%5D+AND+BLAST+2+Sequences%2C+a+new+tool+for+comparing+protein+and+nucleotide+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATatusova+%22BLAST+2+Sequences%2C+a+new+tool+for+comparing+protein+and+nucleotide+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000110-Gray1" id="pone.0000110-Gray1"></a>Gray MW, Burger G, Lang BF (1999) Mitochondrial evolution. Science  283: 1476–1481.  <ul class="find" data-citedArticleID="5292843" data-doi="10.1126/science.283.5407.1476"><li><a href="http://dx.doi.org/10.1126/science.283.5407.1476" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gray%5Bauthor%5D+AND+Mitochondrial+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGray+%22Mitochondrial+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000110-Saito1" id="pone.0000110-Saito1"></a>Saito S, Tamura K, Aotsuka T (2005) Replication origin of mitochondrial DNA in insects. Genetics  171: 1695–1705.  <ul class="find" data-citedArticleID="5292845" data-doi="10.1534/genetics.105.046243"><li><a href="http://dx.doi.org/10.1534/genetics.105.046243" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Saito%5Bauthor%5D+AND+Replication+origin+of+mitochondrial+DNA+in+insects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASaito+%22Replication+origin+of+mitochondrial+DNA+in+insects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000110-Reyes1" id="pone.0000110-Reyes1"></a>Reyes A, Yang MY, Bowmaker M, Holt IJ (2005) Bidirectional replication initiates at sites throughout the mitochondrial genome of birds. J Biol Chem  280: 3242–3250.  <ul class="find" data-citedArticleID="5292847" data-doi="10.1074/jbc.m411916200"><li><a href="http://dx.doi.org/10.1074/jbc.m411916200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reyes%5Bauthor%5D+AND+Bidirectional+replication+initiates+at+sites+throughout+the+mitochondrial+genome+of+birds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AReyes+%22Bidirectional+replication+initiates+at+sites+throughout+the+mitochondrial+genome+of+birds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000110-Shafferman1" id="pone.0000110-Shafferman1"></a>Shafferman A, Flashner Y, Hertman I, Olami Y, Cohen S (1987) Molecular aspects of genetic instability of an artificial 68 bp perfect palindrome in <em>Escherichia coli</em>. Mol Gen Genet  208: 294–300.  <ul class="find" data-citedArticleID="5292849" data-doi="10.1007/bf00330456"><li><a href="http://dx.doi.org/10.1007/bf00330456" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shafferman%5Bauthor%5D+AND+Molecular+aspects+of+genetic+instability+of+an+artificial+68+bp+perfect+palindrome+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AShafferman+%22Molecular+aspects+of+genetic+instability+of+an+artificial+68+bp+perfect+palindrome+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000110-Sinden1" id="pone.0000110-Sinden1"></a>Sinden RR, Zheng GX, Brankamp RG, Allen KN (1991) On the deletion of inverted repeated DNA in <em>Escherichia coli</em>: effects of length, thermal stability, and cruciform formation in vivo. Genetics  129: 991–1005.  <ul class="find" data-citedArticleID="5292851"><li><a href="http://www.crossref.org/guestquery/?auth2=Sinden&amp;atitle2=On+the+deletion+of+inverted+repeated+DNA+in+Escherichia+coli%3A+effects+of+length%2C+thermal+stability%2C+and+cruciform+formation+in+vivo.&amp;auth=Sinden&amp;atitle=On+the+deletion+of+inverted+repeated+DNA+in+Escherichia+coli%3A+effects+of+length%2C+thermal+stability%2C+and+cruciform+formation+in+vivo." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sinden%5Bauthor%5D+AND+On+the+deletion+of+inverted+repeated+DNA+in+Escherichia+coli%3A+effects+of+length%2C+thermal+stability%2C+and+cruciform+formation+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASinden+%22On+the+deletion+of+inverted+repeated+DNA+in+Escherichia+coli%3A+effects+of+length%2C+thermal+stability%2C+and+cruciform+formation+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000110-Pearson1" id="pone.0000110-Pearson1"></a>Pearson CE, Zorbas H, Price GB, Zannis-Hadjopoulos M (1996) Inverted repeats, stem-loops, and cruciforms: significance for initiation of DNA replication. J Cell Biochem  63: 1–22.  <ul class="find" data-citedArticleID="5292853" data-doi="10.1002/(sici)1097-4644(199610)63:1<1::aid-jcb1&gt;3.0.co;2-3"><li><a href="http://dx.doi.org/10.1002/(sici)1097-4644(199610)63:1<1::aid-jcb1&gt;3.0.co;2-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pearson%5Bauthor%5D+AND+Inverted+repeats%2C+stem-loops%2C+and+cruciforms%3A+significance+for+initiation+of+DNA+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APearson+%22Inverted+repeats%2C+stem-loops%2C+and+cruciforms%3A+significance+for+initiation+of+DNA+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000110-Collins1" id="pone.0000110-Collins1"></a>Collins J (1981) Instability of palindromic DNA in <em>Escherichia coli</em>. Cold Spring Harb Symp Quant Biol  45: 409–416.  <ul class="find" data-citedArticleID="5292855" data-doi="10.1101/sqb.1981.045.01.055"><li><a href="http://dx.doi.org/10.1101/sqb.1981.045.01.055" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Collins%5Bauthor%5D+AND+Instability+of+palindromic+DNA+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACollins+%22Instability+of+palindromic+DNA+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000110-Collins2" id="pone.0000110-Collins2"></a>Collins J, Volckaert G, Nevers P (1982) Precise and nearly-precise excision of the symmetrical inverted repeats of Tn5; common features of recA-independent deletion events in <em>Escherichia coli</em>. Gene  19: 139–146.  <ul class="find" data-citedArticleID="5292857" data-doi="10.1016/0378-1119(82)90198-6"><li><a href="http://dx.doi.org/10.1016/0378-1119(82)90198-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Collins%5Bauthor%5D+AND+Precise+and+nearly-precise+excision+of+the+symmetrical+inverted+repeats+of+Tn5%3B+common+features+of+recA-independent+deletion+events+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACollins+%22Precise+and+nearly-precise+excision+of+the+symmetrical+inverted+repeats+of+Tn5%3B+common+features+of+recA-independent+deletion+events+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000110-Albertini1" id="pone.0000110-Albertini1"></a>Albertini AM, Hofer M, Calos MP, Miller JH (1982) On the formation of spontaneous deletions: the importance of short sequence homologies in the generation of large deletions. Cell  29: 319–328.  <ul class="find" data-citedArticleID="5292859" data-doi="10.1016/0092-8674(82)90148-9"><li><a href="http://dx.doi.org/10.1016/0092-8674(82)90148-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Albertini%5Bauthor%5D+AND+On+the+formation+of+spontaneous+deletions%3A+the+importance+of+short+sequence+homologies+in+the+generation+of+large+deletions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAlbertini+%22On+the+formation+of+spontaneous+deletions%3A+the+importance+of+short+sequence+homologies+in+the+generation+of+large+deletions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000110-Glickman1" id="pone.0000110-Glickman1"></a>Glickman BW, Ripley LS (1984) Structural intermediates of deletion mutagenesis: a role for palindromic DNA. Proc Natl Acad Sci U S A  81: 512–516.  <ul class="find" data-citedArticleID="5292861" data-doi="10.1073/pnas.81.2.512"><li><a href="http://dx.doi.org/10.1073/pnas.81.2.512" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glickman%5Bauthor%5D+AND+Structural+intermediates+of+deletion+mutagenesis%3A+a+role+for+palindromic+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGlickman+%22Structural+intermediates+of+deletion+mutagenesis%3A+a+role+for+palindromic+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000110-Nalbantoglu1" id="pone.0000110-Nalbantoglu1"></a>Nalbantoglu J, Hartley D, Phear G, Tear G, Meuth M (1986) Spontaneous deletion formation at the aprt locus of hamster cells: the presence of short sequence homologies and dyad symmetries at deletion termini. Embo J  5: 1199–1204.  <ul class="find" data-citedArticleID="5292863"><li><a href="http://www.crossref.org/guestquery/?auth2=Nalbantoglu&amp;atitle2=Spontaneous+deletion+formation+at+the+aprt+locus+of+hamster+cells%3A+the+presence+of+short+sequence+homologies+and+dyad+symmetries+at+deletion+termini.&amp;auth=Nalbantoglu&amp;atitle=Spontaneous+deletion+formation+at+the+aprt+locus+of+hamster+cells%3A+the+presence+of+short+sequence+homologies+and+dyad+symmetries+at+deletion+termini." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nalbantoglu%5Bauthor%5D+AND+Spontaneous+deletion+formation+at+the+aprt+locus+of+hamster+cells%3A+the+presence+of+short+sequence+homologies+and+dyad+symmetries+at+deletion+termini." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ANalbantoglu+%22Spontaneous+deletion+formation+at+the+aprt+locus+of+hamster+cells%3A+the+presence+of+short+sequence+homologies+and+dyad+symmetries+at+deletion+termini.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000110-Egner1" id="pone.0000110-Egner1"></a>Egner C, Berg DE (1981) Excision of transposon Tn5 is dependent on the inverted repeats but not on the transposase function of Tn5. Proc Natl Acad Sci U S A  78: 459–463.  <ul class="find" data-citedArticleID="5292865" data-doi="10.1073/pnas.78.1.459"><li><a href="http://dx.doi.org/10.1073/pnas.78.1.459" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Egner%5Bauthor%5D+AND+Excision+of+transposon+Tn5+is+dependent+on+the+inverted+repeats+but+not+on+the+transposase+function+of+Tn5." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AEgner+%22Excision+of+transposon+Tn5+is+dependent+on+the+inverted+repeats+but+not+on+the+transposase+function+of+Tn5.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000110-Foster1" id="pone.0000110-Foster1"></a>Foster TJ, Lundblad V, Hanley-Way S, Halling SM, Kleckner N (1981) Three Tn10-associated excision events: relationship to transposition and role of direct and inverted repeats. Cell  23: 215–227.  <ul class="find" data-citedArticleID="5292867" data-doi="10.1016/0092-8674(81)90286-5"><li><a href="http://dx.doi.org/10.1016/0092-8674(81)90286-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Foster%5Bauthor%5D+AND+Three+Tn10-associated+excision+events%3A+relationship+to+transposition+and+role+of+direct+and+inverted+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFoster+%22Three+Tn10-associated+excision+events%3A+relationship+to+transposition+and+role+of+direct+and+inverted+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000110-Lilley1" id="pone.0000110-Lilley1"></a>Lilley DM (1980) The inverted repeat as a recognizable structural feature in supercoiled DNA molecules. Proc Natl Acad Sci U S A  77: 6468–6472.  <ul class="find" data-citedArticleID="5292869" data-doi="10.1073/pnas.77.11.6468"><li><a href="http://dx.doi.org/10.1073/pnas.77.11.6468" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lilley%5Bauthor%5D+AND+The+inverted+repeat+as+a+recognizable+structural+feature+in+supercoiled+DNA+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALilley+%22The+inverted+repeat+as+a+recognizable+structural+feature+in+supercoiled+DNA+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000110-Panayotatos1" id="pone.0000110-Panayotatos1"></a>Panayotatos N, Wells RD (1981) Cruciform structures in supercoiled DNA. Nature  289: 466–470.  <ul class="find" data-citedArticleID="5292871" data-doi="10.1038/289466a0"><li><a href="http://dx.doi.org/10.1038/289466a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Panayotatos%5Bauthor%5D+AND+Cruciform+structures+in+supercoiled+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APanayotatos+%22Cruciform+structures+in+supercoiled+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000110-Sheflin1" id="pone.0000110-Sheflin1"></a>Sheflin LG, Kowalski D (1985) Altered DNA conformations detected by mung bean nuclease occur in promoter and terminator regions of supercoiled pBR322 DNA. Nucleic Acids Res  13: 6137–6154.  <ul class="find" data-citedArticleID="5292873" data-doi="10.1093/nar/13.17.6137"><li><a href="http://dx.doi.org/10.1093/nar/13.17.6137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sheflin%5Bauthor%5D+AND+Altered+DNA+conformations+detected+by+mung+bean+nuclease+occur+in+promoter+and+terminator+regions+of+supercoiled+pBR322+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASheflin+%22Altered+DNA+conformations+detected+by+mung+bean+nuclease+occur+in+promoter+and+terminator+regions+of+supercoiled+pBR322+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000110-Frappier1" id="pone.0000110-Frappier1"></a>Frappier L, Price GB, Martin RG, Zannis-Hadjopoulos M (1989) Characterization of the binding specificity of two anticruciform DNA monoclonal antibodies. J Biol Chem  264: 334–341.  <ul class="find" data-citedArticleID="5292875"><li><a href="http://www.crossref.org/guestquery/?auth2=Frappier&amp;atitle2=Characterization+of+the+binding+specificity+of+two+anticruciform+DNA+monoclonal+antibodies.&amp;auth=Frappier&amp;atitle=Characterization+of+the+binding+specificity+of+two+anticruciform+DNA+monoclonal+antibodies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frappier%5Bauthor%5D+AND+Characterization+of+the+binding+specificity+of+two+anticruciform+DNA+monoclonal+antibodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFrappier+%22Characterization+of+the+binding+specificity+of+two+anticruciform+DNA+monoclonal+antibodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000110-Butler1" id="pone.0000110-Butler1"></a>Butler DK, Yasuda LE, Yao MC (1995) An intramolecular recombination mechanism for the formation of the rRNA gene palindrome of Tetrahymena thermophila. Mol Cell Biol  15: 7117–7126.  <ul class="find" data-citedArticleID="5292877"><li><a href="http://www.crossref.org/guestquery/?auth2=Butler&amp;atitle2=An+intramolecular+recombination+mechanism+for+the+formation+of+the+rRNA+gene+palindrome+of+Tetrahymena+thermophila.&amp;auth=Butler&amp;atitle=An+intramolecular+recombination+mechanism+for+the+formation+of+the+rRNA+gene+palindrome+of+Tetrahymena+thermophila." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Butler%5Bauthor%5D+AND+An+intramolecular+recombination+mechanism+for+the+formation+of+the+rRNA+gene+palindrome+of+Tetrahymena+thermophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AButler+%22An+intramolecular+recombination+mechanism+for+the+formation+of+the+rRNA+gene+palindrome+of+Tetrahymena+thermophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000110-Yasuda1" id="pone.0000110-Yasuda1"></a>Yasuda LF, Yao MC (1991) Short inverted repeats at a free end signal large palindromic DNA formation in Tetrahymena. Cell  67: 505–516.  <ul class="find" data-citedArticleID="5292879" data-doi="10.1016/0092-8674(91)90525-4"><li><a href="http://dx.doi.org/10.1016/0092-8674(91)90525-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yasuda%5Bauthor%5D+AND+Short+inverted+repeats+at+a+free+end+signal+large+palindromic+DNA+formation+in+Tetrahymena." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AYasuda+%22Short+inverted+repeats+at+a+free+end+signal+large+palindromic+DNA+formation+in+Tetrahymena.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000110-Butler2" id="pone.0000110-Butler2"></a>Butler DK, Yasuda LE, Yao MC (1996) Induction of large DNA palindrome formation in yeast: implications for gene amplification and genome stability in eukaryotes. Cell  87: 1115–1122.  <ul class="find" data-citedArticleID="5292881" data-doi="10.1016/s0092-8674(00)81805-x"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81805-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Butler%5Bauthor%5D+AND+Induction+of+large+DNA+palindrome+formation+in+yeast%3A+implications+for+gene+amplification+and+genome+stability+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AButler+%22Induction+of+large+DNA+palindrome+formation+in+yeast%3A+implications+for+gene+amplification+and+genome+stability+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000110-Albrecht1" id="pone.0000110-Albrecht1"></a>Albrecht EB, Hunyady AB, Stark GR, Patterson TE (2000) Mechanisms of <em>sod2</em> gene amplification in <em>Schizosaccharomyces pombe</em>. Mol Biol Cell  11: 873–886.  <ul class="find" data-citedArticleID="5292883" data-doi="10.1091/mbc.11.3.873"><li><a href="http://dx.doi.org/10.1091/mbc.11.3.873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Albrecht%5Bauthor%5D+AND+Mechanisms+of+sod2+gene+amplification+in+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAlbrecht+%22Mechanisms+of+sod2+gene+amplification+in+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000110-Qin1" id="pone.0000110-Qin1"></a>Qin Z, Cohen SN (2000) Long palindromes formed in Streptomyces by nonrecombinational intra-strand annealing. Genes Dev  14: 1789–1796.  <ul class="find" data-citedArticleID="5292885"><li><a href="http://www.crossref.org/guestquery/?auth2=Qin&amp;atitle2=Long+palindromes+formed+in+Streptomyces+by+nonrecombinational+intra-strand+annealing.&amp;auth=Qin&amp;atitle=Long+palindromes+formed+in+Streptomyces+by+nonrecombinational+intra-strand+annealing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Qin%5Bauthor%5D+AND+Long+palindromes+formed+in+Streptomyces+by+nonrecombinational+intra-strand+annealing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AQin+%22Long+palindromes+formed+in+Streptomyces+by+nonrecombinational+intra-strand+annealing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000110-Lobachev1" id="pone.0000110-Lobachev1"></a>Lobachev KS, Gordenin DA, Resnick MA (2002) The Mre11 complex is required for repair of hairpin-capped double-strand breaks and prevention of chromosome rearrangements. Cell  108: 183–193.  <ul class="find" data-citedArticleID="5292887" data-doi="10.1016/s0092-8674(02)00614-1"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)00614-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lobachev%5Bauthor%5D+AND+The+Mre11+complex+is+required+for+repair+of+hairpin-capped+double-strand+breaks+and+prevention+of+chromosome+rearrangements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALobachev+%22The+Mre11+complex+is+required+for+repair+of+hairpin-capped+double-strand+breaks+and+prevention+of+chromosome+rearrangements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000110-Butler3" id="pone.0000110-Butler3"></a>Butler DK, Gillespie D, Steele B (2002) Formation of large palindromic DNA by homologous recombination of short inverted repeat sequences in Saccharomyces cerevisiae. Genetics  161: 1065–1075.  <ul class="find" data-citedArticleID="5292889"><li><a href="http://www.crossref.org/guestquery/?auth2=Butler&amp;atitle2=Formation+of+large+palindromic+DNA+by+homologous+recombination+of+short+inverted+repeat+sequences+in+Saccharomyces+cerevisiae.&amp;auth=Butler&amp;atitle=Formation+of+large+palindromic+DNA+by+homologous+recombination+of+short+inverted+repeat+sequences+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Butler%5Bauthor%5D+AND+Formation+of+large+palindromic+DNA+by+homologous+recombination+of+short+inverted+repeat+sequences+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AButler+%22Formation+of+large+palindromic+DNA+by+homologous+recombination+of+short+inverted+repeat+sequences+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000110-Gray2" id="pone.0000110-Gray2"></a>Gray M, Spencer D (1996) Evolution of microbial life.  Roberts D, Sharp P, Alderson G, Collins M, editors. Cambridge: Cambridge University Press. pp. 107–126.  <ul class="find-nolinks"></ul></li><li><span class="label">45.
              </span><a name="pone.0000110-Andersson1" id="pone.0000110-Andersson1"></a>Andersson SG, Zomorodipour A, Andersson JO, Sicheritz-Ponten T, Alsmark UC, et al.  (1998) The genome sequence of <em>Rickettsia prowazekii</em> and the origin of mitochondria. Nature  396: 133–140.  <ul class="find" data-citedArticleID="5292893" data-doi="10.1038/24094"><li><a href="http://dx.doi.org/10.1038/24094" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Andersson%5Bauthor%5D+AND+The+genome+sequence+of+Rickettsia+prowazekii+and+the+origin+of+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAndersson+%22The+genome+sequence+of+Rickettsia+prowazekii+and+the+origin+of+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000110-Gray3" id="pone.0000110-Gray3"></a>Gray MW (1999) Evolution of organellar genomes. Curr Opin Genet Dev  9: 678–687.  <ul class="find" data-citedArticleID="5292895" data-doi="10.1016/s0959-437x(99)00030-1"><li><a href="http://dx.doi.org/10.1016/s0959-437x(99)00030-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gray%5Bauthor%5D+AND+Evolution+of+organellar+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGray+%22Evolution+of+organellar+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000110-Crozier1" id="pone.0000110-Crozier1"></a>Crozier RH, Crozier YC (1993) The mitochondrial genome of the honeybee <em>Apis mellifera</em>: complete sequence and genome organization. Genetics  133: 97–117.  <ul class="find" data-citedArticleID="5292897"><li><a href="http://www.crossref.org/guestquery/?auth2=Crozier&amp;atitle2=The+mitochondrial+genome+of+the+honeybee+Apis+mellifera%3A+complete+sequence+and+genome+organization.&amp;auth=Crozier&amp;atitle=The+mitochondrial+genome+of+the+honeybee+Apis+mellifera%3A+complete+sequence+and+genome+organization." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crozier%5Bauthor%5D+AND+The+mitochondrial+genome+of+the+honeybee+Apis+mellifera%3A+complete+sequence+and+genome+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACrozier+%22The+mitochondrial+genome+of+the+honeybee+Apis+mellifera%3A+complete+sequence+and+genome+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000110-Dotson1" id="pone.0000110-Dotson1"></a>Dotson EM, Beard CB (2001) Sequence and organization of the mitochondrial genome of the Chagas disease vector, <em>Triatoma dimidiata</em>. Insect Mol Biol  10: 205–215.  <ul class="find" data-citedArticleID="5292899" data-doi="10.1046/j.1365-2583.2001.00258.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2583.2001.00258.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dotson%5Bauthor%5D+AND+Sequence+and+organization+of+the+mitochondrial+genome+of+the+Chagas+disease+vector%2C+Triatoma+dimidiata." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADotson+%22Sequence+and+organization+of+the+mitochondrial+genome+of+the+Chagas+disease+vector%2C+Triatoma+dimidiata.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000110-Brown1" id="pone.0000110-Brown1"></a>Brown W (1985) Molecular Evolutionary Genetics.  Macintyre R, editor. New York: Plenum Press.   <ul class="find-nolinks"></ul></li><li><span class="label">50.
              </span><a name="pone.0000110-Clary1" id="pone.0000110-Clary1"></a>Clary DO, Wolstenholme DR (1987) Drosophila mitochondrial DNA: conserved sequences in the A+T-rich region and supporting evidence for a secondary structure model of the small ribosomal RNA. J Mol Evol  25: 116–125.  <ul class="find" data-citedArticleID="5292903" data-doi="10.1007/bf02101753"><li><a href="http://dx.doi.org/10.1007/bf02101753" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clary%5Bauthor%5D+AND+Drosophila+mitochondrial+DNA%3A+conserved+sequences+in+the+A%2BT-rich+region+and+supporting+evidence+for+a+secondary+structure+model+of+the+small+ribosomal+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AClary+%22Drosophila+mitochondrial+DNA%3A+conserved+sequences+in+the+A%2BT-rich+region+and+supporting+evidence+for+a+secondary+structure+model+of+the+small+ribosomal+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000110-Adams1" id="pone.0000110-Adams1"></a>Adams KL, Palmer JD (2003) Evolution of mitochondrial gene content: gene loss and transfer to the nucleus. Mol Phylogenet Evol  29: 380–395.  <ul class="find" data-citedArticleID="5292905" data-doi="10.1016/s1055-7903(03)00194-5"><li><a href="http://dx.doi.org/10.1016/s1055-7903(03)00194-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adams%5Bauthor%5D+AND+Evolution+of+mitochondrial+gene+content%3A+gene+loss+and+transfer+to+the+nucleus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAdams+%22Evolution+of+mitochondrial+gene+content%3A+gene+loss+and+transfer+to+the+nucleus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000110-Desjardins1" id="pone.0000110-Desjardins1"></a>Desjardins P, Morais R (1990) Sequence and gene organization of the chicken mitochondrial genome. A novel gene order in higher vertebrates. J Mol Biol  212: 599–634.  <ul class="find" data-citedArticleID="5292907" data-doi="10.1016/0022-2836(90)90225-b"><li><a href="http://dx.doi.org/10.1016/0022-2836(90)90225-b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Desjardins%5Bauthor%5D+AND+Sequence+and+gene+organization+of+the+chicken+mitochondrial+genome.+A+novel+gene+order+in+higher+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADesjardins+%22Sequence+and+gene+organization+of+the+chicken+mitochondrial+genome.+A+novel+gene+order+in+higher+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000110-Saccone1" id="pone.0000110-Saccone1"></a>Saccone C, Pesole G, Sbisa E (1991) The main regulatory region of mammalian mitochondrial DNA: structure-function model and evolutionary pattern. J Mol Evol  33: 83–91.  <ul class="find" data-citedArticleID="5292909" data-doi="10.1007/bf02100199"><li><a href="http://dx.doi.org/10.1007/bf02100199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Saccone%5Bauthor%5D+AND+The+main+regulatory+region+of+mammalian+mitochondrial+DNA%3A+structure-function+model+and+evolutionary+pattern." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASaccone+%22The+main+regulatory+region+of+mammalian+mitochondrial+DNA%3A+structure-function+model+and+evolutionary+pattern.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000110-Tapper1" id="pone.0000110-Tapper1"></a>Tapper DP, Clayton DA (1981) Mechanism of replication of human mitochondrial DNA. Localization of the 5′ ends of nascent daughter strands. J Biol Chem  256: 5109–5115.  <ul class="find" data-citedArticleID="5292911"><li><a href="http://www.crossref.org/guestquery/?auth2=Tapper&amp;atitle2=Mechanism+of+replication+of+human+mitochondrial+DNA.+Localization+of+the+5%3F+ends+of+nascent+daughter+strands.&amp;auth=Tapper&amp;atitle=Mechanism+of+replication+of+human+mitochondrial+DNA.+Localization+of+the+5%3F+ends+of+nascent+daughter+strands." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tapper%5Bauthor%5D+AND+Mechanism+of+replication+of+human+mitochondrial+DNA.+Localization+of+the+5%E2%80%B2+ends+of+nascent+daughter+strands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATapper+%22Mechanism+of+replication+of+human+mitochondrial+DNA.+Localization+of+the+5%E2%80%B2+ends+of+nascent+daughter+strands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000110-Chang1" id="pone.0000110-Chang1"></a>Chang DD, Clayton DA (1985) Priming of human mitochondrial DNA replication occurs at the light-strand promoter. Proc Natl Acad Sci U S A  82: 351–355.  <ul class="find" data-citedArticleID="5292913" data-doi="10.1073/pnas.82.2.351"><li><a href="http://dx.doi.org/10.1073/pnas.82.2.351" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chang%5Bauthor%5D+AND+Priming+of+human+mitochondrial+DNA+replication+occurs+at+the+light-strand+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AChang+%22Priming+of+human+mitochondrial+DNA+replication+occurs+at+the+light-strand+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000110-Hixson1" id="pone.0000110-Hixson1"></a>Hixson JE, Wong TW, Clayton DA (1986) Both the conserved stem-loop and divergent 5′-flanking sequences are required for initiation at the human mitochondrial origin of light-strand DNA replication. J Biol Chem  261: 2384–2390.  <ul class="find" data-citedArticleID="5292915"><li><a href="http://www.crossref.org/guestquery/?auth2=Hixson&amp;atitle2=Both+the+conserved+stem-loop+and+divergent+5%3F-flanking+sequences+are+required+for+initiation+at+the+human+mitochondrial+origin+of+light-strand+DNA+replication.&amp;auth=Hixson&amp;atitle=Both+the+conserved+stem-loop+and+divergent+5%3F-flanking+sequences+are+required+for+initiation+at+the+human+mitochondrial+origin+of+light-strand+DNA+replication." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hixson%5Bauthor%5D+AND+Both+the+conserved+stem-loop+and+divergent+5%E2%80%B2-flanking+sequences+are+required+for+initiation+at+the+human+mitochondrial+origin+of+light-strand+DNA+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHixson+%22Both+the+conserved+stem-loop+and+divergent+5%E2%80%B2-flanking+sequences+are+required+for+initiation+at+the+human+mitochondrial+origin+of+light-strand+DNA+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000110-Garesse1" id="pone.0000110-Garesse1"></a>Garesse R (1988) <em>Drosophila melanogaster</em> mitochondrial DNA: gene organization and evolutionary considerations. Genetics  118: 649–663.  <ul class="find" data-citedArticleID="5292917"><li><a href="http://www.crossref.org/guestquery/?auth2=Garesse&amp;atitle2=Drosophila+melanogaster+mitochondrial+DNA%3A+gene+organization+and+evolutionary+considerations.&amp;auth=Garesse&amp;atitle=Drosophila+melanogaster+mitochondrial+DNA%3A+gene+organization+and+evolutionary+considerations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Garesse%5Bauthor%5D+AND+Drosophila+melanogaster+mitochondrial+DNA%3A+gene+organization+and+evolutionary+considerations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGaresse+%22Drosophila+melanogaster+mitochondrial+DNA%3A+gene+organization+and+evolutionary+considerations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000110-Goddard1" id="pone.0000110-Goddard1"></a>Goddard JM, Wolstenholme DR (1978) Origin and direction of replication in mitochondrial DNA molecules from <em>Drosophila melanogaster</em>. Proc Natl Acad Sci U S A  75: 3886–3890.  <ul class="find" data-citedArticleID="5292919" data-doi="10.1073/pnas.75.8.3886"><li><a href="http://dx.doi.org/10.1073/pnas.75.8.3886" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Goddard%5Bauthor%5D+AND+Origin+and+direction+of+replication+in+mitochondrial+DNA+molecules+from+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGoddard+%22Origin+and+direction+of+replication+in+mitochondrial+DNA+molecules+from+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000110-Goddard2" id="pone.0000110-Goddard2"></a>Goddard JM, Wolstenholme DR (1980) Origin and direction of replication in mitochondrial DNA molecules from the genus Drosophila. Nucleic Acids Res  8: 741–757.  <ul class="find" data-citedArticleID="5292921"><li><a href="http://www.crossref.org/guestquery/?auth2=Goddard&amp;atitle2=Origin+and+direction+of+replication+in+mitochondrial+DNA+molecules+from+the+genus+Drosophila.&amp;auth=Goddard&amp;atitle=Origin+and+direction+of+replication+in+mitochondrial+DNA+molecules+from+the+genus+Drosophila." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Goddard%5Bauthor%5D+AND+Origin+and+direction+of+replication+in+mitochondrial+DNA+molecules+from+the+genus+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGoddard+%22Origin+and+direction+of+replication+in+mitochondrial+DNA+molecules+from+the+genus+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.XML" value="83112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.PDF" value="469569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g001.TIF" value="1153692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g001.PNG_I" value="16707"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g001.PNG_S" value="3415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g001.PNG_M" value="40429"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g001.PNG_L" value="967323"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g002.TIF" value="1273292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g002.PNG_I" value="32424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g002.PNG_S" value="4613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g002.PNG_M" value="84961"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g002.PNG_L" value="918043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g003.TIF" value="1477304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g003.PNG_I" value="30486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g003.PNG_S" value="5045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g003.PNG_M" value="76143"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g003.PNG_L" value="837623"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g004.TIF" value="1763560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g004.PNG_I" value="32463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g004.PNG_S" value="5313"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g004.PNG_M" value="80613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g004.PNG_L" value="826677"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g005.TIF" value="3828802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g005.PNG_I" value="53747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g005.PNG_S" value="6772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g005.PNG_M" value="163202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g005.PNG_L" value="3234532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g006.TIF" value="670492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g006.PNG_I" value="44833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g006.PNG_S" value="5799"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g006.PNG_M" value="66242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g006.PNG_L" value="313864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g007.TIF" value="249200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g007.PNG_I" value="42139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g007.PNG_S" value="5515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g007.PNG_M" value="107143"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.g007.PNG_L" value="178737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.s001.XLS" value="95232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.s002.PDF" value="8341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000110.s003.PDF" value="1609936"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000110&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000110" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000110&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000110&volume=&issue=&title=Unusually Long Palindromes Are Abundant in Mitochondrial Control Regions of Insects and Nematodes&author_name=K.%20P.%20Arunkumar%2C%20Javaregowda%20Nagaraju&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000110" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000110" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000110" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000110&amp;t=Unusually%20Long%20Palindromes%20Are%20Abundant%20in%20Mitochondrial%20Control%20Regions%20of%20Insects%20and%20Nematodes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000110&title=Unusually%20Long%20Palindromes%20Are%20Abundant%20in%20Mitochondrial%20Control%20Regions%20of%20Insects%20and%20Nematodes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000110&amp;title=Unusually%20Long%20Palindromes%20Are%20Abundant%20in%20Mitochondrial%20Control%20Regions%20of%20Insects%20and%20Nematodes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000110" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000110&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Unusually Long Palindromes Are Abundant in Mitochondrial Control Regions of Insects and Nematodes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000110';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Unusually%20Long%20Palindromes%20Are%20Abundant%20in%20Mitochondrial%20Control%20Regions%20of%20Insects%20and%20Nematodes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000110" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000110" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+genomics%22" title="Search for articles in the subject area:'Animal genomics'"><div class="flagText">Animal genomics</div></a>
              <div data-categoryid="17455" data-articleid="23058"
                   data-categoryname="Animal genomics"
                   class="flagImage" title="Flag 'Animal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+replication%22" title="Search for articles in the subject area:'DNA replication'"><div class="flagText">DNA replication</div></a>
              <div data-categoryid="19773" data-articleid="23058"
                   data-categoryname="DNA replication"
                   class="flagImage" title="Flag 'DNA replication' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Insects%22" title="Search for articles in the subject area:'Insects'"><div class="flagText">Insects</div></a>
              <div data-categoryid="42993" data-articleid="23058"
                   data-categoryname="Insects"
                   class="flagImage" title="Flag 'Insects' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Invertebrate+genomics%22" title="Search for articles in the subject area:'Invertebrate genomics'"><div class="flagText">Invertebrate genomics</div></a>
              <div data-categoryid="17221" data-articleid="23058"
                   data-categoryname="Invertebrate genomics"
                   class="flagImage" title="Flag 'Invertebrate genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondria%22" title="Search for articles in the subject area:'Mitochondria'"><div class="flagText">Mitochondria</div></a>
              <div data-categoryid="32453" data-articleid="23058"
                   data-categoryname="Mitochondria"
                   class="flagImage" title="Flag 'Mitochondria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondrial+DNA%22" title="Search for articles in the subject area:'Mitochondrial DNA'"><div class="flagText">Mitochondrial DNA</div></a>
              <div data-categoryid="19683" data-articleid="23058"
                   data-categoryname="Mitochondrial DNA"
                   class="flagImage" title="Flag 'Mitochondrial DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="47429" data-articleid="23058"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Vertebrates%22" title="Search for articles in the subject area:'Vertebrates'"><div class="flagText">Vertebrates</div></a>
              <div data-categoryid="43095" data-articleid="23058"
                   data-categoryname="Vertebrates"
                   class="flagImage" title="Flag 'Vertebrates' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3000'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8847'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4801&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
