

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Toxicity Assays in Nanodrops Combining Bioassay and Morphometric Endpoints</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000163"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000163" />

    <meta name="citation_title" content="Toxicity Assays in Nanodrops Combining Bioassay and Morphometric Endpoints"/>
    <meta itemprop="name" content="Toxicity Assays in Nanodrops Combining Bioassay and Morphometric Endpoints"/>

      <meta name="citation_author" content="Frédéric Lemaire"/>
            <meta name="citation_author_institution" content="Commissariat à l'Énergie Atomique (CEA), DSV, Cellular Responses and Dynamics Department (DRDC), Laboratoire Biopuces, Commissariat à l'Energie Atomique Centre de Grenoble, Grenoble, France"/>
      <meta name="citation_author" content="Céline A. Mandon"/>
            <meta name="citation_author_institution" content="Centre National de la Recherche Scientifique UMR 5534, Université Claude Bernard, Laboratoire du Stress Oxydant, Chaperons, Apoptose, Centre de Génétique Moléculaire et Cellulaire, Villeurbanne, France"/>
      <meta name="citation_author" content="Julien Reboud"/>
            <meta name="citation_author_institution" content="Commissariat à l'Énergie Atomique (CEA), DSV, Cellular Responses and Dynamics Department (DRDC), Laboratoire Biopuces, Commissariat à l'Energie Atomique Centre de Grenoble, Grenoble, France"/>
      <meta name="citation_author" content="Alexandre Papine"/>
            <meta name="citation_author_institution" content="IMSTAR S.A., Paris, France"/>
      <meta name="citation_author" content="Jesus Angulo"/>
            <meta name="citation_author_institution" content="Ecole des Mines, Centre de Morphologie Mathématique, Fontainebleau, France"/>
      <meta name="citation_author" content="Hervé Pointu"/>
            <meta name="citation_author_institution" content="Commissariat à l'Énergie Atomique (CEA), DSV, Cellular Responses and Dynamics Department (DRDC), Laboratoire Biopuces, Commissariat à l'Energie Atomique Centre de Grenoble, Grenoble, France"/>
      <meta name="citation_author" content="Chantal Diaz-Latoud"/>
            <meta name="citation_author_institution" content="Centre National de la Recherche Scientifique UMR 5534, Université Claude Bernard, Laboratoire du Stress Oxydant, Chaperons, Apoptose, Centre de Génétique Moléculaire et Cellulaire, Villeurbanne, France"/>
      <meta name="citation_author" content="Christian Lajaunie"/>
            <meta name="citation_author_institution" content="Ecole des Mines, Centre for Computational Biology, Fontainebleau, France"/>
      <meta name="citation_author" content="François Chatelain"/>
            <meta name="citation_author_institution" content="Commissariat à l'Énergie Atomique (CEA), DSV, Cellular Responses and Dynamics Department (DRDC), Laboratoire Biopuces, Commissariat à l'Energie Atomique Centre de Grenoble, Grenoble, France"/>
      <meta name="citation_author" content="André-Patrick Arrigo"/>
            <meta name="citation_author_institution" content="Centre National de la Recherche Scientifique UMR 5534, Université Claude Bernard, Laboratoire du Stress Oxydant, Chaperons, Apoptose, Centre de Génétique Moléculaire et Cellulaire, Villeurbanne, France"/>
      <meta name="citation_author" content="Béatrice Schaack"/>
            <meta name="citation_author_institution" content="Commissariat à l'Énergie Atomique (CEA), DSV, Cellular Responses and Dynamics Department (DRDC), Laboratoire Biopuces, Commissariat à l'Energie Atomique Centre de Grenoble, Grenoble, France"/>

    <meta name="citation_date" content="2007/1/17"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000163.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e163"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The three Rs: the way forward: the report and recommendations of ECVAM Workshop 11.; citation_author=M Balls; citation_author=AM Goldberg; citation_author=JH Fentem; citation_author=CL Broadhead; citation_author=RL Burch; citation_journal_title=Altern Lab Anim; citation_volume=23; citation_number=1; citation_pages=838-866; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Microarray compound screening (microARCS) to identify inhibitors of HIV integrase.; citation_author=CA David; citation_author=T Middleton; citation_author=D Montgomery; citation_author=HB Lim; citation_author=W Kati; citation_journal_title=J Biomol Screen; citation_volume=7; citation_number=2; citation_pages=259-266; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A ‘drop-chip’ cell array for high throughput DNA and siRNA transfection combined with drug screening.; citation_author=J Schaack B Reboud; citation_author=S Combe; citation_author=B Fouqué; citation_author=F Berger; citation_author=S Boccard; citation_author=O Filhol-Cochet; citation_author=F Chatelain; citation_journal_title=NanoBiotechnology; citation_volume=1; citation_number=3; citation_pages=166-194; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=On the complete determination of biological systems.; citation_author=DW Selinger; citation_author=MA Wright; citation_author=GM Church; citation_journal_title=Trends Biotechnol; citation_volume=21; citation_number=4; citation_pages=251-254; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Quantitative analysis of highly parallel transfection in cell microarrays.; citation_author=S Baghdoyan; citation_author=Y Roupioz; citation_author=A Pitaval; citation_author=D Castel; citation_author=E Khomyakova; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=5; citation_pages=e77; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Diffusion dependent cell behavior in microenvironments.; citation_author=H Yu; citation_author=I Meyvantsson; citation_author=IA Shkel; citation_author=DJ Beebe; citation_journal_title=Lab Chip; citation_volume=5; citation_number=6; citation_pages=1089-1095; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Systems biology: deviations in mating.; citation_author=A Eldar; citation_author=M Elowitz; citation_journal_title=Nature; citation_volume=437; citation_number=7; citation_pages=631-632; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of a small molecule that induces mitotic arrest using a simplified high-content screening assay and data analysis method.; citation_author=CJ Wilson; citation_author=Y Si; citation_author=CM Thompsons; citation_author=A Smellie; citation_author=MA Ashwell; citation_journal_title=J Biomol Screen; citation_volume=11; citation_number=8; citation_pages=21-28; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Chemical stress sensitive luminescent human cells: molecular biology approach using inducible Drosophila melanogaster hsp22 promoter.; citation_author=CA Mandon; citation_author=C Diaz; citation_author=AP Arrigo; citation_author=LJ Blum; citation_journal_title=Biochem Biophys Res Commun; citation_volume=335; citation_number=9; citation_pages=536-544; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dithiocarbamate fungicide thiram detection: comparison of bioluminescent and fluorescent whole-cell bioassays based on hsp22 stress promoter induction.; citation_author=CA Mandon; citation_author=C Diaz-Latoud; citation_author=AP Arrigo; citation_author=LJ Blum; citation_journal_title=J Biotechnol; citation_volume=124; citation_number=10; citation_pages=392-402; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evaluation of an in vitro hsp70 induction test for toxicity assessment of complex mixtures: comparison with chemical analyses and ecotoxicity tests.; citation_author=S Ait-Aissa; citation_author=P Pandard; citation_author=H Magaud; citation_author=AP Arrigo; citation_author=E Thybaud; citation_journal_title=Ecotoxicol Environ Saf; citation_volume=54; citation_number=11; citation_pages=92-104; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Tobacco smoke induces coordinate activation of HSF and inhibition of NFkappaB in human monocytes: effects on TNFalpha release.; citation_author=M Vayssier; citation_author=F Favatier; citation_author=F Pinot; citation_author=M Bachelet; citation_author=BS Polla; citation_journal_title=Biochem Biophys Res Commun; citation_volume=252; citation_number=12; citation_pages=249-256; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Heat shock proteins. Facts, thoughts, and dreams. A. De Maio. Shock 11:1–12, 1999.; citation_author=HR Wong; citation_journal_title=Shock; citation_volume=12; citation_number=13; citation_pages=323-325; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Use of human-derived liver cell lines for the detection of environmental and dietary genotoxicants; current state of knowledge.; citation_author=S Knasmuller; citation_author=V Mersch-Sundermann; citation_author=S Kevekordes; citation_author=F Darroudi; citation_author=WW Huber; citation_journal_title=Toxicology; citation_volume=198; citation_number=14; citation_pages=315-328; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=In vitro cytotoxicity assays: comparison of LDH, neutral red, MTT and protein assay in hepatoma cell lines following exposure to cadmium chloride.; citation_author=G Fotakis; citation_author=JA Timbrell; citation_journal_title=Toxicol Lett; citation_volume=160; citation_number=15; citation_pages=171-177; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=In vitro long-term cytotoxicity testing of 27 MEIC chemicals on Hep G2 cells and comparison with acute human toxicity data.; citation_author=EM Scheers; citation_author=B Ekwall; citation_author=PJ Dierickx; citation_journal_title=Toxicol In Vitro; citation_volume=15; citation_number=16; citation_pages=153-161; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Expression of hsp 27, hsp 60, hsc 70, and hsp 70 stress response genes in cultured human urothelial cells (UROtsa) exposed to lethal and sublethal concentrations of sodium arsenite.; citation_author=MR Rossi; citation_author=S Somji; citation_author=SH Garrett; citation_author=MA Sens; citation_author=J Nath; citation_journal_title=Environ Health Perspect; citation_volume=110; citation_number=17; citation_pages=1225-1232; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Cadmium affects the redox state of rat liver glucocorticoid receptor.; citation_author=D Elez; citation_author=J Dundjerski; citation_author=G Matic; citation_journal_title=Cell Biol Toxicol; citation_volume=17; citation_number=18; citation_pages=169-177; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Arsenic induces expression of the multidrug resistance-associated protein 2 (MRP2) gene in primary rat and human hepatocytes.; citation_author=L Vernhet; citation_author=MP Seite; citation_author=N Allain; citation_author=A Guillouzo; citation_author=O Fardel; citation_journal_title=J Pharmacol Exp Ther; citation_volume=298; citation_number=19; citation_pages=234-239; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Acute oxidative stress is associated with cell proliferation in the mouse liver.; citation_author=N Dragin; citation_author=M Smani; citation_author=S Arnaud-Dabernat; citation_author=C Dubost; citation_author=I Moranvillier; citation_journal_title=FEBS Lett; citation_volume=580; citation_number=20; citation_pages=3845-3852; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phenotypic anchoring of arsenic and cadmium toxicity in three hepatic-related cell systems reveals compound- and cell-specific selective up-regulation of stress protein expression: Implications for fingerprint profiling of cytotoxicity.; citation_author=E Gottschalg; citation_author=NE Moore; citation_author=AK Ryan; citation_author=LC Travis; citation_author=RC Waller; citation_journal_title=Chem Biol Interact.; citation_number=21; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Carcinogenic metal induced sites of reactive oxygen species formation in hepatocytes.; citation_author=J Pourahmad; citation_author=PJ O'Brien; citation_author=F Jokar; citation_author=B Daraei; citation_journal_title=Toxicol In Vitro; citation_volume=17; citation_number=22; citation_pages=803-810; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Stress proteins induced by arsenic.; citation_author=LM Del Razo; citation_author=B Quintanilla-Vega; citation_author=E Brambila-Colombres; citation_author=ES Calderon-Aranda; citation_author=M Manno; citation_journal_title=Toxicol Appl Pharmacol; citation_volume=177; citation_number=23; citation_pages=132-148; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Morphological characterisation of BGM (Buffalo Green Monkey) cell line exposed to low doses of cadmium chloride.; citation_author=D Romero; citation_author=M Gomez-Zapata; citation_author=A Luna; citation_author=AJ Garcia-Fernandez; citation_journal_title=Toxicol In Vitro; citation_volume=17; citation_number=24; citation_pages=293-299; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The extracellular matrix guides the orientation of the cell division axis.; citation_author=M Thery; citation_author=V Racine; citation_author=A Pepin; citation_author=M Piel; citation_author=Y Chen; citation_journal_title=Nat Cell Biol; citation_volume=7; citation_number=25; citation_pages=947-953; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Characterization of the cellular response during apoptosis induction in cadmium-treated Hep G2 human hepatoma cells.; citation_author=HH Aydin; citation_author=HA Celik; citation_author=R Deveci; citation_author=E Terzioglu; citation_author=S Karacali; citation_journal_title=Biol Trace Elem Res; citation_volume=95; citation_number=26; citation_pages=139-153; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Proof without prejudice: use of the Kolmogorov-Smirnov test for the analysis of histograms from flow systems and other sources.; citation_author=IT Young; citation_journal_title=J Histochem Cytochem; citation_volume=25; citation_number=27; citation_pages=935-941; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=High-content screening with siRNA optimizes a cell biological approach to drug discovery: defining the role of P53 activation in the cellular response to anticancer drugs.; citation_author=KA Giuliano; citation_author=YT Chen; citation_author=DL Taylor; citation_journal_title=J Biomol Screen; citation_volume=9; citation_number=28; citation_pages=557-568; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Systems cell biology knowledge created from high content screening.; citation_author=KA Giuliano; citation_author=WS Cheung; citation_author=DP Curran; citation_author=BW Day; citation_author=AJ Kassick; citation_journal_title=Assay Drug Dev Technol; citation_volume=3; citation_number=29; citation_pages=501-514; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Detection and classification of threat agents via high-content assays of mammalian cells.; citation_author=SB Tencza; citation_author=MA Sipe; citation_journal_title=J Appl Toxicol; citation_volume=24; citation_number=30; citation_pages=371-377; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A cell array biosensor for environmental toxicity analysis.; citation_author=JH Lee; citation_author=RJ Mitchell; citation_author=BC Kim; citation_author=DC Cullen; citation_author=MB Gu; citation_journal_title=Biosens Bioelectron; citation_volume=21; citation_number=31; citation_pages=500-507; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An extracellular matrix microarray for probing cellular differentiation.; citation_author=CJ Flaim; citation_author=S Chien; citation_author=SN Bhatia; citation_journal_title=Nat Methods; citation_volume=2; citation_number=32; citation_pages=119-125; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Metabolizing enzyme toxicology assay chip (MetaChip) for high-throughput microscale toxicity analyses.; citation_author=MY Lee; citation_author=CB Park; citation_author=JS Dordick; citation_author=DS Clark; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=33; citation_pages=983-987; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=sHsp as novel regulators of programmed cell death and tumorigenicity.; citation_author=AP Arrigo; citation_journal_title=Pathol Biol (Paris); citation_volume=48; citation_number=34; citation_pages=280-288; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Activation of the hsp70 promoter by environmental inorganic and organic chemicals: relationships with cytotoxicity and lipophilicity.; citation_author=S Ait-Aissa; citation_author=J Porcher; citation_author=A Arrigo; citation_author=C Lambre; citation_journal_title=Toxicology; citation_volume=145; citation_number=35; citation_pages=147-157; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Acute exposure to cadmium causes severe liver injury in rats.; citation_author=RE Dudley; citation_author=DJ Svoboda; citation_author=C Klaassen; citation_journal_title=Toxicol Appl Pharmacol; citation_volume=65; citation_number=36; citation_pages=302-313; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Induction of hepatic metallothionein synthesis by endoplasmic reticulum stress in mice.; citation_author=M Kondoh; citation_author=M Tsukada; citation_author=M Kuronaga; citation_author=M Higashimoto; citation_author=M Takiguchi; citation_journal_title=Toxicol Lett; citation_volume=148; citation_number=37; citation_pages=133-139; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=ROS up-regulation mediates Ras-induced changes of cell morphology and motility.; citation_author=AY Alexandrova; citation_author=PB Kopnin; citation_author=JM Vasiliev; citation_author=BP Kopnin; citation_journal_title=Exp Cell Res; citation_number=38; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An Overview of Morphological Segmentation.; citation_author=F Meyer; citation_journal_title=International Journal of Pattern Recognition and Artificial Intelligence Vol.; citation_volume=15; citation_number=39; citation_pages=1089-1118; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Levelings, Image Simplification Filters for Segmentation.; citation_author=F Meyer; citation_journal_title=Journal of Mathematical Imaging and Vision Vol.; citation_volume=20; citation_number=40; citation_pages=59-72; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Course on morphological operators.; citation_author=J Serra; citation_number=41; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Automatic analysis of DNA microarray images using mathematical morphology.; citation_author=J Angulo; citation_author=J Serra; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=42; citation_pages=553-562; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000163" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Toxicity Assays in Nanodrops Combining Bioassay and Morphometric Endpoints"/>
    <meta name="twitter:description" content="BackgroundImproved chemical hazard management such as REACH policy objective as well as drug ADMETOX prediction, while limiting the extent of animal testing, requires the development of increasingly high throughput as well as highly pertinent in vitro toxicity assays.MethodologyThis report describes a new in vitro method for toxicity testing, combining cell-based assays in nanodrop Cell-on-Chip format with the use of a genetically engineered stress sensitive hepatic cell line. We tested the behavior of a stress inducible fluorescent HepG2 model in which Heat Shock Protein promoters controlled Enhanced-Green Fluorescent Protein expression upon exposure to Cadmium Chloride (CdCl2), Sodium Arsenate (NaAsO2) and Paraquat. In agreement with previous studies based on a micro-well format, we could observe a chemical-specific response, identified through differences in dynamics and amplitude. We especially determined IC50 values for CdCl2 and NaAsO2, in agreement with published data. Individual cell identification via image-based screening allowed us to perform multiparametric analyses.ConclusionsUsing pre/sub lethal cell stress instead of cell mortality, we highlighted the high significance and the superior sensitivity of both stress promoter activation reporting and cell morphology parameters in measuring the cell response to a toxicant. These results demonstrate the first generation of high-throughput and high-content assays, capable of assessing chemical hazards in vitro within the REACH policy framework."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000163.g005"/>

  <meta property="og:title" content="Toxicity Assays in Nanodrops Combining Bioassay and Morphometric Endpoints" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000163" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3837'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1892'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4657&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000163">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000163" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000163&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Toxicity Assays in Nanodrops Combining Bioassay and Morphometric Endpoints
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frédéric Lemaire
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Frédéric Lemaire, Céline A. Mandon</p>

              
              

                <p>Affiliation:
                  Commissariat à l'Énergie Atomique (CEA), DSV, Cellular Responses and Dynamics Department (DRDC), Laboratoire Biopuces, Commissariat à l'Energie Atomique Centre de Grenoble, Grenoble, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Céline A. Mandon
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Frédéric Lemaire, Céline A. Mandon</p>

              
              

                <p>Affiliation:
                  Centre National de la Recherche Scientifique UMR 5534, Université Claude Bernard, Laboratoire du Stress Oxydant, Chaperons, Apoptose, Centre de Génétique Moléculaire et Cellulaire, Villeurbanne, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Julien Reboud, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Commissariat à l'Énergie Atomique (CEA), DSV, Cellular Responses and Dynamics Department (DRDC), Laboratoire Biopuces, Commissariat à l'Energie Atomique Centre de Grenoble, Grenoble, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexandre Papine, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  IMSTAR S.A., Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jesus Angulo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Ecole des Mines, Centre de Morphologie Mathématique, Fontainebleau, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hervé Pointu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Commissariat à l'Énergie Atomique (CEA), DSV, Cellular Responses and Dynamics Department (DRDC), Laboratoire Biopuces, Commissariat à l'Energie Atomique Centre de Grenoble, Grenoble, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chantal Diaz-Latoud, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre National de la Recherche Scientifique UMR 5534, Université Claude Bernard, Laboratoire du Stress Oxydant, Chaperons, Apoptose, Centre de Génétique Moléculaire et Cellulaire, Villeurbanne, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christian Lajaunie, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Ecole des Mines, Centre for Computational Biology, Fontainebleau, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            François Chatelain, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Commissariat à l'Énergie Atomique (CEA), DSV, Cellular Responses and Dynamics Department (DRDC), Laboratoire Biopuces, Commissariat à l'Energie Atomique Centre de Grenoble, Grenoble, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            André-Patrick Arrigo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre National de la Recherche Scientifique UMR 5534, Université Claude Bernard, Laboratoire du Stress Oxydant, Chaperons, Apoptose, Centre de Génétique Moléculaire et Cellulaire, Villeurbanne, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Béatrice Schaack
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:beatrice.schaack@ cea.fr">beatrice.schaack@ cea.fr</a>
            </p>

                <p>Affiliation:
                  Commissariat à l'Énergie Atomique (CEA), DSV, Cellular Responses and Dynamics Department (DRDC), Laboratoire Biopuces, Commissariat à l'Energie Atomique Centre de Grenoble, Grenoble, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 17, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000163</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000163" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000163" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000163" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000163" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000163-g001" data-doi="info:doi/10.1371/journal.pone.0000163" data-uri="info:doi/10.1371/journal.pone.0000163.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000163-g002" data-doi="info:doi/10.1371/journal.pone.0000163" data-uri="info:doi/10.1371/journal.pone.0000163.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000163-g003" data-doi="info:doi/10.1371/journal.pone.0000163" data-uri="info:doi/10.1371/journal.pone.0000163.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000163-g004" data-doi="info:doi/10.1371/journal.pone.0000163" data-uri="info:doi/10.1371/journal.pone.0000163.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000163-g005" data-doi="info:doi/10.1371/journal.pone.0000163" data-uri="info:doi/10.1371/journal.pone.0000163.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000163">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000163" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Improved chemical hazard management such as REACH policy objective as well as drug ADMETOX prediction, while limiting the extent of animal testing, requires the development of increasingly high throughput as well as highly pertinent in vitro toxicity assays.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>This report describes a new <em>in vitro</em> method for toxicity testing, combining cell-based assays in nanodrop Cell-on-Chip format with the use of a genetically engineered stress sensitive hepatic cell line. We tested the behavior of a stress inducible fluorescent HepG2 model in which Heat Shock Protein promoters controlled Enhanced-Green Fluorescent Protein expression upon exposure to Cadmium Chloride (CdCl<sub>2</sub>), Sodium Arsenate (NaAsO<sub>2</sub>) and Paraquat. In agreement with previous studies based on a micro-well format, we could observe a chemical-specific response, identified through differences in dynamics and amplitude. We especially determined IC50 values for CdCl<sub>2</sub> and NaAsO<sub>2</sub>, in agreement with published data. Individual cell identification via image-based screening allowed us to perform multiparametric analyses.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Using pre/sub lethal cell stress instead of cell mortality, we highlighted the high significance and the superior sensitivity of both stress promoter activation reporting and cell morphology parameters in measuring the cell response to a toxicant. These results demonstrate the first generation of high-throughput and high-content assays, capable of assessing chemical hazards <em>in vitro</em> within the REACH policy framework.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Lemaire F, Mandon CA, Reboud J, Papine A, Angulo J, et al.  (2007) Toxicity Assays in Nanodrops Combining Bioassay and Morphometric Endpoints. PLoS ONE 2(1):
          e163.
            doi:10.1371/journal.pone.0000163</p><p><strong>Academic Editor: </strong>Axel Imhof, University of Munich, Germany</p><p><strong>Received:</strong> August 7, 2006; <strong>Accepted:</strong> September 18, 2006; <strong>Published:</strong> January 17, 2007</p><p><strong>Copyright:</strong> © 2007 Lemaire et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the European STREP project TOXDROP 513698. The following organizations are part of TOXDROP consortium: CEA, DSV, DRDC, Laboratoire Biopuces, Commissariat à l'Energie Atomique de Grenoble, Bat 40 20, Laboratoire Biopuces, 17 rue des Martyrs, Grenoble, F-38054 France. CNRS, UMR 5534, Laboratoire du Stress Oxydant, Chaperons, Apoptose, Centre de Génétique Moléculaire et Cellulaire, Université Claude Bernard, Bat. Gregor Mendel, 16 rue Dubois, Villeurbanne,F-69622 France. IMSTAR S.A., 60 rue Notre-Dame des Champs, Paris, F-75006 France.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The European Union has elaborated a new set of rules for the Registration, Evaluation and Authorization of Chemicals (REACH; white paper policy, IP/03/1477, Brussels) for all the chemicals registered for use after 1981. This new policy shifts the responsibility to establish proof that a chemical is safe from public health organizations to industry. This improved chemical hazard management will require extensive toxicological evaluation of new chemical entities. In classic laboratory testing, the evaluation of the 3 000 compounds registered since 1981 – with 1 500 considered as high concern – would lead to a significant increase in animal testing, in contradiction with the general aim of reducing the number of animal experiments, <a href="#pone.0000163-Balls1">[1]</a>. Hence <em>in silico</em> approaches such as computer analysis of epidemiological data and extrapolation of chemical structure knowledge (QSAR, Quantitative Structure Activity Relationship) have blossomed with the aim of reducing costs and rationalizing the registration process. Whenever possible the use of validated <em>in vitro</em> toxicology testing methods will be promoted. Innovative high throughput <em>in vitro</em> techniques should raise the standard, reproducibility, accuracy and depth of analysis.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In the race for high throughput performance, new formats of biochips have been created, moving on from cultures in microwells to microfluidics and, in particular, miniature drops on silicon slides. For example an enzyme link-immunoassay has been described by David et al to illustrate the selection of 8640 compounds using an agarose covered microarray <a href="#pone.0000163-David1">[2]</a>.Following this trend, we have developed the Cell-on-Chip device <a href="#pone.0000163-SchaackBReboud1">[3]</a>, where several hundreds of individual nanoliter drops arrayed on a small patterned glass substrate act like as many independent cell cultures. This concept was previously used in drop-based assays to explore gene expression and cellular responses <a href="#pone.0000163-SchaackBReboud1">[3]</a> thus potentially adding functional information to the essentially descriptive large scale studies on genome, transcriptome and proteome performed within the emerging paradigm of Systems Biology <a href="#pone.0000163-Selinger1">[4]</a>. We have combined this device with IMSTAR Pathfinder™ automated image capture and image analysis system to conduct high resolution image-based phenotypic screening on multiple parameters obtained using three fluorescent markers <a href="#pone.0000163-Baghdoyan1">[5]</a>. By this means, we can not only analyze cell viability and fluorescence intensities but also cell morphology, providing invaluable information on the behavior of individual cells in the presence of a compound.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The miniaturized format of the Cell-on-Chip requires only minute amounts of media compared to current micro-well formats. This feature is particularly relevant when: a. Cell availability is limited as in the case of rare differentiated cells and patient biopsies; b. The volume of the tested compound such as potentially toxic compounds needs to be reduced in order to reduce hazards to the manipulators; c. The compounds are expensive products in such cases as candidate drugs and siRNAs. The sequential dispensing on the Cell-on-Chip device allows time for the cells to complete adhesion after cell seeding before the addition of toxicants. Toxicity can result from brief exposure to a significant amount of compound (acute toxicity) or from multiple or long term exposure to a low dose (chronic toxicity). Since we have routinely used a 5 day limit for cell culture on our Cell-on-Chip we focused our efforts on the development of a system for acute toxicity testing.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In addition, the drop reactor is a wall-free system well suited for toxicity assays compared with even small-sized microwells since there is : a) a high efficiency of gas exchange; b) continuous liquid swirling; c) no adsorption of chemicals on plastic walls, hence limited amount of toxic material remains after washing; d) unrestricted analysis of the whole assay in the absence of walls shadowing the liquid. In microfluidic channels diffusion is preponderant and limited mixing has been shown to affect cell behavior essentially in relation with the reduction in height, resembling very much the physics governing small sized wells <a href="#pone.0000163-Yu1">[6]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Stochastic variations in individual cell response to the environment can result in significant differences in the behavior of whole tissues or even organisms, in processes such as stem cell differentiation, immune response, cancer cell drug resistance, tumorigenesis and sexual behavior <a href="#pone.0000163-Eldar1">[7]</a>. With improved phenotyping and data handling techniques, we can now consider High Content Analysis (HCA) for individual cells and deduce cell population distributions potentially granting a deeper understanding of complex cell regulation systems. High Content image-based screening has been applied to high content phenotypic cell-based assays to detect nuclear translocation of proteins, receptor recycling, centrosome duplication and to identify potential new drugs modulating wound healing and mitotic arrest <a href="#pone.0000163-Wilson1">[8]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Several bioassays have been established using microorganisms genetically engineered to emit fluorescence or bioluminescence to survey environmental pollution. Similar stress inducible cell models have also been engineered in human cells <a href="#pone.0000163-Mandon1">[9]</a>–<a href="#pone.0000163-Vayssier1">[12]</a> . Heat Shock Protein (Hsp) expression is increased when environmental conditions become deleterious including heat, hypoxia, heavy metals, oxygen radicals, radiation or osmotic changes <a href="#pone.0000163-Wong1">[13]</a>. The stress-dependent <em>hsp</em> gene induction is under the control of specific regulatory sequences localized into the <em>hsp</em> gene promoter. Hsp response can be used to detect toxicants in the cellular environment by engineering cells with DNA constructs driving the expression of a reporter protein under the control of an <em>hsp</em> promoter. Previous studies have shown that the <em>Drosophila melanogaster hsp22</em> and human <em>hsp70</em> promoters can be used to detect toxic events within stable cell lines expressing the recombinant luciferase or the Enhanced Green Fluorescent Protein (EGFP) reporter genes <a href="#pone.0000163-Mandon2">[10]</a>. The liver is the main target organ for a wide range of toxic chemicals. In the framework of the TOXDROP STREP consortium (<a href="http://toxdrop.vitamib.com/">http://toxdrop.vitamib.com/</a>) <em>hsp22</em> and <em>hsp70</em>-EGFP DNA vector constructs were thus introduced in HepG2 cell line which is considered a suitable liver model for toxicity testing <a href="#pone.0000163-Mandon2">[10]</a>, <a href="#pone.0000163-Knasmuller1">[14]</a>, <a href="#pone.0000163-Fotakis1">[15]</a> and has been used for benchmarking studies <a href="#pone.0000163-Scheers1">[16]</a>.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>For our study we selected compounds for their toxic effects: the poisonous heavy metals Na arsenate NaAsO<sub>2</sub> and cadmium chloride CdCl<sub>2</sub> known to cause damage to the cells associated with reactive oxygen species (ROS), as well as the organic herbicide paraquat. Acute exposure of mammalian cells to Arsenate is a classic model of cellular stress <a href="#pone.0000163-Rossi1">[17]</a>. The liver is the major site for Cadmium accumulation and toxicity in human body <a href="#pone.0000163-Elez1">[18]</a>. Both arsenic and cadmium are hepatotoxic ROS inducers yet they can trigger different cell responses. This is exemplified by the induction in primary rat and human hepatocytes, by arsenic but not cadmium, of the expression of the multidrug resistance protein 2 (MRP2) <a href="#pone.0000163-Vernhet1">[19]</a>. Paraquat is a herbicide also known to induce oxidative stress in liver cells <a href="#pone.0000163-Dragin1">[20]</a>. Arsenate, Cadmium and paraquat exposure all cause liver damage and thus are a relevant hepatotoxicity model.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>We report here the combined use of <em>hsp</em> stress inducible HepG2 cell lines with a Cell–on-chip device to phenotype acute hepatotoxic insult using multiple endpoints in high content fashion. Our results indicate that several cell morphology parameters, along with the EGFP expression level reporting <em>Drosophila melanogaster hsp22</em> promoter activation, are earlier and more sensitive indicators of toxicity than shear cell mortality.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We recorded cytotoxic effects on HepG2 cells cultivated in 100 nL drops (<a href="#pone-0000163-g001">figure 1a</a>) exposed to NaAsO<sub>2</sub>, CdCl<sub>2</sub> and paraquat. As a proof of concept we performed an analysis a) with multiple toxicants b) on two different stress inducible HepG2 clones c) in dose-response fashion d) with quintuplicate measures e) at individual cell resolution f) monitoring multiple endpoint parameters. Analyses were performed between 0.5 µM and 1mM in order to assess a wide range of conditions progressively harmful to the cells in comparison with 0 µM untreated controls (<a href="#pone-0000163-g001">figure 1</a>). All measurements were performed in 5 replicate drops on multiple parameters. One specific toxicant was added to each block of one hundred drops. Each half block was seeded with 2–11/<em>hsp70</em> or A10/<em>hsp22</em> stress inducible clones (top half and bottom half respectively). A subset of the whole chip images series stored in Pathfinder™ image database (IMSTAR) corresponding to A10/<em>hsp22</em> stress inducible clone exposed to Arsenate is presented in <a href="#pone-0000163-g001">figure 1b</a>. A sample image of cells treated with 50 µM Arsenate is presented in <a href="#pone-0000163-g001">figure 1c</a>.</p>
<div class="figure" id="pone-0000163-g001"><div class="img"><a name="pone-0000163-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000163" data-uri="info:doi/10.1371/journal.pone.0000163.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000163.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000163.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Multiplexed toxicity assay in drops.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>The ‘Cell-on-Chip’ device was used to obtain four hundred independent HepG2 stress inducible fluorescent cell based assay measurements using two <em>hsp</em> promoter containing clones and four toxic at ten doses. The experiments were performed in quintuplicate measurements. 1a. Cell dispensing with sciFlexarrayer robot arrayer. 1b Zoom on the assembled mosaic of images corresponding to A10 clone after 6 h exposure to ten doses of Arsenate in quintuplicate (columns); the Hsp induction is monitored by the green EGFP signal, cell nucleus is stained in blue by Hoechst and cell cytoplasm is stained in red by Phalloïdin. 1c: Heterogeneity in cell response is illustrated by an example of Hsp response to 5 10<sup>−5</sup> M Arsenate exposure. Scale bar represents 500 µm. Fully automated image capture with a 10× objective and dedicated image analysis were performed using the same detection protocols by IMSTAR Pathfinder™ Cellscan system. All cells were individually segmented (contour highlighted in white) to extract information (signal intensity, morphology) on every single cell within each drop.</p>
<span>doi:10.1371/journal.pone.0000163.g001</span></div>
<h4>Visual analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A quick visual analysis revealed as shown on <a href="#pone-0000163-g001">figure 1: a</a>. a correlation of cell death with increasing concentrations of NaAsO<sub>2</sub>, CdCl<sub>2</sub> and Paraquat determined by cell counting (<a href="#pone-0000163-g001">figure 1b</a>&amp;<a href="#pone-0000163-g002">2a</a>); b. an induction of EGFP expression for NaAsO<sub>2</sub> and CdCl<sub>2 </sub>with <em>Drosophila melanogaster hsp22</em> promoter (<a href="#pone-0000163-g001">figure 1b</a>&amp;<a href="#pone-0000163-g002">2a</a>); which was c. profoundly heterogeneous (<a href="#pone-0000163-g001">Figure 1c</a>); d. the 2–11/human <em>hsp70</em> promoter clone failed to produce any significant EGFP possibly due to lower activity of the construct (data not shown). From now on the A10/<em>hsp22</em> clone response to toxic insult will be described.</p>
<div class="figure" id="pone-0000163-g002"><div class="img"><a name="pone-0000163-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000163" data-uri="info:doi/10.1371/journal.pone.0000163.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000163.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000163.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Toxicity measured as cell mortality.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Arsenate, cadmium and paraquat were tested in HepG2 A10 stress inducible clone. 2a. A sample image among the quintuplicate experiments is shown for the control and four toxic concentrations around the maximum EGFP induction zone (orange). 2b. After cell detection and cell counting cell viability is plotted versus the log scale of toxic dose with 0M control plotted as 10<sup>−7</sup> M data point. The error bars correspond to the STD of the five replicate independent experiments and are illustrating the variability of the measure. The values for IC50 (cell number) calculated by linear regression on the linear phases of the curves of two independent experiments are displayed below the graphs.</p>
<span>doi:10.1371/journal.pone.0000163.g002</span></div>

<h4>Quantification of cell mortality</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>High Content Analysis algorithms detected the number of HepG2 cells with a consistency of 88%, as checked by visual cell limit determination on sample images representing different cell densities and toxic levels, by detecting the number of Hoechst stained nuclei per spot and subsequently using the red labeled actin signal to segment cell contours. Two independent algorithms were respectively developed by IMSTAR and a group at Ecole des Mines (see <a href="#s4">material and methods</a>) both accurately contouring the cytoplasm boundary as close as possible to the actual cell limit. The results obtained with both detection tools were very similar (Data not shown).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We were able to observe an obvious decrease in cell number (<a href="#pone-0000163-g002">figure 2b</a>) in the presence of Cadmium, Arsenate and Paraquat at doses greater than 20 µM. Almost no cells survived at concentrations above 100 µM for all tested toxic compounds. The variability of the Cell Number parameter was higher than the ones obtained with other high content parameters discussed in the following chapters. This variability may be due to the combination of several minor variations such as detachment of cells during washing steps or variation in initial seeding, cell attachment and cell growth. The ratio of standard deviation vs. mean is 22% in average which remains acceptable for biological systems.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The IC50 values for the Cell Number endpoint, the concentrations corresponding to 50% of the maximum effect, were determined by linear regression on the most linear portions of the curves where toxic effect occurred for each toxicant. The same process was applied to a duplicate chip experiment, showing essentially the same cell response to toxic insult (data not shown), in an attempt to estimate the variability of IC50 measures. For ‘Cell Number’ parameter the IC50 values were: Arsenate-IC50 = 85 µM±2, CdCl2-IC50 = 83 µM±8, Paraquat-IC50 = 88 µM±3. Our results are consistent with published data. Fotakis and Timbrell <a href="#pone.0000163-Fotakis1">[15]</a> compared different cell mortality quantification methods and reported IC50 values in HepG2 cells after 3, 5, 8 and 24 h Cadmium exposure to be 300,100, 80 and 8 µM respectively for neutral red quantification and 500, 100, 40 and 15 µM respectively for MTT assay. In HepG2 cells exposed for 24 h to toxic insult CdCl<sub>2</sub> and NaAsO2 were reported to be of comparable toxicity with IC50 values of 60–70 µM for cytotoxicity. Concentrations of 15 µM CdCl<sub>2</sub> and NaAsO2, 40 µM CdCl<sub>2</sub> and 55 µM NaAsO2, 60 µM CdCl<sub>2</sub> and 70 µM NaAsO<sub>2</sub> were considered as concentrations that elicited minimal (≤5%), mild (20–25%) more severe (approximately 50%) cytotoxicity respectively <a href="#pone.0000163-Gottschalg1">[21]</a>. Mandon et al. optimized the settings for their EGFP expressing stress inducible clones (6 h toxic induction combined with a 12–18 h recovery period) and obtained an IC50 value of 50 µM for CdCl<sub>2</sub> in 96 well format assays, which is consistent with the literature <a href="#pone.0000163-Mandon2">[10]</a>. In HeLa cells engineered with the same constructs LC50 were 5 µM and 50 µM for sodium arsenate and CdCl<sub>2</sub> respectively <a href="#pone.0000163-Mandon1">[9]</a>. Since we used the same genetically engineered cells we followed the same settings.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>For all the following described parameters the measurements obtained at doses greater than 100 µM resulted in very few remaining viable cells or even cell debris and were thus excluded from further analysis.</p>


<h4>Quantification of EGFP expression</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Following arsenate and CdCl<sub>2</sub> exposures a maximum activation of <em>hsp22</em> promoter as determined by EGFP protein expression (<a href="#pone-0000163-g003">figure 3d</a>) was reached around 50 µM then disappeared at 100 µM. This was consistent with the fact that overly stressed cells are unable to initiate protein synthesis before they actually die. In contrast, paraquat did not induce any significant EGFP expression with <em>hsp22</em> promoter. We found that the mean EGFP intensity per pixel, for each cell, was a more accurate measure than total intensity per cell as suggested in other HCA studies <a href="#pone.0000163-Wilson1">[8]</a>. The independence on the size of the object might contribute to this higher reliability.</p>
<div class="figure" id="pone-0000163-g003"><div class="img"><a name="pone-0000163-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000163" data-uri="info:doi/10.1371/journal.pone.0000163.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000163.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000163.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Toxicity measured by novel high content endpoints.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The Dose response curves for a selection of High Content Analysis parameters upon Arsenate, Cadmium and Paraquat exposure are presented. 3a, Cell Area (in µ<sup>2</sup>m). 3b. shape index ( = measured cell perimeter <sup>2</sup>/4 п<sup>2</sup> R<sup>2</sup>, R is the minimum calculated radius). 3c. Roundness ( = R<sup>2</sup> п/area); 3d. EGFP-Gray Level (GL) intensity. At doses greater than 100 µM, too few cells remained to be considered for statistic analysis (grey shading). As on <a href="#pone-0000163-g002">figure 2</a> the concentration of the chemicals on x-axis is plotted in log scale and the 0 M control has been replaced by 10<sup>−7</sup> M value. For each parameter the values for IC50parameter were calculated by linear regression on the linear phases of the curves of two independent experiments and are displayed below the graphs when applicable (significant response to toxic insult, NA not applicable otherwise).</p>
<span>doi:10.1371/journal.pone.0000163.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>As the level and dynamics of EGFP expression differ between NaAsO<sub>2</sub> and CdCl<sub>2</sub>, our reporter system allowed the detection of toxic-specific features. EGFP expression could be detected at 20 µM for arsenate with a maximum at 50 µM, while a more moderate expression could be observed only at 50 µM for CdCl<sub>2</sub>. The induction of the Hsp pathway was consistent with the fact that arsenate and cadmium salts are known to cause oxidative stress <a href="#pone.0000163-Pourahmad1">[22]</a>. Arsenite is even the most efficient inducer of Hsp response in several organs <a href="#pone.0000163-DelRazo1">[23]</a>. The IC50 values for EGFP expression, corresponding to doses at which 50% of the maximum EGFP level is reached, were arsenate-IC50 = 30 µM±1.0 and CdCl<sub>2</sub>-IC50 = 32 µM±0.3. EGFP stress reporting is thus a significantly more sensitive measure than shear mortality in accordance with previous findings <a href="#pone.0000163-Mandon1">[9]</a>, <a href="#pone.0000163-Mandon2">[10]</a>. Using the same stress inducible clones in 96-well format Mandon et al reported a similar peak expression of EGFP around 50 µM CdCl<sub>2</sub> followed by a gradual decrease in promoter activity over 60 µM corresponding to the increasing cellular inability to express the reporter protein upon massive damage <a href="#pone.0000163-Mandon2">[10]</a>. The induction windows were reported around 10–100 µM for CdCl<sub>2</sub> and arsenate consistent with the EGFP inductions observed between 10 and 100 µM on the Cell-on-Chip device.</p>


<h4>Quantification of toxic dose response using morphological end-points</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The development of customized algorithms allowed the detection of individual cell contour and the use of cell morphology endpoints for toxicity reporting (<a href="#pone-0000163-g003">figure 3</a>). High content information such as morphometric parameters (Cell Area, Cell Roundness and Cell Shape Index described in <a href="#s4">Materials and Methods</a>) are obtained as individual cell characteristics calculated from the cell contour detected on Phalloïdin/F-actin signal.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>A dose-dependant decrease in ‘Cell Area’ value starting at sub-lethal low doses (below 10 µM) was observed in response to toxic insult with all compounds (<a href="#pone-0000163-g003">figure 3a</a>). The variability of this parameter was much lower than the one observed for basic cytotoxicity providing high potential for IC50 calculation. While arsenate and cadmium induced a significant cell shrinkage from around 900 to around 400 µm<sup>2</sup>, Paraquat only induced a significant yet milder decrease from 900 to 700 µm<sup>2</sup> pointing again to a different toxic mechanism. The IC50 values for Cell Area were Arsenate-IC50 = 29 µM±0.8, CdCl<sub>2</sub>-IC50 = 29 µM±1.2, and Paraquat-IC50 = 44 µM±0.3. Reduction in cytoplasmic volume has already been associated with CdCl<sub>2</sub> toxicity <a href="#pone.0000163-Romero1">[24]</a>.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>The marked cell shrinkage observed with Arsenate and Cadmium was followed by an increase in Cell Roundness at the late 100 µM dose particularly with cadmium (<a href="#pone-0000163-g003">figure 3c</a>). Cell shrinkage and cell rounding are two well-known events in cell death and more specifically the apoptotic process <a href="#pone.0000163-Thery1">[25]</a>. Apoptosis is a major mode of elimination of HepG2 cells in cadmium toxicity and it precedes necrosis <a href="#pone.0000163-Aydin1">[26]</a>. The IC50 values for ‘Cell Roundness’ were Arsenate-IC50 = 59 µM±1.6, CdCl<sub>2</sub>-IC506 = 52 µM±12. The organic paraquat again behaved differently from heavy metals and produced a less significant effect. It should be noted that cell roundness is probably not the most reliable parameter since only the 100 µM dose produced significant differences. In addition, Cell Roundness was not always consistent on a duplicate chip probably due to minor kinetic differences and differential detachment of these much altered cells during the washing steps. It provides rather qualitative information strengthening the detection of cell shrinkage to point to the occurrence of apoptotic events.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>The Cell Shape Index parameter displayed two distinct behaviors as doses increased depending on the tested toxicant (<a href="#pone-0000163-g003">figure 3b</a>). At high concentrations (above 20 µM) both heavy metal chemicals Arsenate and Cadmium caused a decrease in Shape Index. The cells were then small and round and did not present flat extension such as pods. Interestingly, as doses reached cytotoxic effect (10 µM to 100 µM), the slope of curves became steeper and consequently data points became much less variable. This could illustrate a tighter regulation of cell shape when a selective pressure is applied in the form of toxic stress. In contrast, Paraquat did not induce as significant a decrease of this index. The IC50 values for Shape Index were for Arsenate-IC50 = 24 µM±14, CdCl<sub>2</sub>-IC50 = 32 µM±17, and Paraquat-IC50 = 29 µM±25.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>The morphological effects we observed were not artifacts related to EGFP expression as the 2–11 clone that failed to express EGFP presented the same variations of morphological parameters upon stress (data not shown). In addition effects on ‘Cell Area’ began at low doses where EGFP expression could not be detected. The lower IC50 values and variability obtained for morphological parameters, along with EGFP reporting, highlight these endpoints as better indicators of toxicity than shear cell mortality.</p>


<h4>High Content analysis of Arsenate induced toxicity</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Since Arsenate was the best activator of the <em>hsp22</em> pathway in our assay, and also triggered significant morphological alterations, we performed a finer description of arsenate insult on cell behavior in High Content Analysis. On <a href="#pone-0000163-g004">figure 4</a> the potential correlation of EGFP expression, monitoring the attempt by the cell to cope with damage at mostly sub lethal doses, was studied with regards to morphological alterations. In <a href="#pone-0000163-g004">figure 4a &amp; 4b</a>, representing ‘Cell Area’ and ‘Cell Shape Index’ respectively, we observed that untreated cells presented a broad basis of x-values, meaning that some variability in cell shape and size was possible without stress. Some large and complex shaped cells co-existed with smaller and simpler cells, such as cells exiting a cell division process. As toxic doses were applied to the cells, the Hsp pathway was activated and EGFP protein was produced in coordination with an increasing restriction of morphological parameters values towards small and simple shaped cells.</p>
<div class="figure" id="pone-0000163-g004"><div class="img"><a name="pone-0000163-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000163" data-uri="info:doi/10.1371/journal.pone.0000163.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000163.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000163.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Distribution of morphological endpoints with regards to stress promoter induction.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>The distributions of 3 end-points (4a. Area; 4b. Shape Index; 4c. Roundness) were examined versus the distribution of the EGFP Grey Level intensity in <em>hsp22</em>-HepG2 bioassay cells treated with 0, 20 µM and 50 µM Arsenate. All detected cells within the five independent experiments are aggregated. In the absence of toxic (0 M; red points) around 500 detected cells indicate that cell bodies are spanned over a wide range of size, and present diverse morphologies including irregular and multi-poded (High shape index). In the presence of increasing concentration of the toxic (20 µM, orange triangles; 50 µM, green crosses), the cells tend to get smaller and to present a smaller shape index, thus a simpler morphology.</p>
<span>doi:10.1371/journal.pone.0000163.g004</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>In contrast in <a href="#pone-0000163-g004">figure 4c</a> representing Cell Roundness, the distribution on x-axis remained as spread at toxic doses as in untreated control. This fact points to a different mechanism for cell rounding uncorrelated with Hsp induction. This fits well with the fact that apoptosis related cell rounding is a later event that happens when excessive damage has occurred and cells are unable to repair the damage anymore.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The Kolmogorov-Smirnov (KS) test <a href="#pone.0000163-Young1">[27]</a> has been proposed to identify significant differences in complex parameter distributions associated with High Content Analysis <a href="#pone.0000163-Wilson1">[8]</a>, <a href="#pone.0000163-Giuliano1">[28]</a> with no <em>a priori</em> assumption on the normality of the distributions and the sample sizes. These features are critical for our toxicity study, since cell number decreases with increasing concentrations and the heterogeneity of EGFP expression (<a href="#pone-0000163-g001">figure 1d</a>.) suggests non-normal distribution. KS scores are of increasing importance as High Content Analysis studies develop. A KS score of 0.2 emerges as the threshold for significance in many studies <a href="#pone.0000163-Wilson1">[8]</a>, <a href="#pone.0000163-Giuliano2">[29]</a>. In <a href="#pone-0000163-g005">figure 5</a> we reported KS score versus toxic concentrations for several endpoints. We used the pool of control cells as a reference distribution versus the pool of cells exposed to each toxic dose. We observed that several parameters reach the 0.2 threshold with high significance as the p-value was below 0.000005. ‘Cell Area’ reached the threshold as early as 5 µM dose followed by EGFP Grey Level at 10 µM. Those two parameters are thus the most sensitive and could be used for detection of toxicity at mostly sub-lethal doses. The ‘Shape Index’ distribution was significantly different from control population only after 50 µM dose, but a change in slope occurred around 20 µM. Simplification of cell shape seems to be a later event which could fit with a secondary activation of actin/cell architecture pathways. Again ‘Cell Roundness’ was a significant parameter only as a late event specifically upon CdCl<sub>2</sub> insult (data no shown) hindering its use for blind IC50 determination on a wide range of toxicants.</p>
<div class="figure" id="pone-0000163-g005"><div class="img"><a name="pone-0000163-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000163" data-uri="info:doi/10.1371/journal.pone.0000163.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000163.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000163.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000163.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000163.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000163.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>High Content KS curves analysis.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>The Kolmogorov-Smirnov (KS) score has been calculated comparing the population distribution of all the cells at the tested Arsenate concentrations to the population distribution of all the cells in the 0 M control. KS test is independent on sample size and value scale. KS test does not presuppose any hypothesis on parameter distribution such as Normality. KS increases as the difference in distribution between the two compared populations increases (identical = 0; maximum difference = 1). KS curves have been plotted as dose-response with Arsenate dose in log scale on the <em>x</em>-axis. On the <em>y</em>-axis, the KS scores for different parameters are displayed as fitted curves. Differences are considered significant as KS is greater than 0.2 threshold and p values are indicated by stars (* = p&lt;0.005, ** = p&lt;0.00005, *** = p&lt;0.0000005).</p>
<span>doi:10.1371/journal.pone.0000163.g005</span></div><a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Inspired by IC50 calculations on KS curves proposed by Giuliano et al. <a href="#pone.0000163-Giuliano2">[29]</a>, IC50 calculations were derived from KS curves to evaluate any potential improvement brought by statistical methods accommodating ‘non-Normal’ distributions. These IC50 values for arsenate were then 32 µM, 19 µM and 19 µM for Shape Index, Cell Area and EGFP expression respectively (no significant Roundness effect). The IC50 values for CdCl2 were 48 µM, 60 µM, 25 µM and 28 µM for ‘Shape Index’, ‘Roundness’, ‘Cell Area’ and EGFP induction respectively. An enhanced sensitivity can thus be obtained for arsenate by analyzing KS curves for Cell Area (19 µM vs. 29 µM) and EGFP expression (19 µM vs. 30 µM) parameters probably in relation with non-Normal distributions of these parameters. A more marginal benefit can be observed for these two parameters with CdCl2 (25 µM vs. 29 µM and 28 µM vs. 32 µM, curve not shown). This supplementary data treatment provides some mild refinement in toxicity measures provided the monitored endpoints do not follow perfectly a Gaussian distribution.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The REACH program is a very ambitious challenge and practical innovative approaches are needed in the field of <em>in vitro</em> testing to fulfill European Community directives. Our proposal was to adapt an innovative Cell-on-Chip technology for toxicity screening of chemicals, using cells cultured within hundred nanoliter drops of culture medium. Several key goals have been achieved: a. The miniaturization of parallel cell-based assays using nanodrops for high throughput screening; b. The multiplexed screening of chemicals; e.g. anti-cancerous drugs and siRNA previously published <a href="#pone.0000163-SchaackBReboud1">[3]</a> and the broad screening of 10 concentrations in quintuplicate experiments in the present study; c. The sequential spotting during 5 days and automated chip scanning and smart image captures using a metallic mesh embedded on the glass slide for reproducible positioning; d. The High Content Analysis and data management enabled by the Pathfinder™ system; e. The construction of a cheap and simple glass slide substrate chosen for the Cell-on-Chip device.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>To enhance the chances of success of the novel Cell-on-Chip format, we have improved several key aspects. The hydrophobic surface of the chip was cleaned using strong acids, to allow good attachment of the HepG2 cells. Quality control measurements on the cells were recorded before the experiment using a bench cell analyzer sorter (Guava technology). We believe that quality control values (i.e. cell viability) should be stored in a knowledge database for further results exploitation of cell-based assay experiment. Through the combination of a miniaturized format of Cell-on-Chip and specifically adapted IMSTAR Pathfinder™ automated HCS platform, we may provide novel detection end-points for toxicity screening. The reliability of the measured morphological description of individual cells is provided by complex algorithms embedded in software modules of the HCS system and the combined use of several toxicity endpoints.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Some attempts have been made to adapt high conten/high throughput cell based assays to toxicity or hepatotoxicity <a href="#pone.0000163-Tencza1">[30]</a>. Most of these assays have been performed in 96-well or other microwell plates with bigger sample volumes and simpler readouts than individual cell morphological endpoints. In particular, efforts have been made to improve cells models for <em>in vitro</em> toxicity testing. In the context of rapid environmental toxicity testing, bioassays based on bioluminescent bacteria have been developed with the main advantage of quickness, portability and low cost <a href="#pone.0000163-Lee1">[31]</a>. Yet, single celled prokaryotes necessarily represent toxicity neither in mammalian cells <em>in vitro</em> nor in whole organism <em>in vivo</em>. The liver slices and primary cell cultures are limited by supply, limited lifespan and individual variability making them poorly compatible with regulatory-accepted large-scale assays. In addition, stable EGFP expressing stress inducible clones cannot be obtained in primary cells. Some attempts have been made at using rat primary hepatocytes in miniaturized assay <a href="#pone.0000163-Flaim1">[32]</a>, which are potentially more representative of liver <em>in vivo</em>. However, the smaller size of the 150 µm diameter seeded surface might become too limited to allow the analysis of a statistically relevant number of cells when the cell viability is seriously impaired by toxic insult. In addition rat primary hepatocytes <em>in vitro</em> do not necessarily mimic perfectly human liver <em>in vivo</em>. Chips bearing cells embedded in gel containing liver specific CYP450 family detoxification enzymes have been generated to mimic the bio transformation activities found in liver <em>in vivo</em> 
            <a href="#pone.0000163-Lee2">[33]</a>. However, the dynamic regulation of these enzymes occurring in living cells is not accounted for and the use of MCF7 breast cancer cell line might not necessarily adequately represent the behavior of hepatic cells.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We can evaluate the validity and usefulness of our assay by looking back at the modes of actions of arsenate, cadmium and paraquat. More particularly we highlight the functional information brought by <em>hsp</em> reporting. Heat shock proteins stimulate cellular resistance to different types of stresses including heat shock, oxidative stress and the cytotoxicity induced by drugs and apoptotic agents <a href="#pone.0000163-Arrigo1">[34]</a>. Cells can recover from exposure to sub lethal dose by triggering cell stress response by such pathways as the synthesis of Hsp proteins, which act as chaperones maintaining protein conformation <a href="#pone.0000163-AitAissa1">[11]</a>, <a href="#pone.0000163-AitAissa2">[35]</a>. Beyond critical damage the cells rather orient towards necrosis or apoptosis <a href="#pone.0000163-Arrigo1">[34]</a>. Acute exposure to high doses of cadmium results in major liver damage via hepatocyte necrosis <a href="#pone.0000163-Dudley1">[36]</a>. Both arsenic and cadmium are hepatotoxic, with generation of reactive oxygen species (ROS) as a main cause for cytotoxicity <a href="#pone.0000163-Pourahmad1">[22]</a>. Still they can trigger different cell responses in hepatocytes as exemplified by multidrug resistance protein 2 MRP2 expression induction by arsenic but not cadmium in primary rat and human hepatocytes <a href="#pone.0000163-Vernhet1">[19]</a>. In our study, CdCl<sub>2</sub> and arsenate present accordingly slightly different <em>hsp</em> promoter activation dynamics. Differences between heavy metals and Paraquat could be isolated and fitted existing information. Indeed non redox transition metals AsIII and CdII induce ROS damage with mitochondria as main sites <a href="#pone.0000163-Pourahmad1">[22]</a> while the organic compound Paraquat and redox transition metals also have been reported to induce ROS damage but mostly via lysosomal pathway <a href="#pone.0000163-Kondoh1">[37]</a>. Interestingly, the expression of some Hsp family members has been shown to be induced at non cytotoxic doses with heavy metals while being activated at cytotoxic doses with organic compounds illustrating different mechanisms of induction between heavy metals and organic compounds <a href="#pone.0000163-AitAissa2">[35]</a>. In our study the organic compound Paraquat did not trigger significant EGFP expression reporting <em>hsp22</em> promoter activation compared with Cd and As non redox metals, illustrating toxic specific sensitivity of our bioassay system. The use of EGFP stress inducible clones allows the simple quantification of toxic insult that can be directly analyzed on chip. Intracellular EGFP synthesis could help eliminate several costly steps of cell labeling and washing procedures and consequently diminish the associated handling related experimental variability. Assays using stress inducible clones could be useful to toxicologists implementing fully automated toxicity assays in regulatory compliance and standard operating procedures (SOP) such as the neutral red uptake assay. Although bioassay systems are dependent on the engineering of cell lines, constructs are relatively straightforward to generate and can be largely shared within the scientific community.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In our study tested compounds were all ROS inducers and caused morphological alterations. Effects of ROS on cell morphology have been linked to alterations in Ras pathway modulating cell architecture related proteins such as Rac1 and actin-binding protein Rho1-cofilin <a href="#pone.0000163-Alexandrova1">[38]</a>. Whole cell morphology has not been used to date as a toxicity endpoint in high throughput screens. In another HCA study a decreased nuclear size has been monitored and associated with mitotic arrest <a href="#pone.0000163-Wilson1">[8]</a>. In addition a perinuclear cytoplasmic ring has been additionally used to represent the whole cell upon threat agent exposure <a href="#pone.0000163-Tencza1">[30]</a> but precise quantification of individual cell morphological features for toxicity phenotyping was not reported. In the light of existing methods, our study represents a unique combination of bioassay with high content multiparametric analysis capacity in a pertinent model of human hepatotoxicity. Our study is also one of the few studies to show the significance of morphological parameters to monitor sub lethal stress and it is the only one extending these findings to the analysis of whole cell morphology in high throughput fashion.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In our experiments a relatively high level of variability in individual cell response to toxic insult could be observed in particular for EGFP induction. Variability in individual cell behaviour can be related to multiple parameters such as cell cycle status and crosstalk with neighbouring cells but also stochastic repartition of very rare regulatory proteins between sister cells or local events involving the binding of rare signalling molecules. The use of artificially synchronized cells to ease cell based assays analysis might be to some extent valid for initial (Hit to Lead) drug screening. However such an artificial process probably poorly reproduces the complexity of cell response <em>in vivo</em> and thus probably might not fit toxicity assays. Tencza et al <a href="#pone.0000163-Tencza1">[30]</a> studied the nucleus and a perinuclear cytoplasmic ring as representative of the whole cell and reported heterogeneity in cell response with a small percentage of cells responding vigorously to a toxin consistent with our findings. The development of methods allowing the identification, enrichment and isolation of high responder subpopulations or cells in specific cell cycle status could further improve the sensitivity of cytotoxicity assays.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>We accessed information similar to that of flow cytometry experiments but without trypsinization procedures potentially can affect cell morphology and without the need for multiple sample injections. The combined use of the multiple novel endpoints shows a huge potential for use in toxicology. Information on morphology combined with the induction of specific cell pathways via the use of stress inducible cell lines is leading to a more accurate evaluation of toxicity than cell mortality alone, but also to accurate clustering of families of toxicants. Moreover this functional information could be integrated to QSAR models. With High Content Analysis tools, it would be very interesting in the future to correlate variation in parameter distribution with heterogeneity in cell cycle status, cell membrane composition or particular gene and miRNA expression.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Assays could be developed further to record kinetic evolution of the response to toxic insult. The potential of morphological parameters to detect early toxicity as highlighted in our study could be combined with live dyes (Hoechst, Cell Trackers, etc…) or label free phase contrast imaging methods for high quality time resolved/time lapse toxicity screens.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>This adaptation of Cell-on-Chip technology to acute <em>in vitro</em> hepatotoxicity testing allows the measurement of innovative and sensitive toxicology endpoints such as EGFP and morphological parameters. This appears promising to facilitate the REACH program. Beyond this strict hazard management focus our hepatotoxicity assay could prove very valuable as an early decision tool for ADMETOX studies in pharmaceutics since hepatotoxicity is a major bottleneck in drug development leading to frequent candidate drug failure.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cell lines and culture</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The HepG2 cell line was isolated from hepatocellular carcinoma. HepG2 cells were grown in IMDM Iscove modified Dulbecco's medium + Glutamax I from Invitrogen/GibcoBRL (Cergy Pontoise, France) supplemented with 10% foetal calf serum, from Dominique Dutscher (Brumath, France) 1 mg/ml Fungizone, 50 u/ml Penicillin/Streptomycin and 500 µg/ml Geneticin from Invitrogen to maintain the selection of integrated transgenic plasmid.</p>


<h4>Hsp70 and Hsp22 promoter controlled EGFP Plasmids</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Two DNA promoters of the heat shock protein family were fused upstream of an <em>Enhanced-Green Fluorescent protein</em> (EGFP) reporter gene. The pG-ph70-EGFP-neo and the pG-pd22-<sup>−</sup>400-EGFP-neo reporter vectors contain the human <em>hsp70</em> and <em>Drosophila melanogaster hsp22</em> promoters respectively and were derived from the previously described pG-EGFP-neo host vector, containing the EGFP reporter gene and the neomycin phosphotransferase gene for selection of stably transfected clones <a href="#pone.0000163-Mandon2">[10]</a>. The 2–11 and A10 Clones were selected as highly active clones for these respective constructs monitoring EGFP expression upon the control of heat shock or hyperoxyde stimulation prior to the experiments on chip.</p>


<h4>Culture on chip and cell behavior on chip</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The culture conditions on chip were set so as to obtain approximately 100 cells at fixation time (day4) in the non-exposed cultures, enough to provide statistical analysis significance while avoiding growth inhibition and stress due to excessive cell density.</p>


<h4>Wafer substrates</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The cell culture device was manufactured in compliance with microelectronic fabrication specifications by MEMSCAP (Crolles, France). A molecular monolayer (a few nanometers' thick) of hydrophobic perfluoro-octyl-silane (FDTS) was deposited in a patterned fashion onto a hydrophilic glass layer to form the “Cell-on-Chip” device described elsewhere <a href="#pone.0000163-SchaackBReboud1">[3]</a>. The spot pattern was complemented by a 200 µm wide metallic mesh on the slides for accurate positioning of the automatic microscope with Pathfinder software (Imstar, France).</p>


<h4>Spot preparation</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The hydrophilic/hydrophobic interface allows a contact angle greater than 100° hence the formation of 100 nanoliter drops as cell culture incubators on 500 µm hydrophilic spots. The drop volume resulted from the addition of a controlled number of 500 picoliter droplets. Before cell spotting the slide was rinsed in ultra pure water before surface cleansing with a 20% nitric acid bath under agitation for 20 minutes. The acid was washed with ultra pure water. Then the array was bathed and sterilized for 20 minutes in a 70% ethanol solution, before being air-dried</p>


<h4>Cell preparation and dilution</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>HepG2 cells were trypsinized, homogenized and filtered on a 70 µm Cell Strainer sterile filter from BD Falcon/Dominique Dutscher to prevent nozzle blocking by cell aggregates. The cell suspension was analyzed with a Guava EasyCyte base system from Guava Technologies (Hayward, California, <span class="underline">US</span>) to accurately measure cell density and cell viability via ViaCount kit. The cells were then diluted at 8.10<sup>5</sup> cells/ml and placed in 96-well storage plates from Fisher Scientific Labosi (Elancourt, France) for robot dispensing.</p>


<h4>Cell dispense</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Cells were spotted using the sciFLEXARRAYER robot arrayer from Scienion AG (Berlin, Germany) as described previously <a href="#pone.0000163-SchaackBReboud1">[3]</a>. The HepG2 cell line was dispensed using a 70 µm piezoelectric nozzle. The spotting procedure was conducted under vapor-saturated conditions maintaining the robot atmosphere slightly over the dew point to prevent evaporation of the drops with temperature controlled via a cooling block. The drop ejection speed of 1ms<sup>−1</sup> has been shown to be compatible with good cell culture <a href="#pone.0000163-SchaackBReboud1">[3]</a>
            </p>


<h4>Cell culture of seeded device</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>After dispensing the cell culture drops, the device was set onto a PBS bed as it was reintroduced in the cell culture incubator to ensure smooth warming of the biochip. The cells were left for 48 hours to ensure adhesion to the substrate.</p>


<h4>Toxic compound dilutions and spotting map</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>A set of 3 “proof of concept” compounds was selected: Sodium arsenate (NaAsO<sub>2</sub>), Cadmium chloride (CdCl<sub>2</sub>) and Paraquat from Sigma-Aldrich (St Quentin Fallavier, France). In addition to untreated controls, doses ranging from 5.10<sup>−7</sup> M to 10<sup>−3</sup> M were established for the spotted drops by adding the adequate amount of solutions concentrated 10-fold to the cell culture drops. It should be noted that less than 3 micro moles in total of each compound needed to be handled by the manipulator on the arrayer platform.</p>


<h4>Induction timing and assay optimization</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>Mandon et al. observed a strong induction after 6 hours of toxic substance exposure with the same <em>hsp22</em> and <em>hsp70</em> promoters coupled to the luciferase reporter gene as well as with EGFP reporter constructs <a href="#pone.0000163-Mandon1">[9]</a>, <a href="#pone.0000163-Mandon2">[10]</a>. In our first experiment cells were exposed to the toxic compounds for 1 hour, 6 hours and continuously until the fourth day, with no significant EGFP induction distinguishable after 1 hour treatment (data not shown).</p>
<a id="article1.body1.sec4.sec10.p2" name="article1.body1.sec4.sec10.p2"></a><p>As Mandon et al <a href="#pone.0000163-Mandon1">[9]</a>, <a href="#pone.0000163-Mandon2">[10]</a> pointed out that excessive damage to the cell could prevent the start-up of cell machinery to synthesize the EGFP reporter gene product, we have established a recovery procedure, so cells can recover from the toxicant-induced stress and synthesize EGFP protein. After a 6 hours incubation period with the toxicants, the chip was washed 3 times in PBS solution then placed into a culture medium to allow EGFP expression. Incubation of the culture was resumed until Day4 arrest. However, the biochip no longer supported drops: the cell culture spots were submerged in culture medium.</p>


<h4>Fixation and labeling</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>All 400 Cell-on-Chip spots were fixed with 4% Paraformaldehyde solution from Sigma Aldrich Fluka (St Quentin Fallavier, France) spread over the entire seeded surface for 30 minutes, then stained using Hoechst at 1/6000 dilution and Alexa Fluor 546 Phalloïdin at 1/50 dilution from Molecular Probes Europe BV (Leiden, the Netherlands). The chip was again rinsed twice in PBS prior to mounting in DakoCytomation Fluorescent mounting reagent from DAKO (Trappes, France).</p>


<h4>Automated imaging</h4>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1"></a><p>Using the metallic mesh embedded in the chip for precise positioning <a href="#pone.0000163-SchaackBReboud1">[3]</a>, we performed automated smart capture using IMSTAR (Paris, France) Pathfinder™ system. The Pathfinder™ imaging platform enables fully automated capture of the whole chip with intelligent image-data management (IDB, patent n° 01921459.2) for each spot at 0.6µm resolution compatible with multispectral fluorescence single cell detection and multiparametric cellular characterization. This multiplex capability of the platform makes it a good tool for complex phenotyping in large scale toxicology.</p>


<h4>Image Analysis in Pathfinder™</h4>
<a id="article1.body1.sec4.sec13.p1" name="article1.body1.sec4.sec13.p1"></a><p>After image capture, all cells within each spot were automatically segmented thanks to detection protocols integrated in Pathfinder™ system. The main difficulties for such a segmentation using a unique detection protocol, crucial for comparative studies, are: a. the cell shape variability between cells in each drop potentially depends on toxic compound concentration; b. Phalloïdin labeling is highly inhomogeneous and not limited to the cell membrane; c. there is presence of touching cells, barely separable by eye observation. The core of the detection procedure has been published elsewhere <a href="#pone.0000163-Baghdoyan1">[5]</a> and specifically adapted for HepG2 cells in nanodrops. Basically, the detection involves automated smart thresholds taking into account cells neighborhood, mathematical morphology (watershed-based) segmentation, and several sorting filters in order to discard labeling artifacts and then to reject the cells overlapping too much. Then we obtain an accurate contour detection, which is crucial for morphology characterization.</p>
<a id="article1.body1.sec4.sec13.p2" name="article1.body1.sec4.sec13.p2"></a><p>Subsequent automated analysis integrated in Pathfinder system consists in providing all parameters (Cell area, Roundness, Shape Index, EGFP fluorescence intensity, etc) for each individual cell as well as for each drop.</p>


<h4>Alternative image analysis</h4>
<a id="article1.body1.sec4.sec14.p1" name="article1.body1.sec4.sec14.p1"></a><p>Alternatively, images captured by the Pathfinder™ system were exported and processed by a software developed at Ecole des Mines de Paris, Centre de Morphologie Mathématique to allow the plotting of the distribution of cell populations aggregated by replicate measure points. A customized HepG2 bioassay dedicated segmentation tool was implemented which defines an image mask with the contours of each cell. The approach is based on the application of mathematical morphology techniques, i.e., connected filters and watershed segmentation for the three fluorescence markers <a href="#pone.0000163-Meyer1">[39]</a>, <a href="#pone.0000163-Meyer2">[40]</a>. A tutorial on the main operators of mathematical morphology can be found in <a href="#pone.0000163-Serra1">[41]</a>. For instance these techniques have recently been used to segment cDNA microarray images <a href="#pone.0000163-Angulo1">[42]</a>. The inhomogeneous fluorescence background was removed from the three images using the <em>top-hat transformation</em>. After filtering the structures by <em>area and contrast criterion filters</em>, the Dapi channel allowed the detection of a marker (Hoechst) for each nucleus. Then, from a combination of the Rhod (Phalloïdin) and the Fitc (EGFP) channels, it was computed, on the one hand, an <em>image gradient</em> which described the energy of cytoplasm contours and, on the other hand, a <em>binary image</em> defining the external markers of the cells (and clusters of cells). Before that, the Phalloïdin and the EGFP images were <em>leveled</em> using again several criteria of size and contrast; moreover, the final gradient was obtained by adding several <em>multi-scale gradients</em>. When the gradient and the inner/outer markers were defined, the <em>watershed transformation</em> computed the contours of each individual cell. The values for the parameters of filters were adaptively computed for each cell image by means of a pre-processing step (<em>granulometries</em> according to the different parameters) in order to have a fully automatic algorithm.</p>


<h4>Informative parameters description</h4>
<a id="article1.body1.sec4.sec15.p1" name="article1.body1.sec4.sec15.p1"></a><p>Cell Area (in µ<sup>2</sup>m), Cell. Shape Index ( = measured cell perimeter <sup>2</sup>/4 Π<sup>2</sup> R<sup>2</sup>, R is the radius of the calculated circumscribed circle) and Roundness ( = R<sup>2</sup> Π/area) morphological parameters were found to be following dose response profiles upon toxic exposure to various extent. The EGFP-Gray Level (GL) intensity corresponds to the mean level on Fitc channel by pixel per individual cell showed peak induction upon toxic insult.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>Authors thank for their contribution to this work: Pierre Yves Mitha, Laureline Koutelier, Violaine Chapuis, Sarah Harper and Lamya Ghenim-Ziman.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: FL BS CM JR HP FC AA. Performed the experiments: FL. Analyzed the data: FL AP JA CL. Contributed reagents/materials/analysis tools: FL CM JR AP JA CD CL. Wrote the paper: FL BS. Other: Consortium management, Cell-on-Chip projects leader within CEA Biopuces team: BS. Platform management: HP. Biopuces Team Leader: FC. CNRS Team Leader: AA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000163-Balls1" id="pone.0000163-Balls1"></a>Balls M, Goldberg AM, Fentem JH, Broadhead CL, Burch RL, et al.  (1995) The three Rs: the way forward: the report and recommendations of ECVAM Workshop 11. Altern Lab Anim  23: 838–866.  <ul class="find" data-citedArticleID="926518"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+three+Rs%3A+the+way+forward%3A+the+report+and+recommendations+of+ECVAM+Workshop+11.&amp;auth=&amp;atitle=The+three+Rs%3A+the+way+forward%3A+the+report+and+recommendations+of+ECVAM+Workshop+11." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+three+Rs%3A+the+way+forward%3A+the+report+and+recommendations+of+ECVAM+Workshop+11." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+three+Rs%3A+the+way+forward%3A+the+report+and+recommendations+of+ECVAM+Workshop+11.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000163-David1" id="pone.0000163-David1"></a>David CA, Middleton T, Montgomery D, Lim HB, Kati W, et al.  (2002) Microarray compound screening (microARCS) to identify inhibitors of HIV integrase. J Biomol Screen  7: 259–266.  <ul class="find" data-citedArticleID="926520" data-doi="10.1089/108705702760047754"><li><a href="http://dx.doi.org/10.1089/108705702760047754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+compound+screening+%28microARCS%29+to+identify+inhibitors+of+HIV+integrase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+compound+screening+%28microARCS%29+to+identify+inhibitors+of+HIV+integrase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000163-SchaackBReboud1" id="pone.0000163-SchaackBReboud1"></a>Schaack B Reboud J, Combe S, Fouqué B, Berger F, Boccard S, Filhol-Cochet O, Chatelain F (2005) A ‘drop-chip’ cell array for high throughput DNA and siRNA transfection combined with drug screening. NanoBiotechnology  1: 166–194.  <ul class="find" data-citedArticleID="926564" data-doi="10.1385/nbt:1:2:183"><li><a href="http://dx.doi.org/10.1385/nbt:1:2:183" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+%26%23x2018%3Bdrop-chip%26%23x2019%3B+cell+array+for+high+throughput+DNA+and+siRNA+transfection+combined+with+drug+screening." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+%26%23x2018%3Bdrop-chip%26%23x2019%3B+cell+array+for+high+throughput+DNA+and+siRNA+transfection+combined+with+drug+screening.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000163-Selinger1" id="pone.0000163-Selinger1"></a>Selinger DW, Wright MA, Church GM (2003) On the complete determination of biological systems. Trends Biotechnol  21: 251–254.  <ul class="find" data-citedArticleID="926568" data-doi="10.1016/s0167-7799(03)00113-6"><li><a href="http://dx.doi.org/10.1016/s0167-7799(03)00113-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+complete+determination+of+biological+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+complete+determination+of+biological+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000163-Baghdoyan1" id="pone.0000163-Baghdoyan1"></a>Baghdoyan S, Roupioz Y, Pitaval A, Castel D, Khomyakova E, et al.  (2004) Quantitative analysis of highly parallel transfection in cell microarrays. Nucleic Acids Res  32: e77.  <ul class="find" data-citedArticleID="926516" data-doi="10.1093/nar/gnh074"><li><a href="http://dx.doi.org/10.1093/nar/gnh074" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+analysis+of+highly+parallel+transfection+in+cell+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+analysis+of+highly+parallel+transfection+in+cell+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000163-Yu1" id="pone.0000163-Yu1"></a>Yu H, Meyvantsson I, Shkel IA, Beebe DJ (2005) Diffusion dependent cell behavior in microenvironments. Lab Chip  5: 1089–1095.  <ul class="find" data-citedArticleID="926586" data-doi="10.1039/b504403k"><li><a href="http://dx.doi.org/10.1039/b504403k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diffusion+dependent+cell+behavior+in+microenvironments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diffusion+dependent+cell+behavior+in+microenvironments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000163-Eldar1" id="pone.0000163-Eldar1"></a>Eldar A, Elowitz M (2005) Systems biology: deviations in mating. Nature  437: 631–632.  <ul class="find" data-citedArticleID="926528" data-doi="10.1038/437631a"><li><a href="http://dx.doi.org/10.1038/437631a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systems+biology%3A+deviations+in+mating." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systems+biology%3A+deviations+in+mating.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000163-Wilson1" id="pone.0000163-Wilson1"></a>Wilson CJ, Si Y, Thompsons CM, Smellie A, Ashwell MA, et al.  (2006) Identification of a small molecule that induces mitotic arrest using a simplified high-content screening assay and data analysis method. J Biomol Screen  11: 21–28.  <ul class="find" data-citedArticleID="926580" data-doi="10.1177/1087057105280726"><li><a href="http://dx.doi.org/10.1177/1087057105280726" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+small+molecule+that+induces+mitotic+arrest+using+a+simplified+high-content+screening+assay+and+data+analysis+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+small+molecule+that+induces+mitotic+arrest+using+a+simplified+high-content+screening+assay+and+data+analysis+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000163-Mandon1" id="pone.0000163-Mandon1"></a>Mandon CA, Diaz C, Arrigo AP, Blum LJ (2005) Chemical stress sensitive luminescent human cells: molecular biology approach using inducible Drosophila melanogaster hsp22 promoter. Biochem Biophys Res Commun  335: 536–544.  <ul class="find" data-citedArticleID="926550" data-doi="10.1016/j.bbrc.2005.07.112"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2005.07.112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemical+stress+sensitive+luminescent+human+cells%3A+molecular+biology+approach+using+inducible+Drosophila+melanogaster+hsp22+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemical+stress+sensitive+luminescent+human+cells%3A+molecular+biology+approach+using+inducible+Drosophila+melanogaster+hsp22+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000163-Mandon2" id="pone.0000163-Mandon2"></a>Mandon CA, Diaz-Latoud C, Arrigo AP, Blum LJ (2006) Dithiocarbamate fungicide thiram detection: comparison of bioluminescent and fluorescent whole-cell bioassays based on hsp22 stress promoter induction. J Biotechnol  124: 392–402.  <ul class="find" data-citedArticleID="926552" data-doi="10.1016/j.jbiotec.2006.01.004"><li><a href="http://dx.doi.org/10.1016/j.jbiotec.2006.01.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dithiocarbamate+fungicide+thiram+detection%3A+comparison+of+bioluminescent+and+fluorescent+whole-cell+bioassays+based+on+hsp22+stress+promoter+induction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dithiocarbamate+fungicide+thiram+detection%3A+comparison+of+bioluminescent+and+fluorescent+whole-cell+bioassays+based+on+hsp22+stress+promoter+induction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000163-AitAissa1" id="pone.0000163-AitAissa1"></a>Ait-Aissa S, Pandard P, Magaud H, Arrigo AP, Thybaud E, et al.  (2003) Evaluation of an in vitro hsp70 induction test for toxicity assessment of complex mixtures: comparison with chemical analyses and ecotoxicity tests. Ecotoxicol Environ Saf  54: 92–104.  <ul class="find" data-citedArticleID="926504" data-doi="10.1016/s0147-6513(02)00026-x"><li><a href="http://dx.doi.org/10.1016/s0147-6513(02)00026-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+an+in+vitro+hsp70+induction+test+for+toxicity+assessment+of+complex+mixtures%3A+comparison+with+chemical+analyses+and+ecotoxicity+tests." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+an+in+vitro+hsp70+induction+test+for+toxicity+assessment+of+complex+mixtures%3A+comparison+with+chemical+analyses+and+ecotoxicity+tests.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000163-Vayssier1" id="pone.0000163-Vayssier1"></a>Vayssier M, Favatier F, Pinot F, Bachelet M, Polla BS (1998) Tobacco smoke induces coordinate activation of HSF and inhibition of NFkappaB in human monocytes: effects on TNFalpha release. Biochem Biophys Res Commun  252: 249–256.  <ul class="find" data-citedArticleID="926576" data-doi="10.1006/bbrc.1998.9586"><li><a href="http://dx.doi.org/10.1006/bbrc.1998.9586" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tobacco+smoke+induces+coordinate+activation+of+HSF+and+inhibition+of+NFkappaB+in+human+monocytes%3A+effects+on+TNFalpha+release." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tobacco+smoke+induces+coordinate+activation+of+HSF+and+inhibition+of+NFkappaB+in+human+monocytes%3A+effects+on+TNFalpha+release.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000163-Wong1" id="pone.0000163-Wong1"></a>Wong HR (1999) Heat shock proteins. Facts, thoughts, and dreams. A. De Maio. Shock 11:1–12, 1999. Shock  12: 323–325.  <ul class="find" data-citedArticleID="926582" data-doi="10.1097/00024382-199910000-00012"><li><a href="http://dx.doi.org/10.1097/00024382-199910000-00012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heat+shock+proteins.+Facts%2C+thoughts%2C+and+dreams.+A.+De+Maio.+Shock+11%3A1%26%23x2013%3B12%2C+1999." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heat+shock+proteins.+Facts%2C+thoughts%2C+and+dreams.+A.+De+Maio.+Shock+11%3A1%26%23x2013%3B12%2C+1999.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000163-Knasmuller1" id="pone.0000163-Knasmuller1"></a>Knasmuller S, Mersch-Sundermann V, Kevekordes S, Darroudi F, Huber WW, et al.  (2004) Use of human-derived liver cell lines for the detection of environmental and dietary genotoxicants; current state of knowledge. Toxicology  198: 315–328.  <ul class="find" data-citedArticleID="926542" data-doi="10.1016/j.tox.2004.02.008"><li><a href="http://dx.doi.org/10.1016/j.tox.2004.02.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+human-derived+liver+cell+lines+for+the+detection+of+environmental+and+dietary+genotoxicants%3B+current+state+of+knowledge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+human-derived+liver+cell+lines+for+the+detection+of+environmental+and+dietary+genotoxicants%3B+current+state+of+knowledge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000163-Fotakis1" id="pone.0000163-Fotakis1"></a>Fotakis G, Timbrell JA (2006) In vitro cytotoxicity assays: comparison of LDH, neutral red, MTT and protein assay in hepatoma cell lines following exposure to cadmium chloride. Toxicol Lett  160: 171–177.  <ul class="find" data-citedArticleID="926534" data-doi="10.1016/j.toxlet.2005.07.001"><li><a href="http://dx.doi.org/10.1016/j.toxlet.2005.07.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+cytotoxicity+assays%3A+comparison+of+LDH%2C+neutral+red%2C+MTT+and+protein+assay+in+hepatoma+cell+lines+following+exposure+to+cadmium+chloride." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+cytotoxicity+assays%3A+comparison+of+LDH%2C+neutral+red%2C+MTT+and+protein+assay+in+hepatoma+cell+lines+following+exposure+to+cadmium+chloride.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000163-Scheers1" id="pone.0000163-Scheers1"></a>Scheers EM, Ekwall B, Dierickx PJ (2001) In vitro long-term cytotoxicity testing of 27 MEIC chemicals on Hep G2 cells and comparison with acute human toxicity data. Toxicol In Vitro  15: 153–161.  <ul class="find" data-citedArticleID="926566" data-doi="10.1016/s0887-2333(00)00062-x"><li><a href="http://dx.doi.org/10.1016/s0887-2333(00)00062-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+long-term+cytotoxicity+testing+of+27+MEIC+chemicals+on+Hep+G2+cells+and+comparison+with+acute+human+toxicity+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+long-term+cytotoxicity+testing+of+27+MEIC+chemicals+on+Hep+G2+cells+and+comparison+with+acute+human+toxicity+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000163-Rossi1" id="pone.0000163-Rossi1"></a>Rossi MR, Somji S, Garrett SH, Sens MA, Nath J, et al.  (2002) Expression of hsp 27, hsp 60, hsc 70, and hsp 70 stress response genes in cultured human urothelial cells (UROtsa) exposed to lethal and sublethal concentrations of sodium arsenite. Environ Health Perspect  110: 1225–1232.  <ul class="find" data-citedArticleID="926562" data-doi="10.1289/ehp.021101225"><li><a href="http://dx.doi.org/10.1289/ehp.021101225" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+hsp+27%2C+hsp+60%2C+hsc+70%2C+and+hsp+70+stress+response+genes+in+cultured+human+urothelial+cells+%28UROtsa%29+exposed+to+lethal+and+sublethal+concentrations+of+sodium+arsenite." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+hsp+27%2C+hsp+60%2C+hsc+70%2C+and+hsp+70+stress+response+genes+in+cultured+human+urothelial+cells+%28UROtsa%29+exposed+to+lethal+and+sublethal+concentrations+of+sodium+arsenite.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000163-Elez1" id="pone.0000163-Elez1"></a>Elez D, Dundjerski J, Matic G (2001) Cadmium affects the redox state of rat liver glucocorticoid receptor. Cell Biol Toxicol  17: 169–177.  <ul class="find" data-citedArticleID="926530" data-doi="10.1023/a:1011940414419"><li><a href="http://dx.doi.org/10.1023/a:1011940414419" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cadmium+affects+the+redox+state+of+rat+liver+glucocorticoid+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cadmium+affects+the+redox+state+of+rat+liver+glucocorticoid+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000163-Vernhet1" id="pone.0000163-Vernhet1"></a>Vernhet L, Seite MP, Allain N, Guillouzo A, Fardel O (2001) Arsenic induces expression of the multidrug resistance-associated protein 2 (MRP2) gene in primary rat and human hepatocytes. J Pharmacol Exp Ther  298: 234–239.  <ul class="find" data-citedArticleID="926578"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Arsenic+induces+expression+of+the+multidrug+resistance-associated+protein+2+%28MRP2%29+gene+in+primary+rat+and+human+hepatocytes.&amp;auth=&amp;atitle=Arsenic+induces+expression+of+the+multidrug+resistance-associated+protein+2+%28MRP2%29+gene+in+primary+rat+and+human+hepatocytes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arsenic+induces+expression+of+the+multidrug+resistance-associated+protein+2+%28MRP2%29+gene+in+primary+rat+and+human+hepatocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arsenic+induces+expression+of+the+multidrug+resistance-associated+protein+2+%28MRP2%29+gene+in+primary+rat+and+human+hepatocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000163-Dragin1" id="pone.0000163-Dragin1"></a>Dragin N, Smani M, Arnaud-Dabernat S, Dubost C, Moranvillier I, et al.  (2006) Acute oxidative stress is associated with cell proliferation in the mouse liver. FEBS Lett  580: 3845–3852.  <ul class="find" data-citedArticleID="926524" data-doi="10.1016/j.febslet.2006.06.006"><li><a href="http://dx.doi.org/10.1016/j.febslet.2006.06.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acute+oxidative+stress+is+associated+with+cell+proliferation+in+the+mouse+liver." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acute+oxidative+stress+is+associated+with+cell+proliferation+in+the+mouse+liver.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000163-Gottschalg1" id="pone.0000163-Gottschalg1"></a>Gottschalg E, Moore NE, Ryan AK, Travis LC, Waller RC, et al.  (2006) Phenotypic anchoring of arsenic and cadmium toxicity in three hepatic-related cell systems reveals compound- and cell-specific selective up-regulation of stress protein expression: Implications for fingerprint profiling of cytotoxicity. Chem Biol Interact..   <ul class="find" data-citedArticleID="926540" data-doi="10.1016/j.cbi.2006.04.003"><li><a href="http://dx.doi.org/10.1016/j.cbi.2006.04.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypic+anchoring+of+arsenic+and+cadmium+toxicity+in+three+hepatic-related+cell+systems+reveals+compound-+and+cell-specific+selective+up-regulation+of+stress+protein+expression%3A+Implications+for+fingerprint+profiling+of+cytotoxicity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypic+anchoring+of+arsenic+and+cadmium+toxicity+in+three+hepatic-related+cell+systems+reveals+compound-+and+cell-specific+selective+up-regulation+of+stress+protein+expression%3A+Implications+for+fingerprint+profiling+of+cytotoxicity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000163-Pourahmad1" id="pone.0000163-Pourahmad1"></a>Pourahmad J, O'Brien PJ, Jokar F, Daraei B (2003) Carcinogenic metal induced sites of reactive oxygen species formation in hepatocytes. Toxicol In Vitro  17: 803–810.  <ul class="find" data-citedArticleID="926558" data-doi="10.1016/s0887-2333(03)00123-1"><li><a href="http://dx.doi.org/10.1016/s0887-2333(03)00123-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carcinogenic+metal+induced+sites+of+reactive+oxygen+species+formation+in+hepatocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Carcinogenic+metal+induced+sites+of+reactive+oxygen+species+formation+in+hepatocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000163-DelRazo1" id="pone.0000163-DelRazo1"></a>Del Razo LM, Quintanilla-Vega B, Brambila-Colombres E, Calderon-Aranda ES, Manno M, et al.  (2001) Stress proteins induced by arsenic. Toxicol Appl Pharmacol  177: 132–148.  <ul class="find" data-citedArticleID="926522"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Stress+proteins+induced+by+arsenic.&amp;auth=&amp;atitle=Stress+proteins+induced+by+arsenic." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stress+proteins+induced+by+arsenic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stress+proteins+induced+by+arsenic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000163-Romero1" id="pone.0000163-Romero1"></a>Romero D, Gomez-Zapata M, Luna A, Garcia-Fernandez AJ (2003) Morphological characterisation of BGM (Buffalo Green Monkey) cell line exposed to low doses of cadmium chloride. Toxicol In Vitro  17: 293–299.  <ul class="find" data-citedArticleID="926560" data-doi="10.1016/s0887-2333(03)00017-1"><li><a href="http://dx.doi.org/10.1016/s0887-2333(03)00017-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Morphological+characterisation+of+BGM+%28Buffalo+Green+Monkey%29+cell+line+exposed+to+low+doses+of+cadmium+chloride." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Morphological+characterisation+of+BGM+%28Buffalo+Green+Monkey%29+cell+line+exposed+to+low+doses+of+cadmium+chloride.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000163-Thery1" id="pone.0000163-Thery1"></a>Thery M, Racine V, Pepin A, Piel M, Chen Y, et al.  (2005) The extracellular matrix guides the orientation of the cell division axis. Nat Cell Biol  7: 947–953.  <ul class="find" data-citedArticleID="926574" data-doi="10.1038/ncb1307"><li><a href="http://dx.doi.org/10.1038/ncb1307" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+extracellular+matrix+guides+the+orientation+of+the+cell+division+axis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+extracellular+matrix+guides+the+orientation+of+the+cell+division+axis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000163-Aydin1" id="pone.0000163-Aydin1"></a>Aydin HH, Celik HA, Deveci R, Terzioglu E, Karacali S, et al.  (2003) Characterization of the cellular response during apoptosis induction in cadmium-treated Hep G2 human hepatoma cells. Biol Trace Elem Res  95: 139–153.  <ul class="find" data-citedArticleID="926514" data-doi="10.1385/bter:95:2:139"><li><a href="http://dx.doi.org/10.1385/bter:95:2:139" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+cellular+response+during+apoptosis+induction+in+cadmium-treated+Hep+G2+human+hepatoma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+cellular+response+during+apoptosis+induction+in+cadmium-treated+Hep+G2+human+hepatoma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000163-Young1" id="pone.0000163-Young1"></a>Young IT (1977) Proof without prejudice: use of the Kolmogorov-Smirnov test for the analysis of histograms from flow systems and other sources. J Histochem Cytochem  25: 935–941.  <ul class="find" data-citedArticleID="926584" data-doi="10.1177/25.7.894009"><li><a href="http://dx.doi.org/10.1177/25.7.894009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proof+without+prejudice%3A+use+of+the+Kolmogorov-Smirnov+test+for+the+analysis+of+histograms+from+flow+systems+and+other+sources." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proof+without+prejudice%3A+use+of+the+Kolmogorov-Smirnov+test+for+the+analysis+of+histograms+from+flow+systems+and+other+sources.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000163-Giuliano1" id="pone.0000163-Giuliano1"></a>Giuliano KA, Chen YT, Taylor DL (2004) High-content screening with siRNA optimizes a cell biological approach to drug discovery: defining the role of P53 activation in the cellular response to anticancer drugs. J Biomol Screen  9: 557–568.  <ul class="find" data-citedArticleID="926536" data-doi="10.1177/1087057104265387"><li><a href="http://dx.doi.org/10.1177/1087057104265387" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-content+screening+with+siRNA+optimizes+a+cell+biological+approach+to+drug+discovery%3A+defining+the+role+of+P53+activation+in+the+cellular+response+to+anticancer+drugs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-content+screening+with+siRNA+optimizes+a+cell+biological+approach+to+drug+discovery%3A+defining+the+role+of+P53+activation+in+the+cellular+response+to+anticancer+drugs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000163-Giuliano2" id="pone.0000163-Giuliano2"></a>Giuliano KA, Cheung WS, Curran DP, Day BW, Kassick AJ, et al.  (2005) Systems cell biology knowledge created from high content screening. Assay Drug Dev Technol  3: 501–514.  <ul class="find" data-citedArticleID="926538" data-doi="10.1089/adt.2005.3.501"><li><a href="http://dx.doi.org/10.1089/adt.2005.3.501" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systems+cell+biology+knowledge+created+from+high+content+screening." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systems+cell+biology+knowledge+created+from+high+content+screening.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000163-Tencza1" id="pone.0000163-Tencza1"></a>Tencza SB, Sipe MA (2004) Detection and classification of threat agents via high-content assays of mammalian cells. J Appl Toxicol  24: 371–377.  <ul class="find" data-citedArticleID="926572" data-doi="10.1002/jat.1025"><li><a href="http://dx.doi.org/10.1002/jat.1025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+and+classification+of+threat+agents+via+high-content+assays+of+mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+and+classification+of+threat+agents+via+high-content+assays+of+mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000163-Lee1" id="pone.0000163-Lee1"></a>Lee JH, Mitchell RJ, Kim BC, Cullen DC, Gu MB (2005) A cell array biosensor for environmental toxicity analysis. Biosens Bioelectron  21: 500–507.  <ul class="find" data-citedArticleID="926546" data-doi="10.1016/j.bios.2004.12.015"><li><a href="http://dx.doi.org/10.1016/j.bios.2004.12.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cell+array+biosensor+for+environmental+toxicity+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cell+array+biosensor+for+environmental+toxicity+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000163-Flaim1" id="pone.0000163-Flaim1"></a>Flaim CJ, Chien S, Bhatia SN (2005) An extracellular matrix microarray for probing cellular differentiation. Nat Methods  2: 119–125.  <ul class="find" data-citedArticleID="926532" data-doi="10.1038/nmeth736"><li><a href="http://dx.doi.org/10.1038/nmeth736" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+extracellular+matrix+microarray+for+probing+cellular+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+extracellular+matrix+microarray+for+probing+cellular+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000163-Lee2" id="pone.0000163-Lee2"></a>Lee MY, Park CB, Dordick JS, Clark DS (2005) Metabolizing enzyme toxicology assay chip (MetaChip) for high-throughput microscale toxicity analyses. Proc Natl Acad Sci U S A  102: 983–987.  <ul class="find" data-citedArticleID="926548" data-doi="10.1073/pnas.0406755102"><li><a href="http://dx.doi.org/10.1073/pnas.0406755102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolizing+enzyme+toxicology+assay+chip+%28MetaChip%29+for+high-throughput+microscale+toxicity+analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolizing+enzyme+toxicology+assay+chip+%28MetaChip%29+for+high-throughput+microscale+toxicity+analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000163-Arrigo1" id="pone.0000163-Arrigo1"></a>Arrigo AP (2000) sHsp as novel regulators of programmed cell death and tumorigenicity. Pathol Biol (Paris)  48: 280–288.  <ul class="find" data-citedArticleID="926512"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=sHsp+as+novel+regulators+of+programmed+cell+death+and+tumorigenicity.&amp;auth=&amp;atitle=sHsp+as+novel+regulators+of+programmed+cell+death+and+tumorigenicity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=sHsp+as+novel+regulators+of+programmed+cell+death+and+tumorigenicity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22sHsp+as+novel+regulators+of+programmed+cell+death+and+tumorigenicity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000163-AitAissa2" id="pone.0000163-AitAissa2"></a>Ait-Aissa S, Porcher J, Arrigo A, Lambre C (2000) Activation of the hsp70 promoter by environmental inorganic and organic chemicals: relationships with cytotoxicity and lipophilicity. Toxicology  145: 147–157.  <ul class="find" data-citedArticleID="926506" data-doi="10.1016/s0300-483x(00)00145-1"><li><a href="http://dx.doi.org/10.1016/s0300-483x(00)00145-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+of+the+hsp70+promoter+by+environmental+inorganic+and+organic+chemicals%3A+relationships+with+cytotoxicity+and+lipophilicity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+of+the+hsp70+promoter+by+environmental+inorganic+and+organic+chemicals%3A+relationships+with+cytotoxicity+and+lipophilicity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000163-Dudley1" id="pone.0000163-Dudley1"></a>Dudley RE, Svoboda DJ, Klaassen C (1982) Acute exposure to cadmium causes severe liver injury in rats. Toxicol Appl Pharmacol  65: 302–313.  <ul class="find" data-citedArticleID="926526" data-doi="10.1016/0041-008x(82)90013-8"><li><a href="http://dx.doi.org/10.1016/0041-008x(82)90013-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acute+exposure+to+cadmium+causes+severe+liver+injury+in+rats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acute+exposure+to+cadmium+causes+severe+liver+injury+in+rats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000163-Kondoh1" id="pone.0000163-Kondoh1"></a>Kondoh M, Tsukada M, Kuronaga M, Higashimoto M, Takiguchi M, et al.  (2004) Induction of hepatic metallothionein synthesis by endoplasmic reticulum stress in mice. Toxicol Lett  148: 133–139.  <ul class="find" data-citedArticleID="926544" data-doi="10.1016/j.toxlet.2003.12.066"><li><a href="http://dx.doi.org/10.1016/j.toxlet.2003.12.066" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+hepatic+metallothionein+synthesis+by+endoplasmic+reticulum+stress+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+hepatic+metallothionein+synthesis+by+endoplasmic+reticulum+stress+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000163-Alexandrova1" id="pone.0000163-Alexandrova1"></a>Alexandrova AY, Kopnin PB, Vasiliev JM, Kopnin BP (2006) ROS up-regulation mediates Ras-induced changes of cell morphology and motility. Exp Cell Res.   <ul class="find" data-citedArticleID="926508" data-doi="10.1016/j.yexcr.2006.03.004"><li><a href="http://dx.doi.org/10.1016/j.yexcr.2006.03.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ROS+up-regulation+mediates+Ras-induced+changes+of+cell+morphology+and+motility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ROS+up-regulation+mediates+Ras-induced+changes+of+cell+morphology+and+motility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000163-Meyer1" id="pone.0000163-Meyer1"></a>Meyer F (2001) An Overview of Morphological Segmentation. International Journal of Pattern Recognition and Artificial Intelligence Vol.  15: 1089–1118.  <ul class="find" data-citedArticleID="926554" data-doi="10.1142/s0218001401001337"><li><a href="http://dx.doi.org/10.1142/s0218001401001337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Overview+of+Morphological+Segmentation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Overview+of+Morphological+Segmentation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000163-Meyer2" id="pone.0000163-Meyer2"></a>Meyer F (2004) Levelings, Image Simplification Filters for Segmentation. Journal of Mathematical Imaging and Vision Vol.  20: 59–72.  <ul class="find" data-citedArticleID="926556" data-doi="10.1023/b:jmiv.0000011319.21884.39"><li><a href="http://dx.doi.org/10.1023/b:jmiv.0000011319.21884.39" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Levelings%2C+Image+Simplification+Filters+for+Segmentation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Levelings%2C+Image+Simplification+Filters+for+Segmentation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000163-Serra1" id="pone.0000163-Serra1"></a>Serra J (1999) Course on morphological operators.  
                  <a href="http://cmmensmpfr/">http://cmmensmpfr/</a>~serra/acourshtm; Centre de Morphologie Mathématique, Ecole des Mines de Paris.  <ul class="find" data-citedArticleID="926570"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Course+on+morphological+operators.&amp;auth=&amp;atitle=Course+on+morphological+operators." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Course+on+morphological+operators." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Course+on+morphological+operators.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000163-Angulo1" id="pone.0000163-Angulo1"></a>Angulo J, Serra J (2003) Automatic analysis of DNA microarray images using mathematical morphology. Bioinformatics  19: 553–562.  <ul class="find" data-citedArticleID="926510" data-doi="10.1093/bioinformatics/btg057"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg057" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automatic+analysis+of+DNA+microarray+images+using+mathematical+morphology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automatic+analysis+of+DNA+microarray+images+using+mathematical+morphology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.XML" value="128913"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.PDF" value="366605"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g001.PNG_L" value="1320602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g001.PNG_M" value="151225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g001.PNG_S" value="4834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g001.TIF" value="1685666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g001.PNG_I" value="48256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g002.PNG_L" value="752742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g002.PNG_M" value="134595"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g002.PNG_S" value="4325"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g002.TIF" value="1077928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g002.PNG_I" value="55011"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g003.PNG_L" value="308268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g003.PNG_M" value="82106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g003.PNG_S" value="3456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g003.TIF" value="556720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g003.PNG_I" value="35607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g004.PNG_L" value="384137"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g004.PNG_M" value="44788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g004.PNG_S" value="2566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g004.TIF" value="625354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g004.PNG_I" value="19349"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g005.PNG_L" value="179779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g005.PNG_M" value="100899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g005.PNG_S" value="4688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g005.TIF" value="322050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000163.g005.PNG_I" value="45915"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000163&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000163" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000163&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000163&volume=&issue=&title=Toxicity Assays in Nanodrops Combining Bioassay and Morphometric Endpoints&author_name=Fr%C3%A9d%C3%A9ric%20Lemaire%2C%20C%C3%A9line%20A.%20Mandon%2C%20Julien%20Reboud%2C%20Alexandre%20Papine%2C%20Jesus%20Angulo%2C%20Herv%C3%A9%20Pointu%2C%20Chantal%20Diaz-Latoud%2C%20Christian%20Lajaunie%2C%20Fran%C3%A7ois%20Chatelain%2C%20Andr%C3%A9-Patrick%20Arrigo%2C%20B%C3%A9atrice%20Schaack&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000163" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000163" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000163" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000163&amp;t=Toxicity%20Assays%20in%20Nanodrops%20Combining%20Bioassay%20and%20Morphometric%20Endpoints" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000163&title=Toxicity%20Assays%20in%20Nanodrops%20Combining%20Bioassay%20and%20Morphometric%20Endpoints&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000163&amp;title=Toxicity%20Assays%20in%20Nanodrops%20Combining%20Bioassay%20and%20Morphometric%20Endpoints" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000163" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000163&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Toxicity Assays in Nanodrops Combining Bioassay and Morphometric Endpoints';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000163';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Toxicity%20Assays%20in%20Nanodrops%20Combining%20Bioassay%20and%20Morphometric%20Endpoints http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000163" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000163" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arsenates%22" title="Search for articles in the subject area:'Arsenates'"><div class="flagText">Arsenates</div></a>
              <div data-categoryid="27231" data-articleid="23164"
                   data-categoryname="Arsenates"
                   class="flagImage" title="Flag 'Arsenates' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cadmium%22" title="Search for articles in the subject area:'Cadmium'"><div class="flagText">Cadmium</div></a>
              <div data-categoryid="24835" data-articleid="23164"
                   data-categoryname="Cadmium"
                   class="flagImage" title="Flag 'Cadmium' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+cultures%22" title="Search for articles in the subject area:'Cell cultures'"><div class="flagText">Cell cultures</div></a>
              <div data-categoryid="17567" data-articleid="23164"
                   data-categoryname="Cell cultures"
                   class="flagImage" title="Flag 'Cell cultures' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin+immunoprecipitation%22" title="Search for articles in the subject area:'Chromatin immunoprecipitation'"><div class="flagText">Chromatin immunoprecipitation</div></a>
              <div data-categoryid="40169" data-articleid="23164"
                   data-categoryname="Chromatin immunoprecipitation"
                   class="flagImage" title="Flag 'Chromatin immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cytotoxicity+assay%22" title="Search for articles in the subject area:'Cytotoxicity assay'"><div class="flagText">Cytotoxicity assay</div></a>
              <div data-categoryid="24339" data-articleid="23164"
                   data-categoryname="Cytotoxicity assay"
                   class="flagImage" title="Flag 'Cytotoxicity assay' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fluorescence+imaging%22" title="Search for articles in the subject area:'Fluorescence imaging'"><div class="flagText">Fluorescence imaging</div></a>
              <div data-categoryid="17303" data-articleid="23164"
                   data-categoryname="Fluorescence imaging"
                   class="flagImage" title="Flag 'Fluorescence imaging' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+engineering%22" title="Search for articles in the subject area:'Genetic engineering'"><div class="flagText">Genetic engineering</div></a>
              <div data-categoryid="20621" data-articleid="23164"
                   data-categoryname="Genetic engineering"
                   class="flagImage" title="Flag 'Genetic engineering' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Toxicity%22" title="Search for articles in the subject area:'Toxicity'"><div class="flagText">Toxicity</div></a>
              <div data-categoryid="32015" data-articleid="23164"
                   data-categoryname="Toxicity"
                   class="flagImage" title="Flag 'Toxicity' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5917'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4208'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=27&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
