

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The OSU1/QUA2/TSD2-Encoded Putative Methyltransferase Is a Critical Modulator of Carbon and Nitrogen Nutrient Balance Response in Arabidopsis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001387"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001387" />

    <meta name="citation_title" content="The OSU1/QUA2/TSD2-Encoded Putative Methyltransferase Is a Critical Modulator of Carbon and Nitrogen Nutrient Balance Response in Arabidopsis"/>
    <meta itemprop="name" content="The OSU1/QUA2/TSD2-Encoded Putative Methyltransferase Is a Critical Modulator of Carbon and Nitrogen Nutrient Balance Response in Arabidopsis"/>

      <meta name="citation_author" content="Peng Gao"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Lehman College, City University of New York, Bronx, New York, United States of America"/>
      <meta name="citation_author" content="Zeyu Xin"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Lehman College, City University of New York, Bronx, New York, United States of America"/>
      <meta name="citation_author" content="Zhi-Liang Zheng"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Lehman College, City University of New York, Bronx, New York, United States of America"/>
            <meta name="citation_author_institution" content="Plant Sciences PhD Subprogram, Graduate School and University Center, City University of New York, New York, New York, United States of America"/>

    <meta name="citation_date" content="2008/1/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001387.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1387"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Nitrogen and carbon nutrient and metabolite signaling in plants.; citation_author=G Coruzzi; citation_author=DR Bush; citation_journal_title=Plant Physiol; citation_volume=125; citation_number=1; citation_pages=61-64; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Carbon and nitrogen sensing and signaling in plants: emerging ‘matrix effects’.; citation_author=GM Coruzzi; citation_author=L Zhou; citation_journal_title=Curr Opin Plant Biol; citation_volume=4; citation_number=2; citation_pages=247-253; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Environmental regulation of lateral root initiation in Arabidopsis.; citation_author=JE Malamy; citation_author=KS Ryan; citation_journal_title=Plant Physiol; citation_volume=127; citation_number=3; citation_pages=899-909; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Regulatory links between carbon and nitrogen metabolism.; citation_author=FM Commichau; citation_author=K Forchhammer; citation_author=J Stulke; citation_journal_title=Curr Opin Microbiol; citation_volume=9; citation_number=4; citation_pages=167-172; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Role of insulin, adipocyte hormones, and nutrient-sensing pathways in regulating fuel metabolism and energy homeostasis: a nutritional perspective of diabetes, obesity, and cancer.; citation_author=S Marshall; citation_journal_title=Sci STKE; citation_volume=2006; citation_number=5; citation_pages=re7; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis seedling growth, storage lipid mobilization, and photosynthetic gene expression are regulated by carbon:nitrogen availability.; citation_author=T Martin; citation_author=O Oswald; citation_author=IA Graham; citation_journal_title=Plant Physiol; citation_volume=128; citation_number=6; citation_pages=472-481; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genome-wide patterns of carbon and nitrogen regulation of gene expression validate the combined carbon and nitrogen (CN)-signaling hypothesis in plants.; citation_author=PM Palenchar; citation_author=A Kouranov; citation_author=LV Lejay; citation_author=GM Coruzzi; citation_journal_title=Genome Biol; citation_volume=5; citation_number=7; citation_pages=R91; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Global transcription profiling reveals multiple sugar signal transduction mechanisms in Arabidopsis.; citation_author=J Price; citation_author=A Laxmi; citation_author=SK St Martin; citation_author=JC Jang; citation_journal_title=Plant Cell; citation_volume=16; citation_number=8; citation_pages=2128-2150; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Recent advances in the regulation of the TOR pathway by insulin and nutrients.; citation_author=J Avruch; citation_author=Y Lin; citation_author=X Long; citation_author=S Murthy; citation_author=S Ortiz-Vega; citation_journal_title=Curr Opin Clin Nutr Metab Care; citation_volume=8; citation_number=9; citation_pages=67-72; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sense and sensibility: nutritional response and signal integration in yeast.; citation_author=L Schneper; citation_author=K Duvel; citation_author=JR Broach; citation_journal_title=Curr Opin Microbiol; citation_volume=7; citation_number=10; citation_pages=624-630; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=G protein-coupled receptor Gpr4 senses amino acids and activates the cAMP-PKA pathway in Cryptococcus neoformans.; citation_author=C Xue; citation_author=YS Bahn; citation_author=GM Cox; citation_author=J Heitman; citation_journal_title=Mol Biol Cell; citation_volume=17; citation_number=11; citation_pages=667-679; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Convergence of TOR-nitrogen and Snf1-glucose signaling pathways onto Gln3.; citation_author=PG Bertram; citation_author=JH Choi; citation_author=J Carvalho; citation_author=TF Chan; citation_author=W Ai; citation_journal_title=Mol Cell Biol; citation_volume=22; citation_number=12; citation_pages=1246-1252; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Carbon catabolite repression regulates amino acid permeases in Saccharomyces cerevisiae via the TOR signaling pathway.; citation_author=GJ Peter; citation_author=L During; citation_author=A Ahmed; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=13; citation_pages=5546-5552; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Carbohydrate-modulated gene expression in plants.; citation_author=KE Koch; citation_journal_title=Annu Rev Plant Physiol Plant Mol Biol; citation_volume=47; citation_number=14; citation_pages=509-540; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Sugar sensing and signaling in plants: conserved and novel mechanisms.; citation_author=F Rolland; citation_author=E Baena-Gonzalez; citation_author=J Sheen; citation_journal_title=Annu Rev Plant Biol; citation_volume=57; citation_number=15; citation_pages=675-709; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sugar-Induced Signal Transduction in Plants.; citation_author=S Smeekens; citation_journal_title=Annu Rev Plant Physiol Plant Mol Biol; citation_volume=51; citation_number=16; citation_pages=49-81; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Primary N-assimilation into amino acids in Arabidopsis.; citation_author=G Coruzzi; citation_number=17; citation_date=2003; citation_publisher=American Society of Plant Biologists; " />
      <meta name="citation_reference" content="citation_title=Local and long-range signaling pathways regulating plant responses to nitrate.; citation_author=BG Forde; citation_journal_title=Annu Rev Plant Biol; citation_volume=53; citation_number=18; citation_pages=203-224; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The molecular-genetics of nitrogen assimilation into amino acids in higher plants.; citation_author=HM Lam; citation_author=KT Coschigano; citation_author=IC Oliveira; citation_author=R Melo-Oliveira; citation_author=GM Coruzzi; citation_journal_title=Annu Rev Plant Physiol Plant Mol Biol; citation_volume=47; citation_number=19; citation_pages=569-593; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Qualitative network models and genome-wide expression data define carbon/nitrogen-responsive molecular machines in Arabidopsis.; citation_author=RA Gutierrez; citation_author=LV Lejay; citation_author=A Dean; citation_author=F Chiaromonte; citation_author=DE Shasha; citation_journal_title=Genome Biol; citation_volume=8; citation_number=20; citation_pages=R7; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Reciprocal regulation of distinct asparagine synthetase genes by light and metabolites in Arabidopsis thaliana.; citation_author=HM Lam; citation_author=MH Hsieh; citation_author=G Coruzzi; citation_journal_title=Plant J; citation_volume=16; citation_number=21; citation_pages=345-353; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Metabolic regulation of the gene encoding glutamine-dependent asparagine synthetase in Arabidopsis thaliana.; citation_author=HM Lam; citation_author=SS Peng; citation_author=GM Coruzzi; citation_journal_title=Plant Physiol; citation_volume=106; citation_number=22; citation_pages=1347-1357; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis mutant analysis and gene regulation define a nonredundant role for glutamate dehydrogenase in nitrogen assimilation.; citation_author=R Melo-Oliveira; citation_author=IC Oliveira; citation_author=GM Coruzzi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=23; citation_pages=4718-4723; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Genome-wide reprogramming of primary and secondary metabolism, protein synthesis, cellular growth processes, and the regulatory infrastructure of Arabidopsis in response to nitrogen.; citation_author=WR Scheible; citation_author=R Morcuende; citation_author=T Czechowski; citation_author=C Fritz; citation_author=D Osuna; citation_journal_title=Plant Physiol; citation_volume=136; citation_number=24; citation_pages=2483-2499; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Microarray analysis of the nitrate response in Arabidopsis roots and shoots reveals over 1,000 rapidly responding genes and new linkages to glucose, trehalose-6-phosphate, iron, and sulfate metabolism.; citation_author=R Wang; citation_author=M Okamoto; citation_author=X Xing; citation_author=NM Crawford; citation_journal_title=Plant Physiol; citation_volume=132; citation_number=25; citation_pages=556-567; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Kinetic properties and ammonium-dependent regulation of cytosolic isoenzymes of glutamine synthetase in Arabidopsis.; citation_author=K Ishiyama; citation_author=E Inoue; citation_author=A Watanabe-Takahashi; citation_author=M Obara; citation_author=T Yamaya; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=26; citation_pages=16598-16605; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Overexpression of the ASN1 gene enhances nitrogen status in seeds of Arabidopsis.; citation_author=HM Lam; citation_author=P Wong; citation_author=HK Chan; citation_author=KM Yam; citation_author=L Chen; citation_journal_title=Plant Physiol; citation_volume=132; citation_number=27; citation_pages=926-935; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Carbon and amino acids reciprocally modulate the expression of glutamine synthetase in Arabidopsis.; citation_author=IC Oliveira; citation_author=GM Coruzzi; citation_journal_title=Plant Physiol; citation_volume=121; citation_number=28; citation_pages=301-310; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The putative high-affinity nitrate transporter NRT2.1 represses lateral root initiation in response to nutritional cues.; citation_author=DY Little; citation_author=H Rao; citation_author=S Oliva; citation_author=F Daniel-Vedele; citation_author=A Krapp; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=29; citation_pages=13693-13698; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A PII-like protein in Arabidopsis: putative role in nitrogen sensing.; citation_author=MH Hsieh; citation_author=HM Lam; citation_author=FJ van de Loo; citation_author=G Coruzzi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=30; citation_pages=13965-13970; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Interpreting the plastid carbon, nitrogen, and energy status. A role for PII?; citation_author=GB Moorhead; citation_author=CS Smith; citation_journal_title=Plant Physiol; citation_volume=133; citation_number=31; citation_pages=492-498; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Physiological characterisation of Arabidopsis mutants affected in the expression of the putative regulatory protein PII.; citation_author=S Ferrario-Mery; citation_author=M Bouvet; citation_author=O Leleu; citation_author=G Savino; citation_author=M Hodges; citation_journal_title=Planta; citation_volume=223; citation_number=32; citation_pages=28-39; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Expression and disruption of the Arabidopsis TOR (target of rapamycin) gene.; citation_author=B Menand; citation_author=T Desnos; citation_author=L Nussaume; citation_author=F Berger; citation_author=D Bouchez; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=33; citation_pages=6422-6427; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis AtRaptor genes are essential for post-embryonic plant growth.; citation_author=GH Anderson; citation_author=B Veit; citation_author=MR Hanson; citation_journal_title=BMC Biol; citation_volume=3; citation_number=34; citation_pages=12; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis TARGET OF RAPAMYCIN interacts with RAPTOR, which regulates the activity of S6 kinase in response to osmotic stress signals.; citation_author=MM Mahfouz; citation_author=S Kim; citation_author=AJ Delauney; citation_author=DP Verma; citation_journal_title=Plant Cell; citation_volume=18; citation_number=35; citation_pages=477-490; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The TUMOROUS SHOOT DEVELOPMENT2 gene of Arabidopsis encoding a putative methyltransferase is required for cell adhesion and co-ordinated plant development.; citation_author=E Krupkova; citation_author=P Immerzeel; citation_author=M Pauly; citation_author=T Schmulling; citation_journal_title=Plant J; citation_volume=50; citation_number=36; citation_pages=735-750; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Homogalacturonan synthesis in Arabidopsis thaliana requires a Golgi-localized protein with a putative methyltransferase domain.; citation_author=G Mouille; citation_author=MC Ralet; citation_author=C Cavelier; citation_author=C Eland; citation_author=D Effroy; citation_journal_title=Plant J; citation_volume=50; citation_number=37; citation_pages=605-614; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis abscisic acid response gene ABI5 encodes a basic leucine zipper transcription factor.; citation_author=RR Finkelstein; citation_author=TJ Lynch; citation_journal_title=Plant Cell; citation_volume=12; citation_number=38; citation_pages=599-609; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sucrose-specific induction of anthocyanin biosynthesis in Arabidopsis requires the MYB75/PAP1 gene.; citation_author=S Teng; citation_author=J Keurentjes; citation_author=L Bentsink; citation_author=M Koornneef; citation_author=S Smeekens; citation_journal_title=Plant Physiol; citation_volume=139; citation_number=39; citation_pages=1840-1852; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A negative regulatory role for auxin in sulphate deficiency response in Arabidopsis thaliana.; citation_author=H Dan; citation_author=G Yang; citation_author=ZL Zheng; citation_journal_title=Plant Mol Biol; citation_volume=63; citation_number=40; citation_pages=221-235; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phosphate starvation responses are mediated by sugar signaling in Arabidopsis.; citation_author=AS Karthikeyan; citation_author=DK Varadarajan; citation_author=A Jain; citation_author=MA Held; citation_author=NC Carpita; citation_journal_title=Planta; citation_volume=225; citation_number=41; citation_pages=907-918; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A conserved MYB transcription factor involved in phosphate starvation signaling both in vascular plants and in unicellular algae.; citation_author=V Rubio; citation_author=F Linhares; citation_author=R Solano; citation_author=AC Martin; citation_author=J Iglesias; citation_journal_title=Genes Dev; citation_volume=15; citation_number=42; citation_pages=2122-2133; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Activation tagging identifies a conserved MYB regulator of phenylpropanoid biosynthesis.; citation_author=JO Borevitz; citation_author=Y Xia; citation_author=J Blount; citation_author=RA Dixon; citation_author=C Lamb; citation_journal_title=Plant Cell; citation_volume=12; citation_number=43; citation_pages=2383-2394; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Nitrogen deficiency enhances expression of specific MYB and bHLH transcription factors and accumulation of end products in the flavonoid pathway.; citation_author=US Lea; citation_author=R Slimestad; citation_author=P Smedvig; citation_author=C Lillo; citation_journal_title=Planta; citation_volume=225; citation_number=44; citation_pages=1245-1253; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Suppression of the biosynthesis of proanthocyanidin in Arabidopsis by a chimeric PAP1 repressor.; citation_author=K Matsui; citation_author=H Tanaka; citation_author=M Ohme-Takagi; citation_journal_title=Plant Biotechnol J; citation_volume=2; citation_number=45; citation_pages=487-493; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Functional genomics by integrated analysis of metabolome and transcriptome of Arabidopsis plants over-expressing an MYB transcription factor.; citation_author=T Tohge; citation_author=Y Nishiyama; citation_author=MY Hirai; citation_author=M Yano; citation_author=J Nakajima; citation_journal_title=Plant J; citation_volume=42; citation_number=46; citation_pages=218-235; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dominant repression of target genes by chimeric repressors that include the EAR motif, a repression domain, in Arabidopsis.; citation_author=K Hiratsu; citation_author=K Matsui; citation_author=T Koyama; citation_author=M Ohme-Takagi; citation_journal_title=Plant J; citation_volume=34; citation_number=47; citation_pages=733-739; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Independent modulation of Arabidopsis thaliana polyubiquitin mRNAs in different organs and in response to environmental changes.; citation_author=CW Sun; citation_author=J Callis; citation_journal_title=Plant J; citation_volume=11; citation_number=48; citation_pages=1017-1027; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Positional cloning in Arabidopsis. Why it feels good to have a genome initiative working for you.; citation_author=W Lukowitz; citation_author=CS Gillmor; citation_author=WR Scheible; citation_journal_title=Plant Physiol; citation_volume=123; citation_number=49; citation_pages=795-805; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cloning of cDNAs for genes that are early-responsive to dehydration stress (ERDs) in Arabidopsis thaliana L.: identification of three ERDs as HSP cognate genes.; citation_author=T Kiyosue; citation_author=K Yamaguchi-Shinozaki; citation_author=K Shinozaki; citation_journal_title=Plant Mol Biol; citation_volume=25; citation_number=50; citation_pages=791-798; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=An indole-3-acetic acid carboxyl methyltransferase regulates Arabidopsis leaf development.; citation_author=G Qin; citation_author=H Gu; citation_author=Y Zhao; citation_author=Z Ma; citation_author=G Shi; citation_journal_title=Plant Cell; citation_volume=17; citation_number=51; citation_pages=2693-2704; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulatory functions of nuclear hexokinase1 complex in glucose signaling.; citation_author=YH Cho; citation_author=SD Yoo; citation_author=J Sheen; citation_journal_title=Cell; citation_volume=127; citation_number=52; citation_pages=579-589; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Hexokinase as a sugar sensor in higher plants.; citation_author=JC Jang; citation_author=P Leon; citation_author=L Zhou; citation_author=J Sheen; citation_journal_title=Plant Cell; citation_volume=9; citation_number=53; citation_pages=5-19; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Role of the Arabidopsis glucose sensor HXK1 in nutrient, light, and hormonal signaling.; citation_author=B Moore; citation_author=L Zhou; citation_author=F Rolland; citation_author=Q Hall; citation_author=WH Cheng; citation_journal_title=Science; citation_volume=300; citation_number=54; citation_pages=332-336; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The plastid protein THYLAKOID FORMATION1 and the plasma membrane G-protein GPA1 interact in a novel sugar-signaling mechanism in Arabidopsis.; citation_author=J Huang; citation_author=JP Taylor; citation_author=JG Chen; citation_author=JF Uhrig; citation_author=DJ Schnell; citation_journal_title=Plant Cell; citation_volume=18; citation_number=55; citation_pages=1226-1238; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An Arabidopsis MADS box gene that controls nutrient-induced changes in root architecture.; citation_author=H Zhang; citation_author=BG Forde; citation_journal_title=Science; citation_volume=279; citation_number=56; citation_pages=407-409; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis trehalose-6-P synthase AtTPS1 gene is a regulator of glucose, abscisic acid, and stress signaling.; citation_author=N Avonce; citation_author=B Leyman; citation_author=JO Mascorro-Gallardo; citation_author=P Van Dijck; citation_author=JM Thevelein; citation_journal_title=Plant Physiol; citation_volume=136; citation_number=57; citation_pages=3649-3659; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=SAM (dependent) I AM: the S-adenosylmethionine-dependent methyltransferase fold.; citation_author=JL Martin; citation_author=FM McMillan; citation_journal_title=Curr Opin Struct Biol; citation_volume=12; citation_number=58; citation_pages=783-793; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=An Arabidopsis thaliana T-DNA mutagenized population (GABI-Kat) for flanking sequence tag-based reverse genetics.; citation_author=MG Rosso; citation_author=Y Li; citation_author=N Strizhov; citation_author=B Reiss; citation_author=K Dekker; citation_journal_title=Plant Mol Biol; citation_volume=53; citation_number=59; citation_pages=247-259; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Floral dip: a simplified method for Agrobacterium-mediated transformation of Arabidopsis thaliana.; citation_author=SJ Clough; citation_author=AF Bent; citation_journal_title=Plant J; citation_volume=16; citation_number=60; citation_pages=735-743; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=GUS fusions: beta-glucuronidase as a sensitive and versatile gene fusion marker in higher plants.; citation_author=RA Jefferson; citation_author=TA Kavanagh; citation_author=MW Bevan; citation_journal_title=EMBO J; citation_volume=6; citation_number=61; citation_pages=3901-3907; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Transcriptome analysis reveals specific modulation of abscisic acid signaling by ROP10 small GTPase in Arabidopsis.; citation_author=Z Xin; citation_author=Y Zhao; citation_author=ZL Zheng; citation_journal_title=Plant Physiol; citation_volume=139; citation_number=62; citation_pages=1350-1365; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Plasma membrane-associated ROP10 small GTPase is a specific negative regulator of abscisic acid responses in Arabidopsis.; citation_author=ZL Zheng; citation_author=M Nafisi; citation_author=A Tam; citation_author=H Li; citation_author=DN Crowell; citation_journal_title=Plant Cell; citation_volume=14; citation_number=63; citation_pages=2787-2797; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001387" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The OSU1/QUA2/TSD2-Encoded Putative Methyltransferase Is a Critical Modulator of Carbon and Nitrogen Nutrient Balance Response in Arabidopsis"/>
    <meta name="twitter:description" content="The balance between carbon (C) and nitrogen (N) nutrients must be tightly coordinated so that cells can optimize their opportunity for metabolism, growth and development. However, the C and N nutrient balance perception and signaling mechanism remains poorly understood. Here, we report the isolation and characterization of two allelic oversensitive to sugar1 mutants (osu1-1, osu1-2) in Arabidopsis thaliana. Using the cotyledon anthocyanin accumulation and root growth inhibition assays, we show that the osu1 mutants are more sensitive than wild-type to both of the imbalanced C/N conditions, high C/low N and low C/high N. However, under the balanced C/N conditions (low C/low N or high C/high N), the osu1 mutants have similar anthocyanin levels and root lengths as wild-type. Consistently, the genes encoding two MYB transcription factors (MYB75 and MYB90) and an Asn synthetase isoform (ASN1) are strongly up-regulated by the OSU1 mutation in response to high C/low N and low C/high N, respectively. Furthermore, the enhanced sensitivity of osu1-1 to high C/low N with respect to anthocyanin accumulation but not root growth inhibition can be suppressed by co-suppression of MYB75, indicating that MYB75 acts downstream of OSU1 in the high C/low N imbalance response. Map-based cloning reveals that OSU1 encodes a member of a large family of putative methyltransferases and is allelic to the recently reported QUA2/TSD2 locus identified in genetic screens for cell-adhesion-defective mutants. Accumulation of OSU1/QUA2/TSD2 transcript was not regulated by C and N balance, but the OSU1 promoter was slightly more active in the vascular system. Taken together, our results show that the OSU1/QUA2/TSD2-encoded putative methyltransferase is required for normal C/N nutrient balance response in plants."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001387.g007"/>

  <meta property="og:title" content="The OSU1/QUA2/TSD2-Encoded Putative Methyltransferase Is a Critical Modulator of Carbon and Nitrogen Nutrient Balance Response in Arabidopsis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001387" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4348'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7659'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4141&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001387">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001387" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001387&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The <i>OSU1/QUA2/TSD2</i>-Encoded Putative Methyltransferase Is a Critical Modulator of Carbon and Nitrogen Nutrient Balance Response in <i>Arabidopsis</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peng Gao
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Peng Gao, Zeyu Xin</p>

              
              

                <p>Affiliation:
                  Department of Biological Sciences, Lehman College, City University of New York, Bronx, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zeyu Xin
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Peng Gao, Zeyu Xin</p>

              
              

                <p>Affiliation:
                  Department of Biological Sciences, Lehman College, City University of New York, Bronx, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhi-Liang Zheng
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:zhiliang.zheng@lehman.cuny.edu">zhiliang.zheng@lehman.cuny.edu</a></p>

                <p>Affiliations:
                  Department of Biological Sciences, Lehman College, City University of New York, Bronx, New York, United States of America, 
                  Plant Sciences PhD Subprogram, Graduate School and University Center, City University of New York, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001387</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001387" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001387" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001387" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001387" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001387-g001" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001387-g002" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001387-g003" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001387-g004" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001387-g005" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001387-g006" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001387-g007" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001387">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001387" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The balance between carbon (C) and nitrogen (N) nutrients must be tightly coordinated so that cells can optimize their opportunity for metabolism, growth and development. However, the C and N nutrient balance perception and signaling mechanism remains poorly understood. Here, we report the isolation and characterization of two allelic <em>oversensitive to sugar1</em> mutants (<em>osu1-1, osu1-2</em>) in <em>Arabidopsis thaliana</em>. Using the cotyledon anthocyanin accumulation and root growth inhibition assays, we show that the <em>osu1</em> mutants are more sensitive than wild-type to both of the imbalanced C/N conditions, high C/low N and low C/high N. However, under the balanced C/N conditions (low C/low N or high C/high N), the <em>osu1</em> mutants have similar anthocyanin levels and root lengths as wild-type. Consistently, the genes encoding two MYB transcription factors (MYB75 and MYB90) and an Asn synthetase isoform (ASN1) are strongly up-regulated by the <em>OSU1</em> mutation in response to high C/low N and low C/high N, respectively. Furthermore, the enhanced sensitivity of <em>osu1-1</em> to high C/low N with respect to anthocyanin accumulation but not root growth inhibition can be suppressed by co-suppression of <em>MYB75</em>, indicating that MYB75 acts downstream of OSU1 in the high C/low N imbalance response. Map-based cloning reveals that <em>OSU1</em> encodes a member of a large family of putative methyltransferases and is allelic to the recently reported <em>QUA2/TSD2</em> locus identified in genetic screens for cell-adhesion-defective mutants. Accumulation of <em>OSU1/QUA2/TSD2</em> transcript was not regulated by C and N balance, but the <em>OSU1</em> promoter was slightly more active in the vascular system. Taken together, our results show that the <em>OSU1/QUA2/TSD2</em>-encoded putative methyltransferase is required for normal C/N nutrient balance response in plants.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Gao P, Xin Z, Zheng Z-L (2008) The <em>OSU1/QUA2/TSD2</em>-Encoded Putative Methyltransferase Is a Critical Modulator of Carbon and Nitrogen Nutrient Balance Response in <em>Arabidopsis</em>. PLoS ONE 3(1):
          e1387.
            doi:10.1371/journal.pone.0001387</p><p><strong>Academic Editor: </strong>Brian Dilkes, University of California at Davis, United States of America</p><p><strong>Received:</strong> June 22, 2007; <strong>Accepted:</strong> December 11, 2007; <strong>Published:</strong> January 2, 2008</p><p><strong>Copyright:</strong> © 2008 Gao et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by an NIH-SCORE grant (S06GM008225, Project Number 12 to Z-LZ) and in part by the City University of New York PSC-CUNY program (68436-00-37 to Z-LZ).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The tight coordination of cellular carbon (C) and nitrogen (N) metabolism must be accomplished for proper growth and development. Carbohydrates provide both the energy and the carbon-skeletons used for ammonium assimilation during amino acid biosynthesis, while amino acids and proteins are the key building blocks for the cell <a href="#pone.0001387-Coruzzi1">[1]</a>, <a href="#pone.0001387-Coruzzi2">[2]</a>. Therefore, cells must monitor both the status of C and N nutrients and the balance between C and N to optimize their opportunity for metabolism, growth and development.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Despite numerous studies on the sensing and signaling mechanisms for levels of available C (such as glucose and sucrose) or N (such as nitrate, ammonium, and amino acids), the interaction between C and N has received attention only recently <a href="#pone.0001387-Coruzzi1">[1]</a>–<a href="#pone.0001387-Price1">[8]</a>. In bacteria, the PII protein has been shown to be a central processor for integrating N and C metabolism <a href="#pone.0001387-Commichau1">[4]</a>. In yeast or animal systems, Ras/Rho GTPases, Snf1/AMPK or the “hexosamine” signaling pathway and other components are involved in sensing and signaling the glucose or amino acid status <a href="#pone.0001387-Commichau1">[4]</a>, <a href="#pone.0001387-Avruch1">[9]</a>, <a href="#pone.0001387-Schneper1">[10]</a>. Surprisingly, not much is known about the cross-talk between C and N signaling pathways, although a C and N nutrient coincidence detection system exists (such as reported in <a href="#pone.0001387-Xue1">[11]</a>). Some examples of the cross-talk include the convergence of Snf1-glucose and TOR-nitrogen signaling pathways onto the Gln3 transcription factor <a href="#pone.0001387-Bertram1">[12]</a>, and the TOR pathway-mediated carbon catabolite repression of amino acid permeases <a href="#pone.0001387-Peter1">[13]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The molecular mechanisms by which plants sense the C and N balance or the C:N ratio and transduce the signal remain even more poorly understood <a href="#pone.0001387-Coruzzi1">[1]</a>, <a href="#pone.0001387-Coruzzi2">[2]</a>. Most plants are autotrophic organisms that synthesize both carbohydrates and amino acids and monitor their C and N status. There has been some exciting progress in understanding the mechanisms of sugar signaling <a href="#pone.0001387-Koch1">[14]</a>–<a href="#pone.0001387-Smeekens1">[16]</a> and the regulation of nitrogen metabolism/signaling <a href="#pone.0001387-Coruzzi3">[17]</a>–<a href="#pone.0001387-Lam1">[19]</a>. Recent DNA microarray studies have also shown that many genes are regulated by C alone, N alone, or their interactions, but only a limited number of genes have been shown to be regulated by the C:N ratio <a href="#pone.0001387-Palenchar1">[7]</a>, <a href="#pone.0001387-Price1">[8]</a>, <a href="#pone.0001387-Gutierrez1">[20]</a>–<a href="#pone.0001387-Wang1">[25]</a>. These genes are mostly involved in C or N metabolism such as Asn and Gln synthetase and Glu dehydrogenase genes <a href="#pone.0001387-Lam2">[21]</a>–<a href="#pone.0001387-MeloOliveira1">[23]</a>, <a href="#pone.0001387-Ishiyama1">[26]</a>–<a href="#pone.0001387-Oliveira1">[28]</a> or transport such as a nitrate transporter gene <a href="#pone.0001387-Malamy1">[3]</a>, <a href="#pone.0001387-Little1">[29]</a>. On the other hand, studies on plant orthologs of bacterial PII and mammalian TOR have not provided convincing support for their involvement in C/N balance signaling in plants. For example, the <em>Arabidopsis</em> ortholog of PII, GLB1, was implicated in the C:N ratio sensing, based on an ectopic overexpression study <a href="#pone.0001387-Hsieh1">[30]</a>. Although plant PII orthologs were speculated to function as a sensor for both C and N <a href="#pone.0001387-Moorhead1">[31]</a>, the T-DNA knockout mutants of <em>GLB1</em> exhibit only weak phenotypes <a href="#pone.0001387-FerrarioMery1">[32]</a>. This indicates a subtle involvement of the PII protein in the regulation of some steps of primary C and N metabolism in plants <a href="#pone.0001387-FerrarioMery1">[32]</a>. The TOR ortholog has not been suggested to be involved in vegetative development <a href="#pone.0001387-Menand1">[33]</a>, and the functions of its regulators (such as RAPTOR1A and RAPTOR1B) in nutrient signaling have not been reported <a href="#pone.0001387-Anderson1">[34]</a>, <a href="#pone.0001387-Mahfouz1">[35]</a>. The molecular mechanism underlying the perception or signaling of the C and N balance in plants remains a mystery.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Here, we describe the isolation of the <em>oversensitive to sugar1</em> (<em>osu1)</em> mutants. Using a modified C/N balance bioassay, we show that they are hypersensitive to imbalanced C/N (both high C/low N and low C/high N) but are similar to the wild-type (WT) when C/N ratios are more balanced (low C/low N, or high C/high N). Furthermore, <em>MYB75</em>, a transcription factor gene that is up-regulated by the <em>OSU1</em> mutations, acts downstream of <em>OSU1</em> in response to high C/low N. <em>OSU1</em> suppresses the low C/high N-activation of an Asn synthetase gene (<em>ASN1</em>) which is known to be transcriptionally regulated by the C and N balance. Molecular cloning reveals that <em>OSU1</em> encodes a putative methyltransferase that belongs to a functionally uncharacterized family with a total of 29 members in the Arabidopsis genome. While we were preparing this manuscript, two reports showed that the allelic mutations in <em>OSU1</em> (called <em>QUA2/TSD2</em>) affect cell adhesion and that QUA2/TSD2 is localized in the Golgi <a href="#pone.0001387-Krupkova1">[36]</a>, <a href="#pone.0001387-Mouille1">[37]</a>. <em>OSU1/QUA2/TSD2</em> has a slight preference for expression in the vascular system, consistent with its role in nutrient response. Taken together, our results show that the <em>OSU1/QUA2/TSD2</em>-encoded putative methyltransferase modulates the C and N balance response in <em>Arabidopsis</em>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Isolation of the <em>osu1</em> mutants</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>During the process of screening for homozygous transgenic lines expressing a <em>ROP2-</em>promoter:<em>GUS</em> construct (<em>P<sub>ROP2</sub>:GUS</em>) in half-strength (1/2X) MS medium supplemented with 1% sucrose (Suc) and hygromycin, we found two T2 plants (23G13-2 and 23G9-8) from independent T1 transgenic lines that segregated for reddish purple and normally green cotyledons in the T3 generation. The progeny of seven T2 siblings did not (<a href="#pone-0001387-g001">Figure 1A</a> and data not shown). For convenience, the concentration of supplemental C and N will be expressed as milimolar concentrations in this study. A combination of 30 mM Suc and 30 mM of total N was designated 30C/30N. The 1% Suc (equivalent to 29 mM or 29C in this report) in 1/2X MS medium that contains 30 mM N (composed of 19.7 mM NO<sub>3</sub><sup>−</sup> and 10.3 mM NH<sub>4</sub><sup>+</sup> and designated 30N) did not induce the accumulation of sufficient anthocyanins to cause cotyledons to turn reddish purple in WT plants or the other transformants. This indicates that lineages 23G13-2 and 23G9-8 may carry mutations that led to the induction of anthocyanin production.</p>
<div class="figure" id="pone-0001387-g001"><div class="img"><a name="pone-0001387-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Isolation and characterization of the <em>osu1</em> mutants.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Seedlings shown in this figure were all grown in the commercially prepared 1/2 MS medium which contained 30 mM total N (30N) for seven days. For (D-F), the average of anthocyanin contents from three replicates, each with 8–10 seedlings, is shown, with the bar representing the SD. Statistical analysis by one way ANOVA; the asterisk (*) above the column indicates a significant difference (p&lt;0.05) between <em>osu1-1</em> or <em>osu1-2</em> and wild-type (WT) under the same nutrient condition. (A) T3 seedlings of the two <em>P<sub>ROP2</sub>:GUS</em> lines (23G13-2 and 23G9-8) exhibited segregation of reddish purple cotyledons (indicated by arrows) and normal green cotyledons when grown in the presence of 1% sucrose (approximately 30 mM Suc, designated 30C) and 50 µg/mL hygromycin. Other T3 seedlings from the two parental plants (23G13-4 and 23G9-1) did not show such a phenotype. (B) Failure of 23G13-2 to complement 23G9-8-1 under 120 mM sucrose. 13-2 represents 23G13-2, which is designated <em>osu1-1</em>; 9-8-1 represents 23G-9-8-1, which is designated <em>osu1-2</em>; F1, the progeny from the cross of <em>osu1-1</em> X <em>osu1-2</em>. (C) The two mutants, <em>osu1-1</em> and <em>osu1-2</em>, showed enhanced sensitivities to increasing concentrations of sucrose (0C to 210C) at 30N after 7 days of vertical growth. For 30C/30N+180 Man, 180 mM of mannitol was added to the medium containing 30 mM Suc, so that the total carbohydrate concentration of the medium was equivalent to 210 mM Suc. (D) Quantitative analysis of cotyledon anthocyanin accumulation in response to various Suc (C) concentrations at 30N. FW, fresh weight. (E) Quantitative analysis of cotyledon anothcyanin accumulation in response to the same concentration (210 mM) of Suc, glucose (Glc) and Mannitol (Man). No sugar supplementation was used as the control. (F) Quantitative analysis of cotyledon anothcyanin accumulation in response to various concentrations of maltose and 3-<em>O</em>-methylglucose (OMG). No sugar supplementation was used as the control.</p>
<span>doi:10.1371/journal.pone.0001387.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The progeny of 23G13-2 showed a clear 3:1 segregation ratio (36:14, χ<sup>2</sup>-value = 0.24, <em>p</em>-value &gt;0.05) of green versus reddish purple cotyledons. F1 progeny after backcrossing to WT had normal green cotyledons, and their F2 progeny from the F1 selfcross also showed a clear 3:1 segregation (157:45, χ<sup>2</sup>-value = 0.80, <em>p</em>-value &gt;0.05). T3 progeny from 23G9-8 showed a deviation from 3:1 ratio (42:5, χ<sup>2</sup>-value = 5.17, <em>p</em>-value&lt;0.05). One of the plants showing anthocyanin accumulation, 23G9-8-1, was backcrossed to the Columbia (Col) WT and the resulting F1 showed normal cotyledons (data not shown). F2 plants exhibited a 3:1 ratio of segregating green versus reddish purple cotyledons (57:18, χ<sup>2</sup>-value = 0.04, <em>p</em>-value &gt;0.05). These genetic analyses confirmed that these two lines carried monogenic, recessive mutations that caused the Petri-plate grown seedlings to accumulate more anthocyanins.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>To determine whether the phenotype is genetically linked to the <em>P<sub>ROP2</sub>:GUS</em> construct, co-segregation analysis was performed using the F2 progeny from 23G13-2 crossed to WT. Results showed a segregation of 58 (green cotyledons and GUS positive): 16 (green cotyledon and GUS negative): 19 (reddish purple cotyledons and GUS positive): 4 (reddish purple cotyledons and GUS negative). This is consistent with a segregation ratio of 9:3:3:1 (χ<sup>2</sup>-value = 1.22, <em>p</em>-value &gt;0.05) for independent assortment of two loci. Therefore, the reddish purple cotyledon phenotype in 23G13-2 is unlinked to the T-DNA insertion. Mutations that occurred during T-DNA transformation but unrelated to the T-DNA insertion have been reported such as <em>abi5</em> <a href="#pone.0001387-Finkelstein1">[38]</a>.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>F2 homozygous plants, selected for the absence of the <em>P<sub>ROP2</sub>:GUS</em> T-DNA insertion (determined by the absence of hygromycin resistance and GUS staining) from 23G13-2 and 23G9-8-1, were obtained. To determine whether 23G13-2 and 23G9-8-1 represent different loci or alleles of the same locus, they were crossed. The F1 progeny were indistinguishable from their parental plants when grown on the medium supplemented with 120 mM Suc (<a href="#pone-0001387-g001">Figure 1B</a>). Together, these results suggest that the Suc-enhanced phenotypes of 23G13-2 and 23G9-8-1 are due to mutant alleles at the same locus and therefore are designated <em>osu1-1</em> and <em>osu1-2</em>, respectively.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>To further characterize the <em>osu1</em> Suc response phenotype, homozygous F3 <em>osu1-1</em> and <em>osu1-2</em> mutant plants were grown vertically on agar-solidified media supplemented with various Suc concentrations. Surprisingly, we found that at 30C/30N, cotyledons of 23G13-2 and 23G9-8-1 F3 seedlings did not appear reddish purple (<a href="#pone-0001387-g001">Figure 1C</a>) as was the case for horizontally grown seedlings on 1% Suc (29C/30N; <a href="#pone-0001387-g001">Figure 1A</a>). However, measurement of anthocyanin levels showed that they still had slightly higher anthocyanin levels than WT (<a href="#pone-0001387-g001">Figure 1D</a>). The smaller difference in anthocyanins between WT and the mutants under 30C/30N (<a href="#pone-0001387-g001">Figure 1, C and D</a>) than that under 29C/30N (<a href="#pone-0001387-g001">Figure 1A</a>) was due to vertical versus horizontal growth, instead of the presence or absence of hygromycin in the medium or the difference between 29C and 30C (data not shown). When C was increased at a constant N concentration from 30C/30N to 120C/30N and 210 C/30N, vertically grown WT seedlings showed increasing accumulation of anthocyanins. Both <em>osu1-1</em> and <em>osu1-2</em> mutants accumulated more anthocyanins than WT (<a href="#pone-0001387-g001">Figure 1D</a>). At 300C/30N the anthocyanin levels in WT were similar to that in both mutants (<a href="#pone-0001387-g001">Figure 1D</a>), indicating a possible saturation in the response. In the absence of supplemental Suc (0C/30N), both mutants had similar anthocyanin levels but shorter roots than WT plants (<a href="#pone-0001387-g001">Figure 1C and D</a>).</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>It has been shown that fructose and sorbitol do not induce anthocyanin accumulation, while maltose is similar to Suc, and glucose has an intermediate effect <a href="#pone.0001387-Teng1">[39]</a>. We tested the effects of using glucose and maltose as the source of C on anthocyanin accumluation. When glucose was used, both mutants also exhibited increases in anthocyanin accumulation, although <em>osu1-1</em> was less sensitive than <em>osu1-2</em> (<a href="#pone-0001387-g001">Figure 1E</a>). Similar to the results with Suc (<a href="#pone-0001387-g001">Figure 1D</a>), <em>osu1-1</em> and <em>osu1-2</em> accumulated more anthocyanins than WT on plates supplemented with maltose (<a href="#pone-0001387-g001">Figure 1F</a>). To exclude the possibility that the sugar effect is due to osmosis, we tested the seedling growth in the presence of mannitol and 3-<em>O</em>-methylglucose (a non-metabolizable glucose analog). The mutants were slightly less sensitive than WT to 3-<em>O</em>-methylglucose which weakly induced anthocyanin accumulation (<a href="#pone-0001387-g001">Figure 1F</a> and <a href="#pone.0001387-Dan1">[40]</a>). Growth on the medium supplemented with mannitol had a similar effect on anthocyanin accumulation (<a href="#pone-0001387-g001">Figure 1E</a>). When 180 mM mannitol was added to the medium supplemented with 30 mM Suc, we found that both of the mutants exhibited similar inhibition of root growth as WT (<a href="#pone-0001387-g001">Figure 1C</a>). Taken together, the two mutants greatly enhanced anthocyanin accumulation in response to high concentrations of metabolizable sugars (Suc, maltose and glucose) but not non-metabolizable or slowly metabolized sugars (3-<em>O</em>-methylglucose and mannitol). Thus, the enhanced sugar sensitivity of these mutants is not due to osmotic sensitivity.</p>


<h4>The <em>osu1</em> mutants are sensitive to an imbalance of C and N</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The <em>osu1</em> mutant seedlings were hypersensitive to both high C/low N (120C/30N or 210C/30N) and low C/high N (0C/30N) but exhibited similar anthocyanin levels and root growth as WT plants when grown under more balanced C/N conditions (e.g. 30C/30N; <a href="#pone-0001387-g001">Figure 1C</a>). This led us to hypothesize that <em>osu1</em> mutants could be more sensitive to an imbalance of C and N. If this is the case, <em>osu1</em> seedlings should have similar anthocyanin levels and root lengths as WT when grown in medium with balanced C/N conditions regardless of absolute C or N concentration. Therefore, we formulated growth media with combinations of various C and N levels, and observed root growth and cotyledon anthocyanins in WT and <em>osu1</em> mutant plants <a href="#pone.0001387-Martin1">[6]</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>When grown in medium with 0.5C/0.05N, <em>osu1-1</em> and <em>osu1-2</em> seedlings showed pale green cotyledons similar to WT. The roots of the <em>osu1</em> mutant seedlings were slightly (less than 2 mm or about 10%) shorter than WT (<a href="#pone-0001387-g002">Figure 2, A and C</a>). Quantitation of anthocyanin levels showed that the <em>osu1</em> mutants accumulated similar levels of anthocyanins in the cotyledons as WT (<a href="#pone-0001387-g002">Figure 2B</a>). However, when C increased from 0.5 to 1 or 5 mM while N was kept constant (0.05N), <em>osu1-1</em> and <em>osu1-2</em> accumulated more anthocyanins while WT did not (<a href="#pone-0001387-g002">Figure 2B</a>). As C levels increased further (15C/0.05N and 30C/0.05N), anthocyanin contents increased in WT cotyledons but <em>osu1</em> still had higher anthocyanin levels than WT. Similar results were obtained comparing medium with varying levels of C and 1N. The <em>osu1</em> mutants had higher anthocyanin levels when C increased to 15 or 30 mM. These observations are consistent with the earlier observation that <em>osu1</em> also had higher anthocyanin levels when C increased from 30 to 120 or 210 in the presence of 30N (<a href="#pone-0001387-g001">Figure 1D</a>). Consistent with the observation that the high C-induced anthocyanin accumulation is dependent on low levels of N <a href="#pone.0001387-Martin1">[6]</a>, the anthocyanin levels increase when the N levels in the medium decrease from 1 to 0.05 mM while C is either 5C, 15C, or 30C (<a href="#pone-0001387-g002">Figure 2B</a>). Importantly, there was no difference between <em>osu1</em> and WT under 5C/1N, while decreasing N from 1 to 0.05 mM resulted in higher anthocyanin levels in <em>osu1</em>. Similarly, although <em>osu1</em> had no difference in anthocyanin levels under 15C/4N compared to WT, reducing N from 4 to 1 mM caused <em>osu1</em> mutants to accumulate more anthocyanins while WT did not. If N was further reduced from 1 to 0.05 mM in the presence of 15C, WT plants accumulated anthocyanins although <em>osu1</em> exhibited a similar increase as WT. In media with 30C, decreasing N from 30 to 1 mM resulted in a larger increase of anthocyanin levels for the <em>osu1</em> mutants than in WT. Further decreasing N to 0.05 mM caused a similar increase in WT and <em>osu1</em>. The <em>osu1</em> mutants increased anthocyanin accumulation in response to increased C when N was kept constant and to decreased N when C was kept constant. Thus, the accumulation of anthocyanins in <em>osu1</em> mutant seedlings was hypersensitive to the nutrient combinations with higher concentrations of C relative to N.</p>
<div class="figure" id="pone-0001387-g002"><div class="img"><a name="pone-0001387-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The <em>osu1</em> mutants increase the sensitivity to the imbalanced C/N.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) Representative seedlings of WT, <em>osu1-1</em>, and <em>osu1-2</em> after seven days of growth on 0.5C/0.05N, 0.5C/4N, 15C/0.05N, and 15C/4N. (B) Quantitative analysis of cotyledon anthocyanin accumulation under various C/N conditions. Cotyledons excised from the 7-day-old seedlings were used to measure anthocyanin levels in relative units per g of fresh weight (FW). Shown are the average and the SD of three replicates, each with 8–10 seedlings. (C) Quantitative analysis of root growth inhibition under various C/N. Primary root lengths were measured. The average of four replicates (each with 8–10 seedlings) is shown, with the bar representing the SD. Statistical analysis by one way ANOVA; the asterisk (*) above the column indicates a significant difference (p&lt;0.05) between <em>osu1-1</em> or <em>osu1-2</em> and wild-type (WT) under the same C/N condition.</p>
<span>doi:10.1371/journal.pone.0001387.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The enhanced sensitivity of <em>osu1</em> mutants to the low C/high N imbalanced conditions was also observed in the root growth assay. Increasing N from 0.05 to 30 mM in medium with 0.5C caused progressively shorter root lengths in the <em>osu1</em> mutants as compared to WT plants (<a href="#pone-0001387-g002">Figure 2C</a>). When N was varied across a higher (5C) concentration of Suc (<a href="#pone-0001387-g002">Figure 2C</a>) or no C was provided to the medium (<a href="#pone.0001387.s003">Figure S1</a>), <em>osu1</em> and WT exhibited similar responses to increasing N levels. To exclude the possibility that the observed root growth phenotype in <em>osu1</em> might be due to seed germination variation between <em>osu1</em> and WT, seed germination was recorded every two hours. The <em>osu1</em> mutants had very similar germination profiles as WT under four tested C/N conditions, 0.5C/0.05N, 0.5C/4N, 15C/0.05N, and 15C/4N (<a href="#pone.0001387.s004">Figure S2</a>). In addition, <em>osu1</em> differed from WT in both anthocyanin accumulation and root growth inhibition at 15C/0.05N but only the root growth inhibition at 0.5C/4N. This indicates that anthocyanin accumulation and root growth are likely two separable responses under the imbalanced C/N conditions.</p>


<h4>The <em>osu1</em> mutants do not increase anthocyanin accumulation in response to phosphate or sulphate nutrient deficiencies</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Anthocyanin accumulation is also activated by phosphate or sulphate nutrient deficiencies, which are also dependent on C availability <a href="#pone.0001387-Dan1">[40]</a>–<a href="#pone.0001387-Rubio1">[42]</a>. We tested whether <em>osu1</em> mutants were more sensitive than WT to phosphate or sulphate deficiencies in the presence or absence of 15C. Comparison of cotyledon anthocyanin levels in 0C/4N and 15C/4N-treated 7-day-old seedlings grown vertically in the presence of various phosphate levels showed that C greatly promoted phosphate deficiency-activated anthocyanin accumulation (<a href="#pone-0001387-g003">Figure 3A</a>). At 0C/4N, WT plants accumulated more anthocyanins in the absence than the presence of phosphate (<a href="#pone-0001387-g003">Figure 3A</a>). Anthocyanin accumulation was not sufficient to cause cotyledons to turn visibly reddish purple (data not shown). Nevertheless, <em>osu1</em> did not increase the sensitivity to phosphate deficiency and indeed had slightly lower anthocyanin levels than in WT in most of the conditions. In medium containing 15C/4N, both <em>osu1</em> and WT had similar activation of anthocyanin accumulation in response to phosphate deficiencies except that <em>osu1-1</em> had slightly lower anthocyanin levels than WT (<a href="#pone-0001387-g003">Figure 3A</a>).</p>
<div class="figure" id="pone-0001387-g003"><div class="img"><a name="pone-0001387-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span><em>osu1</em> does not increase the sensitivity to phosphate and sulphate nutrient deficiency-activated anthocyanin accumulation.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Anthocyanin accumulation under phosphate deficiency. Seeds were germinated and grown vertically on two C/N (0C/4N or 15C/4N) conditions with various phosphate (P) concentrations (0, 0.05, and 1.25 mM) for seven days. The average of three replicates (each with 6–8 seedlings) is shown, with the bar representing the SD. (B) Anthocyanin accumulation under sulphate deficiency. Seven-day-old seedlings vertically grown on the 15C/4N plates were transferred to liquid 0C/4N or 15C/4N medium supplemented with various sulphate (S) concentrations (0.0001, 0.0301, and 1.6001 mM) for additional three days. The average of three replicates (each with 5 or 6 seedlings) is shown, with the bar representing the SD. FW, fresh weight. Statistical analysis by one way ANOVA; the asterisk (*) above the column indicates a significant difference (p&lt;0.05) between <em>osu1-1</em> or <em>osu1-2</em> and wild-type (WT) under the same C/N/P or C/N/S condition.</p>
<span>doi:10.1371/journal.pone.0001387.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Seedlings grown vertically on agar-solidified medium with a deficiency of sulphate were not visibly red (data not shown). Seedlings in liquid culture under sulphate deficiency (0.0001 mM) accumulated one-fold higher anthocyanin levels than the sulphate sufficiency (1.6001 mM) control in a previous study <a href="#pone.0001387-Dan1">[40]</a>. We grew seedlings on the medium containing 15C/4N with a sufficient sulphate supply for seven days and then transferred them to liquid media supplemented with three different sulphate levels and both 0C/4N and 15C/4N conditions. Sulphate deficiency activated anthocyanin accumulation, dependent on the availability of C (<a href="#pone-0001387-g003">Figure 3B</a>). This effect was greater than that observed during phosphate deficiency response (<a href="#pone-0001387-g003">Figure 3A</a>). Under 0C/4N, sulphate deficiency did not activate anthocyanin accumulation. The <em>osu1-1</em> and <em>osu1-2</em> mutant plants exhibited very similar patterns of anthocyanin accumulation as WT in response to sulphate deficiency. The only difference was a slightly lower anthocyanin level in <em>osu1</em> than the WT at 0.0301 mM sulphate under 15C/4N (<a href="#pone-0001387-g003">Figure 3B</a>). Phosphate or sulphate nutrient deficiency activated anthocyanin accumulation but <em>osu1</em> mutants were not enhanced in these responses. This suggests that OSU1 functions in the response to C and N nutrient balance but is not involved in a response shared by nutrient deficiencies in general.</p>


<h4><em>OSU1</em> suppresses expression of <em>MYB75</em> in high C/low N</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We investigated whether the <em>osu1</em> alleles affect the transcript levels of two MYB transcription factor genes, <em>MYB75/PAP1</em> and <em>MYB90/PAP2</em>, which are involved in activating anthocyanin biosynthesis in response to C and N nutrient status <a href="#pone.0001387-Teng1">[39]</a>, <a href="#pone.0001387-Borevitz1">[43]</a>–<a href="#pone.0001387-Tohge1">[46]</a>. Real-time RT-PCR analysis showed that in medium with 15C/0.05N, both <em>osu1-1</em> and <em>osu1-2</em> had greater <em>MYB75</em> expression as compared to WT. Only <em>osu1-1</em> had higher <em>MYB75</em> expression levels than WT under 15C/4N, while <em>osu1-2</em> was similar to WT (<a href="#pone-0001387-g004">Figure 4A</a>). At 0.5C/4N, both mutants and WT had similarly low levels of <em>MYB75</em> expression. This pattern of transcriptional regulation is consistent with the phenotypic observation that <em>osu1</em> had similar anthocyanin levels as WT under 0.5C/4N (<a href="#pone-0001387-g002">Figure 2B</a>). A similar response pattern for <em>MYB90</em> was observed except that <em>osu1-1</em> and <em>osu1-2</em> had slightly higher <em>MYB90</em> transcript levels under 0.5C/0.05N (<a href="#pone-0001387-g004">Figure 4B</a>). These results indicate that <em>osu1-1</em> and <em>osu1-2</em> up-regulates <em>MYB75</em> and <em>MYB90</em> induction by high C/low N.</p>
<div class="figure" id="pone-0001387-g004"><div class="img"><a name="pone-0001387-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>OSU1 acts upstream of MYB75 in response to high C/low N.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(A and B) The <em>OSU1</em> mutations altered <em>MYB75</em> and <em>MYB90</em> expression in response to high C/low N. Four-day-old WT, <em>osu1-1</em> and <em>osu1-2</em> seedlings grown vertically on 15C/4N were transferred to freshly prepared media with four different C/N conditions for three days of treatment. <em>MYB75</em> (A) and <em>MYB90</em> (B) mRNA levels were normalized with those of the reference (<em>ACT2</em>), and the relative <em>MYB75</em> or <em>MYB90</em> mRNA level under 0.5C/0.05N was set at 1. The average and the SD bar for <em>MYB75</em> (n = 4) and <em>MYB90</em> (n = 3) are shown. (C) RT-PCR analysis of <em>MYB75</em> expression in <em>MYB75</em> sense (#16, 20 and 24) and dominant negative SRDX lines (#13 and 17). Primers were used to amplify both endogenous and transgenic <em>MYB75</em> expression. <em>ACT2</em>, an internal control. (D) Anthocyanin accumulation of the transgenic lines under high C/low N. The averages and SDs of three replicates, each with 10-15 seedlings, are shown. FW, fresh weight. (E and F) The <em>MYB75</em> co-suppression line (Sense #24) is epistatic to <em>osu1-1</em> at high C/low N. (E) Anthocyanin levels, and (F) root lengths at four different C/N conditions, each with four replicates. <em>osu1-1</em>/Sense #24 represents a homozygous line of <em>osu-1</em> and <em>MYB75</em> co-suppression. Statistical analysis by one way ANOVA; the asterisk (*) above the column indicates a significant difference (p&lt;0.05) from wild-type (WT) under the same C/N condition.</p>
<span>doi:10.1371/journal.pone.0001387.g004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>To confirm that <em>OSU1</em> acts upstream of <em>MYB75</em> and to test the hypothesis that anthocyanin accumulation and root growth are two separable responses under the imbalanced C/N conditions, an epistasis study was performed. We generated transgenic plants in the Col background that express a dominant negative form of MYB75 by fusing MYB75 to the SUPERMAN repression domain (SRDX) which has been shown to suppress the function of several transcription factors <a href="#pone.0001387-Hiratsu1">[47]</a>, under the control of the constitutively active UBQ10 promoter <a href="#pone.0001387-Sun1">[48]</a>. We obtained two lines that overexpressed <em>MYB75-SRDX</em> (lines #13 and 17; <a href="#pone-0001387-g004">Figure 4C</a>) which showed the expected green cotyledon phenotypes under 30C/1N (data not shown) due to reduced anthocyanin accumulation as compared to WT (<a href="#pone-0001387-g004">Figure 4D</a>). Stronger knock-down of <em>MYB75</em> was obtained from co-suppression of the endogenous <em>MYB75</em> by a construct designed to overexpress <em>MYB75</em> cDNA. RT-PCR analysis showed that while two overexpression lines (#16 and 20) accumulated more <em>MYB75</em> transcripts than WT, the co-suppressed line (#24) accumulated far less <em>MYB75</em> mRNA (<a href="#pone-0001387-g004">Figure 4C</a>). Real-time PCR analysis showed that the total transcript level of both endogenous and transgenic <em>MYB75</em> in Sense #24 was reduced to only 9% of <em>MYB75</em> levels in WT. Consistent with their <em>MYB75</em> expression levels, the overexpressors (Sense #16 and #20) had more anthocyanins, while the <em>MYB75</em> co-suppression line (Sense #24) had less anthocyanin accumulation than WT when grown at 15C/0.05N (<a href="#pone-0001387-g004">Figure 4D</a>). This co-suppression line exhibited even stronger suppression of anthocyanin accumulation than the two SRDX lines under 15C/0.05N. Furthermore, in contrast to the two SRDX lines, Sense #24 only exhibited a very mild increase of anthocyanins when C increased from 15 to 30 mM (<a href="#pone-0001387-g004">Figure 4D</a>). Therefore, Sense #24 was used for the subsequent study.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Sense #24 was crossed to <em>osu1-1</em>, and the F3 seedlings from a homozygous <em>osu1-1</em>/Sense #24 plant were then subjected to various C/N treatments. When media containing 0.5C/0.05N and 15C/4N were used, there was no difference between WT and <em>osu1-1</em>, but Sense #24 had a slightly lower level of anthocyanins than WT (<a href="#pone-0001387-g004">Figure 4E</a>). Under these conditions, <em>osu1-1</em>/Sense #24 had similar anthocyanin levels as Sense #24 alone, indicating that <em>MYB75</em> co-suppression can block the production of anthocyanins in both WT and <em>osu1-1</em>. At 15C/0.05N, while <em>osu1-1</em> dramatically increased the anthocyanin levels, <em>osu1-1</em>/Sense #24 showed only a very small increase in anthocyanin accumulation. At 30C/1N, Sense #24 completely eliminated the activation of anthocyanin accumulation in <em>osu1-1</em> (<a href="#pone-0001387-g004">Figure 4E</a>). However, root growth inhibition in <em>osu1-1</em> was not restored by Sense #24 (<a href="#pone-0001387-g004">Figure 4F</a>). Neither did the co-suppression of <em>MYB75</em> alone affect root growth compared to WT. These results show that MYB75 is downstream of OSU1 in modulating anthocyanin induction and that anthocyanin accumulation and root growth in <em>osu1</em> mutants are two separable responses.</p>


<h4><em>OSU1</em> suppresses <em>ASN1</em> expression in response to low C/high N</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The observation that <em>MYB75</em> and <em>MYB90</em> were affected by the <em>osu1-2</em> and <em>osu1-2</em> mutations under high C/low N led us to test whether the expression of an isoform of Asn synthetase (<em>ASN1)</em>, was affected by <em>OSU1</em>. <em>ASN1</em> mRNA expression has been shown to be regulated by light/sugar and amino acids and to correlate with amino acid biosynthesis <a href="#pone.0001387-Lam2">[21]</a>, <a href="#pone.0001387-Lam3">[22]</a>, <a href="#pone.0001387-Lam4">[27]</a>. <em>ASN1</em> expression under 0.5C/0.05N was 40% lower in <em>osu1-2</em> than WT (<a href="#pone-0001387-g005">Figure 5</a>). Increasing the N to 0.5C/4N caused an almost eight-fold increase of <em>ASN1</em> transcript levels in <em>osu1-2</em> compared to <em>osu1-2</em> at 0.5C/0.05N. This resulted in <em>osu1-2</em> mutants accumulating four fold more <em>ASN1</em> transcript than WT. <em>ASN1</em> expression was very low and similar in WT and <em>osu1-2</em> at 15C/4N. The <em>osu1-2</em> mutant was slightly reduced as compared to WT at 15C/4N, similar to what was found at 0.5C/0.05N. These results showed that the <em>OSU1</em> mutation increased the quantity of <em>ASN1</em> up-regulation by low C/high N, while the differences between WT and <em>osu1-2</em> under both of the balanced conditions (0.5C/0.05N and 15C/4N) were similar and small. This effect of <em>osu1-2</em> on <em>ASN1</em> expression under low C/high N is in sharp contrast to that for <em>MYB75</em> and <em>MYB90</em>, which were modulated by loss of OSU1 function under high C/low N growth conditions (<a href="#pone-0001387-g004">Figures 4A and 4B</a>).</p>
<div class="figure" id="pone-0001387-g005"><div class="img"><a name="pone-0001387-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>OSU1 suppresses <em>ASN1</em> expression in response to low C/high N.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Seedlings were treated with various C/N conditions and real-time RT-PCR analysis was performed as described in <a href="#pone-0001387-g004">Figure 4</a> legend. mRNA levels were normalized with those of the <em>ACT2</em> reference, and the relative <em>ASN1</em> mRNA level under 0.5C/0.05N was set at 1. The bar represents the SD of three replicates. Statistical analysis by <em>t</em>-test; the asterisk (*) above the <em>osu1-2</em> column indicates a significant difference (p&lt;0.05) from wild-type (WT) under the same C/N condition.</p>
<span>doi:10.1371/journal.pone.0001387.g005</span></div>

<h4><em>OSU1</em> encodes a member of a large family of putative methyltransferases</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>We cloned the <em>OSU1</em> gene using a map-based strategy <a href="#pone.0001387-Lukowitz1">[49]</a>. The <em>osu1-1</em> mutation was mapped to a 35 kb region between two markers on the two BAC clones T11I11 and F3F9 (<a href="#pone-0001387-g006">Figure 6A</a>, and <a href="#pone.0001387.s001">Table S1</a>). The sequences of open reading frames for seven of the nine genes within this region had DNA sequences identical to the published genome sequence. The remaining two genes, At1g78230 and At1g78240, could not be PCR amplified from <em>osu1-1</em>. Sequencing of these genes from <em>osu1-2</em> revealed that At1g78230 was identical to the reference sequence, and an A to T mutation in the seventh exon of At1g78240 (<a href="#pone-0001387-g006">Figure 6B</a>) resulting in a missense mutation that converts Asn at 560 to Tyr (N560Y). The genomic lesion in the region of the At1g78240 gene was not characterized further in the <em>osu1-1</em> mutant, but it is likely that a DNA rearrangement and/or deletion is responsible for the mutation (<a href="#pone-0001387-g006">Figure 6C</a> and data not shown). To further support this, we isolated homozygous T-DNA insertion lines collected from The Nottingham Arabidopsis Stock Centre (GABI_Kat 107A03) and Arabidopsis Biological Resources Center (Salk_122037 and Salk_062502). RT-PCR analysis showed that the two Salk lines, which have T-DNA insertions in the promoter region, did not affect At1g78240 gene expression (data not shown). However, the 2,055 bp CDS (from ATG to TGA) of the At1g78240 full-length cDNA could not be amplified in the GABI_Kat 107A03 line (<a href="#pone-0001387-g006">Figure 6C</a>) due to a T-DNA inserted into the eighth exon of At1g78240 (<a href="#pone-0001387-g006">Figure 6B</a>). This GABI line was designated <em>osu1-3</em>. Consistent with a deletion in <em>osu1-1</em>, no transcript was detected in RT-PCR reactions from RNA isolated from homozygous mutant plants (<a href="#pone-0001387-g006">Figure 6C</a>). In contrast, At1g78240 expression was not suppressed in <em>osu1-2</em>, indicating that the phenotype is likely due to the N560Y mutation inactivating the OSU1 protein. To quantify the expression level of At1g78240 in these alleles using real-time PCR, two primers were designed to amplify a 98 bp fragment within the second exon 5′ of the <em>osu1-2</em> mutation and the <em>osu1-3</em> T-DNA insertion sites (<a href="#pone-0001387-g006">Figure 6B</a>). Using these primers, At1g78240 transcript levels in <em>osu1-2</em> and <em>osu1-3</em> were respectively 50% and two-fold higher than that in WT (<a href="#pone-0001387-g006">Figure 6D</a>). Surprisingly, these reactions also detected the At1g78240 transcript in <em>osu1-1</em>, although only 2% of that in WT. These results indicate that At1g78240 mRNA expression could be regulated by negative feedback. We found that <em>osu1-3</em> showed similar C and N responses as <em>osu1-1</em> and <em>osu1-2</em> (data not shown). Furthermore, <em>osu1-3</em> failed to complement <em>osu1-1</em> and <em>osu1-2</em> (<a href="#pone-0001387-g006">Figure 6E</a>). These results show that the <em>OSU1</em> locus is At1g78240.</p>
<div class="figure" id="pone-0001387-g006"><div class="img"><a name="pone-0001387-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Molecular cloning of the <em>OSU1</em> gene.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) A diagram showing the <em>osu1</em> mutant gene mapping. The number of recombinants (chromosomes) is indicated for each marker used on the BAC clones. The name and the primers for each marker are listed in <a href="#pone.0001387.s001">Table S1</a>. The <em>osu1-1</em> mutant gene is mapped to a region between two markers (470560 on the BAC clone T11I11, and (470251 on the BAC clone F3F9) that respectively gave one and two recombinants. (B) A schematic representation of the <em>OSU1</em> genomic structure. <em>osu1-3</em> had an insertion of T-DNA surrounded by its two left border (LB) sequences that eliminated 10 bp starting from the 54<sup>th</sup> bp of the eighth exon (sequencing data for the T-DNA and genomic DNA junction not shown). The positions of start codon (ATG) and stop codon (TGA) were indicated. (C) Regular RT-PCR analysis of At1g78240 mRNA expression in the three <em>osu1</em> alleles. PCR reactions with 35 or 39 cycles were used to amplify the 2,055 bp CDS region of At1g78240. <em>ACT2</em> was used as an internal control. (D) Real-time PCR analysis of At1g78240 transcript levels. A 98 bp fragment of At1g78240 cDNA located in the second exon was amplified. Statistical analysis by one way ANOVA; the asterisk (*) above the column indicates a significant difference (p&lt;0.05) from wild-type (WT). The bar represents the SD of three replicates. (E) <em>osu1-3</em> fails to complement <em>osu1-1</em> and <em>osu1-2</em>, respectively. The F1 seeds of the crosses were sown on the medium containing 15C/0.05N and 0.5C/4N, respectively, and two representative seedlings vertically grown for 7 days for each genotype are shown.</p>
<span>doi:10.1371/journal.pone.0001387.g006</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>The <em>OSU1</em> gene is predicted to contain nine exons and encodes a putative methyltransferase with 684 amino acids. In the <em>Arabidopsis</em> genome, there are a total of 28 additional putative methyltransferases that are closely related to OSU1 (<a href="#pone.0001387.s005">Figure S3</a>). Among these, one, At4g19120/<em>Early Response to Drought3</em> (<em>ERD3</em>), has been reported to exhibit mild transcriptional induction to drought <a href="#pone.0001387-Kiyosue1">[50]</a>. The gene tree analysis, using IAMT1 <a href="#pone.0001387-Qin1">[51]</a> as an outgroup, indicates that the OSU1–related putative methyltransferases can be divided into six groups, I to VI (<a href="#pone.0001387.s005">Figure S3</a>). Group I has only one member encoded by At3g56080, while Group II contains OSU1 and two other members encoded by At1g13860 and At2g03480. Groups III-VI can be considered a sister to Group II. ERD3 belongs to Group V which is the largest group with 10 members. Amino acid sequence alignment shows that the <em>osu1-2</em> mutation affects a conserved amino acid N at 560 which is present within a highly conserved seven amino acid-containing motif, WVMNVVP. Only At2g39750 of Group IV (<a href="#pone.0001387.s005">Figure S3</a>), which has a serine residue at this position, lacks N<sub>560</sub>. Together with the fact that <em>osu1-2</em> (N560Y) does not reduce its transcript level (<a href="#pone-0001387-g006">Figure 6C</a>), this indicates that N<sub>560</sub> is critical for the proper function of the <em>OSU1</em>-encoded protein.</p>


<h4>Expression patterns of <em>OSU1</em></h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>We tested whether C and N nutrient imbalance affects <em>OSU1</em> transcription. Real-time RT-PCR analysis of <em>OSU1</em> transcript levels at 0.5C/4N, 15C/0.05N and 15C/4N were quite similar (<a href="#pone-0001387-g007">Figure 7A</a>). The former two nutrient regimes displayed increased root growth inhibition or anthocyanin accumulation in <em>osu1</em> as compared to WT controls (<a href="#pone-0001387-g002">Figure 2</a>). WT plants under these conditions accumulated 20–40% lower levels of <em>OSU1</em> transcript than plants at 0.5C/0.05N (<a href="#pone-0001387-g007">Figure 7A</a>), which did not induce anthocyanins in either <em>osu1</em> mutants or WT. Thus <em>OSU1</em> mNRA accumulation is regulated by C and N nutrient status, but only to a limited degree. To reveal organ-specific regulation, we performed RT-PCR analyses using RNAs extracted from young seedlings (roots and shoots), mature leaves, stems and flowers (<a href="#pone-0001387-g007">Figure 7B</a>). <em>OSU1</em> was expressed in all of these organs, with higher expression in the roots than the shoots of young seedlings. Transgenic plants were generated containing 3.2 kb of <em>OSU1</em> genomic DNA that includes the first exon, the first intron, and part of the second exon (up to 24 bp downstream of the ATG start codon) fused in frame to <em>β-glucuronidase (GUS)</em>. GUS staining of multiple independent transgenic lines showed that the <em>OSU1</em> promoter was more active in the root than in the shoot (<a href="#pone-0001387-g007">Figure 7, C and D</a>), consistent with the RT-PCR result (<a href="#pone-0001387-g007">Figure 7B</a>). Weak GUS staining was detected in root hairs (<a href="#pone-0001387-g007">Figure 7E</a>) and all cell layers of the root tip (<a href="#pone-0001387-g007">Figure 7F</a>). Staining was slightly stronger in the vascular tissues of the root (<a href="#pone-0001387-g007">Figure 7, E and F</a>) and in the vascular tissues of mature leaves (<a href="#pone-0001387-g007">Figure 7G</a>). In inflorescence, the promoter is active in stems, anthers, and stigma surface (<a href="#pone-0001387-g007">Figure 7H</a>).</p>
<div class="figure" id="pone-0001387-g007"><div class="img"><a name="pone-0001387-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001387" data-uri="info:doi/10.1371/journal.pone.0001387.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001387.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001387.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001387.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001387.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Expression patterns of <em>OSU1</em>.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>(A) Quantitative RT-PCR analysis of <em>OSU1</em> under various C/N conditions. WT seedlings were first grown vertically on agar-solidified 15C/4N medium for four days of optimal growth and then transferred to various C/N media for three days. mRNA levels were normalized with those of the <em>ACT2</em> reference, and the relative <em>OSU1</em> mRNA level under 0.5C/0.05N was set at 1. The bar represents the SD of three replicates. Statistical analysis by one way ANOVA; the asterisk (*) above the column indicates a significant difference (p&lt;0.05) compared to the control (0.5C/0.05N). (B) RT-PCR analysis of <em>OSU1</em> expression in different organs. Shown are shoots (Sh) and roots (R) of 7-day-old seedlings, mature leaves (L), stems (St) and flowers (F). <em>ACT2</em>, an internal control. (C-H) Expression patterns revealed by histochemical examination of the <em>OSU1</em> promoter activity using the <em>GUS</em> reporter. Shown are a 7-day-old young seedling stained for 4 hours (C) and another for 10 hours (D), enlarged views of root hairs (E) and a root tip (F), a mature leaf (G), and an inflorescence with several flowers (H).</p>
<span>doi:10.1371/journal.pone.0001387.g007</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Despite the fact that C and N influence the expression of many genes <a href="#pone.0001387-Palenchar1">[7]</a>, <a href="#pone.0001387-Price1">[8]</a>, <a href="#pone.0001387-Gutierrez1">[20]</a>, <a href="#pone.0001387-Scheible1">[24]</a>, the genetic control of the response to the imbalanced C/N conditions remains to be determined. It has been shown that the <em>lin1</em> mutant, which has a mutation in a nitrate transporter (NRT2.1) gene, reduces the sensitivity to high C/low N, but it is unknown whether <em>lin1</em> alters the response to low C/high N <a href="#pone.0001387-Malamy1">[3]</a>, <a href="#pone.0001387-Little1">[29]</a>. Here, we report that the <em>osu1</em> mutants increase the sensitivity to both high C/low N and low C/high N, suggesting that the <em>OSU1</em>-encoded putative methyltransferase acts as a negative modulator of the C/N nutrient imbalance response. <em>osu1-1</em> and <em>osu1-2</em> were initially isolated as sugar oversensitive mutants, but their response to sugars depends in part on the C/N ratio rather than responding solely to C. It should be noted that the <em>osu1</em> mutants did not show consistent hypersensitivity to the same C:N ratios when different absolute levels of C or N were compared. For example, under 15C/4N (a C:N ratio of close to 4), <em>osu1</em> and WT seedlings had similar root lengths and anthocyanin levels (<a href="#pone-0001387-g002">Figure 2</a>), but 120C/30N ( a ratio of 4) led <em>osu1</em> mutants to accumulate more anthocyanins than WT (<a href="#pone-0001387-g001">Figure 1D</a>). More importantly, under intermediate C/intermediate N (15C/4N) and high C/high N (30C/30N), the <em>osu1</em> mutants and WT had no or only a marginal difference in both anthocyanin accumulation and root growth. Phenotypes of previously described mutants or transgenic plants that alter C or N responses should be re-examined under a wider spectrum of C and N levels (<a href="#pone-0001387-g002">Figure 2</a>). Some of the mutants or transgenic plants include genes in regulatory pathways such as the hexokinase pathway <a href="#pone.0001387-Cho1">[52]</a>–<a href="#pone.0001387-Moore1">[54]</a>, heterotrimeric G-proteins <a href="#pone.0001387-Huang1">[55]</a>, and the ANR1 transcription factor <a href="#pone.0001387-Zhang1">[56]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The response to C/N ratio in <em>osu1</em> mutants suggests that OSU1/QUA2/TSD2 could be involved in nutrient utilization or signaling. The utilization of C (such as glycolysis or citric acid cycle) is controlled by N availability and similarly the utilization of N (such as nitrate or ammonium assimilation) is modulated by C availability <a href="#pone.0001387-Coruzzi1">[1]</a>, <a href="#pone.0001387-Coruzzi2">[2]</a>. Therefore, OSU1 might have a strong effect on overall metabolism and affect the response by altering the concentration of substrates critical in the C or N sensing pathways. Proteins involved in C or N transport and metabolism such as HXK1/HXK2, TPS1 and LIN1 also have a regulatory role in C or N signaling <a href="#pone.0001387-Malamy1">[3]</a>, <a href="#pone.0001387-Little1">[29]</a>, <a href="#pone.0001387-Jang1">[53]</a>, <a href="#pone.0001387-Moore1">[54]</a>, <a href="#pone.0001387-Avonce1">[57]</a>. Clearly, it is necessary to dissect the biochemical mechanism by which loss of OSU1 function enhances the response to changes in the ratio of C to N.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Several of our results suggest that the enzyme activity of OSU1/QUA2/TSD2 is necessary for its function in C and N balance signaling. First, the accumulation of <em>OSU1/QUA2/TSD2</em> transcript was not dramatically altered by changes to C and N conditions (<a href="#pone-0001387-g007">Figure 7A</a>). Second, the single amino acid replacement (N560Y) in <em>osu1-2</em> does not lead to its transcriptional suppression but has a strong phenotype (<a href="#pone-0001387-g002">Figures 2</a> and <a href="#pone-0001387-g006">6</a>). This mutation in <em>osu1-2</em> occurs within a conserved seven amino acid motif in the predicted methyltransferase domain (<a href="#pone.0001387.s005">Figure S3</a>). AdoMet-dependent methyltransferases add a methyl group to various biologically active molecules such as proteins, lipids, DNA/RNA, and secondary products. Although they share little sequence identity, around 120 members have been classified (EC 2.1.1.X), based on the substrate specificity and on the atom targeted for methylation <a href="#pone.0001387-Martin2">[58]</a>. The observations that <em>osu1/qua2/tsd2</em> mutants exhibit defects in cell wall composition or structure and hypersensitivity to C/N balance indicate a possible link between cell wall modifications and C/N nutrient balance response. The <em>qua2</em> mutant exhibits a 50% reduction in homogalacturonan content compared to WT <a href="#pone.0001387-Mouille1">[37]</a>. Our observed C/N response phenotypes could be due to these cell wall defects. Alternatively, loss of OSU1 function could result in the production of oligosaccharides with signaling activity, or a disruption in <em>S</em>-adenosylmethionine utilization, either of which might affect the C/N signaling mechanism. The hypothesized role of QUA2/TSD2 as a pectin methyltransferase has not been demonstrated. We failed to express and purify OSU1 in bacteria using various protein tags (data not shown), and the difficulty was also mentioned in another report <a href="#pone.0001387-Mouille1">[37]</a>. Therefore, it remains unclear at this time whether OSU1-modulated cell wall modification is direct and how it might be involved in the C/N balance response including anthocyanin biosynthesis and root growth. Dissection of the OSU1 biochemical and regulatory functions will help elucidate the mechanism by which plants, and possibly other cellular organisms, respond to C and N nutrient balance.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Plant growth and materials</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p><em>Arabidopsis thaliana</em> Columbia (Col) wild-type, transgenic plants and mutants in the Col background were used in this study. Seeds were cold-treated at 4°C for 2–4 days and then allowed for germination and growth in the greenhouse or incubator at 22°C with 16 h light and 8 h dark.</p>


<h4>Isolation of the <em>osu1</em> mutants and cloning of the <em>OSU1</em> mutant gene</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The <em>osu1</em> mutants were isolated from T3 seedlings that carried the <em>P<sub>ROP2</sub>:GUS</em> construct (see below) and grown in the half-strength (1/2X) MS medium supplemented with 1% sucrose and 50 µg/mL hygromycin. The mutants were then backcrossed to Col to remove the T-DNA transgene background. For map-based cloning, <em>osu1-1</em> was out-crossed to the ecotype L<em>er</em>, and the F2 seedlings from the F1 self-cross were selected for the <em>osu1</em> mutant phenotype. Genomic DNAs from a total of 544 F2 plants showing the <em>osu1</em> phenotype were extracted for marker genotyping. Simple Sequence Length Polymorphism (SSLP) markers representing distinct regions of each chromosome were first used for rough mapping according to Lukowitz et al. <a href="#pone.0001387-Lukowitz1">[49]</a>. After we showed that the <em>OSU1</em> mutant gene was mapped to the bottom arm (or the South end) of chromosome 1 between the markers, NGA111 and F23A5, fine mapping was performed by designing a set of primers (<a href="#pone.0001387.s001">Table S1</a>). As shown in <a href="#pone-0001387-g006">Figure 6A</a> and <a href="#pone.0001387.s001">Table S1</a>, the <em>OSU1</em> mutant gene was eventually mapped to a region of 35 kb. Subsequently, various fragments of the open reading frames for the nine genes included in this region were PCR-amplified using the primers listed in <a href="#pone.0001387.s001">Table S1</a> and then sequenced to determine where the mutation was located. The <em>osu1-2</em> allele was also used for DNA amplification in order to verify the mutant gene was represented by At1g78240. The <em>osu1-3</em> allele (GABI_Kat 107A03 in which a T-DNA insertion was verified by sequencing the junction between its left border and the <em>OSU1</em> eighth exon) was isolated from a collection maintained in The Nottingham Arabidopsis Stock Centre. For screening the T-DNA insertional plants, two primers were used to determine the insertion of T-DNA: GZP61 (<a href="#pone.0001387.s001">Table S1</a>) and GAB8409L (5′-ATATTGACCATCATACTCATTGC-3′), according to Rosso et al. <a href="#pone.0001387-Rosso1">[59]</a>. GZP61 and GZP62 (<a href="#pone.0001387.s001">Table S1</a>) were used to screen homozygous plants for the T-DNA insertion. Subsequently, the genomic region for the other side of T-DNA and <em>OSU1</em> junction was sequence verified using the PCR fragment amplified with the primers GZP62 and GAB8409L.</p>


<h4>C and N balance response assays</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The assays were modified from Martin et al. <a href="#pone.0001387-Martin1">[6]</a> and the various C/N media were prepared according to Dan et al. <a href="#pone.0001387-Dan1">[40]</a>. In brief, 1/2XMS salts without KNO<sub>3</sub>, NH<sub>4</sub>NO<sub>3 </sub>and sucrose was prepared at pH value of 5.7, and then various C and N were added to the media. Sucrose was used as the C source, and the combination of NO<sub>3</sub><sup>−</sup> and NH<sub>4</sub><sup>+</sup> in a molar ratio of 2:1 was used as the N source. All the media contained similar amount of K<sup>+</sup> by replacing KNO<sub>3</sub> with KCl, if necessary. 1% PhytoBlend (Caisson Laboratories Inc., USA) was added to solidify the medium. After 2–4 days of cold treatment, seeds were allowed to germinate and grow vertically for seven days. Anthocyanin contents were measured using the methanol-based extraction (99% methanol plus 1% concentrated HCl) method as described <a href="#pone.0001387-Dan1">[40]</a>. Lengths of the primary roots of the seedlings were measured using a ruler. RNA for RT-PCR was prepared from seedlings treated as follows: seeds were germinated and grown vertically under the balanced C/N (15C/4N) for four days and then transferred to the agar-solidified media of various C/N conditions for three more days of vertical growth.</p>


<h4>Phosphate and sulphate deficiency response assay</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>For phosphate deficiency, seeds were sown and seedlings were grown vertically in the plates containing various C and phosphate levels for seven days. Phosphate deficiency media were prepared by adding KH<sub>2</sub>PO<sub>4</sub> at various levels to 0C/4N and 15C/4N, adjusted by adding KCl to keep the [K<sup>+</sup>] constant. For sulphate deficiency, seven-day-old seedlings grown vertically under 15C/4N with sufficient sulphate were transferred to the 0C/4N and 15C/4N liquid media supplemented with various sulphate levels prepared as described elsewhere <a href="#pone.0001387-Dan1">[40]</a>. Anthocyanin contents were measured as described above.</p>


<h4>Production of <em>P<sub>ROP2</sub>:GUS</em> and <em>P<sub>osu1</sub>:GUS</em> transgenic plants</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>For the <em>P<sub>ROP2</sub>:GUS</em> construct, a 2.3 kb <em>ROP2</em> promoter fragment ending 2 bp upstream of ATG was PCR amplified from genomic DNA using the high fidelity DNA polymerase PfuUltra™ (Stratagene, USA) and the following gene-specific primers, with the underlined bases indicating the introduced restriction enzyme sites for cloning: sense (ZZP1: 5′-AT<span class="underline">CTGCAG</span>TTGCCTTCCTTCCTATGTACGTA-3′) and antisense (ZZP3: 5′-CGAT<span class="underline">CCATGG</span>CTGCCGCAAGATCGGAAACAA-3′). The amplified DNA fragment was digested by <em>Pst</em> I and <em>Nco</em> I, and then cloned into <em>Pst</em> I and <em>Nco</em> I sites of the binary vector pCAMBIA1301 (B4) that contains GUS and the CaMV 35S terminator. This resulted in the plasmid, ZZ102. For the <em>P<sub>osu1</sub>:GUS</em> construct, a 3.2 kb promoter fragment, including the first exon, the first intron and part of the second exon (up to 24 bp downstream of ATG), was similarly PCR amplified using the gene-specific primers, with the underlying bases indicating the introduced restriction enzyme sites for cloning: sense (GZP69: 5′- CAT<span class="underline">CTGCAG</span>GTCACGGTCCGCCACACACAAGA-3′) and antisense (GZP71: 5′- CC<span class="underline">TCTAGA</span>GCCACGTTGTAGTGGCATTGACAT-3′). The amplified DNA fragment was digested with <em>Xba</em> I, and then cloned into the <em>Hind</em> III (fill-in) and <em>Xba</em> I sites of the binary vector pBI101 that contains <em>GUS</em> and the <em>NOS</em> terminator. This resulted in the plasmid, GZ59. ZZ102 and GZ59 were then transformed into Col using the floral-dip method <a href="#pone.0001387-Clough1">[60]</a>. Multiple independently transformed lines were used for GUS activity assay as described elsewhere <a href="#pone.0001387-Jefferson1">[61]</a>.</p>


<h4>RT-PCR analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Total RNA was extracted with TRIzol (Invitrogen, USA) and 2–6 µg RNA were reverse transcribed in a 20-µL reaction using Superscript III reverse transcriptase and Oligo(dT)<sub>12–18</sub> primer (Invitrogen), according to the instructions provided by the vender. Regular PCR analysis was performed using Taq DNA polymerase (GenScript, USA), with <em>ACT2</em> as the internal control using the primers ACT2S and ACT2A described elsewhere <a href="#pone.0001387-Xin1">[62]</a>. Gene specific primers for <em>OSU1</em> were: ZZP87 (sense) 5′- CAGGATCC <span class="gene">ATGTCAATGCCACTACAACGTG</span> -3′ and ZZP88 (antisense) 5′- GTCTGCAG <span class="gene">TCAGATTGATTGTCGCTTGGTG</span> -3′. For <em>MYB75</em>, they were: GZP7 (sense) 5′-TTCCATGGAGGGTTCGTCCAAAGGGCTGC-3′, and GZP8 (antisense) 5′-GCTTCAGGAACCAAAATATCTACC-3′. Key parameters of PCR were provided in <a href="#pone.0001387.s002">Table S2</a>.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>For real-time quantitative PCR analysis, the QuantiTect SYBR Green PCR kit (Qiagen, USA) was used according to the instructions provided by the vender. Real-time PCR was carried out in the MasterCycler II (Cypheid, USA) according to the manufacturer's protocol, with key parameters of PCR provided in <a href="#pone.0001387.s002">Table S2</a>. The primers were designed using the software provided online by GenScript (<a href="http://www.genscript.com/bioinformatics.html">http://www.genscript.com/bioinformatics.​html</a>), and the <em>ACT2</em> primers (ActSDS and ActSDA) were designed previously <a href="#pone.0001387-Zheng1">[63]</a>. Gene-specific primers for <em>OSU1</em> were: ZZP89 (sense) 5′- CTTGCTGGTTCTTTCTGGTG-3′ and ZZP90 (antisense) 5′- TCTGAAACAAGCTGCTCCTG-3′. Primers for <em>MYB75</em> were: GZP91 (sense) 5′-GACTGCAACCATCTCAATGC-3′) and GZP92 (antisense) 5′-TTGGTCTTTCTTCTTATCTTTGTTG-3′). Primers for <em>MYB90</em> were: ZZP97 (sense) 5′- CAAGAAGCTGATGCGATTGT-3′ and ZZP98 (antisense) 5′- AACGTCAAACGCCAAAGTG-3′. Primers for <em>ASN1</em> were: ZZP83 (sense) 5′- ACTCTTTCATGGTGGCTCGT-3′ and ZZP84 (antisense) 5′- ACGTTTCGAAATGCTCACAG-3′.</p>


<h4>Production of <em>MYB75</em> transgenic plants</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The vector with the <em>UBQ10</em> promoter, designated ZZ106, was first constructed by replacing the CaMV 35S promoter of the vector pCAMBIA1301 (B4) with the PCR-amplified 1 kb promoter fragment of the <em>UBQ10</em> (At4g05320) gene <a href="#pone.0001387-Sun1">[48]</a>, using the primers ZZP6 (sense, 5′-TA<span class="underline">CTGCAG</span>ACGGATCAGGATATTCTTGT-3′) and ZZP7 (antisense, 5′-CGAT<span class="underline">CCATGG</span>CGGTAGAGAGAATTGAGAGA-3′) with the restriction sites underlined. For <em>MYB75</em>-overexpression and MYB75-SRDX constructs, the SRDX fragment containing twelve amino acids LDLDLELRLGFA encoded by part of the <em>SUP</em> gene as described in <a href="#pone.0001387-Hiratsu1">[47]</a> was first amplified with the primers ZZP48 (sense, <span class="gene">5′-GTAGATCTAGGT</span> ACCCTCGATCTGGATCTAGAACTCCGT-3′) and ZZP49 (antisense, 5′-GTGTTAAGCGAAACCCAAACGGAGTTC-3′) and then cloned into GZ10, resulting in the vector GZ36. The <em>UBQ10</em> promoter fragment was digested with <em>Pst</em> I (fill-in) and <em>Nco</em> I from ZZ106 and then cloned into GZ36, resulting in the vector GZ41. <em>MYB75</em> cDNA was then PCR amplified using the primers GZP7 (sense, 5′-TT<span class="underline">CCATGG</span>AGGGTTCGTCCAAAGGGCTGC-3′) and GZP9 (antisense for MYB75-SRDX, 5′-AT<span class="underline">GGTACC</span>ATCAAATTTCACAGTCTCTCCATCG-3′ that does not include the stop codon) and GZP10 (antisense for <em>MYB75</em>-overexpression, 5′- <span class="gene">AT<span class="underline">GGTACC</span>CTAATCAAATTTCACAGTCTC</span> TC-3′ that includes the stop codon TAG). These fragments were then cloned into GZ41, resulting in the vector GZ44 (MYB75-SRDX) and GZ45 (<em>MYB75</em>-overexpression), respectively. GZ44 and GZ45 were transformed into Col as described above.</p>


<h4>Statistical analysis</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>One way analysis of variance (ANOVA), <em>t</em>-test, and χ<sup>2</sup>-test were performed using the SigmaStat® 3.1 software (Systat Software, Inc., USA). For the multiple comparison procedure in one way ANOVA, the Fisher LSD method was used, according to the user's manual provided by the vendor.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001387"></div><a name="pone.0001387.s001" id="pone.0001387.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001387.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Primers used in the cloning of the <em>OSU1</em> mutant gene</p>
<p class="siDoi">doi:10.1371/journal.pone.0001387.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.10 MB DOC)</p>
<a name="pone.0001387.s002" id="pone.0001387.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001387.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Key parameters used in various RT-PCR reactions</p>
<p class="siDoi">doi:10.1371/journal.pone.0001387.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001387.s003" id="pone.0001387.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001387.s003">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI"><em>osu1-1</em> and <em>osu1-2</em> show similar hypersensitivity to the N-suppressed root growth in the absence of Suc (0C). Primary root lengths were measured after 7 days of vertical growth on agar-solidified media. The average of 7–8 seedlings is shown, with the bar representing the SD. Statistical analysis by one way ANOVA; the asterisk (*) above the column indicates a significant difference (p&lt;0.05) between <em>osu1-1</em> or <em>osu1-2</em> and wild-type (WT) under the same C/N condition.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001387.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.78 MB TIF)</p>
<a name="pone.0001387.s004" id="pone.0001387.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001387.s004">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI"><em>osu1-1</em> and <em>osu1-2</em> show very similar seed germination kinetic profiles as wild-type. Seeds were sown on four representative C/N conditions (0.5C/0.05N, 0.5C/4N, 15C/0.05N, and 15C/4N) and cold-treated for two days before transfer to an incubator with 16 hour light/8 hour dark at room temperature. Germination was scored every two hours, and no seeds germinated before the 22nd hour after incubation. The data shows the average and the SD bar of three replicates, each with about 40 seeds. WT, wild-type.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001387.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(2.24 MB TIF)</p>
<a name="pone.0001387.s005" id="pone.0001387.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001387.s005">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Gene tree analysis of the OSU1-related puative methyltransferase family. A gene tree for all of 29 members of OSU1-related putative methyltransferases (shown on the left) was constructed, using the IAMT1 (encoded by At5g55250) as an outgroup, to analyze the relationships between each member and group. The protein sequences were aligned using the ClustalW 1.8 multiple sequence alignment tool (<a href="http://searchlauncher.bcm.tmc.edu/multi-align/multi-align.html">http://searchlauncher.bcm.tmc.edu/multi-​align/multi-align.html</a>). The alignment result was then used to generate a tree file in the PHYLIP format, and a tree was generated using the rectangular cluster algorithm through the web tool of TreeTop-Phylogenetic Tree Prediction (<a href="http://www.genebee.msu.su/services/phtree_reduced.html">http://www.genebee.msu.su/services/phtre​e_reduced.html</a>), with bootstrap values provided. Shown on the right is the seven amino acid motif, WVMNVVP, for each of 29 members, with the arrow indicating the N560Y mutation site in the <em>osu1-2</em> allele.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001387.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.69 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to Zhenbiao Yang (University of California at Riverside) for insightful discussion of the C and N balance assay and to Yunde Zhao (University of California at San Diego) and Haiyang Wang (Boyce Thompson Institute) for their advice on map-based cloning. We thank Jyan-Chyun Jang (Ohio State University) for his helpful comments on this work and Dring N. Crowell (Indian University-Purdue University at Indianapolis) for his critical reading of the manuscript. We appreciate Joseph Rachlin (CUNY Lehman College) for his advice on gene tree analysis. We thank the Arabidopsis Biological Resources Center at the Ohio State University and the Nottingham Arabidopsis Stock Centre for providing various T-DNA lines. We are grateful to the editor and anonymous reviewers for their constructive comments on the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: ZZ PG. Performed the experiments: PG ZX. Analyzed the data: ZZ PG ZX. Contributed reagents/materials/analysis tools: ZZ PG. Wrote the paper: ZZ. Other: Helped with the manuscript writing: ZX PG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001387-Coruzzi1" id="pone.0001387-Coruzzi1"></a>Coruzzi G, Bush DR (2001) Nitrogen and carbon nutrient and metabolite signaling in plants. Plant Physiol  125: 61–64.  <ul class="find" data-citedArticleID="1037092" data-doi="10.1104/pp.125.1.61"><li><a href="http://dx.doi.org/10.1104/pp.125.1.61" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nitrogen+and+carbon+nutrient+and+metabolite+signaling+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nitrogen+and+carbon+nutrient+and+metabolite+signaling+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001387-Coruzzi2" id="pone.0001387-Coruzzi2"></a>Coruzzi GM, Zhou L (2001) Carbon and nitrogen sensing and signaling in plants: emerging ‘matrix effects’. Curr Opin Plant Biol  4: 247–253.  <ul class="find" data-citedArticleID="1037094" data-doi="10.1016/s1369-5266(00)00168-0"><li><a href="http://dx.doi.org/10.1016/s1369-5266(00)00168-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carbon+and+nitrogen+sensing+and+signaling+in+plants%3A+emerging+%E2%80%98matrix+effects%E2%80%99." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Carbon+and+nitrogen+sensing+and+signaling+in+plants%3A+emerging+%E2%80%98matrix+effects%E2%80%99.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001387-Malamy1" id="pone.0001387-Malamy1"></a>Malamy JE, Ryan KS (2001) Environmental regulation of lateral root initiation in Arabidopsis. Plant Physiol  127: 899–909.  <ul class="find" data-citedArticleID="1037144" data-doi="10.1104/pp.010406"><li><a href="http://dx.doi.org/10.1104/pp.010406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Environmental+regulation+of+lateral+root+initiation+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Environmental+regulation+of+lateral+root+initiation+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001387-Commichau1" id="pone.0001387-Commichau1"></a>Commichau FM, Forchhammer K, Stulke J (2006) Regulatory links between carbon and nitrogen metabolism. Curr Opin Microbiol  9: 167–172.  <ul class="find" data-citedArticleID="1037090" data-doi="10.1016/j.mib.2006.01.001"><li><a href="http://dx.doi.org/10.1016/j.mib.2006.01.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulatory+links+between+carbon+and+nitrogen+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulatory+links+between+carbon+and+nitrogen+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001387-Marshall1" id="pone.0001387-Marshall1"></a>Marshall S (2006) Role of insulin, adipocyte hormones, and nutrient-sensing pathways in regulating fuel metabolism and energy homeostasis: a nutritional perspective of diabetes, obesity, and cancer. Sci STKE  2006: re7.  <ul class="find" data-citedArticleID="1037146" data-doi="10.1126/stke.3462006re7"><li><a href="http://dx.doi.org/10.1126/stke.3462006re7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+insulin%2C+adipocyte+hormones%2C+and+nutrient-sensing+pathways+in+regulating+fuel+metabolism+and+energy+homeostasis%3A+a+nutritional+perspective+of+diabetes%2C+obesity%2C+and+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+insulin%2C+adipocyte+hormones%2C+and+nutrient-sensing+pathways+in+regulating+fuel+metabolism+and+energy+homeostasis%3A+a+nutritional+perspective+of+diabetes%2C+obesity%2C+and+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001387-Martin1" id="pone.0001387-Martin1"></a>Martin T, Oswald O, Graham IA (2002) Arabidopsis seedling growth, storage lipid mobilization, and photosynthetic gene expression are regulated by carbon:nitrogen availability. Plant Physiol  128: 472–481.  <ul class="find" data-citedArticleID="1037148" data-doi="10.1104/pp.010475"><li><a href="http://dx.doi.org/10.1104/pp.010475" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+seedling+growth%2C+storage+lipid+mobilization%2C+and+photosynthetic+gene+expression+are+regulated+by+carbon%3Anitrogen+availability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+seedling+growth%2C+storage+lipid+mobilization%2C+and+photosynthetic+gene+expression+are+regulated+by+carbon%3Anitrogen+availability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001387-Palenchar1" id="pone.0001387-Palenchar1"></a>Palenchar PM, Kouranov A, Lejay LV, Coruzzi GM (2004) Genome-wide patterns of carbon and nitrogen regulation of gene expression validate the combined carbon and nitrogen (CN)-signaling hypothesis in plants. Genome Biol  5: R91.  <ul class="find" data-citedArticleID="1037166" data-doi="10.1186/gb-2004-5-11-r91"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-11-r91" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+patterns+of+carbon+and+nitrogen+regulation+of+gene+expression+validate+the+combined+carbon+and+nitrogen+%28CN%29-signaling+hypothesis+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+patterns+of+carbon+and+nitrogen+regulation+of+gene+expression+validate+the+combined+carbon+and+nitrogen+%28CN%29-signaling+hypothesis+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001387-Price1" id="pone.0001387-Price1"></a>Price J, Laxmi A, St Martin SK, Jang JC (2004) Global transcription profiling reveals multiple sugar signal transduction mechanisms in Arabidopsis. Plant Cell  16: 2128–2150.  <ul class="find" data-citedArticleID="1037170" data-doi="10.1105/tpc.104.022616"><li><a href="http://dx.doi.org/10.1105/tpc.104.022616" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+transcription+profiling+reveals+multiple+sugar+signal+transduction+mechanisms+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+transcription+profiling+reveals+multiple+sugar+signal+transduction+mechanisms+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001387-Avruch1" id="pone.0001387-Avruch1"></a>Avruch J, Lin Y, Long X, Murthy S, Ortiz-Vega S (2005) Recent advances in the regulation of the TOR pathway by insulin and nutrients. Curr Opin Clin Nutr Metab Care  8: 67–72.  <ul class="find" data-citedArticleID="1037080" data-doi="10.1097/00075197-200501000-00010"><li><a href="http://dx.doi.org/10.1097/00075197-200501000-00010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+advances+in+the+regulation+of+the+TOR+pathway+by+insulin+and+nutrients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+advances+in+the+regulation+of+the+TOR+pathway+by+insulin+and+nutrients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001387-Schneper1" id="pone.0001387-Schneper1"></a>Schneper L, Duvel K, Broach JR (2004) Sense and sensibility: nutritional response and signal integration in yeast. Curr Opin Microbiol  7: 624–630.  <ul class="find" data-citedArticleID="1037182" data-doi="10.1016/j.mib.2004.10.002"><li><a href="http://dx.doi.org/10.1016/j.mib.2004.10.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sense+and+sensibility%3A+nutritional+response+and+signal+integration+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sense+and+sensibility%3A+nutritional+response+and+signal+integration+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001387-Xue1" id="pone.0001387-Xue1"></a>Xue C, Bahn YS, Cox GM, Heitman J (2006) G protein-coupled receptor Gpr4 senses amino acids and activates the cAMP-PKA pathway in <em>Cryptococcus neoformans</em>. Mol Biol Cell  17: 667–679.  <ul class="find" data-citedArticleID="1037196" data-doi="10.1091/mbc.e05-07-0699"><li><a href="http://dx.doi.org/10.1091/mbc.e05-07-0699" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=G+protein-coupled+receptor+Gpr4+senses+amino+acids+and+activates+the+cAMP-PKA+pathway+in+Cryptococcus+neoformans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22G+protein-coupled+receptor+Gpr4+senses+amino+acids+and+activates+the+cAMP-PKA+pathway+in+Cryptococcus+neoformans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001387-Bertram1" id="pone.0001387-Bertram1"></a>Bertram PG, Choi JH, Carvalho J, Chan TF, Ai W, et al.  (2002) Convergence of TOR-nitrogen and Snf1-glucose signaling pathways onto Gln3. Mol Cell Biol  22: 1246–1252.  <ul class="find" data-citedArticleID="1037082" data-doi="10.1128/mcb.22.4.1246-1252.2002"><li><a href="http://dx.doi.org/10.1128/mcb.22.4.1246-1252.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Convergence+of+TOR-nitrogen+and+Snf1-glucose+signaling+pathways+onto+Gln3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Convergence+of+TOR-nitrogen+and+Snf1-glucose+signaling+pathways+onto+Gln3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001387-Peter1" id="pone.0001387-Peter1"></a>Peter GJ, During L, Ahmed A (2006) Carbon catabolite repression regulates amino acid permeases in <em>Saccharomyces cerevisiae</em> via the TOR signaling pathway. J Biol Chem  281: 5546–5552.  <ul class="find" data-citedArticleID="1037168" data-doi="10.1074/jbc.m513842200"><li><a href="http://dx.doi.org/10.1074/jbc.m513842200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carbon+catabolite+repression+regulates+amino+acid+permeases+in+Saccharomyces+cerevisiae+via+the+TOR+signaling+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Carbon+catabolite+repression+regulates+amino+acid+permeases+in+Saccharomyces+cerevisiae+via+the+TOR+signaling+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001387-Koch1" id="pone.0001387-Koch1"></a>Koch KE (1996) Carbohydrate-modulated gene expression in plants. Annu Rev Plant Physiol Plant Mol Biol  47: 509–540.  <ul class="find" data-citedArticleID="1037124" data-doi="10.1146/annurev.arplant.47.1.509"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.47.1.509" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carbohydrate-modulated+gene+expression+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Carbohydrate-modulated+gene+expression+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001387-Rolland1" id="pone.0001387-Rolland1"></a>Rolland F, Baena-Gonzalez E, Sheen J (2006) Sugar sensing and signaling in plants: conserved and novel mechanisms. Annu Rev Plant Biol  57: 675–709.  <ul class="find" data-citedArticleID="1037174" data-doi="10.1146/annurev.arplant.57.032905.105441"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.57.032905.105441" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sugar+sensing+and+signaling+in+plants%3A+conserved+and+novel+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sugar+sensing+and+signaling+in+plants%3A+conserved+and+novel+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001387-Smeekens1" id="pone.0001387-Smeekens1"></a>Smeekens S (2000) Sugar-Induced Signal Transduction in Plants. Annu Rev Plant Physiol Plant Mol Biol  51: 49–81.  <ul class="find" data-citedArticleID="1037184" data-doi="10.1146/annurev.arplant.51.1.49"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.51.1.49" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sugar-Induced+Signal+Transduction+in+Plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sugar-Induced+Signal+Transduction+in+Plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001387-Coruzzi3" id="pone.0001387-Coruzzi3"></a>Coruzzi G (2003)  Primary N-assimilation into amino acids in <em>Arabidopsis</em>. In: Somerville CR, Meyerowitz EM, editors. The Arabidopsis Book. Rockville, MD: American Society of Plant Biologists.   DOI: 10.1199/tab.0010, <a href="http://www.aspb.org/publications/arabidopsis/">www.aspb.org/publications/arabidopsis/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0001387-Forde1" id="pone.0001387-Forde1"></a>Forde BG (2002) Local and long-range signaling pathways regulating plant responses to nitrate. Annu Rev Plant Biol  53: 203–224.  <ul class="find" data-citedArticleID="1037104"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Local+and+long-range+signaling+pathways+regulating+plant+responses+to+nitrate.&amp;auth=&amp;atitle=Local+and+long-range+signaling+pathways+regulating+plant+responses+to+nitrate." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Local+and+long-range+signaling+pathways+regulating+plant+responses+to+nitrate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Local+and+long-range+signaling+pathways+regulating+plant+responses+to+nitrate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001387-Lam1" id="pone.0001387-Lam1"></a>Lam HM, Coschigano KT, Oliveira IC, Melo-Oliveira R, Coruzzi GM (1996) The molecular-genetics of nitrogen assimilation into amino acids in higher plants. Annu Rev Plant Physiol Plant Mol Biol  47: 569–593.  <ul class="find" data-citedArticleID="1037128" data-doi="10.1146/annurev.arplant.47.1.569"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.47.1.569" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular-genetics+of+nitrogen+assimilation+into+amino+acids+in+higher+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular-genetics+of+nitrogen+assimilation+into+amino+acids+in+higher+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001387-Gutierrez1" id="pone.0001387-Gutierrez1"></a>Gutierrez RA, Lejay LV, Dean A, Chiaromonte F, Shasha DE, et al.  (2007) Qualitative network models and genome-wide expression data define carbon/nitrogen-responsive molecular machines in Arabidopsis. Genome Biol  8: R7.  <ul class="find" data-citedArticleID="1037106"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Qualitative+network+models+and+genome-wide+expression+data+define+carbon%2Fnitrogen-responsive+molecular+machines+in+Arabidopsis.&amp;auth=&amp;atitle=Qualitative+network+models+and+genome-wide+expression+data+define+carbon%2Fnitrogen-responsive+molecular+machines+in+Arabidopsis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Qualitative+network+models+and+genome-wide+expression+data+define+carbon%2Fnitrogen-responsive+molecular+machines+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Qualitative+network+models+and+genome-wide+expression+data+define+carbon%2Fnitrogen-responsive+molecular+machines+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001387-Lam2" id="pone.0001387-Lam2"></a>Lam HM, Hsieh MH, Coruzzi G (1998) Reciprocal regulation of distinct asparagine synthetase genes by light and metabolites in <em>Arabidopsis thaliana</em>. Plant J  16: 345–353.  <ul class="find" data-citedArticleID="1037130" data-doi="10.1046/j.1365-313x.1998.00302.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1998.00302.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reciprocal+regulation+of+distinct+asparagine+synthetase+genes+by+light+and+metabolites+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reciprocal+regulation+of+distinct+asparagine+synthetase+genes+by+light+and+metabolites+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001387-Lam3" id="pone.0001387-Lam3"></a>Lam HM, Peng SS, Coruzzi GM (1994) Metabolic regulation of the gene encoding glutamine-dependent asparagine synthetase in <em>Arabidopsis thaliana</em>. Plant Physiol  106: 1347–1357.  <ul class="find" data-citedArticleID="1037132" data-doi="10.1104/pp.106.4.1347"><li><a href="http://dx.doi.org/10.1104/pp.106.4.1347" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolic+regulation+of+the+gene+encoding+glutamine-dependent+asparagine+synthetase+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolic+regulation+of+the+gene+encoding+glutamine-dependent+asparagine+synthetase+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001387-MeloOliveira1" id="pone.0001387-MeloOliveira1"></a>Melo-Oliveira R, Oliveira IC, Coruzzi GM (1996) Arabidopsis mutant analysis and gene regulation define a nonredundant role for glutamate dehydrogenase in nitrogen assimilation. Proc Natl Acad Sci U S A  93: 4718–4723.  <ul class="find" data-citedArticleID="1037154" data-doi="10.1073/pnas.93.10.4718"><li><a href="http://dx.doi.org/10.1073/pnas.93.10.4718" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+mutant+analysis+and+gene+regulation+define+a+nonredundant+role+for+glutamate+dehydrogenase+in+nitrogen+assimilation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+mutant+analysis+and+gene+regulation+define+a+nonredundant+role+for+glutamate+dehydrogenase+in+nitrogen+assimilation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001387-Scheible1" id="pone.0001387-Scheible1"></a>Scheible WR, Morcuende R, Czechowski T, Fritz C, Osuna D, et al.  (2004) Genome-wide reprogramming of primary and secondary metabolism, protein synthesis, cellular growth processes, and the regulatory infrastructure of Arabidopsis in response to nitrogen. Plant Physiol  136: 2483–2499.  <ul class="find" data-citedArticleID="1037180" data-doi="10.1104/pp.104.047019"><li><a href="http://dx.doi.org/10.1104/pp.104.047019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+reprogramming+of+primary+and+secondary+metabolism%2C+protein+synthesis%2C+cellular+growth+processes%2C+and+the+regulatory+infrastructure+of+Arabidopsis+in+response+to+nitrogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+reprogramming+of+primary+and+secondary+metabolism%2C+protein+synthesis%2C+cellular+growth+processes%2C+and+the+regulatory+infrastructure+of+Arabidopsis+in+response+to+nitrogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001387-Wang1" id="pone.0001387-Wang1"></a>Wang R, Okamoto M, Xing X, Crawford NM (2003) Microarray analysis of the nitrate response in Arabidopsis roots and shoots reveals over 1,000 rapidly responding genes and new linkages to glucose, trehalose-6-phosphate, iron, and sulfate metabolism. Plant Physiol  132: 556–567.  <ul class="find" data-citedArticleID="1037192" data-doi="10.1104/pp.103.021253"><li><a href="http://dx.doi.org/10.1104/pp.103.021253" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+analysis+of+the+nitrate+response+in+Arabidopsis+roots+and+shoots+reveals+over+1%2C000+rapidly+responding+genes+and+new+linkages+to+glucose%2C+trehalose-6-phosphate%2C+iron%2C+and+sulfate+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+analysis+of+the+nitrate+response+in+Arabidopsis+roots+and+shoots+reveals+over+1%2C000+rapidly+responding+genes+and+new+linkages+to+glucose%2C+trehalose-6-phosphate%2C+iron%2C+and+sulfate+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001387-Ishiyama1" id="pone.0001387-Ishiyama1"></a>Ishiyama K, Inoue E, Watanabe-Takahashi A, Obara M, Yamaya T, et al.  (2004) Kinetic properties and ammonium-dependent regulation of cytosolic isoenzymes of glutamine synthetase in <em>Arabidopsis</em>. J Biol Chem  279: 16598–16605.  <ul class="find" data-citedArticleID="1037114" data-doi="10.1074/jbc.m313710200"><li><a href="http://dx.doi.org/10.1074/jbc.m313710200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kinetic+properties+and+ammonium-dependent+regulation+of+cytosolic+isoenzymes+of+glutamine+synthetase+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kinetic+properties+and+ammonium-dependent+regulation+of+cytosolic+isoenzymes+of+glutamine+synthetase+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001387-Lam4" id="pone.0001387-Lam4"></a>Lam HM, Wong P, Chan HK, Yam KM, Chen L, et al.  (2003) Overexpression of the <em>ASN1</em> gene enhances nitrogen status in seeds of Arabidopsis. Plant Physiol  132: 926–935.  <ul class="find" data-citedArticleID="1037134" data-doi="10.1104/pp.103.020123"><li><a href="http://dx.doi.org/10.1104/pp.103.020123" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overexpression+of+the+ASN1+gene+enhances+nitrogen+status+in+seeds+of+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overexpression+of+the+ASN1+gene+enhances+nitrogen+status+in+seeds+of+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001387-Oliveira1" id="pone.0001387-Oliveira1"></a>Oliveira IC, Coruzzi GM (1999) Carbon and amino acids reciprocally modulate the expression of glutamine synthetase in <em>Arabidopsis</em>. Plant Physiol  121: 301–310.  <ul class="find" data-citedArticleID="1037164" data-doi="10.1104/pp.121.1.301"><li><a href="http://dx.doi.org/10.1104/pp.121.1.301" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carbon+and+amino+acids+reciprocally+modulate+the+expression+of+glutamine+synthetase+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Carbon+and+amino+acids+reciprocally+modulate+the+expression+of+glutamine+synthetase+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001387-Little1" id="pone.0001387-Little1"></a>Little DY, Rao H, Oliva S, Daniel-Vedele F, Krapp A, et al.  (2005) The putative high-affinity nitrate transporter NRT2.1 represses lateral root initiation in response to nutritional cues. Proc Natl Acad Sci U S A  102: 13693–13698.  <ul class="find" data-citedArticleID="1037138" data-doi="10.1073/pnas.0504219102"><li><a href="http://dx.doi.org/10.1073/pnas.0504219102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+putative+high-affinity+nitrate+transporter+NRT2.1+represses+lateral+root+initiation+in+response+to+nutritional+cues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+putative+high-affinity+nitrate+transporter+NRT2.1+represses+lateral+root+initiation+in+response+to+nutritional+cues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001387-Hsieh1" id="pone.0001387-Hsieh1"></a>Hsieh MH, Lam HM, van de Loo FJ, Coruzzi G (1998) A PII-like protein in Arabidopsis: putative role in nitrogen sensing. Proc Natl Acad Sci U S A  95: 13965–13970.  <ul class="find" data-citedArticleID="1037110" data-doi="10.1073/pnas.95.23.13965"><li><a href="http://dx.doi.org/10.1073/pnas.95.23.13965" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+PII-like+protein+in+Arabidopsis%3A+putative+role+in+nitrogen+sensing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+PII-like+protein+in+Arabidopsis%3A+putative+role+in+nitrogen+sensing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001387-Moorhead1" id="pone.0001387-Moorhead1"></a>Moorhead GB, Smith CS (2003) Interpreting the plastid carbon, nitrogen, and energy status. A role for PII? Plant Physiol  133: 492–498.  <ul class="find" data-citedArticleID="1037160" data-doi="10.1104/pp.103.025627"><li><a href="http://dx.doi.org/10.1104/pp.103.025627" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interpreting+the+plastid+carbon%2C+nitrogen%2C+and+energy+status.+A+role+for+PII%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interpreting+the+plastid+carbon%2C+nitrogen%2C+and+energy+status.+A+role+for+PII%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001387-FerrarioMery1" id="pone.0001387-FerrarioMery1"></a>Ferrario-Mery S, Bouvet M, Leleu O, Savino G, Hodges M, et al.  (2005) Physiological characterisation of Arabidopsis mutants affected in the expression of the putative regulatory protein PII. Planta  223: 28–39.  <ul class="find" data-citedArticleID="1037100" data-doi="10.1007/s00425-005-0063-5"><li><a href="http://dx.doi.org/10.1007/s00425-005-0063-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physiological+characterisation+of+Arabidopsis+mutants+affected+in+the+expression+of+the+putative+regulatory+protein+PII." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physiological+characterisation+of+Arabidopsis+mutants+affected+in+the+expression+of+the+putative+regulatory+protein+PII.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001387-Menand1" id="pone.0001387-Menand1"></a>Menand B, Desnos T, Nussaume L, Berger F, Bouchez D, et al.  (2002) Expression and disruption of the Arabidopsis TOR (target of rapamycin) gene. Proc Natl Acad Sci U S A  99: 6422–6427.  <ul class="find" data-citedArticleID="1037156" data-doi="10.1073/pnas.092141899"><li><a href="http://dx.doi.org/10.1073/pnas.092141899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+and+disruption+of+the+Arabidopsis+TOR+%28target+of+rapamycin%29+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+and+disruption+of+the+Arabidopsis+TOR+%28target+of+rapamycin%29+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001387-Anderson1" id="pone.0001387-Anderson1"></a>Anderson GH, Veit B, Hanson MR (2005) The Arabidopsis AtRaptor genes are essential for post-embryonic plant growth. BMC Biol  3: 12.  <ul class="find" data-citedArticleID="1037076"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Arabidopsis+AtRaptor+genes+are+essential+for+post-embryonic+plant+growth.&amp;auth=&amp;atitle=The+Arabidopsis+AtRaptor+genes+are+essential+for+post-embryonic+plant+growth." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Arabidopsis+AtRaptor+genes+are+essential+for+post-embryonic+plant+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Arabidopsis+AtRaptor+genes+are+essential+for+post-embryonic+plant+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001387-Mahfouz1" id="pone.0001387-Mahfouz1"></a>Mahfouz MM, Kim S, Delauney AJ, Verma DP (2006) Arabidopsis TARGET OF RAPAMYCIN interacts with RAPTOR, which regulates the activity of S6 kinase in response to osmotic stress signals. Plant Cell  18: 477–490.  <ul class="find" data-citedArticleID="1037142" data-doi="10.1105/tpc.105.035931"><li><a href="http://dx.doi.org/10.1105/tpc.105.035931" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+TARGET+OF+RAPAMYCIN+interacts+with+RAPTOR%2C+which+regulates+the+activity+of+S6+kinase+in+response+to+osmotic+stress+signals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+TARGET+OF+RAPAMYCIN+interacts+with+RAPTOR%2C+which+regulates+the+activity+of+S6+kinase+in+response+to+osmotic+stress+signals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001387-Krupkova1" id="pone.0001387-Krupkova1"></a>Krupkova E, Immerzeel P, Pauly M, Schmulling T (2007) The <em>TUMOROUS SHOOT DEVELOPMENT2</em> gene of Arabidopsis encoding a putative methyltransferase is required for cell adhesion and co-ordinated plant development. Plant J  50: 735–750.  <ul class="find" data-citedArticleID="1037126" data-doi="10.1111/j.1365-313x.2007.03123.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2007.03123.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+TUMOROUS+SHOOT+DEVELOPMENT2+gene+of+Arabidopsis+encoding+a+putative+methyltransferase+is+required+for+cell+adhesion+and+co-ordinated+plant+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+TUMOROUS+SHOOT+DEVELOPMENT2+gene+of+Arabidopsis+encoding+a+putative+methyltransferase+is+required+for+cell+adhesion+and+co-ordinated+plant+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001387-Mouille1" id="pone.0001387-Mouille1"></a>Mouille G, Ralet MC, Cavelier C, Eland C, Effroy D, et al.  (2007) Homogalacturonan synthesis in <em>Arabidopsis thaliana</em> requires a Golgi-localized protein with a putative methyltransferase domain. Plant J  50: 605–614.  <ul class="find" data-citedArticleID="1037162" data-doi="10.1111/j.1365-313x.2007.03086.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2007.03086.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homogalacturonan+synthesis+in+Arabidopsis+thaliana+requires+a+Golgi-localized+protein+with+a+putative+methyltransferase+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homogalacturonan+synthesis+in+Arabidopsis+thaliana+requires+a+Golgi-localized+protein+with+a+putative+methyltransferase+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001387-Finkelstein1" id="pone.0001387-Finkelstein1"></a>Finkelstein RR, Lynch TJ (2000) The Arabidopsis abscisic acid response gene <em>ABI5</em> encodes a basic leucine zipper transcription factor. Plant Cell  12: 599–609.  <ul class="find" data-citedArticleID="1037102" data-doi="10.1105/tpc.12.4.599"><li><a href="http://dx.doi.org/10.1105/tpc.12.4.599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Arabidopsis+abscisic+acid+response+gene+ABI5+encodes+a+basic+leucine+zipper+transcription+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Arabidopsis+abscisic+acid+response+gene+ABI5+encodes+a+basic+leucine+zipper+transcription+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001387-Teng1" id="pone.0001387-Teng1"></a>Teng S, Keurentjes J, Bentsink L, Koornneef M, Smeekens S (2005) Sucrose-specific induction of anthocyanin biosynthesis in Arabidopsis requires the MYB75/PAP1 gene. Plant Physiol  139: 1840–1852.  <ul class="find" data-citedArticleID="1037188" data-doi="10.1104/pp.105.066688"><li><a href="http://dx.doi.org/10.1104/pp.105.066688" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sucrose-specific+induction+of+anthocyanin+biosynthesis+in+Arabidopsis+requires+the+MYB75%2FPAP1+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sucrose-specific+induction+of+anthocyanin+biosynthesis+in+Arabidopsis+requires+the+MYB75%2FPAP1+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001387-Dan1" id="pone.0001387-Dan1"></a>Dan H, Yang G, Zheng ZL (2007) A negative regulatory role for auxin in sulphate deficiency response in <em>Arabidopsis thaliana</em>. Plant Mol Biol  63: 221–235.  <ul class="find" data-citedArticleID="1037098" data-doi="10.1007/s11103-006-9084-0"><li><a href="http://dx.doi.org/10.1007/s11103-006-9084-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+negative+regulatory+role+for+auxin+in+sulphate+deficiency+response+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+negative+regulatory+role+for+auxin+in+sulphate+deficiency+response+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001387-Karthikeyan1" id="pone.0001387-Karthikeyan1"></a>Karthikeyan AS, Varadarajan DK, Jain A, Held MA, Carpita NC, et al.  (2007) Phosphate starvation responses are mediated by sugar signaling in <em>Arabidopsis</em>. Planta  225: 907–918.  <ul class="find" data-citedArticleID="1037120" data-doi="10.1007/s00425-006-0408-8"><li><a href="http://dx.doi.org/10.1007/s00425-006-0408-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphate+starvation+responses+are+mediated+by+sugar+signaling+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphate+starvation+responses+are+mediated+by+sugar+signaling+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001387-Rubio1" id="pone.0001387-Rubio1"></a>Rubio V, Linhares F, Solano R, Martin AC, Iglesias J, et al.  (2001) A conserved MYB transcription factor involved in phosphate starvation signaling both in vascular plants and in unicellular algae. Genes Dev  15: 2122–2133.  <ul class="find" data-citedArticleID="1037178" data-doi="10.1101/gad.204401"><li><a href="http://dx.doi.org/10.1101/gad.204401" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+conserved+MYB+transcription+factor+involved+in+phosphate+starvation+signaling+both+in+vascular+plants+and+in+unicellular+algae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+conserved+MYB+transcription+factor+involved+in+phosphate+starvation+signaling+both+in+vascular+plants+and+in+unicellular+algae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001387-Borevitz1" id="pone.0001387-Borevitz1"></a>Borevitz JO, Xia Y, Blount J, Dixon RA, Lamb C (2000) Activation tagging identifies a conserved MYB regulator of phenylpropanoid biosynthesis. Plant Cell  12: 2383–2394.  <ul class="find" data-citedArticleID="1037084" data-doi="10.2307/3871236"><li><a href="http://dx.doi.org/10.2307/3871236" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+tagging+identifies+a+conserved+MYB+regulator+of+phenylpropanoid+biosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+tagging+identifies+a+conserved+MYB+regulator+of+phenylpropanoid+biosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001387-Lea1" id="pone.0001387-Lea1"></a>Lea US, Slimestad R, Smedvig P, Lillo C (2007) Nitrogen deficiency enhances expression of specific MYB and bHLH transcription factors and accumulation of end products in the flavonoid pathway. Planta  225: 1245–1253.  <ul class="find" data-citedArticleID="1037136" data-doi="10.1007/s00425-006-0414-x"><li><a href="http://dx.doi.org/10.1007/s00425-006-0414-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nitrogen+deficiency+enhances+expression+of+specific+MYB+and+bHLH+transcription+factors+and+accumulation+of+end+products+in+the+flavonoid+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nitrogen+deficiency+enhances+expression+of+specific+MYB+and+bHLH+transcription+factors+and+accumulation+of+end+products+in+the+flavonoid+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001387-Matsui1" id="pone.0001387-Matsui1"></a>Matsui K, Tanaka H, Ohme-Takagi M (2004) Suppression of the biosynthesis of proanthocyanidin in <em>Arabidopsis</em> by a chimeric PAP1 repressor. Plant Biotechnol J  2: 487–493.  <ul class="find" data-citedArticleID="1037152" data-doi="10.1111/j.1467-7652.2004.00094.x"><li><a href="http://dx.doi.org/10.1111/j.1467-7652.2004.00094.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suppression+of+the+biosynthesis+of+proanthocyanidin+in+Arabidopsis+by+a+chimeric+PAP1+repressor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suppression+of+the+biosynthesis+of+proanthocyanidin+in+Arabidopsis+by+a+chimeric+PAP1+repressor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001387-Tohge1" id="pone.0001387-Tohge1"></a>Tohge T, Nishiyama Y, Hirai MY, Yano M, Nakajima J, et al.  (2005) Functional genomics by integrated analysis of metabolome and transcriptome of Arabidopsis plants over-expressing an MYB transcription factor. Plant J  42: 218–235.  <ul class="find" data-citedArticleID="1037190" data-doi="10.1111/j.1365-313x.2005.02371.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2005.02371.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+genomics+by+integrated+analysis+of+metabolome+and+transcriptome+of+Arabidopsis+plants+over-expressing+an+MYB+transcription+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+genomics+by+integrated+analysis+of+metabolome+and+transcriptome+of+Arabidopsis+plants+over-expressing+an+MYB+transcription+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001387-Hiratsu1" id="pone.0001387-Hiratsu1"></a>Hiratsu K, Matsui K, Koyama T, Ohme-Takagi M (2003) Dominant repression of target genes by chimeric repressors that include the EAR motif, a repression domain, in <em>Arabidopsis</em>. Plant J  34: 733–739.  <ul class="find" data-citedArticleID="1037108" data-doi="10.1046/j.1365-313x.2003.01759.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.2003.01759.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dominant+repression+of+target+genes+by+chimeric+repressors+that+include+the+EAR+motif%2C+a+repression+domain%2C+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dominant+repression+of+target+genes+by+chimeric+repressors+that+include+the+EAR+motif%2C+a+repression+domain%2C+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001387-Sun1" id="pone.0001387-Sun1"></a>Sun CW, Callis J (1997) Independent modulation of <em>Arabidopsis thaliana</em> polyubiquitin mRNAs in different organs and in response to environmental changes. Plant J  11: 1017–1027.  <ul class="find" data-citedArticleID="1037186" data-doi="10.1046/j.1365-313x.1997.11051017.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1997.11051017.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Independent+modulation+of+Arabidopsis+thaliana+polyubiquitin+mRNAs+in+different+organs+and+in+response+to+environmental+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Independent+modulation+of+Arabidopsis+thaliana+polyubiquitin+mRNAs+in+different+organs+and+in+response+to+environmental+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001387-Lukowitz1" id="pone.0001387-Lukowitz1"></a>Lukowitz W, Gillmor CS, Scheible WR (2000) Positional cloning in Arabidopsis. Why it feels good to have a genome initiative working for you. Plant Physiol  123: 795–805.  <ul class="find" data-citedArticleID="1037140" data-doi="10.1104/pp.123.3.795"><li><a href="http://dx.doi.org/10.1104/pp.123.3.795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positional+cloning+in+Arabidopsis.+Why+it+feels+good+to+have+a+genome+initiative+working+for+you." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positional+cloning+in+Arabidopsis.+Why+it+feels+good+to+have+a+genome+initiative+working+for+you.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001387-Kiyosue1" id="pone.0001387-Kiyosue1"></a>Kiyosue T, Yamaguchi-Shinozaki K, Shinozaki K (1994) Cloning of cDNAs for genes that are early-responsive to dehydration stress (ERDs) in <em>Arabidopsis thaliana</em> L.: identification of three ERDs as HSP cognate genes. Plant Mol Biol  25: 791–798.  <ul class="find" data-citedArticleID="1037122" data-doi="10.1007/bf00028874"><li><a href="http://dx.doi.org/10.1007/bf00028874" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+of+cDNAs+for+genes+that+are+early-responsive+to+dehydration+stress+%28ERDs%29+in+Arabidopsis+thaliana+L.%3A+identification+of+three+ERDs+as+HSP+cognate+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+of+cDNAs+for+genes+that+are+early-responsive+to+dehydration+stress+%28ERDs%29+in+Arabidopsis+thaliana+L.%3A+identification+of+three+ERDs+as+HSP+cognate+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001387-Qin1" id="pone.0001387-Qin1"></a>Qin G, Gu H, Zhao Y, Ma Z, Shi G, et al.  (2005) An indole-3-acetic acid carboxyl methyltransferase regulates Arabidopsis leaf development. Plant Cell  17: 2693–2704.  <ul class="find" data-citedArticleID="1037172" data-doi="10.1105/tpc.105.034959"><li><a href="http://dx.doi.org/10.1105/tpc.105.034959" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+indole-3-acetic+acid+carboxyl+methyltransferase+regulates+Arabidopsis+leaf+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+indole-3-acetic+acid+carboxyl+methyltransferase+regulates+Arabidopsis+leaf+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001387-Cho1" id="pone.0001387-Cho1"></a>Cho YH, Yoo SD, Sheen J (2006) Regulatory functions of nuclear hexokinase1 complex in glucose signaling. Cell  127: 579–589.  <ul class="find" data-citedArticleID="1037086" data-doi="10.1016/j.cell.2006.09.028"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.09.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulatory+functions+of+nuclear+hexokinase1+complex+in+glucose+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulatory+functions+of+nuclear+hexokinase1+complex+in+glucose+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001387-Jang1" id="pone.0001387-Jang1"></a>Jang JC, Leon P, Zhou L, Sheen J (1997) Hexokinase as a sugar sensor in higher plants. Plant Cell  9: 5–19.  <ul class="find" data-citedArticleID="1037116" data-doi="10.2307/3870367"><li><a href="http://dx.doi.org/10.2307/3870367" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hexokinase+as+a+sugar+sensor+in+higher+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hexokinase+as+a+sugar+sensor+in+higher+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001387-Moore1" id="pone.0001387-Moore1"></a>Moore B, Zhou L, Rolland F, Hall Q, Cheng WH, et al.  (2003) Role of the <em>Arabidopsis</em> glucose sensor HXK1 in nutrient, light, and hormonal signaling. Science  300: 332–336.  <ul class="find" data-citedArticleID="1037158" data-doi="10.1126/science.1080585"><li><a href="http://dx.doi.org/10.1126/science.1080585" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+Arabidopsis+glucose+sensor+HXK1+in+nutrient%2C+light%2C+and+hormonal+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+Arabidopsis+glucose+sensor+HXK1+in+nutrient%2C+light%2C+and+hormonal+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001387-Huang1" id="pone.0001387-Huang1"></a>Huang J, Taylor JP, Chen JG, Uhrig JF, Schnell DJ, et al.  (2006) The plastid protein THYLAKOID FORMATION1 and the plasma membrane G-protein GPA1 interact in a novel sugar-signaling mechanism in Arabidopsis. Plant Cell  18: 1226–1238.  <ul class="find" data-citedArticleID="1037112" data-doi="10.1105/tpc.105.037259"><li><a href="http://dx.doi.org/10.1105/tpc.105.037259" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+plastid+protein+THYLAKOID+FORMATION1+and+the+plasma+membrane+G-protein+GPA1+interact+in+a+novel+sugar-signaling+mechanism+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+plastid+protein+THYLAKOID+FORMATION1+and+the+plasma+membrane+G-protein+GPA1+interact+in+a+novel+sugar-signaling+mechanism+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001387-Zhang1" id="pone.0001387-Zhang1"></a>Zhang H, Forde BG (1998) An Arabidopsis MADS box gene that controls nutrient-induced changes in root architecture. Science  279: 407–409.  <ul class="find" data-citedArticleID="1037198" data-doi="10.1126/science.279.5349.407"><li><a href="http://dx.doi.org/10.1126/science.279.5349.407" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Arabidopsis+MADS+box+gene+that+controls+nutrient-induced+changes+in+root+architecture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Arabidopsis+MADS+box+gene+that+controls+nutrient-induced+changes+in+root+architecture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001387-Avonce1" id="pone.0001387-Avonce1"></a>Avonce N, Leyman B, Mascorro-Gallardo JO, Van Dijck P, Thevelein JM, et al.  (2004) The Arabidopsis trehalose-6-P synthase AtTPS1 gene is a regulator of glucose, abscisic acid, and stress signaling. Plant Physiol  136: 3649–3659.  <ul class="find" data-citedArticleID="1037078" data-doi="10.1104/pp.104.052084"><li><a href="http://dx.doi.org/10.1104/pp.104.052084" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Arabidopsis+trehalose-6-P+synthase+AtTPS1+gene+is+a+regulator+of+glucose%2C+abscisic+acid%2C+and+stress+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Arabidopsis+trehalose-6-P+synthase+AtTPS1+gene+is+a+regulator+of+glucose%2C+abscisic+acid%2C+and+stress+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001387-Martin2" id="pone.0001387-Martin2"></a>Martin JL, McMillan FM (2002) SAM (dependent) I AM: the S-adenosylmethionine-dependent methyltransferase fold. Curr Opin Struct Biol  12: 783–793.  <ul class="find" data-citedArticleID="1037150" data-doi="10.1016/s0959-440x(02)00391-3"><li><a href="http://dx.doi.org/10.1016/s0959-440x(02)00391-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SAM+%28dependent%29+I+AM%3A+the+S-adenosylmethionine-dependent+methyltransferase+fold." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SAM+%28dependent%29+I+AM%3A+the+S-adenosylmethionine-dependent+methyltransferase+fold.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001387-Rosso1" id="pone.0001387-Rosso1"></a>Rosso MG, Li Y, Strizhov N, Reiss B, Dekker K, et al.  (2003) An <em>Arabidopsis thaliana</em> T-DNA mutagenized population (GABI-Kat) for flanking sequence tag-based reverse genetics. Plant Mol Biol  53: 247–259.  <ul class="find" data-citedArticleID="1037176" data-doi="10.1023/b:plan.0000009297.37235.4a"><li><a href="http://dx.doi.org/10.1023/b:plan.0000009297.37235.4a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Arabidopsis+thaliana+T-DNA+mutagenized+population+%28GABI-Kat%29+for+flanking+sequence+tag-based+reverse+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Arabidopsis+thaliana+T-DNA+mutagenized+population+%28GABI-Kat%29+for+flanking+sequence+tag-based+reverse+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001387-Clough1" id="pone.0001387-Clough1"></a>Clough SJ, Bent AF (1998) Floral dip: a simplified method for <em>Agrobacterium</em>-mediated transformation of <em>Arabidopsis thaliana</em>. Plant J  16: 735–743.  <ul class="find" data-citedArticleID="1037088" data-doi="10.1046/j.1365-313x.1998.00343.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1998.00343.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Floral+dip%3A+a+simplified+method+for+Agrobacterium-mediated+transformation+of+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Floral+dip%3A+a+simplified+method+for+Agrobacterium-mediated+transformation+of+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001387-Jefferson1" id="pone.0001387-Jefferson1"></a>Jefferson RA, Kavanagh TA, Bevan MW (1987) GUS fusions: beta-glucuronidase as a sensitive and versatile gene fusion marker in higher plants. EMBO J  6: 3901–3907.  <ul class="find" data-citedArticleID="1037118"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=GUS+fusions%3A+beta-glucuronidase+as+a+sensitive+and+versatile+gene+fusion+marker+in+higher+plants.&amp;auth=&amp;atitle=GUS+fusions%3A+beta-glucuronidase+as+a+sensitive+and+versatile+gene+fusion+marker+in+higher+plants." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GUS+fusions%3A+beta-glucuronidase+as+a+sensitive+and+versatile+gene+fusion+marker+in+higher+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GUS+fusions%3A+beta-glucuronidase+as+a+sensitive+and+versatile+gene+fusion+marker+in+higher+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001387-Xin1" id="pone.0001387-Xin1"></a>Xin Z, Zhao Y, Zheng ZL (2005) Transcriptome analysis reveals specific modulation of abscisic acid signaling by ROP10 small GTPase in Arabidopsis. Plant Physiol  139: 1350–1365.  <ul class="find" data-citedArticleID="1037194" data-doi="10.1104/pp.105.068064"><li><a href="http://dx.doi.org/10.1104/pp.105.068064" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptome+analysis+reveals+specific+modulation+of+abscisic+acid+signaling+by+ROP10+small+GTPase+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptome+analysis+reveals+specific+modulation+of+abscisic+acid+signaling+by+ROP10+small+GTPase+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001387-Zheng1" id="pone.0001387-Zheng1"></a>Zheng ZL, Nafisi M, Tam A, Li H, Crowell DN, et al.  (2002) Plasma membrane-associated ROP10 small GTPase is a specific negative regulator of abscisic acid responses in Arabidopsis. Plant Cell  14: 2787–2797.  <ul class="find" data-citedArticleID="1037200" data-doi="10.1105/tpc.005611"><li><a href="http://dx.doi.org/10.1105/tpc.005611" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasma+membrane-associated+ROP10+small+GTPase+is+a+specific+negative+regulator+of+abscisic+acid+responses+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasma+membrane-associated+ROP10+small+GTPase+is+a+specific+negative+regulator+of+abscisic+acid+responses+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.XML" value="136460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.PDF" value="614482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g001.PNG_L" value="4546860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g001.PNG_M" value="237829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g001.PNG_S" value="20207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g001.TIF" value="7039790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g001.PNG_I" value="129339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g002.PNG_L" value="2952981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g002.PNG_M" value="99178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g002.PNG_S" value="20261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g002.TIF" value="4800324"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g002.PNG_I" value="158211"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g003.PNG_L" value="401819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g003.PNG_M" value="89654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g003.PNG_S" value="15252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g003.TIF" value="1150702"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g003.PNG_I" value="49882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g004.PNG_L" value="2187552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g004.PNG_M" value="119521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g004.PNG_S" value="14996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g004.TIF" value="4882188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g004.PNG_I" value="43059"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g005.PNG_L" value="275624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g005.PNG_M" value="19273"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g005.PNG_S" value="5364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g005.TIF" value="644124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g005.PNG_I" value="7078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g006.PNG_L" value="4411169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g006.PNG_M" value="111237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g006.PNG_S" value="15829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g006.TIF" value="6066834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g006.PNG_I" value="85231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g007.PNG_L" value="4767994"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g007.PNG_M" value="172972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g007.PNG_S" value="20106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g007.TIF" value="6546196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.g007.PNG_I" value="173828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.s001.DOC" value="101888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.s002.DOC" value="27648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.s003.TIF" value="2763020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.s004.TIF" value="3050900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001387.s005.TIF" value="860692"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001387&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001387" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001387&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001387&volume=&issue=&title=The OSU1/QUA2/TSD2-Encoded Putative Methyltransferase Is a Critical Modulator of Carbon and Nitrogen Nutrient Balance Response in Arabidopsis&author_name=Peng%20Gao%2C%20Zeyu%20Xin%2C%20Zhi-Liang%20Zheng&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001387" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001387" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001387" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001387&amp;t=The%20OSU1%2FQUA2%2FTSD2-Encoded%20Putative%20Methyltransferase%20Is%20a%20Critical%20Modulator%20of%20Carbon%20and%20Nitrogen%20Nutrient%20Balance%20Response%20in%20Arabidopsis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001387&title=The%20OSU1%2FQUA2%2FTSD2-Encoded%20Putative%20Methyltransferase%20Is%20a%20Critical%20Modulator%20of%20Carbon%20and%20Nitrogen%20Nutrient%20Balance%20Response%20in%20Arabidopsis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001387&amp;title=The%20OSU1%2FQUA2%2FTSD2-Encoded%20Putative%20Methyltransferase%20Is%20a%20Critical%20Modulator%20of%20Carbon%20and%20Nitrogen%20Nutrient%20Balance%20Response%20in%20Arabidopsis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001387" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001387&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The OSU1/QUA2/TSD2-Encoded Putative Methyltransferase Is a Critical Modulator of Carbon and Nitrogen Nutrient Balance Response in Arabidopsis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001387';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20OSU1%2FQUA2%2FTSD2-Encoded%20Putative%20Methyltransferase%20Is%20a%20Critical%20Modulator%20of%20Carbon%20and%20Nitrogen%20Nutrient%20Balance%20Response%20in%20Arabidopsis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001387" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001387" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cotyledons+%28botany%29%22" title="Search for articles in the subject area:'Cotyledons (botany)'"><div class="flagText">Cotyledons (botany)</div></a>
              <div data-categoryid="33773" data-articleid="25604"
                   data-categoryname="Cotyledons (botany)"
                   class="flagImage" title="Flag 'Cotyledons (botany)' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phosphates%22" title="Search for articles in the subject area:'Phosphates'"><div class="flagText">Phosphates</div></a>
              <div data-categoryid="19751" data-articleid="25604"
                   data-categoryname="Phosphates"
                   class="flagImage" title="Flag 'Phosphates' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+signaling%22" title="Search for articles in the subject area:'Plant signaling'"><div class="flagText">Plant signaling</div></a>
              <div data-categoryid="18111" data-articleid="25604"
                   data-categoryname="Plant signaling"
                   class="flagImage" title="Flag 'Plant signaling' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="47289" data-articleid="25604"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reverse+transcriptase-polymerase+chain+reaction%22" title="Search for articles in the subject area:'Reverse transcriptase-polymerase chain reaction'"><div class="flagText">Reverse transcriptase-polymerase chain reaction</div></a>
              <div data-categoryid="47921" data-articleid="25604"
                   data-categoryname="Reverse transcriptase-polymerase chain reaction"
                   class="flagImage" title="Flag 'Reverse transcriptase-polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Root+growth%22" title="Search for articles in the subject area:'Root growth'"><div class="flagText">Root growth</div></a>
              <div data-categoryid="21449" data-articleid="25604"
                   data-categoryname="Root growth"
                   class="flagImage" title="Flag 'Root growth' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Seedlings%22" title="Search for articles in the subject area:'Seedlings'"><div class="flagText">Seedlings</div></a>
              <div data-categoryid="43017" data-articleid="25604"
                   data-categoryname="Seedlings"
                   class="flagImage" title="Flag 'Seedlings' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sulfates%22" title="Search for articles in the subject area:'Sulfates'"><div class="flagText">Sulfates</div></a>
              <div data-categoryid="20973" data-articleid="25604"
                   data-categoryname="Sulfates"
                   class="flagImage" title="Flag 'Sulfates' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2507'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4377'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=102&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
