

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Estimating Heritabilities and Genetic Correlations: Comparing the ‘Animal Model’ with Parent-Offspring Regression Using Data from a Natural Population</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001739"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001739" />

    <meta name="citation_title" content="Estimating Heritabilities and Genetic Correlations: Comparing the ‘Animal Model’ with Parent-Offspring Regression Using Data from a Natural Population"/>
    <meta itemprop="name" content="Estimating Heritabilities and Genetic Correlations: Comparing the ‘Animal Model’ with Parent-Offspring Regression Using Data from a Natural Population"/>

      <meta name="citation_author" content="Mikael Åkesson"/>
            <meta name="citation_author_institution" content="Department of Animal Ecology, Lund University, Lund, Sweden"/>
      <meta name="citation_author" content="Staffan Bensch"/>
            <meta name="citation_author_institution" content="Department of Animal Ecology, Lund University, Lund, Sweden"/>
      <meta name="citation_author" content="Dennis Hasselquist"/>
            <meta name="citation_author_institution" content="Department of Animal Ecology, Lund University, Lund, Sweden"/>
      <meta name="citation_author" content="Maja Tarka"/>
            <meta name="citation_author_institution" content="Department of Animal Ecology, Lund University, Lund, Sweden"/>
      <meta name="citation_author" content="Bengt Hansson"/>
            <meta name="citation_author_institution" content="Department of Animal Ecology, Lund University, Lund, Sweden"/>

    <meta name="citation_date" content="2008/3/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001739.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1739"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Natural inheritance.; citation_author=F Galton; citation_number=1; citation_date=1889; citation_publisher=Macmillan; " />
      <meta name="citation_reference" content="citation_title=The origins of theoretical population genetics.; citation_author=WB Provine; citation_number=2; citation_date=2001; citation_publisher=University of Chicago Press; " />
      <meta name="citation_reference" content="citation_title=Natural selection and the heritability of fitness components.; citation_author=TA Mousseau; citation_author=DA Roff; citation_journal_title=Heredity; citation_volume=59; citation_number=3; citation_pages=181-197; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Genetics and the analysis of quantitative traits.; citation_author=M Lynch; citation_author=B Walsh; citation_number=4; citation_date=1998; citation_publisher=Sinauer associates; " />
      <meta name="citation_reference" content="citation_title=Maximum likelihood estimation of genetic parameters in life-history studies using the ‘animal model’.; citation_author=SA Knott; citation_author=RM Sibly; citation_author=RH Smith; citation_author=AP Møller; citation_journal_title=Functional Ecology; citation_volume=9; citation_number=5; citation_pages=122-126; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Estimating genetic parameters in natural populations using the ‘animal model’.; citation_author=LEB Kruuk; citation_journal_title=Philo T Roy Soc B; citation_volume=359; citation_number=6; citation_pages=873-890; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=How to separate genetic and environmental causes of similarity between relatives.; citation_author=LEB Kruuk; citation_author=JD Hadfield; citation_journal_title=J Evol Biol; citation_volume=20; citation_number=7; citation_pages=1890-1903; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Comparing evolvability and variability of quantitative traits.; citation_author=D Houle; citation_journal_title=Genetics; citation_volume=130; citation_number=8; citation_pages=195-204; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Sexual variation in heritability and genetic correlations of morphological traits in house sparrow (Passer domesticus).; citation_author=H Jensen; citation_author=BE Saether; citation_author=TH Ringsby; citation_author=J Tufto; citation_author=SC Griffith; citation_author=H Ellegren; citation_journal_title=J Evol Biol; citation_volume=16; citation_number=9; citation_pages=1296-1307; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Selection and genetic (co)variance in bighorn sheep.; citation_author=DW Coltman; citation_author=P O'Donoghue; citation_author=JT Hogg; citation_author=M Festa-Bianchet; citation_journal_title=Evolution; citation_volume=59; citation_number=10; citation_pages=1372-1382; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Introduction to quantitative genetics.; citation_author=DS Falconer; citation_author=TFC Mackay; citation_number=11; citation_date=1996; citation_publisher=Longman Group Ltd; " />
      <meta name="citation_reference" content="citation_title=Heritability of body mass varies with age and season in wild bighorn sheep.; citation_author=D Réale; citation_author=M Festa-Bianchet; citation_author=JT Jorgenson; citation_journal_title=Heredity; citation_volume=83; citation_number=12; citation_pages=526-532; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Heritability of parental effort in a passerine bird.; citation_author=ADC Maccoll; citation_author=BJ Hatchwell; citation_journal_title=Evolution; citation_volume=57; citation_number=13; citation_pages=2191-2195; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Territory infidelity in the polygynous great reed warbler Acrocephalus arundinaceus: the effect of variation in territory attractiveness.; citation_author=S Bensch; citation_author=D Hasselquist; citation_journal_title=J Anim Ecol; citation_volume=60; citation_number=14; citation_pages=857-871; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Higher fitness for philopatric than for immigrant males in a semi-isolated population of great reed warblers.; citation_author=S Bensch; citation_author=D Hasselquist; citation_author=B Nielsen; citation_author=B Hansson; citation_journal_title=Evolution; citation_volume=52; citation_number=15; citation_pages=877-883; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Lifetime fitness of short- and long-distance dispersing great reed warblers.; citation_author=B Hansson; citation_author=S Bensch; citation_author=D Hasselquist; citation_journal_title=Evolution; citation_volume=58; citation_number=16; citation_pages=2546-2557; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Low frequency of extra-pair paternity in the polygynous great reed warbler, Acrocephalus arundinaceus.; citation_author=D Hasselquist; citation_author=S Bensch; citation_author=T von Schantz; citation_journal_title=Behav Ecol; citation_volume=6; citation_number=17; citation_pages=27-38; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Breeding synchrony does not affect extra-pair paternity in great reed warblers.; citation_author=D Arlt; citation_author=B Hansson; citation_author=S Bensch; citation_author=T von Schantz; citation_author=D Hasselquist; citation_journal_title=Behaviour; citation_volume=141; citation_number=18; citation_pages=863-880; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Do female great reed warblers seek extra-pair fertilizations to avoid inbreeding?; citation_author=B Hansson; citation_author=D Hasselquist; citation_author=S Bensch; citation_journal_title=Biology Letters; citation_volume=271; citation_number=19; citation_pages=290-292; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic and phenotypic associations in morphological traits: a long term study of great reed warblers Acrocephalus arundinaceus.; citation_author=M Åkesson; citation_author=S Bensch; citation_author=D Hasselquist; citation_journal_title=J Avian Biol; citation_volume=38; citation_number=20; citation_pages=58-72; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Does linkage disequilibrium generate heterozygosity-fitness correlations in great reed warblers?; citation_author=B Hansson; citation_author=H Westerdahl; citation_author=D Hasselquist; citation_author=M Åkesson; citation_author=S Bensch; citation_journal_title=Evolution; citation_volume=58; citation_number=21; citation_pages=870-879; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny of Palearctic-African Acrocephalus and Hippolais warblers (Aves : Sylviidae).; citation_author=AJ Helbig; citation_author=I Seibold; citation_journal_title=Mol Phylogenet Evol; citation_volume=11; citation_number=22; citation_pages=246-260; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Handbook of the birds of Europe, the middle East and North Africa.; citation_author=S Cramp; citation_number=23; citation_date=1992; citation_publisher=Oxford Univerity Press; " />
      <meta name="citation_reference" content="citation_title=Polygyny in great reed warblers: a long-term study of factors contributing to male fitness.; citation_author=D Hasselquist; citation_journal_title=Ecology; citation_volume=79; citation_number=24; citation_pages=2376-2390; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Increase of genetic variation over time in a recently founded population of great reed warblers (Acrocephalus arundinaceus) revealed by microsatellites and DNA fingerprinting.; citation_author=B Hansson; citation_author=S Bensch; citation_author=D Hasselquist; citation_author=BG Lillandt; citation_author=L Wennerberg; citation_author=T von Schantz; citation_journal_title=Mol Ecol; citation_volume=9; citation_number=25; citation_pages=1529-1538; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=PEDSTATS: descriptive statistics, graphics and quality assessment for gene mapping data.; citation_author=JE Wigginton; citation_author=GR Abecasis; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=26; citation_pages=3445-3447; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification guide to European passerines.; citation_author=L Svensson; citation_number=27; citation_date=1992; citation_publisher=Fingraf; " />
      <meta name="citation_reference" content="citation_title=SAS System for Mixed Models.; citation_author=RC Littell; citation_author=GA Milliken; citation_author=WW Stroup; citation_author=RD Wolfinger; citation_number=28; citation_date=1996; citation_publisher=SAS Institute Inc; " />
      <meta name="citation_reference" content="citation_title=ASreml user guide. Release 1.0.; citation_author=AR Gilmour; citation_author=BJ Gogel; citation_author=BR Cullis; citation_author=SJ Welham; citation_author=R Thompson; citation_number=29; citation_date=2006; citation_publisher=VSN International; " />
      <meta name="citation_reference" content="citation_title=Unrepeatable repeatabilities: a common mistake.; citation_author=CM Lessells; citation_author=PT Boag; citation_journal_title=Auk; citation_volume=104; citation_number=30; citation_pages=116-121; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Manual of quantitative genetics.; citation_author=WA Becker; citation_number=31; citation_date=1984; citation_publisher=Academic Enterprises; " />
      <meta name="citation_reference" content="citation_title=The sampling variance of the genetic correlation coefficient.; citation_author=A Robertson; citation_journal_title=Biometrics; citation_volume=15; citation_number=32; citation_pages=469-485; citation_date=1959; " />
      <meta name="citation_reference" content="citation_title=SPSS 14.0 for windows; citation_number=33; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Biometry: the principles and practice of statistics in bilogical research.; citation_author=RR Sokal; citation_author=FJ Rohlf; citation_number=34; citation_date=1995; citation_publisher=W.H. Freeman and Co; " />
      <meta name="citation_reference" content="citation_title=Estimating cuckoldry in birds: the heritability method and DNA-fingerprinting give different results.; citation_author=D Hasselquist; citation_author=S Bensch; citation_author=T von Schantz; citation_journal_title=Oikos; citation_volume=72; citation_number=35; citation_pages=173-178; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Avian quantitative genetics.; citation_author=J Merilä; citation_author=BC Sheldon; citation_number=36; citation_pages=179-255; citation_date=2001; citation_publisher=Kluwer Academic/Plenum Publishers; " />
      <meta name="citation_reference" content="citation_title=Evolution of correlated characters.; citation_author=T Price; citation_author=T Langen; citation_journal_title=TREE; citation_volume=7; citation_number=37; citation_pages=307-310; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Inheritance of size and shape in a population of Darwin Finches Geospiza conirostris.; citation_author=PR Grant; citation_journal_title=Proc R Soc Lond B; citation_volume=220; citation_number=38; citation_pages=219-236; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Inheritance of size and shape in a natural population of Collared Flycatchers Ficedula albicollis.; citation_author=J Merilä; citation_author=L Gustafsson; citation_journal_title=J Evol Biol; citation_volume=6; citation_number=39; citation_pages=375-395; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Natural selection and inheritance of breeding time and clutch size in the collared flycatcher.; citation_author=BC Sheldon; citation_author=LEB Kruuk; citation_author=J Merilä; citation_journal_title=Evolution; citation_volume=57; citation_number=40; citation_pages=406-420; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evolutionary quantitative genetics.; citation_author=DA Roff; citation_number=41; citation_date=1997; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=Genetic correlations and maternal effect coefficients obtained from offspring-parent regression.; citation_author=R Lande; citation_author=T Price; citation_journal_title=Genetics; citation_volume=122; citation_number=42; citation_pages=915-922; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The variance of the genetic correlation coefficient.; citation_author=ECR Reeve; citation_journal_title=Biometrics; citation_volume=11; citation_number=43; citation_pages=357-374; citation_date=1955; " />
      <meta name="citation_reference" content="citation_title=The calculation and significance testing of genetic correlations across environments.; citation_author=JJ Windig; citation_journal_title=J Evol Biol; citation_volume=10; citation_number=44; citation_pages=853-874; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Empirical sampling estimates of genetic correlations.; citation_author=LD Van Vleck; citation_author=CR Henderson; citation_journal_title=Biometrics; citation_volume=17; citation_number=45; citation_pages=359-371; citation_date=1961; " />
      <meta name="citation_reference" content="citation_title=An empirical study of distribution of the sample genetic correlation coefficient.; citation_author=GH Brown; citation_journal_title=Biometrics; citation_volume=25; citation_number=46; citation_pages=63-72; citation_date=1969; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001739" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Estimating Heritabilities and Genetic Correlations: Comparing the ‘Animal Model’ with Parent-Offspring Regression Using Data from a Natural Population"/>
    <meta name="twitter:description" content="Quantitative genetic parameters are nowadays more frequently estimated with restricted maximum likelihood using the &lsquo;animal model&rsquo; than with traditional methods such as parent-offspring regressions. These methods have however rarely been evaluated using equivalent data sets. We compare heritabilities and genetic correlations from animal model and parent-offspring analyses, respectively, using data on eight morphological traits in the great reed warbler (Acrocephalus arundinaceus). Animal models were run using either mean trait values or individual repeated measurements to be able to separate between effects of including more extended pedigree information and effects of replicated sampling from the same individuals. We show that the inclusion of more pedigree information by the use of mean traits animal models had limited effect on the standard error and magnitude of heritabilities. In contrast, the use of repeated measures animal model generally had a positive effect on the sampling accuracy and resulted in lower heritabilities; the latter due to lower additive variance and higher phenotypic variance. For most trait combinations, both animal model methods gave genetic correlations that were lower than the parent-offspring estimates, whereas the standard errors were lower only for the mean traits animal model. We conclude that differences in heritabilities between the animal model and parent-offspring regressions were mostly due to the inclusion of individual replicates to the animal model rather than the inclusion of more extended pedigree information. Genetic correlations were, on the other hand, primarily affected by the inclusion of more pedigree information. This study is to our knowledge the most comprehensive empirical evaluation of the performance of the animal model in relation to parent-offspring regressions in a wild population. Our conclusions should be valuable for reconciliation of data obtained in earlier studies as well as for future meta-analyses utilizing estimates from both traditional methods and the animal model."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001739.g002"/>

  <meta property="og:title" content="Estimating Heritabilities and Genetic Correlations: Comparing the ‘Animal Model’ with Parent-Offspring Regression Using Data from a Natural Population" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001739" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5954'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2886'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=557&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001739">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001739" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001739&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Estimating Heritabilities and Genetic Correlations: Comparing the ‘Animal Model’ with Parent-Offspring Regression Using Data from a Natural Population
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mikael Åkesson
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:mikael.akesson@zooekol.lu.se">mikael.akesson@zooekol.lu.se</a></p>

                <p>Affiliation:
                  Department of Animal Ecology, Lund University, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Staffan Bensch, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Animal Ecology, Lund University, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dennis Hasselquist, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Animal Ecology, Lund University, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maja Tarka, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Animal Ecology, Lund University, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bengt Hansson
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Animal Ecology, Lund University, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 05, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001739</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001739" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001739" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001739" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001739" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001739-t001" data-doi="info:doi/10.1371/journal.pone.0001739" data-uri="info:doi/10.1371/journal.pone.0001739.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001739-t002" data-doi="info:doi/10.1371/journal.pone.0001739" data-uri="info:doi/10.1371/journal.pone.0001739.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001739-g001" data-doi="info:doi/10.1371/journal.pone.0001739" data-uri="info:doi/10.1371/journal.pone.0001739.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001739-g002" data-doi="info:doi/10.1371/journal.pone.0001739" data-uri="info:doi/10.1371/journal.pone.0001739.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001739-t003" data-doi="info:doi/10.1371/journal.pone.0001739" data-uri="info:doi/10.1371/journal.pone.0001739.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001739">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001739" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Quantitative genetic parameters are nowadays more frequently estimated with restricted maximum likelihood using the ‘animal model’ than with traditional methods such as parent-offspring regressions. These methods have however rarely been evaluated using equivalent data sets. We compare heritabilities and genetic correlations from animal model and parent-offspring analyses, respectively, using data on eight morphological traits in the great reed warbler (<em>Acrocephalus arundinaceus</em>). Animal models were run using either mean trait values or individual repeated measurements to be able to separate between effects of including more extended pedigree information and effects of replicated sampling from the same individuals. We show that the inclusion of more pedigree information by the use of mean traits animal models had limited effect on the standard error and magnitude of heritabilities. In contrast, the use of repeated measures animal model generally had a positive effect on the sampling accuracy and resulted in lower heritabilities; the latter due to lower additive variance and higher phenotypic variance. For most trait combinations, both animal model methods gave genetic correlations that were lower than the parent-offspring estimates, whereas the standard errors were lower only for the mean traits animal model. We conclude that differences in heritabilities between the animal model and parent-offspring regressions were mostly due to the inclusion of individual replicates to the animal model rather than the inclusion of more extended pedigree information. Genetic correlations were, on the other hand, primarily affected by the inclusion of more pedigree information. This study is to our knowledge the most comprehensive empirical evaluation of the performance of the animal model in relation to parent-offspring regressions in a wild population. Our conclusions should be valuable for reconciliation of data obtained in earlier studies as well as for future meta-analyses utilizing estimates from both traditional methods and the animal model.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Åkesson M, Bensch S, Hasselquist D, Tarka M, Hansson B (2008) Estimating Heritabilities and Genetic Correlations: Comparing the ‘Animal Model’ with Parent-Offspring Regression Using Data from a Natural Population. PLoS ONE 3(3):
          e1739.
            doi:10.1371/journal.pone.0001739</p><p><strong>Academic Editor: </strong>Andy Gardner, University of Edinburgh, United Kingdom</p><p><strong>Received:</strong> November 16, 2007; <strong>Accepted:</strong> January 29, 2008; <strong>Published:</strong> March 5, 2008</p><p><strong>Copyright:</strong> © 2008 Åkesson et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The work was supported by grants from the Swedish Research Council (VR), the Swedish Council for Environment, Agricultural Sciences and Spatial Planning (FORMAS), the National Swedish Environment Protection Board, the Carl Tryggers stiftelse, the Royal Swedish Academy of Science (J.A. Ahlstrand and Hierta-Retzius Foundations), the Lunds Djurskyddsfond, the Magn Bergvalls Stiftelse, the National Geographic Society and the Crafoordska Stiftelsen.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A main aim in evolutionary biology is to predict phenotypic change enforced by natural and sexual selection. This requires, among other things, detailed knowledge about the inheritance of phenotypic traits. Traditionally, heritabilities have been estimated by correlations of close kin, e.g. parent-offspring regressions <a href="#pone.0001739-Galton1">[1]</a>–<a href="#pone.0001739-Lynch1">[4]</a>. During the last decade, the study of evolutionary quantitative genetics in wild populations has made a transition from the traditional use of close-kin comparisons to the more powerful ‘animal model’ using restricted maximum likelihood (REML) <a href="#pone.0001739-Lynch1">[4]</a>, <a href="#pone.0001739-Knott1">[5]</a> to estimate quantitative genetic parameters in natural populations <a href="#pone.0001739-Kruuk1">[6]</a>. An animal model takes into account all relationships in a pedigree and is therefore expected to provide estimates of quantitative genetic parameters with higher precision than estimates restricted to the similarity between close kin. It is also less likely to be biased by complicating factors such as assortative mating, inbreeding, selection and shared environment <a href="#pone.0001739-Kruuk2">[7]</a>. Moreover, the animal model is expected to be statistically more robust to unbalanced data sets compared to parent-offspring models.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>A recent review <a href="#pone.0001739-Kruuk1">[6]</a> found and compared published data on heritabilities, estimated from the same populations by both the animal model and parent-offspring regressions. The comparison included heritabilities of 11 traits from 6 species. The pattern emerging from these studies is that heritabilities and standard errors are generally lower with animal models than with parent-offspring regression <a href="#pone.0001739-Kruuk1">[6]</a>. However, when comparing results of the two methods based on published data one is confronted with several problems. First, the estimates are mostly taken from data that differ in sample size; e.g. larger data sets are frequently accessible in later publications using the animal model technique. Second, comparisons could be hampered, because information about variances and means of traits are lacking in many, in particular older studies <a href="#pone.0001739-Houle1">[8]</a>. Third, many animal model analyses use individual repeated measures <a href="#pone.0001739-Jensen1">[9]</a>, <a href="#pone.0001739-Coltman1">[10]</a> instead of mean trait values that is used in parent-offspring regressions. The within-individual variation is partly due to phenotypic plasticity and partly due to measurement error. The rational behind using mean traits in parent-offspring regressions has been to avoid pseudo-replication and to account for measurement errors. Thus, when comparing (repeated measures) animal model estimates with estimates derived from mean trait values, one needs to take into account that within-individual variance is likely affecting the estimate of phenotypic variance and possibly also the additive and residual variance <a href="#pone.0001739-Falconer1">[11]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Given that heritability estimates for the vast majority of species and traits still come from parent-offspring analyses <a href="#pone.0001739-Mousseau1">[3]</a> it would be valuable to evaluate the accuracy of this method compared to the animal model technique. There are to our knowledge only two published studies directly comparing and evaluating the animal model and parent-offspring regression techniques using the same data sets from natural populations <a href="#pone.0001739-Rale1">[12]</a>, <a href="#pone.0001739-Maccoll1">[13]</a>. The results in one of these studies, on long-tailed tits (<em>Aegithalos caudatus</em>) <a href="#pone.0001739-Maccoll1">[13]</a>, are in line with the general conclusions in Kruuk's review <a href="#pone.0001739-Kruuk1">[6]</a> which include lower heritability accompanied with smaller standard error when using the animal model. In a study of bighorn sheep (<em>Ovus canadensis</em>), the maternal-offspring heritability of age class-specific body mass was similar to the corresponding animal model heritability for older ages, but lower for early age-classes along with only moderate reductions in standard errors <a href="#pone.0001739-Rale1">[12]</a>. Even though these results were not in line with Kruuk's conclusions, the comparison between the two methods in the sheep study was limited by the fact that maternal effects were not fully accounted for in the animal model since all father identities were unknown. Nevertheless, evaluations like these are important in order to reconcile results in studies using different methods. Moreover, there is a need for understanding why these methods may produce different results; whether it could be an effect of including more extended pedigree information or due to replicated sampling from the same individuals in the animal model analyses.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In the present study, we compare different methods to estimate heritabilities and genetic correlations using data from eight morphological traits collected in a natural population of great reed warblers (<em>Acrocephalus arundinaceus</em>). We aimed at evaluating whether the estimated parameters differed when employing the different methods to the same data set, and if so to understand the underlying causes. Our study population has been monitored for more than twenty years <a href="#pone.0001739-Bensch1">[14]</a>–<a href="#pone.0001739-Hansson1">[16]</a> and we have access to a large pedigree in which parentage has been resolved with molecular techniques for the majority of individuals <a href="#pone.0001739-Hasselquist1">[17]</a>–<a href="#pone.0001739-Hansson2">[19]</a>. We therefore expect animal models to be more powerful when estimating quantitative genetic parameters than parent-offspring regressions. Furthermore, there are characteristics of the population that may violate the assumptions of parent-offspring regressions, i.e. some traits are subjected to directional or stabilizing selection <a href="#pone.0001739-kesson1">[20]</a>, some show influence of shared environment between parents and offspring <a href="#pone.0001739-kesson1">[20]</a>, and there are records of relatively high linkage disequilibrium throughout the genome in great reed warblers (<a href="#pone.0001739-Hansson3">[21]</a>, Hansson, B and Csilléry, K unpublished).</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>To separate between the effects of utilizing all relationships in a pedigree when estimating heritability from the effects of using multiple measurements of the same individual, we used two animal models for each trait and trait combination. In the first animal model, we used the arithmetic mean of all measures of an individual and compared this with the parent-offspring model to investigate how bias and precision are affected by the use of all relationships in the pedigree. In the second model, we used repeated measures of the same individual, thereby producing estimates that may not only be influenced by the use of a larger pedigree but also by information about variation within individuals.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Study species and morphometrics</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The great reed warbler (<em>Acrocephalus arundinaceus</em>) is a large-sized warbler belonging to the family Sylviidae <a href="#pone.0001739-Helbig1">[22]</a>. It winters in sub-Saharan Africa and migrates to breed in reed lakes in Eurasia <a href="#pone.0001739-Cramp1">[23]</a>. The great reed warbler has a facultative socially polygynous breeding system and about 40 % of the territorial males form social pair bonds with 2–5 females in a season <a href="#pone.0001739-Hasselquist2">[24]</a>. The breeding population at Lake Kvismaren (50°10′N, 15°25′E) has been monitored since 1983 <a href="#pone.0001739-Bensch1">[14]</a>–<a href="#pone.0001739-Hansson1">[16]</a>, <a href="#pone.0001739-Hasselquist2">[24]</a>. Almost all breeding adults and un-paired males have been captured in mist-nets and then colour-ringed, measured for morphological traits, weighed and blood sampled. Located nests were visited every third day until chicks fledged (when 14–16 days old). When about nine days old, chicks were ringed, measured and blood sampled.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We have taken blood samples from almost all adults and nestlings in the study area since 1987. True parentage of more than eighty percent of these individuals has been assigned with minisatellite DNA fingerprinting <a href="#pone.0001739-Hasselquist1">[17]</a> or microsatellite genotyping (<a href="#pone.0001739-Arlt1">[18]</a>, <a href="#pone.0001739-Hansson2">[19]</a>, unpublished material). The frequency of extra-pair young is ca 3 % in the population and in the following analyses we use the genetic father of all offspring. We estimate that among the non-genotyped families no more than two offspring should be sired by an extra-pair male.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>We used data collected between 1983 and 2002. After the founding event in 1978 the population has increased to a size of about 50 adults (range 42–78 since 1989). The major increase in population size occurred between 1983 and 1989 <a href="#pone.0001739-Hansson4">[25]</a>.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>The pedigree we have used in this study contains 523 adults of which 199 individuals were hatched in Kvismaren and have parents that previously have been caught, ringed and measured. For three individuals, we only know the identities of the fathers whereas the mothers were unringed and thus immigrants providing no morphometric or genetic data. Among the adult great reed warblers in Kvismaren there are 89 sib pairs, 322 half sibs, 94 cousins, 404 parent-offspring pairs, 337 grandparent-grandchild pairs and 145 avuncular pairs (retrieved by PEDSATS 0.6.5 <a href="#pone.0001739-Wigginton1">[26]</a>), indicating a rather complex pedigree.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>We estimated heritabilities and variance components for wing length <a href="#pone.0001739-Svensson1">[27]</a>, wing projection (the distance between the first secondary and the longest primary feather of a relaxed wing), tail length, bill width, bill height, bill length, skull length and tarsus length <a href="#pone.0001739-kesson1">[20]</a>. Adults were measured for all traits from 1991 and onwards. Before this time we only measured the wing length and tarsus length.</p>


<h4>Parent-offspring regression</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We used the same methods as reported in Åkesson <em>et al.</em> <a href="#pone.0001739-kesson1">[20]</a> to estimate (narrow sense) heritabilities and additive variances. Prior to the heritability analyses, we tested for fixed effects on the traits by using a mixed linear model (GLMM) with repeated measurements as a random effect (SAS Proc Mixed; see <a href="#pone.0001739-Littell1">[28]</a>). Each trait was corrected for age, sex, year and/or ringer identity (<a href="#pone.0001739.s001">Table S1</a>) by subtracting the observed value with the proper fixed effects. We lacked the identity of the ringer for 10 measuring events. To avoid reduction in sample size, we fitted these particular measurements with a dummy ringer before the mixed linear model analyses. We calculated heritabilities of the eight traits by regressing the average offspring trait values on average parent values, henceforth referred to as the parent-offspring model. We used the average value of full-sibs to avoid pseudo-replication in the regression analysis. The estimated heritability corresponds to the slope of the midparent-midoffspring regression <a href="#pone.0001739-Falconer1">[11]</a>.</p>


<h4>Animal model analyses</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Heritabilities and variance components of the phenotypic variance were estimated with restricted maximum likelihood (REML) models, which are preferred over maximum likelihood models when fitting a large number of fixed effects <a href="#pone.0001739-Lynch1">[4]</a>. The program we used was ASReml 2.0 <a href="#pone.0001739-Gilmour1">[29]</a>. We fitted animal models with random effects and fixed effects: <strong>y</strong> = <strong><em>X</em></strong><strong>b</strong>+<strong><em>Z</em></strong><sub>a</sub><strong>a</strong>+<strong><em>Z</em></strong><sub>c</sub><strong>c</strong>+<strong><em>Z</em></strong><sub>m</sub><strong>m</strong>+<strong><em>Z</em></strong><sub>n</sub><strong>n</strong>+<strong>e</strong>, where <strong>y</strong> is the vector of observed phenotypic values of the individuals and vectors <strong>b</strong> = fixed effects, <strong>a = </strong>additive effects, <strong>c = </strong>permanent environment effects, <strong>m = </strong>maternal effects, <strong>n = </strong>common-nest effects and <strong>e = </strong>residual effects. <strong><em>X</em></strong>, <strong><em>Z</em></strong><sub>a</sub>, <strong><em>Z</em></strong><sub>c</sub>, <strong><em>Z</em></strong><sub>m</sub> and <strong><em>Z</em></strong><sub>n</sub> are design matrices relating the records to the appropriate fixed and random effects <a href="#pone.0001739-Lynch1">[4]</a>. We collectively refer <strong>c</strong>, <strong>m</strong>, <strong>n</strong> and <strong>e</strong> as environmental effects and their variance as environmental variance. Note that the use of this terminology does not exclude the possibility that they all may incorporate different sources of (non-additive) genetic effects. The repeated measurements of the same individual will group into the permanent environment effects and is likely to incorporate environmental effects that has a long-term effect (e.g. maternal, dominance, epistasis and cohort effects) on an individual <a href="#pone.0001739-Kruuk1">[6]</a>. Maternal effects will group individuals with the same mother and common-nest effects will group those raised in the same nest. To avoid sample size loss due to missing information about mother and nest identity, we fitted unique dummy values to each individual with a missing value. These individuals are almost exclusively immigrants and are therefore very likely to origin from different mothers and nests. We also conducted analyses after deleting individuals with missing values for random factors (such as those with unknown mothers) and the result was very similar but the parameters had larger sampling errors probably due to lower sample size (data not reported in this study).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We used two different animal model approaches to estimate heritabilities and variance components. In the first animal model, we used the mean of the individual trait values. This will henceforth be referred to as the mean traits animal model. The total phenotypic variance (V<sub>P</sub>) was then partitioned into additive genetic variance (V<sub>A</sub>), maternal effect variance (V<sub>M</sub>), common-nest effect variance (V<sub>B</sub>) and residual variance (V<sub>R</sub>). This data set is identical to the data set used for the parent-offspring regression, with exception of the use of average values from individuals in the same brood in the latter method. For comparative purposes we also standardized the phenotypic, additive and environmental variance components of each trait by calculating coefficient of variation (CV), i.e. the square-root of the variance component divided by the non-standardized phenotypic mean (<a href="#pone.0001739.s001">Table S1</a>) of the trait (cf. <a href="#pone.0001739-Houle1">[8]</a>).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In the second animal model we used repeated measurements (if available) from the same individual, henceforth called repeated measures animal model, and this included fixed effects (instead of corrected values). Thus, V<sub>P</sub> was partioned into V<sub>A</sub>, variance due to permanent environment effects (V<sub>PE</sub>), V<sub>M</sub>, V<sub>B</sub> and V<sub>R</sub> in such a way that V<sub>P</sub> = V<sub>A</sub>+V<sub>PE</sub>+V<sub>M</sub>+V<sub>B</sub>+V<sub>R</sub>. The narrow-sense heritability was calculated as the ratio of additive variance to the total phenotypic variance: <em>h</em><sup>2</sup> = V<sub>A</sub>/V<sub>P</sub>, the permanent environment effect as <em>c</em><sup>2</sup> = V<sub>PE</sub>/V<sub>P</sub>, the maternal effect as <em>m</em><sup>2</sup> = V<sub>M</sub>/V<sub>P</sub> and the common-nest effect as <em>b</em><sup>2</sup> = V<sub>B</sub>/V<sub>P</sub>. All data from the repeated measures animal model are reported in <a href="#pone.0001739.s001">Table S1</a>.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Three of eight traits had a significant permanent environment effect in the repeated measures animal model (<a href="#pone.0001739.s001">Table S1</a>), ranging between <em>c</em><sup>2</sup> = 0.13±0.09 (SE) for tarsus length and 0.45±0.11 (SE) for bill depth. In three cases, the estimates of V<sub>PE</sub> were locked at the minimum boundary level of the model and no standard errors were returned. In those cases, the estimates were very small or not accompanied with sampling error and we chose not to present the parameters in <a href="#pone.0001739.s001">Table S1</a>. None of the traits had any significant variance due to maternal or common-nest effect. However, we chose to keep maternal effect in tarsus length and common-nest effect in wing length and wing projection in the models for further analyses, to avoid overestimation of the additive effects.</p>


<h4>Repeatabilities</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The repeatability (<em>r</em><sup>2</sup>) of a trait describes the proportion of variance in the trait that is due to variation among rather than within individuals <a href="#pone.0001739-Lessells1">[30]</a>. We calculated repeatabilities from the components of variance extracted from the repeated measures animal model as the sum of the heritability and the portion of phenotypic variance due to any other random effect (e.g. permanent environment effect) if included into the mixed model (<a href="#pone.0001739.s001">Table S1</a>). The repeatabilities ranged between 0.36 and 0.95 with a mean of 0.61 (<a href="#pone.0001739.s001">Table S1</a>). These <em>r</em><sup>2</sup>s were highly correlated (Pearson correlation: <em>r</em> = 0.987, N = 8, <em>P</em>&lt;0.001), and showed no significant deviation in sign and magnitude from the <em>r</em><sup>2</sup>s reported in Åkesson <em>et al.</em> <a href="#pone.0001739-kesson1">[20]</a> that were calculated (in accordance with <a href="#pone.0001739-Lessells1">[30]</a>) by using repeated values corrected for fixed effects (Wilcoxon signed-rank test: <em>Z</em><sub>W</sub> = 0.84, N = 8, <em>P</em> = 0.4). The standard errors of the repeated measures animal model were very similar to the standard errors estimated according to Lessells and Boag's method <a href="#pone.0001739-Becker1">[31]</a>, as indicated by the high correlation (<em>r</em> = 0.994, N = 8, <em>P</em>&lt;0.001) and non-significant difference in sign and magnitude (<em>Z</em><sub>W</sub> = 0.84, N = 8, <em>P</em> = 0.4) <a href="#pone.0001739-kesson1">[20]</a>. We therefore chose to report only the animal model estimate of repeatability for each trait (<a href="#pone.0001739.s001">Table S1</a>).</p>


<h4>Genetic and phenotypic correlations</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Genetic correlations (<em>r</em><sub>A</sub>) were estimated by regressing average offspring values of trait X on average parent values of trait Y, and vice versa, in accordance with the methods described in <a href="#pone.0001739-Lynch1">[4]</a>. Prior to these analyses, all traits were corrected for significant effects of age, sex, year and ringer (see above). The calculation of <em>r</em><sub>A</sub> involves dividing the covariances between different traits X and Y (covXY) in parents and offspring with the square-root product of the covariances between the same traits (covXX and covYY, respectively). Since there are two possible products of covXY there are also two estimates of <em>r</em><sub>A</sub> (<em>r</em><sub>A1</sub> and <em>r</em><sub>A2</sub>). We present the arithmetic mean of <em>r</em><sub>A1</sub> and <em>r</em><sub>A2</sub> <a href="#pone.0001739-Lynch1">[4]</a>. The data used for estimating <em>r</em><sub>A1</sub> and <em>r</em><sub>A2</sub> were balanced in the sense that there were no missing values for trait X and Y in neither parents nor offspring. Thus, the calculation of <em>r</em><sub>A1</sub> and <em>r</em><sub>A2</sub> for trait X and Y are based on the same individual samples. To estimate the standard error of <em>r</em><sub>A</sub>, we applied the procedures described in Robertson <a href="#pone.0001739-Robertson1">[32]</a> and Falconer and Mackay <a href="#pone.0001739-Falconer1">[11]</a>.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Genetic correlations were also estimated with bivariate animal models based on both arithmetic means and repeated measures. The models included the significant fixed effects and random effects for each trait (estimated from univariate models). Genetic correlations were calculated only for traits that were observed to have significant additive genetic variance because <em>r</em><sub>A</sub> is theoretically undefined when one trait has heritability equal to zero (<a href="#pone.0001739-Lynch1">[4]</a>; see also <a href="#pone.0001739-Coltman1">[10]</a>). We include bill width in the genetic correlations due to its relatively high heritability that tended to be significant. Sample sizes are reported in <a href="#pone.0001739.s002">Table S2</a>.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Phenotypic correlations (<em>r</em><sub>P</sub>) were estimated for each pair of trait as the Pearson product moment correlation coefficient using the mean of the corrected phenotypic values for each individual (see above). Correlation coefficients and their standard errors were extracted from SPSS <a href="#pone.0001739-SPSS1">[33]</a>. Sample sizes are reported in <a href="#pone.0001739.s002">Table S2</a>. We also estimated phenotypic correlations in ASReml for both types of animal models by dividing the phenotypic covariances between the traits with the multiplied standard deviations.</p>


<h4>Statistics</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Parent-offspring analyses were conducted in SPSS version 14.0 <a href="#pone.0001739-SPSS1">[33]</a>, and the animal models in ASReml 2.0 <a href="#pone.0001739-Gilmour1">[29]</a>. In the animal models, the statistical significance of random factors was assessed by comparing the full model with the model without a random factor using the Akaike Information Criteria (see <a href="#pone.0001739-Gilmour1">[29]</a> for details). We kept random factors (i.e. maternal and common-nest effects) that affected the component of additive variance even if non-significant to avoid overestimation of the additive variance. Also, non-significant permanent environment effects were kept in the model to avoid the effects of pseudo-replication. The significance of differences between estimates of <em>h</em><sup>2</sup>, <em>r</em><sup>2</sup>, <em>m</em><sup>2</sup>, <em>b</em><sup>2</sup> and <em>r</em><sub>A</sub> from other estimates or from zero was assessed by calculating <em>z</em> scores<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.e001&amp;representation=PNG" class="inline-graphic"></span><br>where <em>x<sub>i</sub></em> and <em>x<sub>j</sub></em> are the two different estimates and <em>σ<sub>i</sub></em> and <em>σ<sub>j</sub></em> the respective standard errors. In the case of traits being tested against a value of zero the formula is reduced to the ratio between the estimate and the square-root of its standard error. The corresponding two-tailed significance level for <em>z</em> scores were taken from a large sample standard normal distribution.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>We compared the two methods to estimate of V<sub>P</sub>, V<sub>A</sub>, V<sub>R</sub>, V<sub>PE</sub>, <em>h</em><sup>2</sup> and standard error of <em>h</em><sup>2</sup> (SE(<em>h</em><sup>2</sup>)) using Spearman-rank correlation (<em>ρ</em>) and Wilcoxon signed-rank tests (<em>Z</em><sub>W</sub>). We used Wilcoxon signed-rank test and Pearson correlation (<em>r</em>) to test for the difference in elements of <em>r</em><sub>A </sub>matrices estimated by the three models. The significance of the Pearson correlation coefficient between <em>r</em><sub>A</sub> matrices was tested by using a resampling procedure (Mantel test; <a href="#pone.0001739-Sokal1">[34]</a>). The values of the two matrices were randomized N = 10,000 times and correlation coefficients calculated for each randomization were collected. The significance level is given by (n+1)/(N+1), were n is the number of randomized values that are equal to or more extreme than the observed correlation.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Comparing parent-offspring regression and mean traits animal model</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>There were significant heritabilities for 7 of the 8 traits, ranging between 0.39 and 0.97 for the parent-offspring model and between 0.32 and 0.84 for the animal model (<a href="#pone-0001739-t001">Table 1</a>). Bill depth heritability was non-significant for both methods (parent-offspring model; <em>h</em><sup>2</sup> = 0.07±0.16 SE; mean traits animal model: <em>h</em><sup>2</sup> = 0.06±0.12 SE).</p>
<div class="figure" id="pone-0001739-t001"><div class="img"><a name="pone-0001739-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001739" data-uri="info:doi/10.1371/journal.pone.0001739.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001739.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001739.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Heritabilities (<em>h</em><sup>2</sup>) and corresponding standard errors (SE) of eight morphological traits estimated from the different models.</span></strong></p><span>doi:10.1371/journal.pone.0001739.t001</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The heritabilities of the two methods were highly correlated (Spearman rank correlation: <em>ρ</em> = 0.88, N = 8, <em>P</em> = 0.004) and did not differ significantly in magnitude (Wilcoxon sign rank test: <em>Z</em><sub>W</sub> = 0.14, N = 8, <em>P</em> = 0.89; <a href="#pone-0001739-t002">Table 2</a>). The differences in <em>h</em><sup>2</sup> (parent-offspring <em>h</em><sup>2</sup> minus mean traits animal model <em>h</em><sup>2</sup>) ranged between −0.13 and 0.14. When analysing each trait separately there was no significant differences in <em>h</em><sup>2</sup> (range <em>z</em> = 0.003–1.00; range <em>P</em> = 0.32–0.997). Standard errors of heritabilities (SE(<em>h</em><sup>2</sup>)) from the two methods tended to be significantly correlated (<em>ρ</em> = 0.64, N = 8, <em>P</em> = 0.09) and the mean traits animal model generated on average 10.9 % higher standard error than those of parent-offspring (mean traits animal model: mean SE(<em>h</em><sup>2</sup>) 0.14±0.08 SD; parent-offspring model: mean SE(<em>h</em><sup>2</sup>) 0.12±0.02 SD), but over all traits the standard error of the two models did not differ significantly (<em>Z</em><sub>W</sub> = 0.28, N = 8, <em>P</em> = 0.78; <a href="#pone-0001739-t002">Table 2</a>).</p>
<div class="figure" id="pone-0001739-t002"><div class="img"><a name="pone-0001739-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001739" data-uri="info:doi/10.1371/journal.pone.0001739.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001739.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001739.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Correlations and overall differences between parent-offspring estimates of variance components and heritabilities and corresponding estimates generated from two animal models differing in use of replicated values or individual means.</span></strong></p><span>doi:10.1371/journal.pone.0001739.t002</span></div><a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>The phenotypic variance (V<sub>P</sub>) calculated using mean trait animal model was larger in seven of eight traits but very similar in magnitude (<a href="#pone-0001739-t002">Table 2</a>) compared to V<sub>P</sub> calculated using parent-offspring regression (<em>Z</em><sub>W</sub> = 2.10, N = 8, <em>P = </em>0.034; <a href="#pone-0001739-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0001739-g001"><div class="img"><a name="pone-0001739-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001739" data-uri="info:doi/10.1371/journal.pone.0001739.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001739.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001739.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Estimates of heritability (a) and coefficients of variation from three variance components, phenotypic variance (b), additive variance (c) and environmental variance (d), for eight morphological traits.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Each component was estimated from parent-offspring regression, mean traits animal model and repeated measures animal. WL = wing length; WP = wing projection; TL = tail length; BD = bill depth; BW = bill width; BL = bill length; SL = skull length; TR = tarsus length.</p>
<span>doi:10.1371/journal.pone.0001739.g001</span></div><a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>The two methods produced additive variances (V<sub>A</sub>) that were highly correlated (<em>ρ</em> = 1.00, N = 8, <em>P</em>&lt;0.001) and did not differ significantly in magnitude (<em>Z</em><sub>W</sub> = 0.14, N = 8, <em>P</em>  = 0.89; <a href="#pone-0001739-t002">Table 2</a>). The sum of the environmental variance components (V<sub>PE</sub>+V<sub>M</sub>+V<sub>B</sub>+V<sub>R</sub>) for each trait was significantly correlated between the two techniques (<em>ρ</em> = 0.83, N = 8, <em>P</em> = 0.010) and the difference in magnitude was non-significant (<em>Z</em><sub>W</sub><em> = </em>1.12, N = 8, <em>P = </em>0.26).</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>There was no significant maternal effect in any of the investigated traits. For all traits except tarsus length the maternal effect was locked at a minimum value (<a href="#pone.0001739.s001">Table S1</a>). Wing projection had a significant variance component due to a common-nest effect (V<sub>B</sub> = 0.66±0.16). However, due to a large standard error, the ratio between V<sub>B</sub> and the phenotypic variance (<em>b</em><sup>2</sup> = 0.40±0.31) was non-significant (<em>z</em> = 1.28, <em>P</em> = 0.2). Also, wing length showed a common-nest effect variance (V<sub>B</sub> = 0.22±0.23), however it did not differ significantly from zero (<a href="#pone.0001739.s001">Table S1</a>). The additive variance was affected very mildly by the incorporation of maternal and common-nest effects and the major part of these environmental components was extracted from the residual variance (data not reported).</p>


<h4>Comparing parent-offspring regression and repeated measures animal model</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Six of eight morphological traits that were estimated by repeated measures animal model showed significant additive variance (<a href="#pone.0001739.s001">Table S1</a>). The significant <em>h</em><sup>2</sup>s ranged from 0.27 to 0.72 with a mean of 0.54. The <em>h</em><sup>2</sup> of 0.20 of bill width tended towards significance (<em>z</em> = 2.33, <em>P = </em>0.08) whereas the <em>h</em><sup>2 </sup>of 0.05 in bill depth was far from significant (<em>z</em> = 0.60, <em>P</em> = 0.58).</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The <em>h</em><sup>2</sup>s from the repeated measures animal model were numerically lower than the <em>h</em><sup>2</sup>s calculated from parent-offspring regression in all the 8 traits (<a href="#pone-0001739-t001">Table 1</a>), but only significantly so for bill length (<em>z</em> = 2.16, <em>P</em> = 0.031). The difference between <em>h</em><sup>2</sup> from the parent-offspring model and <em>h</em><sup>2</sup> from the repeated measures animal model ranged between 0.01 and 0.26, corresponding to an average difference of 22.1 % of the parent-offspring estimate (repeated measures animal model: <em>h</em><sup>2</sup> = 0.44±0.24 SD; parent-offspring model: <em>h</em><sup>2</sup> = 0.56±0.24 SD; <em>Z</em><sub>W</sub><em> = </em>2.52, N = 8, <em>P</em> = 0.012; <a href="#pone-0001739-t002">Table 2</a>). The SE(<em>h</em><sup>2</sup>) estimated from the repeated measures animal model was lower for all traits compared to those of the parent-offspring model and differed on average 32.6 % (repeated measures animal model: mean SE(<em>h</em><sup>2</sup>) 0.08±0.03 SD; parent-offspring model: mean SE(<em>h</em><sup>2</sup>) 0.12±0.02 SD; <em>Z</em><sub>W</sub> = 2.52, N = 8, <em>P</em> = 0.012; <a href="#pone-0001739-t002">Table 2</a>).</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>The phenotypic variance (V<sub>P</sub>) calculated using repeated measures animal model was larger in all 8 traits compared to V<sub>P</sub> calculated using parent-offspring regression (<em>Z</em><sub>W</sub> = 2.52, N = 8, <em>P = </em>0.012; <a href="#pone-0001739-t002">Table 2</a>). The higher V<sub>P</sub> of the repeated measures animal model was caused by an increase in environmental variance (<em>Z</em><sub>W</sub> = 2.52, N = 8, <em>P</em> = 0.012; <a href="#pone-0001739-g001">Figure 1</a>). For all traits, except tail length, the major part of the increased environmental variance was due to the permanent environment variance, maternal effect variance and common-nest effect variance (<a href="#pone.0001739.s001">Table S1</a>). The higher phenotypic variance obtained when using repeated measures animal model was also a consequence of increased residual variance (V<sub>R</sub>) in wing projection, tail length and bill length.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>The repeated measures animal model generated V<sub>A </sub>values that were highly correlated with V<sub>A </sub>values from the parent-offspring model (<em>ρ</em> = 1.00, N = 8, <em>P</em>&lt;0.001; <a href="#pone-0001739-t002">Table 2</a>). For 7 of 8 traits, the V<sub>A</sub> was lower in the repeated measures animal model compared to parent-offspring model resulting in a significant overall difference in magnitude (repeated measures animal model: V<sub>A</sub> = 0.75; parent-offspring model: V<sub>A</sub> = 0.83; <em>Z</em><sub>W = </sub>2.10, N = 8, <em>P</em> = 0.036).</p>


<h4>Comparing trait correlations</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Genetic correlations (<em>r</em><sub>A</sub>) calculated with the parent-offspring model were significant in 9 of 21 cases and positive in all cases and the significant <em>r</em><sub>A </sub>values ranged between 0.34 and 0.75 (<a href="#pone-0001739-t003">Table 3</a>). The mean traits animal model gave <em>r</em><sub>A </sub>values that were positive in 18 of 21 cases and the six significant estimates ranged between 0.29 and 0.81 (<a href="#pone-0001739-t003">Table 3</a>). The two methods provided values of <em>r</em><sub>A</sub> that were significantly correlated between corresponding trait-pairs (Pearson correlation: <em>r</em> = 0.81, N = 21; Mantel test: <em>P</em>&lt;0.001; <a href="#pone-0001739-g002">Figure 2</a>). For 18 of 21 trait combinations higher estimates of <em>r</em><sub>A</sub> were calculated with the parent-offspring method (mean traits animal model: mean <em>r</em><sub>A</sub> = 0.22±0.24 SD; parent-offspring model: mean <em>r</em><sub>A</sub> = 0.37±0.22 SD; <em>Z</em><sub>W</sub><em> = </em>3.39, N = 21, <em>P</em>&lt;0.001). The standard errors of <em>r</em><sub>A</sub> generated from the two methods was highly correlated (<em>r</em> = 0.904, N = 21; <em>P</em>&lt;0.001) and the mean traits animal model generated overall (19 of 21 cases) smaller standard errors (mean traits animal model: SE(<em>r</em><sub>A</sub>) = 0.16±0.05 SD; parent-offspring model: SE(<em>r</em><sub>A</sub>) = 0.19±0.07 SD; <em>Z</em><sub>W</sub> = 3.60, N = 21, <em>P</em>&lt;0.001).</p>
<div class="figure" id="pone-0001739-g002"><div class="img"><a name="pone-0001739-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001739" data-uri="info:doi/10.1371/journal.pone.0001739.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001739.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001739.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Association between genetic correlations estimated from mean traits animal model and parent-offspring regression (a); and repeated measures animal model and mean traits animal model (b).</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>The dashed line represents the 1:1 relationship.</p>
<span>doi:10.1371/journal.pone.0001739.g002</span></div><div class="figure" id="pone-0001739-t003"><div class="img"><a name="pone-0001739-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001739" data-uri="info:doi/10.1371/journal.pone.0001739.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001739.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001739.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001739.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001739.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Phenotypic correlations (above the diagonal), additive genetic correlations (below the diagonal) among seven morphological traits in the great reed warbler, estimated from (a) parent-offspring regression (b) bivariate animal models using individual mean values and (c) bivariate animal models using repeated measures from the same individual.</span></strong></p><span>doi:10.1371/journal.pone.0001739.t003</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Phenotypic correlations between mean trait values within individuals (<em>r</em><sub>P</sub><sup>*</sup>) were positive in 20 of 21 cases and the 12 significant <em>r</em><sub>P</sub>*-values ranged between 0.11 and 0.57 (<a href="#pone-0001739-t003">Table 3</a>). These data are very similar to the results obtained when using the mean traits animal model approach. Phenotypic correlation calculated with the mean traits animal model was highly correlated with <em>r</em><sub>P</sub><sup>*</sup> for each trait pair (<em>r</em><sub> = </sub>0.997, N = 21; <em>P&lt;</em>0.001), but with a slight downward bias (mean traits animal model: mean <em>r</em><sub>P</sub> = 0.16±0.16 SD; mean <em>r</em><sub>P</sub><sup>*</sup> = 0.17±0.16 SD; <em>Z</em><sub>W</sub> = 2.14, N = 21, <em>P</em> = 0.033).</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Genetic correlations calculated from the repeated measures animal model were positive in 20 of 21 cases and significant in 6 of them (<a href="#pone-0001739-t003">Table 3</a>). Estimates of <em>r</em><sub>A</sub> from the parent-offspring model and <em>r</em><sub>A</sub> from repeated measures animal model were significantly correlated (<em>r = </em>0.87, N = 21; <em>P</em>&lt;0.001; <a href="#pone-0001739-g002">Figure 2</a>). However, in general the repeated measures animal model gave lower <em>r</em><sub>A</sub>s (18 of 21 trait combinations; repeated measures animal model: mean <em>r</em><sub>A</sub> = 0.26±0.22 SD; parent-offspring model: mean <em>r</em><sub>A</sub> = 0.37±0.22 SD; <em>Z</em><sub>W</sub><em> = </em>3.39, N = 21, <em>P</em>&lt;0.001), but none of these differences were significant. The SE(<em>r</em><sub>A</sub>)s calculated with repeated measures animal model were lower in 14 of 21 cases and were not significantly different from the SE(<em>r</em><sub>A</sub>)s from the parent-offspring analyses (repeated measures animal model: SE(<em>r</em><sub>A</sub>) = 0.18±0.07 SD; parent-offspring: SE(<em>r</em><sub>A</sub>) = 0.19±0.07 SD; <em>Z</em><sub>W</sub> = 1.20, N = 21, <em>P</em> = 0.23).</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>Phenotypic correlations calculated from the repeated measures animal model were positive in 20 of 21 cases and the 13 significant estimates ranged between 0.10 and 0.58 (<a href="#pone-0001739-t003">Table 3</a>). The <em>r</em><sub>P</sub> estimates from the repeated measures animal model correlated significantly with the corresponding <em>r</em><sub>P</sub><sup>*</sup> (<em>r</em> = 0.99, N = 21; <em>P</em>&lt;0.001), but tended towards having lower estimates (repeated measures animal model: mean <em>r</em><sub>P</sub> 0.16±0.16 SD; mean <em>r</em><sub>P</sub><sup>*</sup> = 0.17±0.16 SD; <em>Z</em><sub>W</sub> = 1.79, N = 21, <em>P</em> = 0.07).</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>We have compared parent-offspring regression estimates of heritability and genetic correlation with estimates obtained when using animal model. To our knowledge this is one of the first and most exhaustive study directly comparing these methods using the same data set in a population of free-ranging animals exposed to its natural environment (cf. <a href="#pone.0001739-Rale1">[12]</a>, <a href="#pone.0001739-Maccoll1">[13]</a>).</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>We found no overall difference in heritabilities and associated standard errors between parent-offspring regression and mean traits animal model even though the latter utilises much more extensive pedigree information than the former. In general, the heritability was very similar for the two methods. This similarity in <em>h</em><sup>2</sup> was also reflected by strong correlation in the additive variance (V<sub>A</sub>) as well as in the environmental variance (V<sub>M</sub>+V<sub>B</sub>+V<sub>R</sub>). Hence, the estimation of heritability and additive variance with parent-offspring regression were not seriously biased by, e.g. shared environment between parents and offspring. However, tarsus length and wing projection were the only traits that showed a parsimonious model (based on the Akaike Information Criteria) when including an environmental variance component (<a href="#pone.0001739.s001">Table S1</a>). Despite the maternal effect on tarsus length, the heritabilities were very similar for the two methods (0.73 and 0.72 for animal model and parent-offspring regression, respectively). It is worth noting that tarsus length was the only trait that was found to have significantly higher maternal inheritance than paternal inheritance in previous singleparent-midoffspring regression analyses (<a href="#pone.0001739-kesson1">[20]</a>; see also <a href="#pone.0001739-Hasselquist3">[35]</a>). The very similar estimates of heritability of tarsus length would thus suggest that parent-offspring regression is not seriously biased by the common environment shared by the mother and her offspring. However, in great reed warblers, tarsus length was only moderately affected by the mother's identity (<em>m</em><sup>2</sup> = 0.12, <a href="#pone.0001739.s001">Table S1</a>) and it is possible that traits with larger environmental variance (such as life-history traits) are subject to larger bias. In a review of fifteen cross-fostering experiments there was little evidence that shared environment between parents and offspring would seriously bias heritabilities <a href="#pone.0001739-Meril1">[36]</a>. Although the common-nest effect in wing projection is considerable (<em>b</em><sup>2</sup> = 0.40±0.31), this does not affect the differences in heritabilities of the two methods (wing projection <em>h</em><sup>2</sup>: 0.48 vs. 0.47). This is expected since the nest effect is the result of the environmental covariance between offspring from same nests and is not expected to influence the covariance between parents and offspring <a href="#pone.0001739-Falconer1">[11]</a>. The alternative explanation to the high similarity in heritabilities in wing projection calculated from the two methods is that the data-set with individual means do not offer enough power for the animal model to resolve biasing effects on the V<sub>A</sub> estimate. In a recent study <a href="#pone.0001739-Kruuk2">[7]</a> it is highlighted that even a fully specified animal model using considerable pedigree information may produce inflated heritabilities due to common nest effects, when these are considerable. Surprisingly, the standard errors of the heritabilities (SE(<em>h</em><sup>2</sup>)) were largely unaffected by using the mean traits animal model. The exception was SE(<em>h</em><sup>2</sup>) of wing projection that opposite to the prediction was more than twice the magnitude for the mean traits animal model.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The repeated measures animal model resulted in lower heritabilities for all traits as compared with results from parent-offspring regression, and the difference in <em>h</em><sup>2</sup> ranged between 0.01 and 0.26, including a more than 40 % reduction in <em>h</em><sup>2</sup> of wing projection and bill width. Furthermore, the repeated measures animal model gave lower sampling errors for all traits, with an average improved accuracy of 33 %. Two factors contributed to the reduction in heritability. First, the phenotypic variance (V<sub>P</sub>) was larger for the repeated measures animal model as compared with individual means. By using repeated measures a new source of variation is introduced into the model, which is the variation between measuring events of the same individual. This within-individual variance can be caused by a natural variation of a character, e.g. individual variation in development rate and phenotypic plasticity in response to different environmental conditions, but also from measurement errors. The repeatability (<em>r</em><sup>2</sup>) is the proportion of phenotypic variance that is due to variation within individuals <a href="#pone.0001739-Lynch1">[4]</a>, <a href="#pone.0001739-Falconer1">[11]</a>. The repeatability can be used as an indication of how much accuracy in the phenotypic trait might be gained by taking multiple measurements. Highly repeatable traits will only gain marginally in accuracy by using multiple measures, whereas traits with low repeatability may be more accurate if many measurements are taken. This is because the ratio between the phenotypic variance (V<sub>P(<em>n</em>)</sub>) derived from a data-set with average phenotypes and the phenotypic variance (V<sub>P</sub>) from a data-set with single measurements from the same population of individuals, depends on the repeatability (<em>r</em><sup>2</sup>) and number of measuring events (<em>n</em>) according to<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001739.e002&amp;representation=PNG" class="inline-graphic"></span><br><a href="#pone.0001739-Falconer1">[11]</a>. Thus, for a given <em>n</em>, the V<sub>P(<em>n</em>)</sub> is likely to be reduced more in relation to V<sub>P</sub> if the repeatability is low. As expected, there is a positive correlation (<em>r</em> = 0.835, N = 8, <em>P</em> = 0.01) between repeatability and the ratio of V<sub>P</sub> of mean values on repeated measures animal model V<sub>P</sub> (i.e. V<sub>P(<em>n</em>)</sub>/V<sub>P</sub>). Another reason for the lower <em>h</em><sup>2</sup> from repeated measures animal models compared to parent-offspring regressions is the former's lower additive variance (V<sub>A</sub>). This may also be a consequence of the within-individual variance as supported by the tendency to a negative correlation between the repeatability (<em>r</em><sub>A</sub>)) and the ratio of the parent-offspring model V<sub>A</sub> on repeated measures animal model V<sub>A</sub> (V<sub>A(<em>n</em>)</sub>/V<sub>A</sub>) (<em>r</em> = −0.70, N = 8, <em>P</em> = 0.06). Alternatively, the repeated measures animal model is more powerful in correcting for obscuring effects on the estimation of additive variance. These would be effects that violate the assumptions of the parent-offspring heritabilities such as non-random mating, selection, linkage disequilibrium, epistasis and environmental covariances. We found support for this explanation by the increased accuracy of the heritabilities when using repeated measures animal model compared to the mean traits animal model.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>The evolution of a quantitative trait depends on the magnitude of heritability but also on the genetic and environmental correlations with other traits <a href="#pone.0001739-Lynch1">[4]</a>, <a href="#pone.0001739-Falconer1">[11]</a>, <a href="#pone.0001739-Price1">[37]</a>. The genetic correlation shows to what extent two traits have a common genetic background due to pleiotropic effects and linkage disequilibrium (LD; <a href="#pone.0001739-Lynch1">[4]</a>). In the studied great reed warbler population, we have observed a relatively high level of LD (<a href="#pone.0001739-Hansson3">[21]</a>, Hansson, B. and Csilléry K, unpublished), which may have to do with the recent founder event and population expansion in the region <a href="#pone.0001739-Hansson4">[25]</a>. The genetic correlations between traits in the population may thus be partly due to LD between genes, partly due to pleiotropy. The genetic correlations estimated from the repeated measures animal model were largely positive, as has previously been observed in natural populations (e.g. <a href="#pone.0001739-Grant1">[38]</a>–<a href="#pone.0001739-Sheldon1">[40]</a>, but see <a href="#pone.0001739-Jensen1">[9]</a>).</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>Large sample sizes are generally required to accurately estimate genetic correlations since they often are subjected to large sampling errors <a href="#pone.0001739-Lynch1">[4]</a>, <a href="#pone.0001739-Roff1">[41]</a>. Also, estimates of genetic correlations from parent-offspring relationships are easily biased by maternal effects and selection <a href="#pone.0001739-Lande1">[42]</a>. In the present study, we estimated the genetic correlations between 7 traits and compared the parent-offspring approach with the animal model approach (using either individual mean values or repeated measures). These three methods generated highly correlated estimates of the genetic correlations, although there were some differences in overall magnitude. Both animal models generated lower genetic correlations in 18 of 21 trait correlations compared to the parent-offspring model. It is possible that the genetic correlations estimated from the latter model are biased by either shared environment between parents and offspring or by selection acting on the traits (see <a href="#pone.0001739-kesson1">[20]</a>). That the animal models are less biased by such factors seem at least partly to be explained by the use of a large pedigree, because we observed a similar reduction by both types of animal models. However, standard errors were overall lower only in the animal models using mean values. Apparently repeated measures animal models offer less biased estimates of <em>r</em><sub>A</sub>, but does not manage to reduce the sampling error. Alternatively, the standard errors of genetic correlations from the parent-offspring regression are underestimated. The sampling error of genetic correlations <a href="#pone.0001739-Robertson1">[32]</a>, <a href="#pone.0001739-Reeve1">[43]</a> from parent-offspring model is complicated and to a large extent unresolved matter <a href="#pone.0001739-Windig1">[44]</a>. Simulation studies have shown that the sampling error of genetic correlations may be seriously underestimated for sample sizes under 100, especially if the corresponding heritabilities of the traits are low or genetic correlations are high <a href="#pone.0001739-VanVleck1">[45]</a>, <a href="#pone.0001739-Brown1">[46]</a>.</p>

<h4>Conclusion</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Our results suggest that the increased accuracy of heritability estimates when using animal model is mostly due to the inclusion of repeated measures, and that the heritability estimates appear to be lower when using repeated measures animal models. We did not observe any bias caused by the maternal environment on <em>h</em><sup>2</sup>, but it should be kept in mind that only one of the 8 investigated traits showed maternal effects. It should also be kept in mind that morphological traits generally show low levels of dominance variance and that our results may not be applicable to other types of traits, such as life-history traits, that are known to be affected to a larger extent by dominance and epistatis <a href="#pone.0001739-Lynch1">[4]</a>. The lower additive variance from the repeated measures animal model is also likely to be due to the within-individual variance of each trait, as indicated by the tendency for a negative correlation between repeatability and ratio of additive variances between the two methods. This implies that additive variances would be overestimated by parent-offspring regression and mean trait animal models when there is natural variation in trait expression within individuals and when there are measurement errors.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Genetic correlations appear to be lower but more accurate (i.e. having lower standard errors) when estimated by the either of the two animal models than by parent-offspring models. This suggests that genetic correlations from parent-offspring models are sensitive to biasing effects such as selection and environmental covariance between relatives, and highlights the importance of taking into account all relatives in a pedigree when estimating genetic correlations.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>The reconciliation of results from different studies using different estimation procedures depends on finding out and taking potential methodological discrepancies into account before comparing the data (see e.g. <a href="#pone.0001739-Mousseau1">[3]</a>, <a href="#pone.0001739-Houle1">[8]</a>). Only few studies that evaluate the animal model and parent-offspring regressions have been made previously and then on rather limited data <a href="#pone.0001739-Rale1">[12]</a>, <a href="#pone.0001739-Maccoll1">[13]</a>. The present study thus provides important knowledge for future meta-analyses aiming at understanding the concept of evolutionary potential.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001739"></div><a name="pone.0001739.s001" id="pone.0001739.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001739.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">The quantitative genetic parameters for eight morphological traits in the great reed warbler estimated from parent-offspring regression, mean traits animal model and repeated measures animal model.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001739.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.10 MB DOC)</p>
<a name="pone.0001739.s002" id="pone.0001739.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001739.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Sample sizes for trait correlations and genetic covariances among seven morphological traits in the great reed warbler.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001739.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Kathi Foerster and Loeske Kruuk for help with running ASReml and for providing useful comments on the analyses. We thank all the field assistants (in particular Bo Nielsen) for their invaluable efforts in the field. This is report no. 147 from Kvismare Bird Observatory.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SB BH DH MÅ. Analyzed the data: MÅ. Contributed reagents/materials/analysis tools: SB DH. Wrote the paper: BH MÅ. Other: Organized field work and data collection: BH MT DH SB MÅ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001739-Galton1" id="pone.0001739-Galton1"></a>Galton F (1889) Natural inheritance. London: Macmillan.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001739-Provine1" id="pone.0001739-Provine1"></a>Provine WB (2001) The origins of theoretical population genetics. Chicago: University of Chicago Press.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0001739-Mousseau1" id="pone.0001739-Mousseau1"></a>Mousseau TA, Roff DA (1987) Natural selection and the heritability of fitness components. Heredity  59: 181–197.  <ul class="find" data-citedArticleID="1068852" data-doi="10.1038/hdy.1987.113"><li><a href="http://dx.doi.org/10.1038/hdy.1987.113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+and+the+heritability+of+fitness+components." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+and+the+heritability+of+fitness+components.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001739-Lynch1" id="pone.0001739-Lynch1"></a>Lynch M, Walsh B (1998) Genetics and the analysis of quantitative traits. Sunderland, MA: Sinauer associates.   <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0001739-Knott1" id="pone.0001739-Knott1"></a>Knott SA, Sibly RM, Smith RH, Møller AP (1995) Maximum likelihood estimation of genetic parameters in life-history studies using the ‘animal model’. Functional Ecology  9: 122–126.  <ul class="find" data-citedArticleID="1068832" data-doi="10.2307/2390099"><li><a href="http://dx.doi.org/10.2307/2390099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum+likelihood+estimation+of+genetic+parameters+in+life-history+studies+using+the+%E2%80%98animal+model%E2%80%99." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum+likelihood+estimation+of+genetic+parameters+in+life-history+studies+using+the+%E2%80%98animal+model%E2%80%99.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001739-Kruuk1" id="pone.0001739-Kruuk1"></a>Kruuk LEB (2004) Estimating genetic parameters in natural populations using the ‘animal model’. Philo T Roy Soc B  359: 873–890.  <ul class="find" data-citedArticleID="1068834" data-doi="10.1098/rstb.2003.1437"><li><a href="http://dx.doi.org/10.1098/rstb.2003.1437" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+genetic+parameters+in+natural+populations+using+the+%E2%80%98animal+model%E2%80%99." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+genetic+parameters+in+natural+populations+using+the+%E2%80%98animal+model%E2%80%99.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001739-Kruuk2" id="pone.0001739-Kruuk2"></a>Kruuk LEB, Hadfield JD (2007) How to separate genetic and environmental causes of similarity between relatives. J Evol Biol  20: 1890–1903.  <ul class="find" data-citedArticleID="1068836" data-doi="10.1111/j.1420-9101.2007.01377.x"><li><a href="http://dx.doi.org/10.1111/j.1420-9101.2007.01377.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+to+separate+genetic+and+environmental+causes+of+similarity+between+relatives." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+to+separate+genetic+and+environmental+causes+of+similarity+between+relatives.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001739-Houle1" id="pone.0001739-Houle1"></a>Houle D (1992) Comparing evolvability and variability of quantitative traits. Genetics  130: 195–204.  <ul class="find" data-citedArticleID="1068828"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparing+evolvability+and+variability+of+quantitative+traits.&amp;auth=&amp;atitle=Comparing+evolvability+and+variability+of+quantitative+traits." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparing+evolvability+and+variability+of+quantitative+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparing+evolvability+and+variability+of+quantitative+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001739-Jensen1" id="pone.0001739-Jensen1"></a>Jensen H, Saether BE, Ringsby TH, Tufto J, Griffith SC, Ellegren H (2003) Sexual variation in heritability and genetic correlations of morphological traits in house sparrow (<em>Passer domesticus</em>). J Evol Biol  16: 1296–1307.  <ul class="find" data-citedArticleID="1068830" data-doi="10.1046/j.1420-9101.2003.00614.x"><li><a href="http://dx.doi.org/10.1046/j.1420-9101.2003.00614.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sexual+variation+in+heritability+and+genetic+correlations+of+morphological+traits+in+house+sparrow+%28Passer+domesticus%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sexual+variation+in+heritability+and+genetic+correlations+of+morphological+traits+in+house+sparrow+%28Passer+domesticus%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001739-Coltman1" id="pone.0001739-Coltman1"></a>Coltman DW, O'Donoghue P, Hogg JT, Festa-Bianchet M (2005) Selection and genetic (co)variance in bighorn sheep. Evolution  59: 1372–1382.  <ul class="find" data-citedArticleID="1068800" data-doi="10.1554/04-134"><li><a href="http://dx.doi.org/10.1554/04-134" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+and+genetic+%28co%29variance+in+bighorn+sheep." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+and+genetic+%28co%29variance+in+bighorn+sheep.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001739-Falconer1" id="pone.0001739-Falconer1"></a>Falconer DS, Mackay TFC (1996) Introduction to quantitative genetics. Harlow, UK: Longman Group Ltd.   <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0001739-Rale1" id="pone.0001739-Rale1"></a>Réale D, Festa-Bianchet M, Jorgenson JT (1999) Heritability of body mass varies with age and season in wild bighorn sheep. Heredity  83: 526–532.  <ul class="find" data-citedArticleID="1068858" data-doi="10.1038/sj.hdy.6885430"><li><a href="http://dx.doi.org/10.1038/sj.hdy.6885430" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heritability+of+body+mass+varies+with+age+and+season+in+wild+bighorn+sheep." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heritability+of+body+mass+varies+with+age+and+season+in+wild+bighorn+sheep.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001739-Maccoll1" id="pone.0001739-Maccoll1"></a>Maccoll ADC, Hatchwell BJ (2003) Heritability of parental effort in a passerine bird. Evolution  57: 2191–2195.  <ul class="find" data-citedArticleID="1068846" data-doi="10.1111/j.0014-3820.2003.tb00398.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2003.tb00398.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heritability+of+parental+effort+in+a+passerine+bird." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heritability+of+parental+effort+in+a+passerine+bird.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001739-Bensch1" id="pone.0001739-Bensch1"></a>Bensch S, Hasselquist D (1991) Territory infidelity in the polygynous great reed warbler <em>Acrocephalus arundinaceus</em>: the effect of variation in territory attractiveness. J Anim Ecol  60: 857–871.  <ul class="find" data-citedArticleID="1068794" data-doi="10.2307/5418"><li><a href="http://dx.doi.org/10.2307/5418" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Territory+infidelity+in+the+polygynous+great+reed+warbler+Acrocephalus+arundinaceus%3A+the+effect+of+variation+in+territory+attractiveness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Territory+infidelity+in+the+polygynous+great+reed+warbler+Acrocephalus+arundinaceus%3A+the+effect+of+variation+in+territory+attractiveness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001739-Bensch2" id="pone.0001739-Bensch2"></a>Bensch S, Hasselquist D, Nielsen B, Hansson B (1998) Higher fitness for philopatric than for immigrant males in a semi-isolated population of great reed warblers. Evolution  52: 877–883.  <ul class="find" data-citedArticleID="1068796" data-doi="10.2307/2411282"><li><a href="http://dx.doi.org/10.2307/2411282" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Higher+fitness+for+philopatric+than+for+immigrant+males+in+a+semi-isolated+population+of+great+reed+warblers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Higher+fitness+for+philopatric+than+for+immigrant+males+in+a+semi-isolated+population+of+great+reed+warblers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001739-Hansson1" id="pone.0001739-Hansson1"></a>Hansson B, Bensch S, Hasselquist D (2004) Lifetime fitness of short- and long-distance dispersing great reed warblers. Evolution  58: 2546–2557.  <ul class="find" data-citedArticleID="1068812" data-doi="10.1554/04-083"><li><a href="http://dx.doi.org/10.1554/04-083" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lifetime+fitness+of+short-+and+long-distance+dispersing+great+reed+warblers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lifetime+fitness+of+short-+and+long-distance+dispersing+great+reed+warblers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001739-Hasselquist1" id="pone.0001739-Hasselquist1"></a>Hasselquist D, Bensch S, von Schantz T (1995) Low frequency of extra-pair paternity in the polygynous great reed warbler, <em>Acrocephalus arundinaceus</em>. Behav Ecol  6: 27–38.  <ul class="find" data-citedArticleID="1068820" data-doi="10.1093/beheco/6.1.27"><li><a href="http://dx.doi.org/10.1093/beheco/6.1.27" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Low+frequency+of+extra-pair+paternity+in+the+polygynous+great+reed+warbler%2C+Acrocephalus+arundinaceus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Low+frequency+of+extra-pair+paternity+in+the+polygynous+great+reed+warbler%2C+Acrocephalus+arundinaceus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001739-Arlt1" id="pone.0001739-Arlt1"></a>Arlt D, Hansson B, Bensch S, von Schantz T, Hasselquist D (2004) Breeding synchrony does not affect extra-pair paternity in great reed warblers. Behaviour  141: 863–880.  <ul class="find" data-citedArticleID="1068790" data-doi="10.1163/1568539042265699"><li><a href="http://dx.doi.org/10.1163/1568539042265699" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Breeding+synchrony+does+not+affect+extra-pair+paternity+in+great+reed+warblers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Breeding+synchrony+does+not+affect+extra-pair+paternity+in+great+reed+warblers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001739-Hansson2" id="pone.0001739-Hansson2"></a>Hansson B, Hasselquist D, Bensch S (2004) Do female great reed warblers seek extra-pair fertilizations to avoid inbreeding? Biology Letters  271: 290–292.  <ul class="find" data-citedArticleID="1068814" data-doi="10.1098/rsbl.2004.0164"><li><a href="http://dx.doi.org/10.1098/rsbl.2004.0164" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Do+female+great+reed+warblers+seek+extra-pair+fertilizations+to+avoid+inbreeding%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Do+female+great+reed+warblers+seek+extra-pair+fertilizations+to+avoid+inbreeding%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001739-kesson1" id="pone.0001739-kesson1"></a>Åkesson M, Bensch S, Hasselquist D (2007) Genetic and phenotypic associations in morphological traits: a long term study of great reed warblers <em>Acrocephalus arundinaceus</em>. J Avian Biol  38: 58–72.  <ul class="find" data-citedArticleID="1068880" data-doi="10.1111/j.2006.0908-8857.03669.x"><li><a href="http://dx.doi.org/10.1111/j.2006.0908-8857.03669.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+and+phenotypic+associations+in+morphological+traits%3A+a+long+term+study+of+great+reed+warblers+Acrocephalus+arundinaceus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+and+phenotypic+associations+in+morphological+traits%3A+a+long+term+study+of+great+reed+warblers+Acrocephalus+arundinaceus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001739-Hansson3" id="pone.0001739-Hansson3"></a>Hansson B, Westerdahl H, Hasselquist D, Åkesson M, Bensch S (2004) Does linkage disequilibrium generate heterozygosity-fitness correlations in great reed warblers? Evolution  58: 870–879.  <ul class="find" data-citedArticleID="1068816" data-doi="10.1554/03-368"><li><a href="http://dx.doi.org/10.1554/03-368" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Does+linkage+disequilibrium+generate+heterozygosity-fitness+correlations+in+great+reed+warblers%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Does+linkage+disequilibrium+generate+heterozygosity-fitness+correlations+in+great+reed+warblers%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001739-Helbig1" id="pone.0001739-Helbig1"></a>Helbig AJ, Seibold I (1999) Molecular phylogeny of Palearctic-African <em>Acrocephalus</em> and <em>Hippolais</em> warblers (Aves : Sylviidae). Mol Phylogenet Evol  11: 246–260.  <ul class="find" data-citedArticleID="1068826" data-doi="10.1006/mpev.1998.0571"><li><a href="http://dx.doi.org/10.1006/mpev.1998.0571" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogeny+of+Palearctic-African+Acrocephalus+and+Hippolais+warblers+%28Aves+%3A+Sylviidae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogeny+of+Palearctic-African+Acrocephalus+and+Hippolais+warblers+%28Aves+%3A+Sylviidae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001739-Cramp1" id="pone.0001739-Cramp1"></a>Cramp S (1992) Handbook of the birds of Europe, the middle East and North Africa. New York: Oxford Univerity Press.   <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0001739-Hasselquist2" id="pone.0001739-Hasselquist2"></a>Hasselquist D (1998) Polygyny in great reed warblers: a long-term study of factors contributing to male fitness. Ecology  79: 2376–2390.  <ul class="find" data-citedArticleID="1068822" data-doi="10.1890/0012-9658(1998)079[2376:pigrwa]2.0.co;2"><li><a href="http://dx.doi.org/10.1890/0012-9658(1998)079[2376:pigrwa]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polygyny+in+great+reed+warblers%3A+a+long-term+study+of+factors+contributing+to+male+fitness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polygyny+in+great+reed+warblers%3A+a+long-term+study+of+factors+contributing+to+male+fitness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001739-Hansson4" id="pone.0001739-Hansson4"></a>Hansson B, Bensch S, Hasselquist D, Lillandt BG, Wennerberg L, von Schantz T (2000) Increase of genetic variation over time in a recently founded population of great reed warblers (<em>Acrocephalus arundinaceus</em>) revealed by microsatellites and DNA fingerprinting. Mol Ecol  9: 1529–1538.  <ul class="find" data-citedArticleID="1068818" data-doi="10.1046/j.1365-294x.2000.01028.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2000.01028.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Increase+of+genetic+variation+over+time+in+a+recently+founded+population+of+great+reed+warblers+%28Acrocephalus+arundinaceus%29+revealed+by+microsatellites+and+DNA+fingerprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Increase+of+genetic+variation+over+time+in+a+recently+founded+population+of+great+reed+warblers+%28Acrocephalus+arundinaceus%29+revealed+by+microsatellites+and+DNA+fingerprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001739-Wigginton1" id="pone.0001739-Wigginton1"></a>Wigginton JE, Abecasis GR (2005) PEDSTATS: descriptive statistics, graphics and quality assessment for gene mapping data. Bioinformatics  21: 3445–3447.  <ul class="find" data-citedArticleID="1068876" data-doi="10.1093/bioinformatics/bti529"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti529" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PEDSTATS%3A+descriptive+statistics%2C+graphics+and+quality+assessment+for+gene+mapping+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PEDSTATS%3A+descriptive+statistics%2C+graphics+and+quality+assessment+for+gene+mapping+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001739-Svensson1" id="pone.0001739-Svensson1"></a>Svensson L (1992) Identification guide to European passerines. Stockholm: Fingraf.   <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0001739-Littell1" id="pone.0001739-Littell1"></a>Littell RC, Milliken GA, Stroup WW, Wolfinger RD (1996) SAS System for Mixed Models. Cary, NC: SAS Institute Inc.   <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0001739-Gilmour1" id="pone.0001739-Gilmour1"></a>Gilmour AR, Gogel BJ, Cullis BR, Welham SJ, Thompson R (2006) ASreml user guide. Release 1.0. Hemel Hempstead, UK: VSN International.   <ul class="find-nolinks"></ul></li><li><span class="label">30.
              </span><a name="pone.0001739-Lessells1" id="pone.0001739-Lessells1"></a>Lessells CM, Boag PT (1987) Unrepeatable repeatabilities: a common mistake. Auk  104: 116–121.  <ul class="find" data-citedArticleID="1068840" data-doi="10.2307/4087240"><li><a href="http://dx.doi.org/10.2307/4087240" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unrepeatable+repeatabilities%3A+a+common+mistake." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unrepeatable+repeatabilities%3A+a+common+mistake.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001739-Becker1" id="pone.0001739-Becker1"></a>Becker WA (1984) Manual of quantitative genetics. Pullman, WA: Academic Enterprises.   <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0001739-Robertson1" id="pone.0001739-Robertson1"></a>Robertson A (1959) The sampling variance of the genetic correlation coefficient. Biometrics  15: 469–485.  <ul class="find" data-citedArticleID="1068862" data-doi="10.2307/2527750"><li><a href="http://dx.doi.org/10.2307/2527750" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sampling+variance+of+the+genetic+correlation+coefficient." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sampling+variance+of+the+genetic+correlation+coefficient.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001739-SPSS1" id="pone.0001739-SPSS1"></a>SPSS Inc. (2005) SPSS 14.0 for windows.   <ul class="find-nolinks"></ul></li><li><span class="label">34.
              </span><a name="pone.0001739-Sokal1" id="pone.0001739-Sokal1"></a>Sokal RR, Rohlf FJ (1995) Biometry: the principles and practice of statistics in bilogical research. NY: W.H. Freeman and Co.   <ul class="find-nolinks"></ul></li><li><span class="label">35.
              </span><a name="pone.0001739-Hasselquist3" id="pone.0001739-Hasselquist3"></a>Hasselquist D, Bensch S, von Schantz T (1995) Estimating cuckoldry in birds: the heritability method and DNA-fingerprinting give different results. Oikos  72: 173–178.  <ul class="find" data-citedArticleID="1068824" data-doi="10.2307/3546219"><li><a href="http://dx.doi.org/10.2307/3546219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+cuckoldry+in+birds%3A+the+heritability+method+and+DNA-fingerprinting+give+different+results." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+cuckoldry+in+birds%3A+the+heritability+method+and+DNA-fingerprinting+give+different+results.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001739-Meril1" id="pone.0001739-Meril1"></a>Merilä J, Sheldon BC (2001)  Avian quantitative genetics. In: Nolan V Jr, Ketterson ED, Thompson CF, editors. Current ornithology. New York: Kluwer Academic/Plenum Publishers. pp. 179–255.  <ul class="find-nolinks"></ul></li><li><span class="label">37.
              </span><a name="pone.0001739-Price1" id="pone.0001739-Price1"></a>Price T, Langen T (1992) Evolution of correlated characters. TREE  7: 307–310.  <ul class="find" data-citedArticleID="1068854" data-doi="10.1016/0169-5347(92)90229-5"><li><a href="http://dx.doi.org/10.1016/0169-5347(92)90229-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+correlated+characters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+correlated+characters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001739-Grant1" id="pone.0001739-Grant1"></a>Grant PR (1983) Inheritance of size and shape in a population of Darwin Finches <em>Geospiza conirostris</em>. Proc R Soc Lond B  220: 219–236.  <ul class="find" data-citedArticleID="1068810" data-doi="10.1098/rspb.1983.0096"><li><a href="http://dx.doi.org/10.1098/rspb.1983.0096" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inheritance+of+size+and+shape+in+a+population+of+Darwin+Finches+Geospiza+conirostris." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inheritance+of+size+and+shape+in+a+population+of+Darwin+Finches+Geospiza+conirostris.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001739-Meril2" id="pone.0001739-Meril2"></a>Merilä J, Gustafsson L (1993) Inheritance of size and shape in a natural population of Collared Flycatchers <em>Ficedula albicollis</em>. J Evol Biol  6: 375–395.  <ul class="find" data-citedArticleID="1068850" data-doi="10.1046/j.1420-9101.1993.6030375.x"><li><a href="http://dx.doi.org/10.1046/j.1420-9101.1993.6030375.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inheritance+of+size+and+shape+in+a+natural+population+of+Collared+Flycatchers+Ficedula+albicollis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inheritance+of+size+and+shape+in+a+natural+population+of+Collared+Flycatchers+Ficedula+albicollis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001739-Sheldon1" id="pone.0001739-Sheldon1"></a>Sheldon BC, Kruuk LEB, Merilä J (2003) Natural selection and inheritance of breeding time and clutch size in the collared flycatcher. Evolution  57: 406–420.  <ul class="find" data-citedArticleID="1068868" data-doi="10.1111/j.0014-3820.2003.tb00274.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2003.tb00274.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+and+inheritance+of+breeding+time+and+clutch+size+in+the+collared+flycatcher." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+and+inheritance+of+breeding+time+and+clutch+size+in+the+collared+flycatcher.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001739-Roff1" id="pone.0001739-Roff1"></a>Roff DA (1997) Evolutionary quantitative genetics. London: Academic Press.   <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0001739-Lande1" id="pone.0001739-Lande1"></a>Lande R, Price T (1989) Genetic correlations and maternal effect coefficients obtained from offspring-parent regression. Genetics  122: 915–922.  <ul class="find" data-citedArticleID="1068838"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+correlations+and+maternal+effect+coefficients+obtained+from+offspring-parent+regression.&amp;auth=&amp;atitle=Genetic+correlations+and+maternal+effect+coefficients+obtained+from+offspring-parent+regression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+correlations+and+maternal+effect+coefficients+obtained+from+offspring-parent+regression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+correlations+and+maternal+effect+coefficients+obtained+from+offspring-parent+regression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001739-Reeve1" id="pone.0001739-Reeve1"></a>Reeve ECR (1955) The variance of the genetic correlation coefficient. Biometrics  11: 357–374.  <ul class="find" data-citedArticleID="1068860" data-doi="10.2307/3001774"><li><a href="http://dx.doi.org/10.2307/3001774" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+variance+of+the+genetic+correlation+coefficient." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+variance+of+the+genetic+correlation+coefficient.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001739-Windig1" id="pone.0001739-Windig1"></a>Windig JJ (1997) The calculation and significance testing of genetic correlations across environments. J Evol Biol  10: 853–874.  <ul class="find" data-citedArticleID="1068878" data-doi="10.1007/s000360050058"><li><a href="http://dx.doi.org/10.1007/s000360050058" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+calculation+and+significance+testing+of+genetic+correlations+across+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+calculation+and+significance+testing+of+genetic+correlations+across+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001739-VanVleck1" id="pone.0001739-VanVleck1"></a>Van Vleck LD, Henderson CR (1961) Empirical sampling estimates of genetic correlations. Biometrics  17: 359–371.  <ul class="find" data-citedArticleID="1068874" data-doi="10.2307/2527831"><li><a href="http://dx.doi.org/10.2307/2527831" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Empirical+sampling+estimates+of+genetic+correlations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Empirical+sampling+estimates+of+genetic+correlations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001739-Brown1" id="pone.0001739-Brown1"></a>Brown GH (1969) An empirical study of distribution of the sample genetic correlation coefficient. Biometrics  25: 63–72.  <ul class="find" data-citedArticleID="1068798" data-doi="10.2307/2528678"><li><a href="http://dx.doi.org/10.2307/2528678" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+empirical+study+of+distribution+of+the+sample+genetic+correlation+coefficient." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+empirical+study+of+distribution+of+the+sample+genetic+correlation+coefficient.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.XML" value="117996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.PDF" value="256643"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.e001.PNG" value="8126"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.e001.TIF" value="23960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t001.PNG_L" value="113333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t001.PNG_M" value="67055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t001.PNG_S" value="9596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t001.TIF" value="552230"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t001.PNG_I" value="20619"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t002.PNG_L" value="154111"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t002.PNG_M" value="91412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t002.PNG_S" value="10463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t002.TIF" value="687008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t002.PNG_I" value="28930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.g001.PNG_L" value="547550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.g001.PNG_M" value="50393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.g001.PNG_S" value="9147"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.g001.TIF" value="1723982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.g001.PNG_I" value="18521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.g002.PNG_L" value="193024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.g002.PNG_M" value="27701"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.g002.PNG_S" value="5744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.g002.TIF" value="658974"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.g002.PNG_I" value="10516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t003.PNG_L" value="308811"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t003.PNG_M" value="182761"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t003.PNG_S" value="18145"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t003.TIF" value="1303204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.t003.PNG_I" value="56887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.e002.PNG" value="8556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.e002.TIF" value="24480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.s001.DOC" value="99840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001739.s002.DOC" value="49152"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001739&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001739" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001739&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001739&volume=&issue=&title=Estimating Heritabilities and Genetic Correlations: Comparing the ‘Animal Model’ with Parent-Offspring Regression Using Data from a Natural Population&author_name=Mikael%20%C3%85kesson%2C%20Staffan%20Bensch%2C%20Dennis%20Hasselquist%2C%20Maja%20Tarka%2C%20Bengt%20Hansson&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001739" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001739" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001739" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001739&amp;t=Estimating%20Heritabilities%20and%20Genetic%20Correlations%3A%20Comparing%20the%20%E2%80%98Animal%20Model%E2%80%99%20with%20Parent-Offspring%20Regression%20Using%20Data%20from%20a%20Natural%20Population" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001739&title=Estimating%20Heritabilities%20and%20Genetic%20Correlations%3A%20Comparing%20the%20%E2%80%98Animal%20Model%E2%80%99%20with%20Parent-Offspring%20Regression%20Using%20Data%20from%20a%20Natural%20Population&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001739&amp;title=Estimating%20Heritabilities%20and%20Genetic%20Correlations%3A%20Comparing%20the%20%E2%80%98Animal%20Model%E2%80%99%20with%20Parent-Offspring%20Regression%20Using%20Data%20from%20a%20Natural%20Population" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001739" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001739&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Estimating Heritabilities and Genetic Correlations: Comparing the ‘Animal Model’ with Parent-Offspring Regression Using Data from a Natural Population';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001739';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Estimating%20Heritabilities%20and%20Genetic%20Correlations%3A%20Comparing%20the%20%E2%80%98Animal%20Model%E2%80%99%20with%20Parent-Offspring%20Regression%20Using%20Data%20from%20a%20Natural%20Population http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001739" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001739" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+evolution%22" title="Search for articles in the subject area:'Animal evolution'"><div class="flagText">Animal evolution</div></a>
              <div data-categoryid="18779" data-articleid="26302"
                   data-categoryname="Animal evolution"
                   class="flagImage" title="Flag 'Animal evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+models%22" title="Search for articles in the subject area:'Animal models'"><div class="flagText">Animal models</div></a>
              <div data-categoryid="34389" data-articleid="26302"
                   data-categoryname="Animal models"
                   class="flagImage" title="Flag 'Animal models' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+sexual+behavior%22" title="Search for articles in the subject area:'Animal sexual behavior'"><div class="flagText">Animal sexual behavior</div></a>
              <div data-categoryid="21247" data-articleid="26302"
                   data-categoryname="Animal sexual behavior"
                   class="flagImage" title="Flag 'Animal sexual behavior' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="26302"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+modeling%22" title="Search for articles in the subject area:'Evolutionary modeling'"><div class="flagText">Evolutionary modeling</div></a>
              <div data-categoryid="18005" data-articleid="26302"
                   data-categoryname="Evolutionary modeling"
                   class="flagImage" title="Flag 'Evolutionary modeling' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="26302"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="26302"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Wings%22" title="Search for articles in the subject area:'Wings'"><div class="flagText">Wings</div></a>
              <div data-categoryid="48473" data-articleid="26302"
                   data-categoryname="Wings"
                   class="flagImage" title="Flag 'Wings' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3855'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1652'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=153&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=15689">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=12415">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
