

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Sorting Signals, N-Terminal Modifications and Abundance of the Chloroplast Proteome</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001994"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001994" />

    <meta name="citation_title" content="Sorting Signals, N-Terminal Modifications and Abundance of the Chloroplast Proteome"/>
    <meta itemprop="name" content="Sorting Signals, N-Terminal Modifications and Abundance of the Chloroplast Proteome"/>

      <meta name="citation_author" content="Boris Zybailov"/>
            <meta name="citation_author_institution" content="Department of Plant Biology, Cornell University, Ithaca, New York, United States of America"/>
      <meta name="citation_author" content="Heidi Rutschow"/>
            <meta name="citation_author_institution" content="Department of Plant Biology, Cornell University, Ithaca, New York, United States of America"/>
      <meta name="citation_author" content="Giulia Friso"/>
            <meta name="citation_author_institution" content="Department of Plant Biology, Cornell University, Ithaca, New York, United States of America"/>
      <meta name="citation_author" content="Andrea Rudella"/>
            <meta name="citation_author_institution" content="Department of Plant Biology, Cornell University, Ithaca, New York, United States of America"/>
      <meta name="citation_author" content="Olof Emanuelsson"/>
            <meta name="citation_author_institution" content="Stockholm Bioinformatics Center, AlbaNova, Stockholm University, Stockholm, Sweden"/>
      <meta name="citation_author" content="Qi Sun"/>
            <meta name="citation_author_institution" content="Computation Biology Service Unit, Cornell Theory Center, Cornell University, Ithaca, New York, United States of America"/>
      <meta name="citation_author" content="Klaas J. van Wijk"/>
            <meta name="citation_author_institution" content="Department of Plant Biology, Cornell University, Ithaca, New York, United States of America"/>

    <meta name="citation_date" content="2008/4/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001994.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1994"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Domain structure of mitochondrial and chloroplast targeting peptides.; citation_author=G von Heijne; citation_author=J Steppuhn; citation_author=SG Hermann; citation_journal_title=Eur J Biochemistry; citation_volume=80; citation_number=1; citation_pages=535-545; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Locating proteins in the cell using TargetP, SignalP and related tools.; citation_author=O Emanuelsson; citation_author=S Brunak; citation_author=G von Heijne; citation_author=H Nielsen; citation_journal_title=Nat Protoc; citation_volume=2(4); citation_number=2; citation_pages=953-971; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=An improved prediction of chloroplast proteins reveals diversities and commonalities in the chloroplast proteomes of Arabidopsis and rice.; citation_author=E Richly; citation_author=D Leister; citation_journal_title=Gene; citation_volume=329; citation_number=3; citation_pages=11-16; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Analysis of curated and predicted plastid subproteomes of Arabidopsis. Subcellular compartmentalization leads to distinctive proteome properties.; citation_author=Q Sun; citation_author=O Emanuelsson; citation_author=KJ van Wijk; citation_journal_title=Plant Physiol; citation_volume=135(2); citation_number=4; citation_pages=723-734; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Plastid proteomics.; citation_author=KJ van Wijk; citation_journal_title=Plant Physiol Biochem; citation_volume=42(12); citation_number=5; citation_pages=963-77; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Chloroplast proteomics: potentials and challenges.; citation_author=S Baginsky; citation_author=W Gruissem; citation_journal_title=J Exp Bot; citation_volume=55(400); citation_number=6; citation_pages=1213-1220; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Recent surprises in protein targeting to mitochondria and plastids.; citation_author=AH Millar; citation_author=J Whelan; citation_author=I Small; citation_journal_title=Curr Opin Plant Biol; citation_volume=9(6); citation_number=7; citation_pages=610-615; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence for a protein transported through the secretory pathway en route to the higher plant chloroplast.; citation_author=A Villarejo; citation_author=S Buren; citation_author=S Larsson; citation_author=A Dejardin; citation_author=M Monne; citation_author=C Rudhe; citation_author=J Karlsson; citation_author=S Jansson; citation_author=P Lerouge; citation_author=N Rolland; citation_author=G von Heijne; citation_author=M Grebe; citation_author=L Bako; citation_author=G Samuelsson; citation_journal_title=Nat Cell Biol; citation_volume=(12); citation_number=8; citation_pages=1224-1231; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Plant Proteomics and Photosynthesis (Chapter 8), in Photosynthesis: A comprehensive Treatise Biochemistry, Biophysics and Molecular Biology, J.A.T. Eaton-Rye, B., Editor Kluwer.; citation_author=KJ van Wijk; citation_number=9; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis thaliana proteomics: from proteome to genome.; citation_author=S Baginsky; citation_author=W Gruissem; citation_journal_title=J Exp Bot; citation_volume=57(7); citation_number=10; citation_pages=1485-1491; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A versatile method for deciphering plant membrane proteomes.; citation_author=N Rolland; citation_author=M Ferro; citation_author=G Ephritikhine; citation_author=A Marmagne; citation_author=C Ramus; citation_author=S Brugiere; citation_author=D Salvi; citation_author=D Seigneurin-Berny; citation_author=J Bourguignon; citation_author=H Barbier-Brygoo; citation_author=J Joyard; citation_author=J Garin; citation_journal_title=J Exp Bot; citation_volume=57(7); citation_number=11; citation_pages=1579-1589; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Combining experimental and predicted datasets for determination of the subcellular location of proteins in Arabidopsis.; citation_author=JL Heazlewood; citation_author=J Tonti-Filippini; citation_author=RE Verboom; citation_author=AH Millar; citation_journal_title=Plant Physiol; citation_volume=139(2); citation_number=12; citation_pages=598-609; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The role of mass spectrometry in plant systems biology.; citation_author=M Glinski; citation_author=W Weckwerth; citation_journal_title=Mass Spectrom Rev; citation_volume=25(2); citation_number=13; citation_pages=173-214; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Oligomeric Stromal Proteome of Arabidopsis thaliana Chloroplasts.; citation_author=JB Peltier; citation_author=Y Cai; citation_author=Q Sun; citation_author=V Zabrouskov; citation_author=L Giacomelli; citation_author=A Rudella; citation_author=AJ Ytterberg; citation_author=H Rutschow; citation_author=KJ van Wijk; citation_journal_title=Mol Cell Proteomics; citation_volume=5(1); citation_number=14; citation_pages=114-133; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A model for random sampling and estimation of relative protein abundance in shotgun proteomics.; citation_author=H Liu; citation_author=RG Sadygov; citation_author=JR Yates; citation_journal_title=Anal Chem; citation_volume=76(14); citation_number=15; citation_pages=4193-4201; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Correlation of relative abundance ratios derived from peptide ion chromatograms and spectrum counting for quantitative proteomic analysis using stable isotope labeling.; citation_author=B Zybailov; citation_author=MK Coleman; citation_author=L Florens; citation_author=MP Washburn; citation_journal_title=Anal Chem; citation_volume=77(19); citation_number=16; citation_pages=6218-6224; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comparison of label-free methods for quantifying human proteins by shotgun proteomics.; citation_author=WM Old; citation_author=K Meyer-Arendt; citation_author=L Aveline-Wolf; citation_author=KG Pierce; citation_author=A Mendoza; citation_author=JR Sevinsky; citation_author=KA Resing; citation_author=NG Ahn; citation_journal_title=Mol Cell Proteomics; citation_volume=4(10); citation_number=17; citation_pages=1487-1502; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Absolute protein expression profiling estimates the relative contributions of transcriptional and translational regulation.; citation_author=P Lu; citation_author=C Vogel; citation_author=R Wang; citation_author=X Yao; citation_author=EM Marcotte; citation_journal_title=Nat Biotechnol; citation_volume=25(1); citation_number=18; citation_pages=117-124; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Statistical and computational methods for comparative proteomic profiling using liquid chromatography-tandem mass spectrometry.; citation_author=J Listgarten; citation_author=A Emili; citation_journal_title=Mol Cell Proteomics; citation_volume=4(4); citation_number=19; citation_pages=419-434; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis and validation of proteomic data generated by tandem mass spectrometry.; citation_author=AI Nesvizhskii; citation_author=O Vitek; citation_author=R Aebersold; citation_journal_title=Nat Methods; citation_volume=4(10); citation_number=20; citation_pages=787-797; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Quantitative mass spectrometry in proteomics: a critical review.; citation_author=M Bantscheff; citation_author=M Schirle; citation_author=G Sweetman; citation_author=J Rick; citation_author=B Kuster; citation_journal_title=Anal Bioanal Chem; citation_volume=389(4); citation_number=21; citation_pages=1017-1031; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cytoplasmic N-terminal protein acetylation is required for efficient photosynthesis in Arabidopsis.; citation_author=P Pesaresi; citation_author=NA Gardner; citation_author=S Masiero; citation_author=A Dietzmann; citation_author=L Eichacker; citation_author=R Wickner; citation_author=F Salamini; citation_author=D Leister; citation_journal_title=Plant Cell; citation_volume=15(8); citation_number=22; citation_pages=1817-1832; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of eukaryotic peptide deformylases reveals universality of N-terminal protein processing mechanisms.; citation_author=C Giglione; citation_author=A Serero; citation_author=M Pierre; citation_author=B Boisson; citation_author=T Meinnel; citation_journal_title=Embo J; citation_volume=19(21); citation_number=23; citation_pages=5916-5929; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Specificity of chloroplast-localized peptide deformylases as determined with peptide analogs of chloroplast-translated proteins.; citation_author=LM Dirk; citation_author=MA Williams; citation_author=RL Houtz; citation_journal_title=Arch Biochem Biophys; citation_volume=406(1); citation_number=24; citation_pages=135-141; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Eukaryotic peptide deformylases. Nuclear-encoded and chloroplast-targeted enzymes in Arabidopsis.; citation_author=LM Dirk; citation_author=MA Williams; citation_author=RL Houtz; citation_journal_title=Plant Physiol; citation_volume=127(1); citation_number=25; citation_pages=97-107; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Organellar peptide deformylases: universality of the N-terminal methionine cleavage mechanism.; citation_author=C Giglione; citation_author=T Meinnel; citation_journal_title=Trends Plant Sci; citation_volume=6(12); citation_number=26; citation_pages=566-572; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Control of protein life-span by N-terminal methionine excision.; citation_author=C Giglione; citation_author=O Vallon; citation_author=T Meinnel; citation_journal_title=Embo J; citation_volume=22(1); citation_number=27; citation_pages=13-23; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Functional and developmental impact of cytosolic protein N-terminal methionine excision in Arabidopsis.; citation_author=S Ross; citation_author=C Giglione; citation_author=M Pierre; citation_author=C Espagne; citation_author=T Meinnel; citation_journal_title=Plant Physiol; citation_volume=137(2); citation_number=28; citation_pages=623-637; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Impact of the N-terminal amino acid on targeted protein degradation.; citation_author=T Meinnel; citation_author=A Serero; citation_author=C Giglione; citation_journal_title=Biol Chem; citation_volume=387(7); citation_number=29; citation_pages=839-851; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Recent advances in the study of Clp, FtsH and other proteases located in chloroplasts.; citation_author=Z Adam; citation_author=A Rudella; citation_author=KJ van Wijk; citation_journal_title=Curr Opin Plant Biol; citation_volume=9(3); citation_number=30; citation_pages=234-240; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Interpreting the protein language using proteomics.; citation_author=ON Jensen; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=7(6); citation_number=31; citation_pages=391-403; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mass Spectrometric Mapping of Linker Histone H1 Variants Reveals Multiple Acetylations, Methylations, and Phosphorylation as Well as Differences between Cell Culture and Tissue.; citation_author=JR Wisniewski; citation_author=A Zougman; citation_author=S Kruger; citation_author=M Mann; citation_journal_title=Mol Cell Proteomics; citation_volume=6(1); citation_number=32; citation_pages=72-87; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Orbitrap mass analyzer - overview and applications in proteomics.; citation_author=M Scigelova; citation_author=A Makarov; citation_journal_title=Proteomics; citation_volume=6; citation_number=33; citation_pages=16-21; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Parts per million mass accuracy on an Orbitrap mass spectrometer via lock mass injection into a C-trap.; citation_author=JV Olsen; citation_author=LM de Godoy; citation_author=G Li; citation_author=B Macek; citation_author=P Mortensen; citation_author=R Pesch; citation_author=A Makarov; citation_author=O Lange; citation_author=S Horning; citation_author=M Mann; citation_journal_title=Mol Cell Proteomics; citation_volume=4(12); citation_number=34; citation_pages=2010-2021; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Higher-energy C-trap dissociation for peptide modification analysis.; citation_author=JV Olsen; citation_author=B Macek; citation_author=O Lange; citation_author=A Makarov; citation_author=S Horning; citation_author=M Mann; citation_journal_title=Nat Methods; citation_volume=4(9); citation_number=35; citation_pages=709-712; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=In-depth analysis of the thylakoid membrane proteome of Arabidopsis thaliana chloroplasts: new proteins, new functions, and a plastid proteome database.; citation_author=G Friso; citation_author=L Giacomelli; citation_author=AJ Ytterberg; citation_author=JB Peltier; citation_author=A Rudella; citation_author=Q Sun; citation_author=KJ Wijk; citation_journal_title=Plant Cell; citation_volume=16(2); citation_number=36; citation_pages=478-499; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Patterns of beauty–omics meets plant development.; citation_author=L Hennig; citation_journal_title=Trends Plant Sci; citation_volume=12(7); citation_number=37; citation_pages=287-293; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Evaluation of multidimensional chromatography coupled with tandem mass spectrometry (LC/LC-MS/MS) for large-scale protein analysis: the yeast proteome.; citation_author=J Peng; citation_author=JE Elias; citation_author=CC Thoreen; citation_author=LJ Licklider; citation_author=SP Gygi; citation_journal_title=J Proteome Res; citation_volume=2(1); citation_number=38; citation_pages=43-50; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=R Sokal; citation_author=FJJ Rohlf; citation_title=Chapter 5 in Biometry; citation_number=39; citation_date=2005; citation_publisher=W.H. FREEMAN and COMPANY; " />
      <meta name="citation_reference" content="citation_title=Detecting differential and correlated protein expression in label-free shotgun proteomics.; citation_author=B Zhang; citation_author=NC VerBerkmoes; citation_author=MA Langston; citation_author=E Uberbacher; citation_author=RL Hettich; citation_author=NF Samatova; citation_journal_title=J Proteome Res; citation_volume=5(11); citation_number=40; citation_pages=2909-2918; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Statistical analysis of membrane proteome expression changes in Saccharomyces cerevisiae.; citation_author=B Zybailov; citation_author=AL Mosley; citation_author=ME Sardiu; citation_author=MK Coleman; citation_author=L Florens; citation_author=MP Washburn; citation_journal_title=J Proteome Res; citation_volume=5(9); citation_number=41; citation_pages=2339-2347; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Computational prediction of proteotypic peptides for quantitative proteomics.; citation_author=P Mallick; citation_author=M Schirle; citation_author=SS Chen; citation_author=MR Flory; citation_author=H Lee; citation_author=D Martin; citation_author=J Ranish; citation_author=B Raught; citation_author=R Schmitt; citation_author=T Werner; citation_author=B Kuster; citation_author=R Aebersold; citation_journal_title=Nat Biotechnol; citation_volume=25(1); citation_number=42; citation_pages=125-131; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The proteome of the mouse photoreceptor sensory cilium complex.; citation_author=Q Liu; citation_author=G Tan; citation_author=N Levenkova; citation_author=T Li; citation_author=EN Pugh; citation_author=JJ Rux; citation_author=DW Speicher; citation_author=EA Pierce; citation_journal_title=Mol Cell Proteomics; citation_volume=6(8); citation_number=43; citation_pages=1299-1317; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Statistics for engineering and the sciences’. 4th edition ed.; citation_author=WaST Mendenhall; citation_number=44; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=MAPMAN: a user-driven tool to display genomics data sets onto diagrams of metabolic pathways and other biological processes.; citation_author=O Thimm; citation_author=O Blasing; citation_author=Y Gibon; citation_author=A Nagel; citation_author=S Meyer; citation_author=P Kruger; citation_author=J Selbig; citation_author=LA Muller; citation_author=SY Rhee; citation_author=M Stitt; citation_journal_title=Plant J; citation_volume=37(6); citation_number=45; citation_pages=914-939; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Two novel targeting peptide degrading proteases, PrePs, in mitochondria and chloroplasts, so similar and still different.; citation_author=A Stahl; citation_author=S Nilsson; citation_author=P Lundberg; citation_author=S Bhushan; citation_author=H Biverstahl; citation_author=P Moberg; citation_author=M Morisset; citation_author=A Vener; citation_author=L Maler; citation_author=U Langel; citation_author=E Glaser; citation_journal_title=J Mol Biol; citation_volume=349(4); citation_number=46; citation_pages=847-60; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Support for a potential role of E. coli oligopeptidase A in protein degradation.; citation_author=R Jain; citation_author=MK Chan; citation_journal_title=Biochem Biophys Res Commun; citation_volume=359(3); citation_number=47; citation_pages=486-490; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Tripeptidyl peptidase II. An oligomeric protease complex from Arabidopsis.; citation_author=AJ Book; citation_author=P Yang; citation_author=M Scalf; citation_author=LM Smith; citation_author=RD Vierstra; citation_journal_title=Plant Physiol; citation_volume=138(2); citation_number=48; citation_pages=1046-1057; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Clp Protease Complexes from Photosynthetic and Non-photosynthetic Plastids and Mitochondria of Plants, Their Predicted Three-dimensional Structures, and Functional Implications.; citation_author=JB Peltier; citation_author=DR Ripoll; citation_author=G Friso; citation_author=A Rudella; citation_author=Y Cai; citation_author=J Ytterberg; citation_author=L Giacomelli; citation_author=J Pillardy; citation_author=KJ Van Wijk; citation_journal_title=J Biol Chem; citation_volume=279(6); citation_number=49; citation_pages=4768-4781; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structural and functional insights into the chloroplast ATP-dependent Clp protease in Arabidopsis.; citation_author=LL Sjogren; citation_author=TM Stanne; citation_author=B Zheng; citation_author=S Sutinen; citation_author=AK Clarke; citation_journal_title=Plant Cell; citation_volume=18(10); citation_number=50; citation_pages=2635-2649; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Protein profiling of plastoglobules in chloroplasts and chromoplasts; a surprising site for differential accumulation of metabolic enzymes.; citation_author=AJ Ytterberg; citation_author=JB Peltier; citation_author=KJ van Wijk; citation_journal_title=Plant Physiol; citation_volume=140(3); citation_number=51; citation_pages=984-997; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=New functions of the thylakoid membrane proteome of Arabidopsis thaliana revealed by a simple, fast, and versatile fractionation strategy.; citation_author=JB Peltier; citation_author=AJ Ytterberg; citation_author=Q Sun; citation_author=KJ van Wijk; citation_journal_title=J Biol Chem; citation_volume=279(47); citation_number=52; citation_pages=49367-49383; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=SUBA: the Arabidopsis Subcellular Database.; citation_author=JL Heazlewood; citation_author=RE Verboom; citation_author=J Tonti-Filippini; citation_author=I Small; citation_author=AH Millar; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=53; citation_pages=D213-218; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Tandem mass spectrometry identifies sites of three post-translational modifications of spinach light-harvesting chlorophyll protein II. Proteolytic cleavage, acetylation, and phosphorylation.; citation_author=H Michel; citation_author=PR Griffin; citation_author=J Shabanowitz; citation_author=DF Hunt; citation_author=J Bennett; citation_journal_title=J Biol Chem; citation_volume=266(26); citation_number=54; citation_pages=17584-17591; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Tandem mass spectrometry reveals that three photosystem II proteins of spinach chloroplasts contain N-acetyl-O-phosphothreonine at their NH2 termini.; citation_author=H Michel; citation_author=DF Hunt; citation_author=J Shabanowitz; citation_author=J Bennett; citation_journal_title=J Biol Chem; citation_volume=263(3); citation_number=55; citation_pages=1123-1130; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Post-translational modifications, but not transcriptional regulation, of major chloroplast RNA-binding proteins are related to Arabidopsis seedling development.; citation_author=B WangC; citation_author=HX Wang; citation_author=JX Feng; citation_author=DZ Meng; citation_author=LJ Qu; citation_author=YX Zhu; citation_journal_title=Proteomics; citation_volume=6(8); citation_number=56; citation_pages=2555-2563; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Proteome dynamics during plastid differentiation in rice.; citation_author=T Kleffmann; citation_author=A von Zychlinski; citation_author=D Russenberger; citation_author=M Hirsch-Hoffmann; citation_author=P Gehrig; citation_author=W Gruissem; citation_author=S Baginsky; citation_journal_title=Plant Physiol; citation_volume=143(2); citation_number=57; citation_pages=912-923; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=ChloroP a neural network-based method for predicting chloroplast transit peptides and their cleavage sites.; citation_author=O Emanuelsson; citation_author=H Nielsen; citation_author=G von Heijne; citation_journal_title=Protein Sci; citation_volume=8(5); citation_number=58; citation_pages=978-984; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=N-terminal acetyltransferases and sequence requirements for N-terminal acetylation of eukaryotic proteins.; citation_author=B Polevoda; citation_author=F Sherman; citation_journal_title=J Mol Biol; citation_volume=325(4); citation_number=59; citation_pages=595-622; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Central Functions of the Lumenal and Peripheral Thylakoid Proteome of Arabidopsis Determined by Experimentation and Genome-Wide Prediction.; citation_author=JB Peltier; citation_author=O Emanuelsson; citation_author=DE Kalume; citation_author=J Ytterberg; citation_author=G Friso; citation_author=A Rudella; citation_author=DA Liberles; citation_author=L Soderberg; citation_author=P Roepstorff; citation_author=G von Heijne; citation_author=KJ van Wijk; citation_journal_title=Plant Cell; citation_volume=14(1); citation_number=60; citation_pages=211-236; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Proteome map of the chloroplast lumen of Arabidopsis thaliana.; citation_author=M Schubert; citation_author=UA Petersson; citation_author=BJ Haas; citation_author=C Funk; citation_author=WP Schröder; citation_author=T Kieselbach; citation_journal_title=J Biol Chem; citation_volume=277(10); citation_number=61; citation_pages=8354-8365; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Psb27, a cyanobacterial lipoprotein, is involved in the repair cycle of photosystem II.; citation_author=MM Nowaczyk; citation_author=R Hebeler; citation_author=E Schlodder; citation_author=HE Meyer; citation_author=B Warscheid; citation_author=M Rogner; citation_journal_title=Plant Cell; citation_volume=18(11); citation_number=62; citation_pages=3121-3131; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence that D1 processing is required for manganese binding and extrinsic protein assembly into photosystem II.; citation_author=JL Roose; citation_author=HB Pakrasi; citation_journal_title=J Biol Chem; citation_volume=279(44); citation_number=63; citation_pages=45417-45422; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A Psb27 homologue in Arabidopsis thaliana is required for efficient repair of photodamaged photosystem II.; citation_author=H Chen; citation_author=D Zhang; citation_author=J Guo; citation_author=H Wu; citation_author=M Jin; citation_author=Q Lu; citation_author=C Lu; citation_author=L Zhang; citation_journal_title=Plant Mol Biol; citation_volume=61(4–5); citation_number=64; citation_pages=567-575; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mass spectrometry for high throughput quantitative proteomics in plant research: lessons from thylakoid membranes.; citation_author=JP Whitelegge; citation_journal_title=Plant Physiol Biochem; citation_volume=42(12); citation_number=65; citation_pages=919-927; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Domain structure of mitochondrial and chloroplast targeting peptides.; citation_author=G von Heijne; citation_author=J Steppuhn; citation_author=RG Herrmann; citation_journal_title=Eur J Biochem; citation_volume=180(3); citation_number=66; citation_pages=535-45; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Comparative evaluation of mass spectrometry platforms used in large-scale proteomics investigations.; citation_author=JE Elias; citation_author=W Haas; citation_author=BK Faherty; citation_author=SP Gygi; citation_journal_title=Nat Methods; citation_volume=2(9); citation_number=67; citation_pages=667-675; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Experimental analysis of the Arabidopsis mitochondrial proteome highlights signaling and regulatory components, provides assessment of targeting prediction programs, and indicates plant-specific mitochondrial proteins.; citation_author=JL Heazlewood; citation_author=JS Tonti-Filippini; citation_author=AM Gout; citation_author=DA Day; citation_author=J Whelan; citation_author=AH Millar; citation_journal_title=Plant Cell; citation_volume=16(1); citation_number=68; citation_pages=241-256; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=State transitions and light adaptation require chloroplast thylakoid protein kinase STN7.; citation_author=S Bellafiore; citation_author=F Barneche; citation_author=G Peltier; citation_author=JD Rochaix; citation_journal_title=Nature; citation_volume=433(7028); citation_number=69; citation_pages=892-5; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Photosystem II core phosphorylation and photosynthetic acclimation require two different protein kinases.; citation_author=V Bonardi; citation_author=P Pesaresi; citation_author=T Becker; citation_author=E Schleiff; citation_author=R Wagner; citation_author=T Pfannschmidt; citation_author=P Jahns; citation_author=D Leister; citation_journal_title=Nature; citation_volume=437(7062); citation_number=70; citation_pages=1179-1182; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=pTAC2, -6, and -12 are components of the transcriptionally active plastid chromosome that are required for plastid gene expression.; citation_author=J Pfalz; citation_author=K Liere; citation_author=A Kandlbinder; citation_author=KJ Dietz; citation_author=R Oelmuller; citation_journal_title=Plant Cell; citation_volume=18(1); citation_number=71; citation_pages=176-197; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High sensitivity detection of plasma proteins by multiple reaction monitoring of N-glycosites.; citation_author=J Stahl-Zeng; citation_author=V Lange; citation_author=R Ossola; citation_author=K Eckhardt; citation_author=W Krek; citation_author=R Aebersold; citation_author=B Domon; citation_journal_title=Mol Cell Proteomics; citation_volume=6(10); citation_number=72; citation_pages=1809-1817; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Flux an important, but neglected, component of functional genomics.; citation_author=AR Fernie; citation_author=P Geigenberger; citation_author=M Stitt; citation_journal_title=Curr Opin Plant Biol; citation_volume=8(2); citation_number=73; citation_pages=174-182; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The N-end rule pathway for regulated proteolysis: prokaryotic and eukaryotic strategies.; citation_author=A Mogk; citation_author=R Schmidt; citation_author=B Bukau; citation_journal_title=Trends Cell Biol; citation_volume=17(4); citation_number=74; citation_pages=165-172; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=ATP-dependent proteases of bacteria: recognition logic and operating principles.; citation_author=TA Baker; citation_author=RT Sauer; citation_journal_title=Trends Biochem Sci; citation_volume=31(12); citation_number=75; citation_pages=647-653; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=ProMEX: a mass spectral reference database for proteins and protein phosphorylation sites.; citation_author=J Hummel; citation_author=M Niemann; citation_author=S Wienkoop; citation_author=W Schulze; citation_author=D Steinhauser; citation_author=J Selbig; citation_author=D Walther; citation_author=W Weckwerth; citation_journal_title=BMC Bioinformatics; citation_volume=8; citation_number=76; citation_pages=216; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=AraPerox. A database of putative Arabidopsis proteins from plant peroxisomes.; citation_author=S Reumann; citation_author=C Ma; citation_author=S Lemke; citation_author=L Babujee; citation_journal_title=Plant Physiol; citation_volume=136(1); citation_number=77; citation_pages=2587-2608; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis mutants lacking the 43- and 54-kilodalton subunits of the chloroplast signal recognition particle have distinct phenotypes.; citation_author=P Amin; citation_author=DA Sy; citation_author=ML Pilgrim; citation_author=DH Parry; citation_author=L Nussaume; citation_author=NE Hoffman; citation_journal_title=Plant Physiol; citation_volume=121(1); citation_number=78; citation_pages=61-70; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Downregulation of ClpR2 Leads to Reduced Accumulation of the ClpPRS Protease Complex and Defects in Chloroplast Biogenesis in Arabidopsis.; citation_author=A Rudella; citation_author=G Friso; citation_author=JM Alonso; citation_author=JR Ecker; citation_author=KJ van Wijk; citation_journal_title=Plant Cell; citation_volume=18(7); citation_number=79; citation_pages=1704-1721; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Isolation of chloroplast proteins from Arabidopsis thaliana for proteome analysis.; citation_author=KJ van Wijk; citation_author=JB Peltier; citation_author=L Giacomelli; citation_journal_title=Methods Mol Biol; citation_volume=355; citation_number=80; citation_pages=43-48; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mass spectrometric sequencing of proteins silver-stained polyacrylamide gels.; citation_author=A Shevchenko; citation_author=M Wilm; citation_author=O Vorm; citation_author=M Mann; citation_journal_title=Anal Chem; citation_volume=68(5); citation_number=81; citation_pages=850-858; citation_date=1996; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001994" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Sorting Signals, N-Terminal Modifications and Abundance of the Chloroplast Proteome"/>
    <meta name="twitter:description" content="Characterization of the chloroplast proteome is needed to understand the essential contribution of the chloroplast to plant growth and development. Here we present a large scale analysis by nanoLC-Q-TOF and nanoLC-LTQ-Orbitrap mass spectrometry (MS) of ten independent chloroplast preparations from Arabidopsis thaliana which unambiguously identified 1325 proteins. Novel proteins include various kinases and putative nucleotide binding proteins. Based on repeated and independent MS based protein identifications requiring multiple matched peptide sequences, as well as literature, 916 nuclear-encoded proteins were assigned with high confidence to the plastid, of which 86% had a predicted chloroplast transit peptide (cTP). The protein abundance of soluble stromal proteins was calculated from normalized spectral counts from LTQ-Obitrap analysis and was found to cover four orders of magnitude. Comparison to gel-based quantification demonstrates that &lsquo;spectral counting&rsquo; can provide large scale protein quantification for Arabidopsis. This quantitative information was used to determine possible biases for protein targeting prediction by TargetP and also to understand the significance of protein contaminants. The abundance data for 550 stromal proteins was used to understand abundance of metabolic pathways and chloroplast processes. We highlight the abundance of 48 stromal proteins involved in post-translational proteome homeostasis (including aminopeptidases, proteases, deformylases, chaperones, protein sorting components) and discuss the biological implications. N-terminal modifications were identified for a subset of nuclear- and chloroplast-encoded proteins and a novel N-terminal acetylation motif was discovered. Analysis of cTPs and their cleavage sites of Arabidopsis chloroplast proteins, as well as their predicted rice homologues, identified new species-dependent features, which will facilitate improved subcellular localization prediction. No evidence was found for suggested targeting via the secretory system. This study provides the most comprehensive chloroplast proteome analysis to date and an expanded Plant Proteome Database (PPDB) in which all MS data are projected on identified gene models."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001994.g007"/>

  <meta property="og:title" content="Sorting Signals, N-Terminal Modifications and Abundance of the Chloroplast Proteome" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001994" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9835'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8036'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4582&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001994">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001994" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001994&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Sorting Signals, N-Terminal Modifications and Abundance of the Chloroplast Proteome
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Boris Zybailov, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Plant Biology, Cornell University, Ithaca, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Heidi Rutschow, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Plant Biology, Cornell University, Ithaca, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Giulia Friso, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Plant Biology, Cornell University, Ithaca, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrea Rudella, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Plant Biology, Cornell University, Ithaca, New York, United States of America
                </p>

                <p>
                    Current address: Waters Co. EHQ Waters S.A.S., Guyancourt, France
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Olof Emanuelsson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Stockholm Bioinformatics Center, AlbaNova, Stockholm University, Stockholm, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Qi Sun, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computation Biology Service Unit, Cornell Theory Center, Cornell University, Ithaca, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Klaas J. van Wijk
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kv35@cornell.edu">kv35@cornell.edu</a></p>

                <p>Affiliation:
                  Department of Plant Biology, Cornell University, Ithaca, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 23, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001994</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001994" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001994" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001994" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001994" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001994-g001" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001994-t001" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001994-g002" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001994-g003" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001994-g004" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001994-g005" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001994-g006" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001994-t002" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001994-t003" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001994-g007" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001994">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001994" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Characterization of the chloroplast proteome is needed to understand the essential contribution of the chloroplast to plant growth and development. Here we present a large scale analysis by nanoLC-Q-TOF and nanoLC-LTQ-Orbitrap mass spectrometry (MS) of ten independent chloroplast preparations from <em>Arabidopsis thaliana</em> which unambiguously identified 1325 proteins. Novel proteins include various kinases and putative nucleotide binding proteins. Based on repeated and independent MS based protein identifications requiring multiple matched peptide sequences, as well as literature, 916 nuclear-encoded proteins were assigned with high confidence to the plastid, of which 86% had a predicted chloroplast transit peptide (cTP). The protein abundance of soluble stromal proteins was calculated from normalized spectral counts from LTQ-Obitrap analysis and was found to cover four orders of magnitude. Comparison to gel-based quantification demonstrates that ‘spectral counting’ can provide large scale protein quantification for <em>Arabidopsis</em>. This quantitative information was used to determine possible biases for protein targeting prediction by TargetP and also to understand the significance of protein contaminants. The abundance data for 550 stromal proteins was used to understand abundance of metabolic pathways and chloroplast processes. We highlight the abundance of 48 stromal proteins involved in post-translational proteome homeostasis (including aminopeptidases, proteases, deformylases, chaperones, protein sorting components) and discuss the biological implications. N-terminal modifications were identified for a subset of nuclear- and chloroplast-encoded proteins and a novel N-terminal acetylation motif was discovered. Analysis of cTPs and their cleavage sites of <em>Arabidopsis</em> chloroplast proteins, as well as their predicted rice homologues, identified new species-dependent features, which will facilitate improved subcellular localization prediction. No evidence was found for suggested targeting via the secretory system. This study provides the most comprehensive chloroplast proteome analysis to date and an expanded Plant Proteome Database (PPDB) in which all MS data are projected on identified gene models.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zybailov B, Rutschow H, Friso G, Rudella A, Emanuelsson O, et al.  (2008) Sorting Signals, N-Terminal Modifications and Abundance of the Chloroplast Proteome. PLoS ONE 3(4):
          e1994.
            doi:10.1371/journal.pone.0001994</p><p><strong>Editor: </strong>Karl-Wilhelm Koch, University of Oldenburg, Germany</p><p><strong>Received:</strong> January 4, 2008; <strong>Accepted:</strong> March 6, 2008; <strong>Published:</strong> April 23, 2008</p><p><strong>Copyright:</strong> © 2008 Zybailov et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This project was supported by a grant from the National Science Foundation (MCB # 0343444), the U.S. Department of Energy (DE-FG02-04ER15560) and New York Science, Technology and Academic Research (NYSTAR) to K.J.v.W. QS was supported by resources of the Computational Biology Service Unit from Cornell University which is partially funded by the Microsoft Corporation. OE was supported by a grant from the Knut and Alice Wallenberg Foundation.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Chloroplasts are essential organelles of prokaryotic origin and carry out a wide range of metabolic functions. The chloroplast genome only encodes for about 100 proteins, whereas the vast majority of the chloroplast proteome is encoded by the nuclear genome. These proteins are generally synthesized as precursor proteins with cleavable N-terminal chloroplast transit peptides (cTPs) <a href="#pone.0001994-vonHeijne1">[1]</a>. Several subcellular localization programs, such as TargetP <a href="#pone.0001994-Emanuelsson1">[2]</a> are available that predict these cTPs, with the number of predicted chloroplast (plastid) proteins ranging from about 1500 to 4500 proteins <a href="#pone.0001994-Richly1">[3]</a>, <a href="#pone.0001994-Sun1">[4]</a>. However, several known plastid proteins appear to have no obvious cTP, and chloroplast outer envelope proteins never have a cleavable cTP (for discussion see <a href="#pone.0001994-vanWijk1">[5]</a>–<a href="#pone.0001994-Millar1">[7]</a>. It was recently suggested that an <em>Arabidopsis thaliana</em> (from here on referred to as <em>Arabidopsis</em>) chloroplast protein (a carbonic anhydrase) takes an alternative route through the secretory pathway, and becomes N-glycosylated before entering the chloroplast <a href="#pone.0001994-Villarejo1">[8]</a>. It is possible that more chloroplast proteins follow this route. Large scale experimental plastid proteomics studies are needed to evaluate unusual targeting pathways and to provide new training sets to improve subcellular localization prediction.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Driven by developments in mass spectrometry (MS), the <em>Arabidopsis</em> chloroplast proteome has been analyzed by MS in combination with various protein fractionation techniques to assign proteins to chloroplast compartments (reviewed in <a href="#pone.0001994-vanWijk2">[9]</a>–<a href="#pone.0001994-Rolland1">[11]</a>). Collectively, these studies identified 1090 proteins (counting 1 gene model per protein), with an overall cTP prediction rate of 60% by TargetP (data not shown). However, from manual evaluation we estimate that 300–350 proteins likely represent false positive identifications and/or non-chloroplast contaminations. This shows that uncurated experimental proteomics data from isolated subcellular compartments and localization predictors do not provide sufficient quality for localization. However, the combination of multiple independent proteomics experiments, ideally from all compartments of a cell, as well as cross-correlation to detailed functional and localization (eg. with GFP fusion proteins) studies may allow high quality subcellular localization and functional annotation <a href="#pone.0001994-Heazlewood1">[12]</a>. Currently, this curation process cannot be fully automated and requires manual supervision. Thus more experimental work and curation is needed to obtain a more in-depth and accurate overview of the chloroplast proteome in <em>Arabidopsis.</em></p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Protein accumulation levels within a cell, or subcellular compartment such as the chloroplast, span five to ten orders of magnitude. To understand chloroplast function and homeostasis and to accommodate systems biology approaches to model genetic and metabolic networks <a href="#pone.0001994-Glinski1">[13]</a>, it is important to determine protein accumulation levels. A recent analysis of the <em>Arabidopsis</em> stromal proteome used gel based quantification to rank the abundance of 240 stromal proteins spanning several orders of magnitude <a href="#pone.0001994-Peltier1">[14]</a>. The challenge is now to obtain accurate quantification for a larger percentage of the chloroplast proteome. Recently, large scale MS-based studies for yeast, humans, <em>E. coli</em> and other sequenced organisms have shown that the number of MS/MS spectra matched to a protein (spectral counts - SPC) positively correlates with the protein abundance <a href="#pone.0001994-Liu1">[15]</a>–<a href="#pone.0001994-Lu1">[18]</a>. Upon control of several experimental conditions, careful and stringent spectral assignments, and sophisticated normalization procedures, it appears that MS-based quantification can provide an attractive and sensitive tool to obtain large scale measurements of relative protein concentrations. For further review and discussions were refer to <a href="#pone.0001994-Listgarten1">[19]</a>–<a href="#pone.0001994-Bantscheff1">[21]</a>. These new developments provide an excellent opportunity for quantification of the chloroplast proteome as will be demonstrated in the current study.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The half-life and function of proteins is often influenced by post-translational modifications (PTMs). N-terminal modifications of chloroplast proteins have shown to be important for chloroplast viability. For instance, N-terminal acetylation in the cytosol of nuclear-encoded chloroplast proteins is required for chloroplast function <a href="#pone.0001994-Pesaresi1">[22]</a>. Furthermore, both chloroplast localized deformylase <a href="#pone.0001994-Giglione1">[23]</a>–<a href="#pone.0001994-Giglione2">[26]</a> and methionine endopeptidase are essential for <em>Arabidopsis</em> seedling viability <a href="#pone.0001994-Giglione3">[27]</a>, <a href="#pone.0001994-Ross1">[28]</a>. It is quite likely that these N-terminal modifications improve protein stability <a href="#pone.0001994-Meinnel1">[29]</a>, for example to avoid degradation by the abundant chloroplast Clp protease system <a href="#pone.0001994-Adam1">[30]</a>. However, no systematic experimental analysis of N-termini of <em>Arabidopsis</em> chloroplast proteins has been carried out so far. PTMs, such as N-terminal acetylation, typically lead to a well-defined change in molecular mass that can often be detected by high quality MS. The rapid improvements in MS instrumentation, exemplified by the linear ion trap triple quadropole (LTQ) Fourier transform ion cyclotron resonance and LTQ-Orbitrap instruments, now facilitate a high throughput PTM analysis <a href="#pone.0001994-Jensen1">[31]</a>–<a href="#pone.0001994-Olsen2">[35]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The current study determines chloroplast stromal protein abundance and N-terminal modifications, re-evaluates chloroplast transit peptides and cleavage sites, and provides a comprehensive catalogue and annotation of the chloroplast proteome, encompassing existing literature. The plastid proteomics database, PPDB (<a href="http://ppdb.tc.cornell.edu/">http://ppdb.tc.cornell.edu/</a>), first described in <a href="#pone.0001994-Friso1">[36]</a>, is focused on the (cell-type specific) chloroplast proteomes from maize and <em>Arabidopsis</em> and their functional annotation. We recently renamed the Plastid PDB into Plant PDB to better reflect the content. The dataset obtained in the current study is integrated in the PPDB, is expected to serve the plant community in small and large scale analyses where protein subcellular location, protein modification, function and abundance are important. Moreover, based on our experimental and theoretical analysis of the N-terminal portions and cTP cleavage sites, it is expected that the chloroplast data set presented here will facilitate improvement of subcellular protein localization predictors. Finally, the protein coverage and abundance of key chloroplast pathways and processes is discussed. This study demonstrates that ‘spectral counting’ can provide large scale protein quantification for <em>Arabidopsis</em>, which is important in the context of plant systems biology <a href="#pone.0001994-Glinski1">[13]</a>, <a href="#pone.0001994-Hennig1">[37]</a>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Experimental identification of the chloroplast proteome by LTQ-Orbitrap</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To identify stromal and thylakoid proteins in the chloroplast, three independent <em>Arabidopsis</em> chloroplast preparations from mature rosette plants were used. Each preparation was separated into a soluble stromal and membrane fraction and then resolved by 1D SDS-PAGE, followed by in-gel trypsin digestion and identification by online nano-liquid chromatography tandem mass spectrometry (nanoLC-MS/MS) with an LTQ-Orbitrap, using data dependent acquisition (DDA) and dynamic exclusion. Biological replicates were analyzed two or three times to determine the technical variation of MS acquisition, as discussed further below.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Filtering criteria for MS-based protein identification were chosen to give an overall peptide false discovery rate (FDR) of 1%, as estimated by concatenated target/decoy database search <a href="#pone.0001994-Peng1">[38]</a>. Furthermore, only one gene model per protein was selected, which was either the one with the highest protein identification Mowse score, or in case of equal scores, the one with the lowest model number (indicated by the number after the digit). If the same set of peptides could be used to identify several different proteins, these homologous proteins were reported as an ambiguous group with all possible proteins listed, yet counted as one protein identification. For example, AT5G38410.1 and AT5G38420.1, corresponding to Rubisco small subunits 3B and 2B, were counted as one identification because they were described by the same set of peptides. Proteins were not reported if they could only be identified based on subsets of peptides matching to another identified protein. A more detailed description of the filtering procedures is provided in the <a href="#s4">Materials and Methods</a> section.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Overall, 1258 proteins were identified unambiguously. In addition, 22 pairs or small groups of homologous proteins were identified (<a href="#pone.0001994.s002">Table S1</a>). <a href="#pone-0001994-t001">Table 1</a> and <a href="#pone-0001994-g001">Figures 1A–E</a> summarize the identification results and show overlap between the experiments. The number of identified proteins from individual mass-spectrometry analyses were as follows: i) The stromal fraction from chloroplast preparation 1 (P1) was analyzed three times, yielding 656, 653, and 522 proteins each, ii) stromal fractions from P2 and P3 were analyzed once each, yielding 674 and 728 proteins, respectively; iii) fractions enriched in thylakoid membrane proteins were analyzed twice for P2 (534 and 498 identifications), and once for P3 (571 identifications). Additionally, a low density membrane fraction obtained by high speed centrifugation of stromal extract was analyzed for P3, yielding 667 identifications (<a href="#pone-0001994-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001994-g001"><div class="img"><a name="pone-0001994-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Identification of Chloroplast Proteins by nanoLC-LTQ-Orbitrap MS/MS.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Venn diagrams show overlap between proteins identified in different preparations. Letter S denote soluble fractions, letter T denotes membrane fractions, letters LM denote low density membrane fraction, (A) Proteins identified in three technical stromal replicates of Chloroplast preparation 1, (B) Proteins identified in three biological replicates of stromal samples – chloroplast preparation 1 (technical replicate 2), chloroplast preparation 2, and chloroplast preparation 3, (C) Proteins identified in stromal and membrane (two technical replicates thereof) fractions of Chloroplast preparation 2, (D) Proteins identified in the three different fractions of Chloroplast Preparation 3, (E) Overlap between three sample types from all replicates combined. Number in parentheses is the percentage of proteins localized in chloroplast according to TargetP.</p>
<span>doi:10.1371/journal.pone.0001994.g001</span></div><div class="figure" id="pone-0001994-t001"><div class="img"><a name="pone-0001994-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001994.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001994.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Identification of Chloroplast Proteins by nanoLC-LTQ-Orbitrap.</span></strong></p><span>doi:10.1371/journal.pone.0001994.t001</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>To further assess the reproducibility between technical and biological replicates, we then carried out a G-test of independence of unique number of spectral counts (SPC) for each protein <a href="#pone.0001994-Sokal1">[39]</a> <a href="#pone.0001994-Old1">[17]</a>, <a href="#pone.0001994-Zhang1">[40]</a>. This shows that the technical variation resulting from on-line chromatography and MS analysis is very small, while the biological variation was mostly due to undersampling of low-abundant chloroplast proteins and infrequent observation of non-chloroplast contaminants (see <a href="#pone.0001994.s001">Text S1</a> for details).</p>


<h4>Abundance of stromal proteins</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>It has been shown in DDA LC-MS-based analyses of digested protein mixtures that the number of MS/MS of spectral counts (SPC) correlates with protein abundance <a href="#pone.0001994-Liu1">[15]</a>–<a href="#pone.0001994-Lu1">[18]</a>. The matched spectral counts for each protein need to be normalized for protein properties to obtain an accurate abundance measurement. The most promising methods of normalization have been protein length <a href="#pone.0001994-Zybailov2">[41]</a> or number of theoretical and relevant tryptic peptides, possibly corrected for their propensity to be observed <a href="#pone.0001994-Lu1">[18]</a>, <a href="#pone.0001994-Mallick1">[42]</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To estimate abundance levels of the 946 proteins identified in the three stromal preparations, we calculated first the abundance factor (AF) from unique counts, and subsequently normalized AF for number of observable tryptic peptides yielding NAF. In cases where two or more homologous proteins were identified with shared and unique peptides, the number of spectra from shared peptides assigned to each protein was determined based on the ratios of spectra derived from the unique peptides that identified each protein, according to <a href="#pone.0001994-Liu2">[43]</a> (see <a href="#s4">Material and Methods</a> for details). Shared counts contributed only once to the overall normalization.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p><a href="#pone-0001994-g002">Figure 2A</a> shows the frequency distribution of the Log<sub>10</sub> values of the NAF corrected for number of tryptic peptides for the 946 proteins in the initial unfiltered dataset from stromal samples. Each bin on the x-axis corresponds to a 0.25 order of magnitude, with the total population spanning five orders of magnitude. The bins are grouped into seven abundance classes (I–VII, with bin I representing proteins of highest abundance). The percentage of chloroplast predicted nuclear-encoded proteins is indicated, with 80% predicted cTP for the complete set. The decrease in cTP% coincides with the decrease in abundance, which is expected since most of the non-chloroplast contaminants (they have no true cTP) should be present at the lower end of the abundance spectrum.</p>
<div class="figure" id="pone-0001994-g002"><div class="img"><a name="pone-0001994-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Frequency distribution of relative concentrations of soluble chloroplast proteins.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Log10 abundance of stromal proteins were calculated from normalized SPC and corrected for predicted full tryptic peptides of the mature proteins within mass window of 700–3500 Da. Corrections were made for shared peptides as described in the <a href="#s4">Material and Method</a> section. Each bin on the x-axis corresponds to 0.25 order of magnitude, with the total population spanning five orders of magnitude. The bins are grouped into seven abundance classes (I–VII, with I representing proteins of highest abundance). The percentage of chloroplast predicted nuclear-encoded proteins by TargetP (% cTP) is indicated. (A) Frequency distribution for the 946 proteins in the initial (unfiltered) dataset from stromal samples, (B,C,D) Frequency distribution of stromal protein after application of successive filters, as follows i) after known non-stromal chloroplast proteins (<em>i.e.</em> thylakoid, lumen, envelope) were removed (B), when only including proteins observed in 2 or more independent preparations (C), after removal of non-chloroplast contaminants (D).</p>
<span>doi:10.1371/journal.pone.0001994.g002</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>With the objective to get a more accurate quantification of the actual stromal proteome, we applied a series of filters to the initial data set (<a href="#pone-0001994-g002">Figures 2B–D</a>). <a href="#pone-0001994-g002">Figure 2B</a> shows the frequency distribution of the protein abundance after known non-stromal chloroplast proteins (<em>i.e.</em> thylakoid, lumen and envelope) were removed. Furthermore, when we only consider proteins observed in 2 or more independent preparations, 193 proteins are removed mostly from abundance classes IV-V-VI-VII (<a href="#pone-0001994-g002">Figure 2C</a>). Cross-referencing these 590 remaining proteins to the existing literature suggested that 33 of these proteins were non-chloroplast contaminants (<a href="#pone.0001994.s003">Table S2</a>). These were subsequently removed to yield the final set of 557 proteins. The percentage of predicted cTP for these remaining proteins then increased to 87% (<a href="#pone-0001994-g002">Figure 2D</a> and <a href="#pone.0001994.s003">Table S2</a>). The validity of the filtering is supported by the observation that the abundance distribution becomes closer to normal (compare <a href="#pone-0001994-g002">Figure 2A with Figure 2D</a>), in agreement with the central limit theorem <a href="#pone.0001994-Mendenhall1">[44]</a>.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>In a previous study, we quantified accumulation levels of 214 chloroplast stromal proteins (88% cTP) from image analysis of stained 2D native gels (weighted for experimental protein mass), covering ~4 orders of abundance <a href="#pone.0001994-Peltier1">[14]</a>. To evaluate how the current MS based quantification compares to this gel-based quantification, we cross-correlated the abundance of stromal proteins accurately quantified in both studies (<a href="#pone-0001994-g003">Figure 3</a>). This showed a strong positive correlation, as indicated by a Spearman correlation coefficient of 0.56. It appears that the spectral counting technique underestimates the most abundant proteins, in particular for the Rubisco large and small subunits. However, the gel based method is likely less accurate for the lower abundance proteins, as the signal/noise ratio for spot intensity was low – see <a href="#pone.0001994-Peltier1">[14]</a>. When using protein length rather than observable tryptic peptides, the Spearman correlation was slightly lower (0.53) (not shown).</p>
<div class="figure" id="pone-0001994-g003"><div class="img"><a name="pone-0001994-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Cross-correlation between relative concentrations of stromal proteins quantified by spectral counting and by image analysis of gel separated proteins.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Stromal protein quantified in a previous study from image analysis of stained 2-dimensional native gels (weighted for experimental protein mass) <a href="#pone.0001994-Peltier1">[14]</a> were directly correlated to MS based quantified from the current LTQ data set. Log10 abundance of stromal proteins were calculated from normalized SPC and corrected for predicted full tryptic peptides of the mature proteins within mass window of 700–3500 Da. This showed strong positive correlation as indicated by a Spearman correlation coefficient of 0.56.</p>
<span>doi:10.1371/journal.pone.0001994.g003</span></div><h5>Coverage and abundance of chloroplast stromal functions and pathways.</h5><a id="article1.body1.sec2.sec2.sec1.p1" name="article1.body1.sec2.sec2.sec1.p1"></a><p>To obtain better insight in the role of the quantified stromal proteome, all proteins were (re)evaluated for function, using information from papers, functional protein domain predictions, and other resources (<em>e.g</em>.TAIR). We used the MapMan bin system <a href="#pone.0001994-Thimm1">[45]</a> for functional classification. As compared to previous chloroplast proteome studies, the current study significantly increased coverage of lower abundant pathways. Examples are nucleotide synthesis and degradation and nucleotide transfer, represented by 22 proteins out of the 39 cTP predicted plastid proteins assigned to this pathway, with an average abundance (log10(NAF)), of −3.4. Also, a set of 14 low abundant t-RNA synthetases were observed in the stroma with an average of −3.6, while soluble proteins involved in tetrapyrole biosynthesis have an average abundance of −3.3. The quantified stromal proteome also has a high number of proteins involved in protein translation, (un)folding, targeting, processing, aa modifications, and proteolysis. Here we highlight those stromal enzymes involved in the post-translational protein homeostasis network steps, including N-terminal processing and modifications (<a href="#pone-0001994-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0001994-g004"><div class="img"><a name="pone-0001994-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Quantification of the chloroplast protein homeostasis network including processing, (un)folding, maturation and proteolysis.</span></strong></p><a id="article1.body1.sec2.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec2.sec1.fig1.caption1.p1"></a><p>Relative concentrations of 48 stromal proteins involved in the post-translational protein homeostasis network are displayed with color coding. Abbreviations are as follows: SPP, general stromal processing peptidase; PDF1A,B, methionine deformylases 1A,B; MSRA4,B2, methionine sulfoxide reductases; AP, amino-peptidases; CPN10,20,60, chaperone 10,20 and 60 of the GroEL/ES system; cpHSP70, heat shock protein 70; GrpE, nucleotide exchange factor; HSP90, heat shock protein 90; ClpB3, chaperone B3; cpSRP – chloroplast signal recognition particle subunit 43 and 54, involved in protein targeting components cpSRP43, cpSRP54; cpSecA – ATP-dependent Sec targeting component; cpTIG, a homologue of <em>E. coli</em> trigger factor involved in protein folding at the ribosome; ROC4, protein isomerase with unknown function; ClpP/R/S,T,C,D- subunits of the complete Clp protease system, DegP2 – protease of the Deg family; AtPrep1 - a Zn-protease suggested to be involved in degradation of processed cTPs; Zn-oligopeptidase A, homologue of a peptidase that in <em>E. coli</em> was suggested to degrade small peptides down-stream of the Clp protease system; TPPII, tripeptyl peptidase.</p>
<span>doi:10.1371/journal.pone.0001994.g004</span></div><a id="article1.body1.sec2.sec2.sec1.p2" name="article1.body1.sec2.sec2.sec1.p2"></a><p>We quantified 48 proteins involved in the post-translational protein homeostasis network steps. They include the general stromal processing peptidase (SPP), both known methionine deformylases (PDF1A,B), two methionine sulfoxide reductases, seven amino-peptidases, the GroEL/ES and DnaK/J chaperone systems, HSP90 and ClpB3, protein targeting components cpSRP43, cpSRP54 and cpSecA, a homologue of trigger factor (involved in protein folding at the ribosome), the very abundant isomerase (ROC4), the complete Clp protease system (with the exception of ClpS), DegP2 protease, AtPrep1 protease suggested to be involved in degradation of processed cTPs <a href="#pone.0001994-Stahl1">[46]</a>, an oligopeptidase A, - a homologue of a peptidase that in <em>E. coli</em> was suggested to degrade small peptides down-stream of the Clp protease system <a href="#pone.0001994-Jain1">[47]</a>- and finally, a tripeptyl peptidase, TPPII (<a href="#pone-0001994-g004">Figure 4</a>). TPPII was shown to exist as a soluble, approximately 5- to 9-MDa complex and it was suggested that it is an exopeptidase that assists in aa recycling <a href="#pone.0001994-Book1">[48]</a>. These proteins span 3 orders of magnitude, with Roc4 and Cpn20 being the most abundant and some of the amino-peptidases being the least abundant. The average coefficient of variation for abundance for this group of 49 proteins was 56%, which is excellent given that the abundance range of this protein population spans 3 orders of magnitude. These data are consistent with previous quantification of the ClpPRS system (<a href="#pone.0001994-Peltier2">[49]</a>, <a href="#pone.0001994-Sjogren1">[50]</a> and HSP70 and Cpn60 systems <a href="#pone.0001994-Peltier1">[14]</a>. Considering the central role of SPP in processing of all incoming nuclear–encoded chloroplast proteins, it was interesting to note that its abundance is relatively low as compared to the chaperone systems. This suggests that the contact period between SPP and its substrates is relative short compared to the chaperone-substrate interactions.</p>



<h4>Chloroplast stromal proteome analysis by Q-TOF</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In addition to the experimental proteomics data using the LTQ-Orbitrap, we also analyzed seven unpublished independent stromal preparations using a Quadropole-Time-of-Flight instrument (Q-TOF), a high quality, but older generation instrument with lower sensitivity and speed than the LTQ-Orbitrap. These seven independent experiments identified a total of 623 non-redundant proteins (1 gene model) with 1 or more unique peptides; 81% of these proteins had a predicted cTPs (<a href="#pone.0001994.s004">Table S3</a>). The false positive protein identification rate for proteins identified with 2 or more unique peptides was zero. When also including groups of proteins identified with only shared peptides (but not with subsets of peptides identifying other proteins), the total number of ‘identified’ proteins is 720 with a 77% cTP prediction rate (not shown). These data were used only for the evaluation for subcellular protein location, but not used for quantification, nor for PTM analysis, as will be discussed further below.</p>


<h4>Assigning proteins to the chloroplast based on spectral data?</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Using the protein datasets from the ten biological replicates analyzed by LTQ-Orbitrap or Q-TOF, we tested if proteins could be assigned to the chloroplast based on the MS data alone. For that purpose, we merged each of the LTQ thylakoid and stromal data derived from the same chloroplast preparation. We then filtered the identified accessions in these LTQ and Q-TOF data sets each separately, using two simple and objective parameters: i) presence in at least 2 independent preparations, with at least 2 unique matched MS/MS spectra per preparation; the underlying assumption is that contaminations are often different in each independent preparation, and ii) at least 2 different peptide aa sequences across the independent observations. This is to avoid proteins that are identified with repeated matched MS/MS spectra (possibly in different charge states) of only 1 peptide sequence. Examples are AT5G39560 (WAVE protein) and At5G01730 (KELCH protein) each multiple times identified with peptides IEWINSVLTVPL and EDLISPR, respectively. Applying these two simple filters resulted in 739 nuclear-encoded proteins with an 86% cTP prediction rate and 46 chloroplast-encoded proteins in the LTQ-Orbitrap data sets and 340 nuclear-encoded proteins with a 91% cTP prediction rate and 13 chloroplast-encoded proteins in the Q-TOF stromal data sets (<a href="#pone.0001994.s005">Table S4</a>). Reassuringly, 96% of the proteins identified in this filtered Q-TOF dataset are also found in the filtered LTQ datasets, indicating that the chloroplast preparations were highly reproducible and consistent with the higher sensitivity of the LTQ-Orbitrap. Finally, we reanalyzed ‘in-house’ MS/MS data (from Q-TOF) that were underlying published analyses of thylakoid-associated lipoprotein particles or plastoglobules (11 biological preparations) <a href="#pone.0001994-Ytterberg1">[51]</a> and of ‘stripped’ thylakoids (3 biological preparations) <a href="#pone.0001994-Peltier3">[52]</a> and applied the same two filters for the data sets from each of the two types of preparations. Collectively, these highly filtered Q-TOF and LTQ datasets identified 762 nuclear-encoded (85% cTP) and 47 chloroplast-encoded non-redundant proteins (<a href="#pone.0001994.s005">Table S4</a>).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>To evaluate our filter procedure, we determined potential non-chloroplast contaminants in this filtered dataset using an extensive cross-correlation to more than fifty published proteomics papers on <em>Arabidopsis</em> subcellular fractions, as well as information extracted from TAIR, from SUBA <a href="#pone.0001994-Heazlewood2">[53]</a>, and other literature. Details for this manual curation are explained in the <a href="#s4">Method</a> section. Based on these cross-correlations, we found partially conflicting evidence for 21 proteins and we did not assign them to any subcellular location (<a href="#pone.0001994.s005">Table S4</a>). 30 proteins were considered non-plastid contaminations (<a href="#pone.0001994.s005">Table S4</a>). Therefore, 716 nuclear-encoded proteins (including 21 proteins dually targeted to plastid and mitochondria) were assigned to the chloroplast of which 89.5% have a TargetP predicted cTP. This dataset excludes groups of closely related homologues that we could not strictly distinguish by MS data alone.</p>


<h4>Extracting chloroplast/plastid proteins from the literature</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>In addition to the experimentally identified proteins, we also extensively screened the literature for likely plastid proteins. If the data provided for plastid localization were compelling (conclusive Western blot analysis or conclusive GFP/YFP localization), these accessions were also assigned to plastids. We did observe a significant percentage of these proteins by mass spectrometry, but sometimes they did not pass the requirement for multiple independent observations with at least two different peptide sequences, as discussed in the previous paragraph. In addition, several sets of well known chloroplast protein homologues (<em>e.g.</em> some members of the LHCII family) were now also included. In total 200 additional nuclear-encoded proteins were assigned to the plastid based on the literature (<a href="#pone.0001994.s005">Table S4</a>), bringing the total plastid assigned dataset to 916 nuclear-encoded proteins (86% cTP).</p>


<h4>cTP cleavage sites and subsequent N-terminal modifications</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Once nuclear-encoded proteins are inside the chloroplast, the N-terminal cTPs are cleaved by the general SPP. Following cTP cleavage, chloroplast proteins possibly undergo additional cleavage by the various amino-peptidases (as discussed above) and N-terminal residues can be acetylated in an enzyme-catalyzed reaction in which the α-amino group accepts the acetyl group from acetyl-CoA <a href="#pone.0001994-Michel1">[54]</a>–<a href="#pone.0001994-Kleffmann1">[57]</a>. For understanding protein function and location, and also for various practical applications (<em>e.g.</em> overexpression), identification of the correct N-terminus is important. Unfortunately, the only available cTP cleavage site predictor, which is part of TargetP, is not very precise since it was developed in 1999 based on a small training set available at the time <a href="#pone.0001994-Emanuelsson2">[58]</a>.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>To better define the cTP cleavage site and evaluate possible subsequent N-terminal PTMs, all MS data were reanalyzed allowing semi-tryptic peptides and optional N-terminal acetylation, using a narrow ±3 ppm precursor ion tolerance window in the search. We point out that N-terminal acetylation can not occur during sample preparation and occurs only enzymatically <a href="#pone.0001994-Polevoda1">[59]</a> and can thus be taken as evidence for an authentic N-terminus.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>We identified 47 N-terminal acetylated nuclear-encoded proteins (<a href="#pone.0001994.s006">Table S5</a>). Acetylation leads to an increase in retention time (~6 min under our on-line LC conditions) and if both the non-acetylated and acetylated peptides are present, they can be identified as two peptides with different retention times and a mass difference of 43.018 Da. An interesting example is shown for cysteine synthase (<a href="#pone-0001994-g005">Figure 5A, B</a>). In this case, two acetylated N-termini were identified that differ by one aa in length. <a href="#pone-0001994-g005">Figure 5A</a> shows the MS/MS spectrum for the longer doubly-charged N-terminal peptide, Acetyl-AVSIKPEAGVEGLNIADNAAQLIGK, and <a href="#pone-0001994-g005">Figure 5B</a> shows MS/MS spectrum for the shorter doubly-charged N-terminal peptide, Acetyl-VSIKPEAGVEGLNIADNAAQLIGK. Both spectra are of high quality, with respective ion scores of 100 and 79, and with ions supporting assignment of the acetylation of the N-terminal residue (as opposed to lysines) present well above the noise level. The longer peptide was also observed in the triply-charged state in both non-acetylated and N-terminally acetylated forms eluting 6.4 minutes apart (spectra not shown). This example demonstrates that the assignments of acetylated residues are not false positives. It also demonstrates that either cTP cleavage by SPP can occur at more than one precise position, and/or that after the initial cTP cleavage by SPP, additional processing occurs by chloroplast amino peptidases, followed by N-terminal acetylation.</p>
<div class="figure" id="pone-0001994-g005"><div class="img"><a name="pone-0001994-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Tandem MS spectra of N-terminally acetylated peptides suggest presence of two isoforms of Cysteine Synthase, AT2G43750.1.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) Tandem MS spectrum of doubly charged 25 aa-long, N-terminally acetylated AVSIKPEAGVEGLNIADNAAQLIGK peptide. Precursor ion is indicated with red asteric. Singly charged <em>y</em> ions are indicated by <em>blue</em> lines with corresponding aa residues shown on top - peptide sequence should be read <em>right-to-left</em>, starting with the most massive <em>y</em>(20) ion. Singly charged b ions are indicated by <em>red</em> lines with corresponding aa residues shown top – peptide sequence should be read <em>left-to-right</em>, starting with the lightest <em>b</em>(4) ion. Ions, whose presence strengthen the assignment of the N-terminal acetylation, <em>b<sup>0</sup></em>(4), <em>y<sup>++</sup>(</em>23), and <em>y<sup>++</sup></em>(24) are also indicated. (B) Tandem MS spectrum of doubly charged 24 aa-long, N-terminally acetylated VSIKPEAGVEGLNIADNAAQLIGK peptide. Precursor ion is indicated with red asteric. Singly charged <em>y</em> ions are indicated by <em>blue</em> lines with corresponding aa residues shown on top - peptide sequence should be read from <em>right-to-left</em>, starting with the most massive <em>y</em>(20) ion. Ions, whose presence strengthen the assignment of the N-terminal acetylation, <em>y<sup>++</sup></em>(21), and <em>y<sup>++</sup></em>(23) are also indicated.</p>
<span>doi:10.1371/journal.pone.0001994.g005</span></div><a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>The protein sequences around their respective experimentally determined N-termini (±10 aa) from these N-acetylated proteins were aligned in a sequence logo plot (<a href="#pone-0001994-g006">Figure 6A</a>). This illustrates conservation at the −1 and −3 positions ([V/I] and [A,C,S] respectively), as well as the +1,+2 and +3 positions, with the +1 position representing the N-terminus. The +1,+2,+3 positions show a preference for [A/V/S]-[A/S/V/L]-[S/T/A/V] and likely represents a combination of a consensus motif for the SPP and the N-acetylase. Interestingly, while N-terminal acetylation is a very common N-terminal PTM in higher eukaryotes and lower eukaryotes, N-terminal acetylation is much less common in prokaryotes (bacteria and archaea) and relatively few acetylated N-termini have been determined <a href="#pone.0001994-Polevoda1">[59]</a>. Given the bacterial origin of the chloroplast and the prokaryotic-type chloroplast gene expression and protein homeostasis machinery, it is likely that acetylation patterns and the N-acetylases in chloroplasts are more similar to prokaryotes than eukaryotes. Most prokaryotic N-terminal acetylated proteins are ribosomal proteins, with A, M, S being most frequently acetylated, and P, T, G and V less frequently acetylated. The second residue has a strong affect on the frequency of acetylation, and in eukaryotes, acidic residues (D, E) stimulate acetylation, while P and basic residues (K and R) inhibit acetylation <a href="#pone.0001994-Polevoda1">[59]</a>.</p>
<div class="figure" id="pone-0001994-g006"><div class="img"><a name="pone-0001994-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Consensus Sequences of the sites of cTP cleavage in chloroplast proteins.</span></strong></p><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1"></a><p>(A) Sequence logo of the cTP cleavage site, constructed for proteins, for which N-terminally Acetylated semi-tryptic peptides within 10 residues from predicted cTP cleavage site were observed by MS. N-terminally acetylated residue was assumed to represent true cTP cleavage site. (B) Sequence logo of cTP cleavage site, constructed for proteins, for which only non-modified semi-tryptic peptides within 10 residues from predicted cTP cleavage site were observed by MS. N-terminal residue of the semi-tryptic peptide closest to predicted cTP cleavage site was assumed to represent true cTP cleavage site. (C) Sequence logo of 203 stromal proteins for which the most N-terminal peptide (full tryptic or the semi-tryptic peptide) was within 10 residues of the predicted cTP, (D) Sequence logo of the predicted cTP of all 898 annotated <em>Arabidopsis</em> chloroplast proteins, but only those proteins (831) were used with a predicted cTP length of at least 20 aa residues. (E) Sequence logo of the predicted cTP of 802 rice proteins representing the best homologues for 898 annotated <em>Arabidopsis</em> chloroplast proteins, but only those proteins (714) were used with a predicted cTP length of at least 25 aa residues.</p>
<span>doi:10.1371/journal.pone.0001994.g006</span></div><a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p>The prevalence of A,V,S,T (and M) as acetylated residues in our chloroplast proteins suggests that acetylation is carried out by a so-far unidentified NatA type acetylase. Interestingly, the second position for these sequences is occupied by A, S, V or L which is different from the preference in eukaryotes, although none of these residues have inhibitory effects on acetylation in eukaryotes. Additionally, the +3 position is occupied by the small, uncharged residues, S,A,V, and T, in agreement with avoiding charged and very large residues in eukayotes <a href="#pone.0001994-Polevoda1">[59]</a>.</p>
<a id="article1.body1.sec2.sec6.p6" name="article1.body1.sec2.sec6.p6"></a><p>To distinguish between motifs for the SPP and the acetylase, we also assembled those proteins for which the most N-terminal identified peptide was within 10 aa residues from the predicted cTP cleavage site (upstream or down-stream) and for which the N-terminal residue was un-modified. The sequences of these proteins were aligned, assuming that the most N-terminal identified peptide represented the true N-terminus of the protein, resulting in a sequence logo plot of 62 proteins (<a href="#pone-0001994-g006">Figure 6B</a>). The upstream residues at the −3 and −1 positions are similar to the sequence logo plot from acetylated proteins (<a href="#pone-0001994-g006">Figure 6A</a>), but with stronger conservation of the A at the −1 position. The residues at the +1 and +2 position were enriched for A, S, but there was little conservation at the +3 position. This suggests that the conservation at the +3 position in <a href="#pone-0001994-g006">Figure 6A</a> was due to preference for the N-terminal acetylase, rather than SPP. This helps to define the consensus motif for N-acetylation in chloroplasts, which is important for future N-acetylation predictions. Interestingly, the set of acetylated proteins (<a href="#pone-0001994-g006">Figure 6A</a>), but not the unmodified set (<a href="#pone-0001994-g006">Figure 6B</a>), showed also some conservation at the −3 to −7 positions.</p>
<a id="article1.body1.sec2.sec6.p7" name="article1.body1.sec2.sec6.p7"></a><p>There is a potential pitfall in this analysis using tryptic digests (cleaving C-terminal of R and K), in that the −1 position is enriched for R or K. However, using semi-tryptic peptides, rather than full tryptic peptides suppressed this potential pitfall, which is obvious from the weak ‘conservation’ lack of R or K at the −1 position in the sequence logos. We note that the average length of the peptides in our analysis was ~15 residues, explaining why there is little risk for artificial enrichment for R or K within the first 10 residues down-stream of the cleavage site. To illustrate this point, we created a sequence logo of 203 stromal proteins for which the most N-terminal peptide (full tryptic or the semi-tryptic peptide) was within 10 residues of the predicted cTP (<a href="#pone-0001994-g006">Figure 6C</a>); this clearly shows an ‘artificial’ enrichment of R and K at the −1 position, which has no biological relevance.</p>
<a id="article1.body1.sec2.sec6.p8" name="article1.body1.sec2.sec6.p8"></a><p>The sequence logo of the original experimental training set showed a consensus sequence for the cTP cleavage site of [V/I][R/A/V][A/MC]↓[A/S/M]. However, using the same proteins to predict the cTP consensus motif with ChloroP produced a consensus sequence of [V/I/P][R/S/V]↓[A/C/S][S/A/M], essentially moving the cleavage site one residue upstream <a href="#pone.0001994-Emanuelsson2">[58]</a>. When creating a sequence logo of the predicted cTP from a much larger dataset of 898 annotated <em>Arabidopsis</em> chloroplast proteins (outer envelope proteins were removed) (see further below), we observe a similar consensus sequence as for the original training set and a similar upstream shift of one residue as compared to the experimental consensus sequence presented. Moreover, the sequence logo of the predicted cTP of our large dataset shows a similar enrichment for R at the −1 position as originally observed (<a href="#pone-0001994-g006">Figure 6D</a>). However, our experimental data (<a href="#pone-0001994-g006">Figure 6A,B</a>) suggest that there is not such a strong preference for an R. The one residue upstream shift by the cTP cleavage site predictor, is most likely a consequence of the build-in preference (based a small set of <em>in vitro</em> cleavage experiments) for cleavage after an R, as explained in <a href="#pone.0001994-Emanuelsson2">[58]</a>. Based on our experimental data and the low abundance of the amino peptidases quantified in our study, it seems unlikely that all imported and SPP processed proteins undergo systematically a one (or two) aa removal from the N-terminus. We suggest that there is an excellent basis and larger experimental data set for improvement of the cTP cleavage site predictor.</p>


<h4>lTP cleavage site analysis</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>So far, 66 nuclear-encoded proteins assembled from various papers have been identified in the chloroplast lumen (<a href="http://ppdb.tc.cornell.edu/subproteome.aspx">http://ppdb.tc.cornell.edu/subproteome.a​spx</a>). These proteins have a lumenal transit peptide (lTP) down-stream of the cTP which is needed to cross the thylakoid membrane. In addition several integral thylakoid membrane proteins also have an lTP.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>The LTQ-Orbitrap analysis identified 46 out of these 66 lumenal proteins, and except for the lumenal PSI subunit psaN (AT5G64040; see further below), none of them showed N-terminal acetylation, consistent with cleavage of the lTP on the lumenal side of the thylakoid membrane and the lack of a known acetylase in the lumen. This also confirms the low FDR for acetylation. In addition, we identified five integral thylakoid proteins with known lTP (<a href="#pone-0001994-t002">Table 2</a>). For 13 of these lTP proteins, we identified the processed N-terminus that exactly matched the predicted cleavage site and/or N-terminal Edman sequencing data from previous studies (<a href="#pone-0001994-t002">Table 2</a>). For one accession (AT3G01480 - TLP40), we found that the most N-terminal peptide matched the Edman sequencing data, but was 11 residues downstream of the predicted lTP. This either suggests an unusual lTP cleavage after the residues ‘DVS’, instead of after the predicted residues AHA, or this suggests that there was an additional cleavage event. Interestingly, the most N-terminal peptide for OEC-23-like (AT2G39470) and an isomerase (AT1G18170) were upstream of the predicted lTP cleavage site (<a href="#pone-0001994-t002">Table 2</a>), indicating either incorrect lTP cleavage site prediction or identification of an intermediate processing step. The N-terminal Edman sequence (ASA↓NAGVI) of psaN, coinciding with an lTP cleavage site, was 2 residues down-stream of the N-acetylated and unmodified peptides that we identified multiple times by MS. This suggests that biogenesis of psaN is more complicated that anticipated.</p>
<div class="figure" id="pone-0001994-t002"><div class="img"><a name="pone-0001994-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001994.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001994.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Analysis of N-termini of lumenal proteins and potential new lumenal proteins.</span></strong></p><span>doi:10.1371/journal.pone.0001994.t002</span></div>

<h4>The search for new lumenal proteins</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>Genome-wide predictions <a href="#pone.0001994-Sun1">[4]</a>, <a href="#pone.0001994-Peltier4">[60]</a> and manual inspection <a href="#pone.0001994-Schubert1">[61]</a> suggest that there are possibly another 20–100 unknown lumenal proteins. Therefore we investigated those identified proteins for which the most N-terminal (tryptic/semi-tryptic and/or modified) peptide matched the predicted lTP cleavage site or was within 100 residues downstream of the predicted lTP, <span class="underline">without</span> any additional peptides matching upstream of the predicted lTP site. These proteins were then evaluated for experimental mass spectrometry data and other information. This identified four putative lumenal proteins with unknown function, three of which were also identified in our earlier thylakoid analyses <a href="#pone.0001994-Friso1">[36]</a>, <a href="#pone.0001994-Peltier3">[52]</a> (<a href="#pone-0001994-t002">Table 2</a>). The predicted lTP cleavage site (SMA↓ENI) for one of the proteins (AT2G23670) coincides precisely with the most N-terminal peptide identified (ENIPLFGIR). Putative lumenal protein At1G05385 is an orthologue of cyanobacterial protein Psb27 which was suggested to be involved in assembly of the water splitting complex of PSII <a href="#pone.0001994-Nowaczyk1">[62]</a>, <a href="#pone.0001994-Roose1">[63]</a>, consistent with location on the lumenal site of the thylakoid. We note that Psb27 has another homologue in <em>Arabidopsis</em>, At1G03600, for which we previously determined the N-terminus by Edman sequencing and which we assigned to the lumen <a href="#pone.0001994-Peltier4">[60]</a> and was recently showed to be involved in PSII assembly in <em>Arabidopsis</em> <a href="#pone.0001994-Chen1">[64]</a>. Putative lumenal protein AT2G36145.1 has no predicted function, nor predicted TMD and a maize homologue (TC294822) was found in maize thylakoids (Majeran et al, submitted). Putative lumenal protein AT5G42765 was identified with strong MS data in the thylakoid and the most N-terminal (semi-tryptic) peptide suggests an lTP cleavage site of AFA↓FSLGISGPK (<a href="#pone-0001994-t002">Table 2</a>).</p>


<h4>N-terminal modifications of chloroplast-encoded proteins</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>Chloroplast-encoded proteins are synthesized with an N-terminal formylated methionine residue. The N-termini of 59 chloroplast–encoded proteins from various plant species were experimentally determined by Edman sequencing across multiple studies, and subsequently assembled and evaluated <a href="#pone.0001994-Giglione2">[26]</a> (see the table at <a href="http://www.isv.cnrs-gif.fr/tm/maturation/images/chloro.html">http://www.isv.cnrs-gif.fr/tm/maturation​/images/chloro.html</a>). This showed that most N-formyl groups are removed by the chloroplast localized deformylases, PFD1B and possibly PFD1A <a href="#pone.0001994-Dirk2">[25]</a>, which were both quantified in our study (<a href="#pone-0001994-g004">Figure 4</a>). The deformylation is often followed by removal of the methionine, presumably by one or several of the chloroplast methionine aminopeptidases, MAP1B,C,D <a href="#pone.0001994-Giglione1">[23]</a>, <a href="#pone.0001994-Giglione2">[26]</a>. In this study we only observed and quantified MAP1B (<a href="#pone-0001994-g004">Figure 4</a>), suggesting that MAP1B is more abundant than the other two amino peptidases.</p>
<a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>Since no systematic experimental N-terminal analysis for chloroplast-encoded proteins in <em>Arabidopsis,</em> we used the LTQ analysis to evaluate information about the N-termini of 54 identified chloroplast-encoded proteins. For 12 chloroplast-encoded proteins we determined the N-terminus (<a href="#pone-0001994-t003">Table 3</a>), while the N-terminus for several others could not be determined, because they start with a short (semi-)tryptic peptide of less than 7 residues These short peptides are excluded from our database searches because of the relatively low information content. Four out of those 12 proteins (rpl14, and NDH subunits A,I,J) were identified with their N-terminal methionine. The N-termini of Rpl14 and NdhI were also observed with the methionine removed. In case of NDHA, the methionine was formylated; however, we must be careful as samples have been observed to become non-enzymatically formylated during the sample preparation <a href="#pone.0001994-Whitelegge1">[65]</a>. For five proteins (cyt<em>b6</em>, rps8, rps15, D2, CF1ε), we observed that the methionine was always removed and two of those processed N-termini (for D2, CF1ε) were acetylated (both at a T residue). In case of CF1β and RBCL, the first two residues were removed and the new N-termini (respectively an R and P residue) were acetylated. Thus in addition to preferential N-acetylation of A,V,S,T,M observed for the imported nuclear-encoded proteins (<a href="#pone-0001994-g006">Figure 6A</a>), N-acetylation can also occur at T, R and P. Cyt<em>f</em> is the only c-encoded protein with a (SecA-dependent) lTP; this lTP has 36 residues. We did identify the processed N-terminus (starting at residue 37) of mature cyt<em>f</em> which was not acetylated, as expected since it is located on the lumenal side of the thylakoid membrane (<a href="#pone-0001994-t003">Table 3</a>). The MS based data were consistent with the data assembled or predicted in <a href="#pone.0001994-Giglione2">[26]</a>.</p>
<div class="figure" id="pone-0001994-t003"><div class="img"><a name="pone-0001994-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001994.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001994.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>N-terminal modifications of chloroplast-encoded proteins.</span></strong></p><span>doi:10.1371/journal.pone.0001994.t003</span></div>

<h4>Are there chloroplast proteins that are targeted via the secretory system?</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>Recently there was a report that a chloroplast carbonic anhydrase (At3G52720) was targeted to the chloroplast via the secretory system <a href="#pone.0001994-Villarejo1">[8]</a>. It was shown that that this carbonic anhydrase was synthesized with an N-terminal sP and after cleavage in the ER, it reached the chloroplast through an unknown mechanism. It was hypothesized that this maybe a more common pathway for targeting of nuclear-encoded proteins to the chloroplast. Therefore, the 916 nuclear-encoded plastid proteins (but excluding 18 known outer envelope proteins) were passed through the SignalP predictor (<a href="http://www.cbs.dtu.dk/services/SignalP/">http://www.cbs.dtu.dk/services/SignalP/</a>). Only 0.6% of the proteins were predicted to have a sP, while negative test sets, <em>e.g.</em> cytosolic and mitochondrial proteins, showed similar or higher sP prediction rates (not shown). Therefore, it seems unlikely that a significant percentage of chloroplast proteins are targeted via the secretory system.</p>


<h4>cTP properties of chloroplast proteins and species specific differences</h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>In addition to our current analysis discussed above, our previous analyses on smaller data sets <a href="#pone.0001994-Sun1">[4]</a>, <a href="#pone.0001994-Peltier1">[14]</a> also show that the true positive prediction rate for cTP prediction by TargetP (and other predictors) is ~86%, despite various reports of lower true positive rates - for discussion see <a href="#pone.0001994-Emanuelsson1">[2]</a>, <a href="#pone.0001994-vanWijk1">[5]</a>. Properties and aa composition of cTP have been extensively analyzed in the past <a href="#pone.0001994-vonHeijne1">[1]</a>, <a href="#pone.0001994-Emanuelsson2">[58]</a>, but the ~10 fold larger set of chloroplast proteins from this study provides an excellent opportunity to re-evaluate these properties.</p>
<a id="article1.body1.sec2.sec11.p2" name="article1.body1.sec2.sec11.p2"></a><p>We created sequence logos of the 20 most N-terminal residues (<a href="#pone-0001994-g007">Figure 7A</a>) and aa frequency distribution plots over the entire predicted cTP (<a href="#pone-0001994-g007">Figure 7B</a>) for the 916 annotated nuclear-encoded plastid proteins (excluding outer envelope proteins). A weak three domain structure is discernable: (i) an N-terminal domain beginning with MA, rich in S, L, A, and T, potentially terminating with P, (ii) a central domain lacking acidic residues (D/E) and with S, L, and P overrepresented but decreasing slightly towards the cleavage site, while R increases, and (iii) a C-terminal cleavage site domain enriched in R and with a loosely conserved motif VRA↓AA around the cleavage site. In contrast to previous observations <a href="#pone.0001994-Emanuelsson1">[2]</a>, <a href="#pone.0001994-vonHeijne2">[66]</a>, the plots show that the boundary of the N-region is rich in P but not G, that acidic residues (D/E) are underrepresented across the entire cTP (except for the second position), and that apolar L (frequency is 10.4% in total <em>Arabidopsis</em> cTP set) and P (7.3%) are enriched, in addition to the hydroxylated residues S (19.3%) and T (6.9%). The plots also show that the second position is an A residue in only 57% of the cases (<a href="#pone-0001994-g007">Figure 7A,B</a>).</p>
<div class="figure" id="pone-0001994-g007"><div class="img"><a name="pone-0001994-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001994" data-uri="info:doi/10.1371/journal.pone.0001994.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001994.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001994.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001994.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Chloroplast transit peptide analysis of <em>Arabidopsis</em> chloroplast proteins and predicted rice homologues.</span></strong></p><a id="article1.body1.sec2.sec11.fig1.caption1.p1" name="article1.body1.sec2.sec11.fig1.caption1.p1"></a><p>Sequence logos of the first 20 N-terminal residues (A,C) and aa distribution and frequency across the normalized (binned) cTP length (B,D) for 898 annotated <em>Arabidopsis</em> chloroplast proteins (A,B) and the 802 predicted rice homologues (C,D). Only those proteins were used with a predicted cTP length of at least 20 aa residues (831 and 714, respectively from <em>Arabidopsis</em> and rice).</p>
<span>doi:10.1371/journal.pone.0001994.g007</span></div><a id="article1.body1.sec2.sec11.p3" name="article1.body1.sec2.sec11.p3"></a><p>To investigate if there is a significant difference between <em>Arabidopsis</em> and rice chloroplast proteomes (two species about 100 million years apart in evolutionary distance), best rice homologues (E value&lt;0.1) were obtained for the <em>Arabidopsis</em> set. This resulted in 802 rice proteins of which 74% had a predicted cTP (compared to 86% for the Arabidopsis proteins). Sequence logos and aa frequency distribution plots were generated similarly as for <em>Arabidopsis</em> (<a href="#pone-0001994-g007">Figures 7C, D</a>). This showed that the second position in plastid rice cTPs is enriched for A (about 59%), S, and E, followed by a clear preference for small uncharged residues A, S, and L (<a href="#pone-0001994-g007">Figure 7C,D</a>), a pattern very similar to <em>Arabidopsis</em>. On the other hand, a much higher abundance of A (18.0% in predicted rice cTP set versus 7.1% in predicted <em>Arabidopsis</em> cTP set), in particular, but also P, G, and R was observed in rice, while N, K, S, and T were less common in rice. Surprisingly, threonine is slightly underrepresented in these rice cTPs compared to its general abundance in proteins (Swiss-Prot release 54.0). Overall, this is in agreement with an earlier report for smaller datasets <a href="#pone.0001994-Kleffmann1">[57]</a>. There is also a difference in the average length of the predicted cTPs: 50.4 residues for rice and 53.6 residues for <em>Arabidopsis,</em> p-value&lt;0.0002 using Mann-Whitney test (excluding predicted cTPs shorter than 20 residues since these are likely wrong predictions). Moreover, the rice cTP content was slightly more conserved than for <em>Arabidopsis.</em> Finally, the predicted cTP cleavage site for the chloroplast rice homologues is very similar as for <em>Arabidopsis</em>, but with a stronger enrichment for A over S (<a href="#pone-0001994-g006">Figure 6E</a>). Our analysis suggests that specific rice and <em>Arabidopsis</em> cTP predictors might yield higher true positive rates and could also provide better predictions for other sequenced monocotyledons (<em>e.g.</em> maize, sorghum) and dicotyledons (<em>e.g.</em> poplar, medicago, tomato). Similarly, a specific cTP predictor for the sequenced green algae <em>Chlamydomonas reinhardtii</em> may help to alleviate the poor prediction success rates.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>The identified chloroplast proteome and its functions and criteria for contaminations</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>This study identified 1325 proteins in chloroplast samples, including ~400 proteins not yet experimentally observed before. We used two MS platforms and 95% of the proteins identified in the stromal preparations by Q-TOF MS were also found in the LTQ-Orbitrap analysis. This clearly reflects the higher sensitivity and speed of the LTQ-Orbitrap platform, as was recently detailed <a href="#pone.0001994-Elias1">[67]</a>, and also underscores the reproducibility of the stromal preparations.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Uncurated experimental proteomics data from isolated subcellular compartments do not provide sufficient quality for localization, as contamination of even less than 1% protein mass can result in significant numbers of identified proteins from other subcellular compartments. However, by demanding that proteins are identified with multiple high quality MS/MS spectra (with FPR &lt;1%) and multiple unique peptide sequences in multiple independent preparations, many of these contaminations can be eliminated. We showed that this was particularly effective for chloroplasts. Subsequent comparison to proteomics studies of other subcellular compartments and cross-correlation with studies of localization and function removed (most) remaining contaminants.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Some 20 non-chloroplast proteins were frequently observed in our chloroplast preparations, with scores placing them in low abundance classes. For instance, we frequently observed peroxisomal proteins involved in photorespiration. Repeated analysis of total <em>Arabidopsis</em> seedling extracts by LTQ-Orbitrap MS as described here, show that these proteins are among the top scoring proteins in terms of matched MS/MS spectra (Zybailov, Friso and van Wijk, unpublished data). We also noted that most of these contaminants are observed in many proteomics studies of other organelles and membranes, further illustrating the benefits of cross-correlating different studies, made easy through an integrative PPDB. A recent mitochondrial proteome analysis observed many enzymes in the cytosolic glycolytic pathway and it was suggested that they interact tightly with mitochondria, explaining their frequent identification in mitochondrial samples <a href="#pone.0001994-Heazlewood3">[68]</a>. A similar scenario may explain the identification of cytosolic HSP70 and an isomerase in the chloroplast preparations. This brings about the question how one can identify proteins that have less than five orders of magnitude abundance than the most abundant chloroplast proteins. We believe that it will be possible to assign such proteins to the chloroplast based on repeated MS based identification, when many other subcellular compartments are also analyzed with similar high sensitivity setups. A collective effort of the plant (proteomics) community specialized in preparing the various subcellular compartments will be needed to create a high quality ‘proteome map’ of the <em>Arabidopsis</em> cell.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>The stringent filtering of our experimental proteomics data, combined with known chloroplast proteins from the literature, identified 916 nuclear-encoded chloroplast proteins. The experimentally identified proteins include many low abundant regulatory proteins, such as several kinases involved in aa biosynthesis (<em>e.g</em>. shikimate-kinases At2G35500 and At3G26900), a protein tyrosine phosphatase (AT3G44620) and the thylakoid integral membrane protein state-transition kinases, STN7 <a href="#pone.0001994-Bellafiore1">[69]</a> and STN8 <a href="#pone.0001994-Bonardi1">[70]</a>, not earlier observed in MS studies. In addition, we identified multiple low abundant putative RNA and DNA proteins and four RNA polymerases (Rpo-A,B,C1,C2) that were previously only identified by MS in experiments specifically focused on the transcriptional apparatus and the plastid chromosome (the nucleoid) <a href="#pone.0001994-Pfalz1">[71]</a> – here it is worth noting that we identified these polymerases in the thylakoid fraction, consistent with the thylakoid association of the plastid nucleoid. We also identified several new putative thylakoid lumen proteins that may help to clarify the function of this enigmatic compartment. The chloroplast protein dataset will be an important resource for detailed chloroplast studies, as well as large scale <em>Arabidopsis</em> analysis.</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>Ultimately, it would be ideal to identify every protein in the chloroplast, but given the anticipated wide range of protein abundance (&gt;6 orders of magnitude) and the dynamics of the chloroplast proteome, this is not feasible. To better understand coverage and function of the identified chloroplast proteome, we compared the MapMan bin distribution (for functional assignments) of the cTP predicted plastid proteome (4053 - only 1 gene model) with the curated plastid proteome. This shows that the identified chloroplast proteome is overrepresented (as compared to the predicted plastid proteome) in co-factor and vitamin metabolism (59%), Photosynthesis (58%), N-metabolism (+54%), tetrapyrrole metabolism (47%), major CHO metabolism (+45%), redox regulation (+44%), oxidative pentose phosphate pathway (49%), while proteins involved in signaling (−74%), stress (−69%), development (−47%), unassigned (−45%) and RNA (−40%) are underrepresented. Even if functional assignments and cTP predictions are imperfect, this provides an impression of relative protein abundance of the various functional classes and processes within chloroplasts, with overrepresented and underrepresented proteins being respectively of higher and lower abundance. The actual quantification of the stromal proteome using the spectral counting technique provides a more accurate estimate of pathway abundance.</p>


<h4>Stromal proteome quantification and pathways analysis</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Large scale quantification of protein abundance, alongside quantification of transcripts and metabolites, will be needed for systems analysis of <em>Arabidopsis</em> <a href="#pone.0001994-Glinski1">[13]</a>. However, large scale protein quantification remains challenging, in particular due to the large expression range and proteome complexity <a href="#pone.0001994-Listgarten1">[19]</a>–<a href="#pone.0001994-Bantscheff1">[21]</a>. In this study we show that ‘label-free’ MS based quantification does provide meaningful data and we provide a quantification protocol.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The abundance of 557 stromal proteins was determined using SPC that were either normalized for the length of each mature protein, or the number of theoretical tryptic peptides for each protein within the relevant mass range. The two normalization procedures resulted in comparable results, with the normalization by theoretical tryptic peptides yielding somewhat better accuracy, as determined from cross-correlation to published gel-based quantification of some 200 stromal proteins <a href="#pone.0001994-Peltier1">[14]</a>. The cross-correlation showed that the ~10–20 most abundant proteins were somewhat underestimated, as is expected when using DDA with dynamic exclusion (<em>i.e</em>. avoidance of repeatedly sequencing the same peptide). Quantification by spectral counting is likely to improve if a correction is introduced for the propensity of the theoretical peptides to be observed, as discussed in <a href="#pone.0001994-Liu2">[43]</a> <a href="#pone.0001994-Mallick1">[42]</a>. Such propensity is dependent on the proteomics ‘pipeline’ (protein separation and MS platform) <a href="#pone.0001994-Mallick1">[42]</a>, <a href="#pone.0001994-Liu2">[43]</a>. The gel based quantification using native gels (not IEF) as the 1<sup>st</sup> dimension and SDS-PAGE as the second dimension used in our previous study <a href="#pone.0001994-Peltier1">[14]</a> is particularly good for quantification of the most abundant proteins. However, this gel based method fails to accurately detect and quantify the lower abundance proteins and also cannot accurately quantify proteins that co-migrate with other proteins. Gel based quantification is thus not a suitable alternative for large scale quantification. Additional (but far more costly and labor intensive) techniques, such as titration with known amounts of stable isotope labeled ‘proteotypic’ peptides uniquely representing each proteins will be needed to obtain precise quantifications (reviewed in <a href="#pone.0001994-Nesvizhskii1">[20]</a>). Recently, quantitative MS using multiple reaction monitoring (MRM based quantification) without stable isotopes was demonstrated and may provide a good alternative for quantification of known sets of proteins <a href="#pone.0001994-StahlZeng1">[72]</a>. It is clear that while further optimization of MS based quantification is beneficial, our current quantification of the stromal proteome using spectral counting does provide an excellent overview of over 550 stromal proteins, with relatively small variation and a dynamic range of four orders of magnitude. We used this quantitative data to highlight the post-translational protein homeostasis machinery, while the remainder of the dataset is provided as a community resource for functional analysis of the chloroplast.</p>


<h4>Analysis of the cTP and cleavage site suggests potential for improved subcellular localization prediction</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Determination of the protein subcellular localization in <em>Arabidopsis</em> is important since it often dictates the function of a protein and it is also needed to calculate metabolic flux and contributions to signal transduction cascades (e.g. kinases, phosphates, etc) <a href="#pone.0001994-Fernie1">[73]</a>. Accurate prediction of subcellular protein localization may ultimately alleviate the necessity to identify each protein experimentally. Currently, large scale experimental proteome studies are needed to provide larger training sets to improve such predictors and our study provides such a large, carefully curated, positive training set for the chloroplast.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Prediction of protein location in the chloroplast (plastid) is an important tool to identify proteins that so far escaped experimental identification, but true positive rates (~86%) and false positive rates (~35%) need to be further improved <a href="#pone.0001994-Emanuelsson1">[2]</a>. The large set of curated plastid proteins in this study and the detected features in the cTP cleavage site, suggest that there is a good basis for improving plastid localization prediction. Analysis of unfiltered sets of identified proteins in stromal samples suggested that the cTP success rate was much lower for low abundant proteins than for high abundant proteins. However, when chloroplast membrane associated and thylakoid lumen proteins, as well as non-chloroplast contaminants, were removed, this bias was strongly reduced, indicating that cTP prediction rates are relatively independent of protein abundance, although some bias for the very high abundant proteins (class I and II) does seem to exist, as also indicated by the 97% cTP prediction rate of those 156 proteins that were identified in each of the nine LTQ experiments (<a href="#pone.0001994.s007">Table S6</a>). We also observed clear indications that plastid proteins in rice and <em>Arabidopsis</em> are different in their cTPs, confirming less detailed and smaller scale analyses <a href="#pone.0001994-Kleffmann1">[57]</a>. This warrants the development of species-specific (or dicotyledon and monocotyledon specific) plastid localization predictors. Retraining of the TargetP predictor based on information presented in this study, as well as new datasets from the recent literature is in progress (Emanuelsson, unpublished).</p>


<h4>N-terminal processing and modifications and their physiological role</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The proteome complexity is increased by PTMs that play a role in protein location, protein-protein interactions and folding, enzymatic function and protein life-time. Taking advantage of the high accuracy of the Orbitrap, we were able to determine the N-termini of 47 acetylated nuclear-encoded chloroplast proteins and also identified processed (by deformylase and amino-peptidase activity) and unprocessed N-termini of chloroplast-encoded proteins. The biological significance of N-acetylation varies with the particular protein and has been shown to affect protein-protein interactions, protein assembly and enzymatic activities <a href="#pone.0001994-Polevoda1">[59]</a>. Analysis of the acetylated N-termini identified in the current study suggests that in case of nuclear-encoded proteins, small hydrophobic residues A, V, followed by small, hydroxylated residues S and T account for 80% of the acetylated residues. Position 2 is occupied by A, S, V, L and position 3 essentially by S,T,A,V. This illustrates that G, charged residues (D,E,K,R,H), aromatic residues (Y,N,Q) and residues with large side chains (I and L) are largely avoided in the first three positions. This includes primary destabilizing residues in <em>E. coli</em> (F,Y,W,L), as well secondary destabilizing residues (R,K) although the latter is likely a consequence of the tryptic digest. In case of chloroplast-encoded acetylated proteins, all but one were acetylated after f-Met removal. The acetylated N-termini residues are either T, R or P, with the secondary residues L,I,T,Q, which is very different from the residues in nuclear-encoded proteins. This could suggest involvement of two different N-acetylases, one operating co-translationally in case of chloroplast-encoded proteins and the other operating pos-translationally. We are not aware that anyone has investigated the role of acetylation of these or other chloroplast proteins, and this collection of acetylated proteins will provide a basis for such investigations.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>The N-end rule states that the half-life of a protein is determined by the nature of its N-terminal residue. Whereas this fundamental principle is conserved from bacteria to mammals, prokaryotes and eukaryotes employ distinct proteolytic machineries for degradation of N-end rule substrates and the stabilizing and destabilizing residues are also different. The proteolytic machineries in plastids and chloroplast are all distinctly of prokaryotic origin, and it is therefore anticipated that the N-end rule stabilizing and destabilizing residues also follow bacterial rules. Proteolysis in prokaryotes and plastids is entirely independent of ubiquitin. Instead substrate selection is mediated by specialized HSP100 components (in chloroplasts ClpC1,C2, and ClpD) and aided with so-called adaptor proteins (ClpS), which together transfer the substrates to their cognate proteolytic partners. The N-rule and proteolysis in prokaryotes is best studied in <em>E. coli</em> <a href="#pone.0001994-Mogk1">[74]</a>, <a href="#pone.0001994-Baker1">[75]</a>. In <em>E. coli</em>, large hydrophobic (L.F,W,Y) and basic residues (R,K) represent primary and secondary destabilizing residues. Generally, M-aminopeptidases do not generate N-end rule destabilizing N-termini and indeed, the processed N-termini of the chloroplast-encoded N-termini followed this rule. In addition, in <em>E. coli</em>, proteins with an R or K as N-terminus can be destabilized by addition of L or F through an L/F-tRNA protein transferase – but no such transferase has yet been identified in chloroplasts. As mentioned above, we only observed one nuclear-encoded protein with an N-terminal destabilizing residue, Y in AT3G55250.1, which was observed multiple times but always carrying an N-terminal acetylation. Inspection of the sequence logos of the experimental, non-acetylated cTP cleavage site (<a href="#pone-0001994-g006">Figure 6B</a>) showed that the experimental N-termini essentially avoided destabilizing residues.</p>


<h4>Creating resources for the plant community</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>To disseminate our chloroplast proteomics data and integrate these with other types of proteomics information, we had developed the Plastid Proteomics Database, PPDB, initially reported in <a href="#pone.0001994-Friso1">[36]</a>. For this study we have added several features to the PPDB, in particular to better display details of MS based identification such that the user can better evaluate the significance of the protein identification and assigned localization. This includes mass error for all precursor ions and matched peptides, protein Mowse scores, as well as peptides with PTMs and a search option to extract all PTMs (with peptides and accessions). Importantly, all matched MS data are projected on the relevant gene models (using ‘pop-up’ windows), aiding in a better understanding of gene models and PTMs. MS data from earlier published papers (since summer 2004) were all re-searched and filtered and search data extracted and uploaded into PPDB. In addition, all protein accessions can be directly cross-referenced against identified proteins from more than 50 published proteomics papers (from <em>Arabidopsis</em> and other members of the <em>Brassicae</em> family), as well as the subcellular localization in GO (experimental) from TAIR, localization data from SUBA (<a href="http://www.suba.bcs.uwa.edu.au">http://www.suba.bcs.uwa.edu.au</a>) and best matching homologues in maize and rice. Many of these data types can be directly exported from the PPDB as excel files. Since we also curated the location of hundreds of non-chloroplast proteins, and to reflect other available information, we changed the name from Plastid PDB to Plant PDB. Our PPDB is a unique resource for the plant community and complements other plant proteomics resources, such as Promex, a mass spectral reference database for proteins and protein phosphorylation sites <a href="#pone.0001994-Hummel1">[76]</a>, SUBA a database for <em>Arabidopsis</em> protein localization <a href="#pone.0001994-Heazlewood1">[12]</a>, as well as databases specialized in other organelles, such as peroxisomes <a href="#pone.0001994-Reumann1">[77]</a>. We conclude that this study provides the most comprehensive chloroplast proteome analysis to date and a unique web-based <em>Arabidopsis</em> proteome resource.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Plant growth, chloroplast isolation and subfractionation</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p><em>A. thaliana</em> (Col 0) wt and <em>ffc1-2</em> <a href="#pone.0001994-Amin1">[78]</a> were grown on soil under short day (10/14 hours light/dark) with 280 µmol photons.m<sup>−2</sup>.s<sup>−1</sup> light at 23/19°C in controlled growth chambers (Conviron). <em>clpr2-1</em> <a href="#pone.0001994-Rudella1">[79]</a> was grown on soil under short day (10/14 hours light/dark) with 120 µmol photons.m<sup>−2</sup>.s<sup>−1</sup> light at 23/19°C in controlled growth chambers (Conviron). Intact chloroplasts were purified from fully developed leaf rosettes. Chloroplasts were lysed and stroma and thylakoid membranes were collected principally as described in <a href="#pone.0001994-vanWijk3">[80]</a>. A low density membrane-enriched fraction was collected by high speed centrifugation (20 min at 120.000 <em>xg</em>) of stromal extracts. <a href="#pone.0001994.s007">Tables S6A,B</a> show the ratios of weighed SPC between stroma and thylakoid samples for some of the 156 proteins, which were identified in all nine experiments and provides a good indication of their subchloroplast localization.</p>


<h4>Protein analysis by Mass spectrometry</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>400 µg of thylakoid or stromal proteins were separated by SDS-PAGE (12% acrylamide) and stained with Coomassie Brilliant blue. Each gel lane was cut into 12 bands followed by reduction, alkylation, in-gel digestion with trypsin and peptides were extracted, as described in <a href="#pone.0001994-Shevchenko1">[81]</a>. Peptide extracts were dried down and resuspended in 15–20 µl 5% formic acid for MS/MS analysis by either reverse phase nanoLC-ESI-QTOF (Micromass/Waters) or reverse phase nanoLC-ESI-LTQ-Orbitrap (Thermoelectron) and typically 6.4 µl were injected for each run.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>The nanoLC-Q-TOF was interfaced with a CapLC system (Waters) and an autosampler from Waters. Analysis by nanoLC-Q-TOF was as follows: Used automated sample pickups, 6.4 µl peptide extracts were loaded at 15 µl.min<sup>−1</sup> for 6 min on a guard column (LC Packings; MGU-30-C18PM), followed by elution and separation on a PepMap C18 reverse-phase nano column (LC Packings nan75-15-03-C18PM), using 90-min gradients with 95% water, 5% acetonitrile (ACN), 0.1% FA (solvent A), and 95% ACN, 5% water, 0.1% FA (solvent B) at a flow rate of 200 nl/min. A precolumn splitter was used to reduce the flow rate of 10 µl.min<sup>−1</sup> to 200 nl.min<sup>−1</sup>. Each sample injection and analysis was followed by two blank injections using 60-min and 40-min gradients to prevent carry over from sample to sample.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>The Q-TOF was operated in positive ion mode with sample cone voltage of 35 kV, capillary voltage of 3.3 kV and a source temperature of 90°C. The samples were run in data-dependent mode (DDA) where each full MS scan was followed by three consecutive MS/MS scans of the 3 most abundant peptide molecular ions (typically doubly and triply charged ions), which were selected consecutively for collision-induced dissociation (CID). The MS survey scans (m/z 350–1550) were carried out with a scan time of 1 second (s) and a interscan time of 0.08 s. MS/MS spectra were automatically acquired when the peak intensity rose above a threshold of 10 counts.s<sup>−1</sup>. Normalized collision energies for peptide fragmentation were set using the charge–state recognition files for 1+, 2+, 3+, 4+ peptide ions provided by Masslynx (Waters). For tandem MS acquisition we used a scan range from m/z 50–1550 with a scan time of 1.92 s and an interscan of 0.08 s and a dynamic exclusion window of 240 s. Argon was used as collision gas. All MS data from the Q-TOF were processed using Mascot Distiller (v2.0) and the resulting peak lists (in mgf files) were searched again against ATH1v6 database concatenated with a decoy where all the sequences were in reversed orientation using Mascot with a significance threshold of 0.01. For each of the peak lists two searches were performed: 1) tryptic search and 2) a semi-tryptic search with methionine oxidation set as a variable modification and carboxymethylation of cysteins as fixed modification and the maximum mass error tolerance for precursor and products ions at respectively 1.2 and 0.8 Da. Using in-house written filter the results both searches were combined in an excel spreadsheet. Only matches from tryptic search were considered in protein identifications. The Mascot output was automatically processed by in-house software (Qi Sun, B. Zybailov and K.J. Van Wijk; unpublished).</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>The LTQ-Obtrap was interfaced with an LC system (MS surveyor pumps from Thermoelectron) and an MicroAS autosampler (from Thermoelectron). Analysis by nanoLC-LTQ-Orbitrap was as follows: Used automated sample pickups, 6.4 µl peptide extracts were loaded at 20 µl.min<sup>−1</sup> for 6 min on a guard column (LC Packings; MGU-30-C18PM), followed by elution and separation on a PepMap C18 reverse-phase nano column (LC Packings nan75-15-03-C18PM), using 90-min gradients with 95% water, 5% acetonitrile (ACN), 0.1% FA (solvent A), and 95% ACN, 5% water, 0.1% FA (solvent B) at a flow rate of 200 nl/min. A precolumn splitter was used to reduce the flow rate of 25 µl.min<sup>−1</sup> to 200 nl.min<sup>−1</sup>. Each sample injection and analysis was followed by two blank injections using 60-min and 45-min gradients to prevent carry over from sample to sample. The acquisition cycle consisted of a survey FTMS scan at the highest resolving power (100,000), followed by 5 data-dependent MS/MS scans acquired in the LTQ. Dynamic exclusion was used with the following parameters: exclusion size 500, repeat count 2, repeat duration 30 s, exclusion time 180 s, exclusion window ±6 ppm. Target values were set at 5×10<sup>5</sup> and 10<sup>4</sup> for the survey and Tandem MS scans, respectively, and the maximum ion accumulation times were set at 200 ms in both cases. Acquired MS/MS data were searched with Mascot 2.2 using a significant threshold of 0.01. Preliminary search was conducted with broad precursor tolerance window set at ±30 ppm. Peptides with the ion scores above 45 were chosen as benchmarks for off-line recalibration. Recalibration was performed using a Perl script which adjusted precursor masses in the peak lists. The recalibrated peak lists were searched again against ATH1v6 database concatenated with a decoy where all the sequences were in reversed orientation. For each of the peak lists three searches were performed: 1) <em>tryptic</em> search with precursor ion tolerance window set at ±6 ppm, 2) <em>error-tolerant</em> search with precursor ion tolerance window set at ±3 ppm, 3) <em>semi-tryptic</em> search with acetylation of peptide N-terminus set as a variable modification. For all of the three types of searches first and second <sup>13</sup>C peaks were considered as precursors without widening of the precursor ion tolerance, using the corresponding Mascot 2.2 feature. Additionally, in all of the three searches methionine oxidation was set as a variable modification. Using in-house written filter the results of the three searches were combined in an excel spreadsheet. The ion scores threshold were set to 33, which yielded final peptide false positive rate below 1%. Additionally, for proteins represented only by one unique peptide, mass accuracy on the precursor ion was required to be within ±3 ppm. Only matches from tryptic search were considered in protein identifications.</p>


<h4>Calculation of protein abundance</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>For each of the proteins identified in stromal preparations P1–P3, total number of spectral counts (SPC) and number of unique SPC was extracted from the Mascot 2.2 output using in-house written filters. In cases where two or more homologous proteins were identified with shared and unique peptides, the number of spectra from shared peptides assigned to each protein was determined based on the ratios of spectra derived from the unique peptides that identified each protein, similar to <a href="#pone.0001994-Liu2">[43]</a>, using formula:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.e001&amp;representation=PNG" class="inline-graphic"></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.e002&amp;representation=PNG" class="inline-graphic"></span> is the initial number of unique SPC for <em>i</em>-th protein the group, <em>AFi</em> is the abundance factor used for quantification, Shared_SPC – number of total shared SPC within the group, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.e003&amp;representation=PNG" class="inline-graphic"></span> is the sum of all unique SPC within the group. In cases of ambiguous groups (no unique SPC), as well as in the cases of protein subsets <em>AF</em> was set equal to <em>Shared_SPC</em>, with the whole ambiguous group treated as one protein.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>To obtain measure of protein abundance, normalization formula<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001994.e004&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>was used, with normalization factor, nF, derived from mature protein sequence (transit peptide(s) removed) being either length in number of aa residues or, alternatively, number of fully tryptic peptides in the mass range of 700 to 3500 Da.</p>


<h4>Statistical analysis of reproducibility by G-test</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>G-test was performed across three technical and across three biological replicates of chloroplast stroma preparations as described in <a href="#pone.0001994-Old1">[17]</a>, <a href="#pone.0001994-Zhang1">[40]</a> and in <a href="#pone.0001994-Sokal1">[39]</a>. Briefly, for each protein across <em>m</em> conditions tested, the G-statistic is calculated and then compared to the Chi-square distribution with <em>m</em>−1 degrees of freedom, resulting in a p-value. Finally, a correction for multiple testing is applied to the p-values, and proteins with significant changes in abundance are determined. The higher the number of proteins with significant changes, the poorer is reproducibility.</p>


<h4>Analysis of transit peptides and cleavage sites</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Sequence logos were created using the web server at <a href="http://weblogo.berkeley.edu/">http://weblogo.berkeley.edu/</a>. The plots of binned frequencies were obtained by counting the aa frequencies at the two N-terminal positions and 5 positions (2 in cTP, 3 in mature protein) around the predicted cTP cleavage site separately (i.e., one bin for each of these positions), and cutting up the remaining internal part of the cTP in 20 bins. In total 27 bins. This is a simplistic way to investigate regional properties of all cTPs regardless of their length differences. Multiple sequence alignments of the cTPs using Clustalw with standard scoring matrices and parameter settings were uninformative.</p>


<h4>PPDB structure and content</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The Mascot output files were automatically processed by in-house software (Qi Sun, B. Zybailov and K.J. Van Wijk, unpublished) and a number of output parameters (accession number, instrument type, experimental ambiguity, Mowse score, number of matching peptides, number of matched MS/MS spectra (queries), number of unique queries, highest peptide score, highest peptide error (in ppm), lowest absolute error (ppm), sequence coverage, tryptic, semi-tryptic peptide and modified peptide sequences) were uploaded into the PPDB.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>The construction of the PPDB (<a href="http://ppdb.tc.cornell.edu/">http://ppdb.tc.cornell.edu/</a>) was originally described in <a href="#pone.0001994-Friso1">[36]</a>. Since its inception in 2004, the PPDB interface was improved and we renamed the data base from Plastid Proteome DB to Plant Proteome DB to better reflect the content. The ‘backbone’ of the PPDB are all protein-encoding accessions (currently release 6.0 of ATH1.pep) with a theoretical analysis (predicted localization and physical properties of precursor and processed proteins) of all <em>Arabidopsis</em> entries. Also, more detailed curated information about the MS based identification can now be accessed; this will allow the user to better evaluate the strength of protein identification. Mascot and ion scores, mass accuracies, matched aa sequences, number of matching peptides and highest peptide score for each identification and other MS based data are listed. The MapMan Bin system <a href="#pone.0001994-Thimm1">[45]</a> is used for functional assignment and all assignments for identified proteins were verified manually.</p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p>To determine potential non-chloroplast contaminants in this filtered dataset, we used an extensive cross-correlation to more than fifty published proteomics papers on <em>Arabidopsis</em> subcellular fractions, as well as information extracted from TAIR, from SUBA <a href="#pone.0001994-Heazlewood2">[53]</a>, and other literature. The complete list of published proteomics papers and each of their accessions can be downloaded from PPDB (<a href="http://ppdb.tc.cornell.edu/searchpub.aspx">http://ppdb.tc.cornell.edu/searchpub.asp​x</a>). As a rule of thumb, subcellular localization by GFP/YFP and western blots was considered strong evidence, although we noted that there are several examples of incorrect subcellular localization assignment based on GFP/YFP, as discussed in <a href="#pone.0001994-Heazlewood1">[12]</a>. Identification in published proteomics studies was sometimes difficult to judge since information about the ‘strength’ of MS based identification was not easily accessible.</p>
<a id="article1.body1.sec4.sec6.p4" name="article1.body1.sec4.sec6.p4"></a><p>Chloroplast proteins are among the most abundant cellular proteins, and contaminations from the chloroplast are often observed in proteome analyses of other organelles. However, if proteins with lower abundance ranks were identified in multiple non-plastid proteomics studies from the same subcellular fraction (<em>e.g</em>. cell wall or plasma membrane), they were considered contaminants. To avoid circular arguments when evaluating TargetP (below), predicted chloroplast localization was not considered in assigning proteins to the chloroplast.</p>
<a id="article1.body1.sec4.sec6.p5" name="article1.body1.sec4.sec6.p5"></a><p>In addition, all protein accessions can be directly cross-referenced against identified proteins from more than 50 published proteomics papers (from <em>A. thaliana</em> and other members of the <em>Brassicae</em> family), as well as the subcellular localization in GO (experimental) from TAIR, localization data from SUBA (<a href="http://www.suba.bcs.uwa.edu.au">http://www.suba.bcs.uwa.edu.au</a>) and best matching homologues in maize and rice; Many of these data types can be directly exported from the PPDB as excel files.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001994"></div><a name="pone.0001994.s001" id="pone.0001994.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001994.s001">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Analysis of technical and biological variation.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001994.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001994.s002" id="pone.0001994.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001994.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Chloroplast proteome analysis by LTQ, identifying 1258 non-ambiguous proteins and 22 ambiguous protein groups.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001994.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.33 MB XLS)</p>
<a name="pone.0001994.s003" id="pone.0001994.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001994.s003">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Abundance information of 946 stroma-enriched proteins determined by mass spectrometry and correlation to gel-based quantification.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001994.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.40 MB XLS)</p>
<a name="pone.0001994.s004" id="pone.0001994.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001994.s004">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Analysis by Q-TOF of seven independent stromal preparations identifying 623 non-redundant proteins (counting only 1 gene model) with 1 or more unique peptides.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001994.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.10 MB XLS)</p>
<a name="pone.0001994.s005" id="pone.0001994.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001994.s005">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Identified and annotated proteins from filtered experimental MS/MS experiments and additional 200 annotated chloroplast proteins from literature.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001994.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.26 MB XLS)</p>
<a name="pone.0001994.s006" id="pone.0001994.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001994.s006">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Sequences and mass spectrometry details of N-terminal acetylated nuclear-encoded proteins.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001994.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001994.s007" id="pone.0001994.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001994.s007">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Assessing quality of sub-organelle enrichment by comparing protein levels in Stroma and Thylakoids. (A) Top 27 proteins over-represented in stroma protein preparations. (B) Top 27 proteins over-represented in thylakoid preparat</p>
<p class="siDoi">doi:10.1371/journal.pone.0001994.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.57 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We acknowledge Zhang Bing and Nagiza Samatova for general advice on the G-test and Paul Dominic B. Olinares and Wojciech Majeran for comments on the manuscript and discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: Kv BZ. Performed the experiments: BZ HR GF AR. Analyzed the data: QS OE Kv BZ. Contributed reagents/materials/analysis tools: QS. Wrote the paper: OE Kv BZ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001994-vonHeijne1" id="pone.0001994-vonHeijne1"></a>von Heijne G, Steppuhn J, Hermann SG (1989) Domain structure of mitochondrial and chloroplast targeting peptides. Eur J Biochemistry  80: 535–545.  <ul class="find" data-citedArticleID="1091548" data-doi="10.1111/j.1432-1033.1989.tb14679.x"><li><a href="http://dx.doi.org/10.1111/j.1432-1033.1989.tb14679.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Domain+structure+of+mitochondrial+and+chloroplast+targeting+peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Domain+structure+of+mitochondrial+and+chloroplast+targeting+peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001994-Emanuelsson1" id="pone.0001994-Emanuelsson1"></a>Emanuelsson O, Brunak S, von Heijne G, Nielsen H (2007) Locating proteins in the cell using TargetP, SignalP and related tools. Nat Protoc  2(4): 953–971.  <ul class="find" data-citedArticleID="1091416" data-doi="10.1038/nprot.2007.131"><li><a href="http://dx.doi.org/10.1038/nprot.2007.131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Locating+proteins+in+the+cell+using+TargetP%2C+SignalP+and+related+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Locating+proteins+in+the+cell+using+TargetP%2C+SignalP+and+related+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001994-Richly1" id="pone.0001994-Richly1"></a>Richly E, Leister D (2004) An improved prediction of chloroplast proteins reveals diversities and commonalities in the chloroplast proteomes of Arabidopsis and rice. Gene  329: 11–16.  <ul class="find" data-citedArticleID="1091498" data-doi="10.1016/j.gene.2004.01.008"><li><a href="http://dx.doi.org/10.1016/j.gene.2004.01.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+improved+prediction+of+chloroplast+proteins+reveals+diversities+and+commonalities+in+the+chloroplast+proteomes+of+Arabidopsis+and+rice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+improved+prediction+of+chloroplast+proteins+reveals+diversities+and+commonalities+in+the+chloroplast+proteomes+of+Arabidopsis+and+rice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001994-Sun1" id="pone.0001994-Sun1"></a>Sun Q, Emanuelsson O, van Wijk KJ (2004) Analysis of curated and predicted plastid subproteomes of Arabidopsis. Subcellular compartmentalization leads to distinctive proteome properties. Plant Physiol  135(2): 723–734.  <ul class="find" data-citedArticleID="1091522" data-doi="10.1104/pp.104.040717"><li><a href="http://dx.doi.org/10.1104/pp.104.040717" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+curated+and+predicted+plastid+subproteomes+of+Arabidopsis.+Subcellular+compartmentalization+leads+to+distinctive+proteome+properties." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+curated+and+predicted+plastid+subproteomes+of+Arabidopsis.+Subcellular+compartmentalization+leads+to+distinctive+proteome+properties.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001994-vanWijk1" id="pone.0001994-vanWijk1"></a>van Wijk KJ (2004) Plastid proteomics. Plant Physiol Biochem  42(12): 963–77.  <ul class="find" data-citedArticleID="1091542" data-doi="10.1016/j.plaphy.2004.10.015"><li><a href="http://dx.doi.org/10.1016/j.plaphy.2004.10.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plastid+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plastid+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001994-Baginsky1" id="pone.0001994-Baginsky1"></a>Baginsky S, Gruissem W (2004) Chloroplast proteomics: potentials and challenges. J Exp Bot  55(400): 1213–1220.  <ul class="find" data-citedArticleID="1091394" data-doi="10.1093/jxb/erh104"><li><a href="http://dx.doi.org/10.1093/jxb/erh104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chloroplast+proteomics%3A+potentials+and+challenges." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chloroplast+proteomics%3A+potentials+and+challenges.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001994-Millar1" id="pone.0001994-Millar1"></a>Millar AH, Whelan J, Small I (2006) Recent surprises in protein targeting to mitochondria and plastids. Curr Opin Plant Biol  9(6): 610–615.  <ul class="find" data-citedArticleID="1091466" data-doi="10.1016/j.pbi.2006.09.002"><li><a href="http://dx.doi.org/10.1016/j.pbi.2006.09.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+surprises+in+protein+targeting+to+mitochondria+and+plastids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+surprises+in+protein+targeting+to+mitochondria+and+plastids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001994-Villarejo1" id="pone.0001994-Villarejo1"></a>Villarejo A, Buren S, Larsson S, Dejardin A, Monne M, Rudhe C, Karlsson J, Jansson S, Lerouge P, Rolland N, von Heijne G, Grebe M, Bako L, Samuelsson G (2005) Evidence for a protein transported through the secretory pathway en route to the higher plant chloroplast. Nat Cell Biol  (12): 1224–1231.  <ul class="find" data-citedArticleID="1091526" data-doi="10.1038/ncb1330"><li><a href="http://dx.doi.org/10.1038/ncb1330" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+a+protein+transported+through+the+secretory+pathway+en+route+to+the+higher+plant+chloroplast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+a+protein+transported+through+the+secretory+pathway+en+route+to+the+higher+plant+chloroplast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001994-vanWijk2" id="pone.0001994-vanWijk2"></a>van Wijk KJ (2007) Plant Proteomics and Photosynthesis (Chapter 8), in Photosynthesis: A comprehensive Treatise Biochemistry, Biophysics and Molecular Biology, J.A.T. Eaton-Rye, B., Editor Kluwer.   <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0001994-Baginsky2" id="pone.0001994-Baginsky2"></a>Baginsky S, Gruissem W (2006) Arabidopsis thaliana proteomics: from proteome to genome. J Exp Bot  57(7): 1485–1491.  <ul class="find" data-citedArticleID="1091396" data-doi="10.1093/jxb/erj130"><li><a href="http://dx.doi.org/10.1093/jxb/erj130" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+thaliana+proteomics%3A+from+proteome+to+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+thaliana+proteomics%3A+from+proteome+to+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001994-Rolland1" id="pone.0001994-Rolland1"></a>Rolland N, Ferro M, Ephritikhine G, Marmagne A, Ramus C, Brugiere S, Salvi D, Seigneurin-Berny D, Bourguignon J, Barbier-Brygoo H, Joyard J, Garin J (2006) A versatile method for deciphering plant membrane proteomes. J Exp Bot  57(7): 1579–1589.  <ul class="find" data-citedArticleID="1091500"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+versatile+method+for+deciphering+plant+membrane+proteomes.&amp;auth=&amp;atitle=A+versatile+method+for+deciphering+plant+membrane+proteomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+versatile+method+for+deciphering+plant+membrane+proteomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+versatile+method+for+deciphering+plant+membrane+proteomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001994-Heazlewood1" id="pone.0001994-Heazlewood1"></a>Heazlewood JL, Tonti-Filippini J, Verboom RE, Millar AH (2005) Combining experimental and predicted datasets for determination of the subcellular location of proteins in Arabidopsis. Plant Physiol  139(2): 598–609.  <ul class="find" data-citedArticleID="1091432" data-doi="10.1104/pp.105.065532"><li><a href="http://dx.doi.org/10.1104/pp.105.065532" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combining+experimental+and+predicted+datasets+for+determination+of+the+subcellular+location+of+proteins+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combining+experimental+and+predicted+datasets+for+determination+of+the+subcellular+location+of+proteins+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001994-Glinski1" id="pone.0001994-Glinski1"></a>Glinski M, Weckwerth W (2006) The role of mass spectrometry in plant systems biology. Mass Spectrom Rev  25(2): 173–214.  <ul class="find" data-citedArticleID="1091430" data-doi="10.1002/mas.20063"><li><a href="http://dx.doi.org/10.1002/mas.20063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+mass+spectrometry+in+plant+systems+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+mass+spectrometry+in+plant+systems+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001994-Peltier1" id="pone.0001994-Peltier1"></a>Peltier JB, Cai Y, Sun Q, Zabrouskov V, Giacomelli L, Rudella A, Ytterberg AJ, Rutschow H, van Wijk KJ (2006) The Oligomeric Stromal Proteome of Arabidopsis thaliana Chloroplasts. Mol Cell Proteomics  5(1): 114–133.  <ul class="find" data-citedArticleID="1091480"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Oligomeric+Stromal+Proteome+of+Arabidopsis+thaliana+Chloroplasts.&amp;auth=&amp;atitle=The+Oligomeric+Stromal+Proteome+of+Arabidopsis+thaliana+Chloroplasts." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Oligomeric+Stromal+Proteome+of+Arabidopsis+thaliana+Chloroplasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Oligomeric+Stromal+Proteome+of+Arabidopsis+thaliana+Chloroplasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001994-Liu1" id="pone.0001994-Liu1"></a>Liu H, Sadygov RG, Yates JR 3rd (2004) A model for random sampling and estimation of relative protein abundance in shotgun proteomics. Anal Chem  76(14): 4193–4201.  <ul class="find" data-citedArticleID="1091450" data-doi="10.1021/ac0498563"><li><a href="http://dx.doi.org/10.1021/ac0498563" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+model+for+random+sampling+and+estimation+of+relative+protein+abundance+in+shotgun+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+model+for+random+sampling+and+estimation+of+relative+protein+abundance+in+shotgun+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001994-Zybailov1" id="pone.0001994-Zybailov1"></a>Zybailov B, Coleman MK, Florens L, Washburn MP (2005) Correlation of relative abundance ratios derived from peptide ion chromatograms and spectrum counting for quantitative proteomic analysis using stable isotope labeling. Anal Chem  77(19): 6218–6224.  <ul class="find" data-citedArticleID="1091538" data-doi="10.1021/ac050846r"><li><a href="http://dx.doi.org/10.1021/ac050846r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlation+of+relative+abundance+ratios+derived+from+peptide+ion+chromatograms+and+spectrum+counting+for+quantitative+proteomic+analysis+using+stable+isotope+labeling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlation+of+relative+abundance+ratios+derived+from+peptide+ion+chromatograms+and+spectrum+counting+for+quantitative+proteomic+analysis+using+stable+isotope+labeling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001994-Old1" id="pone.0001994-Old1"></a>Old WM, Meyer-Arendt K, Aveline-Wolf L, Pierce KG, Mendoza A, Sevinsky JR, Resing KA, Ahn NG (2005) Comparison of label-free methods for quantifying human proteins by shotgun proteomics. Mol Cell Proteomics  4(10): 1487–1502.  <ul class="find" data-citedArticleID="1091474" data-doi="10.1074/mcp.m500084-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m500084-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+label-free+methods+for+quantifying+human+proteins+by+shotgun+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+label-free+methods+for+quantifying+human+proteins+by+shotgun+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001994-Lu1" id="pone.0001994-Lu1"></a>Lu P, Vogel C, Wang R, Yao X, Marcotte EM (2007) Absolute protein expression profiling estimates the relative contributions of transcriptional and translational regulation. Nat Biotechnol  25(1): 117–124.  <ul class="find" data-citedArticleID="1091454" data-doi="10.1038/nbt1270"><li><a href="http://dx.doi.org/10.1038/nbt1270" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Absolute+protein+expression+profiling+estimates+the+relative+contributions+of+transcriptional+and+translational+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Absolute+protein+expression+profiling+estimates+the+relative+contributions+of+transcriptional+and+translational+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001994-Listgarten1" id="pone.0001994-Listgarten1"></a>Listgarten J, Emili A (2005) Statistical and computational methods for comparative proteomic profiling using liquid chromatography-tandem mass spectrometry. Mol Cell Proteomics  4(4): 419–434.  <ul class="find" data-citedArticleID="1091448" data-doi="10.1074/mcp.r500005-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.r500005-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+and+computational+methods+for+comparative+proteomic+profiling+using+liquid+chromatography-tandem+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+and+computational+methods+for+comparative+proteomic+profiling+using+liquid+chromatography-tandem+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001994-Nesvizhskii1" id="pone.0001994-Nesvizhskii1"></a>Nesvizhskii AI, Vitek O, Aebersold R (2007) Analysis and validation of proteomic data generated by tandem mass spectrometry. Nat Methods  4(10): 787–797.  <ul class="find" data-citedArticleID="1091470" data-doi="10.1038/nmeth1088"><li><a href="http://dx.doi.org/10.1038/nmeth1088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+and+validation+of+proteomic+data+generated+by+tandem+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+and+validation+of+proteomic+data+generated+by+tandem+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001994-Bantscheff1" id="pone.0001994-Bantscheff1"></a>Bantscheff M, Schirle M, Sweetman G, Rick J, Kuster B (2007) Quantitative mass spectrometry in proteomics: a critical review. Anal Bioanal Chem  389(4): 1017–1031.  <ul class="find" data-citedArticleID="1091400" data-doi="10.1007/s00216-007-1486-6"><li><a href="http://dx.doi.org/10.1007/s00216-007-1486-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+mass+spectrometry+in+proteomics%3A+a+critical+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+mass+spectrometry+in+proteomics%3A+a+critical+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001994-Pesaresi1" id="pone.0001994-Pesaresi1"></a>Pesaresi P, Gardner NA, Masiero S, Dietzmann A, Eichacker L, Wickner R, Salamini F, Leister D (2003) Cytoplasmic N-terminal protein acetylation is required for efficient photosynthesis in Arabidopsis. Plant Cell  15(8): 1817–1832.  <ul class="find" data-citedArticleID="1091490" data-doi="10.1105/tpc.012377"><li><a href="http://dx.doi.org/10.1105/tpc.012377" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytoplasmic+N-terminal+protein+acetylation+is+required+for+efficient+photosynthesis+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytoplasmic+N-terminal+protein+acetylation+is+required+for+efficient+photosynthesis+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001994-Giglione1" id="pone.0001994-Giglione1"></a>Giglione C, Serero A, Pierre M, Boisson B, Meinnel T (2000) Identification of eukaryotic peptide deformylases reveals universality of N-terminal protein processing mechanisms. Embo J  19(21): 5916–5929.  <ul class="find" data-citedArticleID="1091424" data-doi="10.1093/emboj/19.21.5916"><li><a href="http://dx.doi.org/10.1093/emboj/19.21.5916" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+eukaryotic+peptide+deformylases+reveals+universality+of+N-terminal+protein+processing+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+eukaryotic+peptide+deformylases+reveals+universality+of+N-terminal+protein+processing+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001994-Dirk1" id="pone.0001994-Dirk1"></a>Dirk LM, Williams MA, Houtz RL (2002) Specificity of chloroplast-localized peptide deformylases as determined with peptide analogs of chloroplast-translated proteins. Arch Biochem Biophys  406(1): 135–141.  <ul class="find" data-citedArticleID="1091410" data-doi="10.1016/s0003-9861(02)00426-5"><li><a href="http://dx.doi.org/10.1016/s0003-9861(02)00426-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specificity+of+chloroplast-localized+peptide+deformylases+as+determined+with+peptide+analogs+of+chloroplast-translated+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specificity+of+chloroplast-localized+peptide+deformylases+as+determined+with+peptide+analogs+of+chloroplast-translated+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001994-Dirk2" id="pone.0001994-Dirk2"></a>Dirk LM, Williams MA, Houtz RL (2001) Eukaryotic peptide deformylases. Nuclear-encoded and chloroplast-targeted enzymes in Arabidopsis. Plant Physiol  127(1): 97–107.  <ul class="find" data-citedArticleID="1091412" data-doi="10.1104/pp.127.1.97"><li><a href="http://dx.doi.org/10.1104/pp.127.1.97" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eukaryotic+peptide+deformylases.+Nuclear-encoded+and+chloroplast-targeted+enzymes+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eukaryotic+peptide+deformylases.+Nuclear-encoded+and+chloroplast-targeted+enzymes+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001994-Giglione2" id="pone.0001994-Giglione2"></a>Giglione C, Meinnel T (2001) Organellar peptide deformylases: universality of the N-terminal methionine cleavage mechanism. Trends Plant Sci  6(12): 566–572.  <ul class="find" data-citedArticleID="1091426" data-doi="10.1016/s1360-1385(01)02151-3"><li><a href="http://dx.doi.org/10.1016/s1360-1385(01)02151-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Organellar+peptide+deformylases%3A+universality+of+the+N-terminal+methionine+cleavage+mechanism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Organellar+peptide+deformylases%3A+universality+of+the+N-terminal+methionine+cleavage+mechanism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001994-Giglione3" id="pone.0001994-Giglione3"></a>Giglione C, Vallon O, Meinnel T (2003) Control of protein life-span by N-terminal methionine excision. Embo J  22(1): 13–23.  <ul class="find" data-citedArticleID="1091428" data-doi="10.1093/emboj/cdg007"><li><a href="http://dx.doi.org/10.1093/emboj/cdg007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+protein+life-span+by+N-terminal+methionine+excision." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+protein+life-span+by+N-terminal+methionine+excision.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001994-Ross1" id="pone.0001994-Ross1"></a>Ross S, Giglione C, Pierre M, Espagne C, Meinnel T (2005) Functional and developmental impact of cytosolic protein N-terminal methionine excision in Arabidopsis. Plant Physiol  137(2): 623–637.  <ul class="find" data-citedArticleID="1091504" data-doi="10.1104/pp.104.056861"><li><a href="http://dx.doi.org/10.1104/pp.104.056861" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+and+developmental+impact+of+cytosolic+protein+N-terminal+methionine+excision+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+and+developmental+impact+of+cytosolic+protein+N-terminal+methionine+excision+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001994-Meinnel1" id="pone.0001994-Meinnel1"></a>Meinnel T, Serero A, Giglione C (2006) Impact of the N-terminal amino acid on targeted protein degradation. Biol Chem  387(7): 839–851.  <ul class="find" data-citedArticleID="1091458" data-doi="10.1515/bc.2006.107"><li><a href="http://dx.doi.org/10.1515/bc.2006.107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impact+of+the+N-terminal+amino+acid+on+targeted+protein+degradation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impact+of+the+N-terminal+amino+acid+on+targeted+protein+degradation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001994-Adam1" id="pone.0001994-Adam1"></a>Adam Z, Rudella A, van Wijk KJ (2006) Recent advances in the study of Clp, FtsH and other proteases located in chloroplasts. Curr Opin Plant Biol  9(3): 234–240.  <ul class="find" data-citedArticleID="1091390" data-doi="10.1016/j.pbi.2006.03.010"><li><a href="http://dx.doi.org/10.1016/j.pbi.2006.03.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+advances+in+the+study+of+Clp%2C+FtsH+and+other+proteases+located+in+chloroplasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+advances+in+the+study+of+Clp%2C+FtsH+and+other+proteases+located+in+chloroplasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001994-Jensen1" id="pone.0001994-Jensen1"></a>Jensen ON (2006) Interpreting the protein language using proteomics. Nat Rev Mol Cell Biol  7(6): 391–403.  <ul class="find" data-citedArticleID="1091444" data-doi="10.1038/nrm1939"><li><a href="http://dx.doi.org/10.1038/nrm1939" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interpreting+the+protein+language+using+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interpreting+the+protein+language+using+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001994-Wisniewski1" id="pone.0001994-Wisniewski1"></a>Wisniewski JR, Zougman A, Kruger S, Mann M (2007) Mass Spectrometric Mapping of Linker Histone H1 Variants Reveals Multiple Acetylations, Methylations, and Phosphorylation as Well as Differences between Cell Culture and Tissue. Mol Cell Proteomics  6(1): 72–87.  <ul class="find" data-citedArticleID="1091532" data-doi="10.1074/mcp.m600255-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m600255-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mass+Spectrometric+Mapping+of+Linker+Histone+H1+Variants+Reveals+Multiple+Acetylations%2C+Methylations%2C+and+Phosphorylation+as+Well+as+Differences+between+Cell+Culture+and+Tissue." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mass+Spectrometric+Mapping+of+Linker+Histone+H1+Variants+Reveals+Multiple+Acetylations%2C+Methylations%2C+and+Phosphorylation+as+Well+as+Differences+between+Cell+Culture+and+Tissue.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001994-Scigelova1" id="pone.0001994-Scigelova1"></a>Scigelova M, Makarov A (2006) Orbitrap mass analyzer - overview and applications in proteomics. Proteomics  6: Suppl 216–21.  <ul class="find" data-citedArticleID="1091510" data-doi="10.1002/pmic.200600528"><li><a href="http://dx.doi.org/10.1002/pmic.200600528" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Orbitrap+mass+analyzer+-+overview+and+applications+in+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Orbitrap+mass+analyzer+-+overview+and+applications+in+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001994-Olsen1" id="pone.0001994-Olsen1"></a>Olsen JV, de Godoy LM, Li G, Macek B, Mortensen P, Pesch R, Makarov A, Lange O, Horning S, Mann M (2005) Parts per million mass accuracy on an Orbitrap mass spectrometer via lock mass injection into a C-trap. Mol Cell Proteomics  4(12): 2010–2021.  <ul class="find" data-citedArticleID="1091476" data-doi="10.1074/mcp.t500030-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.t500030-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parts+per+million+mass+accuracy+on+an+Orbitrap+mass+spectrometer+via+lock+mass+injection+into+a+C-trap." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parts+per+million+mass+accuracy+on+an+Orbitrap+mass+spectrometer+via+lock+mass+injection+into+a+C-trap.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001994-Olsen2" id="pone.0001994-Olsen2"></a>Olsen JV, Macek B, Lange O, Makarov A, Horning S, Mann M (2007) Higher-energy C-trap dissociation for peptide modification analysis. Nat Methods  4(9): 709–712.  <ul class="find" data-citedArticleID="1091478" data-doi="10.1038/nmeth1060"><li><a href="http://dx.doi.org/10.1038/nmeth1060" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Higher-energy+C-trap+dissociation+for+peptide+modification+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Higher-energy+C-trap+dissociation+for+peptide+modification+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001994-Friso1" id="pone.0001994-Friso1"></a>Friso G, Giacomelli L, Ytterberg AJ, Peltier JB, Rudella A, Sun Q, Wijk KJ (2004) In-depth analysis of the thylakoid membrane proteome of Arabidopsis thaliana chloroplasts: new proteins, new functions, and a plastid proteome database. Plant Cell  16(2): 478–499.  <ul class="find" data-citedArticleID="1091422" data-doi="10.1105/tpc.017814"><li><a href="http://dx.doi.org/10.1105/tpc.017814" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In-depth+analysis+of+the+thylakoid+membrane+proteome+of+Arabidopsis+thaliana+chloroplasts%3A+new+proteins%2C+new+functions%2C+and+a+plastid+proteome+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In-depth+analysis+of+the+thylakoid+membrane+proteome+of+Arabidopsis+thaliana+chloroplasts%3A+new+proteins%2C+new+functions%2C+and+a+plastid+proteome+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001994-Hennig1" id="pone.0001994-Hennig1"></a>Hennig L (2007) Patterns of beauty–omics meets plant development. Trends Plant Sci  12(7): 287–293.  <ul class="find" data-citedArticleID="1091438" data-doi="10.1016/j.tplants.2007.05.002"><li><a href="http://dx.doi.org/10.1016/j.tplants.2007.05.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+beauty%E2%80%93omics+meets+plant+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+beauty%E2%80%93omics+meets+plant+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001994-Peng1" id="pone.0001994-Peng1"></a>Peng J, Elias JE, Thoreen CC, Licklider LJ, Gygi SP (2003) Evaluation of multidimensional chromatography coupled with tandem mass spectrometry (LC/LC-MS/MS) for large-scale protein analysis: the yeast proteome. J Proteome Res  2(1): 43–50.  <ul class="find" data-citedArticleID="1091488" data-doi="10.1021/pr025556v"><li><a href="http://dx.doi.org/10.1021/pr025556v" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+multidimensional+chromatography+coupled+with+tandem+mass+spectrometry+%28LC%2FLC-MS%2FMS%29+for+large-scale+protein+analysis%3A+the+yeast+proteome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+multidimensional+chromatography+coupled+with+tandem+mass+spectrometry+%28LC%2FLC-MS%2FMS%29+for+large-scale+protein+analysis%3A+the+yeast+proteome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001994-Sokal1" id="pone.0001994-Sokal1"></a>Sokal R, Rohlf FJJ (2005) Chapter 5 in Biometry. New York: W.H. FREEMAN and COMPANY.   <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0001994-Zhang1" id="pone.0001994-Zhang1"></a>Zhang B, VerBerkmoes NC, Langston MA, Uberbacher E, Hettich RL, Samatova NF (2006) Detecting differential and correlated protein expression in label-free shotgun proteomics. J Proteome Res  5(11): 2909–2918.  <ul class="find" data-citedArticleID="1091536" data-doi="10.1021/pr0600273"><li><a href="http://dx.doi.org/10.1021/pr0600273" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+differential+and+correlated+protein+expression+in+label-free+shotgun+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+differential+and+correlated+protein+expression+in+label-free+shotgun+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001994-Zybailov2" id="pone.0001994-Zybailov2"></a>Zybailov B, Mosley AL, Sardiu ME, Coleman MK, Florens L, Washburn MP (2006) Statistical analysis of membrane proteome expression changes in Saccharomyces cerevisiae. J Proteome Res  5(9): 2339–2347.  <ul class="find" data-citedArticleID="1091540" data-doi="10.1021/pr060161n"><li><a href="http://dx.doi.org/10.1021/pr060161n" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+analysis+of+membrane+proteome+expression+changes+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+analysis+of+membrane+proteome+expression+changes+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001994-Mallick1" id="pone.0001994-Mallick1"></a>Mallick P, Schirle M, Chen SS, Flory MR, Lee H, Martin D, Ranish J, Raught B, Schmitt R, Werner T, Kuster B, Aebersold R (2007) Computational prediction of proteotypic peptides for quantitative proteomics. Nat Biotechnol  25(1): 125–131.  <ul class="find" data-citedArticleID="1091456" data-doi="10.1038/nbt1275"><li><a href="http://dx.doi.org/10.1038/nbt1275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+prediction+of+proteotypic+peptides+for+quantitative+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+prediction+of+proteotypic+peptides+for+quantitative+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001994-Liu2" id="pone.0001994-Liu2"></a>Liu Q, Tan G, Levenkova N, Li T, Pugh EN Jr, Rux JJ, Speicher DW, Pierce EA (2007) The proteome of the mouse photoreceptor sensory cilium complex. Mol Cell Proteomics  6(8): 1299–1317.  <ul class="find" data-citedArticleID="1091452" data-doi="10.1074/mcp.m700054-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m700054-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+proteome+of+the+mouse+photoreceptor+sensory+cilium+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+proteome+of+the+mouse+photoreceptor+sensory+cilium+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001994-Mendenhall1" id="pone.0001994-Mendenhall1"></a>Mendenhall WaST (1995) Statistics for engineering and the sciences’. 4th edition ed.   Prentice-Hall, Inc.  <ul class="find-nolinks"></ul></li><li><span class="label">45.
              </span><a name="pone.0001994-Thimm1" id="pone.0001994-Thimm1"></a>Thimm O, Blasing O, Gibon Y, Nagel A, Meyer S, Kruger P, Selbig J, Muller LA, Rhee SY, Stitt M (2004) MAPMAN: a user-driven tool to display genomics data sets onto diagrams of metabolic pathways and other biological processes. Plant J  37(6): 914–939.  <ul class="find" data-citedArticleID="1091524" data-doi="10.1111/j.1365-313x.2004.02016.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2004.02016.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAPMAN%3A+a+user-driven+tool+to+display+genomics+data+sets+onto+diagrams+of+metabolic+pathways+and+other+biological+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAPMAN%3A+a+user-driven+tool+to+display+genomics+data+sets+onto+diagrams+of+metabolic+pathways+and+other+biological+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001994-Stahl1" id="pone.0001994-Stahl1"></a>Stahl A, Nilsson S, Lundberg P, Bhushan S, Biverstahl H, Moberg P, Morisset M, Vener A, Maler L, Langel U, Glaser E (2005) Two novel targeting peptide degrading proteases, PrePs, in mitochondria and chloroplasts, so similar and still different. J Mol Biol  349(4): 847–60.  <ul class="find" data-citedArticleID="1091518" data-doi="10.1016/j.jmb.2005.04.023"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.04.023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+novel+targeting+peptide+degrading+proteases%2C+PrePs%2C+in+mitochondria+and+chloroplasts%2C+so+similar+and+still+different." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+novel+targeting+peptide+degrading+proteases%2C+PrePs%2C+in+mitochondria+and+chloroplasts%2C+so+similar+and+still+different.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001994-Jain1" id="pone.0001994-Jain1"></a>Jain R, Chan MK (2007) Support for a potential role of E. coli oligopeptidase A in protein degradation. Biochem Biophys Res Commun  359(3): 486–490.  <ul class="find" data-citedArticleID="1091442" data-doi="10.1016/j.bbrc.2007.05.142"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2007.05.142" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Support+for+a+potential+role+of+E.+coli+oligopeptidase+A+in+protein+degradation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Support+for+a+potential+role+of+E.+coli+oligopeptidase+A+in+protein+degradation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001994-Book1" id="pone.0001994-Book1"></a>Book AJ, Yang P, Scalf M, Smith LM, Vierstra RD (2005) Tripeptidyl peptidase II. An oligomeric protease complex from Arabidopsis. Plant Physiol  138(2): 1046–1057.  <ul class="find" data-citedArticleID="1091406" data-doi="10.1104/pp.104.057406"><li><a href="http://dx.doi.org/10.1104/pp.104.057406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tripeptidyl+peptidase+II.+An+oligomeric+protease+complex+from+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tripeptidyl+peptidase+II.+An+oligomeric+protease+complex+from+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001994-Peltier2" id="pone.0001994-Peltier2"></a>Peltier JB, Ripoll DR, Friso G, Rudella A, Cai Y, Ytterberg J, Giacomelli L, Pillardy J, Van Wijk KJ (2004) Clp Protease Complexes from Photosynthetic and Non-photosynthetic Plastids and Mitochondria of Plants, Their Predicted Three-dimensional Structures, and Functional Implications. J Biol Chem  279(6): 4768–4781.  <ul class="find" data-citedArticleID="1091482" data-doi="10.1074/jbc.m309212200"><li><a href="http://dx.doi.org/10.1074/jbc.m309212200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clp+Protease+Complexes+from+Photosynthetic+and+Non-photosynthetic+Plastids+and+Mitochondria+of+Plants%2C+Their+Predicted+Three-dimensional+Structures%2C+and+Functional+Implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clp+Protease+Complexes+from+Photosynthetic+and+Non-photosynthetic+Plastids+and+Mitochondria+of+Plants%2C+Their+Predicted+Three-dimensional+Structures%2C+and+Functional+Implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001994-Sjogren1" id="pone.0001994-Sjogren1"></a>Sjogren LL, Stanne TM, Zheng B, Sutinen S, Clarke AK (2006) Structural and functional insights into the chloroplast ATP-dependent Clp protease in Arabidopsis. Plant Cell  18(10): 2635–2649.  <ul class="find" data-citedArticleID="1091514" data-doi="10.1105/tpc.106.044594"><li><a href="http://dx.doi.org/10.1105/tpc.106.044594" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+and+functional+insights+into+the+chloroplast+ATP-dependent+Clp+protease+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+and+functional+insights+into+the+chloroplast+ATP-dependent+Clp+protease+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001994-Ytterberg1" id="pone.0001994-Ytterberg1"></a>Ytterberg AJ, Peltier JB, van Wijk KJ (2006) Protein profiling of plastoglobules in chloroplasts and chromoplasts; a surprising site for differential accumulation of metabolic enzymes. Plant Physiol  140(3): 984–997.  <ul class="find" data-citedArticleID="1091534" data-doi="10.1104/pp.105.076083"><li><a href="http://dx.doi.org/10.1104/pp.105.076083" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+profiling+of+plastoglobules+in+chloroplasts+and+chromoplasts%3B+a+surprising+site+for+differential+accumulation+of+metabolic+enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+profiling+of+plastoglobules+in+chloroplasts+and+chromoplasts%3B+a+surprising+site+for+differential+accumulation+of+metabolic+enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001994-Peltier3" id="pone.0001994-Peltier3"></a>Peltier JB, Ytterberg AJ, Sun Q, van Wijk KJ (2004) New functions of the thylakoid membrane proteome of Arabidopsis thaliana revealed by a simple, fast, and versatile fractionation strategy. J Biol Chem  279(47): 49367–49383.  <ul class="find" data-citedArticleID="1091484" data-doi="10.1074/jbc.m406763200"><li><a href="http://dx.doi.org/10.1074/jbc.m406763200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+functions+of+the+thylakoid+membrane+proteome+of+Arabidopsis+thaliana+revealed+by+a+simple%2C+fast%2C+and+versatile+fractionation+strategy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+functions+of+the+thylakoid+membrane+proteome+of+Arabidopsis+thaliana+revealed+by+a+simple%2C+fast%2C+and+versatile+fractionation+strategy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001994-Heazlewood2" id="pone.0001994-Heazlewood2"></a>Heazlewood JL, Verboom RE, Tonti-Filippini J, Small I, Millar AH (2007) SUBA: the Arabidopsis Subcellular Database. Nucleic Acids Res  35: D213–218.  <ul class="find" data-citedArticleID="1091434" data-doi="10.1093/nar/gkl863"><li><a href="http://dx.doi.org/10.1093/nar/gkl863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SUBA%3A+the+Arabidopsis+Subcellular+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SUBA%3A+the+Arabidopsis+Subcellular+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001994-Michel1" id="pone.0001994-Michel1"></a>Michel H, Griffin PR, Shabanowitz J, Hunt DF, Bennett J (1991) Tandem mass spectrometry identifies sites of three post-translational modifications of spinach light-harvesting chlorophyll protein II. Proteolytic cleavage, acetylation, and phosphorylation. J Biol Chem  266(26): 17584–17591.  <ul class="find" data-citedArticleID="1091462"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Tandem+mass+spectrometry+identifies+sites+of+three+post-translational+modifications+of+spinach+light-harvesting+chlorophyll+protein+II.+Proteolytic+cleavage%2C+acetylation%2C+and+phosphorylation.&amp;auth=&amp;atitle=Tandem+mass+spectrometry+identifies+sites+of+three+post-translational+modifications+of+spinach+light-harvesting+chlorophyll+protein+II.+Proteolytic+cleavage%2C+acetylation%2C+and+phosphorylation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tandem+mass+spectrometry+identifies+sites+of+three+post-translational+modifications+of+spinach+light-harvesting+chlorophyll+protein+II.+Proteolytic+cleavage%2C+acetylation%2C+and+phosphorylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tandem+mass+spectrometry+identifies+sites+of+three+post-translational+modifications+of+spinach+light-harvesting+chlorophyll+protein+II.+Proteolytic+cleavage%2C+acetylation%2C+and+phosphorylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001994-Michel2" id="pone.0001994-Michel2"></a>Michel H, Hunt DF, Shabanowitz J, Bennett J (1988) Tandem mass spectrometry reveals that three photosystem II proteins of spinach chloroplasts contain N-acetyl-O-phosphothreonine at their NH2 termini. J Biol Chem  263(3): 1123–1130.  <ul class="find" data-citedArticleID="1091464"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Tandem+mass+spectrometry+reveals+that+three+photosystem+II+proteins+of+spinach+chloroplasts+contain+N-acetyl-O-phosphothreonine+at+their+NH2+termini.&amp;auth=&amp;atitle=Tandem+mass+spectrometry+reveals+that+three+photosystem+II+proteins+of+spinach+chloroplasts+contain+N-acetyl-O-phosphothreonine+at+their+NH2+termini." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tandem+mass+spectrometry+reveals+that+three+photosystem+II+proteins+of+spinach+chloroplasts+contain+N-acetyl-O-phosphothreonine+at+their+NH2+termini." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tandem+mass+spectrometry+reveals+that+three+photosystem+II+proteins+of+spinach+chloroplasts+contain+N-acetyl-O-phosphothreonine+at+their+NH2+termini.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001994-WangC1" id="pone.0001994-WangC1"></a>WangC B, Wang HX, Feng JX, Meng DZ, Qu LJ, Zhu YX (2006) Post-translational modifications, but not transcriptional regulation, of major chloroplast RNA-binding proteins are related to Arabidopsis seedling development. Proteomics  6(8): 2555–2563.  <ul class="find" data-citedArticleID="1091528" data-doi="10.1002/pmic.200500657"><li><a href="http://dx.doi.org/10.1002/pmic.200500657" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Post-translational+modifications%2C+but+not+transcriptional+regulation%2C+of+major+chloroplast+RNA-binding+proteins+are+related+to+Arabidopsis+seedling+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Post-translational+modifications%2C+but+not+transcriptional+regulation%2C+of+major+chloroplast+RNA-binding+proteins+are+related+to+Arabidopsis+seedling+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001994-Kleffmann1" id="pone.0001994-Kleffmann1"></a>Kleffmann T, von Zychlinski A, Russenberger D, Hirsch-Hoffmann M, Gehrig P, Gruissem W, Baginsky S (2007) Proteome dynamics during plastid differentiation in rice. Plant Physiol  143(2): 912–923.  <ul class="find" data-citedArticleID="1091446" data-doi="10.1104/pp.106.090738"><li><a href="http://dx.doi.org/10.1104/pp.106.090738" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteome+dynamics+during+plastid+differentiation+in+rice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteome+dynamics+during+plastid+differentiation+in+rice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001994-Emanuelsson2" id="pone.0001994-Emanuelsson2"></a>Emanuelsson O, Nielsen H, von Heijne G (1999) ChloroP a neural network-based method for predicting chloroplast transit peptides and their cleavage sites. Protein Sci  8(5): 978–984.  <ul class="find" data-citedArticleID="1091418" data-doi="10.1110/ps.8.5.978"><li><a href="http://dx.doi.org/10.1110/ps.8.5.978" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ChloroP+a+neural+network-based+method+for+predicting+chloroplast+transit+peptides+and+their+cleavage+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ChloroP+a+neural+network-based+method+for+predicting+chloroplast+transit+peptides+and+their+cleavage+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001994-Polevoda1" id="pone.0001994-Polevoda1"></a>Polevoda B, Sherman F (2003) N-terminal acetyltransferases and sequence requirements for N-terminal acetylation of eukaryotic proteins. J Mol Biol  325(4): 595–622.  <ul class="find" data-citedArticleID="1091494" data-doi="10.1016/s0022-2836(02)01269-x"><li><a href="http://dx.doi.org/10.1016/s0022-2836(02)01269-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=N-terminal+acetyltransferases+and+sequence+requirements+for+N-terminal+acetylation+of+eukaryotic+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22N-terminal+acetyltransferases+and+sequence+requirements+for+N-terminal+acetylation+of+eukaryotic+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001994-Peltier4" id="pone.0001994-Peltier4"></a>Peltier JB, Emanuelsson O, Kalume DE, Ytterberg J, Friso G, Rudella A, Liberles DA, Soderberg L, Roepstorff P, von Heijne G, van Wijk KJ (2002) Central Functions of the Lumenal and Peripheral Thylakoid Proteome of Arabidopsis Determined by Experimentation and Genome-Wide Prediction. Plant Cell  14(1): 211–236.  <ul class="find" data-citedArticleID="1091486" data-doi="10.1105/tpc.010304"><li><a href="http://dx.doi.org/10.1105/tpc.010304" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Central+Functions+of+the+Lumenal+and+Peripheral+Thylakoid+Proteome+of+Arabidopsis+Determined+by+Experimentation+and+Genome-Wide+Prediction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Central+Functions+of+the+Lumenal+and+Peripheral+Thylakoid+Proteome+of+Arabidopsis+Determined+by+Experimentation+and+Genome-Wide+Prediction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001994-Schubert1" id="pone.0001994-Schubert1"></a>Schubert M, Petersson UA, Haas BJ, Funk C, Schröder WP, Kieselbach T (2002) Proteome map of the chloroplast lumen of Arabidopsis thaliana. J Biol Chem  277(10): 8354–8365.  <ul class="find" data-citedArticleID="1091508" data-doi="10.1074/jbc.m108575200"><li><a href="http://dx.doi.org/10.1074/jbc.m108575200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteome+map+of+the+chloroplast+lumen+of+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteome+map+of+the+chloroplast+lumen+of+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001994-Nowaczyk1" id="pone.0001994-Nowaczyk1"></a>Nowaczyk MM, Hebeler R, Schlodder E, Meyer HE, Warscheid B, Rogner M (2006) Psb27, a cyanobacterial lipoprotein, is involved in the repair cycle of photosystem II. Plant Cell  18(11): 3121–3131.  <ul class="find" data-citedArticleID="1091472" data-doi="10.1105/tpc.106.042671"><li><a href="http://dx.doi.org/10.1105/tpc.106.042671" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Psb27%2C+a+cyanobacterial+lipoprotein%2C+is+involved+in+the+repair+cycle+of+photosystem+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Psb27%2C+a+cyanobacterial+lipoprotein%2C+is+involved+in+the+repair+cycle+of+photosystem+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001994-Roose1" id="pone.0001994-Roose1"></a>Roose JL, Pakrasi HB (2004) Evidence that D1 processing is required for manganese binding and extrinsic protein assembly into photosystem II. J Biol Chem  279(44): 45417–45422.  <ul class="find" data-citedArticleID="1091502" data-doi="10.1074/jbc.m408458200"><li><a href="http://dx.doi.org/10.1074/jbc.m408458200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+that+D1+processing+is+required+for+manganese+binding+and+extrinsic+protein+assembly+into+photosystem+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+that+D1+processing+is+required+for+manganese+binding+and+extrinsic+protein+assembly+into+photosystem+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001994-Chen1" id="pone.0001994-Chen1"></a>Chen H, Zhang D, Guo J, Wu H, Jin M, Lu Q, Lu C, Zhang L (2006) A Psb27 homologue in Arabidopsis thaliana is required for efficient repair of photodamaged photosystem II. Plant Mol Biol  61(4–5): 567–575.  <ul class="find" data-citedArticleID="1091408" data-doi="10.1007/s11103-006-0031-x"><li><a href="http://dx.doi.org/10.1007/s11103-006-0031-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Psb27+homologue+in+Arabidopsis+thaliana+is+required+for+efficient+repair+of+photodamaged+photosystem+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Psb27+homologue+in+Arabidopsis+thaliana+is+required+for+efficient+repair+of+photodamaged+photosystem+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001994-Whitelegge1" id="pone.0001994-Whitelegge1"></a>Whitelegge JP (2004) Mass spectrometry for high throughput quantitative proteomics in plant research: lessons from thylakoid membranes. Plant Physiol Biochem  42(12): 919–927.  <ul class="find" data-citedArticleID="1091530" data-doi="10.1016/j.plaphy.2004.11.005"><li><a href="http://dx.doi.org/10.1016/j.plaphy.2004.11.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mass+spectrometry+for+high+throughput+quantitative+proteomics+in+plant+research%3A+lessons+from+thylakoid+membranes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mass+spectrometry+for+high+throughput+quantitative+proteomics+in+plant+research%3A+lessons+from+thylakoid+membranes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001994-vonHeijne2" id="pone.0001994-vonHeijne2"></a>von Heijne G, Steppuhn J, Herrmann RG (1989) Domain structure of mitochondrial and chloroplast targeting peptides. Eur J Biochem  180(3): 535–45.  <ul class="find" data-citedArticleID="1091550" data-doi="10.1111/j.1432-1033.1989.tb14679.x"><li><a href="http://dx.doi.org/10.1111/j.1432-1033.1989.tb14679.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Domain+structure+of+mitochondrial+and+chloroplast+targeting+peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Domain+structure+of+mitochondrial+and+chloroplast+targeting+peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001994-Elias1" id="pone.0001994-Elias1"></a>Elias JE, Haas W, Faherty BK, Gygi SP (2005) Comparative evaluation of mass spectrometry platforms used in large-scale proteomics investigations. Nat Methods  2(9): 667–675.  <ul class="find" data-citedArticleID="1091414" data-doi="10.1038/nmeth785"><li><a href="http://dx.doi.org/10.1038/nmeth785" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+evaluation+of+mass+spectrometry+platforms+used+in+large-scale+proteomics+investigations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+evaluation+of+mass+spectrometry+platforms+used+in+large-scale+proteomics+investigations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001994-Heazlewood3" id="pone.0001994-Heazlewood3"></a>Heazlewood JL, Tonti-Filippini JS, Gout AM, Day DA, Whelan J, Millar AH (2004) Experimental analysis of the Arabidopsis mitochondrial proteome highlights signaling and regulatory components, provides assessment of targeting prediction programs, and indicates plant-specific mitochondrial proteins. Plant Cell  16(1): 241–256.  <ul class="find" data-citedArticleID="1091436" data-doi="10.1105/tpc.016055"><li><a href="http://dx.doi.org/10.1105/tpc.016055" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+analysis+of+the+Arabidopsis+mitochondrial+proteome+highlights+signaling+and+regulatory+components%2C+provides+assessment+of+targeting+prediction+programs%2C+and+indicates+plant-specific+mitochondrial+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+analysis+of+the+Arabidopsis+mitochondrial+proteome+highlights+signaling+and+regulatory+components%2C+provides+assessment+of+targeting+prediction+programs%2C+and+indicates+plant-specific+mitochondrial+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001994-Bellafiore1" id="pone.0001994-Bellafiore1"></a>Bellafiore S, Barneche F, Peltier G, Rochaix JD (2005) State transitions and light adaptation require chloroplast thylakoid protein kinase STN7. Nature  433(7028): 892–5.  <ul class="find" data-citedArticleID="1091402" data-doi="10.1038/nature03286"><li><a href="http://dx.doi.org/10.1038/nature03286" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=State+transitions+and+light+adaptation+require+chloroplast+thylakoid+protein+kinase+STN7." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22State+transitions+and+light+adaptation+require+chloroplast+thylakoid+protein+kinase+STN7.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001994-Bonardi1" id="pone.0001994-Bonardi1"></a>Bonardi V, Pesaresi P, Becker T, Schleiff E, Wagner R, Pfannschmidt T, Jahns P, Leister D (2005) Photosystem II core phosphorylation and photosynthetic acclimation require two different protein kinases. Nature  437(7062): 1179–1182.  <ul class="find" data-citedArticleID="1091404" data-doi="10.1038/nature04016"><li><a href="http://dx.doi.org/10.1038/nature04016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Photosystem+II+core+phosphorylation+and+photosynthetic+acclimation+require+two+different+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Photosystem+II+core+phosphorylation+and+photosynthetic+acclimation+require+two+different+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001994-Pfalz1" id="pone.0001994-Pfalz1"></a>Pfalz J, Liere K, Kandlbinder A, Dietz KJ, Oelmuller R (2006) pTAC2, -6, and -12 are components of the transcriptionally active plastid chromosome that are required for plastid gene expression. Plant Cell  18(1): 176–197.  <ul class="find" data-citedArticleID="1091492" data-doi="10.1105/tpc.105.036392"><li><a href="http://dx.doi.org/10.1105/tpc.105.036392" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=pTAC2%2C+-6%2C+and+-12+are+components+of+the+transcriptionally+active+plastid+chromosome+that+are+required+for+plastid+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22pTAC2%2C+-6%2C+and+-12+are+components+of+the+transcriptionally+active+plastid+chromosome+that+are+required+for+plastid+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001994-StahlZeng1" id="pone.0001994-StahlZeng1"></a>Stahl-Zeng J, Lange V, Ossola R, Eckhardt K, Krek W, Aebersold R, Domon B (2007) High sensitivity detection of plasma proteins by multiple reaction monitoring of N-glycosites. Mol Cell Proteomics  6(10): 1809–1817.  <ul class="find" data-citedArticleID="1091520" data-doi="10.1074/mcp.m700132-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m700132-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+sensitivity+detection+of+plasma+proteins+by+multiple+reaction+monitoring+of+N-glycosites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+sensitivity+detection+of+plasma+proteins+by+multiple+reaction+monitoring+of+N-glycosites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001994-Fernie1" id="pone.0001994-Fernie1"></a>Fernie AR, Geigenberger P, Stitt M (2005) Flux an important, but neglected, component of functional genomics. Curr Opin Plant Biol  8(2): 174–182.  <ul class="find" data-citedArticleID="1091420" data-doi="10.1016/j.pbi.2005.01.008"><li><a href="http://dx.doi.org/10.1016/j.pbi.2005.01.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Flux+an+important%2C+but+neglected%2C+component+of+functional+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Flux+an+important%2C+but+neglected%2C+component+of+functional+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001994-Mogk1" id="pone.0001994-Mogk1"></a>Mogk A, Schmidt R, Bukau B (2007) The N-end rule pathway for regulated proteolysis: prokaryotic and eukaryotic strategies. Trends Cell Biol  17(4): 165–172.  <ul class="find" data-citedArticleID="1091468" data-doi="10.1016/j.tcb.2007.02.001"><li><a href="http://dx.doi.org/10.1016/j.tcb.2007.02.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+N-end+rule+pathway+for+regulated+proteolysis%3A+prokaryotic+and+eukaryotic+strategies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+N-end+rule+pathway+for+regulated+proteolysis%3A+prokaryotic+and+eukaryotic+strategies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001994-Baker1" id="pone.0001994-Baker1"></a>Baker TA, Sauer RT (2006) ATP-dependent proteases of bacteria: recognition logic and operating principles. Trends Biochem Sci  31(12): 647–653.  <ul class="find" data-citedArticleID="1091398" data-doi="10.1016/j.tibs.2006.10.006"><li><a href="http://dx.doi.org/10.1016/j.tibs.2006.10.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ATP-dependent+proteases+of+bacteria%3A+recognition+logic+and+operating+principles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ATP-dependent+proteases+of+bacteria%3A+recognition+logic+and+operating+principles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001994-Hummel1" id="pone.0001994-Hummel1"></a>Hummel J, Niemann M, Wienkoop S, Schulze W, Steinhauser D, Selbig J, Walther D, Weckwerth W (2007) ProMEX: a mass spectral reference database for proteins and protein phosphorylation sites. BMC Bioinformatics  8: 216.  <ul class="find" data-citedArticleID="1091440" data-doi="10.1186/1471-2105-8-216"><li><a href="http://dx.doi.org/10.1186/1471-2105-8-216" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ProMEX%3A+a+mass+spectral+reference+database+for+proteins+and+protein+phosphorylation+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ProMEX%3A+a+mass+spectral+reference+database+for+proteins+and+protein+phosphorylation+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0001994-Reumann1" id="pone.0001994-Reumann1"></a>Reumann S, Ma C, Lemke S, Babujee L (2004) AraPerox. A database of putative Arabidopsis proteins from plant peroxisomes. Plant Physiol  136(1): 2587–2608.  <ul class="find" data-citedArticleID="1091496" data-doi="10.1104/pp.104.043695"><li><a href="http://dx.doi.org/10.1104/pp.104.043695" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AraPerox.+A+database+of+putative+Arabidopsis+proteins+from+plant+peroxisomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AraPerox.+A+database+of+putative+Arabidopsis+proteins+from+plant+peroxisomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0001994-Amin1" id="pone.0001994-Amin1"></a>Amin P, Sy DA, Pilgrim ML, Parry DH, Nussaume L, Hoffman NE (1999) Arabidopsis mutants lacking the 43- and 54-kilodalton subunits of the chloroplast signal recognition particle have distinct phenotypes. Plant Physiol  121(1): 61–70.  <ul class="find" data-citedArticleID="1091392" data-doi="10.1104/pp.121.1.61"><li><a href="http://dx.doi.org/10.1104/pp.121.1.61" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+mutants+lacking+the+43-+and+54-kilodalton+subunits+of+the+chloroplast+signal+recognition+particle+have+distinct+phenotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+mutants+lacking+the+43-+and+54-kilodalton+subunits+of+the+chloroplast+signal+recognition+particle+have+distinct+phenotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0001994-Rudella1" id="pone.0001994-Rudella1"></a>Rudella A, Friso G, Alonso JM, Ecker JR, van Wijk KJ (2006) Downregulation of ClpR2 Leads to Reduced Accumulation of the ClpPRS Protease Complex and Defects in Chloroplast Biogenesis in Arabidopsis. Plant Cell  18(7): 1704–1721.  <ul class="find" data-citedArticleID="1091506" data-doi="10.1105/tpc.106.042861"><li><a href="http://dx.doi.org/10.1105/tpc.106.042861" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Downregulation+of+ClpR2+Leads+to+Reduced+Accumulation+of+the+ClpPRS+Protease+Complex+and+Defects+in+Chloroplast+Biogenesis+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Downregulation+of+ClpR2+Leads+to+Reduced+Accumulation+of+the+ClpPRS+Protease+Complex+and+Defects+in+Chloroplast+Biogenesis+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0001994-vanWijk3" id="pone.0001994-vanWijk3"></a>van Wijk KJ, Peltier JB, Giacomelli L (2007) Isolation of chloroplast proteins from Arabidopsis thaliana for proteome analysis. Methods Mol Biol  355: 43–48.  <ul class="find" data-citedArticleID="1091546" data-doi="10.1385/1-59745-227-0:43"><li><a href="http://dx.doi.org/10.1385/1-59745-227-0:43" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+chloroplast+proteins+from+Arabidopsis+thaliana+for+proteome+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+chloroplast+proteins+from+Arabidopsis+thaliana+for+proteome+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0001994-Shevchenko1" id="pone.0001994-Shevchenko1"></a>Shevchenko A, Wilm M, Vorm O, Mann M (1996) Mass spectrometric sequencing of proteins silver-stained polyacrylamide gels. Anal Chem  68(5): 850–858.  <ul class="find" data-citedArticleID="1091512" data-doi="10.1021/ac950914h"><li><a href="http://dx.doi.org/10.1021/ac950914h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mass+spectrometric+sequencing+of+proteins+silver-stained+polyacrylamide+gels." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mass+spectrometric+sequencing+of+proteins+silver-stained+polyacrylamide+gels.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.XML" value="225072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.PDF" value="676699"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g001.PNG_L" value="93120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g001.PNG_M" value="62901"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g001.PNG_S" value="10247"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g001.TIF" value="125608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g001.PNG_I" value="22606"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t001.PNG_L" value="246822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t001.PNG_M" value="143278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t001.PNG_S" value="13713"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t001.TIF" value="1048526"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t001.PNG_I" value="43832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g002.PNG_L" value="949519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g002.PNG_M" value="52436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g002.PNG_S" value="9384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g002.TIF" value="1695868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g002.PNG_I" value="19005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g003.PNG_L" value="69021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g003.PNG_M" value="37738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g003.PNG_S" value="7301"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g003.TIF" value="225468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g003.PNG_I" value="13608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g004.PNG_L" value="183038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g004.PNG_M" value="131320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g004.PNG_S" value="13398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g004.TIF" value="1191984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g004.PNG_I" value="47242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g005.PNG_L" value="153841"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g005.PNG_M" value="114349"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g005.PNG_S" value="10981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g005.TIF" value="1169824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g005.PNG_I" value="39379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g006.PNG_L" value="1508621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g006.PNG_M" value="224971"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g006.PNG_S" value="16179"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g006.TIF" value="2884834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g006.PNG_I" value="75390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t002.PNG_L" value="661208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t002.PNG_M" value="175316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t002.PNG_S" value="21272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t002.TIF" value="2925172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t002.PNG_I" value="129482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t003.PNG_L" value="352687"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t003.PNG_M" value="198691"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t003.PNG_S" value="16744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t003.TIF" value="1482114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.t003.PNG_I" value="59937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g007.PNG_L" value="769986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g007.PNG_M" value="139853"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g007.PNG_S" value="11018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g007.TIF" value="3007370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.g007.PNG_I" value="46640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.e001.PNG" value="11863"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.e001.TIF" value="27460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.e002.PNG" value="6020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.e002.TIF" value="21952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.e003.PNG" value="7156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.e003.TIF" value="23088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.e004.PNG" value="10846"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.e004.TIF" value="26540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.s001.DOC" value="32256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.s002.XLS" value="326656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.s003.XLS" value="396800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.s004.XLS" value="97792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.s005.XLS" value="255488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.s006.XLS" value="20992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001994.s007.XLS" value="566272"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001994&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001994" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001994&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001994&volume=&issue=&title=Sorting Signals, N-Terminal Modifications and Abundance of the Chloroplast Proteome&author_name=Boris%20Zybailov%2C%20Heidi%20Rutschow%2C%20Giulia%20Friso%2C%20Andrea%20Rudella%2C%20Olof%20Emanuelsson%2C%20Qi%20Sun%2C%20Klaas%20J.%20van%20Wijk&start_page=1&end_page=19" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001994" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001994" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001994" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001994&amp;t=Sorting%20Signals%2C%20N-Terminal%20Modifications%20and%20Abundance%20of%20the%20Chloroplast%20Proteome" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001994&title=Sorting%20Signals%2C%20N-Terminal%20Modifications%20and%20Abundance%20of%20the%20Chloroplast%20Proteome&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001994&amp;title=Sorting%20Signals%2C%20N-Terminal%20Modifications%20and%20Abundance%20of%20the%20Chloroplast%20Proteome" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001994" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001994&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Sorting Signals, N-Terminal Modifications and Abundance of the Chloroplast Proteome';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001994';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Sorting%20Signals%2C%20N-Terminal%20Modifications%20and%20Abundance%20of%20the%20Chloroplast%20Proteome http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001994" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001994" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arabidopsis+thaliana%22" title="Search for articles in the subject area:'Arabidopsis thaliana'"><div class="flagText">Arabidopsis thaliana</div></a>
              <div data-categoryid="32845" data-articleid="26812"
                   data-categoryname="Arabidopsis thaliana"
                   class="flagImage" title="Flag 'Arabidopsis thaliana' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chloroplasts%22" title="Search for articles in the subject area:'Chloroplasts'"><div class="flagText">Chloroplasts</div></a>
              <div data-categoryid="32715" data-articleid="26812"
                   data-categoryname="Chloroplasts"
                   class="flagImage" title="Flag 'Chloroplasts' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+proteins%22" title="Search for articles in the subject area:'Membrane proteins'"><div class="flagText">Membrane proteins</div></a>
              <div data-categoryid="17569" data-articleid="26812"
                   data-categoryname="Membrane proteins"
                   class="flagImage" title="Flag 'Membrane proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Peptides%22" title="Search for articles in the subject area:'Peptides'"><div class="flagText">Peptides</div></a>
              <div data-categoryid="17847" data-articleid="26812"
                   data-categoryname="Peptides"
                   class="flagImage" title="Flag 'Peptides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plastids%22" title="Search for articles in the subject area:'Plastids'"><div class="flagText">Plastids</div></a>
              <div data-categoryid="19679" data-articleid="26812"
                   data-categoryname="Plastids"
                   class="flagImage" title="Flag 'Plastids' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteomes%22" title="Search for articles in the subject area:'Proteomes'"><div class="flagText">Proteomes</div></a>
              <div data-categoryid="18649" data-articleid="26812"
                   data-categoryname="Proteomes"
                   class="flagImage" title="Flag 'Proteomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteomic+databases%22" title="Search for articles in the subject area:'Proteomic databases'"><div class="flagText">Proteomic databases</div></a>
              <div data-categoryid="43483" data-articleid="26812"
                   data-categoryname="Proteomic databases"
                   class="flagImage" title="Flag 'Proteomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26812"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3741'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9916'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9397&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
