

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A North American Yersinia pestis Draft Genome Sequence: SNPs and Phylogenetic Analysis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000220"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000220" />

    <meta name="citation_title" content="A North American Yersinia pestis Draft Genome Sequence: SNPs and Phylogenetic Analysis"/>
    <meta itemprop="name" content="A North American Yersinia pestis Draft Genome Sequence: SNPs and Phylogenetic Analysis"/>

      <meta name="citation_author" content="Jeffrey W. Touchman"/>
            <meta name="citation_author_institution" content="The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America"/>
            <meta name="citation_author_institution" content="The Biodesign Institute, Arizona State University, Tempe, Ariziona, United States of America"/>
      <meta name="citation_author" content="David M. Wagner"/>
            <meta name="citation_author_institution" content="Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America"/>
      <meta name="citation_author" content="Jicheng Hao"/>
            <meta name="citation_author_institution" content="The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America"/>
      <meta name="citation_author" content="Stephen D. Mastrian"/>
            <meta name="citation_author_institution" content="The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America"/>
      <meta name="citation_author" content="Maulik K. Shah"/>
            <meta name="citation_author_institution" content="The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America"/>
      <meta name="citation_author" content="Amy J. Vogler"/>
            <meta name="citation_author_institution" content="Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America"/>
      <meta name="citation_author" content="Christopher J. Allender"/>
            <meta name="citation_author_institution" content="Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America"/>
      <meta name="citation_author" content="Erin A. Clark"/>
            <meta name="citation_author_institution" content="The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America"/>
      <meta name="citation_author" content="Debbie S. Benitez"/>
            <meta name="citation_author_institution" content="The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America"/>
      <meta name="citation_author" content="David J. Youngkin"/>
            <meta name="citation_author_institution" content="The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America"/>
      <meta name="citation_author" content="Jessica M. Girard"/>
            <meta name="citation_author_institution" content="Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America"/>
      <meta name="citation_author" content="Raymond K. Auerbach"/>
            <meta name="citation_author_institution" content="Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America"/>
      <meta name="citation_author" content="Stephen M. Beckstrom-Sternberg"/>
            <meta name="citation_author_institution" content="The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America"/>
            <meta name="citation_author_institution" content="Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America"/>
      <meta name="citation_author" content="Paul Keim"/>
            <meta name="citation_author_institution" content="The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America"/>
            <meta name="citation_author_institution" content="Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America"/>

    <meta name="citation_date" content="2007/2/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000220.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e220"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Yersinia pestis - Etiologic agent of plague.; citation_author=RD Perry; citation_author=JD Fetherston; citation_journal_title=Clinical Microbiology Reviews; citation_volume=10; citation_number=1; citation_pages=35-66; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Plague as a Biological Weapon: Medical and Public Health Management.; citation_author=TV Inglesby; citation_author=DT Dennis; citation_author=DA Henderson; citation_author=JG Bartlett; citation_author=MS Ascher; citation_journal_title=JAMA; citation_volume=283; citation_number=2; citation_pages=2281-2290; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Public health assessment of potential biological terrorism agents.; citation_author=LD Rotz; citation_author=AS Khan; citation_author=SR Lillibridge; citation_author=SM Ostroff; citation_author=JM Hughes; citation_journal_title=Emerging Infectious Diseases; citation_volume=8; citation_number=3; citation_pages=225-230; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Age, descent and genetic diversity within Yersinia pestis.; citation_author=M Achtman; citation_number=4; citation_pages=432; citation_date=2004; citation_publisher=Horizon Bioscience; " />
      <meta name="citation_reference" content="citation_title=Microevolution and history of the plague bacillus, Yersinia pestis.; citation_author=M Achtman; citation_author=G Morelli; citation_author=P Zhu; citation_author=T Wirth; citation_author=I Diehl; citation_journal_title=PNAS; citation_volume=101; citation_number=5; citation_pages=17837-17842; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Yersinia pestis, the cause of plague, is a recently emerged clone of Yersinia pseudotuberculosis.; citation_author=M Achtman; citation_author=K Zurth; citation_author=G Morelli; citation_author=G Torrea; citation_author=A Guiyoule; citation_journal_title=Proceedings Of The National Academy Of Sciences Of The United States Of America; citation_volume=96; citation_number=6; citation_pages=14043-14048; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Insights into the evolution of Yersinia pestis through whole-genome comparison with Yersinia pseudotuberculosis.; citation_author=PSG Chain; citation_author=E Carniel; citation_author=FW Larimer; citation_author=J Lamerdin; citation_author=PO Stoutland; citation_journal_title=PNAS; citation_volume=101; citation_number=7; citation_pages=13826-13831; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A history of plague in the United States of America.; citation_author=VB Link; citation_journal_title=Public Health Monographs; citation_volume=26; citation_number=8; citation_pages=1-120; citation_date=1955; " />
      <meta name="citation_reference" content="citation_title=Interspecific comparisons of sylvatic plague in prairie dogs.; citation_author=JF Cully; citation_author=ES Williams; citation_journal_title=Journal of Mammalogy; citation_volume=82; citation_number=9; citation_pages=894-905; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Incidence of plague associated with increased winter-spring precipitation in New Mexico.; citation_author=RR Parmenter; citation_author=EP Yadav; citation_author=CA Parmenter; citation_author=P Ettestad; citation_author=KL Gage; citation_journal_title=The American Journal Of Tropical Medicine And Hygiene; citation_volume=61; citation_number=10; citation_pages=814-821; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Modeling relationships between climate and the frequency of human plague cases in the southwestern United States, 1960–1997.; citation_author=RE Enscore; citation_author=BJ Biggerstaff; citation_author=TL Brown; citation_author=RE Fulgham; citation_author=PJ Reynolds; citation_journal_title=The American Journal of Tropical Medicine and Hygiene; citation_volume=66; citation_number=11; citation_pages=186-196; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Differential plague-transmission dynamics determine Yersinia pestis population genetic structure on local, regional, and global scales.; citation_author=JM Girard; citation_author=DM Wagner; citation_author=AJ Vogler; citation_author=C Keys; citation_author=CJ Allender; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=12; citation_pages=8408-8413; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of Yersinia pestis, the causative agent of plague.; citation_author=J Parkhill; citation_author=BW Wren; citation_author=NR Thomson; citation_author=RW Titball; citation_author=MT Holden; citation_journal_title=Nature; citation_volume=413; citation_number=13; citation_pages=523-527; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Whole-genome sequence assembly for mammalian genomes: Arachne 2.; citation_author=DB Jaffe; citation_author=J Butler; citation_author=S Gnerre; citation_author=E Mauceli; citation_author=K Lindblad-Toh; citation_journal_title=Genome Res; citation_volume=13; citation_number=14; citation_pages=91-96; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Fast algorithms for large-scale genome alignment and comparison.; citation_author=AL Delcher; citation_author=A Phillippy; citation_author=J Carlton; citation_author=SL Salzberg; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=15; citation_pages=2478-2483; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of Yersinia pestis strains Antiqua and Nepal516: evidence of gene reduction in an emerging pathogen.; citation_author=PS Chain; citation_author=P Hu; citation_author=SA Malfatti; citation_author=L Radnedge; citation_author=F Larimer; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=16; citation_pages=4453-4463; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic discovery bias in Bacillus anthracis using single-nucleotide polymorphisms from whole-genome sequencing.; citation_author=T Pearson; citation_author=JD Busch; citation_author=J Ravel; citation_author=TD Read; citation_author=SD Rhoton; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=17; citation_pages=13536-13541; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An automated hydrodynamic process for controlled, unbiased DNA shearing.; citation_author=YR Thorstenson; citation_author=SP Hunicke-Smith; citation_author=PJ Oefner; citation_author=RW Davis; citation_journal_title=Genome Res; citation_volume=8; citation_number=18; citation_pages=848-855; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. I. Accuracy assessment.; citation_author=B Ewing; citation_author=L Hillier; citation_author=MC Wendl; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=19; citation_pages=175-185; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=MultiPipMaker and supporting tools: alignments and analysis of multiple genomic DNA sequences.; citation_author=S Schwartz; citation_author=L Elnitski; citation_author=M Li; citation_author=M Weirauch; citation_author=C Riemer; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=20; citation_pages=1-8; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities.; citation_author=B Ewing; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=21; citation_pages=186-194; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=REPuter: fast computation of maximal repeats in complete genomes.; citation_author=S Kurtz; citation_author=C Schleiermacher; citation_journal_title=Bioinformatics; citation_volume=15; citation_number=22; citation_pages=426-427; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=MEGA2: molecular evolutionary genetics analysis software.; citation_author=S Kumar; citation_author=K Tamura; citation_author=IB Jakobsen; citation_author=M Nei; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=23; citation_pages=1244-1245; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000220" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A North American Yersinia pestis Draft Genome Sequence: SNPs and Phylogenetic Analysis"/>
    <meta name="twitter:description" content="BackgroundYersinia pestis, the causative agent of plague, is responsible for some of the greatest epidemic scourges of mankind. It is widespread in the western United States, although it has only been present there for just over 100 years. As a result, there has been very little time for diversity to accumulate in this region. Much of the diversity that has been detected among North American isolates is at loci that mutate too quickly to accurately reconstruct large-scale phylogenetic patterns. Slowly-evolving but stable markers such as SNPs could be useful for this purpose, but are difficult to identify due to the monomorphic nature of North American isolates.Methodology/Principal FindingsTo identify SNPs that are polymorphic among North American populations of Y. pestis, a gapped genome sequence of Y. pestis strain FV-1 was generated. Sequence comparison of FV-1 with another North American strain, CO92, identified 19 new SNP loci that differ among North American isolates.Conclusions/SignificanceThe 19 SNP loci identified in this study should facilitate additional studies of the genetic population structure of Y. pestis across North America."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000220.g001"/>

  <meta property="og:title" content="A North American Yersinia pestis Draft Genome Sequence: SNPs and Phylogenetic Analysis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000220" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2563'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1770'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5804&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000220">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000220" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000220&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A North American <i>Yersinia pestis</i> Draft Genome Sequence: SNPs and Phylogenetic Analysis
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jeffrey W. Touchman
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Jeffrey W. Touchman, David M. Wagner</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jtouchman@tgen.org">jtouchman@tgen.org</a></p>

                <p>Affiliations:
                  The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America, 
                  The Biodesign Institute, Arizona State University, Tempe, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David M. Wagner
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Jeffrey W. Touchman, David M. Wagner</p>

              
              

                <p>Affiliation:
                  Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jicheng Hao, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen D. Mastrian, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maulik K. Shah, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Amy J. Vogler, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christopher J. Allender, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Erin A. Clark, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Debbie S. Benitez, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David J. Youngkin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jessica M. Girard, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Raymond K. Auerbach, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen M. Beckstrom-Sternberg, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America, 
                  Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paul Keim
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  The Translational Genomics Research Institute, Phoenix, Ariziona, United States of America, 
                  Center for Microbial Genetics and Genomics, Northern Arizona University, Flagstaff, Ariziona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000220</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000220" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000220" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000220" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000220" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000220-t001" data-doi="info:doi/10.1371/journal.pone.0000220" data-uri="info:doi/10.1371/journal.pone.0000220.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000220.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000220-t002" data-doi="info:doi/10.1371/journal.pone.0000220" data-uri="info:doi/10.1371/journal.pone.0000220.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000220.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000220-g001" data-doi="info:doi/10.1371/journal.pone.0000220" data-uri="info:doi/10.1371/journal.pone.0000220.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000220.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000220-t003" data-doi="info:doi/10.1371/journal.pone.0000220" data-uri="info:doi/10.1371/journal.pone.0000220.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000220.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000220">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000220" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p><em>Yersinia pestis</em>, the causative agent of plague, is responsible for some of the greatest epidemic scourges of mankind. It is widespread in the western United States, although it has only been present there for just over 100 years. As a result, there has been very little time for diversity to accumulate in this region. Much of the diversity that has been detected among North American isolates is at loci that mutate too quickly to accurately reconstruct large-scale phylogenetic patterns. Slowly-evolving but stable markers such as SNPs could be useful for this purpose, but are difficult to identify due to the monomorphic nature of North American isolates.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>To identify SNPs that are polymorphic among North American populations of <em>Y. pestis</em>, a gapped genome sequence of <em>Y. pestis</em> strain FV-1 was generated. Sequence comparison of FV-1 with another North American strain, CO92, identified 19 new SNP loci that differ among North American isolates.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The 19 SNP loci identified in this study should facilitate additional studies of the genetic population structure of <em>Y. pestis</em> across North America.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Touchman JW, Wagner DM, Hao J, Mastrian SD, Shah MK, et al.  (2007) A North American <em>Yersinia pestis</em> Draft Genome Sequence: SNPs and Phylogenetic Analysis. PLoS ONE 2(2):
          e220.
            doi:10.1371/journal.pone.0000220</p><p><strong>Academic Editor: </strong>Jean Carr, Institute of Human Virology, United States of America</p><p><strong>Received:</strong> December 29, 2006; <strong>Accepted:</strong> January 26, 2007; <strong>Published:</strong> February 21, 2007</p><p><strong>Copyright:</strong> © 2007 Touchman et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors have no funding to report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Plague is a disease of both historical and current importance to global health. The first accounts of this disease are from Europe in the 6th century during the Justinian rule of the Roman Empire. After a century of epidemics, plague disappeared until the Middle Ages (14th century), when the devastation of the epidemic known as the Black Death killed millions of Europeans. That second pandemic raged until the 17th century when plague again disappeared. The third and current pandemic began in the mid 19th century when the disease emerged from eastern China. At that time, worldwide commerce allowed it to spread to all inhabited continents. As a result, plague became ecologically established in many locations, including western North America, and active foci are currently found on all continents except Australia and Antarctica <a href="#pone.0000220-Perry1">[1]</a>. In the 20th century, plague was developed as a biological weapon by Japan, the United States, and the former Soviet Union <a href="#pone.0000220-Inglesby1">[2]</a>. It is currently listed as a Category A biological agent for public health preparedness, due to its potential use in biological terrorism <a href="#pone.0000220-Rotz1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Plague is caused by the Gram negative bacterium <em>Yersinia pestis</em>. Genetic analyses suggest that <em>Y. pestis</em> is a recently emerged clone that evolved from <em>Y. pseudotuberculosis</em> within the last 20,000 years <a href="#pone.0000220-Achtman1">[4]</a>–<a href="#pone.0000220-Achtman3">[6]</a>. Despite this close genetic relationship, <em>Y. pestis</em> and <em>Y. pseudotuberculosis</em> exhibit radically different life cycles: <em>Y. pseudotuberculosis</em> lives directly in the environment, whereas <em>Y. pestis</em> is an obligate pathogen that is found only in arthropod vectors or mammalian hosts. This dual life must be evolutionarily challenging and represents a very different and perhaps more restricted niche compared to that of <em>Y. pseudotuberculosis</em>. Comparisons of complete genome sequences from strains of these two species support the idea of a more restricted niche for <em>Y. pestis</em>, as up to 13% of the <em>Y. pseudotuberculosis</em> genes are no longer active in <em>Y. pestis</em> <a href="#pone.0000220-Chain1">[7]</a>. These same analyses identified 32 chromosomal genes unique to <em>Y. pestis</em> <a href="#pone.0000220-Chain1">[7]</a>. These new genes, together with the two plasmids that are unique to <em>Y. pestis</em>, may have facilitated the radical and rapid niche shift observed in this recently emerged pathogen.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Plague was introduced to North America quite recently, within the last 125 years. In the late 1890s and early 1900s, the disease was first documented in human and non-native rat populations in several coastal cities in North America, including: Galveston, Los Angeles, San Francisco, and Seattle <a href="#pone.0000220-Link1">[8]</a>. Following improved hygiene and other measures that decreased populations of non-native rats, the plague mostly disappeared from these cities. However, in 1908, it was documented as the causative agent of large-scale epizootics in native rodent populations near San Francisco <a href="#pone.0000220-Link1">[8]</a>. It is commonly accepted that, following this initial introduction into the native fauna, plague spread rapidly through rodent populations, reaching its current North American distribution by 1950. That distribution is limited to the 17 western-most states, with the eastern boundary consisting roughly of 100°W longitude <a href="#pone.0000220-Cully1">[9]</a>. Plague is now endemic in the western USA, with major foci in California and the Four Corners region, which includes portions of Arizona, Colorado, New Mexico, and Utah. In the United States, the disease is usually cryptic but occasionally responds to environmental cues <a href="#pone.0000220-Parmenter1">[10]</a>, <a href="#pone.0000220-Enscore1">[11]</a>, which results in large-scale epizootics among susceptible hosts <a href="#pone.0000220-Girard1">[12]</a>. These epizootics are primarily associated with two closely-related groups of rodents: prairie dogs (<em>Cynomys</em> spp.) and ground squirrels (<em>Spermophilus</em> spp.).</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Molecular analyses used to differentiate and characterize strains from regional populations across North America are hampered by a lack of informative genetic markers. Variable number tandem repeat markers provide good resolution at a regional scale <a href="#pone.0000220-Girard1">[12]</a> but are less useful at defining the major phylogenetic patterns across North America (Wagner et al., unpublished). To explore genomic variation in North American plague, we have sequenced a second North American isolate, FV-1, from a recent epizootic <a href="#pone.0000220-Girard1">[12]</a> and characterized the differences between it and the available sequence of another North American strain, CO92 <a href="#pone.0000220-Parkhill1">[13]</a>, which was isolated in Colorado. We report here the gapped genomic sequencing of strain FV-1, the comparative analysis of FV-1 with CO92 that has resulted in the identification of multiple informative single nucleotide polymorphism (SNP) markers, and the phylogenetic stratification of 22 different North American <em>Y. pestis</em> isolates using these markers.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Gapped-genome sequencing of <em>Yersinia pestis</em> strain FV-1</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The <em>Y. pestis</em> FV-1 isolate used in this study was obtained from a plague epizootic in a Gunnison's prairie dog (<em>Cynomys gunnisoni</em>) colony near the city of Flagstaff, Arizona, U.S.A. in 2001 <a href="#pone.0000220-Girard1">[12]</a>. The genome of strain FV-1 was sequenced by a whole-genome shotgun strategy that achieved 11-fold average sequence coverage per base pair. A total of 62,649 random sequence reads were assembled into 444 contiguous segments with an N50 length (the length X such that 50% of the sequence is contained in contigs of length X or greater) of 19,182 base pairs. We note that the number of contigs is somewhat high given the level of sequence coverage achieved. We attribute this to the repeat-rich nature of the plague genome (3.7%, <a href="#pone.0000220-Parkhill1">[13]</a>) that can confound assembly programs. Nevertheless, we estimate that roughly 94.4% of the genome sequence of FV-1 is contained in this dataset by aligning it to the highly similar strain CO92. Due to the high level of sequence coverage, the quality of the resulting sequence data is high, with 98% percent of consensus base pairs possessing an estimated error rate of less than 1 in 1,000. The program Arachne <a href="#pone.0000220-Jaffe1">[14]</a> was used to assemble the genome and no assembly errors were observed based on the concordance of forward and reverse read-pairs.</p>


<h4>Single Nucleotide Polymorphisms</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The gapped genome sequence of <em>Y. pestis</em> strain FV-1 was aligned to the complete genome sequence of strain CO92 (GenBank no. NC_003143) using the program MUMmer <a href="#pone.0000220-Delcher1">[15]</a>. We discovered 32 putative single nucleotide polymorphisms (SNPs) by extracting single-base-pair mismatches from the alignment of the two genomes. A criteria was enforced that required that the SNP nucleotide in the FV-1 sequence be of high quality (see <a href="#s3">Methods</a>). We further required the presence of 250 base pairs on either side of the putative SNP in FV-1 to avoid potential poor quality SNPs at contig ends. SNPs were verified by testing genomic DNA from both discovery strains for the allelic difference at each putative SNP locus. Out of 32 putative SNPs identified through genome sequence comparison, 19 were confirmed in both the FV-1 and CO92 genomes by direct testing of purified genomic DNA. Of note, 11 of the 32 SNPs calls were confirmed in FV-1 DNA but were incorrect in CO92 DNA, suggesting that perhaps our CO92 isolate used for SNP validation is slightly dissimilar to the strain sequenced by Parkhill et al. <a href="#pone.0000220-Parkhill1">[13]</a>, or the publicly available sequence of strain CO92 contains an error at these 11 particular positions. An error rate of &lt;3×10−6 would be excellent if the latter is the case.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The 19 confirmed SNPs were located within both coding and non-coding regions of the genome; 14 are located within genes, the other 5 in intergenic regions (<a href="#pone-0000220-t001">Table 1</a>). Based on the CO92 gene density of 83.9% <a href="#pone.0000220-Parkhill1">[13]</a>, this ratio reflects a modest bias towards a lower occurrence of SNPs within genes (binomial test; P-value = 0.11). Of the SNPs occurring within genes, 11 are nonsynonymous changes (nsSNP), two are synonymous (sSNP), and one is a nonsense mutation. All but one of the 11 nsSNPs causes a non-conservative amino acid change. The number of SNPs observed between these two strains is too small to determine whether they were caused by random events, but they are distributed evenly across the genome. Furthermore, the ratio of nsSNPs versus sSNPs, as well as their distribution across many functional gene categories, is consistent with other published reports of inter-strain <em>Y. pestis</em> SNP patterns <a href="#pone.0000220-Chain2">[16]</a>.</p>
<div class="figure" id="pone-0000220-t001"><div class="img"><a name="pone-0000220-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000220.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000220" data-uri="info:doi/10.1371/journal.pone.0000220.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000220.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000220.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000220.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000220.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000220.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000220.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000220.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000220.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000220.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>19 SNPs discovered in the comparison of the CO92 and FV-1 genome sequences.</span></strong></p><span>doi:10.1371/journal.pone.0000220.t001</span></div>

<h4>Phylogenetic analysis</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>All SNPs discovered between CO92 and FV-1 were screened across a collection of DNA from 24 <em>Yerinia</em> isolates (<a href="#pone-0000220-t002">Table 2</a>). The screening set included 20 diverse <em>Y. pestis</em> isolates from North America, as well as the two discovery strains (FV-1 and CO92), which were included as positive controls. Two additional isolates were included to provide an outgroup for rooting the phylogenetic tree: one <em>Y. pseudotuberculosis</em> (nearest-neighbor) isolate from France and one <em>Y. pestis</em> isolate from Kenya. Unique SNP genotypes were identified among the screened samples and one representative of each genotype was included in a neighbor-joining analysis. The subsequent phylogenetic hypothesis was rooted on the genotype assigned to the <em>Y. pseudotuberculosis</em> isolate.</p>
<div class="figure" id="pone-0000220-t002"><div class="img"><a name="pone-0000220-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000220.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000220" data-uri="info:doi/10.1371/journal.pone.0000220.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000220.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000220.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000220.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000220.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000220.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000220.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000220.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000220.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000220.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Twenty-four <em>Yersinia</em> spp. strains used to screen FV-1/CO92 SNPs.</span></strong></p><span>doi:10.1371/journal.pone.0000220.t002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Including the two discovery strains, seven SNP genotypes were identified among the 24 screening isolates (<a href="#pone-0000220-t003">Table 3</a>). As predicted by previous studies <a href="#pone.0000220-Pearson1">[17]</a>, phylogenetic analysis of the seven genotypes yielded a linear phylogeny (<a href="#pone-0000220-g001">Figure 1</a>). A linear phylogeny occurs due to discovery bias and a lack of any reversal mutations in the samples studied. It is important to note that, excluding the two discovery strains, only 20 North American <em>Y. pestis</em> isolates were screened across the SNPs and these isolates are not a complete representation of all of the geographic locations where plague occurs in North America. It seems quite plausible that additional SNP genotypes will be discovered when these SNPs are screened across a larger isolate collection (currently unavailable), though these should still be consistent with a linear phylogenetic topology.</p>
<div class="figure" id="pone-0000220-g001"><div class="img"><a name="pone-0000220-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000220.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000220" data-uri="info:doi/10.1371/journal.pone.0000220.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000220.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000220.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000220.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000220.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000220.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000220.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000220.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000220.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000220.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Phylogenetic relationships among the seven <em>Y. pestis</em> SNP genotypes identified in this study. Neighbor-joining tree was created using the data presented in <a href="#pone-0000220-t003">Table 3</a> and MEGA2 software <a href="#pone.0000220-Kumar1">[23]</a>, and was rooted on genotype 0, as this genotype was assigned to the two outgroup isolates (see text). The position of the 19 SNPs (<a href="#pone-0000220-t001">Table 1</a>) are indicated below the tree. Individual genotypes or nodes were named based upon their position relative to genotype 0.</p>
<span>doi:10.1371/journal.pone.0000220.g001</span></div><div class="figure" id="pone-0000220-t003"><div class="img"><a name="pone-0000220-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000220.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000220" data-uri="info:doi/10.1371/journal.pone.0000220.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000220.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000220.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000220.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000220.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000220.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000220.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000220.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000220.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000220.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Seven SNP genotypes identified in this study.</span></strong></p><span>doi:10.1371/journal.pone.0000220.t003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Some of the SNPs at the ends of the tree may be specific to the discovery strains (e.g., SNPs a–j may be specific to FV-1 and SNPs p–s may be specific to CO92). However, besides CO92, only two additional isolates from Colorado were included in the screening set and no additional Arizona isolates were included in the set. To determine if these SNPs are indeed specific to the discovery strains, it will be necessary to query these SNPs against isolates that are very closely related to the discovery strains.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Genome Sequencing</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The <em>Y. pestis</em> FV-1 isolate used in this study was obtained from a plague epizootic in a Gunnison's prairie dog (<em>Cynomys gunnisoni</em>) colony near the city of Flagstaff, Arizona, U.S.A. in 2001 <a href="#pone.0000220-Girard1">[12]</a>. Approximately10 μg of <em>Y. pestis</em> strain FV-1 genomic DNA was randomly sheared using a GeneMachines Hydroshear instrument <a href="#pone.0000220-Thorstenson1">[18]</a>. The sheared BAC-derived fragments were subjected to end-repair with Klenow and T4 DNA polymerase, ligated to BstXI linkers, and size selected on a 1% low-melt agarose gel. Linker-ligated DNA fragments in the size range of 3–4 kb were purified and ligated to BstXI-digested pOTWI3 vector (provided by the Whitehead Institute/MIT Center for Genome Research). The ligated DNA was electroporated into T1-phage-resistant DH10B <em>E. coli</em> cells (Invitrogen), and the resulting transformed cells were plated onto LB agar plates containing 25 μg/ml chloramphenicol. A total of 33,600 random plasmid subclones were selected and DNA was purified for sequencing using the Concert96 plasmid purification kit (Invitrogen) or by solid-phase reversible immobilization (SPRI) magnetic beads (SprintPrep; Agencourt Bioscience Corporation). Sequencing reactions were performed from both ends of plasmid inserts using BigDye Terminator/Version 3.1 (Applied Biosystems) and universal M13 primers (forward and reverse) for 35 cycles in Perkin Elmer 9700 thermocycyclers. Raw sequence data was produced on Applied Biosystems 3730xl sequencing instruments and analyzed with the Phred basecaller software <a href="#pone.0000220-Ewing1">[19]</a>. A total of 62,649 reads were assembled using the whole-genome assemblers Arachne 2.0 <a href="#pone.0000220-Jaffe1">[14]</a>. This Whole Genome Shotgun project has been deposited at DDBJ/EMBL/GenBank under the project accession AAUB00000000. The version described in this paper is AAUB01000000.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Dotplot analysis was performed using the program PipMaker using the “single coverage” and “ordered and oriented” options <a href="#pone.0000220-Schwartz1">[20]</a>.</p>


<h4>Identification and analysis of single-nucleotide polymorphisms (SNPs)</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The gapped genome sequence of <em>Y. pestis</em> strain FV-1 was compared to the complete genome sequence of strain CO92 (GenBank no. NC_003143) using the program MUMmer <a href="#pone.0000220-Delcher1">[15]</a>. Putative SNPs were identified from this alignment by extracting single-base-pair mismatches. We enforced the criteria that the SNP nucleotide in the FV-1 sequence possess a quality score of Phred-30 or above (sequence quality scores were not available for strain CO92) <a href="#pone.0000220-Ewing2">[21]</a>. We further required that 250 base pairs on either side of the putative SNP in the FV-1 sequence be present (to avoid SNPs at the ends of contigs where sequence quality tends to be poor). A final validation step verified that the SNP was from a unique region of the reference sequence using the program REPuter <a href="#pone.0000220-Kurtz1">[22]</a>. Those SNPs located within repetitive sequence were excluded from further analysis.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Detection of SNPs was performed using the Homogenous MassEXTEND (hME) assay as per manufacturers instructions (Sequenom, San Diego CA). Briefly, multiplexed assays were designed using the Sequenom Assay Design 2.0 software. PCR amplification of genomic DNA was performed using the Sequenom massEXTEND 12-plex multiplexing protocol (Sequenom application note 8876-001, R01, CO 040038) for 45 cycles in Perkin Elmer 9700 thermocyclers. The hME primers for each multiplex reaction were spotted onto SpectroCHIPs using the Sequenom CCU RoboDesign nanodipenser, detected on the Sequenom MALDI-TOF mass spectrophotometer (MS), and adjusted to normalize MALDI-TOF MS peak height intensities for subsequent multiplexing reactions. Shrimp alkaline phosphatase (SAP)-treated PCR products were multiplexed using normalized primer concentrations and amplified for 99 cycles in PE 9700 thermocyclers, followed by nanodispensing and MALDI-TOF MS analysis.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to collaborators at the Institut Pasteur Paris, the California Department of Health Services, the United States Army Medical Research Institute for Infectious Diseases, the Arizona Department of Health Services, and Dugway Proving Grounds for providing DNAs and/or strains of <em>Y. pestis</em>.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PK JT DW SB. Performed the experiments: AV CA JH SM EC DB DY. Analyzed the data: JT DW JG RA JH MS SB. Contributed reagents/materials/analysis tools: AV CA. Wrote the paper: PK JT DW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000220-Perry1" id="pone.0000220-Perry1"></a>Perry RD, Fetherston JD (1997) Yersinia pestis - Etiologic agent of plague. Clinical Microbiology Reviews  10: 35–66.  <ul class="find" data-citedArticleID="930982"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Yersinia+pestis+-+Etiologic+agent+of+plague.&amp;auth=&amp;atitle=Yersinia+pestis+-+Etiologic+agent+of+plague." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yersinia+pestis+-+Etiologic+agent+of+plague." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yersinia+pestis+-+Etiologic+agent+of+plague.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000220-Inglesby1" id="pone.0000220-Inglesby1"></a>Inglesby TV, Dennis DT, Henderson DA, Bartlett JG, Ascher MS, et al.  (2000) Plague as a Biological Weapon: Medical and Public Health Management. JAMA  283: 2281–2290.  <ul class="find" data-citedArticleID="930966" data-doi="10.1001/jama.283.17.2281"><li><a href="http://dx.doi.org/10.1001/jama.283.17.2281" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plague+as+a+Biological+Weapon%3A+Medical+and+Public+Health+Management." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plague+as+a+Biological+Weapon%3A+Medical+and+Public+Health+Management.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000220-Rotz1" id="pone.0000220-Rotz1"></a>Rotz LD, Khan AS, Lillibridge SR, Ostroff SM, Hughes JM (2002) Public health assessment of potential biological terrorism agents. Emerging Infectious Diseases  8: 225–230.  <ul class="find" data-citedArticleID="930984" data-doi="10.3201/eid0802.010164"><li><a href="http://dx.doi.org/10.3201/eid0802.010164" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Public+health+assessment+of+potential+biological+terrorism+agents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Public+health+assessment+of+potential+biological+terrorism+agents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000220-Achtman1" id="pone.0000220-Achtman1"></a>Achtman M (2004)  Age, descent and genetic diversity within Yersinia pestis. In: Carniel E, Hinnebusch BJ, editors. Yersinia: Molecular and Cellular Biology,. Norwich, UK: Horizon Bioscience.   <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0000220-Achtman2" id="pone.0000220-Achtman2"></a>Achtman M, Morelli G, Zhu P, Wirth T, Diehl I, et al.  (2004) Microevolution and history of the plague bacillus, Yersinia pestis. PNAS  101: 17837–17842.  <ul class="find" data-citedArticleID="930946" data-doi="10.1073/pnas.0408026101"><li><a href="http://dx.doi.org/10.1073/pnas.0408026101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microevolution+and+history+of+the+plague+bacillus%2C+Yersinia+pestis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microevolution+and+history+of+the+plague+bacillus%2C+Yersinia+pestis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000220-Achtman3" id="pone.0000220-Achtman3"></a>Achtman M, Zurth K, Morelli G, Torrea G, Guiyoule A, et al.  (1999) Yersinia pestis, the cause of plague, is a recently emerged clone of Yersinia pseudotuberculosis. Proceedings Of The National Academy Of Sciences Of The United States Of America  96: 14043–14048.  <ul class="find" data-citedArticleID="930948" data-doi="10.1073/pnas.96.24.14043"><li><a href="http://dx.doi.org/10.1073/pnas.96.24.14043" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yersinia+pestis%2C+the+cause+of+plague%2C+is+a+recently+emerged+clone+of+Yersinia+pseudotuberculosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yersinia+pestis%2C+the+cause+of+plague%2C+is+a+recently+emerged+clone+of+Yersinia+pseudotuberculosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000220-Chain1" id="pone.0000220-Chain1"></a>Chain PSG, Carniel E, Larimer FW, Lamerdin J, Stoutland PO, et al.  (2004) Insights into the evolution of Yersinia pestis through whole-genome comparison with Yersinia pseudotuberculosis. PNAS  101: 13826–13831.  <ul class="find" data-citedArticleID="930950" data-doi="10.1073/pnas.0404012101"><li><a href="http://dx.doi.org/10.1073/pnas.0404012101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+into+the+evolution+of+Yersinia+pestis+through+whole-genome+comparison+with+Yersinia+pseudotuberculosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+into+the+evolution+of+Yersinia+pestis+through+whole-genome+comparison+with+Yersinia+pseudotuberculosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000220-Link1" id="pone.0000220-Link1"></a>Link VB (1955) A history of plague in the United States of America. Public Health Monographs  26: 1–120.  <ul class="find" data-citedArticleID="930974"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+history+of+plague+in+the+United+States+of+America.&amp;auth=&amp;atitle=A+history+of+plague+in+the+United+States+of+America." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+history+of+plague+in+the+United+States+of+America." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+history+of+plague+in+the+United+States+of+America.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000220-Cully1" id="pone.0000220-Cully1"></a>Cully JF Jr., Williams ES (2001) Interspecific comparisons of sylvatic plague in prairie dogs. Journal of Mammalogy  82: 894–905.  <ul class="find" data-citedArticleID="930954" data-doi="10.1644/1545-1542(2001)082<0894:icospi&gt;2.0.co;2"><li><a href="http://dx.doi.org/10.1644/1545-1542(2001)082<0894:icospi&gt;2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interspecific+comparisons+of+sylvatic+plague+in+prairie+dogs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interspecific+comparisons+of+sylvatic+plague+in+prairie+dogs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000220-Parmenter1" id="pone.0000220-Parmenter1"></a>Parmenter RR, Yadav EP, Parmenter CA, Ettestad P, Gage KL (1999) Incidence of plague associated with increased winter-spring precipitation in New Mexico. The American Journal Of Tropical Medicine And Hygiene  61: 814–821.  <ul class="find" data-citedArticleID="930978"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Incidence+of+plague+associated+with+increased+winter-spring+precipitation+in+New+Mexico.&amp;auth=&amp;atitle=Incidence+of+plague+associated+with+increased+winter-spring+precipitation+in+New+Mexico." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Incidence+of+plague+associated+with+increased+winter-spring+precipitation+in+New+Mexico." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Incidence+of+plague+associated+with+increased+winter-spring+precipitation+in+New+Mexico.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000220-Enscore1" id="pone.0000220-Enscore1"></a>Enscore RE, Biggerstaff BJ, Brown TL, Fulgham RE, Reynolds PJ, et al.  (2002) Modeling relationships between climate and the frequency of human plague cases in the southwestern United States, 1960–1997. The American Journal of Tropical Medicine and Hygiene  66: 186–196.  <ul class="find" data-citedArticleID="930958"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Modeling+relationships+between+climate+and+the+frequency+of+human+plague+cases+in+the+southwestern+United+States%2C+1960%26%23x2013%3B1997.&amp;auth=&amp;atitle=Modeling+relationships+between+climate+and+the+frequency+of+human+plague+cases+in+the+southwestern+United+States%2C+1960%26%23x2013%3B1997." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+relationships+between+climate+and+the+frequency+of+human+plague+cases+in+the+southwestern+United+States%2C+1960%26%23x2013%3B1997." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+relationships+between+climate+and+the+frequency+of+human+plague+cases+in+the+southwestern+United+States%2C+1960%26%23x2013%3B1997.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000220-Girard1" id="pone.0000220-Girard1"></a>Girard JM, Wagner DM, Vogler AJ, Keys C, Allender CJ, et al.  (2004) Differential plague-transmission dynamics determine Yersinia pestis population genetic structure on local, regional, and global scales. Proc Natl Acad Sci U S A  101: 8408–8413.  <ul class="find" data-citedArticleID="930964" data-doi="10.1073/pnas.0401561101"><li><a href="http://dx.doi.org/10.1073/pnas.0401561101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+plague-transmission+dynamics+determine+Yersinia+pestis+population+genetic+structure+on+local%2C+regional%2C+and+global+scales." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+plague-transmission+dynamics+determine+Yersinia+pestis+population+genetic+structure+on+local%2C+regional%2C+and+global+scales.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000220-Parkhill1" id="pone.0000220-Parkhill1"></a>Parkhill J, Wren BW, Thomson NR, Titball RW, Holden MT, et al.  (2001) Genome sequence of Yersinia pestis, the causative agent of plague. Nature  413: 523–527.  <ul class="find" data-citedArticleID="930976" data-doi="10.1038/35097083"><li><a href="http://dx.doi.org/10.1038/35097083" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+Yersinia+pestis%2C+the+causative+agent+of+plague." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+Yersinia+pestis%2C+the+causative+agent+of+plague.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000220-Jaffe1" id="pone.0000220-Jaffe1"></a>Jaffe DB, Butler J, Gnerre S, Mauceli E, Lindblad-Toh K, et al.  (2003) Whole-genome sequence assembly for mammalian genomes: Arachne 2. Genome Res  13: 91–96.  <ul class="find" data-citedArticleID="930968" data-doi="10.1101/gr.828403"><li><a href="http://dx.doi.org/10.1101/gr.828403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+sequence+assembly+for+mammalian+genomes%3A+Arachne+2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+sequence+assembly+for+mammalian+genomes%3A+Arachne+2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000220-Delcher1" id="pone.0000220-Delcher1"></a>Delcher AL, Phillippy A, Carlton J, Salzberg SL (2002) Fast algorithms for large-scale genome alignment and comparison. Nucleic Acids Res  30: 2478–2483.  <ul class="find" data-citedArticleID="930956" data-doi="10.1093/nar/30.11.2478"><li><a href="http://dx.doi.org/10.1093/nar/30.11.2478" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fast+algorithms+for+large-scale+genome+alignment+and+comparison." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fast+algorithms+for+large-scale+genome+alignment+and+comparison.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000220-Chain2" id="pone.0000220-Chain2"></a>Chain PS, Hu P, Malfatti SA, Radnedge L, Larimer F, et al.  (2006) Complete genome sequence of Yersinia pestis strains Antiqua and Nepal516: evidence of gene reduction in an emerging pathogen. J Bacteriol  188: 4453–4463.  <ul class="find" data-citedArticleID="930952" data-doi="10.1128/jb.00124-06"><li><a href="http://dx.doi.org/10.1128/jb.00124-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+Yersinia+pestis+strains+Antiqua+and+Nepal516%3A+evidence+of+gene+reduction+in+an+emerging+pathogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+Yersinia+pestis+strains+Antiqua+and+Nepal516%3A+evidence+of+gene+reduction+in+an+emerging+pathogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000220-Pearson1" id="pone.0000220-Pearson1"></a>Pearson T, Busch JD, Ravel J, Read TD, Rhoton SD, et al.  (2004) Phylogenetic discovery bias in Bacillus anthracis using single-nucleotide polymorphisms from whole-genome sequencing. Proc Natl Acad Sci U S A  101: 13536–13541.  <ul class="find" data-citedArticleID="930980" data-doi="10.1073/pnas.0403844101"><li><a href="http://dx.doi.org/10.1073/pnas.0403844101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+discovery+bias+in+Bacillus+anthracis+using+single-nucleotide+polymorphisms+from+whole-genome+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+discovery+bias+in+Bacillus+anthracis+using+single-nucleotide+polymorphisms+from+whole-genome+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000220-Thorstenson1" id="pone.0000220-Thorstenson1"></a>Thorstenson YR, Hunicke-Smith SP, Oefner PJ, Davis RW (1998) An automated hydrodynamic process for controlled, unbiased DNA shearing. Genome Res  8: 848–855.  <ul class="find" data-citedArticleID="930988"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+automated+hydrodynamic+process+for+controlled%2C+unbiased+DNA+shearing.&amp;auth=&amp;atitle=An+automated+hydrodynamic+process+for+controlled%2C+unbiased+DNA+shearing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+automated+hydrodynamic+process+for+controlled%2C+unbiased+DNA+shearing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+automated+hydrodynamic+process+for+controlled%2C+unbiased+DNA+shearing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000220-Ewing1" id="pone.0000220-Ewing1"></a>Ewing B, Hillier L, Wendl MC, Green P (1998) Base-calling of automated sequencer traces using phred. I. Accuracy assessment. Genome Res  8: 175–185.  <ul class="find" data-citedArticleID="930960" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred.+I.+Accuracy+assessment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.+I.+Accuracy+assessment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000220-Schwartz1" id="pone.0000220-Schwartz1"></a>Schwartz S, Elnitski L, Li M, Weirauch M, Riemer C, et al.  (2003) MultiPipMaker and supporting tools: alignments and analysis of multiple genomic DNA sequences. Nucleic Acids Res  31: 1–8.  <ul class="find" data-citedArticleID="930986" data-doi="10.1093/nar/gkg579"><li><a href="http://dx.doi.org/10.1093/nar/gkg579" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MultiPipMaker+and+supporting+tools%3A+alignments+and+analysis+of+multiple+genomic+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MultiPipMaker+and+supporting+tools%3A+alignments+and+analysis+of+multiple+genomic+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000220-Ewing2" id="pone.0000220-Ewing2"></a>Ewing B, Green P (1998) Base-calling of automated sequencer traces using phred. II. Error probabilities. Genome Res  8: 186–194.  <ul class="find" data-citedArticleID="930962" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000220-Kurtz1" id="pone.0000220-Kurtz1"></a>Kurtz S, Schleiermacher C (1999) REPuter: fast computation of maximal repeats in complete genomes. Bioinformatics  15: 426–427.  <ul class="find" data-citedArticleID="930972" data-doi="10.1093/bioinformatics/15.5.426"><li><a href="http://dx.doi.org/10.1093/bioinformatics/15.5.426" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=REPuter%3A+fast+computation+of+maximal+repeats+in+complete+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22REPuter%3A+fast+computation+of+maximal+repeats+in+complete+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000220-Kumar1" id="pone.0000220-Kumar1"></a>Kumar S, Tamura K, Jakobsen IB, Nei M (2001) MEGA2: molecular evolutionary genetics analysis software. Bioinformatics  17: 1244–1245.  <ul class="find" data-citedArticleID="930970" data-doi="10.1093/bioinformatics/17.12.1244"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.12.1244" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA2%3A+molecular+evolutionary+genetics+analysis+software." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA2%3A+molecular+evolutionary+genetics+analysis+software.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.XML" value="78251"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.PDF" value="91911"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t001.PNG_L" value="218113"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t001.PNG_M" value="76925"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t001.PNG_S" value="4207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t001.TIF" value="965644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t001.PNG_I" value="31764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t002.PNG_L" value="258407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t002.PNG_M" value="87139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t002.PNG_S" value="3665"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t002.TIF" value="1180152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t002.PNG_I" value="37086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.g001.PNG_L" value="4970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.g001.PNG_M" value="5005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.g001.PNG_S" value="1677"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.g001.TIF" value="9032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.g001.PNG_I" value="3149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t003.PNG_L" value="74978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t003.PNG_M" value="29053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t003.PNG_S" value="2440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t003.TIF" value="433642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000220.t003.PNG_I" value="13297"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000220&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000220" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000220&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000220&volume=&issue=&title=A North American Yersinia pestis Draft Genome Sequence: SNPs and Phylogenetic Analysis&author_name=Jeffrey%20W.%20Touchman%2C%20David%20M.%20Wagner%2C%20Jicheng%20Hao%2C%20Stephen%20D.%20Mastrian%2C%20Maulik%20K.%20Shah%2C%20Amy%20J.%20Vogler%2C%20Christopher%20J.%20Allender%2C%20Erin%20A.%20Clark%2C%20Debbie%20S.%20Benitez%2C%20David%20J.%20Youngkin%2C%20Jessica%20M.%20Girard%2C%20Raymond%20K.%20Auerbach%2C%20Stephen%20M.%20Beckstrom-Sternberg%2C%20Paul%20Keim&start_page=1&end_page=5" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000220" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000220" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000220" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000220&amp;t=A%20North%20American%20Yersinia%20pestis%20Draft%20Genome%20Sequence%3A%20SNPs%20and%20Phylogenetic%20Analysis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000220&title=A%20North%20American%20Yersinia%20pestis%20Draft%20Genome%20Sequence%3A%20SNPs%20and%20Phylogenetic%20Analysis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000220&amp;title=A%20North%20American%20Yersinia%20pestis%20Draft%20Genome%20Sequence%3A%20SNPs%20and%20Phylogenetic%20Analysis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000220" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000220&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A North American Yersinia pestis Draft Genome Sequence: SNPs and Phylogenetic Analysis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000220';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20North%20American%20Yersinia%20pestis%20Draft%20Genome%20Sequence%3A%20SNPs%20and%20Phylogenetic%20Analysis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000220" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000220" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Comparative+genomics%22" title="Search for articles in the subject area:'Comparative genomics'"><div class="flagText">Comparative genomics</div></a>
              <div data-categoryid="32937" data-articleid="23278"
                   data-categoryname="Comparative genomics"
                   class="flagImage" title="Flag 'Comparative genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23278"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Invertebrate+genomics%22" title="Search for articles in the subject area:'Invertebrate genomics'"><div class="flagText">Invertebrate genomics</div></a>
              <div data-categoryid="17221" data-articleid="23278"
                   data-categoryname="Invertebrate genomics"
                   class="flagImage" title="Flag 'Invertebrate genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="23278"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plagues%22" title="Search for articles in the subject area:'Plagues'"><div class="flagText">Plagues</div></a>
              <div data-categoryid="50361" data-articleid="23278"
                   data-categoryname="Plagues"
                   class="flagImage" title="Flag 'Plagues' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="23278"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yersinia+pestis%22" title="Search for articles in the subject area:'Yersinia pestis'"><div class="flagText">Yersinia pestis</div></a>
              <div data-categoryid="44253" data-articleid="23278"
                   data-categoryname="Yersinia pestis"
                   class="flagImage" title="Flag 'Yersinia pestis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yersinia+pseudotuberculosis%22" title="Search for articles in the subject area:'Yersinia pseudotuberculosis'"><div class="flagText">Yersinia pseudotuberculosis</div></a>
              <div data-categoryid="44243" data-articleid="23278"
                   data-categoryname="Yersinia pseudotuberculosis"
                   class="flagImage" title="Flag 'Yersinia pseudotuberculosis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3750'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9595'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1111&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
