

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Identification of Prophages in Bacterial Genomes by Dinucleotide Relative Abundance Difference</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001193"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001193" />

    <meta name="citation_title" content="Identification of Prophages in Bacterial Genomes by Dinucleotide Relative Abundance Difference"/>
    <meta itemprop="name" content="Identification of Prophages in Bacterial Genomes by Dinucleotide Relative Abundance Difference"/>

      <meta name="citation_author" content="K. V. Srividhya"/>
            <meta name="citation_author_institution" content="Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India"/>
      <meta name="citation_author" content="V. Alaguraj"/>
            <meta name="citation_author_institution" content="Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India"/>
      <meta name="citation_author" content="G. Poornima"/>
            <meta name="citation_author_institution" content="Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India"/>
      <meta name="citation_author" content="Dinesh Kumar"/>
            <meta name="citation_author_institution" content="Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India"/>
      <meta name="citation_author" content="G. P. Singh"/>
            <meta name="citation_author_institution" content="Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India"/>
      <meta name="citation_author" content="L. Raghavenderan"/>
            <meta name="citation_author_institution" content="Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India"/>
      <meta name="citation_author" content="A. V. S. K. Mohan Katta"/>
            <meta name="citation_author_institution" content="Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India"/>
      <meta name="citation_author" content="Preeti Mehta"/>
            <meta name="citation_author_institution" content="Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India"/>
      <meta name="citation_author" content="S. Krishnaswamy"/>
            <meta name="citation_author_institution" content="Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India"/>

    <meta name="citation_date" content="2007/11/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001193.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1193"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genetic Variation molecular mechanisms and impact on microbial evolution.; citation_author=W Arber; citation_journal_title=FEMS Microbiol; citation_volume=24; citation_number=1; citation_pages=1-7; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Horizontal gene transfer and bacterial diversity.; citation_author=D Chitra; citation_author=P Archana; citation_journal_title=J Biosci; citation_volume=27; citation_number=2; citation_pages=27-33; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Bacteriophages and pathogenicity: more than just providing a toxin?; citation_author=CR Tinsley; citation_author=E Bille; citation_author=X Nassif; citation_journal_title=Microbes Infect; citation_volume=8; citation_number=3; citation_pages=1365-1371; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Lysogeny from Encyclopedia of life sciences,; citation_author=A Campbell; citation_number=4; citation_pages=1-6; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Prophages and bacterial genomics: what have we learned so far?; citation_author=S Casjens; citation_journal_title=Mol Microbiol; citation_volume=49; citation_number=5; citation_pages=277-300; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phages and the evolution of bacterial pathogens: from genomic rearrangements to lysogenic conversion.; citation_author=H Brussow; citation_author=C Canchaya; citation_author=WD Hardt; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=68; citation_number=6; citation_pages=560-602; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phage-host interaction: an ecological perspective.; citation_author=S Chibani-Chennoufi; citation_author=A Bruttin; citation_author=ML Dillmann; citation_author=H Brussow; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=7; citation_pages=3677-3686; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Prophage genomics.; citation_author=C Canchaya; citation_author=C Proux; citation_author=G Fournous; citation_author=A Bruttin; citation_author=H Brussow; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=67; citation_number=8; citation_pages=238-276; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Diversification of Escherichia coli genomes: are bacteriophages the major contributors?; citation_author=M Ohnishi; citation_author=K Kurokawa; citation_author=T Hayashi; citation_journal_title=Trends Microbiol; citation_volume=9; citation_number=9; citation_pages=481-485; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evolutionary relationships among diverse bacteriophages and prophages: all the world's a phage.; citation_author=RW Hendrix; citation_author=MC Smith; citation_author=RN Burns; citation_author=ME Ford; citation_author=GF Hatfull; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=10; citation_pages=2192-2197; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The impact of prophages on bacterial chromosomes.; citation_author=C Canchaya; citation_author=G Fournous; citation_author=H Brussow; citation_journal_title=Mol Microbiol; citation_volume=53; citation_number=11; citation_pages=9-18; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phage as agents of lateral gene transfer.; citation_author=C Canchaya; citation_author=G Fournous; citation_author=S Chibani-Chennoufi; citation_author=ML Dillmann; citation_author=H Brussow; citation_journal_title=Curr Opin Microbiol; citation_volume=6; citation_number=12; citation_pages=417-424; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Bacteriophage control of bacterial virulence.; citation_author=PL Wagner; citation_author=MK Waldor; citation_journal_title=Infect Immun; citation_volume=70; citation_number=13; citation_pages=3985-3993; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Bacteriophage-bacteriophage interactions in the evolution of pathogenic bacteria.; citation_author=EF Boyd; citation_author=BM Davis; citation_author=B Hochhut; citation_journal_title=Trends Microbiol; citation_volume=9; citation_number=14; citation_pages=137-144; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Comparative genomic analyses of the vibrio pathogenicity island and cholera toxin prophage regions in nonepidemic serogroup strains of Vibrio cholerae.; citation_author=M Li; citation_author=M Kotetishvili; citation_author=Y Chen; citation_author=S Sozhamannan; citation_journal_title=Appl Environ Microbiol; citation_volume=69; citation_number=15; citation_pages=1728-1738; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular analyses of a putative CTXphi precursor and evidence for independent acquisition of distinct CTX(phi)s by toxigenic Vibrio cholerae.; citation_author=EF Boyd; citation_author=AJ Heilpern; citation_author=MK Waldor; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=16; citation_pages=5530-5538; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of bacterial pathogenicity.; citation_author=JW Wilson; citation_author=MJ Schurr; citation_author=CL LeBlanc; citation_author=R Ramamurthy; citation_author=KL Buchanan; citation_author=CA Nickerson; citation_journal_title=Postgrad Med J; citation_volume=78; citation_number=17; citation_pages=216-224; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Ecological fitness, genomic islands and bacterial pathogenicity. A Darwinian view of the evolution of microbes.; citation_author=J Hacker; citation_author=E Carniel; citation_journal_title=EMBO Rep; citation_volume=2; citation_number=18; citation_pages=376-381; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Pathogenicity islands in bacterial pathogenesis.; citation_author=H Schmidt; citation_author=M Hensel; citation_journal_title=Clin Microbiol Rev; citation_volume=17; citation_number=19; citation_pages=14-56; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Pathogenicity islands of virulent bacteria: structure, function and impact on microbial evolution.; citation_author=J Hacker; citation_author=G Blum-Oehler; citation_author=I Muhldorfer; citation_author=H Tschape; citation_journal_title=Mol Microbiol; citation_volume=23; citation_number=20; citation_pages=1089-1097; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The social evolution of bacterial pathogenesis.; citation_author=J Smith; citation_journal_title=Proc Biol Sci; citation_volume=268; citation_number=21; citation_pages=61-69; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A computational approach for identifying pathogenicity islands in prokaryotic genomes.; citation_author=SH Yoon; citation_author=CG Hur; citation_author=HY Kang; citation_author=YH Kim; citation_author=TK Oh; citation_author=JF Kim; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=22; citation_pages=184; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=VFDB: a reference database for bacterial virulence factors.; citation_author=L Chen; citation_author=J Yang; citation_author=J Yu; citation_author=Z Yao; citation_author=L Sun; citation_author=Y Shen; citation_author=Q Jin; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=23; citation_pages=D325-D328; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene Transfer is major factor in gene evolution.; citation_author=R Lan; citation_author=PR Reeves; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=24; citation_pages=47-55; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Global dinucleotide signatures and analysis of genomic heterogeneity.; citation_author=S Karlin; citation_journal_title=Curr Opin Microbiol; citation_volume=1; citation_number=25; citation_pages=598-610; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Dinucleotide relative abundance extremes: a genomic signature.; citation_author=S Karlin; citation_author=C Burge; citation_journal_title=Trends Genet; citation_volume=11; citation_number=26; citation_pages=283-290; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes.; citation_author=S Garcia-Vallve; citation_author=E Guzman; citation_author=MA Montero; citation_author=A Romeu; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=27; citation_pages=187-189; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Heterogeneity of genomes: measures and values.; citation_author=S Karlin; citation_author=I Ladunga; citation_author=BE Blaisdell; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=28; citation_pages=12837-12841; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Similarities and dissimilarities of phage genomes.; citation_author=BE Blaisdell; citation_author=AM Campbell; citation_author=S Karlin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=29; citation_pages=5854-5859; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Compositional biases of bacterial genomes and evolutionary implications.; citation_author=S Karlin; citation_author=J Mrazek; citation_author=AM Campbell; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=30; citation_pages=3899-3913; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The relative abundance of dinucleotides in transposable elements in five species.; citation_author=E Lerat; citation_author=P Capy; citation_author=C Biemont; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=31; citation_pages=964-967; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Analysis of the lambdoid prophage element e14 in the E. coli K-12 genome.; citation_author=P Mehta; citation_author=S Casjens; citation_author=S Krishnaswamy; citation_journal_title=BMC Microbiol; citation_volume=4; citation_number=32; citation_pages=4; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A protein similarity approach for detecting prophage regions in bacterial genomes.; citation_author=GV Rao; citation_author=P Mehta; citation_author=KV Srividhya; citation_author=S Krishnaswamy; citation_journal_title=Genome Biology; citation_volume=6; citation_number=33; citation_pages=p11; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Database and Comparative Identification of prophages.; citation_author=KV Srividhya; citation_author=GeetaV Rao; citation_author=L Raghavenderan; citation_author=Preeti Mehta; citation_author=Jaime Pirulsky; citation_author=Sankarnarayanan Manicka; citation_author=JoelL Sussman; citation_author=S Krishnaswamy; citation_journal_title=LNCIS; citation_volume=344; citation_number=34; citation_pages=863-868; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Prophage Finder: a prophage loci prediction tool for prokaryotic genome sequences.; citation_author=M Bose; citation_author=RD Barber; citation_journal_title=In Silico Biol; citation_volume=6; citation_number=35; citation_pages=223-227; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phage_Finder: automated identification and classification of prophage regions in complete bacterial genome sequences.; citation_author=DE Fouts; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=36; citation_pages=5839-5851; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Bacillus anthracis chromosome contains four conserved, excision-proficient, putative prophages.; citation_author=S Sozhamannan; citation_author=MD Chute; citation_author=FD McAfee; citation_author=DE Fouts; citation_author=A Akmal; citation_author=DR Galloway; citation_author=A Mateczun; citation_author=LW Baillie; citation_author=TD Read; citation_journal_title=BMC Microbiol; citation_volume=6; citation_number=37; citation_pages=34; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of the enterobacterial phytopathogen Erwinia carotovora subsp.; citation_author= Bell; citation_journal_title=Atroseptica and characterization of virulence factors Proc Natl Acad Sci; citation_volume=101; citation_number=38; citation_pages=11105-11110; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Did the inheritance of a pathogenicity island modify the virulence of Helicobacter pylori?; citation_author=A Covacci; citation_author=S Falkow; citation_author=DE Berg; citation_author=R Rappuoli; citation_journal_title=Trends Microbiol; citation_volume=5; citation_number=39; citation_pages=205-208; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Alteration of a single amino acid residue reverses fosfomycin resistance of recombinant MurA from Mycobacterium tuberculosis; citation_author=AL Koen; citation_author=D Smet; citation_author=K Kempsell; citation_author=A Gallagher; citation_author=K Duncan; citation_author=D Young; citation_journal_title=Microbiology; citation_volume=145; citation_number=40; citation_pages=3177-3184; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Granuloma-specific expression of Mycobacterium virulence proteins from the glycine-rich PE-PGRS family.; citation_author=L Ramakrishnan; citation_author=NA Federspiel; citation_author=S Falkow; citation_journal_title=Science; citation_volume=288; citation_number=41; citation_pages=1436-1439; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Are the PE PGRS proteins of Mycobacterium tuberculosis variable surface antigens?,; citation_author=S Banu; citation_author=N Honore; citation_author=B Saint-Joanis; citation_author=D Philpott; citation_author=MC Prevost; citation_author=ST Cole; citation_journal_title=Mol Microbiol.,; citation_volume=44; citation_number=42; citation_pages=9-19; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of Salmonella enterica serovar Choleraesuis, a highly invasive and resistant zoonotic pathogen.; citation_author=CH Chiu; citation_author=P Tang; citation_author=C Chu; citation_author=S Hu; citation_author=Q Bao; citation_author=J Yu; citation_author=YY Chou; citation_author=HS Wang; citation_author=YS Lee; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=43; citation_pages=1690-1698; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comparison of genome degradation in Paratyphi A and Typhi, human-restricted serovars of Salmonella enterica that cause typhoid.; citation_author=M McClelland; citation_journal_title=Nat Genet; citation_volume=36; citation_number=44; citation_pages=1268-1274; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=“Black holes” and bacterial pathogenicity: a large genomic deletion that enhances the virulence of Shigella spp. and enteroinvasive Escherichia coli.; citation_author=AT Maurelli; citation_author=RE Fernandez; citation_author=CA Bloch; citation_author=CK Rode; citation_author=A Fasano; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=45; citation_pages=3943-3948; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Infectious CTXPhi and the vibrio pathogenicity island prophage in Vibrio mimicus: evidence for recent horizontal transfer between V. mimicus and V. cholerae.; citation_author=EF Boyd; citation_author=KE Moyer; citation_author=L Shi; citation_author=MK Waldor; citation_journal_title=Infect Immun; citation_volume=68; citation_number=46; citation_pages=1507-1513; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Computer programs for eukaryotic gene prediction.; citation_author=V Makarov; citation_journal_title=Briefings in Bioinformatics; citation_volume=3; citation_number=47; citation_pages=195-199; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001193" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Identification of Prophages in Bacterial Genomes by Dinucleotide Relative Abundance Difference"/>
    <meta name="twitter:description" content="BackgroundProphages are integrated viral forms in bacterial genomes that have been found to contribute to interstrain genetic variability. Many virulence-associated genes are reported to be prophage encoded. Present computational methods to detect prophages are either by identifying possible essential proteins such as integrases or by an extension of this technique, which involves identifying a region containing proteins similar to those occurring in prophages. These methods suffer due to the problem of low sequence similarity at the protein level, which suggests that a nucleotide based approach could be useful.MethodologyEarlier dinucleotide relative abundance (DRA) have been used to identify regions, which deviate from the neighborhood areas, in genomes. We have used the difference in the dinucleotide relative abundance (DRAD) between the bacterial and prophage DNA to aid location of DNA stretches that could be of prophage origin in bacterial genomes. Prophage sequences which deviate from bacterial regions in their dinucleotide frequencies are detected by scanning bacterial genome sequences. The method was validated using a subset of genomes with prophage data from literature reports. A web interface for prophage scan based on this method is available at http://bicmku.in:8082/prophagedb/dra.html. Two hundred bacterial genomes which do not have annotated prophages have been scanned for prophage regions using this method.ConclusionsThe relative dinucleotide distribution difference helps detect prophage regions in genome sequences. The usefulness of this method is seen in the identification of 461 highly probable loci pertaining to prophages which have not been annotated so earlier. This work emphasizes the need to extend the efforts to detect and annotate prophage elements in genome sequences."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001193.g003"/>

  <meta property="og:title" content="Identification of Prophages in Bacterial Genomes by Dinucleotide Relative Abundance Difference" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001193" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1519'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9715'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9526&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001193">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001193" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001193&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Identification of Prophages in Bacterial Genomes by Dinucleotide Relative Abundance Difference
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            K. V. Srividhya, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            V. Alaguraj, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            G. Poornima, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dinesh Kumar, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            G. P. Singh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            L. Raghavenderan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            A. V. S. K. Mohan Katta, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Preeti Mehta, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            S. Krishnaswamy
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:krishna@mrna.tn.nic.in">krishna@mrna.tn.nic.in</a></p>

                <p>Affiliation:
                  Centre of Excellence in Bioinformatics, School of Biotechnology, Madurai Kamaraj University, Madurai, Tamilnadu, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001193</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001193" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001193" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001193" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001193" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001193-t001" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001193-t002" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001193-g001" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001193-g002" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001193-g003" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001193-t003" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001193-t004" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001193">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001193" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Prophages are integrated viral forms in bacterial genomes that have been found to contribute to interstrain genetic variability. Many virulence-associated genes are reported to be prophage encoded. Present computational methods to detect prophages are either by identifying possible essential proteins such as integrases or by an extension of this technique, which involves identifying a region containing proteins similar to those occurring in prophages. These methods suffer due to the problem of low sequence similarity at the protein level, which suggests that a nucleotide based approach could be useful.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Earlier dinucleotide relative abundance (DRA) have been used to identify regions, which deviate from the neighborhood areas, in genomes. We have used the difference in the dinucleotide relative abundance (DRAD) between the bacterial and prophage DNA to aid location of DNA stretches that could be of prophage origin in bacterial genomes. Prophage sequences which deviate from bacterial regions in their dinucleotide frequencies are detected by scanning bacterial genome sequences. The method was validated using a subset of genomes with prophage data from literature reports. A web interface for prophage scan based on this method is available at <a href="http://bicmku.in:8082/prophagedb/dra.html">http://bicmku.in:8082/prophagedb/dra.htm​l</a>. Two hundred bacterial genomes which do not have annotated prophages have been scanned for prophage regions using this method.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The relative dinucleotide distribution difference helps detect prophage regions in genome sequences. The usefulness of this method is seen in the identification of 461 highly probable loci pertaining to prophages which have not been annotated so earlier. This work emphasizes the need to extend the efforts to detect and annotate prophage elements in genome sequences.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Srividhya KV, Alaguraj V, Poornima G, Kumar D, Singh GP, et al.  (2007) Identification of Prophages in Bacterial Genomes by Dinucleotide Relative Abundance Difference. PLoS ONE 2(11):
          e1193.
            doi:10.1371/journal.pone.0001193</p><p><strong>Academic Editor: </strong>Joel Sussman, Weizmann Institute of Science, Israel</p><p><strong>Received:</strong> February 20, 2007; <strong>Accepted:</strong> October 27, 2007; <strong>Published:</strong> November 21, 2007</p><p><strong>Copyright:</strong> © 2007 Srividhya et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Bioinformatics facilities provided by Dept of Biotechnology, Govt of India under CoE. CSIR for fellowship to PM, KVS, AVSKKM,UGC for fellowship to VA. The funding organisation had no role in the design and conduct of the study; collection, analysis, interpretation of data; and in the preparation, review or approval of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Bacterial genomes evolve through a variety of process including horizontal gene transfer to survive under selective pressures exerted by the environment <a href="#pone.0001193-Arber1">[1]</a>. Internal modifications of genome by intergenomic homologous recombination and horizontal gene transfer (HGT) (intragenic recombination) have been prime reasons for bacterial genome diversity <a href="#pone.0001193-Chitra1">[2]</a>. Mobile elements are responsible for the transfer of new functions to a bacterial cell and are recognized as important agents in bacterial evolution <a href="#pone.0001193-Tinsley1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Bacteriophages (phage) are intracellular parasites that infect bacteria. Lytic phages upon infecting a cell, reproduce, lyse the cell and release progeny phages. However lysogenic or temperate phages multiply via the lytic cycle or enter a quiescent state in the cell. Prophages comprise of such DNA from phages in the integrated state. Fully functional prophages are capable of excision from the bacterial chromosome, either spontaneously or in response to specific signals particularly arising from damage to the host DNA. These lyse the host cells at some subsequent generation upon induction <a href="#pone.0001193-Campbell1">[4]</a>. Prophages can also be defective (in a state of mutational decay and not induced to lytic growth) or be satellites (not carrying their own structural protein genes but capable of encapsidation by capsid proteins of other virions) <a href="#pone.0001193-Casjens1">[5]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Prophages can affect the fitness of the bacteria to survive. These, as elaborated by Brussow <em>et al</em>., 2004 <a href="#pone.0001193-Brussow1">[6]</a> include (i) lysogenic conversion (ii) genome rearrangements, (iii) gene disruption, (iv) protection from lytic infection, (v) lysis of competing strains and (vi) introduction of new fitness factors (lysogenic conversion, transduction). Prophage–bacterial interaction has also been looked at from an ecological perspective by Chibani-Chennoufi <em>et al</em>., 2004 <a href="#pone.0001193-ChibaniChennoufi1">[7]</a>. Such interaction becomes an essential survival strategy for both the prophage and the bacteria.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Prophages can constitute as much as 10–20% of a bacterium's genome and contribute to interstrain variability. The most extreme case is currently represented by the food pathogen <em>Escherichia coli</em> O157:H7 strain Sakai contains 18 prophage elements which amount to 16% of its total genome content <a href="#pone.0001193-Canchaya1">[8]</a>, <a href="#pone.0001193-Ohnishi1">[9]</a>. Many of these prophages are cryptic and in a state of mutational decay. Around 230 prophages are reported in 51 genomes <a href="#pone.0001193-Casjens1">[5]</a>. Bacteriophages and prophages are major contributors of diversification in microbes <a href="#pone.0001193-Hendrix1">[10]</a>. The impact of prophages on bacterial chromosomes has been reviewed extensively <a href="#pone.0001193-Canchaya2">[11]</a> and it is seen that prophages are key agents for lateral gene transfer <a href="#pone.0001193-Canchaya3">[12]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Prophages harbor virulence factors and pathogenicity islands, thereby playing an important role in the emergence of pathogens <a href="#pone.0001193-Wagner1">[13]</a>, <a href="#pone.0001193-Boyd1">[14]</a>. This was recognized for diphtheria toxins and botulinum toxins, which are phage encoded. Virulence factor pertaining to prophage loci include toxins, pili (fimbriae), adhesins and secretion systems <a href="#pone.0001193-Brussow1">[6]</a>. The CTXphi prophage of <em>Vibrio cholerae</em> encodes pathogenicity islands which it transfers into <em>Vibrio mimicus</em> . It has been pointed out that gain of virulence is not the only mechanism by which pathogenicity develops <a href="#pone.0001193-Li1">[15]</a>, <a href="#pone.0001193-Boyd2">[16]</a>. In the prophage database (<a href="http://bicmku.in:8082">http://bicmku.in:8082</a>) around 15 prophages are seen to encode virulence factors including toxin and adhesins, which contribute to pathogenicity in microbes <a href="#pone.0001193-Wilson1">[17]</a>.</p>

<h4>Prokaryotic genomes and associated fitness islands</h4>
<a id="article1.body1.sec1.sec1.p1" name="article1.body1.sec1.sec1.p1"></a><p>Genomic islands increase the fitness of the bacterium. Such fitness islands are classified into several subtypes, such as ecological islands, saprophytic islands etc., based on their niche. These islands contribute to the host survival in the given environment. In many cases the fitness factor temporarily or permanently resides in the host either providing some benefits (‘Symbiosis islands’) or cause damage (pathogenicity islands (PAIs)) by interacting with living hosts. This flexible gene pool of bacteria is composed of prophages and other mobile elements or regions contrary to the core gene pool which comprises of the chromosomal segments pertaining to bacterial metabolic functions <a href="#pone.0001193-Hacker1">[18]</a>. Pathogenicity islands are being explored quite frequently to understand disease development and evolution of bacterial pathogenesis <a href="#pone.0001193-Schmidt1">[19]</a>. The role of pathogenicity islands in the microbial evolution has been subject to extensive review <a href="#pone.0001193-Hacker2">[20]</a>, <a href="#pone.0001193-Smith1">[21]</a>. Yoon <em>et al</em> 2005 <a href="#pone.0001193-Yoon1">[22]</a> have looked at 148 prokaryotic sequences and identified 77 candidate PAI's by applying a homology based method combined with abnormalities detected in genomic composition. Interestingly the same aspect could be looked at for understanding the evolution of eukaryotes by analyzing regions which deviate from the template DNA signature <a href="#pone.0001193-Hacker1">[18]</a>.</p>
<a id="article1.body1.sec1.sec1.p2" name="article1.body1.sec1.sec1.p2"></a><p>As reported by Brussow <em>et al</em>., 2004 <a href="#pone.0001193-Brussow1">[6]</a>, prophages harbor morons (more DNA), which provide extra fitness to the organism and are retained, imparting the bacterial host with some unique phenotype. Virulence factors have also been associated with prophages <a href="#pone.0001193-Li1">[15]</a>. A database of bacterial virulence factors (VFs) associated with various medically significant bacterial pathogens is available. VFDB summarizes the conventional VFs (toxins, enzymes, cell-surface structures, such as capsular polysaccharides, lipopolysaccharides and outer membrane proteins, secretion machineries, siderophores, catalases, regulators) which directly or indirectly regulate pathogenesis in 16 important bacterial pathogens <a href="#pone.0001193-Chen1">[23]</a>. The mechanism of bacterial pathogenicity mediated by above VFs has been extensively studied by Wilson <em>et al</em> <a href="#pone.0001193-Wilson1">[17]</a>.</p>


<h4>Detection of genome heterogeneity</h4>
<a id="article1.body1.sec1.sec2.p1" name="article1.body1.sec1.sec2.p1"></a><p>Heterogeneity in genomes is represented in many ways. Some of these include local and global variations in GC content, direct and inverted repeats, oligonucleotide relative abundance, genome mosaicism due to HGT, transposition and recombination events. Methods have been developed to identify potential foreign gene acquired by the bacterial genomes through horizontal gene transfer. A direct experimental method is subtractive hybridization. Comprehensive assessment of the extent of lateral gene transfer can be made easily by genomic subtraction, a procedure to enrich sequences that are present in one genome but not in another by using biotinylated subtractor DNA to fish out the target DNA by hybrid formation. Later after several cycles of hybridization with newly added subtractor DNA removes target DNA with sequences present in both target and subtracter strains. The remaining unbound target DNA is enriched in sequences absent in the subtracter DNA. This has been done for detecting lateral gene transfer, for example, in four strains of <strong><em>Salmonella enterica</em></strong> <a href="#pone.0001193-Lan1">[24]</a>. Indirect approaches include assessment of GC content, codon usage pattern and aminoacid usage <a href="#pone.0001193-Karlin1">[25]</a>, and dinucleotide relative abundance <a href="#pone.0001193-Karlin2">[26]</a>. For example, HGT-DB is a repository of all the prokaryotic HGTs detected based on their deviation in G+C content, codon and amino-acid usage from prokaryotic complete genomes <a href="#pone.0001193-GarciaVallve1">[27]</a>. Genome heterogeneity in terms of short oligonucleotide compositional extremes and dinucleotide relative abundance distances between different parts of genomes have been examined by Karlin <em>et al</em>., 1994 <a href="#pone.0001193-Karlin3">[28]</a>. This method focuses on small DNA sequences as an alternative to whole genome comparison methods and provides a meaningful measure of similarities. It has been observed that the dinucleotide relative abundance signature could discriminate local structure specificity more than sequence specificity. Dinucleotide relative abundance values are regarded as a stable property of DNA of an organism <a href="#pone.0001193-Karlin1">[25]</a>. The method has been applied to phage genomes to understand similarities and dissimilarities associated with them. Compositional biases prevalent in bacterial genomes have also been examined by oligonucleotide distribution <a href="#pone.0001193-Blaisdell1">[29]</a>. The significance of dinucleotide signatures in genome heterogeneity has been extensively reviewed by Karlin <em>et al</em> 1997 <a href="#pone.0001193-Karlin4">[30]</a> in three facets namely, extremes of dinucleotide abundance, difference in genomic signatures in prokaryotes and evolution of genomes with respect to genomic signatures. Dinucleotide TA is seen to be under represented in eukaryotic genomes and not in viral and mitochondrial genomes. Contrarily, viral genomes are seen to be CG dinucleotide suppressed <a href="#pone.0001193-Karlin1">[25]</a>. The transposable elements of <em>A thailana, C elegans D melanogoaster, H sapiens, S cerevisiae</em> display a similar pattern of relative abundance of dinucleotides in comparison with their respective host genomes <a href="#pone.0001193-Lerat1">[31]</a>. This principle was extended over to prophage loci detection in microbial genomes.</p>


<h4>Prophage Identification methods in prokaryotic genomes</h4>
<a id="article1.body1.sec1.sec3.p1" name="article1.body1.sec1.sec3.p1"></a><p>Recognizing prophages in bacterial genome sequences is not a straight-forward task as prophage sequences are mosaic and encode many orphan and hypothetical proteins, hence unambiguous identification is difficult. Extensive work has been done for detecting ‘corner stone genes’ for the purpose of identifying prophages in bacterial genomes. Integrases are usually sufficiently conserved to be recognizable. Although most temperate phages have an integrase gene, it is not a necessary and sufficient condition to prove the existence of a prophage <a href="#pone.0001193-Casjens1">[5]</a>. Prophages do harbor some phage virion assembly proteins such as Terminase, Portal protein, Head maturation protease, Coat protein, Tail tape measure protein.</p>
<a id="article1.body1.sec1.sec3.p2" name="article1.body1.sec1.sec3.p2"></a><p>A comprehensive bioinformatic analysis was earlier carried out for the e14 cryptic prophage sequence <a href="#pone.0001193-Mehta1">[32]</a>. This showed that the e14 is modular and shares a large part of its sequence with <em>Shigella flexneri</em> phage SfV <a href="#pone.0001193-Mehta1">[32]</a>. Based on this similarity, the regulatory region including the repressor and Cro proteins and their promoter binding sites were identified. A protein based comparative approach using the COG database as a starting point was carried out to detect new lambdoid prophage like elements in a set of completely sequenced genomes <a href="#pone.0001193-Mehta1">[32]</a>. This protein similarity approach (PSA) was extended by the use of BLAST similarity searches rather than limiting to the COG database <a href="#pone.0001193-Rao1">[33]</a>, <a href="#pone.0001193-Srividhya1">[34]</a>. The PSA method was tested with bacterial genomes having known reports of prophages and then extended to newly sequenced bacteria. A total of 87 prophage loci could be identified from 61 bacteria <a href="#pone.0001193-Rao1">[33]</a>, <a href="#pone.0001193-Srividhya1">[34]</a>. Bose and Barber 2006 <a href="#pone.0001193-Bose1">[35]</a> have implemented prophage loci prediction tool for prokaryotic genome sequences based on BLASTX sequence comparison against phage proteomes. Subsequently, a heuristic automated program proposed by Fouts 2006 <a href="#pone.0001193-Fouts1">[36]</a> for prophage detection enables multiple curation of identified prophage locus by comparison with HMMs of phage proteins and further facilitates sub classification of the identified locus.</p>
<a id="article1.body1.sec1.sec3.p3" name="article1.body1.sec1.sec3.p3"></a><p>Dinucleotide Relative abundance (DRA) approach takes into account the local heterogeneity within the given bacterial genomes. DRA values are reported to remain relatively uniform within a genome and its closely related organisms. On this basis, the collection of sixteen DRA values has been referred to as a genomic signature. Thus local heterogeneity in DRA values has been used to detect alien regions in bacterial genomes <a href="#pone.0001193-Karlin1">[25]</a>. This method has also been applied to phage genomes to understand similarities and dissimilarities associated with them <a href="#pone.0001193-Blaisdell1">[29]</a>. We have modified this approach to detect prophages in bacterial genomes. Putative prophage regions could be identified by finding local regions of bacterial genomes that show significant deviation in dinucleotide abundance relative to the background. However, these regions should also show similar dinucleotide abundance relative to that of a reference set of non redundant prophage sequences relevant for those bacteria. Hence taking a dinucleotide relative abundance difference (DRAD), with reference to the two cases described, improves the ability to detect the deviant regions. Since not all the dinucleotides show variation, an appropriate selection helps to further increase the discrimination of the prophage regions.</p>

</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>A program to detect prophage regions (both functional and prophage remnants or highly defective prophages) was developed based on comparison of DRAD analysis. From a total of 52 genomes, 325 probable prophage loci could be identified. Of these 95 prophage loci were earlier reported in literature (<a href="#pone-0001193-t001">Table 1</a>). The rest 230 were newly identified loci among which 159 were highly probable loci. Details are available at <a href="http://bicmku.in:8082/prophagedb/newprophages.html">http://bicmku.in:8082/prophagedb/newprop​hages.html</a>.</p>
<div class="figure" id="pone-0001193-t001"><div class="img"><a name="pone-0001193-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001193.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001193.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Prophages identified using dinucleotide relative abundance difference method.</span></strong></p><span>doi:10.1371/journal.pone.0001193.t001</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>The sensitivity and specificity of the method was found to average around 82% and 83% respectively (<a href="#pone-0001193-t002">Table 2</a>) but however varied amongst different genomes. Our analysis suggests that the variation is not related to the GC content. The variation is possibly related to the non redundant nature of the prophage set used for the detection.</p>
<div class="figure" id="pone-0001193-t002"><div class="img"><a name="pone-0001193-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001193.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001193.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Sensitivity and Specificity across genomes.</span></strong></p><span>doi:10.1371/journal.pone.0001193.t002</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>A comparison between the prophages identified by our method, those reported by Casjens <a href="#pone.0001193-Casjens1">[5]</a> and a method phage_finder <a href="#pone.0001193-Bose1">[35]</a> shows a common overlap of 47 prophages (<a href="#pone-0001193-g001">Figure 1</a> and <a href="#pone-0001193-g002">Figure 2</a>). The details on the prophage loci reported by different methods are given at <a href="http://bicmku.in:8082/prophagedb/prophage_different_methods.htm">http://bicmku.in:8082/prophagedb/prophag​e_different_methods.htm</a>. The detection of prophages varies between different genomes suggesting that it would be necessary to use more than one method depending on the genome in order to locate all possible prophages. This probably arises from the mosaic nature of prophages.</p>
<div class="figure" id="pone-0001193-g001"><div class="img"><a name="pone-0001193-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001193.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001193.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Comparative analysis of number of prophages identified by the approach reported here (DRAD), literature reports and another prophage detection method (phage_finder tool).</span></strong></p><span>doi:10.1371/journal.pone.0001193.g001</span></div><div class="figure" id="pone-0001193-g002"><div class="img"><a name="pone-0001193-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001193.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001193.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Variation of prophage number with bacterial genomes.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>– Indicated in green are prophages identified by the method reported here (DRAD), yellow and red represents prophage loci reported in literature <a href="#pone.0001193-Casjens1">[5]</a> , identified by phage_finder program <a href="#pone.0001193-Bose1">[35]</a> respectively.</p>
<span>doi:10.1371/journal.pone.0001193.g002</span></div>
<h4>Bacterial genomes with no earlier report of prophages</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The DRAD method was used to examine genome sequences with no reports of prophages. A total of 200 genome sequences were analyzed for prophage elements using this DRAD approach. Out of the 453 loci identified from 84 bacterial genomes, 207 (from 64 genomes) were seen to be highly probable prophage loci, based on the annotation in the protein table files of the corresponding bacterial genomes. The genome of <em>Shigella sonnei</em> had high incidence of thirteen prophages (<a href="#pone-0001193-g003">Figure 3</a>) <a href="http://bicmku.in:8082/prophagedb/patho_prophages.html">http://bicmku.in:8082/prophagedb/patho_p​rophages.html</a>.</p>
<div class="figure" id="pone-0001193-g003"><div class="img"><a name="pone-0001193-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001193.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001193.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Dinucleotide difference distribution for <em>Shigella sonnei:</em></span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>pink-<em>Shigella sonnei</em> genome Vs <em>Shigella sonnei</em> genome, blue-<em>Shigella sonnei</em> genome Vs prophage dataset , yellow- their dinucleotide relative abundance difference (DRAD) value.</p>
<span>doi:10.1371/journal.pone.0001193.g003</span></div>

<h4>Prophages in bacterial genomes with varied ecological niche</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The acquisition of ecological islands by the bacterial host occurs through horizontal gene transfer <a href="#pone.0001193-Hacker1">[18]</a>. A total of 96 prophage loci could be identified form 35 bacterial genomes (<a href="#pone-0001193-t003">Table 3</a>) which grow in extreme ecological niches or are being exploited for industrial production. The detailed loci of the prophages are available at <a href="http://bicmku.in:8082/prophagedb/eco_prophages.html">http://bicmku.in:8082/prophagedb/eco_pro​phages.html</a>.</p>
<div class="figure" id="pone-0001193-t003"><div class="img"><a name="pone-0001193-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001193.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001193.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Prophages associated with bacterial genomes surviving on varied ecological niches/with industrial significance.</span></strong></p><span>doi:10.1371/journal.pone.0001193.t003</span></div>

<h4>Pathogenicity islands and prophages</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The role of bacteriophages contributing to pathogenicity has been reviewed by Tinsley <em>et al</em>., 2006 <a href="#pone.0001193-Tinsley1">[3]</a>. Prophage loci are seen to encode pathogenicity islands. This study showed that in the 29 pathogenic bacterial genomes screened (<a href="#pone-0001193-t004">Table 4</a>), 207 prophage loci were identified. Of these, 111 were seen to encode virulence or fitness factors. Details of the loci are available at <a href="http://bicmku.in:8082/prophagedb/patho_prophages.html">http://bicmku.in:8082/prophagedb/patho_p​rophages.html</a>. The observations suggest that acquisition of virulence genes through horizontally transferred prophages could be a common strategy of microbes undergoing transformation from a commensal to a pathogen. With the availability of bacterial genomes sequences, it is evident that inter-species transmission of genetic information is pervasive in microbes and that parallely acquisition of foreign genes is counter balanced by loss of native genes, in order to maintain genome size within limits.</p>
<div class="figure" id="pone-0001193-t004"><div class="img"><a name="pone-0001193-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001193" data-uri="info:doi/10.1371/journal.pone.0001193.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001193.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001193.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001193.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001193.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Prophage loci, in pathogenic bacteria, identified by the method reported here (DRAD approach) indicated as * are PAIs reported by Yoon <em>et al</em> 2005 <a href="#pone.0001193-Yoon1">[22]</a>.</span></strong></p><span>doi:10.1371/journal.pone.0001193.t004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The DRAD analysis carried out with <em>Bacillus anthracis</em> showed two prophage loci that encode morons (glucosyl transferase). This supplements the report of four prophages being associated in <em>B anthracis</em> by Sozhamannan <em>et al</em>., 2006 <a href="#pone.0001193-Sozhamannan1">[37]</a> . <em>Erwinia carotovora</em> subsp. <em>atroseptica</em> is an important bacterial plant pathogen causing soft rot and blackleg in potato. As a member of the Enterobacteriaceae, it is related to <em>Escherichia</em> and <em>Shigella</em>, <em>Salmonella</em> and <em>Yersinia</em> <a href="#pone.0001193-Bell1">[38]</a>. In this study, <em>Erwinia</em> was found to harbor a total 7 prophages encoding Type IV pilus protein and flagellar proteins. Similarly, in the pathogenic <em>H pylori</em> genome, the DRAD analysis identified prophage loci that encode Cag island proteins which pertain to pathogenicity <a href="#pone.0001193-Covacci1">[39]</a>. The same Cag island has been reported by Yoon <em>et al</em>., 2005 <a href="#pone.0001193-Yoon1">[22]</a> as potential PAI. Moreover, in <em>Chromobacterium violaceum ATCC 12472 , Bordetella pertussis Tohama I, Helicobacter pylori J99, Photorhabdus luminescens TT01 Vibrio parahaemolyticus RIMD 2210633</em> (<a href="#pone-0001193-t004">Table 4</a>) the prophage loci identified by DRAD compare well with the PAIs reported by Yoon <em>et al</em>., 2005 <a href="#pone.0001193-Yoon1">[22]</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In the case of <em>Mycobacterium avium the</em> prophage region detected by DRAD was found to encode MurA, which has been implicated in <em>M. tuberculosis</em> resistance to a range of broad-spectrum antimicrobial agents <a href="#pone.0001193-Koen1">[40]</a>. With <em>Mycobacterium bovis out</em> of three prophages that were detected one was found to harbor PE-PGRS genes, which are a family encoding numerous repetitive glycine-rich proteins of unknown function <a href="#pone.0001193-Ramakrishnan1">[41]</a>. PE-PGRS proteins are reported to be associated with mycobacterial species (<em>M</em>. <em>tuberculosis, M. bovis BCG, M. smegmatis, M. marinum and M. gordonae</em>) and 11 clinical isolates of M. tuberculosis <a href="#pone.0001193-Banu1">[42]</a>. This again highlights the possible contribution of prophages to the virulence of the associated bacterial species.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p><em>Salmonella enterica subsp. enterica serovar Choleraesuis</em> is a highly invasive serovar among non-typhoidal Salmonella that usually causes sepsis or extra-intestinal focal infections in humans <a href="#pone.0001193-Chiu1">[43]</a>. The DRAD analysis of the bacterial genome showed a high incidence of prophages. The loci identified encode Gifsy-2 and Gifsy-1 prophage like proteins. Most of loci encode a few to many fimbrial proteins, surface presentation antigens and secretion system apparatus which are key genes involved in virulence. In the case of <em>Salmonella enterica Paratyphi</em>, a human-restricted serovars of Salmonella enterica causing typhoid <a href="#pone.0001193-McClelland1">[44]</a>, nine prophage loci could be identified and these predominantly encode pathogenicity islands apart form secretion systems.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Maurelli <em>et al</em> 1998 <a href="#pone.0001193-Maurelli1">[45]</a> have reported the role of genomic deletion (of LCD- lysine decarboxylase) contributing to the pathogenicity of <em>Shigella</em> spp. Among <em>Shigella</em> species, <em>S sonnei</em> involved in mucoid diarrhea, 13 highly probable prophage loci could be detected. With all the three species of <em>Shigella</em> (S. s<em>onnei</em>, S.<em>boydii</em> and S.<em>dysenteriae</em>) almost all the loci are associated with insertion sequence elements, from a minimum of 3 to 10. A few of the possible prophage loci are seen to harbor virulence factors like siderophores. In <em>Vibrio parahaemolyticus</em>, the two prophage loci that have been detected (<a href="#pone-0001193-t004">Table 4</a>) encode pilus assembly protein and restriction proteins. Recently, horizontal gene transfer of CTXphi prophage encoded PAIs have been reported between <em>V mimicus</em> and <em>V cholerae</em> <a href="#pone.0001193-Boyd3">[46]</a> indicating that the <em>Vibrios</em> share such virulence associated gene pools.</p>


<h4>Conclusion</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Prophages, including defective ones, can contribute important biological properties to their bacterial hosts. In order to understand completely the nature of the bacterial behavior, one must be able to recognize the full complement of prophages in bacterial genomes. The extreme variability of prophage sequences, as seen by our comparisons, makes it quite possible that unrecognized prophages are still present in bacterial genome sequences (Casjens, 2003)<a href="#pone.0001193-Casjens1">[5]</a> .We have presented a dinucleotide distribution difference method for identification of prophages from microbial genomes sequences. Prophage detection methods such as the one described here based on dinucleotide composition and those earlier reported based on similarity at the protein level tend to supplement each other. With increasing microbial genome sequences being available, consensus methods will probably emerge for identifying potential prophage loci in microbial genomes. These will help explain the prophage mediated evolution of microbes.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The Dinucleotide Relative Abundance (DRA) <a href="#pone.0001193-Karlin3">[28]</a> was modified for prophage detection.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>For a given dinucleotide XY,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <sup>obs</sup>f<sub>XY</sub> is the observed frequency of the dinucleotide XY occurring in a chosen window and <sup>exp</sup>f<sub>XY</sub> is the expected frequency of the nucleotide XY occurring in the reference set.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>DRA<sup>bact</sup> is calculated using the observed dinucleotide frequencies for a window of the bacterial genome and the expected frequencies of the dinucleotide occurring over the entire bacterial genome. The DRA<sup>bact</sup> values using a sliding window are calculated for the entire genome and plotted against the bacterial genome sequence position. DRA<sup>prophage</sup> is calculated using the observed dinucleotide frequencies for a window of the bacterial genome and the expected frequencies of the dinucleotide occurring over the entire prophage reference set. The DRA<sup>prophage</sup> values using a sliding window are calculated for the entire genome and plotted against the bacterial genome sequence position.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001193.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>The DRAD or DRA<sup>diff</sup> is calculated for each window and plotted against the bacterial genome sequence position. Regions of high DRA<sup>diff</sup> values are used to identify possible prophage-like regions. By trial and error, using known prophage regions, a window size of 25000 with a displacement of 1000 was standardized for the screening. Further the hit was annotated as a potential prophage locus and taken as a true positive if the annotation in protein table (ptt) file for the locus had phage associated genes. Those regions without any phage marker genes were considered as false positives. The annotations of peak locus (corresponding to each prophage) were retrieved from protein table file (ptt) of respective bacterial genomes. False negatives includes prophage set not detected by DRA but reported in literature.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The probable specificity (ratio of true positives to the sum of true positives and false positives) and probable sensitivity (ratio of true positives to the sum of true positives and false negatives) were calculated according to Makarov 2002 <a href="#pone.0001193-Makarov1">[47]</a>. The qualifier probable has been added to the specificity and sensitivity measures since the assumption that the data used for validation is complete is not wholly appropriate, as there could be prophages that are yet to be detected. A server for the detection of prophages based on comparison of Dinucleotide Relative Abundance Difference (DRAD or DRA<sup>diff</sup>) values is available at <a href="http://bicmku.in:8082/prophagedb/dra.html">http://bicmku.in:8082/prophagedb/dra.htm​l</a>.</p>

<h4>Data Source</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Bacteria genomes were downloaded from NCBI ftp site (<a href="http://ftp://ftp.ncbi.nih.gov/genomes/Bacteria/">ftp://ftp.ncbi.nih.gov/genomes/Bacteria/</a>). Prophage positions and sequences obtained from supplementary material of Casjens, (2003) <a href="#pone.0001193-Casjens1">[5]</a> are available in the prophage database (<a href="http://bicmku.in:8082/prophagedb, Srividhya et al 2006">http://bicmku.in:8082/prophagedb, Srividhya et al 2006</a>) <a href="#pone.0001193-Srividhya1">[34]</a>. Location of prophages in bacterial genomes was determined by using protein table file (ptt) from NCBI.</p>


<h4>Construction of Non-redundant Prophage set (NRPS)</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>For detection of new prophages in bacterial genomes a set of non redundant prophages was constructed, which includes prophages (without repetition) from 50 bacterial genomes from the prophage database (<a href="http://bicmku.in:8082">http://bicmku.in:8082</a>). This constitutes the NRPS (non-redundant prophage set) which was used for screening for prophages in any given bacterial genome. The list of prophages taken for NRPS generation is listed in <a href="http://bicmku.in:8082/prophagedb/nrlist.html">http://bicmku.in:8082/prophagedb/nrlist.​html</a>.</p>

</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SK PM. Performed the experiments: KS VA GP LR. Analyzed the data: SK KS GP LR DK. Contributed reagents/materials/analysis tools: VA GS PM DK AM. Wrote the paper: SK KS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001193-Arber1" id="pone.0001193-Arber1"></a>Arber W (2000) Genetic Variation molecular mechanisms and impact on microbial evolution. FEMS Microbiol  24: 1–7.  <ul class="find" data-citedArticleID="1019264" data-doi="10.1111/j.1574-6976.2000.tb00529.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6976.2000.tb00529.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+Variation+molecular+mechanisms+and+impact+on+microbial+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+Variation+molecular+mechanisms+and+impact+on+microbial+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001193-Chitra1" id="pone.0001193-Chitra1"></a>Chitra D, Archana P (2002) Horizontal gene transfer and bacterial diversity. J Biosci  27: 27–33.  <ul class="find" data-citedArticleID="1019296" data-doi="10.1007/bf02703681"><li><a href="http://dx.doi.org/10.1007/bf02703681" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Horizontal+gene+transfer+and+bacterial+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Horizontal+gene+transfer+and+bacterial+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001193-Tinsley1" id="pone.0001193-Tinsley1"></a>Tinsley CR, Bille E, Nassif X (2006) Bacteriophages and pathogenicity: more than just providing a toxin? Microbes Infect  8: 1365–1371.  <ul class="find" data-citedArticleID="1019350" data-doi="10.1016/j.micinf.2005.12.013"><li><a href="http://dx.doi.org/10.1016/j.micinf.2005.12.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteriophages+and+pathogenicity%3A+more+than+just+providing+a+toxin%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteriophages+and+pathogenicity%3A+more+than+just+providing+a+toxin%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001193-Campbell1" id="pone.0001193-Campbell1"></a>Campbell A (2001) Lysogeny from Encyclopedia of life sciences,1–6.  <ul class="find" data-citedArticleID="1019282" data-doi="10.1038/npg.els.0000780"><li><a href="http://dx.doi.org/10.1038/npg.els.0000780" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lysogeny+from+Encyclopedia+of+life+sciences%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lysogeny+from+Encyclopedia+of+life+sciences%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001193-Casjens1" id="pone.0001193-Casjens1"></a>Casjens S (2003) Prophages and bacterial genomics: what have we learned so far? Mol Microbiol  49: 277–300.  <ul class="find" data-citedArticleID="1019290" data-doi="10.1046/j.1365-2958.2003.03580.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03580.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prophages+and+bacterial+genomics%3A+what+have+we+learned+so+far%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prophages+and+bacterial+genomics%3A+what+have+we+learned+so+far%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001193-Brussow1" id="pone.0001193-Brussow1"></a>Brussow H, Canchaya C, Hardt WD (2004) Phages and the evolution of bacterial pathogens: from genomic rearrangements to lysogenic conversion. Microbiol Mol Biol Rev  68: 560–602.  <ul class="find" data-citedArticleID="1019280" data-doi="10.1128/mmbr.68.3.560-602.2004"><li><a href="http://dx.doi.org/10.1128/mmbr.68.3.560-602.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phages+and+the+evolution+of+bacterial+pathogens%3A+from+genomic+rearrangements+to+lysogenic+conversion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phages+and+the+evolution+of+bacterial+pathogens%3A+from+genomic+rearrangements+to+lysogenic+conversion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001193-ChibaniChennoufi1" id="pone.0001193-ChibaniChennoufi1"></a>Chibani-Chennoufi S, Bruttin A, Dillmann ML, Brussow H (2004) Phage-host interaction: an ecological perspective. J Bacteriol  186: 3677–3686.  <ul class="find" data-citedArticleID="1019294" data-doi="10.1128/jb.186.12.3677-3686.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.12.3677-3686.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phage-host+interaction%3A+an+ecological+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phage-host+interaction%3A+an+ecological+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001193-Canchaya1" id="pone.0001193-Canchaya1"></a>Canchaya C, Proux C, Fournous G, Bruttin A, Brussow H (2003) Prophage genomics. Microbiol Mol Biol Rev  67: 238–276.  <ul class="find" data-citedArticleID="1019284" data-doi="10.1128/mmbr.67.2.238-276.2003"><li><a href="http://dx.doi.org/10.1128/mmbr.67.2.238-276.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prophage+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prophage+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001193-Ohnishi1" id="pone.0001193-Ohnishi1"></a>Ohnishi M, Kurokawa K, Hayashi T (2001) Diversification of <em>Escherichia coli</em> genomes: are bacteriophages the major contributors? Trends Microbiol  9: 481–485.  <ul class="find" data-citedArticleID="1019336" data-doi="10.1016/s0966-842x(01)02173-4"><li><a href="http://dx.doi.org/10.1016/s0966-842x(01)02173-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversification+of+Escherichia+coli+genomes%3A+are+bacteriophages+the+major+contributors%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversification+of+Escherichia+coli+genomes%3A+are+bacteriophages+the+major+contributors%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001193-Hendrix1" id="pone.0001193-Hendrix1"></a>Hendrix RW, Smith MC, Burns RN, Ford ME, Hatfull GF (1999) Evolutionary relationships among diverse bacteriophages and prophages: all the world's a phage. Proc Natl Acad Sci U S A  96: 2192–2197.  <ul class="find" data-citedArticleID="1019310" data-doi="10.1073/pnas.96.5.2192"><li><a href="http://dx.doi.org/10.1073/pnas.96.5.2192" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+relationships+among+diverse+bacteriophages+and+prophages%3A+all+the+world%27s+a+phage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+relationships+among+diverse+bacteriophages+and+prophages%3A+all+the+world%27s+a+phage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001193-Canchaya2" id="pone.0001193-Canchaya2"></a>Canchaya C, Fournous G, Brussow H (2004) The impact of prophages on bacterial chromosomes. Mol Microbiol  53: 9–18.  <ul class="find" data-citedArticleID="1019286" data-doi="10.1111/j.1365-2958.2004.04113.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04113.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+impact+of+prophages+on+bacterial+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+impact+of+prophages+on+bacterial+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001193-Canchaya3" id="pone.0001193-Canchaya3"></a>Canchaya C, Fournous G, Chibani-Chennoufi S, Dillmann ML, Brussow H (2003) Phage as agents of lateral gene transfer. Curr Opin Microbiol  6: 417–424.  <ul class="find" data-citedArticleID="1019288" data-doi="10.1016/s1369-5274(03)00086-9"><li><a href="http://dx.doi.org/10.1016/s1369-5274(03)00086-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phage+as+agents+of+lateral+gene+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phage+as+agents+of+lateral+gene+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001193-Wagner1" id="pone.0001193-Wagner1"></a>Wagner PL, Waldor MK (2002) Bacteriophage control of bacterial virulence. Infect Immun  70: 3985–3993.  <ul class="find" data-citedArticleID="1019352" data-doi="10.1128/iai.70.8.3985-3993.2002"><li><a href="http://dx.doi.org/10.1128/iai.70.8.3985-3993.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteriophage+control+of+bacterial+virulence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteriophage+control+of+bacterial+virulence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001193-Boyd1" id="pone.0001193-Boyd1"></a>Boyd EF, Davis BM, Hochhut B (2001) Bacteriophage-bacteriophage interactions in the evolution of pathogenic bacteria. Trends Microbiol  9: 137–144.  <ul class="find" data-citedArticleID="1019274" data-doi="10.1016/s0966-842x(01)01960-6"><li><a href="http://dx.doi.org/10.1016/s0966-842x(01)01960-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteriophage-bacteriophage+interactions+in+the+evolution+of+pathogenic+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteriophage-bacteriophage+interactions+in+the+evolution+of+pathogenic+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001193-Li1" id="pone.0001193-Li1"></a>Li M, Kotetishvili M, Chen Y, Sozhamannan S (2003) Comparative genomic analyses of the vibrio pathogenicity island and cholera toxin prophage regions in nonepidemic serogroup strains of <em>Vibrio cholerae</em>. Appl Environ Microbiol  69: 1728–1738.  <ul class="find" data-citedArticleID="1019326" data-doi="10.1128/aem.69.3.1728-1738.2003"><li><a href="http://dx.doi.org/10.1128/aem.69.3.1728-1738.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomic+analyses+of+the+vibrio+pathogenicity+island+and+cholera+toxin+prophage+regions+in+nonepidemic+serogroup+strains+of+Vibrio+cholerae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomic+analyses+of+the+vibrio+pathogenicity+island+and+cholera+toxin+prophage+regions+in+nonepidemic+serogroup+strains+of+Vibrio+cholerae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001193-Boyd2" id="pone.0001193-Boyd2"></a>Boyd EF, Heilpern AJ, Waldor MK (2000) Molecular analyses of a putative CTXphi precursor and evidence for independent acquisition of distinct CTX(phi)s by toxigenic <em>Vibrio cholerae</em>. J Bacteriol  182: 5530–5538.  <ul class="find" data-citedArticleID="1019276" data-doi="10.1128/jb.182.19.5530-5538.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.19.5530-5538.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+analyses+of+a+putative+CTXphi+precursor+and+evidence+for+independent+acquisition+of+distinct+CTX%28phi%29s+by+toxigenic+Vibrio+cholerae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+analyses+of+a+putative+CTXphi+precursor+and+evidence+for+independent+acquisition+of+distinct+CTX%28phi%29s+by+toxigenic+Vibrio+cholerae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001193-Wilson1" id="pone.0001193-Wilson1"></a>Wilson JW, Schurr MJ, LeBlanc CL, Ramamurthy R, Buchanan KL, Nickerson CA (2002) Mechanisms of bacterial pathogenicity. Postgrad Med J  78: 216–224.  <ul class="find" data-citedArticleID="1019354"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mechanisms+of+bacterial+pathogenicity.&amp;auth=&amp;atitle=Mechanisms+of+bacterial+pathogenicity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+bacterial+pathogenicity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+bacterial+pathogenicity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001193-Hacker1" id="pone.0001193-Hacker1"></a>Hacker J, Carniel E (2001) Ecological fitness, genomic islands and bacterial pathogenicity. A Darwinian view of the evolution of microbes. EMBO Rep  2: 376–381.  <ul class="find" data-citedArticleID="1019306" data-doi="10.1093/embo-reports/kve097"><li><a href="http://dx.doi.org/10.1093/embo-reports/kve097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecological+fitness%2C+genomic+islands+and+bacterial+pathogenicity.+A+Darwinian+view+of+the+evolution+of+microbes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecological+fitness%2C+genomic+islands+and+bacterial+pathogenicity.+A+Darwinian+view+of+the+evolution+of+microbes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001193-Schmidt1" id="pone.0001193-Schmidt1"></a>Schmidt H, Hensel M (2004) Pathogenicity islands in bacterial pathogenesis. Clin Microbiol Rev  17: 14–56.  <ul class="find" data-citedArticleID="1019342" data-doi="10.1128/cmr.17.1.14-56.2004"><li><a href="http://dx.doi.org/10.1128/cmr.17.1.14-56.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathogenicity+islands+in+bacterial+pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathogenicity+islands+in+bacterial+pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001193-Hacker2" id="pone.0001193-Hacker2"></a>Hacker J, Blum-Oehler G, Muhldorfer I, Tschape H (1997) Pathogenicity islands of virulent bacteria: structure, function and impact on microbial evolution. Mol Microbiol  23: 1089–1097.  <ul class="find" data-citedArticleID="1019308" data-doi="10.1046/j.1365-2958.1997.3101672.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1997.3101672.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathogenicity+islands+of+virulent+bacteria%3A+structure%2C+function+and+impact+on+microbial+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathogenicity+islands+of+virulent+bacteria%3A+structure%2C+function+and+impact+on+microbial+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001193-Smith1" id="pone.0001193-Smith1"></a>Smith J (2001) The social evolution of bacterial pathogenesis. Proc Biol Sci  268: 61–69.  <ul class="find" data-citedArticleID="1019344" data-doi="10.1098/rspb.2000.1330"><li><a href="http://dx.doi.org/10.1098/rspb.2000.1330" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+social+evolution+of+bacterial+pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+social+evolution+of+bacterial+pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001193-Yoon1" id="pone.0001193-Yoon1"></a>Yoon SH, Hur CG, Kang HY, Kim YH, Oh TK, Kim JF (2005) A computational approach for identifying pathogenicity islands in prokaryotic genomes. BMC Bioinformatics  6: 184.  <ul class="find" data-citedArticleID="1019356"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+computational+approach+for+identifying+pathogenicity+islands+in+prokaryotic+genomes.&amp;auth=&amp;atitle=A+computational+approach+for+identifying+pathogenicity+islands+in+prokaryotic+genomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+computational+approach+for+identifying+pathogenicity+islands+in+prokaryotic+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+computational+approach+for+identifying+pathogenicity+islands+in+prokaryotic+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001193-Chen1" id="pone.0001193-Chen1"></a>Chen L, Yang J, Yu J, Yao Z, Sun L, Shen Y, Jin Q (2005) VFDB: a reference database for bacterial virulence factors. Nucleic Acids Res  33: D325–D328.  <ul class="find" data-citedArticleID="1019292" data-doi="10.1093/nar/gki008"><li><a href="http://dx.doi.org/10.1093/nar/gki008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=VFDB%3A+a+reference+database+for+bacterial+virulence+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22VFDB%3A+a+reference+database+for+bacterial+virulence+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001193-Lan1" id="pone.0001193-Lan1"></a>Lan R, Reeves PR (1996) Gene Transfer is major factor in gene evolution. Mol Biol Evol  13: 47–55.  <ul class="find" data-citedArticleID="1019322" data-doi="10.1093/oxfordjournals.molbev.a025569"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025569" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+Transfer+is+major+factor+in+gene+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+Transfer+is+major+factor+in+gene+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001193-Karlin1" id="pone.0001193-Karlin1"></a>Karlin S (1998) Global dinucleotide signatures and analysis of genomic heterogeneity. Curr Opin Microbiol  1: 598–610.  <ul class="find" data-citedArticleID="1019312" data-doi="10.1016/s1369-5274(98)80095-7"><li><a href="http://dx.doi.org/10.1016/s1369-5274(98)80095-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+dinucleotide+signatures+and+analysis+of+genomic+heterogeneity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+dinucleotide+signatures+and+analysis+of+genomic+heterogeneity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001193-Karlin2" id="pone.0001193-Karlin2"></a>Karlin S, Burge C (1995) Dinucleotide relative abundance extremes: a genomic signature. Trends Genet  11: 283–290.  <ul class="find" data-citedArticleID="1019314" data-doi="10.1016/s0168-9525(00)89076-9"><li><a href="http://dx.doi.org/10.1016/s0168-9525(00)89076-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dinucleotide+relative+abundance+extremes%3A+a+genomic+signature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dinucleotide+relative+abundance+extremes%3A+a+genomic+signature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001193-GarciaVallve1" id="pone.0001193-GarciaVallve1"></a>Garcia-Vallve S, Guzman E, Montero MA, Romeu A (2003) HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes. Nucleic Acids Res  31: 187–189.  <ul class="find" data-citedArticleID="1019304" data-doi="10.1093/nar/gkg004"><li><a href="http://dx.doi.org/10.1093/nar/gkg004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HGT-DB%3A+a+database+of+putative+horizontally+transferred+genes+in+prokaryotic+complete+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HGT-DB%3A+a+database+of+putative+horizontally+transferred+genes+in+prokaryotic+complete+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001193-Karlin3" id="pone.0001193-Karlin3"></a>Karlin S, Ladunga I, Blaisdell BE (1994) Heterogeneity of genomes: measures and values. Proc Natl Acad Sci U S A  91: 12837–12841.  <ul class="find" data-citedArticleID="1019316" data-doi="10.1073/pnas.91.26.12837"><li><a href="http://dx.doi.org/10.1073/pnas.91.26.12837" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterogeneity+of+genomes%3A+measures+and+values." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterogeneity+of+genomes%3A+measures+and+values.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001193-Blaisdell1" id="pone.0001193-Blaisdell1"></a>Blaisdell BE, Campbell AM, Karlin S (1996) Similarities and dissimilarities of phage genomes. Proc Natl Acad Sci U S A  93: 5854–5859.  <ul class="find" data-citedArticleID="1019270" data-doi="10.1073/pnas.93.12.5854"><li><a href="http://dx.doi.org/10.1073/pnas.93.12.5854" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Similarities+and+dissimilarities+of+phage+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Similarities+and+dissimilarities+of+phage+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001193-Karlin4" id="pone.0001193-Karlin4"></a>Karlin S, Mrazek J, Campbell AM (1997) Compositional biases of bacterial genomes and evolutionary implications. J Bacteriol  179: 3899–3913.  <ul class="find" data-citedArticleID="1019318"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Compositional+biases+of+bacterial+genomes+and+evolutionary+implications.&amp;auth=&amp;atitle=Compositional+biases+of+bacterial+genomes+and+evolutionary+implications." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compositional+biases+of+bacterial+genomes+and+evolutionary+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compositional+biases+of+bacterial+genomes+and+evolutionary+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001193-Lerat1" id="pone.0001193-Lerat1"></a>Lerat E, Capy P, Biemont C (2002) The relative abundance of dinucleotides in transposable elements in five species. Mol Biol Evol  19: 964–967.  <ul class="find" data-citedArticleID="1019324" data-doi="10.1093/oxfordjournals.molbev.a004154"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004154" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+relative+abundance+of+dinucleotides+in+transposable+elements+in+five+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+relative+abundance+of+dinucleotides+in+transposable+elements+in+five+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001193-Mehta1" id="pone.0001193-Mehta1"></a>Mehta P, Casjens S, Krishnaswamy S (2004) Analysis of the lambdoid prophage element e14 in the <em>E. coli K-12</em> genome. BMC Microbiol  4: 4.  <ul class="find" data-citedArticleID="1019334"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+the+lambdoid+prophage+element+e14+in+the+E.+coli+K-12+genome.&amp;auth=&amp;atitle=Analysis+of+the+lambdoid+prophage+element+e14+in+the+E.+coli+K-12+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+lambdoid+prophage+element+e14+in+the+E.+coli+K-12+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+lambdoid+prophage+element+e14+in+the+E.+coli+K-12+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001193-Rao1" id="pone.0001193-Rao1"></a>Rao GV, Mehta P, Srividhya KV, Krishnaswamy S (2005) A protein similarity approach for detecting prophage regions in bacterial genomes. Genome Biology  6: p11.  <ul class="find" data-citedArticleID="1019340" data-doi="10.1186/gb-2005-6-10-p11"><li><a href="http://dx.doi.org/10.1186/gb-2005-6-10-p11" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+protein+similarity+approach+for+detecting+prophage+regions+in+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+protein+similarity+approach+for+detecting+prophage+regions+in+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001193-Srividhya1" id="pone.0001193-Srividhya1"></a>Srividhya KV, Rao GeetaV, Raghavenderan L, Mehta Preeti, Pirulsky Jaime, Manicka Sankarnarayanan, Sussman JoelL, Krishnaswamy S (2006) Database and Comparative Identification of prophages. LNCIS  344: 863–868.  <ul class="find" data-citedArticleID="1019348" data-doi="10.1007/978-3-540-37256-1_110"><li><a href="http://dx.doi.org/10.1007/978-3-540-37256-1_110" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Database+and+Comparative+Identification+of+prophages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Database+and+Comparative+Identification+of+prophages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001193-Bose1" id="pone.0001193-Bose1"></a>Bose M, Barber RD (2006) Prophage Finder: a prophage loci prediction tool for prokaryotic genome sequences. In Silico Biol  6: 223–227.  <ul class="find" data-citedArticleID="1019272"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Prophage+Finder%3A+a+prophage+loci+prediction+tool+for+prokaryotic+genome+sequences.&amp;auth=&amp;atitle=Prophage+Finder%3A+a+prophage+loci+prediction+tool+for+prokaryotic+genome+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prophage+Finder%3A+a+prophage+loci+prediction+tool+for+prokaryotic+genome+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prophage+Finder%3A+a+prophage+loci+prediction+tool+for+prokaryotic+genome+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001193-Fouts1" id="pone.0001193-Fouts1"></a>Fouts DE (2006) Phage_Finder: automated identification and classification of prophage regions in complete bacterial genome sequences. Nucleic Acids Res  34: 5839–5851.  <ul class="find" data-citedArticleID="1019302" data-doi="10.1093/nar/gkl732"><li><a href="http://dx.doi.org/10.1093/nar/gkl732" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phage_Finder%3A+automated+identification+and+classification+of+prophage+regions+in+complete+bacterial+genome+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phage_Finder%3A+automated+identification+and+classification+of+prophage+regions+in+complete+bacterial+genome+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001193-Sozhamannan1" id="pone.0001193-Sozhamannan1"></a>Sozhamannan S, Chute MD, McAfee FD, Fouts DE, Akmal A, Galloway DR, Mateczun A, Baillie LW, Read TD (2006) The <em>Bacillus anthracis</em> chromosome contains four conserved, excision-proficient, putative prophages. BMC Microbiol  6: 34.  <ul class="find" data-citedArticleID="1019346" data-doi="10.1186/1471-2180-6-34"><li><a href="http://dx.doi.org/10.1186/1471-2180-6-34" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Bacillus+anthracis+chromosome+contains+four+conserved%2C+excision-proficient%2C+putative+prophages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Bacillus+anthracis+chromosome+contains+four+conserved%2C+excision-proficient%2C+putative+prophages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001193-Bell1" id="pone.0001193-Bell1"></a>Bell , et al.  (2004) Genome sequence of the enterobacterial phytopathogen <em>Erwinia carotovora</em> subsp. <em>Atroseptica</em> and characterization of virulence factors Proc Natl Acad Sci  101: 11105–11110.  <ul class="find" data-citedArticleID="1019268" data-doi="10.1073/pnas.0402424101"><li><a href="http://dx.doi.org/10.1073/pnas.0402424101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+the+enterobacterial+phytopathogen+Erwinia+carotovora+subsp." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+the+enterobacterial+phytopathogen+Erwinia+carotovora+subsp.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001193-Covacci1" id="pone.0001193-Covacci1"></a>Covacci A, Falkow S, Berg DE, Rappuoli R (1997) Did the inheritance of a pathogenicity island modify the virulence of <em>Helicobacter pylori</em>? Trends Microbiol  5: 205–208.  <ul class="find" data-citedArticleID="1019300" data-doi="10.1016/s0966-842x(97)01035-4"><li><a href="http://dx.doi.org/10.1016/s0966-842x(97)01035-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Did+the+inheritance+of+a+pathogenicity+island+modify+the+virulence+of+Helicobacter+pylori%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Did+the+inheritance+of+a+pathogenicity+island+modify+the+virulence+of+Helicobacter+pylori%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001193-Koen1" id="pone.0001193-Koen1"></a>Koen AL, Smet D, Kempsell K, Gallagher A, Duncan K, Young D (1999) Alteration of a single amino acid residue reverses fosfomycin resistance of recombinant MurA from <em>Mycobacterium tuberculosis</em> Microbiology  145: 3177–3184.  <ul class="find" data-citedArticleID="1019320"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Alteration+of+a+single+amino+acid+residue+reverses+fosfomycin+resistance+of+recombinant+MurA+from+Mycobacterium+tuberculosis&amp;auth=&amp;atitle=Alteration+of+a+single+amino+acid+residue+reverses+fosfomycin+resistance+of+recombinant+MurA+from+Mycobacterium+tuberculosis" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alteration+of+a+single+amino+acid+residue+reverses+fosfomycin+resistance+of+recombinant+MurA+from+Mycobacterium+tuberculosis" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alteration+of+a+single+amino+acid+residue+reverses+fosfomycin+resistance+of+recombinant+MurA+from+Mycobacterium+tuberculosis%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001193-Ramakrishnan1" id="pone.0001193-Ramakrishnan1"></a>Ramakrishnan L, Federspiel NA, Falkow S (2000) Granuloma-specific expression of Mycobacterium virulence proteins from the glycine-rich PE-PGRS family. Science  288: 1436–1439.  <ul class="find" data-citedArticleID="1019338" data-doi="10.1126/science.288.5470.1436"><li><a href="http://dx.doi.org/10.1126/science.288.5470.1436" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Granuloma-specific+expression+of+Mycobacterium+virulence+proteins+from+the+glycine-rich+PE-PGRS+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Granuloma-specific+expression+of+Mycobacterium+virulence+proteins+from+the+glycine-rich+PE-PGRS+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001193-Banu1" id="pone.0001193-Banu1"></a>Banu S, Honore N, Saint-Joanis B, Philpott D, Prevost MC, Cole ST (2002) Are the PE PGRS proteins of <em>Mycobacterium tuberculosis</em> variable surface antigens?, Mol Microbiol.,  44: 9–19.  <ul class="find" data-citedArticleID="1019266" data-doi="10.1046/j.1365-2958.2002.02813.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.02813.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Are+the+PE+PGRS+proteins+of+Mycobacterium+tuberculosis+variable+surface+antigens%3F%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Are+the+PE+PGRS+proteins+of+Mycobacterium+tuberculosis+variable+surface+antigens%3F%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001193-Chiu1" id="pone.0001193-Chiu1"></a>Chiu CH, Tang P, Chu C, Hu S, Bao Q, Yu J, Chou YY, Wang HS, Lee YS (2005) The genome sequence of <em>Salmonella enterica serovar Choleraesuis</em>, a highly invasive and resistant zoonotic pathogen. Nucleic Acids Res  33: 1690–1698.  <ul class="find" data-citedArticleID="1019298" data-doi="10.1093/nar/gki297"><li><a href="http://dx.doi.org/10.1093/nar/gki297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+Salmonella+enterica+serovar+Choleraesuis%2C+a+highly+invasive+and+resistant+zoonotic+pathogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+Salmonella+enterica+serovar+Choleraesuis%2C+a+highly+invasive+and+resistant+zoonotic+pathogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001193-McClelland1" id="pone.0001193-McClelland1"></a>McClelland M, et al.  (2004) Comparison of genome degradation in Paratyphi A and Typhi, human-restricted serovars of <em>Salmonella enterica</em> that cause typhoid. Nat Genet  36: 1268–1274.  <ul class="find" data-citedArticleID="1019332" data-doi="10.1038/ng1470"><li><a href="http://dx.doi.org/10.1038/ng1470" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+genome+degradation+in+Paratyphi+A+and+Typhi%2C+human-restricted+serovars+of+Salmonella+enterica+that+cause+typhoid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+genome+degradation+in+Paratyphi+A+and+Typhi%2C+human-restricted+serovars+of+Salmonella+enterica+that+cause+typhoid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001193-Maurelli1" id="pone.0001193-Maurelli1"></a>Maurelli AT, Fernandez RE, Bloch CA, Rode CK, Fasano A (1998) “Black holes” and bacterial pathogenicity: a large genomic deletion that enhances the virulence of <em>Shigella spp</em>. and enteroinvasive <em>Escherichia coli</em>. Proc Natl Acad Sci U S A  95: 3943–3948.  <ul class="find" data-citedArticleID="1019330" data-doi="10.1073/pnas.95.7.3943"><li><a href="http://dx.doi.org/10.1073/pnas.95.7.3943" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%E2%80%9CBlack+holes%E2%80%9D+and+bacterial+pathogenicity%3A+a+large+genomic+deletion+that+enhances+the+virulence+of+Shigella+spp.+and+enteroinvasive+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%E2%80%9CBlack+holes%E2%80%9D+and+bacterial+pathogenicity%3A+a+large+genomic+deletion+that+enhances+the+virulence+of+Shigella+spp.+and+enteroinvasive+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001193-Boyd3" id="pone.0001193-Boyd3"></a>Boyd EF, Moyer KE, Shi L, Waldor MK (2000) Infectious CTXPhi and the vibrio pathogenicity island prophage in Vibrio mimicus: evidence for recent horizontal transfer between <em>V. mimicus</em> and <em>V. cholerae</em>. Infect Immun  68: 1507–1513.  <ul class="find" data-citedArticleID="1019278" data-doi="10.1128/iai.68.3.1507-1513.2000"><li><a href="http://dx.doi.org/10.1128/iai.68.3.1507-1513.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Infectious+CTXPhi+and+the+vibrio+pathogenicity+island+prophage+in+Vibrio+mimicus%3A+evidence+for+recent+horizontal+transfer+between+V.+mimicus+and+V.+cholerae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Infectious+CTXPhi+and+the+vibrio+pathogenicity+island+prophage+in+Vibrio+mimicus%3A+evidence+for+recent+horizontal+transfer+between+V.+mimicus+and+V.+cholerae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001193-Makarov1" id="pone.0001193-Makarov1"></a>Makarov V (2002) Computer programs for eukaryotic gene prediction. Briefings in Bioinformatics  3: 195–199.  <ul class="find" data-citedArticleID="1019328" data-doi="10.1093/bib/3.2.195"><li><a href="http://dx.doi.org/10.1093/bib/3.2.195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computer+programs+for+eukaryotic+gene+prediction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computer+programs+for+eukaryotic+gene+prediction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.XML" value="119145"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.PDF" value="302282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t001.PNG_L" value="420547"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t001.PNG_M" value="200760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t001.PNG_S" value="18165"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t001.TIF" value="1825694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t001.PNG_I" value="69357"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t002.PNG_L" value="208188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t002.PNG_M" value="115966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t002.PNG_S" value="13406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t002.TIF" value="906402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t002.PNG_I" value="35440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g001.PNG_L" value="336671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g001.PNG_M" value="70934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g001.PNG_S" value="8975"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g001.TIF" value="696346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g001.PNG_I" value="31312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g002.PNG_L" value="1118734"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g002.PNG_M" value="124328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g002.PNG_S" value="10300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g002.TIF" value="3087604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g002.PNG_I" value="43457"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g003.PNG_L" value="861741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g003.PNG_M" value="138743"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g003.PNG_S" value="9185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g003.TIF" value="3678548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.g003.PNG_I" value="49114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t003.PNG_L" value="248198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t003.PNG_M" value="138465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t003.PNG_S" value="12961"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t003.TIF" value="985030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t003.PNG_I" value="40669"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t004.PNG_L" value="453987"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t004.PNG_M" value="244074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t004.PNG_S" value="18583"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t004.TIF" value="1782310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.t004.PNG_I" value="72178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.e001.PNG" value="8749"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.e001.TIF" value="24768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.e002.PNG" value="8045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.e002.TIF" value="24008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.e003.PNG" value="9504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001193.e003.TIF" value="25704"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001193&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001193" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001193&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001193&volume=&issue=&title=Identification of Prophages in Bacterial Genomes by Dinucleotide Relative Abundance Difference&author_name=K.%20V.%20Srividhya%2C%20V.%20Alaguraj%2C%20G.%20Poornima%2C%20Dinesh%20Kumar%2C%20G.%20P.%20Singh%2C%20L.%20Raghavenderan%2C%20A.%20V.%20S.%20K.%20Mohan%20Katta%2C%20Preeti%20Mehta%2C%20S.%20Krishnaswamy&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001193" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001193" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001193" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001193&amp;t=Identification%20of%20Prophages%20in%20Bacterial%20Genomes%20by%20Dinucleotide%20Relative%20Abundance%20Difference" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001193&title=Identification%20of%20Prophages%20in%20Bacterial%20Genomes%20by%20Dinucleotide%20Relative%20Abundance%20Difference&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001193&amp;title=Identification%20of%20Prophages%20in%20Bacterial%20Genomes%20by%20Dinucleotide%20Relative%20Abundance%20Difference" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001193" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001193&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Identification of Prophages in Bacterial Genomes by Dinucleotide Relative Abundance Difference';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001193';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Identification%20of%20Prophages%20in%20Bacterial%20Genomes%20by%20Dinucleotide%20Relative%20Abundance%20Difference http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001193" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001193" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="25218"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+pathogens%22" title="Search for articles in the subject area:'Bacterial pathogens'"><div class="flagText">Bacterial pathogens</div></a>
              <div data-categoryid="42717" data-articleid="25218"
                   data-categoryname="Bacterial pathogens"
                   class="flagImage" title="Flag 'Bacterial pathogens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacteriophages%22" title="Search for articles in the subject area:'Bacteriophages'"><div class="flagText">Bacteriophages</div></a>
              <div data-categoryid="43083" data-articleid="25218"
                   data-categoryname="Bacteriophages"
                   class="flagImage" title="Flag 'Bacteriophages' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25218"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="25218"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+analysis%22" title="Search for articles in the subject area:'Genome analysis'"><div class="flagText">Genome analysis</div></a>
              <div data-categoryid="40539" data-articleid="25218"
                   data-categoryname="Genome analysis"
                   class="flagImage" title="Flag 'Genome analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="25218"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="25218"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1263'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2431'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7376&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
