

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Tempo and Mode of Diversification of Lake Tanganyika Cichlid Fishes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001730"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001730" />

    <meta name="citation_title" content="Tempo and Mode of Diversification of Lake Tanganyika Cichlid Fishes"/>
    <meta itemprop="name" content="Tempo and Mode of Diversification of Lake Tanganyika Cichlid Fishes"/>

      <meta name="citation_author" content="Julia J. Day"/>
            <meta name="citation_author_institution" content="Department of Biology, University College London, London, United Kingdom"/>
            <meta name="citation_author_institution" content="Department of Zoology, The Natural History Museum, London, United Kingdom"/>
      <meta name="citation_author" content="James A. Cotton"/>
            <meta name="citation_author_institution" content="Department of Zoology, The Natural History Museum, London, United Kingdom"/>
            <meta name="citation_author_institution" content="School of Biological and Chemical Sciences, Queen Mary, University of London, London, United Kingdom"/>
      <meta name="citation_author" content="Timothy G. Barraclough"/>
            <meta name="citation_author_institution" content="Division of Biology and NERC Centre for Population Biology, Imperial College London, Silwood Park Campus, Ascot, United Kingdom"/>

    <meta name="citation_date" content="2008/3/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001730.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1730"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The Major Features of Evolution.; citation_author=GG Simpson; citation_number=1; citation_date=1953; citation_publisher=Columbia University Press; " />
      <meta name="citation_reference" content="citation_title=The Ecology of Adaptive Radiation.; citation_author=D Schluter; citation_number=2; citation_date=2000; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Getting the measure of biodiversity.; citation_author=A Purvis; citation_author=A Hector; citation_journal_title=Nature; citation_volume=405; citation_number=3; citation_pages=212-219; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Revealing the factors that promote speciation.; citation_author=TG Barraclough; citation_author=AP Vogler; citation_author=PH Harvey; citation_journal_title=Phil Trans R Soc London B; citation_volume=353; citation_number=4; citation_pages=241-249; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Speciation.; citation_author=J Coyne; citation_author=HA Orr; citation_number=5; citation_date=2004; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=Testing for latitudinal bias in diversification rates: An example using New World birds.; citation_author=M Cardillo; citation_author=CDL Orme; citation_author=IPF Owens; citation_journal_title=Ecology; citation_volume=86; citation_number=6; citation_pages=2278-2287; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Darwin's abominable mystery: Insights from a supertree of the angiosperms.; citation_author=TJ Davies; citation_author=TG Barraclough; citation_author=MW Chase; citation_author=PS Soltis; citation_author=DE Soltis; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=101; citation_number=7; citation_pages=1904-1909; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Global diversification rates of passerine birds.; citation_author=RE Ricklefs; citation_journal_title=Proc R Soc London B; citation_volume=270; citation_number=8; citation_pages=2285-2291; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Environment, area, and diversification in the species-rich flowering plant family Iridaceae.; citation_author=TJ Davies; citation_author=V Savolainen; citation_author=MW Chase; citation_author=P Goldblatt; citation_author=TG Barraclough; citation_journal_title=Am Nat; citation_volume=166; citation_number=9; citation_pages=418-425; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=How does the taxonomic status of allopatric populations influence species richness within African cichlid fish assemblages?; citation_author=MJ Genner; citation_author=O Seehausen; citation_author=DFR Cleary; citation_author=ME Knight; citation_author=E Michel; citation_journal_title=J Biogeogr; citation_volume=31; citation_number=10; citation_pages=93-102; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Late Pleistocene desiccation of Lake Victoria and rapid evolution of cichlid fishes.; citation_author=TC Johnson; citation_author=CA Scholz; citation_author=MR Talbot; citation_author=K Kelts; citation_author=RD Ricketts; citation_journal_title=Science; citation_volume=273; citation_number=11; citation_pages=1091-1093; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Age of Lake Malawi (Nyasa) and water level fluctuations.; citation_author=D Delvaux; citation_journal_title=Mus R Afr Centr Tervuren (Belg) Dept Geol Min Rapp Ann; citation_volume=1995–1996; citation_number=12; citation_pages=99-108; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Effects of sexual selection and life history on speciation: Species flocks in African cichlids and Hawaiian Drosophila.; citation_author=WJ Dominey; citation_number=13; citation_pages=231-250; citation_date=1984; citation_publisher=University Maine at Orono Press; " />
      <meta name="citation_reference" content="citation_title=Speciation in rapidly diverging systems: lessons from Lake Malawi.; citation_author=PD Danley; citation_author=TD Kocher; citation_journal_title=Mol Ecol; citation_volume=10; citation_number=14; citation_pages=1075-1086; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Laboratory mating trials indicate incipient speciation by sexual selection among populations of the cichlid fish Pseudotropheus zebra from Lake Malawi.; citation_author=GF Knight ME Turner; citation_journal_title=Proc. R. Soc. London B.; citation_volume=271; citation_number=15; citation_pages=675-680; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The relative rate of sympatric and allopatric speciation in fishes: tests using DNA sequence divergence between sister species and among clades.; citation_author=AR McCune; citation_author=NR Lovejoy; citation_number=16; citation_pages=172-185; citation_date=1998; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Explosive speciation of African cichlids.; citation_author=GF Turner; citation_number=17; citation_pages=113-129; citation_date=1999; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Estimating the age of formation of lakes - an example from Lake Tanganyika, East-African rift system.; citation_author=AS Cohen; citation_author=MJ Soreghan; citation_author=CA Scholz; citation_journal_title=Geology; citation_volume=21; citation_number=18; citation_pages=511-514; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=New palaeogeographic and lake-level reconstructions of Lake Tanganyika: Implications for tectonic, climatic and biological evolution in a rift lake.; citation_author=AS Cohen; citation_author=KE Lezzar; citation_author=JJ Tiercelin; citation_author=M Soreghan; citation_journal_title=Basin Res; citation_volume=9; citation_number=19; citation_pages=107-132; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Systematics of Tanganyikan cichlids fishes (Teleostei: Perciformes).; citation_author=T Takahashi; citation_journal_title=Ichthy Res; citation_volume=50; citation_number=20; citation_pages=367-382; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Out of Tanganyika: Genesis, explosive speciation, key-innovations and phylogeography of the haplochromine cichlid fishes.; citation_author=W Salzburger; citation_author=T Mack; citation_author=E Verheyen; citation_author=A Meyer; citation_journal_title=BMC Evol Biol; citation_volume=5; citation_number=21; citation_pages=17; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phylogenetics and speciation.; citation_author=TG Barraclough; citation_author=S Nee; citation_journal_title=Trends Ecol Evol; citation_volume=16; citation_number=22; citation_pages=391-399; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=African cichlid fish: a model system in adaptive radiation research.; citation_author=O Seehausen; citation_journal_title=Proc R Soc London B; citation_volume=273; citation_number=23; citation_pages=1987-1998; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Closing of the Indonesian seaway as a precursor to East African aridification around 3–4 million years ago.; citation_author=MA Cane; citation_author=P Molnar; citation_journal_title=Nature; citation_volume=411; citation_number=24; citation_pages=157-162; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The geology of the Tanganyika trough.; citation_author=JJ Tiercelin; citation_author=A Mondeguer; citation_number=25; citation_pages=7-48; citation_date=1991; citation_publisher=Oxford University Press, Oxford; " />
      <meta name="citation_reference" content="citation_title=Comparative phylogenetic analyses of the adaptive radiation of Lake Tanganyika cichlid fish: Nuclear sequences are less homoplasious but also less informative than mitochondrial DNA.; citation_author=C Clabaut; citation_author=W Salzburger; citation_author=A Meyer; citation_journal_title=J Mol Evol; citation_volume=61; citation_number=26; citation_pages=666-681; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Age of cichlids: New dates for ancient lake fish radiations.; citation_author=MJ Genner; citation_author=O Seehausen; citation_author=DH Lunt; citation_author=DA Joyce; citation_author=PW Shaw; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=27; citation_pages=1269-1282; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic relationships and evolution of Tanganyikan cichlids: a molecular perspective.; citation_author=M Nishida; citation_number=28; citation_pages=3-23; citation_date=1997; citation_publisher=Kyoto University Press; " />
      <meta name="citation_reference" content="citation_title=Using SINEs to probe ancient explosive speciation: “Hidden” radiation of African cichlids?; citation_author=Y Terai; citation_author=K Takahashi; citation_author=M Nishida; citation_author=T Sato; citation_author=N Okada; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=29; citation_pages=924-930; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Lake level fluctuations synchronize genetic divergences of cichlid fishes in African lakes.; citation_author=W Salzburger; citation_author=A Meyer; citation_author=S Baric; citation_author=E Verheyen; citation_author=C Strumbauer; citation_journal_title=Syst Biol; citation_volume=1; citation_number=30; citation_pages=1-23; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Relaxed phylogenetics and dating with confidence.; citation_author=AJ Drummond; citation_author=SYW Ho; citation_author=MJ Phillips; citation_author=A Rambaut; citation_journal_title=PLOS Biol; citation_volume=4; citation_number=31; citation_pages=699-710; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The reconstructed evolutionary process.; citation_author=S Nee; citation_author=RM May; citation_author=PH Harvey; citation_journal_title=Phil Trans R Soc Lond B; citation_volume=344; citation_number=32; citation_pages=305-311; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Age and rate of diversification of the Hawaiian silversword alliance (Compositae); citation_author=BG Baldwin; citation_author=MJ Sanderson; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=95; citation_number=33; citation_pages=9402-9406; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=How fast is speciation: molecular, geological and phylogenetic evidence from adaptive radiations of fishes.; citation_author=AR McCune; citation_number=34; citation_pages=585-610; citation_date=1997; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=The origin and age of haplochromine fishes in Lake Victoria, East Africa.; citation_author=S Nagl; citation_author=H Tichy; citation_author=WE Mayer; citation_author=N Takezaki; citation_author=N Takahata; citation_journal_title=Proc R Soc London B; citation_volume=267; citation_number=35; citation_pages=1049-1061; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Macroevolutionary inferences from primate phylogeny.; citation_author=A Purvis; citation_author=S Nee; citation_author=PH Harvey; citation_journal_title=Proc R Soc London B; citation_volume=260; citation_number=36; citation_pages=329-333; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Inferring speciation rates from phylogenies.; citation_author=S Nee; citation_journal_title=Evolution; citation_volume=55; citation_number=37; citation_pages=661-668; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Recent diversification rates in North American tiger beetles estimated from a dated mtDNA phylogenetic tree.; citation_author=TG Barraclough; citation_author=AP Vogler; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=38; citation_pages=1706-1716; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Tanganyika Cichlids in their Natural Habitat.; citation_author=A Konings; citation_number=39; citation_date=1998; citation_publisher=Cichlid Press; " />
      <meta name="citation_reference" content="citation_title=Habitat use in cichlid fishes for breeding.; citation_author=Y Yanagisawa; citation_author=H Ochi; citation_author=MM Gashagaza; citation_number=40; citation_pages=151-173; citation_date=1997; citation_publisher=Kyoto University Press; " />
      <meta name="citation_reference" content="citation_title=Dynamic patterns of adaptive radiation.; citation_author=S Gavrilets; citation_author=A Vose; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=41; citation_pages=18040-18045; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Monophyletic origin of Lake Victoria cichlid fishes suggested by mitochondrial-DNA sequences.; citation_author=A Meyer; citation_author=TD Kocher; citation_author=P Basasibwaki; citation_author= Wilson; citation_journal_title=Nature; citation_volume=347; citation_number=42; citation_pages=550-553; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Lake level fluctuations synchronize genetic divergences of cichlid fishes in African lakes.; citation_author=C Sturmbauer; citation_author=S Baric; citation_author=W Salzburger; citation_author=L Rüber; citation_author=E Verheyen; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=43; citation_pages=144-154; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Testing macro-evolutionary models using incomplete molecular phylogenies.; citation_author=OG Pybus; citation_author=PH Harvey; citation_journal_title=Proc. R. Soc. Lond. B; citation_volume=267; citation_number=44; citation_pages=2267-2272; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The delayed rise of present-day mammals.; citation_author=ORP Binda-Emonds; citation_author=M Cardillo; citation_author=KE Jones; citation_author=RDE MacPhee; citation_author=RMD Beck; citation_journal_title=Nature; citation_volume=446; citation_number=45; citation_pages=507-512; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=On the origin of the Synodontis catfish species flock from Lake Tanganyika.; citation_author=JJ Day; citation_author=M Wilkinson; citation_journal_title=Biol. Letts; citation_volume=2; citation_number=46; citation_pages=548-552; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular evidence for recent divergence of Lake Tanganyika endemic crabs (Decapoda : Platythelphusidae).; citation_author=SAE Marijnissen; citation_author=E Michel; citation_author=SR Daniels; citation_author=D Erpenbeck; citation_author=SBJ Menken; citation_journal_title=Mol. Phylogenet. Evol.; citation_volume=40; citation_number=47; citation_pages=628-634; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cichlid fish diversity threatened by eutrophication that curbs sexual selection.; citation_author=O Seehausen; citation_author=JJM van Alphen; citation_author=F Witte; citation_journal_title=Science; citation_volume=277; citation_number=48; citation_pages=1808-1811; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Comparison of thermal stratification, light attenuation, and chlorophyll-a dynamics between the ends of Lake Tanganyika.; citation_author=VT Langenberg; citation_author=LM Mwape; citation_author=K Tschibangu; citation_author=J-M Tumba; citation_author=AA Koelmans; citation_journal_title=J Aquat Ecosyst Health; citation_volume=5; citation_number=49; citation_pages=255-265; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny of the Carnivora (Mammalia): Assessing the impact of increased sampling on resolving enigmatic relationships.; citation_author=JJ Flynn; citation_author=JA Finarelli; citation_author=S Zehr; citation_author=J Hsu; citation_author=MA Nedbal; citation_journal_title=Syst Biol; citation_volume=54; citation_number=50; citation_pages=317-337; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Increased taxon sampling greatly reduces phylogenetic error.; citation_author=DJ Zwickl; citation_author=DM Hills; citation_journal_title=Syst Biol; citation_volume=51; citation_number=51; citation_pages=588-598; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.; citation_author=JD Thompson; citation_author=TJ Gibson; citation_author=F Plewniak; citation_author=J Jeanmougin; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=52; citation_pages=4876-4882; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Se-Al: Sequence Alignment Editor.; citation_author=A Rambaut; citation_number=53; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=MODELTEST: testing the model of DNA substitution.; citation_author=D Posada; citation_author=K Crandall; citation_journal_title=Bioinform; citation_volume=14; citation_number=54; citation_pages=817-818; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinform; citation_volume=19; citation_number=55; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetic Algorithm Approaches for the Phylogenetic Analysis of Large Biological Sequence Datasets under the Maximum Likelihood Criterion. Ph.D. Dissertation, University of Texas at Austin.; citation_author=DJ Zwickl; citation_number=56; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An approximately unbiased test of phylogenetic tree selection.; citation_author=H Shimodaira; citation_journal_title=Syst Biol; citation_volume=51; citation_number=57; citation_pages=492-508; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=CONSEL: for assessing the confidence of phylogenetic tree selection.; citation_author=H Shimodaira; citation_author=M Hasegawa; citation_journal_title=Bioinform; citation_volume=17; citation_number=58; citation_pages=1246-1247; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=PAUP*: Phylogentic analysis using parsimony (* and other methods), version 4.0b10; citation_author=DL Swofford; citation_number=59; citation_date=2002; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=Phylogeny estimation and hypothesis testing using maximum likelihood.; citation_author=JP Huelsenbeck; citation_author=KA Crandall; citation_journal_title=Ann Rev Ecol Syst; citation_volume=28; citation_number=60; citation_pages=437-466; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=BEAST: Bayesian evolutionary analysis by sampling trees.; citation_author=AJ Drummond; citation_author=A Rambaut; citation_journal_title=BMC Evol Biol; citation_volume=7; citation_number=61; citation_pages=214; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Vicariant patterns and historical explanation in biogeography.; citation_author=DE Rosen; citation_journal_title=Syst Zool; citation_volume=27; citation_number=62; citation_pages=159-188; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=APE: Analyses of Phylogenetics and Evolution in R language.; citation_author=E Paradis; citation_author=J Claude; citation_author=K Strimmer; citation_journal_title=Bioinform; citation_volume=20; citation_number=63; citation_pages=289-290; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing: Vienna, Austria; citation_number=64; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Generalized Additive Models: An Introduction with R:; citation_author=SN Wood; citation_number=65; citation_date=2006; citation_publisher=Chapman & Hall/CRC; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001730" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Tempo and Mode of Diversification of Lake Tanganyika Cichlid Fishes"/>
    <meta name="twitter:description" content="BackgroundUnderstanding the causes of disparities in species diversity across taxonomic groups and regions is a fundamental aim in evolutionary biology. Addressing these questions is difficult because of the need for densely sampled phylogenies and suitable empirical systems.Methodology/Principal FindingsHere we investigate the cichlid fish radiation of Lake Tanganyika and show that per lineage diversification rates have been more than six times slower than in the species flocks of Lakes Victoria and Malawi. The result holds even at peak periods of diversification in Lake Tanganyika, ruling out the age of the lake as an explanation for slow average rates, and is robust to uncertainties over the calibration of cichlid radiations in geological time. Moreover, Lake Tanganyika lineages, irrespective of different biological characteristics (e.g. sexually dichromatic versus sexually monochromatic clades), have diversified at similar rates, falling within typical estimates across a range of plant and animal clades. For example, the mostly sexually dichromatic haplochromines, which have speciated explosively in Lakes Victoria and Malawi, have displayed modest rates in Lake Tanganyika (where they are called Tropheini).Conclusion/SignificanceOur results show that either the Lake Tanganyika environment is less conducive for cichlid speciation or the remarkable diversifying abilities of the haplochromines were inhibited by the prior occupancy of older radiations. Although the results indicate a dominant role for the environment in shaping cichlid diversification, differences in the timing of diversification among the Tanganyikan tribes indicate that biological differences were still important for the dynamics of species build-up in the lake. While we cannot resolve the timing of the radiation relative to the origin of the lake, because of the lack of robust geological date calibrations for cichlids, our results are consistent with a scenario that the different clades reflect independent adaptive radiations into different broad niches in the lake."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001730.g004"/>

  <meta property="og:title" content="Tempo and Mode of Diversification of Lake Tanganyika Cichlid Fishes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001730" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8462'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5629'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1957&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001730">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001730" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001730&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Tempo and Mode of Diversification of Lake Tanganyika Cichlid Fishes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Julia J. Day
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:j.day@ucl.ac.uk">j.day@ucl.ac.uk</a></p>

                <p>Affiliations:
                  Department of Biology, University College London, London, United Kingdom, 
                  Department of Zoology, The Natural History Museum, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James A. Cotton, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Zoology, The Natural History Museum, London, United Kingdom, 
                  School of Biological and Chemical Sciences, Queen Mary, University of London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Timothy G. Barraclough
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Biology and NERC Centre for Population Biology, Imperial College London, Silwood Park Campus, Ascot, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 05, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001730</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001730" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001730" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001730" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001730" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001730-g001" data-doi="info:doi/10.1371/journal.pone.0001730" data-uri="info:doi/10.1371/journal.pone.0001730.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001730-g002" data-doi="info:doi/10.1371/journal.pone.0001730" data-uri="info:doi/10.1371/journal.pone.0001730.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001730-g003" data-doi="info:doi/10.1371/journal.pone.0001730" data-uri="info:doi/10.1371/journal.pone.0001730.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001730-g004" data-doi="info:doi/10.1371/journal.pone.0001730" data-uri="info:doi/10.1371/journal.pone.0001730.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001730-t001" data-doi="info:doi/10.1371/journal.pone.0001730" data-uri="info:doi/10.1371/journal.pone.0001730.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001730">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001730" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Understanding the causes of disparities in species diversity across taxonomic groups and regions is a fundamental aim in evolutionary biology. Addressing these questions is difficult because of the need for densely sampled phylogenies and suitable empirical systems.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here we investigate the cichlid fish radiation of Lake Tanganyika and show that per lineage diversification rates have been more than six times slower than in the species flocks of Lakes Victoria and Malawi. The result holds even at peak periods of diversification in Lake Tanganyika, ruling out the age of the lake as an explanation for slow average rates, and is robust to uncertainties over the calibration of cichlid radiations in geological time. Moreover, Lake Tanganyika lineages, irrespective of different biological characteristics (e.g. sexually dichromatic versus sexually monochromatic clades), have diversified at similar rates, falling within typical estimates across a range of plant and animal clades. For example, the mostly sexually dichromatic haplochromines, which have speciated explosively in Lakes Victoria and Malawi, have displayed modest rates in Lake Tanganyika (where they are called Tropheini).</p>


<h3>Conclusion/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our results show that either the Lake Tanganyika environment is less conducive for cichlid speciation or the remarkable diversifying abilities of the haplochromines were inhibited by the prior occupancy of older radiations. Although the results indicate a dominant role for the environment in shaping cichlid diversification, differences in the timing of diversification among the Tanganyikan tribes indicate that biological differences were still important for the dynamics of species build-up in the lake. While we cannot resolve the timing of the radiation relative to the origin of the lake, because of the lack of robust geological date calibrations for cichlids, our results are consistent with a scenario that the different clades reflect independent adaptive radiations into different broad niches in the lake.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Day JJ, Cotton JA, Barraclough TG (2008) Tempo and Mode of Diversification of Lake Tanganyika Cichlid Fishes. PLoS ONE 3(3):
          e1730.
            doi:10.1371/journal.pone.0001730</p><p><strong>Academic Editor: </strong>Oliver G. Pybus, University of Oxford, United Kingdom</p><p><strong>Received:</strong> August 20, 2007; <strong>Accepted:</strong> February 5, 2008; <strong>Published:</strong> March 5, 2008</p><p><strong>Copyright:</strong> © 2008 Day et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>JJD is supported by a Dorothy Hodgkin Royal Society fellowship.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Explaining why some regions and taxonomic groups contain more species than others is a key goal of evolutionary biology. Different categories of explanations have been offered. For example, if resources place a limit on the number of species in a region, then diversity might often arise by adaptive radiation, in which lineages diversify rapidly to occupy different niches following the invasion of a new region or a shift in habitat usage <a href="#pone.0001730-Simpson1">[1]</a>, <a href="#pone.0001730-Schluter1">[2]</a>. Alternatively, species richness may depend on rates of speciation and extinction: some organisms may evolve reproductive isolation more readily or have a lower risk of extinction than others, irrespective of any ecological limits on biomass <a href="#pone.0001730-Purvis1">[3]</a>. Factors determining net diversification rates could include biological traits <a href="#pone.0001730-Barraclough1">[4]</a>, <a href="#pone.0001730-Coyne1">[5]</a>, the environment <a href="#pone.0001730-Cardillo1">[6]</a>–<a href="#pone.0001730-Ricklefs1">[8]</a>, or the interaction between traits and the environment <a href="#pone.0001730-Davies1">[7]</a>, <a href="#pone.0001730-Davies2">[9]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>One way to evaluate these alternatives is to explore the timing of diversification using molecular phylogenetics. For example, adaptive radiation predicts fast early diversification followed by slower rates of species-turnover once available niches have been filled <a href="#pone.0001730-Schluter1">[2]</a>. Similarly, comparisons among taxa and regions can be used to test the effects of biological traits versus the environment <a href="#pone.0001730-Davies1">[7]</a>–<a href="#pone.0001730-Davies2">[9]</a>. Do diversification rates tend to be more similar between related taxa in different environments, between unrelated taxa in the same environments, or do they depend on an interaction between traits and environment? Such comparisons remain rare because of the need for phylogenetic trees containing nearly all species in the regions and taxa of interest.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here, we use a near-complete phylogenetic tree of cichlid fish species from Lake Tanganyika (LT) to test these ideas in a lake fauna. Cichlids in the East African great lakes have provided classic examples of endemic radiations. Each lake harbors a species flock of many hundred species displaying astonishing levels of ecological, phenotypic and behavioural diversity. Phylogenetic and geological evidence indicates exceptionally rapid radiation from single ancestral species over very short evolutionary timescales in Lake Victoria (LV), 447–535 species and Lake Malawi (LM), 451–600 species <a href="#pone.0001730-Genner1">[10]</a>, based on age estimates assuming near or total desiccation for these lakes at 12.4 Ka <a href="#pone.0001730-Johnson1">[11]</a> and 1 Mya - 0.57 Ka <a href="#pone.0001730-Delvaux1">[12]</a>, respectively. The same tribe, the haplochromines, has radiated in both lakes, where they comprise the majority of cichlid species. A major cause of the explosive radiations appears to be rapid speciation through sexual selection associated with shifts in male breeding coloration and associated female preferences e.g. <a href="#pone.0001730-Dominey1">[13]</a>–<a href="#pone.0001730-KnightMETurner1">[15]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The cichlid fauna of LT is also thought to have radiated rapidly <a href="#pone.0001730-McCune1">[16]</a>, <a href="#pone.0001730-Turner1">[17]</a>. However, LT represents a very different scenario regarding its geological history and cichlid fauna. Despite being the oldest rift lake (9–12 Mya) <a href="#pone.0001730-Cohen1">[18]</a>, and larger than LM, it contains only one third as many cichlid species (200 reported in this study). LT has experienced fluctuations in lake level <a href="#pone.0001730-Cohen2">[19]</a>, but unlike LM and LV, is not thought to have experienced near or complete desiccation. The fauna derives from several distinct lineages (classified in up to 16 tribes <a href="#pone.0001730-Takahashi1">[20]</a>) derived from several invasions rather than from a single ancestor <a href="#pone.0001730-Salzburger1">[21]</a>. Many LT tribes are entirely sexually dichromatic, such as Cyprichromini, Bathybatini. Others are partially sexually dichromatic, such as Ectodini and Tropheini (the LT endemic haplochromines). Conversely, the most species rich lineage, Lamprologini (~90–100 species) are sexually monochromatic. Therefore, LT provides the opportunity to compare the tempo of diversification among biologically distinct lineages occupying the same environment, and among members of the same lineage (the haplochromines) in different lakes.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In order to investigate patterns and rates of diversification within a clade, ideally nearly all the species should be sampled <a href="#pone.0001730-Barraclough2">[22]</a>. Previous comparisons have been limited by incomplete sampling and by a lack of statistical evaluation of expected patterns under hypothesized scenarios. To date, a single study <a href="#pone.0001730-Seehausen1">[23]</a>, using lineage-through-time (LTT) plots, based on published linearised trees of five tribes (Limnochromini, Perissodini, Cyprichromini, Bathybatini, Ectodini) has attempted to evaluate patterns of LT cichlid diversification. The LTT plots of all five tribes were interpreted to show a speciation burst around the same time followed by a period of stasis, but the two most species rich tribes were not included and the conclusion was made from graphical interpretation rather than statistical analysis.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Compiling mtDNA sequence data for 152 (76%) endemic species, plus selected species from elsewhere [<a href="#pone.0001730.s002">Table S1</a>], we reconstruct the timing of diversification of the LT species flock using a Bayesian relaxed molecular clock approach. We also consider the potential effects of missing species on estimates of the timing and rate of diversification. We show that the tempo and mode of cichlid diversification is highly contingent on the environment. LT cichlids have diversified six times more slowly than endemic radiations in the other lakes, even during peak episodes of diversification. This result is robust to uncertainties in age of the cichlid flocks. Diversification rates are remarkably similar among LT tribes considering 95% confidence limits, despite variation in general niche (e.g. littoral versus deep-water) and breeding system (e.g. sexually monochromatic versus dichromatic). We do not find strong evidence for an initial rapid radiation followed by a uniform slow-down in rates. Instead, faster diversification occurred at intermediate periods, possibly coinciding with periods of changing lake levels e.g. <a href="#pone.0001730-Cane1">[24]</a>, <a href="#pone.0001730-Tiercelin1">[25]</a> or with successive invasions and separate adaptive radiations. The different tempo and mode of diversification in LT cichlids could reflect either its different physical environment or the constraining effects of earlier radiations on the remarkable diversifying abilities of the haplochromines.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Phylogeny and Age Estimates of the LT Cichlid Radiation</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Maximum likelihood and Bayesian analyses recover trees with good support for relationships that are largely congruent with taxonomy (<a href="#pone.0001730.s001">Figure S1</a>, <a href="#pone.0001730.s004">Text S1</a>). The majority of species derive from a single well-supported clade (P, <a href="#pone.0001730.s001">Figure S1</a>), comprising two main lineages, the C-lineage <a href="#pone.0001730-Clabaut1">[26]</a>, which contains several tribes including the Haplochromini, and a clade we call the L-lineage, which includes the Lamprologini and Eretmodini. The endemic Tanganyikan haplochromines, termed Tropheini, are closely related to the haplochromine radiations in LM and LV <a href="#pone.0001730-Salzburger1">[21]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Dating cichlid radiations in geological time is problematic because of the lack of fossils and other robust sources of independent calibration. Because of this uncertainty, we adopt, in turn, two widely different approaches used by previous authors. First, we use a date of 12 Mya for the root of the LT radiation (<a href="#pone-0001730-g001">Figure 1</a>), which is the proposed maximum geological age of LT <a href="#pone.0001730-Cohen1">[18]</a> and which is similar to the estimated age for this node based on fossil cichlid calibrations <a href="#pone.0001730-Genner2">[27]</a>. Second, we use a date of 28 Mya for the same node based on calibrations of Gondwanan fragmentation for cichlids <a href="#pone.0001730-Genner2">[27]</a>. The cichlid timescale based on continental fragmentation supports a much older age (Cretaceous), of origin for cichlids, as opposed to the fossil record of the group, which dates back to the Eocene. This consequently has implications for the origin of LT cichlid faunas, in that Gondwanan dates imply the lake would have been colonised independently by the main lineages <a href="#pone.0001730-Genner2">[27]</a>–<a href="#pone.0001730-Terai1">[29]</a>, rather than the accepted view based on the younger calibration, which assumes that early branching events occurred within the lake <a href="#pone.0001730-Genner2">[27]</a>, <a href="#pone.0001730-Salzburger2">[30]</a>. We report results for both calibrations, but our main conclusions depend on relative dates, rather than absolute dates, and so are robust to these uncertainties. Dating the tree using a Bayesian relaxed molecular clock model in BEAST <a href="#pone.0001730-Drummond1">[31]</a> resulted in the ultrametric consensus tree depicted in <a href="#pone-0001730-g001">Figure 1</a>.</p>
<div class="figure" id="pone-0001730-g001"><div class="img"><a name="pone-0001730-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001730" data-uri="info:doi/10.1371/journal.pone.0001730.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001730.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001730.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Phylogeny of LT cichlids reconstructed by Bayesian Inference and dated using a relaxed molecular clock (Yule speciation prior).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Timescales for the two alternative calibrations are shown. The gray bars on each node indicate 95% credible intervals for the node ages. Black circle indicates the node used to date the tree. P = Principle lineage, C = C-lineage, L = L-lineage. Gray branches indicate non-endemic taxa. LM = Lake Malawi species.</p>
<span>doi:10.1371/journal.pone.0001730.g001</span></div>

<h4>Net Diversification Rates in LT</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We used models assuming constant per lineage rates of speciation and extinction to estimate average net diversification rates (speciation rate minus extinction rate) for the LT flock and constituent clades (<a href="#pone.0001730.s003">Table S2</a>) <a href="#pone.0001730-Nee1">[32]</a>. None of the clades displayed evidence for a non-zero extinction rate (<a href="#pone.0001730.s003">Table S2</a>). The average estimate of the per lineage diversification rate for the entire LT endemic flock, assuming a root age of 12 Mya and a Yule Prior for the distribution of node ages, is 0.31 species/Myr (we report per lineage rates throughout). The confidence interval incorporating both the stochastic nature of lineage branching and the uncertainty of relative date estimates across the Bayesian samples is 0.27 to 0.37 (<a href="#pone-0001730-g002">Figure 2</a>). To evaluate the effect of incomplete species sampling, we used taxonomic information about missing species to include them in our phylogeny. Each missing species was added to the tree in turn randomly with equal probability along the branches belonging to its likely clade (<a href="#pone.0001730.s001">Figure S1</a> for placements). Addition of missing species increases the estimates to 0.36 (C.I. 0.30–0.40) for the entire flock (<a href="#pone.0001730.s003">Table S2</a>). We report results for the amended trees including the missing species but conclusions are qualitatively unaffected by their presence or absence. Our rate estimates are lower than previous ones for the LT flock, for example 0.9 <a href="#pone.0001730-McCune1">[16]</a> and 1.42 species/Myr <a href="#pone.0001730-Turner1">[17]</a>. Instead, they fall within the range of typical diversification rates estimated across a wide range of plant and animal taxa <a href="#pone.0001730-Coyne1">[5]</a>, <a href="#pone.0001730-McCune1">[16]</a>, <a href="#pone.0001730-Baldwin1">[33]</a> <a href="#pone.0001730-McCune2">[34]</a>. Diversification rate estimates enforcing an alternative prior, proportional-to distinguishable (PD), are slower than those assuming the Yule prior e.g. 0.26 (C.I. 0.22–0.29) for the LT flock, but there remained no significant evidence for non-zero background extinction rates (<a href="#pone.0001730.s003">Table S2</a>). Similarly, assuming an older age of divergence of the LT radiation (28 Myr), rate estimates fall to 0.15 species/Myr (C.I. 0.13 to 0.17, Yule prior).</p>
<div class="figure" id="pone-0001730-g002"><div class="img"><a name="pone-0001730-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001730" data-uri="info:doi/10.1371/journal.pone.0001730.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001730.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001730.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Average net diversification rates of LT cichlid clades (with missing species added) and the minimum rate estimate for LM and LV cichlid species flocks.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Lines show the 95% confidence intervals incorporating both variation across sampled trees and the inherent stochasticity of the branching process.</p>
<span>doi:10.1371/journal.pone.0001730.g002</span></div>

<h4>Comparison with Lake Malawi and Lake Victora</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Although the uncertainty in calibration dates prevents robust inference of absolute rates, we use the position of LM endemic species in our tree for a conservative comparison of diversification rates between LT and the LM and LV species flocks. We assume that the node separating the LM species from their nearest related LT lineage, i.e. their stem group age, defines an extreme maximum date, <em>t</em>, for the single common ancestor of the LM and LV species, which comprise a monophyletic group together with several riverine taxa <a href="#pone.0001730-Salzburger1">[21]</a>. Our approach, based on the younger calibration, uses an age of 3.1 Myr (C.I. 2.6 to 4.2) whereas the LV flock is believed to be much younger <a href="#pone.0001730-Nagl1">[e.g. 35]</a>. We then estimate the net diversification rate as log(<em>N</em>)/<em>t</em>, where <em>N</em> is the number of species and the 95% confidence interval due to the stochasticity of the branching process is −log(1−0.025<sup>1/<em>N</em></sup>)/<em>t</em> to −log(1−0.975<sup>1/<em>N</em></sup>)/<em>t</em> <a href="#pone.0001730-Purvis2">[36]</a>. Across the Bayesian samples, using the younger calibration yields an average diversification rate of 2.02 species/Myr (C.I. 1.16 to 3.12) with <em>N</em> = 898 (minimum species number for LM and LV flocks) <a href="#pone.0001730-Genner1">[10]</a>, rising to 2.09 species/Myr (C.I. 1.71 to 3.19) with <em>N</em> = 1135 (maximum species number of LM and LV flocks) <a href="#pone.0001730-Genner1">[10]</a>. Using the older calibration yields average diversification rates of 0.87 (C.I. 0.70 to 1.33) and 0.90 (C.I. 0.73 to 1.37) respectively. The confidence intervals do not overlap with the C.I. of any of the LT clades (<a href="#pone-0001730-g002">Figure 2</a>). Therefore, the LM and LV species flocks have diversified at more than five times the rate of the LT flock. This conclusion applies even for the Tropheini, which are closely related to the LM and LV flocks and share their biological characteristics (see next section). Unlike previous comparisons, the conclusion is also independent of how we calibrated the radiations in geological time, because our comparisons derive from a single phylogenetic reconstruction, and is likely to be very conservative. Using a date for the LV flock of 0.12 Myr based on the older, conservative calibration <a href="#pone.0001730-Genner2">[27]</a> would yield a staggering per lineage rate estimate of over 50 species/Myr.</p>


<h4>Comparison of LT Cichlid faunas</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Separate estimates of net diversification rate for the endemic tribes reveal a relatively uniform average speed of diversification among tribes when 95% confidence intervals are considered (<a href="#pone-0001730-g002">Figure 2</a>). Mean values for tribes ranged from 0.19–0.60 species/Myr, assuming 12 Myr calibration, falling to 0.07–0.24 species/Myr using the older calibration (<a href="#pone.0001730.s003">Table S2</a>). To test whether rates varied significantly among tribes, we multiplied the time period between successive nodes in each tribe by the number of lineages present during that interval <a href="#pone.0001730-Nee2">[37]</a>. Under a constant speciation rate model, these transformed internode distances are expected to be equal to the inverse of the diversification rate <a href="#pone.0001730-Purvis2">[36]</a>, <a href="#pone.0001730-Barraclough3">[38]</a>. An anova with clade as a factor revealed no significant difference in diversification rate either between the C and L lineages (<em>F<sub>1,176</sub></em> = 0.1, <em>p</em>&gt;0.5) or among tribes (<em>F<sub>5,158</sub></em> = 1.6, <em>p</em>&gt;0.1).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The fastest rates were recorded in the Tropheini (0.60 species/Myr) and Lamprologini (0.41 species/Myr), assuming a 12 Myr calibration, slowing to 0.16 and 0.24 species/Myr respectively, when the 28 Myr calibration is enforced. That these tribes share similar rates is of interest as they have contrasting evolutionary histories, with the former having a more recent origin than the latter, as well as displaying marked differences in breeding behavior. Lamprologines are substrate brooders and are monochromatic (~5% of species exhibit extreme sexual dimorphism) <a href="#pone.0001730-Konings1">[39]</a>. Conversely, the Tropheini, as with all haplochromines, are mouthbrooders, and while most species are sexually dichromatic, some genera (e.g. <em>Tropheus</em>) are mainly monochromatic. The slowest rate is recorded in a deep-water tribe of strongly sexually dichromatic mouthbrooders, the Bathybatini, (irrespective of inclusion of the basal species, <em>B. minor</em>), with rates estimated from 0.19 – 0.08 species/Myr, assuming a 12 or 28 Myr calibration respectively. Therefore, contrary to previous findings (16), speciation is not uniformly faster in sexually dimorphic groups than in sexually monomorphic groups. The lamprologines also display much greater ecological diversity in trophic morphology and habitat than the Tropheini, consistent with a mode of ecological speciation <a href="#pone.0001730-Schluter1">[2]</a>. Interestingly, a large proportion of lamprologines have exploited gastropod shells to live in and/or breed <a href="#pone.0001730-Yanagisawa1">[40]</a>. Occupation of this niche and the associated dwarfism it entailed may have triggered additional diversification (shell-brooding is significantly associated with increased diversification rate in the lamprologines (<a href="#pone.0001730.s002">Table S1</a>, <a href="#pone.0001730.s004">Text S1</a>). Whatever the mechanism of diversification in each tribe, it is clear that clades with very different biological characteristics have diversified to similar degrees in the lake.</p>


<h4>The Timing of Diversification</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Adaptive radiation, driven by ecological opportunity, predicts rapid early speciation, followed by a subsequent slowdown <a href="#pone.0001730-Schluter1">[2]</a>, <a href="#pone.0001730-Gavrilets1">[41]</a>, whereas speciation by non-ecological sexual selection does not predict this trend <a href="#pone.0001730-Seehausen1">[23]</a>, and may instead predict a speed up of net diversification rate, as recorded in LV cichlids <a href="#pone.0001730-Schluter1">[2]</a>, <a href="#pone.0001730-Meyer1">[after 42]</a>. However, if several lineages colonized the lake and radiated independently, we might expect bursts of diversification following colonization but not necessarily congruent between different clades if they colonized at different times and filled different broad niches. Other scenarios might generate similar patterns, for example the appearance of new niches in the lake, such as shell beds, or if environmental events such as changing lake levels <a href="#pone.0001730-Cane1">[24]</a>, <a href="#pone.0001730-Tiercelin1">[25]</a> triggered an increased speciation rate <a href="#pone.0001730-Sturmbauer1">[43]</a>.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>While distinguishing all these scenarios is difficult, we test whether diversification rates have declined over time and whether any changes have been synchronous among clades. Lineage-through-time (LTT) plots show that rates have slowed down somewhat: a straight line is expected if there has been a constant speciation rate with no extinction (<a href="#pone-0001730-g003">Figure 3</a>) <a href="#pone.0001730-Nee1">[32]</a>. Under both priors (Yule and PD), a significant decline is recorded (Pybus and Harvey <strong>γ</strong> statistic <a href="#pone.0001730-Pybus1">[44]</a>) for the entire endemic flock (<a href="#pone-0001730-g003">Figures 3a</a>, S6a, <strong>γ</strong> = −2.25, <em>p</em> = 0.06; S6e -4.25, <em>p</em> = 0.000) and the principle radiation (<a href="#pone-0001730-g003">Figures 3a</a>, S6a, <strong>γ</strong> = −3.20, <em>p</em> = 0.007; S6e -4.52, <em>p</em> = 0.000). This is also true for the L-lineage under the PD prior (S6e -3.55, <em>p</em> = 0.002), while the decline enforcing the Yule prior is marginally non-significant (<a href="#pone-0001730-g003">Figure 3b</a>, S6a, <strong>γ</strong> = −2.77, <em>p</em> = 0.02). Among the tribes only the Lamprologini (which comprise the majority of the L-lineage) display a significant decline in rates over time (<a href="#pone-0001730-g003">Figure 3c</a>, <a href="#pone.0001730.s003">Table S2</a>). Declines among the other tribes and C-lineage are non-significant (<a href="#pone.0001730.s003">Table S2</a>) and overall there is not the strong plateau expected of an early adaptive radiation and slow-down model. Instead, the curve for the C-lineage is more sigmoid in shape (<a href="#pone-0001730-g003">Figure 3b</a>).</p>
<div class="figure" id="pone-0001730-g003"><div class="img"><a name="pone-0001730-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001730" data-uri="info:doi/10.1371/journal.pone.0001730.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001730.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001730.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Lineage-through-time (LTT) plots, including missing species, based on 1000 sampled Bayesian trees: a) LT radiation (black); b) L-lineage (black) and C-lineage (gray); c) tribes: Lamprologini (purple), Ectodini (green), Tropheini (plum), Limnochromini (fawn), Cyprichromini (aqua), Bathybatini (yellow).</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>A single LLT plot is highlighted for all tribes, but is only displayed for the latter three tribes for clarity. Timescales for the two alternative calibrations are shown.</p>
<span>doi:10.1371/journal.pone.0001730.g003</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>General Additive Models (GAM) were used to explore the net change in diversification rates over time in more detail (<a href="#pone-0001730-g004">Figure 4</a>) <a href="#pone.0001730-BindaEmonds1">[45]</a>. These show that, across the entire flock, diversification rates were highest at intermediate periods, around 3 to 4 Mya (12 Myr calibration) or 7 to 8 Mya (28 Myr calibration, <a href="#pone-0001730-g004">Figure 4a</a>). These findings appear to contradict those of the Pybus and Harvey test <a href="#pone.0001730-Pybus1">[44]</a>, which implied a significant slowdown. While the robustness of GAMs has been questioned in the past, in this case similar conclusions are reached from trees derived from the two different priors of node age distributions and from the repeated samples from each MCMC analysis. Therefore, we believe the GAMs are fitting real features of the distribution of node ages in our sample: the average overall decline in diversification rate is largely due to decline from intermediate to recent periods, which encompass most of the nodes contributing information to the test statistics.</p>
<div class="figure" id="pone-0001730-g004"><div class="img"><a name="pone-0001730-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001730" data-uri="info:doi/10.1371/journal.pone.0001730.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001730.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001730.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Net diversification rate through time fitted by General Additive Models (GAMs) for a) the entire flock and b) the L- (black) and C- (gray) lineages.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>Scale bars (x-axis) represent both 28 Myr and 12 Myr calibrations and the y-axis is the per lineage diversification rate in species per million years. Dashed lines show standard errors combining those across Bayesian samples and those due to uncertainty in the model in a), but only those across Bayesian samples in b). In the latter case, standard errors due to uncertainty in the model are extremely large.</p>
<span>doi:10.1371/journal.pone.0001730.g004</span></div><a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>The pattern differs marginally between the C- and L-lineages (ANOVA comparing GAMs with clade as a factor or not, median F across Bayesian samples = 2.1, median <em>p</em> = 0.033). The C-lineage is typified by an early and late peak in diversification rates, separated by a period of low net rates, during which time the L-lineage sustained higher net rates (<a href="#pone-0001730-g004">Figure 4b</a>). Both lineages reveal a final decline in rates towards the present, which could reflect either a true decline, over-conservative placement of the species missing from our sample, or a taxonomic artifact of failure to recognize recently diverged species. If clades have different peak periods of diversification, even though average diversification rates are relatively similar among tribes, this would suggest that biological differences among tribes did matter in the build-up of present-day diversity. Either clades were affected differentially by different environmental events or they filled broadly different niches in the lake. If biological differences were irrelevant, we would have expected any changes in diversification rate to be congruent among the different clades. However, even with a near complete sample of species, and potentially large differences in average diversification rate at given times, the differences between the two clades are only marginally significant in our present analyses.</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>Interpreting the causes of the diversification pulses is even more difficult. With the younger calibration based on lake age and fossils, then the first peak in the C-lineage corresponds roughly with the onset of full lacustrine conditions and the second peak with a shift in global climate and the aridification of Africa <a href="#pone.0001730-Cane1">[24]</a>, when lake levels are assumed to have dropped. With the older calibration based on Gondwanan fragmentation, then the initial peak would correspond to an early radiation generating the ancestors of the different tribes, perhaps in a now extinct lake <a href="#pone.0001730-Seehausen1">[23]</a>, and the second peak with the onset of full lacustrine conditions. Similarly, the L-lineage may have sustained high rates from basin formation to the onset of full lacustrine conditions (older calibration) or after these events (younger calibrations). The older calibration is appealing because peak diversification across the entire flock (ca. 8 to 9 Mya) would then be congruent with basin formation and development, but this requires that diversification leading to the different tribes occurred at an earlier time. However, the older calibration leads to rather slow estimates of net diversification rates among LT cichlids (although not LM and LV). Clearly, further evidence is needed to resolve these alternatives. Given the difficulty in finding robust calibrations for cichlids, and the limits of sample size being reached for distinguishing alternative diversity models, one possibility would be to compare the timing of diversification across a broad suite of LT endemic taxa, especially those with fossil records or alternative geological calibrations e.g. <a href="#pone.0001730-Day1">[46]</a>, <a href="#pone.0001730-Marijnissen1">[47]</a>.</p>
<a id="article1.body1.sec2.sec5.p6" name="article1.body1.sec2.sec5.p6"></a><p>We conclude that LT cichlid fishes have diversified much more slowly than those of LV and LM, even the endemic LT haplochromines. In contrast to LM and LV, the LT species flock derives from a prolonged accumulation of species, rather than rapid, recent radiation. Clearly, the environment plays a major role in determining cichlid diversification rates. One possibility is that the physical environment of LT is less conducive for cichlid speciation, perhaps by inhibiting whatever special mechanism causes fast speciation in LM and LV. Turbid waters have been shown to impede sexual selection in LV <a href="#pone.0001730-Seehausen2">[48]</a>, raising the possibly that a similar phenomenon may have occurred during the history of LT, although currently these waters are very transparent <a href="#pone.0001730-Langenberg1">[49]</a>. Alternatively, the remarkable diversifying abilities of haplochromines could have been inhibited because the lake was already occupied by older radiations, such as the lamprologines and ectodines, which occupy similar habitats. The similarity of net diversification rates among tribes with very different biological characteristics might seem to imply that traits had minimal influence on the progress of diversification in the lake. However, there is some evidence that clades did vary in the timing of peak rates of diversification: lineage identity was important for determining diversification potential over large periods of the lake's history. Whether these differences reflect differential responses to episodes of climate change or successive adaptive radiations into different broad niches is difficult to resolve until more robust calibrations are available. However, a plausible explanation is that the lake was colonized several times leading to a series of independent adaptive radiations into different broad niches followed by a slow-down in rate in each descendent clade.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Phylogeny</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Previous phylogenetic studies of LT cichlids consider subsets of the data (&lt; one third of sampled species) <a href="#pone.0001730-Clabaut1">[26]</a>, <a href="#pone.0001730-Salzburger2">[30]</a>. However, recent proliferation of sequence data represents an important source for comparative studies on LT cichlids. Densely sampled trees are important in diversification studies as well as providing information on relative timing of cladogensis <a href="#pone.0001730-Barraclough2">[22]</a>. Increased taxon sampling has been shown to improve both the precision and accuracy of phylogenetic reconstruction <a href="#pone.0001730-Flynn1">[50]</a>–<a href="#pone.0001730-Zwickl1">[51]</a>. We reconstruct the evolutionary relationships of 161 cichlid species (152 LT endemics, 9 non-endemics), representing 76% of all LT species (assuming a LT cichlid flock of 200 endemic species, <a href="#pone.0001730.s002">Table S1</a>), based on two mtDNA genes: the protein coding, ND2 (~1047 bp) and the non-coding, control region (CR), (partial sequence data of 347 bp from the 5′-end) using published sequence data (<a href="#pone.0001730.s002">Table S1</a>). Alignment of ND2 sequences was unambiguous, in comparison to the CR, which varied slightly in length among sequences (337-335, with the longest gap consisting of a three bp indel). The CR data was initially aligned using CLUSTAL X <a href="#pone.0001730-Thompson1">[52]</a> and subsequently both datasets were optimised by visual inspection in a manual alignment editor <a href="#pone.0001730-Rambaut1">[53]</a>. Modeltest v.3.7 <a href="#pone.0001730-Posada1">[54]</a> selected the evolutionary model GTR+I+Γ, using Akaike Information Criteria and was implemented in subsequent analyses. Bayesian Inference, using Metropolis-coupled Markov Chain Monte Carlo (MCMCMC), implemented in MrBayes v.3.1.2 <a href="#pone.0001730-Ronquist1">[55]</a>, was run for 2×10<sup>6</sup> generations, sampling every 100 generations (four chains, temperature 0.2), to ensure convergence, with the first 250,000 generations discarded as burn-in. This analysis was partitioned to account for the different behaviour of the genes as well as the third codon of ND2. Maximum likelihood inference was implemented in GARLI v.0.942 (Genetic Algorithm for Rapid Likelihood Inference) <a href="#pone.0001730-Zwickl2">[56]</a> also sampling 2×10<sup>6</sup> generations for multiple runs to ensure similar trees and lnL scores. Branch support was evaluated using non-parametric bootstrapping (BS) consisting of 1000 pseudoreplicates (using GARLI), and Bayesian posterior probabilities (BPP). The tree topologies recovering using Bayesian Inference and Maximum Likelihood were evaluated using the approximately unbiased test <a href="#pone.0001730-Shimodaira1">[57]</a>, implemented in the program CONSEL <a href="#pone.0001730-Shimodaira2">[58]</a> using the site likelihood scores from PAUP* <a href="#pone.0001730-Swofford1">[59]</a> see <a href="#pone.0001730.s004">Text S1</a>.</p>


<h4>Molecular Dating</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>A likelihood ratio test <a href="#pone.0001730-Huelsenbeck1">[60]</a> performed on the concatenated data set [molecular clock enforced (-ln L 41653.91), not enforced (-ln L 41352.23), <em>χ</em><sup>2</sup> = 603.36, df = 157, <em>p</em> = 0.00,)] rejected overall constancy of rates of evolution. Divergence time analyses using a log-normal distributed relaxed molecular clock <a href="#pone.0001730-Drummond1">[31]</a> were performed using BEAST v1.4.6 <a href="#pone.0001730-Drummond2">[61]</a>, which uses Bayesian inference and an MCMC procedure to estimate the posterior distribution of rates and times. We used a constant-rate Yule (speciation process) prior, and all other priors and operators were the default settings, except that the root was constrained with a point prior, and the starting tree was the maximum posterior probability topology form the Bayesian phylogenetic analysis, and this topology remained fixed throughout the MCMC. Three independent chains were run for 2×10<sup>6</sup> generations each, and convergence checked visually by comparing these runs after a burn-in of 2×10<sup>5</sup> generations was discarded. As there is no Bayesian MCMC implementation of speciation models that accommodate extinction, we also performed the same analyses using a Proportional-to-distinguishable (PD) prior. The PD arrangements prior puts an equal probability on every different labelled tree <a href="#pone.0001730-Rosen1">[62]</a>. To investigate the sensitivity of divergence time estimates, we performed the same analyses using exponential relaxed clock and strict clock models in BEAST. Results from these analyses were largely congruent with those from the analysis reported here.</p>


<h4>Estimating Diversification Rates</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The resulting ultrametric trees obtained using both Yule and PD priors using a log-normal distributed relaxed clock calculated by BEAST were imported into the APE 1.2–2 package <a href="#pone.0001730-Paradis1">[63]</a> of the R statistical programming language <a href="#pone.0001730-R1">[64]</a> to generate semi logarithmic LLT plots calculated for 1000 sampled trees (sampling every 2500 generations of the LT radiation and across different taxonomic grouping of LT cichlids We also consider diversification rate (speciation minus extinction rates) using the results from BEAST, implementing different constant speciation and extinction rate models using the birthdeath function in APE, for the same taxonomic groups, similarly calculated from 1000 sampled trees (sampling every 2500 generations). All non-endemic taxa were excluded from these analyses. For each clade we also tested for significant departures from the constant speciation model using the γ statistic <a href="#pone.0001730-Pybus1">[44]</a>. Positive values for this statistic signify that there has been an apparent increase in diversification rate towards the present, whereas negative values indicate a deceleration in diversification rate towards the present. Generalized additive models were constructed using the mgcv library in R <a href="#pone.0001730-R1">[64]</a>. Models of transformed internode intervals as a function of time were fitted assuming a basis dimension (k) of 50 knots. Unbiased risk estimation (UBRE) was then used to estimate smoothing parameters and the effective degrees of freedom for the smooth term <a href="#pone.0001730-Wood1">[65]</a>. Hypothesis-testing comparing a single and different smoothing functions for the C and L lineages was based on unpenalized GAMs <a href="#pone.0001730-Wood1">[65]</a>; for these, <em>k</em> was set to 10 because statistical power falls as <em>k</em> is increased. Nested models (with a single smoothing function or a separate smoothing functions for the two lineages) were compared using ANOVA and F tests. We further applied these models to the data from the PD prior, but found the result was non-significant.</p>


<h4>Web Resources</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>URLs for data presented herein are as follows: GenBank, <a href="http://www.ncbi.nih.gov/Genbank/index.html">http://www.ncbi.nih.gov/Genbank/index.ht​ml</a></p>
<div class="figure" id="pone-0001730-t001"><div class="img"><a name="pone-0001730-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001730" data-uri="info:doi/10.1371/journal.pone.0001730.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001730.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001730.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001730.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001730.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001730.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Lake Tanganyika cichlid clades, including life history traits and age estimates (mean and 95% confidence intervals), based on 12 and 28 Myr calibrations <a href="#pone.0001730-Genner2">[27]</a> using a Bayesian relaxed-clock model enforcing the Yule prior <a href="#pone.0001730-Drummond1">[31]</a>.</span></strong></p><span>doi:10.1371/journal.pone.0001730.t001</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001730"></div><a name="pone.0001730.s001" id="pone.0001730.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001730.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Phylogeny of Lake Tanganyika cichlids based on mtDNA. Bayesian tree recovered from combined analysis of ND2 and control region datasets. Branch support indicated by BPP and BS derived from BI and ML methods (&gt;50%) from left to right respectively. * indicates the same score for both support methods. P = principal lineage, C = C-lineage, L = L-lineage, LM = Lake Malawi. Gray branches indicate non-endemic taxa. Gray circles containing numerical numbers, indicate number of additional species added at that branch, closed gray circle indicates no additional species added beyond that node. Black circle indicates the node used to calibrate the tree. Full details for all species used in this study, including GenBank accession numbers are provided in <a href="#pone.0001730.s003">Table S2</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001730.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.61 MB EPS)</p>
<a name="pone.0001730.s002" id="pone.0001730.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001730.s002">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Cichlid taxa included in phylogenetic analyses and associated Genbank numbers.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001730.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.13 MB PDF)</p>
<a name="pone.0001730.s003" id="pone.0001730.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001730.s003">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Speciation rate statistics for the LT cichlid radiation, major lineages and constituent tribes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001730.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.08 MB PDF)</p>
<a name="pone.0001730.s004" id="pone.0001730.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001730.s004">Text S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Lake Tanganyika cichlid phylogeny Key-innovation test Supporting references</p>
<p class="siDoi">doi:10.1371/journal.pone.0001730.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.07 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank David Gower, Andy Purvis, Oliver Pybus, Lukas Rüber, Mark Wilkinson and anonymous referees for suggestions to improve this manuscript and Andrew Rambaut for help with BEAST.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: TB JD JC. Analyzed the data: TB JD JC. Wrote the paper: TB JD JC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001730-Simpson1" id="pone.0001730-Simpson1"></a>Simpson GG (1953) The Major Features of Evolution. New York: Columbia University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001730-Schluter1" id="pone.0001730-Schluter1"></a>Schluter D (2000) The Ecology of Adaptive Radiation. Oxford: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0001730-Purvis1" id="pone.0001730-Purvis1"></a>Purvis A, Hector A (2001) Getting the measure of biodiversity. Nature  405: 212–219.  <ul class="find" data-citedArticleID="1067928" data-doi="10.1038/35012221"><li><a href="http://dx.doi.org/10.1038/35012221" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Getting+the+measure+of+biodiversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Getting+the+measure+of+biodiversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001730-Barraclough1" id="pone.0001730-Barraclough1"></a>Barraclough TG, Vogler AP, Harvey PH (1998) Revealing the factors that promote speciation. Phil Trans R Soc London B  353: 241–249.  <ul class="find" data-citedArticleID="1067854" data-doi="10.1098/rstb.1998.0206"><li><a href="http://dx.doi.org/10.1098/rstb.1998.0206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Revealing+the+factors+that+promote+speciation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Revealing+the+factors+that+promote+speciation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001730-Coyne1" id="pone.0001730-Coyne1"></a>Coyne J, Orr HA (2004) Speciation. Sunderland, MA: Sinauer.   <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0001730-Cardillo1" id="pone.0001730-Cardillo1"></a>Cardillo M, Orme CDL, Owens IPF (2005) Testing for latitudinal bias in diversification rates: An example using New World birds. Ecology  86: 2278–2287.  <ul class="find" data-citedArticleID="1067864" data-doi="10.1890/05-0112"><li><a href="http://dx.doi.org/10.1890/05-0112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Testing+for+latitudinal+bias+in+diversification+rates%3A+An+example+using+New+World+birds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Testing+for+latitudinal+bias+in+diversification+rates%3A+An+example+using+New+World+birds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001730-Davies1" id="pone.0001730-Davies1"></a>Davies TJ, Barraclough TG, Chase MW, Soltis PS, Soltis DE, et al.  (2004) Darwin's abominable mystery: Insights from a supertree of the angiosperms. Proc Natl Acad Sci USA  101: 1904–1909.  <ul class="find" data-citedArticleID="1067876" data-doi="10.1073/pnas.0308127100"><li><a href="http://dx.doi.org/10.1073/pnas.0308127100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Darwin%27s+abominable+mystery%3A+Insights+from+a+supertree+of+the+angiosperms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Darwin%27s+abominable+mystery%3A+Insights+from+a+supertree+of+the+angiosperms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001730-Ricklefs1" id="pone.0001730-Ricklefs1"></a>Ricklefs RE (2003) Global diversification rates of passerine birds. Proc R Soc London B  270: 2285–2291.  <ul class="find" data-citedArticleID="1067938" data-doi="10.1098/rspb.2003.2489"><li><a href="http://dx.doi.org/10.1098/rspb.2003.2489" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+diversification+rates+of+passerine+birds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+diversification+rates+of+passerine+birds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001730-Davies2" id="pone.0001730-Davies2"></a>Davies TJ, Savolainen V, Chase MW, Goldblatt P, Barraclough TG (2005) Environment, area, and diversification in the species-rich flowering plant family Iridaceae. Am Nat  166: 418–425.  <ul class="find" data-citedArticleID="1067878" data-doi="10.1086/432022"><li><a href="http://dx.doi.org/10.1086/432022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Environment%2C+area%2C+and+diversification+in+the+species-rich+flowering+plant+family+Iridaceae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Environment%2C+area%2C+and+diversification+in+the+species-rich+flowering+plant+family+Iridaceae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001730-Genner1" id="pone.0001730-Genner1"></a>Genner MJ, Seehausen O, Cleary DFR, Knight ME, Michel E, et al.  (2004) How does the taxonomic status of allopatric populations influence species richness within African cichlid fish assemblages? J Biogeogr  31: 93–102.  <ul class="find" data-citedArticleID="1067894" data-doi="10.1046/j.0305-0270.2003.00986.x"><li><a href="http://dx.doi.org/10.1046/j.0305-0270.2003.00986.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+does+the+taxonomic+status+of+allopatric+populations+influence+species+richness+within+African+cichlid+fish+assemblages%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+does+the+taxonomic+status+of+allopatric+populations+influence+species+richness+within+African+cichlid+fish+assemblages%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001730-Johnson1" id="pone.0001730-Johnson1"></a>Johnson TC, Scholz CA, Talbot MR, Kelts K, Ricketts RD, et al.  (1996) Late Pleistocene desiccation of Lake Victoria and rapid evolution of cichlid fishes. Science  273: 1091–1093.  <ul class="find" data-citedArticleID="1067900" data-doi="10.1126/science.273.5278.1091"><li><a href="http://dx.doi.org/10.1126/science.273.5278.1091" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Late+Pleistocene+desiccation+of+Lake+Victoria+and+rapid+evolution+of+cichlid+fishes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Late+Pleistocene+desiccation+of+Lake+Victoria+and+rapid+evolution+of+cichlid+fishes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001730-Delvaux1" id="pone.0001730-Delvaux1"></a>Delvaux D (1996) Age of Lake Malawi (Nyasa) and water level fluctuations. Mus R Afr Centr Tervuren (Belg) Dept Geol Min Rapp Ann  1995–1996: 99–108.  <ul class="find" data-citedArticleID="1067882"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Age+of+Lake+Malawi+%28Nyasa%29+and+water+level+fluctuations.&amp;auth=&amp;atitle=Age+of+Lake+Malawi+%28Nyasa%29+and+water+level+fluctuations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Age+of+Lake+Malawi+%28Nyasa%29+and+water+level+fluctuations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Age+of+Lake+Malawi+%28Nyasa%29+and+water+level+fluctuations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001730-Dominey1" id="pone.0001730-Dominey1"></a>Dominey WJ (1984)  Effects of sexual selection and life history on speciation: Species flocks in African cichlids and Hawaiian <em>Drosophila</em>. In: Echelle AA, Kornfield I, editors. Evolution of Fish Species Flocks. Orono, Maine: University Maine at Orono Press. pp. 231–250.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001730-Danley1" id="pone.0001730-Danley1"></a>Danley PD, Kocher TD (2001) Speciation in rapidly diverging systems: lessons from Lake Malawi. Mol Ecol  10: 1075–1086.  <ul class="find" data-citedArticleID="1067874" data-doi="10.1046/j.1365-294x.2001.01283.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2001.01283.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Speciation+in+rapidly+diverging+systems%3A+lessons+from+Lake+Malawi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Speciation+in+rapidly+diverging+systems%3A+lessons+from+Lake+Malawi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001730-KnightMETurner1" id="pone.0001730-KnightMETurner1"></a>Knight ME Turner GF (2004) Laboratory mating trials indicate incipient speciation by sexual selection among populations of the cichlid fish <em>Pseudotropheus</em> zebra from Lake Malawi. Proc. R. Soc. London B.  271: 675–680.  <ul class="find" data-citedArticleID="1067902" data-doi="10.1098/rspb.2003.2639"><li><a href="http://dx.doi.org/10.1098/rspb.2003.2639" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Laboratory+mating+trials+indicate+incipient+speciation+by+sexual+selection+among+populations+of+the+cichlid+fish+Pseudotropheus+zebra+from+Lake+Malawi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Laboratory+mating+trials+indicate+incipient+speciation+by+sexual+selection+among+populations+of+the+cichlid+fish+Pseudotropheus+zebra+from+Lake+Malawi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001730-McCune1" id="pone.0001730-McCune1"></a>McCune AR, Lovejoy NR (1998)  The relative rate of sympatric and allopatric speciation in fishes: tests using DNA sequence divergence between sister species and among clades. In: Howard D, Berlocher S, editors. Endless Forms: Species and Speciation. Oxford: Oxford University Press. pp. 172–185.  <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0001730-Turner1" id="pone.0001730-Turner1"></a>Turner GF (1999)  Explosive speciation of African cichlids. In: Magurran AE, May RM, editors. Evolution of Biological Diversity. Oxford: Oxford University Press. pp. 113–129.  <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0001730-Cohen1" id="pone.0001730-Cohen1"></a>Cohen AS, Soreghan MJ, Scholz CA (1993) Estimating the age of formation of lakes - an example from Lake Tanganyika, East-African rift system. Geology  21: 511–514.  <ul class="find" data-citedArticleID="1067868" data-doi="10.1130/0091-7613(1993)021<0511:etaofo&gt;2.3.co;2"><li><a href="http://dx.doi.org/10.1130/0091-7613(1993)021<0511:etaofo&gt;2.3.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+the+age+of+formation+of+lakes+-+an+example+from+Lake+Tanganyika%2C+East-African+rift+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+the+age+of+formation+of+lakes+-+an+example+from+Lake+Tanganyika%2C+East-African+rift+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001730-Cohen2" id="pone.0001730-Cohen2"></a>Cohen AS, Lezzar KE, Tiercelin JJ, Soreghan M (1997) New palaeogeographic and lake-level reconstructions of Lake Tanganyika: Implications for tectonic, climatic and biological evolution in a rift lake. Basin Res  9: 107–132.  <ul class="find" data-citedArticleID="1067870" data-doi="10.1046/j.1365-2117.1997.00038.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2117.1997.00038.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+palaeogeographic+and+lake-level+reconstructions+of+Lake+Tanganyika%3A+Implications+for+tectonic%2C+climatic+and+biological+evolution+in+a+rift+lake." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+palaeogeographic+and+lake-level+reconstructions+of+Lake+Tanganyika%3A+Implications+for+tectonic%2C+climatic+and+biological+evolution+in+a+rift+lake.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001730-Takahashi1" id="pone.0001730-Takahashi1"></a>Takahashi T (2003) Systematics of Tanganyikan cichlids fishes (Teleostei: Perciformes). Ichthy Res  50: 367–382.  <ul class="find" data-citedArticleID="1067964" data-doi="10.1007/s10228-003-0181-7"><li><a href="http://dx.doi.org/10.1007/s10228-003-0181-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematics+of+Tanganyikan+cichlids+fishes+%28Teleostei%3A+Perciformes%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematics+of+Tanganyikan+cichlids+fishes+%28Teleostei%3A+Perciformes%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001730-Salzburger1" id="pone.0001730-Salzburger1"></a>Salzburger W, Mack T, Verheyen E, Meyer A (2005) Out of Tanganyika: Genesis, explosive speciation, key-innovations and phylogeography of the haplochromine cichlid fishes. BMC Evol Biol  5: 17.  <ul class="find" data-citedArticleID="1067944"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Out+of+Tanganyika%3A+Genesis%2C+explosive+speciation%2C+key-innovations+and+phylogeography+of+the+haplochromine+cichlid+fishes.&amp;auth=&amp;atitle=Out+of+Tanganyika%3A+Genesis%2C+explosive+speciation%2C+key-innovations+and+phylogeography+of+the+haplochromine+cichlid+fishes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Out+of+Tanganyika%3A+Genesis%2C+explosive+speciation%2C+key-innovations+and+phylogeography+of+the+haplochromine+cichlid+fishes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Out+of+Tanganyika%3A+Genesis%2C+explosive+speciation%2C+key-innovations+and+phylogeography+of+the+haplochromine+cichlid+fishes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001730-Barraclough2" id="pone.0001730-Barraclough2"></a>Barraclough TG, Nee S (2001) Phylogenetics and speciation. Trends Ecol Evol  16: 391–399.  <ul class="find" data-citedArticleID="1067856" data-doi="10.1016/s0169-5347(01)02161-9"><li><a href="http://dx.doi.org/10.1016/s0169-5347(01)02161-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetics+and+speciation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetics+and+speciation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001730-Seehausen1" id="pone.0001730-Seehausen1"></a>Seehausen O (2006) African cichlid fish: a model system in adaptive radiation research. Proc R Soc London B  273: 1987–1998.  <ul class="find" data-citedArticleID="1067950" data-doi="10.1098/rspb.2006.3539"><li><a href="http://dx.doi.org/10.1098/rspb.2006.3539" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=African+cichlid+fish%3A+a+model+system+in+adaptive+radiation+research." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22African+cichlid+fish%3A+a+model+system+in+adaptive+radiation+research.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001730-Cane1" id="pone.0001730-Cane1"></a>Cane MA, Molnar P (2001) Closing of the Indonesian seaway as a precursor to East African aridification around 3–4 million years ago. Nature  411: 157–162.  <ul class="find" data-citedArticleID="1067862" data-doi="10.1038/35075500"><li><a href="http://dx.doi.org/10.1038/35075500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Closing+of+the+Indonesian+seaway+as+a+precursor+to+East+African+aridification+around+3%E2%80%934+million+years+ago." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Closing+of+the+Indonesian+seaway+as+a+precursor+to+East+African+aridification+around+3%E2%80%934+million+years+ago.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001730-Tiercelin1" id="pone.0001730-Tiercelin1"></a>Tiercelin JJ, Mondeguer A (1991)  The geology of the Tanganyika trough. In: Martens K, Goddeeris B, Coulter G, editors. Lake Tanganyika and its life. Oxford: Oxford University Press, Oxford. pp. 7–48.  <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0001730-Clabaut1" id="pone.0001730-Clabaut1"></a>Clabaut C, Salzburger W, Meyer A (2005) Comparative phylogenetic analyses of the adaptive radiation of Lake Tanganyika cichlid fish: Nuclear sequences are less homoplasious but also less informative than mitochondrial DNA. J Mol Evol  61: 666–681.  <ul class="find" data-citedArticleID="1067866" data-doi="10.1007/s00239-004-0217-2"><li><a href="http://dx.doi.org/10.1007/s00239-004-0217-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+phylogenetic+analyses+of+the+adaptive+radiation+of+Lake+Tanganyika+cichlid+fish%3A+Nuclear+sequences+are+less+homoplasious+but+also+less+informative+than+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+phylogenetic+analyses+of+the+adaptive+radiation+of+Lake+Tanganyika+cichlid+fish%3A+Nuclear+sequences+are+less+homoplasious+but+also+less+informative+than+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001730-Genner2" id="pone.0001730-Genner2"></a>Genner MJ, Seehausen O, Lunt DH, Joyce DA, Shaw PW, et al.  (2007) Age of cichlids: New dates for ancient lake fish radiations. Mol Biol Evol  24: 1269–1282.  <ul class="find" data-citedArticleID="1067896" data-doi="10.1093/molbev/msm050"><li><a href="http://dx.doi.org/10.1093/molbev/msm050" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Age+of+cichlids%3A+New+dates+for+ancient+lake+fish+radiations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Age+of+cichlids%3A+New+dates+for+ancient+lake+fish+radiations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001730-Nishida1" id="pone.0001730-Nishida1"></a>Nishida M (1997)  Phylogenetic relationships and evolution of Tanganyikan cichlids: a molecular perspective. In: Kawanabe H, Hori M, Nagoshi M, editors. Fish Communities in Lake. Tanganyika: Kyoto University Press. pp. 3–23.  <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0001730-Terai1" id="pone.0001730-Terai1"></a>Terai Y, Takahashi K, Nishida M, Sato T, Okada N (2003) Using SINEs to probe ancient explosive speciation: “Hidden” radiation of African cichlids? Mol Biol Evol  20: 924–930.  <ul class="find" data-citedArticleID="1067966" data-doi="10.1093/molbev/msg104"><li><a href="http://dx.doi.org/10.1093/molbev/msg104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+SINEs+to+probe+ancient+explosive+speciation%3A+%E2%80%9CHidden%E2%80%9D+radiation+of+African+cichlids%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+SINEs+to+probe+ancient+explosive+speciation%3A+%E2%80%9CHidden%E2%80%9D+radiation+of+African+cichlids%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001730-Salzburger2" id="pone.0001730-Salzburger2"></a>Salzburger W, Meyer A, Baric S, Verheyen E, Strumbauer C (2002) Lake level fluctuations synchronize genetic divergences of cichlid fishes in African lakes. Syst Biol  1: 1–23.  <ul class="find" data-citedArticleID="1067946" data-doi="10.1093/oxfordjournals.molbev.a003788"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003788" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lake+level+fluctuations+synchronize+genetic+divergences+of+cichlid+fishes+in+African+lakes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lake+level+fluctuations+synchronize+genetic+divergences+of+cichlid+fishes+in+African+lakes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001730-Drummond1" id="pone.0001730-Drummond1"></a>Drummond AJ, Ho SYW, Phillips MJ, Rambaut A (2006) Relaxed phylogenetics and dating with confidence. PLOS Biol  4: 699–710.  <ul class="find" data-citedArticleID="1067886" data-doi="10.1371/journal.pbio.0040088"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relaxed+phylogenetics+and+dating+with+confidence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relaxed+phylogenetics+and+dating+with+confidence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001730-Nee1" id="pone.0001730-Nee1"></a>Nee S, May RM, Harvey PH (1994) The reconstructed evolutionary process. Phil Trans R Soc Lond B  344: 305–311.  <ul class="find" data-citedArticleID="1067918" data-doi="10.1098/rstb.1994.0068"><li><a href="http://dx.doi.org/10.1098/rstb.1994.0068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+reconstructed+evolutionary+process." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+reconstructed+evolutionary+process.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001730-Baldwin1" id="pone.0001730-Baldwin1"></a>Baldwin BG, Sanderson MJ (1998) Age and rate of diversification of the Hawaiian silversword alliance (Compositae) Proc Natl Acad Sci USA  95: 9402–9406.  <ul class="find" data-citedArticleID="1067852" data-doi="10.1073/pnas.95.16.9402"><li><a href="http://dx.doi.org/10.1073/pnas.95.16.9402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Age+and+rate+of+diversification+of+the+Hawaiian+silversword+alliance+%28Compositae%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Age+and+rate+of+diversification+of+the+Hawaiian+silversword+alliance+%28Compositae%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001730-McCune2" id="pone.0001730-McCune2"></a>McCune AR (1997)  How fast is speciation: molecular, geological and phylogenetic evidence from adaptive radiations of fishes. In: Givnish TJ, Sytsma K, editors. Molecular Evolution and Adaptive Radiation. Cambridge: Cambridge University Press. pp. 585–610.  <ul class="find-nolinks"></ul></li><li><span class="label">35.
              </span><a name="pone.0001730-Nagl1" id="pone.0001730-Nagl1"></a>Nagl S, Tichy H, Mayer WE, Takezaki N, Takahata N, et al.  (2000) The origin and age of haplochromine fishes in Lake Victoria, East Africa. Proc R Soc London B  267: 1049–1061.  <ul class="find" data-citedArticleID="1067916" data-doi="10.1098/rspb.2000.1109"><li><a href="http://dx.doi.org/10.1098/rspb.2000.1109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+origin+and+age+of+haplochromine+fishes+in+Lake+Victoria%2C+East+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+origin+and+age+of+haplochromine+fishes+in+Lake+Victoria%2C+East+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001730-Purvis2" id="pone.0001730-Purvis2"></a>Purvis A, Nee S, Harvey PH (1995) Macroevolutionary inferences from primate phylogeny. Proc R Soc London B  260: 329–333.  <ul class="find" data-citedArticleID="1067930" data-doi="10.1098/rspb.1995.0100"><li><a href="http://dx.doi.org/10.1098/rspb.1995.0100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Macroevolutionary+inferences+from+primate+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Macroevolutionary+inferences+from+primate+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001730-Nee2" id="pone.0001730-Nee2"></a>Nee S (2001) Inferring speciation rates from phylogenies. Evolution  55: 661–668.  <ul class="find" data-citedArticleID="1067920" data-doi="10.1111/j.0014-3820.2001.tb00801.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2001.tb00801.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+speciation+rates+from+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+speciation+rates+from+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001730-Barraclough3" id="pone.0001730-Barraclough3"></a>Barraclough TG, Vogler AP (2002) Recent diversification rates in North American tiger beetles estimated from a dated mtDNA phylogenetic tree. Mol Biol Evol  19: 1706–1716.  <ul class="find" data-citedArticleID="1067858" data-doi="10.1093/oxfordjournals.molbev.a003993"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003993" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+diversification+rates+in+North+American+tiger+beetles+estimated+from+a+dated+mtDNA+phylogenetic+tree." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+diversification+rates+in+North+American+tiger+beetles+estimated+from+a+dated+mtDNA+phylogenetic+tree.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001730-Konings1" id="pone.0001730-Konings1"></a>Konings A (1998) Tanganyika Cichlids in their Natural Habitat. El Paso, TX: Cichlid Press.   <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0001730-Yanagisawa1" id="pone.0001730-Yanagisawa1"></a>Yanagisawa Y, Ochi H, Gashagaza MM (1997)  Habitat use in cichlid fishes for breeding. In: Kawanabe H, Hori M, Nagoshi M, editors. Fish Communities in Lake. Tanganyika: Kyoto University Press. pp. 151–173.  <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0001730-Gavrilets1" id="pone.0001730-Gavrilets1"></a>Gavrilets S, Vose A (2005) Dynamic patterns of adaptive radiation. Proc Natl Acad Sci USA  102: 18040–18045.  <ul class="find" data-citedArticleID="1067892" data-doi="10.1073/pnas.0506330102"><li><a href="http://dx.doi.org/10.1073/pnas.0506330102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+patterns+of+adaptive+radiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+patterns+of+adaptive+radiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001730-Meyer1" id="pone.0001730-Meyer1"></a>Meyer A, Kocher TD, Basasibwaki P, Wilson  (1990) Monophyletic origin of Lake Victoria cichlid fishes suggested by mitochondrial-DNA sequences. Nature  347: 550–553.  <ul class="find" data-citedArticleID="1067914" data-doi="10.1038/347550a0"><li><a href="http://dx.doi.org/10.1038/347550a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monophyletic+origin+of+Lake+Victoria+cichlid+fishes+suggested+by+mitochondrial-DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monophyletic+origin+of+Lake+Victoria+cichlid+fishes+suggested+by+mitochondrial-DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001730-Sturmbauer1" id="pone.0001730-Sturmbauer1"></a>Sturmbauer C, Baric S, Salzburger W, Rüber L, Verheyen E (2001) Lake level fluctuations synchronize genetic divergences of cichlid fishes in African lakes. Mol Biol Evol  18: 144–154.  <ul class="find" data-citedArticleID="1067960" data-doi="10.1093/oxfordjournals.molbev.a003788"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003788" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lake+level+fluctuations+synchronize+genetic+divergences+of+cichlid+fishes+in+African+lakes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lake+level+fluctuations+synchronize+genetic+divergences+of+cichlid+fishes+in+African+lakes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001730-Pybus1" id="pone.0001730-Pybus1"></a>Pybus OG, Harvey PH (2000) Testing macro-evolutionary models using incomplete molecular phylogenies. Proc. R. Soc. Lond. B  267: 2267–2272.  <ul class="find" data-citedArticleID="1067932" data-doi="10.1098/rspb.2000.1278"><li><a href="http://dx.doi.org/10.1098/rspb.2000.1278" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Testing+macro-evolutionary+models+using+incomplete+molecular+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Testing+macro-evolutionary+models+using+incomplete+molecular+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001730-BindaEmonds1" id="pone.0001730-BindaEmonds1"></a>Binda-Emonds ORP, Cardillo M, Jones KE, MacPhee RDE, Beck RMD, et al.  (2007) The delayed rise of present-day mammals. Nature  446: 507–512.  <ul class="find" data-citedArticleID="1067860" data-doi="10.1038/nature05634"><li><a href="http://dx.doi.org/10.1038/nature05634" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+delayed+rise+of+present-day+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+delayed+rise+of+present-day+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001730-Day1" id="pone.0001730-Day1"></a>Day JJ, Wilkinson M (2006) On the origin of the <em>Synodontis</em> catfish species flock from Lake Tanganyika. Biol. Letts  2: 548–552.  <ul class="find" data-citedArticleID="1067880" data-doi="10.1098/rsbl.2006.0532"><li><a href="http://dx.doi.org/10.1098/rsbl.2006.0532" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+origin+of+the+Synodontis+catfish+species+flock+from+Lake+Tanganyika." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+origin+of+the+Synodontis+catfish+species+flock+from+Lake+Tanganyika.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001730-Marijnissen1" id="pone.0001730-Marijnissen1"></a>Marijnissen SAE, Michel E, Daniels SR, Erpenbeck D, Menken SBJ, et al.  (2006) Molecular evidence for recent divergence of Lake Tanganyika endemic crabs (Decapoda : Platythelphusidae). Mol. Phylogenet. Evol.  40: 628–634.  <ul class="find" data-citedArticleID="1067908" data-doi="10.1016/j.ympev.2006.03.025"><li><a href="http://dx.doi.org/10.1016/j.ympev.2006.03.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evidence+for+recent+divergence+of+Lake+Tanganyika+endemic+crabs+%28Decapoda+%3A+Platythelphusidae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evidence+for+recent+divergence+of+Lake+Tanganyika+endemic+crabs+%28Decapoda+%3A+Platythelphusidae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001730-Seehausen2" id="pone.0001730-Seehausen2"></a>Seehausen O, van Alphen JJM, Witte F (1997) Cichlid fish diversity threatened by eutrophication that curbs sexual selection. Science  277: 1808–1811.  <ul class="find" data-citedArticleID="1067952" data-doi="10.1126/science.277.5333.1808"><li><a href="http://dx.doi.org/10.1126/science.277.5333.1808" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cichlid+fish+diversity+threatened+by+eutrophication+that+curbs+sexual+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cichlid+fish+diversity+threatened+by+eutrophication+that+curbs+sexual+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001730-Langenberg1" id="pone.0001730-Langenberg1"></a>Langenberg VT, Mwape LM, Tschibangu K, Tumba J-M, Koelmans AA, et al.  (2002) Comparison of thermal stratification, light attenuation, and chlorophyll-<em>a</em> dynamics between the ends of Lake Tanganyika. J Aquat Ecosyst Health  5: 255–265.  <ul class="find" data-citedArticleID="1067906" data-doi="10.1080/14634980290031956"><li><a href="http://dx.doi.org/10.1080/14634980290031956" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+thermal+stratification%2C+light+attenuation%2C+and+chlorophyll-a+dynamics+between+the+ends+of+Lake+Tanganyika." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+thermal+stratification%2C+light+attenuation%2C+and+chlorophyll-a+dynamics+between+the+ends+of+Lake+Tanganyika.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001730-Flynn1" id="pone.0001730-Flynn1"></a>Flynn JJ, Finarelli JA, Zehr S, Hsu J, Nedbal MA (2005) Molecular phylogeny of the Carnivora (Mammalia): Assessing the impact of increased sampling on resolving enigmatic relationships. Syst Biol  54: 317–337.  <ul class="find" data-citedArticleID="1067890"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+phylogeny+of+the+Carnivora+%28Mammalia%29%3A+Assessing+the+impact+of+increased+sampling+on+resolving+enigmatic+relationships.&amp;auth=&amp;atitle=Molecular+phylogeny+of+the+Carnivora+%28Mammalia%29%3A+Assessing+the+impact+of+increased+sampling+on+resolving+enigmatic+relationships." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogeny+of+the+Carnivora+%28Mammalia%29%3A+Assessing+the+impact+of+increased+sampling+on+resolving+enigmatic+relationships." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogeny+of+the+Carnivora+%28Mammalia%29%3A+Assessing+the+impact+of+increased+sampling+on+resolving+enigmatic+relationships.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001730-Zwickl1" id="pone.0001730-Zwickl1"></a>Zwickl DJ, Hills DM (2002) Increased taxon sampling greatly reduces phylogenetic error. Syst Biol  51: 588–598.  <ul class="find" data-citedArticleID="1067978"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Increased+taxon+sampling+greatly+reduces+phylogenetic+error.&amp;auth=&amp;atitle=Increased+taxon+sampling+greatly+reduces+phylogenetic+error." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Increased+taxon+sampling+greatly+reduces+phylogenetic+error." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Increased+taxon+sampling+greatly+reduces+phylogenetic+error.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001730-Thompson1" id="pone.0001730-Thompson1"></a>Thompson JD, Gibson TJ, Plewniak F, Jeanmougin J, Higgins DG (1997) The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res  25: 4876–4882.  <ul class="find" data-citedArticleID="1067968" data-doi="10.1093/nar/25.24.4876"><li><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001730-Rambaut1" id="pone.0001730-Rambaut1"></a>Rambaut A (1996) Se-Al: Sequence Alignment Editor.   <a href="http://evolve.zoo.ox.ac.uk/">http://evolve.zoo.ox.ac.uk/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">54.
              </span><a name="pone.0001730-Posada1" id="pone.0001730-Posada1"></a>Posada D, Crandall K (1998) MODELTEST: testing the model of DNA substitution. Bioinform  14: 817–818.  <ul class="find" data-citedArticleID="1067926" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MODELTEST%3A+testing+the+model+of+DNA+substitution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MODELTEST%3A+testing+the+model+of+DNA+substitution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001730-Ronquist1" id="pone.0001730-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinform  19: 1572–1574.  <ul class="find" data-citedArticleID="1067940" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001730-Zwickl2" id="pone.0001730-Zwickl2"></a>Zwickl DJ (2006) Genetic Algorithm Approaches for the Phylogenetic Analysis of Large Biological Sequence Datasets under the Maximum Likelihood Criterion. Ph.D. Dissertation, University of Texas at Austin.   Available from <a href="http://www.zo.utexas.edu/faculty/antisense/garli/Garli.html">http://www.zo.utexas.edu/faculty/antisen​se/garli/Garli.html</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">57.
              </span><a name="pone.0001730-Shimodaira1" id="pone.0001730-Shimodaira1"></a>Shimodaira H (2002) An approximately unbiased test of phylogenetic tree selection. Syst Biol  51: 492–508.  <ul class="find" data-citedArticleID="1067954"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+approximately+unbiased+test+of+phylogenetic+tree+selection.&amp;auth=&amp;atitle=An+approximately+unbiased+test+of+phylogenetic+tree+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+approximately+unbiased+test+of+phylogenetic+tree+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+approximately+unbiased+test+of+phylogenetic+tree+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001730-Shimodaira2" id="pone.0001730-Shimodaira2"></a>Shimodaira H, Hasegawa M (2001) CONSEL: for assessing the confidence of phylogenetic tree selection. Bioinform  17: 1246–1247.  <ul class="find" data-citedArticleID="1067956" data-doi="10.1093/bioinformatics/17.12.1246"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.12.1246" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CONSEL%3A+for+assessing+the+confidence+of+phylogenetic+tree+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CONSEL%3A+for+assessing+the+confidence+of+phylogenetic+tree+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001730-Swofford1" id="pone.0001730-Swofford1"></a>Swofford DL (2002) PAUP*: Phylogentic analysis using parsimony (* and other methods), version 4.0b10. Sunderland, MA: Sinauer.   <ul class="find-nolinks"></ul></li><li><span class="label">60.
              </span><a name="pone.0001730-Huelsenbeck1" id="pone.0001730-Huelsenbeck1"></a>Huelsenbeck JP, Crandall KA (1997) Phylogeny estimation and hypothesis testing using maximum likelihood. Ann Rev Ecol Syst  28: 437–466.  <ul class="find" data-citedArticleID="1067898" data-doi="10.1146/annurev.ecolsys.28.1.437"><li><a href="http://dx.doi.org/10.1146/annurev.ecolsys.28.1.437" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+estimation+and+hypothesis+testing+using+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+estimation+and+hypothesis+testing+using+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001730-Drummond2" id="pone.0001730-Drummond2"></a>Drummond AJ, Rambaut A (2007) BEAST: Bayesian evolutionary analysis by sampling trees. BMC Evol Biol  7: 214.  <ul class="find" data-citedArticleID="1067888" data-doi="10.1186/1471-2148-7-214"><li><a href="http://dx.doi.org/10.1186/1471-2148-7-214" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BEAST%3A+Bayesian+evolutionary+analysis+by+sampling+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BEAST%3A+Bayesian+evolutionary+analysis+by+sampling+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001730-Rosen1" id="pone.0001730-Rosen1"></a>Rosen DE (1978) Vicariant patterns and historical explanation in biogeography. Syst Zool  27: 159–188.  <ul class="find" data-citedArticleID="1067942" data-doi="10.2307/2412970"><li><a href="http://dx.doi.org/10.2307/2412970" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vicariant+patterns+and+historical+explanation+in+biogeography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vicariant+patterns+and+historical+explanation+in+biogeography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001730-Paradis1" id="pone.0001730-Paradis1"></a>Paradis E, Claude J, Strimmer K (2004) APE: Analyses of Phylogenetics and Evolution in R language. Bioinform  20: 289–290.  <ul class="find" data-citedArticleID="1067924" data-doi="10.1093/bioinformatics/btg412"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg412" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=APE%3A+Analyses+of+Phylogenetics+and+Evolution+in+R+language." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22APE%3A+Analyses+of+Phylogenetics+and+Evolution+in+R+language.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001730-R1" id="pone.0001730-R1"></a>R Development Core Team (2006) R: A Language and Environment for Statistical Computing: Vienna, Austria.   <ul class="find-nolinks"></ul></li><li><span class="label">65.
              </span><a name="pone.0001730-Wood1" id="pone.0001730-Wood1"></a>Wood SN (2006) Generalized Additive Models: An Introduction with R:. Boca Raton, Florida: Chapman &amp; Hall/CRC.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.XML" value="108060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.PDF" value="296857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g001.PNG_L" value="122876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g001.PNG_M" value="89234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g001.PNG_S" value="12282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g001.TIF" value="559794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g001.PNG_I" value="39207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g002.PNG_L" value="59825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g002.PNG_M" value="20633"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g002.PNG_S" value="4980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g002.TIF" value="223248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g002.PNG_I" value="7780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g003.PNG_L" value="983335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g003.PNG_M" value="77806"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g003.PNG_S" value="12569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g003.TIF" value="1726256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g003.PNG_I" value="83572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g004.PNG_L" value="79005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g004.PNG_M" value="32142"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g004.PNG_S" value="7027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g004.TIF" value="326554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.g004.PNG_I" value="11461"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.t001.PNG_L" value="172067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.t001.PNG_M" value="98082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.t001.PNG_S" value="12453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.t001.TIF" value="776382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.t001.PNG_I" value="30332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.s001.EPS" value="613417"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.s002.PDF" value="128734"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.s003.PDF" value="79950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001730.s004.PDF" value="71927"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001730&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001730" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001730&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001730&volume=&issue=&title=Tempo and Mode of Diversification of Lake Tanganyika Cichlid Fishes&author_name=Julia%20J.%20Day%2C%20James%20A.%20Cotton%2C%20Timothy%20G.%20Barraclough&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001730" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001730" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001730" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001730&amp;t=Tempo%20and%20Mode%20of%20Diversification%20of%20Lake%20Tanganyika%20Cichlid%20Fishes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001730&title=Tempo%20and%20Mode%20of%20Diversification%20of%20Lake%20Tanganyika%20Cichlid%20Fishes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001730&amp;title=Tempo%20and%20Mode%20of%20Diversification%20of%20Lake%20Tanganyika%20Cichlid%20Fishes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001730" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001730&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Tempo and Mode of Diversification of Lake Tanganyika Cichlid Fishes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001730';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Tempo%20and%20Mode%20of%20Diversification%20of%20Lake%20Tanganyika%20Cichlid%20Fishes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001730" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001730" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+phylogenetics%22" title="Search for articles in the subject area:'Animal phylogenetics'"><div class="flagText">Animal phylogenetics</div></a>
              <div data-categoryid="20335" data-articleid="26284"
                   data-categoryname="Animal phylogenetics"
                   class="flagImage" title="Flag 'Animal phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bayes+theorem%22" title="Search for articles in the subject area:'Bayes theorem'"><div class="flagText">Bayes theorem</div></a>
              <div data-categoryid="32251" data-articleid="26284"
                   data-categoryname="Bayes theorem"
                   class="flagImage" title="Flag 'Bayes theorem' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Conservation+genetics%22" title="Search for articles in the subject area:'Conservation genetics'"><div class="flagText">Conservation genetics</div></a>
              <div data-categoryid="40473" data-articleid="26284"
                   data-categoryname="Conservation genetics"
                   class="flagImage" title="Flag 'Conservation genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fossil+calibration%22" title="Search for articles in the subject area:'Fossil calibration'"><div class="flagText">Fossil calibration</div></a>
              <div data-categoryid="33891" data-articleid="26284"
                   data-categoryname="Fossil calibration"
                   class="flagImage" title="Flag 'Fossil calibration' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lakes%22" title="Search for articles in the subject area:'Lakes'"><div class="flagText">Lakes</div></a>
              <div data-categoryid="18741" data-articleid="26284"
                   data-categoryname="Lakes"
                   class="flagImage" title="Flag 'Lakes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="26284"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Speciation%22" title="Search for articles in the subject area:'Speciation'"><div class="flagText">Speciation</div></a>
              <div data-categoryid="17491" data-articleid="26284"
                   data-categoryname="Speciation"
                   class="flagImage" title="Flag 'Speciation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Species+diversity%22" title="Search for articles in the subject area:'Species diversity'"><div class="flagText">Species diversity</div></a>
              <div data-categoryid="47593" data-articleid="26284"
                   data-categoryname="Species diversity"
                   class="flagImage" title="Flag 'Species diversity' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2839'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1229'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=235&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=4885">Why combine LM and LV?</a><br>Posted by DEEB2008</p>
      <p><a href="/annotation/listThread.action?root=9629">from</a><br>Posted by DEEB2008</p>
      <p><a href="/annotation/listThread.action?root=1753">cladogenesis</a><br>Posted by DEEB2008</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
