

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Probabilistic Inference of Transcription Factor Binding from Multiple Data Sources</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001820"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001820" />

    <meta name="citation_title" content="Probabilistic Inference of Transcription Factor Binding from Multiple Data Sources"/>
    <meta itemprop="name" content="Probabilistic Inference of Transcription Factor Binding from Multiple Data Sources"/>

      <meta name="citation_author" content="Harri Lähdesmäki"/>
            <meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="Alistair G. Rust"/>
            <meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="Ilya Shmulevich"/>
            <meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington, United States of America"/>

    <meta name="citation_date" content="2008/3/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001820.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1820"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_author=EH Davidson; citation_number=1; citation_date=2001; citation_publisher=Academic Press, 1st edition; " />
      <meta name="citation_reference" content="citation_title=Practical strategies for discovering regulatory DNA sequence motifs.; citation_author=KD MacIsaac; citation_author=E Fraenkel; citation_journal_title=PLoS Computational Biology; citation_volume=2; citation_number=2; citation_pages=e36; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Assessing computational tools for the discovery of transcription factor binding sites.; citation_author=M Tompa; citation_author=N Li; citation_author=TL Bailey; citation_author=GM Church; citation_author=B De Moor; citation_journal_title=Nature Biotechnology; citation_volume=23; citation_number=3; citation_pages=137-144; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Computational identification of Cis-regulatory elements associated with groups of functionally related genes in Saccharomyces cerevisiae.; citation_author=JD Hughes; citation_author=PW Estep; citation_author=S Tavazoie; citation_author=GM Church; citation_journal_title=Journal of Molecular Biology; citation_volume=296; citation_number=4; citation_pages=1205-1214; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements.; citation_author=M Kellis; citation_author=N Patterson; citation_author=M Endrizzi; citation_author=B Birren; citation_author=ES Lander; citation_journal_title=Nature; citation_volume=423; citation_number=5; citation_pages=241-254; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=PhyloGibbs: A Gibbs sampling motif finder that incorporates phylogeny.; citation_author=R Siddharthan; citation_author=ED Siggia; citation_author=E van Nimwegen; citation_journal_title=PLoS Computational Biology; citation_volume=1; citation_number=6; citation_pages=e67; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An algorithm for finding protein-DNA binding sites with applications to chromatin-immunoprecipitation microarray experiments.; citation_author=XS Liu; citation_author=DL Brutlag; citation_author=JS Liu; citation_journal_title=Nature Biotechnology; citation_volume=20; citation_number=7; citation_pages=835-839; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulatory code of a eukaryotic genome.; citation_author=CT Harbison; citation_author=DB Gordon; citation_author=TI Lee; citation_author=NJ Rinaldi; citation_author=KD Macisaac; citation_journal_title=Nature; citation_volume=431; citation_number=8; citation_pages=99-104; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=High-resolution computational models of genome binding events.; citation_author=Y Qi; citation_author=A Rolfe; citation_author=KD MacIsaac; citation_author=GK Gerber; citation_author=D Pokholok; citation_journal_title=Nature Biotechnology; citation_volume=24; citation_number=9; citation_pages=963-970; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Informative priors based on transcription factor structural class improve de novo motif discovery.; citation_author=L Narlikar; citation_author=R Gordân; citation_author=U Ohler; citation_author=AJ Hartemink; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=10; citation_pages=e384-e392; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Nucleosome occupancy information improves de novo motif discovery.; citation_author=L Narlikar; citation_author=R Gordân; citation_author=A Hartemink; citation_number=11; citation_pages=107-121; citation_date=2007; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=A nucleosome-guided map of transcription factor binding sites in yeast.; citation_author=L Narlikar; citation_author=R Gordân; citation_author=AJ Hartemink; citation_journal_title=PLoS Computational Biology; citation_number=12; citation_date=In press; " />
      <meta name="citation_reference" content="citation_title=TRANSFAC: an integrated system for gene expression regulation.; citation_author=E Wingender; citation_author=X Chen; citation_author=R Hehl; citation_author=H Karas; citation_author=I Liebich; citation_journal_title=Nucleic Acids Research; citation_volume=28; citation_number=13; citation_pages=316-319; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=JASPAR: an open-access database for eukaryotic transcription factor binding profiles.; citation_author=A Sandelin; citation_author=W Alkema; citation_author=P Engström; citation_author=WW Wasserman; citation_author=B Lenhard; citation_journal_title=Nucleic Acids Research; citation_volume=32; citation_number=14; citation_pages=D91-D94; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rapid analysis of the DNA-binding specificities of transcription factors with DNA microarrays.; citation_author=S Mukherjee; citation_author=MF Berger; citation_author=G Jona; citation_author=XS Wang; citation_author=D Muzzey; citation_journal_title=Nature Genetics; citation_volume=36; citation_number=15; citation_pages=1331-1339; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Compact, universal DNA microarrays to comprehensively determine transcription-factor binding site specificities.; citation_author=MF Berger; citation_author=AA Philippakis; citation_author=AM Qureshi; citation_author=FS He; citation_author=PW Estep; citation_journal_title=Nature Biotechnology; citation_volume=24; citation_number=16; citation_pages=1429-1435; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Defining the sequence-recognition profile of DNA-binding molecules.; citation_author=CL Warren; citation_author=NCS Kratochvil; citation_author=KE Hauschild; citation_author=S Foister; citation_author=ML Brezinski; citation_journal_title=Proceedings of the National Academy of Sciences of the USA; citation_volume=103; citation_number=17; citation_pages=867-872; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A systems approach to measuring the binding energy landscapes of transcription factors.; citation_author=SJ Maerkl; citation_author=SR Quake; citation_journal_title=Science; citation_volume=315; citation_number=18; citation_pages=233-237; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Quantifying DNA-protein binding specificities by using oligonucleotide mass tags and mass spectroscopy.; citation_author=L Zhang; citation_author=S Kasif; citation_author=CR Cantor; citation_journal_title=Proceedings of the National Academy of Sciences of the USA; citation_volume=104; citation_number=19; citation_pages=3061-3066; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A plan for when the ChIPs are down.; citation_author=M Eisenstein; citation_journal_title=Nature Methods; citation_volume=2; citation_number=20; citation_pages=5; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Computer methods to locate signals in nucleic acid sequences.; citation_author=R Staden; citation_journal_title=Nucleic Acids Research; citation_volume=12; citation_number=21; citation_pages=505-519; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=MatInd and MatInspector: new fast and versatile tools for detection of consensus matches in nucleotide sequence data.; citation_author=K Quandt; citation_author=K Frechl; citation_author=H Karas; citation_author=E Wingender; citation_author=T Werner; citation_journal_title=Nucleic Acids Research; citation_volume=23; citation_number=22; citation_pages=4878-4884; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The statistical significance of nucleotide position-weight matrix matches.; citation_author=JM Claverie; citation_author=S Audic; citation_journal_title=Bioinformatics; citation_volume=12; citation_number=23; citation_pages=431-439; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Fast probabilistic analysis of sequence function using scoring matrices.; citation_author=TD Wu; citation_author=CG Nevill-Manning; citation_author=DL Brutlag; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=24; citation_pages=233-244; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Determination of local statistical significance of patterns in Markov sequences with application to promoter element identification.; citation_author=H Huang; citation_author=MCJ Kao; citation_author=X Zhou; citation_author=JS Liu; citation_author=WH Wong; citation_journal_title=Journal of Computational Biology; citation_volume=11; citation_number=25; citation_pages=1-14; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Finding motifs in promoter regions.; citation_author=L Hertzberg; citation_author=O Zuk; citation_author=G Getz; citation_author=E Domany; citation_journal_title=Journal of Computational Biology; citation_volume=12; citation_number=26; citation_pages=314-330; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=CIS: compound importance sampling method for protein-DNA binding site p-value estimation.; citation_author=Y Barash; citation_author=G Elidan; citation_author=T Kaplan; citation_author=N Friedman; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=27; citation_pages=596-600; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Detection-theoretic analysis of MatInspector.; citation_author=IV Bajić; citation_journal_title=IEEE Transactions on Signal Processing; citation_volume=54; citation_number=28; citation_pages=2388-2393; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Applied bioinformatics for the identification of regulatory elements.; citation_author=WW Wasserman; citation_author=A Sandelin; citation_journal_title=Nature Reviews Genetics; citation_volume=5; citation_number=29; citation_pages=276-287; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Nucleotides of transcription factor binding sites exert interdependent effects on the binding affinities of transcription factors.; citation_author=ML Bulyk; citation_author=PLF Johnson; citation_author=GM Church; citation_journal_title=Nucleic Acids Research; citation_volume=30; citation_number=30; citation_pages=1255-1261; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Modeling within-motif dependence for transcription factor binding site predictions.; citation_author=Q Zhou; citation_author=JS Liu; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=31; citation_pages=909-916; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Modeling dependencies in protein-DNA binding sites.; citation_author=Y Barash; citation_author=G Elidan; citation_author=N Friedman; citation_author=T Kaplan; citation_number=32; citation_date=2003; citation_publisher=ACM; " />
      <meta name="citation_reference" content="citation_title=The human genome browser at UCSC.; citation_author=WJ Kent; citation_author=CW Sugnet; citation_author=TS Furey; citation_author=KM Roskin; citation_author=TH Pringle; citation_journal_title=Genome Research; citation_volume=12; citation_number=33; citation_pages=996-1006; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=ORegAnno: An open access database and curation system for literature-derived promoters, transcription factor binding sites and regulatory variation.; citation_author=SB Montgomery; citation_author=OL Griffith; citation_author=MC Sleumer; citation_author=CM Bergman; citation_author=M Bilenky; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=34; citation_pages=637-640; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=ABS: a database of Annotated regulatory Binding Sites from orthologous promoters.; citation_author=E Blanco; citation_author=D Farré; citation_author=MM Albà; citation_author=X Messeguer; citation_author=R Guigó; citation_journal_title=Nucleic Acids Research; citation_volume=34; citation_number=35; citation_pages=D63-D67; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Detection of cis-element clusters in higher eukaryotic DNA.; citation_author=MC Frith; citation_author=U Hansen; citation_author=Z Weng; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=36; citation_pages=878-889; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Computational detection of genomic cis-regulatory modules applied to body patterning in the early Drosophila embryo.; citation_author=N Rajewsky; citation_author=M Vergassola; citation_author=U Gaul; citation_author=ED Siggia; citation_journal_title=BMC Bioinformatics; citation_volume=3; citation_number=37; citation_pages=30; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A probabilistic method to detect regulatory modules.; citation_author=S Sinha; citation_author=E van Nimwegen; citation_author=ED Siggia; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=38; citation_pages=i292-i301; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Detecting subtle sequence signals: A Gibbs sampling strategy for multiple alignment.; citation_author=CE Lawrence; citation_author=SF Altschul; citation_author=MS Boguski; citation_author=JS Liu; citation_author=AF Neuwald; citation_journal_title=Science; citation_volume=262; citation_number=39; citation_pages=208-214; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Fitting a mixture model by expectation maximization to discover motifs in biopolymers.; citation_author=TL Bailey; citation_author=C Elkan; citation_number=40; citation_pages=28-36; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The value of prior knowledge in discovering motifs with MEME.; citation_author=TL Bailey; citation_author=CP Elkan; citation_number=41; citation_pages=21-29; citation_date=1995; citation_publisher=AAAI Press, volume 3; " />
      <meta name="citation_reference" content="citation_title=A Gibbs sampling method to detect overrepresented motifs in the upstream regions of coexpressed genes.; citation_author=G Thijs; citation_author=K Marchal; citation_author=M Lescot; citation_author=S Rombauts; citation_author=BD Moor; citation_journal_title=Journal of Computational Biology; citation_volume=9; citation_number=42; citation_pages=447-464; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Predicting protein-peptide interactions via a network-based motif sampler.; citation_author=DJ Reiss; citation_author=B Schwikowski; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=43; citation_pages=i274-i282; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=On counting position weight matrix matches in a sequence, with application to discriminative motif finding.; citation_author=S Sinha; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=44; citation_pages=e454-e463; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A regularized discriminative model for the prediction of protein-peptide interactions.; citation_author=WP Lehrach; citation_author=D Husmeier; citation_author=CKI Williams; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=45; citation_pages=532-540; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=ConSite: web-based prediction of regulatory elements using cross-species comparison.; citation_author=A Sandelin; citation_author=WW Wasserman; citation_author=B Lenhard; citation_journal_title=Nucleic Acids Research; citation_volume=32; citation_number=46; citation_pages=W249-W252; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MAPPER: a search engine for the computational identification of putative transcription factor binding sites in multiple genomes.; citation_author=VD Marinescu; citation_author=IS Kohane; citation_author=A Riva; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=47; citation_pages=79; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Computational identification of transcriptional regulatory elements in DNA sequence.; citation_author=D GuhaThakurta; citation_journal_title=Nucleic Acids Research; citation_volume=34; citation_number=48; citation_pages=3585-3598; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Integrated assessment and prediction of transcription factor binding.; citation_author=A Beyer; citation_author=C Workman; citation_author=J Hollunder; citation_author=D Radke; citation_author=U Möller; citation_journal_title=PLoS Computatioanl Biology; citation_volume=2; citation_number=49; citation_pages=e70; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Computational framework for the prediction of transcription factor binding sites by multiple data integration.; citation_author=A Ambesi-Impiombato; citation_author=M Bansal; citation_author=P Liò; citation_author=D di Bernardo; citation_journal_title=BMC Neuroscience; citation_volume=7; citation_number=50; citation_pages=S8; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Predicting gene expression from sequence.; citation_author=MA Beer; citation_author=S Tavazoie; citation_journal_title=Cell; citation_volume=117; citation_number=51; citation_pages=185-198; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Combining phylogenetic motif discovery and motif clustering to predict co-regulated genes.; citation_author=ST Jensen; citation_author=L Shen; citation_author=JS Liu; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=52; citation_pages=3832-3839; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=TAMO: a flexible, object-oriented framework for analyzing transcriptional regulation using DNA-sequence motifs.; citation_author=DB Gordon; citation_author=L Nekludova; citation_author=S McCallum; citation_author=E Fraenkel; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=53; citation_pages=3164-3165; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Integrating genomic data to predict transcription factor binding.; citation_author=DT Holloway; citation_author=M Kon; citation_author=C DeLisi; citation_journal_title=Genome Informatics; citation_volume=16; citation_number=54; citation_pages=83-94; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Machine learning methods for transcription data integration.; citation_author=DT Holloway; citation_author=M Kon; citation_author=C DeLisi; citation_journal_title=IBM Journal of Research and Development; citation_volume=6; citation_number=55; citation_pages=631-643; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=KP Murphy; citation_author=S Mian; citation_number=56; citation_date=1999; citation_publisher=Technical report, University of California; " />
      <meta name="citation_reference" content="citation_title=Sensitivity and specificity of inferring genetic regulatory interactions from microarray experiments with dynamic Bayesian networks.; citation_author=D Husmeier; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=57; citation_pages=2271-2282; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Inferring cellular networks using probabilistic graphical models.; citation_author=N Friedman; citation_journal_title=Science; citation_volume=303; citation_number=58; citation_pages=799-805; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Modeling gene expression with differential equations.; citation_author=T Chen; citation_author=HL He; citation_author=GM Church; citation_number=59; citation_pages=29-40; citation_date=1999; " />
      <meta name="citation_reference" content="citation_number=60; citation_date=2001; citation_publisher=The MIT Press, 1st edition edition; " />
      <meta name="citation_reference" content="citation_author=DJ Wilkinson; citation_number=61; citation_date=2006; citation_publisher=Chapman & Hall/CRC, 1st edition; " />
      <meta name="citation_reference" content="citation_title=Informative structure priors: joint learning of dynamic regulatory networks from multiple types of data.; citation_author=A Bernard; citation_author=A Hartemink; citation_number=62; citation_pages=459-470; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Clustering of genes into regulons using integrated modeling-COGRIM.; citation_author=G Chen; citation_author=ST Jensen; citation_author=CJ Stoeckert; citation_journal_title=Genome Biology; citation_volume=8; citation_number=63; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Incorporating gene functions as priors in model-based clustering of microarray gene expression data.; citation_author=W Pan; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=64; citation_pages=795-801; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulatory networks in Saccharomyces cerevisiae.; citation_author=TI Lee; citation_author=NJ Rinaldi; citation_author=F Robert; citation_author=DT Odom; citation_author=Z Bar-Joseph; citation_journal_title=Science; citation_volume=298; citation_number=65; citation_pages=799-804; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Use of the ‘Perceptron’ algorithm to distinguish translational initiation sites in E. coli.; citation_author=GD Stormo; citation_author=TD Schneider; citation_author=L Gold; citation_author=A Ehrenfeucht; citation_journal_title=Nucleic Acids Research; citation_volume=10; citation_number=66; citation_pages=2997-3011; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=A higher-order background model improves the detection of promoter regulatory elements by Gibbs sampling.; citation_author=G Thijs; citation_author=M Lescot; citation_author=K Marchal; citation_author=S Rombauts; citation_author=BD Moor; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=67; citation_pages=1113-1122; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=An introduction to ROC analysis.; citation_author=T Fawcett; citation_journal_title=Pattern Recognition Letters; citation_volume=27; citation_number=68; citation_pages=861-874; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Bioprospector: discovering conserved DNA motifs in upstream regulatory regions of co-expressed genes.; citation_author=X Liu; citation_author=DL Brutlag; citation_author=LJ S; citation_number=69; citation_pages=127-138; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=On the Dirichlet prior and Bayesian regularization.; citation_author=H Steck; citation_author=TS Jaakkola; citation_number=70; citation_pages=697-704; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Systematic discovery of regulatory motifs in human promoters and 30 UTRs by comparison of several mammals.; citation_author=X Xie; citation_author=J Lu; citation_author=EJ Kulbokas; citation_author=TR Golub; citation_author=V Mootha; citation_journal_title=Nature; citation_volume=434; citation_number=71; citation_pages=338-345; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.; citation_author=A Siepel; citation_author=G Bejerano; citation_author=JS Pedersen; citation_author=AS Hinrichs; citation_author=M Hou; citation_journal_title=Genome Research; citation_volume=15; citation_number=72; citation_pages=1034-1050; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=ESPERR: Learning strong and weak signals in genomic sequence alignments to identify functional elements.; citation_author=J Taylor; citation_author=S Tyekucheva; citation_author=DC King; citation_author=RC Hardison; citation_author=W Miller; citation_journal_title=Genome Research; citation_volume=16; citation_number=73; citation_pages=1596-1604; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A genomic code for nucleosome positioning.; citation_author=E Segal; citation_author=Y Fondufe-Mittendorf; citation_author=L Chen; citation_author=A Thåström; citation_author=Y Field; citation_journal_title=Nature; citation_volume=442; citation_number=74; citation_pages=772-778; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The gateway to transcription: identifying, characterizing and understanding promoters in the eukaryotic genome.; citation_author=ND Heintzman; citation_author=B Ren; citation_journal_title=Cellular and Molecular Life Sciences; citation_volume=64; citation_number=75; citation_pages=386-400; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Transcription regulation and animal diversity.; citation_author=M Levine; citation_author=R Tjian; citation_journal_title=Nature; citation_volume=424; citation_number=76; citation_pages=147-151; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide location and function of DNA binding proteins.; citation_author=B Ren; citation_author=F Robert; citation_author=JJ Wyrick; citation_author=O Aparicio; citation_author=EG Jennings; citation_journal_title=Science; citation_volume=290; citation_number=77; citation_pages=2306-2309; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identifying gene regulatory elements by genome-wide recovery of DNase hypersensitive sites.; citation_author=GE Crawford; citation_author=IE Holt; citation_author=JC Mullikin; citation_author=D Tai; citation_journal_title=Proceedings of the National Academy of Sciences of the USA; citation_volume=101; citation_number=78; citation_pages=992-997; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=TRANSCompel®: a database on composite regulatory elements in eukaryotic genes.; citation_author=OV Kel-Margoulis; citation_author=AE Kel; citation_author=I Reuter; citation_author=IV Deineko; citation_author=E Wingender; citation_journal_title=Nucleic Acids Research; citation_volume=30; citation_number=79; citation_pages=332-334; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Repeatmasker computer program.; citation_author=AFA Smit; citation_author=R Hubley; citation_author=P Green; citation_number=80; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Comprehensive curation and analysis of global interaction networks in Saccharomyces cerevisiae.; citation_author=T Reguly; citation_author=A Breitkreutz; citation_author=L Boucher; citation_author=BJ Breitkreutz; citation_author=GC Hon; citation_journal_title=Journal of Biology; citation_volume=5; citation_number=81; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=SP1 transcription factors in male germ cell development and differentiation.; citation_author=K Thomas; citation_author=J Wu; citation_author=DY Sung; citation_author=W Thompson; citation_author=M Powell; citation_journal_title=Molecular and Cellular Endocrinology; citation_volume=270; citation_number=82; citation_pages=1-7; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Unbiased mapping of transcription factor binding sites along human chromosomes 21 and 22 points to widespread regulation of noncoding RNAs.; citation_author=S Cawley; citation_author=S Bekiranov; citation_author=HH Ng; citation_author=P Kapranov; citation_author=EA Sekinger; citation_journal_title=Cell; citation_volume=116; citation_number=83; citation_pages=499-509; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome-wide prediction of mammalian enhancers based on analysis of transcription-factor binding affinity.; citation_author=O Hallikas; citation_author=K Palin; citation_author=N Sinjushina; citation_author=R Rautiainen; citation_author=J Partanen; citation_journal_title=Cell; citation_volume=124; citation_number=84; citation_pages=47-59; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome-wide discovery of transcriptional modules from DNA sequence and gene expression.; citation_author=E Segal; citation_author=R Yelensky; citation_author=D Koller; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=85; citation_pages=i273-i282; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Estimating gene networks from gene expression data by combining Bayesian network model with promoter element detection.; citation_author=Y Tamada; citation_author=S Kim; citation_author=H Bannai; citation_author=S Imoto; citation_author=K Tashiro; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=86; citation_pages=ii227-ii236; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Predicting gene regulation by sigma factors in bacillus subtilis from genome-wide data.; citation_author=MJL de Hoon; citation_author=Y Makita; citation_author=S Imoto; citation_author=K Kobayashi; citation_author=N Ogasawara; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=87; citation_pages=i101-i108; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Probabilistic framework for transcription factor binding prediction.; citation_author=H Lähdesmäki; citation_author=I Shmulevich; citation_number=88; citation_date=2007; " />
      <meta name="citation_reference" content="citation_author=CP Robert; citation_author=G Casella; citation_number=89; citation_date=2005; citation_publisher=Springer-Verlag, 2nd edition; " />
      <meta name="citation_reference" content="citation_title=Nonparametric convergence assessment for MCMC model selection.; citation_author=SP Brooks; citation_author=P Giudici; citation_author=A Philippe; citation_journal_title=Journal of Computational and Graphical Statistics; citation_volume=12; citation_number=90; citation_pages=1-22; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Introduction to learning Bayesian networks from data.; citation_author=D Husmeier; citation_number=91; citation_pages=17-57; citation_date=2005; citation_publisher=Springer-Verlag; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001820" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Probabilistic Inference of Transcription Factor Binding from Multiple Data Sources"/>
    <meta name="twitter:description" content="An important problem in molecular biology is to build a complete understanding of transcriptional regulatory processes in the cell. We have developed a flexible, probabilistic framework to predict TF binding from multiple data sources that differs from the standard hypothesis testing (scanning) methods in several ways. Our probabilistic modeling framework estimates the probability of binding and, thus, naturally reflects our degree of belief in binding. Probabilistic modeling also allows for easy and systematic integration of our binding predictions into other probabilistic modeling methods, such as expression-based gene network inference. The method answers the question of whether the whole analyzed promoter has a binding site, but can also be extended to estimate the binding probability at each nucleotide position. Further, we introduce an extension to model combinatorial regulation by several TFs. Most importantly, the proposed methods can make principled probabilistic inference from multiple evidence sources, such as, multiple statistical models (motifs) of the TFs, evolutionary conservation, regulatory potential, CpG islands, nucleosome positioning, DNase hypersensitive sites, ChIP-chip binding segments and other (prior) sequence-based biological knowledge. We developed both a likelihood and a Bayesian method, where the latter is implemented with a Markov chain Monte Carlo algorithm. Results on a carefully constructed test set from the mouse genome demonstrate that principled data fusion can significantly improve the performance of TF binding prediction methods. We also applied the probabilistic modeling framework to all promoters in the mouse genome and the results indicate a sparse connectivity between transcriptional regulators and their target promoters. To facilitate analysis of other sequences and additional data, we have developed an on-line web tool, ProbTF, which implements our probabilistic TF binding prediction method using multiple data sources. Test data set, a web tool, source codes and supplementary data are available at: http://www.probtf.org."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001820.g015"/>

  <meta property="og:title" content="Probabilistic Inference of Transcription Factor Binding from Multiple Data Sources" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001820" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5033'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2044'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2607&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001820">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001820" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001820&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Probabilistic Inference of Transcription Factor Binding from Multiple Data Sources
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Harri Lähdesmäki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute for Systems Biology, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alistair G. Rust, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute for Systems Biology, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ilya Shmulevich
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:ishmulevich@systemsbiology.org">ishmulevich@systemsbiology.org</a></p>

                <p>Affiliation:
                  Institute for Systems Biology, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 26, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001820</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001820" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001820" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001820" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001820" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001820-g001" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g002" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g003" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g004" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g005" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g006" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g007" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g008" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g009" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g010" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g010" title="Figure 10">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g010&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g011" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g011" title="Figure 11">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g011&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g012" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g012" title="Figure 12">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g012&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g013" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g013" title="Figure 13">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g013&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g014" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g014" title="Figure 14">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g014&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001820-g015" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g015" title="Figure 15">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g015&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001820">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001820" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>An important problem in molecular biology is to build a complete understanding of transcriptional regulatory processes in the cell. We have developed a flexible, probabilistic framework to predict TF binding from multiple data sources that differs from the standard hypothesis testing (scanning) methods in several ways. Our probabilistic modeling framework estimates the probability of binding and, thus, naturally reflects our degree of belief in binding. Probabilistic modeling also allows for easy and systematic integration of our binding predictions into other probabilistic modeling methods, such as expression-based gene network inference. The method answers the question of whether the whole analyzed promoter has a binding site, but can also be extended to estimate the binding probability at each nucleotide position. Further, we introduce an extension to model combinatorial regulation by several TFs. Most importantly, the proposed methods can make principled probabilistic inference from multiple evidence sources, such as, multiple statistical models (motifs) of the TFs, evolutionary conservation, regulatory potential, CpG islands, nucleosome positioning, DNase hypersensitive sites, ChIP-chip binding segments and other (prior) sequence-based biological knowledge. We developed both a likelihood and a Bayesian method, where the latter is implemented with a Markov chain Monte Carlo algorithm. Results on a carefully constructed test set from the mouse genome demonstrate that principled data fusion can significantly improve the performance of TF binding prediction methods. We also applied the probabilistic modeling framework to all promoters in the mouse genome and the results indicate a sparse connectivity between transcriptional regulators and their target promoters. To facilitate analysis of other sequences and additional data, we have developed an on-line web tool, ProbTF, which implements our probabilistic TF binding prediction method using multiple data sources. Test data set, a web tool, source codes and supplementary data are available at: <a href="http://www.probtf.org">http://www.probtf.org</a>.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Lähdesmäki H, Rust AG, Shmulevich I (2008) Probabilistic Inference of Transcription Factor Binding from Multiple Data Sources. PLoS ONE 3(3):
          e1820.
            doi:10.1371/journal.pone.0001820</p><p><strong>Editor: </strong>David Jones, University College London, United Kingdom</p><p><strong>Received:</strong> September 17, 2007; <strong>Accepted:</strong> February 4, 2008; <strong>Published:</strong> March 26, 2008</p><p><strong>Copyright:</strong> © 2008 Lähdesmäki et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants R01 GM072855 and P50 GM076547 from NIH/NIGMS.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Transcriptional regulation is a central control mechanism for many biological processes. Transcriptional regulation generally involves DNA-binding proteins, transcription factors (TFs), that control gene expression by binding to short regulatory sequence motifs in gene promoters <a href="#pone.0001820-Davidson1">[1]</a>. DNA-binding specificities of TFs are encoded in their DNA-binding domains that specialize them to recognize and bind specific types of binding sites. This mechanism is the basis of control in complex transcriptional regulatory networks. Revealing these regulatory mechanisms is one of the key problems in understanding genome-wide transcriptional regulation. Although experimental studies and computational approaches are extending our knowledge of TF binding specificities, relatively little is known about genome-wide binding of TFs to gene promoters. Thus, TF binding prediction remains an important problem in computational biology.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Computational approaches to TF binding site analysis can be divided into two categories, <em>discovery</em> and <em>prediction</em>. Motif <em>discovery</em> focuses on searching for novel binding motifs from a collection of short sequences that are assumed to contain a common regulatory motif. Several algorithms have been proposed for motif discovery (for a recent review and comparison, see <a href="#pone.0001820-MacIsaac1">[2]</a>, <a href="#pone.0001820-Tompa1">[3]</a>). Accurate motif discovery is difficult in general, but incorporating additional information to guide the search for novel sequence signals can improve performance. Such additional data sources include, among others, information about co-regulated genes <a href="#pone.0001820-Hughes1">[4]</a>, evolutionary conservation <a href="#pone.0001820-Kellis1">[5]</a>, <a href="#pone.0001820-Siddharthan1">[6]</a>, physical binding locations as measured by chromatin immunoprecipitation on chip (ChIP-chip) <a href="#pone.0001820-Liu1">[7]</a>–<a href="#pone.0001820-Qi1">[9]</a>, information on the structural class of TFs <a href="#pone.0001820-Narlikar1">[10]</a>, and nucleosome occupancies <a href="#pone.0001820-Narlikar2">[11]</a>, <a href="#pone.0001820-Narlikar3">[12]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>TF binding <em>prediction</em>, in turn, makes use of given DNA-binding specificities to predict putative TF binding sites. The binding preferences can either be the output of a motif discovery algorithm or they can be experimentally measured, such as those reported in curated databases (TRANSFAC <a href="#pone.0001820-Wingender1">[13]</a> and JASPAR <a href="#pone.0001820-Sandelin1">[14]</a>). Regardless of the data source, binding site prediction typically requires some information about binding specificities and is therefore dependent on previous analysis. Current knowledge of binding preferences already allows useful predictions to be made genome-wide. Moreover, several novel measurement techniques to measure DNA-binding specificities have recently been developed <a href="#pone.0001820-Mukherjee1">[15]</a>–<a href="#pone.0001820-Zhang1">[19]</a>. For example, Berger <em>et al.</em> <a href="#pone.0001820-Berger1">[16]</a> have developed a protein binding microarray (PBM) technology to measure binding preferences to all <em>k</em>-mers, <em>k</em> currently being 10 base pairs. These new techniques are rapidly expanding currently available databases by providing estimates of binding specificities of virtually any TF in a high-throughput manner. Consequently, they also offer an approach for rapid and sensitive identification of all TF binding sites genome-wide. In particular, high-throughput screening of TF binding specificities combined with accurate TF binding prediction provides a viable, condition independent, alternative to somewhat complex ChIP-chip experiments <a href="#pone.0001820-Eisenstein1">[20]</a>. At the same time, however, there is a growing need for accurate TF binding prediction methods.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Although motif discovery methods are relatively well-developed, the TF binding prediction problem has attracted less attention. Most of the previous binding site prediction tools have been formulated as hypothesis testing methods, where a significance value of TF binding at a specific sequence position is obtained by comparing a test statistic to a null distribution <a href="#pone.0001820-Staden1">[21]</a>–<a href="#pone.0001820-Baji1">[28]</a>, and possibly correcting the significance level for multiple testing. Traditional scanning methods for TF binding site prediction are known to perform relatively poorly in that they typically have an excessively high false positive rate (see <a href="#pone.0001820-Wasserman1">[29]</a>). This reported poor performance is not directly a shortcoming of previous prediction methods but has more to do with the fact that models to represent binding motifs and background sequences alone do not contain sufficient information for accurate binding site detection. This suggests that one possible approach to improve binding site prediction is to develop better motif (and background) models than the currently used position specific frequency model (PSFM) for binding sites and Markovian models for background. For example, observed dependencies between binding site nucleotides <a href="#pone.0001820-Bulyk1">[30]</a>, <a href="#pone.0001820-Berger1">[16]</a> can be incorporated into motif models <a href="#pone.0001820-Zhou1">[31]</a>. However, the use of more complex models, such as general Bayesian networks, is found to be challenging <a href="#pone.0001820-Barash2">[32]</a>. While developing better motif and background models is important, another more general direction aims at making use of several additional information sources, in a similar manner as has been done in the context of motif discovery (see above) to improve TF binding prediction.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Here, we formulate a probabilistic framework for TF binding prediction that differs from the standard hypothesis testing approaches in three important ways. First, the proposed framework is probabilistic in nature and thus outputs a probability of binding (as opposed to a <em>p</em>-value), which directly reflects our belief of gene's promoter having a binding site. We introduce both likelihood and Bayesian inference methods that naturally allow regularization via various prior distributions. Secondly, the proposed method answers the question of whether the whole promoter has a binding site, as opposed to reporting a <em>p</em>-value for every possible position in the sequence. But it is also straightforward to modify our methods to estimate TF binding separately for each base pair position as well, which we also consider. Since we process each promoter as a whole, in addition to assessing physical TF binding at the individual locations, our computational predictions provide insights into the functional role of a TF in the regulatory program of a target gene. The rationale for this is the fact that the higher binding probability anywhere on the promoter (not just in a particular location) implies higher probability of a regulatory relationship. Thirdly, and most importantly, we propose a principled way of combining multiple data sources, such as evolutionary conservation, regulatory potential, CpG islands, nucleosome positioning, DNase hypersensitive sites, ChIP-chip, and other prior knowledge, into a unified probabilistic framework. Moreover, the proposed data fusion framework is extremely versatile and thus, allows incorporating practically any additional (future) information sources that are indicative of TF binding sites at the genome level.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>To validate our computational methods, we constructed a test set of annotated binding sites in mouse promoters from existing databases <a href="#pone.0001820-Wingender1">[13]</a>, <a href="#pone.0001820-Kent1">[33]</a>–<a href="#pone.0001820-Blanco1">[35]</a>. We demonstrate that our probabilistic inference framework significantly improves TF binding predictions. We also test our probabilistic inference method by applying it to all known mouse promoters. These genome-wide results, which are made publicly available, indicate a sparse connectivity between transcriptional regulators and their target promoters. To provide easy access to this method, we have also implemented a web tool, ProbTF, which allows users to analyze their own promoter sequences and additional data sources.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Because our proposed computational framework is based on probabilistic modeling, it provides an intuitive interpretation (i.e., probabilities, not <em>p</em>-values). Our probabilistic formulation also provides regularization to the inference problem via several informative prior distributions, hence further improving performance. Results on a carefully constructed test set show that the proposed computational methods significantly improve performance when compared to previous binding site prediction methods. This is partly due to the fact that each promoter sequence is analyzed as a whole (i.e., TF binding is not assessed independently at each nucleotide position), taking advantage of multiple binding sites. The most important ingredient, however, is the principled incorporation of multiple additional data sources.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>We construct our basic probabilistic framework using commonly used models for binding and non-binding sites, although generalizations to more complex models are straightforward. Consequently, our initial formulation is similar to other previously proposed TF binding prediction methods <a href="#pone.0001820-Frith1">[36]</a>–<a href="#pone.0001820-Sinha1">[38]</a> and has perhaps even more in common with general motif discovery methods <a href="#pone.0001820-Siddharthan1">[6]</a>, <a href="#pone.0001820-Lawrence1">[39]</a>–<a href="#pone.0001820-Lehrach1">[45]</a>. Notably, the differences are that we further extend our basic TF binding prediction method into a Bayesian setting, incorporate multiple motif models, consider combinatorial regulation with multiple TFs, combine both forward and reverse strands into the modeling framework, provide a way to simultaneously estimate binding probabilities to the whole promoter and at single nucleotide resolution and, most importantly, provide a principled statistical integration of multiple data sources.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>Although applications that combine binding site prediction with other data sources, especially “phylogenetic footprinting,” are plentiful (see e.g <a href="#pone.0001820-Sandelin2">[46]</a>, <a href="#pone.0001820-Marinescu1">[47]</a>, or <a href="#pone.0001820-Wasserman1">[29]</a>, <a href="#pone.0001820-GuhaThakurta1">[48]</a> for review), we are not aware of other probabilistic frameworks for TF binding modeling that can combine several additional data sources at the genome level. Related probabilistic data fusion approaches to TF binding prediction are introduced in <a href="#pone.0001820-Beyer1">[49]</a>, <a href="#pone.0001820-AmbesiImpiombato1">[50]</a>. For example, the method of Beyer <em>et al.</em> <a href="#pone.0001820-Beyer1">[49]</a> provides a general, higher-level, naive Bayes approach to integrate ChIP-chip data with several additional evidence sources. Our approach is different in that we integrate multiple data sources at the genome level, which is indeed necessary for incorporating nucleosome information, regulatory potentials, etc. We also note that our probabilistic predictions can be further used in other methods, such as the one of <a href="#pone.0001820-Beyer1">[49]</a>. Other related previous methods include, among others, probabilistic methods for combining sequences and microarray data <a href="#pone.0001820-Beer1">[51]</a>, <a href="#pone.0001820-Jensen1">[52]</a>, general frameworks for data integration (see <a href="#pone.0001820-Gordon1">[53]</a>), and a supervised method for binding site prediction <a href="#pone.0001820-Holloway1">[54]</a>, <a href="#pone.0001820-Holloway2">[55]</a>. The most closely related previous method is that by Thijs <em>et al.</em> <a href="#pone.0001820-Thijs1">[42]</a>, although that was originally introduced in the context of motif discovery and without an option for data fusion. Note that depending on what additional data sources are available, our method can be applied with either zero or any number of additional information sources.</p>
<a id="article1.body1.sec1.p10" name="article1.body1.sec1.p10"></a><p>Although our general aim is to integrate as many lines of evidence as possible into TF binding prediction, we restrict our focus to those data sources that contain useful information for TF binding at the genome level. For example, we do not consider functional data sources, such as gene expression or protein level measurements, or descriptive higher-level information, such as gene ontology. Despite the fact that gene or protein expression measurements alone can be informative of transcriptional regulatory relationships and, therefore, indirectly informative of TF binding as well, we will not include those data sources into our modeling framework. For example, proper modeling of gene expression or protein level measurements does inevitably require the use of a predictive network model, such as a (dynamic) Bayesian network <a href="#pone.0001820-Murphy1">[56]</a>–<a href="#pone.0001820-Friedman1">[58]</a>, a system of ordinary differential equations or a stochastic kinetic model <a href="#pone.0001820-Chen1">[59]</a>–<a href="#pone.0001820-Wilkinson1">[61]</a>. However, even more comprehensive modeling approaches that integrate all sequence level data sources with functional measurements and gene ontology information become much easier to tackle because our TF binding prediction are probabilistic. Probabilistic methods similar to those presented in <a href="#pone.0001820-Bernard1">[62]</a>–<a href="#pone.0001820-Pan1">[64]</a> are practically straightforward to apply using our sequence level predictions as a building block (see ‘Discussion’ Section for more discussion). From a more general point of view, our approach has much in common with general strategies to infer transcriptional regulation from multiple data sources (see <a href="#pone.0001820-Bernard1">[62]</a>–<a href="#pone.0001820-Lee1">[65]</a> for representative examples). We expect that the method described herein will also prove to be a useful building block in comprehensive transcriptional regulatory modeling.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Probabilistic modeling for binding prediction</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To motivate the results presented in this section, we briefly outline our computational methods—first for the basic probabilistic formulation without any additional data sources and then for general data fusion. A detailed description of the methods is presented in ‘<a href="#s4">Materials and Methods</a>’ Section.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The most commonly used probabilistic models for binding sites and background sequences, on which our methods are also built, are the position specific frequency matrix (PSFM) model <a href="#pone.0001820-Stormo1">[66]</a>, <a href="#pone.0001820-Staden1">[21]</a> and the Markovian model <a href="#pone.0001820-Thijs2">[67]</a>, respectively. Our choice of using PSFM model for binding sites is arbitrary. The same modeling framework can be extended to virtually any binding site model. Although we focus on TF binding prediction, our computational formulation is perhaps more closely related to probabilistic motif discovery methods (see e.g <a href="#pone.0001820-Siddharthan1">[6]</a>, <a href="#pone.0001820-Lawrence1">[39]</a>–<a href="#pone.0001820-Lehrach1">[45]</a>). Note, however, that our goal is not <em>de novo</em> motif discovery but probabilistic inference of TF binding, given some <em>a priori</em> information about TF binding specificities and background sequence properties. Because we assume to have prior knowledge of TF specificities we can use a different approach in our computation (compared to motif discovery methods) to better address our goal of estimating TF binding. Further, these standard motif and background models are combined in a probabilistic framework with multiple additional data sources that are indicative of TF binding or transcriptional regulation in general.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Motif and background models are denoted by θ and φ, respectively. For the cases where a TF is associated with more than one motif model, we denote multiple motifs by Θ = (θ<sup>(1)</sup>,…,θ<sup>(<em>m</em>)</sup>). A key (unknown) quantity is the number of binding sites <em>Q</em> in a promoter sequence <em>S</em> = (<em>s</em><sub>1</sub>,…,<em>s<sub>N</sub></em>). Instead of fixing <em>Q</em> = <em>c</em> binding sites to particular positions, we consider (sum over) all possible non-overlapping motif start positions <em>A</em> = {<em>a</em><sub>1</sub>,…,<em>a<sub>c</sub></em>} and configurations πЄ{1,…,<em>m</em>}<em><sup>c</sup></em> and weight different combinations according to their probability. This is illustrated in <a href="#pone-0001820-g001">Figure 1</a>, which shows four different combinations of the number of binding sites <em>Q</em>, positions <em>A</em> and configurations π for a TF that is associated with two motif models (blue and green boxes).</p>
<div class="figure" id="pone-0001820-g001"><div class="img"><a name="pone-0001820-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>An illustration of four different binding site configurations for a TF that is associated with two motif models (blue and green boxes).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The diagram illustrates the upstream promoter region for a gene, where the direction of transcription is indicated by the direction of the arrows. The arrows are located at the transcription start sites.</p>
<span>doi:10.1371/journal.pone.0001820.g001</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Given <em>S</em>, Θ and φ, we are interested in computing the probability of having <em>c</em> = 0,1,… binding sites<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e001&amp;representation=PNG" class="inline-graphic"></span><br>where the term <em>P</em>(<em>S</em>|<em>Q</em> = <em>c</em>,Θ,φ) involves summing over all motif positions <em>A</em> and configurations π for <em>c</em> binding sites. The prior <em>P</em>(<em>Q</em> = <em>c</em>|Θ,φ) reflects our prior belief of having <em>c</em> binding sites. The probability of binding can be assessed by the probability of having at least one (or any other higher number, if so decided) binding site <em>P</em>(<em>Q</em>&gt;0|<em>S</em>,Θ,φ). A computationally efficient recursive algorithm to compute <em>P</em>(<em>Q</em>&gt;0|<em>S</em>,Θ,φ) is described in ‘<a href="#s4">Materials and Methods</a>’ Section. Our basic formulation can be viewed as a direct generalization of that by Thijs <em>et al.</em> <a href="#pone.0001820-Thijs1">[42]</a> who proposed a practically equivalent framework for a single motif model. Similar probabilistic modeling frameworks have also been constructed using hidden Markov models <a href="#pone.0001820-Frith1">[36]</a>–<a href="#pone.0001820-Sinha1">[38]</a>, <a href="#pone.0001820-Sinha2">[44]</a>.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>Because motif models are typically constructed from a relatively small number of experimentally verified binding sites (or from an output of a motif discovery algorithm) they can contain a considerable amount of uncertainty. Thus, it is also useful to consider a Bayesian approach where Θ and φ are random variables. Given <em>S</em>, <em>A</em> and π and applying Bayes' rule gives<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e002&amp;representation=PNG" class="inline-graphic"></span><br>where <em>P</em>(<em>S</em>|<em>A</em>,π) is the marginal likelihood obtained by integrating over parameters Θ and φ. The probability of binding can again be assessed by the probability of having at least one binding site, i.e., 1−<em>P</em>(<em>A</em> = Ø,π = Ø|<em>S</em>). For the Bayesian approach we develop a Markov chain Monte Carlo (MCMC) estimation method (see ‘MCMC estimation for Bayesian inference’ Section for details). Another important benefit of using a Bayesian approach is that with the proposed MCMC sampling strategy one can solve more complex inference problems (and with more complex prior distributions) than with the efficient recursive algorithm for the likelihood method. In particular, MCMC sampling also makes it possible to jointly estimate binding probabilities to the whole promoter and individual base pair locations.</p>


<h4>Performance on real sequences</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We demonstrate our computational methods on a carefully constructed test set that contains annotated binding sites in 47 mouse promoters. For the positive cases we use those TF-promoter pairs that contain an annotated binding site for a TF in a promoter. Negative cases consist of TF-promoter pairs that (are expected to) have no functional binding sites. Although negative sets in general are likely to contain some true binding sites (those that are not yet discovered), that does not invalidate our performance evaluation but merely introduces a degrading bias in our results. Similarly, some of the annotated binding sites may have been used to construct the PSFM models that we use in our analysis and that in turn can introduce an optimistic bias into our results. The above-mentioned biases are impossible to avoid in practice. Fortunately, this is not a major issue, especially as long as we use the same test set and the same PSFM and Markovian background models for all the new and previously published methods that we compare. The full details of the test set can be found in ‘Data’ Section. For the binding specificities, we use (scaled) motif models from TRANSFAC Professional version 10.3. We measure performance using standard receiver operating characteristics (ROC) curves that plot the fraction of true positives (sensitivity) versus the fraction of false positives (complementary specificity), see e.g <a href="#pone.0001820-Fawcett1">[68]</a>. We also use the area under the curve (AUC) measure that summarizes and represents the ROC with a single number. Given the probabilistic nature of our method, we also found it instructive to visualize the distribution of estimated binding probabilities for positive and negative test cases.</p>


<h4>Comparison of background model orders</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Before proceeding to more interesting results, we first test the effect of some of the parameters in our probabilistic formulation. A natural parameter to start with is the order of the Markovian background model. Several authors have reported that the use of higher-order background models improves motif discovery <a href="#pone.0001820-Thijs2">[67]</a>, <a href="#pone.0001820-Liu2">[69]</a>. <a href="#pone-0001820-g002">Figure 2</a> shows ROC curves for our test set using the basic likelihood-based probabilistic method with varying Markovian background model orders, <em>d</em>Є{0,1,…,4}. <a href="#pone-0001820-g002">Figure 2</a> shows somewhat surprisingly that, overall, TF binding prediction does not seem to depend greatly on the background model order. The best overall performance, according to AUC, is achieved with <em>d</em> = 0. For consistency, we use <em>d</em> = 0 in all our simulations.</p>
<div class="figure" id="pone-0001820-g002"><div class="img"><a name="pone-0001820-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The standard ROC curves for the basic likelihood-based method with varying Markovian background model orders, <em>d</em>Є{0,1,…,4}.</span></strong></p><span>doi:10.1371/journal.pone.0001820.g002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Another useful preliminary test is to vary the prior probability of having <em>Q</em> = 0 binding sites as well as the parameter κ, which specifies how fast the prior probability of having <em>Q</em>&gt;0 binding sites approaches zero (see ‘<a href="#s4">Materials and Methods</a>’ Section for more details). In general, these parameters only affect the overall bias of the posterior binding probabilities: larger values of <em>P</em>(<em>Q</em> = 0|Θ,φ) and smaller values of κ bias probabilities towards small values, and vice versa (results not shown).</p>


<h4>Comparison of likelihood and Bayesian approaches</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Adopting a Bayesian approach allows the modeling of uncertainty in the model parameters as well. Although we use the Dirichlet prior distribution for both the motif and background model parameters, we are primarily concerned with the uncertainty in the motif models, since the background model parameters are estimated from a much larger data set. Hyperparameters of the Dirichlet prior control the amount of uncertainty in the motif model parameters and that can also provide regularization for the inference problem (see <a href="#pone.0001820-Steck1">[70]</a> for a discussion in the context of Bayesian networks). For consistency, we use the same hyperparameters to obtain the scaled PSFMs for the likelihood-based approach. <a href="#pone-0001820-g003">Figure 3</a> shows the ROC results for the likelihood and Bayesian methods with varying prior strengths (see ‘<a href="#s4">Materials and Methods</a>’ Section for more details). In the basic simulations setting without any additional data sources, the likelihood and Bayesian methods perform almost identically.</p>
<div class="figure" id="pone-0001820-g003"><div class="img"><a name="pone-0001820-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>ROC curves for the likelihood and Bayesian probabilistic methods with varying prior strengths.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(a) <em>M</em> = 50. (b) <em>M</em> = 100. Results for (c) likelihood-based and (d) Bayesian methods for various values of <em>M</em>.</p>
<span>doi:10.1371/journal.pone.0001820.g003</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p><a href="#pone-0001820-g004">Figure 4</a> shows histograms of the estimated binding probabilities for the likelihood and Bayesian methods for prior strengths <em>M</em> = 50 and <em>M</em> = 100. A general character of the histograms is that the binding probabilities of annotated sites (red bars) are biased towards high values whereas the binding probabilities of negative cases (blue bars) are approximately uniformly distributed. Smaller prior strengths correspond to more uncertain motif models and that, in turn, allows the data (i.e., promoter sequence) to have a stronger effect on the posterior motif model. Typically, smaller prior strengths also bias overall posterior probabilities towards higher values, especially for the negative set (compare blue bars in <a href="#pone-0001820-g004">Figures 4 (a–b)</a> with the ones in (c–d)). In terms of ROC curves and AUC measures, proposed methods are insensitive to small deviations in prior strengths (within a range of reasonable values). Prior strength <em>M</em> = 100 seems to provide good results and this value was used in all our simulations for consistency.</p>
<div class="figure" id="pone-0001820-g004"><div class="img"><a name="pone-0001820-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Histograms of the estimated binding probabilities for the likelihood and Bayesian methods with varying prior strengths.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>(a) Likelihood <em>M</em> = 50. (b) Bayesian <em>M</em> = 50. (c) Likelihood <em>M</em> = 100. (d) Bayesian <em>M</em> = 100. <em>x</em>-axes correspond to the estimated binding probability and <em>y</em>-axes show the fraction of negative (blue) and positive (red) test cases. Histogram bin edges are located at <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e003&amp;representation=PNG" class="inline-graphic"></span>, <em>i</em> = 0,1,…,10, although the two histograms are shown side by side.</p>
<span>doi:10.1371/journal.pone.0001820.g004</span></div>

<h4>Comparison with traditional promoter scanning</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>In order to better assess the performance of the proposed methods, we also compare our basic likelihood method with traditional promoter scanning (see <a href="#pone.0001820-Staden1">[21]</a>–<a href="#pone.0001820-Wu1">[24]</a>) and a probabilistic scanning-based method that assesses the probability of binding <a href="#pone.0001820-Chen2">[63]</a> (see ‘Comparison with other methods’ Section for more details). Due to a fundamental problem of hypothesis testing approaches, they are prone to systematically finding more significant binding sites in longer promoter sequences just by chance. This inherent bias could be accounted for to an extent by correcting for the multiple testing issue. That would be remarkably challenging because all significance values corresponding to a PSFM model should be corrected simultaneously and that would easily result in millions of <em>p</em>-values per PSFM model. An even more severe issue, however, stems from the fact that exact promoter regions are unknown practically for all the genes. Thus, a particular choice of the promoter regions, especially their lengths, has a potential to impose a significant biasing effect. The easiest and safest way to guarantee a fair and unbiased comparison between different methods is to shorten all the promoter sequences in our controlled test set to have approximately the same length (but we also compare different methods without changing the promoter lengths for comparison purposes). <a href="#pone-0001820-g005">Figure 5</a> shows ROC results for the three methods with two different background model orders. Overall, the performance of all three methods is similar, which is to be expected since they are based on the same motif and background models. However, our probabilistic method performs better for small values of false positives (about &lt;0.2). This is partly due to the fact that our method makes inference using multiple binding sites together and hence, it is able to assign higher probability to sequences that have multiple annotated binding sites. Performance differences become more significant when we integrate multiple data sources into TF binding predictions in the next section. In traditional hypothesis testing based approaches, one has typically been interested in a region of the ROC curves that corresponds to a remarkably small false positive rate, say &lt;0.01. This is mainly due to the fact that each PSFM and nucleotide location pair is assigned its own significance value and therefore even a tiny false positive rate makes the results look like each factor binds to practically every promoter. Fortunately, this issue becomes less severe in our modeling framework since each promoter is processed as a whole (i.e., a single binding probability per TF-promoter pair), although later on we also estimate binding probabilities at a single nucleotide resolution. Further, and perhaps more importantly, one of our main goals is full probabilistic modeling, where different evidence sources are combined in a seemingly continuous fashion (i.e., without making calls of ‘binding’ or ‘not binding’ at each step of the inference process). For example, practically all parts of the ROC curve become equally important when our sequence-based probabilistic modeling is combined with gene expression data and other information sources. Supplemental <a href="#pone.0001820.s002">Figure S1</a> shows the same comparison results as in <a href="#pone-0001820-g005">Figure 5</a>, but without forcing the promoter lengths to be equal. These results show that when promoters have varying lengths, our probabilistic method that processes each promoter as a whole performs even better when compared to scanning-based approaches.</p>
<div class="figure" id="pone-0001820-g005"><div class="img"><a name="pone-0001820-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>ROC curves for the likelihood-based probabilistic method (red), traditional scanning (blue), and a probabilistic scanning-based method that outputs a probability of binding (green).</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>The background model order is (a) <em>d</em> = 0 and (b) <em>d</em> = 1.</p>
<span>doi:10.1371/journal.pone.0001820.g005</span></div>

<h4>Probabilistic integration of multiple data sources</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Many TF binding sequences are relatively short and non-unique and hence, the expected number of their occurrences in a genome by chance is high. These presumably non-functional binding sites cause traditional TF binding site prediction methods to have unacceptably high false positive rates. Although the above probabilistic formulation provides a principled framework for TF binding inference and allows regularization via a Bayesian approach, being built on the same modeling framework as other methods (PSFM motif and Markovian background models), it is also vulnerable to identifying non-functional sites.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>A natural way to improve specificity of TF binding site predictions is to make use of additional biological information in the inference. This idea has been proposed in several articles where, for example, promoter scanning with PSFMs has been constrained to only those parts of the genome that are highly conserved, i.e., conservation scores exceed a threshold (see reviews in <a href="#pone.0001820-Wasserman1">[29]</a>, <a href="#pone.0001820-GuhaThakurta1">[48]</a>). However, not all binding sites are conserved <a href="#pone.0001820-Wasserman1">[29]</a>. Therefore, using additional data in such a binary fashion, is likely to miss some binding sites and does not provide the most efficient use of the data. A key advantage of our formulation is that it provides a principled, probabilistic framework for incorporating multiple data sources.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>A number of additional information sources can be useful for predicting TF binding. First, functional binding sites are typically evolutionarily conserved <a href="#pone.0001820-Xie1">[71]</a>, albeit with the caveat described in the previous paragraph. Alignments of interspecies genomes combined with other modeling efforts can be used to assess the probability that a certain genomic location is under evolutionary selection, and thus more likely to be a functional binding site. We use PhastCons <a href="#pone.0001820-Siepel1">[72]</a> to assess the probability of conservation. PhastCons uses genome alignments of 17 species and a continuous-time Markov model for nucleotide substitutions and a two state phylo-HMM model to compute posterior conservation probabilities.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>Second, in addition to evolutionary conservation, methods exist to assess whether a conserved sequence is neutral or functional. This more detailed information, often called regulatory potential, has a potential to distinguish neutral sequence regions from the functional ones, even within conserved parts of sequences. Regulatory potential scores (log-likelihoods) are obtained using ESPERR <a href="#pone.0001820-Taylor1">[73]</a> that also makes use of multiple genome alignments. After appropriate dimension reduction and alphabet selection, ESPERR applies two variable order Markov models to estimate likelihoods of regulatory and neutral sites. The two Markov models are trained from a set of known regulatory and neutral sites which makes ESPERR essentially a discriminatory method.</p>
<a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p>Third, while evolutionary conservation can help in discriminating functional binding sites that are more prevalently located on conserved parts of the genome from presumably non-functional sites on non-conserved regions, it does not explain the mechanism by which a TF is guided to its functional site. A hypothesis is that this process is controlled by the intrinsic nucleosome organization of genomes <a href="#pone.0001820-Segal1">[74]</a>, <a href="#pone.0001820-Heintzman1">[75]</a>. The likelihood of binding to a non-functional binding site can be decreased by locating a stable nucleosome over those genomic regions while keeping functional sites accessible for TFs, i.e., free of (stable) nucleosomes. The nucleosome occupancy probabilities can be computed using a method by Segal <em>et al.</em> <a href="#pone.0001820-Segal1">[74]</a>, which uses a Markov model whose parameters are estimated from a set of known nucleosome locations.</p>
<a id="article1.body1.sec2.sec6.p6" name="article1.body1.sec2.sec6.p6"></a><p>Although we primarily focus on the three additional evidence sources mentioned above, other information sources can also be directly included into our modeling framework. For example, a general sequence feature of many promoters, and thereby a feature of binding sites within promoters as well, is that they typically have a high CpG dinucleotide content <a href="#pone.0001820-Heintzman1">[75]</a>. Binding sites are also commonly found to be organized into clusters (for a review, see <a href="#pone.0001820-Levine1">[76]</a>). Furthermore, other more important, direct evidence sources include experimental measures of TF binding as measured by chromatin immunoprecipitation on chip <a href="#pone.0001820-Ren1">[77]</a>, <a href="#pone.0001820-Qi1">[9]</a> and DNase hypersensitive sites <a href="#pone.0001820-Crawford1">[78]</a>.</p>
<a id="article1.body1.sec2.sec6.p7" name="article1.body1.sec2.sec6.p7"></a><p>From a computational point of view, we assume that each additional data source is in the form <em>D</em> = (<em>P</em>(1),…,<em>P</em>(<em>N</em>)), where <em>P</em>(<em>i</em>) denotes the probability that the <em>i</em>th nucleotide has one of the above sources of evidence. Although not all data sources are in the form of probabilities they can often be interpreted or transformed such that they conform to this format. Additional data <em>D</em> is assumed to be conditionally independent of the sequence <em>S</em> (given <em>A</em>, π, Θ and φ) and independent of the motif and background models, i.e.,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br></p>
<a id="article1.body1.sec2.sec6.p8" name="article1.body1.sec2.sec6.p8"></a><p>The intuitive rationale for defining the term that captures the additional data, <em>P</em>(<em>D</em>|<em>A</em>,π), is to assign higher probabilities for those configurations (<em>A</em>,π) that are located in regions that are more likely (in light of additional data <em>D</em>) to contain functional binding sites. A similar data fusion technique works in a Bayesian framework as well<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br></p>
<a id="article1.body1.sec2.sec6.p9" name="article1.body1.sec2.sec6.p9"></a><p>We extend the above framework to multiple data sources <em>D<sub>i</sub></em>, 1≤<em>i</em>≤<em>N<sub>D</sub></em>, by combining them using a standard weighting scheme prior to applying Equations (1–2). The details of the computational methods for incorporating multiple data sources are described in ‘Combining multiple information sources’ Section.</p>


<h4>Performance on real multiple data sources</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>The data fusion problem is illustrated in <a href="#pone-0001820-g006">Figure 6</a> for four TF-promoter pairs. The first row in each subplot shows the annotated binding site(s). For illustration purposes, the next rows show the log-likelihood scores of motif model(s) θ<sup>(<em>i</em>)</sup> to the background model φ, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e006&amp;representation=PNG" class="inline-graphic"></span> (see Equation (11) for details). The last three rows show the probabilities of conservation, nucleosome occupancy and the regulatory potential. <a href="#pone-0001820-g006">Figure 6</a> illustrates some general characteristics of the data. First, the highest log-likelihood score <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e007&amp;representation=PNG" class="inline-graphic"></span> is not always obtained at the annotated site but on other, possibly non-functional, positions. Second, for some TFs, such as SRF, motif models θ<sup>(<em>i</em>)</sup> are highly correlated whereas for other TFs, such as SP1, motif models produce “scores” which are distinct from each other. Finally, many of the annotated sites are also associated with a high probability of conservation and regulatory potential and with a low probability of nucleosome occupancy. This correlation is not expected to be perfect though since only about 50% of the functional binding sites are assessed to be conserved (see <a href="#pone.0001820-Wasserman1">[29]</a>). Our goal here is to make principled probabilistic inference from these numbers and to output a single probability of binding for the promoter as a whole (i.e., transcriptional regulation). We will also compute the probability of having a binding site at each nucleotide position later on.</p>
<div class="figure" id="pone-0001820-g006"><div class="img"><a name="pone-0001820-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>An illustration of the data fusion for TF binding prediction.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>(a) Annotated binding sites for SRF on Actc1 promoter. (b) Annotated binding site for SRF on M23768 promoter. (c) Annotated binding site for SP1 on Myod1 promoter. (d) Annotated binding site for TEAD1 on Myh6 promoter. Figure keys are as follows. θ<sup>(<em>i</em>)</sup>: motif models for each TF, Conserv.: sequence conservation probabilities computed by PhastCons <a href="#pone.0001820-Siepel1">[72]</a>, Nuc. pos.: nucleosome occupancy probabilities estimated by a yeast nucleosome model from <a href="#pone.0001820-Segal1">[74]</a>, and Reg. pot.: regulatory potential log-likelihood scores from <a href="#pone.0001820-Taylor1">[73]</a>. The additional evidences range between 0 and 1. Promoters sequence lengths are 2000 base pairs in (a), (c) and (d), and 500 base pairs in (b). See text for more details.</p>
<span>doi:10.1371/journal.pone.0001820.g006</span></div>

<h4>Comparisons between different additional data sources</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p><a href="#pone-0001820-g007">Figure 7 (a)</a> shows ROC curves for the likelihood-based method (blue) when combined with a single additional information source. The use of regulatory potential scores (red) already gives a marginal improvement for the TF binding prediction, but evolutionary conservation (green) significantly improves the detection performance. Although nucleosome occupancy probabilities seem to be low at annotated sites in <a href="#pone-0001820-g006">Figure 6</a>, static predictions of nucleosome positions are not sufficiently informative to help binding prediction. Perhaps the main difficulty in using the predicted nucleosome locations is that the nucleosome model is constructed for yeast (only models for yeast, chicken and human are available in <a href="#pone.0001820-Segal1">[74]</a>). However, as Narlikar <em>et al.</em> <a href="#pone.0001820-Narlikar2">[11]</a> have demonstrated in the case of motif discovery in yeast, nucleosome information can be informative, especially if combined with ChIP-chip data in a discriminatory setting. We also found out that information of CpG-islands does not improve binding predictions.</p>
<div class="figure" id="pone-0001820-g007"><div class="img"><a name="pone-0001820-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>(a) ROC curves for the likelihood-based method (blue) when combined with a single additional information source: regulatory potential (red), and evolutionary conservation (green).</span></strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>Histograms of the estimated binding probabilities for the likelihood-based method when combined with (b) regulatory potential and (c) evolutionary conservation.</p>
<span>doi:10.1371/journal.pone.0001820.g007</span></div><a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p><a href="#pone-0001820-g007">Figures 7 (b) and (c)</a> show the corresponding histograms of the estimated binding probabilities. Combining the probabilistic method with evolutionary conservation produces the most discriminatory histogram (<a href="#pone-0001820-g007">Figure 7 (c)</a>) whereas regulatory potential data (<a href="#pone-0001820-g007">Figure 7 (b)</a>) assign higher probabilities to some of the negative case. This can also indicate that the negative sequences may indeed contain unannotated binding sites.</p>
<a id="article1.body1.sec2.sec8.p3" name="article1.body1.sec2.sec8.p3"></a><p>In these simulations, the scaling parameter for each additional data source (see ‘Combining multiple information sources’ Section for details) is chosen using a grid search over values <em>d</em>Є{0, 0.01,…, 0.5} and taking the one that produces the best AUC measure. We found <em>d</em> = 0.04 for conservation and <em>d</em> = 0.05 for the regulatory potential data, but the results are not sensitive to small deviations in the values of the scaling parameters. To verify that choosing scaling parameters by maximizing the AUC measure does not introduce an optimistic bias, we repeated the same simulation using stratified cross-validation. For conservation scores, the results remained virtually the same and for the regulatory potential the results are also similar (see supplemental <a href="#pone.0001820.s003">Figure S2</a>). ESPERR <a href="#pone.0001820-Taylor1">[73]</a> is trained on a set of human genes, which is further expanded by mapping the set of human genes to orthologous mouse genes, amongst others. This extended data set partly overlaps with our test set. We verified that this overlap does not introduce any bias in our results by removing the overlapping genes from our test set and repeating the simulation (see supplemental material <a href="#pone.0001820.s001">Text S1</a> and <a href="#pone.0001820.s005">Figure S4</a>).</p>
<a id="article1.body1.sec2.sec8.p4" name="article1.body1.sec2.sec8.p4"></a><p>Given the above promising results with a single additional data source, a natural question then is to study whether combinations of additional data sources further improve TF binding prediction. We consider the combination of conservation and regulatory potential for which the ROC curve as well as the corresponding histogram of the estimated binding probabilities are shown in <a href="#pone-0001820-g008">Figure 8</a>. Combining conservation and regulatory potential gives a minor improvement relative to using these data sources alone.</p>
<div class="figure" id="pone-0001820-g008"><div class="img"><a name="pone-0001820-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>(a) ROC curve for the likelihood-based method (blue) when combined with evolutionary conservation (green), regulatory potential (cyan), and a combination of evolutionary conservation and regulatory potential (red).</span></strong></p><a id="article1.body1.sec2.sec8.fig2.caption1.p1" name="article1.body1.sec2.sec8.fig2.caption1.p1"></a><p>(b) Histogram of the estimated binding probabilities for a combination of conservation and regulatory potential.</p>
<span>doi:10.1371/journal.pone.0001820.g008</span></div><a id="article1.body1.sec2.sec8.p5" name="article1.body1.sec2.sec8.p5"></a><p>We used the same scaling parameters as above and tried a set of different weighting schemes and again chose the weighting parameter that gives the best AUC measure over grid <em>w</em><sub>2</sub>Є{0.5, 0.52,…, 1}. We found <em>w</em><sub>1</sub> = 0.14 and <em>w</em><sub>2</sub> = 1−<em>w</em><sub>1</sub> = 0.86 for regulatory potential and conservation, respectively (see ‘Combining multiple information sources’ Section for details). Similarly to scaling, weighting is not sensitive to small deviations in the values of the weighting parameters. Weighting some of the data sources more heavily just biases the results towards those obtained using the particular single data source alone.</p>
<a id="article1.body1.sec2.sec8.p6" name="article1.body1.sec2.sec8.p6"></a><p>The standard practice has been to constrain the scanning with PSFMs to only those regions of the genome whose conservation probability (or score) is sufficiently high <a href="#pone.0001820-Wasserman1">[29]</a>. For comparison purposes, we applied the same strategy to our test set using a similar grid search for the optimal threshold as above. <a href="#pone-0001820-g009">Figure 9</a> shows ROC curves for the traditional scanning with and without conservation data as well as our probabilistic method when combined with conservation information. These results demonstrate the fact that thresholding-based methods cannot achieve optimal performance since not all functional binding sites are conserved. This issue becomes more prevalent when more than one data source is integrated. Supplemental <a href="#pone.0001820.s004">Figure S3</a> shows the same results as in <a href="#pone-0001820-g009">Figure 9</a> but without forcing the promoter sequence lengths to be equal, in which case, our probabilistic method performs again even better than the traditional scanning. These results again demonstrate the potential bias of hypothesis testing based approaches.</p>
<div class="figure" id="pone-0001820-g009"><div class="img"><a name="pone-0001820-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>ROC curves for the traditional scanning (green), traditional scanning combined with thresholded conservation information (blue), probabilistic method combined with conservation information (red), and probabilistic method (cyan).</span></strong></p><span>doi:10.1371/journal.pone.0001820.g009</span></div><a id="article1.body1.sec2.sec8.p7" name="article1.body1.sec2.sec8.p7"></a><p>Evolutionary conservation and regulatory potential are the most informative additional data source in our simulations, whereas estimated nucleosome locations or CpG-islands do not improve TF binding predictions. As mentioned above, the particular estimated nucleosome data that we use might not be optimal for our mouse test set. Once a mouse nucleosome model or high-throughput nucleosome data become available, they can be used in the same framework as well, likely improving TF binding predictions. For example, Narlikar <em>et al.</em> <a href="#pone.0001820-Narlikar3">[12]</a> have found that even low resolution measurements of nucleosome locations give a marginal improvement for motif discovery methods in yeast. Similarly, once more abundant ChIP-chip data becomes available, it can be incorporated into our modeling framework as well. This data fusion step can be performed in two ways: either use an approach where binding probabilities are estimated separately from ChIP-chip and sequence data and then combined (see <a href="#pone.0001820-Beyer1">[49]</a>), or incorporate high-resolution ChIP-chip binding data (see <a href="#pone.0001820-Qi1">[9]</a>) directly into our model.</p>


<h4>Modeling combinatorial regulation</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>Gene regulation in higher organisms commonly requires multiple TFs. Thus, combinatorial regulation by several TFs is another important problem to study. The main difference between a single TF and multiple TFs regulating a gene is that combinatorial regulation requires all TFs to have at least one binding site for (at least) one of their motif models. Although multiple regulatory proteins can also form a complex and the complex can regulate a target gene via a single binding site, we only consider regulation via multiple binding sites, the single binding site case being similar with our previous analysis. Statistical inference for combinatorial regulation can be naturally addressed in our probabilistic framework. For that purpose, we propose to use both the likelihood and Bayesian methods (see ‘Combinatorial regulation’ Section for more computational details).</p>
<a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>Combinatorial regulation by multiple TFs is less well-known and fewer combinatorial annotated binding sites are reported in databases or even in the literature (see <a href="#pone.0001820-KelMargoulis1">[79]</a>). We construct a sufficiently large test set for computational simulations from our test set of annotated binding sites. For each promoter sequence, we pair different TFs that have annotated binding sites on it and consider these TF pairs to jointly regulate a given gene. We add a further constraint that, for each pair of TFs, at least one annotated binding site pair is within a cluster, i.e., sufficiently close to each other, as is typically the case in real promoters as well <a href="#pone.0001820-Levine1">[76]</a>. We construct a negative set such that approximately half of the TF pairs have no annotated binding sites and half of the TF pairs have an annotated binding site for one TF but not for the other.</p>
<a id="article1.body1.sec2.sec9.p3" name="article1.body1.sec2.sec9.p3"></a><p>The problem of inferring combinatorial regulation among many TFs becomes computationally expensive because there are <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e008&amp;representation=PNG" class="inline-graphic"></span> ways to choose <em>k</em> TFs from the set of <em>n</em> TFs. Inference for combinatorial regulation can only be done exactly using an MCMC sampler. This can become a problem since Bayesian inference is considerably slower than the likelihood-based inference. Therefore, in addition to Bayesian inference for combinatorial regulation, we also consider here a naive approximation that estimates the probability of combinatorial regulation by the product of individual TF binding probabilities. <a href="#pone-0001820-g010">Figure 10</a> shows ROC results and the corresponding histograms for the two different methods. Surprisingly, the approximative method performs slightly better than the Bayesian method. <a href="#pone-0001820-g010">Figures 10 (b) and (c)</a> seem to suggest that the Bayesian method assigns higher probabilities to some of the negative cases and, thereby, results in slightly worse ROC curve than the likelihood method. Note that, given the more difficult problem of finding multiple weak sequence signals, histograms of combinatorial regulation probabilities for the positive set (red bars) now span the whole range from 0 to 1.</p>
<div class="figure" id="pone-0001820-g010"><div class="img"><a name="pone-0001820-g010" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g010&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g010"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g010&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g010/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g010/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g010/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g010/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g010.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g010/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g010/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g010.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 10.  <span>(a) ROC curves for combinatorial regulation using the Bayesian method (blue) and a naive likelihood approximation (green).</span></strong></p><a id="article1.body1.sec2.sec9.fig1.caption1.p1" name="article1.body1.sec2.sec9.fig1.caption1.p1"></a><p>Histogram of combinatorial regulation probabilities for (b) the Bayesian method and (c) naive likelihood approximation.</p>
<span>doi:10.1371/journal.pone.0001820.g010</span></div><a id="article1.body1.sec2.sec9.p4" name="article1.body1.sec2.sec9.p4"></a><p>As suggested by previous simulations, the detection of combinatorial regulation can be improved by incorporating additional data sources. We consider using evolutionary conservation for which the results are shown in <a href="#pone-0001820-g011">Figure 11</a>. Comparison of <a href="#pone-0001820-g010">Figures 10</a> and <a href="#pone-0001820-g011">11</a> shows that additional data improves performance significantly. As in the case of no additional information, the naive likelihood approximation performs better than the Bayesian alternative. These results suggest that our proposed methods are well suited for inferring combinatorial regulation as well. Further, the naive likelihood approximation provides a computationally efficient alternative.</p>
<div class="figure" id="pone-0001820-g011"><div class="img"><a name="pone-0001820-g011" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g011&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g011"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g011&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g011/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g011/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g011/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g011/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g011.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g011/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g011/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g011.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 11.  <span>(a) ROC curves for combinatorial regulation using the Bayesian method with evolutionary conservation (blue) and a naive likelihood approximation with evolutionary conservation (green).</span></strong></p><a id="article1.body1.sec2.sec9.fig2.caption1.p1" name="article1.body1.sec2.sec9.fig2.caption1.p1"></a><p>Histogram of combinatorial regulation probabilities for (b) the Bayesian method with evolutionary conservation and (c) a naive likelihood approximation with evolutionary conservation.</p>
<span>doi:10.1371/journal.pone.0001820.g011</span></div>

<h4>Extensions to double-stranded DNA and single nucleotide resolution  Single vs. both strands</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>So far we have assumed that the direction of transcription is known and we have focused on analyzing only a single strand of the DNA. This is not always the case and, therefore, it is useful to generalize TF binding prediction methods such that they use both strands of the DNA. Our probabilistic methods generalize naturally to handle double-stranded DNA. This can be achieved simply by applying the aforementioned methods to both strands, either independently or simultaneously (computational details are described in ‘Single vs. both strands’ Section). To demonstrate performance of our methods on double-stranded DNA, we re-compute the results shown in <a href="#pone-0001820-g007">Figure 7</a>. Here we use a variant of the likelihood method that processes different strands independently. <a href="#pone-0001820-g012">Figure 12</a> shows the resulting ROC curves. Results in <a href="#pone-0001820-g007">Figures 7</a> and <a href="#pone-0001820-g012">12</a> are virtually identical which suggests that our proposed methods perform equally well on both single and double-stranded DNA.</p>
<div class="figure" id="pone-0001820-g012"><div class="img"><a name="pone-0001820-g012" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g012&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g012"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g012&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g012/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g012/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g012/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g012/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g012.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g012/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g012/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g012.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 12.  <span>ROC curves for the likelihood-based method (blue) when both strands of the DNA are used and a single additional information sources is available: regulatory potential (red) and evolutionary conservation (green).</span></strong></p><span>doi:10.1371/journal.pone.0001820.g012</span></div>

<h4>Predicting binding site positions</h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>Our final simulation concerns inferring the probability of having a binding site at a single nucleotide position. Although our proposed methods are primarily designed to process each promoter sequence as a whole, it is also useful to be able to infer binding probabilities at a higher resolution, in particular, at each nucleotide position. Inferring the binding probabilities at each base pair location is more challenging from the computational point of view. In particular, the efficient recursive algorithm developed for the likelihood method cannot be applied. The inference can, however, be easily performed in our Bayesian framework using our MCMC sampler. The binding probability at each base pair location is achieved by integrating out all other locations (see ‘Binding probabilities at single nucleotide resolution’ Section more details). <a href="#pone-0001820-g013">Figure 13</a> shows a representative result of binding probabilities for SRF on the Actc1 and M23768 promoters, SP1 on the Myod1 promoter, and TEAD1 on the Myh6 promoter, with and without evolutionary conservation as an additional data source. These results correspond to the data shown in <a href="#pone-0001820-g006">Figure 6</a>. The Bayesian inference (left column) is able to find and assign a high probability to all the annotated sites, although some presumably non-functional (i.e., non-annotated) sites are identified as well. The use of evolutionary conservation improves the performance of the Bayesian inference by “biasing” the binding sites towards conserved regions. In particular, all the annotated (resp. non-annotated) binding sites are assigned a higher (resp. lower) binding probability than without conservation data. It is also worth noting that our probabilistic inference method does not assign a zero probability to any of the possible binding sites. This is useful because the motif models are known to contain uncertainty and particularly because the binding prediction is based on noisy data. Hence, a probabilistic approach is more flexible and principled.</p>
<div class="figure" id="pone-0001820-g013"><div class="img"><a name="pone-0001820-g013" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g013&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g013"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g013&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g013/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g013/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g013/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g013/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g013.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g013/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g013/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g013.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 13.  <span>Estimated binding probabilities on a single base pair resolution for SRF on (a) the Actc1 and (c) M23768 promoters, (e) SP1 on the Myod1 promoter, and (g) TEAD1 on the Myh6 promoter without any additional information.</span></strong></p><a id="article1.body1.sec2.sec11.fig1.caption1.p1" name="article1.body1.sec2.sec11.fig1.caption1.p1"></a><p>Subplots (b), (d), (f) and (h) show the same results but with evolutionary conservation as the additional data source. The blue and red graphs indicate the start of the binding sites. The annotated binding sites are shown with gray vertical bars. These results correspond to <a href="#pone-0001820-g006">Figure 6</a>.</p>
<span>doi:10.1371/journal.pone.0001820.g013</span></div>

<h4>Genome-wide Analysis</h4>
<a id="article1.body1.sec2.sec12.p1" name="article1.body1.sec2.sec12.p1"></a><p>Encouraged by the above performance evaluations, we applied the proposed likelihood-based binding prediction method to the 2K base pair upstream promoter regions of all 20397 mouse genes, where the genomic locations of the promoters are based on RefSeq gene annotations. Evolutionary conservation was used as an additional data sources as explained above and binding specificities for 266 TFs were again taken from TRANSFAC Professional version 10.3. Prior to analyzing the promoter sequences, DNA repeats were found using RepeatMasker <a href="#pone.0001820-Smit1">[80]</a> and ignored from further analysis. While we mainly focus on general summary results here, the full binding probability results for all mouse TF-gene pairs (20397×266 table) are available on the supplementary web page.</p>
<a id="article1.body1.sec2.sec12.p2" name="article1.body1.sec2.sec12.p2"></a><p><a href="#pone-0001820-g014">Figures 14 (a)–(c)</a> show histograms of the estimated binding probabilities, maximum a posteriori (MAP) number of binding sites and the expected number of binding sites, respectively, over all 5.4 million TF-promoter pairs. A histogram of the estimated binding probabilities in <a href="#pone-0001820-g014">Figure 14 (a)</a>, for example, shows strong bias towards weak binding probabilities. Similarly, zero binding sites is by far the most frequent case among the MAP number of binding sites (<a href="#pone-0001820-g014">Figure 14 (b)</a>) and the expected number of binding sites are also heavily biased towards small values (<a href="#pone-0001820-g014">Figure 14 (c)</a>). These findings are consistent with the current view that, on average, biological interaction networks, such as transcriptional regulatory networks, are sparsely connected (see <a href="#pone.0001820-Reguly1">[81]</a>). Perhaps more interestingly, the histogram of binding probabilities is clearly bi-modal. The second, smaller peak is located at high binding probabilities, close to the probability value 1. Therefore, the histogram of binding probabilities can, for example, be considered as a mixture of two exponentially decreasing (no binding) and increasing (binding) distributions.</p>
<div class="figure" id="pone-0001820-g014"><div class="img"><a name="pone-0001820-g014" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g014&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g014"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g014&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g014/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g014/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g014/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g014/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g014.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g014/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g014/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g014.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 14.  <span>Histogram of (a) the estimated binding probabilities, (b) maximum a posteriori (MAP) number of binding sites, and (c) the expected number of binding sites over all 5.4 million TF-promoter pairs.</span></strong></p><a id="article1.body1.sec2.sec12.fig1.caption1.p1" name="article1.body1.sec2.sec12.fig1.caption1.p1"></a><p>Histogram frequency at bin value 10 in Figure (b) (resp. value about 5 in Figure (c)) includes all values that exceed 10 (resp. 5).</p>
<span>doi:10.1371/journal.pone.0001820.g014</span></div><a id="article1.body1.sec2.sec12.p3" name="article1.body1.sec2.sec12.p3"></a><p><a href="#pone-0001820-g015">Figures 15 (a) and (b)</a> further summarize the estimated binding probabilities over different TFs and promoter sequences, respectively. Although most TFs have relatively low average binding probabilities (<a href="#pone-0001820-g015">Figure 15 (a)</a>), say between 0 and 0.3, average binding probabilities also possess some degree of variability. For example, the TF that has the highest average binding probability (about 0.52) is the well-known Sp1 protein (see <a href="#pone.0001820-Thomas1">[82]</a>) that has been reported to bind practically everywhere in the human genome <a href="#pone.0001820-Cawley1">[83]</a>. <a href="#pone-0001820-g015">Figure 15 (b)</a> displays the histogram of the average binding probability to a promoter sequence. Most of the average binding probabilities are again relatively small, say between 0 and 0.3, but there are also a few promoter sequences for which the average binding probability is higher. This feature is often referred to as scale-free. Note that by scaling the <em>x</em>-axis of <a href="#pone-0001820-g015">Figure 15 (b)</a> by the number of TFs (266) one gets an estimate of the number of regulators per gene.</p>
<div class="figure" id="pone-0001820-g015"><div class="img"><a name="pone-0001820-g015" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g015&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001820" data-uri="info:doi/10.1371/journal.pone.0001820.g015"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.g015&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g015/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g015/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g015/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g015/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g015.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001820.g015/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001820.g015/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001820.g015.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 15.  <span>Histogram of the estimated average binding probabilities over (a) different TFs and (b) promoter sequences.</span></strong></p><span>doi:10.1371/journal.pone.0001820.g015</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In our analysis, we primarily focused on estimating the binding probability of a TF to either the whole analyzed promoter or at single base pair resolution (using the Bayesian method). We also introduced an extension for inferring combinatorial regulation. Given the flexibility of our probabilistic (Bayesian) modeling framework, virtually any question can be answered probabilistically within it. For example, Beer and Tavazoie <a href="#pone.0001820-Beer1">[51]</a> introduced a method for predicting gene expression using positional and combinatorial constraints for local sequence elements. Similar questions can also be answered probabilistically in the proposed framework, e.g., “what is the probability that two TFs, A and B, both have binding sites in a given promoter such that binding site for A is closer to the transcription start site than that of B, binding sites are within 50 base pairs from each other and within 150 base pairs from the transcription start site?” Alternatively, the above type of positional and combinatorial constraints that have been identified in previous studies can be included into the proposed framework via informative prior distributions.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>One popular way of analyzing expression data is based on clustering similarly behaving genes together or finding groups of genes that are differentially expressed. The gene sets found are then typically searched for common (either known or unknown) sequence motifs. A potentially very useful extension of our framework will be to develop a method for computing the probability that a set of genes (or a fraction of them) have a binding site for a TF or for a set of TFs.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>A number of other possible extensions are also easily included in this probabilistic modeling framework. For example, some proteins interact to form a heterodimer and bind as a complex, in which case the potential binding sites of (all or a subset of) the constituent TFs may be more likely to be physically close to each other. Incorporation of protein-protein interaction databases may help in revealing such mechanisms. Evolutionary conservation was included in the framework by utilizing the conservation scores of an input promoter. One can also simultaneously analyze the corresponding promoter in other organisms to check if they have a binding site for the same TF (in the corresponding location), see <a href="#pone.0001820-Sandelin2">[46]</a>. In addition to this, an interesting extension would be to modify the proposed framework to take into account the conservation of TF binding patterns <a href="#pone.0001820-Hallikas1">[84]</a>. An alternative future extension is to incorporate probabilistic evolutionary processes within the proposed binding prediction framework (see <a href="#pone.0001820-Siddharthan1">[6]</a>). Finally, as noted before, our choice of using a PSFM model for binding sites is arbitrary and the same framework can be easily extended to other binding site models as well.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Our final note is devoted to the general distinction between motif discovery and binding site prediction methods. The proposed Bayesian method interprets binding specificities as random variables whose prior parameters define the amount of uncertainty associated with each TF binding model. By gradually forcing all pseudo counts to be equal (unity), i.e., increasing the uncertainty, the Bayesian binding prediction method indeed turns into a pure motif discovery method. Thus, for uncertain binding specificities, the Bayesian method can also be used as a motif discoverer.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Our future work includes developing the framework in the direction of the aforementioned extensions. We are also extending our genome-wide analysis to yeast. Predicting TF binding in yeast is interesting not only because it is the most often considered model organism, but also because yeast has a well-developed nucleosome model <a href="#pone.0001820-Segal1">[74]</a> and more abundant ChIP-chip data (see <a href="#pone.0001820-Lee1">[65]</a>, <a href="#pone.0001820-Qi1">[9]</a>).</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>A central goal in the described computational analysis is accurate TF binding prediction from multiple data sources. However, because TF binding does not necessarily imply transcriptional regulation, it is also important to further extend computational methods to incorporate other, functional data, such as gene expression or protein level (time series) measurements. Statistical inference of transcriptional regulatory networks from a combination of gene expression time series, promoter sequence and binding specificity data has been studied in <a href="#pone.0001820-Segal2">[85]</a>–<a href="#pone.0001820-deHoon1">[87]</a>, <a href="#pone.0001820-Chen2">[63]</a>. Probabilistic formulation of TF binding prediction is particularly useful in such modeling approaches as it naturally allows a principled fusion of various data sources. In a Bayesian context, it is both intuitive and simple to integrate diverse data sources via informative prior distributions. This is similar with what we proposed for the probabilistic integration of multiple data sources above. Similar ideas have already been introduced for the Bayesian inference of transcriptional regulatory networks from gene expression data where a prior distribution of network models is estimated from promoter sequences and TF binding specificities <a href="#pone.0001820-Chen2">[63]</a> or from ChIP-chip data <a href="#pone.0001820-Bernard1">[62]</a>. We are currently developing Bayesian learning methods for transcriptional regulatory networks that can make principled statistical inference from diverse sets of data sources, such as the ones already discussed in this work and other functional data.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>We have developed a flexible and comprehensive framework for TF binding prediction from multiple data sources. The proposed methods are probabilistic in nature and, thus, directly assess our degree of belief in binding or non-binding in terms of probabilities. Instead of assessing TF binding at each nucleotide location separately, we extended the binding prediction methods to analyze each promoter sequence as a whole. This gives a more complete view of a TF binding to and possibly regulating a target gene. Although we primary focused on answering the question of whether the entire promoter has a binding site for a TF, we also developed a method for computing binding probabilities at each nucleotide position by essentially integrating out other locations in a promoter. Most importantly, the proposed methods can make principled inference from multiple data sources that can include, among others, multiple motif models, evolutionary conservation, regulatory potential, CpG islands, nucleosome positioning, DNase hypersensitive sites and ChIP-chip. Results on our carefully constructed test set demonstrate that principled data fusion can significantly improve the performance of binding prediction methods. Recent technological developments, such as protein binding microarrays, now allow accurate measurements of TF binding specificities to be gathered in a high-throughput fashion. Using accurate binding specificity measurements together with principled TF binding prediction methods can provide a competitive alternative to traditional condition specific ChIP-chip experiments, especially when TF binding prediction incorporates multiple additional data sources. Our genome-wide TF-DNA binding results for mouse indicate relatively sparse connectivity between TFs and their target genes, consistent with previous results. The probabilistic formulation of TF binding prediction is particularly useful for integrating our results as building blocks in other computational methods. To that end, we have also implemented a web tool, ProbTF, which allows users to analyze their own promoter sequences and additional data sources.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The computational methods are implemented in Matlab and will be made available as an open-source library upon publication. The test set, including all sequences and additional information sources, will also be made available on a supplementary web site. A preliminary version of the computational methods presented in Sections ‘Modeling framework’, ‘Likelihood approach: one motif model θ’, and ‘Likelihood approach: multiple motif models Θ’ have been reported in our previous conference article <a href="#pone.0001820-Lhdesmki1">[88]</a>.</p>

<h4>Modeling framework</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Let <em>S</em> = (<em>s</em><sub>1</sub>,…,<em>s<sub>L</sub></em>) denote a single strand of a promoter sequence, where <em>s<sub>i</sub></em>Є{A, C, G, T} and <em>L</em> is the length of the sequence. (Generalizations to double stranded DNA sequences are given later on.) Let <em>Q</em> denote the number of (hidden) motif instances in sequence <em>S</em>. This is one of the key quantities estimated from the data. Further, let <em>A</em> denote the (unknown) start positions of non-overlapping motif instances in sequence <em>S</em>. For example, if <em>Q</em> = <em>c</em>, then <em>A</em> = {<em>a</em><sub>1</sub>,…,<em>a<sub>c</sub></em>}. Thus, a promoter consists of <em>c</em> motif instances and <em>c</em>+1 background sequence chunks, some of which can be empty. In the following we assume that <em>A</em> always defines start positions for non-overlapping motifs.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Non-binding background sequence locations are modeled by the commonly used <em>d</em>th order Markovian background model φ. That is, let<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e009&amp;representation=PNG" class="inline-graphic"></span><br>denote the probability of observing nucleotide <em>s<sub>i</sub></em> at the <em>i</em>th position of a promoter sequence <em>S</em> in the background model φ given <em>d</em> previous nucleotides. For simplicity, we assume that for positions <em>i</em>≤<em>d</em> we have access to <em>s</em><sub>−<em>d</em>+1</sub>,…,<em>s</em><sub>0</sub>. We could alternatively define a separate probability distribution for the first <em>d</em> nucleotides. The likelihood of the background model, <em>A</em> = Ø, is thus <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e010&amp;representation=PNG" class="inline-graphic"></span>.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Motifs are modeled using the standard PSFM model θ which is a product of independent multinomial distributions <a href="#pone.0001820-Staden1">[21]</a>. Similarly as above, let<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e011&amp;representation=PNG" class="inline-graphic"></span><br>denote the probability of observing nucleotide <em>s<sub>i</sub></em> at the <em>j</em>th (<em>j</em> = 1,…,ℓ) position of a motif model θ, where ℓ is the length of the motif. Note that <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e012&amp;representation=PNG" class="inline-graphic"></span> for all <em>j</em> and that probabilities for different <em>j</em>s are independent. The probability of sequence <em>S</em>, given non-overlapping motif positions and motif and background models, is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where |<em>A</em>| = <em>Q</em> = <em>c</em> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e014&amp;representation=PNG" class="inline-graphic"></span>.</p>


<h4>Likelihood approach: one motif model θ</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Using Bayes' rule, the probability of <em>c</em> motif instances, given the sequence <em>S</em>, is (see <a href="#pone.0001820-Thijs1">[42]</a>)<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e015&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>where the normalization factor has the following form<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e016&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e017&amp;representation=PNG" class="inline-graphic"></span> is the maximum number of non-overlapping ℓ-length motifs in an <em>L</em>-length sequence. Note that since the sum in Equation (5) has only <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e018&amp;representation=PNG" class="inline-graphic"></span> terms (instead of infinitely many) the normalization factor can be computed exactly. The likelihood of sequence <em>S</em>, given that it contains <em>c</em> motif instances, can be obtained by summing over all possible positions <em>A</em> of <em>c</em> motif instances <a href="#pone.0001820-Thijs1">[42]</a><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e019&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br>where in the last equality we have used <em>P</em>(<em>S</em>|<em>A</em>,<em>Q</em> = <em>c</em>,θ,φ) = <em>P</em>(<em>S</em>|<em>A</em>,θ,φ) and Equation (3). The above probabilistic formulation (Equations (4)–(6)) is practically identical to the one proposed by Thijs <em>et al.</em> <a href="#pone.0001820-Thijs1">[42]</a>.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>As in <a href="#pone.0001820-Thijs1">[42]</a>, let us assume for now that, for a fixed value of <em>Q</em>, the prior over motif positions <em>A</em> is uniform and is inversely proportional to the number of different motif positions, i.e., <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e020&amp;representation=PNG" class="inline-graphic"></span>. Let <em>R</em>(<em>S</em>|<em>Q</em> = <em>c</em>,θ,φ) denote the sum in Equation (6) without the (constant) prior term <em>P</em>(<em>A</em>|<em>Q</em> = <em>c</em>,θ,φ). The likelihood in Equation (6) can be computed efficiently using the following recursion<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e021&amp;representation=PNG" class="inline-graphic"></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e022&amp;representation=PNG" class="inline-graphic"></span> denotes a subsequence of <em>S</em> (note that <em>S</em><sub>1</sub> = <em>S</em>). For the prior over the number of motif instances, we use a probability distribution motivated by previous studies <a href="#pone.0001820-Thijs1">[42]</a><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e023&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e024&amp;representation=PNG" class="inline-graphic"></span>. Since <em>P</em>(<em>Q</em> = <em>c</em>|θ,φ) does not depend on θ or φ we also rewrite <em>P</em>(<em>Q</em> = <em>c</em>|θ,φ) = <em>P</em>(<em>Q</em> = <em>c</em>). One could consider other priors as well, such as ones that depend on the information content of the matrix (see <a href="#pone.0001820-Siddharthan1">[6]</a>).</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>The probability that a given TF (defined by θ) binds to a gene having promoter sequence <em>S</em>, denoted by θ→<em>S</em>, can be computed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e025&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e026&amp;representation=PNG" class="inline-graphic"><span class="note">(9)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e027&amp;representation=PNG" class="inline-graphic"><span class="note">(10)</span></span><br>where <em>P</em>(<em>Q</em> = <em>c</em>|<em>S</em>,θ,φ) can be obtained using Equations (4)–(7). Assuming a single binding position is sufficient for transcriptional regulation, then the above probability of binding can also be interpreted as a probability of transcriptional regulation. It is straightforward to adopt the above probability <em>P</em>(θ→<em>S</em>|<em>S</em>,θ,φ) for the requirement of having multiple binding sites. Also note that we have the distribution of having any number (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e028&amp;representation=PNG" class="inline-graphic"></span>) of binding sites from which we can compute, e.g., the expected (mean) and the maximum a posteriori (MAP) number of binding sites.</p>


<h4>Likelihood approach: multiple motif models Θ</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A TF can recognize several different types of binding sites and is then characterized by several motif models Θ = (θ<sup>(1)</sup>,…,θ<sup>(<em>m</em>)</sup>) each having length ℓ<em><sub>i</sub></em>. Let πЄ{1,…,<em>m</em>}<em><sup>c</sup></em> denote a configuration of motif models from Θ in <em>A</em>. That is, π<em><sub>i</sub></em> specifies the motif model <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e029&amp;representation=PNG" class="inline-graphic"></span> at location <em>a<sub>i</sub></em>. For notational convenience, define<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e030&amp;representation=PNG" class="inline-graphic"><span class="note">(11)</span></span><br>and note that (see also Equation (3))<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e031&amp;representation=PNG" class="inline-graphic"><span class="note">(12)</span></span><br></p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>The probability of <em>c</em> motif instances can be obtained using Bayes' rule as in Equations (4)–(5) but θ replaced with Θ. Further, following Equation (6), the likelihood of sequence <em>S</em> given <em>c</em> motif instances can be obtained by summing over all possible positions and configurations<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e032&amp;representation=PNG" class="inline-graphic"><span class="note">(13)</span></span><br>where ℓ<sub>min</sub> = {ℓ<sub>1</sub>,…,ℓ<em><sub>m</sub></em>}. Let us again start by assuming a uniform prior over motif positions <em>A</em> and configurations π (for each fixed value of <em>Q</em>), and let <em>R</em>(<em>S</em>|<em>Q</em> = <em>c</em>,Θ,φ) denote the sum in Equation (13) without the (constant) prior term <em>P</em>(<em>A</em>,π|Q = <em>c</em>,Θ,φ). A computationally efficient recursive formula can be written as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e033&amp;representation=PNG" class="inline-graphic"></span><br>A closed form formula for uniform <em>P</em>(<em>A</em>,π|<em>Q</em> = <em>c</em>,Θ,φ) is more difficult to obtain in general, but it can be computed numerically using a similar recursion as the one above.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>The prior <em>P</em>(<em>Q</em> = <em>c</em>|Θ,φ) depends now on Θ and thus can be adjusted for multiple motifs. However, it is unrealistic to assume that different motif models (θ<sup>(1)</sup>,…, θ<sup>(<em>m</em>)</sup>) are independent. Indeed, it is likely that they are strongly dependent. Therefore, we use the same prior <em>P</em>(<em>Q</em> = <em>c</em>) as in the case of a single motif model as a first approximation.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>Let Θ→<em>S</em> denote that a TF characterized by Θ binds to a promoter <em>S</em>. The probability that at least one of the motif models in Θ has a binding site in <em>S</em>, <em>P</em>(Θ→<em>S</em>|<em>S</em>,Θ,φ), can be computed as in Equations (8)–(10) but θ replaced with Θ. Under the same premise as above that a single binding site is sufficient for gene regulation, <em>P</em>(Θ→<em>S</em>|<em>S</em>,Θ,φ) can be interpreted as the probability of regulation.</p>
<a id="article1.body1.sec4.sec3.p5" name="article1.body1.sec4.sec3.p5"></a><p>The above probabilistic modeling framework that incorporates multiple motif models can be viewed as an extension of a framework proposed in <a href="#pone.0001820-Thijs1">[42]</a>. Note that the proposed framework is also similar to hidden Markov models (HMM) that have been proposed previously <a href="#pone.0001820-Frith1">[36]</a>–<a href="#pone.0001820-Sinha1">[38]</a>, <a href="#pone.0001820-Sinha2">[44]</a>. An HMM is defined by motif and background models Θ and φ and the transition probabilities (between the states of the HMM) whereas the modeling framework described herein is built on motif and background models alone, with additional information brought into the computation via the priors <em>P</em>(<em>Q</em>|Θ,φ) and <em>P</em>(<em>A</em>,π|<em>Q</em>,Θ,φ).</p>


<h4>Bayesian approach</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>TF binding specificities are derived from experimental data sets, some of which have extremely small sample sizes (as low as five reported binding sequences). PSFM models can therefore contain a considerable amount of uncertainty. Instead of assuming motif models θ to be known exactly, as above, it is useful to take the uncertainty in the motif models themselves into account. This can be done naturally in a Bayesian setting where the parameters/models are considered as random variables. We describe the Bayesian methods directly for the case of multiple motifs. The single motif case can be obtained as a special case by setting <em>m</em> = 1 and omitting π.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Using Bayes' rule, the probability of motif positions <em>A</em> and configurations π, given the sequence <em>S</em>, is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e034&amp;representation=PNG" class="inline-graphic"><span class="note">(14)</span></span><br></p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>The marginal likelihood <em>P</em>(<em>S</em>|<em>A</em>,π) is obtained by integrating over parameters<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e035&amp;representation=PNG" class="inline-graphic"></span><br>where <em>P</em>(<em>S</em>|<em>A</em>,π,Θ,φ) is the same product of multinomial distributions as in Equation (12) and <em>P</em>(Θ,φ|<em>A</em>,π) = <em>P</em>(Θ,φ)defines a prior distribution for the parameters.</p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4"></a><p>TF-DNA binding databases typically provide information in the form of “the number of times a TF has been observed to bind a given sequence.” These sequences are also aligned and aligned counts are summarized in position specific weight matrices (TRANSFAC, JASPAR), which we denote as α<em><sub>ij</sub></em><sup>(<em>k</em>)</sup>. Similar counts can also be obtained for the background model (denoted by α<em><sub>ij</sub></em><sup>(0)</sup>) from genomic sequences that do not contain (known) binding sites. Therefore, it is natural to use a Dirichlet prior for the parameters, which is defined by so-called pseudo-counts.</p>
<a id="article1.body1.sec4.sec4.p5" name="article1.body1.sec4.sec4.p5"></a><p>Let us rewrite the motif model parameters (independent multinomial distributions) for now as θ<sup>(<em>k</em>)</sup>(<em>i</em>, <em>j</em>) = θ<em><sub>ij</sub></em><sup>(<em>k</em>)</sup> which again defines the probability of seeing nucleotide <em>i</em>Є{A, C, G, T} at the <em>j</em>th (1≤<em>j</em>≤ℓ<em><sub>k</sub></em>) position in the <em>k</em>th (1≤<em>k</em>≤<em>m</em>) motif model. Denote θ<em><sub>j</sub></em><sup>(<em>k</em>)</sup> = {θ<em><sub>ij</sub></em><sup>(<em>k</em>)</sup>|<em>i</em>Є{A, C, G, T}}. The Dirichlet prior for each θ<em><sub>j</sub></em><sup>(<em>k</em>)</sup> with hyperparameters α<em><sub>ij</sub></em><sup>(<em>k</em>)</sup> is defined as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e036&amp;representation=PNG" class="inline-graphic"></span><br>where θ<em><sub>ij</sub></em><sup>(<em>k</em>)</sup>≥0, ∑<em><sub>i</sub></em>θ<em><sub>ij</sub></em><sup>(<em>k</em>)</sup> = 1, α<em><sub>ij</sub></em><sup>(<em>k</em>)</sup>&gt;0, and Γ(·) is the Gamma function. Priors for different <em>j</em> and <em>k</em> are assumed to be independent. The Dirichlet prior for the background model is defined similarly.</p>
<a id="article1.body1.sec4.sec4.p6" name="article1.body1.sec4.sec4.p6"></a><p>The Dirichlet prior is also a conjugate prior for multinomials. Consequently, the marginal likelihood has a closed-form solution. Let <em>N<sub>ij</sub></em><sup>(<em>k</em>)</sup> denote the number of times nucleotide <em>i</em> is observed at the <em>j</em>th position in the <em>k</em>th motif model given <em>S</em>, <em>A</em> and π. Denote α<em><sub>j</sub></em><sup>(<em>k</em>)</sup> = ∑<em><sub>i</sub></em>α<em><sub>ij</sub></em><sup>(<em>k</em>)</sup> and <em>N<sub>j</sub></em><sup>(<em>k</em>)</sup> = ∑<em><sub>i</sub>N<sub>ij</sub></em><sup>(<em>k</em>)</sup>. The marginal likelihood can be written as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e037&amp;representation=PNG" class="inline-graphic"><span class="note">(15)</span></span><br>where the first (resp. the second) part corresponds to the background model (resp. <em>m</em> motif models).</p>
<a id="article1.body1.sec4.sec4.p7" name="article1.body1.sec4.sec4.p7"></a><p>To keep likelihood-based and Bayesian approaches comparable, we use the same prior here, i.e.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e038&amp;representation=PNG" class="inline-graphic"><span class="note">(16)</span></span><br>where <em>P</em>(<em>A</em>,π|<em>Q</em> = |<em>A</em>|) and <em>P</em>(<em>Q</em> = |<em>A</em>|) are as in Equations (13) and (7), respectively. Note that we write <em>P</em>(<em>A</em>,π|<em>Q</em> = |<em>A</em>|) instead of <em>P</em>(<em>A</em>,π|<em>Q</em> = |<em>A</em>|, Θ,φ) because the (uniform) prior depends only on the widths of the motif models, ℓ<em><sub>i</sub></em>, and not on the actual parameters Θ or φ.</p>
<a id="article1.body1.sec4.sec4.p8" name="article1.body1.sec4.sec4.p8"></a><p>Because some of the motif models are remarkably diffuse (computed from only a few example sequences), we do not use the PSWMs as pseudo counts directly. We instead use a version that incorporates a so-called prior strength term <em>M</em>, i.e.,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e039&amp;representation=PNG" class="inline-graphic"><span class="note">(17)</span></span><br></p>
<a id="article1.body1.sec4.sec4.p9" name="article1.body1.sec4.sec4.p9"></a><p>This prevents a (single) sequence to have too strong of an influence on the posterior parameter values. For simplicity, we use the same prior strength for all the motif models, although this does not need to be the case in general. In addition, we add a small number (one) to each α<em><sub>i</sub></em><sub>,<em>j</em></sub><sup>(<em>k</em>)</sup> to prevent zero entries. Finally, to preserve comparability between likelihood and Bayesian approaches, the normalized motif models for the likelihood based approach are computed from the recomputed pseudo-counts used in the Bayesian estimation, i.e.,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e040&amp;representation=PNG" class="inline-graphic"><span class="note">(18)</span></span><br></p>
<a id="article1.body1.sec4.sec4.p10" name="article1.body1.sec4.sec4.p10"></a><p>Recall that in the Bayesian framework, the mean of θ<sup>(<em>k</em>)</sup>(<em>i</em>, <em>j</em>) relative to the prior distribution <em>P</em>(θ<em><sub>j</sub></em><sup>(<em>k</em>)</sup>|α) is equal to the quantity in Equation (18).</p>


<h4>MCMC estimation for Bayesian inference</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Unfortunately, there is no efficient recursive formula to compute the probabilities <em>P</em>(<em>A</em>,π|<em>S</em>) for all <em>A</em> and π. However, one can solve the problem by using stochastic estimation methods. Here we propose to sample positions <em>A</em> and configurations π directly from the posterior <em>P</em>(<em>A</em>,π|<em>S</em>) using Markov chain Monte Carlo (MCMC). We develop a Metropolis-Hastings (MH) algorithm for this purpose. For an introduction to MCMC methods, see <a href="#pone.0001820-Robert1">[89]</a>.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>The MH algorithm is completely specified by a proposal distribution <em>G</em>(<em>A</em>′,π′|<em>A</em>,π) which proposes new pairs (<em>A</em>′,π′) given the current (<em>A</em>,π). We define <em>G</em> as follows.</p>


<ul class="bulletlist">

<li>Motif addition with probability <em>p</em>: for a uniformly chosen motif model θ<sup>(<em>i</em>)</sup>ЄΘ, propose a new, non-occupied/non-overlapping motif position uniformly randomly (if a free location exists).</li>

<li>Motif deletion with probability 1−<em>p</em>: delete an existing motif uniformly randomly (if a motif exists).</li>

</ul><a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>We use <em>p</em> = 0.5. The proposed pair (<em>A</em>′,π′) is then accepted with probability<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e041&amp;representation=PNG" class="inline-graphic"></span><br>which satisfies the detailed balance condition. Convergence in distribution to the desired posterior (in the limit of infinitely many samples) is guaranteed if, in addition to satisfying the detailed balance, the resulting chain is also irreducible and aperiodic. Irreducibility of the chain follows from the fact that any pair (<em>A</em>′,π′) can be reached from any pair (<em>A</em>,π) by repeatedly adding or deleting one motif at a time and each step has a positive selection probability. Aperiodicity of the chain can be seen similarly. For example, the probability that the chain first deletes all <em>c</em> = |<em>A</em>| motifs, then stays at <em>A</em> = Ø any number of steps (by trying to delete a motif), and finally adds the same <em>c</em> motifs, is non-zero. In other words, the probability of moving from (<em>A</em>,π) back to (<em>A</em>,π) in 2<em>c</em> or more steps is non-zero. Therefore, the period (the greatest common factor of integers that include at least {2<em>c</em>, 2<em>c</em>+1,…}) for all the states (<em>A</em>,π) is 1 and the chain is aperiodic. Note that computing the above Bayes factor can be done very efficiently because only a single motif is added or deleted at a time. In particular, only two parts of the marginal likelihoods are different, corresponding to the motif that is added/deleted and the background chunk that is deleted/added. For integer-valued pseudo-counts, the computation of the Bayes factor reduces even further because Γ(<em>n</em>) = (<em>n</em>−1)!. After a proper burn-in period <em>B</em>, a dependent sample ((<em>A</em><sup>(<em>B</em>+1)</sup>, π<sup>(<em>B</em>+1)</sup>), (<em>A</em><sup>(<em>B</em>+2)</sup>, π<sup>(<em>B</em>+2)</sup>), …, (<em>A</em><sup>(<em>B</em>+<em>N</em>)</sup>, π<sup>(<em>B</em>+<em>N</em>)</sup>)) is collected.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>Although the chain is ergodic as shown above, it is important to monitor convergence of the MCMC algorithm for finite samples to guarantee the desired output. Bayesian inference in this case can be considered as a model selection problem where the model space consists of all valid pairs (<em>A</em>,π). Although the model space is discrete and finite, standard convergence diagnostics over the full model space are difficult to apply in practice. Better suited diagnostic methods are the ones that are specifically developed for model selection problems, such as the ones in the context of reversible jump MCMC methods (see <a href="#pone.0001820-Brooks1">[90]</a>). A general strategy is to reduce the model space and monitor the convergence in a lower dimensional space. Here we consider a method that compares the marginal probabilities of having <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e042&amp;representation=PNG" class="inline-graphic"></span> binding sites, <em>P</em>(<em>Q</em> = <em>c</em>|<em>S</em>) = ∑<sub>|<em>A</em>| = <em>c</em></sub> ∑<sub>π</sub> <em>P</em>(<em>A</em>,π|<em>S</em>), from two independent chains (see <a href="#pone.0001820-Husmeier2">[91]</a>). Note that <em>P</em>(<em>Q</em> = <em>c</em>|<em>S</em>) is exactly the distribution that we are interested in when assessing TF binding. As for the convergence diagnostic, we use a heuristic that reports two chains as having converged if the <em>L</em><sub>1</sub>-distance between two independent estimates of <em>P</em>(<em>Q</em>|<em>S</em>) is within an accepted error threshold (we use 0.025). We could also use a formal hypothesis testing (e.g. chi-squared or Kolmogorov-Smirnov) for assessing lack of convergence by sub-sampling the two chains to get (approximately) independent samples <a href="#pone.0001820-Brooks1">[90]</a>. The less involved heuristic seems to serve our purposes. We use <em>B</em> = 5·10<sup>5</sup> for the burn-in and <em>N</em> = <em>i</em>·<em>B</em> for the sample size, where index <em>i</em> = 1, 2,… is increased until the chain pair passes the convergence diagnostic.</p>
<a id="article1.body1.sec4.sec5.p5" name="article1.body1.sec4.sec5.p5"></a><p>As above, the quantities of interest include the probability of having at least one binding site for at least one of the motif models, denoted as Θ→<em>S</em> (or, conversely, having no binding sites),<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e043&amp;representation=PNG" class="inline-graphic"></span><br>and the posterior probability of having exactly <em>Q</em> = <em>c</em> binding sites, <em>P</em>(<em>Q</em> = <em>c</em>|<em>S</em>). These quantities can be estimated directly from the chain<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e044&amp;representation=PNG" class="inline-graphic"></span><br>and<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e045&amp;representation=PNG" class="inline-graphic"></span><br>where χ is the indicator function. The sample averages converge to the true posterior probabilities almost surely. The final estimates are obtained by averaging estimates from two independent chains that pass the convergence diagnostic.</p>


<h4>Combinatorial regulation</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>From the point of view of modeling transcriptional regulatory networks, it is also important to study combinatorial regulation by several TFs Ω = (Θ<sup>(1)</sup>,…, Θ<sup>(<em>p</em>)</sup>), each with a set of motif models<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e046&amp;representation=PNG" class="inline-graphic"></span><br>A key quantity then is the probability that each different TF has at least one binding site (for at least one of their motif models). Let the configurations now be defined as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e047&amp;representation=PNG" class="inline-graphic"></span><br>and define a set<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e048&amp;representation=PNG" class="inline-graphic"><span class="note">(19)</span></span><br></p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>In other words, the set <em>C</em> says that every TF is presented with some motif in every configuration π. Note that Equation (19) implies that if πЄ<em>C</em> then |π| = <em>c</em>≥<em>p</em>. The probability of <em>p</em> TFs binding a promoter <em>S</em> jointly can be written as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e049&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p>Probabilities <em>P</em>(<em>A</em>,π|<em>S</em>) are computed as in Equations (14–16) except that <em>m</em> in Equation (15) is replaced with <em>m</em><sub>1</sub>+…+<em>m<sub>p</sub></em> and the prior <em>P</em>(<em>A</em>,π) defined in Equation (16) is adjusted for several TFs. Let <em>c<sub>i</sub></em>≥0 denote the number of binding sites for the <em>i</em>th TF. Assuming that the number of binding sites for different TFs are independent, then one can model the joint number of binding sites, <em>Q</em><sub>Ω</sub>, as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e050&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec4.sec6.p4" name="article1.body1.sec4.sec6.p4"></a><p>There is no efficient formula to compute <em>P</em>(Ω→<em>S</em>|<em>S</em>), or even the analogous likelihood based quantity, so we only formulate this in the Bayesian context and solve it with MCMC sampling. The same MH algorithm as above can be applied except that a motif θ<sup>(<em>i</em>,<em>j</em>)</sup> is now added uniformly randomly from a list of motif sets Ω. Finally, <em>P</em>(Ω→<em>S</em>|<em>S</em>) can be directly estimated from a chain as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e051&amp;representation=PNG" class="inline-graphic"><span class="note">(20)</span></span><br></p>
<a id="article1.body1.sec4.sec6.p5" name="article1.body1.sec4.sec6.p5"></a><p>We also consider a naive (likelihood-based) approximation that estimates the probability of combinatorial regulation by the product of individual binding probabilities, i.e.,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e052&amp;representation=PNG" class="inline-graphic"><span class="note">(21)</span></span><br></p>


<h4>Combining multiple information sources</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>TF binding predictions can be significantly improved by incorporating multiple additional data sources, such as evolutionary conservation, regulatory potential, CpG islands, nucleosome positioning, DNase hypersensitive sites or ChIP-chip, into our probabilistic inference framework. Let <em>D</em> denote a single additional data source that is indicative of functional binding sites. The data is assumed to be in the form <em>D</em> = (<em>P</em>(1),…, <em>P</em>(<em>L</em>)), where <em>P</em>(<em>i</em>) is the probability that the <em>i</em>th base pair location has one of the above mentioned properties (is conserved, belongs to a regulatory region, has a low nucleosome occupancy, etc.). Here we explain how such additional information can be used in a principled way to significantly improve TF binding inference.</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>We model the probability of <em>S</em> and <em>D</em> given <em>A</em>, π, Θ and φ as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e053&amp;representation=PNG" class="inline-graphic"></span><br>where we assume that <em>S</em> and <em>D</em> are conditionally independent and that the probability of <em>D</em> does not depend on the motif and background models. Let <em>I</em> = {1,…, <em>L</em>} denote base pair indices of a promoter and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e054&amp;representation=PNG" class="inline-graphic"></span> be indices of binding sites specified by <em>A</em> and π. The data <em>D</em> can then be modeled as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e055&amp;representation=PNG" class="inline-graphic"><span class="note">(22)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e056&amp;representation=PNG" class="inline-graphic"></span> and<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e057&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec4.sec7.p3" name="article1.body1.sec4.sec7.p3"></a><p>The factorization in Equation (22) is useful as it allows us to write <em>P</em>(<em>S</em>, <em>D</em>|<em>A</em>,π,Θ,φ) in the following compact form<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e058&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec4.sec7.p4" name="article1.body1.sec4.sec7.p4"></a><p>In particular, note that in the likelihood based approach the same efficient recursive formula as in ‘Likelihood approach: multiple motif models Θ’ Section can be applied to compute <em>P</em>(<em>S</em>, <em>D</em>|<em>Q</em> = <em>c</em>,Θ,φ).</p>
<a id="article1.body1.sec4.sec7.p5" name="article1.body1.sec4.sec7.p5"></a><p>In a Bayesian setting, data fusion can be performed similarly<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e059&amp;representation=PNG" class="inline-graphic"><span class="note">(23)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e060&amp;representation=PNG" class="inline-graphic"><span class="note">(24)</span></span><br></p>
<a id="article1.body1.sec4.sec7.p6" name="article1.body1.sec4.sec7.p6"></a><p>If we have several additional data sources <em>D<sub>k</sub></em> = (<em>P<sub>k</sub></em>(1),…, <em>P<sub>k</sub></em>(<em>L</em>)), 1≤<em>k</em>≤<em>N<sub>D</sub></em>, then we propose to combine them directly using<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e061&amp;representation=PNG" class="inline-graphic"></span><br>where <em>w<sub>k</sub></em>≥0 and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e062&amp;representation=PNG" class="inline-graphic"></span>.</p>
<a id="article1.body1.sec4.sec7.p7" name="article1.body1.sec4.sec7.p7"></a><p>Conceptually, the probability <em>P</em>(<em>D</em>|<em>A</em>,π) can be viewed as a positional prior for binding sites. But we do not combine <em>P</em>(<em>D</em>|<em>A</em>,π) with <em>P</em>(<em>A</em>,π) since all aforementioned data sources (probabilities <em>P</em>(<em>i</em>)) depend on the sequence <em>S</em> as well.</p>
<a id="article1.body1.sec4.sec7.p8" name="article1.body1.sec4.sec7.p8"></a><p>In practice, we do not use the above probabilities directly but a scaled version of them. For the <em>k</em>th additional data set we scale the original probabilities between [<em>d<sub>k</sub></em>, 1−<em>d<sub>k</sub></em>], where 0≤<em>d<sub>k</sub></em>≤0.5. We resort to fixed parameter values for <em>w<sub>k</sub></em>s and <em>d<sub>k</sub></em>s here because the estimated binding probabilities are not sensitive to small deviations in the value of <em>w<sub>k</sub></em> or <em>d<sub>k</sub></em> However, it is also possible to develop a full Bayesian treatment by introducing priors over <em>w<sub>k</sub></em>s and <em>d<sub>k</sub></em>s. Our simplified strategy also improves computational efficiency and makes it possible to use the standard MCMC instead of more computationally demanding trans-dimensional MCMC methods.</p>


<h4>Single vs. both strands</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>So far we have only considered using a single, either forward or reverse, strand for inferring TF binding. Extending the above methods to double-stranded DNA is straightforward. Let a promoter be denoted now as <em>S</em> = (<em>S</em>′,<em>S</em>″), where <em>S</em>′ and <em>S</em>″ are the forward and complementary reverse strands. Let us first assume that binding events on separate strands are independent of each other. In the case of double-stranded DNA, we need to compute the probability of an event Θ→<em>S</em>′ or Θ→<em>S</em>″. Let us denote <em>P</em>(Θ→<em>S</em>′|<em>S</em>,Θ,φ) = <em>P<sub>S</sub></em><sub>′</sub> and <em>P</em>(Θ→<em>S</em>″|<em>S</em>,Θ,φ) = <em>P<sub>S</sub></em><sub>″</sub>. Now<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e063&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec4.sec8.p2" name="article1.body1.sec4.sec8.p2"></a><p>Due to the OR-type of event (Θ→<em>S</em>′ or Θ→<em>S</em>″), the effective prior probability of having zero binding sites decreases. We account for this bias by changing <em>P</em>(<em>Q</em> = 0|Θ,φ) from <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e064&amp;representation=PNG" class="inline-graphic"></span>. The above formulation can also make use of the correlation between <em>P<sub>S</sub></em><sub>′</sub> and <em>P<sub>S</sub></em><sub>″</sub>. A similar extension works for the Bayesian case as well with the exception that <em>S</em>′ and <em>S</em>″ should not be analyzed separately even if binding on the two strands is assumed to be independent. Joint analysis of <em>S</em>′ and <em>S</em>″ is easily implemented using the same MCMC/MH algorithm as above. The only difference is that now the model should include separate start positions <em>A</em> = (<em>A</em>′,<em>A</em>″) and configurations π = (π′,π″) for the two strands. This approach also allows incorporating additional constraints, such as binding sites on two strands are less likely to occur at the same position due to physical space constraints.</p>


<h4>Binding probabilities at single nucleotide resolution</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Our computational methods are primarily designed to answer the question of whether the whole promoter has a binding site for a given TF. However, it is also useful to be able to infer binding probabilities at higher, single nucleotide, resolution. A motif position and configuration pair (<em>A</em>,π) contains a binding site (start position) at the <em>i</em>th nucleotide if <em>i</em>Є<em>A</em>. In the Bayesian context, the probability of a binding site at the <em>i</em>th location can be expressed and estimated simply as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001820.e065&amp;representation=PNG" class="inline-graphic"></span><br></p>


<h4>Comparison with other methods</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>We compare our proposed probabilistic method with traditional promoter scanning <a href="#pone.0001820-Staden1">[21]</a>–<a href="#pone.0001820-Claverie1">[23]</a>, <a href="#pone.0001820-Hertzberg1">[26]</a>, <a href="#pone.0001820-Baji1">[28]</a> and with a method that assesses the probability of binding <a href="#pone.0001820-Chen2">[63]</a>. Traditional scanning methods output a significance value for each position in a promoter. It is common practice to use the smallest <em>p</em>-value over a promoter as a measure of binding to the whole promoter, which we also use here. We use the same approach to adopt traditional promoter scanning to handle multiple motif models and report the smallest <em>p</em>-value over the multiple motif models. The same approach to handle multiple motif models is used with the method from <a href="#pone.0001820-Chen2">[63]</a>. Significance values of the traditional scanning method (as well as intermediate quantities in <a href="#pone.0001820-Chen2">[63]</a>) are computed relative to the null distribution derived from the negative promoter sequences.</p>


<h4>Data</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>The data set consists of a merger of annotated TF binding sites for mouse from the ABS <a href="#pone.0001820-Blanco1">[35]</a> and ORegAnno <a href="#pone.0001820-Montgomery1">[34]</a> databases. In total there are 47 annotated promoter sequences. The sequences have sets of length of around 500 nucleotides (ABS) and 2K nucleotides (ORegAnno). The TF binding sites are mapped onto the mm8 (February 2006) mouse assembly.</p>
<a id="article1.body1.sec4.sec11.p2" name="article1.body1.sec4.sec11.p2"></a><p>The promoters are generally upstream of the genes that they are associated with. However, some regions stretch over the first exon into intronic regions. Therefore, some promoters have exons in their sequence. There were around a dozen overlapping promoters between the two databases. The TF binding sites for these promoters were merged onto the longer 2K ORegAnno sequences. The data set also includes 250 upstream, non-coding sequences that can be used for generating background models and statistics.</p>
<a id="article1.body1.sec4.sec11.p3" name="article1.body1.sec4.sec11.p3"></a><p>The test set used in our simulations consists of 47 promoter sequences, each having a varying number of annotated binding sites (<em>positive sequences</em>), and 250 promoter sequences that have no reported binding sites (<em>negative sequences</em>). Additional data sources (evolutionary conservation <a href="#pone.0001820-Siepel1">[72]</a>, regulatory potential <a href="#pone.0001820-Taylor1">[73]</a>, nucleosome location <a href="#pone.0001820-Segal1">[74]</a>, and CpG islands) are available for all the positive sequences. Evolutionary conservation, regulatory potential and CpG island data are downloaded from the UCSC genome browser <a href="#pone.0001820-Kent1">[33]</a> and nucleosome location predictions are obtained using the method and software from <a href="#pone.0001820-Segal1">[74]</a>. After removing unknown/unmatched TFs and TFs for which we do not have prior binding specificities in TRANSFAC, we are left with 70 unique TF-promoter pairs in the <em>positive set</em>, which include 23 unique TFs. As for the <em>negative set</em>, we use all possible pairs of the 23 unique TFs and the 47 promoter sequences that are not in the positive set. We further filter the negative set and ignore those TF-promoter pairs where the TF belongs to the same family as a TF in the positive set (for the same promoter) since their binding specificities are defined by largely overlapping TRANSFAC motif models. For example, Mef2c has an annotated binding site in the Des promoter. Consequently, all other “Mef-family”-Des pairs are ignored from the negative set. This filtering reduces the negative set from all possible 23×47−70 = 1011 pairs to 952. Because all the annotated binding sites for each TF are on the same strand of DNA, we first use only the strand containing the annotated sites. This also doubles the size of the negative set. Because the direction of transcription can be unknown in general, we also extend the analysis to cover both strands. In some simulations we use only 4 randomly chosen TF-promoter pairs for each promoter from the negative set, resulting in 47×4 = 188 unique TF-promoter pairs in the <em>reduced negative set</em>.</p>
<a id="article1.body1.sec4.sec11.p4" name="article1.body1.sec4.sec11.p4"></a><p>For all the simulations, we use (scaled) motif models from TRANSFAC. Parameters of the Markovian background models (model orders 0,1,…, 4 are tested) are estimated from the 250 negative sequences (both strands).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001820"></div><a name="pone.0001820.s001" id="pone.0001820.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001820.s001">Text S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001820.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001820.s002" id="pone.0001820.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001820.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">ROC curves for the likelihood-based probabilistic method (red), traditional scanning (blue), and a probabilistic scanning-based method that outputs a probability of binding (green) for the case where promoter sequence lengths have not been made equal. Background model order is (a) d = 0 and (b) d = 1.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001820.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.49 MB TIF)</p>
<a name="pone.0001820.s003" id="pone.0001820.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001820.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">ROC curves for the likelihood-based method (blue) when combined with a single additional information source: regulatory potential (red), and evolutionary conservation (green). Solid graphs (resp. dashed graphs) correspond to the optimized parameters (resp. results obtained with stratified cross-validation).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001820.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.33 MB TIF)</p>
<a name="pone.0001820.s004" id="pone.0001820.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001820.s004">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">ROC curves for the traditional scanning (green), traditional scanning combined with thresholded conservation information (blue), probabilistic method combined with conservation information (red), and probabilistic method (cyan) for the case where promoter sequence lengths have not been made equal.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001820.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.34 MB TIF)</p>
<a name="pone.0001820.s005" id="pone.0001820.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001820.s005">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">ROC curves for the likelihood-based method (blue) when combined with a single additional information source: evolutionary conservation (green) and regulatory potential (red). Promoter sequences that are used to train the regulatory potential method and that also overlap with our test set have been removed.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001820.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.33 MB TIF)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AR HL IS. Performed the experiments: HL. Analyzed the data: AR HL. Contributed reagents/materials/analysis tools: HL. Wrote the paper: HL. Other: Compiled and created the test set: AR. Supervised the study: IS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001820-Davidson1" id="pone.0001820-Davidson1"></a>Davidson EH (2001) Genomic Regulatory Systems: Development and Evolution. Academic Press, 1st edition.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001820-MacIsaac1" id="pone.0001820-MacIsaac1"></a>MacIsaac KD, Fraenkel E (2006) Practical strategies for discovering regulatory DNA sequence motifs. PLoS Computational Biology  2: e36.  <ul class="find" data-citedArticleID="1076594" data-doi="10.1371/journal.pcbi.0020036"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Practical+strategies+for+discovering+regulatory+DNA+sequence+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Practical+strategies+for+discovering+regulatory+DNA+sequence+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001820-Tompa1" id="pone.0001820-Tompa1"></a>Tompa M, Li N, Bailey TL, Church GM, De Moor B, et al.  (2005) Assessing computational tools for the discovery of transcription factor binding sites. Nature Biotechnology  23: 137–144.  <ul class="find" data-citedArticleID="1076662" data-doi="10.1038/nbt1053"><li><a href="http://dx.doi.org/10.1038/nbt1053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+computational+tools+for+the+discovery+of+transcription+factor+binding+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+computational+tools+for+the+discovery+of+transcription+factor+binding+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001820-Hughes1" id="pone.0001820-Hughes1"></a>Hughes JD, Estep PW, Tavazoie S, Church GM (2000) Computational identification of Cis-regulatory elements associated with groups of functionally related genes in <em>Saccharomyces cerevisiae</em>. Journal of Molecular Biology  296: 1205–1214.  <ul class="find" data-citedArticleID="1076566" data-doi="10.1006/jmbi.2000.3519"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3519" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+identification+of+Cis-regulatory+elements+associated+with+groups+of+functionally+related+genes+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+identification+of+Cis-regulatory+elements+associated+with+groups+of+functionally+related+genes+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001820-Kellis1" id="pone.0001820-Kellis1"></a>Kellis M, Patterson N, Endrizzi M, Birren B, Lander ES (2003) Sequencing and comparison of yeast species to identify genes and regulatory elements. Nature  423: 241–254.  <ul class="find" data-citedArticleID="1076576" data-doi="10.1038/nature01644"><li><a href="http://dx.doi.org/10.1038/nature01644" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001820-Siddharthan1" id="pone.0001820-Siddharthan1"></a>Siddharthan R, Siggia ED, van Nimwegen E (2005) PhyloGibbs: A Gibbs sampling motif finder that incorporates phylogeny. PLoS Computational Biology  1: e67.  <ul class="find" data-citedArticleID="1076636" data-doi="10.1371/journal.pcbi.0010067.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0010067.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PhyloGibbs%3A+A+Gibbs+sampling+motif+finder+that+incorporates+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PhyloGibbs%3A+A+Gibbs+sampling+motif+finder+that+incorporates+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001820-Liu1" id="pone.0001820-Liu1"></a>Liu XS, Brutlag DL, Liu JS (2002) An algorithm for finding protein-DNA binding sites with applications to chromatin-immunoprecipitation microarray experiments. Nature Biotechnology  20: 835–839.  <ul class="find" data-citedArticleID="1076590" data-doi="10.1038/nbt717"><li><a href="http://dx.doi.org/10.1038/nbt717" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+algorithm+for+finding+protein-DNA+binding+sites+with+applications+to+chromatin-immunoprecipitation+microarray+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+algorithm+for+finding+protein-DNA+binding+sites+with+applications+to+chromatin-immunoprecipitation+microarray+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001820-Harbison1" id="pone.0001820-Harbison1"></a>Harbison CT, Gordon DB, Lee TI, Rinaldi NJ, Macisaac KD, et al.  (2004) Transcriptional regulatory code of a eukaryotic genome. Nature  431: 99–104.  <ul class="find" data-citedArticleID="1076554" data-doi="10.1038/nature02800"><li><a href="http://dx.doi.org/10.1038/nature02800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulatory+code+of+a+eukaryotic+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulatory+code+of+a+eukaryotic+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001820-Qi1" id="pone.0001820-Qi1"></a>Qi Y, Rolfe A, MacIsaac KD, Gerber GK, Pokholok D, et al.  (2006) High-resolution computational models of genome binding events. Nature Biotechnology  24: 963–970.  <ul class="find" data-citedArticleID="1076614" data-doi="10.1038/nbt1233"><li><a href="http://dx.doi.org/10.1038/nbt1233" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-resolution+computational+models+of+genome+binding+events." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-resolution+computational+models+of+genome+binding+events.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001820-Narlikar1" id="pone.0001820-Narlikar1"></a>Narlikar L, Gordân R, Ohler U, Hartemink AJ (2006) Informative priors based on transcription factor structural class improve <em>de novo</em> motif discovery. Bioinformatics  22: e384–e392.  <ul class="find" data-citedArticleID="1076606" data-doi="10.1093/bioinformatics/btl251"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl251" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Informative+priors+based+on+transcription+factor+structural+class+improve+de+novo+motif+discovery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Informative+priors+based+on+transcription+factor+structural+class+improve+de+novo+motif+discovery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001820-Narlikar2" id="pone.0001820-Narlikar2"></a>Narlikar L, Gordân R, Hartemink A (2007) Nucleosome occupancy information improves <em>de novo</em> motif discovery. Research in Computational Molecular Biology (RECOMB). Springer. pp. 107–121.  <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0001820-Narlikar3" id="pone.0001820-Narlikar3"></a>Narlikar L, Gordân R, Hartemink AJ (In press) A nucleosome-guided map of transcription factor binding sites in yeast. PLoS Computational Biology.   <ul class="find" data-citedArticleID="1076610" data-doi="10.1371/journal.pcbi.0030215.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0030215.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+nucleosome-guided+map+of+transcription+factor+binding+sites+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+nucleosome-guided+map+of+transcription+factor+binding+sites+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001820-Wingender1" id="pone.0001820-Wingender1"></a>Wingender E, Chen X, Hehl R, Karas H, Liebich I, et al.  (2000) TRANSFAC: an integrated system for gene expression regulation. Nucleic Acids Research  28: 316–319.  <ul class="find" data-citedArticleID="1076670" data-doi="10.1093/nar/28.1.316"><li><a href="http://dx.doi.org/10.1093/nar/28.1.316" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TRANSFAC%3A+an+integrated+system+for+gene+expression+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TRANSFAC%3A+an+integrated+system+for+gene+expression+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001820-Sandelin1" id="pone.0001820-Sandelin1"></a>Sandelin A, Alkema W, Engström P, Wasserman WW, Lenhard B (2004) JASPAR: an open-access database for eukaryotic transcription factor binding profiles. Nucleic Acids Research  32: D91–D94.  <ul class="find" data-citedArticleID="1076628" data-doi="10.1093/nar/gkh012"><li><a href="http://dx.doi.org/10.1093/nar/gkh012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=JASPAR%3A+an+open-access+database+for+eukaryotic+transcription+factor+binding+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22JASPAR%3A+an+open-access+database+for+eukaryotic+transcription+factor+binding+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001820-Mukherjee1" id="pone.0001820-Mukherjee1"></a>Mukherjee S, Berger MF, Jona G, Wang XS, Muzzey D, et al.  (2004) Rapid analysis of the DNA-binding specificities of transcription factors with DNA microarrays. Nature Genetics  36: 1331–1339.  <ul class="find" data-citedArticleID="1076602" data-doi="10.1038/ng1473"><li><a href="http://dx.doi.org/10.1038/ng1473" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+analysis+of+the+DNA-binding+specificities+of+transcription+factors+with+DNA+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+analysis+of+the+DNA-binding+specificities+of+transcription+factors+with+DNA+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001820-Berger1" id="pone.0001820-Berger1"></a>Berger MF, Philippakis AA, Qureshi AM, He FS, Estep PW III, et al.  (2006) Compact, universal DNA microarrays to comprehensively determine transcription-factor binding site specificities. Nature Biotechnology  24: 1429–1435.  <ul class="find" data-citedArticleID="1076514" data-doi="10.1038/nbt1246"><li><a href="http://dx.doi.org/10.1038/nbt1246" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compact%2C+universal+DNA+microarrays+to+comprehensively+determine+transcription-factor+binding+site+specificities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compact%2C+universal+DNA+microarrays+to+comprehensively+determine+transcription-factor+binding+site+specificities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001820-Warren1" id="pone.0001820-Warren1"></a>Warren CL, Kratochvil NCS, Hauschild KE, Foister S, Brezinski ML, et al.  (2007) Defining the sequence-recognition profile of DNA-binding molecules. Proceedings of the National Academy of Sciences of the USA  103: 867–872.  <ul class="find" data-citedArticleID="1076664" data-doi="10.1073/pnas.0509843102"><li><a href="http://dx.doi.org/10.1073/pnas.0509843102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Defining+the+sequence-recognition+profile+of+DNA-binding+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Defining+the+sequence-recognition+profile+of+DNA-binding+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001820-Maerkl1" id="pone.0001820-Maerkl1"></a>Maerkl SJ, Quake SR (2007) A systems approach to measuring the binding energy landscapes of transcription factors. Science  315: 233–237.  <ul class="find" data-citedArticleID="1076596" data-doi="10.1126/science.1131007"><li><a href="http://dx.doi.org/10.1126/science.1131007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+systems+approach+to+measuring+the+binding+energy+landscapes+of+transcription+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+systems+approach+to+measuring+the+binding+energy+landscapes+of+transcription+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001820-Zhang1" id="pone.0001820-Zhang1"></a>Zhang L, Kasif S, Cantor CR (2007) Quantifying DNA-protein binding specificities by using oligonucleotide mass tags and mass spectroscopy. Proceedings of the National Academy of Sciences of the USA  104: 3061–3066.  <ul class="find" data-citedArticleID="1076676" data-doi="10.1073/pnas.0611075104"><li><a href="http://dx.doi.org/10.1073/pnas.0611075104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantifying+DNA-protein+binding+specificities+by+using+oligonucleotide+mass+tags+and+mass+spectroscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantifying+DNA-protein+binding+specificities+by+using+oligonucleotide+mass+tags+and+mass+spectroscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001820-Eisenstein1" id="pone.0001820-Eisenstein1"></a>Eisenstein M (2005) A plan for when the ChIPs are down. Nature Methods  2: 5.  <ul class="find" data-citedArticleID="1076540" data-doi="10.1038/nmeth0105-5"><li><a href="http://dx.doi.org/10.1038/nmeth0105-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+plan+for+when+the+ChIPs+are+down." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+plan+for+when+the+ChIPs+are+down.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001820-Staden1" id="pone.0001820-Staden1"></a>Staden R (1984) Computer methods to locate signals in nucleic acid sequences. Nucleic Acids Research  12: 505–519.  <ul class="find" data-citedArticleID="1076646" data-doi="10.1093/nar/12.1part2.505"><li><a href="http://dx.doi.org/10.1093/nar/12.1part2.505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computer+methods+to+locate+signals+in+nucleic+acid+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computer+methods+to+locate+signals+in+nucleic+acid+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001820-Quandt1" id="pone.0001820-Quandt1"></a>Quandt K, Frechl K, Karas H, Wingender E, Werner T (1995) MatInd and MatInspector: new fast and versatile tools for detection of consensus matches in nucleotide sequence data. Nucleic Acids Research  23: 4878–4884.  <ul class="find" data-citedArticleID="1076616" data-doi="10.1093/nar/23.23.4878"><li><a href="http://dx.doi.org/10.1093/nar/23.23.4878" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MatInd+and+MatInspector%3A+new+fast+and+versatile+tools+for+detection+of+consensus+matches+in+nucleotide+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MatInd+and+MatInspector%3A+new+fast+and+versatile+tools+for+detection+of+consensus+matches+in+nucleotide+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001820-Claverie1" id="pone.0001820-Claverie1"></a>Claverie JM, Audic S (1996) The statistical significance of nucleotide position-weight matrix matches. Bioinformatics  12: 431–439.  <ul class="find" data-citedArticleID="1076534" data-doi="10.1093/bioinformatics/12.5.431"><li><a href="http://dx.doi.org/10.1093/bioinformatics/12.5.431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+statistical+significance+of+nucleotide+position-weight+matrix+matches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+statistical+significance+of+nucleotide+position-weight+matrix+matches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001820-Wu1" id="pone.0001820-Wu1"></a>Wu TD, Nevill-Manning CG, Brutlag DL (2000) Fast probabilistic analysis of sequence function using scoring matrices. Bioinformatics  16: 233–244.  <ul class="find" data-citedArticleID="1076672" data-doi="10.1093/bioinformatics/16.3.233"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.3.233" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fast+probabilistic+analysis+of+sequence+function+using+scoring+matrices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fast+probabilistic+analysis+of+sequence+function+using+scoring+matrices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001820-Huang1" id="pone.0001820-Huang1"></a>Huang H, Kao MCJ, Zhou X, Liu JS, Wong WH (2004) Determination of local statistical significance of patterns in Markov sequences with application to promoter element identification. Journal of Computational Biology  11: 1–14.  <ul class="find" data-citedArticleID="1076564" data-doi="10.1089/106652704773416858"><li><a href="http://dx.doi.org/10.1089/106652704773416858" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determination+of+local+statistical+significance+of+patterns+in+Markov+sequences+with+application+to+promoter+element+identification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determination+of+local+statistical+significance+of+patterns+in+Markov+sequences+with+application+to+promoter+element+identification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001820-Hertzberg1" id="pone.0001820-Hertzberg1"></a>Hertzberg L, Zuk O, Getz G, Domany E (2005) Finding motifs in promoter regions. Journal of Computational Biology  12: 314–330.  <ul class="find" data-citedArticleID="1076558" data-doi="10.1089/cmb.2005.12.314"><li><a href="http://dx.doi.org/10.1089/cmb.2005.12.314" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Finding+motifs+in+promoter+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Finding+motifs+in+promoter+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001820-Barash1" id="pone.0001820-Barash1"></a>Barash Y, Elidan G, Kaplan T, Friedman N (2005) CIS: compound importance sampling method for protein-DNA binding site <em>p</em>-value estimation. Bioinformatics  21: 596–600.  <ul class="find" data-citedArticleID="1076508" data-doi="10.1093/bioinformatics/bti041"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CIS%3A+compound+importance+sampling+method+for+protein-DNA+binding+site+p-value+estimation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CIS%3A+compound+importance+sampling+method+for+protein-DNA+binding+site+p-value+estimation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001820-Baji1" id="pone.0001820-Baji1"></a>Bajić IV (2006) Detection-theoretic analysis of MatInspector. IEEE Transactions on Signal Processing  54: 2388–2393.  <ul class="find" data-citedArticleID="1076506" data-doi="10.1109/tsp.2006.873739"><li><a href="http://dx.doi.org/10.1109/tsp.2006.873739" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection-theoretic+analysis+of+MatInspector." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection-theoretic+analysis+of+MatInspector.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001820-Wasserman1" id="pone.0001820-Wasserman1"></a>Wasserman WW, Sandelin A (2004) Applied bioinformatics for the identification of regulatory elements. Nature Reviews Genetics  5: 276–287.  <ul class="find" data-citedArticleID="1076666" data-doi="10.1038/nrg1315"><li><a href="http://dx.doi.org/10.1038/nrg1315" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Applied+bioinformatics+for+the+identification+of+regulatory+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Applied+bioinformatics+for+the+identification+of+regulatory+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001820-Bulyk1" id="pone.0001820-Bulyk1"></a>Bulyk ML, Johnson PLF, Church GM (2002) Nucleotides of transcription factor binding sites exert interdependent effects on the binding affinities of transcription factors. Nucleic Acids Research  30: 1255–1261.  <ul class="find" data-citedArticleID="1076526" data-doi="10.1093/nar/30.5.1255"><li><a href="http://dx.doi.org/10.1093/nar/30.5.1255" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleotides+of+transcription+factor+binding+sites+exert+interdependent+effects+on+the+binding+affinities+of+transcription+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleotides+of+transcription+factor+binding+sites+exert+interdependent+effects+on+the+binding+affinities+of+transcription+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001820-Zhou1" id="pone.0001820-Zhou1"></a>Zhou Q, Liu JS (2004) Modeling within-motif dependence for transcription factor binding site predictions. Bioinformatics  20: 909–916.  <ul class="find" data-citedArticleID="1076678" data-doi="10.1093/bioinformatics/bth006"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+within-motif+dependence+for+transcription+factor+binding+site+predictions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+within-motif+dependence+for+transcription+factor+binding+site+predictions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001820-Barash2" id="pone.0001820-Barash2"></a>Barash Y, Elidan G, Friedman N, Kaplan T (2003) Modeling dependencies in protein-DNA binding sites. Proceedings of the Seventh Annual International Conference on Computational Biology (RECOMB 2003). ACM.   <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0001820-Kent1" id="pone.0001820-Kent1"></a>Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, et al.  (2002) The human genome browser at UCSC. Genome Research  12: 996–1006.  <ul class="find" data-citedArticleID="1076578" data-doi="10.1101/gr.229102"><li><a href="http://dx.doi.org/10.1101/gr.229102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+genome+browser+at+UCSC." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+genome+browser+at+UCSC.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001820-Montgomery1" id="pone.0001820-Montgomery1"></a>Montgomery SB, Griffith OL, Sleumer MC, Bergman CM, Bilenky M, et al.  (2006) ORegAnno: An open access database and curation system for literature-derived promoters, transcription factor binding sites and regulatory variation. Bioinformatics  22: 637–640.  <ul class="find" data-citedArticleID="1076600" data-doi="10.1093/bioinformatics/btk027"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btk027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ORegAnno%3A+An+open+access+database+and+curation+system+for+literature-derived+promoters%2C+transcription+factor+binding+sites+and+regulatory+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ORegAnno%3A+An+open+access+database+and+curation+system+for+literature-derived+promoters%2C+transcription+factor+binding+sites+and+regulatory+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001820-Blanco1" id="pone.0001820-Blanco1"></a>Blanco E, Farré D, Albà MM, Messeguer X, Guigó R (2006) ABS: a database of Annotated regulatory Binding Sites from orthologous promoters. Nucleic Acids Research  34: D63–D67.  <ul class="find" data-citedArticleID="1076520" data-doi="10.1093/nar/gkj116"><li><a href="http://dx.doi.org/10.1093/nar/gkj116" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ABS%3A+a+database+of+Annotated+regulatory+Binding+Sites+from+orthologous+promoters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ABS%3A+a+database+of+Annotated+regulatory+Binding+Sites+from+orthologous+promoters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001820-Frith1" id="pone.0001820-Frith1"></a>Frith MC, Hansen U, Weng Z (2001) Detection of cis-element clusters in higher eukaryotic DNA. Bioinformatics  17: 878–889.  <ul class="find" data-citedArticleID="1076546" data-doi="10.1093/bioinformatics/17.10.878"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.10.878" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+cis-element+clusters+in+higher+eukaryotic+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+cis-element+clusters+in+higher+eukaryotic+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001820-Rajewsky1" id="pone.0001820-Rajewsky1"></a>Rajewsky N, Vergassola M, Gaul U, Siggia ED (2002) Computational detection of genomic cis-regulatory modules applied to body patterning in the early <em>Drosophila</em> embryo. BMC Bioinformatics  3: 30.  <ul class="find" data-citedArticleID="1076618"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Computational+detection+of+genomic+cis-regulatory+modules+applied+to+body+patterning+in+the+early+Drosophila+embryo.&amp;auth=&amp;atitle=Computational+detection+of+genomic+cis-regulatory+modules+applied+to+body+patterning+in+the+early+Drosophila+embryo." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+detection+of+genomic+cis-regulatory+modules+applied+to+body+patterning+in+the+early+Drosophila+embryo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+detection+of+genomic+cis-regulatory+modules+applied+to+body+patterning+in+the+early+Drosophila+embryo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001820-Sinha1" id="pone.0001820-Sinha1"></a>Sinha S, van Nimwegen E, Siggia ED (2003) A probabilistic method to detect regulatory modules. Bioinformatics  19: i292–i301.  <ul class="find" data-citedArticleID="1076640" data-doi="10.1093/bioinformatics/btg1040"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg1040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+probabilistic+method+to+detect+regulatory+modules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+probabilistic+method+to+detect+regulatory+modules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001820-Lawrence1" id="pone.0001820-Lawrence1"></a>Lawrence CE, Altschul SF, Boguski MS, Liu JS, Neuwald AF, et al.  (1993) Detecting subtle sequence signals: A Gibbs sampling strategy for multiple alignment. Science  262: 208–214.  <ul class="find" data-citedArticleID="1076580" data-doi="10.1126/science.8211139"><li><a href="http://dx.doi.org/10.1126/science.8211139" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+subtle+sequence+signals%3A+A+Gibbs+sampling+strategy+for+multiple+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+subtle+sequence+signals%3A+A+Gibbs+sampling+strategy+for+multiple+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001820-Bailey1" id="pone.0001820-Bailey1"></a>Bailey TL, Elkan C (1994) Fitting a mixture model by expectation maximization to discover motifs in biopolymers. Proceedings of the Second International Conference on Intelligent Systems for Molecular Biology. volume 2. pp. 28–36.  <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0001820-Bailey2" id="pone.0001820-Bailey2"></a>Bailey TL, Elkan CP (1995) The value of prior knowledge in discovering motifs with MEME. Proceedings of the Third International Conference on Intelligent Systems for Molecular Biology. AAAI Press, volume 3. pp. 21–29.  <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0001820-Thijs1" id="pone.0001820-Thijs1"></a>Thijs G, Marchal K, Lescot M, Rombauts S, Moor BD, et al.  (2002) A Gibbs sampling method to detect overrepresented motifs in the upstream regions of coexpressed genes. Journal of Computational Biology  9: 447–464.  <ul class="find" data-citedArticleID="1076656" data-doi="10.1089/10665270252935566"><li><a href="http://dx.doi.org/10.1089/10665270252935566" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Gibbs+sampling+method+to+detect+overrepresented+motifs+in+the+upstream+regions+of+coexpressed+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Gibbs+sampling+method+to+detect+overrepresented+motifs+in+the+upstream+regions+of+coexpressed+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001820-Reiss1" id="pone.0001820-Reiss1"></a>Reiss DJ, Schwikowski B (2004) Predicting protein-peptide interactions via a network-based motif sampler. Bioinformatics  20: i274–i282.  <ul class="find" data-citedArticleID="1076622" data-doi="10.1093/bioinformatics/bth922"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth922" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+protein-peptide+interactions+via+a+network-based+motif+sampler." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+protein-peptide+interactions+via+a+network-based+motif+sampler.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001820-Sinha2" id="pone.0001820-Sinha2"></a>Sinha S (2006) On counting position weight matrix matches in a sequence, with application to discriminative motif finding. Bioinformatics  22: e454–e463.  <ul class="find" data-citedArticleID="1076642" data-doi="10.1093/bioinformatics/btl227"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl227" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+counting+position+weight+matrix+matches+in+a+sequence%2C+with+application+to+discriminative+motif+finding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+counting+position+weight+matrix+matches+in+a+sequence%2C+with+application+to+discriminative+motif+finding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001820-Lehrach1" id="pone.0001820-Lehrach1"></a>Lehrach WP, Husmeier D, Williams CKI (2006) A regularized discriminative model for the prediction of protein-peptide interactions. Bioinformatics  22: 532–540.  <ul class="find" data-citedArticleID="1076584" data-doi="10.1093/bioinformatics/bti804"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti804" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+regularized+discriminative+model+for+the+prediction+of+protein-peptide+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+regularized+discriminative+model+for+the+prediction+of+protein-peptide+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001820-Sandelin2" id="pone.0001820-Sandelin2"></a>Sandelin A, Wasserman WW, Lenhard B (2004) ConSite: web-based prediction of regulatory elements using cross-species comparison. Nucleic Acids Research  32: W249–W252.  <ul class="find" data-citedArticleID="1076630" data-doi="10.1093/nar/gkh372"><li><a href="http://dx.doi.org/10.1093/nar/gkh372" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ConSite%3A+web-based+prediction+of+regulatory+elements+using+cross-species+comparison." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ConSite%3A+web-based+prediction+of+regulatory+elements+using+cross-species+comparison.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001820-Marinescu1" id="pone.0001820-Marinescu1"></a>Marinescu VD, Kohane IS, Riva A (2005) MAPPER: a search engine for the computational identification of putative transcription factor binding sites in multiple genomes. BMC Bioinformatics  6: 79.  <ul class="find" data-citedArticleID="1076598"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=MAPPER%3A+a+search+engine+for+the+computational+identification+of+putative+transcription+factor+binding+sites+in+multiple+genomes.&amp;auth=&amp;atitle=MAPPER%3A+a+search+engine+for+the+computational+identification+of+putative+transcription+factor+binding+sites+in+multiple+genomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAPPER%3A+a+search+engine+for+the+computational+identification+of+putative+transcription+factor+binding+sites+in+multiple+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAPPER%3A+a+search+engine+for+the+computational+identification+of+putative+transcription+factor+binding+sites+in+multiple+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001820-GuhaThakurta1" id="pone.0001820-GuhaThakurta1"></a>GuhaThakurta D (2006) Computational identification of transcriptional regulatory elements in DNA sequence. Nucleic Acids Research  34: 3585–3598.  <ul class="find" data-citedArticleID="1076550" data-doi="10.1093/nar/gkl372"><li><a href="http://dx.doi.org/10.1093/nar/gkl372" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+identification+of+transcriptional+regulatory+elements+in+DNA+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+identification+of+transcriptional+regulatory+elements+in+DNA+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001820-Beyer1" id="pone.0001820-Beyer1"></a>Beyer A, Workman C, Hollunder J, Radke D, Möller U, et al.  (2006) Integrated assessment and prediction of transcription factor binding. PLoS Computatioanl Biology  2: e70.  <ul class="find" data-citedArticleID="1076518" data-doi="10.1371/journal.pcbi.0020070.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020070.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrated+assessment+and+prediction+of+transcription+factor+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrated+assessment+and+prediction+of+transcription+factor+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001820-AmbesiImpiombato1" id="pone.0001820-AmbesiImpiombato1"></a>Ambesi-Impiombato A, Bansal M, Liò P, di Bernardo D (2006) Computational framework for the prediction of transcription factor binding sites by multiple data integration. BMC Neuroscience  7: Suppl 1S8.  <ul class="find" data-citedArticleID="1076500" data-doi="10.1186/1471-2202-7-s1-s8"><li><a href="http://dx.doi.org/10.1186/1471-2202-7-s1-s8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+framework+for+the+prediction+of+transcription+factor+binding+sites+by+multiple+data+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+framework+for+the+prediction+of+transcription+factor+binding+sites+by+multiple+data+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001820-Beer1" id="pone.0001820-Beer1"></a>Beer MA, Tavazoie S (2004) Predicting gene expression from sequence. Cell  117: 185–198.  <ul class="find" data-citedArticleID="1076512" data-doi="10.1016/s0092-8674(04)00304-6"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00304-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+gene+expression+from+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+gene+expression+from+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001820-Jensen1" id="pone.0001820-Jensen1"></a>Jensen ST, Shen L, Liu JS (2005) Combining phylogenetic motif discovery and motif clustering to predict co-regulated genes. Bioinformatics  21: 3832–3839.  <ul class="find" data-citedArticleID="1076572" data-doi="10.1093/bioinformatics/bti628"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti628" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combining+phylogenetic+motif+discovery+and+motif+clustering+to+predict+co-regulated+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combining+phylogenetic+motif+discovery+and+motif+clustering+to+predict+co-regulated+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001820-Gordon1" id="pone.0001820-Gordon1"></a>Gordon DB, Nekludova L, McCallum S, Fraenkel E (2005) TAMO: a flexible, object-oriented framework for analyzing transcriptional regulation using DNA-sequence motifs. Bioinformatics  21: 3164–3165.  <ul class="find" data-citedArticleID="1076548" data-doi="10.1093/bioinformatics/bti481"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti481" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TAMO%3A+a+flexible%2C+object-oriented+framework+for+analyzing+transcriptional+regulation+using+DNA-sequence+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TAMO%3A+a+flexible%2C+object-oriented+framework+for+analyzing+transcriptional+regulation+using+DNA-sequence+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001820-Holloway1" id="pone.0001820-Holloway1"></a>Holloway DT, Kon M, DeLisi C (2005) Integrating genomic data to predict transcription factor binding. Genome Informatics  16: 83–94.  <ul class="find" data-citedArticleID="1076560"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Integrating+genomic+data+to+predict+transcription+factor+binding.&amp;auth=&amp;atitle=Integrating+genomic+data+to+predict+transcription+factor+binding." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrating+genomic+data+to+predict+transcription+factor+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrating+genomic+data+to+predict+transcription+factor+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001820-Holloway2" id="pone.0001820-Holloway2"></a>Holloway DT, Kon M, DeLisi C (2006) Machine learning methods for transcription data integration. IBM Journal of Research and Development  6: 631–643.  <ul class="find" data-citedArticleID="1076562" data-doi="10.1147/rd.506.0631"><li><a href="http://dx.doi.org/10.1147/rd.506.0631" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Machine+learning+methods+for+transcription+data+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Machine+learning+methods+for+transcription+data+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001820-Murphy1" id="pone.0001820-Murphy1"></a>Murphy KP, Mian S (1999) Modelling gene expression data using dynamic Bayesian networks. Berkeley: Technical report, University of California.   <ul class="find-nolinks"></ul></li><li><span class="label">57.
              </span><a name="pone.0001820-Husmeier1" id="pone.0001820-Husmeier1"></a>Husmeier D (2003) Sensitivity and specificity of inferring genetic regulatory interactions from microarray experiments with dynamic Bayesian networks. Bioinformatics  19: 2271–2282.  <ul class="find" data-citedArticleID="1076568" data-doi="10.1093/bioinformatics/btg313"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg313" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sensitivity+and+specificity+of+inferring+genetic+regulatory+interactions+from+microarray+experiments+with+dynamic+Bayesian+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sensitivity+and+specificity+of+inferring+genetic+regulatory+interactions+from+microarray+experiments+with+dynamic+Bayesian+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001820-Friedman1" id="pone.0001820-Friedman1"></a>Friedman N (2004) Inferring cellular networks using probabilistic graphical models. Science  303: 799–805.  <ul class="find" data-citedArticleID="1076544" data-doi="10.1126/science.1094068"><li><a href="http://dx.doi.org/10.1126/science.1094068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+cellular+networks+using+probabilistic+graphical+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+cellular+networks+using+probabilistic+graphical+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001820-Chen1" id="pone.0001820-Chen1"></a>Chen T, He HL, Church GM (1999) Modeling gene expression with differential equations. Proceedings of Pacific Symposium on Biocomputing (PSB 99). pp. 29–40.  World Scientific, Singapore, volume 4.  <ul class="find-nolinks"></ul></li><li><span class="label">60.
              </span><a name="pone.0001820-Bower1" id="pone.0001820-Bower1"></a> Bower JM, Bolouri H, editors.  (2001) Computational Modeling of Genetic and Biochemical Networks. The MIT Press, 1st edition edition.   <ul class="find-nolinks"></ul></li><li><span class="label">61.
              </span><a name="pone.0001820-Wilkinson1" id="pone.0001820-Wilkinson1"></a>Wilkinson DJ (2006) Stochastic Modelling for Systems Biology. Chapman &amp; Hall/CRC, 1st edition.   <ul class="find-nolinks"></ul></li><li><span class="label">62.
              </span><a name="pone.0001820-Bernard1" id="pone.0001820-Bernard1"></a>Bernard A, Hartemink A (2005) Informative structure priors: joint learning of dynamic regulatory networks from multiple types of data. Proceedings of Pacific Symposium on Biocomputing (PSB 05). pp. 459–470.  World Scientific, Singapore, volume 10.  <ul class="find-nolinks"></ul></li><li><span class="label">63.
              </span><a name="pone.0001820-Chen2" id="pone.0001820-Chen2"></a>Chen G, Jensen ST, Stoeckert CJ Jr (2007) Clustering of genes into regulons using integrated modeling-COGRIM. Genome Biology  8:   <ul class="find" data-citedArticleID="1076532"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Clustering+of+genes+into+regulons+using+integrated+modeling-COGRIM.&amp;auth=&amp;atitle=Clustering+of+genes+into+regulons+using+integrated+modeling-COGRIM." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clustering+of+genes+into+regulons+using+integrated+modeling-COGRIM." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clustering+of+genes+into+regulons+using+integrated+modeling-COGRIM.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001820-Pan1" id="pone.0001820-Pan1"></a>Pan W (2006) Incorporating gene functions as priors in model-based clustering of microarray gene expression data. Bioinformatics  22: 795–801.  <ul class="find" data-citedArticleID="1076612" data-doi="10.1093/bioinformatics/btl011"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Incorporating+gene+functions+as+priors+in+model-based+clustering+of+microarray+gene+expression+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Incorporating+gene+functions+as+priors+in+model-based+clustering+of+microarray+gene+expression+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001820-Lee1" id="pone.0001820-Lee1"></a>Lee TI, Rinaldi NJ, Robert F, Odom DT, Bar-Joseph Z, et al.  (2002) Transcriptional regulatory networks in <em>Saccharomyces cerevisiae</em>. Science  298: 799–804.  <ul class="find" data-citedArticleID="1076582" data-doi="10.1126/science.1075090"><li><a href="http://dx.doi.org/10.1126/science.1075090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulatory+networks+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulatory+networks+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001820-Stormo1" id="pone.0001820-Stormo1"></a>Stormo GD, Schneider TD, Gold L, Ehrenfeucht A (1982) Use of the ‘Perceptron’ algorithm to distinguish translational initiation sites in <em>E. coli</em>. Nucleic Acids Research  10: 2997–3011.  <ul class="find" data-citedArticleID="1076650" data-doi="10.1093/nar/10.9.2997"><li><a href="http://dx.doi.org/10.1093/nar/10.9.2997" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+the+%E2%80%98Perceptron%E2%80%99+algorithm+to+distinguish+translational+initiation+sites+in+E.+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+the+%E2%80%98Perceptron%E2%80%99+algorithm+to+distinguish+translational+initiation+sites+in+E.+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001820-Thijs2" id="pone.0001820-Thijs2"></a>Thijs G, Lescot M, Marchal K, Rombauts S, Moor BD, et al.  (2001) A higher-order background model improves the detection of promoter regulatory elements by Gibbs sampling. Bioinformatics  17: 1113–1122.  <ul class="find" data-citedArticleID="1076658" data-doi="10.1093/bioinformatics/17.12.1113"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.12.1113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+higher-order+background+model+improves+the+detection+of+promoter+regulatory+elements+by+Gibbs+sampling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+higher-order+background+model+improves+the+detection+of+promoter+regulatory+elements+by+Gibbs+sampling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001820-Fawcett1" id="pone.0001820-Fawcett1"></a>Fawcett T (2006) An introduction to ROC analysis. Pattern Recognition Letters  27: 861–874.  <ul class="find" data-citedArticleID="1076542" data-doi="10.1016/j.patrec.2005.10.010"><li><a href="http://dx.doi.org/10.1016/j.patrec.2005.10.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+introduction+to+ROC+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+introduction+to+ROC+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001820-Liu2" id="pone.0001820-Liu2"></a>Liu X, Brutlag DL, S LJ (2001) Bioprospector: discovering conserved DNA motifs in upstream regulatory regions of co-expressed genes. Proceedings of Pacific Symposium on Biocomputing (PSB 01). pp. 127–138.  World Scientific, Singapore, volume 6.  <ul class="find-nolinks"></ul></li><li><span class="label">70.
              </span><a name="pone.0001820-Steck1" id="pone.0001820-Steck1"></a>Steck H, Jaakkola TS (2002) On the Dirichlet prior and Bayesian regularization. Advances in Neural Information Processing Systems, volume 15. pp. 697–704.  <ul class="find-nolinks"></ul></li><li><span class="label">71.
              </span><a name="pone.0001820-Xie1" id="pone.0001820-Xie1"></a>Xie X, Lu J, Kulbokas EJ, Golub TR, Mootha V, et al.  (2005) Systematic discovery of regulatory motifs in human promoters and 30 UTRs by comparison of several mammals. Nature  434: 338–345.  <ul class="find" data-citedArticleID="1076674" data-doi="10.1038/nature03441"><li><a href="http://dx.doi.org/10.1038/nature03441" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+discovery+of+regulatory+motifs+in+human+promoters+and+30+UTRs+by+comparison+of+several+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+discovery+of+regulatory+motifs+in+human+promoters+and+30+UTRs+by+comparison+of+several+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001820-Siepel1" id="pone.0001820-Siepel1"></a>Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, et al.  (2005) Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Research  15: 1034–1050.  <ul class="find" data-citedArticleID="1076638" data-doi="10.1101/gr.3715005"><li><a href="http://dx.doi.org/10.1101/gr.3715005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001820-Taylor1" id="pone.0001820-Taylor1"></a>Taylor J, Tyekucheva S, King DC, Hardison RC, Miller W, et al.  (2006) ESPERR: Learning strong and weak signals in genomic sequence alignments to identify functional elements. Genome Research  16: 1596–1604.  <ul class="find" data-citedArticleID="1076654" data-doi="10.1101/gr.4537706"><li><a href="http://dx.doi.org/10.1101/gr.4537706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ESPERR%3A+Learning+strong+and+weak+signals+in+genomic+sequence+alignments+to+identify+functional+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ESPERR%3A+Learning+strong+and+weak+signals+in+genomic+sequence+alignments+to+identify+functional+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001820-Segal1" id="pone.0001820-Segal1"></a>Segal E, Fondufe-Mittendorf Y, Chen L, Thåström A, Field Y, et al.  (2006) A genomic code for nucleosome positioning. Nature  442: 772–778.  <ul class="find" data-citedArticleID="1076632" data-doi="10.1038/nature04979"><li><a href="http://dx.doi.org/10.1038/nature04979" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomic+code+for+nucleosome+positioning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomic+code+for+nucleosome+positioning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001820-Heintzman1" id="pone.0001820-Heintzman1"></a>Heintzman ND, Ren B (2007) The gateway to transcription: identifying, characterizing and understanding promoters in the eukaryotic genome. Cellular and Molecular Life Sciences  64: 386–400.  <ul class="find" data-citedArticleID="1076556" data-doi="10.1007/s00018-006-6295-0"><li><a href="http://dx.doi.org/10.1007/s00018-006-6295-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+gateway+to+transcription%3A+identifying%2C+characterizing+and+understanding+promoters+in+the+eukaryotic+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+gateway+to+transcription%3A+identifying%2C+characterizing+and+understanding+promoters+in+the+eukaryotic+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001820-Levine1" id="pone.0001820-Levine1"></a>Levine M, Tjian R (2003) Transcription regulation and animal diversity. Nature  424: 147–151.  <ul class="find" data-citedArticleID="1076586" data-doi="10.1038/nature01763"><li><a href="http://dx.doi.org/10.1038/nature01763" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+regulation+and+animal+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+regulation+and+animal+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0001820-Ren1" id="pone.0001820-Ren1"></a>Ren B, Robert F, Wyrick JJ, Aparicio O, Jennings EG, et al.  (2000) Genome-wide location and function of DNA binding proteins. Science  290: 2306–2309.  <ul class="find" data-citedArticleID="1076624" data-doi="10.1126/science.290.5500.2306"><li><a href="http://dx.doi.org/10.1126/science.290.5500.2306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+location+and+function+of+DNA+binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+location+and+function+of+DNA+binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0001820-Crawford1" id="pone.0001820-Crawford1"></a>Crawford GE, Holt IE, Mullikin JC, Tai D, National Institutes of Health Intramural Sequencing Center (2004) Identifying gene regulatory elements by genome-wide recovery of DNase hypersensitive sites. Proceedings of the National Academy of Sciences of the USA  101: 992–997.  <ul class="find" data-citedArticleID="1076536" data-doi="10.1073/pnas.0307540100"><li><a href="http://dx.doi.org/10.1073/pnas.0307540100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+gene+regulatory+elements+by+genome-wide+recovery+of+DNase+hypersensitive+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+gene+regulatory+elements+by+genome-wide+recovery+of+DNase+hypersensitive+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0001820-KelMargoulis1" id="pone.0001820-KelMargoulis1"></a>Kel-Margoulis OV, Kel AE, Reuter I, Deineko IV, Wingender E (2002) TRANSCompel®: a database on composite regulatory elements in eukaryotic genes. Nucleic Acids Research  30: 332–334.  <ul class="find" data-citedArticleID="1076574" data-doi="10.1093/nar/30.1.332"><li><a href="http://dx.doi.org/10.1093/nar/30.1.332" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TRANSCompel%C2%AE%3A+a+database+on+composite+regulatory+elements+in+eukaryotic+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TRANSCompel%C2%AE%3A+a+database+on+composite+regulatory+elements+in+eukaryotic+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0001820-Smit1" id="pone.0001820-Smit1"></a>Smit AFA, Hubley R, Green P (1997) Repeatmasker computer program.   Available at <a href="http://www.repeatmasker.org/">http://www.repeatmasker.org/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">81.
              </span><a name="pone.0001820-Reguly1" id="pone.0001820-Reguly1"></a>Reguly T, Breitkreutz A, Boucher L, Breitkreutz BJ, Hon GC, et al.  (2006) Comprehensive curation and analysis of global interaction networks in <em>Saccharomyces cerevisiae</em>. Journal of Biology  5:   <ul class="find" data-citedArticleID="1076620"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comprehensive+curation+and+analysis+of+global+interaction+networks+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Comprehensive+curation+and+analysis+of+global+interaction+networks+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+curation+and+analysis+of+global+interaction+networks+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+curation+and+analysis+of+global+interaction+networks+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0001820-Thomas1" id="pone.0001820-Thomas1"></a>Thomas K, Wu J, Sung DY, Thompson W, Powell M, et al.  (2007) SP1 transcription factors in male germ cell development and differentiation. Molecular and Cellular Endocrinology  270: 1–7.  <ul class="find" data-citedArticleID="1076660" data-doi="10.1016/j.mce.2007.03.001"><li><a href="http://dx.doi.org/10.1016/j.mce.2007.03.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SP1+transcription+factors+in+male+germ+cell+development+and+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SP1+transcription+factors+in+male+germ+cell+development+and+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0001820-Cawley1" id="pone.0001820-Cawley1"></a>Cawley S, Bekiranov S, Ng HH, Kapranov P, Sekinger EA, et al.  (2004) Unbiased mapping of transcription factor binding sites along human chromosomes 21 and 22 points to widespread regulation of noncoding RNAs. Cell  116: 499–509.  <ul class="find" data-citedArticleID="1076528" data-doi="10.1016/s0092-8674(04)00127-8"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00127-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unbiased+mapping+of+transcription+factor+binding+sites+along+human+chromosomes+21+and+22+points+to+widespread+regulation+of+noncoding+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unbiased+mapping+of+transcription+factor+binding+sites+along+human+chromosomes+21+and+22+points+to+widespread+regulation+of+noncoding+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0001820-Hallikas1" id="pone.0001820-Hallikas1"></a>Hallikas O, Palin K, Sinjushina N, Rautiainen R, Partanen J, et al.  (2006) Genome-wide prediction of mammalian enhancers based on analysis of transcription-factor binding affinity. Cell  124: 47–59.  <ul class="find" data-citedArticleID="1076552" data-doi="10.1016/j.cell.2005.10.042"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.10.042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+prediction+of+mammalian+enhancers+based+on+analysis+of+transcription-factor+binding+affinity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+prediction+of+mammalian+enhancers+based+on+analysis+of+transcription-factor+binding+affinity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0001820-Segal2" id="pone.0001820-Segal2"></a>Segal E, Yelensky R, Koller D (2003) Genome-wide discovery of transcriptional modules from DNA sequence and gene expression. Bioinformatics  19: i273–i282.  <ul class="find" data-citedArticleID="1076634" data-doi="10.1093/bioinformatics/btg1038"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg1038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+discovery+of+transcriptional+modules+from+DNA+sequence+and+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+discovery+of+transcriptional+modules+from+DNA+sequence+and+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0001820-Tamada1" id="pone.0001820-Tamada1"></a>Tamada Y, Kim S, Bannai H, Imoto S, Tashiro K, et al.  (2003) Estimating gene networks from gene expression data by combining Bayesian network model with promoter element detection. Bioinformatics  19: ii227–ii236.  <ul class="find" data-citedArticleID="1076652" data-doi="10.1093/bioinformatics/btg1082"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg1082" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+gene+networks+from+gene+expression+data+by+combining+Bayesian+network+model+with+promoter+element+detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+gene+networks+from+gene+expression+data+by+combining+Bayesian+network+model+with+promoter+element+detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0001820-deHoon1" id="pone.0001820-deHoon1"></a>de Hoon MJL, Makita Y, Imoto S, Kobayashi K, Ogasawara N, et al.  (2004) Predicting gene regulation by sigma factors in <em>bacillus subtilis</em> from genome-wide data. Bioinformatics  20: i101–i108.  <ul class="find" data-citedArticleID="1076680" data-doi="10.1093/bioinformatics/bth927"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth927" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+gene+regulation+by+sigma+factors+in+bacillus+subtilis+from+genome-wide+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+gene+regulation+by+sigma+factors+in+bacillus+subtilis+from+genome-wide+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0001820-Lhdesmki1" id="pone.0001820-Lhdesmki1"></a>Lähdesmäki H, Shmulevich I (2007) Probabilistic framework for transcription factor binding prediction. Proceedings of Fifth IEEE International Workshop on Genomic Signal Processing and Statistics (Gensips'07).   <ul class="find-nolinks"></ul></li><li><span class="label">89.
              </span><a name="pone.0001820-Robert1" id="pone.0001820-Robert1"></a>Robert CP, Casella G (2005) Monte Carlo Statistical Methods. Springer-Verlag, 2nd edition.   <ul class="find-nolinks"></ul></li><li><span class="label">90.
              </span><a name="pone.0001820-Brooks1" id="pone.0001820-Brooks1"></a>Brooks SP, Giudici P, Philippe A (2003) Nonparametric convergence assessment for MCMC model selection. Journal of Computational and Graphical Statistics  12: 1–22.  <ul class="find" data-citedArticleID="1076524" data-doi="10.1198/1061860031347"><li><a href="http://dx.doi.org/10.1198/1061860031347" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonparametric+convergence+assessment+for+MCMC+model+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonparametric+convergence+assessment+for+MCMC+model+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0001820-Husmeier2" id="pone.0001820-Husmeier2"></a>Husmeier D (2005)  Introduction to learning Bayesian networks from data. In: Husmeier D, Dybowski R, Roberts S, editors. Probabilistic Modeling in Bioinformatics and Medical Informatics. UK: Springer-Verlag. pp. 17–57.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.XML" value="206704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.PDF" value="1204553"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g001.PNG_L" value="23120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g001.PNG_M" value="15378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g001.PNG_S" value="7958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g001.TIF" value="158332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g001.PNG_I" value="5127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e001.PNG" value="10232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e001.TIF" value="26424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e002.PNG" value="9387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e002.TIF" value="25400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g002.PNG_L" value="198515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g002.PNG_M" value="102466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g002.PNG_S" value="12973"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g002.TIF" value="598112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g002.PNG_I" value="36614"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g003.PNG_L" value="502254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g003.PNG_M" value="107626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g003.PNG_S" value="12407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g003.TIF" value="1553610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g003.PNG_I" value="43275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e003.PNG" value="5002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e003.TIF" value="20960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g004.PNG_L" value="393135"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g004.PNG_M" value="87961"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g004.PNG_S" value="13631"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g004.TIF" value="1381782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g004.PNG_I" value="30747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g005.PNG_L" value="289287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g005.PNG_M" value="69732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g005.PNG_S" value="9045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g005.TIF" value="840196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g005.PNG_I" value="23550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e004.PNG" value="10066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e004.TIF" value="26212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e005.PNG" value="10005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e005.TIF" value="26176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e006.PNG" value="6791"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e006.TIF" value="22748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e007.PNG" value="6728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e007.TIF" value="22732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g006.PNG_L" value="985855"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g006.PNG_M" value="193113"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g006.PNG_S" value="19372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g006.TIF" value="2956194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g006.PNG_I" value="84329"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g007.PNG_L" value="353087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g007.PNG_M" value="87002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g007.PNG_S" value="11945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g007.TIF" value="1330554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g007.PNG_I" value="29635"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g008.PNG_L" value="308116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g008.PNG_M" value="70335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g008.PNG_S" value="9684"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g008.TIF" value="879450"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g008.PNG_I" value="24560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g009.PNG_L" value="185704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g009.PNG_M" value="94720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g009.PNG_S" value="12467"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g009.TIF" value="558184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g009.PNG_I" value="34267"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e008.PNG" value="7872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e008.TIF" value="23728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g010.PNG_L" value="389703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g010.PNG_M" value="84005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g010.PNG_S" value="12017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g010.TIF" value="1383350"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g010.PNG_I" value="29622"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g011.PNG_L" value="388453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g011.PNG_M" value="84335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g011.PNG_S" value="12135"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g011.TIF" value="1380626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g011.PNG_I" value="29482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g012.PNG_L" value="163826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g012.PNG_M" value="84280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g012.PNG_S" value="12450"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g012.TIF" value="517398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g012.PNG_I" value="31783"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g013.PNG_L" value="415651"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g013.PNG_M" value="94023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g013.PNG_S" value="11547"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g013.TIF" value="1521242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g013.PNG_I" value="33367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g014.PNG_L" value="256407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g014.PNG_M" value="51784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g014.PNG_S" value="9238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g014.TIF" value="1100828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g014.PNG_I" value="17972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g015.PNG_L" value="163412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g015.PNG_M" value="40555"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g015.PNG_S" value="8514"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g015.TIF" value="919112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.g015.PNG_I" value="13577"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e009.PNG" value="8615"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e009.TIF" value="24808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e010.PNG" value="9625"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e010.TIF" value="25744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e011.PNG" value="7396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e011.TIF" value="23396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e012.PNG" value="8471"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e012.TIF" value="24388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e013.PNG" value="31766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e013.TIF" value="47984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e014.PNG" value="9887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e014.TIF" value="25716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e015.PNG" value="12353"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e015.TIF" value="28380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e016.PNG" value="11616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e016.TIF" value="27556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e017.PNG" value="5226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e017.TIF" value="21180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e018.PNG" value="5587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e018.TIF" value="21604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e019.PNG" value="17730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e019.TIF" value="33592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e020.PNG" value="8770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e020.TIF" value="24724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e021.PNG" value="23545"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e021.TIF" value="38740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e022.PNG" value="7065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e022.TIF" value="22980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e023.PNG" value="11119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e023.TIF" value="26828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e024.PNG" value="7321"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e024.TIF" value="23216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e025.PNG" value="9443"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e025.TIF" value="25524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e026.PNG" value="9376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e026.TIF" value="25248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e027.PNG" value="7969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e027.TIF" value="23936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e028.PNG" value="6744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e028.TIF" value="22712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e029.PNG" value="5318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e029.TIF" value="21268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e030.PNG" value="14639"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e030.TIF" value="30488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e031.PNG" value="10409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e031.TIF" value="26428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e032.PNG" value="26684"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e032.TIF" value="41708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e033.PNG" value="33037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e033.TIF" value="47756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e034.PNG" value="10701"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e034.TIF" value="26496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e035.PNG" value="11614"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e035.TIF" value="28036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e036.PNG" value="13746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e036.TIF" value="29780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e037.PNG" value="25047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e037.TIF" value="41260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e038.PNG" value="15130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e038.TIF" value="31084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e039.PNG" value="10837"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e039.TIF" value="26432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e040.PNG" value="10837"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e040.TIF" value="26532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e041.PNG" value="13333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e041.TIF" value="29612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e042.PNG" value="6686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e042.TIF" value="22640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e043.PNG" value="12905"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e043.TIF" value="28548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e044.PNG" value="11459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e044.TIF" value="27136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e045.PNG" value="11320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e045.TIF" value="27008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e046.PNG" value="9119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e046.TIF" value="25148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e047.PNG" value="10497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e047.TIF" value="26656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e048.PNG" value="9408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e048.TIF" value="25532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e049.PNG" value="12257"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e049.TIF" value="27924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e050.PNG" value="12645"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e050.TIF" value="28112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e051.PNG" value="11577"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e051.TIF" value="27092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e052.PNG" value="11173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e052.TIF" value="27292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e053.PNG" value="10082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e053.TIF" value="26228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e054.PNG" value="9716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e054.TIF" value="25744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e055.PNG" value="24811"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e055.TIF" value="40992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e056.PNG" value="8841"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e056.TIF" value="24836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e057.PNG" value="11474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e057.TIF" value="27080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e058.PNG" value="12840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e058.TIF" value="29384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e059.PNG" value="11248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e059.TIF" value="26976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e060.PNG" value="11047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e060.TIF" value="26840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e061.PNG" value="8611"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e061.TIF" value="24568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e062.PNG" value="6732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e062.TIF" value="22640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e063.PNG" value="13986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e063.TIF" value="30304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e064.PNG" value="5684"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e064.TIF" value="21608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e065.PNG" value="16114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.e065.TIF" value="31572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.s001.DOC" value="28160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.s002.TIF" value="515544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.s003.TIF" value="346102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.s004.TIF" value="353012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001820.s005.TIF" value="343186"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001820&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001820" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001820&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001820&volume=&issue=&title=Probabilistic Inference of Transcription Factor Binding from Multiple Data Sources&author_name=Harri%20L%C3%A4hdesm%C3%A4ki%2C%20Alistair%20G.%20Rust%2C%20Ilya%20Shmulevich&start_page=1&end_page=24" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001820" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001820" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001820" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001820&amp;t=Probabilistic%20Inference%20of%20Transcription%20Factor%20Binding%20from%20Multiple%20Data%20Sources" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001820&title=Probabilistic%20Inference%20of%20Transcription%20Factor%20Binding%20from%20Multiple%20Data%20Sources&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001820&amp;title=Probabilistic%20Inference%20of%20Transcription%20Factor%20Binding%20from%20Multiple%20Data%20Sources" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001820" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001820&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Probabilistic Inference of Transcription Factor Binding from Multiple Data Sources';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001820';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Probabilistic%20Inference%20of%20Transcription%20Factor%20Binding%20from%20Multiple%20Data%20Sources http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001820" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001820" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bayes+theorem%22" title="Search for articles in the subject area:'Bayes theorem'"><div class="flagText">Bayes theorem</div></a>
              <div data-categoryid="32251" data-articleid="26464"
                   data-categoryname="Bayes theorem"
                   class="flagImage" title="Flag 'Bayes theorem' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Binding+analysis%22" title="Search for articles in the subject area:'Binding analysis'"><div class="flagText">Binding analysis</div></a>
              <div data-categoryid="19597" data-articleid="26464"
                   data-categoryname="Binding analysis"
                   class="flagImage" title="Flag 'Binding analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="26464"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Forecasting%22" title="Search for articles in the subject area:'Forecasting'"><div class="flagText">Forecasting</div></a>
              <div data-categoryid="34789" data-articleid="26464"
                   data-categoryname="Forecasting"
                   class="flagImage" title="Flag 'Forecasting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="26464"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="26464"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleosomes%22" title="Search for articles in the subject area:'Nucleosomes'"><div class="flagText">Nucleosomes</div></a>
              <div data-categoryid="34075" data-articleid="26464"
                   data-categoryname="Nucleosomes"
                   class="flagImage" title="Flag 'Nucleosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26464"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2810'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8424'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4060&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=18873">Publisher's Note: Additional funding information</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=7659">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
