

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Expressed Sequence Tags as a Tool for Phylogenetic Analysis of Placental Mammal Evolution</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000775"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000775" />

    <meta name="citation_title" content="Expressed Sequence Tags as a Tool for Phylogenetic Analysis of Placental Mammal Evolution"/>
    <meta itemprop="name" content="Expressed Sequence Tags as a Tool for Phylogenetic Analysis of Placental Mammal Evolution"/>

      <meta name="citation_author" content="Morgan Kullberg"/>
            <meta name="citation_author_institution" content="Department of Cell and Organism Biology, Division of Evolutionary Molecular Systematics, University of Lund, Lund, Sweden"/>
      <meta name="citation_author" content="Björn Hallström"/>
            <meta name="citation_author_institution" content="Department of Cell and Organism Biology, Division of Evolutionary Molecular Systematics, University of Lund, Lund, Sweden"/>
      <meta name="citation_author" content="Ulfur Arnason"/>
            <meta name="citation_author_institution" content="Department of Cell and Organism Biology, Division of Evolutionary Molecular Systematics, University of Lund, Lund, Sweden"/>
      <meta name="citation_author" content="Axel Janke"/>
            <meta name="citation_author_institution" content="Department of Cell and Organism Biology, Division of Evolutionary Molecular Systematics, University of Lund, Lund, Sweden"/>

    <meta name="citation_date" content="2007/8/22"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000775.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e775"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Mammalian phylogeny: shaking the tree.; citation_author=MJ Novacek; citation_journal_title=Nature; citation_volume=356; citation_number=1; citation_pages=121-125; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Relationship of aardvark to elephants, hyraxes and sea cows from a-crystallin sequences.; citation_author=WW de Jong; citation_author=A Zweers; citation_author=M Goodman; citation_journal_title=Nature; citation_volume=292; citation_number=2; citation_pages=538-540; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Biomolecular systematics of eutherian mammals: phylogenetic patterns and classification.; citation_author=MM Miyamoto; citation_author=M Goodman; citation_journal_title=Syst Zool; citation_volume=35; citation_number=3; citation_pages=230-240; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Evolution of the cytochrome b gene of mammals.; citation_author=DM Irwin; citation_author=TD Kocher; citation_author=AC Wilson; citation_journal_title=J Mol Evol; citation_volume=32; citation_number=4; citation_pages=128-144; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The complete mitochondrial DNA sequence of the harbor seal, Phoca vitulina.; citation_author=U Arnason; citation_author=E Johnsson; citation_journal_title=J Mol Evol.; citation_volume=34; citation_number=5; citation_pages=493-505; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Tempo and mode of mitochondrial DNA evolution in vertebrates at the amino acid sequence level: rapid evolution in warm-blooded vertebrates.; citation_author=J Adachi; citation_author=Y Cao; citation_author=M Hasegawa; citation_journal_title=J Mol Evol; citation_volume=36; citation_number=6; citation_pages=270-281; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The marsupial mitochondrial genome and the evolution of placental mammals.; citation_author=A Janke; citation_author=G Feldmaier-Fuchs; citation_author=WK Thomas; citation_author=A von Haeseler; citation_author=S Pääbo; citation_journal_title=Genetics; citation_volume=137; citation_number=7; citation_pages=243-256; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Pattern and timing of evolutionary divergences among hominoids based on analyses of complete mtDNAs.; citation_author=U Arnason; citation_author=A Gullberg; citation_author=A Janke; citation_author=Y Xu; citation_journal_title=J Mol Evol; citation_volume=43; citation_number=8; citation_pages=650-661; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Mammalian mitogenomic relationships and the root of the eutherian tree.; citation_author=U Arnason; citation_author=JA Adegoke; citation_author=K Bodin; citation_author=EW Born; citation_author=YB Esa; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=9; citation_pages=8151-8156; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The complete mitochondrial DNA sequence of the shark Mustelus manazo: evaluating rooting contradictions to living bony vertebrates.; citation_author=Y Cao; citation_author=PJ Waddell; citation_author=N Okada; citation_author=M Hasegawa; citation_journal_title=Mol Biol Evol; citation_volume=15; citation_number=10; citation_pages=1637-1646; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Long-branch attraction phenomenon and the impact of among-site rate variation on rodent phylogeny.; citation_author=A Reyes; citation_author=G Pesole; citation_author=C Saccone; citation_journal_title=Gene; citation_volume=259; citation_number=11; citation_pages=177-187; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mitogenomic analyses of eutherian relationships.; citation_author=U Arnason; citation_author=A Janke; citation_journal_title=Cytogenet Genome Res; citation_volume=96; citation_number=12; citation_pages=20-32; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Congruent mammalian trees from mitochondrial and nuclear genes using Bayesian methods.; citation_author=A Reyes; citation_author=C Gissi; citation_author=F Catzeflis; citation_author=E Nevo; citation_author=G Pesole; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=13; citation_pages=397-403; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Resolution of the early placental mammal radiation using Bayesian phylogenetics.; citation_author=WJ Murphy; citation_author=E Eizirik; citation_author=SJ O'Brien; citation_author=O Madsen; citation_author=M Scally; citation_journal_title=Science; citation_volume=294; citation_number=14; citation_pages=2348-2351; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Identifying conflicting signal in a multigene analysis reveals a highly resolved tree: the phylogeny of Rodentia (Mammalia).; citation_author=RW DeBry; citation_journal_title=Syst Biol; citation_volume=52; citation_number=15; citation_pages=604-617; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny of coleoid cephalopods (Mollusca: Cephalopoda) using a multigene approach; the effect of data partitioning on resolving phylogenies in a Bayesian framework.; citation_author=J Strugnell; citation_author=M Norman; citation_author=J Jackson; citation_author=AJ Drummond; citation_author=A Cooper; citation_journal_title=Mol Phylogenet Evol; citation_volume=37; citation_number=16; citation_pages=426-441; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Early History of Mammals Is Elucidated with the ENCODE Multiple Species Sequencing Data.; citation_author=S Nikolaev; citation_author=JI Montoya-Burgos; citation_author=EH Margulies; citation_author=NCS Program; citation_author=J Rougemont; citation_author=B Nyffeler; citation_author=S Antonarakis; citation_journal_title=PloS Genet; citation_volume=3; citation_number=17; citation_pages=1; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Retroposed elements as archives for the evolutionary history of placental mammals. PLoS Biol.; citation_author=JO Kriegs; citation_author=G Churakov; citation_author=M Kiefmann; citation_author=U Jordan; citation_author=J Brosius; citation_author=J Schmitz; citation_volume=4; citation_number=18; citation_pages=e91; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Using genomic data to unravel the root of the placental mammal phylogeny.; citation_author=WJ Murphy; citation_author=TH Pringle; citation_author=TA Crider; citation_author=MS Springer; citation_author=W Miller; citation_journal_title=Genome Res; citation_volume=17; citation_number=19; citation_pages=413-421; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Evolutionary History of LINE-1 in the Major Clades of Placental Mammals.; citation_author=PD Waters; citation_author=G Dobigny; citation_author=PJ Waddell; citation_author=TJ Robinson; citation_journal_title=PloS ONE; citation_volume=2; citation_number=20; citation_pages=e158. doi:10.1371/journal.pone.0000158; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Housekeeping genes for phylogenetic analysis of eutherian relationships.; citation_author=M Kullberg; citation_author=MA Nilsson; citation_author=U Arnason; citation_author=EH Harley; citation_author=A Janke; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=21; citation_pages=1493-1503; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Complementary DNA sequencing: expressed sequence tags and human genome project.; citation_author=MD Adams; citation_author=JM Kelley; citation_author=JD Gocayne; citation_author=M Dubnick; citation_author=MH Polymeropoulos; citation_journal_title=Science; citation_volume=252; citation_number=22; citation_pages=1651-1656; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Phylogenomics of eukaryotes: impact of missing data on large alignments.; citation_author=H Philippe; citation_author=EA Snell; citation_author=E Bapteste; citation_author=P Lopez; citation_author=PW Holland; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=23; citation_pages=1740-1752; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of protein coding sequences from human, mouse and the domesticated pig.; citation_author=FG Jorgensen; citation_author=A Hobolth; citation_author=H Hornshoj; citation_author=C Bendixen; citation_author=M Fredholm; citation_journal_title=BMC Biol; citation_volume=3; citation_number=24; citation_pages=2; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=ESTimating plant phylogeny: lessons from partitioning.; citation_author=JE de la Torre; citation_author=MG Egan; citation_author=M Katari; citation_author=ED Brenner; citation_author=DW Stevenson; citation_journal_title=BMC Evol Biol; citation_volume=6; citation_number=25; citation_pages=48; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Deuterostome phylogeny reveals monophyletic chordates and the new phylum Xenoturbellida.; citation_author=SJ Bourlat; citation_author=T Juliusdottir; citation_author=CJ Lowe; citation_author=R Freeman; citation_author=J Aronowicz; citation_author=M Kirschner; citation_author=ES Lander; citation_author=M Thorndyke; citation_author=H Nakano; citation_author=AB Kohn; citation_author=A Heyland; citation_author=LL Moroz; citation_author=RR Copley; citation_author=MJ Telford; citation_journal_title=Nature; citation_volume=444; citation_number=26; citation_pages=85-88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dense taxonomic EST sampling and its applications for molecular systematics of the Coleoptera (beetles).; citation_author=J Hughes; citation_author=SJ Longhorn; citation_author=A Papadopoulou; citation_author=K Theodorides; citation_author=A de Riva; citation_author=M Mejia-Chang; citation_author=PG Foster; citation_author=AP Vogler; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=27; citation_pages=268-278; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Paleontological Evidence to Date the Tree of Life.; citation_author=MJ Benton; citation_author=PCJ Donoghue; citation_journal_title=MBE Advance Access published on October 17; citation_number=28; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Nuclear gene sequences provide evidence for the monophyly of australidelphian marsupials.; citation_author=H Amrine-Madsen; citation_author=M Scally; citation_author=M Westerman; citation_author=MJ Stanhope; citation_author=C Krajewski; citation_journal_title=Mol Phylogenet Evol.; citation_volume=28; citation_number=29; citation_pages=186-196; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Different patterns of selection on the nuclear genes IRBP and DMP-1 affect the efficiency but not the outcome of phylogeny estimation for didelphid marsupials.; citation_author=SA Jansa; citation_author=JF Forsman; citation_author=RS Voss; citation_journal_title=Mol Phylogenet Evol; citation_volume=38; citation_number=30; citation_pages=363-380; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular biology of the gene.; citation_author=JD Watson; citation_author=NH Hopkins; citation_author=JW Roberts; citation_author=AJ Steitz; citation_author=AM Weiner; citation_number=31; citation_date=1965; citation_publisher=Benjamin/Cummings; " />
      <meta name="citation_reference" content="citation_title=Comparison of human adult and fetal expression and identification of 535 housekeeping/maintenance genes.; citation_author=JA Warrington; citation_author=A Nair; citation_author=M Mahadevappa; citation_author=M Tsyganskaya; citation_journal_title=Physiol Genomics; citation_volume=2; citation_number=32; citation_pages=143-147; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A compendium of gene expression in normal human tissues.; citation_author=LL Hsiao; citation_author=F Dangond; citation_author=T Yoshida; citation_author=R Hong; citation_author=RV Jensen; citation_journal_title=Physiol Genomics; citation_volume=7; citation_number=33; citation_pages=97-104; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Using shared genomic synteny and shared protein functions to enhance the identification of orthologous gene pairs.; citation_author=XH Zheng; citation_author=F Lu; citation_author=ZY Wang; citation_author=F Zhong; citation_author=J Hoover; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=34; citation_pages=703-710; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Making sense of EST sequences by CLOBBing them.; citation_author=J Parkinson; citation_author=DB Guiliano; citation_author=M Blaxter; citation_journal_title=BMC Bioinformatics 2002; citation_volume=3; citation_number=35; citation_pages=31; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Large-Scale search for Genes on Which Positive selection May Operate.; citation_author=T Endo; citation_author=K Ikeo; citation_author=T Gojobori; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=36; citation_pages=685-690; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions.; citation_author=M Nei; citation_author=T Gojobori; citation_journal_title=Mol Biol Evol; citation_volume=3; citation_number=37; citation_pages=418-426; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=The Influence of Phylogenetic Uncertainty on the Detection of Positive Darwinian Selection.; citation_author=MR Pie; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=38; citation_pages=2274-2278; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The ENCODE pilot project: Functional annotation of 1% of the human genome.; citation_journal_title=Nature; citation_volume=14; citation_number=39; citation_pages=799-816; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Site specific rates of mitochondrial genomes and the phylogeny of eutheria.; citation_author=KM Kjer; citation_author=RL Honeycutt; citation_journal_title=BMC Evol Biol; citation_volume=7; citation_number=40; citation_pages=8; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A Phylogenomic Study of Human, Dog and Mouse.; citation_author=GM Cannarozzi; citation_author=A Schneider; citation_author=G Gonnet; citation_journal_title=PLoS Computational Biology, e2.eor doi:10.1371/journal.pcbi.0030002.eor; citation_number=41; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Rates of genome evolution and branching order from whole genome analysis.; citation_author=GA Huttley; citation_author=MJ Wakefield; citation_author=S Easteal; citation_journal_title=Mol Biol Evol In press; citation_number=42; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Effect of Branch Lengths on Phylogeny: an Empirical Study Using Highly Conserved Orthologs from Mammalian Genomes, Mol Phylogenet Evol doi:10.1016/j.ympev.2007.04.022; citation_author=AL Hughes; citation_author=R Friedman; citation_number=43; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cases in which parsimony and compatibility methods will be positively misleading.; citation_author=J Felsenstein; citation_journal_title=Syst Zool; citation_volume=27; citation_number=44; citation_pages=401-410; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Are Guinea pigs rodents? The importance of adequate models in molecular phylogenetics.; citation_author=J Sullivan; citation_author=DL Swofford; citation_journal_title=J Mammal Evol; citation_volume=4; citation_number=45; citation_pages=77-86; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Lack of resulution in the animal phylogeny: closely spaced cladogeneses or undetected systematic errors?; citation_author=D Baurain; citation_author=H Brinkmann; citation_author=H Philippe; citation_journal_title=Mol Biol Evol In press; citation_number=46; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Computional molecular evolution.; citation_author=Z Yang; citation_number=47; citation_date=2006; citation_publisher=Oxford University press; " />
      <meta name="citation_reference" content="citation_title=Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction.; citation_author=P Chomczynski; citation_author=N Sacchi; citation_journal_title=Anal Biochem; citation_volume=162; citation_number=48; citation_pages=156-159; citation_date=1987; " />
      <meta name="citation_reference" content="citation_number=49; " />
      <meta name="citation_reference" content="citation_number=50; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Research,; citation_volume=22; citation_number=51; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Se-Alv2.0a11: sequence aignment editor.; citation_author=A Rambaut; citation_number=52; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=PANTHER: A Library of Protein Families and Subfamilies Indexed by Function.; citation_author=PD Thomas; citation_author=MJ Campbell; citation_author=A Kejariwal; citation_author=H Mi; citation_author=B Karlak; citation_journal_title=Genome Res; citation_volume=13; citation_number=53; citation_pages=2129-2141; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The PANTHER database of protein families, subfamilies, functions and pathways Nucl.; citation_author=H Mi; citation_author=B Lazareva-Ulitsky; citation_author=R Loo; citation_author=A Kejariwal; citation_author=J Vandergriff; citation_journal_title=Acids Res; citation_volume=33; citation_number=54; citation_pages=D284-288; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Quartet puzzling: a quartet maximum likelihood method for reconstructing tree topologies.; citation_author=K Strimmer; citation_author=A von Haeseler; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=55; citation_pages=964-969; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=PHYLIP (Phylogeny Inference Package) version 3.5c. Distributed by the author.; citation_author=J Felsenstein; citation_number=56; citation_date=1993; citation_publisher=Department of Genetics, University of Washington; " />
      <meta name="citation_reference" content="citation_title=MOLPHY Version 2.3: Programs for molecular phylogenetics based on maximum likelihood.; citation_author=J Adachi; citation_author=M Hasegawa; citation_journal_title=Computer Science Monographs; citation_volume=28; citation_number=57; citation_pages=1 Inst Stat Mathematics, Tokyo; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=MRBAYES: Bayesian inference of phylogenetic trees.; citation_author=JP Huelsenbeck; citation_author=F Ronquist; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=58; citation_pages=754-755; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum lielihood.; citation_author=Z Yang; citation_journal_title=Computer Applications in BioSciences; citation_volume=13; citation_number=59; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=TREEFINDER: A powerful graphical analysis environment for molecular phylogenetics.; citation_author=G Jobb; citation_author=A von Haeseler; citation_author=K Strimmer; citation_journal_title=BMC Evol Biol; citation_volume=4; citation_number=60; citation_pages=18; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A simple, fast and accurate algorithm to estimate large phylogenies by maximum likelihood.; citation_author=S Guindon; citation_author=S Gascuel; citation_journal_title=Syst Biol; citation_volume=52; citation_number=61; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=PAUP* Phylogenetic Analysis using Parsimony(*and other Methods), version 4.0b8.; citation_author=DL Swofford; citation_number=62; citation_date=2002; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=Modeltest: testing the model of DNA substitution.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=63; citation_pages=817-818; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=ProtTest: selection of the best-fit models of protein evolution.; citation_author=F Abascal; citation_author=R Zardoya; citation_author=D Posada; citation_journal_title=Bioinformatics; citation_volume=7; citation_number=64; citation_pages=2104-2105; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The rapid generation of mutation data matrices from protein sequences.; citation_author=DT Jones; citation_author=WR Taylor; citation_author=JM Thornton; citation_journal_title=CABIOS; citation_volume=8; citation_number=65; citation_pages=275-282; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=A new method for calculating evolutionary substitution rates.; citation_author=C Lanave; citation_author=G Preparata; citation_author=C Saccone; citation_author=G Serio; citation_journal_title=J Mol Evol; citation_volume=20; citation_number=66; citation_pages=86-93; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Maximum likelihood estimation of the heterogeneity of substitution rate among nucleotide sites.; citation_author=X Gu; citation_author=YX Fu; citation_author=W-H Li; citation_journal_title=Mol Biol Evol; citation_volume=12; citation_number=67; citation_pages=546-557; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The Bioportal at University of Oslo.; citation_number=68; " />
      <meta name="citation_reference" content="citation_title=Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level.; citation_author=J Zhang; citation_author=R Nielsen; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=69; citation_pages=2472-2479; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bayes empirical bayes inference of amino acid sites under positive selection.; citation_author=Z Yang; citation_author=WS Wong; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=70; citation_pages=1107-1118; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000775" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Expressed Sequence Tags as a Tool for Phylogenetic Analysis of Placental Mammal Evolution"/>
    <meta name="twitter:description" content="BackgroundWe investigate the usefulness of expressed sequence tags, ESTs, for establishing divergences within the tree of placental mammals. This is done on the example of the established relationships among primates (human), lagomorphs (rabbit), rodents (rat and mouse), artiodactyls (cow), carnivorans (dog) and proboscideans (elephant).Methodology/Principal FindingsWe have produced 2000 ESTs (1.2 mega bases) from a marsupial mouse and characterized the data for their use in phylogenetic analysis. The sequences were used to identify putative orthologous sequences from whole genome projects. Although most ESTs stem from single sequence reads, the frequency of potential sequencing errors was found to be lower than allelic variation. Most of the sequences represented slowly evolving housekeeping-type genes, with an average amino acid distance of 6.6% between human and mouse. Positive Darwinian selection was identified at only a few single sites. Phylogenetic analyses of the EST data yielded trees that were consistent with those established from whole genome projects.ConclusionsThe general quality of EST sequences and the general absence of positive selection in these sequences make ESTs an attractive tool for phylogenetic analysis. The EST approach allows, at reasonable costs, a fast extension of data sampling from species outside the genome projects."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000775.g004"/>

  <meta property="og:title" content="Expressed Sequence Tags as a Tool for Phylogenetic Analysis of Placental Mammal Evolution" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000775" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7015'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9449'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2309&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000775">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000775" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000775&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Expressed Sequence Tags as a Tool for Phylogenetic Analysis of Placental Mammal Evolution
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Morgan Kullberg
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:morgan.kullberg@cob.lu.se">morgan.kullberg@cob.lu.se</a></p>

                <p>Affiliation:
                  Department of Cell and Organism Biology, Division of Evolutionary Molecular Systematics, University of Lund, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Björn Hallström, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Cell and Organism Biology, Division of Evolutionary Molecular Systematics, University of Lund, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ulfur Arnason, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Cell and Organism Biology, Division of Evolutionary Molecular Systematics, University of Lund, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Axel Janke
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Cell and Organism Biology, Division of Evolutionary Molecular Systematics, University of Lund, Lund, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 22, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000775</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000775" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000775" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000775" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000775" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000775-t001" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000775-t002" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000775-g001" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000775-t003" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000775-g002" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000775-g003" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000775-g004" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000775-t004" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000775-t005" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000775-t006" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000775-t007" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t007" title="Table 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000775">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000775" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>We investigate the usefulness of expressed sequence tags, ESTs, for establishing divergences within the tree of placental mammals. This is done on the example of the established relationships among primates (human), lagomorphs (rabbit), rodents (rat and mouse), artiodactyls (cow), carnivorans (dog) and proboscideans (elephant).</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We have produced 2000 ESTs (1.2 mega bases) from a marsupial mouse and characterized the data for their use in phylogenetic analysis. The sequences were used to identify putative orthologous sequences from whole genome projects. Although most ESTs stem from single sequence reads, the frequency of potential sequencing errors was found to be lower than allelic variation. Most of the sequences represented slowly evolving housekeeping-type genes, with an average amino acid distance of 6.6% between human and mouse. Positive Darwinian selection was identified at only a few single sites. Phylogenetic analyses of the EST data yielded trees that were consistent with those established from whole genome projects.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The general quality of EST sequences and the general absence of positive selection in these sequences make ESTs an attractive tool for phylogenetic analysis. The EST approach allows, at reasonable costs, a fast extension of data sampling from species outside the genome projects.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kullberg M, Hallström B, Arnason U, Janke A (2007) Expressed Sequence Tags as a Tool for Phylogenetic Analysis of Placental Mammal Evolution. PLoS ONE 2(8):
          e775.
            doi:10.1371/journal.pone.0000775</p><p><strong>Academic Editor: </strong>Sudhindra Gadagkar, University of Dayton, United States of America</p><p><strong>Received:</strong> January 23, 2007; <strong>Accepted:</strong> July 24, 2007; <strong>Published:</strong> August 22, 2007</p><p><strong>Copyright:</strong> © 2007 Kullberg et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The Swedish Science Council, the Jörgen Lindström Foundation, and the Royal Physiographic Society (Nilsson-Ehle) supported the study.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>In 1992 Novacek <a href="#pone.0000775-Novacek1">[1]</a> presented a widely known hypothesis for the phylogenetic tree of placental mammals based on a synthesis of morphological and molecular findings. At that time only limited amounts of sequence data were available, a circumstance that rendered many ordinal relationships unresolved. During an initial stage phylogenetic analyses of sequence data were generally based on single genes or parts of genes <a href="#pone.0000775-deJong1">[2]</a>–<a href="#pone.0000775-Irwin1">[4]</a>. This changed gradually and during the 1990's sequences of complete mitochondrial (mt) genomes became a common tool in phylogenetic analyses (e.g. <a href="#pone.0000775-Arnason1">[5]</a>, <a href="#pone.0000775-Adachi1">[6]</a>). The combined sequences of all mt protein-coding genes yield alignment lengths of about 10–12 kbp, i.e. about 10-times the sequence amounts commonly used in the 1980s. However, in the absence of a closely related outgroup these analyses could not conclusively establish the direction of evolution in the placental tree. This limitation was amended by the first marsupial mt genome sequence, that of the opossum, <em>Didelphis virginiana</em> <a href="#pone.0000775-Janke1">[7]</a>. The marsupial rooting of the placental tree placed Rodentia (mouse, rat) as the sister group to remaining orders. This position of Rodentia was upheld in the great majority of the following mammalian mitogenomic (mtg) analyses, i.e. phylogenetic analyses based on the protein-coding genes of complete mt genomes (e.g. <a href="#pone.0000775-Arnason2">[8]</a>–<a href="#pone.0000775-Arnason4">[12]</a>). However recent mtg studies joined rodents and primates on a common branch (e.g. <a href="#pone.0000775-Reyes2">[13]</a>). Thus, relationships within some basal parts of the placental tree remained equivocal, even in phylogenetic analyses of complete mt genomes. As some of these analyses demonstrated <a href="#pone.0000775-Arnason3">[9]</a>, <a href="#pone.0000775-Arnason4">[12]</a>, the basal position of the rodents in the mtg tree of placental mammals was sensitive to the sampling of other basal taxa and to the analytical approaches applied.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In 2001 Murphy et al. <a href="#pone.0000775-Murphy1">[14]</a> presented phylogenetic results that challenged some parts of the placental mtg tree. The study was based on both mt data and directly PCR amplified introns of nuclear genes. The contribution of individual taxa to the complete data set differed somewhat and the alignment of the nuclear sequences showed considerable numbers of gaps and ambiguous sites. This was particularly noticeable in three of the nuclear sequences (≈ 50% of the nuclear data) in which the amino acid distance between human and the mouse ranged between 20% and 40%, a circumstance that may adversely affect the aligning of homologous sites. Similarly the concatenation of genes showing great evolutionary rate variation may affect the estimation of model parameters such as the gamma distribution parameter, α <a href="#pone.0000775-DeBry1">[15]</a>, <a href="#pone.0000775-Strugnell1">[16]</a>. The main parts of this nuclear gene tree <a href="#pone.0000775-Murphy1">[14]</a> have nevertheless been supported in later studies based on far more comprehensive alignments [e.g. 17] and genome level characters like retroposon insertion and indel differences <a href="#pone.0000775-Kriegs1">[18]</a>–<a href="#pone.0000775-Waters1">[20]</a>. One of the main differences between this nuclear gene tree and previous mtg findings was that monophyletic Rodentia grouped with Lagomorpha, thereby supporting the morphological Glires hypothesis. Together with Primates, Dermoptera and Scandentia, Glires formed the super-ordinal clade Euarchontoglires. The sister group to the Euarchontoglires, called Laurasiatheria, included Artiodactyla, Carnivora and Perissodactyla among other orders. Euarchontaglires and Laurasiatheria are commonly joined in the Boreoeutheria.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The problems related to resolving basal placental relationships were again underlined in a recent study based on the sequences of eight housekeeping genes that were established by cDNA approaches from 22 placental mammals and three marsupials <a href="#pone.0000775-Kullberg1">[21]</a>. The total length of the alignment was 6 kb and all genes had similar evolutionary rates. Inconsistent with the results of Murphy et al. <a href="#pone.0000775-Murphy1">[14]</a> the analyses favored a tree with Glires in a basal position relative to Primates rather than joining Primates and Glires on a common branch. Furthermore, and despite an extended and more uniform sequence representation of each individual taxon as compared to the study of Murphy et al. <a href="#pone.0000775-Murphy1">[14]</a>, the position of the root of the placental tree was not conclusively established when rate heterogeneity models were applied <a href="#pone.0000775-Kullberg1">[21]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In this study we have selected the Boreoeutheria group to examine the utility ESTs for phylogenetic analyses, as a novel approach to economically obtain large amounts of protein-coding sequences. The procedure rests upon the random retrieval of ESTs from a cDNA library, which represents all expressed genes in a cell at a given time <a href="#pone.0000775-Adams1">[22]</a>. The ensuing database search allows subsequent complementation with orthologous sequences from species that are of interest to the phylogenetic study. This approach has hitherto been applied in only a limited number of phylogenetic studies that have addressed deep relationships among eukaryots, plants, arthropods and mammalians <a href="#pone.0000775-Philippe1">[23]</a>–<a href="#pone.0000775-Hughes1">[27]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Currently, genome projects of some 20 placental and marsupial mammals are in different stages of completion. Sequence data from these projects have allowed resolution of several ordinal and superordinal placental relationships <a href="#pone.0000775-Nikolaev1">[17]</a> with which the results from the EST based phylogenetic analysis can be compared. In order to promote the identification of the root of the placental tree we have, along with the production of placental sequences established the homologous sequences from an Australian marsupial, the fat-tailed dunnart, <em>Sminthopsis crassicaudata</em>. With an upper paleontological limit of about 125 million years before present, MYBP, for the divergence between marsupial and placental mammals <a href="#pone.0000775-Benton1">[28]</a> the inclusion of <em>Sminthopsis</em> constitutes a definite advantage in determining the root of the tree of placental mammals.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>More than 1.200.000 nt sequences representing about 2000 EST sequences were retrieved from the <em>Sminthopsis</em> tissue culture cells (fibroblasts). About 1600 EST sequences with a minimum length of ≈400 bp were collected for further evaluation. After excluding vector and mt sequences, 854 individual nuclear cDNA sequences and contigs remained for the complementary database search. Orthology search against the human mRNA RefSeq database identified 455 protein-coding sequences with E-values &lt;10<sup>−15</sup> that were subsequently aligned. A list of the accession numbers of the putative 455 human orthologous mRNA sequences is provided in the <a href="#pone.0000775.s001">Table S1</a>. Several un-translated sequences were identified during the search. These sequences were not included in the study as it focuses on protein-coding genes. 344 of the 455 human mRNA transcripts could be classified according to the PANTHER classification system, while 109 sequences remained unclassified. <a href="#pone-0000775-t001">Table 1</a> shows the classification for those gene classes that had more than five members.</p>
<div class="figure" id="pone-0000775-t001"><div class="img"><a name="pone-0000775-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Classification of the human homologues</span></strong></p><span>doi:10.1371/journal.pone.0000775.t001</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>Of these 455 sequences a total of 161 sequences were represented by seven placental species (elephant, mouse, rat, rabbit, human, cow and dog). This alignment (named <em>maxspe</em>) that maximized the mammalian representation for all sequences had a length of 77,328 nt (25,776 aa). A second alignment that maximized the number of sequences by allowing some sequences to be missing was also constructed. This alignment, which included 326 sequences (164,466 nt or 54,822 aa) from the eight species, is referred to as the <em>maxgen</em> alignment. Genomes with a low current sequencing coverage such as those of the elephant and the rabbit were allowed to lack 25% of the genes. In a few cases one or two sequences of cetferungulates (cow or dog) and/or rodents (mouse or rat) were allowed to be missing in the <em>maxgen</em> alignment. The chicken was not represented in about 33% of the alignments for both <em>maxspe</em> and <em>maxgen</em> and was therefore excluded from all analysis based on single genes. The general properties of the two datasets are given in <a href="#pone-0000775-t002">Table 2</a>.</p>
<div class="figure" id="pone-0000775-t002"><div class="img"><a name="pone-0000775-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>General statistics of the concatenated data sets</span></strong></p><span>doi:10.1371/journal.pone.0000775.t002</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>The length of the individual and trimmed alignments excluding gaps varied from 126 to 1167 nt, with an average of 505 nt (<a href="#pone-0000775-g001">Figure 1</a>). The genetic distances between human and mouse ranged from 0% to 20% (mean = 4.5±4.5) for aa sequences (<a href="#pone-0000775-g002">Figure 2</a>), 0%–18% (mean = 3.2±2.9) for first and second codon positions (12cdp), and 3%–24% (mean = 11.0±3.3) for all codon positions (123cdp). Alignments with zero aa distance between human and mouse or human and cow were excluded from the analysis. Additional properties such as the number of gaps and number of constant sites in the different alignments are shown in <a href="#pone-0000775-t002">Table 2</a>.</p>
<div class="figure" id="pone-0000775-g001"><div class="img"><a name="pone-0000775-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000775.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000775.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Distribution of the alignment lengths after trimming.</span></strong></p><span>doi:10.1371/journal.pone.0000775.g001</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>Estimates of potential sequencing errors in <em>Sminthopsis</em> ESTs indicated an error rate of approximately 0.01% and allelic variation of about 0.02%. Further evidence that sequence differences had been correctly classified as allelic variation rested on the observation that the sequence differences occurred always at silent 3rd codon positions. Most of the differences constituted frequent naturally occurring C-T transitions. A potential error rate of 0.01% was also recorded in 102,232 nt of mt ESTs with a 10-fold coverage of about 10,000 nt of overlapping mt protein-coding sites. Comparison between the EST data and the mt genome of another <em>Sminthopsis</em> individual showed 134 differences (0.1%). This value is within the expected sequence variation of mt sequences of different individuals. The results suggest that sequence differences related to sequencing errors are less frequent than natural allelic variation, although the statistics behind these differences is limited due to the low total numbers of differences. The findings suggest that potential sequencing errors in the EST sequencing study are at a level that effects the current phylogenetic analyses far less than allelic variation.</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>The aa distances within the two alignments, <em>maxgen</em> and <em>maxspe</em>, are shown in <a href="#pone-0000775-t003">Table 3</a>. Distances between the outgroup and the ingroup taxa differed by ≈ 10%, indicating a limited difference in evolutionary rates among the ingroup species. There is a notable difference between the marsupial and placental distances relative to the chicken, indicating a faster evolution in the placentals. Among the placental mammals the sequences of Glires and the elephant appear to evolve faster than those of <em>Homo</em> and the cetferungulates. A chi-2 test as implemented in TREE-PUZZLE showed that composition of the aa as well as 1st and 2nd codon positions (12cdp) is homogeneous among the mammalian species. However, nt composition was not homogeneous when the same test was applied to all three codon positions.</p>
<div class="figure" id="pone-0000775-t003"><div class="img"><a name="pone-0000775-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Pairwise aa distances between species.</span></strong></p><span>doi:10.1371/journal.pone.0000775.t003</span></div><a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p><a href="#pone-0000775-g003">Figure 3</a> shows the Bayesian tree based on the nt sequences of the <em>maxgen</em> dataset. Posterior probability values were 1.00 for all nodes in the tree. ML rate heterogeneity bootstrap support from the <em>maxspe</em> data was moderate to high (70–100%) for the aa and cdp123 but low for cdp12 (39% for Boreoeutheria). When the chicken, the elephant, and the rabbit were excluded from the alignment there was a 0.85 posterior probability for the Euarchontoglires clade. Unpartitioned ML analyses including all species resulted in the same topology as in <a href="#pone-0000775-g003">Figure 3</a>, but rodents fell basal when the chicken, the elephant, and the rabbit were excluded. NJ and MP analysis generally placed rodents as sister group to all other placentals, regardless of the taxon sampling.</p>
<div class="figure" id="pone-0000775-g002"><div class="img"><a name="pone-0000775-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000775.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000775.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Distribution of the pair wise aa distances between human and mouse.</span></strong></p><span>doi:10.1371/journal.pone.0000775.g002</span></div><div class="figure" id="pone-0000775-g003"><div class="img"><a name="pone-0000775-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000775.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000775.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Phylogenetic relationships based on partitioned Bayesian inference of the <em>maxgen</em> alignment (164466 nt) and a GTR+4Γ+I model of sequence evolution.</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>The partitioning are according to four evolutionary rate classes and codon positions. The model parameters are estimated separate for every partitioning.</p>
<span>doi:10.1371/journal.pone.0000775.g003</span></div><a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>Likelihood ratio tests showed that partitioning of the data significantly improved the fit to the evolutionary model. The largest impact was seen from partitioning according to codon positions, where the increase in the logL values was several thousand. Partitioning according the genetic distances increased the logL values by a few hundred logL values, which was still a statistically significant improvement.</p>
<a id="article1.body1.sec2.p8" name="article1.body1.sec2.p8"></a><p>In order to further investigate the support for the best tree shown in <a href="#pone-0000775-g003">Figure 3</a> the ΔlogL/S.E. ratio and the pSH values were calculated for alternative placental trees, <a href="#pone-0000775-g004">Figure 4</a>. The alternative trees represented the 2nd and 3rd best ML alternatives according to PROTML, an alternative tree based on housekeeping genes <a href="#pone.0000775-Kullberg1">[21]</a>, the MP and NJ aa tree and the mtg tree <a href="#pone.0000775-Arnason3">[9]</a>. Different evolutionary models were used to calculate the likelihood and pSH values for the <em>maxgen</em> and <em>maxspe</em> alignments on the basis of aa, 12cdp and 123cdp nt sequences (<a href="#pone-0000775-t004">Table 4</a>). All analyses identified tree-1 as the best tree, but alternative topologies could not be statistically rejected by all datasets, except in the case of tree-4.</p>
<div class="figure" id="pone-0000775-g004"><div class="img"><a name="pone-0000775-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000775.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000775.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Relationships between species in the tested alternative topologies.</span></strong></p><span>doi:10.1371/journal.pone.0000775.g004</span></div><div class="figure" id="pone-0000775-t004"><div class="img"><a name="pone-0000775-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>ML support for the different phylogenetic hypothesis.</span></strong></p><span>doi:10.1371/journal.pone.0000775.t004</span></div><a id="article1.body1.sec2.p9" name="article1.body1.sec2.p9"></a><p>The 161 genes of the <em>maxspe</em> alignment were individually analyzed in order to estimate the proportion of genes that supported each of the six alternative topologies shown in <a href="#pone-0000775-g004">Figure 4</a>. The logL for each of these topologies was calculated using TREE-PUZZLE. If the ΔlogL/S.E. ratio value between alternative trees exceeded 0.5 the topology was recorded. If the value was less than 0.5, the support was regarded as inconclusive. Although the cut-off ΔlogL/S.E. ratio of 0.5 was arbitrarily chosen, ΔlogL/S.E. ratios of this level indicate some support for the best topology over alternatives. <a href="#pone-0000775-t005">Table 5</a> summarizes the support for alternative trees as recorded for individual genes applying the different analytical approaches. Most genes that contained enough phylogenetic information to distinguish between alternative topologies supported tree-1. The strongest phylogenetic support came from single gene analyses of the 123cdp alignments. However, the majority of the single genes does not carry sufficient phylogenetic information to distinguish between the alternative trees.</p>
<div class="figure" id="pone-0000775-t005"><div class="img"><a name="pone-0000775-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Support of three alternative topologies by ML analysis of single genes.</span></strong></p><span>doi:10.1371/journal.pone.0000775.t005</span></div><a id="article1.body1.sec2.p10" name="article1.body1.sec2.p10"></a><p>The nt and aa sequences of single genes from the <em>maxspe</em> dataset were analyzed with respect to their support for internal branches by calculating and comparing the ML values of different trees using PHYML and quartet puzzling (QP) as implemented in TREE-PUZZLE. The CONSENSE program (PHYLIP) was used to summarize the gene-trees by calculating a majority rule consensus tree. <a href="#pone-0000775-t006">Table 6</a> shows the number of genes that favored selected internal branches according to the ML and QP analyses. Depending on the mode of analysis the support for different clade varied. The only branch receiving strong support was Rodentia, represented by the relatively closely related mouse and rat; all other branches were weakly supported, notably by the QP method.</p>
<div class="figure" id="pone-0000775-t006"><div class="img"><a name="pone-0000775-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Numbers of genes that support a certain clade.</span></strong></p><span>doi:10.1371/journal.pone.0000775.t006</span></div><a id="article1.body1.sec2.p11" name="article1.body1.sec2.p11"></a><p>The effect of taxon sampling and the choice of outgroup species on the phylogenetic reconstruction were evaluated by analyzing the data after excluding one or more species at the time. Inclusion of the chicken as an extra outgroup besides <em>Sminthopsis</em> has no effect on the analysis, but when only the chicken was used as outgroup tree-1 and tree-5 became nearly indistinguishable. Likewise exclusion of rabbit led to the promotion of a basal position of rodents as in trees 4, 5 and 6. Euarchontaglires remained supported after exclusion of the elephant (trees 1, 2 and 3). Exclusion of either mouse or rat, or cow or dog had no effect on the topology.</p>
<a id="article1.body1.sec2.p12" name="article1.body1.sec2.p12"></a><p>In order to investigate whether directional (positive) selection might have affected the tree reconstruction, a ML analysis applying a branch-site model of evolution as implemented in PAML was performed. The analysis was carried out both on the individual genes of the <em>maxspe</em> dataset and the concatenated <em>maxgen</em> data. Among the single genes, 22 had at least one branch with codons that had ω &gt;1, <a href="#pone-0000775-t007">Table 7</a>. This suggests that only a few genes have single sites that might be affected by positive selection. Analyses of the concatenated <em>maxgen</em> dataset confirmed that most branches have only a few single codons with ω &gt;1 that are under selection, <a href="#pone-0000775-t007">Table 7</a>. The concatenated sequences of the elephant appeared to contain more sites (40) under positive selection than in the other mammalian species studied.</p>
<div class="figure" id="pone-0000775-t007"><div class="img"><a name="pone-0000775-t007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000775" data-uri="info:doi/10.1371/journal.pone.0000775.t007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000775.t007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000775.t007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000775.t007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000775.t007.TIF"></span>)
                </a></li></ul></div><p><strong>Table 7.  <span>Sequences showing signs of positive selection on at least one branch under branch-site models. Significant LRT at 1%.</span></strong></p><span>doi:10.1371/journal.pone.0000775.t007</span></div></div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Phylogenetic studies of deeper mammalian relationships, such as those of placental orders, are to a great extent based on sequence analysis of protein-coding genes. Compared to the protein-coding regions the non-coding regions of the genomes evolve in general much faster and are often rearranged and randomized by multiple substitutions. Hitherto, nuclear gene sequences used in phylogenetic analyses have commonly come from genomic PCR-amplifications of single exons or from intron-less protein-coding genes (e.g. <a href="#pone.0000775-Murphy1">[14]</a>, <a href="#pone.0000775-AmrineMadsen1">[29]</a>, <a href="#pone.0000775-Jansa1">[30]</a>). The application of cDNA approaches circumvents this limitation as it allows the production of complete coding sequences from a variety of genes <a href="#pone.0000775-Kullberg1">[21]</a>. Sequencing of ESTs is one method to economically produce large amounts of protein-coding sequences for phylogenetic purposes.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>About 45% of the EST sequences obtained in the current study constituted multiple nuclear sequences and about 7% were mt encoded. About 1/3rd of the nuclear sequences and contigs had putative homologues among the selected species and could be used for phylogenetic analysis. Most of the ESTs produced in the current study represented genes that can be classified as housekeeping genes <a href="#pone.0000775-Watson1">[31]</a>–<a href="#pone.0000775-Hsiao1">[33]</a>. Among mammalian orders the aa distances of housekeeping genes are generally limited to only a few percent, making them an ideal choice for phylogenetic analysis due to the facility with which correct alignments can be established. The limited distances among homologous housekeeping genes contrasts in this respect to nuclear genes such as vWF (von Willebrand factor), IRPB (interphotoreceptor retinoid binding protein) and the BRCA1 (breast and ovarian cancer susceptibility protein 1) sequence, which are commonly included in phylogenetic analyses. These three sequences have aa distance of 20–45% among mammalian orders. In comparison to these three sequences the effect of randomization in housekeeping genes can be considered as being limited. While it can be argued that the conserved nature of the housekeeping genes reduces the phylogenetic content of each single gene, this is compensated by the large amount of different EST sequences that can be produced from each individual taxon. Another advantage of applying cDNA sequences is that the risk of including pseudogenes is low. Although there are reports suggesting that some pseudogenes are transcribed <a href="#pone.0000775-Zheng1">[34]</a>, it is conceivable that pseudogenic sequences, if present, are rare among the large number of genic EST sequences. Thus, even if a few pseudogenes might occur among the ESTs they would be expected to have little or no influence on the phylogenetic outcome. By applying search against mRNA databases such as RefSeq the potential inclusion of pseudogenic sequences is also counteracted.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>One particular difficulty in any phylogenetic study that utilizes nuclear encoded genes is the establishment of their orthology. When PCR based approaches are used to amplify sequences from genomic DNA <a href="#pone.0000775-Murphy1">[14]</a>, <a href="#pone.0000775-AmrineMadsen1">[29]</a>, <a href="#pone.0000775-Jansa1">[30]</a> or cDNA <a href="#pone.0000775-Kullberg1">[21]</a> the orthology can only be assumed by similarity criteria. Criteria that use phylogenetic information <a href="#pone.0000775-delaTorre1">[25]</a> are generally preferred, but these depend on a known tree, the very subject that is under study. Synteny is another powerful criterion that has been used to define orthology <a href="#pone.0000775-Zheng1">[34]</a>. Determination of orthology by the way of synteny analysis could not be achieved in the current study since it requires that the sequence of almost the whole genome is available, which is not the case for <em>Sminthopsis</em>. For this reason a number of precautions were undertaken in the current study in order to minimize the risk of including paralogous or pseudogenic sequences. This included reciprocal BLAST searches between human and the other species, with a cutoff E-value of 10<sup>−15</sup> in order to ensure orthology between species. Also the use of a high quality databases such as RefSeq promotes the inclusion of known functional genes rather than pseudogenes in the analysis. In addition, after finding a possible human homologue, its full-length cDNA sequence instead of the shorter EST sequences was used for reciprocal searches in other species to further increase the chance of identifying orthologous genes in other species. The rigorous approach applied aims to maximize the probability of including only orthologous genes in the analyses.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>There have been concerns about the quality of EST data for phylogenetic analyses <a href="#pone.0000775-Parkinson1">[35]</a>, because most of the individual sequences are based on single reads. This study showed, however, that sequence differences due to sequencing errors of ESTs are at a level similar to that of allelic variation or even lower. The potential effect of errors of this kind can therefore be considered as negligible for the current phylogenetic results.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The phylogenetic analyses appeared stable with respect to the assumption of evolutionary neutrality. The search for positive selection identified only few genes with single sites that had an ω value &gt;1 in one or more branches. Other studies <a href="#pone.0000775-Endo1">[36]</a> have identified even fewer incidents of positive selection, when a pairwise method <a href="#pone.0000775-Nei1">[37]</a> was applied to compare ω between distant groups from different animal classes. The pairwise approach of that study probably identified fewer candidate genes, because the evaluation did not rely on phylogenetic information. Thus, selection may have been active on single branches in the past without the signal being recognized. The discrepancy between the number of sites identified by the analysis of the single genes and the concatenated <em>maxgen</em> dataset may be related to more robust statistics when larger numbers of codons are involved in the analysis.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Compared to the total number of characters the low number of codons and the few genes that appear to be under selection appear to have no practical effect on the phylogenetic reconstruction. Only a few branches are affected by positive selection and the selection was not specific for a particular group, species or branch. If the phylogeny is incorrect, however, e.g. rodents or Glires as sister to the remaining species as in tree-4 to tree-6, fewer sites should actually be selected for, because in suboptimal phylogenies the number of sites under selection becomes over-estimated <a href="#pone.0000775-Pie1">[38]</a>.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Phylogenetic examination of the eight placental taxa that are represented by nearly complete genomes joined primates and Glires as sister groups composing the clade Euarchontoglires. This clade is sister to Boreoeutheria as represented by the cow and the dog. The analyses favored a sister group relationship between the elephant and all other placentals included. The same general topology was also found using sequences from the ENCODE consortium <a href="#pone.0000775-Kriegs1">[18]</a>, <a href="#pone.0000775-The1">[39]</a>. Other studies <a href="#pone.0000775-Murphy1">[14]</a>, <a href="#pone.0000775-Murphy2">[19]</a>, <a href="#pone.0000775-Kjer1">[40]</a> have favored the same topology using shorter alignments but a more comprehensive taxon sampling. These results challenge previous studies using EST <a href="#pone.0000775-Jorgensen1">[24]</a>, large-scale genomic data <a href="#pone.0000775-Cannarozzi1">[41]</a>–<a href="#pone.0000775-Hughes2">[43]</a>, mtgs <a href="#pone.0000775-Arnason3">[9]</a> and the analysis of few housekeeping genes <a href="#pone.0000775-Kullberg1">[21]</a>.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Basal mammalian divergences have proven difficult to resolve, despite the use of large amount of nuclear sequences. This may reflect the potentially narrow temporal window within which the divergences took place, leaving only a low number of phylogenetically informative sites. While this constitutes a limitation that is common to all phylogenetic analyses the impact that taxon sampling may have on the tree is striking. Thus, in the current study Rodents tended to become the sister group to other placentals when the rabbit was not included. The same tendency, which may be the effect of long branch attraction <a href="#pone.0000775-Felsenstein1">[44]</a>, has been observed earlier is studies with a limited taxon sampling <a href="#pone.0000775-Cannarozzi1">[41]</a>–<a href="#pone.0000775-Hughes2">[43]</a>. An indication of a long branch attraction between rodents and the outgroup is that tree-5 was favored by MP and NJ analyses. Especially MP is known to be sensitive to evolutionary rate differences among the taxa and long branch attraction <a href="#pone.0000775-Felsenstein1">[44]</a>.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Among placental mammals the rodents may behave as a so-called rouge taxon, i.e. a lineages that tends to skew phylogenetic analyses <a href="#pone.0000775-Sullivan1">[45]</a>. This effect can be overcome by excluding taxa of this kind <a href="#pone.0000775-Baurain1">[46]</a> or by including less deviating taxa for compensation. In the current study the attraction between rodents and the outgroup could be compensated by the inclusion of lagomorph sequences and complex model of sequence evolution. Other studies have questioned the use of long branch attraction phenomenon as an explanation to a basal position of rodents, because this particular topology received strong support from slowly evolving genes, while fast evolving genes supported Euarchontaglires <a href="#pone.0000775-Hughes2">[43]</a>. The authors concluded that this contradicts the expectation from the effect of long branch attraction. The tendency of ML analysis to join fast (mouse) and slow (human) evolving taxa has instead been coined “long branch repulsion” or “opposite branch attraction” <a href="#pone.0000775-Hughes2">[43]</a>. It is not clear how effects of this kind may have affected the current results, because it is not obvious why the rabbit would promote the opposite branch attraction phenomenon. The shift in the topology favored by ML when the rabbit is excluded form the analysis is more easily explained by long branch attraction between rodents and the outgroup. Further evidence for long branch attraction comes from the observation that ML cannot distinguish between tree-1 and tree-5 when only chicken is used as an outgroup. It appears that in these analyses the rodents are dragged towards the root of the placental tree. This illustrates the importance of choosing a not too distant outgroup and justifies the establishment of the marsupial EST sequences for this study.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>Assumptions about evolutionary models have a major impact on the recovered phylogeny. A parameter rich model naturally fits the data better than a simpler model <a href="#pone.0000775-Yang1">[47]</a>. Dividing the data into partitions increases the number of parameters and thereby the fitness of the model. In order to create evolutionary models that are more realistic Kjer and Honeycutt <a href="#pone.0000775-Kjer1">[40]</a> partitioned the sites in a mtg analysis of eutherian relationship into classes according to their relative MP consistency index. This resulted in a mtg phylogeny with strong support for the Atlantogenata hypothesis, which has not being supported by non-partitioned mtg analysis <a href="#pone.0000775-Arnason3">[9]</a>. The performance of different partition strategies for concatenated data has been studied for mollusk sequence data. Partitioning data according to codon positions, and to a lesser extent also by genes, improves the fitness of models to the data <a href="#pone.0000775-Strugnell1">[16]</a>. In a large and variable dataset such as in this EST study, one can expect that extensive rate heterogeneity that may not be correctly accounted for by using non-partitioned analyses.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>We tried to account for among site rate variation by partitioning the data into codon positions and also after the evolutionary rate of the genes. Obviously, partitioning according to single genes was impossible due to the number of genes included in this study. The size of each partition needs also to be reasonable large in order to get an accurate estimations of each parameter. We therefore divided the data according to distance classes and codon positions. Our approach to account for the among site rate variation satisfy the desire for a realistic model and still keeps the analysis on a computational acceptable level.</p>
<a id="article1.body1.sec3.p12" name="article1.body1.sec3.p12"></a><p>As mentioned above a limited taxon sampling may lead to an incorrect topology due to assuming models that are not consistent with the evolution. This could explain the results of some previous studies <a href="#pone.0000775-Jorgensen1">[24]</a>, <a href="#pone.0000775-Cannarozzi1">[41]</a>, <a href="#pone.0000775-Huttley1">[42]</a>. The strong attraction of rodent to the outgroup disappears for our data when they are partitioned and analyzed under Bayesian approaches.</p>
<a id="article1.body1.sec3.p13" name="article1.body1.sec3.p13"></a><p>While complete genomes are the ultimate data sets for resolving phylogenetic and evolutionary issues of different kinds (e.g. <a href="#pone.0000775-Benton1">[28]</a>, <a href="#pone.0000775-Cannarozzi1">[41]</a>), the costs of producing these data sets are still at a level that that precludes a dense taxonomic sampling among higher organisms. There is therefore a need to establish methods that at reasonable costs allow the production of sequence data that can be of general interest for phylogenetic studies. Producing EST sequences is such a method that will gain more attention in the future.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Total RNA was isolated from fibroblast cell culture (EAECC number: SC 11) of <em>Sminthopsis crassicaudata</em> (fat-tailed dunnart) using the acid phenol guanidinium thiocyanate method, GTC-method <a href="#pone.0000775-Chomczynski1">[48]</a>. Enriched mRNA was reverse transcribed, size fractionated, and cloned, yielding a total of 2000 cDNAs that were sequenced by QIAGEN (Germany) (Accession number EV533153-EV534821). The retrieved sequences were analyzed for identical or overlapping sequences from different clones using the Sequencher version 4.6 (Gene Codes) software. Contigs were assembled from overlapping genes. The sequencing error rate and the proportion of allelic variation were estimated by comparing more than 500 nucleotides (nt) from ESTs that were represented by two or more clones. Nt differences were recorded as allelic variation when at least two different nt occurred at the same site, with each type being represented in at least two sequences. Other differences were counted as potential sequencing errors. As evident with this approach allelic variation becomes automatically underestimated and sequencing errors overestimated unless comprehensive sequence coverage exists. Furthermore, the sequence error rate was estimated from comparing mt EST sequences among themselves and to the complete mt genome from another individual (Accession number NC_007631).</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Individual sequences and contigs were used to search for homologous sequences in the mRNA database using blastn as implemented in the EST-e-mate v1.0 program package (in-house application, code available from authors Hallström and Janke). In short, EST-e-mate blasts the marsupial ESTs against the NCBI human mRNA RefSeq database <a href="#pone.0000775-NCBI1">[49]</a>. The human sequences with the lowest E-value (expect value) were chosen as a template for searching for homologous sequences from the corresponding RefSeq database <a href="#pone.0000775-NCBI1">[49]</a> of chicken (<em>Gallus gallus</em>), mouse (<em>Mus musculus</em>), rat (<em>Rattus norvegicus</em>), cow (<em>Bos taurus</em>) and dog (<em>Canis familiaris</em>) as at 30.Apr.2006. Rabbit (<em>Oryctolagus cuniculus</em>) and elephant (<em>Loxodonta africana</em>) were retrieved from ENSEMBL <a href="#pone.0000775-ENSEMBL1">[50]</a> 1.Apr.2007. Species which represent very close relatives (e.g. chimpanzee to human) were not included in the analysis. The elephant was chosen, because it shows the slowest evolutionary rate among the non-boreoeutherians <a href="#pone.0000775-Nikolaev1">[17]</a>. Sequence hits with E-values above 10<sup>−15</sup> were excluded from further analysis. The program EST-e-mate utilizes ClustalW <a href="#pone.0000775-Thompson1">[51]</a> for aligning the sequences, while keeping the reading frame intact with sequence alignments trimmed relative to the shortest sequence. Gaps and columns with ambiguous characters were removed. Potentially faulty alignments, i.e. alignments in which any taxon pair had amino acid (aa) distance value &gt;0.6 were inspected further. All alignments were manually inspected using the Se-Al v2.0a11 software <a href="#pone.0000775-Rambaut1">[52]</a> and analyzed individually or as concatenated files. The mRNAs were functionally classified using the PANTHER classification system <a href="#pone.0000775-Thomas1">[53]</a>–<a href="#pone.0000775-Mi1">[54]</a>.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Sequence data were analyzed using the TREE-PUZZLE <a href="#pone.0000775-Strimmer1">[55]</a>, PHYLIP <a href="#pone.0000775-Felsenstein2">[56]</a>, MOLPHY <a href="#pone.0000775-Adachi2">[57]</a>, MrBayes v3.1.2 <a href="#pone.0000775-Huelsenbeck1">[58]</a>, PAML3.15 <a href="#pone.0000775-Yang2">[59]</a>, TREEFINDER <a href="#pone.0000775-Jobb1">[60]</a>, PHYML <a href="#pone.0000775-Guindon1">[61]</a> or PAUP* <a href="#pone.0000775-Swofford1">[62]</a> program packages. For the concatenated data the best-fitted model for nt sequence evolution and parameters were determined applying MODELTEST version 3.7 <a href="#pone.0000775-Posada1">[63]</a> and PROTTEST version 1.3 <a href="#pone.0000775-Abascal1">[64]</a>. The JTT model <a href="#pone.0000775-Jones1">[65]</a> of amino acid (aa) sequence evolution and the GTR model <a href="#pone.0000775-Lanave1">[66]</a> of nt evolution were used for distance and likelihood analyses. All phylogenetic analyses were computed assuming a gamma model of rate heterogeneity <a href="#pone.0000775-Gu1">[67]</a> with four classes of variable sites and one class of invariable site (4Γ+I). When a program did not allow for invariable sites, eight classes of variable site were used (8Γ). For the <em>maxgen</em> alignment Bayesian analysis were conducted running two simultaneously analysis with MrBayes applying one cold and three heated chains for 10,000,000 MCMC (Markov chain Monte Carlo) generations, discarding the first 1,000,000 generations as burnin. To compensate for the rate heterogeneity in the data we divided the alignment into twelve partitions, each with its own individual GTR matrix, gamma distribution, proportion of invariable sites and base frequencies. The four main partitions were according to the observed (0–5%, 5–10%, 10–15%, &gt;15%) aa distances between human and mouse. These four partitions were further divided according to codon positions. For aa and cdp12 the dataset were divided only into four partitions according to aa distances. Bayesian analyses were made on the TITAN cluster of the Bioportal <a href="#pone.0000775-1">[68]</a>.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Analyses for potential selection were made for the concatenated sequences and single genes using codeml (PAML3.15) by estimating the non-synonymous (dN), synonymous (dS) substitution rates and ω (dN/dS) for one branch at a time. This approach corresponds to the branch-site model. In this model the branch of interest (foreground) can have sites with an ω-value larger than one and all other branches (background) are restricted to ω-values below or equal to one <a href="#pone.0000775-Zhang1">[69]</a>. A Bayes empirical Bayes (BEB) procedure <a href="#pone.0000775-Yang3">[70]</a> was used to identify the sites evolving under potential positive selection. The codon frequencies were estimated from the data (CodonFreq = 3). All alignment columns containing ambiguities and gaps were excluded during the PAML analysis.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000775"></div><a name="pone.0000775.s001" id="pone.0000775.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000775.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Accession numbers of 455 the human genes homologous to the ESTs</p>
<p class="siDoi">doi:10.1371/journal.pone.0000775.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.12 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors would like to express their gratitude to Maria Nilsson for comments on the manuscript, the Bioportal at University of Oslo for making computer resources public available and two anonymous reviewers for comments on an earlier version of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AJ MK. Performed the experiments: MK. Analyzed the data: MK BH. Contributed reagents/materials/analysis tools: BH. Wrote the paper: UA AJ MK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000775-Novacek1" id="pone.0000775-Novacek1"></a>Novacek MJ (1992) Mammalian phylogeny: shaking the tree. Nature  356: 121–125.  <ul class="find" data-citedArticleID="980928" data-doi="10.1038/356121a0"><li><a href="http://dx.doi.org/10.1038/356121a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+phylogeny%3A+shaking+the+tree." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+phylogeny%3A+shaking+the+tree.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000775-deJong1" id="pone.0000775-deJong1"></a>de Jong WW, Zweers A, Goodman M (1981) Relationship of aardvark to elephants, hyraxes and sea cows from a-crystallin sequences. Nature  292: 538–540.  <ul class="find" data-citedArticleID="980974" data-doi="10.1038/292538a0"><li><a href="http://dx.doi.org/10.1038/292538a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+of+aardvark+to+elephants%2C+hyraxes+and+sea+cows+from+a-crystallin+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+of+aardvark+to+elephants%2C+hyraxes+and+sea+cows+from+a-crystallin+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000775-Miyamoto1" id="pone.0000775-Miyamoto1"></a>Miyamoto MM, Goodman M (1986) Biomolecular systematics of eutherian mammals: phylogenetic patterns and classification. Syst Zool  35: 230–240.  <ul class="find" data-citedArticleID="980916" data-doi="10.2307/2413433"><li><a href="http://dx.doi.org/10.2307/2413433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biomolecular+systematics+of+eutherian+mammals%3A+phylogenetic+patterns+and+classification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biomolecular+systematics+of+eutherian+mammals%3A+phylogenetic+patterns+and+classification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000775-Irwin1" id="pone.0000775-Irwin1"></a>Irwin DM, Kocher TD, Wilson AC (1991) Evolution of the cytochrome b gene of mammals. J Mol Evol  32: 128–144.  <ul class="find" data-citedArticleID="980894" data-doi="10.1007/bf02515385"><li><a href="http://dx.doi.org/10.1007/bf02515385" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+cytochrome+b+gene+of+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+cytochrome+b+gene+of+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000775-Arnason1" id="pone.0000775-Arnason1"></a>Arnason U, Johnsson E (1992) The complete mitochondrial DNA sequence of the harbor seal, Phoca vitulina. J Mol Evol.  34: 493–505.  <ul class="find" data-citedArticleID="980850" data-doi="10.1007/bf00160463"><li><a href="http://dx.doi.org/10.1007/bf00160463" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+mitochondrial+DNA+sequence+of+the+harbor+seal%2C+Phoca+vitulina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+mitochondrial+DNA+sequence+of+the+harbor+seal%2C+Phoca+vitulina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000775-Adachi1" id="pone.0000775-Adachi1"></a>Adachi J, Cao Y, Hasegawa M (1993) Tempo and mode of mitochondrial DNA evolution in vertebrates at the amino acid sequence level: rapid evolution in warm-blooded vertebrates. J Mol Evol  36: 270–281.  <ul class="find" data-citedArticleID="980842" data-doi="10.1007/bf00160483"><li><a href="http://dx.doi.org/10.1007/bf00160483" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tempo+and+mode+of+mitochondrial+DNA+evolution+in+vertebrates+at+the+amino+acid+sequence+level%3A+rapid+evolution+in+warm-blooded+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tempo+and+mode+of+mitochondrial+DNA+evolution+in+vertebrates+at+the+amino+acid+sequence+level%3A+rapid+evolution+in+warm-blooded+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000775-Janke1" id="pone.0000775-Janke1"></a>Janke A, Feldmaier-Fuchs G, Thomas WK, von Haeseler A, Pääbo S (1994) The marsupial mitochondrial genome and the evolution of placental mammals. Genetics  137: 243–256.  <ul class="find" data-citedArticleID="980896"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+marsupial+mitochondrial+genome+and+the+evolution+of+placental+mammals.&amp;auth=&amp;atitle=The+marsupial+mitochondrial+genome+and+the+evolution+of+placental+mammals." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+marsupial+mitochondrial+genome+and+the+evolution+of+placental+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+marsupial+mitochondrial+genome+and+the+evolution+of+placental+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000775-Arnason2" id="pone.0000775-Arnason2"></a>Arnason U, Gullberg A, Janke A, Xu Y (1996) Pattern and timing of evolutionary divergences among hominoids based on analyses of complete mtDNAs. J Mol Evol  43: 650–661.  <ul class="find" data-citedArticleID="980852" data-doi="10.1007/bf02202113"><li><a href="http://dx.doi.org/10.1007/bf02202113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pattern+and+timing+of+evolutionary+divergences+among+hominoids+based+on+analyses+of+complete+mtDNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pattern+and+timing+of+evolutionary+divergences+among+hominoids+based+on+analyses+of+complete+mtDNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000775-Arnason3" id="pone.0000775-Arnason3"></a>Arnason U, Adegoke JA, Bodin K, Born EW, Esa YB, et al.  (2002) Mammalian mitogenomic relationships and the root of the eutherian tree. Proc Natl Acad Sci U S A  99: 8151–8156.  <ul class="find" data-citedArticleID="980854" data-doi="10.1073/pnas.102164299"><li><a href="http://dx.doi.org/10.1073/pnas.102164299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+mitogenomic+relationships+and+the+root+of+the+eutherian+tree." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+mitogenomic+relationships+and+the+root+of+the+eutherian+tree.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000775-Cao1" id="pone.0000775-Cao1"></a>Cao Y, Waddell PJ, Okada N, Hasegawa M (1998) The complete mitochondrial DNA sequence of the shark Mustelus manazo: evaluating rooting contradictions to living bony vertebrates. Mol Biol Evol  15: 1637–1646.  <ul class="find" data-citedArticleID="980866" data-doi="10.1093/oxfordjournals.molbev.a025891"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025891" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+mitochondrial+DNA+sequence+of+the+shark+Mustelus+manazo%3A+evaluating+rooting+contradictions+to+living+bony+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+mitochondrial+DNA+sequence+of+the+shark+Mustelus+manazo%3A+evaluating+rooting+contradictions+to+living+bony+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000775-Reyes1" id="pone.0000775-Reyes1"></a>Reyes A, Pesole G, Saccone C (2000) Long-branch attraction phenomenon and the impact of among-site rate variation on rodent phylogeny. Gene  259: 177–187.  <ul class="find" data-citedArticleID="980940" data-doi="10.1016/s0378-1119(00)00438-8"><li><a href="http://dx.doi.org/10.1016/s0378-1119(00)00438-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long-branch+attraction+phenomenon+and+the+impact+of+among-site+rate+variation+on+rodent+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long-branch+attraction+phenomenon+and+the+impact+of+among-site+rate+variation+on+rodent+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000775-Arnason4" id="pone.0000775-Arnason4"></a>Arnason U, Janke A (2002) Mitogenomic analyses of eutherian relationships. Cytogenet Genome Res  96: 20–32.  <ul class="find" data-citedArticleID="980856" data-doi="10.1159/000063023"><li><a href="http://dx.doi.org/10.1159/000063023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitogenomic+analyses+of+eutherian+relationships." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitogenomic+analyses+of+eutherian+relationships.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000775-Reyes2" id="pone.0000775-Reyes2"></a>Reyes A, Gissi C, Catzeflis F, Nevo E, Pesole G, et al.  (2004) Congruent mammalian trees from mitochondrial and nuclear genes using Bayesian methods. Mol Biol Evol  21: 397–403.  <ul class="find" data-citedArticleID="980942" data-doi="10.1093/molbev/msh033"><li><a href="http://dx.doi.org/10.1093/molbev/msh033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Congruent+mammalian+trees+from+mitochondrial+and+nuclear+genes+using+Bayesian+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Congruent+mammalian+trees+from+mitochondrial+and+nuclear+genes+using+Bayesian+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000775-Murphy1" id="pone.0000775-Murphy1"></a>Murphy WJ, Eizirik E, O'Brien SJ, Madsen O, Scally M, et al.  (2001) Resolution of the early placental mammal radiation using Bayesian phylogenetics. Science  294: 2348–2351.  <ul class="find" data-citedArticleID="980918" data-doi="10.1126/science.1067179"><li><a href="http://dx.doi.org/10.1126/science.1067179" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resolution+of+the+early+placental+mammal+radiation+using+Bayesian+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resolution+of+the+early+placental+mammal+radiation+using+Bayesian+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000775-DeBry1" id="pone.0000775-DeBry1"></a>DeBry RW (2003) Identifying conflicting signal in a multigene analysis reveals a highly resolved tree: the phylogeny of Rodentia (Mammalia). Syst Biol  52: 604–617.  <ul class="find" data-citedArticleID="980870"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identifying+conflicting+signal+in+a+multigene+analysis+reveals+a+highly+resolved+tree%3A+the+phylogeny+of+Rodentia+%28Mammalia%29.&amp;auth=&amp;atitle=Identifying+conflicting+signal+in+a+multigene+analysis+reveals+a+highly+resolved+tree%3A+the+phylogeny+of+Rodentia+%28Mammalia%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+conflicting+signal+in+a+multigene+analysis+reveals+a+highly+resolved+tree%3A+the+phylogeny+of+Rodentia+%28Mammalia%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+conflicting+signal+in+a+multigene+analysis+reveals+a+highly+resolved+tree%3A+the+phylogeny+of+Rodentia+%28Mammalia%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000775-Strugnell1" id="pone.0000775-Strugnell1"></a>Strugnell J, Norman M, Jackson J, Drummond AJ, Cooper A (2005) Molecular phylogeny of coleoid cephalopods (Mollusca: Cephalopoda) using a multigene approach; the effect of data partitioning on resolving phylogenies in a Bayesian framework. Mol Phylogenet Evol  37: 426–441.  <ul class="find" data-citedArticleID="980946" data-doi="10.1016/j.ympev.2005.03.020"><li><a href="http://dx.doi.org/10.1016/j.ympev.2005.03.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogeny+of+coleoid+cephalopods+%28Mollusca%3A+Cephalopoda%29+using+a+multigene+approach%3B+the+effect+of+data+partitioning+on+resolving+phylogenies+in+a+Bayesian+framework." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogeny+of+coleoid+cephalopods+%28Mollusca%3A+Cephalopoda%29+using+a+multigene+approach%3B+the+effect+of+data+partitioning+on+resolving+phylogenies+in+a+Bayesian+framework.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000775-Nikolaev1" id="pone.0000775-Nikolaev1"></a>Nikolaev S, Montoya-Burgos JI, Margulies EH, Program NCS, Rougemont J, Nyffeler B, Antonarakis S (2007) Early History of Mammals Is Elucidated with the ENCODE Multiple Species Sequencing Data. PloS Genet  3: 1.  <ul class="find" data-citedArticleID="980926" data-doi="10.1371/journal.pgen.0030002"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+History+of+Mammals+Is+Elucidated+with+the+ENCODE+Multiple+Species+Sequencing+Data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+History+of+Mammals+Is+Elucidated+with+the+ENCODE+Multiple+Species+Sequencing+Data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000775-Kriegs1" id="pone.0000775-Kriegs1"></a>Kriegs JO, Churakov G, Kiefmann M, Jordan U, Brosius J, Schmitz J (2006) Retroposed elements as archives for the evolutionary history of placental mammals. PLoS Biol. 4: e91.  <ul class="find" data-citedArticleID="980908" data-doi="10.1371/journal.pbio.0040091"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040091" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroposed+elements+as+archives+for+the+evolutionary+history+of+placental+mammals.+PLoS+Biol." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroposed+elements+as+archives+for+the+evolutionary+history+of+placental+mammals.+PLoS+Biol.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000775-Murphy2" id="pone.0000775-Murphy2"></a>Murphy WJ, Pringle TH, Crider TA, Springer MS, Miller W (2007) Using genomic data to unravel the root of the placental mammal phylogeny. Genome Res  17: 413–421.  <ul class="find" data-citedArticleID="980920" data-doi="10.1101/gr.5918807"><li><a href="http://dx.doi.org/10.1101/gr.5918807" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+genomic+data+to+unravel+the+root+of+the+placental+mammal+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+genomic+data+to+unravel+the+root+of+the+placental+mammal+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000775-Waters1" id="pone.0000775-Waters1"></a>Waters PD, Dobigny G, Waddell PJ, Robinson TJ (2007) Evolutionary History of LINE-1 in the Major Clades of Placental Mammals. PloS ONE  2: e158. doi:10.1371/journal.pone.0000158.  <ul class="find" data-citedArticleID="980960" data-doi="10.1371/journal.pone.0000158"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+History+of+LINE-1+in+the+Major+Clades+of+Placental+Mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+History+of+LINE-1+in+the+Major+Clades+of+Placental+Mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000775-Kullberg1" id="pone.0000775-Kullberg1"></a>Kullberg M, Nilsson MA, Arnason U, Harley EH, Janke A (2006) Housekeeping genes for phylogenetic analysis of eutherian relationships. Mol Biol Evol  23: 1493–1503.  <ul class="find" data-citedArticleID="980910" data-doi="10.1093/molbev/msl027"><li><a href="http://dx.doi.org/10.1093/molbev/msl027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Housekeeping+genes+for+phylogenetic+analysis+of+eutherian+relationships." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Housekeeping+genes+for+phylogenetic+analysis+of+eutherian+relationships.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000775-Adams1" id="pone.0000775-Adams1"></a>Adams MD, Kelley JM, Gocayne JD, Dubnick M, Polymeropoulos MH, et al.  (1991) Complementary DNA sequencing: expressed sequence tags and human genome project. Science  252: 1651–1656.  <ul class="find" data-citedArticleID="980846" data-doi="10.1126/science.2047873"><li><a href="http://dx.doi.org/10.1126/science.2047873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complementary+DNA+sequencing%3A+expressed+sequence+tags+and+human+genome+project." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complementary+DNA+sequencing%3A+expressed+sequence+tags+and+human+genome+project.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000775-Philippe1" id="pone.0000775-Philippe1"></a>Philippe H, Snell EA, Bapteste E, Lopez P, Holland PW, et al.  (2004) Phylogenomics of eukaryotes: impact of missing data on large alignments. Mol Biol Evol  21: 1740–1752.  <ul class="find" data-citedArticleID="980932" data-doi="10.1093/molbev/msh182"><li><a href="http://dx.doi.org/10.1093/molbev/msh182" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenomics+of+eukaryotes%3A+impact+of+missing+data+on+large+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenomics+of+eukaryotes%3A+impact+of+missing+data+on+large+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000775-Jorgensen1" id="pone.0000775-Jorgensen1"></a>Jorgensen FG, Hobolth A, Hornshoj H, Bendixen C, Fredholm M, et al.  (2005) Comparative analysis of protein coding sequences from human, mouse and the domesticated pig. BMC Biol  3: 2.  <ul class="find" data-citedArticleID="980904" data-doi="10.1186/1741-7007-3-2"><li><a href="http://dx.doi.org/10.1186/1741-7007-3-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+protein+coding+sequences+from+human%2C+mouse+and+the+domesticated+pig." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+protein+coding+sequences+from+human%2C+mouse+and+the+domesticated+pig.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000775-delaTorre1" id="pone.0000775-delaTorre1"></a>de la Torre JE, Egan MG, Katari M, Brenner ED, Stevenson DW, et al.  (2006) ESTimating plant phylogeny: lessons from partitioning. BMC Evol Biol  6: 48.  <ul class="find" data-citedArticleID="980976" data-doi="10.1186/1471-2148-6-48"><li><a href="http://dx.doi.org/10.1186/1471-2148-6-48" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ESTimating+plant+phylogeny%3A+lessons+from+partitioning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ESTimating+plant+phylogeny%3A+lessons+from+partitioning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000775-Bourlat1" id="pone.0000775-Bourlat1"></a>Bourlat SJ, Juliusdottir T, Lowe CJ, Freeman R, Aronowicz J, Kirschner M, Lander ES, Thorndyke M, Nakano H, Kohn AB, Heyland A, Moroz LL, Copley RR, Telford MJ (2006) Deuterostome phylogeny reveals monophyletic chordates and the new phylum Xenoturbellida. Nature  444: 85–88.  <ul class="find" data-citedArticleID="980862" data-doi="10.1038/nature05241"><li><a href="http://dx.doi.org/10.1038/nature05241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deuterostome+phylogeny+reveals+monophyletic+chordates+and+the+new+phylum+Xenoturbellida." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deuterostome+phylogeny+reveals+monophyletic+chordates+and+the+new+phylum+Xenoturbellida.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000775-Hughes1" id="pone.0000775-Hughes1"></a>Hughes J, Longhorn SJ, Papadopoulou A, Theodorides K, de Riva A, Mejia-Chang M, Foster PG, Vogler AP (2006) Dense taxonomic EST sampling and its applications for molecular systematics of the Coleoptera (beetles). Mol Biol Evol  23: 268–278.  <ul class="find" data-citedArticleID="980888"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Dense+taxonomic+EST+sampling+and+its+applications+for+molecular+systematics+of+the+Coleoptera+%28beetles%29.&amp;auth=&amp;atitle=Dense+taxonomic+EST+sampling+and+its+applications+for+molecular+systematics+of+the+Coleoptera+%28beetles%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dense+taxonomic+EST+sampling+and+its+applications+for+molecular+systematics+of+the+Coleoptera+%28beetles%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dense+taxonomic+EST+sampling+and+its+applications+for+molecular+systematics+of+the+Coleoptera+%28beetles%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000775-Benton1" id="pone.0000775-Benton1"></a>Benton MJ, Donoghue PCJ (2006) Paleontological Evidence to Date the Tree of Life. MBE Advance Access published on October 17.   <ul class="find" data-citedArticleID="980860"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Paleontological+Evidence+to+Date+the+Tree+of+Life.&amp;auth=&amp;atitle=Paleontological+Evidence+to+Date+the+Tree+of+Life." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Paleontological+Evidence+to+Date+the+Tree+of+Life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Paleontological+Evidence+to+Date+the+Tree+of+Life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000775-AmrineMadsen1" id="pone.0000775-AmrineMadsen1"></a>Amrine-Madsen H, Scally M, Westerman M, Stanhope MJ, Krajewski C, et al.  (2003) Nuclear gene sequences provide evidence for the monophyly of australidelphian marsupials. Mol Phylogenet Evol.  28: 186–196.  <ul class="find" data-citedArticleID="980848" data-doi="10.1016/s1055-7903(03)00122-2"><li><a href="http://dx.doi.org/10.1016/s1055-7903(03)00122-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+gene+sequences+provide+evidence+for+the+monophyly+of+australidelphian+marsupials." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+gene+sequences+provide+evidence+for+the+monophyly+of+australidelphian+marsupials.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000775-Jansa1" id="pone.0000775-Jansa1"></a>Jansa SA, Forsman JF, Voss RS (2006) Different patterns of selection on the nuclear genes IRBP and DMP-1 affect the efficiency but not the outcome of phylogeny estimation for didelphid marsupials. Mol Phylogenet Evol  38: 363–380.  <ul class="find" data-citedArticleID="980898" data-doi="10.1016/j.ympev.2005.06.007"><li><a href="http://dx.doi.org/10.1016/j.ympev.2005.06.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Different+patterns+of+selection+on+the+nuclear+genes+IRBP+and+DMP-1+affect+the+efficiency+but+not+the+outcome+of+phylogeny+estimation+for+didelphid+marsupials." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Different+patterns+of+selection+on+the+nuclear+genes+IRBP+and+DMP-1+affect+the+efficiency+but+not+the+outcome+of+phylogeny+estimation+for+didelphid+marsupials.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000775-Watson1" id="pone.0000775-Watson1"></a>Watson JD, Hopkins NH, Roberts JW, Steitz AJ, Weiner AM (1965) Molecular biology of the gene. Menlo Park, California: Benjamin/Cummings.   <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0000775-Warrington1" id="pone.0000775-Warrington1"></a>Warrington JA, Nair A, Mahadevappa M, Tsyganskaya M (2000) Comparison of human adult and fetal expression and identification of 535 housekeeping/maintenance genes. Physiol Genomics  2: 143–147.  <ul class="find" data-citedArticleID="980958"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparison+of+human+adult+and+fetal+expression+and+identification+of+535+housekeeping%2Fmaintenance+genes.&amp;auth=&amp;atitle=Comparison+of+human+adult+and+fetal+expression+and+identification+of+535+housekeeping%2Fmaintenance+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+human+adult+and+fetal+expression+and+identification+of+535+housekeeping%2Fmaintenance+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+human+adult+and+fetal+expression+and+identification+of+535+housekeeping%2Fmaintenance+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000775-Hsiao1" id="pone.0000775-Hsiao1"></a>Hsiao LL, Dangond F, Yoshida T, Hong R, Jensen RV, et al.  (2001) A compendium of gene expression in normal human tissues. Physiol Genomics  7: 97–104.  <ul class="find" data-citedArticleID="980884"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+compendium+of+gene+expression+in+normal+human+tissues.&amp;auth=&amp;atitle=A+compendium+of+gene+expression+in+normal+human+tissues." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+compendium+of+gene+expression+in+normal+human+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+compendium+of+gene+expression+in+normal+human+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000775-Zheng1" id="pone.0000775-Zheng1"></a>Zheng XH, Lu F, Wang ZY, Zhong F, Hoover J, et al.  (2005) Using shared genomic synteny and shared protein functions to enhance the identification of orthologous gene pairs. Bioinformatics  21: 703–710.  <ul class="find" data-citedArticleID="980972" data-doi="10.1093/bioinformatics/bti045"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+shared+genomic+synteny+and+shared+protein+functions+to+enhance+the+identification+of+orthologous+gene+pairs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+shared+genomic+synteny+and+shared+protein+functions+to+enhance+the+identification+of+orthologous+gene+pairs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000775-Parkinson1" id="pone.0000775-Parkinson1"></a>Parkinson J, Guiliano DB, Blaxter M (2002) Making sense of EST sequences by CLOBBing them. BMC Bioinformatics 2002  3: 31.  <ul class="find" data-citedArticleID="980930" data-doi="10.1186/1471-2105-3-31"><li><a href="http://dx.doi.org/10.1186/1471-2105-3-31" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Making+sense+of+EST+sequences+by+CLOBBing+them." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Making+sense+of+EST+sequences+by+CLOBBing+them.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000775-Endo1" id="pone.0000775-Endo1"></a>Endo T, Ikeo K, Gojobori T (1996) Large-Scale search for Genes on Which Positive selection May Operate. Mol Biol Evol  13: 685–690.  <ul class="find" data-citedArticleID="980874" data-doi="10.1093/oxfordjournals.molbev.a025629"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025629" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-Scale+search+for+Genes+on+Which+Positive+selection+May+Operate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-Scale+search+for+Genes+on+Which+Positive+selection+May+Operate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000775-Nei1" id="pone.0000775-Nei1"></a>Nei M, Gojobori T (1986) Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions. Mol Biol Evol  3: 418–426.  <ul class="find" data-citedArticleID="980924"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions.&amp;auth=&amp;atitle=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000775-Pie1" id="pone.0000775-Pie1"></a>Pie MR (2006) The Influence of Phylogenetic Uncertainty on the Detection of Positive Darwinian Selection. Mol Biol Evol  23: 2274–2278.  <ul class="find" data-citedArticleID="980934" data-doi="10.1093/molbev/msl116"><li><a href="http://dx.doi.org/10.1093/molbev/msl116" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Influence+of+Phylogenetic+Uncertainty+on+the+Detection+of+Positive+Darwinian+Selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Influence+of+Phylogenetic+Uncertainty+on+the+Detection+of+Positive+Darwinian+Selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000775-The1" id="pone.0000775-The1"></a>The ENCODE Consortium (2007) The ENCODE pilot project: Functional annotation of 1% of the human genome. Nature  14: 799–816.  <ul class="find" data-citedArticleID="980952"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+ENCODE+pilot+project%3A+Functional+annotation+of+1%25+of+the+human+genome.&amp;auth=&amp;atitle=The+ENCODE+pilot+project%3A+Functional+annotation+of+1%25+of+the+human+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ENCODE+pilot+project%3A+Functional+annotation+of+1%25+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ENCODE+pilot+project%3A+Functional+annotation+of+1%25+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000775-Kjer1" id="pone.0000775-Kjer1"></a>Kjer KM, Honeycutt RL (2007) Site specific rates of mitochondrial genomes and the phylogeny of eutheria. BMC Evol Biol  7: 8.  <ul class="find" data-citedArticleID="980906"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Site+specific+rates+of+mitochondrial+genomes+and+the+phylogeny+of+eutheria.&amp;auth=&amp;atitle=Site+specific+rates+of+mitochondrial+genomes+and+the+phylogeny+of+eutheria." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Site+specific+rates+of+mitochondrial+genomes+and+the+phylogeny+of+eutheria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Site+specific+rates+of+mitochondrial+genomes+and+the+phylogeny+of+eutheria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000775-Cannarozzi1" id="pone.0000775-Cannarozzi1"></a>Cannarozzi GM, Schneider A, Gonnet G (2006) A Phylogenomic Study of Human, Dog and Mouse. PLoS Computational Biology, e2.eor doi:10.1371/journal.pcbi.0030002.eor.   <ul class="find" data-citedArticleID="980864" data-doi="10.1371/journal.pcbi.0030002.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0030002.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Phylogenomic+Study+of+Human%2C+Dog+and+Mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Phylogenomic+Study+of+Human%2C+Dog+and+Mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000775-Huttley1" id="pone.0000775-Huttley1"></a>Huttley GA, Wakefield MJ, Easteal S (2007) Rates of genome evolution and branching order from whole genome analysis. Mol Biol Evol In press.   <ul class="find" data-citedArticleID="980892" data-doi="10.1093/molbev/msm094"><li><a href="http://dx.doi.org/10.1093/molbev/msm094" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rates+of+genome+evolution+and+branching+order+from+whole+genome+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rates+of+genome+evolution+and+branching+order+from+whole+genome+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000775-Hughes2" id="pone.0000775-Hughes2"></a>Hughes AL, Friedman R (2007) The Effect of Branch Lengths on Phylogeny: an Empirical Study Using Highly Conserved Orthologs from Mammalian Genomes, Mol Phylogenet Evol doi:10.1016/j.ympev.2007.04.022  <ul class="find" data-citedArticleID="980890" data-doi="10.1016/j.ympev.2007.04.022"><li><a href="http://dx.doi.org/10.1016/j.ympev.2007.04.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Effect+of+Branch+Lengths+on+Phylogeny%3A+an+Empirical+Study+Using+Highly+Conserved+Orthologs+from+Mammalian+Genomes%2C+Mol+Phylogenet+Evol+doi%3A10.1016%2Fj.ympev.2007.04.022" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Effect+of+Branch+Lengths+on+Phylogeny%3A+an+Empirical+Study+Using+Highly+Conserved+Orthologs+from+Mammalian+Genomes%2C+Mol+Phylogenet+Evol+doi%3A10.1016%2Fj.ympev.2007.04.022%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000775-Felsenstein1" id="pone.0000775-Felsenstein1"></a>Felsenstein J (1978) Cases in which parsimony and compatibility methods will be positively misleading. Syst Zool  27: 401–410.  <ul class="find" data-citedArticleID="980876" data-doi="10.2307/2412923"><li><a href="http://dx.doi.org/10.2307/2412923" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cases+in+which+parsimony+and+compatibility+methods+will+be+positively+misleading." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cases+in+which+parsimony+and+compatibility+methods+will+be+positively+misleading.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000775-Sullivan1" id="pone.0000775-Sullivan1"></a>Sullivan J, Swofford DL (1997) Are Guinea pigs rodents? The importance of adequate models in molecular phylogenetics. J Mammal Evol  4: 77–86.  <ul class="find" data-citedArticleID="980948"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Are+Guinea+pigs+rodents%3F+The+importance+of+adequate+models+in+molecular+phylogenetics.&amp;auth=&amp;atitle=Are+Guinea+pigs+rodents%3F+The+importance+of+adequate+models+in+molecular+phylogenetics." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Are+Guinea+pigs+rodents%3F+The+importance+of+adequate+models+in+molecular+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Are+Guinea+pigs+rodents%3F+The+importance+of+adequate+models+in+molecular+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000775-Baurain1" id="pone.0000775-Baurain1"></a>Baurain D, Brinkmann H, Philippe H (2006) Lack of resulution in the animal phylogeny: closely spaced cladogeneses or undetected systematic errors? Mol Biol Evol In press.   <ul class="find" data-citedArticleID="980858" data-doi="10.1093/molbev/msl137"><li><a href="http://dx.doi.org/10.1093/molbev/msl137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lack+of+resulution+in+the+animal+phylogeny%3A+closely+spaced+cladogeneses+or+undetected+systematic+errors%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lack+of+resulution+in+the+animal+phylogeny%3A+closely+spaced+cladogeneses+or+undetected+systematic+errors%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000775-Yang1" id="pone.0000775-Yang1"></a>Yang Z (2006) Computional molecular evolution. Great Britain: Oxford University press.   <ul class="find-nolinks"></ul></li><li><span class="label">48.
              </span><a name="pone.0000775-Chomczynski1" id="pone.0000775-Chomczynski1"></a>Chomczynski P, Sacchi N (1987) Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction. Anal Biochem  162: 156–159.  <ul class="find" data-citedArticleID="980868" data-doi="10.1016/0003-2697(87)90021-2"><li><a href="http://dx.doi.org/10.1016/0003-2697(87)90021-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single-step+method+of+RNA+isolation+by+acid+guanidinium+thiocyanate-phenol-chloroform+extraction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single-step+method+of+RNA+isolation+by+acid+guanidinium+thiocyanate-phenol-chloroform+extraction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000775-NCBI1" id="pone.0000775-NCBI1"></a>NCBI RefSeq database  (<a href="ftp://ftp.ncbi.nih.gov/genomes">ftp://ftp.ncbi.nih.gov/genomes</a>).  <ul class="find-nolinks"></ul></li><li><span class="label">50.
              </span><a name="pone.0000775-ENSEMBL1" id="pone.0000775-ENSEMBL1"></a>ENSEMBL public ftp  (<a href="ftp://ftp.ensembl.org/pub/">ftp://ftp.ensembl.org/pub/</a>).  <ul class="find-nolinks"></ul></li><li><span class="label">51.
              </span><a name="pone.0000775-Thompson1" id="pone.0000775-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties and weight matrix choice. Nucleic Acids Research,  22: 4673–4680.  <ul class="find" data-citedArticleID="980956" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+positions-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+positions-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000775-Rambaut1" id="pone.0000775-Rambaut1"></a>Rambaut A (1996) Se-Alv2.0a11: sequence aignment editor.   Availible from: <a href="http://evolve.zoo.ox.ac.uk/">http://evolve.zoo.ox.ac.uk/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">53.
              </span><a name="pone.0000775-Thomas1" id="pone.0000775-Thomas1"></a>Thomas PD, Campbell MJ, Kejariwal A, Mi H, Karlak B, et al.  (2003) PANTHER: A Library of Protein Families and Subfamilies Indexed by Function. Genome Res  13: 2129–2141.  <ul class="find" data-citedArticleID="980954" data-doi="10.1101/gr.772403"><li><a href="http://dx.doi.org/10.1101/gr.772403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PANTHER%3A+A+Library+of+Protein+Families+and+Subfamilies+Indexed+by+Function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PANTHER%3A+A+Library+of+Protein+Families+and+Subfamilies+Indexed+by+Function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000775-Mi1" id="pone.0000775-Mi1"></a>Mi H, Lazareva-Ulitsky B, Loo R, Kejariwal A, Vandergriff J, et al.  (2005) The PANTHER database of protein families, subfamilies, functions and pathways Nucl. Acids Res  33: D284–288.  <ul class="find" data-citedArticleID="980914" data-doi="10.1093/nar/gki078"><li><a href="http://dx.doi.org/10.1093/nar/gki078" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+PANTHER+database+of+protein+families%2C+subfamilies%2C+functions+and+pathways+Nucl." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+PANTHER+database+of+protein+families%2C+subfamilies%2C+functions+and+pathways+Nucl.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000775-Strimmer1" id="pone.0000775-Strimmer1"></a>Strimmer K, von Haeseler A (1996) Quartet puzzling: a quartet maximum likelihood method for reconstructing tree topologies. Mol Biol Evol  13: 964–969.  <ul class="find" data-citedArticleID="980944" data-doi="10.1093/oxfordjournals.molbev.a025664"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025664" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quartet+puzzling%3A+a+quartet+maximum+likelihood+method+for+reconstructing+tree+topologies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quartet+puzzling%3A+a+quartet+maximum+likelihood+method+for+reconstructing+tree+topologies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000775-Felsenstein2" id="pone.0000775-Felsenstein2"></a>Felsenstein J (1993) PHYLIP (Phylogeny Inference Package) version 3.5c. Distributed by the author. Seattle: Department of Genetics, University of Washington.   <ul class="find-nolinks"></ul></li><li><span class="label">57.
              </span><a name="pone.0000775-Adachi2" id="pone.0000775-Adachi2"></a>Adachi J, Hasegawa M (1996) MOLPHY Version 2.3: Programs for molecular phylogenetics based on maximum likelihood. Computer Science Monographs  28: 1 Inst Stat Mathematics, Tokyo.  <ul class="find" data-citedArticleID="980844"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=MOLPHY+Version+2.3%3A+Programs+for+molecular+phylogenetics+based+on+maximum+likelihood.&amp;auth=&amp;atitle=MOLPHY+Version+2.3%3A+Programs+for+molecular+phylogenetics+based+on+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MOLPHY+Version+2.3%3A+Programs+for+molecular+phylogenetics+based+on+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MOLPHY+Version+2.3%3A+Programs+for+molecular+phylogenetics+based+on+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000775-Huelsenbeck1" id="pone.0000775-Huelsenbeck1"></a>Huelsenbeck JP, Ronquist F (2001) MRBAYES: Bayesian inference of phylogenetic trees. Bioinformatics  17: 754–755.  <ul class="find" data-citedArticleID="980886" data-doi="10.1093/bioinformatics/17.8.754"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.8.754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MRBAYES%3A+Bayesian+inference+of+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MRBAYES%3A+Bayesian+inference+of+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000775-Yang2" id="pone.0000775-Yang2"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum lielihood. Computer Applications in BioSciences  13: 555–556.  <ul class="find" data-citedArticleID="980966" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+lielihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+lielihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000775-Jobb1" id="pone.0000775-Jobb1"></a>Jobb G, von Haeseler A, Strimmer K (2004) TREEFINDER: A powerful graphical analysis environment for molecular phylogenetics. BMC Evol Biol  4: 18.  <ul class="find" data-citedArticleID="980900" data-doi="10.1186/1471-2148-4-18"><li><a href="http://dx.doi.org/10.1186/1471-2148-4-18" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TREEFINDER%3A+A+powerful+graphical+analysis+environment+for+molecular+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TREEFINDER%3A+A+powerful+graphical+analysis+environment+for+molecular+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000775-Guindon1" id="pone.0000775-Guindon1"></a>Guindon S, Gascuel S (2003) A simple, fast and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol  52: 696–704.  <ul class="find" data-citedArticleID="980882"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple%2C+fast+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.&amp;auth=&amp;atitle=A+simple%2C+fast+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+fast+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+fast+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000775-Swofford1" id="pone.0000775-Swofford1"></a>Swofford DL (2002) PAUP* Phylogenetic Analysis using Parsimony(*and other Methods), version 4.0b8. Sunderland, MA: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">63.
              </span><a name="pone.0000775-Posada1" id="pone.0000775-Posada1"></a>Posada D, Crandall KA (1998) Modeltest: testing the model of DNA substitution. Bioinformatics  14: 817–818.  <ul class="find" data-citedArticleID="980936" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeltest%3A+testing+the+model+of+DNA+substitution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeltest%3A+testing+the+model+of+DNA+substitution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000775-Abascal1" id="pone.0000775-Abascal1"></a>Abascal F, Zardoya R, Posada D (2005) ProtTest: selection of the best-fit models of protein evolution. Bioinformatics  7: 2104–2105.  <ul class="find" data-citedArticleID="980840" data-doi="10.1093/bioinformatics/bti263"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ProtTest%3A+selection+of+the+best-fit+models+of+protein+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ProtTest%3A+selection+of+the+best-fit+models+of+protein+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000775-Jones1" id="pone.0000775-Jones1"></a>Jones DT, Taylor WR, Thornton JM (1992) The rapid generation of mutation data matrices from protein sequences. CABIOS  8: 275–282.  <ul class="find" data-citedArticleID="980902" data-doi="10.1093/bioinformatics/8.3.275"><li><a href="http://dx.doi.org/10.1093/bioinformatics/8.3.275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+rapid+generation+of+mutation+data+matrices+from+protein+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+rapid+generation+of+mutation+data+matrices+from+protein+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000775-Lanave1" id="pone.0000775-Lanave1"></a>Lanave C, Preparata G, Saccone C, Serio G (1984) A new method for calculating evolutionary substitution rates. J Mol Evol  20: 86–93.  <ul class="find" data-citedArticleID="980912" data-doi="10.1007/bf02101990"><li><a href="http://dx.doi.org/10.1007/bf02101990" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+method+for+calculating+evolutionary+substitution+rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+method+for+calculating+evolutionary+substitution+rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000775-Gu1" id="pone.0000775-Gu1"></a>Gu X, Fu YX, Li W-H (1995) Maximum likelihood estimation of the heterogeneity of substitution rate among nucleotide sites. Mol Biol Evol  12: 546–557.  <ul class="find" data-citedArticleID="980880"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maximum+likelihood+estimation+of+the+heterogeneity+of+substitution+rate+among+nucleotide+sites.&amp;auth=&amp;atitle=Maximum+likelihood+estimation+of+the+heterogeneity+of+substitution+rate+among+nucleotide+sites." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum+likelihood+estimation+of+the+heterogeneity+of+substitution+rate+among+nucleotide+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum+likelihood+estimation+of+the+heterogeneity+of+substitution+rate+among+nucleotide+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000775-1" id="pone.0000775-1"></a>The Bioportal at University of Oslo.   <a href="http://www.bioportal.uio.no">www.bioportal.uio.no</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">69.
              </span><a name="pone.0000775-Zhang1" id="pone.0000775-Zhang1"></a>Zhang J, Nielsen R, Yang Z (2005) Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level. Mol Biol Evol  22: 2472–2479.  <ul class="find" data-citedArticleID="980970" data-doi="10.1093/molbev/msi237"><li><a href="http://dx.doi.org/10.1093/molbev/msi237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000775-Yang3" id="pone.0000775-Yang3"></a>Yang Z, Wong WS, Nielsen R (2005) Bayes empirical bayes inference of amino acid sites under positive selection. Mol Biol Evol  22: 1107–1118.  <ul class="find" data-citedArticleID="980968" data-doi="10.1093/molbev/msi097"><li><a href="http://dx.doi.org/10.1093/molbev/msi097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayes+empirical+bayes+inference+of+amino+acid+sites+under+positive+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayes+empirical+bayes+inference+of+amino+acid+sites+under+positive+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.XML" value="143307"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.PDF" value="220721"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t001.PNG_L" value="124055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t001.PNG_M" value="95208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t001.PNG_S" value="15080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t001.TIF" value="723348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t001.PNG_I" value="69682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t002.PNG_L" value="57959"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t002.PNG_M" value="77543"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t002.PNG_S" value="12877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t002.TIF" value="235358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t002.PNG_I" value="26706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g001.PNG_L" value="22348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g001.PNG_M" value="21598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g001.PNG_S" value="7657"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g001.TIF" value="50300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g001.PNG_I" value="6603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t003.PNG_L" value="168585"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t003.PNG_M" value="104854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t003.PNG_S" value="13417"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t003.TIF" value="838728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t003.PNG_I" value="32309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g002.PNG_L" value="25068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g002.PNG_M" value="24705"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g002.PNG_S" value="7859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g002.TIF" value="56872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g002.PNG_I" value="7709"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g003.PNG_L" value="19710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g003.PNG_M" value="21021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g003.PNG_S" value="8726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g003.TIF" value="41240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g003.PNG_I" value="8298"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g004.PNG_L" value="41899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g004.PNG_M" value="47378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g004.PNG_S" value="12596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g004.TIF" value="82528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.g004.PNG_I" value="15406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t004.PNG_L" value="116399"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t004.PNG_M" value="175403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t004.PNG_S" value="21637"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t004.TIF" value="501848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t004.PNG_I" value="60116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t005.PNG_L" value="67451"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t005.PNG_M" value="89391"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t005.PNG_S" value="13922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t005.TIF" value="326598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t005.PNG_I" value="31287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t006.PNG_L" value="77014"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t006.PNG_M" value="102518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t006.PNG_S" value="14544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t006.TIF" value="325526"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t006.PNG_I" value="35600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t007.PNG_L" value="280650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t007.PNG_M" value="116649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t007.PNG_S" value="22612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t007.TIF" value="1325164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.t007.PNG_I" value="178628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000775.s001.DOC" value="121856"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000775&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000775" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000775&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000775&volume=&issue=&title=Expressed Sequence Tags as a Tool for Phylogenetic Analysis of Placental Mammal Evolution&author_name=Morgan%20Kullberg%2C%20Bj%C3%B6rn%20Hallstr%C3%B6m%2C%20Ulfur%20Arnason%2C%20Axel%20Janke&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000775" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000775" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000775" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000775&amp;t=Expressed%20Sequence%20Tags%20as%20a%20Tool%20for%20Phylogenetic%20Analysis%20of%20Placental%20Mammal%20Evolution" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000775&title=Expressed%20Sequence%20Tags%20as%20a%20Tool%20for%20Phylogenetic%20Analysis%20of%20Placental%20Mammal%20Evolution&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000775&amp;title=Expressed%20Sequence%20Tags%20as%20a%20Tool%20for%20Phylogenetic%20Analysis%20of%20Placental%20Mammal%20Evolution" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000775" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000775&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Expressed Sequence Tags as a Tool for Phylogenetic Analysis of Placental Mammal Evolution';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000775';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Expressed%20Sequence%20Tags%20as%20a%20Tool%20for%20Phylogenetic%20Analysis%20of%20Placental%20Mammal%20Evolution http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000775" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000775" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">



















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+phylogenetics%22" title="Search for articles in the subject area:'Animal phylogenetics'"><div class="flagText">Animal phylogenetics</div></a>
              <div data-categoryid="20335" data-articleid="24388"
                   data-categoryname="Animal phylogenetics"
                   class="flagImage" title="Flag 'Animal phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Expressed+sequence+tags%22" title="Search for articles in the subject area:'Expressed sequence tags'"><div class="flagText">Expressed sequence tags</div></a>
              <div data-categoryid="34427" data-articleid="24388"
                   data-categoryname="Expressed sequence tags"
                   class="flagImage" title="Flag 'Expressed sequence tags' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24388"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24388"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Rodents%22" title="Search for articles in the subject area:'Rodents'"><div class="flagText">Rodents</div></a>
              <div data-categoryid="42763" data-articleid="24388"
                   data-categoryname="Rodents"
                   class="flagImage" title="Flag 'Rodents' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="24388"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="24388"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="24388"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2678'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3028'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7519&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
