

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Large Scale Gene Expression Profiles of Regenerating Inner Ear Sensory Epithelia</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000525"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000525" />

    <meta name="citation_title" content="Large Scale Gene Expression Profiles of Regenerating Inner Ear Sensory Epithelia"/>
    <meta itemprop="name" content="Large Scale Gene Expression Profiles of Regenerating Inner Ear Sensory Epithelia"/>

      <meta name="citation_author" content="R. David Hawkins"/>
            <meta name="citation_author_institution" content="Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Stavros Bashiardes"/>
            <meta name="citation_author_institution" content="Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Kara E. Powder"/>
            <meta name="citation_author_institution" content="Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Samin A. Sajan"/>
            <meta name="citation_author_institution" content="Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Veena Bhonagiri"/>
            <meta name="citation_author_institution" content="Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="David M. Alvarado"/>
            <meta name="citation_author_institution" content="Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Judith Speck"/>
            <meta name="citation_author_institution" content="Department of Otolaryngology, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Mark E. Warchol"/>
            <meta name="citation_author_institution" content="Department of Otolaryngology, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Michael Lovett"/>
            <meta name="citation_author_institution" content="Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>

    <meta name="citation_date" content="2007/6/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000525.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e525"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=How hearing happens.; citation_author=AJ Hudspeth; citation_journal_title=Neuron; citation_volume=19; citation_number=1; citation_pages=947-950; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Regenerative proliferation in inner ear sensory epithelia from adult guinea pigs and humans.[see comment].; citation_author=ME Warchol; citation_author=PR Lambert; citation_author=BJ Goldstein; citation_author=A Forge; citation_author=JT Corwin; citation_journal_title=Science; citation_volume=259; citation_number=2; citation_pages=1619-1622; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Ultrastructural evidence for hair cell regeneration in the mammalian inner ear.[see comment].; citation_author=A Forge; citation_author=L Li; citation_author=JT Corwin; citation_author=G Nevill; citation_journal_title=Science; citation_volume=259; citation_number=3; citation_pages=1616-1619; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Cell division in the gerbil cochlea after acoustic trauma.; citation_author=DW Roberson; citation_author=EW Rubel; citation_journal_title=American Journal of Otology; citation_volume=15; citation_number=4; citation_pages=28-34; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Induction of cell proliferation in mammalian inner-ear sensory epithelia by transforming growth factor alpha and epidermal growth factor.; citation_author=H Yamashita; citation_author=EC Oesterle; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=92; citation_number=5; citation_pages=3152-3155; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Ultrastructural analysis of [3H]thymidine-labeled cells in the rat utricular macula.; citation_author=EC Oesterle; citation_author=DE Cunningham; citation_author=LE Westrum; citation_author=EW Rubel; citation_journal_title=Journal of Comparative Neurology; citation_volume=463; citation_number=6; citation_pages=177-195; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Math1: an essential gene for the generation of inner ear hair cells.; citation_author=NA Bermingham; citation_author=BA Hassan; citation_author=SD Price; citation_author=MA Vollrath; citation_author=N Ben-Arie; citation_journal_title=Science; citation_volume=284; citation_number=7; citation_pages=1837-1841; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Math1 gene transfer generates new cochlear hair cells in mature guinea pigs in vivo.; citation_author=K Kawamoto; citation_author=S-I Ishimoto; citation_author=R Minoda; citation_author=DE Brough; citation_author=Y Raphael; citation_journal_title=Journal of Neuroscience; citation_volume=23; citation_number=8; citation_pages=4395-4400; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Auditory hair cell replacement and hearing improvement by Atoh1 gene therapy in deaf mammals.[see comment].; citation_author=M Izumikawa; citation_author=R Minoda; citation_author=K Kawamoto; citation_author=KA Abrashkin; citation_author=DL Swiderski; citation_journal_title=Nature Medicine; citation_volume=11; citation_number=9; citation_pages=271-276; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regeneration of sensory hair cells after acoustic trauma.; citation_author=JT Corwin; citation_author=DA Cotanche; citation_journal_title=Science; citation_volume=240; citation_number=10; citation_pages=1772-1774; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Hair cell regeneration after acoustic trauma in adult Coturnix quail.; citation_author=BM Ryals; citation_author=EW Rubel; citation_journal_title=Science; citation_volume=240; citation_number=11; citation_pages=1774-1776; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Delta1 expression during avian hair cell regeneration.; citation_author=JS Stone; citation_author=EW Rubel; citation_journal_title=Development; citation_volume=126; citation_number=12; citation_pages=961-973; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=FGFR3 expression during development and regeneration of the chick inner ear sensory epithelia.; citation_author=O Bermingham-McDonogh; citation_author=JS Stone; citation_author=TA Reh; citation_author=EW Rubel; citation_journal_title=Developmental Biology; citation_volume=238; citation_number=13; citation_pages=247-259; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=cProx1 immunoreactivity distinguishes progenitor cells and predicts hair cell fate during avian hair cell regeneration.; citation_author=JS Stone; citation_author=JL Shang; citation_author=S Tomarev; citation_journal_title=Developmental Dynamics; citation_volume=230; citation_number=14; citation_pages=597-614; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gene expression differences in quiescent versus regenerating hair cells of avian sensory epithelia: implications for human hearing and balance disorders.; citation_author=RD Hawkins; citation_author=S Bashiardes; citation_author=CA Helms; citation_author=L Hu; citation_author=NL Saccone; citation_journal_title=Human Molecular Genetics; citation_volume=12; citation_number=15; citation_pages=1261-1272; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=An ORFeome-based analysis of human transcription factor genes and the construction of a microarray to interrogate their expression.; citation_author=DN Messina; citation_author=J Glasscock; citation_author=W Gish; citation_author=M Lovett; citation_journal_title=Genome Research; citation_volume=14; citation_number=16; citation_pages=2041-2047; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cell density and N-cadherin interactions regulate cell proliferation in the sensory epithelia of the inner ear.; citation_author=ME Warchol; citation_journal_title=Journal of Neuroscience; citation_volume=22; citation_number=17; citation_pages=2607-2616; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Immune cytokines and dexamethasone influence sensory regeneration in the avian vestibular periphery.; citation_author=ME Warchol; citation_journal_title=Journal of Neurocytology; citation_volume=28; citation_number=18; citation_pages=889-900; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Critical signaling events during the aminoglycoside-induced death of sensory hair cells in vitro.; citation_author=JI Matsui; citation_author=JE Gale; citation_author=ME Warchol; citation_journal_title=Journal of Neurobiology; citation_volume=61; citation_number=19; citation_pages=250-266; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Biologically meaningful expression profiling across species using heterologous hybridization to a cDNA microarray.; citation_author=SCP Renn; citation_author=N Aubin-Horth; citation_author=HA Hofmann; citation_journal_title=BMC Genomics; citation_volume=5; citation_number=20; citation_pages=42; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution.[see comment][erratum appears in Nature. 2005 Feb 17;433(7027):777].; citation_author=LW Hillier; citation_author=W Miller; citation_author=E Birney; citation_author=W Warren; citation_author=RC Hardison; citation_journal_title=Nature; citation_volume=432; citation_number=21; citation_pages=695-716; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Lectin from Griffonia simplicifolia identifies an immature-appearing subpopulation of sensory hair cells in the avian utricle.; citation_author=ME Warchol; citation_journal_title=Journal of Neurocytology; citation_volume=30; citation_number=22; citation_pages=253-264; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The avian inner ear. Continuous production of hair cells in vestibular sensory organs, but not in the auditory papilla.; citation_author=JM Jorgensen; citation_author=C Mathiesen; citation_journal_title=Naturwissenschaften; citation_volume=75; citation_number=23; citation_pages=319-320; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Wnt-1 modulates cell-cell adhesion in mammalian cells by stabilizing beta-catenin binding to the cell adhesion protein cadherin.; citation_author=L Hinck; citation_author=WJ Nelson; citation_author=J Papkoff; citation_journal_title=Journal of Cell Biology; citation_volume=124; citation_number=24; citation_pages=729-741; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Cross-talk between peroxisome proliferator-activated receptor (PPAR) alpha and liver X receptor (LXR) in nutritional regulation of fatty acid metabolism. II. LXRs suppress lipid degradation gene promoters through inhibition of PPAR signaling.[see comment].; citation_author=T Ide; citation_author=H Shimano; citation_author=T Yoshikawa; citation_author=N Yahagi; citation_author=M Amemiya-Kudo; citation_journal_title=Molecular Endocrinology; citation_volume=17; citation_number=25; citation_pages=1255-1267; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The role of liver X receptor-alpha in the fatty acid regulation of hepatic gene expression.; citation_author=A Pawar; citation_author=D Botolin; citation_author=DJ Mangelsdorf; citation_author=DB Jump; citation_journal_title=Journal of Biological Chemistry; citation_volume=278; citation_number=26; citation_pages=40736-40743; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetic networks that regulate B lymphopoiesis.; citation_author=KL Medina; citation_author=H Singh; citation_journal_title=Current Opinion in Hematology; citation_volume=12; citation_number=27; citation_pages=203-209; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Contingent gene regulatory networks and B cell fate specification.; citation_author=H Singh; citation_author=KL Medina; citation_author=JMR Pongubala; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=102; citation_number=28; citation_pages=4949-4953; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Characterization of damage and regeneration in cultured avian utricles.; citation_author=JI Matsui; citation_author=EC Oesterle; citation_author=JS Stone; citation_author=EW Rubel; citation_journal_title=Jaro; citation_volume=1; citation_number=29; citation_pages=46-63; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Differential expression of unconventional myosins in apoptotic and regenerating chick hair cells confirms two regeneration mechanisms.; citation_author=LJ Duncan; citation_author=DA Mangiardi; citation_author=JI Matsui; citation_author=JK Anderson; citation_author=K McLaughlin-Williamson; citation_journal_title=J Comp Neurol; citation_volume=499; citation_number=30; citation_pages=691-701; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transforming growth factor-beta inhibits pulmonary surfactant protein B gene transcription through SMAD3 interactions with NKX2.1 and HNF-3 transcription factors.; citation_author=C Li; citation_author=N-L Zhu; citation_author=RC Tan; citation_author=PL Ballard; citation_author=R Derynck; citation_journal_title=Journal of Biological Chemistry; citation_volume=277; citation_number=31; citation_pages=38399-38408; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=PAX8, TITF1, and FOXE1 gene expression patterns during human development: new insights into human thyroid development and thyroid dysgenesis-associated malformations.; citation_author=SS Trueba; citation_author=J Auge; citation_author=G Mattei; citation_author=H Etchevers; citation_author=J Martinovic; citation_journal_title=Journal of Clinical Endocrinology & Metabolism; citation_volume=90; citation_number=32; citation_pages=455-462; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The paired domain-containing factor Pax8 and the homeodomain-containing factor TTF-1 directly interact and synergistically activate transcription.; citation_author=T Di Palma; citation_author=R Nitsch; citation_author=A Mascia; citation_author=L Nitsch; citation_author=R Di Lauro; citation_journal_title=Journal of Biological Chemistry; citation_volume=278; citation_number=33; citation_pages=3395-3402; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=p27(Kip1) links cell proliferation to morphogenesis in the developing organ of Corti.; citation_author=P Chen; citation_author=N Segil; citation_journal_title=Development; citation_volume=126; citation_number=34; citation_pages=1581-1590; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Mammalian cochlear supporting cells can divide and trans-differentiate into hair cells.; citation_author=PM White; citation_author=A Doetzlhofer; citation_author=YS Lee; citation_author=AK Groves; citation_author=N Segil; citation_journal_title=Nature; citation_volume=441; citation_number=35; citation_pages=984-987; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The COP9 signalosome inhibits p27(kip1) degradation and impedes G1-S phase progression via deneddylation of SCF Cul1.; citation_author=X Yang; citation_author=S Menon; citation_author=K Lykke-Andersen; citation_author=T Tsuge; citation_author=X Di; citation_journal_title=Current Biology; citation_volume=12; citation_number=36; citation_pages=667-672; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Deregulated expression of the homeobox gene Cux-1 in transgenic mice results in downregulation of p27(kip1) expression during nephrogenesis, glomerular abnormalities, and multiorgan hyperplasia.; citation_author=AW Ledford; citation_author=JG Brantley; citation_author=G Kemeny; citation_author=TL Foreman; citation_author=SE Quaggin; citation_journal_title=Developmental Biology; citation_volume=245; citation_number=37; citation_pages=157-171; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Tissue-specific regulation of retinal and pituitary precursor cell proliferation.; citation_author=X Li; citation_author=V Perissi; citation_author=F Liu; citation_author=DW Rose; citation_author=MG Rosenfeld; citation_journal_title=Science; citation_volume=297; citation_number=38; citation_pages=1180-1183; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=GeneCluster 2.0: an advanced toolset for bioarray analysis.; citation_author=M Reich; citation_author=K Ohm; citation_author=M Angelo; citation_author=P Tamayo; citation_author=JP Mesirov; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=39; citation_pages=1797-1798; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Interpreting patterns of gene expression with self-organizing maps: methods and application to hematopoietic differentiation.; citation_author=P Tamayo; citation_author=D Slonim; citation_author=J Mesirov; citation_author=Q Zhu; citation_author=S Kitareewan; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=96; citation_number=40; citation_pages=2907-2912; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Mutation in transcription factor POU4F3 associated with inherited progressive hearing loss in humans.[see comment].; citation_author=O Vahava; citation_author=R Morell; citation_author=ED Lynch; citation_author=S Weiss; citation_author=ME Kagan; citation_journal_title=Science; citation_volume=279; citation_number=41; citation_pages=1950-1954; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Functional analysis of four naturally occurring variants of human constitutive androstane receptor.; citation_author=S Ikeda; citation_author=K Kurose; citation_author=H Jinno; citation_author=K Sai; citation_author=S Ozawa; citation_journal_title=Molecular Genetics & Metabolism; citation_volume=86; citation_number=42; citation_pages=314-319; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Functional constraints on the constitutive androstane receptor inferred from human sequence variation and cross-species comparisons.; citation_author=EE Thompson; citation_author=H Kuttab-Boulos; citation_author=MD Krasowski; citation_author=A Di Rienzo; citation_journal_title=Human Genomics; citation_volume=2; citation_number=43; citation_pages=168-178; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Six3 activation of Pax6 expression is essential for mammalian lens induction and specification.; citation_author=W Liu; citation_author=OV Lagutin; citation_author=M Mende; citation_author=A Streit; citation_author=G Oliver; citation_journal_title=EMBO Journal; citation_volume=25; citation_number=44; citation_pages=5383-5395; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Signal transduction pathways leading to insulin-induced early gene induction.; citation_author=BH Jhun; citation_author=T Haruta; citation_author=JL Meinkoth; citation_author=W Leitner; citation_author=B Draznin; citation_journal_title=Biochemistry; citation_volume=34; citation_number=45; citation_pages=7996-8004; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning and characterization of NF-IL3A, a transcriptional activator of the human interleukin-3 promoter.; citation_author=W Zhang; citation_author=J Zhang; citation_author=M Kornuc; citation_author=K Kwan; citation_author=R Frank; citation_journal_title=Molecular & Cellular Biology; citation_volume=15; citation_number=46; citation_pages=6055-6063; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Development of inner ear afferent connections: forming primary neurons and connecting them to the developing sensory epithelia.; citation_author=B Fritzsch; citation_journal_title=Brain Research Bulletin; citation_volume=60; citation_number=47; citation_pages=423-433; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide expression dynamics during mouse embryonic development reveal similarities to Drosophila development.; citation_author=RA Wagner; citation_author=R Tabibiazar; citation_author=A Liao; citation_author=T Quertermous; citation_journal_title=Developmental Biology; citation_volume=288; citation_number=48; citation_pages=595-611; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Signal transduction and the control of gene expression.; citation_author=AH Brivanlou; citation_author=JE Darnell; citation_journal_title=Science; citation_volume=295; citation_number=49; citation_pages=813-818; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Activation of CAAT enhancer-binding protein delta (C/EBPdelta) by interleukin-1 negatively influences apolipoprotein C-III expression.; citation_author=JM Lacorte; citation_author=E Ktistaki; citation_author=A Beigneux; citation_author=VI Zannis; citation_author=J Chambaz; citation_journal_title=Journal of Biological Chemistry; citation_volume=272; citation_number=50; citation_pages=23578-23584; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Foxp1 regulates cardiac outflow tract, endocardial cushion morphogenesis and myocyte proliferation and maturation.; citation_author=B Wang; citation_author=J Weidenfeld; citation_author=MM Lu; citation_author=S Maika; citation_author=WA Kuziel; citation_journal_title=Development; citation_volume=131; citation_number=51; citation_pages=4477-4487; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome-wide mapping of Polycomb target genes unravels their roles in cell fate transitions.; citation_author=AP Bracken; citation_author=N Dietrich; citation_author=D Pasini; citation_author=KH Hansen; citation_author=K Helin; citation_journal_title=Genes & Development; citation_volume=20; citation_number=52; citation_pages=1123-1136; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Polycomb group gene Ezh2 prevents hematopoietic stem cell exhaustion.; citation_author=LM Kamminga; citation_author=LV Bystrykh; citation_author=A de Boer; citation_author=S Houwer; citation_author=J Douma; citation_journal_title=Blood; citation_volume=107; citation_number=53; citation_pages=2170-2179; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Polycomb group protein EZH2 directly controls DNA methylation.[see comment].; citation_author=E Vire; citation_author=C Brenner; citation_author=R Deplus; citation_author=L Blanchon; citation_author=M Fraga; citation_journal_title=Nature; citation_volume=439; citation_number=54; citation_pages=871-874; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mouse polycomb proteins bind differentially to methylated histone H3 and RNA and are enriched in facultative heterochromatin.; citation_author=E Bernstein; citation_author=EM Duncan; citation_author=O Masui; citation_author=J Gil; citation_author=E Heard; citation_journal_title=Molecular & Cellular Biology; citation_volume=26; citation_number=55; citation_pages=2560-2569; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Polycomb silencing mechanisms and the management of genomic programmes.; citation_author=YB Schwartz; citation_author=V Pirrotta; citation_journal_title=Nature Reviews Genetics; citation_volume=8; citation_number=56; citation_pages=9-22; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Estrogen receptors in the normal adult and developing human inner ear and in Turner's syndrome.; citation_author=AE Stenberg; citation_author=H Wang; citation_author=J Fish; citation_author=A Schrott-Fischer; citation_author=L Sahlin; citation_journal_title=Hearing Research; citation_volume=157; citation_number=57; citation_pages=87-92; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Distinct roles of unliganded and liganded estrogen receptors in transcriptional repression.; citation_author=A Cvoro; citation_author=C Tzagarakis-Foster; citation_author=D Tatomer; citation_author=S Paruthiyil; citation_author=MS Fox; citation_journal_title=Molecular Cell; citation_volume=21; citation_number=58; citation_pages=555-564; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Estrogen receptor and breast cancer.; citation_author=S Sommer; citation_author=SA Fuqua; citation_journal_title=Seminars in Cancer Biology; citation_volume=11; citation_number=59; citation_pages=339-352; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Circadian clocks are seeing the systems biology light.; citation_author=KR Hayes; citation_author=JE Baggs; citation_author=JB Hogenesch; citation_journal_title=Genome Biology; citation_volume=6; citation_number=60; citation_pages=219; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Circadian clock genes as modulators of sensitivity to genotoxic stress.; citation_author=MP Antoch; citation_author=RV Kondratov; citation_author=JS Takahashi; citation_journal_title=Cell Cycle; citation_volume=4; citation_number=61; citation_pages=901-907; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The polycomb group protein EZH2 is required for mammalian circadian clock function.; citation_author=J-P Etchegaray; citation_author=X Yang; citation_author=JP DeBruyne; citation_author=AHFM Peters; citation_author=DR Weaver; citation_journal_title=Journal of Biological Chemistry; citation_volume=281; citation_number=62; citation_pages=21209-21215; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Regenerative proliferation in organ cultures of the avian cochlea: identification of the initial progenitors and determination of the latency of the proliferative response.; citation_author=ME Warchol; citation_author=JT Corwin; citation_journal_title=Journal of Neuroscience; citation_volume=16; citation_number=63; citation_pages=5466-5477; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The modulation of cell adhesion molecule expression and intercellular junction formation in the developing avian inner ear.; citation_author=Y Raphael; citation_author=T Volk; citation_author=KL Crossin; citation_author=GM Edelman; citation_author=B Geiger; citation_journal_title=Developmental Biology; citation_volume=128; citation_number=64; citation_pages=222-235; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Characterization of supporting cell phenotype in the avian inner ear: Implications for sensory regeneration.; citation_author=ME Warchol; citation_journal_title=Hearing Research; citation_volume=(in press); citation_number=65; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000525" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Large Scale Gene Expression Profiles of Regenerating Inner Ear Sensory Epithelia"/>
    <meta name="twitter:description" content="Loss of inner ear sensory hair cells (HC) is a leading cause of human hearing loss and balance disorders. Unlike mammals, many lower vertebrates can regenerate these cells. We used cross-species microarrays to examine this process in the avian inner ear. Specifically, changes in expression of over 1700 transcription factor (TF) genes were investigated in hair cells of auditory and vestibular organs following treatment with two different damaging agents and regeneration in vitro. Multiple components of seven distinct known signaling pathways were clearly identifiable: TGF&beta;, PAX, NOTCH, WNT, NFKappaB, INSULIN/IGF1 and AP1. Numerous components of apoptotic and cell cycle control pathways were differentially expressed, including p27KIP and TFs that regulate its expression. A comparison of expression trends across tissues and treatments revealed identical patterns of expression that occurred at identical times during regenerative proliferation. Network analysis of the patterns of gene expression in this large dataset also revealed the additional presence of many components (and possible network interactions) of estrogen receptor signaling, circadian rhythm genes and parts of the polycomb complex (among others). Equal numbers of differentially expressed genes were identified that have not yet been placed into any known pathway. Specific time points and tissues also exhibited interesting differences: For example, 45 zinc finger genes were specifically up-regulated at later stages of cochlear regeneration. These results are the first of their kind and should provide the starting point for more detailed investigations of the role of these many pathways in HC recovery, and for a description of their possible interactions."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000525.g005"/>

  <meta property="og:title" content="Large Scale Gene Expression Profiles of Regenerating Inner Ear Sensory Epithelia" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000525" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5625'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9069'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8546&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000525">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000525" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000525&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Large Scale Gene Expression Profiles of Regenerating Inner Ear Sensory Epithelia
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            R. David Hawkins
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: R. David Hawkins, Stavros Bashiardes</p>

              
              

                <p>Affiliation:
                  Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stavros Bashiardes
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: R. David Hawkins, Stavros Bashiardes</p>

              
              

                <p>Affiliation:
                  Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kara E. Powder, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Samin A. Sajan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Veena Bhonagiri, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David M. Alvarado, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Judith Speck, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Otolaryngology, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark E. Warchol, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Otolaryngology, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael Lovett
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:lovett@genetics.wustl.edu">lovett@genetics.wustl.edu</a></p>

                <p>Affiliation:
                  Division of Human Genetics, Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 13, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000525</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000525" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000525" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000525" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000525" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000525-t001" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000525-g001" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000525-t002" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000525-g002" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000525-g003" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000525-g004" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000525-g005" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000525">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000525" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Loss of inner ear sensory hair cells (HC) is a leading cause of human hearing loss and balance disorders. Unlike mammals, many lower vertebrates can regenerate these cells. We used cross-species microarrays to examine this process in the avian inner ear. Specifically, changes in expression of over 1700 transcription factor (TF) genes were investigated in hair cells of auditory and vestibular organs following treatment with two different damaging agents and regeneration <em>in vitro</em>. Multiple components of seven distinct known signaling pathways were clearly identifiable: <em>TGFβ, PAX, NOTCH, WNT, NFKappaB, INSULIN/IGF1</em> and <em>AP1</em>. Numerous components of apoptotic and cell cycle control pathways were differentially expressed, including <em>p27<sup>KIP</sup></em> and TFs that regulate its expression. A comparison of expression trends across tissues and treatments revealed identical patterns of expression that occurred at identical times during regenerative proliferation. Network analysis of the patterns of gene expression in this large dataset also revealed the additional presence of many components (and possible network interactions) of estrogen receptor signaling, circadian rhythm genes and parts of the polycomb complex (among others). Equal numbers of differentially expressed genes were identified that have not yet been placed into any known pathway. Specific time points and tissues also exhibited interesting differences: For example, 45 zinc finger genes were specifically up-regulated at later stages of cochlear regeneration. These results are the first of their kind and should provide the starting point for more detailed investigations of the role of these many pathways in HC recovery, and for a description of their possible interactions.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hawkins RD, Bashiardes S, Powder KE, Sajan SA, Bhonagiri V, et al.  (2007) Large Scale Gene Expression Profiles of Regenerating Inner Ear Sensory Epithelia. PLoS ONE 2(6):
          e525.
            doi:10.1371/journal.pone.0000525</p><p><strong>Academic Editor: </strong>Rodolfo Aramayo, Texas A&amp;M University, United States of America</p><p><strong>Received:</strong> December 19, 2006; <strong>Accepted:</strong> May 23, 2007; <strong>Published:</strong> June 13, 2007</p><p><strong>Copyright:</strong> © 2007 Hawkins et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a major grant from the National Organization for Hearing Research Foundation (to ML and MW) and by RO1-DC05632 (ML), R01 DC006283 and P30 DC04665 (MW) and by training grant 5T32HG000045 (supporting DMA) and training grant 5T32GM007067 (supporting KEP).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The human cochlea possesses approximately 16,000 sensory hair cells (HCs), which are necessary for normal hearing <a href="#pone.0000525-Hudspeth1">[1]</a>. These cells are mechanoreceptors that detect sound, head movements and acceleration, and provide input into the auditory and vestibular branches of the eighth cranial nerve. Hair cells in the mammalian ear are produced during a fixed period of embryonic development, and can be lost later in life as a consequence of acoustic trauma, exposure to ototoxic drugs or inner ear infections. The mammalian vestibular organs possess a very limited ability for sensory regeneration <a href="#pone.0000525-Warchol1">[2]</a>, <a href="#pone.0000525-Forge1">[3]</a> and the mammalian cochlea is completely unable to regenerate hair cells <a href="#pone.0000525-Roberson1">[4]</a>. As a result, the loss of sensory cells from the mammalian ear can result in permanent deficits in hearing and balance. Such disorders are very common; so-called ‘sensorineural’ hearing loss afflicts about 30 million Americans <a href="#pone.0000525-Hudspeth1">[1]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Sensory epithelia (SE) in the ears of all vertebrates are populated by two basic cell types: hair cells and supporting cells. The key limitation to regeneration in the mammalian ear is the inability of mammalian supporting cells to proliferate or change phenotype in response to hair cell injury. Several attempts have been made to induce regeneration in the vestibular organs by exposure to exogenous mitogens, but these have had limited success <a href="#pone.0000525-Yamashita1">[5]</a> <a href="#pone.0000525-Oesterle1">[6]</a>. More recently, viral gene transfer techniques have been used to transfect supporting cells in the damaged cochlea with the gene encoding the <em>Atoh1</em> transcription factor. This factor, formerly known as <em>Math1</em>, is required for normal hair cell differentiation <a href="#pone.0000525-Bermingham1">[7]</a> and also appears to be capable of converting the phenotype of mature supporting cells into replacement hair cells. These new sensory receptors can also re-establish synaptic connections with afferent neurons, leading to a moderate restoration of hearing in a guinea pig model of hearing loss <a href="#pone.0000525-Kawamoto1">[8]</a> <a href="#pone.0000525-Izumikawa1">[9]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Unlike mammals, spontaneous regeneration of sensory hair cells has been extensively documented in the hearing and balance organs of non-mammalian vertebrates. The avian inner ear, in particular, has a very robust capacity for sensory regeneration. Death of hair cells quickly triggers renewed proliferation of epithelial supporting cells, and the progeny of these divisions can differentiate as replacement hair cells and supporting cells <a href="#pone.0000525-Corwin1">[10]</a>, <a href="#pone.0000525-Ryals1">[11]</a>. These observations have led to the hope that understanding the biological basis of sensory regeneration in the avian ear might provide insights into how to induce similar repair in humans. Relatively little is known about the genetic regulatory mechanisms that permit regeneration in the avian ear and this has limited the development of strategies for inducing such regeneration in mammals. Previous studies of the avian regenerative process have identified genes on an individual (and essentially “candidate gene”) basis (e.g. <em>DELTA1</em>-<a href="#pone.0000525-Stone1">[12]</a>; <em>FGFR3</em>-<a href="#pone.0000525-BerminghamMcDonogh1">[13]</a>; <em>PROX1</em>-<a href="#pone.0000525-Stone2">[14]</a>).</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In this study we employed two genomic technologies to obtain a more global picture of gene expression changes during avian hair cell regeneration. Specifically, we used micro cDNA amplification methods and custom gene microarrays <a href="#pone.0000525-Hawkins1">[15]</a> <a href="#pone.0000525-Messina1">[16]</a> to interrogate nearly all identified transcription factor (TF) genes in the SE of the chick cochlea and utricle during the early phases of regeneration. We identified activity within known signaling pathways, several of which have not been previously implicated or explored in SE regeneration, and we identified many as-yet unexplored genes that were differentially expressed. We were able to identify treatment, tissue and time point-specific changes in TF expression. Notably, a basal set of TFs was present in both sensory tissues at all stages. We also identified commonalities and tissue-specific differences between the sensory epithelia of the utricle and cochlea that resulted from different damage regimes. This is the first study of its kind to characterize large-scale gene expression profiles during sensory regeneration. These results now provide the starting point for a detailed investigation of the role of these genes in HC recovery, and for a description of their possible interactions.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Study design</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We profiled changes in transcription factor expression in SE of the cochlea and utricle following two distinct forms of <em>in vitro</em> injury: (1) laser ‘wounding’ of cultured SE or; (2) ototoxic hair cell death caused by treatment with the aminoglycoside antibiotic neomycin. In the first case, cultured SE <a href="#pone.0000525-Warchol2">[17]</a> received linear ‘wounds’ with a pulsed laser microbeam. Creation of the lesion typically required 3–5 min/culture; during this time, control cultures were removed from the incubator and kept under identical conditions, but did not receive lesions. Wounded epithelia were allowed to recover for 30 min, 1 hr, 2 hrs or 3 hrs after the lesions. Equal numbers of lesioned and unlesioned specimens were analyzed at each recovery time point. For the second injury regimen, utricles or cochleae were cultured for 24 hr in medium that contained 1 mM neomycin <a href="#pone.0000525-Warchol3">[18]</a>. A sample of SE was collected immediately after this treatment; this constituted the 0 hr time point for the regenerative time course. Other cultures were rinsed and maintained in neomycin-free medium for an additional 24 or 48 hr. Equal numbers of specimens were cultured under identical conditions, but did not receive neomycin; these served as time-matched controls for comparative gene expression profiling.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Hybridizations were conducted on multiple biological samples (see below). The sampling times for neomycin-lesioned specimens were chosen based upon previous studies of hair cell death and supporting cell proliferation, which indicate that S-phase entry by supporting cells peaks at about 48 hrs after ototoxic injury <em>in vitro</em> <a href="#pone.0000525-Matsui1">[19]</a>. On the other hand, the laser time points were expected to provide us with insights into the very early signaling events that occur after epithelial injury. Our expectation in employing these two damage regimes was that we might find some degree of overlap between the two time courses, but that we might also be able to discern treatment-specific, as well as cochlea- and utricle-specific, changes in gene expression.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>After appropriate survival times, RNA was prepared and converted into microarray targets by previously described methods <a href="#pone.0000525-Hawkins1">[15]</a>. TF gene expression was then assayed by comparative hybridization (injured specimens vs. time-matched controls) to custom transcription factor gene microarrays <a href="#pone.0000525-Messina1">[16]</a>. These microarrays contained 50 mer oligonucleotide probes, spotted in duplicate, that interrogate the vast majority of human transcription factor genes (plus a few probes to non-TFs such as sonic hedgehog [SHH]). We used this array as a cross-species profiling tool to measure the expression of the orthologous chicken TF genes. We, and others, have previously demonstrated that cross-species hybridizations can be reliably used on this type of array platform <a href="#pone.0000525-Hawkins1">[15]</a> <a href="#pone.0000525-Renn1">[20]</a>. The study described here was embarked upon before the recent publication of the draft chicken genomic DNA sequence <a href="#pone.0000525-Hillier1">[21]</a> or the availability of commercial chicken gene chips. With the release of most of the chicken genomic DNA sequence it is possible to assess sequence identity between our human probes and their chicken orthologs. An analysis of this type indicates that ~98% of our probes have &gt;70% sequence identity with the correct chicken ortholog (data not shown). Hybridization stringencies were adjusted to accommodate this degree of identity for all of the comparative data described in this study. Each time course was replicated with additional biological samples, including controls. Every treatment time point and time-matched control was hybridized to a minimum of four microarrays; two replicates plus two dye switch experiments. All of our data, array designs and analysis parameters are available through the NCBI Gene Expression Omnibus (GEO; <a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>).</p>


<h4>Array analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In order to quantify gene expression changes, along with associated statistical confidence limits, all expression data were analyzed as described below (see <a href="#s4">Materials and Methods</a>). Briefly, array data were first normalized by LOWESS, a locally weighted linear regression model, to compensate for dye effects. Second, data from the multiple hybridizations for a given time point were hierarchically clustered, along with data from additional time points, to assess the similarity and reproducibility of the data across multiple biological samples and dye switches. T-tests (calculated by conducting a one sample t-test on the adjusted intensity data of the entire array of selected hybridizations at each time point) were performed between samples to assess reproducibility and similarity. The data were also filtered to remove oligonucleotide probes that fell below a threshold for background intensity. This threshold was determined by the intensity of controls spots. During the creation of self-organizing maps (see below), not all genes passed the filtering steps in both time courses. In these cases we extracted the missing values from the primary data and “filled in” the values to construct the patterns of gene expression across all seven time points. In general the vast majority of TFs showed relatively modest gene expression fold changes. This may be due to a compression of the dynamic range in cross-species hybridizations <a href="#pone.0000525-Hawkins1">[15]</a>. Our prior experience in employing this array platform for cross-species hybridizations indicated that changes as low as 1.2-fold frequently reflected higher changes when assessed by q-PCR (<a href="#pone.0000525-Hawkins1">[15]</a>and see q-PCR below).</p>


<h4>Differential gene expression in the four time courses</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In the antibiotic damage regime the 24 and 48 hr time points reflected gene expression changes within supporting cells, as the majority of hair cells had been killed by the ototoxic antibiotic <a href="#pone.0000525-Warchol3">[18]</a>, <a href="#pone.0000525-Warchol4">[22]</a>. By 48 hrs many of the supporting cells had progressed into the S-phase of the cell cycle <a href="#pone.0000525-Matsui1">[19]</a>. By contrast, the laser damage regime resulted in a 100–200 µm-wide ‘wound’ in the cultured sensory epithelia. The wounds typically closed within 16–24 hrs of recovery time. The initial phase of wound repair was due to cell migration, but elevated levels of cell proliferation were also observed at the wound sites (but not at distant, uninjured regions) at 16–48 hrs after injury. For the utricle, after the data analysis steps described above, 143 TFs had passed through the data filters for differential gene expression (&gt;1.2-fold change at one or more time point and a p-value of &lt;0.05) over the three neomycin damage time points (Supplemental <a href="#pone.0000525.s002">Tables S1</a> and <a href="#pone.0000525.s003">S2</a>). Gene expression in laser damaged SE was compared to time-matched controls at 30 min, 1 hr, 2 hrs, and 3 hrs after laser lesions. For the utricle, a total of 66 TFs were differentially expressed across the four laser time points (Supplemental <a href="#pone.0000525.s004">Tables S3</a> and <a href="#pone.0000525.s005">S4</a>).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Analysis of the cochlear treatments revealed a much larger number of significant changes in TF gene expression than were found for the utricle. A total of 484 genes were differentially expressed (&gt;1.2-fold change and p-value of &lt;0.05) across the cochlear neomycin time course (Supplemental <a href="#pone.0000525.s006">Tables S5</a> and <a href="#pone.0000525.s007">S6</a>). Analysis of the cochlear laser comparisons revealed a total of 217 differentially expressed genes (listed in Supplemental <a href="#pone.0000525.s008">Tables S7</a> and <a href="#pone.0000525.s009">S8</a>). Overall, when overlaps between the various lists of genes were taken into account, a total of 605 TFs accounted for all of the statistically significant changes in gene expression observed across the two cochlear time courses, and a total of 188 TF genes were differentially expressed across the two utricle time courses. It is possible that these apparent differences in numbers of differentially expressed genes between the two epithelia, reflect more synchronization of regenerative signaling events in the cochlea when compared to undamaged controls. It is notable that the undamaged avian utricle is in a continual low-level state of hair cell turn-over <a href="#pone.0000525-Jorgensen1">[23]</a>. This process may result in asynchronies in gene expression between injured and uninjured utricles. This might lower apparent fold-changes or increase variability (leading to higher p-values) when the damaged utricles are compared to the undamaged (but constantly regenerating) utricles. It is also possible that the larger number of expression changes in the cochlea reflects a more robust regenerative program in this particular sensory epithelia.</p>


<h4>Validation of expression changes</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To independently verify apparent differences in gene expression we selected 11 genes that showed modest fold-changes for at least one time point within at least one of the four regenerative time courses. The orthologous chicken genes were identified by BLAST searches and PCR primers were designed for use in qPCR assays on new biological samples. In most cases these assays were conducted upon tissues from single time points. However, for three genes (<em>CTNNB1</em>, <em>KIAA0173</em> and <em>TRIP15</em>) the genes were tested across one entire time course (the utricle neomycin time course). A total of 19 time point/tissue/treatment combinations were tested overall. The results of these multiple qPCR assays are shown in <a href="#pone-0000525-t001">Table 1</a> and compared to the apparent fold-changes observed from the array analysis. As observed in our previous study with this type of cross-species array platform <a href="#pone.0000525-Hawkins1">[15]</a>, these data confirmed the trends seen in the microarray data and indicated that the microarray, in general, under-reported the actual levels of changes that occurred across the four time courses. The only example showing &gt;15% deviation from this under reporting was the level of <em>CTNNB1</em> mRNA in the utricle at 48 hrs after neomycin damage. In this case, the q-PCR assay reported an increase of 2.3 fold relative to undamaged utricle, whereas the array reported a 3.1-fold increase.</p>
<div class="figure" id="pone-0000525-t001"><div class="img"><a name="pone-0000525-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000525.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000525.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>QPCR validation of microarray fold-changes.</span></strong></p><span>doi:10.1371/journal.pone.0000525.t001</span></div>

<h4>Comparing and contrasting differential gene expression across tissues and treatments</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Despite the large differences in the sheer volume of changes in the two regenerating epithelia, there were interesting underlying overlaps between the various transcriptional programs. These are shown in summary form as the Venn diagrams in <a href="#pone-0000525-g001">Figure 1</a>. For example, a comparison of all changes in gene expression between both neomycin time courses (utricle and cochlea) revealed 80 shared genes, of which 21 showed identical trends in at least one identical time point. These time point commonalities are shown in <a href="#pone-0000525-t002">Table 2</a> and in a more expanded form in Supplementary <a href="#pone.0000525.s010">Table S9</a>. The largest area of overlap between the cochlea and utricle was at 48 hrs where 11 genes showed identical patterns of gene expression (<a href="#pone-0000525-t002">Table 2</a>). Four of these were down-regulated and seven were up-regulated versus the time-matched controls, but literature searches and pathway analysis did not reveal any known unifying functional network that ties these together. Among this group were; <em>CTNNB1</em> (a component of canonical <em>WNT</em> signaling <a href="#pone.0000525-Hinck1">[24]</a>) <em>NR1H3</em> (a.k.a. <em>LXRalpha</em>), an orphan nuclear receptor that is known to play a role in insulin regulation of cholesterol homeostasis in hepatocytes <a href="#pone.0000525-Ide1">[25]</a>, <a href="#pone.0000525-Pawar1">[26]</a> and; <em>BCL11A</em> (a zinc finger gene associated with various hematopoietic malignancies <a href="#pone.0000525-Medina1">[27]</a>, <a href="#pone.0000525-Singh1">[28]</a>). Antibiotic damage in the avian utricle results in a program of cell division that peaks at 48 hrs after ototoxic injury <a href="#pone.0000525-Matsui2">[29]</a> and returns to normal levels at 4 days post injury. A recent study <a href="#pone.0000525-Duncan1">[30]</a> also indicates that markers of hair cell differentiation are expressed in the SE at &gt;80 hrs after ototoxic injury. Therefore, the pattern of expression of these 11 genes may mark a period at which the proliferative program is declining and the differentiation program is initiating within the regenerating SE.</p>
<div class="figure" id="pone-0000525-g001"><div class="img"><a name="pone-0000525-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000525.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000525.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Overlaps between differentially expressed genes in the four treatment/time course combinations.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>These Venn diagrams illustrate the overlaps between genes that were scored as being either up or down regulated. To be included as an overlap a gene must be differentially expressed in at least one time point in both time courses (&gt;1.2-fold and p-value&lt; = 0.05), but the time points or trends do not have to necessarily match. For a detailed listing of all of the genes in the overlaps see Supplementary <a href="#pone.0000525.s003">Tables S2</a>, <a href="#pone.0000525.s005">S4</a>, <a href="#pone.0000525.s007">S6</a> and <a href="#pone.0000525.s009">S8</a>).</p>
<span>doi:10.1371/journal.pone.0000525.g001</span></div><div class="figure" id="pone-0000525-t002"><div class="img"><a name="pone-0000525-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000525.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000525.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Genes that exhibit identical patterns of expression in at least one identical time-point.</span></strong></p><span>doi:10.1371/journal.pone.0000525.t002</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>A similar set of comparisons were conducted on all the differentially expressed TFs in the laser time courses (<a href="#pone-0000525-g001">Figure 1</a> and <a href="#pone-0000525-t002">Table 2</a>). These shared 27 differentially expressed genes, of which 12 showed the same trends for at least one parallel time point (<a href="#pone-0000525-t001">Table 1</a>). These changes might either reflect laser-specific responses or be early-acting effects in the common pathway of hair cell regeneration. It is interesting to note that the 3 hr laser time points shared two up-regulated genes in common with the 48 hr neomycin time points (<em>BCL11A</em> and another zinc finger gene, <em>ZNF248</em>). The last laser time points also showed up-regulation of five additional genes [<em>JUND, TTLL4</em> (a.k.a. <em>KIAA0173</em>) <em>HOXD8</em> and <em>ZNF75A</em>].</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>A comparison of trends within a given SE (irrespective of the method of injury) revealed 5 genes (<em>CEBPG, HSF1, LOC51131, PSMC5</em> and <em>SSX4</em>) that appear to reflect regenerative responses specific to the utricle (i.e. they were not differentially expressed in the regenerating cochlea SE). A similar comparison of trends in the cochlea identified 90 genes that occurred in both laser and neomycin lists of differentially expressed genes (<a href="#pone-0000525-g001">Figure 1</a>). In this case, 61 of these gene expression changes appeared to be specific to the regenerating cochlea (cf Supplementary <a href="#pone.0000525.s003">Tables S2</a>, <a href="#pone.0000525.s005">S4</a>, <a href="#pone.0000525.s007">S6</a> and <a href="#pone.0000525.s009">S8</a>). Among these were two forkhead genes (<em>FOXC1</em> and <em>FOXH1</em>) three homeobox genes (<em>HOXA4, HOXC10</em> and <em>HOXD1</em>) five zinc finger transcription factor genes (<em>ZFHX1b, ZFP106, ZNF123, ZNF175</em> and <em>ZNF258</em>) and three genes from the circadian rhythm pathway (<em>PER1, PER2</em> and <em>TIMELESS</em>). Several of these will be discussed in more detail below. None of these genes has been previously investigated in the regenerating or developing inner ear.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Notably, we also identified a core group of six genes [<em>BCL11A, HOXD8, TTLL4, ZNF248</em> (a.k.a. <em>LOC57209</em>), <em>NR1H3</em>, and <em>COPS2</em> (a.k.a. <em>TRIP15</em>)] that change their expression levels across all four treatments and tissue combinations. It should be noted that although these genes vary across all of the time courses, they do not necessarily follow exactly parallel patterns of gene expression across time courses. Thus, <em>COPS2</em> was initially down regulated in both neomycin time courses, was up-regulated at 1 hr after laser treatment in the utricle, but was down-regulated at 3 hrs after laser treatment in the cochlea. By contrast, some genes such as <em>BCL11A,</em> appeared quite consistent between the four combinations; <em>BCL11A</em> expression peaked at 3hrs in both laser time courses and peaked at 48 hrs in both neomycin time courses.</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>Beyond these various overlaps between treatments and time courses, the largest group of gene expression changes in each of the four treatment/time point combinations were specific to just one SE and one treatment. For example, changes in the expression of 323 TF genes were specific to cochlea SE that was recovering from antibiotic damage (<a href="#pone-0000525-g001">Figure 1</a>). Many of these changes (for example the large group of zinc finger genes mentioned below) peaked at the 48 hour time point of cochlear SE regeneration.</p>


<h4>Identification of known pathways and processes among the differentially expressed genes</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The comparative data listed in Supplementary <a href="#pone.0000525.s002">Tables S1</a>, <a href="#pone.0000525.s003">S2</a>, <a href="#pone.0000525.s004">S3</a>, <a href="#pone.0000525.s005">S4</a>, <a href="#pone.0000525.s006">S5</a>, <a href="#pone.0000525.s007">S6</a>, <a href="#pone.0000525.s008">S7</a>, <a href="#pone.0000525.s009">S8</a>, <a href="#pone.0000525.s010">S9</a>, <a href="#pone.0000525.s011">S10</a>, <a href="#pone.0000525.s012">S11</a>, <a href="#pone.0000525.s013">S12</a>, <a href="#pone.0000525.s014">S13</a>, <a href="#pone.0000525.s015">S14</a>, <a href="#pone.0000525.s016">S15</a>, <a href="#pone.0000525.s017">S16</a>, <a href="#pone.0000525.s018">S17</a> and <a href="#pone.0000525.s019">S18</a> were manually curated via interrogation of Gene Ontology databases as well as Medline literature citations. This served to identify multiple components or “signatures” of seven distinct signaling pathways within all four regenerative time courses. The identified pathways were those previously shown to be mediated by; <em>TGFβ, PAX, NOTCH, WNT, NFKappaB, Insulin/IGF1</em>, and <em>AP1</em> signaling. All of these have been implicated, in one way or another, in the normal development of the vertebrate inner ear. These signatures (along with their time course(s) and associated references to each component) are listed in Supplementary <a href="#pone.0000525.s011">Tables S10</a>, <a href="#pone.0000525.s012">S11</a>, <a href="#pone.0000525.s013">S12</a>, <a href="#pone.0000525.s014">S13</a>, <a href="#pone.0000525.s015">S14</a>, <a href="#pone.0000525.s016">S15</a>, <a href="#pone.0000525.s017">S16</a>. Again, as with the common genes described above, even within one identified pathway, the profiles of changes in each time course were frequently quite different. Nevertheless, some commonalities could be discerned; for example, the homeobox gene <em>TITF1/NKX2.1</em> (a component of both the <em>TGFβ</em> and <em>PAX</em> pathways) which interacts with both <em>SMAD3</em> and <em>PAX8</em> <a href="#pone.0000525-Li1">[31]</a>–<a href="#pone.0000525-DiPalma1">[33]</a> showed a similar profile in both neomycin time courses.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Not surprisingly, an additional grouping of genes fell within a set that we termed cell cycle/apoptosis genes (listed as Supplementary <a href="#pone.0000525.s018">Table S17</a>). Of interest among this set of genes were three that have been implicated in the regulation of <em>p27<sup>KIP</sup></em>, a cyclin dependent kinase inhibitor that is a key regulator of cell proliferation during cochlear development <a href="#pone.0000525-Chen1">[34]</a>. Although <em>p27<sup>KIP</sup></em> is expressed in supporting cells and may act as a block to cellular proliferation <a href="#pone.0000525-White1">[35]</a>, a probe for this gene was not included on our array. Therefore, we conducted a semi-quantitative PCR analysis of the chicken <em>p27<sup>KIP</sup></em> gene in the utricule neomycin specimens. This is shown in <a href="#pone-0000525-g002">Figure 2</a> and indicates that <em>p27<sup>KIP</sup></em> transcription was down-regulated after utricle SE damage and then returned to normal levels by 48 hrs after the removal of the antibiotic. <a href="#pone-0000525-g002">Figure 2</a> also shows microarray data for four other genes that have been previously shown to regulate <em>p27<sup>KIP</sup></em>. These are: <em>COPS2</em>, a component of the <em>COP9</em> signalosome <a href="#pone.0000525-Yang1">[36]</a>, that can inhibit G1-S transition through interactions with <em>p27<sup>KIP</sup></em>; <em>CUTL1</em> a transcription factor that inhibits <em>p27<sup>KIP</sup></em> transcription <a href="#pone.0000525-Ledford1">[37]</a>; <em>SIX6</em> within the <em>PAX</em> pathway which also represses <em>p27<sup>KIP</sup></em> transcription <a href="#pone.0000525-Li2">[38]</a>; and <em>DACH1</em> (a component of both the <em>PAX</em> and <em>TGF</em>–pathways) which interacts with <em>SIX6</em> to repress <em>p27<sup>KIP</sup></em> transcription <a href="#pone.0000525-Li2">[38]</a>. It is interesting to note that for the <em>COPS2</em> and <em>SIX6</em> genes the microarray data were consistent with their previously described interactions with <em>p27<sup>KIP</sup></em> (i.e. <em>SIX6</em> transcript levels decreased over the time course and <em>COPS2</em> levels initially declined and then increased). <em>CUTL1</em> (a putative repressor of <em>p27<sup>KIP</sup></em>) also appeared to increase in expression level over the time course and <em>DACH1</em> transcript levels did not significantly vary through the time course. This set of five genes is just one example of the many changes in known pathway components that can be constructed into mechanistic and testable hypotheses from this dataset.</p>
<div class="figure" id="pone-0000525-g002"><div class="img"><a name="pone-0000525-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000525.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000525.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Gene expression changes in <em>p27<sup>Kip</sup></em> and four genes that may regulate its expression.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>This diagram shows a combination of semi-quantitative PCR data (for <em>p27<sup>Kip</sup>)</em> and microarray data for the other four genes conducted on the utricle neomycin time courses. Each gene expression profile is color coded with the key to the right of each figure. The X-axis lists time points and the Y-axis is the log<sub>2</sub> fold-change at each time point. Expression values are derived from Supplementary <a href="#pone.0000525.s003">Table S2</a>, except for <em>DACH1</em> which is detectably expressed, but is not significantly differentially expressed across the time course.</p>
<span>doi:10.1371/journal.pone.0000525.g002</span></div>

<h4>Clustering with self organizing maps</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>As described above, literature/database searches plus manual curation of the data assisted us in placing a total of 70 known TFs into possible interactive pathways. However, the vast majority of the TFs in our set have no known function or correlations with known pathways. In order to potentially identify these relationships and to better discern possible patterns of co-expression within these data, we derived self-organizing maps (SOMs) by combining all differentially expressed genes across both time courses for each tissue type. This form of unsupervised clustering (<a href="#pone.0000525-Reich1">[39]</a>, <a href="#pone.0000525-Tamayo1">[40]</a>) produces clusters of genes (with upper and lower limit bars) that show similar patterns of expression across a time course or set of treatments. In this case the situation is somewhat artificial, since in building these graphs we made the arbitrary choice that the 3 hr laser time point would precede the neomycin zero time point changes on the X-axis, whereas in reality the laser time course probably overlaps the early stages (0 hr to 24 hr) of neomycin recovery. Nevertheless, the purpose of these clusters was to visualize apparent patterns and potential clusters of genes within the data. <a href="#pone-0000525-g003">Figure 3A</a> shows a group of 16 SOM centroids (clusters of genes that show similar patterns of differential expression across all the time points) constructed using Genecluster 2 <a href="#pone.0000525-Reich1">[39]</a>, <a href="#pone.0000525-Tamayo1">[40]</a> for the utricle time courses. <a href="#pone-0000525-g003">Figure 3B</a> shows sixteen centroids for the cochlea data. The actual names of every gene in each cluster are listed in Supplementary <a href="#pone.0000525.s019">Tables S18</a> and <a href="#pone.0000525.s020">S19</a>. Some clusters exhibited relatively large temporal fluctuations in gene expression across both time courses. One example of this is centroid 3 in <a href="#pone-0000525-g003">Figure 3A</a> which includes a total of 14 genes such as <em>CEBPG, JUND, FOXP1</em>, and <em>HOXA13</em>. By contrast, centroids 8 and 12 in <a href="#pone-0000525-g003">Figure 3A</a> illustrate genes that show relatively small changes in expression, except at the 48 hour neomycin time point where they were all up regulated. These were the predominantly late genes in the utricle regenerative time course. These two centroids together comprised 19 genes and included <em>POU4F3</em> (previously implicated in hearing loss <a href="#pone.0000525-Vahava1">[41]</a>), <em>CTNNB1</em> and <em>PPARGC1</em> (both in the <em>WNT</em> pathway). At the other end of the spectrum were the 11 genes in centroids 0 and 4 of <a href="#pone-0000525-g003">Figure 3A</a> that appeared to be activated early and peak in expression at the first or second laser time point. Among these are the nuclear hormone receptor <em>NR1I3,</em> which plays a role in transcriptional activation of genes involved in drug metabolism <a href="#pone.0000525-Ikeda1">[42]</a>, <a href="#pone.0000525-Thompson1">[43]</a> <em>SIX3</em>, a homeobox gene that regulates <em>PAX6</em> and <em>SOX2</em> in the developing eye <a href="#pone.0000525-Liu1">[44]</a> and <em>LOC51637</em>, a TF of unknown function, that we previously found to be up-regulated in the chicken utricle <a href="#pone.0000525-Hawkins1">[15]</a> relative to the cochlea.</p>
<div class="figure" id="pone-0000525-g003"><div class="img"><a name="pone-0000525-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000525.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000525.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Analysis of the datasets by Self Organizing Maps.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>All of the differentially expressed genes listed in Supplementary <a href="#pone.0000525.s003">Tables S2</a>, <a href="#pone.0000525.s005">S4</a>, <a href="#pone.0000525.s007">S6</a> and <a href="#pone.0000525.s009">S8</a> were uploaded into Genecluster 2 and 16 centroids per organ were generated. Each box (centroid) in this figure is numbered from C0–C15 and they reflect common patterns of expression for clustered groups of genes within the dataset. The X-axis for each centroid consists of each time point and runs from the laser 30 min time point through 1 hr, 2 hrs, 3 hrs and then into the neomycin 0 time point followed by the neomycin 24 and 48 hr time points. The Y-axis indicates expression level (fold-change). The number in the top left of each centroid indicates the number of genes that fall into this cluster of co-expression. The top line indicates the upper boundary of expression for all of these genes and the lower line indicates the lower boundary. The middle line is the mean. All of the specific genes that fall into each centroid are listed in Supplementary <a href="#pone.0000525.s019">Tables S18</a> and <a href="#pone.0000525.s020">S19</a>. <a href="#pone-0000525-g003">Figure 3A</a> shows the clustering for the utricle time points and <a href="#pone-0000525-g003">Figure 3B</a> shows the clustering for cochlea time points. Arrows indicate various patterns or genes within specific centroids that are described in the text.</p>
<span>doi:10.1371/journal.pone.0000525.g003</span></div><a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>The cochlea regenerative SOMs (16 in total) are shown in <a href="#pone-0000525-g003">Figure 3B</a>. In this case the predominantly late genes fall into centroids 11 and 14 and total 118 genes. However, additional examples of gradual up-regulation occur in centroids 10 and 15 (55 additional genes). Interestingly, of the 118 genes in centroids 11 and 14, a total of 45 are zinc finger transcription factors (as defined by being either ZF or ZNF family members listed in Supplementary <a href="#pone.0000525.s020">Table S19</a>). The vast majority of these are of unknown function and unknown target specificity. If the genes in centroids 10 and 15 are included, the total number of zinc finger TFs peaking in expression at the 48Hr time point rises to 61 (35% of the 173 total genes in these centroids). By contrast, the other twelve centroids in <a href="#pone-0000525-g003">Figure 3B</a> all together contain 19 zinc finger transcription factors (4% of a total of 432 genes in these centroids). Therefore, it appears that a dramatic burst of zinc finger gene expression occurs specifically at these late stages of regenerartive proliferation in the cochlear SE. This contrasts with the utricle SOMs where zinc finger TFs are distributed fairly evenly through the centroids. In common with the utricle time courses, <em>CTNNB1</em> peaks at 48 hrs in the cochlear time courses but, unlike in the utricle, <em>POU4F3</em> peaks earlier, at the 24Hr time point (in centroid 4 of <a href="#pone-0000525-g003">Figure 3B</a>). The predominately early genes (19 in total) in <a href="#pone-0000525-g003">Figure 3B</a> are contained within centroid 3. Of interest within this group are <em>EGR1</em>, which can be induced by <em>IGF</em> signaling <a href="#pone.0000525-Jhun1">[45]</a>, <em>NFIL3</em> which is a nuclear factor regulated by <em>IL3</em> <a href="#pone.0000525-Zhang1">[46]</a>, <a href="#pone.0000525-Fritzsch1">[47]</a> and Neurogenin 1, which is involved in fate choice during inner ear development <a href="#pone.0000525-Fritzsch1">[47]</a>.</p>


<h4>Contrasting patterns of TF genes that are detectably expressed</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>In addition to using the normalized intensity values to identify differentially expressed genes, we also used intensity values to determine which TFs were detectably expressed at any given time point, irrespective of any fold-change. This is a useful dataset since, at the level of detection of our microarrays, it defines lists of TFs that specify the normal functioning of the SE and makes no distinction between genes that never vary and those that change in their expression levels. This involved scoring all genes as “on” that reproducibly exceeded a background intensity level (and likewise any gene that failed to meet this cutoff was arbitrarily scored as “off”). This cut-off was based upon control oligonucleotides that were imbedded within our arrays and have no known homologous sequences in the chicken genome. Venn diagrams (<a href="#pone-0000525-g004">Figures 4A and 4B</a>) illustrate the results of this analysis. It is important to realize the differences between this analysis and the listings of differentially expressed genes in Supplemental <a href="#pone.0000525.s002">tables S1</a> through <a href="#pone.0000525.s009">S8</a>. A gene such as <em>CEBPG</em> is among those that are differentially expressed in both the utricle laser and neomycin time courses (Supplementary <a href="#pone.0000525.s002">Tables S1</a>, <a href="#pone.0000525.s003">S2</a>, <a href="#pone.0000525.s004">S3</a> and <a href="#pone.0000525.s005">S4</a>). However, in the Venn diagrams this gene is scored as being detectably expressed at all time points (albeit at different levels between them). In <a href="#pone-0000525-g004">Figure 4A</a> it therefore falls among the 367 genes that are commonly present in all time-ponts in the neomycin Venn diagram and the 535 common genes in the laser Venn diagram (<a href="#pone-0000525-g004">Figure 4A</a>). Within these two sets of common genes (that are apparently on in either the neomycin or laser time courses) there are 256 that are shared (Supplemental <a href="#pone.0000525.s021">Tables S20</a> and <a href="#pone.0000525.s022">S21</a>). These comprise a core group of expressed TF genes for the sensory epithelium of the utricle, irrespective of time point or treatment. Likewise, the cochlea has a core group of 346 TF genes that are common to both time courses at all time points. There are also a group of 134 genes that are detectably expressed at all times in all four time courses. Additionally, the Venn diagrams identify many genes “uniquely” detectable at individual time points. In some cases these may overlap with those scored as being differentially expressed, or they may only just exceed the background threshold level at those particular time points. This analysis also indicates that the largest number of detectably expressed genes occurs at 0 and 48 hr in the Neomycin time course and at the 1 hr time point in the laser time course. A complete listing of the gene overlaps summarized in <a href="#pone-0000525-g004">Figures 4A and 4B</a> is provided in Supplemental <a href="#pone.0000525.s021">Tables S20</a> and <a href="#pone.0000525.s022">S21</a>.</p>
<div class="figure" id="pone-0000525-g004"><div class="img"><a name="pone-0000525-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000525.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000525.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Detectably expressed TFs in the four treatment/time course combinations.</span></strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>All TFs that were present (as judged by exceeding a background intensity level) at any stage were considered in this analysis, irrespective of differential expression at any time point. Overlaps between these sets of TFs are illustrated in these Venn diagrams. <a href="#pone-0000525-g004">Figure 4A</a> shows overlaps for the utricle time points and treatments. <a href="#pone-0000525-g004">Figure 4B</a> shows overlaps for the cochlea time points and treatments. For a detailed listing of all of the genes in these diagrams see Supplementary <a href="#pone.0000525.s021">Tables S20</a> and <a href="#pone.0000525.s022">S21</a>.</p>
<span>doi:10.1371/journal.pone.0000525.g004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this study we report the first large-scale analysis of changes in gene expression during the process of avian hair cell regeneration. We have detected the signatures of seven known signaling pathways in our data, and changes in expression levels of genes common to particular time courses and treatments. In general, we observed modest fold-changes in TF gene expression across all of the time courses and treatments. This is not unexpected for this class of genes, even when the array platform is within a species (rather than the type of cross-species platform used in this study). For example, in a recent study of gene expression changes of ~25,000 genes in mouse organogenesis (from embryonic day 8.0 to postnatal day 1 <a href="#pone.0000525-Wagner1">[48]</a>), a total of 160 TF genes changed by &gt;1.2-fold in their expression level. In that particular study, the average TF change was 1.63-fold and the range was from a 3.66-fold down-regulation to a 3.63-fold up-regulation. This is remarkably similar to our observations in the current report and it presumably reflects the fact that small changes in these potent transcriptional regulators can have large downstream effects. One should also consider that some TFs are constitutively expressed and their activation is mediated by other events, such as phosphorylation (reviewed in <a href="#pone.0000525-Brivanlou1">[49]</a>). In that regard, it is interesting to note that in our experiments we also detected changes in the expression levels of many of these genes whose protein products are regulated by phosphorylation events. For example, <em>JUND, CEBPG</em> and <em>CEBPB</em> are known to be activated by phosphorylation <a href="#pone.0000525-Brivanlou1">[49]</a>, <a href="#pone.0000525-Lacorte1">[50]</a> and yet we observed clear and reproducible changes in transcript levels for many of these types of TF genes.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Most of the changes we observed have not yet been linked into known networks or pathways. These genes should provide a rich future source of clues into the genetic programming of avian hair cell regeneration and SE development in general. Our descriptions of pathways and trends presented here is just the beginning of a systems biology approach to this important process. Within our data set there are numerous examples of individual, differentially expressed genes that have not previously been implicated in inner ear development or differentiation. For example, the <em>FOXP1</em> gene has previously been implicated in cardiac development <a href="#pone.0000525-Wang1">[51]</a> but from our data appears to be rapidly up-regulated early in the utricle laser time courses (see Supplementary <a href="#pone.0000525.s004">Tables S3</a> and <a href="#pone.0000525.s005">S4</a>). We have recently confirmed that this gene is also differentially expressed during embryonic development of the mouse vestibular organs that give rise to these SE (Sajan, Warchol and Lovett in preparation).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>There are also changes in particular classes of genes that may reflect underlying important pathways. One such example is the change in expression of the Polycomb complex genes <em>EZH1</em>, <em>EZH2</em> (enhancer of zeste 1 and 2) <em>CBX1, CBX3, CBX4, CBX6,</em> and <em>CBX8</em> (chromobox genes) that occur in the regenerative time courses in both the cochlea and utricle SE (Supplementary <a href="#pone.0000525.s002">Table S1</a>, <a href="#pone.0000525.s003">S2</a>, <a href="#pone.0000525.s004">S3</a>, <a href="#pone.0000525.s005">S4</a>, <a href="#pone.0000525.s006">S5</a>, <a href="#pone.0000525.s007">S6</a>, <a href="#pone.0000525.s008">S7</a> and <a href="#pone.0000525.s009">S8</a>). These genes are especially noteworthy because of their roles in controlling cell fate decisions <a href="#pone.0000525-Bracken1">[52]</a> and in preventing stem cell exhaustion <a href="#pone.0000525-Kamminga1">[53]</a> via epigenetic mechanisms <a href="#pone.0000525-Vire1">[54]</a>–<a href="#pone.0000525-Schwartz1">[56]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>A more global method for interrogating the data presented in this report is to make use of web-delivered tools to discover possible networks or canonical pathways. Ingenuity Pathways Analysis (IPA; Ingenuity® Systems, <a href="http://www.ingenuity.com">www.ingenuity.com</a>) is one such set of tools. We uploaded the specific sets of shared genes (fold-changes and p-values) listed in Supplementary <a href="#pone.0000525.s002">Tables S1</a>, <a href="#pone.0000525.s003">S2</a>, <a href="#pone.0000525.s004">S3</a>, <a href="#pone.0000525.s005">S4</a>, <a href="#pone.0000525.s006">S5</a>, <a href="#pone.0000525.s007">S6</a>, <a href="#pone.0000525.s008">S7</a> and <a href="#pone.0000525.s009">S8</a> into the IPA application. These genes were then used to generate biological networks developed form information contained in the Ingenuity Pathways Knowledge Base (IPKB). All connections within the IPKB are supported by at least one reference from the literature (see <a href="http://www.ingenuity.com">www.ingenuity.com</a>). IPA also computed a p-value for each generated network derived from a right-tailed Fisher's exact test, which indicates the probability that the focus genes in a network are found together because of chance alone. A complete description of all of these networks is beyond the scope of the current study. However, it is interesting to note that one of the highest scoring networks (p-value of 2.4×10<sup>−6</sup>) shared between both the cochlea and utricle regenerating SE, involves multiple components of estrogen receptor (ER) signaling. <a href="#pone-0000525-g005">Figure 5A</a> shows an IPA network of these various ER components (analyzed separately from other differentially expressed genes). Previous studies have commented on the presence of estrogen receptor in the developing mammalian inner ear <a href="#pone.0000525-Stenberg1">[57]</a>, but it is not clear what endogenous ligand(s) might activate this pathway or whether this acts via the ligand-independent route <a href="#pone.0000525-Cvoro1">[58]</a>. It is known that ligand–independent activation of ER can be achieved by ER phosphorylation mediated by various signaling pathways<a href="#pone.0000525-Sommer1">[59]</a>.</p>
<div class="figure" id="pone-0000525-g005"><div class="img"><a name="pone-0000525-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000525" data-uri="info:doi/10.1371/journal.pone.0000525.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000525.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000525.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000525.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000525.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000525.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Two examples of Ingenuity gene networks constructed from cochlear differentially expressed genes.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Genes that showed differential expression in both the laser and neomycin cochlear time courses (listed in Supplementary <a href="#pone.0000525.s007">Tables S6</a> and <a href="#pone.0000525.s009">S8</a>) were uploaded to the web-based Ingenuity program (Ingenuity® Systems, <a href="http://www.ingenuity.com">www.ingenuity.com</a>) and the network of interactions shown here was generated. Each interaction is shown according to the following legend and is supported by at least one literature citation (available from the Ingenuity website). <a href="#pone-0000525-g005">Figure 5A</a> shows the network of interactions for genes specifically identified within the cochlear neomycin time course as being part of Estrogen receptor signaling. <a href="#pone-0000525-g005">Figure 5B</a> shows the network of interactions surrounding Circadian rhythm signaling and was generated by uploading all of the cochlear differentially expressed genes (rather than a subset as in 5A). Red denotes up-regulation and green down-regulation in at least one time point. Genes shown in bold with no shading vary across a time course (e.g. <em>GTF2H4</em> in <a href="#pone-0000525-g005">Figure 5A</a> was up-regulated at 24 hrs and down-regulated at 48 hrs). All other genes were either not represented on the microarray or were not significantly differentially expressed. A key to additional Ingenuity labels is listed below.</p>
<span>doi:10.1371/journal.pone.0000525.g005</span></div><a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>An additional example of IPA network building involves the significant enrichment (p-value of 5×10<sup>−7</sup>) in the cochlear regenerating epithelia for genes involved in regulation of circadian rhythm (e.g. <em>BHLHB3, PER1, CREB1, PER2, TIMELESS, CLOCK</em>). This network has not previously been implicated in sensory epithelia regeneration or differentiation. These genes are classically thought of as regulating 24 hour periodicities in gene expression (reviewed in <a href="#pone.0000525-Hayes1">[60]</a>) although they may play roles in other processes, such as genotoxic stress <a href="#pone.0000525-Antoch1">[61]</a>. <a href="#pone-0000525-g005">Figure 5B</a> shows an IPA network for the cochlear differentially expressed genes (in this case, from an analysis of all differentially expressed cochlear genes). This indicates that both circadian rhythm and estrogen receptor signaling may converge within this network. It is also interesting to note that the circadian rhythm pathway may interact with the aforementioned Polycomb complex <em>via EZH2</em> activity, since the mammalian <em>EZH2</em> gene product binds to the <em>PER1</em> and <em>PER2</em> promoters <a href="#pone.0000525-Etchegaray1">[62]</a> .</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>These examples serve to illustrate some of the routes that these data now open up for the further understanding of this complex network of interactions. Together these changes in gene expression lead to the proliferation of supporting cells and the eventual repopulation of the inner ear hair cells. However, unraveling which of these candidate genes are necessary and sufficient for the process will require more direct tests of their effects; such as RNAi treatments, gene knock-out and gene knock-in technologies in both the avian and the mammalian inner ear.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Material and Methods"></a><h3>Material and Methods</h3>
<h4>Culture of sensory organs and isolated sensory epithelia</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>White Leghorn chicks (10–21 days post hatch) were euthanized by CO<sub>2</sub> asphyxiation and decapitated. Cochlea and utricles were isolated as described previously <a href="#pone.0000525-Hawkins1">[15]</a>. Detailed methods for the preparation of organ cultures and cultures of isolated SE have also been reported previously (cochlea: <a href="#pone.0000525-Warchol5">[63]</a>; utricle: <a href="#pone.0000525-Warchol3">[18]</a>; isolated epithelia: <a href="#pone.0000525-Warchol2">[17]</a>).</p>


<h4>Ototoxic hair cell injury</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Cochleae and utricles were placed in small culture wells (Mat-Tek; 1 cm diameter; 1 specimen/well) that contained 100 µm of Medium-199 (with Earles salts, 25 mM HEPES; 2,200 mg/L sodium bicarbonate, and 0.69 mM l-glutamine; Invitrogen), supplemented with 10% fetal bovine serum (FBS; Invitrogen). Half of the specimens also received 1 mM neomycin sulfate (Sigma), in order to lesion sensory hair cells <a href="#pone.0000525-Warchol1">[2]</a>, <a href="#pone.0000525-Warchol3">[18]</a>. Other specimens were cultured under identical conditions, expect that they did not receive neomycin; these served as time-matched controls. Utricles and cochleae were maintained under these conditions for 24 hours. At this point, some specimens were harvested as described below; these formed the 0 hr recovery group. Other specimens were rinsed 3× with fresh Medium-199/10%FBS and maintained in culture for an additional 24 or 48 hr. After appropriate survival times, specimens were rinsed with fresh Medium-199 and incubated for 60 min in 500 µg/ml thermolysin (Sigma; prepared in Medium-199), at 37°C. The sensory organs were then transferred to chilled Medium-199 and the sensory epithelia (consisting of sheets of hair cells and supporting cells) were separated from the associated stromal tissue using a 27-gauge needle. Isolated epithelia were dissolved in 100 µl Trizol and stored at −80°C until further processing. Each biological sample consisted of four neomycin-treated and four time-matched control epithelia at each survival time.</p>


<h4>Laser microbeam ablations</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>SE from the cochlea or utricle were isolated using thermolysin <a href="#pone.0000525-Warchol2">[17]</a>, cut into small pieces, and grown for 7–10 days on fibronectin-coated wells (Mat-Tek) that contained 50 µl Medium-199/10%FBS. Initial immunocytochemical studies confirmed the purity of the epithelial cultures. Cell-cell junctions in the sensory epithelia of the avian cochlea and utricle are mediated by the adhesion molecule N-cadherin, while junctions between cells from the surrounding nonsensory tissue are mediated by E-cadherin <a href="#pone.0000525-Warchol2">[17]</a>, <a href="#pone.0000525-Raphael1">[64]</a>, <a href="#pone.0000525-Warchol6">[65]</a>. Incubation of epithelial cultures with an antibody directed against chick N-cadherin (NCD-2; Zymed) resulted in strong immunoreactivity at nearly all cell-cell junctions (Supplementary <a href="#pone.0000525.s001">Figure S1</a>). This result (which is consistent with previous findings <a href="#pone.0000525-Warchol2">[17]</a>) indicates that the cultures are almost entirely comprised of cells from the sensory epithelia and contain very few extrasensory cells.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Once the cultures were semi-confluent, they were lesioned via laser microsurgery <a href="#pone.0000525-Warchol5">[63]</a>. Individual cultures were placed on the stage of an inverted microscope (Zeiss Axiovert 135) that was equipped with a pulsed laser microbeam device (Photonics Instruments). The beam was focused through a 100× objective lens at a place that was coincident with the cultured cells and the pulse controller was adjusted to run at ~10 pulses/sec. The specimen was then moved slowly through the beam path, using the microscope stage manipulators, resulting in the creation of a linear ‘wound’ in the confluent epithelium. Typically, 2–3 wounds were made in each culture, separated by 100–200 µm. Unlesioned (control) cultures were also removed from the incubator and were placed on the side of the microscope stage during the lesioning of their time-matched counterparts. The lesioned and unlesioned cultures were then returned to the incubator and maintained for an additional 30 min, 1 hr, 2 hr, or 3 hr. Four individual lesioned and unlesioned specimens constituted one biological sample at each of the four recovery times. RNA from the cultures was obtained by addition of 100 µl of Trizol (Invitrogen) to each well. Samples were then stored at −80°C until further processing.</p>


<h4>cDNA amplification</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Isolation of RNA, cDNA synthesis and amplification were carried out as previously described <a href="#pone.0000525-Hawkins1">[15]</a> with the following modifications: following the second round of amplification and Sephadex G50 minicolumn purification, cDNA was added to an <em>in vitro</em> transcription kit (Megascript T7 high yield kit, Ambion) as per the manufacturers instructions. Run-off RNAs were LiCl precipitated, washed with 75% ethanol, dried and resuspended in water at a concentration of 0.5–1 µg/µl. The overall yield and quality of run-off products were assessed by gel electrophoresis.</p>


<h4>Target labeling and microarray hybridizations</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Run-off RNAs from each sample were used as templates in an oligo dT<sub>12–15</sub> primed cDNA synthesis reaction that included amino-allyl dUTP (Sigma, 0.2 mM). The cDNA was then either coupled to Cy3 or Cy5 mono ester dyes (Amersham Pharmacia). Labeled cDNA populations were precipitated and resuspended in 20 µl hybridization buffer (50% formamide, 6× SSPE, 5× Denhart's, 0.5% SDS. 10% dextran sulfate). Microarray slides were hybridized at 37°C for 12 h and were washed as described in <a href="#pone.0000525-Hawkins1">[15]</a>.</p>


<h4>Microarray Design</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The microarrray employed in this study consisted of 50 mer oligonucleotides designed to the vast majority of human transcription factor genes (Messina et al 2004). These probes were designed from coding regions and have previously been shown to accurately report on the majority of chicken TFs when used under appropriate hybridization conditions <a href="#pone.0000525-Hawkins1">[15]</a>. The array also contained probes for a limited number of non-TFs (for example, sonic hedgehog [SHH]). Oligonucleotides were spotted in duplicate on glass slides. The array also contained a number of control 50 mers that have no homolog in the human (or chicken) genome. These were used to measure array sensitivity and to assess background correction levels.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>At least two biological samples (and frequently three or more) were analyzed per time point, treatment and tissue. For every time point to control comparison we conducted at least four separate microarray hybridizations (two comparisons and two dye switches). We also compared each experimental time point to the next experimental time point and control time points to adjacent control time point hybridizations. A total of 226 array comparisons were conducted for this study. These data are available through <a href="http://www.ncbi.nlm.nih.gov/geo/with">http://www.ncbi.nlm.nih.gov/geo/with</a> accession numbers GPL4384, GSE5991, GSE6050, GSE6051 and GSE6052. These data comply with the “minimum information about a microarray experiment” (MIAME) requirements.</p>


<h4>Data Analysis</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Microarray images were analyzed with the BioDiscovery Imagene software package. The Cy3 and Cy5 images were computationally overlaid, aligned and gridded. The intensity of each spot was then measured by laser scanning. The raw intensity files generated by Imagene were analyzed by MIDAS2.17 and TMEV2.2 programs of TIGR software (<a href="http://www.tigr.org">http://www.tigr.org</a>) and GeneCluster 2.0 (<a href="http://www.broad.mit.edu/cancer/software/genecluster2/gc2.html">http://www.broad.mit.edu/cancer/software​/genecluster2/gc2.html</a>). Together, these provide functions for normalizing and analyzing the data. Data were normalized by implementing locally weighted linear regression (LOWESS). After data adjustment, Hierarchical Clustering (HCL) of the data was performed using TIGR TMEV. HCL was implemented to filter out poor hybridizations. Low intensity filtering was performed to exclude genes with intensities lower than the specified thresholds. An arbitrary cut off (usually in the range of 800–1200) was chosen based on the control spot intensities (see Microarray design above). This value was also based on the distribution of intensity values of the array. Approximately 10–20% of the genes were removed in this way. We next selected genes that followed the same trend in at least 80% of the replicated hybridizations. The average fold change was then calculated by averaging the values across the replicate hybridizations. Since the genes identified by these methods (as being differentially expressed) did not necessarily have expression ratios at all the time points (some were filtered out at one stage, but were retained at another) we manually extracted the adjusted intensity values at these time points and calculated the ratios from the corresponding hybridization data. Such “filled-in” expression ratios allowed us to analyze the expression of all the selected genes across the entire time course for a particular treatment. P-values were calculated by conducting a one sample t-test on the adjusted intensity data of the entire array of selected hybridizations at each time point.</p>


<h4>Quantitative PCR (qPCR)</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Validation of microarray fold changes was assessed by qPCR as previously described <a href="#pone.0000525-Hawkins1">[15]</a> using a Stratagene MX3000P machine. cGAPDH was used as a control gene to calculate ΔCt in order to normalize data. Normalized values were used to calculate ΔCt to determine fold changes.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000525"></div><a name="pone.0000525.s001" id="pone.0000525.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Immunoreactivity for N-cadherin in cultures of isolated epithelial cells from the chick cochlea (A) and utricle (B). Epithelial cultures were maintained in vitro for seven days, at which time the specimens were fixed and processed for immunocytochemical localization of N-cadherin. Labeling for N-cadherin (green) was observed at nearly all cell-cell junctions, confirming that the cultured cells originated from the sensory epithelia of the cochlea or utricle (see text). Cell nuclei were also labeled with DAPI (blue). Scale bar = 20 µm.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(2.52 MB DOC)</p>
<a name="pone.0000525.s002" id="pone.0000525.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Utricle Neomycin Differentially Expressed Genes (194 total). This listing shows all genes that exhibited&gt;1.2-fold changes in expression, irrespective of p-value. For p-value filtered data see <a href="#pone.0000525.s003">Table S2</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.21 MB DOC)</p>
<a name="pone.0000525.s003" id="pone.0000525.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s003">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Utricle Neomycin Differential Expression&gt;1.2-fold and P&lt; = 0.05</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.06 MB PDF)</p>
<a name="pone.0000525.s004" id="pone.0000525.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s004">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Utricle Laser Differentially Expressed Genes (261 total). This listing shows all genes that exhibited&gt;1.2-fold changes in expression, irrespective of P-value. For p-value filtered data see <a href="#pone.0000525.s005">Table S4</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.36 MB DOC)</p>
<a name="pone.0000525.s005" id="pone.0000525.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s005">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Utricle Laser Differential Expression&gt;1.2-fold and P&lt; = 0.05</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.05 MB PDF)</p>
<a name="pone.0000525.s006" id="pone.0000525.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s006">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Cochlea Neomycin Differentially Expressed Genes (533 total). This listing shows all genes that exhibited&gt;1.2-fold changes in expression, irrespective of P-value. For p-value filtered data see <a href="#pone.0000525.s007">Table S6</a></p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.71 MB DOC)</p>
<a name="pone.0000525.s007" id="pone.0000525.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s007">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Cochlea Neomycin Differential Expression&gt;1.2-fold and P&lt; = 0.05</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.10 MB PDF)</p>
<a name="pone.0000525.s008" id="pone.0000525.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s008">Table S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Cochlea Laser Differentially Expressed Genes (386 genes total). This listing shows all genes that exhibited&gt;1.2-fold changes in expression, irrespective of P-value. For p-value filtered data see <a href="#pone.0000525.s009">Table S8</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.53 MB DOC)</p>
<a name="pone.0000525.s009" id="pone.0000525.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s009">Table S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Cochlea Laser Differential Expression&gt;1.2-fold and P&lt; = 0.05</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.07 MB PDF)</p>
<a name="pone.0000525.s010" id="pone.0000525.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s010">Table S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">A listing of genes that exhibit similar patterns of expression across the four timecourses. utr-neo = utricle neomycin timecourse , coch-lsr = cochlea laser timecourse and so on. 1 = up-regulation, 0 = no change and −1 = down-regulation relative to untreated controls.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0000525.s011" id="pone.0000525.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s011">Table S10. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Transforming Growth Factor-Beta Signaling. CN = Cochlea Neomycin timecourse. CL = Cochlea Laser timecourse. UN = Utricle Neomycin timecourse. UL = Utricle Laser timecourse.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000525.s012" id="pone.0000525.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s012">Table S11. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">The PAX-EYA-SIX pathway. CN = Cochlea Neomycin timecourse. CL = Cochlea Laser timecourse. UN = Utricle Neomycin timecourse. UL = Utricle Laser timecourse.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000525.s013" id="pone.0000525.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s013">Table S12. </a></strong></p><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Notch signaling. CN = Cochlea Neomycin timecourse. CL = Cochlea Laser timecourse. UN = Utricle Neomycin timecourse. UL = Utricle Laser timecourse.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s013</p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0000525.s014" id="pone.0000525.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s014">Table S13. </a></strong></p><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1"></a><p class="preSiDOI">WNT signaling. CN = Cochlea Neomycin timecourse. CL = Cochlea Laser timecourse. UN = Utricle Neomycin timecourse. UL = Utricle Laser timecourse.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s014</p><a id="article1.body1.sec5.supplementary-material14.caption1.p2" name="article1.body1.sec5.supplementary-material14.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000525.s015" id="pone.0000525.s015"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s015">Table S14. </a></strong></p><a id="article1.body1.sec5.supplementary-material15.caption1.p1" name="article1.body1.sec5.supplementary-material15.caption1.p1"></a><p class="preSiDOI">NFKappaB. CN = Cochlea Neomycin timecourse. CL = Cochlea Laser timecourse. UN = Utricle Neomycin timecourse. UL = Utricle Laser timecourse.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s015</p><a id="article1.body1.sec5.supplementary-material15.caption1.p2" name="article1.body1.sec5.supplementary-material15.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0000525.s016" id="pone.0000525.s016"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s016">Table S15. </a></strong></p><a id="article1.body1.sec5.supplementary-material16.caption1.p1" name="article1.body1.sec5.supplementary-material16.caption1.p1"></a><p class="preSiDOI">Insulin/IGF signaling. CN = Cochlea Neomycin timecourse. CL = Cochlea Laser timecourse. UN = Utricle Neomycin timecourse. UL = Utricle Laser timecourse.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s016</p><a id="article1.body1.sec5.supplementary-material16.caption1.p2" name="article1.body1.sec5.supplementary-material16.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0000525.s017" id="pone.0000525.s017"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s017">Table S16. </a></strong></p><a id="article1.body1.sec5.supplementary-material17.caption1.p1" name="article1.body1.sec5.supplementary-material17.caption1.p1"></a><p class="preSiDOI">AP-1 Signaling. CN = Cochlea Neomycin timecourse. CL = Cochlea Laser timecourse. UN = Utricle Neomycin timecourse. UL = Utricle Laser timecourse.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s017</p><a id="article1.body1.sec5.supplementary-material17.caption1.p2" name="article1.body1.sec5.supplementary-material17.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000525.s018" id="pone.0000525.s018"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s018">Table S17. </a></strong></p><a id="article1.body1.sec5.supplementary-material18.caption1.p1" name="article1.body1.sec5.supplementary-material18.caption1.p1"></a><p class="preSiDOI">Cell Cycle/Apoptosis. CN = Cochlea Neomycin timecourse. CL = Cochlea Laser timecourse. UN = Utricle Neomycin timecourse. UL = Utricle Laser timecourse.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s018</p><a id="article1.body1.sec5.supplementary-material18.caption1.p2" name="article1.body1.sec5.supplementary-material18.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000525.s019" id="pone.0000525.s019"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s019">Table S18. </a></strong></p><a id="article1.body1.sec5.supplementary-material19.caption1.p1" name="article1.body1.sec5.supplementary-material19.caption1.p1"></a><p class="preSiDOI">Utricle Self Organizing Map Centroid Groups shown in <a href="#pone-0000525-g003">Figure 3A</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s019</p><a id="article1.body1.sec5.supplementary-material19.caption1.p2" name="article1.body1.sec5.supplementary-material19.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000525.s020" id="pone.0000525.s020"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s020">Table S19. </a></strong></p><a id="article1.body1.sec5.supplementary-material20.caption1.p1" name="article1.body1.sec5.supplementary-material20.caption1.p1"></a><p class="preSiDOI">Cochlea Self Organizing Map Centroid Groups shown in <a href="#pone-0000525-g003">Figure 3B</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s020</p><a id="article1.body1.sec5.supplementary-material20.caption1.p2" name="article1.body1.sec5.supplementary-material20.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000525.s021" id="pone.0000525.s021"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s021">Table S20. </a></strong></p><a id="article1.body1.sec5.supplementary-material21.caption1.p1" name="article1.body1.sec5.supplementary-material21.caption1.p1"></a><p class="preSiDOI">Utricle Detectably Expressed Genes</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s021</p><a id="article1.body1.sec5.supplementary-material21.caption1.p2" name="article1.body1.sec5.supplementary-material21.caption1.p2"></a><p class="postSiDOI">(0.23 MB DOC)</p>
<a name="pone.0000525.s022" id="pone.0000525.s022"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000525.s022">Table S21. </a></strong></p><a id="article1.body1.sec5.supplementary-material22.caption1.p1" name="article1.body1.sec5.supplementary-material22.caption1.p1"></a><p class="preSiDOI">Cochlea Detectably Expressed Genes</p>
<p class="siDoi">doi:10.1371/journal.pone.0000525.s022</p><a id="article1.body1.sec5.supplementary-material22.caption1.p2" name="article1.body1.sec5.supplementary-material22.caption1.p2"></a><p class="postSiDOI">(0.23 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>Our thanks to Dr. A.M. Bowcock for critical reading of this manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: ML MW. Performed the experiments: RH SB KP DA JS. Analyzed the data: ML RH SB KP VB SS MW. Wrote the paper: ML RH SB SS MW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000525-Hudspeth1" id="pone.0000525-Hudspeth1"></a>Hudspeth AJ (1997) How hearing happens. Neuron  19: 947–950.  <ul class="find" data-citedArticleID="958488" data-doi="10.1016/s0896-6273(00)80385-2"><li><a href="http://dx.doi.org/10.1016/s0896-6273(00)80385-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+hearing+happens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+hearing+happens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000525-Warchol1" id="pone.0000525-Warchol1"></a>Warchol ME, Lambert PR, Goldstein BJ, Forge A, Corwin JT (1993) Regenerative proliferation in inner ear sensory epithelia from adult guinea pigs and humans.[see comment]. Science  259: 1619–1622.  <ul class="find" data-citedArticleID="958562" data-doi="10.1126/science.8456285"><li><a href="http://dx.doi.org/10.1126/science.8456285" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regenerative+proliferation+in+inner+ear+sensory+epithelia+from+adult+guinea+pigs+and+humans.%5Bsee+comment%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regenerative+proliferation+in+inner+ear+sensory+epithelia+from+adult+guinea+pigs+and+humans.%5Bsee+comment%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000525-Forge1" id="pone.0000525-Forge1"></a>Forge A, Li L, Corwin JT, Nevill G (1993) Ultrastructural evidence for hair cell regeneration in the mammalian inner ear.[see comment]. Science  259: 1616–1619.  <ul class="find" data-citedArticleID="958476" data-doi="10.1126/science.8456284"><li><a href="http://dx.doi.org/10.1126/science.8456284" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ultrastructural+evidence+for+hair+cell+regeneration+in+the+mammalian+inner+ear.%5Bsee+comment%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ultrastructural+evidence+for+hair+cell+regeneration+in+the+mammalian+inner+ear.%5Bsee+comment%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000525-Roberson1" id="pone.0000525-Roberson1"></a>Roberson DW, Rubel EW (1994) Cell division in the gerbil cochlea after acoustic trauma. American Journal of Otology  15: 28–34.  <ul class="find" data-citedArticleID="958532"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cell+division+in+the+gerbil+cochlea+after+acoustic+trauma.&amp;auth=&amp;atitle=Cell+division+in+the+gerbil+cochlea+after+acoustic+trauma." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+division+in+the+gerbil+cochlea+after+acoustic+trauma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+division+in+the+gerbil+cochlea+after+acoustic+trauma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000525-Yamashita1" id="pone.0000525-Yamashita1"></a>Yamashita H, Oesterle EC (1995) Induction of cell proliferation in mammalian inner-ear sensory epithelia by transforming growth factor alpha and epidermal growth factor. Proceedings of the National Academy of Sciences of the United States of America  92: 3152–3155.  <ul class="find" data-citedArticleID="958576" data-doi="10.1073/pnas.92.8.3152"><li><a href="http://dx.doi.org/10.1073/pnas.92.8.3152" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+cell+proliferation+in+mammalian+inner-ear+sensory+epithelia+by+transforming+growth+factor+alpha+and+epidermal+growth+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+cell+proliferation+in+mammalian+inner-ear+sensory+epithelia+by+transforming+growth+factor+alpha+and+epidermal+growth+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000525-Oesterle1" id="pone.0000525-Oesterle1"></a>Oesterle EC, Cunningham DE, Westrum LE, Rubel EW (2003) Ultrastructural analysis of [3H]thymidine-labeled cells in the rat utricular macula. Journal of Comparative Neurology  463: 177–195.  <ul class="find" data-citedArticleID="958522" data-doi="10.1002/cne.10756"><li><a href="http://dx.doi.org/10.1002/cne.10756" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ultrastructural+analysis+of+%5B3H%5Dthymidine-labeled+cells+in+the+rat+utricular+macula." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ultrastructural+analysis+of+%5B3H%5Dthymidine-labeled+cells+in+the+rat+utricular+macula.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000525-Bermingham1" id="pone.0000525-Bermingham1"></a>Bermingham NA, Hassan BA, Price SD, Vollrath MA, Ben-Arie N, et al.  (1999) Math1: an essential gene for the generation of inner ear hair cells. Science  284: 1837–1841.  <ul class="find" data-citedArticleID="958454" data-doi="10.1126/science.284.5421.1837"><li><a href="http://dx.doi.org/10.1126/science.284.5421.1837" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Math1%3A+an+essential+gene+for+the+generation+of+inner+ear+hair+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Math1%3A+an+essential+gene+for+the+generation+of+inner+ear+hair+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000525-Kawamoto1" id="pone.0000525-Kawamoto1"></a>Kawamoto K, Ishimoto S-I, Minoda R, Brough DE, Raphael Y (2003) Math1 gene transfer generates new cochlear hair cells in mature guinea pigs in vivo. Journal of Neuroscience  23: 4395–4400.  <ul class="find" data-citedArticleID="958502"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Math1+gene+transfer+generates+new+cochlear+hair+cells+in+mature+guinea+pigs+in+vivo.&amp;auth=&amp;atitle=Math1+gene+transfer+generates+new+cochlear+hair+cells+in+mature+guinea+pigs+in+vivo." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Math1+gene+transfer+generates+new+cochlear+hair+cells+in+mature+guinea+pigs+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Math1+gene+transfer+generates+new+cochlear+hair+cells+in+mature+guinea+pigs+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000525-Izumikawa1" id="pone.0000525-Izumikawa1"></a>Izumikawa M, Minoda R, Kawamoto K, Abrashkin KA, Swiderski DL, et al.  (2005) Auditory hair cell replacement and hearing improvement by Atoh1 gene therapy in deaf mammals.[see comment]. Nature Medicine  11: 271–276.  <ul class="find" data-citedArticleID="958494" data-doi="10.1038/nm1193"><li><a href="http://dx.doi.org/10.1038/nm1193" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Auditory+hair+cell+replacement+and+hearing+improvement+by+Atoh1+gene+therapy+in+deaf+mammals.%5Bsee+comment%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Auditory+hair+cell+replacement+and+hearing+improvement+by+Atoh1+gene+therapy+in+deaf+mammals.%5Bsee+comment%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000525-Corwin1" id="pone.0000525-Corwin1"></a>Corwin JT, Cotanche DA (1988) Regeneration of sensory hair cells after acoustic trauma. Science  240: 1772–1774.  <ul class="find" data-citedArticleID="958466" data-doi="10.1126/science.3381100"><li><a href="http://dx.doi.org/10.1126/science.3381100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regeneration+of+sensory+hair+cells+after+acoustic+trauma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regeneration+of+sensory+hair+cells+after+acoustic+trauma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000525-Ryals1" id="pone.0000525-Ryals1"></a>Ryals BM, Rubel EW (1988) Hair cell regeneration after acoustic trauma in adult Coturnix quail. Science  240: 1774–1776.  <ul class="find" data-citedArticleID="958534" data-doi="10.1126/science.3381101"><li><a href="http://dx.doi.org/10.1126/science.3381101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hair+cell+regeneration+after+acoustic+trauma+in+adult+Coturnix+quail." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hair+cell+regeneration+after+acoustic+trauma+in+adult+Coturnix+quail.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000525-Stone1" id="pone.0000525-Stone1"></a>Stone JS, Rubel EW (1999) Delta1 expression during avian hair cell regeneration. Development  126: 961–973.  <ul class="find" data-citedArticleID="958544"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Delta1+expression+during+avian+hair+cell+regeneration.&amp;auth=&amp;atitle=Delta1+expression+during+avian+hair+cell+regeneration." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Delta1+expression+during+avian+hair+cell+regeneration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Delta1+expression+during+avian+hair+cell+regeneration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000525-BerminghamMcDonogh1" id="pone.0000525-BerminghamMcDonogh1"></a>Bermingham-McDonogh O, Stone JS, Reh TA, Rubel EW (2001) FGFR3 expression during development and regeneration of the chick inner ear sensory epithelia. Developmental Biology  238: 247–259.  <ul class="find" data-citedArticleID="958456" data-doi="10.1006/dbio.2001.0412"><li><a href="http://dx.doi.org/10.1006/dbio.2001.0412" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FGFR3+expression+during+development+and+regeneration+of+the+chick+inner+ear+sensory+epithelia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FGFR3+expression+during+development+and+regeneration+of+the+chick+inner+ear+sensory+epithelia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000525-Stone2" id="pone.0000525-Stone2"></a>Stone JS, Shang JL, Tomarev S (2004) cProx1 immunoreactivity distinguishes progenitor cells and predicts hair cell fate during avian hair cell regeneration. Developmental Dynamics  230: 597–614.  <ul class="find" data-citedArticleID="958546" data-doi="10.1002/dvdy.20087"><li><a href="http://dx.doi.org/10.1002/dvdy.20087" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=cProx1+immunoreactivity+distinguishes+progenitor+cells+and+predicts+hair+cell+fate+during+avian+hair+cell+regeneration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22cProx1+immunoreactivity+distinguishes+progenitor+cells+and+predicts+hair+cell+fate+during+avian+hair+cell+regeneration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000525-Hawkins1" id="pone.0000525-Hawkins1"></a>Hawkins RD, Bashiardes S, Helms CA, Hu L, Saccone NL, et al.  (2003) Gene expression differences in quiescent versus regenerating hair cells of avian sensory epithelia: implications for human hearing and balance disorders. Human Molecular Genetics  12: 1261–1272.  <ul class="find" data-citedArticleID="958480" data-doi="10.1093/hmg/ddg150"><li><a href="http://dx.doi.org/10.1093/hmg/ddg150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+differences+in+quiescent+versus+regenerating+hair+cells+of+avian+sensory+epithelia%3A+implications+for+human+hearing+and+balance+disorders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+differences+in+quiescent+versus+regenerating+hair+cells+of+avian+sensory+epithelia%3A+implications+for+human+hearing+and+balance+disorders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000525-Messina1" id="pone.0000525-Messina1"></a>Messina DN, Glasscock J, Gish W, Lovett M (2004) An ORFeome-based analysis of human transcription factor genes and the construction of a microarray to interrogate their expression. Genome Research  14: 2041–2047.  <ul class="find" data-citedArticleID="958520" data-doi="10.1101/gr.2584104"><li><a href="http://dx.doi.org/10.1101/gr.2584104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+ORFeome-based+analysis+of+human+transcription+factor+genes+and+the+construction+of+a+microarray+to+interrogate+their+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+ORFeome-based+analysis+of+human+transcription+factor+genes+and+the+construction+of+a+microarray+to+interrogate+their+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000525-Warchol2" id="pone.0000525-Warchol2"></a>Warchol ME (2002) Cell density and N-cadherin interactions regulate cell proliferation in the sensory epithelia of the inner ear. Journal of Neuroscience  22: 2607–2616.  <ul class="find" data-citedArticleID="958564"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cell+density+and+N-cadherin+interactions+regulate+cell+proliferation+in+the+sensory+epithelia+of+the+inner+ear.&amp;auth=&amp;atitle=Cell+density+and+N-cadherin+interactions+regulate+cell+proliferation+in+the+sensory+epithelia+of+the+inner+ear." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+density+and+N-cadherin+interactions+regulate+cell+proliferation+in+the+sensory+epithelia+of+the+inner+ear." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+density+and+N-cadherin+interactions+regulate+cell+proliferation+in+the+sensory+epithelia+of+the+inner+ear.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000525-Warchol3" id="pone.0000525-Warchol3"></a>Warchol ME (1999) Immune cytokines and dexamethasone influence sensory regeneration in the avian vestibular periphery. Journal of Neurocytology  28: 889–900.  <ul class="find" data-citedArticleID="958566"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Immune+cytokines+and+dexamethasone+influence+sensory+regeneration+in+the+avian+vestibular+periphery.&amp;auth=&amp;atitle=Immune+cytokines+and+dexamethasone+influence+sensory+regeneration+in+the+avian+vestibular+periphery." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immune+cytokines+and+dexamethasone+influence+sensory+regeneration+in+the+avian+vestibular+periphery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immune+cytokines+and+dexamethasone+influence+sensory+regeneration+in+the+avian+vestibular+periphery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000525-Matsui1" id="pone.0000525-Matsui1"></a>Matsui JI, Gale JE, Warchol ME (2004) Critical signaling events during the aminoglycoside-induced death of sensory hair cells in vitro. Journal of Neurobiology  61: 250–266.  <ul class="find" data-citedArticleID="958514" data-doi="10.1002/neu.20054"><li><a href="http://dx.doi.org/10.1002/neu.20054" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Critical+signaling+events+during+the+aminoglycoside-induced+death+of+sensory+hair+cells+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Critical+signaling+events+during+the+aminoglycoside-induced+death+of+sensory+hair+cells+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000525-Renn1" id="pone.0000525-Renn1"></a>Renn SCP, Aubin-Horth N, Hofmann HA (2004) Biologically meaningful expression profiling across species using heterologous hybridization to a cDNA microarray. BMC Genomics  5: 42.  <ul class="find" data-citedArticleID="958530"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Biologically+meaningful+expression+profiling+across+species+using+heterologous+hybridization+to+a+cDNA+microarray.&amp;auth=&amp;atitle=Biologically+meaningful+expression+profiling+across+species+using+heterologous+hybridization+to+a+cDNA+microarray." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biologically+meaningful+expression+profiling+across+species+using+heterologous+hybridization+to+a+cDNA+microarray." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biologically+meaningful+expression+profiling+across+species+using+heterologous+hybridization+to+a+cDNA+microarray.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000525-Hillier1" id="pone.0000525-Hillier1"></a>Hillier LW, Miller W, Birney E, Warren W, Hardison RC, et al.  (2004) Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution.[see comment][erratum appears in Nature. 2005 Feb 17;433(7027):777]. Nature  432: 695–716.  <ul class="find" data-citedArticleID="958484" data-doi="10.1038/nature03394"><li><a href="http://dx.doi.org/10.1038/nature03394" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+and+comparative+analysis+of+the+chicken+genome+provide+unique+perspectives+on+vertebrate+evolution.%5Bsee+comment%5D%5Berratum+appears+in+Nature.+2005+Feb+17%3B433%287027%29%3A777%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+and+comparative+analysis+of+the+chicken+genome+provide+unique+perspectives+on+vertebrate+evolution.%5Bsee+comment%5D%5Berratum+appears+in+Nature.+2005+Feb+17%3B433%287027%29%3A777%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000525-Warchol4" id="pone.0000525-Warchol4"></a>Warchol ME (2001) Lectin from Griffonia simplicifolia identifies an immature-appearing subpopulation of sensory hair cells in the avian utricle. Journal of Neurocytology  30: 253–264.  <ul class="find" data-citedArticleID="958568"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Lectin+from+Griffonia+simplicifolia+identifies+an+immature-appearing+subpopulation+of+sensory+hair+cells+in+the+avian+utricle.&amp;auth=&amp;atitle=Lectin+from+Griffonia+simplicifolia+identifies+an+immature-appearing+subpopulation+of+sensory+hair+cells+in+the+avian+utricle." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lectin+from+Griffonia+simplicifolia+identifies+an+immature-appearing+subpopulation+of+sensory+hair+cells+in+the+avian+utricle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lectin+from+Griffonia+simplicifolia+identifies+an+immature-appearing+subpopulation+of+sensory+hair+cells+in+the+avian+utricle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000525-Jorgensen1" id="pone.0000525-Jorgensen1"></a>Jorgensen JM, Mathiesen C (1988) The avian inner ear. Continuous production of hair cells in vestibular sensory organs, but not in the auditory papilla. Naturwissenschaften  75: 319–320.  <ul class="find" data-citedArticleID="958498" data-doi="10.1007/bf00367330"><li><a href="http://dx.doi.org/10.1007/bf00367330" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+avian+inner+ear.+Continuous+production+of+hair+cells+in+vestibular+sensory+organs%2C+but+not+in+the+auditory+papilla." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+avian+inner+ear.+Continuous+production+of+hair+cells+in+vestibular+sensory+organs%2C+but+not+in+the+auditory+papilla.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000525-Hinck1" id="pone.0000525-Hinck1"></a>Hinck L, Nelson WJ, Papkoff J (1994) Wnt-1 modulates cell-cell adhesion in mammalian cells by stabilizing beta-catenin binding to the cell adhesion protein cadherin. Journal of Cell Biology  124: 729–741.  <ul class="find" data-citedArticleID="958486" data-doi="10.1083/jcb.124.5.729"><li><a href="http://dx.doi.org/10.1083/jcb.124.5.729" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wnt-1+modulates+cell-cell+adhesion+in+mammalian+cells+by+stabilizing+beta-catenin+binding+to+the+cell+adhesion+protein+cadherin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Wnt-1+modulates+cell-cell+adhesion+in+mammalian+cells+by+stabilizing+beta-catenin+binding+to+the+cell+adhesion+protein+cadherin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000525-Ide1" id="pone.0000525-Ide1"></a>Ide T, Shimano H, Yoshikawa T, Yahagi N, Amemiya-Kudo M, et al.  (2003) Cross-talk between peroxisome proliferator-activated receptor (PPAR) alpha and liver X receptor (LXR) in nutritional regulation of fatty acid metabolism. II. LXRs suppress lipid degradation gene promoters through inhibition of PPAR signaling.[see comment]. Molecular Endocrinology  17: 1255–1267.  <ul class="find" data-citedArticleID="958490" data-doi="10.1210/me.2002-0191"><li><a href="http://dx.doi.org/10.1210/me.2002-0191" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cross-talk+between+peroxisome+proliferator-activated+receptor+%28PPAR%29+alpha+and+liver+X+receptor+%28LXR%29+in+nutritional+regulation+of+fatty+acid+metabolism.+II.+LXRs+suppress+lipid+degradation+gene+promoters+through+inhibition+of+PPAR+signaling.%5Bsee+comment%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cross-talk+between+peroxisome+proliferator-activated+receptor+%28PPAR%29+alpha+and+liver+X+receptor+%28LXR%29+in+nutritional+regulation+of+fatty+acid+metabolism.+II.+LXRs+suppress+lipid+degradation+gene+promoters+through+inhibition+of+PPAR+signaling.%5Bsee+comment%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000525-Pawar1" id="pone.0000525-Pawar1"></a>Pawar A, Botolin D, Mangelsdorf DJ, Jump DB (2003) The role of liver X receptor-alpha in the fatty acid regulation of hepatic gene expression. Journal of Biological Chemistry  278: 40736–40743.  <ul class="find" data-citedArticleID="958524" data-doi="10.1074/jbc.m307973200"><li><a href="http://dx.doi.org/10.1074/jbc.m307973200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+liver+X+receptor-alpha+in+the+fatty+acid+regulation+of+hepatic+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+liver+X+receptor-alpha+in+the+fatty+acid+regulation+of+hepatic+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000525-Medina1" id="pone.0000525-Medina1"></a>Medina KL, Singh H (2005) Genetic networks that regulate B lymphopoiesis. Current Opinion in Hematology  12: 203–209.  <ul class="find" data-citedArticleID="958518" data-doi="10.1097/01.moh.0000160735.67596.a0"><li><a href="http://dx.doi.org/10.1097/01.moh.0000160735.67596.a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+networks+that+regulate+B+lymphopoiesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+networks+that+regulate+B+lymphopoiesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000525-Singh1" id="pone.0000525-Singh1"></a>Singh H, Medina KL, Pongubala JMR (2005) Contingent gene regulatory networks and B cell fate specification. Proceedings of the National Academy of Sciences of the United States of America  102: 4949–4953.  <ul class="find" data-citedArticleID="958538" data-doi="10.1073/pnas.0500480102"><li><a href="http://dx.doi.org/10.1073/pnas.0500480102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contingent+gene+regulatory+networks+and+B+cell+fate+specification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contingent+gene+regulatory+networks+and+B+cell+fate+specification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000525-Matsui2" id="pone.0000525-Matsui2"></a>Matsui JI, Oesterle EC, Stone JS, Rubel EW (2000) Characterization of damage and regeneration in cultured avian utricles. Jaro  1: 46–63.  <ul class="find" data-citedArticleID="958516" data-doi="10.1007/s101620010005"><li><a href="http://dx.doi.org/10.1007/s101620010005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+damage+and+regeneration+in+cultured+avian+utricles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+damage+and+regeneration+in+cultured+avian+utricles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000525-Duncan1" id="pone.0000525-Duncan1"></a>Duncan LJ, Mangiardi DA, Matsui JI, Anderson JK, McLaughlin-Williamson K, et al.  (2006) Differential expression of unconventional myosins in apoptotic and regenerating chick hair cells confirms two regeneration mechanisms. J Comp Neurol  499: 691–701.  <ul class="find" data-citedArticleID="958472" data-doi="10.1002/cne.21114"><li><a href="http://dx.doi.org/10.1002/cne.21114" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+expression+of+unconventional+myosins+in+apoptotic+and+regenerating+chick+hair+cells+confirms+two+regeneration+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+expression+of+unconventional+myosins+in+apoptotic+and+regenerating+chick+hair+cells+confirms+two+regeneration+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000525-Li1" id="pone.0000525-Li1"></a>Li C, Zhu N-L, Tan RC, Ballard PL, Derynck R, et al.  (2002) Transforming growth factor-beta inhibits pulmonary surfactant protein B gene transcription through SMAD3 interactions with NKX2.1 and HNF-3 transcription factors. Journal of Biological Chemistry  277: 38399–38408.  <ul class="find" data-citedArticleID="958508" data-doi="10.1074/jbc.m203188200"><li><a href="http://dx.doi.org/10.1074/jbc.m203188200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transforming+growth+factor-beta+inhibits+pulmonary+surfactant+protein+B+gene+transcription+through+SMAD3+interactions+with+NKX2.1+and+HNF-3+transcription+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transforming+growth+factor-beta+inhibits+pulmonary+surfactant+protein+B+gene+transcription+through+SMAD3+interactions+with+NKX2.1+and+HNF-3+transcription+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000525-Trueba1" id="pone.0000525-Trueba1"></a>Trueba SS, Auge J, Mattei G, Etchevers H, Martinovic J, et al.  (2005) PAX8, TITF1, and FOXE1 gene expression patterns during human development: new insights into human thyroid development and thyroid dysgenesis-associated malformations. Journal of Clinical Endocrinology &amp; Metabolism  90: 455–462.  <ul class="find" data-citedArticleID="958552" data-doi="10.1210/jc.2004-1358"><li><a href="http://dx.doi.org/10.1210/jc.2004-1358" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAX8%2C+TITF1%2C+and+FOXE1+gene+expression+patterns+during+human+development%3A+new+insights+into+human+thyroid+development+and+thyroid+dysgenesis-associated+malformations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAX8%2C+TITF1%2C+and+FOXE1+gene+expression+patterns+during+human+development%3A+new+insights+into+human+thyroid+development+and+thyroid+dysgenesis-associated+malformations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000525-DiPalma1" id="pone.0000525-DiPalma1"></a>Di Palma T, Nitsch R, Mascia A, Nitsch L, Di Lauro R, et al.  (2003) The paired domain-containing factor Pax8 and the homeodomain-containing factor TTF-1 directly interact and synergistically activate transcription. Journal of Biological Chemistry  278: 3395–3402.  <ul class="find" data-citedArticleID="958470" data-doi="10.1074/jbc.m205977200"><li><a href="http://dx.doi.org/10.1074/jbc.m205977200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+paired+domain-containing+factor+Pax8+and+the+homeodomain-containing+factor+TTF-1+directly+interact+and+synergistically+activate+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+paired+domain-containing+factor+Pax8+and+the+homeodomain-containing+factor+TTF-1+directly+interact+and+synergistically+activate+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000525-Chen1" id="pone.0000525-Chen1"></a>Chen P, Segil N (1999) p27(Kip1) links cell proliferation to morphogenesis in the developing organ of Corti. Development  126: 1581–1590.  <ul class="find" data-citedArticleID="958464"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=p27%28Kip1%29+links+cell+proliferation+to+morphogenesis+in+the+developing+organ+of+Corti.&amp;auth=&amp;atitle=p27%28Kip1%29+links+cell+proliferation+to+morphogenesis+in+the+developing+organ+of+Corti." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=p27%28Kip1%29+links+cell+proliferation+to+morphogenesis+in+the+developing+organ+of+Corti." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22p27%28Kip1%29+links+cell+proliferation+to+morphogenesis+in+the+developing+organ+of+Corti.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000525-White1" id="pone.0000525-White1"></a>White PM, Doetzlhofer A, Lee YS, Groves AK, Segil N (2006) Mammalian cochlear supporting cells can divide and trans-differentiate into hair cells. Nature  441: 984–987.  <ul class="find" data-citedArticleID="958574" data-doi="10.1038/nature04849"><li><a href="http://dx.doi.org/10.1038/nature04849" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+cochlear+supporting+cells+can+divide+and+trans-differentiate+into+hair+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+cochlear+supporting+cells+can+divide+and+trans-differentiate+into+hair+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000525-Yang1" id="pone.0000525-Yang1"></a>Yang X, Menon S, Lykke-Andersen K, Tsuge T, Di X, et al.  (2002) The COP9 signalosome inhibits p27(kip1) degradation and impedes G1-S phase progression via deneddylation of SCF Cul1. Current Biology  12: 667–672.  <ul class="find" data-citedArticleID="958578" data-doi="10.1016/s0960-9822(02)00791-1"><li><a href="http://dx.doi.org/10.1016/s0960-9822(02)00791-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+COP9+signalosome+inhibits+p27%28kip1%29+degradation+and+impedes+G1-S+phase+progression+via+deneddylation+of+SCF+Cul1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+COP9+signalosome+inhibits+p27%28kip1%29+degradation+and+impedes+G1-S+phase+progression+via+deneddylation+of+SCF+Cul1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000525-Ledford1" id="pone.0000525-Ledford1"></a>Ledford AW, Brantley JG, Kemeny G, Foreman TL, Quaggin SE, et al.  (2002) Deregulated expression of the homeobox gene Cux-1 in transgenic mice results in downregulation of p27(kip1) expression during nephrogenesis, glomerular abnormalities, and multiorgan hyperplasia. Developmental Biology  245: 157–171.  <ul class="find" data-citedArticleID="958506" data-doi="10.1006/dbio.2002.0636"><li><a href="http://dx.doi.org/10.1006/dbio.2002.0636" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deregulated+expression+of+the+homeobox+gene+Cux-1+in+transgenic+mice+results+in+downregulation+of+p27%28kip1%29+expression+during+nephrogenesis%2C+glomerular+abnormalities%2C+and+multiorgan+hyperplasia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deregulated+expression+of+the+homeobox+gene+Cux-1+in+transgenic+mice+results+in+downregulation+of+p27%28kip1%29+expression+during+nephrogenesis%2C+glomerular+abnormalities%2C+and+multiorgan+hyperplasia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000525-Li2" id="pone.0000525-Li2"></a>Li X, Perissi V, Liu F, Rose DW, Rosenfeld MG (2002) Tissue-specific regulation of retinal and pituitary precursor cell proliferation. Science  297: 1180–1183.  <ul class="find" data-citedArticleID="958510" data-doi="10.1126/science.1073263"><li><a href="http://dx.doi.org/10.1126/science.1073263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tissue-specific+regulation+of+retinal+and+pituitary+precursor+cell+proliferation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tissue-specific+regulation+of+retinal+and+pituitary+precursor+cell+proliferation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000525-Reich1" id="pone.0000525-Reich1"></a>Reich M, Ohm K, Angelo M, Tamayo P, Mesirov JP (2004) GeneCluster 2.0: an advanced toolset for bioarray analysis. Bioinformatics  20: 1797–1798.  <ul class="find" data-citedArticleID="958528" data-doi="10.1093/bioinformatics/bth138"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneCluster+2.0%3A+an+advanced+toolset+for+bioarray+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneCluster+2.0%3A+an+advanced+toolset+for+bioarray+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000525-Tamayo1" id="pone.0000525-Tamayo1"></a>Tamayo P, Slonim D, Mesirov J, Zhu Q, Kitareewan S, et al.  (1999) Interpreting patterns of gene expression with self-organizing maps: methods and application to hematopoietic differentiation. Proceedings of the National Academy of Sciences of the United States of America  96: 2907–2912.  <ul class="find" data-citedArticleID="958548" data-doi="10.1073/pnas.96.6.2907"><li><a href="http://dx.doi.org/10.1073/pnas.96.6.2907" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interpreting+patterns+of+gene+expression+with+self-organizing+maps%3A+methods+and+application+to+hematopoietic+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interpreting+patterns+of+gene+expression+with+self-organizing+maps%3A+methods+and+application+to+hematopoietic+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000525-Vahava1" id="pone.0000525-Vahava1"></a>Vahava O, Morell R, Lynch ED, Weiss S, Kagan ME, et al.  (1998) Mutation in transcription factor POU4F3 associated with inherited progressive hearing loss in humans.[see comment]. Science  279: 1950–1954.  <ul class="find" data-citedArticleID="958554" data-doi="10.1126/science.279.5358.1950"><li><a href="http://dx.doi.org/10.1126/science.279.5358.1950" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+in+transcription+factor+POU4F3+associated+with+inherited+progressive+hearing+loss+in+humans.%5Bsee+comment%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+in+transcription+factor+POU4F3+associated+with+inherited+progressive+hearing+loss+in+humans.%5Bsee+comment%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000525-Ikeda1" id="pone.0000525-Ikeda1"></a>Ikeda S, Kurose K, Jinno H, Sai K, Ozawa S, et al.  (2005) Functional analysis of four naturally occurring variants of human constitutive androstane receptor. Molecular Genetics &amp; Metabolism  86: 314–319.  <ul class="find" data-citedArticleID="958492" data-doi="10.1016/j.ymgme.2005.05.011"><li><a href="http://dx.doi.org/10.1016/j.ymgme.2005.05.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+analysis+of+four+naturally+occurring+variants+of+human+constitutive+androstane+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+analysis+of+four+naturally+occurring+variants+of+human+constitutive+androstane+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000525-Thompson1" id="pone.0000525-Thompson1"></a>Thompson EE, Kuttab-Boulos H, Krasowski MD, Di Rienzo A (2005) Functional constraints on the constitutive androstane receptor inferred from human sequence variation and cross-species comparisons. Human Genomics  2: 168–178.  <ul class="find" data-citedArticleID="958550" data-doi="10.1186/1479-7364-2-3-168"><li><a href="http://dx.doi.org/10.1186/1479-7364-2-3-168" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+constraints+on+the+constitutive+androstane+receptor+inferred+from+human+sequence+variation+and+cross-species+comparisons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+constraints+on+the+constitutive+androstane+receptor+inferred+from+human+sequence+variation+and+cross-species+comparisons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000525-Liu1" id="pone.0000525-Liu1"></a>Liu W, Lagutin OV, Mende M, Streit A, Oliver G (2006) Six3 activation of Pax6 expression is essential for mammalian lens induction and specification. EMBO Journal  25: 5383–5395.  <ul class="find" data-citedArticleID="958512" data-doi="10.1038/sj.emboj.7601398"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601398" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Six3+activation+of+Pax6+expression+is+essential+for+mammalian+lens+induction+and+specification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Six3+activation+of+Pax6+expression+is+essential+for+mammalian+lens+induction+and+specification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000525-Jhun1" id="pone.0000525-Jhun1"></a>Jhun BH, Haruta T, Meinkoth JL, Leitner W, Draznin B, et al.  (1995) Signal transduction pathways leading to insulin-induced early gene induction. Biochemistry  34: 7996–8004.  <ul class="find" data-citedArticleID="958496" data-doi="10.1021/bi00025a005"><li><a href="http://dx.doi.org/10.1021/bi00025a005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signal+transduction+pathways+leading+to+insulin-induced+early+gene+induction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signal+transduction+pathways+leading+to+insulin-induced+early+gene+induction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000525-Zhang1" id="pone.0000525-Zhang1"></a>Zhang W, Zhang J, Kornuc M, Kwan K, Frank R, et al.  (1995) Molecular cloning and characterization of NF-IL3A, a transcriptional activator of the human interleukin-3 promoter. Molecular &amp; Cellular Biology  15: 6055–6063.  <ul class="find" data-citedArticleID="958580"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+cloning+and+characterization+of+NF-IL3A%2C+a+transcriptional+activator+of+the+human+interleukin-3+promoter.&amp;auth=&amp;atitle=Molecular+cloning+and+characterization+of+NF-IL3A%2C+a+transcriptional+activator+of+the+human+interleukin-3+promoter." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+and+characterization+of+NF-IL3A%2C+a+transcriptional+activator+of+the+human+interleukin-3+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+and+characterization+of+NF-IL3A%2C+a+transcriptional+activator+of+the+human+interleukin-3+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000525-Fritzsch1" id="pone.0000525-Fritzsch1"></a>Fritzsch B (2003) Development of inner ear afferent connections: forming primary neurons and connecting them to the developing sensory epithelia. Brain Research Bulletin  60: 423–433.  <ul class="find" data-citedArticleID="958478" data-doi="10.1016/s0361-9230(03)00048-0"><li><a href="http://dx.doi.org/10.1016/s0361-9230(03)00048-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+inner+ear+afferent+connections%3A+forming+primary+neurons+and+connecting+them+to+the+developing+sensory+epithelia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+inner+ear+afferent+connections%3A+forming+primary+neurons+and+connecting+them+to+the+developing+sensory+epithelia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000525-Wagner1" id="pone.0000525-Wagner1"></a>Wagner RA, Tabibiazar R, Liao A, Quertermous T (2005) Genome-wide expression dynamics during mouse embryonic development reveal similarities to Drosophila development. Developmental Biology  288: 595–611.  <ul class="find" data-citedArticleID="958558" data-doi="10.1016/j.ydbio.2005.09.036"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2005.09.036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+expression+dynamics+during+mouse+embryonic+development+reveal+similarities+to+Drosophila+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+expression+dynamics+during+mouse+embryonic+development+reveal+similarities+to+Drosophila+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000525-Brivanlou1" id="pone.0000525-Brivanlou1"></a>Brivanlou AH, Darnell JE Jr (2002) Signal transduction and the control of gene expression. Science  295: 813–818.  <ul class="find" data-citedArticleID="958462" data-doi="10.1126/science.1066355"><li><a href="http://dx.doi.org/10.1126/science.1066355" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signal+transduction+and+the+control+of+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signal+transduction+and+the+control+of+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000525-Lacorte1" id="pone.0000525-Lacorte1"></a>Lacorte JM, Ktistaki E, Beigneux A, Zannis VI, Chambaz J, et al.  (1997) Activation of CAAT enhancer-binding protein delta (C/EBPdelta) by interleukin-1 negatively influences apolipoprotein C-III expression. Journal of Biological Chemistry  272: 23578–23584.  <ul class="find" data-citedArticleID="958504" data-doi="10.1074/jbc.272.38.23578"><li><a href="http://dx.doi.org/10.1074/jbc.272.38.23578" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+of+CAAT+enhancer-binding+protein+delta+%28C%2FEBPdelta%29+by+interleukin-1+negatively+influences+apolipoprotein+C-III+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+of+CAAT+enhancer-binding+protein+delta+%28C%2FEBPdelta%29+by+interleukin-1+negatively+influences+apolipoprotein+C-III+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000525-Wang1" id="pone.0000525-Wang1"></a>Wang B, Weidenfeld J, Lu MM, Maika S, Kuziel WA, et al.  (2004) Foxp1 regulates cardiac outflow tract, endocardial cushion morphogenesis and myocyte proliferation and maturation. Development  131: 4477–4487.  <ul class="find" data-citedArticleID="958560" data-doi="10.1242/dev.01287"><li><a href="http://dx.doi.org/10.1242/dev.01287" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Foxp1+regulates+cardiac+outflow+tract%2C+endocardial+cushion+morphogenesis+and+myocyte+proliferation+and+maturation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Foxp1+regulates+cardiac+outflow+tract%2C+endocardial+cushion+morphogenesis+and+myocyte+proliferation+and+maturation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000525-Bracken1" id="pone.0000525-Bracken1"></a>Bracken AP, Dietrich N, Pasini D, Hansen KH, Helin K (2006) Genome-wide mapping of Polycomb target genes unravels their roles in cell fate transitions. Genes &amp; Development  20: 1123–1136.  <ul class="find" data-citedArticleID="958460" data-doi="10.1101/gad.381706"><li><a href="http://dx.doi.org/10.1101/gad.381706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+mapping+of+Polycomb+target+genes+unravels+their+roles+in+cell+fate+transitions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+mapping+of+Polycomb+target+genes+unravels+their+roles+in+cell+fate+transitions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000525-Kamminga1" id="pone.0000525-Kamminga1"></a>Kamminga LM, Bystrykh LV, de Boer A, Houwer S, Douma J, et al.  (2006) The Polycomb group gene Ezh2 prevents hematopoietic stem cell exhaustion. Blood  107: 2170–2179.  <ul class="find" data-citedArticleID="958500" data-doi="10.1182/blood-2005-09-3585"><li><a href="http://dx.doi.org/10.1182/blood-2005-09-3585" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Polycomb+group+gene+Ezh2+prevents+hematopoietic+stem+cell+exhaustion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Polycomb+group+gene+Ezh2+prevents+hematopoietic+stem+cell+exhaustion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000525-Vire1" id="pone.0000525-Vire1"></a>Vire E, Brenner C, Deplus R, Blanchon L, Fraga M, et al.  (2006) The Polycomb group protein EZH2 directly controls DNA methylation.[see comment]. Nature  439: 871–874.  <ul class="find" data-citedArticleID="958556" data-doi="10.1038/nature04431"><li><a href="http://dx.doi.org/10.1038/nature04431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Polycomb+group+protein+EZH2+directly+controls+DNA+methylation.%5Bsee+comment%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Polycomb+group+protein+EZH2+directly+controls+DNA+methylation.%5Bsee+comment%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000525-Bernstein1" id="pone.0000525-Bernstein1"></a>Bernstein E, Duncan EM, Masui O, Gil J, Heard E, et al.  (2006) Mouse polycomb proteins bind differentially to methylated histone H3 and RNA and are enriched in facultative heterochromatin. Molecular &amp; Cellular Biology  26: 2560–2569.  <ul class="find" data-citedArticleID="958458" data-doi="10.1128/mcb.26.7.2560-2569.2006"><li><a href="http://dx.doi.org/10.1128/mcb.26.7.2560-2569.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mouse+polycomb+proteins+bind+differentially+to+methylated+histone+H3+and+RNA+and+are+enriched+in+facultative+heterochromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mouse+polycomb+proteins+bind+differentially+to+methylated+histone+H3+and+RNA+and+are+enriched+in+facultative+heterochromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000525-Schwartz1" id="pone.0000525-Schwartz1"></a>Schwartz YB, Pirrotta V (2007) Polycomb silencing mechanisms and the management of genomic programmes. Nature Reviews Genetics  8: 9–22.  <ul class="find" data-citedArticleID="958536" data-doi="10.1038/nrg1981"><li><a href="http://dx.doi.org/10.1038/nrg1981" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polycomb+silencing+mechanisms+and+the+management+of+genomic+programmes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polycomb+silencing+mechanisms+and+the+management+of+genomic+programmes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000525-Stenberg1" id="pone.0000525-Stenberg1"></a>Stenberg AE, Wang H, Fish J 3rd, Schrott-Fischer A, Sahlin L, et al.  (2001) Estrogen receptors in the normal adult and developing human inner ear and in Turner's syndrome. Hearing Research  157: 87–92.  <ul class="find" data-citedArticleID="958542" data-doi="10.1016/s0378-5955(01)00280-5"><li><a href="http://dx.doi.org/10.1016/s0378-5955(01)00280-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estrogen+receptors+in+the+normal+adult+and+developing+human+inner+ear+and+in+Turner%27s+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estrogen+receptors+in+the+normal+adult+and+developing+human+inner+ear+and+in+Turner%27s+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000525-Cvoro1" id="pone.0000525-Cvoro1"></a>Cvoro A, Tzagarakis-Foster C, Tatomer D, Paruthiyil S, Fox MS, et al.  (2006) Distinct roles of unliganded and liganded estrogen receptors in transcriptional repression. Molecular Cell  21: 555–564.  <ul class="find" data-citedArticleID="958468" data-doi="10.1016/j.molcel.2006.01.014"><li><a href="http://dx.doi.org/10.1016/j.molcel.2006.01.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+roles+of+unliganded+and+liganded+estrogen+receptors+in+transcriptional+repression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+roles+of+unliganded+and+liganded+estrogen+receptors+in+transcriptional+repression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000525-Sommer1" id="pone.0000525-Sommer1"></a>Sommer S, Fuqua SA (2001) Estrogen receptor and breast cancer. Seminars in Cancer Biology  11: 339–352.  <ul class="find" data-citedArticleID="958540" data-doi="10.1006/scbi.2001.0389"><li><a href="http://dx.doi.org/10.1006/scbi.2001.0389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estrogen+receptor+and+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estrogen+receptor+and+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000525-Hayes1" id="pone.0000525-Hayes1"></a>Hayes KR, Baggs JE, Hogenesch JB (2005) Circadian clocks are seeing the systems biology light. Genome Biology  6: 219.  <ul class="find" data-citedArticleID="958482" data-doi="10.1186/gb-2005-6-5-219"><li><a href="http://dx.doi.org/10.1186/gb-2005-6-5-219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Circadian+clocks+are+seeing+the+systems+biology+light." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Circadian+clocks+are+seeing+the+systems+biology+light.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000525-Antoch1" id="pone.0000525-Antoch1"></a>Antoch MP, Kondratov RV, Takahashi JS (2005) Circadian clock genes as modulators of sensitivity to genotoxic stress. Cell Cycle  4: 901–907.  <ul class="find" data-citedArticleID="958452" data-doi="10.4161/cc.4.7.1792"><li><a href="http://dx.doi.org/10.4161/cc.4.7.1792" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Circadian+clock+genes+as+modulators+of+sensitivity+to+genotoxic+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Circadian+clock+genes+as+modulators+of+sensitivity+to+genotoxic+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000525-Etchegaray1" id="pone.0000525-Etchegaray1"></a>Etchegaray J-P, Yang X, DeBruyne JP, Peters AHFM, Weaver DR, et al.  (2006) The polycomb group protein EZH2 is required for mammalian circadian clock function. Journal of Biological Chemistry  281: 21209–21215.  <ul class="find" data-citedArticleID="958474" data-doi="10.1074/jbc.m603722200"><li><a href="http://dx.doi.org/10.1074/jbc.m603722200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+polycomb+group+protein+EZH2+is+required+for+mammalian+circadian+clock+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+polycomb+group+protein+EZH2+is+required+for+mammalian+circadian+clock+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000525-Warchol5" id="pone.0000525-Warchol5"></a>Warchol ME, Corwin JT (1996) Regenerative proliferation in organ cultures of the avian cochlea: identification of the initial progenitors and determination of the latency of the proliferative response. Journal of Neuroscience  16: 5466–5477.  <ul class="find" data-citedArticleID="958570"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regenerative+proliferation+in+organ+cultures+of+the+avian+cochlea%3A+identification+of+the+initial+progenitors+and+determination+of+the+latency+of+the+proliferative+response.&amp;auth=&amp;atitle=Regenerative+proliferation+in+organ+cultures+of+the+avian+cochlea%3A+identification+of+the+initial+progenitors+and+determination+of+the+latency+of+the+proliferative+response." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regenerative+proliferation+in+organ+cultures+of+the+avian+cochlea%3A+identification+of+the+initial+progenitors+and+determination+of+the+latency+of+the+proliferative+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regenerative+proliferation+in+organ+cultures+of+the+avian+cochlea%3A+identification+of+the+initial+progenitors+and+determination+of+the+latency+of+the+proliferative+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000525-Raphael1" id="pone.0000525-Raphael1"></a>Raphael Y, Volk T, Crossin KL, Edelman GM, Geiger B (1988) The modulation of cell adhesion molecule expression and intercellular junction formation in the developing avian inner ear. Developmental Biology  128: 222–235.  <ul class="find" data-citedArticleID="958526" data-doi="10.1016/0012-1606(88)90284-9"><li><a href="http://dx.doi.org/10.1016/0012-1606(88)90284-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+modulation+of+cell+adhesion+molecule+expression+and+intercellular+junction+formation+in+the+developing+avian+inner+ear." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+modulation+of+cell+adhesion+molecule+expression+and+intercellular+junction+formation+in+the+developing+avian+inner+ear.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000525-Warchol6" id="pone.0000525-Warchol6"></a>Warchol ME (2007) Characterization of supporting cell phenotype in the avian inner ear: Implications for sensory regeneration. Hearing Research  (in press):   <ul class="find" data-citedArticleID="958572" data-doi="10.1016/j.heares.2006.08.014"><li><a href="http://dx.doi.org/10.1016/j.heares.2006.08.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+supporting+cell+phenotype+in+the+avian+inner+ear%3A+Implications+for+sensory+regeneration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+supporting+cell+phenotype+in+the+avian+inner+ear%3A+Implications+for+sensory+regeneration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.XML" value="176320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.PDF" value="434564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.t001.PNG_L" value="172590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.t001.PNG_M" value="61166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.t001.PNG_S" value="3042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.t001.TIF" value="869114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.t001.PNG_I" value="25694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g001.PNG_L" value="94824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g001.PNG_M" value="43784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g001.PNG_S" value="2621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g001.TIF" value="133924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g001.PNG_I" value="16448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.t002.PNG_L" value="201382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.t002.PNG_M" value="67869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.t002.PNG_S" value="3614"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.t002.TIF" value="1276566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.t002.PNG_I" value="35062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g002.PNG_L" value="232632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g002.PNG_M" value="106343"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g002.PNG_S" value="4615"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g002.TIF" value="403866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g002.PNG_I" value="39583"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g003.PNG_L" value="1270666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g003.PNG_M" value="265988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g003.PNG_S" value="14302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g003.TIF" value="1888996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g003.PNG_I" value="194662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g004.PNG_L" value="168771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g004.PNG_M" value="39082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g004.PNG_S" value="3121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g004.TIF" value="219066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g004.PNG_I" value="24703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g005.PNG_L" value="1045008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g005.PNG_M" value="100658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g005.PNG_S" value="10735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g005.TIF" value="1267248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.g005.PNG_I" value="165774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s001.DOC" value="2516992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s002.DOC" value="214528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s003.PDF" value="59728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s004.DOC" value="364032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s005.PDF" value="50375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s006.DOC" value="705024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s007.PDF" value="100299"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s008.DOC" value="532992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s009.PDF" value="69117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s010.PDF" value="44931"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s011.DOC" value="25600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s012.DOC" value="33280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s013.DOC" value="22016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s014.DOC" value="25088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s015.DOC" value="21504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s016.DOC" value="23552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s017.DOC" value="25088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s018.DOC" value="33280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s019.DOC" value="29696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s020.DOC" value="39936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s021.DOC" value="228352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000525.s022.DOC" value="231424"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000525&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000525" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000525&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000525&volume=&issue=&title=Large Scale Gene Expression Profiles of Regenerating Inner Ear Sensory Epithelia&author_name=R.%20David%20Hawkins%2C%20Stavros%20Bashiardes%2C%20Kara%20E.%20Powder%2C%20Samin%20A.%20Sajan%2C%20Veena%20Bhonagiri%2C%20David%20M.%20Alvarado%2C%20Judith%20Speck%2C%20Mark%20E.%20Warchol%2C%20Michael%20Lovett&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000525" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000525" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000525" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000525&amp;t=Large%20Scale%20Gene%20Expression%20Profiles%20of%20Regenerating%20Inner%20Ear%20Sensory%20Epithelia" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000525&title=Large%20Scale%20Gene%20Expression%20Profiles%20of%20Regenerating%20Inner%20Ear%20Sensory%20Epithelia&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000525&amp;title=Large%20Scale%20Gene%20Expression%20Profiles%20of%20Regenerating%20Inner%20Ear%20Sensory%20Epithelia" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000525" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000525&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Large Scale Gene Expression Profiles of Regenerating Inner Ear Sensory Epithelia';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000525';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Large%20Scale%20Gene%20Expression%20Profiles%20of%20Regenerating%20Inner%20Ear%20Sensory%20Epithelia http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000525" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000525" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Birds%22" title="Search for articles in the subject area:'Birds'"><div class="flagText">Birds</div></a>
              <div data-categoryid="42813" data-articleid="23888"
                   data-categoryname="Birds"
                   class="flagImage" title="Flag 'Birds' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chickens%22" title="Search for articles in the subject area:'Chickens'"><div class="flagText">Chickens</div></a>
              <div data-categoryid="42811" data-articleid="23888"
                   data-categoryname="Chickens"
                   class="flagImage" title="Flag 'Chickens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cochlea%22" title="Search for articles in the subject area:'Cochlea'"><div class="flagText">Cochlea</div></a>
              <div data-categoryid="48363" data-articleid="23888"
                   data-categoryname="Cochlea"
                   class="flagImage" title="Flag 'Cochlea' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23888"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lasers%22" title="Search for articles in the subject area:'Lasers'"><div class="flagText">Lasers</div></a>
              <div data-categoryid="19343" data-articleid="23888"
                   data-categoryname="Lasers"
                   class="flagImage" title="Flag 'Lasers' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="23888"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Neomycins%22" title="Search for articles in the subject area:'Neomycins'"><div class="flagText">Neomycins</div></a>
              <div data-categoryid="44283" data-articleid="23888"
                   data-categoryname="Neomycins"
                   class="flagImage" title="Flag 'Neomycins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcription+factors%22" title="Search for articles in the subject area:'Transcription factors'"><div class="flagText">Transcription factors</div></a>
              <div data-categoryid="17335" data-articleid="23888"
                   data-categoryname="Transcription factors"
                   class="flagImage" title="Flag 'Transcription factors' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3476'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1681'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1725&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
