

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: RNA Pol II Promotes Transcription of Centromeric Satellite DNA in Beetles</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001594"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001594" />

    <meta name="citation_title" content="RNA Pol II Promotes Transcription of Centromeric Satellite DNA in Beetles"/>
    <meta itemprop="name" content="RNA Pol II Promotes Transcription of Centromeric Satellite DNA in Beetles"/>

      <meta name="citation_author" content="Željka Pezer"/>
            <meta name="citation_author_institution" content="Department of Molecular Biology, Ruđer Bošković Institute, Zagreb, Croatia"/>
      <meta name="citation_author" content="Đurđica Ugarković"/>
            <meta name="citation_author_institution" content="Department of Molecular Biology, Ruđer Bošković Institute, Zagreb, Croatia"/>

    <meta name="citation_date" content="2008/2/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001594.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1594"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Centromeric heterochromatin: what makes it unique?; citation_author=S Henikoff; citation_author=Y Dalal; citation_journal_title=Curr Opin Genet Dev; citation_volume=15; citation_number=1; citation_pages=177-184; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Satellite DNAs.; citation_author=DM Skinner; citation_journal_title=BioSciences; citation_volume=27; citation_number=2; citation_pages=790-796; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Functional elements residing within satellite DNAs.; citation_author=Đ Ugarković; citation_journal_title=EMBO Rep; citation_volume=6; citation_number=3; citation_pages=1035-1039; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transcription of a satellite DNA on two Y chromosome loops of Drosophila melanogaster.; citation_author=S Bonaccorsi; citation_author=M Gatti; citation_author=C Pisano; citation_author=A Lohe; citation_journal_title=Chromosoma; citation_volume=99; citation_number=4; citation_pages=260-266; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Structure and function of Y chromosomal DNA. II. Analysis of lampbrush loop associated transcripts in nuclei of primary spermatocytes of Drosophila hydei by in situ hybridization using asymetric RNA probes of four different families of repetitive DNA.; citation_author=P Trapitz; citation_author=M Wlaschek; citation_author=H Bunemann; citation_journal_title=Chromosoma; citation_volume=96; citation_number=5; citation_pages=159-170; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Transcription of four satellite DNA subfamilies in Diprion pini (Hymenoptera, Symphyta, Diprionidae).; citation_author=F Roulex-Bonnin; citation_author=S Renault; citation_author=Y Bigot; citation_author=G Periquet; citation_journal_title=Eur J Biochem; citation_volume=238; citation_number=6; citation_pages=752-759; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Genomic organization and transcription of satellite DNA in the ant Aphaenogaster subterranea (Hymenoptera, Formicidae).; citation_author=P Lorite; citation_author=S Renault; citation_author=F Roulex-Bonnin; citation_author=Y Bigot; citation_author=G Periquet; citation_journal_title=Genome; citation_volume=45; citation_number=7; citation_pages=609-616; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Satellite DNA transcription in Diadromus pulchellus (Hymenoptera).; citation_author=S Renault; citation_author=F Roulex-Bonnin; citation_author=G Periquet; citation_author=Y Bigot; citation_journal_title=Insect Biochem Mol Biol; citation_volume=29; citation_number=8; citation_pages=103-111; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=RNA recognition motif 2 directs the recruitment of SF2/ASF to nuclear stress bodies.; citation_author=I Chiodi; citation_author=M Corioni; citation_author=M Giordano; citation_author=R Valgardsdottir; citation_author=C Ghigna; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=9; citation_pages=4127-4136; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A key role for stress-induced satellite III transcripts in the relocalization of splicing factors into nuclear stress granules.; citation_author=A Metz; citation_author=J Soret; citation_author=C Vourc'h; citation_author=J Tazi; citation_author=C Jolly; citation_journal_title=J Cell Sci; citation_volume=117; citation_number=10; citation_pages=4551-4558; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Regulation of heterochromatic silencing and histone H3 lysine-9 methylation by RNAi.; citation_author=TA Volpe; citation_author=C Kidner; citation_author=IM Hall; citation_author=G Teng; citation_author=SIS Grewal; citation_journal_title=Science; citation_volume=297; citation_number=11; citation_pages=1833-1837; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=RNAi-mediated targeting of heterochromatin with the RITS complex.; citation_author=A Verdel; citation_author=S Jia; citation_author=S Gerber; citation_author=T Suglyama; citation_author=S Gygi; citation_journal_title=Science; citation_volume=303; citation_number=12; citation_pages=672-676; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=RNA meets heterochromatin.; citation_author=E Bernstein; citation_author=CD Allis; citation_journal_title=Genes Dev; citation_volume=19; citation_number=13; citation_pages=1635-1655; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Centromere RNA is a key component for the assembly of nucleoproteins at the nucleolus and centromere.; citation_author=LH Wong; citation_author=KH Brettingham-Moore; citation_author=L Chan; citation_author=JM Quach; citation_author=MA Anderson; citation_journal_title=Genome Res; citation_volume=17; citation_number=14; citation_pages=1146-1160; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Sequence of PRAT satellite DNA “frozen” in some coleopteran species.; citation_author=B Mravinac; citation_author=M Plohl; citation_author=N Meštrović; citation_author=Đ Ugarković; citation_journal_title=J Mol Evol; citation_volume=54; citation_number=15; citation_pages=774-783; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Detection of satellite DNA in Palorus ratzeburgii: Analysis of curvature profiles and comparison with Tenebrio molitor satellite DNA.; citation_author=Đ Ugarković; citation_author=M Plohl; citation_author=V Lucijanić-Justić; citation_author=B Borštnik; citation_journal_title=Biochimie; citation_volume=74; citation_number=16; citation_pages=1075-1082; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Evolution of satellite DNAs from the genus Palorus–experimental evidence for the library hypothesis.; citation_author=N Meštrović; citation_author=M Plohl; citation_author=B Mravinac; citation_author=Đ Ugarković; citation_journal_title=Mol Bio. Evol; citation_volume=15; citation_number=17; citation_pages=1062-1068; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Preservation and high sequence conservation of satellite DNAs suggest functional constraints.; citation_author=B Mravinac; citation_author=M Plohl; citation_author=Đ Ugarković; citation_journal_title=J Mol Evol; citation_volume=61; citation_number=18; citation_pages=542-550; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RNA polymerase is required for RNAi-dependent heterochromatin assembly.; citation_author=H Kato; citation_author=DB Goto; citation_author=RA Martienssen; citation_author=T Urano; citation_author=K Furukawa; citation_author=Y Murakami; citation_journal_title=Science; citation_volume=309; citation_number=19; citation_pages=467-469; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Proliferation-dependent and cell cycle-regulated transcription of mouse pericentromeric heterochromatin.; citation_author=J Lu; citation_author=DM Gilbert; citation_journal_title=J Cell Biol; citation_volume=179; citation_number=20; citation_pages=411-421; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Single–Step Method of RNA Isolation by Acid Guanidinium Thiocyanate-Phenol-Chloroform Extraction.; citation_author=P Chomczynski; citation_author=N Sacchi; citation_journal_title=Anal Biochem; citation_volume=162; citation_number=21; citation_pages=156-159; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=A Species of Small Antisense RNA in Posttranscriptional Gene Silencing in Plants.; citation_author=AJ Hamilton; citation_author=DC Baulcombe; citation_journal_title=Science; citation_volume=286; citation_number=22; citation_pages=950-952; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Application of a time-delay neural network to promoter annotation in the Drosophila melanogaster genome.; citation_author=MG Reese; citation_journal_title=Comput Chem; citation_volume=26; citation_number=23; citation_pages=51-56; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A method for parental RNA interference in the wasp Nasonia vitripennis.; citation_author=JA Lynch; citation_author=C Desplan; citation_journal_title=Nature Protocols; citation_volume=1; citation_number=24; citation_pages=486-494; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Differential regulation of strand-specific transcripts from Arabidopsis centromeric satellite repeat.; citation_author=BP May; citation_author=ZB Lippman; citation_author=Y Fang; citation_author=DL Spector; citation_author=RA Martienssen; citation_journal_title=PLoS Genet.; citation_volume=1; citation_number=25; citation_pages=e79; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transcription and evolutionary dynamics of the centromeric satellite repeat CentO in rice.; citation_author=HR Lee; citation_author=P Neumann; citation_author=J Macas; citation_author=J Jiang; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=26; citation_pages=2505-2520; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The assembly and maintenance of heterochromatin initiated by transgene repeats are independent of the RNA interference pathway in mammalian cells.; citation_author=F Wang; citation_author=N Koyama; citation_author=H Nishida; citation_author=T Haraguchi; citation_author=W Reith; citation_journal_title=Mol Cell Biol; citation_volume=26; citation_number=27; citation_pages=4028-4040; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Centromere-encoded RNAs are integral components of the maize kinetochore.; citation_author=CN Topp; citation_author=CX Zhong; citation_author=RK Dawe; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=101; citation_number=28; citation_pages=15986-15991; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Accumulation of small murine minor satellite transcripts leads to impaired centromeric architecture and function.; citation_author=H Bouzinba-Segard; citation_author=A Guais; citation_author=C Francastel; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=29; citation_pages=8709-8714; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Schistosome satellite DNA encodes active hammerhead-ribozymes.; citation_author=G Ferbeyre; citation_author=JM Smith; citation_author=R Cedergren; citation_journal_title=Mol Cell Biol; citation_volume=18; citation_number=30; citation_pages=3880-3888; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Transcription of satellite 2 DNA from the newt is driven by a snRNA type of promoter.; citation_author=SR Coats; citation_author=Y Zhang; citation_author=LM Epstein; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=31; citation_pages=4697-4704; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Evolution of heterochromatic genes of Drosophila.; citation_author=JC Yasuhara; citation_author=CH DeCrease; citation_author=BT Wakimoto; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=32; citation_pages=10958-10963; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A heterochromatin barrier partitions the fission yeast centromere into discrete chromatin domains.; citation_author=KC Scott; citation_author=SL Merrett; citation_author=HF Willard; citation_journal_title=Curr Biol; citation_volume=16; citation_number=33; citation_pages=119-129; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A role for TFIIIC transcription factor complex in genome organization.; citation_author=K Noma; citation_author=HP Cam; citation_author=RJ Maraia; citation_author=SIS Grewal; citation_journal_title=Cell; citation_volume=125; citation_number=34; citation_pages=859-872; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Widespread use of TATA elements in the core promoters for RNA polymerases III, II and I in fission yeast.; citation_author=M Hamada; citation_author=Y Huang; citation_author=TM Lowe; citation_author=RJ Maraia; citation_journal_title=Mol Cell Biol; citation_volume=21; citation_number=35; citation_pages=6870-6881; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001594" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="RNA Pol II Promotes Transcription of Centromeric Satellite DNA in Beetles"/>
    <meta name="twitter:description" content="Transcripts of centromeric satellite DNAs are known to play a role in heterochromatin formation as well as in establishment of the kinetochore. However, little is known about basic mechanisms of satellite DNA expression within constitutive heterochromatin and its regulation. Here we present comprehensive analysis of transcription of abundant centromeric satellite DNA, PRAT from beetle Palorus ratzeburgii (Coleoptera). This satellite is characterized by preservation and extreme sequence conservation among evolutionarily distant insect species. PRAT is expressed in all three developmental stages: larvae, pupae and adults at similar level. Transcripts are abundant comprising 0.033% of total RNA and are heterogeneous in size ranging from 0.5 kb up to more than 5 kb. Transcription proceeds from both strands but with 10 fold different expression intensity and transcripts are not processed into siRNAs. Most of the transcripts (80%) are not polyadenylated and remain in the nucleus while a small portion is exported to the cytoplasm. Multiple, irregularly distributed transcription initiation sites as well as termination sites have been mapped within the PRAT sequence using primer extension and RLM-RACE. The presence of cap structure as well as poly(A) tails in a portion of the transcripts indicate RNA polymerase II&ndash;dependent transcription and a putative polymerase II promoter site overlaps the most conserved part of the PRAT sequence. The treatment of larvae with alpha-amanitin decreases the level of PRAT transcripts at concentrations that selectively inhibit pol II activity. In conclusion, stable, RNA polymerase II dependant transcripts of abundant centromeric satellite DNA, not regulated by RNAi, have been identified and characterized. This study offers a basic understanding of expression of highly abundant heterochromatic DNA which in beetle species constitutes up to 50% of the genome."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001594.g006"/>

  <meta property="og:title" content="RNA Pol II Promotes Transcription of Centromeric Satellite DNA in Beetles" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001594" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=506'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5797'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9097&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001594">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001594" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001594&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    RNA Pol II Promotes Transcription of Centromeric Satellite DNA in Beetles
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Željka Pezer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Biology, Ruđer Bošković Institute, Zagreb, Croatia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Đurđica Ugarković
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:ugarkov@.irb.hr">ugarkov@.irb.hr</a></p>

                <p>Affiliation:
                  Department of Molecular Biology, Ruđer Bošković Institute, Zagreb, Croatia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 13, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001594</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001594" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001594" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001594" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001594" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001594-g001" data-doi="info:doi/10.1371/journal.pone.0001594" data-uri="info:doi/10.1371/journal.pone.0001594.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001594-g002" data-doi="info:doi/10.1371/journal.pone.0001594" data-uri="info:doi/10.1371/journal.pone.0001594.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001594-g003" data-doi="info:doi/10.1371/journal.pone.0001594" data-uri="info:doi/10.1371/journal.pone.0001594.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001594-g004" data-doi="info:doi/10.1371/journal.pone.0001594" data-uri="info:doi/10.1371/journal.pone.0001594.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001594-g005" data-doi="info:doi/10.1371/journal.pone.0001594" data-uri="info:doi/10.1371/journal.pone.0001594.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001594-g006" data-doi="info:doi/10.1371/journal.pone.0001594" data-uri="info:doi/10.1371/journal.pone.0001594.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001594">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001594" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Transcripts of centromeric satellite DNAs are known to play a role in heterochromatin formation as well as in establishment of the kinetochore. However, little is known about basic mechanisms of satellite DNA expression within constitutive heterochromatin and its regulation. Here we present comprehensive analysis of transcription of abundant centromeric satellite DNA, PRAT from beetle <em>Palorus ratzeburgii</em> (Coleoptera). This satellite is characterized by preservation and extreme sequence conservation among evolutionarily distant insect species. PRAT is expressed in all three developmental stages: larvae, pupae and adults at similar level. Transcripts are abundant comprising 0.033% of total RNA and are heterogeneous in size ranging from 0.5 kb up to more than 5 kb. Transcription proceeds from both strands but with 10 fold different expression intensity and transcripts are not processed into siRNAs. Most of the transcripts (80%) are not polyadenylated and remain in the nucleus while a small portion is exported to the cytoplasm. Multiple, irregularly distributed transcription initiation sites as well as termination sites have been mapped within the PRAT sequence using primer extension and RLM-RACE. The presence of cap structure as well as poly(A) tails in a portion of the transcripts indicate RNA polymerase II–dependent transcription and a putative polymerase II promoter site overlaps the most conserved part of the PRAT sequence. The treatment of larvae with alpha-amanitin decreases the level of PRAT transcripts at concentrations that selectively inhibit pol II activity. In conclusion, stable, RNA polymerase II dependant transcripts of abundant centromeric satellite DNA, not regulated by RNAi, have been identified and characterized. This study offers a basic understanding of expression of highly abundant heterochromatic DNA which in beetle species constitutes up to 50% of the genome.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Pezer Ž, Ugarković Đ (2008) RNA Pol II Promotes Transcription of Centromeric Satellite DNA in Beetles. PLoS ONE 3(2):
          e1594.
            doi:10.1371/journal.pone.0001594</p><p><strong>Academic Editor: </strong>Beth Sullivan, Duke University, United States of America</p><p><strong>Received:</strong> November 14, 2007; <strong>Accepted:</strong> January 17, 2008; <strong>Published:</strong> February 13, 2008</p><p><strong>Copyright:</strong> © 2008 Pezer, Ugarković. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Grants 00982604 from the Croatian Ministry of Science and EU FP6 Marie Curie Transfer of Knowledge Grant MTKD-CT-2006-042248</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Satellite DNAs are tandemly repeated sequences that are in the form of long, Mb size arrays located in heterochromatin. Besides being a major constituent of heterochromatin they act as a centromere-building element in many eukaryotes <a href="#pone.0001594-Henikoff1">[1]</a>. Their sequence structure based often on simple repeats as well as heterochromatic localization lead to the belief that they are not transcribed <a href="#pone.0001594-Skinner1">[2]</a>. However, transcripts of satellite DNAs have been reported in various species groups including vertebrates, invertebrates and plants [reviewed in 3]. In invertebrates, satellite transcripts were observed on Y chromosome loops in primary spermatocyte nuclei of <em>Drosophila melanogaster</em> <a href="#pone.0001594-Bonaccorsi1">[4]</a> and <em>Drosophila hydei</em> <a href="#pone.0001594-Trapitz1">[5]</a>. Transcription of satellite DNAs was found often in species of Hymenoptera <a href="#pone.0001594-RoulexBonnin1">[6]</a>, <a href="#pone.0001594-Lorite1">[7]</a>. Some satellite DNA transcripts were found exclusively in nuclei such as <em>D. melanogaster</em> and <em>D. hydei</em> Y chromosome associated transcripts <a href="#pone.0001594-Bonaccorsi1">[4]</a>, <a href="#pone.0001594-Trapitz1">[5]</a> while in most of the cases the transcripts were found as polyadenylated RNA in the cytoplasm. Transcripts exhibit high size heterogeneity and are found in some cases to be strand specific like in wasp <em>Diadromus pulchellus</em> <a href="#pone.0001594-Renault1">[8]</a>, or are transcribed on both DNA strands like in ant <em>Aphaenogaster subterranea</em> <a href="#pone.0001594-Lorite1">[7]</a> and wasp <em>Diprion pini</em> <a href="#pone.0001594-RoulexBonnin1">[6]</a>. The existence of stage specific, differentially expressed transcripts in a number of species is consistent with their regulatory role which is however mostly not elucidated. Recently it has been shown that human satellite III transcripts, induced by stress, recruit the splicing factors to nuclear stress granules regulating in this way splicing function <a href="#pone.0001594-Chiodi1">[9]</a>, <a href="#pone.0001594-Metz1">[10]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Studies of fission yeast revealed the processing of long double strand transcripts deriving from tandemly repeated pericentromeric DNA into small 20–25 bp long RNAs involved in RNA interference (RNAi) pathway <a href="#pone.0001594-Volpe1">[11]</a>. RNAi guides histone modification, in particular methylation of H3 at lysine 9 (H3K9me2) which is characteristic for heterochromatin, via an interaction between complexes containing small interfering RNAs (siRNAs) and nascent transcripts at the locus <a href="#pone.0001594-Verdel1">[12]</a>. In addition to fission yeast the connection between RNAi and centromeric heterochromatin formation has been described for different systems such as plants, insects and mammals <a href="#pone.0001594-Bernstein1">[13]</a>. Recently it has been shown that in addition to small interfering RNAs deriving from alpha satellite DNA, long, single stranded alpha satellite DNA transcripts encompassing a few satellite monomers are functional components of the human kinetochore <a href="#pone.0001594-Wong1">[14]</a>. This alpha satellite RNA is required for the association of kinetochore proteins CENP-C1 and INCENP in the human interphase nucleolus as well as at the centromere.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Despite high evolutionary dynamics some satellite DNAs such as those belonging to the insect genus <em>Palorus</em> (Tenebrionidae, Coleoptera) are preserved and widely distributed among taxonomically distant species <a href="#pone.0001594-Mravinac1">[15]</a>. PRAT represents the major satellite in the species <em>Palorus ratzeburgii</em> comprising 40% of the genomic DNA and is located in regions of centromeric and pericentromeric heterochromatin on all chromosomes <a href="#pone.0001594-Ugarkovi2">[16]</a>. PRAT is distributed among all congeneric species and far beyond the level of genus in the form of low copy number satellite preserving centromeric and pericentromeric location <a href="#pone.0001594-Metrovi1">[17]</a>. The sequence of PRAT cloned from species that are separated by a significant evolutionary period of about 50–60 Myr showed high conservation without any fixed species specific mutation <a href="#pone.0001594-Mravinac1">[15]</a>, <a href="#pone.0001594-Mravinac2">[18]</a>. It was proposed that long evolutionary preservation and sequence conservation of PRAT suggests a possible functional significance.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Most of current research is focused on the explanation of role of satellite DNA transcripts but little is known about the transcription itself although it is likely to be the key regulatory step in biogenesis of functional transcripts. In the case of fission yeast <em>Schizosaccharomyces pombe</em> it is shown that RNA pol II promotes transcription of centromeric DNA <a href="#pone.0001594-Kato1">[19]</a>. Transcription of γ satellite DNA which is a major constituent of mouse pericentromeric heterochromatin is also mediated by RNA pol II <a href="#pone.0001594-Lu1">[20]</a>. The aim of this study is to characterize transcripts of abundant centromeric satellite DNA of beetles and to identify the basic mechanism of satellite DNA transcription. This can help to understand the mechanisms of regulation of expression of highly abundant satellite DNA sequences which in beetle species constitute up to 50% of the genome.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>RNA isolation</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Total, cytoplasmic and nuclear RNAs for the purpose of RT-PCR, real-time PCR, Northern and RLM-RACE analysis were isolated using the Rneasy Mini Kit (Qiagen), whereas polyadenylated RNA was recovered on Oligotex columns (Qiagen). RNA for primer extension reaction as well as RNA to be used for detection of siRNAs on Northern blots were extracted by phenol-chloroform <a href="#pone.0001594-Chomczynski1">[21]</a>. RNA preparations were enriched for small-sized RNA by removing high molecular weight molecules by precipitation with 5% polyethylene glycol and 0.5 M NaCl <a href="#pone.0001594-Hamilton1">[22]</a>. To eliminate traces of co-extracted DNA all preparations were treated with up to 0.5 units of DNase I per microgram of RNA prior to further reactions.</p>


<h4>RT-PCR</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>20 ng of RNA was transcribed into DNA and amplified using the OneStep RT-PCR Kit (Qiagen) and primers RATZEB1 (<span class="gene">5′-GAATCCGAAACAAAATCGTGCC-3′</span>) and RATZEB2 (<span class="gene">5′-GCTGAAATCTTGCAAACTTTAC-3′</span>) specific to the PRAT. To investigate strand origin of transcripts, reverse transcription was performed at 50°C for 30 min with one primer only, followed by heat inactivation of reverse-transcriptase and activation of HotStarTaq DNA Polymerase for 15 min at 95°C. At this step the second primer was added to start 33 cycles of PCR under the following conditions: 1 min at 94°C, 30 s at 58°C and 1 min at 72°C. Final extension was done for 10 min at 72°C. Reactions in which reverse transcription was omitted as well as reactions with no added template were performed as controls for DNA contamination and cross-contamination between samples.</p>


<h4>Northern and dot blot analysis</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>RNA was separated on 1.2% agarose-formaldehyde gels and blotted onto positively charged nylon membranes. For small RNA analysis, 30 µg of RNA enriched for low molecular weight molecules was resolved on 12% polyacrylamide gels containing 8 M urea and transferred to nylon membranes in 0.5× TBE for 45 min at 100 V using wet electroblotter (BioRad). In dot blot analysis, samples of RNA were applied to membrane together with the equal amounts of yeast RNA which served as a control for hybridization specificity. Unlabelled “run off” transcripts produced in vitro from vector containing PRAT dimer sequence were used as positive controls and for calibration after hybridization. Hybridization was performed under high stringency conditions at 50°C and 60°C on dot blots and high molecular weight RNAs, and at 22°C and 40°C on blots with small RNAs, in 10 ml solution containing 50% formamide, 5× Denhardt solution, 6× SSPE, 0.1% SDS, 100µg/ml herring sperm DNA, and 1-2×10<sup>7</sup> cpm of DNA probe labelled by random priming or in vitro transcribed RNA probe. To produce strand specific RNA probes, pGEM-T vector (Promega), with previously cloned PRAT dimer, was digested with <em>Nco</em>I or <em>Sal</em>I and in vitro transcribed by T7 or SP6 RNA polymerase, respectively, in a reaction containing [α-<sup>32</sup>P]UTP. After hybridization, membranes were washed twice in 2× SSC, 0.1% SDS at RT, once in 0.1× SSC, 0.1% SDS at 37°C, and exposed overnight on a phosphor-imager or up to 7 days on autoradiography films. Relative amounts of PRAT transcripts were determined by densitometry, subtracted for the signal corresponding to yeast RNA.</p>


<h4>Primer extension</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Four oligonucleotides, RATZEB1, RATZEB2, RATZEB3 (<span class="gene">5′-GCTGTATTTAAG AGAATCCG-3′</span>) and RATZEB4 (<span class="gene">5′-ATACAGCTGAAACCATGC-3′</span>) corresponding to PRAT sequence were end-labelled with [γ-<sup>32</sup>P]dATP in phosphorylation reaction with T4 polynucleotide kinase (Fermentas). Each of the labelled primers was incubated with total RNA (10–30 µg) at 65°C for 90 min and slowly cooled down to RT to allow hybridization of primer to RNA. 25 U of AMV reverse transcriptase (Roche) was added to 40 µl reaction and incubated at 42°C for 1 h. The reaction was terminated by incubation with RNase A at 37°C for 30 min, and reverse transcripts were recovered by phenol-chloroform extraction and ethanol precipitation. Primer extension products were electrophoresed through a 6% denaturing polyacrylamide gel along with the sequencing products of PRAT dimer clone obtained with the same labelled oligonucleotide used for primer extension.</p>


<h4>RNA Ligase Mediated Rapid Amplification of cDNA Ends (RLM-RACE)</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>In order to identify specific 5′ capped transcripts, potentially transcribed by RNA polymerase II, a 5′ RACE reaction was conducted on total RNA from larvae according to manufacturer's instructions (Ambion). After phosphatase and pyrophosphatase treatment, the RNA adapter was ligated to the 5′ end of decapped target molecules. RT-PCR was performed using the OneStep RT-PCR Kit as described above, with an outer adapter-specific primer and RATZEB1 or RATZEB2 primer. Additional PCR on obtained products was conducted with either RATZEB1 or RATZEB2 satellite specific primer, and inner adapter-specific primer to increase specificity of amplification. Simultaneous experiment without pyrophosphatase treatment (-TAP) served as a negative control. The same kit was used for 3′ RACE reactions to detect transcription termination sites. Total RNA from larvae was reverse transcribed using the supplied 3′ RNA adapter which has an oligo T stretch, and subsequently amplified using adapter specific outer primer and RATZEB1 or RATZEB2 primer.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>All PCR products from both 5′ and 3′ RACE were cloned in the pGEM-T Easy vector (Promega), positive clones were screened by colony lift and hybridization with PRAT probe, followed by sequencing.</p>


<h4>Cloning and sequence analysis</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>A primer extension product of 122 nt size, obtained in reactions with RATZEB2, was amplified by using RATZEB1 and RATZEB2 primers. Amplified PCR product of 131 bp was directly sequenced using RATZEB2 as a sequencing primer. The same amplification product was ligated into the pGEM-T vector and transformed into <em>E. coli</em>. Eleven clones were randomly chosen and sequenced in both directions using vector-specific primers. Accession numbers of PRAT RNA monomers obtained by direct sequencing and cloning are EF209021-EF209033. Nucleotide sequences were aligned and compared using ClustalW. Putative RNA Polymerase II promoter site was annotated using Neural Network Promoter Prediction Tool (<a href="http://www.fruitfly.org/seq_tools/promoter.html">http://www.fruitfly.org/seq_tools/promot​er.html</a>; 23).</p>


<h4>Injection of larvae</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Late instar larvae of <em>P.ratzeburgii</em> were cooled down on ice and fixed onto a double-stick tape placed over a micro slide. Larvae were injected dorsally into the hemolymph using a simple injection apparatus <a href="#pone.0001594-Lynch1">[24]</a>. 23–26 larvae were injected with 0,1–0,15 µl per larvae of injection buffer (1,4 mM NaCl; 0,07 mM Na<sub>2</sub>HPO<sub>4</sub>; 0,03 mM KH<sub>2</sub>PO<sub>4</sub>; 4 mM KCl; 5% green food dye) or alpha-amanitin, 1 µg/ml and 10 µg/ml, diluted in injection buffer. After injection larvae were removed from the tape and grown on whole wheat flour at 30°C for five days. No mortality or toxic effects were noted for larvae treated for five days with such concentrations of alpha-amanitin. The low mortality was observed for larvae injected with 100 µg/ml of alpha-amanitin.</p>


<h4>Real-time PCR</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>0.5 µg of total RNA isolated from injected larvae was reverse transcribed by 20 U of M-MuLV Reverse Transcriptase (Fermentas) using random primers in a final volume of 20 µl. Resulting cDNA (20–40 ng per reaction) was subjected to real-time quantitative PCR analysis. Primers and TaqMan probes for PRAT as well as for 18S rRNA (endogenous control) were obtained from Applied Biosystems as TaqMan Gene Expression Assays. All reactions were performed on ABI 7300 Real-Time PCR System in four replicates in a final volume of 50 µl, under following reaction conditions: pre-incubation at 50°C for 2 min, 95°C for 10 min and 40 cycles of 95°C for 15 sec and 60°C for 1 min. Non-template controls were included. The data were analyzed using the comparative C<sub>T</sub> method (ΔΔ C<sub>T</sub>). cDNA from larvae injected with buffer served as calibrator and 18S rRNA gene was used to normalize PRAT quantities.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>PRAT transcripts are heterogeneous in size and differentially expressed from both strands</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>RT-PCR with primers specific for PRAT satellite DNA revealed the presence of transcripts in total RNA isolated from <em>P. ratzeburgii</em> adults, larvae and pupae (<a href="#pone-0001594-g001">Fig. 1a</a>). The amplified fragments are of expected size corresponding to 142 bp PRAT satellite monomer and its multimers, indicating that transcripts do not terminate within each monomer but are composed of regularly arranged tandem repeats characteristic for PRAT satellite DNA. To ensure that the amplification results are from reverse transcribed RNA and not from residual genomic DNA, each reaction was conducted simultaneously without prior reverse transcription. Similar amplification profiles were obtained using primers complementary to either forward or reverse strand in reverse transcriptase reaction indicating that transcription of PRAT proceeds from both DNA strands.</p>
<div class="figure" id="pone-0001594-g001"><div class="img"><a name="pone-0001594-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001594" data-uri="info:doi/10.1371/journal.pone.0001594.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001594.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001594.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Detection of PRAT satellite transcripts.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p><em>(A)</em> RT-PCR with primers RATZEB1 and RATZEB2 on total RNA from adults (ad), larvae (lar) and pupae (pup). For each reaction PCR was conducted in parallel without prior reverse transcription as a control for genomic DNA contamination, while 0 is control with no template. M is size standard. <em>(B)</em> Total RNA separated on 1.2% denaturing agarose gel (18 µg and 13 µg from adults and larvae, respectively) and corresponding Northern blot after hybridization with <sup>32</sup>P labelled PRAT DNA probe. 18 µg of adult RNA digested with RNase A represents control for residual genomic DNA (+R) while 18 µg of <em>D. melanogaster</em> RNA (c) is used as a negative control. <em>(C)</em> Dot blot analysis of total RNA, using <sup>32</sup>P labelled RNA probes for forward and reverse strand, respectively. 20 µg and 50 µg of larval RNA digested with RNase A represent control for genomic DNA contamination (+RNase) while the same amounts of <em>S. cerevisiae</em> RNA are used as a negative control (−control). Complementary transcripts, in amounts indicated, are used as a positive control (+control) and for calibration. <em>(D)</em> 30 µg of larval low molecular weight RNA (lar) separated on 12% polyacrylamide gel and Northern blot analysis with RNA probe for reverse strand, labelled with <sup>32</sup>P. Hybridization was performed at 22°C and blot was exposed for one day on autoradiography film. Longer exposition as well as rehybridization with RNA probe for forward strand did not reveal signal in the siRNA region. <em>D. melanogaster</em> RNA (dm) serves as a size marker.</p>
<span>doi:10.1371/journal.pone.0001594.g001</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Transcripts were further characterized by Northern hybridization using DNA probe labelled with <sup>32</sup>P (<a href="#pone-0001594-g001">Fig. 1b</a>). Total RNA was isolated from three developmental stages, pretreated with DNase I, and hybridization was performed under high stringency conditions. The hybridization signal appears as a smear ranging from approx. 0.5 kb up to 5 kb without any distinct bands. The heterogeneous size profile of transcripts might result from their rapid processing or could be due to multiple initiation and termination sites. The hybridization profiles were very similar in all three developmental stages (hybridization for pupae not shown). To eliminate the possibility that the signal originates from highly abundant satellite DNA instead of RNA, the RNA samples were digested with RNase A and run on the gel as a control. No signal was detected on RNase A treated samples (<a href="#pone-0001594-g001">Fig. 1b</a>).</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Dot blot analysis of <em>P. ratzeburgii</em> RNA using strand specific RNA probes labelled in vitro by <sup>32</sup>P, revealed different quantities of transcripts from opposite DNA strands: satellite RNA corresponding to forward strand accounts for 0.03% of total RNA, while transcripts corresponding to the reverse strand constitute 0.003% of total RNA (<a href="#pone-0001594-g001">Fig. 1c</a>). Comparison of signals among different developmental stages revealed a similar level of PRAT expression in larvae and adults while the amount of PRAT transcripts from both strands was approx. 20% higher in pupae, indicating slight developmental regulation.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>The simultaneous presence of forward and reverse transcripts may activate the RNA interference (RNAi) pathway and produce small interfering RNAs (siRNA). To verify whether the PRAT transcripts were processed into siRNA, Northern blot hybridization using RNA preparations enriched for small RNA was performed (<a href="#pone-0001594-g001">Fig. 1d</a>). We were unable to detect PRAT satellite DNA-specific siRNA (21–25 nucleotides) deriving either from forward or reverse strand, despite using large quantities of enriched small RNA and employing sensitive hybridization conditions at 40°C. Lowering of stringency conditions for hybridization by reducing the hybridization temperature to 22°C also did not reveal the presence of siRNAs. These results indicate that PRAT is not processed using the RNAi mechanism or that siRNA are very rare, below the level of detection.</p>


<h4>PRAT transcripts are located both in the nucleus and in the cytoplasm and are partially polyadenylated</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Subcellular localization may provide clues to the function of satellite DNA transcripts. Therefore, nuclei were separated from cytoplasm of <em>P. ratzeburgii</em> larvae, and RNA was extracted from each fraction. In addition, polyadenylated RNA was separated from total RNA on Qiagen Oligotex columns and the remaining non-polyadenylated fraction was recovered. All RNA isolations were treated with DNase I to eliminate traces of genomic DNA and subjected to Northern blot hybridization using <sup>32</sup>P strand specific RNA probes (<a href="#pone-0001594-g002">Fig. 2a</a>). Hybridization size profiles obtained by both RNA probes were highly similar. Hybridization signals in the form of smears in the nuclear and cytoplasmic fraction are predominantly localized between 2 and 5 kb. In the non-polyadenylated fraction the majority of the signal is around 5 kb, while the polyadenylated RNA fraction is characterized with a hybridization smear of lower size, with maximal intensity around 3 kb.</p>
<div class="figure" id="pone-0001594-g002"><div class="img"><a name="pone-0001594-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001594" data-uri="info:doi/10.1371/journal.pone.0001594.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001594.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001594.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Subcellular localization and polyadenylation of transcripts.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p><em>(A)</em> Agarose gel electrophoresis of cytoplasmic (cy), nuclear (nu), polyadenylated (a+) and non-polyadenylated (a-) RNA and hybridization profile obtained with RNA probe for forward strand. 13 µg of cy and 1.3 µg of nu RNA extracted in parallel from the same sample of adults are loaded as well as 1 µg of a+ and 17 µg of a-. Cytoplasmic (13 µg) and non-polyadenylated RNAs (17 µg) treated with RNase A (+Rc and +Ra, respectively) serve as controls for genomic DNA contamination. <em>S. cerevisiae</em> RNA (c) represents negative hybridization control. <em>(B)</em> Dot blot analysis of total RNA (45 µg), polyadenylated RNA (a+, 6 µg), non-polyadenylated RNA (a-, 40 µg) using PRAT RNA probes for forward and reverse strand, as well as of cytoplasmic (cy) and nuclear (nu) RNA, in amounts indicated, using RNA probe for reverse strand. <em>S. cerevisiae</em> RNA is used as a negative control in amounts exactly matching those of above mentioned samples (−control). For the purpose of PRAT transcripts quantification in different RNA fractions, signal intensity was reduced for the signal detected in the corresponding negative control. Due to the low signal intensity in the a+ fraction of reverse strand precise quantification was not possible. Complementary PRAT transcripts, in amounts indicated, are used as a positive control (+control) and for calibration. Cytoplasmic and nuclear RNA digested with RNase A represent control for genomic DNA contamination (+RNase).</p>
<span>doi:10.1371/journal.pone.0001594.g002</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>In order to determine relative distribution of PRAT transcripts among different RNA fractions dot-blot hybridization was performed (<a href="#pone-0001594-g002">Fig. 2b</a>). Nuclear and cytoplasmic RNA fractions were isolated in parallel from the same sample of adult insects as well as polyadenylated and non-polyadenylated RNA. Comparison between hybridization signals revealed that the majority of PRAT transcripts, representing approximately 80% of the forward strand, are not polyadenylated, while the remaining 20% have poly(A) tails. Since expression of the reverse strand is 10 times lower relative to the forward one, the signal intensity was very low and not suitable for precise quantification. The comparison of hybridization signals between cytoplasmic and nuclear fractions reveal preferential localization of PRAT transcripts in the nucleus where approx. 80% of them are detected relative to 20% found in cytoplasm (<a href="#pone-0001594-g002">Fig. 2b</a>).</p>


<h4>Multiple transcription initiation are mapped within the PRAT monomer</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>A primer extension analysis was used for mapping the transcription initiation sites on PRAT satellite repeats. Oligonucleotides 5′-labelled with 32P were hybridized individually to total <em>P. ratzeburgii</em> RNA to initiate reverse transcription either from forward or reverse strand. Extended products were visualized on polyacrylamide gel and compared with satellite DNA sequence ladder obtained by the same primer on cloned satellite dimer. Primer extension using primer RATZEB1 revealed a single transcription start site on PRAT reverse strand (<a href="#pone-0001594-g003">Fig. 3a</a>) corresponding to position 46 on PRAT consensus sequence (<a href="#pone-0001594-g003">Fig. 3c</a>). This initiation site was confirmed by another primer RATZEB3 which hybridizes to PRAT upstream from RATZEB1 (not shown, <a href="#pone-0001594-g003">Fig. 3c</a>). Extensions on forward strand using primer RATZEB2 gave as a product strong band of 122 nt and a 108 nt band of low intensity (<a href="#pone-0001594-g003">Fig. 3b</a>), which correspond to positions 10 and 24, respectively, in the PRAT sequence (<a href="#pone-0001594-g003">Fig. 3c</a>). The same, strong initiation site at position 10 was also confirmed by primer RATZEB4 which is positioned upstream of the RATZEB2 (band 127, <a href="#pone-0001594-g003">Figs.3b, 3c</a>). However, primer RATZEB4 revealed three additional strong bands (<a href="#pone-0001594-g003">Fig. 3b</a>) which are also indicated as potential transcription initiation sites on forward strand at positions 16, 41 and 46 (<a href="#pone-0001594-g003">Fig.3c</a>).</p>
<div class="figure" id="pone-0001594-g003"><div class="img"><a name="pone-0001594-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001594" data-uri="info:doi/10.1371/journal.pone.0001594.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001594.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001594.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Primer extension analysis of PRAT satellite transcripts.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p><em>(A)</em> Single extension product is detected on reverse PRAT strand using RATZEB1 primer (line R). The size of the product of 97 nt is determined by comparison with the sequencing reaction of PRAT dimer obtained by the same RATZEB1 primer. <em>(B)</em> Several extension products are detected on the forward strand using primers RATZEB2 and RATZEB4, and their size is indicated (lines R). Products of 122 nt and 127 nt, obtained in a reactions with RATZEB2 and RATZEB4, respectively, correspond to the same start site. <em>(C)</em> Positioning of transcription start sites and polyadenylation sites on PRAT forward and reverse strand. Transcription start sites determined by primer extension are indicated by green bent arrows while those detected by 5′ RLM-RACE are shown by violet bent arrows. Dashed green arrow represents weaker start site corresponding to 108 nt band obtained in reaction with RATZEB2. The sizes of the products obtained in extension assays are indicated under the corresponding nucleotide positions. The lengths of the products from reactions with RATZEB2 are in brackets. Polyadenylation sites are indicated with asterisks. Straight arrows represent primers and are positioned under the sequence they anneal to. Position of putative polymerase II promoter predicted by a neural network method (Reese 2001) is marked in red.</p>
<span>doi:10.1371/journal.pone.0001594.g003</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>In order to confirm that distinct bands obtained by RT extension derive specifically from PRAT transcript we have performed PCR amplification of 122 nt band obtained by extension with RATZEB2 primer, using standard PRAT specific internal primers RATZEB1 and RATZEB2 for PCR reaction. A 131 nt long amplified product was directly sequenced using RATZEB2 as a sequencing primer. The same PCR product was cloned into an AT vector and 11 clones were subsequently sequenced. Sequence analysis revealed high homology and correspondence to randomly cloned monomers of PRAT satellite DNA, previously determined (not shown).</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Mapping of the transcription start sites was also performed using RLM-RACE which involves ligation of a defined RNA adapter to the 5′ ends of full length mRNA followed by RT-PCR using primers RATZEB1 and RATZEB2 for mapping the initiation sites on PRAT reverse and forward strand respectively. Products of amplification (<a href="#pone-0001594-g004">Fig. 4a</a>) were cloned into an AT vector, positive clones detected by hybridization and sequenced. All six positive clones obtained with the RATZEB2 primer have adapter ligated directly on the forward strand of PRAT satellite (<a href="#pone-0001594-g004">Fig. 4b</a>) but at different positions: 22, 24, 61, 73, 77 and 98 (<a href="#pone-0001594-g004">Fig 4b</a>, <a href="#pone-0001594-g003">Fig. 3c</a>). Among them only position 24 overlaps with the initiation site mapped by primer extension (108 nt band in <a href="#pone-0001594-g003">Fig. 3b</a>). The lengths of clones range from 71 nt to 201 nt, corresponding to amplification of satellite monomer and dimer (<a href="#pone-0001594-g004">Fig. 4b</a>). Although longer, multimeric stretches of tandem repeats were amplified (<a href="#pone-0001594-g004">Fig. 4a</a>) and subjected to ligation, they were not detected as positive clones, probably due to inefficient cloning and/or instability of clones. Using the RATZEB1 primer for mapping the initiation sites on the reverse strand, three clones were identified (<a href="#pone-0001594-g004">Fig. 4b</a>). Each clone corresponds to a different initiation site located at positions 28, 55 and 80 (<a href="#pone-0001594-g004">Fig 4b</a>, <a href="#pone-0001594-g003">Fig. 3c</a>). Neither of these sites overlaps with the single initiation site on the reverse strand as determined by primer extension.</p>
<div class="figure" id="pone-0001594-g004"><div class="img"><a name="pone-0001594-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001594" data-uri="info:doi/10.1371/journal.pone.0001594.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001594.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001594.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Mapping of transcription start sites by 5′ RLM-RACE.</span></strong></p><a id="article1.body1.sec3.sec3.fig2.caption1.p1" name="article1.body1.sec3.sec3.fig2.caption1.p1"></a><p><em>(A)</em> RT-PCR products of reactions with either RATZEB1 (left) or RATZEB2 (right) primer, resolved on agarose gel. –TAP lanes represent corresponding negative controls obtained without pyrophosphatase treatment. M is size standard. <em>(B)</em> Sequences of cloned 5′ RACE products obtained with RATZEB2 are aligned with forward strand of consensus PRAT dimer (above). Sequences obtained with RATZEB1 primer are compared to reverse strand of consensus PRAT monomer (below). Missing parts of sequences are marked by a dashed line. Accession numbers of sequences obtained by 5′RLM-RACE are EF209039-EF209047.</p>
<span>doi:10.1371/journal.pone.0001594.g004</span></div>

<h4>Multiple transcription termination sites exist within PRAT</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Transcription termination sites were mapped using 3′ RACE. Reverse transcription was performed using the 3′ RACE adapter which has an oligo T stretch, followed by PCR amplification with satellite specific primer RATZEB1 or RATZEB2, and primer homologous to the adapter. Amplification reactions of forward and reverse strand (<a href="#pone-0001594-g005">Fig. 5a</a>) were randomly cloned into the AT vector and positive clones detected by hybridization. Sequences of three clones corresponding to the forward strand encompass satellite monomer, dimer and trimer, respectively (<a href="#pone-0001594-g005">Fig. 5b</a>). All three have termination sites within the satellite, two polyadenylation sites are mapped at positions 6 and 7 while the third one is located at position 88 of the forward strand (<a href="#pone-0001594-g005">Fig. 5b</a>, <a href="#pone-0001594-g003">Fig. 3c</a>). Since position 7 in the PRAT sequence is immediately followed by a stretch of A nucleotides it is possible that it acts as a false poly(A) tail during reverse transcription. Two clones corresponding to the reverse strand (<a href="#pone-0001594-g005">Fig. 5b</a>) also end within satellite DNA, having poly(A) tails at positions 15 and 76 respectively (<a href="#pone-0001594-g005">Fig. 5b</a>, <a href="#pone-0001594-g003">Fig. 3c</a>).</p>
<div class="figure" id="pone-0001594-g005"><div class="img"><a name="pone-0001594-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001594" data-uri="info:doi/10.1371/journal.pone.0001594.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001594.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001594.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Mapping of polyadenylation sites by 3′ RLM-RACE.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p><em>(A)</em> PCR products of reaction with RATZEB1 (left) and RATZEB2 (right) resolved on agarose gel. –RT lanes represent corresponding negative control reactions in which reverse transcription was omitted. <em>(B)</em> Sequences of cloned 3′ RACE products obtained with RATZEB1 are shown above, aligned with forward strand of consensus PRAT trimer. Products of reaction with RATZEB2, depicted below, are compared to reverse strand of consensus PRAT monomer. Poly(A) tails are marked with grey shading. Missing parts of sequences are marked by a dashed line. Accession numbers of sequences obtained by 3′RLM-RACE are EF209034-EF209038.</p>
<span>doi:10.1371/journal.pone.0001594.g005</span></div><a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>As in the case of 5′ RACE assay, all positive clones detected in 3′ RACE analysis correspond to shorter fragments, up to trimer, although longer multimers have been subjected to ligation (<a href="#pone-0001594-g005">Fig. 5a</a>). This again suggests low efficiency of cloning of longer multimers and/or instability of clones.</p>


<h4>Transcription of PRAT satellite is sensitive to alpha-amanitin</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>The presence of a cap structure as well as poly(A) tails in at least a portion of the transcripts indicates that RNA polymerase II could be responsible for PRAT satellite DNA transcription. In order to test this assumption we injected beetle larvae with alpha-amanitin, which is known to inhibit RNA polymerase II completely at low concentration, while RNA polymerase I activity remains unaffected. Larvae were injected with approx. 0.1 ng and 1 ng of alpha-amanitin respectively, and these concentrations of alpha-amanitin caused no mortality or toxic effects. Five days after injection total RNA was isolated and treated with DNase I to remove traces of genomic DNA. RNA quality was checked by electrophoresis and cDNA library was generated. Real-time PCR analysis evidenced approximately 29% and 42% reduction in PRAT transcript quantity in larvae injected with 0.1 ng and 1 ng of alpha-amanitin, respectively (<a href="#pone-0001594-g006">Fig. 6</a>). 18S rRNA is known to be transcribed by pol I and its transcription was unaffected by inhibitor concentrations applied here, thus it was considered convenient for endogenous control in real-time PCR.</p>
<div class="figure" id="pone-0001594-g006"><div class="img"><a name="pone-0001594-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001594" data-uri="info:doi/10.1371/journal.pone.0001594.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001594.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001594.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001594.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001594.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001594.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Sensitivity of PRAT transcription to alpha-amanitin.</span></strong></p><a id="article1.body1.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec5.fig1.caption1.p1"></a><p>Results of quantitative real-time PCR analysis are shown. Transcription level of PRAT after injection with two concentrations of inhibitor (1 µg/ml and 10 µg/ml) is compared to the transcription level of PRAT from larvae injected with buffer. 18S rRNA gene was used as endogenous control to normalize PRAT quantities. Error bars represent standard deviations from two different real-time PCR experiments.</p>
<span>doi:10.1371/journal.pone.0001594.g006</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The results presented in this paper reveal that evolutionarily conserved PRAT satellite DNA is continuously expressed during all three developmental stages in the beetle <em>P. ratzeburgii</em>: larvae, pupae and adults, at a similar level. PRAT is predominantly transcribed from one strand and the transcripts are relatively abundant comprising 0.03% of total cellular RNA while expression of the opposite strand is 10 times lower. The transcripts are heterogeneous with the size profile ranging from approx. 0.5 kb to more than 5 kb, matching those characteristic for the centromeric, 180 bp satellite of <em>Arabidopsis thaliana</em> <a href="#pone.0001594-May1">[25]</a>. However, unlike <em>Arabidopsis</em> or rice centromeric satellites whose primary transcripts are regulated by RNAi and corresponding siRNAs are readily proved by Northern hybridization <a href="#pone.0001594-May1">[25]</a>, <a href="#pone.0001594-Lee1">[26]</a>, we were unable to detect PRAT satellite-specific siRNA despite using large quantities of enriched small RNA and employing sensitive hybridization conditions. Lowering of stringency conditions for hybridization also did not reveal any presence of siRNAs. The siRNA may be extremely rare or produced in specific tissues and therefore undetectable in RNA isolated from the whole specimen. Alternatively, a distinct mechanism of heterochromatin formation which does not rely on siRNA might be operating in beetles.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Although the RNAi pathway plays a role in heterochromatin establishment in various organisms, it does not seem to be indispensable for assembly and maintenance of heterochromatin, as shown for heterochromatin initiated in mammalian cells by transgene tandem repeats <a href="#pone.0001594-Wang1">[27]</a>. Transcription of major (γ) mouse satellite DNA located within pericentromeric heterochromatin is cell cycle-regulated and proceeds in the form of small, approximately 200 nt long RNA during mitosis while abundant, long, heterogenous transcripts are induced in G1 phase <a href="#pone.0001594-Lu1">[20]</a>. However, no evidence for siRNA-sized molecules at any time during the cell cycle exists, pointing to the difference in heterochromatin expression and establishment between mammals and fission yeast. In addition, numerous examples illustrate the involvement and importance of longer RNAs for kinetochore formation. RNA encoded by centromeric satellite DNA and retrotransposons, ranging in size between 40 and 200 nt, has been shown to be an integral component of the kinetochore in maize, tightly bound to centromeric histone H3 <a href="#pone.0001594-Topp1">[28]</a>. Murine minor satellite DNA associated with the centromeric region is transcribed from both strands and transcripts are processed into 120 nt RNA which localizes to centromeres <a href="#pone.0001594-BouzinbaSegard1">[29]</a>. The accumulation of satellite transcripts is impaired by mislocalization of centromere-associated proteins essential for the formation of centromeric heterochromatin. In addition, long transcripts of alpha satellite DNAs have been shown to be functional component of kinetochore participating in recruitment of kinetochore proteins <a href="#pone.0001594-Wong1">[14]</a>. These results indicate that in addition to RNAi moderating heterochromatin establishment in different systems an alternative mechanism involving longer RNA operates in kinetochore establishment.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Transcription of satellite DNA might be initiated from upstream promoters provided by mobile elements inserted near satellite DNA clusters <a href="#pone.0001594-May1">[25]</a>, <a href="#pone.0001594-Topp1">[28]</a>. However, there are cases when transcription of satellite DNA proceeds from internal promoters such as schistosome satellite having a functional RNA pol III promoter <a href="#pone.0001594-Ferbeyre1">[30]</a>, or newt satellite II with small nuclear RNA (snRNA) promoter that is responsible for RNA pol II transcription <a href="#pone.0001594-Coats1">[31]</a>. Primer extension (PE) and RLM-RACE were used for mapping transcription initiation sites of PRAT transcripts. The results revealed the presence of multiple initiation sites on both strands, all of them located within the satellite monomer. On the forward strand four major and one minor initiation sites were mapped by PE, while RLM-RACE disclosed the presence of five additional start sites. On the reverse strand a single initiation site identified by PE and three additional sites detected by RLM-RACE were mapped. All transcription start sites are probably not used with the same frequency and it is reasonable to propose that those detected by the not very sensitive method of primer extension are used most often. Multiple transcription initiation sites are characteristic for «slippery promoters», a common promoter type found in TATA-box-containing and TATA-less genes of <em>Drosophila</em>, located either in heterochromatin or euchromatin <a href="#pone.0001594-Yasuhara1">[32]</a>.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>The presence of a cap structure as well as poly(A) tails in a portion of the transcripts indicate that pol II could be responsible for PRAT satellite DNA transcription. The sensitivity of PRAT transcription to alpha-amanitin at concentrations that selectively inhibit pol II confirms this assumption. However, since a considerable amount of transcripts does not contain poly(A) tail, the possibility exists that RNA polymerase(s) other than pol II might also be involved in transcription. In the yeast <em>Schizosaccharomyces pombe</em> RNA polymerase II is responsible for transcription of pericentromeric regions which is coupled with RNAi-dependent heterochromatin assembly <a href="#pone.0001594-Kato1">[19]</a>. In addition, at the centromere of <em>S. pombe</em> association of RNA Pol III as well as Pol III transcripts of tRNA genes were identified <a href="#pone.0001594-Scott1">[33]</a>, <a href="#pone.0001594-Noma1">[34]</a>. The examination of PRAT sequence for the presence of A and B box consensus sites <a href="#pone.0001594-Hamada1">[35]</a> which are associated with RNA Pol III transcription was performed. Although no motifs of complete homology to either A or B box were found in PRAT consensus sequence, such motifs can be created by either a single point mutation (A box at position 122 of forward strand) or by two point mutations (B box at position 3 of forward strand). So, it can be proposed that some PRAT satellite monomers could have consensus sites for A and B boxes and are potentially transcribed by RNA Pol III.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>We did not experimentally determine the position of promoter elements but a putative RNA polymerase II promoter site was annotated by a computational approach, using a time-delay neural network method <a href="#pone.0001594-Reese1">[23]</a>. The putative promoter element is detected with a high score of 0.95 in 52% of PRAT DNA monomers as well as in 80% of PRAT RNA monomers, between positions 40–89 nt of reverse strand (<a href="#pone-0001594-g003">Fig. 3c</a>), overlapping exactly with the most conserved part of the PRAT sequence, as previously revealed by sequence comparison <a href="#pone.0001594-Mravinac2">[18]</a>. In a few monomers a putative promoter element is annotated with score of 0.65 between positions 50 and 100 nt in the forward strand (<a href="#pone-0001594-g003">Fig. 3c</a>), overlapping with the conserved part of PRAT sequence. Localization of a putative promoter within the conserved part of the PRAT satellite monomer sequence might indicate possible influence of selection pressure to preserve transcriptional activity of satellite DNA. However, the possibility that transcription proceeds from a promoter located upstream of satellite clusters can not be completely excluded.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>A motif exhibiting perfect homology to the TATA-box was not detected in the PRAT consensus sequence, but can be created by a single point mutation of a motif (TATTTAA) located at position 134 on the forward strand. Due to satellite sequence variability it is probably present in many PRAT monomers. Complementary sequence on opposite DNA strand (TTAAATA) also resembles TATA-box. These TATA-box like motifs are positioned 11 and 18 nt upstream of the most proximate transcription start site detected on forward and reverse strand, respectively. Overlapping of TATA-box like sequences on both DNA strands and close position of transcription initiation sites indicates bidirectional transcription from the same, central promoter.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>RACE analysis revealed the presence of multiple termination sites located within the PRAT sequence. The method specifically picks transcripts ending with poly(A) tails. Some of them could result from polyadenylation, while due to the presence of long A stretches in PRAT sequence it is possible that some of the analyzed transcripts have a false poly(A) tail. In any case, the length of poly(A) tail is relatively short, below 13 nt, similar to those found in rice satellite DNA transcripts <a href="#pone.0001594-Lee1">[26]</a>. A polyadenylation signal (AATAAA) can be created at position 94 of forward strand of PRAT consensus sequence by single point mutation, as well as at position 50 of the reverse strand by two mutations. Based on multiple, irregularly distributed transcription start and stop sites detected within PRAT sequence it can be concluded that they are the major cause of the observed size heterogeneity of transcripts. Although PRAT transcripts are large in size, up to 5 kb, RACE assay did not produce stretches of tandem repeats longer than satellite trimer what can be explained by inefficient cloning of long multimers as well as by their instability in plasmid. The rearrangement and instability of long satellite multimers cloned into plasmid vectors has been previously observed for PRAT satellite DNA and is probably characteristic of many other satellite DNAs. General lack of satellite DNA transcripts in cDNA libraries of different species is most probably related to their tandem repeat substructure which is not stable in most plasmid vectors.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>The function of PRAT transcripts remains highly speculative. Their preferential location in the nucleus might indicate close association with chromosomes and possible influence on chromatin modulation and centromere function. On the other hand, a small portion of PRAT transcripts which are located in the cytoplasm could perform an additional, yet unknown role. The presence of transcription initiation sites within the PRAT sequence as well as a putative promoter for RNA polymerase II point strongly to the functional significance of this evolutionary extremely conserved sequence and transcripts deriving from it. Identification of proteins, either nuclear or cytoplasmic, which interact with long PRAT satellite transcripts, could shed more light on the role of this abundant non-coding RNA and explain its long evolutionary preservation. On the other hand, further characterization of RNA polymerases and their promoters which are responsible for transcription of satellite DNAs could contribute to the elucidation of mechanisms of expression within condensed constitutive heterochromatin.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to Mary Sopta for critical reading of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DU. Performed the experiments: ZP. Analyzed the data: ZP. Contributed reagents/materials/analysis tools: DU. Wrote the paper: DU.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001594-Henikoff1" id="pone.0001594-Henikoff1"></a>Henikoff S, Dalal Y (2005) Centromeric heterochromatin: what makes it unique? Curr Opin Genet Dev  15: 177–184.  <ul class="find" data-citedArticleID="1055372" data-doi="10.1016/j.gde.2005.01.004"><li><a href="http://dx.doi.org/10.1016/j.gde.2005.01.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Centromeric+heterochromatin%3A+what+makes+it+unique%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Centromeric+heterochromatin%3A+what+makes+it+unique%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001594-Skinner1" id="pone.0001594-Skinner1"></a>Skinner DM (1977) Satellite DNAs. BioSciences  27: 790–796.  <ul class="find" data-citedArticleID="1055404" data-doi="10.2307/1297755"><li><a href="http://dx.doi.org/10.2307/1297755" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Satellite+DNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Satellite+DNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001594-Ugarkovi1" id="pone.0001594-Ugarkovi1"></a>Ugarković Đ (2005) Functional elements residing within satellite DNAs. EMBO Rep  6: 1035–1039.  <ul class="find" data-citedArticleID="1055410" data-doi="10.1038/sj.embor.7400558"><li><a href="http://dx.doi.org/10.1038/sj.embor.7400558" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+elements+residing+within+satellite+DNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+elements+residing+within+satellite+DNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001594-Bonaccorsi1" id="pone.0001594-Bonaccorsi1"></a>Bonaccorsi S, Gatti M, Pisano C, Lohe A (1990) Transcription of a satellite DNA on two Y chromosome loops of <em>Drosophila melanogaster</em>. Chromosoma  99: 260–266.  <ul class="find" data-citedArticleID="1055356" data-doi="10.1007/bf01731701"><li><a href="http://dx.doi.org/10.1007/bf01731701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+of+a+satellite+DNA+on+two+Y+chromosome+loops+of+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+of+a+satellite+DNA+on+two+Y+chromosome+loops+of+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001594-Trapitz1" id="pone.0001594-Trapitz1"></a>Trapitz P, Wlaschek M, Bunemann H (1988) Structure and function of Y chromosomal DNA. II. Analysis of lampbrush loop associated transcripts in nuclei of primary spermatocytes of <em>Drosophila hydei</em> by in situ hybridization using asymetric RNA probes of four different families of repetitive DNA. Chromosoma  96: 159–170.  <ul class="find" data-citedArticleID="1055408" data-doi="10.1007/bf00331048"><li><a href="http://dx.doi.org/10.1007/bf00331048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+function+of+Y+chromosomal+DNA.+II.+Analysis+of+lampbrush+loop+associated+transcripts+in+nuclei+of+primary+spermatocytes+of+Drosophila+hydei+by+in+situ+hybridization+using+asymetric+RNA+probes+of+four+different+families+of+repetitive+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+function+of+Y+chromosomal+DNA.+II.+Analysis+of+lampbrush+loop+associated+transcripts+in+nuclei+of+primary+spermatocytes+of+Drosophila+hydei+by+in+situ+hybridization+using+asymetric+RNA+probes+of+four+different+families+of+repetitive+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001594-RoulexBonnin1" id="pone.0001594-RoulexBonnin1"></a>Roulex-Bonnin F, Renault S, Bigot Y, Periquet G (1996) Transcription of four satellite DNA subfamilies in <em>Diprion pini</em> (Hymenoptera, Symphyta, Diprionidae). Eur J Biochem  238: 752–759.  <ul class="find" data-citedArticleID="1055400" data-doi="10.1111/j.1432-1033.1996.0752w.x"><li><a href="http://dx.doi.org/10.1111/j.1432-1033.1996.0752w.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+of+four+satellite+DNA+subfamilies+in+Diprion+pini+%28Hymenoptera%2C+Symphyta%2C+Diprionidae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+of+four+satellite+DNA+subfamilies+in+Diprion+pini+%28Hymenoptera%2C+Symphyta%2C+Diprionidae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001594-Lorite1" id="pone.0001594-Lorite1"></a>Lorite P, Renault S, Roulex-Bonnin F, Bigot Y, Periquet G, et al.  (2002) Genomic organization and transcription of satellite DNA in the ant <em>Aphaenogaster subterranea</em> (Hymenoptera, Formicidae). Genome  45: 609–616.  <ul class="find" data-citedArticleID="1055378" data-doi="10.1139/g02-022"><li><a href="http://dx.doi.org/10.1139/g02-022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+organization+and+transcription+of+satellite+DNA+in+the+ant+Aphaenogaster+subterranea+%28Hymenoptera%2C+Formicidae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+organization+and+transcription+of+satellite+DNA+in+the+ant+Aphaenogaster+subterranea+%28Hymenoptera%2C+Formicidae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001594-Renault1" id="pone.0001594-Renault1"></a>Renault S, Roulex-Bonnin F, Periquet G, Bigot Y (1999) Satellite DNA transcription in <em>Diadromus pulchellus</em> (Hymenoptera). Insect Biochem Mol Biol  29: 103–111.  <ul class="find" data-citedArticleID="1055398" data-doi="10.1016/s0965-1748(98)00113-1"><li><a href="http://dx.doi.org/10.1016/s0965-1748(98)00113-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Satellite+DNA+transcription+in+Diadromus+pulchellus+%28Hymenoptera%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Satellite+DNA+transcription+in+Diadromus+pulchellus+%28Hymenoptera%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001594-Chiodi1" id="pone.0001594-Chiodi1"></a>Chiodi I, Corioni M, Giordano M, Valgardsdottir R, Ghigna C, et al.  (2004) RNA recognition motif 2 directs the recruitment of SF2/ASF to nuclear stress bodies. Nucleic Acids Res  32: 4127–4136.  <ul class="find" data-citedArticleID="1055360" data-doi="10.1093/nar/gkh759"><li><a href="http://dx.doi.org/10.1093/nar/gkh759" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+recognition+motif+2+directs+the+recruitment+of+SF2%2FASF+to+nuclear+stress+bodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+recognition+motif+2+directs+the+recruitment+of+SF2%2FASF+to+nuclear+stress+bodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001594-Metz1" id="pone.0001594-Metz1"></a>Metz A, Soret J, Vourc'h C, Tazi J, Jolly C (2004) A key role for stress-induced satellite III transcripts in the relocalization of splicing factors into nuclear stress granules. J Cell Sci  117: 4551–4558.  <ul class="find" data-citedArticleID="1055388" data-doi="10.1242/jcs.01329"><li><a href="http://dx.doi.org/10.1242/jcs.01329" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+key+role+for+stress-induced+satellite+III+transcripts+in+the+relocalization+of+splicing+factors+into+nuclear+stress+granules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+key+role+for+stress-induced+satellite+III+transcripts+in+the+relocalization+of+splicing+factors+into+nuclear+stress+granules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001594-Volpe1" id="pone.0001594-Volpe1"></a>Volpe TA, Kidner C, Hall IM, Teng G, Grewal SIS, et al.  (2002) Regulation of heterochromatic silencing and histone H3 lysine-9 methylation by RNAi. Science  297: 1833–1837.  <ul class="find" data-citedArticleID="1055416" data-doi="10.1126/science.1074973"><li><a href="http://dx.doi.org/10.1126/science.1074973" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+heterochromatic+silencing+and+histone+H3+lysine-9+methylation+by+RNAi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+heterochromatic+silencing+and+histone+H3+lysine-9+methylation+by+RNAi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001594-Verdel1" id="pone.0001594-Verdel1"></a>Verdel A, Jia S, Gerber S, Suglyama T, Gygi S, et al.  (2004) RNAi-mediated targeting of heterochromatin with the RITS complex. Science  303: 672–676.  <ul class="find" data-citedArticleID="1055414" data-doi="10.1126/science.1093686"><li><a href="http://dx.doi.org/10.1126/science.1093686" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNAi-mediated+targeting+of+heterochromatin+with+the+RITS+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNAi-mediated+targeting+of+heterochromatin+with+the+RITS+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001594-Bernstein1" id="pone.0001594-Bernstein1"></a>Bernstein E, Allis CD (2005) RNA meets heterochromatin. Genes Dev  19: 1635–1655.  <ul class="find" data-citedArticleID="1055354" data-doi="10.1101/gad.1324305"><li><a href="http://dx.doi.org/10.1101/gad.1324305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+meets+heterochromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+meets+heterochromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001594-Wong1" id="pone.0001594-Wong1"></a>Wong LH, Brettingham-Moore KH, Chan L, Quach JM, Anderson MA, et al.  (2007) Centromere RNA is a key component for the assembly of nucleoproteins at the nucleolus and centromere. Genome Res  17: 1146–1160.  <ul class="find" data-citedArticleID="1055420" data-doi="10.1101/gr.6022807"><li><a href="http://dx.doi.org/10.1101/gr.6022807" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Centromere+RNA+is+a+key+component+for+the+assembly+of+nucleoproteins+at+the+nucleolus+and+centromere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Centromere+RNA+is+a+key+component+for+the+assembly+of+nucleoproteins+at+the+nucleolus+and+centromere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001594-Mravinac1" id="pone.0001594-Mravinac1"></a>Mravinac B, Plohl M, Meštrović N, Ugarković Đ (2002) Sequence of PRAT satellite DNA “frozen” in some coleopteran species. J Mol Evol  54: 774–783.  <ul class="find" data-citedArticleID="1055390" data-doi="10.1007/s00239-001-0079-9"><li><a href="http://dx.doi.org/10.1007/s00239-001-0079-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+of+PRAT+satellite+DNA+%E2%80%9Cfrozen%E2%80%9D+in+some+coleopteran+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+of+PRAT+satellite+DNA+%E2%80%9Cfrozen%E2%80%9D+in+some+coleopteran+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001594-Ugarkovi2" id="pone.0001594-Ugarkovi2"></a>Ugarković Đ, Plohl M, Lucijanić-Justić V, Borštnik B (1992) Detection of satellite DNA in <em>Palorus ratzeburgii</em>: Analysis of curvature profiles and comparison with <em>Tenebrio molitor</em> satellite DNA. Biochimie  74: 1075–1082.  <ul class="find" data-citedArticleID="1055412" data-doi="10.1016/0300-9084(92)90005-y"><li><a href="http://dx.doi.org/10.1016/0300-9084(92)90005-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+satellite+DNA+in+Palorus+ratzeburgii%3A+Analysis+of+curvature+profiles+and+comparison+with+Tenebrio+molitor+satellite+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+satellite+DNA+in+Palorus+ratzeburgii%3A+Analysis+of+curvature+profiles+and+comparison+with+Tenebrio+molitor+satellite+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001594-Metrovi1" id="pone.0001594-Metrovi1"></a>Meštrović N, Plohl M, Mravinac B, Ugarković Đ (1998) Evolution of satellite DNAs from the genus <em>Palorus</em>–experimental evidence for the library hypothesis. Mol Bio. Evol  15: 1062–1068.  <ul class="find" data-citedArticleID="1055386" data-doi="10.1093/oxfordjournals.molbev.a026005"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+satellite+DNAs+from+the+genus+Palorus%E2%80%93experimental+evidence+for+the+library+hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+satellite+DNAs+from+the+genus+Palorus%E2%80%93experimental+evidence+for+the+library+hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001594-Mravinac2" id="pone.0001594-Mravinac2"></a>Mravinac B, Plohl M, Ugarković Đ (2005) Preservation and high sequence conservation of satellite DNAs suggest functional constraints. J Mol Evol  61: 542–550.  <ul class="find" data-citedArticleID="1055392" data-doi="10.1007/s00239-004-0342-y"><li><a href="http://dx.doi.org/10.1007/s00239-004-0342-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preservation+and+high+sequence+conservation+of+satellite+DNAs+suggest+functional+constraints." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preservation+and+high+sequence+conservation+of+satellite+DNAs+suggest+functional+constraints.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001594-Kato1" id="pone.0001594-Kato1"></a>Kato H, Goto DB, Martienssen RA, Urano T, Furukawa K, Murakami Y (2005) RNA polymerase is required for RNAi-dependent heterochromatin assembly. Science  309: 467–469.  <ul class="find" data-citedArticleID="1055374" data-doi="10.1126/science.1114955"><li><a href="http://dx.doi.org/10.1126/science.1114955" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+polymerase+is+required+for+RNAi-dependent+heterochromatin+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+polymerase+is+required+for+RNAi-dependent+heterochromatin+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001594-Lu1" id="pone.0001594-Lu1"></a>Lu J, Gilbert DM (2007) Proliferation-dependent and cell cycle-regulated transcription of mouse pericentromeric heterochromatin. J Cell Biol  179: 411–421.  <ul class="find" data-citedArticleID="1055380" data-doi="10.1083/jcb.200706176"><li><a href="http://dx.doi.org/10.1083/jcb.200706176" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proliferation-dependent+and+cell+cycle-regulated+transcription+of+mouse+pericentromeric+heterochromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proliferation-dependent+and+cell+cycle-regulated+transcription+of+mouse+pericentromeric+heterochromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001594-Chomczynski1" id="pone.0001594-Chomczynski1"></a>Chomczynski P, Sacchi N (1987) Single–Step Method of RNA Isolation by Acid Guanidinium Thiocyanate-Phenol-Chloroform Extraction. Anal Biochem  162: 156–159.  <ul class="find" data-citedArticleID="1055362" data-doi="10.1016/0003-2697(87)90021-2"><li><a href="http://dx.doi.org/10.1016/0003-2697(87)90021-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single%E2%80%93Step+Method+of+RNA+Isolation+by+Acid+Guanidinium+Thiocyanate-Phenol-Chloroform+Extraction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single%E2%80%93Step+Method+of+RNA+Isolation+by+Acid+Guanidinium+Thiocyanate-Phenol-Chloroform+Extraction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001594-Hamilton1" id="pone.0001594-Hamilton1"></a>Hamilton AJ, Baulcombe DC (1999) A Species of Small Antisense RNA in Posttranscriptional Gene Silencing in Plants. Science  286: 950–952.  <ul class="find" data-citedArticleID="1055370" data-doi="10.1126/science.286.5441.950"><li><a href="http://dx.doi.org/10.1126/science.286.5441.950" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Species+of+Small+Antisense+RNA+in+Posttranscriptional+Gene+Silencing+in+Plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Species+of+Small+Antisense+RNA+in+Posttranscriptional+Gene+Silencing+in+Plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001594-Reese1" id="pone.0001594-Reese1"></a>Reese MG (2001) Application of a time-delay neural network to promoter annotation in the <em>Drosophila melanogaster</em> genome. Comput Chem  26: 51–56.  <ul class="find" data-citedArticleID="1055396" data-doi="10.1016/s0097-8485(01)00099-7"><li><a href="http://dx.doi.org/10.1016/s0097-8485(01)00099-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+a+time-delay+neural+network+to+promoter+annotation+in+the+Drosophila+melanogaster+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+a+time-delay+neural+network+to+promoter+annotation+in+the+Drosophila+melanogaster+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001594-Lynch1" id="pone.0001594-Lynch1"></a>Lynch JA, Desplan C (2006) A method for parental RNA interference in the wasp <em>Nasonia vitripennis</em>. Nature Protocols  1: 486–494.  <ul class="find" data-citedArticleID="1055382" data-doi="10.1038/nprot.2006.70"><li><a href="http://dx.doi.org/10.1038/nprot.2006.70" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+method+for+parental+RNA+interference+in+the+wasp+Nasonia+vitripennis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+method+for+parental+RNA+interference+in+the+wasp+Nasonia+vitripennis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001594-May1" id="pone.0001594-May1"></a>May BP, Lippman ZB, Fang Y, Spector DL, Martienssen RA (2005) Differential regulation of strand-specific transcripts from <em>Arabidopsis</em> centromeric satellite repeat. PLoS Genet.  1: e79.  <ul class="find" data-citedArticleID="1055384" data-doi="10.1371/journal.pgen.0010079"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010079" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+regulation+of+strand-specific+transcripts+from+Arabidopsis+centromeric+satellite+repeat." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+regulation+of+strand-specific+transcripts+from+Arabidopsis+centromeric+satellite+repeat.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001594-Lee1" id="pone.0001594-Lee1"></a>Lee HR, Neumann P, Macas J, Jiang J (2006) Transcription and evolutionary dynamics of the centromeric satellite repeat CentO in rice. Mol Biol Evol  23: 2505–2520.  <ul class="find" data-citedArticleID="1055376" data-doi="10.1093/molbev/msl127"><li><a href="http://dx.doi.org/10.1093/molbev/msl127" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+and+evolutionary+dynamics+of+the+centromeric+satellite+repeat+CentO+in+rice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+and+evolutionary+dynamics+of+the+centromeric+satellite+repeat+CentO+in+rice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001594-Wang1" id="pone.0001594-Wang1"></a>Wang F, Koyama N, Nishida H, Haraguchi T, Reith W, et al.  (2006) The assembly and maintenance of heterochromatin initiated by transgene repeats are independent of the RNA interference pathway in mammalian cells. Mol Cell Biol  26: 4028–4040.  <ul class="find" data-citedArticleID="1055418" data-doi="10.1128/mcb.02189-05"><li><a href="http://dx.doi.org/10.1128/mcb.02189-05" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+assembly+and+maintenance+of+heterochromatin+initiated+by+transgene+repeats+are+independent+of+the+RNA+interference+pathway+in+mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+assembly+and+maintenance+of+heterochromatin+initiated+by+transgene+repeats+are+independent+of+the+RNA+interference+pathway+in+mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001594-Topp1" id="pone.0001594-Topp1"></a>Topp CN, Zhong CX, Dawe RK (2004) Centromere-encoded RNAs are integral components of the maize kinetochore. Proc Natl Acad Sci USA  101: 15986–15991.  <ul class="find" data-citedArticleID="1055406" data-doi="10.1073/pnas.0407154101"><li><a href="http://dx.doi.org/10.1073/pnas.0407154101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Centromere-encoded+RNAs+are+integral+components+of+the+maize+kinetochore." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Centromere-encoded+RNAs+are+integral+components+of+the+maize+kinetochore.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001594-BouzinbaSegard1" id="pone.0001594-BouzinbaSegard1"></a>Bouzinba-Segard H, Guais A, Francastel C (2006) Accumulation of small murine minor satellite transcripts leads to impaired centromeric architecture and function. Proc Natl Acad Sci USA  103: 8709–8714.  <ul class="find" data-citedArticleID="1055358" data-doi="10.1073/pnas.0508006103"><li><a href="http://dx.doi.org/10.1073/pnas.0508006103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accumulation+of+small+murine+minor+satellite+transcripts+leads+to+impaired+centromeric+architecture+and+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accumulation+of+small+murine+minor+satellite+transcripts+leads+to+impaired+centromeric+architecture+and+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001594-Ferbeyre1" id="pone.0001594-Ferbeyre1"></a>Ferbeyre G, Smith JM, Cedergren R (1998) Schistosome satellite DNA encodes active hammerhead-ribozymes. Mol Cell Biol  18: 3880–3888.  <ul class="find" data-citedArticleID="1055366"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Schistosome+satellite+DNA+encodes+active+hammerhead-ribozymes.&amp;auth=&amp;atitle=Schistosome+satellite+DNA+encodes+active+hammerhead-ribozymes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Schistosome+satellite+DNA+encodes+active+hammerhead-ribozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Schistosome+satellite+DNA+encodes+active+hammerhead-ribozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001594-Coats1" id="pone.0001594-Coats1"></a>Coats SR, Zhang Y, Epstein LM (1994) Transcription of satellite 2 DNA from the newt is driven by a snRNA type of promoter. Nucleic Acids Res  22: 4697–4704.  <ul class="find" data-citedArticleID="1055364" data-doi="10.1093/nar/22.22.4697"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4697" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+of+satellite+2+DNA+from+the+newt+is+driven+by+a+snRNA+type+of+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+of+satellite+2+DNA+from+the+newt+is+driven+by+a+snRNA+type+of+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001594-Yasuhara1" id="pone.0001594-Yasuhara1"></a>Yasuhara JC, DeCrease CH, Wakimoto BT (2005) Evolution of heterochromatic genes of <em>Drosophila</em>. Proc Natl Acad Sci USA  102: 10958–10963.  <ul class="find" data-citedArticleID="1055422" data-doi="10.1073/pnas.0503424102"><li><a href="http://dx.doi.org/10.1073/pnas.0503424102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+heterochromatic+genes+of+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+heterochromatic+genes+of+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001594-Scott1" id="pone.0001594-Scott1"></a>Scott KC, Merrett SL, Willard HF (2006) A heterochromatin barrier partitions the fission yeast centromere into discrete chromatin domains. Curr Biol  16: 119–129.  <ul class="find" data-citedArticleID="1055402" data-doi="10.1016/j.cub.2005.11.065"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.11.065" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+heterochromatin+barrier+partitions+the+fission+yeast+centromere+into+discrete+chromatin+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+heterochromatin+barrier+partitions+the+fission+yeast+centromere+into+discrete+chromatin+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001594-Noma1" id="pone.0001594-Noma1"></a>Noma K, Cam HP, Maraia RJ, Grewal SIS (2006) A role for TFIIIC transcription factor complex in genome organization. Cell  125: 859–872.  <ul class="find" data-citedArticleID="1055394" data-doi="10.1016/j.cell.2006.04.028"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.04.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+role+for+TFIIIC+transcription+factor+complex+in+genome+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+role+for+TFIIIC+transcription+factor+complex+in+genome+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001594-Hamada1" id="pone.0001594-Hamada1"></a>Hamada M, Huang Y, Lowe TM, Maraia RJ (2001) Widespread use of TATA elements in the core promoters for RNA polymerases III, II and I in fission yeast. Mol Cell Biol  21: 6870–6881.  <ul class="find" data-citedArticleID="1055368" data-doi="10.1128/mcb.21.20.6870-6881.2001"><li><a href="http://dx.doi.org/10.1128/mcb.21.20.6870-6881.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Widespread+use+of+TATA+elements+in+the+core+promoters+for+RNA+polymerases+III%2C+II+and+I+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Widespread+use+of+TATA+elements+in+the+core+promoters+for+RNA+polymerases+III%2C+II+and+I+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.XML" value="83885"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.PDF" value="1095938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g001.PNG_L" value="575093"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g001.PNG_M" value="110593"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g001.PNG_S" value="9708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g001.TIF" value="540708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g001.PNG_I" value="40802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g002.PNG_L" value="581009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g002.PNG_M" value="93130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g002.PNG_S" value="7369"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g002.TIF" value="563416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g002.PNG_I" value="29622"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g003.PNG_L" value="782682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g003.PNG_M" value="139828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g003.PNG_S" value="10932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g003.TIF" value="1209236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g003.PNG_I" value="53266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g004.PNG_L" value="529976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g004.PNG_M" value="76920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g004.PNG_S" value="6360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g004.TIF" value="669108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g004.PNG_I" value="26185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g005.PNG_L" value="653994"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g005.PNG_M" value="88412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g005.PNG_S" value="7680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g005.TIF" value="837998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g005.PNG_I" value="29661"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g006.PNG_L" value="93198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g006.PNG_M" value="55070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g006.PNG_S" value="8458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g006.TIF" value="193724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001594.g006.PNG_I" value="20054"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001594&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001594" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001594&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001594&volume=&issue=&title=RNA Pol II Promotes Transcription of Centromeric Satellite DNA in Beetles&author_name=%C5%BDeljka%20Pezer%2C%20%C4%90ur%C4%91ica%20Ugarkovi%C4%87&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001594" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001594" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001594" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001594&amp;t=RNA%20Pol%20II%20Promotes%20Transcription%20of%20Centromeric%20Satellite%20DNA%20in%20Beetles" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001594&title=RNA%20Pol%20II%20Promotes%20Transcription%20of%20Centromeric%20Satellite%20DNA%20in%20Beetles&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001594&amp;title=RNA%20Pol%20II%20Promotes%20Transcription%20of%20Centromeric%20Satellite%20DNA%20in%20Beetles" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001594" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001594&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'RNA Pol II Promotes Transcription of Centromeric Satellite DNA in Beetles';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001594';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20RNA%20Pol%20II%20Promotes%20Transcription%20of%20Centromeric%20Satellite%20DNA%20in%20Beetles http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001594" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001594" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="26014"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="26014"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-RNA+hybridization%22" title="Search for articles in the subject area:'DNA-RNA hybridization'"><div class="flagText">DNA-RNA hybridization</div></a>
              <div data-categoryid="22997" data-articleid="26014"
                   data-categoryname="DNA-RNA hybridization"
                   class="flagImage" title="Flag 'DNA-RNA hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Larvae%22" title="Search for articles in the subject area:'Larvae'"><div class="flagText">Larvae</div></a>
              <div data-categoryid="17821" data-articleid="26014"
                   data-categoryname="Larvae"
                   class="flagImage" title="Flag 'Larvae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+extraction%22" title="Search for articles in the subject area:'RNA extraction'"><div class="flagText">RNA extraction</div></a>
              <div data-categoryid="20119" data-articleid="26014"
                   data-categoryname="RNA extraction"
                   class="flagImage" title="Flag 'RNA extraction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+hybridization%22" title="Search for articles in the subject area:'RNA hybridization'"><div class="flagText">RNA hybridization</div></a>
              <div data-categoryid="19365" data-articleid="26014"
                   data-categoryname="RNA hybridization"
                   class="flagImage" title="Flag 'RNA hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Satellite+DNA%22" title="Search for articles in the subject area:'Satellite DNA'"><div class="flagText">Satellite DNA</div></a>
              <div data-categoryid="25381" data-articleid="26014"
                   data-categoryname="Satellite DNA"
                   class="flagImage" title="Flag 'Satellite DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Small+interfering+RNA%22" title="Search for articles in the subject area:'Small interfering RNA'"><div class="flagText">Small interfering RNA</div></a>
              <div data-categoryid="19855" data-articleid="26014"
                   data-categoryname="Small interfering RNA"
                   class="flagImage" title="Flag 'Small interfering RNA' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1718'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=316'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7945&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=10211">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
