

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Deductive Biocomputing</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000339"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000339" />

    <meta name="citation_title" content="Deductive Biocomputing"/>
    <meta itemprop="name" content="Deductive Biocomputing"/>

      <meta name="citation_author" content="Jeff Shrager"/>
            <meta name="citation_author_institution" content="Department of Plant Biology, Carnegie Institution of Washington, Stanford, California, United States of America"/>
      <meta name="citation_author" content="Richard Waldinger"/>
            <meta name="citation_author_institution" content="Artificial Intelligence Center, SRI International, Menlo Park, California, United States of America"/>
      <meta name="citation_author" content="Mark Stickel"/>
            <meta name="citation_author_institution" content="Artificial Intelligence Center, SRI International, Menlo Park, California, United States of America"/>
      <meta name="citation_author" content="J.P. Massar"/>
            <meta name="citation_author_institution" content="Berkeley, California, United States of America"/>

    <meta name="citation_date" content="2007/4/4"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000339.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e339"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Tools for thought in the age of biological knowledge.; citation_author=N Fedoroff; citation_author=S Racunas; citation_author=J Shrager; citation_journal_title=The Scientist; citation_volume=19; citation_number=1; citation_pages=20-21; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=BioLingua: A programmable knowledge environment for biologists.; citation_author=JP Massar; citation_author=M Travers; citation_author=J Elhai; citation_author=J Shrager; citation_journal_title=BioInformatics; citation_volume=21; citation_number=2; citation_pages=199-207; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Application of theorem proving to problem solving.; citation_author=CC Green; citation_number=3; citation_date=1969; " />
      <meta name="citation_reference" content="citation_title=A deductive approach to program synthesis.; citation_author=Z Manna; citation_author=R Waldinger; citation_journal_title=ACM Transactions on Programming, Languages, and Systems; citation_volume=2; citation_number=4; citation_pages=90-121; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=A Guide to SNARK.; citation_author=ME Stickel; citation_author=RJ Waldinger; citation_author=VK Chaudhri; citation_number=5; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Gene ontology: A tool for the unification of biology.; citation_journal_title=Nature Genet; citation_volume=25; citation_number=6; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The pathway tools software.; citation_author=P Karp; citation_author=S Paley; citation_author=P Romero; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=7; citation_pages=S225-32; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The (re)birth of the knowledge operating system.; citation_author=M Travers; citation_author=JP Massar; citation_author=J Shrager; citation_number=8; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Prochlorococcus, a marine photosynthetic prokaryote of global significance.; citation_author=F Partensky; citation_author=WR Hess; citation_author=D Vaulot; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=63; citation_number=9; citation_pages=106-27; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Analysis of the hli gene family in marine and freshwater cyanobacteria.; citation_author=D Bhaya; citation_author=A Dufresne; citation_author=D Vaulot; citation_author=A Grossman; citation_journal_title=FEMS Microbiology Letters,; citation_volume=215; citation_number=10; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=DNA microarray analysis of cyanobacterial gene expression during acclimation to high light.; citation_author=Y Hihara; citation_author=A Kamei; citation_author=M Kanehisa; citation_author=A Kaplan; citation_author=M Ikeuchi; citation_journal_title=Plant Cell; citation_volume=13; citation_number=11; citation_pages=793-806; citation_date=2001; " />
      <meta name="citation_reference" content="citation_author=S Russell; citation_author=P Norvig; citation_number=12; citation_date=1995; citation_publisher=Prentice Hall; " />
      <meta name="citation_reference" content="citation_title=The fiction of function.; citation_author=J Shrager; citation_journal_title=BioInformatics; citation_volume=19; citation_number=13; citation_pages=1934-1936; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Pathway Logic: Symbolic analysis of biological signaling.; citation_author=S Eker; citation_author=M Knapp; citation_author=K Laderoute; citation_author=P Lincoln; citation_author=J Meseguer; citation_author=K Sonmez; citation_number=14; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Pathway Logic Modeling of Protein Functional Domains in Signal Transduction.; citation_author=C Talcott; citation_author=S Eker; citation_author=M Knapp; citation_author=P Lincoln; citation_author=K Laderoute; citation_number=15; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=HyBrow: a prototype system for computer-aided hypothesis evaluation.; citation_author=SA Racunas; citation_author=NH Shah; citation_author=I Albert; citation_author=NV Fedoroff; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=16; citation_pages=i257-264; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Guiding revision of regulatory models with expression data.; citation_author=J Shrager; citation_author=P Langley; citation_author=A Pohorille; citation_number=17; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Maintaining knowledge about temporal intervals.; citation_author=JF Allen; citation_journal_title=Commun. ACM; citation_volume=26; citation_number=18; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=The Ontological Engineers' Handbook.; citation_number=19; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Towards a standard upper ontology.; citation_author=I Niles; citation_author=A Pease; citation_number=20; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Teknowledge: SUMO Ontology.; citation_number=21; " />
      <meta name="citation_reference" content="citation_author=K Baclawski; citation_author=T Niu; citation_number=22; citation_date=2005; citation_publisher=MIT Press; " />
      <meta name="citation_reference" content="citation_author=M Barr; citation_author=C Wells; citation_number=23; citation_date=1999; citation_publisher=Les Publications CRM; " />
      <meta name="citation_reference" content="citation_title=Putting theories together to make specifications.; citation_author=R Burstall; citation_author=J Goguen; citation_number=24; citation_pages=1045-1058; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Composition by Colimit and Formal Software Development.; citation_author=DR Smith; citation_number=25; citation_date=2006; " />
      <meta name="citation_reference" content="citation_number=26; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=AMPHION: Automatic Programming for Scientific Subroutine Libraries.; citation_author=MR Lowry; citation_author=A Philpot; citation_author=T Pressburger; citation_author=I Underwood; citation_journal_title=ISMIS; citation_number=27; citation_pages=326-335; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Deductive composition of astronomical software from subroutine libraries.; citation_author=ME Stickel; citation_author=R Waldinger; citation_author=M Lowry; citation_author=T Pressburger; citation_author=I Underwood; citation_number=28; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Deductive question answering from multiple resources.; citation_author=R Waldinger; citation_author=J Hobbs; citation_author=DE Appelt; citation_author=J Fry; citation_author=DJ Israel; citation_author=P Jarvis; citation_author=D Martin; citation_author=S Riehemann; citation_author=ME Stickel; citation_author=M Tyson; citation_author=JL Dungan; citation_number=29; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000339" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Deductive Biocomputing"/>
    <meta name="twitter:description" content="BackgroundAs biologists increasingly rely upon computational tools, it is imperative that they be able to appropriately apply these tools and clearly understand the methods the tools employ. Such tools must have access to all the relevant data and knowledge and, in some sense, &ldquo;understand&rdquo; biology so that they can serve biologists' goals appropriately and &ldquo;explain&rdquo; in biological terms how results are computed.Methodology/Principal FindingsWe describe a deduction-based approach to biocomputation that semiautomatically combines knowledge, software, and data to satisfy goals expressed in a high-level biological language. The approach is implemented in an open source web-based biocomputing platform called BioDeducta, which combines SRI's SNARK theorem prover with the BioBike interactive integrated knowledge base. The biologist/user expresses a high-level conjecture, representing a biocomputational goal query, without indicating how this goal is to be achieved. A subject domain theory, represented in SNARK's logical language, transforms the terms in the conjecture into capabilities of the available resources and the background knowledge necessary to link them together. If the subject domain theory enables SNARK to prove the conjecture&mdash;that is, to find paths between the goal and BioBike resources&mdash;then the resulting proofs represent solutions to the conjecture/query. Such proofs provide provenance for each result, indicating in detail how they were computed. We demonstrate BioDeducta by showing how it can approximately replicate a previously published analysis of genes involved in the adaptation of cyanobacteria to different light niches.Conclusions/SignificanceThrough the use of automated deduction guided by a biological subject domain theory, this work is a step towards enabling biologists to conveniently and efficiently marshal integrated knowledge, data, and computational tools toward resolving complex biological queries."/>

  <meta property="og:title" content="Deductive Biocomputing" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000339" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8374'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4648'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6845&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000339">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000339" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000339&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Deductive Biocomputing
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jeff Shrager
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jshrager@stanford.edu">jshrager@stanford.edu</a></p>

                <p>Affiliation:
                  Department of Plant Biology, Carnegie Institution of Washington, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Richard Waldinger, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Artificial Intelligence Center, SRI International, Menlo Park, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark Stickel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Artificial Intelligence Center, SRI International, Menlo Park, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            J.P. Massar
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Berkeley, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 04, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000339</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000339" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000339" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000339" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000339" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000339">Reader Comments (2)</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>As biologists increasingly rely upon computational tools, it is imperative that they be able to appropriately apply these tools and clearly understand the methods the tools employ. Such tools must have access to all the relevant data and knowledge and, in some sense, “understand” biology so that they can serve biologists' goals appropriately and “explain” in biological terms how results are computed.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We describe a deduction-based approach to biocomputation that semiautomatically combines knowledge, software, and data to satisfy goals expressed in a high-level biological language. The approach is implemented in an open source web-based biocomputing platform called BioDeducta, which combines SRI's SNARK theorem prover with the BioBike interactive integrated knowledge base. The biologist/user expresses a high-level conjecture, representing a biocomputational goal query, without indicating how this goal is to be achieved. A subject domain theory, represented in SNARK's logical language, transforms the terms in the conjecture into capabilities of the available resources and the background knowledge necessary to link them together. If the subject domain theory enables SNARK to prove the conjecture—that is, to find paths between the goal and BioBike resources—then the resulting proofs represent solutions to the conjecture/query. Such proofs provide provenance for each result, indicating in detail how they were computed. We demonstrate BioDeducta by showing how it can approximately replicate a previously published analysis of genes involved in the adaptation of cyanobacteria to different light niches.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Through the use of automated deduction guided by a biological subject domain theory, this work is a step towards enabling biologists to conveniently and efficiently marshal integrated knowledge, data, and computational tools toward resolving complex biological queries.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Shrager J, Waldinger R, Stickel M, Massar J (2007) Deductive Biocomputing. PLoS ONE 2(4):
          e339.
            doi:10.1371/journal.pone.0000339</p><p><strong>Academic Editor: </strong>Robert Futrelle, Northeastern University, United States of America</p><p><strong>Received:</strong> January 12, 2007; <strong>Accepted:</strong> March 6, 2007; <strong>Published:</strong> April 4, 2007</p><p><strong>Copyright:</strong> © 2007 Shrager et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported in part by the National Science Foundation, under the Science and Engineering Information Integration and Informatics (SEIII) Program (NSF IIS-0513857), and by a grant from the NASA Advanced Information Systems Research Program through cooperative agreement NO6-4803 between NASA-Ames (Andrew Pohorille P.I.) and The Inst. for the Study of Learning and Expertise.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3>
<h4>Background</h4>
<a id="article1.body1.sec1.sec1.p1" name="article1.body1.sec1.sec1.p1"></a><p>Biologists must increasingly conduct computational analyses across integrated biological knowledge and data <a href="#pone.0000339-Fedoroff1">[1]</a>, but biologists who cannot program are relegated to searching in knowledge bases and carrying out canned computations that have been programmed by others. To conduct novel computations—ones for which no canned solution exists—biologists often hire programmers, but this is either expensive or hit-and-miss (or both), and usually results in overly specific one-off solutions. Moreover, putting non-biologist programmers between biologists and biocomputation removes the biologist from the details of the computation <a href="#pone.0000339-Massar1">[2]</a>, which makes biologists nervous about the correctness of the results—as well it should. Unlike physics, there is no clear categorization within biology of, for example, theoretical vs. experimental vs. computational biologists whose practitioners could naturally collaborate. As a result, it is often nearly impossible for the authors of papers (usually biologists) to accurately and completely describe the computational methods used to solve a given biological problem. Although this will certainly change as computational techniques become standardized and biologists learn to program—indeed a whole new species called “computational biologist” is rapidly evolving—it would be useful if biologists had “intelligent” tools that, in a sense, “understand” biology, could help the biologists conduct analyses, and could explain how the analyses were accomplished, all expressed in terms that are natural to the biologists.</p>
<a id="article1.body1.sec1.sec1.p2" name="article1.body1.sec1.sec1.p2"></a><p>The present paper explores just such an “intelligent” paradigm that we call <em>deductive biocomputing</em>. In our paradigm biologists/users express queries as goals in high-level near-natural terms. Others—usually biologists who are more computationally savvy working with programmers—provide instructions to the platform about how to satisfy those goals. Automatic methods then put these together to satisfy the goals and to explicitly record how the answers were discovered. This is analogous to the widely studied problem of automatic program generation <a href="#pone.0000339-Green1">[3]</a>, <a href="#pone.0000339-Manna1">[4]</a> in which a specification for a program is expressed in a high-level language and then translated into an actual program that implements the specification. (Although fully general automatic program synthesis is an unsolved problem, the present query resolution problem is more tractable because we are looking for an answer to a specific query, not a procedure that will answer an entire class of queries. In particular, because the inputs to the query are generally known in advance, we can evaluate tests and unwind loops at proof-time, this avoiding having to automatically generate conditional branches and iterative or recursive loops.)</p>


<h4>The deductive approach to biocomputation</h4>
<a id="article1.body1.sec1.sec2.p1" name="article1.body1.sec1.sec2.p1"></a><p>Engineers approach complex knowledge-based goals in three general ways: (a) the “procedural” approach, (b) the “relational” approach, and (c) the “deductive” approach. In the procedural approach (a) one specifies each step required to reach the goal, down to the level at which the steps are primitives in the domain language. Such procedures often involve search in knowledge bases, generally traversing the structures that represent complex knowledge, transforming data, and conducting specific calculations along the way. Usually writing programs of this sort is beyond the skill of biologists.</p>
<a id="article1.body1.sec1.sec2.p2" name="article1.body1.sec1.sec2.p2"></a><p>In the relational approach (b) queries are written such that the complex inner loops (searching over records) are implicit; for example: <span class="monospace">“Select genes (g1) of organism1 with the word ‘photosynthesis’ in their annotation field, and where there is a gene (g2) in organism2 such that g1 and g2 are orthologous.”</span> Such queries can be directly executed by the relational database engine, thus simplifying the programming process. Still, in order to make use of this approach in complex analyses the biologist must know a great deal about how the knowledge bases are structured, and what tools are available.</p>
<a id="article1.body1.sec1.sec2.p3" name="article1.body1.sec1.sec2.p3"></a><p>In the deductive approach (c), the biologist specifies a high-level goal, usually without knowing how it will be satisfied, and a runtime executor (called a <em>theorem prover</em> for reasons that will become clear below) does whatever work is required to satisfy the goal. One might, for example, ask: <span class="monospace">“Find photosynthetic orthologs between organism1 and organism2.”</span> Whereas the relational approach goes beyond the procedural approach by leaving loop optimization to the computer, the deductive approach goes beyond even the relational approach by avoiding the necessity of describing an algorithm at all. No computation is explicitly described; instead, the query is expressed in declarative, problem-appropriate terms; it is the job of the theorem prover to bridge the gap between the query-level concepts and the computational ones.</p>
<a id="article1.body1.sec1.sec2.p4" name="article1.body1.sec1.sec2.p4"></a><p>Leaving method determination to the computer is a significant though difficult advance; whereas the implicit search carried out by the relational database engine requires knowledge of how to execute and optimize a certain class of algorithms (complex loops over relational databases), the deductive approach requires that the computer have additional understanding of how aspects of the knowledge base are connected to one another, and how to use these connections in service of high-level goals. The computer must, in a sense, “understand” how the knowledge is organized, and how it relates to high-level goals. Generally the knowledge-base designer provides such information to the theorem prover in what we shall call a <em>subject domain theory</em>. Rather than representing one-off methods, this knowledge can be expressed modularly so that it can be applied to a wide array of different goals. Through this specification of a subject domain theory and application of a general theorem prover, the end-user biologist—the person at the top who needs the results of the computation to begin with—is relying on the knowledge-base designers and the theorem prover to do the heavy lifting. At the end of this paper we will see that in addition to providing great power to the biologist-user, deductive biocomputing affords a number of significant advantages over either task-specific programs or relational queries, including a coherent way of recording where results come from (called <em>provenance</em>). Also, one can nearly (but not quite) get to true natural language programming, in which the biologist can ask a question in plain language and efficiently obtain both results and a detailed explanation of how they were calculated. But there are also efficiency limitations to this technique, and special requirements in terms of the consistency of the symbols (names) used in the subject domain theory. We will return to these points in the discussion.</p>
<a id="article1.body1.sec1.sec2.p5" name="article1.body1.sec1.sec2.p5"></a><p>The core of a deductive approach is the subject domain theory, an ontology comprising definitions of domain concepts, descriptions of the capabilities of available resources including data, knowledge, and tools, and the background knowledge necessary to relate these to high-level queries. While the word <em>ontology</em> is sometimes taken to mean a description of vocabulary and taxonomy of the domain, we mean here a formal axiomatic theory that defines domain concepts and relates them to one another. We use the word <em>axiom</em> to include what is often called a <em>rule</em> in the Semantic Web or Expert Systems literature. We employ full first-order logic as the representation language of the subject domain theory. (Readers familiar with theorem proving technologies will correctly recoil at the undecidability of full first-order logic. We address this issue in some detail in the Discussion section of the paper.)The query, or question to be answered, is also expressed in the language of logic; specifically it is phrased as a conjecture whose validity is to be proved by an automatic proof in the subject domain theory (thus the term <em>theorem prover</em>). The axioms of the subject domain theory allow the query conjecture to be transformed and decomposed into subgoals, leading eventually (one hopes) to a proof. If a proof is found, answer-extraction techniques are applied to the proof to yield an answer <a href="#pone.0000339-Green1">[3]</a>, <a href="#pone.0000339-Manna1">[4]</a>. If more than one proof is found, more than one answer may be extracted.</p>

</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Method"></a><h3>Method</h3>
<h4>BioDeducta = SNARK+BioBike</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>While the applicability of the deductive approach is independent of any particular implementation, our prototype implementation, called BioDeducta, is built largely from existing components. The theorem prover is SRI's reasoner SNARK <a href="#pone.0000339-Stickel1">[5]</a>. The query and subject domain theory are formulated in SNARK's logical language. SNARK has a procedural-attachment mechanism that allows one to consult external resources while a proof is in progress, and a mechanism for extracting answers to queries from discovered proofs. SNARK uses a “sorted logic”, in which each entity bears a syntactic indication of what “sort”, or type, of object it is.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Biology-specific data, knowledge, and software resources are drawn from the BioBike environment <a href="#pone.0000339-Massar1">[2]</a>, an integrated biological data and knowledge repository and biology-specific programming environment. BioBike provides integrated access to a number of knowledge sources, including the Gene Ontology (GO) <a href="#pone.0000339-The1">[6]</a>, Kegg (<a href="http://www.genome.jp/kegg/">www.genome.jp/kegg/</a>), a BioCyc Database <a href="#pone.0000339-Karp1">[7]</a> for Cyanobacteria, and biological literature, as well as important biological software tools such as Meme, BLAST, R, and Clustal. BioBike is built on top of the KnowOS (Knowledge Operating System) <a href="#pone.0000339-Travers1">[8]</a>, which embeds the biological knowledge and data in a frame-based programmable knowledge environment, and provides web-based access to it, as well as the ability to call out to remote resources over the web. The KnowOS includes SNARK as a built-in facility, and SNARK procedural attachments link symbols of the SNARK subject domain theory to BioBike functions. Both SNARK and BioBike are written entirely in Common Lisp, and are offered as open-source freeware. Moreover, there is a fully functional BioBike demo server, including SNARK, on which the examples in this paper can be tested, and wherein users can freely develop new BioDeducta applications of their own design (see <a href="http://www.biobike.org">www.biobike.org</a>).</p>


<h4>The “hli” problem</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We here illustrate the BioDeducta approach with an extended, realistic example. The cyanobacterium subspecies Procholorococcus is widely distributed in the world's oceans, and plays a critical role both in the marine ecosystem and in the global carbon cycle <a href="#pone.0000339-Partensky1">[9]</a>. A problem of interest to biologists is how these organisms are adapted to their environmental niches. Bahya et al. <a href="#pone.0000339-Bhaya1">[10]</a> studied the genomic differences among many strains of the cyanobacteria with respect to their adaptation to niches of differing levels of light and nutrients. Among the Procholorococci, Procholorococcus sp. strain Med4 (aka. ProMed4 or Med4) is adapted to high light, living in the upper part of the ocean, whereas Procholorococcus sp. strain MIT9313 (aka. Pro9313) is adapted to lower light, living in somewhat deeper waters (although still in the “euphotic zone” where there is some available sunlight). Bhaya et al. were interested in which proteins (and their genes) are involved in this adaptation—that is those one might call the <em>high light adaptive genes</em> of ProMed4. (For simplicity in the present discussion we interchangeably refer to genes, and the RNAs and proteins that they code for. As we are dealing here with bacteria, this simplification is not too problematic, and BioBike knows how to automatically determine when translation is needed between these.) One way to address this biological question is to ask which proteins in ProMed4 have no ortholog—that is, no gene of similar apparent function (based upon sequence similarity)—in Pro9313. One can get an even finer bead on this question by examining microarray expression results for the genes that produce those proteins, asking which of those genes unique to ProMed4 demonstrate a significant light response (for example, are 2× up-regulated in a light stress experiment). Unfortunately, microarrays for the Prochlorococci have only recently been developed, so no such experimental work exists. However, there are a number of such studies using the related freshwater cyanobacterium: Synechocystis sp. strain PCC6803 (aka. s6803) <a href="#pone.0000339-Hihara1">[11]</a>. Going one step further, one may focus specifically on the genes that are annotated as photosynthesis-related according to some formalization of gene function, such as the Gene Ontology (GO).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In sum, we can expand this question as follows:</p>

<blockquote><a id="article1.body1.sec2.sec2.disp-quote1.p1" name="article1.body1.sec2.sec2.disp-quote1.p1"></a><p>Which photosynthesis-related proteins in ProMed4 have no ortholog in Pro9313 but do have an ortholog in s6803 such that the genes producing those proteins exhibits a light stress response (greater than 2× ratio in microarray data), and possibly are annotated as light-related genes?</p>
</blockquote>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>One algorithm to find such genes might be:</p>
<pre> For each photosynthesis-related gene/protein in ProMed4,  Find its best protein-level ortholog     in Pro9313 and S6803, When there is no Pro9313 ortholog,   but there is a S6803 ortholog,   and the expression ratio for the genes (mRNAs) that     produce the S6803 ortholog are &gt;2×     up-regulated in the Hihara microarray dataset, collect the ProMed4 gene/protein. </pre><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Expressed in the native BioBike language (BioLisp), this might be written as follows:</p>
<pre>    (loop for pm4prot in (#^proteins promed4)      as 9313ortho  =  (best-ortholog pm4prot pro9313)      as 6803ortho  =  (best-ortholog pm4prot s6803)    when (and (photosynthesis-related pm4prot)         (null 9313ortho)          6803ortho         (&gt; = (array-select 6803ortho Hihara1) 2.0))      collect pm4prot) </pre><a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Although this solution is concise and relatively efficient, its programming requires detailed knowledge of both the BioLisp programming language, and of how to call upon BioBike's knowledge resources. Moreover, this is a one-shot solution specific to this particular problem, not affording of significant reuse (i.e., methodological modularity). Nor does this approach get us any more than the solution reported as an opaque answer that cannot be unpacked into the method that solved it (provenance).</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>In contrast to this approach, the BioDeducta methodology allows us to more conveniently express and solve this problem while simultaneously offering methodological modularity and solution provenance. We begin by expressing our query in high-level terms familiar to the biologist, and then unpack each concept into modular conceptual constituents in the subject domain theory. It is the job of the theorem prover to figure out how to use the guidance offered by the subject domain theory to find a solution to the top query.</p>
<a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>The query might be expressed as follows: What gene enables med4 to adapt to its light environment? Or, expressed in terms of a SNARK formal conjecture, does there exist a gene <span class="monospace">?gene</span> such that:</p>
<a id="article1.body1.sec2.sec2.p8" name="article1.body1.sec2.sec2.p8"></a><p><span class="monospace">(adaptive-gene ?gene med4 light)?</span></p>
<a id="article1.body1.sec2.sec2.p9" name="article1.body1.sec2.sec2.p9"></a><p>The satisfaction of this conjecture in the subject domain theory is to be found by the theorem prover. Terms in the query preceded by question marks (<span class="monospace">?gene</span>) are variables that will be plugged in by SNARK, if possible, in the process of providing a proof for the conjecture; in proving the existence of such entities, the theorem prover will be forced to find consistent values that fit these variables. These descriptions of genes and organisms that satisfy the conditions will then be extracted from the proof. The proof thus constitutes an explanation of why the extracted entities satisfy the desired conditions. The other terms (<span class="monospace">med4</span> and <span class="monospace">light</span>) are constants that are fixed by the theory.</p>


<h4>Subject domain theory</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The meanings of the symbols of a query, such as <span class="monospace">adaptive-gene</span>, are defined by axioms of the subject domain theory. The theorem prover transforms the query in accordance with these axioms. Unlike a logic-programming system such as Prolog, the theorem prover is not constrained to process the query in the order given; rather it follows its own strategic controls.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The subject domain theory has three parts: (1) modular definitions that enable SNARK to translate the high-level query into a search procedure through subgoals that finally ground out in terms of ground knowledge and BioBike procedures, (2) simple ground knowledge that requires no internal (subgoal) computations, and (3) procedural attachments into the BioBike knowledge base that access computed knowledge and which may perform internal “hidden” computations. As we proceed with this exposition, it is important to keep in mind that these various resources are independent of the particular <span class="monospace">adaptive-gene</span> query in the present example; rather, they are conceptually self-contained and modular in that they could be used in any number of queries regarding the properties of genes and proteins. (The subject domain theory will generally have been previously provided by either the knowledge base designers or by other biologists. Here, of course, we have written it for this problem, so it exhibits somewhat less generality. Elsewhere we could engage in a productive debate about how each axiom should properly be cast for maximum generality.)</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In this presentation we begin from the goal, and work our way conceptually downward. Recall that we seek a proof of the theorem that establishes the existence of a gene (<span class="monospace">?gene</span>) such that:</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p><span class="monospace">(adaptive-gene ?gene med4 light)</span></p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>(More precisely we seek a refutation of the negation of this assertion. This process, called “proof by refutation”, is explained in detail in Russell and Norvig <a href="#pone.0000339-Russell1">[12]</a> ch. III.9.)</p>
<a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>Here is the definition of what it means for a gene to be adaptive—that is, for it to be related to the way in which an organism adapts to its environment:</p>
<pre>    (adaptive-gene ?gene1 ?organism1 ?dim)      ⇔ [i.e., if and only if]     (and      (gene-in-organism ?gene1 ?organism1)              (gene-semantics ?gene1 ?dim)              (differentiating-gene ?gene1 ?organism1 ?dim)              (differentially-regulated ?gene1 ?dim)) </pre><a id="article1.body1.sec2.sec3.p7" name="article1.body1.sec2.sec3.p7"></a><p>That asserts that a gene is related to the way in which an organism adapts to its environment along a particular environmental dimension (i.e., light, in the present example) if and only if four conditions are met: First, the gene must be a gene of the given organism. Second, the gene's putative function (per its explicit annotation, provided by the database) must be conceptually related to the relevant dimension. Third, the gene must (genomically) differentiate between organisms that live in environments that differ along the relevant dimension (here: light). And fourth, the gene must be differentially regulated in an experiment that explores the relevant dimension (again: light).</p>
<a id="article1.body1.sec2.sec3.p8" name="article1.body1.sec2.sec3.p8"></a><p>Importantly, none of the specifics of the query are explicit in this formulation of what it means to be an adaptive gene; this axiom is general to this sort of problem, and uses internal formulae, which are likewise general (and which we explain below). One could argue about whether this is precisely what one intends by the biological concept of “adaptive gene”, but it is easy to adjust this definition if one desires.</p>
<a id="article1.body1.sec2.sec3.p9" name="article1.body1.sec2.sec3.p9"></a><p>Continuing to unpack the meaning of terms: A differentiating gene (more precisely: a genomically differentiating gene) is one that exists in one organism and not in another that lives in a different niche, and where the niches differ along the relevant dimension:</p>
<pre>    (differentiating-gene ?gene1 ?organism1 ?dim)    ⇔    (and     (exists      ((?organism2 :sort species))      (and       (differentially-ecotyped ?organism1 ?organism2 ?dim)       (gene-in-organism ?gene1 ?organism1)       (not (exists                    ((?gene2 :sort gene))                    (gene-has-ortholog-in-organism                      ?gene1 ?gene2 ?organism2)                    )))) </pre><a id="article1.body1.sec2.sec3.p10" name="article1.body1.sec2.sec3.p10"></a><p>Note that this axiom asserts the existence of a second organism, unspecified in the formula that calls upon this axiom. When SNARK encounters the <span class="monospace">not exists</span> clause it will attempt to find a gene (<span class="monospace">?gene2</span>) that is an ortholog of the given gene (<span class="monospace">?gene1</span>) in the second organism (<span class="monospace">?organism2</span>). If there <em>is</em> such a gene, this branch of the search will <em>fail</em>, causing the specific gene <span class="monospace">?gene1</span> to be rejected as a differentiating gene, in which case SNARK will try another. On the other hand, if there is no such gene, <span class="monospace">?gene2</span>, this branch of search will succeed, and the proof will continue. (Theorem-proving aficionados will note that this is true negation, not negation-as-failure; we need to know that no ortholog exists, not that we have merely failed to find one. There is a closed-world assumption applied to the procedural attachment: If the attachment finds no ortholog, we assert that no ortholog exists.)</p>
<a id="article1.body1.sec2.sec3.p11" name="article1.body1.sec2.sec3.p11"></a><p>We define the concept of being differentially ecotyped as follows: There is another organism (presumably in the same group as our target organism) that lives in a different environment regarding the specified dimension (light, in the present case). Here we very roughly approximate the qualities of light as low vs. high.</p>
<pre>    (differentially-ecotyped ?organism1 ?organism2 ?dim)    ⇔    (or     (and      (organism ?organism :environment ?dim                 :quality high)      (organism ?organism2 :environment ?dim                 :quality low))     (and      (organism ?organism :environment ?dim :quality low)      (organism ?organism2 :environment ?dim                 :quality high))) </pre><a id="article1.body1.sec2.sec3.p12" name="article1.body1.sec2.sec3.p12"></a><p>Note that this must be expressed in both directions (across the or) to handle the case in which <span class="monospace">?organism1</span> is high and <span class="monospace">?organism2</span> low, and vice versa. (Also, this will only work if the terms “High” and “Low” are both used by the author of the query and by the author of the subject domain theory. Any project of this sort will confront such terminological issues. We have chosen these particular terms because biologists refer to med4 as a the <em>high light strain</em> and mit9313 as a <em>low light strain</em> <a href="#pone.0000339-Bhaya1">[10]</a>, so there is more likely to be agreement in this case.)</p>
<a id="article1.body1.sec2.sec3.p13" name="article1.body1.sec2.sec3.p13"></a><p>Finally, to determine differential regulation we use BioBike's knowledge of microarray experimental results on the given (light) dimension (i.e., from the Hihara et al. <a href="#pone.0000339-Hihara1">[11]</a> data) to find a gene on a microarray in a relevant orthologous organism on which a relevant experiment has been conducted, and then to test for up-regulation of the ortholog:</p>
<pre>    (differentially-regulated ?gene ?dim)    ⇔    (exists     ((?experiment :sort experiment)      (?organism3 :sort species)      (?gene3 :sort gene))    (and     (experiment ?experiment :dimension ?dim                   :organism ?organism3)     (gene-has-ortholog-in-organism ?gene                        ?gene3 ?organism3)     (&gt; (regulation-ratio ?gene3 ?experiment) 2.0)     )) </pre><a id="article1.body1.sec2.sec3.p14" name="article1.body1.sec2.sec3.p14"></a><p>Here <span class="monospace">regulation-ratio</span> is a procedural attachment that returns a value from a given experiment for a given gene, and the whole (&gt; ...) clause ensures that the gene under consideration has a significant (2× up-regulated) response in that experiment. (One may well wonder, after adding all these axioms, whether the subject domain theory is consistent. Asking SNARK to prove “false” will (eventually) root out inconsistencies. Unfortunately the time that this takes is unpredictable so one must decide when to stop it and accept that the theory is consistent.)</p>


<h4>Simple ground knowledge</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>In addition to the axiomatization above, and extensive knowledge of genes, organisms, ontologies, and microarray data built into BioBike and accessed by SNARK via procedural attachments (as described in the next section), we must provide knowledge that is specific to the present problem. Since the query is expressed in terms of the word “light”, most of this relates that term to various organisms and experiments.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>These first three assertions tell SNARK that light varies along three qualitative dimensions, low, medium, and high, and that the light environments of the three organisms, mit9133, s6803, and promed4, are those niches respectively:</p>
<pre>    (organism mit9313 :environment light :quality low)    (organism s6803 :environment light :quality medium)    (organism med4 :environment light :quality high) </pre><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>These assertions are to be read as: The organism “mit9313” on the environmental dimension “light” has the quality value “low”, and so on. We must also relate the Hihara et al. <a href="#pone.0000339-Hihara1">[11]</a> experiment to the light dimension, and indicate that this experiment was conducted on the Synechococcus sp. strain PCC6803 (s6803) organism:</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p><span class="monospace">(experiment hihara :dimension light :organism s6803)</span></p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>If desired, an arbitrary amount of additional irrelevant knowledge could be added to make this example more realistic, but because theorem proving as used here is formally monotonic such additional knowledge would not change the results of the example, although it might slow down the proof process as false leads and dead ends are explored and rejected by the theorem prover.</p>


<h4>Procedural attachments</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Finally, we provide concepts that are implemented by procedural attachment in the BioBike system:</p>


<ul class="bulletlist">

<li><span class="monospace">(gene-in-organism ?gene ?organism)</span>—This relation asserts that ?gene is a gene of the given <span class="monospace">?organism</span>. Procedural attachments to this relation compute either the organism containing a given gene, or, inversely, all the genes contained in a given organism.</li>

<li><span class="monospace">(ortholog ?gene1 ?gene2)</span>—BioBike has a built-in concept of two-way orthology between given genes (more precisely, between the proteins produced by given genes). A procedural attachment to this <span class="monospace">ortholog</span> relation computes the (protein-based, two-way) gene orthologous to a given gene.</li>

<li><span class="monospace">(gene-has-ortholog-in-organism ?gene ?gene1 ?organism)</span></li>

</ul><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Given the concepts of gene-in-organism and ortholog, as above, it is easy to introduce an axiom that conceptually defines gene-has-ortholog-in-organism as follows:</p>
<pre>    (gene-has-ortholog-in-organism ?gene1 ?gene2 ?organism)     ⇔      (and        (ortholog ?gene1 ?gene2)        (gene-in-organism ?gene2 ?organism)) </pre><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>That is, a given gene has an ortholog in a specific given organism if and only if there is a gene <span class="monospace">(?gene2)</span> in the target organism and that gene is orthologous (as defined above) to our given gene.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>When SNARK encounters expressions with procedural attachments, such as:</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p><span class="monospace">(gene-in-organism ?gene med4)</span></p>
<a id="article1.body1.sec2.sec5.p6" name="article1.body1.sec2.sec5.p6"></a><p>a data source is invoked that yields all the genes in the given organism (Procholorococcus sp. strain Med4). In different branches of the search space, the variable <span class="monospace">?gene</span> will be systematically replaced by each of these genes, respectively. The procedural-attachment mechanism allows the theorem prover to behave as if the axioms of the theory express the complete list of the genes of promed4, while in reality that knowledge is imported from the external data source only when it is needed.</p>
<a id="article1.body1.sec2.sec5.p7" name="article1.body1.sec2.sec5.p7"></a><p>Two somewhat more problem-specific primitives are needed in order to work with microarray data, and to examine gene annotations:</p>


<ul class="bulletlist">

<li><span class="monospace">(regulation-ratio ?gene3 ?experiment)</span>—This function returns a number representing the mean up-or-down regulation ratio of a given gene in a given microarray experiment. The experiment name and gene object must both be given. In the present example the data from the experiment of Hihara et al. <a href="#pone.0000339-Hihara1">[11]</a> is available and has been massaged as described in <a href="#pone.0000339.s001">Appendix S1</a> to provide a mean regulation ratio for each gene.</li>

<li><span class="monospace">(photosynthesis-related ?gene)</span>—Finally, among the primitives that we take as built in via BioBike procedural attachments is a test for a gene object being photosynthetically related. This test could work in any number of ways, ranging from reading the annotation field for the gene, to reading information from a functional ontology regarding the given gene, to asking the biologist's opinion. How this is implemented is not relevant to the present discussion so we have simply implemented it to search the gene's Cyanobase (<span class="monospace"><a href="http://www.kazusa.or.jp/cyano/">www.kazusa.or.jp/cyano/</a></span>) annotations (which are built into the BioBike knowledge base) for the string “light” or “photo”. This is not a highly certain way of finding photosynthetically related genes (esp. given the concerns raised by Shrager <a href="#pone.0000339-Shrager1">[13]</a>), but it will do for the present example.</li>

</ul><a id="article1.body1.sec2.sec5.p8" name="article1.body1.sec2.sec5.p8"></a><p>We also add assertions that tell us that when we ask about the light semantics along the light dimension, SNARK should make use of this built-in photosynthesis-related predicate to determine whether or not this is the case for a given gene:</p>
<pre>    (gene-semantics ?gene light)    ⇔    (photosynthesis-related ?gene) </pre><a id="article1.body1.sec2.sec5.p9" name="article1.body1.sec2.sec5.p9"></a><p>We will return in the discussion to consider other interesting ways in which this axiom could be expressed.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We provided BioDeducta all of the above and asked it to find a gene (and other related terms) that satisfy our query:</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p><span class="monospace">(adaptive-gene ?gene med4 light)</span></p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Once the proof is complete, the theorem prover extracts an answer to the query by examining what term replaces the variables <span class="monospace">?gene: PMED4.PMM0817</span>. This is the only answer for this specific query, and it is correct. This proof takes about a minute to find this answer (on a dual Athlon Linux processor with 1G memory). By examining the proof (provided in the online supplementary materials) we find that in order to obtain this answer, SNARK discovered these auxiliary answers, which can be understood by finding their locations in the axioms of the subject domain theory, above (and <a href="#pone.0000339.s002">Appendix S2</a>):</p>
<pre>    ?gene: #$PMED4.PMM0817    ?organism2: #$prochlorococcus_marinus_mit9313    ?experiment: HIHARA  ?organism3: #$synechocystis_pcc6803    ?gene3: #$S6803.ssr2595 </pre><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In other words, a low-light organism that has no ortholog to <span class="monospace">?gene</span> is <span class="monospace">prochlorococcus_marinus_mit9313 (pro9313)</span>. Experiments were performed by Hihara on the organism <span class="monospace">synechocystis_pcc6803 (s6803)</span>, and a high regulation ratio was discovered in those experiments on gene <span class="monospace">S6803.ssr2595</span>, which is an ortholog of PMM0817. The annotation for PMM0817 reads: “possible high-light inducible protein”. Indeed, the high-light inducible (hli) proteins (genes) have been previously identified as being possibly involved in the adaptation of the Procholorococci to different environmental niches by Bhaya et al. <a href="#pone.0000339-Bhaya1">[10]</a>, who identified members of the hli gene family in seven cyanobacterial genomes, including those of Prochlorococcus sp. strain Med4 nad Prochlorococcus sp. strain MIT9313. PMM0817 is called “hli17” in Bhaya et al <a href="#pone.0000339-Bhaya1">[10]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>We conducted a number of additional experiments, demonstrating that BioDeducta's results mirror the results computed by the equivalent BioLisp programs run in the same BioBike database. For example, as was mentioned above, one may quibble with the specifics of our choice of definitions, but it is easy to change the meanings of terms, or to add alternative formulations that modularly work together with existing axioms. For example, one might wish to change the definition of <span class="monospace">adaptive-gene</span> as follows:</p>
<pre>    (adaptive-gene ?gene1 ?organism1 ?dim)    ⇔    (and     (gene-in-organism ?gene1 ?organism1)     (differentiating-gene ?gene1 ?organism1           ?organism2 ?dim)) </pre><a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Note that the <span class="monospace">gene-semantics</span> and <span class="monospace">differentially-regulated</span> clauses are deleted. Re-executing the proof with this axiom produces 340 results (for <span class="monospace">?organism2</span>  =  pro9313). These (and other variations of this example) were checked by equivalent BioLisp code.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Having demonstrated how a combination of axiomatic reasoning, answer extraction, and procedural attachment may offer biologists access to powerful biocomputing analyses, we next turn to discussion of some closely related work, followed by discussion of some of the issues and opportunities raised by our work.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3>
<h4>Closely related work</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>A bevy of activity in biocomputing is concerned with the formal representation and reasoning about biological pathways. An excellent example of this is the Pathway Logic work based on the Maude rewriting logic paradigm <a href="#pone.0000339-Eker1">[14]</a>, <a href="#pone.0000339-Talcott1">[15]</a>. Although based upon somewhat different technology (rewriting systems vs. first-order theorem proving), we share the methodology of expressing a formal subject-domain theory, and then reasoning from it to transform a conjecture into a form that affords various sorts of analyses, such as model checking.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The fact that Pathway Logic operates in the domain of biological signaling pathways, whereas the examples in this paper are in the domain of genomic conjectures, is an incidental difference resulting from our respective choices of problems. Because we both use explicit models, we can in principle do one another's problems by representing one another's subject domain theories. That is, given a subject domain theory containing axioms that define temporally related biological events, BioDeducta will do exactly the same work as Pathway Logic. Indeed, in the BioBike Live Tutorials that come with the BioBike system we develop several other examples, including one that analyzes protein regulation models (see the Software Availability section, below). The subject domain theory for that example includes axioms such as:</p>
<pre>    (and (controls ?protein1 ?protein2)       (controls ?protein2 ?protein3))    ⇒ [i.e., implies]    (controls ?protein1 ?protein3)) </pre><a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>One can consider Maude as a sub-logic of SNARK, specialized to certain forms of rewriting-based reasoning. Although far less general than SNARK, Maude is very efficient for certain kinds of problems, such as reachability (e.g., Can a certain molecule be generated from given precursors?) What is ultimately needed is an integration of SNARK with Maude (and other tools) so that queries that are solvable in principle in first-order logic can be solved efficiently with specialized logically sound algorithms such as the model-checking techniques available in Maude. (We thank Mark-Oliver Stehr for this insightful discussion.)</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>Some other systems, such as HyBrow <a href="#pone.0000339-Racunas1">[16]</a>, try to decide the validity of a declarative conjecture from biological knowledge and/or data. The user of HyBrow expresses fully grounded hypotheses (i.e., conjectures without variables) as sets of temporally related biological events. HyBrow evaluates these hypotheses against genomic and microarray data and may offer alternative “neighboring” hypotheses with improved fit to the data. Although hypotheses in HyBrow are fully grounded, whereas the BioDeducta example given in the present paper contains variables, this is an illusory difference; we could just as well have grounded the <span class="monospace">adaptive-gene</span> conjecture with specific genes, in which case the BioDeducta methodology looks very similar to HyBrow's. Indeed, grounding the conjecture in specific genes (and other abstracted variables) is precisely how SNARK goes about proving it. Moreover, HyBrow's ability to propose “improved” hypotheses is orthogonal to its conjecture validation mechanism, and is a property of many other systems (e.g., <a href="#pone.0000339-Shrager2">[17]</a>). Moreover, the HyBrow concept of “neighboring” hypotheses is ad hoc and dependent upon the specific representation of conjectures, which is (as described above) built into HyBrow's code and is not explicit. Thus, although BioDeducta's ability to replace variables seems to be narrower than HyBrow's neighboring hypothesis concept, this is illusory because of the choice of representation in the present examples. If we had used an example that looked like a HyBrow temporal model, and provided an axiomatization for it, the BioDeducta representation could afford the same representational “hinges” that HyBrow uses to compute neighboring hypotheses. Moreover, because of the declarative axiomatization of the subject domain theory in BioDeducta, these representational hinges could be manipulated much more directly than is the case in HyBrow, which is a “black box” program. In these ways BioDeducta is more general than HyBrow.</p>
<a id="article1.body1.sec4.sec1.p5" name="article1.body1.sec4.sec1.p5"></a><p>Furthermore, as with Pathway Logic, the apparent difference in domain between Hybrow and BioDeducta is merely a happenstance of the examples we have chosen. Axioms such as:</p>
<pre>     (precedes ?event1 ?event2)      □     (exists ((?time1b :sort time)        ...etc...)    (and      (event-begins-at ?event1 ?time1b)      (event-ends-at ?event1 ?time1e)      (event-begins-at ?event2 ?time2b)      (event-ends-at ?event1 ?time2e)      (non-overlapping-ranges ?time1b ?time1e                    ?time1b ?time1e))) </pre><a id="article1.body1.sec4.sec1.p6" name="article1.body1.sec4.sec1.p6"></a><p>could serve the purpose in BioDeducta of computing the same sorts of analyses as HyBrow. Indeed, SNARK has as a built-in version of the Allen temporal calculus <a href="#pone.0000339-Allen1">[18]</a> for reasoning about time points and intervals in just this manner, so building this logic would not be difficult.</p>
<a id="article1.body1.sec4.sec1.p7" name="article1.body1.sec4.sec1.p7"></a><p>The central difference between BioDeducta (or Pathway Logic) and HyBrow is that BioDeducta is an inference engine armed with an explicit axiomatization of its subject domain theory, whereas HyBrow is an ad hoc program in which the equivalent of the subject domain theory is built into special-purpose code. Therefore, HyBrow cannot carry out inference and so HyBrow users cannot use high-level descriptions that are grounded by a subject domain theory in a principled way. Furthermore, HyBrow cannot give explanations that involve such transformations (even if they were carried out in the ad hoc HyBrow code). In these important ways, BioDeducta goes well beyond HyBrow and HyBrow-like systems.</p>


<h4>Issue: Efficiency and the undecidability of full first-order logic</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Being very general, SNARK's proof process will usually be slower than specially written BioLisp programs because the latter may take advantage of specific properties of the problem to speed up search. Our solution to the light acclimation query is slower than a cleverly crafted program, but faster than a naive one. It does, however, require theory-specific domain engineering and strategic work to achieve good performance. This may be chalked up to the price one pays for the flexibility afforded by using a full first-order theorem prover with an explicit subject domain theory.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Full first-order logic is undecideable in general, meaning that there is no hope of developing a fast <em>general-purpose</em> reasoning capability for it. However, within a particular subject-domain theory it is often possible to develop strategic controls that allow the theorem prover to exhibit performance rivaling that of special-purpose systems. Moreover, where the objects involved are finite, as is the case here for genomes, etc., it is likely that most theories are naturally decidable (although one can certainly go out of one's way to build an undecideable theory if one wishes to do so).</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>Through the use of weights and clause ordering, the designer of the subject domain theory can force certain sub-formulae to be treated before others. In the present example, for instance, some symbols have procedural attachments while others do not. Some procedural attachments, such as <span class="monospace">gene-in-organism</span>, can be very expensive, since an organism may have thousands of genes, whereas others, such as <span class="monospace">gene-has-ortholog-in-organism</span>, are relatively cheap because a gene usually has only a small number of orthologs in a given organism. We can weight symbols in proportion to the expected number of solutions offered by the corresponding attached procedure.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>Useful properties may also be stated for relations, such as symmetry or reflexivity. For example, <span class="monospace">ortholog</span> can be declared to be commutative (i.e., symmetric) but in some cases this gives poorer running time. One can also state a reflexivity axiom: (<span class="monospace">ortholog ?gene ?gene</span>), which may will allow some clauses to be dropped by subsumption and improve the search efficiency. Not only is the present theory decidable, but by judicious use of these mechanisms it is also efficient.</p>
<a id="article1.body1.sec4.sec2.p5" name="article1.body1.sec4.sec2.p5"></a><p>Regardless of all these manipulations, given that s6803 has 3722 genes, promed4 has 1760, and pro9313 has 2328, even if it takes several minutes for SNARK to compute a solution, this is far less time than it would take a biologist to do the same work manually or using spreadsheets.</p>


<h4>Issue: The complexity of domain theory formulation</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Even aside from these tuning details, the mere construction of a subject domain theory is a complex and error-prone task. Fortunately, in theory it need be done only once for each subject domain. Furthermore, we do not need to begin from scratch but can import appropriate sections of subject domain theory from such standards as Cycorp's OpenCyc <a href="#pone.0000339-Cycorp1">[19]</a> and Teknowledge's SUMO <a href="#pone.0000339-Niles1">[20]</a>, <a href="#pone.0000339-1">[21]</a> and more specific ontologies for the selected subject domain such as the axiomatization of molecular biology included in the Library of Ontologies of the Laboratory for Applied Ontology [<a href="http://www.loa-cnr.it/">http://www.loa-cnr.it/</a>], which has been under development for many years. Other ontologies that are being developed for biology include the Ontolingua Molecular Biology Theory (<a href="http://www.loa-cnr.it/medicine/molecular-biology">www.loa-cnr.it/medicine/molecular-biolog​y</a>) and the Open Biological Ontologies project (obo.sourceforge.net/main.html); see also <a href="#pone.0000339-Baclawski1">[22]</a>.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>One can also build a subject domain theory by composing simpler theories. One should think of the subject domain theory as a sort of dictionary of biological concepts; it modularly describes how concepts are cached out in terms of other (simpler) concepts, eventually reaching ground facts or underlying computations. The development of any dictionary is not trivial, but because of the generality of the theorem prover, the in-principle modularity of the subject domain theory is essentially guaranteed (although not necessarily its efficiency, unless steps such as those described above are taken as well).</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>Of course, merging subject domain theory components may not be straightforward; different theories may use different symbols for describing the same concept, or may use the same symbol with different meanings. The notions of theory morphism and colimit, obtained from the mathematical theory of categories <a href="#pone.0000339-Barr1">[23]</a>, has been found to be valuable for this purpose <a href="#pone.0000339-Burstall1">[24]</a>, <a href="#pone.0000339-Smith1">[25]</a>. A theory morphism is a meaning-preserving mapping that identifies symbols in one theory with other symbols in a different theory; the colimit then allows us to compose the theories, taking these identifications into account. In this way, one can combine theories even if component theories use different vocabulary for the same concept, or the same vocabulary for different concepts. Specware <a href="#pone.0000339-Kestrel1">[26]</a> is a category-theory-based framework that implements essential theory-manipulation operations, including theory morphism and colimit, required for the composition of multiple ontologies and theories. It provides an interface that allows us to access and generate SNARK theories and proofs.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>Regardless of the specific approaches to mitigating issues of efficiency and of the completeness and correctness of the subject domain theory (not to mention ambiguity and arguments about definitions!), problems are bound to arise in any project of the sort we have described. As with any such project, only the long-term efforts of a dedicated community can work these out. We offer technology that is powerful enough to afford correct, complete, efficient, and explicit solutions—the biocomputation community will, over time, work out the details. BioBike is a collaborative platform within which such a community can engage in efforts of this sort.</p>


<h4>Opportunities: Quasi-natural language and query elicitation</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We opened this paper with the biologist in increasing need of the ability to conduct novel computational analyses without programming, and offered BioDeducta as an approach to this. Whereas BioDeducta may provide significant opportunities in terms of methodological modularity and provenance, it may still be difficult to imagine biologists expressing queries in SNARK's logical notation (or, similarly, writing SNARK axioms). One approach to this problem is to provide graphical support for query formulation, such as was done in NASA's Amphion system <a href="#pone.0000339-Lowry1">[27]</a>, <a href="#pone.0000339-Stickel2">[28]</a> which automatically produces software for planetary astronomers. Amphion accepts user queries formulated with the help of a graphical query-formulation guide, producing a diagrammatic representation of the query. The diagram is then translated into a logical conjecture, which is passed to SNARK for proof.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Another approach to simplifying the query (or theory) formulation task is to use quasi-natural-language. Whereas true natural language programming (or at least querying) has been a holy grail of AI since time immemorial, we do not imagine that this is possible in the near term, even in narrow domains such as biocomputing. However, the fact that the BioDeducta subject domain theory is explicitly formulated makes certain sorts of quasi-natural language a real possibility. SRI's GeoLogica and QUARK <a href="#pone.0000339-Waldinger1">[29]</a> are both experimental systems that use deductive methods very similar to the way this is done in BioDeducta to compose heterogeneous data and software components: A logical form is presented as a conjecture to SNARK, and an answer to the query is extracted from the proof. While GeoLogica responds to queries posed by an Earth systems scientist, QUARK serves as an assistant to intelligence analysts. Both systems accept queries in English which are translated into logical form by a natural-language parser.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>We have conducted preliminary experiments with natural language in BioDeducta, using the same method as described for the GeoLogica system <a href="#pone.0000339-Waldinger1">[29]</a>. Briefly, a query is parsed by Gemini, a general parser, and translated into a logical form, phrased as a conjecture, and submitted to SNARK. As above, the proof produces a set of bindings for free variables in the logical form produced by Gemini. These bindings constitute answers to the question.</p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4"></a><p>For example, the query</p>
<a id="article1.body1.sec4.sec4.p5" name="article1.body1.sec4.sec4.p5"></a><p>“Find a gene that pertains to Promed4 and that does not have an ortholog in Pro9313.”</p>
<a id="article1.body1.sec4.sec4.p6" name="article1.body1.sec4.sec4.p6"></a><p>translates to a logical form that is thence translated by a language subject domain theory into a BioDeducta conjecture, which is thence proved, just as was done in the hli example, above. In the end, the proof produces the answer</p>
<a id="article1.body1.sec4.sec4.p7" name="article1.body1.sec4.sec4.p7"></a><p><span class="monospace">#$PMED4.PMM0030</span></p>
<a id="article1.body1.sec4.sec4.p8" name="article1.body1.sec4.sec4.p8"></a><p>Examining alternative proofs to the theorem produces multiple answers:</p>
<a id="article1.body1.sec4.sec4.p9" name="article1.body1.sec4.sec4.p9"></a><p><span class="monospace">#$PMED4.PMM0038</span></p>
<a id="article1.body1.sec4.sec4.p10" name="article1.body1.sec4.sec4.p10"></a><p><span class="monospace">#$PMED4.PMM0051</span></p>
<a id="article1.body1.sec4.sec4.p11" name="article1.body1.sec4.sec4.p11"></a><p>This example required 5 seconds to produce the first answer, and additional answers were almost instantaneous.</p>
<a id="article1.body1.sec4.sec4.p12" name="article1.body1.sec4.sec4.p12"></a><p>Many other sorts of queries can be addressed by BioDeducta using this approach. Examples include</p>
<a id="article1.body1.sec4.sec4.p13" name="article1.body1.sec4.sec4.p13"></a><p>“Does pmm0226 not have an ortholog in mit9313?”</p>
<a id="article1.body1.sec4.sec4.p14" name="article1.body1.sec4.sec4.p14"></a><p>“What is the Hihara mean regulation ratio of pmm0226?”</p>
<a id="article1.body1.sec4.sec4.p15" name="article1.body1.sec4.sec4.p15"></a><p>Moreover, various semantically equivalent forms are acceptable. For example, one can ask about the “hihara ratio” or the “hihara regulation ratio”, which are taken to be synonymous.</p>
<a id="article1.body1.sec4.sec4.p16" name="article1.body1.sec4.sec4.p16"></a><p>Much as this is encouraging, we do not pretend to have solved the natural language problem for biocomputing. To express such queries (in any language, natural or otherwise) users must know a great deal about the system's capabilities. Natural language provides the illusion that the system can understand everything whereas it is difficult for a system to engage the user in a natural language dialogue to indicate what it does and does not understand.</p>
<a id="article1.body1.sec4.sec4.p17" name="article1.body1.sec4.sec4.p17"></a><p>While naïve users may not be able to formulate the appropriate logical query, we may be able to guide such users to formulate logical queries even if they are ignorant of logical notation or the vocabulary of the subject domain theory. This approach depends upon the use of a sorted theory, one in which each constant is assigned a sort, that is an indicator of a class to which it belongs; thus, promed4 may be declared to be of sort organism (or sort bacterium, a subsort of organism). Similarly, each function symbol is given a declaration of the sorts of arguments it requires and the sort of value it produces, and each relation has a declaration of the sorts of arguments it expects. Such declarations are valuable for a theorem prover in that they restrict search, admit shorter proofs, allow some error detection, and permit more concise axioms and queries. But a sorted theory is of special value in that it allows query elicitation. Let us imagine that a user was trying to formulate the complex query discussed above. The user might select the term promed4 from a menu of known organisms. Since promed4 is of sort organism, the system would offer a menu of relations and functions that accept terms of sort organism as arguments including, for example, <span class="monospace">adaptive-gene</span> and <span class="monospace">gene-in-organism</span>. An English paraphrase of the meanings of these relations could be provided. Alternatively, the user could type an approximation to the desired operator, and the system might offer near matches that accept terms of appropriate sort as arguments. As expression formation proceeds, the types of functions and relations constrain the types of arguments, and vice versa until the query is as complete as the user can make it, at which point it is turned over to SNARK. This same mechanism might be used to introduce new content or axioms into the system, thus enabling a subject domain expert who is ignorant of the language of logic or of the vocabulary of the existing subject-domain theory to be guided to add new axioms to the theory.</p>


<h4>Conclusions: On proof and provenance</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The goal of BioDeducta is to put biological computation directly into the hands of biologists themselves—to enable them to manipulate biological knowledge and data in an interactive computational environment, and to produce results that are backed up by explicit explanations (the proofs). But biocomputation is not a simple art, often requiring one to program complex navigations within and between complex knowledge bases. Although BioBike offers the full power of a mature programming language, it puts the burden of figuring out how to navigate the knowledge bases entirely upon the biologist/users themselves. BioDeducta can in principle assist the user by taking advantage of the guidance of an explicit subject domain theory to find its way through the knowledge base to answer complex queries. Of course, the more meta-knowledge is available to describe knowledge bases and their relationships, the more complete is BioDeducta's ability to offer assistance in this regard.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>In concluding, we wish to emphasize an aspect of the present approach that helps address a critical problem in computational biology: the problem of provenance—that is, tracking how results are calculated, especially as it applies in the annotation of biological function.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>The concept of gene function is highly problematic for a number of reasons, not the least of which being that the way in which function is determined by the annotator is not generally made explicit in the annotation. As more and more genomes come online faster and faster, functional annotation is more and more being done through computational methods rather than by experiment. Because the provenance—the data behind such annotations—is not stored, there is the potential—in fact, the near certainty!—of propagating errors and, complementarily, failing to propagate corrections <a href="#pone.0000339-Shrager1">[13]</a>. One approach to this is to record such provenance, but often there is nothing worthy of recoding because the annotation results from an opaque program such as HyBrow <a href="#pone.0000339-Racunas1">[16]</a>. The general BioDeducta method offers an approach to this problem, as follows.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>Recall that the explicit subject domain theory is not so much a theory of biology as a description of the way to expand high-level biological concepts in terms of more primitive concepts, finally reaching “ground” terms and functions in the knowledge base or BioBike functions. By virtue of this, the proof constructed by SNARK in the process of proving a given conjecture forms an explicit trace or “explanation”, including the specific axioms used, and the ways in which the variables were bound in the axioms that lead to a given result; the more explicit the subject domain theory, the more detailed the explanation. By virtue of this fact, BioDeducta proofs represent precisely the sort of explicit provenance whose absence is endangering the very underpinnings of molecular biology, and by recoding the proofs along with the results that underpinning could be, at least in part, restored!</p>


<h4>Software and example availability</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>BioDeducta is SNARK+BioBike. SNARK is built-in to the BioBike demo server, accessible through <span class="monospace"><a href="http://www.biobike.org">www.biobike.org</a></span>, and is usable without having to register. Because of a time limit on computations taking place in the demo server only simple BioDeducta proofs can be accomplished on that server. Users wishing to seriously experiment with BioDeducta should either ask Jeff Shrager <a href="mailto:(jshrager@stanford.edu)">(jshrager@stanford.edu)</a> for an account on a nondemo BioBike server, or should install BioBike and SNARK themselves. Both of these are Common Lisp programs and are open source freeware for research applications. The BioBike installation instructions (also at <span class="monospace"><a href="http://www.biobike.org">www.biobike.org</a></span>) explain how to download and install SNARK as well.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>The examples in this paper are available as BioBike Live Tutorials, also at <span class="monospace"><a href="http://www.biobike.org">www.biobike.org</a></span>. The BioBike Live Tutorial system walks students through the entire process of developing and running the examples, and includes explanation of some of the more important SNARK parameters.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000339"></div><a name="pone.0000339.s001" id="pone.0000339.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000339.s001">Appendix S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Preparation of the Hihara et al. (2001) Data</p>
<p class="siDoi">doi:10.1371/journal.pone.0000339.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000339.s002" id="pone.0000339.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000339.s002">Appendix S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Complete Refutation Proof for the adaptive gene conjecture resulting in PMM0817</p>
<p class="siDoi">doi:10.1371/journal.pone.0000339.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>Mike Travers designed and initially implemented the BioBike frame system and did much of the threading of the BioBike knowledge base. Yannick Pouliot offered us advice on problem domains and on the use of the BioWarehouse, and Carolyn Talcott and Merill Knapp helped us through discussion of data sources and formalization of the subject domain. The light acclimation example is based on a problem posed by Jeff Elhai of Virginia Commonwealth University. Carolyn Talcott and Mark-Oliver Stehr gave us many comments on this paper.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JS RW. Performed the experiments: JS RW. Analyzed the data: JS RW. Contributed reagents/materials/analysis tools: JS MS JM RW. Wrote the paper: JS RW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000339-Fedoroff1" id="pone.0000339-Fedoroff1"></a>Fedoroff N, Racunas S, Shrager J (2005) Tools for thought in the age of biological knowledge. The Scientist  19(11): 20–21.  <ul class="find" data-citedArticleID="941458"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Tools+for+thought+in+the+age+of+biological+knowledge.&amp;auth=&amp;atitle=Tools+for+thought+in+the+age+of+biological+knowledge." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tools+for+thought+in+the+age+of+biological+knowledge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tools+for+thought+in+the+age+of+biological+knowledge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000339-Massar1" id="pone.0000339-Massar1"></a>Massar JP, Travers M, Elhai J, Shrager J (2005) BioLingua: A programmable knowledge environment for biologists. BioInformatics  21(2): 199–207.  <ul class="find" data-citedArticleID="941472" data-doi="10.1093/bioinformatics/bth465"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth465" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BioLingua%3A+A+programmable+knowledge+environment+for+biologists." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BioLingua%3A+A+programmable+knowledge+environment+for+biologists.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000339-Green1" id="pone.0000339-Green1"></a>Green CC (1969) Application of theorem proving to problem solving.   Proc. International Joint Conference on Artificial Intelligence, Washington, DC.  <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0000339-Manna1" id="pone.0000339-Manna1"></a>Manna Z, Waldinger R (1980) A deductive approach to program synthesis. ACM Transactions on Programming, Languages, and Systems  2: 90–121.  <ul class="find" data-citedArticleID="941470" data-doi="10.1145/357084.357090"><li><a href="http://dx.doi.org/10.1145/357084.357090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+deductive+approach+to+program+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+deductive+approach+to+program+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000339-Stickel1" id="pone.0000339-Stickel1"></a>Stickel ME, Waldinger RJ, Chaudhri VK (2000) A Guide to SNARK.   SRI International, Menlo Park, CA. (also: <a href="http://www.ai.sri.com/SNARK/tutorial/tutorial.html">www.ai.sri.com/SNARK/tutorial/tutorial.h​tml</a>).  <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0000339-The1" id="pone.0000339-The1"></a>The Gene Ontology Consortium (2000) Gene ontology: A tool for the unification of biology. Nature Genet  25: 25–29.  <ul class="find" data-citedArticleID="941494"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+ontology%3A+A+tool+for+the+unification+of+biology.&amp;auth=&amp;atitle=Gene+ontology%3A+A+tool+for+the+unification+of+biology." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+ontology%3A+A+tool+for+the+unification+of+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+ontology%3A+A+tool+for+the+unification+of+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000339-Karp1" id="pone.0000339-Karp1"></a>Karp P, Paley S, Romero P (2002) The pathway tools software. Bioinformatics  18: S225–32.  <ul class="find" data-citedArticleID="941464" data-doi="10.1093/bioinformatics/18.suppl_1.s225"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.suppl_1.s225" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+pathway+tools+software." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+pathway+tools+software.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000339-Travers1" id="pone.0000339-Travers1"></a>Travers M, Massar JP, Shrager J (2005) The (re)birth of the knowledge operating system.   Proc. International Lisp Conference, Stanford, CA.  <ul class="find-nolinks"></ul></li><li><span class="label">9.
              </span><a name="pone.0000339-Partensky1" id="pone.0000339-Partensky1"></a>Partensky F, Hess WR, Vaulot D (1999) Prochlorococcus, a marine photosynthetic prokaryote of global significance. Microbiol Mol Biol Rev  63(1): 106–27.  <ul class="find" data-citedArticleID="941476"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Prochlorococcus%2C+a+marine+photosynthetic+prokaryote+of+global+significance.&amp;auth=&amp;atitle=Prochlorococcus%2C+a+marine+photosynthetic+prokaryote+of+global+significance." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prochlorococcus%2C+a+marine+photosynthetic+prokaryote+of+global+significance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prochlorococcus%2C+a+marine+photosynthetic+prokaryote+of+global+significance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000339-Bhaya1" id="pone.0000339-Bhaya1"></a>Bhaya D, Dufresne A, Vaulot D, Grossman A (2002) Analysis of the hli gene family in marine and freshwater cyanobacteria. FEMS Microbiology Letters,  215:   <ul class="find" data-citedArticleID="941450" data-doi="10.1111/j.1574-6968.2002.tb11393.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.2002.tb11393.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+hli+gene+family+in+marine+and+freshwater+cyanobacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+hli+gene+family+in+marine+and+freshwater+cyanobacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000339-Hihara1" id="pone.0000339-Hihara1"></a>Hihara Y, Kamei A, Kanehisa M, Kaplan A, Ikeuchi M (2001) DNA microarray analysis of cyanobacterial gene expression during acclimation to high light. Plant Cell  13(4): 793–806.  <ul class="find" data-citedArticleID="941462" data-doi="10.2307/3871341"><li><a href="http://dx.doi.org/10.2307/3871341" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+microarray+analysis+of+cyanobacterial+gene+expression+during+acclimation+to+high+light." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+microarray+analysis+of+cyanobacterial+gene+expression+during+acclimation+to+high+light.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000339-Russell1" id="pone.0000339-Russell1"></a>Russell S, Norvig P (1995) Artificial intelligence: A modern approach. Upper Saddle River, , NJ: Prentice Hall.   <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0000339-Shrager1" id="pone.0000339-Shrager1"></a>Shrager J (2003) The fiction of function. BioInformatics  19: 1934–1936.  <ul class="find" data-citedArticleID="941482" data-doi="10.1093/bioinformatics/btg277"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg277" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fiction+of+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fiction+of+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000339-Eker1" id="pone.0000339-Eker1"></a>Eker S, Knapp M, Laderoute K, Lincoln P, Meseguer J, Sonmez K (2002) Pathway Logic: Symbolic analysis of biological signaling.   Proc. Pacific Symposium on BioComputing, World Scientific Press.  <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0000339-Talcott1" id="pone.0000339-Talcott1"></a>Talcott C, Eker S, Knapp M, Lincoln P, Laderoute K (2004) Pathway Logic Modeling of Protein Functional Domains in Signal Transduction.   Proceedings of the Pacific Symposium on Biocomputing, World Scientific Press.  <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0000339-Racunas1" id="pone.0000339-Racunas1"></a>Racunas SA, Shah NH, Albert I, Fedoroff NV (2004) HyBrow: a prototype system for computer-aided hypothesis evaluation. Bioinformatics  20: Suppl 1i257–264.  <ul class="find" data-citedArticleID="941478" data-doi="10.1093/bioinformatics/bth905"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HyBrow%3A+a+prototype+system+for+computer-aided+hypothesis+evaluation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HyBrow%3A+a+prototype+system+for+computer-aided+hypothesis+evaluation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000339-Shrager2" id="pone.0000339-Shrager2"></a>Shrager J, Langley P, Pohorille A (2002) Guiding revision of regulatory models with expression data.   Proc. Pacific Symposium on BioComputing. World Scientific Press.  <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0000339-Allen1" id="pone.0000339-Allen1"></a>Allen JF (1983) Maintaining knowledge about temporal intervals. Commun. ACM  26(11).  <ul class="find" data-citedArticleID="941444" data-doi="10.1145/182.358434"><li><a href="http://dx.doi.org/10.1145/182.358434" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maintaining+knowledge+about+temporal+intervals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maintaining+knowledge+about+temporal+intervals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000339-Cycorp1" id="pone.0000339-Cycorp1"></a>Cycorp (2002) The Ontological Engineers' Handbook.  <a href="http://www.cyc.com/ doc/">http://www.cyc.com/ doc/</a>handbook/oe/oe-handbook-toc-opencyc.html.  <ul class="find" data-citedArticleID="941454"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Ontological+Engineers%27+Handbook.&amp;auth=&amp;atitle=The+Ontological+Engineers%27+Handbook." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Ontological+Engineers%27+Handbook." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Ontological+Engineers%27+Handbook.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000339-Niles1" id="pone.0000339-Niles1"></a>Niles I, Pease A (2001) Towards a standard upper ontology.   Proc. 2nd International Conference on Formal Ontology in Information Systems (FOIS-2001), Chris Welty and Barry Smith, eds, Ogunquit, Maine, October 17–19.  <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0000339-1" id="pone.0000339-1"></a>Teknowledge: SUMO Ontology.  Technical report., 2004, <a href="http://ontology.teknowledge.com/">http://ontology.teknowledge.com/</a>.  <ul class="find" data-citedArticleID="941442"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Teknowledge%3A+SUMO+Ontology.&amp;auth=&amp;atitle=Teknowledge%3A+SUMO+Ontology." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Teknowledge%3A+SUMO+Ontology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Teknowledge%3A+SUMO+Ontology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000339-Baclawski1" id="pone.0000339-Baclawski1"></a>Baclawski K, Niu T (2005) Ontologies for Bioinformatics. MIT Press.   <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0000339-Barr1" id="pone.0000339-Barr1"></a>Barr M, Wells C (1999) Category Theory for Computing Science. Montreal: Les Publications CRM.   third edition.  <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0000339-Burstall1" id="pone.0000339-Burstall1"></a>Burstall R, Goguen J (1977) Putting theories together to make specifications. pp. 1045–1058.  Proc. 5th International Joint Conference on Artificial Intelligence (IJCAI), Cambridge, Massachusetts.  <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0000339-Smith1" id="pone.0000339-Smith1"></a>Smith DR (2006) Composition by Colimit and Formal Software Development.   Technical report, Kestrel Institute.  <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0000339-Kestrel1" id="pone.0000339-Kestrel1"></a>Kestrel Institute (2004)   Specware, <a href="http://www.specware.org/">http://www.specware.org/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0000339-Lowry1" id="pone.0000339-Lowry1"></a>Lowry MR, Philpot A, Pressburger T, Underwood I (1994) AMPHION: Automatic Programming for Scientific Subroutine Libraries. ISMIS 326–335.  <ul class="find" data-citedArticleID="941468" data-doi="10.1007/3-540-58495-1_33"><li><a href="http://dx.doi.org/10.1007/3-540-58495-1_33" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AMPHION%3A+Automatic+Programming+for+Scientific+Subroutine+Libraries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AMPHION%3A+Automatic+Programming+for+Scientific+Subroutine+Libraries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000339-Stickel2" id="pone.0000339-Stickel2"></a>Stickel ME, Waldinger R, Lowry M, Pressburger T, Underwood I (1995)  Deductive composition of astronomical software from subroutine libraries. In: Bundy A, editor. Automated Deduction. Springer-Verlag Lecture Notes in Computer Science, Vol. 814.   <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0000339-Waldinger1" id="pone.0000339-Waldinger1"></a>Waldinger R, Hobbs J, Appelt DE, Fry J, Israel DJ, Jarvis P, Martin D, Riehemann S, Stickel ME, Tyson M, Dungan JL (2003) Deductive question answering from multiple resources.   New Directions in Question Answering. AAAI Press.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000339.XML" value="89477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000339.PDF" value="131692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000339.s001.DOC" value="27648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000339.s002.DOC" value="57344"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000339&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000339" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000339&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000339&volume=&issue=&title=Deductive Biocomputing&author_name=Jeff%20Shrager%2C%20Richard%20Waldinger%2C%20Mark%20Stickel%2C%20J.P.%20Massar&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000339" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000339" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000339" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000339&amp;t=Deductive%20Biocomputing" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000339&title=Deductive%20Biocomputing&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000339&amp;title=Deductive%20Biocomputing" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000339" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000339&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Deductive Biocomputing';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000339';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Deductive%20Biocomputing http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000339" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000339" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biologists%22" title="Search for articles in the subject area:'Biologists'"><div class="flagText">Biologists</div></a>
              <div data-categoryid="46129" data-articleid="23516"
                   data-categoryname="Biologists"
                   class="flagImage" title="Flag 'Biologists' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Controlled+vocabularies%22" title="Search for articles in the subject area:'Controlled vocabularies'"><div class="flagText">Controlled vocabularies</div></a>
              <div data-categoryid="46113" data-articleid="23516"
                   data-categoryname="Controlled vocabularies"
                   class="flagImage" title="Flag 'Controlled vocabularies' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+prediction%22" title="Search for articles in the subject area:'Gene prediction'"><div class="flagText">Gene prediction</div></a>
              <div data-categoryid="40279" data-articleid="23516"
                   data-categoryname="Gene prediction"
                   class="flagImage" title="Flag 'Gene prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23516"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Language%22" title="Search for articles in the subject area:'Language'"><div class="flagText">Language</div></a>
              <div data-categoryid="18829" data-articleid="23516"
                   data-categoryname="Language"
                   class="flagImage" title="Flag 'Language' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="23516"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+language%22" title="Search for articles in the subject area:'Natural language'"><div class="flagText">Natural language</div></a>
              <div data-categoryid="28651" data-articleid="23516"
                   data-categoryname="Natural language"
                   class="flagImage" title="Flag 'Natural language' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ontology+and+logic%22" title="Search for articles in the subject area:'Ontology and logic'"><div class="flagText">Ontology and logic</div></a>
              <div data-categoryid="46289" data-articleid="23516"
                   data-categoryname="Ontology and logic"
                   class="flagImage" title="Flag 'Ontology and logic' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4733'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1185'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3649&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=7313">this -&gt; thus</a><br>Posted by jshrager</p>
      <p><a href="/annotation/listThread.action?root=7307">iff (&lt;=&gt;) symbol displays incorrectly in HTML</a><br>Posted by jshrager</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
