

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Heterologous Tissue Culture Expression Signature Predicts Human Breast Cancer Prognosis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000145"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000145" />

    <meta name="citation_title" content="Heterologous Tissue Culture Expression Signature Predicts Human Breast Cancer Prognosis"/>
    <meta itemprop="name" content="Heterologous Tissue Culture Expression Signature Predicts Human Breast Cancer Prognosis"/>

      <meta name="citation_author" content="Eun Sung Park"/>
            <meta name="citation_author_institution" content="Laboratory of Genetics, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Ju-Seog Lee"/>
            <meta name="citation_author_institution" content="Laboratory of Experimental Carcinogenesis, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Hyun Goo Woo"/>
            <meta name="citation_author_institution" content="Laboratory of Experimental Carcinogenesis, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Fenghuang Zhan"/>
            <meta name="citation_author_institution" content="Donna and Donald Lambert Laboratory of Myeloma Genetics, Myeloma Institute for Research and Therapy, University of Arkansas for Medical Sciences, Little Rock, Arkansas, United States of America"/>
      <meta name="citation_author" content="Joanna H. Shih"/>
            <meta name="citation_author_institution" content="Biometric Research Branch, Division of Cancer Treatment and Diagnosis, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="John D. Shaughnessy Jr."/>
            <meta name="citation_author_institution" content="Donna and Donald Lambert Laboratory of Myeloma Genetics, Myeloma Institute for Research and Therapy, University of Arkansas for Medical Sciences, Little Rock, Arkansas, United States of America"/>
      <meta name="citation_author" content="J. Frederic Mushinski"/>
            <meta name="citation_author_institution" content="Laboratory of Genetics, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>

    <meta name="citation_date" content="2007/1/3"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000145.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e145"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Fibroblasts in cancer.; citation_author=R Kalluri; citation_author=M Zeisberg; citation_journal_title=Nature Reviews Cancer; citation_volume=6; citation_number=1; citation_pages=392-401; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A gene-expression signature as a predictor of survival in breast cancer.; citation_author=MJ van de Vijver; citation_author=YD He; citation_author=LJ van't Veer; citation_author=H Dai; citation_author=AA Hart; citation_journal_title=N Engl J Med; citation_volume=347; citation_number=2; citation_pages=1999-2009; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Gene expression profiling predicts clinical outcome of breast cancer.; citation_author=LJ van't Veer; citation_author=H Dai; citation_author=MJ van de Vijver; citation_author=YD He; citation_author=AA Hart; citation_journal_title=Nature; citation_volume=415; citation_number=3; citation_pages=530-536; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Oncogenic pathway signatures in human cancers as a guide to targeted therapies.; citation_author=AH Bild; citation_author=G Yao; citation_author=JT Chang; citation_author=Q Wang; citation_author=A Potti; citation_journal_title=Nature; citation_volume=439; citation_number=4; citation_pages=353-357; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Estrogen-regulated genes predict survival in hormone receptor-positive breast cancers.; citation_author=DS Oh; citation_author=MA Troester; citation_author=J Usary; citation_author=Z Hu; citation_author=X He; citation_journal_title=J Clin Oncol; citation_volume=24; citation_number=5; citation_pages=1656-1664; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gene expression profiling in breast cancer: understanding the molecular basis of histologic grade to improve prognosis.; citation_author=C Sotiriou; citation_author=P Wirapati; citation_author=S Loi; citation_author=A Harris; citation_author=S Fox; citation_journal_title=J. Natl. Cancer Inst; citation_volume=98; citation_number=6; citation_pages=262-272; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Classification and prediction of survival in hepatocellular carcinoma by gene expression profiling.; citation_author=JS Lee; citation_author=IS Chu; citation_author=JH Heo; citation_author=DF Calvisi; citation_author=ZT Sun; citation_journal_title=Hepatology; citation_volume=40; citation_number=7; citation_pages=667-676; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Distinct types of diffuse large B-cell lymphoma identified by gene expression profiling.; citation_author=A Alizadeh; citation_author=MB Eisen; citation_author=E Davis; citation_author=C Ma; citation_author=IS Lossos; citation_journal_title=Nature; citation_volume=403; citation_number=8; citation_pages=503-511; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular classification and molecular forecasting of breast cancer: Ready for clinical implication?; citation_author=JD Brenton; citation_author=LA Carey; citation_author=AA Ashour; citation_author=C Caldas; citation_journal_title=J. Clin Oncol; citation_volume=23; citation_number=9; citation_pages=7350-7360; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genes involved in invasion and metastasis of gastric cancer identified by array-based hybridization and serial analysis of gene expression.; citation_author=N Oue; citation_author=PP Aung; citation_author=Y Mitani; citation_author=H Kuniyasu; citation_author=H Nakayama; citation_journal_title=Oncology; citation_volume=69; citation_number=10; citation_pages=17-22. Review; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Prediction of venous metastases, recurrence, and prognosis in hepatocellular carcinoma based on a unique immune response signature of the liver microenvironment.; citation_author=A Budhu; citation_author=M Forgues; citation_author=Q-H Ye; citation_author=H-L Jia; citation_author=P He; citation_journal_title=Cancer Cell; citation_volume=10; citation_number=11; citation_pages=99-111; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The proliferation gene expression signature is a quantitative integrator of oncogenic events that predicts survival in mantle cell lymphoma.; citation_author=A Rosenwald; citation_author=G Wright; citation_author=A Wiestner; citation_author=WC Chan; citation_author=JM Connors; citation_journal_title=Cancer Cell; citation_volume=3; citation_number=12; citation_pages=185-97; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A novel prognostic subtype of human hepatocellular carcinoma derived from hepatic progenitor cells.; citation_author=JS Lee; citation_author=JH Heo; citation_author=L Libbrecht; citation_author=IS Chu; citation_author=P Kaposi-Novak; citation_journal_title=Nature Medicine; citation_volume=12; citation_number=13; citation_pages=410-416; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular portraits of human breast tumours.; citation_author=CM Perou; citation_author=T Sorlie; citation_author=MB Eisen; citation_author=M van de Rijn; citation_author=SS Jeffrey; citation_journal_title=Nature; citation_volume=406; citation_number=14; citation_pages=747-752; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Robustness, scalability and integration of a wound response gene expression signature in predicting breast cancer survival.; citation_author=HY Chang; citation_author=DSA Nuyten; citation_author=JB Sneddon; citation_author=T Hastie; citation_author=R Tibshirani; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=15; citation_pages=3738-3743; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene expression signature of fibroblast serum response predicts human cancer progression: similarities between tumors and wounds.; citation_author=HY Chang; citation_author=JB Sneddon; citation_author=AA Alizadeh; citation_author=R Sood; citation_author=RB West; citation_journal_title=PLoS Biology; citation_volume=2; citation_number=16; citation_pages=E7; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gene expression patterns of breast carcinomas distinguish tumor subclasses with clinical implications.; citation_author=T Sorlie; citation_author=CM Perou; citation_author=R Tibshirani; citation_author=T Aas; citation_author=S Geisler; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=17; citation_pages=10869-10874; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Revision of the American Joint Committee on Cancer staging system for breast cancer.; citation_author=PL Fitzgibbons; citation_author=DL Page; citation_author=D Weaver; citation_author=AD Thor; citation_author=DC Allred; citation_journal_title=Arch. Path. Lab. Med.; citation_volume=124; citation_number=18; citation_pages=966-978; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Repeated observation of breast tumor subtypes in independent gene expression data sets.; citation_author=T Sorlie; citation_author=R Tibshirani; citation_author=J Parker; citation_author=T Hastie; citation_author=JS Marron; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=19; citation_pages=8418-8423; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The hallmarks of cancer.; citation_author=D Hanahan; citation_author=RA Weinberg; citation_journal_title=Cell; citation_volume=100; citation_number=20; citation_pages=57-70; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Stromal fibroblasts in cancer initiation and progression.; citation_author=NA Bhowmick; citation_author=EG Neilson; citation_author=HL Moses; citation_journal_title=Nature; citation_volume=432; citation_number=21; citation_pages=332-337; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Expression profiling to predict outcome in breast cancer: the influence of sample selection.; citation_author=SK Gruvberger; citation_author=M Ringner; citation_author=P Eden; citation_author=A Borg; citation_author=M Ferno; citation_journal_title=Breast Cancer Res.; citation_volume=5; citation_number=22; citation_pages=23-26; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A paradigm for class prediction using gene expression profiles.; citation_author=MD Radmacher; citation_author=LM McShane; citation_author=R Simon; citation_journal_title=J Computational Biol; citation_volume=9; citation_number=23; citation_pages=505-511; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Pitfalls in the use of DNA Microarray Data for Diagnostic and Prognostic Classification.; citation_author=R Simon; citation_author=MD Radmacher; citation_author=K Dobbin; citation_author=LM McShane; citation_journal_title=J Natl Can Inst; citation_volume=95; citation_number=24; citation_pages=14-18; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Roadmap for developing and validating therapeutically relevant genomic classifiers.; citation_author=R Simon; citation_journal_title=J Clin Oncol; citation_volume=23; citation_number=25; citation_pages=7332-7341; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Neoplastic development in plasma cells.; citation_author=M Potter; citation_journal_title=Immunol Rev; citation_volume=194; citation_number=26; citation_pages=177-195; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Rapid viral induction of plasmacytomas in pristane primed BALB/c mice.; citation_author=M Potter; citation_author=MD Sklar; citation_author=WP Rowe; citation_journal_title=Science; citation_volume=182; citation_number=27; citation_pages=592-594; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=In vitro culture of primary plasmacytomas requires stromal cell feeder layers.; citation_author=A Degrassi; citation_author=DM Hilbert; citation_author=S Rudikoff; citation_author=AO Anderson; citation_author=M Potter; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=90; citation_number=28; citation_pages=2060-2064; citation_date=1993; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000145" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Heterologous Tissue Culture Expression Signature Predicts Human Breast Cancer Prognosis"/>
    <meta name="twitter:description" content="BackgroundCancer patients have highly variable clinical outcomes owing to many factors, among which are genes that determine the likelihood of invasion and metastasis. This predisposition can be reflected in the gene expression pattern of the primary tumor, which may predict outcomes and guide the choice of treatment better than other clinical predictors.Methodology/Principal FindingsWe developed an mRNA expression-based model that can predict prognosis/outcomes of human breast cancer patients regardless of microarray platform and patient group. Our model was developed using genes differentially expressed in mouse plasma cell tumors growing in vivo versus those growing in vitro. The prediction system was validated using published data from three cohorts of patients for whom microarray and clinical data had been compiled. The model stratified patients into four independent survival groups (BEST, GOOD, BAD, and WORST: log-rank test p&#8202;=&#8202;1.7&times;10&minus;8).ConclusionsOur model significantly improved the survival prediction over other expression-based models and permitted recognition of patients with different prognoses within the estrogen receptor-positive group and within a single pathological tumor class. Basing our predictor on a dataset that originated in a different species and a different cell type may have rendered it less sensitive to proliferation differences and endowed it with wide applicability.SignificancePrognosis prediction for patients with breast cancer is currently based on histopathological typing and estrogen receptor positivity. Yet both assays define groups that are heterogeneous in survival. Gene expression profiling allows subdivision of these groups and recognition of patients whose tumors are very unlikely to be lethal and those with much grimmer outlooks, which can augment the predictive power of conventional tumor analysis and aid the clinician in choosing relaxed vs. aggressive therapy."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000145.g008"/>

  <meta property="og:title" content="Heterologous Tissue Culture Expression Signature Predicts Human Breast Cancer Prognosis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000145" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6823'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=746'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3121&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000145">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000145" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000145&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Heterologous Tissue Culture Expression Signature Predicts Human Breast Cancer Prognosis
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eun Sung Park
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:parkeu@mail.nih.gov">parkeu@mail.nih.gov</a> (ESP); <a href="mailto:jm8p@nih.gov">jm8p@nih.gov</a> (JFM)</p>

                <p>Affiliation:
                  Laboratory of Genetics, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ju-Seog Lee, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Experimental Carcinogenesis, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>

                <p>
                    Current address: M.D. Anderson Cancer Center, Houston, Texas, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hyun Goo Woo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Experimental Carcinogenesis, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fenghuang Zhan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Donna and Donald Lambert Laboratory of Myeloma Genetics, Myeloma Institute for Research and Therapy, University of Arkansas for Medical Sciences, Little Rock, Arkansas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joanna H. Shih, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biometric Research Branch, Division of Cancer Treatment and Diagnosis, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John D. Shaughnessy Jr., 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Donna and Donald Lambert Laboratory of Myeloma Genetics, Myeloma Institute for Research and Therapy, University of Arkansas for Medical Sciences, Little Rock, Arkansas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            J. Frederic Mushinski
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:parkeu@mail.nih.gov">parkeu@mail.nih.gov</a> (ESP); <a href="mailto:jm8p@nih.gov">jm8p@nih.gov</a> (JFM)</p>

                <p>Affiliation:
                  Laboratory of Genetics, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 03, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000145</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000145" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000145" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000145" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000145" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000145-g001" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000145-g002" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000145-g003" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000145-g004" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000145-g005" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000145-t001" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000145-g006" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000145-g007" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000145-g008" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000145">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000145" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Cancer patients have highly variable clinical outcomes owing to many factors, among which are genes that determine the likelihood of invasion and metastasis. This predisposition can be reflected in the gene expression pattern of the primary tumor, which may predict outcomes and guide the choice of treatment better than other clinical predictors.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We developed an mRNA expression-based model that can predict prognosis/outcomes of human breast cancer patients regardless of microarray platform and patient group. Our model was developed using genes differentially expressed in mouse plasma cell tumors growing <em>in vivo</em> versus those growing <em>in vitro</em>. The prediction system was validated using published data from three cohorts of patients for whom microarray and clinical data had been compiled. The model stratified patients into four independent survival groups (BEST, GOOD, BAD, and WORST: log-rank test p = 1.7×10<sup>−8</sup>).</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our model significantly improved the survival prediction over other expression-based models and permitted recognition of patients with different prognoses within the estrogen receptor-positive group and within a single pathological tumor class. Basing our predictor on a dataset that originated in a different species and a different cell type may have rendered it less sensitive to proliferation differences and endowed it with wide applicability.</p>


<h3>Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec4.p1" name="article1.front1.article-meta1.abstract1.sec4.p1"></a><p>Prognosis prediction for patients with breast cancer is currently based on histopathological typing and estrogen receptor positivity. Yet both assays define groups that are heterogeneous in survival. Gene expression profiling allows subdivision of these groups and recognition of patients whose tumors are very unlikely to be lethal and those with much grimmer outlooks, which can augment the predictive power of conventional tumor analysis and aid the clinician in choosing relaxed vs. aggressive therapy.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Park ES, Lee J-S, Woo HG, Zhan F, Shih JH, et al.  (2007) Heterologous Tissue Culture Expression Signature Predicts Human Breast Cancer Prognosis. PLoS ONE 2(1):
          e145.
            doi:10.1371/journal.pone.0000145</p><p><strong>Academic Editor: </strong>Oliver Hofmann, South African National Bioinformatics Institute, South Africa</p><p><strong>Received:</strong> October 18, 2006; <strong>Accepted:</strong> December 11, 2006; <strong>Published:</strong> January 3, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>This research was supported in part by the Intramural Research Program of the NIH and the National Cancer Institute. JDS and FZ were supported by National Institutes of Health grants CA55819 and CA97513, the Fund to Cure Myeloma and the Peninsula Community Foundation.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Cancers are complex tissues whose behavior is strongly influenced by dynamic interactions between the cancer cells, the tumor's stromal cells and the extracellular matrix <a href="#pone.0000145-Kalluri1">[1]</a>. Stromal cells provide growth factors, blood supply, and mechanical support, and changes in this microenvironment can trigger tissue remodeling, setting the stage for tumor progression, invasion and metastasis. Since invasion and metastasis require tumor cells to survive and grow in sites quite different from the milieu in which they arose, we reasoned that adaptation of tumor cells to growth <em>in vitro</em> might require analogous changes in cell physiology, probably mirrored by changes in gene expression. Thus, we undertook the comparison of gene expression profiles between mouse plasma cell tumors (PCTs) growing in mice and PCTs that had been adapted to growth in tissue culture, hoping to gain insights into the genes responsible for the adaptation of this particular tumor to tissue culture conditions. Another goal for this study, which provides the basis for the present paper, was to determine whether these data might be extrapolatable to other tumor types and other species. More particularly, we hypothesized that the alterations in gene expression required for tumor cells to survive <em>in vitro</em> might be markers of human cancers that were particularly suited to growth in distant sites, i.e., more likely to invade or metastasize, two processes associated with poor prognosis and foreshortened survival. Specifically, we sought to test whether expression data from an experimental cancer model in mice, in this case plasma cell tumors, has the potential of uncovering survival/prognosis patterns in human cancers by transcending species-specific and cell lineage-specific gene expression patterns.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Cancer patients have highly variable clinical outcomes based on many factors including the genetic make-up of the patient, the genetic and phenotypic variability of the tumors and the way the tumors interact with their surrounding stroma. It is likely that this spectrum of clinical courses may also reflect different tumor-specific genetic predispositions to metastasize and gene expression heterogeneity that are incompletely recognized by classical diagnosis methods such as histopathological tumor typing and staging. This genetic predisposition might be reflected in specific patterns of gene expression, and it has long been hoped that microarray profiling of tumors' global gene expression could help identify subgroups of patients that differ in prognosis or in their response to available therapeutic modalities <a href="#pone.0000145-vandeVijver1">[2]</a>–<a href="#pone.0000145-Brenton1">[9]</a>. The ultimate goal is that gene expression profiles of a new patient's tumor could be analyzed in the context of a database of gene expression profiles from patients with known outcomes. In this way, treatment could be more precisely tailored to this patient's expected prognosis and predicted response to treatment.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We generated a mouse plasma cell tissue culture (PCT-TC) gene signature by comparing and contrasting the global gene expression of solid mouse plasma cell tumors with that of plasma cell tumors adapted to grow in tissue culture. We then used these signatures in meta-analysis of published reports of human breast cancer patients that included extensive long-term follow–up and survival data along with microarray data from these cancers. We devised three prediction models by which our PCT-TC signature identified subgroups of patients that could be stratified by their different survivals. In this way we identified and validated the existence of four distinct prognostic groups of breast cancer patients with significant differences in clinical outcomes. This method is superior to previously published expression-based survival prediction and may eventually be useful in predicting prognosis of new patients presenting with this disease.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Generation of mouse tissue culture signature</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>For the generation of the PCT-TC signature, microarray-based global gene expression analysis was performed on 27 individual RNA samples composed of 17 solid mouse PCTs and 10 tissue cultured PCT cell lines using Affymetrix U74Av2 microarray chips. We applied Significance Analysis of Microarrays (SAM) at the 99 percentile confidence level, and 1162 genes with a 0.001 False Discovery Rate (FDR) emerged as a signature that characterized the differences in gene expression between these two groups. Cluster analysis of these SAM-filtered genes revealed that most solid tumors showed similar expression patterns and clustered together, while the tissue cultured tumor cells clustered together, separated from their <em>in vivo</em> growing solid tumor counterparts regardless of tumor induction protocols (<a href="#pone-0000145-g001">Fig. 1A</a>). Approximately 70% of these genes showed lower expression in the cells grown <em>in vitro</em> (indicated in green on the heat map in <a href="#pone-0000145-g001">Fig. 1A</a>) compared with the solid tumors. Most of the genes that showed significantly lower expression in cells growing <em>in vitro</em> encode genes involved in angiogenesis, chemotaxis, component of extracellular matrix, complement activation, or cell motility-related genes, while the genes higher in expression in tissue cultured conditions are genes related to cell survival (see <a href="#pone.0000145.s009">Table S1</a>). Since these gene families had been cited in reports analyzing tumor invasion and metastasis <a href="#pone.0000145-Oue1">[10]</a>, <a href="#pone.0000145-Budhu1">[11]</a>, tumor-progression processes associated with poor prognosis and reduced survival; we decided to test this expression signature on the analysis of human cancer survival based on global gene expression patterns.</p>
<div class="figure" id="pone-0000145-g001"><div class="img"><a name="pone-0000145-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Mouse plasma cell tumor tissue culture (PCT-TC) signature and survival analysis of human cancer patients</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>27 RNA samples from 17 solid mouse plasma cell tumors and 10 tissue cultured mouse PCT-TC cell lines (including Baf3, a pre-B cell line) were used for the generation of PCT-TC signature. <strong>A.</strong> Mouse plasma cell tumor tissue culture signature. 1162 genes showing significant differences in expression between solid PCTs and tissue-cultured PCTs were selected by SAM analysis at the 99-percentile confidence level with a 0.001 FDR. <strong>B–D.</strong> Kaplan-Meier survival analysis of human cancer patients groups generated by unsupervised cluster analysis with mouse PCT-TC signature. <strong>B.</strong> Survival analysis of human mantle cell lymphoma patients group <a href="#pone.0000145-Rosenwald1">[12]</a> generated by unsupervised cluster analysis with 694 orthologs of the mouse PCT-TC signature genes. <strong>C.</strong> Survival analysis of human liver cancer patients <a href="#pone.0000145-Lee2">[13]</a> generated by unsupervised cluster analysis with 971 orthologs of the mouse PCT-TC signature genes. <strong>D.</strong> Survival analysis of human breast cancer patients <a href="#pone.0000145-vandeVijver1">[2]</a>, <a href="#pone.0000145-vantVeer1">[3]</a>, <a href="#pone.0000145-Chang1">[15]</a> generated by unsupervised cluster analysis with 470 orthologs of the mouse PCT-TC signature genes.</p>
<span>doi:10.1371/journal.pone.0000145.g001</span></div>

<h4>Cross-comparison of gene expression data between mouse plasma cell tumors with three sets of human cancer gene expression data</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We initially applied our PCT-TC signature to three human datasets in an attempt to stratify tumor patients into homogeneous groups that might reflect prognosis. Using only orthologous1Homologous murine and human genes that have diverged from each other as a consequence of speciation. Cf., <a href="http://www.informatics.jax.org/userdocs/homology_criteria.shtml">http://www.informatics.jax.org/userdocs/​homology_criteria.shtml</a> genes that showed significant difference of expression in SAM analysis of mouse PCT <em>in vivo</em> and <em>in vitro</em>, unsupervised hierarchical cluster analysis was performed with gene expression data from patients with mantle cell lymphoma <a href="#pone.0000145-Rosenwald1">[12]</a>, hepato-cellular carcinoma <a href="#pone.0000145-Lee2">[13]</a>, and breast cancer <a href="#pone.0000145-vandeVijver1">[2]</a>, independently. Kaplan-Meier plots and log-rank test for the patients clustering together based on similarities in gene expression, revealed significantly different survival estimates in each of these three data sets (<a href="#pone-0000145-g001">Fig. 1B–D</a>). Thus, the molecular differences that reflect tissue culture adaptation appeared to be associated with distinct differences in the clinical outcomes of patients with diverse human cancers.</p>


<h4>Construction and validation of prediction models for human breast cancer patient prognosis</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Having demonstrated that our mouse PCT-TC signature can reflect different prognostic features in human patients with three different types of cancer, we focused the application of our PCT-TC expression model to attempt to stratify prognostic subgroups in the well documented human breast cancer microarray data from Netherlands Cancer Institute (NKI) <a href="#pone.0000145-vandeVijver1">[2]</a>, <a href="#pone.0000145-vantVeer1">[3]</a>. Unsupervised hierarchical cluster analysis of half of these samples (the training set) using 460 orthologous genes corresponding to our mouse PCT-TC signature revealed two main clusters (Groups A and B), and each cluster was composed of three sub-clusters (A1, A2, A3 and B1, B2, B3, <a href="#pone-0000145-g002">Fig. 2A</a>).</p>
<div class="figure" id="pone-0000145-g002"><div class="img"><a name="pone-0000145-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Construction of human breast cancer patients' prognosis prediction models and evaluation of outcomes</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>
                     <strong>A.</strong> Unsupervised cluster analysis of NKI training data set (147 samples). It generated two main clusters and six sub-clusters of patients. <strong>B.</strong> Kaplan-Meier survival analysis of the two main clusters (Group A and Group B). <strong>C.</strong> Kaplan-Meier survival analysis of the six subclusters (Group A1–A3 and Group B1–B3). <strong>D.</strong> Kaplan-Meier survival analysis of two sub-clusters (Group A2 and Group B1) showing WORST and BEST prognosis and one group that includes all the others.</p>
<span>doi:10.1371/journal.pone.0000145.g002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Survival analysis of each subgroup by Kaplan-Meier plots with log-rank test showed significant differences between the two main clusters (<a href="#pone-0000145-g002">Fig. 2B</a>). The patients in the group with the better prognosis (Group B) had an overall 85% 10-year survival rate and a 90.7% 5-year survival rate while the group with poorer prognosis (Group A) had a 62.3% 10-year survival rate and a 73.8% 5-year survival rate. The difference was significant with a p value of 0.00245.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Kaplan-Meier plot and log-rank test was also performed using the 6 different subgroups of patients, three from each of the above-studied groups A and B. The log-rank test revealed that these six subgroups also had significant differences in survival, with a p value of 0.00285 (<a href="#pone-0000145-g002">Fig. 2C</a>). The Kaplan-Meier plot showed the most dramatic survival differences were between B1 [hereinafter designated as BEST prognosis subgroup with 96.7% 10-year survival rate (the overall 5-year survival rate was the same 96.7%)] and A2 (designated as WORST prognosis subgroup with 53.1% of survival rate in 10 years and 65.6% in 5 years). The other 4 subgroups exhibited less significant differences and were combined into a single intermediate subgroup (<a href="#pone-0000145-g002">Fig. 2D</a>). These 3 groups had very significant differences in survival (p = 0.000233).</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>This part of the study suggested that our PCT-TC signature may indeed have the potential to provide a novel prognostic model that can predict breast cancer patients' prognosis more precisely than models published heretofore. To refine this model further and to sub-stratify the prediction of outcome, we applied 6 different class prediction algorithms [Compound Covariate Predictor (CCP), Linear Discriminator Analysis (LDA), One Nearest Neighbor (1NN), Three Nearest Neighbor (3NN), Nearest Centroid (NC), and Support Vector Machine (SVM)], which compares two groups at a time, to the NKI microarray data in three independent prediction models for the prediction of patients subtype as good prognosis vs. bad prognosis (predictor 1), BEST prognosis vs. not BEST (predictor 2), and WORST prognosis vs. not WORST (predictor 3). All 6 algorithms yielded very similar results, showing the reliability and robustness of our approach. However, we felt that including all 6, followed by vote taking would overcome any weakness in any single prediction method. <a href="#pone-0000145-g003">Figure 3</a> presents a schematic overview of this strategy for the construction of prediction models and evaluation of outcomes.</p>
<div class="figure" id="pone-0000145-g003"><div class="img"><a name="pone-0000145-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Overview of strategy for the construction of prediction models and evaluation of outcomes.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Based on the unsupervised cluster analysis results, 3 independent prediction models are generated.</p>
<span>doi:10.1371/journal.pone.0000145.g003</span></div><a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>The large numbers of human genes in the Agilent microarray that were most differentially expressed between the three different pairs of prognostic subtypes (Good vs. Bad, BEST vs. not BEST, and WORST vs. not WORST) in the training set, were selected independently (two-sample t-test, p value &lt;0.001) and applied as classifiers that estimate the probability of a particular breast cancer patient belonging to one of these specific subtypes using the above-mentioned six different types of prediction methods (CCP, LDA, 1NN, 3NN, NC, and SVM) (see <a href="#pone.0000145.s010">Table S2</a>). When applied to a different group of 148 NKI breast cancer patients as a validation set, all six prediction methods produced consistent patterns. All Kaplan-Meier plots in the test set showed significant differences in survival of patients with specific subtypes when independently analyzed by these six prediction algorithms (<a href="#pone-0000145-g004">Fig. 4</a>). These results demonstrated not only a strong association of gene expression pattern with the survival of the patients, but also a robust reproducibility of these gene expression-based predictors. It is interesting to note that B3 and B6 in this figure seem to pick a set of almost perfect survivors, although these BEST groups are much smaller than that in the other prediction algorithms, presumably due to stricter selection criteria.</p>
<div class="figure" id="pone-0000145-g004"><div class="img"><a name="pone-0000145-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Kaplan-Meier plots of overall survival with NKI validation set predicted by six different prediction algorithms in 3 independent prediction models.</span></strong></p><a id="article1.body1.sec2.sec3.fig3.caption1.p1" name="article1.body1.sec2.sec3.fig3.caption1.p1"></a><p>
                     <strong>A 1–6.</strong> Group B (with good prognosis) vs. the rest (Group A, with bad prognosis) (Predictor 1). <strong>B 1–6.</strong> Subgroup B1 (BEST prognosis) vs. the rest of all (predictor 2). <strong>C 1–6.</strong> Subgroup A2 (WORST prognosis) vs. the rest of all (predictor 3). The differences between groups were significant in log rank test, with p value indicated above each plot.</p>
<span>doi:10.1371/journal.pone.0000145.g004</span></div>

<h4>Generation and validation of the four distinct subgroups of human breast cancer</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>After we applied these six prediction algorithms to each of the three class-prediction steps, each patient was assigned to one of two possible groups at each of the three stages, as follows. If samples were classified into a particular class three or more times in the six different prediction methods specified above, it was assigned to that group; otherwise this patient was assigned to the other group. Each member of the groups fit satisfactorily onto similar Kaplan-Meier plots generated with the 6 independent prediction methods (<a href="#pone-0000145-g005">Fig. 5 A1–A3</a>). When the 147 patients in the training set were combined with the 148 patients in the validation set and analyzed in this manner as a single group of 295 patients, similar results were obtained, indicating the homogenous character of the clinical outcomes in the same groups of patients in the training set and the validation set (<a href="#pone-0000145-g005">Fig. 5 B1–B3</a>).</p>
<div class="figure" id="pone-0000145-g005"><div class="img"><a name="pone-0000145-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Defining four distinct survival subgroups of human breast cancer</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>
                     <strong>A.</strong> Predicted outcomes in NKI test set (148 patients). Kaplan-Meier plot for the representative groups for 6 different prediction algorithms. If a sample was predicted to belong to the test class (black lines) 3 or more times in the 6 different prediction methods, it was assigned to that group. Otherwise that patient/sample remained in “rest of all” (red lines). There were no 3:3 ties for predictor 1. For predictors 2 and 3, ties were assigned to BEST and WORST, respectively. <strong>B.</strong> Predicted outcomes for combined NKI data sets (295 total patients). Kaplan Meier plots of overall survival of two independent groups identified with two independent analyses (unsupervised clustering in training data set and class prediction in validation set). <strong>C.</strong> Kaplan-Meier plot of four independent prognostic subtypes generated with the NKI data set. Four independent prognostic subtypes (BEST, GOOD, BAD, and WORST) are assigned as follows. Samples that fell into the Group B (good prognosis group) with predictor 1 (Good vs. Bad) but not assigned to the BEST prognosis group with predictor 2 (BEST vs. all the rest) were assigned to an intermediate group designated GOOD. Similarly, samples that did not fall into Group B (i.e., those that belonged to Group A, the bad prognosis group) with predictor 1 (Good vs. Bad) but not assigned to the WORST prognosis group in predictor 3 (WORST vs. all the rest) were assigned to an intermediate group designated BAD.</p>
<span>doi:10.1371/journal.pone.0000145.g005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Based on these three sequential stages of class prediction, samples were assigned into four independent prognostic subtypes (BEST, GOOD, BAD, and WORST) as follows. Samples that were assigned to the Good prognosis group in prediction step 1 but not assigned to the BEST prognosis group in prediction step 2 [BEST vs. not BEST] were assigned to an intermediate group designated GOOD. Similarly, samples that did not fall into Good prognosis group in prediction step 1 (Good vs. Bad) and not assigned to the WORST prognosis group in prediction step 3 (WORST vs. not WORST) were assigned to an intermediate group designated BAD. Kaplan-Meier Survival analysis and log-rank test were performed with these four independent subtypes of patients (<a href="#pone-0000145-g005">Fig. 5C</a>), and differences among them were visually apparent. These differences in patient survival were significant (p = 1.7×10<sup>−8</sup>) by the log-rank test. These findings strongly support the view that the four subgroups of human breast cancer assigned by the PCT-TC signature have distinct patterns of gene expression. These differences may reflect significant differences in the mechanism of malignant transformation.</p>


<h4>Statistical evaluation of PCT-TC-based prediction model in human breast cancer</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To achieve an independent evaluation of the statistical strength and the prognostic value of our PCT-TC signature-based prediction model in human breast cancer, we applied univariate and multivariate analysis to commonly accepted clinical and pathologic risk factors for human breast cancer progression (see <a href="#pone.0000145.s011">Table S3</a>). The BEST and WORST prognosis predicted groups showed strong association with overall survival in univariate Cox proportional hazards analysis. Multivariate analyses that included all relevant pathological variables, and the predicted subtypes revealed that BEST prognosis group prediction was significantly different from the rest of the prognosis groups, independent of ER status and clinico-pathological features of the tumors. This suggests that its predictive potential has real clinical utility, and the close examination of the genes in this signature might also provide better mechanistic understanding of breast cancer progression.</p>


<h4>Comparison of the results from our model of prognosis prediction with previously defined clinical index and gene signatures</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The predicted prognostic subtypes based on our three-stage class-prediction method showed strong association with the status of ER expression, histo-pathological grade <a href="#pone.0000145-Perou1">[14]</a>, 70-gene signature prediction <a href="#pone.0000145-vandeVijver1">[2]</a>, <a href="#pone.0000145-vantVeer1">[3]</a>, core serum-response signature <a href="#pone.0000145-Chang1">[15]</a>, <a href="#pone.0000145-Chang2">[16]</a> and ERBB2 signature <a href="#pone.0000145-Sorlie1">[17]</a> but not with age, tumor size, the number of positive lymph nodes or treatment (<a href="#pone-0000145-t001">Table 1</a>).</p>
<div class="figure" id="pone-0000145-t001"><div class="img"><a name="pone-0000145-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000145.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000145.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Association study of prognosis prediction model with clinical index using the NKI data set.</span></strong></p><span>doi:10.1371/journal.pone.0000145.t001</span></div>

<h4>Estrogen receptor status and PCT-TC stratification</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>It is well accepted that ER-positivity is an indicator of good prognosis in general <a href="#pone.0000145-Oh1">[5]</a>, <a href="#pone.0000145-Brenton1">[9]</a>, while ER-negativity usually indicates a poor prognosis, perhaps because these tumors are unlikely to respond to tamoxifen therapy. Our analysis showed that the ER-negative patients (N = 69) constituted a relatively homogenous group with a bad prognosis [47 patients (68%) in the WORST prognosis group and 13 patients (18.8%) in the BAD prognosis group]. On the other hand the ER-positive patients group (N = 226) proved to be much less homogeneous by our PCT-TC analysis, consisting of mainly 3 subtypes (only 5 patients fell into the WORST group): BEST prognosis group (69 patients, 30.5%), GOOD prognosis group (106 patients, 46.9%) and BAD prognosis group (46 patients, 20.3%). When we performed Kaplan-Meier Survival analysis with these sub-stratified groups of ER-positive patients, our prediction system gave a significant improvement of prognostic prediction (log-rank test p = 0.000739) (<a href="#pone-0000145-g006">Fig. 6A</a>). A combined survival analysis of the group of ER-negative patients with the ER-positive patients that had been stratified into 3 subtypes by our PCT-TC-based predictor, defined 4 independent subtypes with significant differences in survival estimates (log-rank test p = 5.64×10<sup>−9</sup>, <a href="#pone-0000145-g006">Fig. 6B</a>).</p>
<div class="figure" id="pone-0000145-g006"><div class="img"><a name="pone-0000145-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Prediction of breast cancer patients' outcomes based on a combination of gene expression and other criteria</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>The outcome groups previously assigned in the literature based on various criteria (ER status <a href="#pone.0000145-Oh1">[5]</a>, <a href="#pone.0000145-Brenton1">[9]</a>, pathological tumor grade <a href="#pone.0000145-Sotiriou1">[6]</a>, intrinsic-sub type <a href="#pone.0000145-Perou1">[14]</a>, <a href="#pone.0000145-Sorlie1">[17]</a> and 70-gene signature <a href="#pone.0000145-vandeVijver1">[2]</a>, <a href="#pone.0000145-vantVeer1">[3]</a>) were reassessed and further stratified using our prediction system. <strong>A.</strong> Kaplan-Meier plot of ER-positive patients stratified by 3 independent prediction steps. Estrogen receptor-positive patients in the NKI data set were further stratified into the BEST prognosis group (69 patients, 4 deaths), GOOD prognosis group (106 patients, 23 deaths), BAD prognosis group (46 patients, 17 deaths) and WORST prognosis groups (5 patients, 1 death). <strong>B.</strong> Kaplan-Meier plots of survival analysis of ER-negative groups and 3 ER-positive groups that were further stratified by our 3-step prediction analysis. <strong>C.</strong> Kaplan-Meier plot for patients with grade II tumors after further stratification with 3 independent prediction steps. 28 patients were assigned to the BEST prognosis subtype, showing a 96% 15-year survival rate (27 out of 28 patients). The 5 patients assigned to the WORST prognosis subtype had only a 20% (1 of 5 patients) 15-year survival. <strong>D.</strong> Kaplan-Meier plot for intrinsic-sub type. Survival analysis of the complete set of NKI samples (295 patients) previously assigned 5 different breast cancer intrinsic-subtypes (Luminal A, Luminal B, ERBB2, Basal, and Normal Breast-like) by nearest centroid class prediction. <strong>E.</strong> Kaplan-Meier plot for patients with intrinsic-subtypes associated with bad prognosis (Basal, ERBB2+, and Luminal B) after further stratification with our 3-step prediction analysis. This predictor revealed 11 patients that fell into the BEST prognosis group (no deaths within 15 years). <strong>F.</strong> Kaplan-Meier plot for the cell types that could not be assigned (NA) based on correlation coefficients cutting threshold of 0.1. Samples previously not assigned (NA) to any of histological cell types were stratified using our prediction system, revealing subgroups with significantly different clinical outcomes. <strong>G.</strong> Kaplan-Meier plot for the poor prognosis group in the 70-gene-based prediction after further stratification with our 3-step prediction analysis.</p>
<span>doi:10.1371/journal.pone.0000145.g006</span></div>

<h4>Pathologic tumor grading and PCT-TC stratification</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>In an attempt to improve the prognostic usefulness of histological grading of breast cancer we combined our predicting system with classic clinico-pathological phenotypes <a href="#pone.0000145-Fitzgibbons1">[18]</a> for the further stratification of patients, analogous to the approach of Soutiriou et al., <a href="#pone.0000145-Sotiriou1">[6]</a>. When we stratified the 101 NKI patients with grade-II tumors (intermediate malignancy) using our PCT-TC prediction system, 28 patients were assigned to the BEST prognosis subtype (<a href="#pone-0000145-g006">Fig. 6C</a>), showing a 96% survival rate (27 out of 28 patients survived at least 15 years). This indicates that some patients with very optimistic prognosis can be identified within the patients with “intermediate” grade-II tumors.</p>


<h4>Histological tumor-cell type and PCT-TC stratification</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>Two very early publications <a href="#pone.0000145-Perou1">[14]</a>, <a href="#pone.0000145-Sorlie1">[17]</a> used microarray-based global gene expression analysis to sub-classify breast cancer patients and defined five different breast cancer “intrinsic subtypes”; Luminal A, Luminal B, ERBB2, Basal, and Normal Breast-like. These intrinsic subtypes were associated with significant differences in clinical outcomes. Moreover, it was also reported that those 5 intrinsic-subtypes could also be recognized in a subset of the patients in the NKI reports (117 tumors from young patients), and these subclasses were associated with similar clinical outcomes <a href="#pone.0000145-Sorlie2">[19]</a>. When we performed survival analysis of the complete set of the 295-patient set of NKI samples that had been previously assigned to one of the 5 intrinsic-subtypes by nearest centroid classification <a href="#pone.0000145-Sorlie2">[19]</a>, it confirmed the expected differences in clinical outcomes (log rank test p = 3.77×10<sup>−7</sup>) (<a href="#pone-0000145-g006">Fig. 6D</a>). Applying our PCT-TC analysis, we achieved an even further stratification of 165 breast cancer patients with the histological types that had been associated with a bad prognosis, ERBB2-positive, Basal and Luminal B groups <a href="#pone.0000145-Perou1">[14]</a>, <a href="#pone.0000145-Sorlie1">[17]</a>. This analysis separated out a group of patients with a BEST prognosis signature (no deaths in 15 years for these 11 patients) (<a href="#pone-0000145-g006">Fig. 6E</a>).</p>
<a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>In addition, however, the correlation coefficients-based assignment of tumors in the initial publication Sorlie et al. <a href="#pone.0000145-Sorlie1">[17]</a> generated a large number of patients (109 patients from 295 patients) called “NA” (not assigned) even though very low correlation coefficients cutoff values (&lt;0.1) had been applied. Re-assignment of the NA samples based on our PCT-TC prediction system identified 3 subgroups of patients that had significantly different clinical outcomes (<a href="#pone-0000145-g006">Fig. 6F</a>).</p>


<h4>The 70-gene signature and PCT-TC stratification</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>In the analysis of the NKI data <a href="#pone.0000145-vandeVijver1">[2]</a>, <a href="#pone.0000145-vantVeer1">[3]</a>, a 70-gene signature was identified that had a very strong predictive power for a two-way prognosis prediction (good vs. poor). Unfortunately, our analysis showed that this signature was not as strong in prognosis-predicting power when applied to an independent data set, such as that from the Duke University <a href="#pone.0000145-Bild1">[4]</a> (<a href="#pone.0000145.s001">Fig. S1</a> ). One contributor to this loss of prediction power was the diminished numbers of predictor genes [only 18 genes (24 probes) of the 70-gene signature in the Agilent microarray set used for the NKI patients' samples were present in the Affymetrix microarray used at Duke].</p>
<a id="article1.body1.sec2.sec10.p2" name="article1.body1.sec2.sec10.p2"></a><p>We attempted a further stratification of these two groups of patients with our PCT-TC strategy from the NKI data set. Analysis of the “poor” prognosis group from the 70-gene signature analysis using our Predictor 2 (BEST vs. not BEST) culled out a BEST prognosis group [13 patients (8% of the “bad” group) with only one death in 15 years; <a href="#pone-0000145-g006">Fig. 6G</a>].</p>


<h4>Prediction of independent human breast cancer patients</h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>To further validate our prediction system, we tested our PCT-TC-derived predictors on two other groups of breast cancer patients whose survival and microarray expression data have been published but which utilized different microarray platforms [a Duke University report, using Affymetrix arrays <a href="#pone.0000145-Bild1">[4]</a>, and a University of North Carolina (UNC) report, using cDNA spotted arrays <a href="#pone.0000145-Oh1">[5]</a>].</p>
<a id="article1.body1.sec2.sec11.p2" name="article1.body1.sec2.sec11.p2"></a><p>Our independent analyses of these two data sets yielded subgroups with significant differences in survival, similar to our findings with the NKI patients. The summarized results of 6 different prediction algorithms for the Duke University testing samples showed patterns similar to the plots for NKI test set of portraits and showed significance in the log-rank tests for each of the three predictor stages: (p = 0.0124 for Good prognosis vs. Bad prognosis, p = 0.0289 for BEST prognosis vs. not BEST, and p = 0.00245 for WORST prognosis group vs. not WORST) (<a href="#pone-0000145-g007">Fig. 7 A1–A3</a>). Analysis of the UNC data with our prediction system also stratified the patients showing patterns similar to the NKI data (p = 0.0109 for Good prognosis vs. Bad prognosis, p = 0.0321 for BEST prognosis vs. not BEST, and p = 0.0205 for WORST prognosis group vs. not WORST) (<a href="#pone-0000145-g007">Fig. 7 B1–B3</a>). Combined analysis of Duke data with UNC data showed outcomes similar to those revealed in the NKI data set (p = 0.000335 for Good prognosis vs. Bad prognosis, p = 0.00219 for BEST prognosis vs. not BEST, and p = 6.25×10<sup>−5</sup> for WORST prognosis group vs. not WORST) (<a href="#pone-0000145-g007">Fig. 7 C1–C3</a>).</p>
<div class="figure" id="pone-0000145-g007"><div class="img"><a name="pone-0000145-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Prediction of independent cohorts of human breast cancer patients</span></strong></p><a id="article1.body1.sec2.sec11.fig1.caption1.p1" name="article1.body1.sec2.sec11.fig1.caption1.p1"></a><p>The results are shown as the summarized predicted outcomes determined from the results of 6 different prediction algorithms. <strong>A.</strong> Kaplan-Meier plots for the summarized predicted outcomes of Duke University patients <a href="#pone.0000145-Bild1">[4]</a>. <strong>B.</strong> Kaplan-Meier plots for the summarized predicted outcomes of UNC patients <a href="#pone.0000145-Oh1">[5]</a>. <strong>C.</strong> Kaplan-Meier plots for the combined predicted outcomes of UNC data and Duke University patients.</p>
<span>doi:10.1371/journal.pone.0000145.g007</span></div><a id="article1.body1.sec2.sec11.p3" name="article1.body1.sec2.sec11.p3"></a><p>Thus, our method of outcome prediction for two independent groups of patients provided an accurate and precise estimate of clinical outcomes that worked on microarray data sets generated using different microarray platforms and different cohorts of patients treated at different clinical institutions.</p>


<h4>Biological insights into the subtypes of human breast cancer</h4>
<a id="article1.body1.sec2.sec12.p1" name="article1.body1.sec2.sec12.p1"></a><p>A class comparison of the patients in the four different survival subtypes of the 295 NKI breast cancer patients generated by PCT-TC stratification by one-way ANOVA analysis provided a list of the genes that characterized the different prognostic groups. A total of 3307 genes showed significant differences (p&lt;1×10<sup>−8</sup>) in this analysis (<a href="#pone-0000145-g008">Fig. 8A</a>). This list is too long to be included here, but it included many genes that had been noted in previously reported analyses of gene expression in human breast cancer. General agreement between prognosis subtype and clinical predictors, ER status and histopathological grade can be visually appreciated, although some important exceptions can be seen. Regardless of ER status and histological grade, most of the patients' tumors within a single subtype showed similar gene expression patterns.</p>
<div class="figure" id="pone-0000145-g008"><div class="img"><a name="pone-0000145-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000145" data-uri="info:doi/10.1371/journal.pone.0000145.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000145.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000145.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000145.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000145.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Genes showing significant differences in expression among independent groups (BEST, GOOD, BAD and WORST)</span></strong></p><a id="article1.body1.sec2.sec12.fig1.caption1.p1" name="article1.body1.sec2.sec12.fig1.caption1.p1"></a><p>
                     <strong>A.</strong> Gene clustering of 295 NKI samples using genes selected by one-way ANOVA class comparisons. A total of 3307 genes that showed significant expression differences (p&lt;1×10<sup>−8</sup>) in a one-way ANOVA analysis were selected. ER expression status and the histo-pathological grade of each tumor sample are shown in grey-scale bars beneath the colored BEST – WORST classification bar. The key to the grey-scale designations is found beneath the heat map. <strong>B.</strong> PathwayAssist<sup>TM</sup>–generated figure showing networks of transcription factors activated by EGF and showing significantly higher expression in tumors from patients in the WORST prognosis group (indicated by red color) compared to the BEST prognosis group. <strong>C.</strong> PathwayAssist<sup>TM</sup>–generated figure showing networks of genes activated by PTGS2 (COX2) and showing significantly lower expression in tumors from patients in the WORST prognosis group (indicated by green color) compared to the BEST prognosis group.</p>
<span>doi:10.1371/journal.pone.0000145.g008</span></div><a id="article1.body1.sec2.sec12.p2" name="article1.body1.sec2.sec12.p2"></a><p>
               <a href="#pone-0000145-g008">Figures 8B and 8C</a> present genetic networks generated by PathwayAssist<sup>TM</sup> (Ariadne Genomics) analysis of these expression data. <a href="#pone-0000145-g008">Figure 8B</a> shows transcription factors most highly expressed in the WORST subgroup, all of which are activated by EGF. This finding suggests that continuous, spontaneous stimulation of EGFR and Her2 signaling may play central roles in particularly dangerous and metastatic breast cancer tumor types. Figures <a href="#pone.0000145.s002">S2</a>–<a href="#pone.0000145.s003">pone.0000145.s003</a>
               <a href="#pone.0000145.s004">pone.0000145.s004</a>
               <a href="#pone.0000145.s005">S5</a> depict the large number of genes activated by EGF, IFNG, IL4 and CCNA2 that also show increased expression in the WORST group of the NKI patients' tumors, consistent with the previous notion. Of course, these pathways have considerable cross talk or overlap in involved genes. <a href="#pone.0000145.s008">Figure S8</a> is a Venn diagram that depicts the overlapping genes that are highly expressed in patients with the WORST prognosis and are involved in the EGF, IL4 and IFNG pathways. <a href="#pone.0000145.s006">Figures S6</a> and <a href="#pone.0000145.s007">S7</a> depict the large number of genes activated in the WORST group that contribute to the activation of TNF and AKT. This might explain how the anti-apoptotic action of AKT signaling is increased in tumors that fall in the WORST prognosis group and how TNF contributes to the metastasis of the WORST prognosis group.</p>
<a id="article1.body1.sec2.sec12.p3" name="article1.body1.sec2.sec12.p3"></a><p>
               <a href="#pone-0000145-g008">Figure 8C</a> depicts genes activated by PTGS2 (COX2) that shows significantly lower expression in the WORST prognosis subgroup and higher expression in the BEST subgroup. The higher expression of IGF1, BCL2, and CCND1 in the BEST prognosis group (<a href="#pone-0000145-g008">Fig. 8C</a>) was unanticipated, as these genes are commonly considered to be tumor- promoting genes. The activation of IGF1, BCL2 and CCND1 by PTGS2 might be the prolonged legacy of chronic inflammation in early stages of tumor generation. Following the chronic inflammation stage, genetic or epigenetic changes leading to altered expression of other cytokines, growth factors and oncogenes might take over and be responsible for progression of the tumor.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Even though cancers occur in different organs and involve transformation of many cell types, most cancers share certain basic differences that separate them from normal cellular counterparts <a href="#pone.0000145-Hanahan1">[20]</a>. Tumors can evade cell death and bypass mechanisms that normally regulate the cell cycle. Constitutively activated growth factor receptors provide active proliferation signals in many tumors <a href="#pone.0000145-Hanahan1">[20]</a>, <a href="#pone.0000145-Bhowmick1">[21]</a>. Such proliferation-promoting genes commonly appear in expression signatures of cancer, and sometimes they can be used to predict clinical outcomes in tumor patients <a href="#pone.0000145-Rosenwald1">[12]</a>, <a href="#pone.0000145-Chang1">[15]</a>, <a href="#pone.0000145-Chang2">[16]</a>. However, the common accumulation of proliferation signals has made it difficult to use these features to generate highly stratified patient groups with different clinical outcomes and sensitivity to treatment protocols. Hidden beneath the strong proliferation-associated genes may be better clues to variable prognosis, e.g., proteins associated with the ability of cells to live in foreign sites, such as those represented in the PCT-TC signature.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>A closer examination of the nature of those genes that provided better stratification of patients may also improve our understanding of the tumorigenic process. In a Gene Ontology (GO) analysis of this signature (see <a href="#pone.0000145.s009">Table S1</a>), many genes involved in angiogenesis (GO:0001525), chemotaxis (GO:0006935), as well as extracellular matrix structural constituents (GO:0005201) and transmembrane receptor tyrosine kinases (GO:0004714) showed significant lower expression in tissue culture versions of PCTs. Most of the genes were previously shown its roles in tumor metastasis and have provided a basis for different prognostic outcomes of human cancer patients.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Molecular phenotypying of breast cancer by Perou et al. <a href="#pone.0000145-Perou1">[14]</a> delineated four intrinsic-subtypes of breast cancer (Luminal, Basal, ERBB2+, and Normal Epithelial Group) based on 476 genes that showed variable expression in 40 breast cancer samples compared to 20 matched samples that had received doxorubicin treatment. These molecular phenotypes of breast cancer were shown to be conserved in independent analysis of different groups of patients, and they were also associated with significant differences in clinical outcomes that correlated with the tumor cell types.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Another significant advance in the identification of prognostic factors using breast cancer microarray expression analysis was the 70-gene signature generated in axillary lymph node-negative patients by van't Veer et al. <a href="#pone.0000145-vantVeer1">[3]</a>. An extended analysis using the 70-gene signature applied to an additional 234 patients <a href="#pone.0000145-vandeVijver1">[2]</a> documented that this signature-based classification identified classes of patients with significant differences in clinical outcomes more consistently and accurately than any clinical index.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>However, it is a challenge for even the best prognostic models when they are tested on different independent patient populations <a href="#pone.0000145-Gruvberger1">[22]</a>, as most of the model parameters for the signature are generally optimized for the original data <a href="#pone.0000145-Radmacher1">[23]</a>–<a href="#pone.0000145-Simon2">[25]</a>. The source of these difficulties may lie in different sizes of patient cohorts or different compositions of patient populations, differences in microarray platforms employed (one dye system vs. two dye system, etc.), and differences in the way the microarray data were processed (normalization, scaling, etc.). Nonetheless, if the predictor used truly reflects basic genetic characteristics relevant to the mechanisms responsible for the different phenotypes (outcomes, prognosis and survival), one would expect these combinations of gene expression to be conserved among all groups of patients with given tumor types. We have shown in detail that our PCT-TC signature-based outcome-predicting model can outperform previous approaches to prognosis prediction for breast cancer patients, and probably for certain other cancers as well. Just how widely this approach can be applied will require further analysis.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Even though genetic signatures may provide the most precise prognostic prediction for individual tumor patients, it is still not widely accepted as an essential component in the choice of therapy. This might be because array-based outcome prediction has been available for only a short period of time, only small numbers of patients and samples have been used for its validation, and because no single approach has emerged with a clear consensus from the clinical/scientific community.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>We think that a possible solution to this acceptance problem is to apply multi-faceted models for the prediction of prognosis, which include well-characterized and well-accepted clinical indexes along side newly applied genetic signatures. This approach should minimize the number of false positives and false negatives from tumor staging, ER testing and the potentially subjective interpretation of histo-pathological sections. Adding a microarray analysis step offers the potential of classifying “unclassifiable” tumors or those designated “intermediate” in character. We have shown that there is a considerable heterogeneity in ER-positive populations as well as in patients with grade-II tumors. We feel that application of PCT-TC predictors could supplement existing prediction methods and yield more finely tuned, sub-stratified groups of patients based on molecular genetic similarities. These, then, could be the basis for the understanding of basic mechanisms that might be responsible for different clinical outcomes and provide a means of scientifically predicting prognosis and, if further developed, susceptibility to treatment protocols.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>In summary, we generated a novel prognosis prediction model for human breast cancer patients based on a mouse plasma cell tumor tissue culture expression signature. As it is generated based from a data set that originated in a different animal species and a different cell type, it may have side-stepped “over-fitting” problems that often plague models of human cancer patient prognostic prediction. Testing of our prediction model on 3 independent data sets of human breast cancer patients showed similar sub-stratification regardless of microarray type and on patient groups in different clinical institutions.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sample Selection and RNA Preparation</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>A total of 27 samples of RNA were prepared from transplanted mouse PCT tissues and cell lines derived from them. All solid PCT samples used for microarray hybridization had been induced in BALB/c mice by intraperitoneal injections of pristane <a href="#pone.0000145-Potter1">[26]</a> and transplanted at least once. Some had been accelerated by injection of Abelson murine leukemia virus concomitant with the pristane injection <a href="#pone.0000145-Potter2">[27]</a>. PCTs were adapted to growth in tissue culture with difficulty, often requiring cycling through pristane-primed mice before sustained growth <em>in vitro</em> could be obtained <a href="#pone.0000145-Degrassi1">[28]</a>. All tissue cultured cell lines were maintained in RPMI 1620 medium with 10% fetal bovine serum and 10 ng/ml IL-6. All mice were maintained in our conventional colony on the NIH campus under Animal Study Protocol LG-028. Total RNA was prepared from tissue culture pellets or from frozen tissues pulverized by mortar and pestle in liquid nitrogen using extraction in TRIzol (Invitrogen) followed by further purification on RNAeasy columns (Qiagen).</p>


<h4>Microarray Hybridization</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Affymetrix U74Av2 microarrays were used for the hybridization of biotin-labeled cDNA probes synthesized from 5 µg of total RNA or 1 µg poly(A)+ RNA using Superscript double–strand cDNA synthesis kit (Invitrogen), Bioarray High Efficiency RNA transcript labeling kits and Mg-catalyzed fragmentation kit (Enzo) according to the manufacturers' instruction. Microarrays were stained with phycoerythrin-streptavidin (Molecular Probes), scanned with Affymetrix GeneChip scanner and analyzed with Affymetrix Microarray Analysis Suite (MAS) version 5.0.</p>


<h4>Statistical Analysis of Microarray Data</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>BRB ArrayTools Version 3.0 (<a href="http://linus.nci.nih.gov./BRB-ArrayTools.html">http://linus.nci.nih.gov./BRB-ArrayTools​.html</a>) was used for the analysis of the MAS 5.0 data set. A log base 2 transformation was applied to the data set before arrays were normalized. Each array was normalized using median values of gene expression over the entire array (global normalization). A median array was selected as the reference array for normalization. Class comparison analysis was performed with SAM including an estimation of FDR. Cluster analysis was performed with Cluster and Treeview (<a href="http://rana.lbl.gov/EigenSoftware.htm">http://rana.lbl.gov/EigenSoftware.htm</a>). For the cluster analysis, the log base 2 transformed data were centered to the mean values of each gene's expression.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>For the generation of prediction models, samples were divided into two groups as “training set” and “validation set” in such as way that each group had a similar composition of follow-up times since the first diagnosis. Six different prediction methods were applied for the validation of subclasses: LDA, SVM, NC, 1NN, 3NN and CCP. The numbers of genes in the classifiers were optimized to minimize misclassification errors by the leave–one-out cross-validation of the data set. All statistical analyses were performed in R (version 2.0.1). The probabilities of overall survival were estimated according to the Kaplan-Meier method. Significance of difference among subclass was determined by log-rank test.</p>


<h4>Meta-Analysis of Published Data</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We worked with several previously published data sets that combined microarray analysis of gene expression in human cancer samples with clinical and laboratory data from the patients at time of tumor discovery and follow-ups and survival analyses of these patients. Five independent groups published valuable sets of data that were publicly available and could be used in our analysis. Mantle cell lymphoma <a href="#pone.0000145-Rosenwald1">[12]</a> and hepatocellular carcinoma <a href="#pone.0000145-Lee1">[7]</a> patients were the first groups studied to compare clinical outcomes with global gene expression analyses of mouse PCT-TC orthologs. However, we based most of our studies on the NKI patients in which Agilent two-color oligo microarrays were used for analysis of gene expression <a href="#pone.0000145-vandeVijver1">[2]</a>, <a href="#pone.0000145-vantVeer1">[3]</a>, because their total patient number, 295, and length of follow-up were the greatest. We also performed meta-analyses on the results generated from 170 breast cancer patients studied at Duke University using Affymetrix single-color microarrays for expression analysis <a href="#pone.0000145-Bild1">[4]</a>, and from 96 patients from the University of North Carolina (UNC) <a href="#pone.0000145-Oh1">[5]</a> which employed two-color cDNA microarrays. Both data sets provided valuable independent test sets against which we could test our approach for prediction of prognosis. Orthologous genes that were present in the Affymetrix U74Av.2 mouse microarrays and in the human microarrays employed in the previously published studies were selected by using curated mammalian orthology from The Jackson Laboratory. For the analysis of more than one independent data set of breast cancer patients (e.g., NKI data sets and the Duke University data set) each data set was normalized separately and then combined together. Each set was divided into a training set and a test set. Before integrating testing data set with training data set, the expression of each gene was standardized to mean±s.d. of 0±1 independently in both data sets.</p>


<h4>Pathway analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Once genes were identified as useful in the stratification of patients' outcomes, we attempted to gain insight into molecular mechanisms that might be involved in generating this hierarchy of patient outcomes. We employed PathwayAssist<sup>TM</sup> (version 3.0, Ariadne Genomics), as an independent pathway analysis tool to identify connections between differentially expressed genes.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>
               <strong>URL.</strong> The Jackson Laboratory: <a href="http://www.informatics.jax.org">http://www.informatics.jax.org</a>.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>BRB ArrayTools: <a href="http://linus.nci.nih.gov/BRB-">http://linus.nci.nih.gov/BRB-</a>ArrayTools.html.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>NKI data: <a href="http://microarray-pubs.stanford.edu/wound_NKI">http://microarray-pubs.stanford.edu/woun​d_NKI</a>
            </p>
<a id="article1.body1.sec4.sec5.p5" name="article1.body1.sec4.sec5.p5"></a><p>Duke Univ. data: <a href="http://data.cgt.duke.edu/oncogene.php">http://data.cgt.duke.edu/oncogene.php</a>
            </p>
<a id="article1.body1.sec4.sec5.p6" name="article1.body1.sec4.sec5.p6"></a><p>Univ. of North Carolina (UNC) data: <a href="https://genome.unc.edu/">https://genome.unc.edu/</a>
            </p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000145"></div><a name="pone.0000145.s001" id="pone.0000145.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000145.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">70-gene signature-based prognosis prediction of Duke University patients. A. Cluster analysis of patients from Duke University <a href="#pone.0000145-Bild1">[4]</a> using the 70-gene signature from the NKI data set analysis <a href="#pone.0000145-vandeVijver1">[2]</a>. Note that only 24 probes in the Duke University Affymetrix microarray platform match the 70 genes from the NKI Agilent microarray used to define the prognostic predictor in the NKI data set. B. Kaplan-Meier survival plot of the two main clusters generated in A.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000145.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(5.38 MB TIF)</p>
<a name="pone.0000145.s002" id="pone.0000145.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000145.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">PathwayAssist<sup>TM</sup>-generated figure showing genes that are activated by EGF and show significantly increased expression in the tumors from patients in the WORST prognosis group compared to those in the BEST prognosis group.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000145.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(3.18 MB TIF)</p>
<a name="pone.0000145.s003" id="pone.0000145.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000145.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">PathwayAssist<sup>TM</sup>-generated figure showing genes that are activated by interferon γ and show significantly increased expression in the tumors from patients in the WORST prognosis group compared to those in the BEST prognosis group.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000145.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(5.04 MB TIF)</p>
<a name="pone.0000145.s004" id="pone.0000145.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000145.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">PathwayAssist<sup>TM</sup>-generated figure showing genes that are activated by IL-4 and show significantly increased expression in the tumors from patients in the WORST prognosis group compared to those in the BEST prognosis group.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000145.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(4.07 MB TIF)</p>
<a name="pone.0000145.s005" id="pone.0000145.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000145.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">PathwayAssist<sup>TM</sup>-generated figure showing genes that are activated by cyclin A2 and show significantly increased expression in the tumors from patients in the WORST prognosis group compared to those in the BEST prognosis group.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000145.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(3.58 MB TIF)</p>
<a name="pone.0000145.s006" id="pone.0000145.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000145.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">PathwayAssist<sup>TM</sup>-generated figure showing genes that are activated by tumor necrosis factor and show significantly increased expression in the tumors from patients in the WORST prognosis group compared to those in the BEST prognosis group.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000145.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(4.75 MB TIF)</p>
<a name="pone.0000145.s007" id="pone.0000145.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000145.s007">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">PathwayAssist<sup>TM</sup>-generated figure showing genes that are activated by AKT1 and show significantly increased expression in the tumors from patients in the WORST prognosis group compared to those in the BEST prognosis group.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000145.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(5.10 MB TIF)</p>
<a name="pone.0000145.s008" id="pone.0000145.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000145.s008">Figure S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Venn diagram that shows genes activated in common by EGF, IFN-γ or IL-4. EGF expressed a total of 115 genes significantly more highly in tumors from the patients with the WORST prognosis group than in patients in the BEST prognosis group. Of these 115 genes, 26 are also activated by IL-4 and highly expressed in BEST tumors, and 16 of these are highly expressed in BEST patients' tumors and activated by EGF, IL-4 and TNF. The genes activated by more that one of these three factors are listed in the overlapping sectors.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000145.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(3.27 MB TIF)</p>
<a name="pone.0000145.s009" id="pone.0000145.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000145.s009">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Functional enrichment analysis of PCT-TC signature genes. The enrichment of each gene set was estimated by calculating the cumulated hypergeometric p values of each biological process provided by Gene Ontology Consortium (<a href="http://www.gene-ontology.org">www.gene-ontology.org</a>). Gene annotation according to Gene Ontology (GO) terms was downloaded from the NCBI (<a href="ftp://ftp.ncbi.nih.gov/gene">ftp://ftp.ncbi.nih.gov/gene</a>). In order to obtain representative and significantly enriched terms, the terms that belonged to a GO level higher than 2 and that included at least three genes were considered in our calculation. Statistical significance was determined with a cut-off of p&lt;0.001.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000145.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.42 MB DOC)</p>
<a name="pone.0000145.s010" id="pone.0000145.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000145.s010">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Summary of the class prediction results for NKI training set. Genes significantly different between the classes at 0.001 significance level were used for class prediction. 4160 genes were selected as a classifier for Good vs. Bad group and 1651 genes were selected as a classifier for BEST prognosis group. For the prediction of WORST prognosis group 4700 genes were selected as a classifier. Leave-one-out cross-validation method was used to compute misclassification rate based on 100 random permutations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000145.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000145.s011" id="pone.0000145.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000145.s011">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Cox univariate and multivariate analysis of risk factors for death. Parameters showing significance in the Cox proportional hazard model are shown in bold. Age is categorized into two groups based on &gt;45 years old or not. Lymph node status is categorized into two groups, one with one or more tumor cells infiltrated into the lymph node and another group with no tumor cells infiltrated into the lymph node. P value for statistical significance was calculated by log-rank test. T1T2 is categorized based on the diameter of tumor size are &gt;2 cm or not.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000145.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are deeply grateful to Siegfried Janz, M.D., Petr Hausner, M.D., Ph.D., and Charles Rabkin, M.D. who have read our manuscript critically and contributed valuable suggestions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JM EP JL JL. Performed the experiments: FZ. Analyzed the data: EP JL HW JL. Contributed reagents/materials/analysis tools: JS JM. Wrote the paper: JM EP. Other: Reviewed the statistical analysis: JS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000145-Kalluri1" id="pone.0000145-Kalluri1"></a>Kalluri R, Zeisberg M (2006) Fibroblasts in cancer. Nature Reviews Cancer  6: 392–401.  <ul class="find" data-citedArticleID="924842" data-doi="10.1038/nrc1877"><li><a href="http://dx.doi.org/10.1038/nrc1877" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fibroblasts+in+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fibroblasts+in+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000145-vandeVijver1" id="pone.0000145-vandeVijver1"></a>van de Vijver MJ, He YD, van't Veer LJ, Dai H, Hart AA, et al.  (2002) A gene-expression signature as a predictor of survival in breast cancer. N Engl J Med  347: 1999–2009.  <ul class="find" data-citedArticleID="924872" data-doi="10.1056/nejmoa021967"><li><a href="http://dx.doi.org/10.1056/nejmoa021967" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene-expression+signature+as+a+predictor+of+survival+in+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene-expression+signature+as+a+predictor+of+survival+in+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000145-vantVeer1" id="pone.0000145-vantVeer1"></a>van't Veer LJ, Dai H, van de Vijver MJ, He YD, Hart AA, et al.  (2002) Gene expression profiling predicts clinical outcome of breast cancer. Nature  415: 530–536.  <ul class="find" data-citedArticleID="924874" data-doi="10.1038/415530a"><li><a href="http://dx.doi.org/10.1038/415530a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+profiling+predicts+clinical+outcome+of+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+profiling+predicts+clinical+outcome+of+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000145-Bild1" id="pone.0000145-Bild1"></a>Bild AH, Yao G, Chang JT, Wang Q, Potti A (2006) Oncogenic pathway signatures in human cancers as a guide to targeted therapies. Nature  439: 353–357.  <ul class="find" data-citedArticleID="924824" data-doi="10.1038/nature04296"><li><a href="http://dx.doi.org/10.1038/nature04296" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oncogenic+pathway+signatures+in+human+cancers+as+a+guide+to+targeted+therapies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oncogenic+pathway+signatures+in+human+cancers+as+a+guide+to+targeted+therapies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000145-Oh1" id="pone.0000145-Oh1"></a>Oh DS, Troester MA, Usary J, Hu Z, He X, et al.  (2006) Estrogen-regulated genes predict survival in hormone receptor-positive breast cancers. J Clin Oncol  24: 1656–1664.  <ul class="find" data-citedArticleID="924848" data-doi="10.1200/jco.2005.03.2755"><li><a href="http://dx.doi.org/10.1200/jco.2005.03.2755" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estrogen-regulated+genes+predict+survival+in+hormone+receptor-positive+breast+cancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estrogen-regulated+genes+predict+survival+in+hormone+receptor-positive+breast+cancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000145-Sotiriou1" id="pone.0000145-Sotiriou1"></a>Sotiriou C, Wirapati P, Loi S, Harris A, Fox S, et al.  (2006) Gene expression profiling in breast cancer: understanding the molecular basis of histologic grade to improve prognosis. J. Natl. Cancer Inst  98: 262–272.  <ul class="find" data-citedArticleID="924870" data-doi="10.1093/jnci/djj052"><li><a href="http://dx.doi.org/10.1093/jnci/djj052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+profiling+in+breast+cancer%3A+understanding+the+molecular+basis+of+histologic+grade+to+improve+prognosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+profiling+in+breast+cancer%3A+understanding+the+molecular+basis+of+histologic+grade+to+improve+prognosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000145-Lee1" id="pone.0000145-Lee1"></a>Lee JS, Chu IS, Heo JH, Calvisi DF, Sun ZT, et al.  (2004) Classification and prediction of survival in hepatocellular carcinoma by gene expression profiling. Hepatology  40: 667–676.  <ul class="find" data-citedArticleID="924844" data-doi="10.1002/hep.20375"><li><a href="http://dx.doi.org/10.1002/hep.20375" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Classification+and+prediction+of+survival+in+hepatocellular+carcinoma+by+gene+expression+profiling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Classification+and+prediction+of+survival+in+hepatocellular+carcinoma+by+gene+expression+profiling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000145-Alizadeh1" id="pone.0000145-Alizadeh1"></a>Alizadeh A, Eisen MB, Davis E, Ma C, Lossos IS, et al.  (2000) Distinct types of diffuse large B-cell lymphoma identified by gene expression profiling. Nature  403: 503–511.  <ul class="find" data-citedArticleID="924820" data-doi="10.1038/35000501"><li><a href="http://dx.doi.org/10.1038/35000501" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+types+of+diffuse+large+B-cell+lymphoma+identified+by+gene+expression+profiling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+types+of+diffuse+large+B-cell+lymphoma+identified+by+gene+expression+profiling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000145-Brenton1" id="pone.0000145-Brenton1"></a>Brenton JD, Carey LA, Ashour AA, Caldas C (2005) Molecular classification and molecular forecasting of breast cancer: Ready for clinical implication? J. Clin Oncol  23: 7350–7360.  <ul class="find" data-citedArticleID="924826" data-doi="10.1200/jco.2005.03.3845"><li><a href="http://dx.doi.org/10.1200/jco.2005.03.3845" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+classification+and+molecular+forecasting+of+breast+cancer%3A+Ready+for+clinical+implication%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+classification+and+molecular+forecasting+of+breast+cancer%3A+Ready+for+clinical+implication%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000145-Oue1" id="pone.0000145-Oue1"></a>Oue N, Aung PP, Mitani Y, Kuniyasu H, Nakayama H, et al.  (2005) Genes involved in invasion and metastasis of gastric cancer identified by array-based hybridization and serial analysis of gene expression. Oncology  69: Suppl. 117–22. Review.  <ul class="find" data-citedArticleID="924850" data-doi="10.1159/000086627"><li><a href="http://dx.doi.org/10.1159/000086627" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+involved+in+invasion+and+metastasis+of+gastric+cancer+identified+by+array-based+hybridization+and+serial+analysis+of+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+involved+in+invasion+and+metastasis+of+gastric+cancer+identified+by+array-based+hybridization+and+serial+analysis+of+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000145-Budhu1" id="pone.0000145-Budhu1"></a>Budhu A, Forgues M, Ye Q-H, Jia H-L, He P, et al.  (2006) Prediction of venous metastases, recurrence, and prognosis in hepatocellular carcinoma based on a unique immune response signature of the liver microenvironment. Cancer Cell  10: 99–111.  <ul class="find" data-citedArticleID="924828" data-doi="10.1016/j.ccr.2006.06.016"><li><a href="http://dx.doi.org/10.1016/j.ccr.2006.06.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+venous+metastases%2C+recurrence%2C+and+prognosis+in+hepatocellular+carcinoma+based+on+a+unique+immune+response+signature+of+the+liver+microenvironment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+venous+metastases%2C+recurrence%2C+and+prognosis+in+hepatocellular+carcinoma+based+on+a+unique+immune+response+signature+of+the+liver+microenvironment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000145-Rosenwald1" id="pone.0000145-Rosenwald1"></a>Rosenwald A, Wright G, Wiestner A, Chan WC, Connors JM, et al.  (2003) The proliferation gene expression signature is a quantitative integrator of oncogenic events that predicts survival in mantle cell lymphoma. Cancer Cell  3: 185–97.  <ul class="find" data-citedArticleID="924860" data-doi="10.1016/s1535-6108(03)00028-x"><li><a href="http://dx.doi.org/10.1016/s1535-6108(03)00028-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+proliferation+gene+expression+signature+is+a+quantitative+integrator+of+oncogenic+events+that+predicts+survival+in+mantle+cell+lymphoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+proliferation+gene+expression+signature+is+a+quantitative+integrator+of+oncogenic+events+that+predicts+survival+in+mantle+cell+lymphoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000145-Lee2" id="pone.0000145-Lee2"></a>Lee JS, Heo JH, Libbrecht L, Chu IS, Kaposi-Novak P, et al.  (2006) A novel prognostic subtype of human hepatocellular carcinoma derived from hepatic progenitor cells. Nature Medicine  12: 410–416.  <ul class="find" data-citedArticleID="924846" data-doi="10.1038/nm1377"><li><a href="http://dx.doi.org/10.1038/nm1377" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+prognostic+subtype+of+human+hepatocellular+carcinoma+derived+from+hepatic+progenitor+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+prognostic+subtype+of+human+hepatocellular+carcinoma+derived+from+hepatic+progenitor+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000145-Perou1" id="pone.0000145-Perou1"></a>Perou CM, Sorlie T, Eisen MB, van de Rijn M, Jeffrey SS, et al.  (2000) Molecular portraits of human breast tumours. Nature  406: 747–752.  <ul class="find" data-citedArticleID="924852" data-doi="10.1038/35021093"><li><a href="http://dx.doi.org/10.1038/35021093" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+portraits+of+human+breast+tumours." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+portraits+of+human+breast+tumours.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000145-Chang1" id="pone.0000145-Chang1"></a>Chang HY, Nuyten DSA, Sneddon JB, Hastie T, Tibshirani R, et al.  (2005) Robustness, scalability and integration of a wound response gene expression signature in predicting breast cancer survival. Proc Natl Acad Sci USA  102: 3738–3743.  <ul class="find" data-citedArticleID="924830" data-doi="10.1073/pnas.0409462102"><li><a href="http://dx.doi.org/10.1073/pnas.0409462102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robustness%2C+scalability+and+integration+of+a+wound+response+gene+expression+signature+in+predicting+breast+cancer+survival." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robustness%2C+scalability+and+integration+of+a+wound+response+gene+expression+signature+in+predicting+breast+cancer+survival.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000145-Chang2" id="pone.0000145-Chang2"></a>Chang HY, Sneddon JB, Alizadeh AA, Sood R, West RB, et al.  (2004) Gene expression signature of fibroblast serum response predicts human cancer progression: similarities between tumors and wounds. PLoS Biology  2: E7.  <ul class="find" data-citedArticleID="924832" data-doi="10.1371/journal.pbio.0020007"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+signature+of+fibroblast+serum+response+predicts+human+cancer+progression%3A+similarities+between+tumors+and+wounds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+signature+of+fibroblast+serum+response+predicts+human+cancer+progression%3A+similarities+between+tumors+and+wounds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000145-Sorlie1" id="pone.0000145-Sorlie1"></a>Sorlie T, Perou CM, Tibshirani R, Aas T, Geisler S, et al.  (2001) Gene expression patterns of breast carcinomas distinguish tumor subclasses with clinical implications. Proc Natl Acad Sci U S A  98: 10869–10874.  <ul class="find" data-citedArticleID="924866" data-doi="10.1073/pnas.191367098"><li><a href="http://dx.doi.org/10.1073/pnas.191367098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+patterns+of+breast+carcinomas+distinguish+tumor+subclasses+with+clinical+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+patterns+of+breast+carcinomas+distinguish+tumor+subclasses+with+clinical+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000145-Fitzgibbons1" id="pone.0000145-Fitzgibbons1"></a>Fitzgibbons PL, Page DL, Weaver D, Thor AD, Allred DC, et al.  (2000) Revision of the American Joint Committee on Cancer staging system for breast cancer. Arch. Path. Lab. Med.  124: 966–978.  <ul class="find" data-citedArticleID="924836"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Revision+of+the+American+Joint+Committee+on+Cancer+staging+system+for+breast+cancer.&amp;auth=&amp;atitle=Revision+of+the+American+Joint+Committee+on+Cancer+staging+system+for+breast+cancer." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Revision+of+the+American+Joint+Committee+on+Cancer+staging+system+for+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Revision+of+the+American+Joint+Committee+on+Cancer+staging+system+for+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000145-Sorlie2" id="pone.0000145-Sorlie2"></a>Sorlie T, Tibshirani R, Parker J, Hastie T, Marron JS, et al.  (2003) Repeated observation of breast tumor subtypes in independent gene expression data sets. Proc Natl Acad Sci U S A  100: 8418–8423.  <ul class="find" data-citedArticleID="924868" data-doi="10.1073/pnas.0932692100"><li><a href="http://dx.doi.org/10.1073/pnas.0932692100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repeated+observation+of+breast+tumor+subtypes+in+independent+gene+expression+data+sets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repeated+observation+of+breast+tumor+subtypes+in+independent+gene+expression+data+sets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000145-Hanahan1" id="pone.0000145-Hanahan1"></a>Hanahan D, Weinberg RA (2000) The hallmarks of cancer. Cell  100: 57–70.  <ul class="find" data-citedArticleID="924840" data-doi="10.1016/s0092-8674(00)81683-9"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81683-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+hallmarks+of+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+hallmarks+of+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000145-Bhowmick1" id="pone.0000145-Bhowmick1"></a>Bhowmick NA, Neilson EG, Moses HL (2004) Stromal fibroblasts in cancer initiation and progression. Nature  432: 332–337.  <ul class="find" data-citedArticleID="924822" data-doi="10.1038/nature03096"><li><a href="http://dx.doi.org/10.1038/nature03096" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stromal+fibroblasts+in+cancer+initiation+and+progression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stromal+fibroblasts+in+cancer+initiation+and+progression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000145-Gruvberger1" id="pone.0000145-Gruvberger1"></a>Gruvberger SK, Ringner M, Eden P, Borg A, Ferno M, et al.  (2003) Expression profiling to predict outcome in breast cancer: the influence of sample selection. Breast Cancer Res.  5: 23–26.  <ul class="find" data-citedArticleID="924838" data-doi="10.1186/bcr548"><li><a href="http://dx.doi.org/10.1186/bcr548" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+profiling+to+predict+outcome+in+breast+cancer%3A+the+influence+of+sample+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+profiling+to+predict+outcome+in+breast+cancer%3A+the+influence+of+sample+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000145-Radmacher1" id="pone.0000145-Radmacher1"></a>Radmacher MD, McShane LM, Simon R (2002) A paradigm for class prediction using gene expression profiles. J Computational Biol  9: 505–511.  <ul class="find" data-citedArticleID="924858" data-doi="10.1089/106652702760138592"><li><a href="http://dx.doi.org/10.1089/106652702760138592" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+paradigm+for+class+prediction+using+gene+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+paradigm+for+class+prediction+using+gene+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000145-Simon1" id="pone.0000145-Simon1"></a>Simon R, Radmacher MD, Dobbin K, McShane LM (2003) Pitfalls in the use of DNA Microarray Data for Diagnostic and Prognostic Classification. J Natl Can Inst  95: 14–18.  <ul class="find" data-citedArticleID="924862" data-doi="10.1093/jnci/95.1.14"><li><a href="http://dx.doi.org/10.1093/jnci/95.1.14" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pitfalls+in+the+use+of+DNA+Microarray+Data+for+Diagnostic+and+Prognostic+Classification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pitfalls+in+the+use+of+DNA+Microarray+Data+for+Diagnostic+and+Prognostic+Classification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000145-Simon2" id="pone.0000145-Simon2"></a>Simon R (2005) Roadmap for developing and validating therapeutically relevant genomic classifiers. J Clin Oncol  23: 7332–7341.  <ul class="find" data-citedArticleID="924864" data-doi="10.1200/jco.2005.02.8712"><li><a href="http://dx.doi.org/10.1200/jco.2005.02.8712" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Roadmap+for+developing+and+validating+therapeutically+relevant+genomic+classifiers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Roadmap+for+developing+and+validating+therapeutically+relevant+genomic+classifiers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000145-Potter1" id="pone.0000145-Potter1"></a>Potter M (2003) Neoplastic development in plasma cells. Immunol Rev  194: 177–195.  <ul class="find" data-citedArticleID="924854" data-doi="10.1034/j.1600-065x.2003.00061.x"><li><a href="http://dx.doi.org/10.1034/j.1600-065x.2003.00061.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neoplastic+development+in+plasma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neoplastic+development+in+plasma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000145-Potter2" id="pone.0000145-Potter2"></a>Potter M, Sklar MD, Rowe WP (1973) Rapid viral induction of plasmacytomas in pristane primed BALB/c mice. Science  182: 592–594.  <ul class="find" data-citedArticleID="924856" data-doi="10.1126/science.182.4112.592"><li><a href="http://dx.doi.org/10.1126/science.182.4112.592" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+viral+induction+of+plasmacytomas+in+pristane+primed+BALB%2Fc+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+viral+induction+of+plasmacytomas+in+pristane+primed+BALB%2Fc+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000145-Degrassi1" id="pone.0000145-Degrassi1"></a>Degrassi A, Hilbert DM, Rudikoff S, Anderson AO, Potter M, et al.  (1993) In vitro culture of primary plasmacytomas requires stromal cell feeder layers. Proc Natl Acad Sci USA  90: 2060–2064.  <ul class="find" data-citedArticleID="924834" data-doi="10.1073/pnas.90.5.2060"><li><a href="http://dx.doi.org/10.1073/pnas.90.5.2060" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+culture+of+primary+plasmacytomas+requires+stromal+cell+feeder+layers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+culture+of+primary+plasmacytomas+requires+stromal+cell+feeder+layers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.XML" value="152695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.PDF" value="1033246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g001.PNG_L" value="1447793"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g001.PNG_M" value="116259"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g001.PNG_S" value="5934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g001.TIF" value="2091418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g001.PNG_I" value="70339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g002.PNG_L" value="7578949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g002.PNG_M" value="242806"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g002.PNG_S" value="6086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g002.TIF" value="10821666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g002.PNG_I" value="97412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g003.PNG_L" value="848932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g003.PNG_M" value="97882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g003.PNG_S" value="3025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g003.TIF" value="1225398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g003.PNG_I" value="42184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g004.PNG_L" value="1032492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g004.PNG_M" value="108458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g004.PNG_S" value="5339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g004.TIF" value="1508996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g004.PNG_I" value="91044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g005.PNG_L" value="905966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g005.PNG_M" value="87411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g005.PNG_S" value="4167"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g005.TIF" value="1459588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g005.PNG_I" value="51281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.t001.PNG_L" value="303816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.t001.PNG_M" value="30987"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.t001.PNG_S" value="3358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.t001.TIF" value="21163156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.t001.PNG_I" value="19713"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g006.PNG_L" value="1727824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g006.PNG_M" value="74471"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g006.PNG_S" value="5127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g006.TIF" value="2778922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g006.PNG_I" value="71241"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g007.PNG_L" value="979318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g007.PNG_M" value="122238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g007.PNG_S" value="3480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g007.TIF" value="1422022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g007.PNG_I" value="51601"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g008.PNG_L" value="2358857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g008.PNG_M" value="149421"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g008.PNG_S" value="6742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g008.TIF" value="3430112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.g008.PNG_I" value="84598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.s001.TIF" value="2179828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.s002.TIF" value="3103820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.s003.TIF" value="4961542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.s004.TIF" value="4030238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.s005.TIF" value="2801312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.s006.TIF" value="4683540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.s007.TIF" value="4026981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.s008.TIF" value="1547790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.s009.DOC" value="423936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.s010.DOC" value="54784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000145.s011.DOC" value="53248"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000145&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000145" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000145&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000145&volume=&issue=&title=Heterologous Tissue Culture Expression Signature Predicts Human Breast Cancer Prognosis&author_name=Eun%20Sung%20Park%2C%20Ju-Seog%20Lee%2C%20Hyun%20Goo%20Woo%2C%20Fenghuang%20Zhan%2C%20Joanna%20H.%20Shih%2C%20John%20D.%20Shaughnessy%20Jr.%2C%20J.%20Frederic%20Mushinski&start_page=1&end_page=16" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000145" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000145" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000145" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000145&amp;t=Heterologous%20Tissue%20Culture%20Expression%20Signature%20Predicts%20Human%20Breast%20Cancer%20Prognosis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000145&title=Heterologous%20Tissue%20Culture%20Expression%20Signature%20Predicts%20Human%20Breast%20Cancer%20Prognosis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000145&amp;title=Heterologous%20Tissue%20Culture%20Expression%20Signature%20Predicts%20Human%20Breast%20Cancer%20Prognosis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000145" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000145&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Heterologous Tissue Culture Expression Signature Predicts Human Breast Cancer Prognosis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000145';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Heterologous%20Tissue%20Culture%20Expression%20Signature%20Predicts%20Human%20Breast%20Cancer%20Prognosis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000145" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000145" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Breast+cancer%22" title="Search for articles in the subject area:'Breast cancer'"><div class="flagText">Breast cancer</div></a>
              <div data-categoryid="18763" data-articleid="23128"
                   data-categoryname="Breast cancer"
                   class="flagImage" title="Flag 'Breast cancer' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Forecasting%22" title="Search for articles in the subject area:'Forecasting'"><div class="flagText">Forecasting</div></a>
              <div data-categoryid="34789" data-articleid="23128"
                   data-categoryname="Forecasting"
                   class="flagImage" title="Flag 'Forecasting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23128"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="23128"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plasma+cells%22" title="Search for articles in the subject area:'Plasma cells'"><div class="flagText">Plasma cells</div></a>
              <div data-categoryid="41097" data-articleid="23128"
                   data-categoryname="Plasma cells"
                   class="flagImage" title="Flag 'Plasma cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Prognosis%22" title="Search for articles in the subject area:'Prognosis'"><div class="flagText">Prognosis</div></a>
              <div data-categoryid="24099" data-articleid="23128"
                   data-categoryname="Prognosis"
                   class="flagImage" title="Flag 'Prognosis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Survival+analysis%22" title="Search for articles in the subject area:'Survival analysis'"><div class="flagText">Survival analysis</div></a>
              <div data-categoryid="37261" data-articleid="23128"
                   data-categoryname="Survival analysis"
                   class="flagImage" title="Flag 'Survival analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Tissue+cultures%22" title="Search for articles in the subject area:'Tissue cultures'"><div class="flagText">Tissue cultures</div></a>
              <div data-categoryid="25067" data-articleid="23128"
                   data-categoryname="Tissue cultures"
                   class="flagImage" title="Flag 'Tissue cultures' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5343'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3526'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1877&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
