

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Strategy of Transcription Regulation in the Budding Yeast</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000250"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000250" />

    <meta name="citation_title" content="Strategy of Transcription Regulation in the Budding Yeast"/>
    <meta itemprop="name" content="Strategy of Transcription Regulation in the Budding Yeast"/>

      <meta name="citation_author" content="Sagi Levy"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics and Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Jan Ihmels"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics and Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Cellular and Molecular Pharmacology, University of California, San Francisco, California, United States of America"/>
      <meta name="citation_author" content="Miri Carmi"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics and Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Adina Weinberger"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics and Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Gilgi Friedlander"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics and Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Naama Barkai"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics and Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel"/>

    <meta name="citation_date" content="2007/2/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000250.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e250"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Regulation of ribosome biogenesis by the rapamycin-sensitive TOR-signaling pathway in Saccharomyces cerevisiae.; citation_author=T Powers; citation_author=P Walter; citation_journal_title=Mol Biol Cell; citation_volume=10; citation_number=1; citation_pages=987-1000; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=TOR regulates ribosomal protein gene expression via PKA and the Forkhead transcription factor FHL1.; citation_author=DE Martin; citation_author=A Soulard; citation_author=MN Hall; citation_journal_title=Cell; citation_volume=119; citation_number=2; citation_pages=969-979; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Tor and cyclic AMP-protein kinase A: two parallel pathways regulating expression of genes required for cell growth.; citation_author=SA Zurita-Martinez; citation_author=ME Cardenas; citation_journal_title=Eukaryot Cell; citation_volume=4; citation_number=3; citation_pages=63-71; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The economics of ribosome biosynthesis in yeast.; citation_author=JR Warner; citation_journal_title=Trends Biochem Sci; citation_volume=24; citation_number=4; citation_pages=437-440; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Making ribosomes.; citation_author=A Fatica; citation_author=D Tollervey; citation_journal_title=Curr Opin Cell Biol; citation_volume=14; citation_number=5; citation_pages=313-318; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=At the crossroads of growth control; making ribosomal RNA.; citation_author=T Moss; citation_journal_title=Curr Opin Genet Dev; citation_volume=14; citation_number=6; citation_pages=210-217; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A dynamic transcriptional network communicates growth potential to ribosome synthesis and critical cell size.; citation_author=P Jorgensen; citation_author=I Rupes; citation_author=JR Sharom; citation_author=L Schneper; citation_author=JR Broach; citation_journal_title=Genes Dev; citation_volume=18; citation_number=7; citation_pages=2491-2505; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Does the ribosome translate cancer?; citation_author=D Ruggero; citation_author=PP Pandolfi; citation_journal_title=Nat Rev Cancer; citation_volume=3; citation_number=8; citation_pages=179-192; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Overexpression of ribosomal protein L15 is associated with cell proliferation in gastric cancer.; citation_author=H Wang; citation_author=LN Zhao; citation_author=KZ Li; citation_author=R Ling; citation_author=XJ Li; citation_journal_title=BMC Cancer; citation_volume=6; citation_number=9; citation_pages=91; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=rRNA transcription and growth rate-dependent regulation of ribosome synthesis in Escherichia coli.; citation_author=RL Gourse; citation_author=T Gaal; citation_author=MS Bartlett; citation_author=JA Appleman; citation_author=W Ross; citation_journal_title=Annu Rev Microbiol; citation_volume=50; citation_number=10; citation_pages=645-677; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Transcription regulation by initiating NTP concentration: rRNA synthesis in bacteria.; citation_author=T Gaal; citation_author=MS Bartlett; citation_author=W Ross; citation_author=CL Turnbough; citation_author=RL Gourse; citation_journal_title=Science; citation_volume=278; citation_number=11; citation_pages=2092-2097; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=rRNA transcription in Escherichia coli.; citation_author=BJ Paul; citation_author=W Ross; citation_author=T Gaal; citation_author=RL Gourse; citation_journal_title=Annu Rev Genet; citation_volume=38; citation_number=12; citation_pages=749-770; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=NTP-sensing by rRNA promoters in Escherichia coli is direct.; citation_author=DA Schneider; citation_author=T Gaal; citation_author=RL Gourse; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=13; citation_pages=8602-8607; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Escherichia coli Fis promoter is regulated by changes in the levels of its transcription initiation nucleotide CTP.; citation_author=KA Walker; citation_author=P Mallik; citation_author=TS Pratt; citation_author=R Osuna; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=14; citation_pages=50818-50828; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Revisiting the stringent response, ppGpp and starvation signaling.; citation_author=D Chatterji; citation_author=AK Ojha; citation_journal_title=Curr Opin Microbiol; citation_volume=4; citation_number=15; citation_pages=160-165; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=ppGpp: stringent response and survival.; citation_author=V Jain; citation_author=M Kumar; citation_author=D Chatterji; citation_journal_title=J Microbiol; citation_volume=44; citation_number=16; citation_pages=1-10; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dissection of the mechanism for the stringent factor RelA.; citation_author=TM Wendrich; citation_author=G Blaha; citation_author=DN Wilson; citation_author=MA Marahiel; citation_author=KH Nierhaus; citation_journal_title=Mol Cell; citation_volume=10; citation_number=17; citation_pages=779-788; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Growth-rate regulated genes have profound impact on interpretation of transcriptome profiling in Saccharomyces cerevisiae.; citation_author=B Regenberg; citation_author=T Grotkjaer; citation_author=O Winther; citation_author=A Fausboll; citation_author=M Akesson; citation_journal_title=Genome Biol; citation_volume=7; citation_number=18; citation_pages=R107; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Functional discovery via a compendium of expression profiles.; citation_author=TR Hughes; citation_author=MJ Marton; citation_author=AR Jones; citation_author=CJ Roberts; citation_author=R Stoughton; citation_journal_title=Cell; citation_volume=102; citation_number=19; citation_pages=109-126; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Revealing modular organization in the yeast transcriptional network.; citation_author=J Ihmels; citation_author=G Friedlander; citation_author=S Bergmann; citation_author=O Sarig; citation_author=Y Ziv; citation_journal_title=Nat Genet; citation_volume=31; citation_number=20; citation_pages=370-377; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Iterative signature algorithm for the analysis of large-scale gene expression data.; citation_author=S Bergmann; citation_author=J Ihmels; citation_author=N Barkai; citation_journal_title=Phys Rev E Stat Nonlin Soft Matter Phys; citation_volume=67; citation_number=21; citation_pages=031902; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Defining transcription modules using large-scale gene expression data.; citation_author=J Ihmels; citation_author=S Bergmann; citation_author=N Barkai; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=22; citation_pages=1993-2003; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rewiring of the yeast transcriptional network through the evolution of motif usage.; citation_author=J Ihmels; citation_author=S Bergmann; citation_author=M Gerami-Nejad; citation_author=I Yanai; citation_author=M McClellan; citation_journal_title=Science; citation_volume=309; citation_number=23; citation_pages=938-940; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Remodeling of yeast genome expression in response to environmental changes.; citation_author=HC Causton; citation_author=B Ren; citation_author=SS Koh; citation_author=CT Harbison; citation_author=E Kanin; citation_journal_title=Mol Biol Cell; citation_volume=12; citation_number=24; citation_pages=323-337; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genomic expression programs in the response of yeast cells to environmental changes.; citation_author=AP Gasch; citation_author=PT Spellman; citation_author=CM Kao; citation_author=O Carmel-Harel; citation_author=MB Eisen; citation_journal_title=Mol Biol Cell; citation_volume=11; citation_number=25; citation_pages=4241-4257; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Systematic identification of pathways that couple cell growth and division in yeast.; citation_author=P Jorgensen; citation_author=JL Nishikawa; citation_author=BJ Breitkreutz; citation_author=M Tyers; citation_journal_title=Science; citation_volume=297; citation_number=26; citation_pages=395-400; citation_date=2002; " />
      <meta name="citation_reference" content="citation_author=LA Segel; citation_number=27; citation_pages=xx, 300 p; citation_date=1984; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Widespread aneuploidy revealed by DNA microarray expression profiling.; citation_author=TR Hughes; citation_author=CJ Roberts; citation_author=H Dai; citation_author=AR Jones; citation_author=MR Meyer; citation_journal_title=Nat Genet; citation_volume=25; citation_number=28; citation_pages=333-337; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Optimality and evolutionary tuning of the expression level of a protein.; citation_author=E Dekel; citation_author=U Alon; citation_journal_title=Nature; citation_volume=436; citation_number=29; citation_pages=588-592; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Control of rRNA expression by small molecules is dynamic and nonredundant.; citation_author=HD Murray; citation_author=DA Schneider; citation_author=RL Gourse; citation_journal_title=Mol Cell; citation_volume=12; citation_number=30; citation_pages=125-134; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Nutritional homeostasis in batch and steady-state culture of yeast.; citation_author=AJ Saldanha; citation_author=MJ Brauer; citation_author=D Botstein; citation_journal_title=Mol Biol Cell; citation_volume=15; citation_number=31; citation_pages=4089-4104; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000250" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Strategy of Transcription Regulation in the Budding Yeast"/>
    <meta name="twitter:description" content="Cells must adjust their gene expression in order to compete in a constantly changing environment. Two alternative strategies could in principle ensure optimal coordination of gene expression with physiological requirements. First, characters of the internal physiological state, such as growth rate, metabolite levels, or energy availability, could be feedback to tune gene expression. Second, internal needs could be inferred from the external environment, using evolutionary-tuned signaling pathways. Coordination of ribosomal biogenesis with the requirement for protein synthesis is of particular importance, since cells devote a large fraction of their biosynthetic capacity for ribosomal biogenesis. To define the relative contribution of internal vs. external sensing to the regulation of ribosomal biogenesis gene expression in yeast, we subjected S. cerevisiae cells to conditions which decoupled the actual vs. environmentally-expected growth rate. Gene expression followed the environmental signal according to the expected, but not the actual, growth rate. Simultaneous monitoring of gene expression and growth rate in continuous cultures further confirmed that ribosome biogenesis genes responded rapidly to changes in the environments but were oblivious to longer-term changes in growth rate. Our results suggest that the capacity to anticipate and prepare for environmentally-mediated changes in cell growth presented a major selection force during yeast evolution."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000250.g004"/>

  <meta property="og:title" content="Strategy of Transcription Regulation in the Budding Yeast" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000250" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1436'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6102'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6901&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000250">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000250" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000250&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Strategy of Transcription Regulation in the Budding Yeast
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sagi Levy, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Genetics and Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jan Ihmels, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Molecular Genetics and Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel, 
                  Howard Hughes Medical Institute, Department of Cellular and Molecular Pharmacology, University of California, San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Miri Carmi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Genetics and Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Adina Weinberger, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Genetics and Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gilgi Friedlander, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Genetics and Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Naama Barkai
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:naama.barkai@weizmann.ac.il">naama.barkai@weizmann.ac.il</a></p>

                <p>Affiliation:
                  Department of Molecular Genetics and Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000250</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000250" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000250" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000250" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000250" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000250-g001" data-doi="info:doi/10.1371/journal.pone.0000250" data-uri="info:doi/10.1371/journal.pone.0000250.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000250-g002" data-doi="info:doi/10.1371/journal.pone.0000250" data-uri="info:doi/10.1371/journal.pone.0000250.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000250-g003" data-doi="info:doi/10.1371/journal.pone.0000250" data-uri="info:doi/10.1371/journal.pone.0000250.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000250-t001" data-doi="info:doi/10.1371/journal.pone.0000250" data-uri="info:doi/10.1371/journal.pone.0000250.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000250-g004" data-doi="info:doi/10.1371/journal.pone.0000250" data-uri="info:doi/10.1371/journal.pone.0000250.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000250">Reader Comments (4)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000250" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Cells must adjust their gene expression in order to compete in a constantly changing environment. Two alternative strategies could in principle ensure optimal coordination of gene expression with physiological requirements. First, characters of the internal physiological state, such as growth rate, metabolite levels, or energy availability, could be feedback to tune gene expression. Second, internal needs could be inferred from the external environment, using evolutionary-tuned signaling pathways. Coordination of ribosomal biogenesis with the requirement for protein synthesis is of particular importance, since cells devote a large fraction of their biosynthetic capacity for ribosomal biogenesis. To define the relative contribution of internal vs. external sensing to the regulation of ribosomal biogenesis gene expression in yeast, we subjected <em>S. cerevisiae</em> cells to conditions which decoupled the actual vs. environmentally-expected growth rate. Gene expression followed the environmental signal according to the expected, but not the actual, growth rate. Simultaneous monitoring of gene expression and growth rate in continuous cultures further confirmed that ribosome biogenesis genes responded rapidly to changes in the environments but were oblivious to longer-term changes in growth rate. Our results suggest that the capacity to anticipate and prepare for environmentally-mediated changes in cell growth presented a major selection force during yeast evolution.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Levy S, Ihmels J, Carmi M, Weinberger A, Friedlander G, et al.  (2007) Strategy of Transcription Regulation in the Budding Yeast. PLoS ONE 2(2):
          e250.
            doi:10.1371/journal.pone.0000250</p><p><strong>Academic Editor: </strong>Raya Khanin, University of Glasgow, United Kingdom</p><p><strong>Received:</strong> January 5, 2007; <strong>Accepted:</strong> January 30, 2007; <strong>Published:</strong> February 28, 2007</p><p><strong>Copyright:</strong> © 2007 Levy et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a grant from the Kahn Fund for Systems Biology at the Weizmann Institute of Science, and by the Tauber Fund through the Foundations of Cognition Initiative.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Cellular functionality is tightly coupled to the external environment. The type of nutrients available defines the internal metabolic flow, while their abundance often limits the rate of biomass production and energy available for growth. An abundance of toxins impede upon various aspects of cellular machinery, including metabolic capacity, protein stability or DNA integrity. Over evolutionary time scales, cells may encounter virtually endless environmental states at widely different frequencies. Maintaining optimal functionality in the presence of such external variability is a central evolutionary constraint.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Gene expression plays a central role in the adaptation to changing conditions. Studies in the budding yeast <em>S. cerevisiae</em>, for example, have shown that cellular transcription program is dramatically modified by changes in nutrient availability, growth conditions, temperature, and a variety of other environmental condition tested.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>How do cells coordinate their gene expression with varying environments? One strategy is to use feedback mechanisms, which directly link gene expression with internal needs. Within such strategy, internal variables, such as the rate of biomass production or the internal pools of nutrients or energy, feedbacks to properly tune gene expression with the corresponding functional needs. Cells then respond directly to the relevant functional parameter that needs to be monitored, regardless of the specific external perturbation that may have altered this parameter. The primary advantage of such internal feedback strategy is the capacity to ensure optimality of response under a wide diversity of external conditions. It may limit, however, the speed of the response since environmental changes will alter internal characters (<em>e.g.</em> growth rate) only after a delay (<em>e.g.</em> when the intracellular pool of nutrient is diminished).</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>An alternative strategy for coordinating gene expression with physiological requirements is to infer the expected physiological state from the external environment. This can be done using evolutionary-tuned signaling pathways. For example, in yeast, the TOR and the PKA pathways sense the source and availability of carbon and nitrogen and regulate the expression levels of multiple gene groups <a href="#pone.0000250-Powers1">[1]</a>–<a href="#pone.0000250-ZuritaMartinez1">[3]</a>. In principle, the information received by those signaling pathways could coordinate gene expression with the expected growth rate at the specific level of nutrients available. Such a strategy can be used to optimize the speed of the response. It is likely, however, to limit the range of environments that can be properly recognized. In particular, it will hinder optimal response to newly encountered situations.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The need to coordinate gene expression with internal state is best exemplified in the case of ribosomal biogenesis. In rapidly growing yeast cells, the synthesis of ribosomes accounts for the cell's single largest expenditure of biosynthesis energy. With the need to produce ~2000 ribosomes every minute, 60% of total transcription is devoted to ribosomal RNA. Similarly, 50% of RNA polymerase II transcription, as well as 90% of mRNA splicing, are devoted to ribosomal protein <a href="#pone.0000250-Warner1">[4]</a>. Over 150 non-ribosomal genes are involved in various aspects of ribosomal RNA processing or transport <a href="#pone.0000250-Fatica1">[5]</a>. Tight coordination of ribosomal biosynthesis with the requirement for protein translation is thus needed to maintain efficient utilization of biosynthesis resources. Indeed, the number of ribosomes in yeast cells appears to be linked to cell size and growth rate <a href="#pone.0000250-Moss1">[6]</a>, <a href="#pone.0000250-Jorgensen1">[7]</a>. Similarly, tumor cells often express high levels of ribosome and other translation related factors, in accordance with their elevated growth rates <a href="#pone.0000250-Ruggero1">[8]</a>, <a href="#pone.0000250-Wang1">[9]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>In bacteria, the rate of ribosome synthesis increases approximately with the square of the growth rate <a href="#pone.0000250-Gourse1">[10]</a>. Underlying this growth-dependent control are well-studied examples of regulation by an internal-feedback. The purine nucleotide (ATP and GTP), whose concentration reflect the nutritional state as well as the translational activity of the cell, play a key role in this regulation. In fact, purine NTP levels directly regulates rRNA transcription <a href="#pone.0000250-Gaal1">[11]</a>–<a href="#pone.0000250-Walker1">[14]</a>, and thus determine the rate of ribosomal biogenesis. An additional layer of regulation by internal feedback is provided by the stringent response, which is induced by uncharged tRNA. Uncharged tRNAs accumulate when the internal level of amino-acids is not sufficiently to support the rate of protein synthesis. In those instances, the stringent response feedbacks to repress the transcription of genes associated with the translational apparatus, including tRNAs, rRNAs, ribosomal proteins, translational factors, and synthetases <a href="#pone.0000250-Chatterji1">[15]</a>–<a href="#pone.0000250-Wendrich1">[17]</a>.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Growth-rate dependent control of gene expression was implied also in the regulation of gene expression in the budding yeast <a href="#pone.0000250-Regenberg1">[18]</a>. The dependence of gene expression on growth rate was characterized most comprehensively using continuous chemostat cultures growing at steady state <a href="#pone.0000250-Regenberg1">[18]</a>. A large multitude of genes exhibited an expression pattern that was strongly correlated with growth rate. In particular, ribosomal biogenesis gene expression increased with increasing growth rate, whereas the expression of stress-related genes decreased. Based on the genomic distribution of growth-correlated genes, it was suggested that growth-rate dependence is achieved through replication-mediated changes in chromatin modification <a href="#pone.0000250-Regenberg1">[18]</a>. In the context of the internal vs. external regulatory strategies discussed above, however, the interpretation of the observed correlations is difficult, since changes in growth rate necessarily involved change in the amount of the limiting factor (glucose) in the medium. Gene expression was measured at steady state, and, at least formally, it may be that expression levels were tuned by the external glucose concentration, according to the evolutionary-expected growth rate, rather than as feedback by the growth-rate itself.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>More generally, since under most conditions the actual and environmentally-expected states are compatible, it is difficult to discern whether gene expression is regulated predominantly by an internal feedback-mechanism, or whether it is tuned by an environmental signal based on its expected influence on the internal state. To try and overcome this limitation, we undertook two approaches. First, we set to measure gene expression under conditions that decouple the perceived environmental signal from its actual effect on cell growth. This was done by focusing on a specific mutant (<em>adh1</em>) which grows better on glycerol than on glucose. Under such conditions, environmental cues dominate, leading to increased level of ribosomal biogenesis gene expression in slowly-growing cells. Second, we examined the correlation between gene expression and growth rate during the dynamic response of chemostat grown culture to environmental changes. The focus on the dynamics, rather than the steady state behavior, allowed us to examine the relative timing of the two responses. Ribosome biogenesis gene expression responded rapidly to changes in the environments but was rather oblivious to longer-term changes in growth rate. Lack of correlation between growth rate and ribosomal biogenesis gene expression was also observed in a compendium of 196 deletion mutants, for which both growth rates and genome-wide expression profiles were reported <a href="#pone.0000250-Hughes1">[19]</a>. Taken together, our results suggest that the apparent coupling of translation-related gene expression and growth rate is not causal, but reflects the evolutionary fine-tuning of signal transduction mechanisms. The capacity to recognize and prepare to conditions that may alter changes in growth rate was probably a major selection force during yeast evolution.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Ribosomal biogenesis gene expression is highly responsive to changing environments</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We analyzed the expression levels of genes involved in ribosomal biogenesis using an annotated database of over 1500 genome-wide expression profiles in <em>S. cerevisiae</em> <a href="#pone.0000250-Ihmels1">[20]</a>. Our previous studies had identified two groups of co-regulated genes involved in making the ribosome <a href="#pone.0000250-Ihmels1">[20]</a>–<a href="#pone.0000250-Ihmels2">[22]</a>. The first group includes genes that code for the ribosomal proteins (RP) themselves. The second group is composed of genes that assist in the proper assembly of the ribosome, such as genes involved in the processing of rRNA, and is denoted here as ribosomal biogenesis gene module.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>As described previously, genes of both groups are co-expressed under a large number of conditions, and display a strong inverse correlation with genes induced as part of the general stress response <a href="#pone.0000250-Ihmels3">[23]</a>. This correlation pattern was particularly evident in the prototypic responses, termed environmental stress response (ESR), to a variety of environmental stresses (including heat-shock, peroxide shock or high osmolarity) <a href="#pone.0000250-Causton1">[24]</a>, <a href="#pone.0000250-Gasch1">[25]</a>. The ESR is characterized by the repression of ribosomal proteins and ribosomal biogenesis genes together with the induction of a common set of stress-related genes (<a href="#pone-0000250-g001">Fig 1b</a>). We note that although both RP and ribosomal biogenesis genes were repressed by environmental stresses, their responses were characterized by different kinetics (rapid repression of ribosomal biogenesis genes followed by a slower repression of RP genes) leading to their identification as two separate transcription modules (i.e. sets of coordinately expressed genes) <a href="#pone.0000250-Gasch1">[25]</a>.</p>
<div class="figure" id="pone-0000250-g001"><div class="img"><a name="pone-0000250-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000250" data-uri="info:doi/10.1371/journal.pone.0000250.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000250.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000250.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p><strong>(a)</strong> Environmental versus growth dependent modulation of gene expression. Gene expression can be modulated by direct signaling pathways which sense the environmental cues. Alternatively, it can be regulated by internal cues which are coupled directly to cell growth. In this study we characterize the relative contribution of external vs. internal sensing to the regulation of gene expression. (b,c) Expression profile of ESR modules during growth at different environmental conditions (reported in <a href="#pone.0000250-Gasch1">[25]</a>; see <a href="#pone.0000250.s004">table S1</a> for the list of genes). (b) The colormap represents the log<sub>2</sub>-expression-ratio of the respective gene module. The conditions are sorted by the average expression level of the stress module. (c) Average log<sub>2</sub>-expression-ratio of the respective gene module under the indicated condition. Stress-induced genes are activated, while ribosomal biogenesis and RP genes are repressed under conditions associated with slow growth (e.g. non-optimal temperatures, non-fermentable carbon source, or stationary phase).</p>
<span>doi:10.1371/journal.pone.0000250.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Gene expression in unperturbed cells was also reported <a href="#pone.0000250-Gasch1">[25]</a> (<a href="#pone-0000250-g001">Fig. 1c</a>). The expression of both RP and ribosomal biogenesis genes was maximal during logarithmic growth, and decreased as the cultures approached saturation. As expected, the level of ribosomal biogenesis gene expression was also correlated with the efficiency by which the corresponding carbon is utilized: expression was high in cells grown in a fermentable carbon source (glucose), whereas low expression was observed during growth in non-fermentable carbon sources such as ethanol. Similar differences in the level of ribosomal biogenesis genes were observed also when comparing steady state growth at different temperatures. In all the above conditions, the stress-induced part of the ESR exhibited high expression levels in conditions where ribosomal biogenesis genes expression was low, and low expression levels in conditions where ribosomal biogenesis gene expression was high. Although growth rates were not reported, the gene expression pattern seems consistent with the general notion that the expression of ribosomal biogenesis gene increases with growth rate, whereas the expression of stress genes decreases.</p>


<h4>Gene expression in the presence of internal vs. external conflict</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To examine whether the apparent correlation between the level of ribosomal biogenesis gene expression and growth rate reflects a feedback-mediated regulation that is sensitive to the growth rate itself, we sought a situation where the recognized environmental signal as perceived by the cell is separated from its actual effect on growth. Wild-type yeast cells grow most rapidly when provided with glucose as a carbon source. We considered a mutant strain deleted of the gene <em>ADH1</em>, which codes for an enzyme responsible for ethanol production during glucose fermentation. This strain cannot ferment glucose, and consequently grows faster on non-fermentable carbon sources (<em>e.g.</em> glycerol) than on glucose (<a href="#pone-0000250-g002">Fig. 2a</a>, <a href="#pone.0000250.s001">S1</a>).</p>
<div class="figure" id="pone-0000250-g002"><div class="img"><a name="pone-0000250-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000250" data-uri="info:doi/10.1371/journal.pone.0000250.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000250.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000250.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Ribosomal gene expression under “confusing” conditions. (a) Glucose is the preferred carbon source for wild-type yeast cells. In contrast, <em>adh1</em> mutant cells grow faster when glycerol is available (See also <a href="#pone.0000250.s001">Fig S1</a>). (b) Wild-type and mutant cells were grown to log phase on media containing glucose or glycerol as the only carbon source (<a href="#s4">Methods</a>). The expression profile on glycerol vs. glucose was quantified with microarrays. The bars represent the average log<sub>2</sub>-ratio-expression of the ESR modules. Notably, both wild type and mutant cells activate the ESR program. Stress genes are induced while ribosomal biogenesis genes and RP are repressed. (c) Response of <em>adh1</em> mutants to change in carbon source. Overnight cultures of <em>adh1</em> cells were transferred both to glycerol and to fresh glucose media. The bars represent the average expression after 30 minutes with respect to the initial cell culture expression (log<sub>2</sub>-ratios). As expected, the ESR is repressed upon transfer to fresh glucose medium. However, upon transfer to glycerol, the favorable carbon source of these mutant cells, the ESR is activated. Specifically, stress genes are induced while ribosomal biogenesis and RP genes are repressed. (d) Wild-type and <em>adh1</em> mutant cells were monitored upon transfer to glycerol versus their transfer to fresh glucose. As previously reported, the size of slow growing wild type cells (glycerol) is smaller than fast growing ones (glucose) <a href="#pone.0000250-Jorgensen2">[26]</a>. Interestingly, <em>adh1</em> mutants display an opposite behavior. Specifically, slow growing <em>adh1</em> mutants (glucose) are appreciably larger than the fast growing ones (glycerol). Thus, it appears that the cell size may also be tuned to the environmental signals rather than to cell division.</p>
<span>doi:10.1371/journal.pone.0000250.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We reasoned that comparing the gene expression pattern of <em>ADH1</em> cells grown in glucose vs. glycerol would allow us to distinguish between internal vs. external strategies of gene expression regulation. If internal, growth-rate dependent feedback dominates, ribosomal biogenesis gene expression will be higher in glycerol than in glucose, in accordance with the higher growth rate of the cells. Importantly, this prediction holds regardless of the mechanism which hinders <em>ADH1</em> growth in glucose. Alternatively, if signaling by the external environment dominates, ribosomal biogenesis gene expression will be higher in glucose, in accordance with the evolutionary expectation of higher growth rate of wild-type cells in glucose.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Using full-genome microarrays, we compared the expression profile of log-phase <em>adh1</em> mutant cells grown on glycerol vs. glucose medium (<a href="#pone-0000250-g002">Fig. 2b</a>). The expression of ribosomal biogenesis and RP genes was higher in glucose, although the growth rate of the cells was lower. Transfer of cells from glucose to glycerol led to a corresponding repression of ribosomal biogenesis genes (<a href="#pone-0000250-g002">Fig. 2c</a>). Similarly, the expression of stress genes was higher in glycerol (<a href="#pone-0000250-g002">Fig. 2b</a>), and was induced upon transfer from glucose to glycerol (<a href="#pone-0000250-g002">Fig. 2c</a>). Taken together, it appears that the expression of ribosomal biogenesis genes, as well as that of stress genes, was tuned to the environment rather than to the rate of cellular growth.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Interestingly, cell size was also tuned to the environmental signal rather then the growth rate. In <em>S. cerevisiae</em>, poor media and low growth rates correspond to smaller cell size. We observed, however, that <em>adh1</em> cells growing in glucose were larger than <em>adh1</em> cells growing in glycerol (<a href="#pone-0000250-g002">Fig. 2d</a>). Moreover, <em>adh1</em> cells growing on glucose were significantly larger than wild-type cells in glucose. This abnormal size may reflect the imbalance between their (high) ribosomal content and their (slow) growth rate, in consistence with the observed impact of ribosomal content on cell size <a href="#pone.0000250-Jorgensen2">[26]</a>.</p>


<h4>Temporal adaptation to perturbation in continuous cultures</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Our results suggest that upon conflicting signals, when the expected growth rate at a particular environment is different from the actual rate of growth, the environmental signal dominates. Next we asked whether growth-dependent signals play an important role under non-conflicting situations. As we argue above, however, in the absence of conflict the internal physiological state (<em>e.g.</em> growth rate) is compatible with the environmentally-expected one, making it is difficult to disentangle the signals received from the external environment from these that are generated as feedback from internal processes. For examples, changing the steady state growth rate in continuous cultures necessarily involves a change in the amount of limiting factor. It is thus hard to discern whether the changes in gene expression result from the change in growth rate itself, or whether they were generated as a response to the change in the abundance of the limiting factor.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We reasoned that characterizing temporal kinetics, rather than steady state behavior, may overcome this limitation. Consider an environmental perturbation that modulates both the gene expression and the growth rate. At steady state, these changes are likely to be coordinated. By following the temporal evolution of both responses, however, we can conclude whether the change in growth rate followed or precede the changes in gene expression. For example, if internal feedback mechanism is at function, any change in growth rate will be followed by a change in gene expression.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>To define a well-controlled environment where the kinetics of both gene expression and growth rate can be monitored, we grew cells in continuous cultures in a chemostat (<a href="#s4">Methods</a>). Cells were grown to steady state, and were subsequently subjected to perturbations. Microarrays were used to follow their genome-wide transcription response to each perturbation at subsequent time points following the perturbation. Notably, the changes in gene expression changes are analyzed relative to the gene expression in the unperturbed (steady state) culture, and thus reflect only the response to environmental cues. In parallel, we measured also the cells growth rate, by following the changes in their optical density (<a href="#pone-0000250-g003">Fig. 3a, 3b</a>, <a href="#pone.0000250.s002">S2</a>). At steady state, the optical density remains constant in time, and the rate of cell growth is set by the dilution rate of the chemostat (0.2 hr<sup>−1</sup> in our experiment) <a href="#pone.0000250-Segel1">[27]</a>. Since the cells are continuously diluted, and dilution rate remains fixed throughout the experiment, any change in growth rate will be reflected practically immediate by a change in the optical density.</p>
<div class="figure" id="pone-0000250-g003"><div class="img"><a name="pone-0000250-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000250" data-uri="info:doi/10.1371/journal.pone.0000250.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000250.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000250.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Gene expression versus growth rates in continuous cultures. (a) Steady state growing cells were subjected to different environmental stresses. Growth rate was calculated from the measured optical density. Simultaneously, the profile of gene expression was quantified with microarrays. In our study we looked for possible correlations between growth and gene expression (see <a href="#s4">Methods</a> for more details). (b) Cells grown at steady state in a glucose-limited chemostat were subjected to 5 different environmental cues. We examined 4 stress pulses: H<sub>2</sub>O<sub>2</sub> (0.6mM-magenta curve), DTT (6mM-blue curve), NaCl (0.4M-green curve), heat shock (39°C for 5 hours-black curve), and a stress-relieving pulse of glucose (0.2 g/l-red curve). The Y-axis is normalized such that the steady state O.D. (at t&lt;0) is 1. The markers designate O.D. sampling time, and solid lines are spline interpolations of the data. The perturbations in histidine-limited chemostat are shown in <a href="#pone.0000250.s002">figure S2</a>. (c) Dynamics of the average log<sub>2</sub>-ratio-expression for three ESR gene modules. The DTT, NaCl and heat shock pulses were given during glucose depletion, and the clotrimazole perturbation was given in histidine limitation. Bars designate the cells growth rate, which is normalized relative to the steady state growth rate (see <a href="#pone.0000250.s003">Fig. S3</a> for other perturbations dynamics). (d) Each gene was assigned a growth correlation score (GCR), quantifying the correlation between its expression level and growth rate, averaged over all perturbations (<a href="#s4">Methods</a>). The GCR distribution of several gene modules is shown. The X-axis represents the calculated average correlation, and Y-axis represents the fraction of genes within the module. The two vertical lines designate the threshold for the 5% of genes with the highest or lowest GCR. Notably, the distribution for the ribosomal biogenesis genes lies between the thresholds, while the cell-cycle G2-M gene group is enriched with positive GCR genes.</p>
<span>doi:10.1371/journal.pone.0000250.g003</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Since changes in growth rate necessarily involve changes in other factors, such as abundance or level of nutrients or drugs in the media, it was important to diversify the type of perturbations, so that a general correlation with growth-rate could be inferred. We considered four environmental stresses (Heat shock, high osmolarity, peroxide, DTT), a drug (clotrimazole), and supplementation of the limiting factor (histidine or glucose, respectively) (<a href="#pone-0000250-t001">Table 1</a>). Moreover, the experiments were performed using two chemostat cultures, growing in different limiting nutrients (histidine and glucose). Taken together, we analyzed 10 time course experiments, with a total of 83 arrays.</p>
<div class="figure" id="pone-0000250-t001"><div class="img"><a name="pone-0000250-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000250" data-uri="info:doi/10.1371/journal.pone.0000250.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000250.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000250.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Environmental cues imposed on steady state growing yeast cells.</span></strong></p><span>doi:10.1371/journal.pone.0000250.t001</span></div><a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>All perturbations led to a significant change in culture density, ranging from 15% increase in biomass (addition of histidine to the histidine-limited chemostat) to 70% reduction in biomass (addition of DTT to the glucose-limited chemostat) (<a href="#pone-0000250-g003">Fig. 3c</a>, <a href="#pone.0000250.s003">S3</a>). Significant changes in cell growth were observed about ~1 hour after the onset of the perturbation. The rate of growth kept changing for long time periods, adapting back on a scale of 10–30 hours, depending on the experiment. Typically, an ‘over-shoot’ was observed before reaching steady state, where cells grew faster than the dilution rate (<a href="#pone-0000250-g003">Fig. 3c</a>, <a href="#pone.0000250.s003">S3</a>).</p>


<h4>Temporal kinetics of ribosomal biogenesis gene expression vs. growth rate</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Changes in gene expression were observed immediately following the perturbation (<a href="#pone-0000250-g003">Figs. 3c</a>, <a href="#pone.0000250.s003">S3</a>). The gene expression response was similar to that characterized in batch cultures: environmental stresses led to a strong induction of stress-induced genes and the repression of ribosomal biogenesis genes. Also as expected, the addition of glucose to a glucose-limited chemostat led to a decrease in the ESR genes, and increase in RP and ribosomal biogenesis genes (<a href="#pone.0000250.s003">Fig. S3</a>).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Maximal change in gene expression was typically observed already at the first time point examined (10–20 minutes after the onset of the perturbation). In most perturbations, growth rate was hardly altered at that time. Thus, the initial gene expression response appears to be dominated by direct environmental signals, and not by growth-rate dependent feedbacks.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>The strong initial response to environmental changes may mask further growth-dependent signal. We asked whether growth-rate effects may dominate during the recovery from perturbation. The environmental perturbations were given in a pulse-like addition to the media, and were washed out of the media on a time scale of ~5 hours (corresponding to the dilution rate). The long-time temporal recovery from perturbation varied between experiments, but no consistent correlation between growth rate and ribosomal biogenesis gene expression was observed. During the recovery from heat-shock, for example, slow growth was associated with high levels of ribosomal biogenesis genes (<a href="#pone-0000250-g003">Fig. 3c</a>). During the recovery from DTT, temporal changes in growth rate seem to follow gene expression, rather then vice-versa: gene expression adapted to near its steady state level at 8.6 hours, a time where growth rate was minimal (<a href="#pone-0000250-g003">Fig. 3c</a>). Similarly, steady state-levels of gene expression were observed at 27 hours, a time when growth rate was still much higher than its steady state value. In a large number of perturbations, such as NaCl, gene expression was relatively constant during the long-term adaptation, although significant changes in growth rate were observed (<a href="#pone-0000250-g003">Fig. 3c</a>). Some correlation between gene expression and growth rate was observed during the response to clotrimazole (<a href="#pone-0000250-g003">Fig. 3c</a>). However, even in this case, the large increase in growth rate (over-shoot) during the final adaptation back to steady state (at 31 hours) was not accompanied by a corresponding change in gene expression.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>To explore the link between ESR gene expression and growth rate more systematically, we measured the correlation between the average expression of each gene group and growth rate (<a href="#pone.0000250.s005">Table S2</a>). This was done separately for each perturbation and then averaged over all perturbations tested. As expected from the analysis of individual perturbations, no significant correlation was observed for either of the groups considered (RP, ribosomal biogenesis genes and stress genes) (<a href="#pone.0000250.s005">Table S2</a>).</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>Taken together, our results indicate that environmental signal dominate over possible growth-rate dependent feedback in the modulation of gene expression during response to perturbation.</p>


<h4>Ribosomal biogenesis gene expression vs. growth rates in mutant cells</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>An additional means to decouple the growth rate from the environmental signal is by genetic mutations. We have asked whether growth-rate dependent feedback on gene expression can be recognized under such conditions. Gene deletion may impact gene expression through a multitude of effects, depending on its precise function. However, if a direct growth-rate dependent feedback significantly contributes to the regulation of gene expression, it would lead to an apparent correlation between expression levels and growth rates when a large number of unrelated mutants are examined.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We considered the previously published compendium of deletion profiles, characterizing the transcription profiles of 270 deletion mutants <a href="#pone.0000250-Hughes1">[19]</a>. Growth rates of 196 of these deletion strains were measured in an independent experiment, allowing us to examine the correlation of gene expression with cell growth rate. Consistent with our results above, no correlation was found between the expression levels of the ribosomal biogenesis genes and the reported growth rates (r = 0.03). A small inverse correlation was observed with the expression of the stress genes (r = −0.19), but this correlation was only marginally significant and considerably lower than those observed with the highest correlated gene groups (e.g. cell cycle group with r = 0.58, see also <a href="#pone.0000250.s005">Table S2</a>).</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Interestingly, a strong, and highly significant correlation was observed between the RP gene expression and the growth rate (r = 0.53). This is in contrast to the negligible correlation observed in the chemostat experiment. The decoupling of RP gene expression from ribosomal biogenesis gene expression is also unique to this dataset, as in most other datasets expression of these groups is tightly correlated. A possible explanation is that the mutant strains have undergone further genetic adaptation (secondary mutations <em>c.f.</em> Ref <a href="#pone.0000250-Hughes2">[28]</a>) that adjusted the level of ribosomal proteins with growth rates <a href="#pone.0000250-Dekel1">[29]</a>. Further experiments are required to examine this possibility.</p>


<h4>Identifying growth-correlated genes</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Taken together, it appears that growth rate <em>per-se</em> has a minimal impact on the expression levels of genes involved in ribosomal biogenesis. To examine whether other gene groups display a tighter link to growth rate, we extended the analysis described above to all yeast genes. Specifically, we assigned each gene a growth correlation score (GCR), quantifying the correlation between its expression level and growth rate, averaged over all perturbations (<a href="#s4">Methods</a>). A positive GCR indicates a correlation with growth rate, whereas a negative GCR indicates an inverse correlation. We focused on the 5% of genes with the highest or lowest GCR, respectively and characterized their properties by analyzing enrichment within different co-expressed gene groups defined previously <a href="#pone.0000250-Ihmels1">[20]</a>, <a href="#pone.0000250-Ihmels2">[22]</a> (<a href="#pone-0000250-g003">Fig. 3d</a>). To examine the generality of the results, we repeated the same analysis also for the compendium of deletion mutants, using the available gene expression and growth rate of the strains, as described above. The results are summarized in <a href="#pone-0000250-g004">Figure 4</a>.</p>
<div class="figure" id="pone-0000250-g004"><div class="img"><a name="pone-0000250-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000250" data-uri="info:doi/10.1371/journal.pone.0000250.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000250.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000250.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000250.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000250.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Enrichment of positive and negative correlated genes with previously defined gene modules. The table summarizes the analysis which spanned over hundreds of gene modules. For each gene group and experiment we calculated the enrichment with the positive (blue) and negative (red) correlated genes. Note that none of the gene modules were enriched with both positive and negative correlated genes and only the highest score is displayed. Less significant enrichments are shown in black (P-value&gt;10<sup>−3</sup>) (see also <a href="#pone.0000250.s006">Tables S3</a>).</p>
<span>doi:10.1371/journal.pone.0000250.g004</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>The group of genes that are positively correlated with growth was enriched primarily with cell cycle genes. In the chemostat experiment, enrichment was found primarily for genes regulated during the G2/M and M/G1 transition (p-value of 4?10<sup>−7</sup> and &lt;10<sup>−15</sup>, respectively), whereas in the compendium data enrichment was found for the G2/M transition as well as G1/S transition (p-values of 7?10<sup>−9</sup> and &lt;10<sup>−15</sup>, respectively). Other groups that were linked to growth included Mitochondrial ribosomal proteins and oxidative phosphorylation (chemostat experiment), and also gene groups related to histones, ribosomal proteins, phospholipids metabolism and cell wall (deletion mutants). In both experiments, the group of genes that were negatively correlated with growth was enriched with calcium calmodulin related genes (<a href="#pone-0000250-g004">Figure 4</a>).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Summary</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Gene expression is responsive to both external and internal cues. In this study, we wished to characterize the relative importance of these two modes of signaling to the regulation of genes involved in ribosomal biogenesis. Specifically, we sought to characterize the potential importance of internal feedback mechanisms that sense and respond to the cellular physiological state. Such growth-dependent regulation is well characterized in bacteria leading to direct dependency of the rate of ribosomal biogenesis on cellular growth rate. This feedback is mediated, at least in part, by the nucleotide dependency of rRNA transcription, and by the accumulation of uncharged tRNA, through the stringent response <a href="#pone.0000250-Gourse1">[10]</a>, <a href="#pone.0000250-Schneider1">[13]</a>, <a href="#pone.0000250-Chatterji1">[15]</a>, <a href="#pone.0000250-Murray1">[30]</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Our results indicate that in yeast, growth-related signals do not contribute significantly to the expression of ribosomal biogenesis genes. Several observations support this conclusion. First, under “confusion” situation, where the environmental signal and actual growth rate conflict, gene expression appears to be linked to the environment and not to the growth rate. Second, even in the absence of conflicting signals, the linkage of ribosomal biogenesis gene expression to growth rate relies on the availability of a proper environmental signal. Thus, no correlation was observed during the long-term recovery of chemostat-grown cells, when large variations in growth rate were observed, and we did not detect situations where ribosomal biogenesis gene expression was clearly dictated by growth rate. Similarly, no linkage between these factors was observed in a compendium of deletion mutants, where both gene expression and growth rate were modified not by changing an environment but rather by deleting specific genes.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>We identified several gene groups which displayed tight correlation to growth rate. With the exception of the cell cycle genes, most groups identified were specific to a particular experimental condition (see also <a href="#pone.0000250.s006">Table S3</a>). It may be that correlation with growth is observed only for genes involved in rate limiting processes; For example, gluconeogenesis genes displayed a correlation with growth in the glucose-limited chemostat (<a href="#pone.0000250.s006">Table S3</a>). The correlation of amino-acid biosynthesis and phosphate utilization groups with growth rate in the compendium of deletion mutant may reflect limitation of the media used.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>In conclusion, it appears that yeast cells control the level of ribosome biogenesis gene expression primarily by responding to environmental signals. The apparent correlation between gene expression and growth rate, observed under steady state conditions, is likely to reflect the evolutionary fine-tuning of these signals with the expected growth rate, rather than direct growth-dependent signals. This strategy might compensate the exact tuning of gene expression with growth rate, particularly under conditions that are rarely encountered over evolutionary times. On the other hand, it allows for rapid response and preparation to anticipated changes that will impact the growth rate. Our results suggest that the capacity to anticipate and prepare for environmentally-mediated changes in growth rate, before such changes actually occur, presented a major selection force during yeast evolution.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Media and strains</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>For the limiting histidine and glucose chemostats we used the following two strains, respectively: W303-1 bar1::P<sub>ADH1</sub>-CFP-<em>KANMX</em> (<em>MAT</em>a, <em>leu2-Δ1, trp1-Δ63, his3-Δ200, ura3-52, ade2-101</em>) and BY4741 (Euroscarf; <em>MAT</em>a, <em>his3-Δ1, leu2-Δ0, met15-Δ0, ura3-Δ0</em>). For the mutant experiments we used BY4741 <em>adh1</em>::<em>KANMX</em> cells (Euroscarf), and wild-type BY4741 cells were used as a control.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>For the <em>adh1</em> mutant experiment, YPD and YP-glycerol (2%) media were used. Histidine limited chemostat medium consisted of bacto-yeast nitrogen base without amino acids and with ammonium sulfate (0.67%), glucose (2%) and a drop-out mix (0.2%) of the following combination: L-Histidine (1.3 mg), Adenine sulfate (20 mg), Uracil (20 mg), L-Tryptophan (20 mg), L-Leucine (100 mg), L-Arginine (20 mg), L-Methionine (20 mg), L-Tyrosine (30 mg), L-Isoleucine (30 mg), L-Lysine (30 mg), L-Phenylalanine (50 mg), L-Glutamic acid (100 mg), L-Aspartic acid (100 mg), L-Valine (150 mg), L-Threonine (200 mg), L-Serine (400 mg). The final concentration of histidine in this medium is 2 mg/l. Limited glucose medium consisted of bacto-yeast nitrogen base without amino acids and with ammonium sulfate (0.67%), glucose (0.02%) and a drop-out mix (0.2%) as mentioned above but with 20 mg/l of L-histidine. Media for chemostat experiments included also 20 µl/liter of antifoam (Sigma A5758). Importantly, media composition was chosen such that each chemostat has a single limiting factor. Specifically, the response of the yeast culture to addition of different medium components was tested to assure that the cells are sensitive only to changes in histidine or glucose concentrations.</p>


<h4>adh1 mutant experiment</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p><em>adh1</em> deletion strain and its isogenic wild type strain BY4741 were used to generate genome wide expression profiles. For the first experiment (<a href="#pone-0000250-g002">Fig. 2B</a>), cultures were grown in YPD for 8 hours and then washed and diluted into either YPD or YP-Glycerol. The cells were harvested after 4-5 duplications, upon reaching the concentration of ~2*10<sup>6</sup> cells/ml (see also <a href="#pone.0000250.s001">Fig. S1</a>). For the second experiment (<a href="#pone-0000250-g002">Fig. 2C</a>), <em>adh1</em> strain was grown on YPD to O.D. 0.4, and then washed and diluted into either YPD or YP-Glycerol. Cells were harvested after 30 minutes of growth and compared to samples of the initial overnight culture (at time t = 0).</p>


<h4>Continuous culture experiments</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Continuous cultures were grown in BioFlo 110 fermentors (New Brunswick Scientific) in the following conditions: dilution rate = 0.2 h<sup>−1</sup>, fermentor working volume = 750ml, temperature = 30°C, agitation speed = 500 RPM, air flux = 1 LPM, and the dissolved oxygen level was always above 70% of the saturation level. The pH of the media was titrated with NaOH to values of 5 and 5.3–5.6 for the histidine and glucose limited chemostats, respectively. After reaching steady state, cells were subjected to 10 different environmental cues (<a href="#pone-0000250-t001">Table 1</a>). Cell samples were harvested at various times during each perturbation. As a control, batch culture cells were grown in similar media and harvested at late logarithmic phase (Note that patterns of gene expression in steady-state closely resemble those of corresponding batch cultures just before they exhaust the nutrient <a href="#pone.0000250-Saldanha1">[31]</a>).</p>


<h4>Microarray hybridization and data quantification</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Samples were collected at various time points and RNA was isolated in order to generate cDNA, which was then labeled by the indirect amino-allyl method. Cy5-and Cy3-labeled cDNA was hybridized onto microarrays of ORFs representing the complete <em>S. cerevisiae</em> genome (University Health Network, Ontario), using cDNA from untreated cultures as reference. The arrays were scanned, and expression data was extracted by image analysis. Expression ratios were log<sub>2</sub> transformed, corrected for technical biases and normalized (see also <a href="#pone.0000250.s007">Text S1</a>).</p>


<h4>Module identification</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Transcription modules were identified by applying the Iterative Signature Algorithm to the full expression dataset comprising all conditions <a href="#pone.0000250-Ihmels1">[20]</a>, <a href="#pone.0000250-Bergmann1">[21]</a>. The Iterative Signature Algorithm is an iterative extension of the Signature Algorithm, designed for a global decomposition of large-scale expression matrices into transcription modules, in the absence of a-priory information. Modules are fixed-points of the Signature Algorithm, and are identified in a heuristic search by iterating from a large ensemble of randomly composed input sets until convergence is reached. The gene threshold of the Signature Algorithm determines the resolution of the analysis. The full set of transcription modules used in this work is available in the Supplementary Materials and further details are presented at the following URL: <a href="http://barkai-serv.weizmann.ac.il/Modules">http://barkai-serv.weizmann.ac.il/Module​s</a>.</p>


<h4>Correlation between growth rate and gene expression</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The measured O.D. was interpolated using the Matlab csaps function (<a href="#pone-0000250-g003">Fig. 3b</a>, <a href="#pone.0000250.s002">S2</a>). The cells growth rate (µ) was calculated using the chemostat cell density equation: <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.e001&amp;representation=PNG" class="inline-graphic"></span>, where n is the cell density (interpolated O.D) and D is the dilution rate <a href="#pone.0000250-Segel1">[27]</a>. Correlations between growth and gene expression were calculated using the Matlab corrcoef function. Each gene was assigned a growth correlation score (GCR), quantifying the correlation between its expression level and growth rate, averaged over all perturbations. Sorting the genes by their GCR revealed two gene groups: the group of 5% most positive correlated genes (positive GCR) and 5% most negatively correlated genes (negative GCR). In order to reduce noise, we neglected genes that were not affected by the environmental perturbations (see <a href="#pone.0000250.s007">Text S1</a>).</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>Our study focused on finding the enrichment of positive GCR and negative GCR genes in different co-regulated gene groups, which were previously defined by the signature algorithm <a href="#pone.0000250-Ihmels1">[20]</a>, <a href="#pone.0000250-Bergmann1">[21]</a>. The analysis was unbiased and done over all gene modules within all threshold levels. The P-value for each group enrichment with positive and negative GCR was obtained by calculating the chance of getting at least such an enrichment by chance. Specifically: <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.e002&amp;representation=PNG" class="inline-graphic"></span>, where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000250.e003&amp;representation=PNG" class="inline-graphic"></span> is the hyper-geometrical distribution designating the probability for enrichment of X of a possible K correlated genes from a group with N genes (module size) without replacement from a pool of M genes.</p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p>Gene groups were sorted both by their P-value for enrichment with positive GCR, and by their P-value for enrichment with negative GCR. The analysis was done for the continuous culture experiments considering all 10 perturbations (<a href="#pone-0000250-g004">Figure 4</a>). The analysis was done also for each of the two limiting factor experiments separately (only the 5 perturbations in histidine or glucose depletion, <a href="#pone.0000250.s006">Table S3</a>). The same calculation was repeated for the compendium data. Additional figures and tables are available in the Supplementary Material.</p>


<h4>Accession numbers</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>All raw data is available through Gene Expression Omnibus (GEO) accession number GSE6302, at the following <a href="website: http://www.ncbi.nlm.nih.gov/geo">website: http://www.ncbi.nlm.nih.gov/geo</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000250"></div><a name="pone.0000250.s001" id="pone.0000250.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000250.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Growth curves of adh1 and wild type cells</p>
<p class="siDoi">doi:10.1371/journal.pone.0000250.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.42 MB PDF)</p>
<a name="pone.0000250.s002" id="pone.0000250.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000250.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Response of cells grown in steady state to environmental perturbations</p>
<p class="siDoi">doi:10.1371/journal.pone.0000250.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.46 MB PDF)</p>
<a name="pone.0000250.s003" id="pone.0000250.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000250.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Dynamics of average module expression and cell growth upon environmental perturbations. The average log2-ratio-expression is presented for three ESR gene modules. Bars designate the measured cells growth rate, which is normalized relative to the steady state growth rate. </p>
<p class="siDoi">doi:10.1371/journal.pone.0000250.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.44 MB PDF)</p>
<a name="pone.0000250.s004" id="pone.0000250.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000250.s004">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">List of gene modules</p>
<p class="siDoi">doi:10.1371/journal.pone.0000250.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.18 MB PDF)</p>
<a name="pone.0000250.s005" id="pone.0000250.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000250.s005">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Correlations between growth and the modules average expression</p>
<p class="siDoi">doi:10.1371/journal.pone.0000250.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.07 MB PDF)</p>
<a name="pone.0000250.s006" id="pone.0000250.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000250.s006">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Effect of medium limitations on the enrichment of positive and negative correlated genes with gene modules.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000250.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.09 MB PDF)</p>
<a name="pone.0000250.s007" id="pone.0000250.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000250.s007">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Supplementary Methods</p>
<p class="siDoi">doi:10.1371/journal.pone.0000250.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank S. Maizel for the help and advice with setting up the chemostat system. We thank B.Z. Shilo for helpful comments on the manuscript. We also thank I. Tirosh and A. Koren for helpful discussions and careful reading of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: NB JI SL GF. Performed the experiments: MC SL AW GF. Analyzed the data: NB JI SL. Contributed reagents/materials/analysis tools: JI. Wrote the paper: NB SL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000250-Powers1" id="pone.0000250-Powers1"></a>Powers T, Walter P (1999) Regulation of ribosome biogenesis by the rapamycin-sensitive TOR-signaling pathway in Saccharomyces cerevisiae. Mol Biol Cell  10: 987–1000.  <ul class="find" data-citedArticleID="933500" data-doi="10.1091/mbc.10.4.987"><li><a href="http://dx.doi.org/10.1091/mbc.10.4.987" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+ribosome+biogenesis+by+the+rapamycin-sensitive+TOR-signaling+pathway+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+ribosome+biogenesis+by+the+rapamycin-sensitive+TOR-signaling+pathway+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000250-Martin1" id="pone.0000250-Martin1"></a>Martin DE, Soulard A, Hall MN (2004) TOR regulates ribosomal protein gene expression via PKA and the Forkhead transcription factor FHL1. Cell  119: 969–979.  <ul class="find" data-citedArticleID="933492" data-doi="10.1016/j.cell.2004.11.047"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.11.047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TOR+regulates+ribosomal+protein+gene+expression+via+PKA+and+the+Forkhead+transcription+factor+FHL1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TOR+regulates+ribosomal+protein+gene+expression+via+PKA+and+the+Forkhead+transcription+factor+FHL1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000250-ZuritaMartinez1" id="pone.0000250-ZuritaMartinez1"></a>Zurita-Martinez SA, Cardenas ME (2005) Tor and cyclic AMP-protein kinase A: two parallel pathways regulating expression of genes required for cell growth. Eukaryot Cell  4: 63–71.  <ul class="find" data-citedArticleID="933520" data-doi="10.1128/ec.4.1.63-71.2005"><li><a href="http://dx.doi.org/10.1128/ec.4.1.63-71.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tor+and+cyclic+AMP-protein+kinase+A%3A+two+parallel+pathways+regulating+expression+of+genes+required+for+cell+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tor+and+cyclic+AMP-protein+kinase+A%3A+two+parallel+pathways+regulating+expression+of+genes+required+for+cell+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000250-Warner1" id="pone.0000250-Warner1"></a>Warner JR (1999) The economics of ribosome biosynthesis in yeast. Trends Biochem Sci  24: 437–440.  <ul class="find" data-citedArticleID="933516" data-doi="10.1016/s0968-0004(99)01460-7"><li><a href="http://dx.doi.org/10.1016/s0968-0004(99)01460-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+economics+of+ribosome+biosynthesis+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+economics+of+ribosome+biosynthesis+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000250-Fatica1" id="pone.0000250-Fatica1"></a>Fatica A, Tollervey D (2002) Making ribosomes. Curr Opin Cell Biol  14: 313–318.  <ul class="find" data-citedArticleID="933468" data-doi="10.1016/s0955-0674(02)00336-8"><li><a href="http://dx.doi.org/10.1016/s0955-0674(02)00336-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Making+ribosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Making+ribosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000250-Moss1" id="pone.0000250-Moss1"></a>Moss T (2004) At the crossroads of growth control; making ribosomal RNA. Curr Opin Genet Dev  14: 210–217.  <ul class="find" data-citedArticleID="933494" data-doi="10.1016/j.gde.2004.02.005"><li><a href="http://dx.doi.org/10.1016/j.gde.2004.02.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=At+the+crossroads+of+growth+control%3B+making+ribosomal+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22At+the+crossroads+of+growth+control%3B+making+ribosomal+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000250-Jorgensen1" id="pone.0000250-Jorgensen1"></a>Jorgensen P, Rupes I, Sharom JR, Schneper L, Broach JR, et al.  (2004) A dynamic transcriptional network communicates growth potential to ribosome synthesis and critical cell size. Genes Dev  18: 2491–2505.  <ul class="find" data-citedArticleID="933488" data-doi="10.1101/gad.1228804"><li><a href="http://dx.doi.org/10.1101/gad.1228804" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+dynamic+transcriptional+network+communicates+growth+potential+to+ribosome+synthesis+and+critical+cell+size." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+dynamic+transcriptional+network+communicates+growth+potential+to+ribosome+synthesis+and+critical+cell+size.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000250-Ruggero1" id="pone.0000250-Ruggero1"></a>Ruggero D, Pandolfi PP (2003) Does the ribosome translate cancer? Nat Rev Cancer  3: 179–192.  <ul class="find" data-citedArticleID="933504" data-doi="10.1038/nrc1015"><li><a href="http://dx.doi.org/10.1038/nrc1015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Does+the+ribosome+translate+cancer%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Does+the+ribosome+translate+cancer%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000250-Wang1" id="pone.0000250-Wang1"></a>Wang H, Zhao LN, Li KZ, Ling R, Li XJ, et al.  (2006) Overexpression of ribosomal protein L15 is associated with cell proliferation in gastric cancer. BMC Cancer  6: 91.  <ul class="find" data-citedArticleID="933514"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Overexpression+of+ribosomal+protein+L15+is+associated+with+cell+proliferation+in+gastric+cancer.&amp;auth=&amp;atitle=Overexpression+of+ribosomal+protein+L15+is+associated+with+cell+proliferation+in+gastric+cancer." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overexpression+of+ribosomal+protein+L15+is+associated+with+cell+proliferation+in+gastric+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overexpression+of+ribosomal+protein+L15+is+associated+with+cell+proliferation+in+gastric+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000250-Gourse1" id="pone.0000250-Gourse1"></a>Gourse RL, Gaal T, Bartlett MS, Appleman JA, Ross W (1996) rRNA transcription and growth rate-dependent regulation of ribosome synthesis in Escherichia coli. Annu Rev Microbiol  50: 645–677.  <ul class="find" data-citedArticleID="933474" data-doi="10.1146/annurev.micro.50.1.645"><li><a href="http://dx.doi.org/10.1146/annurev.micro.50.1.645" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=rRNA+transcription+and+growth+rate-dependent+regulation+of+ribosome+synthesis+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22rRNA+transcription+and+growth+rate-dependent+regulation+of+ribosome+synthesis+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000250-Gaal1" id="pone.0000250-Gaal1"></a>Gaal T, Bartlett MS, Ross W, Turnbough CL Jr., Gourse RL (1997) Transcription regulation by initiating NTP concentration: rRNA synthesis in bacteria. Science  278: 2092–2097.  <ul class="find" data-citedArticleID="933470" data-doi="10.1126/science.278.5346.2092"><li><a href="http://dx.doi.org/10.1126/science.278.5346.2092" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+regulation+by+initiating+NTP+concentration%3A+rRNA+synthesis+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+regulation+by+initiating+NTP+concentration%3A+rRNA+synthesis+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000250-Paul1" id="pone.0000250-Paul1"></a>Paul BJ, Ross W, Gaal T, Gourse RL (2004) rRNA transcription in Escherichia coli. Annu Rev Genet  38: 749–770.  <ul class="find" data-citedArticleID="933498" data-doi="10.1146/annurev.genet.38.072902.091347"><li><a href="http://dx.doi.org/10.1146/annurev.genet.38.072902.091347" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=rRNA+transcription+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22rRNA+transcription+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000250-Schneider1" id="pone.0000250-Schneider1"></a>Schneider DA, Gaal T, Gourse RL (2002) NTP-sensing by rRNA promoters in Escherichia coli is direct. Proc Natl Acad Sci U S A  99: 8602–8607.  <ul class="find" data-citedArticleID="933508" data-doi="10.1073/pnas.132285199"><li><a href="http://dx.doi.org/10.1073/pnas.132285199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NTP-sensing+by+rRNA+promoters+in+Escherichia+coli+is+direct." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NTP-sensing+by+rRNA+promoters+in+Escherichia+coli+is+direct.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000250-Walker1" id="pone.0000250-Walker1"></a>Walker KA, Mallik P, Pratt TS, Osuna R (2004) The Escherichia coli Fis promoter is regulated by changes in the levels of its transcription initiation nucleotide CTP. J Biol Chem  279: 50818–50828.  <ul class="find" data-citedArticleID="933512" data-doi="10.1074/jbc.m406285200"><li><a href="http://dx.doi.org/10.1074/jbc.m406285200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Escherichia+coli+Fis+promoter+is+regulated+by+changes+in+the+levels+of+its+transcription+initiation+nucleotide+CTP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Escherichia+coli+Fis+promoter+is+regulated+by+changes+in+the+levels+of+its+transcription+initiation+nucleotide+CTP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000250-Chatterji1" id="pone.0000250-Chatterji1"></a>Chatterji D, Ojha AK (2001) Revisiting the stringent response, ppGpp and starvation signaling. Curr Opin Microbiol  4: 160–165.  <ul class="find" data-citedArticleID="933464" data-doi="10.1016/s1369-5274(00)00182-x"><li><a href="http://dx.doi.org/10.1016/s1369-5274(00)00182-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Revisiting+the+stringent+response%2C+ppGpp+and+starvation+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Revisiting+the+stringent+response%2C+ppGpp+and+starvation+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000250-Jain1" id="pone.0000250-Jain1"></a>Jain V, Kumar M, Chatterji D (2006) ppGpp: stringent response and survival. J Microbiol  44: 1–10.  <ul class="find" data-citedArticleID="933486"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=ppGpp%3A+stringent+response+and+survival.&amp;auth=&amp;atitle=ppGpp%3A+stringent+response+and+survival." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ppGpp%3A+stringent+response+and+survival." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ppGpp%3A+stringent+response+and+survival.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000250-Wendrich1" id="pone.0000250-Wendrich1"></a>Wendrich TM, Blaha G, Wilson DN, Marahiel MA, Nierhaus KH (2002) Dissection of the mechanism for the stringent factor RelA. Mol Cell  10: 779–788.  <ul class="find" data-citedArticleID="933518" data-doi="10.1016/s1097-2765(02)00656-1"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00656-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dissection+of+the+mechanism+for+the+stringent+factor+RelA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dissection+of+the+mechanism+for+the+stringent+factor+RelA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000250-Regenberg1" id="pone.0000250-Regenberg1"></a>Regenberg B, Grotkjaer T, Winther O, Fausboll A, Akesson M, et al.  (2006) Growth-rate regulated genes have profound impact on interpretation of transcriptome profiling in Saccharomyces cerevisiae. Genome Biol  7: R107.  <ul class="find" data-citedArticleID="933502" data-doi="10.1186/gb-2006-7-11-r107"><li><a href="http://dx.doi.org/10.1186/gb-2006-7-11-r107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Growth-rate+regulated+genes+have+profound+impact+on+interpretation+of+transcriptome+profiling+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Growth-rate+regulated+genes+have+profound+impact+on+interpretation+of+transcriptome+profiling+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000250-Hughes1" id="pone.0000250-Hughes1"></a>Hughes TR, Marton MJ, Jones AR, Roberts CJ, Stoughton R, et al.  (2000) Functional discovery via a compendium of expression profiles. Cell  102: 109–126.  <ul class="find" data-citedArticleID="933476" data-doi="10.1016/s0092-8674(00)00015-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)00015-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+discovery+via+a+compendium+of+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+discovery+via+a+compendium+of+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000250-Ihmels1" id="pone.0000250-Ihmels1"></a>Ihmels J, Friedlander G, Bergmann S, Sarig O, Ziv Y, et al.  (2002) Revealing modular organization in the yeast transcriptional network. Nat Genet  31: 370–377.  <ul class="find" data-citedArticleID="933480" data-doi="10.1038/ng941"><li><a href="http://dx.doi.org/10.1038/ng941" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Revealing+modular+organization+in+the+yeast+transcriptional+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Revealing+modular+organization+in+the+yeast+transcriptional+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000250-Bergmann1" id="pone.0000250-Bergmann1"></a>Bergmann S, Ihmels J, Barkai N (2003) Iterative signature algorithm for the analysis of large-scale gene expression data. Phys Rev E Stat Nonlin Soft Matter Phys  67: 031902.  <ul class="find" data-citedArticleID="933460" data-doi="10.1103/physreve.67.031902"><li><a href="http://dx.doi.org/10.1103/physreve.67.031902" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Iterative+signature+algorithm+for+the+analysis+of+large-scale+gene+expression+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Iterative+signature+algorithm+for+the+analysis+of+large-scale+gene+expression+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000250-Ihmels2" id="pone.0000250-Ihmels2"></a>Ihmels J, Bergmann S, Barkai N (2004) Defining transcription modules using large-scale gene expression data. Bioinformatics  20: 1993–2003.  <ul class="find" data-citedArticleID="933482" data-doi="10.1093/bioinformatics/bth166"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth166" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Defining+transcription+modules+using+large-scale+gene+expression+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Defining+transcription+modules+using+large-scale+gene+expression+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000250-Ihmels3" id="pone.0000250-Ihmels3"></a>Ihmels J, Bergmann S, Gerami-Nejad M, Yanai I, McClellan M, et al.  (2005) Rewiring of the yeast transcriptional network through the evolution of motif usage. Science  309: 938–940.  <ul class="find" data-citedArticleID="933484" data-doi="10.1126/science.1113833"><li><a href="http://dx.doi.org/10.1126/science.1113833" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rewiring+of+the+yeast+transcriptional+network+through+the+evolution+of+motif+usage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rewiring+of+the+yeast+transcriptional+network+through+the+evolution+of+motif+usage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000250-Causton1" id="pone.0000250-Causton1"></a>Causton HC, Ren B, Koh SS, Harbison CT, Kanin E, et al.  (2001) Remodeling of yeast genome expression in response to environmental changes. Mol Biol Cell  12: 323–337.  <ul class="find" data-citedArticleID="933462" data-doi="10.1091/mbc.12.2.323"><li><a href="http://dx.doi.org/10.1091/mbc.12.2.323" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Remodeling+of+yeast+genome+expression+in+response+to+environmental+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Remodeling+of+yeast+genome+expression+in+response+to+environmental+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000250-Gasch1" id="pone.0000250-Gasch1"></a>Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, et al.  (2000) Genomic expression programs in the response of yeast cells to environmental changes. Mol Biol Cell  11: 4241–4257.  <ul class="find" data-citedArticleID="933472" data-doi="10.1091/mbc.11.12.4241"><li><a href="http://dx.doi.org/10.1091/mbc.11.12.4241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+expression+programs+in+the+response+of+yeast+cells+to+environmental+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+expression+programs+in+the+response+of+yeast+cells+to+environmental+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000250-Jorgensen2" id="pone.0000250-Jorgensen2"></a>Jorgensen P, Nishikawa JL, Breitkreutz BJ, Tyers M (2002) Systematic identification of pathways that couple cell growth and division in yeast. Science  297: 395–400.  <ul class="find" data-citedArticleID="933490" data-doi="10.1126/science.1070850"><li><a href="http://dx.doi.org/10.1126/science.1070850" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+identification+of+pathways+that+couple+cell+growth+and+division+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+identification+of+pathways+that+couple+cell+growth+and+division+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000250-Segel1" id="pone.0000250-Segel1"></a>Segel LA (1984) Modeling dynamic phenomena in molecular and cellular biology. Cambridge ; New York: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0000250-Hughes2" id="pone.0000250-Hughes2"></a>Hughes TR, Roberts CJ, Dai H, Jones AR, Meyer MR, et al.  (2000) Widespread aneuploidy revealed by DNA microarray expression profiling. Nat Genet  25: 333–337.  <ul class="find" data-citedArticleID="933478"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Widespread+aneuploidy+revealed+by+DNA+microarray+expression+profiling.&amp;auth=&amp;atitle=Widespread+aneuploidy+revealed+by+DNA+microarray+expression+profiling." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Widespread+aneuploidy+revealed+by+DNA+microarray+expression+profiling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Widespread+aneuploidy+revealed+by+DNA+microarray+expression+profiling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000250-Dekel1" id="pone.0000250-Dekel1"></a>Dekel E, Alon U (2005) Optimality and evolutionary tuning of the expression level of a protein. Nature  436: 588–592.  <ul class="find" data-citedArticleID="933466" data-doi="10.1038/nature03842"><li><a href="http://dx.doi.org/10.1038/nature03842" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimality+and+evolutionary+tuning+of+the+expression+level+of+a+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimality+and+evolutionary+tuning+of+the+expression+level+of+a+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000250-Murray1" id="pone.0000250-Murray1"></a>Murray HD, Schneider DA, Gourse RL (2003) Control of rRNA expression by small molecules is dynamic and nonredundant. Mol Cell  12: 125–134.  <ul class="find" data-citedArticleID="933496" data-doi="10.1016/s1097-2765(03)00266-1"><li><a href="http://dx.doi.org/10.1016/s1097-2765(03)00266-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+rRNA+expression+by+small+molecules+is+dynamic+and+nonredundant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+rRNA+expression+by+small+molecules+is+dynamic+and+nonredundant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000250-Saldanha1" id="pone.0000250-Saldanha1"></a>Saldanha AJ, Brauer MJ, Botstein D (2004) Nutritional homeostasis in batch and steady-state culture of yeast. Mol Biol Cell  15: 4089–4104.  <ul class="find" data-citedArticleID="933506" data-doi="10.1091/mbc.e04-04-0306"><li><a href="http://dx.doi.org/10.1091/mbc.e04-04-0306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nutritional+homeostasis+in+batch+and+steady-state+culture+of+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nutritional+homeostasis+in+batch+and+steady-state+culture+of+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.XML" value="87304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.PDF" value="467636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g001.PNG_L" value="1368256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g001.PNG_M" value="153691"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g001.PNG_S" value="4954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g001.TIF" value="2074088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g001.PNG_I" value="58715"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g002.PNG_L" value="944439"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g002.PNG_M" value="56108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g002.PNG_S" value="3367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g002.TIF" value="1254168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g002.PNG_I" value="21650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g003.PNG_L" value="1032007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g003.PNG_M" value="133482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g003.PNG_S" value="5081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g003.TIF" value="1614202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g003.PNG_I" value="57165"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.t001.PNG_L" value="74687"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.t001.PNG_M" value="26196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.t001.PNG_S" value="2144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.t001.TIF" value="426570"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.t001.PNG_I" value="11572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g004.PNG_L" value="1198867"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g004.PNG_M" value="165521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g004.PNG_S" value="4970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g004.TIF" value="1698802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.g004.PNG_I" value="63032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.e001.PNG" value="1293"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.e001.TIF" value="22264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.e002.PNG" value="4155"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.e002.TIF" value="25720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.e003.PNG" value="7556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.e003.TIF" value="29232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.s001.PDF" value="416037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.s002.PDF" value="459453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.s003.PDF" value="438734"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.s004.PDF" value="180219"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.s005.PDF" value="67270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.s006.PDF" value="94861"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000250.s007.PDF" value="17766"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000250&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000250" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000250&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000250&volume=&issue=&title=Strategy of Transcription Regulation in the Budding Yeast&author_name=Sagi%20Levy%2C%20Jan%20Ihmels%2C%20Miri%20Carmi%2C%20Adina%20Weinberger%2C%20Gilgi%20Friedlander%2C%20Naama%20Barkai&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000250" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000250" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000250" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000250&amp;t=Strategy%20of%20Transcription%20Regulation%20in%20the%20Budding%20Yeast" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000250&title=Strategy%20of%20Transcription%20Regulation%20in%20the%20Budding%20Yeast&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000250&amp;title=Strategy%20of%20Transcription%20Regulation%20in%20the%20Budding%20Yeast" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000250" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000250&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Strategy of Transcription Regulation in the Budding Yeast';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000250';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Strategy%20of%20Transcription%20Regulation%20in%20the%20Budding%20Yeast http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000250" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000250" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biosynthesis%22" title="Search for articles in the subject area:'Biosynthesis'"><div class="flagText">Biosynthesis</div></a>
              <div data-categoryid="40369" data-articleid="23338"
                   data-categoryname="Biosynthesis"
                   class="flagImage" title="Flag 'Biosynthesis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+growth%22" title="Search for articles in the subject area:'Cell growth'"><div class="flagText">Cell growth</div></a>
              <div data-categoryid="19789" data-articleid="23338"
                   data-categoryname="Cell growth"
                   class="flagImage" title="Flag 'Cell growth' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+evolution%22" title="Search for articles in the subject area:'Fungal evolution'"><div class="flagText">Fungal evolution</div></a>
              <div data-categoryid="17551" data-articleid="23338"
                   data-categoryname="Fungal evolution"
                   class="flagImage" title="Flag 'Fungal evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23338"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Glucose%22" title="Search for articles in the subject area:'Glucose'"><div class="flagText">Glucose</div></a>
              <div data-categoryid="20225" data-articleid="23338"
                   data-categoryname="Glucose"
                   class="flagImage" title="Flag 'Glucose' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Glycerol%22" title="Search for articles in the subject area:'Glycerol'"><div class="flagText">Glycerol</div></a>
              <div data-categoryid="42181" data-articleid="23338"
                   data-categoryname="Glycerol"
                   class="flagImage" title="Flag 'Glycerol' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Histidine%22" title="Search for articles in the subject area:'Histidine'"><div class="flagText">Histidine</div></a>
              <div data-categoryid="40815" data-articleid="23338"
                   data-categoryname="Histidine"
                   class="flagImage" title="Flag 'Histidine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="23338"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9701'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9235'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8607&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=17407">mistake</a><br>Posted by perezje</p>
      <p><a href="/annotation/listThread.action?root=20611">mistake</a><br>Posted by perezje</p>
      <p><a href="/annotation/listThread.action?root=20211">Mistake</a><br>Posted by perezje</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
