

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Breakage-Reunion Domain of Streptococcus pneumoniae Topoisomerase IV: Crystal Structure of a Gram-Positive Quinolone Target</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000301"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000301" />

    <meta name="citation_title" content="Breakage-Reunion Domain of Streptococcus pneumoniae Topoisomerase IV: Crystal Structure of a Gram-Positive Quinolone Target"/>
    <meta itemprop="name" content="Breakage-Reunion Domain of Streptococcus pneumoniae Topoisomerase IV: Crystal Structure of a Gram-Positive Quinolone Target"/>

      <meta name="citation_author" content="Ivan Laponogov"/>
            <meta name="citation_author_institution" content="Randall Division of Cell and Molecular Biophysics, School of Biomedical and Health Sciences, King's College London, London, United Kindgom"/>
      <meta name="citation_author" content="Dennis A. Veselkov"/>
            <meta name="citation_author_institution" content="Randall Division of Cell and Molecular Biophysics, School of Biomedical and Health Sciences, King's College London, London, United Kindgom"/>
      <meta name="citation_author" content="Maninder K. Sohi"/>
            <meta name="citation_author_institution" content="Randall Division of Cell and Molecular Biophysics, School of Biomedical and Health Sciences, King's College London, London, United Kindgom"/>
      <meta name="citation_author" content="Xiao-Su Pan"/>
            <meta name="citation_author_institution" content="Molecular Genetics Group, Molecular and Metabolic Signalling Centre, Division of Basic Medical Sciences, St. George's, University of London, London, United Kindgom"/>
      <meta name="citation_author" content="Aniruddha Achari"/>
            <meta name="citation_author_institution" content="Randall Division of Cell and Molecular Biophysics, School of Biomedical and Health Sciences, King's College London, London, United Kindgom"/>
      <meta name="citation_author" content="Cheng Yang"/>
            <meta name="citation_author_institution" content="Rigaku Americas Corporation, The Woodlands, Texas, United States of America"/>
      <meta name="citation_author" content="Joseph D. Ferrara"/>
            <meta name="citation_author_institution" content="Rigaku Americas Corporation, The Woodlands, Texas, United States of America"/>
      <meta name="citation_author" content="L. Mark Fisher"/>
            <meta name="citation_author_institution" content="Molecular Genetics Group, Molecular and Metabolic Signalling Centre, Division of Basic Medical Sciences, St. George's, University of London, London, United Kindgom"/>
      <meta name="citation_author" content="Mark R. Sanderson"/>
            <meta name="citation_author_institution" content="Randall Division of Cell and Molecular Biophysics, School of Biomedical and Health Sciences, King's College London, London, United Kindgom"/>

    <meta name="citation_date" content="2007/3/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000301.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e301"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Structure of DNA topoisomerases.; citation_author=JM Berger; citation_journal_title=Biochim Biophys Acta; citation_volume=1400; citation_number=1; citation_pages=3-18; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=DNA topoisomerases: structure, function, and mechanism.; citation_author=JJ Champoux; citation_journal_title=Annu Rev Biochem; citation_volume=70; citation_number=2; citation_pages=369-413; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cellular roles of DNA topoisomerases: a molecular perspective.; citation_author=JC Wang; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=3; citation_number=3; citation_pages=430-440; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Type II DNA topoisomerase genes.; citation_author=WM Huang; citation_journal_title=Adv Pharmacol; citation_volume=29A; citation_number=4; citation_pages=201-225; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Protein gates in DNA topoisomerase II.; citation_author=A Maxwell; citation_journal_title=Nat Struct Biol; citation_volume=3; citation_number=5; citation_pages=109-112; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=DNA gyrase, topoisomerase IV, and the 4-quinolones.; citation_author=K Drlica; citation_author=X Zhao; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=61; citation_number=6; citation_pages=377-392; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=DNA gyrase as a drug target.; citation_author=A Maxwell; citation_journal_title=Trends Microbiol; citation_volume=5; citation_number=7; citation_pages=102-109; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Streptococcus pneumoniae DNA gyrase and topoisomerase IV: overexpression, purification, and differential inhibition by fluoroquinolones.; citation_author=XS Pan; citation_author=LM Fisher; citation_journal_title=Antimicrob Agents Chemother; citation_volume=43; citation_number=8; citation_pages=1129-1136; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Structure, molecular mechanisms, and evolutionary relationships in DNA topoisomerases.; citation_author=KD Corbett; citation_author=JM Berger; citation_journal_title=Annu Rev Biophys Biomol Struct; citation_volume=33; citation_number=9; citation_pages=95-118; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phylogenomics of type II DNA topoisomerases.; citation_author=D Gadelle; citation_author=J Filee; citation_author=C Buhler; citation_author=P Forterre; citation_journal_title=Bioessays; citation_volume=25; citation_number=10; citation_pages=232-242; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Moving one DNA double helix through another by a type II DNA topoisomerase: the story of a simple molecular machine.; citation_author=JC Wang; citation_journal_title=Q Rev Biophys; citation_volume=31; citation_number=11; citation_pages=107-144; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Conversion of DNA gyrase into a conventional type II topoisomerase.; citation_author=SC Kampranis; citation_author=A Maxwell; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=12; citation_pages=14416-14421; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Targeting of DNA gyrase in Streptococcus pneumoniae by sparfloxacin: selective targeting of gyrase or topoisomerase IV by quinolones.; citation_author=XS Pan; citation_author=LM Fisher; citation_journal_title=Antimicrob Agents Chemother; citation_volume=41; citation_number=13; citation_pages=471-474; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=DNA gyrase and topoisomerase IV are dual targets of clinafloxacin action in Streptococcus pneumoniae.; citation_author=XS Pan; citation_author=LM Fisher; citation_journal_title=Antimicrob Agents Chemother; citation_volume=42; citation_number=14; citation_pages=2810-2816; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Novel symmetric and asymmetric DNA scission determinants for Streptococcus pneumoniae topoisomerase IV and gyrase are clustered at the DNA breakage site.; citation_author=E Leo; citation_author=KA Gould; citation_author=XS Pan; citation_author=G Capranico; citation_author=MR Sanderson; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=15; citation_pages=14252-14263; citation_date=2005; " />
      <meta name="citation_reference" content="citation_number=16; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Crystal structures of Escherichia coli topoisomerase IV ParE subunit (24 and 43 kilodaltons): a single residue dictates differences in novobiocin potency against topoisomerase IV and DNA gyrase.; citation_author=S Bellon; citation_author=JD Parsons; citation_author=Y Wei; citation_author=K Hayakawa; citation_author=LL Swenson; citation_journal_title=Antimicrob Agents Chemother; citation_volume=48; citation_number=17; citation_pages=1856-1864; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The structural basis for substrate specificity in DNA topoisomerase IV.; citation_author=KD Corbett; citation_author=AJ Schoeffler; citation_author=ND Thomsen; citation_author=JM Berger; citation_journal_title=J Mol Biol; citation_volume=351; citation_number=18; citation_pages=545-561; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the breakage-reunion domain of DNA gyrase.; citation_author=JH Morais-Cabral; citation_author=AP Jackson; citation_author=CV Smith; citation_author=N Shikotra; citation_author=A Maxwell; citation_journal_title=Nature; citation_volume=388; citation_number=19; citation_pages=903-906; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Structure and mechanism of DNA topoisomerase II.; citation_author=JM Berger; citation_author=SJ Gamblin; citation_author=SC Harrison; citation_author=JC Wang; citation_journal_title=Nature; citation_volume=379; citation_number=20; citation_pages=225-232; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Structure of catabolite gene activator protein at 2.9 Å resolution suggests binding to left-handed B-DNA.; citation_author=DB McKay; citation_author=TA Steitz; citation_journal_title=Nature; citation_volume=290; citation_number=21; citation_pages=744-749; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=The structure of a CAP-DNA complex having two cAMP molecules bound to each monomer.; citation_author=JM Passner; citation_author=TA Steitz; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=22; citation_pages=2843-2847; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of a CAP-DNA complex: the DNA is bent by 90 degrees.; citation_author=SC Schultz; citation_author=GC Shields; citation_author=TA Steitz; citation_journal_title=Science; citation_volume=253; citation_number=23; citation_pages=1001-1007; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=A sign inversion mechanism for enzymatic supercoiling of DNA.; citation_author=PO Brown; citation_author=NR Cozzarelli; citation_journal_title=Science; citation_volume=206; citation_number=24; citation_pages=1081-1083; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=DNA gyrase action involves the introduction of transient double-strand breaks into DNA.; citation_author=K Mizuuchi; citation_author=LM Fisher; citation_author=MH O'Dea; citation_author=M Gellert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=77; citation_number=25; citation_pages=1847-1851; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=The C-terminal domain of DNA gyrase A adopts a DNA-bending β-pinwheel fold.; citation_author=KD Corbett; citation_author=RK Shultzaberger; citation_author=JM Berger; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=26; citation_pages=7293-7298; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The C-terminal domain of the Escherichia coli DNA gyrase A subunit is a DNA-binding protein.; citation_author=RJ Reece; citation_author=A Maxwell; citation_journal_title=Nucleic Acids Res; citation_volume=19; citation_number=27; citation_pages=1399-1405; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=A superhelical spiral in the Escherichia coli DNA gyrase A C-terminal domain imparts unidirectional supercoiling bias.; citation_author=AJ Ruthenburg; citation_author=DM Graybosch; citation_author=JC Huetsch; citation_author=GL Verdine; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=28; citation_pages=26177-26184; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Contacts between DNA gyrase and its binding site on DNA: features of symmetry and asymmetry revealed by protection from nucleases.; citation_author=A Morrison; citation_author=NR Cozzarelli; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=78; citation_number=29; citation_pages=1416-1420; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Site-specific interaction of DNA gyrase with DNA.; citation_author=LM Fisher; citation_author=K Mizuuchi; citation_author=MH O'Dea; citation_author=H Ohmori; citation_author=M Gellert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=78; citation_number=30; citation_pages=4165-4169; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Mapping the topography of DNA wrapped around gyrase by nucleolytic and chemical probing of complexes of unique DNA sequences.; citation_author=K Kirkegaard; citation_author=JC Wang; citation_journal_title=Cell; citation_volume=23; citation_number=31; citation_pages=721-729; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=The interaction of Escherichia coli topoisomerase IV with DNA.; citation_author=H Peng; citation_author=KJ Marians; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=32; citation_pages=25286-25290; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=DNA binding and nucleotide flipping by the human DNA repair protein AGT.; citation_author=DS Daniels; citation_author=TT Woo; citation_author=KX Luu; citation_author=DM Noll; citation_author=ND Clarke; citation_journal_title=Nat Struct Mol Biol; citation_volume=11; citation_number=33; citation_pages=714-720; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mechanism of inhibition of DNA gyrase by quinolone antibacterials: specificity and cooperativity of drug binding to DNA.; citation_author=LL Shen; citation_author=J Baranowski; citation_author=AG Pernet; citation_journal_title=Biochemistry; citation_volume=28; citation_number=34; citation_pages=3879-3885; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=A single point mutation in the DNA gyrase A protein greatly reduces binding of fluoroquinolones to the gyrase-DNA complex.; citation_author=CJ Willmott; citation_author=A Maxwell; citation_journal_title=Antimicrob Agents Chemother; citation_volume=37; citation_number=35; citation_pages=126-127; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Clerocidin interacts with the cleavage complex of Streptococcus pneumoniae topoisomerase IV to induce selective irreversible DNA damage.; citation_author=SN Richter; citation_author=E Leo; citation_author=G Giaretta; citation_author=B Gatto; citation_author=LM Fisher; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=36; citation_pages=1982-1991; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Processing of X-ray diffraction data collected in oscillation mode.; citation_author=Z Otwinowski; citation_author=W Minor; citation_journal_title=Method Enzymol; citation_number=37; citation_pages=307-325; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Automatic processing of rotation diffraction data from crystals of initially unknown symmetry and cell constants.; citation_author=W Kabsch; citation_journal_title=J Appl Crystallogr; citation_volume=26; citation_number=38; citation_pages=795; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Detecting and overcoming crystal twinning.; citation_author=TO Yeates; citation_journal_title=Method Enzymol; citation_number=39; citation_pages=344-360; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Crystallography & NMR system: A new software suite for macromolecular structure determination.; citation_author=AT Brunger; citation_author=PD Adams; citation_author=GM Clore; citation_author=WL DeLano; citation_author=P Gros; citation_journal_title=Acta Crystallogr D Biol Crystallogr; citation_volume=54; citation_number=40; citation_pages=905-921; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Enhancement of protein modeling by human intervention in applying the automatic programs 3D-JIGSAW and 3D-PSSM.; citation_author=PA Bates; citation_author=LA Kelley; citation_author=RM MacCallum; citation_author=MJE Sternberg; citation_journal_title=Proteins; citation_volume=45; citation_number=41; citation_pages=39-46; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Model building by comparison at CASP3: using expert knowledge and computer automation.; citation_author=PA Bates; citation_author=MJE Sternberg; citation_journal_title=Proteins; citation_volume=37; citation_number=42; citation_pages=47-54; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Domain Fishing: a first step in protein comparative modelling.; citation_author=B Contreras-Moreira; citation_author=PA Bates; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=43; citation_pages=1141-1142; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The CCP4 suite: programs for protein crystallography.; citation_journal_title=Acta Crystallogr D Biol Crystallogr; citation_volume=50; citation_number=44; citation_pages=760-763; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Implementation of molecular replacement in AMoRe.; citation_author=J Navaza; citation_journal_title=Acta Crystallogr D Biol Crystallogr; citation_volume=57; citation_number=45; citation_pages=1367-1372; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Coot: model-building tools for molecular graphics.; citation_author=P Emsley; citation_author=K Cowtan; citation_journal_title=Acta Crystallogr D Biol Crystallogr; citation_volume=60; citation_number=46; citation_pages=2126-2132; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=PROCHECK: a program to check the stereochemical quality of protein structures.; citation_author=RA Laskowski; citation_author=MW MacArthur; citation_author=DS Moss; citation_author=JM Thornton; citation_journal_title=J Appl Crystallogr; citation_volume=26; citation_number=47; citation_pages=283; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Stereochemical quality of protein structure coordinates.; citation_author=AL Morris; citation_author=MW MacArthur; citation_author=EG Hutchinson; citation_author=JM Thornton; citation_journal_title=Proteins; citation_volume=12; citation_number=48; citation_pages=345-364; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Multiple sequence alignment with the Clustal series of programs.; citation_author=R Chenna; citation_author=H Sugawara; citation_author=T Koike; citation_author=R Lopez; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=49; citation_pages=3497-3500; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=VMD: visual molecular dynamics.; citation_author=W Humphrey; citation_author=A Dalke; citation_author=K Schulten; citation_journal_title=J Mol Graph; citation_volume=14; citation_number=50; citation_pages=27-38; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=GRASP2: visualization, surface properties, and electrostatics of macromolecular structures and sequences.; citation_author=D Petrey; citation_author=B Honig; citation_journal_title=Method Enzymol; citation_volume=374; citation_number=51; citation_pages=492-509; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Calculation of the electric potential in the active site cleft due to α–helix dipoles.; citation_author=J Warwicker; citation_author=HC Watson; citation_journal_title=J Mol Biol; citation_volume=157; citation_number=52; citation_pages=671-679; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=The PyMOL molecular graphics system.; citation_author=WL DeLano; citation_number=53; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000301" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Breakage-Reunion Domain of Streptococcus pneumoniae Topoisomerase IV: Crystal Structure of a Gram-Positive Quinolone Target"/>
    <meta name="twitter:description" content="The 2.7 &Aring; crystal structure of the 55-kDa N-terminal breakage-reunion domain of topoisomerase (topo) IV subunit A (ParC) from Streptococcus pneumoniae, the first for the quinolone targets from a gram-positive bacterium, has been solved and reveals a &lsquo;closed&rsquo; dimer similar in fold to Escherichia coli DNA gyrase subunit A (GyrA), but distinct from the &lsquo;open&rsquo; gate structure of Escherichia coli ParC. Unlike GyrA whose DNA binding groove is largely positively charged, the DNA binding site of ParC exhibits a distinct pattern of alternating positively and negatively charged regions coincident with the predicted positions of the grooves and phosphate backbone of DNA. Based on the ParC structure, a new induced-fit model for sequence-specific recognition of the gate (G) segment by ParC has been proposed. These features may account for the unique DNA recognition and quinolone targeting properties of pneumococcal type II topoisomerases compared to their gram-negative counterparts."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000301.g006"/>

  <meta property="og:title" content="Breakage-Reunion Domain of Streptococcus pneumoniae Topoisomerase IV: Crystal Structure of a Gram-Positive Quinolone Target" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000301" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=11'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2520'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=705&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000301">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000301" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000301&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Breakage-Reunion Domain of <i>Streptococcus pneumoniae</i> Topoisomerase IV: Crystal Structure of a Gram-Positive Quinolone Target
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ivan Laponogov
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Ivan Laponogov, Dennis A. Veselkov, Maninder K. Sohi, Xiao-Su Pan</p>

              
              

                <p>Affiliation:
                  Randall Division of Cell and Molecular Biophysics, School of Biomedical and Health Sciences, King's College London, London, United Kindgom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dennis A. Veselkov
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Ivan Laponogov, Dennis A. Veselkov, Maninder K. Sohi, Xiao-Su Pan</p>

              
              

                <p>Affiliation:
                  Randall Division of Cell and Molecular Biophysics, School of Biomedical and Health Sciences, King's College London, London, United Kindgom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maninder K. Sohi
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Ivan Laponogov, Dennis A. Veselkov, Maninder K. Sohi, Xiao-Su Pan</p>

              
              

                <p>Affiliation:
                  Randall Division of Cell and Molecular Biophysics, School of Biomedical and Health Sciences, King's College London, London, United Kindgom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiao-Su Pan
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Ivan Laponogov, Dennis A. Veselkov, Maninder K. Sohi, Xiao-Su Pan</p>

              
              

                <p>Affiliation:
                  Molecular Genetics Group, Molecular and Metabolic Signalling Centre, Division of Basic Medical Sciences, St. George's, University of London, London, United Kindgom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Aniruddha Achari, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Randall Division of Cell and Molecular Biophysics, School of Biomedical and Health Sciences, King's College London, London, United Kindgom
                </p>

                <p>
                    Current address: Raytheon IS, EM10, NASA Marshall Space Flight Center, Huntsville, Alabama, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Cheng Yang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Rigaku Americas Corporation, The Woodlands, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joseph D. Ferrara, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Rigaku Americas Corporation, The Woodlands, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            L. Mark Fisher
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mark.sanderson@kcl.ac.uk">mark.sanderson@kcl.ac.uk</a> (MS); <a href="mailto:lfisher@sgul.ac.uk">lfisher@sgul.ac.uk</a> (LF)</p>

                <p>Affiliation:
                  Molecular Genetics Group, Molecular and Metabolic Signalling Centre, Division of Basic Medical Sciences, St. George's, University of London, London, United Kindgom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark R. Sanderson
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mark.sanderson@kcl.ac.uk">mark.sanderson@kcl.ac.uk</a> (MS); <a href="mailto:lfisher@sgul.ac.uk">lfisher@sgul.ac.uk</a> (LF)</p>

                <p>Affiliation:
                  Randall Division of Cell and Molecular Biophysics, School of Biomedical and Health Sciences, King's College London, London, United Kindgom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000301</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000301" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000301" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000301" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000301" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000301-g001" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000301-g002" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000301-g003" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000301-g004" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000301-g005" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000301-t001" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000301-g006" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000301">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000301" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The 2.7 Å crystal structure of the 55-kDa N-terminal breakage-reunion domain of topoisomerase (topo) IV subunit A (ParC) from <em>Streptococcus pneumoniae</em>, the first for the quinolone targets from a gram-positive bacterium, has been solved and reveals a ‘closed’ dimer similar in fold to <em>Escherichia coli</em> DNA gyrase subunit A (GyrA), but distinct from the ‘open’ gate structure of <em>Escherichia coli</em> ParC. Unlike GyrA whose DNA binding groove is largely positively charged, the DNA binding site of ParC exhibits a distinct pattern of alternating positively and negatively charged regions coincident with the predicted positions of the grooves and phosphate backbone of DNA. Based on the ParC structure, a new induced-fit model for sequence-specific recognition of the gate (G) segment by ParC has been proposed. These features may account for the unique DNA recognition and quinolone targeting properties of pneumococcal type II topoisomerases compared to their gram-negative counterparts.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Laponogov I, Veselkov DA, Sohi MK, Pan X-S, Achari A, et al.  (2007) Breakage-Reunion Domain of <em>Streptococcus pneumoniae</em> Topoisomerase IV: Crystal Structure of a Gram-Positive Quinolone Target. PLoS ONE 2(3):
          e301.
            doi:10.1371/journal.pone.0000301</p><p><strong>Academic Editor: </strong>Haiwei Song, Institute of Molecular and Cell Biology, Singapore</p><p><strong>Received:</strong> January 5, 2007; <strong>Accepted:</strong> February 21, 2007; <strong>Published:</strong> March 21, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>Project Grant C16747 from the Biotechnology and Biological Sciences Research Council of the United Kingdom (to L.M. Fisher and M.R. Sanderson). I. Laponogov was supported by a scholarship from the School of Biomedical and Health Sciences and ORS Award. Xiao-Su Pan, Dennis A. Veselkov and this work were funded by Project Grant C16747 from the Biotechnology and Biological Sciences Research Council of the United Kingdom (to L.M. Fisher and M.R. Sanderson).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>DNA topoisomerases are enzymes of vital cellular importance due to their ability to alter DNA topology. Replication of duplex DNA or recombination between DNA duplexes may cause the formation of DNA supercoils, knots and catenanes. The major role of topoisomerases is to control DNA topology by transient breakage of DNA strands. This activity is achieved through the formation of covalent enzyme-DNA complexes linked via phosphotyrosine bonds. Type I topoisomerases act on one strand of DNA whereas type II enzymes pass a DNA duplex through a double-strand DNA break, resulting in changes of DNA linking number in units of 1 and 2, respectively <a href="#pone.0000301-Berger1">[1]</a>–<a href="#pone.0000301-Wang1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Type II topoisomerases comprise a family of structurally and evolutionarily conserved enzymes common to all organisms <a href="#pone.0000301-Huang1">[4]</a> and have been implicated in a variety of intracellular processes including DNA replication, transcription and chromosome segregation <a href="#pone.0000301-Maxwell1">[5]</a>. Because of their importance to the cell life cycle, these enzymes have been a subject of intense drug development studies and were identified as targets for a large number of natural toxins, antimicrobial agents and anti-tumour therapeutics <a href="#pone.0000301-Drlica1">[6]</a>–<a href="#pone.0000301-Pan1">[8]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Based on sequence and structural criteria, type II topoisomerases are divided into two subclasses. The most common subclass, type IIA, includes the bacterial enzymes topo IV and gyrase, and eukaryotic topo II. Type IIB topos are structurally and biochemically distinct, and comprise a single family member, topo VI <a href="#pone.0000301-Champoux1">[2]</a>, <a href="#pone.0000301-Corbett1">[9]</a>–<a href="#pone.0000301-Wang2">[11]</a>. All members of the type IIA subclass exhibit significant sequence and mechanistic similarity.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Bacterial genomes usually encode two type IIA enzymes, topo IV and DNA gyrase, each composed of two subunits (ParC and ParE in topo IV, GyrA and GyrB in gyrase) assembled into a functional heterotetramer <em>i.e.</em> C<sub>2</sub>E<sub>2 </sub>and A<sub>2</sub>B<sub>2</sub>, respectively <a href="#pone.0000301-Drlica1">[6]</a>. Both enzymes act by passing a duplex DNA segment through a transient double-stranded DNA break in another helical region. In the currently accepted model <a href="#pone.0000301-Corbett1">[9]</a>, the open enzyme clamp binds a DNA segment termed the ‘gate’ or G-segment. The N-terminal ATPase domains of the ParE (GyrB) subunits dimerize upon ATP binding capturing the DNA duplex to be transported (T-segment). The T-segment is then passed through a transient break in the G-segment (opened by the N-terminal ParC (GyrA) domains), the DNA is resealed and the T-segment released through a protein gate prior to resetting of the enzyme to the open clamp form.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Although topo IV and gyrase share a high degree of similarity, their cellular functions are different. Gyrase controls DNA supercoiling and relieves topological stress arising from the translocation of transcription and replication complexes along DNA. Gyrase is unique in its ability to introduce negative supercoiling into the DNA. By contrast, topo IV is a decatenating enzyme that resolves interlinked daughter chromosomes following DNA replication. The functional differences between gyrase and topo IV are attributed to the non-specific binding of the C-terminal domain of the GyrA subunit that wraps DNA presenting a neighbouring T-segment for passage through the G-gate. Removal of this domain converts gyrase into an enzyme with topo IV-like activities, i.e. ATP-dependent DNA relaxation and decatenation <a href="#pone.0000301-Kampranis1">[12]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>In addition to their unusual mechanisms, topo IV and gyrase are also targets of clinically important quinolone drugs. Quinolones exert their bactericidal effects by interfering with DNA breakage-reunion by topo IV and gyrase leading to double-stranded DNA breaks at specific sites. Recent studies have uncovered fundamental differences in quinolone action in gram-positive vis-à-vis gram-negative bacteria. For example, though gyrase is the universal target of quinolones in gram-negative bacteria, topo IV is the primary (or dual) target of many quinolones in gram-positive pathogens such as <em>Streptococcus pneumoniae</em>, a major cause of pneumonia and other serious infections <a href="#pone.0000301-Pan2">[13]</a>, <a href="#pone.0000301-Pan3">[14]</a>. Second, pneumococcal topo IV is more sensitive than gyrase to quinolone inhibition in vitro, the inverse of that seen in <em>E. coli</em> <a href="#pone.0000301-Leo1">[15]</a>. Third, quinolones trap <em>S. pneumoniae</em> topo IV (and gyrase) at DNA sites (G-segments) that are distinct in sequence from those observed for <em>E. coli</em> topo IV and gyrase <a href="#pone.0000301-Leo1">[15]</a>. These unique features emphasize the need to analyze the structure of <em>S. pneumoniae</em> topo IV and how it binds quinolones and DNA. Progress in these areas will be important in understanding ParC-mediated resistance to quinolones and in developing dual targeting drugs that minimize the emergence of resistance <a href="#pone.0000301-Pan2">[13]</a>, <a href="#pone.0000301-CBSN1">[16]</a>, <a href="#pone.0000301-Bellon1">[17]</a>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Digestion with trypsin revealed that pneumococcal ParC is organized into a 53-kDa N-terminal domain (ParC53, residues 23-487) and a compact 17-kDa C-terminal region (<a href="#pone-0000301-g001">Figure 1A</a>). ParC53 was enzymatically inactive whereas a recombinant ParC55 fragment (residues 1-490), when complemented with topo IV ParE, reconstituted quinolone-promoted DNA cleavage (data not shown). The DNA cleavage specificity of the ParC55-ParE complex was identical to that of pneumococcal topo IV and exhibited the same unique base recognition preferences of −4G/+8C, −2A/+6T and+1G/+4C (relative to the DNA scission site between nucleotide −1 and+1). These determinants are very different from those of <em>E. coli</em> topo IV (preferences −1(A/G) and+1(T/A)) and <em>E. coli</em> gyrase (a degenerate 20-bp consensus) <a href="#pone.0000301-Leo1">[15]</a>. Sequence alignment with the N-terminal domains of <em>E. coli</em> GyrA and ParC whose crystal structures have been solved <a href="#pone.0000301-Corbett2">[18]</a>, <a href="#pone.0000301-MoraisCabral1">[19]</a> shows greater similarity of <em>S. pneumoniae</em> ParC55 with <em>E. coli</em> GyrA and <em>S. pneumoniae</em> GyrA (forms a GyrA58 (residues 17–529) tryptic fragment via GyrA60, <a href="#pone-0000301-g001">Figure 1B</a>) than with <em>E. coli</em> ParC (<a href="#pone-0000301-g001">Figure 1C</a>). However, these comparisons do not explain the distinctive enzyme preferences in G-segment recognition. To gain insight into the mechanism of <em>S. pneumoniae</em> topo IV and its interaction with DNA and quinolones, the structure of the ParC55 fragment has been solved (<a href="#pone-0000301-g002">Figure 2A</a>) and the model was refined to R = 22.30% (R-free = 27.55%) at 2.7 Å resolution using wild-type ParC55 data (twinning fraction = 0.323, twinning operator = (k,-h,l)). The crystallization of ParC55, the quinolone target from gram-positive bacteria, and X-ray analysis were first reported by our group in Acta Cryst. (2005) A61, C176.</p>
<div class="figure" id="pone-0000301-g001"><div class="img"><a name="pone-0000301-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000301.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000301.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Comparison of type II topoisomerases from <em>S. pneumoniae</em> and <em>E. coli</em>. Schematic domain organization of (A) <em>S. pneumoniae</em> topoisomerase IV, (B) <em>S. pneumoniae</em> gyrase (with the proteolytic fragments generated by trypsin digestion superimposed) and (C) graphical representation of the sequence similarity scores for 500-residue N-terminal fragments of ParC (P72525) and GyrA (P72524) from <em>S. pneumoniae</em> and equivalent fragments of ParC (P0AFI2) and GyrA (P0AES4) from <em>E. coli</em> calculated using ClustalW <a href="#pone.0000301-Chenna1">[49]</a>.</p>
<span>doi:10.1371/journal.pone.0000301.g001</span></div><div class="figure" id="pone-0000301-g002"><div class="img"><a name="pone-0000301-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000301.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000301.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>Structure of the ParC55 dimer and illustration of structurally related <em>E. coli</em> ParC and GyrA proteins. (A) Orthogonal views of the ParC55 biological dimer from <em>S. pneumoniae</em>. (B) Structure of the N-terminal region of <em>E. coli</em> ParC (18) (1ZVU) equivalent to ParC55 fragment. (C) Structure of the N-terminal fragment (GyrA59) of <em>E. coli</em> GyrA <a href="#pone.0000301-MoraisCabral1">[19]</a> (1AB4). In (A), (B) and (C) the ‘towers’ and the CAP-like domains are shown in ice blue; the ‘tails’ along with adjacent helices α14, α18 and α19 are in ochre; the helix α4 in red; the helix α3 in cyan and the 100-122 loop in yellow. The active-site tyrosines are shown in green. Residues Ser 79 and Asp 83 responsible for drug-resistance upon mutation are in purple. (D) Schematic conversion of ParC55 from ‘closed’ (red) to ‘open’ (blue) conformation on the basis of the <em>E. coli</em> ParC structure. Panels were rendered using VMD <a href="#pone.0000301-Humphrey1">[50]</a> and Pov-Ray.</p>
<span>doi:10.1371/journal.pone.0000301.g002</span></div>
<h4>Crystal structure of ParC55: a closed dimer with a distinctively charged DNA binding groove</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>ParC55 forms a biological dimer in a ‘closed’ conformation in this crystal form generating a ring-like structure with outer dimensions of 100×100×50 Å and with a central hole approximately 30 Å in diameter, which is wide enough to pass a duplex DNA (<a href="#pone-0000301-g002">Figure 2A</a>). This is in contrast to the ParC structure from <em>E. coli</em> which adopts an ‘open’ dimer conformation <a href="#pone.0000301-Corbett2">[18]</a> (<a href="#pone-0000301-g002">Figure 2B</a>), but similar to the closed dimer of GyrA59 from <em>E. coli</em> <a href="#pone.0000301-MoraisCabral1">[19]</a> (<a href="#pone-0000301-g002">Figure 2C</a>). Each monomer of ParC55 contains two distinct regions labeled ‘head’ and ‘tail’ and exhibits an overall structural fold similar to the one observed for other topoisomerases <a href="#pone.0000301-Corbett2">[18]</a>–<a href="#pone.0000301-Berger2">[20]</a>. The N-terminal proximal head contains the CAP-like (catabolite gene activator protein <a href="#pone.0000301-McKay1">[21]</a>–<a href="#pone.0000301-Schultz1">[23]</a>) DNA-binding domain and the ‘tower’. The CAP-like domain is composed of a helix-turn-helix structural motif (helices α3 and α4) typical for DNA-binding proteins, with the active site Arg 117 and Tyr 118 on the neighboring ‘100-122’ loop (<a href="#pone-0000301-g002">Figure 2A</a>). The residues Ser 79 and Asp 83, which upon mutation give rise to quinolone-resistant pneumococci, are located primarily within the helix α4 in close proximity to the active-site tyrosine and are oriented towards the predicted position of the bound DNA. The ‘tower’ is positioned on the top of the CAP-like domain and has largely α-helical and β-sheet structural organization. High B-factor values and poor definition of several loops indicate a high level of flexibility in this region. In the closed conformation, the ParC55 monomers form a tight contact between the α3 helices with a mean center-to-center distance of 10 Å resulting in the protein-protein interface with a buried surface area of 706 Å<sup>2</sup>. This is less than in the case of <em>E. coli</em> GyrA where the monomers form a protein-protein interface with a buried surface area of 1,380 Å<sup>2</sup>. The ‘recognition’ α4 helices are further apart from each other by 5 Å in the case of the ParC55 dimer compared to the <em>E. coli</em> GyrA dimer.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The ‘tail’ has a primarily α-helical core composition and forms the so-called ‘primary’ dimer interface with a total buried surface area of 2,001 Å<sup>2</sup>. It is very similar to equivalent domains from <em>E. coli</em> GyrA and <em>E. coli</em> ParC in terms of the overall core structure, though its outer appearance is different from the ‘tail’ present in <em>E. coli</em> GyrA due to a lack of extra surface loops (<a href="#pone-0000301-g002">Figure 2A, C</a>). Two long helices (α14 and α18) emanate from the ‘tail’ domain and join it to the ‘head’ domain (α14 directly and α18 through a flexible loop and helix α19, respectively).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Superposition of the ‘closed’ <em>S. pneumoniae</em> ParC55 and ‘open’ <em>E. coli</em> ParC dimers reveals several significant structural differences potentially relevant to topo IV catalysis (<a href="#pone-0000301-g002">Figure 2D</a>). The process of changing conformation from the ‘closed’ form to an ‘open’ one will involve movement of the ‘heads’ away from each other, increasing the average distance between them by approximately 30 Å and rotating them by 15 degrees relative to the tail protein dimer interface and 20 degrees relative to the vertical axis of the dimer. This movement is primarily based on changes in relative orientation of the long helices (α14, α18 and α19) which join the ‘heads’ to the ‘tails’ (<a href="#pone-0000301-g002">Figure 2D</a>) <a href="#pone.0000301-Corbett2">[18]</a>. Interestingly, in addition to several positional changes in the α-helices and β-sheets forming the CAP-like domains, perhaps the most significant difference between <em>S. pneumoniae</em> and <em>E. coli</em> ParC structures is in the fold of the 100-122 loop containing active-site Arg 117 and Tyr 118 residues (<a href="#pone-0000301-g002">Figure 2B</a> and <a href="#pone-0000301-g003">Figure 3A</a>). In the ‘closed’ <em>S. pneumoniae</em> ParC structure, the loop is positioned vertically and stands on the side of the DNA-binding groove orienting the active-site tyrosine towards the expected position of the DNA backbone and exposing the active-site helices (α3 and α4) to the predicted protein-DNA interface. By contrast, in the <em>E. coli</em> ParC, this loop forms two additional turns one of which flips the active-site tyrosine upside-down and moves it away from the dimer interface and surface of the DNA-binding groove. In addition, the loop bends into the DNA-binding groove and covers the active-site helices α3 and α4. It seems possible that the open <em>E. coli</em> ParC structure corresponds to the ‘DNA-free’ form of topoisomerase IV and that the evident flexibility of the 100-122 loop is necessary to allow DNA binding to the closed enzyme conformation.</p>
<div class="figure" id="pone-0000301-g003"><div class="img"><a name="pone-0000301-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000301.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000301.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Structural comparison between active sites of topoisomerases from gram-positive and gram-negative bacteria. (A) Stereo view of the superposition of the active site of ParC from <em>S. pneumoniae</em> (red) and ParC from <em>E. coli</em> (violet). (B) Stereo view of the superposition of the active sites of ParC from <em>S. pneumoniae</em> (red) and GyrA from <em>E. coli</em> (blue). The active-site tyrosines and arginines are represented in CPK mode. Sites responsible for drug-resistance when mutated are represented in Licorice mode. Panels were rendered using VMD <a href="#pone.0000301-Humphrey1">[50]</a> and Pov-Ray.</p>
<span>doi:10.1371/journal.pone.0000301.g003</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p><em>S. pneumoniae</em> ParC55 folds into a similar closed conformation to that of <em>E. coli</em> GyrA (<a href="#pone-0000301-g003">Figure 3B</a>) However, interestingly, the DNA binding site formed by the pneumococcal ParC55 dimer differs markedly in its electrostatic charge distribution from both the <em>E. coli</em> ParC and GyrA dimers (<a href="#pone-0000301-g004">Figure 4</a>). In particular, unlike <em>E. coli</em> GyrA which has uniform positive charge within the DNA-binding groove (except for the DNA scission active site and sides of the neighboring ‘towers’), the DNA-binding groove of <em>S. pneumoniae</em> ParC exhibits a pattern of alternating areas of positive and negative charge (compare <a href="#pone-0000301-g004">Figures 4A and 4C</a>). These areas correspond to the predicted positions of the backbone phosphates and of the major/minor grooves of the bound DNA strand (see below). To determine whether these distinctive charge distributions were retained across bacterial species, we modeled a number of GyrA and ParC sequences on to the <em>E. coli</em> GyrA structure (<a href="#pone.0000301.s001">Figure S1</a>). Modeling of <em>S. pneumoniae</em> ParC recapitulated the experimental charge distribution seen in ParC55. <em>S. pneumoniae</em> GyrA conformed almost exactly to the <em>E. coli</em> GyrA pattern. The ParC and GyrA proteins of <em>Staphylococcus aureus</em> behaved like their pneumococcal counterparts. There were also clear differences between the cognate ParC and GyrA proteins of the Gram-negative species <em>Haemophilus influenzae</em> and less so for <em>Neisseria gonorrhoeae</em>. It is possible that charge distribution differences relate to functional differences in DNA recognition and processing by the two enzymes. Thus, <em>E. coli</em> GyrA uses its C-terminal domain to bend DNA up to 180° presenting a T-segment <em>in cis</em> for passage through the G-segment and thereby introducing negative supercoils <a href="#pone.0000301-Brown1">[24]</a>–<a href="#pone.0000301-Ruthenburg1">[28]</a>. Unlike gyrase which has a 120–150-bp DNA footprint <a href="#pone.0000301-Morrison1">[29]</a>–<a href="#pone.0000301-Kirkegaard1">[31]</a>, topo IV lacks negative supercoiling activity and generally binds a short region of G-segment DNA (~30 bp) <a href="#pone.0000301-Peng1">[32]</a>. The C-terminal DNA binding domain of ParC is used to guide intermolecular T segment translocation facilitating the decatenation of linked DNA molecules <a href="#pone.0000301-Corbett2">[18]</a>, <a href="#pone.0000301-Peng1">[32]</a>. It remains to be determined whether differences in DNA substrate recognition and processing by topo IV and gyrase are partially facilitated by differences in charge distribution in the N-terminal DNA binding site of ParC/GyrA.</p>
<div class="figure" id="pone-0000301-g004"><div class="img"><a name="pone-0000301-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000301.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000301.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Electrostatic surface potential calculation. GRASP2 <a href="#pone.0000301-Petrey1">[51]</a>, <a href="#pone.0000301-Warwicker1">[52]</a> electrostatic surface potentials within the DNA-binding grooves calculated for equivalent N-terminal fragments of: (A) <em>S. pneumoniae</em> ParC (ParC55), (B) <em>E. coli</em> ParC, (C) <em>E. coli</em> GyrA. The <em>E. coli</em> ParC structure was manually brought into the ‘closed’ conformation, but the experimentally determined fold of the active site was preserved. Negatively charged surfaces are in red and positively charged surfaces are in blue.</p>
<span>doi:10.1371/journal.pone.0000301.g004</span></div>

<h4>Induced-fit model for sequence-specific DNA binding and cleavage by ParC</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In the ParC55 dimer, the catalytic Tyr 118 residues are 30 Å apart and are not therefore optimally placed for attack of phosphodiester bonds in the G-segment required for covalent ParC-DNA linkage and DNA breakage. This situation also applies to the <em>E. coli</em> GyrA dimer for which it was proposed that the G-segment is bound in a distorted (perhaps single-stranded) form placing phosphodiester bonds close enough for tyrosine attack <a href="#pone.0000301-MoraisCabral1">[19]</a>. Based on the ParC55 structure and the apparent conformational flexibility of the 100-122 loop (<a href="#pone-0000301-g003">Figure 3</a>), we suggest a possible alternative to the previously proposed DNA “bubble” mechanism <a href="#pone.0000301-Leo1">[15]</a>. In the new model, the catalytic tyrosines are moved to attack DNA as a natural consequence of G-segment recognition and binding by the α3/α4 helices and by the 100-122 loop. Based on a knowledge of the charge distribution for the DNA-binding groove of <em>S. pneumoniae</em> ParC, differences in folding the 100-122 loop between <em>S. pneumoniae</em> ParC and <em>E. coli</em> ParC and the known DNA sequence preferences for <em>S. pneumoniae</em> ParC, we have modeled the protein-DNA binding, DNA sequence recognition and cleavage process using a 24-mer duplex corresponding to the strong E binding site of topo IV <a href="#pone.0000301-Leo1">[15]</a> (<a href="#pone-0000301-g005">Figure 5</a> and <a href="#pone.0000301.s003">Movie S1</a>). First, approximately two turns of DNA helix are captured by the middle regions of the protein heads positioning the phosphate backbone of the DNA molecule against the positively charged grooves on the surface of the CAP-like domains and neighboring towers, with the major groove of the DNA against negatively charged middle regions of the CAP-like domains. The two β-sheets with a joining loop comprising amino acids Gly 166 to Pro 179 form a tight contact with the major groove of the DNA (<a href="#pone-0000301-g005">Figure 5A</a>). In the next stage, the helix α4 enters the minor groove of the DNA molecule constituting the first step in DNA sequence recognition. The possibility of the helix-turn-helix motif binding the minor groove has previously been reported for human DNA repair protein <em>O</em><sup>6</sup>-alkylguanine-DNA alkyltransferase (AGT) <a href="#pone.0000301-Daniels1">[33]</a>. Alternative binding of the helix α4 to either of the neighboring areas of the major groove of the DNA seems less attractive as it would require a strong distortion of the active site which would physically push the active site tyrosines away from the target backbone phosphates.</p>
<div class="figure" id="pone-0000301-g005"><div class="img"><a name="pone-0000301-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000301.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000301.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>DNA binding, recognition and cleavage by <em>S. pneumoniae</em> ParC. Initial (A) and final (B) stages of DNA binding and recognition are modeled and shown as orthogonal stereo representations of the protein-DNA complex (top) together with the electrostatic surface potential of the DNA binding groove (bottom). The ParC dimer is shown in cartoon mode; the helix α4 is in red, the helix α3 in cyan, the 100-122 loop in yellow and Gly 166 - Pro 179 domain in orange. The active-site tyrosines are shown in blue color using CPK mode and indicated by asterisks (magenta for unbound and yellow for bound to the DNA states). The target 5′-end phosphates of the DNA molecule are indicated by circles in magenta. The DNA molecule is represented as a molecular surface model; the backbone is in pink, the cleavage points in purple, the base pairs involved in sequence specific recognition in green and non-specific base pairs in silver. (C) Stereo view of the active site of the modelled protein-DNA complex of <em>S. pneumoniae</em> ParC after cleavage and separation of the DNA fragments in cartoon (left) and surface (right) representation. Topo IV cleaves DNA with a 4-base stagger and the active site tyrosines are covalently attached to the 5′-ends of the DNA. Ser 79 and Asp 83 which upon mutation lead to quinolone resistance in bacteria are shown in purple, the helix α4 is in cyan, the helix α3 in yellow, the 100-122 loop with active site tyrosine in red and Gly 166-Pro 179 domain in orange. DNA molecule backbone is shown in yellow and side chains are in blue. The position of the covalent bond between the active-site tyrosine and the target 5′-end DNA backbone phosphate is indicated by a yellow asterisk in a magenta circle. Panels were rendered using VMD <a href="#pone.0000301-Humphrey1">[50]</a>, Pov-Ray and PyMOL <a href="#pone.0000301-DeLano1">[53]</a>.</p>
<span>doi:10.1371/journal.pone.0000301.g005</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Driven by the process of the helix α4 binding to the minor groove of the DNA, the helix α3 reorients itself moving into the major groove of the DNA bringing the 100-122 loop closer to the same groove. Then, the negatively charged 100-122 loop folds onto and binds to the DNA major groove providing the second step of DNA sequence recognition. As a direct consequence of DNA recognition and binding by the 100-122 loop, the active site tyrosine is moved close to the phosphate backbone of the DNA allowing DNA cleavage to occur (<a href="#pone-0000301-g005">Figure 5B</a>). The process of protein-DNA binding most likely involves slight opening of the protein dimer interface (by approximately 9 Å) and some bending of the DNA molecule towards the active site of the protein (an estimated radius of DNA curvature of about 70 Å).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The model attractively accommodates key features of topo IV action. First, unlike the proposal for DNA-binding to <em>E. coli</em> GyrA, the model requires minimal distortion of protein and DNA molecules <a href="#pone.0000301-MoraisCabral1">[19]</a>. Electrostatic potential calculated for the final stage of the DNA binding process shows that the DNA backbone is properly positioned against the positively charged regions of the protein with the DNA major/minor grooves positioned against the negatively charged regions of the DNA-binding groove along the entire DNA molecule (<a href="#pone-0000301-g005">Figure 5B</a>). Second, it explains the mechanism of selective DNA cleavage as arising from the DNA sequence being ‘sampled’ both from the major groove (mainly by the 100-122 loop and to some extent by the helix α3) and from the minor groove (by the helix α4). Nucleotides+4, +6 and+8 (+1, −2 and −4 on the opposite strand) are placed close to the 100-122 loop and the helices α3/α4 (<a href="#pone-0000301-g005">Figure 5C</a>) consistent with the strong DNA cleavage determinants observed at these positions for pneumococcal topo IV <a href="#pone.0000301-Leo1">[15]</a>. Involvement of the 100-122 loop in DNA sequence recognition may also help explain the divergence of the DNA sequence preferences between ParC and GyrA from different organisms despite very high conservation of the residues within helices α3 and α4 which are involved in protein-DNA contact formation (<a href="#pone.0000301.s002">Figure S2</a>).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Finally, the ParC55 structure and modeling of DNA recognition are relevant to quinolone action. At the final stages of DNA binding and cleavage (<a href="#pone-0000301-g005">Figure 5C</a>), the quinolone resistance sites (Ser 79 and Asp 83, which are located within the helix α4 bound to the minor groove of the DNA molecule) will be positioned next to the 3′-end of the cleaved DNA strand with Ser 79 being closest to the point of cleavage (<a href="#pone-0000301-g005">Figure 5C</a>). It is known that quinolones bind strongly to topoisomerase-DNA complexes stabilizing the cleaved form, but very weakly to the protein or duplex DNA alone <a href="#pone.0000301-Shen1">[34]</a>, <a href="#pone.0000301-Willmott1">[35]</a>. Moreover, quinolones preferentially bind single-stranded DNA <a href="#pone.0000301-Shen1">[34]</a>. These features indicate that the 3′-end of the cleaved DNA (-1 position) is a potential site for quinolone binding due to likely distortion of the double-stranded DNA at the cleavage point, as supported by clerocidin labeling experiments <a href="#pone.0000301-Richter1">[36]</a>. Sequence homology (<a href="#pone-0000301-g001">Figure 1C</a>) and similarity in the disposition of mutational hotspots in ParC and GyrA (<a href="#pone-0000301-g003">Figure 3B</a>) suggest that selective quinolone targeting of ParC in <em>S. pneumoniae</em> and GyrA in <em>E. coli</em> arises from structural conservation of high affinity drug binding sites. The <em>S. pneumoniae</em> ParC55 structure presented here provides a basis for approaching key aspects of DNA recognition and quinolone action in gram-positive bacteria.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Trypsin digestion of <em>S. pneumoniae</em> ParC and GyrA</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>ParC protein (10 µg) was incubated for 1 hour at 37°C with bovine pancreatic trypsin (0.25 to 4 µg) in reaction buffer containing 50 mM Tris-HCl, pH 7.5, 55 mM KCl, 2 mM MgCl<sub>2</sub>, 5 mM DTT and 10% glycerol (total volume of 20 µl). Reactions were stopped by addition of SDS to a final concentration of 1% followed by boiling for 5 minutes. Protein products were separated in a NuPAGE<sup>TM</sup> 4–12% Bis-Tris gradient gel run at 100 Volts for 2.5 hours and stained with Coomassie blue. Proteins were electroblotted onto polyvinylidene fluoride membrane and their N-terminal sequences were determined using the Edman procedure developed by Dr. Arthur Moir, University of Sheffield.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>For tryptic digestion of <em>S. pneumoniae</em> GyrA protein, a time course was performed. The reaction mixture contained 90 µg of GyrA and 2.25 µg of trypsin in the same digestion buffer used for ParC (total volume of 150 µl). At 0, 0.5, 1, 2, 5, 10 and 30 min, a 20 µl aliquot was removed and the reaction was stopped. Proteins were analysed by electrophoresis and sequenced as described for ParC. Recombinant ParC, ParE, GyrA and GyrB were prepared as described previously <a href="#pone.0000301-Pan1">[8]</a>.</p>


<h4>Expression of <em>S. pneumoniae</em> ParC55</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>A 1.4-kb fragment of the <em>parC</em> gene of <em>S. pneumoniae</em> strain 7785 was amplified by PCR using forward primer N6894, 5′-TGGGCTTTGTAT<span class="underline">CATATG</span>TCTAAC (NdeI site overlapping the initiation ATG codon is underlined) and reverse primer VPC4 (5′-TAGCTGTATCAAT<span class="underline">CTCGAG</span>T-GCTTTCGCAG, <em>parC</em> nucleotide position 1482 to 1452, engineered XhoI site underlined). The NdeI-XhoI digested PCR product was ligated into pET29a, yielding an expression plasmid that in <em>E. coli</em> inducibly expressed the 55.5 kDa N-terminal ParC domain (ParC55). The protein constitutes residues 1-490 of <em>S. pneumoniae</em> ParC bearing a C-terminal hexahistidine tag and a single conserved amino acid substitution (I489L) necessary for plasmid construction. Selenomethionine-substituted ParC55 was prepared by induction in <em>E. coli</em> strain B834(DE3)pLysS (Novagen). To produce ParC55 Cys426, the <em>parC</em> Arg426 codon of the expression plasmid was altered using the Quikchange site-directed mutagenesis kit (Stratagene). Mutant proteins were overexpressed in <em>E. coli</em> and purified by Ni<sup>2+</sup>-NTA chromatography using the same conditions as for wild-type ParC55.</p>


<h4>Crystallization, data collection, structure solution and refinement</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The first crystallization of ParC55, the quinolone target from gram-positive bacteria, and preliminary X-ray data analysis were reported by our group (in Acta Cryst. (2005) A61, C176).</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>The purified ParC55 protein was dialyzed against 20 mM Tris-HCl, pH 7.0, 200 mM NaCl, 10% glycerol, 1 mM β-mercaptoethanol, 0.05% NaN<sub>3</sub> then concentrated to 3-4 mg/ml using PEG 35,000 at 4°C. ParC55 was crystallized by hanging drop vapor diffusion in 24-well Limbro plates using 0.5 ml reservoir volumes. Hanging drops were formed by mixing equal volumes of protein and crystallization solutions. Crystals grew at ambient temperature within 3 to 10 days and diffracted to 2.7 Å. The best diffracting wild-type ParC55 crystals were grown from 100 mM Tris-HCl, pH 5.5, 200 mM NaCl, 1 mM β-mercaptoethanol, 0.05% NaN<sub>3</sub> and 10% of 1:1 ethanol-isopropanol as precipitant. Cys426 mutant was crystallized from 100 mM Tris-HCl, pH 6.5, 200 mM NaCl, 1 mM β-mercaptoethanol, 0.05% NaN<sub>3</sub> using 4–8% PEG 400 as a precipitant and 1 mM of hexamine cobalt chloride as an additive.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>The crystals were soaked for 3 to 5 seconds in cryoprotectant solutions composed of either the crystallization solutions plus 25% glycerol or 30% MPD, 150 mM NaCl, 0.05% NaN<sub>3</sub>, 20 mM Tris-HCl, pH 7.0 and immediately frozen in a nitrogen cryostream.</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>Data sets were collected at ESRF (Grenoble, France), SRS (Daresbury Laboratory, UK) and at Rigaku Americas Corporation (The Woodlands, Texas, USA), processed and integrated by HKL2000 <a href="#pone.0000301-Otwinowski1">[37]</a> (ParC Cys426) and XDS <a href="#pone.0000301-Kabsch1">[38]</a> (wild-type ParC55) software. The cell dimensions for the best diffracting ParC crystal were: <em>a</em> = 136.92 Å, <em>b</em> = 137.85 Å, <em>c</em> = 326.02 Å, α = β = γ = 90.0°. Relatively high R<sub>cryst</sub> (7–20% for different crystals) together with high mosaicity (0.8°–1.4°) and peculiarities of the diffraction patterns indicated that the crystal lattice was twinned. The space group was established to be I222 with non-crystallographic symmetry operators mimicking the symmetry of the I4 space group. In addition, most of the crystals were twinned with the twinning operator (k,-h,l) equivalent to one of the symmetry operators within I4. Data sets were tested for twinning using the UCLA Merohedral Crystal Twinning Server <a href="#pone.0000301-Yeates1">[39]</a> and CNS <a href="#pone.0000301-Brunger1">[40]</a>. The twinning fraction varied from 0.32 to 0.43 for different data sets. The only data set which did not show signs of twinning was the one collected on the Cys426 mutant using cobalt hexamine chloride as an additive. The resolution of this data was 3.25 Å and the model could not be refined to an R factor of lower than 30%, which is probably due to an unaccounted contribution from cobalt.</p>
<a id="article1.body1.sec3.sec3.p5" name="article1.body1.sec3.sec3.p5"></a><p>Several attempts to derive experimental <em>ab initio</em> phases from the Se-Met and heavy atom derivatives of ParC were unsuccessful most likely due to the high complexity of the problem, i.e. high values of the twinning fraction and the twinning operator being close to the NCS operator. The structure was solved by Molecular Replacement. The best two data sets in terms of resolution (wild-type ParC55) and completeness (Cys426 mutant ParC55) were used for the structure solution and refinement (<a href="#pone-0000301-t001">Table 1</a>).</p>
<div class="figure" id="pone-0000301-t001"><div class="img"><a name="pone-0000301-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000301.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000301.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of the crystallographic analysis</span></strong></p><span>doi:10.1371/journal.pone.0000301.t001</span></div><a id="article1.body1.sec3.sec3.p6" name="article1.body1.sec3.sec3.p6"></a><p>The initial model was built using 3D-JIGSAW <a href="#pone.0000301-Bates1">[41]</a>–<a href="#pone.0000301-ContrerasMoreira1">[43]</a>. Due to a higher sequence similarity, the GyrA from <em>E. coli</em> was picked automatically as a prototype <a href="#pone.0000301-MoraisCabral1">[19]</a>. The rotational search performed in CNS using the <em>E. coli</em> GyrA-based model gave the same rotational solutions for different data sets and the results were confirmed by AMoRe (CCP4) <a href="#pone.0000301-1">[44]</a>, <a href="#pone.0000301-Navaza1">[45]</a>. The initial translational searches in I222 space group were not successful either in CNS or in AMoRe. The data set collected on Cys426 mutant with chromium wavelength (λ = 2.29 Å) was integrated in P1 space group and used for a thorough translational search to overcome this problem. The monomers were introduced into the translational search in P1 space group one-by-one in different orientations derived from the previous cross-rotational search and I222 space group symmetry operators. One sequential search solution resulted in formation of a reasonable biological dimer and its results were transferred back into the I222 space group giving a half of the asymmetric unit content. The asymmetric unit was completed in I222 space group using NCS-related dimer introduced into the translational search. Finally, the asymmetric unit was established to be composed of two biological dimers related to each other by an approximately 90 degree rotation around the <em>c</em> axis.</p>
<a id="article1.body1.sec3.sec3.p7" name="article1.body1.sec3.sec3.p7"></a><p>The structure was refined in CNS through several cycles of simulated annealing/positional/B-factor refinement using strict NCS operators in the initial cycles and restrained NCS operators in the final ones. Manual backbone and side chain refitting were performed using Coot <a href="#pone.0000301-Emsley1">[46]</a>. The model was refined to R = 22.30% (R-free = 27.55%) at 2.7 Å resolution using wild-type ParC55 data (twinning fraction = 0.323).</p>
<a id="article1.body1.sec3.sec3.p8" name="article1.body1.sec3.sec3.p8"></a><p>The model was cross-checked using the anomalous data collected on the chromium edge for the Cys426 mutant. An anomalous difference Fourier map was generated in CCP4 <a href="#pone.0000301-1">[44]</a> using the ParC Cys426 data set collected at the chromium wavelength and phases from the refined wild-type ParC55 model. The map showed clear anomalous peaks around sulfur atoms of corresponding methionines as well as cysteine 426, confirming the molecular replacement solution (<a href="#pone-0000301-g006">Figure 6A</a>).</p>
<div class="figure" id="pone-0000301-g006"><div class="img"><a name="pone-0000301-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000301" data-uri="info:doi/10.1371/journal.pone.0000301.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000301.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000301.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000301.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000301.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000301.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6. </strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>Electron density maps confirming the molecular replacement solution. (A) Superposition of the anomalous difference Fourier map calculated using Cys426 data collected on the chromium edge contoured at 3.5σ with the final refined model of ParC55. Panels were rendered using Coot <a href="#pone.0000301-Emsley1">[46]</a>. (B) Stereo view of a region of the 2F<sub>obs</sub>-F<sub>calc</sub> electron density map from wild-type ParC55 data contoured at 1.5σ with the final refined model superimposed.</p>
<span>doi:10.1371/journal.pone.0000301.g006</span></div><a id="article1.body1.sec3.sec3.p9" name="article1.body1.sec3.sec3.p9"></a><p>Close inspection of the 2F<sub>obs</sub>-F<sub>calc</sub> electron density maps built using both wild-type ParC55 and Cys426 mutant data sets have shown a very good coverage for most of the backbone and side-chains (ure 6B). The overall B-factors of the structure are relatively high with the average value of 67.42 Å<sup>2</sup> and are similar to those shown for GyrA from <em>E. coli</em>, which indicates a high level of flexibility of the molecules <a href="#pone.0000301-MoraisCabral1">[19]</a>. The terminal residues (1 to 27 and 483 to 496) as well as the residues in flexible regions:</p>
<a id="article1.body1.sec3.sec3.p10" name="article1.body1.sec3.sec3.p10"></a><p>Chain A: 172, 283–285, 291–295, 301–304, 408;</p>
<a id="article1.body1.sec3.sec3.p11" name="article1.body1.sec3.sec3.p11"></a><p>Chain B: 172, 299, 408;</p>
<a id="article1.body1.sec3.sec3.p12" name="article1.body1.sec3.sec3.p12"></a><p>Chain C: 306, 408–409;</p>
<a id="article1.body1.sec3.sec3.p13" name="article1.body1.sec3.sec3.p13"></a><p>Chain D: 110, 171–174, 253, 282–284, 316–318, 408–409</p>
<a id="article1.body1.sec3.sec3.p14" name="article1.body1.sec3.sec3.p14"></a><p>were omitted from the final structure because no convincing electron density was observed for them.</p>
<a id="article1.body1.sec3.sec3.p15" name="article1.body1.sec3.sec3.p15"></a><p>The final Ramachandran plot indicated that most of the residues are in the favored regions: 83.2% most favored, 13.8% allowed, 2.3% generously allowed, 0.7% disallowed according to PROCHECK <a href="#pone.0000301-Laskowski1">[47]</a>, <a href="#pone.0000301-Morris1">[48]</a>.</p>
<a id="article1.body1.sec3.sec3.p16" name="article1.body1.sec3.sec3.p16"></a><p>Data deposition: The atomic coordinates and structure factors have been deposited with the Protein Data Bank, <a href="http://www.rcsb.org">www.rcsb.org</a> (PDB ID code 2NOV).</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000301"></div><a name="pone.0000301.s001" id="pone.0000301.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000301.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Electrostatic surface potential modeling. GRASP2 <a href="#pone.0000301-Petrey1">[51]</a>, <a href="#pone.0000301-Warwicker1">[52]</a> electrostatic surface potentials within the DNA-binding grooves calculated for DNA-binding domains of topoisomerases IIA from different organisms using models generated by 3D-JIGSAW <a href="#pone.0000301-Bates1">[41]</a>–<a href="#pone.0000301-ContrerasMoreira1">[43]</a> on the base of the known structure of <em>E. coli</em> GyrA (1AB4).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000301.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(3.64 MB PDF)</p>
<a name="pone.0000301.s002" id="pone.0000301.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000301.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Modeling of the DNA sequence recognition by topoisomerase IV. (A) Amino acid sequence alignment for the regions of GyrA and ParC from <em>E. coli</em> and <em>S. pneumoniae</em> comprising helices α3 and α4 and the 100-122 loop. The residues within the α-helices which are likely to interact directly with the incoming DNA helix are indicated by color (red for α3 and blue for α4) and asterisks. Active site tyrosines and arginines are in green. (B) Model of the bound state for the protein-DNA complex between <em>S. pneumoniae</em> ParC and the DNA E site <a href="#pone.0000301-Leo1">[15]</a>. The DNA is in Licorice mode and the protein is in cartoon mode. The positions on the DNA helix are given by numbers in yellow circles. The nucleotides are indicated by square boxes. Amino acids of the 100-122 loop are indicated by ovals with corresponding names. Helix α4 is in cyan and helix α3 is in yellow. Active site tyrosine and arginine are in CPK mode and the point of the DNA cleavage is indicated by yellow asterisk in red circle. Ser 79 and Asp 83 are shown using VDW representation and are in yellow and red respectively. The panel was generated using VMD <a href="#pone.0000301-Humphrey1">[50]</a> and Pov-Ray.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000301.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(3.14 MB PDF)</p>
<a name="pone.0000301.s003" id="pone.0000301.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000301.s003">Movie S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">A schematic process of DNA binding and recognition by <em>S. pneumoniae</em> ParC. The process is shown both in cartoon and electrostatic potential surface modes. The positions of the active-site tyrosines are indicated by arrows when the tyrosines approach the target DNA backbone phosphates. The DNA backbone is in rose, the base pairs involved in sequence specific DNA recognition are in green, the nucleotides adjacent to the cleavage points are in purple in Licorice mode and the non-specific base pairs are in silver. The active-site tyrosines of the ParC dimer are represented in CPK mode in blue. The helix α3 is in cyan, the helix α4 is in red, the 100-122 loop is in yellow and the Gly 166-Pro 179 region is in orange. The movie was generated using VMD <a href="#pone.0000301-Humphrey1">[50]</a>, Pov-Ray and PyMOL <a href="#pone.0000301-DeLano1">[53]</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000301.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(4.77 MB MPG)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Drs. Pierre Rizkallah, Rob Kehoe, James Nicholson, Miroslav Papiz, Michele Cianci and Mark Ellis at the SRS, Daresbury laboratory for their invaluable assistance whilst collecting a number of data associated with this project on several beamlines. We also thank Drs. Andrew Thompson and Gordon Lennard for their help in collecting the Se-Met MAD data at ESRF beamlines ID-14 and ID-29, although due to twinning problems this could not ultimately be used in the structure solution. We acknowledge Dr. Juan Fontecilla-Camps and Dr. Michel Roth for giving us access to beamline ESRF BM 30A for data collection. We are grateful to Drs Edward Morris and Paula da Fonseca of the Structural Electron Microscopy Team from Imperial College and the Institute of Cancer Research for their help in collecting electron microscopy data, though due to low resolution the data could not assist us in finding the structure solution. Mark R. Sanderson would like to thank Prof. Nigel Grindley F.R.S. of the Department of Molecular Biophysics and Biochemistry, Yale University, for hosting his sabbatical during which part of this manuscript was written. Dr. Jim Warwicker is thanked for discussion on electrostatics software.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MS LF. Performed the experiments: MS IL DV MS XP. Analyzed the data: MS IL DV LF AA. Contributed reagents/materials/analysis tools: MS XP LF CY JF. Wrote the paper: MS IL DV MS LF.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000301-Berger1" id="pone.0000301-Berger1"></a>Berger JM (1998) Structure of DNA topoisomerases. Biochim Biophys Acta  1400: 3–18.  <ul class="find" data-citedArticleID="938098" data-doi="10.1016/s0167-4781(98)00124-9"><li><a href="http://dx.doi.org/10.1016/s0167-4781(98)00124-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+DNA+topoisomerases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+DNA+topoisomerases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000301-Champoux1" id="pone.0000301-Champoux1"></a>Champoux JJ (2001) DNA topoisomerases: structure, function, and mechanism. Annu Rev Biochem  70: 369–413.  <ul class="find" data-citedArticleID="938108" data-doi="10.1146/annurev.biochem.70.1.369"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.70.1.369" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+topoisomerases%3A+structure%2C+function%2C+and+mechanism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+topoisomerases%3A+structure%2C+function%2C+and+mechanism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000301-Wang1" id="pone.0000301-Wang1"></a>Wang JC (2002) Cellular roles of DNA topoisomerases: a molecular perspective. Nat Rev Mol Cell Biol  3: 430–440.  <ul class="find" data-citedArticleID="938186" data-doi="10.1038/nrm831"><li><a href="http://dx.doi.org/10.1038/nrm831" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cellular+roles+of+DNA+topoisomerases%3A+a+molecular+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cellular+roles+of+DNA+topoisomerases%3A+a+molecular+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000301-Huang1" id="pone.0000301-Huang1"></a>Huang WM (1994) Type II DNA topoisomerase genes. Adv Pharmacol  29A: 201–225.  <ul class="find" data-citedArticleID="938132" data-doi="10.1016/s1054-3589(08)60547-5"><li><a href="http://dx.doi.org/10.1016/s1054-3589(08)60547-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Type+II+DNA+topoisomerase+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Type+II+DNA+topoisomerase+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000301-Maxwell1" id="pone.0000301-Maxwell1"></a>Maxwell A (1996) Protein gates in DNA topoisomerase II. Nat Struct Biol  3: 109–112.  <ul class="find" data-citedArticleID="938146" data-doi="10.1038/nsb0296-109"><li><a href="http://dx.doi.org/10.1038/nsb0296-109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+gates+in+DNA+topoisomerase+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+gates+in+DNA+topoisomerase+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000301-Drlica1" id="pone.0000301-Drlica1"></a>Drlica K, Zhao X (1997) DNA gyrase, topoisomerase IV, and the 4-quinolones. Microbiol Mol Biol Rev  61: 377–392.  <ul class="find" data-citedArticleID="938124"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+gyrase%2C+topoisomerase+IV%2C+and+the+4-quinolones.&amp;auth=&amp;atitle=DNA+gyrase%2C+topoisomerase+IV%2C+and+the+4-quinolones." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+gyrase%2C+topoisomerase+IV%2C+and+the+4-quinolones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+gyrase%2C+topoisomerase+IV%2C+and+the+4-quinolones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000301-Maxwell2" id="pone.0000301-Maxwell2"></a>Maxwell A (1997) DNA gyrase as a drug target. Trends Microbiol  5: 102–109.  <ul class="find" data-citedArticleID="938148" data-doi="10.1016/s0966-842x(96)10085-8"><li><a href="http://dx.doi.org/10.1016/s0966-842x(96)10085-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+gyrase+as+a+drug+target." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+gyrase+as+a+drug+target.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000301-Pan1" id="pone.0000301-Pan1"></a>Pan XS, Fisher LM (1999) <em>Streptococcus pneumoniae</em> DNA gyrase and topoisomerase IV: overexpression, purification, and differential inhibition by fluoroquinolones. Antimicrob Agents Chemother  43: 1129–1136.  <ul class="find" data-citedArticleID="938164"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Streptococcus+pneumoniae+DNA+gyrase+and+topoisomerase+IV%3A+overexpression%2C+purification%2C+and+differential+inhibition+by+fluoroquinolones.&amp;auth=&amp;atitle=Streptococcus+pneumoniae+DNA+gyrase+and+topoisomerase+IV%3A+overexpression%2C+purification%2C+and+differential+inhibition+by+fluoroquinolones." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Streptococcus+pneumoniae+DNA+gyrase+and+topoisomerase+IV%3A+overexpression%2C+purification%2C+and+differential+inhibition+by+fluoroquinolones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Streptococcus+pneumoniae+DNA+gyrase+and+topoisomerase+IV%3A+overexpression%2C+purification%2C+and+differential+inhibition+by+fluoroquinolones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000301-Corbett1" id="pone.0000301-Corbett1"></a>Corbett KD, Berger JM (2004) Structure, molecular mechanisms, and evolutionary relationships in DNA topoisomerases. Annu Rev Biophys Biomol Struct  33: 95–118.  <ul class="find" data-citedArticleID="938114" data-doi="10.1146/annurev.biophys.33.110502.140357"><li><a href="http://dx.doi.org/10.1146/annurev.biophys.33.110502.140357" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure%2C+molecular+mechanisms%2C+and+evolutionary+relationships+in+DNA+topoisomerases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure%2C+molecular+mechanisms%2C+and+evolutionary+relationships+in+DNA+topoisomerases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000301-Gadelle1" id="pone.0000301-Gadelle1"></a>Gadelle D, Filee J, Buhler C, Forterre P (2003) Phylogenomics of type II DNA topoisomerases. Bioessays  25: 232–242.  <ul class="find" data-citedArticleID="938130" data-doi="10.1002/bies.10245"><li><a href="http://dx.doi.org/10.1002/bies.10245" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenomics+of+type+II+DNA+topoisomerases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenomics+of+type+II+DNA+topoisomerases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000301-Wang2" id="pone.0000301-Wang2"></a>Wang JC (1998) Moving one DNA double helix through another by a type II DNA topoisomerase: the story of a simple molecular machine. Q Rev Biophys  31: 107–144.  <ul class="find" data-citedArticleID="938188" data-doi="10.1017/s0033583598003424"><li><a href="http://dx.doi.org/10.1017/s0033583598003424" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Moving+one+DNA+double+helix+through+another+by+a+type+II+DNA+topoisomerase%3A+the+story+of+a+simple+molecular+machine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Moving+one+DNA+double+helix+through+another+by+a+type+II+DNA+topoisomerase%3A+the+story+of+a+simple+molecular+machine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000301-Kampranis1" id="pone.0000301-Kampranis1"></a>Kampranis SC, Maxwell A (1996) Conversion of DNA gyrase into a conventional type II topoisomerase. Proc Natl Acad Sci U S A  93: 14416–14421.  <ul class="find" data-citedArticleID="938138" data-doi="10.1073/pnas.93.25.14416"><li><a href="http://dx.doi.org/10.1073/pnas.93.25.14416" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conversion+of+DNA+gyrase+into+a+conventional+type+II+topoisomerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conversion+of+DNA+gyrase+into+a+conventional+type+II+topoisomerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000301-Pan2" id="pone.0000301-Pan2"></a>Pan XS, Fisher LM (1997) Targeting of DNA gyrase in <em>Streptococcus pneumoniae</em> by sparfloxacin: selective targeting of gyrase or topoisomerase IV by quinolones. Antimicrob Agents Chemother  41: 471–474.  <ul class="find" data-citedArticleID="938166"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Targeting+of+DNA+gyrase+in+Streptococcus+pneumoniae+by+sparfloxacin%3A+selective+targeting+of+gyrase+or+topoisomerase+IV+by+quinolones.&amp;auth=&amp;atitle=Targeting+of+DNA+gyrase+in+Streptococcus+pneumoniae+by+sparfloxacin%3A+selective+targeting+of+gyrase+or+topoisomerase+IV+by+quinolones." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeting+of+DNA+gyrase+in+Streptococcus+pneumoniae+by+sparfloxacin%3A+selective+targeting+of+gyrase+or+topoisomerase+IV+by+quinolones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeting+of+DNA+gyrase+in+Streptococcus+pneumoniae+by+sparfloxacin%3A+selective+targeting+of+gyrase+or+topoisomerase+IV+by+quinolones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000301-Pan3" id="pone.0000301-Pan3"></a>Pan XS, Fisher LM (1998) DNA gyrase and topoisomerase IV are dual targets of clinafloxacin action in <em>Streptococcus pneumoniae</em>. Antimicrob Agents Chemother  42: 2810–2816.  <ul class="find" data-citedArticleID="938168"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+gyrase+and+topoisomerase+IV+are+dual+targets+of+clinafloxacin+action+in+Streptococcus+pneumoniae.&amp;auth=&amp;atitle=DNA+gyrase+and+topoisomerase+IV+are+dual+targets+of+clinafloxacin+action+in+Streptococcus+pneumoniae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+gyrase+and+topoisomerase+IV+are+dual+targets+of+clinafloxacin+action+in+Streptococcus+pneumoniae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+gyrase+and+topoisomerase+IV+are+dual+targets+of+clinafloxacin+action+in+Streptococcus+pneumoniae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000301-Leo1" id="pone.0000301-Leo1"></a>Leo E, Gould KA, Pan XS, Capranico G, Sanderson MR, et al.  (2005) Novel symmetric and asymmetric DNA scission determinants for <em>Streptococcus pneumoniae</em> topoisomerase IV and gyrase are clustered at the DNA breakage site. J Biol Chem  280: 14252–14263.  <ul class="find" data-citedArticleID="938144" data-doi="10.1074/jbc.m500156200"><li><a href="http://dx.doi.org/10.1074/jbc.m500156200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+symmetric+and+asymmetric+DNA+scission+determinants+for+Streptococcus+pneumoniae+topoisomerase+IV+and+gyrase+are+clustered+at+the+DNA+breakage+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+symmetric+and+asymmetric+DNA+scission+determinants+for+Streptococcus+pneumoniae+topoisomerase+IV+and+gyrase+are+clustered+at+the+DNA+breakage+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000301-CBSN1" id="pone.0000301-CBSN1"></a>CBSN: Canadian Bacterial Surveillance Network (2006)   &lt;<a href="http://microbiology.mtsinai.on.ca/data/sp/indexOld.shtml">http://microbiology.mtsinai.on.ca/data/s​p/indexOld.shtml</a>&gt;.  <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0000301-Bellon1" id="pone.0000301-Bellon1"></a>Bellon S, Parsons JD, Wei Y, Hayakawa K, Swenson LL, et al.  (2004) Crystal structures of <em>Escherichia coli</em> topoisomerase IV ParE subunit (24 and 43 kilodaltons): a single residue dictates differences in novobiocin potency against topoisomerase IV and DNA gyrase. Antimicrob Agents Chemother  48: 1856–1864.  <ul class="find" data-citedArticleID="938096" data-doi="10.1128/aac.48.5.1856-1864.2004"><li><a href="http://dx.doi.org/10.1128/aac.48.5.1856-1864.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structures+of+Escherichia+coli+topoisomerase+IV+ParE+subunit+%2824+and+43+kilodaltons%29%3A+a+single+residue+dictates+differences+in+novobiocin+potency+against+topoisomerase+IV+and+DNA+gyrase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structures+of+Escherichia+coli+topoisomerase+IV+ParE+subunit+%2824+and+43+kilodaltons%29%3A+a+single+residue+dictates+differences+in+novobiocin+potency+against+topoisomerase+IV+and+DNA+gyrase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000301-Corbett2" id="pone.0000301-Corbett2"></a>Corbett KD, Schoeffler AJ, Thomsen ND, Berger JM (2005) The structural basis for substrate specificity in DNA topoisomerase IV. J Mol Biol  351: 545–561.  <ul class="find" data-citedArticleID="938116" data-doi="10.1016/j.jmb.2005.06.029"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.06.029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structural+basis+for+substrate+specificity+in+DNA+topoisomerase+IV." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structural+basis+for+substrate+specificity+in+DNA+topoisomerase+IV.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000301-MoraisCabral1" id="pone.0000301-MoraisCabral1"></a>Morais-Cabral JH, Jackson AP, Smith CV, Shikotra N, Maxwell A, et al.  (1997) Crystal structure of the breakage-reunion domain of DNA gyrase. Nature  388: 903–906.  <ul class="find" data-citedArticleID="938154" data-doi="10.1038/42294"><li><a href="http://dx.doi.org/10.1038/42294" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+breakage-reunion+domain+of+DNA+gyrase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+breakage-reunion+domain+of+DNA+gyrase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000301-Berger2" id="pone.0000301-Berger2"></a>Berger JM, Gamblin SJ, Harrison SC, Wang JC (1996) Structure and mechanism of DNA topoisomerase II. Nature  379: 225–232.  <ul class="find" data-citedArticleID="938100" data-doi="10.1038/379225a0"><li><a href="http://dx.doi.org/10.1038/379225a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+mechanism+of+DNA+topoisomerase+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+mechanism+of+DNA+topoisomerase+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000301-McKay1" id="pone.0000301-McKay1"></a>McKay DB, Steitz TA (1981) Structure of catabolite gene activator protein at 2.9 Å resolution suggests binding to left-handed B-DNA. Nature  290: 744–749.  <ul class="find" data-citedArticleID="938150" data-doi="10.1038/290744a0"><li><a href="http://dx.doi.org/10.1038/290744a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+catabolite+gene+activator+protein+at+2.9+%26%23xc5%3B+resolution+suggests+binding+to+left-handed+B-DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+catabolite+gene+activator+protein+at+2.9+%26%23xc5%3B+resolution+suggests+binding+to+left-handed+B-DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000301-Passner1" id="pone.0000301-Passner1"></a>Passner JM, Steitz TA (1997) The structure of a CAP-DNA complex having two cAMP molecules bound to each monomer. Proc Natl Acad Sci U S A  94: 2843–2847.  <ul class="find" data-citedArticleID="938170" data-doi="10.1073/pnas.94.7.2843"><li><a href="http://dx.doi.org/10.1073/pnas.94.7.2843" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+a+CAP-DNA+complex+having+two+cAMP+molecules+bound+to+each+monomer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+a+CAP-DNA+complex+having+two+cAMP+molecules+bound+to+each+monomer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000301-Schultz1" id="pone.0000301-Schultz1"></a>Schultz SC, Shields GC, Steitz TA (1991) Crystal structure of a CAP-DNA complex: the DNA is bent by 90 degrees. Science  253: 1001–1007.  <ul class="find" data-citedArticleID="938182" data-doi="10.1126/science.1653449"><li><a href="http://dx.doi.org/10.1126/science.1653449" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+a+CAP-DNA+complex%3A+the+DNA+is+bent+by+90+degrees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+a+CAP-DNA+complex%3A+the+DNA+is+bent+by+90+degrees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000301-Brown1" id="pone.0000301-Brown1"></a>Brown PO, Cozzarelli NR (1979) A sign inversion mechanism for enzymatic supercoiling of DNA. Science  206: 1081–1083.  <ul class="find" data-citedArticleID="938102" data-doi="10.1126/science.227059"><li><a href="http://dx.doi.org/10.1126/science.227059" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+sign+inversion+mechanism+for+enzymatic+supercoiling+of+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+sign+inversion+mechanism+for+enzymatic+supercoiling+of+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000301-Mizuuchi1" id="pone.0000301-Mizuuchi1"></a>Mizuuchi K, Fisher LM, O'Dea MH, Gellert M (1980) DNA gyrase action involves the introduction of transient double-strand breaks into DNA. Proc Natl Acad Sci U S A  77: 1847–1851.  <ul class="find" data-citedArticleID="938152" data-doi="10.1073/pnas.77.4.1847"><li><a href="http://dx.doi.org/10.1073/pnas.77.4.1847" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+gyrase+action+involves+the+introduction+of+transient+double-strand+breaks+into+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+gyrase+action+involves+the+introduction+of+transient+double-strand+breaks+into+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000301-Corbett3" id="pone.0000301-Corbett3"></a>Corbett KD, Shultzaberger RK, Berger JM (2004) The C-terminal domain of DNA gyrase A adopts a DNA-bending β-pinwheel fold. Proc Natl Acad Sci U S A  101: 7293–7298.  <ul class="find" data-citedArticleID="938118" data-doi="10.1073/pnas.0401595101"><li><a href="http://dx.doi.org/10.1073/pnas.0401595101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+C-terminal+domain+of+DNA+gyrase+A+adopts+a+DNA-bending+%26%23x3b2%3B-pinwheel+fold." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+C-terminal+domain+of+DNA+gyrase+A+adopts+a+DNA-bending+%26%23x3b2%3B-pinwheel+fold.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000301-Reece1" id="pone.0000301-Reece1"></a>Reece RJ, Maxwell A (1991) The C-terminal domain of the <em>Escherichia coli</em> DNA gyrase A subunit is a DNA-binding protein. Nucleic Acids Res  19: 1399–1405.  <ul class="find" data-citedArticleID="938176" data-doi="10.1093/nar/19.7.1399"><li><a href="http://dx.doi.org/10.1093/nar/19.7.1399" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+C-terminal+domain+of+the+Escherichia+coli+DNA+gyrase+A+subunit+is+a+DNA-binding+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+C-terminal+domain+of+the+Escherichia+coli+DNA+gyrase+A+subunit+is+a+DNA-binding+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000301-Ruthenburg1" id="pone.0000301-Ruthenburg1"></a>Ruthenburg AJ, Graybosch DM, Huetsch JC, Verdine GL (2005) A superhelical spiral in the <em>Escherichia coli</em> DNA gyrase A C-terminal domain imparts unidirectional supercoiling bias. J Biol Chem  280: 26177–26184.  <ul class="find" data-citedArticleID="938180" data-doi="10.1074/jbc.m502838200"><li><a href="http://dx.doi.org/10.1074/jbc.m502838200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+superhelical+spiral+in+the+Escherichia+coli+DNA+gyrase+A+C-terminal+domain+imparts+unidirectional+supercoiling+bias." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+superhelical+spiral+in+the+Escherichia+coli+DNA+gyrase+A+C-terminal+domain+imparts+unidirectional+supercoiling+bias.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000301-Morrison1" id="pone.0000301-Morrison1"></a>Morrison A, Cozzarelli NR (1981) Contacts between DNA gyrase and its binding site on DNA: features of symmetry and asymmetry revealed by protection from nucleases. Proc Natl Acad Sci U S A  78: 1416–1420.  <ul class="find" data-citedArticleID="938158" data-doi="10.1073/pnas.78.3.1416"><li><a href="http://dx.doi.org/10.1073/pnas.78.3.1416" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contacts+between+DNA+gyrase+and+its+binding+site+on+DNA%3A+features+of+symmetry+and+asymmetry+revealed+by+protection+from+nucleases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contacts+between+DNA+gyrase+and+its+binding+site+on+DNA%3A+features+of+symmetry+and+asymmetry+revealed+by+protection+from+nucleases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000301-Fisher1" id="pone.0000301-Fisher1"></a>Fisher LM, Mizuuchi K, O'Dea MH, Ohmori H, Gellert M (1981) Site-specific interaction of DNA gyrase with DNA. Proc Natl Acad Sci U S A  78: 4165–4169.  <ul class="find" data-citedArticleID="938128" data-doi="10.1073/pnas.78.7.4165"><li><a href="http://dx.doi.org/10.1073/pnas.78.7.4165" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Site-specific+interaction+of+DNA+gyrase+with+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Site-specific+interaction+of+DNA+gyrase+with+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000301-Kirkegaard1" id="pone.0000301-Kirkegaard1"></a>Kirkegaard K, Wang JC (1981) Mapping the topography of DNA wrapped around gyrase by nucleolytic and chemical probing of complexes of unique DNA sequences. Cell  23: 721–729.  <ul class="find" data-citedArticleID="938140" data-doi="10.1016/0092-8674(81)90435-9"><li><a href="http://dx.doi.org/10.1016/0092-8674(81)90435-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+the+topography+of+DNA+wrapped+around+gyrase+by+nucleolytic+and+chemical+probing+of+complexes+of+unique+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+the+topography+of+DNA+wrapped+around+gyrase+by+nucleolytic+and+chemical+probing+of+complexes+of+unique+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000301-Peng1" id="pone.0000301-Peng1"></a>Peng H, Marians KJ (1995) The interaction of <em>Escherichia coli</em> topoisomerase IV with DNA. J Biol Chem  270: 25286–25290.  <ul class="find" data-citedArticleID="938172" data-doi="10.1074/jbc.270.42.25286"><li><a href="http://dx.doi.org/10.1074/jbc.270.42.25286" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+interaction+of+Escherichia+coli+topoisomerase+IV+with+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+interaction+of+Escherichia+coli+topoisomerase+IV+with+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000301-Daniels1" id="pone.0000301-Daniels1"></a>Daniels DS, Woo TT, Luu KX, Noll DM, Clarke ND, et al.  (2004) DNA binding and nucleotide flipping by the human DNA repair protein AGT. Nat Struct Mol Biol  11: 714–720.  <ul class="find" data-citedArticleID="938120" data-doi="10.1038/nsmb791"><li><a href="http://dx.doi.org/10.1038/nsmb791" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+binding+and+nucleotide+flipping+by+the+human+DNA+repair+protein+AGT." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+binding+and+nucleotide+flipping+by+the+human+DNA+repair+protein+AGT.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000301-Shen1" id="pone.0000301-Shen1"></a>Shen LL, Baranowski J, Pernet AG (1989) Mechanism of inhibition of DNA gyrase by quinolone antibacterials: specificity and cooperativity of drug binding to DNA. Biochemistry  28: 3879–3885.  <ul class="find" data-citedArticleID="938184" data-doi="10.1021/bi00435a038"><li><a href="http://dx.doi.org/10.1021/bi00435a038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+of+inhibition+of+DNA+gyrase+by+quinolone+antibacterials%3A+specificity+and+cooperativity+of+drug+binding+to+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+of+inhibition+of+DNA+gyrase+by+quinolone+antibacterials%3A+specificity+and+cooperativity+of+drug+binding+to+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000301-Willmott1" id="pone.0000301-Willmott1"></a>Willmott CJ, Maxwell A (1993) A single point mutation in the DNA gyrase A protein greatly reduces binding of fluoroquinolones to the gyrase-DNA complex. Antimicrob Agents Chemother  37: 126–127.  <ul class="find" data-citedArticleID="938192" data-doi="10.1128/aac.37.1.126"><li><a href="http://dx.doi.org/10.1128/aac.37.1.126" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+point+mutation+in+the+DNA+gyrase+A+protein+greatly+reduces+binding+of+fluoroquinolones+to+the+gyrase-DNA+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+point+mutation+in+the+DNA+gyrase+A+protein+greatly+reduces+binding+of+fluoroquinolones+to+the+gyrase-DNA+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000301-Richter1" id="pone.0000301-Richter1"></a>Richter SN, Leo E, Giaretta G, Gatto B, Fisher LM, et al.  (2006) Clerocidin interacts with the cleavage complex of <em>Streptococcus pneumoniae</em> topoisomerase IV to induce selective irreversible DNA damage. Nucleic Acids Res  34: 1982–1991.  <ul class="find" data-citedArticleID="938178" data-doi="10.1093/nar/gkl127"><li><a href="http://dx.doi.org/10.1093/nar/gkl127" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clerocidin+interacts+with+the+cleavage+complex+of+Streptococcus+pneumoniae+topoisomerase+IV+to+induce+selective+irreversible+DNA+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clerocidin+interacts+with+the+cleavage+complex+of+Streptococcus+pneumoniae+topoisomerase+IV+to+induce+selective+irreversible+DNA+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000301-Otwinowski1" id="pone.0000301-Otwinowski1"></a>Otwinowski Z, Minor W (1997) Processing of X-ray diffraction data collected in oscillation mode. Method Enzymol 307–325.  <ul class="find" data-citedArticleID="938162" data-doi="10.1016/s0076-6879(97)76066-x"><li><a href="http://dx.doi.org/10.1016/s0076-6879(97)76066-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000301-Kabsch1" id="pone.0000301-Kabsch1"></a>Kabsch W (1993) Automatic processing of rotation diffraction data from crystals of initially unknown symmetry and cell constants. J Appl Crystallogr  26: 795.  <ul class="find" data-citedArticleID="938136" data-doi="10.1107/s0021889893005588"><li><a href="http://dx.doi.org/10.1107/s0021889893005588" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automatic+processing+of+rotation+diffraction+data+from+crystals+of+initially+unknown+symmetry+and+cell+constants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automatic+processing+of+rotation+diffraction+data+from+crystals+of+initially+unknown+symmetry+and+cell+constants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000301-Yeates1" id="pone.0000301-Yeates1"></a>Yeates TO (1997) Detecting and overcoming crystal twinning. Method Enzymol 344–360.  <ul class="find" data-citedArticleID="938194" data-doi="10.1016/s0076-6879(97)76068-3"><li><a href="http://dx.doi.org/10.1016/s0076-6879(97)76068-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+and+overcoming+crystal+twinning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+and+overcoming+crystal+twinning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000301-Brunger1" id="pone.0000301-Brunger1"></a>Brunger AT, Adams PD, Clore GM, DeLano WL, Gros P, et al.  (1998) Crystallography &amp; NMR system: A new software suite for macromolecular structure determination. Acta Crystallogr D Biol Crystallogr  54: 905–921.  <ul class="find" data-citedArticleID="938104" data-doi="10.1107/s0907444998003254"><li><a href="http://dx.doi.org/10.1107/s0907444998003254" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystallography+%26amp%3B+NMR+system%3A+A+new+software+suite+for+macromolecular+structure+determination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystallography+%26amp%3B+NMR+system%3A+A+new+software+suite+for+macromolecular+structure+determination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000301-Bates1" id="pone.0000301-Bates1"></a>Bates PA, Kelley LA, MacCallum RM, Sternberg MJE (2001) Enhancement of protein modeling by human intervention in applying the automatic programs 3D-JIGSAW and 3D-PSSM. Proteins  45: 39–46.  <ul class="find" data-citedArticleID="938092" data-doi="10.1002/prot.1168"><li><a href="http://dx.doi.org/10.1002/prot.1168" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhancement+of+protein+modeling+by+human+intervention+in+applying+the+automatic+programs+3D-JIGSAW+and+3D-PSSM." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhancement+of+protein+modeling+by+human+intervention+in+applying+the+automatic+programs+3D-JIGSAW+and+3D-PSSM.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000301-Bates2" id="pone.0000301-Bates2"></a>Bates PA, Sternberg MJE (1999) Model building by comparison at CASP3: using expert knowledge and computer automation. Proteins  37: 47–54.  <ul class="find" data-citedArticleID="938094" data-doi="10.1002/(sici)1097-0134(1999)37:3+<47::aid-prot7&gt;3.3.co;2-6"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0134(1999)37:3+<47::aid-prot7&gt;3.3.co;2-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Model+building+by+comparison+at+CASP3%3A+using+expert+knowledge+and+computer+automation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Model+building+by+comparison+at+CASP3%3A+using+expert+knowledge+and+computer+automation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000301-ContrerasMoreira1" id="pone.0000301-ContrerasMoreira1"></a>Contreras-Moreira B, Bates PA (2002) Domain Fishing: a first step in protein comparative modelling. Bioinformatics  18: 1141–1142.  <ul class="find" data-citedArticleID="938112" data-doi="10.1093/bioinformatics/18.8.1141"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.8.1141" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Domain+Fishing%3A+a+first+step+in+protein+comparative+modelling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Domain+Fishing%3A+a+first+step+in+protein+comparative+modelling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000301-1" id="pone.0000301-1"></a> (1994) The CCP4 suite: programs for protein crystallography. Acta Crystallogr D Biol Crystallogr  50: 760–763.  <ul class="find" data-citedArticleID="938090" data-doi="10.1107/s0907444994003112"><li><a href="http://dx.doi.org/10.1107/s0907444994003112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CCP4+suite%3A+programs+for+protein+crystallography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CCP4+suite%3A+programs+for+protein+crystallography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000301-Navaza1" id="pone.0000301-Navaza1"></a>Navaza J (2001) Implementation of molecular replacement in AMoRe. Acta Crystallogr D Biol Crystallogr  57: 1367–1372.  <ul class="find" data-citedArticleID="938160" data-doi="10.1107/s0907444901012422"><li><a href="http://dx.doi.org/10.1107/s0907444901012422" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Implementation+of+molecular+replacement+in+AMoRe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Implementation+of+molecular+replacement+in+AMoRe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000301-Emsley1" id="pone.0000301-Emsley1"></a>Emsley P, Cowtan K (2004) Coot: model-building tools for molecular graphics. Acta Crystallogr D Biol Crystallogr  60: 2126–2132.  <ul class="find" data-citedArticleID="938126" data-doi="10.1107/s0907444904019158"><li><a href="http://dx.doi.org/10.1107/s0907444904019158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coot%3A+model-building+tools+for+molecular+graphics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coot%3A+model-building+tools+for+molecular+graphics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000301-Laskowski1" id="pone.0000301-Laskowski1"></a>Laskowski RA, MacArthur MW, Moss DS, Thornton JM (1993) PROCHECK: a program to check the stereochemical quality of protein structures. J Appl Crystallogr  26: 283.  <ul class="find" data-citedArticleID="938142" data-doi="10.1107/s0021889892009944"><li><a href="http://dx.doi.org/10.1107/s0021889892009944" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000301-Morris1" id="pone.0000301-Morris1"></a>Morris AL, MacArthur MW, Hutchinson EG, Thornton JM (1992) Stereochemical quality of protein structure coordinates. Proteins  12: 345–364.  <ul class="find" data-citedArticleID="938156" data-doi="10.1002/prot.340120407"><li><a href="http://dx.doi.org/10.1002/prot.340120407" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stereochemical+quality+of+protein+structure+coordinates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stereochemical+quality+of+protein+structure+coordinates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000301-Chenna1" id="pone.0000301-Chenna1"></a>Chenna R, Sugawara H, Koike T, Lopez R, Gibson TJ, et al.  (2003) Multiple sequence alignment with the Clustal series of programs. Nucleic Acids Res  31: 3497–3500.  <ul class="find" data-citedArticleID="938110" data-doi="10.1093/nar/gkg500"><li><a href="http://dx.doi.org/10.1093/nar/gkg500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+sequence+alignment+with+the+Clustal+series+of+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+sequence+alignment+with+the+Clustal+series+of+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000301-Humphrey1" id="pone.0000301-Humphrey1"></a>Humphrey W, Dalke A, Schulten K (1996) VMD: visual molecular dynamics. J Mol Graph  14: 27–38.  <ul class="find" data-citedArticleID="938134" data-doi="10.1016/0263-7855(96)00018-5"><li><a href="http://dx.doi.org/10.1016/0263-7855(96)00018-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=VMD%3A+visual+molecular+dynamics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22VMD%3A+visual+molecular+dynamics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000301-Petrey1" id="pone.0000301-Petrey1"></a>Petrey D, Honig B (2003) GRASP2: visualization, surface properties, and electrostatics of macromolecular structures and sequences. Method Enzymol  374: 492–509.  <ul class="find" data-citedArticleID="938174" data-doi="10.1016/s0076-6879(03)74021-x"><li><a href="http://dx.doi.org/10.1016/s0076-6879(03)74021-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GRASP2%3A+visualization%2C+surface+properties%2C+and+electrostatics+of+macromolecular+structures+and+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GRASP2%3A+visualization%2C+surface+properties%2C+and+electrostatics+of+macromolecular+structures+and+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000301-Warwicker1" id="pone.0000301-Warwicker1"></a>Warwicker J, Watson HC (1982) Calculation of the electric potential in the active site cleft due to α–helix dipoles. J Mol Biol  157: 671–679.  <ul class="find" data-citedArticleID="938190" data-doi="10.1016/0022-2836(82)90505-8"><li><a href="http://dx.doi.org/10.1016/0022-2836(82)90505-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Calculation+of+the+electric+potential+in+the+active+site+cleft+due+to+%26%23x3b1%3B%26%23x2013%3Bhelix+dipoles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Calculation+of+the+electric+potential+in+the+active+site+cleft+due+to+%26%23x3b1%3B%26%23x2013%3Bhelix+dipoles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000301-DeLano1" id="pone.0000301-DeLano1"></a>DeLano WL (2002) The PyMOL molecular graphics system.   &lt;<a href="http://www.pymol.org">http://www.pymol.org</a>&gt;.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.XML" value="98755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.PDF" value="598299"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g001.PNG_L" value="269771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g001.PNG_M" value="23984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g001.PNG_S" value="2016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g001.TIF" value="363422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g001.PNG_I" value="10781"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g002.PNG_L" value="2996693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g002.PNG_M" value="222819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g002.PNG_S" value="5437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g002.TIF" value="4041688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g002.PNG_I" value="78121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g003.PNG_L" value="1417055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g003.PNG_M" value="282805"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g003.PNG_S" value="9095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g003.TIF" value="1902066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g003.PNG_I" value="135320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g004.PNG_L" value="1944725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g004.PNG_M" value="118462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g004.PNG_S" value="3613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g004.TIF" value="2283852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g004.PNG_I" value="37957"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g005.PNG_L" value="4779858"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g005.PNG_M" value="450689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g005.PNG_S" value="9693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g005.TIF" value="6180686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g005.PNG_I" value="146626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.t001.PNG_L" value="135260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.t001.PNG_M" value="46323"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.t001.PNG_S" value="2652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.t001.TIF" value="729888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.t001.PNG_I" value="19472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g006.PNG_L" value="2188881"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g006.PNG_M" value="372756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g006.PNG_S" value="10169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g006.TIF" value="2691426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.g006.PNG_I" value="160495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.s001.PDF" value="3638756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.s002.PDF" value="3138164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000301.s003.MPG" value="4772340"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000301&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000301" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000301&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000301&volume=&issue=&title=Breakage-Reunion Domain of Streptococcus pneumoniae Topoisomerase IV: Crystal Structure of a Gram-Positive Quinolone Target&author_name=Ivan%20Laponogov%2C%20Dennis%20A.%20Veselkov%2C%20Maninder%20K.%20Sohi%2C%20Xiao-Su%20Pan%2C%20Aniruddha%20Achari%2C%20Cheng%20Yang%2C%20Joseph%20D.%20Ferrara%2C%20L.%20Mark%20Fisher%2C%20Mark%20R.%20Sanderson&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000301" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000301" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000301" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000301&amp;t=Breakage-Reunion%20Domain%20of%20Streptococcus%20pneumoniae%20Topoisomerase%20IV%3A%20Crystal%20Structure%20of%20a%20Gram-Positive%20Quinolone%20Target" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000301&title=Breakage-Reunion%20Domain%20of%20Streptococcus%20pneumoniae%20Topoisomerase%20IV%3A%20Crystal%20Structure%20of%20a%20Gram-Positive%20Quinolone%20Target&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000301&amp;title=Breakage-Reunion%20Domain%20of%20Streptococcus%20pneumoniae%20Topoisomerase%20IV%3A%20Crystal%20Structure%20of%20a%20Gram-Positive%20Quinolone%20Target" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000301" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000301&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Breakage-Reunion Domain of Streptococcus pneumoniae Topoisomerase IV: Crystal Structure of a Gram-Positive Quinolone Target';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000301';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Breakage-Reunion%20Domain%20of%20Streptococcus%20pneumoniae%20Topoisomerase%20IV%3A%20Crystal%20Structure%20of%20a%20Gram-Positive%20Quinolone%20Target http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000301" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000301" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystal+structure%22" title="Search for articles in the subject area:'Crystal structure'"><div class="flagText">Crystal structure</div></a>
              <div data-categoryid="17301" data-articleid="23440"
                   data-categoryname="Crystal structure"
                   class="flagImage" title="Flag 'Crystal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+cleavage%22" title="Search for articles in the subject area:'DNA cleavage'"><div class="flagText">DNA cleavage</div></a>
              <div data-categoryid="31951" data-articleid="23440"
                   data-categoryname="DNA cleavage"
                   class="flagImage" title="Flag 'DNA cleavage' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23440"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+structure%22" title="Search for articles in the subject area:'DNA structure'"><div class="flagText">DNA structure</div></a>
              <div data-categoryid="34399" data-articleid="23440"
                   data-categoryname="DNA structure"
                   class="flagImage" title="Flag 'DNA structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="23440"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gram+positive+bacteria%22" title="Search for articles in the subject area:'Gram positive bacteria'"><div class="flagText">Gram positive bacteria</div></a>
              <div data-categoryid="52001" data-articleid="23440"
                   data-categoryname="Gram positive bacteria"
                   class="flagImage" title="Flag 'Gram positive bacteria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phosphates%22" title="Search for articles in the subject area:'Phosphates'"><div class="flagText">Phosphates</div></a>
              <div data-categoryid="19751" data-articleid="23440"
                   data-categoryname="Phosphates"
                   class="flagImage" title="Flag 'Phosphates' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Tyrosine%22" title="Search for articles in the subject area:'Tyrosine'"><div class="flagText">Tyrosine</div></a>
              <div data-categoryid="40505" data-articleid="23440"
                   data-categoryname="Tyrosine"
                   class="flagImage" title="Flag 'Tyrosine' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9223'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2577'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1803&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
