

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: On the Origin and Spread of the Scab Disease of Apple: Out of Central Asia</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001455"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001455" />

    <meta name="citation_title" content="On the Origin and Spread of the Scab Disease of Apple: Out of Central Asia"/>
    <meta itemprop="name" content="On the Origin and Spread of the Scab Disease of Apple: Out of Central Asia"/>

      <meta name="citation_author" content="Pierre Gladieux"/>
            <meta name="citation_author_institution" content="UMR077, INRA, Angers, France"/>
      <meta name="citation_author" content="Xiu-Guo Zhang"/>
            <meta name="citation_author_institution" content="Department of Plant Pathology, Shandong Agricultural University, Taian, China"/>
      <meta name="citation_author" content="Damien Afoufa-Bastien"/>
            <meta name="citation_author_institution" content="UMR077, INRA, Angers, France"/>
      <meta name="citation_author" content="Rosa-Maria Valdebenito Sanhueza"/>
            <meta name="citation_author_institution" content="National Research Centre for Grape and Wine, EMBRAPA, Bento Gonçalves, Brazil"/>
      <meta name="citation_author" content="Mohamed Sbaghi"/>
            <meta name="citation_author_institution" content="Centre Régional de la Recherche Agronomique de Kenitra, INRA, Kenitra, Morocco"/>
      <meta name="citation_author" content="Bruno Le Cam"/>
            <meta name="citation_author_institution" content="UMR077, INRA, Angers, France"/>

    <meta name="citation_date" content="2008/1/16"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001455.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1455"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A general eco-evolutionary framework for understanding bioinvasions.; citation_author=B Facon; citation_author=B Genton; citation_author=J Shykoff; citation_author=P Jarne; citation_author=A Estoup; citation_journal_title=Trends Ecol Evol; citation_volume=21; citation_number=1; citation_pages=130-135; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Emerging infectious diseases of plants: pathogen pollution, climate change and agrotechnology drivers.; citation_author=P Anderson; citation_author=A Cunningham; citation_author=N Patel; citation_author=F Morales; citation_author=P Epstein; citation_journal_title=Trends Ecol Evol; citation_volume=19; citation_number=2; citation_pages=535-544; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Domesticated nature: shaping landscapes and ecosystems for human welfare.; citation_author=P Kareiva; citation_author=S Watts; citation_author=R McDonald; citation_author=T Boucher; citation_journal_title=Science; citation_volume=316; citation_number=3; citation_pages=1866-1869; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Environmental and Economic Costs of Nonindigenous Species in theUnited States.; citation_author=D Pimentel; citation_author=L Lach; citation_author=R Zuniga; citation_author=D Morrison; citation_journal_title=BioScience; citation_volume=50; citation_number=4; citation_pages=53-64; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Chestnut blight: The classical problem of an introduced pathogen.; citation_author=S Anagnostakis; citation_journal_title=Mycologia; citation_volume=29; citation_number=5; citation_pages=23-37; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Resurgence of the Irish potato famine fungus.; citation_author=W Fry; citation_author=S Goodwin; citation_journal_title=BioScience; citation_volume=47; citation_number=6; citation_pages=363-371; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Systematics and the Impact of Invasive Fungi on Agriculture in theUnited States.; citation_author=M Palm; citation_journal_title=BioScience; citation_volume=51; citation_number=7; citation_pages=141-147; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Worldwide Movement of Exotic Forest Fungi, Especially in theTropics and the Southern Hemisphere.; citation_author=M Wingfield; citation_author=B Slippers; citation_author=J Roux; citation_author=B Wingfield; citation_journal_title=BioScience; citation_volume=51; citation_number=8; citation_pages=134-140; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The Science of Risk Assessment for Phytosanitary Regulation and the Impact of Changing Trade Regulations.; citation_author=F Campbell; citation_journal_title=BioScience; citation_volume=51; citation_number=9; citation_pages=148-153; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Biological invasion risks and the public good: an economic perspective.; citation_author=C Perrings; citation_author=M Williamson; citation_author=E Barbier; citation_author=D Delfino; citation_author=S Dalmazzone; citation_journal_title=Conservation Ecology; citation_volume=6; citation_number=10; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The genetic diversity of Plasmodium vivax populations.; citation_author=L Cui; citation_author=A Escalante; citation_author=M Imwong; citation_author=G Snounou; citation_journal_title=Trends Parasitol; citation_volume=19; citation_number=11; citation_pages=220-226; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Exotic species and fungi: Interactions with fungal, plant and animal communities.; citation_author=D Rizzo; citation_number=12; citation_pages=857-880; citation_date=2005; citation_publisher=CRC Press; " />
      <meta name="citation_reference" content="citation_title=Ergot: A New Disease Threat to Sorghum in the Americas and Australia.; citation_author=R Bandyopadhyay; citation_author=DE Frederickson; citation_author=NW McLaren; citation_author=GN Odvody; citation_author=MJ Ryley; citation_journal_title=Plant Disease; citation_volume=82; citation_number=13; citation_pages=356-367; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=African and Asian Dust: From Desert Soils to Coral Reefs.; citation_author=V Garrison; citation_author=E Shinn; citation_author=W Foreman; citation_author=D Griffin; citation_author=C Holmes; citation_journal_title=BioScience; citation_volume=53; citation_number=14; citation_pages=469-480; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Aerial Dispersal of Pathogens on the Global and Continental Scales and Its Impact on Plant Disease.; citation_author=J Brown; citation_author=M Hovmøller; citation_journal_title=Science; citation_volume=297; citation_number=15; citation_pages=537-541; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Evolutionary Ecology of Novel Plant-Pathogen Interactions.; citation_author=IM Parker; citation_author=GS Gilbert; citation_journal_title=Annu Rev Ecol Evol Syst; citation_volume=35; citation_number=16; citation_pages=675-700; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Man-Made Plant Diseases.; citation_author=CE Yarwood; citation_journal_title=Science; citation_volume=168; citation_number=17; citation_pages=218-220; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=The population biology of invasive species.; citation_author=AK Sakai; citation_author=FW Allendorf; citation_author=JS Holt; citation_author=DM Lodge; citation_author=J Molofsky; citation_journal_title=Annu Rev Ecol Syst; citation_volume=32; citation_number=18; citation_pages=305-332; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Biogeographic range expansion into South America by Coccidioides immitis mirrors New World patterns of human migration.; citation_author=M Fisher; citation_author=G Koenig; citation_author=T White; citation_author=G San-Blas; citation_author=R Negroni; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=19; citation_pages=4558-4562; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Disease surveillance in recombining pathogens: multilocus genotypes identify sources of human Coccidioides infections.; citation_author=M Fisher; citation_author=B Rannala; citation_author=V Chaturvedi; citation_author=J Taylor; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=20; citation_pages=9067-9071; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Migration patterns among global populations of the pathogenic fungus Mycosphaerella graminicola.; citation_author=S Banke; citation_author=BA McDonald; citation_journal_title=Mol Ecol; citation_volume=14; citation_number=21; citation_pages=1881-1896; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Global migration patterns in the fungal wheat pathogen Phaeosphaeria nodorum.; citation_author=E Stukenbrock; citation_author=S Banke; citation_author=B McDonald; citation_journal_title=Mol Ecol; citation_volume=15; citation_number=22; citation_pages=2895-2904; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Bottleneck Effect and Genetic Variability in Populations.; citation_author=M Nei; citation_author=T Maruyama; citation_author=R Chakraborty; citation_journal_title=Evolution; citation_volume=29; citation_number=23; citation_pages=1-10; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=Genetic hitchhiking.; citation_author=N Barton; citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_volume=355; citation_number=24; citation_pages=1553-1562; citation_date=2000; " />
      <meta name="citation_reference" content="citation_author=M Jobling; citation_author=M Hurles; citation_author=C Tyler-Smith; citation_number=25; citation_pages=523; citation_date=2004; citation_publisher=Garland Science Publishing; " />
      <meta name="citation_reference" content="citation_title=Genetic variation in eastern North American and putatively introduced populations of Ceratocystis fimbriata f. platani.; citation_author=CJB Engelbrecht; citation_author=TC Harrington; citation_author=J Steimel; citation_author=P Capretti; citation_journal_title=Mol Ecol; citation_volume=13; citation_number=26; citation_pages=2995-3005; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Microsatellite markers identify three lineages of Phytophthora ramorum in US nurseries, yet single lineages in US forest and European nursery populations.; citation_author=K Ivors; citation_author=M Garbelotto; citation_author=IDE Vries; citation_author=C Ruyter-Spira; citation_author=BT Hekkert; citation_journal_title=Mol Ecol; citation_volume=15; citation_number=27; citation_pages=1493-1505; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Description and Power Analysis of Two Tests for Detecting Recent Population Bottlenecks From Allele Frequency Data.; citation_author=J-M Cornuet; citation_author=G Luikart; citation_journal_title=Genetics; citation_volume=144; citation_number=28; citation_pages=2001-2014; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Founder effects and stochastic dispersal at the continental scale of the fungal pathogen of bananas Mycosphaerella fijiensis.; citation_author=G-G Rivas; citation_author=M-F Zapater; citation_author=C Abadie; citation_author=J Carlier; citation_journal_title=Mol Ecol; citation_volume=13; citation_number=29; citation_pages=471-482; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The Evolution of Asexual Fungi: Reproduction, Speciation and Classification.; citation_author=JW Taylor; citation_author=D Jacobson; citation_author=M Fisher; citation_journal_title=Annu Rev Phytopathol; citation_volume=37; citation_number=30; citation_pages=197-246; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Recombination and the multilocus structure of fungal populations.; citation_author=MG Milgroom; citation_journal_title=Annu Rev Phytopathol; citation_volume=34; citation_number=31; citation_pages=457-477; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Patterns of linkage disequilibrium in the human genome.; citation_author=K Ardlie; citation_author=L Kruglyak; citation_author=M Seielstad; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=32; citation_pages=299-309; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Recombination in Magnaporthe grisea.; citation_author=RS Zeigler; citation_journal_title=Annu Rev Phytopathol; citation_volume=36; citation_number=33; citation_pages=249-275; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Invasions by the late blight pathogen: renewed sex and enhanced fitness.; citation_author=C Smart; citation_author=W Fry; citation_journal_title=Biological Invasions; citation_volume=3; citation_number=34; citation_pages=235-243; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evidence for recent population expansion in the evolutionary history of the malaria vectors Anopheles arabiensis and Anopheles gambiae.; citation_author=M Donnelly; citation_author=M Licht; citation_author=T Lehmann; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=35; citation_pages=1353-1364; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Apple Scab: Biology, Epidemiology and Management.; citation_author=W McHardy; citation_number=36; citation_date=1996; citation_publisher=The American Phytopathological Society; " />
      <meta name="citation_reference" content="citation_title=Deciphering host migrations and origins by means of their microbes.; citation_author=T Wirth; citation_author=A Meyer; citation_author=M Achtman; citation_journal_title=Mol Ecol; citation_volume=14; citation_number=37; citation_pages=3289-3306; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Story Of The Apple.; citation_author=B Juniper; citation_author=D Mabberley; citation_number=38; citation_pages=236; citation_date=2006; citation_publisher=Timber Press; " />
      <meta name="citation_reference" content="citation_title=Genetic clues to the origin of the apple.; citation_author=S Harris; citation_author=J Robinson; citation_author=B Juniper; citation_journal_title=Trends Genet; citation_volume=18; citation_number=39; citation_pages=426-430; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Silk Road.; citation_author=F Wood; citation_number=40; citation_pages=270; citation_date=2003; citation_publisher=The Folio Society; " />
      <meta name="citation_reference" content="citation_title=Plant Evolution and the Origin of Crop Species.; citation_author=JF Hancock; citation_number=41; citation_pages=313; citation_date=2004; citation_publisher=CABI Publishing; " />
      <meta name="citation_reference" content="citation_title=The Book of Apples.; citation_author=J Morgan; citation_author=A Richards; citation_number=42; citation_pages=304; citation_date=1993; citation_publisher=Ebury Press; " />
      <meta name="citation_reference" content="citation_title=Subdivision and genetic structure of four populations of Venturia inaequalis in Switzerland.; citation_author=I Tenzer; citation_author=C Gessler; citation_journal_title=Eur J Plant Pathol; citation_volume=103; citation_number=43; citation_pages=565-571; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genetic Diversity of Venturia inaequalis Across Europe.; citation_author=I Tenzer; citation_author=C Gessler; citation_journal_title=Eur J Plant Pathol; citation_volume=105; citation_number=44; citation_pages=545-552; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Identification of Microsatellite Markers and Their Application to Population Genetics of Venturia inaequalis.; citation_author=I Tenzer; citation_author=S degli Ivanissevich; citation_author=M Morgante; citation_author=C Gessler; citation_journal_title=Phytopathology; citation_volume=89; citation_number=45; citation_pages=748-753; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Fungal multilocus sequence typing–it's not just for bacteria.; citation_author=JW Taylor; citation_author=M Fisher; citation_journal_title=Curr Opin Microbiol; citation_volume=6; citation_number=46; citation_pages=351-356; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Chance and selection in the evolution of barley mildew.; citation_author=JKM Brown; citation_journal_title=Trends Microbiol; citation_volume=2; citation_number=47; citation_pages=470-475; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Breakdown of the scab resistance gene Vf in apple leads to a founder effect in population of the fungal pathogen Venturia inaequalis.; citation_author=F Guérin; citation_author=B Le Cam; citation_journal_title=Phytopathology; citation_volume=94; citation_number=48; citation_pages=364-369; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Origin and colonization history of newly virulent strains of the phytopathogenic fungus Venturia inaequalis.; citation_author=F Guérin; citation_author=P Gladieux; citation_author=B Le Cam; citation_journal_title=Fungal Genet Biol; citation_volume=44; citation_number=49; citation_pages=284-292; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Analysis of Molecular Variance Inferred From Metric Distances Among DNA Haplotypes: Application to Human Mitochondrial DNA Restriction Data.; citation_author=L Excoffier; citation_author=PE Smouse; citation_author=JM Quattro; citation_journal_title=Genetics; citation_volume=131; citation_number=50; citation_pages=479-491; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=genalex 6: genetic analysis in Excel. Population genetic software for teaching and research.; citation_author=ROD Peakall; citation_author=PE Smouse; citation_journal_title=Mol Ecol Notes; citation_volume=6; citation_number=51; citation_pages=288-295; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Isolation of 21 new polymorphic microsatellite loci in the phytopathogenic fungus Venturia inaequalis.; citation_author=F Guérin; citation_author=P Franck; citation_author=A Loiseau; citation_author=M Devaux; citation_author=B Le Cam; citation_journal_title=Mol Ecol Notes; citation_volume=4; citation_number=52; citation_pages=268-270; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Arlequin (version 3.0): An integrated software package for.; citation_author=L Excoffier; citation_author=G Laval; citation_author=S Schneider; citation_journal_title=Evolutionary Bioinformatics Online; citation_volume=1; citation_number=53; citation_pages=47-50; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The global genetic structure of the wheat pathogen Mycosphaerella graminicola is characterized by high nuclear diversity, low mitochondrial diversity, regular recombination, and gene flow.; citation_author=J Zhan; citation_author=R Pettway; citation_author=B McDonald; citation_journal_title=Fungal Genet Biol; citation_volume=38; citation_number=54; citation_pages=286-297; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Sexual Reproduction Plays a Major Role in the Genetic Structure of Populations of the Fungus Mycosphaerella graminicola.; citation_author=RS Chen; citation_author=BA McDonald; citation_journal_title=Genetics; citation_volume=142; citation_number=55; citation_pages=1119-1127; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Analysis of Gene Diversity in Subdivided Populations.; citation_author=M Nei; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=70; citation_number=56; citation_pages=3321-3323; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=High level of genetic differentiation for allelic richness among populations of the argan tree [Argania spinosa (L.) Skeels] endemic to Morocco.; citation_author=A el Mousadik; citation_author=RJ Petit; citation_journal_title=Theor Appl Genet; citation_volume=92; citation_number=57; citation_pages=832-839; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Estimating allelic richness: Effects of sample size and bottlenecks.; citation_author=PL Leberg; citation_journal_title=Mol Ecol; citation_volume=11; citation_number=58; citation_pages=2445-2449; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Multilocus structure of natural populations of Hordeum spontaneum.; citation_author=AHD Brown; citation_author=MW Feldman; citation_author=E Nevo; citation_journal_title=Genetics; citation_volume=96; citation_number=59; citation_pages=523-536; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Indices of multilocus linkage disequilibrium.; citation_author=P-M Agapow; citation_author=A Burt; citation_journal_title=Mol Ecol Notes; citation_volume=1; citation_number=60; citation_pages=101-102; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Global Hierarchical Gene Diversity Analysis Suggests the Fertile Crescent Is Not the Center of Origin of the Barley Scald Pathogen Rhynchosporium secalis.; citation_author=P Zaffarano; citation_author=B McDonald; citation_author=M Zala; citation_author=C Linde; citation_journal_title=Phytopathology; citation_volume=96; citation_number=61; citation_pages=941-950; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=BOTTLENECK: a computer program for detecting recent reductions in the effective size using allele frequency data.; citation_author=S Piry; citation_author=G Luikart; citation_author=J-M Cornuet; citation_journal_title=J Hered; citation_volume=90; citation_number=62; citation_pages=502-503; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Signatures of population expansion in microsatellite repeat data.; citation_author=M Kimmel; citation_author=R Chakraborty; citation_author=J King; citation_author=M Bamshad; citation_author=W Watkins; citation_journal_title=Genetics; citation_volume=148; citation_number=63; citation_pages=1921-1930; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A Power Analysis of Microsatellite-Based Statistics for Inferring Past Population Growth.; citation_author=JP King; citation_author=M Kimmel; citation_author=R Chakraborty; citation_journal_title=Mol Biol Evol; citation_volume=17; citation_number=64; citation_pages=1859-1868; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genetic evidence for a Paleolithic human population expansion in Africa.; citation_author=DE Reich; citation_author=DB Goldstein; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=95; citation_number=65; citation_pages=8119-8123; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Statistical Properties of Two Tests that Use Multilocus Data Sets to Detect Population Expansions.; citation_author=DE Reich; citation_author=MW Feldman; citation_author=DB Goldstein; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=66; citation_pages=453-466; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic Analysis: Models and Estimation Procedures.; citation_author=L Cavalli-Sforza; citation_author=A Edwards; citation_journal_title=Evolution; citation_volume=21; citation_number=67; citation_pages=550-570; citation_date=1967; " />
      <meta name="citation_reference" content="citation_title=microsatellite analyser (MSA): a platform independent analysis tool for large microsatellite data sets.; citation_author=D Dieringer; citation_author=C Schlotterer; citation_journal_title=Mol Ecol Notes; citation_volume=3; citation_number=68; citation_pages=167-169; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Inference of Population Structure Using Multilocus Genotype Data.; citation_author=JK Pritchard; citation_author=M Stephens; citation_author=P Donnelly; citation_journal_title=Genetics; citation_volume=155; citation_number=69; citation_pages=945-959; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Inference of Population Structure Using Multilocus Genotype Data: Linked Loci and Correlated Allele Frequencies.; citation_author=D Falush; citation_author=M Stephens; citation_author=JK Pritchard; citation_journal_title=Genetics; citation_volume=164; citation_number=70; citation_pages=1567-1587; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Bayesian Analysis of Genetic Differentiation Between Populations.; citation_author=J Corander; citation_author=P Waldmann; citation_author=MJ Sillanpaa; citation_journal_title=Genetics; citation_volume=163; citation_number=71; citation_pages=367-374; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=GENECLASS2: A Software for Genetic Assignment and First-Generation Migrant Detection.; citation_author=S Piry; citation_author=A Alapetite; citation_author=J-M Cornuet; citation_author=D Paetkau; citation_author=L Baudouin; citation_journal_title=J Hered; citation_volume=95; citation_number=72; citation_pages=536-539; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Detecting immigration by using multilocus genotypes.; citation_author=B Rannala; citation_author=JL Mountain; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=94; citation_number=73; citation_pages=9197-920; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genetic assignment methods for the direct, real-time estimation of migration rate: a simulation-based exploration of accuracy and power.; citation_author=D Paetkau; citation_author=R Slade; citation_author=M Burden; citation_author=A Estoup; citation_journal_title=Mol Ecol; citation_volume=13; citation_number=74; citation_pages=55-65; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Maximum likelihood estimation of a migration matrix and effective population sizes in n subpopulations by using a coalescent approach.; citation_author=P Beerli; citation_author=J Felsenstein; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=75; citation_pages=4563-4568; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Can assignment tests measure dispersal?; citation_author=O Berry; citation_author=MD Tocher; citation_author=SD Sarre; citation_journal_title=Mol Ecol; citation_volume=13; citation_number=76; citation_pages=551-561; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Assignment methods: matching biological questions with appropriate techniques.; citation_author=S Manel; citation_author=O Gaggiotti; citation_author=R Waples; citation_journal_title=Trends Ecol Evol; citation_volume=20; citation_number=77; citation_pages=136-142; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Using mitochondrial and nuclear DNA markers to reconstruct human evolution.; citation_author=L Jorde; citation_author=M Bamshad; citation_author=A Rogers; citation_journal_title=Bioessays; citation_volume=20; citation_number=78; citation_pages=126-136; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The origin and colonization history of the barley scald pathogen Rhynchosporium secalis.; citation_author=PC Brunner; citation_author=S Schurch; citation_author=BA McDonald; citation_journal_title=J Evol Biol; citation_volume=20; citation_number=79; citation_pages=1-12; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Origins of Host-Specific Populations of the Blast Pathogen Magnaporthe oryzae in Crop Domestication With Subsequent Expansion of Pandemic Clones on Rice and Weeds of Rice.; citation_author=BC Couch; citation_author=I Fudal; citation_author=M-H Lebrun; citation_author=D Tharreau; citation_author=B Valent; citation_journal_title=Genetics; citation_volume=170; citation_number=80; citation_pages=613-630; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An Andean origin of Phytophthora infestans inferred from mitochondrial and nuclear gene genealogies.; citation_author=L Gomez-Alpizar; citation_author=I Carbone; citation_author=JB Ristaino; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=104; citation_number=81; citation_pages=3306-3311; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Evidence for the dispersal of a unique lineage from Asia to America and Africa in the sugarcane fungal pathogen Ustilago scitaminea.; citation_author=L-M Raboin; citation_author=A Selvi; citation_author=KM Oliveira; citation_author=F Paulet; citation_author=C Calatayud; citation_journal_title=Fungal Genet Biol; citation_volume=44; citation_number=82; citation_pages=64-76; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Origin and domestication of the fungal wheat pathogen Mycosphaerella graminicola via sympatric speciation.; citation_author=E Stukenbrock; citation_author=S Banke; citation_author=M Javan-Nikkhah; citation_author=B McDonald; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=83; citation_pages=398-411; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Panglobal Distribution of a Single Clonal Lineage of the Irish Potato Famine Fungus.; citation_author=SB Goodwin; citation_author=BA Cohen; citation_author=WE Fry; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=91; citation_number=84; citation_pages=11591-11595; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Comparison of genotypic diversity in native and introduced populations of Sphaeropsis sapinea isolated from Pinus radiata.; citation_author=T Burgess; citation_author=B Wingfield; citation_author=M Wingfield; citation_journal_title=Mycological Research; citation_volume=105; citation_number=85; citation_pages=1331-1339; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=An initial assessment of genetic relationships among populations of Fusarium circinatum in different parts of the world.; citation_author=K Wikler; citation_author=TR Gordon; citation_journal_title=Can J Bot; citation_volume=78; citation_number=86; citation_pages=709-717; citation_date=2000; " />
      <meta name="citation_reference" content="citation_number=87; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Population structure, stepwise mutations, heterozygote deficiency and their implications in DNA forensics.; citation_author=L Jin; citation_author=R Chakraborty; citation_journal_title=Heredity; citation_volume=74; citation_number=88; citation_pages=274-285; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Evolutionary studies of malaria vectors.; citation_author=M Donnelly; citation_author=F Simard; citation_author=T Lehmann; citation_journal_title=Trends Parasitol; citation_volume=18; citation_number=89; citation_pages=75-80; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=New Populations of Sclerotinia sclerotiorum from Lettuce in California and Peas and Lentils in Washington.; citation_author=G Malvarez; citation_author=I Carbone; citation_author=NJ Grunwald; citation_author=KV Subbarao; citation_author=M Schafer; citation_journal_title=Phytopathology; citation_volume=97; citation_number=90; citation_pages=470-483; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Nonneutral Admixture of Immigrant Genotypes in African Drosophila melanogaster Populations from Zimbabwe.; citation_author=M Kauer; citation_author=D Dieringer; citation_author=C Schlotterer; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=91; citation_pages=1329-1337; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Microsatellite variation and differentiation in African and non-African populations of Drosophila simulans.; citation_author=G Schofl; citation_author=C Schlotterer; citation_journal_title=Mol Ecol; citation_volume=15; citation_number=92; citation_pages=3895-3905; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Population structure in admixed populations: effect of admixture dynamics on the pattern of linkage disequilibrium.; citation_author=C Pfaff; citation_author=E Parra; citation_author=C Bonilla; citation_author=K Hiester; citation_author=P McKeigue; citation_journal_title=Am J Hum Genet; citation_volume=68; citation_number=93; citation_pages=198-207; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Studies of the ascigerous stage of Venturia inaequalis.; citation_author=E Wilson; citation_journal_title=Phytopathology; citation_volume=18; citation_number=94; citation_pages=375-418; citation_date=1928; " />
      <meta name="citation_reference" content="citation_title=Microsatellite Primers Indicate the Presence of Asexual Populations of Venturia inaequalis in Coastal Israeli Apple Orchards.; citation_author=E Boehm; citation_author=S Freeman; citation_author=E Shabi; citation_author=T Michailides; citation_journal_title=Phytoparasitica; citation_volume=31; citation_number=95; citation_pages=236-251; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Human migration, mosquitoes and the evolution of Plasmodium falciparum.; citation_author=J Hume; citation_author=E Lyons; citation_author=K Day; citation_journal_title=Trends Parasitol; citation_volume=19; citation_number=96; citation_pages=144-149; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Early origin and recent expansion of Plasmodium falciparum.; citation_author=D Joy; citation_author=X Feng; citation_author=J Mu; citation_author=T Furuya; citation_author=K Chotivanich; citation_journal_title=Science; citation_volume=300; citation_number=97; citation_pages=318-321; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001455" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="On the Origin and Spread of the Scab Disease of Apple: Out of Central Asia"/>
    <meta name="twitter:description" content="BackgroundVenturia inaequalis is an ascomycete fungus responsible for apple scab, a disease that has invaded almost all apple growing regions worldwide, with the corresponding adverse effects on apple production. Monitoring and predicting the effectiveness of intervention strategies require knowledge of the origin, introduction pathways, and population biology of pathogen populations. Analysis of the variation of genetic markers using the inferential framework of population genetics offers the potential to retrieve this information.Methodology/Principal FindingsHere, we present a population genetic analysis of microsatellite variation in 1,273 strains of V. inaequalis representing 28 orchard samples from seven regions in five continents. Analysis of molecular variance revealed that most of the variation (88%) was distributed within localities, which is consistent with extensive historical migrations of the fungus among and within regions. Despite this shallow population structure, clustering analyses partitioned the data set into separate groups corresponding roughly to geography, indicating that each region hosts a distinct population of the fungus. Comparison of the levels of variability among populations, along with coalescent analyses of migration models and estimates of genetic distances, was consistent with a scenario in which the fungus emerged in Central Asia, where apple was domesticated, before its introduction into Europe and, more recently, into other continents with the expansion of apple growing. Across the novel range, levels of variability pointed to multiple introductions and all populations displayed signatures of significant post-introduction increases in population size. Most populations exhibited high genotypic diversity and random association of alleles across loci, indicating recombination both in native and introduced areas.Conclusions/SignificanceVenturia inaequalis is a model of invasive phytopathogenic fungus that has now reached the ultimate stage of the invasion process with a broad geographic distribution and well-established populations displaying high genetic variability, regular sexual reproduction, and demographic expansion."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001455.g004"/>

  <meta property="og:title" content="On the Origin and Spread of the Scab Disease of Apple: Out of Central Asia" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001455" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=792'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8683'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8022&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001455">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001455" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001455&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    On the Origin and Spread of the Scab Disease of Apple: Out of Central Asia
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pierre Gladieux, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  UMR077, INRA, Angers, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiu-Guo Zhang
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:zhxg@sdau.edu.cn">zhxg@sdau.edu.cn</a> (XGZ); <a href="mailto:lecam@angers.inra.fr">lecam@angers.inra.fr</a> (BLC)</p>

                <p>Affiliation:
                  Department of Plant Pathology, Shandong Agricultural University, Taian, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Damien Afoufa-Bastien, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  UMR077, INRA, Angers, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rosa-Maria Valdebenito Sanhueza, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Research Centre for Grape and Wine, EMBRAPA, Bento Gonçalves, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mohamed Sbaghi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre Régional de la Recherche Agronomique de Kenitra, INRA, Kenitra, Morocco
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bruno Le Cam
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:zhxg@sdau.edu.cn">zhxg@sdau.edu.cn</a> (XGZ); <a href="mailto:lecam@angers.inra.fr">lecam@angers.inra.fr</a> (BLC)</p>

                <p>Affiliation:
                  UMR077, INRA, Angers, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 16, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001455</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001455" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001455" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001455" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001455" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001455-t001" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001455-g001" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001455-t002" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001455-t003" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001455-t004" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001455-t005" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001455-g002" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001455-g003" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001455-g004" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001455-t006" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001455">Reader Comments (4)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001455" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p><em>Venturia inaequalis</em> is an ascomycete fungus responsible for apple scab, a disease that has invaded almost all apple growing regions worldwide, with the corresponding adverse effects on apple production. Monitoring and predicting the effectiveness of intervention strategies require knowledge of the origin, introduction pathways, and population biology of pathogen populations. Analysis of the variation of genetic markers using the inferential framework of population genetics offers the potential to retrieve this information.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here, we present a population genetic analysis of microsatellite variation in 1,273 strains of <em>V. inaequalis</em> representing 28 orchard samples from seven regions in five continents. Analysis of molecular variance revealed that most of the variation (88%) was distributed within localities, which is consistent with extensive historical migrations of the fungus among and within regions. Despite this shallow population structure, clustering analyses partitioned the data set into separate groups corresponding roughly to geography, indicating that each region hosts a distinct population of the fungus. Comparison of the levels of variability among populations, along with coalescent analyses of migration models and estimates of genetic distances, was consistent with a scenario in which the fungus emerged in Central Asia, where apple was domesticated, before its introduction into Europe and, more recently, into other continents with the expansion of apple growing. Across the novel range, levels of variability pointed to multiple introductions and all populations displayed signatures of significant post-introduction increases in population size. Most populations exhibited high genotypic diversity and random association of alleles across loci, indicating recombination both in native and introduced areas.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p><em>Venturia inaequalis</em> is a model of invasive phytopathogenic fungus that has now reached the ultimate stage of the invasion process with a broad geographic distribution and well-established populations displaying high genetic variability, regular sexual reproduction, and demographic expansion.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Gladieux P, Zhang X-G, Afoufa-Bastien D, Valdebenito Sanhueza R-M, Sbaghi M, et al.  (2008) On the Origin and Spread of the Scab Disease of Apple: Out of Central Asia. PLoS ONE 3(1):
          e1455.
            doi:10.1371/journal.pone.0001455</p><p><strong>Academic Editor: </strong>Christophe d'Enfert, Institut Pasteur, France</p><p><strong>Received:</strong> March 12, 2007; <strong>Accepted:</strong> December 20, 2007; <strong>Published:</strong> January 16, 2008</p><p><strong>Copyright:</strong> © 2008 Gladieux et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Part of this work was carried out by using the resources of the Computational Biology Service Unit from Cornell University which is partially funded by Microsoft Corporation.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Biological invasions <a href="#pone.0001455-Facon1">[1]</a> by plant-pathogenic fungi are an unfortunate side effect of globalization, climate change, and more generally of the domestication of nature <a href="#pone.0001455-Anderson1">[2]</a>–<a href="#pone.0001455-Pimentel1">[4]</a>. The Irish potato famine oomycete <em>Phytophthora infestans</em> and the chestnut blight ascomycete <em>Cryphonectria parasitica</em> are notorious examples of invasive phytopathogenic fungi that caused devastating epidemics <a href="#pone.0001455-Anagnostakis1">[5]</a>, <a href="#pone.0001455-Fry1">[6]</a>. Of course, invasions do not always have tragic consequences, but invasive phytopathogenic fungi have had and continue to have diffuse and pernicious impact on agrosystems, ecosystems, and human populations dependent on them <a href="#pone.0001455-Pimentel1">[4]</a>, <a href="#pone.0001455-Palm1">[7]</a>, <a href="#pone.0001455-Wingfield1">[8]</a>. Because attempts to eradicate established invasive phytopathogenic fungi have met little success, the highest priorities should be given to preventing the introduction and limiting the spread and impact of established invaders <a href="#pone.0001455-Campbell1">[9]</a>. The implementation of sound risk-based phytosanitary programs requires a genuinely interdisciplinary approach to seek out and utilize all available information (i) on origins, past and present introduction pathways, and population biology of invasive phytopathogenic fungi; (ii) on the interactions between social, economic, and natural processes; and (iii) on mitigation or alleviation technologies <a href="#pone.0001455-Campbell1">[9]</a>, <a href="#pone.0001455-Perrings1">[10]</a>. In this paper, we focus on the first point, which has important applications for monitoring and predicting the effectiveness of intervention strategies <a href="#pone.0001455-Cui1">[11]</a>. The origin and introduction routes of many invasive phytopathogenic fungi are unknown, even for those causing major economic and ecological impact. One reason is that many introductions occurred when very little attention was paid to risks associated with the disease, as neither the nature of the cause of diseases nor the way in which they spread were understood <a href="#pone.0001455-Palm1">[7]</a>, <a href="#pone.0001455-Wingfield1">[8]</a>. Some invasive phytopathogenic fungi spread so long ago that it probably does not come to mind that they are invasive <a href="#pone.0001455-Palm1">[7]</a>; others have so broad distributions that they are listed as cosmopolitan, though they were initially restricted to a specific area <a href="#pone.0001455-Rizzo1">[12]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Although some invasive phytopathogenic fungi can naturally move over broad geographic areas (e.g., <em>Claviceps africana</em>, <a href="#pone.0001455-Bandyopadhyay1">[13]</a>) or even overseas (e.g., <em>Aspergillus sydowii</em> <a href="#pone.0001455-Garrison1">[14]</a> or <em>Hemileia vastatrix</em> <a href="#pone.0001455-Brown1">[15]</a>), most long-distance movements are assisted by human activities <a href="#pone.0001455-Rizzo1">[12]</a>. Introductions can be deliberate as in the case of biocontrol agents or the unintended consequence of decisions involving the use of nonindigenous species in agriculture and forestry, alteration of habitat, or movements of goods and people <a href="#pone.0001455-Perrings1">[10]</a>, <a href="#pone.0001455-Rizzo1">[12]</a>, <a href="#pone.0001455-Parker1">[16]</a>. The domestication and spread of agricultural food crops provided opportunities for invasions by phytopathogenic fungi. The spread of agriculture and the globalization of travel and trade were associated with extensive movements of crop species and plant products that allowed accidental transportations of fungal pathogens far from their native range <a href="#pone.0001455-Yarwood1">[17]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In the absence of detailed information on the origin, introduction pathways and population biology of invasive phytopathogenic fungi, analysis of the variation of molecular markers in the framework of population genetics theory can serve as a powerful alternative. In the vocabulary of population genetics, bioinvasions are rapid range expansions involving four steps: movement, arrival, establishment, and spread <a href="#pone.0001455-Rizzo1">[12]</a>, <a href="#pone.0001455-Sakai1">[18]</a>. The rationale behind population genetics inference is that each of these steps leaves an imprint in the distribution of genetic variation within and among populations (i.e., the population structure) that can help distinguish among possible competing hypotheses on the history of the bioinvasion process and the population biology of the invasive species.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The first hurdle for invasive phytopathogenic fungi is to arrive in the new range by exploiting introduction pathways <a href="#pone.0001455-Parker1">[16]</a>. Various source populations can contribute to the genetic makeup of introduced populations, and several methods exist that allow determining historical source and sink patterns of migration among populations. For instance, Fisher et al. <a href="#pone.0001455-Fisher1">[19]</a> and Fisher et al. <a href="#pone.0001455-Fisher2">[20]</a> used measures of genetic similarity among genotypes and assignment methods to infer the source populations for isolates of the human pathogen <em>Coccidioides immitis</em> collected outside the endemic area of the fungus. For the wheat pathogens <em>Mycosphaerella graminicola</em> and <em>Phaeosphaeria nodorum</em>, Banke and Mc Donald <a href="#pone.0001455-Banke1">[21]</a> and Stukenbrock et al. <a href="#pone.0001455-Stukenbrock1">[22]</a> used coalescent-based estimates of gene flow to analyze historical patterns of global migration into or among new territories.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Introduction events may involve a population bottleneck because the number of initial colonists is often small <a href="#pone.0001455-Sakai1">[18]</a>, <a href="#pone.0001455-Nei1">[23]</a>. Loss of alleles and reduction in genetic variation can also occur during the early stages of establishment because of random genetic drift due to small population size and selective pressure exerted by novel environments <a href="#pone.0001455-Parker1">[16]</a>, <a href="#pone.0001455-Barton1">[24]</a>. Thus, a newly established population is likely to be much less variable than the older population(s) from which it is derived, and populations from the centre of origin of the invasive phytopathogenic fungi are expected to be the most variable <a href="#pone.0001455-Jobling1">[25]</a>. <em>Ceratocystis fimbriata</em> and <em>Phytophthora ramorum</em>, causal agents of canker stain of plane tree and sudden oak death, are examples of invasive phytopathogenic fungi that have very limited variation in their area of introduction <a href="#pone.0001455-Engelbrecht1">[26]</a>, <a href="#pone.0001455-Ivors1">[27]</a>. For other invasive phytopathogenic fungi such as <em>Phaeosphaeria nodorum</em> or <em>Sphaeropsis sapinea</em>, responsible for leaf and glume blotch of wheat and pine pitch canker, population genetics studies found more substantial levels of variation, pointing toward multiple introduction events <a href="#pone.0001455-Wingfield1">[8]</a>, <a href="#pone.0001455-Stukenbrock1">[22]</a>. Genetic bottlenecks may also transitorily throw populations out of mutation-drift equilibrium <a href="#pone.0001455-Cornuet1">[28]</a>. Rivas et al. <a href="#pone.0001455-Rivas1">[29]</a> used this approach to show that African, Latin American, and Caribbean populations of the causal agent of black leaf streak disease of bananas (<em>M. fijiensis</em>) had been recently founded.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Moving a fungus from its native biogeographic range to a novel environment can also change its population structure and reproductive mode <a href="#pone.0001455-Taylor1">[30]</a>. Random association among alleles from different loci is a reasonable null hypothesis for fungi known to have a sexual stage <a href="#pone.0001455-Milgroom1">[31]</a>. However, even for a normally recombining population, factors such as foundation by a limited number of individuals, random genetic drift in small populations, or immigration of individuals from populations with different allele frequencies can artificially create nonrandom associations between unlinked markers (i.e., linkage disequilibrium) at the time of colonization of a new territory. This spurious linkage disequilibrium should quickly dissipate with periodic recombination and population growth <a href="#pone.0001455-Ardlie1">[32]</a>, provided the introduced population has not lost sexual competence. Indeed, chance effects such as establishment of only one mating type, mating-type linkage to avirulence genes, or introduction into an environment not being conducive to meiospore development can result in the establishment of nonsexual populations <a href="#pone.0001455-Zeigler1">[33]</a>. A well-documented illustration of this phenomenon is the movement of <em>P. infestans</em> outside Mexico: the founding of European and North American populations by a single genotype of the A1 mating type rendered reproduction exclusively mitotic for 120 years <a href="#pone.0001455-Smart1">[34]</a>.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Following movement, arrival, and establishment of a viable population, the final step in a biological invasion is the spread to additional locations within the new territory <a href="#pone.0001455-Rizzo1">[12]</a>. The dispersal mode can range between rare, unpredictable long-distance founder events to a gradual expansion <a href="#pone.0001455-Rivas1">[29]</a>. Depending on factors such as dispersal abilities and reproduction mode of the invasive phytopathogenic fungus, or density and susceptibility of hosts, populations can experience a rapid expansion, producing an increase in effective population size and deviation from mutation-drift equilibrium <a href="#pone.0001455-Donnelly1">[35]</a>. Fisher et al. <a href="#pone.0001455-Fisher1">[19]</a> used this feature to show that South American populations of the human pathogen <em>Coccidiodes immitis</em> had undergone rapid population growth, indicating an epidemic increase in postcolonization population size.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>The fungal pathogen <em>Venturia inaequalis</em> is the agent of the scab disease of apple, the most important disease in apple production. <em>Venturia inaequalis</em> is a heterothallic haploid ascomycete that reproduces both sexually and asexually <a href="#pone.0001455-McHardy1">[36]</a>. During winter, the fungus grows as a saprobe in dead apple leaves and produces meiospores (ascospores). In spring, when temperature and moisture conditions are favorable, ascospores are released and dispersed by wind to initiate epidemics. When an ascospore lands on a susceptible fruit or leaf, it germinates and proceeds to form lesions producing mitospores (conidia) that are blown by wind or splashed by rain to cause secondary infections. Both ascospores and conidia have limited dispersal capacities: conidia are only dispersed over a few meters, ascospores do not spread over a hundred meters, and wind distribution of infected leaves probably does not exceed a few kilometers. The only way to achieve long-distance dispersal is man-mediated transportation of infected fruits or plants. Based on this feature, the population structure of the pathogen is expected to mirror historical movements of its host <a href="#pone.0001455-Wirth1">[37]</a>.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>The history of apple is well documented. It is now widely accepted that the centre of origin of apple (<em>Malus</em>×<em>domestica</em>) is in the mountain ranges of Central Asia <a href="#pone.0001455-Juniper1">[38]</a>, <a href="#pone.0001455-Harris1">[39]</a>. As early as Neolithic times (5,000–8,000 years before present), this region was crossed by the famous Silk Roads stretching from Rome in Italy through Samarkand in Uzbekistan to Luoyang in China <a href="#pone.0001455-Wood1">[40]</a>. Travelers, ably assisted by their domesticated animals, progressively began to domesticate and transport apples westward. Apple cultivation likely began in the region between the Caspian and Black seas, and it had reached the Near East by 3,000 years before present <a href="#pone.0001455-Hancock1">[41]</a>. The Romans introduced and spread apple across the European and Mediterranean areas and European settlers transported it into newfound lands during the last 500 years. Apple is now grown in all temperate regions <a href="#pone.0001455-Morgan1">[42]</a>.</p>
<a id="article1.body1.sec1.p10" name="article1.body1.sec1.p10"></a><p>Today, <em>V. inaequalis</em> has invaded all apple-growing regions. The disease has a negative economic impact due to yield losses, the cost of breeding programs aimed at producing resistant varieties and the use of fungicide inputs, with the corresponding environmental and health hazards. Despite this detrimental effect, the population biology of the fungus outside Europe is virtually unknown <a href="#pone.0001455-Tenzer1">[43]</a>–<a href="#pone.0001455-Tenzer3">[45]</a> and, unlike apple, its origin and introduction pathways are not documented. The present study was conducted to make up for this lack of knowledge. We used multilocus microsatellite typing <a href="#pone.0001455-Taylor2">[46]</a> to describe the population structure of a set of samples from Central Asia, Europe, North Africa, and newfound lands (North and South America, Australasia, South Africa). Our analyses revealed that <em>V. inaequalis</em> emerged in Central Asia and followed its host into Europe along the Silk Roads and more recently into newfound lands with the expansion of apple growing. <em>Venturia inaequalis</em> appeared as a model of invasive phytopathogenic fungus that has reached the ultimate stage of the invasion process with a broad geographic distribution and well-established populations displaying high genetic variability, regular sexual reproduction, and demographic expansion [<em>Nota bene</em>: because this study is the fruit of an international collaborative effort, abstracts in Chinese, Portuguese, French and Moroccan are available as Supplementary Information (<a href="#pone.0001455.s007">Text S1</a>)].</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sample collection</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We collected 1,273 individual fungal strains of <em>V. inaequalis</em> from <em>M.</em>×<em>domestica</em> on 28 locations representing seven regions in five continents: Central Asia (Xinjiang Province of China, Iran, Azerbaijan), Europe (France, Sweden, Spain), North Africa (Morocco), South Africa, North America (Canada, USA), South America (Brazil), and Australasia (New Zealand) (<a href="#pone-0001455-t001">Table 1</a>, <a href="#pone.0001455.s004">Figure S1</a>). All samples represented single orchards, except the sample from Canada that originated from several locations and various host cultivars. In each orchard, infected leaves were sampled randomly and we collected only one leaf per apple tree.</p>
<div class="figure" id="pone-0001455-t001"><div class="img"><a name="pone-0001455-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001455.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001455.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Origin of <em>Venturia inaequalis</em> samples.</span></strong></p><span>doi:10.1371/journal.pone.0001455.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Host resistance can induce selective and/or demographic sweeps in fungal populations, leading to lineages highly divergent from populations found on susceptible cultivars <a href="#pone.0001455-Brown2">[47]</a>–<a href="#pone.0001455-Gurin2">[49]</a>. To avoid confounding geographic structure with possible associations between host cultivars and fungal genotypes (i) we sampled on cultivars with no known effective resistance; (ii) we minimized the total number of cultivars sampled by focusing as much as possible on the commercially leading cultivars Fuji, Royal Gala, and Golden Delicious; and (iii) we collected samples on different cultivars at several locations and checked for the absence of associations between host and <em>V. inaequalis</em> genotypes by calculating pairwise <em>φ<sub>ST</sub></em> between samples (an analog of Wright's <em>F<sub>ST</sub></em> fixation index) <a href="#pone.0001455-Excoffier1">[50]</a> using G<span class="small-caps">enalex</span> <a href="#pone.0001455-Peakall1">[51]</a>. As pairwise <em>φ<sub>ST</sub></em> values were low or nonsignificantly different from zero (<a href="#pone.0001455.s002">Table S2</a>), samples from the same location were pooled for all subsequent analyses, except the samples from Mechraâ Bel Ksiri in Morocco. We obtained a total of 29 samples.</p>


<h4>Microsatellite Multilocus Typing</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>DNA was extracted from monoconidial isolates or directly from infected leaf symptoms according to a protocol described in previous studies <a href="#pone.0001455-Gurin1">[48]</a>, <a href="#pone.0001455-Gurin2">[49]</a>. Samples were genotyped at 12 microsatellite loci: <em>1tc1a, 1tc1b, 1tc1g, 1aac3b</em> <a href="#pone.0001455-Tenzer3">[45]</a>, <em>Vitc1/2, Vitcca7/P, Vitg11/70, Vicacg8/42, Vica9/152, Viga7/116, Vica9/X</em> <a href="#pone.0001455-Gurin3">[52]</a>, and <em>M42</em> <a href="#pone.0001455-Gurin2">[49]</a>. Polymerase chain reaction was performed with the fluorescently labeled primers and conditions described previously <a href="#pone.0001455-Gurin3">[52]</a>. Alleles were scored against a fluorescently labeled size standard in an ABI 310 automated sequencer (Applied Biosystems, Foster City, California). Our data set is accessible via the Internet at <a href="http://www.multilocus.net/">http://www.multilocus.net/</a> (<a href="#pone.0001455.s001">Table S1</a>).</p>


<h4>Data analysis</h4>
<h5>Genetic variation within samples.</h5><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1"></a><p>The number of haplotypes was calculated using A<span class="small-caps">rlequin</span> 3.00 <a href="#pone.0001455-Excoffier2">[53]</a>, and it was used to quantify the clonal fraction <a href="#pone.0001455-Zhan1">[54]</a>. We treated multilocus haplotypes repeated multiple times as clones. For all subsequent analyses, we used a data set in which each multilocus haplotype was represented only once in each sample <a href="#pone.0001455-Chen1">[55]</a>.</p>
<a id="article1.body1.sec2.sec3.sec1.p2" name="article1.body1.sec2.sec3.sec1.p2"></a><p>Expected heterozygosity <a href="#pone.0001455-Nei2">[56]</a>, allelic richness, and unique allele richness were computed using scripts written in M<span class="small-caps">atlab</span> (The Mathworks, Natick, Massachusetts). Unique allele richness represents the number of alleles that are unique to a particular sample in comparisons with all other samples, averaged across loci. To account for differences in sample size, samples were standardized to a uniform size equal to the size of the smallest sample (South Africa: 12 individuals) using random draws with replacement (nonparametric bootstrapping) <a href="#pone.0001455-elMousadik1">[57]</a>, <a href="#pone.0001455-Leberg1">[58]</a>. For each sample, expected heterozygosity, allelic richness, and unique allele richness indices were calculated as the average value of 100 bootstrap replicates <a href="#pone.0001455-Zhan1">[54]</a>. We examined correlations between these variability indices and geographical distance calculated as the arc surface distance from the most eastern Chinese sample. Because the variables tested may not be distributed normally, all correlations were nonparametrically tested using Spearman <em>r</em> available in G<span class="small-caps">raphpad</span> (GraphPad Software Inc., San Diego, California).</p>
<a id="article1.body1.sec2.sec3.sec1.p3" name="article1.body1.sec2.sec3.sec1.p3"></a><p>Associations of alleles among different loci were examined in each sample using the index of association (<em>I<sub>A</sub></em>) statistic, which is a generalized measure of multilocus linkage disequilibrium <a href="#pone.0001455-Brown3">[59]</a>. The null hypothesis of random association of alleles (<em>I<sub>A</sub></em> = 0), consistent with random mating, was tested using the program M<span class="small-caps">ultilocus</span> <a href="#pone.0001455-Agapow1">[60]</a> by comparing the observed value of the statistic to that obtained after 1,000 randomizations to simulate recombination.</p>

<h5>Genetic variation among samples.</h5><a id="article1.body1.sec2.sec3.sec2.p1" name="article1.body1.sec2.sec3.sec2.p1"></a><p>We compared levels of genetic variation among regional groups of samples. To account for differences in group sizes, we used nonparametric bootstrapping to standardize group sizes to the size of the smallest group (South Africa: 12 individuals) using a script written in M<span class="small-caps">atlab</span> <a href="#pone.0001455-Zhan1">[54]</a>, <a href="#pone.0001455-Zaffarano1">[61]</a>. Expected heterozygosity, allelic richness, and unique allele richness were calculated as the average value of 100 bootstrap replicates. The level of genetic variation among groups was compared in SPSS 10 (SPSS Inc., Chicago, Illinois) using a one-way ANOVA followed by a post-hoc Tukey test.</p>
<a id="article1.body1.sec2.sec3.sec2.p2" name="article1.body1.sec2.sec3.sec2.p2"></a><p>One-way and two-way hierarchical analyses of molecular variance (AMOVA) were used to partition microsatellite variation among regions, among samples, and within samples <a href="#pone.0001455-Excoffier1">[50]</a>. Only regions with more than two samples were included in analyses. G<span class="small-caps">enalex</span> <a href="#pone.0001455-Peakall1">[51]</a> was used to compute and test the statistical significance of <em>φ-</em>statistics based on 1,000 permutations.</p>

<h5>Demographic history.</h5><a id="article1.body1.sec2.sec3.sec3.p1" name="article1.body1.sec2.sec3.sec3.p1"></a><p>We used three methods designed to detect historical changes in population size from deviations from mutation-drift equilibrium. The first method, implemented in the program B<span class="small-caps">ottleneck</span> <a href="#pone.0001455-Piry1">[62]</a>, compares the expected heterozygosity estimated from allele frequencies with that estimated from the number of alleles and the sample size, which are expected to be identical in a neutral locus in a population at mutation-drift equilibrium. Inferences about historical demographics are based on the prediction that populations that have experienced a recent reduction of their effective size see their expected heterozygosity estimated from allele frequencies reduce faster than that estimated under a given mutation model at mutation-drift equilibrium; the contrary is expected for growing populations <a href="#pone.0001455-Cornuet1">[28]</a>. The tests were performed under the stepwise-mutation model (SMM) as well as under a two-phase model (TPM), allowing for 30% of multistep changes. Significance was tested using two-sided Wilcoxon signed rank tests.</p>
<a id="article1.body1.sec2.sec3.sec3.p2" name="article1.body1.sec2.sec3.sec3.p2"></a><p>The second method relies on the notion that variance- and homozygosity-based estimates of the population mutation rate <em>θ</em> are expected to be equal in a neutral locus in a population at mutation-drift equilibrium <a href="#pone.0001455-Kimmel1">[63]</a>, <a href="#pone.0001455-King1">[64]</a>. The deviation between the two estimators, measured by the imbalance index <em>β</em> (equation 7 in reference <a href="#pone.0001455-King1">[64]</a>), can be used to detect population expansion: Ln <em>β</em> is expected to be negative for populations that have recently expanded from equilibrium initial conditions and positive for populations that have recently expanded following a bottleneck. 95% confidence intervals estimated by bootstrapping over loci were computed using a script written in M<span class="small-caps">atlab</span> <a href="#pone.0001455-Donnelly1">[35]</a>.</p>
<a id="article1.body1.sec2.sec3.sec3.p3" name="article1.body1.sec2.sec3.sec3.p3"></a><p>The third method uses the principle that the variance of the variance in allele lengths is expected to be larger in a constant-sized than in a growing population, assuming that the loci follow an SMM. This difference was quantified using the interlocus <em>g</em> statistic <a href="#pone.0001455-Reich1">[65]</a> and significance was tested using the fifth-percentile cutoffs of Reich et al. <a href="#pone.0001455-Reich2">[66]</a>. Since the <em>β</em> and <em>g</em> statistics assume that loci evolve under the SMM, loci <em>Vica9/152</em> and <em>Vicacg8/42</em> were excluded from the calculations.</p>

<h5>Clustering and assignment methods.</h5><a id="article1.body1.sec2.sec3.sec4.p1" name="article1.body1.sec2.sec3.sec4.p1"></a><p>We used four different methods to determine the optimal number of populations present in our data set, to assess the level of differentiation, to infer the geographic ancestral relationships among these populations, and to identify recently founded populations, as these are expected to cluster with their source population.</p>
<a id="article1.body1.sec2.sec3.sec4.p2" name="article1.body1.sec2.sec3.sec4.p2"></a><p>First, we calculated principal coordinates on Cavalli-Sforza and Edwards' chord distance among samples <a href="#pone.0001455-CavalliSforza1">[67]</a>. The chord distance matrix was built using the M<span class="small-caps">icrosatellite analyzer</span> (MSA 4.00) software <a href="#pone.0001455-Dieringer1">[68]</a>, and principal component analysis was performed under G<span class="small-caps">enalex</span>.</p>
<a id="article1.body1.sec2.sec3.sec4.p3" name="article1.body1.sec2.sec3.sec4.p3"></a><p>Second, we used the Bayesian clustering algorithm implemented in S<span class="small-caps">tructure</span> 2.1 <a href="#pone.0001455-Pritchard1">[69]</a>, <a href="#pone.0001455-Falush1">[70]</a>. This method relies on the Bayesian Monte Carlo Markov Chain (MCMC) approach to cluster individuals into <em>K</em> distinct populations that minimize Hardy-Weinberg disequilibrium and gametic phase disequilibrium between loci within groups. The model allowed individuals to have mixed ancestry and correlation of allele frequencies. Uniform priors were assumed and the MCMC scheme was run for 500,000 iterations after an initial burn-in period of 50,000. We ran S<span class="small-caps">tructure</span> for <em>K</em> ranging from 1 to 13 and we performed at least six repetitions to check for convergence of likelihood values for each value of <em>K</em>. Convergence of the MCMC could not be achieved for <em>K</em> values higher than 13. The number of populations that best represents the observed data under the model implemented was determined by maximizing the estimated Ln likelihood of the data for different values of <em>K</em>.</p>
<a id="article1.body1.sec2.sec3.sec4.p4" name="article1.body1.sec2.sec3.sec4.p4"></a><p>Third, we used the Bayesian clustering algorithm implemented in B<span class="small-caps">aps</span> 4 <a href="#pone.0001455-Corander1">[71]</a> to identify the optimal number <em>K</em> of partitions among groups of samples. By contrast to the individual-based algorithm applied in S<span class="small-caps">tructure</span>, we used the group-level option in B<span class="small-caps">aps</span> such that clusters are formed by assembling whole samples. B<span class="small-caps">aps</span> 4 relies on stochastic optimization to infer the posterior mode of the genetic structure. The program was run for <em>K</em> ranging from 1 to 29 with five replicates for each value of <em>K</em> to ensure that the stochastic optimization algorithm had not ended up in different solutions in separate runs. Goodness-of-fit levels of the clustering solutions to the data set are compared in terms of natural logarithm of the marginal likelihood of the data. We also used B<span class="small-caps">aps</span> to perform an admixture analysis aiming at estimating individual coefficients of ancestry with regard to the inferred clusters of samples. For this analysis, we used 1,000 iterations to estimate the admixture coefficients for the individuals, we used 200 reference individuals from each cluster, and we repeated the admixture analysis 50 times per individual.</p>
<a id="article1.body1.sec2.sec3.sec4.p5" name="article1.body1.sec2.sec3.sec4.p5"></a><p>Fourth, we used G<span class="small-caps">eneclass</span> 2.0 <a href="#pone.0001455-Piry2">[72]</a> to assign individuals to regional groups of samples. The probability of individuals coming from each area was calculated using the standard criterion described by Rannala and Mountain <a href="#pone.0001455-Rannala1">[73]</a> and by simulating 1,000 individuals per regional group of samples using the method of Paetkau et al. <a href="#pone.0001455-Paetkau1">[74]</a>. Individuals were assigned to a regional group when this group had the highest probability of being the source of this individual.</p>

<h5>Gene flow and effective population size.</h5><a id="article1.body1.sec2.sec3.sec5.p1" name="article1.body1.sec2.sec3.sec5.p1"></a><p>We used the program M<span class="small-caps">igrate</span> 2.0 <a href="#pone.0001455-Beerli1">[75]</a> to assess long-term gene flow and effective population sizes and to determine which migration route was most supported by the data. These analyses were performed on regional groups of samples.</p>
<a id="article1.body1.sec2.sec3.sec5.p2" name="article1.body1.sec2.sec3.sec5.p2"></a><p>M<span class="small-caps">igrate</span> uses an expansion of the coalescent theory to estimate migration rates between populations (<em>N<sub>e</sub>m</em>) and <em>θ</em> (2<em>N<sub>e</sub>μ</em>), where <em>N<sub>e</sub></em> is the effective population size, <em>m</em> is the constant migration rate between population pairs, and <em>μ</em> is the mutation rate per generation at the locus considered. Likelihood surfaces for each parameter were estimated by simulating genealogies using an MCMC approach. The computations were carried out under a Brownian motion approximation of the SMM, with the loci <em>Vica9/152</em> and <em>Vicacg8/42</em> excluded from the data set. We evaluated two migration models: a full migration model with unrestricted migration among all groups (Model 1) and a migration model with the Central Asian group exchanging migrants only with European group and unrestricted migration among all non-Central Asian groups (Model 2). The models were run three times to confirm convergence of parameter estimates, and only the results of the run that yielded the highest Ln likelihood value are presented. The runs consisted of two replicates of 10 short chains (with 10,000 genealogies sampled) and three long chains (with 100,000 genealogies sampled), with the first 10,000 genealogies discarded. A likelihood ratio test was used to compare the likelihoods of all models <a href="#pone.0001455-Beerli1">[75]</a>.</p>


</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Polymorphism and multilocus linkage disequilibrium</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Among the 1,273 individuals analyzed, we found 1,180 unique haplotypes based on 12 microsatellite loci, representing a total of 221 different alleles. The number of alleles at each locus ranged from 6 at <em>1aac3b</em> to 32 at <em>1tc1g</em>, with an average value of 18.4 (±7.1 SD).</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Estimates of variation indices for each sample are reported in <a href="#pone-0001455-t002">Table 2</a>. Allelic richness ranged from 2.71 to 4.85 (mean±SD: 3.84±0.62), expected heterozygosity ranged from 0.43 to 0.65 (mean±SD: 0.56±0.06), and unique allele richness ranged from 0.01 to 0.29 (mean±SD: 0.12±0.09). All three variation indices were negatively correlated with arc surface distance from the most eastern Chinese sample (<em>A</em>: <em>r</em> = −0.74, <em>H<sub>E</sub></em>: <em>r</em> = −0.66, <em>P</em>&lt;0.0001; <em>n<sub>ua</sub></em>: <em>r</em> = −0.58, <em>P</em> = 0.0013) (<a href="#pone-0001455-g001">Figure 1</a>; <a href="#pone.0001455.s005">Figure S2</a>).</p>
<div class="figure" id="pone-0001455-g001"><div class="img"><a name="pone-0001455-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001455.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001455.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Scatterplot of allelic richness and arc surface distance from the most eastern Chinese sample.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>A least-square regression line represents the relationship between the two variables. Significance of the correlation was tested using Spearman's <em>r</em> (<em>r</em> = −0.74, <em>P</em>&lt;0.0001).</p>
<span>doi:10.1371/journal.pone.0001455.g001</span></div><div class="figure" id="pone-0001455-t002"><div class="img"><a name="pone-0001455-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001455.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001455.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Polymorphism summary and test for multilocus linkage disequilibrium in <em>Venturia inaequalis</em> samples.</span></strong></p><span>doi:10.1371/journal.pone.0001455.t002</span></div><a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Bootstrap analysis revealed significant differences in <em>A</em>, <em>H<sub>E</sub></em> and <em>n<sub>ua</sub></em> among regional groups of samples (ANOVA, <em>P</em>&lt;0.001). Central Asian <em>V. inaequalis</em> showed significantly higher values for all three measures of variation (<em>A</em> = 5.03, <em>H<sub>E</sub></em> = 0.65, <em>n<sub>ua</sub></em> = 1.12, <em>P</em>&lt;0.001) than all other regional groups of samples (<a href="#pone-0001455-t003">Table 3</a>). The only nonsignificant comparison was between Central Asian and European groups for <em>H<sub>E</sub></em> (<em>P</em> = 0.898). Unique allele richness was between two and five times higher in the Central Asian group than in any other group. Outside Central Asia, variation measures tended to show the highest variation in Europe (<em>A</em> = 4.82, <em>H<sub>E</sub></em> = 0.66, <em>n<sub>ua</sub></em> = 0.66); intermediate levels of variation in North America (<em>A</em> = 4.23, <em>H<sub>E</sub></em> = 0.59, <em>n<sub>ua</sub></em> = 0.67); and the lowest variation in Morocco, Brazil, South Africa, and New Zealand (<em>A</em>≤3.71, <em>H<sub>E</sub></em>≤0.55, <em>n<sub>ua</sub></em>≤0.14).</p>
<div class="figure" id="pone-0001455-t003"><div class="img"><a name="pone-0001455-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001455.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001455.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Polymorphism summary for regional groupings of <em>Venturia inaequalis</em> samples based on standardization to a group size of 12 individuals.</span></strong></p><span>doi:10.1371/journal.pone.0001455.t003</span></div><a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>Overall, the proportion of haplotypes repeated multiple times was low (<a href="#pone-0001455-t002">Table 2</a>). Thirteen samples had no repeated haplotypes and mean clonal fraction was 5.2%. On average, the clonal fraction was the highest in the Moroccan group (17.3%). The hypothesis of random mating was not rejected in 21 out of the 29 clone-corrected samples analyzed using the <em>I<sub>A</sub></em> statistic (significance level: 0.05).</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>A hierarchical analysis of molecular variance (AMOVA) was performed to describe the distribution of population substructure at different geographic scales (<a href="#pone-0001455-t004">Table 4</a>). AMOVA revealed that, while most of the variation (88%) was distributed within samples, a significant proportion of the variation was also attributable to differences among regions (8%). Only 4% of the variation was partitioned among samples within regions. When each region was analyzed separately, population subdivision within regions was low, albeit significant, and the same order of magnitude was observed in all regions (<em>φ<sub>ST</sub></em> = 0.027–0.084, <em>P</em>&lt;0.001).</p>
<div class="figure" id="pone-0001455-t004"><div class="img"><a name="pone-0001455-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001455.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001455.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Hierarchical analysis of molecular variance of worldwide samples of <em>Venturia inaequalis</em>.</span></strong></p><span>doi:10.1371/journal.pone.0001455.t004</span></div>

<h4>Demographic history</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>We used three different approaches to infer the demographic history of <em>V. inaequalis</em> populations: the test for expected heterozygosity excess/deficiency implemented in the B<span class="small-caps">ottleneck</span> program <a href="#pone.0001455-Cornuet1">[28]</a>, the imbalance index Ln <em>β</em> <a href="#pone.0001455-Kimmel1">[63]</a>, and the interlocus <em>g</em> statistic <a href="#pone.0001455-Reich1">[65]</a>.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Most samples had more loci exhibiting an expected heterozygosity deficit than an expected heterozygosity excess: 27/29 samples under the SMM and 22/29 samples under the TPM showed a majority of loci with expected heterozygosity deficit (<a href="#pone-0001455-t005">Table 5</a>). A two-sided Wilcoxon signed rank test revealed that 13 (resp. 5) samples exhibited a pattern of expected heterozygosity that deviated significantly from mutation-drift equilibrium under the SMM (resp. TPM). Although significance was not observed for a majority of samples especially for the TPM, which may be more realistic for microsatellite loci, it is clear from the results that the trend is consistent with expectations for recent population expansion. The general lack of significance may be explained by the use of an insufficient number of loci that could compromise the power of the test <a href="#pone.0001455-Cornuet1">[28]</a>.</p>
<div class="figure" id="pone-0001455-t005"><div class="img"><a name="pone-0001455-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001455.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001455.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Tests for mutation-drift equilibrium in 29 samples of <em>Venturia inaequalis</em>.</span></strong></p><span>doi:10.1371/journal.pone.0001455.t005</span></div><a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>The imbalance index Ln <em>β</em> was significantly higher than 1 in all samples (<a href="#pone-0001455-t005">Table 5</a>), suggesting that <em>V. inaequalis</em> populations have recently expanded following a bottleneck <a href="#pone.0001455-Kimmel1">[63]</a>. On average, the imbalance was strongest in samples from Morocco (3.45±0.66 SD), lowest in Europe (2.63±0.15 SD) and Central Asia (2.61±0.15 SD), and intermediate in newfound lands (2.91±0.08 SD in Brazil, 2.99 in South Africa, 3.17±0.20 SD in North America and 3.30±0.02 SD in New Zealand). This result is consistent with the bottleneck event predating population expansion being most ancient in Central Asia and Europe, most recent in Morocco, and intermediate in newfound lands <a href="#pone.0001455-Reich1">[65]</a>.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>The interlocus <em>g</em> statistic was lower than 1 in 22 samples (<a href="#pone-0001455-t005">Table 5</a>), which is consistent with population expansion <a href="#pone.0001455-King1">[64]</a>, but none of the values were low enough to be significant at 0.05 according to <a href="#pone-0001455-t001">Table 1</a> of Reich et al. <a href="#pone.0001455-Reich2">[66]</a>. This result may reflect the lower power of the <em>g</em> test to detect recent expansions, particularly when variation in mutation rate across loci is extensive <a href="#pone.0001455-Donnelly1">[35]</a>, <a href="#pone.0001455-King1">[64]</a> as it may be the case with our data set, which combines dinucleotide and trinucleotide loci.</p>


<h4>Clustering and assignment methods</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>In a principal component analysis, the first three principal components accounted for 23.2%, 20.9%, and 18.7% of the variance, respectively. The first two principal components revealed four distinct clusters of samples: a Central Asian cluster, a Brazilian cluster, a cluster formed of Moroccan and North American samples, and a central cluster containing samples from Europe and New Zealand (<a href="#pone-0001455-g002">Figure 2</a>). The third principal component clearly separated Moroccan from North American samples and, along with the first principal component, tended to separate the samples from New Zealand and Europe. The South African sample was placed at the margin of the European cluster in the first and third principal components, and in between the European and North American/Moroccan clusters in the second principal component.</p>
<div class="figure" id="pone-0001455-g002"><div class="img"><a name="pone-0001455-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001455.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001455.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Principal component analysis of a matrix of chord distance <a href="#pone.0001455-CavalliSforza1">[67]</a> among 29 samples of multilocus microsatellite haplotypes of <em>Venturia inaequalis.</em></span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>The first, second and third principal components account for 23.2%, 20.9% and 18.7% of variance respectively. For each samples, the diameter of the disk is proportional to allelic richness. CN = China, IR = Iran, AZ = Azerbaijan, F = France, SE = Sweden, SP = Spain, MA = Morocco, US = USA, CA = Canada, BR = Brazil, SA = South Africa, NZ = New Zealand.</p>
<span>doi:10.1371/journal.pone.0001455.g002</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>S<span class="small-caps">tructure</span> analysis was performed without prior information on the geographic origin of samples, with the number of clusters (<em>K</em>) varying from 1 to 13. The highest Ln likelihood of the data was obtained for <em>K</em> = 7 (<a href="#pone.0001455.s006">Figure S3</a>). The data set was partitioned into clusters corresponding roughly to geography (<a href="#pone-0001455-g003">Figure 3</a>): haplotypes from China, Iran/Azerbaijan, Europe, Morocco, North America, Brazil, and New Zealand tended to be classified in separate clusters. The only noticeable exception was the sample from South Africa, which was assigned in the same cluster as European haplotypes. Overall, individuals from New Zealand, Morocco, Brazil, and North America showed high ancestry fractions in only one group, whereas individuals from Central Asia and Europe/South Africa tended to exhibit more fractional memberships (<a href="#pone-0001455-g003">Figure 3</a>; <a href="#pone.0001455.s003">Table S3</a>).</p>
<div class="figure" id="pone-0001455-g003"><div class="img"><a name="pone-0001455-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001455.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001455.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Population structure of <em>Venturia inaequalis</em> inferred from 1180 multilocus microsatellite haplotypes using the programs B<span class="small-caps">aps</span> <a href="#pone.0001455-Corander1">[71]</a> and S<span class="small-caps">tructure </span><a href="#pone.0001455-Pritchard1">[69]</a>, <a href="#pone.0001455-Falush1">[70]</a>.</span></strong></p><a id="article1.body1.sec3.sec3.fig2.caption1.p1" name="article1.body1.sec3.sec3.fig2.caption1.p1"></a><p>Each haplotype is represented by a line partitioned into <em>K</em> segments that represent the haplotype's estimated membership fractions in <em>K</em> clusters. <em>K</em> = 6 and <em>K</em> = 7 are the population structure models that best fitted the data using B<span class="small-caps">aps</span> and S<span class="small-caps">tructure.</span> CN = China, IR = Iran, AZ = Azerbaijan, F = France, SE = Sweden, SP = Spain, MA = Morocco, US = USA, CA = Canada, BR = Brazil, SA = South Africa, NZ = New Zealand.</p>
<span>doi:10.1371/journal.pone.0001455.g003</span></div><a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>The clustering algorithm implemented in B<span class="small-caps">aps</span> 4 clearly supported six clusters: five clusters corresponded to the Central Asian, Moroccan, North American, Brazilian, New Zealander groups, and the sixth cluster grouped South African and European samples. Unlike analysis with S<span class="small-caps">tructure</span>, B<span class="small-caps">aps</span> did not separate China from Iran/Azerbaijan. Using the admixture analysis implemented in this program, we found lower levels of admixture than with S<span class="small-caps">tructure </span>for the same number of clusters (<a href="#pone-0001455-g003">Figure 3</a>, <a href="#pone.0001455.s003">Table S3</a>). As in S<span class="small-caps">tructure</span> analysis, the highest levels of admixture were observed in the Central Asian and European/South African groups.</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>The exclusion-based method implemented in G<span class="small-caps">eneclass </span>2 produced an accurate assignment rate of 76% (±27 SD) (<a href="#pone-0001455-g004">Figure 4</a>). The rate of accurate assignment was higher for Central Asian, European, and North American samples (&gt;98%) than for other groups of samples (from 25% in South Africa to 78% in Brazil). Overall, the rate of misassignment was high: many individuals tended to be assigned with high probability in multiple groups, which is consistent with a low level of differentiation among groups <a href="#pone.0001455-Berry1">[76]</a>, <a href="#pone.0001455-Manel1">[77]</a>. In particular, all groups showed high rates of misassignment in the Central Asian and European groups (on average 75%±18 SD and 81%±17 SD, respectively) and the South African and New Zealander groups displayed high rate of misassignment in the North American group (71% and 58%, respectively).</p>
<div class="figure" id="pone-0001455-g004"><div class="img"><a name="pone-0001455-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001455.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001455.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Percentage of haplotypes of <em>Venturia inaequalis</em> assigned to 7 regional populations using the assignement method implemented in G<span class="small-caps">eneclass</span> 2 <a href="#pone.0001455-Piry2">[72]</a>.</span></strong></p><span>doi:10.1371/journal.pone.0001455.g004</span></div>

<h4>Gene flow and effective population size</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The migration model with the Central Asian population exchanging migrants only with the European population and unrestricted migration among all non-Central Asian populations (Model 2; Ln(L) = −32266) was found to have significantly higher likelihood than the full model (Model 1; Ln(L) = −37466).</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>For Model 2, parameter estimates for migration rates and effective population sizes (based on <em>θ</em>) varied by population (<a href="#pone-0001455-t006">Table 6</a>). <em>θ</em> values indicated that Central Asia (<em>θ</em> = 1.12) and Europe (<em>θ</em> = 0.91) had higher effective population sizes than Brazil (<em>θ</em> = 0.60), North America (<em>θ</em> = 0.36), Morocco (<em>θ</em> = 0.36), New Zealand (<em>θ</em> = 0.19), and South Africa (<em>θ</em> = 0.04). This pattern is mostly consistent with what could be expected from measures of allelic richness and expected heterozygosity, except for the Brazilian group that showed unexpectedly high <em>θ</em> estimates. Migration rates among regions were generally high (<em>N<sub>e</sub>m</em> = 20.2 on average). Parameter estimates revealed that gene flow between Central Asia and Europe was asymmetrical, with more movements westward than eastward. Outside Central Asia, Europe was the main source of immigrants for all populations. Secondary sources of immigrants were Brazil and, to a lesser extent, Morocco and North America, while South Africa and New Zealand acted as sinks.</p>
<div class="figure" id="pone-0001455-t006"><div class="img"><a name="pone-0001455-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001455" data-uri="info:doi/10.1371/journal.pone.0001455.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001455.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001455.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001455.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001455.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001455.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Estimates for the mean number of migrants exchanged per generation <em>M</em> (2<em>N<sub>e</sub>m</em>) and the mean population mutation rate <em>θ</em> (2<em>N<sub>e</sub>μ</em>) obtained using the program M<span class="small-caps">igrate </span><a href="#pone.0001455-Beerli1">[75]</a>.</span></strong></p><span>doi:10.1371/journal.pone.0001455.t006</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3>
<h4>Origin and introduction pathways</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We used a multilocus microsatellite typing system to describe the worldwide population genetic structure of the apple scab fungus <em>V. inaequalis</em>. Previous studies based on RAPD and PCR-RFLP <a href="#pone.0001455-Tenzer1">[43]</a>, <a href="#pone.0001455-Tenzer2">[44]</a> and microsatellites <a href="#pone.0001455-Tenzer3">[45]</a>, <a href="#pone.0001455-Gurin1">[48]</a>, <a href="#pone.0001455-Gurin2">[49]</a> found high genotypic and genetic diversity in European samples collected on cultivars with no known effective resistance traits. Here, we confirm previous findings at the European scale and we describe the variation in fungal populations from five continents. An older source population is expected to be more variable than a population founded more recently from it <a href="#pone.0001455-Jorde1">[78]</a>. Our finding that genetic variability was higher in Central Asian than in non-Central Asian samples is consistent with a Central Asian origin of the fungus. Just like <em>Mycosphaerella graminicola</em>, <em>Ustilago scitaminea</em>, <em>Magnaporthe oryza,</em> and <em>Phytophthora infestans</em>, respectively pathogen on wheat, sugarcane, rice and potato, and unlike the barley pathogen <em>Rhynchosporium secalis</em>, <em>V. inaequalis</em> seems to share the same geographical origin as its host <a href="#pone.0001455-Banke1">[21]</a>, <a href="#pone.0001455-Zaffarano1">[61]</a>, <a href="#pone.0001455-Brunner1">[79]</a>–<a href="#pone.0001455-Stukenbrock2">[83]</a>. Prospecting and analysis of isolates from Central Asian wild apple should reveal whether the domestication of apple has led to a parallel emergence of apple scab.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The finding of lower levels of variation in non-Central Asian populations suggests that these populations have lost alleles in association with movement, arrival, and establishment outside their native range. However, though less diverse, all these populations were far from being clonal and none displayed extreme reductions in genetic variation such as those reported for other invasive phytopathogenic fungi (e.g., <em>P. infestans</em> <a href="#pone.0001455-Goodwin1">[84]</a>, <em>P. ramorum</em> <a href="#pone.0001455-Ivors1">[27]</a>, <em>Sphaeropsis sapinea</em> <a href="#pone.0001455-Burgess1">[85]</a>, <em>U. scitaminea</em> <a href="#pone.0001455-Raboin1">[82]</a>, <em>Magnaporthe grisea</em> <a href="#pone.0001455-Zeigler1">[33]</a>, <em>Ceratocystis fimbriata</em> f. <em>platani</em> <a href="#pone.0001455-Engelbrecht1">[26]</a>, <em>Fusarium circinatum</em> <a href="#pone.0001455-Wikler1">[86]</a>). Rather, the variability observed in <em>V. inaequalis</em> samples could be compared with that reported for the cereal pathogens <em>R. secalis</em> or <em>Stagonospora nodorum</em> outside their centre of origin. <a href="#pone.0001455-Stukenbrock1">[22]</a>, <a href="#pone.0001455-Zaffarano1">[61]</a>. This level of genetic variation points toward multiple introductions, probably in combination with considerable intraregional gene flow and a significant population expansion as host density increased in new apple-growing regions. In particular, European samples displayed a level of variation close to that observed in Central Asia, suggesting that most of the variation from this region has been introduced into Europe during 2,000 years of travel and trade along the Silk Roads. Considering that apple and potentially its pathogen was introduced in North Africa more than 2,000 years ago <a href="#pone.0001455-Morgan1">[42]</a> we could have expected similar levels of variability in the samples of Morocco. Instead, variation in these samples was significantly lower than in Eurasia and more comparable with the variation displayed in newfound lands. Our hypothesis is that this low level of variation can be attributed to subtle changes in the reproductive mode or the epidemiological structure of the fungus because of the particular mild and dry climatic conditions of this area.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Genetic variability was significantly lower in samples from newfound lands than in samples from Eurasia, and variation indices were linearly correlated with geographic distance from Central Asia, indicating that the farthest populations have received a smaller subset of the original variation. This pattern may reflect that the probability of intercontinental movement of infected material has been limited by distance before recent advances in transportation technology and the advent of global trade. In line with this hypothesis, apple was introduced more recently in countries more distant from Central Asia: with the first settlers during the 16<sup>th</sup> century in North and South America, in 1654 in South Africa, and in 1814 in New Zealand <a href="#pone.0001455-Morgan1">[42]</a>.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>From our coalescent analyses, it appears likely that <em>V. inaequalis</em> followed its host out of Central Asia into Europe, and then into newfound lands. Our migration models indicate that Europe acted as a secondary centre of origin and that very few movements overseas came directly from the actual centre of origin in Central Asia. The intermediate level of variation reported in Europe in comparisons among regions, the central position of European samples in the principal component analysis, and the high rate of misassignment of genotypes from newfound lands in the European group are also consistent with this region being a node in introduction routes.</p>
<a id="article1.body1.sec4.sec1.p5" name="article1.body1.sec4.sec1.p5"></a><p>Exchanges of apple and nursery trees have certainly allowed migrations of <em>V. inaequalis</em> among newfound lands <a href="#pone.0001455-Morgan1">[42]</a>, <a href="#pone.0001455-FAO1">[87]</a>. Estimates of gene flow obtained with M<span class="small-caps">igrate</span> or indirectly using the assignment procedure implemented in G<span class="small-caps">eneclass </span><a href="#pone.0001455-Berry1">[76]</a> indicated that such migrations have been low in front of the historical contribution of Eurasian populations. Surprisingly, analysis with M<span class="small-caps">igrate</span> indicated that Brazil, but also Morocco and North America, were major sources of immigrants, while South Africa and New Zealand acted as sinks for migration. However, we considered these estimates with caution as the assumption of constant population size is likely to be violated in these populations, which have been recently founded.</p>
<a id="article1.body1.sec4.sec1.p6" name="article1.body1.sec4.sec1.p6"></a><p>In summary, the data available to us are consistent with a model of global co-dispersal of apple and its pathogen: the fungus would have first emerged in Central Asia prior to being introduced into Europe along the Silk Roads and more recently into newfound lands with European colonization.</p>


<h4>Gene flow and population subdivision</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We found that most of the genetic variation (88%) was distributed within local samples, which indicates that samples are mostly similar at the genetic level despite distances ranging up to thousands of kilometers. The low differentiation among regions (8% of total variation) can both be explained by levels of gene flow that have been large enough to maintain genetic similarity and/or by insufficient time for differentiation to arise because of recent foundation from a common source. Within regions, the genetic homogeneity observed among samples (4% of total variation) can be explained by the structure of the agrosystem. Apple growing is extensive in all regions where we have sampled and, at least in Central Asia, Europe, and North America, apple trees have covered large areas of land for centuries, both in the form of orchard, meadow, or garden trees <a href="#pone.0001455-Morgan1">[42]</a>, <a href="#pone.0001455-Tenzer1">[43]</a>. This structure may have been prone to significant levels of man-mediated and natural dispersal of <em>V. inaequalis</em>, thereby tending to produce genetic homogeneity among populations within regions.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Note that our study may underestimate population subdivision for two reasons. First, comparative studies have shown that microsatellites could underestimate population differentiation, relative to other markers. The reason given is that the high mutation rate of microsatellite markers increases within-population variance, thereby decreasing the relative magnitude of between-population variance <a href="#pone.0001455-Jorde1">[78]</a>, <a href="#pone.0001455-Jin1">[88]</a>. Second, we found evidence of deviations from mutation-drift equilibrium consistent with a demographic expansion in most populations, and such demographic instabilities are known to downwardly bias estimates of population differentiation <a href="#pone.0001455-Donnelly1">[35]</a>, <a href="#pone.0001455-Donnelly2">[89]</a>.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>Despite a shallow population structure, clustering analyses partitioned the microsatellite data set into separate groups of samples corresponding roughly to geography. Based on these findings, and considering that “a population (…) is a group of conspecific individuals that are at least relatively genetically isolated and that share a common evolutionary origin” <a href="#pone.0001455-Malvarez1">[90]</a>, these regional groups of samples can be referred to as distinct <em>populations</em>. The only noticeable exception is the South African sample, which clustered in the European group with all three assignment methods. This finding reveals that this population has been founded recently from Europe and that introduction is too recent for quantifiable genetic differences to have developed. Considering that apple scab was reported for the first time in 1888 South Africa <a href="#pone.0001455-McHardy1">[36]</a>, populations of more ancient origin probably exist in this region and our sample may not be representative of the real gene pool of <em>V. inaequalis</em> in South Africa.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>Interestingly, samples from newfound lands tended to have high membership in a single cluster, while many individuals from Eurasia had a high membership in non-Eurasian clusters, though they are supposed to be ancestral to all other regional populations. Similar patterns of admixture among ancestral and derived populations have been reported for other organisms (e.g., <em>Drosophila melanogaster</em> <a href="#pone.0001455-Kauer1">[91]</a> or <em>D. simulans</em> <a href="#pone.0001455-Schofl1">[92]</a>). Back migrations of individuals from newfound lands into Eurasia could explain this signal of admixture, but it may also simply reflect the recent foundation of newfound lands populations from a Eurasian source, as recent admixture and shared ancestral variation will give the same signal with Bayesian clustering algorithms <a href="#pone.0001455-Schofl1">[92]</a>.</p>


<h4>Reproductive biology</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Moving a fungus outside its native range can increase linkage disequilibrium or even induce shifts in the reproductive mode because of chance factors (e.g., establishment of only one sexual type) or evolution in response to novel biotic/abiotic conditions <a href="#pone.0001455-Taylor1">[30]</a>, <a href="#pone.0001455-Zeigler1">[33]</a>. In our study, almost all isolates in all samples had unique haplotypes and the majority of samples did not significantly deviate from expectations under random association of alleles as expected for randomly mating populations. This finding concurs with the prevailing view that sexual recombination is a regular feature of <em>V. inaequalis</em> <a href="#pone.0001455-McHardy1">[36]</a> and suggests that our sampling scheme (one isolate per tree) was well suited to avoid collecting clones derived from asexual multiplication. In newfound lands, we would have expected to find more samples with significant multilocus linkage disequilibrium, since multilocus linkage disequilibrium can be inflated by recent foundation and/or admixture between introduced lineages that have different allele frequencies <a href="#pone.0001455-Ardlie1">[32]</a>. However, the proportion of samples showing a significant departure from expectations under random association of alleles was not particularly higher in newfound lands (4/12) than in Eurasia (3/14). One explanation is that multiple generations of recombination, segregation, and population growth may have had time to gradually dissipate multilocus linkage disequilibrium in these populations, even though repeated introductions are known to delay the disruption of multilocus linkage disequilibrium <a href="#pone.0001455-Pfaff1">[93]</a>.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>The Moroccan group displayed a singular population structure: all samples had a low level of genetic and genotypic variation and tended to display multilocus linkage disequilibrium. A possible explanation is that the warm and dry climate of that region could curtail the relative contribution of sexual reproduction. A first hypothesis is that winter temperatures may not be low enough to initiate the sexual stage of the fungus <a href="#pone.0001455-Wilson1">[94]</a>, <a href="#pone.0001455-Boehm1">[95]</a> and/or to induce the falling of leaves, fruiting bodies being only produced on dead leaves <a href="#pone.0001455-McHardy1">[36]</a>. A second hypothesis is that periodic droughts may have cut down genetic variation across historical times <a href="#pone.0001455-Zhan1">[54]</a>; in dry years, rain-splash and moisture may not be sufficient to produce dense epidemics, resulting in lower population sizes and therefore in higher levels of random genetic drift. Further sampling in North Africa should allow determining whether our samples are representative of the population structure of the fungus in that region, and provide more insights into possible changes in the reproductive mode in connection with climate.</p>


<h4>Demographic history</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Tests for mutation-drift equilibrium showed a trend consistent with population expansion in all regions. <em>Venturia inaequalis</em> seems to be in the last step of the invasive process: following multiple introductions, the fungus established viable populations, which are now expanding in their novel environment. The imbalance index also indicated that all populations experienced genetic bottlenecks prior to expansion <a href="#pone.0001455-Kimmel1">[63]</a>, with the bottleneck event being most ancient in Eurasian than in non-Eurasian populations. Eurasian populations may have undergone population shrinkage during the early stages of the putative simultaneous domestication of apple and its fungal pathogen along the Silk Roads <a href="#pone.0001455-Stukenbrock2">[83]</a>, while the genetic structure of populations from newfound lands may bear the signature of more recent bottlenecks resulting from founder events. Such patterns of historical changes in demography have already emerged from population genetic analyses of other eukaryotic invasive pathogens (e.g., <em>M. graminicola</em> <a href="#pone.0001455-Banke1">[21]</a>, <a href="#pone.0001455-Stukenbrock2">[83]</a> and <em>Plasmodium falciparum</em> <a href="#pone.0001455-Hume1">[96]</a>, <a href="#pone.0001455-Joy1">[97]</a>), and it may be a common feature of many invasive species due to the correlation of their demography with changes in human culture and agricultural practices <a href="#pone.0001455-Donnelly1">[35]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001455"></div><a name="pone.0001455.s001" id="pone.0001455.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001455.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Pairwise PhiST between pairs of samples of V. inaequalis collected on different cultivars in the same location.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001455.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0001455.s002" id="pone.0001455.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001455.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Origin and MLMT profiles for 1273 isolates of Venturia inaequalis</p>
<p class="siDoi">doi:10.1371/journal.pone.0001455.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.20 MB PDF)</p>
<a name="pone.0001455.s003" id="pone.0001455.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001455.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Admixture analysis for 1180 haplotypes of Venturia inaequalis from seven regions: (A) average membership coefficients in seven clusters inferred with the individual-based method implemented in the Structure program <a href="#pone.0001455-Pritchard1">[69]</a>, <a href="#pone.0001455-Falush1">[70]</a>, and (B) average membership coefficients in six clusters inferred with the sample-based method implemented in Baps 4 <a href="#pone.0001455-Corander1">[71]</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001455.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0001455.s004" id="pone.0001455.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001455.s004">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Map of approximate sampling locations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001455.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.74 MB PDF)</p>
<a name="pone.0001455.s005" id="pone.0001455.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001455.s005">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Scatterplot expected heterozygosity, unique allele richness and arc surface distance from the most eastern sample. A least-square regression line represents the relationship between the two variables. Significance of the correlation was tested using Spearman's r (expected heterozygosity: r = −0.66, P&lt;0.0001; unique allele richness: r = −0.58, P&lt;0.0013).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001455.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.14 MB PDF)</p>
<a name="pone.0001455.s006" id="pone.0001455.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001455.s006">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Plot of Ln likelihood of the data for several value of K, the parameter representing the number of populations in the Bayesian clustering algorithm implemented in the STRUCTURE program <a href="#pone.0001455-Pritchard1">[69]</a>, <a href="#pone.0001455-Falush1">[70]</a>. Ln likelihood values were averaged across at least 6 independent runs of the program.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001455.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.10 MB PDF)</p>
<a name="pone.0001455.s007" id="pone.0001455.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001455.s007">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Abstract in Chinese, French, Arabic and Portuguese</p>
<p class="siDoi">doi:10.1371/journal.pone.0001455.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to all the people who helped us to collect strains and to Martine Devaux, François Coupry and Guillaume Lorin for technical assistance. We thank Jean-Pierre Paulin, Valérie Caffier, Damien Meyer, Jacqui Shykoff and John Hartman for critical reading of the manuscript, and Tatiana Giraud for useful suggestions. Part of this work was carried out by using the resources of the Computational Biology Service Unit from Cornell University which is partially funded by Microsoft Corporation. We thank Matthew Fisher and collaborators for hosting our microsatellite dataset on <a href="http://www.multilocus.net">www.multilocus.net</a>.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BL PG XZ. Performed the experiments: PG DA RV MS XZ. Analyzed the data: PG. Contributed reagents/materials/analysis tools: BL RV MS XZ. Wrote the paper: BL PG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001455-Facon1" id="pone.0001455-Facon1"></a>Facon B, Genton B, Shykoff J, Jarne P, Estoup A, et al.  (2006) A general eco-evolutionary framework for understanding bioinvasions. Trends Ecol Evol  21: 130–135.  <ul class="find" data-citedArticleID="1043646" data-doi="10.1016/j.tree.2005.10.012"><li><a href="http://dx.doi.org/10.1016/j.tree.2005.10.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+eco-evolutionary+framework+for+understanding+bioinvasions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+eco-evolutionary+framework+for+understanding+bioinvasions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001455-Anderson1" id="pone.0001455-Anderson1"></a>Anderson P, Cunningham A, Patel N, Morales F, Epstein P, et al.  (2004) Emerging infectious diseases of plants: pathogen pollution, climate change and agrotechnology drivers. Trends Ecol Evol  19: 535–544.  <ul class="find" data-citedArticleID="1043592" data-doi="10.1016/j.tree.2004.07.021"><li><a href="http://dx.doi.org/10.1016/j.tree.2004.07.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emerging+infectious+diseases+of+plants%3A+pathogen+pollution%2C+climate+change+and+agrotechnology+drivers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emerging+infectious+diseases+of+plants%3A+pathogen+pollution%2C+climate+change+and+agrotechnology+drivers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001455-Kareiva1" id="pone.0001455-Kareiva1"></a>Kareiva P, Watts S, McDonald R, Boucher T (2007) Domesticated nature: shaping landscapes and ecosystems for human welfare. Science  316: 1866–1869.  <ul class="find" data-citedArticleID="1043686" data-doi="10.1126/science.1140170"><li><a href="http://dx.doi.org/10.1126/science.1140170" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Domesticated+nature%3A+shaping+landscapes+and+ecosystems+for+human+welfare." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Domesticated+nature%3A+shaping+landscapes+and+ecosystems+for+human+welfare.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001455-Pimentel1" id="pone.0001455-Pimentel1"></a>Pimentel D, Lach L, Zuniga R, Morrison D (2000) Environmental and Economic Costs of Nonindigenous Species in theUnited States. BioScience  50: 53–64.  <ul class="find" data-citedArticleID="1043722" data-doi="10.1641/0006-3568(2000)050[0053:eaecon]2.3.co;2"><li><a href="http://dx.doi.org/10.1641/0006-3568(2000)050[0053:eaecon]2.3.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Environmental+and+Economic+Costs+of+Nonindigenous+Species+in+theUnited+States." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Environmental+and+Economic+Costs+of+Nonindigenous+Species+in+theUnited+States.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001455-Anagnostakis1" id="pone.0001455-Anagnostakis1"></a>Anagnostakis S (1987) Chestnut blight: The classical problem of an introduced pathogen. Mycologia  29: 23–37.  <ul class="find" data-citedArticleID="1043590" data-doi="10.2307/3807741"><li><a href="http://dx.doi.org/10.2307/3807741" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chestnut+blight%3A+The+classical+problem+of+an+introduced+pathogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chestnut+blight%3A+The+classical+problem+of+an+introduced+pathogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001455-Fry1" id="pone.0001455-Fry1"></a>Fry W, Goodwin S (1997) Resurgence of the Irish potato famine fungus. BioScience  47: 363–371.  <ul class="find" data-citedArticleID="1043654" data-doi="10.2307/1313151"><li><a href="http://dx.doi.org/10.2307/1313151" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resurgence+of+the+Irish+potato+famine+fungus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resurgence+of+the+Irish+potato+famine+fungus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001455-Palm1" id="pone.0001455-Palm1"></a>Palm M (2001) Systematics and the Impact of Invasive Fungi on Agriculture in theUnited States. BioScience  51: 141–147.  <ul class="find" data-citedArticleID="1043712" data-doi="10.1641/0006-3568(2001)051[0141:satioi]2.0.co;2"><li><a href="http://dx.doi.org/10.1641/0006-3568(2001)051[0141:satioi]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematics+and+the+Impact+of+Invasive+Fungi+on+Agriculture+in+theUnited+States." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematics+and+the+Impact+of+Invasive+Fungi+on+Agriculture+in+theUnited+States.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001455-Wingfield1" id="pone.0001455-Wingfield1"></a>Wingfield M, Slippers B, Roux J, Wingfield B (2001) Worldwide Movement of Exotic Forest Fungi, Especially in theTropics and the Southern Hemisphere. BioScience  51: 134–140.  <ul class="find" data-citedArticleID="1043766" data-doi="10.1641/0006-3568(2001)051[0134:wmoeff]2.0.co;2"><li><a href="http://dx.doi.org/10.1641/0006-3568(2001)051[0134:wmoeff]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Worldwide+Movement+of+Exotic+Forest+Fungi%2C+Especially+in+theTropics+and+the+Southern+Hemisphere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Worldwide+Movement+of+Exotic+Forest+Fungi%2C+Especially+in+theTropics+and+the+Southern+Hemisphere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001455-Campbell1" id="pone.0001455-Campbell1"></a>Campbell F (2001) The Science of Risk Assessment for Phytosanitary Regulation and the Impact of Changing Trade Regulations. BioScience  51: 148–153.  <ul class="find" data-citedArticleID="1043618" data-doi="10.1641/0006-3568(2001)051[0148:tsoraf]2.0.co;2"><li><a href="http://dx.doi.org/10.1641/0006-3568(2001)051[0148:tsoraf]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Science+of+Risk+Assessment+for+Phytosanitary+Regulation+and+the+Impact+of+Changing+Trade+Regulations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Science+of+Risk+Assessment+for+Phytosanitary+Regulation+and+the+Impact+of+Changing+Trade+Regulations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001455-Perrings1" id="pone.0001455-Perrings1"></a>Perrings C, Williamson M, Barbier E, Delfino D, Dalmazzone S, et al.  (2002) Biological invasion risks and the public good: an economic perspective. Conservation Ecology  6:   Article 1 (online).  <ul class="find" data-citedArticleID="1043718"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Biological+invasion+risks+and+the+public+good%3A+an+economic+perspective.&amp;auth=&amp;atitle=Biological+invasion+risks+and+the+public+good%3A+an+economic+perspective." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biological+invasion+risks+and+the+public+good%3A+an+economic+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biological+invasion+risks+and+the+public+good%3A+an+economic+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001455-Cui1" id="pone.0001455-Cui1"></a>Cui L, Escalante A, Imwong M, Snounou G (2003) The genetic diversity of <em>Plasmodium vivax</em> populations. Trends Parasitol  19: 220–226.  <ul class="find" data-citedArticleID="1043630" data-doi="10.1016/s1471-4922(03)00085-0"><li><a href="http://dx.doi.org/10.1016/s1471-4922(03)00085-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+diversity+of+Plasmodium+vivax+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+diversity+of+Plasmodium+vivax+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001455-Rizzo1" id="pone.0001455-Rizzo1"></a>Rizzo D (2005)  Exotic species and fungi: Interactions with fungal, plant and animal communities. In: Dighton J, Oudemans P, White J-F, editors. The Fungal Community: Its Organization and Role in the Ecosystem. Boca Raton: CRC Press. pp. 857–880.  <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0001455-Bandyopadhyay1" id="pone.0001455-Bandyopadhyay1"></a>Bandyopadhyay R, Frederickson DE, McLaren NW, Odvody GN, Ryley MJ (1998) Ergot: A New Disease Threat to Sorghum in the Americas and Australia. Plant Disease  82: 356–367.  <ul class="find" data-citedArticleID="1043596" data-doi="10.1094/pdis.1998.82.4.356"><li><a href="http://dx.doi.org/10.1094/pdis.1998.82.4.356" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ergot%3A+A+New+Disease+Threat+to+Sorghum+in+the+Americas+and+Australia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ergot%3A+A+New+Disease+Threat+to+Sorghum+in+the+Americas+and+Australia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001455-Garrison1" id="pone.0001455-Garrison1"></a>Garrison V, Shinn E, Foreman W, Griffin D, Holmes C, et al.  (2003) African and Asian Dust: From Desert Soils to Coral Reefs. BioScience  53: 469–480.  <ul class="find" data-citedArticleID="1043656" data-doi="10.1641/0006-3568(2003)053[0469:aaadfd]2.0.co;2"><li><a href="http://dx.doi.org/10.1641/0006-3568(2003)053[0469:aaadfd]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=African+and+Asian+Dust%3A+From+Desert+Soils+to+Coral+Reefs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22African+and+Asian+Dust%3A+From+Desert+Soils+to+Coral+Reefs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001455-Brown1" id="pone.0001455-Brown1"></a>Brown J, Hovmøller M (2002) Aerial Dispersal of Pathogens on the Global and Continental Scales and Its Impact on Plant Disease. Science  297: 537–541.  <ul class="find" data-citedArticleID="1043608" data-doi="10.1126/science.1072678"><li><a href="http://dx.doi.org/10.1126/science.1072678" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aerial+Dispersal+of+Pathogens+on+the+Global+and+Continental+Scales+and+Its+Impact+on+Plant+Disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aerial+Dispersal+of+Pathogens+on+the+Global+and+Continental+Scales+and+Its+Impact+on+Plant+Disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001455-Parker1" id="pone.0001455-Parker1"></a>Parker IM, Gilbert GS (2004) The Evolutionary Ecology of Novel Plant-Pathogen Interactions. Annu Rev Ecol Evol Syst  35: 675–700.  <ul class="find" data-citedArticleID="1043714" data-doi="10.1146/annurev.ecolsys.34.011802.132339"><li><a href="http://dx.doi.org/10.1146/annurev.ecolsys.34.011802.132339" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Evolutionary+Ecology+of+Novel+Plant-Pathogen+Interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Evolutionary+Ecology+of+Novel+Plant-Pathogen+Interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001455-Yarwood1" id="pone.0001455-Yarwood1"></a>Yarwood CE (1970) Man-Made Plant Diseases. Science  168: 218–220.  <ul class="find" data-citedArticleID="1043772" data-doi="10.1126/science.168.3928.218"><li><a href="http://dx.doi.org/10.1126/science.168.3928.218" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Man-Made+Plant+Diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Man-Made+Plant+Diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001455-Sakai1" id="pone.0001455-Sakai1"></a>Sakai AK, Allendorf FW, Holt JS, Lodge DM, Molofsky J, et al.  (2001) The population biology of invasive species. Annu Rev Ecol Syst  32: 305–332.  <ul class="find" data-citedArticleID="1043742" data-doi="10.1146/annurev.ecolsys.32.081501.114037"><li><a href="http://dx.doi.org/10.1146/annurev.ecolsys.32.081501.114037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+population+biology+of+invasive+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+population+biology+of+invasive+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001455-Fisher1" id="pone.0001455-Fisher1"></a>Fisher M, Koenig G, White T, San-Blas G, Negroni R, et al.  (2001) Biogeographic range expansion into South America by <em>Coccidioides immitis</em> mirrors New World patterns of human migration. Proc Natl Acad Sci USA  98: 4558–4562.  <ul class="find" data-citedArticleID="1043650" data-doi="10.1073/pnas.071406098"><li><a href="http://dx.doi.org/10.1073/pnas.071406098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biogeographic+range+expansion+into+South+America+by+Coccidioides+immitis+mirrors+New+World+patterns+of+human+migration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biogeographic+range+expansion+into+South+America+by+Coccidioides+immitis+mirrors+New+World+patterns+of+human+migration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001455-Fisher2" id="pone.0001455-Fisher2"></a>Fisher M, Rannala B, Chaturvedi V, Taylor J (2002) Disease surveillance in recombining pathogens: multilocus genotypes identify sources of human Coccidioides infections. Proc Natl Acad Sci USA  99: 9067–9071.  <ul class="find" data-citedArticleID="1043652" data-doi="10.1073/pnas.132178099"><li><a href="http://dx.doi.org/10.1073/pnas.132178099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Disease+surveillance+in+recombining+pathogens%3A+multilocus+genotypes+identify+sources+of+human+Coccidioides+infections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Disease+surveillance+in+recombining+pathogens%3A+multilocus+genotypes+identify+sources+of+human+Coccidioides+infections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001455-Banke1" id="pone.0001455-Banke1"></a>Banke S, McDonald BA (2005) Migration patterns among global populations of the pathogenic fungus <em>Mycosphaerella graminicola</em>. Mol Ecol  14: 1881–1896.  <ul class="find" data-citedArticleID="1043598" data-doi="10.1111/j.1365-294x.2005.02536.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2005.02536.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Migration+patterns+among+global+populations+of+the+pathogenic+fungus+Mycosphaerella+graminicola." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Migration+patterns+among+global+populations+of+the+pathogenic+fungus+Mycosphaerella+graminicola.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001455-Stukenbrock1" id="pone.0001455-Stukenbrock1"></a>Stukenbrock E, Banke S, McDonald B (2006) Global migration patterns in the fungal wheat pathogen <em>Phaeosphaeria nodorum</em>. Mol Ecol  15: 2895–2904.  <ul class="find" data-citedArticleID="1043748" data-doi="10.1111/j.1365-294x.2006.02986.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2006.02986.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+migration+patterns+in+the+fungal+wheat+pathogen+Phaeosphaeria+nodorum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+migration+patterns+in+the+fungal+wheat+pathogen+Phaeosphaeria+nodorum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001455-Nei1" id="pone.0001455-Nei1"></a>Nei M, Maruyama T, Chakraborty R (1975) The Bottleneck Effect and Genetic Variability in Populations. Evolution  29: 1–10.  <ul class="find" data-citedArticleID="1043706" data-doi="10.2307/2407137"><li><a href="http://dx.doi.org/10.2307/2407137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Bottleneck+Effect+and+Genetic+Variability+in+Populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Bottleneck+Effect+and+Genetic+Variability+in+Populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001455-Barton1" id="pone.0001455-Barton1"></a>Barton N (2000) Genetic hitchhiking. Philos Trans R Soc Lond B Biol Sci  355: 1553–1562.  <ul class="find" data-citedArticleID="1043600"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+hitchhiking.&amp;auth=&amp;atitle=Genetic+hitchhiking." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+hitchhiking." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+hitchhiking.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001455-Jobling1" id="pone.0001455-Jobling1"></a>Jobling M, Hurles M, Tyler-Smith C (2004) Human Evolutionary Genetics: Origins, Peoples &amp; Disease. London/New York: Garland Science Publishing.   <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0001455-Engelbrecht1" id="pone.0001455-Engelbrecht1"></a>Engelbrecht CJB, Harrington TC, Steimel J, Capretti P (2004) Genetic variation in eastern North American and putatively introduced populations of <em>Ceratocystis fimbriata</em> f. <em>platani</em>. Mol Ecol  13: 2995–3005.  <ul class="find" data-citedArticleID="1043638" data-doi="10.1111/j.1365-294x.2004.02312.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2004.02312.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+variation+in+eastern+North+American+and+putatively+introduced+populations+of+Ceratocystis+fimbriata+f.+platani." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+variation+in+eastern+North+American+and+putatively+introduced+populations+of+Ceratocystis+fimbriata+f.+platani.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001455-Ivors1" id="pone.0001455-Ivors1"></a>Ivors K, Garbelotto M, Vries IDE, Ruyter-Spira C, Hekkert BT, et al.  (2006) Microsatellite markers identify three lineages of <em>Phytophthora ramorum</em> in US nurseries, yet single lineages in US forest and European nursery populations. Mol Ecol  15: 1493–1505.  <ul class="find" data-citedArticleID="1043674" data-doi="10.1111/j.1365-294x.2006.02864.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2006.02864.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microsatellite+markers+identify+three+lineages+of+Phytophthora+ramorum+in+US+nurseries%2C+yet+single+lineages+in+US+forest+and+European+nursery+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microsatellite+markers+identify+three+lineages+of+Phytophthora+ramorum+in+US+nurseries%2C+yet+single+lineages+in+US+forest+and+European+nursery+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001455-Cornuet1" id="pone.0001455-Cornuet1"></a>Cornuet J-M, Luikart G (1996) Description and Power Analysis of Two Tests for Detecting Recent Population Bottlenecks From Allele Frequency Data. Genetics  144: 2001–2014.  <ul class="find" data-citedArticleID="1043626"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Description+and+Power+Analysis+of+Two+Tests+for+Detecting+Recent+Population+Bottlenecks+From+Allele+Frequency+Data.&amp;auth=&amp;atitle=Description+and+Power+Analysis+of+Two+Tests+for+Detecting+Recent+Population+Bottlenecks+From+Allele+Frequency+Data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Description+and+Power+Analysis+of+Two+Tests+for+Detecting+Recent+Population+Bottlenecks+From+Allele+Frequency+Data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Description+and+Power+Analysis+of+Two+Tests+for+Detecting+Recent+Population+Bottlenecks+From+Allele+Frequency+Data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001455-Rivas1" id="pone.0001455-Rivas1"></a>Rivas G-G, Zapater M-F, Abadie C, Carlier J (2004) Founder effects and stochastic dispersal at the continental scale of the fungal pathogen of bananas <em>Mycosphaerella fijiensis</em>. Mol Ecol  13: 471–482.  <ul class="find" data-citedArticleID="1043738" data-doi="10.1046/j.1365-294x.2003.02043.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2003.02043.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Founder+effects+and+stochastic+dispersal+at+the+continental+scale+of+the+fungal+pathogen+of+bananas+Mycosphaerella+fijiensis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Founder+effects+and+stochastic+dispersal+at+the+continental+scale+of+the+fungal+pathogen+of+bananas+Mycosphaerella+fijiensis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001455-Taylor1" id="pone.0001455-Taylor1"></a>Taylor JW, Jacobson D, Fisher M (1999) The Evolution of Asexual Fungi: Reproduction, Speciation and Classification. Annu Rev Phytopathol  37: 197–246.  <ul class="find" data-citedArticleID="1043752" data-doi="10.1146/annurev.phyto.37.1.197"><li><a href="http://dx.doi.org/10.1146/annurev.phyto.37.1.197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Evolution+of+Asexual+Fungi%3A+Reproduction%2C+Speciation+and+Classification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Evolution+of+Asexual+Fungi%3A+Reproduction%2C+Speciation+and+Classification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001455-Milgroom1" id="pone.0001455-Milgroom1"></a>Milgroom MG (1996) Recombination and the multilocus structure of fungal populations. Annu Rev Phytopathol  34: 457–477.  <ul class="find" data-citedArticleID="1043702" data-doi="10.1146/annurev.phyto.34.1.457"><li><a href="http://dx.doi.org/10.1146/annurev.phyto.34.1.457" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+and+the+multilocus+structure+of+fungal+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+and+the+multilocus+structure+of+fungal+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001455-Ardlie1" id="pone.0001455-Ardlie1"></a>Ardlie K, Kruglyak L, Seielstad M (2002) Patterns of linkage disequilibrium in the human genome. Nat Rev Genet  3: 299–309.  <ul class="find" data-citedArticleID="1043594" data-doi="10.1038/nrg777"><li><a href="http://dx.doi.org/10.1038/nrg777" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+linkage+disequilibrium+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+linkage+disequilibrium+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001455-Zeigler1" id="pone.0001455-Zeigler1"></a>Zeigler RS (1998) Recombination in <em>Magnaporthe grisea</em>. Annu Rev Phytopathol  36: 249–275.  <ul class="find" data-citedArticleID="1043776" data-doi="10.1146/annurev.phyto.36.1.249"><li><a href="http://dx.doi.org/10.1146/annurev.phyto.36.1.249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+in+Magnaporthe+grisea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+in+Magnaporthe+grisea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001455-Smart1" id="pone.0001455-Smart1"></a>Smart C, Fry W (2001) Invasions by the late blight pathogen: renewed sex and enhanced fitness. Biological Invasions  3: 235–243.  <ul class="find" data-citedArticleID="1043746" data-doi="10.1023/a:1015200920772"><li><a href="http://dx.doi.org/10.1023/a:1015200920772" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Invasions+by+the+late+blight+pathogen%3A+renewed+sex+and+enhanced+fitness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Invasions+by+the+late+blight+pathogen%3A+renewed+sex+and+enhanced+fitness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001455-Donnelly1" id="pone.0001455-Donnelly1"></a>Donnelly M, Licht M, Lehmann T (2001) Evidence for recent population expansion in the evolutionary history of the malaria vectors <em>Anopheles arabiensis</em> and <em>Anopheles gambiae</em>. Mol Biol Evol  18: 1353–1364.  <ul class="find" data-citedArticleID="1043634" data-doi="10.1093/oxfordjournals.molbev.a003919"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003919" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+recent+population+expansion+in+the+evolutionary+history+of+the+malaria+vectors+Anopheles+arabiensis+and+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+recent+population+expansion+in+the+evolutionary+history+of+the+malaria+vectors+Anopheles+arabiensis+and+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001455-McHardy1" id="pone.0001455-McHardy1"></a>McHardy W (1996) Apple Scab: Biology, Epidemiology and Management. St Paul: The American Phytopathological Society.   <ul class="find-nolinks"></ul></li><li><span class="label">37.
              </span><a name="pone.0001455-Wirth1" id="pone.0001455-Wirth1"></a>Wirth T, Meyer A, Achtman M (2005) Deciphering host migrations and origins by means of their microbes. Mol Ecol  14: 3289–3306.  <ul class="find" data-citedArticleID="1043768" data-doi="10.1111/j.1365-294x.2005.02687.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2005.02687.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deciphering+host+migrations+and+origins+by+means+of+their+microbes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deciphering+host+migrations+and+origins+by+means+of+their+microbes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001455-Juniper1" id="pone.0001455-Juniper1"></a>Juniper B, Mabberley D (2006) The Story Of The Apple. Portland: Timber Press.   <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0001455-Harris1" id="pone.0001455-Harris1"></a>Harris S, Robinson J, Juniper B (2002) Genetic clues to the origin of the apple. Trends Genet  18: 426–430.  <ul class="find" data-citedArticleID="1043670" data-doi="10.1016/s0168-9525(02)02689-6"><li><a href="http://dx.doi.org/10.1016/s0168-9525(02)02689-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+clues+to+the+origin+of+the+apple." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+clues+to+the+origin+of+the+apple.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001455-Wood1" id="pone.0001455-Wood1"></a>Wood F (2003) The Silk Road. London: The Folio Society.   <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0001455-Hancock1" id="pone.0001455-Hancock1"></a>Hancock JF (2004) Plant Evolution and the Origin of Crop Species. Wallingford: CABI Publishing.   <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0001455-Morgan1" id="pone.0001455-Morgan1"></a>Morgan J, Richards A (1993) The Book of Apples. London: Ebury Press.   <ul class="find-nolinks"></ul></li><li><span class="label">43.
              </span><a name="pone.0001455-Tenzer1" id="pone.0001455-Tenzer1"></a>Tenzer I, Gessler C (1997) Subdivision and genetic structure of four populations of <em>Venturia inaequalis</em> in Switzerland. Eur J Plant Pathol  103: 565–571.  <ul class="find" data-citedArticleID="1043756"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Subdivision+and+genetic+structure+of+four+populations+of+Venturia+inaequalis+in+Switzerland.&amp;auth=&amp;atitle=Subdivision+and+genetic+structure+of+four+populations+of+Venturia+inaequalis+in+Switzerland." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subdivision+and+genetic+structure+of+four+populations+of+Venturia+inaequalis+in+Switzerland." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subdivision+and+genetic+structure+of+four+populations+of+Venturia+inaequalis+in+Switzerland.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001455-Tenzer2" id="pone.0001455-Tenzer2"></a>Tenzer I, Gessler C (1999) Genetic Diversity of <em>Venturia inaequalis</em> Across Europe. Eur J Plant Pathol  105: 545–552.  <ul class="find" data-citedArticleID="1043758"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+Diversity+of+Venturia+inaequalis+Across+Europe.&amp;auth=&amp;atitle=Genetic+Diversity+of+Venturia+inaequalis+Across+Europe." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+Diversity+of+Venturia+inaequalis+Across+Europe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+Diversity+of+Venturia+inaequalis+Across+Europe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001455-Tenzer3" id="pone.0001455-Tenzer3"></a>Tenzer I, degli Ivanissevich S, Morgante M, Gessler C (1999) Identification of Microsatellite Markers and Their Application to Population Genetics of <em>Venturia inaequalis</em>. Phytopathology  89: 748–753.  <ul class="find" data-citedArticleID="1043760" data-doi="10.1094/phyto.1999.89.9.748"><li><a href="http://dx.doi.org/10.1094/phyto.1999.89.9.748" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+Microsatellite+Markers+and+Their+Application+to+Population+Genetics+of+Venturia+inaequalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+Microsatellite+Markers+and+Their+Application+to+Population+Genetics+of+Venturia+inaequalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001455-Taylor2" id="pone.0001455-Taylor2"></a>Taylor JW, Fisher M (2003) Fungal multilocus sequence typing–it's not just for bacteria. Curr Opin Microbiol  6: 351–356.  <ul class="find" data-citedArticleID="1043754" data-doi="10.1016/s1369-5274(03)00088-2"><li><a href="http://dx.doi.org/10.1016/s1369-5274(03)00088-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fungal+multilocus+sequence+typing%E2%80%93it%27s+not+just+for+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fungal+multilocus+sequence+typing%E2%80%93it%27s+not+just+for+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001455-Brown2" id="pone.0001455-Brown2"></a>Brown JKM (1994) Chance and selection in the evolution of barley mildew. Trends Microbiol  2: 470–475.  <ul class="find" data-citedArticleID="1043610" data-doi="10.1016/0966-842x(94)90650-5"><li><a href="http://dx.doi.org/10.1016/0966-842x(94)90650-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chance+and+selection+in+the+evolution+of+barley+mildew." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chance+and+selection+in+the+evolution+of+barley+mildew.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001455-Gurin1" id="pone.0001455-Gurin1"></a>Guérin F, Le Cam B (2004) Breakdown of the scab resistance gene <em>Vf</em> in apple leads to a founder effect in population of the fungal pathogen <em>Venturia inaequalis</em>. Phytopathology  94: 364–369.  <ul class="find" data-citedArticleID="1043662" data-doi="10.1094/phyto.2004.94.4.364"><li><a href="http://dx.doi.org/10.1094/phyto.2004.94.4.364" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Breakdown+of+the+scab+resistance+gene+Vf+in+apple+leads+to+a+founder+effect+in+population+of+the+fungal+pathogen+Venturia+inaequalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Breakdown+of+the+scab+resistance+gene+Vf+in+apple+leads+to+a+founder+effect+in+population+of+the+fungal+pathogen+Venturia+inaequalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001455-Gurin2" id="pone.0001455-Gurin2"></a>Guérin F, Gladieux P, Le Cam B (2007) Origin and colonization history of newly virulent strains of the phytopathogenic fungus <em>Venturia inaequalis</em>. Fungal Genet Biol  44: 284–292.  <ul class="find" data-citedArticleID="1043664" data-doi="10.1016/j.fgb.2006.10.005"><li><a href="http://dx.doi.org/10.1016/j.fgb.2006.10.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+and+colonization+history+of+newly+virulent+strains+of+the+phytopathogenic+fungus+Venturia+inaequalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+and+colonization+history+of+newly+virulent+strains+of+the+phytopathogenic+fungus+Venturia+inaequalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001455-Excoffier1" id="pone.0001455-Excoffier1"></a>Excoffier L, Smouse PE, Quattro JM (1992) Analysis of Molecular Variance Inferred From Metric Distances Among DNA Haplotypes: Application to Human Mitochondrial DNA Restriction Data. Genetics  131: 479–491.  <ul class="find" data-citedArticleID="1043640"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+Molecular+Variance+Inferred+From+Metric+Distances+Among+DNA+Haplotypes%3A+Application+to+Human+Mitochondrial+DNA+Restriction+Data.&amp;auth=&amp;atitle=Analysis+of+Molecular+Variance+Inferred+From+Metric+Distances+Among+DNA+Haplotypes%3A+Application+to+Human+Mitochondrial+DNA+Restriction+Data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+Molecular+Variance+Inferred+From+Metric+Distances+Among+DNA+Haplotypes%3A+Application+to+Human+Mitochondrial+DNA+Restriction+Data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+Molecular+Variance+Inferred+From+Metric+Distances+Among+DNA+Haplotypes%3A+Application+to+Human+Mitochondrial+DNA+Restriction+Data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001455-Peakall1" id="pone.0001455-Peakall1"></a>Peakall ROD, Smouse PE (2006) genalex 6: genetic analysis in Excel. Population genetic software for teaching and research. Mol Ecol Notes  6: 288–295.  <ul class="find" data-citedArticleID="1043716" data-doi="10.1111/j.1471-8286.2005.01155.x"><li><a href="http://dx.doi.org/10.1111/j.1471-8286.2005.01155.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=genalex+6%3A+genetic+analysis+in+Excel.+Population+genetic+software+for+teaching+and+research." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22genalex+6%3A+genetic+analysis+in+Excel.+Population+genetic+software+for+teaching+and+research.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001455-Gurin3" id="pone.0001455-Gurin3"></a>Guérin F, Franck P, Loiseau A, Devaux M, Le Cam B (2004) Isolation of 21 new polymorphic microsatellite loci in the phytopathogenic fungus <em>Venturia inaequalis</em>. Mol Ecol Notes  4: 268–270.  <ul class="find" data-citedArticleID="1043666" data-doi="10.1111/j.1471-8286.2004.00637.x"><li><a href="http://dx.doi.org/10.1111/j.1471-8286.2004.00637.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+21+new+polymorphic+microsatellite+loci+in+the+phytopathogenic+fungus+Venturia+inaequalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+21+new+polymorphic+microsatellite+loci+in+the+phytopathogenic+fungus+Venturia+inaequalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001455-Excoffier2" id="pone.0001455-Excoffier2"></a>Excoffier L, Laval G, Schneider S (2005) Arlequin (version 3.0): An integrated software package for. Evolutionary Bioinformatics Online  1: 47–50.  <ul class="find" data-citedArticleID="1043642"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Arlequin+%28version+3.0%29%3A+An+integrated+software+package+for.&amp;auth=&amp;atitle=Arlequin+%28version+3.0%29%3A+An+integrated+software+package+for." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arlequin+%28version+3.0%29%3A+An+integrated+software+package+for." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arlequin+%28version+3.0%29%3A+An+integrated+software+package+for.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001455-Zhan1" id="pone.0001455-Zhan1"></a>Zhan J, Pettway R, McDonald B (2003) The global genetic structure of the wheat pathogen Mycosphaerella graminicola is characterized by high nuclear diversity, low mitochondrial diversity, regular recombination, and gene flow. Fungal Genet Biol  38: 286–297.  <ul class="find" data-citedArticleID="1043778" data-doi="10.1016/s1087-1845(02)00538-8"><li><a href="http://dx.doi.org/10.1016/s1087-1845(02)00538-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+global+genetic+structure+of+the+wheat+pathogen+Mycosphaerella+graminicola+is+characterized+by+high+nuclear+diversity%2C+low+mitochondrial+diversity%2C+regular+recombination%2C+and+gene+flow." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+global+genetic+structure+of+the+wheat+pathogen+Mycosphaerella+graminicola+is+characterized+by+high+nuclear+diversity%2C+low+mitochondrial+diversity%2C+regular+recombination%2C+and+gene+flow.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001455-Chen1" id="pone.0001455-Chen1"></a>Chen RS, McDonald BA (1996) Sexual Reproduction Plays a Major Role in the Genetic Structure of Populations of the Fungus <em>Mycosphaerella graminicola</em>. Genetics  142: 1119–1127.  <ul class="find" data-citedArticleID="1043622"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sexual+Reproduction+Plays+a+Major+Role+in+the+Genetic+Structure+of+Populations+of+the+Fungus+Mycosphaerella+graminicola.&amp;auth=&amp;atitle=Sexual+Reproduction+Plays+a+Major+Role+in+the+Genetic+Structure+of+Populations+of+the+Fungus+Mycosphaerella+graminicola." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sexual+Reproduction+Plays+a+Major+Role+in+the+Genetic+Structure+of+Populations+of+the+Fungus+Mycosphaerella+graminicola." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sexual+Reproduction+Plays+a+Major+Role+in+the+Genetic+Structure+of+Populations+of+the+Fungus+Mycosphaerella+graminicola.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001455-Nei2" id="pone.0001455-Nei2"></a>Nei M (1973) Analysis of Gene Diversity in Subdivided Populations. Proc Natl Acad Sci USA  70: 3321–3323.  <ul class="find" data-citedArticleID="1043708" data-doi="10.1073/pnas.70.12.3321"><li><a href="http://dx.doi.org/10.1073/pnas.70.12.3321" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+Gene+Diversity+in+Subdivided+Populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+Gene+Diversity+in+Subdivided+Populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001455-elMousadik1" id="pone.0001455-elMousadik1"></a>el Mousadik A, Petit RJ (1996) High level of genetic differentiation for allelic richness among populations of the argan tree [<em>Argania spinosa</em> (L.) Skeels] endemic to Morocco. Theor Appl Genet  92: 832–839.  <ul class="find" data-citedArticleID="1043780" data-doi="10.1007/bf00221895"><li><a href="http://dx.doi.org/10.1007/bf00221895" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+level+of+genetic+differentiation+for+allelic+richness+among+populations+of+the+argan+tree+%5BArgania+spinosa+%28L.%29+Skeels%5D+endemic+to+Morocco." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+level+of+genetic+differentiation+for+allelic+richness+among+populations+of+the+argan+tree+%5BArgania+spinosa+%28L.%29+Skeels%5D+endemic+to+Morocco.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001455-Leberg1" id="pone.0001455-Leberg1"></a>Leberg PL (2002) Estimating allelic richness: Effects of sample size and bottlenecks. Mol Ecol  11: 2445–2449.  <ul class="find" data-citedArticleID="1043694" data-doi="10.1046/j.1365-294x.2002.01612.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2002.01612.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+allelic+richness%3A+Effects+of+sample+size+and+bottlenecks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+allelic+richness%3A+Effects+of+sample+size+and+bottlenecks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001455-Brown3" id="pone.0001455-Brown3"></a>Brown AHD, Feldman MW, Nevo E (1980) Multilocus structure of natural populations of <em>Hordeum spontaneum</em>. Genetics  96: 523–536.  <ul class="find" data-citedArticleID="1043612"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Multilocus+structure+of+natural+populations+of+Hordeum+spontaneum.&amp;auth=&amp;atitle=Multilocus+structure+of+natural+populations+of+Hordeum+spontaneum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multilocus+structure+of+natural+populations+of+Hordeum+spontaneum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multilocus+structure+of+natural+populations+of+Hordeum+spontaneum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001455-Agapow1" id="pone.0001455-Agapow1"></a>Agapow P-M, Burt A (2001) Indices of multilocus linkage disequilibrium. Mol Ecol Notes  1: 101–102.  <ul class="find" data-citedArticleID="1043588" data-doi="10.1046/j.1471-8278.2000.00014.x"><li><a href="http://dx.doi.org/10.1046/j.1471-8278.2000.00014.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Indices+of+multilocus+linkage+disequilibrium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Indices+of+multilocus+linkage+disequilibrium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001455-Zaffarano1" id="pone.0001455-Zaffarano1"></a>Zaffarano P, McDonald B, Zala M, Linde C (2006) Global Hierarchical Gene Diversity Analysis Suggests the Fertile Crescent Is Not the Center of Origin of the Barley Scald Pathogen <em>Rhynchosporium secalis</em>. Phytopathology  96: 941–950.  <ul class="find" data-citedArticleID="1043774" data-doi="10.1094/phyto-96-0941"><li><a href="http://dx.doi.org/10.1094/phyto-96-0941" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+Hierarchical+Gene+Diversity+Analysis+Suggests+the+Fertile+Crescent+Is+Not+the+Center+of+Origin+of+the+Barley+Scald+Pathogen+Rhynchosporium+secalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+Hierarchical+Gene+Diversity+Analysis+Suggests+the+Fertile+Crescent+Is+Not+the+Center+of+Origin+of+the+Barley+Scald+Pathogen+Rhynchosporium+secalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001455-Piry1" id="pone.0001455-Piry1"></a>Piry S, Luikart G, Cornuet J-M (1999) BOTTLENECK: a computer program for detecting recent reductions in the effective size using allele frequency data. J Hered  90: 502–503.  <ul class="find" data-citedArticleID="1043724" data-doi="10.1093/jhered/90.4.502"><li><a href="http://dx.doi.org/10.1093/jhered/90.4.502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BOTTLENECK%3A+a+computer+program+for+detecting+recent+reductions+in+the+effective+size+using+allele+frequency+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BOTTLENECK%3A+a+computer+program+for+detecting+recent+reductions+in+the+effective+size+using+allele+frequency+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001455-Kimmel1" id="pone.0001455-Kimmel1"></a>Kimmel M, Chakraborty R, King J, Bamshad M, Watkins W, et al.  (1998) Signatures of population expansion in microsatellite repeat data. Genetics  148: 1921–1930.  <ul class="find" data-citedArticleID="1043690"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Signatures+of+population+expansion+in+microsatellite+repeat+data.&amp;auth=&amp;atitle=Signatures+of+population+expansion+in+microsatellite+repeat+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signatures+of+population+expansion+in+microsatellite+repeat+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signatures+of+population+expansion+in+microsatellite+repeat+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001455-King1" id="pone.0001455-King1"></a>King JP, Kimmel M, Chakraborty R (2000) A Power Analysis of Microsatellite-Based Statistics for Inferring Past Population Growth. Mol Biol Evol  17: 1859–1868.  <ul class="find" data-citedArticleID="1043692" data-doi="10.1093/oxfordjournals.molbev.a026287"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026287" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Power+Analysis+of+Microsatellite-Based+Statistics+for+Inferring+Past+Population+Growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Power+Analysis+of+Microsatellite-Based+Statistics+for+Inferring+Past+Population+Growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001455-Reich1" id="pone.0001455-Reich1"></a>Reich DE, Goldstein DB (1998) Genetic evidence for a Paleolithic human population expansion in Africa. Proc Natl Acad Sci USA  95: 8119–8123.  <ul class="find" data-citedArticleID="1043734" data-doi="10.1073/pnas.95.14.8119"><li><a href="http://dx.doi.org/10.1073/pnas.95.14.8119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+evidence+for+a+Paleolithic+human+population+expansion+in+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+evidence+for+a+Paleolithic+human+population+expansion+in+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001455-Reich2" id="pone.0001455-Reich2"></a>Reich DE, Feldman MW, Goldstein DB (1999) Statistical Properties of Two Tests that Use Multilocus Data Sets to Detect Population Expansions. Mol Biol Evol  16: 453–466.  <ul class="find" data-citedArticleID="1043736" data-doi="10.1093/oxfordjournals.molbev.a026127"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026127" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+Properties+of+Two+Tests+that+Use+Multilocus+Data+Sets+to+Detect+Population+Expansions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+Properties+of+Two+Tests+that+Use+Multilocus+Data+Sets+to+Detect+Population+Expansions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001455-CavalliSforza1" id="pone.0001455-CavalliSforza1"></a>Cavalli-Sforza L, Edwards A (1967) Phylogenetic Analysis: Models and Estimation Procedures. Evolution  21: 550–570.  <ul class="find" data-citedArticleID="1043620" data-doi="10.2307/2406616"><li><a href="http://dx.doi.org/10.2307/2406616" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+Analysis%3A+Models+and+Estimation+Procedures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+Analysis%3A+Models+and+Estimation+Procedures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001455-Dieringer1" id="pone.0001455-Dieringer1"></a>Dieringer D, Schlotterer C (2003) microsatellite analyser (MSA): a platform independent analysis tool for large microsatellite data sets. Mol Ecol Notes  3: 167–169.  <ul class="find" data-citedArticleID="1043632" data-doi="10.1046/j.1471-8286.2003.00351.x"><li><a href="http://dx.doi.org/10.1046/j.1471-8286.2003.00351.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=microsatellite+analyser+%28MSA%29%3A+a+platform+independent+analysis+tool+for+large+microsatellite+data+sets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22microsatellite+analyser+%28MSA%29%3A+a+platform+independent+analysis+tool+for+large+microsatellite+data+sets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001455-Pritchard1" id="pone.0001455-Pritchard1"></a>Pritchard JK, Stephens M, Donnelly P (2000) Inference of Population Structure Using Multilocus Genotype Data. Genetics  155: 945–959.  <ul class="find" data-citedArticleID="1043728"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inference+of+Population+Structure+Using+Multilocus+Genotype+Data.&amp;auth=&amp;atitle=Inference+of+Population+Structure+Using+Multilocus+Genotype+Data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inference+of+Population+Structure+Using+Multilocus+Genotype+Data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inference+of+Population+Structure+Using+Multilocus+Genotype+Data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001455-Falush1" id="pone.0001455-Falush1"></a>Falush D, Stephens M, Pritchard JK (2003) Inference of Population Structure Using Multilocus Genotype Data: Linked Loci and Correlated Allele Frequencies. Genetics  164: 1567–1587.  <ul class="find" data-citedArticleID="1043648"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inference+of+Population+Structure+Using+Multilocus+Genotype+Data%3A+Linked+Loci+and+Correlated+Allele+Frequencies.&amp;auth=&amp;atitle=Inference+of+Population+Structure+Using+Multilocus+Genotype+Data%3A+Linked+Loci+and+Correlated+Allele+Frequencies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inference+of+Population+Structure+Using+Multilocus+Genotype+Data%3A+Linked+Loci+and+Correlated+Allele+Frequencies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inference+of+Population+Structure+Using+Multilocus+Genotype+Data%3A+Linked+Loci+and+Correlated+Allele+Frequencies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001455-Corander1" id="pone.0001455-Corander1"></a>Corander J, Waldmann P, Sillanpaa MJ (2003) Bayesian Analysis of Genetic Differentiation Between Populations. Genetics  163: 367–374.  <ul class="find" data-citedArticleID="1043624"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bayesian+Analysis+of+Genetic+Differentiation+Between+Populations.&amp;auth=&amp;atitle=Bayesian+Analysis+of+Genetic+Differentiation+Between+Populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+Analysis+of+Genetic+Differentiation+Between+Populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+Analysis+of+Genetic+Differentiation+Between+Populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001455-Piry2" id="pone.0001455-Piry2"></a>Piry S, Alapetite A, Cornuet J-M, Paetkau D, Baudouin L, et al.  (2004) GENECLASS2: A Software for Genetic Assignment and First-Generation Migrant Detection. J Hered  95: 536–539.  <ul class="find" data-citedArticleID="1043726" data-doi="10.1093/jhered/esh074"><li><a href="http://dx.doi.org/10.1093/jhered/esh074" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GENECLASS2%3A+A+Software+for+Genetic+Assignment+and+First-Generation+Migrant+Detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GENECLASS2%3A+A+Software+for+Genetic+Assignment+and+First-Generation+Migrant+Detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001455-Rannala1" id="pone.0001455-Rannala1"></a>Rannala B, Mountain JL (1997) Detecting immigration by using multilocus genotypes. Proc Natl Acad Sci USA  94: 9197–920.  <ul class="find" data-citedArticleID="1043732" data-doi="10.1073/pnas.94.17.9197"><li><a href="http://dx.doi.org/10.1073/pnas.94.17.9197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+immigration+by+using+multilocus+genotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+immigration+by+using+multilocus+genotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001455-Paetkau1" id="pone.0001455-Paetkau1"></a>Paetkau D, Slade R, Burden M, Estoup A (2004) Genetic assignment methods for the direct, real-time estimation of migration rate: a simulation-based exploration of accuracy and power. Mol Ecol  13: 55–65.  <ul class="find" data-citedArticleID="1043710" data-doi="10.1046/j.1365-294x.2004.02008.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2004.02008.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+assignment+methods+for+the+direct%2C+real-time+estimation+of+migration+rate%3A+a+simulation-based+exploration+of+accuracy+and+power." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+assignment+methods+for+the+direct%2C+real-time+estimation+of+migration+rate%3A+a+simulation-based+exploration+of+accuracy+and+power.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001455-Beerli1" id="pone.0001455-Beerli1"></a>Beerli P, Felsenstein J (2001) Maximum likelihood estimation of a migration matrix and effective population sizes in n subpopulations by using a coalescent approach. Proc Natl Acad Sci USA  98: 4563–4568.  <ul class="find" data-citedArticleID="1043602" data-doi="10.1073/pnas.081068098"><li><a href="http://dx.doi.org/10.1073/pnas.081068098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum+likelihood+estimation+of+a+migration+matrix+and+effective+population+sizes+in+n+subpopulations+by+using+a+coalescent+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum+likelihood+estimation+of+a+migration+matrix+and+effective+population+sizes+in+n+subpopulations+by+using+a+coalescent+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001455-Berry1" id="pone.0001455-Berry1"></a>Berry O, Tocher MD, Sarre SD (2004) Can assignment tests measure dispersal? Mol Ecol  13: 551–561.  <ul class="find" data-citedArticleID="1043604" data-doi="10.1046/j.1365-294x.2004.2081.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2004.2081.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Can+assignment+tests+measure+dispersal%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Can+assignment+tests+measure+dispersal%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0001455-Manel1" id="pone.0001455-Manel1"></a>Manel S, Gaggiotti O, Waples R (2005) Assignment methods: matching biological questions with appropriate techniques. Trends Ecol Evol  20: 136–142.  <ul class="find" data-citedArticleID="1043698" data-doi="10.1016/j.tree.2004.12.004"><li><a href="http://dx.doi.org/10.1016/j.tree.2004.12.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assignment+methods%3A+matching+biological+questions+with+appropriate+techniques." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assignment+methods%3A+matching+biological+questions+with+appropriate+techniques.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0001455-Jorde1" id="pone.0001455-Jorde1"></a>Jorde L, Bamshad M, Rogers A (1998) Using mitochondrial and nuclear DNA markers to reconstruct human evolution. Bioessays  20: 126–136.  <ul class="find" data-citedArticleID="1043680" data-doi="10.1002/(sici)1521-1878(199802)20:2<126::aid-bies5&gt;3.0.co;2-r"><li><a href="http://dx.doi.org/10.1002/(sici)1521-1878(199802)20:2<126::aid-bies5&gt;3.0.co;2-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+mitochondrial+and+nuclear+DNA+markers+to+reconstruct+human+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+mitochondrial+and+nuclear+DNA+markers+to+reconstruct+human+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0001455-Brunner1" id="pone.0001455-Brunner1"></a>Brunner PC, Schurch S, McDonald BA (2007) The origin and colonization history of the barley scald pathogen <em>Rhynchosporium secalis</em>. J Evol Biol  20: 1–12.  <ul class="find" data-citedArticleID="1043614" data-doi="10.1111/j.1420-9101.2007.01347.x"><li><a href="http://dx.doi.org/10.1111/j.1420-9101.2007.01347.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+origin+and+colonization+history+of+the+barley+scald+pathogen+Rhynchosporium+secalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+origin+and+colonization+history+of+the+barley+scald+pathogen+Rhynchosporium+secalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0001455-Couch1" id="pone.0001455-Couch1"></a>Couch BC, Fudal I, Lebrun M-H, Tharreau D, Valent B, et al.  (2005) Origins of Host-Specific Populations of the Blast Pathogen <em>Magnaporthe oryzae</em> in Crop Domestication With Subsequent Expansion of Pandemic Clones on Rice and Weeds of Rice. Genetics  170: 613–630.  <ul class="find" data-citedArticleID="1043628" data-doi="10.1534/genetics.105.041780"><li><a href="http://dx.doi.org/10.1534/genetics.105.041780" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origins+of+Host-Specific+Populations+of+the+Blast+Pathogen+Magnaporthe+oryzae+in+Crop+Domestication+With+Subsequent+Expansion+of+Pandemic+Clones+on+Rice+and+Weeds+of+Rice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origins+of+Host-Specific+Populations+of+the+Blast+Pathogen+Magnaporthe+oryzae+in+Crop+Domestication+With+Subsequent+Expansion+of+Pandemic+Clones+on+Rice+and+Weeds+of+Rice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0001455-GomezAlpizar1" id="pone.0001455-GomezAlpizar1"></a>Gomez-Alpizar L, Carbone I, Ristaino JB (2007) An Andean origin of <em>Phytophthora infestans</em> inferred from mitochondrial and nuclear gene genealogies. Proc Natl Acad Sci USA  104: 3306–3311.  <ul class="find" data-citedArticleID="1043658" data-doi="10.1073/pnas.0611479104"><li><a href="http://dx.doi.org/10.1073/pnas.0611479104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Andean+origin+of+Phytophthora+infestans+inferred+from+mitochondrial+and+nuclear+gene+genealogies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Andean+origin+of+Phytophthora+infestans+inferred+from+mitochondrial+and+nuclear+gene+genealogies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0001455-Raboin1" id="pone.0001455-Raboin1"></a>Raboin L-M, Selvi A, Oliveira KM, Paulet F, Calatayud C, et al.  (2007) Evidence for the dispersal of a unique lineage from Asia to America and Africa in the sugarcane fungal pathogen <em>Ustilago scitaminea</em>. Fungal Genet Biol  44: 64–76.  <ul class="find" data-citedArticleID="1043730" data-doi="10.1016/j.fgb.2006.07.004"><li><a href="http://dx.doi.org/10.1016/j.fgb.2006.07.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+the+dispersal+of+a+unique+lineage+from+Asia+to+America+and+Africa+in+the+sugarcane+fungal+pathogen+Ustilago+scitaminea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+the+dispersal+of+a+unique+lineage+from+Asia+to+America+and+Africa+in+the+sugarcane+fungal+pathogen+Ustilago+scitaminea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0001455-Stukenbrock2" id="pone.0001455-Stukenbrock2"></a>Stukenbrock E, Banke S, Javan-Nikkhah M, McDonald B (2007) Origin and domestication of the fungal wheat pathogen <em>Mycosphaerella graminicola</em> via sympatric speciation. Mol Biol Evol  24: 398–411.  <ul class="find" data-citedArticleID="1043750" data-doi="10.1093/molbev/msl169"><li><a href="http://dx.doi.org/10.1093/molbev/msl169" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+and+domestication+of+the+fungal+wheat+pathogen+Mycosphaerella+graminicola+via+sympatric+speciation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+and+domestication+of+the+fungal+wheat+pathogen+Mycosphaerella+graminicola+via+sympatric+speciation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0001455-Goodwin1" id="pone.0001455-Goodwin1"></a>Goodwin SB, Cohen BA, Fry WE (1994) Panglobal Distribution of a Single Clonal Lineage of the Irish Potato Famine Fungus. Proc Natl Acad Sci USA  91: 11591–11595.  <ul class="find" data-citedArticleID="1043660" data-doi="10.1073/pnas.91.24.11591"><li><a href="http://dx.doi.org/10.1073/pnas.91.24.11591" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Panglobal+Distribution+of+a+Single+Clonal+Lineage+of+the+Irish+Potato+Famine+Fungus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Panglobal+Distribution+of+a+Single+Clonal+Lineage+of+the+Irish+Potato+Famine+Fungus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0001455-Burgess1" id="pone.0001455-Burgess1"></a>Burgess T, Wingfield B, Wingfield M (2001) Comparison of genotypic diversity in native and introduced populations of <em>Sphaeropsis sapinea</em> isolated from <em>Pinus radiata</em>. Mycological Research  105: 1331–1339.  <ul class="find" data-citedArticleID="1043616" data-doi="10.1017/s0953756201005056"><li><a href="http://dx.doi.org/10.1017/s0953756201005056" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+genotypic+diversity+in+native+and+introduced+populations+of+Sphaeropsis+sapinea+isolated+from+Pinus+radiata." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+genotypic+diversity+in+native+and+introduced+populations+of+Sphaeropsis+sapinea+isolated+from+Pinus+radiata.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0001455-Wikler1" id="pone.0001455-Wikler1"></a>Wikler K, Gordon TR (2000) An initial assessment of genetic relationships among populations of <em>Fusarium circinatum</em> in different parts of the world. Can J Bot  78: 709–717.  <ul class="find" data-citedArticleID="1043762" data-doi="10.1139/b00-044"><li><a href="http://dx.doi.org/10.1139/b00-044" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+initial+assessment+of+genetic+relationships+among+populations+of+Fusarium+circinatum+in+different+parts+of+the+world." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+initial+assessment+of+genetic+relationships+among+populations+of+Fusarium+circinatum+in+different+parts+of+the+world.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0001455-FAO1" id="pone.0001455-FAO1"></a>FAO (2004)   FAOSTAT Archives (<a href="http://faostat.fao.org/">http://faostat.fao.org/</a>).  <ul class="find-nolinks"></ul></li><li><span class="label">88.
              </span><a name="pone.0001455-Jin1" id="pone.0001455-Jin1"></a>Jin L, Chakraborty R (1995) Population structure, stepwise mutations, heterozygote deficiency and their implications in DNA forensics. Heredity  74: 274–285.  <ul class="find" data-citedArticleID="1043676" data-doi="10.1038/hdy.1995.41"><li><a href="http://dx.doi.org/10.1038/hdy.1995.41" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+structure%2C+stepwise+mutations%2C+heterozygote+deficiency+and+their+implications+in+DNA+forensics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+structure%2C+stepwise+mutations%2C+heterozygote+deficiency+and+their+implications+in+DNA+forensics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0001455-Donnelly2" id="pone.0001455-Donnelly2"></a>Donnelly M, Simard F, Lehmann T (2002) Evolutionary studies of malaria vectors. Trends Parasitol  18: 75–80.  <ul class="find" data-citedArticleID="1043636" data-doi="10.1016/s1471-4922(01)02198-5"><li><a href="http://dx.doi.org/10.1016/s1471-4922(01)02198-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+studies+of+malaria+vectors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+studies+of+malaria+vectors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0001455-Malvarez1" id="pone.0001455-Malvarez1"></a>Malvarez G, Carbone I, Grunwald NJ, Subbarao KV, Schafer M, et al.  (2007) New Populations of <em>Sclerotinia sclerotiorum</em> from Lettuce in California and Peas and Lentils in Washington. Phytopathology  97: 470–483.  <ul class="find" data-citedArticleID="1043696" data-doi="10.1094/phyto-97-4-0470"><li><a href="http://dx.doi.org/10.1094/phyto-97-4-0470" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+Populations+of+Sclerotinia+sclerotiorum+from+Lettuce+in+California+and+Peas+and+Lentils+in+Washington." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+Populations+of+Sclerotinia+sclerotiorum+from+Lettuce+in+California+and+Peas+and+Lentils+in+Washington.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0001455-Kauer1" id="pone.0001455-Kauer1"></a>Kauer M, Dieringer D, Schlotterer C (2003) Nonneutral Admixture of Immigrant Genotypes in African <em>Drosophila melanogaster</em> Populations from Zimbabwe. Mol Biol Evol  20: 1329–1337.  <ul class="find" data-citedArticleID="1043688" data-doi="10.1093/molbev/msg148"><li><a href="http://dx.doi.org/10.1093/molbev/msg148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonneutral+Admixture+of+Immigrant+Genotypes+in+African+Drosophila+melanogaster+Populations+from+Zimbabwe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonneutral+Admixture+of+Immigrant+Genotypes+in+African+Drosophila+melanogaster+Populations+from+Zimbabwe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0001455-Schofl1" id="pone.0001455-Schofl1"></a>Schofl G, Schlotterer C (2006) Microsatellite variation and differentiation in African and non-African populations of <em>Drosophila simulans</em>. Mol Ecol  15: 3895–3905.  <ul class="find" data-citedArticleID="1043744" data-doi="10.1111/j.1365-294x.2006.03065.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2006.03065.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microsatellite+variation+and+differentiation+in+African+and+non-African+populations+of+Drosophila+simulans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microsatellite+variation+and+differentiation+in+African+and+non-African+populations+of+Drosophila+simulans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0001455-Pfaff1" id="pone.0001455-Pfaff1"></a>Pfaff C, Parra E, Bonilla C, Hiester K, McKeigue P, et al.  (2001) Population structure in admixed populations: effect of admixture dynamics on the pattern of linkage disequilibrium. Am J Hum Genet  68: 198–207.  <ul class="find" data-citedArticleID="1043720" data-doi="10.1086/316935"><li><a href="http://dx.doi.org/10.1086/316935" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+structure+in+admixed+populations%3A+effect+of+admixture+dynamics+on+the+pattern+of+linkage+disequilibrium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+structure+in+admixed+populations%3A+effect+of+admixture+dynamics+on+the+pattern+of+linkage+disequilibrium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">94.
              </span><a name="pone.0001455-Wilson1" id="pone.0001455-Wilson1"></a>Wilson E (1928) Studies of the ascigerous stage of <em>Venturia inaequalis</em>. Phytopathology  18: 375–418.  <ul class="find" data-citedArticleID="1043764"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Studies+of+the+ascigerous+stage+of+Venturia+inaequalis.&amp;auth=&amp;atitle=Studies+of+the+ascigerous+stage+of+Venturia+inaequalis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Studies+of+the+ascigerous+stage+of+Venturia+inaequalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Studies+of+the+ascigerous+stage+of+Venturia+inaequalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">95.
              </span><a name="pone.0001455-Boehm1" id="pone.0001455-Boehm1"></a>Boehm E, Freeman S, Shabi E, Michailides T (2003) Microsatellite Primers Indicate the Presence of Asexual Populations of <em>Venturia inaequalis</em> in Coastal Israeli Apple Orchards. Phytoparasitica  31: 236–251.  <ul class="find" data-citedArticleID="1043606" data-doi="10.1007/bf02980833"><li><a href="http://dx.doi.org/10.1007/bf02980833" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microsatellite+Primers+Indicate+the+Presence+of+Asexual+Populations+of+Venturia+inaequalis+in+Coastal+Israeli+Apple+Orchards." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microsatellite+Primers+Indicate+the+Presence+of+Asexual+Populations+of+Venturia+inaequalis+in+Coastal+Israeli+Apple+Orchards.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">96.
              </span><a name="pone.0001455-Hume1" id="pone.0001455-Hume1"></a>Hume J, Lyons E, Day K (2003) Human migration, mosquitoes and the evolution of <em>Plasmodium falciparum</em>. Trends Parasitol  19: 144–149.  <ul class="find" data-citedArticleID="1043672" data-doi="10.1016/s1471-4922(03)00008-4"><li><a href="http://dx.doi.org/10.1016/s1471-4922(03)00008-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+migration%2C+mosquitoes+and+the+evolution+of+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+migration%2C+mosquitoes+and+the+evolution+of+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">97.
              </span><a name="pone.0001455-Joy1" id="pone.0001455-Joy1"></a>Joy D, Feng X, Mu J, Furuya T, Chotivanich K, et al.  (2003) Early origin and recent expansion of <em>Plasmodium falciparum</em>. Science  300: 318–321.  <ul class="find" data-citedArticleID="1043682" data-doi="10.1126/science.1081449"><li><a href="http://dx.doi.org/10.1126/science.1081449" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+origin+and+recent+expansion+of+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+origin+and+recent+expansion+of+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.XML" value="223052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.PDF" value="335816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t001.PNG_L" value="578962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t001.PNG_M" value="232766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t001.PNG_S" value="22222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t001.TIF" value="2297196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t001.PNG_I" value="103978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g001.PNG_L" value="26294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g001.PNG_M" value="46560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g001.PNG_S" value="7680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g001.TIF" value="211420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g001.PNG_I" value="18495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t002.PNG_L" value="335888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t002.PNG_M" value="170210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t002.PNG_S" value="19268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t002.TIF" value="1680304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t002.PNG_I" value="64748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t003.PNG_L" value="108538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t003.PNG_M" value="146725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t003.PNG_S" value="18592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t003.TIF" value="447346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t003.PNG_I" value="50893"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t004.PNG_L" value="96055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t004.PNG_M" value="127533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t004.PNG_S" value="16770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t004.TIF" value="448744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t004.PNG_I" value="42649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t005.PNG_L" value="285047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t005.PNG_M" value="98601"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t005.PNG_S" value="23779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t005.TIF" value="1402592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t005.PNG_I" value="175764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g002.PNG_L" value="378395"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g002.PNG_M" value="62755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g002.PNG_S" value="9043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g002.TIF" value="726602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g002.PNG_I" value="21906"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g003.PNG_L" value="421842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g003.PNG_M" value="145912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g003.PNG_S" value="12665"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g003.TIF" value="2744442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g003.PNG_I" value="48836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g004.PNG_L" value="135105"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g004.PNG_M" value="65134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g004.PNG_S" value="9443"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g004.TIF" value="436300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.g004.PNG_I" value="24678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t006.PNG_L" value="172918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t006.PNG_M" value="68621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t006.PNG_S" value="9456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t006.TIF" value="720952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.t006.PNG_I" value="20814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.s001.PDF" value="7831"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.s002.PDF" value="200695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.s003.PDF" value="43048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.s004.PDF" value="742351"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.s005.PDF" value="140325"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.s006.PDF" value="102742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001455.s007.DOC" value="35328"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001455&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001455" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001455&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001455&volume=&issue=&title=On the Origin and Spread of the Scab Disease of Apple: Out of Central Asia&author_name=Pierre%20Gladieux%2C%20Xiu-Guo%20Zhang%2C%20Damien%20Afoufa-Bastien%2C%20Rosa-Maria%20Valdebenito%20Sanhueza%2C%20Mohamed%20Sbaghi%2C%20Bruno%20Le%20Cam&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001455" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001455" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001455" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001455&amp;t=On%20the%20Origin%20and%20Spread%20of%20the%20Scab%20Disease%20of%20Apple%3A%20Out%20of%20Central%20Asia" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001455&title=On%20the%20Origin%20and%20Spread%20of%20the%20Scab%20Disease%20of%20Apple%3A%20Out%20of%20Central%20Asia&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001455&amp;title=On%20the%20Origin%20and%20Spread%20of%20the%20Scab%20Disease%20of%20Apple%3A%20Out%20of%20Central%20Asia" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001455" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001455&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'On the Origin and Spread of the Scab Disease of Apple: Out of Central Asia';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001455';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20On%20the%20Origin%20and%20Spread%20of%20the%20Scab%20Disease%20of%20Apple%3A%20Out%20of%20Central%20Asia http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001455" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001455" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Apples%22" title="Search for articles in the subject area:'Apples'"><div class="flagText">Apples</div></a>
              <div data-categoryid="17779" data-articleid="25738"
                   data-categoryname="Apples"
                   class="flagImage" title="Flag 'Apples' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Asia%22" title="Search for articles in the subject area:'Asia'"><div class="flagText">Asia</div></a>
              <div data-categoryid="40353" data-articleid="25738"
                   data-categoryname="Asia"
                   class="flagImage" title="Flag 'Asia' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Europe%22" title="Search for articles in the subject area:'Europe'"><div class="flagText">Europe</div></a>
              <div data-categoryid="40225" data-articleid="25738"
                   data-categoryname="Europe"
                   class="flagImage" title="Flag 'Europe' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+genetics%22" title="Search for articles in the subject area:'Fungal genetics'"><div class="flagText">Fungal genetics</div></a>
              <div data-categoryid="18483" data-articleid="25738"
                   data-categoryname="Fungal genetics"
                   class="flagImage" title="Flag 'Fungal genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+structure%22" title="Search for articles in the subject area:'Fungal structure'"><div class="flagText">Fungal structure</div></a>
              <div data-categoryid="19535" data-articleid="25738"
                   data-categoryname="Fungal structure"
                   class="flagImage" title="Flag 'Fungal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungi%22" title="Search for articles in the subject area:'Fungi'"><div class="flagText">Fungi</div></a>
              <div data-categoryid="43047" data-articleid="25738"
                   data-categoryname="Fungi"
                   class="flagImage" title="Flag 'Fungi' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="25738"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="25738"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4633'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9069'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=397&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=41957"></a><br>Posted by atila</p>
      <p><a href="/annotation/listThread.action?root=12307">Correction to Funding </a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=2675">Referee Comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
