

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Multi-Locus Sequence Typing of Bartonella henselae Isolates from Three Continents Reveals Hypervirulent and Feline-Associated Clones</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001346"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001346" />

    <meta name="citation_title" content="Multi-Locus Sequence Typing of Bartonella henselae Isolates from Three Continents Reveals Hypervirulent and Feline-Associated Clones"/>
    <meta itemprop="name" content="Multi-Locus Sequence Typing of Bartonella henselae Isolates from Three Continents Reveals Hypervirulent and Feline-Associated Clones"/>

      <meta name="citation_author" content="Mardjan Arvand"/>
            <meta name="citation_author_institution" content="Institut für Medizinische Mikrobiologie, Virologie und Hygiene, Universität Rostock, Rostock, Germany"/>
      <meta name="citation_author" content="Edward J. Feil"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Bath, Claverton Down, Bath, United Kingdom"/>
      <meta name="citation_author" content="Michael Giladi"/>
            <meta name="citation_author_institution" content="Bernard Pridan Laboratory for Molecular Biology of Infectious Diseases, Tel Aviv Medical Center, Tel Aviv, Israel"/>
      <meta name="citation_author" content="Henri-Jean Boulouis"/>
            <meta name="citation_author_institution" content="Unité Mixte de Recherche, Ecole Nationale Veterinaire d'Alfort, Maisons-Alfort, France"/>
      <meta name="citation_author" content="Juliane Viezens"/>
            <meta name="citation_author_institution" content="Institut für Medizinische Mikrobiologie, Virologie und Hygiene, Universität Rostock, Rostock, Germany"/>

    <meta name="citation_date" content="2007/12/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001346.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1346"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Cat scratch disease.; citation_author=AM Margileth; citation_journal_title=Adv Pediatr Infect Dis; citation_volume=8; citation_number=1; citation_pages=1-21; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Isolation of Rochalimaea species from cutaneous and osseous lesions of bacillary angiomatosis.; citation_author=JE Koehler; citation_author=FD Quinn; citation_author=TG Berger; citation_author=PE LeBoit; citation_author=JW Tappero; citation_journal_title=N Engl J Med; citation_volume=327; citation_number=2; citation_pages=1625-1631; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Experimental infection of young specific pathogen-free cats with Bartonella henselae.; citation_author=L Guptill; citation_author=L Slater; citation_author=CC Wu; citation_author=TL Lin; citation_author=LT Glickman; citation_journal_title=J Infect Dis; citation_volume=176; citation_number=3; citation_pages=206-216; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Detection of Bartonella henselae DNA by two different PCR assays and determination of the genotypes of strains involved in histologically defined cat scratch disease.; citation_author=A Sander; citation_author=M Posselt; citation_author=N Bohm; citation_author=M Ruess; citation_author=M Altwegg; citation_journal_title=J Clin Microbiol; citation_volume=37; citation_number=4; citation_pages=993-997; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=DNA amplification for the diagnosis of cat-scratch disease in small-quantity clinical specimens.; citation_author=B Avidor; citation_author=M Varon; citation_author=S Marmor; citation_author=B Lifschitz-Mercer; citation_author=Y Kletter; citation_journal_title=Am J Clin Pathol; citation_volume=115; citation_number=5; citation_pages=900-909; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Isolation of Bartonella henselae DNA from the peripheral blood of a patient with cat scratch disease up to 4 months after the cat scratch injury.; citation_author=M Arvand; citation_author=SG Schäd; citation_journal_title=J Clin Microbiol; citation_volume=44; citation_number=6; citation_pages=2288-2290; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A newly recognized fastidious gram-negative pathogen as a cause of fever and bacteremia.; citation_author=LN Slater; citation_author=DF Welch; citation_author=D Hensel; citation_author=DW Coody; citation_journal_title=N Engl J Med; citation_volume=323; citation_number=7; citation_pages=1587-1593; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Syndrome of Rochalimaea henselae adenitis suggesting cat scratch disease.; citation_author=MJ Dolan; citation_author=MT Wong; citation_author=RL Regnery; citation_author=JH Jorgensen; citation_author=M Garcia; citation_journal_title=Ann Intern Med; citation_volume=118; citation_number=8; citation_pages=331-336; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Limited Diversity among Human Isolates of Bartonella henselae.; citation_author=B Dillon; citation_author=J Valenzuela; citation_author=R Don; citation_author=D Blanckenberg; citation_author=DI Wigney; citation_journal_title=J Clin Microbiol; citation_volume=40; citation_number=9; citation_pages=4691-4699; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=New serotype of Bartonella henselae in endocarditis and cat-scratch disease.; citation_author=M Drancourt; citation_author=R Birtles; citation_author=G Chaumentin; citation_author=F Vandenesch; citation_author=J Etienne; citation_journal_title=Lancet; citation_volume=347; citation_number=10; citation_pages=441-443; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Bartonella henselae prevalence in domestic cats in California: risk factors and association between bacteremia and antibody titers.; citation_author=BB Chomel; citation_author=RC Abbott; citation_author=RW Kasten; citation_author=KA Floyd-Hawkins; citation_author=PH Kass; citation_journal_title=J Clin Microbiol; citation_volume=33; citation_number=11; citation_pages=2445-2450; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Prevalence of Bartonella henselae and Bartonella clarridgeiae in stray cats.; citation_author=R Heller; citation_author=M Artois; citation_author=V Xemar; citation_author=D De Briel; citation_author=H Gehin; citation_journal_title=J Clin Microbiol; citation_volume=35; citation_number=12; citation_pages=1327-1331; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Detection and identification of two Bartonella henselae variants in domestic cats in Germany.; citation_author=A Sander; citation_author=C Buhler; citation_author=K Pelz; citation_author=E von Cramm; citation_author=W Bredt; citation_journal_title=J Clin Microbiol; citation_volume=35; citation_number=13; citation_pages=584-587; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Prevalence of Bartonella species in domestic cats in The Netherlands.; citation_author=AM Bergmans; citation_author=CM de Jong; citation_author=G van Amerongen; citation_author=CS Schot; citation_author=LM Schouls; citation_journal_title=J Clin Microbiol; citation_volume=35; citation_number=14; citation_pages=2256-2261; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Prevalence of Bartonella henselae bacteremia, the causative agent of cat scratch disease, in an Australian cat population.; citation_author=J Branley; citation_author=C Wolfson; citation_author=P Waters; citation_author=T Gottlieb; citation_author=R Bradbury; citation_journal_title=Pathology; citation_volume=28; citation_number=15; citation_pages=262-265; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Comparison of different DNA fingerprinting techniques for molecular typing of Bartonella henselae isolates.; citation_author=A Sander; citation_author=M Ruess; citation_author=S Bereswill; citation_author=M Schuppler; citation_author=B Steinbrueckner; citation_journal_title=J Clin Microbiol; citation_volume=36; citation_number=16; citation_pages=2973-2981; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genomic fingerprinting of Bartonella species by repetitive element PCR for distinguishing species and isolates.; citation_author=MC Rodriguez-Barradas; citation_author=RJ Hamill; citation_author=ED Houston; citation_author=PR Georghiou; citation_author=JE Clarridge; citation_journal_title=J Clin Microbiol; citation_volume=33; citation_number=17; citation_pages=1089-1093; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Genomic variation of Bartonella henselae strains detected in lymph nodes of patients with cat scratch disease.; citation_author=Z Zeaiter; citation_author=PE Fournier; citation_author=D Raoult; citation_journal_title=J Clin Microbiol; citation_volume=40; citation_number=18; citation_pages=1023-1030; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Predominance of two Bartonella henselae variants among cat-scratch disease patients in the Netherlands.; citation_author=AM Bergmans; citation_author=JF Schellekens; citation_author=JD van Embden; citation_author=LM Schouls; citation_journal_title=J Clin Microbiol; citation_volume=34; citation_number=19; citation_pages=254-260; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Genetic variability and prevalence of Bartonella henselae in cats in Berlin, Germany, and analysis of its genetic relatedness to a strain from Berlin that is pathogenic for humans.; citation_author=M Arvand; citation_author=AJ Klose; citation_author=D Schwartz-Porsche; citation_author=H Hahn; citation_author=C Wendt; citation_journal_title=J Clin Microbiol; citation_volume=39; citation_number=20; citation_pages=743-746; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Sequence variation in the ftsZ gene of Bartonella henselae isolates and clinical samples.; citation_author=C Ehrenborg; citation_author=L Wesslen; citation_author=A Jakobson; citation_author=G Friman; citation_author=M Holmberg; citation_journal_title=J Clin Microbiol; citation_volume=38; citation_number=21; citation_pages=682-687; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Characterization of the natural population of Bartonella henselae by multilocus sequence typing.; citation_author=J Iredell; citation_author=D Blanckenberg; citation_author=M Arvand; citation_author=S Grauling; citation_author=EJ Feil; citation_journal_title=J Clin Microbiol; citation_volume=41; citation_number=22; citation_pages=5071-5079; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evaluation of pulsed-field gel electrophoresis and multi-locus sequence typing for the analysis of clonal relatedness among Bartonella henselae isolates.; citation_author=M Arvand; citation_author=J Viezens; citation_journal_title=Int J Med Microbiol; citation_volume=297; citation_number=23; citation_pages=255-262; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Analyses of clonality and the evolution of bacterial pathogens.; citation_author=EJ Feil; citation_author=MC Enright; citation_journal_title=Curr Opin Microbiol; citation_volume=7; citation_number=24; citation_pages=308-313; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=eBURST: inferring patterns of evolutionary descent among clusters of related bacterial genotypes from multilocus sequence typing data.; citation_author=EJ Feil; citation_author=BC Li; citation_author=DM Aanensen; citation_author=WP Hanage; citation_author=BG Spratt; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=25; citation_pages=1518-1530; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Application of phylogenetic networks in evolutionary studies.; citation_author=DH Huson; citation_author=D Bryant; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=26; citation_pages=254-267; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Use of rpoB gene analysis for detection and identification of Bartonella species.; citation_author=P Renesto; citation_author=J Gouvernet; citation_author=M Drancourt; citation_author=V Roux; citation_author=D Raoult; citation_journal_title=J Clin Microbiol; citation_volume=39; citation_number=27; citation_pages=430-437; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genome Rearrangements, Deletions, and Amplifications in the Natural Population of Bartonella henselae.; citation_author=H Lindroos; citation_author=O Vinnere; citation_author=A Mira; citation_author=D Repsilber; citation_author=K Naslund; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=28; citation_pages=7426-7439; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genotypic characteristics of two serotypes of Bartonella henselae.; citation_author=B La Scola; citation_author=Z Liang; citation_author=Z Zeaiter; citation_author=P Houpikian; citation_author=PA Grimont; citation_journal_title=J Clin Microbiol; citation_volume=40; citation_number=29; citation_pages=2002-2008; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0.; citation_author=K Tamura; citation_author=J Dudley; citation_author=M Nei; citation_author=S Kumar; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=30; citation_pages=1596-1599; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Assessing the reliability of eBURST using simulated populations with known ancestry.; citation_author=KM Turner; citation_author=WP Hanage; citation_author=C Fraser; citation_author=TR Connor; citation_author=BG Spratt; citation_journal_title=BMC Microbiol; citation_volume=7; citation_number=31; citation_pages=30; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Detection and characterization of feline Bartonella henselae in the Czech Republic.; citation_author=O Melter; citation_author=K Hercik; citation_author=RS Weyant; citation_author=J Janecek; citation_author=A Nemec; citation_journal_title=Vet Microbiol; citation_volume=93; citation_number=32; citation_pages=261-273; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Epidemiology of Bartonella infection in domestic cats in France.; citation_author=AN Gurfield; citation_author=HJ Boulouis; citation_author=BB Chomel; citation_author=RW Kasten; citation_author=R Heller; citation_journal_title=Vet Microbiol; citation_volume=80; citation_number=33; citation_pages=185-198; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Characterization of Bartonella henselae isolated from bacillary angiomatosis lesions in a human immunodeficiency virus-infected patient in Germany.; citation_author=M Arvand; citation_author=C Wendt; citation_author=T Regnath; citation_author=R Ullrich; citation_author=H Hahn; citation_journal_title=Clin Infect Dis; citation_volume=26; citation_number=34; citation_pages=1296-1299; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Prevalence of Bartonella henselae in Italian stray cats: evaluation of serology to assess the risk of transmission of Bartonella to humans.; citation_author=M Fabbi; citation_author=L De Giuli; citation_author=M Tranquillo; citation_author=R Bragoni; citation_author=M Casiraghi; citation_journal_title=J Clin Microbiol; citation_volume=42; citation_number=35; citation_pages=264-268; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Prevalence of Bartonella species causing bacteraemia in domesticated and companion animals in the United Kingdom.; citation_author=RJ Birtles; citation_author=G Laycock; citation_author=MJ Kenny; citation_author=SE Shaw; citation_author=MJ Day; citation_journal_title=Vet Rec; citation_volume=151; citation_number=36; citation_pages=225-229; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Prevalence, risk factors, and genetic diversity of Bartonella henselae infections in pet cats in four regions of the United States.; citation_author=L Guptill; citation_author=CC Wu; citation_author=H HogenEsch; citation_author=LN Slater; citation_author=N Glickman; citation_journal_title=J Clin Microbiol; citation_volume=42; citation_number=37; citation_pages=652-659; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001346" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Multi-Locus Sequence Typing of Bartonella henselae Isolates from Three Continents Reveals Hypervirulent and Feline-Associated Clones"/>
    <meta name="twitter:description" content="Bartonella henselae is a zoonotic pathogen and the causative agent of cat scratch disease and a variety of other disease manifestations in humans. Previous investigations have suggested that a limited subset of B. henselae isolates may be associated with human disease. In the present study, 182 human and feline B. henselae isolates from Europe, North America and Australia were analysed by multi-locus sequence typing (MLST) to detect any associations between sequence type (ST), host species and geographical distribution of the isolates. A total of 14 sequence types were detected, but over 66% (16/24) of the isolates recovered from human disease corresponded to a single genotype, ST1, and this type was detected in all three continents. In contrast, 27.2% (43/158) of the feline isolates corresponded to ST7, but this ST was not recovered from humans and was restricted to Europe. The difference in host association of STs 1 (human) and 7 (feline) was statistically significant (P&le;0.001). eBURST analysis assigned the 14 STs to three clonal lineages, which contained two or more STs, and a singleton comprising ST7. These groups were broadly consistent with a neighbour-joining tree, although splits decomposition analysis was indicative of a history of recombination. These data indicate that B. henselae lineages differ in their virulence properties for humans and contribute to a better understanding of the population structure of B. henselae."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001346.g005"/>

  <meta property="og:title" content="Multi-Locus Sequence Typing of Bartonella henselae Isolates from Three Continents Reveals Hypervirulent and Feline-Associated Clones" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001346" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8864'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7598'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1440&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001346">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001346" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001346&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Multi-Locus Sequence Typing of <i>Bartonella henselae</i> Isolates from Three Continents Reveals Hypervirulent and Feline-Associated Clones
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mardjan Arvand
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mardjan.arvand@med.uni-rostock.de">mardjan.arvand@med.uni-rostock.de</a></p>

                <p>Affiliation:
                  Institut für Medizinische Mikrobiologie, Virologie und Hygiene, Universität Rostock, Rostock, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Edward J. Feil, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Bath, Claverton Down, Bath, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael Giladi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bernard Pridan Laboratory for Molecular Biology of Infectious Diseases, Tel Aviv Medical Center, Tel Aviv, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Henri-Jean Boulouis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité Mixte de Recherche, Ecole Nationale Veterinaire d'Alfort, Maisons-Alfort, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Juliane Viezens
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institut für Medizinische Mikrobiologie, Virologie und Hygiene, Universität Rostock, Rostock, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001346</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001346" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001346" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001346" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001346" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001346-t001" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001346-g001" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001346-t002" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001346-g002" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001346-t003" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001346-g003" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001346-g004" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001346-g005" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001346-t004" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001346">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001346" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p><em>Bartonella henselae</em> is a zoonotic pathogen and the causative agent of cat scratch disease and a variety of other disease manifestations in humans. Previous investigations have suggested that a limited subset of <em>B. henselae</em> isolates may be associated with human disease. In the present study, 182 human and feline <em>B. henselae</em> isolates from Europe, North America and Australia were analysed by multi-locus sequence typing (MLST) to detect any associations between sequence type (ST), host species and geographical distribution of the isolates. A total of 14 sequence types were detected, but over 66% (16/24) of the isolates recovered from human disease corresponded to a single genotype, ST1, and this type was detected in all three continents. In contrast, 27.2% (43/158) of the feline isolates corresponded to ST7, but this ST was not recovered from humans and was restricted to Europe. The difference in host association of STs 1 (human) and 7 (feline) was statistically significant (P≤0.001). eBURST analysis assigned the 14 STs to three clonal lineages, which contained two or more STs, and a singleton comprising ST7. These groups were broadly consistent with a neighbour-joining tree, although splits decomposition analysis was indicative of a history of recombination. These data indicate that <em>B. henselae</em> lineages differ in their virulence properties for humans and contribute to a better understanding of the population structure of <em>B. henselae</em>.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Arvand M, Feil EJ, Giladi M, Boulouis H-J, Viezens J (2007) Multi-Locus Sequence Typing of <em>Bartonella henselae</em> Isolates from Three Continents Reveals Hypervirulent and Feline-Associated Clones. PLoS ONE 2(12):
          e1346.
            doi:10.1371/journal.pone.0001346</p><p><strong>Academic Editor: </strong>Rosemary Redfield, University of British Columbia, Canada</p><p><strong>Received:</strong> October 31, 2007; <strong>Accepted:</strong> November 28, 2007; <strong>Published:</strong> December 19, 2007</p><p><strong>Copyright:</strong> © 2007 Arvand et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This project was supported by a grant from the Deutsche Forschungsgemeinschaft to MA.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p><em>Bartonella henselae</em> is a fastidious bacterium associated with a broad spectrum of clinical disease manifestations in humans, including cat scratch disease (CSD) and bacillary angiomatosis (BA). CSD is characterized by subacute regional lymphadenopathy that usually occurs in immunocompetent individuals <a href="#pone.0001346-Margileth1">[1]</a>. BA is a vasculoproliferative disorder which is predominantly encompassed in immunocompromised patients and often associated with chronic or relapsing bacteremia <a href="#pone.0001346-Koehler1">[2]</a>. Cats represent the natural host and main reservoir for <em>B. henselae</em>. Infected animals develop relapsing bacteremia of several months duration without overt clinical symptoms <a href="#pone.0001346-Guptill1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Isolation of <em>B. henselae</em> is hampered by the fastidious nature of the organism. The sensitivity of cultural detection of <em>B. henselae</em> from tissues other than blood (e.g. lymph node biopsy specimen) is relatively low. The diagnosis of CSD and most other disease manifestations relies on detection of bacterial DNA in tissue specimens by PCR or serology <a href="#pone.0001346-Sander1">[4]</a>, <a href="#pone.0001346-Avidor1">[5]</a>, <a href="#pone.0001346-Arvand1">[6]</a>. Therefore, only few human-derived <em>B. henselae</em> isolates are available worldwide <a href="#pone.0001346-Slater1">[7]</a>, <a href="#pone.0001346-Dolan1">[8]</a>, <a href="#pone.0001346-Dillon1">[9]</a>, <a href="#pone.0001346-Drancourt1">[10]</a>. In contrast, Bartonellae can be more easily isolated from the blood of infected cats. Several feline isolates have been collected during prevalence studies from different geographical regions <a href="#pone.0001346-Chomel1">[11]</a>, <a href="#pone.0001346-Heller1">[12]</a>, <a href="#pone.0001346-Sander2">[13]</a>, <a href="#pone.0001346-Bergmans1">[14]</a>, <a href="#pone.0001346-Branley1">[15]</a>. Thus, feline isolates usually outnumber the human-derived isolates in investigations of the molecular epidemiology of <em>B. henselae</em>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Previous studies have shown a considerable genetic heterogeneity among <em>B. henselae</em> isolates by using different DNA fingerprinting methods <a href="#pone.0001346-Drancourt1">[10]</a>, <a href="#pone.0001346-Sander3">[16]</a>, <a href="#pone.0001346-RodriguezBarradas1">[17]</a>, <a href="#pone.0001346-Zeaiter1">[18]</a>. The first suggestion that human-associated isolates represent a limited subset of the total <em>B. henselae</em> population came from a Dutch study of lymph nodes obtained from CSD patients, which revealed a higher prevalence of isolates displaying the 16S RNA-type I in tissue samples of CSD patients than among feline isolates obtained from the same geographic region <a href="#pone.0001346-Bergmans2">[19]</a>. Subsequent studies from Germany <a href="#pone.0001346-Sander1">[4]</a>, <a href="#pone.0001346-Sander3">[16]</a>, <a href="#pone.0001346-Arvand2">[20]</a> and Australia <a href="#pone.0001346-Dillon1">[9]</a> further supported the hypothesis that isolates responsible for human disease are not drawn randomly from the feline reservoir. Recent studies have shown that the delineation of <em>B. henselae</em> isolates into two genotypes based on the 16S rRNA sequence is not congruent with phylogenetic classifications using other genetic loci such as <em>gro</em>EL, <em>fts</em>Z and <em>rpo</em>B <a href="#pone.0001346-Zeaiter1">[18]</a>, <a href="#pone.0001346-Ehrenborg1">[21]</a>. In 2003, Iredell et al. developed a MLST scheme for <em>B. henselae</em> based on comparison of the nucleotide sequences of nine genetic loci <a href="#pone.0001346-Iredell1">[22]</a>. Analysis of 37 feline and human <em>B. henselae</em> isolates from Australia by MLST revealed a considerable genetic diversity among feline isolates, while human isolates were more homogeneous <a href="#pone.0001346-Iredell1">[22]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>We have recently validated the use of MLST for the definition of <em>B. henselae</em> strains by comparison with pulsed-field gel electrophoresis (PFGE) analysis <a href="#pone.0001346-Arvand3">[23]</a>. MLST is a pangenomic approach that identifies very closely related bacterial isolates and allows the reconstruction of micro-evolutionary events <a href="#pone.0001346-Feil1">[24]</a>. In the present study, MLST was applied to a larger collection of feline and human <em>B. henselae</em> isolates from Europe, Australia and the USA in order to further investigate the association between ST, host species and geographical distribution. The clonal and phylogenetic relationships among the isolates was analysed using three different procedures: i) eBURST was used to define clonal lineages and reconstruct very recent events within each lineage <a href="#pone.0001346-Feil2">[25]</a>, ii) a neighbour-joining tree was reconstructed based on concatenated MLST alleles, and iii) splits decomposition was used to detect inconsistent phylogenetic signals in the data indicative of recombination <a href="#pone.0001346-Huson1">[26]</a>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Assignment of the <em>B. henselae</em> isolates to STs</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>From the 184 isolates studied, 182 isolates were assigned to 14 different STs. Two isolates could not be assigned to a ST because they contained different 16S RNA alleles; they will be described elsewhere. A new allele for <em>rpoB</em> was obtained from a feline isolate from Israel and designated as <em>rpoB</em>-allele 4. The nucleotide sequence of this allele has been deposited in Genbank (Accession No. EU289215) and the MLST web site (in preparation). Six STs were encountered for the first time in this study and designated as ST9 to ST14 in order of detection. The allelic profile, frequency of isolation and geographical distribution of STs, as well as a reference strain for each ST have been presented in <a href="#pone-0001346-t001">Table 1</a>. The average sequence divergence between all pairwise allelic comparisons was 0.5, 0.6, 0.5, 0.4, 0.3, 0.2, 1 and 0.3 percent for the <em>rrs</em>, <em>bat</em>R, <em>fts</em>Z, <em>glt</em>A, <em>gro</em>EL, <em>nlp</em>D, <em>rib</em>C, and <em>rpo</em>B locus, respectively.</p>
<div class="figure" id="pone-0001346-t001"><div class="img"><a name="pone-0001346-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001346.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001346.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Allelic profile, frequency of isolation, and geographic distribution of STs.</span></strong></p><span>doi:10.1371/journal.pone.0001346.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The Urlly8 (Marseille) isolate displayed the <em>rpo</em>B-allele 2 and was assigned to ST6 in the present study (<a href="#pone-0001346-t001">Table 1</a>). These results are in accordance with data by Renesto et al. <a href="#pone.0001346-Renesto1">[27]</a>, Iredell et al. <a href="#pone.0001346-Iredell1">[22]</a>, and our previous results <a href="#pone.0001346-Arvand3">[23]</a>, but differ from the data presented by Lindroos et al. <a href="#pone.0001346-Lindroos1">[28]</a>, who found an extra single nucleotide polymorphism in the <em>rpo</em>B allele of the Urlly8 isolate and assigned it to a new ST. The CA-1 isolate displayed the <em>rrs</em>- and <em>gro</em>EL-alleles 2 in our study, corresponding to ST5. This is in accordance with previous results of different groups <a href="#pone.0001346-Zeaiter1">[18]</a>, <a href="#pone.0001346-Ehrenborg1">[21]</a>, <a href="#pone.0001346-LaScola1">[29]</a>, but again differs from the data presented by Lindroos et al. <a href="#pone.0001346-Lindroos1">[28]</a>, who obtained the <em>rrs</em>- and <em>gro</em>EL-alleles 1 for the CA-1 isolate and assigned it to ST1.</p>


<h4>Geographical distribution and frequency of STs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>ST1, ST5, ST6, and ST7 were the most common STs, representing 23.6%, 20.9%, 15.4%, and 23.6% of the isolates, respectively. ST1, ST5 and ST6 were isolated in Europe, America and Australia, while ST7 was only distributed in Europe. ST4 and ST9 were distributed in two continents only, being absent from the USA and Australian samples respectively. The less common STs, noted in 1-7 isolates, were found in one continent only. <a href="#pone-0001346-g001">Figure 1</a> shows the distribution of STs in different continents and among European countries that were represented by more than 10 isolates. The distribution of the major STs in different continents was found to be significantly non-random by chi-square test for STs 1 and 7 (p&lt;0.00001 each), and ST6 (p = 0.039). In contrast, ST5 was evenly distributed among the three continents (p&gt;0.1). The distribution of STs varied also considerably between different countries within Europe. The dominance of ST1 in Italy and Israel, but near absence in France, UK and Germany was particularly striking. To examine this further, we divided the European isolates in two subgroups: i) Mediterranean isolates including all isolates from Israel, Italy, and the Urlly8 (Marseille) isolate, and ii) North-western European isolates (NW-Europe) including all other European isolates. The relative frequency of major STs in different geographic regions was again evaluated by chi-square test (<a href="#pone-0001346-t002">Table 2</a>). This revealed a highly non-random distribution for STs 1 and 7 in Europe, these being over-represented in the Mediterranean and NW Europe, respectively.</p>
<div class="figure" id="pone-0001346-g001"><div class="img"><a name="pone-0001346-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001346.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001346.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Geographical distribution of <em>B. henselae</em> STs in different continents.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The lower panel shows the ST distribution in European countries that were represented by at least 10 isolates.</p>
<span>doi:10.1371/journal.pone.0001346.g001</span></div><div class="figure" id="pone-0001346-t002"><div class="img"><a name="pone-0001346-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001346.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001346.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Correlation between ST and geographic origin among 182 <em>B. henselae</em> isolates analysed.</span></strong></p><span>doi:10.1371/journal.pone.0001346.t002</span></div>

<h4>Relationship between ST and host species</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The feline isolates (n = 158) were assigned to 13 STs. The human-derived isolates were assigned to 4 STs, including ST1 (16 isolates), ST5 (5 isolates), ST6 (2 isolates), and ST2 (1 isolate). Interestingly, ST7 was not encountered among the human isolates, although it was displayed by 43/121 (35.5%) of the feline isolates from Europe. <a href="#pone-0001346-g002">Figure 2</a> shows the frequency of feline and human isolates within each ST, along with geographical source. The relative frequency of feline and human isolates within each ST was compared with their frequency in the whole panel by using Fisher's exact test. Human-derived isolates were over proportionally allocated to ST1 and under proportionally associated with ST7 (p≤0.001). The other major STs did not show a disproportional distribution pattern (<a href="#pone-0001346-t003">Table 3</a>).</p>
<div class="figure" id="pone-0001346-g002"><div class="img"><a name="pone-0001346-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001346.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001346.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Frequency of feline and human <em>B. henselae</em> isolates within each ST in correlation with the geographic origin of the isolates.</span></strong></p><span>doi:10.1371/journal.pone.0001346.g002</span></div><div class="figure" id="pone-0001346-t003"><div class="img"><a name="pone-0001346-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001346.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001346.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Correlation between ST and host species in 182 <em>B. henselae</em> isolates analysed.</span></strong></p><span>doi:10.1371/journal.pone.0001346.t003</span></div>

<h4>Evaluation of <em>eno</em> for the MLST scheme</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The original MLST scheme proposed by Iredell et al. <a href="#pone.0001346-Iredell1">[22]</a> contained <em>eno</em>, however, no allelic variability was found in the latter study or in subsequent studies <a href="#pone.0001346-Lindroos1">[28]</a>. We therefore decided to determine the <em>eno</em> sequence in a selected panel of isolates to evaluate its appropriateness for the <em>B. henselae</em> MLST scheme. Fifty isolates that represented every ST and corresponded with their frequency of isolation were analysed. We did not find any allelic diversity among these isolates. Since the selected 50 isolates represent the most heterogeneous <em>B. henselae</em> strain collection analysed so far, we conclude that this region of the <em>eno</em> gene is not an appropriate target for the <em>B. henselae</em> MLST scheme.</p>


<h4>Phylogenetic analysis</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The relationships between the STs was first examined using eBURST (<a href="#pone-0001346-g003">Figure 3</a>), which uses allele profiles rather than sequences and does not attempt to reconstruct the relationships between the different clonal lineages. The majority of the isolates (107/182; 58.8%) corresponded to nine STs, which formed a single clonal complex, Group 1, with ST5 as primary founder. A clonal complex contains STs that have 7 out of 8 alleles in common and a primary founder is the ST with the largest number of single locus variants (SLVs) within a clonal complex. ST5 corresponded to 38/182 (20.8%) of the isolates, which is consistent with its positioning as the founder of STs 9, 4, 12 and 2. The links connecting the other STs in this complex, STs 1, 3, 8 and 14, are less certain, and may have been obscured by recombination. Two doublets were also identified (ST6-ST10, Group 2, and ST13-ST11, Group 3), whilst ST7 differed in two or more genes from every other isolate and was therefore assigned as a singleton.</p>
<div class="figure" id="pone-0001346-g003"><div class="img"><a name="pone-0001346-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001346.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001346.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Phylogenetic relationship between different <em>B. henselae</em> STs as determined by eBURST.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>A clonal complex contains STs that have 7 out of 8 alleles in common. ST7 is assigned as a singleton since it differed in 3–7 alleles from all other STs. The size of the circles relates to the frequency of the corresponding ST, and illustrates that the assigned primary founder of the major clonal complex (ST5) is a common clone.</p>
<span>doi:10.1371/journal.pone.0001346.g003</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Phylogenetic analysis of the data was carried out by reconstructing a neighbour-joining tree based on concatenated sequences as implemented in MEGA4 <a href="#pone.0001346-Tamura1">[30]</a> (<a href="#pone-0001346-g004">Figure 4</a>). Although the bootstrap support on this tree was generally poor, owing to a paucity of informative sites and possibly a history of recombination, 75% of 1,000 bootstrap trees supported the delineation of Group 1 from the other STs. This is consistent with the hypothesis that it represents a real division within the <em>B. henselae</em> population.</p>
<div class="figure" id="pone-0001346-g004"><div class="img"><a name="pone-0001346-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001346.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001346.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Neighbour-joining tree of the concatenated sequences of <em>B. henselae</em> STs as reconstructed by MEGA4.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>1,000 bootstrap replicates were used to examine the confidence in the tree. The only node to score above 60 is the one leading to Group 1 (75%, as indicated), indicating that the delineation of Group 1 represents a real division in the <em>B. henselae</em> population.</p>
<span>doi:10.1371/journal.pone.0001346.g004</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Iredell et al <a href="#pone.0001346-Iredell1">[22]</a> noted evidence for recombination from their MLST data, and to explore this issue further we used splits decomposition analysis as implemented in Splitstree4 <a href="#pone.0001346-Huson1">[26]</a>. The approach examines the degree to which the data correspond to a bifurcating tree, which would indicate limited recombination, or alternatively a network structure, which would be consistent with more frequent recombination. <a href="#pone-0001346-g005">Figure 5</a> shows that the approach resulted in extensive reticulation between the STs, which is consistent with a history of recombination. Furthermore, the phi test, as implemented in Splitstree4, revealed significant evidence for recombination (P&lt;0.004). Splits decomposition analysis also confirmed the delineation of Group 1, and placed ST7 as a distinct genotype more closely related to Groups 2 and 3 than to Group 1.</p>
<div class="figure" id="pone-0001346-g005"><div class="img"><a name="pone-0001346-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001346.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001346.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Splits decomposition was used to detect evidence for a past history of recombination in the sequences.</span></strong></p><a id="article1.body1.sec2.sec5.fig3.caption1.p1" name="article1.body1.sec2.sec5.fig3.caption1.p1"></a><p>The extensive reticulation suggests that recombination has occurred relatively frequently. However, Group 1 remains distinct (as indicated by the filled oval).</p>
<span>doi:10.1371/journal.pone.0001346.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this study, a collection of 182 <em>B. henselae</em> isolates from 12 countries and three continents was analysed by MLST to elucidate i) the relationship between ST and host species, ii) the geographical distribution of STs, and iii) the phylogenetic relationship among different STs. To our knowledge, this is the largest <em>B. henselae</em> collection that has been analysed by MLST or other molecular typing techniques hitherto. We have tried to minimise sampling artefacts by making every effort to include human isolates from all geographic regions that were represented by feline isolates. However, human isolates were not available from some regions or were outnumbered by feline isolates in other areas. In addition, as the isolates were collected by different investigators in different settings and during a long period of time (approximately 15 years), we can not exclude temporal or seasonal variations or a bias caused by the population (stray versus pet) or breed of the cats examined. Therefore, the panel is still not truly representative of the natural population of <em>B. henselae</em>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Fourteen STs were encountered among 182 <em>B. henselae</em> isolates. ST1, ST5, ST6 and ST7 represented major STs and accounted for 83.5% of the isolates. The geographical distribution of STs was not homogeneous. ST1, ST5 and ST6 were found in three continents, suggesting that they may be distributed world wide. In contrast, ST7 was detected only in Europe, suggesting that its distribution may be restricted to Europe. The differences in distribution of STs 1, 6 and 7 on three continents were statistically significant. The distribution of STs varied also between different European countries. ST7 was more prevalent in North and West Europe (UK, Sweden, Denmark, Germany, the Netherlands, France), whereas ST1 was more frequently obtained from the Mediterranean region (Italy, Israel).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We found a significant correlation between distinct STs and human disease. ST1 was statistically significantly associated with human infection, suggesting that it represents a hypervirulent strain. This finding is in accordance with data by Iredell et al. <a href="#pone.0001346-Iredell1">[22]</a>, who found a significant association of ST1 with CSD in Australia. They contradict the results by Lindroos et al. <a href="#pone.0001346-Lindroos1">[28]</a>, who did not find a disproportional association between a distinct ST and human-derived isolates. This discrepancy might be due to differences in size and composition of the panels of isolates. In the latter study, the panel was smaller (n = 38), composed to 60.5% of ST1, and did not contain matched human and feline isolates from the same geographic regions.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>ST7 was underrepresented among the human isolates in our study, suggesting that ST7 may be less virulent for humans. However, we can not completely rule out the possibility that the absence of ST7 among the human isolates could be due to a bias in composition of our panel, which contained more feline than human isolates from countries with a higher prevalence of ST7 (e.g. Germany, UK, and France). Further studies with more human-derived isolates from Europe would help to evaluate this hypothesis.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>eBURST and phylogenetic analyses were broadly consistent and revealed a major division within the population of <em>B. henselae</em>. Of the four predominant genotypes, ST1 and ST5 are related and belong to the major clade, Group 1. ST6 belongs to the minor clade, Group 2, and ST7 is a distinct genotype, probably more closely related to Groups 2 and 3 than to Group 1. Our analysis also supports previous studies which have suggested a history of recombination between the isolates. This is further supported by the “straggly” shape of the major clonal complex as revealed by eBURST. Recent simulation studies have shown that such a structure is indicative of frequent recombination <a href="#pone.0001346-Turner1">[31]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In summary, our data indicate that different STs of <em>B. henselae</em> may vary with regard to virulence for humans. It can be hypothesized that ST1 might possess additional virulence factors, which could encode for a more effective transmission from cats to humans, or a better survival of the pathogen in the human host. It can also be speculated that ST7 may lack one or more virulence determinants, and lower transmission potential may possibly account for the restriction of this genotype to Europe. Future studies using comparative genomic or proteomic approaches could help to identify and characterize these factors. The MLST approach has been previously used for tracking hypervirulent or antibiotic resistant lineages in other bacterial pathogens, e.g. <em>Neisseria meningitidis</em>, <em>Streptococcus pneumoniae</em> or <em>Staphylococcus aureus</em> <a href="#pone.0001346-Feil1">[24]</a>, <a href="#pone.0001346-Feil2">[25]</a>. MLST data are unambiguous and can be easily transferred electronically between laboratories. Furthermore, MLST can be applied directly to clinical specimens and is therefore not strictly dependent on culture. It can be expected that more MLST data will become available in future, and the establishment of a <em>B. henselae</em> site on <a href="http://www.mlst.net">www.mlst.net</a> should greatly facilitate this.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Bartonella isolates</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>One-hundred and eighty four <em>B. henselae</em> isolates collected by different investigators in several European countries, Australia, and the USA were analysed. One-hundred and sixty isolates were isolated from feline blood, and 24 isolates were obtained from human tissue specimens, including lymph node, cutaneous BA lesion, and blood. <a href="#pone-0001346-t004">Table 4</a> summarises the epidemiological data of the isolates studied. Bacteria were stored at −20°C or −80°C until use. The isolates were grown on Columbia blood agar with 5% sheep blood (Becton Dickenson) at 37°C in 5% CO<sub>2</sub> for 7–14 d, and passaged once on agar prior to isolation of bacterial DNA.</p>
<div class="figure" id="pone-0001346-t004"><div class="img"><a name="pone-0001346-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001346" data-uri="info:doi/10.1371/journal.pone.0001346.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001346.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001346.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001346.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001346.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001346.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Geographic origin, host species and clinical source of <em>B. henselae</em> isolates.</span></strong></p><span>doi:10.1371/journal.pone.0001346.t004</span></div>

<h4>MLST</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Nucleotide sequence data were collected from all <em>B. henselae</em> isolates for approximately 320–500 bp fragments of eight genetic loci (16S rRNA [<em>rrs</em>], <em>bat</em>R, <em>glt</em>A, <em>gro</em>EL, <em>fts</em>Z, <em>nlp</em>D, <em>rib</em>C, and <em>rpo</em>B) as described previously <a href="#pone.0001346-Iredell1">[22]</a>, <a href="#pone.0001346-Arvand3">[23]</a>. In addition, the partial sequence of <em>eno</em> was determined for 50 isolates <a href="#pone.0001346-Iredell1">[22]</a>. All sequences were determined for both strands and the results were confirmed by repeats when necessary. The reliability of the sequence data was controlled by subjecting 20 randomly selected isolates in a blinded manner as “quality control strains” to MLST analysis. The results of the quality control strains were compared with the data obtained from the “original isolates”. The MLST results were 100% consistent for each pair of quality control strain-original isolate.</p>


<h4>Analysis of MLST Data</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The nucleotide sequences were analysed with the DNASTAR Lasergene software package 7 (DNASTAR, Madison, USA). Alleles and STs were assigned in accordance with the published data <a href="#pone.0001346-Iredell1">[22]</a>, <a href="#pone.0001346-Arvand3">[23]</a>. New alleles were confirmed by repeats and the sequence was deposited in GenBank (see below). New allelic combinations that were encountered for the first time in this study were assigned to new STs in order of detection.</p>


<h4>Phylogenetic analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The definition of clonal complexes and the examination of relationships between STs within clonal complexes were carried out by using eBURST (<a href="http://eburst.mlst.net">http://eburst.mlst.net</a>). A neighbour-joining tree was reconstructed from the concatenated MLST alleles using the kimura-2-parameter distance measures as implemented in MEGA4 <a href="#pone.0001346-Tamura1">[30]</a>. Splits decomposition analysis and the phi test were carried out using the default settings in Splitstree4 <a href="#pone.0001346-Huson1">[26]</a>.</p>


<h4>Statistical analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Chi square test was used to compare the geographical distribution patterns of major STs. Two-tailed Fisher's exact test was used to compare the frequency of feline and human-derived isolates within a ST with the frequency within the whole panel. <em>P</em> values of &lt;0.05 were considered significant.</p>


<h4>Nucleotide sequence accession number</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>A new <em>rpo</em>B allele was encountered from the isolate Is-959 and was designated as <em>rpo</em>B-allele 4. This allele contains a single nucleotide variation (G instead of A) at position 711758 of the <em>B. henselae</em> Houston-1 chromosome (Accession No. BX897699.1). The <em>rpo</em>B-allele 4 sequence has been deposited in GenBank under the Accession No. EU289215. The data were also deposited at <a href="http://www.mlst.net">http://www.mlst.net</a> (in preparation).</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Drs. Burt Anderson, Anneke Bergmans, Richard Birtles, Massimo Fabbi, Pierre-Edouard Fournier, John Iredell, Lynn Guptill, Eva Hjelm, Oto Melter, Yves Piémont, Anna Sander, and Dietrich Schimmel for providing <em>B. henselae</em> isolates.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MA. Performed the experiments: JV. Analyzed the data: EF MA JV. Contributed reagents/materials/analysis tools: EF MA MG HB. Wrote the paper: EF MA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001346-Margileth1" id="pone.0001346-Margileth1"></a>Margileth AM (1993) Cat scratch disease. Adv Pediatr Infect Dis  8: 1–21.  <ul class="find" data-citedArticleID="1033084" data-doi="10.1007/s11908-000-0026-8"><li><a href="http://dx.doi.org/10.1007/s11908-000-0026-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cat+scratch+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cat+scratch+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001346-Koehler1" id="pone.0001346-Koehler1"></a>Koehler JE, Quinn FD, Berger TG, LeBoit PE, Tappero JW (1992) Isolation of <em>Rochalimaea species</em> from cutaneous and osseous lesions of bacillary angiomatosis. N Engl J Med  327: 1625–1631.  <ul class="find" data-citedArticleID="1033078" data-doi="10.1056/nejm199212033272303"><li><a href="http://dx.doi.org/10.1056/nejm199212033272303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+Rochalimaea+species+from+cutaneous+and+osseous+lesions+of+bacillary+angiomatosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+Rochalimaea+species+from+cutaneous+and+osseous+lesions+of+bacillary+angiomatosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001346-Guptill1" id="pone.0001346-Guptill1"></a>Guptill L, Slater L, Wu CC, Lin TL, Glickman LT, et al.  (1997) Experimental infection of young specific pathogen-free cats with <em>Bartonella henselae</em>. J Infect Dis  176: 206–216.  <ul class="find" data-citedArticleID="1033066" data-doi="10.1086/514026"><li><a href="http://dx.doi.org/10.1086/514026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+infection+of+young+specific+pathogen-free+cats+with+Bartonella+henselae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+infection+of+young+specific+pathogen-free+cats+with+Bartonella+henselae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001346-Sander1" id="pone.0001346-Sander1"></a>Sander A, Posselt M, Bohm N, Ruess M, Altwegg M (1999) Detection of <em>Bartonella henselae</em> DNA by two different PCR assays and determination of the genotypes of strains involved in histologically defined cat scratch disease. J Clin Microbiol  37: 993–997.  <ul class="find" data-citedArticleID="1033092"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Detection+of+Bartonella+henselae+DNA+by+two+different+PCR+assays+and+determination+of+the+genotypes+of+strains+involved+in+histologically+defined+cat+scratch+disease.&amp;auth=&amp;atitle=Detection+of+Bartonella+henselae+DNA+by+two+different+PCR+assays+and+determination+of+the+genotypes+of+strains+involved+in+histologically+defined+cat+scratch+disease." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+Bartonella+henselae+DNA+by+two+different+PCR+assays+and+determination+of+the+genotypes+of+strains+involved+in+histologically+defined+cat+scratch+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+Bartonella+henselae+DNA+by+two+different+PCR+assays+and+determination+of+the+genotypes+of+strains+involved+in+histologically+defined+cat+scratch+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001346-Avidor1" id="pone.0001346-Avidor1"></a>Avidor B, Varon M, Marmor S, Lifschitz-Mercer B, Kletter Y, et al.  (2001) DNA amplification for the diagnosis of cat-scratch disease in small-quantity clinical specimens. Am J Clin Pathol  115: 900–909.  <ul class="find" data-citedArticleID="1033040" data-doi="10.1309/y468-82g5-achw-yrmv"><li><a href="http://dx.doi.org/10.1309/y468-82g5-achw-yrmv" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+amplification+for+the+diagnosis+of+cat-scratch+disease+in+small-quantity+clinical+specimens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+amplification+for+the+diagnosis+of+cat-scratch+disease+in+small-quantity+clinical+specimens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001346-Arvand1" id="pone.0001346-Arvand1"></a>Arvand M, Schäd SG (2006) Isolation of <em>Bartonella henselae</em> DNA from the peripheral blood of a patient with cat scratch disease up to 4 months after the cat scratch injury. J Clin Microbiol  44: 2288–2290.  <ul class="find" data-citedArticleID="1033032" data-doi="10.1128/jcm.00239-06"><li><a href="http://dx.doi.org/10.1128/jcm.00239-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+Bartonella+henselae+DNA+from+the+peripheral+blood+of+a+patient+with+cat+scratch+disease+up+to+4+months+after+the+cat+scratch+injury." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+Bartonella+henselae+DNA+from+the+peripheral+blood+of+a+patient+with+cat+scratch+disease+up+to+4+months+after+the+cat+scratch+injury.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001346-Slater1" id="pone.0001346-Slater1"></a>Slater LN, Welch DF, Hensel D, Coody DW (1990) A newly recognized fastidious gram-negative pathogen as a cause of fever and bacteremia. N Engl J Med  323: 1587–1593.  <ul class="find" data-citedArticleID="1033098" data-doi="10.1056/nejm199012063232303"><li><a href="http://dx.doi.org/10.1056/nejm199012063232303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+newly+recognized+fastidious+gram-negative+pathogen+as+a+cause+of+fever+and+bacteremia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+newly+recognized+fastidious+gram-negative+pathogen+as+a+cause+of+fever+and+bacteremia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001346-Dolan1" id="pone.0001346-Dolan1"></a>Dolan MJ, Wong MT, Regnery RL, Jorgensen JH, Garcia M, et al.  (1993) Syndrome of <em>Rochalimaea henselae</em> adenitis suggesting cat scratch disease. Ann Intern Med  118: 331–336.  <ul class="find" data-citedArticleID="1033054" data-doi="10.7326/0003-4819-118-5-199303010-00002"><li><a href="http://dx.doi.org/10.7326/0003-4819-118-5-199303010-00002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Syndrome+of+Rochalimaea+henselae+adenitis+suggesting+cat+scratch+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Syndrome+of+Rochalimaea+henselae+adenitis+suggesting+cat+scratch+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001346-Dillon1" id="pone.0001346-Dillon1"></a>Dillon B, Valenzuela J, Don R, Blanckenberg D, Wigney DI, et al.  (2002) Limited Diversity among Human Isolates of <em>Bartonella henselae</em>. J Clin Microbiol  40: 4691–4699.  <ul class="find" data-citedArticleID="1033052" data-doi="10.1128/jcm.40.12.4691-4699.2002"><li><a href="http://dx.doi.org/10.1128/jcm.40.12.4691-4699.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Limited+Diversity+among+Human+Isolates+of+Bartonella+henselae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Limited+Diversity+among+Human+Isolates+of+Bartonella+henselae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001346-Drancourt1" id="pone.0001346-Drancourt1"></a>Drancourt M, Birtles R, Chaumentin G, Vandenesch F, Etienne J, et al.  (1996) New serotype of <em>Bartonella henselae</em> in endocarditis and cat-scratch disease. Lancet  347: 441–443.  <ul class="find" data-citedArticleID="1033056" data-doi="10.1016/s0140-6736(96)90012-4"><li><a href="http://dx.doi.org/10.1016/s0140-6736(96)90012-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+serotype+of+Bartonella+henselae+in+endocarditis+and+cat-scratch+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+serotype+of+Bartonella+henselae+in+endocarditis+and+cat-scratch+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001346-Chomel1" id="pone.0001346-Chomel1"></a>Chomel BB, Abbott RC, Kasten RW, Floyd-Hawkins KA, Kass PH, et al.  (1995) <em>Bartonella henselae</em> prevalence in domestic cats in California: risk factors and association between bacteremia and antibody titers. J Clin Microbiol  33: 2445–2450.  <ul class="find" data-citedArticleID="1033050"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bartonella+henselae+prevalence+in+domestic+cats+in+California%3A+risk+factors+and+association+between+bacteremia+and+antibody+titers.&amp;auth=&amp;atitle=Bartonella+henselae+prevalence+in+domestic+cats+in+California%3A+risk+factors+and+association+between+bacteremia+and+antibody+titers." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bartonella+henselae+prevalence+in+domestic+cats+in+California%3A+risk+factors+and+association+between+bacteremia+and+antibody+titers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bartonella+henselae+prevalence+in+domestic+cats+in+California%3A+risk+factors+and+association+between+bacteremia+and+antibody+titers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001346-Heller1" id="pone.0001346-Heller1"></a>Heller R, Artois M, Xemar V, De Briel D, Gehin H, et al.  (1997) Prevalence of <em>Bartonella henselae</em> and <em>Bartonella clarridgeiae</em> in stray cats. J Clin Microbiol  35: 1327–1331.  <ul class="find" data-citedArticleID="1033072"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Prevalence+of+Bartonella+henselae+and+Bartonella+clarridgeiae+in+stray+cats.&amp;auth=&amp;atitle=Prevalence+of+Bartonella+henselae+and+Bartonella+clarridgeiae+in+stray+cats." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalence+of+Bartonella+henselae+and+Bartonella+clarridgeiae+in+stray+cats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalence+of+Bartonella+henselae+and+Bartonella+clarridgeiae+in+stray+cats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001346-Sander2" id="pone.0001346-Sander2"></a>Sander A, Buhler C, Pelz K, von Cramm E, Bredt W (1997) Detection and identification of two <em>Bartonella henselae</em> variants in domestic cats in Germany. J Clin Microbiol  35: 584–587.  <ul class="find" data-citedArticleID="1033094"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Detection+and+identification+of+two+Bartonella+henselae+variants+in+domestic+cats+in+Germany.&amp;auth=&amp;atitle=Detection+and+identification+of+two+Bartonella+henselae+variants+in+domestic+cats+in+Germany." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+and+identification+of+two+Bartonella+henselae+variants+in+domestic+cats+in+Germany." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+and+identification+of+two+Bartonella+henselae+variants+in+domestic+cats+in+Germany.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001346-Bergmans1" id="pone.0001346-Bergmans1"></a>Bergmans AM, de Jong CM, van Amerongen G, Schot CS, Schouls LM (1997) Prevalence of <em>Bartonella species</em> in domestic cats in The Netherlands. J Clin Microbiol  35: 2256–2261.  <ul class="find" data-citedArticleID="1033042"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Prevalence+of+Bartonella+species+in+domestic+cats+in+The+Netherlands.&amp;auth=&amp;atitle=Prevalence+of+Bartonella+species+in+domestic+cats+in+The+Netherlands." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalence+of+Bartonella+species+in+domestic+cats+in+The+Netherlands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalence+of+Bartonella+species+in+domestic+cats+in+The+Netherlands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001346-Branley1" id="pone.0001346-Branley1"></a>Branley J, Wolfson C, Waters P, Gottlieb T, Bradbury R (1996) Prevalence of <em>Bartonella henselae</em> bacteremia, the causative agent of cat scratch disease, in an Australian cat population. Pathology  28: 262–265.  <ul class="find" data-citedArticleID="1033048" data-doi="10.1080/00313029600169124"><li><a href="http://dx.doi.org/10.1080/00313029600169124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalence+of+Bartonella+henselae+bacteremia%2C+the+causative+agent+of+cat+scratch+disease%2C+in+an+Australian+cat+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalence+of+Bartonella+henselae+bacteremia%2C+the+causative+agent+of+cat+scratch+disease%2C+in+an+Australian+cat+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001346-Sander3" id="pone.0001346-Sander3"></a>Sander A, Ruess M, Bereswill S, Schuppler M, Steinbrueckner B (1998) Comparison of different DNA fingerprinting techniques for molecular typing of <em>Bartonella henselae</em> isolates. J Clin Microbiol  36: 2973–2981.  <ul class="find" data-citedArticleID="1033096"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparison+of+different+DNA+fingerprinting+techniques+for+molecular+typing+of+Bartonella+henselae+isolates.&amp;auth=&amp;atitle=Comparison+of+different+DNA+fingerprinting+techniques+for+molecular+typing+of+Bartonella+henselae+isolates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+different+DNA+fingerprinting+techniques+for+molecular+typing+of+Bartonella+henselae+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+different+DNA+fingerprinting+techniques+for+molecular+typing+of+Bartonella+henselae+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001346-RodriguezBarradas1" id="pone.0001346-RodriguezBarradas1"></a>Rodriguez-Barradas MC, Hamill RJ, Houston ED, Georghiou PR, Clarridge JE, et al.  (1995) Genomic fingerprinting of <em>Bartonella species</em> by repetitive element PCR for distinguishing species and isolates. J Clin Microbiol  33: 1089–1093.  <ul class="find" data-citedArticleID="1033090"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genomic+fingerprinting+of+Bartonella+species+by+repetitive+element+PCR+for+distinguishing+species+and+isolates.&amp;auth=&amp;atitle=Genomic+fingerprinting+of+Bartonella+species+by+repetitive+element+PCR+for+distinguishing+species+and+isolates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+fingerprinting+of+Bartonella+species+by+repetitive+element+PCR+for+distinguishing+species+and+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+fingerprinting+of+Bartonella+species+by+repetitive+element+PCR+for+distinguishing+species+and+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001346-Zeaiter1" id="pone.0001346-Zeaiter1"></a>Zeaiter Z, Fournier PE, Raoult D (2002) Genomic variation of <em>Bartonella henselae</em> strains detected in lymph nodes of patients with cat scratch disease. J Clin Microbiol  40: 1023–1030.  <ul class="find" data-citedArticleID="1033104" data-doi="10.1128/jcm.40.3.1023-1030.2002"><li><a href="http://dx.doi.org/10.1128/jcm.40.3.1023-1030.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+variation+of+Bartonella+henselae+strains+detected+in+lymph+nodes+of+patients+with+cat+scratch+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+variation+of+Bartonella+henselae+strains+detected+in+lymph+nodes+of+patients+with+cat+scratch+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001346-Bergmans2" id="pone.0001346-Bergmans2"></a>Bergmans AM, Schellekens JF, van Embden JD, Schouls LM (1996) Predominance of two <em>Bartonella henselae</em> variants among cat-scratch disease patients in the Netherlands. J Clin Microbiol  34: 254–260.  <ul class="find" data-citedArticleID="1033044"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Predominance+of+two+Bartonella+henselae+variants+among+cat-scratch+disease+patients+in+the+Netherlands.&amp;auth=&amp;atitle=Predominance+of+two+Bartonella+henselae+variants+among+cat-scratch+disease+patients+in+the+Netherlands." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predominance+of+two+Bartonella+henselae+variants+among+cat-scratch+disease+patients+in+the+Netherlands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predominance+of+two+Bartonella+henselae+variants+among+cat-scratch+disease+patients+in+the+Netherlands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001346-Arvand2" id="pone.0001346-Arvand2"></a>Arvand M, Klose AJ, Schwartz-Porsche D, Hahn H, Wendt C (2001) Genetic variability and prevalence of <em>Bartonella henselae</em> in cats in Berlin, Germany, and analysis of its genetic relatedness to a strain from Berlin that is pathogenic for humans. J Clin Microbiol  39: 743–746.  <ul class="find" data-citedArticleID="1033034" data-doi="10.1128/jcm.39.2.743-746.2001"><li><a href="http://dx.doi.org/10.1128/jcm.39.2.743-746.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+variability+and+prevalence+of+Bartonella+henselae+in+cats+in+Berlin%2C+Germany%2C+and+analysis+of+its+genetic+relatedness+to+a+strain+from+Berlin+that+is+pathogenic+for+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+variability+and+prevalence+of+Bartonella+henselae+in+cats+in+Berlin%2C+Germany%2C+and+analysis+of+its+genetic+relatedness+to+a+strain+from+Berlin+that+is+pathogenic+for+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001346-Ehrenborg1" id="pone.0001346-Ehrenborg1"></a>Ehrenborg C, Wesslen L, Jakobson A, Friman G, Holmberg M (2000) Sequence variation in the ftsZ gene of <em>Bartonella henselae</em> isolates and clinical samples. J Clin Microbiol  38: 682–687.  <ul class="find" data-citedArticleID="1033058"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sequence+variation+in+the+ftsZ+gene+of+Bartonella+henselae+isolates+and+clinical+samples.&amp;auth=&amp;atitle=Sequence+variation+in+the+ftsZ+gene+of+Bartonella+henselae+isolates+and+clinical+samples." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+variation+in+the+ftsZ+gene+of+Bartonella+henselae+isolates+and+clinical+samples." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+variation+in+the+ftsZ+gene+of+Bartonella+henselae+isolates+and+clinical+samples.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001346-Iredell1" id="pone.0001346-Iredell1"></a>Iredell J, Blanckenberg D, Arvand M, Grauling S, Feil EJ, et al.  (2003) Characterization of the natural population of <em>Bartonella henselae</em> by multilocus sequence typing. J Clin Microbiol  41: 5071–5079.  <ul class="find" data-citedArticleID="1033076" data-doi="10.1128/jcm.41.11.5071-5079.2003"><li><a href="http://dx.doi.org/10.1128/jcm.41.11.5071-5079.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+natural+population+of+Bartonella+henselae+by+multilocus+sequence+typing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+natural+population+of+Bartonella+henselae+by+multilocus+sequence+typing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001346-Arvand3" id="pone.0001346-Arvand3"></a>Arvand M, Viezens J (2007) Evaluation of pulsed-field gel electrophoresis and multi-locus sequence typing for the analysis of clonal relatedness among <em>Bartonella henselae</em> isolates. Int J Med Microbiol  297: 255–262.  Epub 2007 Mar 2030. <ul class="find" data-citedArticleID="1033036" data-doi="10.1016/j.ijmm.2007.02.001"><li><a href="http://dx.doi.org/10.1016/j.ijmm.2007.02.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+pulsed-field+gel+electrophoresis+and+multi-locus+sequence+typing+for+the+analysis+of+clonal+relatedness+among+Bartonella+henselae+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+pulsed-field+gel+electrophoresis+and+multi-locus+sequence+typing+for+the+analysis+of+clonal+relatedness+among+Bartonella+henselae+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001346-Feil1" id="pone.0001346-Feil1"></a>Feil EJ, Enright MC (2004) Analyses of clonality and the evolution of bacterial pathogens. Curr Opin Microbiol  7: 308–313.  <ul class="find" data-citedArticleID="1033062" data-doi="10.1016/j.mib.2004.04.002"><li><a href="http://dx.doi.org/10.1016/j.mib.2004.04.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analyses+of+clonality+and+the+evolution+of+bacterial+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analyses+of+clonality+and+the+evolution+of+bacterial+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001346-Feil2" id="pone.0001346-Feil2"></a>Feil EJ, Li BC, Aanensen DM, Hanage WP, Spratt BG (2004) eBURST: inferring patterns of evolutionary descent among clusters of related bacterial genotypes from multilocus sequence typing data. J Bacteriol  186: 1518–1530.  <ul class="find" data-citedArticleID="1033064" data-doi="10.1128/jb.186.5.1518-1530.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.5.1518-1530.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=eBURST%3A+inferring+patterns+of+evolutionary+descent+among+clusters+of+related+bacterial+genotypes+from+multilocus+sequence+typing+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22eBURST%3A+inferring+patterns+of+evolutionary+descent+among+clusters+of+related+bacterial+genotypes+from+multilocus+sequence+typing+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001346-Huson1" id="pone.0001346-Huson1"></a>Huson DH, Bryant D (2006) Application of phylogenetic networks in evolutionary studies. Mol Biol Evol  23: 254–267.  Epub 2005 Oct 2012. <ul class="find" data-citedArticleID="1033074" data-doi="10.1093/molbev/msj030"><li><a href="http://dx.doi.org/10.1093/molbev/msj030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+phylogenetic+networks+in+evolutionary+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+phylogenetic+networks+in+evolutionary+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001346-Renesto1" id="pone.0001346-Renesto1"></a>Renesto P, Gouvernet J, Drancourt M, Roux V, Raoult D (2001) Use of rpoB gene analysis for detection and identification of <em>Bartonella species</em>. J Clin Microbiol  39: 430–437.  <ul class="find" data-citedArticleID="1033088" data-doi="10.1128/jcm.39.2.430-437.2001"><li><a href="http://dx.doi.org/10.1128/jcm.39.2.430-437.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+rpoB+gene+analysis+for+detection+and+identification+of+Bartonella+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+rpoB+gene+analysis+for+detection+and+identification+of+Bartonella+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001346-Lindroos1" id="pone.0001346-Lindroos1"></a>Lindroos H, Vinnere O, Mira A, Repsilber D, Naslund K, et al.  (2006) Genome Rearrangements, Deletions, and Amplifications in the Natural Population of <em>Bartonella henselae</em>. J Bacteriol  188: 7426–7439.  Epub 2006 Aug 7425.  <ul class="find" data-citedArticleID="1033082" data-doi="10.1128/jb.00472-06"><li><a href="http://dx.doi.org/10.1128/jb.00472-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+Rearrangements%2C+Deletions%2C+and+Amplifications+in+the+Natural+Population+of+Bartonella+henselae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+Rearrangements%2C+Deletions%2C+and+Amplifications+in+the+Natural+Population+of+Bartonella+henselae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001346-LaScola1" id="pone.0001346-LaScola1"></a>La Scola B, Liang Z, Zeaiter Z, Houpikian P, Grimont PA, et al.  (2002) Genotypic characteristics of two serotypes of <em>Bartonella henselae</em>. J Clin Microbiol  40: 2002–2008.  <ul class="find" data-citedArticleID="1033080" data-doi="10.1128/jcm.40.6.2002-2008.2002"><li><a href="http://dx.doi.org/10.1128/jcm.40.6.2002-2008.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotypic+characteristics+of+two+serotypes+of+Bartonella+henselae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotypic+characteristics+of+two+serotypes+of+Bartonella+henselae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001346-Tamura1" id="pone.0001346-Tamura1"></a>Tamura K, Dudley J, Nei M, Kumar S (2007) MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0. Mol Biol Evol  24: 1596–1599.  Epub 2007 May 1597.  <ul class="find" data-citedArticleID="1033100" data-doi="10.1093/molbev/msm092"><li><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA4%3A+Molecular+Evolutionary+Genetics+Analysis+%28MEGA%29+software+version+4.0." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA4%3A+Molecular+Evolutionary+Genetics+Analysis+%28MEGA%29+software+version+4.0.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001346-Turner1" id="pone.0001346-Turner1"></a>Turner KM, Hanage WP, Fraser C, Connor TR, Spratt BG (2007) Assessing the reliability of eBURST using simulated populations with known ancestry. BMC Microbiol  7: 30.  <ul class="find" data-citedArticleID="1033102" data-doi="10.1186/1471-2180-7-30"><li><a href="http://dx.doi.org/10.1186/1471-2180-7-30" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+the+reliability+of+eBURST+using+simulated+populations+with+known+ancestry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+the+reliability+of+eBURST+using+simulated+populations+with+known+ancestry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001346-Melter1" id="pone.0001346-Melter1"></a>Melter O, Hercik K, Weyant RS, Janecek J, Nemec A, et al.  (2003) Detection and characterization of feline <em>Bartonella henselae</em> in the Czech Republic. Vet Microbiol  93: 261–273.  <ul class="find" data-citedArticleID="1033086" data-doi="10.1016/s0378-1135(03)00032-4"><li><a href="http://dx.doi.org/10.1016/s0378-1135(03)00032-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+and+characterization+of+feline+Bartonella+henselae+in+the+Czech+Republic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+and+characterization+of+feline+Bartonella+henselae+in+the+Czech+Republic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001346-Gurfield1" id="pone.0001346-Gurfield1"></a>Gurfield AN, Boulouis HJ, Chomel BB, Kasten RW, Heller R, et al.  (2001) Epidemiology of Bartonella infection in domestic cats in France. Vet Microbiol  80: 185–198.  <ul class="find" data-citedArticleID="1033070" data-doi="10.1016/s0378-1135(01)00304-2"><li><a href="http://dx.doi.org/10.1016/s0378-1135(01)00304-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epidemiology+of+Bartonella+infection+in+domestic+cats+in+France." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epidemiology+of+Bartonella+infection+in+domestic+cats+in+France.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001346-Arvand4" id="pone.0001346-Arvand4"></a>Arvand M, Wendt C, Regnath T, Ullrich R, Hahn H (1998) Characterization of <em>Bartonella henselae</em> isolated from bacillary angiomatosis lesions in a human immunodeficiency virus-infected patient in Germany. Clin Infect Dis  26: 1296–1299.  <ul class="find" data-citedArticleID="1033038" data-doi="10.1086/516348"><li><a href="http://dx.doi.org/10.1086/516348" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+Bartonella+henselae+isolated+from+bacillary+angiomatosis+lesions+in+a+human+immunodeficiency+virus-infected+patient+in+Germany." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+Bartonella+henselae+isolated+from+bacillary+angiomatosis+lesions+in+a+human+immunodeficiency+virus-infected+patient+in+Germany.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001346-Fabbi1" id="pone.0001346-Fabbi1"></a>Fabbi M, De Giuli L, Tranquillo M, Bragoni R, Casiraghi M, et al.  (2004) Prevalence of <em>Bartonella henselae</em> in Italian stray cats: evaluation of serology to assess the risk of transmission of Bartonella to humans. J Clin Microbiol  42: 264–268.  <ul class="find" data-citedArticleID="1033060" data-doi="10.1128/jcm.42.1.264-268.2004"><li><a href="http://dx.doi.org/10.1128/jcm.42.1.264-268.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalence+of+Bartonella+henselae+in+Italian+stray+cats%3A+evaluation+of+serology+to+assess+the+risk+of+transmission+of+Bartonella+to+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalence+of+Bartonella+henselae+in+Italian+stray+cats%3A+evaluation+of+serology+to+assess+the+risk+of+transmission+of+Bartonella+to+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001346-Birtles1" id="pone.0001346-Birtles1"></a>Birtles RJ, Laycock G, Kenny MJ, Shaw SE, Day MJ (2002) Prevalence of <em>Bartonella species</em> causing bacteraemia in domesticated and companion animals in the United Kingdom. Vet Rec  151: 225–229.  <ul class="find" data-citedArticleID="1033046" data-doi="10.1136/vr.151.8.225"><li><a href="http://dx.doi.org/10.1136/vr.151.8.225" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalence+of+Bartonella+species+causing+bacteraemia+in+domesticated+and+companion+animals+in+the+United+Kingdom." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalence+of+Bartonella+species+causing+bacteraemia+in+domesticated+and+companion+animals+in+the+United+Kingdom.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001346-Guptill2" id="pone.0001346-Guptill2"></a>Guptill L, Wu CC, HogenEsch H, Slater LN, Glickman N, et al.  (2004) Prevalence, risk factors, and genetic diversity of <em>Bartonella henselae</em> infections in pet cats in four regions of the United States. J Clin Microbiol  42: 652–659.  <ul class="find" data-citedArticleID="1033068" data-doi="10.1128/jcm.42.2.652-659.2004"><li><a href="http://dx.doi.org/10.1128/jcm.42.2.652-659.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalence%2C+risk+factors%2C+and+genetic+diversity+of+Bartonella+henselae+infections+in+pet+cats+in+four+regions+of+the+United+States." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalence%2C+risk+factors%2C+and+genetic+diversity+of+Bartonella+henselae+infections+in+pet+cats+in+four+regions+of+the+United+States.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.XML" value="101384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.PDF" value="344841"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t001.PNG_L" value="140071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t001.PNG_M" value="83853"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t001.PNG_S" value="12556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t001.TIF" value="695036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t001.PNG_I" value="26246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g001.PNG_L" value="293273"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g001.PNG_M" value="88105"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g001.PNG_S" value="12764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g001.TIF" value="499880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g001.PNG_I" value="29878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t002.PNG_L" value="114683"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t002.PNG_M" value="124377"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t002.PNG_S" value="18743"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t002.TIF" value="574202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t002.PNG_I" value="57964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g002.PNG_L" value="122045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g002.PNG_M" value="56178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g002.PNG_S" value="10130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g002.TIF" value="429066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g002.PNG_I" value="19397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t003.PNG_L" value="107993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t003.PNG_M" value="130633"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t003.PNG_S" value="18384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t003.TIF" value="527026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t003.PNG_I" value="50237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g003.PNG_L" value="63467"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g003.PNG_M" value="43285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g003.PNG_S" value="7443"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g003.TIF" value="85160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g003.PNG_I" value="14204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g004.PNG_L" value="17311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g004.PNG_M" value="17311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g004.PNG_S" value="8977"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g004.TIF" value="27508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g004.PNG_I" value="5938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g005.PNG_L" value="93230"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g005.PNG_M" value="92117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g005.PNG_S" value="12904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g005.TIF" value="156798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.g005.PNG_I" value="56111"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t004.PNG_L" value="103435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t004.PNG_M" value="57749"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t004.PNG_S" value="9912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t004.TIF" value="571416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001346.t004.PNG_I" value="16996"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001346&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001346" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001346&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001346&volume=&issue=&title=Multi-Locus Sequence Typing of Bartonella henselae Isolates from Three Continents Reveals Hypervirulent and Feline-Associated Clones&author_name=Mardjan%20Arvand%2C%20Edward%20J.%20Feil%2C%20Michael%20Giladi%2C%20Henri-Jean%20Boulouis%2C%20Juliane%20Viezens&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001346" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001346" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001346" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001346&amp;t=Multi-Locus%20Sequence%20Typing%20of%20Bartonella%20henselae%20Isolates%20from%20Three%20Continents%20Reveals%20Hypervirulent%20and%20Feline-Associated%20Clones" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001346&title=Multi-Locus%20Sequence%20Typing%20of%20Bartonella%20henselae%20Isolates%20from%20Three%20Continents%20Reveals%20Hypervirulent%20and%20Feline-Associated%20Clones&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001346&amp;title=Multi-Locus%20Sequence%20Typing%20of%20Bartonella%20henselae%20Isolates%20from%20Three%20Continents%20Reveals%20Hypervirulent%20and%20Feline-Associated%20Clones" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001346" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001346&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Multi-Locus Sequence Typing of Bartonella henselae Isolates from Three Continents Reveals Hypervirulent and Feline-Associated Clones';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001346';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Multi-Locus%20Sequence%20Typing%20of%20Bartonella%20henselae%20Isolates%20from%20Three%20Continents%20Reveals%20Hypervirulent%20and%20Feline-Associated%20Clones http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001346" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001346" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Australia%22" title="Search for articles in the subject area:'Australia'"><div class="flagText">Australia</div></a>
              <div data-categoryid="40109" data-articleid="25522"
                   data-categoryname="Australia"
                   class="flagImage" title="Flag 'Australia' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Blood%22" title="Search for articles in the subject area:'Blood'"><div class="flagText">Blood</div></a>
              <div data-categoryid="47189" data-articleid="25522"
                   data-categoryname="Blood"
                   class="flagImage" title="Flag 'Blood' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25522"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Europe%22" title="Search for articles in the subject area:'Europe'"><div class="flagText">Europe</div></a>
              <div data-categoryid="40225" data-articleid="25522"
                   data-categoryname="Europe"
                   class="flagImage" title="Flag 'Europe' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Geographic+distribution%22" title="Search for articles in the subject area:'Geographic distribution'"><div class="flagText">Geographic distribution</div></a>
              <div data-categoryid="19113" data-articleid="25522"
                   data-categoryname="Geographic distribution"
                   class="flagImage" title="Flag 'Geographic distribution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Geography%22" title="Search for articles in the subject area:'Geography'"><div class="flagText">Geography</div></a>
              <div data-categoryid="1863" data-articleid="25522"
                   data-categoryname="Geography"
                   class="flagImage" title="Flag 'Geography' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="25522"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="25522"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8476'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5561'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9581&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
