

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Ancestral Inference and the Study of Codon Bias Evolution: Implications for Molecular Evolutionary Analyses of the Drosophila melanogaster Subgroup</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001065"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001065" />

    <meta name="citation_title" content="Ancestral Inference and the Study of Codon Bias Evolution: Implications for Molecular Evolutionary Analyses of the Drosophila melanogaster Subgroup"/>
    <meta itemprop="name" content="Ancestral Inference and the Study of Codon Bias Evolution: Implications for Molecular Evolutionary Analyses of the Drosophila melanogaster Subgroup"/>

      <meta name="citation_author" content="Hiroshi Akashi"/>
            <meta name="citation_author_institution" content="Institute of Molecular Evolutionary Genetics, Department of Biology, Pennsylvania State University, State College, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Piyush Goel"/>
            <meta name="citation_author_institution" content="Institute of Molecular Evolutionary Genetics, Department of Biology, Pennsylvania State University, State College, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Anoop John"/>
            <meta name="citation_author_institution" content="Institute of Molecular Evolutionary Genetics, Department of Biology, Pennsylvania State University, State College, Pennsylvania, United States of America"/>

    <meta name="citation_date" content="2007/10/24"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001065.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1065"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Natural selection and the origin of jingwei, a chimeric processed functional gene in Drosophila.; citation_author=M Long; citation_author=CH Langley; citation_journal_title=Science; citation_volume=260; citation_number=1; citation_pages=91-95; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Molecular evolution between Drosophila melanogaster and D. simulans: reduced codon bias, faster rates of amino acid substitution, and larger proteins in D. melanogaster.; citation_author=H Akashi; citation_journal_title=Genetics; citation_volume=144; citation_number=2; citation_pages=1297-1307; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Historical selection, amino acid polymorphism and lineage-specific divergence at the G6pd locus in Drosophila melanogaster and D. simulans.; citation_author=WF Eanes; citation_author=M Kirchner; citation_author=J Yoon; citation_author=CH Biermann; citation_author=IN Wang; citation_journal_title=Genetics; citation_volume=144; citation_number=3; citation_pages=1027-1041; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Long term trends in the evolution of H(3) HA1 human influenza type A.; citation_author=WM Fitch; citation_author=RM Bush; citation_author=CA Bender; citation_author=NJ Cox; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=4; citation_pages=7712-7718; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Episodic adaptive evolution of primate lysozymes.; citation_author=W Messier; citation_author=CB Stewart; citation_journal_title=Nature; citation_volume=385; citation_number=5; citation_pages=151-154; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Local changes in GC/AT substitution biases and in crossover frequencies on Drosophila chromosomes.; citation_author=T Takano-Shimizu; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=6; citation_pages=606-619; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of a duplicated pancreatic ribonuclease gene in a leaf-eating monkey.; citation_author=J Zhang; citation_author=YP Zhang; citation_author=HF Rosenberg; citation_journal_title=Nat Genet; citation_volume=30; citation_number=7; citation_pages=411-415; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Inferring weak selection from patterns of polymorphism and divergence at “silent” sites in Drosophila DNA.; citation_author=H Akashi; citation_journal_title=Genetics; citation_volume=139; citation_number=8; citation_pages=1067-1076; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Inferring the fitness effects of DNA mutations from polymorphism and divergence data: statistical power to detect directional selection under stationarity and free recombination.; citation_author=H Akashi; citation_journal_title=Genetics; citation_volume=151; citation_number=9; citation_pages=221-238; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Contingency tests of neutrality using intra/interspecific gene trees: the rejection of neutrality for the evolution of the mitochondrial cytochrome oxidase II gene in the hominoid primates.; citation_author=AR Templeton; citation_journal_title=Genetics; citation_volume=144; citation_number=10; citation_pages=1263-1270; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A method for detecting positive selection at single amino acid sites.; citation_author=Y Suzuki; citation_author=T Gojobori; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=11; citation_pages=1315-1328; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Hitchhiking under positive Darwinian selection.; citation_author=JC Fay; citation_author=CI Wu; citation_journal_title=Genetics; citation_volume=155; citation_number=12; citation_pages=1405-1413; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Simultaneous inference of selection and population growth from patterns of variation in the human genome.; citation_author=SH Williamson; citation_author=R Hernandez; citation_author=A Fledel-Alon; citation_author=L Zhu; citation_author=R Nielsen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=13; citation_pages=7882-7887; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Recent selection on synonymous codon usage in Drosophila.; citation_author=RM Kliman; citation_journal_title=J Mol Evol; citation_volume=49; citation_number=14; citation_pages=343-351; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The frequency distribution of nucleotide variation in Drosophila simulans.; citation_author=DJ Begun; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=15; citation_pages=1343-1352; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=DNA variability and divergence at the notch locus in Drosophila melanogaster and D. simulans: a case of accelerated synonymous site divergence.; citation_author=VB DuMont; citation_author=JC Fay; citation_author=PP Calabrese; citation_author=CF Aquadro; citation_journal_title=Genetics; citation_volume=167; citation_number=16; citation_pages=171-185; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Intragenic Hill-Robertson Interference Influences Selection Intensity on Synonymous Mutations in Drosophila.; citation_author=JM Comeron; citation_author=TB Guthrie; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=17; citation_pages=2519-2530; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Recombination enhances protein adaptation in Drosophila melanogaster.; citation_author=DC Presgraves; citation_journal_title=Curr Biol; citation_volume=15; citation_number=18; citation_pages=1651-1656; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Compositional Bias, Character-State Bias, and Character-State Reconstruction Using Parsimony.; citation_author=TM Collins; citation_author=PH Wimberger; citation_author=GJP Naylor; citation_journal_title=Systematic Biology; citation_volume=43; citation_number=19; citation_pages=482-496; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Synonymous rates at the RpII215 gene of Drosophila: variation among species and across the coding region.; citation_author=A Llopart; citation_author=M Aguade; citation_journal_title=Genetics; citation_volume=152; citation_number=20; citation_pages=269-280; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The evolution of codon preferences in Drosophila: a maximum-likelihood approach to parameter estimation and hypothesis testing.; citation_author=GA McVean; citation_author=J Vieira; citation_journal_title=J Mol Evol; citation_volume=49; citation_number=21; citation_pages=63-75; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Evidence for a high ancestral GC content in Drosophila.; citation_author=F Rodriguez-Trelles; citation_author=R Tarrio; citation_author=FJ Ayala; citation_journal_title=Mol Biol Evol; citation_volume=17; citation_number=22; citation_pages=1710-1717; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Fluctuating mutation bias and the evolution of base composition in Drosophila.; citation_author=F Rodriguez-Trelles; citation_author=R Tarrio; citation_author=FJ Ayala; citation_journal_title=J Mol Evol; citation_volume=50; citation_number=23; citation_pages=1-10; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular population genetics of Xdh and the evolution of base composition in Drosophila.; citation_author=DJ Begun; citation_author=P Whitley; citation_journal_title=Genetics; citation_volume=162; citation_number=24; citation_pages=1725-1735; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Protein evolution and codon usage bias on the neo-sex chromosomes of Drosophila miranda.; citation_author=D Bachtrog; citation_journal_title=Genetics; citation_volume=165; citation_number=25; citation_pages=1221-1232; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Nucleotide polymorphism in the RpII215 gene region of the insular species Drosophila guanche: reduced efficacy of weak selection on synonymous variation.; citation_author=JA Perez; citation_author=A Munte; citation_author=J Rozas; citation_author=C Segarra; citation_author=M Aguade; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=26; citation_pages=1867-1875; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Analysis of a shift in codon usage in Drosophila.; citation_author=JR Powell; citation_author=E Sezzi; citation_author=EN Moriyama; citation_author=JM Gleason; citation_author=A Caccone; citation_journal_title=J Mol Evol 57 Suppl; citation_volume=1; citation_number=27; citation_pages=S214-225; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Vanishing GC-rich isochores in mammalian genomes.; citation_author=L Duret; citation_author=M Semon; citation_author=G Piganeau; citation_author=D Mouchiroud; citation_author=N Galtier; citation_journal_title=Genetics; citation_volume=162; citation_number=28; citation_pages=1837-1847; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The compositional evolution of the murid genome.; citation_author=NG Smith; citation_author=A Eyre-Walker; citation_journal_title=J Mol Evol; citation_volume=55; citation_number=29; citation_pages=197-201; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Distinct changes of genomic biases in nucleotide substitution at the time of Mammalian radiation.; citation_author=PF Arndt; citation_author=DA Petrov; citation_author=T Hwa; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=30; citation_pages=1887-1896; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Compositional evolution of noncoding DNA in the human and chimpanzee genomes.; citation_author=MT Webster; citation_author=NG Smith; citation_author=H Ellegren; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=31; citation_pages=278-286; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The decline of isochores in mammals: an assessment of the GC content variation along the mammalian phylogeny.; citation_author=EM Belle; citation_author=L Duret; citation_author=N Galtier; citation_author=A Eyre-Walker; citation_journal_title=J Mol Evol; citation_volume=58; citation_number=32; citation_pages=653-660; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Reconstruction of ancestral nucleotide sequences and estimation of substitution frequencies in a star phylogeny.; citation_author=PF Arndt; citation_journal_title=Gene; citation_volume=390; citation_number=33; citation_pages=75-83; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Codon bias differentiates between the duplicated amylase loci following gene duplication in Drosophila.; citation_author=Z Zhang; citation_author=N Inomata; citation_author=T Ohba; citation_author=ML Cariou; citation_author=T Yamazaki; citation_journal_title=Genetics; citation_volume=161; citation_number=34; citation_pages=1187-1196; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular evolution in the Drosophila melanogaster species subgroup: frequent parameter fluctuations on the timescale of molecular divergence.; citation_author=H Akashi; citation_author=WY Ko; citation_author=S Piao; citation_author=A John; citation_author=P Goel; citation_journal_title=Genetics; citation_volume=172; citation_number=35; citation_pages=1711-1726; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Strong regional heterogeneity in base composition evolution on the Drosophila X chromosome.; citation_author=WY Ko; citation_author=S Piao; citation_author=H Akashi; citation_journal_title=Genetics; citation_volume=174; citation_number=36; citation_pages=349-362; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Patterns of molecular variation and evolution in Drosophila americana and its relatives.; citation_author=X Maside; citation_author=B Charlesworth; citation_journal_title=Genetics; citation_number=37; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Unequal Base Frequencies and Estimation of Substitution Rates.; citation_author=NT Perna; citation_author=TD Kocher; citation_journal_title=Molecular Biology and Evolution; citation_volume=12; citation_number=38; citation_pages=359-361; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Problems with parsimony in sequences of biased base composition.; citation_author=A Eyre-Walker; citation_journal_title=J Mol Evol; citation_volume=47; citation_number=39; citation_pages=686-690; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A new method for locating changes in a tree reveals distinct nucleotide polymorphism vs. divergence patterns in mouse mitochondrial control region.; citation_author=N Galtier; citation_author=P Boursot; citation_journal_title=J Mol Evol; citation_volume=50; citation_number=40; citation_pages=224-231; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Inaccurate reconstruction of ancestral GC levels creates a “vanishing isochores” effect.; citation_author=F Alvarez-Valin; citation_author=O Clay; citation_author=S Cruveiller; citation_author=G Bernardi; citation_journal_title=Mol Phylogenet Evol; citation_volume=31; citation_number=41; citation_pages=788-793; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Dating of the human-ape splitting by a molecular clock of mitochondrial DNA.; citation_author=M Hasegawa; citation_author=H Kishino; citation_author=T Yano; citation_journal_title=J Mol Evol; citation_volume=22; citation_number=42; citation_pages=160-174; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms.; citation_author=T Ikemura; citation_journal_title=Mol Biol Evol; citation_volume=2; citation_number=43; citation_pages=13-34; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=“Silent” sites in Drosophila genes are not neutral: evidence of selection among synonymous codons.; citation_author=DC Shields; citation_author=PM Sharp; citation_author=DG Higgins; citation_author=F Wright; citation_journal_title=Mol Biol Evol; citation_volume=5; citation_number=44; citation_pages=704-716; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Codon preferences in free-living microorganisms.; citation_author=SG Andersson; citation_author=CG Kurland; citation_journal_title=Microbiol Rev; citation_volume=54; citation_number=45; citation_pages=198-210; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=DNA sequence evolution: the sounds of silence.; citation_author=PM Sharp; citation_author=M Averof; citation_author=AT Lloyd; citation_author=G Matassi; citation_author=JF Peden; citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_volume=349; citation_number=46; citation_pages=241-247; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Gene expression and molecular evolution.; citation_author=H Akashi; citation_journal_title=Curr Opin Genet Dev; citation_volume=11; citation_number=47; citation_pages=660-666; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evolution of synonymous codon usage in metazoans.; citation_author=L Duret; citation_journal_title=Curr Opin Genet Dev; citation_volume=12; citation_number=48; citation_pages=640-649; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of Bacillus subtilis tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis.; citation_author=S Kanaya; citation_author=Y Yamada; citation_author=Y Kudo; citation_author=T Ikemura; citation_journal_title=Gene; citation_volume=238; citation_number=49; citation_pages=143-155; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Codon usage and tRNA genes in eukaryotes: correlation of codon usage diversity with translation efficiency and with CG-dinucleotide usage as assessed by multivariate analysis.; citation_author=S Kanaya; citation_author=Y Yamada; citation_author=M Kinouchi; citation_author=Y Kudo; citation_author=T Ikemura; citation_journal_title=J Mol Evol; citation_volume=53; citation_number=50; citation_pages=290-298; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Codon usage bias and tRNA abundance in Drosophila.; citation_author=EN Moriyama; citation_author=JR Powell; citation_journal_title=J Mol Evol; citation_volume=45; citation_number=51; citation_pages=514-523; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=tRNA gene number and codon usage in the C. elegans genome are co-adapted for optimal translation of highly expressed genes.; citation_author=L Duret; citation_journal_title=Trends Genet; citation_volume=16; citation_number=52; citation_pages=287-289; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Effects of gene expression on molecular evolution in Arabidopsis thaliana and Arabidopsis lyrata.; citation_author=SI Wright; citation_author=CB Yau; citation_author=M Looseley; citation_author=BC Meyers; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=53; citation_pages=1719-1726; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Translational accuracy and the fitness of bacteria.; citation_author=CG Kurland; citation_journal_title=Annu Rev Genet; citation_volume=26; citation_number=54; citation_pages=29-50; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Natural selection and the frequency distributions of “silent” DNA polymorphism in Drosophila.; citation_author=H Akashi; citation_author=SW Schaeffer; citation_journal_title=Genetics; citation_volume=146; citation_number=55; citation_pages=295-307; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Nucleotide polymorphism at the RpII215 gene in Drosophila subobscura. Weak selection on synonymous mutations.; citation_author=A Llopart; citation_author=M Aguade; citation_journal_title=Genetics; citation_volume=155; citation_number=56; citation_pages=1245-1252; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Selection on codon usage in Drosophila americana.; citation_author=X Maside; citation_author=AW Lee; citation_author=B Charlesworth; citation_journal_title=Curr Biol; citation_volume=14; citation_number=57; citation_pages=150-154; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Patterns of selection on synonymous and nonsynonymous variants in Drosophila miranda.; citation_author=C Bartolome; citation_author=X Maside; citation_author=S Yi; citation_author=AL Grant; citation_author=B Charlesworth; citation_journal_title=Genetics; citation_volume=169; citation_number=58; citation_pages=1495-1507; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A population genetic model for the evolution of synonymous codon usage: patterns and predictions.; citation_author=GAT McVean; citation_author=B Charlesworth; citation_journal_title=Genetical Research; citation_volume=74; citation_number=59; citation_pages=145-158; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny of the Drosophila melanogaster species subgroup.; citation_author=WY Ko; citation_author=RM David; citation_author=H Akashi; citation_journal_title=J Mol Evol; citation_volume=57; citation_number=60; citation_pages=562-573; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic incongruence in the Drosophila melanogaster species group.; citation_author=A Wong; citation_author=JD Jensen; citation_author=JE Pool; citation_author=CF Aquadro; citation_journal_title=Mol Phylogenet Evol; citation_number=61; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Widespread discordance of gene trees with species tree in Drosophila: evidence for incomplete lineage sorting.; citation_author=DA Pollard; citation_author=VN Iyer; citation_author=AM Moses; citation_author=MB Eisen; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=62; citation_pages=e173; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=Comput Appl Biosci; citation_volume=13; citation_number=63; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions.; citation_author=M Nei; citation_author=T Gojobori; citation_journal_title=Mol Biol Evol; citation_volume=3; citation_number=64; citation_pages=418-426; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=A new method of inference of ancestral nucleotide and amino acid sequences.; citation_author=Z Yang; citation_author=S Kumar; citation_author=M Nei; citation_journal_title=Genetics; citation_volume=141; citation_number=65; citation_pages=1641-1650; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Evolution of protein molecules.; citation_author=TH Jukes; citation_author=CR Cantor; citation_number=66; citation_pages=21-132; citation_date=1969; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=Ancestral sequence reconstruction in primate mitochondrial DNA: compositional bias and effect on functional inference.; citation_author=NM Krishnan; citation_author=H Seligmann; citation_author=CB Stewart; citation_author=AP De Koning; citation_author=DD Pollock; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=67; citation_pages=1871-1883; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Statistical Methods: 8th Edition.; citation_author=GW Snedecor; citation_author=WG Cochran; citation_number=68; citation_pages=503; citation_date=1989; citation_publisher=Iowa State University Press; " />
      <meta name="citation_reference" content="citation_title=Accuracies of ancestral amino acid sequences inferred by the parsimony, likelihood, and distance methods.; citation_author=J Zhang; citation_author=M Nei; citation_journal_title=J Mol Evol; citation_volume=44; citation_number=69; citation_pages=S139-146; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Local recombination and mutation effects on molecular evolution in Drosophila.; citation_author=T Takano-Shimizu; citation_journal_title=Genetics; citation_volume=153; citation_number=70; citation_pages=1285-1296; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Switch in codon bias and increased rates of amino acid substitution in the Drosophila saltans species group.; citation_author=F Rodriguez-Trelles; citation_author=R Tarrio; citation_author=FJ Ayala; citation_journal_title=Genetics; citation_volume=153; citation_number=71; citation_pages=339-350; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Reconstructing large regions of an ancestral mammalian genome in silico.; citation_author=M Blanchette; citation_author=ED Green; citation_author=W Miller; citation_author=D Haussler; citation_journal_title=Genome Res; citation_volume=14; citation_number=72; citation_pages=2412-2423; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy.; citation_author=H Akashi; citation_journal_title=Genetics; citation_volume=136; citation_number=73; citation_pages=927-935; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Gene length and codon usage bias in Drosophila melanogaster, Saccharomyces cerevisiae and Escherichia coli.; citation_author=EN Moriyama; citation_author=JR Powell; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=74; citation_pages=3188-3193; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Patterns of base composition within the genes of Drosophila melanogaster.; citation_author=RM Kliman; citation_author=A Eyre-Walker; citation_journal_title=J Mol Evol; citation_volume=46; citation_number=75; citation_pages=534-541; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A test of translational selection at ‘silent’ sites in the human genome: base composition comparisons in alternatively spliced genes.; citation_author=K Iida; citation_author=H Akashi; citation_journal_title=Gene; citation_volume=261; citation_number=76; citation_pages=93-105; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Population, evolutionary and genomic consequences of interference selection.; citation_author=JM Comeron; citation_author=M Kreitman; citation_journal_title=Genetics; citation_volume=161; citation_number=77; citation_pages=389-410; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Intragenic spatial patterns of codon usage bias in prokaryotic and eukaryotic genomes.; citation_author=H Qin; citation_author=WB Wu; citation_author=JM Comeron; citation_author=M Kreitman; citation_author=WH Li; citation_journal_title=Genetics; citation_volume=168; citation_number=78; citation_pages=2245-2260; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Empirical and hierarchical Bayesian estimation of ancestral states.; citation_author=JP Huelsenbeck; citation_author=JP Bollback; citation_journal_title=Syst Biol; citation_volume=50; citation_number=79; citation_pages=351-366; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mapping mutations on phylogenies.; citation_author=R Nielsen; citation_journal_title=Syst Biol; citation_volume=51; citation_number=80; citation_pages=729-739; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Bayesian estimation of ancestral character states on phylogenies.; citation_author=M Pagel; citation_author=A Meade; citation_author=D Barker; citation_journal_title=Syst Biol; citation_volume=53; citation_number=81; citation_pages=673-684; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=On the use of nucleic acid sequences to infer early branchings in the tree of life.; citation_author=Z Yang; citation_author=D Roberts; citation_journal_title=Mol Biol Evol; citation_volume=12; citation_number=82; citation_pages=451-458; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Inferring pattern and process: maximum-likelihood implementation of a nonhomogeneous model of DNA sequence evolution for phylogenetic analysis.; citation_author=N Galtier; citation_author=M Gouy; citation_journal_title=Mol Biol Evol; citation_volume=15; citation_number=83; citation_pages=871-879; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Step matrices and the interpretation of homoplasy.; citation_author=RH Ree; citation_author=MJ Donoghue; citation_journal_title=Syst Biol; citation_volume=47; citation_number=84; citation_pages=582-588; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Molecular evolution.; citation_author=W-H Li; citation_number=85; citation_pages=xv, 487; citation_date=1997; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=Maximum likelihood estimation of ancestral codon usage bias parameters in Drosophila.; citation_author=R Nielsen; citation_author=VL Bauer DuMont; citation_author=MJ Hubisz; citation_author=CF Aquadro; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=86; citation_pages=228-235; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Context dependence, ancestral misidentification, and spurious signatures of natural selection.; citation_author=RD Hernandez; citation_author=SH Williamson; citation_author=CD Bustamante; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=87; citation_pages=1792-1800; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Confidence interval for the number of selectively neutral amino acid polymorphisms.; citation_author=SA Sawyer; citation_author=DE Dykhuizen; citation_author=DL Hartl; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=84; citation_number=88; citation_pages=6225-6228; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Adaptive protein evolution at the Adh locus in Drosophila.; citation_author=JH McDonald; citation_author=M Kreitman; citation_journal_title=Nature; citation_volume=351; citation_number=89; citation_pages=652-654; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The cost of inbreeding in Arabidopsis.; citation_author=CD Bustamante; citation_author=R Nielsen; citation_author=SA Sawyer; citation_author=KM Olsen; citation_author=MD Purugganan; citation_journal_title=Nature; citation_volume=416; citation_number=90; citation_pages=531-534; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Directional selection and the site-frequency spectrum.; citation_author=CD Bustamante; citation_author=J Wakeley; citation_author=S Sawyer; citation_author=DL Hartl; citation_journal_title=Genetics; citation_volume=159; citation_number=91; citation_pages=1779-1788; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of non-coding DNA in Drosophila.; citation_author=P Andolfatto; citation_journal_title=Nature; citation_volume=437; citation_number=92; citation_pages=1149-1152; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Effect of misoriented sites on neutrality tests with outgroup.; citation_author=E Baudry; citation_author=F Depaulis; citation_journal_title=Genetics; citation_volume=165; citation_number=93; citation_pages=1619-1622; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001065" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Ancestral Inference and the Study of Codon Bias Evolution: Implications for Molecular Evolutionary Analyses of the Drosophila melanogaster Subgroup"/>
    <meta name="twitter:description" content="Reliable inference of ancestral sequences can be critical to identifying both patterns and causes of molecular evolution. Robustness of ancestral inference is often assumed among closely related species, but tests of this assumption have been limited. Here, we examine the performance of inference methods for data simulated under scenarios of codon bias evolution within the Drosophila melanogaster subgroup. Genome sequence data for multiple, closely related species within this subgroup make it an important system for studying molecular evolutionary genetics. The effects of asymmetric and lineage-specific substitution rates (i.e., varying levels of codon usage bias and departures from equilibrium) on the reliability of ancestral codon usage was investigated. Maximum parsimony inference, which has been widely employed in analyses of Drosophila codon bias evolution, was compared to an approach that attempts to account for uncertainty in ancestral inference by weighting ancestral reconstructions by their posterior probabilities. The latter approach employs maximum likelihood estimation of rate and base composition parameters. For equilibrium and most non-equilibrium scenarios that were investigated, the probabilistic method appears to generate reliable ancestral codon bias inferences for molecular evolutionary studies within the D. melanogaster subgroup. These reconstructions are more reliable than parsimony inference, especially when codon usage is strongly skewed. However, inference biases are considerable for both methods under particular departures from stationarity (i.e., when adaptive evolution is prevalent). Reliability of inference can be sensitive to branch lengths, asymmetry in substitution rates, and the locations and nature of lineage-specific processes within a gene tree. Inference reliability, even among closely related species, can be strongly affected by (potentially unknown) patterns of molecular evolution in lineages ancestral to those of interest."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001065.g009"/>

  <meta property="og:title" content="Ancestral Inference and the Study of Codon Bias Evolution: Implications for Molecular Evolutionary Analyses of the Drosophila melanogaster Subgroup" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001065" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5018'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5337'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9323&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001065">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001065" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001065&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Ancestral Inference and the Study of Codon Bias Evolution: Implications for Molecular Evolutionary Analyses of the <i>Drosophila melanogaster</i> Subgroup
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hiroshi Akashi
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:akashi@psu.edu">akashi@psu.edu</a></p>

                <p>Affiliation:
                  Institute of Molecular Evolutionary Genetics, Department of Biology, Pennsylvania State University, State College, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Piyush Goel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Molecular Evolutionary Genetics, Department of Biology, Pennsylvania State University, State College, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anoop John
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Molecular Evolutionary Genetics, Department of Biology, Pennsylvania State University, State College, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 24, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001065</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001065" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001065" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001065" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001065" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001065-g001" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001065-g002" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001065-g003" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001065-g004" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001065-t001" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001065-g005" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001065-g006" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001065-g007" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001065-g008" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001065-g009" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001065">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001065" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Reliable inference of ancestral sequences can be critical to identifying both patterns and causes of molecular evolution. Robustness of ancestral inference is often assumed among closely related species, but tests of this assumption have been limited. Here, we examine the performance of inference methods for data simulated under scenarios of codon bias evolution within the <em>Drosophila melanogaster</em> subgroup. Genome sequence data for multiple, closely related species within this subgroup make it an important system for studying molecular evolutionary genetics. The effects of asymmetric and lineage-specific substitution rates (<em>i.e</em>., varying levels of codon usage bias and departures from equilibrium) on the reliability of ancestral codon usage was investigated. Maximum parsimony inference, which has been widely employed in analyses of <em>Drosophila</em> codon bias evolution, was compared to an approach that attempts to account for uncertainty in ancestral inference by weighting ancestral reconstructions by their posterior probabilities. The latter approach employs maximum likelihood estimation of rate and base composition parameters. For equilibrium and most non-equilibrium scenarios that were investigated, the probabilistic method appears to generate reliable ancestral codon bias inferences for molecular evolutionary studies within the <em>D. melanogaster</em> subgroup. These reconstructions are more reliable than parsimony inference, especially when codon usage is strongly skewed. However, inference biases are considerable for both methods under particular departures from stationarity (<em>i.e</em>., when adaptive evolution is prevalent). Reliability of inference can be sensitive to branch lengths, asymmetry in substitution rates, and the locations and nature of lineage-specific processes within a gene tree. Inference reliability, even among closely related species, can be strongly affected by (potentially unknown) patterns of molecular evolution in lineages ancestral to those of interest.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Akashi H, Goel P, John A (2007) Ancestral Inference and the Study of Codon Bias Evolution: Implications for Molecular Evolutionary Analyses of the <em>Drosophila melanogaster</em> Subgroup. PLoS ONE 2(10):
          e1065.
            doi:10.1371/journal.pone.0001065</p><p><strong>Academic Editor: </strong>Justin Fay, Washington University in St. Louis School of Medicine, United States of America</p><p><strong>Received:</strong> June 8, 2007; <strong>Accepted:</strong> September 21, 2007; <strong>Published:</strong> October 24, 2007</p><p><strong>Copyright:</strong> © 2007 Akashi et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research is supported by a grant from the US National Science Foundation.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Inference of ancestral and derived nucleotides within populations or among lineages is a critical step in a number of approaches to identify mechanisms of molecular evolution. Ancestral state inference has been employed to reveal episodic, or lineage-specific base composition and protein evolution <a href="#pone.0001065-Long1">[<em>e.g</em>.]</a>, <a href="#pone.0001065-Akashi1">[01]</a>, <a href="#pone.0001065-Eanes1">[02]</a>, <a href="#pone.0001065-Fitch1">[03]</a>, <a href="#pone.0001065-Messier1">[04]</a>, <a href="#pone.0001065-TakanoShimizu1">[05]</a>, <a href="#pone.0001065-Zhang1">[06, 07]</a>. In addition, several population genetic tests rely on ancestral reconstructions to reveal the action of natural selection on functional classes of mutations or in particular genetic regions <a href="#pone.0001065-Akashi2">[<em>e.g</em>.08]</a>, <a href="#pone.0001065-Akashi3">[09]</a>, <a href="#pone.0001065-Templeton1">[10]</a>, <a href="#pone.0001065-Suzuki1">[11]</a>, <a href="#pone.0001065-Fay1">[12]</a>, <a href="#pone.0001065-Williamson1">[13]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>This study addresses the accuracy of ancestral codon usage inference using the phylogenetic relationships and distances among species in the <em>Drosophila melanogaster</em> subgroup as a model tree. This group of species has been the focus of a large number of studies of mechanisms of molecular evolution. In addition, genome sequences are now available for five species in this subgroup and large-scale polymorphism studies are underway for at least two species, <em>D. melanogaster</em> and <em>D. simulans</em>. Most studies that have incorporated ancestral inference in the <em>D. melanogaster</em> subgroup have employed maximum parsimony because it is simple to implement and is assumed to be accurate among closely related lineages <a href="#pone.0001065-Akashi2">[<em>e.g</em>.08]</a>, <a href="#pone.0001065-Kliman1">[14]</a>, <a href="#pone.0001065-TakanoShimizu1">[06]</a>, <a href="#pone.0001065-Begun1">[15]</a>, <a href="#pone.0001065-DuMont1">[16]</a>, <a href="#pone.0001065-Comeron1">[17]</a>, <a href="#pone.0001065-Presgraves1">[18]</a>. However, parsimony inference can be biased under non-random character state usage (<em>i.e</em>., base composition bias) and long branch lengths <a href="#pone.0001065-Collins1">[19]</a>. Genome-wide declines of GC content have been inferred in a number of Drosophila lineages <a href="#pone.0001065-Akashi1">[02]</a>, <a href="#pone.0001065-Llopart1">[20]</a>, <a href="#pone.0001065-McVean1">[21]</a>, <a href="#pone.0001065-RodriguezTrelles1">[22]</a>, <a href="#pone.0001065-RodriguezTrelles2">[23]</a>, <a href="#pone.0001065-Begun1">[15]</a>, <a href="#pone.0001065-Begun2">[24]</a>, <a href="#pone.0001065-Bachtrog1">[25]</a>, <a href="#pone.0001065-Perez1">[26]</a>, <a href="#pone.0001065-Powell1">[27]</a>, <a href="#pone.0001065-DuMont1">[16]</a> as well as among mammals <a href="#pone.0001065-Duret1">[28]</a>, <a href="#pone.0001065-Smith1">[29]</a>, <a href="#pone.0001065-Arndt1">[30]</a>, <a href="#pone.0001065-Webster1">[31]</a>, <a href="#pone.0001065-Belle1">[32]</a>; <a href="#pone.0001065-Arndt2">[33]</a>. Support for increases in GC content in Drosophila are confined to specific genes or small genetic regions and/or a limited number of lineages <a href="#pone.0001065-TakanoShimizu1">[06]</a>, <a href="#pone.0001065-Zhang2">[34]</a>, <a href="#pone.0001065-Akashi4">[35]</a>, <a href="#pone.0001065-Ko1">[36]</a>, <a href="#pone.0001065-Maside1">[37]</a>. Some studies have employed inference methods that account for rate variation among types of nucleotide changes or among lineages, but many claims of base composition differentiation are based solely on parsimony reconstructions. Because parsimony is biased toward inference of changes from common to rare states, genes whose base composition is skewed toward GC can show apparent declines of GC in the absence of actual changes in base composition <a href="#pone.0001065-Collins1">[19]</a>, <a href="#pone.0001065-Perna1">[38]</a>, <a href="#pone.0001065-EyreWalker1">[39]</a>, <a href="#pone.0001065-Galtier1">[40]</a>, <a href="#pone.0001065-AlvarezValin1">[41]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here, we employ computer simulations to determine the reliability of ancestral codon usage inference under parsimony and likelihood approaches that have been employed in studies of Drosophila codon bias evolution. The likelihood method implements the HKY85 model <a href="#pone.0001065-Hasegawa1">[42]</a> which includes parameters for base composition bias. This study begins with a description of “major codon preference”, a model of weak selection at silent sites. Sequence data generated by computer simulations of major codon preference were used to determine the magnitude and direction of ancestral inference biases. Factors examined include levels of codon bias, branch lengths, departures from equilibrium, and the location of lineages within a phylogeny. We show that the direction and magnitude of biases in ancestral inference are dependent on all of these factors and are relevant to the study of molecular evolution among closely related species within the <em>D. melanogaster</em> subgroup. More general implications for inferring ancestral sequences under asymmetric and/or fluctuating substitution rates are discussed.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods and Results"></a><h3>Methods and Results</h3>
<h4>Major codon preference</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Synonymous codon usage appears to evolve under a balance among weak evolutionary forces in Drosophila and in a wide range of taxa <a href="#pone.0001065-Ikemura1">[reviewed in 43]</a>, <a href="#pone.0001065-Shields1">[44]</a>, <a href="#pone.0001065-Andersson1">[45]</a>, <a href="#pone.0001065-Sharp1">[46]</a>, <a href="#pone.0001065-Akashi5">[47]</a>, <a href="#pone.0001065-Duret2">[48]</a>. Codons that are used preferentially in highly expressed genes tend to be recognized by abundant tRNA isoacceptors <a href="#pone.0001065-Ikemura1">[43]</a>, <a href="#pone.0001065-Kanaya1">[49]</a>, <a href="#pone.0001065-Kanaya2">[50]</a>, <a href="#pone.0001065-Moriyama1">[51]</a>, <a href="#pone.0001065-Duret3">[52]</a>, <a href="#pone.0001065-Wright1">[53]</a>. Such codons are termed “preferred” or “major” codons. Major codon usage is defined as the overall percentage of major codons at redundant codons in a gene, MCU = #major codons/(#major+#minor codons). MCU is positively correlated with gene expression levels <a href="#pone.0001065-Akashi5">[reviewed in 47]</a> and biochemical studies have shown faster and more accurate translation at major codons than at other codons that encode the same amino acid, termed “minor” or “unpreferred” codons <a href="#pone.0001065-Andersson1">[reviewed in 45]</a>, <a href="#pone.0001065-Kurland1">[54]</a>. Major codon preference posits that translationally superior codons confer a fitness advantage sufficient to bias codon usage (especially in highly expressed genes) but small enough to allow minor codons to persist through mutation pressure and genetic drift. Within-and between-species comparisons of silent mutations in a number of Drosophila species are consistent with small fitness benefits to major codon usage <a href="#pone.0001065-Akashi2">[08]</a>, <a href="#pone.0001065-Akashi6">[55]</a>, <a href="#pone.0001065-Kliman1">[14]</a>, <a href="#pone.0001065-Llopart2">[56]</a>, <a href="#pone.0001065-Begun1">[15]</a>, <a href="#pone.0001065-Perez1">[26]</a>, <a href="#pone.0001065-Maside2">[57]</a>, <a href="#pone.0001065-Bartolome1">[58]</a>, <a href="#pone.0001065-Comeron1">[17]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Under major codon preference, the evolutionary dynamics of synonymous changes are determined by the joint effects of mutation, genetic drift, and weak selection. Consider a two-fold redundant codon where A<sub>1</sub> represents a major codon (relative fitness = 1) and A<sub>2</sub> represents a minor codon (relative fitness = 1−<em>s</em>). Mutations may occur at different rates <em>u</em>, from A<sub>1</sub> to A<sub>2</sub>, and <em>v</em>, in the reverse direction. A<sub>1</sub> to A<sub>2 </sub>changes will be designated “<em>pu</em>” (for preferred to unpreferred) and A<sub>2</sub> to A<sub>1 </sub>changes will be referred to as “<em>up</em>” (for unpreferred to preferred). At a locus consisting of a number of such codons, the frequency of major codons at the locus, MCU, will reach a steady state if <em>u</em>/<em>v</em> and <em>N</em><sub>e</sub><em>s</em> remain constant over a large number of generations (see <a href="#pone.0001065.s001">Methods S1</a> for details). However, small changes in parameter values can cause departures from steady-state that are both strong and long-lasting; new equilibria are approached on a time scale on the order of the reciprocal of <em>per</em> site mutation rate (10<sup>8</sup>–10<sup>9</sup> generations in Drosophila). If changes in mutation rates or selection intensity occur on a faster time scale than the approach to equilibrium, codon usage may rarely be at steady-state.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Identifying lineage-specific codon bias changes and their causes is important to our understanding of evolution at synonymous sites and may help to reveal mechanisms of protein evolution <a href="#pone.0001065-Akashi1">[02]</a>. <a href="#pone-0001065-g001">Figures 1A and C</a> show expected rates of <em>up</em> and <em>pu</em> substitutions following three-fold decreases and two-fold increases in <em>N</em><sub>e</sub>, respectively. These <em>N</em><sub>e</sub> ratios give approximately two-fold excesses of <em>pu</em> and <em>up</em> changes, respectively, for a gene with initial MCU = 0.7. Changes in <em>N</em><sub>e</sub> are assumed to be constant across loci (<em>i.e</em>., in <a href="#pone-0001065-g001">Figures 1A and B</a>, all loci experience a three-fold decrease from their initial <em>N</em><sub>e</sub><em>s</em> values). A measure of skew, <em>d</em><sub>up,pu</sub> = (#<em>up</em>–#<em>pu</em>)/(#<em>up</em>+#<em>pu</em>), is employed as an index of the direction and magnitude of departures from steady-state. Under non-stationary <em>N</em><sub>e</sub><em>s</em>, expected <em>d</em><sub>up,pu</sub> varies considerably as a function of MCU.</p>
<div class="figure" id="pone-0001065-g001"><div class="img"><a name="pone-0001065-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Silent divergence under departures from equilibrium major codon usage.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The x-axis shows MCU values prior to a change in <em>N</em><sub>e</sub>. A: expected instantaneous <em>per</em> locus rates of <em>pu</em> and <em>up</em> fixations when <em>N</em><sub>e</sub> decreases to 1/3 its original value across genes. B: expected <em>d</em><sub>up,pu</sub> = (#<em>up</em>−#<em>pu</em>)/(#<em>up</em>+#<em>pu</em>) for the 1/3<em>N</em><sub>e</sub> scenario (decreasing codon bias). C and D: expected instantaneous silent rates and <em>d</em><sub>up,pu</sub> after a doubling of <em>N</em><sub>e</sub> (increasing codon bias). Legends and X-axis scales apply to graphs in the same column. See text for details of the model. The curves assume that variation in selection coefficients underlies MCU variation among genes (<em>u/v</em> = 1 across genes).</p>
<span>doi:10.1371/journal.pone.0001065.g001</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Mutational variation can also cause genome-wide changes in codon bias. Because fixation rates are linearly dependent on mutation rates, the <em>pu</em>:<em>up</em> substitution ratio will reflect the altered ratio of <em>per</em> locus mutation rates. If codon bias variation among lineages is caused by uniform parameter changes, then comparisons of <em>d</em><sub>up,pu</sub> among codons experiencing different selection intensities (<em>i.e</em>., codons in different genes or regions within genes or in different synonymous families) can identify whether changes in scaled selection coefficients or mutational biases underlie codon bias differences. It should be noted, however, that the predictions above are based on a model that does not account for ancestral polymorphism. When <em>N</em><sub>e</sub><em>u</em> is large, the frequency distribution of silent polymorphisms may need to be considered when predicting <em>up</em>:<em>pu</em> ratios following parameter changes. McVean and Charlesworth <a href="#pone.0001065-McVean2">[59]</a> showed that equilibrium MCU predictions are robust to polymorphism, but this result has not been confirmed for non-stationary evolution.</p>


<h4>Simulations of codon bias evolution</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Computer simulations were employed to test the performance of ancestral reconstruction methods with a focus on inference of non-stationary codon bias and its causes in the <em>D. melanogaster</em> subgroup.</p>
<h5>Tree topology and branch lengths.</h5><a id="article1.body1.sec2.sec2.sec1.p1" name="article1.body1.sec2.sec2.sec1.p1"></a><p>Data were simulated for six extant nodes given a topology and branch lengths set to estimates for the <em>D</em>. <em>melanogaster</em> subgroup (<a href="#pone-0001065-g002">Figure 2</a>). This tree topology is strongly supported in analyses of nuclear genes using sequences from relatively closely related outgroup species from the <em>takahashii</em> and <em>suzukii</em> subgroups <a href="#pone.0001065-Ko2">[60]</a>, <a href="#pone.0001065-Wong1">[61]</a>. Analyses of much larger numbers of loci (but with more distantly related outgroups) are generally consistent with this tree, but also support gene-specific topologies that vary with respect to the placement of the <em>D. yakuba</em> and <em>D. erecta</em> clades <a href="#pone.0001065-Pollard1">[62]</a>.</p>
<div class="figure" id="pone-0001065-g002"><div class="img"><a name="pone-0001065-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Synonymous distance trees for six <em>Drosophila melanogaster</em> subgroup species.</span></strong></p><a id="article1.body1.sec2.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec2.sec1.fig1.caption1.p1"></a><p><em>m, s, t, y, e,</em> and <em>o</em> refer to <em>D. melanogaster, D. simulans, D. teissieri, D. yakuba, D. erecta,</em> and <em>D. orena</em>, respectively. The assumed tree topology ((<em>m, s</em>), ((<em>t, y</em>), (<em>e, o</em>))) is based on <a href="#pone.0001065-Ko2">[60]</a>. Silent distances were calculated using CODEML <a href="#pone.0001065-Yang1">[63]</a> and averaged across 22 genes. (ML) unrooted tree showing maximum likelihood distances under a codon-based substitution model. Equilibrium frequencies of each codon were calculated from the nucleotides frequencies at three codon positions (F3x4). (NG) unrooted neighbor-joining tree based on Nei-Gojobori <a href="#pone.0001065-Nei1">[64]</a> pairwise distances. Numbers shown on each branch are per site synonymous distances x1000. (1x) topology employed in 1x simulations. Mutation rates and numbers of generations per branch were set to give expected <em>per</em> site silent divergence of 0.05 for the <em>m, s, t, y, e, o, ty</em>, and <em>eo</em> lineages and 0.075 and 0.025 for the <em>ms</em> and <em>tyeo</em> lineages, respectively, for equilibrium MCU = 0.7. Abbreviations for ancestral nodes are shown below and to the right of the nodes.</p>
<span>doi:10.1371/journal.pone.0001065.g002</span></div><a id="article1.body1.sec2.sec2.sec1.p2" name="article1.body1.sec2.sec2.sec1.p2"></a><p>Average synonymous divergence is shown for six species in the <em>D. melanogaster</em> subgroup in <a href="#pone-0001065-g002">Figure 2</a>. The branch lengths in the “1x” tree in <a href="#pone-0001065-g002">Figure 2</a> were employed in the simulations. Extant nodes on the simulated tree will be referred to by the first letter of the species' name. Internal nodes are named according to their child nodes (<em>i.e</em>., the node connecting <em>t</em> and <em>y</em> is “<em>ty</em>” and the node connecting <em>ty</em> and <em>eo</em> is “<em>tyeo</em>”). Lineages will be referred to by the name of the upper node (<em>i.e</em>., “<em>m</em>” refers to the branch connecting the <em>ms</em> and <em>m</em> nodes and “<em>ms”</em> refers to the branch between the <em>mstyeo</em> and <em>ms</em> nodes). Branch lengths were chosen to give a symmetric (unrooted) tree with silent divergence levels between maximum likelihood <a href="#pone.0001065-Yang1">[63]</a> and Nei-Gojobori (NG) <a href="#pone.0001065-Nei1">[64]</a> estimates (see <a href="#pone-0001065-g002">Figure 2</a> legend for details). Although data were simulated for the ancestral lineage (<em>mstyeo</em>) as well as the ten lineages in the 1x tree shown in <a href="#pone-0001065-g002">Figure 2</a>, ancestral states were reconstructed on an unrooted tree. Thus, silent changes were assigned to eight lineages: <em>m</em>, <em>s</em>, <em>t</em>, <em>y</em>, <em>ty</em>, <em>e</em>, <em>o</em>, and <em>eo</em>.</p>

<h5>Simulating sequence evolution.</h5><a id="article1.body1.sec2.sec2.sec2.p1" name="article1.body1.sec2.sec2.sec2.p1"></a><p>Nucleotide changes were simulated for sequences with random amino acid usage (<em>i.e</em>., each amino acid was represented in the sequence in proportion to its level of redundancy in the standard genetic code). Sequence lengths were 4,941 (61×81) codons. Only synonymous changes at third codon positions were simulated (six-fold redundant codons evolved as two-fold or four-fold families). For each codon, state-transition or “substitution” probabilities to other synonymous codons were calculated by multiplying a <em>per</em> site <em>per</em> generation mutation rate of 2×10<sup>−5</sup> and the fixation probability for the mutation. Codon changes in the simulated trees will be referred to as “fixations” or “substitutions” although population data were not simulated. Equal mutation rates were assumed among all nucleotides. All G- and C-ending synonymous codons were assigned a fitness of 1 and all A- and T-ending codons were assigned a fitness of 1-<em>s</em>. <em>N</em><sub>e</sub> was set to 5,000 diploid individuals and values of <em>s</em> were assigned to give the following equilibrium MCU values: 0.5, 0.6, 0.7, 0.8, 0.9, and 0.95 (in these simulations, MCU is equivalent to %GC). Nucleotide ambiguity codes S and W will be used to refer to G or C and A or T, respectively. This substitution model will be referred to as the “GCpref codon” model.</p>
<a id="article1.body1.sec2.sec2.sec2.p2" name="article1.body1.sec2.sec2.sec2.p2"></a><p>Variable parameters in the simulations were: the numbers of generations on each branch, mutation rates, the fitness advantage of G- or C-ending codons, <em>s</em>, and effective population size, <em>N</em><sub>e</sub>. The tree topology was not varied among simulations. All simulations were initiated with a “burn-in” period of ≥2/<em>u</em> generations to insure independence among replicates (longer burn-in periods were used for high MCU scenarios). A total tree length of 9,000 generations from the <em>mstyeo</em> node gave distances of approximately 5% synonymous divergence on the <em>m</em>, <em>s</em>, <em>t</em>, <em>y</em>, <em>e</em>, <em>o</em>, <em>ty</em>, and <em>eo</em> lineages for the equilibrium MCU = 0.7 case. Simulations under this set of parameter values will be referred to as the “1x” scenario. Data were also produced for trees with the same topology but double (2×) and half (0.5×) the numbers of generations on each branch.</p>
<a id="article1.body1.sec2.sec2.sec2.p3" name="article1.body1.sec2.sec2.sec2.p3"></a><p>Non-equilibrium codon usage was simulated by adjusting effective population size or mutation parameters at the <em>mstyeo</em> node following the burn-in. Both increasing codon bias (larger <em>N</em><sub>e</sub> or reduced <em>u</em>/<em>v</em>) and decreasing bias (smaller <em>N</em><sub>e</sub> or elevated <em>u</em>/<em>v</em>) were simulated. Values of 2× and 1/3× the initial <em>N</em><sub>e</sub> and 1/2× and 2× the initial <em>u</em>/<em>v</em> were chosen to give <em>pu</em>:<em>up</em> fixation ratios of approximately half and double the equilibrium values for an initial MCU of 0.7. All parameters were held constant following the change at the ancestral node. For all simulations, the lineage and fitness class of each fixation was recorded and the resulting extant sequences were stored. Extant sequences were employed to generate inferences of the numbers and types of changes for comparisons to the numbers of “actual” (simulated) changes.</p>



<h4>Reliability of Ancestral Inference</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Simulated sequence data were analyzed using BASEML in the PAML software package (Version 2.0k) <a href="#pone.0001065-Yang1">[63]</a>. This program takes a gene tree and sequence data as input and determines maximum likelihood estimates for branch lengths and parameters of a substitution model. These MLEs are employed in calculations of the posterior probabilities of joint reconstructions of nucleotides at ancestral nodes <a href="#pone.0001065-Yang2">[65]</a>. Two substitution models were employed for inferences of ancestral states among simulated sequences. Maximum parsimony (MP) inference was emulated using a Jukes-Cantor <a href="#pone.0001065-Jukes1">[66]</a> one parameter model with all branch lengths set to small and equal values. Under this model, all nucleotide changes occur at equal rates and the set of ancestral nucleotides (joint reconstruction) that requires the fewest number of changes in the tree has the highest probability. A unique joint reconstruction requiring the lowest number of changes was assumed to reflect the true ancestral states. This procedure emulates “simple” or equally-weighted parsimony. In cases where multiple best reconstructions had the same probability (<em>i.e</em>., the same number of changes), the codon was not included in the analysis. <em>Up</em> and <em>pu</em> counts inferred using this procedure were compared to results from an iterative method of parsimony inference for 5 replicates each for initial MCU = 0.5, 0.7, and 0.9 for stationary (1x), variable <em>N</em><sub>e</sub> (1/3<em>N</em><sub>e</sub>, 2<em>N</em><sub>e</sub>) and variable mutation (2<em>u</em>/<em>v</em>, 0.5<em>u</em>/<em>v)</em> simulations scenarios (described below). Inferred numbers of changes were identical in all cases. Simple parsimony inference was chosen for this analysis because it has been employed extensively in Drosophila studies <a href="#pone.0001065-Akashi2">[<em>e.g</em>.08]</a>, <a href="#pone.0001065-Akashi1">[02]</a>, <a href="#pone.0001065-Kliman1">[14]</a>, <a href="#pone.0001065-Begun1">[15]</a>, <a href="#pone.0001065-TakanoShimizu1">[06]</a>, <a href="#pone.0001065-Begun2">[24]</a>, <a href="#pone.0001065-Bachtrog1">[25]</a>, <a href="#pone.0001065-Perez1">[26]</a>, <a href="#pone.0001065-DuMont1">[16]</a>, <a href="#pone.0001065-Presgraves1">[18]</a>.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The HKY85 substitution model <a href="#pone.0001065-Hasegawa1">[42]</a> was also employed to infer ancestral codons using BASEML. This model incorporates unequal base composition and different transition and transversion rates. Four base composition parameters are estimated from the average base composition of the extant sequences. A transition/transversion rate ratio and branch lengths are estimated by maximizing their likelihoods over the data. The approach is similar to one employed by Galtier and Boursot <a href="#pone.0001065-Galtier1">[40]</a> except that the BASEML model does not include a rate heterogeneity parameter and calculates substitution parameters separately for first, second, and third codon positions (in our simulations, only the third codon position is variable). It is important to note a difference in the parameterization of the HKY85 and GCpref codon substitution models; under HKY85, transversions to the same nucleotide are assigned the same rate (discussed below). The joint probabilities of codon reconstructions were calculated assuming independence among nucleotide reconstructions at different sites and a minimal evolution model was used to infer codon changes between nodes in the phylogeny. The probability of a given ancestral codon configuration was treated as the “count” of inferred substitutions on a given reconstruction. For example, consider a case where a configuration [G, A, G, G, G, G] for extant nodes [<em>m</em>, <em>s</em>, <em>t</em>, <em>y</em>, <em>e</em>, <em>o</em>] gave inferred ancestral configurations [G, G, G, G] and [A, G, G, G] for ancestral nodes [<em>ms</em>, <em>tyeo</em>, <em>ty</em>, <em>eo</em>] with probabilities 0.9 and 0.1, respectively. For this site, 0.9 G-&gt;A changes were recorded in the <em>s</em> lineage and 0.1 A-&gt;G changes were recorded in the <em>m</em> lineage. The latter change also requires either a G-&gt;A change in the <em>ms</em> lineage or an A-&gt;G change in the <em>tyeo</em> lineage but reconstructions were performed on an unrooted tree. This inference method will be referred to as “ML”. It is important to note that this method averages over possible ancestral reconstructions given their probabilities under the HKY85 substitution model and differs from ML implementations that treat the most probable reconstruction as pseudodata. The latter can yield strongly biased ancestral nucleotide frequencies for scenarios similar to those considered here <a href="#pone.0001065-Krishnan1">[67]</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>MP and ML inferred codon bias changes were compared across 200–500 replicates for each scenario (numbers for each scenario are given in the figure legends). Mantel-Haentzel tests <a href="#pone.0001065-Snedecor1">[68]</a> were employed to detect consistent differences in ratios of MP and ML inferred counts of <em>up</em> and <em>pu</em> changes (summed across synonymous families). Wilcoxon ranked signs tests <a href="#pone.0001065-Snedecor1">[68]</a> were conducted to determine if differences between actual and inferred <em>d</em><sub>uppu</sub> were consistently smaller for MP or ML. We refer to a method as “more reliable” if inferred <em>up</em>/<em>pu</em> ratios differ between MP and ML (Mantel-Haentzel test, <em>P</em>&lt;0.05) and <em>d</em><sub>uppu</sub> values are closer to actual values (Wilcoxon ranked signs test, <em>P</em>&lt;0.05) for one of the methods. Computer programs for simulating sequence evolution and for analyses of BASEML results were written in the C computer language and are available upon request from HA.</p>
<h5>Parsimony vs likelihood.</h5><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1"></a><p><a href="#pone-0001065-g003">Figures 3A and B</a> show the actual and inferred numbers of <em>pu</em> and <em>up</em> changes in the <em>m</em> lineage under a simulated equilibrium 1x tree. S-&gt;W changes are pooled in the <em>pu</em> class and W-&gt;S changes are pooled in the <em>up</em> class. Data for changes within preference classes (S-&gt;S and W-&gt;W) are not shown. Both MP and ML underestimate the numbers of changes and the underestimation is greater for MP inference, especially for <em>up</em> changes at high MCU. MP biases are similar to those described in a number of previous studies <a href="#pone.0001065-Collins1">[19]</a>, <a href="#pone.0001065-Perna1">[38]</a>, <a href="#pone.0001065-EyreWalker1">[39]</a>, <a href="#pone.0001065-Galtier1">[40]</a>, <a href="#pone.0001065-AlvarezValin1">[41]</a>. <a href="#pone-0001065-g003">Figures 3C and D</a> depict the ratio of inferred to actual numbers of changes and show that, for both methods, underestimation decreases with MCU for <em>pu</em> changes and increases with MCU for <em>up</em> changes. This results in a bias toward negative <em>d</em><sub>up,up</sub> (inflated inference of the ratio of <em>pu</em> to <em>up</em> fixations; <a href="#pone-0001065-g003">Figure 3E</a>) which increases with MCU for both MP and ML. The magnitude of the bias is considerably greater for parsimony. Under ML, inferred <em>pu</em>:<em>up</em> ratios are inflated by 4, 8, and 13% for MCU values of 0.5, 0.7, and 0.9, whereas for parsimony, <em>pu</em>:<em>up</em> is inflated by 16, 28, and 47% for the same MCU values. Even among closely related lineages at equilibrium base composition, biases in ancestral inference can generate patterns consistent with a genome-wide decline of MCU. Furthermore, the decrease in <em>d</em><sub>up,pu</sub> as a function of MCU mimics the expected trend following a reduction in <em>N</em><sub>e</sub><em>s</em> (<a href="#pone-0001065-g001">Figure 1B</a>).</p>
<div class="figure" id="pone-0001065-g003"><div class="img"><a name="pone-0001065-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Inference of <em>pu</em> and <em>up</em> substitutions on the <em>m</em> lineage under equilibrium codon bias evolution.</span></strong></p><a id="article1.body1.sec2.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec3.sec1.fig1.caption1.p1"></a><p>The numbers of hits and the ratios of inferred to actual hits reflect averages across 300 replicates of the 1x equilibrium scenario. X-axis scales apply to graphs in the same column. The legend in A applies to B and E and the legend in C applies to D. Reliability of <em>d</em><sub>up,pu</sub> inference was greater for ML than for MP for MCU≥0.6 (see text for criteria).</p>
<span>doi:10.1371/journal.pone.0001065.g003</span></div><a id="article1.body1.sec2.sec3.sec1.p2" name="article1.body1.sec2.sec3.sec1.p2"></a><p>Examples of extant and ancestral codon configurations illustrate some of the causes of these inference biases. Under neutrality (MCU = 0.5), the probabilities of <em>pu</em> and <em>up</em> fixations are equal. MP underestimation of the numbers of changes (<a href="#pone-0001065-g003">Figure 3A and B</a>) partly reflects the absence of inference in cases of multiple most parsimonious reconstructions. Greater inference bias as a function of MCU reflects a combination of differences in fixation probabilities and <em>per</em> locus mutation rates for <em>pu</em> and <em>up</em> changes. <a href="#pone-0001065-g004">Figure 4B–D</a> shows several ancestral codon configurations (ACCs) that could underlie an extant codon configuration (ECC) that MP would infer as a single <em>pu</em> change in the <em>m</em> lineage. The preference states of codons present in the extant nodes [<em>m</em>, <em>s</em>, <em>t</em>, <em>y</em>, <em>e</em>, and <em>o</em>] are [u, p, p, p, p, p] respectively (<a href="#pone-0001065-g004">Figure 4A</a>). Such an extant codon configuration will be abbreviated “ECC_uppppp” assuming all “p” states are identical. In the 1x equilibrium simulations, three ACCs underlie over 98% of ECC_uppppp's (<a href="#pone-0001065-t001">Table 1</a>). For MCU = 0.5, codons that underwent a single <em>pu</em> in the <em>m</em> lineage are the predominant scenario (94.6%), and double-hit codons with a <em>up</em> change in <em>s</em> and either a <em>pu</em> change in <em>ms</em> or a <em>up</em> change in <em>tyeo</em> underlie 3.5% of the configurations. In all cases, parsimony infers a single <em>up</em> change in the <em>m</em> lineage. For MCU = 0.5, the expected frequencies of each of the pairs of ACC types B and F, C and G, and D and H are equal since <em>pu</em> and <em>up</em> changes have the same rates [ECC_puuuuu and its ancestral codon configurations exchange p and u states at all nodes with those in ECC_uppppp (<a href="#pone-0001065-g004">Figures 4E–H</a>)]. Under the symmetric model of no selection and equal mutation rates, inference errors cancel. The scenarios depicted in <a href="#pone-0001065-g004">Figures 4C and G</a> will be referred to more generally as “child/ancestor reverse” changes (a child lineage is a direct descendent of an ancestral lineage; in these cases, <em>m</em> and <em>s</em> are the child lineages and <em>ms</em> is the ancestor) and those in <a href="#pone-0001065-g004">Figures 4D and H</a> will be referred to as “child/sib-ancestor parallel” changes (in these cases, <em>m</em> and <em>s</em> are child lineage and <em>tyeo</em> is the sibling to the ancestral <em>ms</em> lineage).</p>
<div class="figure" id="pone-0001065-g004"><div class="img"><a name="pone-0001065-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Ancestral codon configurations in simulations of codon bias evolution.</span></strong></p><a id="article1.body1.sec2.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec3.sec1.fig2.caption1.p1"></a><p>Trees representing extant codon configurations consistent with single silent changes in the <em>m</em> lineage, ECC_uppppp (A) and ECC_puuuuu (E), are shown. The three most common ancestral codon configurations underlying these extant codon configurations are shown in B, C, and D for ECC_uppppp and in E, F, and G for ECC_puuuuu. Trees C and G reflect child/ancestor reverse changes and trees D and H show child/sib-ancestor parallel changes. The relative frequencies of ancestral codon configurations underlying ECC_uppppp and ECC_puuuuu in the codon bias simulations are shown as bubble plots beneath the trees. The sizes of the bubbles reflect the relative numbers of ancestral codon configurations in each class for three different MCU values. For the non-equilibrium scenarios (1/3<em>N</em><sub>e</sub> and 2<em>N</em><sub>e</sub>), the proportion of each ancestral codon configuration among the extant configurations relative to the proportion under equilibrium codon bias evolution are given. The data are from <a href="#pone-0001065-t001">Table 1</a>.</p>
<span>doi:10.1371/journal.pone.0001065.g004</span></div><div class="figure" id="pone-0001065-t001"><div class="img"><a name="pone-0001065-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001065.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001065.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Ancestral codon configurations underlying ECC_uppppp and ECC_puuuuu trees</span></strong></p><span>doi:10.1371/journal.pone.0001065.t001</span></div><a id="article1.body1.sec2.sec3.sec1.p3" name="article1.body1.sec2.sec3.sec1.p3"></a><p>An important difference between the ECC_puuuuu and ECC_uppppp scenarios emerges when selection causes rate variation between mutation classes. For MCU = 0.9, ECC_uppppp's that result from multiply-hit codons with a <em>up</em> change in <em>s</em> (<a href="#pone-0001065-g004">Figures 4C and D</a>) increase to over 7% of the observations, whereas the fraction of ECC_puuuuu's resulting from a <em>pu</em> change in <em>s</em> at a multiply-hit codon (<a href="#pone-0001065-g004">Figures 4G and H</a>) <em>decreases</em> with MCU to less than 2% (<a href="#pone-0001065-t001">Table 1</a>). The asymmetry causes MP to overestimate the numbers of low probability (<em>pu</em>) changes, and underestimate the numbers of high probability (<em>up</em>) changes. This bias increases with the difference in the probabilities (<em>i.e</em>., with MCU). In addition, misinference of <em>up</em> changes in the <em>m</em> lineage following <em>pu</em> changes in <em>ms</em> (resulting in ECC_pupppp) as <em>pu</em> changes in <em>s</em> will further contribute to overestimation of <em>pu</em> in <em>m</em> and underestimation of <em>up</em> in <em>s</em>. Finally, MP inference attributes parallel changes in sibling lineages to single changes in the ancestral lineage. Among codons that experienced a <em>pu</em> change, the fraction that experienced parallel substitutions (another <em>pu</em> in a different lineage) decreases with MCU as selection reduces the probability of <em>pu</em> fixations. The proportion of multiple-<em>up</em> hit codons increases with MCU as selection elevates their fixation probabilities. This asymmetry contributes to parsimony underestimation of <em>d</em><sub>up,pu</sub>.</p>

<h5>Branch lengths.</h5><a id="article1.body1.sec2.sec3.sec2.p1" name="article1.body1.sec2.sec3.sec2.p1"></a><p>Reliability of ancestral inference is strongly dependent on distances among nodes <a href="#pone.0001065-Zhang3">[69]</a>. Simulations of equilibrium codon bias were conducted for a tree with the same topology as shown in <a href="#pone-0001065-g002">Figure 2</a> but with varying branch lengths. A 0.5× tree (half the 1× numbers of generations on all branches) and a 2× tree (double the 1× numbers of generations on all branches) were examined. Inference biases are smaller in the shorter tree (<a href="#pone-0001065-g005">Figure 5C</a>) and considerably greater in a 2× tree (<a href="#pone-0001065-g005">Figure 5F</a>). The MP bias in <em>d</em><sub>up,pu</sub> inference as a function of MCU in the 2× scenario is dramatic and reflects a striking underestimation of <em>up</em> changes (inferred values &lt;50% of the actual numbers for MCU&gt;0.8). These patterns reflect the increase in the proportions of multiply-hit codons with longer branch lengths (1.7, 5.7, and 16.7% of codons experienced &gt;1 change since the <em>mstyeo</em> node in the 0.5×, 1×, and 2× scenarios, respectively for MCU = 0.7).</p>
<div class="figure" id="pone-0001065-g005"><div class="img"><a name="pone-0001065-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Reliability of ancestral codon bias inference under equilibrium evolution.</span></strong></p><a id="article1.body1.sec2.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec3.sec2.fig1.caption1.p1"></a><p>A, B, and C show ratios of inferred to actual values and the <em>d</em><sub>up,pu</sub> for the 0.5× tree (averages among 500 replicates) and D, E, and F show values for the 2× tree (averages among 200 replicates). Actual and inferred numbers of hits are not shown. The legend in A applies to B, D, and E and the legend in C applies to F. X-axis scales are identical for graphs in the same column.</p>
<span>doi:10.1371/journal.pone.0001065.g005</span></div>
<h5>ML ancestral reconstruction bias.</h5><a id="article1.body1.sec2.sec3.sec3.p1" name="article1.body1.sec2.sec3.sec3.p1"></a><p>ML inference, though more reliable than MP, also shows a bias toward negative <em>d</em><sub>up,pu</sub> with MCU (especially in the 2× tree; <a href="#pone-0001065-g005">Figure 5F</a>). Under the HKY85 model, four base composition parameters [<strong><em>π</em></strong>  =  (<em>π</em><sub>A</sub>, <em>π</em><sub>T</sub>, <em>π</em><sub>C</sub>, <em>π</em><sub>G</sub>)] are estimated from extant sequences and the transition/transversion rate ratio (<em>κ</em>) is estimated from patterns of sequence divergence. Because the parameters are calculated separately for the three codon positions and because the simulated scenario is one of equilibrium base composition, ancestral inference should be more accurate than MP. Differences in the parameterizations of the HKY85 and GCpref codon models appear to underlie the observed ML biases. Under HKY85, transversion changes to the same nucleotide occur at the same rate. For example, C-&gt;A and T-&gt;A occur at rate <em>π</em><sub>A</sub> and A-&gt;C and G-&gt;C occur at rate <em>π</em><sub>C</sub>. Under the GCpref codon model, transversions to the same nucleotide consist of one mutation affecting fitness (C-&gt;A is unpreferred and A-&gt;C is preferred) and one neutral mutation (within a fitness class: T-&gt;A and G-&gt;C). If a single rate is estimated for transversions to A, the C-&gt;A (unpreferred) rate will be overestimated and the T-&gt;A (neutral) rate will be underestimated. For transversions to C, the A-&gt;C rate will be underestimated and the G-&gt;C rate will be overestimated. Overall, the rates of preferred and unpreferred transversions will be underestimated and overestimated, respectively, and this error will increase with MCU. This leads to underestimation of <em>d</em><sub>up,pu</sub> with MCU, a bias similar to MP. ML inference for data simulated under the exact parameterization of the HKY85 model showed no bias in <em>d</em><sub>up,pu</sub>, even under strong base composition bias in a 2× tree (<a href="#pone-0001065-g006">Figure 6</a>). Seemingly minor deviations from model assumptions can lead to considerable biases for relatively highly biased genes and/or long branches.</p>
<div class="figure" id="pone-0001065-g006"><div class="img"><a name="pone-0001065-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Reliability of ML inference for evolution under the GCpref codon and HKY85 models.</span></strong></p><a id="article1.body1.sec2.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.sec3.fig1.caption1.p1"></a><p><em>d</em><sub>up,pu</sub> values are shown for 4-fold synonymous codons for ML inference under the equilibrium GCpref codon model and for data simulated under the HKY85 substitution matrix. The latter matrix was set to give identical expected numbers of substitutions and equilibrium GC content for the two scenarios. The legend applies to both graphs and the y-axis scales are identical in the two graphs. Note that <em>d</em><sub>up,pu</sub> inference biases are larger for 4-fold redundant codons than for 2-fold redundant codons under the GCpref codon model. Data are averaged across 300 replicates.</p>
<span>doi:10.1371/journal.pone.0001065.g006</span></div>
<h5>Changes in selection intensity.</h5><a id="article1.body1.sec2.sec3.sec4.p1" name="article1.body1.sec2.sec3.sec4.p1"></a><p>Constancy of parameters governing molecular evolution may often be violated in Drosophila among closely related lineages <a href="#pone.0001065-Akashi2">[08]</a>, <a href="#pone.0001065-Akashi1">[02]</a>, <a href="#pone.0001065-Llopart1">[20]</a>, <a href="#pone.0001065-McVean1">[21]</a>, <a href="#pone.0001065-TakanoShimizu1">[06]</a>, <a href="#pone.0001065-TakanoShimizu2">[70]</a>, <a href="#pone.0001065-Begun1">[15]</a>, <a href="#pone.0001065-Begun2">[24]</a>, <a href="#pone.0001065-Bachtrog1">[25]</a>, <a href="#pone.0001065-Perez1">[26]</a>, <a href="#pone.0001065-DuMont1">[16]</a>, <a href="#pone.0001065-Akashi4">[35]</a>, <a href="#pone.0001065-Maside1">[37]</a> as well as among distantly related taxa <a href="#pone.0001065-RodriguezTrelles3">[71]</a>, <a href="#pone.0001065-RodriguezTrelles1">[22]</a>, <a href="#pone.0001065-RodriguezTrelles2">[23]</a>, <a href="#pone.0001065-Powell1">[27]</a>. Ancestral inference under fluctuating codon bias is examined below.</p>
<a id="article1.body1.sec2.sec3.sec4.p2" name="article1.body1.sec2.sec3.sec4.p2"></a><p>Two scenarios of changes in selection intensity were examined: a three-fold decrease in <em>N</em><sub>e</sub> (1/3<em>N</em><sub>e</sub>) and a two-fold increase in <em>N</em><sub>e</sub> (2<em>N</em><sub>e</sub>). Parameter changes were invoked at the <em>mstyeo</em> node and substitution probabilities differing from those in the <em>mstyeo</em> lineage were employed for evolution within the <em>D. melanogaster</em> subgroup. These probabilities were kept constant within the subgroup. However, because base composition changes directionally during the course of the simulation, <em>per</em> locus mutation rates are not constant.</p>
<a id="article1.body1.sec2.sec3.sec4.p3" name="article1.body1.sec2.sec3.sec4.p3"></a><p>For the 1/3<em>N</em><sub>e</sub> scenario (GC content decline), MP and ML show contrasting biases in <em>d</em><sub>up,pu</sub> inference (<a href="#pone-0001065-g007">Figures 7A to C</a>). Parsimony biases are similar to the equilibrium case; both <em>pu</em> and <em>up</em> changes are underestimated, but the magnitude of <em>pu</em> underestimation <em>decreases</em> with MCU whereas <em>up</em> underestimation <em>increases</em> as a function of codon bias. <em>d</em><sub>up,pu</sub> is biased downward and the bias increases with MCU in a manner similar to the equilibrium case. Because the actual <em>d</em><sub>up,pu</sub> is negative (<a href="#pone-0001065-g001">Figure 1B</a>), MP exaggerates reductions of MCU.</p>
<div class="figure" id="pone-0001065-g007"><div class="img"><a name="pone-0001065-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Reliability of ancestral codon bias inference under non-equilibrium evolution: variable selection intensity.</span></strong></p><a id="article1.body1.sec2.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec3.sec4.fig1.caption1.p1"></a><p>The legend in A applies to B, D, and E. The legend in C also applies to F. X-axis scales are identical for graphs in the same column. Note that the MCU values reflect values at the <em>m</em> node and differ from ancestral values. For the 1/3<em>N</em><sub>e</sub> scenario, reliability of <em>d</em><sub>up,pu</sub> inference was greater for ML than for MP for 0.6≤MCU≤0.8. For 2<em>N</em><sub>e</sub>, ML was more reliable for MCU≥0.8 (see text for criteria). Data are averaged across 300 replicates.</p>
<span>doi:10.1371/journal.pone.0001065.g007</span></div><a id="article1.body1.sec2.sec3.sec4.p4" name="article1.body1.sec2.sec3.sec4.p4"></a><p>Examination of extant and ancestral codon configurations reveals differences in the causes of similar MP reconstruction biases under equilibrium and decreasing codon bias. Under reduced <em>N</em><sub>e</sub>, the ratio of fixation probabilities of <em>pu</em> and <em>up</em> mutations decreases less as a function of MCU than under stationarity. <a href="#pone-0001065-t001">Table 1</a> and <a href="#pone-0001065-g004">Figure 4</a> show that, among ECC_uppppp's, the frequencies of multiply-hit codons do not increase with MCU as in the equilibrium case. However, increases in the <em>numbers</em> of ECC_uppppp's with MCU (in contrast to decreases under equilibrium) lead to considerable numbers of misinferred <em>up</em> changes at high MCU. For ECC_puuuuu, the frequency of multiply-hit codons increases with MCU rather than decreasing as in the equilibrium scenario. The prevalence of ECC_uppppp results in <em>d</em><sub>up,pu</sub> inference biases similar to the equilibrium case.</p>
<a id="article1.body1.sec2.sec3.sec4.p5" name="article1.body1.sec2.sec3.sec4.p5"></a><p>ML inference of <em>pu</em> and <em>up</em> changes under relaxed selection differs considerably from MP results. <em>pu</em> underestimation increases and <em>up</em> changes are <em>overestimated</em> with increasing <em>N</em><sub>e</sub><em>s</em>. This causes overestimation of <em>d</em><sub>up,pu</sub> as a function of MCU (bringing <em>d</em><sub>up,pu</sub> closer to zero; <a href="#pone-0001065-g007">Figure 7C</a>). The bias in the opposite direction compared to ML inference under equilibrium (<a href="#pone-0001065-g003">Figure 3E</a>) is a consequence of the method for estimating base composition parameters. The lineages examined are relatively short compared to the time required to reach equilibrium. The base composition of extant sequences reflects parameter values (<em>N</em><sub>e</sub><em>s</em> and <em>u</em>/<em>v</em>) from the ancestral <em>mstyeo</em> lineage rather than values on the lineages on which fixations are inferred. For a given ECC, ancestral reconstructions are assigned probabilities according to equilibrium expectations given the base composition of the extant sequences; rate parameters are biased upward for <em>up</em> changes and downward for <em>pu</em> changes. <em>d</em><sub>up,pu</sub> inference is biased toward positive values because parallel <em>up</em> changes are assigned inflated probabilities and parallel <em>pu</em> changes are given underestimated probabilities. For ECC_uppppp, the proportions of codons with multiple <em>up</em> changes are considerably smaller than under equilibrium (roughly 50 and 14% for MCU = 0.7 and 0.9, respectively; <a href="#pone-0001065-g004">Figure 4</a>; <a href="#pone-0001065-t001">Table 1</a>) and ML overestimates this proportion. The larger <em>numbers</em> of codons with <em>pu</em> changes leads to considerable <em>up</em> overestimation (<a href="#pone-0001065-g007">Figure 7B</a>). For ECC_puuuuu, the proportions of codons with multiple <em>pu</em> changes are much <em>larger</em> than under equilibrium (roughly 2× and 15× for MCU = 0.7 and 0.9, respectively) and ML underestimates their probabilities.</p>
<a id="article1.body1.sec2.sec3.sec4.p6" name="article1.body1.sec2.sec3.sec4.p6"></a><p>Under the 2<em>N</em><sub>e</sub> scenario (increasing GC), both MP and ML underestimate <em>d</em><sub>up,pu</sub> as a function of MCU (<a href="#pone-0001065-g007">Figures 7D–F</a>). Actual <em>d</em><sub>up,pu</sub> values are positive (<a href="#pone-0001065-g001">Figures 1C and D</a>) and both methods bias the statistic toward zero. Because base composition in extant sequences reflects parameter values prior to the increase in <em>N</em><sub>e</sub> (<em>i.e</em>., lower GC content), HKY85 underestimates <em>up</em> rates and overestimates <em>pu</em> rates. Ancestral unpreferred sites have a much larger probability of undergoing <em>up</em> changes in multiple lineages than under equilibrium (<a href="#pone-0001065-g004">Figure 4</a>; <a href="#pone-0001065-t001">Table 1</a>); for MCU = 0.9, over 30% of ECC_uppppp observations reflect multiply-hit codons. Both ML and MP underestimation of <em>up</em> is larger than in the equilibrium case, but ML underestimation is less severe because multiple-<em>up</em> hit scenarios are assigned some probability. For ECC_puuuuu, multiple-hit scenarios are less common than under equilibrium (<a href="#pone-0001065-g004">Figure 4</a>; <a href="#pone-0001065-t001">Table 1</a>). Thus, MP shows less underestimation of <em>pu</em> as a function of MCU than in the equilibrium case, but ML overestimates <em>pu</em> changes. In contrast to the 1/3<em>N</em><sub>e</sub> scenario, the numbers of ECC_puuuuu observations are larger than the numbers of ECC_uppppp's for high MCU values. The magnitude of the resulting <em>pu</em> overestimation is substantial for both MP and ML (<a href="#pone-0001065-g007">Figure 7D</a>). Under this particular departure from the assumptions of both methods, ML and MP biases in <em>d</em><sub>up,pu</sub> inference are similar across a wide range of MCU.</p>

<h5>Changes in mutation bias.</h5><a id="article1.body1.sec2.sec3.sec5.p1" name="article1.body1.sec2.sec3.sec5.p1"></a><p>Ancestral inference were also examined for non-stationary codon bias evolution following changes in mutation rates. For these simulations, expected <em>up</em>:<em>pu</em> ratios of 0.5 and 2.0 were achieved by doubling either <em>u</em> (<em>per</em> site rate of S-&gt;W mutations) or <em>v</em> (<em>per</em> site rate of W-&gt;S mutations) relative to their values in the stationary 1× simulations. The scenarios will be referred to as 2<em>u</em>/<em>v</em> and 0.5<em>u</em>/<em>v</em>, for doublings of <em>u</em> and <em>v</em>, respectively. Within-fitness class mutation rates, W-&gt;W and S-&gt;S, were not changed. Mutation rate changes were implemented at the <em>mstyeo</em> node and all parameters were kept constant following the change. Expected <em>pu</em>:<em>up</em> fixation ratios are equal to the new <em>u</em>:<em>v</em> ratios immediately following a change in the mutation rate, but the fixation ratios gradually approach the equilibrium value of one (rates of approach to equilibrium are positive functions of initial MCU). Thus, the actual excesses of <em>up</em> or <em>pu</em> changes on the <em>m</em> lineage are smaller than two-fold and decrease with MCU (<a href="#pone-0001065-g008">Figures 8C and F</a>).</p>
<div class="figure" id="pone-0001065-g008"><div class="img"><a name="pone-0001065-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Reliability of ancestral codon bias inference under non-equilibrium evolution: variable mutation.</span></strong></p><a id="article1.body1.sec2.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec3.sec5.fig1.caption1.p1"></a><p>The legend in A applies to B, D, and E. The legend in C applies to F. X-axis scales are identical for graphs in the same column. Note that MCU values are given for the <em>m</em> node and have shifted from ancestral values. For the 2<em>u/v</em> scenario, reliability of <em>d</em><sub>up,pu</sub> inference was greater for ML than for MP for MCU≥0.8, but MP was more reliable for MCU≤0.6. For 0.5<em>u/v</em>, <em>d</em><sub>up,pu</sub> was more reliably inferred by ML than by MP for MCU≥0.7, but MP was more reliable for MCU = 0.5. Data are averaged across 300 replicates.</p>
<span>doi:10.1371/journal.pone.0001065.g008</span></div><a id="article1.body1.sec2.sec3.sec5.p2" name="article1.body1.sec2.sec3.sec5.p2"></a><p>For the 2<em>u</em>/<em>v</em> scenario of mutation-driven reductions in MCU, parsimony underestimates decreases in codon bias for low MCU genes, but overestimates codon bias changes for higher MCU (<a href="#pone-0001065-g008">Figure 8C</a>). In the absence of selection (MCU ≈ 0.5), parsimony underestimation is greater for the more common <em>pu</em> changes than for <em>up</em> changes. Increasing underestimation of <em>d</em><sub>up,pu</sub> as a function of MCU is similar to the pattern observed under equilibrium (<a href="#pone-0001065-g003">Figure 3E</a>). As selection increases the ratio of <em>up</em>:<em>pu</em> fixation probabilities, parsimony misinference at multiple <em>up</em>-hit codons leads to underestimation of <em>up</em> changes.</p>
<a id="article1.body1.sec2.sec3.sec5.p3" name="article1.body1.sec2.sec3.sec5.p3"></a><p>ML underestimates mutation-driven decreases in codon bias for low and intermediate MCU. Because parameters are estimated from sequences that have not reached equilibrium base composition, rates of <em>up</em> and <em>pu</em> are over- and under-estimated, respectively. This bias in parameter estimation leads to error in the probabilities of joint reconstructions. Decreasing underestimation of <em>d</em><sub>up,pu</sub> with MCU (the decline of codon bias is overestimated at very high MCU) probably reflects differences between the HKY85 and GC codon pref models (rates are estimated for pooled classes of neutral and non-neutral mutations; see above). With increasing MCU, the latter bias compensates, then over-compensates, for the bias introduced by non-stationary base composition.</p>
<a id="article1.body1.sec2.sec3.sec5.p4" name="article1.body1.sec2.sec3.sec5.p4"></a><p>Inference biases were considerable for mutation-driven increases in codon bias. In the 0.5<em>u</em>/<em>v</em> scenario, both selection and mutation elevate ratios of <em>per</em> site <em>up</em>:<em>pu</em> substitutions. Parsimony misinference at multiple-<em>up</em> hit codons increases dramatically as a function of MCU (<a href="#pone-0001065-g008">Figure 8E</a>) and results in large underestimation of <em>d</em><sub>up,pu</sub> (<a href="#pone-0001065-g008">Figure 8F</a>). For initial MCU = 0.95, parsimony infers <em>d</em><sub>up,pu</sub> = −0.22 where the actual value is 0.25.</p>
<a id="article1.body1.sec2.sec3.sec5.p5" name="article1.body1.sec2.sec3.sec5.p5"></a><p>ML biases were similar in direction, but much smaller in magnitude, than parsimony biases for the 0.5<em>u</em>/<em>v</em> scenario. Inference biases reflect under- and over-estimation of <em>up</em> and <em>pu</em> rates because base composition parameters are estimated from sequences that are far from equilibrium. Fitting data generated under GCpref codon to the HKY85 model exacerbates these biases, leading to greater differences between inferred and actual <em>d</em><sub>up,pu</sub> than for ML in the 2<em>u</em>/<em>v</em> scenario.</p>

<h5>Lineage-specific departures from equilibrium.</h5><a id="article1.body1.sec2.sec3.sec6.p1" name="article1.body1.sec2.sec3.sec6.p1"></a><p>The non-stationary scenarios considered above have employed a single parameter change at the <em>mstyeo</em> node and similar departures from equilibrium among lineages. A large number of scenarios of lineage-specific departures from equilibrium are possible, but some general results can be obtained from consideration of a set of scenarios based on findings in the <em>D. melanogaster</em> subgroup.</p>
<a id="article1.body1.sec2.sec3.sec6.p2" name="article1.body1.sec2.sec3.sec6.p2"></a><p>Data from nineteen loci are consistent with reductions in codon bias in the <em>m</em>, <em>y</em>, <em>o</em>, and <em>eo</em> lineages, and increases in codon bias in the <em>t</em> and <em>ty</em> lineages <a href="#pone.0001065-Akashi4">[35]</a>. For these genes, patterns on the <em>s</em> and <em>e</em> lineages appear to be consistent with equilibrium MCU. Changes in codon bias were assigned accordingly in the simulation. For simplicity, all reductions in codon bias were modeled as 1/3<em>N</em><sub>e</sub> and all increases were modeled as 2<em>N</em><sub>e</sub>. Because changes in the ancestral <em>ms</em> and <em>tyeo</em> lineages are unknown, all combinations of increases (2<em>N</em><sub>e</sub>), decreases (1/3<em>N</em><sub>e</sub>), and stationarity for these two lineages were employed (nine scenarios). Parameters were changed at the ancestral node of a lineage and were held constant within the lineage.</p>
<a id="article1.body1.sec2.sec3.sec6.p3" name="article1.body1.sec2.sec3.sec6.p3"></a><p>To allow comparisons between equilibrium and lineage-specific non-equilibrium simulations, <em>differences</em> between inferred and actual <em>d</em><sub>up,pu</sub> are plotted for each lineage in <a href="#pone-0001065-g009">Figure 9</a> and <a href="#pone.0001065.s002">Figure S1</a>. Given the phylogenetic distances in the <em>D. melanogaster</em> subgroup, <em>d</em><sub>up,pu</sub> inference biases under the mixed non-equilibrium simulation can be understood, to a large extent, by considering extant codon configurations consistent with both a single change or changes in two lineages (ECC_SD for extant codon configurations consistent with single or double hits). Because the <em>y</em> lineage is decreasing in bias and the ancestral <em>ty</em> lineage is increasing in MCU, <em>up</em> in <em>ty</em> followed by <em>pu</em> in <em>y</em> has an elevated occurrence relative to the 1× stationary case. <em>d</em><sub>up,pu</sub> under ML inference is elevated in <em>t</em> and <em>y</em>, and reduced in <em>ty</em>. Because <em>y</em> and <em>eo</em> are both losing codon bias, parallel <em>pu</em> changes in these lineages are more common, but such a scenario does not result in an ECC_SD. Relative to the equilibrium case, a decline of codon bias in the <em>m</em> lineage elevates the probability of <em>up</em> changes in <em>ms</em> followed by <em>pu</em> reversals in <em>m</em>. Because ML underestimates this probability, <em>d</em><sub>up,pu</sub> is elevated in both <em>m</em> and <em>s</em>.</p>
<div class="figure" id="pone-0001065-g009"><div class="img"><a name="pone-0001065-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001065" data-uri="info:doi/10.1371/journal.pone.0001065.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001065.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001065.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001065.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001065.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Reliability of ML codon bias inference under lineage-specific non-stationarity.</span></strong></p><a id="article1.body1.sec2.sec3.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec3.sec6.fig1.caption1.p1"></a><p>Differences between inferred and actual <em>d</em><sub>up,pu</sub> values are plotted as a function of MCU for each lineage (averages across 300 simulations are plotted). The legend applies to all graphs. X-axis scales apply to all graphs in the same column. Y-axis scales are identical among all graphs. The lineage-specific scenarios are as follows: stationary MCU (st) in &lt;l&gt;s and <em>e</em>, decreasing MCU (1/3<em>N</em><sub>e</sub>) in <em>m, y, o,</em> and <em>eo</em>, and increasing MCU (2<em>N</em><sub>e</sub>) in <em>t</em> and <em>ty</em>. Scenarios were varied in the ancestral <em>ms</em> and <em>tyeo</em> lineages. For the <em>ms</em> lineage: black (st), red (1/3<em>N</em><sub>e</sub>), blue (2<em>N</em><sub>e</sub>). For the <em>tyeo</em> lineage: thick (st), thin (1/3<em>N</em><sub>e</sub>), dotted (2<em>N</em><sub>e</sub>).</p>
<span>doi:10.1371/journal.pone.0001065.g009</span></div><a id="article1.body1.sec2.sec3.sec6.p4" name="article1.body1.sec2.sec3.sec6.p4"></a><p>Rate heterogeneity in ancestral lineages can have a strong impact on inference in their descendent lineages. If codon bias is increasing in the <em>ms</em> lineage (blue lines in <a href="#pone-0001065-g009">Figure 9</a>), then <em>up</em> in <em>ms</em>/<em>pu</em> in <em>m</em> reversals become more common. Such changes elevate <em>d</em><sub>up,pu</sub> for both the <em>m</em> and <em>s</em> lineages. In addition, increased frequencies of parallel <em>up</em> changes in the <em>ms</em> and <em>ty</em> lineages decrease <em>d</em><sub>up,pu</sub> in both <em>ty</em> and in <em>eo</em>. The length of the <em>ms</em> lineage contributes to substantial effects of these double-hit scenarios on <em>d</em><sub>up,pu</sub> inference. Decreases in MCU on the <em>ms</em> lineage (red lines in <a href="#pone-0001065-g009">Figure 9</a>) have the opposite effect on <em>d</em><sub>up,pu</sub> inference. Reverse changes, <em>pu</em> in <em>ms</em>/<em>up</em> in <em>s,</em> cause decreased <em>d</em><sub>up,pu</sub> inference in <em>m</em> and <em>s</em>, and parallel <em>pu</em> changes in <em>ms</em> and <em>eo</em> result in elevated <em>d</em><sub>up,pu</sub> in <em>ty</em> and <em>eo</em> (relative to equilibrium in <em>ms</em>). Note that <em>d</em><sub>up,pu</sub> inference on the <em>t</em>, <em>y</em>, <em>e</em>, and <em>o</em> lineages is relatively insensitive to departures from stationarity in <em>ms</em> and <em>tyeo</em> (<a href="#pone-0001065-g009">Figure 9</a>).</p>
<a id="article1.body1.sec2.sec3.sec6.p5" name="article1.body1.sec2.sec3.sec6.p5"></a><p>Non-stationary evolution on the ancestral <em>tyeo</em> lineage has a smaller impact on <em>d</em><sub>up,pu</sub> inference than departures from equilibrium on the longer <em>ms</em> lineage. Increasing MCU on the <em>tyeo</em> lineage enhances the probability of <em>up</em> in <em>tyeo</em>/<em>pu</em> in <em>eo</em> reversals. This results in elevated <em>d</em><sub>up,pu</sub> in both <em>eo</em> and <em>ty</em>. Higher occurrences of parallel <em>up</em> changes in <em>s</em> and <em>tyeo</em> result in decreased <em>d</em><sub>up,pu</sub> in both <em>s</em> and <em>m</em>. Decreasing codon bias in <em>tyeo</em> has the opposite effects on <em>d</em><sub>up,pu</sub> inference in these lineages. The elevated occurrence of <em>pu</em> in <em>tyeo</em>/<em>up</em> in <em>ty</em> reversals decreases <em>d</em><sub>up,pu</sub> in <em>ty</em> and <em>eo</em> and parallel <em>pu</em> changes in <em>tyeo</em> and <em>m</em> leads to elevated <em>d</em><sub>up,pu</sub> for the <em>s</em> and <em>m</em> lineages.</p>
<a id="article1.body1.sec2.sec3.sec6.p6" name="article1.body1.sec2.sec3.sec6.p6"></a><p>MP performance can be understood by considering the same sets of ECC_SD's. Inference biases (relative to the stationary 1× case) for the mixed non-equilibrium simulations are similar in direction, but generally greater in magnitude, to those under ML. <a href="#pone.0001065.s002">Figure S1</a> shows considerable MP underestimation of <em>d</em><sub>up,pu</sub>, even at intermediate MCU, for the <em>m</em>, <em>s</em>, <em>e</em>, <em>o</em>, and <em>eo</em> lineages for most of these scenarios.</p>
<a id="article1.body1.sec2.sec3.sec6.p7" name="article1.body1.sec2.sec3.sec6.p7"></a><p>These results suggest that ML generally allows more reliable inference of ancestral codon usage than MP for lineages within the <em>D. melanogaster</em> subgroup. In addition, inference biases do not appear to have contributed substantially to the conclusion of frequent codon bias fluctuations within the <em>D. melanogaster</em> subgroup <a href="#pone.0001065-Akashi4">[35]</a>. In particular, biases in ML <em>d</em><sub>up,pu</sub> inference for the <em>t</em>, <em>y</em>, <em>e</em>, and <em>o</em> lineages appear to be relatively small if the magnitude and direction of departures from equilibrium in our simulations are correct. Inference biases on the <em>s</em>, <em>ty</em> and <em>eo</em> lineages, however, can be considerable if codon bias has increased on the <em>ms</em> lineage. The sensitivity of inference on the <em>s</em> lineage to processes occurring in ancestral lineages appears to result from the combination of a strong departure from equilibrium on a sibling lineage, a long parent lineage, and the lack of an outgroup for the <em>ms</em> clade. It is important to note that longer lineages or greater departures from equilibrium could elevate <em>d</em><sub>up,pu</sub> biases from those shown in <a href="#pone-0001065-g009">Figure 9</a>. Other factors that affect ancestral inference (<em>i.e</em>., the locations of lineages in the phylogeny and species sampling) are discussed in <a href="#pone.0001065.s003">Results S1</a> and <a href="#pone.0001065.s004">S2</a>.</p>


</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Reliability of ancestral codon usage inference</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The accuracy of ancestral inference, even among relatively closely related species, has a complex dependence on branch lengths, the location of lineages within a phylogeny, rate heterogeneity among mutation classes (levels of base composition bias) and among lineages (changes in base composition). Parsimony is susceptible to errors in ancestral reconstruction when base composition is biased <a href="#pone.0001065-Collins1">[19]</a>, <a href="#pone.0001065-Perna1">[38]</a>, <a href="#pone.0001065-EyreWalker1">[39]</a>, <a href="#pone.0001065-Galtier1">[40]</a>, <a href="#pone.0001065-AlvarezValin1">[41]</a>. For lineages at stationary base composition, inference biases can yield patterns mimicking departures from equilibrium codon bias toward a loss of the common state (excess of unpreferred substitutions); the increase in this bias with the degree of base composition skew can lead to false inference of reductions in selection intensity for codon bias.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>For the scenarios considered in this study, maximum likelihood implementation of the HKY85 model is generally (often considerably) less biased than MP. Estimation of separate substitution rates for classes of nucleotide changes is desirable when base composition is skewed. However, departures from steady-state can cause strong biases in ancestral reconstruction among closely related species. The long lag-time between parameter changes and the achievement of a new equilibrium base composition is problematic for models that infer substitution rates from the base composition of extant sequences. Because the time-scale on which parameters vary appears to often be shorter than the time scale over which equilibrium is achieved <a href="#pone.0001065-Akashi4">[35]</a>, <a href="#pone.0001065-Maside1">[37]</a>, then inference using stationary models must be treated with caution.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>The direction and magnitude of departures from equilibrium as well as the branch lengths of a given lineage, its sibling lineage, direct ancestral lineages, and the sibling lineages to its direct ancestors can have a strong impact on the reliability of ancestral inference. For example, child/ancestor reversals will be prevalent if departures from equilibrium are in opposing directions in branches with direct descendent relationships. Parallel changes will be common when departures from equilibrium are in the same direction in sibling lineages or in a lineage and the sibling lineage of a direct ancestor. Bursts of adaptive codon bias evolution on multiple lineages can substantially reduce the reliability of ancestral inference.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>Among the lineages in our simulated data, ancestral inference is generally less biased in the <em>t</em>, <em>y</em>, <em>e</em>, and <em>o</em> lineages than in the <em>m</em>, <em>s</em> and <em>ty</em>, <em>eo</em> lineages. The greater accuracy of inference for terminal lineages in the <em>tyeo</em> clade results, in part, from shorter parental lineages than for the <em>m</em> and <em>s</em> lineages. In addition, for the <em>t, y, e,</em> and <em>o</em> lineages, data from the outgroup <em>ms</em> clade often eliminates the possibility of single-hit ancestral configurations consistent with child/sib-parent parallel changes. For these lineages, inference biases can be predicted given sequence data from species within the subgroup whereas biases for the <em>m, s</em>, <em>ty</em> and <em>eo</em> lineages are strongly dependent on unknown processes in the ancestral <em>ms</em> and <em>tyeo</em> lineages (<a href="#pone-0001065-g009">Figure 9</a>).</p>


<h4>Improving inference methods for ancestral codon usage</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Reliability of ancestral reconstructions is compromised by incorrect trees, errors in sequence alignment, incorrect substitution models, and insufficient data to estimate parameters of the substitution model. The analyses above assumed knowledge of the correct tree topology, no alignment errors, and sufficient data for ML parameter estimation. This study focused on the effects of incorrect substitution models.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Ancestral reconstructions can be improved both by data selection and by model improvements. Blanchette <em>et al</em>. <a href="#pone.0001065-Blanchette1">[72]</a> found that a “star-like” phylogeny allows reliable inference of ancestral mammal sequences given stationary evolution and sufficient sampling of extant taxa. Branch lengths are critical determinants of the accuracy of ancestral inference and errors can be minimized by choosing data from closely related species (with short ancestral lineages). However, even for the low levels of divergence examined in this study, departures from substitution model assumptions led to biases in ancestral reconstructions for high MCU genes (for MP) and for departures from steady-state (for MP and ML). These effects can be minimized by restricting data to moderately biased genes (MCU&lt;0.8). However, biases were clearly detectable for intermediate MCU genes in a number of examined scenarios, and longer lineages or greater fluctuations in base composition than those studied here can inflate these biases. For example, our simulations assumed homogeneity of parameters within genes. A number of findings support within-gene heterogeneity in <em>N</em><sub>e</sub><em>s</em> at silent sites <a href="#pone.0001065-Akashi7">[73]</a>, <a href="#pone.0001065-Moriyama2">[74]</a>, <a href="#pone.0001065-Kliman2">[75]</a>, <a href="#pone.0001065-Llopart1">[20]</a>, <a href="#pone.0001065-McVean1">[21]</a>, <a href="#pone.0001065-Iida1">[76]</a>, <a href="#pone.0001065-Comeron2">[77]</a>, <a href="#pone.0001065-Comeron1">[17]</a>, <a href="#pone.0001065-Qin1">[78]</a> which could elevate inference biases because rate parameters at variable sites will be underestimated.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>A number of improvements in ancestral inference methods could enhance comparisons among closely related species. Uncertainties in both tree topologies and parameter estimates have been incorporated in Bayesian approaches to ancestral state reconstruction <a href="#pone.0001065-Huelsenbeck1">[79]</a>, <a href="#pone.0001065-Nielsen1">[80]</a>, <a href="#pone.0001065-Krishnan1">[67]</a>, <a href="#pone.0001065-Pagel1">[81]</a>. Rate heterogeneity parameters are included in Galtier and Boursot's <a href="#pone.0001065-Galtier1">[40]</a> ML method. The equilibrium assumption appears to be a critical limitation in the methods employed here and Yang and Roberts <a href="#pone.0001065-Yang3">[82]</a> and Galtier and Guoy <a href="#pone.0001065-Galtier2">[83]</a> have incorporated parameters for fluctuating base composition in methods to infer ancestral GC content (these implementations do not calculate posterior probabilities of ancestral states).</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>Modifications to the simple parsimony approach could also improve inference of ancestral codon usage. The MP method employed above eliminated data for codons at which multiple reconstructions give the least number of changes in the gene tree. Such data could be included by allocating equal probabilities among most parsimonious reconstructions. In addition, classes of mutations can be assigned “weights” to account for asymmetric substitution rates <a href="#pone.0001065-Ree1">[84]</a>.</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>Key considerations for substitution models may vary considerably among taxa and genetic distances. Neighboring base effects play a strong role in mammalian genomes and transition/transversion rate ratios can be large in mtDNA evolution <a href="#pone.0001065-Li1">[85]</a>. For evolutionary studies of coding regions within the <em>D. melanogaster</em> subgroup, a codon substitution model that accommodates both heterogeneity in rates among sites and fluctuations in base composition may be appropriate. Nielsen et al. <a href="#pone.0001065-Nielsen2">[86]</a> have recently developed a method that allows fluctuations in codon bias within a gene tree and Arndt <a href="#pone.0001065-Arndt2">[33]</a> and Hernandez and co-workers <a href="#pone.0001065-Hernandez1">[87]</a> have proposed methods that account for non-stationarity and neighboring base effects. However, such models may entail trade-offs; increases in the amount of data required to estimate a larger number of parameters may preclude analyses of gene- and lineage-specific evolution except for large genes and/or long lineages <a href="#pone.0001065-Galtier2">[83]</a>. The average inferred numbers of <em>pu</em> and <em>up</em> per gene per lineage are 8.0 and 5.7, respectively, within the <em>D. melanogaster</em> subgroup for the nineteen genes examined in <a href="#pone.0001065-Akashi4">[35]</a>.</p>
<a id="article1.body1.sec3.sec2.p6" name="article1.body1.sec3.sec2.p6"></a><p>Finally, the data simulated for this study were generated using an instantaneous substitution model and did not consider variation within species. For several lineages in the <em>D. melanogaster</em> subgroup, the most recent common ancestor for within-species alleles may lie close to ancestral nodes. Contrasts of the numbers and the site frequency spectra of newly arisen mutations within populations and among closely related species <a href="#pone.0001065-Sawyer1">[88]</a>, <a href="#pone.0001065-McDonald1">[89]</a>, <a href="#pone.0001065-Akashi2">[08]</a>, <a href="#pone.0001065-Templeton1">[10]</a>, <a href="#pone.0001065-Bustamante1">[90]</a>, <a href="#pone.0001065-Williamson1">[13]</a> can be powerful approaches to identify the roles of weak selection and adaptive evolution for different functional classes of mutations <a href="#pone.0001065-Akashi3">[09]</a>, <a href="#pone.0001065-Bustamante2">[91]</a>. Such methods can determine the causes of codon bias and protein evolution as well as mechanisms of regulatory region and intron evolution <a href="#pone.0001065-Andolfatto1">[<em>e.g</em>., 92]</a>. Studies of the reliability of ancestral inference for polymorphism/divergence data <a href="#pone.0001065-Baudry1">[<em>e.g</em>., 93]</a> will be necessary to determine the robustness of such analyses.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001065"></div><a name="pone.0001065.s001" id="pone.0001065.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001065.s001">Methods S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">The major codon preference model.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001065.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001065.s002" id="pone.0001065.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001065.s002">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Reliability of parsimony codon bias inference under lineage-specific non-stationarity. Differences between inferred and actual <em>d</em><sup>up,pu</sup> values are plotted as a function of MCU for each lineage (averages across 300 simulations are plotted). The legend applies to all graphs. X-axis scales apply to all graphs in the same column. Y-axis scales are identical among all graphs and are identical to those in <a href="#pone-0001065-g009">Figure 9</a> to allow comparisons between methods. The lineage-specific scenarios are identical to those in <a href="#pone-0001065-g009">Figure 9:</a> stationary MCU (st) in <em>s</em> and <em>e</em>, decreasing MCU (1/3<em>N</em><sup>e</sup>) in <em>m, y, o,</em> and <em>eo</em>, and increasing MCU (2<em>N</em><sup>e</sup>) in <em>t</em> and <em>ty</em>. Scenarios were varied in the ancestral <em>ms</em> and <em>tyeo</em> lineages. For the ms lineage: black (st), red (1/3<em>N</em><sup>e</sup>), blue (2<em>N</em><sup>e</sup>). For the <em>tyeo</em> lineage: thick (st), thin (1/3<em>N</em><sup>e</sup>), dotted (2<em>N</em><sup>e</sup>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001065.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(16.60 MB TIF)</p>
<a name="pone.0001065.s003" id="pone.0001065.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001065.s003">Results S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Lineage-dependent ancestral reconstruction biases.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001065.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.68 MB DOC)</p>
<a name="pone.0001065.s004" id="pone.0001065.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001065.s004">Results S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Species composition and ancestral reconstruction biases.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001065.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.64 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to Ziheng Yang, Alex Wong, Wen-Ya Ko and several anonymous reviewers who helped to improve this study.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: HA PG. Analyzed the data: HA PG. Wrote the paper: HA. Other: Developed the C programs: AJ PG HA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001065-Long1" id="pone.0001065-Long1"></a>Long M, Langley CH (1993) Natural selection and the origin of jingwei, a chimeric processed functional gene in Drosophila. Science  260: 91–95.  <ul class="find" data-citedArticleID="1007968" data-doi="10.1126/science.7682012"><li><a href="http://dx.doi.org/10.1126/science.7682012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+and+the+origin+of+jingwei%2C+a+chimeric+processed+functional+gene+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+and+the+origin+of+jingwei%2C+a+chimeric+processed+functional+gene+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001065-Akashi1" id="pone.0001065-Akashi1"></a>Akashi H (1996) Molecular evolution between Drosophila melanogaster and D. simulans: reduced codon bias, faster rates of amino acid substitution, and larger proteins in D. melanogaster. Genetics  144: 1297–1307.  <ul class="find" data-citedArticleID="1007866"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+evolution+between+Drosophila+melanogaster+and+D.+simulans%3A+reduced+codon+bias%2C+faster+rates+of+amino+acid+substitution%2C+and+larger+proteins+in+D.+melanogaster.&amp;auth=&amp;atitle=Molecular+evolution+between+Drosophila+melanogaster+and+D.+simulans%3A+reduced+codon+bias%2C+faster+rates+of+amino+acid+substitution%2C+and+larger+proteins+in+D.+melanogaster." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolution+between+Drosophila+melanogaster+and+D.+simulans%3A+reduced+codon+bias%2C+faster+rates+of+amino+acid+substitution%2C+and+larger+proteins+in+D.+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolution+between+Drosophila+melanogaster+and+D.+simulans%3A+reduced+codon+bias%2C+faster+rates+of+amino+acid+substitution%2C+and+larger+proteins+in+D.+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001065-Eanes1" id="pone.0001065-Eanes1"></a>Eanes WF, Kirchner M, Yoon J, Biermann CH, Wang IN, et al.  (1996) Historical selection, amino acid polymorphism and lineage-specific divergence at the G6pd locus in Drosophila melanogaster and D. simulans. Genetics  144: 1027–1041.  <ul class="find" data-citedArticleID="1007922"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Historical+selection%2C+amino+acid+polymorphism+and+lineage-specific+divergence+at+the+G6pd+locus+in+Drosophila+melanogaster+and+D.+simulans.&amp;auth=&amp;atitle=Historical+selection%2C+amino+acid+polymorphism+and+lineage-specific+divergence+at+the+G6pd+locus+in+Drosophila+melanogaster+and+D.+simulans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Historical+selection%2C+amino+acid+polymorphism+and+lineage-specific+divergence+at+the+G6pd+locus+in+Drosophila+melanogaster+and+D.+simulans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Historical+selection%2C+amino+acid+polymorphism+and+lineage-specific+divergence+at+the+G6pd+locus+in+Drosophila+melanogaster+and+D.+simulans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001065-Fitch1" id="pone.0001065-Fitch1"></a>Fitch WM, Bush RM, Bender CA, Cox NJ (1997) Long term trends in the evolution of H(3) HA1 human influenza type A. Proc Natl Acad Sci U S A  94: 7712–7718.  <ul class="find" data-citedArticleID="1007928" data-doi="10.1073/pnas.94.15.7712"><li><a href="http://dx.doi.org/10.1073/pnas.94.15.7712" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long+term+trends+in+the+evolution+of+H%283%29+HA1+human+influenza+type+A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long+term+trends+in+the+evolution+of+H%283%29+HA1+human+influenza+type+A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001065-Messier1" id="pone.0001065-Messier1"></a>Messier W, Stewart CB (1997) Episodic adaptive evolution of primate lysozymes. Nature  385: 151–154.  <ul class="find" data-citedArticleID="1007980" data-doi="10.1038/385151a0"><li><a href="http://dx.doi.org/10.1038/385151a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Episodic+adaptive+evolution+of+primate+lysozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Episodic+adaptive+evolution+of+primate+lysozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001065-TakanoShimizu1" id="pone.0001065-TakanoShimizu1"></a>Takano-Shimizu T (2001) Local changes in GC/AT substitution biases and in crossover frequencies on Drosophila chromosomes. Mol Biol Evol  18: 606–619.  <ul class="find" data-citedArticleID="1008026" data-doi="10.1093/oxfordjournals.molbev.a003841"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003841" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Local+changes+in+GC%2FAT+substitution+biases+and+in+crossover+frequencies+on+Drosophila+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Local+changes+in+GC%2FAT+substitution+biases+and+in+crossover+frequencies+on+Drosophila+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001065-Zhang1" id="pone.0001065-Zhang1"></a>Zhang J, Zhang YP, Rosenberg HF (2002) Adaptive evolution of a duplicated pancreatic ribonuclease gene in a leaf-eating monkey. Nat Genet  30: 411–415.  <ul class="find" data-citedArticleID="1008046" data-doi="10.1038/ng852"><li><a href="http://dx.doi.org/10.1038/ng852" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+a+duplicated+pancreatic+ribonuclease+gene+in+a+leaf-eating+monkey." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+a+duplicated+pancreatic+ribonuclease+gene+in+a+leaf-eating+monkey.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001065-Akashi2" id="pone.0001065-Akashi2"></a>Akashi H (1995) Inferring weak selection from patterns of polymorphism and divergence at “silent” sites in Drosophila DNA. Genetics  139: 1067–1076.  <ul class="find" data-citedArticleID="1007868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inferring+weak+selection+from+patterns+of+polymorphism+and+divergence+at+%3Fsilent%3F+sites+in+Drosophila+DNA.&amp;auth=&amp;atitle=Inferring+weak+selection+from+patterns+of+polymorphism+and+divergence+at+%3Fsilent%3F+sites+in+Drosophila+DNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+weak+selection+from+patterns+of+polymorphism+and+divergence+at+%E2%80%9Csilent%E2%80%9D+sites+in+Drosophila+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+weak+selection+from+patterns+of+polymorphism+and+divergence+at+%E2%80%9Csilent%E2%80%9D+sites+in+Drosophila+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001065-Akashi3" id="pone.0001065-Akashi3"></a>Akashi H (1999) Inferring the fitness effects of DNA mutations from polymorphism and divergence data: statistical power to detect directional selection under stationarity and free recombination. Genetics  151: 221–238.  <ul class="find" data-citedArticleID="1007870"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inferring+the+fitness+effects+of+DNA+mutations+from+polymorphism+and+divergence+data%3A+statistical+power+to+detect+directional+selection+under+stationarity+and+free+recombination.&amp;auth=&amp;atitle=Inferring+the+fitness+effects+of+DNA+mutations+from+polymorphism+and+divergence+data%3A+statistical+power+to+detect+directional+selection+under+stationarity+and+free+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+the+fitness+effects+of+DNA+mutations+from+polymorphism+and+divergence+data%3A+statistical+power+to+detect+directional+selection+under+stationarity+and+free+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+the+fitness+effects+of+DNA+mutations+from+polymorphism+and+divergence+data%3A+statistical+power+to+detect+directional+selection+under+stationarity+and+free+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001065-Templeton1" id="pone.0001065-Templeton1"></a>Templeton AR (1996) Contingency tests of neutrality using intra/interspecific gene trees: the rejection of neutrality for the evolution of the mitochondrial cytochrome oxidase II gene in the hominoid primates. Genetics  144: 1263–1270.  <ul class="find" data-citedArticleID="1008030"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Contingency+tests+of+neutrality+using+intra%2Finterspecific+gene+trees%3A+the+rejection+of+neutrality+for+the+evolution+of+the+mitochondrial+cytochrome+oxidase+II+gene+in+the+hominoid+primates.&amp;auth=&amp;atitle=Contingency+tests+of+neutrality+using+intra%2Finterspecific+gene+trees%3A+the+rejection+of+neutrality+for+the+evolution+of+the+mitochondrial+cytochrome+oxidase+II+gene+in+the+hominoid+primates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contingency+tests+of+neutrality+using+intra%2Finterspecific+gene+trees%3A+the+rejection+of+neutrality+for+the+evolution+of+the+mitochondrial+cytochrome+oxidase+II+gene+in+the+hominoid+primates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contingency+tests+of+neutrality+using+intra%2Finterspecific+gene+trees%3A+the+rejection+of+neutrality+for+the+evolution+of+the+mitochondrial+cytochrome+oxidase+II+gene+in+the+hominoid+primates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001065-Suzuki1" id="pone.0001065-Suzuki1"></a>Suzuki Y, Gojobori T (1999) A method for detecting positive selection at single amino acid sites. Mol Biol Evol  16: 1315–1328.  <ul class="find" data-citedArticleID="1008024" data-doi="10.1093/oxfordjournals.molbev.a026042"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+method+for+detecting+positive+selection+at+single+amino+acid+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+method+for+detecting+positive+selection+at+single+amino+acid+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001065-Fay1" id="pone.0001065-Fay1"></a>Fay JC, Wu CI (2000) Hitchhiking under positive Darwinian selection. Genetics  155: 1405–1413.  <ul class="find" data-citedArticleID="1007926"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hitchhiking+under+positive+Darwinian+selection.&amp;auth=&amp;atitle=Hitchhiking+under+positive+Darwinian+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hitchhiking+under+positive+Darwinian+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hitchhiking+under+positive+Darwinian+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001065-Williamson1" id="pone.0001065-Williamson1"></a>Williamson SH, Hernandez R, Fledel-Alon A, Zhu L, Nielsen R, et al.  (2005) Simultaneous inference of selection and population growth from patterns of variation in the human genome. Proc Natl Acad Sci U S A  102: 7882–7887.  <ul class="find" data-citedArticleID="1008034" data-doi="10.1073/pnas.0502300102"><li><a href="http://dx.doi.org/10.1073/pnas.0502300102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simultaneous+inference+of+selection+and+population+growth+from+patterns+of+variation+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simultaneous+inference+of+selection+and+population+growth+from+patterns+of+variation+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001065-Kliman1" id="pone.0001065-Kliman1"></a>Kliman RM (1999) Recent selection on synonymous codon usage in Drosophila. J Mol Evol  49: 343–351.  <ul class="find" data-citedArticleID="1007950" data-doi="10.1007/pl00006557"><li><a href="http://dx.doi.org/10.1007/pl00006557" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+selection+on+synonymous+codon+usage+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+selection+on+synonymous+codon+usage+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001065-Begun1" id="pone.0001065-Begun1"></a>Begun DJ (2001) The frequency distribution of nucleotide variation in Drosophila simulans. Mol Biol Evol  18: 1343–1352.  <ul class="find" data-citedArticleID="1007896" data-doi="10.1093/oxfordjournals.molbev.a003918"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003918" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+frequency+distribution+of+nucleotide+variation+in+Drosophila+simulans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+frequency+distribution+of+nucleotide+variation+in+Drosophila+simulans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001065-DuMont1" id="pone.0001065-DuMont1"></a>DuMont VB, Fay JC, Calabrese PP, Aquadro CF (2004) DNA variability and divergence at the notch locus in Drosophila melanogaster and D. simulans: a case of accelerated synonymous site divergence. Genetics  167: 171–185.  <ul class="find" data-citedArticleID="1007914" data-doi="10.1534/genetics.167.1.171"><li><a href="http://dx.doi.org/10.1534/genetics.167.1.171" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+variability+and+divergence+at+the+notch+locus+in+Drosophila+melanogaster+and+D.+simulans%3A+a+case+of+accelerated+synonymous+site+divergence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+variability+and+divergence+at+the+notch+locus+in+Drosophila+melanogaster+and+D.+simulans%3A+a+case+of+accelerated+synonymous+site+divergence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001065-Comeron1" id="pone.0001065-Comeron1"></a>Comeron JM, Guthrie TB (2005) Intragenic Hill-Robertson Interference Influences Selection Intensity on Synonymous Mutations in Drosophila. Mol Biol Evol  22: 2519–2530.  <ul class="find" data-citedArticleID="1007910" data-doi="10.1093/molbev/msi246"><li><a href="http://dx.doi.org/10.1093/molbev/msi246" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intragenic+Hill-Robertson+Interference+Influences+Selection+Intensity+on+Synonymous+Mutations+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intragenic+Hill-Robertson+Interference+Influences+Selection+Intensity+on+Synonymous+Mutations+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001065-Presgraves1" id="pone.0001065-Presgraves1"></a>Presgraves DC (2005) Recombination enhances protein adaptation in Drosophila melanogaster. Curr Biol  15: 1651–1656.  <ul class="find" data-citedArticleID="1008002" data-doi="10.1016/j.cub.2005.07.065"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.07.065" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+enhances+protein+adaptation+in+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+enhances+protein+adaptation+in+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001065-Collins1" id="pone.0001065-Collins1"></a>Collins TM, Wimberger PH, Naylor GJP (1994) Compositional Bias, Character-State Bias, and Character-State Reconstruction Using Parsimony. Systematic Biology  43: 482–496.  <ul class="find" data-citedArticleID="1007908"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Compositional+Bias%2C+Character-State+Bias%2C+and+Character-State+Reconstruction+Using+Parsimony.&amp;auth=&amp;atitle=Compositional+Bias%2C+Character-State+Bias%2C+and+Character-State+Reconstruction+Using+Parsimony." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compositional+Bias%2C+Character-State+Bias%2C+and+Character-State+Reconstruction+Using+Parsimony." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compositional+Bias%2C+Character-State+Bias%2C+and+Character-State+Reconstruction+Using+Parsimony.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001065-Llopart1" id="pone.0001065-Llopart1"></a>Llopart A, Aguade M (1999) Synonymous rates at the RpII215 gene of Drosophila: variation among species and across the coding region. Genetics  152: 269–280.  <ul class="find" data-citedArticleID="1007964"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Synonymous+rates+at+the+RpII215+gene+of+Drosophila%3A+variation+among+species+and+across+the+coding+region.&amp;auth=&amp;atitle=Synonymous+rates+at+the+RpII215+gene+of+Drosophila%3A+variation+among+species+and+across+the+coding+region." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synonymous+rates+at+the+RpII215+gene+of+Drosophila%3A+variation+among+species+and+across+the+coding+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synonymous+rates+at+the+RpII215+gene+of+Drosophila%3A+variation+among+species+and+across+the+coding+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001065-McVean1" id="pone.0001065-McVean1"></a>McVean GA, Vieira J (1999) The evolution of codon preferences in Drosophila: a maximum-likelihood approach to parameter estimation and hypothesis testing. J Mol Evol  49: 63–75.  <ul class="find" data-citedArticleID="1007976" data-doi="10.1007/pl00006535"><li><a href="http://dx.doi.org/10.1007/pl00006535" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+codon+preferences+in+Drosophila%3A+a+maximum-likelihood+approach+to+parameter+estimation+and+hypothesis+testing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+codon+preferences+in+Drosophila%3A+a+maximum-likelihood+approach+to+parameter+estimation+and+hypothesis+testing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001065-RodriguezTrelles1" id="pone.0001065-RodriguezTrelles1"></a>Rodriguez-Trelles F, Tarrio R, Ayala FJ (2000) Evidence for a high ancestral GC content in Drosophila. Mol Biol Evol  17: 1710–1717.  <ul class="find" data-citedArticleID="1008008" data-doi="10.1093/oxfordjournals.molbev.a026269"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026269" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+a+high+ancestral+GC+content+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+a+high+ancestral+GC+content+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001065-RodriguezTrelles2" id="pone.0001065-RodriguezTrelles2"></a>Rodriguez-Trelles F, Tarrio R, Ayala FJ (2000) Fluctuating mutation bias and the evolution of base composition in Drosophila. J Mol Evol  50: 1–10.  <ul class="find" data-citedArticleID="1008010"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fluctuating+mutation+bias+and+the+evolution+of+base+composition+in+Drosophila.&amp;auth=&amp;atitle=Fluctuating+mutation+bias+and+the+evolution+of+base+composition+in+Drosophila." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fluctuating+mutation+bias+and+the+evolution+of+base+composition+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fluctuating+mutation+bias+and+the+evolution+of+base+composition+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001065-Begun2" id="pone.0001065-Begun2"></a>Begun DJ, Whitley P (2002) Molecular population genetics of Xdh and the evolution of base composition in Drosophila. Genetics  162: 1725–1735.  <ul class="find" data-citedArticleID="1007898"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+population+genetics+of+Xdh+and+the+evolution+of+base+composition+in+Drosophila.&amp;auth=&amp;atitle=Molecular+population+genetics+of+Xdh+and+the+evolution+of+base+composition+in+Drosophila." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+population+genetics+of+Xdh+and+the+evolution+of+base+composition+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+population+genetics+of+Xdh+and+the+evolution+of+base+composition+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001065-Bachtrog1" id="pone.0001065-Bachtrog1"></a>Bachtrog D (2003) Protein evolution and codon usage bias on the neo-sex chromosomes of Drosophila miranda. Genetics  165: 1221–1232.  <ul class="find" data-citedArticleID="1007890"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Protein+evolution+and+codon+usage+bias+on+the+neo-sex+chromosomes+of+Drosophila+miranda.&amp;auth=&amp;atitle=Protein+evolution+and+codon+usage+bias+on+the+neo-sex+chromosomes+of+Drosophila+miranda." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+evolution+and+codon+usage+bias+on+the+neo-sex+chromosomes+of+Drosophila+miranda." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+evolution+and+codon+usage+bias+on+the+neo-sex+chromosomes+of+Drosophila+miranda.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001065-Perez1" id="pone.0001065-Perez1"></a>Perez JA, Munte A, Rozas J, Segarra C, Aguade M (2003) Nucleotide polymorphism in the RpII215 gene region of the insular species Drosophila guanche: reduced efficacy of weak selection on synonymous variation. Mol Biol Evol  20: 1867–1875.  <ul class="find" data-citedArticleID="1007994" data-doi="10.1093/molbev/msg199"><li><a href="http://dx.doi.org/10.1093/molbev/msg199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleotide+polymorphism+in+the+RpII215+gene+region+of+the+insular+species+Drosophila+guanche%3A+reduced+efficacy+of+weak+selection+on+synonymous+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleotide+polymorphism+in+the+RpII215+gene+region+of+the+insular+species+Drosophila+guanche%3A+reduced+efficacy+of+weak+selection+on+synonymous+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001065-Powell1" id="pone.0001065-Powell1"></a>Powell JR, Sezzi E, Moriyama EN, Gleason JM, Caccone A (2003) Analysis of a shift in codon usage in Drosophila. J Mol Evol 57 Suppl  1: S214–225.  <ul class="find" data-citedArticleID="1008000" data-doi="10.1007/s00239-003-0030-3"><li><a href="http://dx.doi.org/10.1007/s00239-003-0030-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+a+shift+in+codon+usage+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+a+shift+in+codon+usage+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001065-Duret1" id="pone.0001065-Duret1"></a>Duret L, Semon M, Piganeau G, Mouchiroud D, Galtier N (2002) Vanishing GC-rich isochores in mammalian genomes. Genetics  162: 1837–1847.  <ul class="find" data-citedArticleID="1007916"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Vanishing+GC-rich+isochores+in+mammalian+genomes.&amp;auth=&amp;atitle=Vanishing+GC-rich+isochores+in+mammalian+genomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vanishing+GC-rich+isochores+in+mammalian+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vanishing+GC-rich+isochores+in+mammalian+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001065-Smith1" id="pone.0001065-Smith1"></a>Smith NG, Eyre-Walker A (2002) The compositional evolution of the murid genome. J Mol Evol  55: 197–201.  <ul class="find" data-citedArticleID="1008020" data-doi="10.1007/s00239-002-2316-2"><li><a href="http://dx.doi.org/10.1007/s00239-002-2316-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+compositional+evolution+of+the+murid+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+compositional+evolution+of+the+murid+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001065-Arndt1" id="pone.0001065-Arndt1"></a>Arndt PF, Petrov DA, Hwa T (2003) Distinct changes of genomic biases in nucleotide substitution at the time of Mammalian radiation. Mol Biol Evol  20: 1887–1896.  <ul class="find" data-citedArticleID="1007886" data-doi="10.1093/molbev/msg204"><li><a href="http://dx.doi.org/10.1093/molbev/msg204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+changes+of+genomic+biases+in+nucleotide+substitution+at+the+time+of+Mammalian+radiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+changes+of+genomic+biases+in+nucleotide+substitution+at+the+time+of+Mammalian+radiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001065-Webster1" id="pone.0001065-Webster1"></a>Webster MT, Smith NG, Ellegren H (2003) Compositional evolution of noncoding DNA in the human and chimpanzee genomes. Mol Biol Evol  20: 278–286.  <ul class="find" data-citedArticleID="1008032" data-doi="10.1093/molbev/msg037"><li><a href="http://dx.doi.org/10.1093/molbev/msg037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compositional+evolution+of+noncoding+DNA+in+the+human+and+chimpanzee+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compositional+evolution+of+noncoding+DNA+in+the+human+and+chimpanzee+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001065-Belle1" id="pone.0001065-Belle1"></a>Belle EM, Duret L, Galtier N, Eyre-Walker A (2004) The decline of isochores in mammals: an assessment of the GC content variation along the mammalian phylogeny. J Mol Evol  58: 653–660.  <ul class="find" data-citedArticleID="1007900" data-doi="10.1007/s00239-004-2587-x"><li><a href="http://dx.doi.org/10.1007/s00239-004-2587-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+decline+of+isochores+in+mammals%3A+an+assessment+of+the+GC+content+variation+along+the+mammalian+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+decline+of+isochores+in+mammals%3A+an+assessment+of+the+GC+content+variation+along+the+mammalian+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001065-Arndt2" id="pone.0001065-Arndt2"></a>Arndt PF (2007) Reconstruction of ancestral nucleotide sequences and estimation of substitution frequencies in a star phylogeny. Gene  390: 75–83.  <ul class="find" data-citedArticleID="1007888" data-doi="10.1016/j.gene.2006.11.022"><li><a href="http://dx.doi.org/10.1016/j.gene.2006.11.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstruction+of+ancestral+nucleotide+sequences+and+estimation+of+substitution+frequencies+in+a+star+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstruction+of+ancestral+nucleotide+sequences+and+estimation+of+substitution+frequencies+in+a+star+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001065-Zhang2" id="pone.0001065-Zhang2"></a>Zhang Z, Inomata N, Ohba T, Cariou ML, Yamazaki T (2002) Codon bias differentiates between the duplicated amylase loci following gene duplication in Drosophila. Genetics  161: 1187–1196.  <ul class="find" data-citedArticleID="1008048"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Codon+bias+differentiates+between+the+duplicated+amylase+loci+following+gene+duplication+in+Drosophila.&amp;auth=&amp;atitle=Codon+bias+differentiates+between+the+duplicated+amylase+loci+following+gene+duplication+in+Drosophila." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+bias+differentiates+between+the+duplicated+amylase+loci+following+gene+duplication+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+bias+differentiates+between+the+duplicated+amylase+loci+following+gene+duplication+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001065-Akashi4" id="pone.0001065-Akashi4"></a>Akashi H, Ko WY, Piao S, John A, Goel P, et al.  (2006) Molecular evolution in the Drosophila melanogaster species subgroup: frequent parameter fluctuations on the timescale of molecular divergence. Genetics  172: 1711–1726.  <ul class="find" data-citedArticleID="1007872" data-doi="10.1534/genetics.105.049676"><li><a href="http://dx.doi.org/10.1534/genetics.105.049676" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolution+in+the+Drosophila+melanogaster+species+subgroup%3A+frequent+parameter+fluctuations+on+the+timescale+of+molecular+divergence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolution+in+the+Drosophila+melanogaster+species+subgroup%3A+frequent+parameter+fluctuations+on+the+timescale+of+molecular+divergence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001065-Ko1" id="pone.0001065-Ko1"></a>Ko WY, Piao S, Akashi H (2006) Strong regional heterogeneity in base composition evolution on the Drosophila X chromosome. Genetics  174: 349–362.  <ul class="find" data-citedArticleID="1007954" data-doi="10.1534/genetics.105.054346"><li><a href="http://dx.doi.org/10.1534/genetics.105.054346" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strong+regional+heterogeneity+in+base+composition+evolution+on+the+Drosophila+X+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strong+regional+heterogeneity+in+base+composition+evolution+on+the+Drosophila+X+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001065-Maside1" id="pone.0001065-Maside1"></a>Maside X, Charlesworth B (2007) Patterns of molecular variation and evolution in Drosophila americana and its relatives. Genetics.   <ul class="find" data-citedArticleID="1007970" data-doi="10.1534/genetics.107.071191"><li><a href="http://dx.doi.org/10.1534/genetics.107.071191" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+molecular+variation+and+evolution+in+Drosophila+americana+and+its+relatives." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+molecular+variation+and+evolution+in+Drosophila+americana+and+its+relatives.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001065-Perna1" id="pone.0001065-Perna1"></a>Perna NT, Kocher TD (1995) Unequal Base Frequencies and Estimation of Substitution Rates. Molecular Biology and Evolution  12: 359–361.  <ul class="find" data-citedArticleID="1007996"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Unequal+Base+Frequencies+and+Estimation+of+Substitution+Rates.&amp;auth=&amp;atitle=Unequal+Base+Frequencies+and+Estimation+of+Substitution+Rates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unequal+Base+Frequencies+and+Estimation+of+Substitution+Rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unequal+Base+Frequencies+and+Estimation+of+Substitution+Rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001065-EyreWalker1" id="pone.0001065-EyreWalker1"></a>Eyre-Walker A (1998) Problems with parsimony in sequences of biased base composition. J Mol Evol  47: 686–690.  <ul class="find" data-citedArticleID="1007924" data-doi="10.1007/pl00006427"><li><a href="http://dx.doi.org/10.1007/pl00006427" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Problems+with+parsimony+in+sequences+of+biased+base+composition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Problems+with+parsimony+in+sequences+of+biased+base+composition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001065-Galtier1" id="pone.0001065-Galtier1"></a>Galtier N, Boursot P (2000) A new method for locating changes in a tree reveals distinct nucleotide polymorphism vs. divergence patterns in mouse mitochondrial control region. J Mol Evol  50: 224–231.  <ul class="find" data-citedArticleID="1007930"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+new+method+for+locating+changes+in+a+tree+reveals+distinct+nucleotide+polymorphism+vs.+divergence+patterns+in+mouse+mitochondrial+control+region.&amp;auth=&amp;atitle=A+new+method+for+locating+changes+in+a+tree+reveals+distinct+nucleotide+polymorphism+vs.+divergence+patterns+in+mouse+mitochondrial+control+region." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+method+for+locating+changes+in+a+tree+reveals+distinct+nucleotide+polymorphism+vs.+divergence+patterns+in+mouse+mitochondrial+control+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+method+for+locating+changes+in+a+tree+reveals+distinct+nucleotide+polymorphism+vs.+divergence+patterns+in+mouse+mitochondrial+control+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001065-AlvarezValin1" id="pone.0001065-AlvarezValin1"></a>Alvarez-Valin F, Clay O, Cruveiller S, Bernardi G (2004) Inaccurate reconstruction of ancestral GC levels creates a “vanishing isochores” effect. Mol Phylogenet Evol  31: 788–793.  <ul class="find" data-citedArticleID="1007880" data-doi="10.1016/j.ympev.2004.01.016"><li><a href="http://dx.doi.org/10.1016/j.ympev.2004.01.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inaccurate+reconstruction+of+ancestral+GC+levels+creates+a+%E2%80%9Cvanishing+isochores%E2%80%9D+effect." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inaccurate+reconstruction+of+ancestral+GC+levels+creates+a+%E2%80%9Cvanishing+isochores%E2%80%9D+effect.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001065-Hasegawa1" id="pone.0001065-Hasegawa1"></a>Hasegawa M, Kishino H, Yano T (1985) Dating of the human-ape splitting by a molecular clock of mitochondrial DNA. J Mol Evol  22: 160–174.  <ul class="find" data-citedArticleID="1007934" data-doi="10.1007/bf02101694"><li><a href="http://dx.doi.org/10.1007/bf02101694" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dating+of+the+human-ape+splitting+by+a+molecular+clock+of+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dating+of+the+human-ape+splitting+by+a+molecular+clock+of+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001065-Ikemura1" id="pone.0001065-Ikemura1"></a>Ikemura T (1985) Codon usage and tRNA content in unicellular and multicellular organisms. Mol Biol Evol  2: 13–34.  <ul class="find" data-citedArticleID="1007942"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Codon+usage+and+tRNA+content+in+unicellular+and+multicellular+organisms.&amp;auth=&amp;atitle=Codon+usage+and+tRNA+content+in+unicellular+and+multicellular+organisms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+usage+and+tRNA+content+in+unicellular+and+multicellular+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+usage+and+tRNA+content+in+unicellular+and+multicellular+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001065-Shields1" id="pone.0001065-Shields1"></a>Shields DC, Sharp PM, Higgins DG, Wright F (1988) “Silent” sites in Drosophila genes are not neutral: evidence of selection among synonymous codons. Mol Biol Evol  5: 704–716.  <ul class="find" data-citedArticleID="1008018"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=%3FSilent%3F+sites+in+Drosophila+genes+are+not+neutral%3A+evidence+of+selection+among+synonymous+codons.&amp;auth=&amp;atitle=%3FSilent%3F+sites+in+Drosophila+genes+are+not+neutral%3A+evidence+of+selection+among+synonymous+codons." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%E2%80%9CSilent%E2%80%9D+sites+in+Drosophila+genes+are+not+neutral%3A+evidence+of+selection+among+synonymous+codons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%E2%80%9CSilent%E2%80%9D+sites+in+Drosophila+genes+are+not+neutral%3A+evidence+of+selection+among+synonymous+codons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001065-Andersson1" id="pone.0001065-Andersson1"></a>Andersson SG, Kurland CG (1990) Codon preferences in free-living microorganisms. Microbiol Rev  54: 198–210.  <ul class="find" data-citedArticleID="1007882"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Codon+preferences+in+free-living+microorganisms.&amp;auth=&amp;atitle=Codon+preferences+in+free-living+microorganisms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+preferences+in+free-living+microorganisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+preferences+in+free-living+microorganisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001065-Sharp1" id="pone.0001065-Sharp1"></a>Sharp PM, Averof M, Lloyd AT, Matassi G, Peden JF (1995) DNA sequence evolution: the sounds of silence. Philos Trans R Soc Lond B Biol Sci  349: 241–247.  <ul class="find" data-citedArticleID="1008016" data-doi="10.1098/rstb.1995.0108"><li><a href="http://dx.doi.org/10.1098/rstb.1995.0108" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+sequence+evolution%3A+the+sounds+of+silence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+sequence+evolution%3A+the+sounds+of+silence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001065-Akashi5" id="pone.0001065-Akashi5"></a>Akashi H (2001) Gene expression and molecular evolution. Curr Opin Genet Dev  11: 660–666.  <ul class="find" data-citedArticleID="1007874" data-doi="10.1016/s0959-437x(00)00250-1"><li><a href="http://dx.doi.org/10.1016/s0959-437x(00)00250-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+and+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+and+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001065-Duret2" id="pone.0001065-Duret2"></a>Duret L (2002) Evolution of synonymous codon usage in metazoans. Curr Opin Genet Dev  12: 640–649.  <ul class="find" data-citedArticleID="1007918" data-doi="10.1016/s0959-437x(02)00353-2"><li><a href="http://dx.doi.org/10.1016/s0959-437x(02)00353-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+synonymous+codon+usage+in+metazoans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+synonymous+codon+usage+in+metazoans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001065-Kanaya1" id="pone.0001065-Kanaya1"></a>Kanaya S, Yamada Y, Kudo Y, Ikemura T (1999) Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of Bacillus subtilis tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis. Gene  238: 143–155.  <ul class="find" data-citedArticleID="1007946" data-doi="10.1016/s0378-1119(99)00225-5"><li><a href="http://dx.doi.org/10.1016/s0378-1119(99)00225-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Studies+of+codon+usage+and+tRNA+genes+of+18+unicellular+organisms+and+quantification+of+Bacillus+subtilis+tRNAs%3A+gene+expression+level+and+species-specific+diversity+of+codon+usage+based+on+multivariate+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Studies+of+codon+usage+and+tRNA+genes+of+18+unicellular+organisms+and+quantification+of+Bacillus+subtilis+tRNAs%3A+gene+expression+level+and+species-specific+diversity+of+codon+usage+based+on+multivariate+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001065-Kanaya2" id="pone.0001065-Kanaya2"></a>Kanaya S, Yamada Y, Kinouchi M, Kudo Y, Ikemura T (2001) Codon usage and tRNA genes in eukaryotes: correlation of codon usage diversity with translation efficiency and with CG-dinucleotide usage as assessed by multivariate analysis. J Mol Evol  53: 290–298.  <ul class="find" data-citedArticleID="1007948" data-doi="10.1007/s002390010219"><li><a href="http://dx.doi.org/10.1007/s002390010219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+usage+and+tRNA+genes+in+eukaryotes%3A+correlation+of+codon+usage+diversity+with+translation+efficiency+and+with+CG-dinucleotide+usage+as+assessed+by+multivariate+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+usage+and+tRNA+genes+in+eukaryotes%3A+correlation+of+codon+usage+diversity+with+translation+efficiency+and+with+CG-dinucleotide+usage+as+assessed+by+multivariate+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001065-Moriyama1" id="pone.0001065-Moriyama1"></a>Moriyama EN, Powell JR (1997) Codon usage bias and tRNA abundance in Drosophila. J Mol Evol  45: 514–523.  <ul class="find" data-citedArticleID="1007982" data-doi="10.1007/pl00006256"><li><a href="http://dx.doi.org/10.1007/pl00006256" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+usage+bias+and+tRNA+abundance+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+usage+bias+and+tRNA+abundance+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001065-Duret3" id="pone.0001065-Duret3"></a>Duret L (2000) tRNA gene number and codon usage in the C. elegans genome are co-adapted for optimal translation of highly expressed genes. Trends <strong>Genet</strong>  16: 287–289.  <ul class="find" data-citedArticleID="1007920" data-doi="10.1016/s0168-9525(00)02041-2"><li><a href="http://dx.doi.org/10.1016/s0168-9525(00)02041-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNA+gene+number+and+codon+usage+in+the+C.+elegans+genome+are+co-adapted+for+optimal+translation+of+highly+expressed+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNA+gene+number+and+codon+usage+in+the+C.+elegans+genome+are+co-adapted+for+optimal+translation+of+highly+expressed+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001065-Wright1" id="pone.0001065-Wright1"></a>Wright SI, Yau CB, Looseley M, Meyers BC (2004) Effects of gene expression on molecular evolution in Arabidopsis thaliana and Arabidopsis lyrata. Mol Biol Evol  21: 1719–1726.  <ul class="find" data-citedArticleID="1008038" data-doi="10.1093/molbev/msh191"><li><a href="http://dx.doi.org/10.1093/molbev/msh191" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+gene+expression+on+molecular+evolution+in+Arabidopsis+thaliana+and+Arabidopsis+lyrata." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+gene+expression+on+molecular+evolution+in+Arabidopsis+thaliana+and+Arabidopsis+lyrata.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001065-Kurland1" id="pone.0001065-Kurland1"></a>Kurland CG (1992) Translational accuracy and the fitness of bacteria. Annu Rev Genet  26: 29–50.  <ul class="find" data-citedArticleID="1007960" data-doi="10.1146/annurev.ge.26.120192.000333"><li><a href="http://dx.doi.org/10.1146/annurev.ge.26.120192.000333" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translational+accuracy+and+the+fitness+of+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translational+accuracy+and+the+fitness+of+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001065-Akashi6" id="pone.0001065-Akashi6"></a>Akashi H, Schaeffer SW (1997) Natural selection and the frequency distributions of “silent” DNA polymorphism in Drosophila. Genetics  146: 295–307.  <ul class="find" data-citedArticleID="1007876"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Natural+selection+and+the+frequency+distributions+of+%3Fsilent%3F+DNA+polymorphism+in+Drosophila.&amp;auth=&amp;atitle=Natural+selection+and+the+frequency+distributions+of+%3Fsilent%3F+DNA+polymorphism+in+Drosophila." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+and+the+frequency+distributions+of+%E2%80%9Csilent%E2%80%9D+DNA+polymorphism+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+and+the+frequency+distributions+of+%E2%80%9Csilent%E2%80%9D+DNA+polymorphism+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001065-Llopart2" id="pone.0001065-Llopart2"></a>Llopart A, Aguade M (2000) Nucleotide polymorphism at the RpII215 gene in Drosophila subobscura. Weak selection on synonymous mutations. Genetics  155: 1245–1252.  <ul class="find" data-citedArticleID="1007966"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Nucleotide+polymorphism+at+the+RpII215+gene+in+Drosophila+subobscura.+Weak+selection+on+synonymous+mutations.&amp;auth=&amp;atitle=Nucleotide+polymorphism+at+the+RpII215+gene+in+Drosophila+subobscura.+Weak+selection+on+synonymous+mutations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleotide+polymorphism+at+the+RpII215+gene+in+Drosophila+subobscura.+Weak+selection+on+synonymous+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleotide+polymorphism+at+the+RpII215+gene+in+Drosophila+subobscura.+Weak+selection+on+synonymous+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001065-Maside2" id="pone.0001065-Maside2"></a>Maside X, Lee AW, Charlesworth B (2004) Selection on codon usage in Drosophila americana. Curr Biol  14: 150–154.  <ul class="find" data-citedArticleID="1007972" data-doi="10.1016/j.cub.2003.12.055"><li><a href="http://dx.doi.org/10.1016/j.cub.2003.12.055" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+on+codon+usage+in+Drosophila+americana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+on+codon+usage+in+Drosophila+americana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001065-Bartolome1" id="pone.0001065-Bartolome1"></a>Bartolome C, Maside X, Yi S, Grant AL, Charlesworth B (2005) Patterns of selection on synonymous and nonsynonymous variants in Drosophila miranda. Genetics  169: 1495–1507.  <ul class="find" data-citedArticleID="1007892" data-doi="10.1534/genetics.104.033068"><li><a href="http://dx.doi.org/10.1534/genetics.104.033068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+selection+on+synonymous+and+nonsynonymous+variants+in+Drosophila+miranda." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+selection+on+synonymous+and+nonsynonymous+variants+in+Drosophila+miranda.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001065-McVean2" id="pone.0001065-McVean2"></a>McVean GAT, Charlesworth B (1999) A population genetic model for the evolution of synonymous codon usage: patterns and predictions. Genetical Research  74: 145–158.  <ul class="find" data-citedArticleID="1007978" data-doi="10.1017/s0016672399003912"><li><a href="http://dx.doi.org/10.1017/s0016672399003912" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+population+genetic+model+for+the+evolution+of+synonymous+codon+usage%3A+patterns+and+predictions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+population+genetic+model+for+the+evolution+of+synonymous+codon+usage%3A+patterns+and+predictions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001065-Ko2" id="pone.0001065-Ko2"></a>Ko WY, David RM, Akashi H (2003) Molecular phylogeny of the Drosophila melanogaster species subgroup. J Mol Evol  57: 562–573.  <ul class="find" data-citedArticleID="1007956" data-doi="10.1007/s00239-003-2510-x"><li><a href="http://dx.doi.org/10.1007/s00239-003-2510-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogeny+of+the+Drosophila+melanogaster+species+subgroup." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogeny+of+the+Drosophila+melanogaster+species+subgroup.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001065-Wong1" id="pone.0001065-Wong1"></a>Wong A, Jensen JD, Pool JE, Aquadro CF (2006) Phylogenetic incongruence in the Drosophila melanogaster species group. Mol Phylogenet Evol.   <ul class="find" data-citedArticleID="1008036" data-doi="10.1016/j.ympev.2006.09.002"><li><a href="http://dx.doi.org/10.1016/j.ympev.2006.09.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+incongruence+in+the+Drosophila+melanogaster+species+group." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+incongruence+in+the+Drosophila+melanogaster+species+group.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001065-Pollard1" id="pone.0001065-Pollard1"></a>Pollard DA, Iyer VN, Moses AM, Eisen MB (2006) Widespread discordance of gene trees with species tree in Drosophila: evidence for incomplete lineage sorting. PLoS Genet  2: e173.  <ul class="find" data-citedArticleID="1007998" data-doi="10.1371/journal.pgen.0020173"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020173" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Widespread+discordance+of+gene+trees+with+species+tree+in+Drosophila%3A+evidence+for+incomplete+lineage+sorting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Widespread+discordance+of+gene+trees+with+species+tree+in+Drosophila%3A+evidence+for+incomplete+lineage+sorting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001065-Yang1" id="pone.0001065-Yang1"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Comput Appl Biosci  13: 555–556.  <ul class="find" data-citedArticleID="1008040" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001065-Nei1" id="pone.0001065-Nei1"></a>Nei M, Gojobori T (1986) Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions. Mol Biol Evol  3: 418–426.  <ul class="find" data-citedArticleID="1007986"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions.&amp;auth=&amp;atitle=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001065-Yang2" id="pone.0001065-Yang2"></a>Yang Z, Kumar S, Nei M (1995) A new method of inference of ancestral nucleotide and amino acid sequences. Genetics  141: 1641–1650.  <ul class="find" data-citedArticleID="1008042"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+new+method+of+inference+of+ancestral+nucleotide+and+amino+acid+sequences.&amp;auth=&amp;atitle=A+new+method+of+inference+of+ancestral+nucleotide+and+amino+acid+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+method+of+inference+of+ancestral+nucleotide+and+amino+acid+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+method+of+inference+of+ancestral+nucleotide+and+amino+acid+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001065-Jukes1" id="pone.0001065-Jukes1"></a>Jukes TH, Cantor CR (1969)  Evolution of protein molecules. In: Munro HN, editor. Mammalian Protein Metabolism III. New York: Academic Press. pp. 21–132.  <ul class="find-nolinks"></ul></li><li><span class="label">67.
              </span><a name="pone.0001065-Krishnan1" id="pone.0001065-Krishnan1"></a>Krishnan NM, Seligmann H, Stewart CB, De Koning AP, Pollock DD (2004) Ancestral sequence reconstruction in primate mitochondrial DNA: compositional bias and effect on functional inference. Mol Biol Evol  21: 1871–1883.  <ul class="find" data-citedArticleID="1007958" data-doi="10.1093/molbev/msh198"><li><a href="http://dx.doi.org/10.1093/molbev/msh198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancestral+sequence+reconstruction+in+primate+mitochondrial+DNA%3A+compositional+bias+and+effect+on+functional+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancestral+sequence+reconstruction+in+primate+mitochondrial+DNA%3A+compositional+bias+and+effect+on+functional+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001065-Snedecor1" id="pone.0001065-Snedecor1"></a>Snedecor GW, Cochran WG (1989) Statistical Methods: 8th Edition. Ames, Iowa: Iowa State University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">69.
              </span><a name="pone.0001065-Zhang3" id="pone.0001065-Zhang3"></a>Zhang J, Nei M (1997) Accuracies of ancestral amino acid sequences inferred by the parsimony, likelihood, and distance methods. J Mol Evol  44: Suppl 1S139–146.  <ul class="find" data-citedArticleID="1008050" data-doi="10.1007/pl00000067"><li><a href="http://dx.doi.org/10.1007/pl00000067" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accuracies+of+ancestral+amino+acid+sequences+inferred+by+the+parsimony%2C+likelihood%2C+and+distance+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accuracies+of+ancestral+amino+acid+sequences+inferred+by+the+parsimony%2C+likelihood%2C+and+distance+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001065-TakanoShimizu2" id="pone.0001065-TakanoShimizu2"></a>Takano-Shimizu T (1999) Local recombination and mutation effects on molecular evolution in Drosophila. Genetics  153: 1285–1296.  <ul class="find" data-citedArticleID="1008028"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Local+recombination+and+mutation+effects+on+molecular+evolution+in+Drosophila.&amp;auth=&amp;atitle=Local+recombination+and+mutation+effects+on+molecular+evolution+in+Drosophila." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Local+recombination+and+mutation+effects+on+molecular+evolution+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Local+recombination+and+mutation+effects+on+molecular+evolution+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001065-RodriguezTrelles3" id="pone.0001065-RodriguezTrelles3"></a>Rodriguez-Trelles F, Tarrio R, Ayala FJ (1999) Switch in codon bias and increased rates of amino acid substitution in the Drosophila saltans species group. Genetics  153: 339–350.  <ul class="find" data-citedArticleID="1008012"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Switch+in+codon+bias+and+increased+rates+of+amino+acid+substitution+in+the+Drosophila+saltans+species+group.&amp;auth=&amp;atitle=Switch+in+codon+bias+and+increased+rates+of+amino+acid+substitution+in+the+Drosophila+saltans+species+group." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Switch+in+codon+bias+and+increased+rates+of+amino+acid+substitution+in+the+Drosophila+saltans+species+group." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Switch+in+codon+bias+and+increased+rates+of+amino+acid+substitution+in+the+Drosophila+saltans+species+group.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001065-Blanchette1" id="pone.0001065-Blanchette1"></a>Blanchette M, Green ED, Miller W, Haussler D (2004) Reconstructing large regions of an ancestral mammalian genome in silico. Genome Res  14: 2412–2423.  <ul class="find" data-citedArticleID="1007902" data-doi="10.1101/gr.2800104"><li><a href="http://dx.doi.org/10.1101/gr.2800104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstructing+large+regions+of+an+ancestral+mammalian+genome+in+silico." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstructing+large+regions+of+an+ancestral+mammalian+genome+in+silico.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001065-Akashi7" id="pone.0001065-Akashi7"></a>Akashi H (1994) Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy. Genetics  136: 927–935.  <ul class="find" data-citedArticleID="1007878"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Synonymous+codon+usage+in+Drosophila+melanogaster%3A+natural+selection+and+translational+accuracy.&amp;auth=&amp;atitle=Synonymous+codon+usage+in+Drosophila+melanogaster%3A+natural+selection+and+translational+accuracy." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synonymous+codon+usage+in+Drosophila+melanogaster%3A+natural+selection+and+translational+accuracy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synonymous+codon+usage+in+Drosophila+melanogaster%3A+natural+selection+and+translational+accuracy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001065-Moriyama2" id="pone.0001065-Moriyama2"></a>Moriyama EN, Powell JR (1998) Gene length and codon usage bias in Drosophila melanogaster, Saccharomyces cerevisiae and Escherichia coli. Nucleic Acids Res  26: 3188–3193.  <ul class="find" data-citedArticleID="1007984" data-doi="10.1093/nar/26.13.3188"><li><a href="http://dx.doi.org/10.1093/nar/26.13.3188" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+length+and+codon+usage+bias+in+Drosophila+melanogaster%2C+Saccharomyces+cerevisiae+and+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+length+and+codon+usage+bias+in+Drosophila+melanogaster%2C+Saccharomyces+cerevisiae+and+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001065-Kliman2" id="pone.0001065-Kliman2"></a>Kliman RM, Eyre-Walker A (1998) Patterns of base composition within the genes of Drosophila melanogaster. J Mol Evol  46: 534–541.  <ul class="find" data-citedArticleID="1007952" data-doi="10.1007/pl00006334"><li><a href="http://dx.doi.org/10.1007/pl00006334" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+base+composition+within+the+genes+of+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+base+composition+within+the+genes+of+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001065-Iida1" id="pone.0001065-Iida1"></a>Iida K, Akashi H (2000) A test of translational selection at ‘silent’ sites in the human genome: base composition comparisons in alternatively spliced genes. Gene  261: 93–105.  <ul class="find" data-citedArticleID="1007940" data-doi="10.1016/s0378-1119(00)00482-0"><li><a href="http://dx.doi.org/10.1016/s0378-1119(00)00482-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+test+of+translational+selection+at+%E2%80%98silent%E2%80%99+sites+in+the+human+genome%3A+base+composition+comparisons+in+alternatively+spliced+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+test+of+translational+selection+at+%E2%80%98silent%E2%80%99+sites+in+the+human+genome%3A+base+composition+comparisons+in+alternatively+spliced+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0001065-Comeron2" id="pone.0001065-Comeron2"></a>Comeron JM, Kreitman M (2002) Population, evolutionary and genomic consequences of interference selection. Genetics  161: 389–410.  <ul class="find" data-citedArticleID="1007912"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Population%2C+evolutionary+and+genomic+consequences+of+interference+selection.&amp;auth=&amp;atitle=Population%2C+evolutionary+and+genomic+consequences+of+interference+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population%2C+evolutionary+and+genomic+consequences+of+interference+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population%2C+evolutionary+and+genomic+consequences+of+interference+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0001065-Qin1" id="pone.0001065-Qin1"></a>Qin H, Wu WB, Comeron JM, Kreitman M, Li WH (2004) Intragenic spatial patterns of codon usage bias in prokaryotic and eukaryotic genomes. Genetics  168: 2245–2260.  <ul class="find" data-citedArticleID="1008004" data-doi="10.1534/genetics.104.030866"><li><a href="http://dx.doi.org/10.1534/genetics.104.030866" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intragenic+spatial+patterns+of+codon+usage+bias+in+prokaryotic+and+eukaryotic+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intragenic+spatial+patterns+of+codon+usage+bias+in+prokaryotic+and+eukaryotic+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0001065-Huelsenbeck1" id="pone.0001065-Huelsenbeck1"></a>Huelsenbeck JP, Bollback JP (2001) Empirical and hierarchical Bayesian estimation of ancestral states. Syst Biol  50: 351–366.  <ul class="find" data-citedArticleID="1007938"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Empirical+and+hierarchical+Bayesian+estimation+of+ancestral+states.&amp;auth=&amp;atitle=Empirical+and+hierarchical+Bayesian+estimation+of+ancestral+states." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Empirical+and+hierarchical+Bayesian+estimation+of+ancestral+states." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Empirical+and+hierarchical+Bayesian+estimation+of+ancestral+states.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0001065-Nielsen1" id="pone.0001065-Nielsen1"></a>Nielsen R (2002) Mapping mutations on phylogenies. Syst Biol  51: 729–739.  <ul class="find" data-citedArticleID="1007988"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mapping+mutations+on+phylogenies.&amp;auth=&amp;atitle=Mapping+mutations+on+phylogenies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+mutations+on+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+mutations+on+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0001065-Pagel1" id="pone.0001065-Pagel1"></a>Pagel M, Meade A, Barker D (2004) Bayesian estimation of ancestral character states on phylogenies. Syst Biol  53: 673–684.  <ul class="find" data-citedArticleID="1007992"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bayesian+estimation+of+ancestral+character+states+on+phylogenies.&amp;auth=&amp;atitle=Bayesian+estimation+of+ancestral+character+states+on+phylogenies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+estimation+of+ancestral+character+states+on+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+estimation+of+ancestral+character+states+on+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0001065-Yang3" id="pone.0001065-Yang3"></a>Yang Z, Roberts D (1995) On the use of nucleic acid sequences to infer early branchings in the tree of life. Mol Biol Evol  12: 451–458.  <ul class="find" data-citedArticleID="1008044"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+the+use+of+nucleic+acid+sequences+to+infer+early+branchings+in+the+tree+of+life.&amp;auth=&amp;atitle=On+the+use+of+nucleic+acid+sequences+to+infer+early+branchings+in+the+tree+of+life." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+use+of+nucleic+acid+sequences+to+infer+early+branchings+in+the+tree+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+use+of+nucleic+acid+sequences+to+infer+early+branchings+in+the+tree+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0001065-Galtier2" id="pone.0001065-Galtier2"></a>Galtier N, Gouy M (1998) Inferring pattern and process: maximum-likelihood implementation of a nonhomogeneous model of DNA sequence evolution for phylogenetic analysis. Mol Biol Evol  15: 871–879.  <ul class="find" data-citedArticleID="1007932" data-doi="10.1093/oxfordjournals.molbev.a025991"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025991" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+pattern+and+process%3A+maximum-likelihood+implementation+of+a+nonhomogeneous+model+of+DNA+sequence+evolution+for+phylogenetic+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+pattern+and+process%3A+maximum-likelihood+implementation+of+a+nonhomogeneous+model+of+DNA+sequence+evolution+for+phylogenetic+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0001065-Ree1" id="pone.0001065-Ree1"></a>Ree RH, Donoghue MJ (1998) Step matrices and the interpretation of homoplasy. Syst Biol  47: 582–588.  <ul class="find" data-citedArticleID="1008006"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Step+matrices+and+the+interpretation+of+homoplasy.&amp;auth=&amp;atitle=Step+matrices+and+the+interpretation+of+homoplasy." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Step+matrices+and+the+interpretation+of+homoplasy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Step+matrices+and+the+interpretation+of+homoplasy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0001065-Li1" id="pone.0001065-Li1"></a>Li W-H (1997) Molecular evolution. Sunderland, Mass.: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">86.
              </span><a name="pone.0001065-Nielsen2" id="pone.0001065-Nielsen2"></a>Nielsen R, Bauer DuMont VL, Hubisz MJ, Aquadro CF (2007) Maximum likelihood estimation of ancestral codon usage bias parameters in Drosophila. Mol Biol Evol  24: 228–235.  <ul class="find" data-citedArticleID="1007990" data-doi="10.1093/molbev/msl146"><li><a href="http://dx.doi.org/10.1093/molbev/msl146" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum+likelihood+estimation+of+ancestral+codon+usage+bias+parameters+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum+likelihood+estimation+of+ancestral+codon+usage+bias+parameters+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0001065-Hernandez1" id="pone.0001065-Hernandez1"></a>Hernandez RD, Williamson SH, Bustamante CD (2007) Context dependence, ancestral misidentification, and spurious signatures of natural selection. Mol Biol Evol  24: 1792–1800.  <ul class="find" data-citedArticleID="1007936" data-doi="10.1093/molbev/msm108"><li><a href="http://dx.doi.org/10.1093/molbev/msm108" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Context+dependence%2C+ancestral+misidentification%2C+and+spurious+signatures+of+natural+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Context+dependence%2C+ancestral+misidentification%2C+and+spurious+signatures+of+natural+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0001065-Sawyer1" id="pone.0001065-Sawyer1"></a>Sawyer SA, Dykhuizen DE, Hartl DL (1987) Confidence interval for the number of selectively neutral amino acid polymorphisms. Proc Natl Acad Sci U S A  84: 6225–6228.  <ul class="find" data-citedArticleID="1008014" data-doi="10.1073/pnas.84.17.6225"><li><a href="http://dx.doi.org/10.1073/pnas.84.17.6225" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Confidence+interval+for+the+number+of+selectively+neutral+amino+acid+polymorphisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Confidence+interval+for+the+number+of+selectively+neutral+amino+acid+polymorphisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0001065-McDonald1" id="pone.0001065-McDonald1"></a>McDonald JH, Kreitman M (1991) Adaptive protein evolution at the Adh locus in Drosophila. Nature  351: 652–654.  <ul class="find" data-citedArticleID="1007974" data-doi="10.1038/351652a0"><li><a href="http://dx.doi.org/10.1038/351652a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+protein+evolution+at+the+Adh+locus+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+protein+evolution+at+the+Adh+locus+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0001065-Bustamante1" id="pone.0001065-Bustamante1"></a>Bustamante CD, Nielsen R, Sawyer SA, Olsen KM, Purugganan MD, et al.  (2002) The cost of inbreeding in Arabidopsis. Nature  416: 531–534.  <ul class="find" data-citedArticleID="1007904" data-doi="10.1038/416531a"><li><a href="http://dx.doi.org/10.1038/416531a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+cost+of+inbreeding+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+cost+of+inbreeding+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0001065-Bustamante2" id="pone.0001065-Bustamante2"></a>Bustamante CD, Wakeley J, Sawyer S, Hartl DL (2001) Directional selection and the site-frequency spectrum. Genetics  159: 1779–1788.  <ul class="find" data-citedArticleID="1007906"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Directional+selection+and+the+site-frequency+spectrum.&amp;auth=&amp;atitle=Directional+selection+and+the+site-frequency+spectrum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Directional+selection+and+the+site-frequency+spectrum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Directional+selection+and+the+site-frequency+spectrum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0001065-Andolfatto1" id="pone.0001065-Andolfatto1"></a>Andolfatto P (2005) Adaptive evolution of non-coding DNA in Drosophila. Nature  437: 1149–1152.  <ul class="find" data-citedArticleID="1007884" data-doi="10.1038/nature04107"><li><a href="http://dx.doi.org/10.1038/nature04107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+non-coding+DNA+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+non-coding+DNA+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0001065-Baudry1" id="pone.0001065-Baudry1"></a>Baudry E, Depaulis F (2003) Effect of misoriented sites on neutrality tests with outgroup. Genetics  165: 1619–1622.  <ul class="find" data-citedArticleID="1007894"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effect+of+misoriented+sites+on+neutrality+tests+with+outgroup.&amp;auth=&amp;atitle=Effect+of+misoriented+sites+on+neutrality+tests+with+outgroup." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+misoriented+sites+on+neutrality+tests+with+outgroup." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+misoriented+sites+on+neutrality+tests+with+outgroup.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.XML" value="167781"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.PDF" value="491453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g001.PNG_L" value="185398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g001.PNG_M" value="40085"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g001.PNG_S" value="7558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g001.TIF" value="294408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g001.PNG_I" value="14228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g002.PNG_L" value="275209"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g002.PNG_M" value="23163"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g002.PNG_S" value="6085"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g002.TIF" value="453384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g002.PNG_I" value="8859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g003.PNG_L" value="227420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g003.PNG_M" value="30602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g003.PNG_S" value="5678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g003.TIF" value="383306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g003.PNG_I" value="10679"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g004.PNG_L" value="341984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g004.PNG_M" value="41287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g004.PNG_S" value="8837"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g004.TIF" value="522800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g004.PNG_I" value="23762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.t001.PNG_L" value="193178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.t001.PNG_M" value="114552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.t001.PNG_S" value="12069"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.t001.TIF" value="793762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.t001.PNG_I" value="35599"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g005.PNG_L" value="265044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g005.PNG_M" value="35401"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g005.PNG_S" value="6353"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g005.TIF" value="451406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g005.PNG_I" value="12298"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g006.PNG_L" value="80554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g006.PNG_M" value="22976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g006.PNG_S" value="5060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g006.TIF" value="128234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g006.PNG_I" value="8330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g007.PNG_L" value="296216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g007.PNG_M" value="38874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g007.PNG_S" value="6776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g007.TIF" value="490468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g007.PNG_I" value="13610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g008.PNG_L" value="296730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g008.PNG_M" value="38242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g008.PNG_S" value="6716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g008.TIF" value="484522"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g008.PNG_I" value="13200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g009.PNG_L" value="868509"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g009.PNG_M" value="109133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g009.PNG_S" value="9349"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g009.TIF" value="1315332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.g009.PNG_I" value="37899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.s001.DOC" value="40960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.s002.TIF" value="16621677"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.s003.DOC" value="1678848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001065.s004.DOC" value="643072"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001065&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001065" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001065&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001065&volume=&issue=&title=Ancestral Inference and the Study of Codon Bias Evolution: Implications for Molecular Evolutionary Analyses of the Drosophila melanogaster Subgroup&author_name=Hiroshi%20Akashi%2C%20Piyush%20Goel%2C%20Anoop%20John&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001065" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001065" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001065" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001065&amp;t=Ancestral%20Inference%20and%20the%20Study%20of%20Codon%20Bias%20Evolution%3A%20Implications%20for%20Molecular%20Evolutionary%20Analyses%20of%20the%20Drosophila%20melanogaster%20Subgroup" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001065&title=Ancestral%20Inference%20and%20the%20Study%20of%20Codon%20Bias%20Evolution%3A%20Implications%20for%20Molecular%20Evolutionary%20Analyses%20of%20the%20Drosophila%20melanogaster%20Subgroup&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001065&amp;title=Ancestral%20Inference%20and%20the%20Study%20of%20Codon%20Bias%20Evolution%3A%20Implications%20for%20Molecular%20Evolutionary%20Analyses%20of%20the%20Drosophila%20melanogaster%20Subgroup" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001065" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001065&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Ancestral Inference and the Study of Codon Bias Evolution: Implications for Molecular Evolutionary Analyses of the Drosophila melanogaster Subgroup';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001065';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Ancestral%20Inference%20and%20the%20Study%20of%20Codon%20Bias%20Evolution%3A%20Implications%20for%20Molecular%20Evolutionary%20Analyses%20of%20the%20Drosophila%20melanogaster%20Subgroup http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001065" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001065" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="24964"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drosophila+melanogaster%22" title="Search for articles in the subject area:'Drosophila melanogaster'"><div class="flagText">Drosophila melanogaster</div></a>
              <div data-categoryid="32461" data-articleid="24964"
                   data-categoryname="Drosophila melanogaster"
                   class="flagImage" title="Flag 'Drosophila melanogaster' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="24964"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24964"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Molecular+evolution%22" title="Search for articles in the subject area:'Molecular evolution'"><div class="flagText">Molecular evolution</div></a>
              <div data-categoryid="17533" data-articleid="24964"
                   data-categoryname="Molecular evolution"
                   class="flagImage" title="Flag 'Molecular evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="24964"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24964"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Substitution+mutation%22" title="Search for articles in the subject area:'Substitution mutation'"><div class="flagText">Substitution mutation</div></a>
              <div data-categoryid="35145" data-articleid="24964"
                   data-categoryname="Substitution mutation"
                   class="flagImage" title="Flag 'Substitution mutation' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8862'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1282'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3574&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=14231">Referee Comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
