

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Worldwide Phylogeography for the Human X Chromosome</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000557"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000557" />

    <meta name="citation_title" content="A Worldwide Phylogeography for the Human X Chromosome"/>
    <meta itemprop="name" content="A Worldwide Phylogeography for the Human X Chromosome"/>

      <meta name="citation_author" content="Simone S. Santos-Lopes"/>
            <meta name="citation_author_institution" content="Departamento de Bioquímica e Imunologia, Universidade Federal de Minas Gerais, Belo Horizonte, Minas Gerais, Brazil"/>
      <meta name="citation_author" content="Rinaldo W. Pereira"/>
            <meta name="citation_author_institution" content="Departamento de Bioquímica e Imunologia, Universidade Federal de Minas Gerais, Belo Horizonte, Minas Gerais, Brazil"/>
            <meta name="citation_author_institution" content="Programa de Pós Graduação em Ciências Genômicas e Biotecnologia, Catholic University of Brasília (UCB), Brasília, Brazil"/>
      <meta name="citation_author" content="Ian J. Wilson"/>
            <meta name="citation_author_institution" content="Institute of Human Genetics, Newcastle University, Newcastle, United Kingdom"/>
      <meta name="citation_author" content="Sérgio D.J. Pena"/>
            <meta name="citation_author_institution" content="Departamento de Bioquímica e Imunologia, Universidade Federal de Minas Gerais, Belo Horizonte, Minas Gerais, Brazil"/>

    <meta name="citation_date" content="2007/6/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000557.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e557"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The human Y chromosome: an evolutionary marker comes of age.; citation_author=MA Jobling; citation_author=C Tyler-Smith; citation_journal_title=Nat Rev Genet; citation_volume=4; citation_number=1; citation_pages=598-612; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The X chromosome in population genetics.; citation_author=SF Schaffner; citation_journal_title=Nat Rev Genet; citation_volume=5; citation_number=2; citation_pages=43-51; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sex-biased migration in humans: what should we expect from genetic data?; citation_author=JF Wilkins; citation_author=FW Marlowe; citation_journal_title=Bioessays; citation_volume=28; citation_number=3; citation_pages=290-300; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=X chromosome map at 75-kb STS resolution, revealing extremes of recombination and GC content.; citation_author=R Nagaraja; citation_author=S MacMillan; citation_author=J Kere; citation_author=C Jones; citation_author=S Griffin; citation_journal_title=Genome Res; citation_volume=7; citation_number=4; citation_pages=210-222; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A novel polymorphic Alu insertion embedded in a LINE 1 retrotransposon in the human X chromosome (DXS225): identification and worldwide population study.; citation_author=RW Pereira; citation_author=SS Santos; citation_author=SD Pena; citation_journal_title=Genet Mol Res; citation_volume=5; citation_number=5; citation_pages=63-71; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Standardized subsets of the HGDP-CEPH Human Genome Diversity Cell Line Panel, accounting for atypical and duplicated samples and pairs of close relatives.; citation_author=NA Rosenberg; citation_journal_title=Ann Hum Genet.; citation_volume=70; citation_number=6; citation_pages=841-847; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=L Storto; citation_author=FFV Velden; citation_number=7; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=SNPSTRs: empirically derived, rapidly typed, autosomal haplotypes for inference of population history and mutational processes.; citation_author=JL Mountain; citation_author=A Knight; citation_author=M Jobin; citation_author=C Gignoux; citation_author=A Miller; citation_journal_title=Genome Res; citation_volume=12; citation_number=8; citation_pages=1766-1772; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A comprehensive genetic map of the human genome based on 5,264 microsatellites.; citation_author=C Dib; citation_author=S Faure; citation_author=C Fizames; citation_author=D Samson; citation_author=N Drouot; citation_journal_title=Nature; citation_volume=380; citation_number=9; citation_pages=152-154; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences.; citation_author=G Benson; citation_journal_title=Nuc Acid Res; citation_volume=27; citation_number=10; citation_pages=573-580; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data.; citation_author=L Excoffier; citation_author=PE Smouse; citation_author=JM Quattro; citation_journal_title=Genetics; citation_volume=131; citation_number=11; citation_pages=479-491; citation_date=1992; " />
      <meta name="citation_reference" content="citation_author=S Schneider; citation_author=D Roessli; citation_author=L Excoffier; citation_number=12; citation_date=2000; citation_publisher=University of Geneva; " />
      <meta name="citation_reference" content="citation_title=Modeling linkage disequilibrium and identifying recombination hotspots using single-nucleotide polymorphism data.; citation_author=N Li; citation_author=M Stephens; citation_journal_title=Genetics; citation_volume=165; citation_number=13; citation_pages=2213-2233; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Median-joining networks for inferring intraspecific phylogenies.; citation_author=HJ Bandelt; citation_author=P Forster; citation_author=A Rohl; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=14; citation_pages=37-48; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genealogical inference from microsatellite data.; citation_author=IJ Wilson; citation_author=DG Balding; citation_journal_title=Genetics; citation_volume=150; citation_number=15; citation_pages=499-510; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Inferences from DNA data: population histories, evolutionary processes and forensic match probabilities.; citation_author=IJ Wilson; citation_author=ME Weale; citation_author=DJ Balding; citation_journal_title=J R Stat Soc Ser A; citation_volume=166; citation_number=16; citation_pages=155-201; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gametic disequilibrium measures: proceed with caution.; citation_author=WP Hedrick; citation_journal_title=Genetics; citation_volume=117; citation_number=17; citation_pages=331-341; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Mutation rate variation at human dinucleotide microsatellites.; citation_author=H Xu; citation_author=R Chakraborty; citation_author=Y-X Fu; citation_journal_title=Genetics; citation_volume=170; citation_number=18; citation_pages=305-312; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The mutational spectrum of human autosomal tetranucleotide microsatellites.; citation_author=AM Leopoldino; citation_author=SD Pena; citation_journal_title=Hum Mutat; citation_volume=21; citation_number=19; citation_pages=71-79; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetic structure of human populations.; citation_author=NA Rosenberg; citation_author=JK Pritchard; citation_author=JL Weber; citation_author=HM Cann; citation_author=KK Kidd; citation_author=LA Zhivotovsky; citation_author=MW Feldman; citation_journal_title=Science.; citation_volume=298; citation_number=20; citation_pages=2381-2385; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A haplotype map of the human genome.; citation_journal_title=Nature; citation_volume=437; citation_number=21; citation_pages=1299-1320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Haploview: analysis and visualization of LD and haplotype maps.; citation_author=JC Barrett; citation_author=B Fry; citation_author=J Maller; citation_author=MJ Daly; citation_journal_title=Bioinformatics:; citation_volume=21; citation_number=22; citation_pages=263-265; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A fine-scale map of recombination rates and hotspots across the human genome.; citation_author=S Myers; citation_author=L Bottolo; citation_author=C Freeman; citation_author=G McVean; citation_author=P Donnelly; citation_journal_title=Science; citation_volume=310; citation_number=23; citation_pages=321-324; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The application of molecular genetic approaches to the study of human evolution.; citation_author=LL Cavalli-Sforza; citation_author=MW Feldman; citation_journal_title=Nat Genet; citation_volume=33; citation_number=24; citation_pages=266-275; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The evolution and structure of human genetic diversity.; citation_author=SDJ Pena; citation_number=25; citation_date=2007; citation_publisher=Landes Bioscience; " />
      <meta name="citation_reference" content="citation_author=LL Cavalli-Sforza; citation_author=P Menozzi; citation_author=A Piazza; citation_number=26; citation_date=1994; citation_publisher=Princeton University Press; " />
      <meta name="citation_reference" content="citation_title=The central Siberian origin for Native American chromosomes.; citation_author=FR Santos; citation_author=A Pandya; citation_author=C Tyler-Smith; citation_author=SDJ Pena; citation_author=M Schanfield; citation_journal_title=Am J Hum Genet; citation_volume=64; citation_number=27; citation_pages=619-628; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Harvesting the fruit of the human mtDNA tree.; citation_author=A Torroni; citation_author=A Achilli; citation_author=V Macaulay; citation_author=M Richards; citation_author=HJ Bandelt; citation_journal_title=Trends Genet; citation_volume=22; citation_number=28; citation_pages=339-345; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Larger genetic differences within africans than between Africans and Eurasians.; citation_author=N Yu; citation_author=FC Chen; citation_author=S Ota; citation_author=LB Jorde; citation_author=P Pamilo; citation_journal_title=Genetics; citation_volume=161; citation_number=29; citation_pages=269-274; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Geography predicts neutral genetic diversity of human populations.; citation_author=F Prugnolle; citation_author=A Manica; citation_author=F Balloux; citation_journal_title=Curr Biol; citation_volume=15; citation_number=30; citation_pages=159-160; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Support from the relationship of genetic and geographic distance in human populations for a serial founder effect originating in Africa.; citation_author=S Ramachandran; citation_author=O Deshpande; citation_author=CC Roseman; citation_author=NA Rosenberg; citation_author=MW Feldman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=31; citation_pages=15942-15947; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A geographically explicit genetic model of worldwide human-settlement history.; citation_author=H Liu; citation_author=F Prugnolle; citation_author=A Manica; citation_author=F Balloux; citation_journal_title=Am J Hum Genet; citation_volume=79; citation_number=32; citation_pages=230-237; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Stratigraphic placement and age of modern humans from Kibish, Ethiopia.; citation_author=I McDougall; citation_author=FH Brown; citation_author=JG Fleagle; citation_journal_title=Nature; citation_volume=433; citation_number=33; citation_pages=733-736; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Going East: new genetic and archaeological perspectives on the modern human colonization of Eurasia.; citation_author=P Mellars; citation_journal_title=Science; citation_volume=313; citation_number=34; citation_pages=796-800; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A worldwide survey of haplotype variation and linkage disequilibrium in the human genome.; citation_author=DF Conrad; citation_author=M Jakobsson; citation_author=G Coop; citation_author=X Wen; citation_author=JD Wall; citation_journal_title=Nat Genet; citation_volume=38; citation_number=35; citation_pages=1251-1260; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000557" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Worldwide Phylogeography for the Human X Chromosome"/>
    <meta name="twitter:description" content="BackgroundWe reasoned that by identifying genetic markers on human X chromosome regions where recombination is rare or absent, we should be able to construct X chromosome genealogies analogous to those based on Y chromosome and mitochondrial DNA polymorphisms, with the advantage of providing information about both male and female components of the population.Methodology/Principal FindingsWe identified a 47 Kb interval containing an Alu insertion polymorphism (DXS225) and four microsatellites in complete linkage disequilibrium in a low recombination rate region of the long arm of the human X chromosome. This haplotype block was studied in 667 males from the HGDP-CEPH Human Genome Diversity Panel. The haplotypic diversity was highest in Africa (0.992&plusmn;0.0025) and lowest in the Americas (0.839&plusmn;0.0378), where no insertion alleles of DXS225 were observed. Africa shared few haplotypes with other geographical areas, while those exhibited significant sharing among themselves. Median joining networks revealed that the African haplotypes were numerous, occupied the periphery of the graph and had low frequency, whereas those from the other continents were few, central and had high frequency. Altogether, our data support a single origin of modern man in Africa and migration to occupy the other continents by serial founder effects. Coalescent analysis permitted estimation of the time of the most recent common ancestor as 182,000 years (56,700&ndash;479,000) and the estimated time of the DXS225 Alu insertion of 94,400 years (24,300&ndash;310,000). These dates are fully compatible with the current widely accepted scenario of the origin of modern mankind in Africa within the last 195,000 years and migration out-of-Africa circa 55,000&ndash;65,000 years ago.Conclusions/SignificanceA haplotypic block combining an Alu insertion polymorphism and four microsatellite markers on the human X chromosome is a useful marker to evaluate genetic diversity of human populations and provides a highly informative tool for evolutionary studies."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000557.g003"/>

  <meta property="og:title" content="A Worldwide Phylogeography for the Human X Chromosome" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000557" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1612'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3289'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9950&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000557">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000557" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000557&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Worldwide Phylogeography for the Human X Chromosome
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Simone S. Santos-Lopes, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departamento de Bioquímica e Imunologia, Universidade Federal de Minas Gerais, Belo Horizonte, Minas Gerais, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rinaldo W. Pereira, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Departamento de Bioquímica e Imunologia, Universidade Federal de Minas Gerais, Belo Horizonte, Minas Gerais, Brazil, 
                  Programa de Pós Graduação em Ciências Genômicas e Biotecnologia, Catholic University of Brasília (UCB), Brasília, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ian J. Wilson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Human Genetics, Newcastle University, Newcastle, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sérgio D.J. Pena
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:spena@dcc.ufmg.br">spena@dcc.ufmg.br</a></p>

                <p>Affiliation:
                  Departamento de Bioquímica e Imunologia, Universidade Federal de Minas Gerais, Belo Horizonte, Minas Gerais, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 27, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000557</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000557" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000557" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000557" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000557" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000557-t001" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000557-t002" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000557-g001" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000557-t003" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000557-g002" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000557-g003" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000557-t004" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000557">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000557" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>We reasoned that by identifying genetic markers on human X chromosome regions where recombination is rare or absent, we should be able to construct X chromosome genealogies analogous to those based on Y chromosome and mitochondrial DNA polymorphisms, with the advantage of providing information about both male and female components of the population.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We identified a 47 Kb interval containing an <em>Alu</em> insertion polymorphism (<em>DXS225</em>) and four microsatellites in complete linkage disequilibrium in a low recombination rate region of the long arm of the human X chromosome. This haplotype block was studied in 667 males from the HGDP-CEPH Human Genome Diversity Panel. The haplotypic diversity was highest in Africa (0.992±0.0025) and lowest in the Americas (0.839±0.0378), where no insertion alleles of <em>DXS225</em> were observed. Africa shared few haplotypes with other geographical areas, while those exhibited significant sharing among themselves. Median joining networks revealed that the African haplotypes were numerous, occupied the periphery of the graph and had low frequency, whereas those from the other continents were few, central and had high frequency. Altogether, our data support a single origin of modern man in Africa and migration to occupy the other continents by serial founder effects. Coalescent analysis permitted estimation of the time of the most recent common ancestor as 182,000 years (56,700–479,000) and the estimated time of the <em>DXS225 Alu</em> insertion of 94,400 years (24,300–310,000). These dates are fully compatible with the current widely accepted scenario of the origin of modern mankind in Africa within the last 195,000 years and migration out-of-Africa <em>circa</em> 55,000–65,000 years ago.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>A haplotypic block combining an <em>Alu</em> insertion polymorphism and four microsatellite markers on the human X chromosome is a useful marker to evaluate genetic diversity of human populations and provides a highly informative tool for evolutionary studies.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Santos-Lopes SS, Pereira RW, Wilson IJ, Pena SD (2007) A Worldwide Phylogeography for the Human X Chromosome. PLoS ONE 2(6):
          e557.
            doi:10.1371/journal.pone.0000557</p><p><strong>Academic Editor: </strong>Neil Gemmell, University of Canterbury, New Zealand</p><p><strong>Received:</strong> March 22, 2007; <strong>Accepted:</strong> May 28, 2007; <strong>Published:</strong> June 27, 2007</p><p><strong>Copyright:</strong> © 2007 Santos-Lopes et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Supported by CNPq-Brazil through the Universal Grants program and the “Institutos do Milênio” program. Further support to Ian Wilson from the Royal Society Relocation Fellowship scheme.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Human Y chromosomes are haploid and lack recombination over most of their length. Thus, they are transmitted by males to their male offspring and remain unaltered from generation to generation, establishing patrilineages that remain stable until a mutation supervenes. Human Y chromosomal DNA polymorphisms are consequently paternal lineage markers that have been extremely useful in human evolutionary studies <a href="#pone.0000557-Jobling1">[1]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Since in males the X chromosome is also haploid, determination of haplotypes is straightforward. We reasoned that if we could identify genetic markers on the human X chromosome in regions where recombination is rare or absent, we might be able to study human X chromosome genealogies in an analogous fashion to those based on investigations of Y chromosome and mitochondrial DNA polymorphisms. These X chromosome genealogies would have the interesting peculiarity that in every generation half of the X chromosomes in females and all X chromosomes in males (2/3 of the total) will change sexes <a href="#pone.0000557-Schaffner1">[2]</a>. Thus, X chromosome lineages should provide simultaneous information about both the male and female components of the population. This contrasts with Y chromosome genealogies, which examine only patrilineages, and with mtDNA genealogies, which examine only matrilineages. Several authors have emphasized that the history of patrilineages and matrilineages in human populations are diverse <a href="#pone.0000557-Wilkins1">[3]</a>. Thus, the comparison of X chromosome genealogies with those of Y chromosomes and mtDNA should be informative of past population history.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>With this in mind, we decided to study a region located between Xq13.3 and Xq21.3, with a recombination rate of 0.6 cM/Mb, a low rate when compared with the average X chromosome recombination rate of 1.3 cM/Mb <a href="#pone.0000557-Nagaraja1">[4]</a>. Within this region we located a young <em>Alu</em> element embedded within a <em>LINE-1</em> element, which proved to be polymorphic in humans. We recently reported <a href="#pone.0000557-Pereira1">[5]</a> a survey of the worldwide frequency distribution of the new polymorphic <em>Alu</em> insertion (named <em>DXS225</em>; GDB:11524531) in 677 males from the HGDP-CEPH Human Genome Diversity Panel <a href="#pone.0000557-Rosenberg1">[6]</a>. All regions of the globe, namely Africa, Middle East, Central Asia, Oceania, Europe and America, showed presence of the <em>Alu</em> sequence in polymorphic frequencies, indicating that insertion event took place before the modern human spread from Africa. Further analysis, however, revealed that among the five Amerindian populations in the CEPH panel and two other studied, only the Karitiana showed presence of the <em>Alu</em> insertion. The Karitiana are a very small group known to have had contact with European and African descendants in the early 20<sup>th</sup> century <a href="#pone.0000557-Storto1">[7]</a> and it is thus most likely that the <em>Alu</em> insertion allele was introduced into their gene pool by admixture. Thus, we believe that the <em>DXS225</em> is monomorphic in pre-Columbian Amerindians, conceivably because of a founder effect. Because of that, the Karitiana were removed from the analyses in the present article.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In an effort to increase the resolution power of our X-chromosome molecular analysis we searched for and identified seven microsatellites in a 118 Kb region containing the Alu insertion polymorphism. We typed these microsatellites in all 677 male samples of the HGDP-CEPH panel. Here, we report that four of these microsatellites, spanning a 47 Kb interval containing the <em>DXS225</em> locus, are in complete linkage disequilibrium, thus providing a hypervariable and highly informative haplotype block for inference about human evolution <a href="#pone.0000557-Mountain1">[8]</a>. The study of the worldwide variation of haplotypes in this region and its exploration using haplotype networks and coalescent analysis provides interesting new knowledge about the population history of humanity after its exodus from Africa.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Population samples</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>All unrelated male samples from HGDP-CEPH Human Genome Diversity Cell Line Panel <a href="#pone.0000557-Rosenberg1">[6]</a> were analyzed in this study. A total of 677 male individuals representing 52 different populations from seven regional groups worldwide (Africa, Europe, Middle East, Central/South Asia, East Asia, Oceania and America). However, as evidence obtained in our previous study with <em>DXS225</em> had shown that the Karitiana may have received gene flow from European and/or African populations and also because the group represents a single extended family <a href="#pone.0000557-Storto1">[7]</a>, we removed them from all further analyses. Thus, our final study sample numbered 667 males.</p>


<h4>DNA typing</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>DNA from each individual was independently typed for the <em>DXS225 Alu</em> insertion on X chromosome (Genome Data Base accession number GDB: 11524531) exactly as described elsewhere <a href="#pone.0000557-Pereira1">[5]</a>. As before, the two <em>DXS225</em> allelic states were identified as 0 (pre-insertion allele) or 1 (<em>Alu</em> insertion allele).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The following seven microsatellites located on Xq21 were also analyzed in all samples: <em>DXS995, DXS8076, DXS1012, DXS1002, DXS1019, DXS8114 and DXS1050.</em> The dinucleotide repeat microsatellites <em>DXS995, DXS8076, DXS1002, DXS8114</em> and <em>DXS1050</em> had been previously mapped to the chosen region by Dib et al. <a href="#pone.0000557-Dib1">[9]</a> and we used the primers described by them, with exception of the reverse primer of <em>DXS995</em> to which was added a tail of ten adenine residues in order to increase the amplicon size and avoid overlap with alleles of the locus <em>DXS8114</em> in the multiplex analysis. The pentanucleotide repeat microsatellite <em>DXS1012</em> and the dinucleotide microsatellite <em>DXS1019</em> were identified using the Tandem Repeats Finder program <a href="#pone.0000557-Benson1">[10]</a> in the interval 84,261,735 to 84,391,735 of the human X chromosome (GenBank Accession # NT_011651.16). All microsatellite alleles were identified by their repeat numbers. Primers were designed by routine techniques and after verification that the microsatellites were polymorphic, they were registered in the Genome DataBase with accession numbers GDB:11524532 for <em>DXS1012</em> and GDB:11524534 for <em>DXS1019</em>.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Microsatellites were amplified in multiplex PCR reactions, in a final volume of 10 µl, containing 50 ng of genomic DNA, and separated in multiplex reaction in the capillary automatic sequencer MegaBACE 1000 (GE Healthcare). The results were analyzed using the program Fragment Profile version 1.2 (GE Healthcare).</p>


<h4>Statistical Analyses</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The genetic structure of the populations and basic parameters of molecular diversity, including analyses of molecular variance (AMOVA) <a href="#pone.0000557-Excoffier1">[11]</a>, haplotype frequency, haplotype diversity, haplotype sharing and linkage disequilibrium analyses were calculated using the package <em>Arlequin</em> 2.0 <a href="#pone.0000557-Schneider1">[12]</a>. The Product of Approximate Conditionals model of Li and Stephens <a href="#pone.0000557-Li1">[13]</a> was used to further investigate the recombination rate over the entire region and over the proposed non-recombining block (<em>DSX1012, DXS1002 DXS225, DXS1019 and DXS8114</em>). This method depends on using a fixed value for the scaled population mutation rate, <em>ϑ</em> which Li and Stephens <a href="#pone.0000557-Li1">[13]</a> call <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.e001&amp;representation=PNG" class="inline-graphic"></span>. Values for <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.e002&amp;representation=PNG" class="inline-graphic"></span> from 10 to 60 were investigated, consistent with known microsatellite mutation rates and effective population size for the X chromosome. While the Li and Stephens <a href="#pone.0000557-Li1">[13]</a> model is generally corrected for the number of sites and sequences, we were not interested in a precise estimate of <em>ρ</em>, rather we wanted to test whether it was different from zero and thus we did not apply their correction.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Median-joining networks were constructed using the software Network 4.1.0.6 <a href="#pone.0000557-Bandelt1">[14]</a> available at <a href="http://www.fluxus-engineering.com">www.fluxus-engineering.com</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The program BATWING <a href="#pone.0000557-Wilson1">[15]</a>, <a href="#pone.0000557-Wilson2">[16]</a> was used for a genealogical analysis. BATWING uses Markov chain Monte Carlo (MCMC) techniques to sample many reconstructed genealogies proportional to their probability under the coalescent model (for background see Wilson et al. <a href="#pone.0000557-Wilson2">[16]</a>) in a Bayesian framework. These reconstructed population histories depend on models for mutation and the expected genealogical structure and <em>prior</em> distributions for parameters of interest. By summarizing the population histories we can see the sorts of population history and ranges of parameters that are consistent with the data in the present.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Further modeling of the population structure is achieved by having a <em>supertree</em> that describes each population's history as a sequence of splitting events; this is different to <em>island</em> models of structure that assume fixed populations with migrations between them. While the supertree model should not be taken too literally-splitting may take place over many generations and later admixture is always likely–this allows us to take account of the non-random nature of sampling and the correlations between the population histories of individuals within subpopulations.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Linkage disequilibrium</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We used the <em>Arlequin</em> 2.0 program <a href="#pone.0000557-Schneider1">[12]</a> to perform linkage disequilibrium (LD) analyses of the seven microsatellites (<em>DXS995, DXS8076, DXS1012, DXS1002, DXS1019, DXS8114, DXS1050</em>) and the <em>Alu</em> insertion (<em>DXS225</em>) using data from 667 males in the HGDP-CEPH Diversity Panel <a href="#pone.0000557-Rosenberg1">[6]</a>. According to the March 2006 version of the UCSC Genome Browser (<a href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a>), the loci are in the order given below and occupy the following positions in contig NT_011651.16 that contains the sequence of the X chromosome: <em>DXS995</em> (82,643,697-82,644,081 pb); <em>DXS8076</em> (82,665,965-82,666,202 pb); <em>DXS1012</em> (85,409,962-85,410,320 pb); <em>DXS1002</em> (85,413,714-85,414,062 pb); <em>DXS225</em> (85,424,344-85,424,694 pb ); <em>DXS1019</em> (85,425,383-85,425,527 pb); <em>DXS8114</em> (85,500,625-85,501,030 pb); <em>DXS1050</em> (87,160,603-87,160,886 pb).</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The linkage disequilibrium test performed by the <em>Arlequin</em> 2.0 program is an extension of Fisher exact probability test on contingency tables and the results are reported as <em>P</em>-values with standard errors <a href="#pone.0000557-Schneider1">[12]</a>. Obviously, small <em>P</em>-values indicate high linkage disequilibrium. As shown in the part below the diagonal of <a href="#pone-0000557-t001">Table 1</a>, we observed linkage disequilibrium for all pairwise tests of markers <em>DXS1012, DXS1002, DXS225, DXS1019</em> and <em>DXS8114</em> (<a href="#pone-0000557-t001">Table 1</a>). However, no significant linkage disequilibrium was observed between the external loci <em>DXS995, DXS8076 and DXS1050</em> (<a href="#pone-0000557-t001">Table 1</a>).</p>
<div class="figure" id="pone-0000557-t001"><div class="img"><a name="pone-0000557-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000557.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000557.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Pairwise linkage disequilibrium between the seven microsatellites and the polymorphic <em>Alu</em> insertion.</span></strong></p><span>doi:10.1371/journal.pone.0000557.t001</span></div><a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Multiallelic D' values <a href="#pone.0000557-Hedrick1">[17]</a> are also shown in <a href="#pone-0000557-t001">Table 1</a>, above the diagonal. It should be observed that for the <em>DXS1012, DXS1002, DXS225, DXS1019</em> and <em>DXS8114</em> block the values go from a high of 0.94 down to 0.39 (<a href="#pone-0000557-t001">Table 1</a>). The problem is that it is difficult to predict theoretically exactly what range of values we would expect for highly variable microsatellites under complete linkage disequilibrium. To ascertain whether our D' values were consistent with zero recombination, we performed a small scale simulation study for four completely linked microsatellites with the same mutation rates as our sample (see below) and a single UEP. The simulations are a standard coalescent with 667 samples, with stepwise mutations for the microsatellites. In <a href="#pone.0000557.s002">Fig. S1</a> histograms of the minimum, mean (observed mean = 0.6) and maximum D' values seen in 1000 replicates are displayed. Inspection of the histograms reveals that our data are perfectly compatible with absolute linkage disequilibrium.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>As an additional test of linkage disequilibrium, the PAC method of recombination rate estimation <a href="#pone.0000557-Li1">[13]</a>, modified to deal with high mutation rate markers, was used to estimate the recombination rate for the region containing the markers <em>DXS1012, DXS1002, DXS225, DXS1019 and DXS8114</em>. The PAC analyses showed no evidence that the population recombination rate, <em>ρ</em>, was different from zero for the putative non-recombining sub-block while the entire region had a maximum <em>ρ</em> of about 1.5 cm/Mb when <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.e003&amp;representation=PNG" class="inline-graphic"></span> = 40 (further details are shown in <a href="#pone.0000557.s003">Fig. S2</a>).</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>From the above we conclude that our marker loci <em>DXS1012, DXS1002, DXS225, DXS1019</em> and <em>DXS8114</em> constitute a non-recombining haplotype block. All subsequent analyses were made using only these markers.</p>


<h4>Haplotypes and their diversity</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Among the 667 individuals studied (after removal of the Karitiana) we observed 187 different haplotypes of <em>DXS1012, DXS1002, DXS225, DXS1019</em> and <em>DXS8114.</em> The number of individuals studied and of haplotypes seen in each of the five major regions is shown in <a href="#pone-0000557-t002">Table 2</a>, together with haplotypic diversity estimates and their standard errors. The haplotypic diversity was highest in Africa (0.992±0.0025) and lowest in the Americas (0.839±0.0378), where no insertion alleles of <em>DXS225</em> were observed.</p>
<div class="figure" id="pone-0000557-t002"><div class="img"><a name="pone-0000557-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000557.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000557.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Number of haplotypes and the haplotypic diversity of the five regions from CEPH panel.</span></strong></p><span>doi:10.1371/journal.pone.0000557.t002</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Of the 187 haplotypes encountered, 129 (69.0%) were observed in one single geographical region. Africa contained only 14.7% of the individuals studied, but 44.2% (57/129) of the unique haplotypes. The proportion of shared haplotypes between the different regions is shown in <a href="#pone-0000557-g001">Fig. 1</a>. It is noticeable that Africa shares few haplotypes with the other geographical areas, which in turn display significant sharing among themselves.</p>
<div class="figure" id="pone-0000557-g001"><div class="img"><a name="pone-0000557-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000557.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000557.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Haplotypes shared among different regions of the world.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>The area of the rectangles is proportional to the size of the sample from each region. Arrow widths are proportional to the percentage of haplotype sharing from one region to another and the percentages are displayed in the arrows. For instance, 50% of the haplotypes of Oceania are present in East Asia, 20% are present in America, 20% are present in Africa and 30% are present in Eurasia. In contrast only 7.4% of East Asian haplotypes are shared with Oceania. This asymmetry suggests that East Asia is a parental population of Oceania. This figure was inspired by a similar diagram in Conrad et al. <a href="#pone.0000557-McDougall1">[33]</a>.</p>
<span>doi:10.1371/journal.pone.0000557.g001</span></div><a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>A hierarchical analysis of molecular variance (AMOVA) was performed on the haplotype data and is shown in <a href="#pone-0000557-t003">Table 3</a>. Our analysis of genetic variance showed very little genetic structure, with 95.2% within-population, 1.7% among-populations within-regions and 3.1% among-regions components of genetic variance. The same was true for each geographical group, with &gt;95% of the genetic variability occurring within the population level, except for Oceania (only two populations studied; 14.95% among-populations within-regions component) and the Americas (15.29% among-populations within-regions component).</p>
<div class="figure" id="pone-0000557-t003"><div class="img"><a name="pone-0000557-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000557.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000557.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Analysis of molecular variance (AMOVA) for the X haplotype block<span class="xref"><sup>*</sup></span>.</span></strong></p><span>doi:10.1371/journal.pone.0000557.t003</span></div>

<h4>Network analysis</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Median joining networks (<a href="#pone-0000557-g002">Fig. 2</a>) of haplotypes were drawn using the Network software v.4.201 <a href="#pone.0000557-Bandelt1">[14]</a>. In the networks, the 51 populations were color-coded into five groups: Africa, Eurasia, East Asia, Oceania and America. In the global network (<a href="#pone-0000557-g002">Fig. 2a</a>) it was noteworthy the fact that the African haplotypes (red) occupied the periphery of the graph and had low frequency, being often single.</p>
<div class="figure" id="pone-0000557-g002"><div class="img"><a name="pone-0000557-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000557.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000557.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>(A) Median joining network of all the haplotypes found in 667 individuals from the HGDP-CEPH Diversity Panel <a href="#pone.0000557-Rosenberg1">[6]</a>, color coded according to region of origin.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p><strong>(B)</strong> The same median joining network as in (A) with only the haplotypes containing the <em>DXS225</em><sup>0</sup> allele shown in color. The most widespread and most common haplotypes are concentrated on two clusters (arrows). <strong>(C)</strong> The same median joining network as in (A) with only the haplotypes containing the <em>DXS225</em><sup>1</sup> allele shown in color. The most frequent haplotypes belong to a cluster composed of 1,25,18,14,11 and 1,25,17,14,10 (arrows) and a few others.</p>
<span>doi:10.1371/journal.pone.0000557.g002</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>In <a href="#pone-0000557-g002">Fig. 2B</a> we can see the network with only the haplotypes containing the <em>DXS225</em><sup>0</sup> allele shown in color. The haplotype 0,27,21,15,9 (<em>DXS225, DXS1019, DXS8114, DXS1002</em> and <em>DXS1012</em> respectively) is the most common and also the only one seen in all five geographical regions of the world (<a href="#pone-0000557-g002">Fig.2B</a>, arrow). Immediately beside it are two closely related haplotypes also indicated by arrows in <a href="#pone-0000557-g002">Fig.2B:</a> 0,27,21,15,10 (seen in all regions, except Oceania) and 0,27,20,15,9 (seen in East Asia, Oceania and America). The wide geographical spread of these three haplotypes, suggests that one or more of them were among the founder haplotypes in the migrant group that emerged from Africa to populate other continents. A second very common haplotype is 0,27,20,16,13 (arrow in <a href="#pone-0000557-g002">Fig. 2B</a>) seen in Eurasia and East Asia. Clustered with it are two other haplotypes (0,27,21,16,13 and 0,27,21,16,14; <a href="#pone-0000557-g002">Fig. 2B</a>, arrows) both found in Eurasia, East Asia and Oceania. Because of its frequency and wide geographical spread, this family of closely related haplotypes is also a candidate for a second founding effect in the emergence of <em>Homo sapiens</em> from Africa.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Moving now to the haplotypes containing the <em>DXS225</em><sup>1</sup> allele (<a href="#pone-0000557-g002">Fig.2C</a>) we observe that the most frequent haplotypes belong to a cluster composed of 1,25,18,14,11 and 1,25,17,14,10 (arrows) and a few others. This again suggests a founder effect in the out-of-Africa migration.</p>


<h4>Coalescent analyses</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Within each subpopulation, we modeled the genealogy using the coalescent with growth from a constant sized population, as described in Wilson et al. <a href="#pone.0000557-Wilson2">[16]</a>. This model assumes that a small ancestral population (with ancestral population size <em>N</em>) grows at a rate α% per generation until it has size Nexp(κ) in the present; this determines how long ago growth started. In this analysis <em>priors</em> are needed for the parameters estimated in the model: the mutation rate (<em>μ</em>) the growth rate per generation (<em>α</em>), the relative sizes of the current and ancestral populations (κ), the ancestral population size (<em>N</em>), and parameters that determine the expected shape of the population supertree.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>We assumed a stepwise mutation model for the four STR loci: <em>DXS1019, DXS8114, DXS1002, and DXS1012</em>, and assumed that <em>DXS225</em> is a unique event polymorphism (UEP), i.e., only one mutation has historically occurred at this site. We have used a different mutation rate for each STR, with a gamma distribution with parameters 1.35 and 740.4 to give a mean mutation rate of 0.0018 with a standard deviation of 0.0016. The shape parameter was estimated from the survey of relative mutation rates in Xu et al. <a href="#pone.0000557-Xu1">[18]</a>. The scale parameter was chosen to give an overall mean mutation rate of approximately 2×10<sup>3</sup> <a href="#pone.0000557-Mountain1">[8]</a>, <a href="#pone.0000557-Leopoldino1">[19]</a>. All datasets analyzed gave a similar signal for the relative mutation rates of the four loci, with <em>DXS1019</em> having an order of magnitude lower mutation rate than <em>DXS8114</em> and <em>DXS1012</em>. <em>DXS1002</em> had an intermediate value. A comparison of the prior and posterior population parameters is shown in <a href="#pone-0000557-g003">Fig. 3</a>.</p>
<div class="figure" id="pone-0000557-g003"><div class="img"><a name="pone-0000557-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000557.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000557.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Results from BATWING analysis of the full data set (except Karitiana and Oceania samples).</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>Posteriors histograms (shared bars) and prior densities (solid lines) shown for the mutation rate per generation for the four microsatellites (labels in figure) within the non-recombining region. Posterior histograms are from 200,000 BATWING outputs. Prior density is a gamma(1.35,740.4) for all four microsatellites.</p>
<span>doi:10.1371/journal.pone.0000557.g003</span></div><a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>For the coalescent analysis the populations from Europe, Central Asia and Middle East were treated separately, rather than as a single Eurasian group. Since the sample from Oceania was small it was left out of the analysis, as was the Karitiana as explained above. All analyses used five independent B<span class="small-caps">atwing</span> runs of 42,000 samples with the first 2000 removed from each and with 100 tree rearrangements between each attempted change to the population parameters and only every 200<sup>th</sup> sample taken. This gave a sample size of 200,000 to construct the empirical posterior distribution. These were very long runs but were used to ensure that the very complex joint model spaces of genealogies and population trees were explored. The results are shown in <a href="#pone-0000557-t004">Table 4</a>. Of special interest are the estimates of the time of the most recent common ancestor (TMRCA) of 182,000 years (95% confidence limits 56,700–479,000) and the estimated time of the <em>DXS225</em> Alu insertion of 94,400 years (95% confidence limits 24,300-310,000).</p>
<div class="figure" id="pone-0000557-t004"><div class="img"><a name="pone-0000557-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000557" data-uri="info:doi/10.1371/journal.pone.0000557.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000557.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000557.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000557.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000557.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000557.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Posterior means and quantiles for BATWING analysis.</span></strong></p><span>doi:10.1371/journal.pone.0000557.t004</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>In this study we have used a haplotypic block in X chromosome formed by four microsatellites and one <em>Alu</em> insertion to study a worldwide sample of human DNA from the HGDP-CEPH Human Genome Diversity Cell Line Panel <a href="#pone.0000557-Rosenberg1">[6]</a>. The individuals studied belonged to 52 different populations from seven continental groups (Africa, Europe, Middle East, Central/South Asia, East Asia, Oceania and America). Because Europe, Middle East, Central/South Asia are known to have a very similar genetic structure <a href="#pone.0000557-Rosenberg2">[20]</a>, we decided to pool them into a single group that we called Eurasia. Also, because of evidence indicative that the Karitiana may present admixture from European and/or African sources <a href="#pone.0000557-Pereira1">[5]</a>, we elected to exclude them from our analyses.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>We reasoned that if we could identify genetic markers on the human X chromosome in regions where recombination is absent, we might be able to unravel human X chromosome genealogies in an analogous fashion to those based on investigations of Y chromosome and mitochondrial DNA polymorphisms. These X chromosome lineages should provide simultaneous information about both the male and female components of the population.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>To validate the haplotypic block as a useful non-recombining X-chromosome lineage marker we first determined, using three different statistical approaches that the loci <em>DXS1012, DXS1002, DXS225, DXS1019</em> and <em>DXS8114</em> were in absolute linkage disequilibrium (<a href="#pone-0000557-t001">Table 1</a>). <em>DXS225</em> is a unique event polymorphism characterized by a variable <em>Alu</em> insertion that can be seen in all worldwide populations, except in Amerindians <a href="#pone.0000557-Pereira1">[5]</a>. <em>DXS1019, DXS1002,</em> and <em>DXS8114</em> are dinucleotide repeats and <em>DXS1012</em> is a pentanucleotide repeat microsatellite.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Three indirect lines of evidence add support to our idea that recombination between our markers did not happen at all or was a very rare event that should not affect our analyses. First, we obtained no evidence of crossover from European, Chinese and Japanese data for the HapMap <a href="#pone.0000557-The1">[21]</a> (data from HapMap release #22), with the vast majority of SNPs being in complete LD over the entire region (approximately 90% of D' = 1). There were some SNPs not in complete LD, but these were consistent with a small number of gene conversions affecting single SNPs. All LD calculations were performed using Haploview <a href="#pone.0000557-Barrett1">[22]</a> Second, we could not find any evidence from the maps of Myers et al <a href="#pone.0000557-Myers1">[23]</a> that there were any recombination hotspots within the region delimited by <em>DXS1012</em> and <em>DXS8114</em>. Finally, the fact that we have observed in the networks a fairly strong founder effect with a small cluster of haplotypes (separated by single step mutations) constituting the vast majority outside Africa provided further evidence for absence of recombination. If recombination had occurred, these founder effects would have dissipated by now.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>As expected from the combined variability of the four microsatellites, we have observed 187 different haplotypes among the 667 individuals studied, with a very high haplotypic diversity of 0. 9756±0.0022. Most individuals (477/667; 71.5%) carried the pre-insertion allele at <em>DXS225</em> (<em>DXS225</em><sup>0</sup>) and accounted for a total of 141 haplotypes, a diversity of 0.9718±0.0031. The haplotype diversity for carriers of the insertion allele (<em>DXS225</em><sup>1</sup>) was 0.8794±0.0167.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>The haplotypic diversity of each of the five regions is shown in <a href="#pone-0000557-t002">Table 2</a>. As expected, Africa presented the largest haplotypic diversity. While encompassing only 14.3% of all the individuals studied, Africa contained 44.2% of the unique haplotypes. Accordingly, Africa had small levels of haplotype sharing with other regions (<a href="#pone-0000557-g001">Fig. 1</a>) and in the networks (<a href="#pone-0000557-g002">Fig. 2</a>) African haplotypes mostly occupied the periphery of the graph. Altogether, these observations suggest that the African samples present in the HGDP-CEPH Human Genome Diversity Cell Line Panel embrace only a small portion of the total African haplotypic variability. These data are compatible with the view that modern man emerged in Africa and migrated from that continent to populate all other areas of Earth (reviewed in <a href="#pone.0000557-CavalliSforza1">[24]</a>, <a href="#pone.0000557-Pena1">[25]</a>).</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>A hierarchical analysis of molecular variance (AMOVA) was performed (<a href="#pone-0000557-t003">Table 3</a>) and revealed negligible amounts of genetic structure. The only exceptions were Oceania and America with, respectively, 14.95% and 15.29% among-populations within-regions component of variation. Since only two populations were studied in the former, it is difficult to ascertain the meaning of this relatively elevated component. As to America, it is well known <a href="#pone.0000557-CavalliSforza2">[26]</a> that the genetic structure of Amerindian populations is characterized by high levels of genetic drift.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>Because of the asymmetric nature of haplotype sharing between populations with unequal sample size, its analysis (<a href="#pone-0000557-g001">Fig. 1</a>) reveal genealogical relationships between populations. For instance, our East Asian sample contains 50% of the haplotypes seen in Oceania, but the latter includes only 7% of the haplotypes observed in East Asia. This asymmetry suggests that East Asia has a parental genealogical relationship with Oceania. Likewise, 70% of the haplotypes seen in America are also seen in East Asia while these shared haplotypes make up only 10.4% of the haplotypes of East Asia. In principle this not only suggests that East Asia has a parental genealogical relationship with America (as is known to be the case; see, for instance, <a href="#pone.0000557-Santos1">[27]</a>), but also that this did not occur too long ago, otherwise there would have been time for much higher levels of mutation-driven haplotypic diversification in Amerindians.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p>As a consequence of the stepwise nature of microsatellite mutation it is possible to construct useful haplotypic networks (<a href="#pone-0000557-g002">Fig. 2</a>). These show significant different patterns for Africa and the other continents. In the former, as noted above, haplotypes (red) occupy the periphery of the graph and occur at low frequencies, being often single (<a href="#pone-0000557-g002">Fig 2</a>). On the other hand, in Eurasia, East Asia, Oceania and America, the network for carriers of the <em>DXS225</em><sup>0</sup> allele was concentrated primarily on two haplotypic clusters (<a href="#pone-0000557-g002">Fig. 2B</a>) while for carriers of the <em>DXS225</em><sup>1</sup> allele it was concentrated primarily on one single haplotypic clusters (<a href="#pone-0000557-g002">Fig. 2C</a>). This concentration on few haplotypic clusters outside of Africa can be verified by the estimates of haplotype diversity, which for carriers of the for <em>DXS225</em><sup>0</sup> allele were 0.9918±0.0034 in Africa and 0.9623±0.0040 in the other continents. For <em>DXS225</em><sup>1</sup> carriers the haplotype diversity was 0.9523±0.0040 in Africa and 0.8549±0.0201 in the other continents. Thus, as expected from the lower frequency of the <em>DXS225</em><sup>1</sup> the bottleneck was more severe for carriers of the insertion allele in <em>DXS225</em>. All these observations match perfectly the known fact that the migration of modern humanity out-of-Africa was associated with a population size bottleneck and reduction of variability, as has been shown for mitochondrial DNA <a href="#pone.0000557-Torroni1">[28]</a> and several other markers <a href="#pone.0000557-Yu1">[29]</a>.</p>
<a id="article1.body1.sec4.p10" name="article1.body1.sec4.p10"></a><p>Prugnolle et al. <a href="#pone.0000557-Prugnolle1">[30]</a> showed that geographic distance–not genetic distance–from East Africa along likely colonization routes was highly correlated with the genetic diversity of human populations. These and other observations led to an attractive model of human expansion out-of-Africa having occurred by a series of founder effects <a href="#pone.0000557-Ramachandran1">[31]</a>, <a href="#pone.0000557-Liu1">[32]</a>. Our data are quite in harmony with this serial founder effect model. We first observed a large reduction of haplotypic diversity in the passage from Africa (haplotypic diversity = 0.992±0.003) to East Asia (0.967±0.005), followed by a smaller reduction of haplotypic variability from East Asia to Eurasia (0.953±0.006) and a steeper one from East Asia to Oceania (0.885±0.047). Finally, the peopling of the Americas from East Asia was again accompanied by a significant bottleneck that led to a large reduction of haplotypic diversity (0.839±0.038) and, we believe, to the exclusion of the less frequent <em>DXS225</em> insertion allele. Our asymmetric haplotype sharing model (<a href="#pone-0000557-g001">Fig. 1</a>) is completely coherent with this view.</p>
<a id="article1.body1.sec4.p11" name="article1.body1.sec4.p11"></a><p>A corollary of the stepwise mutation nature of microsatellite mutations is the fact that we can use microsatellite variability to estimate the timing of evolutionary events. From the coalescent analysis we could estimate the time of the most recent ancestor (TMRCA) as 182,000 years (95% confidence limits 56,700–479,000) for all the individuals studied (<a href="#pone-0000557-t004">Table 4</a>). This is fully compatible with the genetic and paleontological evidence of the origin of modern mankind in Africa within the last 195,000 years <a href="#pone.0000557-McDougall1">[33]</a>. We also estimated (<a href="#pone-0000557-t004">Table 4</a>) that the <em>Alu</em> insertion event in DXS225 occurred 94,400 years (95% confidence limits 24,300–310,000). This date is quite consistent with the notion that the Alu insertion occurred in Africa, predating the out-of-Africa migration of man to Asia, which was calculated to have occurred 55,000–65,000 years ago <a href="#pone.0000557-Liu1">[32]</a>, <a href="#pone.0000557-Mellars1">[34]</a>.</p>
<a id="article1.body1.sec4.p12" name="article1.body1.sec4.p12"></a><p>In conclusion, our results demonstrate that a haplotypic block on the human X chromosome is a useful marker to evaluate genetic diversity of human populations and the combination of an <em>Alu</em> insertion polymorphism and microsatellite markers provides a highly informative tool for population and evolutionary studies. Although this is essentially the tale of a single “gene”, similarly to other haplotypes blocks such as the Y chromosome and mitochondrial DNA, it presents a novel, less sex-biased adjunct to studies performed with these traditional markers. Other non-recombining regions in the human genome, as they are identified, can likewise be explored for phylogeographical studies, each one providing a fresh perspective on human evolutionary history. Hopefully, from this multitude of points of view, a coherent picture of the genealogy of the human species will emerge.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000557"></div><a name="pone.0000557.s001" id="pone.0000557.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000557.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Table of prior means and quantiles for the parameters in the model.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000557.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000557.s002" id="pone.0000557.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000557.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Values for the minimum, mean and maximum values of D' <a href="#pone.0000557-Xu1">[18]</a> calculated from 10,000 coalescent simulations of 667 individuals each with four completely linked microsatellites (with theta = 4,10,30, and 30 respectively) and one unique event polymorphism (that was selected to have a frequency close to 0.3). The simulations are for a single panmictic population of constant size. Observed values from data are shown by blue arrows.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000557.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.01 MB EPS)</p>
<a name="pone.0000557.s003" id="pone.0000557.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000557.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Scaled approximate log-likelihood curves from the PAC model of Li&amp;Stephens <a href="#pone.0000557-Li1">[13]</a> for the entire regions (black lines) and for the proposed non-recombining block of DSX1012, DXS1002 DXS225, DXS1019 and DXS8114 (blue lines) for estimated theta = 10 (dashed lines) and estimated theta = 40 (solid lines). All lines are scaled so that the maximum value is at zero, so allow comparison of lines.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000557.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.01 MB EPS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Neuza A. Rodrigues and Katia B. Gonçalves for expert technical assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SP. Performed the experiments: SS RP. Analyzed the data: SP IW. Wrote the paper: SP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000557-Jobling1" id="pone.0000557-Jobling1"></a>Jobling MA, Tyler-Smith C (2003) The human Y chromosome: an evolutionary marker comes of age. Nat Rev Genet  4: 598–612.  <ul class="find" data-citedArticleID="961346" data-doi="10.1038/nrg1124"><li><a href="http://dx.doi.org/10.1038/nrg1124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+Y+chromosome%3A+an+evolutionary+marker+comes+of+age." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+Y+chromosome%3A+an+evolutionary+marker+comes+of+age.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000557-Schaffner1" id="pone.0000557-Schaffner1"></a>Schaffner SF (2004) The X chromosome in population genetics. Nat Rev Genet  5: 43–51.  <ul class="find" data-citedArticleID="961378" data-doi="10.1038/nrg1247"><li><a href="http://dx.doi.org/10.1038/nrg1247" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+X+chromosome+in+population+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+X+chromosome+in+population+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000557-Wilkins1" id="pone.0000557-Wilkins1"></a>Wilkins JF, Marlowe FW (2006) Sex-biased migration in humans: what should we expect from genetic data? Bioessays  28: 290–300.  <ul class="find" data-citedArticleID="961388" data-doi="10.1002/bies.20378"><li><a href="http://dx.doi.org/10.1002/bies.20378" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sex-biased+migration+in+humans%3A+what+should+we+expect+from+genetic+data%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sex-biased+migration+in+humans%3A+what+should+we+expect+from+genetic+data%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000557-Nagaraja1" id="pone.0000557-Nagaraja1"></a>Nagaraja R, MacMillan S, Kere J, Jones C, Griffin S, et al.  (1997) X chromosome map at 75-kb STS resolution, revealing extremes of recombination and GC content. Genome Res  7: 210–222.  <ul class="find" data-citedArticleID="961362" data-doi="10.1101/gr.7.3.210"><li><a href="http://dx.doi.org/10.1101/gr.7.3.210" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=X+chromosome+map+at+75-kb+STS+resolution%2C+revealing+extremes+of+recombination+and+GC+content." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22X+chromosome+map+at+75-kb+STS+resolution%2C+revealing+extremes+of+recombination+and+GC+content.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000557-Pereira1" id="pone.0000557-Pereira1"></a>Pereira RW, Santos SS, Pena SD (2006) A novel polymorphic Alu insertion embedded in a LINE 1 retrotransposon in the human X chromosome (DXS225): identification and worldwide population study. Genet Mol Res  5: 63–71.  <ul class="find" data-citedArticleID="961366"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+novel+polymorphic+Alu+insertion+embedded+in+a+LINE+1+retrotransposon+in+the+human+X+chromosome+%28DXS225%29%3A+identification+and+worldwide+population+study.&amp;auth=&amp;atitle=A+novel+polymorphic+Alu+insertion+embedded+in+a+LINE+1+retrotransposon+in+the+human+X+chromosome+%28DXS225%29%3A+identification+and+worldwide+population+study." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+polymorphic+Alu+insertion+embedded+in+a+LINE+1+retrotransposon+in+the+human+X+chromosome+%28DXS225%29%3A+identification+and+worldwide+population+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+polymorphic+Alu+insertion+embedded+in+a+LINE+1+retrotransposon+in+the+human+X+chromosome+%28DXS225%29%3A+identification+and+worldwide+population+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000557-Rosenberg1" id="pone.0000557-Rosenberg1"></a>Rosenberg NA (2006) Standardized subsets of the HGDP-CEPH Human Genome Diversity Cell Line Panel, accounting for atypical and duplicated samples and pairs of close relatives. Ann Hum Genet.  70: 841–847.  <ul class="find" data-citedArticleID="961372" data-doi="10.1111/j.1469-1809.2006.00285.x"><li><a href="http://dx.doi.org/10.1111/j.1469-1809.2006.00285.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Standardized+subsets+of+the+HGDP-CEPH+Human+Genome+Diversity+Cell+Line+Panel%2C+accounting+for+atypical+and+duplicated+samples+and+pairs+of+close+relatives." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Standardized+subsets+of+the+HGDP-CEPH+Human+Genome+Diversity+Cell+Line+Panel%2C+accounting+for+atypical+and+duplicated+samples+and+pairs+of+close+relatives.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000557-Storto1" id="pone.0000557-Storto1"></a>Storto L, Velden FFV (2005) Povos Indígenas no Brasil-Karitiana.   Available:<a href="http://www.socioambiental.org/pib/epi/karitiana/karitiana.shtm">http://www.socioambiental.org/pib/epi/ka​ritiana/karitiana.shtm</a>, Accessed 15 March 2007.  <ul class="find-nolinks"></ul></li><li><span class="label">8.
              </span><a name="pone.0000557-Mountain1" id="pone.0000557-Mountain1"></a>Mountain JL, Knight A, Jobin M, Gignoux C, Miller A, et al.  (2002) SNPSTRs: empirically derived, rapidly typed, autosomal haplotypes for inference of population history and mutational processes. Genome Res  12: 1766–1772.  <ul class="find" data-citedArticleID="961358" data-doi="10.1101/gr.238602"><li><a href="http://dx.doi.org/10.1101/gr.238602" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SNPSTRs%3A+empirically+derived%2C+rapidly+typed%2C+autosomal+haplotypes+for+inference+of+population+history+and+mutational+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SNPSTRs%3A+empirically+derived%2C+rapidly+typed%2C+autosomal+haplotypes+for+inference+of+population+history+and+mutational+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000557-Dib1" id="pone.0000557-Dib1"></a>Dib C, Faure S, Fizames C, Samson D, Drouot N, et al.  (1996) A comprehensive genetic map of the human genome based on 5,264 microsatellites. Nature  380: 152–154.  <ul class="find" data-citedArticleID="961340" data-doi="10.1038/380152a0"><li><a href="http://dx.doi.org/10.1038/380152a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+genetic+map+of+the+human+genome+based+on+5%2C264+microsatellites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+genetic+map+of+the+human+genome+based+on+5%2C264+microsatellites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000557-Benson1" id="pone.0000557-Benson1"></a>Benson G (1999) Tandem repeats finder: a program to analyze DNA sequences. Nuc Acid Res  27: 573–580.  <ul class="find" data-citedArticleID="961332" data-doi="10.1093/nar/27.2.573"><li><a href="http://dx.doi.org/10.1093/nar/27.2.573" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tandem+repeats+finder%3A+a+program+to+analyze+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tandem+repeats+finder%3A+a+program+to+analyze+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000557-Excoffier1" id="pone.0000557-Excoffier1"></a>Excoffier L, Smouse PE, Quattro JM (1992) Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data. Genetics  131: 479–491.  <ul class="find" data-citedArticleID="961342"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+application+to+human+mitochondrial+DNA+restriction+data.&amp;auth=&amp;atitle=Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+application+to+human+mitochondrial+DNA+restriction+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+application+to+human+mitochondrial+DNA+restriction+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+application+to+human+mitochondrial+DNA+restriction+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000557-Schneider1" id="pone.0000557-Schneider1"></a>Schneider S, Roessli D, Excoffier L (2000) Arlequin ver. 2000: a software for population genetics data analysis. Genetics and biometry laboratory, Switzerland: University of Geneva.   <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0000557-Li1" id="pone.0000557-Li1"></a>Li N, Stephens M (2003) Modeling linkage disequilibrium and identifying recombination hotspots using single-nucleotide polymorphism data. Genetics  165: 2213–2233.  <ul class="find" data-citedArticleID="961350"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Modeling+linkage+disequilibrium+and+identifying+recombination+hotspots+using+single-nucleotide+polymorphism+data.&amp;auth=&amp;atitle=Modeling+linkage+disequilibrium+and+identifying+recombination+hotspots+using+single-nucleotide+polymorphism+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+linkage+disequilibrium+and+identifying+recombination+hotspots+using+single-nucleotide+polymorphism+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+linkage+disequilibrium+and+identifying+recombination+hotspots+using+single-nucleotide+polymorphism+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000557-Bandelt1" id="pone.0000557-Bandelt1"></a>Bandelt HJ, Forster P, Rohl A (1999) Median-joining networks for inferring intraspecific phylogenies. Mol Biol Evol  16: 37–48.  <ul class="find" data-citedArticleID="961328" data-doi="10.1093/oxfordjournals.molbev.a026036"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Median-joining+networks+for+inferring+intraspecific+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Median-joining+networks+for+inferring+intraspecific+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000557-Wilson1" id="pone.0000557-Wilson1"></a>Wilson IJ, Balding DG (1998) Genealogical inference from microsatellite data. Genetics  150: 499–510.  <ul class="find" data-citedArticleID="961390"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genealogical+inference+from+microsatellite+data.&amp;auth=&amp;atitle=Genealogical+inference+from+microsatellite+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genealogical+inference+from+microsatellite+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genealogical+inference+from+microsatellite+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000557-Wilson2" id="pone.0000557-Wilson2"></a>Wilson IJ, Weale ME, Balding DJ (2003) Inferences from DNA data: population histories, evolutionary processes and forensic match probabilities. J R Stat Soc Ser A  166: 155–201.  <ul class="find" data-citedArticleID="961392" data-doi="10.1111/1467-985x.00264"><li><a href="http://dx.doi.org/10.1111/1467-985x.00264" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferences+from+DNA+data%3A+population+histories%2C+evolutionary+processes+and+forensic+match+probabilities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferences+from+DNA+data%3A+population+histories%2C+evolutionary+processes+and+forensic+match+probabilities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000557-Hedrick1" id="pone.0000557-Hedrick1"></a>Hedrick WP (1987) Gametic disequilibrium measures: proceed with caution. Genetics  117: 331–341.  <ul class="find" data-citedArticleID="961344"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gametic+disequilibrium+measures%3A+proceed+with+caution.&amp;auth=&amp;atitle=Gametic+disequilibrium+measures%3A+proceed+with+caution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gametic+disequilibrium+measures%3A+proceed+with+caution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gametic+disequilibrium+measures%3A+proceed+with+caution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000557-Xu1" id="pone.0000557-Xu1"></a>Xu H, Chakraborty R, Fu Y-X (2005) Mutation rate variation at human dinucleotide microsatellites. Genetics  170: 305–312.  <ul class="find" data-citedArticleID="961394" data-doi="10.1534/genetics.104.036855"><li><a href="http://dx.doi.org/10.1534/genetics.104.036855" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+rate+variation+at+human+dinucleotide+microsatellites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+rate+variation+at+human+dinucleotide+microsatellites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000557-Leopoldino1" id="pone.0000557-Leopoldino1"></a>Leopoldino AM, Pena SD (2003) The mutational spectrum of human autosomal tetranucleotide microsatellites. Hum Mutat  21: 71–79.  <ul class="find" data-citedArticleID="961348" data-doi="10.1002/humu.10153"><li><a href="http://dx.doi.org/10.1002/humu.10153" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mutational+spectrum+of+human+autosomal+tetranucleotide+microsatellites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mutational+spectrum+of+human+autosomal+tetranucleotide+microsatellites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000557-Rosenberg2" id="pone.0000557-Rosenberg2"></a>Rosenberg NA, Pritchard JK, Weber JL, Cann HM, Kidd KK, Zhivotovsky LA, Feldman MW (2002) Genetic structure of human populations. Science.  298: 2381–2385.  <ul class="find" data-citedArticleID="961374" data-doi="10.1126/science.1078311"><li><a href="http://dx.doi.org/10.1126/science.1078311" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+structure+of+human+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+structure+of+human+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000557-The1" id="pone.0000557-The1"></a>The International HapMap Consortium (2005) A haplotype map of the human genome. Nature  437: 1299–1320.  <ul class="find" data-citedArticleID="961384" data-doi="10.1038/nature04226"><li><a href="http://dx.doi.org/10.1038/nature04226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+haplotype+map+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+haplotype+map+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000557-Barrett1" id="pone.0000557-Barrett1"></a>Barrett JC, Fry B, Maller J, Daly MJ (2005) Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics:  21: 263–265.  <ul class="find" data-citedArticleID="961330" data-doi="10.1093/bioinformatics/bth457"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth457" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haploview%3A+analysis+and+visualization+of+LD+and+haplotype+maps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Haploview%3A+analysis+and+visualization+of+LD+and+haplotype+maps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000557-Myers1" id="pone.0000557-Myers1"></a>Myers S, Bottolo L, Freeman C, McVean G, Donnelly P (2005) A fine-scale map of recombination rates and hotspots across the human genome. Science  310: 321–324.  <ul class="find" data-citedArticleID="961360" data-doi="10.1126/science.1117196"><li><a href="http://dx.doi.org/10.1126/science.1117196" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+fine-scale+map+of+recombination+rates+and+hotspots+across+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+fine-scale+map+of+recombination+rates+and+hotspots+across+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000557-CavalliSforza1" id="pone.0000557-CavalliSforza1"></a>Cavalli-Sforza LL, Feldman MW (2003) The application of molecular genetic approaches to the study of human evolution. Nat Genet  33: 266–275.  <ul class="find" data-citedArticleID="961334" data-doi="10.1038/ng1113"><li><a href="http://dx.doi.org/10.1038/ng1113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+application+of+molecular+genetic+approaches+to+the+study+of+human+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+application+of+molecular+genetic+approaches+to+the+study+of+human+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000557-Pena1" id="pone.0000557-Pena1"></a>Pena SDJ (2007)  The evolution and structure of human genetic diversity. In: Suarez-Kurtz G, editor. Pharmacogenomics in Admixed Populations. Austin: Landes Bioscience.   Available: <a href="http://www.eurekah.com/chapter/3190">http://www.eurekah.com/chapter/3190</a>, Accessed: 10 March 2007.  <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0000557-CavalliSforza2" id="pone.0000557-CavalliSforza2"></a>Cavalli-Sforza LL, Menozzi P, Piazza A (1994) The History and Geography of Human Genes. Princeton: Princeton University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0000557-Santos1" id="pone.0000557-Santos1"></a>Santos FR, Pandya A, Tyler-Smith C, Pena SDJ, Schanfield M, et al.  (1999) The central Siberian origin for Native American chromosomes. Am J Hum Genet  64: 619–628.  <ul class="find" data-citedArticleID="961376" data-doi="10.1086/302242"><li><a href="http://dx.doi.org/10.1086/302242" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+central+Siberian+origin+for+Native+American+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+central+Siberian+origin+for+Native+American+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000557-Torroni1" id="pone.0000557-Torroni1"></a>Torroni A, Achilli A, Macaulay V, Richards M, Bandelt HJ (2006) Harvesting the fruit of the human mtDNA tree. Trends Genet  22: 339–345.  <ul class="find" data-citedArticleID="961386" data-doi="10.1016/j.tig.2006.04.001"><li><a href="http://dx.doi.org/10.1016/j.tig.2006.04.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Harvesting+the+fruit+of+the+human+mtDNA+tree." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Harvesting+the+fruit+of+the+human+mtDNA+tree.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000557-Yu1" id="pone.0000557-Yu1"></a>Yu N, Chen FC, Ota S, Jorde LB, Pamilo P, et al.  (2002) Larger genetic differences within africans than between Africans and Eurasians. Genetics  161: 269–274.  <ul class="find" data-citedArticleID="961396"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Larger+genetic+differences+within+africans+than+between+Africans+and+Eurasians.&amp;auth=&amp;atitle=Larger+genetic+differences+within+africans+than+between+Africans+and+Eurasians." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Larger+genetic+differences+within+africans+than+between+Africans+and+Eurasians." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Larger+genetic+differences+within+africans+than+between+Africans+and+Eurasians.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000557-Prugnolle1" id="pone.0000557-Prugnolle1"></a>Prugnolle F, Manica A, Balloux F (2005) Geography predicts neutral genetic diversity of human populations. Curr Biol  15: 159–160.  <ul class="find" data-citedArticleID="961368" data-doi="10.1016/j.cub.2005.02.038"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.02.038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Geography+predicts+neutral+genetic+diversity+of+human+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Geography+predicts+neutral+genetic+diversity+of+human+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000557-Ramachandran1" id="pone.0000557-Ramachandran1"></a>Ramachandran S, Deshpande O, Roseman CC, Rosenberg NA, Feldman MW, et al.  (2005) Support from the relationship of genetic and geographic distance in human populations for a serial founder effect originating in Africa. Proc Natl Acad Sci U S A  102: 15942–15947.  <ul class="find" data-citedArticleID="961370" data-doi="10.1073/pnas.0507611102"><li><a href="http://dx.doi.org/10.1073/pnas.0507611102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Support+from+the+relationship+of+genetic+and+geographic+distance+in+human+populations+for+a+serial+founder+effect+originating+in+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Support+from+the+relationship+of+genetic+and+geographic+distance+in+human+populations+for+a+serial+founder+effect+originating+in+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000557-Liu1" id="pone.0000557-Liu1"></a>Liu H, Prugnolle F, Manica A, Balloux F (2006) A geographically explicit genetic model of worldwide human-settlement history. Am J Hum Genet  79: 230–237.  <ul class="find" data-citedArticleID="961352" data-doi="10.1086/505436"><li><a href="http://dx.doi.org/10.1086/505436" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+geographically+explicit+genetic+model+of+worldwide+human-settlement+history." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+geographically+explicit+genetic+model+of+worldwide+human-settlement+history.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000557-McDougall1" id="pone.0000557-McDougall1"></a>McDougall I, Brown FH, Fleagle JG (2005) Stratigraphic placement and age of modern humans from Kibish, Ethiopia. Nature  433: 733–736.  <ul class="find" data-citedArticleID="961354" data-doi="10.1038/nature03258"><li><a href="http://dx.doi.org/10.1038/nature03258" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stratigraphic+placement+and+age+of+modern+humans+from+Kibish%2C+Ethiopia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stratigraphic+placement+and+age+of+modern+humans+from+Kibish%2C+Ethiopia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000557-Mellars1" id="pone.0000557-Mellars1"></a>Mellars P (2006) Going East: new genetic and archaeological perspectives on the modern human colonization of Eurasia. Science  313: 796–800.  <ul class="find" data-citedArticleID="961356" data-doi="10.1126/science.1128402"><li><a href="http://dx.doi.org/10.1126/science.1128402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Going+East%3A+new+genetic+and+archaeological+perspectives+on+the+modern+human+colonization+of+Eurasia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Going+East%3A+new+genetic+and+archaeological+perspectives+on+the+modern+human+colonization+of+Eurasia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000557-Conrad1" id="pone.0000557-Conrad1"></a>Conrad DF, Jakobsson M, Coop G, Wen X, Wall JD, et al.  (2006) A worldwide survey of haplotype variation and linkage disequilibrium in the human genome. Nat Genet  38: 1251–1260.  <ul class="find" data-citedArticleID="961338" data-doi="10.1038/ng1911"><li><a href="http://dx.doi.org/10.1038/ng1911" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+worldwide+survey+of+haplotype+variation+and+linkage+disequilibrium+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+worldwide+survey+of+haplotype+variation+and+linkage+disequilibrium+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.XML" value="90756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.PDF" value="260407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.e001.PNG" value="404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.e001.TIF" value="20884"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.e002.PNG" value="404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.e002.TIF" value="21228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t001.PNG_L" value="110413"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t001.PNG_M" value="39895"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t001.PNG_S" value="2655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t001.TIF" value="509552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t001.PNG_I" value="17281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.e003.PNG" value="404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.e003.TIF" value="21228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t002.PNG_L" value="49748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t002.PNG_M" value="48319"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t002.PNG_S" value="2707"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t002.TIF" value="252718"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t002.PNG_I" value="22020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g001.PNG_L" value="154214"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g001.PNG_M" value="33502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g001.PNG_S" value="2391"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g001.TIF" value="461476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g001.PNG_I" value="16619"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t003.PNG_L" value="76904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t003.PNG_M" value="80870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t003.PNG_S" value="4302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t003.TIF" value="418288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t003.PNG_I" value="35636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g002.PNG_L" value="939421"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g002.PNG_M" value="107212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g002.PNG_S" value="10916"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g002.TIF" value="1273136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g002.PNG_I" value="166446"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g003.PNG_L" value="52208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g003.PNG_M" value="71590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g003.PNG_S" value="2997"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g003.TIF" value="477102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.g003.PNG_I" value="30796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t004.PNG_L" value="111105"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t004.PNG_M" value="118629"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t004.PNG_S" value="5561"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t004.TIF" value="484282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.t004.PNG_I" value="52071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.s001.DOC" value="33792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.s002.EPS" value="7638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000557.s003.EPS" value="6956"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000557&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000557" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000557&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000557&volume=&issue=&title=A Worldwide Phylogeography for the Human X Chromosome&author_name=Simone%20S.%20Santos-Lopes%2C%20Rinaldo%20W.%20Pereira%2C%20Ian%20J.%20Wilson%2C%20S%C3%A9rgio%20D.J.%20Pena&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000557" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000557" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000557" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000557&amp;t=A%20Worldwide%20Phylogeography%20for%20the%20Human%20X%20Chromosome" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000557&title=A%20Worldwide%20Phylogeography%20for%20the%20Human%20X%20Chromosome&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000557&amp;title=A%20Worldwide%20Phylogeography%20for%20the%20Human%20X%20Chromosome" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000557" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000557&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Worldwide Phylogeography for the Human X Chromosome';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000557';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Worldwide%20Phylogeography%20for%20the%20Human%20X%20Chromosome http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000557" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000557" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Africa%22" title="Search for articles in the subject area:'Africa'"><div class="flagText">Africa</div></a>
              <div data-categoryid="40287" data-articleid="23952"
                   data-categoryname="Africa"
                   class="flagImage" title="Flag 'Africa' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alu+elements%22" title="Search for articles in the subject area:'Alu elements'"><div class="flagText">Alu elements</div></a>
              <div data-categoryid="20809" data-articleid="23952"
                   data-categoryname="Alu elements"
                   class="flagImage" title="Flag 'Alu elements' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Asia%22" title="Search for articles in the subject area:'Asia'"><div class="flagText">Asia</div></a>
              <div data-categoryid="40353" data-articleid="23952"
                   data-categoryname="Asia"
                   class="flagImage" title="Flag 'Asia' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplotypes%22" title="Search for articles in the subject area:'Haplotypes'"><div class="flagText">Haplotypes</div></a>
              <div data-categoryid="32131" data-articleid="23952"
                   data-categoryname="Haplotypes"
                   class="flagImage" title="Flag 'Haplotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="23952"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oceania%22" title="Search for articles in the subject area:'Oceania'"><div class="flagText">Oceania</div></a>
              <div data-categoryid="41137" data-articleid="23952"
                   data-categoryname="Oceania"
                   class="flagImage" title="Flag 'Oceania' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="23952"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22X+chromosomes%22" title="Search for articles in the subject area:'X chromosomes'"><div class="flagText">X chromosomes</div></a>
              <div data-categoryid="17587" data-articleid="23952"
                   data-categoryname="X chromosomes"
                   class="flagImage" title="Flag 'X chromosomes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6544'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2253'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5667&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=10589">Referee Comments: Referee 3</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=10565">Referee Comments: Referee 1 (David G Smith)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
